
<html lang="en"     class="pb-page"  data-request-id="8ab1492e-cff9-4499-a934-973737927c3f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/jm2014698;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2012.55.issue-6"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Implications of Promiscuous Pim-1 Kinase Fragment Inhibitor Hydrophobic Interactions for Fragment-Based Drug Design" /></meta><meta name="dc.Creator" content="Andrew C.  Good" /></meta><meta name="dc.Creator" content="Jinyu  Liu" /></meta><meta name="dc.Creator" content="Bradford  Hirth" /></meta><meta name="dc.Creator" content="Gary  Asmussen" /></meta><meta name="dc.Creator" content="Yibin  Xiang" /></meta><meta name="dc.Creator" content="Hans-Peter  Biemann" /></meta><meta name="dc.Creator" content="Kimberly A.  Bishop" /></meta><meta name="dc.Creator" content="Trisha  Fremgen" /></meta><meta name="dc.Creator" content="Maria  Fitzgerald" /></meta><meta name="dc.Creator" content="Tatiana  Gladysheva" /></meta><meta name="dc.Creator" content="Annuradha  Jain" /></meta><meta name="dc.Creator" content="Katherine  Jancsics" /></meta><meta name="dc.Creator" content="Markus  Metz" /></meta><meta name="dc.Creator" content="Andrew  Papoulis" /></meta><meta name="dc.Creator" content="Renato  Skerlj" /></meta><meta name="dc.Creator" content="J. David  Stepp" /></meta><meta name="dc.Creator" content="Ronnie R.  Wei" /></meta><meta name="dc.Description" content="We have studied the subtleties of fragment docking and binding using data generated in a Pim-1 kinase inhibitor program. Crystallographic and docking data analyses have been undertaken using inhibi..." /></meta><meta name="Description" content="We have studied the subtleties of fragment docking and binding using data generated in a Pim-1 kinase inhibitor program. Crystallographic and docking data analyses have been undertaken using inhibi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 5, 2012" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm2014698" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2012 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm2014698" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm2014698" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm2014698" /></link>
        
    
    

<title>Implications of Promiscuous Pim-1 Kinase Fragment Inhibitor Hydrophobic Interactions for Fragment-Based Drug Design | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm2014698" /></meta><meta property="og:title" content="Implications of Promiscuous Pim-1 Kinase Fragment Inhibitor Hydrophobic Interactions for Fragment-Based Drug Design" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0011.jpeg" /></meta><meta property="og:description" content="We have studied the subtleties of fragment docking and binding using data generated in a Pim-1 kinase inhibitor program. Crystallographic and docking data analyses have been undertaken using inhibitor complexes derived from an in-house surface plasmon resonance (SPR) fragment screen, a virtual needle screen, and a de novo designed fragment inhibitor hybrid. These investigations highlight that fragments that do not fill their binding pocket can exhibit promiscuous hydrophobic interactions due to the lack of steric constraints imposed on them by the boundaries of said pocket. As a result, docking modes that disagree with an observed crystal structure but maintain key crystallographically observed hydrogen bonds still have potential value in ligand design and optimization. This observation runs counter to the lore in fragment-based drug design that all fragment elaboration must be based on the parent crystal structure alone." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm2014698"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm2014698">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm2014698&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm2014698&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm2014698&amp;href=/doi/10.1021/jm2014698" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2012</span><span class="cit-fg-volume">, 55</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2641-2648</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/55/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm201454n" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm201482p" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Implications of Promiscuous Pim-1 Kinase Fragment Inhibitor Hydrophobic Interactions for Fragment-Based Drug Design<a class="ref internalNav" href="#notes-1" aria-label=""></a></span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+C.++Good">Andrew C. Good</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jinyu++Liu">Jinyu Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bradford++Hirth">Bradford Hirth</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gary++Asmussen">Gary Asmussen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yibin++Xiang">Yibin Xiang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hans-Peter++Biemann">Hans-Peter Biemann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kimberly+A.++Bishop">Kimberly A. Bishop</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Trisha++Fremgen">Trisha Fremgen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Maria++Fitzgerald">Maria Fitzgerald</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tatiana++Gladysheva">Tatiana Gladysheva</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Annuradha++Jain">Annuradha Jain</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Katherine++Jancsics">Katherine Jancsics</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Markus++Metz">Markus Metz</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew++Papoulis">Andrew Papoulis</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Renato++Skerlj">Renato Skerlj</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+David++Stepp">J. David Stepp</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ronnie+R.++Wei">Ronnie R. Wei</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>â¡</sup>Department of Medicinal Chemistry, <sup>Â§</sup>Department of In Vitro Biology, and <sup>â¥</sup>Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span></div><div class="aff-info" id="aff4"><span class="aff-symbol">â¥</span> <span class="aff-text">Department of Therapeutic Protein Research, Genzyme Corp., 1 Mountain Road, Framingham, Massachusetts 01701, United States </span></div><div class="aff-info" id="aff5"><span class="aff-symbol">#</span> <span class="aff-text">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts 01701, United States </span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#3859565c4a5d4f165f57575c785f5d564241555d165b5755"><span class="__cf_email__" data-cfemail="2a4b444e584f5d044d45454e6a4d4f445053474f04494547">[emailÂ protected]</span></a>. Tel: 781 464 3831. Fax: 781 290 5890.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm2014698&amp;href=/doi/10.1021%2Fjm2014698" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2012</span></span><span class="cit-volume">, 55</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2641â2648</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 16, 2012</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 October 2011</li><li><span class="item_label"><b>Published</b> online</span>5 March 2012</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 March 2012</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm2014698" title="DOI URL">https://doi.org/10.1021/jm2014698</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2012 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2641%26pageCount%3D8%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAndrew%2BC.%2BGood%252C%2BJinyu%2BLiu%252C%2BBradford%2BHirth%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D55%26issueNum%3D6%26contentID%3Djm2014698%26title%3DImplications%2Bof%2BPromiscuous%2BPim-1%2BKinase%2BFragment%2BInhibitor%2BHydrophobic%2BInteractions%2Bfor%2BFragment-Based%2BDrug%2BDesign%26numPages%3D8%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2648%26publicationDate%3DMarch%2B2012">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm2014698"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2702</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">23</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm2014698" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Implications of Promiscuous Pim-1 Kinase Fragment Inhibitor Hydrophobic Interactions for Fragment-Based Drug Design&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;C. Good&quot;},{&quot;first_name&quot;:&quot;Jinyu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Bradford&quot;,&quot;last_name&quot;:&quot;Hirth&quot;},{&quot;first_name&quot;:&quot;Gary&quot;,&quot;last_name&quot;:&quot;Asmussen&quot;},{&quot;first_name&quot;:&quot;Yibin&quot;,&quot;last_name&quot;:&quot;Xiang&quot;},{&quot;first_name&quot;:&quot;Hans-Peter&quot;,&quot;last_name&quot;:&quot;Biemann&quot;},{&quot;first_name&quot;:&quot;Kimberly&quot;,&quot;last_name&quot;:&quot;A. Bishop&quot;},{&quot;first_name&quot;:&quot;Trisha&quot;,&quot;last_name&quot;:&quot;Fremgen&quot;},{&quot;first_name&quot;:&quot;Maria&quot;,&quot;last_name&quot;:&quot;Fitzgerald&quot;},{&quot;first_name&quot;:&quot;Tatiana&quot;,&quot;last_name&quot;:&quot;Gladysheva&quot;},{&quot;first_name&quot;:&quot;Annuradha&quot;,&quot;last_name&quot;:&quot;Jain&quot;},{&quot;first_name&quot;:&quot;Katherine&quot;,&quot;last_name&quot;:&quot;Jancsics&quot;},{&quot;first_name&quot;:&quot;Markus&quot;,&quot;last_name&quot;:&quot;Metz&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;Papoulis&quot;},{&quot;first_name&quot;:&quot;Renato&quot;,&quot;last_name&quot;:&quot;Skerlj&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;David Stepp&quot;},{&quot;first_name&quot;:&quot;Ronnie&quot;,&quot;last_name&quot;:&quot;R. Wei&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2012&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;55&quot;,&quot;pages&quot;:&quot;2641-2648&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm2014698&quot;},&quot;abstract&quot;:&quot;We have studied the subtleties of fragment docking and binding using data generated in a Pim-1 kinase inhibitor program. Crystallographic and docking data analyses have been undertaken using inhibitor complexes derived from an in-house surface plasmon resonance (SPR) fragment screen, a virtual needle screen, and a de novo designed fragment inhibitor hybrid. These investigations highlight that fragments that do not fill their binding pocket can exhibit promiscuous hydrophobic interactions due to the lack of steric constraints imposed on them by the boundaries of said pocket. As a result, docking modes that disagree with an observed crystal structure but maintain key crystallographically observed hydrogen bonds still have potential value in ligand design and optimization. This observation runs counter to the lore in fragment-based drug design that all fragment elaboration must be based on the parent crystal structure alone.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm2014698&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm2014698" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm2014698&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm2014698" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm2014698&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm2014698" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm2014698&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm2014698&amp;href=/doi/10.1021/jm2014698" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm2014698" /></input><a href="/doi/pdf/10.1021/jm2014698" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm2014698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm2014698%26sid%3Dliteratum%253Aachs%26pmid%3D22339127%26genre%3Darticle%26aulast%3DGood%26date%3D2012%26atitle%3DImplications%2Bof%2BPromiscuous%2BPim-1%2BKinase%2BFragment%2BInhibitor%2BHydrophobic%2BInteractions%2Bfor%2BFragment-Based%2BDrug%2BDesign%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D55%26issue%3D6%26spage%3D2641%26epage%3D2648%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292580" title="Binding modes">Binding modes</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/55/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jmcmar.2012.55.issue-6/production/jmcmar.2012.55.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We have studied the subtleties of fragment docking and binding using data generated in a Pim-1 kinase inhibitor program. Crystallographic and docking data analyses have been undertaken using inhibitor complexes derived from an in-house surface plasmon resonance (SPR) fragment screen, a virtual needle screen, and a de novo designed fragment inhibitor hybrid. These investigations highlight that fragments that do not fill their binding pocket can exhibit promiscuous hydrophobic interactions due to the lack of steric constraints imposed on them by the boundaries of said pocket. As a result, docking modes that disagree with an observed crystal structure but maintain key crystallographically observed hydrogen bonds still have potential value in ligand design and optimization. This observation runs counter to the lore in fragment-based drug design that all fragment elaboration must be based on the parent crystal structure alone.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote withdrawn"><h4><span>â </span>Â Accession Codes</h4><p class="last">Crystal structure PDB IDs for inhibitors disclosed in this paper: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBQ">3VBQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBT">3VBT</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBV">3VBV</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBW">3VBW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBX">3VBX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBY">3VBY</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VC4">3VC4</a>.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15226" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15226" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The accepted wisdom around fragment-based drug design (FBDD) is that the challenges inherent to docking fragments render crystal structure data critical for ligand design. It is also generally accepted that fragments are harder to dock relative to larger drug sized molecules. More recent studies focusing on this issue, however, suggest that there is little difference in docking performance between the two.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> These investigations also surmise that incorrect docking of fragments tends to occur due to incorrect scoring, while for druglike molecules sampling issues are the primary problem. The scoring issue appears to be due to the fact that small energy differences often exist between fragment binding modes.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> These differences are beyond the ranking resolution of existing scoring functions to differentiate, even when sampling is sufficient to find binding modes close to the observed crystal structure.</div><div class="NLM_p">We have investigated these issues in the context of a legacy FBDD program run within Genzyme. The Pim family of serine/threonine kinases has been implicated as having a functional role in cell survival.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> This has provided a general mechanism for the potential of Pim kinases as targets in oncology and immunology regulation.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4-6)</a> Pim kinases are transcriptionally activated by FLT3,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> rendering them constitutively active.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In addition, Pim kinase structures benefit from the presence of an unusual proline in the hinge region of ATP binding site, which both expands the ATP pocket and alters the kinase hinge hydrogen-bonding motif.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> As a result of these features, the potential for finding selective small-molecule inhibitors for the Pim family relative to other kinases is significantly enhanced. On the basis of these and other data, Pim-1 was selected as a Genzyme oncology target and a drug discovery program initiated. Our primary strategy involved the application of fragment-based screening using surface plasmon resonance (SPR) to find early hit molecules, followed by X-ray structure guided medicinal chemistry optimization. The screen yielded a number of useful hits, one of which was rapidly optimized to high potency.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> A key facet of our drug discovery approach has been to continue focusing some resource on broadening our chemotype SAR even when entering hit/lead optimization. With that in mind, an analysis of the Genzyme screening deck was undertaken to find additional fragment-like inhibitors. The high concentration screening of such âneedleâ compounds in biochemical assays was pioneered at Roche,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and the technique has since been adapted to allow for high concentration screening of low complexity molecule collections in parallel with high throughput screening (HTS).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> For this work, we returned to the original approach favored by Boehm et al.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> in which a 3D-constrained target-biased subset of compounds was chosen for screening from the available needles. Pim-1 was considered appropriate for this approach given the number of SPR fragment hits we found with potencies sensitive to a standard biochemical screen (IC<sub>50</sub> < 125 Î¼M).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p last">In concert with these hit-finding efforts, we leveraged our understanding of the key components of Pim-1 binding to explore potential new lead chemotype opportunities. These studies focused on an interesting Pim-1 screening hit for which a postulated binding mode had been published in what we deemed to be an incorrect orientation.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The molecule was redocked and a chimera designed leveraging another Pim-1 inhibitor series postulated to share a similar binding mode.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The results of these studies are described below, with a focus on the docking and crystal structure data gathered and their implications for FBDD.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73207" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73207" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Needle Screening</h3><div class="NLM_p">The Genzyme screening deck was analyzed using a simplified version of the Astex rule of 3 (molecular weight < 300, ClogP â¤ 3, and number of rotatable bonds â¤ 3).<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> A total of 13â888 molecules passed this filter and became part of a virtual needle screening deck for the Glide docking program.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> At the time this screen was run, no in-house crystal structures of Pim-1 were available, so virtual screens were constructed using structures available in the Protein Data Bank (PDB).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> An analysis of available PDB structures highlighted the importance and diversity of hydrogen-bonding interactions in the vicinity of the catalytic domain lysine (K67). The variability comes about due to conformational switching of the c helix E89 residue producing a variety of gatekeeper water molecule hydrogen-bonding motifs. Searches were constructed based on the motifs observed in four Pim-1 inhibitor complexes<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>), taking advantage of Glideâs versatile pharmacophore constraint tools. Hydrogen-bond constraints were focused in the region of K67, with an additional positional constraint allowing aromatic carbons (all inhibitors shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> exhibit such interactions at this position) or hydrogen-bond donors in the vicinity of the E121 hinge carbonyl group (see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for further details). The final selection of 500 compounds was checked for availability, resulting in 462 molecules chosen for screening. The IC<sub>50</sub>s of all molecules showing >50% inhibition at 125 Î¼M were determined, with 46 exhibiting IC<sub>50</sub>s below 100 Î¼M (the full structure list is shown in <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Hydrogen-bonding networks (green dotted lines for heteroatom distance <3.0 Ã) around catalytic domain K67 highlighted for each of the four structures used in virtual screening calculations. The 2D structures of the parent inhibitors associated with each complex are shown for reference, together with their Pim-1 IC<sub>50</sub>/<i>K</i><sub>i</sub> data and LEs. The structures used are (a) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OBJ">2OBJ</a>,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (b) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGQ">3BGQ</a>,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (c) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGP">3BGP</a>,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and (d) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGZ">3BGZ</a>.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The conformational switching of E89 in structure b and the general variability in positioning and number of water-mediated hydrogen bonds are of particular note. For <b>1a</b> the hydroxypyridine tautomer was hypothesized as preferred due to the presence of a stabilizing internal H bond from the pendant phenolic hydroxyl. It was this tautomer that was used in protein preparation. If the generally preferred pyridone tautomer is used instead, the pendant phenolic hydroxyl hydrogen can no longer form either an inter- or intramolecular hydrogen bond, as it cannot pair with the gatekeeper water. This leads to an unpaired hydrogen bond donor exposed to a hydrophobic environment. We suspect that the internal hydrogen bond and the ortho cyano substituent either stabilize this tautomer or allow the formation of the phenolate anion given the preference of this active site region to bind acidic functionality.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Six molecules were repurchased and recrystallized with the target both to confirm binding and determine binding mode. Comparisons between the crystallized and Glide docked structures of five of these molecules are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> (the sixth molecule was an analogue of <b>2c</b> and was found to exhibit the same binding mode. It was thus excluded as it provides little additional insight).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Top scoring docked structures (orange ligand and pink protein) and crystal structure (green ligand and purple protein) for five hits from the needle screen. The reference structure is taken from the top score generating reference structure in each case. Note that water molecules associated with each reference structure are shown with pink hydrogens. 2D structures, Pim-1 IC<sub>50</sub>s, and LEs are shown for each ligand and key hydrogen bonds are highlighted (green dotted lines for heteroatom distance < 3.0 Ã, orange for distance â¼ 3.2 Ã). The PDB codes of the crystal structure of these compounds are as follows: (a) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBW">3VBW</a>, (b) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VBV">2VBV</a>, (c) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBX">3VBX</a>, (d) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBY">3VBY</a>, and (e) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBT">3VBT</a>. Note that <b>2a</b> may exist as an enolate tautomer; however, this is not how it was modeled in the virtual screening studies. Note also that for <b>2c</b> an alternate location is included for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=D186">D186</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Novel Lead Design</h3><div class="NLM_p">A survey of the literature highlighted <b>3</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>), a Pim-1 thiazolidinedione (THZ) screening hit with impressive potency and ligand efficiency (LE).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The accompanying docking study suggested that the THZ hit formed a hydrogen bond between the amide hydrogen and the hinge carbonyl group of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=E121">E121</a>, while simultaneously interacting with the hydrophobic pocket under the P loop (F49). The primary issue with this docking calculation is that the THZ is protonated, while THZs are known to function as carboxylate mimics in PPARÎ³ agonists.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> We had already found a number of carboxylate-containing fragment hits in our original SPR fragment screen<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (molecules <b>4a</b> and <b>4b</b>, Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>), and this led us to believe that the THZ was most likely deprotonated on binding. With this in mind, <b>3</b> was redocked using Glide [standard precision (SP) and extra precision (XP)] into the crystal structure of <b>4a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R00">3R00</a>), the most ligand efficient acid discovered in the SPR screen. The Glide SP results were studied in depth as they both reproduced the expected hydrogen-bonding interactions and returned multiple binding modes for visual inspection. The top scoring docking modes are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>. An alternate docking (the fifth docking mode in the Glide SP calculation) is shown in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>. At this point a new Pim-1 structure was deposited in the PDB based on a novel series of carboxylate inhibitors determined in an HTS screen (compound <b>5</b>, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F2A">3F2A</a>; see Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Superposition of the alternate docking mode of <b>3</b> with the crystal structure of <b>5</b> suggested the possibility of creating a chimera molecule linking through the trifluoromethyl bond (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). Chimera design based on this superposition using the most potent fragment abstracted from the original reference<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> produced <b>6</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>). The top scoring Glide docking modes for <b>6</b> agreed with the postulated binding mode (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>). Synthesis of <b>6</b> produced an inhibitor of Pim-1 with an IC<sub>50</sub> of 467 pM. The crystal structure of <b>6</b> is shown in Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>. Note that the crystal structure binding mode of <b>3</b> is also shown in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>. This was solved subsequent to the design and synthesis of <b>6</b>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Pim-1 THZ screening hit<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> used in chimera lead design with Pim-1 IC<sub>50</sub> and LE shown for reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structures of carboxylate-containing hits from internal SPR fragment screen<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> [PDB reference codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R00">3R00</a> (<b>4a</b>) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R01">3R01</a> (<b>4b</b>)]. Key hydrogen bonds are highlighted with Pim-1 IC<sub>50</sub>s and LEs shown for reference. Note that the protein structures of these complexes have essentially identical tertiary structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. THZ crystal structure of <b>3</b> (pink) versus unconstrained Glide XP/SP highest scoring docking modes (identical and in orange), with <b>4b</b> crystal structure (green) of carboxylate-containing hit shown for reference. Proteins of both crystal structures are also essentially superimposable, so only the protein structure of <b>4b</b> has been included. The crystal structure of <b>3</b> has been deposited in the PDB using code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VC4">3VC4</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Alternate Glide SP docking mode for deprotonated model of <b>3</b> (orange) superimposed on reference crystal structure <b>4a</b> (green) used in docking.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Alternate Glide SP docking mode of <b>3</b> (orange) superimposed on crystal structure of molecule <b>5</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F2A">3F2A</a>). The protein reference structure shown is also derived from <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F2A">3F2A</a>. Pim-1 IC<sub>50</sub> and LE of <b>5</b> shown for reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Crystal structure of <b>6</b> (green) superimposed on constrained Glide SP (orange), unconstrained Glide SP (pink), and Glide XP (maroon) top scoring binding modes on docking to the crystal structure of <b>5</b> (key hydrogen bonds highlighted). The crystal structure of <b>6</b> has been deposited in the PDB using code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBQ">3VBQ</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21312" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21312" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The crystal structures for the needle screen hits shown in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> highlight all the classic pitfalls that can trip up docking calculations. Protein residues and water molecule positions in the structure around <b>2a</b> match those of the structure used for docking (from protein complex of molecule <b>1a</b>) closely, and the resulting docking mode is a close match to the crystal structure (rmsd 0.88 Ã). The gatekeeper water interaction between the crystal structure of <b>2b</b> and its docking reference structure (<b>1c</b>) is also well-matched, resulting in a good superposition of amide hydrogen-bonding interactions between docked and crystallographic binding modes. The inability of docking calculations to deal with shifting water molecule positions confounds the positioning of the hydroxyquinoline ring, however. The docking model postulates an internal hydrogen bond with the hinge carbonyl group of E121, while in reality the phenolic hydroxyl forms an internal water-mediated hydrogen bond and flips to interact with F49 on the P loop. This is perhaps not surprising, as the preference for aromatic CH interactions with the hinge carbonyl group has precedence<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and is well-documented in existing Pim-1 inhibitors (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). As a consequence, while the key hydrogen-bonding motif constraint is correctly matched, the rmsd is still 2.6 Ã between docked and crystallographic binding modes. To test the effects of the constraints on the search, an unconstrained Glide SP calculation was also run on <b>2b</b> in the <b>1c</b> active site. This search yields the same top scoring docking mode as seen in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>, showing that the presence of the docked mode hinge interaction is not due solely to the positional constraint used. The closest rmsd (which is the second top scorer) found in the constrained searches is 1.1 Ã (Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>, pink structure), with the quinoline rings nearly superimposed but with the hydrogen-bonding interactions completely disrupted. The highest scoring docking mode for <b>2c</b> (<b>1b</b> used for docking) is found to superimpose well with the crystal with an rmsd of 1.0 Ã. This is a case where a good docking mode is obtained despite the changes occurring around the ligand on binding. E89 shifts position to accommodate no less than 3 water molecules in the gatekeeper pocket, fortunately without disrupting the position of the key water-mediated hydrogen bond. In addition, D186 lifts up in the site to form a water-mediated hydrogen bond with the hydroxyl off the back of the coumarin ring, again with little effect on the binding mode. A slight shift in hydrogen-bonding interactions is seen, with the docking mode seemingly overemphasizing the potential for hydrogen bonding with the ester oxygen of the coumarin ring. Instead, the molecule shifts to permit a bifurcated interaction between K67 and the gatekeeper water molecule. Molecule <b>2d</b> produces shifts in the active site that begin to confound the docking calculation (to <b>1b</b>) to a significant degree. In particular, the side chain of K67 lifts up in the active site to allow the furan ring to sit in a hydrophobic sandwich between the carbon side chain atoms of K67 and D186. This interaction is impossible to replicate in any of the sites used for docking and a high rmsd ensues (2.9 Ã). Nevertheless, the key water-mediated hydrogen bond to the carbonyl oxygen is essentially maintained, but as in the case of <b>2b</b>, the formation of a hydrogen bond to the hinge (this time through the indole nitrogen) proves irresistible. For molecule <b>2e</b>, we see a complete breakdown in docking mode accuracy both from an rmsd (2.3 Ã) and interaction standpoint. The reference docking structure that produces the highest Glide score (<b>1d</b>) shows no conservation in water-mediated hydrogen-bonding interactions and does not take into account the shift in position of D186, which also forms a key hydrogen bond with pyrazole of <b>2e</b>.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Crystal structure (light green) and the docked binding modes of <b>2b</b> for the four virtual screens run. Ligands have been extracted from their respective binding poses after superposition of the Pim-1 references structures used in docking.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">If we look at the overall performance of the docking calculations, while only two of the highest scoring docked modes are docked correctly in their entirety, four of five systems correctly map the key hydrogen-bonding constraints. This once again highlights the utility of incorporating hydrogen-bonding constraints, even when screening for fragment-sized hits. It also underlines the value of crystal structures given the resolution limits of docking scores, particularly in the context of changes in water molecule binding patterns and side chain shifts.</div><div class="NLM_p">To explore the notion of flat energy surfaces from a more quantitative perspective, the difference in scores between top scoring docked and crystal structure binding modes has also been explored. The use of postscore minimization in enhancing docking accuracy was recently shown to have potential in fragment docking.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> With this in mind and to allow for relaxation of the crystal structure on cross-docking, all docked, cross-docked crystal and native crystal structures were refined using Impref-constrained minimizations (see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>). The resulting binding modes were then rescored using Glide SP. The degree of diversity in the docked poses produced for each hit has been explored by studying both the count of duplicate binding modes and hinge hydrogen-bonding propensity. The results of these studies are shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>. Molecules <b>2a</b> and <b>2e</b> both show little difference in scoring between the cross-docked crystal, native crystal, and virtual screening docked structures. One might expect this given the high level of similarity of the native and docked binding modes. It is nonetheless interesting that, although Glideâs hbond score has increased from â0.29 to â0.49 in the presence of an interstitial water molecule in the native <b>2c</b> structure, there is little overall effect in the Glide SP score. It is also of note that these are the two inhibitors with highest LE, given that the correlation between said efficiency and docking accuracy has been observed previously.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> For <b>2b</b>, the virtual screening score is higher than the cross-docked X-ray crystal structure score by a significant margin. The refined native crystal structure scores the highest, but only when the interstitial water molecule that sits between the carbonyl and phenol groups is retained. If this is removed, the resulting score is similar to the docked mode. This highlights both  potential effects of interstitial water (in contrast to <b>2c</b>), which increases the Glide hbond score from â0.48 to â1.01 while also increasing both Glide reward and Glide Emodel contributions. Molecule <b>2d</b> has no cross-docking scores, as the clash between the crystal structure inhibitor and the docking protein prevents a successful docking mode from being formed. For <b>2e</b>, while the cross-docked crystal inhibitor can be scored, the lack of interactions with D186 in the docking protein relative to the native structure compromises the score, highlighting the potential importance of side chain conformation changes. For <b>2d</b>, the top-scoring docked complex out scores the native binding mode, with a significant part of the difference derived from the hinge hydrogen bond contribution (Glide hbond score â0.61 vs â0.32, respectively). For <b>2e</b>, although the docked and native binding modes are quite distinct, the resulting Glide SP scores are similar, an example of scoring resolution issues when docking fragments. For <b>2b</b> and <b>2d</b>, the two systems for which there are hinge hydrogen bonds observed in the virtual screens, two of four modes for <b>2b</b> and two of three modes for <b>2d</b> show this interaction. Overall, however, there is little duplication in binding modes between the top scoring hits from each virtual screen, with the high diversity of hydrogen-bonding constraints around K67 creating a wide array of binding configurations (Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Additional Docking Data for Hits <b>2a</b>â<b>e</b><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">top docked complex</th><th class="colsep0 rowsep0" align="center">transplanted X-ray ligand</th><th class="colsep0 rowsep0" align="center">native X-ray structure</th><th class="colsep0 rowsep0" align="center">consensus/duplicate/hinge H bond count</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2a</b></td><td class="colsep0 rowsep0" align="left">â7.4</td><td class="colsep0 rowsep0" align="left">â7.9</td><td class="colsep0 rowsep0" align="left">â7.8</td><td class="colsep0 rowsep0" align="left">4/2/0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2b</b></td><td class="colsep0 rowsep0" align="left">â9.0</td><td class="colsep0 rowsep0" align="left">â7.7</td><td class="colsep0 rowsep0" align="left">â8.0Â /â9.3<a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">4/0/2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2c</b></td><td class="colsep0 rowsep0" align="left">â7.1</td><td class="colsep0 rowsep0" align="left">â7.0</td><td class="colsep0 rowsep0" align="left">â7.0</td><td class="colsep0 rowsep0" align="left">2/2/0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2d</b></td><td class="colsep0 rowsep0" align="left">â9.1</td><td class="colsep0 rowsep0" align="left">1000<a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">â8.4</td><td class="colsep0 rowsep0" align="left">3/0/2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2e</b></td><td class="colsep0 rowsep0" align="left">â7.8</td><td class="colsep0 rowsep0" align="left">â5.7</td><td class="colsep0 rowsep0" align="left">â7.7</td><td class="colsep0 rowsep0" align="left">3/0/0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">Columns 2â4 summarize the Glide SP scores of Impref-refined needle screen hit complexes for the top docked ligand, the X-ray ligand binding mode in the top scoring docking complex protein, and the native X-ray structure. Column 4 shows the number of times inhibitor met the â6 score cutoff for the four searches run/the number of duplicate binding modes (RMSD < 0.5 Ã) observed/number of times that a hydrogen bond to hinge was observed.</p></div><div class="footnote" id="t1fn1"><sup>b</sup><p class="last">Scores with and without the presence of water molecule bridging phenol and carbonyl groups.</p></div><div class="footnote" id="t1fn2"><sup>c</sup><p class="last">Calculation contains a significant clash with docking protein.</p></div></div></div><div class="NLM_p last">The results above suggest that while docking is useful for finding hits, crystal structure data are indeed critical for FBDD molecular design. This presupposes, however, that the binding energy differences between crystal and docked modes are generally large enough to render the docking mode useless. A closer look at the binding mode changes seen in the carboxylate- and THZ-containing chemotypes show that this is not always the case. The potential for binding mode shifts inherent in hydrophobic interactions with few shape constraints is nicely illustrated by the benzfuran carboxylate containing analogue SPR hits shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>. While the addition of a methoxy produces a steric-clash-induced ring flip, the lack of additional shape constrained interactions allows this to occur while both key hydrogen bonds and binding potency are maintained. This is a potential feature of fragment binding, since they often do not completely fill an active site, with the result that their shapes are unable to act as a strong directional constraint. This, as well as the importance of understanding key hydrogen-bonding interactions in an active site, is nicely illustrated in the chimera inhibitor design example of <b>6</b>. An understanding of the hydrogen-bonding preferences of Pim-1 led to the correct assertion of the THZ binding of <b>3</b> occurring via K67 and water-mediated hydrogen bonds through a deprotonated headgroup. The top scoring Glide SP docking mode (score â8.4) showed the hydrophobic portion of <b>3</b> binding in a <b>4b</b>-like manner (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). An alternate docking mode of <b>3</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>, score â6.8) places its phenyl group deeper into the pocket (similar to <b>4a</b> and <b>5</b>). Given the conformational flips already observed in the carboxylate-containing inhibitors and the excellent mapping of this docking mode onto <b>5</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>), we decided to pursue the synthesis of <b>6</b>. This proved a wise choice, as the activity of <b>6</b> is over 20 times greater than the most potent compounds from either parent series, with a LE higher than the original lead <b>5</b>. The crystal structure of <b>6</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>), which is in agreement with the Glide docking calculation, confirms the binding hypothesis. Interestingly, the crystal structure of <b>3</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>, solved subsequent to initiation of molecule <b>6</b> synthesis) confirms THZ binding in acidic form but shows a flipped orientation for the hydrophobic portion of the molecule, closer to the top scoring Glide SP docking mode. Glide SP data for <b>3</b> highlights a preference for binding the trifluoromethyl along rather than toward the hinge. This is illustrated by the fact that two of the ten Glide SP docking modes are within 1 Ã rmsd of the crystal structure but score significantly lower (â6.8 and â6.8 versus â8.4). These results are consistent with the observations of Verdonk et al., which point to scoring rather than sampling as the biggest obstacle to accurate fragment docking.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> As a consequence, molecular design based on docking derived binding modes remains a technique worthy of consideration when used appropriately.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31718" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31718" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have used data extracted from our Pim-1 project efforts to further investigate the subtleties of fragment inhibitor binding. The success of our needle screen at retrieving actives and the ability of the constrained docking calculations used to reproduce the crystallographically observed primary hydrogen-bonding patterns are noteworthy. It is nevertheless clear that, as one might expect, crystal complexes are still key to understanding the subtleties of fragment binding, particularly in regard to new water molecule interactions and side chain shifts. When water and side chain sampling are not an issue, our data point toward fragment scoring rather than binding mode sampling as the primary docking limitation, in keeping with earlier studies. Our subsequent chimera inhibitor design study suggests, however, that fragment docking modes for which hydrophobic interactions are in disagreement with the parent crystal structure may still have value when key hydrogen bonds are maintained. While the design was based on a docking mode for which the hydrophobic interactions were found to be incorrect by crystallography, the predicted binding mode of the resultant highly potent chimera was nonetheless confirmed. The data presented here highlight the low level of hydrophobic interaction directionality possible when fragment hits do not completely fill the pocket to which they bind. This lack of directionality can both confound docking scores and provide additional design opportunities based on alternate âincorrectâ docking poses during fragment elaboration.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Modeling</h3><div class="NLM_p">All virtual screening and docking experiments were carried out within the Maestro suite of software.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> All protein complexes were prepared using the protein preparation wizard workflow by applying default settings (preprocess keeping water molecules within 5 Ã of ligand, water optimization sampling with interactive optimization of gatekeeper water molecule hydrogen positions where appropriate, followed by Impref refinement to 0.3 Ã rmsd). Glide<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> was used to perform all docking experiments by applying the default calculation settings. The reference proteins used in each calculation are detailed in the <a class="ref internalNav" href="#sec3" aria-label="Discussion">Discussion</a> and <a class="ref internalNav" href="#sec2" aria-label="Results">Results</a> sections. Glide SP was used for all virtual screening experiments, while both Glide XP and Glide SP were applied to individual compound docking studies. For each virtual screen hydrogen-bonding constraints were placed on K67 and any relevant gatekeeper water molecules shown to create water-mediated hydrogen bonding with the complexed inhibitor. In addition, a 1.0 Ã radius positional constraint was added on the basis of the position of the nearest inhibitor aromatic carbon atom of each complex to the inner hinge backbone carbonyl group of E121, with aromatic carbon atoms and heteroatom hydrogen bond donors being allowed to map the constraint. For each search, the positional constraint and a hydrogen bond match up to either the gatekeeper water molecule and/or K67 were required for a total of eight searches. Comparison of the âandâ versus âorâ hydrogen bond constraint searches showed significant overlap of results of compound binding modes and selections, so the âandâ searches were dropped from the analysis. The top scoring hit for each molecule was retained for each virtual screen run, and this resulted in a total of 1927 hits scoring â6 or less using Glide SP.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> This score was used to define a threshold for general relevance, which from our perspective was the limit of its resolution given all the uncertainties inherent in docking. The hit lists were combined and reordered multiple times using a number of the scoring columns provided by the Glide calculation (docking score, glide lipo, glide h bond, and glide ligand efficiency). In each case, the hit lists were then analyzed visually within Vida, which is well-suited for rapid visual analysis of large data sets.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Compounds were selected primarily on the basis of the observed diversity of hydrogen bond interaction(s) in the vicinity of K67, together with an experienced eye âthis looks reasonable and interestingâ criteria. Multiple analogues were chosen for particularly interesting chemotypes to hedge against potential purity issues, given that the compounds were taken from DMSO stock rather than direct from solid as is the case for fragment deck screens. Selections were also focused on compounds appearing in the hit list multiple times (i.e., with a Glide score < â6 in multiple searches), providing a qualitative consensus scoring bias. All database molecules were prepared using Ligprep<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> with tautomer probability set to 30, maximum tautomers equal to 4, maximum stereoisomers set to 2, with a maximum of two ring conformations being equal using idealized geometries and both neutral and charged molecule generation permitted.</div><div class="NLM_p">For the inhibitor docked and crystal structure binding mode comparisons in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, the crystal structure protein and the parent virtual screening protein for the top scoring docking mode of each hit (<b>2a</b>â<b>e</b>) were superimposed. The crystal structure of each inhibitor was then abstracted into the virtual screening protein to create a cross-docking mode of each hit crystal structure. This then underwent a constrained minimization using the Impref routine in the protein preparation wizard of Maestro 9.2.0.15<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (maximum heavy atom rmsd shift permitted during minimization set to 0.3 Ã using the OPLS 2005 force field). The same Impref routine was also used to minimize each top scoring docked pose to provide a consistent frame of reference. In the same vein each crystal structure also underwent a full protein preparation (including an Impref refinement). All the resulting refined conformations were then rescored using Glide SP (version 57109).</div><div class="NLM_p last">For the chimera design, <b>3</b> was docked with the THZ deprotonated. For this study the top 10 scoring ligand binding poses were retained from the Glide SP docking experiment with each analyzed visually.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Chemistry</h3><div class="NLM_p last">All solvents and reagents were used as purchased from various commercial sources. Reactions were conducted under a dry nitrogen atmosphere unless otherwise indicated. Chromatography was carried out using a Teledyne Isco Combiflash R<sub>f</sub> system employing prepacked RediSep silica cartridges. <sup>1</sup>H and <sup>13</sup>C NMR spectra were acquired on a Bruker Ultrashield 400 instrument operated at 400 and 100 MHz, respectively. NMR samples were prepared as CDCl<sub>3</sub> or DMSO-<i>d</i><sub>6</sub> solutions and referenced as follows: <sup>1</sup>H NMR, TMS (Î´ = 0.00 ppm); <sup>13</sup>C NMR, DMSO-<i>d</i><sub>6</sub> (Î´ = 39.52 ppm). The purity of <b>6</b> was determined using a Waters Acquity UPLC system equipped with an Acquity PDA detector and in-line Waters ZQ2000 mass spectrometer (ESI). UPLC conditions were as follows: column, Acquity UPLC BEH C18 (2.1 Ã 50 mm, 1.7 Î¼m); column temperature, 50 Â°C; flow rate, 1.0 mL/min; mobile phase A, 0.1% formic acid in water; mobile phase B, 0.1% formic acid in acetonitrile; gradient, 5%â95% B (0â1.9 min), 95% B (1.9â2.2 min), 95%â5% B (2.2â2.3 min), 5%â0% B (2.3â10 min). Purity was calculated as a percentage of total area at 210 and 254 nm.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> (<i>Z</i>)-5-(3-(6-((<i>trans</i>-4-Aminocyclohexyl)amino)pyrazin-2-yl)benzylidene)thiazolidine-2,4-dione 2,2,2-Trifluoroacetate (Compound 6)</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Step 1</h4><div class="NLM_p last">To a stirred solution of <i>trans</i>-1,4-diaminocyclohexane (3.90 g, 34.1 mmol) and 2,6-dichloropyrazine (2.04 g, 13.7 mmol) in 1,4-dioxane (100 mL) was added diisopropylamine (5.9 mL, 34 mmol). The mixture was heated at reflux for 16 h and then concentrated. The residue was taken up in aqueous sodium bicarbonate solution (â¼150 mL) and extracted with dichloromethane (3 Ã â¼100 mL). The combined extracts were dried over sodium sulfate and concentrated. The resulting oil was purified by flash chromatography over silica (5â15% gradient of 2 N ammonia/methanol in methylene chloride) to provide <i>trans</i>-<i>N</i>-(6-chloropyrazin-2-yl)cyclohexane-1,4-diamine as tacky tan solid (2.12 g, 69%): <sup>1</sup>H NMR (CDCl<sub>3</sub>) Î´ 7.75 (s, 1H), 7.70 (s, 1H), 4.66 (d, <i>J</i> = 7.4 Hz, 1H), 3.78â3.56 (m, 1H), 3.47 (s, 3H), 2.81â2.63 (m, 1H), 2.18â2.04 (m, 2H), 2.03â1.83 (m, 2H), 1.63 (s, 5H), 1.38â1.14 (m, 4H) ppm.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Step 2</h4><div class="NLM_p last">To a stirred solution of step 1 product (2.12 g, 9.37 mmol) in methylene chloride (90 mL) was added di-<i>tert</i>-butyl dicarbonate (2.45 g, 11.2 mmol). The solution was stirred at room temperature for 16 h and then washed with aqueous sodium bicarbonate (â¼75 mL), dried over sodium sulfate, and concentrated. The resulting solid was purified by flash chromatography over silica (0â5% gradient of methanol in methylene chloride) to provide <i>tert</i>-butyl (<i>trans</i>-4-((6-chloropyrazin-2-yl)amino)cyclohexyl)carbamate as a tan solid (2.30 g, 75%): <sup>1</sup>H NMR (CDCl<sub>3</sub>) Î´ 7.76 (s, 1H), 7.70 (s, 1H), 4.55 (d, <i>J</i> = 7.9 Hz, 1H), 4.50â4.28 (m, 1H), 3.77â3.61 (m, 1H), 3.61â3.39 (m, 1H), 2.20â2.12 (m, 2H), 2.12â2.02 (m, 2H), 1.45 (s, 9H), 1.36â1.19 (m, 4H) ppm.</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Step 3</h4><div class="NLM_p last">Step 2 product (1.00 g, 3.07 mmol), 3-formylphenylboronic acid (0.506 g, 3.37 mmol), and potassium carbonate (2.97 g, 21.5 mmol) were taken up in DMF (35 mL) and water (10 mL). Nitrogen was bubbled through the stirred mixture for 15 min. Following degassing, bis(triphenylphosphine)palladium(II) dichloride (0.054 g, 0.077 mmol) was added and the mixture was heated at 100 Â°C for 1.5 h. The reaction was then concentrated and the residue partitioned between ethyl acetate (â¼70 mL) and water (â¼100 mL). The organic layer was washed with additional water (2 Ã â¼100 mL), dried over sodium sulfate, and concentrated. The resulting tan solid was purified by flash chromatography over silica (0â5% gradient of methanol in methylene chloride). <i>tert</i>-Butyl (<i>trans</i>-4-((6-(3-formylphenyl)pyrazin-2-yl)amino)cyclohexyl)carbamate was afforded as a light yellow solid (1.12 g, 92%): <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 10.11 (s, 1H), 8.49â8.46 (m, 1H), 8.32 (s, 1H), 8.25â8.21 (m, 1H), 7.96â7.91 (m, 1H), 7.84 (s, 1H), 7.67â7.61 (m, 1H), 4.62 (d, <i>J</i> = 7.6 Hz, 1H), 4.54â4.38 (m, 1H), 3.88â3.74 (m, 1H), 3.59â3.41 (m, 1H), 2.29â2.05 (m, 4H), 1.46 (s, 9H), 1.44â1.28 (m, 4H) ppm.</div></div><div id="sec5_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Step 4</h4><div class="NLM_p last">To a stirred suspension of step 3 product (1.10 g, 2.77 mmol) and 2,4-thiazolidinedione (0.341 g, 2.91 mmol) in <i>tert</i>-butanol (20 mL) was added piperidine (0.22 mL, 2.22 mmol). The mixture was heated at reflux overnight, cooled slightly, and suction filtered to remove suspended solid. The filter cake was rinsed with ethanol (2 Ã â¼5 mL) and diethyl ether (2 Ã â¼15 mL). Drying on the filter frit under house vacuum provided <i>tert</i>-butyl (<i>trans</i>-4-((6-(3-((<i>Z</i>)-(2,4-dioxothiazolidin-5-ylidene)methyl)phenyl)pyrazin-2-yl)amino)cyclohexyl)carbamate as a pale yellow solid (0.862 g, 63%): <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 12.67 (br s, 1H), 8.35â8.31 (m, 1H), 8.29 (s, 1H), 8.13â8.08 (m, 1H), 7.89 (s, 1H), 7.87 (s, 1H), 7.72â7.67 (m, 1H), 7.65â7.60 (m, 1H), 7.14 (d, <i>J</i> = 7.3 Hz, 1H), 6.52 (d, <i>J</i> = 8.0 Hz, 1H), 4.26â4.07 (m, 1H), 3.90â3.72 (m, 1H), 2.15â1.98 (m, 2H), 1.95â1.77 (m, 2H), 1.39 (s, 9H), 1.39â1.23 (m, 4H) ppm.</div></div><div id="sec5_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Step 5</h4><div class="NLM_p last">To a stirred suspension of step 4 product (0.300 g, 0.605 mmol) in methylene chloride (8 mL) was added trifluoroacetic acid (2.0 mL, 26 mmol). After 45 min the reaction was concentrated. The residue was further dried in a vacuum oven (60 Â°C) overnight. The title compound was afforded as a bright yellow solid (0.317 g, 100%): <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 12.70 (br s, 1H), 8.32â8.20 (m, 2H), 8.12â8.03 (m, 1H), 7.98â7.77 (m, 5H), 7.71â7.53 (m, 2H), 7.20 (br s, 1H), 3.87â3.66 (m, 1H), 3.15â2.95 (m, 1H), 2.24â1.88 (m, 4H), 1.58â1.22 (m, 4H) ppm; <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 167.7, 167.3, 158.3 (d, <i>J</i> = 34.3 Hz; TFA), 153.6, 146.9, 137.9, 133.5, 132.8, 131.5, 131.1, 129.7, 127.9, 127.4, 127.2, 124.0, 116.3 (q, <i>J</i> = 294.9 Hz; TFA); UPLC purity 100% (210 nm), 98% (254 nM); retention time, 0.74 min; ESMS <i>m</i>/<i>z</i> 396.2 (M + H).</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Pim-1 DoseâResponse Assay</h3><div class="NLM_p last">Reagents and consumables were purchased from Sigma Aldrich or Caliper Life Sciences. Human Pim-1 was produced internally at Genzyme. All assay reaction conditions for IC<sub>50</sub> determinations were within the linear range with respect to time and enzyme concentration. In a 384-well polypropylene plate, human Pim-1 (1.2 nM) was preincubated with a compound in a 100 mM HepesâNaOH pH 7.5 buffer containing 0.01% Triton X-100, 10 mM MgCl<sub>2</sub>, 0.1% BSA, 1 mM DTT, 10 Î¼M sodium orthovanadate, 10 Î¼M Î²-glycerophosphate, and 2.5% DMSO for 15 min at room temperature. The reaction was initiated with an equal volume of a peptide substrate (5-FAM-RSRHSSYPAGT-CONH<sub>2</sub>, Caliper Life Sciences) and ATP mixture in the aforementioned buffer. The final concentrations in the reaction were 600 pM Pim-1, 1 Î¼M peptide substrate, and 150 Î¼M ATP (ATP Km). The reaction was incubated at room temperature for 45 min and terminated with a buffer containing excess EDTA (100 mM HepesâNaOH pH 7.5, 0.02% Brij, 0.1% CR-3, 0.36% DMSO, and 100 mM EDTA). The plate was run for one cycle on a LabChip 3000 (Caliper Life Sciences, Hopkinton, MA) in an off-chip mobility shift assay with an upstream voltage of â2250 V, a downstream voltage of â500 V, and a vacuum pressure of â1.6 psi. The LabChip 3000 separates and measures the fluorescent signal of fluorescein-labeled peptide substrate and fluorescein-labeled peptide product present in each well. Results are expressed as percent conversion by measuring peak heights for both the substrate and product and dividing the product peak height by the sum of peak heights for both substrate and product. On every plate 100% inhibition (with a saturating concentration of staurosporine) and 0% inhibition (substrate with enzyme and DMSO) controls were used to calculate percent inhibition of tested compounds and a Zâ²-value.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Crystallography</h3><div class="NLM_p last">Purified human Pim-1 protein (aa 29â313) with a C-terminal His tag was concentrated to 10â13 mg/mL in 20 mM HEPES, pH 8, 120 mM NaCl, 5 mM DTT. Pim-1 was crystallized in 0.1 M imidazole, pH 6.4, anhydrous 1â1.4 M sodium acetate by sitting drop vapor diffusion at 4 Â°C and reached a maximum size after about 5 days. Pim-1 crystals were soaked with 10 mM compound in the well solution overnight at 4 Â°C and flash frozen with 30% glycerol in the presence of the compound. Diffraction data were collected at home and at ALS503 (Advanced Light Source at Lawrence Berkeley National Laboratory, Berkeley, CA) and processed using HKL2000 and Scala. Crystals belong to space group <i>P</i>65, with <i>a</i> = <i>b</i> = 98 Ã, <i>c</i> = 80 Ã, Î± = Î² = 90.00Â°, Î³ = 120.00Â°. The structures were solved by molecular replacement using the apo Pim-1 structure as a search model.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Iterative manual model building was carried out with Coot,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> coupled with refinement using Refmac5. The resolutions (Ã) of the structures elucidated were as follows: <b>2a</b>, 2.48; <b>2b</b>, 2.08; <b>2c</b>, 2.03; <b>2d</b>; 2.27; <b>2e</b>, 2.23; <b>3</b>, 2.23; <b>6</b>, 1.85.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20"><a href="/doi/suppl/10.1021/jm2014698">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83326" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83326" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">IC<sub>50</sub>s and structures of all needle screening hits with a Pim-1 IC<sub>50</sub> < 100 Î¼M. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm2014698/suppl_file/jm2014698_si_001.pdf">jm2014698_si_001.pdf (195.19 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm2014698" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35032" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew C. Good</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#62030c061007154c050d0d062205070c181b0f074c010d0f"><span class="__cf_email__" data-cfemail="73121d170116045d141c1c173314161d090a1e165d101c1e">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jinyu Liu</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bradford Hirth</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gary Asmussen</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yibin Xiang</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hans-Peter Biemann</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kimberly A. Bishop</span> - <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Trisha Fremgen</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maria Fitzgerald</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tatiana Gladysheva</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Annuradha Jain</span> - <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katherine Jancsics</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Markus Metz</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew Papoulis</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renato Skerlj</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. David Stepp</span> - <span class="hlFld-Affiliation affiliation">â¡Department
of Medicinal Chemistry, Â§Department of In Vitro Biology, and â¥Department of DMPK, Drug and Biomaterial R&D, Genzyme Corp., 153 Second Avenue, Waltham, Massachusetts 02451, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span>;Â 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein Discovery, Genzyme Corp., 49 New York Ave., Framingham, Massachusetts
01701, United States </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ronnie R. Wei</span> - <span class="hlFld-Affiliation affiliation">Department of Therapeutic Protein
Research, Genzyme Corp., 1 Mountain Road,
Framingham, Massachusetts 01701, United States </span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i21" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i21"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i22" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i22"> Abbreviations Used</h2><tr><td class="NLM_term">FBDD</td><td class="NLM_def"><p class="first last">fragment-based drug design</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structureâactivity relationship</p></td></tr><tr><td class="NLM_term">SP</td><td class="NLM_def"><p class="first last">standard precision</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">THZ</td><td class="NLM_def"><p class="first last">thiazolidinedione</p></td></tr><tr><td class="NLM_term">XP</td><td class="NLM_def"><p class="first last">extra precision</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65505" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65505" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 27 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giangreco, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortenson, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span> </span><span class="NLM_article-title">Docking performance of fragments and druglike compounds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5422</span><span class="NLM_x">â</span> <span class="NLM_lpage">5431</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200558u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFCgsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5422-5431&author=M.+L.+Verdonkauthor=I.+Giangrecoauthor=R.+J.+Hallauthor=O.+Korbauthor=P.+N.+Mortensonauthor=C.+W.+Murray&title=Docking+performance+of+fragments+and+druglike+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Performance of Fragments and Druglike Compounds</span></div><div class="casAuthors">Verdonk, Marcel L.; Giangreco, Ilenia; Hall, Richard J.; Korb, Oliver; Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5422-5431</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper addresses two questions of key interest to researchers working with protein-ligand docking methods: (i) Why is there such a large variation in docking performance between different test sets reported in the literature (ii) Are fragments more difficult to dock than druglike compds..  To answer these, the authors construct a test set of inhouse x-ray structures of protein-ligand complexes from drug discovery projects, half of which contain fragment ligands, the other half drug-like ligands.  The authors find that a key factor affecting docking performance is ligand efficiency (LE).  High LE compds. are significantly easier to dock than low LE compds., which the authors believe could explain the differences obsd. between test sets reported in the literature.  There is no significant difference in docking performance between fragments and drug-like compds., but the reasons why dockings fail appear to be different.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVFTOFvmiFxrVg90H21EOLACvtfcHk0lie4hnTNlgUXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFCgsLo%253D&md5=8d7a5519d6e7fa2ab403ea76b8b9a586</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm200558u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200558u%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DGiangreco%26aufirst%3DI.%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DDocking%2520performance%2520of%2520fragments%2520and%2520druglike%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5422%26epage%3D5431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">SÃ¡ndor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiss, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KeserÅ±, G. M.</span><span> </span><span class="NLM_article-title">Virtual fragment docking by Glide: A validation study on 190 proteinâfragment complexes</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1165</span><span class="NLM_x">â</span> <span class="NLM_lpage">1172</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci1000407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVaksLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1165-1172&author=M.+S%C3%A1ndorauthor=R.+Kissauthor=G.+M.+Keser%C5%B1&title=Virtual+fragment+docking+by+Glide%3A+A+validation+study+on+190+protein%E2%80%93fragment+complexes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Fragment Docking by Glide: a Validation Study on 190 Protein-Fragment Complexes</span></div><div class="casAuthors">Sandor, Mark; Kiss, Robert; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1165-1172</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The docking accuracy of Glide was evaluated using 16 different docking protocols on 190 protein-fragment complexes representing 78 targets.  Std. precision docking (Glide SP) based protocols showed the best performance.  The av. root-mean-square deviation (RMSD) between the docked and cocrystd. poses achieved by Glide SP with pre- and postprocessing was 1.17 Ã, and an acceptable binding mode with RMSD. < 2 Ã could be found in 80% of the cases.  Comparison of the docking results produced by different protocols suggests that the sampling efficacy of Glide is adequate for fragment docking.  The docking accuracy seems to be limited by the performance of scoring schemes, which is supported by the weak correlation between exptl. binding affinities and GlideScores.  Cross-docking expts. performed on 8 targets represented by 63 complexes revealed that Glide SP gave similar results to that of the computationally more intensive Glide XP.  The av. RMSD achieved by Glide SP with pre- and postprocessing was 2.06 Ã, and an acceptable binding mode with RMSD < 2 Ã could be found in 63% of the cases.  These cross-docking results were improved significantly selecting the optimal X-ray structure for each target (av. RMSD = 1.3 Ã, success rate = 77%), indicating the importance of enrichment studies and the use of multiple X-ray structures in virtual fragment screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodKfjH9BbnrLVg90H21EOLACvtfcHk0lg0F3Wh9j3qTA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVaksLY%253D&md5=35d7acb7374360828beb668e2f207bab</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fci1000407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1000407%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1ndor%26aufirst%3DM.%26aulast%3DKiss%26aufirst%3DR.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DVirtual%2520fragment%2520docking%2520by%2520Glide%253A%2520A%2520validation%2520study%2520on%2520190%2520protein%25E2%2580%2593fragment%2520complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1165%26epage%3D1172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Amaravadi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. B.</span><span> </span><span class="NLM_article-title">The survival kinases Akt and Pim as potential pharmacological targets</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">2618</span><span class="NLM_x">â</span> <span class="NLM_lpage">2624</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2005&pages=2618-2624&author=R.+Amaravadiauthor=C.+B.+Thompson&title=The+survival+kinases+Akt+and+Pim+as+potential+pharmacological+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmaravadi%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DThe%2520survival%2520kinases%2520Akt%2520and%2520Pim%2520as%2520potential%2520pharmacological%2520targets%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2005%26volume%3D115%26spage%3D2618%26epage%3D2624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Amson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigaux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Przedborski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flandrin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givol, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telerman, A.</span><span> </span><span class="NLM_article-title">The human protooncogene product p33pim is expressed during fetal hematopoiesis and in diverse leukemias</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">8857</span><span class="NLM_x">â</span> <span class="NLM_lpage">8861</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=8857-8861&author=R.+Amsonauthor=F.+Sigauxauthor=S.+Przedborskiauthor=G.+Flandrinauthor=D.+Givolauthor=A.+Telerman&title=The+human+protooncogene+product+p33pim+is+expressed+during+fetal+hematopoiesis+and+in+diverse+leukemias"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmson%26aufirst%3DR.%26aulast%3DSigaux%26aufirst%3DF.%26aulast%3DPrzedborski%26aufirst%3DS.%26aulast%3DFlandrin%26aufirst%3DG.%26aulast%3DGivol%26aufirst%3DD.%26aulast%3DTelerman%26aufirst%3DA.%26atitle%3DThe%2520human%2520protooncogene%2520product%2520p33pim%2520is%2520expressed%2520during%2520fetal%2520hematopoiesis%2520and%2520in%2520diverse%2520leukemias%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1989%26volume%3D86%26spage%3D8857%26epage%3D8861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Shah, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K-G</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lilly, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salto-Tellez, M</span><span> </span><span class="NLM_article-title">Potential roles for the PIM-1 kinase in human cancerâA molecular and therapeutic appraisal</span> <span class="citation_source-journal">Eur. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">2144</span><span class="NLM_x">â</span> <span class="NLM_lpage">2151</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2008&pages=2144-2151&author=N.+Shahauthor=B.+Pangauthor=K-G+Yeohauthor=S.+Thornauthor=C.+S.+Chenauthor=M.+B.+Lillyauthor=M+Salto-Tellez&title=Potential+roles+for+the+PIM-1+kinase+in+human+cancer%E2%80%94A+molecular+and+therapeutic+appraisal"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DN.%26aulast%3DPang%26aufirst%3DB.%26aulast%3DYeoh%26aufirst%3DK-G%26aulast%3DThorn%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DC.%2BS.%26aulast%3DLilly%26aufirst%3DM.%2BB.%26aulast%3DSalto-Tellez%26aufirst%3DM%26atitle%3DPotential%2520roles%2520for%2520the%2520PIM-1%2520kinase%2520in%2520human%2520cancer%25E2%2580%2594A%2520molecular%2520and%2520therapeutic%2520appraisal%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2008%26volume%3D44%26spage%3D2144%26epage%3D2151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Magnuson, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reeves, R.</span><span> </span><span class="NLM_article-title">Why target PIM-1 for cancer diagnosis and treatment?</span> <span class="citation_source-journal">Future Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1461</span><span class="NLM_x">â</span> <span class="NLM_lpage">1478</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.2217%2Ffon.10.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=20919829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1GnsrjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1461-1478&author=N.+S.+Magnusonauthor=Z.+Wangauthor=G.+Dingauthor=R.+Reeves&title=Why+target+PIM-1+for+cancer+diagnosis+and+treatment%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Why target PIM1 for cancer diagnosis and treatment?</span></div><div class="casAuthors">Magnuson, Nancy S.; Wang, Zeping; Ding, Gang; Reeves, Raymond</div><div class="citationInfo"><span class="NLM_cas:title">Future Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1461-1478</span>CODEN:
                <span class="NLM_cas:coden">FOUNBN</span>;
        ISSN:<span class="NLM_cas:issn">1479-6694</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  The highly conserved proto-oncogenic protein PIM1 is an unusual serine or threonine kinase, in part because it is constitutively active.  Overexpression of PIM1 exptl. leads to tumor formation in mice, while complete knockout of the protein has no observable phenotype.  It appears to contribute to cancer development in three major ways when it is overexpressed; by inhibiting apoptosis, by promoting cell proliferation and by promoting genomic instability.  Expression in normal tissues is nearly undetectable.  However, in hematopoietic malignancies and in a variety of solid tumors, increased PIM1 expression has been shown to correlate with the stage of disease.  This characteristic suggests it can serve as a useful biomarker for cancer diagnosis and prognosis.  Several specific and potent inhibitors of PIM1's kinase activity have also been shown to induce apoptotic death of cancer cells, to sensitize cancer cells to chemotherapy and to synergize with other anti-tumor agents, thus making it an attractive therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8Np_ubohH1bVg90H21EOLACvtfcHk0ljTbMsiW9N9HQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1GnsrjE&md5=c072a8db7336d7c7dc5032c56b9a923b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2217%2Ffon.10.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Ffon.10.106%26sid%3Dliteratum%253Aachs%26aulast%3DMagnuson%26aufirst%3DN.%2BS.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DDing%26aufirst%3DG.%26aulast%3DReeves%26aufirst%3DR.%26atitle%3DWhy%2520target%2520PIM-1%2520for%2520cancer%2520diagnosis%2520and%2520treatment%253F%26jtitle%3DFuture%2520Oncol.%26date%3D2010%26volume%3D6%26spage%3D1461%26epage%3D1478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Kim, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, D.</span><span> </span><span class="NLM_article-title">Constitutively activated FLT3 phosphorylates BAD partially through Pim-1 Br</span> <span class="citation_source-journal">J. Haematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">500</span><span class="NLM_x">â</span> <span class="NLM_lpage">509</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.1111%2Fj.1365-2141.2006.06225.x" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=500-509&author=K.+T.+Kimauthor=M.+Levisauthor=D.+Small&title=Constitutively+activated+FLT3+phosphorylates+BAD+partially+through+Pim-1+Br"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2141.2006.06225.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2141.2006.06225.x%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BT.%26aulast%3DLevis%26aufirst%3DM.%26aulast%3DSmall%26aufirst%3DD.%26atitle%3DConstitutively%2520activated%2520FLT3%2520phosphorylates%2520BAD%2520partially%2520through%2520Pim-1%2520Br%26jtitle%3DJ.%2520Haematol.%26date%3D2006%26volume%3D124%26spage%3D500%26epage%3D509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Qian, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Studts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barringer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kronkaitis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mische, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farmer, B.</span><span> </span><span class="NLM_article-title">Structural basis of constitutive activity and a unique nucleotide binding mode of human Pim-1 kinase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">6130</span><span class="NLM_x">â</span> <span class="NLM_lpage">6137</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=6130-6137&author=K.+C.+Qianauthor=L.+Wangauthor=E.+R.+Hickeyauthor=J.+Studtsauthor=K.+Barringerauthor=C.+Pengauthor=A.+Kronkaitisauthor=J.+Liauthor=A.+Whiteauthor=S.+Mischeauthor=B.+Farmer&title=Structural+basis+of+constitutive+activity+and+a+unique+nucleotide+binding+mode+of+human+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DK.%2BC.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DStudts%26aufirst%3DJ.%26aulast%3DBarringer%26aufirst%3DK.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DKronkaitis%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DMische%26aufirst%3DS.%26aulast%3DFarmer%26aufirst%3DB.%26atitle%3DStructural%2520basis%2520of%2520constitutive%2520activity%2520and%2520a%2520unique%2520nucleotide%2520binding%2520mode%2520of%2520human%2520Pim-1%2520kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D6130%26epage%3D6137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Schenone, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tintori, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botta, M.</span><span> </span><span class="NLM_article-title">Using insights into Pim-1 structure to design new anticancer drugs</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3964</span><span class="NLM_x">â</span> <span class="NLM_lpage">3978</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.2174%2F138161210794454996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=21158732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFKqtLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=3964-3978&author=S.+Schenoneauthor=C.+Tintoriauthor=M.+Botta&title=Using+insights+into+Pim-1+structure+to+design+new+anticancer+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Using insights into Pim1 structure to design new anticancer drugs</span></div><div class="casAuthors">Schenone, Silvia; Tintori, Cristina; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">3964-3978</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Human Pim1 (proviral integration site for Moloney murine leukemia virus) kinase is a 313-amino acid serine-threonine kinase that possesses several biol. functions in cell survival, proliferation and differentiation, and its over-expression has been obsd. in a no. of human cancers.  Indeed, this kinase is a proto-oncogene that has been implicated in early transformation and tumor progression, esp. in hematopoietic malignancies and prostate carcinoma where it is a marker of a poor prognosis.  For these reasons, Pim1 is emerging as an important target in drug discovery, and many Pim1 inhibitors have been reported in the last three years.  The challenge of this research is to obtain compds. that specifically inhibit only Pim1 and not Pim2 and Pim3, the other members of the Pim family, with the aim of providing selective inhibitors as potential therapeutic agents and also of studying the different roles of three enzymes.  In this review Pim1 functions and Pim1 role in human cancer are summarized, but the primary focus of the article is on the Pim1 three-dimensional structure that was deeply analyzed by a detailed inspection of the available crystallog. data and all complexes of small mol. inhibitors reported in the literature to this point.  Finally, the use of mol. modeling techniques for the identification and optimization of Pim1 inhibitors is extensively discussed.  This data collection, which to the best of the authors' knowledge was not previously reviewed in such detail, could offer a useful tool for medicinal chemists working in the field of small mol. kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEKuEDdrcDV7Vg90H21EOLACvtfcHk0ljTbMsiW9N9HQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFKqtLs%253D&md5=1f446cd7cdd6ad5fb098e4471e807c27</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F138161210794454996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161210794454996%26sid%3Dliteratum%253Aachs%26aulast%3DSchenone%26aufirst%3DS.%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DUsing%2520insights%2520into%2520Pim-1%2520structure%2520to%2520design%2520new%2520anticancer%2520drugs%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2010%26volume%3D16%26spage%3D3964%26epage%3D3978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Xiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asmussen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biemann, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bishop, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Good, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gladysheva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jancsics, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papoulis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skerlj, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stepp, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, R. R.</span><span> </span><span class="NLM_article-title">The discovery of novel benzofuran-2-carboxylic acids as potent Pim-1 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">3050</span><span class="NLM_x">â</span> <span class="NLM_lpage">3056</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=3050-3056&author=Y.+Xiangauthor=B.+Hirthauthor=G.+Asmussenauthor=H.+P.+Biemannauthor=K.+A.+Bishopauthor=A.+Goodauthor=M.+Fitzgeraldauthor=T.+Gladyshevaauthor=A.+Jainauthor=K.+Jancsicsauthor=J.+Liuauthor=M.+Metzauthor=A.+Papoulisauthor=R.+Skerljauthor=J.+D.+Steppauthor=R.+R.+Wei&title=The+discovery+of+novel+benzofuran-2-carboxylic+acids+as+potent+Pim-1+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DHirth%26aufirst%3DB.%26aulast%3DAsmussen%26aufirst%3DG.%26aulast%3DBiemann%26aufirst%3DH.%2BP.%26aulast%3DBishop%26aufirst%3DK.%2BA.%26aulast%3DGood%26aufirst%3DA.%26aulast%3DFitzgerald%26aufirst%3DM.%26aulast%3DGladysheva%26aufirst%3DT.%26aulast%3DJain%26aufirst%3DA.%26aulast%3DJancsics%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DMetz%26aufirst%3DM.%26aulast%3DPapoulis%26aufirst%3DA.%26aulast%3DSkerlj%26aufirst%3DR.%26aulast%3DStepp%26aufirst%3DJ.%2BD.%26aulast%3DWei%26aufirst%3DR.%2BR.%26atitle%3DThe%2520discovery%2520of%2520novel%2520benzofuran-2-carboxylic%2520acids%2520as%2520potent%2520Pim-1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D3050%26epage%3D3056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Boehm, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boehringer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bur, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gmuender, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kostrewa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuehne, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luebbers, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meunier-Keller, N</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, F.</span><span> </span><span class="NLM_article-title">Novel inhibitors of DNA gyrase: 3D structure based biased needle screening, hit validation by biophysical methods, and 3D guided optimization. A promising alternative to random screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">2664</span><span class="NLM_x">â</span> <span class="NLM_lpage">2674</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000017s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BD3cXktFOgsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=2664-2674&author=H.-J.+Boehmauthor=M.+Boehringerauthor=D.+Burauthor=H.+Gmuenderauthor=H.+Walterauthor=W.+Klausauthor=D.+Kostrewaauthor=H.+Kuehneauthor=T.+Luebbersauthor=N+Meunier-Kellerauthor=F.+Mueller&title=Novel+inhibitors+of+DNA+gyrase%3A+3D+structure+based+biased+needle+screening%2C+hit+validation+by+biophysical+methods%2C+and+3D+guided+optimization.+A+promising+alternative+to+random+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Novel inhibitors of DNA gyrase: 3D structure based biased needle screening, hit validation by biophysical methods, and 3D guided optimization. A promising alternative to random screening</span></div><div class="casAuthors">Boehm, Hans-Joachim; Boehringer, Markus; Bur, Daniel; Gmuender, Hans; Huber, Walter; Klaus, Werner; Kostrewa, Dirk; Kuehne, Holger; Luebbers, Thomas; Meunier-Keller, Nathalie; Mueller, Francis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2664-2674</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Random screening provided no suitable lead structures in a search for novel inhibitors of the bacterial enzyme DNA gyrase.  Therefore, an alternative approach had to be developed.  Relying on the detailed 3D structural information of the targeted ATP binding site, our approach combines as key techniques (1) an in silico screening for potential low mol. wt. inhibitors, (2) a biased high throughput DNA gyrase screen, (3) validation of the screening hits by biophys. methods, and (4) a 3D-guided optimization process.  When the in silico screening was performed, the initial data set contg. 350,000 compds. could be reduced to 3000 mols.  Testing these 3000 selected compds. in the DNA gyrase assay provided 150 hits clustered in 14 classes.  Seven classes could be validated as true, novel DNA gyrase inhibitors that act by binding to the ATP binding site located on subunit B: phenols, 2-aminotriazines, 4-aminopyrimidines, 2-aminopyrimidines, pyrrolopyrimidines, indazoles, and 2-hydroxymethylindoles.  The 3D guided optimization provided highly potent DNA gyrase inhibitors, e.g., the 3,4-disubstituted indazole (I) being a 10 times more potent DNA gyrase inhibitor than novobiocin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosDAeZrjuWibVg90H21EOLACvtfcHk0liPa02gTjZWnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXktFOgsrc%253D&md5=bb7f072e82e89371e6e2da109bbd84af</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm000017s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000017s%26sid%3Dliteratum%253Aachs%26aulast%3DBoehm%26aufirst%3DH.-J.%26aulast%3DBoehringer%26aufirst%3DM.%26aulast%3DBur%26aufirst%3DD.%26aulast%3DGmuender%26aufirst%3DH.%26aulast%3DWalter%26aufirst%3DH.%26aulast%3DKlaus%26aufirst%3DW.%26aulast%3DKostrewa%26aufirst%3DD.%26aulast%3DKuehne%26aufirst%3DH.%26aulast%3DLuebbers%26aufirst%3DT.%26aulast%3DMeunier-Keller%26aufirst%3DN%26aulast%3DMueller%26aufirst%3DF.%26atitle%3DNovel%2520inhibitors%2520of%2520DNA%2520gyrase%253A%25203D%2520structure%2520based%2520biased%2520needle%2520screening%252C%2520hit%2520validation%2520by%2520biophysical%2520methods%252C%2520and%25203D%2520guided%2520optimization.%2520A%2520promising%2520alternative%2520to%2520random%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D2664%26epage%3D2674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrows, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffen, E. J.</span><span> </span><span class="NLM_article-title">Fragment screening: An introduction</span> <span class="citation_source-journal">Mol. Biosyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">â</span> <span class="NLM_lpage">446</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2006&pages=430-446&author=A.+R.+Leachauthor=M.+M.+Hannauthor=J.+N.+Burrowsauthor=E.+J.+Griffen&title=Fragment+screening%3A+An+introduction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DBurrows%26aufirst%3DJ.%2BN.%26aulast%3DGriffen%26aufirst%3DE.%2BJ.%26atitle%3DFragment%2520screening%253A%2520An%2520introduction%26jtitle%3DMol.%2520Biosyst.%26date%3D2006%26volume%3D9%26spage%3D430%26epage%3D446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Xia, Z</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knaak, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beharry, Z. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McInnes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraft, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C. D.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of novel inhibitors of Pim-1 and Pim-2 protein kinases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">74</span><span class="NLM_x">â</span> <span class="NLM_lpage">86</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800937p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=74-86&author=Z+Xiaauthor=C.+Knaakauthor=J.+Maauthor=Z.+M.+Beharryauthor=C.+McInnesauthor=W.+Wangauthor=A.+S.+Kraftauthor=C.+D.+Smith&title=Synthesis+and+evaluation+of+novel+inhibitors+of+Pim-1+and+Pim-2+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm800937p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800937p%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DZ%26aulast%3DKnaak%26aufirst%3DC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DBeharry%26aufirst%3DZ.%2BM.%26aulast%3DMcInnes%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKraft%26aufirst%3DA.%2BS.%26aulast%3DSmith%26aufirst%3DC.%2BD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520inhibitors%2520of%2520Pim-1%2520and%2520Pim-2%2520protein%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D74%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Qian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cywin, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farmer, B. T.,  II</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kashem, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magboo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pack, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prokopowicz, A.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welzel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morwick, T.</span><span> </span><span class="NLM_article-title">Hit to lead account of the discovery of a new class of inhibitors of Pim kinases and crystallographic studies revealing an unusual kinase binding mode</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1814</span><span class="NLM_x">â</span> <span class="NLM_lpage">1827</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm801242y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1814-1827&author=K.+Qianauthor=L.+Wangauthor=C.+L.+Cywinauthor=B.+T.+Farmerauthor=E.+Hickeyauthor=C.+Homonauthor=S.+Jakesauthor=M.+A.+Kashemauthor=G.+Leeauthor=S.+Leonardauthor=J.+Liauthor=R.+Magbooauthor=W.+Maoauthor=E.+Packauthor=C.+Pengauthor=A.+Prokopowiczauthor=M.+Welzelauthor=J.+Wolakauthor=T.+Morwick&title=Hit+to+lead+account+of+the+discovery+of+a+new+class+of+inhibitors+of+Pim+kinases+and+crystallographic+studies+revealing+an+unusual+kinase+binding+mode"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm801242y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801242y%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DCywin%26aufirst%3DC.%2BL.%26aulast%3DFarmer%26aufirst%3DB.%2BT.%26aulast%3DHickey%26aufirst%3DE.%26aulast%3DHomon%26aufirst%3DC.%26aulast%3DJakes%26aufirst%3DS.%26aulast%3DKashem%26aufirst%3DM.%2BA.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DLeonard%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMagboo%26aufirst%3DR.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DPack%26aufirst%3DE.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DProkopowicz%26aufirst%3DA.%26aulast%3DWelzel%26aufirst%3DM.%26aulast%3DWolak%26aufirst%3DJ.%26aulast%3DMorwick%26aufirst%3DT.%26atitle%3DHit%2520to%2520lead%2520account%2520of%2520the%2520discovery%2520of%2520a%2520new%2520class%2520of%2520inhibitors%2520of%2520Pim%2520kinases%2520and%2520crystallographic%2520studies%2520revealing%2520an%2520unusual%2520kinase%2520binding%2520mode%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1814%26epage%3D1827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">A rule of three for fragment-based lead discovery?</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">876</span><span class="NLM_x">â</span> <span class="NLM_lpage">877</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=876-877&author=M.+Congreveauthor=R.+Carrauthor=C.+Murrayauthor=H.+Jhoti&title=A+rule+of+three+for+fragment-based+lead+discovery%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DA%2520rule%2520of%2520three%2520for%2520fragment-based%2520lead%2520discovery%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D876%26epage%3D877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span> <span class="citation_source-book">Glide (version 50207 unless otherwise mentioned)</span>; <span class="NLM_publisher-name">SchrÃ¶dinger, Inc.</span>, <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span><span class="NLM_x">; </span><a href="http://www.schrodinger.com" class="extLink">www.schrodinger.com</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Glide+%28version+50207+unless+otherwise+mentioned%29%3B+Schr%C3%B6dinger%2C+Inc.%2C+New+York%2C+2009%3B+www.schrodinger.com"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGlide%2520%2528version%252050207%2520unless%2520otherwise%2520mentioned%2529%26pub%3DSchr%25C3%25B6dinger%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">â</span> <span class="NLM_lpage">242</span><div class="note"><p class="first last"> (<a href="http://www.pdb.org" class="extLink">www.pdb.org</a>)</p></div></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0liaVYIy5Z-PPw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Cheney, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Applebya, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walkera, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yaoa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamatakea, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honga, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wua, J. Z.</span><span> </span><span class="NLM_article-title">Identification and structureâactivity relationships of substituted pyridones as inhibitors of Pim-1 kinase</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1679</span><span class="NLM_x">â</span> <span class="NLM_lpage">1683</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=1679-1683&author=I.+W.+Cheneyauthor=S.+Yanaauthor=T.+Applebyaauthor=H.+Walkeraauthor=T.+Voaauthor=N.+Yaoaauthor=R.+Hamatakeaauthor=Z.+Hongaauthor=J.+Z.+Wua&title=Identification+and+structure%E2%80%93activity+relationships+of+substituted+pyridones+as+inhibitors+of+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCheney%26aufirst%3DI.%2BW.%26aulast%3DYana%26aufirst%3DS.%26aulast%3DApplebya%26aufirst%3DT.%26aulast%3DWalkera%26aufirst%3DH.%26aulast%3DVoa%26aufirst%3DT.%26aulast%3DYaoa%26aufirst%3DN.%26aulast%3DHamatakea%26aufirst%3DR.%26aulast%3DHonga%26aufirst%3DZ.%26aulast%3DWua%26aufirst%3DJ.%2BZ.%26atitle%3DIdentification%2520and%2520structure%25E2%2580%2593activity%2520relationships%2520of%2520substituted%2520pyridones%2520as%2520inhibitors%2520of%2520Pim-1%2520kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D1679%26epage%3D1683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuver-Moody, C.</span><span> </span><span class="NLM_article-title">Docking study yields four novel inhibitors of the protooncogene Pim-1 kinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1972</span><span class="NLM_x">â</span> <span class="NLM_lpage">1975</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701248t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOmtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1972-1975&author=A.+C.+Pierceauthor=M.+Jacobsauthor=C.+Stuver-Moody&title=Docking+study+yields+four+novel+inhibitors+of+the+protooncogene+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Study Yields Four Novel Inhibitors of the Protooncogene Pim-1 Kinase</span></div><div class="casAuthors">Pierce, Albert C.; Jacobs, Marc; Stuver-Moody, Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1972-1975</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To supplement the hits from a high throughput screen, docking was performed against Pim-1 kinase.  Glide docking was augmented with a filter to require traditional or arom. CHÂ·Â·O hydrogen bonds to the kinase hinge.  Four diverse actives, of 96 mols. assayed, had Ki values between 0.091 and 4.5 Î¼M.  This gives a 14-fold enrichment over the earlier HTS run, and the two crystal structures solved confirmed the binding modes predicted by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH6GwwgMwC27Vg90H21EOLACvtfcHk0liaVYIy5Z-PPw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOmtLY%253D&md5=fafba52528730d4e250d489ad24a6cd8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm701248t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701248t%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DJacobs%26aufirst%3DM.%26aulast%3DStuver-Moody%26aufirst%3DC.%26atitle%3DDocking%2520study%2520yields%2520four%2520novel%2520inhibitors%2520of%2520the%2520protooncogene%2520Pim-1%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1972%26epage%3D1975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Gampe, R. T.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montana, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bledsoe, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milburn, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kliewer, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H. E.</span><span> </span><span class="NLM_article-title">Asymmetry in the PPARgamma/RXRalpha crystal structure reveals the molecular basis of heterodimerization among nuclear receptors</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">545</span><span class="NLM_x">â</span> <span class="NLM_lpage">555</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=545-555&author=R.+T.+Gampeauthor=V.+G.+Montanaauthor=M.+H.+Lambertauthor=A.+B.+Millerauthor=R.+K.+Bledsoeauthor=M.+V.+Milburnauthor=S.+A.+Kliewerauthor=T.+M.+Willsonauthor=H.+E.+Xu&title=Asymmetry+in+the+PPARgamma%2FRXRalpha+crystal+structure+reveals+the+molecular+basis+of+heterodimerization+among+nuclear+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGampe%26aufirst%3DR.%2BT.%26aulast%3DMontana%26aufirst%3DV.%2BG.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DMiller%26aufirst%3DA.%2BB.%26aulast%3DBledsoe%26aufirst%3DR.%2BK.%26aulast%3DMilburn%26aufirst%3DM.%2BV.%26aulast%3DKliewer%26aufirst%3DS.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DAsymmetry%2520in%2520the%2520PPARgamma%252FRXRalpha%2520crystal%2520structure%2520reveals%2520the%2520molecular%2520basis%2520of%2520heterodimerization%2520among%2520nuclear%2520receptors%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D5%26spage%3D545%26epage%3D555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandretto, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span> </span><span class="NLM_article-title">Kinase inhibitors and the case for CHÂ·Â·Â·O hydrogen bonds in proteinâligand binding</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">567</span><span class="NLM_x">â</span> <span class="NLM_lpage">576</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2002&pages=567-576&author=A.+C.+Pierceauthor=K.+L.+Sandrettoauthor=G.+W.+Bemis&title=Kinase+inhibitors+and+the+case+for+CH%C2%B7%C2%B7%C2%B7O+hydrogen+bonds+in+protein%E2%80%93ligand+binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DSandretto%26aufirst%3DK.%2BL.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26atitle%3DKinase%2520inhibitors%2520and%2520the%2520case%2520for%2520CH%25C2%25B7%25C2%25B7%25C2%25B7O%2520hydrogen%2520bonds%2520in%2520protein%25E2%2580%2593ligand%2520binding%26jtitle%3DProteins%26date%3D2002%26volume%3D49%26spage%3D567%26epage%3D576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span> <span class="citation_source-book">Maestro (version 9.0 unless otherwise mentioned)</span>; <span class="NLM_publisher-name">SchrÃ¶dinger, Inc.</span>, <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>. <a href="http://www.schrodinger.com" class="extLink">www.schrodinger.com</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Maestro+%28version+9.0+unless+otherwise+mentioned%29%3B+Schr%C3%B6dinger%2C+Inc.%2C+New+York%2C+2009.+www.schrodinger.com"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMaestro%2520%2528version%25209.0%2520unless%2520otherwise%2520mentioned%2529%26pub%3DSchr%25C3%25B6dinger%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span>Vida version 3.1. Open Eye Inc.; <a href="http://www.eyesopen.com" class="extLink">www.eyesopen.com</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Vida+version+3.1.+Open+Eye+Inc.%3B+www.eyesopen.com"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span> <span class="citation_source-book">Ligprep, version 22207</span>; <span class="NLM_publisher-name">SchrÃ¶dinger, Inc.</span>, <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span><span class="NLM_x">; </span><a href="http://www.schrodinger.com" class="extLink">www.schrodinger.com</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Ligprep%2C+version+22207%3B+Schr%C3%B6dinger%2C+Inc.%2C+New+York%2C+2009%3B+www.schrodinger.com"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DLigprep%252C%2520version%252022207%26pub%3DSchr%25C3%25B6dinger%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Zhang, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oldenburg, K. R.</span><span> </span><span class="NLM_article-title">A simple statistical parameter for use in evaluation and validation of high throughput screening assays</span> <span class="citation_source-journal">J. Biomol. Screen</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">â</span> <span class="NLM_lpage">73</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.1177%2F108705719900400206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=10838414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&pages=67-73&author=J.+H.+Zhangauthor=T.+D.+Chungauthor=K.+R.+Oldenburg&title=A+simple+statistical+parameter+for+use+in+evaluation+and+validation+of+high+throughput+screening+assays"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span></div><div class="casAuthors">Zhang; Chung; Oldenburg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ability to identify active compounds (3hits2) from large chemical libraries accurately and rapidly has been the ultimate goal in developing high-throughput screening (HTS) assays.  The ability to identify hits from a particular HTS assay depends largely on the suitability or quality of the assay used in the screening.  The criteria or parameters for evaluating the 3suitability2 of an HTS assay for hit identification are not well defined and hence it still remains difficult to compare the quality of assays directly.  In this report, a screening window coefficient, called 3Z-factor,2 is defined.  This coefficient is reflective of both the assay signal dynamic range and the data variation associated with the signal measurements, and therefore is suitable for assay quality assessment.  The Z-factor is a dimensionless, simple statistical characteristic for each HTS assay.  The Z-factor provides a useful tool for comparison and evaluation of the quality of assays, and can be utilized in assay optimization and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkwW5JDdXnbsNIRw0GKUoEfW6udTcc2eYLqqPI8LjvCLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D&md5=0ec4ed2ccdd3b4a16521567a0975d31f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1177%2F108705719900400206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705719900400206%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DChung%26aufirst%3DT.%2BD.%26aulast%3DOldenburg%26aufirst%3DK.%2BR.%26atitle%3DA%2520simple%2520statistical%2520parameter%2520for%2520use%2520in%2520evaluation%2520and%2520validation%2520of%2520high%2520throughput%2520screening%2520assays%26jtitle%3DJ.%2520Biomol.%2520Screen%26date%3D1999%26spage%3D67%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandiyan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artis, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bremer, R.</span><span> </span><span class="NLM_article-title">Crystal structures of proto-oncogene kinase Pim-1: A target of aberrant somatic hypermutations in diffuse large cell lymphoma</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">348</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">â</span> <span class="NLM_lpage">193</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2005&pages=183-193&author=A.+Kumarauthor=V.+Mandiyanauthor=Y.+Suzukiauthor=C.+Zhangauthor=J.+Riceauthor=J.+Tsaiauthor=D.+R.+Artisauthor=P.+Ibrahimauthor=R.+Bremer&title=Crystal+structures+of+proto-oncogene+kinase+Pim-1%3A+A+target+of+aberrant+somatic+hypermutations+in+diffuse+large+cell+lymphoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DMandiyan%26aufirst%3DV.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DBremer%26aufirst%3DR.%26atitle%3DCrystal%2520structures%2520of%2520proto-oncogene%2520kinase%2520Pim-1%253A%2520A%2520target%2520of%2520aberrant%2520somatic%2520hypermutations%2520in%2520diffuse%2520large%2520cell%2520lymphoma%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D348%26spage%3D183%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">â</span> <span class="NLM_lpage">501</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0li8EGhNDkvCxw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 23 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Panagiotis K.  Chrysanthopoulos</span>, <span class="hlFld-ContribAuthor ">Prashant  Mujumdar</span>, <span class="hlFld-ContribAuthor ">Lucy A.  Woods</span>, <span class="hlFld-ContribAuthor ">Olan  Dolezal</span>, <span class="hlFld-ContribAuthor ">Bin  Ren</span>, <span class="hlFld-ContribAuthor ">Thomas S.  Peat</span>, and <span class="hlFld-ContribAuthor ">Sally-Ann  Poulsen</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of a New Zinc Binding Chemotype by Fragment Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (17)
                                     , 7333-7349. <a href="https://doi.org/10.1021/acs.jmedchem.7b00606" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00606</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00606%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Ba%252BNew%252BZinc%252BBinding%252BChemotype%252Bby%252BFragment%252BScreening%26aulast%3DChrysanthopoulos%26aufirst%3DPanagiotis%2BK.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D27042017%26date%3D01092017%26date%3D14092017%26date%3D17082017%26volume%3D60%26issue%3D17%26spage%3D7333%26epage%3D7349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah  Barelier</span>, <span class="hlFld-ContribAuthor ">Oliv  Eidam</span>, <span class="hlFld-ContribAuthor ">Inbar  Fish</span>, <span class="hlFld-ContribAuthor ">Johan  Hollander</span>, <span class="hlFld-ContribAuthor ">Francis  Figaroa</span>, <span class="hlFld-ContribAuthor ">Ruta  Nachane</span>, <span class="hlFld-ContribAuthor ">John J.  Irwin</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>, and <span class="hlFld-ContribAuthor ">Gregg  Siegal</span>  . </span><span class="cited-content_cbyCitation_article-title">Increasing Chemical Space Coverage by Combining Empirical and Computational Fragment Screens. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2014,</strong> <em>9 </em>
                                    (7)
                                     , 1528-1535. <a href="https://doi.org/10.1021/cb5001636" title="DOI URL">https://doi.org/10.1021/cb5001636</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb5001636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb5001636%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DIncreasing%252BChemical%252BSpace%252BCoverage%252Bby%252BCombining%252BEmpirical%252Band%252BComputational%252BFragment%252BScreens%26aulast%3DBarelier%26aufirst%3DSarah%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D03032014%26date%3D07052014%26date%3D20052014%26date%3D18072014%26date%3D07052014%26volume%3D9%26issue%3D7%26spage%3D1528%26epage%3D1535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simone  Di Micco</span>, <span class="hlFld-ContribAuthor ">Stefania  Terracciano</span>, <span class="hlFld-ContribAuthor ">Dafne  Ruggiero</span>, <span class="hlFld-ContribAuthor ">Marianna  Potenza</span>, <span class="hlFld-ContribAuthor ">Maria C.  Vaccaro</span>, <span class="hlFld-ContribAuthor ">Katrin  Fischer</span>, <span class="hlFld-ContribAuthor ">Oliver  Werz</span>, <span class="hlFld-ContribAuthor ">Ines  Bruno</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Bifulco</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of 2-(thiophen-2-yl)acetic Acid-Based Lead Compound for mPGES-1 Inhibition. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.676631" title="DOI URL">https://doi.org/10.3389/fchem.2021.676631</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.676631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.676631%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DIdentification%252Bof%252B2-%252528thiophen-2-yl%252529acetic%252BAcid-Based%252BLead%252BCompound%252Bfor%252BmPGES-1%252BInhibition%26aulast%3DDi%2BMicco%26aufirst%3DSimone%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Victor Sukbong  Hong</span>, <span class="hlFld-ContribAuthor ">Seungik  Jeong</span>, <span class="hlFld-ContribAuthor ">Yanghwan  Yun</span>, <span class="hlFld-ContribAuthor ">Hyeonseong  Choo</span>, <span class="hlFld-ContribAuthor ">Jongin  Won</span>, <span class="hlFld-ContribAuthor ">Jinho  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">1,3,
              4âOxadiazole
              â2(
              
                3
                H
              
              )âthione Analogs as
              PIM
              Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bulletin of the Korean Chemical Society</span><span> <strong>2020,</strong> <em>41 </em>
                                    (10)
                                     , 994-1001. <a href="https://doi.org/10.1002/bkcs.12101" title="DOI URL">https://doi.org/10.1002/bkcs.12101</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/bkcs.12101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbkcs.12101%26sid%3Dliteratum%253Aachs%26jtitle%3DBulletin%2520of%2520the%2520Korean%2520Chemical%2520Society%26atitle%3D1%25252C3%25252C%252B4%2525E2%252580%252590Oxadiazole%252B%2525E2%252580%2525902%252528%252B3%252BH%252B%252529%2525E2%252580%252590thione%252BAnalogs%252Bas%252BPIM%252BKinase%252BInhibitors%26aulast%3DHong%26aufirst%3DVictor%2BSukbong%26date%3D2020%26date%3D2020%26volume%3D41%26issue%3D10%26spage%3D994%26epage%3D1001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">CÃ©lien  Jacquemard</span>, <span class="hlFld-ContribAuthor ">Malgorzata N.  Drwal</span>, <span class="hlFld-ContribAuthor ">JÃ©rÃ©my  Desaphy</span>, <span class="hlFld-ContribAuthor ">Esther  Kellenberger</span>. </span><span class="cited-content_cbyCitation_article-title">Binding mode information improves fragment docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-019-0346-7" title="DOI URL">https://doi.org/10.1186/s13321-019-0346-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-019-0346-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-019-0346-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DBinding%252Bmode%252Binformation%252Bimproves%252Bfragment%252Bdocking%26aulast%3DJacquemard%26aufirst%3DC%25C3%25A9lien%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Purvi  Shah</span>, <span class="hlFld-ContribAuthor ">Leila F.  Abadi</span>, <span class="hlFld-ContribAuthor ">Shraddha  Gaikwad</span>, <span class="hlFld-ContribAuthor ">Dasharath  Chaudhari</span>, <span class="hlFld-ContribAuthor ">Varun  Kushwah</span>, <span class="hlFld-ContribAuthor ">Sanyog  Jain</span>, <span class="hlFld-ContribAuthor ">Kamlesh K.  Bhutani</span>, <span class="hlFld-ContribAuthor ">Smita  Kulkarni</span>, <span class="hlFld-ContribAuthor ">Inder P.  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Biological Evaluation of 8-Hydroxyquinoline-hydrazones for Anti-HIV-1âand Anticancer Potential. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2018,</strong> <em>3 </em>
                                    (38)
                                     , 10727-10731. <a href="https://doi.org/10.1002/slct.201802283" title="DOI URL">https://doi.org/10.1002/slct.201802283</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.201802283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.201802283%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252B8-Hydroxyquinoline-hydrazones%252Bfor%252BAnti-HIV-1%2525E2%252580%252589and%252BAnticancer%252BPotential%26aulast%3DShah%26aufirst%3DPurvi%26date%3D2018%26date%3D2018%26volume%3D3%26issue%3D38%26spage%3D10727%26epage%3D10731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simone  Di Micco</span>, <span class="hlFld-ContribAuthor ">Stefania  Terracciano</span>, <span class="hlFld-ContribAuthor ">Vincenza  Cantone</span>, <span class="hlFld-ContribAuthor ">Katrin  Fischer</span>, <span class="hlFld-ContribAuthor ">Andreas  Koeberle</span>, <span class="hlFld-ContribAuthor ">Antonio  Foglia</span>, <span class="hlFld-ContribAuthor ">Raffaele  Riccio</span>, <span class="hlFld-ContribAuthor ">Oliver  Werz</span>, <span class="hlFld-ContribAuthor ">Ines  Bruno</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Bifulco</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of new potent molecular entities able to inhibit mPGES-1. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>143 </em>, 1419-1427. <a href="https://doi.org/10.1016/j.ejmech.2017.10.039" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.10.039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.10.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.10.039%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnew%252Bpotent%252Bmolecular%252Bentities%252Bable%252Bto%252Binhibit%252BmPGES-1%26aulast%3DDi%2BMicco%26aufirst%3DSimone%26date%3D2018%26volume%3D143%26spage%3D1419%26epage%3D1427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bassem H.  Naguib</span>, <span class="hlFld-ContribAuthor ">Hala B.  El-Nassan</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of new thieno[2,3-
              b
              ]pyridine derivatives as pim-1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2016,</strong> <em>31 </em>
                                    (6)
                                     , 1718-1725. <a href="https://doi.org/10.3109/14756366.2016.1158711" title="DOI URL">https://doi.org/10.3109/14756366.2016.1158711</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3109/14756366.2016.1158711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3109%2F14756366.2016.1158711%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Bof%252Bnew%252Bthieno%25255B2%25252C3-%252Bb%252B%25255Dpyridine%252Bderivatives%252Bas%252Bpim-1%252Binhibitors%26aulast%3DNaguib%26aufirst%3DBassem%2BH.%26date%3D2016%26date%3D2016%26volume%3D31%26issue%3D6%26spage%3D1718%26epage%3D1725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashutosh  Kumar</span>, <span class="hlFld-ContribAuthor ">Kam Y. J.  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Prospective evaluation of shape similarity based pose prediction method in D3R Grand Challenge 2015. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2016,</strong> <em>30 </em>
                                    (9)
                                     , 685-693. <a href="https://doi.org/10.1007/s10822-016-9931-2" title="DOI URL">https://doi.org/10.1007/s10822-016-9931-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-016-9931-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-016-9931-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DProspective%252Bevaluation%252Bof%252Bshape%252Bsimilarity%252Bbased%252Bpose%252Bprediction%252Bmethod%252Bin%252BD3R%252BGrand%252BChallenge%252B2015%26aulast%3DKumar%26aufirst%3DAshutosh%26date%3D2016%26date%3D2016%26volume%3D30%26issue%3D9%26spage%3D685%26epage%3D693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Junhao  Xing</span>, <span class="hlFld-ContribAuthor ">Lingyun  Yang</span>, <span class="hlFld-ContribAuthor ">Hui  Li</span>, <span class="hlFld-ContribAuthor ">Qing  Li</span>, <span class="hlFld-ContribAuthor ">Leilei  Zhao</span>, <span class="hlFld-ContribAuthor ">Xinning  Wang</span>, <span class="hlFld-ContribAuthor ">Yuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Muxing  Zhou</span>, <span class="hlFld-ContribAuthor ">Jinpei  Zhou</span>, <span class="hlFld-ContribAuthor ">Huibin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of anthranilamide derivatives as potential factor Xa inhibitors: Drug design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>95 </em>, 388-399. <a href="https://doi.org/10.1016/j.ejmech.2015.03.052" title="DOI URL">https://doi.org/10.1016/j.ejmech.2015.03.052</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2015.03.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2015.03.052%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Banthranilamide%252Bderivatives%252Bas%252Bpotential%252Bfactor%252BXa%252Binhibitors%25253A%252BDrug%252Bdesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DXing%26aufirst%3DJunhao%26date%3D2015%26volume%3D95%26spage%3D388%26epage%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryan P.  Wurz</span>, <span class="hlFld-ContribAuthor ">Liping H.  Pettus</span>, <span class="hlFld-ContribAuthor ">Claire  Jackson</span>, <span class="hlFld-ContribAuthor ">Bin  Wu</span>, <span class="hlFld-ContribAuthor ">Hui-Ling  Wang</span>, <span class="hlFld-ContribAuthor ">Brad  Herberich</span>, <span class="hlFld-ContribAuthor ">Victor  Cee</span>, <span class="hlFld-ContribAuthor ">Brian A.  Lanman</span>, <span class="hlFld-ContribAuthor ">Anthony B.  Reed</span>, <span class="hlFld-ContribAuthor ">Frank  Chavez</span>, <span class="hlFld-ContribAuthor ">Thomas  Nixey</span>, <span class="hlFld-ContribAuthor ">Jimmy  Laszlo</span>, <span class="hlFld-ContribAuthor ">Paul  Wang</span>, <span class="hlFld-ContribAuthor ">Yen  Nguyen</span>, <span class="hlFld-ContribAuthor ">Christine  Sastri</span>, <span class="hlFld-ContribAuthor ">Nadia  Guerrero</span>, <span class="hlFld-ContribAuthor ">Jeff  Winston</span>, <span class="hlFld-ContribAuthor ">J. Russell  Lipford</span>, <span class="hlFld-ContribAuthor ">Matthew R.  Lee</span>, <span class="hlFld-ContribAuthor ">Kristin L.  Andrews</span>, <span class="hlFld-ContribAuthor ">Christopher  Mohr</span>, <span class="hlFld-ContribAuthor ">Yang  Xu</span>, <span class="hlFld-ContribAuthor ">Yihong  Zhou</span>, <span class="hlFld-ContribAuthor ">Darren L.  Reid</span>, <span class="hlFld-ContribAuthor ">Andrew S.  Tasker</span>. </span><span class="cited-content_cbyCitation_article-title">The discovery and optimization of aminooxadiazoles as potent Pim kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2015,</strong> <em>25 </em>
                                    (4)
                                     , 847-855. <a href="https://doi.org/10.1016/j.bmcl.2014.12.067" title="DOI URL">https://doi.org/10.1016/j.bmcl.2014.12.067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2014.12.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2014.12.067%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DThe%252Bdiscovery%252Band%252Boptimization%252Bof%252Baminooxadiazoles%252Bas%252Bpotent%252BPim%252Bkinase%252Binhibitors%26aulast%3DWurz%26aufirst%3DRyan%2BP.%26date%3D2015%26volume%3D25%26issue%3D4%26spage%3D847%26epage%3D855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jon A.  Erickson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Design of Kinase Inhibitors: A Practical Guide. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 157-183. <a href="https://doi.org/10.1007/978-1-4939-2486-8_13" title="DOI URL">https://doi.org/10.1007/978-1-4939-2486-8_13</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-2486-8_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-2486-8_13%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BDesign%252Bof%252BKinase%252BInhibitors%25253A%252BA%252BPractical%252BGuide%26aulast%3DErickson%26aufirst%3DJon%2BA.%26date%3D2015%26date%3D2015%26spage%3D157%26epage%3D183%26pub%3DSpringer%2520New%2520York%26atitle%3DFragment-Based%252BMethods%252Bin%252BDrug%252BDiscovery%26aulast%3DKlon%26aufirst%3DAnthony%2BE.%26date%3D2015%26volume%3D1289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu'ning  Song</span>, <span class="hlFld-ContribAuthor ">Hao  Xu</span>, <span class="hlFld-ContribAuthor ">Wenmin  Chen</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">8-Hydroxyquinoline: a privileged structure with a broad-ranging pharmacological potential. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2015,</strong> <em>6 </em>
                                    (1)
                                     , 61-74. <a href="https://doi.org/10.1039/C4MD00284A" title="DOI URL">https://doi.org/10.1039/C4MD00284A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00284A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00284A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3D8-Hydroxyquinoline%25253A%252Ba%252Bprivileged%252Bstructure%252Bwith%252Ba%252Bbroad-ranging%252Bpharmacological%252Bpotential%26aulast%3DSong%26aufirst%3DYu%2527ning%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D1%26spage%3D61%26epage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huaqing  Duan</span>, <span class="hlFld-ContribAuthor ">Zhong  Chen</span>, <span class="hlFld-ContribAuthor ">Li  Han</span>, <span class="hlFld-ContribAuthor ">Yulin  Feng</span>, <span class="hlFld-ContribAuthor ">Yongming  Zhu</span>, <span class="hlFld-ContribAuthor ">Shilin  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium-catalyzed chemoselective synthesis of indane-1,3-dione derivatives via tert-butyl isocyanide insertion. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2015,</strong> <em>13 </em>
                                    (24)
                                     , 6782-6788. <a href="https://doi.org/10.1039/C5OB00472A" title="DOI URL">https://doi.org/10.1039/C5OB00472A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C5OB00472A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC5OB00472A%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DPalladium-catalyzed%252Bchemoselective%252Bsynthesis%252Bof%252Bindane-1%25252C3-dione%252Bderivatives%252Bvia%252Btert-butyl%252Bisocyanide%252Binsertion%26aulast%3DDuan%26aufirst%3DHuaqing%26date%3D2015%26date%3D2015%26volume%3D13%26issue%3D24%26spage%3D6782%26epage%3D6788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jinho  Lee</span>, <span class="hlFld-ContribAuthor ">Kunal N.  More</span>, <span class="hlFld-ContribAuthor ">Seun-Ah  Yang</span>, <span class="hlFld-ContribAuthor ">Victor S.  Hong</span>. </span><span class="cited-content_cbyCitation_article-title">3,5-Bis(aminopyrimidinyl)indole Derivatives: Synthesis and Evaluation of Pim Kinase Inhibitory Activities. </span><span class="cited-content_cbyCitation_journal-name">Bulletin of the Korean Chemical Society</span><span> <strong>2014,</strong> <em>35 </em>
                                    (7)
                                     , 2123-2129. <a href="https://doi.org/10.5012/bkcs.2014.35.7.2123" title="DOI URL">https://doi.org/10.5012/bkcs.2014.35.7.2123</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5012/bkcs.2014.35.7.2123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5012%2Fbkcs.2014.35.7.2123%26sid%3Dliteratum%253Aachs%26jtitle%3DBulletin%2520of%2520the%2520Korean%2520Chemical%2520Society%26atitle%3D3%25252C5-Bis%252528aminopyrimidinyl%252529indole%252BDerivatives%25253A%252BSynthesis%252Band%252BEvaluation%252Bof%252BPim%252BKinase%252BInhibitory%252BActivities%26aulast%3DLee%26aufirst%3DJinho%26date%3D2014%26volume%3D35%26issue%3D7%26spage%3D2123%26epage%3D2129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kunal N.  More</span>, <span class="hlFld-ContribAuthor ">Hyo Weon  Jang</span>, <span class="hlFld-ContribAuthor ">Victor S.  Hong</span>, <span class="hlFld-ContribAuthor ">Jinho  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Pim kinase inhibitory and antiproliferative activity of a novel series of meridianin C derivatives. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>24 </em>
                                    (11)
                                     , 2424-2428. <a href="https://doi.org/10.1016/j.bmcl.2014.04.035" title="DOI URL">https://doi.org/10.1016/j.bmcl.2014.04.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2014.04.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2014.04.035%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPim%252Bkinase%252Binhibitory%252Band%252Bantiproliferative%252Bactivity%252Bof%252Ba%252Bnovel%252Bseries%252Bof%252Bmeridianin%252BC%252Bderivatives%26aulast%3DMore%26aufirst%3DKunal%2BN.%26date%3D2014%26volume%3D24%26issue%3D11%26spage%3D2424%26epage%3D2428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mingfeng  Shao</span>, <span class="hlFld-ContribAuthor ">Yiming  Yuan</span>, <span class="hlFld-ContribAuthor ">Kun  Yu</span>, <span class="hlFld-ContribAuthor ">Kai  Lei</span>, <span class="hlFld-ContribAuthor ">Guonian  Zhu</span>, <span class="hlFld-ContribAuthor ">Lijuan  Chen</span>, <span class="hlFld-ContribAuthor ">Mingli  Xiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and identification of PIM-1 kinase inhibitors through a hybrid screening approach. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2014,</strong> <em>18 </em>
                                    (2)
                                     , 335-344. <a href="https://doi.org/10.1007/s11030-014-9504-z" title="DOI URL">https://doi.org/10.1007/s11030-014-9504-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-014-9504-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-014-9504-z%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DDiscovery%252Band%252Bidentification%252Bof%252BPIM-1%252Bkinase%252Binhibitors%252Bthrough%252Ba%252Bhybrid%252Bscreening%252Bapproach%26aulast%3DShao%26aufirst%3DMingfeng%26date%3D2014%26date%3D2014%26volume%3D18%26issue%3D2%26spage%3D335%26epage%3D344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jinho  Lee</span>, <span class="hlFld-ContribAuthor ">Jongseong  Park</span>, <span class="hlFld-ContribAuthor ">Victor Sukbong  Hong</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Evaluation of 5-(3-(Pyrazin-2-yl)benzylidene)thiazolidine-2,4-dione Derivatives as PanâPim Kinases Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">CHEMICAL & PHARMACEUTICAL BULLETIN</span><span> <strong>2014,</strong> <em>62 </em>
                                    (9)
                                     , 906-914. <a href="https://doi.org/10.1248/cpb.c14-00325" title="DOI URL">https://doi.org/10.1248/cpb.c14-00325</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1248/cpb.c14-00325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1248%2Fcpb.c14-00325%26sid%3Dliteratum%253Aachs%26jtitle%3DCHEMICAL%2520%2526%2520PHARMACEUTICAL%2520BULLETIN%26atitle%3DSynthesis%252Band%252BEvaluation%252Bof%252B5-%2525283-%252528Pyrazin-2-yl%252529benzylidene%252529thiazolidine-2%25252C4-dione%252BDerivatives%252Bas%252BPan%2525E2%252580%252593Pim%252BKinases%252BInhibitors%26aulast%3DLee%26aufirst%3DJinho%26date%3D2014%26volume%3D62%26issue%3D9%26spage%3D906%26epage%3D914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Douglas B.  Kell</span>. </span><span class="cited-content_cbyCitation_article-title">Finding novel pharmaceuticals in the systems biology era using multiple effective drug targets, phenotypic screening and knowledge of transporters: where drug discovery went wrong and how to fix it. </span><span class="cited-content_cbyCitation_journal-name">FEBS Journal</span><span> <strong>2013,</strong> <em>280 </em>
                                    (23)
                                     , 5957-5980. <a href="https://doi.org/10.1111/febs.12268" title="DOI URL">https://doi.org/10.1111/febs.12268</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/febs.12268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Ffebs.12268%26sid%3Dliteratum%253Aachs%26jtitle%3DFEBS%2520Journal%26atitle%3DFinding%252Bnovel%252Bpharmaceuticals%252Bin%252Bthe%252Bsystems%252Bbiology%252Bera%252Busing%252Bmultiple%252Beffective%252Bdrug%252Btargets%25252C%252Bphenotypic%252Bscreening%252Band%252Bknowledge%252Bof%252Btransporters%25253A%252Bwhere%252Bdrug%252Bdiscovery%252Bwent%252Bwrong%252Band%252Bhow%252Bto%252Bfix%252Bit%26aulast%3DKell%26aufirst%3DDouglas%2BB.%26date%3D2013%26date%3D2013%26volume%3D280%26issue%3D23%26spage%3D5957%26epage%3D5980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rocco  Caliandro</span>, <span class="hlFld-ContribAuthor ">Danilo Benny  Belviso</span>, <span class="hlFld-ContribAuthor ">Brunella Maria  Aresta</span>, <span class="hlFld-ContribAuthor ">Modesto  de Candia</span>, <span class="hlFld-ContribAuthor ">Cosimo Damiano  Altomare</span>. </span><span class="cited-content_cbyCitation_article-title">Protein crystallography and fragment-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2013,</strong> <em>5 </em>
                                    (10)
                                     , 1121-1140. <a href="https://doi.org/10.4155/fmc.13.84" title="DOI URL">https://doi.org/10.4155/fmc.13.84</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.13.84&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.13.84%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DProtein%252Bcrystallography%252Band%252Bfragment-based%252Bdrug%252Bdesign%26aulast%3DCaliandro%26aufirst%3DRocco%26date%3D2013%26volume%3D5%26issue%3D10%26spage%3D1121%26epage%3D1140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Douglas B.  Kell</span>, <span class="hlFld-ContribAuthor ">Paul D.  Dobson</span>, <span class="hlFld-ContribAuthor ">Elizabeth  Bilsland</span>, <span class="hlFld-ContribAuthor ">Stephen G.  Oliver</span>. </span><span class="cited-content_cbyCitation_article-title">The promiscuous binding of pharmaceutical drugs and their transporter-mediated uptake into cells: what we (need to) know and how we can do so. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2013,</strong> <em>18 </em>
                                    (5-6)
                                     , 218-239. <a href="https://doi.org/10.1016/j.drudis.2012.11.008" title="DOI URL">https://doi.org/10.1016/j.drudis.2012.11.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2012.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2012.11.008%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DThe%252Bpromiscuous%252Bbinding%252Bof%252Bpharmaceutical%252Bdrugs%252Band%252Btheir%252Btransporter-mediated%252Buptake%252Binto%252Bcells%25253A%252Bwhat%252Bwe%252B%252528need%252Bto%252529%252Bknow%252Band%252Bhow%252Bwe%252Bcan%252Bdo%252Bso%26aulast%3DKell%26aufirst%3DDouglas%2BB.%26date%3D2013%26volume%3D18%26issue%3D5-6%26spage%3D218%26epage%3D239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zorik  Chilingaryan</span>, <span class="hlFld-ContribAuthor ">Zhou  Yin</span>, <span class="hlFld-ContribAuthor ">Aaron J.  Oakley</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Screening by Protein Crystallography: Successes and Pitfalls. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2012,</strong> <em>13 </em>
                                    (12)
                                     , 12857-12879. <a href="https://doi.org/10.3390/ijms131012857" title="DOI URL">https://doi.org/10.3390/ijms131012857</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms131012857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms131012857%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DFragment-Based%252BScreening%252Bby%252BProtein%252BCrystallography%25253A%252BSuccesses%252Band%252BPitfalls%26aulast%3DChilingaryan%26aufirst%3DZorik%26date%3D2012%26date%3D2012%26volume%3D13%26issue%3D12%26spage%3D12857%26epage%3D12879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Les A.  Dakin</span>, <span class="hlFld-ContribAuthor ">Michael H.  Block</span>, <span class="hlFld-ContribAuthor ">Huawei  Chen</span>, <span class="hlFld-ContribAuthor ">Erin  Code</span>, <span class="hlFld-ContribAuthor ">James E.  Dowling</span>, <span class="hlFld-ContribAuthor ">Xiaomei  Feng</span>, <span class="hlFld-ContribAuthor ">Andrew D.  Ferguson</span>, <span class="hlFld-ContribAuthor ">Isabelle  Green</span>, <span class="hlFld-ContribAuthor ">Alexander W.  Hird</span>, <span class="hlFld-ContribAuthor ">Tina  Howard</span>, <span class="hlFld-ContribAuthor ">Erika K.  Keeton</span>, <span class="hlFld-ContribAuthor ">Michelle L.  Lamb</span>, <span class="hlFld-ContribAuthor ">Paul D.  Lyne</span>, <span class="hlFld-ContribAuthor ">Hannah  Pollard</span>, <span class="hlFld-ContribAuthor ">Jon  Read</span>, <span class="hlFld-ContribAuthor ">Allan J.  Wu</span>, <span class="hlFld-ContribAuthor ">Tao  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaolan  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel benzylidene-1,3-thiazolidine-2,4-diones as potent and selective inhibitors of the PIM-1, PIM-2, and PIM-3 protein kinases. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2012,</strong> <em>22 </em>
                                    (14)
                                     , 4599-4604. <a href="https://doi.org/10.1016/j.bmcl.2012.05.098" title="DOI URL">https://doi.org/10.1016/j.bmcl.2012.05.098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2012.05.098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2012.05.098%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bnovel%252Bbenzylidene-1%25252C3-thiazolidine-2%25252C4-diones%252Bas%252Bpotent%252Band%252Bselective%252Binhibitors%252Bof%252Bthe%252BPIM-1%25252C%252BPIM-2%25252C%252Band%252BPIM-3%252Bprotein%252Bkinases%26aulast%3DDakin%26aufirst%3DLes%2BA.%26date%3D2012%26volume%3D22%26issue%3D14%26spage%3D4599%26epage%3D4604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Hydrogen-bonding networks (green dotted lines for heteroatom distance <3.0 Ã) around catalytic domain K67 highlighted for each of the four structures used in virtual screening calculations. The 2D structures of the parent inhibitors associated with each complex are shown for reference, together with their Pim-1 IC<sub>50</sub>/<i>K</i><sub>i</sub> data and LEs. The structures used are (a) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OBJ">2OBJ</a>,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (b) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGQ">3BGQ</a>,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (c) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGP">3BGP</a>,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and (d) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGZ">3BGZ</a>.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The conformational switching of E89 in structure b and the general variability in positioning and number of water-mediated hydrogen bonds are of particular note. For <b>1a</b> the hydroxypyridine tautomer was hypothesized as preferred due to the presence of a stabilizing internal H bond from the pendant phenolic hydroxyl. It was this tautomer that was used in protein preparation. If the generally preferred pyridone tautomer is used instead, the pendant phenolic hydroxyl hydrogen can no longer form either an inter- or intramolecular hydrogen bond, as it cannot pair with the gatekeeper water. This leads to an unpaired hydrogen bond donor exposed to a hydrophobic environment. We suspect that the internal hydrogen bond and the ortho cyano substituent either stabilize this tautomer or allow the formation of the phenolate anion given the preference of this active site region to bind acidic functionality.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Top scoring docked structures (orange ligand and pink protein) and crystal structure (green ligand and purple protein) for five hits from the needle screen. The reference structure is taken from the top score generating reference structure in each case. Note that water molecules associated with each reference structure are shown with pink hydrogens. 2D structures, Pim-1 IC<sub>50</sub>s, and LEs are shown for each ligand and key hydrogen bonds are highlighted (green dotted lines for heteroatom distance < 3.0 Ã, orange for distance â¼ 3.2 Ã). The PDB codes of the crystal structure of these compounds are as follows: (a) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBW">3VBW</a>, (b) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VBV">2VBV</a>, (c) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBX">3VBX</a>, (d) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBY">3VBY</a>, and (e) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBT">3VBT</a>. Note that <b>2a</b> may exist as an enolate tautomer; however, this is not how it was modeled in the virtual screening studies. Note also that for <b>2c</b> an alternate location is included for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=D186">D186</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Pim-1 THZ screening hit<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> used in chimera lead design with Pim-1 IC<sub>50</sub> and LE shown for reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structures of carboxylate-containing hits from internal SPR fragment screen<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> [PDB reference codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R00">3R00</a> (<b>4a</b>) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R01">3R01</a> (<b>4b</b>)]. Key hydrogen bonds are highlighted with Pim-1 IC<sub>50</sub>s and LEs shown for reference. Note that the protein structures of these complexes have essentially identical tertiary structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. THZ crystal structure of <b>3</b> (pink) versus unconstrained Glide XP/SP highest scoring docking modes (identical and in orange), with <b>4b</b> crystal structure (green) of carboxylate-containing hit shown for reference. Proteins of both crystal structures are also essentially superimposable, so only the protein structure of <b>4b</b> has been included. The crystal structure of <b>3</b> has been deposited in the PDB using code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VC4">3VC4</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Alternate Glide SP docking mode for deprotonated model of <b>3</b> (orange) superimposed on reference crystal structure <b>4a</b> (green) used in docking.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Alternate Glide SP docking mode of <b>3</b> (orange) superimposed on crystal structure of molecule <b>5</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F2A">3F2A</a>). The protein reference structure shown is also derived from <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F2A">3F2A</a>. Pim-1 IC<sub>50</sub> and LE of <b>5</b> shown for reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Crystal structure of <b>6</b> (green) superimposed on constrained Glide SP (orange), unconstrained Glide SP (pink), and Glide XP (maroon) top scoring binding modes on docking to the crystal structure of <b>5</b> (key hydrogen bonds highlighted). The crystal structure of <b>6</b> has been deposited in the PDB using code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBQ">3VBQ</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/medium/jm-2011-014698_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Crystal structure (light green) and the docked binding modes of <b>2b</b> for the four virtual screens run. Ligands have been extracted from their respective binding poses after superposition of the Pim-1 references structures used in docking.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-6/jm2014698/production/images/large/jm-2011-014698_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm2014698&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20155" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20155" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 27 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giangreco, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortenson, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span> </span><span class="NLM_article-title">Docking performance of fragments and druglike compounds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5422</span><span class="NLM_x">â</span> <span class="NLM_lpage">5431</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200558u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFCgsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5422-5431&author=M.+L.+Verdonkauthor=I.+Giangrecoauthor=R.+J.+Hallauthor=O.+Korbauthor=P.+N.+Mortensonauthor=C.+W.+Murray&title=Docking+performance+of+fragments+and+druglike+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Performance of Fragments and Druglike Compounds</span></div><div class="casAuthors">Verdonk, Marcel L.; Giangreco, Ilenia; Hall, Richard J.; Korb, Oliver; Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5422-5431</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper addresses two questions of key interest to researchers working with protein-ligand docking methods: (i) Why is there such a large variation in docking performance between different test sets reported in the literature (ii) Are fragments more difficult to dock than druglike compds..  To answer these, the authors construct a test set of inhouse x-ray structures of protein-ligand complexes from drug discovery projects, half of which contain fragment ligands, the other half drug-like ligands.  The authors find that a key factor affecting docking performance is ligand efficiency (LE).  High LE compds. are significantly easier to dock than low LE compds., which the authors believe could explain the differences obsd. between test sets reported in the literature.  There is no significant difference in docking performance between fragments and drug-like compds., but the reasons why dockings fail appear to be different.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVFTOFvmiFxrVg90H21EOLACvtfcHk0lg36Z3se5CeaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFCgsLo%253D&md5=8d7a5519d6e7fa2ab403ea76b8b9a586</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm200558u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200558u%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DGiangreco%26aufirst%3DI.%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DDocking%2520performance%2520of%2520fragments%2520and%2520druglike%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5422%26epage%3D5431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">SÃ¡ndor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiss, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KeserÅ±, G. M.</span><span> </span><span class="NLM_article-title">Virtual fragment docking by Glide: A validation study on 190 proteinâfragment complexes</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1165</span><span class="NLM_x">â</span> <span class="NLM_lpage">1172</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci1000407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVaksLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1165-1172&author=M.+S%C3%A1ndorauthor=R.+Kissauthor=G.+M.+Keser%C5%B1&title=Virtual+fragment+docking+by+Glide%3A+A+validation+study+on+190+protein%E2%80%93fragment+complexes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Fragment Docking by Glide: a Validation Study on 190 Protein-Fragment Complexes</span></div><div class="casAuthors">Sandor, Mark; Kiss, Robert; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1165-1172</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The docking accuracy of Glide was evaluated using 16 different docking protocols on 190 protein-fragment complexes representing 78 targets.  Std. precision docking (Glide SP) based protocols showed the best performance.  The av. root-mean-square deviation (RMSD) between the docked and cocrystd. poses achieved by Glide SP with pre- and postprocessing was 1.17 Ã, and an acceptable binding mode with RMSD. < 2 Ã could be found in 80% of the cases.  Comparison of the docking results produced by different protocols suggests that the sampling efficacy of Glide is adequate for fragment docking.  The docking accuracy seems to be limited by the performance of scoring schemes, which is supported by the weak correlation between exptl. binding affinities and GlideScores.  Cross-docking expts. performed on 8 targets represented by 63 complexes revealed that Glide SP gave similar results to that of the computationally more intensive Glide XP.  The av. RMSD achieved by Glide SP with pre- and postprocessing was 2.06 Ã, and an acceptable binding mode with RMSD < 2 Ã could be found in 63% of the cases.  These cross-docking results were improved significantly selecting the optimal X-ray structure for each target (av. RMSD = 1.3 Ã, success rate = 77%), indicating the importance of enrichment studies and the use of multiple X-ray structures in virtual fragment screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodKfjH9BbnrLVg90H21EOLACvtfcHk0lg36Z3se5CeaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVaksLY%253D&md5=35d7acb7374360828beb668e2f207bab</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fci1000407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1000407%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1ndor%26aufirst%3DM.%26aulast%3DKiss%26aufirst%3DR.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DVirtual%2520fragment%2520docking%2520by%2520Glide%253A%2520A%2520validation%2520study%2520on%2520190%2520protein%25E2%2580%2593fragment%2520complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1165%26epage%3D1172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Amaravadi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. B.</span><span> </span><span class="NLM_article-title">The survival kinases Akt and Pim as potential pharmacological targets</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">2618</span><span class="NLM_x">â</span> <span class="NLM_lpage">2624</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2005&pages=2618-2624&author=R.+Amaravadiauthor=C.+B.+Thompson&title=The+survival+kinases+Akt+and+Pim+as+potential+pharmacological+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmaravadi%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DThe%2520survival%2520kinases%2520Akt%2520and%2520Pim%2520as%2520potential%2520pharmacological%2520targets%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2005%26volume%3D115%26spage%3D2618%26epage%3D2624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Amson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigaux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Przedborski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flandrin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givol, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telerman, A.</span><span> </span><span class="NLM_article-title">The human protooncogene product p33pim is expressed during fetal hematopoiesis and in diverse leukemias</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">8857</span><span class="NLM_x">â</span> <span class="NLM_lpage">8861</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=8857-8861&author=R.+Amsonauthor=F.+Sigauxauthor=S.+Przedborskiauthor=G.+Flandrinauthor=D.+Givolauthor=A.+Telerman&title=The+human+protooncogene+product+p33pim+is+expressed+during+fetal+hematopoiesis+and+in+diverse+leukemias"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmson%26aufirst%3DR.%26aulast%3DSigaux%26aufirst%3DF.%26aulast%3DPrzedborski%26aufirst%3DS.%26aulast%3DFlandrin%26aufirst%3DG.%26aulast%3DGivol%26aufirst%3DD.%26aulast%3DTelerman%26aufirst%3DA.%26atitle%3DThe%2520human%2520protooncogene%2520product%2520p33pim%2520is%2520expressed%2520during%2520fetal%2520hematopoiesis%2520and%2520in%2520diverse%2520leukemias%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1989%26volume%3D86%26spage%3D8857%26epage%3D8861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Shah, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K-G</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lilly, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salto-Tellez, M</span><span> </span><span class="NLM_article-title">Potential roles for the PIM-1 kinase in human cancerâA molecular and therapeutic appraisal</span> <span class="citation_source-journal">Eur. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">2144</span><span class="NLM_x">â</span> <span class="NLM_lpage">2151</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2008&pages=2144-2151&author=N.+Shahauthor=B.+Pangauthor=K-G+Yeohauthor=S.+Thornauthor=C.+S.+Chenauthor=M.+B.+Lillyauthor=M+Salto-Tellez&title=Potential+roles+for+the+PIM-1+kinase+in+human+cancer%E2%80%94A+molecular+and+therapeutic+appraisal"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DN.%26aulast%3DPang%26aufirst%3DB.%26aulast%3DYeoh%26aufirst%3DK-G%26aulast%3DThorn%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DC.%2BS.%26aulast%3DLilly%26aufirst%3DM.%2BB.%26aulast%3DSalto-Tellez%26aufirst%3DM%26atitle%3DPotential%2520roles%2520for%2520the%2520PIM-1%2520kinase%2520in%2520human%2520cancer%25E2%2580%2594A%2520molecular%2520and%2520therapeutic%2520appraisal%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2008%26volume%3D44%26spage%3D2144%26epage%3D2151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Magnuson, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reeves, R.</span><span> </span><span class="NLM_article-title">Why target PIM-1 for cancer diagnosis and treatment?</span> <span class="citation_source-journal">Future Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1461</span><span class="NLM_x">â</span> <span class="NLM_lpage">1478</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.2217%2Ffon.10.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=20919829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1GnsrjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1461-1478&author=N.+S.+Magnusonauthor=Z.+Wangauthor=G.+Dingauthor=R.+Reeves&title=Why+target+PIM-1+for+cancer+diagnosis+and+treatment%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Why target PIM1 for cancer diagnosis and treatment?</span></div><div class="casAuthors">Magnuson, Nancy S.; Wang, Zeping; Ding, Gang; Reeves, Raymond</div><div class="citationInfo"><span class="NLM_cas:title">Future Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1461-1478</span>CODEN:
                <span class="NLM_cas:coden">FOUNBN</span>;
        ISSN:<span class="NLM_cas:issn">1479-6694</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  The highly conserved proto-oncogenic protein PIM1 is an unusual serine or threonine kinase, in part because it is constitutively active.  Overexpression of PIM1 exptl. leads to tumor formation in mice, while complete knockout of the protein has no observable phenotype.  It appears to contribute to cancer development in three major ways when it is overexpressed; by inhibiting apoptosis, by promoting cell proliferation and by promoting genomic instability.  Expression in normal tissues is nearly undetectable.  However, in hematopoietic malignancies and in a variety of solid tumors, increased PIM1 expression has been shown to correlate with the stage of disease.  This characteristic suggests it can serve as a useful biomarker for cancer diagnosis and prognosis.  Several specific and potent inhibitors of PIM1's kinase activity have also been shown to induce apoptotic death of cancer cells, to sensitize cancer cells to chemotherapy and to synergize with other anti-tumor agents, thus making it an attractive therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8Np_ubohH1bVg90H21EOLACvtfcHk0liYSrPDx22k4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1GnsrjE&md5=c072a8db7336d7c7dc5032c56b9a923b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2217%2Ffon.10.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Ffon.10.106%26sid%3Dliteratum%253Aachs%26aulast%3DMagnuson%26aufirst%3DN.%2BS.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DDing%26aufirst%3DG.%26aulast%3DReeves%26aufirst%3DR.%26atitle%3DWhy%2520target%2520PIM-1%2520for%2520cancer%2520diagnosis%2520and%2520treatment%253F%26jtitle%3DFuture%2520Oncol.%26date%3D2010%26volume%3D6%26spage%3D1461%26epage%3D1478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Kim, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, D.</span><span> </span><span class="NLM_article-title">Constitutively activated FLT3 phosphorylates BAD partially through Pim-1 Br</span> <span class="citation_source-journal">J. Haematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">500</span><span class="NLM_x">â</span> <span class="NLM_lpage">509</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.1111%2Fj.1365-2141.2006.06225.x" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=500-509&author=K.+T.+Kimauthor=M.+Levisauthor=D.+Small&title=Constitutively+activated+FLT3+phosphorylates+BAD+partially+through+Pim-1+Br"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2141.2006.06225.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2141.2006.06225.x%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BT.%26aulast%3DLevis%26aufirst%3DM.%26aulast%3DSmall%26aufirst%3DD.%26atitle%3DConstitutively%2520activated%2520FLT3%2520phosphorylates%2520BAD%2520partially%2520through%2520Pim-1%2520Br%26jtitle%3DJ.%2520Haematol.%26date%3D2006%26volume%3D124%26spage%3D500%26epage%3D509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Qian, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Studts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barringer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kronkaitis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mische, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farmer, B.</span><span> </span><span class="NLM_article-title">Structural basis of constitutive activity and a unique nucleotide binding mode of human Pim-1 kinase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">6130</span><span class="NLM_x">â</span> <span class="NLM_lpage">6137</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=6130-6137&author=K.+C.+Qianauthor=L.+Wangauthor=E.+R.+Hickeyauthor=J.+Studtsauthor=K.+Barringerauthor=C.+Pengauthor=A.+Kronkaitisauthor=J.+Liauthor=A.+Whiteauthor=S.+Mischeauthor=B.+Farmer&title=Structural+basis+of+constitutive+activity+and+a+unique+nucleotide+binding+mode+of+human+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DK.%2BC.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DStudts%26aufirst%3DJ.%26aulast%3DBarringer%26aufirst%3DK.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DKronkaitis%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DMische%26aufirst%3DS.%26aulast%3DFarmer%26aufirst%3DB.%26atitle%3DStructural%2520basis%2520of%2520constitutive%2520activity%2520and%2520a%2520unique%2520nucleotide%2520binding%2520mode%2520of%2520human%2520Pim-1%2520kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D6130%26epage%3D6137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Schenone, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tintori, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botta, M.</span><span> </span><span class="NLM_article-title">Using insights into Pim-1 structure to design new anticancer drugs</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3964</span><span class="NLM_x">â</span> <span class="NLM_lpage">3978</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.2174%2F138161210794454996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=21158732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFKqtLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=3964-3978&author=S.+Schenoneauthor=C.+Tintoriauthor=M.+Botta&title=Using+insights+into+Pim-1+structure+to+design+new+anticancer+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Using insights into Pim1 structure to design new anticancer drugs</span></div><div class="casAuthors">Schenone, Silvia; Tintori, Cristina; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">3964-3978</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Human Pim1 (proviral integration site for Moloney murine leukemia virus) kinase is a 313-amino acid serine-threonine kinase that possesses several biol. functions in cell survival, proliferation and differentiation, and its over-expression has been obsd. in a no. of human cancers.  Indeed, this kinase is a proto-oncogene that has been implicated in early transformation and tumor progression, esp. in hematopoietic malignancies and prostate carcinoma where it is a marker of a poor prognosis.  For these reasons, Pim1 is emerging as an important target in drug discovery, and many Pim1 inhibitors have been reported in the last three years.  The challenge of this research is to obtain compds. that specifically inhibit only Pim1 and not Pim2 and Pim3, the other members of the Pim family, with the aim of providing selective inhibitors as potential therapeutic agents and also of studying the different roles of three enzymes.  In this review Pim1 functions and Pim1 role in human cancer are summarized, but the primary focus of the article is on the Pim1 three-dimensional structure that was deeply analyzed by a detailed inspection of the available crystallog. data and all complexes of small mol. inhibitors reported in the literature to this point.  Finally, the use of mol. modeling techniques for the identification and optimization of Pim1 inhibitors is extensively discussed.  This data collection, which to the best of the authors' knowledge was not previously reviewed in such detail, could offer a useful tool for medicinal chemists working in the field of small mol. kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEKuEDdrcDV7Vg90H21EOLACvtfcHk0liHz0hWdfPZ7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFKqtLs%253D&md5=1f446cd7cdd6ad5fb098e4471e807c27</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F138161210794454996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161210794454996%26sid%3Dliteratum%253Aachs%26aulast%3DSchenone%26aufirst%3DS.%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DUsing%2520insights%2520into%2520Pim-1%2520structure%2520to%2520design%2520new%2520anticancer%2520drugs%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2010%26volume%3D16%26spage%3D3964%26epage%3D3978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Xiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asmussen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biemann, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bishop, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Good, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gladysheva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jancsics, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papoulis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skerlj, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stepp, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, R. R.</span><span> </span><span class="NLM_article-title">The discovery of novel benzofuran-2-carboxylic acids as potent Pim-1 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">3050</span><span class="NLM_x">â</span> <span class="NLM_lpage">3056</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=3050-3056&author=Y.+Xiangauthor=B.+Hirthauthor=G.+Asmussenauthor=H.+P.+Biemannauthor=K.+A.+Bishopauthor=A.+Goodauthor=M.+Fitzgeraldauthor=T.+Gladyshevaauthor=A.+Jainauthor=K.+Jancsicsauthor=J.+Liuauthor=M.+Metzauthor=A.+Papoulisauthor=R.+Skerljauthor=J.+D.+Steppauthor=R.+R.+Wei&title=The+discovery+of+novel+benzofuran-2-carboxylic+acids+as+potent+Pim-1+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DHirth%26aufirst%3DB.%26aulast%3DAsmussen%26aufirst%3DG.%26aulast%3DBiemann%26aufirst%3DH.%2BP.%26aulast%3DBishop%26aufirst%3DK.%2BA.%26aulast%3DGood%26aufirst%3DA.%26aulast%3DFitzgerald%26aufirst%3DM.%26aulast%3DGladysheva%26aufirst%3DT.%26aulast%3DJain%26aufirst%3DA.%26aulast%3DJancsics%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DMetz%26aufirst%3DM.%26aulast%3DPapoulis%26aufirst%3DA.%26aulast%3DSkerlj%26aufirst%3DR.%26aulast%3DStepp%26aufirst%3DJ.%2BD.%26aulast%3DWei%26aufirst%3DR.%2BR.%26atitle%3DThe%2520discovery%2520of%2520novel%2520benzofuran-2-carboxylic%2520acids%2520as%2520potent%2520Pim-1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D3050%26epage%3D3056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Boehm, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boehringer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bur, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gmuender, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kostrewa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuehne, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luebbers, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meunier-Keller, N</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, F.</span><span> </span><span class="NLM_article-title">Novel inhibitors of DNA gyrase: 3D structure based biased needle screening, hit validation by biophysical methods, and 3D guided optimization. A promising alternative to random screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">2664</span><span class="NLM_x">â</span> <span class="NLM_lpage">2674</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000017s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BD3cXktFOgsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=2664-2674&author=H.-J.+Boehmauthor=M.+Boehringerauthor=D.+Burauthor=H.+Gmuenderauthor=H.+Walterauthor=W.+Klausauthor=D.+Kostrewaauthor=H.+Kuehneauthor=T.+Luebbersauthor=N+Meunier-Kellerauthor=F.+Mueller&title=Novel+inhibitors+of+DNA+gyrase%3A+3D+structure+based+biased+needle+screening%2C+hit+validation+by+biophysical+methods%2C+and+3D+guided+optimization.+A+promising+alternative+to+random+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Novel inhibitors of DNA gyrase: 3D structure based biased needle screening, hit validation by biophysical methods, and 3D guided optimization. A promising alternative to random screening</span></div><div class="casAuthors">Boehm, Hans-Joachim; Boehringer, Markus; Bur, Daniel; Gmuender, Hans; Huber, Walter; Klaus, Werner; Kostrewa, Dirk; Kuehne, Holger; Luebbers, Thomas; Meunier-Keller, Nathalie; Mueller, Francis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2664-2674</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Random screening provided no suitable lead structures in a search for novel inhibitors of the bacterial enzyme DNA gyrase.  Therefore, an alternative approach had to be developed.  Relying on the detailed 3D structural information of the targeted ATP binding site, our approach combines as key techniques (1) an in silico screening for potential low mol. wt. inhibitors, (2) a biased high throughput DNA gyrase screen, (3) validation of the screening hits by biophys. methods, and (4) a 3D-guided optimization process.  When the in silico screening was performed, the initial data set contg. 350,000 compds. could be reduced to 3000 mols.  Testing these 3000 selected compds. in the DNA gyrase assay provided 150 hits clustered in 14 classes.  Seven classes could be validated as true, novel DNA gyrase inhibitors that act by binding to the ATP binding site located on subunit B: phenols, 2-aminotriazines, 4-aminopyrimidines, 2-aminopyrimidines, pyrrolopyrimidines, indazoles, and 2-hydroxymethylindoles.  The 3D guided optimization provided highly potent DNA gyrase inhibitors, e.g., the 3,4-disubstituted indazole (I) being a 10 times more potent DNA gyrase inhibitor than novobiocin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosDAeZrjuWibVg90H21EOLACvtfcHk0lhqLeEon0buBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXktFOgsrc%253D&md5=bb7f072e82e89371e6e2da109bbd84af</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm000017s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000017s%26sid%3Dliteratum%253Aachs%26aulast%3DBoehm%26aufirst%3DH.-J.%26aulast%3DBoehringer%26aufirst%3DM.%26aulast%3DBur%26aufirst%3DD.%26aulast%3DGmuender%26aufirst%3DH.%26aulast%3DWalter%26aufirst%3DH.%26aulast%3DKlaus%26aufirst%3DW.%26aulast%3DKostrewa%26aufirst%3DD.%26aulast%3DKuehne%26aufirst%3DH.%26aulast%3DLuebbers%26aufirst%3DT.%26aulast%3DMeunier-Keller%26aufirst%3DN%26aulast%3DMueller%26aufirst%3DF.%26atitle%3DNovel%2520inhibitors%2520of%2520DNA%2520gyrase%253A%25203D%2520structure%2520based%2520biased%2520needle%2520screening%252C%2520hit%2520validation%2520by%2520biophysical%2520methods%252C%2520and%25203D%2520guided%2520optimization.%2520A%2520promising%2520alternative%2520to%2520random%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D2664%26epage%3D2674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrows, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffen, E. J.</span><span> </span><span class="NLM_article-title">Fragment screening: An introduction</span> <span class="citation_source-journal">Mol. Biosyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">â</span> <span class="NLM_lpage">446</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2006&pages=430-446&author=A.+R.+Leachauthor=M.+M.+Hannauthor=J.+N.+Burrowsauthor=E.+J.+Griffen&title=Fragment+screening%3A+An+introduction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DBurrows%26aufirst%3DJ.%2BN.%26aulast%3DGriffen%26aufirst%3DE.%2BJ.%26atitle%3DFragment%2520screening%253A%2520An%2520introduction%26jtitle%3DMol.%2520Biosyst.%26date%3D2006%26volume%3D9%26spage%3D430%26epage%3D446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Xia, Z</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knaak, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beharry, Z. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McInnes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraft, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C. D.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of novel inhibitors of Pim-1 and Pim-2 protein kinases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">74</span><span class="NLM_x">â</span> <span class="NLM_lpage">86</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800937p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=74-86&author=Z+Xiaauthor=C.+Knaakauthor=J.+Maauthor=Z.+M.+Beharryauthor=C.+McInnesauthor=W.+Wangauthor=A.+S.+Kraftauthor=C.+D.+Smith&title=Synthesis+and+evaluation+of+novel+inhibitors+of+Pim-1+and+Pim-2+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm800937p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800937p%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DZ%26aulast%3DKnaak%26aufirst%3DC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DBeharry%26aufirst%3DZ.%2BM.%26aulast%3DMcInnes%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKraft%26aufirst%3DA.%2BS.%26aulast%3DSmith%26aufirst%3DC.%2BD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520inhibitors%2520of%2520Pim-1%2520and%2520Pim-2%2520protein%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D74%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Qian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cywin, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farmer, B. T.,  II</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kashem, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magboo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pack, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prokopowicz, A.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welzel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morwick, T.</span><span> </span><span class="NLM_article-title">Hit to lead account of the discovery of a new class of inhibitors of Pim kinases and crystallographic studies revealing an unusual kinase binding mode</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1814</span><span class="NLM_x">â</span> <span class="NLM_lpage">1827</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm801242y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1814-1827&author=K.+Qianauthor=L.+Wangauthor=C.+L.+Cywinauthor=B.+T.+Farmerauthor=E.+Hickeyauthor=C.+Homonauthor=S.+Jakesauthor=M.+A.+Kashemauthor=G.+Leeauthor=S.+Leonardauthor=J.+Liauthor=R.+Magbooauthor=W.+Maoauthor=E.+Packauthor=C.+Pengauthor=A.+Prokopowiczauthor=M.+Welzelauthor=J.+Wolakauthor=T.+Morwick&title=Hit+to+lead+account+of+the+discovery+of+a+new+class+of+inhibitors+of+Pim+kinases+and+crystallographic+studies+revealing+an+unusual+kinase+binding+mode"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm801242y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801242y%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DCywin%26aufirst%3DC.%2BL.%26aulast%3DFarmer%26aufirst%3DB.%2BT.%26aulast%3DHickey%26aufirst%3DE.%26aulast%3DHomon%26aufirst%3DC.%26aulast%3DJakes%26aufirst%3DS.%26aulast%3DKashem%26aufirst%3DM.%2BA.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DLeonard%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMagboo%26aufirst%3DR.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DPack%26aufirst%3DE.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DProkopowicz%26aufirst%3DA.%26aulast%3DWelzel%26aufirst%3DM.%26aulast%3DWolak%26aufirst%3DJ.%26aulast%3DMorwick%26aufirst%3DT.%26atitle%3DHit%2520to%2520lead%2520account%2520of%2520the%2520discovery%2520of%2520a%2520new%2520class%2520of%2520inhibitors%2520of%2520Pim%2520kinases%2520and%2520crystallographic%2520studies%2520revealing%2520an%2520unusual%2520kinase%2520binding%2520mode%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1814%26epage%3D1827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">A rule of three for fragment-based lead discovery?</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">876</span><span class="NLM_x">â</span> <span class="NLM_lpage">877</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=876-877&author=M.+Congreveauthor=R.+Carrauthor=C.+Murrayauthor=H.+Jhoti&title=A+rule+of+three+for+fragment-based+lead+discovery%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DA%2520rule%2520of%2520three%2520for%2520fragment-based%2520lead%2520discovery%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D876%26epage%3D877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span> <span class="citation_source-book">Glide (version 50207 unless otherwise mentioned)</span>; <span class="NLM_publisher-name">SchrÃ¶dinger, Inc.</span>, <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span><span class="NLM_x">; </span><a href="http://www.schrodinger.com" class="extLink">www.schrodinger.com</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Glide+%28version+50207+unless+otherwise+mentioned%29%3B+Schr%C3%B6dinger%2C+Inc.%2C+New+York%2C+2009%3B+www.schrodinger.com"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGlide%2520%2528version%252050207%2520unless%2520otherwise%2520mentioned%2529%26pub%3DSchr%25C3%25B6dinger%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">â</span> <span class="NLM_lpage">242</span><div class="note"><p class="first last"> (<a href="http://www.pdb.org" class="extLink">www.pdb.org</a>)</p></div></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0ljs2VaSPbdEUA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Cheney, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Applebya, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walkera, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yaoa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamatakea, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honga, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wua, J. Z.</span><span> </span><span class="NLM_article-title">Identification and structureâactivity relationships of substituted pyridones as inhibitors of Pim-1 kinase</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1679</span><span class="NLM_x">â</span> <span class="NLM_lpage">1683</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=1679-1683&author=I.+W.+Cheneyauthor=S.+Yanaauthor=T.+Applebyaauthor=H.+Walkeraauthor=T.+Voaauthor=N.+Yaoaauthor=R.+Hamatakeaauthor=Z.+Hongaauthor=J.+Z.+Wua&title=Identification+and+structure%E2%80%93activity+relationships+of+substituted+pyridones+as+inhibitors+of+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCheney%26aufirst%3DI.%2BW.%26aulast%3DYana%26aufirst%3DS.%26aulast%3DApplebya%26aufirst%3DT.%26aulast%3DWalkera%26aufirst%3DH.%26aulast%3DVoa%26aufirst%3DT.%26aulast%3DYaoa%26aufirst%3DN.%26aulast%3DHamatakea%26aufirst%3DR.%26aulast%3DHonga%26aufirst%3DZ.%26aulast%3DWua%26aufirst%3DJ.%2BZ.%26atitle%3DIdentification%2520and%2520structure%25E2%2580%2593activity%2520relationships%2520of%2520substituted%2520pyridones%2520as%2520inhibitors%2520of%2520Pim-1%2520kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D1679%26epage%3D1683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuver-Moody, C.</span><span> </span><span class="NLM_article-title">Docking study yields four novel inhibitors of the protooncogene Pim-1 kinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1972</span><span class="NLM_x">â</span> <span class="NLM_lpage">1975</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701248t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOmtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1972-1975&author=A.+C.+Pierceauthor=M.+Jacobsauthor=C.+Stuver-Moody&title=Docking+study+yields+four+novel+inhibitors+of+the+protooncogene+Pim-1+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Study Yields Four Novel Inhibitors of the Protooncogene Pim-1 Kinase</span></div><div class="casAuthors">Pierce, Albert C.; Jacobs, Marc; Stuver-Moody, Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1972-1975</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To supplement the hits from a high throughput screen, docking was performed against Pim-1 kinase.  Glide docking was augmented with a filter to require traditional or arom. CHÂ·Â·O hydrogen bonds to the kinase hinge.  Four diverse actives, of 96 mols. assayed, had Ki values between 0.091 and 4.5 Î¼M.  This gives a 14-fold enrichment over the earlier HTS run, and the two crystal structures solved confirmed the binding modes predicted by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH6GwwgMwC27Vg90H21EOLACvtfcHk0lhNTH7moUJU5Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOmtLY%253D&md5=fafba52528730d4e250d489ad24a6cd8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm701248t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701248t%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DJacobs%26aufirst%3DM.%26aulast%3DStuver-Moody%26aufirst%3DC.%26atitle%3DDocking%2520study%2520yields%2520four%2520novel%2520inhibitors%2520of%2520the%2520protooncogene%2520Pim-1%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1972%26epage%3D1975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Gampe, R. T.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montana, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bledsoe, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milburn, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kliewer, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H. E.</span><span> </span><span class="NLM_article-title">Asymmetry in the PPARgamma/RXRalpha crystal structure reveals the molecular basis of heterodimerization among nuclear receptors</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">545</span><span class="NLM_x">â</span> <span class="NLM_lpage">555</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=545-555&author=R.+T.+Gampeauthor=V.+G.+Montanaauthor=M.+H.+Lambertauthor=A.+B.+Millerauthor=R.+K.+Bledsoeauthor=M.+V.+Milburnauthor=S.+A.+Kliewerauthor=T.+M.+Willsonauthor=H.+E.+Xu&title=Asymmetry+in+the+PPARgamma%2FRXRalpha+crystal+structure+reveals+the+molecular+basis+of+heterodimerization+among+nuclear+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGampe%26aufirst%3DR.%2BT.%26aulast%3DMontana%26aufirst%3DV.%2BG.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DMiller%26aufirst%3DA.%2BB.%26aulast%3DBledsoe%26aufirst%3DR.%2BK.%26aulast%3DMilburn%26aufirst%3DM.%2BV.%26aulast%3DKliewer%26aufirst%3DS.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DAsymmetry%2520in%2520the%2520PPARgamma%252FRXRalpha%2520crystal%2520structure%2520reveals%2520the%2520molecular%2520basis%2520of%2520heterodimerization%2520among%2520nuclear%2520receptors%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D5%26spage%3D545%26epage%3D555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandretto, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span> </span><span class="NLM_article-title">Kinase inhibitors and the case for CHÂ·Â·Â·O hydrogen bonds in proteinâligand binding</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">567</span><span class="NLM_x">â</span> <span class="NLM_lpage">576</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2002&pages=567-576&author=A.+C.+Pierceauthor=K.+L.+Sandrettoauthor=G.+W.+Bemis&title=Kinase+inhibitors+and+the+case+for+CH%C2%B7%C2%B7%C2%B7O+hydrogen+bonds+in+protein%E2%80%93ligand+binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DSandretto%26aufirst%3DK.%2BL.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26atitle%3DKinase%2520inhibitors%2520and%2520the%2520case%2520for%2520CH%25C2%25B7%25C2%25B7%25C2%25B7O%2520hydrogen%2520bonds%2520in%2520protein%25E2%2580%2593ligand%2520binding%26jtitle%3DProteins%26date%3D2002%26volume%3D49%26spage%3D567%26epage%3D576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span> <span class="citation_source-book">Maestro (version 9.0 unless otherwise mentioned)</span>; <span class="NLM_publisher-name">SchrÃ¶dinger, Inc.</span>, <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>. <a href="http://www.schrodinger.com" class="extLink">www.schrodinger.com</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Maestro+%28version+9.0+unless+otherwise+mentioned%29%3B+Schr%C3%B6dinger%2C+Inc.%2C+New+York%2C+2009.+www.schrodinger.com"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMaestro%2520%2528version%25209.0%2520unless%2520otherwise%2520mentioned%2529%26pub%3DSchr%25C3%25B6dinger%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span>Vida version 3.1. Open Eye Inc.; <a href="http://www.eyesopen.com" class="extLink">www.eyesopen.com</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Vida+version+3.1.+Open+Eye+Inc.%3B+www.eyesopen.com"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span> <span class="citation_source-book">Ligprep, version 22207</span>; <span class="NLM_publisher-name">SchrÃ¶dinger, Inc.</span>, <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span><span class="NLM_x">; </span><a href="http://www.schrodinger.com" class="extLink">www.schrodinger.com</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Ligprep%2C+version+22207%3B+Schr%C3%B6dinger%2C+Inc.%2C+New+York%2C+2009%3B+www.schrodinger.com"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DLigprep%252C%2520version%252022207%26pub%3DSchr%25C3%25B6dinger%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Zhang, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oldenburg, K. R.</span><span> </span><span class="NLM_article-title">A simple statistical parameter for use in evaluation and validation of high throughput screening assays</span> <span class="citation_source-journal">J. Biomol. Screen</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">â</span> <span class="NLM_lpage">73</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.1177%2F108705719900400206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=10838414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&pages=67-73&author=J.+H.+Zhangauthor=T.+D.+Chungauthor=K.+R.+Oldenburg&title=A+simple+statistical+parameter+for+use+in+evaluation+and+validation+of+high+throughput+screening+assays"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span></div><div class="casAuthors">Zhang; Chung; Oldenburg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ability to identify active compounds (3hits2) from large chemical libraries accurately and rapidly has been the ultimate goal in developing high-throughput screening (HTS) assays.  The ability to identify hits from a particular HTS assay depends largely on the suitability or quality of the assay used in the screening.  The criteria or parameters for evaluating the 3suitability2 of an HTS assay for hit identification are not well defined and hence it still remains difficult to compare the quality of assays directly.  In this report, a screening window coefficient, called 3Z-factor,2 is defined.  This coefficient is reflective of both the assay signal dynamic range and the data variation associated with the signal measurements, and therefore is suitable for assay quality assessment.  The Z-factor is a dimensionless, simple statistical characteristic for each HTS assay.  The Z-factor provides a useful tool for comparison and evaluation of the quality of assays, and can be utilized in assay optimization and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkwW5JDdXnbsNIRw0GKUoEfW6udTcc2eYXAE0nVsEY7bntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D&md5=0ec4ed2ccdd3b4a16521567a0975d31f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1177%2F108705719900400206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705719900400206%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DChung%26aufirst%3DT.%2BD.%26aulast%3DOldenburg%26aufirst%3DK.%2BR.%26atitle%3DA%2520simple%2520statistical%2520parameter%2520for%2520use%2520in%2520evaluation%2520and%2520validation%2520of%2520high%2520throughput%2520screening%2520assays%26jtitle%3DJ.%2520Biomol.%2520Screen%26date%3D1999%26spage%3D67%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandiyan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artis, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bremer, R.</span><span> </span><span class="NLM_article-title">Crystal structures of proto-oncogene kinase Pim-1: A target of aberrant somatic hypermutations in diffuse large cell lymphoma</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">348</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">â</span> <span class="NLM_lpage">193</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2005&pages=183-193&author=A.+Kumarauthor=V.+Mandiyanauthor=Y.+Suzukiauthor=C.+Zhangauthor=J.+Riceauthor=J.+Tsaiauthor=D.+R.+Artisauthor=P.+Ibrahimauthor=R.+Bremer&title=Crystal+structures+of+proto-oncogene+kinase+Pim-1%3A+A+target+of+aberrant+somatic+hypermutations+in+diffuse+large+cell+lymphoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DMandiyan%26aufirst%3DV.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DBremer%26aufirst%3DR.%26atitle%3DCrystal%2520structures%2520of%2520proto-oncogene%2520kinase%2520Pim-1%253A%2520A%2520target%2520of%2520aberrant%2520somatic%2520hypermutations%2520in%2520diffuse%2520large%2520cell%2520lymphoma%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D348%26spage%3D183%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">â</span> <span class="NLM_lpage">501</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm2014698&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm2014698&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm2014698&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhN9eCL3vtddQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBQ','PDB','3VBQ'); return false;">PDB: 3VBQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBT','PDB','3VBT'); return false;">PDB: 3VBT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBV','PDB','3VBV'); return false;">PDB: 3VBV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBW','PDB','3VBW'); return false;">PDB: 3VBW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBX','PDB','3VBX'); return false;">PDB: 3VBX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VBY','PDB','3VBY'); return false;">PDB: 3VBY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VC4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VC4','PDB','3VC4'); return false;">PDB: 3VC4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OBJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OBJ','PDB','2OBJ'); return false;">PDB: 2OBJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGQ','PDB','3BGQ'); return false;">PDB: 3BGQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGP','PDB','3BGP'); return false;">PDB: 3BGP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BGZ','PDB','3BGZ'); return false;">PDB: 3BGZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VBV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VBV','PDB','2VBV'); return false;">PDB: 2VBV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=D186" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=D186','PDB','D186'); return false;">PDB: D186</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=E121" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=E121','PDB','E121'); return false;">PDB: E121</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R00" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R00','PDB','3R00'); return false;">PDB: 3R00</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F2A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F2A','PDB','3F2A'); return false;">PDB: 3F2A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R01" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R01','PDB','3R01'); return false;">PDB: 3R01</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i20"><a href="/doi/suppl/10.1021/jm2014698">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_17208"></div></div></div></div></div><hr /></hr><p class="last">IC<sub>50</sub>s and structures of all needle screening hits with a Pim-1 IC<sub>50</sub> < 100 Î¼M. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm2014698/suppl_file/jm2014698_si_001.pdf">jm2014698_si_001.pdf (195.19 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm2014698&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Fjm2014698%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2012.55.issue-6" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm2014698" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a3d958993192e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
