
<html lang="en"     class="pb-page"  data-request-id="be1cac11-5ec6-4639-832f-9ba1aea8922a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b00930;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2018.61.issue-19;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Optimization of a Novel Class of Aldehyde Dehydrogenase 1A (ALDH1A) Subfamily-Selective Inhibitors as Potential Adjuncts to Ovarian Cancer Chemotherapy" /></meta><meta name="dc.Creator" content="Brandt&#xA;C.  Huddle" /></meta><meta name="dc.Creator" content="Edward  Grimley" /></meta><meta name="dc.Creator" content="Cameron D.  Buchman" /></meta><meta name="dc.Creator" content="Mikhail  Chtcherbinine" /></meta><meta name="dc.Creator" content="Bikash  Debnath" /></meta><meta name="dc.Creator" content="Pooja  Mehta" /></meta><meta name="dc.Creator" content="Kun  Yang" /></meta><meta name="dc.Creator" content="Cynthia A.  Morgan" /></meta><meta name="dc.Creator" content="Siwei  Li" /></meta><meta name="dc.Creator" content="Jeremy  Felton" /></meta><meta name="dc.Creator" content="Duxin  Sun" /></meta><meta name="dc.Creator" content="Geeta  Mehta" /></meta><meta name="dc.Creator" content="Nouri  Neamati" /></meta><meta name="dc.Creator" content="Ronald J.  Buckanovich" /></meta><meta name="dc.Creator" content="Thomas D.  Hurley" /></meta><meta name="dc.Creator" content="Scott D.  Larsen" /></meta><meta name="dc.Description" content="Aldehyde dehydrogenase (ALDH) activity is commonly used as a marker to identify cancer stem-like cells. The three ALDH1A isoforms have all been individually implicated in cancer stem-like cells and..." /></meta><meta name="Description" content="Aldehyde dehydrogenase (ALDH) activity is commonly used as a marker to identify cancer stem-like cells. The three ALDH1A isoforms have all been individually implicated in cancer stem-like cells and..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 17, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00930" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00930" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00930" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00930" /></link>
        
    
    

<title>Structure-Based Optimization of a Novel Class of Aldehyde Dehydrogenase 1A (ALDH1A) Subfamily-Selective Inhibitors as Potential Adjuncts to Ovarian Cancer Chemotherapy | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00930" /></meta><meta property="og:title" content="Structure-Based Optimization of a Novel Class of Aldehyde Dehydrogenase 1A (ALDH1A) Subfamily-Selective Inhibitors as Potential Adjuncts to Ovarian Cancer Chemotherapy" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0017.jpeg" /></meta><meta property="og:description" content="Aldehyde dehydrogenase (ALDH) activity is commonly used as a marker to identify cancer stem-like cells. The three ALDH1A isoforms have all been individually implicated in cancer stem-like cells and in chemoresistance; however, which isoform is preferentially expressed varies between cell lines. We sought to explore the structural determinants of ALDH1A isoform selectivity in a series of small-molecule inhibitors in support of research into the role of ALDH1A in cancer stem cells. An SAR campaign guided by a cocrystal structure of the HTS hit CM39 (7) with ALDH1A1 afforded first-in-class inhibitors of the ALDH1A subfamily with excellent selectivity over the homologous ALDH2 isoform. We also discovered the first reported modestly selective single isoform 1A2 and 1A3 inhibitors. Two compounds, 13g and 13h, depleted the CD133+ putative cancer stem cell pool, synergized with cisplatin, and achieved efficacious concentrations in vivo following IP administration. Compound 13h additionally synergized with cisplatin in a patient-derived ovarian cancer spheroid model." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00930"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00930">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00930&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00930&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00930&amp;href=/doi/10.1021/acs.jmedchem.8b00930" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 19</span><span class="cit-fg-pageRange">, 8754-8773</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/19" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00898" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00932" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Optimization of a Novel Class of Aldehyde Dehydrogenase 1A (ALDH1A) Subfamily-Selective Inhibitors as Potential Adjuncts to Ovarian Cancer Chemotherapy</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Brandt C. Huddle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brandt C. Huddle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brandt%0AC.++Huddle">Brandt C. Huddle</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Edward Grimley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Edward Grimley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Obstetrics, Gynecology, and Reproductive Sciences, University of Pittsburgh Medical Center and the Magee Womens Research Institute, Pittsburgh Pennsylvania 15213, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Edward++Grimley">Edward Grimley</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Cameron D. Buchman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cameron D. Buchman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cameron+D.++Buchman">Cameron D. Buchman</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Mikhail Chtcherbinine</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mikhail Chtcherbinine</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mikhail++Chtcherbinine">Mikhail Chtcherbinine</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Bikash Debnath</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bikash Debnath</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bikash++Debnath">Bikash Debnath</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Pooja Mehta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pooja Mehta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Materials Science Engineering, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pooja++Mehta">Pooja Mehta</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Kun Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kun Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Hematology Oncology, Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kun++Yang">Kun Yang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Cynthia A. Morgan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cynthia A. Morgan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cynthia+A.++Morgan">Cynthia A. Morgan</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Siwei Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Siwei Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy; University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Siwei++Li">Siwei Li</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jeremy Felton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeremy Felton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy; University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeremy++Felton">Jeremy Felton</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Duxin Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Duxin Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy; University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Duxin++Sun">Duxin Sun</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Geeta Mehta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Geeta Mehta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Materials Science Engineering, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Biomedical Engineering, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Macromolecular Science and Engineering, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Geeta++Mehta">Geeta Mehta</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Nouri Neamati</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nouri Neamati</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nouri++Neamati">Nouri Neamati</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ronald J. Buckanovich</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ronald J. Buckanovich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Hematology-Oncology, Department of Internal Medicine, University of Pittsburgh Medical Center and the Magee Womens Research Institute, Pittsburgh Pennsylvania 15213, United States</div><div class="loa-info-affiliations-info">Division of Hematology Oncology, Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail for R.J.B.: <a href="/cdn-cgi/l/email-protection#345641575f555a5b425d575c465e745943465d1a59555351511a515041"><span class="__cf_email__" data-cfemail="3c5e495f575d52534a555f544e567c514b4e5512515d5b595912595849">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ronald+J.++Buckanovich">Ronald J. Buckanovich</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Thomas D. Hurley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas D. Hurley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+D.++Hurley">Thomas D. Hurley</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Scott D. Larsen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott D. Larsen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vahlteich Medicinal Chemistry Core,Â  Department of Medicinal Chemistry, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail for S.D.L.: <a href="/cdn-cgi/l/email-protection#e89b8c84899a9b8d86a8858d8cc69d85818b80c68d8c9d"><span class="__cf_email__" data-cfemail="6310070f021110060d230e06074d160e0a000b4d060716">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott+D.++Larsen">Scott D. Larsen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8440-4990" title="Orcid link">http://orcid.org/0000-0001-8440-4990</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00930&amp;href=/doi/10.1021%2Facs.jmedchem.8b00930" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 19</span><span class="cit-pageRange">, 8754â8773</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 17, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 June 2018</li><li><span class="item_label"><b>Published</b> online</span>17 September 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 October 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00930" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00930</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8754%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DBrandt%250AC.%2BHuddle%252C%2BEdward%2BGrimley%252C%2BCameron%2BD.%2BBuchman%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D19%26contentID%3Dacs.jmedchem.8b00930%26title%3DStructure-Based%2BOptimization%2Bof%2Ba%2BNovel%2BClass%2Bof%2BAldehyde%2BDehydrogenase%2B1A%2B%2528ALDH1A%2529%2BSubfamily-Selective%2BInhibitors%2Bas%2BPotential%2BAdjuncts%2Bto%2BOvarian%2BCancer%2BChemotherapy%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8773%26publicationDate%3DOctober%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00930"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2450</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">21</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00930" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Optimization of a Novel Class of Aldehyde Dehydrogenase 1A (ALDH1A) Subfamily-Selective Inhibitors as Potential Adjuncts to Ovarian Cancer Chemotherapy&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Brandt\nC.&quot;,&quot;last_name&quot;:&quot;Huddle&quot;},{&quot;first_name&quot;:&quot;Edward&quot;,&quot;last_name&quot;:&quot;Grimley&quot;},{&quot;first_name&quot;:&quot;Cameron&quot;,&quot;last_name&quot;:&quot;D. Buchman&quot;},{&quot;first_name&quot;:&quot;Mikhail&quot;,&quot;last_name&quot;:&quot;Chtcherbinine&quot;},{&quot;first_name&quot;:&quot;Bikash&quot;,&quot;last_name&quot;:&quot;Debnath&quot;},{&quot;first_name&quot;:&quot;Pooja&quot;,&quot;last_name&quot;:&quot;Mehta&quot;},{&quot;first_name&quot;:&quot;Kun&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Cynthia&quot;,&quot;last_name&quot;:&quot;A. Morgan&quot;},{&quot;first_name&quot;:&quot;Siwei&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jeremy&quot;,&quot;last_name&quot;:&quot;Felton&quot;},{&quot;first_name&quot;:&quot;Duxin&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Geeta&quot;,&quot;last_name&quot;:&quot;Mehta&quot;},{&quot;first_name&quot;:&quot;Nouri&quot;,&quot;last_name&quot;:&quot;Neamati&quot;},{&quot;first_name&quot;:&quot;Ronald&quot;,&quot;last_name&quot;:&quot;J. Buckanovich&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;D. Hurley&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;D. Larsen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;19&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;8754-8773&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00930&quot;},&quot;abstract&quot;:&quot;Aldehyde dehydrogenase (ALDH) activity is commonly used as a marker to identify cancer stem-like cells. The three ALDH1A isoforms have all been individually implicated in cancer stem-like cells and in chemoresistance; however, which isoform is preferentially expressed varies between cell lines. We sought to explore the structural determinants of ALDH1A isoform selectivity in a series of small-molecule inhibitors in support of research into the role of ALDH1A in cancer stem cells. An SAR campaign guided by a cocrystal structure of the HTS hit CM39 (7) with ALDH1A1 afforded first-in-class inhibitors of the ALDH1A subfamily with excellent selectivity over the homologous ALDH2 isoform. We also discovered the first reported modestly selective single isoform 1A2 and 1A3 inhibitors. Two compounds, 13g and 13h, depleted the CD133+ putative cancer stem cell pool, synergized with cisplatin, and achieved efficacious concentrations in vivo following IP administration. Compound 13h additionally synergized with cisplatin i&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00930&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00930" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00930&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00930" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00930&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00930" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00930&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00930&amp;href=/doi/10.1021/acs.jmedchem.8b00930" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00930" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00930" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (15 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00930%26sid%3Dliteratum%253Aachs%26pmid%3D30221940%26genre%3Darticle%26aulast%3DHuddle%26date%3D2018%26atitle%3DStructure-Based%2BOptimization%2Bof%2Ba%2BNovel%2BClass%2Bof%2BAldehyde%2BDehydrogenase%2B1A%2B%2528ALDH1A%2529%2BSubfamily-Selective%2BInhibitors%2Bas%2BPotential%2BAdjuncts%2Bto%2BOvarian%2BCancer%2BChemotherapy%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D19%26spage%3D8754%26epage%3D8773%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/19" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/jmcmar.2018.61.issue-19/20181011/jmcmar.2018.61.issue-19.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Aldehyde dehydrogenase (ALDH) activity is commonly used as a marker to identify cancer stem-like cells. The three ALDH1A isoforms have all been individually implicated in cancer stem-like cells and in chemoresistance; however, which isoform is preferentially expressed varies between cell lines. We sought to explore the structural determinants of ALDH1A isoform selectivity in a series of small-molecule inhibitors in support of research into the role of ALDH1A in cancer stem cells. An SAR campaign guided by a cocrystal structure of the HTS hit <b>CM39</b> (<b>7</b>) with ALDH1A1 afforded first-in-class inhibitors of the ALDH1A subfamily with excellent selectivity over the homologous ALDH2 isoform. We also discovered the first reported modestly selective single isoform 1A2 and 1A3 inhibitors. Two compounds, <b>13g</b> and <b>13h</b>, depleted the CD133<sup>+</sup> putative cancer stem cell pool, synergized with cisplatin, and achieved efficacious concentrations in vivo following IP administration. Compound <b>13h</b> additionally synergized with cisplatin in a patient-derived ovarian cancer spheroid model.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47618" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47618" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Currently, more than half of the 24,000 U.S. women diagnosed with epithelial ovarian cancer (EOC) annually are expected to succumb to the disease within 5 years.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The first line therapy for the majority of EOC cases is surgical debulking of the primary tumor with adjuvant platinum- and taxane-based chemotherapeutics to treat the residual disease.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Approximately 70% of EOC patients are initially responsive to chemotherapeutics; however, most relapse and ultimately become unresponsive to further chemotherapy.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">EOC tumors contain a hierarchy of heterogeneous cells consistent with the Cancer Stem Cell (CSC) hypothesis.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8">(5â8)</a> New therapies which target these CSCs may improve patient outcomes alone or when combined with chemotherapy.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> In EOC, elevated aldehyde dehydrogenase (ALDH) activity is a marker of CSCs.<a onclick="showRef(event, 'ref5 ref6 ref7 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref10 ref11 ref12">(5â7,10â12)</a> ALDH<sup>bright</sup> cells (the subpopulation of cells with the greatest activity in the ALDEFLUOR assay) are more tumorigenic and chemoresistant than ALDH<sup>dim</sup> cells, are more prevalent in chemoresistant tumors following chemotherapy, and their presence within a tumor is predictive of poorer patient outcomes.<a onclick="showRef(event, 'ref6 ref7 ref8 ref12'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref12">(6â8,12)</a></div><div class="NLM_p">There are 19 distinct genes for ALDH superfamily members in humans. The primary function of ALDH is to oxidize endogenous aldehydes generated through various cellular processes to the corresponding carboxylic acids. In addition to neutralization of these reactive species, the 3 members of the ALDH1A subfamily also function in cellular signaling by generating the nuclear hormone all-trans retinoic acid (ATRA) from retinal.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In some solid tumors, ATRA has been shown to activate transcription of oncogenes such as c-MYC, PDK-1, and cyclin D1.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Although the strongest body of evidence supports the role of ALDH1A1 (1A1) in CSCs, other isoforms of the ALDH1A family are often simultaneously expressed.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><div class="NLM_p">Given that ALDH plays a potentially critical role in CSCs, inhibition of ALDH is a potential strategy to target CSC and reverse resistance to chemotherapy. Indeed, knockdown or inhibition of 1A1 increases chemosensitivity in ovarian and other cancers.<a onclick="showRef(event, 'ref5 ref16 ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref5 ref16 ref17 ref18 ref19 ref20 ref21">(5,16â21)</a> ALDH1A2 (1A2) and ALDH1A3 (1A3) have similarly been implicated in chemoresistance in other tumor types.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> Some ALDH isoforms, including 1A1, are able to divert cyclophosphamide metabolism, preventing generation of the active phosphoramide mustard; however, their ability to attenuate the effects of other chemotherapeutics such as cisplatin and paclitaxel is poorly understood.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><div class="NLM_p">A panel of cell-permeable, single or dual isoform-selective inhibitors for 1A1, 1A2, and 1A3 could have utility as probes for dissecting the role of the various isoforms in any number of applications which currently rely on siRNA knockdown. Various tumors and cancer cell lines differ in which ALDH1A isoforms are highly expressed. <i>Importantly, high grade serous ovarian tumors, the histologic subtype responsible for 70â80% of EOC deaths, show strong elevation of 1A3 expression but only modest elevation in 1A1 expression.</i> Other histologic subtypes show significant elevation of both isoforms.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> The inability of existing 1A1 selective inhibitors to inhibit ALDEFLUOR in 1A3 high cell lines has previously been demonstrated.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p">There are a number of small molecule ALDH inhibitors reported in the literature (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref13 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref15">(13,15)</a> Many bear an electrophilic warhead and rely on reversible or irreversible covalent interaction with the ALDH catalytic cysteine to achieve potency. Design of highly isoform selective compounds employing these warheads is complicated by the presence of this critical cysteine throughout the ALDH family. One of most widely studied ALDH inhibitors, DEAB (<b>1</b>), inhibits at least 6 isoforms of ALDH with an IC<sub>50</sub> < 15 Î¼M and is a substrate of at least 5 isoforms, including 1A1.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Compound <b>1</b> requires concentrations of â¼100 Î¼M to induce chemosensitization of CSCs.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Disulfiram (<b>2</b>), a nonselective covalent ALDH inhibitor approved for treatment of alcoholism, is rapidly metabolized into several covalent ALDH inhibitors with variable isoform selectivity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> While <b>2</b> is also reported to deplete CSCs in combination with chemotherapy, this effect likely does not result from its action on ALDH.<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29â31)</a> Win 18,446 (<b>3</b>) is a potent inhibitor of the ALDH1A subfamily (1A1, 1A2, 1A3 <i>K</i><sub><i>i</i></sub> = 285, 56, and 261 nM, respectively).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Unfortunately, <b>3</b> also inhibits ALDH2 and is a known teratogen due to its ability to chelate zinc, making it an imperfect tool for the study of ALDH1A biology.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Hurley and colleagues disclosed the noncovalent 1A1 inhibitor high throughput screening (HTS) hits <b>4</b> and <b>5a</b> along with cocrystal structures.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> Soon thereafter, compound <b>5b</b> was disclosed elsewhere.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Recently, a scaffold-hopping approach to improve the cell permeability of <b>5b</b> yielded compound <b>6</b> and other similar analogues. Compound <b>6</b> exhibits potent and highly selective 1A1 inhibition (1A1 IC<sub>50</sub> = 7 nM; 1A2, 1A3, ALDH2, ALDH3A1 IC<sub>50</sub> values > 1 Î¼M) and potent inhibition of ALDEFLUOR in MIA PaCa-2 cells, which preferentially express 1A1. Furthermore, this series was able to sensitize a paclitaxel resistant ovarian cancer cell line to treatment with paclitaxel.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> To our knowledge, there are no selective inhibitors of 1A2 or 1A3.<a onclick="showRef(event, 'ref13 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref15">(13,15)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative ALDH inhibitors <b>1â6</b> and lead compound <b>7</b> (CM39).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The high-throughput screen which identified <b>4</b> and <b>5a</b> also yielded the pyrazolopyrimidinone <b>CM39</b> (<b>7</b>). This compound was an attractive lead for the development of ALDH1A subfamily inhibitors because it exhibited reasonably ligand-efficient inhibition of 1A1 (IC<sub>50</sub> = 0.9 Î¼M, LE = 0.32) and good selectivity (<20% inhibition of ALDH2 and 3A1 at 20uM).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> A further screen of selectivity showed â¼50% inhibition of 1A2 and 1A3 at 20 Î¼M and little activity against other ALDH isoforms (vide infra). Following our successful acquisition of an X-ray crystal structure of <b>7</b> bound in the 1A1 active site (vide infra), we undertook the rational design and synthesis of analogues. Herein we report the ability of these analogues to inhibit ALDH1A with widely varying selectivity profiles. We also examine the ability of selected compounds to inhibit cellular ALDH function using ALDEFLUOR, to deplete the CD133<sup>+</sup> putative cancer stem cell pool, and to synergize with cisplatin in two ovarian cancer cell lines and in patient derived ovarian cancer spheroids. Aqueous solubility and metabolic stability were assessed for select compounds to guide future optimization of in vivo probes.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, synthesis of compounds <b>7</b>, <b>11</b>, <b>12a</b>â<b>d</b>,<b>f</b>â<b>h</b>, and <b>13a</b>â<b>h</b> began with the commercially available pyrazole <b>8a</b>.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Treatment of <b>8a</b> with the appropriate aryl isothiocyanate in refluxing toluene afforded the thioureas <b>9a</b> and <b>9b</b> which precipitated from the reaction mixture upon cooling. Ring closure to generate thiopyrimidinones <b>10a</b> and <b>10b</b> proceeded in aqueous NaOH at reflux. Selective S-alkylation with 3-fluorobenzyl bromide yielded compounds <b>7</b> and <b>11</b>. Subsequent N-alkylation resulted in a mixture of regioisomers <b>12a</b>-<b>d</b>,<b>f-h</b> and <b>13a</b>â<b>h</b> which were easily separated by flash chromatography on silica. Regioisomeric assignment was made by NOE; in all cases the 1-alkyl-pyrazoles <b>12</b> eluted before the corresponding 2-alkyl-pyrazoles <b>13</b>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>7</b>, <b>11</b>, <b>12a</b>â<b>d</b>,<b>f</b>â<b>h</b>, and <b>13a</b>â<b>h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ArNCS, toluene, reflux; (b) 1N NaOH, reflux; (c) 3-F-PhCH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, RT; (d) R<sup>1</sup>-X, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 Â°C X = Br, OMs</p></p></figure><div class="NLM_p">In a departure from our earlier synthetic strategy, we envisioned an alternative route to <b>12e</b> and <b>15a</b>â<b>m</b> that would use readily available amines rather than the corresponding isothiocyanates to introduce diverse R<sup>3</sup> substituents as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The synthesis of <b>14</b> has been previously accomplished by treating commercially available pyrazole <b>8b</b> with thiophosgene under SchottenâBaumann conditions; however, we found the low yield, difficult purification, and use of noxious reagents to be unsatisfactory.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> After screening many reported isothiocyanate-forming conditions employing carbon disulfide as the thiocarbonyl source, we found that treatment of <b>8b</b> with NaH and CS<sub>2</sub> followed by the slow addition of I<sub>2</sub> afforded <b>14</b> in good yield following a simple purification.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Isothiocyanate <b>14</b> reacted with amines rapidly under mild conditions to form thioureas. Subsequent addition of NaH in the same pot facilitated cyclization to the thiopyrimidinone. For the majority of arylamines, adding 3-fluorobenzyl bromide at this point completed the transformation of <b>14</b> to <b>12e</b>, <b>15a</b>â<b>i</b> in one pot. For alkylamines and 2-aminopyridine (<b>15j</b>â<b>l</b>), an aqueous workup prior to S-alkylation with 3-fluorobenzyl bromide reduced the formation of several unidentified side products. The thiourea resulting from treatment of <b>14</b> with 8N ethanolic methylamine spontaneously cyclized to form the thiopyrimidinone. Following removal of volatiles, S-alkylation of this intermediate with 3-fluorobenzyl bromide under basic conditions afforded <b>15m</b>.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Analogues <b>12e</b>, <b>15a</b>â<b>m</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, CS<sub>2</sub>, I<sub>2</sub>, THF, 0â40 Â°C; (b) R<sup>3</sup>-NH<sub>2</sub>, 3-F-PhCH<sub>2</sub>Br, NaH, DMF, 0â20 Â°C; (c) (i) R<sup>3</sup>-NH<sub>2</sub>, NaH, DMF, 0â20 Â°C; (ii) 3-F-PhCH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, RT; (d) (i) MeNH<sub>2</sub>, EtOH; (ii) 3-F-PhCH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, RT</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Characterization of Lead <b>7</b></h3><div class="NLM_p">To establish a benchmark and guide design of new analogues, we further characterized compound <b>7</b>. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, <b>7</b> has excellent selectivity for 1A1 vs 2, 1B1, 3A1, 4A1, 5A1, and 1L1 yet modestly inhibits 1A2, 1A3. Despite the presence of a potentially electrophilic isothiourea, <b>7</b>-mediated inhibition of 1A1 was fully reversed following gel filtration, indicating it is noncovalent (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplemental Figure S1</a>). Similar to other noncovalent inhibitors of 1A1, <b>7</b> was found to be noncompetitive with respect to the substrate acetaldehyde with a <i>K</i><sub><i>i</i></sub> = 0.38 Â± 0.05 Î¼M and uncompetitive with respect to NAD<sup>+</sup>.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Liabilities of the lead with regard to its potential as a therapeutic included a half-life of 9 min in the presence of MLMs and poor aqueous solubility (2 Î¼M), both of which could be attributed to high lipophilicity (cLogP of 5.2).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, compound <b>7</b> was inferior to the widely used ALDEFLUOR control compound DEAB (<b>1</b>) at reducing ALDH<sup>Bright</sup> PEO1 cells at concentrations up to 30 Î¼M. This result suggested that ALDH1A1 selective inhibition is not sufficient to inhibit ALDEFLUOR in this 1A3 high cell line. It is important to note that while DEAB is a 57 nM 1A1 selective inhibitor, it only inhibits ALDEFLUOR above its 1A3 IC<sub>50</sub> (3 Î¼M).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> More potent ALDEFLUOR inhibition for DEAB in two 1A1 high cell lines, MIA PaCa-2 (IC<sub>50</sub> 3.4 Î¼M) and HT-29 (IC<sub>50</sub> 1.7 Î¼M), has previously been reported.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Fraction of enzyme activity remaining for 9 ALDH isoforms in the presence of 20 Î¼M <b>7</b>. (B) Percent control ALDH<sup>Bright</sup> cells remaining following treatment of PEO1 cells at the specified concentration with DEAB (<b>1</b>) or compound <b>7</b> (<i>N</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Crystal Structure of ALDH1A1 Complexed with <b>7</b></h3><div class="NLM_p">To guide the design of new analogues, an X-ray crystal structure of <b>7</b> bound to N121S ALDH1A1 was obtained with a resolution of 2.1 Ã (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI">5TEI</a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Table S1</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Figure S2</a>). The known single nucleotide polymorphism (NCBI rs1049981) resulting in the N121S missense mutation has been found in a small percentage of the HapMap-CEU population. There is no known clinical significance to the mutation and its structure is highly similar to the wild-type.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> The N121S enzyme crystallizes under identical conditions to the WT enzyme, but complexes with inhibitors are frequently easier to obtain, presumably due to dynamic fluctuations within the active site that are not detectable by crystallography and have little impact on observed potency. Furthermore, the IC<sub>50</sub> of <b>7</b> was comparable for the WT and N121S (0.9 and 0.7 Î¼M respectively, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplementary Figure S1</a>) Similar to the complex with <b>5a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a>), the bicyclic ring binds across the exterior 1/3 of the substrate binding pocket. The 3-fluorobenzyl group extends in toward the catalytic nucleophile (C303) and makes aromatic contacts with F171, W178 and F466. This binding mode is orthogonal to that for <b>4</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L">4X4L</a>), in which its tricyclic ring system extends toward both C303 and W178 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The core heterocycle of <b>7</b> binds similarly to the substituted xanthine ring of <b>5a</b>, binding beneath Y297 and occupying the additional active site volume afforded by the 1A1-specific residue, G458. Other ALDH1/2 isoenzymes have either an Asn or Asp residue at position 458 which may drive selectivity by prohibiting a similar binding mode in these isoforms. There appears to be room for small substitutions on the pyrazole at N-1 but not N-2, suggesting that larger substituents at these positions will not be tolerated without alteration of the binding pose. The lipophilic 2-methylphenyl ring binds near a relatively polar region of the 1A1 active site, suggesting that appropriate polar substituents could improve or maintain binding affinity while lowering cLogP. The 3-fluorobenzyl ring attached to the sulfur projects into the deepest section of the active site and maximizes contacts with the aromatic residues near the catalytic nucleophile, C303. The apparent goodness of fit of the 3-fluorobenzyl substituent, as well as its expected metabolic stability, encouraged us to retain it for the initial set of analogues based on <b>7</b>.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of ALDH1A1 N121S with <b>7</b> (2.10 Ã, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI">5TEI</a>). (A) The fluorophenyl of compound <b>7</b> projects into the lipophilic pocket toward the catalytic cysteine (C303). Key residues are labeled. (B) Overlay of compounds <b>4</b> (cyan, PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a>), <b>5a</b> (gray, PDB Entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L">4X4L</a>), and <b>7</b> (orange) bound in ALDH1A1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Characterization of N-1 Alkylated Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0014.gif" alt="" id="GRAPHIC-d7e1145-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="4" align="center">ALDHÂ IC<sub>50</sub><a class="ref internalNav" href="#t1fn2" aria-label="a">a</a>Â (Î¼M)Â orÂ %Â controlÂ atÂ 20Â Î¼M<a class="ref internalNav" href="#t1fn3" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd no.</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">1A1</th><th class="colsep0 rowsep0" align="center">1A2</th><th class="colsep0 rowsep0" align="center">1A3</th><th class="colsep0 rowsep0" align="center">2</th><th class="colsep0 rowsep0" align="center">cLogP</th><th class="colsep0 rowsep0" align="center">aq.Â sol.<a class="ref internalNav" href="#t1fn4" aria-label="c">c</a> (Î¼M)</th><th class="colsep0 rowsep0" align="center">MLM <i>t</i><sub>1/2</sub> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>7</b></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">0.9Â Â±Â 0.2</td><td class="colsep0 rowsep0" align="center">53%</td><td class="colsep0 rowsep0" align="center">53%</td><td class="colsep0 rowsep0" align="center">85%</td><td class="colsep0 rowsep0" align="center">5.2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>11</b></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">0.45Â Â±Â 0.03</td><td class="colsep0 rowsep0" align="center">0.67Â Â±Â 0.08</td><td class="colsep0 rowsep0" align="center">0.39Â Â±Â 0.02</td><td class="colsep0 rowsep0" align="center">94%</td><td class="colsep0 rowsep0" align="center">4.7</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12a</b></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">0.66Â Â±Â 0.06</td><td class="colsep0 rowsep0" align="center">52%</td><td class="colsep0 rowsep0" align="center">4Â Â±Â 1</td><td class="colsep0 rowsep0" align="center">94%</td><td class="colsep0 rowsep0" align="center">4.8</td><td class="colsep0 rowsep0" align="center">38</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12b</b></td><td class="colsep0 rowsep0" align="center">c-PrCH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">0.18Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">>10</td><td class="colsep0 rowsep0" align="center">>10</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">5.6</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12c</b></td><td class="colsep0 rowsep0" align="center">3-oxetanyl</td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">0.26Â Â±Â 0.03</td><td class="colsep0 rowsep0" align="center">6Â Â±Â 1</td><td class="colsep0 rowsep0" align="center">60%</td><td class="colsep0 rowsep0" align="center">96%</td><td class="colsep0 rowsep0" align="center">4.7</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12d</b></td><td class="colsep0 rowsep0" align="center">3-oxetanyl-CH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">0.42Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">45%</td><td class="colsep0 rowsep0" align="center">58%</td><td class="colsep0 rowsep0" align="center">97%</td><td class="colsep0 rowsep0" align="center">4.7</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12e</b></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">0.08Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">0.15Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">0.09Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">4.3</td><td class="colsep0 rowsep0" align="center"><0.7Â (BLQ)<a class="ref internalNav" href="#t1fn5" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12f</b></td><td class="colsep0 rowsep0" align="center">c-PrCH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">0.56Â Â±Â 0.03</td><td class="colsep0 rowsep0" align="center">1.26Â Â±Â 0.08</td><td class="colsep0 rowsep0" align="center">0.55Â Â±Â 0.02</td><td class="colsep0 rowsep0" align="center">83%</td><td class="colsep0 rowsep0" align="center">5.1</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12g</b></td><td class="colsep0 rowsep0" align="center">3-oxetanyl</td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">0.13Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">0.23Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">0.17Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">94%</td><td class="colsep0 rowsep0" align="center">4.1</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12h</b></td><td class="colsep0 rowsep0" align="center">3-oxetanyl-CH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">0.34Â Â±Â 0.07</td><td class="colsep0 rowsep0" align="center">1.1Â Â±Â 0.1</td><td class="colsep0 rowsep0" align="center">0.61Â Â±Â 0.09</td><td class="colsep0 rowsep0" align="center">96%</td><td class="colsep0 rowsep0" align="center">4.1</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn2"><div class="footnote" id="t1fn2"><sup><sup>a</sup></sup><p class="last">Mean Â± SEM (<i>n</i> = 3).</p></div><div class="footnote" id="t1fn3"><sup><sup>b</sup></sup><p class="last">Mean (<i>n</i> = 3).</p></div><div class="footnote" id="t1fn4"><sup><sup>c</sup></sup><p class="last">Thermodynamic solubility analysis was performed by Analiza Inc. using quantitative nitrogen detection (<a href="http://www.analiza.com" class="extLink">www.analiza.com</a>).</p></div><div class="footnote" id="t1fn5"><sup><sup>d</sup></sup><p class="last">BLQ = below limit of quantitation.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> StructureâActivity Relationships (SAR)</h3><div class="NLM_p">All new compounds were tested for inhibition of ALDH1A1, 1A2, 1A3, and 2. Selected compounds were also tested for thermodynamic aqueous solubility and stability in the presence of mouse liver microsomes (MLM).</div><div class="NLM_p">We began our SAR campaign by generating N-alkylated analogues to fill the small lipophilic pocket adjacent to Ile304. We also hypothesized that eliminating the moderately acidic (p<i>K</i><sub>a</sub> = 8.1<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a>) pyrazole NâH might improve cell permeability. Encouragingly, methylation at N-1 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>12a</b>) resulted in a modest improvement in 1A1 inhibition compared to <b>7</b>. Consistent with a reduction in cLogP,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> aqueous solubility was also improved. Installing the larger cyclopropyl-methyl substituent (<b>12b</b>) improved the 1A1 IC<sub>50</sub> but did not improve potency against 1A2 or 1A3. The unfavorable cLogP (5.6) of <b>12b</b> encouraged us to explore less lipophilic oxetanyl analogues <b>12c</b>,<b>d</b>. Compound <b>12c</b> maintained comparable potency against 1A1 with substantially lower cLogP (4.7). The homologated oxetane <b>12d</b> was 2-fold less potent against 1A1, suggesting that larger substituents would not be tolerated.</div><div class="NLM_p">In an effort to pare down the <b>CM39</b> scaffold to the key pharmacophore, we discovered that removing the R<sup>2</sup> methyl markedly improved inhibition of 1A2 and 1A3 affording compound <b>11</b>, which potently inhibited all three ALDH1A isoforms but retained excellent selectivity over ALDH2. We revisited the panel of N-1 substituents (<b>12e</b>â<b>h</b>) and observed different SAR trends. The small <i>N</i>-methyl substituent of <b>12e</b> afforded the most potent of the four analogues tested but exhibited very poor solubility. Given the lower cLogP of <b>12e</b> compared to <b>7</b> or <b>12a</b>, we believe the poor solubility is best explained by the increased planarity of <b>12e</b>. The MLM stability of <b>12e</b> was unchanged relative to <b>7</b>. The bulkier cyclopropyl-methyl and oxetanyl-methyl substituents of <b>12f</b> and <b>12h</b> both resulted in 4-fold or greater reduction in potency against the ALDH1A family relative to <b>12e</b>. The smaller oxetanyl substituent of <b>12g</b> was only slightly less potent than <b>12e</b> and afforded a modest improvement in aqueous solubility. <i>The divergent SAR trends for</i><b><i>12aâd</i></b><i>compared to</i><b><i>12e</i></b>â<b><i>h</i></b><i>suggested that, upon removal of the o-methyl, a new binding mode was accessible.</i></div><div class="NLM_p">N-2 alkylated analogues were also tested to further probe the flexibility of the binding mode depicted in the <b>CM39</b> crystal structure (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). <b>13a</b> exhibited a similar ALDH1A inhibition profile to <b>12a</b>. Extending to bulkier substituents (<b>13b</b>â<b>d</b>) resulted in improved 1A1 potency with the cyclopropyl-methyl being optimal, providing strong evidence for an alternative binding mode. <b>13a</b>â<b>d</b> were more potent against 1A3 than the corresponding N-1 regioisomers (<b>12a</b>â<b>d</b>), as exemplified by the modestly 1A1/1A3 selective cyclopropyl-methyl analogue <b>13b</b>. As we observed previously, removal of the <i>o</i>-methyl significantly improved the inhibition of 1A2 and 1A3 in analogues <b>13e</b>â<b>h</b>. Oxetane was the optimal N-2 substituent to achieve inhibition of all three ALDH1A isoforms (<b>13g</b>), while cyclopropyl-methyl (<b>13f</b>) afforded a potent 1A1/1A3 inhibitor with >5-fold selectivity over 1A2. Despite having higher values for cLogP than the corresponding N-1 regioisomers, N-2 analogues were observed to be more polar by normal and reverse phase chromatography. Consistent with this observation, the solubility of N-2 methyl analogue <b>13e</b> was substantially improved over regioisomer <b>12e</b>. Compound <b>13e</b> was also moderately more stable in the presence of MLMs, leading us to prioritize N-2 substituted analogues for further pharmacokinetic studies. Curiously, appending the oxetanyl substituent at N-2 (<b>13g</b>) was not favorable for solubility as was observed at N-1 (<b>12g</b>). Reasonable aqueous solubility was restored for the homologated analogue <b>13h</b>. Perhaps resulting from steric blocking of N-dealkylation, <b>13g</b> and <b>13h</b> were about 2-fold more stable in the MLM assay than <b>13e</b>. Exquisite selectivity over the closely related ALDH2 isoform was maintained for all tested analogues, positioning the series as a useful set of tool compounds to probe the biological effect of ALDH1A inhibition.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Characterization of N-2 Alkylated Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0015.gif" alt="" id="GRAPHIC-d7e1699-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="4" align="center">ALDHÂ IC<sub>50</sub><a class="ref internalNav" href="#t2fn2" aria-label="a">a</a>Â (Î¼M)Â orÂ %Â controlÂ atÂ 20Â Î¼M<a class="ref internalNav" href="#t2fn3" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd no.</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">1A1</th><th class="colsep0 rowsep0" align="center">1A2</th><th class="colsep0 rowsep0" align="center">1A3</th><th class="colsep0 rowsep0" align="center">2</th><th class="colsep0 rowsep0" align="center">cLogP</th><th class="colsep0 rowsep0" align="center">aq.Â sol.<a class="ref internalNav" href="#t2fn4" aria-label="c">c</a> (Î¼M)</th><th class="colsep0 rowsep0" align="center">MLM <i>t</i><sub>1/2</sub> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13a</b></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">0.67Â Â±Â 0.07</td><td class="colsep0 rowsep0" align="center">6Â Â±Â 1</td><td class="colsep0 rowsep0" align="center">2.2Â Â±Â 0.4</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">5.3</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13b</b></td><td class="colsep0 rowsep0" align="center">c-Pr-CH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">0.10Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">>10</td><td class="colsep0 rowsep0" align="center">1.0Â Â±Â 0.1</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">6.1</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13c</b></td><td class="colsep0 rowsep0" align="center">3-oxetanyl</td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">0.44Â Â±Â 0.06</td><td class="colsep0 rowsep0" align="center">8Â Â±Â 1</td><td class="colsep0 rowsep0" align="center">3.9Â Â±Â 0.4</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">5.2</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13d</b></td><td class="colsep0 rowsep0" align="center">3-oxetanyl-CH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">0.20Â Â±Â 0.03</td><td class="colsep0 rowsep0" align="center">3.9Â Â±Â 0.3</td><td class="colsep0 rowsep0" align="center">2.6Â Â±Â 0.3</td><td class="colsep0 rowsep0" align="center">98%</td><td class="colsep0 rowsep0" align="center">5.2</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13e</b></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">0.25Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">0.34Â Â±Â 0.05</td><td class="colsep0 rowsep0" align="center">0.13Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">94%</td><td class="colsep0 rowsep0" align="center">4.8</td><td class="colsep0 rowsep0" align="center">69</td><td class="colsep0 rowsep0" align="center">13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13f</b></td><td class="colsep0 rowsep0" align="center">c-Pr-CH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">0.13Â Â±Â 0.03</td><td class="colsep0 rowsep0" align="center">1.1Â Â±Â 0.2</td><td class="colsep0 rowsep0" align="center">0.17Â Â±Â 0.03</td><td class="colsep0 rowsep0" align="center">87%</td><td class="colsep0 rowsep0" align="center">5.6</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13g</b></td><td class="colsep0 rowsep0" align="center">3-oxetanyl</td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">0.08Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">0.25Â Â±Â 0.04</td><td class="colsep0 rowsep0" align="center">0.12Â Â±Â 0.02</td><td class="colsep0 rowsep0" align="center">97%</td><td class="colsep0 rowsep0" align="center">4.7</td><td class="colsep0 rowsep0" align="center">5</td><td class="colsep0 rowsep0" align="center">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13h</b></td><td class="colsep0 rowsep0" align="center">3-oxetanyl-CH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">0.27Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">0.48Â Â±Â 0.04</td><td class="colsep0 rowsep0" align="center">0.13Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">99%</td><td class="colsep0 rowsep0" align="center">4.7</td><td class="colsep0 rowsep0" align="center">53</td><td class="colsep0 rowsep0" align="center">27</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn2"><div class="footnote" id="t2fn2"><sup><sup>a</sup></sup><p class="last">Mean Â± SEM (<i>n</i> = 3).</p></div><div class="footnote" id="t2fn3"><sup><sup>b</sup></sup><p class="last">Mean (<i>n</i> = 3).</p></div><div class="footnote" id="t2fn4"><sup><sup>c</sup></sup><p class="last">Thermodynamic solubility analysis was performed by Analiza Inc. using quantitative nitrogen detection (<a href="http://www.analiza.com" class="extLink">www.analiza.com</a>).</p></div></div></div><div class="NLM_p">The substantial increase in inhibition across the ALDH1A subfamily upon removal of the <i>o</i>-methyl in R<sup>3</sup> (<b>12e</b>) prompted us to further probe the effect of varied substitutions at this position as shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. We retained the N-1 methyl substituent of <b>12e</b> for synthetic ease. Although we had evidence to suggest the binding mode of <b>7</b> depicted in the crystal structure was flexible, we began by designing R<sup>3</sup> analogues according to the crystal structure. The R<sup>3</sup> region of <b>7</b> is flanked by a region of exposed peptide backbone (residues 458â460) and the side chain of N121 (S121 in WT), suggesting that polar interactions in this region might improve binding over the lipophilic 2-methylphenyl. Additionally, as the R<sup>3</sup> methyl-phenyl is partially solvent exposed, polar solubilizing moieties were expected to improve pharmacokinetic properties without substantial loss of potency. Unfortunately, all attempts to alter the phenyl present in <b>12e</b> resulted in significant loss of activity across the ALDH1A subfamily. Nevertheless, interesting selectivity trends did emerge. Substituents other than methyl at the ortho position (Cl, OMe) also rendered compounds 1A1 selective, but decreased 1A1 potency (<b>15a</b> and <b>15e</b>) relative to proteo analogue <b>12e</b>. Chloro analogue <b>15a</b> showed a 2-fold improvement in potency (1A1 IC<sub>50</sub> = 250 nM) compared to <b>12a</b>. Because of the comparable size of Cl and CH<sub>3</sub>, this difference suggested a preference for electron withdrawing groups; however, the electron-deficient fluorinated analogue <b>15d</b> was merely equipotent with <b>12e</b> despite the similar size of the substituents. Electron-rich methoxy analogue <b>15e</b> was 4-fold less potent against 1A1 than <b>15a</b>. Meta substituents rendered analogues <b>15b</b> and <b>15f</b> modestly 1A2 selective. Consistent with our hypothesis that disruption of planarity is favorable for solubility in this series, <b>15b</b> was considerably more soluble than <b>12e</b>. Addition of a chlorine in the para position resulted in a 1A2/1A3-selective inhibitor vs 1A1 (<b>15c</b>), while para methoxy analogue <b>15g</b> exhibited very modest selectivity for 1A3. Pyridyl analogues <b>15h</b>â<b>j</b> were substantially less potent across the ALDH1A family, perhaps reflecting a desolvation penalty without formation of productive hydrogen bonds within the active site. Benzyl analogue <b>15k</b> inhibited 1A1 and 1A2 with good selectivity over 1A3. Homologation to phenylethyl analogue <b>15l</b> resulted in potent inhibition of 1A3 (IC<sub>50</sub> = 180 nM) with more than 5-fold selectivity over 1A1 and 1A2 and greater selectivity over other ALDH isoforms as shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. Replacing the phenyl of <b>12e</b> with a methyl group led to a large decrease in potency (<b>15m</b>). The steep SAR at R,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> inconsistent with partial solvent exposure, was further evidence that the binding mode depicted in the crystal structure of <b>7</b> was no longer relevant.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Characterization of N-Phenyl Modifications</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0016.gif" alt="" id="GRAPHIC-d7e2119-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="4" align="center">IC<sub>50</sub><a class="ref internalNav" href="#t3fn2" aria-label="a">a</a>Â (Î¼M)Â orÂ %Â controlÂ atÂ 20Â Î¼M<a class="ref internalNav" href="#t3fn3" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">Â </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" colspan="4" align="center">ALDH</th><th class="colsep0 rowsep0" align="center">Â </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd no.</th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">1A1</th><th class="colsep0 rowsep0" align="center">1A2</th><th class="colsep0 rowsep0" align="center">1A3</th><th class="colsep0 rowsep0" align="center">2</th><th class="colsep0 rowsep0" align="center">aq.Â sol.<a class="ref internalNav" href="#t3fn5" aria-label="d">d</a> (Î¼M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12a</b></td><td class="colsep0 rowsep0" align="center">2-Me-Ph</td><td class="colsep0 rowsep0" align="center">0.66Â Â±Â 0.06</td><td class="colsep0 rowsep0" align="center">52%</td><td class="colsep0 rowsep0" align="center">4Â Â±Â 1</td><td class="colsep0 rowsep0" align="center">94%</td><td class="colsep0 rowsep0" align="center">38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>12e</b></td><td class="colsep0 rowsep0" align="center">Ph</td><td class="colsep0 rowsep0" align="center">0.08Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">0.15Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">0.09Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center"><0.7Â (BLQ)<a class="ref internalNav" href="#t3fn6" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15a</b></td><td class="colsep0 rowsep0" align="center">2-Cl-Ph</td><td class="colsep0 rowsep0" align="center">0.25Â Â±Â 0.04</td><td class="colsep0 rowsep0" align="center">75%</td><td class="colsep0 rowsep0" align="center">10Â Â±Â 2</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15b</b></td><td class="colsep0 rowsep0" align="center">3-Cl-Ph</td><td class="colsep0 rowsep0" align="center">4.1Â Â±Â 0.6</td><td class="colsep0 rowsep0" align="center">0.56Â Â±Â 0.06</td><td class="colsep0 rowsep0" align="center">2.6Â Â±Â 0.3</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15c</b></td><td class="colsep0 rowsep0" align="center">4-Cl-Ph</td><td class="colsep0 rowsep0" align="center">69%</td><td class="colsep0 rowsep0" align="center">1.1Â Â±Â 0.1</td><td class="colsep0 rowsep0" align="center">0.75Â Â±Â 0.05</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15d</b></td><td class="colsep0 rowsep0" align="center">2-F-Ph</td><td class="colsep0 rowsep0" align="center">0.11Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="center">8%<a class="ref internalNav" href="#t3fn4" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">1%<sup>c</sup></td><td class="colsep0 rowsep0" align="center">90%</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15e</b></td><td class="colsep0 rowsep0" align="center">2-OMe-Ph</td><td class="colsep0 rowsep0" align="center">0.97Â Â±Â 0.10</td><td class="colsep0 rowsep0" align="center">>10</td><td class="colsep0 rowsep0" align="center">>10</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15f</b></td><td class="colsep0 rowsep0" align="center">3-OMe-Ph</td><td class="colsep0 rowsep0" align="center">41%</td><td class="colsep0 rowsep0" align="center">1.3Â Â±Â 0.1</td><td class="colsep0 rowsep0" align="center">44%</td><td class="colsep0 rowsep0" align="center">82%</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15g</b></td><td class="colsep0 rowsep0" align="center">4-OMe-Ph</td><td class="colsep0 rowsep0" align="center">8.2Â Â±Â 0.7</td><td class="colsep0 rowsep0" align="center">1.77Â Â±Â 0.09</td><td class="colsep0 rowsep0" align="center">0.78Â Â±Â 0.09</td><td class="colsep0 rowsep0" align="center">86%</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15h</b></td><td class="colsep0 rowsep0" align="center">3-Pyridyl</td><td class="colsep0 rowsep0" align="center">3.83Â Â±Â 0.03</td><td class="colsep0 rowsep0" align="center">45%</td><td class="colsep0 rowsep0" align="center">117%</td><td class="colsep0 rowsep0" align="center">99%</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15i</b></td><td class="colsep0 rowsep0" align="center">4-Pyridyl</td><td class="colsep0 rowsep0" align="center">52%</td><td class="colsep0 rowsep0" align="center">52%</td><td class="colsep0 rowsep0" align="center">60%</td><td class="colsep0 rowsep0" align="center">97%</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15j</b></td><td class="colsep0 rowsep0" align="center">2-Pyridyl</td><td class="colsep0 rowsep0" align="center">2.4Â Â±Â 0.4</td><td class="colsep0 rowsep0" align="center">64%</td><td class="colsep0 rowsep0" align="center">67%</td><td class="colsep0 rowsep0" align="center">99%</td><td class="colsep0 rowsep0" align="center">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15k</b></td><td class="colsep0 rowsep0" align="center">Bn</td><td class="colsep0 rowsep0" align="center">0.89Â Â±Â 0.13</td><td class="colsep0 rowsep0" align="center">0.72Â Â±Â 0.04</td><td class="colsep0 rowsep0" align="center">>10</td><td class="colsep0 rowsep0" align="center">100%</td><td class="colsep0 rowsep0" align="center">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15l</b></td><td class="colsep0 rowsep0" align="center">Ph-CH<sub>2</sub>-CH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">1.4Â Â±Â 0.15</td><td class="colsep0 rowsep0" align="center">0.9Â Â±Â 0.1</td><td class="colsep0 rowsep0" align="center">0.18Â Â±Â 0.04</td><td class="colsep0 rowsep0" align="center">87%</td><td class="colsep0 rowsep0" align="center">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>15m</b></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="center">4.1Â Â±Â 0.37</td><td class="colsep0 rowsep0" align="center">79%</td><td class="colsep0 rowsep0" align="center">66%</td><td class="colsep0 rowsep0" align="center">95%</td><td class="colsep0 rowsep0" align="center">Â </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn2"><div class="footnote" id="t3fn2"><sup><sup>a</sup></sup><p class="last">Mean Â± SEM (<i>n</i> = 3).</p></div><div class="footnote" id="t3fn3"><sup><sup>b</sup></sup><p class="last">Mean (<i>n</i> = 3).</p></div><div class="footnote" id="t3fn4"><sup><sup>c</sup></sup><p class="last">Dose response curves were not satisfactory for IC<sub>50</sub> determination;</p></div><div class="footnote" id="t3fn5"><sup><sup>d</sup></sup><p class="last">Thermodynamic solubility analysis was performed by Analiza Inc. using quantitative nitrogen detection (<a href="http://www.analiza.com" class="extLink">www.analiza.com</a>);</p></div><div class="footnote" id="t3fn6"><sup><sup>e</sup></sup><p class="last">BLQ = below limit of quantitation.</p></div></div></div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fraction of enzyme activity remaining for 9 ALDH isoforms in the presence of 20 Î¼M <b>15l</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Crystal Structure of ALDH1A1 Complexed with Compound <b>13g</b></h3><div class="NLM_p">To elucidate the change in binding mode we suspected, we obtained a cocrystal structure of N121S ALDH1A1 with compound <b>13g</b> resolved to 2.0 Ã (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DUM">6DUM</a>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Table S1</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Figure S3</a>). The structure revealed a dramatically different binding pose for <b>13g</b> in which the R<sup>3</sup><i>N</i>-phenyl is situated near the catalytic cysteine. The <i>N</i>-phenyl is flanked by Phe171 and Phe466, potentially explaining why steric bulk and polarity are not tolerated at the ortho and meta positions (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). It is more difficult to justify why para substituents are so poorly tolerated. In contrast to the binding pose of compound <b>7</b>, there is room for fairly large N-2 pyrazole substituents to extend across the mouth of the active site. Consistent with this observation, larger substituents at N-2 improve 1A1 potency (<b>13f</b> and <b>13g</b>) relative to methyl analogue <b>13e</b>. As expected, given that large substituents at N-1 would project into solvent, compounds <b>12f</b>â<b>h</b> do not improve potency over methyl analogue <b>12e</b>.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) X-ray crystal structure of <b>13g</b> bound to ALDH1A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DUM">6DUM</a>). (B) Overlay of compounds <b>7</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI">5TEI</a>) (orange) and <b>13g</b> (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Homology Modeling</h3><div class="NLM_p">In order to better understand the SAR of these compounds, we performed a molecular docking study of select compounds against all isoforms of the ALDH1A class. At the time this manuscript was drafted, no high quality crystal structures were available for 1A2 and 1A3. Recently, Chen et al. disclosed high quality structures of 1A2 with small molecules bound in the active site (e.g., PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5H">6B5H</a>).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> We opted for building our own homology models of 1A2 and 1A3 isoforms based on the PDB structure of ALDH1A1 reported here. Homology models were developed on Swiss-Model Workspace using ProMod3. The quality of the models was assessed by QMEAN scores as well as Ramachandran plots (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplementary Figure S4,S5</a>). Furthermore, when comparing all residues in our 1A2 homology model with PDB:<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5H">6B5H</a>, the average RMSD was 0.76 Ã, indicative of an excellent homology model (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplementary Figure S4</a>). In order to prove robustness of our docking methods, we docked cocrystal ligands <b>7</b> and <b>13g</b> into the binding site of ALDH1A1. The docked poses for both ligands were in good agreement with the crystal structures with RMSD values of 0.8 and 1.0 Ã for <b>7</b> and <b>13g</b>, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplementary Figure S6</a>).</div><div class="NLM_p">The molecular docking study of compound <b>7</b> in ALDH1A2 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B) and 1A3 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C) shows that the 2-methyl substituent on the phenyl ring would clash with nonconserved residues Leu477 (1A2) or Leu471 (1A3) if it adopted the same binding pose as in 1A1. The corresponding residue Val460 in ALDH1A1 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A) avoids this steric clash with 2-methyl substituent. These observations agree with the observed 1A1 selectivity of <b>7</b>.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. ALDH1A2 and 1A3 homology models provide rationale for isoform selectivity of Compound <b>7</b>. (A) Val460 does not clash with <i>o</i>-methyl substituent of <i>N</i>-Phenyl in ALDH1A1- Compound <b>7</b> structure. (B, C) Docking poses of <b>7</b> in ALDH1A2 and ALDH1A3, respectively. Red arrow indicates that in both ALDH1A2 and A3, 2-methyl substituent at phenyl clashes with Leu477/471.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Des-methyl compound <b>11</b>, which is active against all ALDH1A isoforms, docks well into the active site of all three ALDH1A proteins (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). For ALDH1A1, the 3-fluorobenzyl ring engages in piâpi interactions with both Phe171 and Trp178. The bicyclic ring makes a piâpi interaction with Tyr297 and van der Waals interactions with Gly458 and Tyr297, while the phenyl ring participates in van der Waals interactions with Val460 and Asn121. In the active site of ALDH1A2, the bicyclic ring forms piâpi interaction with Phe314 and a van der Waals interaction with Asn475. The 3-fluorobenzyl ring makes a piâpi interaction with Phe188 and van der Waals hydrophobic interactions with Trp195, while the phenyl ring engages in van der Waals hydrophobic interactions with Val138 and Leu477. In the case of ALDH1A3, the bicyclic ring of compound <b>11</b> makes a piâpi interaction with Phe308 and a van der Waals interaction with Asn469. The 3-fluorobenzyl ring forms a piâpi interaction with Phe182 and a van der Waals hydrophobic interaction with Trp189, while the phenyl ring makes van der Waals hydrophobic interactions with Ile132, Phe182 and Leu477.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>11</b> docks without clashes into the active sites of ALDH1A1, 1A2 and 1A3. (A) Structure and ligand interaction plot of Compound <b>11</b> docked into ALDH1A1 crystal structure. (B,C) Structure and ligand interaction plot of compound <b>11</b> docked into ALDH1A2, 1A3 homology models, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">When inhibitor <b>12g</b> was docked, it adopted a similar pose to the crystal structure of the regioisomer <b>13g</b>. The similar potency and isoform selectivity for these compounds is consistent with the relatively solvent exposed nature of the oxetane in both cases. The phenyl ring docks deep in the cavity and forms piâpi interactions with Phe171 and Trp178 (1A1), Phe188(1A2), Phe182(1A3) and van der Waals interactions with Trp195(1A2), Trp189(1A3) as shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. The 3-fluorobenzyl ring makes van der Waals interactions with Val460 and Asn121 (1A1), Leu477 and Val138 (1A2), and Leu471 and Ile132 (1A3). The bicyclic ring of <b>12g</b> engages the same residues as <b>11</b> for the three isoforms. The predicted binding mode of compound <b>13h</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplementary Figure S7</a>), which is also active against all three isoforms, is similar to that of <b>12g</b>.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Docking of compound <b>12g</b> predicts a similar binding mode to the <b>13g</b> crystal structure for ALDH1A1, 1A2, and 1A3. (A) Structure and ligand interaction plot of Compound <b>12g</b> docked into ALDH1A1 crystal structure. (B,C) Structure and ligand interaction plot of compound <b>12g</b> docked into ALDH1A2, 1A3 homology models, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The modestly ALDH1A3 selective compound <b>15g</b>, which has a methoxy group at the 4-position of the phenyl ring, is predicted to bind ALDH1A3 in a similar fashion to <b>7</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). The positioning of the methoxy group near the ALDH1A3 specific residue Glu135, may explain the selectivity of <b>7</b> toward ALDH1A3. In contrast, ALDH1A1 bears Ala124 at this position which could not form a favorable polar contact with the methoxy.</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. ALDH1A3 specific residue Glu135 may explain the observed selectivity of compound <b>15g</b>. (A) <b>15g</b> docked into the ALDH1A3 homology model. (B) Overlay of ALDH1A1 (pink) crystal structure with <b>15g</b> docked in the 1A3 homology model (cyan).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Confirming Activity in Live Cells</h3><div class="NLM_p">We next performed the ALDEFLUOR assay (Stem Cell Technologies) in PEO1 and OVCAR5 cells, to confirm ALDH inhibition in live cells (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A, B). Given the association between ALDH1A3 expression and the high grade serous histology we chose cell lines with high 1A3 activity (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplementary Figure S8</a>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> PEO-1 is classified as a high grade serous ovarian cancer cell line.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> OVCAR5, a commonly used cell line originally classified as high grade serous ovarian, may be gastrointestinal in origin.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The ALDH1A subfamily inhibitors <b>12g</b>, <b>13g</b>, and <b>13h</b> all exhibited strong inhibition of ALDEFLUOR signal with significant activity in both cell lines at 1 Î¼M and reductions of greater than 50% at 10 Î¼M. While both compounds inhibited ALDEFLUOR to some extent, the 1A3 selective compound <b>15l</b> was significantly better than 1A1 selective <b>DEAB</b>. The 1A2 selective inhibitor <b>15b</b> showed little to no inhibition at 30 Î¼M. Compound <b>15b</b> is reasonably soluble and has a lower TPSA and molecular weight than the efficacious inhibitors, suggesting that this effect is indeed due to its isoform selectivity profile and not simply an inability to enter the cell. Despite having biochemical 1A3 IC<sub>50</sub>s (0.12â0.17 Î¼M) that are very similar to <b>15l</b> (0.18 Î¼M), the inhibitors of all 3 ALDH1A isoforms, <b>12g</b>, <b>13g</b>, and <b>13h</b>, were somewhat superior in the ALDEFLUOR assay (statistically significant differences at 3 Î¼M are highlighted in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a>B). Although we cannot rule out variable cell penetration for the compounds as a potential cause, this may indicate that low levels of 1A1 and/or 1A2 expression in these cell lines slightly diminish the effective ALDEFLUOR inhibition of 1A3-selective <b>15l</b>. Low levels of 1A1 and 1A2 mRNA were in fact detected in OVCAR5 but not PEO1 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplementary Figure S8</a>).</div><figure id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compounds <b>15l</b>, <b>12g</b>, <b>13g</b>, and <b>13h</b> inhibit ALDH in cells and deplete cells bearing the putative CSC marker CD133. (A,B) Graphs showing changes in ALDEFLUOR activity in PEO1 (A) and OVCAR5 (B) cells following the specified treatment (<i>N</i> = 3). (C,D) Graphs showing changes in the percent of CD133+ cells as determined by flow cytometry following 48 h drug treatment at the specified concentration (<i>N</i> = 3â6) *<i>p</i> â¤ 0.05, **<i>p</i> â¤ 0.01, ***<i>p</i> â¤ 0.0001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In addition to ALDH, the surface glycoprotein CD133 has also been used as a marker of stemness in many cancers including ovarian cancer, and its expression is correlated with poorer patient outcomes in ovarian cancer.<a onclick="showRef(event, 'ref7 ref46'); return false;" href="javascript:void(0);" class="ref ref7 ref46">(7,46)</a> Interestingly, treating PEO1 and OVCAR5 cells with inhibitors <b>12g</b>, <b>13g</b>, and <b>13h</b> also elicited a dose-dependent preferential reduction in the percentage of CD133<sup>+</sup> cells (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>C,D). The 1A1-selective <b>DEAB</b> was not effective in this assay, and the ALDH1A2 selective <b>15b</b> had only a modest dose-independent effect. The 1A3 selective compound <b>15l</b> depleted CD133<sup>+</sup> cells at 30 Î¼M, but had no effect at 10 Î¼M.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Evaluation of Synergy with Cisplatin</h3><div class="NLM_p">ALDH1A family members are known to play a role in resistance to multiple chemotherapy agents<a onclick="showRef(event, 'ref47 ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49 ref50">(47â50)</a> and a reduction in their activity has been shown to restore sensitivity.<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref51 ref52 ref53">(16â22,51â53)</a> Therefore, we investigated the ability of our inhibitors with the best ALDEFLUOR activity to increase sensitivity to and potentially synergize with cisplatin in PEO1 and OVCAR5 cells. Using the ChouâTalalay method, we determined that <b>12g</b>, <b>13g</b>, and <b>13h</b> are highly synergistic (Combination Index values <1 indicate synergy with smaller numbers indicating stronger synergy) with cisplatin (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A,B, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplemental Figure S9</a>).<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The observed synergy for <b>12g</b> and <b>13g</b> at 1 Î¼M was already strong but it was improved by increasing the concentration of these compounds to 10 Î¼M. While <b>12g</b> and <b>13g</b> alone did not show cytotoxicity at these doses, <b>13h</b> did and, therefore, had to be tested at lower doses. Nonetheless, synergy with <b>13h</b> was observed at concentrations as low as 0.3 Î¼M. As a control, we included the ALDEFLUOR inactive <b>15b</b>, which was somewhat synergistic but was overall the least effective compound, especially at the lower concentrations of cisplatin.</div><figure id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Compounds <b>12g</b>, <b>13g</b>, and <b>13h</b> synergistically enhance the effect of cisplatin on ovarian cancer cell lines and patient derived spheroids. (A,B) Representative heatmaps of combination indices for PEO1 (A) and OVCAR5 (B) cells cultured for 72 h in the presence of cisplatin and the indicated ALDH inhibitor at the concentrations shown. (<i>N</i> = 3) (C) Normalized average viability as determined using Alamarblue dye following exposure to the specified compounds for 72 h. (<i>N</i> = 5â8) *<i>p</i> â¤ 0.05, ****<i>p</i> â¤ 0.0001 one-way ANOVA compared to Control. Combination indices for the specified drug combinations calculated using CompuSyn software. (CI < 1 indicates synergy) (<a href="http://www.combosyn.com" class="extLink">http://www.combosyn.com</a>).<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We similarly evaluated synergy using primary patient derived samples in a 3-D tumor spheroid assay; a platform which mimics in vivo growth (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>C).<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> As <b>13h</b> showed comparable synergy to <b>12g</b> and <b>13g</b> at 3-fold lower concentrations in the initial synergy studies, we tested this compound. Ovarian cancer spheroids, compared with a 2D monolayer culture, are more resistant to standard therapies and express higher levels of ALDH1A1.<a onclick="showRef(event, 'ref20 ref57'); return false;" href="javascript:void(0);" class="ref ref20 ref57">(20,57)</a> The spheroids employed in this experiment express both 1A1 and 1A3 as determined by qRT-PCR (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">Supplementary Figure S10</a>). Treatment of patient-derived ovarian cancer spheroids with 0.3 Î¼M <b>13h</b> did not significantly reduce viability as monotherapy. However, it was highly synergistic with all three concentrations of cisplatin tested (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>C, Combination indices 0.08â0.27).<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Similar to our observations, Condello et al. report that concentrations of <b>4</b> below its ALDEFLUOR IC<sub>50</sub> are able to potentiate the effects of cisplatin against ovarian cancer spheroids.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> In Vivo Murine Metabolic Stability</h3><div class="NLM_p">Given that <b>13g</b> and <b>13h</b> showed promising cellular activity and were the most stable in the presence of MLM among compounds tested, we assessed in vivo exposure following IP injection in mice (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Despite the two compounds having very similar MLM stability, the AUC for <b>13g</b> was more than double that of <b>13h</b>. Plasma concentration of <b>13g</b> exceeded 1 Î¼M, the concentration needed for 50% inhibition of ALDEFLUOR (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A,B), for nearly 7 h. <b>13h</b> exceeded 0.3 Î¼M, an efficacious dose in the spheroid cisplatin synergy assay (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>C), for almost 4 h.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. In Vivo Exposure Following IP Injection in Mice<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="4" align="center">plasma concentration (Î¼M)</th><th class="colsep0 rowsep0" align="center">Â </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">0.5Â h</th><th class="colsep0 rowsep0" align="center">2Â h</th><th class="colsep0 rowsep0" align="center">4Â h</th><th class="colsep0 rowsep0" align="center">7Â h</th><th class="colsep0 rowsep0" align="center">AUC<sub>0â7h</sub><sup>obs</sup> (hrÂ·Î¼M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13g</b></td><td class="colsep0 rowsep0" align="center">7.6Â Â±Â 0.8</td><td class="colsep0 rowsep0" align="center">5.2Â Â±Â 1.0</td><td class="colsep0 rowsep0" align="center">3.0Â Â±Â 1.3</td><td class="colsep0 rowsep0" align="center">0.9Â Â±Â 0.7</td><td class="colsep0 rowsep0" align="center">27.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>13h</b></td><td class="colsep0 rowsep0" align="center">5.6Â Â±Â 0.2</td><td class="colsep0 rowsep0" align="center">2.6Â Â±Â 0.6</td><td class="colsep0 rowsep0" align="center">0.6Â Â±Â 0.2</td><td class="colsep0 rowsep0" align="center">0.1Â Â±Â 0.1</td><td class="colsep0 rowsep0" align="center">11</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Compounds at the specified doses were administered by a single intraperitoneal injection of 10 mg/kg to CD-1 mice. Values expressed as Mean Â± SD for 3 mice at each time point.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In order to explore the structural determinants of ALDH1A isoform selectivity, we generated a number of analogues of lead compound <b>7</b> guided by a cocrystal structure with 1A1. Expedient synthetic routes were developed to allow late-stage diversification at multiple sites. The observation that the ortho-methyl group on the <i>N</i>-phenyl was disfavorable to binding across the ALDH1A subfamily led to the first potent inhibitors of the three ALDH1A isoforms with excellent selectivity over ALDH2 (e.g., <b>12e</b>). Small structural changes led to a new binding mode for compound <b>13g</b>, explaining SAR trends which did not agree with the 1A1 crystal structure for lead compound <b>7</b>. We demonstrated that remarkable changes in ALDH1A isoform-selectivity can be achieved by minor structural changes to the <i>N</i>-phenyl substituent, which binds in the narrow lipophilic region of the active site near the catalytic cysteine according to the <b>13g-1A1</b> structure. In the course of exploring SAR at this position we discovered the 1A2 selective compounds <b>15b</b> and <b>15f</b>, and the 1A3 selective compounds <b>15g</b> and <b>15l</b>. Although selectivity within the ALDH1A subfamily is modest, <i>to our knowledge, these are the first examples of ALDH1A2 and ALDH1A3 selective inhibitors with excellent selectivity against ALDH2</i>. Further efforts to create more potent and selective 1A2 and 1A3 inhibitors are underway.</div><div class="NLM_p">Our preliminary data lead us to hypothesize that ALDEFLUOR activity will be improved by inhibiting multiple ALDH1A isoforms simultaneously as compared to an equipotent 1A3 selective compound (<b>12g</b>, <b>13g</b>, <b>13h</b> vs <b>15l</b>), even in cell lines which only express detectable levels of 1A3. This hypothesis will require more extensive cell testing to confirm our preliminary findings. Testing optimized compounds in an expanded panel of cell lines which express various ALDH1A isoforms will be necessary and is underway.</div><div class="NLM_p last">Interestingly <b>12g</b>, <b>13g</b>, and <b>13h</b> depleted the CD133<sup>+</sup> putative stem cell pool in a dose-dependent manner and synergized with cisplatin at a range of concentrations. <b>13h</b> also showed synergy with cisplatin in patient derived ovarian cancer spheroids. Compound <b>13g</b> furthermore shows promising PK properties for in vivo studies, exceeding its efficacious cellular concentration in plasma for nearly 7 h following 10 mg/kg IP administration in a preliminary study. Xenograft studies on optimal compounds will be reported in due course. Future medicinal chemistry development of this series will focus on further improving enzymatic and cellular potency and pharmacokinetic properties, as well as tuning the selectivity profile among the ALDH1A isoforms in order to generate diverse probes for establishing the contribution of individual isoforms to cellular activity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03185" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03185" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Crystal Structure Determination</h3><div class="NLM_p last">Crystals of ALDH1A1 N121S were grown by equilibrating 4â8 mg/mL ALDH1A1 N121S against 100 mM sodium BisTris, pH 6.2â6.5, 6â11% PEG3350, 200 mM NaCl, and 5â10 mM YbCl<sub>3</sub>. The crystal of ALDH1A1 N121S in complex with <b>7</b> was prepared by soaking approximately 2-week old apo crystals for 6 h in crystallization solution containing 500 Î¼M <b>7</b> with 1% (v/v) DMSO and 1 mM NAD. The crystal of ALDH1A1 N121S in complex with <b>13g</b> was prepared by soaking apo crystals for 1 h in crystallization solution containing 1 mM NAD followed by soaking overnight in crystallization solution containing 500 Î¼M <b>13g</b> with 2% (v/v) DMSO and 1 mM NAD. Cryoprotection for flash-freezing was 20% (v/v) ethylene glycol in ligand soaking solution. Diffraction data for <b>7</b> in complex with ALDH1A1 N121S was collected at Beamline 23-ID-D (GM/CA), National Institute of General Medicine Sciences and National Cancer Institute of the NIH at APS. Diffraction data for <b>13g</b> in complex with ALDH1A1 N121S was collected at Beamline 19-ID operated by the Structural Biology Consortium at the Advance Photon Source (APS), Argonne National Laboratory. Diffraction data were indexed, integrated, and scaled using HKL3000.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The CCP4 program suite was used for molecular replacement and refinement.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The Coot molecular graphic application was used for model building.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The TLSMD (translation/libration/screw motion determination) server was used to determine dynamic properties of ALDH1A1 N121S.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61,62)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Protein Purification and Enzymatic Assays</h3><div class="NLM_p">Human ALDH1A1, ALDH1A2, ALDH1A3, ALDH2, ALDH1B1, ALDH3A1, ALDH4A1, ALDH5A1, and rat ALDH1L1 were prepared and purified as previously described.<a onclick="showRef(event, 'ref27 ref63 ref64 ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref27 ref63 ref64 ref65 ref66">(27,63â66)</a></div><div class="NLM_p last">Inhibition of ALDH activity by compounds and IC<sub>50</sub> curves were determined by measuring the formation of NAD(P)H spectrophotometrically at 340 nm (molar extinction coefficient of 6200 M<sup>â1</sup> cm<sup>â1</sup>) on the Beckman DU-640 and Spectramax 340PC spectrophotometers using purified recombinant enzyme. Reaction components for ALDH1A and ALDH2 assays consisted of 100â200 nM enzyme, 200 Î¼M NAD<sup>+</sup>, 100 Î¼M propionaldehyde, and 1% DMSO in 25 mM BES buffer, pH 7.5. For ALDH1B1, the assay included 500 Î¼M NAD<sup>+</sup> and 200 Î¼M propionaldehyde. For ALDH3A1, the assay included 300 Î¼M NADP<sup>+</sup>, 20 nM enzyme, and 300 Î¼M benzaldehyde. For ALDH4A1 and ALDH5A1, the assay included 1.5 mM NAD<sup>+</sup>, 100 nM enzyme, with 20 mM propionaldehyde for ALDH4A1 and 2 mM propionaldehyde for ALDH5A1. For rat ALDH1L1, the assay included 0.5 mM NADP<sup>+</sup>, 200 nM enzyme, and 4 mM propionaldehyde. All assays were performed at 25 Â°C and were initiated by addition of substrate after a 2 min incubation period. IC<sub>50</sub> curves were collected for compounds which substantially inhibited ALDH1A activity at 20 Î¼M compound. Data were fit to the four parameter EC<sub>50</sub> equation using SigmaPlot (v12) and the values represent the mean/SEM of three independent experiments (each <i>n</i> = 3).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> ALDEFLUOR Assay</h3><div class="NLM_p last">PEO1 and OVCAR5 cells were grown in RPMI 1640 media (Corning) containing 2 mM glutamine, 2 mM sodium pyruvate (PEO1 only, Gibco), 10% FBS (Sigma), and 1Ã Pen/Strep (Gibco). When the cells reached 80% confluency, they were harvested by trypsinization and assayed for ALDH activity using the aldefluor assay (STEMCELL Technologies). Briefly, the cells were washed with PBS before being centrifuged and resuspended in ALDEFLUOR buffer. Once the cells were resuspended in ALDEFLUOR buffer, the ALDEFLUOR reagent was added. The cells were quickly mixed and evenly distributed into 1.5 mL Eppendorf tubes containing inhibitor or vehicle. The tubes were incubated for 30 min at 37 Â°C. Next, the tubes were centrifuged, the buffer containing the ALDEFLUOR reagent was removed, and the cells were resuspended in fresh ALDEFLUOR buffer that had been kept on ice. The tubes were kept on ice until they were analyzed on an Accuri C6 flow cytometer (BD). The percent of control values were calculated using the percentage of ALDEFLUOR positive cells for a particular sample and the percentage of ALDEFLUOR positive cells in the control sample (vehicle treated). The percentage of ALDEFLUOR positive cells was graphed in Prism 7 (GraphPad) and is displayed as mean Â± SD. The Two-way ANOVA with Tukeyâs multiple comparison test within Prism 7 (GraphPad) was used to determine statistical significance between samples treated with compound or vehicle.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> CD133<sup>+</sup> Cell Assay</h3><div class="NLM_p last">PEO1 and OVCAR5 cells were grown in RPMI 1640 media (Corning) containing 2 mM glutamine, 2 mM sodium pyruvate (PEO1 only, Gibco)10% FBS (Sigma), and 1Ã Pen/Strep (Gibco). When the cells reached 80% confluency, they were harvested by trypsinization, washed with PBS, and reseeded in 6-well plates at 1 Ã 10<sup>5</sup> cells/well. The cells were allowed to recover for 24 h, inhibitor or vehicle was added, and the cells were put back into the incubator for 48 h. Following the 48 h incubation, the cells were harvested and counted. An equal number of cells were isolated from each sample. The cells were washed with PBS and resuspended in fresh PBS. CD133/2-APC antibody (Miltenyi Biotec) was added at a dilution of 1:100. The cells were mixed and incubated at room temperature for 10 min. Next, 10 times the volume of PBS was added, and the cells were centrifuged. The cells were washed with PBS and resuspended in fresh PBS that had been kept on ice. The tubes were kept on ice until they could be analyzed on a Cytoflex S flow cytometer (BeckmanCoulter). The percentage of CD133 positive cells was graphed in Prism 7 (GraphPad) and is displayed as mean Â± SD. The Two-way ANOVA with Tukeyâs multiple comparison test within Prism 7 (GraphPad) was used to determine statistical significance between samples treated with compound or vehicle.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Cell Viability and Synergy Assays</h3><div class="NLM_p last">PEO1 and OVCAR5 cells were grown in RPMI 1640 media (Corning) containing 2 mM glutamine, 2 mM sodium pyruvate (PEO1 only, Gibco)10% FBS (Sigma), and 1Ã Pen/Strep (Gibco). When the cells reached 80% confluency, they were harvested by trypsinization, washed with PBS, and reseeded in 96-well plates at 4000 cells/well (PEO1) or 1500 cells/well (OVCAR5). The cells were allowed to recover for 24 h, inhibitor or vehicle was added, cisplatin was added, and the cells were put back into the incubator for 72 h. Following the 72 h incubation, the media was removed, and a 1Ã solution of Cell-Titer Glo 2.0 (Promega) was added. The plates were mixed and allowed to incubate at room temperature for 10 min before luminescence was read. Normalized viability was calculated by comparing the luminesence of drug-treated wells to vehicle treated wells and expressed as a percentage. The percentage of viable cells was graphed in Prism 7, and all data are displayed as mean Â± SD. Synergy was assessed using ChouâTalalay method and the CompuSyn program.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Hanging Drop Culture for Patient Derived Spheroid Formation</h3><div class="NLM_p last">Graded patient tumor samples (high-grade epithelial stage IIIC or IV, obtained from IRB approved protocol, and collected from consented patients) were dissected, digested enzymatically using a mixture of Collagenase IV/Dispase, recovered by centrifugation, and filtered through a 500 Î¼m mesh to remove large debris. Resulting patient cells were suspended in serum-free medium (SFM) supplemented with 5 ng/mL FGF, 5 ng/mL EGF, B27, 1Ã insulin-transferrin-selenium supplement, 1Ã nonessential amino acids, antibiotics, and antimycotics. Cell counts were obtained using a hemocytometer and adjusted such that a 20 Î¼L volume contained 100 cells. Similar to previously established protocol, patient derived spheroids were initiated using 100 CD133<sup>+</sup> ALDH<sup>+</sup> cells per spheroid.<a onclick="showRef(event, 'ref55 ref57 ref67'); return false;" href="javascript:void(0);" class="ref ref55 ref57 ref67">(55,57,67)</a> Spheroids were maintained in SFM for a period of 10 days, and they imaged using live phase contrast microscopy to follow spheroid formation and maintenance.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Drug Treatment on Patient Derived Spheroids in Hanging Drop Array Culture</h3><div class="NLM_p">Patient derived spheroids were initiated in 384-well hanging drop arrays using 100 cells per spheroid. Spheroids were allowed to aggregate and form a 3D microtissue over a period of 10 days. For drug treatment, a 10Ã stock of drug was prepared independently, and 2 Î¼L of drug was added to the 20 Î¼L hanging drop containing spheroids, to result in a final concentration of 1Ã. Several drug-dosing regimens were carried out, including cisplatin in the range of 1â50 Î¼M and ALDH inhibitor <b>13h</b> in the range of 0.03â30 Î¼M.</div><div class="NLM_p last">The effect of drug was assayed on spheroids 72 h, using the alamarBlue assay to determine viability after drug treatment. Control untreated spheroids were maintained for the same duration in culture. Cell viability after drug treatment was normalized to untreated controls and quantified. At least 20 spheroids (technical replicates) were assayed per experiment, with 5 to 8 biological replicates. Drug-treated spheroids were imaged using phase-contrast microscopy to observe morphologic differences in spheroids exposed to drug compared with control untreated spheroids.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Metabolic Activity in Hanging Drop Ovarian Cancer Spheroids</h3><div class="NLM_p last">alamarBlue dye (Life Technologies, Carlsbad CA) was added in a 1:10 dilution to 100 cells/drop spheroids. Following 4 h of alamarBlue addition and incubation, the 384 hanging drop array was placed in a fluorescence plate reader (Synergy HT, BioTek Instruments, Winooski, VT). alamarBlue fluorescence readings were obtained at 530 nm excitation and 590 nm emission. To quantify viability within spheroids, alamarBlue readings of treated as well untreated spheroids were obtained at Day 10, and compared to the untreated Control spheroids.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Statistical Analysis</h3><div class="NLM_p last">Drug viability data were determined and quantified using the alamarBlue assay, as outlined previously.<a onclick="showRef(event, 'ref55 ref57'); return false;" href="javascript:void(0);" class="ref ref55 ref57">(55,57)</a> Briefly, normalized viability was calculated by comparing the alamarBlue fluorescence of drug-treated spheroids to control untreated spheroids and expressed as a percentage. Statistical analysis was performed using one-way ANOVAs and levels of statistical significance are indicated in the figures. All data are expressed as mean Â± SEM and are an average of at least 5 to 8 independent experiments.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Metabolic Stability in Mouse Liver Microsomes</h3><div class="NLM_p last">The metabolic stability was assessed using CD-1 mouse liver microsomes. One Î¼M of each compound was incubated with 0.5 mg/mL microsomes and 1.7 mM cofactor Î²-NADPH in 0.1 M phosphate buffer (pH = 7.4) containing 3.3 mM MgCl<sub>2</sub> at 37 Â°C. The DMSO concentration was less than 0.1% in the final incubation system. At 0, 5, 10, 15, 30, 45, and 60 min of incubation, 40 Î¼L of reaction mixture were taken out, and the reaction is quenched by adding 3-fold excess of cold acetonitrile containing 100 ng/mL of internal standard for quantification. The collected fractions were centrifuged at 15â¯000 rpm for 10 min to collect the supernatant for LC-MS/MS analysis, from which the amount of compound remaining was determined. The natural log of the amount of compound remaining was plotted against time to determine the disappearance rate and the half-life of tested compounds.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Pharmacokinetic Studies in Mice</h3><div class="NLM_p last">All animal experiments in this study were approved by the University of Michigan Committee on Use and Care of Animals and Unit for Laboratory Animal Medicine (ULAM). The abbreviated pharmacokinetics of compounds <b>13g</b> and <b>13h</b> were determined in female CD-1 mice following intraperitoneal (ip) injection of 10 mg/kg, respectively. Compounds were dissolved in the vehicle containing 20% DMSO, 50% PEG-400, and 30% PBS. Four blood samples (50 Î¼L) were collected over 7 h (at 0.5, 2, 4, and 7 h), centrifuged at 3500 rpm for 10 min, and plasma was frozen at â80 Â°C for later analysis. Plasma concentrations of the compounds were determined by the LC-MS/MS method developed and validated for this study. The LC-MS/MS method consisted of a Shimadzu HPLC system, and chromatographic separation of tested compound which was achieved using a Waters Xbridge-C18 column (5 cm Ã 2.1 mm, 3.5 Î¼m). An AB Sciex QTrap 4500 mass spectrometer equipped with an electrospray ionization source (ABI-Sciex, Toronto, Canada) in the positive-ion multiple reaction monitoring (MRM) mode was used for detection. All pharmacokinetic parameters were calculated by noncompartmental methods using WinNonlin, version 3.2 (Pharsight Corporation, Mountain View, CA, U.S.A.).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Molecular Modeling</h3><div class="NLM_p">Molecular modeling was performed on a Dell Precision T7400n Mini-Tower, Quad Core Xeon Proc X5450 dual processor with 32 nodes computer.</div><div id="sec4_12_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Homology Modeling</h4><div class="NLM_p">Homology models for ALDH1A2 and 1A3 were built on Swiss-Model Workspace (SMW) using ALDH1A1 PDB (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI">5TEI</a>) as a template.<a onclick="showRef(event, 'ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref68 ref69">(68,69)</a> The homology modeling on SMW can be briefly described as alignment of a target sequence and a template sequence using the âautomated modeâ since the similarity between ALDH1A isoforms are approximately 70%. The target/template alignment along with 3D coordinates of the template were used as input for generating an all-atom model for the target sequence using ProMod3, a comparative modeling engine. Side chains were remodeled using a backbone dependent rotamer library. Finally, the geometry of the resulting model was energy minimized by using the force field OpenMM.<a onclick="showRef(event, 'ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref68 ref69">(68,69)</a></div><div class="NLM_p last">Global and per-residue quality of the model was assessed by the QMEAN scoring function, where several statistical potential terms, that is, geometrical features of the model as well as each residue (pairwise atomic distances, torsion angles, solvent accessibility, etc.), are compared to statistical parameters obtained from experimental structures.<a onclick="showRef(event, 'ref68 ref70'); return false;" href="javascript:void(0);" class="ref ref68 ref70">(68,70)</a> The local scores are linear combinations of the four statistical potential terms evaluated on a per-residue basis. Each residue gets a score between 0 and 1, with 1 being the best. To assess the overall quality of the model, global QMEAN scores were calculated as a <i>Z</i>-score which compares the obtained values to scores calculated from a set of high-resolution X-ray structures. GMQE (Global Model Quality Estimation) is another global quality estimation which combines properties from the target-template alignment.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> The resulting GMQE score is between 0 and 1 and expresses the accuracy of a model built with that alignment and template. The quality of the model increases as the number approaches 1. Additionally, the model quality was evaluated by a Ramachandran plot within the PROCHECK server.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> PROCHECK assesses the stereochemical quality of a protein structure. A good quality model would be expected to have over 90% in the most favored regions.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a></div></div><div id="sec4_12_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Molecular Docking</h4><div class="NLM_p last">Molecular docking studies were performed using the HYBRID docking program from OpenEye Scientific, Santa Fe, NM against ALDH1A isoforms with cocrystal <b>7</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI">5TEI</a>) or <b>13g</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DUM">6DUM</a>) as the reference ligands.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Prior to docking, 500 different conformations were generated for each ligand using Omega (OpenEye Scientific, Santa Fe, NM), a systematic, knowledge-based conformer generator.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> HYBRID performs a systematic, exhaustive, nonstochastic examination of poses within the protein active site. However, HYBRID reduces this search space based on shape and chemical complementarity to reference bound ligands. It treats ligand conformers as rigid during the docking process, although ligand flexibility is implicitly included by docking multiple conformers of each ligand. The input for HYBRID is a protein structure and a multiconformer representation of the ligand to be docked. In the exhaustive search using the Chemical Gaussian Overlay (CGO) scoring function, each ligand conformation is systematically rotated and translated within the active site at a resolution of 1 Ã. Every pose that passes a bump check is scored.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></div></div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Chemistry</h3><div class="NLM_p">All reagents were used in the condition received from commercial sources. <sup>1</sup>H NMR and <sup>13</sup>C NMR were taken in CDCl<sub>3</sub> or DMSO-<i>d</i><sub>6</sub> at room temperature on Varian Inova 400 or 500 MHz instruments. Reported chemical shifts are expressed in parts per million (ppm) on the Î´ scale from an internal standard of tetramethylsilane (0 ppm). Mass spectrometry data were obtained on either an Agilent TOF or Agilent Q-TOF. An Agilent 1100 series HPLC with an Agilent Zorbax Eclipse PlusâC18 column was used to determine purity of biologically tested compounds. All tested compounds were determined to be >95% pure using a 6 min gradient of 10â90% acetonitrile in water followed by a 2 min hold at 90% acetonitrile with detection at 254 nm. Flash chromatographic purifications were performed using a Teledyne ISCO Combiflash RF with Redisep Gold RF columns.</div><div id="sec4_13_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 6-((3-Fluorobenzyl)thio)-5-(o-tolyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>7</b>)</h4><div class="NLM_p last">To a flask charged with potassium carbonate (1.56 g, 11.3 mmol), <b>9a</b> (1.46 g, 5.7 mmol), and 10 mL DMF was added 3-fluorobenzyl bromide (0.69 mL, 5.7 mmol). After stirring under N<sub>2</sub> for 16 h at RT the reaction was neutralized with sat. aq. NH<sub>4</sub>Cl and the mixture was diluted with water and extracted 2Ã with EtOAc. The combined organic portions were washed three times with brine and then dried with sodium sulfate and the solvent removed. The crude was purified by flash (50% EtOAc in Hex) yielding the titled compound as a white solid (1.26g, 61% yield). HRMS (ESI): <i>m</i>/<i>z</i> 367.1022 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.40 (br. s, 1H), 8.15 (s, 1H), 7.40 (t, <i>J</i> = 8.30 Hz, 1H), 7.34 (d, <i>J</i> = 7.83 Hz, 1H), 7.31 (t, <i>J</i> = 7.83 Hz, 1H), 7.21 (t, <i>J</i> = 7.80 Hz, 1H), 7.18 (d, <i>J</i> = 7.83 Hz, 1H), 7.07 (d, <i>J</i> = 7.34 Hz, 1H), 7.02 (d, <i>J</i> = 9.29 Hz, 1H), 6.90 (dt, <i>J</i> = 1.96, 8.56 Hz, 1H), 4.28 (d, <i>J</i> = 13.69 Hz, 1H), 4.31 (d, <i>J</i> = 13.69 Hz, 1H), 2.10 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.59 (d, <i>J</i> = 246.5 Hz), 161.52, 157.71, 153.63, 138.25 (d, <i>J</i> = 7.5 Hz), 136.92, 135.62, 134.42, 131.40, 130.42, 130.01 (d, <i>J</i> = 8.2 Hz), 129.36, 127.37, 124.82 (d, <i>J</i> = 3.2 Hz), 116.08 (d, <i>J</i> = 21.9 Hz), 114.54 (d, <i>J</i> = 21.1 Hz), 102.86, 36.55, 17.28. HPLC Purity: 99%.</div></div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Method A: General Method for Synthesis of Pyrazolopyrimidinones <b>10a</b>,<b>b</b></h3><div class="NLM_p">A solution of the appropriate isothiocyanate (12.9 mmol) and ethyl 5-amino-1<i>H</i>-pyrazole-4-carboxylate (<b>8a</b>) (12.9 mmol) in 15 mL of toluene was refluxed under N<sub>2</sub> for 3 h. The thiourea intermediate, which precipitated upon cooling to RT, was collected and added to 10 mL of 1N NaOH and refluxed for 3 h. The product was precipitated by the addition of concentrated HCl and collected by filtration.</div><div id="sec4_14_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 6-Mercapto-5-(o-tolyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>10a</b>)</h4><div class="NLM_p last">Method A. White solid. 44% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 13.77 (br. s., 1H), 13.41 (br. s., 1H), 8.57 (br. s., 1H), 7.19â7.31 (m, 3H), 7.09 (d, <i>J</i> = 5.87 Hz, 1H), 1.99 (s, 3H).</div></div><div id="sec4_14_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 5-Phenyl-6-thioxo-6,7-dihydro-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>10b</b>)</h4><div class="NLM_p last">Method A. White solid. 64% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 13.73 (s, 1H), 13.37 (s, 1H), 8.58 (s, 1H), 7.44 (t, <i>J</i> = 7.6 Hz, 2H), 7.37 (t, <i>J</i> = 7.4 Hz, 1H), 7.20 (d, <i>J</i> = 7.7 Hz, 2H).</div></div><div id="sec4_14_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 6-((3-Fluorobenzyl)thio)-5-phenyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>11</b>)</h4><div class="NLM_p last">Starting from compound <b>9b</b> the titled compound was synthesized and purified in a similar manner to compound <b>7</b> and obtained as a white solid. 69% yield. HRMS (ESI): <i>m</i>/<i>z</i> 353.0867 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 12.54 (br. s., 1H), 8.08 (s, 1H), 7.47 (s, 3H), 7.30 (s, 2H), 7.14â7.22 (m, 1H), 7.06 (d, <i>J</i> = 7.83 Hz, 1H), 7.02 (d, <i>J</i> = 9.39 Hz, 1H), 6.89 (t, <i>J</i> = 8.60 Hz, 1H), 4.22â4.35 (m, 2H), <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 162.46 (d, <i>J</i> = 246.6 Hz), 161.46, 158.34, 153.26, 138.12 (d, <i>J</i> = 7.6 Hz), 135.27, 135.13, 129.98, 129.91 (d, <i>J</i> = 9.0 Hz), 129.60, 129.27, 124.84 (d, <i>J</i> = 2.9 Hz), 116.07 (d, <i>J</i> = 21.8 Hz), 114.42 (d, <i>J</i> = 21.1 Hz), 102.73, 36.68. HPLC Purity: 99%.</div></div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Method B. General Method for Synthesis of <i>N</i>-Alkylated Pyrazolopyrimidinones <b>12a</b>â<b>d</b>, <b>f</b>â<b>h</b>, <b>13a</b>â<b>h</b></h3><div class="NLM_p">To a dry flask charged with <b>7</b> or <b>11</b> (0.27 mmol) and potassium carbonate (0.55 mmol) under N<sub>2</sub> was added 2 mLof DMF and the appropriate alkyl halide or mesylate (0.34 mmol) by syringe. The mixture was heated to 50 Â°C for 16 h at which point starting material was consumed by HPLC. The mixture was diluted with ethyl acetate and washed with water and three times with brine. The organic portion was dried over sodium sulfate, and the solvent was removed. The regioisomers were separated by flash (EtOAc in Hex gradient). The 1-alkyl pyrazoles (<b>12a</b>â<b>h</b>) eluted more quickly than the 2-alkyl pyrazoles (<b>13a</b>â<b>h</b>). The alkylating reagents used are designated below for each compound. All were commercially available except for oxetan-3-ylmethylmethanesulfonate which was prepared as previously described.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a></div><div id="sec4_15_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 6-((3-Fluorobenzyl)thio)-1-methyl-5-(o-tolyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>12a</b>)</h4><div class="NLM_p last">Method B. Methyl iodide. White solid. 43% yield. HRMS (ESI): <i>m</i>/<i>z</i> 381.1179 [M + H]<sup>+Â 1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 8.06 (s, 1H), 7.41 (dt, <i>J</i> = 1.20, 6.40 Hz, 2H), 7.31â7.38 (m, 2H), 7.23â7.31 (m, 3H), 7.06 (tt, <i>J</i> = 1.60, 8.60 Hz, 1H), 4.38 (d, <i>J</i> = 13.30 Hz, 1H), 4.43 (d, <i>J</i> = 13.30 Hz, 1H), 3.96 (s, 3H), 1.95 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.63 (d, <i>J</i> = 246.5 Hz), 161.21, 157.07, 151.01, 138.62 (d, <i>J</i> = 7.4 Hz), 136.93, 135.40, 134.49, 131.41, 130.40, 130.01 (d, <i>J</i> = 8.4 Hz), 129.35, 127.37, 124.67 (d, <i>J</i> = 2.9 Hz), 116.05 (d, <i>J</i> = 22.0 Hz), 114.54 (d, <i>J</i> = 21.0 Hz), 102.86, 36.56 (d, <i>J</i> = 2.0 Hz), 34.09, 17.29. HPLC Purity: 97%.</div></div><div id="sec4_15_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 1-(Cyclopropylmethyl)-6-((3-fluorobenzyl)thio)-5-(o-tolyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>12b</b>)</h4><div class="NLM_p last">Method B. Cyclopropylmethyl bromide. Colorless oil. 32% yield. HRMS (ESI): <i>m</i>/<i>z</i> 421.1493 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.05 (s, 1H), 7.42 (t, <i>J</i> = 7.80 Hz, 1H), 7.31â7.39 (m, 2H), 7.21â7.28 (m, 1H), 7.17 (d, <i>J</i> = 7.34 Hz, 1H), 7.12 (d, <i>J</i> = 7.83 Hz, 1H), 7.07 (d, <i>J</i> = 9.78 Hz, 1H), 6.94 (dt, <i>J</i> = 2.20, 8.44 Hz, 1H), 4.32 (d, <i>J</i> = 13.69 Hz, 1H), 4.29 (d, <i>J</i> = 14.18 Hz, 1H), 4.19 (d, <i>J</i> = 7.20 Hz, 2H), 2.12 (s, 3H), 1.32â1.44 (m, 1H), 0.62 (td, <i>J</i> = 5.50, 7.80 Hz, 2H), 0.44 (dt, <i>J</i> = 4.80, 5.50 Hz, 2H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.66 (d, <i>J</i> = 246.7 Hz), 160.99, 157.09, 150.50, 138.64 (d, <i>J</i> = 7.4 Hz), 136.96, 135.33, 134.53, 131.37, 130.36, 130.01 (d, <i>J</i> = 8.2 Hz), 129.34, 127.33, 124.45 (d, <i>J</i> = 2.7 Hz), 115.81 (d, <i>J</i> = 22.1 Hz), 114.47 (d, <i>J</i> = 21.0 Hz), 102.82, 52.05, 36.49 (d, <i>J</i> = 1.9 Hz), 17.33, 11.16, 3.97. HPLC Purity: 96%.</div></div><div id="sec4_15_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 6-((3-Fluorobenzyl)thio)-1-(oxetan-3-yl)-5-(o-tolyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>12c</b>)</h4><div class="NLM_p last">Method B. 3-Bromooexetane. White solid. 18% yield. HRMS (ESI): <i>m</i>/<i>z</i> 423.1292 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.14 (s, 1H), 7.41â7.46 (m, 1H), 7.32â7.39 (m, 2H), 7.24â7.29 (m, 1H), 7.11â7.17 (m, 2H), 7.06â7.10 (m, 1H), 6.95 (dt, <i>J</i> = 2.20, 8.44 Hz, 1H), 5.91â5.99 (m, 1H), 5.28â5.35 (m, 2H), 5.04â5.09 (m, 2H), 4.30 (d, <i>J</i> = 13.69 Hz, 1H), 4.34 (d, <i>J</i> = 13.69 Hz, 1H), 2.10 (s, 1H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.71 (d, <i>J</i> = 246.9 Hz), 161.93, 156.91, 151.04, 138.31 (d, <i>J</i> = 7.6 Hz), 136.85, 136.37, 134.35, 131.51, 130.55, 130.10 (d, <i>J</i> = 8.4 Hz), 129.26, 127.47, 124.57 (d, <i>J</i> = 2.9 Hz), 115.92 (d, <i>J</i> = 22.0 Hz), 114.66 (d, <i>J</i> = 21.1 Hz), 103.38, 76.85, 50.69, 36.62, 17.32. HPLC Purity: 98%.</div></div><div id="sec4_15_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 6-((3-Fluorobenzyl)thio)-1-(oxetan-3-ylmethyl)-5-(o-tolyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>12d</b>)</h4><div class="NLM_p last">Method B. Oxetan-3-ylmethylmethanesulfonate. White crystalline solid. 42% yield. HRMS (ESI): <i>m</i>/<i>z</i> 437.1449 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.03 (s, 1H), 7.42â7.47 (m, 1H), 7.33â7.40 (m, 2H), 7.25â7.31 (m, 1H), 7.17 (d, <i>J</i> = 7.34 Hz, 1H), 7.14 (d, <i>J</i> = 7.34 Hz, 1H), 7.06â7.11 (m, 1H), 6.97 (dt, <i>J</i> = 2.45, 8.31 Hz, 1H), 4.86 (dt, <i>J</i> = 1.47, 7.09 Hz, 2H), 4.59â4.66 (m, 4H), 4.35 (d, <i>J</i> = 13.69 Hz, 1H), 4.31 (d, <i>J</i> = 14.18 Hz, 1H), 3.52â3.61 (m, 1H), 2.12 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.68 (d, <i>J</i> = 246.7 Hz), 161.69, 156.89, 150.99, 138.43 (d, <i>J</i> = 7.5 Hz), 136.90, 135.82, 134.41, 131.44, 130.46, 130.08 (d, <i>J</i> = 8.2 Hz), 129.28, 127.40, 124.50 (d, <i>J</i> = 2.9 Hz), 115.86 (d, <i>J</i> = 22.0 Hz), 114.58 (d, <i>J</i> = 21.3 Hz), 102.85, 74.85, 74.77, 49.40, 36.64 (d, <i>J</i> = 1.9 Hz), 35.21, 17.32. HPLC Purity: 99%.</div></div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Method C. General Synthesis of Substituted Pyrazolothiopyrimidinones <b>12e</b>, <b>15a</b>â<b>i</b> from <b>14</b></h3><div class="NLM_p">To a dry flask under N<sub>2</sub> charged with <b>14</b> (0.24 mmol) and 2 mL of dry DMF was added the appropriate amine (0.24 mmol) by syringe. The reaction was stirred at RT for 1 h and then cooled to 0 Â°C. 60 wt % NaH in mineral oil(0.24 mmol) was added, and the flask was stirred at 0 Â°C for 20 min and then allowed to warm to RT and stirred for 3 h. The flask was once again cooled to 0 Â°C, and 3-fluorobenzyl bromide (0.24 mmol) was added by syringe. The mixture was stirred at 0 Â°C for 30 min at which point the reaction was complete by HPLC. The reaction was diluted with 10 mL of water and extracted two times with 10 mL of EtOAc. The combined organic portion was washed three times with brine, dried over sodium sulfate, and the solvent removed. The crude product was purified by flash (0â100% EtOAc in Hexanes).</div><div id="sec4_16_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 6-((3-Fluorobenzyl)thio)-1-methyl-5-phenyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>12e</b>)</h4><div class="NLM_p last">Method C. White crystalline solid. 84% yield. HRMS (ESI): <i>m</i>/<i>z</i> 367.1023 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.03 (s, 1H), 7.47â7.58 (m, 3H), 7.22â7.30 (m, 3H), 7.13 (d, <i>J</i> = 7.83 Hz, 1H), 7.09 (d, <i>J</i> = 9.78 Hz, 1H), 6.95 (dt, <i>J</i> = 2.20, 8.44 Hz, 1H), 4.34 (s, 2H), 4.01 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.67 (d, <i>J</i> = 246.6 Hz), 161.31, 157.68, 150.89, 138.47 (d, <i>J</i> = 7.4 Hz), 135.50, 135.42, 130.11, 130.06 (d, <i>J</i> = 8.3 Hz), 129.75, 129.36, 124.76 (d, <i>J</i> = 3.1 Hz), 116.15 (d, <i>J</i> = 21.9 Hz), 114.61 (d, <i>J</i> = 21.0 Hz), 102.95, 36.88, 34.12. HPLC Purity: 98%.</div></div><div id="sec4_16_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 1-(Cyclopropylmethyl)-6-((3-fluorobenzyl)thio)-5-phenyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>12f</b>)</h4><div class="NLM_p last">Method B. Cyclopropylmethyl bromide. White Solid. 47.7% yield. HRMS (ESI): <i>m</i>/<i>z</i> 407.1335 [M + H]<sup>+Â 1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.05 (s, 1H), 7.48â7.57 (m, 3H), 7.20â7.33 (m, 3H), 7.12 (d, <i>J</i> = 7.83 Hz, 1H), 7.08 (d, <i>J</i> = 9.39 Hz, 1H), 6.95 (t, <i>J</i> = 8.41 Hz, 1H), 4.31 (s, 2H), 4.19 (d, <i>J</i> = 7.04 Hz, 2H), 1.38 (s, 1H), 0.54â0.69 (m, 2H), 0.36â0.50 (m, 2H) <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 162.69 (d, <i>J</i> = 246.7 Hz), 161.07, 157.71, 150.37, 138.46 (d, <i>J</i> = 7.6 Hz), 135.53, 135.35 (d, <i>J</i> = 3.7 Hz), 130.07, 130.05 (d, <i>J</i> = 8.1 Hz), 129.72, 129.35, 124.55 (d, <i>J</i> = 2.9 Hz), 115.92 (d, <i>J</i> = 21.9 Hz), 114.54 (d, <i>J</i> = 21.3 Hz), 102.91, 52.05, 36.82, 11.22, 3.98. HPLC Purity: 99%.</div></div><div id="sec4_16_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 6-((3-Fluorobenzyl)thio)-1-(oxetan-3-yl)-5-phenyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>12g</b>)</h4><div class="NLM_p last">Method B. 3-bromooexetane. White solid. 15.5% yield. HRMS (ESI): <i>m</i>/<i>z</i> 409.1131 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.13 (s, 1H), 7.50â7.57 (m, 3H), 7.21â7.31 (m, 3H), 7.12 (d, <i>J</i> = 7.83 Hz, 1H), 7.08 (d, <i>J</i> = 9.29 Hz, 1H), 6.92â6.99 (m, 1H), 5.94 (quin, <i>J</i> = 7.21 Hz, 1H), 5.30 (t, <i>J</i> = 6.36 Hz, 2H), 5.06 (t, <i>J</i> = 7.09 Hz, 2H), 4.33 (s, 2H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.71 (d, <i>J</i> = 246.6 Hz), 161.98, 157.48, 150.90, 138.14 (d, <i>J</i> = 7.4 Hz), 136.33, 135.33, 130.22, 130.12 (d, <i>J</i> = 8.3 Hz), 129.82, 129.25, 124.64 (d, <i>J</i> = 2.9 Hz), 115.99 (d, <i>J</i> = 22.1 Hz), 114.69 (d, <i>J</i> = 21.1 Hz), 103.47, 76.81, 50.75, 36.91. HPLC Purity: 96%.</div></div><div id="sec4_16_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 6-((3-Fluorobenzyl)thio)-1-(oxetan-3-ylmethyl)-5-phenyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>12h</b>)</h4><div class="NLM_p last">Method B. Oxetan-3-ylmethylmethanesulfonate. White solid. 32.5% yield. HRMS (ESI): <i>m</i>/<i>z</i> 423.1289 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.00 (s, 1H), 7.48â7.56 (m, 3H), 7.23â7.31 (m, 3H), 7.14 (d, <i>J</i> = 7.83 Hz, 1H), 7.08 (d, <i>J</i> = 9.78 Hz, 1H), 6.92â6.98 (m, 1H), 4.83 (t, <i>J</i> = 7.09 Hz, 2H), 4.62 (d, <i>J</i> = 7.34 Hz, 2H), 4.60 (t, <i>J</i> = 6.10 Hz, 2H), 4.32 (s, 2H), 3.48â3.59 (m, 1H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.59 (d, <i>J</i> = 246.6 Hz), 161.67, 157.39, 150.77, 138.22 (d, <i>J</i> = 7.6 Hz), 135.68, 135.34, 130.05, 130.2 (d, <i>J</i> = 7.6) 129.99, 129.67, 129.21, 124.51 (d, <i>J</i> = 3.0 Hz), 115.84 (d, <i>J</i> = 22.0 Hz), 114.52 (d, <i>J</i> = 21.0 Hz), 102.84, 74.67, 49.29, 36.85, 35.15. HPLC Purity: 99%.</div></div><div id="sec4_16_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 6-((3-Fluorobenzyl)thio)-2-methyl-5-(o-tolyl)-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>13a</b>)</h4><div class="NLM_p last">Method B. Methyl iodide. Colorless Oil. 27% yield. HRMS (ESI): <i>m</i>/<i>z</i> 381.1179 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.03 (s, 1H), 7.40 (t, <i>J</i> = 7.58 Hz, 1H), 7.29â7.36 (m, 2H), 7.19â7.25 (m, 1H), 7.17 (d, <i>J</i> = 7.83 Hz, 1H), 7.13 (d, <i>J</i> = 7.83 Hz, 1H), 7.07 (d, <i>J</i> = 9.78 Hz, 1H), 6.91 (dt, <i>J</i> = 2.00, 8.30 Hz, 1H), 4.33â4.42 (m, 2H), 4.07 (s, 3H), 2.10 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.61 (d, <i>J</i> = 246.1 Hz), 159.88, 158.40, 157.99, 138.73 (d, <i>J</i> = 7.4 Hz), 137.12, 134.53, 131.25, 130.26, 129.89 (d, <i>J</i> = 8.5 Hz), 129.59, 129.31, 127.20, 124.93 (d, <i>J</i> = 2.9 Hz), 116.09 (d, <i>J</i> = 21.6 Hz), 114.34 (d, <i>J</i> = 21.1 Hz), 104.89, 40.29, 36.60 (d, <i>J</i> = 2.1 Hz), 17.29. HPLC Purity: 99%.</div></div><div id="sec4_16_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 2-(Cyclopropylmethyl)-6-((3-fluorobenzyl)thio)-5-(o-tolyl)-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>13b</b>)</h4><div class="NLM_p last">Method B. Cyclopropylmethyl bromide. White Crystalline solid. 38% yield. HRMS (ESI): <i>m</i>/<i>z</i> 421.1492 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.21 (s, 1H), 7.40 (t, <i>J</i> = 7.34 Hz, 1H), 7.28â7.36 (m, 2H), 7.19â7.25 (m, 1H), 7.17 (d, <i>J</i> = 7.83 Hz, 1H), 7.12 (d, <i>J</i> = 7.83 Hz, 1H), 7.06 (d, <i>J</i> = 9.78 Hz, 1H), 6.91 (t, <i>J</i> = 8.31 Hz, 1H), 4.41 (d, <i>J</i> = 14.18 Hz, 1H), 4.38 (d, <i>J</i> = 14.20 Hz, 1H), 4.13 (d, <i>J</i> = 6.85 Hz, 2H), 2.12 (s, 3H), 1.36â1.47 (m, 1H), 0.69â0.78 (m, 2H), 0.42â0.49 (m, 2H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.58 (d, <i>J</i> = 246.2 Hz), 159.74, 158.16, 158.10, 138.60 (d, <i>J</i> = 7.3 Hz), 137.13, 134.55, 131.22, 130.21, 129.87 (d, <i>J</i> = 8.4 Hz), 129.58, 127.85, 127.17, 124.90 (d, <i>J</i> = 3.1 Hz), 116.08 (d, <i>J</i> = 21.9 Hz), 114.32 (d, <i>J</i> = 21.2 Hz), 104.54, 58.20, 36.64, 17.30, 10.71, 4.19. HPLC Purity: 97%.</div></div><div id="sec4_16_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 6-((3-Fluorobenzyl)thio)-2-(oxetan-3-yl)-5-(o-tolyl)-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>13c</b>)</h4><div class="NLM_p last">Method B. 3-Bromooexetane. White solid. 15% yield. HRMS (ESI): <i>m</i>/<i>z</i> 423.1295 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.21 (s, 1H), 7.39â7.45 (m, 1H), 7.30â7.37 (m, 2H), 7.23 (dt, <i>J</i> = 5.87, 7.83 Hz, 1H), 7.15â7.19 (m, 1H), 7.12 (d, <i>J</i> = 7.83 Hz, 1H), 7.04â7.09 (m, 1H), 6.92 (dt, <i>J</i> = 1.96, 8.56 Hz, 1H), 5.53â5.62 (m, 1H), 5.23 (t, <i>J</i> = 6.60 Hz, 2H), 5.10 (t, <i>J</i> = 7.34 Hz, 2H), 4.38â4.46 (m, 2H), 2.11 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.70 (d, <i>J</i> = 246.7 Hz), 160.89, 158.58, 157.91, 138.34 (d, <i>J</i> = 7.8 Hz), 137.11, 134.40, 131.36, 130.41, 130.00 (d, <i>J</i> = 8.2 Hz), 129.58, 128.09, 127.31, 124.97 (d, <i>J</i> = 3.2 Hz), 116.18 (d, <i>J</i> = 22.2 Hz), 114.51 (d, <i>J</i> = 21.0 Hz), 105.25, 76.81 (d, <i>J</i> = 2.7 Hz), 56.83, 36.87, 17.34. HPLC Purity: 96%.</div></div><div id="sec4_16_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 6-((3-Fluorobenzyl)thio)-2-(oxetan-3-ylmethyl)-5-(o-tolyl)-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>13d</b>)</h4><div class="NLM_p last">Method B. Oxetan-3-ylmethylmethanesulfonate. Colorless oil. 24.3% yield. HRMS (ESI): <i>m</i>/<i>z</i> 437.1449 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.07 (s, 1H), 7.38â7.44 (m, 1H), 7.29â7.37 (m, 2H), 7.22 (dt, <i>J</i> = 6.11, 7.95 Hz, 1H), 7.16 (d, <i>J</i> = 7.34 Hz, 1H), 7.10 (d, <i>J</i> = 7.83 Hz, 1H), 7.05 (dd, <i>J</i> = 1.96, 9.78 Hz, 1H), 6.91 (dt, <i>J</i> = 2.45, 8.31 Hz, 1H), 4.88 (t, <i>J</i> = 7.09 Hz, 2H), 4.59 (d, <i>J</i> = 7.83 Hz, 2H), 4.55 (t, <i>J</i> = 6.36 Hz, 2H), 4.35â4.43 (m, 2H), 3.62â3.71 (m, 1H), 2.11 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.66 (d, <i>J</i> = 246.2 Hz), 160.42, 158.56, 157.96, 138.42 (d, <i>J</i> = 7.7 Hz), 137.13, 134.45, 131.32, 130.36, 129.96 (d, <i>J</i> = 8.3 Hz), 129.57, 128.70, 127.28, 124.95 (d, <i>J</i> = 2.6 Hz), 116.18 (d, <i>J</i> = 21.9 Hz), 114.45 (d, <i>J</i> = 21.1 Hz), 104.89, 74.48 (d, <i>J</i> = 2.7 Hz), 55.85, 36.79, 35.36, 17.35. HPLC Purity: 99%.</div></div><div id="sec4_16_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 6-((3-Fluorobenzyl)thio)-2-methyl-5-phenyl-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>13e</b>)</h4><div class="NLM_p last">Method B. Methyl Iodide. White solid. 24.0% yield. HRMS (ESI): <i>m</i>/<i>z</i> 367.1021 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.02 (s, 1H), 7.46â7.56 (m, 3H), 7.26â7.31 (m, 2H), 7.23 (q, <i>J</i> = 7.34 Hz, 1H), 7.14 (d, <i>J</i> = 7.83 Hz, 1H), 7.08 (d, <i>J</i> = 9.29 Hz, 1H), 6.92 (t, <i>J</i> = 8.07 Hz, 1H), 4.39 (s, 2H), 4.07 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.64 (d, <i>J</i> = 246.3 Hz), 159.88, 158.52, 158.23, 138.61 (d, <i>J</i> = 7.6 Hz), 135.54, 129.91 (d, <i>J</i> = 8.3 Hz), 129.91, 129.57, 129.54, 129.25, 124.98 (d, <i>J</i> = 3.2 Hz), 116.17 (d, <i>J</i> = 21.9 Hz), 114.38 (d, <i>J</i> = 21.3 Hz), 104.95, 40.28, 36.90. HPLC Purity: 96%.</div></div><div id="sec4_16_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 2-(Cyclopropylmethyl)-6-((3-fluorobenzyl)thio)-5-phenyl-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>13f</b>)</h4><div class="NLM_p last">Method B. Cyclopropylmethyl bromide. White Solid. 36.8% yield. HRMS (ESI): <i>m</i>/<i>z</i> 407.1334 [M + H]<sup>+Â 1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.19 (s, 1H), 7.42â7.58 (m, 3H), 7.27 (d, <i>J</i> = 4.30 Hz, 2H), 7.16â7.24 (m, 1H), 7.11 (d, <i>J</i> = 7.43 Hz, 1H), 7.06 (d, <i>J</i> = 9.78 Hz, 1H), 6.90 (t, <i>J</i> = 8.41 Hz, 1H), 4.39 (s, 2H), 4.12 (d, <i>J</i> = 7.04 Hz, 2H), 1.34â1.49 (m, 1H), 0.72 (d, <i>J</i> = 7.83 Hz, 2H), 0.44 (d, <i>J</i> = 4.30 Hz, 2H) <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 162.7, 159.8, 158.8, 158.0, 138.5, 135.6, 130.0, 129.9, 129.6, 129.6, 127.9, 125.0, 116.2, 114.4, 104.7, 58.3, 37.0, 10.5, 4.3. HPLC Purity: 99%.</div></div><div id="sec4_16_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 6-((3-Fluorobenzyl)thio)-2-(oxetan-3-yl)-5-phenyl-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>13g</b>)</h4><div class="NLM_p last">Method B. 3-bromooexetane. White solid. 21.6% yield. HRMS (ESI): <i>m</i>/<i>z</i> 409.1130 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.20 (s, 1H), 7.46â7.54 (m, 3H), 7.26â7.31 (m, 2H), 7.19â7.25 (m, 1H), 7.12 (d, <i>J</i> = 7.34 Hz, 1H), 7.06 (d, <i>J</i> = 9.78 Hz, 1H), 6.92 (dt, <i>J</i> = 1.71, 8.44 Hz, 1H), 5.56 (quin, <i>J</i> = 6.97 Hz, 1H), 5.21 (t, <i>J</i> = 6.60 Hz, 2H), 5.07 (t, <i>J</i> = 7.34 Hz, 2H), 4.42 (s, 2H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.71 (d, <i>J</i> = 246.3 Hz), 160.83, 158.50, 158.42, 138.22 (d, <i>J</i> = 7.4 Hz), 135.42, 130.09, 130.02 (d, <i>J</i> = 6.8 Hz), 129.68, 129.58, 128.18, 125.05 (d, <i>J</i> = 3.0 Hz), 116.26 (d, <i>J</i> = 21.9 Hz), 114.55 (d, <i>J</i> = 21.1 Hz), 105.26, 76.82, 56.81, 37.17. HPLC Purity: 96%.</div></div><div id="sec4_16_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 6-((3-Fluorobenzyl)thio)-2-(oxetan-3-ylmethyl)-5-phenyl-2<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>13h</b>)</h4><div class="NLM_p last">Method B. Oxetan-3-ylmethylmethanesulfonate. White solid. 22.5% yield. HRMS (ESI): <i>m</i>/<i>z</i> 423.1286 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.07 (s, 1H), 7.47â7.54 (m, 3H), 7.25â7.30 (m, 2H), 7.19â7.25 (m, 1H), 7.11 (d, <i>J</i> = 7.34 Hz, 1H), 7.05 (d, <i>J</i> = 9.29 Hz, 1H), 6.92 (dt, <i>J</i> = 1.47, 8.31 Hz, 1H), 4.86 (t, <i>J</i> = 7.09 Hz, 2H), 4.57 (d, <i>J</i> = 7.34 Hz, 2H), 4.54 (t, <i>J</i> = 6.11 Hz, 2H), 4.38 (s, 2H), 3.58â3.70 (m, 1H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.63 (d, <i>J</i> = 246.5 Hz), 160.34, 158.48, 158.36, 138.26 (d, <i>J</i> = 7.5 Hz), 135.42, 129.98, 129.95 (d, <i>J</i> = 7.8 Hz), 129.59, 129.52, 128.68, 124.98 (d, <i>J</i> = 2.9 Hz), 116.21 (d, <i>J</i> = 21.9 Hz), 114.45 (d, <i>J</i> = 21.0 Hz), 104.90, 74.41, 55.76, 37.03, 35.31. HPLC Purity: 99%.</div></div><div id="sec4_16_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Ethyl 5-isothiocyanato-1-methyl-1<i>H</i>-pyrazole-4-carboxylate (<b>14)</b></h4><div class="NLM_p last">To a dry flask under N<sub>2</sub> at 0 Â°C charged with 60 wt % NaH in mineral oil (296 mg, 7.39 mmol) and <b>8b</b> (500 mg, 2.96 mmol) was added 10 mL of THF. The mixture was stirred for 10 min at which point CS<sub>2</sub> (1.8 mL, 29.6 mmol) was added by syringe. The mixture was allowed to warm to room temperature and then heated to 40 Â°C and stirred for 3 h. After cooling the flask to 0 Â°C, iodine was added portion-wise over 10 min. The mixture was stirred for 1 h at 0 Â°C and then 30 mL of diethyl ether was added and the precipitate was filtered off. The filtrate was washed three times with 1 N HCl, one time with brine, and the organic portion was dried over sodium sulfate and the solvent removed yielding a reddish black solid. The crude product was purified by flash (0â30% EA in Hex) yielding the titled compound as a yellow solid (460 mg, 2.18 mmol, 74% yield). MS (ESI): <i>m</i>/<i>z</i> 212.0 [M + H]<sup>+Â 1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.81 (s, 1H), 4.25â4.44 (m, 2H), 3.80 (s, 3H), 1.30â1.44 (m, 3H).</div></div><div id="sec4_16_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 5-(2-Chlorophenyl)-6-((3-fluorobenzyl)thio)-1-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15a</b>)</h4><div class="NLM_p last">Method C. White crystalline solid. 83% yield. HRMS (ESI): <i>m</i>/<i>z</i> 401.0629 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.04 (s, 1H), 7.58 (dd, <i>J</i> = 1.47, 7.83 Hz, 1H), 7.48 (dt, <i>J</i> = 1.71, 7.70 Hz, 1H), 7.43 (dt, <i>J</i> = 1.47, 7.58 Hz, 1H), 7.33 (d, <i>J</i> = 7.83 Hz, 1H), 7.23â7.30 (m, 1H), 7.15 (d, <i>J</i> = 7.34 Hz, 1H), 7.10 (d, <i>J</i> = 9.78 Hz, 1H), 6.95 (dt, <i>J</i> = 1.96, 8.31 Hz, 1H), 4.32â4.42 (m, 2H), 4.01 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.67 (d, <i>J</i> = 246.8 Hz), 160.82, 156.77, 150.91, 138.29 (d, <i>J</i> = 7.3 Hz), 135.54, 133.78, 133.22, 131.56, 131.21, 130.73, 130.05 (d, <i>J</i> = 8.5 Hz), 128.07, 124.69 (d, <i>J</i> = 2.9 Hz), 116.07 (d, <i>J</i> = 22.1 Hz), 114.63 (d, <i>J</i> = 21.0 Hz), 102.72, 36.59, 34.15. HPLC Purity: 99%.</div></div><div id="sec4_16_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 5-(3-Chlorophenyl)-6-((3-fluorobenzyl)thio)-1-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15b</b>)</h4><div class="NLM_p last">Method C. White crystalline solid. 44% yield. HRMS (ESI): <i>m</i>/<i>z</i> 401.0629 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.03 (s, 1H), 7.51 (td, <i>J</i> = 1.60, 8.00 Hz, 1H), 7.47 (t, <i>J</i> = 8.00 Hz, 1H), 7.26â7.32 (m, 2H), 7.18 (td, <i>J</i> = 1.53, 7.70 Hz, 1H), 7.15 (d, <i>J</i> = 7.83 Hz, 1H), 7.11 (td, <i>J</i> = 2.08, 9.54 Hz, 1H), 6.98 (dt, <i>J</i> = 2.69, 8.44 Hz, 1H), 4.32â4.41 (m, 2H), 4.02 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.70 (d, <i>J</i> = 246.8 Hz), 160.84, 157.42, 150.77, 138.20 (d, <i>J</i> = 7.6 Hz), 136.53, 135.47, 135.27, 130.66, 130.47, 130.15 (d, <i>J</i> = 8.1 Hz), 129.77, 127.77, 124.78 (d, <i>J</i> = 3.2 Hz), 116.19 (d, <i>J</i> = 21.7 Hz), 114.74 (d, <i>J</i> = 21.0 Hz), 102.80, 36.93, 34.18. HPLC Purity: 98%.</div></div><div id="sec4_16_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 5-(4-Chlorophenyl)-6-((3-fluorobenzyl)thio)-1-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15c</b>)</h4><div class="NLM_p last">Method C. White crystalline solid. 44% yield. HRMS (ESI): <i>m</i>/<i>z</i> 401.0629 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.02 (s, 1H), 7.49 (d, <i>J</i> = 8.31 Hz, 2H), 7.24â7.30 (m, 1H), 7.20 (d, <i>J</i> = 8.31 Hz, 2H), 7.13 (d, <i>J</i> = 7.34 Hz, 1H), 7.09 (d, <i>J</i> = 9.78 Hz, 1H), 6.96 (dt, <i>J</i> = 2.45, 8.31 Hz, 1H), 4.34 (s, 2H), 4.01 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.69 (d, <i>J</i> = 246.7 Hz), 160.96, 157.53, 150.81, 138.22 (d, <i>J</i> = 7.6 Hz), 136.33, 135.44, 133.89, 130.75, 130.17, 130.11, 124.76 (d, <i>J</i> = 2.9 Hz), 116.16 (d, <i>J</i> = 22.1 Hz), 114.73 (d, <i>J</i> = 21.0 Hz), 102.81, 36.92, 34.16. HPLC Purity: 99%.</div></div><div id="sec4_16_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 6-((3-Fluorobenzyl)thio)-5-(2-fluorophenyl)-1-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15d</b>)</h4><div class="NLM_p last">Method C. White crystalline solid. 70% yield. HRMS (ESI): <i>m</i>/<i>z</i> 385.0932 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.03 (s, 1H), 7.57â7.49 (m, 1H), 7.36â7.20 (m, 4H), 7.14 (d, <i>J</i> = 7.6 Hz, 1H), 7.10 (d, <i>J</i> = 9.5 Hz, 1H), 6.96 (td, <i>J</i> = 8.4, 2.5 Hz, 1H), 4.40 (d, <i>J</i> = 13.7 Hz, 1H), 4.34 (d, <i>J</i> = 13.6 Hz, 1H), 4.01 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.70 (d, <i>J</i> = 247.0 Hz), 161.09, 158.28 (d, <i>J</i> = 253.4 Hz), 156.94, 150.92, 138.20 (d, <i>J</i> = 7.6 Hz), 135.52, 132.33 (d, <i>J</i> = 8.0 Hz), 131.24, 130.12 (d, <i>J</i> = 8.5 Hz), 125.05 (d, <i>J</i> = 3.9 Hz), 124.75 (d, <i>J</i> = 3.1 Hz), 123.11 (d, <i>J</i> = 14.1 Hz), 117.03 (d, <i>J</i> = 19.5 Hz), 116.15 (d, <i>J</i> = 22.2 Hz), 114.71 (d, <i>J</i> = 21.0 Hz), 102.69, 36.73, 34.18. HPLC Purity: 98%.</div></div><div id="sec4_16_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 6-((3-Fluorobenzyl)thio)-5-(2-methoxyphenyl)-1-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15e</b>)</h4><div class="NLM_p last">Method C. White crystalline solid. 19% yield. HRMS (ESI): <i>m</i>/<i>z</i> 397.1127 [M + H]<sup>+</sup> (400 MHz, CDCl<sub>3</sub>) Î´ 8.02 (s, 1H), 7.49 (t, <i>J</i> = 8.02 Hz, 1H), 7.22â7.29 (m, 1H), 7.19 (d, <i>J</i> = 7.83 Hz, 1H), 7.02â7.16 (m, 4H), 6.94 (t, <i>J</i> = 8.22 Hz, 1H), 4.33 (s, 2H), 3.99 (s, 3H), 3.78 (s, 3H) <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 162.64 (d, <i>J</i> = 246.3 Hz), 161.76, 157.31, 155.40, 151.04, 138.76 (d, <i>J</i> = 7.6 Hz), 135.39, 131.80, 130.64, 129.92 (d, <i>J</i> = 8.4 Hz), 124.70 (d, <i>J</i> = 2.9 Hz), 123.89, 121.08, 116.00 (d, <i>J</i> = 21.9 Hz), 114.44 (d, <i>J</i> = 21.0 Hz), 112.34, 102.95, 55.82, 36.48, 34.06. HPLC Purity: 99%.</div></div><div id="sec4_16_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 6-((3-Fluorobenzyl)thio)-5-(3-methoxyphenyl)-1-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15f</b>)</h4><div class="NLM_p last">Method C. White Solid. 80% yield. HRMS (ESI): <i>m</i>/<i>z</i> 397.1125 [M + H]<sup>+Â 1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.99 (s, 1H), 7.41 (t, <i>J</i> = 8.22 Hz, 1H), 7.21â7.30 (m, 1H), 7.07â7.17 (m, 2H), 7.04 (dd, <i>J</i> = 2.35, 8.22 Hz, 1H), 6.90â6.98 (m, 1H), 6.85 (dd, <i>J</i> = 0.78, 7.83 Hz, 1H), 6.80 (d, <i>J</i> = 1.96 Hz, 1H), 4.33 (s, 2H), 3.99 (s, 3H), 3.80 (s, 3H) <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 162.55 (d, <i>J</i> = 246.6 Hz), 161.20, 160.39, 157.50, 150.73, 138.41 (d, <i>J</i> = 7.5 Hz), 136.38, 135.24, 130.29, 129.97 (d, <i>J</i> = 8.3 Hz), 124.72 (d, <i>J</i> = 3.0 Hz), 121.31, 116.06 (d, <i>J</i> = 21.8 Hz), 115.82, 114.83, 114.48 (d, <i>J</i> = 21.1 Hz), 102.79, 55.39, 36.74, 34.04. HPLC Purity: 97%.</div></div><div id="sec4_16_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 6-((3-Fluorobenzyl)thio)-5-(4-methoxyphenyl)-1-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15g</b>)</h4><div class="NLM_p last">Method C. White needles. 47% yield. HRMS (ESI): <i>m</i>/<i>z</i> 397.1123 [M + H]<sup>+Â 1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.01 (s, 1H), 7.22â7.30 (m, 1H), 7.07â7.19 (m, 4H), 6.98â7.04 (m, 2H), 6.95 (dt, <i>J</i> = 2.35, 8.41 Hz, 1H), 4.32 (s, 2H), 4.00 (s, 3H), 3.85 (s, 3H) <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 162.64 (d, <i>J</i> = 246.4 Hz), 161.92, 160.63, 157.91, 150.86, 138.52 (d, <i>J</i> = 7.4 Hz), 135.38, 130.40, 130.00, 127.75, 124.76 (d, <i>J</i> = 2.7 Hz), 116.16 (d, <i>J</i> = 22.6 Hz), 114.96, 114.56 (d, <i>J</i> = 21.0 Hz), 102.91, 55.45, 36.93, 34.14. HPLC Purity: 96%.</div></div><div id="sec4_16_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 6-((3-Fluorobenzyl)thio)-1-methyl-5-(pyridin-3-yl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15h</b>)</h4><div class="NLM_p last">Method C. White solid. 64% yield. HRMS (ESI): <i>m</i>/<i>z</i> 368.0972 [M + H]<sup>+</sup> (500 MHz, CDCl<sub>3</sub>) Î´ 8.72 (d, <i>J</i> = 4.40 Hz, 1H), 8.52 (s, 1H), 7.97 (s, 1H), 7.63 (d, <i>J</i> = 8.31 Hz, 1H), 7.45 (dd, <i>J</i> = 4.89, 7.83 Hz, 1H), 7.21â7.30 (m, 1H), 7.12 (d, <i>J</i> = 7.34 Hz, 1H), 7.07 (d, <i>J</i> = 9.78 Hz, 1H), 6.94 (t, <i>J</i> = 8.30 Hz, 1H), 4.37 (d, <i>J</i> = 14.18 Hz, 1H), 4.33 (d, <i>J</i> = 14.18 Hz, 1H), 4.00 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.54 (d, <i>J</i> = 246.9 Hz), 160.65, 157.29, 150.83, 150.64, 150.08, 137.96 (d, <i>J</i> = 7.6 Hz), 137.13, 135.25, 132.31, 130.05 (d, <i>J</i> = 8.2 Hz), 124.66 (d, <i>J</i> = 3.2 Hz), 124.07, 116.02 (d, <i>J</i> = 21.9 Hz), 114.64 (d, <i>J</i> = 21.0 Hz), 102.54, 36.80, 34.09. HPLC Purity: 99%.</div></div><div id="sec4_16_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 6-((3-Fluorobenzyl)thio)-1-methyl-5-(pyridin-4-yl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15i</b>)</h4><div class="NLM_p last">Method C. Off white solid. 62% yield. HRMS (ESI): <i>m</i>/<i>z</i> 368.0974 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.82 (d, <i>J</i> = 4.89 Hz, 2H), 8.00 (s, 1H), 7.26â7.32 (m, 1H), 7.25 (d, <i>J</i> = 5.38 Hz, 2H), 7.13 (d, <i>J</i> = 7.83 Hz, 1H), 7.09 (d, <i>J</i> = 9.78 Hz, 1H), 6.96 (dt, <i>J</i> = 2.45, 8.31 Hz, 1H), 4.37 (s, 2H), 4.01 (s, 3H) <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 162.48 (d, <i>J</i> = 246.8 Hz), 162.30, 159.50, 156.74, 151.49, 150.51, 143.30, 137.86 (d, <i>J</i> = 7.5 Hz), 135.19, 130.03 (d, <i>J</i> = 8.3 Hz), 124.62 (d, <i>J</i> = 3.0 Hz), 124.20, 115.96 (d, <i>J</i> = 22.1 Hz), 114.63 (d, <i>J</i> = 21.0 Hz), 102.45, 36.28, 34.06 (d, <i>J</i> = 4.9 Hz). HPLC Purity: 99%.</div></div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Method D. General Synthesis of Substituted Pyrazolothiopyrimidinones <b>15j</b>â<b>l</b> from <b>14</b></h3><div class="NLM_p">To a dry flask under N<sub>2</sub> charged with <b>14</b> (0.24 mmol) and 2 mL of dry DMF was added the appropriate amine (0.24 mmol) by syringe. The reaction was stirred at RT for 1 h and then cooled to 0 Â°C. 60 wt % NaH in mineral oil (0.71 mmol) was added, and the flask was stirred at 0 Â°C for 20 min then allowed to warm to RT and stirred for 3 h. The reaction was quenched with sat. aq. NH<sub>4</sub>Cl and extracted two times with ethyl acetate. The combined organics were dried with sodium sulfate and the solvent removed. The residue was dissolved in 2 mL of DMF and 3-fluorobenzyl bromide (0.03 mL, 0.24 mmol) and sodium bicarbonate (80 mg, 0.95 mmol) were added. The mixture was stirred at RT overnight at which point the reaction was complete by HPLC. The mixture was diluted with water and extracted two times with EtOAc. The combined organics were washed three times with brine, dried over sodium sulfate, and the solvent removed. The crude product was purified by flash chromatography (0â100% EtOAc in Hex) yielding the titled compounds.</div><div id="sec4_17_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 6-((3-Fluorobenzyl)thio)-1-methyl-5-(pyridin-2-yl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15j</b>)</h4><div class="NLM_p last">Method D. White needles. 40% yield. HRMS (ESI): <i>m</i>/<i>z</i> 368.0977 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.69 (dd, <i>J</i> = 1.47, 4.89 Hz, 1H), 8.03 (s, 1H), 7.92 (dt, <i>J</i> = 1.96, 7.58 Hz, 1H), 7.44â7.49 (m, 1H), 7.38 (d, <i>J</i> = 7.83 Hz, 1H), 7.21â7.29 (m, 1H), 7.13 (d, <i>J</i> = 7.83 Hz, 1H), 7.06â7.11 (m, 1H), 6.95 (dt, <i>J</i> = 2.20, 8.44 Hz, 1H), 4.38 (s, 2H), 4.01 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.70 (d, <i>J</i> = 246.7 Hz), 160.19, 157.59, 150.88, 150.26, 149.38, 138.83, 138.17 (d, <i>J</i> = 7.6 Hz), 135.52, 130.10 (d, <i>J</i> = 8.5 Hz), 125.06, 124.86, 124.81 (d, <i>J</i> = 3.1 Hz), 116.19 (d, <i>J</i> = 22.0 Hz), 114.70 (d, <i>J</i> = 21.4 Hz), 102.96, 36.66, 34.18. HPLC Purity: 98%.</div></div><div id="sec4_17_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 5-Benzyl-6-((3-fluorobenzyl)thio)-1-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15k</b>)</h4><div class="NLM_p last">Method D. White crystalline solid. 57% yield. HRMS (ESI): <i>m</i>/<i>z</i> 381.1180 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.02 (s, 1H), 7.22â7.35 (m, 6H), 7.16 (d, <i>J</i> = 7.83 Hz, 1H), 7.11 (d, <i>J</i> = 9.78 Hz, 1H), 6.97 (t, <i>J</i> = 8.31 Hz, 1H), 5.36 (br. s., 2H), 4.43 (s, 2H), 3.96 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.69 (d, <i>J</i> = 246.9 Hz), 160.36, 157.77, 150.44, 138.33 (d, <i>J</i> = 7.5 Hz), 135.33, 135.20, 130.09 (d, <i>J</i> = 8.3 Hz), 128.57, 127.67, 127.28, 124.70 (d, <i>J</i> = 3.1 Hz), 116.09 (d, <i>J</i> = 22.0 Hz), 114.70 (d, <i>J</i> = 21.0 Hz), 102.58, 46.60, 36.51, 33.97. HPLC Purity: 99%.</div></div><div id="sec4_17_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 6-((3-Fluorobenzyl)thio)-1-methyl-5-phenethyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15l</b>)</h4><div class="NLM_p last">Method D. White Needles. 72% yield. HRMS (ESI): <i>m</i>/<i>z</i> 395.1341 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.00 (s, 1H), 7.27â7.36 (m, 5H), 7.23 (t, <i>J</i> = 7.58 Hz, 2H), 7.17 (d, <i>J</i> = 9.78 Hz, 1H), 7.00 (dt, <i>J</i> = 2.20, 8.44 Hz, 1H), 4.48 (s, 2H), 4.24â4.31 (m, 2H), 3.96 (s, 3H), 2.97â3.04 (m, 2H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.76 (d, <i>J</i> = 246.8 Hz), 159.65, 157.37, 150.47, 138.42 (d, <i>J</i> = 7.6 Hz), 137.64, 135.00, 130.18 (d, <i>J</i> = 8.3 Hz), 128.90, 128.65, 126.81, 124.76 (d, <i>J</i> = 2.9 Hz), 116.19 (d, <i>J</i> = 22.0 Hz), 114.78 (d, <i>J</i> = 21.2 Hz), 102.75, 45.68, 36.34, 34.23, 33.98. HPLC Purity: 99%.</div></div><div id="sec4_17_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 6-((3-Fluorobenzyl)thio)-1,5-dimethyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4(5<i>H</i>)-one (<b>15m</b>)</h4><div class="NLM_p last">Compound <b>14</b> (100 mg, 0.47 mmol) was dissolved in 1 mL of 8N methylamine in ethanol and stirred at RT for 1 h. The volatiles were removed, and the residue was dissolved in 1 mL of DMF to which sodium bicarbonate (80 mg, 0.947 mmol) and 3-fluorobenzyl bromide (70 Î¼L, 0.57 mmol) were added. The reaction was stirred for 1 h at which point the product was precipitated out by the addition of water. The precipitate was collected by filtration and washed with water and then hexanes. The precipitate was then taken up in dichloromethane and dried with sodium sulfate. The solvent was removed to yield the titled compound as a white crystalline solid (122 mg, 0.40 mmol, 85% yield). HRMS (ESI): <i>m</i>/<i>z</i> 305.0864 [M + H]<sup>+Â 1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 7.99 (s, 1H), 7.29â7.35 (m, 1H), 7.24 (d, <i>J</i> = 7.83 Hz, 1H), 7.19 (d, <i>J</i> = 9.78 Hz, 1H), 7.00 (dt, <i>J</i> = 2.45, 8.56 Hz, 1H), 4.49 (s, 2H), 3.96 (s, 3H), 3.55 (s, 3H) <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 162.74 (d, <i>J</i> = 246.7 Hz), 160.32, 157.62, 150.46, 138.42 (d, <i>J</i> = 7.5 Hz), 134.95, 130.17 (d, <i>J</i> = 8.2 Hz), 124.75 (d, <i>J</i> = 2.9 Hz), 116.17 (d, <i>J</i> = 22.0 Hz), 114.77 (d, <i>J</i> = 21.0 Hz), 102.44, 36.28, 33.95, 29.70. HPLC Purity: 99%.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00930" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00876" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00876" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00930" class="ext-link">10.1021/acs.jmedchem.8b00930</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional information regarding the reversible inhibition assay, inhibition of the N121S mutant used for crystallography, crystallographic data collection and refinement, electron density maps, validation of ALDH1A2 and 1A3 homology models, additional docking studies, and additional cytotoxicity data, and ALDH expression for cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx" class="ext-link">DOCX</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Coordinates for ALDH1A2 and ALDH1A3 homology models and docking studies (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_019.zip" class="ext-link">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx">jm8b00930_si_001.docx (2.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_002.csv">jm8b00930_si_002.csv (3.03 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_019.zip">jm8b00930_si_019.zip (2.38 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates of ALDH1A1 bound to compounds <b>7</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI">5TEI</a>), <b>13g</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WDV">5WDV</a>) have been deposited with the Protein Data Bank. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b00930" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54887" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54887" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ronald J. Buckanovich</span> - <span class="hlFld-Affiliation affiliation">Division of Hematology-Oncology, Department of Internal Medicine,Â 
    , , University
of Pittsburgh Medical Center and the Magee Womens Research Institute, Pittsburgh Pennsylvania 15213, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Division
of Hematology Oncology, Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#096b7c6a626867667f606a617b6349647e7b602764686e6c6c276c6d7c"><span class="__cf_email__" data-cfemail="3b594e58505a55544d52585349517b564c495215565a5c5e5e155e5f4e">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott D. Larsen</span> - <span class="hlFld-Affiliation affiliation">Vahlteich
Medicinal Chemistry Core,Â 
    Department of Medicinal Chemistry,
College of Pharmacy,Â 
     and , University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8440-4990" title="Orcid link">http://orcid.org/0000-0001-8440-4990</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c9baada5a8bbbaaca789a4acade7bca4a0aaa1e7acadbc"><span class="__cf_email__" data-cfemail="d7a4b3bbb6a5a4b2b997bab2b3f9a2babeb4bff9b2b3a2">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brandt
C. Huddle</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry,
College of Pharmacy and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edward Grimley</span> - <span class="hlFld-Affiliation affiliation">Department of Obstetrics,
Gynecology, and Reproductive Sciences, , University
of Pittsburgh Medical Center and the Magee Womens Research Institute, Pittsburgh Pennsylvania 15213, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cameron D. Buchman</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mikhail Chtcherbinine</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bikash Debnath</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry,
College of Pharmacy and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pooja Mehta</span> - <span class="hlFld-Affiliation affiliation">Department
of Materials Science Engineering, University
of Michigan, Ann Arbor, Michigan 48109, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kun Yang</span> - <span class="hlFld-Affiliation affiliation">Division
of Hematology Oncology, Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cynthia A. Morgan</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Siwei Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, College of Pharmacy; University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeremy Felton</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, College of Pharmacy; University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Duxin Sun</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, College of Pharmacy; University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Geeta Mehta</span> - <span class="hlFld-Affiliation affiliation">Department
of Materials Science Engineering, University
of Michigan, Ann Arbor, Michigan 48109, United
States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, University of
Michigan, Ann Arbor, Michigan 48109, United
States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Macromolecular
Science and Engineering, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nouri Neamati</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry,
College of Pharmacy and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas D. Hurley</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, Indianapolis, Indiana 46202, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>B.C.H. and E.G. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i77">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Results shown in this report are derived from work performed at Argonne National Laboratory, Structural Biology Center at the Advanced Photon Source. Argonne is operated by UChicago Argonne, LLC, for the U.S. Department of Energy, Office of Biological and Environmental Research under contract DE-AC02-06CH11357. This work was supported by grants from the Center for the Discovery of New Medicines, University of Michigan (to R.J.B.), the Michigan Ovarian Cancer Alliance (to G.M.), the Rivkin Center for Ovarian Cancer (to G.M.), NIH R01 CA214567-01 (to S.D.L. and R.J.B.), and DOD OCRP Early Career Investigator Award W81XWH-13-1-0134 (to G.M.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS USED</td></tr><tr><td class="NLM_term">ALDH</td><td class="NLM_def"><p class="first last">aldehyde dehydrogenase</p></td></tr><tr><td class="NLM_term">ATRA</td><td class="NLM_def"><p class="first last">all-trans retinoic acid</p></td></tr><tr><td class="NLM_term">CI</td><td class="NLM_def"><p class="first last">combination index</p></td></tr><tr><td class="NLM_term">CSC</td><td class="NLM_def"><p class="first last">cancer stem cell</p></td></tr><tr><td class="NLM_term">DEAB</td><td class="NLM_def"><p class="first last">diethylamino benzoic acid</p></td></tr><tr><td class="NLM_term">EOC</td><td class="NLM_def"><p class="first last">epithelial ovarian cancer</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">FACS</td><td class="NLM_def"><p class="first last">fluorescence-activated cell sorting</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">Hex</td><td class="NLM_def"><p class="first last">Hexanes</p></td></tr><tr><td class="NLM_term">NCS</td><td class="NLM_def"><p class="first last">isothiocyanate</p></td></tr><tr><td class="NLM_term">MLM</td><td class="NLM_def"><p class="first last">mouse liver microsomes</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">SFM</td><td class="NLM_def"><p class="first last">serum free medium</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76142" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76142" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 75 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span> <i>Cancer Facts & Figures</i>; <span class="NLM_publisher-name">American Cancer
Society</span>: <span class="NLM_publisher-loc">Atlanta, GA</span>, <span class="NLM_year">2015</span>. See the following: <a href="https://www.cancer.org/content/dam/cancer-org/research/cancer-facts-and-statistics/annual-cancer-facts-and-figures/2015/cancer-facts-and-figures-2015.pdf" class="extLink">https://www.cancer.org/content/dam/cancer-org/research/cancer-facts-and-statistics/annual-cancer-facts-and-figures/2015/cancer-facts-and-figures-2015.pdf</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Cancer+Facts+%26+Figures%3B+American+Cancer%0ASociety%3A+Atlanta%2C+GA%2C+2015.+See+the+following%3A+https%3A%2F%2Fwww.cancer.org%2Fcontent%2Fdam%2Fcancer-org%2Fresearch%2Fcancer-facts-and-statistics%2Fannual-cancer-facts-and-figures%2F2015%2Fcancer-facts-and-figures-2015.pdf."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Facts%2520%2526%2520Figures%26pub%3DAmerican%2520Cancer%250ASociety%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozols, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bundy, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greer, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke-Pearson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burger, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannel, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGeest, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartenbach, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baergen, R.</span></span> <span> </span><span class="NLM_article-title">Phase III Trial of Carboplatin and Paclitaxel Compared With Cisplatin and Paclitaxel in Patients With Optimally Resected Stage III Ovarian Cancer: A Gynecologic Oncology Group Study</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">3194</span>â <span class="NLM_lpage">3200</span>, <span class="refDoi">Â DOI: 10.1200/JCO.2003.02.153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1200%2FJCO.2003.02.153" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=12860964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD2cXps1aktbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2003&pages=3194-3200&issue=17&author=R.+F.+Ozolsauthor=B.+N.+Bundyauthor=B.+E.+Greerauthor=J.+M.+Fowlerauthor=D.+Clarke-Pearsonauthor=R.+A.+Burgerauthor=R.+S.+Mannelauthor=K.+DeGeestauthor=E.+M.+Hartenbachauthor=R.+Baergen&title=Phase+III+Trial+of+Carboplatin+and+Paclitaxel+Compared+With+Cisplatin+and+Paclitaxel+in+Patients+With+Optimally+Resected+Stage+III+Ovarian+Cancer%3A+A+Gynecologic+Oncology+Group+Study&doi=10.1200%2FJCO.2003.02.153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Phase III trial of carboplatin and paclitaxel compared with cisplatin and paclitaxel in patients with optimally resected stage III ovarian cancer: a Gynecologic Oncology Group Study</span></div><div class="casAuthors">Ozols, Robert F.; Bundy, Brian N.; Greer, Benjamin E.; Fowler, Jeffrey M.; Clarke-Pearson, Daniel; Burger, Robert A.; Mannel, Robert S.; de Geest, Koen; Hartenbach, Ellen M.; Baergen, Rebecca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3194-3200</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">In randomized trials the combination of cisplatin and paclitaxel was superior to cisplatin and cyclophosphamide in advanced-stage epithelial ovarian cancer.  Although in nonrandomized trials, carboplatin and paclitaxel was a less toxic and highly active combination regimen, there remained concern regarding its efficacy in patients with small-vol., resected, stage III disease.  Thus, we conducted a noninferiority trial of cisplatin and paclitaxel vs. carboplatin and paclitaxel in this population.  Patients with advanced ovarian cancer and no residual mass greater than 1.0 cm after surgery were randomly assigned to receive cisplatin 75 mg/m2 plus a 24-h infusion of paclitaxel 135 mg/m2 (arm I), or carboplatin area under the curve 7.5 i.v. plus paclitaxel 175 mg/m2 over 3 h (arm II).  Seven hundred ninety-two eligible patients were enrolled onto the study.  Prognostic factors were similar in the two treatment groups.  Gastrointestinal, renal, and metabolic toxicity, as well as grade 4 leukopenia, were significantly more frequent in arm I.  Grade 2 or greater thrombocytopenia was more common in arm II.  Neurol. toxicity was similar in both regimens.  Median progression-free survival and overall survival were 19.4 and 48.7 mo, resp., for arm I compared with 20.7 and 57.4 mo, resp., for arm II.  The relative risk (RR) of progression for the carboplatin plus paclitaxel group was 0.88 (95% confidence interval [CI], 0.75 to 1.03) and the RR of death was 0.84 (95% CI, 0.70 to 1.02).  In patients with advanced ovarian cancer, a chemotherapy regimen consisting of carboplatin plus paclitaxel results in less toxicity, is easier to administer, and is not inferior, when compared with cisplatin plus paclitaxel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob190CVtYZirVg90H21EOLACvtfcHk0lim2eRN8N19Zg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXps1aktbk%253D&md5=83a519c6c3759f8ac7e8fc86444f546f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1200%2FJCO.2003.02.153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2003.02.153%26sid%3Dliteratum%253Aachs%26aulast%3DOzols%26aufirst%3DR.%2BF.%26aulast%3DBundy%26aufirst%3DB.%2BN.%26aulast%3DGreer%26aufirst%3DB.%2BE.%26aulast%3DFowler%26aufirst%3DJ.%2BM.%26aulast%3DClarke-Pearson%26aufirst%3DD.%26aulast%3DBurger%26aufirst%3DR.%2BA.%26aulast%3DMannel%26aufirst%3DR.%2BS.%26aulast%3DDeGeest%26aufirst%3DK.%26aulast%3DHartenbach%26aufirst%3DE.%2BM.%26aulast%3DBaergen%26aufirst%3DR.%26atitle%3DPhase%2520III%2520Trial%2520of%2520Carboplatin%2520and%2520Paclitaxel%2520Compared%2520With%2520Cisplatin%2520and%2520Paclitaxel%2520in%2520Patients%2520With%2520Optimally%2520Resected%2520Stage%2520III%2520Ovarian%2520Cancer%253A%2520A%2520Gynecologic%2520Oncology%2520Group%2520Study%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2003%26volume%3D21%26issue%3D17%26spage%3D3194%26epage%3D3200%26doi%3D10.1200%2FJCO.2003.02.153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thigpen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">duBois, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlpine, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiSaia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoskins, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristensen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfisterer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swart, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallardo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poveda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinstein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitchener, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, G. C.</span></span> <span> </span><span class="NLM_article-title">First-line Therapy in Ovarian Cancer Trials</span>. <i>Int. J. Gynecol Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">756</span>â <span class="NLM_lpage">762</span>, <span class="refDoi">Â DOI: 10.1097/IGC.0b013e31821ce75d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1097%2FIGC.0b013e31821ce75d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21543937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A280%3ADC%252BC3MnitVWltg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=756-762&issue=4&author=T.+Thigpenauthor=A.+duBoisauthor=J.+McAlpineauthor=P.+DiSaiaauthor=K.+Fujiwaraauthor=W.+Hoskinsauthor=G.+Kristensenauthor=R.+Mannelauthor=M.+Markmanauthor=J.+Pfistererauthor=M.+Quinnauthor=N.+Reedauthor=A.+M.+Swartauthor=J.+Berekauthor=N.+Colomboauthor=G.+Freyerauthor=D.+Gallardoauthor=M.+Planteauthor=A.+Povedaauthor=L.+Rubinsteinauthor=M.+Baconauthor=H.+Kitchenerauthor=G.+C.+Stuart&title=First-line+Therapy+in+Ovarian+Cancer+Trials&doi=10.1097%2FIGC.0b013e31821ce75d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">First-line therapy in ovarian cancer trials</span></div><div class="casAuthors">Thigpen Tate; duBois Andreas; McAlpine Jessica; DiSaia Philip; Fujiwara Keiichi; Hoskins William; Kristensen Gunnar; Mannel Robert; Markman Maurie; Pfisterer Jacobus; Quinn Michael; Reed Nick; Swart Ann Marie; Berek Jonathan; Colombo Nicoletta; Freyer Gilles; Gallardo Dolores; Plante Marie; Poveda Andres; Rubinstein Lawrence; Bacon Monica; Kitchener Henry; Stuart Gavin C E</div><div class="citationInfo"><span class="NLM_cas:title">International journal of gynecological cancer : official journal of the International Gynecological Cancer Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">756-62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">At the 4th Ovarian Cancer Consensus Conference of the Gynecologic Cancer InterGroup (GCIG) held in Vancouver, Canada, in June 2010, representatives of 23 cooperative research groups studying gynecologic cancers gathered to establish international consensus on issues critical to the conduct of large randomized trials.  The process focused on 13 predetermined questions.  Group A, 1 of the 3 discussion groups, addressed the first 5 questions, examining first-line therapies in newly diagnosed ovarian cancer patients.  A1: What are the appropriate end points for different trials (maintenance, upfront chemotherapy trials including molecular drugs)? A2: Are there any subgroups defined by tumor biology who need specific treatment options/trials? A3: Is the 2004 GCIG-recommended standard comparator arm still valid? A4: What is the role of modifying dose, schedule, and delivery of chemotherapy? A5: What role does surgery play today?</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZIyqqc7wuZub6AVA-44PTfW6udTcc2eYBq66RdRRBGbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MnitVWltg%253D%253D&md5=e6cb9d1cd93b8099f3e7430b803c1fb1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1097%2FIGC.0b013e31821ce75d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FIGC.0b013e31821ce75d%26sid%3Dliteratum%253Aachs%26aulast%3DThigpen%26aufirst%3DT.%26aulast%3DduBois%26aufirst%3DA.%26aulast%3DMcAlpine%26aufirst%3DJ.%26aulast%3DDiSaia%26aufirst%3DP.%26aulast%3DFujiwara%26aufirst%3DK.%26aulast%3DHoskins%26aufirst%3DW.%26aulast%3DKristensen%26aufirst%3DG.%26aulast%3DMannel%26aufirst%3DR.%26aulast%3DMarkman%26aufirst%3DM.%26aulast%3DPfisterer%26aufirst%3DJ.%26aulast%3DQuinn%26aufirst%3DM.%26aulast%3DReed%26aufirst%3DN.%26aulast%3DSwart%26aufirst%3DA.%2BM.%26aulast%3DBerek%26aufirst%3DJ.%26aulast%3DColombo%26aufirst%3DN.%26aulast%3DFreyer%26aufirst%3DG.%26aulast%3DGallardo%26aufirst%3DD.%26aulast%3DPlante%26aufirst%3DM.%26aulast%3DPoveda%26aufirst%3DA.%26aulast%3DRubinstein%26aufirst%3DL.%26aulast%3DBacon%26aufirst%3DM.%26aulast%3DKitchener%26aufirst%3DH.%26aulast%3DStuart%26aufirst%3DG.%2BC.%26atitle%3DFirst-line%2520Therapy%2520in%2520Ovarian%2520Cancer%2520Trials%26jtitle%3DInt.%2520J.%2520Gynecol%2520Cancer%26date%3D2011%26volume%3D21%26issue%3D4%26spage%3D756%26epage%3D762%26doi%3D10.1097%2FIGC.0b013e31821ce75d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cannistra, S. A.</span></span> <span> </span><span class="NLM_article-title">Cancer of the Ovary</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>351</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">2519</span>â <span class="NLM_lpage">2529</span>, <span class="refDoi">Â DOI: 10.1056/NEJMra041842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1056%2FNEJMra041842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=15590954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGksbbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2004&pages=2519-2529&issue=24&author=S.+A.+Cannistra&title=Cancer+of+the+Ovary&doi=10.1056%2FNEJMra041842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer of the ovary</span></div><div class="casAuthors">Cannistra, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2519-2529</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Epithelial cancer of the ovary is a relatively uncommon gynecol. cancer in the United States, with approx. 25,580 new cases and 16,090 deaths anticipated in 2004.  Most patients present with advanced disease, which is managed with surgical resection followed by platinum-based chemotherapy.  During the past decade, advances in chemotherapy have resulted in improved survival and in more effective treatment of relapsed disease.  In addn., a better understanding of genetic risk factors has permitted a tailored approach to preventive strategies, such as bilateral salpingo-oophorectomy in selected women.  This review describes the clin. features of epithelial ovarian cancer, with an emphasis on recent advances in postoperative management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSCl_VGPoSdLVg90H21EOLACvtfcHk0ljwTV0dOb41yg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGksbbM&md5=e9f22d658bf6281d6e99ce86c192ac5c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1056%2FNEJMra041842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra041842%26sid%3Dliteratum%253Aachs%26aulast%3DCannistra%26aufirst%3DS.%2BA.%26atitle%3DCancer%2520of%2520the%2520Ovary%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2004%26volume%3D351%26issue%3D24%26spage%3D2519%26epage%3D2529%26doi%3D10.1056%2FNEJMra041842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landen, C. N.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katre, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steg, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nick, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mejia, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershenson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span> <span> </span><span class="NLM_article-title">Targeting Aldehyde Dehydrogenase Cancer Stem Cells in Ovarian Cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3186</span>â <span class="NLM_lpage">3199</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.MCT-10-0563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F1535-7163.MCT-10-0563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=20889728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsL%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=3186-3199&issue=12&author=C.+N.+Landenauthor=B.+Goodmanauthor=A.+A.+Katreauthor=A.+D.+Stegauthor=A.+M.+Nickauthor=R.+L.+Stoneauthor=L.+D.+Millerauthor=P.+V.+Mejiaauthor=N.+B.+Jenningsauthor=D.+M.+Gershensonauthor=R.+C.+Bastauthor=R.+L.+Colemanauthor=G.+Lopez-Beresteinauthor=A.+K.+Sood&title=Targeting+Aldehyde+Dehydrogenase+Cancer+Stem+Cells+in+Ovarian+Cancer&doi=10.1158%2F1535-7163.MCT-10-0563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Aldehyde Dehydrogenase Cancer Stem Cells in Ovarian Cancer</span></div><div class="casAuthors">Landen, Charles N., Jr.; Goodman, Blake; Katre, Ashwini A.; Steg, Adam D.; Nick, Alpa M.; Stone, Rebecca L.; Miller, Lance D.; Mejia, Pablo Vivas; Jennings, Nicolas B.; Gershenson, David M.; Bast, Robert C., Jr.; Coleman, Robert L.; Lopez-Berestein, Gabriel; Sood, Anil K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3186-3199</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenase-1A1 (ALDH1A1) expression characterizes a subpopulation of cells with tumor-initiating or cancer stem cell properties in several malignancies.  Our goal was to characterize the phenotype of ALDH1A1-pos. ovarian cancer cells and examine the biol. effects of ALDH1A1 gene silencing.  In our anal. of multiple ovarian cancer cell lines, we found that ALDH1A1 expression and activity was significantly higher in taxane- and platinum-resistant cell lines.  In patient samples, 72.9% of ovarian cancers had ALDH1A1 expression in which the percentage of ALDH1A1-pos. cells correlated neg. with progression-free survival (6.05 vs. 13.81 mo; P < 0.035).  Subpopulations of A2780cp20 cells with ALDH1A1 activity were isolated for orthotopic tumor-initiating studies, where tumorigenicity was approx. 50-fold higher with ALDH1A1-pos. cells.  Interestingly, tumors derived from ALDH1A1-pos. cells gave rise to both ALDH1A1-pos. and ALDH1A1-neg. populations, but ALDH1A1-neg. cells could not generate ALDH1A1-pos. cells.  In an in vivo orthotopic mouse model of ovarian cancer, ALDH1A1 silencing using nanoliposomal siRNA sensitized both taxane- and platinum-resistant cell lines to chemotherapy, significantly reducing tumor growth in mice compared with chemotherapy alone (a 74%-90% redn.; P < 0.015).  These data show that the ALDH1A1 subpopulation is assocd. with chemoresistance and outcome in ovarian cancer patients, and targeting ALDH1A1 sensitizes resistant cells to chemotherapy.  ALDH1A1-pos. cells have enhanced, but not abs., tumorigenicity but do have differentiation capacity lacking in ALDH1A1-neg. cells.  This enzyme may be important for identification and targeting of chemoresistant cell populations in ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeKpHNJYkfVLVg90H21EOLACvtfcHk0ljwTV0dOb41yg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsL%252FP&md5=2ddb105fb819e82928e7c95994ee317e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-10-0563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-10-0563%26sid%3Dliteratum%253Aachs%26aulast%3DLanden%26aufirst%3DC.%2BN.%26aulast%3DGoodman%26aufirst%3DB.%26aulast%3DKatre%26aufirst%3DA.%2BA.%26aulast%3DSteg%26aufirst%3DA.%2BD.%26aulast%3DNick%26aufirst%3DA.%2BM.%26aulast%3DStone%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DL.%2BD.%26aulast%3DMejia%26aufirst%3DP.%2BV.%26aulast%3DJennings%26aufirst%3DN.%2BB.%26aulast%3DGershenson%26aufirst%3DD.%2BM.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DColeman%26aufirst%3DR.%2BL.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DSood%26aufirst%3DA.%2BK.%26atitle%3DTargeting%2520Aldehyde%2520Dehydrogenase%2520Cancer%2520Stem%2520Cells%2520in%2520Ovarian%2520Cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26issue%3D12%26spage%3D3186%26epage%3D3199%26doi%3D10.1158%2F1535-7163.MCT-10-0563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steg, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevis, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katre, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobbin, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landen, C. N.</span></span> <span> </span><span class="NLM_article-title">Stem Cell Pathways Contribute to Clinical Chemoresistance in Ovarian Cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">869</span>â <span class="NLM_lpage">881</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-11-2188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F1078-0432.CCR-11-2188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22142828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOku7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=869-881&author=A.+D.+Stegauthor=K.+S.+Bevisauthor=A.+A.+Katreauthor=A.+Ziebarthauthor=Z.+C.+Dobbinauthor=R.+D.+Alvarezauthor=K.+Zhangauthor=M.+Connerauthor=C.+N.+Landen&title=Stem+Cell+Pathways+Contribute+to+Clinical+Chemoresistance+in+Ovarian+Cancer&doi=10.1158%2F1078-0432.CCR-11-2188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Stem Cell Pathways Contribute to Clinical Chemoresistance in Ovarian Cancer</span></div><div class="casAuthors">Steg, Adam D.; Bevis, Kerri S.; Katre, Ashwini A.; Ziebarth, Angela; Dobbin, Zachary C.; Alvarez, Ronald D.; Zhang, Kui; Conner, Michael; Landen, Charles N.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">869-881</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Within heterogeneous tumors, subpopulations often labeled cancer stem cells (CSC) have been identified that have enhanced tumorigenicity and chemoresistance in ex vivo models.  However, whether these populations are more capable of surviving chemotherapy in de novo tumors is unknown.  Exptl. Design: We examd. 45 matched primary/recurrent tumor pairs of high-grade ovarian adenocarcinomas for expression of CSC markers ALDH1A1, CD44, and CD133 using immunohistochem.  Tumors collected immediately after completion of primary therapy were then laser capture microdissected and subjected to a quant. PCR array examg. stem cell biol. pathways (Hedgehog, Notch, TGF-Î², and Wnt).  Select genes of interest were validated as important targets using siRNA-mediated downregulation.  RESULTS: Primary samples were composed of low densities of ALDH1A1, CD44, and CD133.  Tumors collected immediately after primary therapy were more densely composed of each marker, whereas samples collected at first recurrence, before initiating secondary therapy, were composed of similar percentages of each marker as their primary tumor.  In tumors collected from recurrent platinum-resistant patients, only CD133 was significantly increased.  Of stem cell pathway members examd., 14% were significantly overexpressed in recurrent compared with matched primary tumors.  Knockdown of genes of interest, including endoglin/CD105 and the hedgehog mediators Gli1 and Gli2, led to decreased ovarian cancer cell viability, with Gli2 showing a novel contribution to cisplatin resistance.  CONCLUSIONS: These data indicate that ovarian tumors are enriched with CSCs and stem cell pathway mediators, esp. at the completion of primary therapy.  This suggests that stem cell subpopulations contribute to tumor chemoresistance and ultimately recurrent disease.  Clin Cancer Res; 18(3); 869-81.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvmyFTyD3DorVg90H21EOLACvtfcHk0lhLWuxx_AYJ2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOku7k%253D&md5=3dd7be6381c376fa6caacafb86b8c85a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-2188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-2188%26sid%3Dliteratum%253Aachs%26aulast%3DSteg%26aufirst%3DA.%2BD.%26aulast%3DBevis%26aufirst%3DK.%2BS.%26aulast%3DKatre%26aufirst%3DA.%2BA.%26aulast%3DZiebarth%26aufirst%3DA.%26aulast%3DDobbin%26aufirst%3DZ.%2BC.%26aulast%3DAlvarez%26aufirst%3DR.%2BD.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DConner%26aufirst%3DM.%26aulast%3DLanden%26aufirst%3DC.%2BN.%26atitle%3DStem%2520Cell%2520Pathways%2520Contribute%2520to%2520Clinical%2520Chemoresistance%2520in%2520Ovarian%2520Cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D869%26epage%3D881%26doi%3D10.1158%2F1078-0432.CCR-11-2188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silva, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginestier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kueck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicha, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckanovich, R. J.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenase in Combination with CD133 Defines Angiogenic Ovarian Cancer Stem Cells that Portend Poor Patient Survival</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3991</span>â <span class="NLM_lpage">4001</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-10-3175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F0008-5472.CAN-10-3175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21498635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFWksrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=3991-4001&issue=11&author=I.+A.+Silvaauthor=S.+Baiauthor=K.+McLeanauthor=K.+Yangauthor=K.+Griffithauthor=D.+Thomasauthor=C.+Ginestierauthor=C.+Johnstonauthor=A.+Kueckauthor=R.+K.+Reynoldsauthor=M.+S.+Wichaauthor=R.+J.+Buckanovich&title=Aldehyde+Dehydrogenase+in+Combination+with+CD133+Defines+Angiogenic+Ovarian+Cancer+Stem+Cells+that+Portend+Poor+Patient+Survival&doi=10.1158%2F0008-5472.CAN-10-3175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde Dehydrogenase in Combination with CD133 Defines Angiogenic Ovarian Cancer Stem Cells That Portend Poor Patient Survival</span></div><div class="casAuthors">Silva, Ines A.; Bai, Shoumei; McLean, Karen; Yang, Kun; Griffith, Kent; Thomas, Dafydd; Ginestier, Christophe; Johnston, Carolyn; Kueck, Angela; Reynolds, R. Kevin; Wicha, Max S.; Buckanovich, Ronald J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3991-4001</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Markers that reliably identify cancer stem cells (CSC) in ovarian cancer could assist prognosis and improve strategies for therapy.  CD133 is a reported marker of ovarian CSC.  Aldehyde dehydrogenase (ALDH) activity is a reported CSC marker in several solid tumors, but it was not studied in ovarian CSC.  Here we report that dual positivity of CD133 and ALDH defines a compelling marker set in ovarian CSC.  All human ovarian tumors and cell lines displayed ALDH activity.  ALDH+ cells isolated from ovarian cancer cell lines were chemoresistant and preferentially grew tumors, compared with ALDH- cells, validating ALDH as a marker of ovarian CSC in cell lines.  Notably, as few as 1,000 ALDH+ cells isolated directly from CD133- human ovarian tumors were sufficient to generate tumors in immunocompromised mice, whereas 50,000 ALDH- cells were unable to initiate tumors.  Using ALDH in combination with CD133 to analyze ovarian cancer cell lines, we obsd. even greater growth in the ALDH+CD133+ cells compared with ALDH+CD133- cells, suggesting a further enrichment of ovarian CSC in ALDH+CD133+ cells.  Strikingly, as few as 11 ALDH+CD133+ cells isolated directly from human tumors were sufficient to initiate tumors in mice.  Like other CSC, ovarian CSC exhibited increased angiogenic capacity compared with bulk tumor cells.  Finally, the presence of ALDH+CD133+ cells in debulked primary tumor specimens correlated with reduced disease-free and overall survival in ovarian cancer patients.  Taken together,these findings define ALDH and CD133 as a functionally significant set of markers to identify ovarian CSCs.  Cancer Res; 71(11); 3991-4001.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy2Tirx6N8kbVg90H21EOLACvtfcHk0lhLWuxx_AYJ2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFWksrs%253D&md5=b5719c401bcb6ed36e2d831f0356cf5a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-3175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-3175%26sid%3Dliteratum%253Aachs%26aulast%3DSilva%26aufirst%3DI.%2BA.%26aulast%3DBai%26aufirst%3DS.%26aulast%3DMcLean%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DGriffith%26aufirst%3DK.%26aulast%3DThomas%26aufirst%3DD.%26aulast%3DGinestier%26aufirst%3DC.%26aulast%3DJohnston%26aufirst%3DC.%26aulast%3DKueck%26aufirst%3DA.%26aulast%3DReynolds%26aufirst%3DR.%2BK.%26aulast%3DWicha%26aufirst%3DM.%2BS.%26aulast%3DBuckanovich%26aufirst%3DR.%2BJ.%26atitle%3DAldehyde%2520Dehydrogenase%2520in%2520Combination%2520with%2520CD133%2520Defines%2520Angiogenic%2520Ovarian%2520Cancer%2520Stem%2520Cells%2520that%2520Portend%2520Poor%2520Patient%2520Survival%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26issue%3D11%26spage%3D3991%26epage%3D4001%26doi%3D10.1158%2F0008-5472.CAN-10-3175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingram, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyengar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckanovich, R. J.</span></span> <span> </span><span class="NLM_article-title">Identifying an Ovarian Cancer Cell Hierarchy Regulated by Bone Morphogenetic Protein 2</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span> (<span class="NLM_issue">50</span>),  <span class="NLM_fpage">E6882</span>â <span class="NLM_lpage">E6888</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1507899112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1073%2Fpnas.1507899112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E6882-E6888&issue=50&author=Y.-J.+Choiauthor=P.+N.+Ingramauthor=K.+Yangauthor=L.+Coffmanauthor=M.+Iyengarauthor=S.+Baiauthor=D.+G.+Thomasauthor=E.+Yoonauthor=R.+J.+Buckanovich&title=Identifying+an+Ovarian+Cancer+Cell+Hierarchy+Regulated+by+Bone+Morphogenetic+Protein+2&doi=10.1073%2Fpnas.1507899112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1507899112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1507899112%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DY.-J.%26aulast%3DIngram%26aufirst%3DP.%2BN.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DCoffman%26aufirst%3DL.%26aulast%3DIyengar%26aufirst%3DM.%26aulast%3DBai%26aufirst%3DS.%26aulast%3DThomas%26aufirst%3DD.%2BG.%26aulast%3DYoon%26aufirst%3DE.%26aulast%3DBuckanovich%26aufirst%3DR.%2BJ.%26atitle%3DIdentifying%2520an%2520Ovarian%2520Cancer%2520Cell%2520Hierarchy%2520Regulated%2520by%2520Bone%2520Morphogenetic%2520Protein%25202%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26issue%3D50%26spage%3DE6882%26epage%3DE6888%26doi%3D10.1073%2Fpnas.1507899112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Strategies Targeting Cancer Stem Cells</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">5</span>â <span class="NLM_lpage">11</span>, <span class="refDoi">Â DOI: 10.1111/cas.12817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1111%2Fcas.12817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=26362755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7bM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=5-11&issue=1&author=G.+J.+Yoshidaauthor=H.+Saya&title=Therapeutic+Strategies+Targeting+Cancer+Stem+Cells&doi=10.1111%2Fcas.12817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic strategies targeting cancer stem cells</span></div><div class="casAuthors">Yoshida, Go J.; Saya, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-11</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cancer stem cells (CSCs) are undifferentiated cancer cells with a high tumorigenic activity, the ability to undergo self-renewal, and a multilineage differentiation potential.  Cancer stem cells are responsible for the development of tumor cell heterogeneity, a key feature for resistance to anticancer treatments including conventional chemotherapy, radiation therapy, and molecularly targeted therapy.  Furthermore, minimal residual disease, the major cause of cancer recurrence and metastasis, is enriched in CSCs.  Cancer stem cells also possess the property of "robustness", which encompasses several characteristics including a slow cell cycle, the ability to detoxify or mediate the efflux of cytotoxic agents, resistance to oxidative stress, and a rapid response to DNA damage, all of which contribute to the development of therapeutic resistance.  The identification of mechanisms underlying such characteristics and the development of novel approaches to target them will be required for the therapeutic elimination of CSCs and the complete eradication of tumors.  In this review, we focus on two prospective therapeutic approaches that target CSCs with the aim of disrupting their quiescence or redox defense capability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt-ai0C0VCYLVg90H21EOLACvtfcHk0lhLWuxx_AYJ2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7bM&md5=043f5210853d44b7cb4f228a86cd90dc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fcas.12817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12817%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DG.%2BJ.%26aulast%3DSaya%26aufirst%3DH.%26atitle%3DTherapeutic%2520Strategies%2520Targeting%2520Cancer%2520Stem%2520Cells%26jtitle%3DCancer%2520Sci.%26date%3D2016%26volume%3D107%26issue%3D1%26spage%3D5%26epage%3D11%26doi%3D10.1111%2Fcas.12817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kryczek, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vatan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szeliga, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotarski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicha, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, W.</span></span> <span> </span><span class="NLM_article-title">Expression of Aldehyde Dehydrogenase and CD133 Defines Ovarian Cancer Stem Cells</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>130</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">29</span>â <span class="NLM_lpage">39</span>, <span class="refDoi">Â DOI: 10.1002/ijc.25967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fijc.25967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21480217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOmsL7N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2012&pages=29-39&issue=1&author=I.+Kryczekauthor=S.+Liuauthor=M.+Rohauthor=L.+Vatanauthor=W.+Szeligaauthor=S.+Weiauthor=M.+Banerjeeauthor=Y.+Maoauthor=J.+Kotarskiauthor=M.+S.+Wichaauthor=R.+Liuauthor=W.+Zou&title=Expression+of+Aldehyde+Dehydrogenase+and+CD133+Defines+Ovarian+Cancer+Stem+Cells&doi=10.1002%2Fijc.25967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of aldehyde dehydrogenase and CD133 defines ovarian cancer stem cells</span></div><div class="casAuthors">Kryczek, Ilona; Liu, Suling; Roh, Michael; Vatan, Linhua; Szeliga, Wojciech; Wei, Shuang; Banerjee, Mousumi; Mao, Yujun; Kotarski, Jan; Wicha, Max S.; Liu, Rebecca; Zou, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-39</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Identification of cancer stem cells is crucial for advancing cancer biol. and therapy.  Several markers including CD24, CD44, CD117, CD133, the G subfamily of ATP-binding cassette transporters (ABCG), epithelial specific antigen (ESA) and aldehyde dehydrogenase (ALDH) are used to identify and investigate human epithelial cancer stem cells in the literature.  We have now systemically analyzed and compared the expression of these markers in fresh ovarian epithelial carcinomas.  Although the expression levels of these markers were unexpectedly variable and partially overlapping in fresh ovarian cancer cells from different donors, we reliably detected important levels of CD133 and ALDH in the majority of fresh ovarian cancer.  Furthermore, most of these stem cell markers including CD133 and ALDH were gradually lost following in vitro passage of primary tumor cells.  However, the expression of ALDH and CD133, but not CD24, CD44 and CD117, could be partially rescued by the in vitro serum-free and sphere cultures and by the in vivo passage in the immune-deficient xenografts.  ALDH+ and CD133+ cells formed three-dimensional spheres more efficiently than their neg. counterparts.  These sphere-forming cells expressed high levels of stem cell core gene transcripts and could be expanded and form addnl. spheres in long-term culture.  ALDH+, CD133+ and ALDH+CD133+ cells from fresh tumors developed larger tumors more rapidly than their neg. counterparts.  This property was preserved in the xenografted tumors.  Altogether, the data suggest that ALDH+ and CD133+ cells are enriched with ovarian cancer-initiating (stem) cells and that ALDH and CD133 may be widely used as reliable markers to investigate ovarian cancer stem cell biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYTMDKETxyV7Vg90H21EOLACvtfcHk0ljswsB4AGdZhA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOmsL7N&md5=cbc76214c56d0ff4d4ca9ddf8549d42a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fijc.25967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25967%26sid%3Dliteratum%253Aachs%26aulast%3DKryczek%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRoh%26aufirst%3DM.%26aulast%3DVatan%26aufirst%3DL.%26aulast%3DSzeliga%26aufirst%3DW.%26aulast%3DWei%26aufirst%3DS.%26aulast%3DBanerjee%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DKotarski%26aufirst%3DJ.%26aulast%3DWicha%26aufirst%3DM.%2BS.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DZou%26aufirst%3DW.%26atitle%3DExpression%2520of%2520Aldehyde%2520Dehydrogenase%2520and%2520CD133%2520Defines%2520Ovarian%2520Cancer%2520Stem%2520Cells%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D130%26issue%3D1%26spage%3D29%26epage%3D39%26doi%3D10.1002%2Fijc.25967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roby, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orsulic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montone, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butzow, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coukos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Distinct Expression Levels and Patterns of Stem Cell Marker, Aldehyde Dehydrogenase Isoform 1 (ALDH1), in Human Epithelial Cancers</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">e10277</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0010277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0010277" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e10277&issue=4&author=S.+Dengauthor=X.+Yangauthor=H.+Lassusauthor=S.+Liangauthor=S.+Kaurauthor=Q.+Yeauthor=C.+Liauthor=L.+P.+Wangauthor=K.+F.+Robyauthor=S.+Orsulicauthor=D.+C.+Connollyauthor=Y.+Zhangauthor=K.+Montoneauthor=R.+Butzowauthor=G.+Coukosauthor=L.+Zhang&title=Distinct+Expression+Levels+and+Patterns+of+Stem+Cell+Marker%2C+Aldehyde+Dehydrogenase+Isoform+1+%28ALDH1%29%2C+in+Human+Epithelial+Cancers&doi=10.1371%2Fjournal.pone.0010277"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0010277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0010277%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLassus%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DKaur%26aufirst%3DS.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DL.%2BP.%26aulast%3DRoby%26aufirst%3DK.%2BF.%26aulast%3DOrsulic%26aufirst%3DS.%26aulast%3DConnolly%26aufirst%3DD.%2BC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMontone%26aufirst%3DK.%26aulast%3DButzow%26aufirst%3DR.%26aulast%3DCoukos%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DDistinct%2520Expression%2520Levels%2520and%2520Patterns%2520of%2520Stem%2520Cell%2520Marker%252C%2520Aldehyde%2520Dehydrogenase%2520Isoform%25201%2520%2528ALDH1%2529%252C%2520in%2520Human%2520Epithelial%2520Cancers%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26issue%3D4%26spage%3De10277%26doi%3D10.1371%2Fjournal.pone.0010277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolluru, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekaran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baby, B. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirimulla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alatassi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankem, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damodaran, C.</span></span> <span> </span><span class="NLM_article-title">Targeting Aberrant Expression of Notch-1 in ALDH(+) Cancer Stem Cells in Breast Cancer</span>. <i>Mol. Carcinog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1127</span>â <span class="NLM_lpage">1136</span>, <span class="refDoi">Â DOI: 10.1002/mc.22579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fmc.22579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=27753148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslyktrzN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=1127-1136&issue=3&author=D.+Palauthor=V.+Kolluruauthor=B.+Chandrasekaranauthor=B.+V.+Babyauthor=M.+Amanauthor=S.+Sumanauthor=S.+Sirimullaauthor=M.+A.+Sandersauthor=H.+Alatassiauthor=M.+K.+Ankemauthor=C.+Damodaran&title=Targeting+Aberrant+Expression+of+Notch-1+in+ALDH%28%2B%29+Cancer+Stem+Cells+in+Breast+Cancer&doi=10.1002%2Fmc.22579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting aberrant expression of Notch-1 in ALDH+ cancer stem cells in breast cancer</span></div><div class="casAuthors">Pal, Deeksha; Kolluru, Venkatesh; Chandrasekaran, Balaji; Baby, Becca V.; Aman, Masarath; Suman, Suman; Sirimulla, Suman; Sanders, Mary Ann; Alatassi, Houda; Ankem, Murali K.; Damodaran, Chendil</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Carcinogenesis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1127-1136</span>CODEN:
                <span class="NLM_cas:coden">MOCAE8</span>;
        ISSN:<span class="NLM_cas:issn">0899-1987</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">We have previously reported that high aldehyde dehydrogenase (ALDH) enzyme activity in breast cancer cells results in breast cancer stem cell (BCSC) properties by upregualting Notch-1 and epithelial mesenchymal markers.  This results in chemoresistance in breast cancer.  Here, we examd. the functional and clin. significance of ALDH expression by measuring the ALDH levels in breast cancer tissues by immunohistochem.  There was a significantly higher ALDH expression in higher grade breast cancer tumor tissues (Grade- II and III) vs. normal breast tissues.  Injection of BCSC (ALDH+ and CD44+/CD22-) cells resulted in aggressive tumor growth in athymic mice vs. ALDH- cells.  The ALDH+ and CD44+/CD22- tumors grow rapidly and are larger than ALDH- tumors which were slow growing and smaller.  Molecularly, ALDH+ tumors expressed higher expression of Notch-1 and EMT markers than ALDH- tumors.  Oral administration of the naturally occurring Psoralidin (Pso, 25 mg/kg of body wt.) significantly inhibited the growth in ALDH+ and ALDH- tumors as well.  Psoralidin inhibited Notch-1 mediated EMT activation in ALDH+ and ALDH- tumors-this confirms our in vitro findings.  Our results suggest that Notch-1 could be an attractive target and inhibition of Notch-1 by Psoralidin may prevent pathogenesis of breast cancer as well as metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU_GYURGg64LVg90H21EOLACvtfcHk0ljswsB4AGdZhA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslyktrzN&md5=6eb43ddd85d0d86f80fc8ddb8381769a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fmc.22579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmc.22579%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DD.%26aulast%3DKolluru%26aufirst%3DV.%26aulast%3DChandrasekaran%26aufirst%3DB.%26aulast%3DBaby%26aufirst%3DB.%2BV.%26aulast%3DAman%26aufirst%3DM.%26aulast%3DSuman%26aufirst%3DS.%26aulast%3DSirimulla%26aufirst%3DS.%26aulast%3DSanders%26aufirst%3DM.%2BA.%26aulast%3DAlatassi%26aufirst%3DH.%26aulast%3DAnkem%26aufirst%3DM.%2BK.%26aulast%3DDamodaran%26aufirst%3DC.%26atitle%3DTargeting%2520Aberrant%2520Expression%2520of%2520Notch-1%2520in%2520ALDH%2528%252B%2529%2520Cancer%2520Stem%2520Cells%2520in%2520Breast%2520Cancer%26jtitle%3DMol.%2520Carcinog.%26date%3D2017%26volume%3D56%26issue%3D3%26spage%3D1127%26epage%3D1136%26doi%3D10.1002%2Fmc.22579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koppaka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juvonen, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deitrich, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenase Inhibitors: a Comprehensive Review of the Pharmacology, Mechanism of Action, Substrate Specificity, and Clinical Application</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>64</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">520</span>â <span class="NLM_lpage">539</span>, <span class="refDoi">Â DOI: 10.1124/pr.111.005538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1124%2Fpr.111.005538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22544865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslaktw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2012&pages=520-539&issue=3&author=V.+Koppakaauthor=D.+C.+Thompsonauthor=Y.+Chenauthor=M.+Ellermannauthor=K.+C.+Nicolaouauthor=R.+O.+Juvonenauthor=D.+Petersenauthor=R.+A.+Deitrichauthor=T.+D.+Hurleyauthor=V.+Vasiliou&title=Aldehyde+Dehydrogenase+Inhibitors%3A+a+Comprehensive+Review+of+the+Pharmacology%2C+Mechanism+of+Action%2C+Substrate+Specificity%2C+and+Clinical+Application&doi=10.1124%2Fpr.111.005538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenase inhibitors: a comprehensive review of the pharmacology, mechanism of action, substrate specificity, and clinical application</span></div><div class="casAuthors">Koppaka, Vindhya; Thompson, David C.; Chen, Ying; Ellermann, Manuel; Nicolaou, Kyriacos C.; Juvonen, Risto O.; Petersen, Dennis; Deitrich, Richard A.; Hurley, Thomas D.; Vasiliou, Vasilis</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">520-539</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Aldehyde dehydrogenases (ALDHs) belong to a superfamily of enzymes that play a key role in the metab. of aldehydes of both endogenous and exogenous derivation.  The human ALDH superfamily comprises 19 isoenzymes that possess important physiol. and toxicol. functions.  The ALDH1A subfamily plays a pivotal role in embryogenesis and development by mediating retinoic acid signaling.  ALDH2, as a key enzyme that oxidizes acetaldehyde, is crucial for alc. metab.  ALDH1A1 and ALDH3A1 are lens and corneal crystallins, which are essential elements of the cellular defense mechanism against UV radiation-induced damage in ocular tissues.  Many ALDH isoenzymes are important in oxidizing reactive aldehydes derived from lipid peroxidn. and thereby help maintain cellular homeostasis.  Increased expression and activity of ALDH isoenzymes have been reported in various human cancers and are assocd. with cancer relapse.  As a direct consequence of their significant physiol. and toxicol. roles, inhibitors of the ALDH enzymes have been developed to treat human diseases.  This review summarizes known ALDH inhibitors, their mechanisms of action, isoenzyme selectivity, potency, and clin. uses.  The purpose of this review is to 1) establish the current status of pharmacol. inhibition of the ALDHs, 2) provide a rationale for the continued development of ALDH isoenzyme-selective inhibitors, and 3) identify the challenges and potential therapeutic rewards assocd. with the creation of such agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOl6kdU4MnaLVg90H21EOLACvtfcHk0ljswsB4AGdZhA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslaktw%253D%253D&md5=1de904ae3778b8715a6ec10e8d0242db</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1124%2Fpr.111.005538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.111.005538%26sid%3Dliteratum%253Aachs%26aulast%3DKoppaka%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DJuvonen%26aufirst%3DR.%2BO.%26aulast%3DPetersen%26aufirst%3DD.%26aulast%3DDeitrich%26aufirst%3DR.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DAldehyde%2520Dehydrogenase%2520Inhibitors%253A%2520a%2520Comprehensive%2520Review%2520of%2520the%2520Pharmacology%252C%2520Mechanism%2520of%2520Action%252C%2520Substrate%2520Specificity%252C%2520and%2520Clinical%2520Application%26jtitle%3DPharmacol.%2520Rev.%26date%3D2012%26volume%3D64%26issue%3D3%26spage%3D520%26epage%3D539%26doi%3D10.1124%2Fpr.111.005538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenases and Cancer Stem Cells</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>369</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">50</span>â <span class="NLM_lpage">57</span>, <span class="refDoi">Â DOI: 10.1016/j.canlet.2015.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.canlet.2015.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=26319899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVent77M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2015&pages=50-57&issue=1&author=X.+Xuauthor=S.+Chaiauthor=P.+Wangauthor=C.+Zhangauthor=Y.+Yangauthor=Y.+Yangauthor=K.+Wang&title=Aldehyde+Dehydrogenases+and+Cancer+Stem+Cells&doi=10.1016%2Fj.canlet.2015.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenases and cancer stem cells</span></div><div class="casAuthors">Xu, Xia; Chai, Shoujie; Wang, Pingli; Zhang, Chenchen; Yang, Yiming; Yang, Ying; Wang, Kai</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-57</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDHs), as essential regulators of aldehyde metab. in the human body, protect organisms from damage induced by active aldehydes.  Given their roles in different cancer types, ALDHs have been evaluated as potential prognostic markers of cancer.  ALDHs exhibit high activity in cancer stem cells (CSCs) and may serve as markers of CSCs.  Moreover, studies indicated that ALDHs and their regulated retinoic acid, reactive oxygen species and reactive aldehydes metab. were strongly related with various properties of CSCs.  Besides, recent research evidences have demonstrated the transcriptional and post-translational regulation of ALDH expression and activation in CSCs.  Thus, this review focuses on the function and regulation of ALDHs in CSCs, particularly ALDH1A1 and ALDH1A3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu8flgrvV35rVg90H21EOLACvtfcHk0ljNZYgNnatVNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVent77M&md5=4b4e93f5f8faf061d6061841fcc187d3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2015.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2015.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DChai%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DK.%26atitle%3DAldehyde%2520Dehydrogenases%2520and%2520Cancer%2520Stem%2520Cells%26jtitle%3DCancer%2520Lett.%26date%3D2015%26volume%3D369%26issue%3D1%26spage%3D50%26epage%3D57%26doi%3D10.1016%2Fj.canlet.2015.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pors, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreb, J. S.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenases in Cancer: an Opportunity for Biomarker and Drug Development?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1953</span>â <span class="NLM_lpage">1963</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2014.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.drudis.2014.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25256776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt7vL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1953-1963&issue=12&author=K.+Porsauthor=J.+S.+Moreb&title=Aldehyde+Dehydrogenases+in+Cancer%3A+an+Opportunity+for+Biomarker+and+Drug+Development%3F&doi=10.1016%2Fj.drudis.2014.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenases in cancer: an opportunity for biomarker and drug development?</span></div><div class="casAuthors">Pors, Klaus; Moreb, Jan S.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1953-1963</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Aldehyde dehydrogenases (ALDHs) belong to a superfamily of 19 isoenzymes that are known to participate in many physiol. important biosynthetic processes including detoxification of specific endogenous and exogenous aldehyde substrates.  The high expression levels of an emerging no. of ALDHs in various cancer tissues suggest that these enzymes have pivotal roles in cancer cell survival and progression.  Mapping out the heterogeneity of tumors and their cancer stem cell (CSC) component will be key to successful design of strategies involving therapeutics that are targeted against specific ALDH isoenzymes.  This review summarises recent progress in ALDH-focused cancer research and discovery of small-mol.-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2K6XIiKHROLVg90H21EOLACvtfcHk0ljNZYgNnatVNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt7vL&md5=3b8dac676074a792064a1c6c36e16dcb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DPors%26aufirst%3DK.%26aulast%3DMoreb%26aufirst%3DJ.%2BS.%26atitle%3DAldehyde%2520Dehydrogenases%2520in%2520Cancer%253A%2520an%2520Opportunity%2520for%2520Biomarker%2520and%2520Drug%2520Development%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26issue%3D12%26spage%3D1953%26epage%3D1963%26doi%3D10.1016%2Fj.drudis.2014.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreb, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostmark, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, W.</span></span> <span> </span><span class="NLM_article-title">ALDH Isozymes Downregulation Affects Cell Growth, Cell Motility and Gene Expression in Lung Cancer Cells</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">87</span>, <span class="refDoi">Â DOI: 10.1186/1476-4598-7-87</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1186%2F1476-4598-7-87" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=19025616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252Fit1ajtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=87&author=J.+S.+Morebauthor=H.+V.+Bakerauthor=L.+J.+Changauthor=M.+Amayaauthor=M.+C.+Lopezauthor=B.+Ostmarkauthor=W.+Chou&title=ALDH+Isozymes+Downregulation+Affects+Cell+Growth%2C+Cell+Motility+and+Gene+Expression+in+Lung+Cancer+Cells&doi=10.1186%2F1476-4598-7-87"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH isozymes downregulation affects cell growth, cell motility and gene expression in lung cancer cells</span></div><div class="casAuthors">Moreb Jan S; Baker Henry V; Chang Lung-Ji; Amaya Maria; Lopez M Cecilia; Ostmark Blanca; Chou Wayne</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">87</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Aldehyde dehydrogenase isozymes ALDH1A1 and ALDH3A1 are highly expressed in non small cell lung cancer.  Neither the mechanisms nor the biologic significance for such over expression have been studied.  METHODS:  We have employed oligonucleotide microarrays to analyze changes in gene profiles in A549 lung cancer cell line in which ALDH activity was reduced by up to 95% using lentiviral mediated expression of siRNA against both isozymes (Lenti 1+3).  Stringent analysis methods were used to identify gene expression patterns that are specific to the knock down of ALDH activity and significantly different in comparison to wild type A549 cells (WT) or cells similarly transduced with green fluorescent protein (GFP) siRNA.  RESULTS:  We confirmed significant and specific down regulation of ALDH1A1 and ALDH3A1 in Lenti 1+3 cells and in comparison to 12 other ALDH genes detected.  The results of the microarray analysis were validated by real time RT-PCR on RNA obtained from Lenti 1+3 or WT cells treated with ALDH activity inhibitors.  Detailed functional analysis was performed on 101 genes that were significantly different (P < 0.001) and their expression changed by > or = 2 folds in the Lenti 1+3 group versus the control groups.  There were 75 down regulated and 26 up regulated genes.  Protein binding, organ development, signal transduction, transcription, lipid metabolism, and cell migration and adhesion were among the most affected pathways.  CONCLUSION:  These molecular effects of the ALDH knock-down are associated with in vitro functional changes in the proliferation and motility of these cells and demonstrate the significance of ALDH enzymes in cell homeostasis with a potentially significant impact on the treatment of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSIE4CRrX3AXY2y4bB4mnvYfW6udTcc2eap8Kd_XEsmmbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252Fit1ajtg%253D%253D&md5=141521d11ed323a2ef5def283846753a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-7-87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-7-87%26sid%3Dliteratum%253Aachs%26aulast%3DMoreb%26aufirst%3DJ.%2BS.%26aulast%3DBaker%26aufirst%3DH.%2BV.%26aulast%3DChang%26aufirst%3DL.%2BJ.%26aulast%3DAmaya%26aufirst%3DM.%26aulast%3DLopez%26aufirst%3DM.%2BC.%26aulast%3DOstmark%26aufirst%3DB.%26aulast%3DChou%26aufirst%3DW.%26atitle%3DALDH%2520Isozymes%2520Downregulation%2520Affects%2520Cell%2520Growth%252C%2520Cell%2520Motility%2520and%2520Gene%2520Expression%2520in%2520Lung%2520Cancer%2520Cells%26jtitle%3DMol.%2520Cancer%26date%3D2008%26volume%3D7%26spage%3D87%26doi%3D10.1186%2F1476-4598-7-87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dallaglio, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roop, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, M.</span></span> <span> </span><span class="NLM_article-title">ALDH1A Isozymes are Markers of Human Melanoma Stem Cells and Potential Therapeutic Targets</span>. <i>Stem Cells</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2100</span>â <span class="NLM_lpage">2113</span>, <span class="refDoi">Â DOI: 10.1002/stem.1193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fstem.1193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22887839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2ntrfK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=2100-2113&issue=10&author=Y.+Luoauthor=K.+Dallaglioauthor=Y.+Chenauthor=W.+A.+Robinsonauthor=S.+E.+Robinsonauthor=M.+D.+McCarterauthor=J.+Wangauthor=R.+Gonzalezauthor=D.+C.+Thompsonauthor=D.+A.+Norrisauthor=D.+R.+Roopauthor=V.+Vasiliouauthor=M.+Fujita&title=ALDH1A+Isozymes+are+Markers+of+Human+Melanoma+Stem+Cells+and+Potential+Therapeutic+Targets&doi=10.1002%2Fstem.1193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH1A isozymes are markers of human melanoma stem cells and potential therapeutic targets</span></div><div class="casAuthors">Luo, Yuchun; Dallaglio, Katiuscia; Chen, Ying; Robinson, William A.; Robinson, Steven E.; McCarter, Martin D.; Wang, Jianbin; Gonzalez, Rene; Thompson, David C.; Norris, David A.; Roop, Dennis R.; Vasiliou, Vasilis; Fujita, Mayumi</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cells (Durham, NC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2100-2113</span>CODEN:
                <span class="NLM_cas:coden">STCEEJ</span>;
        ISSN:<span class="NLM_cas:issn">1066-5099</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">Although the concept of cancer stem cells (CSCs) is well-accepted for many tumors, the existence of such cells in human melanoma has been the subject of debate.  In this study, we demonstrate the existence of human melanoma cells that fulfill the criteria for CSCs (self-renewal and differentiation) by serially xenotransplanting cells into non-obese diabetic/severe combined immunodeficiency (NOD/SCID) mice.  These cells possess high aldehyde dehydrogenase (ALDH) activity with ALDH1A1 and ALDH1A3 being the predominant ALDH isoenzymes.  ALDH-pos. melanoma cells are more tumorigenic than ALDH-neg. cells in both NOD/SCID mice and NSG mice.  Biol. analyses of the ALDH-pos. melanoma cells reveal the ALDH isoenzymes to be key mols. regulating the function of these cells.  Silencing ALDH1A by siRNA or shRNA leads to cell cycle arrest, apoptosis, decreased cell viability in vitro, and reduced tumorigenesis in vivo.  ALDH-pos. melanoma cells are more resistant to chemotherapeutic agents and silencing ALDH1A by siRNA sensitizes melanoma cells to drug-induced cell death.  Furthermore, we, for the first time, examd. the mol. signatures of ALDH-pos. CSCs from patient-derived tumor specimens.  The signatures of melanoma CSCs include retinoic acid (RA)-driven target genes with RA response elements and genes assocd. with stem cell function.  These findings implicate that ALDH isoenzymes are not only biomarkers of CSCs but also attractive therapeutic targets for human melanoma.  Further investigation of these isoenzymes and genes will enhance our understanding of the mol. mechanisms governing CSCs and reveal new mol. targets for therapeutic intervention of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryIxi-Gq5gk7Vg90H21EOLACvtfcHk0ljs_c6Y1nindw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2ntrfK&md5=89c568bb117628a4034c949835d43a92</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fstem.1193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1193%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DDallaglio%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DRobinson%26aufirst%3DW.%2BA.%26aulast%3DRobinson%26aufirst%3DS.%2BE.%26aulast%3DMcCarter%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGonzalez%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26aulast%3DNorris%26aufirst%3DD.%2BA.%26aulast%3DRoop%26aufirst%3DD.%2BR.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DFujita%26aufirst%3DM.%26atitle%3DALDH1A%2520Isozymes%2520are%2520Markers%2520of%2520Human%2520Melanoma%2520Stem%2520Cells%2520and%2520Potential%2520Therapeutic%2520Targets%26jtitle%3DStem%2520Cells%26date%3D2012%26volume%3D30%26issue%3D10%26spage%3D2100%26epage%3D2113%26doi%3D10.1002%2Fstem.1193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, P.</span></span> <span> </span><span class="NLM_article-title">ALDH Maintains the Stemness of Lung Adenoma Stem Cells by Suppressing the Notch/CDK2/CCNE Pathway</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e92669</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0092669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0092669" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e92669&issue=3&author=Z.+Liauthor=Y.+Xiangauthor=L.+Xiangauthor=Y.+Xiaoauthor=F.+Liauthor=P.+Hao&title=ALDH+Maintains+the+Stemness+of+Lung+Adenoma+Stem+Cells+by+Suppressing+the+Notch%2FCDK2%2FCCNE+Pathway&doi=10.1371%2Fjournal.pone.0092669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0092669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0092669%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHao%26aufirst%3DP.%26atitle%3DALDH%2520Maintains%2520the%2520Stemness%2520of%2520Lung%2520Adenoma%2520Stem%2520Cells%2520by%2520Suppressing%2520the%2520Notch%252FCDK2%252FCCNE%2520Pathway%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D3%26spage%3De92669%26doi%3D10.1371%2Fjournal.pone.0092669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duong, H. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, I.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenase 1A1 Confers Intrinsic and Acquired Resistance to Gemcitabine in Human Pancreatic Adenocarcinoma MIA PaCa-2 Cells</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">855</span>â <span class="NLM_lpage">861</span>, <span class="refDoi">Â DOI: 10.3892/ijo.2012.1516</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.3892%2Fijo.2012.1516" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22710732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWhurnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=855-861&issue=3&author=H.+Q.+Duongauthor=J.+S.+Hwangauthor=H.+J.+Kimauthor=H.+J.+Kangauthor=Y.+S.+Seongauthor=I.+Bae&title=Aldehyde+Dehydrogenase+1A1+Confers+Intrinsic+and+Acquired+Resistance+to+Gemcitabine+in+Human+Pancreatic+Adenocarcinoma+MIA+PaCa-2+Cells&doi=10.3892%2Fijo.2012.1516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenase 1A1 confers intrinsic and acquired resistance to gemcitabine in human pancreatic adenocarcinoma MIA PaCa-2 cells</span></div><div class="casAuthors">Duong, Hong-Quan; Hwang, Jae Seok; Kim, Hee Jeong; Kang, Hyo Jin; Seong, Yeon-Sun; Bae, Insoo</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">Gemcitabine (GEM) is the front-line std. chemotherapy used for the treatment of pancreatic cancer; however, chemoresistance to GEM remains the major obstacle to the successful control of this disease.  Both the expression levels and activity of aldehyde dehydrogenase 1A1 (ALDH1A1) are important features of tumor-initiating and/or cancer stem cell properties in multiple types of human cancer.  As one of the intrinsic properties of cancer stem cells is drug resistance, in this study, we examd. the correlation between the level and activity of endogenous ALDHlAl and GEM resistance in the MIA PaCa-2 cell line that contains high expression levels and activity 1 1.DHl Al.  We used small interfering RNAs (siRNAs) to deple1 ALDHl A1 and investigate its potential role in conferring GEM resistance.  The ALDH1A1 knockdown markedly reduc1 ALDHl A1 expression and activity and inhibited cell proliferation.  Moreover, the combination of ALDHlAl-siRNA and GEM significantly decreased cell viability, increased apoptotic cell death and increased the accumulation of cells at the S-phase compared to the controls.  Our data also demonstrated that ALDHlA1 expression and activity were significantly higher in the GEM-resistant MIA PaCa-2 cell line (MIA PaCa-2/GR), compared to the parental MIA PaCa-2 cell line (MIA PaCa-2/P).  In the MIA PaCa-2/GR cells, the combination of ALDHLA1-siRNA and GEM also showed a significant decrease in cell viability and an increase in apoptotic cell death, emphasizing the importance ALDHlA1 in both intrinsic and acquired GEM resistance.  This potentially powerful combination treatment of ALDHlAl-siRNA and GEM warrants further investigation as an effective therapeutic regimen to overcome the resistance of pancreatic cancer to GEM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYdDPtBAWCfbVg90H21EOLACvtfcHk0ljs_c6Y1nindw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWhurnE&md5=ca0d05f83a1e06ae3b02e16fa4e9e517</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3892%2Fijo.2012.1516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2012.1516%26sid%3Dliteratum%253Aachs%26aulast%3DDuong%26aufirst%3DH.%2BQ.%26aulast%3DHwang%26aufirst%3DJ.%2BS.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DKang%26aufirst%3DH.%2BJ.%26aulast%3DSeong%26aufirst%3DY.%2BS.%26aulast%3DBae%26aufirst%3DI.%26atitle%3DAldehyde%2520Dehydrogenase%25201A1%2520Confers%2520Intrinsic%2520and%2520Acquired%2520Resistance%2520to%2520Gemcitabine%2520in%2520Human%2520Pancreatic%2520Adenocarcinoma%2520MIA%2520PaCa-2%2520Cells%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2012%26volume%3D41%26issue%3D3%26spage%3D855%26epage%3D861%26doi%3D10.3892%2Fijo.2012.1516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Condello, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagdas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matei, D.</span></span> <span> </span><span class="NLM_article-title">Beta-Catenin-Regulated ALDH1A1 is a Target in Ovarian Cancer Spheroids</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">2297</span>â <span class="NLM_lpage">2308</span>, <span class="refDoi">Â DOI: 10.1038/onc.2014.178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fonc.2014.178" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=2297-2308&issue=18&author=S.+Condelloauthor=C.+A.+Morganauthor=S.+Nagdasauthor=L.+Caoauthor=J.+Turekauthor=T.+D.+Hurleyauthor=D.+Matei&title=Beta-Catenin-Regulated+ALDH1A1+is+a+Target+in+Ovarian+Cancer+Spheroids&doi=10.1038%2Fonc.2014.178"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.178%26sid%3Dliteratum%253Aachs%26aulast%3DCondello%26aufirst%3DS.%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DNagdas%26aufirst%3DS.%26aulast%3DCao%26aufirst%3DL.%26aulast%3DTurek%26aufirst%3DJ.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DMatei%26aufirst%3DD.%26atitle%3DBeta-Catenin-Regulated%2520ALDH1A1%2520is%2520a%2520Target%2520in%2520Ovarian%2520Cancer%2520Spheroids%26jtitle%3DOncogene%26date%3D2015%26volume%3D34%26issue%3D18%26spage%3D2297%26epage%3D2308%26doi%3D10.1038%2Fonc.2014.178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenkov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickramasinghe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palozola, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showe, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaret, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speicher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ordog, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitler, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span> <span> </span><span class="NLM_article-title">BET Inhibitors Suppress ALDH Activity by Targeting ALDH1A1 Super-Enhancer in Ovarian Cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">6320</span>â <span class="NLM_lpage">6330</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-16-0854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F0008-5472.CAN-16-0854" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=6320-6330&issue=21&author=Y.+Yokoyamaauthor=H.+Zhuauthor=J.+H.+Leeauthor=A.+V.+Kossenkovauthor=S.+Y.+Wuauthor=J.+M.+Wickramasingheauthor=X.+Yinauthor=K.+C.+Palozolaauthor=A.+Gardiniauthor=L.+C.+Showeauthor=K.+S.+Zaretauthor=Q.+Liuauthor=D.+Speicherauthor=J.+R.+Conejo-Garciaauthor=J.+E.+Bradnerauthor=Z.+Zhangauthor=A.+K.+Soodauthor=T.+Ordogauthor=B.+G.+Bitlerauthor=R.+Zhang&title=BET+Inhibitors+Suppress+ALDH+Activity+by+Targeting+ALDH1A1+Super-Enhancer+in+Ovarian+Cancer&doi=10.1158%2F0008-5472.CAN-16-0854"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-0854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-0854%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DKossenkov%26aufirst%3DA.%2BV.%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DWickramasinghe%26aufirst%3DJ.%2BM.%26aulast%3DYin%26aufirst%3DX.%26aulast%3DPalozola%26aufirst%3DK.%2BC.%26aulast%3DGardini%26aufirst%3DA.%26aulast%3DShowe%26aufirst%3DL.%2BC.%26aulast%3DZaret%26aufirst%3DK.%2BS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSpeicher%26aufirst%3DD.%26aulast%3DConejo-Garcia%26aufirst%3DJ.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DOrdog%26aufirst%3DT.%26aulast%3DBitler%26aufirst%3DB.%2BG.%26aulast%3DZhang%26aufirst%3DR.%26atitle%3DBET%2520Inhibitors%2520Suppress%2520ALDH%2520Activity%2520by%2520Targeting%2520ALDH1A1%2520Super-Enhancer%2520in%2520Ovarian%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26issue%3D21%26spage%3D6320%26epage%3D6330%26doi%3D10.1158%2F0008-5472.CAN-16-0854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. N.</span></span> <span> </span><span class="NLM_article-title">ALDH1A3, the Major Aldehyde Dehydrogenase Isoform in Human Cholangiocarcinoma Cells, Affects Prognosis and Gemcitabine Resistance in Cholangiocarcinoma Patients</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">4225</span>â <span class="NLM_lpage">4235</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-15-1800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F1078-0432.CCR-15-1800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=27076629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKrsbbO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=4225-4235&issue=16&author=M.+H.+Chenauthor=J.+J.+Wengauthor=C.+T.+Chengauthor=R.+C.+Wuauthor=S.+C.+Huangauthor=C.+E.+Wuauthor=Y.+H.+Chungauthor=C.+Y.+Liuauthor=M.+H.+Changauthor=M.+H.+Chenauthor=K.+C.+Chiangauthor=T.+S.+Yehauthor=Y.+Suauthor=C.+N.+Yeh&title=ALDH1A3%2C+the+Major+Aldehyde+Dehydrogenase+Isoform+in+Human+Cholangiocarcinoma+Cells%2C+Affects+Prognosis+and+Gemcitabine+Resistance+in+Cholangiocarcinoma+Patients&doi=10.1158%2F1078-0432.CCR-15-1800"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH1A3, the Major Aldehyde Dehydrogenase Isoform in Human Cholangiocarcinoma Cells, Affects Prognosis and Gemcitabine Resistance in Cholangiocarcinoma Patients</span></div><div class="casAuthors">Chen, Ming-Huang; Weng, Jing-Jie; Cheng, Chi-Tung; Wu, Ren-Chin; Huang, Shih-Chiang; Wu, Chiao-En; Chung, Yi-Hsiu; Liu, Chun-Yu; Chang, Mu-Hsin; Chen, Ming-Han; Chiang, Kun-Chun; Yeh, Ta-Sen; Su, Yeu; Yeh, Chun-Nan</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4225-4235</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Intrahepatic cholangiocarcinoma is a fatal primary liver cancer resulting from diagnosis at an advanced stage.  Understanding the mechanisms of drug resistance and metastasis of cholangiocarcinoma may improve the disease prognosis.  Enhanced aldehyde dehydrogenase (ALDH) activity is suggested to be assocd. with increased drug resistance and the metastasis.  This study aims to investigate the roles of the ALDH isoforms in cholangiocarcinoma.  Exptl. Design: Aldefluor assays, RT-PCR, and Western blot anal. were used to identify the major ALDH isoforms contributing to Aldefluor activity in human cholangiocarcinoma cell lines.  We manipulated isoform expression in HuCCT1 cells to elucidate the role of ALDH1A3 in the malignant progression of these cells.  Finally, we used immunohistochem. staining to evaluate the clin. significance of ALDH1A3 in 77 hepatectomized cholangiocarcinoma patients and an addnl. 31 patients with advanced cholangiocarcinoma who received gemcitabine-based therapy.  Results: ALDHhigh cholangiocarcinoma cells not only migrated faster but were more resistant to gemcitabine.  Among the 19 ALDH isoforms studied, ALDH1A3 was found to be the main contributor to Aldefluor activity.  In addn., we also found that knockdown of ALDH1A3 expression in HuCCT1 cells markedly reduced not only their sensitivity to gemcitabine, which might be attributed to a decreased expression of ribonucleotide reductase M1, but also their migration.  Most importantly, this enzyme was also identified as an independent poor prognostic factor for patients with intrahepatic cholangiocarcinoma, as well as a prognostic biomarker of gemcitabine-treated patients.  Conclusions: ALDH1A3 plays an important role in enhancing malignant behavior of cholangiocarcinoma and serves as a new therapeutic target.  Clin Cancer Res; 22(16); 4225-35. Â©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSsIYLXC1BjLVg90H21EOLACvtfcHk0lg4XQzh68SN7A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKrsbbO&md5=94376c2d13426323f216f9a7c2c4a42c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-15-1800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-15-1800%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DM.%2BH.%26aulast%3DWeng%26aufirst%3DJ.%2BJ.%26aulast%3DCheng%26aufirst%3DC.%2BT.%26aulast%3DWu%26aufirst%3DR.%2BC.%26aulast%3DHuang%26aufirst%3DS.%2BC.%26aulast%3DWu%26aufirst%3DC.%2BE.%26aulast%3DChung%26aufirst%3DY.%2BH.%26aulast%3DLiu%26aufirst%3DC.%2BY.%26aulast%3DChang%26aufirst%3DM.%2BH.%26aulast%3DChen%26aufirst%3DM.%2BH.%26aulast%3DChiang%26aufirst%3DK.%2BC.%26aulast%3DYeh%26aufirst%3DT.%2BS.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DYeh%26aufirst%3DC.%2BN.%26atitle%3DALDH1A3%252C%2520the%2520Major%2520Aldehyde%2520Dehydrogenase%2520Isoform%2520in%2520Human%2520Cholangiocarcinoma%2520Cells%252C%2520Affects%2520Prognosis%2520and%2520Gemcitabine%2520Resistance%2520in%2520Cholangiocarcinoma%2520Patients%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2016%26volume%3D22%26issue%3D16%26spage%3D4225%26epage%3D4235%26doi%3D10.1158%2F1078-0432.CCR-15-1800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cortes-Dericks, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froment, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoubi, G.</span></span> <span> </span><span class="NLM_article-title">Cisplatin-Resistant Cells in Malignant Pleural Mesothelioma Cell Lines Show ALDH(high)CD44(+) Phenotype and Sphere-Forming Capacity</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">304</span>, <span class="refDoi">Â DOI: 10.1186/1471-2407-14-304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1186%2F1471-2407-14-304" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=304&author=L.+Cortes-Dericksauthor=L.+Fromentauthor=R.+Boeschauthor=R.+A.+Schmidauthor=G.+Karoubi&title=Cisplatin-Resistant+Cells+in+Malignant+Pleural+Mesothelioma+Cell+Lines+Show+ALDH%28high%29CD44%28%2B%29+Phenotype+and+Sphere-Forming+Capacity&doi=10.1186%2F1471-2407-14-304"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-304%26sid%3Dliteratum%253Aachs%26aulast%3DCortes-Dericks%26aufirst%3DL.%26aulast%3DFroment%26aufirst%3DL.%26aulast%3DBoesch%26aufirst%3DR.%26aulast%3DSchmid%26aufirst%3DR.%2BA.%26aulast%3DKaroubi%26aufirst%3DG.%26atitle%3DCisplatin-Resistant%2520Cells%2520in%2520Malignant%2520Pleural%2520Mesothelioma%2520Cell%2520Lines%2520Show%2520ALDH%2528high%2529CD44%2528%252B%2529%2520Phenotype%2520and%2520Sphere-Forming%2520Capacity%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D304%26doi%3D10.1186%2F1471-2407-14-304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saw, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, W.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S.-W.</span></span> <span> </span><span class="NLM_article-title">Characterization of Aldehyde Dehydrogenase Isozymes in Ovarian Cancer Tissues and Sphere Cultures</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">329</span>, <span class="refDoi">Â DOI: 10.1186/1471-2407-12-329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1186%2F1471-2407-12-329" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=329&issue=1&author=Y.-T.+Sawauthor=J.+Yangauthor=S.-K.+Ngauthor=S.+Liuauthor=S.+Singhauthor=M.+Singhauthor=W.+R.+Welchauthor=H.+Tsudaauthor=W.-P.+Fongauthor=D.+Thompsonauthor=V.+Vasiliouauthor=R.+S.+Berkowitzauthor=S.-W.+Ng&title=Characterization+of+Aldehyde+Dehydrogenase+Isozymes+in+Ovarian+Cancer+Tissues+and+Sphere+Cultures&doi=10.1186%2F1471-2407-12-329"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-12-329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-12-329%26sid%3Dliteratum%253Aachs%26aulast%3DSaw%26aufirst%3DY.-T.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DNg%26aufirst%3DS.-K.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DWelch%26aufirst%3DW.%2BR.%26aulast%3DTsuda%26aufirst%3DH.%26aulast%3DFong%26aufirst%3DW.-P.%26aulast%3DThompson%26aufirst%3DD.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DBerkowitz%26aufirst%3DR.%2BS.%26aulast%3DNg%26aufirst%3DS.-W.%26atitle%3DCharacterization%2520of%2520Aldehyde%2520Dehydrogenase%2520Isozymes%2520in%2520Ovarian%2520Cancer%2520Tissues%2520and%2520Sphere%2520Cultures%26jtitle%3DBMC%2520Cancer%26date%3D2012%26volume%3D12%26issue%3D1%26spage%3D329%26doi%3D10.1186%2F1471-2407-12-329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowtell, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BÃ¶hm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuria, P.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beral, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berek, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birrer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bookman, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiappinelli, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coukos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drapkin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotopoulou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabra, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gourley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heong, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntsman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwanicki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlan, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengyel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeish, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narod, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nephew, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pharoah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, D. J.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stronach, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balkwill, F. R.</span></span> <span> </span><span class="NLM_article-title">Rethinking Ovarian Cancer II: Reducing Mortality From High-grade Serous Ovarian Cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">668</span>, <span class="refDoi">Â DOI: 10.1038/nrc4019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fnrc4019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=26493647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWhsLjF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=668&author=D.+D.+Bowtellauthor=S.+B%C3%B6hmauthor=A.+A.+Ahmedauthor=P.-J.+Aspuriaauthor=R.+C.+Bastauthor=V.+Beralauthor=J.+S.+Berekauthor=M.+J.+Birrerauthor=S.+Blagdenauthor=M.+A.+Bookmanauthor=J.+D.+Brentonauthor=K.+B.+Chiappinelliauthor=F.+C.+Martinsauthor=G.+Coukosauthor=R.+Drapkinauthor=R.+Edmondsonauthor=C.+Fotopoulouauthor=H.+Gabraauthor=J.+Galonauthor=C.+Gourleyauthor=V.+Heongauthor=D.+G.+Huntsmanauthor=M.+Iwanickiauthor=B.+Y.+Karlanauthor=A.+Kayeauthor=E.+Lengyelauthor=D.+A.+Levineauthor=K.+H.+Luauthor=I.+A.+McNeishauthor=U.+Menonauthor=S.+A.+Narodauthor=B.+H.+Nelsonauthor=K.+P.+Nephewauthor=P.+Pharoahauthor=D.+J.+Powellauthor=P.+Ramosauthor=I.+L.+Romeroauthor=C.+L.+Scottauthor=A.+K.+Soodauthor=E.+A.+Stronachauthor=F.+R.+Balkwill&title=Rethinking+Ovarian+Cancer+II%3A+Reducing+Mortality+From+High-grade+Serous+Ovarian+Cancer&doi=10.1038%2Fnrc4019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking ovarian cancer II: reducing mortality from high-grade serous ovarian cancer</span></div><div class="casAuthors">Bowtell, David D.; Bohm, Steffen; Ahmed, Ahmed A.; Aspuria, Paul-Joseph; Bast, Robert C., Jr.; Beral, Valerie; Berek, Jonathan S.; Birrer, Michael J.; Blagden, Sarah; Bookman, Michael A.; Brenton, James D.; Chiappinelli, Katherine B.; Martins, Filipe Correia; Coukos, George; Drapkin, Ronny; Edmondson, Richard; Fotopoulou, Christina; Gabra, Hani; Galon, Jerome; Gourley, Charlie; Heong, Valerie; Huntsman, David G.; Iwanicki, Marcin; Karlan, Beth Y.; Kaye, Allyson; Lengyel, Ernst; Levine, Douglas A.; Lu, Karen H.; McNeish, Iain A.; Menon, Usha; Narod, Steven A.; Nelson, Brad H.; Nephew, Kenneth P.; Pharoah, Paul; Powell, Daniel J. Jr; Ramos, Pilar; Romero, Iris L.; Scott, Clare L.; Sood, Anil K.; Stronach, Euan A.; Balkwill, Frances R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">668-679</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">High-grade serous ovarian cancer (HGSOC) accounts for 70-80% of ovarian cancer deaths, and overall survival has not changed significantly for several decades.  In this Opinion article, we outline a set of research priorities that we believe will reduce incidence and improve outcomes for women with this disease.  This 'roadmap' for HGSOC was detd. after extensive discussions at an Ovarian Cancer Action meeting in Jan. 2015.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4-9JF1j3Wv7Vg90H21EOLACvtfcHk0ljiJPba97AoeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWhsLjF&md5=a23413ed49019485b07b170c3eae7fcb</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrc4019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc4019%26sid%3Dliteratum%253Aachs%26aulast%3DBowtell%26aufirst%3DD.%2BD.%26aulast%3DB%25C3%25B6hm%26aufirst%3DS.%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26aulast%3DAspuria%26aufirst%3DP.-J.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DBeral%26aufirst%3DV.%26aulast%3DBerek%26aufirst%3DJ.%2BS.%26aulast%3DBirrer%26aufirst%3DM.%2BJ.%26aulast%3DBlagden%26aufirst%3DS.%26aulast%3DBookman%26aufirst%3DM.%2BA.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DChiappinelli%26aufirst%3DK.%2BB.%26aulast%3DMartins%26aufirst%3DF.%2BC.%26aulast%3DCoukos%26aufirst%3DG.%26aulast%3DDrapkin%26aufirst%3DR.%26aulast%3DEdmondson%26aufirst%3DR.%26aulast%3DFotopoulou%26aufirst%3DC.%26aulast%3DGabra%26aufirst%3DH.%26aulast%3DGalon%26aufirst%3DJ.%26aulast%3DGourley%26aufirst%3DC.%26aulast%3DHeong%26aufirst%3DV.%26aulast%3DHuntsman%26aufirst%3DD.%2BG.%26aulast%3DIwanicki%26aufirst%3DM.%26aulast%3DKarlan%26aufirst%3DB.%2BY.%26aulast%3DKaye%26aufirst%3DA.%26aulast%3DLengyel%26aufirst%3DE.%26aulast%3DLevine%26aufirst%3DD.%2BA.%26aulast%3DLu%26aufirst%3DK.%2BH.%26aulast%3DMcNeish%26aufirst%3DI.%2BA.%26aulast%3DMenon%26aufirst%3DU.%26aulast%3DNarod%26aufirst%3DS.%2BA.%26aulast%3DNelson%26aufirst%3DB.%2BH.%26aulast%3DNephew%26aufirst%3DK.%2BP.%26aulast%3DPharoah%26aufirst%3DP.%26aulast%3DPowell%26aufirst%3DD.%2BJ.%26aulast%3DRamos%26aufirst%3DP.%26aulast%3DRomero%26aufirst%3DI.%2BL.%26aulast%3DScott%26aufirst%3DC.%2BL.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DStronach%26aufirst%3DE.%2BA.%26aulast%3DBalkwill%26aufirst%3DF.%2BR.%26atitle%3DRethinking%2520Ovarian%2520Cancer%2520II%253A%2520Reducing%2520Mortality%2520From%2520High-grade%2520Serous%2520Ovarian%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D668%26doi%3D10.1038%2Fnrc4019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Titus, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danchik, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. J.</span></span> <span> </span><span class="NLM_article-title">Correction: A High-Content Assay Enables the Automated Screening and Identification of Small Molecules with Specific ALDH1A1-Inhibitory Activity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">e0197292</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0197292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0197292" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=e0197292&issue=5&author=A.+Yasgarauthor=S.+A.+Titusauthor=Y.+Wangauthor=C.+Danchikauthor=S.-M.+Yangauthor=V.+Vasiliouauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=N.+J.+Martinez&title=Correction%3A+A+High-Content+Assay+Enables+the+Automated+Screening+and+Identification+of+Small+Molecules+with+Specific+ALDH1A1-Inhibitory+Activity&doi=10.1371%2Fjournal.pone.0197292"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0197292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0197292%26sid%3Dliteratum%253Aachs%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DTitus%26aufirst%3DS.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDanchik%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DS.-M.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DMartinez%26aufirst%3DN.%2BJ.%26atitle%3DCorrection%253A%2520A%2520High-Content%2520Assay%2520Enables%2520the%2520Automated%2520Screening%2520and%2520Identification%2520of%2520Small%2520Molecules%2520with%2520Specific%2520ALDH1A1-Inhibitory%2520Activity%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3De0197292%26doi%3D10.1371%2Fjournal.pone.0197292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchman, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dria, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">N,N-Diethylaminobenzaldehyde (DEAB) as a Substrate and Mechanism-Based Inhibitor for Human ALDH Isoenzymes</span>. <i>Chem.-Biol. Interact.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>234</i></span> (<span class="NLM_issue">0</span>),  <span class="NLM_fpage">18</span>â <span class="NLM_lpage">28</span>, <span class="refDoi">Â DOI: 10.1016/j.cbi.2014.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.cbi.2014.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25512087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkslWmsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2015&pages=18-28&issue=0&author=C.+A.+Morganauthor=B.+Parajuliauthor=C.+D.+Buchmanauthor=K.+Driaauthor=T.+D.+Hurley&title=N%2CN-Diethylaminobenzaldehyde+%28DEAB%29+as+a+Substrate+and+Mechanism-Based+Inhibitor+for+Human+ALDH+Isoenzymes&doi=10.1016%2Fj.cbi.2014.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">N,N-diethylaminobenzaldehyde (DEAB) as a substrate and mechanism-based inhibitor for human ALDH isoenzymes</span></div><div class="casAuthors">Morgan, Cynthia A.; Parajuli, Bibek; Buchman, Cameron D.; Dria, Karl; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18-28</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">4-(N,N-Diethylamino)benzaldehyde(DEAB) is a commonly used "selective" inhibitor of human NAD(P)-dependent aldehyde dehydrogenase (ALDH) isoenzymes in cancer stem cell biol. due to its inclusion as a neg. control compd. in the widely utilized Aldefluor assay.  Recent evidence has accumulated that DEAB is not a selective inhibitory agent when assayed in vitro vs. ALDH1, ALDH2, and ALDH3 family members.  Here, the authors sought to det. the selectivity of DEAB toward ALDH1A1, ALDH1A2, ALDH1A3, ALDH1B1, ALDH1L1, ALDH2, ALDH3A1, ALDH4A1, and ALDH5A1 isoenzymes and det. the mechanism by which DEAB exerts its inhibitory action.  The authors found that DEAB was an excellent substrate for ALDH3A1, exhibiting a Vmax/Km that exceeded that of its commonly used substrate, benzaldehyde.  DEAB was also a substrate for ALDH1A1, albeit an exceptionally slow one (turnover rate of â¼0.03 min-1).  In contrast, little if any turnover of DEAB was obsd. when incubated with ALDH1A2, ALDH1A3, ALDH1B1, ALDH2, or ALDH5A1.  DEAB was neither a substrate nor an inhibitor for ALDH1L1 or ALDH4A1.  Anal. by enzyme kinetics and QTOF mass spectrometry demonstrated that DEAB was an irreversible inhibitor of ALDH1A2 and ALDH2 with apparent bimol. rate consts. of 2900 and 86,000 M-1 s-1, resp.  The mechanism of inactivation was consistent with the formation of a quinoid-like resonance state following hydride transfer that was stabilized by local structural features that exist in several of the ALDH isoenzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrREVT_lTwCJLVg90H21EOLACvtfcHk0ljiJPba97AoeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkslWmsA%253D%253D&md5=82753ba4991367a9a5da5e6d64770ced</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DBuchman%26aufirst%3DC.%2BD.%26aulast%3DDria%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DN%252CN-Diethylaminobenzaldehyde%2520%2528DEAB%2529%2520as%2520a%2520Substrate%2520and%2520Mechanism-Based%2520Inhibitor%2520for%2520Human%2520ALDH%2520Isoenzymes%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D2015%26volume%3D234%26issue%3D0%26spage%3D18%26epage%3D28%26doi%3D10.1016%2Fj.cbi.2014.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Croker, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allan, A. L.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Aldehyde Dehydrogenase (ALDH) Activity Reduces Chemotherapy and Radiation Resistance of Stem-Like ALDHhiCD44+ Human Breast Cancer Cells</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>133</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">75</span>â <span class="NLM_lpage">87</span>, <span class="refDoi">Â DOI: 10.1007/s10549-011-1692-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1007%2Fs10549-011-1692-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21818590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtlOlurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2012&pages=75-87&issue=1&author=A.+K.+Crokerauthor=A.+L.+Allan&title=Inhibition+of+Aldehyde+Dehydrogenase+%28ALDH%29+Activity+Reduces+Chemotherapy+and+Radiation+Resistance+of+Stem-Like+ALDHhiCD44%2B+Human+Breast+Cancer+Cells&doi=10.1007%2Fs10549-011-1692-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of aldehyde dehydrogenase (ALDH) activity reduces chemotherapy and radiation resistance of stem-like ALDHhiCD44+ human breast cancer cells</span></div><div class="casAuthors">Croker, Alysha K.; Allan, Alison L.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-87</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The majority of breast cancer deaths are because of ineffective treatment of metastatic disease.  We previously identified a subpopulation of cells in human breast cancer cell lines that demonstrate high activity of aldehyde dehydrogenase (ALDH) and high expression of CD44.  These ALDHhiCD44+ cells displayed enhanced metastatic behavior in vitro and in vivo relative to ALDHlowCD44- cells.  The goal of this study was to test the hypothesis that ALDHhiCD44+ breast cancer cells are more resistant to std. cancer therapy, and that inhibiting ALDH activity through all-trans retinoic acid (ATRA) or the specific ALDH inhibitor diethylaminobenzaldehyde (DEAB) sensitizes these cells to treatment.  ALDHhiCD44+ and ALDHlowCD44- populations were isolated from MDA-MB-231 and MDA-MB-468 cells lines and exposed to chemotherapy (doxorubicin/paclitaxel) or radiotherapy Â± ATRA or DEAB.  Cell populations were assessed for differences in survival, colony formation, and protein expression related to therapy resistance and differentiation.  Significantly more ALDHhiCD44+ cells survived chemotherapy/radiotherapy relative to ALDHlowCD44- cells (P < 0.001).  Glutathione-S-transferase pi, p-glycoprotein, and/or CHK1 were overexpressed in ALDHhiCD44+ populations compared with ALDHlowCD44- populations (P < 0.05).  Pre-treatment of cell populations with DEAB or ATRA had no effect on ALDHlowCD44- cells, but resulted in significant initial sensitization of ALDHhiCD44+ cells to chemotherapy/radiotherapy.  However, only DEAB had a long-term effect, resulting in reduced colony formation (P < 0.01).  ATRA also significantly increased expression of CK8/18/19 in MDA-MB-468 ALDHhiCD44+ cells compared with control (P < 0.05).  Our novel findings indicate that ALDHhiCD44+ breast cancer cells contribute to both chemotherapy and radiation resistance and suggest a much broader role for ALDH in treatment response than previously reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqugwcSmbKKLbVg90H21EOLACvtfcHk0lgk1sVEam-DDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtlOlurc%253D&md5=64822493337b70996c7515e3368c2b50</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs10549-011-1692-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-011-1692-y%26sid%3Dliteratum%253Aachs%26aulast%3DCroker%26aufirst%3DA.%2BK.%26aulast%3DAllan%26aufirst%3DA.%2BL.%26atitle%3DInhibition%2520of%2520Aldehyde%2520Dehydrogenase%2520%2528ALDH%2529%2520Activity%2520Reduces%2520Chemotherapy%2520and%2520Radiation%2520Resistance%2520of%2520Stem-Like%2520ALDHhiCD44%252B%2520Human%2520Breast%2520Cancer%2520Cells%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2012%26volume%3D133%26issue%3D1%26spage%3D75%26epage%3D87%26doi%3D10.1007%2Fs10549-011-1692-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goktug, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Channathodiyil, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannappan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hugnot, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichet, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armesilla, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darling, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span> <span> </span><span class="NLM_article-title">Cytotoxic Effect of Disulfiram/Copper on Human Glioblastoma Cell Lines and ALDH-Positive Cancer-Stem-Like Cells</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>107</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1488</span>â <span class="NLM_lpage">1497</span>, <span class="refDoi">Â DOI: 10.1038/bjc.2012.442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fbjc.2012.442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=23033007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOntLjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2012&pages=1488-1497&issue=9&author=P.+Liuauthor=S.+Brownauthor=T.+Goktugauthor=P.+Channathodiyilauthor=V.+Kannappanauthor=J.+P.+Hugnotauthor=P.+O.+Guichetauthor=X.+Bianauthor=A.+L.+Armesillaauthor=J.+L.+Darlingauthor=W.+Wang&title=Cytotoxic+Effect+of+Disulfiram%2FCopper+on+Human+Glioblastoma+Cell+Lines+and+ALDH-Positive+Cancer-Stem-Like+Cells&doi=10.1038%2Fbjc.2012.442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Cytotoxic effect of disulfiram/copper on human glioblastoma cell lines and ALDH-positive cancer-stem-like cells</span></div><div class="casAuthors">Liu, P.; Brown, S.; Goktug, T.; Channathodiyil, P.; Kannappan, V.; Hugnot, J-P.; Guichet, P.-O.; Bian, X.; Armesilla, A. L.; Darling, J. L.; Wang, W.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1488-1497</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Background: Glioblastoma multiforme (GBM) cells are resistant to anticancer drugs.  Cancer stem cells (CSCs) are a key mediator of chemoresistance.  We have reported that disulfiram (DS), an aldehyde dehydrogenase (ALDH) inhibitor, targets breast CSC-like cells.  In this study, the effect of DS and combination of DS and gemcitabine (dFdC) on GBM cells and GBM stem-like cells was investigated.  Methods: 1-(4,5-Dimethylthiazol-2-yl)-3,5-diphenylformazan (MTT), combination index (CI)-isobologram, western blot, luciferase reporter gene assay, electrophoretic mobility-shift assay and ALDH anal. were used in this study.  Results: Disulfiram is cytotoxic in GBM cell lines in a copper (Cu)-dependent manner.  Disulfiram/copper enhances the cytotoxicity of dFdC.  Combination index-isobologram anal. indicates a synergistic effect between DS/Cu and dFdC.  Disulfiram/copper induces reactive oxygen species (ROS), activates JNK and p38 pathways and inhibits nuclear factor-kappa B activity in GBM cell lines.  Disulfiram/copper may trigger intrinsic apoptotic pathway via modulation of the Bcl2 family.  Disulfiram/copper abolishes stem-like cell population in GBM cell lines.  Conclusion: Our findings indicate that the cytotoxicity of DS/Cu and the enhancing effect of DS/Cu on the cytotoxicity of dFdC in GBM stem-like cells may be caused by induction of ROS and inhibition of both ALDH and the NFkB pathway.  Both DS and dFdC can traverse the blood-brain barrier.  Further study may lead them into GBM chemotherapy.  British Journal of Cancer (2012) 107, 1488-1497; doi:10.1038/bjc.2012.442 www.bjcancer.com Published online 2 Oct. 2012.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4qo20htmw2bVg90H21EOLACvtfcHk0lgk1sVEam-DDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOntLjO&md5=d66fc5467e594264c42dd3e6f61f7083</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2012.442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2012.442%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DGoktug%26aufirst%3DT.%26aulast%3DChannathodiyil%26aufirst%3DP.%26aulast%3DKannappan%26aufirst%3DV.%26aulast%3DHugnot%26aufirst%3DJ.%2BP.%26aulast%3DGuichet%26aufirst%3DP.%2BO.%26aulast%3DBian%26aufirst%3DX.%26aulast%3DArmesilla%26aufirst%3DA.%2BL.%26aulast%3DDarling%26aufirst%3DJ.%2BL.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DCytotoxic%2520Effect%2520of%2520Disulfiram%252FCopper%2520on%2520Human%2520Glioblastoma%2520Cell%2520Lines%2520and%2520ALDH-Positive%2520Cancer-Stem-Like%2520Cells%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2012%26volume%3D107%26issue%3D9%26spage%3D1488%26epage%3D1497%26doi%3D10.1038%2Fbjc.2012.442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nechushtan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamreh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotfried, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalev, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nisman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peretz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peylan-Ramu, N.</span></span> <span> </span><span class="NLM_article-title">A Phase IIb Trial Assessing the Addition of Disulfiram to Chemotherapy for the Treatment of Metastatic Non-Small Cell Lung Cancer</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">366</span>â <span class="NLM_lpage">367</span>, <span class="refDoi">Â DOI: 10.1634/theoncologist.2014-0424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1634%2Ftheoncologist.2014-0424" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=366-367&issue=4&author=H.+Nechushtanauthor=Y.+Hamamrehauthor=S.+Nidalauthor=M.+Gotfriedauthor=A.+Baronauthor=Y.+I.+Shalevauthor=B.+Nismanauthor=T.+Peretzauthor=N.+Peylan-Ramu&title=A+Phase+IIb+Trial+Assessing+the+Addition+of+Disulfiram+to+Chemotherapy+for+the+Treatment+of+Metastatic+Non-Small+Cell+Lung+Cancer&doi=10.1634%2Ftheoncologist.2014-0424"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2014-0424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2014-0424%26sid%3Dliteratum%253Aachs%26aulast%3DNechushtan%26aufirst%3DH.%26aulast%3DHamamreh%26aufirst%3DY.%26aulast%3DNidal%26aufirst%3DS.%26aulast%3DGotfried%26aufirst%3DM.%26aulast%3DBaron%26aufirst%3DA.%26aulast%3DShalev%26aufirst%3DY.%2BI.%26aulast%3DNisman%26aufirst%3DB.%26aulast%3DPeretz%26aufirst%3DT.%26aulast%3DPeylan-Ramu%26aufirst%3DN.%26atitle%3DA%2520Phase%2520IIb%2520Trial%2520Assessing%2520the%2520Addition%2520of%2520Disulfiram%2520to%2520Chemotherapy%2520for%2520the%2520Treatment%2520of%2520Metastatic%2520Non-Small%2520Cell%2520Lung%2520Cancer%26jtitle%3DOncologist%26date%3D2015%26volume%3D20%26issue%3D4%26spage%3D366%26epage%3D367%26doi%3D10.1634%2Ftheoncologist.2014-0424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skrott, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistrik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozdian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moudry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaclavkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrobel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouckova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedlacek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miklovicova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driessen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Q. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">Alcohol-Abuse Drug Disulfiram Targets Cancer Via p97 Segregase Adaptor NPL4</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>552</i></span>,  <span class="NLM_fpage">194</span>, <span class="refDoi">Â DOI: 10.1038/nature25016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fnature25016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=29211715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFGlsbrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=552&publication_year=2017&pages=194&author=Z.+Skrottauthor=M.+Mistrikauthor=K.+K.+Andersenauthor=S.+Friisauthor=D.+Majeraauthor=J.+Gurskyauthor=T.+Ozdianauthor=J.+Bartkovaauthor=Z.+Turiauthor=P.+Moudryauthor=M.+Krausauthor=M.+Michalovaauthor=J.+Vaclavkovaauthor=P.+Dzubakauthor=I.+Vrobelauthor=P.+Pouckovaauthor=J.+Sedlacekauthor=A.+Miklovicovaauthor=A.+Kuttauthor=J.+Liauthor=J.+Mattovaauthor=C.+Driessenauthor=Q.+P.+Douauthor=J.+Olsenauthor=M.+Hajduchauthor=B.+Cvekauthor=R.+J.+Deshaiesauthor=J.+Bartek&title=Alcohol-Abuse+Drug+Disulfiram+Targets+Cancer+Via+p97+Segregase+Adaptor+NPL4&doi=10.1038%2Fnature25016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Alcohol-abuse drug disulfiram targets cancer via p97 segregase adaptor NPL4</span></div><div class="casAuthors">Skrott, Zdenek; Mistrik, Martin; Andersen, Klaus Kaae; Friis, Soeren; Majera, Dusana; Gursky, Jan; Ozdian, Tomas; Bartkova, Jirina; Turi, Zsofia; Moudry, Pavel; Kraus, Marianne; Michalova, Martina; Vaclavkova, Jana; Dzubak, Petr; Vrobel, Ivo; Pouckova, Pavla; Sedlacek, Jindrich; Miklovicova, Andrea; Kutt, Anne; Li, Jing; Mattova, Jana; Driessen, Christoph; Dou, Q. Ping; Olsen, Joergen; Hajduch, Marian; Cvek, Boris; Deshaies, Raymond J.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">552</span>
        (<span class="NLM_cas:issue">7684</span>),
    <span class="NLM_cas:pages">194-199</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cancer incidence is rising and this global challenge is further exacerbated by tumor resistance to available medicines.  A promising approach to meet the need for improved cancer treatment is drug repurposing.  Here we highlight the potential for repurposing disulfiram (also known by the trade name Antabuse), an old alc.-aversion drug that has been shown to be effective against diverse cancer types in preclin. studies.  Our nationwide epidemiol. study reveals that patients who continuously used disulfiram have a lower risk of death from cancer compared to those who stopped using the drug at their diagnosis.  Moreover, we identify the ditiocarb-copper complex as the metabolite of disulfiram that is responsible for its anti-cancer effects, and provide methods to detect preferential accumulation of the complex in tumors and candidate biomarkers to analyze its effect on cells and tissues.  Finally, our functional and biophys. analyses reveal the mol. target of disulfiram's tumor-suppressing effects as NPL4, an adaptor of p97 (also known as VCP) segregase, which is essential for the turnover of proteins involved in multiple regulatory and stress-response pathways in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp_sAn4LKB57Vg90H21EOLACvtfcHk0ljmP4CdVHeLjA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFGlsbrN&md5=b53235723e8478ab49b7d569d0dd35ac</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature25016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25016%26sid%3Dliteratum%253Aachs%26aulast%3DSkrott%26aufirst%3DZ.%26aulast%3DMistrik%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DK.%2BK.%26aulast%3DFriis%26aufirst%3DS.%26aulast%3DMajera%26aufirst%3DD.%26aulast%3DGursky%26aufirst%3DJ.%26aulast%3DOzdian%26aufirst%3DT.%26aulast%3DBartkova%26aufirst%3DJ.%26aulast%3DTuri%26aufirst%3DZ.%26aulast%3DMoudry%26aufirst%3DP.%26aulast%3DKraus%26aufirst%3DM.%26aulast%3DMichalova%26aufirst%3DM.%26aulast%3DVaclavkova%26aufirst%3DJ.%26aulast%3DDzubak%26aufirst%3DP.%26aulast%3DVrobel%26aufirst%3DI.%26aulast%3DPouckova%26aufirst%3DP.%26aulast%3DSedlacek%26aufirst%3DJ.%26aulast%3DMiklovicova%26aufirst%3DA.%26aulast%3DKutt%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMattova%26aufirst%3DJ.%26aulast%3DDriessen%26aufirst%3DC.%26aulast%3DDou%26aufirst%3DQ.%2BP.%26aulast%3DOlsen%26aufirst%3DJ.%26aulast%3DHajduch%26aufirst%3DM.%26aulast%3DCvek%26aufirst%3DB.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DAlcohol-Abuse%2520Drug%2520Disulfiram%2520Targets%2520Cancer%2520Via%2520p97%2520Segregase%2520Adaptor%2520NPL4%26jtitle%3DNature%26date%3D2017%26volume%3D552%26spage%3D194%26doi%3D10.1038%2Fnature25016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogarth, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlatt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haenisch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griswold, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amory, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isoherranen, N.</span></span> <span> </span><span class="NLM_article-title">Importance of ALDH1A Enzymes in Determining Human Testicular Retinoic Acid Concentrations</span>. <i>J. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">342</span>â <span class="NLM_lpage">357</span>, <span class="refDoi">Â DOI: 10.1194/jlr.M054718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1194%2Fjlr.M054718" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2015&pages=342-357&issue=2&author=S.+L.+Arnoldauthor=T.+Kentauthor=C.+A.+Hogarthauthor=S.+Schlattauthor=B.+Prasadauthor=M.+Haenischauthor=T.+Walshauthor=C.+H.+Mullerauthor=M.+D.+Griswoldauthor=J.+K.+Amoryauthor=N.+Isoherranen&title=Importance+of+ALDH1A+Enzymes+in+Determining+Human+Testicular+Retinoic+Acid+Concentrations&doi=10.1194%2Fjlr.M054718"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M054718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M054718%26sid%3Dliteratum%253Aachs%26aulast%3DArnold%26aufirst%3DS.%2BL.%26aulast%3DKent%26aufirst%3DT.%26aulast%3DHogarth%26aufirst%3DC.%2BA.%26aulast%3DSchlatt%26aufirst%3DS.%26aulast%3DPrasad%26aufirst%3DB.%26aulast%3DHaenisch%26aufirst%3DM.%26aulast%3DWalsh%26aufirst%3DT.%26aulast%3DMuller%26aufirst%3DC.%2BH.%26aulast%3DGriswold%26aufirst%3DM.%2BD.%26aulast%3DAmory%26aufirst%3DJ.%2BK.%26aulast%3DIsoherranen%26aufirst%3DN.%26atitle%3DImportance%2520of%2520ALDH1A%2520Enzymes%2520in%2520Determining%2520Human%2520Testicular%2520Retinoic%2520Acid%2520Concentrations%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2015%26volume%3D56%26issue%3D2%26spage%3D342%26epage%3D357%26doi%3D10.1194%2Fjlr.M054718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oster, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilburn, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegal, F. P.</span></span> <span> </span><span class="NLM_article-title">Chemically Induced Congenital Thymic Dysgenesis in the Rat: a Model of the DiGeorge Syndrome</span>. <i>Clin. Immunol. Immunopathol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">128</span>â <span class="NLM_lpage">134</span>, <span class="refDoi">Â DOI: 10.1016/0090-1229(83)90196-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2F0090-1229%2883%2990196-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=6872357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A280%3ADyaL3s3mtVSrsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1983&pages=128-134&issue=1&author=G.+Osterauthor=K.+H.+Kilburnauthor=F.+P.+Siegal&title=Chemically+Induced+Congenital+Thymic+Dysgenesis+in+the+Rat%3A+a+Model+of+the+DiGeorge+Syndrome&doi=10.1016%2F0090-1229%2883%2990196-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced congenital thymic dysgenesis in the rat: a model of the DiGeorge syndrome</span></div><div class="casAuthors">Oster G; Kilburn K H; Siegal F P</div><div class="citationInfo"><span class="NLM_cas:title">Clinical immunology and immunopathology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">128-34</span>
        ISSN:<span class="NLM_cas:issn">0090-1229</span>.
    </div><div class="casAbstract">The DiGeorge syndrome, a variable complex of thymic aplasia, congenital heart disease, hypoparathyroidism, and anomalies of the face and neck, is thought to result from exposure to teratogenic agents.  A group of congenital defects closely resembling this syndrome can be produced in newborn rats by the administration of the fat-soluble zinc chelating agent WIN 18,446, a bis-dichloroacetylamine.  This drug, nontoxic to adult animals, is a powerful teratogen when administered to pregnant rats during days 9-12 of the 21-day gestation period.  Our animal data suggest that the human syndrome results from exposure in utero to agents like WIN 18,446, which damage the fetus during a critical period of organogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD3_j8PryB7zAEfvC7EH1LfW6udTcc2eayEVtySwHQrLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3s3mtVSrsA%253D%253D&md5=6a30bdb6cac84c93af43abc3d80ac78b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2F0090-1229%2883%2990196-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0090-1229%252883%252990196-4%26sid%3Dliteratum%253Aachs%26aulast%3DOster%26aufirst%3DG.%26aulast%3DKilburn%26aufirst%3DK.%2BH.%26aulast%3DSiegal%26aufirst%3DF.%2BP.%26atitle%3DChemically%2520Induced%2520Congenital%2520Thymic%2520Dysgenesis%2520in%2520the%2520Rat%253A%2520a%2520Model%2520of%2520the%2520DiGeorge%2520Syndrome%26jtitle%3DClin.%2520Immunol.%2520Immunopathol.%26date%3D1983%26volume%3D28%26issue%3D1%26spage%3D128%26epage%3D134%26doi%3D10.1016%2F0090-1229%2883%2990196-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Development of a High-Throughput In Vitro Assay to Identify Selective Inhibitors for Human ALDH1A1</span>. <i>Chem.-Biol. Interact.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>234</i></span> (<span class="NLM_issue">0</span>),  <span class="NLM_fpage">29</span>â <span class="NLM_lpage">37</span>, <span class="refDoi">Â DOI: 10.1016/j.cbi.2014.10.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.cbi.2014.10.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25450233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2gsbzP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2015&pages=29-37&issue=0&author=C.+A.+Morganauthor=T.+D.+Hurley&title=Development+of+a+High-Throughput+In+Vitro+Assay+to+Identify+Selective+Inhibitors+for+Human+ALDH1A1&doi=10.1016%2Fj.cbi.2014.10.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a high-throughput in vitro assay to identify selective inhibitors for human ALDH1A1</span></div><div class="casAuthors">Morgan, Cynthia A.; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29-37</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">The human aldehyde dehydrogenase (ALDH) superfamily consists of at least 19 enzymes that metabolize endogenous and exogenous aldehydes.  Currently, there are no com. available inhibitors that target ALDH1A1 but have little to no effect on the structurally and functionally similar ALDH2.  Here we present the first human ALDH1A1 structure, as the apo-enzyme and in complex with its cofactor NADH to a resoln. of 1.75 and 2.1 Ã, respectfully.  Structural comparisons of the cofactor binding sites in ALDH1A1 with other closely related ALDH enzymes illustrate a high degree of similarity.  In order to minimize discovery of compds. that inhibit both isoenzymes by interfering with their conserved cofactor binding sites, this study reports the use of an in vitro, NAD+-independent, esterase-based high-throughput screen (HTS) of 64,000 compds. to discover novel, selective inhibitors of ALDH1A1.  We describe 256 hits that alter the esterase activity of ALDH1A1.  The effects on aldehyde oxidn. of 67 compds. were further analyzed, with 30 selectively inhibiting ALDH1A1 compared to ALDH2 and ALDH3A1.  One compd. inhibited ALDH1A1 and ALDH2, while another inhibited ALDH1A1, ALDH2, and the more distantly related ALDH3A1.  The results presented here indicate that this in vitro enzyme activity screening protocol successfully identified ALDH1A1 inhibitors with a high degree of isoenzyme selectivity.  The compds. identified via this screen plus the screening methodol. itself represent a starting point for the development of highly potent and selective inhibitors of ALDH1A1 that may be utilized to better understand the role of this enzyme in both normal and disease states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0H8M4JN5aC7Vg90H21EOLACvtfcHk0liqpFcfGKPVBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2gsbzP&md5=64ce33cd18ae04a4ef8796bd9c4474a7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.10.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.10.028%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520a%2520High-Throughput%2520In%2520Vitro%2520Assay%2520to%2520Identify%2520Selective%2520Inhibitors%2520for%2520Human%2520ALDH1A1%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D2015%26volume%3D234%26issue%3D0%26spage%3D29%26epage%3D37%26doi%3D10.1016%2Fj.cbi.2014.10.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Characterization of Two Distinct Structural Classes of Selective Aldehyde Dehydrogenase 1A1 Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1964</span>â <span class="NLM_lpage">1975</span>, <span class="refDoi">Â DOI: 10.1021/jm501900s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501900s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12ltr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1964-1975&issue=4&author=C.+A.+Morganauthor=T.+D.+Hurley&title=Characterization+of+Two+Distinct+Structural+Classes+of+Selective+Aldehyde+Dehydrogenase+1A1+Inhibitors&doi=10.1021%2Fjm501900s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Two Distinct Structural Classes of Selective Aldehyde Dehydrogenase 1A1 Inhibitors</span></div><div class="casAuthors">Morgan, Cynthia A.; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1964-1975</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDH) catalyze the irreversible oxidn. of aldehydes to their corresponding carboxylic acid.  Alterations in ALDH1A1 activity are assocd. with such diverse diseases as cancer, Parkinson's disease, obesity, and cataracts.  Inhibitors of ALDH1A1 could aid in illuminating the role of this enzyme in disease processes.  However, there are no com. available selective inhibitors for ALDH1A1.  Here we characterize two distinct chem. classes of inhibitors that are selective for human ALDH1A1 compared to eight other ALDH isoenzymes.  The prototypical members of each structural class, CM026 and CM037, exhibit submicromolar inhibition consts. but have different mechanisms of inhibition.  The crystal structures of these compds. bound to ALDH1A1 demonstrate that they bind within the aldehyde binding pocket of ALDH1A1 and exploit the presence of a unique glycine residue to achieve their selectivity.  These two novel and selective ALDH1A1 inhibitors may serve as chem. tools to better understand the contributions of ALDH1A1 to normal biol. and to disease states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUpN6wS1yVQLVg90H21EOLACvtfcHk0liqpFcfGKPVBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12ltr0%253D&md5=102a2c49cbf0bc89f620934e8b7872f2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm501900s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501900s%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DCharacterization%2520of%2520Two%2520Distinct%2520Structural%2520Classes%2520of%2520Selective%2520Aldehyde%2520Dehydrogenase%25201A1%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D4%26spage%3D1964%26epage%3D1975%26doi%3D10.1021%2Fjm501900s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal-Nag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of NCT-501, a Potent and Selective Theophylline-Based Inhibitor of Aldehyde Dehydrogenase 1A1 (ALDH1A1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">5967</span>â <span class="NLM_lpage">5978</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00577</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00577" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5967-5978&issue=15&author=S.+M.+Yangauthor=A.+Yasgarauthor=B.+Millerauthor=M.+Lal-Nagauthor=K.+Brimacombeauthor=X.+Huauthor=H.+Sunauthor=A.+Wangauthor=X.+Xuauthor=K.+Nguyenauthor=U.+Oppermannauthor=M.+Ferrerauthor=V.+Vasiliouauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+J.+Maloney&title=Discovery+of+NCT-501%2C+a+Potent+and+Selective+Theophylline-Based+Inhibitor+of+Aldehyde+Dehydrogenase+1A1+%28ALDH1A1%29&doi=10.1021%2Facs.jmedchem.5b00577"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00577%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%2BM.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DB.%26aulast%3DLal-Nag%26aufirst%3DM.%26aulast%3DBrimacombe%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520NCT-501%252C%2520a%2520Potent%2520and%2520Selective%2520Theophylline-Based%2520Inhibitor%2520of%2520Aldehyde%2520Dehydrogenase%25201A1%2520%2528ALDH1A1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D15%26spage%3D5967%26epage%3D5978%26doi%3D10.1021%2Facs.jmedchem.5b00577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danchik, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baljinnyam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheff, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal-Nag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunford, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Orally Bioavailable, Quinoline-Based Aldehyde Dehydrogenase 1A1 (ALDH1A1) Inhibitors with Potent Cellular Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">4883</span>â <span class="NLM_lpage">4903</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00270</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00270" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps12msL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4883-4903&issue=11&author=S.+M.+Yangauthor=N.+J.+Martinezauthor=A.+Yasgarauthor=C.+Danchikauthor=C.+Johanssonauthor=Y.+Wangauthor=B.+Baljinnyamauthor=A.+Q.+Wangauthor=X.+Xuauthor=P.+Shahauthor=D.+Cheffauthor=X.+S.+Wangauthor=J.+Rothauthor=M.+Lal-Nagauthor=J.+E.+Dunfordauthor=U.+Oppermannauthor=V.+Vasiliouauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+J.+Maloney&title=Discovery+of+Orally+Bioavailable%2C+Quinoline-Based+Aldehyde+Dehydrogenase+1A1+%28ALDH1A1%29+Inhibitors+with+Potent+Cellular+Activity&doi=10.1021%2Facs.jmedchem.8b00270"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Orally Bioavailable, Quinoline-Based Aldehyde Dehydrogenase 1A1 (ALDH1A1) Inhibitors with Potent Cellular Activity</span></div><div class="casAuthors">Yang, Shyh-Ming; Martinez, Natalia J.; Yasgar, Adam; Danchik, Carina; Johansson, Catrine; Wang, Yuhong; Baljinnyam, Bolormaa; Wang, Amy Q.; Xu, Xin; Shah, Pranav; Cheff, Dorian; Wang, Xinran S.; Roth, Jacob; Lal-Nag, Madhu; Dunford, James E.; Oppermann, Udo; Vasiliou, Vasilis; Simeonov, Anton; Jadhav, Ajit; Maloney, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4883-4903</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDHs) are responsible for the metab. of aldehydes (exogenous and endogenous) and possess vital physiol. and toxicol. functions in areas such as CNS, inflammation, metabolic disorders, and cancers.  Overexpression of certain ALDHs (e.g., ALDH1A1) is an important biomarker in cancers and cancer stem cells (CSCs) indicating the potential need for the identification and development of small mol. ALDH inhibitors.  Herein, a newly designed series of quinoline-based analogs of ALDH1A1 inhibitors is described.  Extensive medicinal chem. optimization and biol. characterization led to the identification of analogs with significantly improved enzymic and cellular ALDH inhibition.  Selected analogs, e.g., 86 (NCT-505) and 91 (NCT-506), demonstrated target engagement in a cellular thermal shift assay (CETSA), inhibited the formation of 3D spheroid cultures of OV-90 cancer cells, and potentiated the cytotoxicity of paclitaxel in SKOV-3-TR, a paclitaxel resistant ovarian cancer cell line.  Lead compds. also exhibit high specificity over other ALDH isoenzymes and unrelated dehydrogenases.  The in vitro ADME profiles and pharmacokinetic evaluation of selected analogs are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMdOrSgvA10rVg90H21EOLACvtfcHk0liqpFcfGKPVBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps12msL8%253D&md5=6f63e6c9803cb8f0e113fb1f81bebf8a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00270%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%2BM.%26aulast%3DMartinez%26aufirst%3DN.%2BJ.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DDanchik%26aufirst%3DC.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBaljinnyam%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DCheff%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%2BS.%26aulast%3DRoth%26aufirst%3DJ.%26aulast%3DLal-Nag%26aufirst%3DM.%26aulast%3DDunford%26aufirst%3DJ.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520Orally%2520Bioavailable%252C%2520Quinoline-Based%2520Aldehyde%2520Dehydrogenase%25201A1%2520%2528ALDH1A1%2529%2520Inhibitors%2520with%2520Potent%2520Cellular%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D11%26spage%3D4883%26epage%3D4903%26doi%3D10.1021%2Facs.jmedchem.8b00270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span>; <span class="NLM_string-name">Mao, L.</span>; <span class="NLM_string-name">Wang, X.</span>; <span class="NLM_string-name">Xu, X.</span>; <span class="NLM_string-name">Zhao, L.</span></span> <i>Pyrazolopyrimidine Derivatives and Uses as Anticancer Agents</i>. Patent No. <span class="NLM_patent">WO2012097196A1</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=J.+Liu&author=L.+Mao&author=X.+Wang&author=X.+Xu&author=L.+Zhao&title=Pyrazolopyrimidine+Derivatives+and+Uses+as+Anticancer+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26jtitle%3DPyrazolopyrimidine%2520Derivatives%2520and%2520Uses%2520as%2520Anticancer%2520Agents%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffaelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barretta, G. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolaro, L. M.</span></span> <span> </span><span class="NLM_article-title">Potential of Alkyl Orthoformates as Alkylating Agents of Non-Electron Rich Nitrogen Heterocycles</span>. <i>J. Heterocycl. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1149</span>â <span class="NLM_lpage">1158</span>, <span class="refDoi">Â DOI: 10.1002/jhet.5570320409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fjhet.5570320409" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1995&pages=1149-1158&issue=4&author=S.+Guccioneauthor=F.+Russoauthor=A.+Raffaelliauthor=G.+U.+Barrettaauthor=S.+Pucciauthor=L.+M.+Scolaro&title=Potential+of+Alkyl+Orthoformates+as+Alkylating+Agents+of+Non-Electron+Rich+Nitrogen+Heterocycles&doi=10.1002%2Fjhet.5570320409"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570320409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570320409%26sid%3Dliteratum%253Aachs%26aulast%3DGuccione%26aufirst%3DS.%26aulast%3DRusso%26aufirst%3DF.%26aulast%3DRaffaelli%26aufirst%3DA.%26aulast%3DBarretta%26aufirst%3DG.%2BU.%26aulast%3DPucci%26aufirst%3DS.%26aulast%3DScolaro%26aufirst%3DL.%2BM.%26atitle%3DPotential%2520of%2520Alkyl%2520Orthoformates%2520as%2520Alkylating%2520Agents%2520of%2520Non-Electron%2520Rich%2520Nitrogen%2520Heterocycles%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D1995%26volume%3D32%26issue%3D4%26spage%3D1149%26epage%3D1158%26doi%3D10.1002%2Fjhet.5570320409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, B. K.</span></span> <span> </span><span class="NLM_article-title">Molecular Iodine Mediated Preparation of Isothiocyanates from Dithiocarbamic Acid Salts</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2009</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1849</span>â <span class="NLM_lpage">1851</span>, <span class="refDoi">Â DOI: 10.1002/ejoc.200801270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fejoc.200801270" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2009&publication_year=2009&pages=1849-1851&issue=12&author=J.+Nathauthor=H.+Ghoshauthor=R.+Yellaauthor=B.+K.+Patel&title=Molecular+Iodine+Mediated+Preparation+of+Isothiocyanates+from+Dithiocarbamic+Acid+Salts&doi=10.1002%2Fejoc.200801270"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fejoc.200801270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.200801270%26sid%3Dliteratum%253Aachs%26aulast%3DNath%26aufirst%3DJ.%26aulast%3DGhosh%26aufirst%3DH.%26aulast%3DYella%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520Iodine%2520Mediated%2520Preparation%2520of%2520Isothiocyanates%2520from%2520Dithiocarbamic%2520Acid%2520Salts%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2009%26volume%3D2009%26issue%3D12%26spage%3D1849%26epage%3D1851%26doi%3D10.1002%2Fejoc.200801270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinzler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilbert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelstein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preisinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedge, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKechnie, D.</span></span> <span> </span><span class="NLM_article-title">Identification of FAP Locus Genes from Chromosome 5q21</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>253</i></span> (<span class="NLM_issue">5020</span>),  <span class="NLM_fpage">661</span>â <span class="NLM_lpage">665</span>, <span class="refDoi">Â DOI: 10.1126/science.1651562</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1126%2Fscience.1651562" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1991&pages=661-665&issue=5020&author=K.+Kinzlerauthor=M.+Nilbertauthor=L.+Suauthor=B.+Vogelsteinauthor=T.+Bryanauthor=D.+Levyauthor=K.+Smithauthor=A.+Preisingerauthor=P.+Hedgeauthor=D.+McKechnie&title=Identification+of+FAP+Locus+Genes+from+Chromosome+5q21&doi=10.1126%2Fscience.1651562"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1126%2Fscience.1651562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1651562%26sid%3Dliteratum%253Aachs%26aulast%3DKinzler%26aufirst%3DK.%26aulast%3DNilbert%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DBryan%26aufirst%3DT.%26aulast%3DLevy%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DK.%26aulast%3DPreisinger%26aufirst%3DA.%26aulast%3DHedge%26aufirst%3DP.%26aulast%3DMcKechnie%26aufirst%3DD.%26atitle%3DIdentification%2520of%2520FAP%2520Locus%2520Genes%2520from%2520Chromosome%25205q21%26jtitle%3DScience%26date%3D1991%26volume%3D253%26issue%3D5020%26spage%3D661%26epage%3D665%26doi%3D10.1126%2Fscience.1651562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span>MarvinSketch
Calculator Plugins were used for ADME predictions.  <i>Marvin 15.6.15</i>, <span class="NLM_year">2015</span>, <span class="NLM_publisher-name">ChemAxon</span>. See the following: <a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a> (accessed
June 5, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MarvinSketch%0ACalculator+Plugins+were+used+for+ADME+predictions.+Marvin+15.6.15%2C+2015%2C+ChemAxon.+See+the+following%3A+http%3A%2F%2Fwww.chemaxon.com+%28accessed%0AJune+5%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMarvin%252015.6.15%26date%3D2015%26pub%3DChemAxon" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikles, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amory, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of ALDH1A2 Inhibition by Irreversible and Reversible Small Molecule Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">582</span>â <span class="NLM_lpage">590</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.7b00685</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00685" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOgt7bK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=582-590&issue=3&author=Y.+Chenauthor=J.+Y.+Zhuauthor=K.+H.+Hongauthor=D.+C.+Miklesauthor=G.+I.+Georgauthor=A.+S.+Goldsteinauthor=J.+K.+Amoryauthor=E.+Schonbrunn&title=Structural+Basis+of+ALDH1A2+Inhibition+by+Irreversible+and+Reversible+Small+Molecule+Inhibitors&doi=10.1021%2Facschembio.7b00685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of ALDH1A2 Inhibition by Irreversible and Reversible Small Molecule Inhibitors</span></div><div class="casAuthors">Chen, Yan; Zhu, Jin-Yi; Hong, Kwon Ho; Mikles, David C.; Georg, Gunda I.; Goldstein, Alex S.; Amory, John K.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">582-590</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymes of the ALDH1A subfamily of aldehyde dehydrogenases are crucial in regulating retinoic acid (RA) signaling and have received attention as potential drug targets.  ALDH1A2 is the primary RA-synthesizing enzyme in mammalian spermatogenesis and is therefore considered a viable drug target for male contraceptive development.  However, only a small no. of ALDH1A2 inhibitors have been reported, and information on the structure of ALDH1A2 was limited to the NAD-liganded enzyme void of substrate or inhibitors.  Herein, the authors describe the mechanism of action of structurally unrelated reversible and irreversible inhibitors of human ALDH1A2 using direct binding studies and x-ray crystallog.  All inhibitors bind to the active sites of tetrameric ALDH1A2.  Compd. WIN18,446 covalently reacts with the side chain of the catalytic residue Cys 320, resulting in a chiral adduct in (R) configuration.  The covalent adduct directly affects the neighboring NAD mol., which assumes a contracted conformation suboptimal for the dehydrogenase reaction.  The reversible inhibitors interact predominantly through direct hydrogen bonding interactions with residues in the vicinity of Cys 320 without affecting NAD.  Upon interaction with inhibitors, a large flexible loop assumes regular structure, thereby shielding the active site from solvent.  The precise knowledge of the binding modes provides a new framework for the rational design of novel inhibitors of ALDH1A2 with improved potency and selectivity profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprC4rddNQ7u7Vg90H21EOLACvtfcHk0liCZkAtGP3Ivg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOgt7bK&md5=dff6eef4a876a603048f3ccd53f60dee</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00685%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DHong%26aufirst%3DK.%2BH.%26aulast%3DMikles%26aufirst%3DD.%2BC.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DGoldstein%26aufirst%3DA.%2BS.%26aulast%3DAmory%26aufirst%3DJ.%2BK.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DStructural%2520Basis%2520of%2520ALDH1A2%2520Inhibition%2520by%2520Irreversible%2520and%2520Reversible%2520Small%2520Molecule%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D3%26spage%3D582%26epage%3D590%26doi%3D10.1021%2Facschembio.7b00685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beaufort, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helmijr, J. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piskorz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoogstraat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruigrok-Ritstier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besselink, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murtaza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Ijcken, W. F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koudijs, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berns, E. M. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleman, J.</span></span> <span> </span><span class="NLM_article-title">Ovarian Cancer Cell Line Panel (OCCP): Clinical Importance of In Vitro Morphological Subtypes</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">e103988</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0103988</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0103988" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25230021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslaqsbbE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e103988&issue=9&author=C.+M.+Beaufortauthor=J.+C.+A.+Helmijrauthor=A.+M.+Piskorzauthor=M.+Hoogstraatauthor=K.+Ruigrok-Ritstierauthor=N.+Besselinkauthor=M.+Murtazaauthor=W.+F.+J.+van+Ijckenauthor=A.+A.+J.+Heineauthor=M.+Smidauthor=M.+J.+Koudijsauthor=J.+D.+Brentonauthor=E.+M.+J.+J.+Bernsauthor=J.+Helleman&title=Ovarian+Cancer+Cell+Line+Panel+%28OCCP%29%3A+Clinical+Importance+of+In+Vitro+Morphological+Subtypes&doi=10.1371%2Fjournal.pone.0103988"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian Cancer Cell Line Panel (OCCP): clinical importance of in vitro morphological subtypes</span></div><div class="casAuthors">Beaufort, Corine M.; Helmijr, Jean C. A.; Piskorz, Anna M.; Hoogstraat, Marlous; Ruigrok-Ritstier, Kirsten; Besselink, Nicolle; Murtaza, Muhammed; van Ijcken, Wilfred F. J.; Heine, Anouk A. J.; Smid, Marcel; Koudijs, Marco J.; Brenton, James D.; Berns, Els M. J. J.; Helleman, Jozien</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e103988/1-e103988/16, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Epithelial ovarian cancer is a highly heterogeneous disease and remains the most lethal gynaecol. malignancy in the Western world.  Therapeutic approaches need to account for inter-patient and intra-tumoral heterogeneity and detailed characterization of in vitro models representing the different histol. and mol. ovarian cancer subtypes is crit. to enable reliable preclin. testing.  There are approx. 100 publicly available ovarian cancer cell lines but their cellular and mol. characteristics are largely undescribed.  We have characterized 39 ovarian cancer cell lines under uniform conditions for growth characteristics, mRNA/microRNA expression, exon sequencing, drug response for clin.-relevant therapeutics and collated all available information on the original clin. features and site of origin.  We tested for statistical assocns. between the cellular and mol. features of the lines and clin. features.  Of the 39 ovarian cancer cell lines, 14 were assigned as high-grade serous, four serous-type, one low-grade serous and 20 non-serous type.  Three morphol. subtypes: Epithelial (n = 21), Round (n = 7) and Spindle (n = 12) were identified that showed distinct biol. and mol. characteristics, including overexpression of cell movement and migration-assocd. genes in the Spindle subtype.  Comparison with the original clin. data showed assocn. of the spindle-like tumors with metastasis, advanced stage, suboptimal debulking and poor prognosis.  In addn., the expression profiles of Spindle, Round and Epithelial morphologies clustered with the previously described C1-stromal, C5-mesenchymal and C4 ovarian subtype expression profiles resp.  Comprehensive profiling of 39 ovarian cancer cell lines under controlled, uniform conditions demonstrates clin. relevant cellular and genomic characteristics.  This data provides a rational basis for selecting models to develop specific treatment approaches for histol. and mol. subtypes of ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI3caC6_-9C7Vg90H21EOLACvtfcHk0lhbZ-6cw9LIvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslaqsbbE&md5=39a63787ab20dabe8003c08d1908a90a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0103988&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0103988%26sid%3Dliteratum%253Aachs%26aulast%3DBeaufort%26aufirst%3DC.%2BM.%26aulast%3DHelmijr%26aufirst%3DJ.%2BC.%2BA.%26aulast%3DPiskorz%26aufirst%3DA.%2BM.%26aulast%3DHoogstraat%26aufirst%3DM.%26aulast%3DRuigrok-Ritstier%26aufirst%3DK.%26aulast%3DBesselink%26aufirst%3DN.%26aulast%3DMurtaza%26aufirst%3DM.%26aulast%3Dvan%2BIjcken%26aufirst%3DW.%2BF.%2BJ.%26aulast%3DHeine%26aufirst%3DA.%2BA.%2BJ.%26aulast%3DSmid%26aufirst%3DM.%26aulast%3DKoudijs%26aufirst%3DM.%2BJ.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DBerns%26aufirst%3DE.%2BM.%2BJ.%2BJ.%26aulast%3DHelleman%26aufirst%3DJ.%26atitle%3DOvarian%2520Cancer%2520Cell%2520Line%2520Panel%2520%2528OCCP%2529%253A%2520Clinical%2520Importance%2520of%2520In%2520Vitro%2520Morphological%2520Subtypes%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3De103988%26doi%3D10.1371%2Fjournal.pone.0103988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blayney, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCluggage, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capes-Davis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harkin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gourley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, R. D.</span></span> <span> </span><span class="NLM_article-title">Prior Knowledge Transfer Across Transcriptional Data Sets and Technologies Using Compositional Statistics Yields New Mislabelled Ovarian Cell Line</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">e137</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkw578</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1093%2Fnar%2Fgkw578" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e137&issue=17&author=J.+K.+Blayneyauthor=T.+Davisonauthor=N.+McCabeauthor=S.+Walkerauthor=K.+Keatingauthor=T.+Delaneyauthor=C.+Greenanauthor=A.+R.+Williamsauthor=W.+G.+McCluggageauthor=A.+Capes-Davisauthor=D.+P.+Harkinauthor=C.+Gourleyauthor=R.+D.+Kennedy&title=Prior+Knowledge+Transfer+Across+Transcriptional+Data+Sets+and+Technologies+Using+Compositional+Statistics+Yields+New+Mislabelled+Ovarian+Cell+Line&doi=10.1093%2Fnar%2Fgkw578"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw578&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw578%26sid%3Dliteratum%253Aachs%26aulast%3DBlayney%26aufirst%3DJ.%2BK.%26aulast%3DDavison%26aufirst%3DT.%26aulast%3DMcCabe%26aufirst%3DN.%26aulast%3DWalker%26aufirst%3DS.%26aulast%3DKeating%26aufirst%3DK.%26aulast%3DDelaney%26aufirst%3DT.%26aulast%3DGreenan%26aufirst%3DC.%26aulast%3DWilliams%26aufirst%3DA.%2BR.%26aulast%3DMcCluggage%26aufirst%3DW.%2BG.%26aulast%3DCapes-Davis%26aufirst%3DA.%26aulast%3DHarkin%26aufirst%3DD.%2BP.%26aulast%3DGourley%26aufirst%3DC.%26aulast%3DKennedy%26aufirst%3DR.%2BD.%26atitle%3DPrior%2520Knowledge%2520Transfer%2520Across%2520Transcriptional%2520Data%2520Sets%2520and%2520Technologies%2520Using%2520Compositional%2520Statistics%2520Yields%2520New%2520Mislabelled%2520Ovarian%2520Cell%2520Line%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26issue%3D17%26spage%3De137%26doi%3D10.1093%2Fnar%2Fgkw578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, M.</span></span> <span> </span><span class="NLM_article-title">Prognostic Value of Cancer Stem Cell Marker CD133 in Ovarian Cancer: a Meta-Analysis</span>. <i>International Journal of Clinical and Experimental Medicine</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">3080</span>â <span class="NLM_lpage">3088</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=3080-3088&issue=3&author=Q.+Zhouauthor=A.+Chenauthor=H.+Songauthor=J.+Taoauthor=H.+Yangauthor=M.+Zuo&title=Prognostic+Value+of+Cancer+Stem+Cell+Marker+CD133+in+Ovarian+Cancer%3A+a+Meta-Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZuo%26aufirst%3DM.%26atitle%3DPrognostic%2520Value%2520of%2520Cancer%2520Stem%2520Cell%2520Marker%2520CD133%2520in%2520Ovarian%2520Cancer%253A%2520a%2520Meta-Analysis%26jtitle%3DInternational%2520Journal%2520of%2520Clinical%2520and%2520Experimental%2520Medicine%26date%3D2015%26volume%3D8%26issue%3D3%26spage%3D3080%26epage%3D3088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanabe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tezuka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunisada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, S.</span></span> <span> </span><span class="NLM_article-title">A More Efficient Method to Generate Integration-Free Human iPS Cells</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">409</span>â <span class="NLM_lpage">412</span>, <span class="refDoi">Â DOI: 10.1038/nmeth.1591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fnmeth.1591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21460823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFahtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2011&pages=409-412&issue=5&author=K.+Okitaauthor=Y.+Matsumuraauthor=Y.+Satoauthor=A.+Okadaauthor=A.+Morizaneauthor=S.+Okamotoauthor=H.+Hongauthor=M.+Nakagawaauthor=K.+Tanabeauthor=K.+Tezukaauthor=T.+Shibataauthor=T.+Kunisadaauthor=M.+Takahashiauthor=J.+Takahashiauthor=H.+Sajiauthor=S.+Yamanaka&title=A+More+Efficient+Method+to+Generate+Integration-Free+Human+iPS+Cells&doi=10.1038%2Fnmeth.1591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A more efficient method to generate integration-free human iPS cells</span></div><div class="casAuthors">Okita, Keisuke; Matsumura, Yasuko; Sato, Yoshiko; Okada, Aki; Morizane, Asuka; Okamoto, Satoshi; Hong, Hyenjong; Nakagawa, Masato; Tanabe, Koji; Tezuka, Ken-ichi; Shibata, Toshiyuki; Kunisada, Takahiro; Takahashi, Masayo; Takahashi, Jun; Saji, Hiroh; Yamanaka, Shinya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report a simple method, using p53 suppression and nontransforming L-Myc, to generate human induced pluripotent stem cells (iPSCs) with episomal plasmid vectors.  We generated human iPSCs from multiple donors, including two putative human leukocyte antigen (HLA)-homozygous donors who match â¼20% of the Japanese population at major HLA loci; most iPSCs are integrated transgene-free.  This method may provide iPSCs suitable for autologous and allologous stem-cell therapy in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN2ueECgZQZbVg90H21EOLACvtfcHk0lhbZ-6cw9LIvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFahtrc%253D&md5=9d48be9c24ff58bf9ab862ba9ce3e2b0</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1591%26sid%3Dliteratum%253Aachs%26aulast%3DOkita%26aufirst%3DK.%26aulast%3DMatsumura%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DY.%26aulast%3DOkada%26aufirst%3DA.%26aulast%3DMorizane%26aufirst%3DA.%26aulast%3DOkamoto%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DTanabe%26aufirst%3DK.%26aulast%3DTezuka%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DT.%26aulast%3DKunisada%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DJ.%26aulast%3DSaji%26aufirst%3DH.%26aulast%3DYamanaka%26aufirst%3DS.%26atitle%3DA%2520More%2520Efficient%2520Method%2520to%2520Generate%2520Integration-Free%2520Human%2520iPS%2520Cells%26jtitle%3DNat.%2520Methods%26date%3D2011%26volume%3D8%26issue%3D5%26spage%3D409%26epage%3D412%26doi%3D10.1038%2Fnmeth.1591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bretz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noske, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbe-Hofmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlange, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salnikov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moldenhauer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristiansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altevogt, P.</span></span> <span> </span><span class="NLM_article-title">CD24 Promotes Tumor Cell Invasion by Suppressing Tissue Factor Pathway Inhibitor-2 (TFPI-2) in a c-Src-Dependent Fashion</span>. <i>Clin. Exp. Metastasis</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">27</span>â <span class="NLM_lpage">38</span>, <span class="refDoi">Â DOI: 10.1007/s10585-011-9426-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1007%2Fs10585-011-9426-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21984372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsF2qsr7F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=27-38&issue=1&author=N.+Bretzauthor=A.+Noskeauthor=S.+Kellerauthor=N.+Erbe-Hofmannauthor=T.+Schlangeauthor=A.+V.+Salnikovauthor=G.+Moldenhauerauthor=G.+Kristiansenauthor=P.+Altevogt&title=CD24+Promotes+Tumor+Cell+Invasion+by+Suppressing+Tissue+Factor+Pathway+Inhibitor-2+%28TFPI-2%29+in+a+c-Src-Dependent+Fashion&doi=10.1007%2Fs10585-011-9426-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">CD24 promotes tumor cell invasion by suppressing tissue factor pathway inhibitor-2 (TFPI-2) in a c-Src-dependent fashion</span></div><div class="casAuthors">Bretz, Niko; Noske, Aurelia; Keller, Sascha; Erbe-Hofmann, Natalie; Schlange, Thomas; Salnikov, Alexei V.; Moldenhauer, Gerd; Kristiansen, Glen; Altevogt, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Clinical & Experimental Metastasis</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">CEXMD2</span>;
        ISSN:<span class="NLM_cas:issn">0262-0898</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">CD24 is a glycosyl-phosphatidylinositol-anchored protein with mucin-type structure that resides exclusively in membrane microdomains.  CD24 is often highly expressed in carcinomas and correlates with poor prognosis.  Exptl., the over-expression or depletion of CD24 alters cell proliferation, adhesion, and invasion in vitro and tumor growth in vivo.  However, little is known about the mechanisms by which CD24 mediates these cellular effects.  Here we have studied the mechanism of CD24-dependent cell invasion using transient CD24 knock-down or over-expression in human cancer cell lines.  We show that CD24 depletion reduced tumor cell invasion and up-regulated expression of Tissue Factor Pathway Inhibitor 2 (TFPI-2), a potent inhibitor of extracellular matrix degrdn. that can block metastases formation and tumor cell invasion.  Over-expression of CD24 in A125 cells resulted in reduced TFPI-2 expression and enhanced invasion.  We provide evidence that the activity of c-Src is reduced upon CD24 knock-down.  The silencing of c-Src, similar to CD24, was able to enhance TFPI-2 expression and reduce tumor cell invasion.  An inverse expression of CD24 and TFPI-2 was obsd. by immunohistochem. anal. of primary breast cancers (N = 1,174).  TFPI-2 expression was highest in CD24 neg. samples and lowered with increasing CD24 expression.  Patients with a CD24 low/TFPI-2 high phenotype showed significantly better survival compared to CD24 high/TFPI-2 low patients.  Our results provide evidence that CD24 can regulate cell invasion via TFPI-2 and suggests a role of c-Src in this process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7X2BvIbJpwLVg90H21EOLACvtfcHk0lhbZ-6cw9LIvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsF2qsr7F&md5=d3f9718ee880e1c4b1bd7ee6917c9c66</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs10585-011-9426-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10585-011-9426-4%26sid%3Dliteratum%253Aachs%26aulast%3DBretz%26aufirst%3DN.%26aulast%3DNoske%26aufirst%3DA.%26aulast%3DKeller%26aufirst%3DS.%26aulast%3DErbe-Hofmann%26aufirst%3DN.%26aulast%3DSchlange%26aufirst%3DT.%26aulast%3DSalnikov%26aufirst%3DA.%2BV.%26aulast%3DMoldenhauer%26aufirst%3DG.%26aulast%3DKristiansen%26aufirst%3DG.%26aulast%3DAltevogt%26aufirst%3DP.%26atitle%3DCD24%2520Promotes%2520Tumor%2520Cell%2520Invasion%2520by%2520Suppressing%2520Tissue%2520Factor%2520Pathway%2520Inhibitor-2%2520%2528TFPI-2%2529%2520in%2520a%2520c-Src-Dependent%2520Fashion%26jtitle%3DClin.%2520Exp.%2520Metastasis%26date%3D2012%26volume%3D29%26issue%3D1%26spage%3D27%26epage%3D38%26doi%3D10.1007%2Fs10585-011-9426-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dylla, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beviglia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chartier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raval, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazetic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith-Berdan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewicki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurney, A. L.</span></span> <span> </span><span class="NLM_article-title">Colorectal Cancer Stem Cells are Enriched in Xenogeneic Tumors Following Chemotherapy</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">e2428</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0002428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0002428" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e2428&issue=6&author=S.+J.+Dyllaauthor=L.+Bevigliaauthor=I.+K.+Parkauthor=C.+Chartierauthor=J.+Ravalauthor=L.+Nganauthor=K.+Pickellauthor=J.+Aguilarauthor=S.+Lazeticauthor=S.+Smith-Berdanauthor=M.+F.+Clarkeauthor=T.+Hoeyauthor=J.+Lewickiauthor=A.+L.+Gurney&title=Colorectal+Cancer+Stem+Cells+are+Enriched+in+Xenogeneic+Tumors+Following+Chemotherapy&doi=10.1371%2Fjournal.pone.0002428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0002428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0002428%26sid%3Dliteratum%253Aachs%26aulast%3DDylla%26aufirst%3DS.%2BJ.%26aulast%3DBeviglia%26aufirst%3DL.%26aulast%3DPark%26aufirst%3DI.%2BK.%26aulast%3DChartier%26aufirst%3DC.%26aulast%3DRaval%26aufirst%3DJ.%26aulast%3DNgan%26aufirst%3DL.%26aulast%3DPickell%26aufirst%3DK.%26aulast%3DAguilar%26aufirst%3DJ.%26aulast%3DLazetic%26aufirst%3DS.%26aulast%3DSmith-Berdan%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DM.%2BF.%26aulast%3DHoey%26aufirst%3DT.%26aulast%3DLewicki%26aufirst%3DJ.%26aulast%3DGurney%26aufirst%3DA.%2BL.%26atitle%3DColorectal%2520Cancer%2520Stem%2520Cells%2520are%2520Enriched%2520in%2520Xenogeneic%2520Tumors%2520Following%2520Chemotherapy%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26issue%3D6%26spage%3De2428%26doi%3D10.1371%2Fjournal.pone.0002428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Throndset, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whelan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zare, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manger, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daridon, A.</span></span> <span> </span><span class="NLM_article-title">Microfluidic Device for Single-Cell Analysis</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>75</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">3581</span>â <span class="NLM_lpage">3586</span>, <span class="refDoi">Â DOI: 10.1021/ac0340758</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac0340758" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjs12ktr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=3581-3586&issue=14&author=A.+R.+Wheelerauthor=W.+R.+Throndsetauthor=R.+J.+Whelanauthor=A.+M.+Leachauthor=R.+N.+Zareauthor=Y.+H.+Liaoauthor=K.+Farrellauthor=I.+D.+Mangerauthor=A.+Daridon&title=Microfluidic+Device+for+Single-Cell+Analysis&doi=10.1021%2Fac0340758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Microfluidic device for single-cell analysis</span></div><div class="casAuthors">Wheeler, Aaron R.; Throndset, William R.; Whelan, Rebecca J.; Leach, Andrew M.; Zare, Richard N.; Liao, Yish Hann; Farrell, Kevin; Manger, Ian D.; Daridon, Antoine</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3581-3586</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have developed a novel microfluidic device constructed from poly(dimethylsiloxane) using multilayer soft lithog. technol. for the anal. of single cells.  The microfluidic network enables the passive and gentle sepn. of a single cell from the bulk cell suspension, and integrated valves and pumps enable the precise delivery of nanoliter vols. of reagents to that cell.  Various applications are demonstrated, including cell viability assays, ionophore-mediated intracellular Ca2+ flux measurements, and multistep receptor-mediated Ca2+ measurements.  These assays, and others, are achieved with significant improvements in reagent consumption, anal. time, and temporal resoln. over macroscale alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjEXGz2K_yFbVg90H21EOLACvtfcHk0lhHvvIPLKMeDQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjs12ktr0%253D&md5=7ca8c890337654cd79c400aa839ac4c9</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fac0340758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac0340758%26sid%3Dliteratum%253Aachs%26aulast%3DWheeler%26aufirst%3DA.%2BR.%26aulast%3DThrondset%26aufirst%3DW.%2BR.%26aulast%3DWhelan%26aufirst%3DR.%2BJ.%26aulast%3DLeach%26aufirst%3DA.%2BM.%26aulast%3DZare%26aufirst%3DR.%2BN.%26aulast%3DLiao%26aufirst%3DY.%2BH.%26aulast%3DFarrell%26aufirst%3DK.%26aulast%3DManger%26aufirst%3DI.%2BD.%26aulast%3DDaridon%26aufirst%3DA.%26atitle%3DMicrofluidic%2520Device%2520for%2520Single-Cell%2520Analysis%26jtitle%3DAnal.%2520Chem.%26date%3D2003%26volume%3D75%26issue%3D14%26spage%3D3581%26epage%3D3586%26doi%3D10.1021%2Fac0340758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcato, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pass, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cioce, M.</span></span> <span> </span><span class="NLM_article-title">A STAT3-NFkB/DDIT3/CEBPÎ² Axis Modulates ALDH1A3 Expression in Chemoresistant Cell Subpopulations</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">12637</span>â <span class="NLM_lpage">12653</span>, <span class="refDoi">Â DOI: 10.18632/oncotarget.3703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.18632%2Foncotarget.3703" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=12637-12653&issue=14&author=C.+Caninoauthor=Y.+Luoauthor=P.+Marcatoauthor=G.+Blandinoauthor=H.+I.+Passauthor=M.+Cioce&title=A+STAT3-NFkB%2FDDIT3%2FCEBP%CE%B2+Axis+Modulates+ALDH1A3+Expression+in+Chemoresistant+Cell+Subpopulations&doi=10.18632%2Foncotarget.3703"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3703%26sid%3Dliteratum%253Aachs%26aulast%3DCanino%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DMarcato%26aufirst%3DP.%26aulast%3DBlandino%26aufirst%3DG.%26aulast%3DPass%26aufirst%3DH.%2BI.%26aulast%3DCioce%26aufirst%3DM.%26atitle%3DA%2520STAT3-NFkB%252FDDIT3%252FCEBP%25CE%25B2%2520Axis%2520Modulates%2520ALDH1A3%2520Expression%2520in%2520Chemoresistant%2520Cell%2520Subpopulations%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26issue%3D14%26spage%3D12637%26epage%3D12653%26doi%3D10.18632%2Foncotarget.3703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golubovskaya, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conroy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cance, W. G.</span></span> <span> </span><span class="NLM_article-title">Down-Regulation of ALDH1A3, CD44 or MDR1 Sensitizes Resistant Cancer Cells to FAK Autophosphorylation Inhibitor Y15</span>. <i>J. Cancer Res. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>141</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1613</span>â <span class="NLM_lpage">1631</span>, <span class="refDoi">Â DOI: 10.1007/s00432-015-1924-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1007%2Fs00432-015-1924-3" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2015&pages=1613-1631&issue=9&author=V.+Golubovskayaauthor=S.+O%E2%80%99Brienauthor=B.+Hoauthor=M.+Hefflerauthor=J.+Conroyauthor=Q.+Huauthor=D.+Wangauthor=S.+Liuauthor=W.+G.+Cance&title=Down-Regulation+of+ALDH1A3%2C+CD44+or+MDR1+Sensitizes+Resistant+Cancer+Cells+to+FAK+Autophosphorylation+Inhibitor+Y15&doi=10.1007%2Fs00432-015-1924-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2Fs00432-015-1924-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-015-1924-3%26sid%3Dliteratum%253Aachs%26aulast%3DGolubovskaya%26aufirst%3DV.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DS.%26aulast%3DHo%26aufirst%3DB.%26aulast%3DHeffler%26aufirst%3DM.%26aulast%3DConroy%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DCance%26aufirst%3DW.%2BG.%26atitle%3DDown-Regulation%2520of%2520ALDH1A3%252C%2520CD44%2520or%2520MDR1%2520Sensitizes%2520Resistant%2520Cancer%2520Cells%2520to%2520FAK%2520Autophosphorylation%2520Inhibitor%2520Y15%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D141%26issue%3D9%26spage%3D1613%26epage%3D1631%26doi%3D10.1007%2Fs00432-015-1924-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raha, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">The Cancer Stem Cell Marker Aldehyde Dehydrogenase is Required to Maintain a Drug-Tolerant Tumor Cell Subpopulation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3579</span>â <span class="NLM_lpage">3590</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-3456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F0008-5472.CAN-13-3456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=24812274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKlurnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=3579-3590&issue=13&author=D.+Rahaauthor=T.+R.+Wilsonauthor=J.+Pengauthor=D.+Petersonauthor=P.+Yueauthor=M.+Evangelistaauthor=C.+Wilsonauthor=M.+Merchantauthor=J.+Settleman&title=The+Cancer+Stem+Cell+Marker+Aldehyde+Dehydrogenase+is+Required+to+Maintain+a+Drug-Tolerant+Tumor+Cell+Subpopulation&doi=10.1158%2F0008-5472.CAN-13-3456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The Cancer Stem Cell Marker Aldehyde Dehydrogenase Is Required to Maintain a Drug-Tolerant Tumor Cell Subpopulation</span></div><div class="casAuthors">Raha, Debasish; Wilson, Timothy R.; Peng, Jing; Peterson, David; Yue, Peng; Evangelista, Marie; Wilson, Catherine; Merchant, Mark; Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3579-3590</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Selective kinase inhibitors have emerged as an important class of cancer therapeutics, and several such drugs are now routinely used to treat advanced-stage disease.  However, their clin. benefit is typically short-lived because of the relatively rapid acquisition of drug resistance following treatment response.  A Accumlating preclin. and clin. data point to a role for a heterogeneous response to treatment within a subpopulation of tumor cells that are intrinsically drug-resistant, such as cancer stem cells.  We have previously described an epigenetically detd. reversibly drug-tolerant subpopulation of cancer cells that share some properties with cancer stem cells.  Here, we define a requirement for the previously established cancer stem cell marker ALDH (aldehyde dehydrogenase) in the maintenance of this drug-tolerant subpopulation.  We find that ALDH protects the drug-tolerant subpopulation from the potentially toxic effects of elevated levels of reactive oxygen species (ROS) in these cells, and pharmacol. disruption of ALDH activity leads to accumulation of ROS to toxic levels, consequent DNA damage, and apoptosis specifically within the drug-tolerant subpopulation.  Combining ALDH inhibition with other kinase-directed treatments delayed treatment relapse in vitro and in vivo, revealing a novel combination treatment strategy for cancers that might otherwise rapidly relapse following single-agent therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLJmyYM8YDRrVg90H21EOLACvtfcHk0lhHvvIPLKMeDQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKlurnE&md5=d9180579641124fa4c565d8ff0a7cf46</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-3456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-3456%26sid%3Dliteratum%253Aachs%26aulast%3DRaha%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DT.%2BR.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DEvangelista%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DThe%2520Cancer%2520Stem%2520Cell%2520Marker%2520Aldehyde%2520Dehydrogenase%2520is%2520Required%2520to%2520Maintain%2520a%2520Drug-Tolerant%2520Tumor%2520Cell%2520Subpopulation%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26issue%3D13%26spage%3D3579%26epage%3D3590%26doi%3D10.1158%2F0008-5472.CAN-13-3456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chou, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talalay, P.</span></span> <span> </span><span class="NLM_article-title">Quantitative Analysis of Dose-Effect Relationships: the Combined Effects of Multiple Drugs or Enzyme Inhibitors</span>. <i>Adv. Enzyme Regul.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">27</span>â <span class="NLM_lpage">55</span>, <span class="refDoi">Â DOI: 10.1016/0065-2571(84)90007-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2F0065-2571%2884%2990007-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=6382953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADyaL2cXktlaksLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1984&pages=27-55&author=T.+C.+Chouauthor=P.+Talalay&title=Quantitative+Analysis+of+Dose-Effect+Relationships%3A+the+Combined+Effects+of+Multiple+Drugs+or+Enzyme+Inhibitors&doi=10.1016%2F0065-2571%2884%2990007-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative analysis of dose-effect relationships:  the combined effects of multiple drugs or enzyme inhibitors</span></div><div class="casAuthors">Chou, Ting Chao; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Enzyme Regulation</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27-55</span>CODEN:
                <span class="NLM_cas:coden">AEZRA2</span>;
        ISSN:<span class="NLM_cas:issn">0065-2571</span>.
    </div><div class="casAbstract">A simplified exptl. design method for the anal. of multiple drug effects and for detg. summation, synergism, and antagonism is described.  The method is not limited by: whether the dose-effect relations are hyperbolic or sigmoidal, whether the effects are mutually exclusive or nonexclusive, whether the ligand interactions are competitive, noncompetitive, or uncompetitive, or whether the drugs are agonists or antagonists.  The method is independent of the no. of drugs involved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX4wNupCqgpbVg90H21EOLACvtfcHk0liBkzSorSwPmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXktlaksLk%253D&md5=055a2711b52dde49389c3f526352333d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2F0065-2571%2884%2990007-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0065-2571%252884%252990007-4%26sid%3Dliteratum%253Aachs%26aulast%3DChou%26aufirst%3DT.%2BC.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DQuantitative%2520Analysis%2520of%2520Dose-Effect%2520Relationships%253A%2520the%2520Combined%2520Effects%2520of%2520Multiple%2520Drugs%2520or%2520Enzyme%2520Inhibitors%26jtitle%3DAdv.%2520Enzyme%2520Regul.%26date%3D1984%26volume%3D22%26spage%3D27%26epage%3D55%26doi%3D10.1016%2F0065-2571%2884%2990007-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takayama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckanovich, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span> <span> </span><span class="NLM_article-title">Formation of Stable Small Cell Number Three-Dimensional Ovarian Cancer Spheroids Using Hanging Drop Arrays for Preclinical Drug Sensitivity Assays</span>. <i>Gynecol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>138</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">181</span>â <span class="NLM_lpage">189</span>, <span class="refDoi">Â DOI: 10.1016/j.ygyno.2015.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.ygyno.2015.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25913133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXot1WnsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2015&pages=181-189&issue=1&author=S.+Raghavanauthor=M.+R.+Wardauthor=K.+R.+Rowleyauthor=R.+M.+Woldauthor=S.+Takayamaauthor=R.+J.+Buckanovichauthor=G.+Mehta&title=Formation+of+Stable+Small+Cell+Number+Three-Dimensional+Ovarian+Cancer+Spheroids+Using+Hanging+Drop+Arrays+for+Preclinical+Drug+Sensitivity+Assays&doi=10.1016%2Fj.ygyno.2015.04.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of stable small cell number three-dimensional ovarian cancer spheroids using hanging drop arrays for preclinical drug sensitivity assays</span></div><div class="casAuthors">Raghavan, Shreya; Ward, Maria R.; Rowley, Katelyn R.; Wold, Rachel M.; Takayama, Shuichi; Buckanovich, Ronald J.; Mehta, Geeta</div><div class="citationInfo"><span class="NLM_cas:title">Gynecologic Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">181-189</span>CODEN:
                <span class="NLM_cas:coden">GYNOA3</span>;
        ISSN:<span class="NLM_cas:issn">0090-8258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Ovarian cancer grows and metastasizes from multicellular spheroidal aggregates within the ascites fluid.  Multicellular tumor spheroids are therefore physiol. significant 3D in vitro models for ovarian cancer research.  Conventional hanging drop cultures require high starting cell nos., and are tedious for long-term maintenance.  In this study, we generate stable, uniform multicellular spheroids using very small no. of ovarian cancer cells in a novel 384 well hanging drop array platform.  We used novel tumor spheroid platform and two ovarian cancer cell lines (A2780 and OVCAR3) to demonstrate the stable incorporation of as few as 10 cells into a single spheroid.  Spheroids had uniform geometry, with projected areas (42.60 Ã 103 Î¼m-475.22 Ã 103 Î¼m2 for A2780 spheroids and 37.24 Ã 103 Î¼m2-281.01 Ã 103 Î¼m2 for OVCAR3 spheroids) that varied as a function of the initial cell seeding d.  Phalloidin and nuclear stains indicated cells formed tightly packed spheroids with demarcated boundaries and cell-cell interaction within spheroids.  Cells within spheroids demonstrated over 85% viability. 3D tumor spheroids demonstrated greater resistance (70-80% viability) to cisplatin chemotherapy compared to 2D cultures (30-50% viability).  Ovarian cancer spheroids can be generated from limited cell nos. in high throughput 384 well plates with high viability.  Spheroids demonstrate therapeutic resistance relative to cells in traditional 2D culture.  Stable incorporation of low cell nos. is advantageous when translating this research to rare patient-derived cells.  This system can be used to understand ovarian cancer spheroid biol., as well as carry out preclin. drug sensitivity assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZZNJLQFmx2LVg90H21EOLACvtfcHk0liBkzSorSwPmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXot1WnsrY%253D&md5=0e327f491b2234aa686994b785a05e88</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.ygyno.2015.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ygyno.2015.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DRaghavan%26aufirst%3DS.%26aulast%3DWard%26aufirst%3DM.%2BR.%26aulast%3DRowley%26aufirst%3DK.%2BR.%26aulast%3DWold%26aufirst%3DR.%2BM.%26aulast%3DTakayama%26aufirst%3DS.%26aulast%3DBuckanovich%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26atitle%3DFormation%2520of%2520Stable%2520Small%2520Cell%2520Number%2520Three-Dimensional%2520Ovarian%2520Cancer%2520Spheroids%2520Using%2520Hanging%2520Drop%2520Arrays%2520for%2520Preclinical%2520Drug%2520Sensitivity%2520Assays%26jtitle%3DGynecol.%2520Oncol.%26date%3D2015%26volume%3D138%26issue%3D1%26spage%3D181%26epage%3D189%26doi%3D10.1016%2Fj.ygyno.2015.04.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shield, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackland, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, G. E.</span></span> <span> </span><span class="NLM_article-title">Multicellular Spheroids in Ovarian Cancer Metastases: Biology and Pathology</span>. <i>Gynecol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">143</span>â <span class="NLM_lpage">148</span>, <span class="refDoi">Â DOI: 10.1016/j.ygyno.2008.11.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.ygyno.2008.11.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=19135710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A280%3ADC%252BD1M3is12itg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=143-148&issue=1&author=K.+Shieldauthor=M.+L.+Acklandauthor=N.+Ahmedauthor=G.+E.+Rice&title=Multicellular+Spheroids+in+Ovarian+Cancer+Metastases%3A+Biology+and+Pathology&doi=10.1016%2Fj.ygyno.2008.11.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Multicellular spheroids in ovarian cancer metastases: Biology and pathology</span></div><div class="casAuthors">Shield Kristy; Ackland M Leigh; Ahmed Nuzhat; Rice Gregory E</div><div class="citationInfo"><span class="NLM_cas:title">Gynecologic oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">143-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epithelial ovarian cancer (EOC) has a relatively high mortality rate ( approximately 55%).  One of the presiding causes is that the current chemotherapeutic regimes are unable to achieve sustained remission, despite frequently producing a positive response at first treatment.  One of the reasons that EOC is difficult to treat is that the mechanism of dissemination is unusual.  EOC dissemination characteristically involves local invasion of pelvic and abdominal organs.  Unlike many epithelial cancers, initial dissemination rarely requires the vasculature, although the vasculature is often implicated in the advanced stages of disease.  Recently, it has become apparent that aggregates of malignant cells (spheroids) contained within malignant ascites represent a significant impediment to efficacious treatment of late stage EOC.  In vivo, spheroids are present in the malignant ascites of EOC patients, while in vitro cultured spheroids are capable of tumorgenesis in vivo and display a reduced response to chemotherapeutic drugs when compared to monolayers.  A major problem associated with the current generation of chemotherapy agents is that they do not address the anchorage- and vascular-independent growth conditions associated with a 3-dimensional structure that has formed and/or grown in suspension.  Thus, spheroid formation may represent a key component of platinum/taxane-sensitive recurrence.  If this is correct, a better understanding of spheroid biology may contribute to the identification of new treatment opportunities for the sustained treatment of metastatic EOC.  This review article outlines the key biological features of spheroids, specifically discussing their role in EOC dissemination and chemo-response as well as providing insights into spheroid functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTd5-MIaWwb6Xqj_yg7twvpfW6udTcc2eY37ugNndI8cbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3is12itg%253D%253D&md5=e12bb83edf45c5406122f11c7764266d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.ygyno.2008.11.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ygyno.2008.11.032%26sid%3Dliteratum%253Aachs%26aulast%3DShield%26aufirst%3DK.%26aulast%3DAckland%26aufirst%3DM.%2BL.%26aulast%3DAhmed%26aufirst%3DN.%26aulast%3DRice%26aufirst%3DG.%2BE.%26atitle%3DMulticellular%2520Spheroids%2520in%2520Ovarian%2520Cancer%2520Metastases%253A%2520Biology%2520and%2520Pathology%26jtitle%3DGynecol.%2520Oncol.%26date%3D2009%26volume%3D113%26issue%3D1%26spage%3D143%26epage%3D148%26doi%3D10.1016%2Fj.ygyno.2008.11.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bregenzer, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleck, E. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckanovich, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span> <span> </span><span class="NLM_article-title">Personalized Medicine-Based Approach to Model Patterns of Chemoresistance and Tumor Recurrence Using Ovarian Cancer Stem Cell Spheroids</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">6934</span>â <span class="NLM_lpage">6945</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-17-0133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F1078-0432.CCR-17-0133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=28814433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnurbN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=6934-6945&issue=22&author=S.+Raghavanauthor=P.+Mehtaauthor=M.+R.+Wardauthor=M.+E.+Bregenzerauthor=E.+M.+A.+Fleckauthor=L.+Tanauthor=K.+McLeanauthor=R.+J.+Buckanovichauthor=G.+Mehta&title=Personalized+Medicine-Based+Approach+to+Model+Patterns+of+Chemoresistance+and+Tumor+Recurrence+Using+Ovarian+Cancer+Stem+Cell+Spheroids&doi=10.1158%2F1078-0432.CCR-17-0133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Personalized Medicine-Based Approach to Model Patterns of Chemoresistance and Tumor Recurrence Using Ovarian Cancer Stem Cell Spheroids</span></div><div class="casAuthors">Raghavan, Shreya; Mehta, Pooja; Ward, Maria R.; Bregenzer, Michael E.; Fleck, Elyse M. A.; Tan, Lijun; McLean, Karen; Buckanovich, Ronald J.; Mehta, Geeta</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6934-6945</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Chemoresistant ovarian cancers grow in suspension within the ascites fluid.  To screen the effect of chemotherapeutics and biologics on resistant ovarian cancers with a personalized basis, we developed a 3D hanging drop spheroid platform.  Exptl. Design: We initiated spheroids with primary aldehyde dehydrogenase-pos. (ALDH+) CD133+ ovarian cancer stem cells (OvCSC) from different patient samples and demonstrated that stem cell progeny from harvested spheroids was similar to the primary tumor.  OvCSC spheroids were utilized to initiate tumors in immunodeficient mice.  Drug responses to cisplatin and ALDH-targeting compd. or JAK2 inhibitor detd. whether the OvCSC population within the spheroids could be targeted.  Cells that escaped therapy were isolated and used to initiate new spheroids and model tumor reemergence in a personalized manner.  Results: OvCSC spheroids from different patients exhibited varying and personalized responses to chemotherapeutics.  Xenografts were established from OvCSC spheroids, even with a single spheroid.  Distinct responses to therapy were obsd. in distinct primary tumor xenografts similar to those obsd. in spheroids.  Spheroids resistant to cisplatin/ALDH inhibitor therapy had persistent, albeit lower ALDH expression and complete loss of CD133 expression, whereas those resistant to cisplatin/JAK2 inhibitor therapy were enriched for ALDH+ cells.  Conclusions: Our 3D hanging drop suspension platform can be used to propagate primary OvCSCs that represent individual patient tumors effectively by differentiating in vitro and initiating tumors in mice.  Therefore, our platform can be used to study cancer stem cell biol. and model tumor reemergence to identify new targeted therapeutics from an effective personalized medicine standpoint.  Clin Cancer Res; 23(22); 6934-45. Â©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrym4L94HYJs7Vg90H21EOLACvtfcHk0liBkzSorSwPmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnurbN&md5=b8b2a038d780b726ff785287a0ea4489</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-17-0133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-17-0133%26sid%3Dliteratum%253Aachs%26aulast%3DRaghavan%26aufirst%3DS.%26aulast%3DMehta%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DM.%2BR.%26aulast%3DBregenzer%26aufirst%3DM.%2BE.%26aulast%3DFleck%26aufirst%3DE.%2BM.%2BA.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DMcLean%26aufirst%3DK.%26aulast%3DBuckanovich%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26atitle%3DPersonalized%2520Medicine-Based%2520Approach%2520to%2520Model%2520Patterns%2520of%2520Chemoresistance%2520and%2520Tumor%2520Recurrence%2520Using%2520Ovarian%2520Cancer%2520Stem%2520Cell%2520Spheroids%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26issue%3D22%26spage%3D6934%26epage%3D6945%26doi%3D10.1158%2F1078-0432.CCR-17-0133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhWta6qL83krg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Collaborative
Computational Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4 Suite: Programs for Protein Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">Pt 5</span>),  <span class="NLM_fpage">760</span>â <span class="NLM_lpage">763</span>, <span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&issue=Pt+5&author=Collaborative%0AComputational+Project%2C+Number+4&title=The+CCP4+Suite%3A+Programs+for+Protein+Crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520Suite%253A%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26issue%3DPt%25205%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title"><i>Coot</i>: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2126</span>â <span class="NLM_lpage">2132</span>, <span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&issue=12&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhWta6qL83krg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26issue%3D12%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">Optimal Description of a Protein Structure in Terms of Multiple Groups Undergoing TLS Motion</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">439</span>â <span class="NLM_lpage">450</span>, <span class="refDoi">Â DOI: 10.1107/S0907444906005270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0907444906005270" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=16552146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=439-450&issue=4&author=J.+Painterauthor=E.+A.+Merritt&title=Optimal+Description+of+a+Protein+Structure+in+Terms+of+Multiple+Groups+Undergoing+TLS+Motion&doi=10.1107%2FS0907444906005270"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-450</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A single protein crystal structure contains information about dynamic properties of the protein as well as providing a static view of one three-dimensional conformation.  This addnl. information is to be found in the distribution of obsd. electron d. about the mean position of each atom.  It is general practice to account for this by refining a sep. at. displacement parameter (ADP) for each at. center.  However, these same displacements are often described well by simpler models based on TLS (translation/libration/screw) rigid-body motion of large groups of atoms, for example interdomain hinge motion.  A procedure, TLSMD, has been developed that analyzes the distribution of ADPs in a previously refined protein crystal structure in order to generate optimal multi-group TLS descriptions of the constituent protein chains.  TLSMD is applicable to crystal structures at any resoln.  The models generated by TLSMD anal. can significantly improve the std. crystallog. residuals R and Rfree and can reveal intrinsic dynamic properties of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIR_NcneJBNbVg90H21EOLACvtfcHk0lhWta6qL83krg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D&md5=6289483d19fb6dd64fe9b032e9c45005</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444906005270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906005270%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DOptimal%2520Description%2520of%2520a%2520Protein%2520Structure%2520in%2520Terms%2520of%2520Multiple%2520Groups%2520Undergoing%2520TLS%2520Motion%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26issue%3D4%26spage%3D439%26epage%3D450%26doi%3D10.1107%2FS0907444906005270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">A Molecular Viewer for the Analysis of TLS Rigid-Body Motion in Macromolecules</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">465</span>â <span class="NLM_lpage">471</span>, <span class="refDoi">Â DOI: 10.1107/S0907444905001897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0907444905001897" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=465-471&issue=4&author=J.+Painterauthor=E.+A.+Merritt&title=A+Molecular+Viewer+for+the+Analysis+of+TLS+Rigid-Body+Motion+in+Macromolecules&doi=10.1107%2FS0907444905001897"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444905001897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905001897%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DA%2520Molecular%2520Viewer%2520for%2520the%2520Analysis%2520of%2520TLS%2520Rigid-Body%2520Motion%2520in%2520Macromolecules%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26issue%3D4%26spage%3D465%26epage%3D471%26doi%3D10.1107%2FS0907444905001897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammen, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallenga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, H.</span></span> <span> </span><span class="NLM_article-title">Multiple Conformations of NAD and NADH when Bound to Human Cytosolic and Mitochondrial Aldehyde Dehydrogenase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">7156</span>â <span class="NLM_lpage">7168</span>, <span class="refDoi">Â DOI: 10.1021/bi012197t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi012197t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=7156-7168&issue=22&author=P.+K.+Hammenauthor=A.+Allali-Hassaniauthor=K.+Hallengaauthor=T.+D.+Hurleyauthor=H.+Weiner&title=Multiple+Conformations+of+NAD+and+NADH+when+Bound+to+Human+Cytosolic+and+Mitochondrial+Aldehyde+Dehydrogenase&doi=10.1021%2Fbi012197t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fbi012197t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi012197t%26sid%3Dliteratum%253Aachs%26aulast%3DHammen%26aufirst%3DP.%2BK.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHallenga%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DMultiple%2520Conformations%2520of%2520NAD%2520and%2520NADH%2520when%2520Bound%2520to%2520Human%2520Cytosolic%2520and%2520Mitochondrial%2520Aldehyde%2520Dehydrogenase%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26issue%3D22%26spage%3D7156%26epage%3D7168%26doi%3D10.1021%2Fbi012197t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimble-Hill, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanova, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meroueh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Regulators of Aldehyde Dehydrogenase Isoenzymes</span>. <i>Chem.-Biol. Interact.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>191</i></span> (<span class="NLM_issue">1â3</span>),  <span class="NLM_fpage">153</span>â <span class="NLM_lpage">158</span>, <span class="refDoi">Â DOI: 10.1016/j.cbi.2011.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.cbi.2011.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21349255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFajtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2011&pages=153-158&issue=1%E2%80%933&author=B.+Parajuliauthor=A.+C.+Kimble-Hillauthor=M.+Khannaauthor=Y.+Ivanovaauthor=S.+Merouehauthor=T.+D.+Hurley&title=Discovery+of+Novel+Regulators+of+Aldehyde+Dehydrogenase+Isoenzymes&doi=10.1016%2Fj.cbi.2011.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel regulators of aldehyde dehydrogenase isoenzymes</span></div><div class="casAuthors">Parajuli, Bibek; Kimble-Hill, Ann C.; Khanna, May; Ivanova, Yvelina; Meroueh, Samy; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">153-158</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Over the past three years we have been involved in high-throughput screening in an effort to discover novel small mol. modulators of aldehyde dehydrogenase (ALDH) activity.  In particular, we have been interested in both the activation and inhibition of the three commonly studied isoenzymes, ALDH1A1, ALDH2 and ALDH3A1, as their distinct, yet overlapping substrate specificities, present a particularly difficult challenge for inhibitor discovery and design.  Activation of ALDH2 has been shown to benefit cardiovascular outcome following periods of ischemia and renewed interest in specific inhibition of ALDH2 has application for alc. aversion therapy, and more recently, in cocaine addiction.  In contrast, inhibition of either ALDH1A1 or ALDH3A1 has application in cancer treatments where the isoenzymes are commonly over-expressed and serve as markers for cancer stem cells.  We are taking two distinct approaches for these screens: in vitro enzyme activity screens using chem. libraries and virtual computational screens using the structures of the target enzymes as filters for identifying potential inhibitors, followed by in vitro testing of their ability to inhibit their intended targets.  We have identified selective inhibitors of each of these three isoenzymes with inhibition consts. in the high nanomolar to low micromolar range from these screening procedures.  Together, these inhibitors provide proof for concept that selective inhibition of these broad specificity general detoxication enzymes through small mol. discovery and design is possible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcKIxSAscaUbVg90H21EOLACvtfcHk0ljx-gTTYMNBHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFajtbc%253D&md5=f8c284f0a5ad00a61f68cd49e5d4bff1</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2011.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2011.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DKimble-Hill%26aufirst%3DA.%2BC.%26aulast%3DKhanna%26aufirst%3DM.%26aulast%3DIvanova%26aufirst%3DY.%26aulast%3DMeroueh%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDiscovery%2520of%2520Novel%2520Regulators%2520of%2520Aldehyde%2520Dehydrogenase%2520Isoenzymes%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D2011%26volume%3D191%26issue%3D1%25E2%2580%25933%26spage%3D153%26epage%3D158%26doi%3D10.1016%2Fj.cbi.2011.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Development of Selective Inhibitors for Human Aldehyde Dehydrogenase 3A1 (ALDH3A1) for the Enhancement of Cyclophosphamide Cytotoxicity</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">701</span>â <span class="NLM_lpage">712</span>, <span class="refDoi">Â DOI: 10.1002/cbic.201300625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fcbic.201300625" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=701-712&issue=5&author=B.+Parajuliauthor=T.+M.+Georgiadisauthor=M.+L.+Fishelauthor=T.+D.+Hurley&title=Development+of+Selective+Inhibitors+for+Human+Aldehyde+Dehydrogenase+3A1+%28ALDH3A1%29+for+the+Enhancement+of+Cyclophosphamide+Cytotoxicity&doi=10.1002%2Fcbic.201300625"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201300625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201300625%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DGeorgiadis%26aufirst%3DT.%2BM.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520Selective%2520Inhibitors%2520for%2520Human%2520Aldehyde%2520Dehydrogenase%25203A1%2520%2528ALDH3A1%2529%2520for%2520the%2520Enhancement%2520of%2520Cyclophosphamide%2520Cytotoxicity%26jtitle%3DChemBioChem%26date%3D2014%26volume%3D15%26issue%3D5%26spage%3D701%26epage%3D712%26doi%3D10.1002%2Fcbic.201300625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchman, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the Aldehyde Dehydrogenase 1/2 Family by Psoralen and Coumarin Derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2439</span>â <span class="NLM_lpage">2455</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2439-2455&issue=6&author=C.+D.+Buchmanauthor=T.+D.+Hurley&title=Inhibition+of+the+Aldehyde+Dehydrogenase+1%2F2+Family+by+Psoralen+and+Coumarin+Derivatives&doi=10.1021%2Facs.jmedchem.6b01825"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01825%26sid%3Dliteratum%253Aachs%26aulast%3DBuchman%26aufirst%3DC.%2BD.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DInhibition%2520of%2520the%2520Aldehyde%2520Dehydrogenase%25201%252F2%2520Family%2520by%2520Psoralen%2520and%2520Coumarin%2520Derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D6%26spage%3D2439%26epage%3D2455%26doi%3D10.1021%2Facs.jmedchem.6b01825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horst, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span> <span> </span><span class="NLM_article-title">Comparative Analysis of Tumor Spheroid Generation Techniques for Differential In Vitro Drug Toxicity</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">16948</span>â <span class="NLM_lpage">16961</span>, <span class="refDoi">Â DOI: 10.18632/oncotarget.7659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.18632%2Foncotarget.7659" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=16948-16961&issue=13&author=S.+Raghavanauthor=P.+Mehtaauthor=E.+N.+Horstauthor=M.+R.+Wardauthor=K.+R.+Rowleyauthor=G.+Mehta&title=Comparative+Analysis+of+Tumor+Spheroid+Generation+Techniques+for+Differential+In+Vitro+Drug+Toxicity&doi=10.18632%2Foncotarget.7659"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7659%26sid%3Dliteratum%253Aachs%26aulast%3DRaghavan%26aufirst%3DS.%26aulast%3DMehta%26aufirst%3DP.%26aulast%3DHorst%26aufirst%3DE.%2BN.%26aulast%3DWard%26aufirst%3DM.%2BR.%26aulast%3DRowley%26aufirst%3DK.%2BR.%26aulast%3DMehta%26aufirst%3DG.%26atitle%3DComparative%2520Analysis%2520of%2520Tumor%2520Spheroid%2520Generation%2520Techniques%2520for%2520Differential%2520In%2520Vitro%2520Drug%2520Toxicity%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26issue%3D13%26spage%3D16948%26epage%3D16961%26doi%3D10.18632%2Foncotarget.7659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biasini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bienert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterhouse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Studer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo Cassarino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwede, T.</span></span> <span> </span><span class="NLM_article-title">SWISS-MODEL: Modelling Protein Tertiary and Quaternary Structure Using Evolutionary Information</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W252</span>â <span class="NLM_lpage">W258</span>, <span class="refDoi">Â DOI: 10.1093/nar/gku340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1093%2Fnar%2Fgku340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=24782522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W252-W258&issue=W1&author=M.+Biasiniauthor=S.+Bienertauthor=A.+Waterhouseauthor=K.+Arnoldauthor=G.+Studerauthor=T.+Schmidtauthor=F.+Kieferauthor=T.+Gallo+Cassarinoauthor=M.+Bertoniauthor=L.+Bordoliauthor=T.+Schwede&title=SWISS-MODEL%3A+Modelling+Protein+Tertiary+and+Quaternary+Structure+Using+Evolutionary+Information&doi=10.1093%2Fnar%2Fgku340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information</span></div><div class="casAuthors">Biasini, Marco; Bienert, Stefan; Waterhouse, Andrew; Arnold, Konstantin; Studer, Gabriel; Schmidt, Tobias; Kiefer, Florian; Cassarino, Tiziano Gallo; Bertoni, Martino; Bordoli, Lorenza; Schwede, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W252-W258</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein structure homol. modeling has become a routine technique to generate 3D models for proteins when exptl. structures are not available.  Fully automated servers such as SWISS-MODEL with user-friendly web interfaces generate reliable models without the need for complex software packages or downloading large databases.  Here, we describe the latest version of the SWISS-MODEL expert system for protein structure modeling.  The SWISS-MODEL template library provides annotation of quaternary structure and essential ligands and co-factors to allow for building of complete structural models, including their oligomeric structure.  The improved SWISS-MODEL pipeline makes extensive use of model quality estn. for selection of the most suitable templates and provides ests. of the expected accuracy of the resulting models.  The accuracy of the models generated by SWISS-MODEL is continuously evaluated by the CAMEO system.  The new web site allows users to interactively search for templates, cluster them by sequence similarity, structurally compare alternative templates and select the ones to be used for model building.  In cases where multiple alternative template structures are available for a protein of interest, a user-guided template selection step allows building models in different functional states.  SWISS-MODEL is available at http://swissmodel.expasy.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevqhyplQxI7Vg90H21EOLACvtfcHk0ljfTN8pUqxEDA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs73I&md5=51509ba8353b06f286d954ebe7e6673a</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku340%26sid%3Dliteratum%253Aachs%26aulast%3DBiasini%26aufirst%3DM.%26aulast%3DBienert%26aufirst%3DS.%26aulast%3DWaterhouse%26aufirst%3DA.%26aulast%3DArnold%26aufirst%3DK.%26aulast%3DStuder%26aufirst%3DG.%26aulast%3DSchmidt%26aufirst%3DT.%26aulast%3DKiefer%26aufirst%3DF.%26aulast%3DGallo%2BCassarino%26aufirst%3DT.%26aulast%3DBertoni%26aufirst%3DM.%26aulast%3DBordoli%26aufirst%3DL.%26aulast%3DSchwede%26aufirst%3DT.%26atitle%3DSWISS-MODEL%253A%2520Modelling%2520Protein%2520Tertiary%2520and%2520Quaternary%2520Structure%2520Using%2520Evolutionary%2520Information%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DW1%26spage%3DW252%26epage%3DW258%26doi%3D10.1093%2Fnar%2Fgku340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwede, T.</span></span> <span> </span><span class="NLM_article-title">The SWISS-MODEL Workspace: a Web-Based Environment for Protein Structure Homology Modelling</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">195</span>â <span class="NLM_lpage">201</span>, <span class="refDoi">Â DOI: 10.1093/bioinformatics/bti770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1093%2Fbioinformatics%2Fbti770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=16301204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD28XovVCltw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=195-201&issue=2&author=K.+Arnoldauthor=L.+Bordoliauthor=J.+Koppauthor=T.+Schwede&title=The+SWISS-MODEL+Workspace%3A+a+Web-Based+Environment+for+Protein+Structure+Homology+Modelling&doi=10.1093%2Fbioinformatics%2Fbti770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling</span></div><div class="casAuthors">Arnold, Konstantin; Bordoli, Lorenza; Kopp, Juergen; Schwede, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">195-201</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Homol. models of proteins are of great interest for planning and analyzing biol. expts. when no exptl. three-dimensional structures are available.  Building homol. models requires specialized programs and up-to-date sequence and structural databases.  Integrating all required tools, programs and databases into a single web-based workspace facilitates access to homol. modeling from a computer with web connection without the need of downloading and installing large program packages and databases.  Results: SWISS-MODEL workspace is a web-based integrated service dedicated to protein structure homol. modeling.  It assists and guides the user in building protein homol. models at different levels of complexity.  A personal working environment is provided for each user where several modeling projects can be carried out in parallel.  Protein sequence and structure databases necessary for modeling are accessible from the workspace and are updated in regular intervals.  Tools for template selection, model building and structure quality evaluation can be invoked from within the workspace.  Workflow and usage of the workspace are illustrated by modeling human Cyclin A1 and human Transmembrane Protease 3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDwXLAG5Kz27Vg90H21EOLACvtfcHk0ljfTN8pUqxEDA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XovVCltw%253D%253D&md5=66fa733d2ad6d2abf7d3d87ae837ebce</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbti770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbti770%26sid%3Dliteratum%253Aachs%26aulast%3DArnold%26aufirst%3DK.%26aulast%3DBordoli%26aufirst%3DL.%26aulast%3DKopp%26aufirst%3DJ.%26aulast%3DSchwede%26aufirst%3DT.%26atitle%3DThe%2520SWISS-MODEL%2520Workspace%253A%2520a%2520Web-Based%2520Environment%2520for%2520Protein%2520Structure%2520Homology%2520Modelling%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26issue%3D2%26spage%3D195%26epage%3D201%26doi%3D10.1093%2Fbioinformatics%2Fbti770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benkert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biasini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwede, T.</span></span> <span> </span><span class="NLM_article-title">Toward the Estimation of the Absolute Quality of Individual Protein Structure Models</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">343</span>â <span class="NLM_lpage">350</span>, <span class="refDoi">Â DOI: 10.1093/bioinformatics/btq662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1093%2Fbioinformatics%2Fbtq662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21134891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Gisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=343-350&issue=3&author=P.+Benkertauthor=M.+Biasiniauthor=T.+Schwede&title=Toward+the+Estimation+of+the+Absolute+Quality+of+Individual+Protein+Structure+Models&doi=10.1093%2Fbioinformatics%2Fbtq662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the estimation of the absolute quality of individual protein structure models</span></div><div class="casAuthors">Benkert, Pascal; Biasini, Marco; Schwede, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">343-350</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Quality assessment of protein structures is an important part of exptl. structure validation and plays a crucial role in protein structure prediction, where the predicted models may contain substantial errors.  Most current scoring functions are primarily designed to rank alternative models of the same sequence supporting model selection, whereas the prediction of the abs. quality of an individual protein model has received little attention in the field.  However, reliable abs. quality ests. are crucial to assess the suitability of a model for specific biomedical applications.  In this work, we present a new abs. measure for the quality of protein models, which provides an est. of the degree of nativeness' of the structural features obsd. in a model and describes the likelihood that a given model is of comparable quality to exptl. structures.  Model quality ests. based on the QMEAN scoring function were normalized with respect to the no. of interactions.  The resulting scoring function is independent of the size of the protein and may therefore be used to assess both monomers and entire oligomeric assemblies.  Model quality scores for individual models are then expressed as Z-scores' in comparison to scores obtained for high-resoln. crystal structures.  We demonstrate the ability of the newly introduced QMEAN Z-score to detect exptl. solved protein structures contg. significant errors, as well as to evaluate theor. protein models.  In a comprehensive QMEAN Z-score anal. of all exptl. structures in the PDB, membrane proteins accumulate on one side of the score spectrum and thermostable proteins on the other.  Proteins from the thermophilic organism Thermatoga maritima received significantly higher QMEAN Z-scores in a pairwise comparison with their homologous mesophilic counterparts, underlining the significance of the QMEAN Z-score as an est. of protein stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo81whOYLkpN7Vg90H21EOLACvtfcHk0ljfTN8pUqxEDA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Gisb8%253D&md5=eb1b789a978f8db7beff4dbcd03296ee</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq662%26sid%3Dliteratum%253Aachs%26aulast%3DBenkert%26aufirst%3DP.%26aulast%3DBiasini%26aufirst%3DM.%26aulast%3DSchwede%26aufirst%3DT.%26atitle%3DToward%2520the%2520Estimation%2520of%2520the%2520Absolute%2520Quality%2520of%2520Individual%2520Protein%2520Structure%2520Models%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26issue%3D3%26spage%3D343%26epage%3D350%26doi%3D10.1093%2Fbioinformatics%2Fbtq662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walther, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenet, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deutsch, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jostes, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goulding, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plachov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balling, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruss, P.</span></span> <span> </span><span class="NLM_article-title">Pax: a Murine Multigene Family of Paired Box-Containing Genes</span>. <i>Genomics</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">424</span>â <span class="NLM_lpage">434</span>, <span class="refDoi">Â DOI: 10.1016/0888-7543(91)90151-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2F0888-7543%2891%2990151-4" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=424-434&issue=2&author=C.+Waltherauthor=J.+L.+Guenetauthor=D.+Simonauthor=U.+Deutschauthor=B.+Jostesauthor=M.+D.+Gouldingauthor=D.+Plachovauthor=R.+Ballingauthor=P.+Gruss&title=Pax%3A+a+Murine+Multigene+Family+of+Paired+Box-Containing+Genes&doi=10.1016%2F0888-7543%2891%2990151-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2F0888-7543%2891%2990151-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0888-7543%252891%252990151-4%26sid%3Dliteratum%253Aachs%26aulast%3DWalther%26aufirst%3DC.%26aulast%3DGuenet%26aufirst%3DJ.%2BL.%26aulast%3DSimon%26aufirst%3DD.%26aulast%3DDeutsch%26aufirst%3DU.%26aulast%3DJostes%26aufirst%3DB.%26aulast%3DGoulding%26aufirst%3DM.%2BD.%26aulast%3DPlachov%26aufirst%3DD.%26aulast%3DBalling%26aufirst%3DR.%26aulast%3DGruss%26aufirst%3DP.%26atitle%3DPax%253A%2520a%2520Murine%2520Multigene%2520Family%2520of%2520Paired%2520Box-Containing%2520Genes%26jtitle%3DGenomics%26date%3D1991%26volume%3D11%26issue%3D2%26spage%3D424%26epage%3D434%26doi%3D10.1016%2F0888-7543%2891%2990151-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">PROCHECK - a Program to Check the Stereochemical Quality of Protein Structures</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">283</span>â <span class="NLM_lpage">291</span>, <span class="refDoi">Â DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=PROCHECK+-+a+Program+to+Check+the+Stereochemical+Quality+of+Protein+Structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0liL2zQ6IqDRZQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%2520-%2520a%2520Program%2520to%2520Check%2520the%2520Stereochemical%2520Quality%2520of%2520Protein%2520Structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGann, M.</span></span> <span> </span><span class="NLM_article-title">FRED and HYBRID Docking Performance on Standardized Datasets</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">897</span>â <span class="NLM_lpage">906</span>, <span class="refDoi">Â DOI: 10.1007/s10822-012-9584-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1007%2Fs10822-012-9584-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22669221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ens7bO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=897-906&issue=8&author=M.+McGann&title=FRED+and+HYBRID+Docking+Performance+on+Standardized+Datasets&doi=10.1007%2Fs10822-012-9584-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">FRED and HYBRID docking performance on standardized datasets</span></div><div class="casAuthors">McGann, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">897-906</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The docking performance of the FRED and HYBRID programs are evaluated on two standardized datasets from the Docking and Scoring Symposium of the ACS Spring 2011 national meeting.  The evaluation includes cognate docking and virtual screening performance.  FRED docks 70 % of the structures to within 2 Ã in the cognate docking test.  In the virtual screening test, FRED is found to have a mean AUC of 0.75.  The HYBRID program uses a modified version of FRED's algorithm that uses both ligand- and structure-based information to dock mols., which increases its mean AUC to 0.78.  HYBRID can also implicitly account for protein flexibility by making use of multiple crystal structures.  Using multiple crystal structures improves HYBRID's performance (mean AUC 0.80) with a negligible increase in docking time (â¼15 %).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDNF1kmgp-WLVg90H21EOLACvtfcHk0liL2zQ6IqDRZQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ens7bO&md5=299f2e795ca922412f57df9434b52661</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9584-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9584-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcGann%26aufirst%3DM.%26atitle%3DFRED%2520and%2520HYBRID%2520Docking%2520Performance%2520on%2520Standardized%2520Datasets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26issue%3D8%26spage%3D897%26epage%3D906%26doi%3D10.1007%2Fs10822-012-9584-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">572</span>â <span class="NLM_lpage">584</span>, <span class="refDoi">Â DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&issue=4&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+Generation+with+OMEGA%3A+Algorithm+and+Validation+Using+High+Quality+Structures+from+the+Protein+Databank+and+Cambridge+Structural+Database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lhNPm_idatO0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520Generation%2520with%2520OMEGA%253A%2520Algorithm%2520and%2520Validation%2520Using%2520High%2520Quality%2520Structures%2520from%2520the%2520Protein%2520Databank%2520and%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D4%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kehler, J.</span>; <span class="NLM_string-name">Rasmussen, L. K.</span>; <span class="NLM_string-name">Jessing, M.</span></span> <span> </span><span class="NLM_article-title">Triazolopyrazinones as PDE1 Inhibitors</span>. Patent No. <span class="NLM_patent">WO2016055618 A1</span>. <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+Kehler&author=L.+K.+Rasmussen&author=M.+Jessing&title=Triazolopyrazinones+as+PDE1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKehler%26aufirst%3DJ.%26atitle%3DTriazolopyrazinones%2520as%2520PDE1%2520Inhibitors%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 21 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gera  Narendra</span>, <span class="hlFld-ContribAuthor ">Baddipadige  Raju</span>, <span class="hlFld-ContribAuthor ">Himanshu  Verma</span>, <span class="hlFld-ContribAuthor ">Bharti  Sapra</span>, <span class="hlFld-ContribAuthor ">Om  Silakari</span>. </span><span class="cited-content_cbyCitation_article-title">Multiple machine learning models combined with virtual screening and molecular docking to identify selective human ALDH1A1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2021,</strong> <em>107 </em>, 107950. <a href="https://doi.org/10.1016/j.jmgm.2021.107950" title="DOI URL">https://doi.org/10.1016/j.jmgm.2021.107950</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2021.107950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2021.107950%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DMultiple%252Bmachine%252Blearning%252Bmodels%252Bcombined%252Bwith%252Bvirtual%252Bscreening%252Band%252Bmolecular%252Bdocking%252Bto%252Bidentify%252Bselective%252Bhuman%252BALDH1A1%252Binhibitors%26aulast%3DNarendra%26aufirst%3DGera%26date%3D2021%26volume%3D107%26spage%3D107950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">ThiÂ BaoÂ Chau  Bui</span>, <span class="hlFld-ContribAuthor ">Shohei  Nosaki</span>, <span class="hlFld-ContribAuthor ">Mito  Kokawa</span>, <span class="hlFld-ContribAuthor ">Yuqun  Xu</span>, <span class="hlFld-ContribAuthor ">Yutaka  Kitamura</span>, <span class="hlFld-ContribAuthor ">Masaru  Tanokura</span>, <span class="hlFld-ContribAuthor ">Satoshi  Hachimura</span>, <span class="hlFld-ContribAuthor ">Takuya  Miyakawa</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of spice and herb as phyto-derived selective modulators of human retinaldehyde dehydrogenases using a simple
              in vitro
              method. </span><span class="cited-content_cbyCitation_journal-name">Bioscience Reports</span><span> <strong>2021,</strong> <em>41 </em>
                                    (5)
                                     <a href="https://doi.org/10.1042/BSR20210491" title="DOI URL">https://doi.org/10.1042/BSR20210491</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BSR20210491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBSR20210491%26sid%3Dliteratum%253Aachs%26jtitle%3DBioscience%2520Reports%26atitle%3DEvaluation%252Bof%252Bspice%252Band%252Bherb%252Bas%252Bphyto-derived%252Bselective%252Bmodulators%252Bof%252Bhuman%252Bretinaldehyde%252Bdehydrogenases%252Busing%252Ba%252Bsimple%252Bin%252Bvitro%252Bmethod%26aulast%3DBui%26aufirst%3DThi%25C2%25A0Bao%25C2%25A0Chau%26date%3D2021%26date%3D2021%26volume%3D41%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohammad  Jalalirad</span>, <span class="hlFld-ContribAuthor ">Tufia C.  Haddad</span>, <span class="hlFld-ContribAuthor ">Jeffrey L.  Salisbury</span>, <span class="hlFld-ContribAuthor ">Derek  Radisky</span>, <span class="hlFld-ContribAuthor ">Minzhi  Zhang</span>, <span class="hlFld-ContribAuthor ">Mark  Schroeder</span>, <span class="hlFld-ContribAuthor ">Ann  Tuma</span>, <span class="hlFld-ContribAuthor ">Eduard  Leof</span>, <span class="hlFld-ContribAuthor ">Jodi M.  Carter</span>, <span class="hlFld-ContribAuthor ">Amy C.  Degnim</span>, <span class="hlFld-ContribAuthor ">Judy C.  Boughey</span>, <span class="hlFld-ContribAuthor ">Jann  Sarkaria</span>, <span class="hlFld-ContribAuthor ">Jia  Yu</span>, <span class="hlFld-ContribAuthor ">Liewei  Wang</span>, <span class="hlFld-ContribAuthor ">Minetta C.  Liu</span>, <span class="hlFld-ContribAuthor ">Luca  Zammataro</span>, <span class="hlFld-ContribAuthor ">Lorenzo  Malatino</span>, <span class="hlFld-ContribAuthor ">Evanthia  Galanis</span>, <span class="hlFld-ContribAuthor ">James N.  Ingle</span>, <span class="hlFld-ContribAuthor ">Matthew P.  Goetz</span>, <span class="hlFld-ContribAuthor ">Antonino B.  DâAssoro</span>. </span><span class="cited-content_cbyCitation_article-title">Aurora-A kinase oncogenic signaling mediates TGF-Î²-induced triple-negative breast cancer plasticity and chemoresistance. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2021,</strong> <em>40 </em>
                                    (14)
                                     , 2509-2523. <a href="https://doi.org/10.1038/s41388-021-01711-x" title="DOI URL">https://doi.org/10.1038/s41388-021-01711-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-021-01711-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-021-01711-x%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DAurora-A%252Bkinase%252Boncogenic%252Bsignaling%252Bmediates%252BTGF-%2525CE%2525B2-induced%252Btriple-negative%252Bbreast%252Bcancer%252Bplasticity%252Band%252Bchemoresistance%26aulast%3DJalalirad%26aufirst%3DMohammad%26date%3D2021%26date%3D2021%26volume%3D40%26issue%3D14%26spage%3D2509%26epage%3D2523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandt C.  Huddle</span>, <span class="hlFld-ContribAuthor ">Edward  Grimley</span>, <span class="hlFld-ContribAuthor ">Mikhail  Chtcherbinine</span>, <span class="hlFld-ContribAuthor ">Cameron D.  Buchman</span>, <span class="hlFld-ContribAuthor ">Cyrus  Takahashi</span>, <span class="hlFld-ContribAuthor ">Bikash  Debnath</span>, <span class="hlFld-ContribAuthor ">Stacy C.  McGonigal</span>, <span class="hlFld-ContribAuthor ">Shuai  Mao</span>, <span class="hlFld-ContribAuthor ">Siwei  Li</span>, <span class="hlFld-ContribAuthor ">Jeremy  Felton</span>, <span class="hlFld-ContribAuthor ">Shu  Pan</span>, <span class="hlFld-ContribAuthor ">Bo  Wen</span>, <span class="hlFld-ContribAuthor ">Duxin  Sun</span>, <span class="hlFld-ContribAuthor ">Nouri  Neamati</span>, <span class="hlFld-ContribAuthor ">Ronald J.  Buckanovich</span>, <span class="hlFld-ContribAuthor ">Thomas D.  Hurley</span>, <span class="hlFld-ContribAuthor ">Scott D.  Larsen</span>. </span><span class="cited-content_cbyCitation_article-title">Development of 2,5-dihydro-4H-pyrazolo[3,4-d]pyrimidin-4-one inhibitors of aldehyde dehydrogenase 1A (ALDH1A) as potential adjuncts to ovarian cancer chemotherapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>211 </em>, 113060. <a href="https://doi.org/10.1016/j.ejmech.2020.113060" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.113060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.113060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.113060%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252B2%25252C5-dihydro-4H-pyrazolo%25255B3%25252C4-d%25255Dpyrimidin-4-one%252Binhibitors%252Bof%252Baldehyde%252Bdehydrogenase%252B1A%252B%252528ALDH1A%252529%252Bas%252Bpotential%252Badjuncts%252Bto%252Bovarian%252Bcancer%252Bchemotherapy%26aulast%3DHuddle%26aufirst%3DBrandt%2BC.%26date%3D2021%26volume%3D211%26spage%3D113060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bingyan  Li</span>, <span class="hlFld-ContribAuthor ">Kang  Yang</span>, <span class="hlFld-ContribAuthor ">Dailin  Liang</span>, <span class="hlFld-ContribAuthor ">Cheng  Jiang</span>, <span class="hlFld-ContribAuthor ">Zonghui  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and development of selective aldehyde dehydrogenase 1A1 (ALDH1A1) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112940. <a href="https://doi.org/10.1016/j.ejmech.2020.112940" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112940</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112940%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252Bdevelopment%252Bof%252Bselective%252Baldehyde%252Bdehydrogenase%252B1A1%252B%252528ALDH1A1%252529%252Binhibitors%26aulast%3DLi%26aufirst%3DBingyan%26date%3D2021%26volume%3D209%26spage%3D112940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eddie L  Myers</span>. </span><span class="cited-content_cbyCitation_article-title">Bicyclic 5â6 Systems: Other Four Heteroatoms 2:2. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/B978-0-12-818655-8.00056-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-818655-8.00056-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-818655-8.00056-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-818655-8.00056-1%26sid%3Dliteratum%253Aachs%26atitle%3DBicyclic%252B5%2525E2%252580%2525936%252BSystems%25253A%252BOther%252BFour%252BHeteroatoms%252B2%25253A2%26aulast%3DMyers%26aufirst%3DEddie%2BL%26date%3D2021%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2021%26volume%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Himanshu  Verma</span>, <span class="hlFld-ContribAuthor ">Om  Silakari</span>. </span><span class="cited-content_cbyCitation_article-title">Investigating the Role of Missense SNPs on ALDH 1A1 mediated pharmacokinetic resistance to cyclophosphamide. </span><span class="cited-content_cbyCitation_journal-name">Computers in Biology and Medicine</span><span> <strong>2020,</strong> <em>125 </em>, 103979. <a href="https://doi.org/10.1016/j.compbiomed.2020.103979" title="DOI URL">https://doi.org/10.1016/j.compbiomed.2020.103979</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiomed.2020.103979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiomed.2020.103979%26sid%3Dliteratum%253Aachs%26jtitle%3DComputers%2520in%2520Biology%2520and%2520Medicine%26atitle%3DInvestigating%252Bthe%252BRole%252Bof%252BMissense%252BSNPs%252Bon%252BALDH%252B1A1%252Bmediated%252Bpharmacokinetic%252Bresistance%252Bto%252Bcyclophosphamide%26aulast%3DVerma%26aufirst%3DHimanshu%26date%3D2020%26volume%3D125%26spage%3D103979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Davide M.  Ferraris</span>, <span class="hlFld-ContribAuthor ">Edoardo L. M.  Gelardi</span>, <span class="hlFld-ContribAuthor ">Silvia  Garavaglia</span>, <span class="hlFld-ContribAuthor ">Riccardo  Miggiano</span>, <span class="hlFld-ContribAuthor ">Menico  Rizzi</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting NAD-dependent dehydrogenases in drug discovery against infectious diseases and cancer. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Society Transactions</span><span> <strong>2020,</strong> <em>48 </em>
                                    (2)
                                     , 693-707. <a href="https://doi.org/10.1042/BST20191261" title="DOI URL">https://doi.org/10.1042/BST20191261</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BST20191261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBST20191261%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Society%2520Transactions%26atitle%3DTargeting%252BNAD-dependent%252Bdehydrogenases%252Bin%252Bdrug%252Bdiscovery%252Bagainst%252Binfectious%252Bdiseases%252Band%252Bcancer%26aulast%3DFerraris%26aufirst%3DDavide%2BM.%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D2%26spage%3D693%26epage%3D707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vaishnavi  Muralikrishnan</span>, <span class="hlFld-ContribAuthor ">Thomas D.  Hurley</span>, <span class="hlFld-ContribAuthor ">Kenneth P.  Nephew</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Aldehyde Dehydrogenases to Eliminate Cancer Stem Cells in Gynecologic Malignancies. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (4)
                                     , 961. <a href="https://doi.org/10.3390/cancers12040961" title="DOI URL">https://doi.org/10.3390/cancers12040961</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12040961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12040961%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DTargeting%252BAldehyde%252BDehydrogenases%252Bto%252BEliminate%252BCancer%252BStem%252BCells%252Bin%252BGynecologic%252BMalignancies%26aulast%3DMuralikrishnan%26aufirst%3DVaishnavi%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D4%26spage%3D961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dailin  Liang</span>, <span class="hlFld-ContribAuthor ">Yazhou  Fan</span>, <span class="hlFld-ContribAuthor ">Zhou  Yang</span>, <span class="hlFld-ContribAuthor ">Zhenguo  Zhang</span>, <span class="hlFld-ContribAuthor ">Meiyang  Liu</span>, <span class="hlFld-ContribAuthor ">Li  Liu</span>, <span class="hlFld-ContribAuthor ">Cheng  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of coumarin-based selective aldehyde dehydrogenase 1A1 inhibitors with glucose metabolism improving activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 111923. <a href="https://doi.org/10.1016/j.ejmech.2019.111923" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111923</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111923%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bcoumarin-based%252Bselective%252Baldehyde%252Bdehydrogenase%252B1A1%252Binhibitors%252Bwith%252Bglucose%252Bmetabolism%252Bimproving%252Bactivity%26aulast%3DLiang%26aufirst%3DDailin%26date%3D2020%26volume%3D187%26spage%3D111923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Saketh S.  Dinavahi</span>, <span class="hlFld-ContribAuthor ">Raghavendra  Gowda</span>, <span class="hlFld-ContribAuthor ">Christopher G.  Bazewicz</span>, <span class="hlFld-ContribAuthor ">Madhu Babu  Battu</span>, <span class="hlFld-ContribAuthor ">Jyh Ming  Lin</span>, <span class="hlFld-ContribAuthor ">Robert J.  Chitren</span>, <span class="hlFld-ContribAuthor ">Manoj K.  Pandey</span>, <span class="hlFld-ContribAuthor ">Shantu  Amin</span>, <span class="hlFld-ContribAuthor ">Gavin P.  Robertson</span>, <span class="hlFld-ContribAuthor ">Krishne  Gowda</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis characterization and biological evaluation of novel multi-isoform ALDH inhibitors as potential anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 111962. <a href="https://doi.org/10.1016/j.ejmech.2019.111962" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111962%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Bcharacterization%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bmulti-isoform%252BALDH%252Binhibitors%252Bas%252Bpotential%252Banticancer%252Bagents%26aulast%3DDinavahi%26aufirst%3DSaketh%2BS.%26date%3D2020%26volume%3D187%26spage%3D111962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lu  Liu</span>, <span class="hlFld-ContribAuthor ">Shurui  Cai</span>, <span class="hlFld-ContribAuthor ">Chunhua  Han</span>, <span class="hlFld-ContribAuthor ">Ananya  Banerjee</span>, <span class="hlFld-ContribAuthor ">Dayong  Wu</span>, <span class="hlFld-ContribAuthor ">Tiantian  Cui</span>, <span class="hlFld-ContribAuthor ">Guozhen  Xie</span>, <span class="hlFld-ContribAuthor ">Junran  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaoli  Zhang</span>, <span class="hlFld-ContribAuthor ">Eric  McLaughlin</span>, <span class="hlFld-ContribAuthor ">Ming  Yin</span>, <span class="hlFld-ContribAuthor ">Floor J.  Backes</span>, <span class="hlFld-ContribAuthor ">Arnab  Chakravarti</span>, <span class="hlFld-ContribAuthor ">Yanfang  Zheng</span>, <span class="hlFld-ContribAuthor ">Qi-En  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">ALDH1A1 Contributes to PARP Inhibitor Resistance via Enhancing DNA Repair in BRCA2
              â/â
              Ovarian Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2020,</strong> <em>19 </em>
                                    (1)
                                     , 199-210. <a href="https://doi.org/10.1158/1535-7163.MCT-19-0242" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-19-0242</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-19-0242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-19-0242%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DALDH1A1%252BContributes%252Bto%252BPARP%252BInhibitor%252BResistance%252Bvia%252BEnhancing%252BDNA%252BRepair%252Bin%252BBRCA2%252B%2525E2%252588%252592%25252F%2525E2%252588%252592%252BOvarian%252BCancer%252BCells%26aulast%3DLiu%26aufirst%3DLu%26date%3D2020%26date%3D2019%26volume%3D19%26issue%3D1%26spage%3D199%26epage%3D210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zdenek  Skrott</span>, <span class="hlFld-ContribAuthor ">Dusana  Majera</span>, <span class="hlFld-ContribAuthor ">Jan  Gursky</span>, <span class="hlFld-ContribAuthor ">Tereza  Buchtova</span>, <span class="hlFld-ContribAuthor ">Marian  Hajduch</span>, <span class="hlFld-ContribAuthor ">Martin  Mistrik</span>, <span class="hlFld-ContribAuthor ">Jiri  Bartek</span>. </span><span class="cited-content_cbyCitation_article-title">Disulfiramâs anti-cancer activity reflects targeting NPL4, not inhibition of aldehyde dehydrogenase. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2019,</strong> <em>38 </em>
                                    (40)
                                     , 6711-6722. <a href="https://doi.org/10.1038/s41388-019-0915-2" title="DOI URL">https://doi.org/10.1038/s41388-019-0915-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-019-0915-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-019-0915-2%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DDisulfiram%2525E2%252580%252599s%252Banti-cancer%252Bactivity%252Breflects%252Btargeting%252BNPL4%25252C%252Bnot%252Binhibition%252Bof%252Baldehyde%252Bdehydrogenase%26aulast%3DSkrott%26aufirst%3DZdenek%26date%3D2019%26date%3D2019%26volume%3D38%26issue%3D40%26spage%3D6711%26epage%3D6722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Saketh S.  Dinavahi</span>, <span class="hlFld-ContribAuthor ">Christopher G.  Bazewicz</span>, <span class="hlFld-ContribAuthor ">Raghavendra  Gowda</span>, <span class="hlFld-ContribAuthor ">Gavin P.  Robertson</span>. </span><span class="cited-content_cbyCitation_article-title">Aldehyde Dehydrogenase Inhibitors for Cancer Therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2019,</strong> <em>40 </em>
                                    (10)
                                     , 774-789. <a href="https://doi.org/10.1016/j.tips.2019.08.002" title="DOI URL">https://doi.org/10.1016/j.tips.2019.08.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2019.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2019.08.002%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DAldehyde%252BDehydrogenase%252BInhibitors%252Bfor%252BCancer%252BTherapeutics%26aulast%3DDinavahi%26aufirst%3DSaketh%2BS.%26date%3D2019%26volume%3D40%26issue%3D10%26spage%3D774%26epage%3D789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xingyue  Zong</span>, <span class="hlFld-ContribAuthor ">Kenneth P.  Nephew</span>. </span><span class="cited-content_cbyCitation_article-title">Ovarian Cancer Stem Cells: Role in Metastasis and Opportunity for Therapeutic Targeting. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (7)
                                     , 934. <a href="https://doi.org/10.3390/cancers11070934" title="DOI URL">https://doi.org/10.3390/cancers11070934</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11070934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11070934%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DOvarian%252BCancer%252BStem%252BCells%25253A%252BRole%252Bin%252BMetastasis%252Band%252BOpportunity%252Bfor%252BTherapeutic%252BTargeting%26aulast%3DZong%26aufirst%3DXingyue%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D7%26spage%3D934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shanni  Hong</span>, <span class="hlFld-ContribAuthor ">Pi  Ding</span>, <span class="hlFld-ContribAuthor ">Yu  Luo</span>, <span class="hlFld-ContribAuthor ">Tian  Gao</span>, <span class="hlFld-ContribAuthor ">Ye  Zhang</span>, <span class="hlFld-ContribAuthor ">Renjun  Pei</span>. </span><span class="cited-content_cbyCitation_article-title">Aptamerâintegrated Î±âGal liposomes as bispecific agents to trigger immune response for killing tumor cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomedical Materials Research Part A</span><span> <strong>2019,</strong> <em>107 </em>
                                    (6)
                                     , 1176-1183. <a href="https://doi.org/10.1002/jbm.a.36609" title="DOI URL">https://doi.org/10.1002/jbm.a.36609</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jbm.a.36609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjbm.a.36609%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomedical%2520Materials%2520Research%2520Part%2520A%26atitle%3DAptamer%2525E2%252580%252590integrated%252B%2525CE%2525B1%2525E2%252580%252590Gal%252Bliposomes%252Bas%252Bbispecific%252Bagents%252Bto%252Btrigger%252Bimmune%252Bresponse%252Bfor%252Bkilling%252Btumor%252Bcells%26aulast%3DHong%26aufirst%3DShanni%26date%3D2019%26date%3D2019%26volume%3D107%26issue%3D6%26spage%3D1176%26epage%3D1183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Himanshu  Verma</span>, <span class="hlFld-ContribAuthor ">Malkeet  Singh Bahia</span>, <span class="hlFld-ContribAuthor ">Shalki  Choudhary</span>, <span class="hlFld-ContribAuthor ">Pankaj  Kumar Singh</span>, <span class="hlFld-ContribAuthor ">Om  Silakari</span>. </span><span class="cited-content_cbyCitation_article-title">Drug metabolizing enzymes-associated chemo resistance and strategies to overcome it. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism Reviews</span><span> <strong>2019,</strong> <em>51 </em>
                                    (2)
                                     , 196-223. <a href="https://doi.org/10.1080/03602532.2019.1632886" title="DOI URL">https://doi.org/10.1080/03602532.2019.1632886</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/03602532.2019.1632886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F03602532.2019.1632886%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520Reviews%26atitle%3DDrug%252Bmetabolizing%252Benzymes-associated%252Bchemo%252Bresistance%252Band%252Bstrategies%252Bto%252Bovercome%252Bit%26aulast%3DVerma%26aufirst%3DHimanshu%26date%3D2019%26date%3D2019%26volume%3D51%26issue%3D2%26spage%3D196%26epage%3D223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nkechiyere  Nwani</span>, <span class="hlFld-ContribAuthor ">Salvatore  Condello</span>, <span class="hlFld-ContribAuthor ">Yinu  Wang</span>, <span class="hlFld-ContribAuthor ">Wendy  Swetzig</span>, <span class="hlFld-ContribAuthor ">Emma  Barber</span>, <span class="hlFld-ContribAuthor ">Thomas  Hurley</span>, <span class="hlFld-ContribAuthor ">Daniela  Matei</span>. </span><span class="cited-content_cbyCitation_article-title">A Novel ALDH1A1 Inhibitor Targets Cells with Stem Cell Characteristics in Ovarian Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (4)
                                     , 502. <a href="https://doi.org/10.3390/cancers11040502" title="DOI URL">https://doi.org/10.3390/cancers11040502</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11040502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11040502%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DA%252BNovel%252BALDH1A1%252BInhibitor%252BTargets%252BCells%252Bwith%252BStem%252BCell%252BCharacteristics%252Bin%252BOvarian%252BCancer%26aulast%3DNwani%26aufirst%3DNkechiyere%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D4%26spage%3D502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ilana  Chefetz</span>, <span class="hlFld-ContribAuthor ">Edward  Grimley</span>, <span class="hlFld-ContribAuthor ">Kun  Yang</span>, <span class="hlFld-ContribAuthor ">Linda  Hong</span>, <span class="hlFld-ContribAuthor ">Ekaterina V.  Vinogradova</span>, <span class="hlFld-ContribAuthor ">Radu  Suciu</span>, <span class="hlFld-ContribAuthor ">Ilya  Kovalenko</span>, <span class="hlFld-ContribAuthor ">David  Karnak</span>, <span class="hlFld-ContribAuthor ">Cynthia A.  Morgan</span>, <span class="hlFld-ContribAuthor ">Mikhail  Chtcherbinine</span>, <span class="hlFld-ContribAuthor ">Cameron  Buchman</span>, <span class="hlFld-ContribAuthor ">Brandt  Huddle</span>, <span class="hlFld-ContribAuthor ">Scott  Barraza</span>, <span class="hlFld-ContribAuthor ">Meredith  Morgan</span>, <span class="hlFld-ContribAuthor ">Kara A.  Bernstein</span>, <span class="hlFld-ContribAuthor ">Euisik  Yoon</span>, <span class="hlFld-ContribAuthor ">David B.  Lombard</span>, <span class="hlFld-ContribAuthor ">Andrea  Bild</span>, <span class="hlFld-ContribAuthor ">Geeta  Mehta</span>, <span class="hlFld-ContribAuthor ">Iris  Romero</span>, <span class="hlFld-ContribAuthor ">Chun-Yi  Chiang</span>, <span class="hlFld-ContribAuthor ">Charles  Landen</span>, <span class="hlFld-ContribAuthor ">Benjamin  Cravatt</span>, <span class="hlFld-ContribAuthor ">Thomas D.  Hurley</span>, <span class="hlFld-ContribAuthor ">Scott D.  Larsen</span>, <span class="hlFld-ContribAuthor ">Ronald J.  Buckanovich</span>. </span><span class="cited-content_cbyCitation_article-title">A Pan-ALDH1A Inhibitor Induces Necroptosis in Ovarian Cancer Stem-like Cells. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2019,</strong> <em>26 </em>
                                    (11)
                                     , 3061-3075.e6. <a href="https://doi.org/10.1016/j.celrep.2019.02.032" title="DOI URL">https://doi.org/10.1016/j.celrep.2019.02.032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2019.02.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2019.02.032%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DA%252BPan-ALDH1A%252BInhibitor%252BInduces%252BNecroptosis%252Bin%252BOvarian%252BCancer%252BStem-like%252BCells%26aulast%3DChefetz%26aufirst%3DIlana%26date%3D2019%26volume%3D26%26issue%3D11%26spage%3D3061%26epage%3D3075.e6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppe  Vassalli</span>. </span><span class="cited-content_cbyCitation_article-title">Aldehyde Dehydrogenases: Not Just Markers, but Functional Regulators of Stem Cells. </span><span class="cited-content_cbyCitation_journal-name">Stem Cells International</span><span> <strong>2019,</strong> <em>2019 </em>, 1-15. <a href="https://doi.org/10.1155/2019/3904645" title="DOI URL">https://doi.org/10.1155/2019/3904645</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2019/3904645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2019%2F3904645%26sid%3Dliteratum%253Aachs%26jtitle%3DStem%2520Cells%2520International%26atitle%3DAldehyde%252BDehydrogenases%25253A%252BNot%252BJust%252BMarkers%25252C%252Bbut%252BFunctional%252BRegulators%252Bof%252BStem%252BCells%26aulast%3DVassalli%26aufirst%3DGiuseppe%26date%3D2019%26volume%3D2019%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative ALDH inhibitors <b>1â6</b> and lead compound <b>7</b> (CM39).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>7</b>, <b>11</b>, <b>12a</b>â<b>d</b>,<b>f</b>â<b>h</b>, and <b>13a</b>â<b>h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ArNCS, toluene, reflux; (b) 1N NaOH, reflux; (c) 3-F-PhCH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, RT; (d) R<sup>1</sup>-X, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 Â°C X = Br, OMs</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Analogues <b>12e</b>, <b>15a</b>â<b>m</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, CS<sub>2</sub>, I<sub>2</sub>, THF, 0â40 Â°C; (b) R<sup>3</sup>-NH<sub>2</sub>, 3-F-PhCH<sub>2</sub>Br, NaH, DMF, 0â20 Â°C; (c) (i) R<sup>3</sup>-NH<sub>2</sub>, NaH, DMF, 0â20 Â°C; (ii) 3-F-PhCH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, RT; (d) (i) MeNH<sub>2</sub>, EtOH; (ii) 3-F-PhCH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, RT</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Fraction of enzyme activity remaining for 9 ALDH isoforms in the presence of 20 Î¼M <b>7</b>. (B) Percent control ALDH<sup>Bright</sup> cells remaining following treatment of PEO1 cells at the specified concentration with DEAB (<b>1</b>) or compound <b>7</b> (<i>N</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of ALDH1A1 N121S with <b>7</b> (2.10 Ã, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI">5TEI</a>). (A) The fluorophenyl of compound <b>7</b> projects into the lipophilic pocket toward the catalytic cysteine (C303). Key residues are labeled. (B) Overlay of compounds <b>4</b> (cyan, PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a>), <b>5a</b> (gray, PDB Entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L">4X4L</a>), and <b>7</b> (orange) bound in ALDH1A1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fraction of enzyme activity remaining for 9 ALDH isoforms in the presence of 20 Î¼M <b>15l</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) X-ray crystal structure of <b>13g</b> bound to ALDH1A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DUM">6DUM</a>). (B) Overlay of compounds <b>7</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI">5TEI</a>) (orange) and <b>13g</b> (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. ALDH1A2 and 1A3 homology models provide rationale for isoform selectivity of Compound <b>7</b>. (A) Val460 does not clash with <i>o</i>-methyl substituent of <i>N</i>-Phenyl in ALDH1A1- Compound <b>7</b> structure. (B, C) Docking poses of <b>7</b> in ALDH1A2 and ALDH1A3, respectively. Red arrow indicates that in both ALDH1A2 and A3, 2-methyl substituent at phenyl clashes with Leu477/471.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>11</b> docks without clashes into the active sites of ALDH1A1, 1A2 and 1A3. (A) Structure and ligand interaction plot of Compound <b>11</b> docked into ALDH1A1 crystal structure. (B,C) Structure and ligand interaction plot of compound <b>11</b> docked into ALDH1A2, 1A3 homology models, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Docking of compound <b>12g</b> predicts a similar binding mode to the <b>13g</b> crystal structure for ALDH1A1, 1A2, and 1A3. (A) Structure and ligand interaction plot of Compound <b>12g</b> docked into ALDH1A1 crystal structure. (B,C) Structure and ligand interaction plot of compound <b>12g</b> docked into ALDH1A2, 1A3 homology models, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. ALDH1A3 specific residue Glu135 may explain the observed selectivity of compound <b>15g</b>. (A) <b>15g</b> docked into the ALDH1A3 homology model. (B) Overlay of ALDH1A1 (pink) crystal structure with <b>15g</b> docked in the 1A3 homology model (cyan).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compounds <b>15l</b>, <b>12g</b>, <b>13g</b>, and <b>13h</b> inhibit ALDH in cells and deplete cells bearing the putative CSC marker CD133. (A,B) Graphs showing changes in ALDEFLUOR activity in PEO1 (A) and OVCAR5 (B) cells following the specified treatment (<i>N</i> = 3). (C,D) Graphs showing changes in the percent of CD133+ cells as determined by flow cytometry following 48 h drug treatment at the specified concentration (<i>N</i> = 3â6) *<i>p</i> â¤ 0.05, **<i>p</i> â¤ 0.01, ***<i>p</i> â¤ 0.0001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/medium/jm-2018-00930g_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Compounds <b>12g</b>, <b>13g</b>, and <b>13h</b> synergistically enhance the effect of cisplatin on ovarian cancer cell lines and patient derived spheroids. (A,B) Representative heatmaps of combination indices for PEO1 (A) and OVCAR5 (B) cells cultured for 72 h in the presence of cisplatin and the indicated ALDH inhibitor at the concentrations shown. (<i>N</i> = 3) (C) Normalized average viability as determined using Alamarblue dye following exposure to the specified compounds for 72 h. (<i>N</i> = 5â8) *<i>p</i> â¤ 0.05, ****<i>p</i> â¤ 0.0001 one-way ANOVA compared to Control. Combination indices for the specified drug combinations calculated using CompuSyn software. (CI < 1 indicates synergy) (<a href="http://www.combosyn.com" class="extLink">http://www.combosyn.com</a>).<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-19/acs.jmedchem.8b00930/20181004/images/large/jm-2018-00930g_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00930&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72453" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72453" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 75 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span> <i>Cancer Facts & Figures</i>; <span class="NLM_publisher-name">American Cancer
Society</span>: <span class="NLM_publisher-loc">Atlanta, GA</span>, <span class="NLM_year">2015</span>. See the following: <a href="https://www.cancer.org/content/dam/cancer-org/research/cancer-facts-and-statistics/annual-cancer-facts-and-figures/2015/cancer-facts-and-figures-2015.pdf" class="extLink">https://www.cancer.org/content/dam/cancer-org/research/cancer-facts-and-statistics/annual-cancer-facts-and-figures/2015/cancer-facts-and-figures-2015.pdf</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Cancer+Facts+%26+Figures%3B+American+Cancer%0ASociety%3A+Atlanta%2C+GA%2C+2015.+See+the+following%3A+https%3A%2F%2Fwww.cancer.org%2Fcontent%2Fdam%2Fcancer-org%2Fresearch%2Fcancer-facts-and-statistics%2Fannual-cancer-facts-and-figures%2F2015%2Fcancer-facts-and-figures-2015.pdf."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Facts%2520%2526%2520Figures%26pub%3DAmerican%2520Cancer%250ASociety%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozols, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bundy, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greer, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke-Pearson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burger, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannel, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGeest, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartenbach, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baergen, R.</span></span> <span> </span><span class="NLM_article-title">Phase III Trial of Carboplatin and Paclitaxel Compared With Cisplatin and Paclitaxel in Patients With Optimally Resected Stage III Ovarian Cancer: A Gynecologic Oncology Group Study</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">3194</span>â <span class="NLM_lpage">3200</span>, <span class="refDoi">Â DOI: 10.1200/JCO.2003.02.153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1200%2FJCO.2003.02.153" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=12860964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD2cXps1aktbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2003&pages=3194-3200&issue=17&author=R.+F.+Ozolsauthor=B.+N.+Bundyauthor=B.+E.+Greerauthor=J.+M.+Fowlerauthor=D.+Clarke-Pearsonauthor=R.+A.+Burgerauthor=R.+S.+Mannelauthor=K.+DeGeestauthor=E.+M.+Hartenbachauthor=R.+Baergen&title=Phase+III+Trial+of+Carboplatin+and+Paclitaxel+Compared+With+Cisplatin+and+Paclitaxel+in+Patients+With+Optimally+Resected+Stage+III+Ovarian+Cancer%3A+A+Gynecologic+Oncology+Group+Study&doi=10.1200%2FJCO.2003.02.153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Phase III trial of carboplatin and paclitaxel compared with cisplatin and paclitaxel in patients with optimally resected stage III ovarian cancer: a Gynecologic Oncology Group Study</span></div><div class="casAuthors">Ozols, Robert F.; Bundy, Brian N.; Greer, Benjamin E.; Fowler, Jeffrey M.; Clarke-Pearson, Daniel; Burger, Robert A.; Mannel, Robert S.; de Geest, Koen; Hartenbach, Ellen M.; Baergen, Rebecca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3194-3200</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">In randomized trials the combination of cisplatin and paclitaxel was superior to cisplatin and cyclophosphamide in advanced-stage epithelial ovarian cancer.  Although in nonrandomized trials, carboplatin and paclitaxel was a less toxic and highly active combination regimen, there remained concern regarding its efficacy in patients with small-vol., resected, stage III disease.  Thus, we conducted a noninferiority trial of cisplatin and paclitaxel vs. carboplatin and paclitaxel in this population.  Patients with advanced ovarian cancer and no residual mass greater than 1.0 cm after surgery were randomly assigned to receive cisplatin 75 mg/m2 plus a 24-h infusion of paclitaxel 135 mg/m2 (arm I), or carboplatin area under the curve 7.5 i.v. plus paclitaxel 175 mg/m2 over 3 h (arm II).  Seven hundred ninety-two eligible patients were enrolled onto the study.  Prognostic factors were similar in the two treatment groups.  Gastrointestinal, renal, and metabolic toxicity, as well as grade 4 leukopenia, were significantly more frequent in arm I.  Grade 2 or greater thrombocytopenia was more common in arm II.  Neurol. toxicity was similar in both regimens.  Median progression-free survival and overall survival were 19.4 and 48.7 mo, resp., for arm I compared with 20.7 and 57.4 mo, resp., for arm II.  The relative risk (RR) of progression for the carboplatin plus paclitaxel group was 0.88 (95% confidence interval [CI], 0.75 to 1.03) and the RR of death was 0.84 (95% CI, 0.70 to 1.02).  In patients with advanced ovarian cancer, a chemotherapy regimen consisting of carboplatin plus paclitaxel results in less toxicity, is easier to administer, and is not inferior, when compared with cisplatin plus paclitaxel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob190CVtYZirVg90H21EOLACvtfcHk0lhPQNUKOgrMpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXps1aktbk%253D&md5=83a519c6c3759f8ac7e8fc86444f546f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1200%2FJCO.2003.02.153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2003.02.153%26sid%3Dliteratum%253Aachs%26aulast%3DOzols%26aufirst%3DR.%2BF.%26aulast%3DBundy%26aufirst%3DB.%2BN.%26aulast%3DGreer%26aufirst%3DB.%2BE.%26aulast%3DFowler%26aufirst%3DJ.%2BM.%26aulast%3DClarke-Pearson%26aufirst%3DD.%26aulast%3DBurger%26aufirst%3DR.%2BA.%26aulast%3DMannel%26aufirst%3DR.%2BS.%26aulast%3DDeGeest%26aufirst%3DK.%26aulast%3DHartenbach%26aufirst%3DE.%2BM.%26aulast%3DBaergen%26aufirst%3DR.%26atitle%3DPhase%2520III%2520Trial%2520of%2520Carboplatin%2520and%2520Paclitaxel%2520Compared%2520With%2520Cisplatin%2520and%2520Paclitaxel%2520in%2520Patients%2520With%2520Optimally%2520Resected%2520Stage%2520III%2520Ovarian%2520Cancer%253A%2520A%2520Gynecologic%2520Oncology%2520Group%2520Study%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2003%26volume%3D21%26issue%3D17%26spage%3D3194%26epage%3D3200%26doi%3D10.1200%2FJCO.2003.02.153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thigpen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">duBois, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlpine, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiSaia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoskins, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristensen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfisterer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swart, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallardo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poveda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinstein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitchener, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, G. C.</span></span> <span> </span><span class="NLM_article-title">First-line Therapy in Ovarian Cancer Trials</span>. <i>Int. J. Gynecol Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">756</span>â <span class="NLM_lpage">762</span>, <span class="refDoi">Â DOI: 10.1097/IGC.0b013e31821ce75d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1097%2FIGC.0b013e31821ce75d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21543937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A280%3ADC%252BC3MnitVWltg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=756-762&issue=4&author=T.+Thigpenauthor=A.+duBoisauthor=J.+McAlpineauthor=P.+DiSaiaauthor=K.+Fujiwaraauthor=W.+Hoskinsauthor=G.+Kristensenauthor=R.+Mannelauthor=M.+Markmanauthor=J.+Pfistererauthor=M.+Quinnauthor=N.+Reedauthor=A.+M.+Swartauthor=J.+Berekauthor=N.+Colomboauthor=G.+Freyerauthor=D.+Gallardoauthor=M.+Planteauthor=A.+Povedaauthor=L.+Rubinsteinauthor=M.+Baconauthor=H.+Kitchenerauthor=G.+C.+Stuart&title=First-line+Therapy+in+Ovarian+Cancer+Trials&doi=10.1097%2FIGC.0b013e31821ce75d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">First-line therapy in ovarian cancer trials</span></div><div class="casAuthors">Thigpen Tate; duBois Andreas; McAlpine Jessica; DiSaia Philip; Fujiwara Keiichi; Hoskins William; Kristensen Gunnar; Mannel Robert; Markman Maurie; Pfisterer Jacobus; Quinn Michael; Reed Nick; Swart Ann Marie; Berek Jonathan; Colombo Nicoletta; Freyer Gilles; Gallardo Dolores; Plante Marie; Poveda Andres; Rubinstein Lawrence; Bacon Monica; Kitchener Henry; Stuart Gavin C E</div><div class="citationInfo"><span class="NLM_cas:title">International journal of gynecological cancer : official journal of the International Gynecological Cancer Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">756-62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">At the 4th Ovarian Cancer Consensus Conference of the Gynecologic Cancer InterGroup (GCIG) held in Vancouver, Canada, in June 2010, representatives of 23 cooperative research groups studying gynecologic cancers gathered to establish international consensus on issues critical to the conduct of large randomized trials.  The process focused on 13 predetermined questions.  Group A, 1 of the 3 discussion groups, addressed the first 5 questions, examining first-line therapies in newly diagnosed ovarian cancer patients.  A1: What are the appropriate end points for different trials (maintenance, upfront chemotherapy trials including molecular drugs)? A2: Are there any subgroups defined by tumor biology who need specific treatment options/trials? A3: Is the 2004 GCIG-recommended standard comparator arm still valid? A4: What is the role of modifying dose, schedule, and delivery of chemotherapy? A5: What role does surgery play today?</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZIyqqc7wuZub6AVA-44PTfW6udTcc2ebA9zzfZ8eRJ7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MnitVWltg%253D%253D&md5=e6cb9d1cd93b8099f3e7430b803c1fb1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1097%2FIGC.0b013e31821ce75d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FIGC.0b013e31821ce75d%26sid%3Dliteratum%253Aachs%26aulast%3DThigpen%26aufirst%3DT.%26aulast%3DduBois%26aufirst%3DA.%26aulast%3DMcAlpine%26aufirst%3DJ.%26aulast%3DDiSaia%26aufirst%3DP.%26aulast%3DFujiwara%26aufirst%3DK.%26aulast%3DHoskins%26aufirst%3DW.%26aulast%3DKristensen%26aufirst%3DG.%26aulast%3DMannel%26aufirst%3DR.%26aulast%3DMarkman%26aufirst%3DM.%26aulast%3DPfisterer%26aufirst%3DJ.%26aulast%3DQuinn%26aufirst%3DM.%26aulast%3DReed%26aufirst%3DN.%26aulast%3DSwart%26aufirst%3DA.%2BM.%26aulast%3DBerek%26aufirst%3DJ.%26aulast%3DColombo%26aufirst%3DN.%26aulast%3DFreyer%26aufirst%3DG.%26aulast%3DGallardo%26aufirst%3DD.%26aulast%3DPlante%26aufirst%3DM.%26aulast%3DPoveda%26aufirst%3DA.%26aulast%3DRubinstein%26aufirst%3DL.%26aulast%3DBacon%26aufirst%3DM.%26aulast%3DKitchener%26aufirst%3DH.%26aulast%3DStuart%26aufirst%3DG.%2BC.%26atitle%3DFirst-line%2520Therapy%2520in%2520Ovarian%2520Cancer%2520Trials%26jtitle%3DInt.%2520J.%2520Gynecol%2520Cancer%26date%3D2011%26volume%3D21%26issue%3D4%26spage%3D756%26epage%3D762%26doi%3D10.1097%2FIGC.0b013e31821ce75d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cannistra, S. A.</span></span> <span> </span><span class="NLM_article-title">Cancer of the Ovary</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>351</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">2519</span>â <span class="NLM_lpage">2529</span>, <span class="refDoi">Â DOI: 10.1056/NEJMra041842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1056%2FNEJMra041842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=15590954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGksbbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2004&pages=2519-2529&issue=24&author=S.+A.+Cannistra&title=Cancer+of+the+Ovary&doi=10.1056%2FNEJMra041842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer of the ovary</span></div><div class="casAuthors">Cannistra, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2519-2529</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Epithelial cancer of the ovary is a relatively uncommon gynecol. cancer in the United States, with approx. 25,580 new cases and 16,090 deaths anticipated in 2004.  Most patients present with advanced disease, which is managed with surgical resection followed by platinum-based chemotherapy.  During the past decade, advances in chemotherapy have resulted in improved survival and in more effective treatment of relapsed disease.  In addn., a better understanding of genetic risk factors has permitted a tailored approach to preventive strategies, such as bilateral salpingo-oophorectomy in selected women.  This review describes the clin. features of epithelial ovarian cancer, with an emphasis on recent advances in postoperative management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSCl_VGPoSdLVg90H21EOLACvtfcHk0lhPQNUKOgrMpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGksbbM&md5=e9f22d658bf6281d6e99ce86c192ac5c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1056%2FNEJMra041842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra041842%26sid%3Dliteratum%253Aachs%26aulast%3DCannistra%26aufirst%3DS.%2BA.%26atitle%3DCancer%2520of%2520the%2520Ovary%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2004%26volume%3D351%26issue%3D24%26spage%3D2519%26epage%3D2529%26doi%3D10.1056%2FNEJMra041842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landen, C. N.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katre, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steg, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nick, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mejia, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershenson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span> <span> </span><span class="NLM_article-title">Targeting Aldehyde Dehydrogenase Cancer Stem Cells in Ovarian Cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3186</span>â <span class="NLM_lpage">3199</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.MCT-10-0563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F1535-7163.MCT-10-0563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=20889728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsL%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=3186-3199&issue=12&author=C.+N.+Landenauthor=B.+Goodmanauthor=A.+A.+Katreauthor=A.+D.+Stegauthor=A.+M.+Nickauthor=R.+L.+Stoneauthor=L.+D.+Millerauthor=P.+V.+Mejiaauthor=N.+B.+Jenningsauthor=D.+M.+Gershensonauthor=R.+C.+Bastauthor=R.+L.+Colemanauthor=G.+Lopez-Beresteinauthor=A.+K.+Sood&title=Targeting+Aldehyde+Dehydrogenase+Cancer+Stem+Cells+in+Ovarian+Cancer&doi=10.1158%2F1535-7163.MCT-10-0563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Aldehyde Dehydrogenase Cancer Stem Cells in Ovarian Cancer</span></div><div class="casAuthors">Landen, Charles N., Jr.; Goodman, Blake; Katre, Ashwini A.; Steg, Adam D.; Nick, Alpa M.; Stone, Rebecca L.; Miller, Lance D.; Mejia, Pablo Vivas; Jennings, Nicolas B.; Gershenson, David M.; Bast, Robert C., Jr.; Coleman, Robert L.; Lopez-Berestein, Gabriel; Sood, Anil K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3186-3199</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenase-1A1 (ALDH1A1) expression characterizes a subpopulation of cells with tumor-initiating or cancer stem cell properties in several malignancies.  Our goal was to characterize the phenotype of ALDH1A1-pos. ovarian cancer cells and examine the biol. effects of ALDH1A1 gene silencing.  In our anal. of multiple ovarian cancer cell lines, we found that ALDH1A1 expression and activity was significantly higher in taxane- and platinum-resistant cell lines.  In patient samples, 72.9% of ovarian cancers had ALDH1A1 expression in which the percentage of ALDH1A1-pos. cells correlated neg. with progression-free survival (6.05 vs. 13.81 mo; P < 0.035).  Subpopulations of A2780cp20 cells with ALDH1A1 activity were isolated for orthotopic tumor-initiating studies, where tumorigenicity was approx. 50-fold higher with ALDH1A1-pos. cells.  Interestingly, tumors derived from ALDH1A1-pos. cells gave rise to both ALDH1A1-pos. and ALDH1A1-neg. populations, but ALDH1A1-neg. cells could not generate ALDH1A1-pos. cells.  In an in vivo orthotopic mouse model of ovarian cancer, ALDH1A1 silencing using nanoliposomal siRNA sensitized both taxane- and platinum-resistant cell lines to chemotherapy, significantly reducing tumor growth in mice compared with chemotherapy alone (a 74%-90% redn.; P < 0.015).  These data show that the ALDH1A1 subpopulation is assocd. with chemoresistance and outcome in ovarian cancer patients, and targeting ALDH1A1 sensitizes resistant cells to chemotherapy.  ALDH1A1-pos. cells have enhanced, but not abs., tumorigenicity but do have differentiation capacity lacking in ALDH1A1-neg. cells.  This enzyme may be important for identification and targeting of chemoresistant cell populations in ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeKpHNJYkfVLVg90H21EOLACvtfcHk0lhSXB7qszMp7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsL%252FP&md5=2ddb105fb819e82928e7c95994ee317e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-10-0563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-10-0563%26sid%3Dliteratum%253Aachs%26aulast%3DLanden%26aufirst%3DC.%2BN.%26aulast%3DGoodman%26aufirst%3DB.%26aulast%3DKatre%26aufirst%3DA.%2BA.%26aulast%3DSteg%26aufirst%3DA.%2BD.%26aulast%3DNick%26aufirst%3DA.%2BM.%26aulast%3DStone%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DL.%2BD.%26aulast%3DMejia%26aufirst%3DP.%2BV.%26aulast%3DJennings%26aufirst%3DN.%2BB.%26aulast%3DGershenson%26aufirst%3DD.%2BM.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DColeman%26aufirst%3DR.%2BL.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DSood%26aufirst%3DA.%2BK.%26atitle%3DTargeting%2520Aldehyde%2520Dehydrogenase%2520Cancer%2520Stem%2520Cells%2520in%2520Ovarian%2520Cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26issue%3D12%26spage%3D3186%26epage%3D3199%26doi%3D10.1158%2F1535-7163.MCT-10-0563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steg, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevis, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katre, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobbin, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landen, C. N.</span></span> <span> </span><span class="NLM_article-title">Stem Cell Pathways Contribute to Clinical Chemoresistance in Ovarian Cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">869</span>â <span class="NLM_lpage">881</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-11-2188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F1078-0432.CCR-11-2188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22142828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOku7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=869-881&author=A.+D.+Stegauthor=K.+S.+Bevisauthor=A.+A.+Katreauthor=A.+Ziebarthauthor=Z.+C.+Dobbinauthor=R.+D.+Alvarezauthor=K.+Zhangauthor=M.+Connerauthor=C.+N.+Landen&title=Stem+Cell+Pathways+Contribute+to+Clinical+Chemoresistance+in+Ovarian+Cancer&doi=10.1158%2F1078-0432.CCR-11-2188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Stem Cell Pathways Contribute to Clinical Chemoresistance in Ovarian Cancer</span></div><div class="casAuthors">Steg, Adam D.; Bevis, Kerri S.; Katre, Ashwini A.; Ziebarth, Angela; Dobbin, Zachary C.; Alvarez, Ronald D.; Zhang, Kui; Conner, Michael; Landen, Charles N.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">869-881</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Within heterogeneous tumors, subpopulations often labeled cancer stem cells (CSC) have been identified that have enhanced tumorigenicity and chemoresistance in ex vivo models.  However, whether these populations are more capable of surviving chemotherapy in de novo tumors is unknown.  Exptl. Design: We examd. 45 matched primary/recurrent tumor pairs of high-grade ovarian adenocarcinomas for expression of CSC markers ALDH1A1, CD44, and CD133 using immunohistochem.  Tumors collected immediately after completion of primary therapy were then laser capture microdissected and subjected to a quant. PCR array examg. stem cell biol. pathways (Hedgehog, Notch, TGF-Î², and Wnt).  Select genes of interest were validated as important targets using siRNA-mediated downregulation.  RESULTS: Primary samples were composed of low densities of ALDH1A1, CD44, and CD133.  Tumors collected immediately after primary therapy were more densely composed of each marker, whereas samples collected at first recurrence, before initiating secondary therapy, were composed of similar percentages of each marker as their primary tumor.  In tumors collected from recurrent platinum-resistant patients, only CD133 was significantly increased.  Of stem cell pathway members examd., 14% were significantly overexpressed in recurrent compared with matched primary tumors.  Knockdown of genes of interest, including endoglin/CD105 and the hedgehog mediators Gli1 and Gli2, led to decreased ovarian cancer cell viability, with Gli2 showing a novel contribution to cisplatin resistance.  CONCLUSIONS: These data indicate that ovarian tumors are enriched with CSCs and stem cell pathway mediators, esp. at the completion of primary therapy.  This suggests that stem cell subpopulations contribute to tumor chemoresistance and ultimately recurrent disease.  Clin Cancer Res; 18(3); 869-81.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvmyFTyD3DorVg90H21EOLACvtfcHk0lhSXB7qszMp7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOku7k%253D&md5=3dd7be6381c376fa6caacafb86b8c85a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-2188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-2188%26sid%3Dliteratum%253Aachs%26aulast%3DSteg%26aufirst%3DA.%2BD.%26aulast%3DBevis%26aufirst%3DK.%2BS.%26aulast%3DKatre%26aufirst%3DA.%2BA.%26aulast%3DZiebarth%26aufirst%3DA.%26aulast%3DDobbin%26aufirst%3DZ.%2BC.%26aulast%3DAlvarez%26aufirst%3DR.%2BD.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DConner%26aufirst%3DM.%26aulast%3DLanden%26aufirst%3DC.%2BN.%26atitle%3DStem%2520Cell%2520Pathways%2520Contribute%2520to%2520Clinical%2520Chemoresistance%2520in%2520Ovarian%2520Cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D869%26epage%3D881%26doi%3D10.1158%2F1078-0432.CCR-11-2188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silva, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginestier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kueck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicha, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckanovich, R. J.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenase in Combination with CD133 Defines Angiogenic Ovarian Cancer Stem Cells that Portend Poor Patient Survival</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3991</span>â <span class="NLM_lpage">4001</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-10-3175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F0008-5472.CAN-10-3175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21498635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFWksrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=3991-4001&issue=11&author=I.+A.+Silvaauthor=S.+Baiauthor=K.+McLeanauthor=K.+Yangauthor=K.+Griffithauthor=D.+Thomasauthor=C.+Ginestierauthor=C.+Johnstonauthor=A.+Kueckauthor=R.+K.+Reynoldsauthor=M.+S.+Wichaauthor=R.+J.+Buckanovich&title=Aldehyde+Dehydrogenase+in+Combination+with+CD133+Defines+Angiogenic+Ovarian+Cancer+Stem+Cells+that+Portend+Poor+Patient+Survival&doi=10.1158%2F0008-5472.CAN-10-3175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde Dehydrogenase in Combination with CD133 Defines Angiogenic Ovarian Cancer Stem Cells That Portend Poor Patient Survival</span></div><div class="casAuthors">Silva, Ines A.; Bai, Shoumei; McLean, Karen; Yang, Kun; Griffith, Kent; Thomas, Dafydd; Ginestier, Christophe; Johnston, Carolyn; Kueck, Angela; Reynolds, R. Kevin; Wicha, Max S.; Buckanovich, Ronald J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3991-4001</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Markers that reliably identify cancer stem cells (CSC) in ovarian cancer could assist prognosis and improve strategies for therapy.  CD133 is a reported marker of ovarian CSC.  Aldehyde dehydrogenase (ALDH) activity is a reported CSC marker in several solid tumors, but it was not studied in ovarian CSC.  Here we report that dual positivity of CD133 and ALDH defines a compelling marker set in ovarian CSC.  All human ovarian tumors and cell lines displayed ALDH activity.  ALDH+ cells isolated from ovarian cancer cell lines were chemoresistant and preferentially grew tumors, compared with ALDH- cells, validating ALDH as a marker of ovarian CSC in cell lines.  Notably, as few as 1,000 ALDH+ cells isolated directly from CD133- human ovarian tumors were sufficient to generate tumors in immunocompromised mice, whereas 50,000 ALDH- cells were unable to initiate tumors.  Using ALDH in combination with CD133 to analyze ovarian cancer cell lines, we obsd. even greater growth in the ALDH+CD133+ cells compared with ALDH+CD133- cells, suggesting a further enrichment of ovarian CSC in ALDH+CD133+ cells.  Strikingly, as few as 11 ALDH+CD133+ cells isolated directly from human tumors were sufficient to initiate tumors in mice.  Like other CSC, ovarian CSC exhibited increased angiogenic capacity compared with bulk tumor cells.  Finally, the presence of ALDH+CD133+ cells in debulked primary tumor specimens correlated with reduced disease-free and overall survival in ovarian cancer patients.  Taken together,these findings define ALDH and CD133 as a functionally significant set of markers to identify ovarian CSCs.  Cancer Res; 71(11); 3991-4001.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy2Tirx6N8kbVg90H21EOLACvtfcHk0lhSXB7qszMp7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFWksrs%253D&md5=b5719c401bcb6ed36e2d831f0356cf5a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-3175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-3175%26sid%3Dliteratum%253Aachs%26aulast%3DSilva%26aufirst%3DI.%2BA.%26aulast%3DBai%26aufirst%3DS.%26aulast%3DMcLean%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DGriffith%26aufirst%3DK.%26aulast%3DThomas%26aufirst%3DD.%26aulast%3DGinestier%26aufirst%3DC.%26aulast%3DJohnston%26aufirst%3DC.%26aulast%3DKueck%26aufirst%3DA.%26aulast%3DReynolds%26aufirst%3DR.%2BK.%26aulast%3DWicha%26aufirst%3DM.%2BS.%26aulast%3DBuckanovich%26aufirst%3DR.%2BJ.%26atitle%3DAldehyde%2520Dehydrogenase%2520in%2520Combination%2520with%2520CD133%2520Defines%2520Angiogenic%2520Ovarian%2520Cancer%2520Stem%2520Cells%2520that%2520Portend%2520Poor%2520Patient%2520Survival%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26issue%3D11%26spage%3D3991%26epage%3D4001%26doi%3D10.1158%2F0008-5472.CAN-10-3175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingram, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyengar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckanovich, R. J.</span></span> <span> </span><span class="NLM_article-title">Identifying an Ovarian Cancer Cell Hierarchy Regulated by Bone Morphogenetic Protein 2</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span> (<span class="NLM_issue">50</span>),  <span class="NLM_fpage">E6882</span>â <span class="NLM_lpage">E6888</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1507899112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1073%2Fpnas.1507899112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E6882-E6888&issue=50&author=Y.-J.+Choiauthor=P.+N.+Ingramauthor=K.+Yangauthor=L.+Coffmanauthor=M.+Iyengarauthor=S.+Baiauthor=D.+G.+Thomasauthor=E.+Yoonauthor=R.+J.+Buckanovich&title=Identifying+an+Ovarian+Cancer+Cell+Hierarchy+Regulated+by+Bone+Morphogenetic+Protein+2&doi=10.1073%2Fpnas.1507899112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1507899112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1507899112%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DY.-J.%26aulast%3DIngram%26aufirst%3DP.%2BN.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DCoffman%26aufirst%3DL.%26aulast%3DIyengar%26aufirst%3DM.%26aulast%3DBai%26aufirst%3DS.%26aulast%3DThomas%26aufirst%3DD.%2BG.%26aulast%3DYoon%26aufirst%3DE.%26aulast%3DBuckanovich%26aufirst%3DR.%2BJ.%26atitle%3DIdentifying%2520an%2520Ovarian%2520Cancer%2520Cell%2520Hierarchy%2520Regulated%2520by%2520Bone%2520Morphogenetic%2520Protein%25202%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26issue%3D50%26spage%3DE6882%26epage%3DE6888%26doi%3D10.1073%2Fpnas.1507899112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Strategies Targeting Cancer Stem Cells</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">5</span>â <span class="NLM_lpage">11</span>, <span class="refDoi">Â DOI: 10.1111/cas.12817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1111%2Fcas.12817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=26362755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7bM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=5-11&issue=1&author=G.+J.+Yoshidaauthor=H.+Saya&title=Therapeutic+Strategies+Targeting+Cancer+Stem+Cells&doi=10.1111%2Fcas.12817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic strategies targeting cancer stem cells</span></div><div class="casAuthors">Yoshida, Go J.; Saya, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-11</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cancer stem cells (CSCs) are undifferentiated cancer cells with a high tumorigenic activity, the ability to undergo self-renewal, and a multilineage differentiation potential.  Cancer stem cells are responsible for the development of tumor cell heterogeneity, a key feature for resistance to anticancer treatments including conventional chemotherapy, radiation therapy, and molecularly targeted therapy.  Furthermore, minimal residual disease, the major cause of cancer recurrence and metastasis, is enriched in CSCs.  Cancer stem cells also possess the property of "robustness", which encompasses several characteristics including a slow cell cycle, the ability to detoxify or mediate the efflux of cytotoxic agents, resistance to oxidative stress, and a rapid response to DNA damage, all of which contribute to the development of therapeutic resistance.  The identification of mechanisms underlying such characteristics and the development of novel approaches to target them will be required for the therapeutic elimination of CSCs and the complete eradication of tumors.  In this review, we focus on two prospective therapeutic approaches that target CSCs with the aim of disrupting their quiescence or redox defense capability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt-ai0C0VCYLVg90H21EOLACvtfcHk0liGpOgGKL2Syg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7bM&md5=043f5210853d44b7cb4f228a86cd90dc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fcas.12817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12817%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DG.%2BJ.%26aulast%3DSaya%26aufirst%3DH.%26atitle%3DTherapeutic%2520Strategies%2520Targeting%2520Cancer%2520Stem%2520Cells%26jtitle%3DCancer%2520Sci.%26date%3D2016%26volume%3D107%26issue%3D1%26spage%3D5%26epage%3D11%26doi%3D10.1111%2Fcas.12817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kryczek, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vatan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szeliga, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotarski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicha, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, W.</span></span> <span> </span><span class="NLM_article-title">Expression of Aldehyde Dehydrogenase and CD133 Defines Ovarian Cancer Stem Cells</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>130</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">29</span>â <span class="NLM_lpage">39</span>, <span class="refDoi">Â DOI: 10.1002/ijc.25967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fijc.25967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21480217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOmsL7N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2012&pages=29-39&issue=1&author=I.+Kryczekauthor=S.+Liuauthor=M.+Rohauthor=L.+Vatanauthor=W.+Szeligaauthor=S.+Weiauthor=M.+Banerjeeauthor=Y.+Maoauthor=J.+Kotarskiauthor=M.+S.+Wichaauthor=R.+Liuauthor=W.+Zou&title=Expression+of+Aldehyde+Dehydrogenase+and+CD133+Defines+Ovarian+Cancer+Stem+Cells&doi=10.1002%2Fijc.25967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of aldehyde dehydrogenase and CD133 defines ovarian cancer stem cells</span></div><div class="casAuthors">Kryczek, Ilona; Liu, Suling; Roh, Michael; Vatan, Linhua; Szeliga, Wojciech; Wei, Shuang; Banerjee, Mousumi; Mao, Yujun; Kotarski, Jan; Wicha, Max S.; Liu, Rebecca; Zou, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-39</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Identification of cancer stem cells is crucial for advancing cancer biol. and therapy.  Several markers including CD24, CD44, CD117, CD133, the G subfamily of ATP-binding cassette transporters (ABCG), epithelial specific antigen (ESA) and aldehyde dehydrogenase (ALDH) are used to identify and investigate human epithelial cancer stem cells in the literature.  We have now systemically analyzed and compared the expression of these markers in fresh ovarian epithelial carcinomas.  Although the expression levels of these markers were unexpectedly variable and partially overlapping in fresh ovarian cancer cells from different donors, we reliably detected important levels of CD133 and ALDH in the majority of fresh ovarian cancer.  Furthermore, most of these stem cell markers including CD133 and ALDH were gradually lost following in vitro passage of primary tumor cells.  However, the expression of ALDH and CD133, but not CD24, CD44 and CD117, could be partially rescued by the in vitro serum-free and sphere cultures and by the in vivo passage in the immune-deficient xenografts.  ALDH+ and CD133+ cells formed three-dimensional spheres more efficiently than their neg. counterparts.  These sphere-forming cells expressed high levels of stem cell core gene transcripts and could be expanded and form addnl. spheres in long-term culture.  ALDH+, CD133+ and ALDH+CD133+ cells from fresh tumors developed larger tumors more rapidly than their neg. counterparts.  This property was preserved in the xenografted tumors.  Altogether, the data suggest that ALDH+ and CD133+ cells are enriched with ovarian cancer-initiating (stem) cells and that ALDH and CD133 may be widely used as reliable markers to investigate ovarian cancer stem cell biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYTMDKETxyV7Vg90H21EOLACvtfcHk0liGpOgGKL2Syg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOmsL7N&md5=cbc76214c56d0ff4d4ca9ddf8549d42a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fijc.25967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25967%26sid%3Dliteratum%253Aachs%26aulast%3DKryczek%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRoh%26aufirst%3DM.%26aulast%3DVatan%26aufirst%3DL.%26aulast%3DSzeliga%26aufirst%3DW.%26aulast%3DWei%26aufirst%3DS.%26aulast%3DBanerjee%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DKotarski%26aufirst%3DJ.%26aulast%3DWicha%26aufirst%3DM.%2BS.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DZou%26aufirst%3DW.%26atitle%3DExpression%2520of%2520Aldehyde%2520Dehydrogenase%2520and%2520CD133%2520Defines%2520Ovarian%2520Cancer%2520Stem%2520Cells%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D130%26issue%3D1%26spage%3D29%26epage%3D39%26doi%3D10.1002%2Fijc.25967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roby, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orsulic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montone, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butzow, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coukos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Distinct Expression Levels and Patterns of Stem Cell Marker, Aldehyde Dehydrogenase Isoform 1 (ALDH1), in Human Epithelial Cancers</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">e10277</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0010277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0010277" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e10277&issue=4&author=S.+Dengauthor=X.+Yangauthor=H.+Lassusauthor=S.+Liangauthor=S.+Kaurauthor=Q.+Yeauthor=C.+Liauthor=L.+P.+Wangauthor=K.+F.+Robyauthor=S.+Orsulicauthor=D.+C.+Connollyauthor=Y.+Zhangauthor=K.+Montoneauthor=R.+Butzowauthor=G.+Coukosauthor=L.+Zhang&title=Distinct+Expression+Levels+and+Patterns+of+Stem+Cell+Marker%2C+Aldehyde+Dehydrogenase+Isoform+1+%28ALDH1%29%2C+in+Human+Epithelial+Cancers&doi=10.1371%2Fjournal.pone.0010277"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0010277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0010277%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLassus%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DKaur%26aufirst%3DS.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DL.%2BP.%26aulast%3DRoby%26aufirst%3DK.%2BF.%26aulast%3DOrsulic%26aufirst%3DS.%26aulast%3DConnolly%26aufirst%3DD.%2BC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMontone%26aufirst%3DK.%26aulast%3DButzow%26aufirst%3DR.%26aulast%3DCoukos%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DDistinct%2520Expression%2520Levels%2520and%2520Patterns%2520of%2520Stem%2520Cell%2520Marker%252C%2520Aldehyde%2520Dehydrogenase%2520Isoform%25201%2520%2528ALDH1%2529%252C%2520in%2520Human%2520Epithelial%2520Cancers%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26issue%3D4%26spage%3De10277%26doi%3D10.1371%2Fjournal.pone.0010277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolluru, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekaran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baby, B. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirimulla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alatassi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankem, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damodaran, C.</span></span> <span> </span><span class="NLM_article-title">Targeting Aberrant Expression of Notch-1 in ALDH(+) Cancer Stem Cells in Breast Cancer</span>. <i>Mol. Carcinog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1127</span>â <span class="NLM_lpage">1136</span>, <span class="refDoi">Â DOI: 10.1002/mc.22579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fmc.22579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=27753148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslyktrzN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=1127-1136&issue=3&author=D.+Palauthor=V.+Kolluruauthor=B.+Chandrasekaranauthor=B.+V.+Babyauthor=M.+Amanauthor=S.+Sumanauthor=S.+Sirimullaauthor=M.+A.+Sandersauthor=H.+Alatassiauthor=M.+K.+Ankemauthor=C.+Damodaran&title=Targeting+Aberrant+Expression+of+Notch-1+in+ALDH%28%2B%29+Cancer+Stem+Cells+in+Breast+Cancer&doi=10.1002%2Fmc.22579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting aberrant expression of Notch-1 in ALDH+ cancer stem cells in breast cancer</span></div><div class="casAuthors">Pal, Deeksha; Kolluru, Venkatesh; Chandrasekaran, Balaji; Baby, Becca V.; Aman, Masarath; Suman, Suman; Sirimulla, Suman; Sanders, Mary Ann; Alatassi, Houda; Ankem, Murali K.; Damodaran, Chendil</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Carcinogenesis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1127-1136</span>CODEN:
                <span class="NLM_cas:coden">MOCAE8</span>;
        ISSN:<span class="NLM_cas:issn">0899-1987</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">We have previously reported that high aldehyde dehydrogenase (ALDH) enzyme activity in breast cancer cells results in breast cancer stem cell (BCSC) properties by upregualting Notch-1 and epithelial mesenchymal markers.  This results in chemoresistance in breast cancer.  Here, we examd. the functional and clin. significance of ALDH expression by measuring the ALDH levels in breast cancer tissues by immunohistochem.  There was a significantly higher ALDH expression in higher grade breast cancer tumor tissues (Grade- II and III) vs. normal breast tissues.  Injection of BCSC (ALDH+ and CD44+/CD22-) cells resulted in aggressive tumor growth in athymic mice vs. ALDH- cells.  The ALDH+ and CD44+/CD22- tumors grow rapidly and are larger than ALDH- tumors which were slow growing and smaller.  Molecularly, ALDH+ tumors expressed higher expression of Notch-1 and EMT markers than ALDH- tumors.  Oral administration of the naturally occurring Psoralidin (Pso, 25 mg/kg of body wt.) significantly inhibited the growth in ALDH+ and ALDH- tumors as well.  Psoralidin inhibited Notch-1 mediated EMT activation in ALDH+ and ALDH- tumors-this confirms our in vitro findings.  Our results suggest that Notch-1 could be an attractive target and inhibition of Notch-1 by Psoralidin may prevent pathogenesis of breast cancer as well as metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU_GYURGg64LVg90H21EOLACvtfcHk0liGpOgGKL2Syg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslyktrzN&md5=6eb43ddd85d0d86f80fc8ddb8381769a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fmc.22579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmc.22579%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DD.%26aulast%3DKolluru%26aufirst%3DV.%26aulast%3DChandrasekaran%26aufirst%3DB.%26aulast%3DBaby%26aufirst%3DB.%2BV.%26aulast%3DAman%26aufirst%3DM.%26aulast%3DSuman%26aufirst%3DS.%26aulast%3DSirimulla%26aufirst%3DS.%26aulast%3DSanders%26aufirst%3DM.%2BA.%26aulast%3DAlatassi%26aufirst%3DH.%26aulast%3DAnkem%26aufirst%3DM.%2BK.%26aulast%3DDamodaran%26aufirst%3DC.%26atitle%3DTargeting%2520Aberrant%2520Expression%2520of%2520Notch-1%2520in%2520ALDH%2528%252B%2529%2520Cancer%2520Stem%2520Cells%2520in%2520Breast%2520Cancer%26jtitle%3DMol.%2520Carcinog.%26date%3D2017%26volume%3D56%26issue%3D3%26spage%3D1127%26epage%3D1136%26doi%3D10.1002%2Fmc.22579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koppaka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juvonen, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deitrich, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenase Inhibitors: a Comprehensive Review of the Pharmacology, Mechanism of Action, Substrate Specificity, and Clinical Application</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>64</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">520</span>â <span class="NLM_lpage">539</span>, <span class="refDoi">Â DOI: 10.1124/pr.111.005538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1124%2Fpr.111.005538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22544865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslaktw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2012&pages=520-539&issue=3&author=V.+Koppakaauthor=D.+C.+Thompsonauthor=Y.+Chenauthor=M.+Ellermannauthor=K.+C.+Nicolaouauthor=R.+O.+Juvonenauthor=D.+Petersenauthor=R.+A.+Deitrichauthor=T.+D.+Hurleyauthor=V.+Vasiliou&title=Aldehyde+Dehydrogenase+Inhibitors%3A+a+Comprehensive+Review+of+the+Pharmacology%2C+Mechanism+of+Action%2C+Substrate+Specificity%2C+and+Clinical+Application&doi=10.1124%2Fpr.111.005538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenase inhibitors: a comprehensive review of the pharmacology, mechanism of action, substrate specificity, and clinical application</span></div><div class="casAuthors">Koppaka, Vindhya; Thompson, David C.; Chen, Ying; Ellermann, Manuel; Nicolaou, Kyriacos C.; Juvonen, Risto O.; Petersen, Dennis; Deitrich, Richard A.; Hurley, Thomas D.; Vasiliou, Vasilis</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">520-539</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Aldehyde dehydrogenases (ALDHs) belong to a superfamily of enzymes that play a key role in the metab. of aldehydes of both endogenous and exogenous derivation.  The human ALDH superfamily comprises 19 isoenzymes that possess important physiol. and toxicol. functions.  The ALDH1A subfamily plays a pivotal role in embryogenesis and development by mediating retinoic acid signaling.  ALDH2, as a key enzyme that oxidizes acetaldehyde, is crucial for alc. metab.  ALDH1A1 and ALDH3A1 are lens and corneal crystallins, which are essential elements of the cellular defense mechanism against UV radiation-induced damage in ocular tissues.  Many ALDH isoenzymes are important in oxidizing reactive aldehydes derived from lipid peroxidn. and thereby help maintain cellular homeostasis.  Increased expression and activity of ALDH isoenzymes have been reported in various human cancers and are assocd. with cancer relapse.  As a direct consequence of their significant physiol. and toxicol. roles, inhibitors of the ALDH enzymes have been developed to treat human diseases.  This review summarizes known ALDH inhibitors, their mechanisms of action, isoenzyme selectivity, potency, and clin. uses.  The purpose of this review is to 1) establish the current status of pharmacol. inhibition of the ALDHs, 2) provide a rationale for the continued development of ALDH isoenzyme-selective inhibitors, and 3) identify the challenges and potential therapeutic rewards assocd. with the creation of such agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOl6kdU4MnaLVg90H21EOLACvtfcHk0liu580lDAxgOQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslaktw%253D%253D&md5=1de904ae3778b8715a6ec10e8d0242db</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1124%2Fpr.111.005538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.111.005538%26sid%3Dliteratum%253Aachs%26aulast%3DKoppaka%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DJuvonen%26aufirst%3DR.%2BO.%26aulast%3DPetersen%26aufirst%3DD.%26aulast%3DDeitrich%26aufirst%3DR.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DAldehyde%2520Dehydrogenase%2520Inhibitors%253A%2520a%2520Comprehensive%2520Review%2520of%2520the%2520Pharmacology%252C%2520Mechanism%2520of%2520Action%252C%2520Substrate%2520Specificity%252C%2520and%2520Clinical%2520Application%26jtitle%3DPharmacol.%2520Rev.%26date%3D2012%26volume%3D64%26issue%3D3%26spage%3D520%26epage%3D539%26doi%3D10.1124%2Fpr.111.005538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenases and Cancer Stem Cells</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>369</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">50</span>â <span class="NLM_lpage">57</span>, <span class="refDoi">Â DOI: 10.1016/j.canlet.2015.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.canlet.2015.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=26319899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVent77M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2015&pages=50-57&issue=1&author=X.+Xuauthor=S.+Chaiauthor=P.+Wangauthor=C.+Zhangauthor=Y.+Yangauthor=Y.+Yangauthor=K.+Wang&title=Aldehyde+Dehydrogenases+and+Cancer+Stem+Cells&doi=10.1016%2Fj.canlet.2015.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenases and cancer stem cells</span></div><div class="casAuthors">Xu, Xia; Chai, Shoujie; Wang, Pingli; Zhang, Chenchen; Yang, Yiming; Yang, Ying; Wang, Kai</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-57</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDHs), as essential regulators of aldehyde metab. in the human body, protect organisms from damage induced by active aldehydes.  Given their roles in different cancer types, ALDHs have been evaluated as potential prognostic markers of cancer.  ALDHs exhibit high activity in cancer stem cells (CSCs) and may serve as markers of CSCs.  Moreover, studies indicated that ALDHs and their regulated retinoic acid, reactive oxygen species and reactive aldehydes metab. were strongly related with various properties of CSCs.  Besides, recent research evidences have demonstrated the transcriptional and post-translational regulation of ALDH expression and activation in CSCs.  Thus, this review focuses on the function and regulation of ALDHs in CSCs, particularly ALDH1A1 and ALDH1A3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu8flgrvV35rVg90H21EOLACvtfcHk0liu580lDAxgOQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVent77M&md5=4b4e93f5f8faf061d6061841fcc187d3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2015.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2015.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DChai%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DK.%26atitle%3DAldehyde%2520Dehydrogenases%2520and%2520Cancer%2520Stem%2520Cells%26jtitle%3DCancer%2520Lett.%26date%3D2015%26volume%3D369%26issue%3D1%26spage%3D50%26epage%3D57%26doi%3D10.1016%2Fj.canlet.2015.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pors, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreb, J. S.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenases in Cancer: an Opportunity for Biomarker and Drug Development?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1953</span>â <span class="NLM_lpage">1963</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2014.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.drudis.2014.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25256776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt7vL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1953-1963&issue=12&author=K.+Porsauthor=J.+S.+Moreb&title=Aldehyde+Dehydrogenases+in+Cancer%3A+an+Opportunity+for+Biomarker+and+Drug+Development%3F&doi=10.1016%2Fj.drudis.2014.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenases in cancer: an opportunity for biomarker and drug development?</span></div><div class="casAuthors">Pors, Klaus; Moreb, Jan S.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1953-1963</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Aldehyde dehydrogenases (ALDHs) belong to a superfamily of 19 isoenzymes that are known to participate in many physiol. important biosynthetic processes including detoxification of specific endogenous and exogenous aldehyde substrates.  The high expression levels of an emerging no. of ALDHs in various cancer tissues suggest that these enzymes have pivotal roles in cancer cell survival and progression.  Mapping out the heterogeneity of tumors and their cancer stem cell (CSC) component will be key to successful design of strategies involving therapeutics that are targeted against specific ALDH isoenzymes.  This review summarises recent progress in ALDH-focused cancer research and discovery of small-mol.-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2K6XIiKHROLVg90H21EOLACvtfcHk0liu580lDAxgOQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt7vL&md5=3b8dac676074a792064a1c6c36e16dcb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DPors%26aufirst%3DK.%26aulast%3DMoreb%26aufirst%3DJ.%2BS.%26atitle%3DAldehyde%2520Dehydrogenases%2520in%2520Cancer%253A%2520an%2520Opportunity%2520for%2520Biomarker%2520and%2520Drug%2520Development%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26issue%3D12%26spage%3D1953%26epage%3D1963%26doi%3D10.1016%2Fj.drudis.2014.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreb, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostmark, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, W.</span></span> <span> </span><span class="NLM_article-title">ALDH Isozymes Downregulation Affects Cell Growth, Cell Motility and Gene Expression in Lung Cancer Cells</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">87</span>, <span class="refDoi">Â DOI: 10.1186/1476-4598-7-87</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1186%2F1476-4598-7-87" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=19025616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252Fit1ajtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=87&author=J.+S.+Morebauthor=H.+V.+Bakerauthor=L.+J.+Changauthor=M.+Amayaauthor=M.+C.+Lopezauthor=B.+Ostmarkauthor=W.+Chou&title=ALDH+Isozymes+Downregulation+Affects+Cell+Growth%2C+Cell+Motility+and+Gene+Expression+in+Lung+Cancer+Cells&doi=10.1186%2F1476-4598-7-87"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH isozymes downregulation affects cell growth, cell motility and gene expression in lung cancer cells</span></div><div class="casAuthors">Moreb Jan S; Baker Henry V; Chang Lung-Ji; Amaya Maria; Lopez M Cecilia; Ostmark Blanca; Chou Wayne</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">87</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Aldehyde dehydrogenase isozymes ALDH1A1 and ALDH3A1 are highly expressed in non small cell lung cancer.  Neither the mechanisms nor the biologic significance for such over expression have been studied.  METHODS:  We have employed oligonucleotide microarrays to analyze changes in gene profiles in A549 lung cancer cell line in which ALDH activity was reduced by up to 95% using lentiviral mediated expression of siRNA against both isozymes (Lenti 1+3).  Stringent analysis methods were used to identify gene expression patterns that are specific to the knock down of ALDH activity and significantly different in comparison to wild type A549 cells (WT) or cells similarly transduced with green fluorescent protein (GFP) siRNA.  RESULTS:  We confirmed significant and specific down regulation of ALDH1A1 and ALDH3A1 in Lenti 1+3 cells and in comparison to 12 other ALDH genes detected.  The results of the microarray analysis were validated by real time RT-PCR on RNA obtained from Lenti 1+3 or WT cells treated with ALDH activity inhibitors.  Detailed functional analysis was performed on 101 genes that were significantly different (P < 0.001) and their expression changed by > or = 2 folds in the Lenti 1+3 group versus the control groups.  There were 75 down regulated and 26 up regulated genes.  Protein binding, organ development, signal transduction, transcription, lipid metabolism, and cell migration and adhesion were among the most affected pathways.  CONCLUSION:  These molecular effects of the ALDH knock-down are associated with in vitro functional changes in the proliferation and motility of these cells and demonstrate the significance of ALDH enzymes in cell homeostasis with a potentially significant impact on the treatment of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSIE4CRrX3AXY2y4bB4mnvYfW6udTcc2eZ9FNqKAIt7qLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252Fit1ajtg%253D%253D&md5=141521d11ed323a2ef5def283846753a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-7-87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-7-87%26sid%3Dliteratum%253Aachs%26aulast%3DMoreb%26aufirst%3DJ.%2BS.%26aulast%3DBaker%26aufirst%3DH.%2BV.%26aulast%3DChang%26aufirst%3DL.%2BJ.%26aulast%3DAmaya%26aufirst%3DM.%26aulast%3DLopez%26aufirst%3DM.%2BC.%26aulast%3DOstmark%26aufirst%3DB.%26aulast%3DChou%26aufirst%3DW.%26atitle%3DALDH%2520Isozymes%2520Downregulation%2520Affects%2520Cell%2520Growth%252C%2520Cell%2520Motility%2520and%2520Gene%2520Expression%2520in%2520Lung%2520Cancer%2520Cells%26jtitle%3DMol.%2520Cancer%26date%3D2008%26volume%3D7%26spage%3D87%26doi%3D10.1186%2F1476-4598-7-87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dallaglio, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roop, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, M.</span></span> <span> </span><span class="NLM_article-title">ALDH1A Isozymes are Markers of Human Melanoma Stem Cells and Potential Therapeutic Targets</span>. <i>Stem Cells</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2100</span>â <span class="NLM_lpage">2113</span>, <span class="refDoi">Â DOI: 10.1002/stem.1193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fstem.1193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22887839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2ntrfK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=2100-2113&issue=10&author=Y.+Luoauthor=K.+Dallaglioauthor=Y.+Chenauthor=W.+A.+Robinsonauthor=S.+E.+Robinsonauthor=M.+D.+McCarterauthor=J.+Wangauthor=R.+Gonzalezauthor=D.+C.+Thompsonauthor=D.+A.+Norrisauthor=D.+R.+Roopauthor=V.+Vasiliouauthor=M.+Fujita&title=ALDH1A+Isozymes+are+Markers+of+Human+Melanoma+Stem+Cells+and+Potential+Therapeutic+Targets&doi=10.1002%2Fstem.1193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH1A isozymes are markers of human melanoma stem cells and potential therapeutic targets</span></div><div class="casAuthors">Luo, Yuchun; Dallaglio, Katiuscia; Chen, Ying; Robinson, William A.; Robinson, Steven E.; McCarter, Martin D.; Wang, Jianbin; Gonzalez, Rene; Thompson, David C.; Norris, David A.; Roop, Dennis R.; Vasiliou, Vasilis; Fujita, Mayumi</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cells (Durham, NC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2100-2113</span>CODEN:
                <span class="NLM_cas:coden">STCEEJ</span>;
        ISSN:<span class="NLM_cas:issn">1066-5099</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">Although the concept of cancer stem cells (CSCs) is well-accepted for many tumors, the existence of such cells in human melanoma has been the subject of debate.  In this study, we demonstrate the existence of human melanoma cells that fulfill the criteria for CSCs (self-renewal and differentiation) by serially xenotransplanting cells into non-obese diabetic/severe combined immunodeficiency (NOD/SCID) mice.  These cells possess high aldehyde dehydrogenase (ALDH) activity with ALDH1A1 and ALDH1A3 being the predominant ALDH isoenzymes.  ALDH-pos. melanoma cells are more tumorigenic than ALDH-neg. cells in both NOD/SCID mice and NSG mice.  Biol. analyses of the ALDH-pos. melanoma cells reveal the ALDH isoenzymes to be key mols. regulating the function of these cells.  Silencing ALDH1A by siRNA or shRNA leads to cell cycle arrest, apoptosis, decreased cell viability in vitro, and reduced tumorigenesis in vivo.  ALDH-pos. melanoma cells are more resistant to chemotherapeutic agents and silencing ALDH1A by siRNA sensitizes melanoma cells to drug-induced cell death.  Furthermore, we, for the first time, examd. the mol. signatures of ALDH-pos. CSCs from patient-derived tumor specimens.  The signatures of melanoma CSCs include retinoic acid (RA)-driven target genes with RA response elements and genes assocd. with stem cell function.  These findings implicate that ALDH isoenzymes are not only biomarkers of CSCs but also attractive therapeutic targets for human melanoma.  Further investigation of these isoenzymes and genes will enhance our understanding of the mol. mechanisms governing CSCs and reveal new mol. targets for therapeutic intervention of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryIxi-Gq5gk7Vg90H21EOLACvtfcHk0lgVK0B7tQpy_Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2ntrfK&md5=89c568bb117628a4034c949835d43a92</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fstem.1193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1193%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DDallaglio%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DRobinson%26aufirst%3DW.%2BA.%26aulast%3DRobinson%26aufirst%3DS.%2BE.%26aulast%3DMcCarter%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGonzalez%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26aulast%3DNorris%26aufirst%3DD.%2BA.%26aulast%3DRoop%26aufirst%3DD.%2BR.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DFujita%26aufirst%3DM.%26atitle%3DALDH1A%2520Isozymes%2520are%2520Markers%2520of%2520Human%2520Melanoma%2520Stem%2520Cells%2520and%2520Potential%2520Therapeutic%2520Targets%26jtitle%3DStem%2520Cells%26date%3D2012%26volume%3D30%26issue%3D10%26spage%3D2100%26epage%3D2113%26doi%3D10.1002%2Fstem.1193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, P.</span></span> <span> </span><span class="NLM_article-title">ALDH Maintains the Stemness of Lung Adenoma Stem Cells by Suppressing the Notch/CDK2/CCNE Pathway</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e92669</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0092669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0092669" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e92669&issue=3&author=Z.+Liauthor=Y.+Xiangauthor=L.+Xiangauthor=Y.+Xiaoauthor=F.+Liauthor=P.+Hao&title=ALDH+Maintains+the+Stemness+of+Lung+Adenoma+Stem+Cells+by+Suppressing+the+Notch%2FCDK2%2FCCNE+Pathway&doi=10.1371%2Fjournal.pone.0092669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0092669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0092669%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHao%26aufirst%3DP.%26atitle%3DALDH%2520Maintains%2520the%2520Stemness%2520of%2520Lung%2520Adenoma%2520Stem%2520Cells%2520by%2520Suppressing%2520the%2520Notch%252FCDK2%252FCCNE%2520Pathway%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D3%26spage%3De92669%26doi%3D10.1371%2Fjournal.pone.0092669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duong, H. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, I.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Dehydrogenase 1A1 Confers Intrinsic and Acquired Resistance to Gemcitabine in Human Pancreatic Adenocarcinoma MIA PaCa-2 Cells</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">855</span>â <span class="NLM_lpage">861</span>, <span class="refDoi">Â DOI: 10.3892/ijo.2012.1516</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.3892%2Fijo.2012.1516" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22710732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWhurnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=855-861&issue=3&author=H.+Q.+Duongauthor=J.+S.+Hwangauthor=H.+J.+Kimauthor=H.+J.+Kangauthor=Y.+S.+Seongauthor=I.+Bae&title=Aldehyde+Dehydrogenase+1A1+Confers+Intrinsic+and+Acquired+Resistance+to+Gemcitabine+in+Human+Pancreatic+Adenocarcinoma+MIA+PaCa-2+Cells&doi=10.3892%2Fijo.2012.1516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenase 1A1 confers intrinsic and acquired resistance to gemcitabine in human pancreatic adenocarcinoma MIA PaCa-2 cells</span></div><div class="casAuthors">Duong, Hong-Quan; Hwang, Jae Seok; Kim, Hee Jeong; Kang, Hyo Jin; Seong, Yeon-Sun; Bae, Insoo</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">Gemcitabine (GEM) is the front-line std. chemotherapy used for the treatment of pancreatic cancer; however, chemoresistance to GEM remains the major obstacle to the successful control of this disease.  Both the expression levels and activity of aldehyde dehydrogenase 1A1 (ALDH1A1) are important features of tumor-initiating and/or cancer stem cell properties in multiple types of human cancer.  As one of the intrinsic properties of cancer stem cells is drug resistance, in this study, we examd. the correlation between the level and activity of endogenous ALDHlAl and GEM resistance in the MIA PaCa-2 cell line that contains high expression levels and activity 1 1.DHl Al.  We used small interfering RNAs (siRNAs) to deple1 ALDHl A1 and investigate its potential role in conferring GEM resistance.  The ALDH1A1 knockdown markedly reduc1 ALDHl A1 expression and activity and inhibited cell proliferation.  Moreover, the combination of ALDHlAl-siRNA and GEM significantly decreased cell viability, increased apoptotic cell death and increased the accumulation of cells at the S-phase compared to the controls.  Our data also demonstrated that ALDHlA1 expression and activity were significantly higher in the GEM-resistant MIA PaCa-2 cell line (MIA PaCa-2/GR), compared to the parental MIA PaCa-2 cell line (MIA PaCa-2/P).  In the MIA PaCa-2/GR cells, the combination of ALDHLA1-siRNA and GEM also showed a significant decrease in cell viability and an increase in apoptotic cell death, emphasizing the importance ALDHlA1 in both intrinsic and acquired GEM resistance.  This potentially powerful combination treatment of ALDHlAl-siRNA and GEM warrants further investigation as an effective therapeutic regimen to overcome the resistance of pancreatic cancer to GEM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYdDPtBAWCfbVg90H21EOLACvtfcHk0lgVK0B7tQpy_Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWhurnE&md5=ca0d05f83a1e06ae3b02e16fa4e9e517</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3892%2Fijo.2012.1516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2012.1516%26sid%3Dliteratum%253Aachs%26aulast%3DDuong%26aufirst%3DH.%2BQ.%26aulast%3DHwang%26aufirst%3DJ.%2BS.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DKang%26aufirst%3DH.%2BJ.%26aulast%3DSeong%26aufirst%3DY.%2BS.%26aulast%3DBae%26aufirst%3DI.%26atitle%3DAldehyde%2520Dehydrogenase%25201A1%2520Confers%2520Intrinsic%2520and%2520Acquired%2520Resistance%2520to%2520Gemcitabine%2520in%2520Human%2520Pancreatic%2520Adenocarcinoma%2520MIA%2520PaCa-2%2520Cells%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2012%26volume%3D41%26issue%3D3%26spage%3D855%26epage%3D861%26doi%3D10.3892%2Fijo.2012.1516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Condello, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagdas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matei, D.</span></span> <span> </span><span class="NLM_article-title">Beta-Catenin-Regulated ALDH1A1 is a Target in Ovarian Cancer Spheroids</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">2297</span>â <span class="NLM_lpage">2308</span>, <span class="refDoi">Â DOI: 10.1038/onc.2014.178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fonc.2014.178" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=2297-2308&issue=18&author=S.+Condelloauthor=C.+A.+Morganauthor=S.+Nagdasauthor=L.+Caoauthor=J.+Turekauthor=T.+D.+Hurleyauthor=D.+Matei&title=Beta-Catenin-Regulated+ALDH1A1+is+a+Target+in+Ovarian+Cancer+Spheroids&doi=10.1038%2Fonc.2014.178"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.178%26sid%3Dliteratum%253Aachs%26aulast%3DCondello%26aufirst%3DS.%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DNagdas%26aufirst%3DS.%26aulast%3DCao%26aufirst%3DL.%26aulast%3DTurek%26aufirst%3DJ.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DMatei%26aufirst%3DD.%26atitle%3DBeta-Catenin-Regulated%2520ALDH1A1%2520is%2520a%2520Target%2520in%2520Ovarian%2520Cancer%2520Spheroids%26jtitle%3DOncogene%26date%3D2015%26volume%3D34%26issue%3D18%26spage%3D2297%26epage%3D2308%26doi%3D10.1038%2Fonc.2014.178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenkov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickramasinghe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palozola, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showe, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaret, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speicher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ordog, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitler, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span> <span> </span><span class="NLM_article-title">BET Inhibitors Suppress ALDH Activity by Targeting ALDH1A1 Super-Enhancer in Ovarian Cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">6320</span>â <span class="NLM_lpage">6330</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-16-0854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F0008-5472.CAN-16-0854" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=6320-6330&issue=21&author=Y.+Yokoyamaauthor=H.+Zhuauthor=J.+H.+Leeauthor=A.+V.+Kossenkovauthor=S.+Y.+Wuauthor=J.+M.+Wickramasingheauthor=X.+Yinauthor=K.+C.+Palozolaauthor=A.+Gardiniauthor=L.+C.+Showeauthor=K.+S.+Zaretauthor=Q.+Liuauthor=D.+Speicherauthor=J.+R.+Conejo-Garciaauthor=J.+E.+Bradnerauthor=Z.+Zhangauthor=A.+K.+Soodauthor=T.+Ordogauthor=B.+G.+Bitlerauthor=R.+Zhang&title=BET+Inhibitors+Suppress+ALDH+Activity+by+Targeting+ALDH1A1+Super-Enhancer+in+Ovarian+Cancer&doi=10.1158%2F0008-5472.CAN-16-0854"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-0854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-0854%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DKossenkov%26aufirst%3DA.%2BV.%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DWickramasinghe%26aufirst%3DJ.%2BM.%26aulast%3DYin%26aufirst%3DX.%26aulast%3DPalozola%26aufirst%3DK.%2BC.%26aulast%3DGardini%26aufirst%3DA.%26aulast%3DShowe%26aufirst%3DL.%2BC.%26aulast%3DZaret%26aufirst%3DK.%2BS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSpeicher%26aufirst%3DD.%26aulast%3DConejo-Garcia%26aufirst%3DJ.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DOrdog%26aufirst%3DT.%26aulast%3DBitler%26aufirst%3DB.%2BG.%26aulast%3DZhang%26aufirst%3DR.%26atitle%3DBET%2520Inhibitors%2520Suppress%2520ALDH%2520Activity%2520by%2520Targeting%2520ALDH1A1%2520Super-Enhancer%2520in%2520Ovarian%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26issue%3D21%26spage%3D6320%26epage%3D6330%26doi%3D10.1158%2F0008-5472.CAN-16-0854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. N.</span></span> <span> </span><span class="NLM_article-title">ALDH1A3, the Major Aldehyde Dehydrogenase Isoform in Human Cholangiocarcinoma Cells, Affects Prognosis and Gemcitabine Resistance in Cholangiocarcinoma Patients</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">4225</span>â <span class="NLM_lpage">4235</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-15-1800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F1078-0432.CCR-15-1800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=27076629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKrsbbO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=4225-4235&issue=16&author=M.+H.+Chenauthor=J.+J.+Wengauthor=C.+T.+Chengauthor=R.+C.+Wuauthor=S.+C.+Huangauthor=C.+E.+Wuauthor=Y.+H.+Chungauthor=C.+Y.+Liuauthor=M.+H.+Changauthor=M.+H.+Chenauthor=K.+C.+Chiangauthor=T.+S.+Yehauthor=Y.+Suauthor=C.+N.+Yeh&title=ALDH1A3%2C+the+Major+Aldehyde+Dehydrogenase+Isoform+in+Human+Cholangiocarcinoma+Cells%2C+Affects+Prognosis+and+Gemcitabine+Resistance+in+Cholangiocarcinoma+Patients&doi=10.1158%2F1078-0432.CCR-15-1800"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH1A3, the Major Aldehyde Dehydrogenase Isoform in Human Cholangiocarcinoma Cells, Affects Prognosis and Gemcitabine Resistance in Cholangiocarcinoma Patients</span></div><div class="casAuthors">Chen, Ming-Huang; Weng, Jing-Jie; Cheng, Chi-Tung; Wu, Ren-Chin; Huang, Shih-Chiang; Wu, Chiao-En; Chung, Yi-Hsiu; Liu, Chun-Yu; Chang, Mu-Hsin; Chen, Ming-Han; Chiang, Kun-Chun; Yeh, Ta-Sen; Su, Yeu; Yeh, Chun-Nan</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4225-4235</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Intrahepatic cholangiocarcinoma is a fatal primary liver cancer resulting from diagnosis at an advanced stage.  Understanding the mechanisms of drug resistance and metastasis of cholangiocarcinoma may improve the disease prognosis.  Enhanced aldehyde dehydrogenase (ALDH) activity is suggested to be assocd. with increased drug resistance and the metastasis.  This study aims to investigate the roles of the ALDH isoforms in cholangiocarcinoma.  Exptl. Design: Aldefluor assays, RT-PCR, and Western blot anal. were used to identify the major ALDH isoforms contributing to Aldefluor activity in human cholangiocarcinoma cell lines.  We manipulated isoform expression in HuCCT1 cells to elucidate the role of ALDH1A3 in the malignant progression of these cells.  Finally, we used immunohistochem. staining to evaluate the clin. significance of ALDH1A3 in 77 hepatectomized cholangiocarcinoma patients and an addnl. 31 patients with advanced cholangiocarcinoma who received gemcitabine-based therapy.  Results: ALDHhigh cholangiocarcinoma cells not only migrated faster but were more resistant to gemcitabine.  Among the 19 ALDH isoforms studied, ALDH1A3 was found to be the main contributor to Aldefluor activity.  In addn., we also found that knockdown of ALDH1A3 expression in HuCCT1 cells markedly reduced not only their sensitivity to gemcitabine, which might be attributed to a decreased expression of ribonucleotide reductase M1, but also their migration.  Most importantly, this enzyme was also identified as an independent poor prognostic factor for patients with intrahepatic cholangiocarcinoma, as well as a prognostic biomarker of gemcitabine-treated patients.  Conclusions: ALDH1A3 plays an important role in enhancing malignant behavior of cholangiocarcinoma and serves as a new therapeutic target.  Clin Cancer Res; 22(16); 4225-35. Â©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSsIYLXC1BjLVg90H21EOLACvtfcHk0lgLA-C7Na1SUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKrsbbO&md5=94376c2d13426323f216f9a7c2c4a42c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-15-1800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-15-1800%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DM.%2BH.%26aulast%3DWeng%26aufirst%3DJ.%2BJ.%26aulast%3DCheng%26aufirst%3DC.%2BT.%26aulast%3DWu%26aufirst%3DR.%2BC.%26aulast%3DHuang%26aufirst%3DS.%2BC.%26aulast%3DWu%26aufirst%3DC.%2BE.%26aulast%3DChung%26aufirst%3DY.%2BH.%26aulast%3DLiu%26aufirst%3DC.%2BY.%26aulast%3DChang%26aufirst%3DM.%2BH.%26aulast%3DChen%26aufirst%3DM.%2BH.%26aulast%3DChiang%26aufirst%3DK.%2BC.%26aulast%3DYeh%26aufirst%3DT.%2BS.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DYeh%26aufirst%3DC.%2BN.%26atitle%3DALDH1A3%252C%2520the%2520Major%2520Aldehyde%2520Dehydrogenase%2520Isoform%2520in%2520Human%2520Cholangiocarcinoma%2520Cells%252C%2520Affects%2520Prognosis%2520and%2520Gemcitabine%2520Resistance%2520in%2520Cholangiocarcinoma%2520Patients%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2016%26volume%3D22%26issue%3D16%26spage%3D4225%26epage%3D4235%26doi%3D10.1158%2F1078-0432.CCR-15-1800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cortes-Dericks, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froment, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoubi, G.</span></span> <span> </span><span class="NLM_article-title">Cisplatin-Resistant Cells in Malignant Pleural Mesothelioma Cell Lines Show ALDH(high)CD44(+) Phenotype and Sphere-Forming Capacity</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">304</span>, <span class="refDoi">Â DOI: 10.1186/1471-2407-14-304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1186%2F1471-2407-14-304" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=304&author=L.+Cortes-Dericksauthor=L.+Fromentauthor=R.+Boeschauthor=R.+A.+Schmidauthor=G.+Karoubi&title=Cisplatin-Resistant+Cells+in+Malignant+Pleural+Mesothelioma+Cell+Lines+Show+ALDH%28high%29CD44%28%2B%29+Phenotype+and+Sphere-Forming+Capacity&doi=10.1186%2F1471-2407-14-304"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-304%26sid%3Dliteratum%253Aachs%26aulast%3DCortes-Dericks%26aufirst%3DL.%26aulast%3DFroment%26aufirst%3DL.%26aulast%3DBoesch%26aufirst%3DR.%26aulast%3DSchmid%26aufirst%3DR.%2BA.%26aulast%3DKaroubi%26aufirst%3DG.%26atitle%3DCisplatin-Resistant%2520Cells%2520in%2520Malignant%2520Pleural%2520Mesothelioma%2520Cell%2520Lines%2520Show%2520ALDH%2528high%2529CD44%2528%252B%2529%2520Phenotype%2520and%2520Sphere-Forming%2520Capacity%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D304%26doi%3D10.1186%2F1471-2407-14-304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saw, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, W.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S.-W.</span></span> <span> </span><span class="NLM_article-title">Characterization of Aldehyde Dehydrogenase Isozymes in Ovarian Cancer Tissues and Sphere Cultures</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">329</span>, <span class="refDoi">Â DOI: 10.1186/1471-2407-12-329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1186%2F1471-2407-12-329" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=329&issue=1&author=Y.-T.+Sawauthor=J.+Yangauthor=S.-K.+Ngauthor=S.+Liuauthor=S.+Singhauthor=M.+Singhauthor=W.+R.+Welchauthor=H.+Tsudaauthor=W.-P.+Fongauthor=D.+Thompsonauthor=V.+Vasiliouauthor=R.+S.+Berkowitzauthor=S.-W.+Ng&title=Characterization+of+Aldehyde+Dehydrogenase+Isozymes+in+Ovarian+Cancer+Tissues+and+Sphere+Cultures&doi=10.1186%2F1471-2407-12-329"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-12-329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-12-329%26sid%3Dliteratum%253Aachs%26aulast%3DSaw%26aufirst%3DY.-T.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DNg%26aufirst%3DS.-K.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DWelch%26aufirst%3DW.%2BR.%26aulast%3DTsuda%26aufirst%3DH.%26aulast%3DFong%26aufirst%3DW.-P.%26aulast%3DThompson%26aufirst%3DD.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DBerkowitz%26aufirst%3DR.%2BS.%26aulast%3DNg%26aufirst%3DS.-W.%26atitle%3DCharacterization%2520of%2520Aldehyde%2520Dehydrogenase%2520Isozymes%2520in%2520Ovarian%2520Cancer%2520Tissues%2520and%2520Sphere%2520Cultures%26jtitle%3DBMC%2520Cancer%26date%3D2012%26volume%3D12%26issue%3D1%26spage%3D329%26doi%3D10.1186%2F1471-2407-12-329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowtell, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BÃ¶hm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuria, P.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beral, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berek, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birrer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bookman, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiappinelli, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coukos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drapkin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotopoulou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabra, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gourley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heong, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntsman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwanicki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlan, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengyel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeish, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narod, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nephew, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pharoah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, D. J.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stronach, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balkwill, F. R.</span></span> <span> </span><span class="NLM_article-title">Rethinking Ovarian Cancer II: Reducing Mortality From High-grade Serous Ovarian Cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">668</span>, <span class="refDoi">Â DOI: 10.1038/nrc4019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fnrc4019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=26493647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWhsLjF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=668&author=D.+D.+Bowtellauthor=S.+B%C3%B6hmauthor=A.+A.+Ahmedauthor=P.-J.+Aspuriaauthor=R.+C.+Bastauthor=V.+Beralauthor=J.+S.+Berekauthor=M.+J.+Birrerauthor=S.+Blagdenauthor=M.+A.+Bookmanauthor=J.+D.+Brentonauthor=K.+B.+Chiappinelliauthor=F.+C.+Martinsauthor=G.+Coukosauthor=R.+Drapkinauthor=R.+Edmondsonauthor=C.+Fotopoulouauthor=H.+Gabraauthor=J.+Galonauthor=C.+Gourleyauthor=V.+Heongauthor=D.+G.+Huntsmanauthor=M.+Iwanickiauthor=B.+Y.+Karlanauthor=A.+Kayeauthor=E.+Lengyelauthor=D.+A.+Levineauthor=K.+H.+Luauthor=I.+A.+McNeishauthor=U.+Menonauthor=S.+A.+Narodauthor=B.+H.+Nelsonauthor=K.+P.+Nephewauthor=P.+Pharoahauthor=D.+J.+Powellauthor=P.+Ramosauthor=I.+L.+Romeroauthor=C.+L.+Scottauthor=A.+K.+Soodauthor=E.+A.+Stronachauthor=F.+R.+Balkwill&title=Rethinking+Ovarian+Cancer+II%3A+Reducing+Mortality+From+High-grade+Serous+Ovarian+Cancer&doi=10.1038%2Fnrc4019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking ovarian cancer II: reducing mortality from high-grade serous ovarian cancer</span></div><div class="casAuthors">Bowtell, David D.; Bohm, Steffen; Ahmed, Ahmed A.; Aspuria, Paul-Joseph; Bast, Robert C., Jr.; Beral, Valerie; Berek, Jonathan S.; Birrer, Michael J.; Blagden, Sarah; Bookman, Michael A.; Brenton, James D.; Chiappinelli, Katherine B.; Martins, Filipe Correia; Coukos, George; Drapkin, Ronny; Edmondson, Richard; Fotopoulou, Christina; Gabra, Hani; Galon, Jerome; Gourley, Charlie; Heong, Valerie; Huntsman, David G.; Iwanicki, Marcin; Karlan, Beth Y.; Kaye, Allyson; Lengyel, Ernst; Levine, Douglas A.; Lu, Karen H.; McNeish, Iain A.; Menon, Usha; Narod, Steven A.; Nelson, Brad H.; Nephew, Kenneth P.; Pharoah, Paul; Powell, Daniel J. Jr; Ramos, Pilar; Romero, Iris L.; Scott, Clare L.; Sood, Anil K.; Stronach, Euan A.; Balkwill, Frances R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">668-679</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">High-grade serous ovarian cancer (HGSOC) accounts for 70-80% of ovarian cancer deaths, and overall survival has not changed significantly for several decades.  In this Opinion article, we outline a set of research priorities that we believe will reduce incidence and improve outcomes for women with this disease.  This 'roadmap' for HGSOC was detd. after extensive discussions at an Ovarian Cancer Action meeting in Jan. 2015.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4-9JF1j3Wv7Vg90H21EOLACvtfcHk0lgnevKYniRMnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWhsLjF&md5=a23413ed49019485b07b170c3eae7fcb</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrc4019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc4019%26sid%3Dliteratum%253Aachs%26aulast%3DBowtell%26aufirst%3DD.%2BD.%26aulast%3DB%25C3%25B6hm%26aufirst%3DS.%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26aulast%3DAspuria%26aufirst%3DP.-J.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DBeral%26aufirst%3DV.%26aulast%3DBerek%26aufirst%3DJ.%2BS.%26aulast%3DBirrer%26aufirst%3DM.%2BJ.%26aulast%3DBlagden%26aufirst%3DS.%26aulast%3DBookman%26aufirst%3DM.%2BA.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DChiappinelli%26aufirst%3DK.%2BB.%26aulast%3DMartins%26aufirst%3DF.%2BC.%26aulast%3DCoukos%26aufirst%3DG.%26aulast%3DDrapkin%26aufirst%3DR.%26aulast%3DEdmondson%26aufirst%3DR.%26aulast%3DFotopoulou%26aufirst%3DC.%26aulast%3DGabra%26aufirst%3DH.%26aulast%3DGalon%26aufirst%3DJ.%26aulast%3DGourley%26aufirst%3DC.%26aulast%3DHeong%26aufirst%3DV.%26aulast%3DHuntsman%26aufirst%3DD.%2BG.%26aulast%3DIwanicki%26aufirst%3DM.%26aulast%3DKarlan%26aufirst%3DB.%2BY.%26aulast%3DKaye%26aufirst%3DA.%26aulast%3DLengyel%26aufirst%3DE.%26aulast%3DLevine%26aufirst%3DD.%2BA.%26aulast%3DLu%26aufirst%3DK.%2BH.%26aulast%3DMcNeish%26aufirst%3DI.%2BA.%26aulast%3DMenon%26aufirst%3DU.%26aulast%3DNarod%26aufirst%3DS.%2BA.%26aulast%3DNelson%26aufirst%3DB.%2BH.%26aulast%3DNephew%26aufirst%3DK.%2BP.%26aulast%3DPharoah%26aufirst%3DP.%26aulast%3DPowell%26aufirst%3DD.%2BJ.%26aulast%3DRamos%26aufirst%3DP.%26aulast%3DRomero%26aufirst%3DI.%2BL.%26aulast%3DScott%26aufirst%3DC.%2BL.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DStronach%26aufirst%3DE.%2BA.%26aulast%3DBalkwill%26aufirst%3DF.%2BR.%26atitle%3DRethinking%2520Ovarian%2520Cancer%2520II%253A%2520Reducing%2520Mortality%2520From%2520High-grade%2520Serous%2520Ovarian%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D668%26doi%3D10.1038%2Fnrc4019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Titus, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danchik, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. J.</span></span> <span> </span><span class="NLM_article-title">Correction: A High-Content Assay Enables the Automated Screening and Identification of Small Molecules with Specific ALDH1A1-Inhibitory Activity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">e0197292</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0197292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0197292" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=e0197292&issue=5&author=A.+Yasgarauthor=S.+A.+Titusauthor=Y.+Wangauthor=C.+Danchikauthor=S.-M.+Yangauthor=V.+Vasiliouauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=N.+J.+Martinez&title=Correction%3A+A+High-Content+Assay+Enables+the+Automated+Screening+and+Identification+of+Small+Molecules+with+Specific+ALDH1A1-Inhibitory+Activity&doi=10.1371%2Fjournal.pone.0197292"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0197292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0197292%26sid%3Dliteratum%253Aachs%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DTitus%26aufirst%3DS.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDanchik%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DS.-M.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DMartinez%26aufirst%3DN.%2BJ.%26atitle%3DCorrection%253A%2520A%2520High-Content%2520Assay%2520Enables%2520the%2520Automated%2520Screening%2520and%2520Identification%2520of%2520Small%2520Molecules%2520with%2520Specific%2520ALDH1A1-Inhibitory%2520Activity%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3De0197292%26doi%3D10.1371%2Fjournal.pone.0197292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchman, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dria, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">N,N-Diethylaminobenzaldehyde (DEAB) as a Substrate and Mechanism-Based Inhibitor for Human ALDH Isoenzymes</span>. <i>Chem.-Biol. Interact.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>234</i></span> (<span class="NLM_issue">0</span>),  <span class="NLM_fpage">18</span>â <span class="NLM_lpage">28</span>, <span class="refDoi">Â DOI: 10.1016/j.cbi.2014.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.cbi.2014.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25512087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkslWmsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2015&pages=18-28&issue=0&author=C.+A.+Morganauthor=B.+Parajuliauthor=C.+D.+Buchmanauthor=K.+Driaauthor=T.+D.+Hurley&title=N%2CN-Diethylaminobenzaldehyde+%28DEAB%29+as+a+Substrate+and+Mechanism-Based+Inhibitor+for+Human+ALDH+Isoenzymes&doi=10.1016%2Fj.cbi.2014.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">N,N-diethylaminobenzaldehyde (DEAB) as a substrate and mechanism-based inhibitor for human ALDH isoenzymes</span></div><div class="casAuthors">Morgan, Cynthia A.; Parajuli, Bibek; Buchman, Cameron D.; Dria, Karl; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18-28</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">4-(N,N-Diethylamino)benzaldehyde(DEAB) is a commonly used "selective" inhibitor of human NAD(P)-dependent aldehyde dehydrogenase (ALDH) isoenzymes in cancer stem cell biol. due to its inclusion as a neg. control compd. in the widely utilized Aldefluor assay.  Recent evidence has accumulated that DEAB is not a selective inhibitory agent when assayed in vitro vs. ALDH1, ALDH2, and ALDH3 family members.  Here, the authors sought to det. the selectivity of DEAB toward ALDH1A1, ALDH1A2, ALDH1A3, ALDH1B1, ALDH1L1, ALDH2, ALDH3A1, ALDH4A1, and ALDH5A1 isoenzymes and det. the mechanism by which DEAB exerts its inhibitory action.  The authors found that DEAB was an excellent substrate for ALDH3A1, exhibiting a Vmax/Km that exceeded that of its commonly used substrate, benzaldehyde.  DEAB was also a substrate for ALDH1A1, albeit an exceptionally slow one (turnover rate of â¼0.03 min-1).  In contrast, little if any turnover of DEAB was obsd. when incubated with ALDH1A2, ALDH1A3, ALDH1B1, ALDH2, or ALDH5A1.  DEAB was neither a substrate nor an inhibitor for ALDH1L1 or ALDH4A1.  Anal. by enzyme kinetics and QTOF mass spectrometry demonstrated that DEAB was an irreversible inhibitor of ALDH1A2 and ALDH2 with apparent bimol. rate consts. of 2900 and 86,000 M-1 s-1, resp.  The mechanism of inactivation was consistent with the formation of a quinoid-like resonance state following hydride transfer that was stabilized by local structural features that exist in several of the ALDH isoenzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrREVT_lTwCJLVg90H21EOLACvtfcHk0lgnevKYniRMnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkslWmsA%253D%253D&md5=82753ba4991367a9a5da5e6d64770ced</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DBuchman%26aufirst%3DC.%2BD.%26aulast%3DDria%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DN%252CN-Diethylaminobenzaldehyde%2520%2528DEAB%2529%2520as%2520a%2520Substrate%2520and%2520Mechanism-Based%2520Inhibitor%2520for%2520Human%2520ALDH%2520Isoenzymes%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D2015%26volume%3D234%26issue%3D0%26spage%3D18%26epage%3D28%26doi%3D10.1016%2Fj.cbi.2014.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Croker, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allan, A. L.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Aldehyde Dehydrogenase (ALDH) Activity Reduces Chemotherapy and Radiation Resistance of Stem-Like ALDHhiCD44+ Human Breast Cancer Cells</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>133</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">75</span>â <span class="NLM_lpage">87</span>, <span class="refDoi">Â DOI: 10.1007/s10549-011-1692-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1007%2Fs10549-011-1692-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21818590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtlOlurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2012&pages=75-87&issue=1&author=A.+K.+Crokerauthor=A.+L.+Allan&title=Inhibition+of+Aldehyde+Dehydrogenase+%28ALDH%29+Activity+Reduces+Chemotherapy+and+Radiation+Resistance+of+Stem-Like+ALDHhiCD44%2B+Human+Breast+Cancer+Cells&doi=10.1007%2Fs10549-011-1692-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of aldehyde dehydrogenase (ALDH) activity reduces chemotherapy and radiation resistance of stem-like ALDHhiCD44+ human breast cancer cells</span></div><div class="casAuthors">Croker, Alysha K.; Allan, Alison L.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-87</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The majority of breast cancer deaths are because of ineffective treatment of metastatic disease.  We previously identified a subpopulation of cells in human breast cancer cell lines that demonstrate high activity of aldehyde dehydrogenase (ALDH) and high expression of CD44.  These ALDHhiCD44+ cells displayed enhanced metastatic behavior in vitro and in vivo relative to ALDHlowCD44- cells.  The goal of this study was to test the hypothesis that ALDHhiCD44+ breast cancer cells are more resistant to std. cancer therapy, and that inhibiting ALDH activity through all-trans retinoic acid (ATRA) or the specific ALDH inhibitor diethylaminobenzaldehyde (DEAB) sensitizes these cells to treatment.  ALDHhiCD44+ and ALDHlowCD44- populations were isolated from MDA-MB-231 and MDA-MB-468 cells lines and exposed to chemotherapy (doxorubicin/paclitaxel) or radiotherapy Â± ATRA or DEAB.  Cell populations were assessed for differences in survival, colony formation, and protein expression related to therapy resistance and differentiation.  Significantly more ALDHhiCD44+ cells survived chemotherapy/radiotherapy relative to ALDHlowCD44- cells (P < 0.001).  Glutathione-S-transferase pi, p-glycoprotein, and/or CHK1 were overexpressed in ALDHhiCD44+ populations compared with ALDHlowCD44- populations (P < 0.05).  Pre-treatment of cell populations with DEAB or ATRA had no effect on ALDHlowCD44- cells, but resulted in significant initial sensitization of ALDHhiCD44+ cells to chemotherapy/radiotherapy.  However, only DEAB had a long-term effect, resulting in reduced colony formation (P < 0.01).  ATRA also significantly increased expression of CK8/18/19 in MDA-MB-468 ALDHhiCD44+ cells compared with control (P < 0.05).  Our novel findings indicate that ALDHhiCD44+ breast cancer cells contribute to both chemotherapy and radiation resistance and suggest a much broader role for ALDH in treatment response than previously reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqugwcSmbKKLbVg90H21EOLACvtfcHk0lgnevKYniRMnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtlOlurc%253D&md5=64822493337b70996c7515e3368c2b50</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs10549-011-1692-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-011-1692-y%26sid%3Dliteratum%253Aachs%26aulast%3DCroker%26aufirst%3DA.%2BK.%26aulast%3DAllan%26aufirst%3DA.%2BL.%26atitle%3DInhibition%2520of%2520Aldehyde%2520Dehydrogenase%2520%2528ALDH%2529%2520Activity%2520Reduces%2520Chemotherapy%2520and%2520Radiation%2520Resistance%2520of%2520Stem-Like%2520ALDHhiCD44%252B%2520Human%2520Breast%2520Cancer%2520Cells%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2012%26volume%3D133%26issue%3D1%26spage%3D75%26epage%3D87%26doi%3D10.1007%2Fs10549-011-1692-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goktug, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Channathodiyil, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannappan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hugnot, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichet, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armesilla, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darling, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span> <span> </span><span class="NLM_article-title">Cytotoxic Effect of Disulfiram/Copper on Human Glioblastoma Cell Lines and ALDH-Positive Cancer-Stem-Like Cells</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>107</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1488</span>â <span class="NLM_lpage">1497</span>, <span class="refDoi">Â DOI: 10.1038/bjc.2012.442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fbjc.2012.442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=23033007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOntLjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2012&pages=1488-1497&issue=9&author=P.+Liuauthor=S.+Brownauthor=T.+Goktugauthor=P.+Channathodiyilauthor=V.+Kannappanauthor=J.+P.+Hugnotauthor=P.+O.+Guichetauthor=X.+Bianauthor=A.+L.+Armesillaauthor=J.+L.+Darlingauthor=W.+Wang&title=Cytotoxic+Effect+of+Disulfiram%2FCopper+on+Human+Glioblastoma+Cell+Lines+and+ALDH-Positive+Cancer-Stem-Like+Cells&doi=10.1038%2Fbjc.2012.442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Cytotoxic effect of disulfiram/copper on human glioblastoma cell lines and ALDH-positive cancer-stem-like cells</span></div><div class="casAuthors">Liu, P.; Brown, S.; Goktug, T.; Channathodiyil, P.; Kannappan, V.; Hugnot, J-P.; Guichet, P.-O.; Bian, X.; Armesilla, A. L.; Darling, J. L.; Wang, W.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1488-1497</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Background: Glioblastoma multiforme (GBM) cells are resistant to anticancer drugs.  Cancer stem cells (CSCs) are a key mediator of chemoresistance.  We have reported that disulfiram (DS), an aldehyde dehydrogenase (ALDH) inhibitor, targets breast CSC-like cells.  In this study, the effect of DS and combination of DS and gemcitabine (dFdC) on GBM cells and GBM stem-like cells was investigated.  Methods: 1-(4,5-Dimethylthiazol-2-yl)-3,5-diphenylformazan (MTT), combination index (CI)-isobologram, western blot, luciferase reporter gene assay, electrophoretic mobility-shift assay and ALDH anal. were used in this study.  Results: Disulfiram is cytotoxic in GBM cell lines in a copper (Cu)-dependent manner.  Disulfiram/copper enhances the cytotoxicity of dFdC.  Combination index-isobologram anal. indicates a synergistic effect between DS/Cu and dFdC.  Disulfiram/copper induces reactive oxygen species (ROS), activates JNK and p38 pathways and inhibits nuclear factor-kappa B activity in GBM cell lines.  Disulfiram/copper may trigger intrinsic apoptotic pathway via modulation of the Bcl2 family.  Disulfiram/copper abolishes stem-like cell population in GBM cell lines.  Conclusion: Our findings indicate that the cytotoxicity of DS/Cu and the enhancing effect of DS/Cu on the cytotoxicity of dFdC in GBM stem-like cells may be caused by induction of ROS and inhibition of both ALDH and the NFkB pathway.  Both DS and dFdC can traverse the blood-brain barrier.  Further study may lead them into GBM chemotherapy.  British Journal of Cancer (2012) 107, 1488-1497; doi:10.1038/bjc.2012.442 www.bjcancer.com Published online 2 Oct. 2012.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4qo20htmw2bVg90H21EOLACvtfcHk0liP6g2D43j29Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOntLjO&md5=d66fc5467e594264c42dd3e6f61f7083</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2012.442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2012.442%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DGoktug%26aufirst%3DT.%26aulast%3DChannathodiyil%26aufirst%3DP.%26aulast%3DKannappan%26aufirst%3DV.%26aulast%3DHugnot%26aufirst%3DJ.%2BP.%26aulast%3DGuichet%26aufirst%3DP.%2BO.%26aulast%3DBian%26aufirst%3DX.%26aulast%3DArmesilla%26aufirst%3DA.%2BL.%26aulast%3DDarling%26aufirst%3DJ.%2BL.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DCytotoxic%2520Effect%2520of%2520Disulfiram%252FCopper%2520on%2520Human%2520Glioblastoma%2520Cell%2520Lines%2520and%2520ALDH-Positive%2520Cancer-Stem-Like%2520Cells%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2012%26volume%3D107%26issue%3D9%26spage%3D1488%26epage%3D1497%26doi%3D10.1038%2Fbjc.2012.442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nechushtan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamreh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotfried, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalev, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nisman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peretz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peylan-Ramu, N.</span></span> <span> </span><span class="NLM_article-title">A Phase IIb Trial Assessing the Addition of Disulfiram to Chemotherapy for the Treatment of Metastatic Non-Small Cell Lung Cancer</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">366</span>â <span class="NLM_lpage">367</span>, <span class="refDoi">Â DOI: 10.1634/theoncologist.2014-0424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1634%2Ftheoncologist.2014-0424" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=366-367&issue=4&author=H.+Nechushtanauthor=Y.+Hamamrehauthor=S.+Nidalauthor=M.+Gotfriedauthor=A.+Baronauthor=Y.+I.+Shalevauthor=B.+Nismanauthor=T.+Peretzauthor=N.+Peylan-Ramu&title=A+Phase+IIb+Trial+Assessing+the+Addition+of+Disulfiram+to+Chemotherapy+for+the+Treatment+of+Metastatic+Non-Small+Cell+Lung+Cancer&doi=10.1634%2Ftheoncologist.2014-0424"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2014-0424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2014-0424%26sid%3Dliteratum%253Aachs%26aulast%3DNechushtan%26aufirst%3DH.%26aulast%3DHamamreh%26aufirst%3DY.%26aulast%3DNidal%26aufirst%3DS.%26aulast%3DGotfried%26aufirst%3DM.%26aulast%3DBaron%26aufirst%3DA.%26aulast%3DShalev%26aufirst%3DY.%2BI.%26aulast%3DNisman%26aufirst%3DB.%26aulast%3DPeretz%26aufirst%3DT.%26aulast%3DPeylan-Ramu%26aufirst%3DN.%26atitle%3DA%2520Phase%2520IIb%2520Trial%2520Assessing%2520the%2520Addition%2520of%2520Disulfiram%2520to%2520Chemotherapy%2520for%2520the%2520Treatment%2520of%2520Metastatic%2520Non-Small%2520Cell%2520Lung%2520Cancer%26jtitle%3DOncologist%26date%3D2015%26volume%3D20%26issue%3D4%26spage%3D366%26epage%3D367%26doi%3D10.1634%2Ftheoncologist.2014-0424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skrott, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistrik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozdian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moudry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaclavkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrobel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouckova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedlacek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miklovicova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driessen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Q. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">Alcohol-Abuse Drug Disulfiram Targets Cancer Via p97 Segregase Adaptor NPL4</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>552</i></span>,  <span class="NLM_fpage">194</span>, <span class="refDoi">Â DOI: 10.1038/nature25016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fnature25016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=29211715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFGlsbrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=552&publication_year=2017&pages=194&author=Z.+Skrottauthor=M.+Mistrikauthor=K.+K.+Andersenauthor=S.+Friisauthor=D.+Majeraauthor=J.+Gurskyauthor=T.+Ozdianauthor=J.+Bartkovaauthor=Z.+Turiauthor=P.+Moudryauthor=M.+Krausauthor=M.+Michalovaauthor=J.+Vaclavkovaauthor=P.+Dzubakauthor=I.+Vrobelauthor=P.+Pouckovaauthor=J.+Sedlacekauthor=A.+Miklovicovaauthor=A.+Kuttauthor=J.+Liauthor=J.+Mattovaauthor=C.+Driessenauthor=Q.+P.+Douauthor=J.+Olsenauthor=M.+Hajduchauthor=B.+Cvekauthor=R.+J.+Deshaiesauthor=J.+Bartek&title=Alcohol-Abuse+Drug+Disulfiram+Targets+Cancer+Via+p97+Segregase+Adaptor+NPL4&doi=10.1038%2Fnature25016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Alcohol-abuse drug disulfiram targets cancer via p97 segregase adaptor NPL4</span></div><div class="casAuthors">Skrott, Zdenek; Mistrik, Martin; Andersen, Klaus Kaae; Friis, Soeren; Majera, Dusana; Gursky, Jan; Ozdian, Tomas; Bartkova, Jirina; Turi, Zsofia; Moudry, Pavel; Kraus, Marianne; Michalova, Martina; Vaclavkova, Jana; Dzubak, Petr; Vrobel, Ivo; Pouckova, Pavla; Sedlacek, Jindrich; Miklovicova, Andrea; Kutt, Anne; Li, Jing; Mattova, Jana; Driessen, Christoph; Dou, Q. Ping; Olsen, Joergen; Hajduch, Marian; Cvek, Boris; Deshaies, Raymond J.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">552</span>
        (<span class="NLM_cas:issue">7684</span>),
    <span class="NLM_cas:pages">194-199</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cancer incidence is rising and this global challenge is further exacerbated by tumor resistance to available medicines.  A promising approach to meet the need for improved cancer treatment is drug repurposing.  Here we highlight the potential for repurposing disulfiram (also known by the trade name Antabuse), an old alc.-aversion drug that has been shown to be effective against diverse cancer types in preclin. studies.  Our nationwide epidemiol. study reveals that patients who continuously used disulfiram have a lower risk of death from cancer compared to those who stopped using the drug at their diagnosis.  Moreover, we identify the ditiocarb-copper complex as the metabolite of disulfiram that is responsible for its anti-cancer effects, and provide methods to detect preferential accumulation of the complex in tumors and candidate biomarkers to analyze its effect on cells and tissues.  Finally, our functional and biophys. analyses reveal the mol. target of disulfiram's tumor-suppressing effects as NPL4, an adaptor of p97 (also known as VCP) segregase, which is essential for the turnover of proteins involved in multiple regulatory and stress-response pathways in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp_sAn4LKB57Vg90H21EOLACvtfcHk0liP6g2D43j29Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFGlsbrN&md5=b53235723e8478ab49b7d569d0dd35ac</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature25016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25016%26sid%3Dliteratum%253Aachs%26aulast%3DSkrott%26aufirst%3DZ.%26aulast%3DMistrik%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DK.%2BK.%26aulast%3DFriis%26aufirst%3DS.%26aulast%3DMajera%26aufirst%3DD.%26aulast%3DGursky%26aufirst%3DJ.%26aulast%3DOzdian%26aufirst%3DT.%26aulast%3DBartkova%26aufirst%3DJ.%26aulast%3DTuri%26aufirst%3DZ.%26aulast%3DMoudry%26aufirst%3DP.%26aulast%3DKraus%26aufirst%3DM.%26aulast%3DMichalova%26aufirst%3DM.%26aulast%3DVaclavkova%26aufirst%3DJ.%26aulast%3DDzubak%26aufirst%3DP.%26aulast%3DVrobel%26aufirst%3DI.%26aulast%3DPouckova%26aufirst%3DP.%26aulast%3DSedlacek%26aufirst%3DJ.%26aulast%3DMiklovicova%26aufirst%3DA.%26aulast%3DKutt%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMattova%26aufirst%3DJ.%26aulast%3DDriessen%26aufirst%3DC.%26aulast%3DDou%26aufirst%3DQ.%2BP.%26aulast%3DOlsen%26aufirst%3DJ.%26aulast%3DHajduch%26aufirst%3DM.%26aulast%3DCvek%26aufirst%3DB.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DAlcohol-Abuse%2520Drug%2520Disulfiram%2520Targets%2520Cancer%2520Via%2520p97%2520Segregase%2520Adaptor%2520NPL4%26jtitle%3DNature%26date%3D2017%26volume%3D552%26spage%3D194%26doi%3D10.1038%2Fnature25016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogarth, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlatt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haenisch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griswold, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amory, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isoherranen, N.</span></span> <span> </span><span class="NLM_article-title">Importance of ALDH1A Enzymes in Determining Human Testicular Retinoic Acid Concentrations</span>. <i>J. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">342</span>â <span class="NLM_lpage">357</span>, <span class="refDoi">Â DOI: 10.1194/jlr.M054718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1194%2Fjlr.M054718" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2015&pages=342-357&issue=2&author=S.+L.+Arnoldauthor=T.+Kentauthor=C.+A.+Hogarthauthor=S.+Schlattauthor=B.+Prasadauthor=M.+Haenischauthor=T.+Walshauthor=C.+H.+Mullerauthor=M.+D.+Griswoldauthor=J.+K.+Amoryauthor=N.+Isoherranen&title=Importance+of+ALDH1A+Enzymes+in+Determining+Human+Testicular+Retinoic+Acid+Concentrations&doi=10.1194%2Fjlr.M054718"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M054718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M054718%26sid%3Dliteratum%253Aachs%26aulast%3DArnold%26aufirst%3DS.%2BL.%26aulast%3DKent%26aufirst%3DT.%26aulast%3DHogarth%26aufirst%3DC.%2BA.%26aulast%3DSchlatt%26aufirst%3DS.%26aulast%3DPrasad%26aufirst%3DB.%26aulast%3DHaenisch%26aufirst%3DM.%26aulast%3DWalsh%26aufirst%3DT.%26aulast%3DMuller%26aufirst%3DC.%2BH.%26aulast%3DGriswold%26aufirst%3DM.%2BD.%26aulast%3DAmory%26aufirst%3DJ.%2BK.%26aulast%3DIsoherranen%26aufirst%3DN.%26atitle%3DImportance%2520of%2520ALDH1A%2520Enzymes%2520in%2520Determining%2520Human%2520Testicular%2520Retinoic%2520Acid%2520Concentrations%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2015%26volume%3D56%26issue%3D2%26spage%3D342%26epage%3D357%26doi%3D10.1194%2Fjlr.M054718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oster, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilburn, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegal, F. P.</span></span> <span> </span><span class="NLM_article-title">Chemically Induced Congenital Thymic Dysgenesis in the Rat: a Model of the DiGeorge Syndrome</span>. <i>Clin. Immunol. Immunopathol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">128</span>â <span class="NLM_lpage">134</span>, <span class="refDoi">Â DOI: 10.1016/0090-1229(83)90196-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2F0090-1229%2883%2990196-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=6872357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A280%3ADyaL3s3mtVSrsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1983&pages=128-134&issue=1&author=G.+Osterauthor=K.+H.+Kilburnauthor=F.+P.+Siegal&title=Chemically+Induced+Congenital+Thymic+Dysgenesis+in+the+Rat%3A+a+Model+of+the+DiGeorge+Syndrome&doi=10.1016%2F0090-1229%2883%2990196-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced congenital thymic dysgenesis in the rat: a model of the DiGeorge syndrome</span></div><div class="casAuthors">Oster G; Kilburn K H; Siegal F P</div><div class="citationInfo"><span class="NLM_cas:title">Clinical immunology and immunopathology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">128-34</span>
        ISSN:<span class="NLM_cas:issn">0090-1229</span>.
    </div><div class="casAbstract">The DiGeorge syndrome, a variable complex of thymic aplasia, congenital heart disease, hypoparathyroidism, and anomalies of the face and neck, is thought to result from exposure to teratogenic agents.  A group of congenital defects closely resembling this syndrome can be produced in newborn rats by the administration of the fat-soluble zinc chelating agent WIN 18,446, a bis-dichloroacetylamine.  This drug, nontoxic to adult animals, is a powerful teratogen when administered to pregnant rats during days 9-12 of the 21-day gestation period.  Our animal data suggest that the human syndrome results from exposure in utero to agents like WIN 18,446, which damage the fetus during a critical period of organogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD3_j8PryB7zAEfvC7EH1LfW6udTcc2eZukwUqzlFS5Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3s3mtVSrsA%253D%253D&md5=6a30bdb6cac84c93af43abc3d80ac78b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2F0090-1229%2883%2990196-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0090-1229%252883%252990196-4%26sid%3Dliteratum%253Aachs%26aulast%3DOster%26aufirst%3DG.%26aulast%3DKilburn%26aufirst%3DK.%2BH.%26aulast%3DSiegal%26aufirst%3DF.%2BP.%26atitle%3DChemically%2520Induced%2520Congenital%2520Thymic%2520Dysgenesis%2520in%2520the%2520Rat%253A%2520a%2520Model%2520of%2520the%2520DiGeorge%2520Syndrome%26jtitle%3DClin.%2520Immunol.%2520Immunopathol.%26date%3D1983%26volume%3D28%26issue%3D1%26spage%3D128%26epage%3D134%26doi%3D10.1016%2F0090-1229%2883%2990196-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Development of a High-Throughput In Vitro Assay to Identify Selective Inhibitors for Human ALDH1A1</span>. <i>Chem.-Biol. Interact.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>234</i></span> (<span class="NLM_issue">0</span>),  <span class="NLM_fpage">29</span>â <span class="NLM_lpage">37</span>, <span class="refDoi">Â DOI: 10.1016/j.cbi.2014.10.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.cbi.2014.10.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25450233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2gsbzP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2015&pages=29-37&issue=0&author=C.+A.+Morganauthor=T.+D.+Hurley&title=Development+of+a+High-Throughput+In+Vitro+Assay+to+Identify+Selective+Inhibitors+for+Human+ALDH1A1&doi=10.1016%2Fj.cbi.2014.10.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a high-throughput in vitro assay to identify selective inhibitors for human ALDH1A1</span></div><div class="casAuthors">Morgan, Cynthia A.; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29-37</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">The human aldehyde dehydrogenase (ALDH) superfamily consists of at least 19 enzymes that metabolize endogenous and exogenous aldehydes.  Currently, there are no com. available inhibitors that target ALDH1A1 but have little to no effect on the structurally and functionally similar ALDH2.  Here we present the first human ALDH1A1 structure, as the apo-enzyme and in complex with its cofactor NADH to a resoln. of 1.75 and 2.1 Ã, respectfully.  Structural comparisons of the cofactor binding sites in ALDH1A1 with other closely related ALDH enzymes illustrate a high degree of similarity.  In order to minimize discovery of compds. that inhibit both isoenzymes by interfering with their conserved cofactor binding sites, this study reports the use of an in vitro, NAD+-independent, esterase-based high-throughput screen (HTS) of 64,000 compds. to discover novel, selective inhibitors of ALDH1A1.  We describe 256 hits that alter the esterase activity of ALDH1A1.  The effects on aldehyde oxidn. of 67 compds. were further analyzed, with 30 selectively inhibiting ALDH1A1 compared to ALDH2 and ALDH3A1.  One compd. inhibited ALDH1A1 and ALDH2, while another inhibited ALDH1A1, ALDH2, and the more distantly related ALDH3A1.  The results presented here indicate that this in vitro enzyme activity screening protocol successfully identified ALDH1A1 inhibitors with a high degree of isoenzyme selectivity.  The compds. identified via this screen plus the screening methodol. itself represent a starting point for the development of highly potent and selective inhibitors of ALDH1A1 that may be utilized to better understand the role of this enzyme in both normal and disease states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0H8M4JN5aC7Vg90H21EOLACvtfcHk0lggH9BjylEjZg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2gsbzP&md5=64ce33cd18ae04a4ef8796bd9c4474a7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.10.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.10.028%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520a%2520High-Throughput%2520In%2520Vitro%2520Assay%2520to%2520Identify%2520Selective%2520Inhibitors%2520for%2520Human%2520ALDH1A1%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D2015%26volume%3D234%26issue%3D0%26spage%3D29%26epage%3D37%26doi%3D10.1016%2Fj.cbi.2014.10.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Characterization of Two Distinct Structural Classes of Selective Aldehyde Dehydrogenase 1A1 Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1964</span>â <span class="NLM_lpage">1975</span>, <span class="refDoi">Â DOI: 10.1021/jm501900s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501900s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12ltr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1964-1975&issue=4&author=C.+A.+Morganauthor=T.+D.+Hurley&title=Characterization+of+Two+Distinct+Structural+Classes+of+Selective+Aldehyde+Dehydrogenase+1A1+Inhibitors&doi=10.1021%2Fjm501900s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Two Distinct Structural Classes of Selective Aldehyde Dehydrogenase 1A1 Inhibitors</span></div><div class="casAuthors">Morgan, Cynthia A.; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1964-1975</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDH) catalyze the irreversible oxidn. of aldehydes to their corresponding carboxylic acid.  Alterations in ALDH1A1 activity are assocd. with such diverse diseases as cancer, Parkinson's disease, obesity, and cataracts.  Inhibitors of ALDH1A1 could aid in illuminating the role of this enzyme in disease processes.  However, there are no com. available selective inhibitors for ALDH1A1.  Here we characterize two distinct chem. classes of inhibitors that are selective for human ALDH1A1 compared to eight other ALDH isoenzymes.  The prototypical members of each structural class, CM026 and CM037, exhibit submicromolar inhibition consts. but have different mechanisms of inhibition.  The crystal structures of these compds. bound to ALDH1A1 demonstrate that they bind within the aldehyde binding pocket of ALDH1A1 and exploit the presence of a unique glycine residue to achieve their selectivity.  These two novel and selective ALDH1A1 inhibitors may serve as chem. tools to better understand the contributions of ALDH1A1 to normal biol. and to disease states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUpN6wS1yVQLVg90H21EOLACvtfcHk0lggH9BjylEjZg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12ltr0%253D&md5=102a2c49cbf0bc89f620934e8b7872f2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm501900s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501900s%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DCharacterization%2520of%2520Two%2520Distinct%2520Structural%2520Classes%2520of%2520Selective%2520Aldehyde%2520Dehydrogenase%25201A1%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D4%26spage%3D1964%26epage%3D1975%26doi%3D10.1021%2Fjm501900s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal-Nag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of NCT-501, a Potent and Selective Theophylline-Based Inhibitor of Aldehyde Dehydrogenase 1A1 (ALDH1A1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">5967</span>â <span class="NLM_lpage">5978</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00577</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00577" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5967-5978&issue=15&author=S.+M.+Yangauthor=A.+Yasgarauthor=B.+Millerauthor=M.+Lal-Nagauthor=K.+Brimacombeauthor=X.+Huauthor=H.+Sunauthor=A.+Wangauthor=X.+Xuauthor=K.+Nguyenauthor=U.+Oppermannauthor=M.+Ferrerauthor=V.+Vasiliouauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+J.+Maloney&title=Discovery+of+NCT-501%2C+a+Potent+and+Selective+Theophylline-Based+Inhibitor+of+Aldehyde+Dehydrogenase+1A1+%28ALDH1A1%29&doi=10.1021%2Facs.jmedchem.5b00577"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00577%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%2BM.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DB.%26aulast%3DLal-Nag%26aufirst%3DM.%26aulast%3DBrimacombe%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520NCT-501%252C%2520a%2520Potent%2520and%2520Selective%2520Theophylline-Based%2520Inhibitor%2520of%2520Aldehyde%2520Dehydrogenase%25201A1%2520%2528ALDH1A1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D15%26spage%3D5967%26epage%3D5978%26doi%3D10.1021%2Facs.jmedchem.5b00577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danchik, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baljinnyam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheff, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal-Nag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunford, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Orally Bioavailable, Quinoline-Based Aldehyde Dehydrogenase 1A1 (ALDH1A1) Inhibitors with Potent Cellular Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">4883</span>â <span class="NLM_lpage">4903</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00270</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00270" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps12msL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4883-4903&issue=11&author=S.+M.+Yangauthor=N.+J.+Martinezauthor=A.+Yasgarauthor=C.+Danchikauthor=C.+Johanssonauthor=Y.+Wangauthor=B.+Baljinnyamauthor=A.+Q.+Wangauthor=X.+Xuauthor=P.+Shahauthor=D.+Cheffauthor=X.+S.+Wangauthor=J.+Rothauthor=M.+Lal-Nagauthor=J.+E.+Dunfordauthor=U.+Oppermannauthor=V.+Vasiliouauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+J.+Maloney&title=Discovery+of+Orally+Bioavailable%2C+Quinoline-Based+Aldehyde+Dehydrogenase+1A1+%28ALDH1A1%29+Inhibitors+with+Potent+Cellular+Activity&doi=10.1021%2Facs.jmedchem.8b00270"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Orally Bioavailable, Quinoline-Based Aldehyde Dehydrogenase 1A1 (ALDH1A1) Inhibitors with Potent Cellular Activity</span></div><div class="casAuthors">Yang, Shyh-Ming; Martinez, Natalia J.; Yasgar, Adam; Danchik, Carina; Johansson, Catrine; Wang, Yuhong; Baljinnyam, Bolormaa; Wang, Amy Q.; Xu, Xin; Shah, Pranav; Cheff, Dorian; Wang, Xinran S.; Roth, Jacob; Lal-Nag, Madhu; Dunford, James E.; Oppermann, Udo; Vasiliou, Vasilis; Simeonov, Anton; Jadhav, Ajit; Maloney, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4883-4903</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDHs) are responsible for the metab. of aldehydes (exogenous and endogenous) and possess vital physiol. and toxicol. functions in areas such as CNS, inflammation, metabolic disorders, and cancers.  Overexpression of certain ALDHs (e.g., ALDH1A1) is an important biomarker in cancers and cancer stem cells (CSCs) indicating the potential need for the identification and development of small mol. ALDH inhibitors.  Herein, a newly designed series of quinoline-based analogs of ALDH1A1 inhibitors is described.  Extensive medicinal chem. optimization and biol. characterization led to the identification of analogs with significantly improved enzymic and cellular ALDH inhibition.  Selected analogs, e.g., 86 (NCT-505) and 91 (NCT-506), demonstrated target engagement in a cellular thermal shift assay (CETSA), inhibited the formation of 3D spheroid cultures of OV-90 cancer cells, and potentiated the cytotoxicity of paclitaxel in SKOV-3-TR, a paclitaxel resistant ovarian cancer cell line.  Lead compds. also exhibit high specificity over other ALDH isoenzymes and unrelated dehydrogenases.  The in vitro ADME profiles and pharmacokinetic evaluation of selected analogs are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMdOrSgvA10rVg90H21EOLACvtfcHk0ljePCSn610ZiA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps12msL8%253D&md5=6f63e6c9803cb8f0e113fb1f81bebf8a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00270%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%2BM.%26aulast%3DMartinez%26aufirst%3DN.%2BJ.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DDanchik%26aufirst%3DC.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBaljinnyam%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DCheff%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%2BS.%26aulast%3DRoth%26aufirst%3DJ.%26aulast%3DLal-Nag%26aufirst%3DM.%26aulast%3DDunford%26aufirst%3DJ.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520Orally%2520Bioavailable%252C%2520Quinoline-Based%2520Aldehyde%2520Dehydrogenase%25201A1%2520%2528ALDH1A1%2529%2520Inhibitors%2520with%2520Potent%2520Cellular%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D11%26spage%3D4883%26epage%3D4903%26doi%3D10.1021%2Facs.jmedchem.8b00270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span>; <span class="NLM_string-name">Mao, L.</span>; <span class="NLM_string-name">Wang, X.</span>; <span class="NLM_string-name">Xu, X.</span>; <span class="NLM_string-name">Zhao, L.</span></span> <i>Pyrazolopyrimidine Derivatives and Uses as Anticancer Agents</i>. Patent No. <span class="NLM_patent">WO2012097196A1</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=J.+Liu&author=L.+Mao&author=X.+Wang&author=X.+Xu&author=L.+Zhao&title=Pyrazolopyrimidine+Derivatives+and+Uses+as+Anticancer+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26jtitle%3DPyrazolopyrimidine%2520Derivatives%2520and%2520Uses%2520as%2520Anticancer%2520Agents%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffaelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barretta, G. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolaro, L. M.</span></span> <span> </span><span class="NLM_article-title">Potential of Alkyl Orthoformates as Alkylating Agents of Non-Electron Rich Nitrogen Heterocycles</span>. <i>J. Heterocycl. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1149</span>â <span class="NLM_lpage">1158</span>, <span class="refDoi">Â DOI: 10.1002/jhet.5570320409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fjhet.5570320409" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1995&pages=1149-1158&issue=4&author=S.+Guccioneauthor=F.+Russoauthor=A.+Raffaelliauthor=G.+U.+Barrettaauthor=S.+Pucciauthor=L.+M.+Scolaro&title=Potential+of+Alkyl+Orthoformates+as+Alkylating+Agents+of+Non-Electron+Rich+Nitrogen+Heterocycles&doi=10.1002%2Fjhet.5570320409"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570320409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570320409%26sid%3Dliteratum%253Aachs%26aulast%3DGuccione%26aufirst%3DS.%26aulast%3DRusso%26aufirst%3DF.%26aulast%3DRaffaelli%26aufirst%3DA.%26aulast%3DBarretta%26aufirst%3DG.%2BU.%26aulast%3DPucci%26aufirst%3DS.%26aulast%3DScolaro%26aufirst%3DL.%2BM.%26atitle%3DPotential%2520of%2520Alkyl%2520Orthoformates%2520as%2520Alkylating%2520Agents%2520of%2520Non-Electron%2520Rich%2520Nitrogen%2520Heterocycles%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D1995%26volume%3D32%26issue%3D4%26spage%3D1149%26epage%3D1158%26doi%3D10.1002%2Fjhet.5570320409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, B. K.</span></span> <span> </span><span class="NLM_article-title">Molecular Iodine Mediated Preparation of Isothiocyanates from Dithiocarbamic Acid Salts</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2009</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1849</span>â <span class="NLM_lpage">1851</span>, <span class="refDoi">Â DOI: 10.1002/ejoc.200801270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fejoc.200801270" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2009&publication_year=2009&pages=1849-1851&issue=12&author=J.+Nathauthor=H.+Ghoshauthor=R.+Yellaauthor=B.+K.+Patel&title=Molecular+Iodine+Mediated+Preparation+of+Isothiocyanates+from+Dithiocarbamic+Acid+Salts&doi=10.1002%2Fejoc.200801270"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fejoc.200801270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.200801270%26sid%3Dliteratum%253Aachs%26aulast%3DNath%26aufirst%3DJ.%26aulast%3DGhosh%26aufirst%3DH.%26aulast%3DYella%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520Iodine%2520Mediated%2520Preparation%2520of%2520Isothiocyanates%2520from%2520Dithiocarbamic%2520Acid%2520Salts%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2009%26volume%3D2009%26issue%3D12%26spage%3D1849%26epage%3D1851%26doi%3D10.1002%2Fejoc.200801270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinzler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilbert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelstein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preisinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedge, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKechnie, D.</span></span> <span> </span><span class="NLM_article-title">Identification of FAP Locus Genes from Chromosome 5q21</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>253</i></span> (<span class="NLM_issue">5020</span>),  <span class="NLM_fpage">661</span>â <span class="NLM_lpage">665</span>, <span class="refDoi">Â DOI: 10.1126/science.1651562</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1126%2Fscience.1651562" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1991&pages=661-665&issue=5020&author=K.+Kinzlerauthor=M.+Nilbertauthor=L.+Suauthor=B.+Vogelsteinauthor=T.+Bryanauthor=D.+Levyauthor=K.+Smithauthor=A.+Preisingerauthor=P.+Hedgeauthor=D.+McKechnie&title=Identification+of+FAP+Locus+Genes+from+Chromosome+5q21&doi=10.1126%2Fscience.1651562"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1126%2Fscience.1651562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1651562%26sid%3Dliteratum%253Aachs%26aulast%3DKinzler%26aufirst%3DK.%26aulast%3DNilbert%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DBryan%26aufirst%3DT.%26aulast%3DLevy%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DK.%26aulast%3DPreisinger%26aufirst%3DA.%26aulast%3DHedge%26aufirst%3DP.%26aulast%3DMcKechnie%26aufirst%3DD.%26atitle%3DIdentification%2520of%2520FAP%2520Locus%2520Genes%2520from%2520Chromosome%25205q21%26jtitle%3DScience%26date%3D1991%26volume%3D253%26issue%3D5020%26spage%3D661%26epage%3D665%26doi%3D10.1126%2Fscience.1651562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span>MarvinSketch
Calculator Plugins were used for ADME predictions.  <i>Marvin 15.6.15</i>, <span class="NLM_year">2015</span>, <span class="NLM_publisher-name">ChemAxon</span>. See the following: <a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a> (accessed
June 5, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MarvinSketch%0ACalculator+Plugins+were+used+for+ADME+predictions.+Marvin+15.6.15%2C+2015%2C+ChemAxon.+See+the+following%3A+http%3A%2F%2Fwww.chemaxon.com+%28accessed%0AJune+5%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMarvin%252015.6.15%26date%3D2015%26pub%3DChemAxon" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikles, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amory, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of ALDH1A2 Inhibition by Irreversible and Reversible Small Molecule Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">582</span>â <span class="NLM_lpage">590</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.7b00685</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00685" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOgt7bK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=582-590&issue=3&author=Y.+Chenauthor=J.+Y.+Zhuauthor=K.+H.+Hongauthor=D.+C.+Miklesauthor=G.+I.+Georgauthor=A.+S.+Goldsteinauthor=J.+K.+Amoryauthor=E.+Schonbrunn&title=Structural+Basis+of+ALDH1A2+Inhibition+by+Irreversible+and+Reversible+Small+Molecule+Inhibitors&doi=10.1021%2Facschembio.7b00685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of ALDH1A2 Inhibition by Irreversible and Reversible Small Molecule Inhibitors</span></div><div class="casAuthors">Chen, Yan; Zhu, Jin-Yi; Hong, Kwon Ho; Mikles, David C.; Georg, Gunda I.; Goldstein, Alex S.; Amory, John K.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">582-590</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymes of the ALDH1A subfamily of aldehyde dehydrogenases are crucial in regulating retinoic acid (RA) signaling and have received attention as potential drug targets.  ALDH1A2 is the primary RA-synthesizing enzyme in mammalian spermatogenesis and is therefore considered a viable drug target for male contraceptive development.  However, only a small no. of ALDH1A2 inhibitors have been reported, and information on the structure of ALDH1A2 was limited to the NAD-liganded enzyme void of substrate or inhibitors.  Herein, the authors describe the mechanism of action of structurally unrelated reversible and irreversible inhibitors of human ALDH1A2 using direct binding studies and x-ray crystallog.  All inhibitors bind to the active sites of tetrameric ALDH1A2.  Compd. WIN18,446 covalently reacts with the side chain of the catalytic residue Cys 320, resulting in a chiral adduct in (R) configuration.  The covalent adduct directly affects the neighboring NAD mol., which assumes a contracted conformation suboptimal for the dehydrogenase reaction.  The reversible inhibitors interact predominantly through direct hydrogen bonding interactions with residues in the vicinity of Cys 320 without affecting NAD.  Upon interaction with inhibitors, a large flexible loop assumes regular structure, thereby shielding the active site from solvent.  The precise knowledge of the binding modes provides a new framework for the rational design of novel inhibitors of ALDH1A2 with improved potency and selectivity profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprC4rddNQ7u7Vg90H21EOLACvtfcHk0ljePCSn610ZiA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOgt7bK&md5=dff6eef4a876a603048f3ccd53f60dee</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00685%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DHong%26aufirst%3DK.%2BH.%26aulast%3DMikles%26aufirst%3DD.%2BC.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DGoldstein%26aufirst%3DA.%2BS.%26aulast%3DAmory%26aufirst%3DJ.%2BK.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DStructural%2520Basis%2520of%2520ALDH1A2%2520Inhibition%2520by%2520Irreversible%2520and%2520Reversible%2520Small%2520Molecule%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D3%26spage%3D582%26epage%3D590%26doi%3D10.1021%2Facschembio.7b00685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beaufort, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helmijr, J. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piskorz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoogstraat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruigrok-Ritstier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besselink, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murtaza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Ijcken, W. F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koudijs, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berns, E. M. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleman, J.</span></span> <span> </span><span class="NLM_article-title">Ovarian Cancer Cell Line Panel (OCCP): Clinical Importance of In Vitro Morphological Subtypes</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">e103988</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0103988</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0103988" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25230021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslaqsbbE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e103988&issue=9&author=C.+M.+Beaufortauthor=J.+C.+A.+Helmijrauthor=A.+M.+Piskorzauthor=M.+Hoogstraatauthor=K.+Ruigrok-Ritstierauthor=N.+Besselinkauthor=M.+Murtazaauthor=W.+F.+J.+van+Ijckenauthor=A.+A.+J.+Heineauthor=M.+Smidauthor=M.+J.+Koudijsauthor=J.+D.+Brentonauthor=E.+M.+J.+J.+Bernsauthor=J.+Helleman&title=Ovarian+Cancer+Cell+Line+Panel+%28OCCP%29%3A+Clinical+Importance+of+In+Vitro+Morphological+Subtypes&doi=10.1371%2Fjournal.pone.0103988"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian Cancer Cell Line Panel (OCCP): clinical importance of in vitro morphological subtypes</span></div><div class="casAuthors">Beaufort, Corine M.; Helmijr, Jean C. A.; Piskorz, Anna M.; Hoogstraat, Marlous; Ruigrok-Ritstier, Kirsten; Besselink, Nicolle; Murtaza, Muhammed; van Ijcken, Wilfred F. J.; Heine, Anouk A. J.; Smid, Marcel; Koudijs, Marco J.; Brenton, James D.; Berns, Els M. J. J.; Helleman, Jozien</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e103988/1-e103988/16, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Epithelial ovarian cancer is a highly heterogeneous disease and remains the most lethal gynaecol. malignancy in the Western world.  Therapeutic approaches need to account for inter-patient and intra-tumoral heterogeneity and detailed characterization of in vitro models representing the different histol. and mol. ovarian cancer subtypes is crit. to enable reliable preclin. testing.  There are approx. 100 publicly available ovarian cancer cell lines but their cellular and mol. characteristics are largely undescribed.  We have characterized 39 ovarian cancer cell lines under uniform conditions for growth characteristics, mRNA/microRNA expression, exon sequencing, drug response for clin.-relevant therapeutics and collated all available information on the original clin. features and site of origin.  We tested for statistical assocns. between the cellular and mol. features of the lines and clin. features.  Of the 39 ovarian cancer cell lines, 14 were assigned as high-grade serous, four serous-type, one low-grade serous and 20 non-serous type.  Three morphol. subtypes: Epithelial (n = 21), Round (n = 7) and Spindle (n = 12) were identified that showed distinct biol. and mol. characteristics, including overexpression of cell movement and migration-assocd. genes in the Spindle subtype.  Comparison with the original clin. data showed assocn. of the spindle-like tumors with metastasis, advanced stage, suboptimal debulking and poor prognosis.  In addn., the expression profiles of Spindle, Round and Epithelial morphologies clustered with the previously described C1-stromal, C5-mesenchymal and C4 ovarian subtype expression profiles resp.  Comprehensive profiling of 39 ovarian cancer cell lines under controlled, uniform conditions demonstrates clin. relevant cellular and genomic characteristics.  This data provides a rational basis for selecting models to develop specific treatment approaches for histol. and mol. subtypes of ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI3caC6_-9C7Vg90H21EOLACvtfcHk0lhGIl1npbHMsg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslaqsbbE&md5=39a63787ab20dabe8003c08d1908a90a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0103988&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0103988%26sid%3Dliteratum%253Aachs%26aulast%3DBeaufort%26aufirst%3DC.%2BM.%26aulast%3DHelmijr%26aufirst%3DJ.%2BC.%2BA.%26aulast%3DPiskorz%26aufirst%3DA.%2BM.%26aulast%3DHoogstraat%26aufirst%3DM.%26aulast%3DRuigrok-Ritstier%26aufirst%3DK.%26aulast%3DBesselink%26aufirst%3DN.%26aulast%3DMurtaza%26aufirst%3DM.%26aulast%3Dvan%2BIjcken%26aufirst%3DW.%2BF.%2BJ.%26aulast%3DHeine%26aufirst%3DA.%2BA.%2BJ.%26aulast%3DSmid%26aufirst%3DM.%26aulast%3DKoudijs%26aufirst%3DM.%2BJ.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DBerns%26aufirst%3DE.%2BM.%2BJ.%2BJ.%26aulast%3DHelleman%26aufirst%3DJ.%26atitle%3DOvarian%2520Cancer%2520Cell%2520Line%2520Panel%2520%2528OCCP%2529%253A%2520Clinical%2520Importance%2520of%2520In%2520Vitro%2520Morphological%2520Subtypes%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3De103988%26doi%3D10.1371%2Fjournal.pone.0103988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blayney, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCluggage, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capes-Davis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harkin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gourley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, R. D.</span></span> <span> </span><span class="NLM_article-title">Prior Knowledge Transfer Across Transcriptional Data Sets and Technologies Using Compositional Statistics Yields New Mislabelled Ovarian Cell Line</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">e137</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkw578</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1093%2Fnar%2Fgkw578" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e137&issue=17&author=J.+K.+Blayneyauthor=T.+Davisonauthor=N.+McCabeauthor=S.+Walkerauthor=K.+Keatingauthor=T.+Delaneyauthor=C.+Greenanauthor=A.+R.+Williamsauthor=W.+G.+McCluggageauthor=A.+Capes-Davisauthor=D.+P.+Harkinauthor=C.+Gourleyauthor=R.+D.+Kennedy&title=Prior+Knowledge+Transfer+Across+Transcriptional+Data+Sets+and+Technologies+Using+Compositional+Statistics+Yields+New+Mislabelled+Ovarian+Cell+Line&doi=10.1093%2Fnar%2Fgkw578"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw578&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw578%26sid%3Dliteratum%253Aachs%26aulast%3DBlayney%26aufirst%3DJ.%2BK.%26aulast%3DDavison%26aufirst%3DT.%26aulast%3DMcCabe%26aufirst%3DN.%26aulast%3DWalker%26aufirst%3DS.%26aulast%3DKeating%26aufirst%3DK.%26aulast%3DDelaney%26aufirst%3DT.%26aulast%3DGreenan%26aufirst%3DC.%26aulast%3DWilliams%26aufirst%3DA.%2BR.%26aulast%3DMcCluggage%26aufirst%3DW.%2BG.%26aulast%3DCapes-Davis%26aufirst%3DA.%26aulast%3DHarkin%26aufirst%3DD.%2BP.%26aulast%3DGourley%26aufirst%3DC.%26aulast%3DKennedy%26aufirst%3DR.%2BD.%26atitle%3DPrior%2520Knowledge%2520Transfer%2520Across%2520Transcriptional%2520Data%2520Sets%2520and%2520Technologies%2520Using%2520Compositional%2520Statistics%2520Yields%2520New%2520Mislabelled%2520Ovarian%2520Cell%2520Line%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26issue%3D17%26spage%3De137%26doi%3D10.1093%2Fnar%2Fgkw578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, M.</span></span> <span> </span><span class="NLM_article-title">Prognostic Value of Cancer Stem Cell Marker CD133 in Ovarian Cancer: a Meta-Analysis</span>. <i>International Journal of Clinical and Experimental Medicine</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">3080</span>â <span class="NLM_lpage">3088</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=3080-3088&issue=3&author=Q.+Zhouauthor=A.+Chenauthor=H.+Songauthor=J.+Taoauthor=H.+Yangauthor=M.+Zuo&title=Prognostic+Value+of+Cancer+Stem+Cell+Marker+CD133+in+Ovarian+Cancer%3A+a+Meta-Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZuo%26aufirst%3DM.%26atitle%3DPrognostic%2520Value%2520of%2520Cancer%2520Stem%2520Cell%2520Marker%2520CD133%2520in%2520Ovarian%2520Cancer%253A%2520a%2520Meta-Analysis%26jtitle%3DInternational%2520Journal%2520of%2520Clinical%2520and%2520Experimental%2520Medicine%26date%3D2015%26volume%3D8%26issue%3D3%26spage%3D3080%26epage%3D3088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanabe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tezuka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunisada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, S.</span></span> <span> </span><span class="NLM_article-title">A More Efficient Method to Generate Integration-Free Human iPS Cells</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">409</span>â <span class="NLM_lpage">412</span>, <span class="refDoi">Â DOI: 10.1038/nmeth.1591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1038%2Fnmeth.1591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21460823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFahtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2011&pages=409-412&issue=5&author=K.+Okitaauthor=Y.+Matsumuraauthor=Y.+Satoauthor=A.+Okadaauthor=A.+Morizaneauthor=S.+Okamotoauthor=H.+Hongauthor=M.+Nakagawaauthor=K.+Tanabeauthor=K.+Tezukaauthor=T.+Shibataauthor=T.+Kunisadaauthor=M.+Takahashiauthor=J.+Takahashiauthor=H.+Sajiauthor=S.+Yamanaka&title=A+More+Efficient+Method+to+Generate+Integration-Free+Human+iPS+Cells&doi=10.1038%2Fnmeth.1591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A more efficient method to generate integration-free human iPS cells</span></div><div class="casAuthors">Okita, Keisuke; Matsumura, Yasuko; Sato, Yoshiko; Okada, Aki; Morizane, Asuka; Okamoto, Satoshi; Hong, Hyenjong; Nakagawa, Masato; Tanabe, Koji; Tezuka, Ken-ichi; Shibata, Toshiyuki; Kunisada, Takahiro; Takahashi, Masayo; Takahashi, Jun; Saji, Hiroh; Yamanaka, Shinya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report a simple method, using p53 suppression and nontransforming L-Myc, to generate human induced pluripotent stem cells (iPSCs) with episomal plasmid vectors.  We generated human iPSCs from multiple donors, including two putative human leukocyte antigen (HLA)-homozygous donors who match â¼20% of the Japanese population at major HLA loci; most iPSCs are integrated transgene-free.  This method may provide iPSCs suitable for autologous and allologous stem-cell therapy in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN2ueECgZQZbVg90H21EOLACvtfcHk0lhGIl1npbHMsg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFahtrc%253D&md5=9d48be9c24ff58bf9ab862ba9ce3e2b0</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1591%26sid%3Dliteratum%253Aachs%26aulast%3DOkita%26aufirst%3DK.%26aulast%3DMatsumura%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DY.%26aulast%3DOkada%26aufirst%3DA.%26aulast%3DMorizane%26aufirst%3DA.%26aulast%3DOkamoto%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DTanabe%26aufirst%3DK.%26aulast%3DTezuka%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DT.%26aulast%3DKunisada%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DJ.%26aulast%3DSaji%26aufirst%3DH.%26aulast%3DYamanaka%26aufirst%3DS.%26atitle%3DA%2520More%2520Efficient%2520Method%2520to%2520Generate%2520Integration-Free%2520Human%2520iPS%2520Cells%26jtitle%3DNat.%2520Methods%26date%3D2011%26volume%3D8%26issue%3D5%26spage%3D409%26epage%3D412%26doi%3D10.1038%2Fnmeth.1591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bretz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noske, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbe-Hofmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlange, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salnikov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moldenhauer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristiansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altevogt, P.</span></span> <span> </span><span class="NLM_article-title">CD24 Promotes Tumor Cell Invasion by Suppressing Tissue Factor Pathway Inhibitor-2 (TFPI-2) in a c-Src-Dependent Fashion</span>. <i>Clin. Exp. Metastasis</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">27</span>â <span class="NLM_lpage">38</span>, <span class="refDoi">Â DOI: 10.1007/s10585-011-9426-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1007%2Fs10585-011-9426-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21984372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsF2qsr7F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=27-38&issue=1&author=N.+Bretzauthor=A.+Noskeauthor=S.+Kellerauthor=N.+Erbe-Hofmannauthor=T.+Schlangeauthor=A.+V.+Salnikovauthor=G.+Moldenhauerauthor=G.+Kristiansenauthor=P.+Altevogt&title=CD24+Promotes+Tumor+Cell+Invasion+by+Suppressing+Tissue+Factor+Pathway+Inhibitor-2+%28TFPI-2%29+in+a+c-Src-Dependent+Fashion&doi=10.1007%2Fs10585-011-9426-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">CD24 promotes tumor cell invasion by suppressing tissue factor pathway inhibitor-2 (TFPI-2) in a c-Src-dependent fashion</span></div><div class="casAuthors">Bretz, Niko; Noske, Aurelia; Keller, Sascha; Erbe-Hofmann, Natalie; Schlange, Thomas; Salnikov, Alexei V.; Moldenhauer, Gerd; Kristiansen, Glen; Altevogt, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Clinical & Experimental Metastasis</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">CEXMD2</span>;
        ISSN:<span class="NLM_cas:issn">0262-0898</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">CD24 is a glycosyl-phosphatidylinositol-anchored protein with mucin-type structure that resides exclusively in membrane microdomains.  CD24 is often highly expressed in carcinomas and correlates with poor prognosis.  Exptl., the over-expression or depletion of CD24 alters cell proliferation, adhesion, and invasion in vitro and tumor growth in vivo.  However, little is known about the mechanisms by which CD24 mediates these cellular effects.  Here we have studied the mechanism of CD24-dependent cell invasion using transient CD24 knock-down or over-expression in human cancer cell lines.  We show that CD24 depletion reduced tumor cell invasion and up-regulated expression of Tissue Factor Pathway Inhibitor 2 (TFPI-2), a potent inhibitor of extracellular matrix degrdn. that can block metastases formation and tumor cell invasion.  Over-expression of CD24 in A125 cells resulted in reduced TFPI-2 expression and enhanced invasion.  We provide evidence that the activity of c-Src is reduced upon CD24 knock-down.  The silencing of c-Src, similar to CD24, was able to enhance TFPI-2 expression and reduce tumor cell invasion.  An inverse expression of CD24 and TFPI-2 was obsd. by immunohistochem. anal. of primary breast cancers (N = 1,174).  TFPI-2 expression was highest in CD24 neg. samples and lowered with increasing CD24 expression.  Patients with a CD24 low/TFPI-2 high phenotype showed significantly better survival compared to CD24 high/TFPI-2 low patients.  Our results provide evidence that CD24 can regulate cell invasion via TFPI-2 and suggests a role of c-Src in this process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7X2BvIbJpwLVg90H21EOLACvtfcHk0ljewkuGz8VGOQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsF2qsr7F&md5=d3f9718ee880e1c4b1bd7ee6917c9c66</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs10585-011-9426-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10585-011-9426-4%26sid%3Dliteratum%253Aachs%26aulast%3DBretz%26aufirst%3DN.%26aulast%3DNoske%26aufirst%3DA.%26aulast%3DKeller%26aufirst%3DS.%26aulast%3DErbe-Hofmann%26aufirst%3DN.%26aulast%3DSchlange%26aufirst%3DT.%26aulast%3DSalnikov%26aufirst%3DA.%2BV.%26aulast%3DMoldenhauer%26aufirst%3DG.%26aulast%3DKristiansen%26aufirst%3DG.%26aulast%3DAltevogt%26aufirst%3DP.%26atitle%3DCD24%2520Promotes%2520Tumor%2520Cell%2520Invasion%2520by%2520Suppressing%2520Tissue%2520Factor%2520Pathway%2520Inhibitor-2%2520%2528TFPI-2%2529%2520in%2520a%2520c-Src-Dependent%2520Fashion%26jtitle%3DClin.%2520Exp.%2520Metastasis%26date%3D2012%26volume%3D29%26issue%3D1%26spage%3D27%26epage%3D38%26doi%3D10.1007%2Fs10585-011-9426-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dylla, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beviglia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chartier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raval, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazetic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith-Berdan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewicki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurney, A. L.</span></span> <span> </span><span class="NLM_article-title">Colorectal Cancer Stem Cells are Enriched in Xenogeneic Tumors Following Chemotherapy</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">e2428</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0002428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1371%2Fjournal.pone.0002428" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e2428&issue=6&author=S.+J.+Dyllaauthor=L.+Bevigliaauthor=I.+K.+Parkauthor=C.+Chartierauthor=J.+Ravalauthor=L.+Nganauthor=K.+Pickellauthor=J.+Aguilarauthor=S.+Lazeticauthor=S.+Smith-Berdanauthor=M.+F.+Clarkeauthor=T.+Hoeyauthor=J.+Lewickiauthor=A.+L.+Gurney&title=Colorectal+Cancer+Stem+Cells+are+Enriched+in+Xenogeneic+Tumors+Following+Chemotherapy&doi=10.1371%2Fjournal.pone.0002428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0002428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0002428%26sid%3Dliteratum%253Aachs%26aulast%3DDylla%26aufirst%3DS.%2BJ.%26aulast%3DBeviglia%26aufirst%3DL.%26aulast%3DPark%26aufirst%3DI.%2BK.%26aulast%3DChartier%26aufirst%3DC.%26aulast%3DRaval%26aufirst%3DJ.%26aulast%3DNgan%26aufirst%3DL.%26aulast%3DPickell%26aufirst%3DK.%26aulast%3DAguilar%26aufirst%3DJ.%26aulast%3DLazetic%26aufirst%3DS.%26aulast%3DSmith-Berdan%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DM.%2BF.%26aulast%3DHoey%26aufirst%3DT.%26aulast%3DLewicki%26aufirst%3DJ.%26aulast%3DGurney%26aufirst%3DA.%2BL.%26atitle%3DColorectal%2520Cancer%2520Stem%2520Cells%2520are%2520Enriched%2520in%2520Xenogeneic%2520Tumors%2520Following%2520Chemotherapy%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26issue%3D6%26spage%3De2428%26doi%3D10.1371%2Fjournal.pone.0002428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Throndset, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whelan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zare, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manger, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daridon, A.</span></span> <span> </span><span class="NLM_article-title">Microfluidic Device for Single-Cell Analysis</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>75</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">3581</span>â <span class="NLM_lpage">3586</span>, <span class="refDoi">Â DOI: 10.1021/ac0340758</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac0340758" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjs12ktr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=3581-3586&issue=14&author=A.+R.+Wheelerauthor=W.+R.+Throndsetauthor=R.+J.+Whelanauthor=A.+M.+Leachauthor=R.+N.+Zareauthor=Y.+H.+Liaoauthor=K.+Farrellauthor=I.+D.+Mangerauthor=A.+Daridon&title=Microfluidic+Device+for+Single-Cell+Analysis&doi=10.1021%2Fac0340758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Microfluidic device for single-cell analysis</span></div><div class="casAuthors">Wheeler, Aaron R.; Throndset, William R.; Whelan, Rebecca J.; Leach, Andrew M.; Zare, Richard N.; Liao, Yish Hann; Farrell, Kevin; Manger, Ian D.; Daridon, Antoine</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3581-3586</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have developed a novel microfluidic device constructed from poly(dimethylsiloxane) using multilayer soft lithog. technol. for the anal. of single cells.  The microfluidic network enables the passive and gentle sepn. of a single cell from the bulk cell suspension, and integrated valves and pumps enable the precise delivery of nanoliter vols. of reagents to that cell.  Various applications are demonstrated, including cell viability assays, ionophore-mediated intracellular Ca2+ flux measurements, and multistep receptor-mediated Ca2+ measurements.  These assays, and others, are achieved with significant improvements in reagent consumption, anal. time, and temporal resoln. over macroscale alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjEXGz2K_yFbVg90H21EOLACvtfcHk0ljewkuGz8VGOQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjs12ktr0%253D&md5=7ca8c890337654cd79c400aa839ac4c9</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fac0340758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac0340758%26sid%3Dliteratum%253Aachs%26aulast%3DWheeler%26aufirst%3DA.%2BR.%26aulast%3DThrondset%26aufirst%3DW.%2BR.%26aulast%3DWhelan%26aufirst%3DR.%2BJ.%26aulast%3DLeach%26aufirst%3DA.%2BM.%26aulast%3DZare%26aufirst%3DR.%2BN.%26aulast%3DLiao%26aufirst%3DY.%2BH.%26aulast%3DFarrell%26aufirst%3DK.%26aulast%3DManger%26aufirst%3DI.%2BD.%26aulast%3DDaridon%26aufirst%3DA.%26atitle%3DMicrofluidic%2520Device%2520for%2520Single-Cell%2520Analysis%26jtitle%3DAnal.%2520Chem.%26date%3D2003%26volume%3D75%26issue%3D14%26spage%3D3581%26epage%3D3586%26doi%3D10.1021%2Fac0340758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcato, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pass, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cioce, M.</span></span> <span> </span><span class="NLM_article-title">A STAT3-NFkB/DDIT3/CEBPÎ² Axis Modulates ALDH1A3 Expression in Chemoresistant Cell Subpopulations</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">12637</span>â <span class="NLM_lpage">12653</span>, <span class="refDoi">Â DOI: 10.18632/oncotarget.3703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.18632%2Foncotarget.3703" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=12637-12653&issue=14&author=C.+Caninoauthor=Y.+Luoauthor=P.+Marcatoauthor=G.+Blandinoauthor=H.+I.+Passauthor=M.+Cioce&title=A+STAT3-NFkB%2FDDIT3%2FCEBP%CE%B2+Axis+Modulates+ALDH1A3+Expression+in+Chemoresistant+Cell+Subpopulations&doi=10.18632%2Foncotarget.3703"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3703%26sid%3Dliteratum%253Aachs%26aulast%3DCanino%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DMarcato%26aufirst%3DP.%26aulast%3DBlandino%26aufirst%3DG.%26aulast%3DPass%26aufirst%3DH.%2BI.%26aulast%3DCioce%26aufirst%3DM.%26atitle%3DA%2520STAT3-NFkB%252FDDIT3%252FCEBP%25CE%25B2%2520Axis%2520Modulates%2520ALDH1A3%2520Expression%2520in%2520Chemoresistant%2520Cell%2520Subpopulations%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26issue%3D14%26spage%3D12637%26epage%3D12653%26doi%3D10.18632%2Foncotarget.3703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golubovskaya, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conroy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cance, W. G.</span></span> <span> </span><span class="NLM_article-title">Down-Regulation of ALDH1A3, CD44 or MDR1 Sensitizes Resistant Cancer Cells to FAK Autophosphorylation Inhibitor Y15</span>. <i>J. Cancer Res. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>141</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1613</span>â <span class="NLM_lpage">1631</span>, <span class="refDoi">Â DOI: 10.1007/s00432-015-1924-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1007%2Fs00432-015-1924-3" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2015&pages=1613-1631&issue=9&author=V.+Golubovskayaauthor=S.+O%E2%80%99Brienauthor=B.+Hoauthor=M.+Hefflerauthor=J.+Conroyauthor=Q.+Huauthor=D.+Wangauthor=S.+Liuauthor=W.+G.+Cance&title=Down-Regulation+of+ALDH1A3%2C+CD44+or+MDR1+Sensitizes+Resistant+Cancer+Cells+to+FAK+Autophosphorylation+Inhibitor+Y15&doi=10.1007%2Fs00432-015-1924-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2Fs00432-015-1924-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-015-1924-3%26sid%3Dliteratum%253Aachs%26aulast%3DGolubovskaya%26aufirst%3DV.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DS.%26aulast%3DHo%26aufirst%3DB.%26aulast%3DHeffler%26aufirst%3DM.%26aulast%3DConroy%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DCance%26aufirst%3DW.%2BG.%26atitle%3DDown-Regulation%2520of%2520ALDH1A3%252C%2520CD44%2520or%2520MDR1%2520Sensitizes%2520Resistant%2520Cancer%2520Cells%2520to%2520FAK%2520Autophosphorylation%2520Inhibitor%2520Y15%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D141%26issue%3D9%26spage%3D1613%26epage%3D1631%26doi%3D10.1007%2Fs00432-015-1924-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raha, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">The Cancer Stem Cell Marker Aldehyde Dehydrogenase is Required to Maintain a Drug-Tolerant Tumor Cell Subpopulation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3579</span>â <span class="NLM_lpage">3590</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-3456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F0008-5472.CAN-13-3456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=24812274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKlurnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=3579-3590&issue=13&author=D.+Rahaauthor=T.+R.+Wilsonauthor=J.+Pengauthor=D.+Petersonauthor=P.+Yueauthor=M.+Evangelistaauthor=C.+Wilsonauthor=M.+Merchantauthor=J.+Settleman&title=The+Cancer+Stem+Cell+Marker+Aldehyde+Dehydrogenase+is+Required+to+Maintain+a+Drug-Tolerant+Tumor+Cell+Subpopulation&doi=10.1158%2F0008-5472.CAN-13-3456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The Cancer Stem Cell Marker Aldehyde Dehydrogenase Is Required to Maintain a Drug-Tolerant Tumor Cell Subpopulation</span></div><div class="casAuthors">Raha, Debasish; Wilson, Timothy R.; Peng, Jing; Peterson, David; Yue, Peng; Evangelista, Marie; Wilson, Catherine; Merchant, Mark; Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3579-3590</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Selective kinase inhibitors have emerged as an important class of cancer therapeutics, and several such drugs are now routinely used to treat advanced-stage disease.  However, their clin. benefit is typically short-lived because of the relatively rapid acquisition of drug resistance following treatment response.  A Accumlating preclin. and clin. data point to a role for a heterogeneous response to treatment within a subpopulation of tumor cells that are intrinsically drug-resistant, such as cancer stem cells.  We have previously described an epigenetically detd. reversibly drug-tolerant subpopulation of cancer cells that share some properties with cancer stem cells.  Here, we define a requirement for the previously established cancer stem cell marker ALDH (aldehyde dehydrogenase) in the maintenance of this drug-tolerant subpopulation.  We find that ALDH protects the drug-tolerant subpopulation from the potentially toxic effects of elevated levels of reactive oxygen species (ROS) in these cells, and pharmacol. disruption of ALDH activity leads to accumulation of ROS to toxic levels, consequent DNA damage, and apoptosis specifically within the drug-tolerant subpopulation.  Combining ALDH inhibition with other kinase-directed treatments delayed treatment relapse in vitro and in vivo, revealing a novel combination treatment strategy for cancers that might otherwise rapidly relapse following single-agent therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLJmyYM8YDRrVg90H21EOLACvtfcHk0lim6D4rDj7RgQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKlurnE&md5=d9180579641124fa4c565d8ff0a7cf46</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-3456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-3456%26sid%3Dliteratum%253Aachs%26aulast%3DRaha%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DT.%2BR.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DEvangelista%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DThe%2520Cancer%2520Stem%2520Cell%2520Marker%2520Aldehyde%2520Dehydrogenase%2520is%2520Required%2520to%2520Maintain%2520a%2520Drug-Tolerant%2520Tumor%2520Cell%2520Subpopulation%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26issue%3D13%26spage%3D3579%26epage%3D3590%26doi%3D10.1158%2F0008-5472.CAN-13-3456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chou, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talalay, P.</span></span> <span> </span><span class="NLM_article-title">Quantitative Analysis of Dose-Effect Relationships: the Combined Effects of Multiple Drugs or Enzyme Inhibitors</span>. <i>Adv. Enzyme Regul.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">27</span>â <span class="NLM_lpage">55</span>, <span class="refDoi">Â DOI: 10.1016/0065-2571(84)90007-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2F0065-2571%2884%2990007-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=6382953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADyaL2cXktlaksLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1984&pages=27-55&author=T.+C.+Chouauthor=P.+Talalay&title=Quantitative+Analysis+of+Dose-Effect+Relationships%3A+the+Combined+Effects+of+Multiple+Drugs+or+Enzyme+Inhibitors&doi=10.1016%2F0065-2571%2884%2990007-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative analysis of dose-effect relationships:  the combined effects of multiple drugs or enzyme inhibitors</span></div><div class="casAuthors">Chou, Ting Chao; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Enzyme Regulation</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27-55</span>CODEN:
                <span class="NLM_cas:coden">AEZRA2</span>;
        ISSN:<span class="NLM_cas:issn">0065-2571</span>.
    </div><div class="casAbstract">A simplified exptl. design method for the anal. of multiple drug effects and for detg. summation, synergism, and antagonism is described.  The method is not limited by: whether the dose-effect relations are hyperbolic or sigmoidal, whether the effects are mutually exclusive or nonexclusive, whether the ligand interactions are competitive, noncompetitive, or uncompetitive, or whether the drugs are agonists or antagonists.  The method is independent of the no. of drugs involved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX4wNupCqgpbVg90H21EOLACvtfcHk0lim6D4rDj7RgQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXktlaksLk%253D&md5=055a2711b52dde49389c3f526352333d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2F0065-2571%2884%2990007-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0065-2571%252884%252990007-4%26sid%3Dliteratum%253Aachs%26aulast%3DChou%26aufirst%3DT.%2BC.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DQuantitative%2520Analysis%2520of%2520Dose-Effect%2520Relationships%253A%2520the%2520Combined%2520Effects%2520of%2520Multiple%2520Drugs%2520or%2520Enzyme%2520Inhibitors%26jtitle%3DAdv.%2520Enzyme%2520Regul.%26date%3D1984%26volume%3D22%26spage%3D27%26epage%3D55%26doi%3D10.1016%2F0065-2571%2884%2990007-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takayama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckanovich, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span> <span> </span><span class="NLM_article-title">Formation of Stable Small Cell Number Three-Dimensional Ovarian Cancer Spheroids Using Hanging Drop Arrays for Preclinical Drug Sensitivity Assays</span>. <i>Gynecol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>138</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">181</span>â <span class="NLM_lpage">189</span>, <span class="refDoi">Â DOI: 10.1016/j.ygyno.2015.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.ygyno.2015.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=25913133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2MXot1WnsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2015&pages=181-189&issue=1&author=S.+Raghavanauthor=M.+R.+Wardauthor=K.+R.+Rowleyauthor=R.+M.+Woldauthor=S.+Takayamaauthor=R.+J.+Buckanovichauthor=G.+Mehta&title=Formation+of+Stable+Small+Cell+Number+Three-Dimensional+Ovarian+Cancer+Spheroids+Using+Hanging+Drop+Arrays+for+Preclinical+Drug+Sensitivity+Assays&doi=10.1016%2Fj.ygyno.2015.04.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of stable small cell number three-dimensional ovarian cancer spheroids using hanging drop arrays for preclinical drug sensitivity assays</span></div><div class="casAuthors">Raghavan, Shreya; Ward, Maria R.; Rowley, Katelyn R.; Wold, Rachel M.; Takayama, Shuichi; Buckanovich, Ronald J.; Mehta, Geeta</div><div class="citationInfo"><span class="NLM_cas:title">Gynecologic Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">181-189</span>CODEN:
                <span class="NLM_cas:coden">GYNOA3</span>;
        ISSN:<span class="NLM_cas:issn">0090-8258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Ovarian cancer grows and metastasizes from multicellular spheroidal aggregates within the ascites fluid.  Multicellular tumor spheroids are therefore physiol. significant 3D in vitro models for ovarian cancer research.  Conventional hanging drop cultures require high starting cell nos., and are tedious for long-term maintenance.  In this study, we generate stable, uniform multicellular spheroids using very small no. of ovarian cancer cells in a novel 384 well hanging drop array platform.  We used novel tumor spheroid platform and two ovarian cancer cell lines (A2780 and OVCAR3) to demonstrate the stable incorporation of as few as 10 cells into a single spheroid.  Spheroids had uniform geometry, with projected areas (42.60 Ã 103 Î¼m-475.22 Ã 103 Î¼m2 for A2780 spheroids and 37.24 Ã 103 Î¼m2-281.01 Ã 103 Î¼m2 for OVCAR3 spheroids) that varied as a function of the initial cell seeding d.  Phalloidin and nuclear stains indicated cells formed tightly packed spheroids with demarcated boundaries and cell-cell interaction within spheroids.  Cells within spheroids demonstrated over 85% viability. 3D tumor spheroids demonstrated greater resistance (70-80% viability) to cisplatin chemotherapy compared to 2D cultures (30-50% viability).  Ovarian cancer spheroids can be generated from limited cell nos. in high throughput 384 well plates with high viability.  Spheroids demonstrate therapeutic resistance relative to cells in traditional 2D culture.  Stable incorporation of low cell nos. is advantageous when translating this research to rare patient-derived cells.  This system can be used to understand ovarian cancer spheroid biol., as well as carry out preclin. drug sensitivity assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZZNJLQFmx2LVg90H21EOLACvtfcHk0lim6D4rDj7RgQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXot1WnsrY%253D&md5=0e327f491b2234aa686994b785a05e88</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.ygyno.2015.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ygyno.2015.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DRaghavan%26aufirst%3DS.%26aulast%3DWard%26aufirst%3DM.%2BR.%26aulast%3DRowley%26aufirst%3DK.%2BR.%26aulast%3DWold%26aufirst%3DR.%2BM.%26aulast%3DTakayama%26aufirst%3DS.%26aulast%3DBuckanovich%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26atitle%3DFormation%2520of%2520Stable%2520Small%2520Cell%2520Number%2520Three-Dimensional%2520Ovarian%2520Cancer%2520Spheroids%2520Using%2520Hanging%2520Drop%2520Arrays%2520for%2520Preclinical%2520Drug%2520Sensitivity%2520Assays%26jtitle%3DGynecol.%2520Oncol.%26date%3D2015%26volume%3D138%26issue%3D1%26spage%3D181%26epage%3D189%26doi%3D10.1016%2Fj.ygyno.2015.04.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shield, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackland, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, G. E.</span></span> <span> </span><span class="NLM_article-title">Multicellular Spheroids in Ovarian Cancer Metastases: Biology and Pathology</span>. <i>Gynecol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">143</span>â <span class="NLM_lpage">148</span>, <span class="refDoi">Â DOI: 10.1016/j.ygyno.2008.11.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.ygyno.2008.11.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=19135710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A280%3ADC%252BD1M3is12itg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=143-148&issue=1&author=K.+Shieldauthor=M.+L.+Acklandauthor=N.+Ahmedauthor=G.+E.+Rice&title=Multicellular+Spheroids+in+Ovarian+Cancer+Metastases%3A+Biology+and+Pathology&doi=10.1016%2Fj.ygyno.2008.11.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Multicellular spheroids in ovarian cancer metastases: Biology and pathology</span></div><div class="casAuthors">Shield Kristy; Ackland M Leigh; Ahmed Nuzhat; Rice Gregory E</div><div class="citationInfo"><span class="NLM_cas:title">Gynecologic oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">143-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epithelial ovarian cancer (EOC) has a relatively high mortality rate ( approximately 55%).  One of the presiding causes is that the current chemotherapeutic regimes are unable to achieve sustained remission, despite frequently producing a positive response at first treatment.  One of the reasons that EOC is difficult to treat is that the mechanism of dissemination is unusual.  EOC dissemination characteristically involves local invasion of pelvic and abdominal organs.  Unlike many epithelial cancers, initial dissemination rarely requires the vasculature, although the vasculature is often implicated in the advanced stages of disease.  Recently, it has become apparent that aggregates of malignant cells (spheroids) contained within malignant ascites represent a significant impediment to efficacious treatment of late stage EOC.  In vivo, spheroids are present in the malignant ascites of EOC patients, while in vitro cultured spheroids are capable of tumorgenesis in vivo and display a reduced response to chemotherapeutic drugs when compared to monolayers.  A major problem associated with the current generation of chemotherapy agents is that they do not address the anchorage- and vascular-independent growth conditions associated with a 3-dimensional structure that has formed and/or grown in suspension.  Thus, spheroid formation may represent a key component of platinum/taxane-sensitive recurrence.  If this is correct, a better understanding of spheroid biology may contribute to the identification of new treatment opportunities for the sustained treatment of metastatic EOC.  This review article outlines the key biological features of spheroids, specifically discussing their role in EOC dissemination and chemo-response as well as providing insights into spheroid functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTd5-MIaWwb6Xqj_yg7twvpfW6udTcc2eb2FGY2r1W5grntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3is12itg%253D%253D&md5=e12bb83edf45c5406122f11c7764266d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.ygyno.2008.11.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ygyno.2008.11.032%26sid%3Dliteratum%253Aachs%26aulast%3DShield%26aufirst%3DK.%26aulast%3DAckland%26aufirst%3DM.%2BL.%26aulast%3DAhmed%26aufirst%3DN.%26aulast%3DRice%26aufirst%3DG.%2BE.%26atitle%3DMulticellular%2520Spheroids%2520in%2520Ovarian%2520Cancer%2520Metastases%253A%2520Biology%2520and%2520Pathology%26jtitle%3DGynecol.%2520Oncol.%26date%3D2009%26volume%3D113%26issue%3D1%26spage%3D143%26epage%3D148%26doi%3D10.1016%2Fj.ygyno.2008.11.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bregenzer, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleck, E. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckanovich, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span> <span> </span><span class="NLM_article-title">Personalized Medicine-Based Approach to Model Patterns of Chemoresistance and Tumor Recurrence Using Ovarian Cancer Stem Cell Spheroids</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">6934</span>â <span class="NLM_lpage">6945</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-17-0133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1158%2F1078-0432.CCR-17-0133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=28814433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnurbN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=6934-6945&issue=22&author=S.+Raghavanauthor=P.+Mehtaauthor=M.+R.+Wardauthor=M.+E.+Bregenzerauthor=E.+M.+A.+Fleckauthor=L.+Tanauthor=K.+McLeanauthor=R.+J.+Buckanovichauthor=G.+Mehta&title=Personalized+Medicine-Based+Approach+to+Model+Patterns+of+Chemoresistance+and+Tumor+Recurrence+Using+Ovarian+Cancer+Stem+Cell+Spheroids&doi=10.1158%2F1078-0432.CCR-17-0133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Personalized Medicine-Based Approach to Model Patterns of Chemoresistance and Tumor Recurrence Using Ovarian Cancer Stem Cell Spheroids</span></div><div class="casAuthors">Raghavan, Shreya; Mehta, Pooja; Ward, Maria R.; Bregenzer, Michael E.; Fleck, Elyse M. A.; Tan, Lijun; McLean, Karen; Buckanovich, Ronald J.; Mehta, Geeta</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6934-6945</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Chemoresistant ovarian cancers grow in suspension within the ascites fluid.  To screen the effect of chemotherapeutics and biologics on resistant ovarian cancers with a personalized basis, we developed a 3D hanging drop spheroid platform.  Exptl. Design: We initiated spheroids with primary aldehyde dehydrogenase-pos. (ALDH+) CD133+ ovarian cancer stem cells (OvCSC) from different patient samples and demonstrated that stem cell progeny from harvested spheroids was similar to the primary tumor.  OvCSC spheroids were utilized to initiate tumors in immunodeficient mice.  Drug responses to cisplatin and ALDH-targeting compd. or JAK2 inhibitor detd. whether the OvCSC population within the spheroids could be targeted.  Cells that escaped therapy were isolated and used to initiate new spheroids and model tumor reemergence in a personalized manner.  Results: OvCSC spheroids from different patients exhibited varying and personalized responses to chemotherapeutics.  Xenografts were established from OvCSC spheroids, even with a single spheroid.  Distinct responses to therapy were obsd. in distinct primary tumor xenografts similar to those obsd. in spheroids.  Spheroids resistant to cisplatin/ALDH inhibitor therapy had persistent, albeit lower ALDH expression and complete loss of CD133 expression, whereas those resistant to cisplatin/JAK2 inhibitor therapy were enriched for ALDH+ cells.  Conclusions: Our 3D hanging drop suspension platform can be used to propagate primary OvCSCs that represent individual patient tumors effectively by differentiating in vitro and initiating tumors in mice.  Therefore, our platform can be used to study cancer stem cell biol. and model tumor reemergence to identify new targeted therapeutics from an effective personalized medicine standpoint.  Clin Cancer Res; 23(22); 6934-45. Â©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrym4L94HYJs7Vg90H21EOLACvtfcHk0lgJsAXDBeRbvQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnurbN&md5=b8b2a038d780b726ff785287a0ea4489</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-17-0133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-17-0133%26sid%3Dliteratum%253Aachs%26aulast%3DRaghavan%26aufirst%3DS.%26aulast%3DMehta%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DM.%2BR.%26aulast%3DBregenzer%26aufirst%3DM.%2BE.%26aulast%3DFleck%26aufirst%3DE.%2BM.%2BA.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DMcLean%26aufirst%3DK.%26aulast%3DBuckanovich%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26atitle%3DPersonalized%2520Medicine-Based%2520Approach%2520to%2520Model%2520Patterns%2520of%2520Chemoresistance%2520and%2520Tumor%2520Recurrence%2520Using%2520Ovarian%2520Cancer%2520Stem%2520Cell%2520Spheroids%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26issue%3D22%26spage%3D6934%26epage%3D6945%26doi%3D10.1158%2F1078-0432.CCR-17-0133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lgJsAXDBeRbvQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Collaborative
Computational Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4 Suite: Programs for Protein Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">Pt 5</span>),  <span class="NLM_fpage">760</span>â <span class="NLM_lpage">763</span>, <span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&issue=Pt+5&author=Collaborative%0AComputational+Project%2C+Number+4&title=The+CCP4+Suite%3A+Programs+for+Protein+Crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520Suite%253A%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26issue%3DPt%25205%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title"><i>Coot</i>: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2126</span>â <span class="NLM_lpage">2132</span>, <span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&issue=12&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgJsAXDBeRbvQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26issue%3D12%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">Optimal Description of a Protein Structure in Terms of Multiple Groups Undergoing TLS Motion</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">439</span>â <span class="NLM_lpage">450</span>, <span class="refDoi">Â DOI: 10.1107/S0907444906005270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0907444906005270" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=16552146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=439-450&issue=4&author=J.+Painterauthor=E.+A.+Merritt&title=Optimal+Description+of+a+Protein+Structure+in+Terms+of+Multiple+Groups+Undergoing+TLS+Motion&doi=10.1107%2FS0907444906005270"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-450</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A single protein crystal structure contains information about dynamic properties of the protein as well as providing a static view of one three-dimensional conformation.  This addnl. information is to be found in the distribution of obsd. electron d. about the mean position of each atom.  It is general practice to account for this by refining a sep. at. displacement parameter (ADP) for each at. center.  However, these same displacements are often described well by simpler models based on TLS (translation/libration/screw) rigid-body motion of large groups of atoms, for example interdomain hinge motion.  A procedure, TLSMD, has been developed that analyzes the distribution of ADPs in a previously refined protein crystal structure in order to generate optimal multi-group TLS descriptions of the constituent protein chains.  TLSMD is applicable to crystal structures at any resoln.  The models generated by TLSMD anal. can significantly improve the std. crystallog. residuals R and Rfree and can reveal intrinsic dynamic properties of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIR_NcneJBNbVg90H21EOLACvtfcHk0lgv1_KbDXQ0DQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D&md5=6289483d19fb6dd64fe9b032e9c45005</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444906005270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906005270%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DOptimal%2520Description%2520of%2520a%2520Protein%2520Structure%2520in%2520Terms%2520of%2520Multiple%2520Groups%2520Undergoing%2520TLS%2520Motion%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26issue%3D4%26spage%3D439%26epage%3D450%26doi%3D10.1107%2FS0907444906005270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">A Molecular Viewer for the Analysis of TLS Rigid-Body Motion in Macromolecules</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">465</span>â <span class="NLM_lpage">471</span>, <span class="refDoi">Â DOI: 10.1107/S0907444905001897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0907444905001897" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=465-471&issue=4&author=J.+Painterauthor=E.+A.+Merritt&title=A+Molecular+Viewer+for+the+Analysis+of+TLS+Rigid-Body+Motion+in+Macromolecules&doi=10.1107%2FS0907444905001897"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444905001897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905001897%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DA%2520Molecular%2520Viewer%2520for%2520the%2520Analysis%2520of%2520TLS%2520Rigid-Body%2520Motion%2520in%2520Macromolecules%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26issue%3D4%26spage%3D465%26epage%3D471%26doi%3D10.1107%2FS0907444905001897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammen, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallenga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, H.</span></span> <span> </span><span class="NLM_article-title">Multiple Conformations of NAD and NADH when Bound to Human Cytosolic and Mitochondrial Aldehyde Dehydrogenase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">7156</span>â <span class="NLM_lpage">7168</span>, <span class="refDoi">Â DOI: 10.1021/bi012197t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi012197t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=7156-7168&issue=22&author=P.+K.+Hammenauthor=A.+Allali-Hassaniauthor=K.+Hallengaauthor=T.+D.+Hurleyauthor=H.+Weiner&title=Multiple+Conformations+of+NAD+and+NADH+when+Bound+to+Human+Cytosolic+and+Mitochondrial+Aldehyde+Dehydrogenase&doi=10.1021%2Fbi012197t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fbi012197t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi012197t%26sid%3Dliteratum%253Aachs%26aulast%3DHammen%26aufirst%3DP.%2BK.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHallenga%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DMultiple%2520Conformations%2520of%2520NAD%2520and%2520NADH%2520when%2520Bound%2520to%2520Human%2520Cytosolic%2520and%2520Mitochondrial%2520Aldehyde%2520Dehydrogenase%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26issue%3D22%26spage%3D7156%26epage%3D7168%26doi%3D10.1021%2Fbi012197t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimble-Hill, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanova, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meroueh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Regulators of Aldehyde Dehydrogenase Isoenzymes</span>. <i>Chem.-Biol. Interact.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>191</i></span> (<span class="NLM_issue">1â3</span>),  <span class="NLM_fpage">153</span>â <span class="NLM_lpage">158</span>, <span class="refDoi">Â DOI: 10.1016/j.cbi.2011.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2Fj.cbi.2011.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21349255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFajtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2011&pages=153-158&issue=1%E2%80%933&author=B.+Parajuliauthor=A.+C.+Kimble-Hillauthor=M.+Khannaauthor=Y.+Ivanovaauthor=S.+Merouehauthor=T.+D.+Hurley&title=Discovery+of+Novel+Regulators+of+Aldehyde+Dehydrogenase+Isoenzymes&doi=10.1016%2Fj.cbi.2011.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel regulators of aldehyde dehydrogenase isoenzymes</span></div><div class="casAuthors">Parajuli, Bibek; Kimble-Hill, Ann C.; Khanna, May; Ivanova, Yvelina; Meroueh, Samy; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">153-158</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Over the past three years we have been involved in high-throughput screening in an effort to discover novel small mol. modulators of aldehyde dehydrogenase (ALDH) activity.  In particular, we have been interested in both the activation and inhibition of the three commonly studied isoenzymes, ALDH1A1, ALDH2 and ALDH3A1, as their distinct, yet overlapping substrate specificities, present a particularly difficult challenge for inhibitor discovery and design.  Activation of ALDH2 has been shown to benefit cardiovascular outcome following periods of ischemia and renewed interest in specific inhibition of ALDH2 has application for alc. aversion therapy, and more recently, in cocaine addiction.  In contrast, inhibition of either ALDH1A1 or ALDH3A1 has application in cancer treatments where the isoenzymes are commonly over-expressed and serve as markers for cancer stem cells.  We are taking two distinct approaches for these screens: in vitro enzyme activity screens using chem. libraries and virtual computational screens using the structures of the target enzymes as filters for identifying potential inhibitors, followed by in vitro testing of their ability to inhibit their intended targets.  We have identified selective inhibitors of each of these three isoenzymes with inhibition consts. in the high nanomolar to low micromolar range from these screening procedures.  Together, these inhibitors provide proof for concept that selective inhibition of these broad specificity general detoxication enzymes through small mol. discovery and design is possible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcKIxSAscaUbVg90H21EOLACvtfcHk0lgv1_KbDXQ0DQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFajtbc%253D&md5=f8c284f0a5ad00a61f68cd49e5d4bff1</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2011.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2011.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DKimble-Hill%26aufirst%3DA.%2BC.%26aulast%3DKhanna%26aufirst%3DM.%26aulast%3DIvanova%26aufirst%3DY.%26aulast%3DMeroueh%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDiscovery%2520of%2520Novel%2520Regulators%2520of%2520Aldehyde%2520Dehydrogenase%2520Isoenzymes%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D2011%26volume%3D191%26issue%3D1%25E2%2580%25933%26spage%3D153%26epage%3D158%26doi%3D10.1016%2Fj.cbi.2011.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Development of Selective Inhibitors for Human Aldehyde Dehydrogenase 3A1 (ALDH3A1) for the Enhancement of Cyclophosphamide Cytotoxicity</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">701</span>â <span class="NLM_lpage">712</span>, <span class="refDoi">Â DOI: 10.1002/cbic.201300625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1002%2Fcbic.201300625" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=701-712&issue=5&author=B.+Parajuliauthor=T.+M.+Georgiadisauthor=M.+L.+Fishelauthor=T.+D.+Hurley&title=Development+of+Selective+Inhibitors+for+Human+Aldehyde+Dehydrogenase+3A1+%28ALDH3A1%29+for+the+Enhancement+of+Cyclophosphamide+Cytotoxicity&doi=10.1002%2Fcbic.201300625"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201300625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201300625%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DGeorgiadis%26aufirst%3DT.%2BM.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520Selective%2520Inhibitors%2520for%2520Human%2520Aldehyde%2520Dehydrogenase%25203A1%2520%2528ALDH3A1%2529%2520for%2520the%2520Enhancement%2520of%2520Cyclophosphamide%2520Cytotoxicity%26jtitle%3DChemBioChem%26date%3D2014%26volume%3D15%26issue%3D5%26spage%3D701%26epage%3D712%26doi%3D10.1002%2Fcbic.201300625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchman, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the Aldehyde Dehydrogenase 1/2 Family by Psoralen and Coumarin Derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2439</span>â <span class="NLM_lpage">2455</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2439-2455&issue=6&author=C.+D.+Buchmanauthor=T.+D.+Hurley&title=Inhibition+of+the+Aldehyde+Dehydrogenase+1%2F2+Family+by+Psoralen+and+Coumarin+Derivatives&doi=10.1021%2Facs.jmedchem.6b01825"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01825%26sid%3Dliteratum%253Aachs%26aulast%3DBuchman%26aufirst%3DC.%2BD.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DInhibition%2520of%2520the%2520Aldehyde%2520Dehydrogenase%25201%252F2%2520Family%2520by%2520Psoralen%2520and%2520Coumarin%2520Derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D6%26spage%3D2439%26epage%3D2455%26doi%3D10.1021%2Facs.jmedchem.6b01825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horst, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span> <span> </span><span class="NLM_article-title">Comparative Analysis of Tumor Spheroid Generation Techniques for Differential In Vitro Drug Toxicity</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">16948</span>â <span class="NLM_lpage">16961</span>, <span class="refDoi">Â DOI: 10.18632/oncotarget.7659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.18632%2Foncotarget.7659" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=16948-16961&issue=13&author=S.+Raghavanauthor=P.+Mehtaauthor=E.+N.+Horstauthor=M.+R.+Wardauthor=K.+R.+Rowleyauthor=G.+Mehta&title=Comparative+Analysis+of+Tumor+Spheroid+Generation+Techniques+for+Differential+In+Vitro+Drug+Toxicity&doi=10.18632%2Foncotarget.7659"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7659%26sid%3Dliteratum%253Aachs%26aulast%3DRaghavan%26aufirst%3DS.%26aulast%3DMehta%26aufirst%3DP.%26aulast%3DHorst%26aufirst%3DE.%2BN.%26aulast%3DWard%26aufirst%3DM.%2BR.%26aulast%3DRowley%26aufirst%3DK.%2BR.%26aulast%3DMehta%26aufirst%3DG.%26atitle%3DComparative%2520Analysis%2520of%2520Tumor%2520Spheroid%2520Generation%2520Techniques%2520for%2520Differential%2520In%2520Vitro%2520Drug%2520Toxicity%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26issue%3D13%26spage%3D16948%26epage%3D16961%26doi%3D10.18632%2Foncotarget.7659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biasini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bienert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterhouse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Studer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo Cassarino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwede, T.</span></span> <span> </span><span class="NLM_article-title">SWISS-MODEL: Modelling Protein Tertiary and Quaternary Structure Using Evolutionary Information</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W252</span>â <span class="NLM_lpage">W258</span>, <span class="refDoi">Â DOI: 10.1093/nar/gku340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1093%2Fnar%2Fgku340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=24782522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W252-W258&issue=W1&author=M.+Biasiniauthor=S.+Bienertauthor=A.+Waterhouseauthor=K.+Arnoldauthor=G.+Studerauthor=T.+Schmidtauthor=F.+Kieferauthor=T.+Gallo+Cassarinoauthor=M.+Bertoniauthor=L.+Bordoliauthor=T.+Schwede&title=SWISS-MODEL%3A+Modelling+Protein+Tertiary+and+Quaternary+Structure+Using+Evolutionary+Information&doi=10.1093%2Fnar%2Fgku340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information</span></div><div class="casAuthors">Biasini, Marco; Bienert, Stefan; Waterhouse, Andrew; Arnold, Konstantin; Studer, Gabriel; Schmidt, Tobias; Kiefer, Florian; Cassarino, Tiziano Gallo; Bertoni, Martino; Bordoli, Lorenza; Schwede, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W252-W258</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein structure homol. modeling has become a routine technique to generate 3D models for proteins when exptl. structures are not available.  Fully automated servers such as SWISS-MODEL with user-friendly web interfaces generate reliable models without the need for complex software packages or downloading large databases.  Here, we describe the latest version of the SWISS-MODEL expert system for protein structure modeling.  The SWISS-MODEL template library provides annotation of quaternary structure and essential ligands and co-factors to allow for building of complete structural models, including their oligomeric structure.  The improved SWISS-MODEL pipeline makes extensive use of model quality estn. for selection of the most suitable templates and provides ests. of the expected accuracy of the resulting models.  The accuracy of the models generated by SWISS-MODEL is continuously evaluated by the CAMEO system.  The new web site allows users to interactively search for templates, cluster them by sequence similarity, structurally compare alternative templates and select the ones to be used for model building.  In cases where multiple alternative template structures are available for a protein of interest, a user-guided template selection step allows building models in different functional states.  SWISS-MODEL is available at http://swissmodel.expasy.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevqhyplQxI7Vg90H21EOLACvtfcHk0lgU_OWZLZa9aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs73I&md5=51509ba8353b06f286d954ebe7e6673a</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku340%26sid%3Dliteratum%253Aachs%26aulast%3DBiasini%26aufirst%3DM.%26aulast%3DBienert%26aufirst%3DS.%26aulast%3DWaterhouse%26aufirst%3DA.%26aulast%3DArnold%26aufirst%3DK.%26aulast%3DStuder%26aufirst%3DG.%26aulast%3DSchmidt%26aufirst%3DT.%26aulast%3DKiefer%26aufirst%3DF.%26aulast%3DGallo%2BCassarino%26aufirst%3DT.%26aulast%3DBertoni%26aufirst%3DM.%26aulast%3DBordoli%26aufirst%3DL.%26aulast%3DSchwede%26aufirst%3DT.%26atitle%3DSWISS-MODEL%253A%2520Modelling%2520Protein%2520Tertiary%2520and%2520Quaternary%2520Structure%2520Using%2520Evolutionary%2520Information%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DW1%26spage%3DW252%26epage%3DW258%26doi%3D10.1093%2Fnar%2Fgku340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwede, T.</span></span> <span> </span><span class="NLM_article-title">The SWISS-MODEL Workspace: a Web-Based Environment for Protein Structure Homology Modelling</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">195</span>â <span class="NLM_lpage">201</span>, <span class="refDoi">Â DOI: 10.1093/bioinformatics/bti770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1093%2Fbioinformatics%2Fbti770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=16301204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BD28XovVCltw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=195-201&issue=2&author=K.+Arnoldauthor=L.+Bordoliauthor=J.+Koppauthor=T.+Schwede&title=The+SWISS-MODEL+Workspace%3A+a+Web-Based+Environment+for+Protein+Structure+Homology+Modelling&doi=10.1093%2Fbioinformatics%2Fbti770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling</span></div><div class="casAuthors">Arnold, Konstantin; Bordoli, Lorenza; Kopp, Juergen; Schwede, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">195-201</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Homol. models of proteins are of great interest for planning and analyzing biol. expts. when no exptl. three-dimensional structures are available.  Building homol. models requires specialized programs and up-to-date sequence and structural databases.  Integrating all required tools, programs and databases into a single web-based workspace facilitates access to homol. modeling from a computer with web connection without the need of downloading and installing large program packages and databases.  Results: SWISS-MODEL workspace is a web-based integrated service dedicated to protein structure homol. modeling.  It assists and guides the user in building protein homol. models at different levels of complexity.  A personal working environment is provided for each user where several modeling projects can be carried out in parallel.  Protein sequence and structure databases necessary for modeling are accessible from the workspace and are updated in regular intervals.  Tools for template selection, model building and structure quality evaluation can be invoked from within the workspace.  Workflow and usage of the workspace are illustrated by modeling human Cyclin A1 and human Transmembrane Protease 3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDwXLAG5Kz27Vg90H21EOLACvtfcHk0lgU_OWZLZa9aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XovVCltw%253D%253D&md5=66fa733d2ad6d2abf7d3d87ae837ebce</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbti770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbti770%26sid%3Dliteratum%253Aachs%26aulast%3DArnold%26aufirst%3DK.%26aulast%3DBordoli%26aufirst%3DL.%26aulast%3DKopp%26aufirst%3DJ.%26aulast%3DSchwede%26aufirst%3DT.%26atitle%3DThe%2520SWISS-MODEL%2520Workspace%253A%2520a%2520Web-Based%2520Environment%2520for%2520Protein%2520Structure%2520Homology%2520Modelling%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26issue%3D2%26spage%3D195%26epage%3D201%26doi%3D10.1093%2Fbioinformatics%2Fbti770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benkert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biasini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwede, T.</span></span> <span> </span><span class="NLM_article-title">Toward the Estimation of the Absolute Quality of Individual Protein Structure Models</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">343</span>â <span class="NLM_lpage">350</span>, <span class="refDoi">Â DOI: 10.1093/bioinformatics/btq662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1093%2Fbioinformatics%2Fbtq662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=21134891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Gisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=343-350&issue=3&author=P.+Benkertauthor=M.+Biasiniauthor=T.+Schwede&title=Toward+the+Estimation+of+the+Absolute+Quality+of+Individual+Protein+Structure+Models&doi=10.1093%2Fbioinformatics%2Fbtq662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the estimation of the absolute quality of individual protein structure models</span></div><div class="casAuthors">Benkert, Pascal; Biasini, Marco; Schwede, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">343-350</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Quality assessment of protein structures is an important part of exptl. structure validation and plays a crucial role in protein structure prediction, where the predicted models may contain substantial errors.  Most current scoring functions are primarily designed to rank alternative models of the same sequence supporting model selection, whereas the prediction of the abs. quality of an individual protein model has received little attention in the field.  However, reliable abs. quality ests. are crucial to assess the suitability of a model for specific biomedical applications.  In this work, we present a new abs. measure for the quality of protein models, which provides an est. of the degree of nativeness' of the structural features obsd. in a model and describes the likelihood that a given model is of comparable quality to exptl. structures.  Model quality ests. based on the QMEAN scoring function were normalized with respect to the no. of interactions.  The resulting scoring function is independent of the size of the protein and may therefore be used to assess both monomers and entire oligomeric assemblies.  Model quality scores for individual models are then expressed as Z-scores' in comparison to scores obtained for high-resoln. crystal structures.  We demonstrate the ability of the newly introduced QMEAN Z-score to detect exptl. solved protein structures contg. significant errors, as well as to evaluate theor. protein models.  In a comprehensive QMEAN Z-score anal. of all exptl. structures in the PDB, membrane proteins accumulate on one side of the score spectrum and thermostable proteins on the other.  Proteins from the thermophilic organism Thermatoga maritima received significantly higher QMEAN Z-scores in a pairwise comparison with their homologous mesophilic counterparts, underlining the significance of the QMEAN Z-score as an est. of protein stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo81whOYLkpN7Vg90H21EOLACvtfcHk0lgfwfi-i9hFmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Gisb8%253D&md5=eb1b789a978f8db7beff4dbcd03296ee</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq662%26sid%3Dliteratum%253Aachs%26aulast%3DBenkert%26aufirst%3DP.%26aulast%3DBiasini%26aufirst%3DM.%26aulast%3DSchwede%26aufirst%3DT.%26atitle%3DToward%2520the%2520Estimation%2520of%2520the%2520Absolute%2520Quality%2520of%2520Individual%2520Protein%2520Structure%2520Models%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26issue%3D3%26spage%3D343%26epage%3D350%26doi%3D10.1093%2Fbioinformatics%2Fbtq662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walther, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenet, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deutsch, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jostes, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goulding, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plachov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balling, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruss, P.</span></span> <span> </span><span class="NLM_article-title">Pax: a Murine Multigene Family of Paired Box-Containing Genes</span>. <i>Genomics</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">424</span>â <span class="NLM_lpage">434</span>, <span class="refDoi">Â DOI: 10.1016/0888-7543(91)90151-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1016%2F0888-7543%2891%2990151-4" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=424-434&issue=2&author=C.+Waltherauthor=J.+L.+Guenetauthor=D.+Simonauthor=U.+Deutschauthor=B.+Jostesauthor=M.+D.+Gouldingauthor=D.+Plachovauthor=R.+Ballingauthor=P.+Gruss&title=Pax%3A+a+Murine+Multigene+Family+of+Paired+Box-Containing+Genes&doi=10.1016%2F0888-7543%2891%2990151-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2F0888-7543%2891%2990151-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0888-7543%252891%252990151-4%26sid%3Dliteratum%253Aachs%26aulast%3DWalther%26aufirst%3DC.%26aulast%3DGuenet%26aufirst%3DJ.%2BL.%26aulast%3DSimon%26aufirst%3DD.%26aulast%3DDeutsch%26aufirst%3DU.%26aulast%3DJostes%26aufirst%3DB.%26aulast%3DGoulding%26aufirst%3DM.%2BD.%26aulast%3DPlachov%26aufirst%3DD.%26aulast%3DBalling%26aufirst%3DR.%26aulast%3DGruss%26aufirst%3DP.%26atitle%3DPax%253A%2520a%2520Murine%2520Multigene%2520Family%2520of%2520Paired%2520Box-Containing%2520Genes%26jtitle%3DGenomics%26date%3D1991%26volume%3D11%26issue%3D2%26spage%3D424%26epage%3D434%26doi%3D10.1016%2F0888-7543%2891%2990151-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">PROCHECK - a Program to Check the Stereochemical Quality of Protein Structures</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">283</span>â <span class="NLM_lpage">291</span>, <span class="refDoi">Â DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=PROCHECK+-+a+Program+to+Check+the+Stereochemical+Quality+of+Protein+Structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0lgfwfi-i9hFmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%2520-%2520a%2520Program%2520to%2520Check%2520the%2520Stereochemical%2520Quality%2520of%2520Protein%2520Structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGann, M.</span></span> <span> </span><span class="NLM_article-title">FRED and HYBRID Docking Performance on Standardized Datasets</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">897</span>â <span class="NLM_lpage">906</span>, <span class="refDoi">Â DOI: 10.1007/s10822-012-9584-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=10.1007%2Fs10822-012-9584-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=22669221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ens7bO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=897-906&issue=8&author=M.+McGann&title=FRED+and+HYBRID+Docking+Performance+on+Standardized+Datasets&doi=10.1007%2Fs10822-012-9584-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">FRED and HYBRID docking performance on standardized datasets</span></div><div class="casAuthors">McGann, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">897-906</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The docking performance of the FRED and HYBRID programs are evaluated on two standardized datasets from the Docking and Scoring Symposium of the ACS Spring 2011 national meeting.  The evaluation includes cognate docking and virtual screening performance.  FRED docks 70 % of the structures to within 2 Ã in the cognate docking test.  In the virtual screening test, FRED is found to have a mean AUC of 0.75.  The HYBRID program uses a modified version of FRED's algorithm that uses both ligand- and structure-based information to dock mols., which increases its mean AUC to 0.78.  HYBRID can also implicitly account for protein flexibility by making use of multiple crystal structures.  Using multiple crystal structures improves HYBRID's performance (mean AUC 0.80) with a negligible increase in docking time (â¼15 %).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDNF1kmgp-WLVg90H21EOLACvtfcHk0lgfwfi-i9hFmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ens7bO&md5=299f2e795ca922412f57df9434b52661</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9584-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9584-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcGann%26aufirst%3DM.%26atitle%3DFRED%2520and%2520HYBRID%2520Docking%2520Performance%2520on%2520Standardized%2520Datasets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26issue%3D8%26spage%3D897%26epage%3D906%26doi%3D10.1007%2Fs10822-012-9584-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">572</span>â <span class="NLM_lpage">584</span>, <span class="refDoi">Â DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&issue=4&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+Generation+with+OMEGA%3A+Algorithm+and+Validation+Using+High+Quality+Structures+from+the+Protein+Databank+and+Cambridge+Structural+Database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lgfwfi-i9hFmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520Generation%2520with%2520OMEGA%253A%2520Algorithm%2520and%2520Validation%2520Using%2520High%2520Quality%2520Structures%2520from%2520the%2520Protein%2520Databank%2520and%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D4%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kehler, J.</span>; <span class="NLM_string-name">Rasmussen, L. K.</span>; <span class="NLM_string-name">Jessing, M.</span></span> <span> </span><span class="NLM_article-title">Triazolopyrazinones as PDE1 Inhibitors</span>. Patent No. <span class="NLM_patent">WO2016055618 A1</span>. <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+Kehler&author=L.+K.+Rasmussen&author=M.+Jessing&title=Triazolopyrazinones+as+PDE1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKehler%26aufirst%3DJ.%26atitle%3DTriazolopyrazinones%2520as%2520PDE1%2520Inhibitors%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI','PDB','5TEI'); return false;">PDB: 5TEI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7','PDB','4WP7'); return false;">PDB: 4WP7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L','PDB','4X4L'); return false;">PDB: 4X4L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DUM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DUM','PDB','6DUM'); return false;">PDB: 6DUM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5H','PDB','6B5H'); return false;">PDB: 6B5H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WDV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WDV','PDB','5WDV'); return false;">PDB: 5WDV</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i74"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00930">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_31322"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00930">10.1021/acs.jmedchem.8b00930</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional information regarding the reversible inhibition assay, inhibition of the N121S mutant used for crystallography, crystallographic data collection and refinement, electron density maps, validation of ALDH1A2 and 1A3 homology models, additional docking studies, and additional cytotoxicity data, and ALDH expression for cell lines (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx">DOCX</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_002.csv">CSV</a>)</p></li><li><p class="inline">Coordinates for ALDH1A2 and ALDH1A3 homology models and docking studies (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_019.zip">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_001.docx">jm8b00930_si_001.docx (2.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_002.csv">jm8b00930_si_002.csv (3.03 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00930/suppl_file/jm8b00930_si_019.zip">jm8b00930_si_019.zip (2.38 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates of ALDH1A1 bound to compounds <b>7</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TEI">5TEI</a>), <b>13g</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WDV">5WDV</a>) have been deposited with the Protein Data Bank. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00930&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00930%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-19%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00930" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                15MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a158c9832d1c7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
