
<html lang="en"     class="pb-page"  data-request-id="a5b05f20-37c4-4f60-933b-57c2430aeaad"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01010;issue:issue:10.1021/jmcmar.2019.62.issue-24;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis" /></meta><meta name="dc.Creator" content="Fei  Jiang" /></meta><meta name="dc.Creator" content="Qinghua  Hu" /></meta><meta name="dc.Creator" content="Zhimin  Zhang" /></meta><meta name="dc.Creator" content="Hongmei  Li" /></meta><meta name="dc.Creator" content="Huili  Li" /></meta><meta name="dc.Creator" content="Dewei  Zhang" /></meta><meta name="dc.Creator" content="Hanwen  Li" /></meta><meta name="dc.Creator" content="Yu  Ma" /></meta><meta name="dc.Creator" content="Jingjing  Xu" /></meta><meta name="dc.Creator" content="Haifang  Chen" /></meta><meta name="dc.Creator" content="Yong  Cui" /></meta><meta name="dc.Creator" content="Yanle  Zhi" /></meta><meta name="dc.Creator" content="Yanmin  Zhang" /></meta><meta name="dc.Creator" content="Junyu  Xu" /></meta><meta name="dc.Creator" content="Jiapeng  Zhu" /></meta><meta name="dc.Creator" content="Tao  Lu" /></meta><meta name="dc.Creator" content="Yadong  Chen" /></meta><meta name="dc.Description" content="The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues. Each BET bromodomain protein contains two highly homo..." /></meta><meta name="Description" content="The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues. Each BET bromodomain protein contains two highly homo..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 30, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01010" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01010" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01010" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01010" /></link>
        
    
    

<title>Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01010" /></meta><meta property="og:title" content="Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0009.jpeg" /></meta><meta property="og:description" content="The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues. Each BET bromodomain protein contains two highly homologous domains: the first bromodomain (BD1) and the second bromodomain (BD2). Pan-BET bromodomain inhibition is a potential therapy for various cancers and immune-inflammatory diseases, but only few reported inhibitors show selectivity within the BET family. Herein, we identified a series of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de]quinolin-2(1H)-ones with good selectivity for BET BD1. Through structure-based optimization, highly active and selective compounds are ultimately obtained. The representative compounds are the first reported inhibitors with selectivity more than 100-fold for BRD4(1) over BRD4(2). Among them, we further show that 68 (LT052) mediates BRD4/NF-κB/NLRP3 signaling inflammatory pathways with comparable protein expression and significantly improves symptoms of gout arthritis in a rat model. Therefore, selective pharmacological modulation of individual bromodomains could represent a strategy for the treatment of acute gouty arthritis." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01010"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01010">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01010&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01010&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01010&amp;href=/doi/10.1021/acs.jmedchem.9b01010" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 11080-11107</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00918" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01126" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Benzo[<i>cd</i>]indol-2(1<i>H</i>)-ones and Pyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Fei Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fei Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div><div class="loa-info-affiliations-info">State Key Laboratory of Natural Medicines, China Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fei++Jiang">Fei Jiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qinghua Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qinghua Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qinghua++Hu">Qinghua Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhimin Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhimin Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div><div class="loa-info-affiliations-info">Key Laboratory of Neuropsychiatric Drug Research of Zhejiang Province, Institute of Materia Medica, Zhejiang Academy of Medical Sciences, Hangzhou 310013, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhimin++Zhang">Zhimin Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongmei Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongmei Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div><div class="loa-info-affiliations-info">State Key Laboratory of Natural Medicines, China Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongmei++Li">Hongmei Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huili Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huili Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huili++Li">Huili Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dewei Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dewei Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dewei++Zhang">Dewei Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hanwen Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hanwen Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hanwen++Li">Hanwen Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu Ma</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu Ma</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Ma">Yu Ma</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jingjing Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jingjing Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Medicine and Life Sciences, State Key Laboratory Cultivation Base for TCM Quality and Efficacy, Jiangsu Key Laboratory for Pharmacology and Safety Evaluation of Chinese Materia Medica, Nanjing University of Chinese Medicine, Nanjing 210023, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jingjing++Xu">Jingjing Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Haifang Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haifang Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haifang++Chen">Haifang Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yong Cui</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yong Cui</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Cui">Yong Cui</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanle Zhi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanle Zhi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanle++Zhi">Yanle Zhi</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6066-2231" title="Orcid link">http://orcid.org/0000-0001-6066-2231</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanmin Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanmin Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanmin++Zhang">Yanmin Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Junyu Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Junyu Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Junyu++Xu">Junyu Xu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4918-0693" title="Orcid link">http://orcid.org/0000-0002-4918-0693</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiapeng Zhu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiapeng Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Medicine and Life Sciences, State Key Laboratory Cultivation Base for TCM Quality and Efficacy, Jiangsu Key Laboratory for Pharmacology and Safety Evaluation of Chinese Materia Medica, Nanjing University of Chinese Medicine, Nanjing 210023, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#02786a77686b6372676c65426a6d766f636b6e2c616d6f"><span class="__cf_email__" data-cfemail="374d5f425d5e5647525950775f58435a565e5b1954585a">[email protected]</span></a>. Tel: +86-25-85811382 (J.Z.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiapeng++Zhu">Jiapeng Zhu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tao Lu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tao Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div><div class="loa-info-affiliations-info">State Key Laboratory of Natural Medicines, China Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, P. R. China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#d9b5acadb8b699baa9acf7bcbdacf7bab7"><span class="__cf_email__" data-cfemail="4f233a3b2e200f2c3f3a612a2b3a612c21">[email protected]</span></a>. Tel: +86-25-86185180. Fax: +86-25-86185179 (T.L.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tao++Lu">Tao Lu</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Yadong Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yadong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</div><div class="loa-info-affiliations-info">State Key Laboratory of Natural Medicines, China Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, P. R. China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#0178656269646f416271742f6465742f626f"><span class="__cf_email__" data-cfemail="83fae7e0ebe6edc3e0f3f6ade6e7f6ade0ed">[email protected]</span></a>. Tel: +86-25-86185153. Fax: +86-25-861851170 (Y.C.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yadong++Chen">Yadong Chen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2189-9356" title="Orcid link">http://orcid.org/0000-0002-2189-9356</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01010&amp;href=/doi/10.1021%2Facs.jmedchem.9b01010" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 11080–11107</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 30, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 June 2019</li><li><span class="item_label"><b>Published</b> online</span>30 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 December 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01010" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01010</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D11080%26pageCount%3D28%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFei%2BJiang%252C%2BQinghua%2BHu%252C%2BZhimin%2BZhang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D24%26contentID%3Dacs.jmedchem.9b01010%26title%3DDiscovery%2Bof%2BBenzo%255Bcd%255Dindol-2%25281H%2529-ones%2Band%2BPyrrolo%255B4%252C3%252C2-de%255Dquinolin-2%25281H%2529-ones%2Bas%2BBromodomain%2Band%2BExtra-Terminal%2BDomain%2B%2528BET%2529%2BInhibitors%2Bwith%2BSelectivity%2Bfor%2Bthe%2BFirst%2BBromodomain%2Bwith%2BPotential%2BHigh%2BEfficiency%2Bagainst%2BAcute%2BGouty%2BArthritis%26numPages%3D28%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D11107%26publicationDate%3DDecember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01010"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1554</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">14</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01010" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Fei&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Qinghua&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Zhimin&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Hongmei&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Huili&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Dewei&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Hanwen&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Jingjing&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Haifang&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Cui&quot;},{&quot;first_name&quot;:&quot;Yanle&quot;,&quot;last_name&quot;:&quot;Zhi&quot;},{&quot;first_name&quot;:&quot;Yanmin&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Junyu&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Jiapeng&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Tao&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Yadong&quot;,&quot;last_name&quot;:&quot;Chen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;11080-11107&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01010&quot;},&quot;abstract&quot;:&quot;The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues. Each BET bromodomain protein contains two highly homologous domains: the first bromodomain (BD1) and the second bromodomain (BD2). Pan-BET bromodomain inhibition is a potential therapy for various cancers and immune-inflammatory diseases, but only few reported inhibitors show selectivity within the BET family. Herein, we identified a series of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de]quinolin-2(1H)-ones with good selectivity for BET BD1. Through structure-based optimization, highly active and selective compounds are ultimately obtained. The representative compounds are the first reported inhibitors with selectivity more than 100-fold for BRD4(1) over BRD4(2). Among them, we further show that 68 (LT052) mediates BRD4/NF-κB/NLRP3 signaling inflammatory pathways with comparable protein expression and significantly improves symptoms of gout arthritis in a rat model.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01010&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01010" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01010&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01010" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01010&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01010" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01010&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01010&amp;href=/doi/10.1021/acs.jmedchem.9b01010" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01010" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01010" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01010%26sid%3Dliteratum%253Aachs%26pmid%3D31789032%26genre%3Darticle%26aulast%3DJiang%26date%3D2019%26atitle%3DDiscovery%2Bof%2BBenzo%255Bcd%255Dindol-2%25281H%2529-ones%2Band%2BPyrrolo%255B4%252C3%252C2-de%255Dquinolin-2%25281H%2529-ones%2Bas%2BBromodomain%2Band%2BExtra-Terminal%2BDomain%2B%2528BET%2529%2BInhibitors%2Bwith%2BSelectivity%2Bfor%2Bthe%2BFirst%2BBromodomain%2Bwith%2BPotential%2BHigh%2BEfficiency%2Bagainst%2BAcute%2BGouty%2BArthritis%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D24%26spage%3D11080%26epage%3D11107%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290758" title="Anatomy">Anatomy</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/jmcmar.2019.62.issue-24/20191226/jmcmar.2019.62.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues. Each BET bromodomain protein contains two highly homologous domains: the first bromodomain (BD1) and the second bromodomain (BD2). Pan-BET bromodomain inhibition is a potential therapy for various cancers and immune-inflammatory diseases, but only few reported inhibitors show selectivity within the BET family. Herein, we identified a series of benzo[<i>cd</i>]indol-2(1<i>H</i>)-ones and pyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-ones with good selectivity for BET BD1. Through structure-based optimization, highly active and selective compounds are ultimately obtained. The representative compounds are the first reported inhibitors with selectivity more than 100-fold for BRD4(1) over BRD4(2). Among them, we further show that <b>68</b> (LT052) mediates BRD4/NF-κB/NLRP3 signaling inflammatory pathways with comparable protein expression and significantly improves symptoms of gout arthritis in a rat model. Therefore, selective pharmacological modulation of individual bromodomains could represent a strategy for the treatment of acute gouty arthritis.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07615" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07615" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Gout is the most common type of inflammatory arthritis with increasing prevalence in recent years, which is characterized by the precipitation of monosodium urate (MSU) crystals in the joint.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Proinflammatory cytokine IL-1β is a critical manifestation in response to MSU crystal attack. IL-1β secretion is dependent on a thoroughly studied inflammasome, nucleotide-binding oligomerization domain-like receptor, pyrin domain containing 3 (NLRP3).<a onclick="showRef(event, 'ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5">(2−5)</a> The activation of NLRP3 involves a two-step process. The first signal triggers NF-κB-dependent pro-IL-1β and NLRP3 expression.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The second signal triggers the formation of inflammasome multimers by microbial molecules or dangerous signals (such as MSU) of different structures. NLRP3 induces self-cleavage and activation of caspase-1. Activated caspase-1 promotes the hydrolysis and maturation of IL-1β and IL-18 and the formation of pores on the cell membrane and triggers proinflammatory cell death, pyroptosis.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a></div><div class="NLM_p">In recent years, Bromodomain-containing proteins (BRDs) have been found to play an important role in the development and progression of inflammation and cancer as epigenetic targets. Bromodomain-containing proteins (BRDs) are epigenetic reader proteins of histone acetylation involved in chromatin remodeling and transcriptional regulation, which recruit transcriptional regulator complexes to chromatin and binding to acetylated lysine (KAc) residues on the <i>N</i>-terminal tails of histones.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> So far, 61 bromodomains have been identified in 46 human proteins, which are classified into eight families.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The second family of bromodomain and extra-terminal (BET) includes BRD2, BRD3, BRD4, and BRDT, each of which contains two bromodomains (BD1 and BD2). The BD2 domains in these four BET proteins display a high degree of structural similarity, as do the four BD1 domains.</div><div class="NLM_p">BET family members recruit different partners. BRD4 recruits positive transcriptional elongation factor complex (P-TEFb), which plays an essential role in the regulation of transcription by RNA polymerase II (RNA Pol II) in eukaryotes.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> BET proteins facilitate the transcription of tumor-growth-promoting and antiapoptotic genes, including c-Myc, and their dysfunction plays critical roles in a variety of human diseases.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> Small-molecule BET inhibitors have shown efficacy against a range of cancers, and several inhibitors have entered clinical trials in oncology.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> BET inhibition has been shown to potently modulate inflammatory autoimmune gene expression.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> BET bromodomains control the differentiation of CD4<sup>+</sup> T cells to inflammatory Th17 cells through genomic localization and their effects on cytokine regulation.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Jiang et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> reported that BRD4 ligands have dual effects on the HMGB1 and NF-κB signaling pathways and BRD4 is a potential therapeutic target for osteoarthritis. Meanwhile, BRD4 proteins regulate gene transcription during lineage-specific differentiation of naïve CD4<sup>+</sup> T cells to produce mature T-helper cells. Pan-BET bromodomain (BRD) inhibitors have distinct immune functions, thus limiting their therapeutic potential.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> In human monocytes, BET inhibition suppresses the transcriptional response of HMGB1 and NF-κB pathway to cytokine stimulation, and downregulates proinflammatory cytokine production in macrophages,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> dendritic cells,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and rheumatoid arthritis synovial fibroblasts.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In vivo, BET inhibitors exhibit therapeutic effects in a variety of anti-inflammatory disease models, such as colitis,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> arthritis,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> psoriasis,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and pathologic bone loss (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative BET bromodomain family selective inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the past decade, multiple structural cores that mimic acetylated lysine (KAc) binding have been discovered. Among BET inhibitors, the triazolodiazepine chemotype has been extensively explored. With (+)-JQ1 (<b>1</b>)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> as a representative widely used in biological mechanism research, its derivatives have been advanced into clinical trials, including i-BET762,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> OTX-015,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> TEN-10,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and CPI-0610.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Except RVX-208 (<b>2</b>),<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> these clinical-stage molecules exhibit little or no selectivity between the eight bromodomains of the BET family. Given the fact that eight bromodomains in BET family are highly homologous, it is still a challenge to obtain highly selective inhibitors. It is still not clear if side effects and toxicity were related to their selectivity in biological function.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Therefore, the development of highly selective inhibitors is urgently needed.</div><div class="NLM_p">Currently, only a few selective BET inhibitors were reported. Among them, RVX-208 (<b>2</b>) is a compound with a high degree of selectivity for BET BD2 and good response in the treatment of metabolic diseases in clinical trials.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Based on AlphaScreen data, RVX-208 (2) has about 170-fold selectivity toward BD2 over BD2 (IC<sub>50</sub> values of 87 ± 10 and 0.510 ± 0.041 μM for BD1 of BRD3 and BD2 of BRD3, respectively).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The sequence alignment (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A–C) shows that the BC loop-specific DDI of the first bromodomain of the four BET proteins is different from the BC loop hepatitis delta virus (HDV) or hepatitis E virus (HEV) sequence of the second bromodomain. RVX-208 (<b>2</b>) selectively binds to the second bromodomain by utilizing the fine structural differences of the BC loop-specific histidine in the two bromodomains.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Compound <b>3</b> selectively binds to the second bromodomain, and His433, which is unique to BRD4 BD2, is utilized to form hydrogen-bond interactions to increase selectivity.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Weak (∼10-fold) BRD4 BD1 bias is also observed for the cyclopent-2-en-1-one inhibitor MS402 (<b>4</b>), which blocked Th17 maturation from mouse naive CD4<sup>+</sup> T cells, with limited or no effects on Th1, Th2, or Treg cells.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Experimental models of mouse colitis induced by T cell transfer have been observed to potentially treat inflammatory bowel diseases. The diazobenzene MS611 (<b>5</b>) was reported to be moderately active for BRD4 BD1 with <i>K</i><sub>d</sub> = 0.41 μM and 100-fold selectivity over BRD4 BD2, where the phenol acts as KAc mimetic.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0012.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Sequence alignment of BET bromodomains with the residues highlighted. (B) Comparison of BRD4 BD1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>, green) and BD2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0">5uf0</a>, gray) with (+)-JQ1 (<b>1</b>). The cartoons and lines show the binding pockets of BD1 and BD2, and the sticks are shown to highlight key differential residues. (C) Comparison of BRD4 BD1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>, green) and BD2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0">5uf0</a>, gray) with (+)-JQ1 (<b>1</b>). The surfaces and lines show the binding pockets of BD1 and BD2, and the color surface is shown to highlight the volume of the space differential residues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There are currently very few small-molecule inhibitors that selectively bind to a single bromodomain of the BET family proteins. It remains a challenge to identify highly potent and selective-bromodomain small-molecule BET inhibitors. Importantly, Ding et al.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> reported a novel chemical core of 1-ethylbenzo[<i>cd</i>]indol-2(1<i>H</i>)-one based on virtual screening, which showed promising BRD4 binding activity. Subsequently, Chen et al.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> also reported that this core has excellent BRD4 activity. Although derivatives of this core only displayed moderate anticancer cell proliferation activity in vitro compared to triazole derivatives (e.g., JQ1), they show great potential for the development of anticancer therapeutics. Further investigation on benzo[<i>cd</i>]indol-2(1<i>H</i>)-ones may stimulate discovery of selective-bromodomain BET inhibitors.</div><div class="NLM_p last">A previous fragment-based screen carried out in our laboratories identified benzo[<i>cd</i>]indol-2(1<i>H</i>)-one <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which bound weakly but with unique structural features to BRD4 BD1 (IC<sub>50</sub> = 15.53 μM and Δ<i>T</i><sub>m</sub> = 2.23 °C). Through a series of structural modifications, we obtained a class of inhibitors that specifically bind to BRD4 BD1. Among them, we identified a lead compound with nanomolar potency toward BD1, and more importantly, it displayed more than 100-fold selectivity toward BD1 over BD2. The lead compound has a potential in the treatment of BRD4/NF-κB/NLRP3-mediated gout arthritis, providing a very useful tool for the disease phenotype of subsequent selective-bromodomain BET inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24567" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24567" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The synthesis of compounds <b>9</b> and <b>12</b>–<b>34</b> is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Benzo[<i>cd</i>]indol-2(1<i>H</i>)-one was used as the starting material, and compound <b>9</b> was prepared through methylation, electrophilic aromatic substitution with chlorosulfonic acid, and nucleophilic addition with aniline. Compounds <b>11a</b>–<b>11c</b> are synthesized by alkylation and nitration reaction, followed by hydrogenation. Starting from the intermediate <b>11a</b>, the title compounds <b>12</b>–<b>14</b> were prepared through the nucleophilic aromatic substitution with appropriate halides or benzoyl chlorides. Further, the title compounds <b>15</b>–<b>34</b> were prepared by appropriate benzenesulfonyl chlorides with <b>11a</b>–<b>11b</b> in the presence of a small amount of pyridine.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes of Compounds <b>9</b> and <b>12</b>–<b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) appropriate iodide, 60% NaH, dimethylformamide (DMF); (b) CISO<sub>3</sub>H, 0 °C; (c) aniline, pyridine, dichloromethane (DCM), rt; (d) 69% HNO<sub>3</sub>, AcOH, rt; (e) H<sub>2</sub>, 10% Pd/C, MeOH, tetrahydrofuran (THF), rt; (f) appropriate bromide or benzoyl chloride, 60% NaH, DMF, rt; (g) appropriate sulfonyl chlorides, pyridine, DCM, rt; (h) CH<sub>3</sub>I, 60% NaH, DMF.</p></p></figure><div class="NLM_p">In <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, benzo[<i>cd</i>]indol-2(1<i>H</i>)-one derivatives were prepared (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). The synthesis of intermediate <b>39</b> is no longer described in the same way as reported by Yang et al.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Starting from inexpensively available 4-chloroindoline-2,3-dione, the intermediate was nitrated with nitric acid sulfuric acid system from −25 to −10 °C to afford compound <b>40</b>. Intermediates <b>43</b> and <b>44</b> were synthesized from <b>39</b> by hydrogenation and dechlorination. Meanwhile, <b>43a</b>–<b>43n</b> were prepared from <b>40</b> with different amines by nucleophilic reaction, followed by hydrogenation of the nitro group, and <b>43o</b>–<b>43v</b> were prepared from <b>40</b> using the Suzuki–Miyaura cross-coupling reaction with different boronic acids/esters, followed by hydrogenation of the nitro group. Finally, the title compounds <b>46</b>–<b>70</b> were prepared with different benzenesulfonyl chlorides in the presence of a small amount of pyridine.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0017.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Routes of Compounds <b>46</b>–<b>70</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CH<sub>3</sub>I, 60% NaH, DMF; (b) BocNH<sub>2</sub>, Pd(AcO)<sub>2</sub>, <i>x</i>-PHOS, K<sub>2</sub>CO<sub>3</sub>, anhydrous 1,4-dioxane, 80 °C, 10 h; (c) <i>tert</i>-butyl 2-(triphenyl-λ<sup>5</sup>-phosphanylidene)acetate, DMF, rt; (d) TFA, 100 °C; (e) POCl<sub>3</sub>, 100 °C; (f) H<sub>2</sub>SO<sub>4</sub>, HNO<sub>3</sub>, −25 to −10 °C; (g) appropriate boronic ester, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane, N<sub>2</sub>, 80 °C; (h) appropriate amine derivatives, anhydrous 1,4-dioxane, 100 °C; (i) K<sub>2</sub>CO<sub>3</sub>, 10% P/C, H<sub>2</sub>, THF; (j) 10% P/C, H<sub>2</sub>, THF, MeOH; (k) appropriate sulfonyl chlorides, pyridine, DCM, rt.</p></p></figure><div class="NLM_p">In <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, benzo[<i>cd</i>]indol-2(1<i>H</i>)-one derivatives were prepared (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Starting from inexpensively available 1,8-naphthalic anhydride, intermediate <b>71</b> was synthesized through halogenation reaction with bromine and silver sulfate in concentrated sulfuric acid. Two key steps, nucleophilic substitution and strong base decarboxylation, were carried out in one pot in acceptable yields to obtain <b>72</b>. Subsequently, <b>75a</b>–<b>75g</b> were synthesized through a three-step reaction, including methylation with methyl iodide, nitrification under acetic acid–nitric acid conditions, and Suzuki coupling reaction with different boric acid or borate esters. The intermediates were subjected to hydrogenation and were used directly for a next step without further purification. Finally, the title compounds <b>77</b>–<b>90</b> were prepared with different benzenesulfonyl chlorides and <b>76a</b>–<b>76g</b> in the presence of a small amount of pyridine.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0018.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Routes of Compounds <b>77</b>–<b>90</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Br<sub>2</sub>, H<sub>2</sub>SO<sub>4</sub>, Ag<sub>2</sub>SO<sub>4</sub>, 60 °C; (b) i: NH<sub>2</sub>OH HCl, TsCl, pyridine, 90 °C; ii: NaOH, HCl, EtOH, H<sub>2</sub>O; (c) HNO<sub>3</sub>, AcOH, 60 °C; (d) CH<sub>3</sub>I, 60% NaH, DMF; (e) appropriate boronic este, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane, N<sub>2</sub>, 80 °C; (f) Fe, NH<sub>4</sub>Cl, EtOH, 80 °C; (g) appropriate sulfonyl chlorides, pyridine, DCM, rt.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52013" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52013" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Optimization of the Linker to the WPF Shelf</h3><div class="NLM_p">To improve the potency of benzo[<i>cd</i>]indol-2(1<i>H</i>)-one-based BET bromodomain inhibitors such as <b>6</b>, we explored interactions between the inhibitor core and the conserved Asn140 residue and the WPF shelf of the BET protein. Compounds were evaluated using differential scanning fluorimetry (DSF) and amplified luminescent proximity homogeneous assay (AlphaScreen). The DSF binding assay and AlphaScreen assay were used to determine the thermal stability (Δ<i>T</i><sub>m</sub>) and the affinities (IC<sub>50</sub>) of compounds for a construct containing the two bromodomains of BRD4. All derivatives were tested for their inhibitory potency toward recombinant human BRD4 BD1 His and BRD4 BD2 His.</div><div class="NLM_p">From the binding mode of <b>6</b> with BRD4 BD1 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A), we can see that the carbon-based oxygen of the benzo[<i>cd</i>]indol-2(1<i>H</i>)-one forms a molecular key hydrogen bond with the amino group of Asn140 and indirectly forms another molecular key hydrogen bond with the water molecule and Tyr97. The phenylmethyl group of compound <b>6</b> extended to the WPF shelf via a sulfonamide linker, not in the WPF hydrophobic center region. To better fit the WPF shelf, we examined a series of linkers in compounds <b>9</b>, <b>12</b>–<b>15</b>, including methylene, ethylene, amide, and back-linked sulfonamides. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, methylene, ethylene, and amide did not achieve the desired effect and instead had a large increase in the non-BET family. It is very exciting that the value of Δ<i>T</i><sub>m</sub> of <b>15</b> was elevated by 3.53 °C over the forward sulfonyl linker in compound <b>9</b>, still remaining selective for the BET family. The binding mode (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) showed that the benzene ring of <b>15</b> has a high degree of fit with the space of the benzene ring of JQ1 (<b>1</b>) at the center of the WPF shelf.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0019.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode analysis of compounds <b>6</b> and <b>15</b> to BRD4 BD1 protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>). (A) Compound <b>6</b> (shown as blue) docked into BRD4 BD1 overlaid <b>JQ1</b> (shown as green) with BRD4(1) protein. (B) Compounds <b>6</b> (shown as blue) and <b>15</b> (shown as purple) docked into BRD4 BD1 overlaid <b>JQ1</b> (shown as green) with BRD4(1) protein (displayed as gray surface).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures and Biological Evaluation of Target Compounds</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0011.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">Δ<i>T</i><sub>m</sub> (°C)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compounds</th><th class="colsep0 rowsep0" align="center" char=".">BRD4(1)</th><th class="colsep0 rowsep0" align="center">BRD9(1)</th><th class="colsep0 rowsep0" align="center">CECR2(1)</th><th class="colsep0 rowsep0" align="center" char=".">CREBBP(1)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char=".">2.19</td><td class="colsep0 rowsep0" align="left">0.47</td><td class="colsep0 rowsep0" align="left">–0.03</td><td class="colsep0 rowsep0" align="char" char=".">0.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char=".">2.70</td><td class="colsep0 rowsep0" align="left">–0.90</td><td class="colsep0 rowsep0" align="left">1.92</td><td class="colsep0 rowsep0" align="char" char=".">1.60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">3.15</td><td class="colsep0 rowsep0" align="left">4.33</td><td class="colsep0 rowsep0" align="left">0.13</td><td class="colsep0 rowsep0" align="char" char=".">0.55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">2.17</td><td class="colsep0 rowsep0" align="left">4.10</td><td class="colsep0 rowsep0" align="left">2.17</td><td class="colsep0 rowsep0" align="char" char=".">0.93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">5.72</td><td class="colsep0 rowsep0" align="left">1.93</td><td class="colsep0 rowsep0" align="left">–0.31</td><td class="colsep0 rowsep0" align="char" char=".">1.51</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Using differential scanning fluorimetry (DSF) at 10 μM concentration.</p></div></div></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Optimization of the WPF Shelf and the KAc Pocket</h3><div class="NLM_p">With these encouraging results, the benzene ring moiety of <b>15</b> was investigated with a suitable aromatic ring and a substituted ortho- or para-monosubstituted benzene ring (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) while maintaining the sulfonamide linker of <b>15</b>. From the comparison of BRD4 BD1 protein and BRD4 BD2 sequence (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), it was found that the amino acids (Tyr, Pro, and Phe) constituting the WPF region were significantly different in terms of space volume, electrical property, and flexibility. First, the spatial volume and electrical properties of the WPF shelf were explored using five-membered thiophene in <b>16</b>, electron-deficient pyridine in <b>17</b>, and bulkier quinoline ring in <b>18</b>. Among them, the IC<sub>50</sub> values of <b>15</b> are 709.1 ± 4.5 and 2621 ± 10 nM for BD1 and BD2, respectively, suggesting about 3.7-fold selectivity (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The selectivity was reduced by the increase in the size of R<sup>1</sup> and the activity of the methyl group (<b>19</b>) appears to be the best. Changes in methylation of <b>22</b>, selectivity, and binding were not significant compared to <b>15</b>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structures and Biological Evaluation of Target Compounds <b>15</b>–<b>28</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a><a class="ref internalNav" href="#t2fn3" aria-label="c">c</a><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0013.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0006.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Using differential scanning fluorimetry (DSF) at 10 μM concentration.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Using AlphaScreen assay, the values are mean ± SD from two independent experiments.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">nd: not determined.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last"><i>n</i> = 2.</p></div></div><div></div></div><div class="NLM_p">To investigate the effect of ortho substituents on the potency and selectivity, we designed chloro, methyl, ethoxy trifluoromethoxy, and ethoxy substituents. The increase in the potency is very limited with the 2-position benzene ring replaced with electron-withdrawing chlorine (<b>23</b>) and trifluoromethoxy (<b>27</b>) and electron-donating methyl (<b>24</b>) and ethoxy groups (<b>28</b>). Surprisingly, a breakthrough level in the selectivity and the affinity was found in compound <b>25</b> (IC<sub>50</sub>, 49.5 ± 10.3 and 2479 ± 170 nM for BD1 and BD2, respectively, thus about 50-fold selectivity) with a methoxy group at the ortho-phenyl.</div><div class="NLM_p">We further designed a series of derivatives with multiple substitutions on the phenyl ring and examined their effects on the activity and selectivity of BD1 and BD2, as shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. The introduction of 2,6-dichloro (<b>29</b>) and 2,6-difluoro (<b>30</b>) substitutions in WPF shelf has a greater decreasing effect on the activity, while the activity of the 2,3-substituents (<b>31</b>) on the same side of the benzene ring is restored. In addition, it was found that 2,4-substituent groups (<b>32</b>–<b>34</b>) are more selective to BRD4 BD1. Among them, the IC<sub>50</sub> values of <b>33</b> were 91.3 ± 3.3 and 3828 ± 48 nM for BD1 and BD2, respectively, thus about 42-fold selectivity (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Taking into account the physical and chemical properties, we have moderately decreased the activity and selectivity of the 5-position C of the benzofluorene ring with N in compound <b>48</b>, and its physical and chemical properties (<b>48</b> predicted tPSA: 88 and C log <i>P</i>: 1.29; <b>25</b> predicted tPSA: 75 and C log <i>P</i>: 2.03) are improved.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structures and Biological Evaluation of Target Compounds <b>29</b>–<b>28</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a><a class="ref internalNav" href="#t3fn3" aria-label="c">c</a><a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0014.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0023.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Using differential scanning fluorimetry (DSF) at 10 μM concentration.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Using AlphaScreen assay, the values are mean ± standard deviation (SD) from two independent experiments.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">nd: not determined.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last"><i>n</i> = 2.</p></div></div><div></div></div><div class="NLM_p">To clarify the outstanding activity and selectivity of compound <b>25</b>, we determined the crystal structure of compound <b>25</b> with BRD4 BD1, as shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A. The phenyl group of compound <b>25</b> differs from the conformation of the phenyl group of JQ1 (<b>1</b>) or other above derivatives (<b>6</b> and <b>15</b>, the binding mode in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) and does not completely occupy the WPF shelf, but is more adjacent to the region between Asp144 and Lue92. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, steric hindrance may exist between the 2-methoxyphenyl of <b>25</b> and the imidazole of BD2-specific His437 (with a distance of around 3.2 Å from <b>25</b>), which suggests a very unfavorable conformation binding to BRD4 BD2. The 2-position methoxy group may play a crucial role in the selective mechanism, and this hypothesis was verified by the selection and activity in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. The 2-methoxyphenyl module was moved forward in further optimization, and new modules were added to further improve selectivity and activity.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0020.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Conformation of compound <b>25</b> in the BRD4 BD1 and BRD4 BD2 proteins. (A) X-ray crystal structure of compound <b>25</b> bound to BRD4 BD1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJB">6JJB</a>; the authors will release the atomic coordinates and experimental data upon article publication). The ligands are shown as sticks models, and the protein is shown as surface and a cartoon with line model. (B) X-ray crystal structure of <b>25</b> (surface and line, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJB">6JJB</a>) bound to BRD4 BD1, overlaid with BRD4 BD2 (stick and line, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0">5uf0</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Optimization of the BC Loop Solvent Region</h3><div class="NLM_p">The BC loop solvent region formed by B loop and C loop was found to have a large difference in residues between the two domains BD1 and BD2, although the two domains are highly homologous in the BRD4 protein. The greatest differences lie in the BC loop, with Asp144 in BD1 changed to His437 in BD2 and an aspartic to glutamic acid switch of the adjacent residue. It was suspected that the introduction of appropriate hydrophilic groups in this region may be beneficial for activity and selectivity. Therefore, a series of derivatives in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> were synthesized and evaluated for the binding potency of BRD4 BD1 and BRD4 BD2. First, <b>49</b> and <b>50</b> moderately decreased BRD4 BD1 efficacy compared to 5-position chlorine substitutions in <b>46</b> and <b>48</b>. Cyclic amine had good potency in different volumes of amino groups, including <i>tert</i>-butylamine and cyclopropylamine.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Optimization of the Hydrophilic Group in the Solvent-Accessible Region<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0001.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0016.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Using AlphaScreen assay, the values are mean ± SD from two independent experiments.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">nd: not determined.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last"><i>n</i> = 2.</p></div></div><div></div></div><div class="NLM_p">Inspired by this structure–activity relationship, the difference in binding potency tolerance between BD1 and BD2 was examined using groups (different chain lengths and hydrophilic contributions at the 5-position R<sup>3</sup>). The IC<sub>50</sub> values from the AlphaScreen data showed that these derivatives had approximately a 5-fold change in potency (except <b>57</b>) for BRD4 BD1, but their binding potency to BRD4 BD2 was greatly affected. The methylpiperidine of <b>53</b> was more suitable for the solvent region of BRD4 BD1 than for the tetrahydro-2<i>H</i>-pyran of <b>54</b>. Interestingly, subtle changes of nitrogen atoms in <b>55</b>–<b>57</b> could obtain unexpected results. Among them, the IC<sub>50</sub> values of <b>55</b> are 328 ± 7.8 and 7068 ± 69 nM for BD1 and BD2, respectively, thus about 21-fold selectivity. In contrast, the introduction of moderately hydrophobic pyridine <b>57</b> was nearly 20-fold lower for BD1 on potency. To further explore the characteristics of BD1 and BD2 in the solvent accessible region of BC, we designed a larger volume of six-membered cyclic hydrophilic fragments directly linked to the pyridine 2-position. As shown in the AlphaScreen assay of <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, the morpholine ring and the nitrogen methylpiperazine ring exhibited good potency against BD1 over BD2. However, the <i>N</i>-methyl group of <i>N</i>-methylpiperazine (<b>60</b>–<b>61</b>) did not show improvement when it was replaced by other groups.</div><div class="NLM_p">To better understand the mechanism of selective binding of compound <b>60</b> to BRD4 BD1 over BRD4 BD1, we determined a co-crystal structure of compound <b>60</b> with BRD4 BD1. In <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, the 2-methoxyphenyl group of <b>60</b> differs from the selective dominant conformation of <b>25</b>, with deflection fully entering the WPF shelf. Therefore, the selectivity decrease of <b>60</b> is due to the fact that R<sup>3</sup><i>N</i>-methylpiperazinyl changes the original conformation of 2-oxyphenyl group, resulting in reduced steric hindrance to imidazole H437 in BD2 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) without affecting the binding of BD1. Although the selective contribution of 2-methoxyphenyl was weakened in <b>60</b>, a new steric hindrance effect was obtained between <i>N</i>-methylpiperazinyl and imidazole H437 in BD2 (the closest distance of 1.5 Å, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). As shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, the selectivity of <b>60</b> was 21-fold lower than that of <b>25</b> for BD1 over BD2.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Conformation of compound <b>60</b> in the BRD4 BD1 and BRD4 BD2 proteins. (A) X-ray crystal structure of compound <b>60</b> bound to BRD4 BD1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJ3">6JJ3</a>; the authors will release the atomic coordinates and experimental data upon article publication). The ligands are shown as sticks models, and the protein is shown as surface and a cartoon with line model. (B) X-ray crystal structure of <b>60</b> (stick) bound to BRD4 BD1 (line and cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJ3">6JJ3</a>), overlaid with BRD4 BD2 (line and cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0">5uf0</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Based on the structure–activity relationship, we made an assumption that an appropriate rigid group substitution in the solvent region of R<sup>3</sup> may render a ligand with a high selectivity for BRD4 BD1 over BRD4 BD2. The rigid ring is more likely to have a repulsive effect with the imidazole ring of the His437 residue in BD2 and reduce the affinity for BD2. We designed multiple rigid five-membered heterocycles to validate this hypothesis. The activity evaluation results showed excellent protein binding ability and selectivity (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). When X is a quinoline core and R<sup>3</sup> is unsubstituted 2-furan (<b>65</b>) and 3-furan (<b>66</b>), the IC<sub>50</sub> values for BD1 are 35.1 ± 2.2 and 37.0 ± 1.3 nM, respectively. Next, we introduced an imidazole in <b>67</b> and a methylimidazole ring in <b>68</b> with IC<sub>50</sub> values of 889.0 ± 32.9 and 87.7 ± 4.9 nM for BD1, respectively. Excitingly, the IC<sub>50</sub> values of compound <b>68</b> are 87.7 ± 4.9 and 12130 ± 308 nM for BD1 and BD2, respectively, thus about 138-fold selectivity. The methyl group of <b>68</b> resulted in a significant steric hindrance effect between the methylimidazole (<b>68</b>) and the H437 residue of BD2, which in turn increased selectivity without affecting the binding of BD1. Compound <b>69</b> merely changes the position of the nitrogen atom compared to compound <b>68</b> resulting in a loss of selectivity of more than 100-fold. However, when 5-methylfuran-2-yl was introduced in <b>79</b>, the IC<sub>50</sub> values suggested 857-fold selectivity (IC<sub>50</sub> = 54.8 ± 6.7 and 47 000 ± 542 nM for BD1 and BD2, respectively). The hetero five-membered ring and a methyl group at R<sup>3</sup> in (<b>68</b>, <b>69</b>, <b>77</b>, and <b>79</b>) showed good selectivity and activity. This also shows that the volume is favorable for the steric hindrance effect of the His437 of BRD4(2). In addition, due to the imidazole electron-deficient ring of His437, the hetero atom position of R<sup>3</sup> has a great contribution to the selectivity (<b>68</b> and <b>79</b>). The affinity of <b>80</b> for BD1 was reduced by more than 3-fold by the introduction of 3-furyl. When Ar is a 2-methoxyphenyl group in the WPF shelf and R<sup>3</sup> is a methyl- (<b>69</b> and <b>77</b>), propyl- (<b>82</b>), cyclopropyl- (<b>70</b> and <b>84</b>), and benzyl (<b>90</b>)-substituted pyrazole, its IC<sub>50</sub> value for BD1 was between 14.4 ± 5.7 and 205.2 ± 16.2 nM, its selectivity being within 25-fold. Furthermore, when Ar in the WPF shelf is substituted by other substituted phenyl in <b>85</b>–<b>89</b> (except 2-methoxy-4-chlorophenyl), the activity is drastically reduced or even lost.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Optimization of the Hydrophilic Group in the Solvent-Accessible Area<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a><a class="ref internalNav" href="#t5fn2" aria-label="b">b</a><a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0002.gif" alt="" id="fx8" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0015.gif" alt="" id="fx9" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0007.gif" alt="" id="fx10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Using AlphaScreen assay, the values are mean ± SD from two independent experiments.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">nd: not determined.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last"><i>n</i> = 2.</p></div></div><div></div></div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> In Vitro Evaluation of Selected Compounds on Anti-Inflammatory Activity</h3><div class="NLM_p">To investigate the role of single Bromodomains in the development of inflammatory diseases, we evaluated the selective-bromodomain BET inhibitors and pan-BET inhibitors to inhibit NF-κB activity in HUVECs cells and inhibit nitric oxide (NO) production in RAW264.7 cells. Detection of NO in macrophages is widely used in the screening of anti-inflammatory drugs.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> Leal et al.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> reported that pan-BRD4 inhibitors play an important role in the tumor microenvironment. These drugs suppressed the production of NO in RAW264.7 cells. Therefore, we selected 17 compounds at 1 μM to evaluate NO inhibition rate treated with 1 μg/mL of lipopolysaccharide (LPS) in macrophages. From the results shown (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>), most of the compounds showed significant activity to reduce the concentration of NO production. The inhibition rate of seven compounds was greater than 90%, and the selectivity of compounds <b>33</b> and <b>68</b> was more than 40-fold. Remarkably, the NO inhibition rate of RVX-208 with selective binding to BD2 was significantly lower than the NO inhibition rate of inhibitors with high activity for BD1 (JQ1 and other compounds). In addition, the highly selective compounds <b>25</b> and <b>79</b> failed to decrease NO production, which may be due to poor permeability (<b>79</b> with low permeability 0.060 nM/s and <b>68</b> with moderate permeability 1.172 nM/s in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf" class="ext-link">Table S2</a>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. In RAW264.7 Cells, the Downregulate NO Production Activities of Selected Compounds at 1 μM Treated with 1 μg/mL of LPS</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">BRD4 IC<sub>50</sub> (nM)</th><th class="rowsep1 colsep0" align="center" char=".">selectivity</th><th class="rowsep0 colsep0" align="center" char="±"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd.</th><th class="colsep0 rowsep0" align="center">BD1</th><th class="colsep0 rowsep0" align="center">BD2</th><th class="colsep0 rowsep0" align="center" char=".">BD2/BD1</th><th class="colsep0 rowsep0" align="center" char="±">at 1 μM NO inhibition rate (%)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">167 ± 14</td><td class="colsep0 rowsep0" align="left">2717 ± 401</td><td class="colsep0 rowsep0" align="char" char=".">16.2</td><td class="colsep0 rowsep0" align="char" char="±">15.09 ± 8.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">49.5 ± 10.3</td><td class="colsep0 rowsep0" align="left">2479 ± 170</td><td class="colsep0 rowsep0" align="char" char=".">50.1</td><td class="colsep0 rowsep0" align="char" char="±">51.37 ± 4.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">172.1 ± 12.8</td><td class="colsep0 rowsep0" align="left">1640 ± 68</td><td class="colsep0 rowsep0" align="char" char=".">9.5</td><td class="colsep0 rowsep0" align="char" char="±">101.89 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">91.3 ± 3.3</td><td class="colsep0 rowsep0" align="left">3828 ± 48</td><td class="colsep0 rowsep0" align="char" char=".">41.9</td><td class="colsep0 rowsep0" align="char" char="±">98.11 ± 3.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="left">205.2 ± 22.4</td><td class="colsep0 rowsep0" align="left">4599 ± 53</td><td class="colsep0 rowsep0" align="char" char=".">22.4</td><td class="colsep0 rowsep0" align="char" char="±">94.34 ± 3.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="left">113.0 ± 1.8</td><td class="colsep0 rowsep0" align="left">1284 ± 62</td><td class="colsep0 rowsep0" align="char" char=".">11.4</td><td class="colsep0 rowsep0" align="char" char="±">47.17 ± 18.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">328 ± 7.8</td><td class="colsep0 rowsep0" align="left">7068 ± 69</td><td class="colsep0 rowsep0" align="char" char=".">21.5</td><td class="colsep0 rowsep0" align="char" char="±">43.40 ± 8.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>56</b></td><td class="colsep0 rowsep0" align="left">223 ± 15.3</td><td class="colsep0 rowsep0" align="left">1520 ± 61</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char="±">39.62 ± 1.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>58</b></td><td class="colsep0 rowsep0" align="left">95.2 ± 12.3</td><td class="colsep0 rowsep0" align="left">808.4 ± 26.9</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char="±">100.00 ± 3.28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60</b></td><td class="colsep0 rowsep0" align="left">100.0 ± 6.8</td><td class="colsep0 rowsep0" align="left">2117 ± 42</td><td class="colsep0 rowsep0" align="char" char=".">21.1</td><td class="colsep0 rowsep0" align="char" char="±">92.45 ± 3.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>65</b></td><td class="colsep0 rowsep0" align="left">35.1 ± 2.2</td><td class="colsep0 rowsep0" align="left">981.7 ± 42.7</td><td class="colsep0 rowsep0" align="char" char=".">27.3</td><td class="colsep0 rowsep0" align="char" char="±">64.86 ± 8.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>66</b></td><td class="colsep0 rowsep0" align="left">37.0 ± 1.3</td><td class="colsep0 rowsep0" align="left">458.7 ± 52.5</td><td class="colsep0 rowsep0" align="char" char=".">12.37</td><td class="colsep0 rowsep0" align="char" char="±">66.28 ± 2.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>68</b></td><td class="colsep0 rowsep0" align="left">87.7 ± 4.9</td><td class="colsep0 rowsep0" align="left">12130 ± 308</td><td class="colsep0 rowsep0" align="char" char=".">138.3</td><td class="colsep0 rowsep0" align="char" char="±">101.89 ± 1.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>77</b></td><td class="colsep0 rowsep0" align="left">17.4 ± 5.6</td><td class="colsep0 rowsep0" align="left">434.9 ± 7.3</td><td class="colsep0 rowsep0" align="char" char=".">24.99</td><td class="colsep0 rowsep0" align="char" char="±">101.89 ± 5.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>79</b></td><td class="colsep0 rowsep0" align="left">54.8 ± 6.7</td><td class="colsep0 rowsep0" align="left">47000 ± 542</td><td class="colsep0 rowsep0" align="char" char=".">857.2</td><td class="colsep0 rowsep0" align="char" char="±">58.49 ± 17.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>82</b></td><td class="colsep0 rowsep0" align="left">43.7 ± 12.6</td><td class="colsep0 rowsep0" align="left">265.6 ± 7.9</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char="±">105.66 ± 3.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84</b></td><td class="colsep0 rowsep0" align="left">14.4 ± 5.7</td><td class="colsep0 rowsep0" align="left">237.6 ± 7.4</td><td class="colsep0 rowsep0" align="char" char=".">16.9</td><td class="colsep0 rowsep0" align="char" char="±">83.73 ± 1.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>JQ1</b></td><td class="colsep0 rowsep0" align="left">42.24 ± 5.08</td><td class="colsep0 rowsep0" align="left">24.71 ± 1.01</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char="±">47.56 ± 0.55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>RVX-208</b><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">1800</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="char" char=".">0.02</td><td class="colsep0 rowsep0" align="char" char="±">9.17 ± 0.41</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Using Griess reagents method, the values are mean ± SDs from three independent experiments, n = 3. NO inhibition rate % = (model group NO concentration–compound group NO concentration)/(model group NO concentration–control group NO concentration) × 100%.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">The data were assayed by TR-FRET.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></p></div></div></div><div class="NLM_p">The dual luciferase reporter gene assay data (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf" class="ext-link">Table S3</a>) showed that the selective BD1 inhibitors and pan-BET inhibitor (JQ1) were significantly more potent in downregulating the NF-κB activity than the selective BD2 inhibitor (RVX-208). The NF-κB transfection activity of compound <b>68</b> was superior to that of compounds <b>65</b> and <b>79</b>, comparable to the JQ1 inhibitory activity. Obviously, these results suggest that inhibitors selectively binding to BD1 have a better anti-inflammatory activity than inhibitors selectively binding to BD2 in vitro.</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0021.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Examination for the NF-κB transcriptional activity in HUVECs undergoing three concentration degrees (0.1, 1, and 10 μM) treated JQ1, RVX-208, and compounds for 24 h, respectively. The models were established by TNF-A 50 ng/mL. The values of fold changes are mean ± SDs from three independent experiments, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In the evaluation of in vitro inflammatory activity, the selective BET BD1 inhibitor <b>68</b> (inhibition rate: 101.89 ± 1.01%) was significantly superior to the pan-BET inhibitor JQ1 (inhibition rate: 47.56 ± 0.55%) in inhibiting the NO production of macrophage RAW264.7. The NO production is regulated by multiple inflammatory signaling pathways, and NF-κB signaling pathway is one of the important pathways regulating NO production. Selective BET BD1 inhibitor <b>68</b> of NF-κB transfection activity was comparable to pan-BET inhibitor JQ1 inhibitory activity. It is demonstrated that the selective BET BD1 inhibitor (<b>68</b> BRD4(1): 87.7 ± 4.9 nM; BRD4(2): 12130 ± 308 nM) still maintains comparable or better anti-inflammatory activity than the pan-BET inhibitor (JQ1 BRD4(1): 42.24 ± 5.08 nM; BRD4(2): 24.71 ± 1.01 nM) compared to the protein weak activity. Based on our analysis, compound <b>68</b> was considered for further studies in biological mechanisms and diseases.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Bromodomains Activity Profile of Compound <b>68</b></h3><div class="NLM_p">The lead compound <b>68</b> displayed the highest inhibitory activity against BRD4(1) (IC<sub>50</sub>: 87.7 ± 4.9 nM), BRD3(1) (IC<sub>50</sub>: 246.3 ± 20.2 nM), and BRDT(1) (IC<sub>50</sub>: 357.1 ± 8.3 nM). It should be noted that <b>68</b> also has inhibitory activities against BRPF1b (IC<sub>50</sub>: 567.5 ± 16.9 nM). In addition, compound <b>68</b> showed 102-fold selectivity for BRD4(1) over BRD2(1). The <i>K</i><sub>d</sub> values from the BROMOscan <i>K</i><sub>d</sub> data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf" class="ext-link">Figure S2</a>) suggest 238-fold selectivity toward BD1 over BD2 (<i>K</i><sub>d</sub> = 105 ± 5 and >25 μM for BD1 and BD2, respectively). Therefore, compound <b>68</b> was identified as a highly selective BET BD1 inhibitor and the inhibition of single bromodomain by compound <b>68</b> may be beneficial to its anti-inflammatory efficiency and reduce potential toxicity (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>).</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Inhibitory Activities of Compound <b>68</b> toward BRD2(1), BRD2(2), BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRDT(1), and BRPF1b</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" align="center" char="±"><a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" align="center" char=".">selectivity</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep0 colsep0" align="center" char="±"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">selectivity</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">BRDs</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">BRDs/BRD4(1)</th><th class="colsep0 rowsep0" align="center">BRDs</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">BRDs/BRD4(1)</th><th class="colsep0 rowsep0" align="center">BD2/BD1</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD2(1)</td><td class="colsep0 rowsep0" align="char" char="±">8948 ± 457</td><td class="colsep0 rowsep0" align="char" char=".">102.3</td><td class="colsep0 rowsep0" align="left">BRD2(2)</td><td class="colsep0 rowsep0" align="char" char="±">21450 ± 127</td><td class="colsep0 rowsep0" align="char" char=".">244.5</td><td class="colsep0 rowsep0" align="left">2.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD3(1)</td><td class="colsep0 rowsep0" align="char" char="±">246.3 ± 20.2</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="left">BRD3(2)</td><td class="colsep0 rowsep0" align="char" char="±">49770 ± 258</td><td class="colsep0 rowsep0" align="char" char=".">567.5</td><td class="colsep0 rowsep0" align="left">202.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4(1)</td><td class="colsep0 rowsep0" align="char" char="±">87.7 ± 4.9</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="left">BRD4(2)</td><td class="colsep0 rowsep0" align="char" char="±">12130 ± 308</td><td class="colsep0 rowsep0" align="char" char=".">138.3</td><td class="colsep0 rowsep0" align="left">138.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRDT(1)</td><td class="colsep0 rowsep0" align="char" char="±">357.1 ± 8.3</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="left">BRPF1b</td><td class="colsep0 rowsep0" align="char" char="±">567.5 ± 16.9</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Using AlphaScreen assay, the values are mean ± SD from two independent experiments, <i>n</i> = 2.</p></div></div></div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Pharmacokinetics of <b>68</b> in Liver Microsomes of Multiple Species</h3><div class="NLM_p">The preliminary metabolic stability study of compound <b>68</b> was performed by using human liver microsomes (HML), rat liver microsomes (RLM), mouse liver microsomes (MLM), dog liver microsomes (DLM), and monkey liver microsomes (CLM) (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Tables <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf" class="ext-link">S1</a>). The terminal <i>t</i><sub>1/2</sub> was moderate in human liver microsomes, rat liver microsomes, mouse liver microsomes, and monkey liver microsomes (ranged between 9.4 and 14.8 min). Compound <b>68</b> has a high clearance rate in the range of 93.517 μL/(min mg) proteins to 146.685 μL/(min mg) proteins for different species. Overall, compound <b>68</b> exhibited moderately stable levels of in vitro liver microsomal metabolism. Therefore, intra-articular injections for the treatment of acute gout arthritis were superior to oral and intravenous injections.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Pharmacokinetics of Compound <b>68</b> in Liver Microsomes of Multiple Species</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compounds</th><th class="colsep0 rowsep0" align="center">cpecies</th><th class="colsep0 rowsep0" align="center" char=".">R<sup>2</sup><a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (min)<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>int,microsome</sub> (μL/(min mg) proteins)<a class="ref internalNav" href="#t8fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>int,in vivo</sub> (mL/(min kg))<a class="ref internalNav" href="#t8fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left"><b>68</b></td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char=".">0.9840</td><td class="colsep0 rowsep0" align="char" char=".">14.821</td><td class="colsep0 rowsep0" align="char" char=".">93.517</td><td class="colsep0 rowsep0" align="char" char=".">88.373</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">monkey</td><td class="colsep0 rowsep0" align="char" char=".">0.9918</td><td class="colsep0 rowsep0" align="char" char=".">10.263</td><td class="colsep0 rowsep0" align="char" char=".">135.043</td><td class="colsep0 rowsep0" align="char" char=".">194.461</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="char" char=".">0.9105</td><td class="colsep0 rowsep0" align="char" char=".">10.409</td><td class="colsep0 rowsep0" align="char" char=".">133.159</td><td class="colsep0 rowsep0" align="char" char=".">191.748</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="char" char=".">0.9635</td><td class="colsep0 rowsep0" align="char" char=".">9.449</td><td class="colsep0 rowsep0" align="char" char=".">146.685</td><td class="colsep0 rowsep0" align="char" char=".">264.034</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">R<sup>2</sup> is the correlation coefficient of the linear regression for the determination of kinetic constant.</p></div><div class="footnote" id="t8fn2"><sup><sup>b</sup></sup><p class="last"><i>T</i><sub>1/2</sub> is the half-life.</p></div><div class="footnote" id="t8fn3"><sup><sup>c</sup></sup><p class="last">CL<sub>int</sub> is the intrinsic clearance.</p></div><div class="footnote" id="t8fn4"><sup><sup>d</sup></sup><p class="last">CL<sub>int,in vivo</sub> is the intrinsic hepatic clearance.</p></div></div></div></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Compound <b>68</b> Inhibited MSU-Induced Pyroptosis of THP-1 Cells through BRD4/NF-κB/NLRP3 Signaling Pathways</h3><div class="NLM_p">To investigate the anti-inflammatory mechanisms, we explored the relationship between compound <b>68</b> with high selectivity for BET BD1 in inflammation-related pyroptosis and signaling pathways in vitro. Pyroptosis was defined to activate in the presence of propidium iodide (PI) and active Caspase-1 double positivity by flow cytometry. MSU exposure in the model group exhibited a significant increment in the amount of PI and Caspase-1 double-positive cells in comparison to normal group, which could be reversed by pretreatment of compound <b>68</b> at 1 μM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). More importantly, as shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B, MSU stimulation (500 μg/mL) contributed to the upregulation of BRD4, phosphorylation of NF-κB p65, and activation of NLRP3 inflammasome compared to the normal group. These increased levels were notably attenuated by compound <b>68</b> intervention. Inhibition of compound <b>68</b> in the activation of NRLP3 inflammasome was also confirmed by immunofluorescence data (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Increased fluorescence intensity in NLRP3 and ASC staining as well as the co-localization of NLRP3 and ASC in merged data were significantly decreased by compound <b>68</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Moreover, MSU-induced elevation of IL-1β concentrations was ameliorated by compound <b>68</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D), suggesting that compound <b>68</b> with high selectivity for the first bromodomain could improve MSU-induced inflammatory cascade and pyroptosis via inhibiting BRD4/NF-κB/NLRP3 signaling pathways.</div><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0004.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>68</b> inhibits MSU-induced pyroptosis in THP-1 cells. After PMA treatment, THP-1 cells were incubated in the absence or presence of serum-free medium containing <b>68</b> (1 μM) for 1 h, followed by the stimulation with MSU (500 μg/mL) for 12 h. The rate of pyroptotic cell death was examined with active PI and Caspase-1 double staining by flow cytometry (A). The expression of BRD4 and NRLP3 inflammasome activation was measured by western blotting and protein quantitative analysis normalized to β-action (B). Representative confocal microscopy photographs of THP-1 cells with immunofluorescence changes (C). The culture supernatants level of IL-1β was detected using the Enzyme-linked immunosorbent assay (ELISA) kit (D). Data were presented as mean ± SD. Compared to the control group: <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, <sup>###</sup><i>P</i> < 0.001. Compared to the model group: *<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001. Each group (<i>n</i> = 4). Independent experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Compound <b>68</b> Improved MSU-Induced Acute Gouty Arthritis</h3><div class="NLM_p">With the promising in vitro results, we further investigated the pharmaceutical effect of compound <b>68</b> in rat. As shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C, after treatment with compound <b>68</b> via intra-articular injection (ia) at a dose of 1 mg/kg, the challenged joints were subjected to circumference detection at 0, 2, 4, 8, 12, and 24 h after MSU crystals induction at 4 mg/kg intra-articular injection (ia). As shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A, the joint circumference in the model group increased approximately 1.28-fold at 2, 4, and 8 h time points and enlarged to 1.52-fold at 12 h point than that of control group, while treatment with compound <b>68</b> restored the joint circumference to normal level, suggesting the powerful defense against swelling pathology brought by MSU-induced acute gouty arthritis. These results were supported by histopathological analysis of H&E-stained synovial tissue specimens, which revealed that the rats receiving compound <b>68</b> exhibited suppressed synovial hyperplasia as well as severe neutrophil infiltration compared to the model group (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B). In addition, an attenuation of MSU-induced raise in the synovial IL-1β level was observed after the treatment of compound <b>68</b>, which was consistent with in vitro study (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D). The in vivo study suggested that compound <b>68</b> has a good therapeutic effect on MSU-induced acute gouty arthritis.</div><figure id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0005.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Effect of BRD4 inhibitor on MSU-induced acute gouty arthritis (4 mg/kg, intra-articular injection) in vivo. Representative photographs to show the swelling of joints (A). Representative photographs of histopathologic changes in synovium (B). The injected ankle joint circumference of each rat was determined at 0, 2, 4, 8, 12, and 24 h after MSU stimulation (C). The IL-1β level in synovium homogenate was detected by the ELISA kit (D). The data were presented as mean ± SD. Compared to the control group: <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, <sup>###</sup><i>P</i> < 0.001. Compared to the model group: *<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001. Data in panels represent mean ± SD from eight animals. Independent experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Compound <b>68</b> Suppressed Pyroptosis of Macrophages in Rat Synovial Tissues through Regulating BRD4/NF-κB/NLRP3 Signaling Pathway</h3><div class="NLM_p">To further validate the in vivo biological mechanisms, we investigated whether compound <b>68</b> suppresses pyroptosis of macrophages in rat synovial tissue through regulating the BRD4/NF-kB/NLRP3 signaling pathway. To determine the rate of pyroptotic cell death in synovial tissue, macrophages were classified based on positive staining for CD68. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A, there was a remarkable elevation in the number of PI and Caspase-1 double-positive macrophages obtained from rat synovium of model rats in contrast to those of the normal group, while this increased number was pulled down closer to the normal level by the intervention of compound <b>68</b>. Consistent with in vitro results, western blotting analysis revealed a significant increase in BRD4 upregulation, NF-κB p65 phosphorylation, and NLRP3 inflammasome activation after MSU exposure in synovium specimens, which could be restored by compound <b>68</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B). Besides, immunofluorescence assay of synovial samples showed that compound <b>68</b> treatment could inhibit the enhanced fluorescence intensity of synovial NLRP3, ASC, and their co-localization induced by MSU injection (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C). This study confirmed that compound <b>68</b> was able to improve MSU-induced acute gouty arthritis through regulating BRD4/NF-κB/NLRP3 signaling pathways.</div><figure id="fig9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0022.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound <b>68</b> regulates the pyroptosis death in synovial tissue. Pyroptosis assay was performed in the macrophages obtained from synovium using active PI and Caspase-1 double staining by flow cytometry (A). Protein expression of BRD4 and NLRP3 inflammasome pathway in synovium after compound <b>68</b> treatment at 1 mg/kg intra-articular injection (ia) by western blotting and protein quantitative analysis normalized to β-action (B). NLPR3 inflammasome activation in synovial tissues evaluated with immunofluorescence staining (C). Data were presented as mean ± SD. Compared to the control group: <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, <sup>###</sup><i>P</i> < 0.001. Compared to the model group: *<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001. Data in panels represent mean ± SD from four animals. Independent experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43265" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43265" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, a promising BD1-selective BET inhibitor with a benzo[<i>cd</i>]indol-2(1<i>H</i>)-one core was obtained by virtual screening. Starting from this scaffold, we designed and synthesized a series of derivatives, which demonstrated high BRD4 BD1 potency, significant selectivity over the BRD4 BD2, and excellent selectivity for all other bromodomains and activities. Among them, compound <b>68</b> (LT052) showed nanomolar BRD4 BD1 potency and 138-fold selectivity over BRD4 BD2. Our detailed high-resolution crystal structural analysis of the BRD4 BD1 bound to the lead benzo[<i>cd</i>]indol-2(1<i>H</i>)-one ligands reveals that the steric hindrance effect of the amino residues in the BC loop region of BRD4 BD2 has an important contribution to the selectivity of BD1, especially His437 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><div class="NLM_p">Compound <b>68</b> (inhibition rate: 101.89 ± 1.01%) displayed better activity in reducing NO production in RAW264.7 cells than JQ1 (inhibition rate: 47.56 ± 0.55%) and other compounds in the same series (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Besides, <b>68</b> displayed good activity inhibiting NF-κB transcriptional activity in HUVECs. Due to the inhibitory activity of pan-BET inhibitors on multiple Bromodomains, pan-BET inhibitors exhibit various toxicities and side effects such as thrombocytopenia, fatigue, and nausea in clinical trials.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Among them, some toxicity and side effects have a dose-dependent limitation of drug treatment.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Moreover, the selectivity for BRD4 BD1 was more closely related to their antifibrosis activity, whereas the selective BD2 inhibitor (RVX-208) did not indicate any of this activity.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Furthermore, NUE20798 with potent and benign safety against BET BD1 in fibrosis diseases and atopic dermatitis has been developed and without the observation of the typical adverse toxicity seen with nonselective BET inhibition.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Encouraged by the results, NUE20798 hopes to launch human clinical trials. It provides a value therapeutic strategy for drug development in the inhibition of the single bromodomain of the intra-BET family.</div><div class="NLM_p last">Further studies revealed that compound <b>68</b> with BRD4/NF-κB/NLRP3 signaling pathway inhibits MSU-induced apoptosis of THP-1 cells and improves MSU-induced acute gouty arthritis. In the rat animal model, compound <b>68</b> inhibits the pyrophosphorylation of macrophages in rat synovial tissue by modulating this signaling pathway, thereby achieving treatment of acute gout arthritis. Currently, there are few specific drugs for the treatment of acute gout arthritis, and more toxic side effects.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> We explored a new therapeutic mechanism for the treatment of acute gout joints by selectively inhibiting BET BD1, which has clinical therapeutic significance. Meanwhile, the identification of the selective BET BD1-bromodomain inhibitors may facilitate studies on the biological role of the BET BD1-bromodomain and bromodomain-selective treatment of acute gout sources.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05515" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05515" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Protein Inhibition Assay</h3><div class="NLM_p">Differential scanning fluorimetry (DSF) was tested on a Mx3005p Real Time polymerase chain reaction (PCR) machine (Stratagene) device. The BRD4 protein was added to a buffer solution containing 10 nM HEPES, 500 nM NaCl, pH 7.5, and finally formulated into a 2 μM/20 μL solution in a 96-well plate. The test compound was added to the solution at a concentration of 10 μM while adding the fluorescent probe SYPRO Orange at a ratio of 1:1000. The excitation and emission wavelengths of SYPRO Orange are set at 465 and 590 nm, respectively. The temperature is raised from 25 to 96 °C by 3 °C per minute and fluorescence readings are taken at each interval. Then, according to the theoretical conversion formula between fluorescence intensity and temperature change, the temperature difference between the protein before and after binding of the ligand is calculated by mathematical statistics, as a method for evaluating the binding ability.</div><div class="NLM_p">The AlphaScreen technology assay<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> is used as the binding assay of the readers, which was performed by Reaction Biology Corp. (Malvern, PA), as described previously. The biotinylated peptide binding to the reader domain of His-tagged protein is monitored by the singlet oxygen transfer from the Streptavidin-coated donor beads to the AlphaScreen Ni-chelate acceptor beads. <i>Reagent</i>: Reaction buffer for BRD4s: 50 mM HEPES, pH 7.5, 100 mM NaCl, 0.05% CHAPS, 0.1% bovine serum albumin (BSA), and 1% dimethyl sulfoxide (DMSO) final. Reaction buffer for TAF1-2: 20 mM Tris, pH 8, 25 mM NaCl, 0.05% Tween20, 3 mM Bmercaptethanol, and 0.25% DMSO final (the final DMSO concentration may be different depending on compound stock and test concentrations). Bromodomain BRD4-1: RBC Cat# RD-11-140 Recombinant Human Bromodomain-containing protein 4, bromodomain 1 (aa 44–170; Genbank Accesstion # NM_058243), expressed in <i>Escherichia coli</i> with N-terminal His-tag. MW = 17.8 kDa. BRD4-2: RBC Cat# RD-11-141 Recombinant Human Bromodomain-containing protein 4, bromodomain 2 (aa 349–460; Genbank Accesstion # NM_058243), expressed in <i>E. coli</i>with N-terminal His-tag. MW = 15.7 kDa. bromodomain 2 (aa 1502–1635; Genbank Accesstion # NM_138923), expressed in <i>E. coli</i> with N-terminal His-tag. MW = 18.4 kDa. Ligand (C-term-Biotin) Histone H4 peptide (1–21) K5/8/12/16Ac-Biotin Detection beads: PerkinElmer Donor beads: Streptavidin-coated donor beads Acceptor beads: AlphaScreen Ni acceptor beads <i>Reaction Procedure:</i> 1. Deliver 2.5X BRD in wells of reaction plate except No BRD control wells. Add buffer instead. 2. Deliver compounds in 100% DMSO into the BRD mixture by acoustic technology (Echo550; nanoliter range). Spindown and preincubation for 30 min. 3. Deliver 5X Ligand. Spin and shake. 4. Incubate for 30 min at room temperature with gentle shaking. IC<sub>50</sub> values and curve fits were obtained by GraphPad Prism 7.0 software.</div><div class="NLM_p last">BROMOscan is a novel industry leading platform for identifying small-molecule bromodomain inhibitors, which was performed by DiscoverX Corp., as described previously (<a href="http://www.eurofinsdiscoveryservices.com" class="extLink">www.eurofinsdiscoveryservices.com</a>).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The assay process shows five steps: 1. Assemble assay components: <i>E. coli</i> or mammalian cell-expressed bromodomain labeled with DNA tag for quantitative polymerase chain reaction (qPCR) readout □Known bromodomain ligand immobilized on a solid support □Test compound or control (solvent only); 2. Equilibrate; 3. Wash solid support to remove unbound bromodomain; 4. Quantify bromodomain captured on solid support (qPCR); 5. Compare captured bromodomain levels in test compound and control samples. The compound(s) were screened at the concentration(s) requested, and the results for primary screen binding interactions are reported as “% Ctrl”. An 11-point threefold serial dilution of each test compound was prepared in 100% DMSO at 1000× final test concentration. All compounds for <i>K</i><sub>d</sub> measurements are distributed by acoustic transfer (noncontact dispensing) in 100% DMSO. The <i>K</i><sub>d</sub> values and curve fits were obtained by GraphPad Prism 7.0 software.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Griess Reagents Method for Detecting NO in RAW264.7 Cells</h3><div class="NLM_p last">Dulbecco’s modified Eagle medium (DMEM) was cultured in RAW264.7. The cell concentration was 5 × 10<sup>4</sup> cells/mL, and the cells were seeded in a 96-well plate at 200 μL per well, cultured at 37 °C, 5% CO<sub>2</sub> for 24 h, and the medium was discarded. The medium (197 μL, no serum) was added: blank control group, LPS model group, and test drug group. Subsequently, 2 μL of the medium was added to the blank control group. The LPS model group was spiked with 2 μL of DMSO at a concentration of 1 μM in the test group and three replicate wells per sample. After dosing, incubation was conducted for 2 h at 37 °C in a 5% CO<sub>2</sub> incubator, and then 1 μL of PBS-configured LPS (200 μg/mL) was added to the LPS control group and each test group to a final concentration of 1 μg/mL. After continuing to culture for 22 h, the cells were centrifuged and the supernatant was weighed to determine the NO content by Griess reagents. Mean and standard deviation (STDEV) are shown. Inhibition rate = (model group NO concentration–compound group NO concentration)/(model group NO concentration – control group NO concentration) × 100%.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Dual Luciferase Reporter Assay for Detecting NF-kB Activity in HUVEC Cell Lines</h3><div class="NLM_p last">To confirm the alteration of NF-kB activity by inhibitors in HUVEC cell lines, we performed a dual luciferase reporter assay. The p-NF-kB luc vector (Clontech, Palo Alto, CA) contains the firefly luciferase gene and four copies of the NF-kB consensus sequence. The phRL-null vector (Promega) contains a renilla luciferase gene, and the transfection efficiency was normalized by cotransfection with this vector. The p-NF-kB luc and the phRL-null vectors were cotransfected in a 30:1 ratio using SuperFect (Qiagen, Valencia, CA) according to the manufacturer of instructions. After 12 h transfection, the medium was treated with compounds and positive inhibitors (except the control group) for 0.5 h, followed by treatment with TNF-A (50 ng/mL). Another 24 h later, luciferase activity was measured by a luminometer (Lumat LB 9507; EG&G Berthold Tokyo, Japan) using the dual luciferase reporter assay system (Promega).</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Metabolic Stability in Liver Microsomes</h3><div class="NLM_p last">The metabolic stability of compound <b>68</b> in rat liver microsomes (RLM), dog liver microsomes (DLM), monkey liver microsomes (CLM), and human liver microsomes (HLM) was tested following the same procedures by 3D BioOptima Co. Ltd. Compound <b>68</b> (10 μL) or control working solution/well was added to all plates (T0, T5, T15, T30, T60, NCF60) except the blank matrix. Then, 80 μL/well was added to every plate by Apricot and the mixture of microsome solution and compound was, incubated at 37 °C for about 10 min. Then, 10 μL of 100 mM potassium phosphate buffer/well was added to NCF60 and the mixture was incubated at 37 °C for 1 h. After prewarming, 10 μL/well was added to every plate by Apricot to start reaction. The reactions were terminated at 5, 15, 30, and 60 min of incubation with a chilled mixture of Tolbutamide and Labetalol (1:1). The mixture was vortexed for 5 min, centrifuged at 4000 rpm for 20 min at 4 °C, and the supernatants were analyzed by LC-MS/MS. The data were analyzed by the first-order kinetics to calculate <i>t</i><sub>1/2</sub> and Cl. Drug elimination rate constant <i>k</i> (min<sup>–1</sup>), elimination half-life <i>T</i><sub>1/2</sub> (min), and in vitro intrinsic clearance, CL<sub>int,in vitro</sub> (μL/(min mg) protein) were calculated according to the following equations: <i>k</i> = −slope, <i>T</i><sub>1/2</sub> = 0.693/<i>k</i>, CL<sub>int,in vitro</sub> = <i>k</i>/C<sub>protein</sub>, where C<sub>protein</sub> (mg/mL) is the microsomal protein concentration in the incubation.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Molecular Modeling</h3><div class="NLM_p last">The X-ray crystal structures of BRD4(1) binding with (+)JQ1(<b>1</b>) (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>) and BRD4(2) (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0">5uf0</a>) were retrieved from the Protein Data Bank.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Compounds <b>6</b> and <b>15</b> were prepared using the protein preparation wizard in Maestro with standard settings. Grids of BRD4(1) and BRD4(2) were generated using Glide, version 10.2, following the standard procedure recommended by Schrodinger. The conformational ensembles were docked flexibly using Glide with standard settings in both standard and extra precision modes. Only poses with low energy conformations and good hydrogen-bond geometries were considered. Molecular-protein docking results and small-molecule–protein co-crystal structures were shown to use Pymol. At the same time, the overlap of BRD4(1) and BRD4(2) was the align module of the Pymol software. The numbering of the amino residues of the protein and the editing of the figure were performed using HprSnap7.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> X-Ray Crystallography</h3><div id="sec5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Protein Crystallization</h4><div class="NLM_p last">Crystallization was performed by using the sitting drop vapor diffusion method. Protein solution (1 μL) containing 7 mg/mL N-terminal bromodomain of BRD4, 5 mM inhibitors, and 10 mM methyl 2-oxo-2,3-dihydro-1<i>H</i>-pyrrolo[3,2-<i>c</i>]pyridine-6-carboxylate in 20 mM Tris, pH 8.0, 50 mM NaCl, was mixed with 1 μL of well solution containing 4.0 M sodium formate. The crystallization drop was incubated against 50 μL of well solution at 295 K. The best rodlike crystals of maximum size 200 × 50 × 50 μm appeared after 12 h and reached final size after 24 h.</div></div><div id="sec5_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Data Collection and Processing</h4><div class="NLM_p last">Crystals were transferred to a cryoprotectant solution comprising 4.0 M sodium formate and 20% glycerol then flash-frozen in liquid nitrogen. X-ray data of each protein–inhibtor complex were collected from a single crystal at 100 K at BL17U1 of Shanghai Synchrotron Radiation Facility. Images were processed with Mosflm.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The crystals belong to space group <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>, with one molecule per asymmetric unit (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf" class="ext-link">Table S5</a>).</div></div><div id="sec5_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Phasing, Model Building, and Refinement</h4><div class="NLM_p last">The structures of protein–inhibitor complexes were solved by molecular replacement implemented in Phaser.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Coordinates of N-terminal bromodomain of BRD4 (PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PCE">4PCE</a>) were used as the original search model. For each protein–inhibtor complex, the initial electron density difference map clearly showed extra density indentified as the inhibitor in the binding pocket of BRD4. The structure models of the inhibitors were manually fit in the extra electron density using COOT.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The built models were refined by the program PHENIX.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Figures of protein structures were created with Pymol (<a href="http://www.pymol.org" class="extLink">http://www.pymol.org</a>).</div></div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Cell Culture</h3><div class="NLM_p last">THP-1 cell line was obtained from American Type Culture Collection (Manassas, VA). The THP-1 cells were cultured at 37 °C under 5% CO<sub>2</sub> in RPMI1640 medium supplemented with 10% fetal bovine serum (Gibco), 100 IU/mL penicillin, and 100 IU/mL streptomycin. After phorbol 12-myristate 13-acetate (PMA; Sigma-Aldrich, St. Louis, MO) treatment, the THP-1 cells were incubated with the absence or presence of serum-free medium containing inhibitors (1 μM) for 1 h, followed by the stimulation with MSU (0.5 mg/mL) for 12 h. Subsequently, the culture supernatants were collected for IL-1β detection and the cell lysates were utilized for protein extraction.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Animal Models</h3><div class="NLM_p last">A total of 24 male adult Sprague-Dawley rats (250–280 g) purchased from Central Institute for Experimental Animals of Zhejiang were allowed 1 week to adapt to the laboratory environment before used for experiments. All animal studies were performed in accordance with the Animal Ethics Committee of China Pharmaceutical University. Acute gouty arthritis animal models were established by MSU crystals injection (1.0 mg suspended in 50 μL of sterile saline, 4 mg/kg intra-articular injection). The rats were randomly assigned into three groups (n = 8 per group): (1) normal group, (2) model group, and (3) <b>68</b>(LT052) group. The rats received intra-articular injection of vehicle (0.1% DMSO) or <b>68</b>(LT052) (1 mg/kg) 1 h before treatment of MSU. The injected ankle joint circumference of each rat was determined 0, 2, 4, 8, 12, and 24 h after MSU stimulation. Then, the animals were sacrificed, and serum and synovial tissues were collected for further detection.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Measurements of IL-1β</h3><div class="NLM_p last">The levels of IL-1β in THP-1 cell supernatants and rat synovial tissue homogenates were detected using ELISA Kits (R&D, Minneapolis) following the manufacturer’s instructions.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Pyroptosis Assay</h3><div class="NLM_p last">Pyroptosis was determined following the instruction of FLICA 660 in vitro Caspase-1 Detection Kit (Immuno Chemistry Technologies, Bloomington) by flow cytometry, while propidium iodide (PI) was used to mark cells with membrane pores (Beyotime Biotechnology, Nanjing, CN). To determine the rate of pyroptotic cell death in synovial tissue, macrophages were classified using CD68 staining.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Immunofluorescence</h3><div class="NLM_p last">The fixed synovial tissues from rats were embedded, decalcified, and sectioned, while THP-1 cells were fixed with 4% paraformaldehyde fix solution at room temperature after experiments. Immunofluorescence staining was conducted using primary antibodies at 4 °C overnight. Subsequently, the samples were incubated with different polyclonal secondary antibodies at a 1:200 dilution for the primary antibodies detection. After being washed in PBS, the samples were incubated for 1 h at room temperature with DAPI, followed by observation using a confocal laser scanning microscope (LSM 700, Zeiss, CA). The antibodies are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf" class="ext-link">Table S6</a>.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Western Blot</h3><div class="NLM_p last">The protein extractions of THP-1 cells and rat synovial tissues were prepared for western blot analyses of BRD4, p-p65 NF-κB, p65 NF-κB, NLRP3, ASC, Caspase-1 p20, and β-actin. Western blot analysis was performed as in our previous study. The levels of target proteins were determined using a gel imaging system (ChemiScope 2850, Clinx Science Instruments Co., Ltd., Shanghai, China) and normalized to that of the reference band. Primary antibodies are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf" class="ext-link">Table S7</a>.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Hematoxylin-Eosin (H&E) Staining</h3><div class="NLM_p last">The synovium tissue samples were collected immediately, fixed in 4% paraformaldehyde solution for 48 h, and then embedded with paraffin. After that, coronal sections having penumbral cortex of 5 μm thickness were dewaxed according to the standard protocol. Then, the histopathological evaluation was performed by two pathologists in blinded manners.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> General Chemical Procedure</h3><div class="NLM_p last">Unless otherwise specified, reagents were purchased from commercial suppliers and used without further purification. Melting points were determined by X-4 digital display micro-melting-point apparatus (Beijing Tech Instrument Co., Ltd.). NMR spectra were recorded on a Bruker AVANCE AV-600 spectrometer (400 or 300 MHz for 1H, 200 or 150 MHz for <sup>13</sup>C). Mass spectra were obtained on an Agilent 1100 LC/mass selective detector (MSD) mass spectrometer (Agilent) and Q-Tof micro MS (Micromass Company). All reactions were monitored by thin-layer chromatography (TLC, Merck Kieselgel GF254), and spots were visualized with UV light or iodine. The purity of biologically evaluated compounds was ≥95% as determined by high-performance liquid chromatography (HPLC).</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> General Procedure A for Synthesis of Compounds <b>9</b>, <b>12</b>–<b>34</b></h3><div class="NLM_p">A mixture of benzo[<i>cd</i>]indol-2(1<i>H</i>)-one (1 equiv) in anhydrous DMF was stirred at 0 °C. The solution was added to 60% NaH (1.5 equiv) in four batches and stirred for 30 min. Appropriate iodide (1.5 equiv) was added dropwise to the mixture, and the whole mixture was stirred at room temperature for 3 h and then poured into ice water. This mixture was extracted with AcOEt. The organic layer was washed with H<sub>2</sub>O and brine, dried (MgSO<sub>4</sub>), and evaporated to afford <b>7a</b>–<b>7b</b> as a yellow solid (75–90% yield).</div><div class="NLM_p">A suspension of compound <b>7a</b> (500 mg, 2.73 mmol, 1 equiv) in ClSO<sub>2</sub>OH (27.32 mmol, 10 equiv) was stirred at 0 °C for 2 h. The reaction mixture was slowly added dropwise to ice water and extracted with AcOEt. Concentration in vacuo and purification by silica gel column chromatography afforded a yellow intermediate <b>8</b> (0.56 g, 2.0 mmol, yield 74%). To a solution of compound <b>8</b> (150 mg, 0.53 mmol) and aniline (65 mg, 0.69 mmol) in DCM (10 mL) was added pyridine (0.5 mL). The mixture was stirred at rt for 2 h. The reaction mixture was concentrated under reduced pressure. The resulting crude product was purified by silica gel column chromatography (DCM/petroleum ether (PE)) to yield the title compound <b>9</b> (96 mg, 0.28 mmol, yield 54%) as a yellow solid.</div><div class="NLM_p">To a solution of benzo[<i>cd</i>]indol-2(1<i>H</i>)-one or compounds <b>7a</b>–<b>7b</b> (1 equiv) in AcOH was added 69% HNO<sub>3</sub> (1.5 equiv) and heated to 50–65 °C. This mixture was stirred for 4 h. When TLC analysis showed complete conversion of the starting material, the reaction mixture was poured into water filtrated through Celite and the filtrate was concentrated in vacuum. The crude product was purified by silica gel column chromatography (PE/DCM) to yield the title compound as a yellow solid.</div><div class="NLM_p">To a solution of compounds <b>10a</b>–<b>10c</b> (1 equiv) in THF/MeOH (1:1) was added Pd/C (10% mol). This mixture was flushed thrice with argon, then thrice with hydrogen. The flask was charged with hydrogen gas, and the reaction mixture was left to stir for 5 h. The suspension was filtered through Celite and rinsed with EtOAc. The filtrate was evaporated and purified by column chromatography (SiO<sub>2</sub>: 45% EtOAc/petroleum ether) to yield the intermediate compounds <b>11a</b>–<b>11c</b> as a red solid.</div><div class="NLM_p">A reaction mixture of compound <b>11a</b> (1 equiv), halogen (1.5 equiv), and K<sub>2</sub>CO<sub>3</sub> (3 equiv) in anhydrous DMF was stirred at 80 °C for 8 h. After the TLC detection reaction was complete, it was extracted with H<sub>2</sub>O and ethyl acetate (EA). The organic layer was concentrated in vacuum. The crude products were purified by silica gel column chromatography (PE/DCM) to yield the title compounds <b>12–13</b> as a light yellow solid.</div><div class="NLM_p last">To a solution of compounds <b>11a</b>–<b>11c</b> (1 equiv) and proper sulfonyl chloride (1.5 equiv) in DCM (10 mL) was added pyridine (0.5 mL). The mixture was stirred at rt. for 2 h and concentrated under reduced pressure. The resulting crude product was purified by silica gel column chromatography (DCM/PE) to yield the title compounds <b>1525</b> and <b>2734</b> (yield 20–70%) as a yellow solid. The synthesis method of <b>26</b> described here is not described as similar to the preparation of <b>7a</b>.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> General Procedure A for the Synthesis of Intermediate Compound</h3><div id="sec5_16_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 1-Methylbenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>7a</b>)</h4><div class="NLM_p last">Compound <b>7a</b> was prepared according to general procedure A on 41.2 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound (6.48 g, 35.4 mmol, yield 86%). [M + H]<sup>+</sup>: 184.1. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.16 (d, <i>J</i> = 8.1 Hz, 1H), 8.04 (d, <i>J</i> = 7.0 Hz, 1H), 7.79 (t, <i>J</i> = 7.5 Hz, 1H), 7.63 (d, <i>J</i> = 8.4 Hz, 1H), 7.54 (t, <i>J</i> = 7.7 Hz, 1H), 7.13 (d, <i>J</i> = 6.9 Hz, 1H), 3.38 (s, 3H).</div></div><div id="sec5_16_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 1-Ethylbenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>7b</b>)</h4><div class="NLM_p last">Compound <b>7b</b> was prepared according to general procedure A on 23.5 mmol scale. Purification by column chromatography (15% EA/PE) yielded the intermediate compound (3.9 g, 19.7 mmol, yield 84%). [M + H]<sup>+</sup>: 198.2. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.99 (d, <i>J</i> = 7.0 Hz, 1H), 7.91 (d, <i>J</i> = 8.1 Hz, 1H), 7.65–7.58 (m, 1H), 7.47–7.34 (m, 2H), 6.82 (d, <i>J</i> = 6.7 Hz, 1H), 3.92 (q, <i>J</i> = 7.2 Hz, 2H), 1.34 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec5_16_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 1-Methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indole-6-sulfonyl chloride (<b>8</b>)</h4><div class="NLM_p last">Compound <b>8</b> was prepared according to general procedure A on 2.7 mmol scale. Purification by column chromatography (50% PE/DCM) yielded the intermediate compound (0.56 g, 2.0 mmol, yield 74%). [M + H]<sup>+</sup>: 282.2. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.72 (d, <i>J</i> = 8.4 Hz, 1H), 8.34 (d, <i>J</i> = 7.8 Hz, 1H), 8.17 (d, <i>J</i> = 7.1 Hz, 1H), 7.96 (t, <i>J</i> = 7.7 Hz, 1H), 6.97 (d, <i>J</i> = 7.8 Hz, 1H), 3.49 (s, 3H).</div></div><div id="sec5_16_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 1-Methyl-2-oxo-<i>N</i>-phenyl-1,2-dihydrobenzo[<i>cd</i>]indole-6-sulfonamide (<b>9</b>)</h4><div class="NLM_p last">Compound <b>9</b> was prepared according to general procedure A on 0.53 mmol scale. Purification by column chromatography (DCM) yielded the title compound (96 mg, 0.28 mmol, yield 54%). Yellow solid; mp 219–220 °C. HPLC analysis: retention time = 6.047 min; peak area, 97.71%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.59 (s, 1H), 8.70 (d, <i>J</i> = 8.4 Hz, 1H), 8.15 (dd, <i>J</i> = 11.1, 7.5 Hz, 2H), 7.93 (t, <i>J</i> = 7.3 Hz, 2H), 7.17 (dd, <i>J</i> = 19.1, 7.6 Hz, 3H), 7.04 (d, <i>J</i> = 8.1 Hz, 2H), 6.94 (t, <i>J</i> = 7.2 Hz, 1H), 3.34 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.4, 144.6, 137.9, 133.8, 130.9, 129.5, 128.2, 126.4, 125.5, 125.1, 124.2, 124.1, 119.7, 104.2, 26.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>15</sub>N<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 339.0798, found: 339.0803.</div></div><div id="sec5_16_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 6-Nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>10a</b>)</h4><div class="NLM_p last">Compound <b>10a</b> was prepared according to general procedure A on 5.9 mmol scale. Purification by column chromatography (70% DCM/PE) yielded the intermediate compound (0.93 g, 4.5 mmol, yield 74%). [M – H]<sup>−</sup>: 213.0. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.40 (s, 1H), 8.86 (d, <i>J</i> = 8.4 Hz, 1H), 8.61 (d, <i>J</i> = 8.1 Hz, 1H), 8.16 (d, <i>J</i> = 7.0 Hz, 1H), 8.05 (dd, <i>J</i> = 8.4, 7.0 Hz, 1H), 7.11 (d, <i>J</i> = 8.0 Hz, 1H).</div></div><div id="sec5_16_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 1-Methyl-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>10b</b>)</h4><div class="NLM_p last">Compound <b>10a</b> was prepared according to general procedure A on 21.8 mmol scale. Purification by column chromatography (60% DCM/PE) yielded the intermediate compound (3.58 g, 15.7 mmol, yield 72%). [M + H]<sup>+</sup>: 229.2. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.77–8.68 (m, 1H), 8.56 (d, <i>J</i> = 8.2 Hz, 1H), 8.08 (d, <i>J</i> = 7.0 Hz, 1H), 8.00–7.88 (m, 1H), 7.20 (dd, <i>J</i> = 8.1, 1.1 Hz, 1H), 3.35 (s, 3H).</div></div><div id="sec5_16_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 1-Ethyl-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>10c</b>)</h4><div class="NLM_p last">Compound <b>10c</b> was prepared according to general procedure A on 5.1 mmol scale. Purification by column chromatography (50% DCM/PE) yielded the intermediate compound (0.85 g, 3.5 mmol, yield 69%). [M + H]<sup>+</sup>: 243.2. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.78 (dd, <i>J</i> = 8.5, 0.6 Hz, 1H), 8.60 (d, <i>J</i> = 8.1 Hz, 1H), 8.18–8.09 (m, 1H), 7.99 (dd, <i>J</i> = 8.5, 7.0 Hz, 1H), 7.32 (d, <i>J</i> = 8.1 Hz, 1H), 3.94 (q, <i>J</i> = 7.2 Hz, 2H), 1.28 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec5_16_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 6-(Benzylamino)-1-methylbenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>12</b>)</h4><div class="NLM_p last">Compound <b>12</b> was prepared according to general procedure A on 0.53 mmol scale. Purification by column chromatography (DCM) yielded the title compound (96 mg, 0.28 mmol, yield 54%). Yellow solid; mp 135–136 °C. HPLC analysis: retention time = 4.867 min; peak area, 96.24%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.53 (d, <i>J</i> = 8.2 Hz, 1H), 8.00 (d, <i>J</i> = 7.0 Hz, 1H), 7.72 (t, <i>J</i> = 7.6 Hz, 1H), 7.42 (d, <i>J</i> = 7.3 Hz, 2H), 7.32 (t, <i>J</i> = 7.5 Hz, 2H), 7.26–7.11 (m, 2H), 6.85 (d, <i>J</i> = 7.7 Hz, 1H), 6.20 (d, <i>J</i> = 7.7 Hz, 1H), 4.48 (d, <i>J</i> = 5.8 Hz, 2H), 3.28 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 156.8, 138.0, 135.4, 130.5, 129.1 (2C), 128.4, 127.9, 126.8, 126.4, 125.9, 124.7 (2C), 120.4, 113.0, 106.0, 56.3, 26.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>17</sub>N<sub>2</sub>O<sup>+</sup>: 289.1335, found: 289.1340.</div></div><div id="sec5_16_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 1-Methyl-6-((1-phenylethyl)amino)benzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>13</b>)</h4><div class="NLM_p last">Compound <b>13</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (41 mg, 0.18 mmol, yield 36%). Yellow solid; mp 145–146 °C. HPLC analysis: retention time = 4.924 min; peak area, 96.00%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.70 (d, <i>J</i> = 8.2 Hz, 1H), 7.99 (d, <i>J</i> = 6.9 Hz, 1H), 7.73 (dd, <i>J</i> = 8.3, 7.0 Hz, 1H), 7.48–7.39 (m, 2H), 7.28 (t, <i>J</i> = 7.6 Hz, 2H), 7.23–7.11 (m, 1H), 6.81–6.72 (m, 2H), 6.07 (d, <i>J</i> = 7.8 Hz, 1H), 4.64 (t, <i>J</i> = 6.7 Hz, 1H), 3.25 (s, 3H), 1.57 (d, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 146.0, 140.4, 129.1, 128.8 (2C), 127.4, 127.1, 126.9, 126.7, 126.3 (2C), 125.1, 124.1, 121.5, 108.0, 104.9, 52.9, 26.4, 25.0. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>18</sub>N<sub>2</sub>O<sup>+</sup>: 302.1419, found: 303.1487.</div></div><div id="sec5_16_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzamide (<b>14</b>)</h4><div class="NLM_p last">Compound <b>14</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (63 mg, 0.21 mmol, yield 41%). Yellow solid; mp 207–208 °C. HPLC analysis: retention time = 6.102 min; peak area, 96.83%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.50 (s, 1H), 8.21 (s, 1H), 8.15–8.00 (m, 3H), 7.81 (d, <i>J</i> = 8.2 Hz, 1H), 7.73–7.51 (m, 4H), 7.18 (q, <i>J</i> = 6.5 Hz, 1H), 3.39 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.5, 166.7, 137.9, 134.8, 132.1, 129.4, 129.0, 128.9 (2C), 128.3 (2C), 126.6, 125.7, 125.1, 125.0, 124.6, 106.0, 26.7. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>15</sub>N<sub>2</sub>O<sub>2</sub><sup>+</sup>: 303.1128, found: 303.1132.</div></div><div id="sec5_16_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>15</b>)</h4><div class="NLM_p last">Compound <b>15</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (63 mg, 0.22 mmol, yield 41%). Yellow solid; mp 185–186 °C. HPLC analysis: retention time = 5.864 min; peak area, 99.23%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.31 (s, 1H), 8.09 (d, <i>J</i> = 8.4 Hz, 1H), 8.00 (d, <i>J</i> = 6.8 Hz, 1H), 7.72–7.66 (m, 3H), 7.62–7.55 (m, 1H), 7.54–7.45 (m, 2H), 7.16 (d, <i>J</i> = 7.4 Hz, 1H), 7.05 (d, <i>J</i> = 7.5 Hz, 1H), 3.32 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 139.7, 138.5, 133.2, 129.8, 129.5 (2C), 129.2, 128.2, 127.5, 127.2 (2C), 126.4, 126.3, 126.2, 124.7, 105.9, 26.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>15</sub>N<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 339.0798, found: 339.0802.</div></div><div id="sec5_16_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)thiophene-2-sulfonamide (<b>16</b>)</h4><div class="NLM_p last">Compound <b>15</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (108 mg, 0.32 mmol, yield 62%). Yellow solid; mp 183–184 °C. HPLC analysis: retention time = 5.638 min; peak area, 98.76%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.44 (s, 1H), 8.04 (dd, <i>J</i> = 15.9, 7.6 Hz, 2H), 7.85 (dd, <i>J</i> = 5.0, 1.1 Hz, 1H), 7.77–7.65 (m, 1H), 7.39 (dd, <i>J</i> = 3.7, 1.2 Hz, 1H), 7.23 (d, <i>J</i> = 7.6 Hz, 1H), 7.11 (d, <i>J</i> = 7.6 Hz, 1H), 7.07–7.01 (m, 1H), 3.34 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 140.1, 138.7, 133.7, 132.9, 129.30, 128.1 (2C), 127.3, 126.7, 126.4, 126.3, 124.9, 124.7, 105.9, 26.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>13</sub>N<sub>2</sub>O<sub>3</sub>S<sub>2</sub><sup>+</sup>: 345.0362, found: 345.0360.</div></div><div id="sec5_16_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)pyridine-3-sulfonamide (<b>17</b>)</h4><div class="NLM_p last">Compound <b>17</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (89 mg, 0.26 mmol, yield 52%). Yellow solid; mp 212–214 °C. HPLC analysis: retention time = 4.003 min; peak area, 96.27%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.47 (s, 1H), 8.79 (d, <i>J</i> = 1.8 Hz, 1H), 8.75 (dd, <i>J</i> = 4.8, 1.6 Hz, 1H), 8.06 (d, <i>J</i> = 8.5 Hz, 1H), 8.04–7.98 (m, 2H), 7.71 (dd, <i>J</i> = 8.3, 7.0 Hz, 1H), 7.54 (dd, <i>J</i> = 8.1, 4.8 Hz, 1H), 7.18 (d, <i>J</i> = 7.6 Hz, 1H), 7.06 (d, <i>J</i> = 7.6 Hz, 1H), 3.33 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 153.8, 147.59, 138.9, 136.0, 135.3, 129.4, 128.1, 127.0, 126.8, 126.5, 126.2, 125.0, 124.9, 124.6, 106.0, 26.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>14</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup>: 340.0750, found: 340.0757.</div></div><div id="sec5_16_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)quinoline-8-sulfonamide (<b>18</b>)</h4><div class="NLM_p last">Compound <b>18</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (117 mg, 0.30 mmol, yield 52%). Yellow solid; mp 274–275 °C. HPLC analysis: retention time = 6.062 min; peak area, 97.97%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.12 (s, 1H), 9.22 (dd, <i>J</i> = 4.3, 1.8 Hz, 1H), 8.54 (dd, <i>J</i> = 8.4, 1.8 Hz, 1H), 8.29–8.11 (m, 3H), 7.92 (d, <i>J</i> = 6.9 Hz, 1H), 7.77 (dd, <i>J</i> = 8.4, 4.3 Hz, 1H), 7.67–7.52 (m, 2H), 7.02 (d, <i>J</i> = 7.7 Hz, 1H), 6.89 (d, <i>J</i> = 7.7 Hz, 1H), 3.24 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 152.0, 143.35, 138.1, 137.5, 135.7, 134.6, 132.3, 128.89, 128.83, 128.6, 128.0, 126.3, 126.09, 126.04, 124.8, 124.6, 124.5, 123.1, 105.8, 26.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>16</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup>: 390.0907, found: 390.0914.</div></div><div id="sec5_16_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 4-Chloro-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>19</b>)</h4><div class="NLM_p last">Compound <b>19</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (89 mg, 0.24 mmol, yield 47%). Yellow solid; mp 221–222 °C. HPLC analysis: retention time = 5.873 min; peak area, 95.02%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.35 (s, 1H), 8.04 (dd, <i>J</i> = 13.7, 7.6 Hz, 2H), 7.72 (dd, <i>J</i> = 8.4, 6.9 Hz, 1H), 7.67–7.51 (m, 4H), 7.10 (q, <i>J</i> = 7.6 Hz, 2H), 3.33 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 138.8, 138.7, 138.1, 129.7 (2C), 129.3, 129.2 (2C), 128.2, 127.2, 126.6, 126.5, 126.3, 125.1, 124.8, 105.9, 26.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>14</sub>ClN<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 373.0408, found: 373.0406.</div></div><div id="sec5_16_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 4-Chloro-<i>N</i>-(2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>20</b>)</h4><div class="NLM_p last">Compound <b>20</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (82 mg, 0.23 mmol, yield 45%). Yellow solid; mp 182–184 °C. HPLC analysis: retention time = 5.707 min; peak area, 96.61%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.79 (s, 1H), 10.16 (s, 1H), 8.10 (d, <i>J</i> = 8.2 Hz, 1H), 7.97 (d, <i>J</i> = 6.9 Hz, 1H), 7.70 (dd, <i>J</i> = 8.2, 7.1 Hz, 1H), 7.53 (d, <i>J</i> = 8.2 Hz, 2H), 7.28 (d, <i>J</i> = 8.1 Hz, 2H), 7.07 (d, <i>J</i> = 7.6 Hz, 1H), 6.87 (d, <i>J</i> = 7.5 Hz, 1H), 3.38 (s, 3H), 2.31 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.1, 138.7, 138.0, 137.3, 129.7 (2C), 129.3, 129.2 (2C), 128.1, 127.3, 126.9, 126.8, 126.5, 126.4, 124.5, 106.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>12</sub>ClN<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 359.0252, found: 359.0245.</div></div><div id="sec5_16_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 4-Chloro-<i>N</i>-(1-ethyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>21</b>)</h4><div class="NLM_p last">Compound <b>21</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (86 mg, 0.23 mmol, yield 44%). Yellow solid; mp 241–242 °C. HPLC analysis: retention time = 6.208 min; peak area, 98.22%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.38 (s, 1H), 8.05 (dd, <i>J</i> = 21.0, 7.6 Hz, 2H), 7.75–7.71 (m, 1H), 7.70–7.65 (m, 2H), 7.59–7.52 (m, 2H), 7.19–7.06 (m, 2H), 3.86 (q, <i>J</i> = 7.1 Hz, 2H), 1.23 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.9, 138.7, 138.1, 137.7, 129.7, 129.3, 129.2, 128.2, 127.1, 126.6, 126.5, 126.4, 125.1, 124.9, 106.1, 34.9, 14.2. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>16</sub>ClN<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 387.0565, found: 387.0560.</div></div><div id="sec5_16_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 4-Methyl-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>22</b>)</h4><div class="NLM_p last">Compound <b>22</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (100 mg, 0.28 mmol, yield 56%). Yellow solid; mp 237–238 °C. HPLC analysis: retention time = 5.647 min; peak area, 97.95%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.79 (s, 1H), 10.16 (s, 1H), 8.10 (d, <i>J</i> = 8.2 Hz, 1H), 7.97 (d, <i>J</i> = 6.9 Hz, 1H), 7.70 (dd, <i>J</i> = 8.2, 7.1 Hz, 1H), 7.53 (d, <i>J</i> = 8.2 Hz, 2H), 7.28 (d, <i>J</i> = 8.1 Hz, 2H), 7.07 (d, <i>J</i> = 7.6 Hz, 1H), 6.87 (d, <i>J</i> = 7.5 Hz, 1H), 3.38 (s, 3H), 2.31 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.1, 143.5, 137.0, 129.9 (2C), 129.2, 128.3, 127.3 (2C), 127.0, 126.7, 126.3, 126.3, 124.5, 106.6, 21.4. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>17</sub>N<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 353.0954, found: 353.0957.</div></div><div id="sec5_16_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 2-Chloro-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>23</b>)</h4><div class="NLM_p last">Compound <b>23</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (125 mg, 0.33 mmol, yield 56%). Yellow solid; mp 229–230 °C. HPLC analysis: retention time = 5.699 min; peak area, 97.45%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (s, 1H), 8.28 (d, <i>J</i> = 8.3 Hz, 1H), 8.02 (d, <i>J</i> = 6.9 Hz, 1H), 7.83–7.76 (m, 2H), 7.66 (d, <i>J</i> = 7.9 Hz, 1H), 7.62–7.53 (m, 1H), 7.43–7.35 (m, 1H), 7.21 (d, <i>J</i> = 7.6 Hz, 1H), 7.03 (d, <i>J</i> = 7.6 Hz, 1H), 3.30 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 138.5, 137.1, 134.9, 132.2, 131.8, 131.3, 129.3, 128.1, 128.0, 126.9, 126.4, 126.2, 125.7, 124.96, 124.90, 105.8, 26.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>14</sub>ClN<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 373.0408, found: 373.0408.</div></div><div id="sec5_16_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 2-Methyl-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>24</b>)</h4><div class="NLM_p last">Compound <b>24</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (102 mg, 0.29 mmol, yield 57%). Yellow solid; mp 190–192 °C. HPLC analysis: retention time = 6.084 min; peak area, 97.79%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.27 (s, 1H), 8.15 (d, <i>J</i> = 8.3 Hz, 1H), 8.00 (d, <i>J</i> = 6.9 Hz, 1H), 7.77–7.63 (m, 2H), 7.45 (td, <i>J</i> = 7.5, 1.4 Hz, 1H), 7.34 (d, <i>J</i> = 9.5 Hz, 1H), 7.24 (td, <i>J</i> = 7.4, 1.3 Hz, 1H), 7.13 (d, <i>J</i> = 7.6 Hz, 1H), 7.00 (d, <i>J</i> = 7.6 Hz, 1H), 3.31 (s, 3H), 2.59 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 138.4, 138.0, 137.2, 133.3, 132.9, 129.6, 129.3, 128.1, 127.4, 126.7, 126.5, 126.2, 125.7, 124.9, 124.8, 105.9, 26.6, 20.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>17</sub>N<sub>2</sub>O<sub>3</sub>S: 353.0954, found: 353.0957.</div></div><div id="sec5_16_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 2-Methoxy-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>25</b>)</h4><div class="NLM_p last">Compound <b>25</b> was prepared according to general procedure A on 1.53 mmol scale. Purification by column chromatography (DCM) yielded the title compound (327 mg, 0.88 mmol, yield 56%). Yellow solid; mp 187–188 °C. HPLC analysis: retention time = 5.852 min; peak area, 95.15%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.13 (s, 1H), 8.31 (d, <i>J</i> = 8.3 Hz, 1H), 8.00 (d, <i>J</i> = 6.9 Hz, 1H), 7.76 (t, <i>J</i> = 7.6 Hz, 1H), 7.52 (dd, <i>J</i> = 20.1, 7.8 Hz, 2H), 7.24 (d, <i>J</i> = 7.6 Hz, 1H), 7.14 (d, <i>J</i> = 8.3 Hz, 1H), 7.00 (d, <i>J</i> = 7.6 Hz, 1H), 6.91 (t, <i>J</i> = 7.6 Hz, 1H), 3.90 (s, 3H), 3.28 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 156.8, 137.9, 135.4, 130.6, 129.0, 128.3, 127.9, 126.9, 126.4, 125.9, 124.8, 124.7, 120.4, 112.9, 105.9, 56.3, 26.4, HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>17</sub>N<sub>2</sub>O<sub>4</sub>S<sup>+</sup>: 369.0904, found: 369.0909.</div></div><div id="sec5_16_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 2-Methoxy-<i>N</i>-methyl-N-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>26</b>)</h4><div class="NLM_p last">Compound <b>26</b> was prepared according to general procedure A on 0.27 mmol scale. Purification by column chromatography (DCM) yielded the title compound (64 mg, 0.17 mmol, yield 56%). Yellow solid; mp 223–224 °C. HPLC analysis: retention time = 5.975 min; peak area, 97.61%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.11 (d, <i>J</i> = 8.2 Hz, 1H), 8.05 (d, <i>J</i> = 6.9 Hz, 1H), 7.82 (dd, <i>J</i> = 8.3, 7.0 Hz, 1H), 7.64 (dd, <i>J</i> = 8.3, 7.4 Hz, 1H), 7.53 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 7.33 (d, <i>J</i> = 8.3 Hz, 1H), 7.10 (d, <i>J</i> = 7.5 Hz, 1H), 7.04–6.92 (m, 2H), 3.95 (s, 3H), 3.42 (s, 3H), 3.32 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.6, 157.2, 140.0, 135.6, 132.3, 131.3, 129.8, 129.6, 128.5, 128.1, 126.8, 126.6, 125.3, 124.9, 120.5, 113.5, 105.8, 56.5, 41.2, 26.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub>S<sup>+</sup>: 383.1060, found: 383.1063.</div></div><div id="sec5_16_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)-2-(trifluoromethoxy)benzenesulfonamide (<b>27</b>)</h4><div class="NLM_p last">Compound <b>27</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (79 mg, 0.19 mmol, yield 37%). Yellow solid; mp 189–190 °C. HPLC analysis: retention time = 5.655 min; peak area, 95.39%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.57 (s, 1H), 8.20 (d, <i>J</i> = 8.3 Hz, 1H), 8.02 (d, <i>J</i> = 6.9 Hz, 1H), 7.87 (dd, <i>J</i> = 7.9, 1.5 Hz, 1H), 7.80–7.69 (m, 2H), 7.54–7.43 (m, 2H), 7.26 (d, <i>J</i> = 7.6 Hz, 1H), 7.03 (d, <i>J</i> = 7.6 Hz, 1H), 3.33 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 145.6, 138.7, 135.65, 132.0, 131.4, 129.2, 127.9, 127.7, 127.0, 126.5, 126.3, 126.2, 125.0, 124.7, 121.4, 121.1, 105.7, 26.4. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>14</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub>S<sup>+</sup>: 423.0621, found: 423.0622.</div></div><div id="sec5_16_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 2-Ethoxy-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>28</b>)</h4><div class="NLM_p last">Compound <b>28</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (70 mg, 0.18 mmol, yield 36%). Yellow solid; mp 180–182 °C. HPLC analysis: retention time = 6.024 min; peak area, 96.37%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.85 (s, 1H), 8.29 (d, <i>J</i> = 8.2 Hz, 1H), 8.00 (d, <i>J</i> = 6.9 Hz, 1H), 7.74 (dd, <i>J</i> = 8.3, 7.0 Hz, 1H), 7.61 (d, <i>J</i> = 7.9 Hz, 1H), 7.49 (dd, <i>J</i> = 8.8, 7.5 Hz, 1H), 7.23 (d, <i>J</i> = 7.6 Hz, 1H), 7.13 (d, <i>J</i> = 8.4 Hz, 1H), 7.00 (d, <i>J</i> = 7.6 Hz, 1H), 6.91 (t, <i>J</i> = 7.6 Hz, 1H), 4.15 (q, <i>J</i> = 7.0 Hz, 2H), 3.30 (s, 4H), 1.32 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 156.2, 137.9, 135.3, 130.8, 129.0, 128.4, 128.0, 127.1, 126.4, 125.9, 124.8, 124.7, 124.7, 120.2, 113.8, 106.0, 64.7, 26.5, 14.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub>S<sup>+</sup>: 383.1060, found: 383.1057.</div></div><div id="sec5_16_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 2,6-Dichloro-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>29</b>)</h4><div class="NLM_p last">Compound <b>29</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (82 mg, 0.20 mmol, yield 40%). Yellow solid; mp 244–245 °C. HPLC analysis: retention time = 5.599 min; peak area, 97.99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.82 (s, 1H), 8.27 (d, <i>J</i> = 8.2 Hz, 1H), 8.04 (d, <i>J</i> = 6.9 Hz, 1H), 7.79 (dd, <i>J</i> = 8.2, 7.1 Hz, 1H), 7.60 (d, <i>J</i> = 2.0 Hz, 1H), 7.57 (s, 1H), 7.50 (dd, <i>J</i> = 9.3, 6.5 Hz, 1H), 7.25 (d, <i>J</i> = 7.6 Hz, 1H), 7.06 (d, <i>J</i> = 7.6 Hz, 1H), 3.32 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 138.7, 134.9 (2C), 134.8, 134.3, 132.2 (2C), 129.5, 128.0, 126.5 (2C), 126.1, 125.4, 125.0, 124.9, 105.8, 26.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>13</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 407.0018, found: 407.0013.</div></div><div id="sec5_16_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 3-Chloro-2-fluoro-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>30</b>)</h4><div class="NLM_p last">Compound <b>30</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (62 mg, 0.16 mmol, yield 33%). Yellow solid; mp 213–214 °C. HPLC analysis: retention time = 6.190 min; peak area, 95.26%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.65 (s, 1H), 8.25 (d, <i>J</i> = 8.3 Hz, 1H), 8.04 (d, <i>J</i> = 6.9 Hz, 1H), 7.90–7.67 (m, 3H), 7.28 (td, <i>J</i> = 8.7, 2.5 Hz, 1H), 7.20 (d, <i>J</i> = 7.6 Hz, 1H), 7.04 (d, <i>J</i> = 7.6 Hz, 1H), 3.31 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 138.9, 136.1, 129.5, 129.4, 129.2, 129.1, 127.7, 126.9, 126.5, 126.4, 126.3, 126.2, 124.9, 122.0, 121.7, 105.9, 26.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>13</sub>ClFN<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 391.0314, found: 391.0316.</div></div><div id="sec5_16_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 2,6-Difluoro-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>31</b>)</h4><div class="NLM_p last">Compound <b>31</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (72 mg, 0.19 mmol, yield 56%). Yellow solid; mp 238–239 °C. HPLC analysis: retention time = 5.697 min; peak area, 98.58%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.95 (s, 1H), 8.18 (d, <i>J</i> = 8.3 Hz, 1H), 8.04 (d, <i>J</i> = 6.9 Hz, 1H), 7.83–7.72 (m, 1H), 7.66 (dd, <i>J</i> = 14.5, 8.4 Hz, 1H), 7.28 (d, <i>J</i> = 7.6 Hz, 1H), 7.21 (t, <i>J</i> = 9.1 Hz, 2H), 7.09 (d, <i>J</i> = 7.6 Hz, 1H), 3.33 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3 (2C), 161.0, 157.6, 138.9, 136.3, 129.5, 127.8, 126.5, 126.4, 126.1 (2C), 124.9, 124.9, 114.0, 113.7, 105.9, 26.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>12</sub>F<sub>2</sub>N<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 375.0609, found: 375.0618.</div></div><div id="sec5_16_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 4-Fluoro-2-methyl-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>32</b>)</h4><div class="NLM_p last">Compound <b>32</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (66 mg, 0.17 mmol, yield 35%). Yellow solid; mp 214–215 °C. HPLC analysis: retention time = 2.254 min; peak area, 96.97%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.31 (s, 1H), 8.15 (d, <i>J</i> = 8.3 Hz, 1H), 8.02 (d, <i>J</i> = 6.9 Hz, 1H), 7.82–7.66 (m, 2H), 7.25 (dd, <i>J</i> = 9.8, 2.7 Hz, 1H), 7.14 (d, <i>J</i> = 7.8 Hz, 1H), 7.08 (dt, <i>J</i> = 8.5, 4.6 Hz, 1H), 7.02 (d, <i>J</i> = 7.6 Hz, 1H), 3.33 (s, 3H), 2.61 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 141.1 (d, <i>J</i> = 9.4 Hz), 134.4, 134.4, 132.8, 132.6, 129.3, 128.0, 127.2, 126.5, 126.1, 124.8, 119.5 (d, <i>J</i> = 22.3 Hz), 113.5 (d, <i>J</i> = 21.8 Hz), 105.9, 26.5, 20.5, 20.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>16</sub>FN<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 371.0860, found: 371.0864.</div></div><div id="sec5_16_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 2-Chloro-4-fluoro-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>33</b>)</h4><div class="NLM_p last">Compound <b>33</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (63 mg, 0.16 mmol, yield 56%). Yellow solid; mp 228–229 °C. HPLC analysis: retention time = 6.577 min; peak area, 97.37%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.85 (s, 1H), 8.13 (d, <i>J</i> = 8.3 Hz, 1H), 8.04 (d, <i>J</i> = 6.9 Hz, 1H), 7.88–7.80 (m, 1H), 7.80–7.73 (m, 1H), 7.64–7.53 (m, 1H), 7.27 (dd, <i>J</i> = 15.5, 7.8 Hz, 2H), 7.08 (d, <i>J</i> = 7.6 Hz, 1H), 3.40 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 138.8, 134.2, 134.0, 133.9, 133.2, 129.4, 128.1, 126.7, 126.5, 126.4, 126.2, 125.0, 124.9, 119.8 (d, <i>J</i> = 25.9 Hz), 115.3 (d, <i>J</i> = 21.9 Hz), 105.9, 26.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>13</sub>ClFN<sub>2</sub>O<sub>3</sub>S<sup>+</sup>: 391.0314, found: 391.0313.</div></div><div id="sec5_16_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 4-Chloro-2-methoxy-<i>N</i>-(1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>34</b>)</h4><div class="NLM_p last">Compound <b>34</b> was prepared according to general procedure A on 0.51 mmol scale. Purification by column chromatography (DCM) yielded the title compound (47 mg, 0.12 mmol, yield 24%). Yellow solid; mp 152–154 °C. HPLC analysis: retention time = 5.900 min; peak area, 98.79%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.17 (s, 1H), 8.32 (d, <i>J</i> = 8.3 Hz, 1H), 8.01 (d, <i>J</i> = 6.9 Hz, 1H), 7.79 (dd, <i>J</i> = 8.4, 7.0 Hz, 1H), 7.59 (d, <i>J</i> = 8.5 Hz, 1H), 7.32–7.21 (m, 2H), 6.99 (dd, <i>J</i> = 8.5, 2.0 Hz, 2H), 5.76 (d, <i>J</i> = 0.9 Hz, 1H), 3.95 (s, 3H), 3.31 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 157.5, 139.7, 138.2, 131.9, 129.2, 128.3, 127.6, 126.4, 126.1, 126.0, 125.1, 124.9, 124.7, 120.5, 113.5, 105.9, 57.0, 55.3, 26.5. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>16</sub>ClN<sub>2</sub>O<sub>4</sub>S<sup>+</sup>: 403.0514, found: 403.0516.</div></div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> General Procedure B for Synthesis of Compound <b>45</b>–<b>70</b></h3><div class="NLM_p">To a solution of benzo[<i>cd</i>]indol-2(1<i>H</i>)-one (20 g, 111.11 mmol, 1 equiv) in anhydrous DMF (400 mL) was stirred at 0 °C. The solution was added to 60% NaH (6.67, 166.67 mmol, 1.5 equiv) in three batches and stirred for 30 min. Methyl iodide (10.31 mL, 166.67 mmol, 1.5 equiv) was added dropwise to the mixture, and the whole mixture was stirred at room temperature for 3 h and then poured into ice water. This mixture was extracted with AcOEt. The organic layer was washed with H<sub>2</sub>O and brine four times, dried (MgSO<sub>4</sub>), and evaporated to afford <b>35</b> as a red solid (17.12 g, 87.78 mmol, yield 79%).</div><div class="NLM_p">A solution of <b>35</b> (8 g, 41.03 mmol, 1 equiv), palladium acetate (91 mg, 0.41 mmol, 1 mol %), XPhos (585 mg, 1.23 mmol, 3 mol %), K<sub>2</sub>CO<sub>3</sub> (8.5 g, 61.55 mmol, 1.5 equiv), and <i>tert</i>-butyl carbamate (5.75 g, 49.20 mmol, 1.2 equiv) in anhydrous 1,4-dioxane (150 mL) was stirred at 100 °C for 8 h. After cooling the reaction mixture, the product was poured into CH<sub>2</sub>Cl<sub>2</sub> (500 mL), filtered, and washed with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic layers were dried with Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. The crude mixture was purified by flash column chromatography (EtOAc/petroleum ether, 1:4) to afford <b>36</b> as a red solid (27.90 g, 101.11 mmol, yield 91%).</div><div class="NLM_p">To a solution of <b>36</b> (4.0 g, 14.49 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (100 mL) was slowly added a solution of <i>tert</i>-butyl (triphenylphosphoranylidene)acetate (5.56 g, 14.49 mmol, 1 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (40 mL) by using a constant pressure funnel. The solution was stirred for 2 h and then concentrated. The crude mixture was purified by flash column chromatography (DCM/petroleum ether, 1:5) to give <b>37</b> as a 3:1 mixture of isomers (4.45 g, 12.17 mmol, yield 84%).</div><div class="NLM_p">A solution of <b>37</b> (8.0 g, 21.39 mmol) in TFA (20 mL) was heated at 70 °C for 8 h. The mixture was cooled and concentrated. The crude mixture was purified by recrystallization to give <b>38</b> as a yellow solid (3.42 g, 17.11 mmol, yield 80%). A solution of compound <b>38</b> (6.0 g, 30 mmol) in POCl<sub>3</sub> (50 mL) was heated at 90 °C for 2 h. The solution was cooled and then carefully added by a pipette to a stirred solution of ice/water (500 mL). The aqueous layer was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 200 mL). The combined extracts were washed with water (200 mL) and brine (200 mL), dried, filtered, and concentrated. The product was loaded onto silica gel and purified by column chromatography (CH<sub>2</sub>Cl<sub>2</sub>) to give <b>39</b> as a yellow solid (5.43 g, 24.9 mmol, yield 83%). To a mixture of HNO<sub>3</sub> and H<sub>2</sub>SO<sub>4</sub> (40 mL) was added a solution of <b>39</b> in H<sub>2</sub>SO<sub>4</sub> (10 mL) under −15 to −10 °C. The mixture was stirred at −10 °C for 7 h, poured into ice-cold water (300 mL), filtered, and washed with hot water to give <b>40</b> as a yellow solid (4.91 g, 18.67 mmol, yield 75%).</div><div class="NLM_p">To a solution of compound <b>40</b> (1.0 g, 3.80 mmol, 1 equiv) and KHCO<sub>3</sub> (1.14 g, 11.40 mmol, 3 equiv) in THF/MeOH (1:1) was added 10% Pd/C (200 mg, 10% mmol). The mixture reaction was flushed thrice with argon then thrice with hydrogen. The flask was charged with hydrogen gas, and reaction mixture was left to stir for overnight. The suspension was filtered and concentrated to give the crude product <b>43</b> for further reaction. To a solution of the intermediate <b>43</b> (1 equiv) and proper phenylsulfonyl chloride (1.5 equiv) in CH<sub>2</sub>Cl<sub>2</sub> was added pyridine (0.5 mL) and stirred at room temperature for overnight. The mixture was concentrated and purified by silica gel column chromatography to afford the target compounds <b>45</b>–<b>48</b> as a yellow solid (30–60% yield).</div><div class="NLM_p">A solution of compound <b>40</b> (200 mg, 1 equiv) and appropriate amine (2 equiv) in anhydrous 1,4-dioxine (15 mL) was heated at 100–130 °C for 6–20 h. When TLC analysis is complete, the reaction was concentrated and purified by silica gel column chromatography to afford the title compound <b>41a</b>–<b>41l</b> as a yellow solid (40–95% yield).</div><div class="NLM_p">To a mixture of <b>40</b> (1 equiv), Pd(PPh<sub>3</sub>)<sub>4</sub> (0.05 equiv), K<sub>2</sub>CO<sub>3</sub> (3 equiv), H<sub>2</sub>O (10 equiv), and proper boric acid or borate (1.5 equiv) was dissolved 1,4-dioxane. The reaction system was exchanged by N<sub>2</sub> and then heated to 80 °C for 7 h. When TLC analysis showed complete conversion of the starting material, the mixture was extracted with AcOEt and water. The organic layer was washed with H<sub>2</sub>O and brine, dried (MgSO<sub>4</sub>), concentrated, and purified by silica gel column chromatography to afford the title compound <b>41m</b>–<b>41r</b> as a yellow solid (45–90% yield).</div><div class="NLM_p last">To a solution of compounds <b>40</b>, <b>41a</b>–<b>41l</b> (1 equiv) in THF/MeOH (1:1) was added Pd/C (10% mol). The mixture reaction was flushed thrice with argon then thrice with hydrogen. The flask was charged with hydrogen gas, and reaction mixture was left to stir for 5 h. The suspension was filtered through Celite and rinsed with EtOAc. The filtrate was evaporated to afford the crude product <b>43</b>, <b>44a</b>–<b>44l</b> to further reactions. To a solution of the intermediate <b>43</b>, <b>44a</b>–<b>44l</b> (1 equiv), and proper phenylsulfonyl chloride (1.5 equiv) in CH<sub>2</sub>Cl<sub>2</sub> was added pyridine (15 equiv) and stirred at room temperature for overnight. When TLC analysis showed complete conversion of the starting material, the mixture was concentrated and purified by silica gel column chromatography to afford the title compounds <b>49</b>–<b>71</b> as a yellow solid (30–70% yield).</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> General Procedure B for Synthesis of Intermediate Compound</h3><div id="sec5_18_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 4-Chloro-1-methylindoline-2,3-dione (<b>35</b>)</h4><div class="NLM_p last">Compound <b>35</b> was prepared according to general procedure B on 111.11 mmol scale. Purification by column chromatography (80% DCM/PE) yielded the intermediate compound (17.12 g, 87.78 mmol, yield 79%). [M + H]<sup>+</sup>: 196.1. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.63 (t, <i>J</i> = 8.0 Hz, 1H), 7.11 (t, <i>J</i> = 7.5 Hz, 2H), 3.14 (s, 3H).</div></div><div id="sec5_18_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> <i>Tert</i>-butyl (1-Methyl-2,3-dioxoindolin-4-yl)carbamate (<b>36</b>)</h4><div class="NLM_p last">Compound <b>36</b> was prepared according to general procedure B on 41.03 mmol scale. Purification by column chromatography (80% DCM/PE) yielded the intermediate compound (10.34 g, 37.34 mmol, yield 79%). [M + Na]<sup>+</sup>: 299.2. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.71 (s, 1H), 7.61 (d, <i>J</i> = 7.7 Hz, 2H), 6.74 (d, <i>J</i> = 6.8 Hz, 1H), 3.12 (s, 3H), 1.50 (s, 9H).</div></div><div id="sec5_18_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>Tert</i>-butyl 2-(4-((<i>tert</i>-Butoxycarbonyl)amino)-1-methyl-2-oxoindolin-3-ylidene)acetate (<b>37</b>)</h4><div class="NLM_p last">Compound <b>37</b> was prepared according to general procedure B on 14.49 mmol scale. Purification by column chromatography (80% DCM/PE) yielded the intermediate compound (4.45 g, 12.17 mmol, yield 84%). [M + H]<sup>+</sup>: 375.2. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 9.72 (s, 1H), 7.77 (dd, <i>J</i> = 8.6, 0.7 Hz, 1H), 7.39–7.27 (m, 1H), 6.93 (s, 1H), 6.46 (dd, <i>J</i> = 7.6, 0.9 Hz, 1H), 3.21 (s, 3H), 1.59 (s, 9H), 1.51 (s, 9H).</div></div><div id="sec5_18_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 1-Methylpyrrolo[4,3,2-<i>de</i>]quinoline-2,4(1<i>H</i>,5<i>H</i>)-dione (<b>38</b>)</h4><div class="NLM_p last">Compound <b>38</b> was prepared according to general procedure B on 21.39 mmol scale. Purification by recrystallization (80% EtOH/H<sub>2</sub>0) yielded the intermediate compound (3.42 g, 17.11 mmol, yield 80%). [M + H]<sup>+</sup>: 201.07. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.56–7.46 (m, 1H), 7.17 (s, 1H), 6.97 (d, <i>J</i> = 8.4 Hz, 1H), 6.68 (d, <i>J</i> = 7.5 Hz, 1H), 3.36 (s, 3H).</div></div><div id="sec5_18_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 4-Chloro-1-methylpyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>39</b>)</h4><div class="NLM_p last">Compound <b>39</b> was prepared according to general procedure B on 17.11 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a yellow solid (5.43 g, 24.9 mmol, yield 83%). [M + H]<sup>+</sup>: 219. 3. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.83 (s, 1H), 7.70–7.61 (m, 2H), 6.92 (dd, <i>J</i> = 6.2, 1.4 Hz, 1H), 3.43 (s, 3H).</div></div><div id="sec5_18_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 4-Chloro-1-methyl-6-nitropyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>40</b>)</h4><div class="NLM_p last">Compound <b>40</b> was prepared according to general procedure B on 24.9 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a yellow solid (4.91 g, 18.67 mmol, yield 75%). [M + H]<sup>+</sup>: 264.2. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.65 (d, <i>J</i> = 8.0 Hz, 1H), 8.24 (s, 1H), 7.33 (d, <i>J</i> = 8.0 Hz, 1H), 3.38 (s, 3H).</div></div><div id="sec5_18_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 4-(<i>Tert</i>-butylamino)-1-methyl-6-nitropyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41a</b>)</h4><div class="NLM_p last">Compound <b>41a</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by recrystallization (60% EA/PE) yielded the intermediate compound as a yellow solid (147 mg, 0.49 mmol, 43%). [M + H]<sup>+</sup>: 301.1. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.34 (d, <i>J</i> = 8.1 Hz, 1H), 7.81 (s, 1H), 7.28 (s, 1H), 6.91 (d, <i>J</i> = 8.1 Hz, 1H), 3.30 (s, 3H), 1.51 (s, 9H).</div></div><div id="sec5_18_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 4-(Cyclopropylamino)-1-methyl-6-nitropyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41b</b>)</h4><div class="NLM_p last">Compound <b>41b</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by recrystallization (50% EA/PE) yielded the intermediate compound as a yellow solid (220 mg, 0.78 mmol, yield 68%). [M + H]<sup>+</sup>: 285.2. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.34 (d, <i>J</i> = 8.2 Hz, 2H), 7.21 (s, 1H), 6.92 (d, <i>J</i> = 8.1 Hz, 1H), 3.31 (s, 3H), 3.25–2.74 (m, 1H), 0.87–0.79 (m, 2H), 0.61–0.52 (m, 2H).</div></div><div id="sec5_18_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 1-Methyl-4-(((1-methylpiperidin-4-yl)methyl)amino)-6-nitropyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41c</b>)</h4><div class="NLM_p last">Compound <b>41c</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by recrystallization (50% EA/PE) yielded the intermediate compound as a yellow solid (227 mg, 0.64 mmol, yield 56%). [M + H]<sup>+</sup>: 356.3. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.32 (dd, <i>J</i> = 8.1, 1.4 Hz, 1H), 8.15 (t, <i>J</i> = 5.5 Hz, 1H), 7.25 (s, 1H), 6.90 (d, <i>J</i> = 6.4 Hz, 1H), 3.42–3.36 (t, <i>J</i> = 5.9 Hz, 4H), 3.30 (s, 3H), 2.76 (d, <i>J</i> = 11.3 Hz, 2H), 2.13 (s, 3H), 1.87–1.67 (m, 4H), 1.65–1.60 (s, 1H).</div></div><div id="sec5_18_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 1-Methyl-6-nitro-4-(((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)amino)pyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41d</b>)</h4><div class="NLM_p last">Compound <b>41d</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a yellow solid (281 mg, 0.82 mmol, yield 72%). [M + H]<sup>+</sup>: 343.3. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.43–8.06 (m, 2H), 7.24 (d, <i>J</i> = 5.9 Hz, 1H), 6.89 (s, 1H), 3.87 (s, 2H), 3.34–3.18 (m, 7H), 2.09–1.80 (m, 2H), 1.78–1.53 (m, 4H), 1.46–0.96 (m, 6H).</div></div><div id="sec5_18_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 1-Methyl-6-nitro-4-((2-(piperidin-1-yl)ethyl)amino)pyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41e</b>)</h4><div class="NLM_p last">Compound <b>41e</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by recrystallization (60% EA/PE) yielded the intermediate compound as a yellow solid (219 mg, 0.62 mmol, yield 54%). [M + K]<sup>+</sup>: 356.2. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.90 (s, 1H), 8.63 (s, 1H), 8.39 (d, <i>J</i> = 8.1 Hz, 1H), 7.31 (s, 1H), 6.97 (d, <i>J</i> = 8.1 Hz, 1H), 3.88 (d, <i>J</i> = 6.2 Hz, 2H), 3.57 (s, 2H), 3.38 (s, 2H), 3.32 (s, 3H), 2.97 (s, 2H), 1.86–1.26 (m, 6H).</div></div><div id="sec5_18_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 1-Methyl-4-((2-(4-methylpiperazin-1-yl)ethyl)amino)-6-nitropyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41f</b>)</h4><div class="NLM_p last">Compound <b>41f</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by recrystallization (EA) yielded the intermediate compound as a yellow solid (262 mg, 0.71 mmol, yield 62%). [M + Na]<sup>+</sup>: 393.3. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.39 (d, <i>J</i> = 8.1 Hz, 1H), 7.80 (s, 1H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 3.89 (t, <i>J</i> = 4.9 Hz, 3H), 3.32 (s, 3H), 2.44 (t, <i>J</i> = 5.0 Hz, 3H), 2.24 (s, 3H).</div></div><div id="sec5_18_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> (<i>R</i>)-1-methyl-6-nitro-4-((1-(pyridin-2-yl)ethyl)amino)pyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41g</b>)</h4><div class="NLM_p last">Compound <b>41g</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by recrystallization (EA) yielded the intermediate compound as a yellow solid (171 mg, 0.49 mmol, yield 43%). [M + H]<sup>+</sup>: 350.3. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.83–8.59 (m, 2H), 8.52–8.26 (m, 2H), 7.87 (s, 1H), 7.35 (s, 1H), 7.24 (s, 1H), 7.00–6.70 (m, 1H), 3.38–3.30 (m, 1H), 3.28 (s, 3H), 1.60 (s, 3H).</div></div><div id="sec5_18_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 1-Methyl-4-morpholino-6-nitropyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41h</b>)</h4><div class="NLM_p last">Compound <b>41h</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a yellow solid (208 mg, 0.66 mmol, yield 58%). [M + H]<sup>+</sup>: 315.2. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.42 (d, <i>J</i> = 8.1 Hz, 1H), 7.82 (s, 1H), 6.97 (d, <i>J</i> = 8.1 Hz, 1H), 3.88 (t, <i>J</i> = 4.7 Hz, 4H), 3.74 (t, <i>J</i> = 4.8 Hz, 4H), 3.33 (s, 4H).</div></div><div id="sec5_18_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 1-Methyl-4-(4-methylpiperazin-1-yl)-6-nitropyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41i</b>)</h4><div class="NLM_p last">Compound <b>41i</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a yellow solid (269 mg, 0.82 mmol, yield 72%). [M + H]<sup>+</sup>: 328.3. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.40 (d, <i>J</i> = 8.1 Hz, 1H), 7.80 (s, 1H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 3.89 (t, <i>J</i> = 5.1 Hz, 4H), 3.32 (s, 3H), 2.44 (d, <i>J</i> = 5.0 Hz, 4H), 2.24 (s, 3H).</div></div><div id="sec5_18_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 4-(4-(Hydroxymethyl)piperazin-1-yl)-1-methyl-6-nitropyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-one (<b>41j</b>)</h4><div class="NLM_p last">Compound <b>41j</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a yellow solid (191 mg, 0.56 mmol, yield 49%). [M + H]<sup>+</sup>: 343.3. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.35 (d, <i>J</i> = 8.1 Hz, 1H), 7.74 (s, 1H), 6.87 (d, <i>J</i> = 8.0 Hz, 1H), 4.75 (d, <i>J</i> = 13.4 Hz, 2H), 4.52 (s, 1H), 3.30 (s, 3H), 3.05 (t, <i>J</i> = 12.7 Hz, 2H), 1.79 (d, <i>J</i> = 13.4 Hz, 4H), 1.24–1.10 (m, 2H).</div></div><div id="sec5_18_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> Ethyl2-(4-(1-methyl-6-nitro-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-4-yl)piperazin-1-yl)acetate (<b>41k</b>)</h4><div class="NLM_p last">Compound <b>41k</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a yellow solid (241 mg, 0.60 mmol, yield 53%). [M + H]<sup>+</sup>: 399.3. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.39 (d, <i>J</i> = 8.1 Hz, 1H), 7.81 (s, 1H), 6.93 (d, <i>J</i> = 8.1 Hz, 1H), 4.76 (d, <i>J</i> = 7.0 Hz, 2H), 4.07 (q, <i>J</i> = 7.0 Hz, 2H), 3.32 (s, 3H), 3.09 (t, <i>J</i> = 12.6 Hz, 2H), 2.27 (d, <i>J</i> = 6.9 Hz, 2H), 2.06 (s, 1H), 1.79 (d, <i>J</i> = 12.5 Hz, 2H), 1.27–1.16 (m, 5H).</div></div><div id="sec5_18_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> <i>Tert</i>-butyl-4-(1-methyl-6-nitro-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-4-yl)piperazine-1-carboxylate (<b>41l</b>)</h4><div class="NLM_p last">Compound <b>41l</b> was prepared according to general procedure B on 1.14 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a yellow solid (212 mg, 0.51 mmol, yield 45%). [M-Boc]<sup>+</sup>: 314.3. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.42 (d, <i>J</i> = 8.0 Hz, 1H), 7.44 (s, 1H), 6.64 (d, <i>J</i> = 8.0 Hz, 1H), 3.92 (d, <i>J</i> = 5.7 Hz, 4H), 3.63 (dd, <i>J</i> = 6.5, 4.0 Hz, 4H), 3.40 (s, 3H), 1.50 (s, 9H).</div></div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> General Procedure B for Synthesis of Title Compound <b>45</b>–<b>70</b></h3><div id="sec5_19_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 2-Chloro-4-fluoro-<i>N</i>-(1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>45</b>)</h4><div class="NLM_p last">Compound <b>45</b> was prepared according to general procedure B on 0.50 mmol scale. Purification by column chromatography (DCM) yielded the title compound <b>45</b> (74 mg, 0.19 mmol, yield 38%). Yellow solid; mp 177–178 °C. HPLC analysis: retention time = 5.843 min; peak area, 100.00%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.57 (s, 1H), 9.10 (d, <i>J</i> = 4.4 Hz, 1H), 7.97 (q, <i>J</i> = 5.3 Hz, 2H), 7.66 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.50 (d, <i>J</i> = 7.6 Hz, 1H), 7.29 (td, <i>J</i> = 8.5, 2.6 Hz, 1H), 7.13 (d, <i>J</i> = 7.6 Hz, 1H), 3.32 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.7, 153.3, 140.5, 138.6, 134.3, 128.1, 127.1, 120.1, 119.8, 119.5, 119.1, 118.5, 115.1, 114.8, 106.9, 27.1, 26.9. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>12</sub>ClFN<sub>3</sub>O<sub>3</sub>S<sup>+</sup>: 392.0266, found: 392.0269.</div></div><div id="sec5_19_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 4-Chloro-<i>N</i>-(1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>46</b>)</h4><div class="NLM_p last">Compound <b>46</b> was prepared according to general procedure B on 0.50 mmol scale. Purification by column chromatography (DCM) yielded the title compound <b>46</b> (59 mg, 0.16 mmol, yield 32%). Yellow solid; mp 213–214 °C. HPLC analysis: retention time = 5.902 min; peak area, 98.39%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.05 (s, 1H), 7.70 (t, <i>J</i> = 8.2 Hz, 4H), 7.58–7.49 (m, 6H), 7.37 (s, 1H), 7.19 (d, <i>J</i> = 7.6 Hz, 1H), 7.02 (d, <i>J</i> = 7.7 Hz, 1H), 3.93 (s, 3H), 3.31 (d, <i>J</i> = 7.8 Hz, 6H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 153.2, 140.2, 139.3, 138.3, 137.9, 134.2, 129.5 (2C), 129.2 (2C), 128.4, 126.5, 119.1, 118.4, 107.0, 26.9. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>13</sub>ClN<sub>3</sub>O<sub>3</sub>S<sup>+</sup>: 374.0361, found: 374.0363.</div></div><div id="sec5_19_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> 4-Methyl-<i>N</i>-(1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>47</b>)</h4><div class="NLM_p last">Compound <b>47</b> was prepared according to general procedure B on 0.50 mmol scale. Purification by column chromatography (DCM) yielded the title compound <b>47</b> (91 mg, 0.26 mmol, yield 52%). Yellow solid; mp 194–195 °C. HPLC analysis: retention time = 5.877 min; peak area, 96.16%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.42 (s, 1H), 9.12 (d, <i>J</i> = 4.4 Hz, 1H), 7.96 (d, <i>J</i> = 4.4 Hz, 1H), 7.70 (d, <i>J</i> = 8.1 Hz, 2H), 7.52 (d, <i>J</i> = 7.7 Hz, 1H), 7.26 (d, <i>J</i> = 8.0 Hz, 2H), 7.12 (d, <i>J</i> = 7.7 Hz, 1H), 3.31 (s, 3H), 2.28 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 153.1, 143.5, 139.9, 137.7, 137.6, 134.3, 129.8, 129.0, 127.3, 124.6, 118.4, 107.1, 26.9, 21.3. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup>: 354.0907, found: 354.0905.</div></div><div id="sec5_19_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> 2-Methoxy-<i>N</i>-(1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>48</b>)</h4><div class="NLM_p last">Compound <b>48</b> was prepared according to general procedure B on 0.50 mmol scale. Purification by column chromatography (DCM) yielded the title compound <b>48</b> (79 mg, 0.215 mmol, yield 43%). Yellow solid; mp 229–230 °C. HPLC analysis: retention time = 5.838 min; peak area, 95.18%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.50 (s, 1H), 9.17 (d, <i>J</i> = 4.4 Hz, 1H), 7.99 (d, <i>J</i> = 4.4 Hz, 1H), 7.71 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.59–7.44 (m, 2H), 7.16–7.03 (m, 2H), 6.96 (td, <i>J</i> = 7.6, 1.0 Hz, 1H), 3.77 (s, 3H), 3.30 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 153.1, 143.5, 139.9, 137.7, 137.6, 134.3, 129.8 (2C), 129.0, 127.3 (2C), 124.6, 118.4, 107.1, 26.9, 21.3. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>3</sub>O4S<sup>+</sup>: 370.0856, found: 370.0856.</div></div><div id="sec5_19_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> 4-Chloro-<i>N</i>-(4-chloro-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>49</b>)</h4><div class="NLM_p last">Compound <b>49</b> was prepared according to general procedure B on 0.43 mmol scale. Purification by column chromatography (50% EA/PE) yielded the title compound (73 mg, 0.18 mmol, yield 42%). Yellow solid; mp 197–198 °C. HPLC analysis: retention time = 4.537 min; peak area, 95.20%. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.93 (s, 1H), 7.78 (t, <i>J</i> = 4.3 Hz, 4H), 7.34 (d, <i>J</i> = 8.3 Hz, 2H), 6.83 (d, <i>J</i> = 7.7 Hz, 1H), 3.38 (s, 3H). <sup>13</sup>C NMR (75 MHz, chloroform-<i>d</i>) δ 165.1, 153.1, 139.8, 137.6, 137.3, 137.1, 136.8, 129.3 (2C), 128.8 (2C), 127.4, 121.9, 120.2, 118.2, 106.4, 26.9. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>12</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup>: 407.9971, found: 407.9969.</div></div><div id="sec5_19_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> <i>N</i>-(4-chloro-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)-2-methoxybenzenesulfonamide (<b>50</b>)</h4><div class="NLM_p last">Compound <b>50</b> was prepared according to general procedure B on 0.43 mmol scale. Purification by column chromatography (50% EA/PE) yielded the title compound (54 mg, 0.13 mmol, yield 31%). Yellow solid; mp 214–215 °C. HPLC analysis: retention time = 6.046 min; peak area, 95.02%. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.57 (s, 1H), 7.87–7.74 (m, 3H), 7.41 (dd, <i>J</i> = 8.3, 7.4 Hz, 1H), 6.90 (dd, <i>J</i> = 8.2, 6.8 Hz, 2H), 6.78 (d, <i>J</i> = 7.8 Hz, 1H), 4.05 (s, 3H), 3.34 (s, 3H). <sup>13</sup>C NMR (75 MHz, chloroform-<i>d</i>) δ 165.2, 156.4, 152.6, 137.6, 137.4, 136.2, 135.0, 130.6, 128.4, 125.9, 121.6, 120.0, 119.8, 118.1, 111.7, 106.7, 56.4, 26.8. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>15</sub>ClN<sub>3</sub>O<sub>4</sub>S<sup>+</sup>: 404.0466, found: 404.0474.</div></div><div id="sec5_19_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> <i>N</i>-(4-(<i>Tert</i>-butylamino)-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)-2-methoxybenzenesulfonamide (<b>51</b>)</h4><div class="NLM_p last">Compound <b>51</b> was prepared according to general procedure B on 0.37 mmol scale. Purification by column chromatography (50% EA/PE) yielded the title compound (57 mg, 0.13 mmol, yield 35%). Yellow solid; mp 252–254 °C. HPLC analysis: retention time = 4.768 min; peak area, 95.63%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.38 (s, 1H), 7.88 (dd, <i>J</i> = 7.8, 1.6 Hz, 1H), 7.57 (dd, <i>J</i> = 9.0, 7.3 Hz, 1H), 7.42 (s, 1H), 7.21 (t, <i>J</i> = 3.8 Hz, 2H), 7.15 (d, <i>J</i> = 8.4 Hz, 1H), 7.07 (t, <i>J</i> = 7.6 Hz, 1H), 6.59 (d, <i>J</i> = 7.7 Hz, 1H), 5.76 (s, 1H), 3.68 (s, 3H), 3.19 (s, 3H), 1.50 (s, 9H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.4, 159.5, 156.8, 136.5, 135.7, 134.8, 130.8, 126.7, 126.6, 120.6, 119.2, 113.7, 111.9, 111.8, 101.7, 56.8, 55.3, 51.9, 28.7 (3C), 26.8. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 441.1591, found: 441.1594.</div></div><div id="sec5_19_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> <i>N</i>-(4-(cyclopropylamino)-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)-2-methoxybenzenesulfonamide (<b>52</b>)</h4><div class="NLM_p last">Compound <b>52</b> was prepared according to general procedure B on 0.39 mmol scale. Purification by column chromatography (50% EA/PE) yielded the title compound (60 mg, 0.14 mmol, yield 36%). Yellow solid; mp 217–218 °C. HPLC analysis: retention time = 2.508 min; peak area, 96.68%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.60 (s, 1H), 7.94 (d, <i>J</i> = 3.6 Hz, 1H), 7.85 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.54 (dd, <i>J</i> = 8.4, 7.4 Hz, 1H), 7.31 (d, <i>J</i> = 7.7 Hz, 1H), 7.20 (s, 1H), 7.13–7.01 (m, 2H), 6.59 (d, <i>J</i> = 7.8 Hz, 1H), 3.79 (s, 3H), 3.19 (s, 3H), 2.99-2.94 (m, 1H), 0.86–0.83 (m, 2H), 0.60–0.49 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3, 161.2, 156.6, 136.9, 136.3, 135.7 (2C), 130.7 (2C), 126.6, 126.2, 120.5, 118.3, 113.3, 112.7, 102.0, 56.7, 26.8, 24.7, 14.4, 7.1. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>21</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 425.1278, found: 425.1280.</div></div><div id="sec5_19_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> 2-Methoxy-<i>N</i>-(1-methyl-4-(((1-methylpiperidin-4-yl)methyl)amino)-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>53</b>)</h4><div class="NLM_p last">Compound <b>53</b> was prepared according to general procedure B on 0.31 mmol scale. Purification by column chromatography (70% EA/PE) yielded the title compound (65 mg, 0.13 mmol, yield 43%). Yellow solid; mp 230–232 °C. HPLC analysis: retention time = 8.380 min; peak area, 95.78%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.65 (s, 1H), 7.82 (d, <i>J</i> = 7.8 Hz, 1H), 7.72 (t, <i>J</i> = 5.3 Hz, 1H), 7.54 (t, <i>J</i> = 7.8 Hz, 1H), 7.32 (d, <i>J</i> = 7.7 Hz, 1H), 7.19 (s, 1H), 7.12 (d, <i>J</i> = 8.3 Hz, 1H), 7.03 (t, <i>J</i> = 7.6 Hz, 1H), 6.60 (d, <i>J</i> = 7.8 Hz, 1H), 3.75 (s, 3H), 3.19 (s, 3H), 2.90 (d, <i>J</i> = 11.1 Hz, 2H), 2.26 (s, 3H), 2.03 (t, <i>J</i> = 11.8 Hz, 2H), 1.94–1.49 (m, 4H), 1.47–1.13 (m, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.4, 160.2, 156.7, 137.2, 136.5, 135.6 (2C), 135.1, 130.5, 126.8, 126.5, 120.5 (2C), 119.9, 113.4, 101.8, 56.7, 55.2, 47.1, 46.0, 34.8, 29.9, 26.8. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>30</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup>: 496.2013, found: 496.2020.</div></div><div id="sec5_19_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> 2-Methoxy-<i>N</i>-(1-methyl-2-oxo-4-(((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)amino)-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>54</b>)</h4><div class="NLM_p last">Compound <b>54</b> was prepared according to general procedure B on 0.32 mmol scale. Purification by column chromatography (70% EA/PE) yielded the title compound (54 mg, 0.11 mmol, yield 35%). Yellow solid; mp 289–290 °C. HPLC analysis: retention time = 5.951 min; peak area, 95.30%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.71 (s, 1H), 7.83 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.70 (t, <i>J</i> = 5.4 Hz, 1H), 7.62–7.49 (m, 1H), 7.32 (d, <i>J</i> = 7.7 Hz, 1H), 7.19 (s, 1H), 7.12 (d, <i>J</i> = 8.4 Hz, 1H), 7.03 (t, <i>J</i> = 7.6 Hz, 1H), 6.60 (d, <i>J</i> = 7.8 Hz, 1H), 3.96–3.84 (m, 2H), 3.75 (s, 3H), 3.43–3.37 (m, 2H), 3.30 (dd, <i>J</i> = 11.6, 2.0 Hz, 2H), 3.19 (s, 3H), 1.95–1.80 (m, 1H), 1.70 (dd, <i>J</i> = 12.4, 3.1 Hz, 2H), 1.36–1.27 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.4, 160.2, 156.7 (2C), 137.3, 136.5, 135.6, 135.2, 130.5, 126.9, 126.5, 120.5, 120.1, 113.4, 112.3, 101.8, 67.3 (2C), 56.7, 47.4, 34.9, 31.1 (2C), 26.8. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd forC<sub>24</sub>H<sub>27</sub>N<sub>4</sub>O<sub>5</sub>S<sup>+</sup>: 483.1697, found: 483.1702.</div></div><div id="sec5_19_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> 2-Methoxy-<i>N</i>-(1-methyl-2-oxo-4-((2-(piperidin-1-yl)ethyl)amino)-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>55</b>)</h4><div class="NLM_p last">Compound <b>55</b> was prepared according to general procedure B on 0.31 mmol scale. Purification by column chromatography (70% EA/PE) yielded the title compound (55 mg, 0.11 mmol, yield 36%). Yellow solid; mp 220–221 °C. HPLC analysis: retention time = 6.393 min; peak area, 96.875%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.72 (s, 0H), 7.82 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.69 (s, 1H), 7.54 (dd, <i>J</i> = 8.8, 7.4 Hz, 1H), 7.30 (d, <i>J</i> = 7.7 Hz, 1H), 7.22 (s, 1H), 7.12 (d, <i>J</i> = 8.3 Hz, 1H), 7.03 (td, <i>J</i> = 7.6, 1.0 Hz, 1H), 6.61 (d, <i>J</i> = 7.7 Hz, 1H), 3.75 (s, 3H), 3.69–3.54 (m, 2H), 3.19 (s, 3H), 2.87–2.26 (m, 6H), 1.70–1.29 (m, 6H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.4, 159.9, 156.7, 137.3, 136.5, 135.5 (2C), 135.2, 130.5, 126.9, 126.7, 120.5 (2C), 113.4, 112.1, 101.9, 56.7 (4C), 54.4, 39.5, 26.8 (3C).HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>30</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup>: 496.2013, found: 496.2010.</div></div><div id="sec5_19_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> 2-Methoxy-<i>N</i>-(1-methyl-4-((2-(4-methylpiperazin-1-yl)ethyl)amino)-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>56</b>)</h4><div class="NLM_p last">Compound <b>56</b> was prepared according to general procedure B on 0.29 mmol scale. Purification by column chromatography (EA) yielded the title compound (55 mg, 0.11 mmol, yield 37%). Yellow solid; mp 117–118 °C. HPLC analysis: retention time = 6.501 min; peak area, 96.108%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.67 (s, 1H), 7.83 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.61 (s, 1H), 7.54 (dd, <i>J</i> = 8.9, 7.4Hz, 1H), 7.31 (d, <i>J</i> = 7.8 Hz, 1H), 7.22 (s, 1H), 7.13–7.09 (m, 1H), 7.03 (td, <i>J</i> = 7.6, 1.0 Hz, 1H), 6.60 (d, <i>J</i> = 7.8 Hz, 1H), 3.76 (s, 3H), 3.56 (t, <i>J</i> = 6.1 Hz, 2H), 3.19 (s, 3H), 2.57 (t, <i>J</i> = 6.6 Hz, 2H), 2.51–2.25 (m, 8H), 2.16 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.4, 160.0, 156.7 (2C), 137.2, 136.5, 135.6, 135.2, 130.5, 126.7, 126.6, 120.5, 119.8, 113.3, 112.4, 101.9, 56.9, 56.7 (2C), 55.1 (2C), 53.3, 46.2, 40.9-39.1 (1C). 26.8. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>31</sub>N<sub>6</sub>O<sub>4</sub>S<sup>+</sup>: 511.2122, found: 511.2117.</div></div><div id="sec5_19_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> (<i>R</i>)-2-methoxy-<i>N</i>-(1-methyl-2-oxo-4-((1-(pyridin-2-yl)ethyl)amino)-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>57</b>)</h4><div class="NLM_p last">Compound <b>57</b> was prepared according to general procedure B on 0.31 mmol scale. Purification by column chromatography (50% EA/PE) yielded the title compound (71 mg, 0.14 mmol, yield 46%). Yellow solid; mp 241–242 °C. HPLC analysis: retention time = 4.065 min; peak area, 97.71%. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.60 (d, <i>J</i> = 4.2 Hz, 1H), 8.25 (s, 1H), 7.99 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.73 (t, <i>J</i> = 7.7 Hz, 1H), 7.51–7.35 (m, 3H), 7.26–7.17 (m, 1H), 7.11 (s, 1H), 6.99 (t, <i>J</i> = 7.6 Hz, 1H), 6.84 (d, <i>J</i> = 8.3 Hz, 1H), 6.60 (d, <i>J</i> = 6.9 Hz, 1H), 6.39 (d, <i>J</i> = 7.8 Hz, 1H), 5.40 (p, <i>J</i> = 6.8 Hz, 1H), 3.79 (s, 3H), 3.25 (s, 3H), 1.65 (d, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.5, 160.2, 156.8 (2C), 137.3, 136.7, 136.3, 135.4 (2C), 130.4 (2C), 127.2 (2C), 126.9 (2C), 121.9, 120.4 (2C), 113.3, 107.2, 102.7, 66.4, 56.6, 45.9, 26.9.HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>24</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup>: 490.1544, found: 490.1532.</div></div><div id="sec5_19_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> 2-Methoxy-<i>N</i>-(1-methyl-4-morpholino-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>58</b>)</h4><div class="NLM_p last">Compound <b>58</b> was prepared according to general procedure B on 0.35 mmol scale. Purification by column chromatography (DCM) yielded the title compound (72 mg, 0.16 mmol, yield 45%). Yellow solid; mp 247–248 °C. HPLC analysis: retention time = 7.919 min; peak area, 95.52%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.03 (d, <i>J</i> = 2.3 Hz, 1H), 7.78 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.62 (d, <i>J</i> = 2.4 Hz, 1H), 7.53 (ddd, <i>J</i> = 8.8, 7.5, 1.8 Hz, 1H), 7.39 (dd, <i>J</i> = 7.7, 1.0 Hz, 1H), 7.10 (dd, <i>J</i> = 8.5, 0.9 Hz, 1H), 7.00 (td, <i>J</i> = 7.6, 1.0 Hz, 1H), 6.68 (dd, <i>J</i> = 7.7, 3.1 Hz, 1H), 3.74 (s, 8H), 3.72 (s, 3H), 3.22 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 160.2, 156.8, 137.3, 136.7, 136.3, 135.4, 130.4, 127.2, 126.9, 121.8, 120.4, 113.3, 112.9, 107.2, 102.7, 66.4 (2C), 56.6, 45.9 (2C), 26.91. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>23</sub>N<sub>4</sub>O<sub>5</sub>S<sup>+</sup>: 455.1384, found: 455.138.</div></div><div id="sec5_19_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> 4-Chloro-<i>N</i>-(1-methyl-4-morpholino-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>59</b>)</h4><div class="NLM_p last">Compound <b>59</b> was prepared according to general procedure B on 0.35 mmol scale. Purification by column chromatography (70% EA/PE) yielded the title compound (69 mg, 0.15 mmol, yield 43%). Yellow solid; mp 223–225 °C. HPLC analysis: retention time = 4.882 min; peak area, 98.26%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.08 (s, 1H), 7.70 (s, 1H), 7.67 (s, 1H), 7.61 (s, 1H), 7.50 (s, 1H), 7.47 (s, 1H), 7.45 (d, <i>J</i> = 7.7 Hz, 1H), 6.81 (d, <i>J</i> = 7.7 Hz, 1H), 3.95–3.69 (m, 4H), 3.27 (s, 3H), 2.80–2.55 (s, 4H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.8, 159.7, 139.7, 138.9, 138.4, 137.7, 136.2, 129.4, 129.0, 128.3, 128.1, 127.9, 125.7, 113.0, 107.2, 102.9, 54.2 (2C), 44.6 (2C), 26.9. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>20</sub>ClN<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 459.0888, found: 459.0892.</div></div><div id="sec5_19_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> 2-Methoxy-<i>N</i>-(1-methyl-4-(4-methylpiperazin-1-yl)-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>60</b>)</h4><div class="NLM_p last">Compound <b>60</b> was prepared according to general procedure B on 0.34 mmol scale. Purification by column chromatography (90% EA/PE) yielded the title compound (57 mg, 0.12 mmol, yield 36%). Yellow solid; mp 227–248 °C. HPLC analysis: retention time = 5.148 min; peak area, 97.19%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.98 (s, 1H), 7.78 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.63 (s, 1H), 7.52 (ddd, <i>J</i> = 8.8, 7.4, 1.8 Hz, 1H), 7.39 (d, <i>J</i> = 7.7 Hz, 1H), 7.10 (d, <i>J</i> = 8.3 Hz, 1H), 7.00 (t, <i>J</i> = 7.6 Hz, 1H), 6.67 (d, <i>J</i> = 7.7 Hz, 1H), 3.77 (t, <i>J</i> = 4.8 Hz, 4H), 3.73 (s, 3H), 3.22 (s, 3H), 2.44 (t, <i>J</i> = 5.0 Hz, 4H), 2.25 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 160.1, 156.8, 136.7, 136.3, 135.4 (2C), 130.3, 127.2, 126.8, 121.8, 120.4 (2C), 113.3, 112.8, 102.6, 56.6, 54.9 (2C), 46.1, 45.4 (2C), 26.9. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>26</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup>: 468.1700, found: 468.1708.</div></div><div id="sec5_19_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> 4-Chloro-<i>N</i>-(1-methyl-4-(4-methylpiperazin-1-yl)-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>61</b>)</h4><div class="NLM_p last">Compound <b>61</b> was prepared according to general procedure B on 0.34 mmol scale. Purification by column chromatography (80% EA/PE) yielded the title compound (48 mg, 0.10 mmol, yield 30%). Yellow solid; mp 247–248 °C. HPLC analysis: retention time = 4.331 min; peak area, 97.97%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.03 (s, 1H), 7.67 (d, <i>J</i> = 8.6 Hz, 2H), 7.55 (s, 1H), 7.50 (s, 1H), 7.48 (d, <i>J</i> = 1.9 Hz, 1H), 7.44 (d, <i>J</i> = 7.6 Hz, 1H), 6.79 (d, <i>J</i> = 7.7 Hz, 1H), 3.76–3.62 (m, 8H), 3.26 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.8, 160.0, 139.7, 139.0, 138.4, 137.7, 136.1, 129.4 (2C), 129.0 (2C), 128.5, 125.7, 113.1, 106.9, 102.8, 66.5 (2C), 45.8 (3C), 26.9. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>23</sub>ClN<sub>5</sub>O<sub>3</sub>S<sup>+</sup>: 472.1205, found: 472.1206.</div></div><div id="sec5_19_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> <i>N</i>-(4-(4-(hydroxymethyl)piperidin-1-yl)-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)-2-methoxybenzenesulfonamide (<b>62</b>)</h4><div class="NLM_p last">Compound <b>62</b> was prepared according to general procedure B on 0.32 mmol scale. Purification by column chromatography (80% EA/PE) yielded the title compound (48 mg, 0.10 mmol, yield 31%). Yellow solid; mp 191–192 °C. HPLC analysis: retention time = 4.297 min; peak area, 96.22%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.92 (s, 1H), 8.27 (s, 1H), 7.80 (d, <i>J</i> = 7.8 Hz, 1H), 7.63 (s, 1H), 7.52 (dd, <i>J</i> = 8.9, 7.5 Hz, 1H), 7.37 (d, <i>J</i> = 7.7 Hz, 1H), 7.16–6.95 (m, 2H), 6.64 (d, <i>J</i> = 7.7 Hz, 1H), 4.70 (d, <i>J</i> = 13.2 Hz, 2H), 4.02 (s, 2H), 3.73 (s, 3H), 3.34 (s, 1H), 3.21 (s, 3H), 2.99 (t, <i>J</i> = 12.1 Hz, 2H), 2.04–1.95 (m, 1H), 1.91-1.80 (m, 2H), 1.33–1.12 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 162.6, 159.8, 156.8, 137.4, 136.6, 136.3, 135.4, 130.4, 127.1, 126.7, 121.3, 120.4, 113.3, 107.5, 102.3, 67.7, 56.6, 45.4 (2C), 35.6, 28.4, 26.8 (2C). HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>27</sub>N<sub>4</sub>O<sub>5</sub>S+: 483.1697, found: 483.1698.</div></div><div id="sec5_19_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> Ethyl2-(1-(6-((2-methoxyphenyl)sulfonamido)-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-4-yl)piperidin-4-yl)acetate (<b>63</b>)</h4><div class="NLM_p last">Compound <b>63</b> was prepared according to general procedure B on 0.27 mmol scale. Purification by column chromatography (60% EA/PE) yielded the title compound (50 mg, 0.09 mmol, yield 34%). Yellow solid; mp 195–196 °C. HPLC analysis: retention time = 3.833 min; peak area, 97.88%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.91 (s, 1H), 7.79 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.63 (s, 1H), 7.52 (dd, <i>J</i> = 8.3, 7.4 Hz, 1H), 7.37 (d, <i>J</i> = 7.7 Hz, 1H), 7.10 (dd, <i>J</i> = 8.5, 1.0 Hz, 1H), 7.00 (td, <i>J</i> = 7.6, 1.0 Hz, 1H), 6.65 (d, <i>J</i> = 7.7 Hz, 1H), 4.09 (q, <i>J</i> = 7.1 Hz, 2H), 3.72 (s, 3H), 3.21 (s, 3H), 3.04–2.91 (m, 2H), 2.51 (p, <i>J</i> = 1.8 Hz, 2H), 2.28 (d, <i>J</i> = 7.0 Hz, 2H), 2.04 (dd, <i>J</i> = 7.3, 3.6 Hz, 1H), 1.78 (d, <i>J</i> = 12.8 Hz, 2H), 1.20 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.3 (2C), 166.6, 159.9, 156.8, 137.4, 136.6, 136.3, 135.5, 130.4, 127.1, 126.7, 121.2, 120.4, 113.3, 107.5, 102.3, 60.2, 56.6, 45.7 (2C), 40.7, 33.3, 31.6 (2C), 26.9, 14.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>31</sub>N<sub>4</sub>O<sub>6</sub>S<sup>+</sup>: 539.1959, found: 539.1957.</div></div><div id="sec5_19_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> <i>Tert</i>-butyl4-(6-((2-methoxyphenyl)sulfonamido)-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-4-yl)piperazine-1-carboxylate (<b>64</b>)</h4><div class="NLM_p last">Compound <b>64</b> was prepared according to general procedure B on 0.26 mmol scale. Purification by column chromatography (50% EA/PE) yielded the title compound (77 mg, 0.14 mmol, yield 53%). Yellow solid; mp 204–205 °C. HPLC analysis: retention time = 4.523 min; peak area, 98.21%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.07 (s, 1H), 7.79 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.63 (s, 1H), 7.52 (dd <i>J</i> = 8.3, 7.4 Hz, 1H), 7.39 (d, <i>J</i> = 7.7 Hz, 1H), 7.14–6.94 (m, 2H), 6.67 (d, <i>J</i> = 7.7 Hz, 1H), 3.85–3.75 (m, 4H), 3.51–3.41 (m, 4H), 3.34 (s, 2H), 3.22 (s, 3H), 1.45 (s, 9H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 159.8, 156.8, 154.4, 137.3, 136.67, 136.40, 135.4, 130.4, 127.2, 126.9, 121.7, 120.4, 113.3, 112.8, 107.4, 102.7, 79.5, 56.6 (3C), 45.2 (2C), 28.5 (3C), 26.9. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>32</sub>N<sub>5</sub>O<sub>6</sub>S<sup>+</sup>: 554.2068, found: 554.2061.</div></div><div id="sec5_19_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> <i>N</i>-(4-(furan-2-yl)-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)-2-methoxybenzenesulfonamide (<b>65</b>)</h4><div class="NLM_p last">Compound <b>65</b> was prepared according to general procedure B on 0.38 mmol scale. Purification by column chromatography (50% EA/PE) yielded the title compound (74 mg, 0.17 mmol, yield 45%). Yellow solid; mp 251–252 °C. HPLC analysis: retention time = 4.423 min; peak area, 96.72%. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.78 (s, 1H), 8.31 (s, 1H), 7.97 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.75–7.63 (m, 2H), 7.45–7.33 (m, 2H), 7.02–6.90 (m, 1H), 6.78 (d, <i>J</i> = 8.5 Hz, 1H), 6.72–6.62 (m, 2H), 3.81 (s, 3H), 3.35 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 156.6, 153.2, 151.2, 146.2, 137.9, 136.7, 135.7, 135.6, 130.5, 128.8, 126.4, 120.9, 120.4, 117.6, 114.7, 113.38, 113.27, 113.2, 106.7, 56.6, 27.0. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>18</sub>N<sub>3</sub>O<sub>5</sub>S<sup>+</sup>: 436.0962, found: 436.0958.</div></div><div id="sec5_19_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> <i>N</i>-(4-(furan-3-yl)-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)-2-methoxybenzenesulfonamide (<b>66</b>)</h4><div class="NLM_p last">Compound <b>66</b> was prepared according to general procedure B on 0.38 mmol scale. Purification by column chromatography (DCM) yielded the title compound (92 mg, 0.21 mmol, yield 56%). Yellow solid; mp 248–250 °C. HPLC analysis: retention time = 6.069 min; peak area, 95.52%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.63 (s, 1H), 8.86–8.81 (m, 1H), 8.44 (s, 1H), 7.88–7.78 (m, 2H), 7.56 (d, <i>J</i> = 7.8 Hz, 1H), 7.52–7.43 (m, 2H), 7.06–6.97 (m, 3H), 3.63 (s, 3H), 3.29 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.7, 154.4, 145.1, 144.9, 137.8, 136.5, 135.6, 135.5, 130.6, 129.0, 128.0, 126.6, 120.5, 120.4, 117.6, 116.6, 113.2, 109.9, 106.4, 99.9, 56.5, 26.9. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>18</sub>N<sub>3</sub>O<sub>5</sub>S<sup>+</sup>: 436.0962, found: 436.0968.</div></div><div id="sec5_19_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> <i>N</i>-(4-(1<i>H</i>-imidazol-1-yl)-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)-2-methoxybenzenesulfonamide (<b>67</b>)</h4><div class="NLM_p last">Compound <b>67</b> was prepared according to general procedure B on 0.38 mmol scale. Purification by column chromatography (DCM) yielded the title compound (64 mg, 0.15 mmol, yield 39%). Yellow solid; mp 246–247 °C. HPLC analysis: retention time = 5.899 min; peak area, 95.83%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.15 (s, 1H), 9.05 (t, <i>J</i> = 1.1 Hz, 1H), 8.52 (s, 1H), 8.48 (t, <i>J</i> = 1.4 Hz, 1H), 7.79 (d, <i>J</i> = 7.7 Hz, 1H), 7.60 (d, <i>J</i> = 7.8 Hz, 1H), 7.47 (dd, <i>J</i> = 8.4, 7.4 Hz, 1H), 7.22–7.15 (m, 1H), 7.00 (q, <i>J</i> = 7.8 Hz, 3H), 3.61 (s, 3H), 3.29 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.8, 156.9, 150.2, 138.2, 137.1, 136.8, 136.6, 135.5, 130.7, 130.6, 129.0, 127.0, 123.0, 120.3, 118.2, 117.6, 113.2, 110.2, 106.8, 56.3, 27.1. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>18</sub>N5O<sub>4</sub>S<sup>+</sup>: 436.1074, found: 436.1069.</div></div><div id="sec5_19_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> 2-Methoxy-<i>N</i>-(1-methyl-4-(4-methyl-1<i>H</i>-imidazol-1-yl)-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>68</b>)</h4><div class="NLM_p last">Compound <b>68</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (DCM) yielded the title compound (51 mg, 0.11 mmol, yield 32%). Yellow solid; mp 269–270 °C. HPLC analysis: retention time = 4.383 min; peak area, 100.00%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.07 (s, 1H), 8.92 (s, 1H), 8.48–8.39 (m, 1H), 8.17–8.09 (m, 1H), 7.79 (d, <i>J</i> = 7.8 Hz, 1H), 7.59 (d, <i>J</i> = 7.7 Hz, 1H), 7.54–7.42 (m, 1H), 7.01 (dt, <i>J</i> = 7.5, 3.3 Hz, 3H), 3.62 (s, 3H), 3.29 (s, 3H), 2.22 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.8, 156.9, 150.2, 139.3, 138.1, 136.9, 136.7, 136.4, 135.5, 130.6, 128.9, 127.0, 123.0, 120.3, 117.4, 114.1, 113.2, 109.9, 106.6, 56.3, 27.1, 14.1. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>20</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup>: 450.1231, found: 450.1228.</div></div><div id="sec5_19_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> 2-Methoxy-<i>N</i>-(1-methyl-4-(1-methyl-1<i>H</i>-pyrazol-4-yl)-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)benzenesulfonamide (<b>69</b>)</h4><div class="NLM_p last">Compound <b>69</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (80% EA/PE) yielded the title compound (79 mg, 0.18 mmol, yield 49%). Yellow solid; mp over 300 °C. HPLC analysis: retention time = 4.479 min; peak area, 95.27%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.46 (s, 1H), 8.74 (s, 1H), 8.41 (s, 1H), 8.31 (s, 1H), 7.90–7.77 (m, 1H), 7.50 (dd, <i>J</i> = 9.6, 6.9 Hz, 2H), 7.01 (t, <i>J</i> = 7.4 Hz, 2H), 6.93 (d, <i>J</i> = 7.7 Hz, 1H), 3.96 (s, 3H), 3.66 (s, 3H), 3.26 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.5, 156.6 (2C), 155.0, 139.0, 137.8, 136.4, 135.7, 135.4, 131.9, 130.7, 128.7, 126.4, 123.6, 120.4, 120.0, 117.1, 116.1, 113.2, 105.9, 56.6, 26.9. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>20</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup>: 450.1231, found: 450.1238.</div></div><div id="sec5_19_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> <i>N</i>-(4-(1-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-2-oxo-1,2-dihydropyrrolo[4,3,2-<i>de</i>]quinolin-6-yl)-2-methoxybenzenesulfonamide (<b>70</b>)</h4><div class="NLM_p last">Compound <b>70</b> was prepared according to general procedure B on 0.33 mmol scale. Purification by column chromatography (90% EA/PE) yielded the title compound (78 mg, 0.16 mmol, yield 50%). Yellow solid; mp 253–255 °C. HPLC analysis: retention time = 6.168 min; peak area, 96.75%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.55 (s, 1H), 8.84 (s, 1H), 8.41 (s, 1H), 8.34 (s, 1H), 7.83 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.53 (d, <i>J</i> = 7.8 Hz, 1H), 7.51–7.42 (m, 1H), 7.07–6.95 (m, 2H), 6.93 (d, <i>J</i> = 7.7 Hz, 1H), 3.85 (t, <i>J</i> = 7.4 Hz, 1H), 3.64 (s, 3H), 3.26 (s, 3H), 1.25–1.00 (m, 4H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.5, 156.7, 154.9, 139.0, 137.8, 136.5, 135.6, 135.4, 131.0, 130.6, 128.8, 126.6, 123.3, 120.4, 120.2, 117.1, 116.1, 113.2, 105.9, 56.5, 33.5, 26.9, 6.9 (2C). HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>22</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup>: 476.1387, found: 476.1380.</div></div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i123" class="anchor-spacer"></div><h3 class="article-section__title" id="_i123"> General Procedure C for Synthesis of Compound <b>77</b>–<b>84</b></h3><div class="NLM_p">A solution of 1,8-naphthalic anhydride (10.0 g, 50.5 mmol) and silver sulfate (7.9 g, 25.25 mmol, 0.50 equiv) in concentrated sulfuric acid (200 mL) was stirred at room temperature. Bromine (3.2 mL, 63.0 mmol, 1.26 equiv) was added over 30 min and then heated to 60 °C for 8–10 h, before cooling back to 20 °C. The solid silver bromide byproduct was filtered off at a water pump to give a clear orange solution. This mixture reaction was added dropwise to ice–water mixture (1 L). An off-white solid precipitate was filtered. The filter cake was washed once by displacement with water (50 mL) and twice with cold ethanol (100 mL each) and dried in vacuo at 60 °C to constant weight to yield the title compound <b>71</b> as a white solid (12.54 g, 45.5 mmol, 90% yield).</div><div class="NLM_p">A solution of 5-bromo-1<i>H</i>,3<i>H</i>-benzo[<i>de</i>]isochromene-1,3-dione <b>71</b> (27.59 g, 100 mmol) and hydroxylamine hydrochloride (6.9 g, 100 mmol) in pyridine (200 mL) was refluxed for 2 h, followed by cooling to 80 °C. Then, <i>p</i>-toluenesulfonyl chloride (38.14 g, 200 mmol) was added to the reaction system. After addition, the temperature was raised and the reaction was carried out under reflux for 2 h, followed by cooling. The reaction mixture was poured into 0.50 L of water and stirred to precipitate crystals, which were collected by filtration. The crystals were transferred to a beaker and washed successively with 0.5 L of a NaHCO<sub>3</sub> aqueous solution and 0.5 L of water, followed by filtration. The crystals were washed with water and dried to give an intermediate for further reaction. The whole amount of the intermediate, 85 mL of ethanol, and 100 mL of water were put in a reactor and stirred. Then, 325 mL of a 1.4 mol/L aqueous solution of NaOH was added dropwise to the mixture. Thereafter, the mixture was heated to refluxing temperature, at which the reaction was carried out for 3 h while distilling off ethanol. After completion of the reaction, the reaction mixture was cooled to 75 °C, and 60 mL of concentrated HCl was added dropwise. In the meantime, crystals were precipitated at 60 °C. After completion of the dropwise addition, the mixture was further cooled. The precipitated crystals were collected by filtration, washed with ion-exchanged water, and dried to give the title compound <b>72</b> as a white solid (7.17 g, 29.14 mmol, yield 29%).</div><div class="NLM_p">To a solution of 4-bromo-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one <b>72</b> (7 g, 28.5 mmol) in AcOH (15 mL) was added 69% HNO<sub>3</sub> (2 mL, 42.75 mmol) and heated to 50–65 °C. The mixture reaction was stirred for 4 h. When TLC analysis showed complete conversion of the starting material, the reaction mixture was poured into water filtrated through Celite and the filtrate was concentrated in vacuum. The crude product was purified by silica gel column chromatography (PE/DCM) to yield the title compound <b>73</b> as a yellow solid (6.9 g, 23.65 mmol, yield 83%).</div><div class="NLM_p">A solution of 4-bromo-1-methyl-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one <b>73</b> (6.5 g, 22.26 mmol) in anhydrous DMF (100 mL) was stirred at 0 °C. The solution was added with 60% NaH (1.34 g, 33.35 mmol) in three batches and stirred for 30 min. Methyl iodide (2.06 mL, 33.35 mmol) was added dropwise to the mixture, and the whole mixture was stirred at room temperature for 3 h and then poured into ice water. This mixture was extracted with AcOEt. The organic layer was washed with H<sub>2</sub>O and brine four times, dried (MgSO<sub>4</sub>), concentrated, and purified by silica gel column chromatography to afford the title compound <b>74</b> as a yellow solid (5.52 g, 18.04 mmol, 81% yield).</div><div class="NLM_p">A mixture of 4-bromo-1-methyl-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one <b>74</b> (1 equiv), Pd(PPh<sub>3</sub>)<sub>4</sub> (0.05 equiv), K<sub>2</sub>CO<sub>3</sub> (3 equiv), H<sub>2</sub>O (10 equiv), and proper boric acid or borate (1.5 equiv) was dissolved in 1,4-dioxane. The reaction system was exchanged by N<sub>2</sub> and then heated to 80 °C for 7 h. When TLC analysis showed complete conversion of the starting material, the mixture was extracted with AcOEt and water. The organic layer was washed with H<sub>2</sub>O and brine, dried (MgSO<sub>4</sub>), concentrated, and purified by silica gel column chromatography to afford the title compound <b>75a–75g</b> as a yellow solid (50–80% yield).</div><div class="NLM_p last">To a solution of compounds <b>75a–75g</b> (1 equiv) in THF/MeOH (1:1) was added Pd/C (10% mol). The mixture reaction was flushed thrice with argon, then thrice with hydrogen. The flask was charged with hydrogen gas, and reaction mixture was left to stir for 5 h. The suspension was filtered through Celite and rinsed with EtOAc. The filtrate was evaporated to afford the crude product <b>76a–76g</b> as a yellow solid (70–90% yield). To a solution of the intermediate <b>76a–76g</b> (1 equiv) and proper phenylsulfonyl chloride (1.5 equiv) in CH<sub>2</sub>Cl<sub>2</sub> was added pyridine (15 equiv) and stirred at room temperature overnight. When TLC analysis showed complete conversion of the starting material, the mixture was concentrated and purified by silica gel column chromatography to afford the target compounds <b>75a–75g</b> as a yellow solid (30–70% yield).</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i124" class="anchor-spacer"></div><h3 class="article-section__title" id="_i124"> General Procedure C for Synthesis of Intermediate</h3><div id="sec5_21_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> 5-Bromo-1<i>H</i>,3<i>H</i>-benzo[<i>de</i>]isochromene-1,3-dione (<b>71</b>)</h4><div class="NLM_p last">Compound <b>71</b> was prepared according to general procedure B on 50.5 mmol scale. Purification by column chromatography (20% EA/DCM) yielded the intermediate compound as a white solid (12.54 g, 45.5 mmol, 90% yield). [M + H]<sup>+</sup>: 301.13. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.83 (s, 1H), 8.57–8.44 (m, 3H), 7.93 (q, <i>J</i> = 9.4, 8.6 Hz, 1H).</div></div><div id="sec5_21_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> 4-Bromobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>72</b>)</h4><div class="NLM_p last">Compound <b>72</b> was prepared according to general procedure B on 100 mmol scale. Purification by column chromatography (20% EA/DCM) yielded the intermediate compound as a white solid (7.17 g, 29.14 mmol, yield 29%). [M + H]<sup>+</sup>: 247.9. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.92 (s, 1H), 8.44 (s, 1H), 8.10 (s, 1H), 7.58 (d, <i>J</i> = 3.5 Hz, 2H), 7.01 (dd, <i>J</i> = 5.9, 1.8 Hz, 1H).</div></div><div id="sec5_21_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> 4-Bromo-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>73</b>)</h4><div class="NLM_p last">Compound <b>73</b> was prepared according to general procedure B on 28.5 mmol scale. Purification by column chromatography (20% EA/DCM) yielded the intermediate compound as a light yellow solid (6.9 g, 23.65 mmol, yield 83%). [M + H]<sup>+</sup>: 291.0. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.91 (s, 1H), 8.67 (d, <i>J</i> = 8.1 Hz, 1H), 8.31 (s, 1H), 7.32 (d, <i>J</i> = 8.1 Hz, 1H), 3.38 (s, 3H).</div></div><div id="sec5_21_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> 4-Bromo-1-methyl-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>74</b>)</h4><div class="NLM_p last">Compound <b>74</b> was prepared according to general procedure B on 22.6 mmol scale. Purification by column chromatography (20% EA/DCM) yielded the intermediate compound as a light yellow solid (5.52 g, 18.04 mmol, yield 81%). [M + H]<sup>+</sup>: 306.9. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.91 (s, 1H), 8.67 (d, <i>J</i> = 8.1 Hz, 1H), 8.31 (s, 1H), 7.32 (d, <i>J</i> = 8.1 Hz, 1H), 3.38 (s, 3H).</div></div><div id="sec5_21_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> 1-Methyl-4-(1-methyl-1<i>H</i>-pyrazol-4-yl)-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>75a</b>)</h4><div class="NLM_p last">Compound <b>75a</b> was prepared according to general procedure B on 0.82 mmol scale. Purification by column chromatography (20% EA/DCM) yielded the intermediate compound as a white solid (197 mg, 23.65 mmol, yield 86%). [M + H]<sup>+</sup>: 309.2. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.50 (s, 1H), 8.95 (d, <i>J</i> = 1.3 Hz, 1H), 8.61 (d, <i>J</i> = 8.1 Hz, 1H), 8.27 (d, <i>J</i> = 1.3 Hz, 1H), 7.10 (d, <i>J</i> = 8.1 Hz, 1H).</div></div><div id="sec5_21_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> 4-(1-Ethyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>75b</b>)</h4><div class="NLM_p last">Compound <b>75b</b> was prepared according to general procedure B on 0.82 mmol scale. Purification by column chromatography (10% EA/DCM) yielded the intermediate compound as a white solid (207 mg, 0.62 mmol, yield 75%). [M + H]<sup>+</sup>: 337.1.<sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.80 (d, <i>J</i> = 1.2 Hz, 1H), 8.57 (d, <i>J</i> = 8.1 Hz, 1H), 8.52 (s, 1H), 8.34 (d, <i>J</i> = 1.1 Hz, 1H), 8.12 (s, 1H), 7.17 (d, <i>J</i> = 8.1 Hz, 1H), 4.13 (t, <i>J</i> = 7.0 Hz, 2H), 3.35 (s, 3H), 1.87 (h, <i>J</i> = 7.2 Hz, 2H), 0.90 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec5_21_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> 4-(1-Cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>75c</b>)</h4><div class="NLM_p last">Compound <b>75c</b> was prepared according to general procedure B on 0.82 mmol scale. Purification by column chromatography (10% EA/DCM) yielded the intermediate compound as a white solid (225 mg, 0.67 mmol, yield 82%). [M + H]<sup>+</sup>: 335.1. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.86 (s, 1H), 8.61 (d, <i>J</i> = 7.8 Hz, 2H), 8.41 (s, 1H), 8.11 (s, 1H), 7.21 (d, <i>J</i> = 8.2 Hz, 1H), 3.80 (s, 1H), 3.38 (s, 3H), 1.16–1.09 (m, 2H), 1.08–1.00 (m, 2H).</div></div><div id="sec5_21_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i132" class="anchor-spacer"></div><h4 class="article-section__title" id="_i132"> 1-Methyl-4-(5-methylfuran-2-yl)-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>75d</b>)</h4><div class="NLM_p last">Compound <b>75d</b> was prepared according to general procedure B on 0.82 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a white solid (159 mg, 0.52 mmol, yield 63%). [M + H]<sup>+</sup>: 309.1. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.95 (s, 1H), 8.65 (d, <i>J</i> = 8.2 Hz, 1H), 8.47 (s, 1H), 7.30 (d, <i>J</i> = 21.2 Hz, 1H), 6.48–6.26 (m, 1H), 5.76 (s, 1H), 3.39 (s, 3H), 2.43 (s, 3H).</div></div><div id="sec5_21_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> 4-(1-Benzyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-6-nitrobenzo[<i>cd</i>]indol-2(1<i>H</i>)-one (<b>75e</b>)</h4><div class="NLM_p last">Compound <b>75e</b> was prepared according to general procedure B on 0.82 mmol scale. Purification by column chromatography (DCM) yielded the intermediate compound as a light yellow solid (258 mg, 0.67 mmol, yield 82%). [M + H]<sup>+</sup>: 385.1. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.87 (d, <i>J</i> = 1.1 Hz, 1H), 8.68 (s, 1H), 8.61 (d, <i>J</i> = 8.1 Hz, 1H), 8.42 (d, <i>J</i> = 1.1 Hz, 1H), 8.20 (s, 1H), 7.35 (qd, <i>J</i> = 6.9, 2.4 Hz, 5H), 7.20 (d, <i>J</i> = 8.1 Hz, 1H), 5.40 (s, 2H), 3.37 (s, 3H).</div></div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i134" class="anchor-spacer"></div><h3 class="article-section__title" id="_i134"> General Procedure C for Synthesis of Target Compound <b>77</b>–<b>83</b></h3><div id="sec5_22_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i135" class="anchor-spacer"></div><h4 class="article-section__title" id="_i135"> 2-Methoxy-<i>N</i>-(1-methyl-4-(1-methyl-1<i>H</i>-pyrazol-4-yl)-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>77</b>)</h4><div class="NLM_p last">Compound <b>77</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (84 mg, 0.19 mmol, yield 52%). Yellow solid; mp 285–288 °C. HPLC analysis: retention time = 5.893 min; peak area, 97.36%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.08 (s, 1H), 8.59 (d, <i>J</i> = 1.1 Hz, 1H), 8.33 (s, 1H), 8.26 (d, <i>J</i> = 1.0 Hz, 1H), 8.05 (s, 1H), 7.66 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.49 (ddd, <i>J</i> = 8.8, 7.4, 1.8 Hz, 1H), 7.29 (d, <i>J</i> = 7.7 Hz, 1H), 7.10 (d, <i>J</i> = 7.8 Hz, 1H), 7.01–6.93 (m, 1H), 6.92 (d, <i>J</i> = 7.6 Hz, 1H), 3.81 (s, 3H), 3.36 (s, 1H), 3.28 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 156.83, 137.4, 136.9, 135.5, 134.0, 130.8, 129.1, 128.1, 127.3, 126.8, 125.3, 123.2, 122.9, 122.8, 122.5, 122.3, 120.4, 113.1, 105.4, 56.5, 39.3, 26.6. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>21</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 449.1278, found: 449.1274.</div></div><div id="sec5_22_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i136" class="anchor-spacer"></div><h4 class="article-section__title" id="_i136"> 4-Chloro-<i>N</i>-(1-methyl-4-(1-methyl-1<i>H</i>-pyrazol-4-yl)-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>78</b>)</h4><div class="NLM_p last">Compound <b>7</b> was prepared according to general procedure B on 0.50 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (78 mg, 0.17 mmol, yield 48%). Yellow solid; mp 212–213 °C. HPLC analysis: retention time = 6.047 min; peak area, 97.28%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.29 (s, 1H), 8.25 (d, <i>J</i> = 2.1 Hz, 2H), 8.13–8.06 (m, 1H), 7.92 (s, 1H), 7.73–7.63 (m, 2H), 7.60–7.49 (m, 2H), 7.22 (d, <i>J</i> = 7.6 Hz, 1H), 7.01 (d, <i>J</i> = 7.6 Hz, 1H), 3.93 (s, 3H), 3.33 (s, 5H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.4, 138.6, 138.5, 138.2, 136.8, 134.2, 129.7 (2C), 129.2 (2C), 129.0, 127.4, 127.0, 126.8, 126.2, 123.4, 122.6, 122.5, 121.9, 105.4, 39.3, 26.7. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>18</sub>ClN<sub>4</sub>O<sub>3</sub>S<sup>+</sup>: 453.0783, found: 453.0777.</div></div><div id="sec5_22_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> 2-Methoxy-<i>N</i>-(1-methyl-4-(5-methylfuran-2-yl)-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>79</b>)</h4><div class="NLM_p last">Compound <b>79</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (20% EA/DCM) yielded the title compound (98 mg, 0.22 mmol, yield 61%). Yellow solid; mp over 300 °C. HPLC analysis: retention time = 5.781 min; peak area, 98.08%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.18 (s, 1H), 8.49 (s, 1H), 8.31 (s, 1H), 7.56 (dd, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.49 (t, <i>J</i> = 7.9 Hz, 1H), 7.31 (d, <i>J</i> = 7.6 Hz, 1H), 7.22–7.13 (m, 2H), 6.98 (d, <i>J</i> = 7.7 Hz, 1H), 6.95–6.84 (m, 1H), 6.30 (dd, <i>J</i> = 3.2, 1.2 Hz, 1H), 3.97 (s, 3H), 3.30 (s, 3H), 2.44 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.1, 156.8, 153.2, 151.5, 137.9, 135.4, 131.6, 130.5, 128.0, 127.0, 125.9, 125.5, 123.4, 120.8, 120.7, 120.3, 113.0, 109.5, 109.2, 105.9, 99.9, 56.5, 26.6, 14.0. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>21</sub>N<sub>2</sub>O<sub>5</sub>S<sup>+</sup>:449.1166, found: 449.1167.</div></div><div id="sec5_22_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i138" class="anchor-spacer"></div><h4 class="article-section__title" id="_i138"> <i>N</i>-(4-(furan-3-yl)-1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)-2-methoxybenzenesulfonamide (<b>80</b>)</h4><div class="NLM_p last">Compound <b>80</b> was prepared according to general procedure B on 0.38 mmol scale. Purification by column chromatography (20% EA/DCM) yielded the title compound (74 mg, 0.17 mmol, yield 45%). Yellow solid; mp 273–275 °C. HPLC analysis: retention time = 5.358 min; peak area, 98.95%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.15 (s, 1H), 8.61 (s, 1H), 8.44 (s, 1H), 8.34 (s, 1H), 7.87 (s, 1H), 7.77–7.62 (m, 1H), 7.53–7.43 (m, 1H), 7.35 (d, <i>J</i> = 7.7 Hz, 1H), 7.18–7.06 (m, 2H), 6.96 (d, <i>J</i> = 7.9 Hz, 2H), 3.80 (s, 3H), 3.29 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 145.2, 141.0, 138.2, 133.4, 129.7, 129.2, 127.4, 127.1, 126.9, 126.0, 125.9, 123.8, 123.5, 122.9, 109.0, 105.8, 26.7. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>19</sub>N<sub>2</sub>O<sub>5</sub>S<sup>+</sup>: 435.1009, found: 435.1007.</div></div><div id="sec5_22_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i139" class="anchor-spacer"></div><h4 class="article-section__title" id="_i139"> 4-Chloro-<i>N</i>-(4-(furan-3-yl)-1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>81</b>)</h4><div class="NLM_p last">Compound <b>81</b> was prepared according to general procedure B on 0.38 mmol scale. Purification by column chromatography (20% EA/DCM) yielded the title compound (78 mg, 0.18 mmol, yield 47%). Yellow solid; mp 270–272 °C. HPLC analysis: retention time = 6.301 min; peak area, 98.44%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.32 (s, 1H), 8.35 (d, <i>J</i> = 14.6 Hz, 2H), 8.09 (s, 1H), 7.85 (s, 1H), 7.67 (d, <i>J</i> = 8.6 Hz, 2H), 7.52 (d, <i>J</i> = 8.5 Hz, 2H), 7.28 (d, <i>J</i> = 7.6 Hz, 1H), 7.06 (d, <i>J</i> = 7.6 Hz, 1H), 6.95 (d, <i>J</i> = 1.8 Hz, 1H), 3.34 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 145.2, 141.04, 138.6, 138.5, 138.2, 133.4, 129.7 (2C), 129.2 (2C), 127.4, 127.1, 126.9, 126.0, 125.9, 123.8, 123.5, 122.9, 109.0, 105.8, 26.7. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>16</sub>ClN<sub>2</sub>O<sub>4</sub>S<sup>+</sup>: 439.0514, found: 439.0516.</div></div><div id="sec5_22_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i140" class="anchor-spacer"></div><h4 class="article-section__title" id="_i140"> 2-Methoxy-<i>N</i>-(1-methyl-2-oxo-4-(1-propyl-1<i>H</i>-pyrazol-4-yl)-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>82</b>)</h4><div class="NLM_p last">Compound <b>82</b> was prepared according to general procedure B on 0.33 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (94 mg, 0.20 mmol, yield 62%). Yellow solid; mp 264–265 °C. HPLC analysis: retention time = 4.614 min; peak area, 97.54%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.17 (s, 0H), 8.53 (s, 1H), 8.38 (s, 1H), 8.29 (s, 1H), 8.05 (s, 1H), 7.64 (d, <i>J</i> = 8.4 Hz, 1H), 7.31–7.19 (m, 2H), 7.03 (d, <i>J</i> = 8.2 Hz, 1H), 6.95 (d, <i>J</i> = 7.7 Hz, 1H), 4.14 (t, <i>J</i> = 6.7 Hz, 2H), 3.81 (s, 3H), 3.30 (s, 3H), 1.87 (q, <i>J</i> = 7.2 Hz, 2H), 0.89 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 157.5, 139.9, 137.6, 136.8, 134.2, 132.2, 128.3, 127.8, 127.4, 125.9, 125.4, 123.36, 123.32, 122.78, 122.72, 121.9, 120.6, 113.7, 105.4, 57.1, 53.6, 26.6, 23.6, 11.4. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 477.1591, found: 477.1592.</div></div><div id="sec5_22_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i141" class="anchor-spacer"></div><h4 class="article-section__title" id="_i141"> 4-Chloro-2-methoxy-<i>N</i>-(1-methyl-2-oxo-4-(1-propyl-1<i>H</i>-pyrazol-4-yl)-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>83</b>)</h4><div class="NLM_p last">Compound <b>83</b> was prepared according to general procedure B on 0.33 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (92 mg, 0.19 mmol, yield 57%). Yellow solid; mp 271–273 °C. HPLC analysis: retention time = 6.200 min; peak area, 97.51%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.18 (s, 1H), 8.57–8.50 (m, 1H), 8.38 (s, 1H), 8.30 (d, <i>J</i> = 1.0 Hz, 1H), 8.06 (s, 1H), 7.65 (d, <i>J</i> = 8.4 Hz, 1H), 7.32–7.20 (m, 2H), 7.03 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 6.94 (d, <i>J</i> = 7.6 Hz, 1H), 3.82 (s, 3H), 3.30 (s, 3H), 1.87 (q, <i>J</i> = 7.2 Hz, 2H), 0.90 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 157.5, 139.9, 137.6, 136.8, 134.2, 132.2, 128.3, 127.8, 127.4, 125.9, 125.4, 123.3, 122.7, 122.7, 121.9, 120.6, 113.7, 105.4, 57.1, 53.6, 26.6, 23.6, 11.4. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>24</sub>ClN<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 511.1201, found: 511.1206.</div></div><div id="sec5_22_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i142" class="anchor-spacer"></div><h4 class="article-section__title" id="_i142"> <i>N</i>-(4-(1-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)-2-methoxybenzenesulfonamide (<b>84</b>)</h4><div class="NLM_p last">Compound <b>84</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (90 mg, 0.19 mmol, yield 53%). Yellow solid; mp 258–259 °C. HPLC analysis: retention time = 2.613 min; peak area, 97.84%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.17 (s, 1H), 8.56 (s, 1H), 8.46 (s, 1H), 8.32 (s, 1H), 8.03 (s, 1H), 7.65 (d, <i>J</i> = 8.4 Hz, 1H), 7.28 (d, <i>J</i> = 7.7 Hz, 1H), 7.23 (d, <i>J</i> = 1.9 Hz, 1H), 7.03 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 6.94 (d, <i>J</i> = 7.7 Hz, 1H), 3.84 (dd, <i>J</i> = 7.1, 3.7 Hz, 1H), 3.81 (s, 3H), 1.21–1.09 (m, 2H), 1.05 (qd, <i>J</i> = 7.5, 3.6 Hz, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 157.5, 139.9, 137.6, 136.8, 134.0, 132.2, 128.3, 127.8, 127.4, 125.9, 125.3, 123.3, 123.0, 122.8, 122.7, 121.9, 120.6, 113.7, 105.4, 57.1, 33.4, 26.6, 6.8 (2C). HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>23</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 475.1435, found: 475.1437.</div></div><div id="sec5_22_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i143" class="anchor-spacer"></div><h4 class="article-section__title" id="_i143"> 4-Chloro-<i>N</i>-(4-(1-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>85</b>)</h4><div class="NLM_p last">Compound <b>85</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (95 mg, 0.20 mmol, yield 55%). Yellow solid; mp 262–264 °C. HPLC analysis: retention time = 6.031 min; peak area, 96.78%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.30 (s, 1H), 8.39 (s, 1H), 8.29 (s, 1H), 8.13 (s, 1H), 7.92 (s, 1H), 7.71 (d, <i>J</i> = 8.3 Hz, 2H), 7.54 (d, <i>J</i> = 8.3 Hz, 2H), 7.25 (d, <i>J</i> = 7.6 Hz, 1H), 7.01 (d, <i>J</i> = 7.6 Hz, 1H), 3.82 (tt, <i>J</i> = 7.4, 3.9 Hz, 1H), 3.33 (s, 3H), 1.15 (q, <i>J</i> = 6.2, 4.7 Hz, 2H), 1.06 (dd, <i>J</i> = 7.5, 5.0 Hz, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 138.6, 138.4, 138.2, 136.7, 134.1, 129.6 (2C), 129.2 (2C), 128.2, 127.4, 127.0, 126.5, 126.0, 123.4, 122.7, 122.5, 121.7, 105.4, 33.4, 26.6, 6.8 (2C). HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>20</sub>ClN<sub>4</sub>O<sub>3</sub>S<sup>+</sup>: 479.0939, found: 479.0939.</div></div><div id="sec5_22_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i144" class="anchor-spacer"></div><h4 class="article-section__title" id="_i144"> 4-Chloro-<i>N</i>-(4-(1-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)-2-methoxybenzenesulfonamide (<b>86</b>)</h4><div class="NLM_p last">Compound <b>86</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (95 mg, 0.19 mmol, yield 52%). Yellow solid; mp 263–265 °C. HPLC analysis: retention time = 4.191 min; peak area, 97.08%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.17 (s, 1H), 8.56 (s, 1H), 8.46 (s, 1H), 8.32 (s, 1H), 8.03 (s, 1H), 7.66 (d, <i>J</i> = 8.4 Hz, 1H), 7.33–7.19 (m, 2H), 7.03 (d, <i>J</i> = 8.5 Hz, 1H), 6.94 (d, <i>J</i> = 7.7 Hz, 1H), 3.91–3.82 (m, 1H) 3.81 (s, 3H), 3.30 (s, 3H), 1.17–1.10 (m, 2H), 1.04 (q, <i>J</i> = 7.4, 6.1 Hz, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 157.5, 139.9, 137.6, 136.8, 134.0, 132.2, 128.3, 127.8, 127.4, 125.9, 125.2, 123.3, 123.0, 122.7, 122.7, 121.9, 120.5, 113.7, 105.4, 57.1, 33.4, 26.6, 6.8 (2C). HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>22</sub>ClN<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 509.1045, found: 509.1044.</div></div><div id="sec5_22_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i145" class="anchor-spacer"></div><h4 class="article-section__title" id="_i145"> 4-Chloro-<i>N</i>-(4-(1-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)-2-fluorobenzenesulfonamide (<b>87</b>)</h4><div class="NLM_p last">Compound <b>87</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (91 mg, 0.18 mmol, yield 51%). Yellow solid; mp 260–262 °C. HPLC analysis: retention time = 3.745 min; peak area, 98.26%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.69 (s, 1H), 8.42 (s, 1H), 8.30 (d, <i>J</i> = 8.9 Hz, 2H), 7.93 (s, 1H), 7.76–7.60 (m, 2H), 7.37 (dd, <i>J</i> = 8.5, 2.0 Hz, 1H), 7.27 (d, <i>J</i> = 7.6 Hz, 1H), 7.00 (d, <i>J</i> = 7.6 Hz, 1H), 3.82 (tt, <i>J</i> = 7.4, 3.9 Hz, 1H), 3.33 (s, 4H), 1.14 (q, <i>J</i> = 6.1, 4.3 Hz, 2H), 1.07–0.97 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 139.8 (d, 1C), 138.6, 136.6, 134.2, 132.1, 128.3, 127.4, 126.8, 126.6, 126.5, 126.2, 126.1, 125.7, 123.4, 122.9, 122.3, 121.7, 118.2 (d, 1C), 105.4, 33.4, 26.6, 6.8 (2C). HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>19</sub>ClFN<sub>4</sub>O<sub>3</sub>S<sup>+</sup>: 497.0845, found: 497.0844.</div></div><div id="sec5_22_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i146" class="anchor-spacer"></div><h4 class="article-section__title" id="_i146"> 5-Chloro-<i>N</i>-(4-(1-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)-2-methoxybenzenesulfonamide (<b>88</b>)</h4><div class="NLM_p last">Compound <b>88</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (99 mg, 0.19 mmol, yield 54%). Yellow solid; mp 260–262 °C. HPLC analysis: retention time = 3.880 min; peak area, 96.50%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.24 (s, 1H), 8.52 (s, 1H), 8.45 (s, 1H), 8.31 (s, 1H), 8.01 (s, 1H), 7.64–7.51 (m, 2H), 7.30 (d, <i>J</i> = 7.7 Hz, 1H), 7.14 (d, <i>J</i> = 8.8 Hz, 1H), 6.97 (d, <i>J</i> = 7.7 Hz, 1H), 4.03 (q, <i>J</i> = 7.1 Hz, 1H), 3.77 (s, 3H), 3.30 (s, 3H), 1.13 (q, <i>J</i> = 4.7, 3.8 Hz, 2H), 1.04 (d, <i>J</i> = 7.5 Hz, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 155.7, 137.7, 136.8, 135.1, 134.0, 129.7, 128.4, 128.3, 127.6, 127.4, 125.3, 124.0, 123.5, 123.3, 122.8, 122.6, 121.9, 115.3, 105.5, 56.9, 33.4, 26.6, 6.8 (2C). HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>22</sub>ClN<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 509.1045, found: 509.1041.</div></div><div id="sec5_22_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i147" class="anchor-spacer"></div><h4 class="article-section__title" id="_i147"> 2,4-Dichloro-<i>N</i>-(4-(1-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)benzenesulfonamide (<b>89</b>)</h4><div class="NLM_p last">Compound <b>89</b> was prepared according to general procedure B on 0.36 mmol scale. Purification by column chromatography (50% EA/DCM) yielded the title compound (103 mg, 0.20 mmol, yield 56%). Yellow solid; mp over 300 °C. HPLC analysis: retention time = 7.476 min; peak area, 98.01%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.71 (s, 1H), 8.48–8.37 (m, 2H), 8.32 (s, 1H), 7.98 (s, 1H), 7.90–7.80 (m, 2H), 7.49 (dd, <i>J</i> = 8.6, 2.1 Hz, 1H), 7.29 (d, <i>J</i> = 7.6 Hz, 1H), 6.98 (d, <i>J</i> = 7.6 Hz, 1H), 3.83 (tt, <i>J</i> = 7.5, 3.9 Hz, 1H), 3.31 (s, 3H), 1.15 (dq, <i>J</i> = 7.8, 4.0 Hz, 2H), 1.04 (dd, <i>J</i> = 7.0, 2.1 Hz, 2H). 13C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 138.9, 138.4, 136.6, 136.1, 134.2, 133.3, 132.5, 131.7, 128.3, 128.3, 127.4, 126.5, 125.9, 125.4, 122.9, 122.3, 121.8, 105.4, 33.4, 26.6, 6.8 (2C). HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>22</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup>: 513.0549, found: 513.0543.</div></div><div id="sec5_22_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i148" class="anchor-spacer"></div><h4 class="article-section__title" id="_i148"> <i>N</i>-(4-(1-benzyl-1<i>H</i>-pyrazol-4-yl)-1-methyl-2-oxo-1,2-dihydrobenzo[<i>cd</i>]indol-6-yl)-2-methoxybenzenesulfonamide (<b>90</b>)</h4><div class="NLM_p last">Compound <b>90</b> was prepared according to general procedure B on 0.33 mmol scale. Purification by column chromatography (20% EA/DCM) yielded the title compound (74 mg, 0.14 mmol, yield 43%). Yellow solid; mp 243–245 °C. HPLC analysis: retention time = 4.944 min; peak area, 95.51%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.07 (s, 1H), 8.55 (s, 1H), 8.42 (s, 1H), 8.29 (s, 1H), 8.12 (s, 1H), 7.64 (d, <i>J</i> = 7.8 Hz, 1H), 7.51–7.24 (m, 7H), 7.05 (d, <i>J</i> = 8.4 Hz, 1H), 6.93 (dt, <i>J</i> = 8.0, 3.9 Hz, 2H), 5.43 (s, 2H), 3.74 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 156.7, 137.6, 137.3, 135.5, 133.8, 130.8, 129.1 (3C), 128.4, 128.2 (3C), 128.1, 127.3, 126.8, 125.2, 123.3, 123.0 (2C), 122.6 (2C), 120.4, 113.0, 105.5, 56.4, 55.7, 26.6.. HRMS-EI <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>18</sub>ClN<sub>4</sub>O<sub>3</sub>S<sup>+</sup>: 525.1591, found: 525.1584.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i149"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01010" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31822" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31822" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01010?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01010</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Molecular modeling information (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Pharmacokinetics of <b>68</b> in liver microsomes; permeability assay of compounds in PAMPA; NF-κB transcriptional activity of compounds in HUVECs; concentration of compounds in different medium condition; data collection and refinement statistics; antibodies used for immunofluorescence; antibodies used for western blot analysis; X-ray electron density maps of <b>25</b> and <b>60</b>; the <i>K</i><sub>d</sub> curve images of <b>68</b>; the AlphaScreen assay of <b>68</b> for BET families and BRPF1b; uncropped Western blot images; NMR spectra; and HPLC analysis result data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_001.csv">jm9b01010_si_001.csv (6.57 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_002.pdb">jm9b01010_si_002.pdb (180.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf">jm9b01010_si_003.pdf (20.31 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01010" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiapeng Zhu</span> - <span class="hlFld-Affiliation affiliation">School
of Medicine and Life Sciences, State Key Laboratory Cultivation Base
for TCM Quality and Efficacy, Jiangsu Key Laboratory for Pharmacology
and Safety Evaluation of Chinese Materia Medica, Nanjing University of Chinese Medicine, Nanjing 210023, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#98e2f0edf2f1f9e8fdf6ffd8f0f7ecf5f9f1f4b6fbf7f5"><span class="__cf_email__" data-cfemail="5c26342936353d2c39323b1c343328313d3530723f3331">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tao Lu</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Natural Medicines, China
Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, P. R. China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c0acb5b4a1af80a3b0b5eea5a4b5eea3ae"><span class="__cf_email__" data-cfemail="066a737267694665767328636273286568">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yadong Chen</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Natural Medicines, China
Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2189-9356" title="Orcid link">http://orcid.org/0000-0002-2189-9356</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#09706d6a616c67496a797c276c6d7c276a67"><span class="__cf_email__" data-cfemail="08716c6b606d66486b787d266d6c7d266b66">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fei Jiang</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Natural Medicines, China
Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qinghua Hu</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhimin Zhang</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">Key
Laboratory of Neuropsychiatric Drug Research of Zhejiang Province,
Institute of Materia Medica, Zhejiang Academy
of Medical Sciences, Hangzhou 310013, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongmei Li</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Natural Medicines, China
Pharmaceutical University, 24 Tongjiaxiang, Nanjing 210009, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huili Li</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dewei Zhang</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hanwen Li</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Ma</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jingjing Xu</span> - <span class="hlFld-Affiliation affiliation">School
of Medicine and Life Sciences, State Key Laboratory Cultivation Base
for TCM Quality and Efficacy, Jiangsu Key Laboratory for Pharmacology
and Safety Evaluation of Chinese Materia Medica, Nanjing University of Chinese Medicine, Nanjing 210023, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haifang Chen</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Cui</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanle Zhi</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6066-2231" title="Orcid link">http://orcid.org/0000-0001-6066-2231</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanmin Zhang</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junyu Xu</span> - <span class="hlFld-Affiliation affiliation">School
of Sciences, China Pharmaceutical University, 639 Longmian Avenue, Nanjing 211198, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4918-0693" title="Orcid link">http://orcid.org/0000-0002-4918-0693</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>F.J. and Q.H. contributed equally to this work. The authors declare no competing financial interest. The manuscript was written through contributions of all authors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The authors will release the atomic coordinates and experimental data upon article publication.</p></div></li></ul></div><div class="ack" id="ACK-d7e9908-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i152">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46310" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46310" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors are grateful to the National Natural Science Foundation of China (Program No. 81673301 and  81773745) for the financial support. This work was financially supported by the “Double First-Class” University project from China Pharmaceutical University (Program No. CPU2018GF02), the Priority Academic Program Development of Jiangsu Higher Education Institutions (Integration of Chinese and Western Medicine), and the China Postdoctoral Science Foundation (2018M632428). They also express their gratitude to Dr. Bingxu Zhang (3D BioOptima Co. Ltd.) and Dr. Jamie Planck (RBC, Pennsylvania) for their helpful support in biological evaluation. They thank the Shanghai Synchrotron Radiation Facility (SSRF) for access to their synchrotron facilities.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS</td></tr><tr><td class="NLM_term">BRD4(1)</td><td class="NLM_def"><p class="first last">first Bromodomain of Bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">BRD4(2)</td><td class="NLM_def"><p class="first last">second Bromodomain of Bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">NF-κB</td><td class="NLM_def"><p class="first last">nuclear factor kappa-light-chain-enhancer of activated B cells</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i154">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33582" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33582" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 58 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonifacio, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso-Valdivielso, J. L.</span></span> <span> </span><span class="NLM_article-title">Gout treatment: when a tablet is not enough</span>. <i>Nat. Rev. Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1038/nrrheum.2013.173-c1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fnrrheum.2013.173-c1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=194-195&author=A.+L.+Bonifacioauthor=J.+L.+Alonso-Valdivielso&title=Gout+treatment%3A+when+a+tablet+is+not+enough&doi=10.1038%2Fnrrheum.2013.173-c1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrrheum.2013.173-c1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrrheum.2013.173-c1%26sid%3Dliteratum%253Aachs%26aulast%3DBonifacio%26aufirst%3DA.%2BL.%26aulast%3DAlonso-Valdivielso%26aufirst%3DJ.%2BL.%26atitle%3DGout%2520treatment%253A%2520when%2520a%2520tablet%2520is%2520not%2520enough%26jtitle%3DNat.%2520Rev.%2520Rheumatol.%26date%3D2014%26volume%3D10%26spage%3D194%26epage%3D195%26doi%3D10.1038%2Fnrrheum.2013.173-c1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.2217/epi-2016-0144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.2217%2Fepi-2016-0144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27925476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=573-584&author=D.+F.+Toughauthor=R.+K.+Prinjha&title=Immune+disease-associated+variants+in+gene+enhancers+point+to+BET+epigenetic+mechanisms+for+therapeutic+intervention&doi=10.2217%2Fepi-2016-0144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span></div><div class="casAuthors">Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-584</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Genome-wide assocn. studies have identified thousands of single nucleotide polymorphisms in the human genome that are statistically assocd. with particular disease traits.  In this perspective, we review emerging data suggesting that most single nucleotide polymorphisms assocd. with immune-mediated diseases are found in regulatory regions of the DNA - parts of the genome that control expression of the protein encoding genes - rather than causing mutations in proteins.  We discuss how the emerging understanding of particular gene regulatory regions, gene enhancers and the epigenetic mechanisms by which they are regulated is opening up new opportunities for the treatment of immune-mediated diseases, focusing particularly on the BET family of epigenetic reader proteins as potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFU7Sh3r75ZbVg90H21EOLACvtfcHk0liG6Xhh2783EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D&md5=8a2d44dd000c465b904d4ef2e6f37b77</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0144%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DImmune%2520disease-associated%2520variants%2520in%2520gene%2520enhancers%2520point%2520to%2520BET%2520epigenetic%2520mechanisms%2520for%2520therapeutic%2520intervention%26jtitle%3DEpigenomics%26date%3D2017%26volume%3D9%26spage%3D573%26epage%3D584%26doi%3D10.2217%2Fepi-2016-0144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jhang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, G. C.</span></span> <span> </span><span class="NLM_article-title">Beneficial properties of phytochemicals on NLRP3 inflammasome-mediated gout and complication</span>. <i>J. Agric. Food Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">765</span>, <span class="refDoi"> DOI: 10.1021/acs.jafc.7b05113</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jafc.7b05113" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVWhug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=765&author=J.+J.+Jhangauthor=J.+H.+Linauthor=G.+C.+Yen&title=Beneficial+properties+of+phytochemicals+on+NLRP3+inflammasome-mediated+gout+and+complication&doi=10.1021%2Facs.jafc.7b05113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Beneficial Properties of Phytochemicals on NLRP3 Inflammasome-Mediated Gout and Complication</span></div><div class="casAuthors">Jhang, Jhih-Jia; Lin, Jia-Hong; Yen, Gow-Chin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Agricultural and Food Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">765-772</span>CODEN:
                <span class="NLM_cas:coden">JAFCAU</span>;
        ISSN:<span class="NLM_cas:issn">0021-8561</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Gouty arthritis is characterized by the pptn. of monosodium urate (MSU) crystals in the joint.  Pro-inflammatory cytokine IL-1β is a crit. manifestation in response to MSU crystals attack.  IL-1β secretion is dependent on the nucleotide-binding oligomerization domain-like receptor pyrin domain contg. 3 (NLRP3) inflammasome.  Abnormal activation of the NLRP inflammasome is related to cellular oxidative stress.  However, recent studies have illustrated that phytochems. with potent antioxidant activity exert inhibitory effects on NLRP3 inflammasome-mediated diseases.  This review focuses on the current findings of studies on the NLRP3 inflammasome and the proposed mechanisms that MSU crystals trigger inflammation via activation of the NLRP3 inflammasome.  The authors also summarized the potential use of phytochems. on NLRP3 inflammasome-mediated diseases, suggesting that phytochems. can further prevent acute gout attack.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprORHvlyxeibVg90H21EOLACvtfcHk0liG6Xhh2783EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVWhug%253D%253D&md5=f6b3d4ee6df2c0f6ad5135e029a53708</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jafc.7b05113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jafc.7b05113%26sid%3Dliteratum%253Aachs%26aulast%3DJhang%26aufirst%3DJ.%2BJ.%26aulast%3DLin%26aufirst%3DJ.%2BH.%26aulast%3DYen%26aufirst%3DG.%2BC.%26atitle%3DBeneficial%2520properties%2520of%2520phytochemicals%2520on%2520NLRP3%2520inflammasome-mediated%2520gout%2520and%2520complication%26jtitle%3DJ.%2520Agric.%2520Food%2520Chem.%26date%3D2018%26volume%3D66%26spage%3D765%26doi%3D10.1021%2Facs.jafc.7b05113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, S.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.-S.</span></span> <span> </span><span class="NLM_article-title">Poly-gamma-glutamic acid from Bacillus subtilis upregulates pro-inflammatory cytokines while inhibiting NLRP3, NLRC4 and AIM2 inflammasome activation</span>. <i>Cell. Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">111</span>, <span class="refDoi"> DOI: 10.1038/cmi.2016.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fcmi.2016.13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27133472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=111&author=H.+Ahnauthor=S.+G.+Kangauthor=S.-i.+Yoonauthor=P.-H.+Kimauthor=D.+Kimauthor=G.-S.+Lee&title=Poly-gamma-glutamic+acid+from+Bacillus+subtilis+upregulates+pro-inflammatory+cytokines+while+inhibiting+NLRP3%2C+NLRC4+and+AIM2+inflammasome+activation&doi=10.1038%2Fcmi.2016.13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Poly-gamma-glutamic acid from Bacillus subtilis upregulates pro-inflammatory cytokines while inhibiting NLRP3, NLRC4 and AIM2 inflammasome activation</span></div><div class="casAuthors">Ahn, Huijeong; Kang, Seung Goo; Yoon, Sung-il; Kim, Pyeung-Hyeun; Kim, Doo; Lee, Geun-Shik</div><div class="citationInfo"><span class="NLM_cas:title">Cellular & Molecular Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">CMIEAO</span>;
        ISSN:<span class="NLM_cas:issn">1672-7681</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Poly-gamma-glutamic acid (γ-PGA) is a natural, edible and non-toxic polymer synthesized by Bacillus subtilis and is suggested as a safe biomaterial for the use in hydrogels and vaccine adjuvants.  However, the effect of γ-PGA on inflammasome activation has not yet been studied in macrophages.  Inflammasomes, which are intracellular multi-protein complexes, promote acute and chronic inflammation via interleukin-1β or interleukin-18 maturation, and they are known targets for metabolic syndromes and cancer.  In this study, we obsd. that γ-PGA attenuated NLRP3, NLRC4 and AIM2 inflammasome activation, whereas it upregulated pro-inflammatory cytokine expression in human and murine macrophages.  Although γ-PGA had conflicting effects on cytokine prodn. and maturation, it clearly alleviated the severity of lipopolysaccharide-induced endotoxin shock in an animal model.  Thus, we suggest γ-PGA as a candidate to control inflammasome-mediated disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb13M4HBqOD7Vg90H21EOLACvtfcHk0lhx86FzJg-4Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGrsb4%253D&md5=10d604b0efb3a74c08a880199acdc18b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fcmi.2016.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcmi.2016.13%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DH.%26aulast%3DKang%26aufirst%3DS.%2BG.%26aulast%3DYoon%26aufirst%3DS.-i.%26aulast%3DKim%26aufirst%3DP.-H.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DG.-S.%26atitle%3DPoly-gamma-glutamic%2520acid%2520from%2520Bacillus%2520subtilis%2520upregulates%2520pro-inflammatory%2520cytokines%2520while%2520inhibiting%2520NLRP3%252C%2520NLRC4%2520and%2520AIM2%2520inflammasome%2520activation%26jtitle%3DCell.%2520Mol.%2520Immunol.%26date%3D2018%26volume%3D15%26spage%3D111%26doi%3D10.1038%2Fcmi.2016.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, E.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, D.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasakawa, C.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms regulating NLRP3 inflammasome activation</span>. <i>Cell. Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">148</span>, <span class="refDoi"> DOI: 10.1038/cmi.2015.95</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fcmi.2015.95" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=26549800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs1Gms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=148&author=E.-K.+Joauthor=J.+K.+Kimauthor=D.-M.+Shinauthor=C.+Sasakawa&title=Molecular+mechanisms+regulating+NLRP3+inflammasome+activation&doi=10.1038%2Fcmi.2015.95"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms regulating NLRP3 inflammasome activation</span></div><div class="casAuthors">Jo, Eun-Kyeong; Kim, Jin Kyung; Shin, Dong-Min; Sasakawa, Chihiro</div><div class="citationInfo"><span class="NLM_cas:title">Cellular & Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-159</span>CODEN:
                <span class="NLM_cas:coden">CMIEAO</span>;
        ISSN:<span class="NLM_cas:issn">1672-7681</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Inflammasomes are multi-protein signaling complexes that trigger the activation of inflammatory caspases and the maturation of interleukin-1β.  Among various inflammasome complexes, the NLRP3 inflammasome is best characterized and has been linked with various human autoinflammatory and autoimmune diseases.  Thus, the NLRP3 inflammasome may be a promising target for anti-inflammatory therapies.  In this review, we summarize the current understanding of the mechanisms by which the NLRP3 inflammasome is activated in the cytosol.  We also describe the binding partners of NLRP3 inflammasome complexes activating or inhibiting the inflammasome assembly.  Our knowledge of the mechanisms regulating NLRP3 inflammasome signaling and how these influence inflammatory responses offers further insight into potential therapeutic strategies to treat inflammatory diseases assocd. with dysregulation of the NLRP3 inflammasome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJBv_NNzcwFLVg90H21EOLACvtfcHk0lhx86FzJg-4Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs1Gms7s%253D&md5=90c508a823912cf69f86cefac7aaaaec</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fcmi.2015.95&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcmi.2015.95%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DE.-K.%26aulast%3DKim%26aufirst%3DJ.%2BK.%26aulast%3DShin%26aufirst%3DD.-M.%26aulast%3DSasakawa%26aufirst%3DC.%26atitle%3DMolecular%2520mechanisms%2520regulating%2520NLRP3%2520inflammasome%2520activation%26jtitle%3DCell.%2520Mol.%2520Immunol.%26date%3D2016%26volume%3D13%26spage%3D148%26doi%3D10.1038%2Fcmi.2015.95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Samra, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Said, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsherbiny, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liou, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Shishtawy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eissa, L. A.</span></span> <span> </span><span class="NLM_article-title">Cepharanthine and piperine ameliorate diabetic nephropathy in rats: role of NF-κB and NLRP3 inflammasome</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1016/j.lfs.2016.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.lfs.2016.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27266851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGiurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2016&pages=187-199&author=Y.+A.+Samraauthor=H.+S.+Saidauthor=N.+M.+Elsherbinyauthor=G.+I.+Liouauthor=M.+M.+El-Shishtawyauthor=L.+A.+Eissa&title=Cepharanthine+and+piperine+ameliorate+diabetic+nephropathy+in+rats%3A+role+of+NF-%CE%BAB+and+NLRP3+inflammasome&doi=10.1016%2Fj.lfs.2016.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Cepharanthine and Piperine ameliorate diabetic nephropathy in rats: role of NF-κB and NLRP3 inflammasome</span></div><div class="casAuthors">Samra, Yara A.; Said, Heba S.; Elsherbiny, Nehal M.; Liou, Gregory I.; El-Shishtawy, Mamdouh M.; Eissa, Laila A.</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">187-199</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Hyperglycemia leads to elevation of oxidative stress and proinflammatory cytokines which are the main causes of diabetic nephropathy (DN).  NLRP3 inflammasome and thioredoxin-interacting protein (TXNIP) are recently assumed to participate in the development of DN.  We aimed to investigate the effects of Cepharanthine (CEP), Piperine (Pip) and their combination in streptozotocin (STZ)-induced DN focusing on their role to modulate NLRP3 and TXNIP induced inflammation.  Diabetic rats were treated with i.p. (i.p.) injection of CEP (10 mg/kg/day), Pip (30 mg/kg/day) or their combination for 8 wk.  Nuclear factor kappa B (NF-κB), tumor necrosis factor alpha (TNF-α) and interleukin-1β (IL-1β) were assessed by ELISA technique.  TXNIP and NLRP3 genes expressions were evaluated by real time-PCR.  Diabetic rats showed significant increase in renal TXNIP and NLRP3 expression.  CEP, Pip or their combination significantly decreased TXNIP and NLRP3 expression in diabetic kidneys.  Hyperglycemia induced NF-κB activation leading to increased IL-1β and TNF-α levels.  CEP, Pip or their combination showed significant inhibition of NF-κB together with decreased IL-1β and TNF-α levels in diabetic rats.  Also, diabetic rats showed significant decrease in creatinine clearance and increase in blood glucose, serum creatinine, blood urea nitrogen, malondialdehyde, proteinuria, and kidney wt. to body Wt. ratio.  All of these changes were reversed by CEP, Pip or their combination.  The antioxidant and anti-inflammatory effects of CEP and Pip which were accompanied by inhibition of NF-κB and NLRP3 activation might be helpful mechanisms to halt the progression of DN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZEuultRHBi7Vg90H21EOLACvtfcHk0lhx86FzJg-4Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGiurfN&md5=aa668b4f5d012de254f66f16b6cec0fe</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.lfs.2016.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lfs.2016.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DSamra%26aufirst%3DY.%2BA.%26aulast%3DSaid%26aufirst%3DH.%2BS.%26aulast%3DElsherbiny%26aufirst%3DN.%2BM.%26aulast%3DLiou%26aufirst%3DG.%2BI.%26aulast%3DEl-Shishtawy%26aufirst%3DM.%2BM.%26aulast%3DEissa%26aufirst%3DL.%2BA.%26atitle%3DCepharanthine%2520and%2520piperine%2520ameliorate%2520diabetic%2520nephropathy%2520in%2520rats%253A%2520role%2520of%2520NF-%25CE%25BAB%2520and%2520NLRP3%2520inflammasome%26jtitle%3DLife%2520Sci.%26date%3D2016%26volume%3D157%26spage%3D187%26epage%3D199%26doi%3D10.1016%2Fj.lfs.2016.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergsbaken, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cookson, B. T.</span></span> <span> </span><span class="NLM_article-title">Macrophage activation redirects yersinia-infected host cell death from apoptosis to caspase-1-dependent pyroptosis</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e161</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.0030161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1371%2Fjournal.ppat.0030161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=17983266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A280%3ADC%252BD2sjhvFymtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&author=T.+Bergsbakenauthor=B.+T.+Cookson&title=Macrophage+activation+redirects+yersinia-infected+host+cell+death+from+apoptosis+to+caspase-1-dependent+pyroptosis&doi=10.1371%2Fjournal.ppat.0030161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Macrophage activation redirects yersinia-infected host cell death from apoptosis to caspase-1-dependent pyroptosis</span></div><div class="casAuthors">Bergsbaken Tessa; Cookson Brad T</div><div class="citationInfo"><span class="NLM_cas:title">PLoS pathogens</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e161</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Infection of macrophages by Yersinia species results in YopJ-dependent apoptosis, and naive macrophages are highly susceptible to this form of cell death.  Previous studies have demonstrated that macrophages activated with lipopolysaccharide (LPS) prior to infection are resistant to YopJ-dependent cell death; we found this simultaneously renders macrophages susceptible to killing by YopJ(-) Yersinia pseudotuberculosis (Yptb).  YopJ(-) Yptb-induced macrophage death was dependent on caspase-1 activation, resulting in rapid permeability to small molecules, followed by membrane breakdown and DNA damage, and accompanied by cleavage and release of proinflammatory interleukin-18.  Induction of caspase-1-dependent death, or pyroptosis, required the bacterial type III translocon but none of its known translocated proteins.  Wild-type Yptb infection also triggered pyroptosis: YopJ-dependent activation of proapoptotic caspase-3 was significantly delayed in activated macrophages and resulted in caspase-1-dependent pyroptosis.  The transition to susceptibility was not limited to LPS activation; it was also seen in macrophages activated with other Toll-like receptor (TLR) ligands and intact nonviable bacteria.  Yptb infection triggered macrophage activation and activation of caspase-1 in vivo.  Y. pestis infection of activated macrophages also stimulated caspase-1 activation.  These results indicate that host signaling triggered by TLR and other activating ligands during the course of Yersinia infection redirects both the mechanism of host cell death and the downstream consequences of death by shifting from noninflammatory apoptosis to inflammatory pyroptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRL3C0hooqMyzEwO064Z95rfW6udTcc2eZTKPtsUdDiwLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjhvFymtQ%253D%253D&md5=0489d306a5ee2f455204cf65cbed236a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.0030161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.0030161%26sid%3Dliteratum%253Aachs%26aulast%3DBergsbaken%26aufirst%3DT.%26aulast%3DCookson%26aufirst%3DB.%2BT.%26atitle%3DMacrophage%2520activation%2520redirects%2520yersinia-infected%2520host%2520cell%2520death%2520from%2520apoptosis%2520to%2520caspase-1-dependent%2520pyroptosis%26jtitle%3DPLoS%2520Pathog.%26date%3D2007%26volume%3D3%26doi%3D10.1371%2Fjournal.ppat.0030161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miao, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leaf, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treuting, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dors, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wewers, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aderem, A.</span></span> <span> </span><span class="NLM_article-title">Caspase-1-induced pyroptosis is an innate immune effector mechanism against intracellular bacteria</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1136</span>, <span class="refDoi"> DOI: 10.1038/ni.1960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fni.1960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=21057511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2jtrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=1136&author=E.+A.+Miaoauthor=I.+A.+Leafauthor=P.+M.+Treutingauthor=D.+P.+Maoauthor=M.+Dorsauthor=A.+Sarkarauthor=S.+E.+Warrenauthor=M.+D.+Wewersauthor=A.+Aderem&title=Caspase-1-induced+pyroptosis+is+an+innate+immune+effector+mechanism+against+intracellular+bacteria&doi=10.1038%2Fni.1960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Caspase-1-induced pyroptosis is an innate immune effector mechanism against intracellular bacteria</span></div><div class="casAuthors">Miao, Edward A.; Leaf, Irina A.; Treuting, Piper M.; Mao, Dat P.; Dors, Monica; Sarkar, Anasuya; Warren, Sarah E.; Wewers, Mark D.; Aderem, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1136-1142</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macrophages mediate crucial innate immune responses via caspase-1-dependent processing and secretion of interleukin 1β (IL-1β) and IL-18.  Although infection with wild-type Salmonella typhimurium is lethal to mice, we show here that a strain that persistently expresses flagellin was cleared by the cytosolic flagellin-detection pathway through the activation of caspase-1 by the NLRC4 inflammasome; however, this clearance was independent of IL-1β and IL-18.  Instead, caspase-1-induced pyroptotic cell death released bacteria from macrophages and exposed the bacteria to uptake and killing by reactive oxygen species in neutrophils.  Similarly, activation of caspase-1 cleared unmanipulated Legionella pneumophila and Burkholderia thailandensis by cytokine-independent mechanisms.  This demonstrates that activation of caspase-1 clears intracellular bacteria in vivo independently of IL-1β and IL-18 and establishes pyroptosis as an efficient mechanism of bacterial clearance by the innate immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXdPstV9EOz7Vg90H21EOLACvtfcHk0lgfVXoGz5Mf3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2jtrrK&md5=1739613c63616f5c401a543819698d49</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fni.1960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.1960%26sid%3Dliteratum%253Aachs%26aulast%3DMiao%26aufirst%3DE.%2BA.%26aulast%3DLeaf%26aufirst%3DI.%2BA.%26aulast%3DTreuting%26aufirst%3DP.%2BM.%26aulast%3DMao%26aufirst%3DD.%2BP.%26aulast%3DDors%26aufirst%3DM.%26aulast%3DSarkar%26aufirst%3DA.%26aulast%3DWarren%26aufirst%3DS.%2BE.%26aulast%3DWewers%26aufirst%3DM.%2BD.%26aulast%3DAderem%26aufirst%3DA.%26atitle%3DCaspase-1-induced%2520pyroptosis%2520is%2520an%2520innate%2520immune%2520effector%2520mechanism%2520against%2520intracellular%2520bacteria%26jtitle%3DNat.%2520Immunol.%26date%3D2010%26volume%3D11%26spage%3D1136%26doi%3D10.1038%2Fni.1960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Measures, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of bromodomain–acetyl-lysine interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1021/cb500996u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500996u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=22-39&author=M.+Brandauthor=A.+M.+Measuresauthor=B.+G.+Wilsonauthor=W.+A.+Cortopassiauthor=R.+Alexanderauthor=M.+H%C3%B6ssauthor=D.+S.+Hewingsauthor=T.+P.+Rooneyauthor=R.+S.+Patonauthor=S.+J.+Conway&title=Small+molecule+inhibitors+of+bromodomain%E2%80%93acetyl-lysine+interactions&doi=10.1021%2Fcb500996u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Bromodomain-Acetyl-lysine Interactions</span></div><div class="casAuthors">Brand, Michael; Measures, Angelina R.; Wilson, Brian G.; Cortopassi, Wilian A.; Alexander, Rikki; Hoss, Matthias; Hewings, David S.; Rooney, Timothy P. C.; Paton, Robert S.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-39</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Bromodomains are protein modules that bind to acetylated lysine residues.  Their interaction with histone proteins suggests that they function as "readers" of histone lysine acetylation, a component of the proposed "histone code".  Bromodomain-contg. proteins are often found as components of larger protein complexes with roles in fundamental cellular process including transcription.  The publication of two potent ligands for the BET bromodomains in 2010 demonstrated that small mols. can inhibit the bromodomain-acetyl-lysine protein-protein interaction.  These mols. display strong phenotypic effects in a no. of cell lines and affect a range of cancers in vivo.  This work stimulated intense interest in developing further ligands for the BET bromodomains and the design of ligands for non-BET bromodomains.  Here we review the recent progress in the field with particular attention paid to ligand design, the assays employed in early ligand discovery, and the use of computational approaches to inform ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1JZtWpBpqS7Vg90H21EOLACvtfcHk0lgfVXoGz5Mf3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D&md5=e40e85fc300109654712a00de400c8ba</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fcb500996u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500996u%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DMeasures%26aufirst%3DA.%2BM.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DAlexander%26aufirst%3DR.%26aulast%3DH%25C3%25B6ss%26aufirst%3DM.%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520bromodomain%25E2%2580%2593acetyl-lysine%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D22%26epage%3D39%26doi%3D10.1021%2Fcb500996u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winston, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">The bromodomain: a chromatin-targeting module?</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">601</span>, <span class="refDoi"> DOI: 10.1038/10640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2F10640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10404206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADyaK1MXksVentbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=601&author=F.+Winstonauthor=C.+D.+Allis&title=The+bromodomain%3A+a+chromatin-targeting+module%3F&doi=10.1038%2F10640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain: a chromatin-targeting module?</span></div><div class="casAuthors">Winston, Fred; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">601-604</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">A review and discussion with 69 refs.  It has recently been demonstrated that bromodomains - motifs found in several eukaryotic transcription factors - bind to acetyl-lysine, a modification of histones that is important for transcription.  This finding suggests that the regulatory effects of histone acetylation may be exerted by bromodomain-contg. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-zV1Z0XMw8LVg90H21EOLACvtfcHk0liK8Hy-9lOpXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXksVentbs%253D&md5=1d5aa880eaaa4734b39d49c49c874bb3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F10640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F10640%26sid%3Dliteratum%253Aachs%26aulast%3DWinston%26aufirst%3DF.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520bromodomain%253A%2520a%2520chromatin-targeting%2520module%253F%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D601%26doi%3D10.1038%2F10640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuppusetty, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">3,5-dimethylisoxazoles act as acetyl-lysine-mimetic bromodomain ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6761</span>– <span class="NLM_lpage">6770</span>, <span class="refDoi"> DOI: 10.1021/jm200640v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&author=D.+S.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knapp&title=3%2C5-dimethylisoxazoles+act+as+acetyl-lysine-mimetic+bromodomain+ligands&doi=10.1021%2Fjm200640v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Dimethylisoxazoles Act As Acetyl-lysine-mimetic Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Wang, Minghua; Philpott, Martin; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Vuppusetty, Chaitanya; Marsden, Brian; Knapp, Stefan; Conway, Stuart J.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6761-6770</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-lysine acetylation is a vital chromatin post-translational modification involved in the epigenetic regulation of gene transcription.  Bromodomains bind acetylated lysines, acting as readers of the histone-acetylation code.  Competitive inhibitors of this interaction have antiproliferative and anti-inflammatory properties.  With 57 distinct bromodomains known, the discovery of subtype-selective inhibitors of the histone-bromodomain interaction is of great importance.  We have identified the 3,5-dimethylisoxazole moiety as a novel acetyl-lysine bioisostere, which displaces acetylated histone-mimicking peptides from bromodomains.  Using X-ray crystallog. anal., we have detd. the interactions responsible for the activity and selectivity of 4-substituted 3,5-dimethylisoxazoles against a selection of phylogenetically diverse bromodomains.  By exploiting these interactions, we have developed compd. 4d, which has IC50 values of <5 μM for the bromodomain-contg. proteins BRD2(1) and BRD4(1).  These compds. are promising leads for the further development of selective probes for the bromodomain and extra C-terminal domain (BET) family and CREBBP bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEzK4xMvJhXrVg90H21EOLACvtfcHk0liK8Hy-9lOpXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO&md5=7d6f9eff2735c8a7189dec4fb5c5d13d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3D3%252C5-dimethylisoxazoles%2520act%2520as%2520acetyl-lysine-mimetic%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6761%26epage%3D6770%26doi%3D10.1021%2Fjm200640v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangos&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0liK8Hy-9lOpXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. H.</span></span> <span> </span><span class="NLM_article-title">RNA polymerase II elongation control</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">119</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-052610-095910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1146%2Fannurev-biochem-052610-095910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=22404626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVGls7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2012&pages=119&author=Q.+Zhouauthor=T.+Liauthor=D.+H.+Price&title=RNA+polymerase+II+elongation+control&doi=10.1146%2Fannurev-biochem-052610-095910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RNA polymerase II elongation control</span></div><div class="casAuthors">Zhou, Qiang; Li, Tiandao; Price, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">119-143</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Regulation of the elongation phase of transcription by RNA polymerase II (Pol II) is utilized extensively to generate the pattern of mRNAs needed to specify cell types and to respond to environmental changes.  After Pol II initiates, neg. elongation factors cause it to pause in a promoter proximal position.  These polymerases are poised to respond to the pos. transcription elongation factor P-TEFb, and then enter productive elongation only under the appropriate set of signals to generate full-length properly processed mRNAs.  Recent global analyses of Pol II and elongation factors, mechanisms that regulate P-TEFb involving the 7SK small nuclear ribonucleoprotein (snRNP), factors that control both the neg. and pos. elongation properties of Pol II, and the mRNA processing events that are coupled with elongation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60CSYt9WFzLVg90H21EOLACvtfcHk0liK8Hy-9lOpXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVGls7bN&md5=38b677c9655caf0e767b1ea61e4b47b9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-052610-095910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-052610-095910%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DPrice%26aufirst%3DD.%2BH.%26atitle%3DRNA%2520polymerase%2520II%2520elongation%2520control%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2012%26volume%3D81%26spage%3D119%26doi%3D10.1146%2Fannurev-biochem-052610-095910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolgemuth, D. J.</span></span> <span> </span><span class="NLM_article-title">The first bromodomain of Brdt, a testis-specific member of the BET sub-family of double-bromodomain-containing proteins, is essential for male germ cell differentiation</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">3507</span>, <span class="refDoi"> DOI: 10.1242/dev.004481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1242%2Fdev.004481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=17728347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlSitbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2007&pages=3507&author=E.+Shangauthor=H.+D.+Nickersonauthor=D.+Wenauthor=X.+Wangauthor=D.+J.+Wolgemuth&title=The+first+bromodomain+of+Brdt%2C+a+testis-specific+member+of+the+BET+sub-family+of+double-bromodomain-containing+proteins%2C+is+essential+for+male+germ+cell+differentiation&doi=10.1242%2Fdev.004481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The first bromodomain of Brdt, a testis-specific member of the BET sub-family of double-bromodomain-containing proteins, is essential for male germ cell differentiation</span></div><div class="casAuthors">Shang, Enyuan; Nickerson, Helen D.; Wen, Duancheng; Wang, Xiangyuan; Wolgemuth, Debra J.</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3507-3515</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Brdt is a testis-specific member of the distinctive BET sub-family of bromodomain motif-contg. proteins, a motif that binds acetylated lysines and is implicated in chromatin remodeling.  Its expression is restricted to the germ line, specifically to pachytene and diplotene spermatocytes and early spermatids.  Targeted mutagenesis was used to generate mice carrying a mutant allele of Brdt, BrdtΔBD1, which lacks only the first of the two bromodomains that uniquely characterize BET proteins.  Homozygous BrdtΔBD1/ΔBD1 mice were viable but males were sterile, producing fewer and morphol. abnormal sperm.  Aberrant morphogenesis was first detected in step 9 elongating spermatids, and those elongated spermatids that were formed lacked the distinctive foci of heterochromatin at the peri-nuclear envelope.  Quant. reverse transcription (RT)-PCR showed threefold increased levels of histone H1t (Hist1h1t) in BrdtΔBD1/ΔBD1 testes and chromatin immunopptn. revealed that Brdt protein, but not BrdtΔBD1 protein, was assocd. with the promoter of H1t.  Intracytoplasmic sperm injection suggested that the DNA in the BrdtΔBD1 mutant sperm could support early embryonic development and yield functional embryonic stem cells.  This is the first demonstration that deletion of just one of the two bromodomains in members of the BET sub-family of bromodomain-contg. proteins has profound effects on in vivo differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDa2lSw-mPaLVg90H21EOLACvtfcHk0lh8e7N9LGtEtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlSitbfJ&md5=f8ef8a8ab721f0eebf4441ffc35eee93</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1242%2Fdev.004481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.004481%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DE.%26aulast%3DNickerson%26aufirst%3DH.%2BD.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWolgemuth%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520first%2520bromodomain%2520of%2520Brdt%252C%2520a%2520testis-specific%2520member%2520of%2520the%2520BET%2520sub-family%2520of%2520double-bromodomain-containing%2520proteins%252C%2520is%2520essential%2520for%2520male%2520germ%2520cell%2520differentiation%26jtitle%3DDevelopment%26date%3D2007%26volume%3D134%26spage%3D3507%26doi%3D10.1242%2Fdev.004481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0lh8e7N9LGtEtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delmore, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilpatrick, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qi&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0lh8e7N9LGtEtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Q.+Liuauthor=P.+Y.+Wangauthor=H.+Y.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lhtH385nFWTFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%2BQ.%26aulast%3DWang%26aufirst%3DP.%2BY.%26aulast%3DChen%26aufirst%3DH.%2BY.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medzhitov, R.</span></span> <span> </span><span class="NLM_article-title">Control of inducible gene expression by signal-dependent transcriptional elongation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.05.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.cell.2009.05.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=19596240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1ygurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2009&pages=129-145&author=D.+C.+Hargreavesauthor=T.+Horngauthor=R.+Medzhitov&title=Control+of+inducible+gene+expression+by+signal-dependent+transcriptional+elongation&doi=10.1016%2Fj.cell.2009.05.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Control of inducible gene expression by signal-dependent transcriptional elongation</span></div><div class="casAuthors">Hargreaves, Diana C.; Horng, Tiffany; Medzhitov, Ruslan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-145</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Most inducible transcriptional programs consist of primary and secondary response genes (PRGs and SRGs) that differ in their kinetics of expression and in their requirements for new protein synthesis and chromatin remodeling.  Here we show that many PRGs, in contrast to SRGs, have preassembled RNA polymerase II (Pol II) and pos. histone modifications at their promoters in the basal state.  Pol II at PRGs generates low levels of full-length unspliced transcripts but fails to make mature, protein-coding transcripts in the absence of stimulation.  Induction of PRGs is controlled at the level of transcriptional elongation and mRNA processing, through the signal-dependent recruitment of P-TEFb.  P-TEFb is in turn recruited by the bromodomain-contg. protein Brd4, which detects H4K5/8/12Ac inducibly acquired at PRG promoters.  Our findings suggest that the permissive structure of PRGs both stipulates their unique regulation in the basal state by corepressor complexes and enables their rapid induction in multiple cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFmjH5y-GkbrVg90H21EOLACvtfcHk0lhtH385nFWTFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1ygurk%253D&md5=e5292556b16506183db147693db3450a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.05.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.05.047%26sid%3Dliteratum%253Aachs%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHorng%26aufirst%3DT.%26aulast%3DMedzhitov%26aufirst%3DR.%26atitle%3DControl%2520of%2520inducible%2520gene%2520expression%2520by%2520signal-dependent%2520transcriptional%2520elongation%26jtitle%3DCell%26date%3D2009%26volume%3D138%26spage%3D129%26epage%3D145%26doi%3D10.1016%2Fj.cell.2009.05.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">BET N-terminal bromodomain inhibition selectively blocks Th17 cell differentiation and ameliorates colitis in mice</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2952</span>, <span class="refDoi"> DOI: 10.1073/pnas.1615601114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1073%2Fpnas.1615601114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=28265070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVyjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=2952&author=K.+Cheungauthor=G.+Luauthor=R.+Sharmaauthor=A.+Vincekauthor=R.+Zhangauthor=A.+N.+Plotnikovauthor=F.+Zhangauthor=Q.+Zhangauthor=Y.+Juauthor=Y.+Hu&title=BET+N-terminal+bromodomain+inhibition+selectively+blocks+Th17+cell+differentiation+and+ameliorates+colitis+in+mice&doi=10.1073%2Fpnas.1615601114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">BET N-terminal bromodomain inhibition selectively blocks Th17 cell differentiation and ameliorates colitis in mice</span></div><div class="casAuthors">Cheung, Kalung; Lu, Geming; Sharma, Rajal; Vincek, Adam; Zhang, Ruihua; Plotnikov, Alexander N.; Zhang, Fan; Zhang, Qiang; Ju, Ying; Hu, Yuan; Zhao, Li; Han, Xinye; Meslamani, Jamel; Xu, Feihong; Jaganathan, Anbalagan; Shen, Tong; Zhu, Hongfa; Rusinova, Elena; Zeng, Lei; Zhou, Jiachi; Yang, Jianjun; Peng, Liang; Ohlmeyer, Michael; Walsh, Martin J.; Zhang, David Y.; Xiong, Huabao; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2952-2957</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">T-helper 17 (Th17) cells have important functions in adaptor immunity and have also been implicated in inflammatory disorders.  The bromodomain and extraterminal domain (BET) family proteins regulate gene transcription during lineage-specific differentiation of naive CD4+ T cells to produce mature T-helper cells.  Inhibition of acetyl-lysine binding of the BET proteins by pan-BET bromodomain (BrD) inhibitors, such as JQ1, broadly affects differentiation of Th17, Th1, and Th2 cells that have distinct immune functions, thus limiting their therapeutic potential.  Whether these BET proteins represent viable new epigenetic drug targets for inflammatory disorders has remained an unanswered question.  In this study, we report that selective inhibition of the first bromodomain of BET proteins with our newly designed small mol. MS402 inhibits primarily Th17 cell differentiation with a little or almost no effect on Th1 or Th2 and Treg cells.  MS402 preferentially renders Brd4 binding to Th17 signature gene loci over those of housekeeping genes and reduces Brd4 recruitment of p-TEFb to phosphorylate and activate RNA polymerase II for transcription elongation.  We further show that MS402 prevents and ameliorates T-cell transfer-induced colitis in mice by blocking Th17 cell overdevelopment.  Thus, selective pharmacol. modulation of individual bromodomains likely represents a strategy for treatment of inflammatory bowel diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBIvlTKnR97LVg90H21EOLACvtfcHk0lhtH385nFWTFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVyjsbo%253D&md5=367b7c0545edde6cc0ef107e03e1310f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1615601114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1615601114%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DJu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DY.%26atitle%3DBET%2520N-terminal%2520bromodomain%2520inhibition%2520selectively%2520blocks%2520Th17%2520cell%2520differentiation%2520and%2520ameliorates%2520colitis%2520in%2520mice%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3D2952%26doi%3D10.1073%2Fpnas.1615601114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span>; <span class="NLM_string-name">Zhu, L.</span>; <span class="NLM_string-name">Zhang, T.</span>; <span class="NLM_string-name">Lu, H.</span>; <span class="NLM_string-name">Wang, C.</span>; <span class="NLM_string-name">Xue, B.</span>; <span class="NLM_string-name">Xu, X.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Cai, Z.</span>; <span class="NLM_string-name">Sang, W.</span></span> <span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">BRD4 has dual effects on the HMGB1 and NF-κB signalling pathways and is a potential therapeutic target for osteoarthritis</span>. <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume">1863</span>,  <span class="NLM_fpage">3001</span>. <span class="refDoi"> DOI: 10.1016/j.bbadis.2017.08.009</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.bbadis.2017.08.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1863&publication_year=2017&pages=3001&author=Y.+Jiang&author=L.+Zhu&author=T.+Zhang&author=H.+Lu&author=C.+Wang&author=B.+Xue&author=X.+Xu&author=Y.+Liu&author=Z.+Cai&author=W.+Sang&title=BRD4+has+dual+effects+on+the+HMGB1+and+NF-%CE%BAB+signalling+pathways+and+is+a+potential+therapeutic+target+for+osteoarthritis&doi=10.1016%2Fj.bbadis.2017.08.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbadis.2017.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbadis.2017.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DBRD4%2520has%2520dual%2520effects%2520on%2520the%2520HMGB1%2520and%2520NF-%25CE%25BAB%2520signalling%2520pathways%2520and%2520is%2520a%2520potential%2520therapeutic%2520target%2520for%2520osteoarthritis%26date%3D2017%26volume%3D1863%26spage%3D3001%26doi%3D10.1016%2Fj.bbadis.2017.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ball, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipman, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liby, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pioli, P. A.</span></span> <span> </span><span class="NLM_article-title">CDDO-me redirects activation of breast tumor associated macrophages</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0149600</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0149600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1371%2Fjournal.pone.0149600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=26918785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVylsrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=M.+S.+Ballauthor=E.+P.+Shipmanauthor=H.+Kimauthor=K.+T.+Libyauthor=P.+A.+Pioli&title=CDDO-me+redirects+activation+of+breast+tumor+associated+macrophages&doi=10.1371%2Fjournal.pone.0149600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">CDDO-Me redirects activation of breast tumor associated macrophages</span></div><div class="casAuthors">Ball, Michael S.; Shipman, Emilie P.; Kim, Hyunjung; Liby, Karen T.; Pioli, Patricia A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e0149600/1-e0149600/18</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Tumor-assocd. macrophages can account for up to 50% of the tumor mass in breast cancer patients and high TAM d. is assocd. with poor clin. prognosis.  Because TAMs enhance tumor growth, development, and metastatic potential, redirection of TAM activation may have significant therapeutic benefit.  Our studies in primary human macrophages and murine breast TAMs suggest that the synthetic oleanane triterpenoid CDDOmethyl ester (CDDO-Me) reprograms the activation profile of TAMs from tumor-promoting to tumor-inhibiting.  We show that CDDO-Me treatment inhibits expression of IL-10 and VEGF in stimulated human M2 macrophages and TAMs but increases expression of TNF-α and IL-6.  Surface expression of CD206 and CD163, which are characteristic of M2 activation, is significantly attenuated by CDDO-Me.  In contrast, CDDO-Me up-regulates surface expression of HLA-DR and CD80, which are markers of M1 activation, and importantly potentiates macrophage activation of autologous T cells but inhibits endothelial cell vascularization.  These results show for the first time that CDDO-Me redirects activation of M2 macrophages and TAMs from immune-suppressive to immune-stimulatory, and implicate a role for CDDO-Me as an immunotherapeutic in the treatment of breast and potentially other types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO7q7YdphPZ7Vg90H21EOLACvtfcHk0lggtHnAmAePXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVylsrzN&md5=6b3b749c446c1deca82158999752e153</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0149600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0149600%26sid%3Dliteratum%253Aachs%26aulast%3DBall%26aufirst%3DM.%2BS.%26aulast%3DShipman%26aufirst%3DE.%2BP.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DLiby%26aufirst%3DK.%2BT.%26aulast%3DPioli%26aufirst%3DP.%2BA.%26atitle%3DCDDO-me%2520redirects%2520activation%2520of%2520breast%2520tumor%2520associated%2520macrophages%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0149600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0lggtHnAmAePXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabiec, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ospelt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422-429&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+E.+Gayauthor=C.+Kollingauthor=L.+L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- α, interleukin (IL)-1β and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- α and IL-1β .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0lggtHnAmAePXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BE.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.%2BL.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D422%26epage%3D429%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span> <span> </span><span class="NLM_article-title">Bromodomain protein inhibition: a novel therapeutic strategy in rheumatic diseases</span>. <i>RMD Open</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e000744</span> <span class="refDoi"> DOI: 10.1136/rmdopen-2018-000744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1136%2Frmdopen-2018-000744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30564450" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&author=K.+Klein&title=Bromodomain+protein+inhibition%3A+a+novel+therapeutic+strategy+in+rheumatic+diseases&doi=10.1136%2Frmdopen-2018-000744"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1136%2Frmdopen-2018-000744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Frmdopen-2018-000744%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26atitle%3DBromodomain%2520protein%2520inhibition%253A%2520a%2520novel%2520therapeutic%2520strategy%2520in%2520rheumatic%2520diseases%26jtitle%3DRMD%2520Open%26date%3D2018%26volume%3D4%26doi%3D10.1136%2Frmdopen-2018-000744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sherbeeny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoheir, K. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attia, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hosaini, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Sharary, S. D.</span></span> <span> </span><span class="NLM_article-title">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.phrs.2015.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=26149470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=248-257&author=A.+Nadeemauthor=N.+O.+Al-Harbiauthor=M.+M.+Al-Harbiauthor=A.+M.+El-Sherbeenyauthor=S.+F.+Ahmadauthor=N.+Siddiquiauthor=M.+A.+Ansariauthor=K.+M.+A.+Zoheirauthor=S.+M.+Attiaauthor=K.+A.+Al-Hosainiauthor=S.+D.+Al-Sharary&title=Imiquimod-induced+psoriasis-like+skin+inflammation+is+suppressed+by+BET+bromodomain+inhibitor+in+mice+through+RORC%2FIL-17A+pathway+modulation&doi=10.1016%2Fj.phrs.2015.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span></div><div class="casAuthors">Nadeem, Ahmed; Al-Harbi, Naif O.; Al-Harbi, Mohamed M.; El-Sherbeeny, Ahmed M.; Ahmad, Sheikh F.; Siddiqui, Nahid; Ansari, Mushtaq A.; Zoheir, Khairy M. A.; Attia, Sabry M.; Al-Hosaini, Khaled A.; Al-Sharary, Shakir D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Psoriasis is one of the most common skin disorders characterized by erythematous plaques that result from hyperproliferative keratinocytes and infiltration of inflammatory leukocytes into dermis and epidermis.  Recent studies suggest that IL-23/IL-17A/IL-22 cytokine axis plays an important role in the pathogenesis of psoriasis.  The small mol. bromodomain and extraterminal domain (BET) inhibitors, that disrupt interaction of BET proteins with acetylated histones have recently demonstrated efficacy in various models of inflammation through suppression of several pathways, one of them being synthesis of IL-17A/IL-22 which primarily depends on transcription factor, retinoic acid receptor-related orphan receptor C (RORC).  However, the efficacy and mechanistic aspect of a BET inhibitor in mouse model of skin inflammation has not been explored previously.  Therefore, this study investigated the role of BET inhibitor, JQ-1 in mouse model of psoriasis-like inflammation.  Mice were topically applied imiquimod (IMQ) to develop psoriasis-like inflammation on the shaved back and ear followed by assessment of skin inflammation (myeloperoxidase activity, ear thickness, and histopathol.), RORC and its signature cytokines (IL-17A/IL-22).  JQ-1 suppressed IMQ-induced skin inflammation as reflected by a decrease in ear thickness/myeloperoxidase activity, and RORC/IL-17A/IL-22 expression.  Addnl., a RORα/γ agonist SR1078 was utilized to investigate the role of RORC in BET-mediated skin inflammation.  SR1078 reversed the protective effect of JQ-1 on skin inflammation at both histol. and mol. levels in the IMQ model.  The current study suggests that BET bromodomains are involved in psoriasis-like inflammation through induction of RORC/IL-17A pathway.  Therefore, inhibition of BET bromodomains may provide a new therapy against skin inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOnC_sPUMSLVg90H21EOLACvtfcHk0lgBit-H4O-jYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I&md5=3edb10526e30c18bce037dc0aca611a4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%26aufirst%3DA.%26aulast%3DAl-Harbi%26aufirst%3DN.%2BO.%26aulast%3DAl-Harbi%26aufirst%3DM.%2BM.%26aulast%3DEl-Sherbeeny%26aufirst%3DA.%2BM.%26aulast%3DAhmad%26aufirst%3DS.%2BF.%26aulast%3DSiddiqui%26aufirst%3DN.%26aulast%3DAnsari%26aufirst%3DM.%2BA.%26aulast%3DZoheir%26aufirst%3DK.%2BM.%2BA.%26aulast%3DAttia%26aufirst%3DS.%2BM.%26aulast%3DAl-Hosaini%26aufirst%3DK.%2BA.%26aulast%3DAl-Sharary%26aufirst%3DS.%2BD.%26atitle%3DImiquimod-induced%2520psoriasis-like%2520skin%2520inflammation%2520is%2520suppressed%2520by%2520BET%2520bromodomain%2520inhibitor%2520in%2520mice%2520through%2520RORC%252FIL-17A%2520pathway%2520modulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D248%26epage%3D257%26doi%3D10.1016%2Fj.phrs.2015.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, W. G.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and extra-terminal (BET) protein inhibitors suppress chondrocyte differentiation and restrain bone growth</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">26647</span>– <span class="NLM_lpage">26657</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.749697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1074%2Fjbc.M116.749697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27821592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCmurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=26647-26657&author=N.+N.+Niuauthor=R.+Shaoauthor=G.+Yanauthor=W.+G.+Zou&title=Bromodomain+and+extra-terminal+%28BET%29+protein+inhibitors+suppress+chondrocyte+differentiation+and+restrain+bone+growth&doi=10.1074%2Fjbc.M116.749697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain and Extra-terminal (BET) Protein Inhibitors Suppress Chondrocyte Differentiation and Restrain Bone Growth</span></div><div class="casAuthors">Niu, Ningning; Shao, Rui; Yan, Guang; Zou, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">26647-26657</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Small mol. inhibitors for bromodomain and extra-terminal (BET) proteins have recently emerged as potential therapeutic agents in clin. trials for various cancers.  However, to date, it is unknown whether these inhibitors have side effects on bone structures.  Here, we report that inhibition of BET bromodomain proteins may suppress chondrocyte differentiation and restrain bone growth.  We generated a luciferase reporter system using the chondrogenic cell line ATDC5 in which the luciferase gene was driven by the promoter of Col2a1, an elementary collagen of the chondrocyte.  The Col2a1-luciferase ATDC5 system was used for rapidly screening both activators and repressors of human collagen Col2a1 gene expression, and we found that BET bromodomain inhibitors reduce the Col2a1-luciferase.  Consistent with the luciferase assay, BET inhibitors decrease the expression of Col2a1.  Furthermore, we constructed a zebrafish line in which the enhanced green fluorescent protein (EGFP) expression was driven by col2a1 promoter.  The transgenic (col2a1-EGFP) zebrafish line demonstrated that BET inhibitors I-BET151 and (+)-JQ1 may affect EGFP expression in zebrafish.  Furthermore, we found that I-BET151 and (+)-JQ1 may affect chondrocyte differentiation in vitro and inhibit zebrafish growth in vivo.  Mechanistic anal. revealed that BET inhibitors influenced the depletion of RNA polymerase II from the Col2a1 promoter.  Collectively, these results suggest that BET bromodomain inhibition may have side effects on skeletal bone structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr97hnBq7K0OrVg90H21EOLACvtfcHk0lgBit-H4O-jYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCmurnF&md5=d0518ae2f1e7d71ca1e87148848f8707</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.749697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.749697%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DN.%2BN.%26aulast%3DShao%26aufirst%3DR.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DZou%26aufirst%3DW.%2BG.%26atitle%3DBromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520protein%2520inhibitors%2520suppress%2520chondrocyte%2520differentiation%2520and%2520restrain%2520bone%2520growth%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D26647%26epage%3D26657%26doi%3D10.1074%2Fjbc.M116.749697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletar&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lgBit-H4O-jYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romain, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jérôme, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mélanie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean-Marie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miriam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=G.+Romainauthor=T.+J%C3%A9r%C3%B4meauthor=C.+A.+Cl%C3%A9mentauthor=B.+M%C3%A9lanieauthor=B.+Jean-Marieauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=C.+Miriamauthor=P.+Olivier&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0lhpAVTI36eIkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRomain%26aufirst%3DG.%26aulast%3DJ%25C3%25A9r%25C3%25B4me%26aufirst%3DT.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DM%25C3%25A9lanie%26aufirst%3DB.%26aulast%3DJean-Marie%26aufirst%3DB.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DMiriam%26aufirst%3DC.%26aulast%3DOlivier%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coudé, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berrou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffoux, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itzykson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17698</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.4131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.18632%2Foncotarget.4131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=25989842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17698&author=M.-M.+Coud%C3%A9author=T.+Braunauthor=J.+Berrouauthor=M.+Dupontauthor=S.+Bertrandauthor=A.+Masseauthor=E.+Raffouxauthor=R.+Itzyksonauthor=M.+Delordauthor=M.+E.+Riveiro&title=BET+inhibitor+OTX015+targets+BRD2+and+BRD4+and+decreases+c-MYC+in+acute+leukemia+cells&doi=10.18632%2Foncotarget.4131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span></div><div class="casAuthors">Coude Marie-Magdelaine; Braun Thorsten; Berrou Jeannig; Dupont Melanie; Bertrand Sibyl; Masse Aline; Raffoux Emmanuel; Itzykson Raphael; Baruchel Andre; Dombret Herve; Gardin Claude; Coude Marie-Magdelaine; Braun Thorsten; Gardin Claude; Raffoux Emmanuel; Itzykson Raphael; Dombret Herve; Delord Marc; Riveiro Maria E; Herait Patrice; Baruchel Andre</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">17698-712</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The bromodomain (BRD) and extraterminal (BET) proteins including BRD2, BRD3 and BRD4 have been identified as key targets for leukemia maintenance.  A novel oral inhibitor of BRD2/3/4, the thienotriazolodiazepine compound OTX015, suitable for human use, is available.  Here we report its biological effects in AML and ALL cell lines and leukemic samples.  Exposure to OTX015 lead to cell growth inhibition, cell cycle arrest and apoptosis at submicromolar concentrations in acute leukemia cell lines and patient-derived leukemic cells, as described with the canonical JQ1 BET inhibitor.  Treatment with JQ1 and OTX15 induces similar gene expression profiles in sensitive cell lines, including a c-MYC decrease and an HEXIM1 increase.  OTX015 exposure also induced a strong decrease of BRD2, BRD4 and c-MYC and increase of HEXIM1 proteins, while BRD3 expression was unchanged. c-MYC, BRD2, BRD3, BRD4 and HEXIM1 mRNA levels did not correlate however with viability following exposure to OTX015.  Sequential combinations of OTX015 with other epigenetic modifying drugs, panobinostat and azacitidine have a synergic effect on growth of the KASUMI cell line.  Our results indicate that OTX015 and JQ1 have similar biological effects in leukemic cells, supporting OTX015 evaluation in a Phase Ib trial in relapsed/refractory leukemia patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSlHRu3Td8hIa_dlEzl_fBafW6udTcc2eYRXbRXpk8TiLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D&md5=0db953a0a24b12ec2918a02d1a2df00b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4131%26sid%3Dliteratum%253Aachs%26aulast%3DCoud%25C3%25A9%26aufirst%3DM.-M.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DBerrou%26aufirst%3DJ.%26aulast%3DDupont%26aufirst%3DM.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DMasse%26aufirst%3DA.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DItzykson%26aufirst%3DR.%26aulast%3DDelord%26aufirst%3DM.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26atitle%3DBET%2520inhibitor%2520OTX015%2520targets%2520BRD2%2520and%2520BRD4%2520and%2520decreases%2520c-MYC%2520in%2520acute%2520leukemia%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17698%26doi%3D10.18632%2Foncotarget.4131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowlati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, S.</span></span> <span> </span><span class="NLM_article-title">Clinically efficacy of the BET bromodomain inhibitor TEN-010 in an open-label substudy with patients with documented NUT-midline carcinoma (NMC)</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">AbstractnrA49</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=AbstractnrA49&author=G.+Shapiroauthor=A.+Dowlatiauthor=P.+LoRussoauthor=J.+Ederauthor=A.+Andersonauthor=K.+Doauthor=M.+Kageyauthor=C.+Sirardauthor=J.+Bradnerauthor=S.+Landau&title=Clinically+efficacy+of+the+BET+bromodomain+inhibitor+TEN-010+in+an+open-label+substudy+with+patients+with+documented+NUT-midline+carcinoma+%28NMC%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShapiro%26aufirst%3DG.%26aulast%3DDowlati%26aufirst%3DA.%26aulast%3DLoRusso%26aufirst%3DP.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DK.%26aulast%3DKagey%26aufirst%3DM.%26aulast%3DSirard%26aufirst%3DC.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DLandau%26aufirst%3DS.%26atitle%3DClinically%2520efficacy%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520TEN-010%2520in%2520an%2520open-label%2520substudy%2520with%2520patients%2520with%2520documented%2520NUT-midline%2520carcinoma%2520%2528NMC%2529%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DAbstractnrA49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koulnis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N. S.</span></span> <span> </span><span class="NLM_article-title">Effect of the BET inhibitor, CPI-0610, alone and in combination with lenalidomide in multiple myeloma</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">4255</span>, <span class="refDoi"> DOI: 10.1182/blood.V126.23.4255.4255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1182%2Fblood.V126.23.4255.4255" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=4255&author=K.+T.+Siuauthor=H.+Edaauthor=L.+Santoauthor=J.+Ramachandranauthor=M.+Koulnisauthor=J.+Mertzauthor=R.+J.+Simsauthor=M.+Cooperauthor=N.+S.+Raje&title=Effect+of+the+BET+inhibitor%2C+CPI-0610%2C+alone+and+in+combination+with+lenalidomide+in+multiple+myeloma&doi=10.1182%2Fblood.V126.23.4255.4255"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1182%2Fblood.V126.23.4255.4255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V126.23.4255.4255%26sid%3Dliteratum%253Aachs%26aulast%3DSiu%26aufirst%3DK.%2BT.%26aulast%3DEda%26aufirst%3DH.%26aulast%3DSanto%26aufirst%3DL.%26aulast%3DRamachandran%26aufirst%3DJ.%26aulast%3DKoulnis%26aufirst%3DM.%26aulast%3DMertz%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DCooper%26aufirst%3DM.%26aulast%3DRaje%26aufirst%3DN.%2BS.%26atitle%3DEffect%2520of%2520the%2520BET%2520inhibitor%252C%2520CPI-0610%252C%2520alone%2520and%2520in%2520combination%2520with%2520lenalidomide%2520in%2520multiple%2520myeloma%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26spage%3D4255%26doi%3D10.1182%2Fblood.V126.23.4255.4255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingard&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lhKPDbLc9K0gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olp, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprague, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovering and exploiting selectivity in BET tandem bromodomain recognition of epigenetic lysine acylation</span>. <i>Faseb J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=2&author=B.+C.+Smithauthor=M.+D.+Olpauthor=D.+J.+Sprague&title=Discovering+and+exploiting+selectivity+in+BET+tandem+bromodomain+recognition+of+epigenetic+lysine+acylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BC.%26aulast%3DOlp%26aufirst%3DM.%2BD.%26aulast%3DSprague%26aufirst%3DD.%2BJ.%26atitle%3DDiscovering%2520and%2520exploiting%2520selectivity%2520in%2520BET%2520tandem%2520bromodomain%2520recognition%2520of%2520epigenetic%2520lysine%2520acylation%26jtitle%3DFaseb%2520J.%26date%3D2018%26volume%3D32%26spage%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A.</span></span> <span> </span><span class="NLM_article-title">RVX 208: a novel BET protein inhibitor, role as an inducer of apo A-I/HDL and beyond</span>. <i>Cardiovasc. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>, <span class="NLM_elocation-id">e12265</span> <span class="refDoi"> DOI: 10.1111/1755-5922.12265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1111%2F1755-5922.12265" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&author=G.+C.+Ghoshauthor=R.+Bhadraauthor=R.+K.+Ghoshauthor=K.+Banerjeeauthor=A.+Gupta&title=RVX+208%3A+a+novel+BET+protein+inhibitor%2C+role+as+an+inducer+of+apo+A-I%2FHDL+and+beyond&doi=10.1111%2F1755-5922.12265"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2F1755-5922.12265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252F1755-5922.12265%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DG.%2BC.%26aulast%3DBhadra%26aufirst%3DR.%26aulast%3DGhosh%26aufirst%3DR.%2BK.%26aulast%3DBanerjee%26aufirst%3DK.%26aulast%3DGupta%26aufirst%3DA.%26atitle%3DRVX%2520208%253A%2520a%2520novel%2520BET%2520protein%2520inhibitor%252C%2520role%2520as%2520an%2520inducer%2520of%2520apo%2520A-I%252FHDL%2520and%2520beyond%26jtitle%3DCardiovasc.%2520Ther.%26date%3D2017%26volume%3D35%26doi%3D10.1111%2F1755-5922.12265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, E. L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. J.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition mechanism of RVX-208 to the second bromodomain of bromo and extraterminal proteins: insight from microsecond molecular dynamics simulations</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">8857</span> <span class="refDoi"> DOI: 10.1038/s41598-017-08909-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fs41598-017-08909-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=28821780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A280%3ADC%252BC1cfpslartg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&author=Q.+Q.+Wangauthor=Y.+Liauthor=J.+H.+Xuauthor=Y.+W.+Wangauthor=E.+L.+H.+Leungauthor=L.+Liuauthor=X.+J.+Yao&title=Selective+inhibition+mechanism+of+RVX-208+to+the+second+bromodomain+of+bromo+and+extraterminal+proteins%3A+insight+from+microsecond+molecular+dynamics+simulations&doi=10.1038%2Fs41598-017-08909-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition mechanism of RVX-208 to the second bromodomain of bromo and extraterminal proteins: insight from microsecond molecular dynamics simulations</span></div><div class="casAuthors">Wang Qianqian; Li Ying; Xu Jiahui; Wang Yuwei; Leung Elaine Lai-Han; Liu Liang; Yao Xiaojun</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8857</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">RVX-208 is a recently reported inhibitor of bromo and extraterminal (BET) family proteins (including BRD2-4 and BRDT) with selectivity for the second bromodomain (BD2), currently in phase III clinical trials.  Despite of its promising antitumor activity, due to the conserved folds of the first and second bromodomains (BD1 and BD2), the detailed selectivity mechanism of RVX-208 towards BD2 over BD1 is still unknown.  To elucidate selective inhibition mechanism of RVX-208 to BD2, microsecond molecular dynamics simulations were performed in this study for BRD2-BD1, BRD2-BD2 and BRD4-BD1 with and without RVX-208, respectively.  Binding free energy calculations show that there exists strongest interaction between RVX-208 and BRD2-BD2.  Leu383 and Asn429 are two most important residues of BRD2-BD2 for binding to RVX-208.  Structural network analysis reveals that RVX-208 can shorten the communication path of ZA and BC loops in BRD2-BD2 pocket, making pocket more suitable to accommodate RVX-208.  Additionally, different behaviors of His433 (Asp160 in BRD2-BD1) and Val435 (Ile162 in BRD2-BD1) in BRD2-BD2 are key factors responsible for selective binding of RVX-208 to BRD2-BD2.  The proposed selective inhibition mechanism of RVX-208 to BRD2-BD2 can be helpful for rational design of novel selective inhibitors of the second bromodomain of BET family proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnFJC4cRlg1GlSCSgBhwJFfW6udTcc2eb6Nib1ZqIKT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfpslartg%253D%253D&md5=30494be9728cf5d0dc16f68ea4b0c9da</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-08909-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-08909-8%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%2BQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%2BH.%26aulast%3DWang%26aufirst%3DY.%2BW.%26aulast%3DLeung%26aufirst%3DE.%2BL.%2BH.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DX.%2BJ.%26atitle%3DSelective%2520inhibition%2520mechanism%2520of%2520RVX-208%2520to%2520the%2520second%2520bromodomain%2520of%2520bromo%2520and%2520extraterminal%2520proteins%253A%2520insight%2520from%2520microsecond%2520molecular%2520dynamics%2520simulations%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26doi%3D10.1038%2Fs41598-017-08909-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajb, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>– <span class="NLM_lpage">4334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=W.+Ajbauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0ljsdLbVNv4mwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DAjb%26aufirst%3DW.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, E. L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. J.</span></span> <span> </span><span class="NLM_article-title">Computational study on the selective inhibition mechanism of MS402 to the first and second bromodomains of BRD4</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1002/prot.25611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1002%2Fprot.25611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30260047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFOis7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2019&pages=3-11&author=Q.+Q.+Wangauthor=Y.+Liauthor=J.+H.+Xuauthor=Y.+W.+Wangauthor=D.+F.+Shiauthor=L.+Liuauthor=E.+L.+H.+Leungauthor=X.+J.+Yao&title=Computational+study+on+the+selective+inhibition+mechanism+of+MS402+to+the+first+and+second+bromodomains+of+BRD4&doi=10.1002%2Fprot.25611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Computational study on the selective inhibition mechanism of MS402 to the first and second bromodomains of BRD4</span></div><div class="casAuthors">Wang, Qianqian; Li, Ying; Xu, Jiahui; Wang, Yuwei; Shi, Danfeng; Liu, Liang; Leung, Elaine Lai-Han; Yao, Xiaojun</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-11</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>;
        ISSN:<span class="NLM_cas:issn">1097-0134</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">As a member of the bromodomain and extraterminal domain (BET) family, BRD4 is considered as a potential target for cancer treatment.  However, because of the highly conservation of its two homologous bromodomains (BD1/BD2), selective inhibition of each bromodomain remains a challenge.  MS402 is a domain-selective inhibitor of BRD4-BD1 over BRD4-BD2 reported recently.  Understanding the selectivity mechanism would be very useful for the further design of more potent BD1-selectivity inhibitors.  Mol. dynamics simulation, adaptive biasing force and multiple-walker adaptive biasing force were performed to study the inhibition and domain-selective mechanism of MS402 toward BRD4-BD1 over BRD4-BD2 here.  Results demonstrate BRD4-BD1 binds to MS402 with lower binding free energy than BRD4-BD2.  Residues Gln85, Pro86, Asn140, and Ile146 are crucial for MS402's selectively binding to BRD4-BD1.  MS402 needs to overcome more energy barrier to dissoc. from BD1 than from BD2 pocket.  These findings will be helpful for rational structural modification of existing inhibitors to increase their BD1-selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoenr3BSeSNnLVg90H21EOLACvtfcHk0ljsdLbVNv4mwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFOis7vI&md5=4e2e93201e34b93a2acd09de604d07d2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fprot.25611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.25611%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%2BQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%2BH.%26aulast%3DWang%26aufirst%3DY.%2BW.%26aulast%3DShi%26aufirst%3DD.%2BF.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLeung%26aufirst%3DE.%2BL.%2BH.%26aulast%3DYao%26aufirst%3DX.%2BJ.%26atitle%3DComputational%2520study%2520on%2520the%2520selective%2520inhibition%2520mechanism%2520of%2520MS402%2520to%2520the%2520first%2520and%2520second%2520bromodomains%2520of%2520BRD4%26jtitle%3DProteins%26date%3D2019%26volume%3D87%26spage%3D3%26epage%3D11%26doi%3D10.1002%2Fprot.25611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of potent diazobenzene inhibitors for the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9251</span>– <span class="NLM_lpage">9264</span>, <span class="refDoi"> DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&author=G.+T.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.+M.+Zhou&title=Structure-guided+design+of+potent+diazobenzene+inhibitors+for+the+BET+bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0ljsdLbVNv4mwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%2BT.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure-guided%2520design%2520of%2520potent%2520diazobenzene%2520inhibitors%2520for%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzo[<i>cd</i>]indol-2(1<i>H</i>)-ones as potent and specific BET bromodomain inhibitors: structure-based virtual screening, optimization, and biological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1565</span>– <span class="NLM_lpage">1579</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01511</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01511" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs1yqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1565-1579&author=X.+Xueauthor=Y.+Zhangauthor=Z.+Liuauthor=M.+Songauthor=Y.+Xingauthor=Q.+Xiangauthor=Z.+Wangauthor=Z.+Tuauthor=Y.+Zhouauthor=K.+Ding&title=Discovery+of+benzo%5Bcd%5Dindol-2%281H%29-ones+as+potent+and+specific+BET+bromodomain+inhibitors%3A+structure-based+virtual+screening%2C+optimization%2C+and+biological+evaluation&doi=10.1021%2Facs.jmedchem.5b01511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzo[cd]indol-2(1H)-ones as Potent and Specific BET Bromodomain Inhibitors: Structure-Based Virtual Screening, Optimization, and Biological Evaluation</span></div><div class="casAuthors">Xue, Xiaoqian; Zhang, Yan; Liu, Zhaoxuan; Song, Ming; Xing, Yanli; Xiang, Qiuping; Wang, Zhen; Tu, Zhengchao; Zhou, Yulai; Ding, Ke; Xu, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1565-1579</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of inhibitors of bromodomain and extra terminal domain (BET) has achieved great progress, and at least seven inhibitors have progressed into clin. trials for the treatment of cancer or inflammatory diseases.  Here, the authors describe the identification, optimization, and evaluation of benzo[cd]indol-2(1H)-one contg. compds. as a new class of BET bromodomain inhibitors, starting from structure-based virtual screening (SBVS).  Through structure-based optimization, potent compds. were obtained with significantly improved activity.  The two most potent compds. bind to the BRD4 bromodomain, with Kd values of 124 and 137 nM.  Selected compds. exhibited high selectivity over other non-BET subfamily members.  Notably, compd. I demonstrated a reasonable antiproliferation effect on MV4;11 leukemia cells and exhibited a good pharmacokinetic profile with high oral bioavailability (75.8%) and moderate half-life (T1/2 = 3.95 h).  The resulting lead mol. I represents a new, potent, and selective class of BET bromodomain inhibitors for the development of therapeutics to treat cancer and inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl7CO2ESpHObVg90H21EOLACvtfcHk0lgIEsqoSpJb4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs1yqsA%253D%253D&md5=d1cd7b961fd4c417c886a1c2274cdacb</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01511%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DXing%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DTu%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520benzo%255Bcd%255Dindol-2%25281H%2529-ones%2520as%2520potent%2520and%2520specific%2520BET%2520bromodomain%2520inhibitors%253A%2520structure-based%2520virtual%2520screening%252C%2520optimization%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1565%26epage%3D1579%26doi%3D10.1021%2Facs.jmedchem.5b01511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of benzo[<i>cd</i>]indol-2(1<i>H</i>)-ones derivatives as BRD4 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.04.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.ejmech.2018.04.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=29730189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptFSjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2018&pages=264-273&author=Y.+Fengauthor=S.+Xiaoauthor=Y.+Chenauthor=H.+Jiangauthor=N.+Liuauthor=C.+Luoauthor=S.+Chenauthor=H.+Chen&title=Design%2C+synthesis+and+biological+evaluation+of+benzo%5Bcd%5Dindol-2%281H%29-ones+derivatives+as+BRD4+inhibitors&doi=10.1016%2Fj.ejmech.2018.04.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of benzo[cd]indol-2(1H)-one derivatives as BRD4 inhibitors</span></div><div class="casAuthors">Feng, Yuxin; Xiao, Senhao; Chen, Yantao; Jiang, Hao; Liu, Na; Luo, Cheng; Chen, Shijie; Chen, Hua</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">264-273</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of benzo[cd]indol-2(1H)-one derivs. were designed by structural optimization based on compd. I and evaluated for their BRD4 (bromodomain-contg. protein 4) inhibitory activity.  The results showed that four compds. are the most potential ones with the IC50 values of 1.02 μM, 1.43 μM, 1.55 μM and 3.02 μM.  According to their co-crystal structures in complex with BRD4_BD1 and the protein thermal shift assays, the binding modes were revealed that the addnl. indirect hydrogen bonds and hydrophobic interactions make such four compds. more active than I against BRD4.  Furthermore, compds. I, 1-((1-ethyl-2-oxo-1,2-dihydrobenzo[cd]indol-6-yl)sulfonyl)pyrrolidine-2-carbohydrazide and 1-((1-ethyl-2-oxo-1,2-dihydrobenzo[cd]indol-6-yl)sulfonyl)pyrrolidine-2-carboxylic acid were chosen to evaluate for their antiproliferative activities on the MLL-AF4-expression acute leukemia cell line (MV4-11), other cancer cell lines and the non-cancer cell lines.  The results showed that these compds. exhibited good and selective inhibitory activities against MV4-11 cells with the IC50 values of 11.67 μM, 5.55 μM, and 11.54 μM, resp., and could act on the cell proliferation by blocking cell cycle at G1 phase.  They could markedly down-regulate the expressions of the c-Myc, Bcl-2 and CDK6 oncogenes in MV4-11 in the qRT-PCR and western blot studies, which further demonstrated that compd. I and its derivs. could serve as a promising therapeutic strategy for MLL leukemia by targeting BRD4_BD1 protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqumq3_lulaVLVg90H21EOLACvtfcHk0lgIEsqoSpJb4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptFSjtrw%253D&md5=0652f895d702325d904e64136c44505c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.04.048%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520benzo%255Bcd%255Dindol-2%25281H%2529-ones%2520derivatives%2520as%2520BRD4%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D152%26spage%3D264%26epage%3D273%26doi%3D10.1016%2Fj.ejmech.2018.04.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, K. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L. P.</span></span> <span> </span><span class="NLM_article-title">An efficient approach to the total synthesis of Ammosamide B</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">1050</span>– <span class="NLM_lpage">1053</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201501560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1002%2Fejoc.201501560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1050-1053&author=S.+W.+Yangauthor=C.+M.+Wangauthor=K.+X.+Tangauthor=J.+X.+Wangauthor=L.+P.+Sun&title=An+efficient+approach+to+the+total+synthesis+of+Ammosamide+B&doi=10.1002%2Fejoc.201501560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">An Efficient Approach to the Total Synthesis of Ammosamide B</span></div><div class="casAuthors">Yang, Sheng-Wei; Wang, Chun-Meng; Tang, Kai-Xiang; Wang, Jin-Xin; Sun, Li-Ping</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2016</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1050-1053</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A new approach for the total synthesis of ammosamide B (I) was realized.  The strategy is based on an intermol. Pd-catalyzed N-arylation of 4-chloro-1-methylindoline-2,3-dione (2, II), which enables construction of the key pyrroloquinoline skeleton precursor 6 (VI) within three steps in an overall yield of 80 %.  The ammosamide B total synthesis is achieved in 12 simple transformations by starting from a com. available starting material and proceeds with an overall yield of 23 %.  Addnl., key precursor 6 represents an important intermediate in the synthesis of other pyrroloquinoline-contg. natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJE0yskdRlu7Vg90H21EOLACvtfcHk0lgIEsqoSpJb4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVeitb4%253D&md5=21c20b338972b193d9f4255e1befa19e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201501560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201501560%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%2BW.%26aulast%3DWang%26aufirst%3DC.%2BM.%26aulast%3DTang%26aufirst%3DK.%2BX.%26aulast%3DWang%26aufirst%3DJ.%2BX.%26aulast%3DSun%26aufirst%3DL.%2BP.%26atitle%3DAn%2520efficient%2520approach%2520to%2520the%2520total%2520synthesis%2520of%2520Ammosamide%2520B%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2016%26spage%3D1050%26epage%3D1053%26doi%3D10.1002%2Fejoc.201501560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cinelli, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miley, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. B.</span></span> <span> </span><span class="NLM_article-title">Inducible nitric oxide synthase: regulation, structure, and inhibition</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">593</span>, <span class="refDoi"> DOI: 10.1002/med.21599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1002%2Fmed.21599" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=593&author=M.+A.+Cinelliauthor=H.+T.+Doauthor=G.+P.+Mileyauthor=R.+B.+Silverman&title=Inducible+nitric+oxide+synthase%3A+regulation%2C+structure%2C+and+inhibition&doi=10.1002%2Fmed.21599"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fmed.21599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21599%26sid%3Dliteratum%253Aachs%26aulast%3DCinelli%26aufirst%3DM.%2BA.%26aulast%3DDo%26aufirst%3DH.%2BT.%26aulast%3DMiley%26aufirst%3DG.%2BP.%26aulast%3DSilverman%26aufirst%3DR.%2BB.%26atitle%3DInducible%2520nitric%2520oxide%2520synthase%253A%2520regulation%252C%2520structure%252C%2520and%2520inhibition%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2019%26spage%3D593%26doi%3D10.1002%2Fmed.21599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span>; <span class="NLM_string-name">Huq, S.</span>; <span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Targeting cyclooxygenase and nitric oxide pathway cross-talk: a new signal transduction pathway for developing more effective anti-inflammatory drugs</span>. <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume">4</span>,  <span class="NLM_fpage">66</span>  <span class="NLM_lpage">75</span>. <span class="refDoi"> DOI: 10.2174/157436209787048702</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.2174%2F157436209787048702" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=66-75&author=L.+Liu&author=S.+Huq&author=W.+Xu&title=Targeting+cyclooxygenase+and+nitric+oxide+pathway+cross-talk%3A+a+new+signal+transduction+pathway+for+developing+more+effective+anti-inflammatory+drugs&doi=10.2174%2F157436209787048702"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.2174%2F157436209787048702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157436209787048702%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26atitle%3DTargeting%2520cyclooxygenase%2520and%2520nitric%2520oxide%2520pathway%2520cross-talk%253A%2520a%2520new%2520signal%2520transduction%2520pathway%2520for%2520developing%2520more%2520effective%2520anti-inflammatory%2520drugs%26date%3D2009%26volume%3D4%26spage%3D66%26epage%3D75%26doi%3D10.2174%2F157436209787048702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leal, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royce, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pioli, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sporn, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liby, K. T.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitors, JQ1 and I-BET 762, as potential therapies for pancreatic cancer</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>394</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2017.02.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.canlet.2017.02.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=28254412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVWrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2017&pages=76-87&author=A.+S.+Lealauthor=C.+R.+Williamsauthor=D.+B.+Royceauthor=P.+A.+Pioliauthor=M.+B.+Spornauthor=K.+T.+Liby&title=Bromodomain+inhibitors%2C+JQ1+and+I-BET+762%2C+as+potential+therapies+for+pancreatic+cancer&doi=10.1016%2Fj.canlet.2017.02.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibitors, JQ1 and I-BET 762, as potential therapies for pancreatic cancer</span></div><div class="casAuthors">Leal, Ana S.; Williams, Charlotte R.; Royce, Darlene B.; Pioli, Patricia A.; Sporn, Michael B.; Liby, Karen T.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76-87</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Bromodomain inhibitors (JQ1 and I-BET 762) are a new generation of selective, small mol. inhibitors that target BET (bromodomain and extra terminal) proteins.  By impairing their ability to bind to acetylated lysines on histones, bromodomain inhibitors interfere with transcriptional initiation and elongation.  BET proteins regulate several genes responsible for cell cycle, apoptosis and inflammation.  In this study, JQ1 and I-BET 762 decreased c-Myc and p-Erk 1/2 protein levels and inhibited proliferation in pancreatic cancer cells.  The tumor microenvironment is known to play an important role in pancreatic cancer, and these drugs suppressed the prodn. of nitric oxide and a variety of inflammatory cytokines, including IL-6, CCL2, and GM-CSF, in both immune and pancreatic cancer cells in vitro.  Notably, the bromodomain inhibitors also reduced protein levels of p-Erk 1/2 and p-STAT3 in mouse models of pancreatic cancer.  All of these proteins are essential for tumor promotion, progression and metastasis.  In conclusion, the bromodomain inhibitors JQ1 and I-BET 762 targeted and suppressed multiple pathways in pancreatic cancer.  I-BET 762 and a no. of other bromodomain inhibitors are currently being tested in several clin. trials, making them potentially promising drugs for the treatment of pancreatic cancer, an often-fatal disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9YkBNLO0LkrVg90H21EOLACvtfcHk0lgnFvBdRMQdTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVWrurk%253D&md5=818a84ed01487793f657a74ec5c972ef</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.02.021%26sid%3Dliteratum%253Aachs%26aulast%3DLeal%26aufirst%3DA.%2BS.%26aulast%3DWilliams%26aufirst%3DC.%2BR.%26aulast%3DRoyce%26aufirst%3DD.%2BB.%26aulast%3DPioli%26aufirst%3DP.%2BA.%26aulast%3DSporn%26aufirst%3DM.%2BB.%26aulast%3DLiby%26aufirst%3DK.%2BT.%26atitle%3DBromodomain%2520inhibitors%252C%2520JQ1%2520and%2520I-BET%2520762%252C%2520as%2520potential%2520therapies%2520for%2520pancreatic%2520cancer%26jtitle%3DCancer%2520Lett.%26date%3D2017%26volume%3D394%26spage%3D76%26epage%3D87%26doi%3D10.1016%2Fj.canlet.2017.02.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujikawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e83190</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0083190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1371%2Fjournal.pone.0083190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=24391744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=K.+G.+McLureauthor=E.+M.+Gesnerauthor=L.+Tsujikawaauthor=O.+A.+Kharenkoauthor=S.+Attwellauthor=E.+Campeauauthor=S.+Wasiakauthor=A.+Steinauthor=A.+Whiteauthor=E.+Fontano&title=RVX-208%2C+an+inducer+of+ApoA-I+in+humans%2C+is+a+BET+bromodomain+antagonist&doi=10.1371%2Fjournal.pone.0083190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span></div><div class="casAuthors">McLure, Kevin G.; Gesner, Emily M.; Tsujikawa, Laura; Kharenko, Olesya A.; Attwell, Sarah; Campeau, Eric; Wasiak, Sylwia; Stein, Adam; White, Andre; Fontano, Eric; Suto, Robert K.; Wong, Norman C. W.; Wagner, Gregory S.; Hansen, Henrik C.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e83190/1-e83190/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increased synthesis of Apolipoprotein A-I (ApoA-I) and HDL is believed to provide a new approach to treating atherosclerosis through the stimulation of reverse cholesterol transport.  RVX-208 increases the prodn. of ApoA-I in hepatocytes in vitro and in vivo in monkeys and humans, which results in increased HDL-C, but the mol. target was not previously reported.  Using binding assays and X-ray crystallog., we now show that RVX-208 selectively binds to bromodomains of the BET (Bromodomain and Extra Terminal) family, competing for a site bound by the endogenous ligand, acetylated lysine and that this accounts for its pharmacol. activity. siRNA expts. further suggest that induction of ApoA-I mRNA is mediated by BET family member BRD4.  These data indicate that RVX-208 increases ApoA-I prodn. through an epigenetic mechanism and suggests that BET inhibition may be a promising new approach to the treatment of atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHPlAY0x6NLVg90H21EOLACvtfcHk0lgnFvBdRMQdTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D&md5=519ef7ed6d06f97ef8a1359832c11d92</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083190%26sid%3Dliteratum%253Aachs%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DWasiak%26aufirst%3DS.%26aulast%3DStein%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26atitle%3DRVX-208%252C%2520an%2520inducer%2520of%2520ApoA-I%2520in%2520humans%252C%2520is%2520a%2520BET%2520bromodomain%2520antagonist%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0083190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alqahtani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choucair, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashraf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammouda, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alloghbi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senzer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemunaitis, J.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and extra-terminal motif inhibitors: a review of preclinical and clinical advances in cancer therapy</span>. <i>Future Sci. OA</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">FSO372</span>, <span class="refDoi"> DOI: 10.4155/fsoa-2018-0115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.4155%2Ffsoa-2018-0115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30906568" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=FSO372&author=A.+Alqahtaniauthor=K.+Choucairauthor=M.+Ashrafauthor=D.+M.+Hammoudaauthor=A.+Alloghbiauthor=T.+Khanauthor=N.+Senzerauthor=J.+Nemunaitis&title=Bromodomain+and+extra-terminal+motif+inhibitors%3A+a+review+of+preclinical+and+clinical+advances+in+cancer+therapy&doi=10.4155%2Ffsoa-2018-0115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.4155%2Ffsoa-2018-0115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffsoa-2018-0115%26sid%3Dliteratum%253Aachs%26aulast%3DAlqahtani%26aufirst%3DA.%26aulast%3DChoucair%26aufirst%3DK.%26aulast%3DAshraf%26aufirst%3DM.%26aulast%3DHammouda%26aufirst%3DD.%2BM.%26aulast%3DAlloghbi%26aufirst%3DA.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DSenzer%26aufirst%3DN.%26aulast%3DNemunaitis%26aufirst%3DJ.%26atitle%3DBromodomain%2520and%2520extra-terminal%2520motif%2520inhibitors%253A%2520a%2520review%2520of%2520preclinical%2520and%2520clinical%2520advances%2520in%2520cancer%2520therapy%26jtitle%3DFuture%2520Sci.%2520OA%26date%3D2019%26volume%3D5%26spage%3DFSO372%26doi%3D10.4155%2Ffsoa-2018-0115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: a novel epigenetic approach</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1776</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&author=D.+Doroshowauthor=J.+Ederauthor=P.+LoRusso&title=BET+inhibitors%3A+a+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2eb8yqEYdlCfsrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DLoRusso%26aufirst%3DP.%26atitle%3DBET%2520inhibitors%253A%2520a%2520novel%2520epigenetic%2520approach%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span> <span> </span><span class="NLM_article-title">Binding pocket-based design, synthesis and biological evaluation of novel selective BRD4-BD1 inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1871</span>– <span class="NLM_lpage">1881</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.03.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.bmc.2019.03.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30926312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=1871-1881&author=J.+Maauthor=H.+Chenauthor=J.+Yangauthor=Z.+Yuauthor=P.+Huangauthor=H.+Yangauthor=B.+Zhengauthor=R.+Liuauthor=Q.+Liauthor=G.+Huauthor=Z.+Chen&title=Binding+pocket-based+design%2C+synthesis+and+biological+evaluation+of+novel+selective+BRD4-BD1+inhibitors&doi=10.1016%2Fj.bmc.2019.03.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Binding pocket-based design, synthesis and biological evaluation of novel selective BRD4-BD1 inhibitors</span></div><div class="casAuthors">Ma, Junlong; Chen, Heng; Yang, Jie; Yu, Zutao; Huang, Pan; Yang, Haofeng; Zheng, Bifeng; Liu, Rangru; Li, Qianbin; Hu, Gaoyun; Chen, Zhuo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1871-1881</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4), consisting of two tandem bromodomains (BD1 and BD2), is key epigenetic regulator in fibrosis and cancer, which has been reported that BD1 and BD2 have distinct roles in post-translational modification.  But there are few selective inhibitors toward those two domains.  Herein, this study designed and synthesized a series of novel selective BRD4-BD1 inhibitors, using computer-aided drug design (CADD) approach focused on exploring the difference of the binding pockets of BD1 and BD2, and finding the His437 a crucial way to achieve BRD4-BD1 selectivity.  Our results revealed that the compd. 3u(I) is a potent selective BRD4-BD1 inhibitor with IC50 values of 0.56 μM for BD1 but >100 μM for BD2.  I exhibited a broad spectrum of anti-proliferative activity against several human cancer and fibroblastic cell lines, which might be related to its capability of reducing the expression of c-Myc and collagen I.  Furthermore, I could induce apoptosis in A375 cells.  To the contrary, the selective BD2 inhibitor, RVX-208, did not indicate any of these activities.  Our findings highlight that the function of BRD4-BD1 might be predominant in fibrosis and cancer.  And it is rational to further develop novel selective BRD4-BD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp09D43P7pU8LVg90H21EOLACvtfcHk0lgEYX4df-mzTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCit74%253D&md5=8e0166a4372a3ea0919762c2a770aa7a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.03.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.03.037%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DHu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26atitle%3DBinding%2520pocket-based%2520design%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520selective%2520BRD4-BD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D1871%26epage%3D1881%26doi%3D10.1016%2Fj.bmc.2019.03.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seitzberg, J. G.</span>; <span class="NLM_string-name">Kronborg, T. T. A.</span>; <span class="NLM_string-name">Poljak, V.</span>; <span class="NLM_string-name">Friberg, G.</span>; <span class="NLM_string-name">Teuber, L.</span></span> <span> </span><span class="NLM_article-title">Compounds Active towards Bromodomains</span>. U.S. Patent Application <span class="NLM_patent">US20170349607A1</span><span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+G.+Seitzberg&author=T.+T.+A.+Kronborg&author=V.+Poljak&author=G.+Friberg&author=L.+Teuber&title=Compounds+Active+towards+Bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSeitzberg%26aufirst%3DJ.%2BG.%26atitle%3DCompounds%2520Active%2520towards%2520Bromodomains%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W. S.</span></span> <span> </span><span class="NLM_article-title">Design, characterization and comparison of transdermal delivery of colchicine via borneol-chemically-modified and borneol-physically-modified ethosome</span>. <i>Drug Delivery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1080/10717544.2018.1559258</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1080%2F10717544.2018.1559258" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30744424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVSisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=70-77&author=Y.+J.+Zhangauthor=N.+Zhangauthor=H.+Songauthor=H.+Liauthor=J.+Wenauthor=X.+C.+Tanauthor=W.+S.+Zheng&title=Design%2C+characterization+and+comparison+of+transdermal+delivery+of+colchicine+via+borneol-chemically-modified+and+borneol-physically-modified+ethosome&doi=10.1080%2F10717544.2018.1559258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Design, characterization and comparison of transdermal delivery of colchicine via borneol-chemically-modified and borneol-physically-modified ethosome</span></div><div class="casAuthors">Zhang, Yujia; Zhang, Nan; Song, Hui; Li, He; Wen, Jin; Tan, Xiaochuan; Zheng, Wensheng</div><div class="citationInfo"><span class="NLM_cas:title">Drug Delivery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-77</span>CODEN:
                <span class="NLM_cas:coden">DDELEB</span>;
        ISSN:<span class="NLM_cas:issn">1071-7544</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Gout is a kind of joint disease characterized by the accumulation of monosodium urate (MSU) crystals in the joint and its surrounding tissue, causing persistent hyperuricemia.  Colchicine is the first choice of treatment for acute gout attacks.  Due to strong toxicity of colchicines oral tablets, there are high fluctuations of blood drug concn. and serious irritation of gastrointestinal tract, and hence a transdermal prepn. can help to slow down the blood drug concn., reduce the frequency of drug taking, and improve the patients' compliance of the drug.  The ethosome is a lipid carrier with high concn. of ethanol and has been proved to promote the penetration of drugs into the skin.  Borneol (BO) is an excellent penetration enhancer in Chinese medicine, which can promote the entry of drugs into the skin.  This paper prepd. the borneol-phys.-modified colchicine ethosome (COL-bpES) and used the prepd. borneol-dioleoyl phosphoethanloamine (BO-DOPE) to prep. borneol-chem.-modified colchicine ethosome (COL-bcES).  Compared to the free colchicine aq. soln. (free COL) and normal colchicine ethosome (COL-ES), the borneol-modified colchicine ethosome (COL-bES) demonstrated better drug penetration effect, while the particle size of the COL-bcES was lower than that of the COL-bpES.  Toxicity, in vitro diffusion, pharmacokinetics and pharmacodynamics are superior to those of COL-bpES, providing a better delivery system for the treatment of small mol. inflammatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-I-ofY_fSJrVg90H21EOLACvtfcHk0lgEYX4df-mzTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVSisrg%253D&md5=0c494e7f594b8a6f07f97780acacdada</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1080%2F10717544.2018.1559258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10717544.2018.1559258%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%2BJ.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DX.%2BC.%26aulast%3DZheng%26aufirst%3DW.%2BS.%26atitle%3DDesign%252C%2520characterization%2520and%2520comparison%2520of%2520transdermal%2520delivery%2520of%2520colchicine%2520via%2520borneol-chemically-modified%2520and%2520borneol-physically-modified%2520ethosome%26jtitle%3DDrug%2520Delivery%26date%3D2019%26volume%3D26%26spage%3D70%26epage%3D77%26doi%3D10.1080%2F10717544.2018.1559258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saseen, J. J.</span></span> <span> </span><span class="NLM_article-title">Gouty arthritis: a review of acute management and prevention</span>. <i>Pharmacotherapy</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">906</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1002/phar.1788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1002%2Fphar.1788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27318031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKrtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=906-922&author=L.+Wilsonauthor=J.+J.+Saseen&title=Gouty+arthritis%3A+a+review+of+acute+management+and+prevention&doi=10.1002%2Fphar.1788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Gouty Arthritis: A Review of Acute Management and Prevention</span></div><div class="casAuthors">Wilson, Liza; Saseen, Joseph J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">906-922</span>CODEN:
                <span class="NLM_cas:coden">PHPYDQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-9114</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Gouty arthritis is one of the most common rheumatic diseases.  The clin. burden of gouty arthritis has historically been well recognized; however, gout is often misdiagnosed and mismanaged.  The prevalence of gout is rising and is likely attributed to several factors including increased incidence of comorbidities, lifestyle factors, and increased use of causative medications.  With the increasing prevalence, there have been several innovations and evidence-based updates related to the diagnosis and management of gout.  Acute gouty arthritis should be treated with nonsteroidal antiinflammatory drugs (NSAIDs), colchicine, or corticosteroids, or a combination of two agents.  Xanthine oxidase inhibitor therapy remains the consensus first-line treatment option for the prevention of recurrent gout.  Add-on therapies that reduce serum urate concn. include traditional uricosuric agents and a novel uric acid reabsorption inhibitor.  Prophylaxis of acute gout with NSAIDs, colchicine, or corticosteroids is universally recommended when initiating any urate-lowering therapy in order to prevent acute gouty arthritis for a period of at least 6 mo.  In this review, we discuss the epidemiol. and risk factors for gouty arthritis and evaluate diagnostic strategies and therapeutic regimens for the management of gout, including a new drug approval.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreo-TcsyQUP7Vg90H21EOLACvtfcHk0liZmNmJalx-Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKrtrvJ&md5=6e67005f24e6eba7afcf4faec5cfb92f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fphar.1788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fphar.1788%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DL.%26aulast%3DSaseen%26aufirst%3DJ.%2BJ.%26atitle%3DGouty%2520arthritis%253A%2520a%2520review%2520of%2520acute%2520management%2520and%2520prevention%26jtitle%3DPharmacotherapy%26date%3D2016%26volume%3D36%26spage%3D906%26epage%3D922%26doi%3D10.1002%2Fphar.1788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eglen, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reisine, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roby, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouleau, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Illy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossé, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielefeld, M.</span></span> <span> </span><span class="NLM_article-title">The use of AlphaScreen technology in HTS: current status</span>. <i>Curr. Chem. Genomics</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">2</span>, <span class="refDoi"> DOI: 10.2174/1875397300801010002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.2174%2F1875397300801010002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=20161822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjvVWrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2008&pages=2&author=R.+M.+Eglenauthor=T.+Reisineauthor=P.+Robyauthor=N.+Rouleauauthor=C.+Illyauthor=R.+Boss%C3%A9author=M.+Bielefeld&title=The+use+of+AlphaScreen+technology+in+HTS%3A+current+status&doi=10.2174%2F1875397300801010002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The use of AlphaScreen technology in HTS: Current status</span></div><div class="casAuthors">Eglen, Richard M.; Reisine, Terry; Roby, Philippe; Rouleau, Nathalie; Illy, Chantal; Bosse, Roger; Bielefeld, Martina</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2-10</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  AlphaScreen (Amplified Luminescent Proximity Homogeneous Assay Screen) is versatile assay technol. developed to measuring analytes using a homogeneous protocol.  This technol. is an example of a bead-based proximity assay and was developed from a diagnostic assay technol. known as LOCI (Luminescent Oxygen Channeling Assay).  Here, singlet oxygen mols., generated by high energy irradn. of Donor beads, travel over a constrained distance (approx. 200 nm) to Acceptor beads.  This results in excitation of a cascading series of chem. reactions, ultimately causing generation of a chemiluminescent signal.  In the past decade, a wide variety of applications has been reported, ranging from detection of analytes involved in cell signaling, including protein:protein, protein:peptide, protein:small mol. or peptide:peptide interactions.  Numerous homogeneous HTS-optimized assays have been reported using the approach, including generation of second messengers (such as accumulation of cAMP, cyclic GMP, inositol [1, 4, 5] trisphosphate or phosphorylated ERK) from liganded GPCRs or tyrosine kinase receptors, post-translational modification of proteins (such as proteolytic cleavage, phosphorylation, ubiquitination and sumoylation) as well as protein-protein and protein-nucleic acid interactions.  Recently, the basic AlphaScreen technol. was extended in that the chem. of the Acceptor bead was modified such that emitted light is more intense and spectrally defined, thereby markedly reducing interference from biol. fluid matrixes (such as trace hemolysis in serum and plasma).  In this format, referred to as AlphaLISA, it provides an alternative technol. to classical ELISA assays and is suitable for high throughput automated fluid dispensing and detection systems.  Collectively, AlphaScreen and AlphaLISA technologies provide a facile assay platform with which one can quantitate complex cellular processes using simple no-wash microtiter plate based assays.  They provide the means by which large compd. libraries can be screened in a high throughput fashion at a diverse range of therapeutically important targets, often not readily undertaken using other homogeneous assay technologies.  This review assesses the current status of the technol. in drug discovery, in general, and high throughput screening (HTS), in particular.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4coo45wvUbbVg90H21EOLACvtfcHk0liZmNmJalx-Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjvVWrtro%253D&md5=4d35f83091184d6eb4d1b76444754707</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.2174%2F1875397300801010002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397300801010002%26sid%3Dliteratum%253Aachs%26aulast%3DEglen%26aufirst%3DR.%2BM.%26aulast%3DReisine%26aufirst%3DT.%26aulast%3DRoby%26aufirst%3DP.%26aulast%3DRouleau%26aufirst%3DN.%26aulast%3DIlly%26aufirst%3DC.%26aulast%3DBoss%25C3%25A9%26aufirst%3DR.%26aulast%3DBielefeld%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520AlphaScreen%2520technology%2520in%2520HTS%253A%2520current%2520status%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2008%26volume%3D1%26spage%3D2%26doi%3D10.2174%2F1875397300801010002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, E.</span>; <span class="NLM_string-name">Wodicka, L.</span>; <span class="NLM_string-name">Ciceri, P.</span>; <span class="NLM_string-name">Pallares, G.</span>; <span class="NLM_string-name">Pickle, E.</span>; <span class="NLM_string-name">Torrey, A.</span>; <span class="NLM_string-name">Floyd, M.</span>; <span class="NLM_string-name">Hunt, J.</span>; <span class="NLM_string-name">Treiber, D.</span></span> <i>BROMOscan a High Throughput, Quantitative Ligand Binding Platform Identifies Best-in-Class Bromodomain Inhibitors from a Screen of Mature Compounds Targeting Other Protein Classes</i>; <span class="NLM_publisher-name">AACR</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=E.+Quinn&author=L.+Wodicka&author=P.+Ciceri&author=G.+Pallares&author=E.+Pickle&author=A.+Torrey&author=M.+Floyd&author=J.+Hunt&author=D.+Treiber&title=BROMOscan+a+High+Throughput%2C+Quantitative+Ligand+Binding+Platform+Identifies+Best-in-Class+Bromodomain+Inhibitors+from+a+Screen+of+Mature+Compounds+Targeting+Other+Protein+Classes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DE.%26btitle%3DBROMOscan%2520a%2520High%2520Throughput%252C%2520Quantitative%2520Ligand%2520Binding%2520Platform%2520Identifies%2520Best-in-Class%2520Bromodomain%2520Inhibitors%2520from%2520a%2520Screen%2520of%2520Mature%2520Compounds%2520Targeting%2520Other%2520Protein%2520Classes%26pub%3DAACR%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holms, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClellan, W. J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Fragment-based, structure-enabled discovery of novel pyridones and pyridone macrocycles as potent bromodomain and extra-terminal domain (BET) family bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3828</span>– <span class="NLM_lpage">3850</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00017</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00017" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsVygu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3828-3850&author=L.+Wangauthor=J.+K.+Prattauthor=T.+Soltwedelauthor=G.+S.+Sheppardauthor=S.+D.+Fidanzeauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=R.+A.+Manteiauthor=J.+H.+Holmsauthor=W.+J.+McClellan&title=Fragment-based%2C+structure-enabled+discovery+of+novel+pyridones+and+pyridone+macrocycles+as+potent+bromodomain+and+extra-terminal+domain+%28BET%29+family+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.7b00017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based, Structure-Enabled Discovery of Novel Pyridones and Pyridone Macrocycles as Potent Bromodomain and Extra-Terminal Domain (BET) Family Bromodomain Inhibitors</span></div><div class="casAuthors">Wang, Le; Pratt, John K.; Soltwedel, Todd; Sheppard, George S.; Fidanze, Steven D.; Liu, Dachun; Hasvold, Lisa A.; Mantei, Robert A.; Holms, James H.; McClellan, William J.; Wendt, Michael D.; Wada, Carol; Frey, Robin; Hansen, T. Matthew; Hubbard, Robert; Park, Chang H.; Li, Leiming; Magoc, Terrance J.; Albert, Daniel H.; Lin, Xiaoyu; Warder, Scott E.; Kovar, Peter; Huang, Xiaoli; Wilcox, Denise; Wang, Rongqi; Rajaraman, Ganesh; Petros, Andrew M.; Hutchins, Charles W.; Panchal, Sanjay C.; Sun, Chaohong; Elmore, Steven W.; Shen, Yu; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3828-3850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the BET family of bromodomain contg. proteins have been identified as potential targets for blocking proliferation in a variety of cancer cell lines.  A 2-dimensional NMR fragment screen for binders to the bromodomains of BRD4 identified a Ph pyridazinone fragment with a weak binding affinity (I, Ki = 160 μM).  SAR investigation of fragment I, aided by X-ray structure-based design, enabled the synthesis of potent pyridone and macrocyclic pyridone inhibitors exhibiting single digit nanomolar potency in both biochem. and cell based assays, e.g. II.  Advanced analogs in these series exhibited high oral exposures in rodent PK studies and demonstrated significant tumor growth inhibition efficacy in mouse flank xenograft models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDaiYdyL4LtrVg90H21EOLACvtfcHk0liZmNmJalx-Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsVygu7k%253D&md5=91750e660bafec08b8bcbe6bcd5e818f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00017%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DHolms%26aufirst%3DJ.%2BH.%26aulast%3DMcClellan%26aufirst%3DW.%2BJ.%26atitle%3DFragment-based%252C%2520structure-enabled%2520discovery%2520of%2520novel%2520pyridones%2520and%2520pyridone%2520macrocycles%2520as%2520potent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3828%26epage%3D3850%26doi%3D10.1021%2Facs.jmedchem.7b00017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G. G.</span>; <span class="NLM_string-name">Kontogiannis, L.</span>; <span class="NLM_string-name">Johnson, O.</span>; <span class="NLM_string-name">Powell, H. R.</span>; <span class="NLM_string-name">Leslie, A. G.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>. <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume">67</span>,  <span class="NLM_fpage">271</span>  <span class="NLM_lpage">281</span>. <span class="refDoi"> DOI: 10.1107/S0907444910048675</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battye&author=L.+Kontogiannis&author=O.+Johnson&author=H.+R.+Powell&author=A.+G.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgM6qQUTJS1tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span>; <span class="NLM_string-name">Lohkamp, B.</span>; <span class="NLM_string-name">Scott, W. G.</span>; <span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume">66</span>,  <span class="NLM_fpage">486</span>  <span class="NLM_lpage">501</span>. <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsley&author=B.+Lohkamp&author=W.+G.+Scott&author=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span>; <span class="NLM_string-name">Afonine, P. V.</span>; <span class="NLM_string-name">Bunkóczi, G.</span>; <span class="NLM_string-name">Chen, V. B.</span>; <span class="NLM_string-name">Davis, I. W.</span>; <span class="NLM_string-name">Echols, N.</span>; <span class="NLM_string-name">Headd, J. J.</span>; <span class="NLM_string-name">Hung, L.-W.</span>; <span class="NLM_string-name">Kapral, G. J.</span>; <span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span> <span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume">66</span>,  <span class="NLM_fpage">213</span>  <span class="NLM_lpage">221</span>. <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adams&author=P.+V.+Afonine&author=G.+Bunk%C3%B3czi&author=V.+B.+Chen&author=I.+W.+Davis&author=N.+Echols&author=J.+J.+Headd&author=L.-W.+Hung&author=G.+J.+Kapral&author=R.+W.+Grosse-Kunstleve&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 14 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Francesco Rianjongdee, Stephen J. Atkinson, Chun-wa Chung, Paola Grandi, James R. J. Gray, Laura J. Kaushansky, Patricia Medeiros, Cassie Messenger, Alex Phillipou, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Alexander L. Satz, Simon Taylor, Ian D. Wall, Robert J. Watson, Gang Yao, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00412%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BHighly%252BSelective%252BBET%252BBD2%252BInhibitor%252Bfrom%252Ba%252BDNA-Encoded%252BLibrary%252BTechnology%252BScreening%252BHit%26aulast%3DRianjongdee%26aufirst%3DFrancesco%26date%3D2021%26date%3D2021%26date%3D08032021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Helen E. Aylott, Stephen J. Atkinson, Paul Bamborough, Anna Bassil, Chun-wa Chung, Laurie Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Thomas G. Hayhow, Cassie Messenger, Darren Mitchell, Alexander Phillipou, Alex Preston, Rab K. Prinjha, Francesco Rianjongdee, Inmaculada Rioja, Jonathan T. Seal, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Template-Hopping Approach Leads to Potent, Selective, and Highly Soluble Bromo and Extraterminal Domain (BET) Second Bromodomain (BD2) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3249-3281. <a href="https://doi.org/10.1021/acs.jmedchem.0c02156" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02156</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02156%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTemplate-Hopping%252BApproach%252BLeads%252Bto%252BPotent%25252C%252BSelective%25252C%252Band%252BHighly%252BSoluble%252BBromo%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BSecond%252BBromodomain%252B%252528BD2%252529%252BInhibitors%26aulast%3DAylott%26aufirst%3DHelen%2BE.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D04032021%26volume%3D64%26issue%3D6%26spage%3D3249%26epage%3D3281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pan Tang, Jifa Zhang, Jie Liu, Cheng-Ming Chiang, <span class="NLM_string-name hlFld-ContribAuthor">Liang Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2419-2435. <a href="https://doi.org/10.1021/acs.jmedchem.0c01487" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01487%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BBromodomain%252Band%252BExtraterminal%252BProteins%252Bfor%252BDrug%252BDiscovery%25253A%252BFrom%252BCurrent%252BProgress%252Bto%252BTechnological%252BDevelopment%26aulast%3DTang%26aufirst%3DPan%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26082020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2419%26epage%3D2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert J. Watson, Paul Bamborough, Heather Barnett, Chun-wa Chung, Rob Davis, Laurie Gordon, Paola Grandi, Massimo Petretich, Alex Phillipou, Rab K. Prinjha, Inmaculada Rioja, Peter Soden, Thilo Werner, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9045-9069. <a href="https://doi.org/10.1021/acs.jmedchem.0c00614" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGSK789%25253A%252BA%252BSelective%252BInhibitor%252Bof%252Bthe%252BFirst%252BBromodomains%252B%252528BD1%252529%252Bof%252Bthe%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BProteins%26aulast%3DWatson%26aufirst%3DRobert%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14042020%26date%3D04082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9045%26epage%3D9069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yuqin Zhu, Yuxiang Ma, Weidong Zu, Jianing Song, Hua Wang, You Zhong, Hongmei Li, Yanmin Zhang, Qianqian Gao, Bo Kong, Junyu Xu, Fei Jiang, Xinren Wang, Shuwen Li, Chenhe Liu, Haichun Liu, Tao Lu, <span class="NLM_string-name hlFld-ContribAuthor">Yadong Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of N-Phenyl-7H-pyrrolo[2,3-d]pyrimidin-4-amine Derivatives as Novel, Potent, and Selective NF-κB Inducing Kinase (NIK) Inhibitors for the Treatment of Psoriasis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (13)
                                     , 6748-6773. <a href="https://doi.org/10.1021/acs.jmedchem.0c00055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BN-Phenyl-7H-pyrrolo%25255B2%25252C3-d%25255Dpyrimidin-4-amine%252BDerivatives%252Bas%252BNovel%25252C%252BPotent%25252C%252Band%252BSelective%252BNF-%2525CE%2525BAB%252BInducing%252BKinase%252B%252528NIK%252529%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BPsoriasis%26aulast%3DZhu%26aufirst%3DYuqin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12012020%26date%3D16062020%26date%3D01062020%26volume%3D63%26issue%3D13%26spage%3D6748%26epage%3D6773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nian  Wang</span>, <span class="hlFld-ContribAuthor ">Runliu  Wu</span>, <span class="hlFld-ContribAuthor ">Daolin  Tang</span>, <span class="hlFld-ContribAuthor ">Rui  Kang</span>. </span><span class="cited-content_cbyCitation_article-title">The BET family in immunity and disease. </span><span class="cited-content_cbyCitation_journal-name">Signal Transduction and Targeted Therapy</span><span> <strong>2021,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41392-020-00384-4" title="DOI URL">https://doi.org/10.1038/s41392-020-00384-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41392-020-00384-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41392-020-00384-4%26sid%3Dliteratum%253Aachs%26jtitle%3DSignal%2520Transduction%2520and%2520Targeted%2520Therapy%26atitle%3DThe%252BBET%252Bfamily%252Bin%252Bimmunity%252Band%252Bdisease%26aulast%3DWang%26aufirst%3DNian%26date%3D2021%26date%3D2021%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabelle A.  Engelberg</span>, <span class="hlFld-ContribAuthor ">Caroline A.  Foley</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Improved methods for targeting epigenetic reader domains of acetylated and methylated lysine. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>63 </em>, 132-144. <a href="https://doi.org/10.1016/j.cbpa.2021.03.002" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.03.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.03.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DImproved%252Bmethods%252Bfor%252Btargeting%252Bepigenetic%252Breader%252Bdomains%252Bof%252Bacetylated%252Band%252Bmethylated%252Blysine%26aulast%3DEngelberg%26aufirst%3DIsabelle%2BA.%26date%3D2021%26volume%3D63%26spage%3D132%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maofeng  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhuyun  Liu</span>, <span class="hlFld-ContribAuthor ">Lizhong  Wang</span>, <span class="hlFld-ContribAuthor ">Yan  Li</span>, <span class="hlFld-ContribAuthor ">Yonggang  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, evaluation and in silico studies of novel BRD4 bromodomain inhibitors bearing a benzo[d]isoxazol scaffold. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Sciences</span><span> <strong>2021,</strong> <em>133 </em>
                                    (1)
                                     <a href="https://doi.org/10.1007/s12039-020-01874-2" title="DOI URL">https://doi.org/10.1007/s12039-020-01874-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12039-020-01874-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12039-020-01874-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Sciences%26atitle%3DSynthesis%25252C%252Bevaluation%252Band%252Bin%252Bsilico%252Bstudies%252Bof%252Bnovel%252BBRD4%252Bbromodomain%252Binhibitors%252Bbearing%252Ba%252Bbenzo%25255Bd%25255Disoxazol%252Bscaffold%26aulast%3DZhang%26aufirst%3DMaofeng%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ellen E.  Guest</span>, <span class="hlFld-ContribAuthor ">Stephen D.  Pickett</span>, <span class="hlFld-ContribAuthor ">Jonathan D.  Hirst</span>. </span><span class="cited-content_cbyCitation_article-title">Structural variation of protein–ligand complexes of the first bromodomain of BRD4. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>1 </em><a href="https://doi.org/10.1039/D1OB00658D" title="DOI URL">https://doi.org/10.1039/D1OB00658D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1OB00658D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1OB00658D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DStructural%252Bvariation%252Bof%252Bprotein%2525E2%252580%252593ligand%252Bcomplexes%252Bof%252Bthe%252Bfirst%252Bbromodomain%252Bof%252BBRD4%26aulast%3DGuest%26aufirst%3DEllen%2BE.%26date%3D2021%26date%3D2021%26volume%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qifei  Li</span>, <span class="hlFld-ContribAuthor ">Jieming  Li</span>, <span class="hlFld-ContribAuthor ">Yan  Cai</span>, <span class="hlFld-ContribAuthor ">Yuxing  Zou</span>, <span class="hlFld-ContribAuthor ">Bin  Chen</span>, <span class="hlFld-ContribAuthor ">Feng  Zou</span>, <span class="hlFld-ContribAuthor ">Jiaxian  Mo</span>, <span class="hlFld-ContribAuthor ">Ting  Han</span>, <span class="hlFld-ContribAuthor ">Weiwei  Guo</span>, <span class="hlFld-ContribAuthor ">Wenlong  Huang</span>, <span class="hlFld-ContribAuthor ">Qianqian  Qiu</span>, <span class="hlFld-ContribAuthor ">Hai  Qian</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel 6-phenyl-1,3a,4,10b-tetrahydro-2H-benzo[c]thiazolo[4,5-e]azepin-2-one derivatives as potential BRD4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (15)
                                     , 115601. <a href="https://doi.org/10.1016/j.bmc.2020.115601" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115601</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115601%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252B6-phenyl-1%25252C3a%25252C4%25252C10b-tetrahydro-2H-benzo%25255Bc%25255Dthiazolo%25255B4%25252C5-e%25255Dazepin-2-one%252Bderivatives%252Bas%252Bpotential%252BBRD4%252Binhibitors%26aulast%3DLi%26aufirst%3DQifei%26date%3D2020%26volume%3D28%26issue%3D15%26spage%3D115601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Massimo  Petretich</span>, <span class="hlFld-ContribAuthor ">Emmanuel H.  Demont</span>, <span class="hlFld-ContribAuthor ">Paola  Grandi</span>. </span><span class="cited-content_cbyCitation_article-title">Domain-selective targeting of BET proteins in cancer and immunological diseases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 184-193. <a href="https://doi.org/10.1016/j.cbpa.2020.02.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.02.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.02.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDomain-selective%252Btargeting%252Bof%252BBET%252Bproteins%252Bin%252Bcancer%252Band%252Bimmunological%252Bdiseases%26aulast%3DPetretich%26aufirst%3DMassimo%26date%3D2020%26volume%3D57%26spage%3D184%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandra  Cipriano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic reader domains by chemical biology. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 82-94. <a href="https://doi.org/10.1016/j.cbpa.2020.05.006" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.05.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.05.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTargeting%252Bepigenetic%252Breader%252Bdomains%252Bby%252Bchemical%252Bbiology%26aulast%3DCipriano%26aufirst%3DAlessandra%26date%3D2020%26volume%3D57%26spage%3D82%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chia-Chun  Tseng</span>, <span class="hlFld-ContribAuthor ">Man Chun  Wong</span>, <span class="hlFld-ContribAuthor ">Wei-Ting  Liao</span>, <span class="hlFld-ContribAuthor ">Chung-Jen  Chen</span>, <span class="hlFld-ContribAuthor ">Su-Chen  Lee</span>, <span class="hlFld-ContribAuthor ">Jeng-Hsien  Yen</span>, <span class="hlFld-ContribAuthor ">Shun-Jen  Chang</span>. </span><span class="cited-content_cbyCitation_article-title">Systemic Investigation of Promoter-wide Methylome and Genome Variations in Gout. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (13)
                                     , 4702. <a href="https://doi.org/10.3390/ijms21134702" title="DOI URL">https://doi.org/10.3390/ijms21134702</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21134702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21134702%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DSystemic%252BInvestigation%252Bof%252BPromoter-wide%252BMethylome%252Band%252BGenome%252BVariations%252Bin%252BGout%26aulast%3DTseng%26aufirst%3DChia-Chun%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D13%26spage%3D4702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative BET bromodomain family selective inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0012.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Sequence alignment of BET bromodomains with the residues highlighted. (B) Comparison of BRD4 BD1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>, green) and BD2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0">5uf0</a>, gray) with (+)-JQ1 (<b>1</b>). The cartoons and lines show the binding pockets of BD1 and BD2, and the sticks are shown to highlight key differential residues. (C) Comparison of BRD4 BD1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>, green) and BD2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0">5uf0</a>, gray) with (+)-JQ1 (<b>1</b>). The surfaces and lines show the binding pockets of BD1 and BD2, and the color surface is shown to highlight the volume of the space differential residues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes of Compounds <b>9</b> and <b>12</b>–<b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) appropriate iodide, 60% NaH, dimethylformamide (DMF); (b) CISO<sub>3</sub>H, 0 °C; (c) aniline, pyridine, dichloromethane (DCM), rt; (d) 69% HNO<sub>3</sub>, AcOH, rt; (e) H<sub>2</sub>, 10% Pd/C, MeOH, tetrahydrofuran (THF), rt; (f) appropriate bromide or benzoyl chloride, 60% NaH, DMF, rt; (g) appropriate sulfonyl chlorides, pyridine, DCM, rt; (h) CH<sub>3</sub>I, 60% NaH, DMF.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0017.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Routes of Compounds <b>46</b>–<b>70</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CH<sub>3</sub>I, 60% NaH, DMF; (b) BocNH<sub>2</sub>, Pd(AcO)<sub>2</sub>, <i>x</i>-PHOS, K<sub>2</sub>CO<sub>3</sub>, anhydrous 1,4-dioxane, 80 °C, 10 h; (c) <i>tert</i>-butyl 2-(triphenyl-λ<sup>5</sup>-phosphanylidene)acetate, DMF, rt; (d) TFA, 100 °C; (e) POCl<sub>3</sub>, 100 °C; (f) H<sub>2</sub>SO<sub>4</sub>, HNO<sub>3</sub>, −25 to −10 °C; (g) appropriate boronic ester, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane, N<sub>2</sub>, 80 °C; (h) appropriate amine derivatives, anhydrous 1,4-dioxane, 100 °C; (i) K<sub>2</sub>CO<sub>3</sub>, 10% P/C, H<sub>2</sub>, THF; (j) 10% P/C, H<sub>2</sub>, THF, MeOH; (k) appropriate sulfonyl chlorides, pyridine, DCM, rt.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0018.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Routes of Compounds <b>77</b>–<b>90</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Br<sub>2</sub>, H<sub>2</sub>SO<sub>4</sub>, Ag<sub>2</sub>SO<sub>4</sub>, 60 °C; (b) i: NH<sub>2</sub>OH HCl, TsCl, pyridine, 90 °C; ii: NaOH, HCl, EtOH, H<sub>2</sub>O; (c) HNO<sub>3</sub>, AcOH, 60 °C; (d) CH<sub>3</sub>I, 60% NaH, DMF; (e) appropriate boronic este, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane, N<sub>2</sub>, 80 °C; (f) Fe, NH<sub>4</sub>Cl, EtOH, 80 °C; (g) appropriate sulfonyl chlorides, pyridine, DCM, rt.</p></p></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0019.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode analysis of compounds <b>6</b> and <b>15</b> to BRD4 BD1 protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>). (A) Compound <b>6</b> (shown as blue) docked into BRD4 BD1 overlaid <b>JQ1</b> (shown as green) with BRD4(1) protein. (B) Compounds <b>6</b> (shown as blue) and <b>15</b> (shown as purple) docked into BRD4 BD1 overlaid <b>JQ1</b> (shown as green) with BRD4(1) protein (displayed as gray surface).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0020.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Conformation of compound <b>25</b> in the BRD4 BD1 and BRD4 BD2 proteins. (A) X-ray crystal structure of compound <b>25</b> bound to BRD4 BD1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJB">6JJB</a>; the authors will release the atomic coordinates and experimental data upon article publication). The ligands are shown as sticks models, and the protein is shown as surface and a cartoon with line model. (B) X-ray crystal structure of <b>25</b> (surface and line, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJB">6JJB</a>) bound to BRD4 BD1, overlaid with BRD4 BD2 (stick and line, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0">5uf0</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Conformation of compound <b>60</b> in the BRD4 BD1 and BRD4 BD2 proteins. (A) X-ray crystal structure of compound <b>60</b> bound to BRD4 BD1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJ3">6JJ3</a>; the authors will release the atomic coordinates and experimental data upon article publication). The ligands are shown as sticks models, and the protein is shown as surface and a cartoon with line model. (B) X-ray crystal structure of <b>60</b> (stick) bound to BRD4 BD1 (line and cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJ3">6JJ3</a>), overlaid with BRD4 BD2 (line and cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0">5uf0</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0021.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Examination for the NF-κB transcriptional activity in HUVECs undergoing three concentration degrees (0.1, 1, and 10 μM) treated JQ1, RVX-208, and compounds for 24 h, respectively. The models were established by TNF-A 50 ng/mL. The values of fold changes are mean ± SDs from three independent experiments, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0004.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>68</b> inhibits MSU-induced pyroptosis in THP-1 cells. After PMA treatment, THP-1 cells were incubated in the absence or presence of serum-free medium containing <b>68</b> (1 μM) for 1 h, followed by the stimulation with MSU (500 μg/mL) for 12 h. The rate of pyroptotic cell death was examined with active PI and Caspase-1 double staining by flow cytometry (A). The expression of BRD4 and NRLP3 inflammasome activation was measured by western blotting and protein quantitative analysis normalized to β-action (B). Representative confocal microscopy photographs of THP-1 cells with immunofluorescence changes (C). The culture supernatants level of IL-1β was detected using the Enzyme-linked immunosorbent assay (ELISA) kit (D). Data were presented as mean ± SD. Compared to the control group: <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, <sup>###</sup><i>P</i> < 0.001. Compared to the model group: *<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001. Each group (<i>n</i> = 4). Independent experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0005.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Effect of BRD4 inhibitor on MSU-induced acute gouty arthritis (4 mg/kg, intra-articular injection) in vivo. Representative photographs to show the swelling of joints (A). Representative photographs of histopathologic changes in synovium (B). The injected ankle joint circumference of each rat was determined at 0, 2, 4, 8, 12, and 24 h after MSU stimulation (C). The IL-1β level in synovium homogenate was detected by the ELISA kit (D). The data were presented as mean ± SD. Compared to the control group: <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, <sup>###</sup><i>P</i> < 0.001. Compared to the model group: *<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001. Data in panels represent mean ± SD from eight animals. Independent experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/medium/jm9b01010_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0022.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound <b>68</b> regulates the pyroptosis death in synovial tissue. Pyroptosis assay was performed in the macrophages obtained from synovium using active PI and Caspase-1 double staining by flow cytometry (A). Protein expression of BRD4 and NLRP3 inflammasome pathway in synovium after compound <b>68</b> treatment at 1 mg/kg intra-articular injection (ia) by western blotting and protein quantitative analysis normalized to β-action (B). NLPR3 inflammasome activation in synovial tissues evaluated with immunofluorescence staining (C). Data were presented as mean ± SD. Compared to the control group: <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, <sup>###</sup><i>P</i> < 0.001. Compared to the model group: *<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001. Data in panels represent mean ± SD from four animals. Independent experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01010/20191219/images/large/jm9b01010_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01010&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i154">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80004" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80004" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 58 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonifacio, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso-Valdivielso, J. L.</span></span> <span> </span><span class="NLM_article-title">Gout treatment: when a tablet is not enough</span>. <i>Nat. Rev. Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1038/nrrheum.2013.173-c1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fnrrheum.2013.173-c1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=194-195&author=A.+L.+Bonifacioauthor=J.+L.+Alonso-Valdivielso&title=Gout+treatment%3A+when+a+tablet+is+not+enough&doi=10.1038%2Fnrrheum.2013.173-c1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrrheum.2013.173-c1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrrheum.2013.173-c1%26sid%3Dliteratum%253Aachs%26aulast%3DBonifacio%26aufirst%3DA.%2BL.%26aulast%3DAlonso-Valdivielso%26aufirst%3DJ.%2BL.%26atitle%3DGout%2520treatment%253A%2520when%2520a%2520tablet%2520is%2520not%2520enough%26jtitle%3DNat.%2520Rev.%2520Rheumatol.%26date%3D2014%26volume%3D10%26spage%3D194%26epage%3D195%26doi%3D10.1038%2Fnrrheum.2013.173-c1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.2217/epi-2016-0144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.2217%2Fepi-2016-0144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27925476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=573-584&author=D.+F.+Toughauthor=R.+K.+Prinjha&title=Immune+disease-associated+variants+in+gene+enhancers+point+to+BET+epigenetic+mechanisms+for+therapeutic+intervention&doi=10.2217%2Fepi-2016-0144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span></div><div class="casAuthors">Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-584</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Genome-wide assocn. studies have identified thousands of single nucleotide polymorphisms in the human genome that are statistically assocd. with particular disease traits.  In this perspective, we review emerging data suggesting that most single nucleotide polymorphisms assocd. with immune-mediated diseases are found in regulatory regions of the DNA - parts of the genome that control expression of the protein encoding genes - rather than causing mutations in proteins.  We discuss how the emerging understanding of particular gene regulatory regions, gene enhancers and the epigenetic mechanisms by which they are regulated is opening up new opportunities for the treatment of immune-mediated diseases, focusing particularly on the BET family of epigenetic reader proteins as potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFU7Sh3r75ZbVg90H21EOLACvtfcHk0liYc9KQv4d6Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D&md5=8a2d44dd000c465b904d4ef2e6f37b77</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0144%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DImmune%2520disease-associated%2520variants%2520in%2520gene%2520enhancers%2520point%2520to%2520BET%2520epigenetic%2520mechanisms%2520for%2520therapeutic%2520intervention%26jtitle%3DEpigenomics%26date%3D2017%26volume%3D9%26spage%3D573%26epage%3D584%26doi%3D10.2217%2Fepi-2016-0144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jhang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, G. C.</span></span> <span> </span><span class="NLM_article-title">Beneficial properties of phytochemicals on NLRP3 inflammasome-mediated gout and complication</span>. <i>J. Agric. Food Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">765</span>, <span class="refDoi"> DOI: 10.1021/acs.jafc.7b05113</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jafc.7b05113" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVWhug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=765&author=J.+J.+Jhangauthor=J.+H.+Linauthor=G.+C.+Yen&title=Beneficial+properties+of+phytochemicals+on+NLRP3+inflammasome-mediated+gout+and+complication&doi=10.1021%2Facs.jafc.7b05113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Beneficial Properties of Phytochemicals on NLRP3 Inflammasome-Mediated Gout and Complication</span></div><div class="casAuthors">Jhang, Jhih-Jia; Lin, Jia-Hong; Yen, Gow-Chin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Agricultural and Food Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">765-772</span>CODEN:
                <span class="NLM_cas:coden">JAFCAU</span>;
        ISSN:<span class="NLM_cas:issn">0021-8561</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Gouty arthritis is characterized by the pptn. of monosodium urate (MSU) crystals in the joint.  Pro-inflammatory cytokine IL-1β is a crit. manifestation in response to MSU crystals attack.  IL-1β secretion is dependent on the nucleotide-binding oligomerization domain-like receptor pyrin domain contg. 3 (NLRP3) inflammasome.  Abnormal activation of the NLRP inflammasome is related to cellular oxidative stress.  However, recent studies have illustrated that phytochems. with potent antioxidant activity exert inhibitory effects on NLRP3 inflammasome-mediated diseases.  This review focuses on the current findings of studies on the NLRP3 inflammasome and the proposed mechanisms that MSU crystals trigger inflammation via activation of the NLRP3 inflammasome.  The authors also summarized the potential use of phytochems. on NLRP3 inflammasome-mediated diseases, suggesting that phytochems. can further prevent acute gout attack.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprORHvlyxeibVg90H21EOLACvtfcHk0liYc9KQv4d6Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVWhug%253D%253D&md5=f6b3d4ee6df2c0f6ad5135e029a53708</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jafc.7b05113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jafc.7b05113%26sid%3Dliteratum%253Aachs%26aulast%3DJhang%26aufirst%3DJ.%2BJ.%26aulast%3DLin%26aufirst%3DJ.%2BH.%26aulast%3DYen%26aufirst%3DG.%2BC.%26atitle%3DBeneficial%2520properties%2520of%2520phytochemicals%2520on%2520NLRP3%2520inflammasome-mediated%2520gout%2520and%2520complication%26jtitle%3DJ.%2520Agric.%2520Food%2520Chem.%26date%3D2018%26volume%3D66%26spage%3D765%26doi%3D10.1021%2Facs.jafc.7b05113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, S.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.-S.</span></span> <span> </span><span class="NLM_article-title">Poly-gamma-glutamic acid from Bacillus subtilis upregulates pro-inflammatory cytokines while inhibiting NLRP3, NLRC4 and AIM2 inflammasome activation</span>. <i>Cell. Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">111</span>, <span class="refDoi"> DOI: 10.1038/cmi.2016.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fcmi.2016.13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27133472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=111&author=H.+Ahnauthor=S.+G.+Kangauthor=S.-i.+Yoonauthor=P.-H.+Kimauthor=D.+Kimauthor=G.-S.+Lee&title=Poly-gamma-glutamic+acid+from+Bacillus+subtilis+upregulates+pro-inflammatory+cytokines+while+inhibiting+NLRP3%2C+NLRC4+and+AIM2+inflammasome+activation&doi=10.1038%2Fcmi.2016.13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Poly-gamma-glutamic acid from Bacillus subtilis upregulates pro-inflammatory cytokines while inhibiting NLRP3, NLRC4 and AIM2 inflammasome activation</span></div><div class="casAuthors">Ahn, Huijeong; Kang, Seung Goo; Yoon, Sung-il; Kim, Pyeung-Hyeun; Kim, Doo; Lee, Geun-Shik</div><div class="citationInfo"><span class="NLM_cas:title">Cellular & Molecular Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">CMIEAO</span>;
        ISSN:<span class="NLM_cas:issn">1672-7681</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Poly-gamma-glutamic acid (γ-PGA) is a natural, edible and non-toxic polymer synthesized by Bacillus subtilis and is suggested as a safe biomaterial for the use in hydrogels and vaccine adjuvants.  However, the effect of γ-PGA on inflammasome activation has not yet been studied in macrophages.  Inflammasomes, which are intracellular multi-protein complexes, promote acute and chronic inflammation via interleukin-1β or interleukin-18 maturation, and they are known targets for metabolic syndromes and cancer.  In this study, we obsd. that γ-PGA attenuated NLRP3, NLRC4 and AIM2 inflammasome activation, whereas it upregulated pro-inflammatory cytokine expression in human and murine macrophages.  Although γ-PGA had conflicting effects on cytokine prodn. and maturation, it clearly alleviated the severity of lipopolysaccharide-induced endotoxin shock in an animal model.  Thus, we suggest γ-PGA as a candidate to control inflammasome-mediated disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb13M4HBqOD7Vg90H21EOLACvtfcHk0lgWMdTNYiRTsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGrsb4%253D&md5=10d604b0efb3a74c08a880199acdc18b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fcmi.2016.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcmi.2016.13%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DH.%26aulast%3DKang%26aufirst%3DS.%2BG.%26aulast%3DYoon%26aufirst%3DS.-i.%26aulast%3DKim%26aufirst%3DP.-H.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DG.-S.%26atitle%3DPoly-gamma-glutamic%2520acid%2520from%2520Bacillus%2520subtilis%2520upregulates%2520pro-inflammatory%2520cytokines%2520while%2520inhibiting%2520NLRP3%252C%2520NLRC4%2520and%2520AIM2%2520inflammasome%2520activation%26jtitle%3DCell.%2520Mol.%2520Immunol.%26date%3D2018%26volume%3D15%26spage%3D111%26doi%3D10.1038%2Fcmi.2016.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, E.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, D.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasakawa, C.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms regulating NLRP3 inflammasome activation</span>. <i>Cell. Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">148</span>, <span class="refDoi"> DOI: 10.1038/cmi.2015.95</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fcmi.2015.95" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=26549800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs1Gms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=148&author=E.-K.+Joauthor=J.+K.+Kimauthor=D.-M.+Shinauthor=C.+Sasakawa&title=Molecular+mechanisms+regulating+NLRP3+inflammasome+activation&doi=10.1038%2Fcmi.2015.95"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms regulating NLRP3 inflammasome activation</span></div><div class="casAuthors">Jo, Eun-Kyeong; Kim, Jin Kyung; Shin, Dong-Min; Sasakawa, Chihiro</div><div class="citationInfo"><span class="NLM_cas:title">Cellular & Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-159</span>CODEN:
                <span class="NLM_cas:coden">CMIEAO</span>;
        ISSN:<span class="NLM_cas:issn">1672-7681</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Inflammasomes are multi-protein signaling complexes that trigger the activation of inflammatory caspases and the maturation of interleukin-1β.  Among various inflammasome complexes, the NLRP3 inflammasome is best characterized and has been linked with various human autoinflammatory and autoimmune diseases.  Thus, the NLRP3 inflammasome may be a promising target for anti-inflammatory therapies.  In this review, we summarize the current understanding of the mechanisms by which the NLRP3 inflammasome is activated in the cytosol.  We also describe the binding partners of NLRP3 inflammasome complexes activating or inhibiting the inflammasome assembly.  Our knowledge of the mechanisms regulating NLRP3 inflammasome signaling and how these influence inflammatory responses offers further insight into potential therapeutic strategies to treat inflammatory diseases assocd. with dysregulation of the NLRP3 inflammasome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJBv_NNzcwFLVg90H21EOLACvtfcHk0lgWMdTNYiRTsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs1Gms7s%253D&md5=90c508a823912cf69f86cefac7aaaaec</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fcmi.2015.95&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcmi.2015.95%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DE.-K.%26aulast%3DKim%26aufirst%3DJ.%2BK.%26aulast%3DShin%26aufirst%3DD.-M.%26aulast%3DSasakawa%26aufirst%3DC.%26atitle%3DMolecular%2520mechanisms%2520regulating%2520NLRP3%2520inflammasome%2520activation%26jtitle%3DCell.%2520Mol.%2520Immunol.%26date%3D2016%26volume%3D13%26spage%3D148%26doi%3D10.1038%2Fcmi.2015.95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Samra, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Said, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsherbiny, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liou, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Shishtawy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eissa, L. A.</span></span> <span> </span><span class="NLM_article-title">Cepharanthine and piperine ameliorate diabetic nephropathy in rats: role of NF-κB and NLRP3 inflammasome</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1016/j.lfs.2016.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.lfs.2016.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27266851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGiurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2016&pages=187-199&author=Y.+A.+Samraauthor=H.+S.+Saidauthor=N.+M.+Elsherbinyauthor=G.+I.+Liouauthor=M.+M.+El-Shishtawyauthor=L.+A.+Eissa&title=Cepharanthine+and+piperine+ameliorate+diabetic+nephropathy+in+rats%3A+role+of+NF-%CE%BAB+and+NLRP3+inflammasome&doi=10.1016%2Fj.lfs.2016.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Cepharanthine and Piperine ameliorate diabetic nephropathy in rats: role of NF-κB and NLRP3 inflammasome</span></div><div class="casAuthors">Samra, Yara A.; Said, Heba S.; Elsherbiny, Nehal M.; Liou, Gregory I.; El-Shishtawy, Mamdouh M.; Eissa, Laila A.</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">187-199</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Hyperglycemia leads to elevation of oxidative stress and proinflammatory cytokines which are the main causes of diabetic nephropathy (DN).  NLRP3 inflammasome and thioredoxin-interacting protein (TXNIP) are recently assumed to participate in the development of DN.  We aimed to investigate the effects of Cepharanthine (CEP), Piperine (Pip) and their combination in streptozotocin (STZ)-induced DN focusing on their role to modulate NLRP3 and TXNIP induced inflammation.  Diabetic rats were treated with i.p. (i.p.) injection of CEP (10 mg/kg/day), Pip (30 mg/kg/day) or their combination for 8 wk.  Nuclear factor kappa B (NF-κB), tumor necrosis factor alpha (TNF-α) and interleukin-1β (IL-1β) were assessed by ELISA technique.  TXNIP and NLRP3 genes expressions were evaluated by real time-PCR.  Diabetic rats showed significant increase in renal TXNIP and NLRP3 expression.  CEP, Pip or their combination significantly decreased TXNIP and NLRP3 expression in diabetic kidneys.  Hyperglycemia induced NF-κB activation leading to increased IL-1β and TNF-α levels.  CEP, Pip or their combination showed significant inhibition of NF-κB together with decreased IL-1β and TNF-α levels in diabetic rats.  Also, diabetic rats showed significant decrease in creatinine clearance and increase in blood glucose, serum creatinine, blood urea nitrogen, malondialdehyde, proteinuria, and kidney wt. to body Wt. ratio.  All of these changes were reversed by CEP, Pip or their combination.  The antioxidant and anti-inflammatory effects of CEP and Pip which were accompanied by inhibition of NF-κB and NLRP3 activation might be helpful mechanisms to halt the progression of DN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZEuultRHBi7Vg90H21EOLACvtfcHk0lgWMdTNYiRTsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGiurfN&md5=aa668b4f5d012de254f66f16b6cec0fe</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.lfs.2016.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lfs.2016.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DSamra%26aufirst%3DY.%2BA.%26aulast%3DSaid%26aufirst%3DH.%2BS.%26aulast%3DElsherbiny%26aufirst%3DN.%2BM.%26aulast%3DLiou%26aufirst%3DG.%2BI.%26aulast%3DEl-Shishtawy%26aufirst%3DM.%2BM.%26aulast%3DEissa%26aufirst%3DL.%2BA.%26atitle%3DCepharanthine%2520and%2520piperine%2520ameliorate%2520diabetic%2520nephropathy%2520in%2520rats%253A%2520role%2520of%2520NF-%25CE%25BAB%2520and%2520NLRP3%2520inflammasome%26jtitle%3DLife%2520Sci.%26date%3D2016%26volume%3D157%26spage%3D187%26epage%3D199%26doi%3D10.1016%2Fj.lfs.2016.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergsbaken, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cookson, B. T.</span></span> <span> </span><span class="NLM_article-title">Macrophage activation redirects yersinia-infected host cell death from apoptosis to caspase-1-dependent pyroptosis</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e161</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.0030161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1371%2Fjournal.ppat.0030161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=17983266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A280%3ADC%252BD2sjhvFymtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&author=T.+Bergsbakenauthor=B.+T.+Cookson&title=Macrophage+activation+redirects+yersinia-infected+host+cell+death+from+apoptosis+to+caspase-1-dependent+pyroptosis&doi=10.1371%2Fjournal.ppat.0030161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Macrophage activation redirects yersinia-infected host cell death from apoptosis to caspase-1-dependent pyroptosis</span></div><div class="casAuthors">Bergsbaken Tessa; Cookson Brad T</div><div class="citationInfo"><span class="NLM_cas:title">PLoS pathogens</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e161</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Infection of macrophages by Yersinia species results in YopJ-dependent apoptosis, and naive macrophages are highly susceptible to this form of cell death.  Previous studies have demonstrated that macrophages activated with lipopolysaccharide (LPS) prior to infection are resistant to YopJ-dependent cell death; we found this simultaneously renders macrophages susceptible to killing by YopJ(-) Yersinia pseudotuberculosis (Yptb).  YopJ(-) Yptb-induced macrophage death was dependent on caspase-1 activation, resulting in rapid permeability to small molecules, followed by membrane breakdown and DNA damage, and accompanied by cleavage and release of proinflammatory interleukin-18.  Induction of caspase-1-dependent death, or pyroptosis, required the bacterial type III translocon but none of its known translocated proteins.  Wild-type Yptb infection also triggered pyroptosis: YopJ-dependent activation of proapoptotic caspase-3 was significantly delayed in activated macrophages and resulted in caspase-1-dependent pyroptosis.  The transition to susceptibility was not limited to LPS activation; it was also seen in macrophages activated with other Toll-like receptor (TLR) ligands and intact nonviable bacteria.  Yptb infection triggered macrophage activation and activation of caspase-1 in vivo.  Y. pestis infection of activated macrophages also stimulated caspase-1 activation.  These results indicate that host signaling triggered by TLR and other activating ligands during the course of Yersinia infection redirects both the mechanism of host cell death and the downstream consequences of death by shifting from noninflammatory apoptosis to inflammatory pyroptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRL3C0hooqMyzEwO064Z95rfW6udTcc2ebIjLSl6n0vmbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjhvFymtQ%253D%253D&md5=0489d306a5ee2f455204cf65cbed236a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.0030161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.0030161%26sid%3Dliteratum%253Aachs%26aulast%3DBergsbaken%26aufirst%3DT.%26aulast%3DCookson%26aufirst%3DB.%2BT.%26atitle%3DMacrophage%2520activation%2520redirects%2520yersinia-infected%2520host%2520cell%2520death%2520from%2520apoptosis%2520to%2520caspase-1-dependent%2520pyroptosis%26jtitle%3DPLoS%2520Pathog.%26date%3D2007%26volume%3D3%26doi%3D10.1371%2Fjournal.ppat.0030161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miao, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leaf, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treuting, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dors, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wewers, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aderem, A.</span></span> <span> </span><span class="NLM_article-title">Caspase-1-induced pyroptosis is an innate immune effector mechanism against intracellular bacteria</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1136</span>, <span class="refDoi"> DOI: 10.1038/ni.1960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fni.1960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=21057511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2jtrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=1136&author=E.+A.+Miaoauthor=I.+A.+Leafauthor=P.+M.+Treutingauthor=D.+P.+Maoauthor=M.+Dorsauthor=A.+Sarkarauthor=S.+E.+Warrenauthor=M.+D.+Wewersauthor=A.+Aderem&title=Caspase-1-induced+pyroptosis+is+an+innate+immune+effector+mechanism+against+intracellular+bacteria&doi=10.1038%2Fni.1960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Caspase-1-induced pyroptosis is an innate immune effector mechanism against intracellular bacteria</span></div><div class="casAuthors">Miao, Edward A.; Leaf, Irina A.; Treuting, Piper M.; Mao, Dat P.; Dors, Monica; Sarkar, Anasuya; Warren, Sarah E.; Wewers, Mark D.; Aderem, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1136-1142</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macrophages mediate crucial innate immune responses via caspase-1-dependent processing and secretion of interleukin 1β (IL-1β) and IL-18.  Although infection with wild-type Salmonella typhimurium is lethal to mice, we show here that a strain that persistently expresses flagellin was cleared by the cytosolic flagellin-detection pathway through the activation of caspase-1 by the NLRC4 inflammasome; however, this clearance was independent of IL-1β and IL-18.  Instead, caspase-1-induced pyroptotic cell death released bacteria from macrophages and exposed the bacteria to uptake and killing by reactive oxygen species in neutrophils.  Similarly, activation of caspase-1 cleared unmanipulated Legionella pneumophila and Burkholderia thailandensis by cytokine-independent mechanisms.  This demonstrates that activation of caspase-1 clears intracellular bacteria in vivo independently of IL-1β and IL-18 and establishes pyroptosis as an efficient mechanism of bacterial clearance by the innate immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXdPstV9EOz7Vg90H21EOLACvtfcHk0lhw6WKtCWwAlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2jtrrK&md5=1739613c63616f5c401a543819698d49</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fni.1960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.1960%26sid%3Dliteratum%253Aachs%26aulast%3DMiao%26aufirst%3DE.%2BA.%26aulast%3DLeaf%26aufirst%3DI.%2BA.%26aulast%3DTreuting%26aufirst%3DP.%2BM.%26aulast%3DMao%26aufirst%3DD.%2BP.%26aulast%3DDors%26aufirst%3DM.%26aulast%3DSarkar%26aufirst%3DA.%26aulast%3DWarren%26aufirst%3DS.%2BE.%26aulast%3DWewers%26aufirst%3DM.%2BD.%26aulast%3DAderem%26aufirst%3DA.%26atitle%3DCaspase-1-induced%2520pyroptosis%2520is%2520an%2520innate%2520immune%2520effector%2520mechanism%2520against%2520intracellular%2520bacteria%26jtitle%3DNat.%2520Immunol.%26date%3D2010%26volume%3D11%26spage%3D1136%26doi%3D10.1038%2Fni.1960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Measures, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of bromodomain–acetyl-lysine interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1021/cb500996u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500996u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=22-39&author=M.+Brandauthor=A.+M.+Measuresauthor=B.+G.+Wilsonauthor=W.+A.+Cortopassiauthor=R.+Alexanderauthor=M.+H%C3%B6ssauthor=D.+S.+Hewingsauthor=T.+P.+Rooneyauthor=R.+S.+Patonauthor=S.+J.+Conway&title=Small+molecule+inhibitors+of+bromodomain%E2%80%93acetyl-lysine+interactions&doi=10.1021%2Fcb500996u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Bromodomain-Acetyl-lysine Interactions</span></div><div class="casAuthors">Brand, Michael; Measures, Angelina R.; Wilson, Brian G.; Cortopassi, Wilian A.; Alexander, Rikki; Hoss, Matthias; Hewings, David S.; Rooney, Timothy P. C.; Paton, Robert S.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-39</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Bromodomains are protein modules that bind to acetylated lysine residues.  Their interaction with histone proteins suggests that they function as "readers" of histone lysine acetylation, a component of the proposed "histone code".  Bromodomain-contg. proteins are often found as components of larger protein complexes with roles in fundamental cellular process including transcription.  The publication of two potent ligands for the BET bromodomains in 2010 demonstrated that small mols. can inhibit the bromodomain-acetyl-lysine protein-protein interaction.  These mols. display strong phenotypic effects in a no. of cell lines and affect a range of cancers in vivo.  This work stimulated intense interest in developing further ligands for the BET bromodomains and the design of ligands for non-BET bromodomains.  Here we review the recent progress in the field with particular attention paid to ligand design, the assays employed in early ligand discovery, and the use of computational approaches to inform ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1JZtWpBpqS7Vg90H21EOLACvtfcHk0lhw6WKtCWwAlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D&md5=e40e85fc300109654712a00de400c8ba</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fcb500996u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500996u%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DMeasures%26aufirst%3DA.%2BM.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DAlexander%26aufirst%3DR.%26aulast%3DH%25C3%25B6ss%26aufirst%3DM.%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520bromodomain%25E2%2580%2593acetyl-lysine%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D22%26epage%3D39%26doi%3D10.1021%2Fcb500996u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winston, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">The bromodomain: a chromatin-targeting module?</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">601</span>, <span class="refDoi"> DOI: 10.1038/10640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2F10640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10404206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADyaK1MXksVentbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=601&author=F.+Winstonauthor=C.+D.+Allis&title=The+bromodomain%3A+a+chromatin-targeting+module%3F&doi=10.1038%2F10640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain: a chromatin-targeting module?</span></div><div class="casAuthors">Winston, Fred; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">601-604</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">A review and discussion with 69 refs.  It has recently been demonstrated that bromodomains - motifs found in several eukaryotic transcription factors - bind to acetyl-lysine, a modification of histones that is important for transcription.  This finding suggests that the regulatory effects of histone acetylation may be exerted by bromodomain-contg. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-zV1Z0XMw8LVg90H21EOLACvtfcHk0lhn-wBoFLDyUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXksVentbs%253D&md5=1d5aa880eaaa4734b39d49c49c874bb3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F10640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F10640%26sid%3Dliteratum%253Aachs%26aulast%3DWinston%26aufirst%3DF.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520bromodomain%253A%2520a%2520chromatin-targeting%2520module%253F%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D601%26doi%3D10.1038%2F10640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuppusetty, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">3,5-dimethylisoxazoles act as acetyl-lysine-mimetic bromodomain ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6761</span>– <span class="NLM_lpage">6770</span>, <span class="refDoi"> DOI: 10.1021/jm200640v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&author=D.+S.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knapp&title=3%2C5-dimethylisoxazoles+act+as+acetyl-lysine-mimetic+bromodomain+ligands&doi=10.1021%2Fjm200640v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Dimethylisoxazoles Act As Acetyl-lysine-mimetic Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Wang, Minghua; Philpott, Martin; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Vuppusetty, Chaitanya; Marsden, Brian; Knapp, Stefan; Conway, Stuart J.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6761-6770</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-lysine acetylation is a vital chromatin post-translational modification involved in the epigenetic regulation of gene transcription.  Bromodomains bind acetylated lysines, acting as readers of the histone-acetylation code.  Competitive inhibitors of this interaction have antiproliferative and anti-inflammatory properties.  With 57 distinct bromodomains known, the discovery of subtype-selective inhibitors of the histone-bromodomain interaction is of great importance.  We have identified the 3,5-dimethylisoxazole moiety as a novel acetyl-lysine bioisostere, which displaces acetylated histone-mimicking peptides from bromodomains.  Using X-ray crystallog. anal., we have detd. the interactions responsible for the activity and selectivity of 4-substituted 3,5-dimethylisoxazoles against a selection of phylogenetically diverse bromodomains.  By exploiting these interactions, we have developed compd. 4d, which has IC50 values of <5 μM for the bromodomain-contg. proteins BRD2(1) and BRD4(1).  These compds. are promising leads for the further development of selective probes for the bromodomain and extra C-terminal domain (BET) family and CREBBP bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEzK4xMvJhXrVg90H21EOLACvtfcHk0lhn-wBoFLDyUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO&md5=7d6f9eff2735c8a7189dec4fb5c5d13d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3D3%252C5-dimethylisoxazoles%2520act%2520as%2520acetyl-lysine-mimetic%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6761%26epage%3D6770%26doi%3D10.1021%2Fjm200640v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangos&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lhn-wBoFLDyUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. H.</span></span> <span> </span><span class="NLM_article-title">RNA polymerase II elongation control</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">119</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-052610-095910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1146%2Fannurev-biochem-052610-095910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=22404626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVGls7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2012&pages=119&author=Q.+Zhouauthor=T.+Liauthor=D.+H.+Price&title=RNA+polymerase+II+elongation+control&doi=10.1146%2Fannurev-biochem-052610-095910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RNA polymerase II elongation control</span></div><div class="casAuthors">Zhou, Qiang; Li, Tiandao; Price, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">119-143</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Regulation of the elongation phase of transcription by RNA polymerase II (Pol II) is utilized extensively to generate the pattern of mRNAs needed to specify cell types and to respond to environmental changes.  After Pol II initiates, neg. elongation factors cause it to pause in a promoter proximal position.  These polymerases are poised to respond to the pos. transcription elongation factor P-TEFb, and then enter productive elongation only under the appropriate set of signals to generate full-length properly processed mRNAs.  Recent global analyses of Pol II and elongation factors, mechanisms that regulate P-TEFb involving the 7SK small nuclear ribonucleoprotein (snRNP), factors that control both the neg. and pos. elongation properties of Pol II, and the mRNA processing events that are coupled with elongation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60CSYt9WFzLVg90H21EOLACvtfcHk0lhn-wBoFLDyUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVGls7bN&md5=38b677c9655caf0e767b1ea61e4b47b9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-052610-095910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-052610-095910%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DPrice%26aufirst%3DD.%2BH.%26atitle%3DRNA%2520polymerase%2520II%2520elongation%2520control%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2012%26volume%3D81%26spage%3D119%26doi%3D10.1146%2Fannurev-biochem-052610-095910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolgemuth, D. J.</span></span> <span> </span><span class="NLM_article-title">The first bromodomain of Brdt, a testis-specific member of the BET sub-family of double-bromodomain-containing proteins, is essential for male germ cell differentiation</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">3507</span>, <span class="refDoi"> DOI: 10.1242/dev.004481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1242%2Fdev.004481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=17728347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlSitbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2007&pages=3507&author=E.+Shangauthor=H.+D.+Nickersonauthor=D.+Wenauthor=X.+Wangauthor=D.+J.+Wolgemuth&title=The+first+bromodomain+of+Brdt%2C+a+testis-specific+member+of+the+BET+sub-family+of+double-bromodomain-containing+proteins%2C+is+essential+for+male+germ+cell+differentiation&doi=10.1242%2Fdev.004481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The first bromodomain of Brdt, a testis-specific member of the BET sub-family of double-bromodomain-containing proteins, is essential for male germ cell differentiation</span></div><div class="casAuthors">Shang, Enyuan; Nickerson, Helen D.; Wen, Duancheng; Wang, Xiangyuan; Wolgemuth, Debra J.</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3507-3515</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Brdt is a testis-specific member of the distinctive BET sub-family of bromodomain motif-contg. proteins, a motif that binds acetylated lysines and is implicated in chromatin remodeling.  Its expression is restricted to the germ line, specifically to pachytene and diplotene spermatocytes and early spermatids.  Targeted mutagenesis was used to generate mice carrying a mutant allele of Brdt, BrdtΔBD1, which lacks only the first of the two bromodomains that uniquely characterize BET proteins.  Homozygous BrdtΔBD1/ΔBD1 mice were viable but males were sterile, producing fewer and morphol. abnormal sperm.  Aberrant morphogenesis was first detected in step 9 elongating spermatids, and those elongated spermatids that were formed lacked the distinctive foci of heterochromatin at the peri-nuclear envelope.  Quant. reverse transcription (RT)-PCR showed threefold increased levels of histone H1t (Hist1h1t) in BrdtΔBD1/ΔBD1 testes and chromatin immunopptn. revealed that Brdt protein, but not BrdtΔBD1 protein, was assocd. with the promoter of H1t.  Intracytoplasmic sperm injection suggested that the DNA in the BrdtΔBD1 mutant sperm could support early embryonic development and yield functional embryonic stem cells.  This is the first demonstration that deletion of just one of the two bromodomains in members of the BET sub-family of bromodomain-contg. proteins has profound effects on in vivo differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDa2lSw-mPaLVg90H21EOLACvtfcHk0lgFwoGx9l1c9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlSitbfJ&md5=f8ef8a8ab721f0eebf4441ffc35eee93</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1242%2Fdev.004481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.004481%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DE.%26aulast%3DNickerson%26aufirst%3DH.%2BD.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWolgemuth%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520first%2520bromodomain%2520of%2520Brdt%252C%2520a%2520testis-specific%2520member%2520of%2520the%2520BET%2520sub-family%2520of%2520double-bromodomain-containing%2520proteins%252C%2520is%2520essential%2520for%2520male%2520germ%2520cell%2520differentiation%26jtitle%3DDevelopment%26date%3D2007%26volume%3D134%26spage%3D3507%26doi%3D10.1242%2Fdev.004481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0lgFwoGx9l1c9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delmore, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilpatrick, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qi&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0lgFwoGx9l1c9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Q.+Liuauthor=P.+Y.+Wangauthor=H.+Y.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lhj3KWVUMrBpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%2BQ.%26aulast%3DWang%26aufirst%3DP.%2BY.%26aulast%3DChen%26aufirst%3DH.%2BY.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medzhitov, R.</span></span> <span> </span><span class="NLM_article-title">Control of inducible gene expression by signal-dependent transcriptional elongation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.05.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.cell.2009.05.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=19596240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1ygurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2009&pages=129-145&author=D.+C.+Hargreavesauthor=T.+Horngauthor=R.+Medzhitov&title=Control+of+inducible+gene+expression+by+signal-dependent+transcriptional+elongation&doi=10.1016%2Fj.cell.2009.05.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Control of inducible gene expression by signal-dependent transcriptional elongation</span></div><div class="casAuthors">Hargreaves, Diana C.; Horng, Tiffany; Medzhitov, Ruslan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-145</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Most inducible transcriptional programs consist of primary and secondary response genes (PRGs and SRGs) that differ in their kinetics of expression and in their requirements for new protein synthesis and chromatin remodeling.  Here we show that many PRGs, in contrast to SRGs, have preassembled RNA polymerase II (Pol II) and pos. histone modifications at their promoters in the basal state.  Pol II at PRGs generates low levels of full-length unspliced transcripts but fails to make mature, protein-coding transcripts in the absence of stimulation.  Induction of PRGs is controlled at the level of transcriptional elongation and mRNA processing, through the signal-dependent recruitment of P-TEFb.  P-TEFb is in turn recruited by the bromodomain-contg. protein Brd4, which detects H4K5/8/12Ac inducibly acquired at PRG promoters.  Our findings suggest that the permissive structure of PRGs both stipulates their unique regulation in the basal state by corepressor complexes and enables their rapid induction in multiple cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFmjH5y-GkbrVg90H21EOLACvtfcHk0lhj3KWVUMrBpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1ygurk%253D&md5=e5292556b16506183db147693db3450a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.05.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.05.047%26sid%3Dliteratum%253Aachs%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHorng%26aufirst%3DT.%26aulast%3DMedzhitov%26aufirst%3DR.%26atitle%3DControl%2520of%2520inducible%2520gene%2520expression%2520by%2520signal-dependent%2520transcriptional%2520elongation%26jtitle%3DCell%26date%3D2009%26volume%3D138%26spage%3D129%26epage%3D145%26doi%3D10.1016%2Fj.cell.2009.05.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">BET N-terminal bromodomain inhibition selectively blocks Th17 cell differentiation and ameliorates colitis in mice</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2952</span>, <span class="refDoi"> DOI: 10.1073/pnas.1615601114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1073%2Fpnas.1615601114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=28265070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVyjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=2952&author=K.+Cheungauthor=G.+Luauthor=R.+Sharmaauthor=A.+Vincekauthor=R.+Zhangauthor=A.+N.+Plotnikovauthor=F.+Zhangauthor=Q.+Zhangauthor=Y.+Juauthor=Y.+Hu&title=BET+N-terminal+bromodomain+inhibition+selectively+blocks+Th17+cell+differentiation+and+ameliorates+colitis+in+mice&doi=10.1073%2Fpnas.1615601114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">BET N-terminal bromodomain inhibition selectively blocks Th17 cell differentiation and ameliorates colitis in mice</span></div><div class="casAuthors">Cheung, Kalung; Lu, Geming; Sharma, Rajal; Vincek, Adam; Zhang, Ruihua; Plotnikov, Alexander N.; Zhang, Fan; Zhang, Qiang; Ju, Ying; Hu, Yuan; Zhao, Li; Han, Xinye; Meslamani, Jamel; Xu, Feihong; Jaganathan, Anbalagan; Shen, Tong; Zhu, Hongfa; Rusinova, Elena; Zeng, Lei; Zhou, Jiachi; Yang, Jianjun; Peng, Liang; Ohlmeyer, Michael; Walsh, Martin J.; Zhang, David Y.; Xiong, Huabao; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2952-2957</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">T-helper 17 (Th17) cells have important functions in adaptor immunity and have also been implicated in inflammatory disorders.  The bromodomain and extraterminal domain (BET) family proteins regulate gene transcription during lineage-specific differentiation of naive CD4+ T cells to produce mature T-helper cells.  Inhibition of acetyl-lysine binding of the BET proteins by pan-BET bromodomain (BrD) inhibitors, such as JQ1, broadly affects differentiation of Th17, Th1, and Th2 cells that have distinct immune functions, thus limiting their therapeutic potential.  Whether these BET proteins represent viable new epigenetic drug targets for inflammatory disorders has remained an unanswered question.  In this study, we report that selective inhibition of the first bromodomain of BET proteins with our newly designed small mol. MS402 inhibits primarily Th17 cell differentiation with a little or almost no effect on Th1 or Th2 and Treg cells.  MS402 preferentially renders Brd4 binding to Th17 signature gene loci over those of housekeeping genes and reduces Brd4 recruitment of p-TEFb to phosphorylate and activate RNA polymerase II for transcription elongation.  We further show that MS402 prevents and ameliorates T-cell transfer-induced colitis in mice by blocking Th17 cell overdevelopment.  Thus, selective pharmacol. modulation of individual bromodomains likely represents a strategy for treatment of inflammatory bowel diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBIvlTKnR97LVg90H21EOLACvtfcHk0lhj3KWVUMrBpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVyjsbo%253D&md5=367b7c0545edde6cc0ef107e03e1310f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1615601114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1615601114%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DJu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DY.%26atitle%3DBET%2520N-terminal%2520bromodomain%2520inhibition%2520selectively%2520blocks%2520Th17%2520cell%2520differentiation%2520and%2520ameliorates%2520colitis%2520in%2520mice%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3D2952%26doi%3D10.1073%2Fpnas.1615601114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span>; <span class="NLM_string-name">Zhu, L.</span>; <span class="NLM_string-name">Zhang, T.</span>; <span class="NLM_string-name">Lu, H.</span>; <span class="NLM_string-name">Wang, C.</span>; <span class="NLM_string-name">Xue, B.</span>; <span class="NLM_string-name">Xu, X.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Cai, Z.</span>; <span class="NLM_string-name">Sang, W.</span></span> <span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">BRD4 has dual effects on the HMGB1 and NF-κB signalling pathways and is a potential therapeutic target for osteoarthritis</span>. <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume">1863</span>,  <span class="NLM_fpage">3001</span>. <span class="refDoi"> DOI: 10.1016/j.bbadis.2017.08.009</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.bbadis.2017.08.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1863&publication_year=2017&pages=3001&author=Y.+Jiang&author=L.+Zhu&author=T.+Zhang&author=H.+Lu&author=C.+Wang&author=B.+Xue&author=X.+Xu&author=Y.+Liu&author=Z.+Cai&author=W.+Sang&title=BRD4+has+dual+effects+on+the+HMGB1+and+NF-%CE%BAB+signalling+pathways+and+is+a+potential+therapeutic+target+for+osteoarthritis&doi=10.1016%2Fj.bbadis.2017.08.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbadis.2017.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbadis.2017.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DBRD4%2520has%2520dual%2520effects%2520on%2520the%2520HMGB1%2520and%2520NF-%25CE%25BAB%2520signalling%2520pathways%2520and%2520is%2520a%2520potential%2520therapeutic%2520target%2520for%2520osteoarthritis%26date%3D2017%26volume%3D1863%26spage%3D3001%26doi%3D10.1016%2Fj.bbadis.2017.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ball, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipman, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liby, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pioli, P. A.</span></span> <span> </span><span class="NLM_article-title">CDDO-me redirects activation of breast tumor associated macrophages</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0149600</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0149600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1371%2Fjournal.pone.0149600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=26918785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVylsrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=M.+S.+Ballauthor=E.+P.+Shipmanauthor=H.+Kimauthor=K.+T.+Libyauthor=P.+A.+Pioli&title=CDDO-me+redirects+activation+of+breast+tumor+associated+macrophages&doi=10.1371%2Fjournal.pone.0149600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">CDDO-Me redirects activation of breast tumor associated macrophages</span></div><div class="casAuthors">Ball, Michael S.; Shipman, Emilie P.; Kim, Hyunjung; Liby, Karen T.; Pioli, Patricia A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e0149600/1-e0149600/18</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Tumor-assocd. macrophages can account for up to 50% of the tumor mass in breast cancer patients and high TAM d. is assocd. with poor clin. prognosis.  Because TAMs enhance tumor growth, development, and metastatic potential, redirection of TAM activation may have significant therapeutic benefit.  Our studies in primary human macrophages and murine breast TAMs suggest that the synthetic oleanane triterpenoid CDDOmethyl ester (CDDO-Me) reprograms the activation profile of TAMs from tumor-promoting to tumor-inhibiting.  We show that CDDO-Me treatment inhibits expression of IL-10 and VEGF in stimulated human M2 macrophages and TAMs but increases expression of TNF-α and IL-6.  Surface expression of CD206 and CD163, which are characteristic of M2 activation, is significantly attenuated by CDDO-Me.  In contrast, CDDO-Me up-regulates surface expression of HLA-DR and CD80, which are markers of M1 activation, and importantly potentiates macrophage activation of autologous T cells but inhibits endothelial cell vascularization.  These results show for the first time that CDDO-Me redirects activation of M2 macrophages and TAMs from immune-suppressive to immune-stimulatory, and implicate a role for CDDO-Me as an immunotherapeutic in the treatment of breast and potentially other types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO7q7YdphPZ7Vg90H21EOLACvtfcHk0lg_OURDQ6tx_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVylsrzN&md5=6b3b749c446c1deca82158999752e153</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0149600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0149600%26sid%3Dliteratum%253Aachs%26aulast%3DBall%26aufirst%3DM.%2BS.%26aulast%3DShipman%26aufirst%3DE.%2BP.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DLiby%26aufirst%3DK.%2BT.%26aulast%3DPioli%26aufirst%3DP.%2BA.%26atitle%3DCDDO-me%2520redirects%2520activation%2520of%2520breast%2520tumor%2520associated%2520macrophages%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0149600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0lg_OURDQ6tx_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabiec, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ospelt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422-429&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+E.+Gayauthor=C.+Kollingauthor=L.+L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- α, interleukin (IL)-1β and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- α and IL-1β .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0lg_OURDQ6tx_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BE.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.%2BL.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D422%26epage%3D429%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span> <span> </span><span class="NLM_article-title">Bromodomain protein inhibition: a novel therapeutic strategy in rheumatic diseases</span>. <i>RMD Open</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e000744</span> <span class="refDoi"> DOI: 10.1136/rmdopen-2018-000744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1136%2Frmdopen-2018-000744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30564450" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&author=K.+Klein&title=Bromodomain+protein+inhibition%3A+a+novel+therapeutic+strategy+in+rheumatic+diseases&doi=10.1136%2Frmdopen-2018-000744"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1136%2Frmdopen-2018-000744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Frmdopen-2018-000744%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26atitle%3DBromodomain%2520protein%2520inhibition%253A%2520a%2520novel%2520therapeutic%2520strategy%2520in%2520rheumatic%2520diseases%26jtitle%3DRMD%2520Open%26date%3D2018%26volume%3D4%26doi%3D10.1136%2Frmdopen-2018-000744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sherbeeny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoheir, K. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attia, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hosaini, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Sharary, S. D.</span></span> <span> </span><span class="NLM_article-title">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.phrs.2015.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=26149470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=248-257&author=A.+Nadeemauthor=N.+O.+Al-Harbiauthor=M.+M.+Al-Harbiauthor=A.+M.+El-Sherbeenyauthor=S.+F.+Ahmadauthor=N.+Siddiquiauthor=M.+A.+Ansariauthor=K.+M.+A.+Zoheirauthor=S.+M.+Attiaauthor=K.+A.+Al-Hosainiauthor=S.+D.+Al-Sharary&title=Imiquimod-induced+psoriasis-like+skin+inflammation+is+suppressed+by+BET+bromodomain+inhibitor+in+mice+through+RORC%2FIL-17A+pathway+modulation&doi=10.1016%2Fj.phrs.2015.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span></div><div class="casAuthors">Nadeem, Ahmed; Al-Harbi, Naif O.; Al-Harbi, Mohamed M.; El-Sherbeeny, Ahmed M.; Ahmad, Sheikh F.; Siddiqui, Nahid; Ansari, Mushtaq A.; Zoheir, Khairy M. A.; Attia, Sabry M.; Al-Hosaini, Khaled A.; Al-Sharary, Shakir D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Psoriasis is one of the most common skin disorders characterized by erythematous plaques that result from hyperproliferative keratinocytes and infiltration of inflammatory leukocytes into dermis and epidermis.  Recent studies suggest that IL-23/IL-17A/IL-22 cytokine axis plays an important role in the pathogenesis of psoriasis.  The small mol. bromodomain and extraterminal domain (BET) inhibitors, that disrupt interaction of BET proteins with acetylated histones have recently demonstrated efficacy in various models of inflammation through suppression of several pathways, one of them being synthesis of IL-17A/IL-22 which primarily depends on transcription factor, retinoic acid receptor-related orphan receptor C (RORC).  However, the efficacy and mechanistic aspect of a BET inhibitor in mouse model of skin inflammation has not been explored previously.  Therefore, this study investigated the role of BET inhibitor, JQ-1 in mouse model of psoriasis-like inflammation.  Mice were topically applied imiquimod (IMQ) to develop psoriasis-like inflammation on the shaved back and ear followed by assessment of skin inflammation (myeloperoxidase activity, ear thickness, and histopathol.), RORC and its signature cytokines (IL-17A/IL-22).  JQ-1 suppressed IMQ-induced skin inflammation as reflected by a decrease in ear thickness/myeloperoxidase activity, and RORC/IL-17A/IL-22 expression.  Addnl., a RORα/γ agonist SR1078 was utilized to investigate the role of RORC in BET-mediated skin inflammation.  SR1078 reversed the protective effect of JQ-1 on skin inflammation at both histol. and mol. levels in the IMQ model.  The current study suggests that BET bromodomains are involved in psoriasis-like inflammation through induction of RORC/IL-17A pathway.  Therefore, inhibition of BET bromodomains may provide a new therapy against skin inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOnC_sPUMSLVg90H21EOLACvtfcHk0lhmZmPY6msPTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I&md5=3edb10526e30c18bce037dc0aca611a4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%26aufirst%3DA.%26aulast%3DAl-Harbi%26aufirst%3DN.%2BO.%26aulast%3DAl-Harbi%26aufirst%3DM.%2BM.%26aulast%3DEl-Sherbeeny%26aufirst%3DA.%2BM.%26aulast%3DAhmad%26aufirst%3DS.%2BF.%26aulast%3DSiddiqui%26aufirst%3DN.%26aulast%3DAnsari%26aufirst%3DM.%2BA.%26aulast%3DZoheir%26aufirst%3DK.%2BM.%2BA.%26aulast%3DAttia%26aufirst%3DS.%2BM.%26aulast%3DAl-Hosaini%26aufirst%3DK.%2BA.%26aulast%3DAl-Sharary%26aufirst%3DS.%2BD.%26atitle%3DImiquimod-induced%2520psoriasis-like%2520skin%2520inflammation%2520is%2520suppressed%2520by%2520BET%2520bromodomain%2520inhibitor%2520in%2520mice%2520through%2520RORC%252FIL-17A%2520pathway%2520modulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D248%26epage%3D257%26doi%3D10.1016%2Fj.phrs.2015.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, W. G.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and extra-terminal (BET) protein inhibitors suppress chondrocyte differentiation and restrain bone growth</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">26647</span>– <span class="NLM_lpage">26657</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.749697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1074%2Fjbc.M116.749697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27821592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCmurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=26647-26657&author=N.+N.+Niuauthor=R.+Shaoauthor=G.+Yanauthor=W.+G.+Zou&title=Bromodomain+and+extra-terminal+%28BET%29+protein+inhibitors+suppress+chondrocyte+differentiation+and+restrain+bone+growth&doi=10.1074%2Fjbc.M116.749697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain and Extra-terminal (BET) Protein Inhibitors Suppress Chondrocyte Differentiation and Restrain Bone Growth</span></div><div class="casAuthors">Niu, Ningning; Shao, Rui; Yan, Guang; Zou, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">26647-26657</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Small mol. inhibitors for bromodomain and extra-terminal (BET) proteins have recently emerged as potential therapeutic agents in clin. trials for various cancers.  However, to date, it is unknown whether these inhibitors have side effects on bone structures.  Here, we report that inhibition of BET bromodomain proteins may suppress chondrocyte differentiation and restrain bone growth.  We generated a luciferase reporter system using the chondrogenic cell line ATDC5 in which the luciferase gene was driven by the promoter of Col2a1, an elementary collagen of the chondrocyte.  The Col2a1-luciferase ATDC5 system was used for rapidly screening both activators and repressors of human collagen Col2a1 gene expression, and we found that BET bromodomain inhibitors reduce the Col2a1-luciferase.  Consistent with the luciferase assay, BET inhibitors decrease the expression of Col2a1.  Furthermore, we constructed a zebrafish line in which the enhanced green fluorescent protein (EGFP) expression was driven by col2a1 promoter.  The transgenic (col2a1-EGFP) zebrafish line demonstrated that BET inhibitors I-BET151 and (+)-JQ1 may affect EGFP expression in zebrafish.  Furthermore, we found that I-BET151 and (+)-JQ1 may affect chondrocyte differentiation in vitro and inhibit zebrafish growth in vivo.  Mechanistic anal. revealed that BET inhibitors influenced the depletion of RNA polymerase II from the Col2a1 promoter.  Collectively, these results suggest that BET bromodomain inhibition may have side effects on skeletal bone structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr97hnBq7K0OrVg90H21EOLACvtfcHk0lhmZmPY6msPTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCmurnF&md5=d0518ae2f1e7d71ca1e87148848f8707</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.749697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.749697%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DN.%2BN.%26aulast%3DShao%26aufirst%3DR.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DZou%26aufirst%3DW.%2BG.%26atitle%3DBromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520protein%2520inhibitors%2520suppress%2520chondrocyte%2520differentiation%2520and%2520restrain%2520bone%2520growth%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D26647%26epage%3D26657%26doi%3D10.1074%2Fjbc.M116.749697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletar&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lgwoFXfey3VHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romain, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jérôme, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mélanie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean-Marie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miriam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=G.+Romainauthor=T.+J%C3%A9r%C3%B4meauthor=C.+A.+Cl%C3%A9mentauthor=B.+M%C3%A9lanieauthor=B.+Jean-Marieauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=C.+Miriamauthor=P.+Olivier&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0lgwoFXfey3VHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRomain%26aufirst%3DG.%26aulast%3DJ%25C3%25A9r%25C3%25B4me%26aufirst%3DT.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DM%25C3%25A9lanie%26aufirst%3DB.%26aulast%3DJean-Marie%26aufirst%3DB.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DMiriam%26aufirst%3DC.%26aulast%3DOlivier%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coudé, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berrou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffoux, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itzykson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17698</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.4131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.18632%2Foncotarget.4131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=25989842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17698&author=M.-M.+Coud%C3%A9author=T.+Braunauthor=J.+Berrouauthor=M.+Dupontauthor=S.+Bertrandauthor=A.+Masseauthor=E.+Raffouxauthor=R.+Itzyksonauthor=M.+Delordauthor=M.+E.+Riveiro&title=BET+inhibitor+OTX015+targets+BRD2+and+BRD4+and+decreases+c-MYC+in+acute+leukemia+cells&doi=10.18632%2Foncotarget.4131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span></div><div class="casAuthors">Coude Marie-Magdelaine; Braun Thorsten; Berrou Jeannig; Dupont Melanie; Bertrand Sibyl; Masse Aline; Raffoux Emmanuel; Itzykson Raphael; Baruchel Andre; Dombret Herve; Gardin Claude; Coude Marie-Magdelaine; Braun Thorsten; Gardin Claude; Raffoux Emmanuel; Itzykson Raphael; Dombret Herve; Delord Marc; Riveiro Maria E; Herait Patrice; Baruchel Andre</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">17698-712</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The bromodomain (BRD) and extraterminal (BET) proteins including BRD2, BRD3 and BRD4 have been identified as key targets for leukemia maintenance.  A novel oral inhibitor of BRD2/3/4, the thienotriazolodiazepine compound OTX015, suitable for human use, is available.  Here we report its biological effects in AML and ALL cell lines and leukemic samples.  Exposure to OTX015 lead to cell growth inhibition, cell cycle arrest and apoptosis at submicromolar concentrations in acute leukemia cell lines and patient-derived leukemic cells, as described with the canonical JQ1 BET inhibitor.  Treatment with JQ1 and OTX15 induces similar gene expression profiles in sensitive cell lines, including a c-MYC decrease and an HEXIM1 increase.  OTX015 exposure also induced a strong decrease of BRD2, BRD4 and c-MYC and increase of HEXIM1 proteins, while BRD3 expression was unchanged. c-MYC, BRD2, BRD3, BRD4 and HEXIM1 mRNA levels did not correlate however with viability following exposure to OTX015.  Sequential combinations of OTX015 with other epigenetic modifying drugs, panobinostat and azacitidine have a synergic effect on growth of the KASUMI cell line.  Our results indicate that OTX015 and JQ1 have similar biological effects in leukemic cells, supporting OTX015 evaluation in a Phase Ib trial in relapsed/refractory leukemia patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSlHRu3Td8hIa_dlEzl_fBafW6udTcc2ebuo6yEMABIWbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D&md5=0db953a0a24b12ec2918a02d1a2df00b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4131%26sid%3Dliteratum%253Aachs%26aulast%3DCoud%25C3%25A9%26aufirst%3DM.-M.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DBerrou%26aufirst%3DJ.%26aulast%3DDupont%26aufirst%3DM.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DMasse%26aufirst%3DA.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DItzykson%26aufirst%3DR.%26aulast%3DDelord%26aufirst%3DM.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26atitle%3DBET%2520inhibitor%2520OTX015%2520targets%2520BRD2%2520and%2520BRD4%2520and%2520decreases%2520c-MYC%2520in%2520acute%2520leukemia%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17698%26doi%3D10.18632%2Foncotarget.4131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowlati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, S.</span></span> <span> </span><span class="NLM_article-title">Clinically efficacy of the BET bromodomain inhibitor TEN-010 in an open-label substudy with patients with documented NUT-midline carcinoma (NMC)</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">AbstractnrA49</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=AbstractnrA49&author=G.+Shapiroauthor=A.+Dowlatiauthor=P.+LoRussoauthor=J.+Ederauthor=A.+Andersonauthor=K.+Doauthor=M.+Kageyauthor=C.+Sirardauthor=J.+Bradnerauthor=S.+Landau&title=Clinically+efficacy+of+the+BET+bromodomain+inhibitor+TEN-010+in+an+open-label+substudy+with+patients+with+documented+NUT-midline+carcinoma+%28NMC%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShapiro%26aufirst%3DG.%26aulast%3DDowlati%26aufirst%3DA.%26aulast%3DLoRusso%26aufirst%3DP.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DK.%26aulast%3DKagey%26aufirst%3DM.%26aulast%3DSirard%26aufirst%3DC.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DLandau%26aufirst%3DS.%26atitle%3DClinically%2520efficacy%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520TEN-010%2520in%2520an%2520open-label%2520substudy%2520with%2520patients%2520with%2520documented%2520NUT-midline%2520carcinoma%2520%2528NMC%2529%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DAbstractnrA49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koulnis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N. S.</span></span> <span> </span><span class="NLM_article-title">Effect of the BET inhibitor, CPI-0610, alone and in combination with lenalidomide in multiple myeloma</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">4255</span>, <span class="refDoi"> DOI: 10.1182/blood.V126.23.4255.4255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1182%2Fblood.V126.23.4255.4255" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=4255&author=K.+T.+Siuauthor=H.+Edaauthor=L.+Santoauthor=J.+Ramachandranauthor=M.+Koulnisauthor=J.+Mertzauthor=R.+J.+Simsauthor=M.+Cooperauthor=N.+S.+Raje&title=Effect+of+the+BET+inhibitor%2C+CPI-0610%2C+alone+and+in+combination+with+lenalidomide+in+multiple+myeloma&doi=10.1182%2Fblood.V126.23.4255.4255"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1182%2Fblood.V126.23.4255.4255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V126.23.4255.4255%26sid%3Dliteratum%253Aachs%26aulast%3DSiu%26aufirst%3DK.%2BT.%26aulast%3DEda%26aufirst%3DH.%26aulast%3DSanto%26aufirst%3DL.%26aulast%3DRamachandran%26aufirst%3DJ.%26aulast%3DKoulnis%26aufirst%3DM.%26aulast%3DMertz%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DCooper%26aufirst%3DM.%26aulast%3DRaje%26aufirst%3DN.%2BS.%26atitle%3DEffect%2520of%2520the%2520BET%2520inhibitor%252C%2520CPI-0610%252C%2520alone%2520and%2520in%2520combination%2520with%2520lenalidomide%2520in%2520multiple%2520myeloma%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26spage%3D4255%26doi%3D10.1182%2Fblood.V126.23.4255.4255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingard&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lj4y0dOMj_gQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olp, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprague, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovering and exploiting selectivity in BET tandem bromodomain recognition of epigenetic lysine acylation</span>. <i>Faseb J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=2&author=B.+C.+Smithauthor=M.+D.+Olpauthor=D.+J.+Sprague&title=Discovering+and+exploiting+selectivity+in+BET+tandem+bromodomain+recognition+of+epigenetic+lysine+acylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BC.%26aulast%3DOlp%26aufirst%3DM.%2BD.%26aulast%3DSprague%26aufirst%3DD.%2BJ.%26atitle%3DDiscovering%2520and%2520exploiting%2520selectivity%2520in%2520BET%2520tandem%2520bromodomain%2520recognition%2520of%2520epigenetic%2520lysine%2520acylation%26jtitle%3DFaseb%2520J.%26date%3D2018%26volume%3D32%26spage%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A.</span></span> <span> </span><span class="NLM_article-title">RVX 208: a novel BET protein inhibitor, role as an inducer of apo A-I/HDL and beyond</span>. <i>Cardiovasc. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>, <span class="NLM_elocation-id">e12265</span> <span class="refDoi"> DOI: 10.1111/1755-5922.12265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1111%2F1755-5922.12265" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&author=G.+C.+Ghoshauthor=R.+Bhadraauthor=R.+K.+Ghoshauthor=K.+Banerjeeauthor=A.+Gupta&title=RVX+208%3A+a+novel+BET+protein+inhibitor%2C+role+as+an+inducer+of+apo+A-I%2FHDL+and+beyond&doi=10.1111%2F1755-5922.12265"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2F1755-5922.12265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252F1755-5922.12265%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DG.%2BC.%26aulast%3DBhadra%26aufirst%3DR.%26aulast%3DGhosh%26aufirst%3DR.%2BK.%26aulast%3DBanerjee%26aufirst%3DK.%26aulast%3DGupta%26aufirst%3DA.%26atitle%3DRVX%2520208%253A%2520a%2520novel%2520BET%2520protein%2520inhibitor%252C%2520role%2520as%2520an%2520inducer%2520of%2520apo%2520A-I%252FHDL%2520and%2520beyond%26jtitle%3DCardiovasc.%2520Ther.%26date%3D2017%26volume%3D35%26doi%3D10.1111%2F1755-5922.12265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, E. L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. J.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition mechanism of RVX-208 to the second bromodomain of bromo and extraterminal proteins: insight from microsecond molecular dynamics simulations</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">8857</span> <span class="refDoi"> DOI: 10.1038/s41598-017-08909-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1038%2Fs41598-017-08909-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=28821780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A280%3ADC%252BC1cfpslartg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&author=Q.+Q.+Wangauthor=Y.+Liauthor=J.+H.+Xuauthor=Y.+W.+Wangauthor=E.+L.+H.+Leungauthor=L.+Liuauthor=X.+J.+Yao&title=Selective+inhibition+mechanism+of+RVX-208+to+the+second+bromodomain+of+bromo+and+extraterminal+proteins%3A+insight+from+microsecond+molecular+dynamics+simulations&doi=10.1038%2Fs41598-017-08909-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition mechanism of RVX-208 to the second bromodomain of bromo and extraterminal proteins: insight from microsecond molecular dynamics simulations</span></div><div class="casAuthors">Wang Qianqian; Li Ying; Xu Jiahui; Wang Yuwei; Leung Elaine Lai-Han; Liu Liang; Yao Xiaojun</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8857</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">RVX-208 is a recently reported inhibitor of bromo and extraterminal (BET) family proteins (including BRD2-4 and BRDT) with selectivity for the second bromodomain (BD2), currently in phase III clinical trials.  Despite of its promising antitumor activity, due to the conserved folds of the first and second bromodomains (BD1 and BD2), the detailed selectivity mechanism of RVX-208 towards BD2 over BD1 is still unknown.  To elucidate selective inhibition mechanism of RVX-208 to BD2, microsecond molecular dynamics simulations were performed in this study for BRD2-BD1, BRD2-BD2 and BRD4-BD1 with and without RVX-208, respectively.  Binding free energy calculations show that there exists strongest interaction between RVX-208 and BRD2-BD2.  Leu383 and Asn429 are two most important residues of BRD2-BD2 for binding to RVX-208.  Structural network analysis reveals that RVX-208 can shorten the communication path of ZA and BC loops in BRD2-BD2 pocket, making pocket more suitable to accommodate RVX-208.  Additionally, different behaviors of His433 (Asp160 in BRD2-BD1) and Val435 (Ile162 in BRD2-BD1) in BRD2-BD2 are key factors responsible for selective binding of RVX-208 to BRD2-BD2.  The proposed selective inhibition mechanism of RVX-208 to BRD2-BD2 can be helpful for rational design of novel selective inhibitors of the second bromodomain of BET family proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnFJC4cRlg1GlSCSgBhwJFfW6udTcc2eYIxegFjHlYwbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfpslartg%253D%253D&md5=30494be9728cf5d0dc16f68ea4b0c9da</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-08909-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-08909-8%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%2BQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%2BH.%26aulast%3DWang%26aufirst%3DY.%2BW.%26aulast%3DLeung%26aufirst%3DE.%2BL.%2BH.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DX.%2BJ.%26atitle%3DSelective%2520inhibition%2520mechanism%2520of%2520RVX-208%2520to%2520the%2520second%2520bromodomain%2520of%2520bromo%2520and%2520extraterminal%2520proteins%253A%2520insight%2520from%2520microsecond%2520molecular%2520dynamics%2520simulations%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26doi%3D10.1038%2Fs41598-017-08909-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajb, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>– <span class="NLM_lpage">4334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=W.+Ajbauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0liOGOQoQb84_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DAjb%26aufirst%3DW.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, E. L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. J.</span></span> <span> </span><span class="NLM_article-title">Computational study on the selective inhibition mechanism of MS402 to the first and second bromodomains of BRD4</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1002/prot.25611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1002%2Fprot.25611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30260047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFOis7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2019&pages=3-11&author=Q.+Q.+Wangauthor=Y.+Liauthor=J.+H.+Xuauthor=Y.+W.+Wangauthor=D.+F.+Shiauthor=L.+Liuauthor=E.+L.+H.+Leungauthor=X.+J.+Yao&title=Computational+study+on+the+selective+inhibition+mechanism+of+MS402+to+the+first+and+second+bromodomains+of+BRD4&doi=10.1002%2Fprot.25611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Computational study on the selective inhibition mechanism of MS402 to the first and second bromodomains of BRD4</span></div><div class="casAuthors">Wang, Qianqian; Li, Ying; Xu, Jiahui; Wang, Yuwei; Shi, Danfeng; Liu, Liang; Leung, Elaine Lai-Han; Yao, Xiaojun</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-11</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>;
        ISSN:<span class="NLM_cas:issn">1097-0134</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">As a member of the bromodomain and extraterminal domain (BET) family, BRD4 is considered as a potential target for cancer treatment.  However, because of the highly conservation of its two homologous bromodomains (BD1/BD2), selective inhibition of each bromodomain remains a challenge.  MS402 is a domain-selective inhibitor of BRD4-BD1 over BRD4-BD2 reported recently.  Understanding the selectivity mechanism would be very useful for the further design of more potent BD1-selectivity inhibitors.  Mol. dynamics simulation, adaptive biasing force and multiple-walker adaptive biasing force were performed to study the inhibition and domain-selective mechanism of MS402 toward BRD4-BD1 over BRD4-BD2 here.  Results demonstrate BRD4-BD1 binds to MS402 with lower binding free energy than BRD4-BD2.  Residues Gln85, Pro86, Asn140, and Ile146 are crucial for MS402's selectively binding to BRD4-BD1.  MS402 needs to overcome more energy barrier to dissoc. from BD1 than from BD2 pocket.  These findings will be helpful for rational structural modification of existing inhibitors to increase their BD1-selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoenr3BSeSNnLVg90H21EOLACvtfcHk0liOGOQoQb84_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFOis7vI&md5=4e2e93201e34b93a2acd09de604d07d2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fprot.25611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.25611%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%2BQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%2BH.%26aulast%3DWang%26aufirst%3DY.%2BW.%26aulast%3DShi%26aufirst%3DD.%2BF.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLeung%26aufirst%3DE.%2BL.%2BH.%26aulast%3DYao%26aufirst%3DX.%2BJ.%26atitle%3DComputational%2520study%2520on%2520the%2520selective%2520inhibition%2520mechanism%2520of%2520MS402%2520to%2520the%2520first%2520and%2520second%2520bromodomains%2520of%2520BRD4%26jtitle%3DProteins%26date%3D2019%26volume%3D87%26spage%3D3%26epage%3D11%26doi%3D10.1002%2Fprot.25611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of potent diazobenzene inhibitors for the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9251</span>– <span class="NLM_lpage">9264</span>, <span class="refDoi"> DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&author=G.+T.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.+M.+Zhou&title=Structure-guided+design+of+potent+diazobenzene+inhibitors+for+the+BET+bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0liOGOQoQb84_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%2BT.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure-guided%2520design%2520of%2520potent%2520diazobenzene%2520inhibitors%2520for%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzo[<i>cd</i>]indol-2(1<i>H</i>)-ones as potent and specific BET bromodomain inhibitors: structure-based virtual screening, optimization, and biological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1565</span>– <span class="NLM_lpage">1579</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01511</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01511" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs1yqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1565-1579&author=X.+Xueauthor=Y.+Zhangauthor=Z.+Liuauthor=M.+Songauthor=Y.+Xingauthor=Q.+Xiangauthor=Z.+Wangauthor=Z.+Tuauthor=Y.+Zhouauthor=K.+Ding&title=Discovery+of+benzo%5Bcd%5Dindol-2%281H%29-ones+as+potent+and+specific+BET+bromodomain+inhibitors%3A+structure-based+virtual+screening%2C+optimization%2C+and+biological+evaluation&doi=10.1021%2Facs.jmedchem.5b01511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzo[cd]indol-2(1H)-ones as Potent and Specific BET Bromodomain Inhibitors: Structure-Based Virtual Screening, Optimization, and Biological Evaluation</span></div><div class="casAuthors">Xue, Xiaoqian; Zhang, Yan; Liu, Zhaoxuan; Song, Ming; Xing, Yanli; Xiang, Qiuping; Wang, Zhen; Tu, Zhengchao; Zhou, Yulai; Ding, Ke; Xu, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1565-1579</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of inhibitors of bromodomain and extra terminal domain (BET) has achieved great progress, and at least seven inhibitors have progressed into clin. trials for the treatment of cancer or inflammatory diseases.  Here, the authors describe the identification, optimization, and evaluation of benzo[cd]indol-2(1H)-one contg. compds. as a new class of BET bromodomain inhibitors, starting from structure-based virtual screening (SBVS).  Through structure-based optimization, potent compds. were obtained with significantly improved activity.  The two most potent compds. bind to the BRD4 bromodomain, with Kd values of 124 and 137 nM.  Selected compds. exhibited high selectivity over other non-BET subfamily members.  Notably, compd. I demonstrated a reasonable antiproliferation effect on MV4;11 leukemia cells and exhibited a good pharmacokinetic profile with high oral bioavailability (75.8%) and moderate half-life (T1/2 = 3.95 h).  The resulting lead mol. I represents a new, potent, and selective class of BET bromodomain inhibitors for the development of therapeutics to treat cancer and inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl7CO2ESpHObVg90H21EOLACvtfcHk0liIuTw4OMbg1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs1yqsA%253D%253D&md5=d1cd7b961fd4c417c886a1c2274cdacb</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01511%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DXing%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DTu%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520benzo%255Bcd%255Dindol-2%25281H%2529-ones%2520as%2520potent%2520and%2520specific%2520BET%2520bromodomain%2520inhibitors%253A%2520structure-based%2520virtual%2520screening%252C%2520optimization%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1565%26epage%3D1579%26doi%3D10.1021%2Facs.jmedchem.5b01511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of benzo[<i>cd</i>]indol-2(1<i>H</i>)-ones derivatives as BRD4 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.04.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.ejmech.2018.04.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=29730189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptFSjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2018&pages=264-273&author=Y.+Fengauthor=S.+Xiaoauthor=Y.+Chenauthor=H.+Jiangauthor=N.+Liuauthor=C.+Luoauthor=S.+Chenauthor=H.+Chen&title=Design%2C+synthesis+and+biological+evaluation+of+benzo%5Bcd%5Dindol-2%281H%29-ones+derivatives+as+BRD4+inhibitors&doi=10.1016%2Fj.ejmech.2018.04.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of benzo[cd]indol-2(1H)-one derivatives as BRD4 inhibitors</span></div><div class="casAuthors">Feng, Yuxin; Xiao, Senhao; Chen, Yantao; Jiang, Hao; Liu, Na; Luo, Cheng; Chen, Shijie; Chen, Hua</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">264-273</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of benzo[cd]indol-2(1H)-one derivs. were designed by structural optimization based on compd. I and evaluated for their BRD4 (bromodomain-contg. protein 4) inhibitory activity.  The results showed that four compds. are the most potential ones with the IC50 values of 1.02 μM, 1.43 μM, 1.55 μM and 3.02 μM.  According to their co-crystal structures in complex with BRD4_BD1 and the protein thermal shift assays, the binding modes were revealed that the addnl. indirect hydrogen bonds and hydrophobic interactions make such four compds. more active than I against BRD4.  Furthermore, compds. I, 1-((1-ethyl-2-oxo-1,2-dihydrobenzo[cd]indol-6-yl)sulfonyl)pyrrolidine-2-carbohydrazide and 1-((1-ethyl-2-oxo-1,2-dihydrobenzo[cd]indol-6-yl)sulfonyl)pyrrolidine-2-carboxylic acid were chosen to evaluate for their antiproliferative activities on the MLL-AF4-expression acute leukemia cell line (MV4-11), other cancer cell lines and the non-cancer cell lines.  The results showed that these compds. exhibited good and selective inhibitory activities against MV4-11 cells with the IC50 values of 11.67 μM, 5.55 μM, and 11.54 μM, resp., and could act on the cell proliferation by blocking cell cycle at G1 phase.  They could markedly down-regulate the expressions of the c-Myc, Bcl-2 and CDK6 oncogenes in MV4-11 in the qRT-PCR and western blot studies, which further demonstrated that compd. I and its derivs. could serve as a promising therapeutic strategy for MLL leukemia by targeting BRD4_BD1 protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqumq3_lulaVLVg90H21EOLACvtfcHk0liIuTw4OMbg1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptFSjtrw%253D&md5=0652f895d702325d904e64136c44505c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.04.048%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520benzo%255Bcd%255Dindol-2%25281H%2529-ones%2520derivatives%2520as%2520BRD4%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D152%26spage%3D264%26epage%3D273%26doi%3D10.1016%2Fj.ejmech.2018.04.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, K. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L. P.</span></span> <span> </span><span class="NLM_article-title">An efficient approach to the total synthesis of Ammosamide B</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">1050</span>– <span class="NLM_lpage">1053</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201501560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1002%2Fejoc.201501560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1050-1053&author=S.+W.+Yangauthor=C.+M.+Wangauthor=K.+X.+Tangauthor=J.+X.+Wangauthor=L.+P.+Sun&title=An+efficient+approach+to+the+total+synthesis+of+Ammosamide+B&doi=10.1002%2Fejoc.201501560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">An Efficient Approach to the Total Synthesis of Ammosamide B</span></div><div class="casAuthors">Yang, Sheng-Wei; Wang, Chun-Meng; Tang, Kai-Xiang; Wang, Jin-Xin; Sun, Li-Ping</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2016</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1050-1053</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A new approach for the total synthesis of ammosamide B (I) was realized.  The strategy is based on an intermol. Pd-catalyzed N-arylation of 4-chloro-1-methylindoline-2,3-dione (2, II), which enables construction of the key pyrroloquinoline skeleton precursor 6 (VI) within three steps in an overall yield of 80 %.  The ammosamide B total synthesis is achieved in 12 simple transformations by starting from a com. available starting material and proceeds with an overall yield of 23 %.  Addnl., key precursor 6 represents an important intermediate in the synthesis of other pyrroloquinoline-contg. natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJE0yskdRlu7Vg90H21EOLACvtfcHk0liIuTw4OMbg1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVeitb4%253D&md5=21c20b338972b193d9f4255e1befa19e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201501560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201501560%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%2BW.%26aulast%3DWang%26aufirst%3DC.%2BM.%26aulast%3DTang%26aufirst%3DK.%2BX.%26aulast%3DWang%26aufirst%3DJ.%2BX.%26aulast%3DSun%26aufirst%3DL.%2BP.%26atitle%3DAn%2520efficient%2520approach%2520to%2520the%2520total%2520synthesis%2520of%2520Ammosamide%2520B%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2016%26spage%3D1050%26epage%3D1053%26doi%3D10.1002%2Fejoc.201501560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cinelli, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miley, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. B.</span></span> <span> </span><span class="NLM_article-title">Inducible nitric oxide synthase: regulation, structure, and inhibition</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">593</span>, <span class="refDoi"> DOI: 10.1002/med.21599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1002%2Fmed.21599" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=593&author=M.+A.+Cinelliauthor=H.+T.+Doauthor=G.+P.+Mileyauthor=R.+B.+Silverman&title=Inducible+nitric+oxide+synthase%3A+regulation%2C+structure%2C+and+inhibition&doi=10.1002%2Fmed.21599"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fmed.21599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21599%26sid%3Dliteratum%253Aachs%26aulast%3DCinelli%26aufirst%3DM.%2BA.%26aulast%3DDo%26aufirst%3DH.%2BT.%26aulast%3DMiley%26aufirst%3DG.%2BP.%26aulast%3DSilverman%26aufirst%3DR.%2BB.%26atitle%3DInducible%2520nitric%2520oxide%2520synthase%253A%2520regulation%252C%2520structure%252C%2520and%2520inhibition%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2019%26spage%3D593%26doi%3D10.1002%2Fmed.21599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span>; <span class="NLM_string-name">Huq, S.</span>; <span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Targeting cyclooxygenase and nitric oxide pathway cross-talk: a new signal transduction pathway for developing more effective anti-inflammatory drugs</span>. <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume">4</span>,  <span class="NLM_fpage">66</span>  <span class="NLM_lpage">75</span>. <span class="refDoi"> DOI: 10.2174/157436209787048702</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.2174%2F157436209787048702" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=66-75&author=L.+Liu&author=S.+Huq&author=W.+Xu&title=Targeting+cyclooxygenase+and+nitric+oxide+pathway+cross-talk%3A+a+new+signal+transduction+pathway+for+developing+more+effective+anti-inflammatory+drugs&doi=10.2174%2F157436209787048702"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.2174%2F157436209787048702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157436209787048702%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26atitle%3DTargeting%2520cyclooxygenase%2520and%2520nitric%2520oxide%2520pathway%2520cross-talk%253A%2520a%2520new%2520signal%2520transduction%2520pathway%2520for%2520developing%2520more%2520effective%2520anti-inflammatory%2520drugs%26date%3D2009%26volume%3D4%26spage%3D66%26epage%3D75%26doi%3D10.2174%2F157436209787048702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leal, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royce, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pioli, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sporn, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liby, K. T.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitors, JQ1 and I-BET 762, as potential therapies for pancreatic cancer</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>394</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2017.02.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.canlet.2017.02.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=28254412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVWrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2017&pages=76-87&author=A.+S.+Lealauthor=C.+R.+Williamsauthor=D.+B.+Royceauthor=P.+A.+Pioliauthor=M.+B.+Spornauthor=K.+T.+Liby&title=Bromodomain+inhibitors%2C+JQ1+and+I-BET+762%2C+as+potential+therapies+for+pancreatic+cancer&doi=10.1016%2Fj.canlet.2017.02.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibitors, JQ1 and I-BET 762, as potential therapies for pancreatic cancer</span></div><div class="casAuthors">Leal, Ana S.; Williams, Charlotte R.; Royce, Darlene B.; Pioli, Patricia A.; Sporn, Michael B.; Liby, Karen T.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76-87</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Bromodomain inhibitors (JQ1 and I-BET 762) are a new generation of selective, small mol. inhibitors that target BET (bromodomain and extra terminal) proteins.  By impairing their ability to bind to acetylated lysines on histones, bromodomain inhibitors interfere with transcriptional initiation and elongation.  BET proteins regulate several genes responsible for cell cycle, apoptosis and inflammation.  In this study, JQ1 and I-BET 762 decreased c-Myc and p-Erk 1/2 protein levels and inhibited proliferation in pancreatic cancer cells.  The tumor microenvironment is known to play an important role in pancreatic cancer, and these drugs suppressed the prodn. of nitric oxide and a variety of inflammatory cytokines, including IL-6, CCL2, and GM-CSF, in both immune and pancreatic cancer cells in vitro.  Notably, the bromodomain inhibitors also reduced protein levels of p-Erk 1/2 and p-STAT3 in mouse models of pancreatic cancer.  All of these proteins are essential for tumor promotion, progression and metastasis.  In conclusion, the bromodomain inhibitors JQ1 and I-BET 762 targeted and suppressed multiple pathways in pancreatic cancer.  I-BET 762 and a no. of other bromodomain inhibitors are currently being tested in several clin. trials, making them potentially promising drugs for the treatment of pancreatic cancer, an often-fatal disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9YkBNLO0LkrVg90H21EOLACvtfcHk0lhv5BxIGQGe_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVWrurk%253D&md5=818a84ed01487793f657a74ec5c972ef</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.02.021%26sid%3Dliteratum%253Aachs%26aulast%3DLeal%26aufirst%3DA.%2BS.%26aulast%3DWilliams%26aufirst%3DC.%2BR.%26aulast%3DRoyce%26aufirst%3DD.%2BB.%26aulast%3DPioli%26aufirst%3DP.%2BA.%26aulast%3DSporn%26aufirst%3DM.%2BB.%26aulast%3DLiby%26aufirst%3DK.%2BT.%26atitle%3DBromodomain%2520inhibitors%252C%2520JQ1%2520and%2520I-BET%2520762%252C%2520as%2520potential%2520therapies%2520for%2520pancreatic%2520cancer%26jtitle%3DCancer%2520Lett.%26date%3D2017%26volume%3D394%26spage%3D76%26epage%3D87%26doi%3D10.1016%2Fj.canlet.2017.02.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujikawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e83190</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0083190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1371%2Fjournal.pone.0083190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=24391744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=K.+G.+McLureauthor=E.+M.+Gesnerauthor=L.+Tsujikawaauthor=O.+A.+Kharenkoauthor=S.+Attwellauthor=E.+Campeauauthor=S.+Wasiakauthor=A.+Steinauthor=A.+Whiteauthor=E.+Fontano&title=RVX-208%2C+an+inducer+of+ApoA-I+in+humans%2C+is+a+BET+bromodomain+antagonist&doi=10.1371%2Fjournal.pone.0083190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span></div><div class="casAuthors">McLure, Kevin G.; Gesner, Emily M.; Tsujikawa, Laura; Kharenko, Olesya A.; Attwell, Sarah; Campeau, Eric; Wasiak, Sylwia; Stein, Adam; White, Andre; Fontano, Eric; Suto, Robert K.; Wong, Norman C. W.; Wagner, Gregory S.; Hansen, Henrik C.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e83190/1-e83190/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increased synthesis of Apolipoprotein A-I (ApoA-I) and HDL is believed to provide a new approach to treating atherosclerosis through the stimulation of reverse cholesterol transport.  RVX-208 increases the prodn. of ApoA-I in hepatocytes in vitro and in vivo in monkeys and humans, which results in increased HDL-C, but the mol. target was not previously reported.  Using binding assays and X-ray crystallog., we now show that RVX-208 selectively binds to bromodomains of the BET (Bromodomain and Extra Terminal) family, competing for a site bound by the endogenous ligand, acetylated lysine and that this accounts for its pharmacol. activity. siRNA expts. further suggest that induction of ApoA-I mRNA is mediated by BET family member BRD4.  These data indicate that RVX-208 increases ApoA-I prodn. through an epigenetic mechanism and suggests that BET inhibition may be a promising new approach to the treatment of atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHPlAY0x6NLVg90H21EOLACvtfcHk0lhv5BxIGQGe_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D&md5=519ef7ed6d06f97ef8a1359832c11d92</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083190%26sid%3Dliteratum%253Aachs%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DWasiak%26aufirst%3DS.%26aulast%3DStein%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26atitle%3DRVX-208%252C%2520an%2520inducer%2520of%2520ApoA-I%2520in%2520humans%252C%2520is%2520a%2520BET%2520bromodomain%2520antagonist%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0083190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alqahtani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choucair, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashraf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammouda, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alloghbi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senzer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemunaitis, J.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and extra-terminal motif inhibitors: a review of preclinical and clinical advances in cancer therapy</span>. <i>Future Sci. OA</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">FSO372</span>, <span class="refDoi"> DOI: 10.4155/fsoa-2018-0115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.4155%2Ffsoa-2018-0115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30906568" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=FSO372&author=A.+Alqahtaniauthor=K.+Choucairauthor=M.+Ashrafauthor=D.+M.+Hammoudaauthor=A.+Alloghbiauthor=T.+Khanauthor=N.+Senzerauthor=J.+Nemunaitis&title=Bromodomain+and+extra-terminal+motif+inhibitors%3A+a+review+of+preclinical+and+clinical+advances+in+cancer+therapy&doi=10.4155%2Ffsoa-2018-0115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.4155%2Ffsoa-2018-0115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffsoa-2018-0115%26sid%3Dliteratum%253Aachs%26aulast%3DAlqahtani%26aufirst%3DA.%26aulast%3DChoucair%26aufirst%3DK.%26aulast%3DAshraf%26aufirst%3DM.%26aulast%3DHammouda%26aufirst%3DD.%2BM.%26aulast%3DAlloghbi%26aufirst%3DA.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DSenzer%26aufirst%3DN.%26aulast%3DNemunaitis%26aufirst%3DJ.%26atitle%3DBromodomain%2520and%2520extra-terminal%2520motif%2520inhibitors%253A%2520a%2520review%2520of%2520preclinical%2520and%2520clinical%2520advances%2520in%2520cancer%2520therapy%26jtitle%3DFuture%2520Sci.%2520OA%26date%3D2019%26volume%3D5%26spage%3DFSO372%26doi%3D10.4155%2Ffsoa-2018-0115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: a novel epigenetic approach</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1776</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&author=D.+Doroshowauthor=J.+Ederauthor=P.+LoRusso&title=BET+inhibitors%3A+a+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2eajS3kHTRQKj7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DLoRusso%26aufirst%3DP.%26atitle%3DBET%2520inhibitors%253A%2520a%2520novel%2520epigenetic%2520approach%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span> <span> </span><span class="NLM_article-title">Binding pocket-based design, synthesis and biological evaluation of novel selective BRD4-BD1 inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1871</span>– <span class="NLM_lpage">1881</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.03.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1016%2Fj.bmc.2019.03.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30926312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=1871-1881&author=J.+Maauthor=H.+Chenauthor=J.+Yangauthor=Z.+Yuauthor=P.+Huangauthor=H.+Yangauthor=B.+Zhengauthor=R.+Liuauthor=Q.+Liauthor=G.+Huauthor=Z.+Chen&title=Binding+pocket-based+design%2C+synthesis+and+biological+evaluation+of+novel+selective+BRD4-BD1+inhibitors&doi=10.1016%2Fj.bmc.2019.03.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Binding pocket-based design, synthesis and biological evaluation of novel selective BRD4-BD1 inhibitors</span></div><div class="casAuthors">Ma, Junlong; Chen, Heng; Yang, Jie; Yu, Zutao; Huang, Pan; Yang, Haofeng; Zheng, Bifeng; Liu, Rangru; Li, Qianbin; Hu, Gaoyun; Chen, Zhuo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1871-1881</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4), consisting of two tandem bromodomains (BD1 and BD2), is key epigenetic regulator in fibrosis and cancer, which has been reported that BD1 and BD2 have distinct roles in post-translational modification.  But there are few selective inhibitors toward those two domains.  Herein, this study designed and synthesized a series of novel selective BRD4-BD1 inhibitors, using computer-aided drug design (CADD) approach focused on exploring the difference of the binding pockets of BD1 and BD2, and finding the His437 a crucial way to achieve BRD4-BD1 selectivity.  Our results revealed that the compd. 3u(I) is a potent selective BRD4-BD1 inhibitor with IC50 values of 0.56 μM for BD1 but >100 μM for BD2.  I exhibited a broad spectrum of anti-proliferative activity against several human cancer and fibroblastic cell lines, which might be related to its capability of reducing the expression of c-Myc and collagen I.  Furthermore, I could induce apoptosis in A375 cells.  To the contrary, the selective BD2 inhibitor, RVX-208, did not indicate any of these activities.  Our findings highlight that the function of BRD4-BD1 might be predominant in fibrosis and cancer.  And it is rational to further develop novel selective BRD4-BD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp09D43P7pU8LVg90H21EOLACvtfcHk0lh2uKwExbTWeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCit74%253D&md5=8e0166a4372a3ea0919762c2a770aa7a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.03.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.03.037%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DHu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26atitle%3DBinding%2520pocket-based%2520design%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520selective%2520BRD4-BD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D1871%26epage%3D1881%26doi%3D10.1016%2Fj.bmc.2019.03.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seitzberg, J. G.</span>; <span class="NLM_string-name">Kronborg, T. T. A.</span>; <span class="NLM_string-name">Poljak, V.</span>; <span class="NLM_string-name">Friberg, G.</span>; <span class="NLM_string-name">Teuber, L.</span></span> <span> </span><span class="NLM_article-title">Compounds Active towards Bromodomains</span>. U.S. Patent Application <span class="NLM_patent">US20170349607A1</span><span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+G.+Seitzberg&author=T.+T.+A.+Kronborg&author=V.+Poljak&author=G.+Friberg&author=L.+Teuber&title=Compounds+Active+towards+Bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSeitzberg%26aufirst%3DJ.%2BG.%26atitle%3DCompounds%2520Active%2520towards%2520Bromodomains%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W. S.</span></span> <span> </span><span class="NLM_article-title">Design, characterization and comparison of transdermal delivery of colchicine via borneol-chemically-modified and borneol-physically-modified ethosome</span>. <i>Drug Delivery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1080/10717544.2018.1559258</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1080%2F10717544.2018.1559258" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=30744424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVSisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=70-77&author=Y.+J.+Zhangauthor=N.+Zhangauthor=H.+Songauthor=H.+Liauthor=J.+Wenauthor=X.+C.+Tanauthor=W.+S.+Zheng&title=Design%2C+characterization+and+comparison+of+transdermal+delivery+of+colchicine+via+borneol-chemically-modified+and+borneol-physically-modified+ethosome&doi=10.1080%2F10717544.2018.1559258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Design, characterization and comparison of transdermal delivery of colchicine via borneol-chemically-modified and borneol-physically-modified ethosome</span></div><div class="casAuthors">Zhang, Yujia; Zhang, Nan; Song, Hui; Li, He; Wen, Jin; Tan, Xiaochuan; Zheng, Wensheng</div><div class="citationInfo"><span class="NLM_cas:title">Drug Delivery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-77</span>CODEN:
                <span class="NLM_cas:coden">DDELEB</span>;
        ISSN:<span class="NLM_cas:issn">1071-7544</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Gout is a kind of joint disease characterized by the accumulation of monosodium urate (MSU) crystals in the joint and its surrounding tissue, causing persistent hyperuricemia.  Colchicine is the first choice of treatment for acute gout attacks.  Due to strong toxicity of colchicines oral tablets, there are high fluctuations of blood drug concn. and serious irritation of gastrointestinal tract, and hence a transdermal prepn. can help to slow down the blood drug concn., reduce the frequency of drug taking, and improve the patients' compliance of the drug.  The ethosome is a lipid carrier with high concn. of ethanol and has been proved to promote the penetration of drugs into the skin.  Borneol (BO) is an excellent penetration enhancer in Chinese medicine, which can promote the entry of drugs into the skin.  This paper prepd. the borneol-phys.-modified colchicine ethosome (COL-bpES) and used the prepd. borneol-dioleoyl phosphoethanloamine (BO-DOPE) to prep. borneol-chem.-modified colchicine ethosome (COL-bcES).  Compared to the free colchicine aq. soln. (free COL) and normal colchicine ethosome (COL-ES), the borneol-modified colchicine ethosome (COL-bES) demonstrated better drug penetration effect, while the particle size of the COL-bcES was lower than that of the COL-bpES.  Toxicity, in vitro diffusion, pharmacokinetics and pharmacodynamics are superior to those of COL-bpES, providing a better delivery system for the treatment of small mol. inflammatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-I-ofY_fSJrVg90H21EOLACvtfcHk0li0c9fJWCC_Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVSisrg%253D&md5=0c494e7f594b8a6f07f97780acacdada</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1080%2F10717544.2018.1559258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10717544.2018.1559258%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%2BJ.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DX.%2BC.%26aulast%3DZheng%26aufirst%3DW.%2BS.%26atitle%3DDesign%252C%2520characterization%2520and%2520comparison%2520of%2520transdermal%2520delivery%2520of%2520colchicine%2520via%2520borneol-chemically-modified%2520and%2520borneol-physically-modified%2520ethosome%26jtitle%3DDrug%2520Delivery%26date%3D2019%26volume%3D26%26spage%3D70%26epage%3D77%26doi%3D10.1080%2F10717544.2018.1559258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saseen, J. J.</span></span> <span> </span><span class="NLM_article-title">Gouty arthritis: a review of acute management and prevention</span>. <i>Pharmacotherapy</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">906</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1002/phar.1788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1002%2Fphar.1788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=27318031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKrtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=906-922&author=L.+Wilsonauthor=J.+J.+Saseen&title=Gouty+arthritis%3A+a+review+of+acute+management+and+prevention&doi=10.1002%2Fphar.1788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Gouty Arthritis: A Review of Acute Management and Prevention</span></div><div class="casAuthors">Wilson, Liza; Saseen, Joseph J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">906-922</span>CODEN:
                <span class="NLM_cas:coden">PHPYDQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-9114</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Gouty arthritis is one of the most common rheumatic diseases.  The clin. burden of gouty arthritis has historically been well recognized; however, gout is often misdiagnosed and mismanaged.  The prevalence of gout is rising and is likely attributed to several factors including increased incidence of comorbidities, lifestyle factors, and increased use of causative medications.  With the increasing prevalence, there have been several innovations and evidence-based updates related to the diagnosis and management of gout.  Acute gouty arthritis should be treated with nonsteroidal antiinflammatory drugs (NSAIDs), colchicine, or corticosteroids, or a combination of two agents.  Xanthine oxidase inhibitor therapy remains the consensus first-line treatment option for the prevention of recurrent gout.  Add-on therapies that reduce serum urate concn. include traditional uricosuric agents and a novel uric acid reabsorption inhibitor.  Prophylaxis of acute gout with NSAIDs, colchicine, or corticosteroids is universally recommended when initiating any urate-lowering therapy in order to prevent acute gouty arthritis for a period of at least 6 mo.  In this review, we discuss the epidemiol. and risk factors for gouty arthritis and evaluate diagnostic strategies and therapeutic regimens for the management of gout, including a new drug approval.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreo-TcsyQUP7Vg90H21EOLACvtfcHk0li0c9fJWCC_Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKrtrvJ&md5=6e67005f24e6eba7afcf4faec5cfb92f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fphar.1788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fphar.1788%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DL.%26aulast%3DSaseen%26aufirst%3DJ.%2BJ.%26atitle%3DGouty%2520arthritis%253A%2520a%2520review%2520of%2520acute%2520management%2520and%2520prevention%26jtitle%3DPharmacotherapy%26date%3D2016%26volume%3D36%26spage%3D906%26epage%3D922%26doi%3D10.1002%2Fphar.1788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eglen, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reisine, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roby, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouleau, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Illy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossé, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielefeld, M.</span></span> <span> </span><span class="NLM_article-title">The use of AlphaScreen technology in HTS: current status</span>. <i>Curr. Chem. Genomics</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">2</span>, <span class="refDoi"> DOI: 10.2174/1875397300801010002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.2174%2F1875397300801010002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=20161822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjvVWrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2008&pages=2&author=R.+M.+Eglenauthor=T.+Reisineauthor=P.+Robyauthor=N.+Rouleauauthor=C.+Illyauthor=R.+Boss%C3%A9author=M.+Bielefeld&title=The+use+of+AlphaScreen+technology+in+HTS%3A+current+status&doi=10.2174%2F1875397300801010002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The use of AlphaScreen technology in HTS: Current status</span></div><div class="casAuthors">Eglen, Richard M.; Reisine, Terry; Roby, Philippe; Rouleau, Nathalie; Illy, Chantal; Bosse, Roger; Bielefeld, Martina</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2-10</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  AlphaScreen (Amplified Luminescent Proximity Homogeneous Assay Screen) is versatile assay technol. developed to measuring analytes using a homogeneous protocol.  This technol. is an example of a bead-based proximity assay and was developed from a diagnostic assay technol. known as LOCI (Luminescent Oxygen Channeling Assay).  Here, singlet oxygen mols., generated by high energy irradn. of Donor beads, travel over a constrained distance (approx. 200 nm) to Acceptor beads.  This results in excitation of a cascading series of chem. reactions, ultimately causing generation of a chemiluminescent signal.  In the past decade, a wide variety of applications has been reported, ranging from detection of analytes involved in cell signaling, including protein:protein, protein:peptide, protein:small mol. or peptide:peptide interactions.  Numerous homogeneous HTS-optimized assays have been reported using the approach, including generation of second messengers (such as accumulation of cAMP, cyclic GMP, inositol [1, 4, 5] trisphosphate or phosphorylated ERK) from liganded GPCRs or tyrosine kinase receptors, post-translational modification of proteins (such as proteolytic cleavage, phosphorylation, ubiquitination and sumoylation) as well as protein-protein and protein-nucleic acid interactions.  Recently, the basic AlphaScreen technol. was extended in that the chem. of the Acceptor bead was modified such that emitted light is more intense and spectrally defined, thereby markedly reducing interference from biol. fluid matrixes (such as trace hemolysis in serum and plasma).  In this format, referred to as AlphaLISA, it provides an alternative technol. to classical ELISA assays and is suitable for high throughput automated fluid dispensing and detection systems.  Collectively, AlphaScreen and AlphaLISA technologies provide a facile assay platform with which one can quantitate complex cellular processes using simple no-wash microtiter plate based assays.  They provide the means by which large compd. libraries can be screened in a high throughput fashion at a diverse range of therapeutically important targets, often not readily undertaken using other homogeneous assay technologies.  This review assesses the current status of the technol. in drug discovery, in general, and high throughput screening (HTS), in particular.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4coo45wvUbbVg90H21EOLACvtfcHk0ljNsvGYqBY-Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjvVWrtro%253D&md5=4d35f83091184d6eb4d1b76444754707</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.2174%2F1875397300801010002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397300801010002%26sid%3Dliteratum%253Aachs%26aulast%3DEglen%26aufirst%3DR.%2BM.%26aulast%3DReisine%26aufirst%3DT.%26aulast%3DRoby%26aufirst%3DP.%26aulast%3DRouleau%26aufirst%3DN.%26aulast%3DIlly%26aufirst%3DC.%26aulast%3DBoss%25C3%25A9%26aufirst%3DR.%26aulast%3DBielefeld%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520AlphaScreen%2520technology%2520in%2520HTS%253A%2520current%2520status%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2008%26volume%3D1%26spage%3D2%26doi%3D10.2174%2F1875397300801010002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, E.</span>; <span class="NLM_string-name">Wodicka, L.</span>; <span class="NLM_string-name">Ciceri, P.</span>; <span class="NLM_string-name">Pallares, G.</span>; <span class="NLM_string-name">Pickle, E.</span>; <span class="NLM_string-name">Torrey, A.</span>; <span class="NLM_string-name">Floyd, M.</span>; <span class="NLM_string-name">Hunt, J.</span>; <span class="NLM_string-name">Treiber, D.</span></span> <i>BROMOscan a High Throughput, Quantitative Ligand Binding Platform Identifies Best-in-Class Bromodomain Inhibitors from a Screen of Mature Compounds Targeting Other Protein Classes</i>; <span class="NLM_publisher-name">AACR</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=E.+Quinn&author=L.+Wodicka&author=P.+Ciceri&author=G.+Pallares&author=E.+Pickle&author=A.+Torrey&author=M.+Floyd&author=J.+Hunt&author=D.+Treiber&title=BROMOscan+a+High+Throughput%2C+Quantitative+Ligand+Binding+Platform+Identifies+Best-in-Class+Bromodomain+Inhibitors+from+a+Screen+of+Mature+Compounds+Targeting+Other+Protein+Classes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DE.%26btitle%3DBROMOscan%2520a%2520High%2520Throughput%252C%2520Quantitative%2520Ligand%2520Binding%2520Platform%2520Identifies%2520Best-in-Class%2520Bromodomain%2520Inhibitors%2520from%2520a%2520Screen%2520of%2520Mature%2520Compounds%2520Targeting%2520Other%2520Protein%2520Classes%26pub%3DAACR%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holms, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClellan, W. J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Fragment-based, structure-enabled discovery of novel pyridones and pyridone macrocycles as potent bromodomain and extra-terminal domain (BET) family bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3828</span>– <span class="NLM_lpage">3850</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00017</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00017" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsVygu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3828-3850&author=L.+Wangauthor=J.+K.+Prattauthor=T.+Soltwedelauthor=G.+S.+Sheppardauthor=S.+D.+Fidanzeauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=R.+A.+Manteiauthor=J.+H.+Holmsauthor=W.+J.+McClellan&title=Fragment-based%2C+structure-enabled+discovery+of+novel+pyridones+and+pyridone+macrocycles+as+potent+bromodomain+and+extra-terminal+domain+%28BET%29+family+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.7b00017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based, Structure-Enabled Discovery of Novel Pyridones and Pyridone Macrocycles as Potent Bromodomain and Extra-Terminal Domain (BET) Family Bromodomain Inhibitors</span></div><div class="casAuthors">Wang, Le; Pratt, John K.; Soltwedel, Todd; Sheppard, George S.; Fidanze, Steven D.; Liu, Dachun; Hasvold, Lisa A.; Mantei, Robert A.; Holms, James H.; McClellan, William J.; Wendt, Michael D.; Wada, Carol; Frey, Robin; Hansen, T. Matthew; Hubbard, Robert; Park, Chang H.; Li, Leiming; Magoc, Terrance J.; Albert, Daniel H.; Lin, Xiaoyu; Warder, Scott E.; Kovar, Peter; Huang, Xiaoli; Wilcox, Denise; Wang, Rongqi; Rajaraman, Ganesh; Petros, Andrew M.; Hutchins, Charles W.; Panchal, Sanjay C.; Sun, Chaohong; Elmore, Steven W.; Shen, Yu; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3828-3850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the BET family of bromodomain contg. proteins have been identified as potential targets for blocking proliferation in a variety of cancer cell lines.  A 2-dimensional NMR fragment screen for binders to the bromodomains of BRD4 identified a Ph pyridazinone fragment with a weak binding affinity (I, Ki = 160 μM).  SAR investigation of fragment I, aided by X-ray structure-based design, enabled the synthesis of potent pyridone and macrocyclic pyridone inhibitors exhibiting single digit nanomolar potency in both biochem. and cell based assays, e.g. II.  Advanced analogs in these series exhibited high oral exposures in rodent PK studies and demonstrated significant tumor growth inhibition efficacy in mouse flank xenograft models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDaiYdyL4LtrVg90H21EOLACvtfcHk0ljNsvGYqBY-Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsVygu7k%253D&md5=91750e660bafec08b8bcbe6bcd5e818f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00017%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DHolms%26aufirst%3DJ.%2BH.%26aulast%3DMcClellan%26aufirst%3DW.%2BJ.%26atitle%3DFragment-based%252C%2520structure-enabled%2520discovery%2520of%2520novel%2520pyridones%2520and%2520pyridone%2520macrocycles%2520as%2520potent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3828%26epage%3D3850%26doi%3D10.1021%2Facs.jmedchem.7b00017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G. G.</span>; <span class="NLM_string-name">Kontogiannis, L.</span>; <span class="NLM_string-name">Johnson, O.</span>; <span class="NLM_string-name">Powell, H. R.</span>; <span class="NLM_string-name">Leslie, A. G.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>. <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume">67</span>,  <span class="NLM_fpage">271</span>  <span class="NLM_lpage">281</span>. <span class="refDoi"> DOI: 10.1107/S0907444910048675</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battye&author=L.+Kontogiannis&author=O.+Johnson&author=H.+R.+Powell&author=A.+G.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhJY-NitVVS3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span>; <span class="NLM_string-name">Lohkamp, B.</span>; <span class="NLM_string-name">Scott, W. G.</span>; <span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume">66</span>,  <span class="NLM_fpage">486</span>  <span class="NLM_lpage">501</span>. <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsley&author=B.+Lohkamp&author=W.+G.+Scott&author=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span>; <span class="NLM_string-name">Afonine, P. V.</span>; <span class="NLM_string-name">Bunkóczi, G.</span>; <span class="NLM_string-name">Chen, V. B.</span>; <span class="NLM_string-name">Davis, I. W.</span>; <span class="NLM_string-name">Echols, N.</span>; <span class="NLM_string-name">Headd, J. J.</span>; <span class="NLM_string-name">Hung, L.-W.</span>; <span class="NLM_string-name">Kapral, G. J.</span>; <span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span> <span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume">66</span>,  <span class="NLM_fpage">213</span>  <span class="NLM_lpage">221</span>. <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adams&author=P.+V.+Afonine&author=G.+Bunk%C3%B3czi&author=V.+B.+Chen&author=I.+W.+Davis&author=N.+Echols&author=J.+J.+Headd&author=L.-W.+Hung&author=G.+J.+Kapral&author=R.+W.+Grosse-Kunstleve&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF','PDB','3MXF'); return false;">PDB: 3MXF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5uf0','PDB','5uf0'); return false;">PDB: 5uf0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJB','PDB','6JJB'); return false;">PDB: 6JJB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJ3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6JJ3','PDB','6JJ3'); return false;">PDB: 6JJ3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PCE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PCE','PDB','4PCE'); return false;">PDB: 4PCE</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i149"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01010">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_30287"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01010?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01010</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_001.csv">CSV</a>)</p></li><li><p class="inline">Molecular modeling information (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Pharmacokinetics of <b>68</b> in liver microsomes; permeability assay of compounds in PAMPA; NF-κB transcriptional activity of compounds in HUVECs; concentration of compounds in different medium condition; data collection and refinement statistics; antibodies used for immunofluorescence; antibodies used for western blot analysis; X-ray electron density maps of <b>25</b> and <b>60</b>; the <i>K</i><sub>d</sub> curve images of <b>68</b>; the AlphaScreen assay of <b>68</b> for BET families and BRPF1b; uncropped Western blot images; NMR spectra; and HPLC analysis result data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_001.csv">jm9b01010_si_001.csv (6.57 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_002.pdb">jm9b01010_si_002.pdb (180.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01010/suppl_file/jm9b01010_si_003.pdf">jm9b01010_si_003.pdf (20.31 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01010%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-24%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01010" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a59930e6e248d","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
