
<html lang="en"     class="pb-page"  data-request-id="c187b2a5-41ed-42bc-8738-698efc664690"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.5b01323;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-4;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Identification of SNAIL1 Peptide-Based Irreversible Lysine-Specific Demethylase 1-Selective Inactivators" /></meta><meta name="dc.Creator" content="Yukihiro  Itoh" /></meta><meta name="dc.Creator" content="Keisuke  Aihara" /></meta><meta name="dc.Creator" content="Paolo  Mellini" /></meta><meta name="dc.Creator" content="Toshifumi  Tojo" /></meta><meta name="dc.Creator" content="Yosuke  Ota" /></meta><meta name="dc.Creator" content="Hiroki  Tsumoto" /></meta><meta name="dc.Creator" content="Viswas Raja  Solomon" /></meta><meta name="dc.Creator" content="Peng  Zhan" /></meta><meta name="dc.Creator" content="Miki  Suzuki" /></meta><meta name="dc.Creator" content="Daisuke  Ogasawara" /></meta><meta name="dc.Creator" content="Akira  Shigenaga" /></meta><meta name="dc.Creator" content="Tsubasa  Inokuma" /></meta><meta name="dc.Creator" content="Hidehiko  Nakagawa" /></meta><meta name="dc.Creator" content="Naoki  Miyata" /></meta><meta name="dc.Creator" content="Tamio  Mizukami" /></meta><meta name="dc.Creator" content="Akira  Otaka" /></meta><meta name="dc.Creator" content="Takayoshi  Suzuki" /></meta><meta name="dc.Description" content="Inhibition of lysine-specific demethylase 1 (LSD1), a flavin-dependent histone demethylase, has recently emerged as a new strategy for treating cancer and other diseases. LSD1 interacts physically ..." /></meta><meta name="Description" content="Inhibition of lysine-specific demethylase 1 (LSD1), a flavin-dependent histone demethylase, has recently emerged as a new strategy for treating cancer and other diseases. LSD1 interacts physically ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 4, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01323" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01323" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01323" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01323" /></link>
        
    
    

<title>Identification of SNAIL1 Peptide-Based Irreversible Lysine-Specific Demethylase 1-Selective Inactivators | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01323" /></meta><meta property="og:title" content="Identification of SNAIL1 Peptide-Based Irreversible Lysine-Specific Demethylase 1-Selective Inactivators" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0018.jpeg" /></meta><meta property="og:description" content="Inhibition of lysine-specific demethylase 1 (LSD1), a flavin-dependent histone demethylase, has recently emerged as a new strategy for treating cancer and other diseases. LSD1 interacts physically with SNAIL1, a member of the SNAIL/SCRATCH family of transcription factors. This study describes the discovery of SNAIL1 peptide-based inactivators of LSD1. We designed and prepared SNAIL1 peptides bearing a propargyl amine, hydrazine, or phenylcyclopropane moiety. Among them, peptide 3, bearing hydrazine, displayed the most potent LSD1-inhibitory activity in enzyme assays. Kinetic study and mass spectrometric analysis indicated that peptide 3 is a mechanism-based LSD1 inhibitor. Furthermore, peptides 37 and 38, which consist of cell-membrane-permeable oligoarginine conjugated with peptide 3, induced a dose-dependent increase of dimethylated Lys4 of histone H3 in HeLa cells, suggesting that they are likely to exhibit LSD1-inhibitory activity intracellularly. In addition, peptide 37 decreased the viability of HeLa cells. We believe this new approach for targeting LSD1 provides a basis for development of potent selective inhibitors and biological probes for LSD1." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01323"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01323">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01323&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01323&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01323&amp;href=/doi/10.1021/acs.jmedchem.5b01323" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1531-1544</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01267" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01493" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Identification of SNAIL1 Peptide-Based Irreversible Lysine-Specific Demethylase 1-Selective Inactivators</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yukihiro++Itoh">Yukihiro Itoh</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Keisuke++Aihara">Keisuke Aihara</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paolo++Mellini">Paolo Mellini</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Toshifumi++Tojo">Toshifumi Tojo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yosuke++Ota">Yosuke Ota</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hiroki++Tsumoto">Hiroki Tsumoto</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Viswas+Raja++Solomon">Viswas Raja Solomon</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peng++Zhan">Peng Zhan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Miki++Suzuki">Miki Suzuki</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daisuke++Ogasawara">Daisuke Ogasawara</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Akira++Shigenaga">Akira Shigenaga</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tsubasa++Inokuma">Tsubasa Inokuma</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hidehiko++Nakagawa">Hidehiko Nakagawa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Naoki++Miyata">Naoki Miyata</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tamio++Mizukami">Tamio Mizukami</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Akira++Otaka">Akira Otaka</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Takayoshi++Suzuki">Takayoshi Suzuki</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">â </span> <span class="aff-text">Graduate School of Medical Science, Kyoto Prefectural University of Medicine, 1-5 Shimogamohangi-Cho, Sakyo-Ku, Kyoto, 606-0823, Japan</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">â¡</span> <span class="aff-text">Institute of Biomedical Sciences and Graduate School of Pharmaceutical Sciences, Tokushima University, Shomachi, Tokushima 770-8505, Japan</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">Â§</span> <span class="aff-text">Graduate School of Pharmaceutical Sciences, Nagoya City University, 3-1 Tanabe-dori, Mizuho-ku, Nagoya, Aichi 467-8603, Japan</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">â¥</span> <span class="aff-text">Research Team for Mechanism of Aging Tokyo Metropolitan Institute of Gerontology, 35-2 Sakae-Cho, Itabashi-ku, Tokyo 173-0015, Japan</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">â¥</span> <span class="aff-text">Graduate School of Bio-Science, Nagahama Institute of Bio-Science and Technology, 1266 Tamura-cho, Nagahama, Shiga 526-0829, Japan</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">CREST, Japan Science and Technology Agency (JST), 4-1-8 Honcho Kawaguchi, Saitama 332-0012, Japan</span></div><div class="corresp-info"><strong>*</strong>E-mail (A.O.): <a href="/cdn-cgi/l/email-protection#b0d1dfc4d1dbd1f0c4dfdbc5c3d8d9ddd19dc59ed1d39edac0"><span class="__cf_email__" data-cfemail="0f6e607b6e646e4f7b60647a7c6766626e227a216e6c21657f">[emailÂ protected]</span></a></div><div class="corresp-info"><strong>*</strong>E-mail (T.S.): <a href="/cdn-cgi/l/email-protection#1162646b647a7865517a7e657e3f7a61643c7c3f70723f7b61"><span class="__cf_email__" data-cfemail="097a7c737c62607d4962667d662762797c246427686a276379">[emailÂ protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01323&amp;href=/doi/10.1021%2Facs.jmedchem.5b01323" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1531â1544</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 23, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 August 2015</li><li><span class="item_label"><b>Published</b> online</span>4 January 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 February 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01323" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01323</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1531%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYukihiro%2BItoh%252C%2BKeisuke%2BAihara%252C%2BPaolo%2BMellini%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D4%26contentID%3Dacs.jmedchem.5b01323%26title%3DIdentification%2Bof%2BSNAIL1%2BPeptide-Based%2BIrreversible%2BLysine-Specific%2BDemethylase%2B1-Selective%2BInactivators%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1544%26publicationDate%3DFebruary%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01323"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2050</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">20</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01323" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Identification of SNAIL1 Peptide-Based Irreversible Lysine-Specific Demethylase 1-Selective Inactivators&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yukihiro&quot;,&quot;last_name&quot;:&quot;Itoh&quot;},{&quot;first_name&quot;:&quot;Keisuke&quot;,&quot;last_name&quot;:&quot;Aihara&quot;},{&quot;first_name&quot;:&quot;Paolo&quot;,&quot;last_name&quot;:&quot;Mellini&quot;},{&quot;first_name&quot;:&quot;Toshifumi&quot;,&quot;last_name&quot;:&quot;Tojo&quot;},{&quot;first_name&quot;:&quot;Yosuke&quot;,&quot;last_name&quot;:&quot;Ota&quot;},{&quot;first_name&quot;:&quot;Hiroki&quot;,&quot;last_name&quot;:&quot;Tsumoto&quot;},{&quot;first_name&quot;:&quot;Viswas&quot;,&quot;last_name&quot;:&quot;Raja Solomon&quot;},{&quot;first_name&quot;:&quot;Peng&quot;,&quot;last_name&quot;:&quot;Zhan&quot;},{&quot;first_name&quot;:&quot;Miki&quot;,&quot;last_name&quot;:&quot;Suzuki&quot;},{&quot;first_name&quot;:&quot;Daisuke&quot;,&quot;last_name&quot;:&quot;Ogasawara&quot;},{&quot;first_name&quot;:&quot;Akira&quot;,&quot;last_name&quot;:&quot;Shigenaga&quot;},{&quot;first_name&quot;:&quot;Tsubasa&quot;,&quot;last_name&quot;:&quot;Inokuma&quot;},{&quot;first_name&quot;:&quot;Hidehiko&quot;,&quot;last_name&quot;:&quot;Nakagawa&quot;},{&quot;first_name&quot;:&quot;Naoki&quot;,&quot;last_name&quot;:&quot;Miyata&quot;},{&quot;first_name&quot;:&quot;Tamio&quot;,&quot;last_name&quot;:&quot;Mizukami&quot;},{&quot;first_name&quot;:&quot;Akira&quot;,&quot;last_name&quot;:&quot;Otaka&quot;},{&quot;first_name&quot;:&quot;Takayoshi&quot;,&quot;last_name&quot;:&quot;Suzuki&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;1531-1544&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01323&quot;},&quot;abstract&quot;:&quot;Inhibition of lysine-specific demethylase 1 (LSD1), a flavin-dependent histone demethylase, has recently emerged as a new strategy for treating cancer and other diseases. LSD1 interacts physically with SNAIL1, a member of the SNAIL/SCRATCH family of transcription factors. This study describes the discovery of SNAIL1 peptide-based inactivators of LSD1. We designed and prepared SNAIL1 peptides bearing a propargyl amine, hydrazine, or phenylcyclopropane moiety. Among them, peptide 3, bearing hydrazine, displayed the most potent LSD1-inhibitory activity in enzyme assays. Kinetic study and mass spectrometric analysis indicated that peptide 3 is a mechanism-based LSD1 inhibitor. Furthermore, peptides 37 and 38, which consist of cell-membrane-permeable oligoarginine conjugated with peptide 3, induced a dose-dependent increase of dimethylated Lys4 of histone H3 in HeLa cells, suggesting that they are likely to exhibit LSD1-inhibitory activity intracellularly. In addition, peptide 37 decreased the viability of HeLa ce&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01323&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01323" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01323&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01323" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01323&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01323" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01323&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01323&amp;href=/doi/10.1021/acs.jmedchem.5b01323" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01323" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01323" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01323%26sid%3Dliteratum%253Aachs%26pmid%3D26700437%26genre%3Darticle%26aulast%3DItoh%26date%3D2016%26atitle%3DIdentification%2Bof%2BSNAIL1%2BPeptide-Based%2BIrreversible%2BLysine-Specific%2BDemethylase%2B1-Selective%2BInactivators%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D4%26spage%3D1531%26epage%3D1544%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291485" title="Solid phase synthesis">Solid phase synthesis</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/jmcmar.2016.59.issue-4/20160225/jmcmar.2016.59.issue-4.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inhibition of lysine-specific demethylase 1 (LSD1), a flavin-dependent histone demethylase, has recently emerged as a new strategy for treating cancer and other diseases. LSD1 interacts physically with SNAIL1, a member of the SNAIL/SCRATCH family of transcription factors. This study describes the discovery of SNAIL1 peptide-based inactivators of LSD1. We designed and prepared SNAIL1 peptides bearing a propargyl amine, hydrazine, or phenylcyclopropane moiety. Among them, peptide <b>3</b>, bearing hydrazine, displayed the most potent LSD1-inhibitory activity in enzyme assays. Kinetic study and mass spectrometric analysis indicated that peptide <b>3</b> is a mechanism-based LSD1 inhibitor. Furthermore, peptides <b>37</b> and <b>38</b>, which consist of cell-membrane-permeable oligoarginine conjugated with peptide <b>3</b>, induced a dose-dependent increase of dimethylated Lys4 of histone H3 in HeLa cells, suggesting that they are likely to exhibit LSD1-inhibitory activity intracellularly. In addition, peptide <b>37</b> decreased the viability of HeLa cells. We believe this new approach for targeting LSD1 provides a basis for development of potent selective inhibitors and biological probes for LSD1.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/59/4"><issue-title>Epigenetics</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82673" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82673" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Chromatin modification, including DNA methylation and histone modifications such as acetylation and methylation, plays a pivotal role in transcriptional regulation.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> These modifications epigenetically control biological functions such as cellular differentiation and developmental changes of organisms. In addition, aberrations of chromatin modification are closely associated with several diseases, such as cancer and neurodegenerative disorders. Therefore, modulators that can regulate chromatin modification have attracted a great deal of attention as therapeutic agents.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Among post-translation histone modifications, lysine methylation is one of the most widely studied modifications.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Methylation of histone lysine residues is reversible and is regulated by histone lysine methyltransferases and histone lysine demethylases (KDMs).<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> Lysine-specific demethylase 1 (LSD1, also known as KDM1A) is a demethylase that removes methyl groups from mono- and dimethylated Lys4 of histone H3 (H3K4me1/2) through flavin adenine dinucleotide (FAD)-dependent enzymatic oxidation.<a onclick="showRef(event, 'cit4a'); return false;" href="javascript:void(0);" class="ref cit4a">(4a)</a> LSD1 binds with several transcription factors and regulates the expression of a number of genes, such as <i>synapsin</i> and <i>muscarinic acetylcholine receptor 4</i>.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> LSD1 has also been reported to be associated with several disease states. For example, it is involved in proliferation of neuroblastoma cells and leukemia cells.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> It is also associated with latent infection of Î±-herpesvirus and globin disorder.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Therefore, LSD1 inhibitors are of interest as chemical tools for studying the functions of LSD1 and as candidate therapeutic agents targeting LSD1.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">LSD1 has been reported to interact with SNAIL1, a member of the SNAIL/SCRATCH family of transcription factors, and to play a role in the expression of cancer-associated SNAIL1 target genes.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Further, inhibition of the interaction between LSD1 and SNAIL1 blocks cancer cell invasion.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Therefore, inhibitors of the LSD1/SNAIL1 interaction appear to be novel anticancer agent candidates. Indeed, Tortorici et al. recently reported SNAIL1-derived peptides as reversible LSD1 inhibitors.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> On the other hand, we and other groups revealed that small molecules containing a propargyl amine, hydrazine, or phenylcyclopropylamine (PCPA) moiety (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) irreversibly inhibit LSD1.<a onclick="showRef(event, 'ref8 ref12'); return false;" href="javascript:void(0);" class="ref ref8 ref12">(8, 12)</a> Such mechanism-based irreversible inhibitors are expected to maintain the pharmacological effect because of a long target residence time.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In light of these findings, we initiated a search for SNAIL1 peptide-based irreversible LSD1 inactivators. Herein we report the design, synthesis, LSD1-inhibitory activity, and cellular activity of a series of SNAIL1-derived peptides bearing various functional groups.</div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Examples of Reported Small-Molecular LSD1 Inactivators</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81734" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81734" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The routes used for the synthesis of peptides <b>1</b>â<b>5</b> (<a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>), which were prepared for this study, are shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>â<a class="ref internalNav" href="#sch3" aria-label="3">3</a>. <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> shows the preparation of peptides <b>1</b> and <b>2</b> with a propargylamine moiety. Boc-Asp(OBn)-OH (<b>6</b>) was treated with <i>tert</i>-BuOH in the presence of EDCI and DMAP to give Boc-Asp(OBn)-O<sup><i>t</i></sup>Bu (<b>7</b>). The benzyl group of compound <b>7</b> was removed by hydrogenolysis to give Boc-Asp-O<sup><i>t</i></sup>Bu (<b>8</b>). Conversion of the carboxylic acid of <b>8</b> to mixed anhydride by using ClCOOEt and subsequent reduction with NaBH<sub>4</sub> yielded alcohol <b>9</b>. Propargylamines <b>11</b> and <b>12</b> were synthesized from the alcohol <b>9</b> via the mesylate <b>10</b>.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Removal of the Boc group and <i>tert</i>-butyl group of <b>11</b> and <b>12</b> using hydrochloric acid afforded amino acids <b>13</b> and <b>14</b>. Treatment of the amino acids <b>13</b> and <b>14</b> with Fmoc-OSu in the presence of NaHCO<sub>3</sub> gave the Fmoc-protected amino acids <b>15</b> and <b>16</b>, from which the propargylated peptides <b>1</b> and <b>2</b> were prepared using Fmoc-based solid-phase peptide synthesis.</div><figure id="cht2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Structures of SNAIL1-Based Peptides <b>1</b>â<b>5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0013.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Peptides <b>1</b> and <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <sup><i>t</i></sup>BuOH, EDCI, DMAP, CH<sub>2</sub>Cl<sub>2</sub>, room temp, 93%; (b) H<sub>2</sub>, Pd/C, MeOH, room temp, 97%; (c) ClCOOEt, Et<sub>3</sub>N, THF, â10 Â°C; (d) NaBH<sub>4</sub>, THF, H<sub>2</sub>O, 0 Â°C to room temp, 74% (two steps); (e) MsCl, Et<sub>3</sub>N, DMAP, CH<sub>2</sub>Cl<sub>2</sub>, â20 Â°C, 58%; (f) <i>N</i>-benzylpropargylamine or <i>N</i>-methylpropargylamine, Cs<sub>2</sub>CO<sub>3</sub> for <b>11</b> or Et<sub>3</sub>N, DMAP for <b>12</b>, DMF, 65 Â°C for <b>11</b> or 75 Â°C for <b>12</b>, 19% for <b>11</b>, 23% for <b>12</b>; (g) HCl, 1,4-dioxane, CH<sub>2</sub>Cl<sub>2</sub>, 0 Â°C to room temp, 88% for <b>13</b>, 92% for <b>14</b>; (h) Fmoc-OSu, NaHCO<sub>3</sub>, 1,4-dioxane, H<sub>2</sub>O, 0 Â°C to room temp, 20% for <b>15</b>, 19% for <b>16</b>; (i) Fmoc-based solid-phase peptide synthesis, 17% for <b>1</b>, 16% for <b>2</b>.</p></p></figure><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0014.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Peptides <b>3</b> and <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) BnBr, K<sub>2</sub>CO<sub>3</sub>, NaOH, H<sub>2</sub>O, reflux, 29%; (b) DIBAL, THF, â10 to 0 Â°C, 92%; (c) DMSO, (COCl)<sub>2</sub>, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, â78 Â°C, 85%; (d) R<sup>2</sup>R<sup>3</sup>NNH<sub>2</sub>, THF, room temp, 94% for <b>21</b>, 93% for <b>22</b>; (e) NaBH<sub>3</sub>CN, EtOH, AcOH, room temp, 69% for <b>23</b>, 92% for <b>24</b>; (f) (Boc)<sub>2</sub>O, DMAP, CH<sub>2</sub>Cl<sub>2</sub>, room temp, 92% for <b>25</b>, 40% for <b>26</b>; (g) Pd black, HCOOH, MeOH, â 40 Â°C to room temp; (h) Fmoc-OSu, NaHCO<sub>3</sub>, 1,4-dioxane, H<sub>2</sub>O, 0 Â°C to room temp, 89% for <b>29</b> (2 steps), 82% for <b>30</b> (2 steps); (i) Fmoc-based solid-phase peptide synthesis, 16% for <b>3</b>, 29% for <b>4</b>.</p></p></figure><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0015.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Peptide <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) MsCl, Et<sub>3</sub>N, DMAP, CH<sub>2</sub>Cl<sub>2</sub>, â20 Â°C, 66%; (b) PCPA, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 75 Â°C, 75%; (c) HCl, dioxane, 0 Â°C to room temp, 78%; (d) FmocCl, NaHCO<sub>3</sub>, 1,4-dioxane, H<sub>2</sub>O, 0 Â°C to room temp, 14%; (e) (Boc)<sub>2</sub>O, NaHCO<sub>3</sub>, 1,4-dioxane, H<sub>2</sub>O, 0 Â°C to room temp, 28%; (f) Fmoc-based solid-phase peptide synthesis, 14%.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> shows the synthesis of peptides <b>3</b> and <b>4</b>. Benzylation of aspartic acid <b>17</b> afforded Bn<sub>2</sub>-Asp(OBn)-OBn (<b>18</b>). Selective reduction of the less-hindered benzyl ester of <b>18</b> using DIBAL yielded alcohol <b>19</b>. The alcohol <b>19</b> was converted to aldehyde <b>20</b> by Swern oxidation. Reductive amino alkylation of BocNHNH<sub>2</sub> or BocNMeNH<sub>2</sub> with the aldehyde <b>20</b> in the presence of sodium cyanoborohydride gave <i>N</i>-Boc hydrazine compounds <b>23</b> and <b>24</b>. Fmoc-protected unnatural amino acids <b>29</b> and <b>30</b> were prepared from compounds <b>23</b> and <b>24</b> in three steps: Boc protection using (Boc)<sub>2</sub>O, removal of the benzyl groups by hydrogenolysis, and Fmoc protection using Fmoc-OSu. The Fmoc-protected amino acids <b>29</b> and <b>30</b> were incorporated into peptides <b>3</b> and <b>4</b>, respectively, by means of Fmoc-based solid-phase peptide synthesis.</div><div class="NLM_p">Peptide <b>5</b> was prepared from alcohol <b>31</b><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> as outlined in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The alcohol <b>31</b> was reacted with mesyl chloride to obtain mesylate <b>32</b>. Treatment of the mesylate <b>32</b> with PCPA in the presence of Cs<sub>2</sub>CO<sub>3</sub> gave compound <b>33</b>. Deprotection of the Boc and <i>tert</i>-butyl groups of <b>33</b> afforded amino acid <b>34</b>. The amino acid <b>34</b> was converted to Fmoc-protected PCPA-conjugated amino acid <b>36</b> by Fmoc protection of <b>34</b> and Boc protection of <b>35</b>. The PCPA-conjugated SNAIL1-based peptide <b>5</b> was prepared from <b>36</b> via standard Fmoc-based solid-phase peptide synthesis.</div><div class="NLM_p">Oligoarginine-conjugated peptides <b>37</b>â<b>40</b> (<a class="ref internalNav" href="#cht3" aria-label="Chart 3">Chart 3</a>) were also synthesized by Fmoc-based solid-phase peptide synthesis using a lysine derivative with ivDde protection (see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>).</div><figure id="cht3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Chart 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 3. Structures of Oligoarginine-Conjugated Peptides <b>37</b>â<b>40</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=cht3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14428" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14428" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We designed SNAIL1 peptide-based LSD1 inactivators based on the crystal structure of LSD1 complexed with a 20-amino-acid SNAIL1 <i>N</i>-terminal peptide<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and based on the work of Cole and co-workers on histone H3 peptide-based LSD1 inactivators containing a propargyl amine, a hydrazine, or a cyclopropylamine moiety.<a onclick="showRef(event, 'cit12i'); return false;" href="javascript:void(0);" class="ref cit12i">(12i)</a> The X-ray crystal structure of the LSD1/SNAIL1 peptide complex revealed that Phe 4 of the SNAIL1 peptide is positioned in front of the flavin in the active site of LSD1 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). On the basis of this structure, we designed SNAIL1-based peptides <b>1</b>â<b>5</b> (<a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>), in which the phenylalanine residue is replaced with non-natural amino acid residues bearing propargyl amine (<b>1</b> and <b>2</b>), hydrazine (<b>3</b> and <b>4</b>), or PCPA (<b>5</b>). We expected that the designed SNAIL1 peptides <b>1</b>â<b>5</b> would selectively bind LSD1 over LSD2, because it has been reported that the affinity of LSD2 for a SNAIL1 peptide is at least 10 times lower than that of LSD1.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In addition, these SNAIL1-based peptides were expected to be recognized effectively by LSD1 (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S1">Supporting Information Figure S1</a>) and to inactivate LSD1 irreversibly through amine oxidation by FAD and subsequent covalent bond formation with FAD in the active site of LSD1 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). Specifically, peptides <b>1</b> and <b>2</b> bearing propargyl amine were expected to be oxidized to an Î±,Î²-unsaturated iminium cation species that would be nucleophilically attacked by <i>N</i>5 of the reduced FAD, leading to LSD1 inactivation (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The expected mechanism of LSD1 inactivation by peptides <b>3</b> and <b>4</b> bearing hydrazine is shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b. The first oxidation of the hydrazine of peptide <b>3</b> by FAD would yield the diazene, and the second oxidation of the diazene would produce the primary diazonium species, which would inactivate LSD1 through S<sub>N</sub>2 reaction between the diazonium cation and FADH<sup>â</sup>.<a onclick="showRef(event, 'cit12i'); return false;" href="javascript:void(0);" class="ref cit12i">(12i)</a> On the other hand, peptide <b>4</b> with a monomethylhydrazine group would be converted to peptide <b>3</b> through enzymatic demethylation by LSD1. Then, it would inhibit LSD1 similarly to peptide <b>3</b>. Introduction of the methyl group into the hydrazine moiety of peptide <b>3</b> was expected to enhance the affinity for LSD1, because methylated lysine is a substrate of LSD1. The expected mechanism of LSD1 inactivation by peptide <b>5</b> involves formation of a PCPA-FAD adduct, following the oxidation by FAD and cyclopropyl ring opening (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>c).<a onclick="showRef(event, 'cit12f'); return false;" href="javascript:void(0);" class="ref cit12f">(12f)</a></div><figure id="fig1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. X-ray crystal structure of LSD1 complex with a SNAIL1 peptide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y48">2Y48</a>). FAD and the SNAIL1 peptide are displayed as tube and ball-and-stick models, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Plausible mechanism of inactivation of LSD1 by <i>N</i>-propargyl lysine peptide <b>1</b> or <b>2</b>. (b) Plausible mechanism of inactivation of LSD1 by peptides <b>3</b> and <b>4</b> bearing a hydrazine moiety. (c) Plausible mechanism of inactivation of LSD1 by peptide <b>5</b> bearing a PCPA moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The SNAIL1-based peptides <b>1</b>â<b>5</b> were prepared (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>â<a class="ref internalNav" href="#sch3" aria-label="3">3</a>) and evaluated for LSD1-inhibitory activity using a peroxidase-coupled assay as described before.<a onclick="showRef(event, 'cit12d cit12e cit12f'); return false;" href="javascript:void(0);" class="ref cit12d cit12e cit12f">(12d-12f)</a> The results are summarized in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> as IC<sub>50</sub> values. Among the tested peptides, peptide <b>3</b> with a hydrazine moiety was found to be the most potent LSD1 inactivator (<b>3</b>: IC<sub>50</sub> = 0.44 Î¼M). The activity of peptide <b>4</b> with a monomethylhydrazine group was moderate (<b>4</b>: IC<sub>50</sub> = 8.4 Î¼M). We also examined the inhibitory activity of peptides <b>3</b> and <b>4</b> toward LSD2, which is a paralogue of LSD1 monoamine oxidase (MAO) A/B, which are other FAD-dependent oxidases, and Jumonji AT-rich interactive domain 1A (JARID1A), which is one of the histone H3K4me3/me2-specific demethylases.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> As shown in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, peptides <b>3</b> and <b>4</b> were inactive at concentrations up to 50 Î¼M in LSD2-inhibitory assays. These results are consistent with the previous report that the affinity of LSD2 for SNAIL peptide is lower than that of LSD1.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Furthermore, the two peptides were also almost inactive against MAOs and JARID1A. These results demonstrated that peptides <b>3</b> and <b>4</b> are inhibitors with high selectivity for LSD1 over LSD2, MAO A/B, and JARID1A.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. In Vitro LSD1-, LSD2-, MAO-A-, MAO-B-, and JARID1A-Inhibitory Activities of SNAIL1-Based Peptides <b>1</b>â<b>5</b>.<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0016.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0017.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Values are means of at least two experiments.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">PCPA (IC<sub>50</sub> = 41 Î¼M for LSD1; IC<sub>50</sub> = 4.2 Î¼M for MAO A; IC<sub>50</sub> = 3.5 Î¼M for MAO B) and GSK2879552 (IC<sub>50</sub> = 0.61 Î¼M for LSD1) were used as positive controls.</p></div><div class="footnote" id="t1fn3"><sup>Table c</sup><p class="last">N.D. = Not determined.</p></div></div><div></div></div><div class="NLM_p">Because peptide <b>3</b>, which is the most potent LSD1 inhibitor, was designed to inactivate LSD1 irreversibly as shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b, we next investigated the mechanism of LSD1 inhibition by peptide <b>3</b>. We initially examined whether the inhibition is time-dependent. The time course of product formation was monitored in the absence or presence of peptide <b>3</b>. As shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>, peptide <b>3</b> was found to be a time-dependent inhibitor of LSD1.</div><figure id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Time- and concentration-dependent inhibition of LSD1 by peptide <b>3</b>. Steady-state progress curves obtained for inactivation of LSD1 by 0 (â ), 0.1 (â) 0.2 (â²), 0.4 ( Ã ), 0.8 (*), 1.6 (+), 3.2 (â), and 6.4 (â) Î¼M <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To gain further insight into the mechanism of LSD1 inhibition by peptide <b>3</b>, mass spectrometric analysis of a mixture of LSD1 incubated with FAD and peptide <b>3</b> was performed. If peptide <b>3</b> reacts with FAD as expected, peptide <b>3</b>-FAD adduct (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b) should be generated. As depicted in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a, a significant peak at <i>m</i>/<i>z</i> 3123.4 was observed. This peak corresponds to the predicted molecular weight of peptide <b>3</b>-FAD adduct. The peak was dependent on the presence of LSD1; that is, it was not detected in the absence of LSD1 (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b). In addition, an excess of H3K4me2 peptide, a substrate of LSD1, inhibited the generation of the <b>3</b>-FAD adduct (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S2">Supporting Information Figure S2</a>). The data from the mass spectrometric analysis support the idea that peptide <b>3</b> oxidized by FAD is nucleophilically attacked by FAD<sup>â</sup> in the active site of LSD1 to afford peptide <b>3</b>-FAD conjugate, and this reaction results in inhibition of LSD1. Kinetic analysis and mass analysis results indicate that peptide <b>3</b> is an irreversible LSD1 inactivator, as we had hoped.</div><figure id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mass spectrometric detection of <b>3</b>-FAD adduct. Mass spectra (positive mode) were obtained from (a) LSD1-<b>3</b> mixture, (b) FAD-<b>3</b> mixture in the absence of LSD1, (c) LSD1-<b>4</b> mixture, and (d) FAD-<b>4</b> mixture in the absence of LSD1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Mass analysis of peptide <b>4</b> bearing a methylated hydrazine moiety was also performed. As shown in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S3">Supporting Information Figure S3</a>, the peak corresponding to peptide <b>3</b>, the demethylated form of peptide <b>4</b>, was observed dependently on the presence of LSD1. In addition, the peak of peptide <b>3</b>-FAD adduct was detected dependently on LSD1, though the signal intensity was weak (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>d). These results suggest that peptide <b>4</b> inhibits LSD1 through the generation of peptide <b>3</b> by demethylation of a part of peptide <b>4</b>, followed by formation of peptide <b>3</b>-FAD adduct. However, the LSD1-inhibitory activity of peptide <b>4</b> was less than that of peptide <b>3</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). These results indicate that the demethylation of methylated hydrazine in peptide <b>4</b> is unfavorable as compared with the oxidation of hydrazine in peptide <b>3</b>.</div><div class="NLM_p">Although peptide compounds generally show poor cell permeability because of their high polarity, previous studies have established that oligomers of arginine covalently attached to a variety of molecules can efficiently cross the cell membrane.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> For the application of SNAIL1-based peptide <b>3</b>, the most potent LSD1 inactivator in this study (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), to cell-based assays, we therefore designed and synthesized oligoarginine-conjugated derivatives of peptide <b>3</b>. Oligoarginine-conjugated peptides <b>37</b>â<b>39</b> (<a class="ref internalNav" href="#cht3" aria-label="Chart 3">Chart 3</a>) were designed based on the X-ray crystal structure of LSD1/SNAIL1 peptide complex (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). As shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, the side chain of Lys 8 of the SNAIL1 peptide is located outside the active pocket of LSD1. Thus, peptides <b>37</b> and <b>38</b>, in which oligoarginine is conjugated with the side chain of Lys 8 of peptide <b>3</b>, were expected to show little or no decrease of LSD1-inhibitory activity. In addition, we designed peptide <b>39</b> in which oligoarginine is conjugated with the side chain of Lys 15 of peptide <b>3</b>. It was reported that residues 10â20 of the 20-amino-acid SNAIL1 <i>N</i>-terminal peptide could not be identified in terms of electron density in the X-ray crystal structure analysis of the LSD1/SNAIL1 peptide complex.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> This suggest that the residues 10â20 of the SNAIL1 peptide lack an ordered conformation and Lys 15 of the SNAIL1 peptide is likely to be positioned out of the active pocket of LSD1. Therefore, peptide <b>39</b> was also expected to show well-maintained LSD1-inhibitory activity, as compared with peptide <b>3</b>.</div><figure id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0009.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Surface of the catalytic pocket of LSD1 in complex with a SNAIL1 peptide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y48">2Y48</a>). The SNAIL1 peptide is displayed as a ball-and-stick model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Oligoarginine-conjugated peptides <b>37</b>â<b>39</b> were prepared and initially evaluated for in vitro LSD1-inhibitory activity (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Interestingly, oligoarginine-conjugated peptides <b>37</b>â<b>39</b> were found to be more potent LSD1 inhibitors than peptide <b>3</b>. The increase in LSD1-inhibitory activity of <b>37</b> and <b>38</b> may be due to the interaction between the cationic oligoarginine and anionic amino acid residues of LSD1, such as Asp 556, Asp 557, and Glu 559 (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). Furthermore, peptides <b>37</b>â<b>39</b> displayed only weak inhibitory activity toward LSD2, MAO A/B, and JARID1A (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>), confirming their high selectivity for LSD1 over LSD2, MAO A/B, and JARID1A (LSD2 IC<sub>50</sub>/LSD1 IC<sub>50</sub> = 23â80; MAO A IC<sub>50</sub>/LSD1 IC<sub>50</sub> > 94; MAO B IC<sub>50</sub>/LSD1 IC<sub>50</sub> = 94â104; JARID1A IC<sub>50</sub>/LSD1 IC<sub>50</sub> = 16â44).</div><figure id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0010.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. View of the catalytic pocket of LSD1 in the complex with a SNAIL1 peptide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y48">2Y48</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. In Vitro LSD1-, LSD2-, MAO-A-, MAO-B-, and JARID1A-Inhibitory Activities of Oligoarginine-Conjugated Peptides <b>37</b>â<b>39</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="5" align="center" char=".">IC<sub>50</sub> (Î¼M)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Compd</th><th class="colsep0 rowsep0" align="center" char=".">LSD1</th><th class="colsep0 rowsep0" align="center" char=".">LSD2</th><th class="colsep0 rowsep0" align="center" char=".">MAOÂ A</th><th class="colsep0 rowsep0" align="center" char=".">MAOÂ B</th><th class="colsep0 rowsep0" align="center" char=".">JARID1A</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">0.44</td><td class="colsep0 rowsep0" align="char" char="."> > 50</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="char" char=".">0.28</td><td class="colsep0 rowsep0" align="char" char=".">13.20</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">6.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="char" char=".">0.29</td><td class="colsep0 rowsep0" align="char" char=".">23.20</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">12.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="char" char=".">0.32</td><td class="colsep0 rowsep0" align="char" char=".">7.51</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">5.05</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Values are means of at least three experiments.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">PCPA (IC<sub>50</sub> = 41 Î¼M for LSD1; IC<sub>50</sub> = 4.2 Î¼M for MAO A; IC<sub>50</sub> = 3.5 Î¼M for MAO B) and GSK2879552 (IC<sub>50</sub> = 0.61 Î¼M for LSD1) were used as positive controls.</p></div></div></div><div class="NLM_p">We also carried out mass spectrometric analysis of a mixture of LSD1 incubated with peptides <b>37</b>â<b>39</b>. The peak of peptide <b>37</b>-, <b>38</b>-, or <b>39</b>-FAD adduct was detected dependently on LSD1, as in the case of peptide <b>3</b> (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S4">Supporting Information Figure S4</a>).</div><div class="NLM_p">Next, we examined the activity of peptides <b>37</b>â<b>39</b> and <b>40</b> in cell-based assays. Peptide <b>40</b> (<a class="ref internalNav" href="#cht3" aria-label="Chart 3">Chart 3</a>), in which the hydrazine of peptide <b>37</b> is replaced with an amine, was prepared as a reference compound to confirm that the covalent modification is essential for the in-cell activity of the peptides. We performed a cellular assay using Western blot analysis. Because LSD1 is known to catalyze demethylation of H3K4me2, H3K4me1, and H3K9me2,<a onclick="showRef(event, 'cit4a'); return false;" href="javascript:void(0);" class="ref cit4a">(4a)</a> the effect of oligoarginine peptides on the methylation levels of H3K4 and H3K9 in HeLa cells was examined. As shown in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>a, treatment with 20 Î¼M peptides <b>37</b> and <b>38</b> induced a large increase in dimethylated H3K4 as compared with that induced by peptides <b>3</b>, <b>39</b>, and <b>40</b>. Loss in the H3K4 dimethylating activity of amine <b>40</b> suggests that the covalent bond formation of the peptide with FAD is responsible for the activity in cells. Furthermore, the increase of H3K4me2 in the presence of peptides <b>37</b> and <b>38</b> was dose-dependent without affecting the levels of H3K4me3, H3K4me1, and H3K9me2 (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>b). Similar results have been obtained with the reported hydrazine-based LSD1 inhibitor.<a onclick="showRef(event, 'cit12l'); return false;" href="javascript:void(0);" class="ref cit12l">(12l)</a> The results in both enzyme assays and cellular assays suggest that peptides <b>37</b> and <b>38</b> bearing oligoarginine moieties permeate through the cell membrane and selectively inhibit LSD1 over JARID1 in the nucleus.</div><figure id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0011.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Western blot detection of methylation level of H3K4 or H3K9 in HeLa cells after 24 h treatment with peptides in the absence of fetal bovine serum (FBS). Values of H3K4me2/H3, H3K4me1/H3, H3K4me3/H3, or H3K9me2/H3 ratio determined by optical density measurement of the blots are shown. (a) Treatment with 20 Î¼M of GSK2879552, peptides <b>3</b>, and <b>37</b>â<b>40</b>. (b) Dose-dependency for peptides <b>37</b> and <b>38</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Finally, we tested the effects of peptides <b>37</b>â<b>40</b> on the viability of cancer cells (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). In this study, we used HeLa cells because it has been reported that LSD1 is overexpressed in HeLa cells, and LSD1 inhibition suppresses HeLa cell growth.<a onclick="showRef(event, 'cit12d cit12e cit12f'); return false;" href="javascript:void(0);" class="ref cit12d cit12e cit12f">(12d-12f)</a> Peptide <b>37</b> reduced the cell viability at concentrations of 25â100 Î¼M. Although peptides <b>38</b> and <b>39</b> were moderately effective, their activity was weaker than that of peptide <b>37</b>. On the other hand, peptide <b>40</b> did not inhibit the cell viability strongly. Thus, peptide <b>37</b> showed the strongest antiproliferative activity against HeLa cells among the three peptides.</div><figure id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0012.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Effect of peptides <b>37</b>â<b>40</b> on viability of HeLa cells. The cells were incubated for 48 h after 24 h treatment with peptides <b>37</b>â<b>40</b> at the indicated concentrations in the absence of fetal bovine serum (FBS). Error bars represent the mean standard deviation (SD) of at least three samples. 300 Î¼M GSK2879552 reduced the cell viability by 9.96 Â± 0.47% (GSK2879552 was inactive at 100 Î¼M).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84146" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84146" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have designed SNAIL1-based peptides bearing a propargyl amine (<b>1</b> and <b>2</b>), hydrazine (<b>3</b> and <b>4</b>), or phenylcyclopropane (<b>5</b>) moiety based on the X-ray crystal structure of LSD1 complexed with a 20-amino-acid SNAIL1 <i>N</i>-terminal peptide. Among these five peptides, hydrazine-containing SNAIL1-based peptide <b>3</b> showed the most potent LSD1-inhibitory activity with an IC<sub>50</sub> of 0.44 Î¼M. In addition, kinetic study and mass spectrometric analysis demonstrated that peptide <b>3</b> inhibits LSD1 through covalent bond formation with FAD after oxidation of the hydrazine moiety in the active site of LSD1. Furthermore, cell-penetrating peptides <b>37</b> and <b>38</b>, which were designed based on peptide <b>3</b>, increased H3K4-methylation, suggesting that they are likely to inhibit intracellular LSD1 activity. In addition, peptide <b>37</b> decreased the viability of HeLa cells. To our knowledge, this is the first report of cell-active peptide-based LSD1 inactivators. These SNAIL1-based peptide derivatives represent an entry into a new class of LSD1-selective inhibitors, and should be useful tools for probing the biology of LSD1.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51264" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51264" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry</h3><div class="NLM_p last">Melting points were determined using a Yanagimoto micro melting point apparatus or a BÃ¼chi 545 melting point apparatus. Proton nuclear magnetic resonance spectra (<sup>1</sup>H NMR) and carbon nuclear magnetic resonance spectra (<sup>13</sup>C NMR) were recorded on a JEOL JNM-LA500, JEOL JNM-A500, BURKER AVANCE 300, or BRUKER AVANCE600 spectrometer in the indicated solvent. Chemical shifts (Î´) are reported in parts per million relative to the internal standard tetramethylsilane. Electron ionization (EI) mass spectra and fast atom bombardment (FAB) mass spectra were recorded on a JEOL JMS-SX102A mass spectrometer. Electrospray ionization (ESI) mass spectra were recorded on a BRUKER HCTplus mass spectrometer (ESI-IT) or a Waters MICROMASS LCT PREMIER (ESI-TOF). The specific rotation was measured on a HORIBA SEPA-300 polarimeter. HPLC analysis for unnatural amino acids was performed on a Cosmosil 5C<sub>18</sub>-AR-II analytical column (Nacalai Tesque, 4.6 Ã 250 mm, flow rate 1.0 mL/min) and eluted products were detected by UV absorbance measurement at 254 nm. A solvent system consisting of 0.1% (v/v) TFA aqueous solution (solvent A) and 0.1% (v/v) TFA in MeCN (solvent B) was used for HPLC elution. For HPLC separation of synthetic peptides, a Cosmosil 5C<sub>18</sub>-AR-II analytical column (Nacalai Tesque, 4.6 Ã 250 mm, flow rate 1.0 mL/min) or a Cosmosil 5C<sub>18</sub>-AR-II preparative column (Nacalai Tesque, 20 Ã 250 mm, flow rate 10 mL/min) was employed and eluted products were detected by UV absorbance measurement at 220 nm. A solvent system consisting of 0.1% (v/v) TFA aqueous solution (solvent A) and 0.1% (v/v) TFA in MeCN (solvent B) was used for HPLC elution. Chiral HPLC was performed on a CHIRALPAK IA column (4.6 Ã 250 mm, flow rate 1 mL/min) and eluted products were detected by UV absorbance measurement at 254 nm. A solvent system consisting of hexane/<i>i</i>PrOH = 90/10 (v/v) was used for chiral HPLC elution. The purity of the unnatural amino acid compounds for peptide synthesis and the tested peptides was determined by HPLC and the purity level was â¥95%. Reagents and solvents were purchased from Aldrich, Tokyo Kasei Kogyo, Wako Pure Chemical Industries, and Kanto Kagaku and used without purification. Flash column chromatography was performed using silica gel supplied by Merck or Toyota Silica Gel.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Fmoc-Based Solid-Phase Peptide Synthesis</h3><div class="NLM_p">An Fmoc-protected amino acid (5.0 equiv) was coupled to NovaSyn TGR resin (0.25 mmol amine/g), which affords peptide amide, with the aid of <i>N</i>,<i>N</i>â²-diisopropylcarbodiimide (5.0 equiv) and 1-hydroxy-1<i>H</i>-benzotriazole monohydrate (5.0 equiv) in DMF at room temperature for 2 h. Then the Fmoc group was removed with 20% (v/v) piperidine in DMF at room temperature for 10 min. After peptide chain elongation, Boc-Pro-OH was introduced as an N-terminal residue. The resulting resin was treated with TFA/<i>m</i>-cresol/thioanisole/1,2-ethanedithiol/H<sub>2</sub>O (80/5/5/5/5 (v/v), 50 Î¼L/1 mg resin) at room temperature for 1.5 h. After removal of the resin by filtration, cooled Et<sub>2</sub>O was added to the filtrate, and the precipitate was collected by centrifugation. The precipitate was washed with Et<sub>2</sub>O, and the obtained crude peptide was purified by preparative HPLC and then lyophilized. All purified peptides were obtained as white lyophilized powders. For introduction of an oligoarginine unit, an <i>N</i>-Îµ-[1-(4,4-dimethyl-2,6-dioxocyclohex-1-ylidene)-3-methylbutyl] (ivDde) group was used as a protecting group for a side chain of lysine. Fmoc-Lys(ivDde)âOH was incorporated at the site of Lys8 or Lys15. After sequence elongation, the ivDde group was removed by treatment with 4% (v/v) hydrazine monohydrate in DMF at room temperature for 5 h. Then, oligoarginine (R8) or an oligoarginine (R8)-miniPEG was introduced on the generated Îµâamine using standard Fmoc-based solid-phase peptide synthesis as described above. Results of characterization of peptides are presented in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> SNAIL1â20-Peptide-Based Propargylbenzylamine (<b>1</b>). Step 1: Preparation of (<i>S</i>)-5-Benzyl 1-<i>tert</i>-Butyl 2-<i>tert</i>-butoxycarbonylaminopentanedioate (<b>7</b>)</h4><div class="NLM_p last">A solution of (<i>S</i>)-5-benzyloxy-2-<i>tert</i>-butoxycarbonylamino-5-oxopentanoic acid (<b>6</b>) (6.00 g, 17.8 mmol), <i>tert</i>-butanol (1.70 mL, 18.0 mmol), EDCI (3.45 g, 18.0 mmol), and DMAP (2.20 g, 18.0 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (60 mL) was stirred at room temperature for 12 h. The reaction mixture was washed with saturated aqueous NaHCO<sub>3</sub> and brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane = 1/9) gave 6.50 g (93%) of <b>7</b> as a colorless oil: <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, Î´; ppm) 7.35â7.30 (5H, m), 5.12 (2H, s), 4.03â4.00 (1H, m), 2.47â2.44 (2H, m), 2.14â2.07 (1H, m), 1.89â1.84 (1H, m), 1.45 (9H, s), 1.43 (9H, s); MS (EI) <i>m</i>/<i>z</i>: 393 (M<sup>+</sup>); [Î±]<sup>24</sup><sub>D</sub> â6.74 (c 0.089, MeOH).</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Step 2: Preparation of (<i>S</i>)-5-<i>tert</i>-Butoxy-4-<i>tert</i>-butoxycarbonylamino-5-oxopentanoic acid (<b>8</b>)</h4><div class="NLM_p last">To a solution of <b>7</b> (6.00 g, 15.3 mmol) in MeOH (3 mL) was added 850 mg of 5% Pd/C. The mixture was stirred under H<sub>2</sub> at room temperature for 8 h, and then filtered through Celite. The filtrate was concentrated in vacuo to give 4.50 g (97%) of <b>8</b> as a colorless oil: <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, Î´; ppm) 4.02â3.99 (1H, m), 2.38 (2H, t, <i>J</i> = 7.5 Hz), 2.08â2.04 (1H, m), 1.88â1.81 (1H, m), 1.47 (9H, s), 1.44 (9H, s); MS (EI) <i>m</i>/<i>z</i>: 303 (M<sup>+</sup>); [Î±]<sup>23</sup><sub>D</sub> â12.22 (c 0.090, MeOH).</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Step 3: Preparation of (<i>S</i>)-<i>tert</i>-Butyl 2-<i>tert</i>-butoxycarbonylamino-5-hydroxypentanoate (<b>9</b>)</h4><div class="NLM_p last">To a solution of <b>8</b> (1.53 g, 5.06 mmol) and triethylamine (3.5 mL, 25.1 mmol) in THF (21 mL) was added ethyl chloroformate (2.5 mL, 26.1 mmol) at â10 Â°C. The mixture was stirred at â10 Â°C for 1 h and insoluble materials were filtered off. To a solution of NaBH<sub>4</sub> (956 mg, 25.3 mmol) in THF (20 mL) and water (5 mL) was added the filtrate at 0 Â°C, and the reaction mixture was stirred at room temperature for 5 h. After acidification with 1 N aqueous HCl (to pH 3), the mixture was extracted with AcOEt. The organic layer was separated, washed with 10% aqueous solution of NaOH, water, and brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane =1/3) gave 1.08 g (74%) of <b>9</b> as a colorless oil: <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, Î´; ppm) 3.96â3.94 (1H, m), 3.56â3.53 (2H, m), 1.86â1.80 (1H, m), 1.68â1.64 (1H, m), 1.60â1.55 (2H, m), 1.46 (9H, s), 1.44 (9H, s); MS (EI) <i>m</i>/<i>z</i>: 289 (M<sup>+</sup>); [Î±]<sup>23</sup><sub>D</sub> â5.62 (c 0.169, MeOH).</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Step 4: Preparation of (<i>S</i>)-<i>tert</i>-Butyl 2-<i>tert</i>-butoxycarbonylamino-5-methylsulfonyloxypentanoate (<b>10</b>)</h4><div class="NLM_p last">To a solution of <b>9</b> (500 mg, 1.73 mmol), triethylamine (3.5 mL, 25.1 mmol), and DMAP (21.0 mg, 0.173 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) was added dropwise methanesulfonyl chloride (270 Î¼L, 3.46 mmol) at â20 Â°C. The reaction mixture was stirred at â20 Â°C for 1 h, then poured into water and extracted with AcOEt. The organic layer was separated, washed with brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane = 1/1) gave 370 mg (58%) of <b>10</b> as a colorless oil: <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, Î´; ppm) 4.25 (2H, t, <i>J</i> = 6.5 Hz), 4.00â3.98 (1H, m), 3.06 (1H, s), 1.92â1.85 (1H, m), 1.83â1.77 (2H, m), 1.74â1.66 (1H, m), 1.46 (9H, s), 1.44 (9H, s); MS (EI) <i>m</i>/<i>z</i>: 367 (M<sup>+</sup>).</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Step 5: Preparation of (<i>S</i>)-<i>tert</i>-Butyl 5-benzyl(prop-2-yn-1-yl)amino-2-<i>tert</i>-butoxycarbonylaminopentanoate (<b>11</b>)</h4><div class="NLM_p last">To a solution of <b>10</b> (368 mg, 1.00 mmol) in DMF (1 mL) was added benzylpropargylamine (217 mg, 1.50 mmol) and Cs<sub>2</sub>CO<sub>3</sub> (500 mg, 1.50 mmol) and the reaction mixture was stirred at 65 Â°C for 2 h. The reaction mixture was poured into water and extracted with CHCl<sub>3</sub>. The organic layer was separated and washed with saturated aqueous NaHCO<sub>3</sub> and brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane =1/1) gave 70 mg (19%) of <b>11</b> as a colorless oil: <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, Î´; ppm) 7.34 (2H, d, <i>J</i> = 7.5 Hz), 7.28â7.25 (3H, m), 4.58 (2H, s), 4.16 (2H, s), 4.04â4.03 (2H, s), 4.00â3.99 (2H, m), 2.65 (1H, d, <i>J</i> = 2.5 Hz), 1.70â1.68 (4H, m), 1.46 (9H, s), 1.44 (9H, s); MS (EI) <i>m</i>/<i>z</i>: 416 (M<sup>+</sup>).</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Step 6: Preparation of (<i>S</i>)-2-Amino-5-benzyl(prop-2-yn-1-yl)aminopentanoic Acid Hydrochloric Acid Salt (<b>13Â·HCl</b>)</h4><div class="NLM_p last">To a solution of <b>11</b> (500 mg, 1.20 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) was added dropwise 4 N HCl in dioxane (5 mL, 20.0 mmol) at 0 Â°C. The reaction mixture was stirred at 0 Â°C for 1 h, and then concentrated in vacuo to give 330 mg (80%) of <b>13Â·HCl</b> as a colorless amorphous solid: <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, Î´; ppm) 7.58 (2H, broad s), 7.33 (2H, d, J = 7.5 Hz), 7.29â7.28 (3H, m), 4.58 (2H, s), 4.21 (2H, s), 4.04â3.98 (4H, m), 2.66 (1H, d, <i>J</i> = 2.5 Hz), 2.11â1.98 (4H, m); MS (EI) <i>m</i>/<i>z</i>: 260 (M<sup>+</sup>).</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Step 7: Preparation of (<i>S</i>)-2-[(9<i>H</i>-Fluoren-9-yl)methoxy]carbonylamino-5-benzyl(prop-2-yn-1-yl)aminopentanoic Acid (<b>15</b>)</h4><div class="NLM_p last">To a solution of <b>13Â·HCl</b> (330 mg, 1.27 mmol) and NaHCO<sub>3</sub> (640 mg, 7.61 mmol) in water (2 mL) and dioxane (1 mL) was added <i>N</i>-(9-fluorenylmethoxycarbonyloxy)succinimide (427 mg, 1.27 mmol) in dioxane (1 mL) at 0 Â°C. The reaction mixture was stirred at room temperature for 15 h. After acidification with 2 N aqueous HCl (to pH 6), the mixture was extracted with AcOEt. The organic layer was separated, washed with brine and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (CHCl<sub>3</sub>/MeOH = 1/9) gave 120 mg (20%) of <b>15</b> as a colorless solid: <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, Î´; ppm) 7.79 (2H, d, <i>J</i> = 7.5 Hz), 7.68 (2H, d, <i>J</i> = 7.5 Hz), 7.39 (2H, d, <i>J</i> = 7.5 Hz), 7.31â7.28 (5H, m), 7.27â7.25 (2H, m), 4.57 (2H, s), 4.37â4.35 (2H, m), 4.22â4.17 (4H, m), 4.05â3.98 (2H, m), 2.65 (1H, d, <i>J</i> = 2.5 Hz), 2.12â1.98 (2H, m), 1.81â1.77 (2H, m); MS (EI) <i>m</i>/<i>z</i>: 482 (M<sup>+</sup>); HPLC, retention time = 43.4 min, purity 95.6% (analytical HPLC conditions: a linear gradient, 0 min (5% B)-45 min (50% B)-55 min (50% B)-60 min (90% B)); [Î±]<sup>20</sup><sub>D</sub> + 6.94 (c 0.072, MeOH).</div></div><div id="sec5_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Step 8: Preparation of SNAIL1â20-Peptide-Based Propargylbenzylamine (<b>1</b>)</h4><div class="NLM_p last">By employing <b>15</b>, peptide <b>1</b> was obtained in 17% isolated yield based on the starting resin: HPLC, retention time =19.4 min, purity 96.0% (analytical HPLC conditions: linear gradient of solvent B in solvent A, 5 to 30% over 30 min); preparative HPLC conditions: linear gradient of solvent B in solvent A 13% to 23% over 30 min); LRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>111</sub>H<sub>177</sub>N<sub>35</sub>O<sub>30</sub> ([M + 3H]<sup>3+</sup>): 827.8, found: 827.6.</div></div><div id="sec5_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> SNAIL1â20-Peptide-Based Propargylmethylamine (<b>2</b>). Preparation of (<i>S</i>)-2-({[(9<i>H</i>-Fluoren-9-yl)methoxy]carbonyl}amino)-5-[methyl(prop-2-yn-1-yl)amino]pentanoic Acid (<b>16</b>)</h4><div class="NLM_p last">Compound <b>16</b> was prepared from <b>10</b> using the procedure described for <b>1</b> (steps 5â7, yield 4.0% from <b>10</b>): <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD, Î´; ppm): 7.88 (2H, d, <i>J</i> = 7.5 Hz), 7.71â7.64 (2H, m), 7.37 (2H, t, <i>J</i> = 7.5 Hz), 7.29 (2H, t, <i>J</i> = 7.5 Hz), 4.33 (2H, m), 4.19 (1H, t, <i>J</i> = 6.5 Hz), 4.07 (1H, q, <i>J</i> = 4.5 Hz), 3.78 (2H, d, <i>J</i> = 2.5 Hz), 3.05 (1H, t, <i>J</i> = 2.5 Hz), 2.90â2.87 (2H, m), 2.65 (3H, s), 1.86â1.83 (1H, m), 1.72â1.61 (2H, m), 1.41â1.38 (2H, m); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>, Î´; ppm): 177.88, 158.40, 145.26, 128.22, 128.20, 126.30, 120.96, 78.64, 75.44, 68.01, 57.01, 56.51, 56.37, 45.96, 41.06, 33.11, 26.10, 24.06; MS (FAB) <i>m</i>/<i>z</i>: 423 (MH<sup>+</sup>); HPLC, retention time =25.2 min, purity 99.2% (analytical HPLC conditions: a linear gradient, 0 min (5% B)-25 min (50% B)-35 min (50% B)-40 min (90% B)-50 min (90% B)); [Î±]<sup>22</sup><sub>D</sub> + 2.19 (c 0.228, MeOH).</div></div><div id="sec5_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Preparation of SNAIL1â20-Peptide-Based Propargylmethylamine (<b>2</b>)</h4><div class="NLM_p last">By employing <b>16</b>, peptide <b>2</b> was obtained in 16% isolated yield based on the starting resin: HPLC, retention time =15.9 min, purity 96.8% (analytical HPLC conditions: linear gradient of solvent B in solvent A, 5 to 30% over 30 min); preparative HPLC conditions: linear gradient of solvent B in solvent A 6% to 16% over 30 min); LRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>105</sub>H<sub>173</sub>N<sub>35</sub>O<sub>30</sub> ([M + 3H]<sup>3+</sup>): 802.4, found: 802.2.</div></div><div id="sec5_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> SNAIL1â20-Peptide-Based Hydrazine (<b>3</b>). Step 1: Preparation of (<i>S</i>)-Dibenzyl 2-(Dibenzylamino)pentanedioate (<b>18</b>)</h4><div class="NLM_p last">To a solution of <span class="smallcaps smallerCapital">l</span>-glutamic acid (<b>17</b>) (4.26 g, 29.0 mmol), K<sub>2</sub>CO<sub>3</sub> (16.2 g, 118 mmol), and NaOH (2.33 g, 58.0 mmol) in water (100 mL) was slowly added benzyl bromide (20.2 g, 118 mmol). The reaction mixture was stirred at reflux temperature for 3 h, and then extracted with AcOEt. The organic layer was separated and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane =1/9) gave 4.31 g (29%) of <b>18</b> as a colorless oil: <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, Î´; ppm): 7.45â7.18 (20H, m), 5.23 (1H, d, <i>J</i> = 12.3 Hz), 5.22 (1H, d, <i>J</i> = 12.3 Hz), 5.00 (1H, d, <i>J</i> = 12.6 Hz), 5.00 (1H, d, <i>J</i> = 12.6 Hz), 4.95 (1H, d, <i>J</i> = 12.6 Hz), 3.88 (2H, d, <i>J</i> = 13.8 Hz), 3.50 (2H, d, <i>J</i> = 13.8 Hz), 3.41 (1H, t, <i>J</i> = 7.7 Hz), 2.50 (1H, m), 2.34 (1H, m), 2.05 (2H, m); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>, Î´; ppm): 173.53, 172.91, 139.95, 136.70, 129.65, 129.37, 129.28, 129.24, 129.11, 129.05, 128.92, 127.82, 66.95, 66.90, 60.60, 55.23, 31.42, 25.05; MS (ESI-IT) <i>m</i>/<i>z</i>: 508 [(M + H)<sup>+</sup>]; [Î±]<sup>23</sup><sub>D</sub> â83.52 (c 0.182, MeOH).</div></div><div id="sec5_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Step 2: Preparation of (<i>S</i>)-Benzyl 2-(dibenzylamino)-5-hydroxypentanoate (<b>19</b>)</h4><div class="NLM_p last">To a solution of <b>18</b> (1.92 g, 3.78 mmol) in THF (12.5 mL) was slowly added DIBAL (8.32 mL of a 1 M solution in toluene, 8.32 mmol) at â15 Â°C. The reaction mixture was stirred at 0 Â°C for 30 min, and then water (7 mL) was added. Stirring was continued at 0 Â°C for 30 min. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane =1/4 to 1/2) gave 1.40 g (92%) of <b>19</b> as a colorless oil: <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, Î´; ppm): 7.45â7.34 (5H, m), 7.33â7.18 (10H, m), 5.26 (1H, d, <i>J</i> = 12.3 Hz), 5.14 (1H, d, <i>J</i> = 12.3 Hz), 3.93 (2H, d, <i>J</i> = 14.1 Hz), 3.50 (1H, d, <i>J</i> = 13.5 Hz), 3.54â3.43 (4H, m), 3.38 (1H, t, <i>J</i> = 7.5 Hz), 1.87â1.61 (3H, m), 1.58â1.39 (1H, m), 1.30 (1H, br); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>, Î´; ppm): 173.48, 140.18, 136.83, 129.68, 129.37, 129.28, 129.02, 127.81, 66.81, 63.18, 61.36, 55.29, 30.10, 26.58; MS (ESI-IT) <i>m</i>/<i>z</i>: 404 [(M + H)<sup>+</sup>]; [Î±]<sup>24</sup><sub>D</sub> â111.54 (c 0.117, MeOH).</div></div><div id="sec5_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Step 3: Preparation of (<i>S</i>)-Benzyl 2-(dibenzylamino)-5-oxopentanoate (<b>20</b>)</h4><div class="NLM_p last">To a solution of oxalyl chloride (690 Î¼L, 7.92 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) was slowly added a solution of DMSO (675 Î¼L, 9.50 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) at â78 Â°C. The mixture was stirred for 25 min at â78 Â°C, then a solution of <b>19</b> (1.28 g, 3.17 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) at â78 Â°C was slowly added, and stirring was continued for 2 h at â78 Â°C. To the mixture was added Et<sub>3</sub>N (1.3 mL, 9.38 mmol), and stirring was continued for 1 h at â78 Â°C. The mixture was further stirred for 15 min 0 Â°C, then poured into water, and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The organic layer was separated, washed with brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane =1/9 to 1/2) gave 1.08 g (85%) of <b>20</b> as a colorless oil: <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, Î´; ppm): 9.59 (1H, t, <i>J</i> = 1.2 Hz), 7.46â7.35 (5H, m), 7.34â7.20 (10H, m), 5.27 (1H, d, <i>J</i> = 12.3 Hz), 5.17 (1H, d, <i>J</i> = 12.3 Hz), 3.87 (2H, d, <i>J</i> = 13.8 Hz), 3.50 (2H, d, <i>J</i> = 13.8 Hz), 3.35 (1H, t, <i>J</i> = 7.8 Hz), 2.60â2.32 (2H, m), 2.08â1.98 (2H, m); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>, Î´; ppm): 201.38, 172.12, 139.12. 135.95, 128.93, 128.67, 128.58, 128.45, 128.36, 127.19, 66.27, 59.87, 54.54, 40.47, 21.66; MS (ESI-IT) <i>m</i>/<i>z</i>: 402 [(M + H)<sup>+</sup>]; [Î±]<sup>22</sup><sub>D</sub> â72.22 (c 0.126, MeOH).</div></div><div id="sec5_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Step 4: Preparation of (<i>S</i>)-<i>tert</i>-Butyl 2-[5-(benzyloxy)-4-(dibenzylamino)-5-oxopentylidene]hydrazinecarboxylate (<b>21</b>)</h4><div class="NLM_p last">To a solution of <b>20</b> (382 mg, 0.951 mmol) in THF (10 mL) was added <i>tert</i>-butyl carbazate (257 mg, 1.94 mmol). The mixture was stirred for 5 h at room temperature, then diluted with AcOEt, washed with brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane =1/5 to 1/2) gave 459 mg (94%) of <b>21</b> as a colorless oil: <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, Î´; ppm), small rotamers observed but not reported: 7.43â7.32 (5H, m), 7.30â7.18 (10H, m), 6.78 (1H, m), 5.27 (1H, d, <i>J</i> = 12.3 Hz), 5.17 (1H, d, <i>J</i> = 12.3 Hz), 3.88 (2H, d, <i>J</i> = 13.8 Hz), 3.50 (2H, d, <i>J</i> = 13.8 Hz), 3.38 (1H, m), 2.50â2.36 (1H, m), 2.28â2.16 (1H, m), 2.05â1.84 (2H, m), 1.49 (9H, s); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>, Î´; ppm), small rotamers observed but not reported: 171.37, 138.40, 135.09, 128.08, 128.01, 127.80, 127.73, 127.64, 127.49, 127.42, 126.18, 80.12, 65.26, 58.99, 53.54, 27.84, 27.39, 25.44; MS (ESI-IT) <i>m</i>/<i>z</i>: 516 [(M + H)<sup>+</sup>].</div></div><div id="sec5_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Step 5: Preparation of (<i>S</i>)-<i>tert</i>-Butyl 2-[5-(benzyloxy)-4-(dibenzylamino)-5-oxopentyl]hydrazinecarboxylate (<b>23</b>)</h4><div class="NLM_p last">To a solution of <b>21</b> (230 mg, 0.446 mmol) in EtOH (5 mL) were added acetic acid (500 Î¼L, 8.74 mmol) and sodium cyanoborohydride (138 mg, 1.98 mmol). The reaction mixture was stirred for 12 h at room temperature, then diluted with AcOEt, washed with aqueous saturated sodium bicarbonate and brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane =1/4 to 1/1) gave 160 mg (69%) of <b>23</b> as a colorless oil: <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, Î´; ppm): 7.45â7.35 (5H, m), 7.32â7.18 (10H, m), 5.85 (1H, br), 5.26 (1H, d, <i>J</i> = 12.3 Hz), 5.14 (1H, d, <i>J</i> = 12.3 Hz), 3.88 (2H, d, <i>J</i> = 13.8 Hz), 3.50 (2H, d, <i>J</i> = 13.8 Hz), 3.35 (1H, t, <i>J</i> = 7.5 Hz), 2.68 (2H, t, <i>J</i> = 7.0 Hz), 1.86â1.73 (2H, m), 1.60 (13H, m); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>, Î´; ppm): 168.82, 135.65, 132.21, 124.69, 124.58, 124.41, 124.34, 123.07, 76.47, 62.06, 56.78, 50.58, 47.61, 24.46, 23.04, 20.48; MS (ESI-IT) <i>m</i>/<i>z</i>: 518 [(M + H)<sup>+</sup>].</div></div><div id="sec5_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Step 6: Preparation of (<i>S</i>)-<i>tert</i>-Butyl 2-[5-(benzyloxy)-4-(dibenzylamino)-5-oxopentyl]hydrazinecarboxylate (<b>25</b>)</h4><div class="NLM_p last">To a solution of <b>23</b> (160 mg, 0.309 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) were added di-<i>tert</i>-butyl dicarbonate (82.1 mg, 0.376 mmol) and 4-dimethylaminopyridine (10.2 mg). The mixture was stirred at room temperature for 7 h, then diluted with CH<sub>2</sub>Cl<sub>2</sub>, washed with water and brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (AcOEt/<i>n</i>âhexane =1/4) gave 148 mg (78%) of <b>25</b> as a colorless oil: <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, Î´; ppm) 7.52â7.35 (5H, m), 7.32â7.18 (10H, m), 5.26 (1H, d, <i>J</i> = 12.3 Hz), 5.12 (1H, d, <i>J</i> = 12.3 Hz), 4.38 (1H, br), 3.89 (2H, d, <i>J</i> = 13.8 Hz), 3.49 (2H, d, <i>J</i> = 13.8 Hz), 3.35 (1H, t, <i>J</i> = 7.3 Hz), 2.70 (2H, t, <i>J</i> = 7.3 Hz), 1.86â1.35 (22H, m); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>, Î´; ppm): 172.64, 152.37, 139.50, 136.12, 128.84, 128.60, 128.49, 128.31, 128.26, 126.98, 83.06, 65.97, 60.79, 54.46, 50.88, 28.20, 28.09, 27.11, 24.51; MS (ESI) <i>m</i>/<i>z</i>: 618 [(M + H)<sup>+</sup>].</div></div><div id="sec5_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Step 7: Preparation of (<i>S</i>)-2-Amino-5-[1,2-bis(<i>tert</i>-butoxycarbonyl)hydrazinyl]pentanoic Acid (<b>27</b>)</h4><div class="NLM_p last">To a solution of <b>25</b> (185 mg, 0.299 mmol) in MeOH (20 mL) was added a solution of formic acid (260 Î¼L) in MeOH (8.7 mL) and palladium black (100 mg, high surface area, 99.8%) at â40 Â°C. The reaction mixture was stirred at room temperature for 2 h. The palladium was filtered off and the filtrate was concentrated in vacuo to give 125 mg of crude <b>27</b> which was used for the next step without further purification: MS (ESI-IT) <i>m</i>/<i>z</i>: 348 [(M + H)<sup>+</sup>]; [Î±]<sup>25</sup><sub>D</sub> â50.00 (c 0.030, MeOH).</div></div><div id="sec5_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Step 8: Preparation of (<i>S</i>)-2-[(9<i>H</i>-Fluoren-9-ylmethoxycarbonyl)amino]-5-[1,2-bis(<i>tert</i>-butoxycarbonyl)hydrazinyl]pentanoic Acid (<b>29</b>)</h4><div class="NLM_p last">To a solution of crude <b>27</b> (125 mg) and NaHCO<sub>3</sub> (151 mg, 1.80 mmol) in water (1.5 mL) and dioxane (3 mL) was added <i>N</i>-(9-fluorenylmethoxycarbonyloxy)succinimide (242 mg, 0.720 mmol) in dioxane (1 mL) at 0 Â°C. The reaction was stirred at room temperature for 3 h. After acidification with 10% citric acid, the mixture was extracted with AcOEt. The organic layer was separated, washed with brine and dried over Na<sub>2</sub>SO<sub>4</sub>. Filtration, concentration in vacuo, and purification of the residue by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub> = 1/99 to 1/6) followed by preparative TLC (MeOH/CH<sub>2</sub>Cl<sub>2</sub> = 1/12) gave 151 mg (89%, 2 steps) of <b>29</b> as a colorless solid: <sup>1</sup>H NMR (CDCl<sub>3</sub>, 300 MHz, Î´; ppm), small rotamers observed but not reported: 7.74 (2H, d, <i>J</i> = 7.2 Hz), 7.59 (2H, m), 7.38 (2H, t, <i>J</i> = 7.2 Hz), 7.29 (2H, t, <i>J</i> = 7.2 Hz), 5.89 (1H, brd, <i>J</i> = 7.2 Hz), 4.60â4.32 (3H, m), 4.20 (1H, m), 2.87 (2H, m), 2.02â1.85 (2H, m), 1.81â1.69 (2H, m), 1.50 (18H, s); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>, Î´; ppm), small rotamers observed but not reported: 156.21, 152.32, 143.97, 143.84, 141.84, 127.66, 127.08, 125.20, 119.92, 83.94, 67.09, 53.86, 50.36, 47.16, 29.96, 28.05, 23.06; MS (ESI-IT) <i>m</i>/<i>z</i>: 570 [(M + H)<sup>+</sup>]; HPLC, retention time =20.2 min, purity 96.7% (rotamer mixture) (analytical HPLC conditions: a linear gradient, 0 min (50% B)-30 min (50% B)-35 min (90% B)-40 min (90% B)); [Î±]<sup>20</sup><sub>D</sub> + 1.90 (c 0.079, MeOH); Chiral HPLC, > 99% ee, retention time =59.0 min (major, <i>S</i> form), 28.0 min (minor, <i>R</i> form) (This compound was protected as methyl ester by using trimethylsilyldiazomethane<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and the ester form was analyzed by chiral HPLC).</div></div><div id="sec5_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Step 9: Preparation of SNAIL1â20-Peptide-Based Hydrazine (<b>3</b>)</h4><div class="NLM_p last">By employing <b>29</b>, peptide <b>3</b> was obtained in 16% isolated yield based on the starting resin: HPLC, retention time =18.0 min, purity 97.7% (analytical HPLC conditions: linear gradient of solvent B in solvent A, 1 to 30% over 30 min); preparative HPLC conditions: linear gradient of solvent B in solvent A 6% to 16% over 30 min); LRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>101</sub>H<sub>170</sub>N<sub>36</sub>O<sub>30</sub> ([M + 3H]<sup>3+</sup>): 790.1, found: 789.9.</div></div><div id="sec5_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> SNAIL1â20-Peptide-Based Hydrazine (<b>4</b>): Preparation of (<i>S</i>)-2-({[(9<i>H</i>-Fluoren-9-yl)methoxy]carbonyl}amino)-5-[1,2-bis(<i>tert</i>-butoxycarbonyl)-2-methylhydrazinyl]pentanoic Acid (<b>30</b>)</h4><div class="NLM_p last">Compound <b>30</b> was prepared from <b>20</b> using the procedure described for <b>3</b> (steps 5â8, yield 28% from <b>20</b>): <sup>1</sup>H NMR (DMSO, 300 MHz, Î´; ppm), small rotamers observed but not reported: 12.61 (1H, br), 7.89 (2H, d, <i>J</i> = 7.6 Hz), 7.65â7.63 (3H, m), 7.41 (2H, t, <i>J</i> = 7.6 Hz), 7.32 (2H, t, <i>J</i> = 7.6 Hz), 4.26â4.21 (3H, m), 4.07â3.96 (1H, m), 2.95 (3H, m), 1.83â1.68 (1H, m), 1.66â1.49 (3H, m), 1.41â1.34 (18H, m); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>, Î´; ppm), small rotamers observed but not reported: 175.95, 156.66, 155.91, 155.24, 143.91, 141.34, 127.76, 127.13, 125.19, 120.01, 81.61, 67.27, 53.64, 48.19, 47.19, 37.45, 29.86, 28.33, 24.34; MS (ESI-IT) <i>m</i>/<i>z</i>: 584 (MH<sup>+</sup>); HPLC, retention time =18.6 min, purity 98.3% (analytical HPLC conditions: a linear gradient, 0 min (50% B)-30 min (90% B)-40 min (90% B)); [Î±]<sup>20</sup><sub>D</sub> + 4.11 (c 0.073, MeOH).</div></div><div id="sec5_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Preparation of SNAIL1â20-Peptide-Based Hydrazine (<b>4</b>)</h4><div class="NLM_p last">By employing <b>30</b>, peptide <b>4</b> was obtained in 29% isolated yield based on the starting resin: HPLC, retention time =18.2 min, purity 96.0% (analytical HPLC conditions: linear gradient of solvent B in solvent A, 1 to 30% over 30 min); preparative HPLC conditions: linear gradient of solvent B in solvent A 6% to 16% over 30 min); LRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>102</sub>H<sub>172</sub>N<sub>36</sub>O<sub>30</sub> ([M + 3H]<sup>3+</sup>): 794.8, found: 794.6.</div></div><div id="sec5_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> SNAIL1â20-Peptide-Based Phenylcyclopropylamine (<b>5</b>). Preparation of (2<i>S</i>)-2-({[(9<i>H</i>-Fluoren-9-yl)methoxy]carbonyl}amino)-6-[(<i>tert</i>-butoxycarbonyl)(2-phenylcyclopropyl)amino]hexanoic Acid (<b>36</b>)</h4><div class="NLM_p last">Compound <b>36</b> was prepared from <b>31</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> using the procedure described for <b>1</b> (steps 4â7) and <b>3</b> (step 6): yield 2%; <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD, Î´; ppm) 7.90 (2H, t, <i>J</i> = 7.5 Hz), 7.78 (2H, d, <i>J</i> = 7.5 Hz), 7.39 (2H, d, <i>J</i> = 7.0 Hz), 7.29â7.27 (2H, m), 7.22â7.19 (2H, m), 7.12 (2H, d, <i>J</i> = 8.0 Hz), 7.09â7.06 (1H, m), 4.59â4.58 (1H, m), 4.20â4.18 (1H, m), 4.04â4.01 (1H, m), 2.57â2.56 (1H, m), 2.07â2.06 (1H, m), 1.51â1.42 (6H, m), 1.40 (9H, s), 1.28â1.12 (4H, m); MS (FAB): <i>m</i>/<i>z</i> 585 [(M + H)<sup>+</sup>]; <sup>13</sup>C NMR (500 MHz, CD<sub>3</sub>OD, Î´; ppm) HPLC, retention time =25.8 min, purity 97.0% (analytical HPLC conditions: a linear gradient, 0 min (10% B)-15 min (75% B)-35 min (75% B); [Î±]<sup>20</sup><sub>D</sub> + 0.197 (c 0.254, MeOH).</div></div><div id="sec5_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Preparation of SNAIL1â20-Peptide-Based Phenylcyclopropylamine (<b>5</b>)</h4><div class="NLM_p last">By employing <b>36</b>, peptide <b>5</b> was obtained in 14% isolated yield from the starting resin: HPLC, retention time =15.8 min, purity 95.2% (analytical HPLC conditions: linear gradient of solvent B in solvent A, 5 to 45% over 30 min); preparative HPLC conditions: linear gradient of solvent B in solvent A 15% to 25% over 30 min); LRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>111</sub>H<sub>179</sub>N<sub>35</sub>O<sub>30</sub> ([M + 3H]<sup>3+</sup>): 828.8, found: 828.5.</div></div><div id="sec5_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Oligoarginine-Conjugated Peptide <b>37</b></h4><div class="NLM_p last">Peptide <b>37</b> was prepared according to the general procedure and was obtained in 10% isolated yield from the starting resin: retention time =19.2 min, purity 95.0% (analytical HPLC conditions: linear gradient of solvent B in solvent A, 1 to 30% over 30 min); preparative HPLC conditions: linear gradient of solvent B in solvent A 8% to 18% over 30 min); LRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>149</sub>H<sub>266</sub>N<sub>68</sub>O<sub>38</sub> ([M + 4H]<sup>4+</sup>): 905.0, found: 904.8.</div></div><div id="sec5_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Oligoarginine-Conjugated Peptide <b>38</b></h4><div class="NLM_p last">Peptide <b>38</b> was prepared according to the general procedure and was obtained in 10% isolated yield from the starting resin: HPLC, retention time =19.6 min, purity 95.5% (analytical HPLC conditions: linear gradient of solvent B in solvent A, 1 to 30% over 30 min); preparative HPLC conditions: linear gradient of solvent B in solvent A 8% to 18% over 30 min); LRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>155</sub>H<sub>277</sub>N<sub>69</sub>O<sub>41</sub> ([M + 4H]<sup>4+</sup>): 941.3, found: 941.1.</div></div><div id="sec5_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Oligoarginine-Conjugated Peptide <b>39</b></h4><div class="NLM_p last">Peptide <b>39</b> was prepared according to the general procedure and was obtained in 21% isolated yield from the starting resin: HPLC, retention time =18.8 min, purity 97.5% (analytical HPLC conditions: linear gradient of solvent B in solvent A, 1 to 30% over 30 min); preparative HPLC conditions: linear gradient of solvent B in solvent A 7% to 17% over 30 min); LRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>149</sub>H<sub>266</sub>N<sub>68</sub>O<sub>38</sub> ([M + 4H]<sup>4+</sup>): 905.0, found: 904.9.</div></div><div id="sec5_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Oligoarginine-Conjugated Peptide <b>40</b></h4><div class="NLM_p last">Peptide <b>40</b> was prepared according to the general procedure and was obtained in 33% isolated yield from the starting resin: HPLC, retention time =20.0 min, purity 97.3% (analytical HPLC conditions: linear gradient of solvent B in solvent A, 1 to 30% over 30 min); preparative HPLC conditions: linear gradient of solvent B in solvent A 8% to 30% over 30 min); LRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>150</sub>H<sub>267</sub>N<sub>67</sub>O<sub>38</sub> ([M + 4H]<sup>4+</sup>): 904.8, found: 904.3.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Biology</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> LSD1 Inhibition Assay</h4><div class="NLM_p last">LSD1 activity was assayed at 25 Â°C by using the peroxidase-coupled method, as described previously.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a><sup>d,e,f</sup> The chemically synthesized peptide consisting of the first 21 amino acid residues of histone H3, incorporating dimethylated lysine at position 4 (H3K4me2 peptide) (Sigma-Aldrich), was used as the substrate of LSD1. The reaction mixture contained 50 mM HEPES-NaOH, pH 7.5, 0.1 mM 4-aminoantipyrine, 1 mM 3,5-dichloro-2-hydroxybenzenesulfonic acid, 5.5 units/ml horseradish peroxidase, 20 Î¼M H3K4me2 peptide, and an appropriate amount of LSD1. To assess the inhibitory effect of test compounds on the LSD1 activity, we used partially purified LSD1 (obtained at the purification step of HisTrap HP chromatography) dissolved in buffer C. Reaction in the absence of inhibitors was also done as a control. Reaction mixtures (18 Î¼L), containing all the materials except H3K4me2 peptide were first incubated for 5 min. Then, the reactions were initiated by adding 2 Î¼L of 0.2 mM peptide solution to the assay mixtures. Absorbance at 515 nm was monitored for 30 min in a 384-well plate (Nunc) by using a microplate reader (SpectraMax M2<sup>e</sup>; Molecular Devices). Enzyme activity was determined from the linear part of the reaction curve. The ratio of the enzyme activity measured in the presence of inhibitor to the activity of the control was plotted against log[Inh].</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> LSD2 Inhibition Assay</h4><div class="NLM_p last">Production and purification of recombinant His-tagged human full-length LSD2 was performed as described previously.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The histone demethylation reaction for assaying LSD2 carried out for 1 h at 37 C in 20 Î¼L of 50 mM HEPES-NaOH (pH 8.5) containing 0.1% BSA, 2 Î¼g of His-tagged full-length LSD2, and 10 Î¼M H3K4me2, the chemically synthesized peptide consisted of the first 21 amino acid residues of histone H3, incorporating dimethylated lysine at position 4. The reaction was stopped by adding 20 Î¼L of 100% acetonitrile including 0.1% trifluoroacetic acid. 0.5 Î¼L of 10 mg/mL Î±-Cyano-4-hydroxycinnamic acid was added to the 0.5 Î¼L of reaction mixture, and 1 Î¼L of this mixture was spotted on the sample plate, dried and analyzed by matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOFMS) using JMS-S3000 (JEOL). The enzymatic removal of H3K4me0/1 from H3K4me2 substrate was quantified by this protocol. The LSD2 inhibition activity of the test compounds was calculated from the remaining amount of H3K4me2. The 50% inhibitory concentration (IC<sub>50</sub>) of the test compounds was calculated as the concentration at which the half amount of H3K4me2 was removed compared to that removed when the enzyme was added.</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> MAO Inhibition Assays</h4><div class="NLM_p last">The MAO activity assays were performed using a MAO-Glo assay kit (Promega), according to the supplierâs protocol. MAO A (18 unit/mg) or MAO B (6 unit/mg) (Sigma-Aldrich) (25 Î¼L/well), 160 Î¼M (for MAO-A) or 16 Î¼M (for MAO-B) (4<i>S</i>)-4,5-dihydro-2-(6-hydroxybenzothiazolyl)-4-thiazolecarboxylic acid (12.5 Î¼L/well), a MAO substrate, and various concentrations of inhibitors (12.5 Î¼L/well) were incubated at room temperature. Reactions were stopped after 60 min by adding reconstituted Luciferin Detection Reagent (50 Î¼L/well). Then, 20 min after addition of this reagent, the fluorescence at 562 nm of the wells was measured with a fluorescence reader. The value of % inhibition was calculated from the fluorescence readings of inhibited wells relative to those of control wells.</div></div><div id="sec5_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> JARID1A Inhibition Assay</h4><div class="NLM_p last">The JARID1A activity was measured as described in ref <a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">22</a>.</div></div><div id="sec5_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Kinetic Analysis</h4><div class="NLM_p last">The kinetic analysis was performed using a LSD1 fluorometric drug discovery kit (Enzo #NML-AK544-0001). Peptide <b>3</b>, H3K4me2 peptide (final concentration, 20 Î¼M) and HRP/CELLestial Red were mixed in assay buffer (total volume, 80 Î¼L) on each well of a 96-plate. The reactions (in 100 Î¼L) were initiated by adding LSD1 (10 Î¼g) in 20 Î¼L of assay buffer to each well. After incubation at room temperature for 5 min, the fluorescence in each well was measured with an ARVO X3 microplate reader (excitation at 540 nm, emission, at 590 nm) every 30 s for 15 min.</div></div><div id="sec5_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> MALDI-TOF/MS Analysis</h4><div class="NLM_p last">LSD1 (12 Î¼M) was incubated with 100 Î¼M peptides <b>3</b> and <b>4</b> for 22 h at room temperature in 50 mM sodium phosphate buffer (pH 7.5), 4% glycerol and 1% DMSO. The control was run with FAD instead of the enzyme. The reaction mixtures (10 Î¼L) were then denatured with 6 M guanidine hydrochloride (10 Î¼L) and acidified with 1% TFA (5 Î¼L). An aliquot of the reaction mixture (10 Î¼L) was desalted and concentrated using ZipTip Î¼-C18 (Millipore) (eluted with 5 Î¼L of 50% MeCN containing 0.1% TFA). The eluate was analyzed on a 5800 MALDI-TOF/TOF mass spectrometer (AB SCIEX). MALDI mass spectra were acquired between <i>m</i>/<i>z</i> 800 and 4000 in the positive ion mode. Î±-Cyano-4-hydroxycinnamic acid (CHCA, LaserBio) was used as the MALDI matrix without further purification. CHCA was dissolved in 50% MeCN containing 0.1% TFA to a concentration of 5 mg/mL. Samples for MALDI-TOF MS analysis were prepared by mixing the desalted reaction mixture (2 Î¼L) with the matrix solution (2 Î¼L). The applied samples (1 Î¼L) were allowed to dry at room temperature.</div></div><div id="sec5_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Western Blot Analysis</h4><div class="NLM_p last">Human cervical carcinoma HeLa cells were cultured in RPMI 1640 containing 10% heat-inactivated fetal bovine serum (FBS), penicillin and streptomycin mixture at 37 Â°C in a humidified atmosphere of 5% CO<sub>2</sub> in air. The HeLa cells (5 Ã 10<sup>5</sup>) were plated in 6-well plates (Iwaki) and cultured for 24 h. After removal of the medium, the cells were washed with PBS. The cells were incubated with synthetic peptides at the indicated concentrations in RPMI1640 medium without FBS for 24 h, then collected and extracted with SDS buffer. Protein concentrations of the lysates were determined using a BCA protein assay. Equivalent amounts of protein from each lysate were resolved in 5â20% SDS-polyacrylamide gels and transferred onto PVDF membranes. After blocking with TBS-T containing 5% skim milk, the transblotted membranes were probed with rabbit monoclonal H3K4me3 antibody (Abcam, #ab8580) (1:500 dilution), rabbit polyclonal H3K4me2 antibody (Abcam, #ab32356) (1:1000 dilution), rabbit polyclonal H3K4me1 antibody (Abcam, #ab194691) (1:500 dilution), rabbit polyclonal H3K9me2 antibody (Abcam, #ab194680) (1:500 dilution) or rabbit polyclonal H3 antibody (Abcam, #ab1791) (1:200000 dilution) in TBS-T. Each probed membrane was washed with TBS-T, incubated with ECL rabbit IgG, HRP-linked whole antibody (GE Healthcare Life Sciences, #NA934) (1:2500 dilution), and again washed with TBS-T. The immunoblots were visualized by enhanced chemiluminescence with Immobilon Western Chemiluminescent HRP Substrate (Millipore, #P90718).</div></div><div id="sec5_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Cell Viability Assay</h4><div class="NLM_p last">The cells were plated in 96-well plates at the initial density of 1 Ã 10<sup>3</sup> cells/well (50 Î¼L/well, HeLa) and incubated at 37 Â°C. After 24 h, the medium was removed and the cells were washed with PBS. The cells were incubated with synthetic peptides at the indicated concentrations in 50 Î¼L of RPMI1640 medium without FBS for 24 h. Then, 50 Î¼L of RPMI1640 culture medium was added to each well and the plates were incubated at 37 Â°C under 5% CO<sub>2</sub> in air for a further 48 h. The mixtures were then treated with 10 Î¼L of AlamarBlue (AbD Serotec, #BUF012A), and incubation was continued at 37 Â°C for 3 h. The fluorescence in each well was measured with an ARVO X3 microplate reader (excitation at 540 nm, emission, at 590 nm). The cell viability (%) was calculated from the fluorescence readings.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01323">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31035" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31035" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01323" class="ext-link">10.1021/acs.jmedchem.5b01323</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">View of the designed peptides <b>1</b>â<b>5</b> in the active site of LSD1; mass spectrometric detection and quantitative analysis of generation of 3-FAD adduct; mass spectrometric detection of LSD1-mediated demethylation of peptide <b>4</b> and of the <b>37</b>-, <b>38</b>-, or <b>39</b>-FAD adduct; synthesis and chiral HPLC data of the methyl ester compounds derivatized from unnatural amino acid <b>29</b>; HPLC data; and MS spectra of synthetic peptides (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01323/suppl_file/jm5b01323_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01323/suppl_file/jm5b01323_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01323/suppl_file/jm5b01323_si_001.pdf">jm5b01323_si_001.pdf (1.27 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01323/suppl_file/jm5b01323_si_002.csv">jm5b01323_si_002.csv (4.01 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01323" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88786" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88786" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Akira Otaka</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biomedical Sciences and Graduate School of Pharmaceutical Sciences, Tokushima University, Shomachi, Tokushima 770-8505, Japan</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#64050b10050f0524100b0f11170c0d090549114a05074a0e14"><span class="__cf_email__" data-cfemail="88e9e7fce9e3e9c8fce7e3fdfbe0e1e5e9a5fda6e9eba6e2f8">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Takayoshi Suzuki</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Medical Science, Kyoto Prefectural
University of Medicine, 1-5 Shimogamohangi-Cho, Sakyo-Ku, Kyoto, 606-0823, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">CREST, Japan Science
and Technology Agency (JST), 4-1-8
Honcho Kawaguchi, Saitama 332-0012, Japan</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2b585e515e40425f6b40445f4405405b5e0646054a4805415b"><span class="__cf_email__" data-cfemail="1b686e616e70726f5b70746f7435706b6e3676357a7835716b">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yukihiro Itoh</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Medical Science, Kyoto Prefectural
University of Medicine, 1-5 Shimogamohangi-Cho, Sakyo-Ku, Kyoto, 606-0823, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Keisuke Aihara</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biomedical Sciences and Graduate School of Pharmaceutical Sciences, Tokushima University, Shomachi, Tokushima 770-8505, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paolo Mellini</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Medical Science, Kyoto Prefectural
University of Medicine, 1-5 Shimogamohangi-Cho, Sakyo-Ku, Kyoto, 606-0823, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Toshifumi Tojo</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Medical Science, Kyoto Prefectural
University of Medicine, 1-5 Shimogamohangi-Cho, Sakyo-Ku, Kyoto, 606-0823, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yosuke Ota</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Medical Science, Kyoto Prefectural
University of Medicine, 1-5 Shimogamohangi-Cho, Sakyo-Ku, Kyoto, 606-0823, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hiroki Tsumoto</span> - <span class="hlFld-Affiliation affiliation">Research
Team for Mechanism of Aging Tokyo Metropolitan Institute of Gerontology, 35-2 Sakae-Cho, Itabashi-ku, Tokyo 173-0015, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Viswas Raja Solomon</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Pharmaceutical Sciences, Nagoya
City University, 3-1
Tanabe-dori, Mizuho-ku, Nagoya, Aichi 467-8603, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peng Zhan</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Medical Science, Kyoto Prefectural
University of Medicine, 1-5 Shimogamohangi-Cho, Sakyo-Ku, Kyoto, 606-0823, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Miki Suzuki</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Medical Science, Kyoto Prefectural
University of Medicine, 1-5 Shimogamohangi-Cho, Sakyo-Ku, Kyoto, 606-0823, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daisuke Ogasawara</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Medical Science, Kyoto Prefectural
University of Medicine, 1-5 Shimogamohangi-Cho, Sakyo-Ku, Kyoto, 606-0823, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Akira Shigenaga</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biomedical Sciences and Graduate School of Pharmaceutical Sciences, Tokushima University, Shomachi, Tokushima 770-8505, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tsubasa Inokuma</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biomedical Sciences and Graduate School of Pharmaceutical Sciences, Tokushima University, Shomachi, Tokushima 770-8505, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hidehiko Nakagawa</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Pharmaceutical Sciences, Nagoya
City University, 3-1
Tanabe-dori, Mizuho-ku, Nagoya, Aichi 467-8603, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Naoki Miyata</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Pharmaceutical Sciences, Nagoya
City University, 3-1
Tanabe-dori, Mizuho-ku, Nagoya, Aichi 467-8603, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tamio Mizukami</span> - <span class="hlFld-Affiliation affiliation">Graduate
School of Bio-Science, Nagahama Institute
of Bio-Science and Technology, 1266 Tamura-cho, Nagahama, Shiga 526-0829, Japan</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d86e3711-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i53">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66371" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66371" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Mie Tsuchida and Miho Hosoi for their technical support. This work was supported in part by the JST CREST program (T.S.), a Grant-in-Aid for Scientific Research from the Japan Society for the Promotion of Science (T.S.), the Takeda Science Foundation (T.S.), the Mochida Memorial Foundation for Medical and Pharmaceutical Research (T.S.), the Terumo Foundation for Life Science and Arts (Y.I.), and Kyoto-Funding for Innovation in Health-related R&D Fields (Y.I.).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i54" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i54"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i55" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i55"> Abbreviations</h2><tr><td class="NLM_term">KDM</td><td class="NLM_def"><p class="first last">lysine demethylase</p></td></tr><tr><td class="NLM_term">LSD</td><td class="NLM_def"><p class="first last">lysine-specific demethylase</p></td></tr><tr><td class="NLM_term">FAD</td><td class="NLM_def"><p class="first last">flavin adenine dinucleotide</p></td></tr><tr><td class="NLM_term">PCPA</td><td class="NLM_def"><p class="first last">phenylcyclopropylamine</p></td></tr><tr><td class="NLM_term">JARID</td><td class="NLM_def"><p class="first last">Jumonji AT-rich interactive domain</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i56">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16180" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16180" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 22 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">â</span> <span class="NLM_lpage">705</span><span class="refDoi">Â DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0ljG_dBN6P1PGQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Liu, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. H.</span><span> </span><span class="NLM_article-title">An Iridium(III) Complex Inhibits JMJD2 Activities and Acts as a Potential Epigenetic Modulator</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">6697</span><span class="NLM_x">â</span> <span class="NLM_lpage">6703</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00375</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00375" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6697-6703&author=L.+J.+Liuauthor=L.+Luauthor=H.+J.+Zhongauthor=B.+Heauthor=D.+W.+Kwongauthor=D.+L.+Maauthor=C.+H.+Leung&title=An+Iridium%28III%29+Complex+Inhibits+JMJD2+Activities+and+Acts+as+a+Potential+Epigenetic+Modulator&doi=10.1021%2Facs.jmedchem.5b00375"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00375%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%2BJ.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DH.%2BJ.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DKwong%26aufirst%3DD.%2BW.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DAn%2520Iridium%2528III%2529%2520Complex%2520Inhibits%2520JMJD2%2520Activities%2520and%2520Acts%2520as%2520a%2520Potential%2520Epigenetic%2520Modulator%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6697%26epage%3D6703%26doi%3D10.1021%2Facs.jmedchem.5b00375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Zhong, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C. C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H. M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. H.</span><span> </span><span class="NLM_article-title">An iridium(III)-based irreversible proteinâprotein interaction inhibitor of BRD4 as a potent anticancer agent</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">5400</span><span class="NLM_x">â</span> <span class="NLM_lpage">5408</span><span class="refDoi">Â DOI: 10.1039/C5SC02321A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1039%2FC5SC02321A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=28757943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grtr7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5400-5408&author=H.+J.+Zhongauthor=L.+Luauthor=K.+H.+Leungauthor=C.+C.+L.+Wongauthor=C.+Pengauthor=S.+C.+Yanauthor=D.+L.+Maauthor=Z.+Caiauthor=H.+M.+D.+Wangauthor=C.+H.+Leung&title=An+iridium%28III%29-based+irreversible+protein%E2%80%93protein+interaction+inhibitor+of+BRD4+as+a+potent+anticancer+agent&doi=10.1039%2FC5SC02321A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">An iridium(III)-based irreversible protein-protein interaction inhibitor of BRD4 as a potent anticancer agent</span></div><div class="casAuthors">Zhong, Hai-Jing; Lu, Lihua; Leung, Ka-Ho; Wong, Catherine C. L.; Peng, Chao; Yan, Siu-Cheong; Ma, Dik-Lung; Cai, Zongwei; David Wang, Hui-Min; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5400-5408</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) has recently emerged as an attractive epigenetic target for anticancer therapy.  In this study, an iridium(III) complex is reported as the first metal-based, irreversible inhibitor of BRD4.  Complex 1a is able to antagonize the BRD4-acetylated histone protein-protein interaction (PPI) in vitro, and to bind BRD4 and down-regulate c-myc oncogenic expression in cellulo.  Chromatin immunopptn. (ChIP) anal. revealed that 1a could modulate the interaction between BRD4 and chromatin in melanoma cells, particular at the MYC promoter.  Finally, the complex showed potent activity against melanoma xenografts in an in vivo mouse model.  To our knowledge, this is the first report of a Group 9 metal complex inhibiting the PPI of a member of the bromodomain and extraterminal domain (BET) family.  We envision that complex 1a may serve as a useful scaffold for the development of more potent epigenetic agents against cancers such as melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqMXspNfXFlbVg90H21EOLACvtfcHk0liWde5-GLlDuw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grtr7E&md5=ae176cbfd3621dd7939c234d643db84e</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1039%2FC5SC02321A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC02321A%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.%2BJ.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DLeung%26aufirst%3DK.%2BH.%26aulast%3DWong%26aufirst%3DC.%2BC.%2BL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DS.%2BC.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DCai%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DH.%2BM.%2BD.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DAn%2520iridium%2528III%2529-based%2520irreversible%2520protein%25E2%2580%2593protein%2520interaction%2520inhibitor%2520of%2520BRD4%2520as%2520a%2520potent%2520anticancer%2520agent%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D5400%26epage%3D5408%26doi%3D10.1039%2FC5SC02321A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Schnekenburger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dicato, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, M.</span><span> </span><span class="NLM_article-title">Epigenetic modulators from â³The Big Blueâ³: a treasure to fight against cancer</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">351</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">â</span> <span class="NLM_lpage">197</span><span class="refDoi">Â DOI: 10.1016/j.canlet.2014.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.canlet.2014.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=24964131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGmtLbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2014&pages=182-197&author=M.+Schnekenburgerauthor=M.+Dicatoauthor=M.+Diederich&title=Epigenetic+modulators+from+%E2%80%B3The+Big+Blue%E2%80%B3%3A+a+treasure+to+fight+against+cancer&doi=10.1016%2Fj.canlet.2014.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2cR"><div class="casContent"><span class="casTitleNuber">2c</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic modulators from "The Big Blue": A treasure to fight against cancer</span></div><div class="casAuthors">Schnekenburger, Michael; Dicato, Mario; Diederich, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">182-197</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Cancer remains a major public health problem in our society.  The development of potent novel anti-cancer drugs selective for tumor cells is therefore still required.  Deregulation of the epigenetic machinery including DNA methylation, histone modifications and non-coding RNAs is a hallmark of cancer, which provides potential new therapeutic targets.  Natural products or their derivs. represent a major class of anti-cancer drugs in the arsenal available to the clinician.  However, regarding epigenetically active anti-cancer agents for clinics, the oceans represent a largely untapped resource.  This review focuses on marine natural compds. with epigenetic activities and their synthetic derivs. displaying anti-cancer properties including largazole, psammaplins, trichostatins and azumamides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvkq7FWSi1ALVg90H21EOLACvtfcHk0liWde5-GLlDuw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGmtLbJ&md5=8b354ad10012c11b443e5130bf4e86ce</span></div><a href="/servlet/linkout?suffix=cit2c&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DSchnekenburger%26aufirst%3DM.%26aulast%3DDicato%26aufirst%3DM.%26aulast%3DDiederich%26aufirst%3DM.%26atitle%3DEpigenetic%2520modulators%2520from%2520%25E2%2580%25B3The%2520Big%2520Blue%25E2%2580%25B3%253A%2520a%2520treasure%2520to%2520fight%2520against%2520cancer%26jtitle%3DCancer%2520Lett.%26date%3D2014%26volume%3D351%26spage%3D182%26epage%3D197%26doi%3D10.1016%2Fj.canlet.2014.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">LÃ¶tsch, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parnham, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geisslinger, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doehring, A.</span><span> </span><span class="NLM_article-title">Common non-epigenetic drugs as epigenetic modulators</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">â</span> <span class="NLM_lpage">753</span><span class="refDoi">Â DOI: 10.1016/j.molmed.2013.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.molmed.2013.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=24054876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FgtValtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=742-753&author=J.+L%C3%B6tschauthor=G.+Schneiderauthor=D.+Rekerauthor=M.+J.+Parnhamauthor=P.+Schneiderauthor=G.+Geisslingerauthor=A.+Doehring&title=Common+non-epigenetic+drugs+as+epigenetic+modulators&doi=10.1016%2Fj.molmed.2013.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2dR"><div class="casContent"><span class="casTitleNuber">2d</span><div class="casTitle"><span class="NLM_cas:atitle">Common non-epigenetic drugs as epigenetic modulators</span></div><div class="casAuthors">Lotsch Jorn; Schneider Gisbert; Reker Daniel; Parnham Michael J; Schneider Petra; Geisslinger Gerd; Doehring Alexandra</div><div class="citationInfo"><span class="NLM_cas:title">Trends in molecular medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">742-53</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetic effects are exerted by a variety of factors and evidence increases that common drugs such as opioids, cannabinoids, valproic acid, or cytostatics may induce alterations in DNA methylation patterns or histone conformations.  These effects occur via chemical structural interactions with epigenetic enzymes, through interactions with DNA repair mechanisms.  Computational predictions indicate that one-twentieth of all drugs might potentially interact with human histone deacetylase, which was prospectively experimentally verified for the compound with the highest predicted interaction probability.  These epigenetic effects add to wanted and unwanted drug effects, contributing to mechanisms of drug resistance or disease-related and unrelated phenotypes.  Because epigenetic changes might be transmitted to offspring, the need for reliable and cost-effective epigenetic screening tools becomes acute.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRjgfDAzktxzC-bLEOO4dYpfW6udTcc2ebjiG51svugTbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FgtValtA%253D%253D&md5=a01992b5ad8dac32dd0db860502ad25d</span></div><a href="/servlet/linkout?suffix=cit2d&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2013.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2013.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6tsch%26aufirst%3DJ.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DReker%26aufirst%3DD.%26aulast%3DParnham%26aufirst%3DM.%2BJ.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DGeisslinger%26aufirst%3DG.%26aulast%3DDoehring%26aufirst%3DA.%26atitle%3DCommon%2520non-epigenetic%2520drugs%2520as%2520epigenetic%2520modulators%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D742%26epage%3D753%26doi%3D10.1016%2Fj.molmed.2013.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinmann, H.</span><span> </span><span class="NLM_article-title">Small-molecule modulators for epigenetics targets</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1885</span><span class="NLM_x">â</span> <span class="NLM_lpage">1891</span><span class="refDoi">Â DOI: 10.1002/cmdc.201300344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1002%2Fcmdc.201300344" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1885-1891&author=S.+Knappauthor=H.+Weinmann&title=Small-molecule+modulators+for+epigenetics+targets&doi=10.1002%2Fcmdc.201300344"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2e&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300344%26sid%3Dliteratum%253Aachs%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DSmall-molecule%2520modulators%2520for%2520epigenetics%2520targets%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1885%26epage%3D1891%26doi%3D10.1002%2Fcmdc.201300344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">A crack in histone lysine methylation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">903</span><span class="NLM_x">â</span> <span class="NLM_lpage">906</span><span class="refDoi">Â DOI: 10.1016/j.cell.2004.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.cell.2004.12.006" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=903-906&author=S.+Kubicekauthor=T.+Jenuwein&title=A+crack+in+histone+lysine+methylation&doi=10.1016%2Fj.cell.2004.12.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DA%2520crack%2520in%2520histone%2520lysine%2520methylation%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D903%26epage%3D906%26doi%3D10.1016%2Fj.cell.2004.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">â</span> <span class="NLM_lpage">953</span><span class="refDoi">Â DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0ljPuEPkOknvyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallin, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">JmjC-domain-containing proteins and histone demethylation</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">715</span><span class="NLM_x">â</span> <span class="NLM_lpage">727</span><span class="refDoi">Â DOI: 10.1038/nrg1945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnrg1945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=16983801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1Ggu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=715-727&author=R.+J.+Kloseauthor=E.+M.+Kallinauthor=Y.+Zhang&title=JmjC-domain-containing+proteins+and+histone+demethylation&doi=10.1038%2Fnrg1945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">JmjC-domain-containing proteins and histone demethylation</span></div><div class="casAuthors">Klose, Robert J.; Kallin, Eric M.; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">715-727</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation has important roles in regulating gene expression and forms part of the epigenetic memory system that regulates cell fate and identity.  Enzymes that directly remove Me marks from histones have recently been identified, revealing a new level of plasticity within this epigenetic modification system.  Here we analyze the evolutionary relationship between Jumonji C (JmjC)-domain-contg. proteins and discuss their cellular functions in relation to their potential enzymic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPb9c8M0AJzLVg90H21EOLACvtfcHk0ljPuEPkOknvyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1Ggu74%253D&md5=1ac8a66426993770ae33b78ea35f5879</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1038%2Fnrg1945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1945%26sid%3Dliteratum%253Aachs%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DKallin%26aufirst%3DE.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DJmjC-domain-containing%2520proteins%2520and%2520histone%2520demethylation%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2006%26volume%3D7%26spage%3D715%26epage%3D727%26doi%3D10.1038%2Fnrg1945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone lysine demethylases: emerging roles in development, physiology and disease</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">829</span><span class="NLM_x">â</span> <span class="NLM_lpage">833</span><span class="refDoi">Â DOI: 10.1038/nrg2218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnrg2218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=17909537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOksL7P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=829-833&author=Y.+Shi&title=Histone+lysine+demethylases%3A+emerging+roles+in+development%2C+physiology+and+disease&doi=10.1038%2Fnrg2218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4cR"><div class="casContent"><span class="casTitleNuber">4c</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylases: emerging roles in development, physiology and disease</span></div><div class="casAuthors">Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">829-833</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Functions of histone lysine demethylases in a range of developmental and physiol. processes are rapidly being uncovered, as are the roles of these enzymes in disease.  Histone demethylases also provide a promising new route towards the therapeutic targeting of epigenetic regulators.  The discovery of an increasing no. of histone demethylases has highlighted the dynamic nature of the regulation of histone methylation, a key chromatin modification that is involved in eukaryotic genome and gene regulation.  A flurry of recent studies has offered glimpses into the specific biol. roles of these enzymes and their potential connections to human diseases.  These advances have also catalyzed a resurgence of interest in epigenetic regulators as potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ_dGIbAr6QLVg90H21EOLACvtfcHk0ljglFT3UhhhhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOksL7P&md5=d8bdeeb298c8afdcd9d0d5d5a62f6cc9</span></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.1038%2Fnrg2218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2218%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520lysine%2520demethylases%253A%2520emerging%2520roles%2520in%2520development%252C%2520physiology%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2007%26volume%3D8%26spage%3D829%26epage%3D833%26doi%3D10.1038%2Fnrg2218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooch, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">â</span> <span class="NLM_lpage">435</span><span class="refDoi">Â DOI: 10.1038/nature04021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnature04021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=16079794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=432-435&author=M.+G.+Leeauthor=C.+Wynderauthor=N.+Coochauthor=R.+Shiekhattar&title=An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation&doi=10.1038%2Fnature04021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Cooch, Neil; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">432-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors have previously described a multiprotein complex termed the BHC or BRAF-HDAC complex, which is required for the repression of neuronal-specific genes.  The authors have shown that the BHC complex is recruited by a neuronal silencer, REST (RE1-silencing transcription factor), and mediates the repression of REST-responsive genes.  BHC is a multiprotein complex consisting of two enzymic activities: a histone deacetylase (HDAC1 or 2) and a recently described histone demethylase (BHC110, also known as LSD1 or AOF2).  Here the authors show that BHC110-contg. complexes show a nearly fivefold increase in demethylation of histone H3 lysine 4 (H3K4) compared to recombinant BHC110.  Furthermore, recombinant BHC110 is unable to demethylate H3K4 on nucleosomes, but BHC110-contg. complexes readily demethylate nucleosomes.  In vitro reconstitution of the BHC complex using recombinant subunits reveals an essential role for the REST corepressor CoREST, not only in stimulating demethylation on core histones but also promoting demethylation of nucleosomal substrates.  The authors find that nucleosomal demethylation is the result of CoREST enhancing the assocn. between BHC110 and nucleosomes.  Depletion of CoREST in in vivo cell culture results in de-repression of REST-responsive gene expression and increased methylation of H3K4.  Together, these results highlight an essential role for CoREST in demethylation of H3K4 both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmTdZApayNhrVg90H21EOLACvtfcHk0ljglFT3UhhhhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D&md5=535a6d800bd6672192be133516b4dee7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature04021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04021%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DAn%2520essential%2520role%2520for%2520CoREST%2520in%2520nucleosomal%2520histone%25203%2520lysine%25204%2520demethylation%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D432%26epage%3D435%26doi%3D10.1038%2Fnature04021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SchÃ¼le, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: implications for therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">â</span> <span class="NLM_lpage">2071</span><span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+is+strongly+expressed+in+poorly+differentiated+neuroblastoma%3A+implications+for+therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0ljglFT3UhhhhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520strongly%2520expressed%2520in%2520poorly%2520differentiated%2520neuroblastoma%253A%2520implications%2520for%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Schenk, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">GÃ¶llner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebestreit, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popescu, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marton, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MÃ¼ller-Tidow, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zelent, A.</span><span> </span><span class="NLM_article-title">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">â</span> <span class="NLM_lpage">611</span><span class="refDoi">Â DOI: 10.1038/nm.2661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnm.2661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=22406747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=605-611&author=T.+Schenkauthor=W.+C.+Chenauthor=S.+G%C3%B6llnerauthor=L.+Howellauthor=L.+Jinauthor=K.+Hebestreitauthor=H.+U.+Kleinauthor=A.+C.+Popescuauthor=A.+Burnettauthor=K.+Millsauthor=R.+A.+Caseroauthor=L.+Martonauthor=P.+Wosterauthor=M.+D.+Mindenauthor=M.+Dugasauthor=J.+C.+Wangauthor=J.+E.+Dickauthor=C.+M%C3%BCller-Tidowauthor=K.+Petrieauthor=A.+Zelent&title=Inhibition+of+the+LSD1+%28KDM1A%29+demethylase+reactivates+the+all-trans-retinoic+acid+differentiation+pathway+in+acute+myeloid+leukemia&doi=10.1038%2Fnm.2661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span></div><div class="casAuthors">Schenk, Tino; Chen, Weihsu Claire; Goellner, Stefanie; Howell, Louise; Jin, Liqing; Hebestreit, Katja; Klein, Hans-Ulrich; Popescu, Andreea C.; Burnett, Alan; Mills, Ken; Casero, Robert A.; Marton, Laurence; Woster, Patrick; Minden, Mark D.; Dugas, Martin; Wang, Jean C. Y.; Dick, John E.; Mueller-Tidow, Carsten; Petrie, Kevin; Zelent, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">605-611</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acute promyelocytic leukemia (APL), a cytogenetically distinct subtype of acute myeloid leukemia (AML), characterized by the t(15;17)-assocd. PML-RARA fusion, has been successfully treated with therapy utilizing all-trans-retinoic acid (ATRA) to differentiate leukemic blasts.  However, among patients with non-APL AML, ATRA-based treatment has not been effective.  Here we show that, through epigenetic reprogramming, inhibitors of lysine-specific demethylase 1 (LSD1, also called KDM1A), including tranylcypromine (TCP), unlocked the ATRA-driven therapeutic response in non-APL AML.  LSD1 inhibition did not lead to a large-scale increase in histone 3 Lys4 dimethylation (H3K4me2) across the genome, but it did increase H3K4me2 and expression of myeloid-differentiation-assocd. genes.  Notably, treatment with ATRA plus TCP markedly diminished the engraftment of primary human AML cells in vivo in nonobese diabetic (NOD)-severe combined immunodeficient (SCID) mice, suggesting that ATRA in combination with TCP may target leukemia-initiating cells.  Furthermore, initiation of ATRA plus TCP treatment 15 d after engraftment of human AML cells in NOD-SCID Î³ (with interleukin-2 (IL-2) receptor Î³ chain deficiency) mice also revealed the ATRA plus TCP drug combination to have a potent anti-leukemic effect that was superior to treatment with either drug alone.  These data identify LSD1 as a therapeutic target and strongly suggest that it may contribute to AML pathogenesis by inhibiting the normal pro-differentiative function of ATRA, paving the way for new combinatorial therapies for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU7ebz8Ar4AbVg90H21EOLACvtfcHk0ljjT81hL1zCFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D&md5=57d968de189123149a2714df0208725f</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fnm.2661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2661%26sid%3Dliteratum%253Aachs%26aulast%3DSchenk%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DW.%2BC.%26aulast%3DG%25C3%25B6llner%26aufirst%3DS.%26aulast%3DHowell%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHebestreit%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DH.%2BU.%26aulast%3DPopescu%26aufirst%3DA.%2BC.%26aulast%3DBurnett%26aufirst%3DA.%26aulast%3DMills%26aufirst%3DK.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DMarton%26aufirst%3DL.%26aulast%3DWoster%26aufirst%3DP.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DDugas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BC.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DM%25C3%25BCller-Tidow%26aufirst%3DC.%26aulast%3DPetrie%26aufirst%3DK.%26aulast%3DZelent%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520the%2520LSD1%2520%2528KDM1A%2529%2520demethylase%2520reactivates%2520the%2520all-trans-retinoic%2520acid%2520differentiation%2520pathway%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D605%26epage%3D611%26doi%3D10.1038%2Fnm.2661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogel, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayanan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristie, T. M.</span><span> </span><span class="NLM_article-title">Inhibition of the histone demethylase LSD1 blocks Î±-herpesvirus lytic replication and reactivation from latency</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1312</span><span class="NLM_x">â</span> <span class="NLM_lpage">1317</span><span class="refDoi">Â DOI: 10.1038/nm.2051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnm.2051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=19855399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSqsLbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=1312-1317&author=Y.+Liangauthor=J.+L.+Vogelauthor=A.+Narayananauthor=H.+Pengauthor=T.+M.+Kristie&title=Inhibition+of+the+histone+demethylase+LSD1+blocks+%CE%B1-herpesvirus+lytic+replication+and+reactivation+from+latency&doi=10.1038%2Fnm.2051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the histone demethylase LSD1 blocks Î±-herpesvirus lytic replication and reactivation from latency</span></div><div class="casAuthors">Liang, Yu; Vogel, Jodi L.; Narayanan, Aarthi; Peng, Hua; Kristie, Thomas M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1312-1317</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reversible methylation of histone tails serves as either a pos. signal recognized by transcriptional assemblies or a neg. signal that result in repression.  Invading viral pathogens that depend upon the host cell's transcriptional app. are also subject to the regulatory impact of chromatin assembly and modifications.  Here we show that infection by the Î±-herpesviruses, herpes simplex virus (HSV) and varicella zoster virus (VZV), results in the rapid accumulation of chromatin bearing repressive histone H3 Lys9 methylation.  To enable expression of viral immediate early (IE) genes, both viruses use the cellular transcriptional coactivator host cell factor-1 (HCF-1) to recruit the lysine-specific demethylase-1 (LSD1) to the viral immediate early promoters.  Depletion of LSD1 or inhibition of its activity with monoamine oxidase inhibitors (MAOIs) results in the accumulation of repressive chromatin and a block to viral gene expression.  As HCF-1 is a component of the Set1 and MLL1 histone H3 Lys4 methyltransferase complexes, it thus coordinates modulation of repressive H3 Lys9 methylation levels with addn. of activating H3 Lys4 trimethylation marks.  Strikingly, MAOIs also block the reactivation of HSV from latency in sensory neurons, indicating that the HCF-1 complex is a crucial component of the reactivation mechanism.  The results support pharmaceutical control of histone modifying enzymes as a strategy for controlling herpesvirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppT5vH8lPRnbVg90H21EOLACvtfcHk0ljjT81hL1zCFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSqsLbF&md5=bd9aacfdbcba54e30f9482f8fcc96777</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1038%2Fnm.2051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2051%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DVogel%26aufirst%3DJ.%2BL.%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DPeng%26aufirst%3DH.%26aulast%3DKristie%26aufirst%3DT.%2BM.%26atitle%3DInhibition%2520of%2520the%2520histone%2520demethylase%2520LSD1%2520blocks%2520%25CE%25B1-herpesvirus%2520lytic%2520replication%2520and%2520reactivation%2520from%2520latency%26jtitle%3DNat.%2520Med.%26date%3D2009%26volume%3D15%26spage%3D1312%26epage%3D1317%26doi%3D10.1038%2Fnm.2051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanabe, O.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is a therapeutic target for fetal hemoglobin induction</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">â</span> <span class="NLM_lpage">294</span><span class="refDoi">Â DOI: 10.1038/nm.3101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnm.3101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=23416702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Wrurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=291-294&author=L.+Shiauthor=S.+Cuiauthor=J.+D.+Engelauthor=O.+Tanabe&title=Lysine-specific+demethylase+1+is+a+therapeutic+target+for+fetal+hemoglobin+induction&doi=10.1038%2Fnm.3101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is a therapeutic target for fetal hemoglobin induction</span></div><div class="casAuthors">Shi, Lihong; Cui, Shuaiying; Engel, James D.; Tanabe, Osamu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">291-294</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Enhanced fetal Î³-globin synthesis alleviates symptoms of Î²-globinopathies such as sickle cell disease and Î²-thalassemia, but current Î³-globin-inducing drugs offer limited beneficial effects.  We show here that lysine-specific demethylase 1 (LSD1) inhibition by RNAi in human erythroid cells or by the monoamine oxidase inhibitor tranylcypromine in human erythroid cells or Î²-type globin-transgenic mice enhances Î³-globin expression.  LSD1 is thus a promising therapeutic target for Î³-globin induction, and tranylcypromine may serve as a lead compd. for the development of a new Î³-globin inducer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPuLk6KrTQ-rVg90H21EOLACvtfcHk0ljjT81hL1zCFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Wrurs%253D&md5=3747256331d40eb16b2ce54a85df1761</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1038%2Fnm.3101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3101%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DS.%26aulast%3DEngel%26aufirst%3DJ.%2BD.%26aulast%3DTanabe%26aufirst%3DO.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520a%2520therapeutic%2520target%2520for%2520fetal%2520hemoglobin%2520induction%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D291%26epage%3D294%26doi%3D10.1038%2Fnm.3101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine demethylases inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">â</span> <span class="NLM_lpage">8250</span><span class="refDoi">Â DOI: 10.1021/jm201048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+demethylases+inhibitors&doi=10.1021%2Fjm201048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0lhjoKc-JuxpCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520demethylases%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250%26doi%3D10.1021%2Fjm201048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chi, Y. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">The SNAG domain of Snail1 functions as a molecular hook for recruiting lysine-specific demethylase 1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1803</span><span class="NLM_x">â</span> <span class="NLM_lpage">1816</span><span class="refDoi">Â DOI: 10.1038/emboj.2010.63</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Femboj.2010.63" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=1803-1816&author=Y.+Linauthor=Y.+Wuauthor=J.+Liauthor=C.+Dongauthor=X.+Yeauthor=Y.+I.+Chiauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=The+SNAG+domain+of+Snail1+functions+as+a+molecular+hook+for+recruiting+lysine-specific+demethylase+1&doi=10.1038%2Femboj.2010.63"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.63&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.63%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DC.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DChi%26aufirst%3DY.%2BI.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DThe%2520SNAG%2520domain%2520of%2520Snail1%2520functions%2520as%2520a%2520molecular%2520hook%2520for%2520recruiting%2520lysine-specific%2520demethylase%25201%26jtitle%3DEMBO%2520J.%26date%3D2010%26volume%3D29%26spage%3D1803%26epage%3D1816%26doi%3D10.1038%2Femboj.2010.63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit9b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span> </span><span class="NLM_article-title">Requirement of the histone demethylase LSD1 in Snai1-mediated transcriptional repression during epithelial-mesenchymal transition</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">4896</span><span class="NLM_x">â</span> <span class="NLM_lpage">4904</span><span class="refDoi">Â DOI: 10.1038/onc.2010.234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fonc.2010.234" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=4896-4904&author=T.+Linauthor=A.+Ponnauthor=X.+Huauthor=B.+K.+Lawauthor=J.+Lu&title=Requirement+of+the+histone+demethylase+LSD1+in+Snai1-mediated+transcriptional+repression+during+epithelial-mesenchymal+transition&doi=10.1038%2Fonc.2010.234"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.234%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DT.%26aulast%3DPonn%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLaw%26aufirst%3DB.%2BK.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DRequirement%2520of%2520the%2520histone%2520demethylase%2520LSD1%2520in%2520Snai1-mediated%2520transcriptional%2520repression%2520during%2520epithelial-mesenchymal%2520transition%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26spage%3D4896%26epage%3D4904%26doi%3D10.1038%2Fonc.2010.234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Ferrari-Amorotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fragliasso, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esteki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prudente, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soliera, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cattelani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manzotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grisendi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieraccioli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">RaschellÃ , G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiodoni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calabretta, B.</span><span> </span><span class="NLM_article-title">Inhibiting interactions of lysine demethylase LSD1 with snail/slug blocks cancer cell invasion</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">â</span> <span class="NLM_lpage">245</span><span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-12-1739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1158%2F0008-5472.CAN-12-1739" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=235-245&author=G.+Ferrari-Amorottiauthor=V.+Fragliassoauthor=R.+Estekiauthor=Z.+Prudenteauthor=A.+R.+Solieraauthor=S.+Cattelaniauthor=G.+Manzottiauthor=G.+Grisendiauthor=M.+Dominiciauthor=M.+Pieraccioliauthor=G.+Raschell%C3%A0author=C.+Chiodoniauthor=M.+P.+Colomboauthor=B.+Calabretta&title=Inhibiting+interactions+of+lysine+demethylase+LSD1+with+snail%2Fslug+blocks+cancer+cell+invasion&doi=10.1158%2F0008-5472.CAN-12-1739"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-1739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-1739%26sid%3Dliteratum%253Aachs%26aulast%3DFerrari-Amorotti%26aufirst%3DG.%26aulast%3DFragliasso%26aufirst%3DV.%26aulast%3DEsteki%26aufirst%3DR.%26aulast%3DPrudente%26aufirst%3DZ.%26aulast%3DSoliera%26aufirst%3DA.%2BR.%26aulast%3DCattelani%26aufirst%3DS.%26aulast%3DManzotti%26aufirst%3DG.%26aulast%3DGrisendi%26aufirst%3DG.%26aulast%3DDominici%26aufirst%3DM.%26aulast%3DPieraccioli%26aufirst%3DM.%26aulast%3DRaschell%25C3%25A0%26aufirst%3DG.%26aulast%3DChiodoni%26aufirst%3DC.%26aulast%3DColombo%26aufirst%3DM.%2BP.%26aulast%3DCalabretta%26aufirst%3DB.%26atitle%3DInhibiting%2520interactions%2520of%2520lysine%2520demethylase%2520LSD1%2520with%2520snail%252Fslug%2520blocks%2520cancer%2520cell%2520invasion%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D235%26epage%3D245%26doi%3D10.1158%2F0008-5472.CAN-12-1739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrello, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiarelli, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vellore, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâConnell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1677</span><span class="NLM_x">â</span> <span class="NLM_lpage">1682</span><span class="refDoi">Â DOI: 10.1021/cb4001926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4001926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1677-1682&author=M.+Tortoriciauthor=M.+T.+Borrelloauthor=M.+Tardugnoauthor=L.+R.+Chiarelliauthor=S.+Pilottoauthor=G.+Ciossaniauthor=N.+A.+Velloreauthor=S.+G.+Baileyauthor=J.+Cowanauthor=M.+O%E2%80%99Connellauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Maiauthor=R.+Baronauthor=A.+Ganesanauthor=A.+Mattevi&title=Protein+recognition+by+short+peptide+reversible+inhibitors+of+the+chromatin-modifying+LSD1%2FCoREST+lysine+demethylase&doi=10.1021%2Fcb4001926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span></div><div class="casAuthors">Tortorici, Marcello; Borrello, Maria Teresa; Tardugno, Maria; Chiarelli, Laurent R.; Pilotto, Simona; Ciossani, Giuseppe; Vellore, Nadeem A.; Bailey, Sarah G.; Cowan, Jonathan; O'Connell, Maria; Crabb, Simon J.; Packham, Graham; Mai, Antonello; Baron, Riccardo; Ganesan, A.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1677-1682</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The combinatorial assembly of protein complexes is at the heart of chromatin biol.  Lysine demethylase LSD1(KDM1A)/CoREST beautifully exemplifies this concept.  The active site of the enzyme tightly assocs. to the N-terminal domain of transcription factors of the SNAIL1 family, which therefore can competitively inhibit the binding of the N-terminal tail of the histone substrate.  Our enzymic, crystallog., spectroscopic, and computational studies reveal that LSD1/CoREST can bind to a hexapeptide derived from the SNAIL sequence through recognition of a pos. charged Î±-helical turn that forms upon binding to the enzyme.  Variations in sequence and length of this six amino acid ligand modulate affinities enabling the same binding site to differentially interact with proteins that exert distinct biol. functions.  The discovered short peptide inhibitors exhibit antiproliferative activities and lay the foundation for the development of peptidomimetic small mol. inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7SufQOwK-y7Vg90H21EOLACvtfcHk0lhjoKc-JuxpCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D&md5=a8479b513e392e8acb78979e116dcffa</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb4001926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4001926%26sid%3Dliteratum%253Aachs%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DBorrello%26aufirst%3DM.%2BT.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DChiarelli%26aufirst%3DL.%2BR.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DVellore%26aufirst%3DN.%2BA.%26aulast%3DBailey%26aufirst%3DS.%2BG.%26aulast%3DCowan%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DM.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DProtein%2520recognition%2520by%2520short%2520peptide%2520reversible%2520inhibitors%2520of%2520the%2520chromatin-modifying%2520LSD1%252FCoREST%2520lysine%2520demethylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1677%26epage%3D1682%26doi%3D10.1021%2Fcb4001926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Small-molecular modulators of cancer-associated epigenetic mechanisms</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">â</span> <span class="NLM_lpage">896</span><span class="refDoi">Â DOI: 10.1039/c3mb25410k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1039%2Fc3mb25410k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=23511667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVOlsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=873-896&author=Y.+Itohauthor=T.+Suzukiauthor=N.+Miyata&title=Small-molecular+modulators+of+cancer-associated+epigenetic+mechanisms&doi=10.1039%2Fc3mb25410k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecular modulators of cancer-associated epigenetic mechanisms</span></div><div class="casAuthors">Itoh, Yukihiro; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">873-896</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  DNA methylation and posttranslational histone modifications regulate expression of various genes independently of changes in the DNA sequence.  Such epigenetic mechanisms play important roles in controlling cellular functions, including the cell cycle, immunoresponses and signal transduction.  On the other hand, epigenetic aberrations are assocd. with oncogenesis and proliferation of cancer cells, and epigenetic alterations have been identified in many human cancer cells.  Furthermore, chem.-biol. approaches have uncovered relationships between epigenetic dysregulation and cancer, and several small mols. that modulate epigenetic mechanisms have already been approved for cancer therapy.  In this review, we deal with chem. epigenetics relevant to cancer therapy, focusing esp. on small mols. that regulate epigenetic mechanisms related to DNA methylation and histone modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo76xFXo9kCe7Vg90H21EOLACvtfcHk0lh8MEdPnQU-DA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVOlsr4%253D&md5=ecc7cd46637f6010872510329d51ac90</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1039%2Fc3mb25410k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3mb25410k%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSmall-molecular%2520modulators%2520of%2520cancer-associated%2520epigenetic%2520mechanisms%26jtitle%3DMol.%2520BioSyst.%26date%3D2013%26volume%3D9%26spage%3D873%26epage%3D896%26doi%3D10.1039%2Fc3mb25410k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title"><i>trans</i>-2-Phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">4408</span><span class="NLM_x">â</span> <span class="NLM_lpage">4416</span><span class="refDoi">Â DOI: 10.1021/bi0618621</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0618621" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=4408-4416&author=D.+M.+Schmidtauthor=D.+G.+McCafferty&title=trans-2-Phenylcyclopropylamine+is+a+mechanism-based+inactivator+of+the+histone+demethylase+LSD1&doi=10.1021%2Fbi0618621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">trans-2-Phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span></div><div class="casAuthors">Schmidt, Dawn M. Z.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4408-4416</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic domain of the flavin-dependent human histone demethylase, lysine-specific demethylase 1 (LSD1), belongs to the family of amine oxidases including polyamine oxidase and monoamine oxidase (MAO).  The authors previously assessed MAO inhibitors (MAOIs) for their ability to inhibit the reaction catalyzed by LSD1, demonstrating that trans-2-phenylcyclopropylamine (2-PCPA; tranylcypromine, Parnate) was the most potent inhibitor with respect to LSD1.  Here, the authors showed that 2-PCPA was a time-dependent, mechanism-based irreversible inhibitor of LSD1 with a Ki of 242 Î¼M and a kinact of 0.0106 s-1.  2-PCPA showed limited selectivity for human MAOs vs. LSD1, with kinact/Ki values only 16-fold and 2.4-fold higher for MAO B and MAO A, resp.  Profiles of LSD1 activity and inactivation by 2-PCPA as a function of pH were consistent with a mechanism of inactivation dependent upon enzyme catalysis.  Mass spectrometry supported a role for FAD as the site of covalent modification by 2-PCPA.  These results will provide a foundation for the design of cyclopropylamine-based inhibitors that are selective for LSD1 to probe its role in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVK59U7IsBvbVg90H21EOLACvtfcHk0lh8MEdPnQU-DA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D&md5=02925342b86c7dbfb5f1a99fa5264adc</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1021%2Fbi0618621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0618621%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3Dtrans-2-Phenylcyclopropylamine%2520is%2520a%2520mechanism-based%2520inactivator%2520of%2520the%2520histone%2520demethylase%2520LSD1%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D4408%26epage%3D4416%26doi%3D10.1021%2Fbi0618621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jalili, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ball, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span> </span><span class="NLM_article-title">Structural basis for the inhibition of the LSD1 histone demethylase by the antidepressant <i>trans</i>-2-phenylcyclopropylamine</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">8058</span><span class="NLM_x">â</span> <span class="NLM_lpage">8065</span><span class="refDoi">Â DOI: 10.1021/bi700664y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi700664y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsVKmsr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=8058-8065&author=M.+Yangauthor=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=P.+Jaliliauthor=H.+L.+Ballauthor=M.+Machiusauthor=P.+A.+Coleauthor=H.+Yu&title=Structural+basis+for+the+inhibition+of+the+LSD1+histone+demethylase+by+the+antidepressant+trans-2-phenylcyclopropylamine&doi=10.1021%2Fbi700664y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12cR"><div class="casContent"><span class="casTitleNuber">12c</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Inhibition of the LSD1 Histone Demethylase by the Antidepressant trans-2-Phenylcyclopropylamine</span></div><div class="casAuthors">Yang, Maojun; Culhane, Jeffrey C.; Szewczuk, Lawrence M.; Jalili, Pegah; Ball, Haydn L.; Machius, Mischa; Cole, Philip A.; Yu, Hongtao</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8058-8065</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone modifications, such as acetylation and methylation, are important epigenetic marks that regulate diverse biol. processes that use chromatin as the template, including transcription.  Dysregulation of histone acetylation and methylation leads to the silencing of tumor suppressor genes and contributes to cancer progression.  Inhibitors of enzymes that catalyze the addn. and removal of these epigenetic marks thus have therapeutic potential for treating cancer.  Lysine-specific demethylase 1 (LSD1) is the first discovered histone lysine demethylase and with the help of its cofactor CoREST, specifically demethylates mono- and dimethylated histone H3 lysine 4 (H3-K4), thus repressing transcription.  Because LSD1 belongs to the family of FAD (FAD)-dependent amine oxidases, certain inhibitors of monoamine oxidases (MAOs), including the clin. used antidepressant trans-2-phenylcyclopropylamine (PCPA; tranylcypromine; Parnate), are also capable of inhibiting LSD1.  In this study, we have further measured the kinetic parameters of the inhibition of LSD1 by PCPA and detd. the crystal structure of LSD1-CoREST in the presence of PCPA.  Our structural and mass spectrometry analyses are consistent with PCPA forming a covalent adduct with FAD in LSD1 that is distinct from the FAD-PCPA adduct of MAO B.  The structure also reveals that the Ph ring of the FAD-PCPA adduct in LSD1 does not form extensive interactions with active-site residues.  This study thus provides the basis for designing more potent inhibitors of LSD1 that contain substitutions on the Ph ring of PCPA to fully engage neighboring residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLqcLmzpB1S7Vg90H21EOLACvtfcHk0lh8MEdPnQU-DA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsVKmsr8%253D&md5=e11fe93832d46962b6c4848f682042d2</span></div><a href="/servlet/linkout?suffix=cit12c&amp;dbid=16384&amp;doi=10.1021%2Fbi700664y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi700664y%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DJalili%26aufirst%3DP.%26aulast%3DBall%26aufirst%3DH.%2BL.%26aulast%3DMachius%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DH.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibition%2520of%2520the%2520LSD1%2520histone%2520demethylase%2520by%2520the%2520antidepressant%2520trans-2-phenylcyclopropylamine%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D8058%26epage%3D8065%26doi%3D10.1021%2Fbi700664y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Identification of cell-active lysine specific demethylase 1-selective inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">17536</span><span class="NLM_x">â</span> <span class="NLM_lpage">17537</span><span class="refDoi">Â DOI: 10.1021/ja907055q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja907055q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVWrtbzO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=17536-17537&author=R.+Uedaauthor=T.+Suzukiauthor=K.+Minoauthor=H.+Tsumotoauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyata&title=Identification+of+cell-active+lysine+specific+demethylase+1-selective+inhibitors&doi=10.1021%2Fja907055q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12dR"><div class="casContent"><span class="casTitleNuber">12d</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of cell-active lysine specific demethylase 1-selective inhibitors</span></div><div class="casAuthors">Ueda, Rie; Suzuki, Takayoshi; Mino, Koshiki; Tsumoto, Hiroki; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">17536-17537</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated Lys4 of histone H3 (H3K4).  Here we report the identification of the first small-mol. LSD1-selective inhibitors.  These inhibitors show in vivo H3K4-methylating activity and antiproliferative activity and should be useful as lead structures for anticancer drugs and as tools for studying the biol. roles of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH7uDFiErmKbVg90H21EOLACvtfcHk0lg8MuffO-PejA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVWrtbzO&md5=243e81a1d8d8cb3747f8307d5f42cd29</span></div><a href="/servlet/linkout?suffix=cit12d&amp;dbid=16384&amp;doi=10.1021%2Fja907055q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja907055q%26sid%3Dliteratum%253Aachs%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520cell-active%2520lysine%2520specific%2520demethylase%25201-selective%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D17536%26epage%3D17537%26doi%3D10.1021%2Fja907055q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsubara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Synthesis and biological activity of optically active NCL-1, a lysine-specific demethylase 1 selective inhibitor</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3702</span><span class="NLM_x">â</span> <span class="NLM_lpage">3708</span><span class="refDoi">Â DOI: 10.1016/j.bmc.2010.12.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.bmc.2010.12.024" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3702-3708&author=D.+Ogasawaraauthor=T.+Suzukiauthor=K.+Minoauthor=R.+Uedaauthor=M.+N.+Khanauthor=T.+Matsubaraauthor=K.+Kosekiauthor=M.+Hasegawaauthor=R.+Sasakiauthor=H.+Nakagawaauthor=T.+Mizukamiauthor=N.+Miyata&title=Synthesis+and+biological+activity+of+optically+active+NCL-1%2C+a+lysine-specific+demethylase+1+selective+inhibitor&doi=10.1016%2Fj.bmc.2010.12.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12e&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.12.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.12.024%26sid%3Dliteratum%253Aachs%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DKhan%26aufirst%3DM.%2BN.%26aulast%3DMatsubara%26aufirst%3DT.%26aulast%3DKoseki%26aufirst%3DK.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSynthesis%2520and%2520biological%2520activity%2520of%2520optically%2520active%2520NCL-1%252C%2520a%2520lysine-specific%2520demethylase%25201%2520selective%2520inhibitor%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3702%26epage%3D3708%26doi%3D10.1016%2Fj.bmc.2010.12.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12f"><span><span class="NLM_label">(f) </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuhara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1-selective inactivators: protein-targeted drug delivery mechanism</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">8620</span><span class="NLM_x">â</span> <span class="NLM_lpage">8624</span><span class="refDoi">Â DOI: 10.1002/anie.201303999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1002%2Fanie.201303999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=8620-8624&author=D.+Ogasawaraauthor=Y.+Itohauthor=H.+Tsumotoauthor=T.+Kakizawaauthor=K.+Minoauthor=K.+Fukuharaauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyataauthor=T.+Suzuki&title=Lysine-specific+demethylase+1-selective+inactivators%3A+protein-targeted+drug+delivery+mechanism&doi=10.1002%2Fanie.201303999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12fR"><div class="casContent"><span class="casTitleNuber">12f</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1-selective inactivators: Protein-targeted drug delivery mechanism</span></div><div class="casAuthors">Ogasawara, Daisuke; Itoh, Yukihiro; Tsumoto, Hiroki; Kakizawa, Taeko; Mino, Koshiki; Fukuhara, Kiyoshi; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">8620-8624</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of novel LSD1 (LSD1 = lysine-specific demethylase) inactivators based on new concept of LSD1-targeted PCPA (PCPA = trans-2-phenylcyclopropylamine) delivery have been designed.  This concept was tested with peptide 1a [1a = (ART-X-QTARKSTGGKAPRKQLA)] X = (I) and confirmed that 1a does indeed inhibit LSD1 selectively and efficiently by delivering PCPA directly to the LSD1 active site.  The enzyme is then inactivated by FAD-PCPA adduct formation, as in the case of PCPA itself.  This strategy could be applied to the design of nonpeptide, small-mol. LSD1 inactivators.  Mechanistic studies and biol. evaluations revealed that these small-mol. compds. are potent LSD1-selective inactivators (comparable to peptide 1a) in enzyme assays, and they also exhibit potent cell growth inhibitory activities against cancer cell lines.  This is the first time that a drug-carrier-type system has been employed to target a specific mol.  These novel LSD1-selective inactivators are considered to be candidates for anticancer agents, as well as bioprobes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqQOLhLwmv9rVg90H21EOLACvtfcHk0lg8MuffO-PejA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF&md5=71bd05a50aeb73431444f2588fba2dac</span></div><a href="/servlet/linkout?suffix=cit12f&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303999%26sid%3Dliteratum%253Aachs%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DKakizawa%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DFukuhara%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DLysine-specific%2520demethylase%25201-selective%2520inactivators%253A%2520protein-targeted%2520drug%2520delivery%2520mechanism%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D8620%26epage%3D8624%26doi%3D10.1002%2Fanie.201303999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12g"><span><span class="NLM_label">(g) </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">â</span> <span class="NLM_lpage">6833</span><span class="refDoi">Â DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12gR"><div class="casContent"><span class="casTitleNuber">12g</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0lg8MuffO-PejA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit12g&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12h"><span><span class="NLM_label">(h) </span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally designed <i>trans</i>-2-phenylcyclopropylamine derivatives potently inhibit histone demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">â</span> <span class="NLM_lpage">6503</span><span class="refDoi">Â DOI: 10.1021/bi100299r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12h&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+designed+trans-2-phenylcyclopropylamine+derivatives+potently+inhibit+histone+demethylase+LSD1%2FKDM1&doi=10.1021%2Fbi100299r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12hR"><div class="casContent"><span class="casTitleNuber">12h</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0ljNveSNZ9WVew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit12h&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520designed%2520trans-2-phenylcyclopropylamine%2520derivatives%2520potently%2520inhibit%2520histone%2520demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503%26doi%3D10.1021%2Fbi100299r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12i"><span><span class="NLM_label">(i) </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Comparative analysis of small molecules and histone substrate analogues as LSD1 lysine demethylase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">3164</span><span class="NLM_x">â</span> <span class="NLM_lpage">3176</span><span class="refDoi">Â DOI: 10.1021/ja909996p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja909996p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=3164-3176&author=J.+C.+Culhaneauthor=D.+Wangauthor=P.+M.+Yenauthor=P.+A.+Cole&title=Comparative+analysis+of+small+molecules+and+histone+substrate+analogues+as+LSD1+lysine+demethylase+inhibitors&doi=10.1021%2Fja909996p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12i&amp;dbid=16384&amp;doi=10.1021%2Fja909996p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja909996p%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DYen%26aufirst%3DP.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DComparative%2520analysis%2520of%2520small%2520molecules%2520and%2520histone%2520substrate%2520analogues%2520as%2520LSD1%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D3164%26epage%3D3176%26doi%3D10.1021%2Fja909996p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12j"><span><span class="NLM_label">(j) </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MÃ¼ller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">GÃ¼nther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SchÃ¼le, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">â</span> <span class="NLM_lpage">439</span><span class="refDoi">Â DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12j&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12j&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12j&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12jR"><div class="casContent"><span class="casTitleNuber">12j</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0ljNveSNZ9WVew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit12j&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12k"><span><span class="NLM_label">(k) </span><span class="NLM_contrib-group">Schmitt, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pippel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Fincke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yiu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SchÃ¼le, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Nonpeptidic propargylamines as inhibitors of lysine specific demethylase 1 (LSD1) with cellular activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7334</span><span class="NLM_x">â</span> <span class="NLM_lpage">7342</span><span class="refDoi">Â DOI: 10.1021/jm400792m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400792m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12k&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7334-7342&author=M.+L.+Schmittauthor=A.+T.+Hauserauthor=L.+Carlinoauthor=M.+Pippelauthor=J.+Schulz-Finckeauthor=E.+Metzgerauthor=D.+Willmannauthor=T.+Yiuauthor=M.+Bartonauthor=R.+Sch%C3%BCleauthor=W.+Sipplauthor=M.+Jung&title=Nonpeptidic+propargylamines+as+inhibitors+of+lysine+specific+demethylase+1+%28LSD1%29+with+cellular+activity&doi=10.1021%2Fjm400792m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12kR"><div class="casContent"><span class="casTitleNuber">12k</span><div class="casTitle"><span class="NLM_cas:atitle">Nonpeptidic Propargylamines as Inhibitors of Lysine Specific Demethylase 1 (LSD1) with Cellular Activity</span></div><div class="casAuthors">Schmitt, Martin L.; Hauser, Alexander-Thomas; Carlino, Luca; Pippel, Martin; Schulz-Fincke, Johannes; Metzger, Eric; Willmann, Dominica; Yiu, Teresa; Barton, Michelle; Schuele, Roland; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7334-7342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine demethylases play an important role in epigenetic regulation and thus in the development of diseases like cancer or neurodegenerative disorders.  As the lysine specific demethylase 1 (LSD1/KDM1) has been strongly connected to androgen and estrogen dependent gene expression, it serves as a promising target for the therapy of hormone dependent cancer.  Here, we report on the discovery of new small mol. inhibitors of LSD1 contg. a propargylamine warhead, starting out from lysine contg. substrate analogs.  On the basis of these substrate mimicking inhibitors, we were able to increase potency by a combination of similarity-based virtual screening and subsequent synthetic optimization resulting in more druglike LSD1 inhibitors that led to histone hypermethylation in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRGRr4rUPfsLVg90H21EOLACvtfcHk0lgFWiabr63bXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO&md5=d4de914dec37f011a7b9488ec539a395</span></div><a href="/servlet/linkout?suffix=cit12k&amp;dbid=16384&amp;doi=10.1021%2Fjm400792m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400792m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DM.%2BL.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DPippel%26aufirst%3DM.%26aulast%3DSchulz-Fincke%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DYiu%26aufirst%3DT.%26aulast%3DBarton%26aufirst%3DM.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DNonpeptidic%2520propargylamines%2520as%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520with%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7334%26epage%3D7342%26doi%3D10.1021%2Fjm400792m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12l"><span><span class="NLM_label">(l) </span><span class="NLM_contrib-group">Prusevich, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalin, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ming, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basso, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cieniewicz, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsiao, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adejola, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avery, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taverna, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tackett, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratan, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, O. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">A selective phenelzine analogue inhibitor of histone demethylase LSD1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1284</span><span class="NLM_x">â</span> <span class="NLM_lpage">1293</span><span class="refDoi">Â DOI: 10.1021/cb500018s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500018s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12l&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1284-1293&author=P.+Prusevichauthor=J.+H.+Kalinauthor=S.+A.+Mingauthor=M.+Bassoauthor=J.+Givensauthor=X.+Liauthor=J.+Huauthor=M.+S.+Taylorauthor=A.+M.+Cieniewiczauthor=P.+Y.+Hsiaoauthor=R.+Huangauthor=H.+Robersonauthor=N.+Adejolaauthor=L.+B.+Averyauthor=R.+A.+Caseroauthor=S.+D.+Tavernaauthor=J.+Qianauthor=A.+J.+Tackettauthor=R.+R.+Ratanauthor=O.+G.+McDonaldauthor=A.+P.+Feinbergauthor=P.+A.+Cole&title=A+selective+phenelzine+analogue+inhibitor+of+histone+demethylase+LSD1&doi=10.1021%2Fcb500018s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12lR"><div class="casContent"><span class="casTitleNuber">12l</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective Phenelzine Analogue Inhibitor of Histone Demethylase LSD1</span></div><div class="casAuthors">Prusevich, Polina; Kalin, Jay H.; Ming, Shonoi A.; Basso, Manuela; Givens, Jeffrey; Li, Xin; Hu, Jianfei; Taylor, Martin S.; Cieniewicz, Anne M.; Hsiao, Po-Yuan; Huang, Rong; Roberson, Heather; Adejola, Nkosi; Avery, Lindsay B.; Casero, Robert A., Jr.; Taverna, Sean D.; Qian, Jiang; Tackett, Alan J.; Ratan, Rajiv R.; McDonald, Oliver G.; Feinberg, Andrew P.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1284-1293</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is an epigenetic enzyme that oxidatively cleaves Me groups from monomethyl and di-Me Lys 4 of histone H3 (H3K4Me1, H3K4Me2) and can contribute to gene silencing.  This study describes the design and synthesis of analogs of a monoamine oxidase antidepressant, phenelzine, and their LSD1 inhibitory properties.  A novel phenelzine analog (bizine) contg. a phenyl-butyrylamide appendage was shown to be a potent LSD1 inhibitor in vitro and was selective vs. monoamine oxidases A/B and the LSD1 homolog, LSD2.  Bizine was found to be effective at modulating bulk histone methylation in cancer cells, and ChIP-seq expts. revealed a statistically significant overlap in the H3K4 methylation pattern of genes affected by bizine and those altered in LSD1-/- cells.  Treatment of two cancer cell lines, LNCaP and H460, with bizine conferred a redn. in proliferation rate, and bizine showed additive to synergistic effects on cell growth when used in combination with two out of five HDAC inhibitors tested.  Moreover, neurons exposed to oxidative stress were protected by the presence of bizine, suggesting potential applications in neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Z1w5uKT7qbVg90H21EOLACvtfcHk0lgFWiabr63bXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D&md5=36c9f3fb9b28579fa3cad41052cf335a</span></div><a href="/servlet/linkout?suffix=cit12l&amp;dbid=16384&amp;doi=10.1021%2Fcb500018s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500018s%26sid%3Dliteratum%253Aachs%26aulast%3DPrusevich%26aufirst%3DP.%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DMing%26aufirst%3DS.%2BA.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DGivens%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DCieniewicz%26aufirst%3DA.%2BM.%26aulast%3DHsiao%26aufirst%3DP.%2BY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DRoberson%26aufirst%3DH.%26aulast%3DAdejola%26aufirst%3DN.%26aulast%3DAvery%26aufirst%3DL.%2BB.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DRatan%26aufirst%3DR.%2BR.%26aulast%3DMcDonald%26aufirst%3DO.%2BG.%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520selective%2520phenelzine%2520analogue%2520inhibitor%2520of%2520histone%2520demethylase%2520LSD1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1284%26epage%3D1293%26doi%3D10.1021%2Fcb500018s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12m"><span><span class="NLM_label">(m) </span><span class="NLM_contrib-group">Kakizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ota, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Histone H3 peptide based LSD1-selective inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1925</span><span class="NLM_x">â</span> <span class="NLM_lpage">1928</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.03.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12m&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.bmcl.2015.03.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12m&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=25827526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12m&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFWgs7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1925-1928&author=T.+Kakizawaauthor=Y.+Otaauthor=Y.+Itohauthor=H.+Tsumotoauthor=T.+Suzuki&title=Histone+H3+peptide+based+LSD1-selective+inhibitors&doi=10.1016%2Fj.bmcl.2015.03.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12mR"><div class="casContent"><span class="casTitleNuber">12m</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 peptide based LSD1-selective inhibitors</span></div><div class="casAuthors">Kakizawa, Taeko; Ota, Yosuke; Itoh, Yukihiro; Tsumoto, Hiroki; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1925-1928</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of candidates for the histone H3 peptide based LSD1-selective inhibitor were designed and synthesized.  Among peptides, ART-X-QTARKSTGGKAPRKQLA and ARTKQTAR-X-STGGKAPRKQLA [X = substituted with phenylcyclopropylamine, 5-dihydro-1H-pyrrole and 1,2,3,6-tetrahydropyridine lysine (I), (II) and (III)].  Peptide ART-I-QTARKSTGGKAPRKQLA, which has a phenylcyclopropylamine (PCPA) moiety at Lys-4 of the 21 amino acid residues of histone H3, was the most potent LSD1-selective inhibitor.  Truncation studies of peptide ART-I-QTARKSTGGKAPRKQLA revealed the significance of the peptide sequence length.  These findings will be useful for the further development of histone H3 peptide based LSD1-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxO8fMkITrQ7Vg90H21EOLACvtfcHk0lhab3CHYxtBUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFWgs7Y%253D&md5=1ea3add93c3eaa8a367433a6ddcb4dd9</span></div><a href="/servlet/linkout?suffix=cit12m&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.03.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.03.030%26sid%3Dliteratum%253Aachs%26aulast%3DKakizawa%26aufirst%3DT.%26aulast%3DOta%26aufirst%3DY.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DHistone%2520H3%2520peptide%2520based%2520LSD1-selective%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D1925%26epage%3D1928%26doi%3D10.1016%2Fj.bmcl.2015.03.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12n"><span><span class="NLM_label">(n) </span><span class="NLM_contrib-group">Khan, M. N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ota, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Design, synthesis, and biological activity of N-alkylated analogue of NCL1, a selective inhibitor of lysine-specific demethylase 1</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">407</span><span class="NLM_x">â</span> <span class="NLM_lpage">412</span><span class="refDoi">Â DOI: 10.1039/C4MD00330F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12n&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1039%2FC4MD00330F" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=407-412&author=M.+N.+A.+Khanauthor=H.+Tsumotoauthor=Y.+Itohauthor=Y.+Otaauthor=M.+Suzukiauthor=D.+Ogasawaraauthor=H.+Nakagawaauthor=T.+Mizukamiauthor=N.+Miyataauthor=T.+Suzuki&title=Design%2C+synthesis%2C+and+biological+activity+of+N-alkylated+analogue+of+NCL1%2C+a+selective+inhibitor+of+lysine-specific+demethylase+1&doi=10.1039%2FC4MD00330F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12n&amp;dbid=16384&amp;doi=10.1039%2FC4MD00330F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00330F%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BN.%2BA.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOta%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520activity%2520of%2520N-alkylated%2520analogue%2520of%2520NCL1%252C%2520a%2520selective%2520inhibitor%2520of%2520lysine-specific%2520demethylase%25201%26jtitle%3DMedChemComm%26date%3D2015%26volume%3D6%26spage%3D407%26epage%3D412%26doi%3D10.1039%2FC4MD00330F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12o"><span><span class="NLM_label">(o) </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ota, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Synthesis, LSD1 inhibitory activity, and LSD1 binding model of optically pure lysine-PCPA conjugates</span> <span class="citation_source-journal">Comput. Struct. Biotechnol. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">â</span> <span class="NLM_lpage">9</span><span class="refDoi">Â DOI: 10.5936/csbj.201402002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12o&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.5936%2Fcsbj.201402002" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1-9&author=Y.+Itohauthor=D.+Ogasawaraauthor=Y.+Otaauthor=T.+Mizukamiauthor=T.+Suzuki&title=Synthesis%2C+LSD1+inhibitory+activity%2C+and+LSD1+binding+model+of+optically+pure+lysine-PCPA+conjugates&doi=10.5936%2Fcsbj.201402002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12o&amp;dbid=16384&amp;doi=10.5936%2Fcsbj.201402002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5936%252Fcsbj.201402002%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DOta%26aufirst%3DY.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DSynthesis%252C%2520LSD1%2520inhibitory%2520activity%252C%2520and%2520LSD1%2520binding%2520model%2520of%2520optically%2520pure%2520lysine-PCPA%2520conjugates%26jtitle%3DComput.%2520Struct.%2520Biotechnol.%2520J.%26date%3D2014%26volume%3D9%26spage%3D1%26epage%3D9%26doi%3D10.5936%2Fcsbj.201402002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12p"><span><span class="NLM_label">(p) </span><span class="NLM_contrib-group">Mohammad, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smitheman, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamat, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federowicz, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneck, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butticello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnette, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degenhardt, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasparec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNulty, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDevitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouthamel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Concha, N. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpenter, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span> </span><span class="NLM_article-title">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">â</span> <span class="NLM_lpage">69</span><span class="refDoi">Â DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12p&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12p&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12p&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+Hypomethylation+Signature+Predicts+Antitumor+Activity+of+LSD1+Inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12pR"><div class="casContent"><span class="casTitleNuber">12p</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lhab3CHYxtBUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit12p&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520Hypomethylation%2520Signature%2520Predicts%2520Antitumor%2520Activity%2520of%2520LSD1%2520Inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meek, T. D.</span><span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">â</span> <span class="NLM_lpage">739</span><span class="refDoi">Â DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0ljAQm-AktMzHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Ito, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span> </span><span class="NLM_article-title">Metal-free reductive cyclization and isomerization of sulfanyl-1,6-diynes using sodium borohydride</span> <span class="citation_source-journal">Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1758</span><span class="NLM_x">â</span> <span class="NLM_lpage">1760</span><span class="refDoi">Â DOI: 10.1246/cl.140688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1246%2Fcl.140688" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=1758-1760&author=Y.+Itoauthor=M.+Yoshimatsu&title=Metal-free+reductive+cyclization+and+isomerization+of+sulfanyl-1%2C6-diynes+using+sodium+borohydride&doi=10.1246%2Fcl.140688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1246%2Fcl.140688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1246%252Fcl.140688%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DY.%26aulast%3DYoshimatsu%26aufirst%3DM.%26atitle%3DMetal-free%2520reductive%2520cyclization%2520and%2520isomerization%2520of%2520sulfanyl-1%252C6-diynes%2520using%2520sodium%2520borohydride%26jtitle%3DChem.%2520Lett.%26date%3D2014%26volume%3D43%26spage%3D1758%26epage%3D1760%26doi%3D10.1246%2Fcl.140688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Lee, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fekner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Au, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. K.</span><span> </span><span class="NLM_article-title">A click-and-release pyrrolysine analogue</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">805</span><span class="NLM_x">â</span> <span class="NLM_lpage">808</span><span class="refDoi">Â DOI: 10.1002/cbic.201300124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1002%2Fcbic.201300124" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=805-808&author=M.+M.+Leeauthor=T.+Feknerauthor=T.+H.+Tangauthor=L.+Wangauthor=A.+H.+Chanauthor=P.+H.+Hsuauthor=S.+W.+Auauthor=M.+K.+Chan&title=A+click-and-release+pyrrolysine+analogue&doi=10.1002%2Fcbic.201300124"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201300124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201300124%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BM.%26aulast%3DFekner%26aufirst%3DT.%26aulast%3DTang%26aufirst%3DT.%2BH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChan%26aufirst%3DA.%2BH.%26aulast%3DHsu%26aufirst%3DP.%2BH.%26aulast%3DAu%26aufirst%3DS.%2BW.%26aulast%3DChan%26aufirst%3DM.%2BK.%26atitle%3DA%2520click-and-release%2520pyrrolysine%2520analogue%26jtitle%3DChemBioChem%26date%3D2013%26volume%3D14%26spage%3D805%26epage%3D808%26doi%3D10.1002%2Fcbic.201300124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Shetty, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeong, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. C.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of macrocyclic amino acid derivatives for tumor imaging by gallium-68 positron emission tomography</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">7338</span><span class="NLM_x">â</span> <span class="NLM_lpage">7347</span><span class="refDoi">Â DOI: 10.1016/j.bmc.2010.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.bmc.2010.09.022" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=7338-7347&author=D.+Shettyauthor=J.+M.+Jeongauthor=C.+H.+Juauthor=Y.+Kimauthor=J.+Y.+Leeauthor=Y.+S.+Leeauthor=D.+S.+Leeauthor=J.+K.+Chungauthor=M.+C.+Lee&title=Synthesis+and+evaluation+of+macrocyclic+amino+acid+derivatives+for+tumor+imaging+by+gallium-68+positron+emission+tomography&doi=10.1016%2Fj.bmc.2010.09.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DShetty%26aufirst%3DD.%26aulast%3DJeong%26aufirst%3DJ.%2BM.%26aulast%3DJu%26aufirst%3DC.%2BH.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DY.%2BS.%26aulast%3DLee%26aufirst%3DD.%2BS.%26aulast%3DChung%26aufirst%3DJ.%2BK.%26aulast%3DLee%26aufirst%3DM.%2BC.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520macrocyclic%2520amino%2520acid%2520derivatives%2520for%2520tumor%2520imaging%2520by%2520gallium-68%2520positron%2520emission%2520tomography%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D7338%26epage%3D7347%26doi%3D10.1016%2Fj.bmc.2010.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Molecular mimicry and ligand recognition in binding and catalysis by the histone demethylase LSD1-CoREST complex</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">212</span><span class="NLM_x">â</span> <span class="NLM_lpage">220</span><span class="refDoi">Â DOI: 10.1016/j.str.2011.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.str.2011.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=21300290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyitL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=212-220&author=R.+Baronauthor=C.+Bindaauthor=M.+Tortoriciauthor=J.+A.+McCammonauthor=A.+Mattevi&title=Molecular+mimicry+and+ligand+recognition+in+binding+and+catalysis+by+the+histone+demethylase+LSD1-CoREST+complex&doi=10.1016%2Fj.str.2011.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mimicry and Ligand Recognition in Binding and Catalysis by the Histone Demethylase LSD1-CoREST Complex</span></div><div class="casAuthors">Baron, Riccardo; Binda, Claudia; Tortorici, Marcello; McCammon, J. Andrew; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">212-220</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone demethylases LSD1 and LSD2 (KDM1A/B) catalyze the oxidative demethylation of Lys4 of histone H3.  We used mol. dynamics simulations to probe the diffusion of the oxygen substrate.  Oxygen can reach the catalytic center independently from the presence of a bound histone peptide, implying that LSD1 can complete subsequent demethylation cycles without detaching from the nucleosomal particle.  The simulations highlight the role of a strictly conserved active-site Lys residue providing general insight into the enzymic mechanism of oxygen-reacting flavoenzymes.  The crystal structure of LSD1-CoREST bound to a peptide of the transcription factor SNAIL1 unravels a fascinating example of mol. mimicry.  The SNAIL1 N-terminal residues bind to the enzyme active-site cleft, effectively mimicking the H3 tail.  This finding predicts that other members of the SNAIL/Scratch transcription factor family might assoc. to LSD1/2.  The combination of selective histone-modifying activity with the distinct recognition mechanisms underlies the biol. complexity of LSD1/2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGdKxxD93hm7Vg90H21EOLACvtfcHk0ljAQm-AktMzHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyitL4%253D&md5=61b4d3cf137e45643c3066f3369f18d5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DMolecular%2520mimicry%2520and%2520ligand%2520recognition%2520in%2520binding%2520and%2520catalysis%2520by%2520the%2520histone%2520demethylase%2520LSD1-CoREST%2520complex%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D212%26epage%3D220%26doi%3D10.1016%2Fj.str.2011.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmorstein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">A mechanism-based inactivator for histone demethylase LSD1</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">4536</span><span class="NLM_x">â</span> <span class="NLM_lpage">4537</span><span class="refDoi">Â DOI: 10.1021/ja0602748</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0602748" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=4536-4537&author=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=X.+Liuauthor=G.+Daauthor=R.+Marmorsteinauthor=P.+A.+Cole&title=A+mechanism-based+inactivator+for+histone+demethylase+LSD1&doi=10.1021%2Fja0602748"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1021%2Fja0602748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0602748%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDa%26aufirst%3DG.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520mechanism-based%2520inactivator%2520for%2520histone%2520demethylase%2520LSD1%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D4536%26epage%3D4537%26doi%3D10.1021%2Fja0602748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gocke, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brautigam, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomchick, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span> </span><span class="NLM_article-title">Structural basis of histone demethylation by LSD1 revealed by suicide inactivation</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">535</span><span class="NLM_x">â</span> <span class="NLM_lpage">539</span><span class="refDoi">Â DOI: 10.1038/nsmb1255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnsmb1255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=17529991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVyjtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=535-539&author=M.+Yangauthor=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=C.+B.+Gockeauthor=C.+A.+Brautigamauthor=D.+R.+Tomchickauthor=M.+Machiusauthor=P.+A.+Coleauthor=H.+Yu&title=Structural+basis+of+histone+demethylation+by+LSD1+revealed+by+suicide+inactivation&doi=10.1038%2Fnsmb1255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17bR"><div class="casContent"><span class="casTitleNuber">17b</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of histone demethylation by LSD1 revealed by suicide inactivation</span></div><div class="casAuthors">Yang, Maojun; Culhane, Jeffrey C.; Szewczuk, Lawrence M.; Gocke, Christian B.; Brautigam, Chad A.; Tomchick, Diana R.; Machius, Mischa; Cole, Philip A.; Yu, Hongtao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">535-539</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation regulates diverse chromatin-templated processes, including transcription.  The recent discovery of the first histone lysine-specific demethylase (LSD1) has changed the long-held view that histone methylation is a permanent epigenetic mark.  LSD1 is a FAD-dependent amine oxidase that demethylates histone H3 Lys4 (H3-K4).  However, the mechanism by which LSD1 achieves its substrate specificity is unclear.  We report the crystal structure of human LSD1 with a propargylamine-derivatized H3 peptide covalently tethered to FAD.  H3 adopts three consecutive Î³-turns, enabling an ideal side chain spacing that places its N terminus into an anionic pocket and positions methyl-Lys4 near FAD for catalysis.  The LSD1 active site cannot productively accommodate more than three residues on the N-terminal side of the methyllysine, explaining its H3-K4 specificity.  The unusual backbone conformation of LSD1-bound H3 suggests a strategy for designing potent LSD1 inhibitors with therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocskIRV1aPVbVg90H21EOLACvtfcHk0lgB9bgZgopVEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVyjtbs%253D&md5=51ba4fc92701ed0952d3f37e11ecda00</span></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1255%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DGocke%26aufirst%3DC.%2BB.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DTomchick%26aufirst%3DD.%2BR.%26aulast%3DMachius%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DH.%26atitle%3DStructural%2520basis%2520of%2520histone%2520demethylation%2520by%2520LSD1%2520revealed%2520by%2520suicide%2520inactivation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D535%26epage%3D539%26doi%3D10.1038%2Fnsmb1255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cloos, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sennels, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappsilber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salcini, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">RBP2 belongs to a family of demethylases, specific for tri-and dimethylated lysine 4 on histone 3</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">1063</span><span class="NLM_x">â</span> <span class="NLM_lpage">1076</span><span class="refDoi">Â DOI: 10.1016/j.cell.2007.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.cell.2007.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=17320161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslGms78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=1063-1076&author=J.+Christensenauthor=K.+Aggerauthor=P.+A.+Cloosauthor=D.+Pasiniauthor=S.+Roseauthor=L.+Sennelsauthor=J.+Rappsilberauthor=K.+H.+Hansenauthor=A.+E.+Salciniauthor=K.+Helin&title=RBP2+belongs+to+a+family+of+demethylases%2C+specific+for+tri-and+dimethylated+lysine+4+on+histone+3&doi=10.1016%2Fj.cell.2007.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">RBP2 belongs to a family of demethylases, specific for triand dimethylated lysine 4 on histone 3</span></div><div class="casAuthors">Christensen, Jesper; Agger, Karl; Cloos, Paul A. C.; Pasini, Diego; Rose, Simon; Sennels, Lau; Rappsilber, Juri; Hansen, Klaus H.; Salcini, Anna Elisabetta; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1063-1076</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of histones has been regarded as a stable modification defining the epigenetic program of the cell, which regulates chromatin structure and transcription.  However, the recent discovery of histone demethylases has challenged the stable nature of histone methylation.  Here we demonstrate that the JARID1 proteins RBP2, PLU1, and SMCX are histone demethylases specific for di- and trimethylated histone 3 lysine 4 (H3K4).  Consistent with a role for the JARID1 Drosophila homolog Lid in regulating expression of homeotic genes during development, we show that RBP2 is displaced from Hox genes during embryonic stem (ES) cell differentiation correlating with an increase of their H3K4me3 levels and expression.  Furthermore, we show that mutation or RNAi depletion of the C. elegans JARID1 homolog rbr-2 leads to increased levels of H3K4me3 during larval development and defects in vulva formation.  Taken together, these results suggest that H3K4me3/me2 demethylation regulated by the JARID1 family plays an important role during development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoiLQh_7D2g7Vg90H21EOLACvtfcHk0lgB9bgZgopVEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslGms78%253D&md5=c1f04b597253ddc6d89b62bf0b99b13c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DCloos%26aufirst%3DP.%2BA.%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DRose%26aufirst%3DS.%26aulast%3DSennels%26aufirst%3DL.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DSalcini%26aufirst%3DA.%2BE.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DRBP2%2520belongs%2520to%2520a%2520family%2520of%2520demethylases%252C%2520specific%2520for%2520tri-and%2520dimethylated%2520lysine%25204%2520on%2520histone%25203%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D1063%26epage%3D1076%26doi%3D10.1016%2Fj.cell.2007.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Futaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yagami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiura, Y.</span><span> </span><span class="NLM_article-title">Arginine-rich peptides. An abundant source of membrane-permeable peptides having potential as carriers for intracellular protein delivery</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">5836</span><span class="NLM_x">â</span> <span class="NLM_lpage">5840</span><span class="refDoi">Â DOI: 10.1074/jbc.M007540200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1074%2Fjbc.M007540200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=11084031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhs1KmtL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=5836-5840&author=S.+Futakiauthor=T.+Suzukiauthor=W.+Ohashiauthor=T.+Yagamiauthor=S.+Tanakaauthor=K.+Uedaauthor=Y.+Sugiura&title=Arginine-rich+peptides.+An+abundant+source+of+membrane-permeable+peptides+having+potential+as+carriers+for+intracellular+protein+delivery&doi=10.1074%2Fjbc.M007540200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine-rich peptides: an abundant source of membrane-permeable peptides having potential as carriers for intracellular protein delivery</span></div><div class="casAuthors">Futaki, Shiroh; Suzuki, Tomoki; Ohashi, Wakana; Yagami, Takeshi; Tanaka, Seigo; Ueda, Kunihiro; Sugiura, Yukio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">5836-5840</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A basic peptide derived from human immunodeficiency virus (HIV)-1 Tat protein (positions 48-60) has been reported to have the ability to translocate through the cell membranes and accumulate in the nucleus, the characteristics of which are utilized for the delivery of exogenous proteins into cells.  Based on the fluorescence microscopic observations of mouse macrophage RAW264.7 cells, we found that various arginine-rich peptides have a translocation activity very similar to Tat-(48-60).  These included such peptides as the D-amino acid- and arginine-substituted Tat-(48-60), the RNA-binding peptides derived from virus proteins, such as HIV-1 Rev, and flock house virus coat proteins, and the DNA binding segments of leucine zipper proteins, such as cancer-related proteins c-Fos and c-Jan, and the yeast transcription factor GCN4.  These segments have no specific primary and secondary structures in common except that they have several arginine residues in the sequences.  Moreover, these peptides were internalized even at 4Â°.  These results strongly suggested the possible existence of a common internalization mechanism ubiquitous to arginine-rich peptides, which is not explained by a typical endocytosis.  Using (Arg)n (n = 4-16) peptides, we also demonstrated that there would be an optimal no. of arginine residues (n â¼ 8) for the efficient translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorVdj5ycrHWrVg90H21EOLACvtfcHk0lgB9bgZgopVEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhs1KmtL0%253D&md5=9d0b4040f7eede7e405734a3cd2e33d5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M007540200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M007540200%26sid%3Dliteratum%253Aachs%26aulast%3DFutaki%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DOhashi%26aufirst%3DW.%26aulast%3DYagami%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DSugiura%26aufirst%3DY.%26atitle%3DArginine-rich%2520peptides.%2520An%2520abundant%2520source%2520of%2520membrane-permeable%2520peptides%2520having%2520potential%2520as%2520carriers%2520for%2520intracellular%2520protein%2520delivery%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D5836%26epage%3D5840%26doi%3D10.1074%2Fjbc.M007540200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Hashimoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shioiri, T.</span><span> </span><span class="NLM_article-title">New methods and reagents in organic synthesis. 14. A simple efficient preparation of methyl esters with trimethylsilyldiazomethane (TMSCHN<sub>2</sub>) and itsaApplication to gas chromatographic analysis of fatty acids</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1475</span><span class="NLM_x">â</span> <span class="NLM_lpage">1478</span><span class="refDoi">Â DOI: 10.1248/cpb.29.1475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1248%2Fcpb.29.1475" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1981&pages=1475-1478&author=N.+Hashimotoauthor=T.+Aoyamaauthor=T.+Shioiri&title=New+methods+and+reagents+in+organic+synthesis.+14.+A+simple+efficient+preparation+of+methyl+esters+with+trimethylsilyldiazomethane+%28TMSCHN2%29+and+itsaApplication+to+gas+chromatographic+analysis+of+fatty+acids&doi=10.1248%2Fcpb.29.1475"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1248%2Fcpb.29.1475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.29.1475%26sid%3Dliteratum%253Aachs%26aulast%3DHashimoto%26aufirst%3DN.%26aulast%3DAoyama%26aufirst%3DT.%26aulast%3DShioiri%26aufirst%3DT.%26atitle%3DNew%2520methods%2520and%2520reagents%2520in%2520organic%2520synthesis.%252014.%2520A%2520simple%2520efficient%2520preparation%2520of%2520methyl%2520esters%2520with%2520trimethylsilyldiazomethane%2520%2528TMSCHN2%2529%2520and%2520itsaApplication%2520to%2520gas%2520chromatographic%2520analysis%2520of%2520fatty%2520acids%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1981%26volume%3D29%26spage%3D1475%26epage%3D1478%26doi%3D10.1248%2Fcpb.29.1475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ninomiya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tujii, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsumori, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuchida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hosoi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murakami-Yamaguchi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span> </span><span class="NLM_article-title">Regulation of tissue factor pathway inhibitor-2 (TFPI-2) expression by lysine-specific demethylase 1 and 2 (LSD1 and LSD2)</span> <span class="citation_source-journal">Biosci., Biotechnol., Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">1010</span><span class="NLM_x">â</span> <span class="NLM_lpage">1017</span><span class="refDoi">Â DOI: 10.1080/09168451.2014.910104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1080%2F09168451.2014.910104" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2014&pages=1010-1017&author=K.+Minoauthor=S.+Nishimuraauthor=S.+Ninomiyaauthor=H.+Tujiiauthor=Y.+Matsumoriauthor=M.+Tsuchidaauthor=M.+Hosoiauthor=K.+Kosekiauthor=S.+Wadaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=Y.+Murakami-Yamaguchiauthor=H.+Naritaauthor=T.+Suzukiauthor=N.+Miyataauthor=T.+Mizukami&title=Regulation+of+tissue+factor+pathway+inhibitor-2+%28TFPI-2%29+expression+by+lysine-specific+demethylase+1+and+2+%28LSD1+and+LSD2%29&doi=10.1080%2F09168451.2014.910104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1080%2F09168451.2014.910104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F09168451.2014.910104%26sid%3Dliteratum%253Aachs%26aulast%3DMino%26aufirst%3DK.%26aulast%3DNishimura%26aufirst%3DS.%26aulast%3DNinomiya%26aufirst%3DS.%26aulast%3DTujii%26aufirst%3DH.%26aulast%3DMatsumori%26aufirst%3DY.%26aulast%3DTsuchida%26aufirst%3DM.%26aulast%3DHosoi%26aufirst%3DM.%26aulast%3DKoseki%26aufirst%3DK.%26aulast%3DWada%26aufirst%3DS.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMurakami-Yamaguchi%26aufirst%3DY.%26aulast%3DNarita%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DMizukami%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520tissue%2520factor%2520pathway%2520inhibitor-2%2520%2528TFPI-2%2529%2520expression%2520by%2520lysine-specific%2520demethylase%25201%2520and%25202%2520%2528LSD1%2520and%2520LSD2%2529%26jtitle%3DBiosci.%252C%2520Biotechnol.%252C%2520Biochem.%26date%3D2014%26volume%3D78%26spage%3D1010%26epage%3D1017%26doi%3D10.1080%2F09168451.2014.910104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozasa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohkubo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Identification of the KDM2/7 histone lysine demethylase subfamily inhibitor and its antiproliferative activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7222</span><span class="NLM_x">â</span> <span class="NLM_lpage">7231</span><span class="refDoi">Â DOI: 10.1021/jm400624b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400624b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7222-7231&author=T.+Suzukiauthor=H.+Ozasaauthor=Y.+Itohauthor=P.+Zhanauthor=H.+Sawadaauthor=K.+Minoauthor=L.+Walportauthor=R.+Ohkuboauthor=A.+Kawamuraauthor=M.+Yonezawaauthor=Y.+Tsukadaauthor=A.+Tumberauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=C.+J.+Schofieldauthor=N.+Miyata&title=Identification+of+the+KDM2%2F7+histone+lysine+demethylase+subfamily+inhibitor+and+its+antiproliferative+activity&doi=10.1021%2Fjm400624b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=10.1021%2Fjm400624b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400624b%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DOzasa%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DSawada%26aufirst%3DH.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DWalport%26aufirst%3DL.%26aulast%3DOhkubo%26aufirst%3DR.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DTsukada%26aufirst%3DY.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520the%2520KDM2%252F7%2520histone%2520lysine%2520demethylase%2520subfamily%2520inhibitor%2520and%2520its%2520antiproliferative%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7222%26epage%3D7231%26doi%3D10.1021%2Fjm400624b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit22b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tojo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Identification of jumonji AT-rich interactive domain 1A inhibitors and their effect on cancer cells</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">665</span><span class="NLM_x">â</span> <span class="NLM_lpage">670</span><span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFeisbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=665-670&author=Y.+Itohauthor=H.+Sawadaauthor=M.+Suzukiauthor=T.+Tojoauthor=R.+Sasakiauthor=M.+Hasegawaauthor=T.+Mizukamiauthor=T.+Suzuki&title=Identification+of+jumonji+AT-rich+interactive+domain+1A+inhibitors+and+their+effect+on+cancer+cells&doi=10.1021%2Facsmedchemlett.5b00083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22bR"><div class="casContent"><span class="casTitleNuber">22b</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Jumonji AT-Rich Interactive Domain 1A Inhibitors and Their Effect on Cancer Cells</span></div><div class="casAuthors">Itoh, Yukihiro; Sawada, Hideyuki; Suzuki, Miki; Tojo, Toshifumi; Sasaki, Ryuzo; Hasegawa, Makoto; Mizukami, Tamio; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">665-670</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Jumonji AT-rich interactive domain 1A (JARID1A), one of the jumonji C domain-contg. histone demethylase (JHDM) family members, plays key roles in cancer cell proliferation and development of drug tolerance.  Therefore, selective JARID1A inhibitors are potential anticancer agents.  In this study, the authors searched for cell-active JARID1A inhibitors by screening hydroxamate compds. in the inhouse library and the structural optimization based on docking study of the hit-compd. to a homol. model of JARID1A.  As a result, the authors identified compd. I, which selectively inhibits JARID1A over three other JHDM family members.  Compd. II, a prodrug form of compd. I, induced a selective increase in the level of trimethylation of histone H3 lysine 4, a substrate of JARID1A.  Furthermore, compd. II synergistically enhanced A549 human lung cancer cell growth inhibition induced by vorinostat, a histone deacetylase inhibitor.  These findings support the idea that JARID1A inhibitors have potential as anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAGPzR-uSQybVg90H21EOLACvtfcHk0lhTGqgbgEqKqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFeisbk%253D&md5=056d2209c293d95153c09d7d8a1e0248</span></div><a href="/servlet/linkout?suffix=cit22b&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00083%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSawada%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DTojo%26aufirst%3DT.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DIdentification%2520of%2520jumonji%2520AT-rich%2520interactive%2520domain%25201A%2520inhibitors%2520and%2520their%2520effect%2520on%2520cancer%2520cells%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D665%26epage%3D670%26doi%3D10.1021%2Facsmedchemlett.5b00083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2a','cit2b','cit2c','cit2d','cit2e'],'ref3':['cit3'],'ref4':['cit4a','cit4b','cit4c'],'ref5':['cit5'],'ref6':['cit6a','cit6b'],'ref7':['cit7a','cit7b'],'ref8':['cit8'],'ref9':['cit9a','cit9b'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12a','cit12b','cit12c','cit12d','cit12e','cit12f','cit12g','cit12h','cit12i','cit12j','cit12k','cit12l','cit12m','cit12n','cit12o','cit12p'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15a','cit15b'],'ref16':['cit16'],'ref17':['cit17a','cit17b'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22a','cit22b']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 20 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dominik K.  KÃ¶lmel</span> and <span class="hlFld-ContribAuthor ">Eric T.  Kool</span>  . </span><span class="cited-content_cbyCitation_article-title">Oximes and Hydrazones in Bioconjugation: Mechanism and Catalysis. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2017,</strong> <em>117 </em>
                                    (15)
                                     , 10358-10376. <a href="https://doi.org/10.1021/acs.chemrev.7b00090" title="DOI URL">https://doi.org/10.1021/acs.chemrev.7b00090</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.7b00090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.7b00090%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DOximes%252Band%252BHydrazones%252Bin%252BBioconjugation%25253A%252BMechanism%252Band%252BCatalysis%26aulast%3DK%25C3%25B6lmel%26aufirst%3DDominik%2BK.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D09022017%26date%3D22062017%26date%3D09082017%26volume%3D117%26issue%3D15%26spage%3D10358%26epage%3D10376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stijn M.  Agten</span>, <span class="hlFld-ContribAuthor ">Ingrid  Dijkgraaf</span>, <span class="hlFld-ContribAuthor ">StanÂ H. E.  Beelen</span>, <span class="hlFld-ContribAuthor ">Tilman M.  Hackeng</span>. </span><span class="cited-content_cbyCitation_article-title">Bioâorthogonal Imine Chemistry in Chemical Protein Synthesis. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 327-356. <a href="https://doi.org/10.1002/9783527823567.ch12" title="DOI URL">https://doi.org/10.1002/9783527823567.ch12</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527823567.ch12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527823567.ch12%26sid%3Dliteratum%253Aachs%26atitle%3DBio%2525E2%252580%252590orthogonal%252BImine%252BChemistry%252Bin%252BChemical%252BProtein%252BSynthesis%26aulast%3DAgten%26aufirst%3DStijn%2BM.%26date%3D2021%26date%3D2021%26spage%3D327%26epage%3D356%26pub%3DWiley%26atitle%3DTotal%252BChemical%252BSynthesis%252Bof%252BProteins%26aulast%3DBrik%26aufirst%3DAshraf%26date%3D2021%26date%3D2021%26volume%3D117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samir  Mehndiratta</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine specific demethylase 1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (9)
                                     , 969-981. <a href="https://doi.org/10.1039/D0MD00141D" title="DOI URL">https://doi.org/10.1039/D0MD00141D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00141D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00141D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DHistone%252Blysine%252Bspecific%252Bdemethylase%252B1%252Binhibitors%26aulast%3DMehndiratta%26aufirst%3DSamir%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D9%26spage%3D969%26epage%3D981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarder  Arifuzzaman</span>, <span class="hlFld-ContribAuthor ">Mst Reshma  Khatun</span>, <span class="hlFld-ContribAuthor ">Rabeya  Khatun</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging of lysine demethylases (KDMs): From pathophysiological insights to novel therapeutic opportunities. </span><span class="cited-content_cbyCitation_journal-name">Biomedicine & Pharmacotherapy</span><span> <strong>2020,</strong> <em>129 </em>, 110392. <a href="https://doi.org/10.1016/j.biopha.2020.110392" title="DOI URL">https://doi.org/10.1016/j.biopha.2020.110392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biopha.2020.110392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biopha.2020.110392%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicine%2520%2526%2520Pharmacotherapy%26atitle%3DEmerging%252Bof%252Blysine%252Bdemethylases%252B%252528KDMs%252529%25253A%252BFrom%252Bpathophysiological%252Binsights%252Bto%252Bnovel%252Btherapeutic%252Bopportunities%26aulast%3DArifuzzaman%26aufirst%3DSarder%26date%3D2020%26volume%3D129%26spage%3D110392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liyun  Liang</span>, <span class="hlFld-ContribAuthor ">Haiwen  Wang</span>, <span class="hlFld-ContribAuthor ">Yongliang  Du</span>, <span class="hlFld-ContribAuthor ">Bingling  Luo</span>, <span class="hlFld-ContribAuthor ">Ning  Meng</span>, <span class="hlFld-ContribAuthor ">Meifeng  Cen</span>, <span class="hlFld-ContribAuthor ">Peng  Huang</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>, <span class="hlFld-ContribAuthor ">Shijun  Wen</span>. </span><span class="cited-content_cbyCitation_article-title">New tranylcypromine derivatives containing sulfonamide motif as potent LSD1 inhibitors to target acute myeloid leukemia: Design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>99 </em>, 103808. <a href="https://doi.org/10.1016/j.bioorg.2020.103808" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.103808</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.103808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.103808%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DNew%252Btranylcypromine%252Bderivatives%252Bcontaining%252Bsulfonamide%252Bmotif%252Bas%252Bpotent%252BLSD1%252Binhibitors%252Bto%252Btarget%252Bacute%252Bmyeloid%252Bleukemia%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DLiang%26aufirst%3DLiyun%26date%3D2020%26volume%3D99%26spage%3D103808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Discovery Researches on Modulators of Lysine-Modifying Enzymes Based on Strategic Chemistry Approaches. </span><span class="cited-content_cbyCitation_journal-name">Chemical and Pharmaceutical Bulletin</span><span> <strong>2020,</strong> <em>68 </em>
                                    (1)
                                     , 34-45. <a href="https://doi.org/10.1248/cpb.c19-00741" title="DOI URL">https://doi.org/10.1248/cpb.c19-00741</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1248/cpb.c19-00741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1248%2Fcpb.c19-00741%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520and%2520Pharmaceutical%2520Bulletin%26atitle%3DDrug%252BDiscovery%252BResearches%252Bon%252BModulators%252Bof%252BLysine-Modifying%252BEnzymes%252BBased%252Bon%252BStrategic%252BChemistry%252BApproaches%26aulast%3DItoh%26aufirst%3DYukihiro%26date%3D2020%26volume%3D68%26issue%3D1%26spage%3D34%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yorick  Janssens</span>, <span class="hlFld-ContribAuthor ">Evelien  Wynendaele</span>, <span class="hlFld-ContribAuthor ">Wim  Vanden Berghe</span>, <span class="hlFld-ContribAuthor ">Bart  De Spiegeleer</span>. </span><span class="cited-content_cbyCitation_article-title">Peptides as epigenetic modulators: therapeutic implications. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-019-0700-7" title="DOI URL">https://doi.org/10.1186/s13148-019-0700-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-019-0700-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-019-0700-7%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DPeptides%252Bas%252Bepigenetic%252Bmodulators%25253A%252Btherapeutic%252Bimplications%26aulast%3DJanssens%26aufirst%3DYorick%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhong-Rui  Li</span>, <span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Linlin  Yang</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Experience-based discovery (EBD) of aryl hydrazines as new scaffolds for the development of LSD1/KDM1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>166 </em>, 432-444. <a href="https://doi.org/10.1016/j.ejmech.2019.01.075" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.075%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExperience-based%252Bdiscovery%252B%252528EBD%252529%252Bof%252Baryl%252Bhydrazines%252Bas%252Bnew%252Bscaffolds%252Bfor%252Bthe%252Bdevelopment%252Bof%252BLSD1%25252FKDM1A%252Binhibitors%26aulast%3DLi%26aufirst%3DZhong-Rui%26date%3D2019%26volume%3D166%26spage%3D432%26epage%3D444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Technology for Epigenetics Toward Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 219-255. <a href="https://doi.org/10.1002/9783527823987.vol2_c11" title="DOI URL">https://doi.org/10.1002/9783527823987.vol2_c11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527823987.vol2_c11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527823987.vol2_c11%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%252BTechnology%252Bfor%252BEpigenetics%252BToward%252BDrug%252BDiscovery%26aulast%3DSuzuki%26aufirst%3DTakayoshi%26date%3D2019%26date%3D2019%26spage%3D219%26epage%3D255%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DMolecular%252BTechnology%26aulast%3DYamamoto%26aufirst%3DHisashi%26date%3D2018%26volume%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Lee</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Borrello</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">LSD
              (LysineâSpecific Demethylase): A DecadeâLong Trip from Discovery to Clinical Trials. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 221-261. <a href="https://doi.org/10.1002/9783527809257.ch10" title="DOI URL">https://doi.org/10.1002/9783527809257.ch10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch10%26sid%3Dliteratum%253Aachs%26atitle%3DLSD%252B%252528Lysine%2525E2%252580%252590Specific%252BDemethylase%252529%25253A%252BA%252BDecade%2525E2%252580%252590Long%252BTrip%252Bfrom%252BDiscovery%252Bto%252BClinical%252BTrials%26aulast%3DLee%26aufirst%3DAdam%26date%3D2019%26date%3D2019%26spage%3D221%26epage%3D261%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yosuke  Ota</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Design Concepts for LSD1âSelective Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1782-1791. <a href="https://doi.org/10.1002/tcr.201810031" title="DOI URL">https://doi.org/10.1002/tcr.201810031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201810031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201810031%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DDrug%252BDesign%252BConcepts%252Bfor%252BLSD1%2525E2%252580%252590Selective%252BInhibitors%26aulast%3DOta%26aufirst%3DYosuke%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1782%26epage%3D1791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingchao  Duan</span>, <span class="hlFld-ContribAuthor ">Wenping  Qin</span>, <span class="hlFld-ContribAuthor ">Fengzhi  Suo</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Zhai</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Guan</span>, <span class="hlFld-ContribAuthor ">Xiaojuan  Wang</span>, <span class="hlFld-ContribAuthor ">Yichao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and in vitro evaluation of stilbene derivatives as novel LSD1 inhibitors for AML therapy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (23-24)
                                     , 6000-6014. <a href="https://doi.org/10.1016/j.bmc.2018.10.037" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.10.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.10.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.10.037%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bin%252Bvitro%252Bevaluation%252Bof%252Bstilbene%252Bderivatives%252Bas%252Bnovel%252BLSD1%252Binhibitors%252Bfor%252BAML%252Btherapy%26aulast%3DDuan%26aufirst%3DYingchao%26date%3D2018%26volume%3D26%26issue%3D23-24%26spage%3D6000%26epage%3D6014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guan-Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Pui-Man  Lei</span>, <span class="hlFld-ContribAuthor ">Suk-Yu  Wong</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>, <span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological Inhibition of LSD1 for Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (12)
                                     , 3194. <a href="https://doi.org/10.3390/molecules23123194" title="DOI URL">https://doi.org/10.3390/molecules23123194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23123194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23123194%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPharmacological%252BInhibition%252Bof%252BLSD1%252Bfor%252BCancer%252BTreatment%26aulast%3DYang%26aufirst%3DGuan-Jun%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D12%26spage%3D3194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yosuke  Ota</span>, <span class="hlFld-ContribAuthor ">Taeko  Kakizawa</span>, <span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and In Vitro Evaluation of Novel Histone H3 Peptide-Based LSD1 Inactivators Incorporating Î±,Î±-Disubstituted Amino Acids with Î³-Turn-Inducing Structures. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (5)
                                     , 1099. <a href="https://doi.org/10.3390/molecules23051099" title="DOI URL">https://doi.org/10.3390/molecules23051099</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23051099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23051099%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BIn%252BVitro%252BEvaluation%252Bof%252BNovel%252BHistone%252BH3%252BPeptide-Based%252BLSD1%252BInactivators%252BIncorporating%252B%2525CE%2525B1%25252C%2525CE%2525B1-Disubstituted%252BAmino%252BAcids%252Bwith%252B%2525CE%2525B3-Turn-Inducing%252BStructures%26aulast%3DOta%26aufirst%3DYosuke%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D5%26spage%3D1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yosuke  Ota</span>, <span class="hlFld-ContribAuthor ">Shin  Miyamura</span>, <span class="hlFld-ContribAuthor ">Misaho  Araki</span>, <span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Shusuke  Yasuda</span>, <span class="hlFld-ContribAuthor ">Mitsuharu  Masuda</span>, <span class="hlFld-ContribAuthor ">Tomoyuki  Taniguchi</span>, <span class="hlFld-ContribAuthor ">Yoshihiro  Sowa</span>, <span class="hlFld-ContribAuthor ">Toshiyuki  Sakai</span>, <span class="hlFld-ContribAuthor ">Kenichiro  Itami</span>, <span class="hlFld-ContribAuthor ">Junichiro  Yamaguchi</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and evaluation of Î³-turn mimetics as LSD1-selective inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (3)
                                     , 775-785. <a href="https://doi.org/10.1016/j.bmc.2017.12.045" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.12.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.12.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.12.045%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bevaluation%252Bof%252B%2525CE%2525B3-turn%252Bmimetics%252Bas%252BLSD1-selective%252Binhibitors%26aulast%3DOta%26aufirst%3DYosuke%26date%3D2018%26volume%3D26%26issue%3D3%26spage%3D775%26epage%3D785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Taeko  Kakizawa</span>, <span class="hlFld-ContribAuthor ">Yosuke  Ota</span>, <span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Histone H3 peptides incorporating modified lysine residues as lysine-specific demethylase 1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (2)
                                     , 167-169. <a href="https://doi.org/10.1016/j.bmcl.2017.11.035" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.11.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.11.035%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DHistone%252BH3%252Bpeptides%252Bincorporating%252Bmodified%252Blysine%252Bresidues%252Bas%252Blysine-specific%252Bdemethylase%252B1%252Binhibitors%26aulast%3DKakizawa%26aufirst%3DTaeko%26date%3D2018%26volume%3D28%26issue%3D2%26spage%3D167%26epage%3D169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 340-352. <a href="https://doi.org/10.1080/15592294.2017.1290032" title="DOI URL">https://doi.org/10.1080/15592294.2017.1290032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2017.1290032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2017.1290032%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DStructural%252Binsight%252Binto%252Binhibitors%252Bof%252Bflavin%252Badenine%252Bdinucleotide-dependent%252Blysine%252Bdemethylases%26aulast%3DNiwa%26aufirst%3DHideaki%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D5%26spage%3D340%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">âDrugâ Discovery with the Help of Organic Chemistry. </span><span class="cited-content_cbyCitation_journal-name">YAKUGAKU ZASSHI</span><span> <strong>2017,</strong> <em>137 </em>
                                    (3)
                                     , 283-292. <a href="https://doi.org/10.1248/yakushi.16-00231-1" title="DOI URL">https://doi.org/10.1248/yakushi.16-00231-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1248/yakushi.16-00231-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1248%2Fyakushi.16-00231-1%26sid%3Dliteratum%253Aachs%26jtitle%3DYAKUGAKU%2520ZASSHI%26atitle%3D%2525E2%252580%25259CDrug%2525E2%252580%25259D%252BDiscovery%252Bwith%252Bthe%252BHelp%252Bof%252BOrganic%252BChemistry%26aulast%3DItoh%26aufirst%3DYukihiro%26date%3D2017%26volume%3D137%26issue%3D3%26spage%3D283%26epage%3D292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James W.  Walton</span>, <span class="hlFld-ContribAuthor ">Jasmine M.  Cross</span>, <span class="hlFld-ContribAuthor ">Tina  Riedel</span>, <span class="hlFld-ContribAuthor ">Paul J.  Dyson</span>. </span><span class="cited-content_cbyCitation_article-title">Perfluorinated HDAC inhibitors as selective anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2017,</strong> <em>15 </em>
                                    (43)
                                     , 9186-9190. <a href="https://doi.org/10.1039/C7OB02339A" title="DOI URL">https://doi.org/10.1039/C7OB02339A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7OB02339A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7OB02339A%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DPerfluorinated%252BHDAC%252Binhibitors%252Bas%252Bselective%252Banticancer%252Bagents%26aulast%3DWalton%26aufirst%3DJames%2BW.%26date%3D2017%26date%3D2017%26volume%3D15%26issue%3D43%26spage%3D9186%26epage%3D9190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Marabelli</span>, <span class="hlFld-ContribAuthor ">Biagina  Marrocco</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>. </span><span class="cited-content_cbyCitation_article-title">The growing structural and functional complexity of the LSD1/KDM1A histone demethylase. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2016,</strong> <em>41 </em>, 135-144. <a href="https://doi.org/10.1016/j.sbi.2016.07.011" title="DOI URL">https://doi.org/10.1016/j.sbi.2016.07.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2016.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2016.07.011%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DThe%252Bgrowing%252Bstructural%252Band%252Bfunctional%252Bcomplexity%252Bof%252Bthe%252BLSD1%25252FKDM1A%252Bhistone%252Bdemethylase%26aulast%3DMarabelli%26aufirst%3DChiara%26date%3D2016%26volume%3D41%26spage%3D135%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Examples of Reported Small-Molecular LSD1 Inactivators</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Structures of SNAIL1-Based Peptides <b>1</b>â<b>5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0013.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Peptides <b>1</b> and <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <sup><i>t</i></sup>BuOH, EDCI, DMAP, CH<sub>2</sub>Cl<sub>2</sub>, room temp, 93%; (b) H<sub>2</sub>, Pd/C, MeOH, room temp, 97%; (c) ClCOOEt, Et<sub>3</sub>N, THF, â10 Â°C; (d) NaBH<sub>4</sub>, THF, H<sub>2</sub>O, 0 Â°C to room temp, 74% (two steps); (e) MsCl, Et<sub>3</sub>N, DMAP, CH<sub>2</sub>Cl<sub>2</sub>, â20 Â°C, 58%; (f) <i>N</i>-benzylpropargylamine or <i>N</i>-methylpropargylamine, Cs<sub>2</sub>CO<sub>3</sub> for <b>11</b> or Et<sub>3</sub>N, DMAP for <b>12</b>, DMF, 65 Â°C for <b>11</b> or 75 Â°C for <b>12</b>, 19% for <b>11</b>, 23% for <b>12</b>; (g) HCl, 1,4-dioxane, CH<sub>2</sub>Cl<sub>2</sub>, 0 Â°C to room temp, 88% for <b>13</b>, 92% for <b>14</b>; (h) Fmoc-OSu, NaHCO<sub>3</sub>, 1,4-dioxane, H<sub>2</sub>O, 0 Â°C to room temp, 20% for <b>15</b>, 19% for <b>16</b>; (i) Fmoc-based solid-phase peptide synthesis, 17% for <b>1</b>, 16% for <b>2</b>.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0014.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Peptides <b>3</b> and <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) BnBr, K<sub>2</sub>CO<sub>3</sub>, NaOH, H<sub>2</sub>O, reflux, 29%; (b) DIBAL, THF, â10 to 0 Â°C, 92%; (c) DMSO, (COCl)<sub>2</sub>, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, â78 Â°C, 85%; (d) R<sup>2</sup>R<sup>3</sup>NNH<sub>2</sub>, THF, room temp, 94% for <b>21</b>, 93% for <b>22</b>; (e) NaBH<sub>3</sub>CN, EtOH, AcOH, room temp, 69% for <b>23</b>, 92% for <b>24</b>; (f) (Boc)<sub>2</sub>O, DMAP, CH<sub>2</sub>Cl<sub>2</sub>, room temp, 92% for <b>25</b>, 40% for <b>26</b>; (g) Pd black, HCOOH, MeOH, â 40 Â°C to room temp; (h) Fmoc-OSu, NaHCO<sub>3</sub>, 1,4-dioxane, H<sub>2</sub>O, 0 Â°C to room temp, 89% for <b>29</b> (2 steps), 82% for <b>30</b> (2 steps); (i) Fmoc-based solid-phase peptide synthesis, 16% for <b>3</b>, 29% for <b>4</b>.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0015.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Peptide <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) MsCl, Et<sub>3</sub>N, DMAP, CH<sub>2</sub>Cl<sub>2</sub>, â20 Â°C, 66%; (b) PCPA, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 75 Â°C, 75%; (c) HCl, dioxane, 0 Â°C to room temp, 78%; (d) FmocCl, NaHCO<sub>3</sub>, 1,4-dioxane, H<sub>2</sub>O, 0 Â°C to room temp, 14%; (e) (Boc)<sub>2</sub>O, NaHCO<sub>3</sub>, 1,4-dioxane, H<sub>2</sub>O, 0 Â°C to room temp, 28%; (f) Fmoc-based solid-phase peptide synthesis, 14%.</p></p></figure><figure data-id="cht3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Chart 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 3. Structures of Oligoarginine-Conjugated Peptides <b>37</b>â<b>40</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=cht3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. X-ray crystal structure of LSD1 complex with a SNAIL1 peptide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y48">2Y48</a>). FAD and the SNAIL1 peptide are displayed as tube and ball-and-stick models, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Plausible mechanism of inactivation of LSD1 by <i>N</i>-propargyl lysine peptide <b>1</b> or <b>2</b>. (b) Plausible mechanism of inactivation of LSD1 by peptides <b>3</b> and <b>4</b> bearing a hydrazine moiety. (c) Plausible mechanism of inactivation of LSD1 by peptide <b>5</b> bearing a PCPA moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Time- and concentration-dependent inhibition of LSD1 by peptide <b>3</b>. Steady-state progress curves obtained for inactivation of LSD1 by 0 (â ), 0.1 (â) 0.2 (â²), 0.4 ( Ã ), 0.8 (*), 1.6 (+), 3.2 (â), and 6.4 (â) Î¼M <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mass spectrometric detection of <b>3</b>-FAD adduct. Mass spectra (positive mode) were obtained from (a) LSD1-<b>3</b> mixture, (b) FAD-<b>3</b> mixture in the absence of LSD1, (c) LSD1-<b>4</b> mixture, and (d) FAD-<b>4</b> mixture in the absence of LSD1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0009.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Surface of the catalytic pocket of LSD1 in complex with a SNAIL1 peptide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y48">2Y48</a>). The SNAIL1 peptide is displayed as a ball-and-stick model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0010.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. View of the catalytic pocket of LSD1 in the complex with a SNAIL1 peptide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y48">2Y48</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0011.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Western blot detection of methylation level of H3K4 or H3K9 in HeLa cells after 24 h treatment with peptides in the absence of fetal bovine serum (FBS). Values of H3K4me2/H3, H3K4me1/H3, H3K4me3/H3, or H3K9me2/H3 ratio determined by optical density measurement of the blots are shown. (a) Treatment with 20 Î¼M of GSK2879552, peptides <b>3</b>, and <b>37</b>â<b>40</b>. (b) Dose-dependency for peptides <b>37</b> and <b>38</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/medium/jm-2015-01323m_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0012.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Effect of peptides <b>37</b>â<b>40</b> on viability of HeLa cells. The cells were incubated for 48 h after 24 h treatment with peptides <b>37</b>â<b>40</b> at the indicated concentrations in the absence of fetal bovine serum (FBS). Error bars represent the mean standard deviation (SD) of at least three samples. 300 Î¼M GSK2879552 reduced the cell viability by 9.96 Â± 0.47% (GSK2879552 was inactive at 100 Î¼M).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01323/20160219/images/large/jm-2015-01323m_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01323&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i56">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08336" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08336" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 22 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">â</span> <span class="NLM_lpage">705</span><span class="refDoi">Â DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lganWiznjomWw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Liu, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. H.</span><span> </span><span class="NLM_article-title">An Iridium(III) Complex Inhibits JMJD2 Activities and Acts as a Potential Epigenetic Modulator</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">6697</span><span class="NLM_x">â</span> <span class="NLM_lpage">6703</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00375</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00375" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6697-6703&author=L.+J.+Liuauthor=L.+Luauthor=H.+J.+Zhongauthor=B.+Heauthor=D.+W.+Kwongauthor=D.+L.+Maauthor=C.+H.+Leung&title=An+Iridium%28III%29+Complex+Inhibits+JMJD2+Activities+and+Acts+as+a+Potential+Epigenetic+Modulator&doi=10.1021%2Facs.jmedchem.5b00375"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00375%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%2BJ.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DH.%2BJ.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DKwong%26aufirst%3DD.%2BW.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DAn%2520Iridium%2528III%2529%2520Complex%2520Inhibits%2520JMJD2%2520Activities%2520and%2520Acts%2520as%2520a%2520Potential%2520Epigenetic%2520Modulator%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6697%26epage%3D6703%26doi%3D10.1021%2Facs.jmedchem.5b00375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Zhong, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C. C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H. M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. H.</span><span> </span><span class="NLM_article-title">An iridium(III)-based irreversible proteinâprotein interaction inhibitor of BRD4 as a potent anticancer agent</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">5400</span><span class="NLM_x">â</span> <span class="NLM_lpage">5408</span><span class="refDoi">Â DOI: 10.1039/C5SC02321A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1039%2FC5SC02321A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=28757943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grtr7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5400-5408&author=H.+J.+Zhongauthor=L.+Luauthor=K.+H.+Leungauthor=C.+C.+L.+Wongauthor=C.+Pengauthor=S.+C.+Yanauthor=D.+L.+Maauthor=Z.+Caiauthor=H.+M.+D.+Wangauthor=C.+H.+Leung&title=An+iridium%28III%29-based+irreversible+protein%E2%80%93protein+interaction+inhibitor+of+BRD4+as+a+potent+anticancer+agent&doi=10.1039%2FC5SC02321A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">An iridium(III)-based irreversible protein-protein interaction inhibitor of BRD4 as a potent anticancer agent</span></div><div class="casAuthors">Zhong, Hai-Jing; Lu, Lihua; Leung, Ka-Ho; Wong, Catherine C. L.; Peng, Chao; Yan, Siu-Cheong; Ma, Dik-Lung; Cai, Zongwei; David Wang, Hui-Min; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5400-5408</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) has recently emerged as an attractive epigenetic target for anticancer therapy.  In this study, an iridium(III) complex is reported as the first metal-based, irreversible inhibitor of BRD4.  Complex 1a is able to antagonize the BRD4-acetylated histone protein-protein interaction (PPI) in vitro, and to bind BRD4 and down-regulate c-myc oncogenic expression in cellulo.  Chromatin immunopptn. (ChIP) anal. revealed that 1a could modulate the interaction between BRD4 and chromatin in melanoma cells, particular at the MYC promoter.  Finally, the complex showed potent activity against melanoma xenografts in an in vivo mouse model.  To our knowledge, this is the first report of a Group 9 metal complex inhibiting the PPI of a member of the bromodomain and extraterminal domain (BET) family.  We envision that complex 1a may serve as a useful scaffold for the development of more potent epigenetic agents against cancers such as melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqMXspNfXFlbVg90H21EOLACvtfcHk0lgOMakgW-1AOg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grtr7E&md5=ae176cbfd3621dd7939c234d643db84e</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1039%2FC5SC02321A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC02321A%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.%2BJ.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DLeung%26aufirst%3DK.%2BH.%26aulast%3DWong%26aufirst%3DC.%2BC.%2BL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DS.%2BC.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DCai%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DH.%2BM.%2BD.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DAn%2520iridium%2528III%2529-based%2520irreversible%2520protein%25E2%2580%2593protein%2520interaction%2520inhibitor%2520of%2520BRD4%2520as%2520a%2520potent%2520anticancer%2520agent%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D5400%26epage%3D5408%26doi%3D10.1039%2FC5SC02321A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Schnekenburger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dicato, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, M.</span><span> </span><span class="NLM_article-title">Epigenetic modulators from â³The Big Blueâ³: a treasure to fight against cancer</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">351</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">â</span> <span class="NLM_lpage">197</span><span class="refDoi">Â DOI: 10.1016/j.canlet.2014.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.canlet.2014.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=24964131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGmtLbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2014&pages=182-197&author=M.+Schnekenburgerauthor=M.+Dicatoauthor=M.+Diederich&title=Epigenetic+modulators+from+%E2%80%B3The+Big+Blue%E2%80%B3%3A+a+treasure+to+fight+against+cancer&doi=10.1016%2Fj.canlet.2014.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2cR"><div class="casContent"><span class="casTitleNuber">2c</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic modulators from "The Big Blue": A treasure to fight against cancer</span></div><div class="casAuthors">Schnekenburger, Michael; Dicato, Mario; Diederich, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">182-197</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Cancer remains a major public health problem in our society.  The development of potent novel anti-cancer drugs selective for tumor cells is therefore still required.  Deregulation of the epigenetic machinery including DNA methylation, histone modifications and non-coding RNAs is a hallmark of cancer, which provides potential new therapeutic targets.  Natural products or their derivs. represent a major class of anti-cancer drugs in the arsenal available to the clinician.  However, regarding epigenetically active anti-cancer agents for clinics, the oceans represent a largely untapped resource.  This review focuses on marine natural compds. with epigenetic activities and their synthetic derivs. displaying anti-cancer properties including largazole, psammaplins, trichostatins and azumamides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvkq7FWSi1ALVg90H21EOLACvtfcHk0lgOMakgW-1AOg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGmtLbJ&md5=8b354ad10012c11b443e5130bf4e86ce</span></div><a href="/servlet/linkout?suffix=cit2c&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DSchnekenburger%26aufirst%3DM.%26aulast%3DDicato%26aufirst%3DM.%26aulast%3DDiederich%26aufirst%3DM.%26atitle%3DEpigenetic%2520modulators%2520from%2520%25E2%2580%25B3The%2520Big%2520Blue%25E2%2580%25B3%253A%2520a%2520treasure%2520to%2520fight%2520against%2520cancer%26jtitle%3DCancer%2520Lett.%26date%3D2014%26volume%3D351%26spage%3D182%26epage%3D197%26doi%3D10.1016%2Fj.canlet.2014.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">LÃ¶tsch, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parnham, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geisslinger, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doehring, A.</span><span> </span><span class="NLM_article-title">Common non-epigenetic drugs as epigenetic modulators</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">â</span> <span class="NLM_lpage">753</span><span class="refDoi">Â DOI: 10.1016/j.molmed.2013.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.molmed.2013.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=24054876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FgtValtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=742-753&author=J.+L%C3%B6tschauthor=G.+Schneiderauthor=D.+Rekerauthor=M.+J.+Parnhamauthor=P.+Schneiderauthor=G.+Geisslingerauthor=A.+Doehring&title=Common+non-epigenetic+drugs+as+epigenetic+modulators&doi=10.1016%2Fj.molmed.2013.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2dR"><div class="casContent"><span class="casTitleNuber">2d</span><div class="casTitle"><span class="NLM_cas:atitle">Common non-epigenetic drugs as epigenetic modulators</span></div><div class="casAuthors">Lotsch Jorn; Schneider Gisbert; Reker Daniel; Parnham Michael J; Schneider Petra; Geisslinger Gerd; Doehring Alexandra</div><div class="citationInfo"><span class="NLM_cas:title">Trends in molecular medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">742-53</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetic effects are exerted by a variety of factors and evidence increases that common drugs such as opioids, cannabinoids, valproic acid, or cytostatics may induce alterations in DNA methylation patterns or histone conformations.  These effects occur via chemical structural interactions with epigenetic enzymes, through interactions with DNA repair mechanisms.  Computational predictions indicate that one-twentieth of all drugs might potentially interact with human histone deacetylase, which was prospectively experimentally verified for the compound with the highest predicted interaction probability.  These epigenetic effects add to wanted and unwanted drug effects, contributing to mechanisms of drug resistance or disease-related and unrelated phenotypes.  Because epigenetic changes might be transmitted to offspring, the need for reliable and cost-effective epigenetic screening tools becomes acute.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRjgfDAzktxzC-bLEOO4dYpfW6udTcc2ebgOctR2V3Ufbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FgtValtA%253D%253D&md5=a01992b5ad8dac32dd0db860502ad25d</span></div><a href="/servlet/linkout?suffix=cit2d&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2013.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2013.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6tsch%26aufirst%3DJ.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DReker%26aufirst%3DD.%26aulast%3DParnham%26aufirst%3DM.%2BJ.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DGeisslinger%26aufirst%3DG.%26aulast%3DDoehring%26aufirst%3DA.%26atitle%3DCommon%2520non-epigenetic%2520drugs%2520as%2520epigenetic%2520modulators%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D742%26epage%3D753%26doi%3D10.1016%2Fj.molmed.2013.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinmann, H.</span><span> </span><span class="NLM_article-title">Small-molecule modulators for epigenetics targets</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1885</span><span class="NLM_x">â</span> <span class="NLM_lpage">1891</span><span class="refDoi">Â DOI: 10.1002/cmdc.201300344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1002%2Fcmdc.201300344" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1885-1891&author=S.+Knappauthor=H.+Weinmann&title=Small-molecule+modulators+for+epigenetics+targets&doi=10.1002%2Fcmdc.201300344"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2e&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300344%26sid%3Dliteratum%253Aachs%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DSmall-molecule%2520modulators%2520for%2520epigenetics%2520targets%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1885%26epage%3D1891%26doi%3D10.1002%2Fcmdc.201300344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">A crack in histone lysine methylation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">903</span><span class="NLM_x">â</span> <span class="NLM_lpage">906</span><span class="refDoi">Â DOI: 10.1016/j.cell.2004.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.cell.2004.12.006" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=903-906&author=S.+Kubicekauthor=T.+Jenuwein&title=A+crack+in+histone+lysine+methylation&doi=10.1016%2Fj.cell.2004.12.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DA%2520crack%2520in%2520histone%2520lysine%2520methylation%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D903%26epage%3D906%26doi%3D10.1016%2Fj.cell.2004.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">â</span> <span class="NLM_lpage">953</span><span class="refDoi">Â DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lhTdUgOfmddAQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallin, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">JmjC-domain-containing proteins and histone demethylation</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">715</span><span class="NLM_x">â</span> <span class="NLM_lpage">727</span><span class="refDoi">Â DOI: 10.1038/nrg1945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnrg1945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=16983801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1Ggu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=715-727&author=R.+J.+Kloseauthor=E.+M.+Kallinauthor=Y.+Zhang&title=JmjC-domain-containing+proteins+and+histone+demethylation&doi=10.1038%2Fnrg1945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">JmjC-domain-containing proteins and histone demethylation</span></div><div class="casAuthors">Klose, Robert J.; Kallin, Eric M.; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">715-727</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation has important roles in regulating gene expression and forms part of the epigenetic memory system that regulates cell fate and identity.  Enzymes that directly remove Me marks from histones have recently been identified, revealing a new level of plasticity within this epigenetic modification system.  Here we analyze the evolutionary relationship between Jumonji C (JmjC)-domain-contg. proteins and discuss their cellular functions in relation to their potential enzymic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPb9c8M0AJzLVg90H21EOLACvtfcHk0lh_uLZt_oc_mQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1Ggu74%253D&md5=1ac8a66426993770ae33b78ea35f5879</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1038%2Fnrg1945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1945%26sid%3Dliteratum%253Aachs%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DKallin%26aufirst%3DE.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DJmjC-domain-containing%2520proteins%2520and%2520histone%2520demethylation%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2006%26volume%3D7%26spage%3D715%26epage%3D727%26doi%3D10.1038%2Fnrg1945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone lysine demethylases: emerging roles in development, physiology and disease</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">829</span><span class="NLM_x">â</span> <span class="NLM_lpage">833</span><span class="refDoi">Â DOI: 10.1038/nrg2218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnrg2218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=17909537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOksL7P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=829-833&author=Y.+Shi&title=Histone+lysine+demethylases%3A+emerging+roles+in+development%2C+physiology+and+disease&doi=10.1038%2Fnrg2218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4cR"><div class="casContent"><span class="casTitleNuber">4c</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylases: emerging roles in development, physiology and disease</span></div><div class="casAuthors">Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">829-833</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Functions of histone lysine demethylases in a range of developmental and physiol. processes are rapidly being uncovered, as are the roles of these enzymes in disease.  Histone demethylases also provide a promising new route towards the therapeutic targeting of epigenetic regulators.  The discovery of an increasing no. of histone demethylases has highlighted the dynamic nature of the regulation of histone methylation, a key chromatin modification that is involved in eukaryotic genome and gene regulation.  A flurry of recent studies has offered glimpses into the specific biol. roles of these enzymes and their potential connections to human diseases.  These advances have also catalyzed a resurgence of interest in epigenetic regulators as potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ_dGIbAr6QLVg90H21EOLACvtfcHk0lh_uLZt_oc_mQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOksL7P&md5=d8bdeeb298c8afdcd9d0d5d5a62f6cc9</span></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.1038%2Fnrg2218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2218%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520lysine%2520demethylases%253A%2520emerging%2520roles%2520in%2520development%252C%2520physiology%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2007%26volume%3D8%26spage%3D829%26epage%3D833%26doi%3D10.1038%2Fnrg2218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooch, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">â</span> <span class="NLM_lpage">435</span><span class="refDoi">Â DOI: 10.1038/nature04021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnature04021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=16079794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=432-435&author=M.+G.+Leeauthor=C.+Wynderauthor=N.+Coochauthor=R.+Shiekhattar&title=An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation&doi=10.1038%2Fnature04021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Cooch, Neil; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">432-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors have previously described a multiprotein complex termed the BHC or BRAF-HDAC complex, which is required for the repression of neuronal-specific genes.  The authors have shown that the BHC complex is recruited by a neuronal silencer, REST (RE1-silencing transcription factor), and mediates the repression of REST-responsive genes.  BHC is a multiprotein complex consisting of two enzymic activities: a histone deacetylase (HDAC1 or 2) and a recently described histone demethylase (BHC110, also known as LSD1 or AOF2).  Here the authors show that BHC110-contg. complexes show a nearly fivefold increase in demethylation of histone H3 lysine 4 (H3K4) compared to recombinant BHC110.  Furthermore, recombinant BHC110 is unable to demethylate H3K4 on nucleosomes, but BHC110-contg. complexes readily demethylate nucleosomes.  In vitro reconstitution of the BHC complex using recombinant subunits reveals an essential role for the REST corepressor CoREST, not only in stimulating demethylation on core histones but also promoting demethylation of nucleosomal substrates.  The authors find that nucleosomal demethylation is the result of CoREST enhancing the assocn. between BHC110 and nucleosomes.  Depletion of CoREST in in vivo cell culture results in de-repression of REST-responsive gene expression and increased methylation of H3K4.  Together, these results highlight an essential role for CoREST in demethylation of H3K4 both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmTdZApayNhrVg90H21EOLACvtfcHk0lh_uLZt_oc_mQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D&md5=535a6d800bd6672192be133516b4dee7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature04021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04021%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DAn%2520essential%2520role%2520for%2520CoREST%2520in%2520nucleosomal%2520histone%25203%2520lysine%25204%2520demethylation%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D432%26epage%3D435%26doi%3D10.1038%2Fnature04021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SchÃ¼le, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: implications for therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">â</span> <span class="NLM_lpage">2071</span><span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+is+strongly+expressed+in+poorly+differentiated+neuroblastoma%3A+implications+for+therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0lgJAFG7ihR5vw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520strongly%2520expressed%2520in%2520poorly%2520differentiated%2520neuroblastoma%253A%2520implications%2520for%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Schenk, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">GÃ¶llner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebestreit, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popescu, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marton, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MÃ¼ller-Tidow, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zelent, A.</span><span> </span><span class="NLM_article-title">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">â</span> <span class="NLM_lpage">611</span><span class="refDoi">Â DOI: 10.1038/nm.2661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnm.2661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=22406747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=605-611&author=T.+Schenkauthor=W.+C.+Chenauthor=S.+G%C3%B6llnerauthor=L.+Howellauthor=L.+Jinauthor=K.+Hebestreitauthor=H.+U.+Kleinauthor=A.+C.+Popescuauthor=A.+Burnettauthor=K.+Millsauthor=R.+A.+Caseroauthor=L.+Martonauthor=P.+Wosterauthor=M.+D.+Mindenauthor=M.+Dugasauthor=J.+C.+Wangauthor=J.+E.+Dickauthor=C.+M%C3%BCller-Tidowauthor=K.+Petrieauthor=A.+Zelent&title=Inhibition+of+the+LSD1+%28KDM1A%29+demethylase+reactivates+the+all-trans-retinoic+acid+differentiation+pathway+in+acute+myeloid+leukemia&doi=10.1038%2Fnm.2661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span></div><div class="casAuthors">Schenk, Tino; Chen, Weihsu Claire; Goellner, Stefanie; Howell, Louise; Jin, Liqing; Hebestreit, Katja; Klein, Hans-Ulrich; Popescu, Andreea C.; Burnett, Alan; Mills, Ken; Casero, Robert A.; Marton, Laurence; Woster, Patrick; Minden, Mark D.; Dugas, Martin; Wang, Jean C. Y.; Dick, John E.; Mueller-Tidow, Carsten; Petrie, Kevin; Zelent, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">605-611</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acute promyelocytic leukemia (APL), a cytogenetically distinct subtype of acute myeloid leukemia (AML), characterized by the t(15;17)-assocd. PML-RARA fusion, has been successfully treated with therapy utilizing all-trans-retinoic acid (ATRA) to differentiate leukemic blasts.  However, among patients with non-APL AML, ATRA-based treatment has not been effective.  Here we show that, through epigenetic reprogramming, inhibitors of lysine-specific demethylase 1 (LSD1, also called KDM1A), including tranylcypromine (TCP), unlocked the ATRA-driven therapeutic response in non-APL AML.  LSD1 inhibition did not lead to a large-scale increase in histone 3 Lys4 dimethylation (H3K4me2) across the genome, but it did increase H3K4me2 and expression of myeloid-differentiation-assocd. genes.  Notably, treatment with ATRA plus TCP markedly diminished the engraftment of primary human AML cells in vivo in nonobese diabetic (NOD)-severe combined immunodeficient (SCID) mice, suggesting that ATRA in combination with TCP may target leukemia-initiating cells.  Furthermore, initiation of ATRA plus TCP treatment 15 d after engraftment of human AML cells in NOD-SCID Î³ (with interleukin-2 (IL-2) receptor Î³ chain deficiency) mice also revealed the ATRA plus TCP drug combination to have a potent anti-leukemic effect that was superior to treatment with either drug alone.  These data identify LSD1 as a therapeutic target and strongly suggest that it may contribute to AML pathogenesis by inhibiting the normal pro-differentiative function of ATRA, paving the way for new combinatorial therapies for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU7ebz8Ar4AbVg90H21EOLACvtfcHk0lgJAFG7ihR5vw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D&md5=57d968de189123149a2714df0208725f</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fnm.2661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2661%26sid%3Dliteratum%253Aachs%26aulast%3DSchenk%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DW.%2BC.%26aulast%3DG%25C3%25B6llner%26aufirst%3DS.%26aulast%3DHowell%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHebestreit%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DH.%2BU.%26aulast%3DPopescu%26aufirst%3DA.%2BC.%26aulast%3DBurnett%26aufirst%3DA.%26aulast%3DMills%26aufirst%3DK.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DMarton%26aufirst%3DL.%26aulast%3DWoster%26aufirst%3DP.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DDugas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BC.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DM%25C3%25BCller-Tidow%26aufirst%3DC.%26aulast%3DPetrie%26aufirst%3DK.%26aulast%3DZelent%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520the%2520LSD1%2520%2528KDM1A%2529%2520demethylase%2520reactivates%2520the%2520all-trans-retinoic%2520acid%2520differentiation%2520pathway%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D605%26epage%3D611%26doi%3D10.1038%2Fnm.2661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogel, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayanan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristie, T. M.</span><span> </span><span class="NLM_article-title">Inhibition of the histone demethylase LSD1 blocks Î±-herpesvirus lytic replication and reactivation from latency</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1312</span><span class="NLM_x">â</span> <span class="NLM_lpage">1317</span><span class="refDoi">Â DOI: 10.1038/nm.2051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnm.2051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=19855399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSqsLbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=1312-1317&author=Y.+Liangauthor=J.+L.+Vogelauthor=A.+Narayananauthor=H.+Pengauthor=T.+M.+Kristie&title=Inhibition+of+the+histone+demethylase+LSD1+blocks+%CE%B1-herpesvirus+lytic+replication+and+reactivation+from+latency&doi=10.1038%2Fnm.2051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the histone demethylase LSD1 blocks Î±-herpesvirus lytic replication and reactivation from latency</span></div><div class="casAuthors">Liang, Yu; Vogel, Jodi L.; Narayanan, Aarthi; Peng, Hua; Kristie, Thomas M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1312-1317</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reversible methylation of histone tails serves as either a pos. signal recognized by transcriptional assemblies or a neg. signal that result in repression.  Invading viral pathogens that depend upon the host cell's transcriptional app. are also subject to the regulatory impact of chromatin assembly and modifications.  Here we show that infection by the Î±-herpesviruses, herpes simplex virus (HSV) and varicella zoster virus (VZV), results in the rapid accumulation of chromatin bearing repressive histone H3 Lys9 methylation.  To enable expression of viral immediate early (IE) genes, both viruses use the cellular transcriptional coactivator host cell factor-1 (HCF-1) to recruit the lysine-specific demethylase-1 (LSD1) to the viral immediate early promoters.  Depletion of LSD1 or inhibition of its activity with monoamine oxidase inhibitors (MAOIs) results in the accumulation of repressive chromatin and a block to viral gene expression.  As HCF-1 is a component of the Set1 and MLL1 histone H3 Lys4 methyltransferase complexes, it thus coordinates modulation of repressive H3 Lys9 methylation levels with addn. of activating H3 Lys4 trimethylation marks.  Strikingly, MAOIs also block the reactivation of HSV from latency in sensory neurons, indicating that the HCF-1 complex is a crucial component of the reactivation mechanism.  The results support pharmaceutical control of histone modifying enzymes as a strategy for controlling herpesvirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppT5vH8lPRnbVg90H21EOLACvtfcHk0ljS1Opdw5QDpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSqsLbF&md5=bd9aacfdbcba54e30f9482f8fcc96777</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1038%2Fnm.2051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2051%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DVogel%26aufirst%3DJ.%2BL.%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DPeng%26aufirst%3DH.%26aulast%3DKristie%26aufirst%3DT.%2BM.%26atitle%3DInhibition%2520of%2520the%2520histone%2520demethylase%2520LSD1%2520blocks%2520%25CE%25B1-herpesvirus%2520lytic%2520replication%2520and%2520reactivation%2520from%2520latency%26jtitle%3DNat.%2520Med.%26date%3D2009%26volume%3D15%26spage%3D1312%26epage%3D1317%26doi%3D10.1038%2Fnm.2051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanabe, O.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is a therapeutic target for fetal hemoglobin induction</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">â</span> <span class="NLM_lpage">294</span><span class="refDoi">Â DOI: 10.1038/nm.3101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnm.3101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=23416702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Wrurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=291-294&author=L.+Shiauthor=S.+Cuiauthor=J.+D.+Engelauthor=O.+Tanabe&title=Lysine-specific+demethylase+1+is+a+therapeutic+target+for+fetal+hemoglobin+induction&doi=10.1038%2Fnm.3101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is a therapeutic target for fetal hemoglobin induction</span></div><div class="casAuthors">Shi, Lihong; Cui, Shuaiying; Engel, James D.; Tanabe, Osamu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">291-294</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Enhanced fetal Î³-globin synthesis alleviates symptoms of Î²-globinopathies such as sickle cell disease and Î²-thalassemia, but current Î³-globin-inducing drugs offer limited beneficial effects.  We show here that lysine-specific demethylase 1 (LSD1) inhibition by RNAi in human erythroid cells or by the monoamine oxidase inhibitor tranylcypromine in human erythroid cells or Î²-type globin-transgenic mice enhances Î³-globin expression.  LSD1 is thus a promising therapeutic target for Î³-globin induction, and tranylcypromine may serve as a lead compd. for the development of a new Î³-globin inducer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPuLk6KrTQ-rVg90H21EOLACvtfcHk0ljS1Opdw5QDpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Wrurs%253D&md5=3747256331d40eb16b2ce54a85df1761</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1038%2Fnm.3101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3101%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DS.%26aulast%3DEngel%26aufirst%3DJ.%2BD.%26aulast%3DTanabe%26aufirst%3DO.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520a%2520therapeutic%2520target%2520for%2520fetal%2520hemoglobin%2520induction%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D291%26epage%3D294%26doi%3D10.1038%2Fnm.3101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine demethylases inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">â</span> <span class="NLM_lpage">8250</span><span class="refDoi">Â DOI: 10.1021/jm201048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+demethylases+inhibitors&doi=10.1021%2Fjm201048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0ljS1Opdw5QDpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520demethylases%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250%26doi%3D10.1021%2Fjm201048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chi, Y. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">The SNAG domain of Snail1 functions as a molecular hook for recruiting lysine-specific demethylase 1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1803</span><span class="NLM_x">â</span> <span class="NLM_lpage">1816</span><span class="refDoi">Â DOI: 10.1038/emboj.2010.63</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Femboj.2010.63" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=1803-1816&author=Y.+Linauthor=Y.+Wuauthor=J.+Liauthor=C.+Dongauthor=X.+Yeauthor=Y.+I.+Chiauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=The+SNAG+domain+of+Snail1+functions+as+a+molecular+hook+for+recruiting+lysine-specific+demethylase+1&doi=10.1038%2Femboj.2010.63"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.63&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.63%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DC.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DChi%26aufirst%3DY.%2BI.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DThe%2520SNAG%2520domain%2520of%2520Snail1%2520functions%2520as%2520a%2520molecular%2520hook%2520for%2520recruiting%2520lysine-specific%2520demethylase%25201%26jtitle%3DEMBO%2520J.%26date%3D2010%26volume%3D29%26spage%3D1803%26epage%3D1816%26doi%3D10.1038%2Femboj.2010.63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit9b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span> </span><span class="NLM_article-title">Requirement of the histone demethylase LSD1 in Snai1-mediated transcriptional repression during epithelial-mesenchymal transition</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">4896</span><span class="NLM_x">â</span> <span class="NLM_lpage">4904</span><span class="refDoi">Â DOI: 10.1038/onc.2010.234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fonc.2010.234" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=4896-4904&author=T.+Linauthor=A.+Ponnauthor=X.+Huauthor=B.+K.+Lawauthor=J.+Lu&title=Requirement+of+the+histone+demethylase+LSD1+in+Snai1-mediated+transcriptional+repression+during+epithelial-mesenchymal+transition&doi=10.1038%2Fonc.2010.234"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.234%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DT.%26aulast%3DPonn%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLaw%26aufirst%3DB.%2BK.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DRequirement%2520of%2520the%2520histone%2520demethylase%2520LSD1%2520in%2520Snai1-mediated%2520transcriptional%2520repression%2520during%2520epithelial-mesenchymal%2520transition%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26spage%3D4896%26epage%3D4904%26doi%3D10.1038%2Fonc.2010.234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Ferrari-Amorotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fragliasso, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esteki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prudente, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soliera, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cattelani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manzotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grisendi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieraccioli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">RaschellÃ , G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiodoni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calabretta, B.</span><span> </span><span class="NLM_article-title">Inhibiting interactions of lysine demethylase LSD1 with snail/slug blocks cancer cell invasion</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">â</span> <span class="NLM_lpage">245</span><span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-12-1739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1158%2F0008-5472.CAN-12-1739" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=235-245&author=G.+Ferrari-Amorottiauthor=V.+Fragliassoauthor=R.+Estekiauthor=Z.+Prudenteauthor=A.+R.+Solieraauthor=S.+Cattelaniauthor=G.+Manzottiauthor=G.+Grisendiauthor=M.+Dominiciauthor=M.+Pieraccioliauthor=G.+Raschell%C3%A0author=C.+Chiodoniauthor=M.+P.+Colomboauthor=B.+Calabretta&title=Inhibiting+interactions+of+lysine+demethylase+LSD1+with+snail%2Fslug+blocks+cancer+cell+invasion&doi=10.1158%2F0008-5472.CAN-12-1739"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-1739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-1739%26sid%3Dliteratum%253Aachs%26aulast%3DFerrari-Amorotti%26aufirst%3DG.%26aulast%3DFragliasso%26aufirst%3DV.%26aulast%3DEsteki%26aufirst%3DR.%26aulast%3DPrudente%26aufirst%3DZ.%26aulast%3DSoliera%26aufirst%3DA.%2BR.%26aulast%3DCattelani%26aufirst%3DS.%26aulast%3DManzotti%26aufirst%3DG.%26aulast%3DGrisendi%26aufirst%3DG.%26aulast%3DDominici%26aufirst%3DM.%26aulast%3DPieraccioli%26aufirst%3DM.%26aulast%3DRaschell%25C3%25A0%26aufirst%3DG.%26aulast%3DChiodoni%26aufirst%3DC.%26aulast%3DColombo%26aufirst%3DM.%2BP.%26aulast%3DCalabretta%26aufirst%3DB.%26atitle%3DInhibiting%2520interactions%2520of%2520lysine%2520demethylase%2520LSD1%2520with%2520snail%252Fslug%2520blocks%2520cancer%2520cell%2520invasion%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D235%26epage%3D245%26doi%3D10.1158%2F0008-5472.CAN-12-1739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrello, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiarelli, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vellore, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâConnell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1677</span><span class="NLM_x">â</span> <span class="NLM_lpage">1682</span><span class="refDoi">Â DOI: 10.1021/cb4001926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4001926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1677-1682&author=M.+Tortoriciauthor=M.+T.+Borrelloauthor=M.+Tardugnoauthor=L.+R.+Chiarelliauthor=S.+Pilottoauthor=G.+Ciossaniauthor=N.+A.+Velloreauthor=S.+G.+Baileyauthor=J.+Cowanauthor=M.+O%E2%80%99Connellauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Maiauthor=R.+Baronauthor=A.+Ganesanauthor=A.+Mattevi&title=Protein+recognition+by+short+peptide+reversible+inhibitors+of+the+chromatin-modifying+LSD1%2FCoREST+lysine+demethylase&doi=10.1021%2Fcb4001926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span></div><div class="casAuthors">Tortorici, Marcello; Borrello, Maria Teresa; Tardugno, Maria; Chiarelli, Laurent R.; Pilotto, Simona; Ciossani, Giuseppe; Vellore, Nadeem A.; Bailey, Sarah G.; Cowan, Jonathan; O'Connell, Maria; Crabb, Simon J.; Packham, Graham; Mai, Antonello; Baron, Riccardo; Ganesan, A.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1677-1682</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The combinatorial assembly of protein complexes is at the heart of chromatin biol.  Lysine demethylase LSD1(KDM1A)/CoREST beautifully exemplifies this concept.  The active site of the enzyme tightly assocs. to the N-terminal domain of transcription factors of the SNAIL1 family, which therefore can competitively inhibit the binding of the N-terminal tail of the histone substrate.  Our enzymic, crystallog., spectroscopic, and computational studies reveal that LSD1/CoREST can bind to a hexapeptide derived from the SNAIL sequence through recognition of a pos. charged Î±-helical turn that forms upon binding to the enzyme.  Variations in sequence and length of this six amino acid ligand modulate affinities enabling the same binding site to differentially interact with proteins that exert distinct biol. functions.  The discovered short peptide inhibitors exhibit antiproliferative activities and lay the foundation for the development of peptidomimetic small mol. inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7SufQOwK-y7Vg90H21EOLACvtfcHk0li1fc0kqN7VlA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D&md5=a8479b513e392e8acb78979e116dcffa</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb4001926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4001926%26sid%3Dliteratum%253Aachs%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DBorrello%26aufirst%3DM.%2BT.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DChiarelli%26aufirst%3DL.%2BR.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DVellore%26aufirst%3DN.%2BA.%26aulast%3DBailey%26aufirst%3DS.%2BG.%26aulast%3DCowan%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DM.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DProtein%2520recognition%2520by%2520short%2520peptide%2520reversible%2520inhibitors%2520of%2520the%2520chromatin-modifying%2520LSD1%252FCoREST%2520lysine%2520demethylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1677%26epage%3D1682%26doi%3D10.1021%2Fcb4001926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Small-molecular modulators of cancer-associated epigenetic mechanisms</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">â</span> <span class="NLM_lpage">896</span><span class="refDoi">Â DOI: 10.1039/c3mb25410k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1039%2Fc3mb25410k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=23511667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVOlsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=873-896&author=Y.+Itohauthor=T.+Suzukiauthor=N.+Miyata&title=Small-molecular+modulators+of+cancer-associated+epigenetic+mechanisms&doi=10.1039%2Fc3mb25410k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecular modulators of cancer-associated epigenetic mechanisms</span></div><div class="casAuthors">Itoh, Yukihiro; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">873-896</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  DNA methylation and posttranslational histone modifications regulate expression of various genes independently of changes in the DNA sequence.  Such epigenetic mechanisms play important roles in controlling cellular functions, including the cell cycle, immunoresponses and signal transduction.  On the other hand, epigenetic aberrations are assocd. with oncogenesis and proliferation of cancer cells, and epigenetic alterations have been identified in many human cancer cells.  Furthermore, chem.-biol. approaches have uncovered relationships between epigenetic dysregulation and cancer, and several small mols. that modulate epigenetic mechanisms have already been approved for cancer therapy.  In this review, we deal with chem. epigenetics relevant to cancer therapy, focusing esp. on small mols. that regulate epigenetic mechanisms related to DNA methylation and histone modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo76xFXo9kCe7Vg90H21EOLACvtfcHk0ljeaWh-5_0H6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVOlsr4%253D&md5=ecc7cd46637f6010872510329d51ac90</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1039%2Fc3mb25410k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3mb25410k%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSmall-molecular%2520modulators%2520of%2520cancer-associated%2520epigenetic%2520mechanisms%26jtitle%3DMol.%2520BioSyst.%26date%3D2013%26volume%3D9%26spage%3D873%26epage%3D896%26doi%3D10.1039%2Fc3mb25410k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title"><i>trans</i>-2-Phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">4408</span><span class="NLM_x">â</span> <span class="NLM_lpage">4416</span><span class="refDoi">Â DOI: 10.1021/bi0618621</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0618621" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=4408-4416&author=D.+M.+Schmidtauthor=D.+G.+McCafferty&title=trans-2-Phenylcyclopropylamine+is+a+mechanism-based+inactivator+of+the+histone+demethylase+LSD1&doi=10.1021%2Fbi0618621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">trans-2-Phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span></div><div class="casAuthors">Schmidt, Dawn M. Z.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4408-4416</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic domain of the flavin-dependent human histone demethylase, lysine-specific demethylase 1 (LSD1), belongs to the family of amine oxidases including polyamine oxidase and monoamine oxidase (MAO).  The authors previously assessed MAO inhibitors (MAOIs) for their ability to inhibit the reaction catalyzed by LSD1, demonstrating that trans-2-phenylcyclopropylamine (2-PCPA; tranylcypromine, Parnate) was the most potent inhibitor with respect to LSD1.  Here, the authors showed that 2-PCPA was a time-dependent, mechanism-based irreversible inhibitor of LSD1 with a Ki of 242 Î¼M and a kinact of 0.0106 s-1.  2-PCPA showed limited selectivity for human MAOs vs. LSD1, with kinact/Ki values only 16-fold and 2.4-fold higher for MAO B and MAO A, resp.  Profiles of LSD1 activity and inactivation by 2-PCPA as a function of pH were consistent with a mechanism of inactivation dependent upon enzyme catalysis.  Mass spectrometry supported a role for FAD as the site of covalent modification by 2-PCPA.  These results will provide a foundation for the design of cyclopropylamine-based inhibitors that are selective for LSD1 to probe its role in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVK59U7IsBvbVg90H21EOLACvtfcHk0ljeaWh-5_0H6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D&md5=02925342b86c7dbfb5f1a99fa5264adc</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1021%2Fbi0618621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0618621%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3Dtrans-2-Phenylcyclopropylamine%2520is%2520a%2520mechanism-based%2520inactivator%2520of%2520the%2520histone%2520demethylase%2520LSD1%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D4408%26epage%3D4416%26doi%3D10.1021%2Fbi0618621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jalili, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ball, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span> </span><span class="NLM_article-title">Structural basis for the inhibition of the LSD1 histone demethylase by the antidepressant <i>trans</i>-2-phenylcyclopropylamine</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">8058</span><span class="NLM_x">â</span> <span class="NLM_lpage">8065</span><span class="refDoi">Â DOI: 10.1021/bi700664y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi700664y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsVKmsr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=8058-8065&author=M.+Yangauthor=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=P.+Jaliliauthor=H.+L.+Ballauthor=M.+Machiusauthor=P.+A.+Coleauthor=H.+Yu&title=Structural+basis+for+the+inhibition+of+the+LSD1+histone+demethylase+by+the+antidepressant+trans-2-phenylcyclopropylamine&doi=10.1021%2Fbi700664y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12cR"><div class="casContent"><span class="casTitleNuber">12c</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Inhibition of the LSD1 Histone Demethylase by the Antidepressant trans-2-Phenylcyclopropylamine</span></div><div class="casAuthors">Yang, Maojun; Culhane, Jeffrey C.; Szewczuk, Lawrence M.; Jalili, Pegah; Ball, Haydn L.; Machius, Mischa; Cole, Philip A.; Yu, Hongtao</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8058-8065</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone modifications, such as acetylation and methylation, are important epigenetic marks that regulate diverse biol. processes that use chromatin as the template, including transcription.  Dysregulation of histone acetylation and methylation leads to the silencing of tumor suppressor genes and contributes to cancer progression.  Inhibitors of enzymes that catalyze the addn. and removal of these epigenetic marks thus have therapeutic potential for treating cancer.  Lysine-specific demethylase 1 (LSD1) is the first discovered histone lysine demethylase and with the help of its cofactor CoREST, specifically demethylates mono- and dimethylated histone H3 lysine 4 (H3-K4), thus repressing transcription.  Because LSD1 belongs to the family of FAD (FAD)-dependent amine oxidases, certain inhibitors of monoamine oxidases (MAOs), including the clin. used antidepressant trans-2-phenylcyclopropylamine (PCPA; tranylcypromine; Parnate), are also capable of inhibiting LSD1.  In this study, we have further measured the kinetic parameters of the inhibition of LSD1 by PCPA and detd. the crystal structure of LSD1-CoREST in the presence of PCPA.  Our structural and mass spectrometry analyses are consistent with PCPA forming a covalent adduct with FAD in LSD1 that is distinct from the FAD-PCPA adduct of MAO B.  The structure also reveals that the Ph ring of the FAD-PCPA adduct in LSD1 does not form extensive interactions with active-site residues.  This study thus provides the basis for designing more potent inhibitors of LSD1 that contain substitutions on the Ph ring of PCPA to fully engage neighboring residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLqcLmzpB1S7Vg90H21EOLACvtfcHk0ljeaWh-5_0H6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsVKmsr8%253D&md5=e11fe93832d46962b6c4848f682042d2</span></div><a href="/servlet/linkout?suffix=cit12c&amp;dbid=16384&amp;doi=10.1021%2Fbi700664y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi700664y%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DJalili%26aufirst%3DP.%26aulast%3DBall%26aufirst%3DH.%2BL.%26aulast%3DMachius%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DH.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibition%2520of%2520the%2520LSD1%2520histone%2520demethylase%2520by%2520the%2520antidepressant%2520trans-2-phenylcyclopropylamine%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D8058%26epage%3D8065%26doi%3D10.1021%2Fbi700664y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Identification of cell-active lysine specific demethylase 1-selective inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">17536</span><span class="NLM_x">â</span> <span class="NLM_lpage">17537</span><span class="refDoi">Â DOI: 10.1021/ja907055q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja907055q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVWrtbzO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=17536-17537&author=R.+Uedaauthor=T.+Suzukiauthor=K.+Minoauthor=H.+Tsumotoauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyata&title=Identification+of+cell-active+lysine+specific+demethylase+1-selective+inhibitors&doi=10.1021%2Fja907055q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12dR"><div class="casContent"><span class="casTitleNuber">12d</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of cell-active lysine specific demethylase 1-selective inhibitors</span></div><div class="casAuthors">Ueda, Rie; Suzuki, Takayoshi; Mino, Koshiki; Tsumoto, Hiroki; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">17536-17537</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated Lys4 of histone H3 (H3K4).  Here we report the identification of the first small-mol. LSD1-selective inhibitors.  These inhibitors show in vivo H3K4-methylating activity and antiproliferative activity and should be useful as lead structures for anticancer drugs and as tools for studying the biol. roles of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH7uDFiErmKbVg90H21EOLACvtfcHk0liKXSYsjCBwow"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVWrtbzO&md5=243e81a1d8d8cb3747f8307d5f42cd29</span></div><a href="/servlet/linkout?suffix=cit12d&amp;dbid=16384&amp;doi=10.1021%2Fja907055q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja907055q%26sid%3Dliteratum%253Aachs%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520cell-active%2520lysine%2520specific%2520demethylase%25201-selective%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D17536%26epage%3D17537%26doi%3D10.1021%2Fja907055q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsubara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Synthesis and biological activity of optically active NCL-1, a lysine-specific demethylase 1 selective inhibitor</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3702</span><span class="NLM_x">â</span> <span class="NLM_lpage">3708</span><span class="refDoi">Â DOI: 10.1016/j.bmc.2010.12.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.bmc.2010.12.024" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3702-3708&author=D.+Ogasawaraauthor=T.+Suzukiauthor=K.+Minoauthor=R.+Uedaauthor=M.+N.+Khanauthor=T.+Matsubaraauthor=K.+Kosekiauthor=M.+Hasegawaauthor=R.+Sasakiauthor=H.+Nakagawaauthor=T.+Mizukamiauthor=N.+Miyata&title=Synthesis+and+biological+activity+of+optically+active+NCL-1%2C+a+lysine-specific+demethylase+1+selective+inhibitor&doi=10.1016%2Fj.bmc.2010.12.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12e&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.12.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.12.024%26sid%3Dliteratum%253Aachs%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DKhan%26aufirst%3DM.%2BN.%26aulast%3DMatsubara%26aufirst%3DT.%26aulast%3DKoseki%26aufirst%3DK.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSynthesis%2520and%2520biological%2520activity%2520of%2520optically%2520active%2520NCL-1%252C%2520a%2520lysine-specific%2520demethylase%25201%2520selective%2520inhibitor%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3702%26epage%3D3708%26doi%3D10.1016%2Fj.bmc.2010.12.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12f"><span><span class="NLM_label">(f) </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuhara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1-selective inactivators: protein-targeted drug delivery mechanism</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">8620</span><span class="NLM_x">â</span> <span class="NLM_lpage">8624</span><span class="refDoi">Â DOI: 10.1002/anie.201303999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1002%2Fanie.201303999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=8620-8624&author=D.+Ogasawaraauthor=Y.+Itohauthor=H.+Tsumotoauthor=T.+Kakizawaauthor=K.+Minoauthor=K.+Fukuharaauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyataauthor=T.+Suzuki&title=Lysine-specific+demethylase+1-selective+inactivators%3A+protein-targeted+drug+delivery+mechanism&doi=10.1002%2Fanie.201303999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12fR"><div class="casContent"><span class="casTitleNuber">12f</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1-selective inactivators: Protein-targeted drug delivery mechanism</span></div><div class="casAuthors">Ogasawara, Daisuke; Itoh, Yukihiro; Tsumoto, Hiroki; Kakizawa, Taeko; Mino, Koshiki; Fukuhara, Kiyoshi; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">8620-8624</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of novel LSD1 (LSD1 = lysine-specific demethylase) inactivators based on new concept of LSD1-targeted PCPA (PCPA = trans-2-phenylcyclopropylamine) delivery have been designed.  This concept was tested with peptide 1a [1a = (ART-X-QTARKSTGGKAPRKQLA)] X = (I) and confirmed that 1a does indeed inhibit LSD1 selectively and efficiently by delivering PCPA directly to the LSD1 active site.  The enzyme is then inactivated by FAD-PCPA adduct formation, as in the case of PCPA itself.  This strategy could be applied to the design of nonpeptide, small-mol. LSD1 inactivators.  Mechanistic studies and biol. evaluations revealed that these small-mol. compds. are potent LSD1-selective inactivators (comparable to peptide 1a) in enzyme assays, and they also exhibit potent cell growth inhibitory activities against cancer cell lines.  This is the first time that a drug-carrier-type system has been employed to target a specific mol.  These novel LSD1-selective inactivators are considered to be candidates for anticancer agents, as well as bioprobes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqQOLhLwmv9rVg90H21EOLACvtfcHk0liKXSYsjCBwow"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF&md5=71bd05a50aeb73431444f2588fba2dac</span></div><a href="/servlet/linkout?suffix=cit12f&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303999%26sid%3Dliteratum%253Aachs%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DKakizawa%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DFukuhara%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DLysine-specific%2520demethylase%25201-selective%2520inactivators%253A%2520protein-targeted%2520drug%2520delivery%2520mechanism%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D8620%26epage%3D8624%26doi%3D10.1002%2Fanie.201303999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12g"><span><span class="NLM_label">(g) </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">â</span> <span class="NLM_lpage">6833</span><span class="refDoi">Â DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12gR"><div class="casContent"><span class="casTitleNuber">12g</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0lgmTrpTgflMig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit12g&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12h"><span><span class="NLM_label">(h) </span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally designed <i>trans</i>-2-phenylcyclopropylamine derivatives potently inhibit histone demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">â</span> <span class="NLM_lpage">6503</span><span class="refDoi">Â DOI: 10.1021/bi100299r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12h&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+designed+trans-2-phenylcyclopropylamine+derivatives+potently+inhibit+histone+demethylase+LSD1%2FKDM1&doi=10.1021%2Fbi100299r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12hR"><div class="casContent"><span class="casTitleNuber">12h</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0lgmTrpTgflMig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit12h&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520designed%2520trans-2-phenylcyclopropylamine%2520derivatives%2520potently%2520inhibit%2520histone%2520demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503%26doi%3D10.1021%2Fbi100299r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12i"><span><span class="NLM_label">(i) </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Comparative analysis of small molecules and histone substrate analogues as LSD1 lysine demethylase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">3164</span><span class="NLM_x">â</span> <span class="NLM_lpage">3176</span><span class="refDoi">Â DOI: 10.1021/ja909996p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja909996p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=3164-3176&author=J.+C.+Culhaneauthor=D.+Wangauthor=P.+M.+Yenauthor=P.+A.+Cole&title=Comparative+analysis+of+small+molecules+and+histone+substrate+analogues+as+LSD1+lysine+demethylase+inhibitors&doi=10.1021%2Fja909996p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12i&amp;dbid=16384&amp;doi=10.1021%2Fja909996p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja909996p%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DYen%26aufirst%3DP.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DComparative%2520analysis%2520of%2520small%2520molecules%2520and%2520histone%2520substrate%2520analogues%2520as%2520LSD1%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D3164%26epage%3D3176%26doi%3D10.1021%2Fja909996p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12j"><span><span class="NLM_label">(j) </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MÃ¼ller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">GÃ¼nther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SchÃ¼le, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">â</span> <span class="NLM_lpage">439</span><span class="refDoi">Â DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12j&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12j&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12j&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12jR"><div class="casContent"><span class="casTitleNuber">12j</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0lgRIlcT0aqnBg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit12j&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12k"><span><span class="NLM_label">(k) </span><span class="NLM_contrib-group">Schmitt, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pippel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Fincke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yiu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SchÃ¼le, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Nonpeptidic propargylamines as inhibitors of lysine specific demethylase 1 (LSD1) with cellular activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7334</span><span class="NLM_x">â</span> <span class="NLM_lpage">7342</span><span class="refDoi">Â DOI: 10.1021/jm400792m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400792m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12k&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7334-7342&author=M.+L.+Schmittauthor=A.+T.+Hauserauthor=L.+Carlinoauthor=M.+Pippelauthor=J.+Schulz-Finckeauthor=E.+Metzgerauthor=D.+Willmannauthor=T.+Yiuauthor=M.+Bartonauthor=R.+Sch%C3%BCleauthor=W.+Sipplauthor=M.+Jung&title=Nonpeptidic+propargylamines+as+inhibitors+of+lysine+specific+demethylase+1+%28LSD1%29+with+cellular+activity&doi=10.1021%2Fjm400792m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12kR"><div class="casContent"><span class="casTitleNuber">12k</span><div class="casTitle"><span class="NLM_cas:atitle">Nonpeptidic Propargylamines as Inhibitors of Lysine Specific Demethylase 1 (LSD1) with Cellular Activity</span></div><div class="casAuthors">Schmitt, Martin L.; Hauser, Alexander-Thomas; Carlino, Luca; Pippel, Martin; Schulz-Fincke, Johannes; Metzger, Eric; Willmann, Dominica; Yiu, Teresa; Barton, Michelle; Schuele, Roland; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7334-7342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine demethylases play an important role in epigenetic regulation and thus in the development of diseases like cancer or neurodegenerative disorders.  As the lysine specific demethylase 1 (LSD1/KDM1) has been strongly connected to androgen and estrogen dependent gene expression, it serves as a promising target for the therapy of hormone dependent cancer.  Here, we report on the discovery of new small mol. inhibitors of LSD1 contg. a propargylamine warhead, starting out from lysine contg. substrate analogs.  On the basis of these substrate mimicking inhibitors, we were able to increase potency by a combination of similarity-based virtual screening and subsequent synthetic optimization resulting in more druglike LSD1 inhibitors that led to histone hypermethylation in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRGRr4rUPfsLVg90H21EOLACvtfcHk0lgRIlcT0aqnBg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO&md5=d4de914dec37f011a7b9488ec539a395</span></div><a href="/servlet/linkout?suffix=cit12k&amp;dbid=16384&amp;doi=10.1021%2Fjm400792m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400792m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DM.%2BL.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DPippel%26aufirst%3DM.%26aulast%3DSchulz-Fincke%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DYiu%26aufirst%3DT.%26aulast%3DBarton%26aufirst%3DM.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DNonpeptidic%2520propargylamines%2520as%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520with%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7334%26epage%3D7342%26doi%3D10.1021%2Fjm400792m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12l"><span><span class="NLM_label">(l) </span><span class="NLM_contrib-group">Prusevich, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalin, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ming, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basso, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cieniewicz, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsiao, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adejola, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avery, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taverna, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tackett, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratan, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, O. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">A selective phenelzine analogue inhibitor of histone demethylase LSD1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1284</span><span class="NLM_x">â</span> <span class="NLM_lpage">1293</span><span class="refDoi">Â DOI: 10.1021/cb500018s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500018s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12l&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1284-1293&author=P.+Prusevichauthor=J.+H.+Kalinauthor=S.+A.+Mingauthor=M.+Bassoauthor=J.+Givensauthor=X.+Liauthor=J.+Huauthor=M.+S.+Taylorauthor=A.+M.+Cieniewiczauthor=P.+Y.+Hsiaoauthor=R.+Huangauthor=H.+Robersonauthor=N.+Adejolaauthor=L.+B.+Averyauthor=R.+A.+Caseroauthor=S.+D.+Tavernaauthor=J.+Qianauthor=A.+J.+Tackettauthor=R.+R.+Ratanauthor=O.+G.+McDonaldauthor=A.+P.+Feinbergauthor=P.+A.+Cole&title=A+selective+phenelzine+analogue+inhibitor+of+histone+demethylase+LSD1&doi=10.1021%2Fcb500018s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12lR"><div class="casContent"><span class="casTitleNuber">12l</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective Phenelzine Analogue Inhibitor of Histone Demethylase LSD1</span></div><div class="casAuthors">Prusevich, Polina; Kalin, Jay H.; Ming, Shonoi A.; Basso, Manuela; Givens, Jeffrey; Li, Xin; Hu, Jianfei; Taylor, Martin S.; Cieniewicz, Anne M.; Hsiao, Po-Yuan; Huang, Rong; Roberson, Heather; Adejola, Nkosi; Avery, Lindsay B.; Casero, Robert A., Jr.; Taverna, Sean D.; Qian, Jiang; Tackett, Alan J.; Ratan, Rajiv R.; McDonald, Oliver G.; Feinberg, Andrew P.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1284-1293</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is an epigenetic enzyme that oxidatively cleaves Me groups from monomethyl and di-Me Lys 4 of histone H3 (H3K4Me1, H3K4Me2) and can contribute to gene silencing.  This study describes the design and synthesis of analogs of a monoamine oxidase antidepressant, phenelzine, and their LSD1 inhibitory properties.  A novel phenelzine analog (bizine) contg. a phenyl-butyrylamide appendage was shown to be a potent LSD1 inhibitor in vitro and was selective vs. monoamine oxidases A/B and the LSD1 homolog, LSD2.  Bizine was found to be effective at modulating bulk histone methylation in cancer cells, and ChIP-seq expts. revealed a statistically significant overlap in the H3K4 methylation pattern of genes affected by bizine and those altered in LSD1-/- cells.  Treatment of two cancer cell lines, LNCaP and H460, with bizine conferred a redn. in proliferation rate, and bizine showed additive to synergistic effects on cell growth when used in combination with two out of five HDAC inhibitors tested.  Moreover, neurons exposed to oxidative stress were protected by the presence of bizine, suggesting potential applications in neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Z1w5uKT7qbVg90H21EOLACvtfcHk0liGhSkZPUY2dA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D&md5=36c9f3fb9b28579fa3cad41052cf335a</span></div><a href="/servlet/linkout?suffix=cit12l&amp;dbid=16384&amp;doi=10.1021%2Fcb500018s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500018s%26sid%3Dliteratum%253Aachs%26aulast%3DPrusevich%26aufirst%3DP.%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DMing%26aufirst%3DS.%2BA.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DGivens%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DCieniewicz%26aufirst%3DA.%2BM.%26aulast%3DHsiao%26aufirst%3DP.%2BY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DRoberson%26aufirst%3DH.%26aulast%3DAdejola%26aufirst%3DN.%26aulast%3DAvery%26aufirst%3DL.%2BB.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DRatan%26aufirst%3DR.%2BR.%26aulast%3DMcDonald%26aufirst%3DO.%2BG.%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520selective%2520phenelzine%2520analogue%2520inhibitor%2520of%2520histone%2520demethylase%2520LSD1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1284%26epage%3D1293%26doi%3D10.1021%2Fcb500018s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12m"><span><span class="NLM_label">(m) </span><span class="NLM_contrib-group">Kakizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ota, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Histone H3 peptide based LSD1-selective inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1925</span><span class="NLM_x">â</span> <span class="NLM_lpage">1928</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.03.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12m&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.bmcl.2015.03.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12m&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=25827526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12m&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFWgs7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1925-1928&author=T.+Kakizawaauthor=Y.+Otaauthor=Y.+Itohauthor=H.+Tsumotoauthor=T.+Suzuki&title=Histone+H3+peptide+based+LSD1-selective+inhibitors&doi=10.1016%2Fj.bmcl.2015.03.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12mR"><div class="casContent"><span class="casTitleNuber">12m</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 peptide based LSD1-selective inhibitors</span></div><div class="casAuthors">Kakizawa, Taeko; Ota, Yosuke; Itoh, Yukihiro; Tsumoto, Hiroki; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1925-1928</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of candidates for the histone H3 peptide based LSD1-selective inhibitor were designed and synthesized.  Among peptides, ART-X-QTARKSTGGKAPRKQLA and ARTKQTAR-X-STGGKAPRKQLA [X = substituted with phenylcyclopropylamine, 5-dihydro-1H-pyrrole and 1,2,3,6-tetrahydropyridine lysine (I), (II) and (III)].  Peptide ART-I-QTARKSTGGKAPRKQLA, which has a phenylcyclopropylamine (PCPA) moiety at Lys-4 of the 21 amino acid residues of histone H3, was the most potent LSD1-selective inhibitor.  Truncation studies of peptide ART-I-QTARKSTGGKAPRKQLA revealed the significance of the peptide sequence length.  These findings will be useful for the further development of histone H3 peptide based LSD1-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxO8fMkITrQ7Vg90H21EOLACvtfcHk0liGhSkZPUY2dA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFWgs7Y%253D&md5=1ea3add93c3eaa8a367433a6ddcb4dd9</span></div><a href="/servlet/linkout?suffix=cit12m&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.03.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.03.030%26sid%3Dliteratum%253Aachs%26aulast%3DKakizawa%26aufirst%3DT.%26aulast%3DOta%26aufirst%3DY.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DHistone%2520H3%2520peptide%2520based%2520LSD1-selective%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D1925%26epage%3D1928%26doi%3D10.1016%2Fj.bmcl.2015.03.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12n"><span><span class="NLM_label">(n) </span><span class="NLM_contrib-group">Khan, M. N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ota, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Design, synthesis, and biological activity of N-alkylated analogue of NCL1, a selective inhibitor of lysine-specific demethylase 1</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">407</span><span class="NLM_x">â</span> <span class="NLM_lpage">412</span><span class="refDoi">Â DOI: 10.1039/C4MD00330F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12n&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1039%2FC4MD00330F" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=407-412&author=M.+N.+A.+Khanauthor=H.+Tsumotoauthor=Y.+Itohauthor=Y.+Otaauthor=M.+Suzukiauthor=D.+Ogasawaraauthor=H.+Nakagawaauthor=T.+Mizukamiauthor=N.+Miyataauthor=T.+Suzuki&title=Design%2C+synthesis%2C+and+biological+activity+of+N-alkylated+analogue+of+NCL1%2C+a+selective+inhibitor+of+lysine-specific+demethylase+1&doi=10.1039%2FC4MD00330F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12n&amp;dbid=16384&amp;doi=10.1039%2FC4MD00330F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00330F%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BN.%2BA.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOta%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520activity%2520of%2520N-alkylated%2520analogue%2520of%2520NCL1%252C%2520a%2520selective%2520inhibitor%2520of%2520lysine-specific%2520demethylase%25201%26jtitle%3DMedChemComm%26date%3D2015%26volume%3D6%26spage%3D407%26epage%3D412%26doi%3D10.1039%2FC4MD00330F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12o"><span><span class="NLM_label">(o) </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ota, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Synthesis, LSD1 inhibitory activity, and LSD1 binding model of optically pure lysine-PCPA conjugates</span> <span class="citation_source-journal">Comput. Struct. Biotechnol. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">â</span> <span class="NLM_lpage">9</span><span class="refDoi">Â DOI: 10.5936/csbj.201402002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12o&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.5936%2Fcsbj.201402002" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1-9&author=Y.+Itohauthor=D.+Ogasawaraauthor=Y.+Otaauthor=T.+Mizukamiauthor=T.+Suzuki&title=Synthesis%2C+LSD1+inhibitory+activity%2C+and+LSD1+binding+model+of+optically+pure+lysine-PCPA+conjugates&doi=10.5936%2Fcsbj.201402002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12o&amp;dbid=16384&amp;doi=10.5936%2Fcsbj.201402002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5936%252Fcsbj.201402002%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DOta%26aufirst%3DY.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DSynthesis%252C%2520LSD1%2520inhibitory%2520activity%252C%2520and%2520LSD1%2520binding%2520model%2520of%2520optically%2520pure%2520lysine-PCPA%2520conjugates%26jtitle%3DComput.%2520Struct.%2520Biotechnol.%2520J.%26date%3D2014%26volume%3D9%26spage%3D1%26epage%3D9%26doi%3D10.5936%2Fcsbj.201402002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12p"><span><span class="NLM_label">(p) </span><span class="NLM_contrib-group">Mohammad, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smitheman, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamat, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federowicz, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneck, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butticello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnette, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degenhardt, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasparec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNulty, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDevitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouthamel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Concha, N. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpenter, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span> </span><span class="NLM_article-title">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">â</span> <span class="NLM_lpage">69</span><span class="refDoi">Â DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12p&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12p&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12p&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+Hypomethylation+Signature+Predicts+Antitumor+Activity+of+LSD1+Inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12pR"><div class="casContent"><span class="casTitleNuber">12p</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lhLBbF5OtPpwg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit12p&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520Hypomethylation%2520Signature%2520Predicts%2520Antitumor%2520Activity%2520of%2520LSD1%2520Inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meek, T. D.</span><span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">â</span> <span class="NLM_lpage">739</span><span class="refDoi">Â DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0lhDHQSIJ2VAAg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Ito, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span> </span><span class="NLM_article-title">Metal-free reductive cyclization and isomerization of sulfanyl-1,6-diynes using sodium borohydride</span> <span class="citation_source-journal">Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1758</span><span class="NLM_x">â</span> <span class="NLM_lpage">1760</span><span class="refDoi">Â DOI: 10.1246/cl.140688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1246%2Fcl.140688" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=1758-1760&author=Y.+Itoauthor=M.+Yoshimatsu&title=Metal-free+reductive+cyclization+and+isomerization+of+sulfanyl-1%2C6-diynes+using+sodium+borohydride&doi=10.1246%2Fcl.140688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1246%2Fcl.140688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1246%252Fcl.140688%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DY.%26aulast%3DYoshimatsu%26aufirst%3DM.%26atitle%3DMetal-free%2520reductive%2520cyclization%2520and%2520isomerization%2520of%2520sulfanyl-1%252C6-diynes%2520using%2520sodium%2520borohydride%26jtitle%3DChem.%2520Lett.%26date%3D2014%26volume%3D43%26spage%3D1758%26epage%3D1760%26doi%3D10.1246%2Fcl.140688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Lee, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fekner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Au, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. K.</span><span> </span><span class="NLM_article-title">A click-and-release pyrrolysine analogue</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">805</span><span class="NLM_x">â</span> <span class="NLM_lpage">808</span><span class="refDoi">Â DOI: 10.1002/cbic.201300124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1002%2Fcbic.201300124" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=805-808&author=M.+M.+Leeauthor=T.+Feknerauthor=T.+H.+Tangauthor=L.+Wangauthor=A.+H.+Chanauthor=P.+H.+Hsuauthor=S.+W.+Auauthor=M.+K.+Chan&title=A+click-and-release+pyrrolysine+analogue&doi=10.1002%2Fcbic.201300124"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201300124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201300124%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BM.%26aulast%3DFekner%26aufirst%3DT.%26aulast%3DTang%26aufirst%3DT.%2BH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChan%26aufirst%3DA.%2BH.%26aulast%3DHsu%26aufirst%3DP.%2BH.%26aulast%3DAu%26aufirst%3DS.%2BW.%26aulast%3DChan%26aufirst%3DM.%2BK.%26atitle%3DA%2520click-and-release%2520pyrrolysine%2520analogue%26jtitle%3DChemBioChem%26date%3D2013%26volume%3D14%26spage%3D805%26epage%3D808%26doi%3D10.1002%2Fcbic.201300124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Shetty, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeong, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. C.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of macrocyclic amino acid derivatives for tumor imaging by gallium-68 positron emission tomography</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">7338</span><span class="NLM_x">â</span> <span class="NLM_lpage">7347</span><span class="refDoi">Â DOI: 10.1016/j.bmc.2010.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.bmc.2010.09.022" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=7338-7347&author=D.+Shettyauthor=J.+M.+Jeongauthor=C.+H.+Juauthor=Y.+Kimauthor=J.+Y.+Leeauthor=Y.+S.+Leeauthor=D.+S.+Leeauthor=J.+K.+Chungauthor=M.+C.+Lee&title=Synthesis+and+evaluation+of+macrocyclic+amino+acid+derivatives+for+tumor+imaging+by+gallium-68+positron+emission+tomography&doi=10.1016%2Fj.bmc.2010.09.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DShetty%26aufirst%3DD.%26aulast%3DJeong%26aufirst%3DJ.%2BM.%26aulast%3DJu%26aufirst%3DC.%2BH.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DY.%2BS.%26aulast%3DLee%26aufirst%3DD.%2BS.%26aulast%3DChung%26aufirst%3DJ.%2BK.%26aulast%3DLee%26aufirst%3DM.%2BC.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520macrocyclic%2520amino%2520acid%2520derivatives%2520for%2520tumor%2520imaging%2520by%2520gallium-68%2520positron%2520emission%2520tomography%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D7338%26epage%3D7347%26doi%3D10.1016%2Fj.bmc.2010.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Molecular mimicry and ligand recognition in binding and catalysis by the histone demethylase LSD1-CoREST complex</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">212</span><span class="NLM_x">â</span> <span class="NLM_lpage">220</span><span class="refDoi">Â DOI: 10.1016/j.str.2011.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.str.2011.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=21300290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyitL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=212-220&author=R.+Baronauthor=C.+Bindaauthor=M.+Tortoriciauthor=J.+A.+McCammonauthor=A.+Mattevi&title=Molecular+mimicry+and+ligand+recognition+in+binding+and+catalysis+by+the+histone+demethylase+LSD1-CoREST+complex&doi=10.1016%2Fj.str.2011.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mimicry and Ligand Recognition in Binding and Catalysis by the Histone Demethylase LSD1-CoREST Complex</span></div><div class="casAuthors">Baron, Riccardo; Binda, Claudia; Tortorici, Marcello; McCammon, J. Andrew; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">212-220</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone demethylases LSD1 and LSD2 (KDM1A/B) catalyze the oxidative demethylation of Lys4 of histone H3.  We used mol. dynamics simulations to probe the diffusion of the oxygen substrate.  Oxygen can reach the catalytic center independently from the presence of a bound histone peptide, implying that LSD1 can complete subsequent demethylation cycles without detaching from the nucleosomal particle.  The simulations highlight the role of a strictly conserved active-site Lys residue providing general insight into the enzymic mechanism of oxygen-reacting flavoenzymes.  The crystal structure of LSD1-CoREST bound to a peptide of the transcription factor SNAIL1 unravels a fascinating example of mol. mimicry.  The SNAIL1 N-terminal residues bind to the enzyme active-site cleft, effectively mimicking the H3 tail.  This finding predicts that other members of the SNAIL/Scratch transcription factor family might assoc. to LSD1/2.  The combination of selective histone-modifying activity with the distinct recognition mechanisms underlies the biol. complexity of LSD1/2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGdKxxD93hm7Vg90H21EOLACvtfcHk0ljqtheDv7HfMg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyitL4%253D&md5=61b4d3cf137e45643c3066f3369f18d5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DMolecular%2520mimicry%2520and%2520ligand%2520recognition%2520in%2520binding%2520and%2520catalysis%2520by%2520the%2520histone%2520demethylase%2520LSD1-CoREST%2520complex%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D212%26epage%3D220%26doi%3D10.1016%2Fj.str.2011.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmorstein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">A mechanism-based inactivator for histone demethylase LSD1</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">4536</span><span class="NLM_x">â</span> <span class="NLM_lpage">4537</span><span class="refDoi">Â DOI: 10.1021/ja0602748</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0602748" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=4536-4537&author=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=X.+Liuauthor=G.+Daauthor=R.+Marmorsteinauthor=P.+A.+Cole&title=A+mechanism-based+inactivator+for+histone+demethylase+LSD1&doi=10.1021%2Fja0602748"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1021%2Fja0602748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0602748%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDa%26aufirst%3DG.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520mechanism-based%2520inactivator%2520for%2520histone%2520demethylase%2520LSD1%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D4536%26epage%3D4537%26doi%3D10.1021%2Fja0602748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gocke, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brautigam, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomchick, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span> </span><span class="NLM_article-title">Structural basis of histone demethylation by LSD1 revealed by suicide inactivation</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">535</span><span class="NLM_x">â</span> <span class="NLM_lpage">539</span><span class="refDoi">Â DOI: 10.1038/nsmb1255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1038%2Fnsmb1255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=17529991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVyjtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=535-539&author=M.+Yangauthor=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=C.+B.+Gockeauthor=C.+A.+Brautigamauthor=D.+R.+Tomchickauthor=M.+Machiusauthor=P.+A.+Coleauthor=H.+Yu&title=Structural+basis+of+histone+demethylation+by+LSD1+revealed+by+suicide+inactivation&doi=10.1038%2Fnsmb1255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17bR"><div class="casContent"><span class="casTitleNuber">17b</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of histone demethylation by LSD1 revealed by suicide inactivation</span></div><div class="casAuthors">Yang, Maojun; Culhane, Jeffrey C.; Szewczuk, Lawrence M.; Gocke, Christian B.; Brautigam, Chad A.; Tomchick, Diana R.; Machius, Mischa; Cole, Philip A.; Yu, Hongtao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">535-539</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation regulates diverse chromatin-templated processes, including transcription.  The recent discovery of the first histone lysine-specific demethylase (LSD1) has changed the long-held view that histone methylation is a permanent epigenetic mark.  LSD1 is a FAD-dependent amine oxidase that demethylates histone H3 Lys4 (H3-K4).  However, the mechanism by which LSD1 achieves its substrate specificity is unclear.  We report the crystal structure of human LSD1 with a propargylamine-derivatized H3 peptide covalently tethered to FAD.  H3 adopts three consecutive Î³-turns, enabling an ideal side chain spacing that places its N terminus into an anionic pocket and positions methyl-Lys4 near FAD for catalysis.  The LSD1 active site cannot productively accommodate more than three residues on the N-terminal side of the methyllysine, explaining its H3-K4 specificity.  The unusual backbone conformation of LSD1-bound H3 suggests a strategy for designing potent LSD1 inhibitors with therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocskIRV1aPVbVg90H21EOLACvtfcHk0ljqtheDv7HfMg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVyjtbs%253D&md5=51ba4fc92701ed0952d3f37e11ecda00</span></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1255%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DGocke%26aufirst%3DC.%2BB.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DTomchick%26aufirst%3DD.%2BR.%26aulast%3DMachius%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DH.%26atitle%3DStructural%2520basis%2520of%2520histone%2520demethylation%2520by%2520LSD1%2520revealed%2520by%2520suicide%2520inactivation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D535%26epage%3D539%26doi%3D10.1038%2Fnsmb1255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cloos, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sennels, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappsilber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salcini, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">RBP2 belongs to a family of demethylases, specific for tri-and dimethylated lysine 4 on histone 3</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">1063</span><span class="NLM_x">â</span> <span class="NLM_lpage">1076</span><span class="refDoi">Â DOI: 10.1016/j.cell.2007.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1016%2Fj.cell.2007.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=17320161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslGms78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=1063-1076&author=J.+Christensenauthor=K.+Aggerauthor=P.+A.+Cloosauthor=D.+Pasiniauthor=S.+Roseauthor=L.+Sennelsauthor=J.+Rappsilberauthor=K.+H.+Hansenauthor=A.+E.+Salciniauthor=K.+Helin&title=RBP2+belongs+to+a+family+of+demethylases%2C+specific+for+tri-and+dimethylated+lysine+4+on+histone+3&doi=10.1016%2Fj.cell.2007.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">RBP2 belongs to a family of demethylases, specific for triand dimethylated lysine 4 on histone 3</span></div><div class="casAuthors">Christensen, Jesper; Agger, Karl; Cloos, Paul A. C.; Pasini, Diego; Rose, Simon; Sennels, Lau; Rappsilber, Juri; Hansen, Klaus H.; Salcini, Anna Elisabetta; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1063-1076</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of histones has been regarded as a stable modification defining the epigenetic program of the cell, which regulates chromatin structure and transcription.  However, the recent discovery of histone demethylases has challenged the stable nature of histone methylation.  Here we demonstrate that the JARID1 proteins RBP2, PLU1, and SMCX are histone demethylases specific for di- and trimethylated histone 3 lysine 4 (H3K4).  Consistent with a role for the JARID1 Drosophila homolog Lid in regulating expression of homeotic genes during development, we show that RBP2 is displaced from Hox genes during embryonic stem (ES) cell differentiation correlating with an increase of their H3K4me3 levels and expression.  Furthermore, we show that mutation or RNAi depletion of the C. elegans JARID1 homolog rbr-2 leads to increased levels of H3K4me3 during larval development and defects in vulva formation.  Taken together, these results suggest that H3K4me3/me2 demethylation regulated by the JARID1 family plays an important role during development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoiLQh_7D2g7Vg90H21EOLACvtfcHk0ljITo2I3-0ozg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslGms78%253D&md5=c1f04b597253ddc6d89b62bf0b99b13c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DCloos%26aufirst%3DP.%2BA.%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DRose%26aufirst%3DS.%26aulast%3DSennels%26aufirst%3DL.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DSalcini%26aufirst%3DA.%2BE.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DRBP2%2520belongs%2520to%2520a%2520family%2520of%2520demethylases%252C%2520specific%2520for%2520tri-and%2520dimethylated%2520lysine%25204%2520on%2520histone%25203%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D1063%26epage%3D1076%26doi%3D10.1016%2Fj.cell.2007.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Futaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yagami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiura, Y.</span><span> </span><span class="NLM_article-title">Arginine-rich peptides. An abundant source of membrane-permeable peptides having potential as carriers for intracellular protein delivery</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">5836</span><span class="NLM_x">â</span> <span class="NLM_lpage">5840</span><span class="refDoi">Â DOI: 10.1074/jbc.M007540200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1074%2Fjbc.M007540200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=11084031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhs1KmtL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=5836-5840&author=S.+Futakiauthor=T.+Suzukiauthor=W.+Ohashiauthor=T.+Yagamiauthor=S.+Tanakaauthor=K.+Uedaauthor=Y.+Sugiura&title=Arginine-rich+peptides.+An+abundant+source+of+membrane-permeable+peptides+having+potential+as+carriers+for+intracellular+protein+delivery&doi=10.1074%2Fjbc.M007540200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine-rich peptides: an abundant source of membrane-permeable peptides having potential as carriers for intracellular protein delivery</span></div><div class="casAuthors">Futaki, Shiroh; Suzuki, Tomoki; Ohashi, Wakana; Yagami, Takeshi; Tanaka, Seigo; Ueda, Kunihiro; Sugiura, Yukio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">5836-5840</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A basic peptide derived from human immunodeficiency virus (HIV)-1 Tat protein (positions 48-60) has been reported to have the ability to translocate through the cell membranes and accumulate in the nucleus, the characteristics of which are utilized for the delivery of exogenous proteins into cells.  Based on the fluorescence microscopic observations of mouse macrophage RAW264.7 cells, we found that various arginine-rich peptides have a translocation activity very similar to Tat-(48-60).  These included such peptides as the D-amino acid- and arginine-substituted Tat-(48-60), the RNA-binding peptides derived from virus proteins, such as HIV-1 Rev, and flock house virus coat proteins, and the DNA binding segments of leucine zipper proteins, such as cancer-related proteins c-Fos and c-Jan, and the yeast transcription factor GCN4.  These segments have no specific primary and secondary structures in common except that they have several arginine residues in the sequences.  Moreover, these peptides were internalized even at 4Â°.  These results strongly suggested the possible existence of a common internalization mechanism ubiquitous to arginine-rich peptides, which is not explained by a typical endocytosis.  Using (Arg)n (n = 4-16) peptides, we also demonstrated that there would be an optimal no. of arginine residues (n â¼ 8) for the efficient translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorVdj5ycrHWrVg90H21EOLACvtfcHk0ljITo2I3-0ozg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhs1KmtL0%253D&md5=9d0b4040f7eede7e405734a3cd2e33d5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M007540200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M007540200%26sid%3Dliteratum%253Aachs%26aulast%3DFutaki%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DOhashi%26aufirst%3DW.%26aulast%3DYagami%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DSugiura%26aufirst%3DY.%26atitle%3DArginine-rich%2520peptides.%2520An%2520abundant%2520source%2520of%2520membrane-permeable%2520peptides%2520having%2520potential%2520as%2520carriers%2520for%2520intracellular%2520protein%2520delivery%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D5836%26epage%3D5840%26doi%3D10.1074%2Fjbc.M007540200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Hashimoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shioiri, T.</span><span> </span><span class="NLM_article-title">New methods and reagents in organic synthesis. 14. A simple efficient preparation of methyl esters with trimethylsilyldiazomethane (TMSCHN<sub>2</sub>) and itsaApplication to gas chromatographic analysis of fatty acids</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1475</span><span class="NLM_x">â</span> <span class="NLM_lpage">1478</span><span class="refDoi">Â DOI: 10.1248/cpb.29.1475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1248%2Fcpb.29.1475" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1981&pages=1475-1478&author=N.+Hashimotoauthor=T.+Aoyamaauthor=T.+Shioiri&title=New+methods+and+reagents+in+organic+synthesis.+14.+A+simple+efficient+preparation+of+methyl+esters+with+trimethylsilyldiazomethane+%28TMSCHN2%29+and+itsaApplication+to+gas+chromatographic+analysis+of+fatty+acids&doi=10.1248%2Fcpb.29.1475"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1248%2Fcpb.29.1475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.29.1475%26sid%3Dliteratum%253Aachs%26aulast%3DHashimoto%26aufirst%3DN.%26aulast%3DAoyama%26aufirst%3DT.%26aulast%3DShioiri%26aufirst%3DT.%26atitle%3DNew%2520methods%2520and%2520reagents%2520in%2520organic%2520synthesis.%252014.%2520A%2520simple%2520efficient%2520preparation%2520of%2520methyl%2520esters%2520with%2520trimethylsilyldiazomethane%2520%2528TMSCHN2%2529%2520and%2520itsaApplication%2520to%2520gas%2520chromatographic%2520analysis%2520of%2520fatty%2520acids%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1981%26volume%3D29%26spage%3D1475%26epage%3D1478%26doi%3D10.1248%2Fcpb.29.1475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ninomiya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tujii, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsumori, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuchida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hosoi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murakami-Yamaguchi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span> </span><span class="NLM_article-title">Regulation of tissue factor pathway inhibitor-2 (TFPI-2) expression by lysine-specific demethylase 1 and 2 (LSD1 and LSD2)</span> <span class="citation_source-journal">Biosci., Biotechnol., Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">1010</span><span class="NLM_x">â</span> <span class="NLM_lpage">1017</span><span class="refDoi">Â DOI: 10.1080/09168451.2014.910104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=10.1080%2F09168451.2014.910104" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2014&pages=1010-1017&author=K.+Minoauthor=S.+Nishimuraauthor=S.+Ninomiyaauthor=H.+Tujiiauthor=Y.+Matsumoriauthor=M.+Tsuchidaauthor=M.+Hosoiauthor=K.+Kosekiauthor=S.+Wadaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=Y.+Murakami-Yamaguchiauthor=H.+Naritaauthor=T.+Suzukiauthor=N.+Miyataauthor=T.+Mizukami&title=Regulation+of+tissue+factor+pathway+inhibitor-2+%28TFPI-2%29+expression+by+lysine-specific+demethylase+1+and+2+%28LSD1+and+LSD2%29&doi=10.1080%2F09168451.2014.910104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1080%2F09168451.2014.910104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F09168451.2014.910104%26sid%3Dliteratum%253Aachs%26aulast%3DMino%26aufirst%3DK.%26aulast%3DNishimura%26aufirst%3DS.%26aulast%3DNinomiya%26aufirst%3DS.%26aulast%3DTujii%26aufirst%3DH.%26aulast%3DMatsumori%26aufirst%3DY.%26aulast%3DTsuchida%26aufirst%3DM.%26aulast%3DHosoi%26aufirst%3DM.%26aulast%3DKoseki%26aufirst%3DK.%26aulast%3DWada%26aufirst%3DS.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMurakami-Yamaguchi%26aufirst%3DY.%26aulast%3DNarita%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DMizukami%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520tissue%2520factor%2520pathway%2520inhibitor-2%2520%2528TFPI-2%2529%2520expression%2520by%2520lysine-specific%2520demethylase%25201%2520and%25202%2520%2528LSD1%2520and%2520LSD2%2529%26jtitle%3DBiosci.%252C%2520Biotechnol.%252C%2520Biochem.%26date%3D2014%26volume%3D78%26spage%3D1010%26epage%3D1017%26doi%3D10.1080%2F09168451.2014.910104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozasa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohkubo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Identification of the KDM2/7 histone lysine demethylase subfamily inhibitor and its antiproliferative activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7222</span><span class="NLM_x">â</span> <span class="NLM_lpage">7231</span><span class="refDoi">Â DOI: 10.1021/jm400624b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400624b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7222-7231&author=T.+Suzukiauthor=H.+Ozasaauthor=Y.+Itohauthor=P.+Zhanauthor=H.+Sawadaauthor=K.+Minoauthor=L.+Walportauthor=R.+Ohkuboauthor=A.+Kawamuraauthor=M.+Yonezawaauthor=Y.+Tsukadaauthor=A.+Tumberauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=C.+J.+Schofieldauthor=N.+Miyata&title=Identification+of+the+KDM2%2F7+histone+lysine+demethylase+subfamily+inhibitor+and+its+antiproliferative+activity&doi=10.1021%2Fjm400624b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=10.1021%2Fjm400624b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400624b%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DOzasa%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DSawada%26aufirst%3DH.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DWalport%26aufirst%3DL.%26aulast%3DOhkubo%26aufirst%3DR.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DTsukada%26aufirst%3DY.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520the%2520KDM2%252F7%2520histone%2520lysine%2520demethylase%2520subfamily%2520inhibitor%2520and%2520its%2520antiproliferative%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7222%26epage%3D7231%26doi%3D10.1021%2Fjm400624b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit22b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tojo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Identification of jumonji AT-rich interactive domain 1A inhibitors and their effect on cancer cells</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">665</span><span class="NLM_x">â</span> <span class="NLM_lpage">670</span><span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFeisbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=665-670&author=Y.+Itohauthor=H.+Sawadaauthor=M.+Suzukiauthor=T.+Tojoauthor=R.+Sasakiauthor=M.+Hasegawaauthor=T.+Mizukamiauthor=T.+Suzuki&title=Identification+of+jumonji+AT-rich+interactive+domain+1A+inhibitors+and+their+effect+on+cancer+cells&doi=10.1021%2Facsmedchemlett.5b00083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22bR"><div class="casContent"><span class="casTitleNuber">22b</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Jumonji AT-Rich Interactive Domain 1A Inhibitors and Their Effect on Cancer Cells</span></div><div class="casAuthors">Itoh, Yukihiro; Sawada, Hideyuki; Suzuki, Miki; Tojo, Toshifumi; Sasaki, Ryuzo; Hasegawa, Makoto; Mizukami, Tamio; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">665-670</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Jumonji AT-rich interactive domain 1A (JARID1A), one of the jumonji C domain-contg. histone demethylase (JHDM) family members, plays key roles in cancer cell proliferation and development of drug tolerance.  Therefore, selective JARID1A inhibitors are potential anticancer agents.  In this study, the authors searched for cell-active JARID1A inhibitors by screening hydroxamate compds. in the inhouse library and the structural optimization based on docking study of the hit-compd. to a homol. model of JARID1A.  As a result, the authors identified compd. I, which selectively inhibits JARID1A over three other JHDM family members.  Compd. II, a prodrug form of compd. I, induced a selective increase in the level of trimethylation of histone H3 lysine 4, a substrate of JARID1A.  Furthermore, compd. II synergistically enhanced A549 human lung cancer cell growth inhibition induced by vorinostat, a histone deacetylase inhibitor.  These findings support the idea that JARID1A inhibitors have potential as anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAGPzR-uSQybVg90H21EOLACvtfcHk0lgZq8lV3jQBNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFeisbk%253D&md5=056d2209c293d95153c09d7d8a1e0248</span></div><a href="/servlet/linkout?suffix=cit22b&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00083%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSawada%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DTojo%26aufirst%3DT.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DIdentification%2520of%2520jumonji%2520AT-rich%2520interactive%2520domain%25201A%2520inhibitors%2520and%2520their%2520effect%2520on%2520cancer%2520cells%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D665%26epage%3D670%26doi%3D10.1021%2Facsmedchemlett.5b00083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y48" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Y48','PDB','2Y48'); return false;">PDB: 2Y48</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i52"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01323">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_23568"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01323">10.1021/acs.jmedchem.5b01323</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">View of the designed peptides <b>1</b>â<b>5</b> in the active site of LSD1; mass spectrometric detection and quantitative analysis of generation of 3-FAD adduct; mass spectrometric detection of LSD1-mediated demethylation of peptide <b>4</b> and of the <b>37</b>-, <b>38</b>-, or <b>39</b>-FAD adduct; synthesis and chiral HPLC data of the methyl ester compounds derivatized from unnatural amino acid <b>29</b>; HPLC data; and MS spectra of synthetic peptides (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01323/suppl_file/jm5b01323_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01323/suppl_file/jm5b01323_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01323/suppl_file/jm5b01323_si_001.pdf">jm5b01323_si_001.pdf (1.27 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01323/suppl_file/jm5b01323_si_002.csv">jm5b01323_si_002.csv (4.01 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01323&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01323%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-4%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01323" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799394ea84d3cdc","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
