
<html lang="en"     class="pb-page"  data-request-id="dd9e4b66-3fd5-48f9-b8dd-3d65a3b97bcd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-23;article:article:10.1021/acs.jmedchem.9b00966"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Optimizing Targeted Inhibitors of P-Glycoprotein Using Computational and Structure-Guided Approaches" /></meta><meta name="dc.Creator" content="John G.  Wise" /></meta><meta name="dc.Creator" content="Amila K.  Nanayakkara" /></meta><meta name="dc.Creator" content="Maha  Aljowni" /></meta><meta name="dc.Creator" content="Gang  Chen" /></meta><meta name="dc.Creator" content="Maisa C.  De Oliveira" /></meta><meta name="dc.Creator" content="Lauren  Ammerman" /></meta><meta name="dc.Creator" content="Ketetha  Olengue" /></meta><meta name="dc.Creator" content="Alexander R.  Lippert" /></meta><meta name="dc.Creator" content="Pia D.  Vogel" /></meta><meta name="dc.Description" content="Overexpression of ABC transporters like P-glycoprotein (P-gp) has been correlated with resistances in cancer chemotherapy. Intensive efforts to identify P-gp inhibitors for use in combination thera..." /></meta><meta name="Description" content="Overexpression of ABC transporters like P-glycoprotein (P-gp) has been correlated with resistances in cancer chemotherapy. Intensive efforts to identify P-gp inhibitors for use in combination thera..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 8, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00966" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00966" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00966" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00966" /></link>
        
    
    

<title>Optimizing Targeted Inhibitors of P-Glycoprotein Using Computational and Structure-Guided Approaches | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00966" /></meta><meta property="og:title" content="Optimizing Targeted Inhibitors of P-Glycoprotein Using Computational and Structure-Guided Approaches" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0001.jpeg" /></meta><meta property="og:description" content="Overexpression of ABC transporters like P-glycoprotein (P-gp) has been correlated with resistances in cancer chemotherapy. Intensive efforts to identify P-gp inhibitors for use in combination therapy have not led to clinically approved inhibitors to date. Here, we describe computational approaches combined with structure-based design to improve the characteristics of a P-gp inhibitor previously identified by us. This hit compound represents a novel class of P-gp inhibitors that specifically targets and inhibits P-gp ATP hydrolysis while not being transported by the pump. We describe here a new program for virtual chemical synthesis and computational assessment, ChemGen, to produce hit compound variants with improved binding characteristics. The chemical syntheses of several variants, efficacy in reversing multidrug resistance in cell culture, and biochemical assessment of the inhibition mechanism are described. The usefulness of the computational predictions of binding characteristics of the inhibitor variants is discussed and compared to more traditional structure-based approaches." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00966"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00966">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00966&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00966&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00966&amp;href=/doi/10.1021/acs.jmedchem.9b00966" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 10645-10663</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00939" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00998" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Optimizing Targeted Inhibitors of P-Glycoprotein Using Computational and Structure-Guided Approaches</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">John G. Wise</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John G. Wise</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences,Â  The Center for Drug Discovery, Design and Delivery,Â  The Center for Scientific Computation, Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#204a5749534560534d550e454455"><span class="__cf_email__" data-cfemail="0e6479677d6b4e7d637b206b6a7b">[emailÂ protected]</span></a>. Phone: 214-768-3426 (J.G.W.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+G.++Wise">John G. Wise</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Amila K. Nanayakkara</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amila K. Nanayakkara</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences,Â  The Center for Drug Discovery, Design and Delivery,Â  The Center for Scientific Computation, Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amila+K.++Nanayakkara">Amila K. Nanayakkara</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Maha Aljowni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maha Aljowni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Center for Drug Discovery, Design and Delivery,Â  Department of Chemistry, Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maha++Aljowni">Maha Aljowni</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Gang Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gang Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences,Â  The Center for Drug Discovery, Design and Delivery,Â  The Center for Scientific Computation, Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gang++Chen">Gang Chen</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Maisa C. De Oliveira</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maisa C. De Oliveira</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences,Â  The Center for Drug Discovery, Design and Delivery,Â  The Center for Scientific Computation, Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maisa+C.++De+Oliveira">Maisa C. De Oliveira</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Lauren Ammerman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lauren Ammerman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences,Â  The Center for Drug Discovery, Design and Delivery,Â  The Center for Scientific Computation, Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lauren++Ammerman">Lauren Ammerman</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ketetha Olengue</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ketetha Olengue</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences,Â  The Center for Drug Discovery, Design and Delivery,Â  The Center for Scientific Computation, Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ketetha++Olengue">Ketetha Olengue</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Alexander R. Lippert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexander R. Lippert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Center for Drug Discovery, Design and Delivery,Â  Department of Chemistry, Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexander+R.++Lippert">Alexander R. Lippert</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4396-0848" title="Orcid link">http://orcid.org/0000-0003-4396-0848</a></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Pia D. Vogel</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pia D. Vogel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences,Â  The Center for Drug Discovery, Design and Delivery,Â  The Center for Scientific Computation, Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#5727213830323b17243a2279323322"><span class="__cf_email__" data-cfemail="3747415850525b77445a4219525342">[emailÂ protected]</span></a>. Phone: 214-768-1790 (P.D.V.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pia+D.++Vogel">Pia D. Vogel</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8135-4649" title="Orcid link">http://orcid.org/0000-0002-8135-4649</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00966&amp;href=/doi/10.1021%2Facs.jmedchem.9b00966" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 10645â10663</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 8, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 June 2019</li><li><span class="item_label"><b>Published</b> online</span>8 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 December 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00966" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00966</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10645%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJohn%2BG.%2BWise%252C%2BAmila%2BK.%2BNanayakkara%252C%2BMaha%2BAljowni%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D23%26contentID%3Dacs.jmedchem.9b00966%26title%3DOptimizing%2BTargeted%2BInhibitors%2Bof%2BP-Glycoprotein%2BUsing%2BComputational%2Band%2BStructure-Guided%2BApproaches%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10663%26publicationDate%3DDecember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00966"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1088</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">6</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00966" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Optimizing Targeted Inhibitors of P-Glycoprotein Using Computational and Structure-Guided Approaches&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;G. Wise&quot;},{&quot;first_name&quot;:&quot;Amila&quot;,&quot;last_name&quot;:&quot;K. Nanayakkara&quot;},{&quot;first_name&quot;:&quot;Maha&quot;,&quot;last_name&quot;:&quot;Aljowni&quot;},{&quot;first_name&quot;:&quot;Gang&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Maisa&quot;,&quot;last_name&quot;:&quot;C. De Oliveira&quot;},{&quot;first_name&quot;:&quot;Lauren&quot;,&quot;last_name&quot;:&quot;Ammerman&quot;},{&quot;first_name&quot;:&quot;Ketetha&quot;,&quot;last_name&quot;:&quot;Olengue&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;R. Lippert&quot;},{&quot;first_name&quot;:&quot;Pia&quot;,&quot;last_name&quot;:&quot;D. Vogel&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10645-10663&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00966&quot;},&quot;abstract&quot;:&quot;Overexpression of ABC transporters like P-glycoprotein (P-gp) has been correlated with resistances in cancer chemotherapy. Intensive efforts to identify P-gp inhibitors for use in combination therapy have not led to clinically approved inhibitors to date. Here, we describe computational approaches combined with structure-based design to improve the characteristics of a P-gp inhibitor previously identified by us. This hit compound represents a novel class of P-gp inhibitors that specifically targets and inhibits P-gp ATP hydrolysis while not being transported by the pump. We describe here a new program for virtual chemical synthesis and computational assessment, ChemGen, to produce hit compound variants with improved binding characteristics. The chemical syntheses of several variants, efficacy in reversing multidrug resistance in cell culture, and biochemical assessment of the inhibition mechanism are described. The usefulness of the computational predictions of binding characteristics of the inhibitor variant&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00966&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00966" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00966&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00966" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00966&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00966" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00966&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00966&amp;href=/doi/10.1021/acs.jmedchem.9b00966" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00966" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00966" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00966%26sid%3Dliteratum%253Aachs%26pmid%3D31702922%26genre%3Darticle%26aulast%3DWise%26date%3D2019%26atitle%3DOptimizing%2BTargeted%2BInhibitors%2Bof%2BP-Glycoprotein%2BUsing%2BComputational%2Band%2BStructure-Guided%2BApproaches%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D23%26spage%3D10645%26epage%3D10663%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292227" title="Drug resistance">Drug resistance</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=290768" title="Antineoplastic agents">Antineoplastic agents</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/jmcmar.2019.62.issue-23/20191212/jmcmar.2019.62.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Overexpression of ABC transporters like P-glycoprotein (P-gp) has been correlated with resistances in cancer chemotherapy. Intensive efforts to identify P-gp inhibitors for use in combination therapy have not led to clinically approved inhibitors to date. Here, we describe computational approaches combined with structure-based design to improve the characteristics of a P-gp inhibitor previously identified by us. This hit compound represents a novel class of P-gp inhibitors that specifically targets and inhibits P-gp ATP hydrolysis while not being transported by the pump. We describe here a new program for virtual chemical synthesis and computational assessment, ChemGen, to produce hit compound variants with improved binding characteristics. The chemical syntheses of several variants, efficacy in reversing multidrug resistance in cell culture, and biochemical assessment of the inhibition mechanism are described. The usefulness of the computational predictions of binding characteristics of the inhibitor variants is discussed and compared to more traditional structure-based approaches.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34813" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34813" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Resistances of cancers to chemically unrelated anticancer drugs are frequently caused by the expression of members of the ABC transporter superfamily, including ABCB1 (P-glycoprotein or P-gp)<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1â3)</a> and/or ABCG2 (the breast cancer resistance protein or BCRP).<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> The phenomenon of multidrug resistance (MDR) remains a major obstacle in the treatment of both adult and pediatric cancers.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5â7)</a> Despite nearly 40 years of intense research, no inhibitor of these proteins has yet been approved for clinical use.<a onclick="showRef(event, 'ref5 ref8 ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref5 ref8 ref9 ref10 ref11 ref12 ref13">(5,8â13)</a> A significant number of failures during the clinical trials may be attributed to the fact that many of the assessed MDR inhibitors were also good transport substrates of the pumps. This latter characteristic likely required elevated systemic inhibitor concentrations for efficacy that may have resulted in off-target toxicities. The fact that a phase III trial using the immunosuppressant, cyclosporine A, led to improved patient outcomes in poor-risk acute myeloid lymphoma patients<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> suggests that these proteins are important targets for drug discovery and development, despite the limited success in finding clinically successful inhibitors of MDR pumps to date. Over the last several years, significant advances in our knowledge of the structure<a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19 ref20">(15â20)</a> and mechanism<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25">(21â25)</a> of P-gp, BCRP, and other related pumps have emerged that may enable the design of potentent inhibitors of pumps and prove to be more successful in clinical applications.</div><div class="NLM_p">Using the evolutionary relationship of different ABC transporters and the structural knowledge of both prokaryotic and eukaryotic ABC transporters, we created dynamic models of human P-gp<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and simulated a putative catalytic cycle<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> that correlated well with published biochemical and biophysical studies as well as with the recently elucidated outward facing structure of the human P-gp.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> We previously used these conformationally dynamic models of human P-gp in ultrahigh throughput in silico screenings to identify and characterize inhibitors of P-glycoprotein.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> One novel characteristic sought in these screens was that the potential inhibitors identified should interact strongly with the nucleotide-binding domains (NBD) of the protein but not be transport substrates of the pump.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> For this reason, compounds were discarded from further evaluation if significant binding to the drug-binding sites was predicted by the in silico docking calculations.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Using this approach, we initially identified and characterized three compounds that reversed the multidrug resistance phenotype of various P-gp overexpressing cancer cell lines in both conventional and microtumor-spheroid cell cultures.<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26â28)</a> All three compounds were observed to increase cancer cell lethality to various chemotherapeutics. This increased lethality was directly correlated with increased cellular retention of the chemotherapeutics when an inhibitor was present.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Importantly, studies also showed that these three P-gp inhibitors were not significantly transported by P-gp,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> supporting our predictions that the inhibitors would not bind effectively to the drug-binding domains of the pump.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p">One of these compounds that reversed MDR phenotypes in cancer cells was computationally predicted to be an allosteric inhibitor of P-glycoprotein (âcompound <b>29</b>â, 2-[(5-cyclopropyl-4<i>H</i>-1,2,4-triazol-3-yl)sulfanyl]-<i>N</i>-[2-phenyl-5-(2,4,5-trimethylphenyl)-pyrazol-3-yl, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The presence of compound <b>29</b> caused increased penetration of a P-gp pump substrate into microtumors of a highly P-gp-overexpressing prostate cancer,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and coadministration of <b>29</b> with a chemotherapeutic resulted in increased cell death via apoptotic mechanisms and size reduction of tumor spheroids.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The binding pose of <b>29</b> docked at the highest affinity interaction site on P-gp as observed in the original study<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> is presented in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B. This site is located in the N-terminal half of the protein near the interface of the two nucleotide-binding domains and significantly outside of the ATP-binding sites. The computational prediction that <b>29</b> acted as a potential allosteric inhibitor of P-gp ATP hydrolysis as assayed using purified, recombinantly expressed P-gp was supported by the observation that <b>29</b> did not affect the binding of an ESR active analog of ATP (SL-ATP, 2â²,3â²-(2,2,5,5,-tetramethyl-3-pyrroline-1-oxyl-3-carboxylic acid ester ATP) to P-gp.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Three other compounds assessed in that same study inhibited SL-ATP binding to P-gp as well as P-gp ATP hydrolysis. In contrast to compound <b>29</b>, these latter three compounds had been computationally predicted to partially overlap with the ATP-binding sites of the transporter<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and were therefore anticipated to be competitive inhibitors of ATP binding. All four compounds inhibited both basal and transport-substrate stimulated ATP hydrolysis activities of purified P-gp,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> suggesting direct interaction with the energy-harvesting steps of substrate pumping by P-gp.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0010.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Putative interactions of compound <b>29</b> at an allosteric inhibitory site on human P-glycoprotein. (A) Structure of compound <b>29</b>. For ease of reference, the âEasternâ and âWesternâ portions of the molecule are marked. (B) Compound <b>29</b> (shown in van der Waalsâ surfaces) docked to a potential allosteric site in the N-terminal nucleotide-binding domain (blue cartoon) of P-gp. This conformation has nucleotide-binding domains in a closed conformation and is similar to that of the Sav1866 X-ray structure (PDB accession: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HYD">2HYD</a>) published by Dawson and Locher.<a onclick="showRef(event, 'ref55 ref57'); return false;" href="javascript:void(0);" class="ref ref55 ref57">(55,57)</a> It was derived from the conformational dynamics investigations of Wise<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and McCormick et al.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The C-terminal nucleotide-binding domain is shown in a charcoal cartoon representation. Bound nucleotides are shown in licorice representations. (C) The P-gp inhibitor compound <b>29</b> (licorice) is shown docked to the putative allosteric site. The surface of P-gp is shown as a wireframe colored by electrostatic potential, and the view point is from within the protein. A large void is visible in the protein near the âWesternâ end of compound <b>29</b> distal to its cyclopropyl group). (D) Same as (C) except that the orientation of the molecule was turned approximately 30Â° around the <i>z</i> axis to show the docked molecule in a slightly different orientation. Small molecules are shown in the figure with nonpolar hydrogens removed for clarity. Images were created with VMD, the SURF surface renderer, and the Tachyon image renderer in VMD; the electrostatic surface of P-gp was calculated with pdb2pqr and the ABPS programs (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Inspection of the computational docking of <b>29</b> at the putative allosteric site on P-gp (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,D) indicated that considerable space was available for additional interactions between the protein and some parts of compound <b>29</b>, especially around what we call the âWesternâ end of the inhibitor (see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,C,D). In contrast, the âEasternâ end of <b>29</b> docked to P-gp was partly exposed to the external surface of P-gp (the phenyl group at the top of <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> B), while the trimethylphenyl group seemed to be nearly optimally fit into a cavity in P-gp (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B,C,D).</div><div class="NLM_p last">A number of computational approaches exist that seek to analyze the potential chemical space of a hit compound by creating virtual libraries of variants of a given hit molecule.<a onclick="showRef(event, 'ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32">(29â32)</a> We present here our initial efforts at creating variants of compound <b>29</b> with optimized binding affinity to P-glycoprotein using a novel computer-aided, structure-based approach that was applied to the âWesternâ end of compound <b>29</b>. In this pilot study, a small number of variants of the nearly infinite chemical space around hit compound <b>29</b> were synthesized and assessed for reversal of the MDR phenotype in a multidrug-resistant prostate cancer cell line that overexpresses P-gp (DU145TXR<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a>). After initial evaluation of the computationally predicted inhibitor variants in P-gp inhibition in cell-based and biochemical assays, structure-based rational design and visual inspection of the putative <b>29</b> binding site on P-gp led us to synthesize and analyze a small number of <b>29</b>-derivatives with different structural and physicochemical characteristics. These latter compounds were not initially computationally evaluated using the subtractive binding routines as described in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> but were chosen mostly for the shape and size of the Western half of the molecule and physicochemical characteristics like polar surface areas and solubility. The work led to the discovery of several variants of the P-gp inhibitor hit compound <b>29</b> with improved efficacy in reversing MDR in P-glycoprotein-overexpressing cancer cells by inhibiting P-gp-catalyzed substrate pumping. All of the computationally evaluated variants were shown to be more efficient in increasing lethality of P-gp-overexpressing cancer cells, while none of them was observed to be transported by the pump. By contrast, only one of the traditionally rationally designed variants was not a transport substrate of the pump. This result strongly suggests that the combination of computational medicinal chemistry and computational pre-assessment of the predicted variants described here may be a promising path for drug optimization projects, especially for hard-to-target proteins where some protein interactions are not desirable.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78372" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78372" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Virtual Synthesis of Novel Variants of the âWesternâ Half of SMU29 Using the ChemGen Computational Suite</h3><div class="NLM_p">Evaluation of the fit of compound <b>29</b> into a putative allosteric binding site on P-gp was visualized from the results of docking studies as in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and led us to hypothesize that, if variants of inhibitor <b>29</b> were made larger and more hydrophobic on one end of the molecule, they would likely fill the relatively large hydrophobic pocket in the protein where the cyclopropyl group of <b>29</b> interacts (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,D), increasing the overall binding affinity of the variant.</div><div class="NLM_p">In order to test this hypothesis, a central retrosynthetic disconnection of a carbonâsulfur bond in compound <b>29</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) was generalized for virtual syntheses (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The ChemGen program (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S1</a>) was used to virtually synthesize a number of variants of compound <b>29</b> that had different substituents at the âWesternâ thiol-derived part of the molecule. Several thousands of sulfur-containing compounds were collected from the ZINC database<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> by search of carbonâsulfur single bonds. Using the ChemGen precursor marking procedures (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Supporting Information</a>), this set of compounds was pruned for those that could be converted to thiols. A total of 647 thiol-containing molecules remained that were then used in the final reaction step of the computational retrosynthesis shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B. The scaffold compound used to react with these thiols was 2-chloro-<i>N</i>-[1-phenyl-3-(2,4,5-trimethylphenyl)-1<i>H</i>-pyrazol-5-yl]acetamide, which includes the âEasternâ part of compound <b>29</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Retrosynthesis of the P-glycoprotein inhibitor, compound <b>29</b>, and the generalized scheme for creating variants of compound <b>29</b>. (A) Retrosynthetic scheme for compound <b>29</b>. (B) Generalized synthesis scheme used in the ChemGen program for the virtual synthesis of variants of compound <b>29</b>: In this reaction scheme, any amine can be used to react with chloroacetyl chloride to generate 2-chloroacetamide derivatives. Reaction of the latter with any thiol is then performed to create the sulfide-linked derivatives of which compound <b>29</b> is one. Computationally, large numbers of different amines and thiols can be used to generate large numbers of derivatives. In addition to these generalized schemes for variation of a given structure, localized substitutions of hydrogens for halides, for example, are quite easily accomplished by ChemGen (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Virtual syntheses of the resulting 647 variants of compound <b>29</b> were performed using ChemGen applied to the single scaffold and thiol precursors as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Final geometrical optimization of the virtually synthesized molecules was performed using the phenix.elbow<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> program. Some of the potential chemical spaces of these 647 ChemGen-produced variants of <b>29</b> (âgroup 1â compounds) are visualized in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S2</a>. This figure shows the 647 virtual <b>29</b>-variants aligned on the heavy atoms of the common 1-phenyl-3-(2,4,5-trimethylphenyl)-1<i>H</i>-pyrazol-5-yl group.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Docking to P-Glycoprotein and Chemical Synthesis of Compound <b>29</b>-Variants</h3><div class="NLM_p">The 647 group 1 molecules created by ChemGen were used in docking studies to the same structural model of P-gp that was employed in our previous work and led to the identification of compound <b>29</b> as a potential inhibitor<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> (see steps 8 through 10 of <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S1</a>, the âin silico docking routineâ). Docking was performed as in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> but using a target box that encompassed the putative allosteric site on P-gp (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B and <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>) instead of the larger target box described in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>. The original unmodified compound <b>29</b> was included in these calculations so that the estimated binding affinities calculated by docking software for each of the variants could be compared to the affinities estimated for parental compound <b>29</b>. Sixty-seven of these group 1 variants of compound <b>29</b> were identified in these calculations that were predicted to interact with the putative allosteric site on P-gp more strongly than parental <b>29</b>. Since it was desirable to identify variants that are not pump substrates for P-gp, these 67 variants of <b>29</b> were then docked in a second step and counterscreened for low predicted interaction affinities with the drug-binding domains of P-gp (as described in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>). Eleven of the promising hit variants had molecular weights less than 600 Da and ratios of the estimated <i>K</i><sub>d</sub> of compound <b>29</b>/estimated <i>K</i><sub>d</sub> of the <b>29</b>-variant that were greater than 10 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), suggesting that these variants might have higher affinity to P-gp than that of the original <b>29</b>. One additional compound (<b>29-551</b>), which contains a bromo substituent and has a molecular weight of 627 Da, was later added for consideration because of its otherwise favorable predicted properties. <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S3</a> shows compound <b>29</b> (left panel) and an overlay of the 12 variants shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> (right panel) in the putative allosteric site on P-gp. It can be seen from the figure that the ChemGen synthesis and subsequent docking routines generated and identified several variant compounds that were predicted to better fill the void in the putative allosteric site on P-gp than the parental compound <b>29</b>. Docking calculations suggested that these variants might show improvements in binding affinities to P-gp. In this pilot study, a small number (five) of these group 1 ChemGen<i>-</i>generated compounds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>29-216</b>, <b>-227</b>, <b>-231</b>, <b>-541</b>, and <b>-551</b>) were selected for chemical synthesis and subsequent testing for potentially improved efficacy in inhibiting P-gp. At this stage of our study, no consideration was given to âdrug-likenessâ of the chosen compounds except for trying to keep the relative molecular weight as low as possible. Instead, the choice for synthesis of these five particular compounds was made on the basis of ease of synthesis and commercial availability of the precursor molecules.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. ChemGen/docking Routine-Identified Variants of P-gp Inhibitor <b>29</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">variant name</th><th class="colsep0 rowsep0" align="center">synthesized variant</th><th class="colsep0 rowsep0" align="center" char=".">estimated Î<i>G</i><sub>binding</sub> (kcalÂ mol<sup>â1</sup>)</th><th class="colsep0 rowsep0" align="center" char=".">estimated <i>K</i><sub>d</sub> (nM)</th><th class="colsep0 rowsep0" align="center">ratio of <i>K</i><sub>d</sub>Â <b>29</b>/<i>K</i><sub>d</sub>Â variant</th><th class="colsep0 rowsep0" align="center">molecular weight (Da)</th><th class="colsep0 rowsep0" align="center" char=".">topological polar surface area (Ã<sup>2</sup>)</th><th class="colsep0 rowsep0" align="center" char=".">consensus logÂ <i>P</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0009_19</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">â12.4</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">599</td><td class="colsep0 rowsep0" align="char" char=".">173.8</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0005_53</td><td class="colsep0 rowsep0" align="left"><b>29-541</b></td><td class="colsep0 rowsep0" align="char" char=".">â12.0</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">522</td><td class="colsep0 rowsep0" align="char" char=".">132.5</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0009_29</td><td class="colsep0 rowsep0" align="left"><b>29-216</b></td><td class="colsep0 rowsep0" align="char" char=".">â11.8</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">558</td><td class="colsep0 rowsep0" align="char" char=".">89.3</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0010_33</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">â11.8</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">576</td><td class="colsep0 rowsep0" align="char" char=".">151.5</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0005_47</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">â11.8</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">591</td><td class="colsep0 rowsep0" align="char" char=".">138.6</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0000_30</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">â11.7</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">541</td><td class="colsep0 rowsep0" align="char" char=".">157</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0002_34</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">â11.7</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">579</td><td class="colsep0 rowsep0" align="char" char=".">124</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0005_27</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">â11.7</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">584</td><td class="colsep0 rowsep0" align="char" char=".">131.4</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0010_49</td><td class="colsep0 rowsep0" align="left"><b>29-551</b></td><td class="colsep0 rowsep0" align="char" char=".">â11.6</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">627</td><td class="colsep0 rowsep0" align="char" char=".">114.2</td><td class="colsep0 rowsep0" align="char" char=".">5.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0009_33</td><td class="colsep0 rowsep0" align="left"><b>29-231</b></td><td class="colsep0 rowsep0" align="char" char=".">â11.5</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">535</td><td class="colsep0 rowsep0" align="char" char=".">101.3</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0004_12</td><td class="colsep0 rowsep0" align="left"><b>29-227</b></td><td class="colsep0 rowsep0" align="char" char=".">â11.5</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">587</td><td class="colsep0 rowsep0" align="char" char=".">92.5</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">opt_0006_13</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">â11.5</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">596</td><td class="colsep0 rowsep0" align="char" char=".">130.1</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ZINC08767731</td><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="char" char=".">â10.1</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">459</td><td class="colsep0 rowsep0" align="char" char=".">113.8</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Estimated <i>K</i><sub>d</sub> values for the ligand interactions with P-gp were calculated from the lowest estimated binding Î<i>G</i> values from the AutoDock calculations. The ratio of <i>K</i><sub>d</sub> values is shown as a relative value for potentially increased affinity of the variants over the parent P-gp inhibitor <b>29</b>. Molecular weights, topological polar surface areas, and consensus log <i>P</i> values were calculated at the SwissADME website (<a href="http://www.swissadme.ch/" class="extLink">http://www.swissadme.ch/</a>) as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>.</p></div></div></div><div class="NLM_p">Syntheses were performed using the scheme shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S4</a>. The virtual retrosynthetic scheme shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B was slightly modified so that the conserved pyrazole âEasternâ half would be formed from a thiol precursor. The Lewis acid-catalyzed reaction of aldehyde <b>1</b> with diazoacetonitrile <b>2</b> provided the Î±-cyanoketone <b>3</b>.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The core pyrazole motif was formed through condensation of <b>3</b> with phenylhydrazine, providing the aminopyrazole <b>4</b> in good yield.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> After nucleophilic acyl substitution with chloroacetyl chloride, the thiol was formed in two steps by substitution with potassium thioacetate followed by thioester hydrolysis.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> With this thiol in hand, S<sub>N</sub>2 reactions with various alkyl chlorides provided a modular approach to diverse derivatives of compound <b>29</b>. While most of the synthesized derivatives of compound <b>29</b> consisted of an amide on the âWesternâ half, derivative <b>29-216</b> (<b>216</b>) required a ketone. The target Î±-chloro ketone was prepared by FriedelâCrafts acylation of fluorene with chloroacteyl chloride.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The resultant chloride was then subjected to a similar sequence as described above to form the thiol after nucleophilic substitution with thioacetate. The aromatic sulfide compounds <b>541</b> and <b>551</b> were prepared by substituting the alkyl chloride <b>5</b> with the respective aromatic thiols. Details of the syntheses and product analyses are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Supporting Information</a>, âSynthetic Proceduresâ section.</div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A shows the structures of group 1 variants <b>29-216</b> (<b>216</b>), <b>29-227</b> (<b>227</b>), <b>29-231</b> (<b>231</b>), <b>29-541</b> (<b>541</b>), and <b>29-551</b> (<b>551</b>) in the highest estimated affinity docking pose (shown as licorice and colored by their atoms) in the putative allosteric site of P-gp.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0008.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Interactions of compound <b>29</b> group 1 variants at a putative allosteric inhibitory site on human P-glycoprotein. (A) The putative allosteric site on P-gp is shown in a surface representation with synthesized variants of compound <b>29</b> shown in licorice and colored by their atoms. Variants (labeled in the panel) are shown as in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> with the surface of P-glycoprotein colored by its calculated electrostatic potential. (B) The structures of the variant compounds <b>216</b>, <b>227</b>, <b>231</b>, <b>541</b>, and <b>551</b> identified using ChemGen are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B shows the chemical structures of the <b>29</b>-variants below the respective docking images. The nearly identical binding of the âEasternâ portions of compound <b>29</b> and the ChemGen<i>/</i>docking routine-generated <b>29</b>-variants is also clearly observable in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S3</a> where the right panel displays the superposition of the docking poses of all 12 of the <b>29</b>-variants shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Group 1 ChemGen/Docking Routine-Produced <b>29</b>-Variants Resensitize a P-gp-Overexpressing Prostate Cancer Cell Line, DU145TXR, to Paclitaxel</h3><div class="NLM_p">The mitochondrial reduction potential of cells is often used as an indicator for cell viability using MTT assays.<a onclick="showRef(event, 'ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref40 ref41 ref42">(40â42)</a> Using these assays, we observed that the five <b>29</b>-derivatives in group 1, compounds <b>216</b>, <b>227</b>, <b>231</b>, <b>541</b>, and <b>551</b>, indeed resensitized the P-gp overexpressing prostate cancer cell line, DU145TXR,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> to the chemotherapeutic paclitaxel (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0007.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Reversal of paclitaxel resistance by ChemGen-identified variants of the P-glycoprotein inhibitor, compound <b>29</b>. The prostate cancer cell line, DU145TXR, that overexpresses P-glycoprotein was treated with either paclitaxel (PTX) alone or paclitaxel and either 3, 5, 7, or 10 Î¼M P-gp inhibitors (upper left, upper right, lower left, and lower right panels, respectively). Metabolic activity assays using MTT were performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Calculated IC<sub>50</sub> values for paclitaxel in the presence of the respective compounds calculated from this data are reported in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Analysis of the data from <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> revealed that, in the presence of 3 Î¼M <b>29</b>-variant <b>216</b>, the observed IC<sub>50</sub> value of paclitaxel was decreased by approximately 2.4-fold when compared to the presence of the parental compound <b>29</b> and approximately 8-fold when compared to paclitaxel alone. At higher concentrations, the efficacy of variant <b>216</b> in increasing paclitaxel toxicity decreased when compared to the parental compound <b>29</b>. At the highest concentration tested (10 Î¼M), <b>216</b> was observed to be somewhat less effective than parental compound <b>29</b> (0.4-fold compared to the 1-fold of paclitaxel + <b>29</b>). Unlike <b>216</b>, variants <b>227, 231</b>, <b>541</b>, and <b>551</b> were more effective than <b>29</b> at all concentrations tested. At 3 Î¼M, the presence of variants <b>227, 231</b>, <b>541</b>, and <b>551</b> resulted in 4- to 11-fold-decreased paclitaxel IC<sub>50</sub> when compared to parental compound <b>29</b> and up to 38-fold overall sensitization to paclitaxel when compared to paclitaxel alone (compound <b>551</b>). At higher concentrations (5 to 10 Î¼M), addition of these variants resulted in increased paclitaxel toxicity and decreased paclitaxel IC<sub>50</sub> of up to 500â1000-fold at 10 Î¼M, as compared to â¼100-fold sensitization caused by the parental compound <b>29</b> at 10 Î¼M. This data indicated that variants <b>227</b>, <b>231</b>, <b>541</b>, and <b>551</b> were better resensitizers of the multidrug resistant cells to paclitaxel than the original compound <b>29</b> at all concentrations tested, while variant <b>216</b> appeared to be marginally better than <b>29</b> at lower concentrations. The data therefore support the hypothesis that the ChemGen-generated and docking analyses-selected group 1 variants of compound <b>29</b> had increased affinity for P-gp resulting in improved efficacy for reversing chemotherapy resistance in a P-gp overexpressing cancer cell line compared with the parental compound.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Accumulation and Cellular Retention of Calcein AM in DU145TXR Cells upon Incubation with Group 1 SMU29 Variants</h3><div class="NLM_p">Calcein AM accumulation assays have been used by us previously to evaluate P-gp-substrate accumulation in real time in the presence or absence of P-gp inhibitors.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> For these assays, P-gp-overexpressing DU145TXR cells were incubated with the respective inhibitors in the presence of the P-gp substrate, calcein AM. Inhibition of P-gp results in cellular accumulation of calcein AM and cleavage of its acetoxymethyl ester groups, generating the highly fluorescent compound, calcein. The anionic calcein is not a substrate of P-gp and remains in the cells. In these assays, the relative fluorescence of cellular calcein was measured over time and the results of these assays are shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> (left panel). The data indicated that when these cells were treated with any of the five group 1 <b>29</b>-variants, the observed cellular accumulation of fluorescent calcein was lower than that upon treatment with the parental compound <b>29</b>. Only compound <b>551</b> resulted in marginally higher calcein accumulation than that of parental compound <b>29</b>.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cellular accumulation of the fluorescent dye, calcein, by inclusion of group 1 variants of the P-glycoprotein inhibitor, compound <b>29</b>, identified with ChemGen. The prostate cancer cell line, DU145TXR, that overexpresses P-glycoprotein was treated with calcein AM and without (left) or with (right) a 6 h preincubation with 2 Î¼M P-gp inhibitors indicated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To test whether the lower accumulation of calcein in the presence of the group 1 <b>29</b>-variants was the result of retention of the compounds in the cellular membrane due to their increased log <i>P</i> values relative to <b>29</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), similar calcein accumulation assays were performed after a 6 h preincubation with the <b>29</b>-variants and parental compound <b>29</b>, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> (right panel). We hypothesized that increased partitioning of variants into the hydrophobic part of the cell membrane may result in lowered cytosolic concentrations of the inhibitors and decreased binding to the putative allosteric site on P-gp, which is located in the cytoplasm, therefore potentially slowing the inhibitory effect of the compounds. The data of <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, right, suggested that calcein accumulation in the presence of the variants was improved by the 6 h preincubation for all group 1 compounds compared to the âno preincubationâ experiments, supporting our hypothesis that partitioning into the membrane may have been a contributing factor. Compound <b>541</b>, which has the lowest log <i>P</i> of the group 1 variants, performed relatively similar to <b>29</b> without preincubation. All of the variants except <b>216</b>, which has the highest log <i>P</i>, exceeded <b>29</b> in efficacy significantly upon the 6 h preincubation. Compound <b>216</b> was observed to be equivalent to <b>29</b> in efficacy even after the 6 h preincubation<named-content content-type="anchor" rid="tbl2" type="simple"></named-content>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Increased Toxicity of Paclitaxel to DU145TXR in the Presence of P-gp Inhibitors Identified by ChemGen/Docking Routine<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="19" align="center">resensitization to paclitaxel with the indicated treatment and fold-increased sensitivity in the presence of inhibitors</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" align="center">PTX alone</th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>29</b></th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>216</b></th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>227</b></th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>231</b></th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>541</b></th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>551</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">inhibitor concentration</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3 Î¼M</td><td class="colsep0 rowsep0" align="left">2120</td><td class="colsep0 rowsep0" align="left">629</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">266</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="left">164</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="left">153</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="left">426</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="left">56</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="char" char=".">11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5 Î¼M</td><td class="colsep0 rowsep0" align="left">2120</td><td class="colsep0 rowsep0" align="left">194</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">154</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="left">46</td><td class="colsep0 rowsep0" align="left">46</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">101</td><td class="colsep0 rowsep0" align="char" char=".">9.2</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">106</td><td class="colsep0 rowsep0" align="char" char=".">9.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7 Î¼M</td><td class="colsep0 rowsep0" align="left">2120</td><td class="colsep0 rowsep0" align="left">59</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">353</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">303</td><td class="colsep0 rowsep0" align="char" char=".">8.4</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">125</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">236</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10 Î¼M</td><td class="colsep0 rowsep0" align="left">2120</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">101</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">530</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1060</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">125</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">353</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Cytotoxicity of the chemotherapeutic paclitaxel (PTX) to P-gp-overexpressing prostate cancer cells, DU145TXR, was determined in the absence and presence of the ChemGen-designed <b>29</b>-variants, <b>216</b>, <b>227</b>, <b>231</b>, <b>541</b>, and <b>551</b>. For each experimental compound IC<sub>50</sub> values of PTX alone or in the presence of inhibitors, fold improvement of PTX sensitivity in the presence of the inhibitor and fold improvement of PTX sensitivity by variants compared to parental compound <b>29</b> are given.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Assessing the Roles of Polarity and Size of <b>29</b>-Variants in Improving Efficacy of Compound <b>29</b> Variants</h3><div class="NLM_p">To assess the contributions of overall hydrophobicity and size of the âWestern halvesâ of the <b>29</b>-variants on efficacy in inhibiting P-gp and reversing multidrug resistance in cancer cells, five structural derivatives of <b>29</b> (âgroup 2 variantsâ, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) were synthesized, which varied in size, shape, polar surface areas, and overall hydrophobicity as judged by calculated values of molecular weight, topological polar surface areas, and log <i>P</i> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The variations of structure in these molecules were again made in the âWesternâ half of the molecule (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) similar to group 1 variants.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Rationally designed <b>29</b>-variants (group 2 variants) docked at a putative allosteric inhibitory site on human P-glycoprotein. (A) The putative allosteric site on P-gp is shown in a surface representation with synthesized variants of compound <b>29</b> shown in licorice representations colored by their atoms (as indicated). Variants are shown as in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> with the surface of P-glycoprotein colored by its calculated electrostatic potential. (B) Structures of the rationally designed variant compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. <b>29</b>-Variants Differing in Overall Shape, Size, and Polarity that Did Not Undergo a Docking Routine<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">synthesized variant name</th><th class="colsep0 rowsep0" align="center" char=".">estimated Î<i>G</i><sub>binding</sub> (kcalÂ mol<sup>â1</sup>)</th><th class="colsep0 rowsep0" align="center">estimated <i>K</i><sub>d</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char=".">ratio of <i>K</i><sub>d</sub>Â 29/<i>K</i><sub>d</sub>Â variant</th><th class="colsep0 rowsep0" align="center">molecular weight (Da)</th><th class="colsep0 rowsep0" align="center" char=".">topological polar surface area (Ã<sup>2</sup>)</th><th class="colsep0 rowsep0" align="center" char=".">consensus log P</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29-238</b></td><td class="colsep0 rowsep0" align="char" char=".">â9.6</td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="left">575</td><td class="colsep0 rowsep0" align="char" char=".">129.0</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29-255</b></td><td class="colsep0 rowsep0" align="char" char=".">â11.0</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="left">542</td><td class="colsep0 rowsep0" align="char" char=".">142.5</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29-278</b></td><td class="colsep0 rowsep0" align="char" char=".">â9.9</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="left">543</td><td class="colsep0 rowsep0" align="char" char=".">101.3</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29-280</b></td><td class="colsep0 rowsep0" align="char" char=".">â9.6</td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="left">575</td><td class="colsep0 rowsep0" align="char" char=".">101.3</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29-286</b></td><td class="colsep0 rowsep0" align="char" char=".">â9.9</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="left">517</td><td class="colsep0 rowsep0" align="char" char=".">101.3</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ZINC08767731, â<b>29</b>â</td><td class="colsep0 rowsep0" align="char" char=".">â10.1</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">459</td><td class="colsep0 rowsep0" align="char" char=".">113.8</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Estimated <i>K</i><sub>d</sub> values for the ligand interactions with P-gp were calculated from the lowest estimated binding Î<i>G</i> values from the AutoDock calculations. The ratio of <i>K</i><sub>d</sub> values is given as a relative value for potentially changed affinities exhibited by the respective variants over the parent P-gp inhibitor compound <b>29</b>. Molecular weights, topological polar surface areas, and consensus log <i>P</i> values were calculated at the SwissADME website (<a href="http://www.swissadme.ch/" class="extLink">http://www.swissadme.ch/</a>) as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>.</p></div></div></div><div class="NLM_p">Group 2 variants were chosen without consideration of docking results for predicted high affinity to the putative allosteric site on P-gp; they were also not âcounter-selectedâ against binding affinity to the drug-binding domains of P-gp using computational docking studies. Instead, the group 2 variants were rationally designed upon visual inspection of the putative binding site to provide a larger volume to fill the void visible around <b>29</b> in the putative allosteric site (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) while at the same time somewhat decreasing the hydrophobicity (compare log <i>P</i> and TPSA values for group 1 compounds, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, with those for group 2 derivatives, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The chemical synthesis is given in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> shows that all five of these group 2 variants are somewhat larger than <b>29</b>, but the calculated log <i>P</i> values for these group 2 compounds (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) are closer to that of <b>29</b> than the log <i>P</i> values of the group 1 ChemGen-generated variants with the exception of <b>541</b>, which has the lowest log <i>P</i> value of that group (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The consensus log <i>P</i> values<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> of <b>238</b> and <b>255</b> were calculated to be 4.8 and 5.3, while those of <b>278</b>, <b>280</b>, and <b>286</b> were calculated to be 5.7, 6.0, and 5.2, respectively. The topological polar surface areas (TPSA) of <b>238</b> and <b>255</b> were calculated to be higher than those of <b>278</b>, <b>280</b>, and <b>286</b>. These TPSA values were also higher than those of the group 1 variants, <b>216</b>, <b>227</b>, and <b>231</b> (compare <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Of the five structural variants in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, two had increased calculated topological polar surface areas and three had reduced calculated topological polar surface areas when compared with <b>29</b>.</div><div class="NLM_p last">Docking of the group 2 variants to the putative allosteric inhibitor binding site of P-gp showed that the âWesternâ portions of compounds <b>238</b>, <b>255</b>, and <b>286</b> could penetrate deeper into the hydrophobic void than compound <b>29</b> but compounds <b>278</b> and <b>280</b> did not seem to penetrate as well as compound <b>29</b> (compare <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C,D).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Effects of Group 2 Variants on the Paclitaxel Sensitivity of DU145TXR Cancer Cells</h3><div class="NLM_p">MTT assays were again used to assess the efficacy of the new structural <b>29-</b>variants on sensitizing the chemotherapy-resistant prostate cancer cell line, DU145TXR, to paclitaxel (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Reversal of paclitaxel resistance by rationally designed variants of the P-glycoprotein inhibitor <b>29</b>. The prostate cancer cell line, DU145TXR, that overexpresses P-glycoprotein was treated with either paclitaxel (PTX) or paclitaxel and either 3, 5, 7, or 10 Î¼M P-gp inhibitors (upper left, upper right, lower left, and lower right panels, respectively). Metabolic activity assays using MTT were performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Calculated IC<sub>50</sub> values for paclitaxel in the presence of the respective compounds calculated from this data are reported in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Increased Toxicity of Paclitaxel in the Presence of Rationally Designed P-gp Inhibitors<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="19" align="center">resensitization to paclitaxel with the indicated treatment and fold-increased sensitivity in the presence of inhibitors</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" align="center">PTX alone</th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>29</b></th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">PTX + <b>238</b></th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">PTX + <b>255</b></th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>278</b></th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>280</b></th><th class="rowsep1 colsep0" colspan="3" align="center">PTX + <b>286</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">inhibitor concentration</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> PTX (nM)</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX</th><th class="colsep0 rowsep0" align="center" char=".">fold vs PTX + <b>29</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3 Î¼M</td><td class="colsep0 rowsep0" align="left">2120</td><td class="colsep0 rowsep0" align="left">629</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">185</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">172</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="left">231</td><td class="colsep0 rowsep0" align="char" char=".">9.2</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="left">160</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="left">458</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5 Î¼M</td><td class="colsep0 rowsep0" align="left">2120</td><td class="colsep0 rowsep0" align="left">194</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="left">1116</td><td class="colsep0 rowsep0" align="char" char=".">102</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="left">372</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td><td class="colsep0 rowsep0" align="left">109</td><td class="colsep0 rowsep0" align="char" char=".">19</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="left">56</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="left">379</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7 Î¼M</td><td class="colsep0 rowsep0" align="left">2120</td><td class="colsep0 rowsep0" align="left">59</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">0.64</td><td class="colsep0 rowsep0" align="left">3312</td><td class="colsep0 rowsep0" align="char" char=".">92</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="left">731</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="char" char=".">101</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">133</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="left">192</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10 Î¼M</td><td class="colsep0 rowsep0" align="left">2120</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">101</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">0.07</td><td class="colsep0 rowsep0" align="left">30,286</td><td class="colsep0 rowsep0" align="char" char=".">300</td><td class="colsep0 rowsep0" align="char" char=".">0.78</td><td class="colsep0 rowsep0" align="left">2118</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="char" char=".">163</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">193</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="left">131</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Cytotoxicity of the chemotherapeutic paclitaxel (PTX) in P-gp-overexpressing prostate cancer cells, DU145TXR, was determined in the absence and presence of the structural <b>29</b>-variants, <b>238</b>, <b>255</b>, <b>276</b>, <b>280</b>, and <b>286</b>. For each experimental compound, (a) IC 50 values of PTX alone or in the presence of inhibitors (in nM), (b) fold improvement of PTX sensitivity in the presence of an inhibitor, and (c) fold improvement of PTX sensitivity by variants compared to parental compound <b>29</b> are given.</p></div></div></div><div class="NLM_p">The data suggest that the presence of all of the variants except for <b>286</b> increased the paclitaxel toxicities to these P-gp-overexpressing cells (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Compounds <b>238</b> and <b>255</b> increased paclitaxel toxicities to the greatest extents: At a 5 Î¼M concentration, compound <b>238</b> decreased the paclitaxel IC<sub>50</sub> by about a thousandfold, similar to compound <b>255</b> at 7 Î¼M. Comparison with parental compound <b>29</b> showed that <b>238</b> improved sensitization of DU145TXR to paclitaxel by 3.4-fold at 3 Î¼M and by approximately 100-fold at 5 and 7 Î¼M. At 10 Î¼M, compound <b>238</b> resulted in a 300-fold decreased paclitaxel IC<sub>50</sub> of DU145TXR compared to parental compound <b>29</b> at the same concentration. Compound <b>255</b> also showed substantial improvement in sensitization of DU145TXR to paclitaxel that was comparable to <b>238</b> at 3 Î¼M, but the effect was not as pronounced at higher concentrations. The other three compounds, <b>278</b>, <b>280</b>, and <b>286</b>, were similar to or less effective than the group 1 variants, <b>216</b>, <b>227</b>, and <b>231</b>.</div><div class="NLM_p last">In addition, compound <b>280</b> seemed to exhibit some toxicity to the multidrug-resistant cancer cells as judged by the lowered cell viability at very low concentrations of paclitaxel in the presence of <b>280</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>).</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Accumulation and Cellular Retention of Calcein AM in DU145TXR Cells in the Presence of <b>29</b>-Variants from Group 2</h3><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, left, shows that without preincubation, the presence of 5 Î¼M compounds <b>238</b> and <b>255</b> resulted in considerably increased calcein accumulation in DU145TXR cells when compared to <b>29</b>, consistent with a log <i>P</i> close to that of the parental compound. The effect of compound <b>286</b> was comparable to <b>29</b>, while <b>278</b> and <b>280</b> caused less calcein accumulation than that of the parental compound. After a 6 h preincubation (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, right panel), the effectiveness of inhibiting P-gp-catalyzed transport of calcein AM remained strongest for the more polar <b>29</b>-variants <b>238</b> and <b>255</b>, while that of the more hydrophobic variants <b>278</b>, <b>280</b>, and <b>286</b> was comparable to <b>29</b>. Even though variant <b>286</b> has a consensus log <i>P</i> value similar to the log <i>P</i> calculated for <b>238</b> and <b>255</b>, it seems to group better with regard to efficacy in blocking calcein AM export with compounds that have comparable or lower calculated polar surface areas, that is, <b>278</b> and <b>280</b>, see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, and <b>216</b>, <b>227</b>, and <b>231</b> from group 1, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Group 1 compounds <b>216</b>, <b>227</b>, and <b>231</b>, in addition, have increased consensus log <i>P</i> values, which might explain their somewhat reduced efficacy in blocking P-gp-catalyzed calcein AM export (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Cellular accumulation of the fluorescent dye, calcein, by inclusion of group 2 variants of the P-glycoprotein inhibitor, compound <b>29</b>. The prostate cancer cell line, DU145TXR, that overexpresses P-glycoprotein was treated with calcein AM and without (left) or with (right) a 6 h preincubation with 2 Î¼M indicated P-gp inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Evaluation of the Mode of Inhibition of P-Glycoprotein by Group 1 and Group 2 Variants of Compound <b>29</b></h3><div class="NLM_p">To assess the mode of inhibition of P-gp by the novel variants of P-gp inhibitor <b>29</b>, ATP hydrolysis by P-gp was evaluated in the presence or absence of the variants. Both âbasalâ ATP hydrolysis (assayed in the absence of an added transport substrate) and âstimulatedâ ATP hydrolysis (assayed in the presence of the P-gp transport substrate verapamil) were assessed as described in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>. We used murine P-gp (MDR3) expressed in <i>Pichia pastoris</i> (<i>P</i>. <i>pastoris</i>) that had all naturally occurring cysteines replaced with alanine.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> It is widely assumed that the ATP hydrolytic rate of P-gp is stimulated in the presence of transport substrates when compared to ATP hydrolysis in the absence of transport substrates. Assays comparing these rates were therefore thought to be useful in identifying inhibitors of P-gp-catalyzed ATP hydrolysis and may also be a predictor for whether a compound might be a transport substrate if basal ATPase rates are stimulated by the addition of a compound.</div><div id="sec2_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Effects of Compound <b>29</b> Variants on Verapamil-Stimulated ATP Hydrolysis by P-gp</h4><div class="NLM_p">The effects of compound <b>29</b>-variants on P-gp ATP hydrolysis rates assayed in the presence of verapamil (a good substrate for transport by P-gp) are presented in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> (âstimulated ATPaseâ). The respective percent ATPase activity is shown, normalized to ATP hydrolysis in the presence of a DMSO carrier and with no experimental compound added. Interestingly, the group 1 compounds differed in their effects on âstimulatedâ ATPase: <b>216</b> did not affect stimulated ATP hydrolysis activities, while compounds <b>227</b>, <b>541</b>, and <b>551</b> inhibited activity similar to parental compound <b>29</b>. Compound <b>231</b> slightly stimulated ATP hydrolysis rates in the presence of verapamil. For group 2 compounds, <b>238</b> stimulated the âstimulatedâ ATPase rates by approximately 2-fold, while variant <b>280</b> showed only a slight stimulation of hydrolysis rates, and compounds <b>255</b> and <b>286</b> had no significant effect. Only compound <b>278</b> of the group 2 variants inhibited verapamil-stimulated ATP hydrolysis by P-gp similar to the parental compound <b>29</b>.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Mode of Inhibition of Cysteine-less Mouse MDR3 P-Glycoprotein by Group 1 and Group 2 Compound <b>29</b> Variants<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="Â±" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="Â±" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" colspan="2" align="center" char="Â±">stimulated ATPase (% DMSO, significance)</th><th class="colsep0 rowsep0" align="center">effect on stimulated ATPase</th><th class="colsep0 rowsep0" colspan="2" align="center" char="Â±">basal ATPase (% DMSO, significance)</th><th class="colsep0 rowsep0" align="center">effect on basal ATPase</th><th class="colsep0 rowsep0" colspan="2" align="center" char=".">cellular accumulation: ratio of plus tariquidar over no tariquidar</th><th class="colsep0 rowsep0" align="center">transport substrate for P-gp</th><th class="colsep0 rowsep0" align="center" char="Â±">maximum ATP binding (molÂ ofÂ SL-ANPÂ bound/molÂ ofÂ P-gp)</th><th class="colsep0 rowsep0" align="center" char="Â±">SL-ANP binding to P-gp, apparent <i>K</i><sub>d</sub> (Î¼M)</th><th class="colsep0 rowsep0" align="center">effect on SL-ANP binding</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DMSO</td><td class="colsep0 rowsep0" align="char" char="Â±">100Â Â±Â 8</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">100Â Â±Â 7</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">1.8 Â± 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">36.5Â Â±Â 3.6</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29</td><td class="colsep0 rowsep0" align="char" char="Â±">49 Â± 2</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">95 Â± 11</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="char" char="Â±">1.9Â Â±Â 0.1<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="Â±">71.0 Â± 12.0</td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="13" align="left">Group 1: ChemGen- and docking-selected</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-216</td><td class="colsep0 rowsep0" align="char" char="Â±">108Â Â±Â 2</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="char" char="Â±">105Â Â±Â 7</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="char" char="Â±">1.7Â Â±Â 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">23.1Â Â±Â 3.9</td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-227</td><td class="colsep0 rowsep0" align="char" char="Â±">50 Â± 2</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">88 Â± 4</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="char" char="Â±">1.8 Â± 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">36.9 Â± 4.1</td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-231</td><td class="colsep0 rowsep0" align="char" char="Â±">141 Â± 18</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">stimulator</td><td class="colsep0 rowsep0" align="char" char="Â±">70 Â± 0</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="char" char="Â±">1.6 Â± 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">22.2 Â± 3.8</td><td class="colsep0 rowsep0" align="left">marginally</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-541</td><td class="colsep0 rowsep0" align="char" char="Â±">68 Â± 7</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">101 Â± 12</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="char" char="Â±">1.8 Â± 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">24.1 Â± 4.0</td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-551</td><td class="colsep0 rowsep0" align="char" char="Â±">62 Â± 5</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">150 Â± 6</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">stimulator</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="char" char="Â±">1.7 Â± 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">22.8 Â± 3.6</td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="13" align="left">Group 2: rationally designed/no docking selection</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-238</td><td class="colsep0 rowsep0" align="char" char="Â±">194Â Â±Â 22</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">stimulator</td><td class="colsep0 rowsep0" align="char" char="Â±">1143Â Â±Â 46</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">stimulator</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="char" char="Â±">1.2Â Â±Â 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">20.1Â Â±Â 4.0</td><td class="colsep0 rowsep0" align="left">yes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-255</td><td class="colsep0 rowsep0" align="char" char="Â±">123 Â± 15</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="char" char="Â±">355 Â± 7</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="left">stimulator</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="char" char="Â±">1.9 Â± 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">25.6 Â± 4.7</td><td class="colsep0 rowsep0" align="left">no</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-278</td><td class="colsep0 rowsep0" align="char" char="Â±">41 Â± 7</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">78 Â± 4</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="char" char="Â±">1.3 Â± 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">22.9 Â± 4.5</td><td class="colsep0 rowsep0" align="left">yes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-280</td><td class="colsep0 rowsep0" align="char" char="Â±">116 Â± 4</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">stimulator</td><td class="colsep0 rowsep0" align="char" char="Â±">148 Â± 4</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">stimulator</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="char" char="Â±">1.6 Â± 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">21.6 Â± 4.6</td><td class="colsep0 rowsep0" align="left">marginally</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-286</td><td class="colsep0 rowsep0" align="char" char="Â±">98 Â± 2</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="char" char="Â±">143 Â± 32</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="char" char="Â±">1.9 Â± 0.1</td><td class="colsep0 rowsep0" align="char" char="Â±">25.7 Â± 4.9</td><td class="colsep0 rowsep0" align="left">no</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">ATP hydrolysis assays using purified P-gp were performed without added transport substrate (âbasal ATPaseâ) or in the presence of verapamil (âstimulated ATPaseâ). Results are presented compared to DMSO control Â± standard deviation (three independent experiments with duplicate samples). The basal activity of P-gp was 20 to 30 nmol min<sup>â1</sup> mg<sup>â1</sup>; verapamil-stimulated rates were 200 to 400 nmol min<sup>â1</sup> mg<sup>â1</sup> of P-gp. Stimulation of basal ATPase by <b>29</b>-variants was used as an indicator that a compound may be a P-gp transport substrate. Effects on stimulated P-gp ATPase activity indicated whether a compound directly interfered with ATP usage by the protein (***, <i>p</i> < 0.001; **, <i>p</i> < 0.01; *, <i>p</i> < 0.1; NS, not significant). Quantitative cellular accumulation of <b>29</b>-variants was performed using LC-MS/MS (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>) and is presented as a ratio of the cellular amounts of <b>29</b>-variants in the presence of the P-gp inhibitor tariquidar, divided by amounts accumulated in its absence. A ratio of >1 indicates that the compound likely is a transport substrate of P-gp (***, very significant; *, significant; NS, not significant). The binding of an ATP analog, SL-ATP, to P-gp was used to determine whether ATP binding to P-gp was affected by the <b>29</b>-variants (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Values Â± standard deviations are shown for at least three different P-gp preparations and three independent SL-ATP titration experiments.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">The values for SL-ATP binding in the presence of <b>29</b> were taken directly from Brewer et al.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></p></div></div></div></div><div id="sec2_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Effects on âBasalâ ATP Hydrolysis Rates of Compound <b>29</b> Variants</h4><div class="NLM_p last">Group 1 compounds <b>216</b>, <b>227</b>, and <b>541</b> did not significantly affect basal ATP hydrolysis by P-gp, while compound <b>231</b> was inhibitory. Only <b>551</b> of the group 1 molecules stimulated basal ATPase activities of P-gp. Of the group 2 compounds, <b>238</b> and <b>255</b> stimulated basal ATPase by â¼10- and â¼3-fold, respectively. Compounds <b>280</b> and <b>286</b> stimulated basal ATPase marginally or with no statistical significance. Only compound <b>278</b> inhibited basal ATPase of P-gp. The relatively strong activation of basal ATPase by compounds <b>238</b> and <b>255</b> was suggestive that these two compounds and, potentially to a lesser extent, compound <b>280</b>, may be transport substrates of the pump. Compound <b>278</b> was not assumed to be a good transport substrate for P-gp since it inhibited basal ATPase by P-gp.</div></div><div id="sec2_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Intracellular Accumulation of Compound <b>29</b> Variants</h4><div class="NLM_p">To more directly assess whether the <b>29</b> variants were indeed transport substrates for P-gp, cell accumulation assays were performed as in ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>. These assays measured the intracellular accumulation of the experimental compounds using LCâMS/MS methods after incubation with the P-gp-overexpressing cell line, DU145TXR, in the absence and presence of the strong P-gp inhibitor, tariquidar<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> (TQR). Low levels of cellular accumulation of a compound in the absence of tariquidar accompanied by much higher levels of accumulation in the presence of tariquidar suggests that the compound in question may be a transport substrate of P-gp. If a compound is not a substrate of P-gp, no significant difference in intracellular accumulation of the compound with or without tariquidar is expected. Daunorubicin (DNR) was used as a positive control of a good transport substrate for P-gp and showed very strong cellular accumulation in these assays when P-gp was inhibited by tariquidar, but much less accumulation in the cells when P-gp was not inhibited (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S5</a>, DNR). <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S5</a> â<b>29</b>â shows that compound <b>29</b> is not a transport substrate for P-gp as already discussed in ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and no significant difference in cellular accumulation of <b>29</b> was observed with or without addition of tariquidar (TQR). <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S5</a> also presents the fold accumulation of each of the experimental <b>29</b>-variants in these assays, normalized to the amount of the compound found in the absence of TQR. This data is numerically presented in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> as the ratio of observed accumulation in the presence of tariquidar divided by the accumulation observed in the absence of tariquidar for each of the experimental compounds. Ratios that are significantly greater than 1.0 indicate that a compound is very likely a transport substrate of P-gp.</div><div class="NLM_p">None of the group 1 molecules resulted in intracellular accumulations that were considerably different in the absence or presence of TQR, similar to the parental compound <b>29</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S5</a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>), indicating that these variants were likely not transport substrates of the pump in human cells in culture.</div><div class="NLM_p">Of the group 2 compounds, variant <b>238</b> showed a very large and significant increase in intracellular accumulation in the presence of TQR (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S5</a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Compounds <b>255</b> and <b>286</b> showed more modest but statistically significant increases in intracellular accumulation when P-gp was inhibited in the presence of TQR. Compounds <b>278</b> and Compound <b>280</b> did not show significantly different intracellular accumulations in the absence or presence of TQR.</div><div class="NLM_p">To assess whether the observed discrepancies of compounds stimulating basal P-gp ATPase activity but are not being transport substrates of the human pump in the cell culture assessments was due to the fact that, in the biochemical assays, we used a cysteine-less variant of the mouse MDR3 P-glycoprotein, we repeated the ATP hydrolysis experiments using normal human MDR1 P-gp. In order to stabilize the human protein for the activity assays, the protein was assembled into membrane nanodiscs as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. The results of the experiments are shown in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>. Interestingly, neither compound <b>29</b> nor any of its variants had a stimulatory effect on basal ATPase activity of the normal human protein reconstituted into membrane nanodiscs, while all of them significantly inhibited transport substrate (verapamil)-stimulated activity. The results clearly indicate that the source (human vs mouse) and potentially also the membrane environment of P-glycoprotein strongly affect the overall behavior of potential biochemical inhibitors and their impact on the catalytic mechanism of the enzyme.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Effects of Group 1 and Group 2 Compound <b>29</b> Variants on ATP Hydrolysis by Normal Human MDR1 P-glycoprotein<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="Â±" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="Â±" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" colspan="2" align="center" char="Â±">stimulated ATPase (% DMSO, significance)</th><th class="colsep0 rowsep0" align="center">effect on stimulated ATPase</th><th class="colsep0 rowsep0" colspan="2" align="center" char="Â±">basal ATPase (% DMSO, significance)</th><th class="colsep0 rowsep0" align="center">effect on basal ATPase</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DMSO</td><td class="colsep0 rowsep0" align="char" char="Â±">100Â Â±Â 6</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">100Â Â±Â 5</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29</td><td class="colsep0 rowsep0" align="char" char="Â±">62 Â± 3</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">88 Â± 9</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">Group 1: ChemGen and docking selected</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-216</td><td class="colsep0 rowsep0" align="char" char="Â±">64Â Â±Â 6</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">69Â Â±Â 8</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-227</td><td class="colsep0 rowsep0" align="char" char="Â±">62 Â± 5</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">83 Â± 7</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-231</td><td class="colsep0 rowsep0" align="char" char="Â±">40 Â± 3</td><td class="colsep0 rowsep0" align="left">****</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">70 Â± 4</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="left">inhibitor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-541</td><td class="colsep0 rowsep0" align="char" char="Â±">66 Â± 6</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">87 Â± 9</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-551</td><td class="colsep0 rowsep0" align="char" char="Â±">27 Â± 1</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">72 Â± 9</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">inhibitor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">Group 2: rationally designed/no docking selection</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-238</td><td class="colsep0 rowsep0" align="char" char="Â±">61Â Â±Â 7</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">89Â Â±Â 7</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-255</td><td class="colsep0 rowsep0" align="char" char="Â±">59 Â± 6</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">84 Â± 9</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-278</td><td class="colsep0 rowsep0" align="char" char="Â±">63 Â± 6</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">74 Â± 6</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="left">inhibitor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-280</td><td class="colsep0 rowsep0" align="char" char="Â±">54 Â± 7</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">76 Â± 8</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="left">inhibitor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMU29-286</td><td class="colsep0 rowsep0" align="char" char="Â±">59 Â± 4</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="char" char="Â±">95 Â± 3</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="left">none</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">ATP hydrolysis assays using purified human P-glycoprotein were performed (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>) without added transport substrate (âbasal ATPaseâ) or in the presence of verapamil (âstimulated ATPaseâ). Results are presented compared to DMSO control Â± standard deviation (three independent experiments with duplicate samples). The specific basal activity of normal human MDR1 P-gp was between 123 and 193 nmol min<sup>â1</sup> g<sup>â1</sup>, and transport substrate (verapamil)-stimulated activity was between 193 and 263 nmol min<sup>â1</sup> mg<sup>â1</sup>. Effects on stimulated P-gp ATPase activity indicated whether a compound directly interfered with ATP usage by the protein (***, <i>p</i> < 0.001; **, <i>p</i> < 0.01; *, <i>p</i> < 0.1; NS, not significant).</p></div></div></div></div><div id="sec2_8_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Effects of <b>29</b>-Variants on Binding of an ATP Analog to Purified P-Glycoprotein</h4><div class="NLM_p last">ATP binding in the presence of the <b>29</b>-variants was assessed in titration assays using a spin-labeled analog of ATP, 2â²,3â²-SL-ATP (2â²,3â²-(2,2,5,5,-tetramethyl-3-pyrroline-l-oxyl-3-carboxylic acid ester)ATP; (2â²,3â² indicates a rapid equilibrium of the ester bond between the C2â² and C3â² of the ribose moiety)<a onclick="showRef(event, 'ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50">(48â50)</a> and electron spin resonance spectroscopy as described in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>. Due to the lower stability of the human P-glycoprotein in the extended times needed for these experiments, the cysteine-less mouse protein was used here. The goal was to assess whether binding of the <b>29</b>-variants to P-gp affected nucleotide binding to the protein. Results of these assays are presented in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. Except for compounds <b>238</b> and <b>278</b>, neither of which initially underwent the selective docking routines used for group 1 compounds, none of the novel inhibitors affected maximal binding of the ATP analog or the apparent <i>K</i><sub>d</sub>, supporting the hypothesis that the inhibitors were indeed interacting with the putative allosteric binding site on P-gp. Compounds <b>238</b> and <b>278</b> reduced SL-ATP binding to approximately 1 mol of SL-ANP (ANP, adenine nucleotide with an undefined number of phosphoryl groups) bound/mole of enzyme, suggesting that these inhibitors may also interact with the nucleotide-binding sites or may indirectly induce changes in P-gp that affect ATP binding.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77032" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Using Computational Approaches To Create Novel Variants of âHitâ Molecules from Drug Discovery Programs</h3><div class="NLM_p">A number of virtual chemical synthesis computer programs have been previously described: some use fragments annotated with reaction rules<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> or compound scaffolds with chemically reactive linkers<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and again others use popular click chemistries,<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> just to mention a few. To make more informed choices about which of the vast numbers of possible compound variants to synthesize for subsequent testing, we have written and developed a set of computational routines (collectively called ChemGen) to in silico synthesize what could be very large numbers of variant compounds. Our methods differ from predecessor methods in that retrosynthetic approaches to the discovered hit molecule synthetic routes are mimicked in the computations. This results in advantageous translation to actual chemical syntheses of identified variants of interest and is not constrained to one or a few chemical reaction types. A disadvantage of the approach is that each reaction type must be programmed ahead of its implementation; however, the <i>ChemGen</i> platform can be relatively easily adapted to new chemistries.</div><div class="NLM_p">In this presented pilot study, we created hit variants computationally and then assessed them computationally for variants that bind with increased affinity to desirable protein structures and not well to structures that were undesired for this particular protein target, that is, improved binding to regions in the nucleotide-binding domains of P-glycoprotein but low affinity to the drug-binding domains in our efforts to find inhibitors of the protein that are not transport substrates themselves. This latter part of the approach, namely, computational counter-selection for compounds with undesirable binding characteristics, was successfully used by us previously to identify molecules, including hit compound <b>29</b>, that inhibited P-gp catalysis but were not transport substrates of P-gp.<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26â28)</a> When coupled with virtual synthesis of hit variants, increased efficiency and cost-effectiveness of synthesis projects are practically assured.</div><div class="NLM_p">Compound <b>29</b> and the other hits discovered were evaluated in biochemical and biophysical studies for their mechanism of inhibition of P-gp action<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> as well as for their potential to reverse multidrug resistance in different cancer cell lines in culture.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Compound <b>29</b> was chosen as an initial compound for further development in this present study mostly for the fact that binding of compound <b>29</b> was predicted to be at an allosteric site, away from the nucleotide-binding sites of P-gp,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> which was hypothesized to reduce potential off-target effects of the inhibitor and its derivatives. Earlier biophysical assessment using electron spin resonance spectroscopy and a spin-labeled ATP analog suggested that ATP binding was not affected in the presence of <b>29</b>, while ATP hydrolysis assays showed inhibition of ATPase activity.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The putative mechanism for P-gp inhibition by <b>29</b> can be envisioned as shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B when comparing the position to which <b>29</b> docked with high affinity<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> to the recently published cryo-EM structure of the protein<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a> shows a pronounced steric clash of P-gp amino acid side chains with the bound inhibitor when P-gp adopts a conformation similar to that of the published cryo-EM structure. This clash may suggest that, in the presence of bound <b>29</b>, P-gp cannot undergo conformational changes that may be needed for catalytic activity. Creating variants of compound <b>29</b> with increased affinity to this particular binding site was therefore viewed as a promising strategy toward further development of specific P-gp inhibitors that would not function as a transport substrate of the protein.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0002.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Putative allosteric inhibitory site on human P-glycoprotein with the ATPase inhibitor compound <b>29</b> bound. Compound <b>29</b> shown in a licorice representation with its solvent-excluded surface in a wire mesh as it was docked in the P-gp model shown in a blue cartoon. Shown in a red cartoon representation is the cryoEM-derived structure determined by Kim and Chen (2018, PDB accession <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0C">6V0C</a>) superposed on the P-gp model used for docking. One can clearly observe a steric clash of <b>29</b> with the cryoEM structure (black arrow). Residues involved in the steric clash with <b>29</b> (atoms within 2.0 Ã) are T422, E566, Q570, V584, S590, T591, and N594. Small molecules are shown in the figure with nonpolar hydrogens removed for clarity. Images were created with VMD with the MSMS surface renderer from M. Sanner and the Tachyon image renderer in VMD (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Closer evaluation of the high-affinity allosteric docking site of compound <b>29</b> to P-gp revealed a relatively large hydrophobic pocket where the cyclopropyl moiety of the âWesternâ half of the molecule interacted with the protein (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,C,D). In our efforts to assess putative P-gp inhibitors with increased affinity to the protein, we set out to âvirtually synthesizeâ a number of <b>29</b>-variants with larger moieties at the âWesternâ half of the protein. We used the synthesis scheme shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B and the ChemGen protocols described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Supporting Information</a> to accomplish this goal. The resulting 647 derivatives of hit compound <b>29</b> were evaluated for binding to the allosteric site and counter-screened for low-affinity interactions to the drug-binding domains of the protein using docking methods that were similar to those described in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and that led to the discovery of the parental compound <b>29</b>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><div class="NLM_p">The ChemGen in silico synthesized and docking routine-selected group 1 variants of <b>29</b> were ranked by binding affinity to the allosteric site on P-gp. Compounds with molecular weights that exceeded 600 Da were originally excluded from synthesis and further evaluation. Twelve variants of <b>29</b> that were predicted to bind with relatively high affinity to the proposed allosteric site were further evaluated for some physicochemical characteristics (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). All of the variants showed higher molecular weights as was to be assumed when adding larger fragments to the âWesternâ part of the molecule. TPSA and consensus log <i>P</i> values differed between the variants. Visual evaluation of the docking poses of the variants showed clear overlap of the âEasternâ parts of the molecules, highlighting the consistency of docking to this site on P-gp. The âWesternâ portions of the Group 1 <b>29</b>-variants were observed to extend further into the hydrophobic pocket of the protein compared with the original compound <b>29</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S3</a>).</div><div class="NLM_p">Of the twelve <b>29</b>-variants given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, five were chosen for actual chemical synthesis mostly based on their perceived ease of synthesis and relatively low expense of precursor fragments (<b>216</b>, <b>227</b>, <b>231</b>, <b>541</b>, and <b>551</b>). Variant <b>551</b> was also chosen for synthesis even though its molecular weight of 627 Da exceeded our initial <i>M</i><sub>w</sub> cutoff mostly because of the ease of synthesis and also for its otherwise favorable characteristics. All variants added more volume to the âWesternâ half of the molecules, and all but <b>551</b> and to some extent <b>541</b> had lower TPSA than that of the original compound <b>29</b>. All variants had somewhat higher consensus log <i>P</i> values than those of <b>29.</b> Closer inspection of the docking poses of the five variants (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) revealed that <b>29</b>-derivatives <b>216</b> and <b>231</b> both reached significantly farther into the hydrophobic pocket compared with the parental compound <b>29</b>, while variant <b>227</b> seems to make significant protein interactions at the âmouthâ of the hydrophobic pocket. Both <b>541</b> and <b>551</b> reached deeply into the previously observed pocket within P-gp. We hypothesized that these five variants should show improvements in reversing the MDR phenotype of cancer cells that overexpress P-gp as demonstrated for <b>29</b> in refs <a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and would therefore be reasonable initial choices for this proof of principle study.</div><div class="NLM_p">Cell viability assays using a P-gp-overexpressing prostate cancer cell line indicated that all five of the <b>29</b>-variants had improved characteristics over the parental <b>29</b> in causing increased cell mortality (summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) in the presence of the chemotherapeutic paclitaxel. This remarkable result that five out of five group 1 ChemGen-produced variants showed between 2.4-fold and 11-fold improvement over the performance of <b>29</b> in reversing the P-glycoprotein-conveyed multidrug resistance phenotype (summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) underscores the utility of the ChemGen virtual synthesis and docking approach for increasing variant affinity to P-gp. It is a reasonable conclusion that the larger group 1 variants were able to interact more strongly with the protein as depicted in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> than the parental compound <b>29</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S3</a>), adding to the overall binding affinity of the variants.</div><div class="NLM_p">In assays designed to allow quantification of the accumulation of a P-gp transport substrate, calcein AM, in cells that overexpress P-gp, however, the larger, more hydrophobic group 1 variants did not initially perform better than <b>29</b>. Comparison of the calculated consensus log <i>P</i> values for variants <b>216</b>, <b>227</b>, <b>231</b>, <b>541</b>, and <b>551</b> (6.6, 6.4, 5.7, 5.4, and 5.9, respectively, see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) with the log <i>P</i> of the parental compound <b>29</b> (4.6, see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) led us to hypothesize that the lack of efficacy in inhibiting P-gp-catalyzed export of calcein AM may have been due to <b>29</b>-variants being too hydrophobic to efficiently transfer across the cellular membrane to the cytosol-located nucleotide-binding domains of P-gp for efficious inhibition in the nucleotide-binding domain of the protein to occur. The relatively short incubation times that are used in the calcein AM assays as shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, left, would hypothetically exacerbate this problem when compared to much longer exposure times in the cellular toxicity assays (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Supporting this hypothesis was the observation of significant improvements in efficacy in the calcein AM assays when longer preincubation with the variants was performed, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, right. Except for variant <b>216</b>, which has the lowest TPSA and highest consensus log <i>P</i> and which under these conditions performed similar to parental compound <b>29</b>, all other variants performed better in retaining the P-gp substrate than <b>29</b>. The variants with the most promising physicochemical characteristics (relatively high TPSA/ relatively low log <i>P</i> values), compounds <b>541</b> and <b>551,</b> performed best in these assays. This finding supports the hypothesis that given enough time to enter the cells, the group 1 variants performed better or at least equally to the parental compound.</div><div class="NLM_p last">Since the ChemGen/docking routine computational approach for selecting variants at the putative allosteric site resulted in larger but also more hydrophobic âWesternâ fragments in variants <b>216</b>, <b>227</b>, <b>231</b>, <b>541</b>, and <b>551</b> than those of the parental compound <b>29</b> and increased hydrophobicity may be viewed as pharmacologically less desirable in drug candidates, we set out to rationally design another set of variants of the âWesternâ portion of compound <b>29</b> that might also result in improved interactions with P-gp but may in addition have also more favorable physicochemical characteristics. In this light, we synthesized five additional âWesternâ derivatives of <b>29</b> with varying shapes, sizes, and physicochemical properties, compounds <b>238</b>, <b>255</b>, <b>278</b>, <b>280</b>, and <b>286</b> (group 2 variants). The compounds were chosen for relative ease of synthesis as well as availability and cost of precursors. Unlike the previous set of ChemGen-generated group 1 variants, this latter group did not initially undergo the docking routines that selected for high-affinity binding to the nucleotide-binding domains and low-affinity binding to the drug-binding domains. Instead, these variants were chosen by visual inspection of the putative binding site as well as by the calculated physicochemical properties of the resulting variants.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Efficacy of Rationally Designed Group 2 Variants in Reversing MDR Caused by P-gp Overexpression</h3><div class="NLM_p">If the hypothesis was correct that compounds <b>216</b>, <b>227</b>, <b>231</b>, <b>541</b>, and <b>551</b> were too hydrophobic for a kinetically favorable passage through the cellular membrane into the cytoplasm, thereby causing less efficient P-gp inhibition and decreased calcein AM accumulation in the P-gp overexpressing cells, then the more hydrophilic group 2 derivatives, <b>238</b>, <b>255</b>, as judged by their TPSA and log <i>P</i> values, and possibly <b>286</b> as judged by its log <i>P</i> value, might show improved calcein AM retention as compared to the other derivatives. Indeed, improved calcein AM retention was observed for compounds <b>238</b> and <b>255</b> already for the shorter incubation times (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, left), while <b>286</b> performed similar to <b>29</b>.</div><div class="NLM_p">The more hydrophobic compounds <b>278</b> and <b>280</b> (as judged by <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) performed similar to the parental compound only upon extended exposure to the cells.</div><div class="NLM_p last">All in all, TPSA values greater than â¼110 Ã<sup>3</sup> and consensus log <i>P</i> values lower than 6 correlated best with MDR reversal efficacy in both the group 1 and group 2 variants and the most pronounced retention of the P-gp substrate, calcein AM.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Effectiveness of the Selective Docking Routines Used in Group 1 Variants To Ensure Targeting to the Nucleotide-Binding Domains and Avoiding the Drug Transport Domains of P-gp</h3><div class="NLM_p">Since our initial premise was to generate a new class of P-gp inhibitors that are not transport substrates of the pump, it was of interest to investigate whether the subtractive docking routine performed on group 1 variants added value to the overall selection variants for synthesis. Comparison of the results from group 1 compounds to the more traditional, rationally designed group 2 variants that were chosen without additional input by docking-derived binding affinities to different substructures of the protein was therefore made. To this end, we assessed the effects of the <b>29</b>-variants on basal (no transport substrate present) ATPase activity of P-gp as well as the intracellular accumulation of the hit variants in the P-gp-overexpressing cancer cells, DU145TXR, in the presence or absence of a strong inhibitor of P-gp, tariquidar. Stimulation of basal ATPase activity of P-gp is commonly believed to be a good indicator that a transport substrate is present in the assay. Interestingly, the cellular accumulation assays (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Figure S5</a>) and ATP hydrolysis assays did not correlate all too well. While all of the group 1 variants were shown not to be transport substrates in the cellular accumulation assays as in ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>, only <b>231</b> resulted in inhibition of basal ATPase activity, while <b>551</b> stimulated the activity, and the rest had no effect.</div><div class="NLM_p">To evaluate whether this lack of correlation may have been due to the fact that these biochemical assays were performed using a cysteine-less variant of the mouse MDR3 P-glycoprotein and not the human isoform, we repeated the experiments using normal (not cysteine-less) human P-gp that we had reconstituted into nanodiscs for enhanced stability. The results presented in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> indicate that none of the <b>29</b>-variants nor <b>29</b> itself stimulated basal ATP hydrolysis but that, in contrast, several of them inhibited basal ATPase activity. This finding places doubt on the general assumption in the field that transport substrates stimulate basal ATP hydrolysis. We therefore conclude that simple assessment of basal P-gp hydrolysis activities may not be a good predictor for the biochemical mechanism of an inhibitor. While not giving any information about whether or not an inhibitor is a transport substrate of P-gp, a much better prediction can be gleaned by assessing the inhibition of transport substrate stimulated ATPase activity using the normal human P-gp as shown in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>. All of the variants of compound <b>29</b> inhibited stimulated activity, indicating interaction with the nucleotide-binding domain as predicted from the putative binding of <b>29</b> and its variants to an allosteric site as depicted in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>.</div><div class="NLM_p last">Although the general case cannot be statistically proven since the number of synthesized and tested variants was small, these experiments support the hypothesis that the ChemGen-produced variants that were selected against interactions at drug-transporting structures on P-gp were much more likely not to be transport substrates of P-gp (five of five group 1 variants were not observed to be transport substrates) than the group 2 molecules that did not undergo the docking counter-selection procedures (where four out of five molecules were likely or potentially transport substrates for P-gp). The studies strongly suggest that the docking and counter-selections procedures used to select the group 1 variants outperformed the more traditionally rationally designed molecules that were not subjected to these selections.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Mechanism of Inhibition of the <b>29</b>-Variants</h3><div class="NLM_p">To assess whether the <b>29</b>-variants likely interacted with the putative allosteric site or potentially also interacted with the nucleotide-binding sites of P-glycoprotein, titration experiments using an electron spin resonance (ESR)-active ATP analog, SL-ATP, were performed and the amount of P-gp bound SL-ANP was determined in the presence of the <b>29</b>-variants (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). The results showed that none of the group 1 variants significantly affected ATP binding, while two of the group 2 variants reduced ATP binding from 2 to approximately 1 mol of the nucleotide analog/mol of protein. This again indicates that the selection process through ChemGen/selective docking was much more predictive of the effects the potential inhibitors had on the enzyme.</div><div class="NLM_p">Among all compounds tested, the strongest reversal of MDR and the strongest stimulator of both âbasalâ and âverapamil-stimulatedâ ATPase activities was variant <b>238</b> of the group 2 molecules. Compound <b>238</b>, while strongly reversing MDR, was also the best transport substrate of all the variants tested, a characteristic that we deemed undesirable for any clinically relevant P-gp modulator lead as discussed above. Although our curiosity on the mechanism of action of <b>238</b> is piqued by the observation that this compound also inhibited SL-ANP binding to P-gp (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>), we will likely not consider it for further clinical lead development.</div><div class="NLM_p">The study presented here supports the hypothesis that the virtual synthesis of hit variants with a program suite like ChemGen combined with selection for desired characteristics predicted from docking experiments, that is, increased affinity to targeted structures and decreased affinity against substructures that should be avoided, as employed here, resulted in efficient and cost-effective identification of five out of five variants assessed that met these goals. A comparison of the synthetic capabilities of ChemGen with, for example, commercially available programs that perform synthetic chemistry reactions from ChemAxon (i.e., Reactor), shows that the potential for both packages to perform a wide variety of different chemical reactions is similar. Both programs deal with a virtual synthesis engine that can use a large library of prescreened or more randomly acquired reactant molecules to create structural variants in structure-based lead optimization studies. ChemGen also gives the option for fairly rigorous quantum mechanical optimizations of the bonding geometries of synthetic products and looks for steric interference between reactants for the identification and rejection of variants that would likely be impossible or difficult to obtain synthetically. In addition, ChemGen is set up for convenient prescreening of reactant molecules for optimized pharmacological and physiochemical fragment properties. Each program requires the expert knowledge of a synthetic chemist to specify each specific chemical reaction to be performed. While the ChemAxon Reactor has more known chemistries that have been preprogrammed to draw on due to its longer and necessarily much wider use, it should be noted that the known chemistry libraries utilized by ChemGen can be expanded on demand by any user. While both programs require either an atomic resolution structure and molecular docking of the starting âhitâ ligand or ligands to start the processes, the most prominent advantage of ChemGen is that, in addition to medicinal chemistry, target selection is included in the suite. The ability to screen for preferential binding to one substructure of a target (here, a putative allosteric site in the nucleotide-binding domains) while avoiding binding to an undesirable structure (here, the drug-binding domains) was shown in the study to be highly successful. The main goal to this study was to discover derivatives of hit compound <b>29</b> that were not transport substrates of P-glycoprotein while inhibiting ATP hydrolysis and reversing drug resistance in drug-resistant cancer cells. All five variants synthesized and assayed from this group fulfilled all of these requirements. None of the variants affected nucleotide binding to the protein, suggesting that all variants were binding to the allosteric site targeted. All of the five variants rationally designed to fit into the putative allosteric binding site of hit compound <b>29</b> reversed drug resistance in cancer cells, and all inhibited transport substrate stimulated ATP hydrolysis by the human MDR1 protein, but two out of the five variants affected nucleotide binding to P-gp, indicating that these variants may also directly bind to the nucleotide binding sites that were not originally targeted. Four out of the five variants were also shown to be transport substrates of the pump, a characteristic that was not predictable from simple visual inspection of the desired binding interactions with the protein.</div><div class="NLM_p last">Even though this presented study is a pilot study in scope, the exceptionally high success rate of 100% for all desired characteristics of the ChemGen variants as compared to 60% for binding specificity to the allosteric site and only 20% for avoiding the drug-binding domains of the more traditionally rationally designed variants is a good example of how computational evaluation and selection of potential inhibitors before their actual synthesis adds to the speed and overall success rates in identifying hit-to-lead variants that possess desired characteristics. The use of ever more sophisticated presynthesis computational selections will likely yield even better variants in the future. Computational routines that employ parameters such as those used here, that is, increased affinity to targeted structures and avoidance of undesirable interactions, can in the future be easily expanded to include avoidance of targeting other proteins, optimized physicochemical properties for solubilities and membrane penetration, and even complex parameters like decreased cellular toxicity and avoidance of adverse ADME properties that can be potentially learned through deep learning derived computational studies. Such expansion of computational preselection routines would be expected to yield better molecules faster with less financial and time investment than those of current, lesser-guided rational design programs.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48535" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48535" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Virtual Synthesis of Compound <b>29</b>-Derivatives Using ChemGen</h3><div class="NLM_p last">Because of the nature of the putative allosteric site shown for compound <b>29</b> and in light of rational design considerations (see text), only a moderate set of variants were virtually synthesized for this study using the ChemGen programs. The ChemGen program is described in detail in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Supporting Information</a> to this paper. In the virtual syntheses performed here, a scaffold molecule, 2-chloro-<i>N</i>-[1-phenyl-3-(2,4,5-trimethylphenyl)-1<i>H</i>-pyrazol-5-yl]acetamide, which is equivalent to the chloroacetamide that retains the âEasternâ substituent group from compound <b>29</b>, was reacted with approximately 650 thiol compounds obtained by simple structural searches for thiols from the âclean drug-likeâ commercially available molecule set at the ZINC database.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The 2-chloro-<i>N</i>-[1-phenyl-3-(2,4,5-trimethylphenyl)-1<i>H</i>-pyrazol-5-yl]acetamide scaffold and the thiol precursor molecules were âmarkedâ for reactions in ChemGen as described for the second reaction shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Supporting Information</a> for details about the ChemGen marking and molecule building methods). The build process successfully created 647 derivatives of the âEasternâ substituents of <b>29</b>, which were then geometrically optimized with the electronic Ligand Builder and Optimisation Workbench (eLBOW) â Phenix<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> program.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> In Silico Docking of Compound <b>29</b> Variants to a Model of Human P-gp</h3><div class="NLM_p last">AutoDock Vina<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> and AutoDock 4.2<a onclick="showRef(event, 'ref52 ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref52 ref53 ref54">(52â54)</a> were used with a model of P-glycoprotein with drug-binding domains open to the outside and nucleotide-binding sites fully formed that was extracted from targeted molecular dynamics trajectories as described in refs <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and <a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a>. This conformation of P-gp is one that is very similar to the homologous Sav1866 crystal structure reported by Dawson and Locher<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> and was the conformation with which compound <b>29</b> was originally identified.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Ligand docking was limited to a volume equivalent to 20 Ã 24 Ã 26 Ã<sup>3</sup> centered on the putative allosteric site of P-gp (see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). One hundred twenty-eight replicates for each ligand were calculated and then ranked by the lowest estimated binding energies to the protein target calculated by the docking programs.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Calculation of Physicochemical Properties</h3><div class="NLM_p last">The SWISS-ADME server at <a href="http://www.swissadme.ch/" class="extLink">http://www.swissadme.ch/</a> was used for the calculation of the physicochemical properties of the compounds as discussed in ref <a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Imaging of P-Glycoprotein and Ligands</h3><div class="NLM_p last">The VMD (visual molecular dynamics) program suite was used extensively in this work for the analysis of structural data and for the presentation of visual images<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and included the SURF surface representation program<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> as well as the pdb2pqr<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> and APBS<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> programs for electrostatic/solvation calculations.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Synthetic Procedures</h3><div class="NLM_p last">All synthetic procedures and analyses of products are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">Supporting Information</a>, âSynthetic Proceduresâ. All compounds were â¥95% pure as determined by <sup>1</sup>H NMR and high-performance liquid chromatography (HPLC). The chromatographic separations were accomplished by using an Ascentis C18 25 cm Ã 2.1 mm, 5 Î¼m column with 10 Î¼L injection volumes. A flow rate of 0.2 mL min<sup>â1</sup> was used and eluted with a mobile phase of 100% water for 3 min, a linear gradient to 100% acetonitrile over 2 min, and 15 min of 100% acetonitrile.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Cell Lines and Cell Culture</h3><div class="NLM_p last">The chemotherapeutic sensitive DU145 human prostate cancer cells<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> as well as the multidrug-resistant sub-line, DU145TXR<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> were generous gifts from Dr. Evan Keller (University of Michigan, Ann Arbor, MI). The multidrug-resistant DU145TXR was maintained under positive selection pressure by supplementing complete medium with 10 nM paclitaxel (Acros Organics, NJ). Both cell lines were maintained in complete media consisting of RPMI-1640 with <span class="smallcaps smallerCapital">l</span>-glutamine, 10% fetal bovine serum (FBS; BioWest, Logan, UT), 100 U mL<sup>â1</sup> penicillin, and 100 Î¼g mL<sup>â1</sup> streptomycin in a humidified incubator at 37 Â°C and 5% CO<sub>2</sub>. The noncancerous human fetal lung cell line, HFL1,<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> was kindly provided by Dr. Robert Harrod (Southern Methodist University, Dallas, TX) and maintained in complete media consisting of F-12K with <span class="smallcaps smallerCapital">l</span>-glutamine, 10% FBS (BioWest, Logan, UT), 100 U mL<sup>â1</sup> penicillin, and 100 Î¼g mL<sup>â1</sup> streptomycin in a humidified incubator at 37 Â°C and 5% CO<sub>2</sub>. To promote attachment of HFL1 cells, growth surfaces were treated with 0.1 mg mL<sup>â1</sup> rat tail collagen (BD Biosciences, Palo Alto, CA) in 0.02 N acetic acid for 10 min and rinsed with PBS prior to use. Cell culture materials were purchased from Corning Inc. (Corning, NY) unless otherwise stated.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> MTT Cell Viability Assay</h3><div class="NLM_p">Cells were trypsinized from monolayers and seeded with 3000 cells in 150âÎ¼L of complete medium in a 96-well plate. After 24âh, cells were treated for 48âh with paclitaxel (Acros Organics, NJ) and/or P-gp inhibitory compounds dissolved in DMSO or DMSO controls. All additions were diluted into complete medium. After 48âh of treatment, MTT assays were performed as described<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> using 5âmg mL<sup>â1</sup> of MTT (Acros Organics, NJ) solution prepared in PBS (137âmM NaCl, 2.7âmM KCl, 10âmM Na<sub>2</sub>HPO<sub>4</sub>, 1.8âmM KH<sub>2</sub>PO<sub>4</sub>, pH 7.4). After 4âh of incubation with MTT, the media was removed and the formazan crystals were dissolved in 100âÎ¼L of DMSO. The absorbance at 570ânm was then measured using a BioTek Cytation 5 imaging multimode reader (Bio-Tek, Winooski, VT). Percent viability was calculated using DMSO-treated cells as representative for 100% viability. Background absorbance was determined using MTT and complete medium without cells and subtracted from all the test values.<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_m001.gif" alt="" id="_i35" /></img></span></div><div class="NLM_p last">The results were plotted as the mean with standard deviation (SD) of eight replicates per concentration from at least two independent experiments with <i>n</i> = 8. The graphical representations and IC<sub>50</sub> values were determined using four-parameter variable slope nonlinear regression, using the following equation: <i>Y</i> = bottom + (top â bottom)/(1 + 10^((log IC<sub>50</sub> â <i>X</i>)*hill slope) (GraphPad Prism, La Jolla California, U.S.A., version 6.05). The reported âfold sensitizationâ was calculated as follows.<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_m002.gif" alt="" id="_i36" /></img></span></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Calcein AM Assay</h3><div class="NLM_p last">To assess inhibition of P-gp-catalyzed transport of the P-gp pump substrate, calcein AM, DU145TXR cells were seeded in 96-well plates and allowed to grow in complete medium until confluency was reached. The medium was removed, and cells were treated without 2 Î¼M P-gp inhibitory compounds and 1 Î¼g mL<sup>â1</sup> calcein AM (Life Technologies, OR) and diluted into phenol red free RPMI 1640 media. To study the effect of preincubation of compounds, cells were treated with just P-gp inhibitory compounds and incubated at 37 Â°C for 6 h before adding calcein AM. Fluorescence excitation at 485 nm with a 20 nm gate and at emission at 535 nm with a 20 nm gate was measured using a BioTek Cytation 5 imaging multimode reader (Bio-Tek, Winooski, VT) over 60 min in 20 min intervals. DMSO was used as vehicle. Results were plotted as the mean with standard deviation (SD) of six replicates per concentration and are representative of at least two independent experiments.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Cellular Accumulation Assays for Experimental P-gp Inhibitors</h3><div class="NLM_p last">Cell use (DU145TXR), cell culturing, cell exposure to compounds, cellular handling, and extractions were performed as described in ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>. LCâMS/MS methods were performed as described in ref <a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> and as modified in ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> P-Glycoprotein Purification</h3><div class="NLM_p last">Cysteine-less MDR3 and the human normal MDR1 P-glycoprotein were recombinantly expressed in the yeast <i>P</i>. <i>pastoris</i> and purified essentially as described in refs <a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a><a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a>, and <a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and were used for assaying ATP hydrolysis and ATP binding to the protein as described in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> in the presence of the <b>29</b>-variants.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Nanodisc Assembly</h3><div class="NLM_p last">Human P-gp in mixed detergent micelles obtained during protein purification was reconstituted into nanodiscs similar to that in refs <a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> and <a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> with small modifications. P-gp was assembled with membrane scaffold protein MSP1E3D1 (Sigma-Aldrich) expressed in BL21 (DE3) and <span class="smallcaps smallerCapital">l</span>-alpha-phosphatidylcholine (Sigma-Aldrich) at a ratio of 1:10:500 (P-gp:MSP:PC) in 50 mM Tris-CL (pH 8). The detergent was removed with Bio-Beads SM-2 adsorbent media (BioRad). Ni-NTA agarose (Qiagen) chromatography was used to purify P-gp reconstituted nanodiscs using six bed volumes of the start buffer (20% (v/v) glycerol, 50 mM Tris-CL pH 7.5 at 4 Â°C, 50 mM NaCl) and five bed volumes of the elution buffer (20% (v/v) glycerol, 50 mM Tris-Cl pH 7.5 at 4 Â°C, 50 mM NaCl, 300 mM imidazole).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> ATPase Activity Assays</h3><div class="NLM_p last">ATP hydrolysis activity was measured using a coupled enzyme assay<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> as modified in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>. The specific basal activity of the mouse MDR3 cysteine-less P-gp was between 20 and 30 nmol min<sup>â1</sup> mg<sup>â1</sup>, and the transport substrate (verapamil)-stimulated activity was 200â400 nmol min<sup>â1</sup> mg<sup>â1</sup>. The specific basal activity of normal human MDR1 P-gp was between 123 and 193 nmol min<sup>â1</sup> mg<sup>â1</sup>, and the transport substrate (verapamil)-stimulated activity was between 193 and 263 nmol min<sup>â1</sup> mg<sup>â1</sup>.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> ESR Measurements</h3><div class="NLM_p last">ESR measurements were as described in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>. The amount of the protein-bound spin-labeled (SL) adenine nucleotide was determined as the difference between the known total concentration of SL-ATP (2â²,3â²-(2,2,5,5,-tetramethyl-3-pyrroline-1-oxyl-3-carboxylic acid ester)âATP<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a>) added and the free spin-labeled nucleotide (SL-ANP) observed in the experiment. Hyperbolic curve fitting of the results was performed using GraphPad Prism7 to determine maximum binding and apparent affinity for the spin-labeled nucleotide. The equation used for the fitting the curves was <i>y</i> = <i>P</i><sub>1</sub> Ã <i>x</i>/(<i>P</i><sub>2</sub> + <i>x</i>) where <i>P</i><sub>1</sub> corresponds to the maximum binding of SL-ANP (moles of SL-ANP bound per mole P-gp) and <i>P</i><sub>2</sub> equals the apparent dissociation constant for SL-ANP. To quantify the amount of free SL-ANP, standard curves were established where the signal amplitude of the high field signal of the ESR spectra of free SL-ANP in the absence of protein was correlated to the concentration of SL-ANP added. All ESR measurements were performed using a Bruker EMX 6/1 ESR spectrophotometer operating in the X-band mode and equipped with a high-sensitivity cavity. Spectra were acquired at a microwave frequency of 9.33 GHz, microwave power of 12.63 mW, 100 kHz modulation frequency, and a resolution of 1024 points. The centerfield of the scan was at 3325 G and an area of 100 G was scanned. The peak to peak modulation amplitude was 1 G, and the time constant was set to 10.240 ms. The conversion time was 163.84 ms, resulting in a total time sweep of 167.772 s. The signal gain was adjusted for the SL-ATP concentrations in the different experiments.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Data Availability</h3><div class="NLM_p last">The datasets generated during and/or analyzed during the current study are available from the corresponding authors on reasonable request.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00966" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32323" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32323" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b00966?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00966</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Description of ChemGen programs and workflow, figure of chemical spaces of variants produced, figure of binding interactions of variants with P-glycoprotein, chemical synthesis scheme, synthetic procedures, analytical data: HRMS and NMR, and NMR spectra and molecular strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">SMILES, inhibition of P-gp ATPase, and inhibition of ATP binding to P-gp of given molecules (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Human P-glycoprotein (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf">jm9b00966_si_001.pdf (6.22 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_002.csv">jm9b00966_si_002.csv (1.24 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_003.pdb">jm9b00966_si_003.pdb (711.07 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00966" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60027" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60027" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John G. Wise</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences,Â 
    The Center for Drug Discovery, Design and
Delivery,Â 
    The Center for Scientific Computation,Â 
    , , , and , Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0e6479677d6b4e7d637b206b6a7b"><span class="__cf_email__" data-cfemail="553f223c2630152638207b303120">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pia D. Vogel</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences,Â 
    The Center for Drug Discovery, Design and
Delivery,Â 
    The Center for Scientific Computation,Â 
    , , , and , Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8135-4649" title="Orcid link">http://orcid.org/0000-0002-8135-4649</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f9898f969e9c95b98a948cd79c9d8c"><span class="__cf_email__" data-cfemail="5121273e36343d11223c247f343524">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amila K. Nanayakkara</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences,Â 
    The Center for Drug Discovery, Design and
Delivery,Â 
    The Center for Scientific Computation,Â 
    , , , and , Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maha Aljowni</span> - <span class="hlFld-Affiliation affiliation">The Center for Drug Discovery, Design and
Delivery,Â 
    Department of Chemistry,Â 
    , , , and , Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gang Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences,Â 
    The Center for Drug Discovery, Design and
Delivery,Â 
    The Center for Scientific Computation,Â 
    , , , and , Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maisa C. De Oliveira</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences,Â 
    The Center for Drug Discovery, Design and
Delivery,Â 
    The Center for Scientific Computation,Â 
    , , , and , Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lauren Ammerman</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences,Â 
    The Center for Drug Discovery, Design and
Delivery,Â 
    The Center for Scientific Computation,Â 
    , , , and , Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ketetha Olengue</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences,Â 
    The Center for Drug Discovery, Design and
Delivery,Â 
    The Center for Scientific Computation,Â 
    , , , and , Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</span>;Â 
    <span>Present Address:
                        Present address: Keck School of Medicine, University of Southern California, Health Sciences Campus, 1975 Zonal Avenue, Los Angeles, California 90033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander R. Lippert</span> - <span class="hlFld-Affiliation affiliation">The Center for Drug Discovery, Design and
Delivery,Â 
    Department of Chemistry,Â 
    , , , and , Southern Methodist University, 6501 Airline Rd., Dallas, Texas 75205, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4396-0848" title="Orcid link">http://orcid.org/0000-0003-4396-0848</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.K.N., M.A., G.C., M.C.D.O., and L.A. contributed equally to this work. J.G.W. designed and coded the ChemGen program suite, performed ChemGen calculations, contributed to the designing of experiments, contributed to results interpretation, and co-wrote the manuscript. A.K.N. designed and performed the cell-based analyses. M.A. performed the chemical syntheses. G.C. performed the biochemical/biophysical analyses using MDR3 P-glycoprotein. M.C.D.O. performed biochemical assays using MDR1 P-glycoprotein. L.A. performed cheminformatic and molecular docking analyses. K.O. contributed to the coding of early versions of the ChemGen program. A.R.L. designed the chemical syntheses. P.D.V. contributed to experimental design, interpreted results, and was mostly responsible for writing the manuscript. All authors contributed to the writing of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): A.R.L. discloses a financial stake in BioLum Sciences, LLC.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40464" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40464" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The project described was supported by grant no. R15 GM094771-02 from the National Institute of General Medical Sciences (NIH/NIGMS) to J.G.W.. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institute of General Medical Sciences or the National Institutes of Health. The authors also wish to thank the SMU Hamilton Undergraduate Research Scholars Program, SMU Undergraduate Research Assistantships, SMU Undergraduate Engaged Learning Program, Communities Foundation of Texas, and Ms. Suzy Ruff of Dallas, Texas, and Ms. Myra Williams, Ph.D. of Naples, Florida, for private gifts. The authors thank Tristan Smyth for experimental assistance.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">P-gp</td><td class="NLM_def"><p class="first last">P-glycoprotein</p></td></tr><tr><td class="NLM_term">SMU29</td><td class="NLM_def"><p class="first last">2-[(5-cyclopropyl-1<i>H</i>-1,2,4-triazol-3-yl)sulfanyl]-<i>N</i>-[2-phenyl-5-(2,4,5-trimethylphenyl)pyrazol-3-yl]acetamide (ZINC 08767731, CID 17555821)</p></td></tr><tr><td class="NLM_term">NBD</td><td class="NLM_def"><p class="first last">the nucleotide-binding domains of P-gp</p></td></tr><tr><td class="NLM_term">DBD</td><td class="NLM_def"><p class="first last">the transmembrane domains of P-gp that bind to transport substrates</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethylsulfoxide</p></td></tr><tr><td class="NLM_term">ESR</td><td class="NLM_def"><p class="first last">electron spin resonance spectroscopy</p></td></tr><tr><td class="NLM_term">SL-ATP</td><td class="NLM_def"><p class="first last">2â²,3â²-(2,2,5,5,-tetramethyl-3-pyrroline-1-oxyl-3-carboxylic acid ester)</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">2â²,3â² indicates a rapid equilibrium of the ester bond between the 2â² and 3â² hydroxyl groups of ATP</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 68 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dos Santos, D. J. V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, M.-J. U.</span></span> <span> </span><span class="NLM_article-title">P-glycoprotein and membrane roles in multidrug resistance</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">929</span>â <span class="NLM_lpage">946</span>, <span class="refDoi">Â DOI: 10.4155/fmc.15.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.4155%2Ffmc.15.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26061109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpslGhu7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=929-946&author=R.+J.+Ferreiraauthor=D.+J.+V.+A.+dos+Santosauthor=M.-J.+U.+Ferreira&title=P-glycoprotein+and+membrane+roles+in+multidrug+resistance&doi=10.4155%2Ffmc.15.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">P-glycoprotein and membrane roles in multidrug resistance</span></div><div class="casAuthors">Ferreira, Ricardo J.; dos Santos, Daniel J. V. A.; Ferreira, Maria-Jose U.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">929-946</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Multidrug-resistance (MDR) phenomena are a worldwide health concern.  ATP-binding cassette efflux pumps as P-glycoprotein have been thoroughly studied in a frantic run to develop new efflux modulators capable to reverse MDR phenotypes.  The study of efflux pumps has provided some key aspects on drug extrusion, however the answers could not be found solely on ATP-binding cassette transporters.  Its counterpart - the plasma membrane - is now emerging as a crit. structure able to modify drug behavior and efflux pump activity.  Alterations in the membrane surrounding P-glycoprotein are now known to modulate drug efflux, with membrane-related biophys., biochem. and mech. aspects further increasing the complexity of an already multifaceted phenomena.  This review summarizes the main knowledge comprising the plasma membrane role in MDR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVK5Cuqc-j2rVg90H21EOLACvtfcHk0lg1NYrtgbYVaA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpslGhu7c%253D&md5=bdcd93a9c7cdcb16c6cef975f810145f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.36%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%26aufirst%3DR.%2BJ.%26aulast%3Ddos%2BSantos%26aufirst%3DD.%2BJ.%2BV.%2BA.%26aulast%3DFerreira%26aufirst%3DM.-J.%2BU.%26atitle%3DP-glycoprotein%2520and%2520membrane%2520roles%2520in%2520multidrug%2520resistance%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D929%26epage%3D946%26doi%3D10.4155%2Ffmc.15.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharom, F. J.</span></span> <span> </span><span class="NLM_article-title">ABC multidrug transporters: structure, function and role in chemoresistance</span>. <i>Pharmacogenomics</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">105</span>â <span class="NLM_lpage">127</span>, <span class="refDoi">Â DOI: 10.2217/14622416.9.1.105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.2217%2F14622416.9.1.105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=18154452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFOjsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=105-127&author=F.+J.+Sharom&title=ABC+multidrug+transporters%3A+structure%2C+function+and+role+in+chemoresistance&doi=10.2217%2F14622416.9.1.105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">ABC multidrug transporters: structure, function and role in chemoresistance</span></div><div class="casAuthors">Sharom, Frances J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacogenomics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">105-127</span>CODEN:
                <span class="NLM_cas:coden">PARMFL</span>;
        ISSN:<span class="NLM_cas:issn">1462-2416</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  Three ATP-binding cassette (ABC)-superfamily multidrug efflux pumps are known to be responsible for chemoresistance; P-glycoprotein (ABCB1), MRP1 (ABCC1) and ABCG2 (BCRP).  These transporters play an important role in normal physiol. by protecting tissues from toxic xenobiotics and endogenous metabolites.  Hydrophobic amphipathic compds., including many clin. used drugs, interact with the substrate-binding pocket of these proteins via flexible hydrophobic and H-bonding interactions.  These efflux pumps are expressed in many human tumors, where they likely contribute to resistance to chemotherapy treatment.  However, the use of efflux-pump modulators in clin. cancer treatment has proved disappointing.  Single nucleotide polymorphisms in ABC drug-efflux pumps may play a role in responses to drug therapy and disease susceptibility.  The effect of various genotypes and haplotypes on the expression and function of these proteins is not yet clear, and their true impact remains controversial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodtANKweyrPrVg90H21EOLACvtfcHk0liUhcqzlrF32Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFOjsg%253D%253D&md5=7594f33bc3a524c65d3622b00750dbe6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2217%2F14622416.9.1.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252F14622416.9.1.105%26sid%3Dliteratum%253Aachs%26aulast%3DSharom%26aufirst%3DF.%2BJ.%26atitle%3DABC%2520multidrug%2520transporters%253A%2520structure%252C%2520function%2520and%2520role%2520in%2520chemoresistance%26jtitle%3DPharmacogenomics%26date%3D2008%26volume%3D9%26spage%3D105%26epage%3D127%26doi%3D10.2217%2F14622416.9.1.105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jaramillo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Saig, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, G. J.</span></span> <span> </span><span class="NLM_article-title">How to overcome ATP-binding cassette drug efflux transporter-mediated drug resistance?</span>. <i>Cancer Drug Resist.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">6</span>â <span class="NLM_lpage">29</span>, <span class="refDoi">Â DOI: 10.20517/cdr.2018.02</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.20517%2Fcdr.2018.02" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=6-29&author=A.+C.+Jaramilloauthor=F.+Al+Saigauthor=J.+Cloosauthor=G.+Jansenauthor=G.+J.+Peters&title=How+to+overcome+ATP-binding+cassette+drug+efflux+transporter-mediated+drug+resistance%3F&doi=10.20517%2Fcdr.2018.02"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.20517%2Fcdr.2018.02&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.20517%252Fcdr.2018.02%26sid%3Dliteratum%253Aachs%26aulast%3DJaramillo%26aufirst%3DA.%2BC.%26aulast%3DAl%2BSaig%26aufirst%3DF.%26aulast%3DCloos%26aufirst%3DJ.%26aulast%3DJansen%26aufirst%3DG.%26aulast%3DPeters%26aufirst%3DG.%2BJ.%26atitle%3DHow%2520to%2520overcome%2520ATP-binding%2520cassette%2520drug%2520efflux%2520transporter-mediated%2520drug%2520resistance%253F%26jtitle%3DCancer%2520Drug%2520Resist.%26date%3D2018%26volume%3D1%26spage%3D6%26epage%3D29%26doi%3D10.20517%2Fcdr.2018.02" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unadkat, J. D.</span></span> <span> </span><span class="NLM_article-title">Role of the breast cancer resistance protein (BCRP/ABCG2) in drug transport--an update</span>. <i>AAPS J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">65</span>â <span class="NLM_lpage">82</span>, <span class="refDoi">Â DOI: 10.1208/s12248-014-9668-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1208%2Fs12248-014-9668-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=25236865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFygu7fM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=65-82&author=Q.+Maoauthor=J.+D.+Unadkat&title=Role+of+the+breast+cancer+resistance+protein+%28BCRP%2FABCG2%29+in+drug+transport%2D%2Dan+update&doi=10.1208%2Fs12248-014-9668-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the breast cancer resistance protein (BCRP/ABCG2) in drug transport-an update</span></div><div class="casAuthors">Mao, Qingcheng; Unadkat, Jashvant D.</div><div class="citationInfo"><span class="NLM_cas:title">AAPS Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-82</span>CODEN:
                <span class="NLM_cas:coden">AJAOB6</span>;
        ISSN:<span class="NLM_cas:issn">1550-7416</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The human breast cancer resistance protein (BCRP, gene symbol ABCG2) is an ATP-binding cassette (ABC) efflux transporter.  It was so named because it was initially cloned from a multidrug-resistant breast cancer cell line where it was found to confer resistance to chemotherapeutic agents such as mitoxantrone and topotecan.  Since its discovery in 1998, the substrates of BCRP have been rapidly expanding to include not only therapeutic agents but also physiol. substances such as estrone-3-sulfate, 17Î²-estradiol 17-(Î²-D-glucuronide) and uric acid.  Likewise, at least hundreds of BCRP inhibitors have been identified.  Among normal human tissues, BCRP is highly expressed on the apical membranes of the placental syncytiotrophoblasts, the intestinal epithelium, the liver hepatocytes, the endothelial cells of brain microvessels, and the renal proximal tubular cells, contributing to the absorption, distribution, and elimination of drugs and endogenous compds. as well as tissue protection against xenobiotic exposure.  As a result, BCRP has now been recognized by the FDA to be one of the key drug transporters involved in clin. relevant drug disposition.  We published a highly-accessed review article on BCRP in 2005, and much progress has been made since then.  In this review, we provide an update of current knowledge on basic biochem. and pharmacol. functions of BCRP as well as its relevance to drug resistance and drug disposition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojBFIoPBT7CbVg90H21EOLACvtfcHk0liUhcqzlrF32Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFygu7fM&md5=c4b8d6e1286fc6340cf4a8159f525c22</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1208%2Fs12248-014-9668-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-014-9668-6%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DQ.%26aulast%3DUnadkat%26aufirst%3DJ.%2BD.%26atitle%3DRole%2520of%2520the%2520breast%2520cancer%2520resistance%2520protein%2520%2528BCRP%252FABCG2%2529%2520in%2520drug%2520transport--an%2520update%26jtitle%3DAAPS%2520J.%26date%3D2015%26volume%3D17%26spage%3D65%26epage%3D82%26doi%3D10.1208%2Fs12248-014-9668-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span> <span> </span><span class="NLM_article-title">Cancer stem cells and chemoresistance: The smartest survives the raid</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">145</span>â <span class="NLM_lpage">158</span>, <span class="refDoi">Â DOI: 10.1016/j.pharmthera.2016.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.pharmthera.2016.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26899500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVChtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2016&pages=145-158&author=J.+Zhao&title=Cancer+stem+cells+and+chemoresistance%3A+The+smartest+survives+the+raid&doi=10.1016%2Fj.pharmthera.2016.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer stem cells and chemoresistance: The smartest survives the raid</span></div><div class="casAuthors">Zhao, Jihe</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">145-158</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Chemoresistant metastatic relapse of minimal residual disease plays a significant role for poor prognosis of cancer.  Growing evidence supports a crit. role of cancer stem cell (CSC) behind the mechanisms for this deadly disease.  This review briefly introduces the basics of the conventional chemotherapies, updates the CSC theories, highlights the mol. and cellular mechanisms by which CSC smartly designs and utilizes multiple lines of self-defense to avoid being killed by chemotherapy, and concisely summarizes recent progress in studies on CSC-targeted therapies in the end, with the hope to help guide future research toward developing more effective therapeutic strategies to eradicate tumor cells in the patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS9bQmJ1OaX7Vg90H21EOLACvtfcHk0lgFasKS7oNJ6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVChtrY%253D&md5=2e3676a52f325cdc7426c76d9eb4da81</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2016.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2016.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DJ.%26atitle%3DCancer%2520stem%2520cells%2520and%2520chemoresistance%253A%2520The%2520smartest%2520survives%2520the%2520raid%26jtitle%3DPharmacol.%2520Ther.%26date%3D2016%26volume%3D160%26spage%3D145%26epage%3D158%26doi%3D10.1016%2Fj.pharmthera.2016.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wijdeven, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assaraf, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neefjes, J.</span></span> <span> </span><span class="NLM_article-title">Old drugs, novel ways out: Drug resistance toward cytotoxic chemotherapeutics</span>. <i>Drug Resist Updates</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">65</span>â <span class="NLM_lpage">81</span>, <span class="refDoi">Â DOI: 10.1016/j.drup.2016.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.drup.2016.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=27620955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC2svgtVWqtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2016&pages=65-81&author=R.+H.+Wijdevenauthor=B.+Pangauthor=Y.+G.+Assarafauthor=J.+Neefjes&title=Old+drugs%2C+novel+ways+out%3A+Drug+resistance+toward+cytotoxic+chemotherapeutics&doi=10.1016%2Fj.drup.2016.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Old drugs, novel ways out: Drug resistance toward cytotoxic chemotherapeutics</span></div><div class="casAuthors">Wijdeven Ruud H; Pang Baoxu; Assaraf Yehuda G; Neefjes Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Drug resistance updates : reviews and commentaries in antimicrobial and anticancer chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">65-81</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Efficacy of chemotherapy in the treatment of distinct malignancies is often hampered by drug resistance arising in the tumor.  Understanding the molecular basis of drug resistance and translating this knowledge into personalized treatment decisions can enhance therapeutic efficacy and even curative outcome.  Over the years, multiple drug resistance mechanisms have been identified that enable tumors to cope with the damage instigated by a specific drug or group of anti-tumor agents.  Here we provide an overview of the molecular pathways leading to resistance against conventional anti-cancer drugs, with emphasis on the utility of these pathways for rational selection of treatments for individual cancer patients.  We further complement the review by discussing the pitfalls and difficulties in translating these findings into novel treatment strategies for cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQB6_n5sF8u3dts4TFFB11gfW6udTcc2eaK3DYMuTsnp7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2svgtVWqtg%253D%253D&md5=e2c1879c6086cfd6ab7140a5f80af6d3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2016.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2016.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DWijdeven%26aufirst%3DR.%2BH.%26aulast%3DPang%26aufirst%3DB.%26aulast%3DAssaraf%26aufirst%3DY.%2BG.%26aulast%3DNeefjes%26aufirst%3DJ.%26atitle%3DOld%2520drugs%252C%2520novel%2520ways%2520out%253A%2520Drug%2520resistance%2520toward%2520cytotoxic%2520chemotherapeutics%26jtitle%3DDrug%2520Resist%2520Updates%26date%3D2016%26volume%3D28%26spage%3D65%26epage%3D81%26doi%3D10.1016%2Fj.drup.2016.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fruci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nobili, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locatelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alisi, A.</span></span> <span> </span><span class="NLM_article-title">Drug transporters and multiple drug resistance in the most common pediatric solid tumors</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">308</span>â <span class="NLM_lpage">316</span>, <span class="refDoi">Â DOI: 10.2174/1567205010666131212110948</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.2174%2F1567205010666131212110948" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26694726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC28XktlClsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=308-316&author=D.+Fruciauthor=W.+C.+S.+Choauthor=V.+Nobiliauthor=F.+Locatelliauthor=A.+Alisi&title=Drug+transporters+and+multiple+drug+resistance+in+the+most+common+pediatric+solid+tumors&doi=10.2174%2F1567205010666131212110948"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Transporters and Multiple Drug Resistance in the Most Common Pediatric Solid Tumors</span></div><div class="casAuthors">Fruci, Doriana; Cho, William C. S.; Romania, Paolo; Nobili, Valerio; Locatelli, Franco; Alisi, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">308-316</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Solid tumors account for approx. 30% of all pediatric cancers.  Although chemotherapy has largely contributed to strongly improve patient outcome, multidrug resistance (MDR) remains one of the major mechanisms limiting the overall survival.  The enhanced efflux rate of chemotherapeutic drugs from tumor cells through drug transporters is one of the most important mechanisms of MDR.  Drug transporters play a pivotal role in preserving the balance between sensitivity and resistance of tumor cells to anti-neoplastic drugs.  Their functional activities have been barely investigated in pediatric solid malignancies.  Here, we provide evidence from the current literatures on drug transporters and MDR in the most common types of pediatric solid tumors, including neuroblastoma, Wilms' tumor, rhabdomyosarcoma, retinoblastoma, medulloblastoma and hepatoblastoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO9B7XwflfBLVg90H21EOLACvtfcHk0lgFasKS7oNJ6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktlClsLo%253D&md5=b7923a609527acc658e6c24c112ab858</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F1567205010666131212110948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1567205010666131212110948%26sid%3Dliteratum%253Aachs%26aulast%3DFruci%26aufirst%3DD.%26aulast%3DCho%26aufirst%3DW.%2BC.%2BS.%26aulast%3DNobili%26aufirst%3DV.%26aulast%3DLocatelli%26aufirst%3DF.%26aulast%3DAlisi%26aufirst%3DA.%26atitle%3DDrug%2520transporters%2520and%2520multiple%2520drug%2520resistance%2520in%2520the%2520most%2520common%2520pediatric%2520solid%2520tumors%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2016%26volume%3D17%26spage%3D308%26epage%3D316%26doi%3D10.2174%2F1567205010666131212110948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ierano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szakacs, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span> <span> </span><span class="NLM_article-title">The controversial role of ABC transporters in clinical oncology</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">209</span>â <span class="NLM_lpage">232</span>, <span class="refDoi">Â DOI: 10.1042/bse0500209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1042%2Fbse0500209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=21967059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlGrsb7O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=209-232&author=A.+Tamakiauthor=C.+Ieranoauthor=G.+Szakacsauthor=R.+W.+Robeyauthor=S.+E.+Bates&title=The+controversial+role+of+ABC+transporters+in+clinical+oncology&doi=10.1042%2Fbse0500209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The controversial role of ABC transporters in clinical oncology</span></div><div class="casAuthors">Tamaki, Akina; Ierano, Caterina; Szakacs, Gergely; Robey, Robert W.; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Essays in Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">209-232</span>CODEN:
                <span class="NLM_cas:coden">ESBIAV</span>;
        ISSN:<span class="NLM_cas:issn">0071-1365</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The phenomenon of multidrug resistance in cancer is often assocd. with the overexpression of the ABC (ATP-binding cassette) transporters Pgp (P-glycoprotein) (ABCB1), MRP1 (multidrug resistance-assocd. protein 1) (ABCC1) and ABCG2 [BCRP (breast cancer resistance protein)].  Since the discovery of Pgp over 35 years ago, studies have convincingly linked ABC transporter expression to poor outcome in several cancer types, leading to the development of transporter inhibitors.  Three generations of inhibitors later, we are still no closer to validating the 'Pgp hypothesis', the idea that increased chemotherapy efficacy can be achieved by inhibition of transporter-mediated efflux.  In this chapter, we highlight the difficulties and past failures encountered in the development of clin. inhibitors of ABC transporters.  We discuss the challenges that remain in our effort to exploit decades of work on ABC transporters in oncol.  In learning from past mistakes, it is hoped that ABC transporters can be developed as targets for clin. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTf-JsvF3tlLVg90H21EOLACvtfcHk0lgFasKS7oNJ6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlGrsb7O&md5=3c63559612797c2339f046019985def9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2Fbse0500209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbse0500209%26sid%3Dliteratum%253Aachs%26aulast%3DTamaki%26aufirst%3DA.%26aulast%3DIerano%26aufirst%3DC.%26aulast%3DSzakacs%26aufirst%3DG.%26aulast%3DRobey%26aufirst%3DR.%2BW.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DThe%2520controversial%2520role%2520of%2520ABC%2520transporters%2520in%2520clinical%2520oncology%26jtitle%3DEssays%2520Biochem.%26date%3D2011%26volume%3D50%26spage%3D209%26epage%3D232%26doi%3D10.1042%2Fbse0500209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palmeira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasconcelos, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, M. M.</span></span> <span> </span><span class="NLM_article-title">Three decades of P-gp inhibitors: skimming through several generations and scaffolds</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1946</span>â <span class="NLM_lpage">2025</span>, <span class="refDoi">Â DOI: 10.2174/092986712800167392</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.2174%2F092986712800167392" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=22257057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsF2htbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1946-2025&author=A.+Palmeiraauthor=E.+Sousaauthor=M.+H.+Vasconcelosauthor=M.+M.+Pinto&title=Three+decades+of+P-gp+inhibitors%3A+skimming+through+several+generations+and+scaffolds&doi=10.2174%2F092986712800167392"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Three decades of P-gp inhibitors: skimming through several generations and scaffolds</span></div><div class="casAuthors">Palmeira, A.; Sousa, E.; Vasconcelos, M. H.; Pinto, M. M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1946-2025</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Many tumor cells become resistant to commonly used cytotoxic drugs due to the overexpression of ATP-binding cassette (ABC) transporters, namely P-glycoprotein (P-gp).  The discovery of the reversal of multidrug resistance (MDR) by verapamil occurred in 1981, and in 1968 MDR Chinese hamster cell lines were isolated for the first time.  Since then, P-gp inhibitors have been intensively studied as potential MDR reversers.  Initially, drugs to reverse MDR were not specifically developed for inhibiting P-gp; in fact, they had other pharmacol. properties, as well as a relatively low affinity for MDR transporters.  An example of this first generation P-gp inhibitors is verapamil.  The second generation included more specific with less side-effect inhibitors, such as dexverapamil or dexniguldipine.  A third generation of P-gp inhibitors comprised compds. such as tariquidar, with high affinity to P-gp at nanomolar concns.  These generations of inhibitors of P-gp have been examd. in preclin. and clin. studies; however, these trials have largely failed to demonstrate an improvement in therapeutic efficacy.  Therefore, new and innovative strategies, such as the fallback to natural products, the design of peptidomimetics and dual activity ligands emerged as a fourth generation of P-gp inhibitors.  The chem. of P-gp inhibitors, as well as their in vitro, in vivo and clin. trials are discussed, and the most recent advances concerning P-gp modulators are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooLS8EJNaESbVg90H21EOLACvtfcHk0ljLJMs0Dldhfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsF2htbo%253D&md5=37ba7b1ce5f786a456edc7ed68f33f0b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F092986712800167392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712800167392%26sid%3Dliteratum%253Aachs%26aulast%3DPalmeira%26aufirst%3DA.%26aulast%3DSousa%26aufirst%3DE.%26aulast%3DVasconcelos%26aufirst%3DM.%2BH.%26aulast%3DPinto%26aufirst%3DM.%2BM.%26atitle%3DThree%2520decades%2520of%2520P-gp%2520inhibitors%253A%2520skimming%2520through%2520several%2520generations%2520and%2520scaffolds%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D1946%26epage%3D2025%26doi%3D10.2174%2F092986712800167392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Binkhathlan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavasanifar, A.</span></span> <span> </span><span class="NLM_article-title">P-glycoprotein inhibition as a therapeutic approach for overcoming multidrug resistance in cancer: current status and future perspectives</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">326</span>â <span class="NLM_lpage">346</span>, <span class="refDoi">Â DOI: 10.2174/15680096113139990076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.2174%2F15680096113139990076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=23369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVOjsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=326-346&author=Z.+Binkhathlanauthor=A.+Lavasanifar&title=P-glycoprotein+inhibition+as+a+therapeutic+approach+for+overcoming+multidrug+resistance+in+cancer%3A+current+status+and+future+perspectives&doi=10.2174%2F15680096113139990076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">P-glycoprotein inhibition as a therapeutic approach for overcoming multidrug resistance in cancer: current status and future perspectives</span></div><div class="casAuthors">Binkhathlan, Ziyad; Lavasanifar, Afsaneh</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">326-346</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  One of the major causes of failure in cancer chemotherapy is multidrug resistance (MDR), where cancer cells simultaneously become resistant to different anticancer drugs.  Over-expression of membrane efflux pumps like P-glycoprotein (P-gp) that recognizes different chemotherapeutic agents and transports them out of the cell, plays a major role in MDR.  The shortcoming of P-gp inhibitors in clinic was attributed to their non-specific action on P-gp and/or non-selective distribution to non-target organs that leads to intolerable side effects by the P-gp inhibitor at doses required for P-gp inhibition upon systemic administration.  Another major issue is the reduced elimination of P-gp substrates (e.g. anticancer drugs) and intolerable toxicities by anticancer drugs when co-administered with P-gp inhibitors.  To overcome these shortcomings, new generation of P-gp inhibitors with improved specificity for P-gp were developed.  More recently, attention was paid to the use of drug delivery systems primarily to restrict P-gp inhibition to tumor and reduce the non-selective inhibition of P-gp in non-target organs.  This review will provide an overview and update on the status of P-gp inhibition approaches and the role of drug delivery systems in overcoming P-gp mediated MDR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6kCA4RDN2s7Vg90H21EOLACvtfcHk0ljLJMs0Dldhfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVOjsbY%253D&md5=9fc1e50504313ae7735049fcaadcb9ff</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F15680096113139990076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F15680096113139990076%26sid%3Dliteratum%253Aachs%26aulast%3DBinkhathlan%26aufirst%3DZ.%26aulast%3DLavasanifar%26aufirst%3DA.%26atitle%3DP-glycoprotein%2520inhibition%2520as%2520a%2520therapeutic%2520approach%2520for%2520overcoming%2520multidrug%2520resistance%2520in%2520cancer%253A%2520current%2520status%2520and%2520future%2520perspectives%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2013%26volume%3D13%26spage%3D326%26epage%3D346%26doi%3D10.2174%2F15680096113139990076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kathawala, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashby, C. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.-S.</span></span> <span> </span><span class="NLM_article-title">The modulation of ABC transporter-mediated multidrug resistance in cancer: A review of the past decade</span>. <i>Drug Resist. Updates</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">17</span>, <span class="refDoi">Â DOI: 10.1016/j.drup.2014.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.drup.2014.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=25554624" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC2MvjtFGmtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1-17&author=R.+J.+Kathawalaauthor=P.+Guptaauthor=C.+R.+Ashbyauthor=Z.-S.+Chen&title=The+modulation+of+ABC+transporter-mediated+multidrug+resistance+in+cancer%3A+A+review+of+the+past+decade&doi=10.1016%2Fj.drup.2014.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The modulation of ABC transporter-mediated multidrug resistance in cancer: a review of the past decade</span></div><div class="casAuthors">Kathawala Rishil J; Gupta Pranav; Ashby Charles R Jr; Chen Zhe-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Drug resistance updates : reviews and commentaries in antimicrobial and anticancer chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ATP-binding cassette (ABC) transporters represent one of the largest and oldest families of membrane proteins in all extant phyla from prokaryotes to humans, which couple the energy derived from ATP hydrolysis essentially to translocate, among various substrates, toxic compounds across the membrane.  The fundamental functions of these multiple transporter proteins include: (1) conserved mechanisms related to nutrition and pathogenesis in bacteria, (2) spore formation in fungi, and (3) signal transduction, protein secretion and antigen presentation in eukaryotes.  Moreover, one of the major causes of multidrug resistance (MDR) and chemotherapeutic failure in cancer therapy is believed to be the ABC transporter-mediated active efflux of a multitude of structurally and mechanistically distinct cytotoxic compounds across membranes.  It has been postulated that ABC transporter inhibitors known as chemosensitizers may be used in combination with standard chemotherapeutic agents to enhance their therapeutic efficacy.  The current paper reviews the advance in the past decade in this important domain of cancer chemoresistance and summarizes the development of new compounds and the re-evaluation of compounds originally designed for other targets as transport inhibitors of ATP-dependent drug efflux pumps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRNzfrj3o9ghtHxxy173TfNfW6udTcc2eZv5T5hFiuQArntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvjtFGmtQ%253D%253D&md5=db6a95a46f45a829f02b1af34fc11cc4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2014.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2014.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DKathawala%26aufirst%3DR.%2BJ.%26aulast%3DGupta%26aufirst%3DP.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26aulast%3DChen%26aufirst%3DZ.-S.%26atitle%3DThe%2520modulation%2520of%2520ABC%2520transporter-mediated%2520multidrug%2520resistance%2520in%2520cancer%253A%2520A%2520review%2520of%2520the%2520past%2520decade%26jtitle%3DDrug%2520Resist.%2520Updates%26date%3D2015%26volume%3D18%26spage%3D1%26epage%3D17%26doi%3D10.1016%2Fj.drup.2014.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottesman, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastan, I. H.</span></span> <span> </span><span class="NLM_article-title">The Role of multidrug resistance efflux pumps in cancer: revisiting a JNCI publication exploring expression of the MDR1 (P-glycoprotein) gene</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">djv222</span>, <span class="refDoi">Â DOI: 10.1093/jnci/djv222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1093%2Fjnci%2Fdjv222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26286731" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2015&pages=djv222&author=M.+M.+Gottesmanauthor=I.+H.+Pastan&title=The+Role+of+multidrug+resistance+efflux+pumps+in+cancer%3A+revisiting+a+JNCI+publication+exploring+expression+of+the+MDR1+%28P-glycoprotein%29+gene&doi=10.1093%2Fjnci%2Fdjv222"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjv222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjv222%26sid%3Dliteratum%253Aachs%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26aulast%3DPastan%26aufirst%3DI.%2BH.%26atitle%3DThe%2520Role%2520of%2520multidrug%2520resistance%2520efflux%2520pumps%2520in%2520cancer%253A%2520revisiting%2520a%2520JNCI%2520publication%2520exploring%2520expression%2520of%2520the%2520MDR1%2520%2528P-glycoprotein%2529%2520gene%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2015%26volume%3D107%26spage%3Ddjv222%26doi%3D10.1093%2Fjnci%2Fdjv222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marin, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RodrÃ­guez-Macias, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DÃ­ez-MartÃ­n, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macias, R. I. R.</span></span> <span> </span><span class="NLM_article-title">Role of drug transport and metabolism in the chemoresistance of acute myeloid leukemia</span>. <i>Blood Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">55</span>â <span class="NLM_lpage">64</span>, <span class="refDoi">Â DOI: 10.1016/j.blre.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.blre.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26321049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOmt7bL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=55-64&author=J.+J.+G.+Marinauthor=O.+Brizauthor=G.+Rodr%C3%ADguez-Maciasauthor=J.+L.+D%C3%ADez-Mart%C3%ADnauthor=R.+I.+R.+Macias&title=Role+of+drug+transport+and+metabolism+in+the+chemoresistance+of+acute+myeloid+leukemia&doi=10.1016%2Fj.blre.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Role of drug transport and metabolism in the chemoresistance of acute myeloid leukemia</span></div><div class="casAuthors">Marin, Jose J. G.; Briz, Oscar; Rodriguez-Macias, Gabriela; Diez-Martin, Jose L.; Macias, Rocio I. R.</div><div class="citationInfo"><span class="NLM_cas:title">Blood Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-64</span>CODEN:
                <span class="NLM_cas:coden">BLOREB</span>;
        ISSN:<span class="NLM_cas:issn">0268-960X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Acute myeloid leukemia is a clonal but heterogeneous disease differing in mol. pathogenesis, clin. features and response to chemotherapy.  This latter frequently consists of a combination of cytarabine and anthracyclines, although etoposide, demethylating agents, and other drugs are also used.  Unfortunately, chemoresistance is a common and serious problem.  Multiple mechanisms account for impaired effectiveness of drugs and reduced levels of active agents in target cells.  The latter can be due to lower drug uptake, increased export or decreased intracellular proportion of active/inactive agent due to changes in the expression/function of enzymes responsible for the activation of pro-drugs and the inactivation of active agents.  Characterization of the "resistome", or profile of expressed genes accounting for multi-drug resistance (MDR) phenotype, would permit to predict the lack of response to chemotherapy and would help in the selection of the best pharmacol. regime for each patient and moment, and to develop strategies of chemosensitization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfZ9jGbdMhrrVg90H21EOLACvtfcHk0ljF4raXa6PGhg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOmt7bL&md5=945e0f62a3800f906f24421dfea5eca6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.blre.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.blre.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DMarin%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DBriz%26aufirst%3DO.%26aulast%3DRodr%25C3%25ADguez-Macias%26aufirst%3DG.%26aulast%3DD%25C3%25ADez-Mart%25C3%25ADn%26aufirst%3DJ.%2BL.%26aulast%3DMacias%26aufirst%3DR.%2BI.%2BR.%26atitle%3DRole%2520of%2520drug%2520transport%2520and%2520metabolism%2520in%2520the%2520chemoresistance%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DBlood%2520Rev.%26date%3D2016%26volume%3D30%26spage%3D55%26epage%3D64%26doi%3D10.1016%2Fj.blre.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">List, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopecky, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willman, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persons, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slovak, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karanes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shurafa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appelbaum, F. R.</span></span> <span> </span><span class="NLM_article-title">Benefit of cyclosporine modulation of drug resistance in patients with poor-risk acute myeloid leukemia: a Southwest Oncology Group study</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">3212</span>â <span class="NLM_lpage">3220</span>, <span class="refDoi">Â DOI: 10.1182/blood.V98.12.3212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1182%2Fblood.V98.12.3212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=11719356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovVSgsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=3212-3220&author=A.+F.+Listauthor=K.+J.+Kopeckyauthor=C.+L.+Willmanauthor=D.+R.+Headauthor=D.+L.+Personsauthor=M.+L.+Slovakauthor=R.+Dorrauthor=C.+Karanesauthor=H.+E.+Hynesauthor=J.+H.+Doroshowauthor=M.+Shurafaauthor=F.+R.+Appelbaum&title=Benefit+of+cyclosporine+modulation+of+drug+resistance+in+patients+with+poor-risk+acute+myeloid+leukemia%3A+a+Southwest+Oncology+Group+study&doi=10.1182%2Fblood.V98.12.3212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Benefit of cyclosporine modulation of drug resistance in patients with poor-risk acute myeloid leukemia: a Southwest Oncology Group study</span></div><div class="casAuthors">List, Alan F.; Kopecky, Kenneth J.; Willman, Cheryl L.; Head, David R.; Persons, Diane L.; Slovak, Marilyn L.; Dorr, Robert; Karanes, Chatchada; Hynes, Harry E.; Doroshow, James H.; Shurafa, Muhammad; Appelbaum, Frederick R.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3212-3220</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Cyclosporine A (CsA) inhibits P-glycoprotein (Pgp)-mediated cellular export of anthracyclines at clin. achievable concns.  This randomized controlled trial was performed to test the benefit of CsA addn. to treatment with cytarabine and daunorubicin (DNR) in patients with poor-risk acute myeloid leukemia (AML).  A total of 226 patients were randomly assigned to sequential treatment with cytarabine and Infusional DNR with or without i.v. CsA.  Remitting patients received one course of consolidation chemotherapy that included DNR with or without CsA as assigned during induction.  Addn. of CsA significantly reduced the frequency of resistance to induction chemotherapy (31% vs. 47%, P = .0077).  Whereas the rate of complete remission was not significantly improved (39% vs. 33%, P = 14), relapse-free survival (34% vs. 9% at 2 yr, P = .031) and overall survival (22% vs. 12%, P = .046) were significantly increased with CsA.  The effect of CsA on survival was greatest in patients with moderate or bright Pgp expression (median 12 mo with CsA vs. 4 mo for controls) compared to patients with absent or low Pgp expression (median 6 mo in both arms).  The frequency of Induction deaths was 15% with CsA and 18% in controls.  Steady-state serum concns. of DNR (P = .0089) and daunorubicinol (P < .0001) were significantly higher in CsA-treated patients.  Survival (P = .0003) and induction response (P = .028) improved with increasing DNR concn. in CsA-treated patients but not in controls, suggesting a targeted Interaction by CsA to enhance anthracycline cytotoxicity.  These results indicate that addn. of CsA to an induction and consolidation regimen contg. infusional DNR significantly reduces resistance to DNR, prolongs the duration of remission, and improves overall survival in patients with poor-risk AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8IxQr3SlAlrVg90H21EOLACvtfcHk0ljF4raXa6PGhg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovVSgsrc%253D&md5=6d53b5b27e1a51791db770858ac40047</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1182%2Fblood.V98.12.3212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V98.12.3212%26sid%3Dliteratum%253Aachs%26aulast%3DList%26aufirst%3DA.%2BF.%26aulast%3DKopecky%26aufirst%3DK.%2BJ.%26aulast%3DWillman%26aufirst%3DC.%2BL.%26aulast%3DHead%26aufirst%3DD.%2BR.%26aulast%3DPersons%26aufirst%3DD.%2BL.%26aulast%3DSlovak%26aufirst%3DM.%2BL.%26aulast%3DDorr%26aufirst%3DR.%26aulast%3DKaranes%26aufirst%3DC.%26aulast%3DHynes%26aufirst%3DH.%2BE.%26aulast%3DDoroshow%26aufirst%3DJ.%2BH.%26aulast%3DShurafa%26aufirst%3DM.%26aulast%3DAppelbaum%26aufirst%3DF.%2BR.%26atitle%3DBenefit%2520of%2520cyclosporine%2520modulation%2520of%2520drug%2520resistance%2520in%2520patients%2520with%2520poor-risk%2520acute%2520myeloid%2520leukemia%253A%2520a%2520Southwest%2520Oncology%2520Group%2520study%26jtitle%3DBlood%26date%3D2001%26volume%3D98%26spage%3D3212%26epage%3D3220%26doi%3D10.1182%2Fblood.V98.12.3212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaimes, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aller, S. G.</span></span> <span> </span><span class="NLM_article-title">Refined structures of mouse P-glycoprotein</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">34</span>â <span class="NLM_lpage">46</span>, <span class="refDoi">Â DOI: 10.1002/pro.2387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fpro.2387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=24155053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFGktLfN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2014&pages=34-46&author=J.+Liauthor=K.+F.+Jaimesauthor=S.+G.+Aller&title=Refined+structures+of+mouse+P-glycoprotein&doi=10.1002%2Fpro.2387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Refined structures of mouse P-glycoprotein</span></div><div class="casAuthors">Li, Jingzhi; Jaimes, Kimberly F.; Aller, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-46</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The recently detd. C. elegans P-glycoprotein (Pgp) structure revealed significant deviations compared to the original mouse Pgp structure, which suggested possible misinterpretations in the latter model.  To address this concern, we generated an exptl. electron d. map from single-wavelength anomalous dispersion phasing of an original mouse Pgp dataset to 3.8 Ã resoln.  The map exhibited significantly more detail compared to the original MAD map and revealed several regions of the structure that required de novo model building.  The improved drug-free structure was refined to 3.8 Ã resoln. with a 9.4 and 8.1% decrease in Rwork and Rfree, resp., (Rwork = 21.2%, Rfree = 26.6%) and a significant improvement in protein geometry.  The improved mouse Pgp model contains â¼95% of residues in the favorable Ramachandran region compared to only 57% for the original model.  The registry of six transmembrane helixes was cor., revealing amino acid residues involved in drug binding that were previously unrecognized.  Registry shifts (rotations and translations) for three transmembrane (TM)4 and TM5 and the addn. of three N-terminal residues were necessary, and were validated with new mercury labeling and anomalous Fourier d.  The cor. position of TM4, which forms the frame of a portal for drug entry, had backbone atoms shifted >6 Ã from their original positions.  The drug translocation pathway of mouse Pgp is 96% identical to human Pgp and is enriched in arom. residues that likely play a collective role in allowing a high degree of polyspecific substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreAlq0kWSXj7Vg90H21EOLACvtfcHk0lhj18HPAlDSHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFGktLfN&md5=7c6fd47f821dca3d47cf78c7d2e2020f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fpro.2387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2387%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJaimes%26aufirst%3DK.%2BF.%26aulast%3DAller%26aufirst%3DS.%2BG.%26atitle%3DRefined%2520structures%2520of%2520mouse%2520P-glycoprotein%26jtitle%3DProtein%2520Sci.%26date%3D2014%26volume%3D23%26spage%3D34%26epage%3D46%26doi%3D10.1002%2Fpro.2387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Molecular structure of human P-glycoprotein in the ATP-bound, outward-facing conformation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">915</span>, <span class="refDoi">Â DOI: 10.1126/science.aar7389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1126%2Fscience.aar7389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=29371429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVKgs7vN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=915&author=Y.+Kimauthor=J.+Chen&title=Molecular+structure+of+human+P-glycoprotein+in+the+ATP-bound%2C+outward-facing+conformation&doi=10.1126%2Fscience.aar7389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Co-infection of Ureaplasma urealyticum and human papilloma virus in asymptomatic sexually active individuals</span></div><div class="casAuthors">Kim, Sang Il; Yoon, Joo Hee; Park, Dong Choon; Lee, Dong Sup; Lee, Seung Ju; Choe, Hyun Sop; Hwikim, Jin; Park, Tae Chul; Lee, Sung Jong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Medical Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">915-920</span>CODEN:
                <span class="NLM_cas:coden">IJMSGZ</span>;
        ISSN:<span class="NLM_cas:issn">1449-1907</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">This study aimed to det. the role of asymptomatic bacterial sexually transmitted infections (STIs), such as Chlamydia trachomatis (Ct), Mycoplasma genitalium (Mg), Mycoplasma hominis (Mh), and Ureaplasma urealyticum (Uu) in human papillomavirus (HPV) infection.  In total, 264 asymptomatic outpatients aged between 21 and 80 years were prospectively enrolled in this study during routine gynecol. screening tests.  Specimens collected with a Cervex Brush were routinely analyzed with the Hybrid Capture 2 assay for HPV.  Simultaneously, a specimen obtained with an endocervical swab was used to detect Ct and Mg with a monoplex real-time polymerase chain reaction (PCR) and to confirm Mh and Uu with a Mycoplasma IST 2 kit.  The detection rates (%) of HPV, Ct, Mg, Mh, and Uu were 82/264 (31.1), 6/264 (2.3), 5/264 (1.9), 16/264 (6.1), and 95/264 (36.0), resp.  Of 95 Uu, 32 (33.7%) showed high d. colonization (HDC, â¥104 color-changing units/mL).  HDC-Uu was significantly assocd. with HPV infection (p=0.014, chi-square test).  Mg infection and Mh infection were not assocd. with HPV infection (p=0.981 and p=0.931, chi-square test).  Age was not assocd. with HPV infection or bacterial infection.  Our data suggested that asymptomatic HDC-Uu was closely assocd. with HPV infection.  Therefore, simultaneous evaluation for Uu and HPV should be performed during gynecol. screening, even in asymptomatic individuals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrADIFePxanKrVg90H21EOLACvtfcHk0lhj18HPAlDSHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVKgs7vN&md5=f7093af4dc290e028e859999e25b743f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.aar7389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aar7389%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DMolecular%2520structure%2520of%2520human%2520P-glycoprotein%2520in%2520the%2520ATP-bound%252C%2520outward-facing%2520conformation%26jtitle%3DScience%26date%3D2018%26spage%3D915%26doi%3D10.1126%2Fscience.aar7389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aller, S. G.</span></span> <span> </span><span class="NLM_article-title">Equilibrated atomic models of outward-facing P-glycoprotein and effect of ATP binding on structural dynamics</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">7880</span>, <span class="refDoi">Â DOI: 10.1038/srep07880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fsrep07880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=25600711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVCltrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=7880&author=L.+Panauthor=S.+G.+Aller&title=Equilibrated+atomic+models+of+outward-facing+P-glycoprotein+and+effect+of+ATP+binding+on+structural+dynamics&doi=10.1038%2Fsrep07880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Equilibrated Atomic Models of Outward-Facing P-glycoprotein and Effect of ATP Binding on Structural Dynamics</span></div><div class="casAuthors">Pan, Lurong; Aller, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7880</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) is an ATP-binding cassette (ABC) transporter that alternates between inward- and outward-facing conformations to capture and force substrates out of cells like a peristaltic pump.  The high degree of similarity in outward-facing structures across evolution of ABC transporters allowed construction of a high-confidence outward-facing Pgp at. model based on crystal structures of outward-facing Sav1866 and inward-facing Pgp.  The model adhered to previous exptl. detd. secondary- and tertiary- configurations during all-atom mol. dynamics simulations in the presence or absence of MgATP.  Three long lasting (>100 ns) meta-stable states were apparent in the presence of MgATP revealing new insights into alternating access.  The two ATP-binding pockets are highly asym. resulting in differential control of overall structural dynamics and allosteric regulation of the drug-binding pocket.  Equilibrated Pgp has a considerably different electrostatic profile compared to Sav1866 that implicates significant kinetic and thermodn. differences in transport mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonLxb3soLs7LVg90H21EOLACvtfcHk0lhj18HPAlDSHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVCltrY%253D&md5=197b7b93e90191f9347156277530b2a0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fsrep07880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep07880%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DL.%26aulast%3DAller%26aufirst%3DS.%2BG.%26atitle%3DEquilibrated%2520atomic%2520models%2520of%2520outward-facing%2520P-glycoprotein%2520and%2520effect%2520of%2520ATP%2520binding%2520on%2520structural%2520dynamics%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D7880%26doi%3D10.1038%2Fsrep07880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the multidrug transporter P-glycoprotein from Caenorhabditis elegans</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>490</i></span>,  <span class="NLM_fpage">566</span>â <span class="NLM_lpage">569</span>, <span class="refDoi">Â DOI: 10.1038/nature11448</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fnature11448" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=23000902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtlymu7%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=566-569&author=M.+S.+Jinauthor=M.+L.+Oldhamauthor=Q.+Zhangauthor=J.+Chen&title=Crystal+structure+of+the+multidrug+transporter+P-glycoprotein+from+Caenorhabditis+elegans&doi=10.1038%2Fnature11448"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the multidrug transporter P-glycoprotein from Caenorhabditis elegans</span></div><div class="casAuthors">Jin, Mi Sun; Oldham, Michael L.; Zhang, Qiuju; Chen, Jue</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">490</span>
        (<span class="NLM_cas:issue">7421</span>),
    <span class="NLM_cas:pages">566-569</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P-glycoprotein (P-gp) is an ATP-binding cassette transporter that confers multidrug resistance in cancer cells.  It also affects the absorption, distribution and clearance of cancer-unrelated drugs and xenobiotics.  For these reasons, the structure and function of P-gp have been studied extensively for decades.  Here we present biochem. characterization of P-gp from Caenorhabditis elegans and its crystal structure at a resoln. of 3.4 angstroems.  We find that the apparent affinities of P-gp for anticancer drugs actinomycin D and paclitaxel are approx. 4,000 and 100 times higher, resp., in the membrane bilayer than in detergent.  This affinity enhancement highlights the importance of membrane partitioning when a drug accesses the transporter in the membrane.  Furthermore, the transporter in the crystal structure opens its drug pathway at the level of the membrane's inner leaflet.  In the helixes flanking the opening to the membrane, we observe extended loops that may mediate drug binding, function as hinges to gate the pathway or both.  We also find that the interface between the transmembrane and nucleotide-binding domains, which couples ATP hydrolysis to transport, contains a ball-and-socket joint and salt bridges similar to the ATP-binding cassette importers, suggesting that ATP-binding cassette exporters and importers may use similar mechanisms to achieve alternating access for transport.  Finally, a model of human P-gp derived from the structure of C. elegans P-gp not only is compatible with decades of biochem. anal., but also helps to explain perplexing functional data regarding the Phe335Ala mutant.  These results increase our understanding of the structure and function of this important mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdu-UyIUtRP7Vg90H21EOLACvtfcHk0li26WTjSODo1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtlymu7%252FN&md5=73d67b98d098c05a3975ed6f371027cb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature11448&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11448%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DOldham%26aufirst%3DM.%2BL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520multidrug%2520transporter%2520P-glycoprotein%2520from%2520Caenorhabditis%2520elegans%26jtitle%3DNature%26date%3D2012%26volume%3D490%26spage%3D566%26epage%3D569%26doi%3D10.1038%2Fnature11448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, N. M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manolaridis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahlberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locher, K. P.</span></span> <span> </span><span class="NLM_article-title">Structure of the human multidrug transporter ABCG2</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span>,  <span class="NLM_fpage">504</span>â <span class="NLM_lpage">509</span>, <span class="refDoi">Â DOI: 10.1038/nature22345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fnature22345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=28554189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosFWntbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=546&publication_year=2017&pages=504-509&author=N.+M.+I.+Taylorauthor=I.+Manolaridisauthor=S.+M.+Jacksonauthor=J.+Kowalauthor=H.+Stahlbergauthor=K.+P.+Locher&title=Structure+of+the+human+multidrug+transporter+ABCG2&doi=10.1038%2Fnature22345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the human multidrug transporter ABCG2</span></div><div class="casAuthors">Taylor, Nicholas M. I.; Manolaridis, Ioannis; Jackson, Scott M.; Kowal, Julia; Stahlberg, Henning; Locher, Kaspar P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7659</span>),
    <span class="NLM_cas:pages">504-509</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">ABCG2 is a constitutively expressed ATP-binding cassette (ABC) transporter that protects many tissues against xenobiotic mols.  Its activity affects the pharmacokinetics of commonly used drugs and limits the delivery of therapeutics into tumor cells, thus contributing to multidrug resistance.  Here we present the structure of human ABCG2 detd. by cryo-electron microscopy, providing the first high-resoln. insight into a human multidrug transporter.  We visualize ABCG2 in complex with two antigen-binding fragments of the human-specific, inhibitory antibody 5D3 that recognizes extracellular loops of the transporter.  We observe two cholesterol mols. bound in the multidrug-binding pocket that is located in a central, hydrophobic, inward-facing translocation pathway between the transmembrane domains.  Combined with functional in vitro analyses, our results suggest a multidrug recognition and transport mechanism of ABCG2, rationalize disease-causing single nucleotide polymorphisms (SNP) and the allosteric inhibition by the 5D3 antibody, and provide the structural basis of cholesterol recognition by other G-subfamily ABC transporters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYOlfrPNtVhrVg90H21EOLACvtfcHk0li26WTjSODo1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosFWntbk%253D&md5=a3fd5157f639d2958e5abd996e2a6470</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature22345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22345%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DN.%2BM.%2BI.%26aulast%3DManolaridis%26aufirst%3DI.%26aulast%3DJackson%26aufirst%3DS.%2BM.%26aulast%3DKowal%26aufirst%3DJ.%26aulast%3DStahlberg%26aufirst%3DH.%26aulast%3DLocher%26aufirst%3DK.%2BP.%26atitle%3DStructure%2520of%2520the%2520human%2520multidrug%2520transporter%2520ABCG2%26jtitle%3DNature%26date%3D2017%26volume%3D546%26spage%3D504%26epage%3D509%26doi%3D10.1038%2Fnature22345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis of substrate recognition by the multidrug resistance protein MRP1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">1075</span>â <span class="NLM_lpage">1085.e9</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2017.01.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.cell.2017.01.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=28238471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlGrurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=1075-1085.e9&author=Z.+L.+Johnsonauthor=J.+Chen&title=Structural+basis+of+substrate+recognition+by+the+multidrug+resistance+protein+MRP1&doi=10.1016%2Fj.cell.2017.01.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Recognition by the Multidrug Resistance Protein MRP1</span></div><div class="casAuthors">Johnson, Zachary Lee; Chen, Jue</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1075-1085.e9</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The multidrug resistance protein MRP1 is an ATP-binding cassette (ABC) transporter that confers resistance to many anticancer drugs and plays a role in the disposition and efficacy of several opiates, antidepressants, statins, and antibiotics.  In addn., MRP1 regulates redox homeostasis, inflammation, and hormone secretion.  Using electron cryomicroscopy, we detd. the mol. structures of bovine MRP1 in two conformations: an apo form at 3.5 Ã without any added substrate and a complex form at 3.3 Ã with one of its physiol. substrates, leukotriene C4.  These structures show that by forming a single bipartite binding site, MRP1 can recognize a spectrum of substrates with different chem. structures.  We also obsd. large conformational changes induced by leukotriene C4, explaining how substrate binding primes the transporter for ATP hydrolysis.  Structural comparison of MRP1 and P-glycoprotein advances our understanding of the common and unique properties of these two important mols. in multidrug resistance to chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo87MmRBg2HsbVg90H21EOLACvtfcHk0li26WTjSODo1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlGrurg%253D&md5=1799bd0d0180f8c524ebe97271a08c85</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.01.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.01.041%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DZ.%2BL.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520recognition%2520by%2520the%2520multidrug%2520resistance%2520protein%2520MRP1%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D1075%26epage%3D1085.e9%26doi%3D10.1016%2Fj.cell.2017.01.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span> <span> </span><span class="NLM_article-title">Catalytic transitions in the human MDR1 P-glycoprotein drug binding sites</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5125</span>â <span class="NLM_lpage">5141</span>, <span class="refDoi">Â DOI: 10.1021/bi300299z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi300299z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnslaht7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5125-5141&author=J.+G.+Wise&title=Catalytic+transitions+in+the+human+MDR1+P-glycoprotein+drug+binding+sites&doi=10.1021%2Fbi300299z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic Transitions in the Human MDR1 P-Glycoprotein Drug Binding Sites</span></div><div class="casAuthors">Wise, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5125-5141</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multidrug resistance proteins that belong to the ATP-binding cassette family like the human P-glycoprotein (ABCB1 or Pgp) are responsible for many failed cancer and antiviral chemotherapies because these membrane transporters remove the chemotherapeutics from the targeted cells.  Understanding the details of the catalytic mechanism of Pgp is therefore crit. to the development of inhibitors that might overcome these resistances.  In this work, targeted mol. dynamics techniques were used to elucidate catalytically relevant structures of Pgp.  Crystal structures of homologs in four different conformations were used as intermediate targets in the dynamics simulations.  Transitions from conformations that were wide open to the cytoplasm to transition state conformations that were wide open to the extracellular space were studied.  Twenty-six nonredundant transitional protein structures were identified from these targeted mol. dynamics simulations using evolutionary structure analyses.  Coupled movement of nucleotide binding domains (NBDs) and transmembrane domains (TMDs) that form the drug binding cavities were obsd.  Pronounced twisting of the NBDs as they approached each other as well as the quantification of a dramatic opening of the TMDs to the extracellular space as the ATP hydrolysis transition state was reached were obsd.  Docking interactions of 21 known transport ligands or inhibitors were analyzed with each of the 26 transitional structures.  Many of the docking results obtained here were validated by previously published biochem. detns.  As the ATP hydrolysis transition state was approached, drug docking in the extracellular half of the transmembrane domains seemed to be destabilized as transport ligand exit gates opened to the extracellular space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOOus-mymdBLVg90H21EOLACvtfcHk0lj3RKoOTw82Ig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnslaht7k%253D&md5=0b1f63d47f37a3ea618d281c87650bc8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fbi300299z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi300299z%26sid%3Dliteratum%253Aachs%26aulast%3DWise%26aufirst%3DJ.%2BG.%26atitle%3DCatalytic%2520transitions%2520in%2520the%2520human%2520MDR1%2520P-glycoprotein%2520drug%2520binding%2520sites%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5125%26epage%3D5141%26doi%3D10.1021%2Fbi300299z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span> <span> </span><span class="NLM_article-title">Multiple drug transport pathways through human P-glycoprotein</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4374</span>â <span class="NLM_lpage">4390</span>, <span class="refDoi">Â DOI: 10.1021/acs.biochem.5b00018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b00018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=4374-4390&author=J.+W.+McCormickauthor=P.+D.+Vogelauthor=J.+G.+Wise&title=Multiple+drug+transport+pathways+through+human+P-glycoprotein&doi=10.1021%2Facs.biochem.5b00018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Drug Transport Pathways through Human P-Glycoprotein</span></div><div class="casAuthors">McCormick, James W.; Vogel, Pia D.; Wise, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">4374-4390</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-Glycoprotein (P-gp) is a plasma membrane efflux pump that is commonly assocd. with therapy resistances in cancers and infectious diseases.  P-gp can lower the intracellular concns. of many drugs to subtherapeutic levels by translocating them out of the cell.  Because of the broad range of substrates transported by P-gp, overexpression of P-gp causes multidrug resistance.  We reported previously on dynamic transitions of P-gp as it moved through conformations based on crystal structures of homologous ABCB1 proteins using in silico targeted mol. dynamics techniques.  We expanded these studies here by docking transport substrates to drug binding sites of P-gp in conformations open to the cytoplasm, followed by cycling the pump through conformations that opened to the extracellular space.  We obsd. reproducible transport of two substrates, daunorubicin and verapamil, by an av. of 11-12 Ã through the plane of the membrane as P-gp progressed through a catalytic cycle.  Methylpyrophosphate, a ligand that should not be transported by P-gp, did not show this movement through P-gp.  Drug binding to either of two subsites on P-gp appeared to det. the initial pathway used for drug movement through the membrane.  The specific side-chain interactions with drugs within each pathway seemed to be, at least in part, stochastic.  The docking and transport properties of a P-gp inhibitor, tariquidar, were also studied.  A mechanism of inhibition by tariquidar that involves stabilization of an outward open conformation with tariquidar bound in intracellular loops or at the drug binding domain of P-gp is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA2Lu18uESV7Vg90H21EOLACvtfcHk0lj3RKoOTw82Ig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLrI&md5=af16b2b12607eb23bc9b88555f26a5a1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b00018%26sid%3Dliteratum%253Aachs%26aulast%3DMcCormick%26aufirst%3DJ.%2BW.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26aulast%3DWise%26aufirst%3DJ.%2BG.%26atitle%3DMultiple%2520drug%2520transport%2520pathways%2520through%2520human%2520P-glycoprotein%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D4374%26epage%3D4390%26doi%3D10.1021%2Facs.biochem.5b00018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frank, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgnia, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earl, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottesman, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambudkar, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span> <span> </span><span class="NLM_article-title">Cryo-EM analysis of the conformational landscape of human P-glycoprotein (ABCB1) during its catalytic cycle</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">35</span>â <span class="NLM_lpage">41</span>, <span class="refDoi">Â DOI: 10.1124/mol.116.104190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1124%2Fmol.116.104190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=27190212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVejs7rO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2016&pages=35-41&author=G.+A.+Frankauthor=S.+Shuklaauthor=P.+Raoauthor=M.+J.+Borgniaauthor=A.+Bartesaghiauthor=A.+Merkauthor=A.+Mobinauthor=L.+Esserauthor=L.+A.+Earlauthor=M.+M.+Gottesmanauthor=D.+Xiaauthor=S.+V.+Ambudkarauthor=S.+Subramaniam&title=Cryo-EM+analysis+of+the+conformational+landscape+of+human+P-glycoprotein+%28ABCB1%29+during+its+catalytic+cycle&doi=10.1124%2Fmol.116.104190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM analysis of the conformational landscape of human P-glycoprotein (ABCB1) during its catalytic cycle</span></div><div class="casAuthors">Frank, Gabriel A.; Shukla, Suneet; Rao, Prashant; Borgnia, Mario J.; Bartesaghi, Alberto; Merk, Alan; Mobin, Aerfa; Esser, Lothar; Earl, Lesley A.; Gottesman, Michael M.; Xia, Di; Ambudkar, Suresh V.; Subramaniam, Sriram</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-41</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The multidrug transporter P-glycoprotein (P-gp, ABCB1) is an ATP-dependent pump that mediates the efflux of structurally diverse drugs and xenobiotics across cell membranes, affecting drug pharmacokinetics and contributing to the development of multidrug resistance.  Structural information about the conformational changes in human P-gp during the ATP hydrolysis cycle has not been directly demonstrated, although mechanistic information has been inferred from biochem. and biophys. studies conducted with P-gp and its orthologs, or from structures of other ATP-binding cassette transporters.  Using single-particle cryo-electron microscopy, we report the surprising discovery that, in the absence of the transport substrate and nucleotides, human P-gp can exist in both open [nucleotide binding domains (NBDs) apart; inward facing] and closed (NBDs close; outward-facing) conformations.  We also probe conformational states of human P-gp during the catalytic cycle, and demonstrate that, following ATP hydrolysis, P-gp transitions through a complete closed conformation to a complete open conformation in the presence of ADP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1zy4LAdfsg7Vg90H21EOLACvtfcHk0lj3RKoOTw82Ig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVejs7rO&md5=8993b0baf446fde3801846cea3a141aa</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1124%2Fmol.116.104190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.116.104190%26sid%3Dliteratum%253Aachs%26aulast%3DFrank%26aufirst%3DG.%2BA.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DRao%26aufirst%3DP.%26aulast%3DBorgnia%26aufirst%3DM.%2BJ.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DMerk%26aufirst%3DA.%26aulast%3DMobin%26aufirst%3DA.%26aulast%3DEsser%26aufirst%3DL.%26aulast%3DEarl%26aufirst%3DL.%2BA.%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26aulast%3DXia%26aufirst%3DD.%26aulast%3DAmbudkar%26aufirst%3DS.%2BV.%26aulast%3DSubramaniam%26aufirst%3DS.%26atitle%3DCryo-EM%2520analysis%2520of%2520the%2520conformational%2520landscape%2520of%2520human%2520P-glycoprotein%2520%2528ABCB1%2529%2520during%2520its%2520catalytic%2520cycle%26jtitle%3DMol.%2520Pharmacol.%26date%3D2016%26volume%3D90%26spage%3D35%26epage%3D41%26doi%3D10.1124%2Fmol.116.104190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khunweeraphong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchler, K.</span></span> <span> </span><span class="NLM_article-title">The structure of the human ABC transporter ABCG2 reveals a novel mechanism for drug extrusion</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13767</span>, <span class="refDoi">Â DOI: 10.1038/s41598-017-11794-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fs41598-017-11794-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=29061978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC1M7js1yksQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=13767&author=N.+Khunweeraphongauthor=T.+Stocknerauthor=K.+Kuchler&title=The+structure+of+the+human+ABC+transporter+ABCG2+reveals+a+novel+mechanism+for+drug+extrusion&doi=10.1038%2Fs41598-017-11794-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of the human ABC transporter ABCG2 reveals a novel mechanism for drug extrusion</span></div><div class="casAuthors">Khunweeraphong Narakorn; Kuchler Karl; Stockner Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13767</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The human ABC transporter ABCG2 (Breast Cancer Resistance Protein, BCRP) is implicated in anticancer resistance, in detoxification across barriers and linked to gout.  Here, we generate a novel atomic model of ABCG2 using the crystal structure of ABCG5/G8.  Extensive mutagenesis verifies the structure, disclosing hitherto unrecognized essential residues and domains in the homodimeric ABCG2 transporter.  The elbow helix, the first intracellular loop (ICL1) and the nucleotide-binding domain (NBD) constitute pivotal elements of the architecture building the transmission interface that borders a central cavity which acts as a drug trap.  The transmission interface is stabilized by salt-bridge interactions between the elbow helix and ICL1, as well as within ICL1, which is essential to control the conformational switch of ABCG2 to the outward-open drug-releasing conformation.  Importantly, we propose that ICL1 operates like a molecular spring that holds the NBD dimer close to the membrane, thereby enabling efficient coupling of ATP hydrolysis during the catalytic cycle.  These novel mechanistic data open new opportunities to therapeutically target ABCG2 in the context of related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9zxYb5ouxkL0AGQpJZWpJfW6udTcc2eaVb_ZWiqJdSbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7js1yksQ%253D%253D&md5=13b9d4f177ed8f979a173b5e2ce45fb8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-11794-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-11794-w%26sid%3Dliteratum%253Aachs%26aulast%3DKhunweeraphong%26aufirst%3DN.%26aulast%3DStockner%26aufirst%3DT.%26aulast%3DKuchler%26aufirst%3DK.%26atitle%3DThe%2520structure%2520of%2520the%2520human%2520ABC%2520transporter%2520ABCG2%2520reveals%2520a%2520novel%2520mechanism%2520for%2520drug%2520extrusion%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D13767%26doi%3D10.1038%2Fs41598-017-11794-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoghbi, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swartz, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendley, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altenberg, G. A.</span></span> <span> </span><span class="NLM_article-title">Substrate-induced conformational changes in the nucleotide-binding domains of lipid bilayer-associated P-glycoprotein during ATP hydrolysis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">20412</span>â <span class="NLM_lpage">20424</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M117.814186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1074%2Fjbc.M117.814186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=29018094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFygs77K" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=20412-20424&author=M.+E.+Zoghbiauthor=L.+Mokauthor=D.+J.+Swartzauthor=A.+Singhauthor=G.+A.+Fendleyauthor=I.+L.+Urbatschauthor=G.+A.+Altenberg&title=Substrate-induced+conformational+changes+in+the+nucleotide-binding+domains+of+lipid+bilayer-associated+P-glycoprotein+during+ATP+hydrolysis&doi=10.1074%2Fjbc.M117.814186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-induced conformational changes in the nucleotide-binding domains of lipid bilayer-associated P-glycoprotein during ATP hydrolysis</span></div><div class="casAuthors">Zoghbi, Maria E.; Mok, Leo; Swartz, Douglas J.; Singh, Anukriti; Fendley, Gregory A.; Urbatsch, Ina L.; Altenberg, Guillermo A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">20412-20424</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) is an efflux pump important in multidrug resistance of cancer cells and in detg. drug pharmacokinetics.  Pgp is a prototype ATP-binding cassette transporter with 2 nucleotide-binding domains (NBDs) that bind and hydrolyze ATP.  Conformational changes at the NBDs (the Pgp engines) lead to changes across Pgp transmembrane domains that result in substrate translocation.  According to current alternating access models (substrate-binding pocket accessible only to one side of the membrane at a time), binding of ATP promotes NBD dimerization, resulting in external accessibility of the drug-binding site (outward-facing, closed NBD conformation), and ATP hydrolysis leads to dissocn. of the NBDs with the subsequent return of the accessibility of the binding site to the cytoplasmic side (inward-facing, open NBD conformation).  However, previous work has not investigated these events under near-physiol. conditions in a lipid bilayer and in the presence of transport substrate.  Here, we used luminescence resonance energy transfer (LRET) to measure the distances between the 2 Pgp NBDs.  Pgp was labeled with LRET probes, reconstituted in lipid nanodisks, and the distance between the NBDs was measured at 37Â°.  In the presence of verapamil, a substrate that activates ATP hydrolysis, the NBDs of Pgp reconstituted in nanodiscs were never far apart during the hydrolysis cycle, and we never obsd. the NBD-NBD distances of tens of Ã that have previously been reported.  However, we found 2 main conformations that coexist in a dynamic equil. under all conditions studied.  These observations highlight the importance of performing studies of efflux pumps under near-physiol. conditions, in a lipid bilayer, at 37Â°, and during substrate-stimulated hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLqpb0i6gl3bVg90H21EOLACvtfcHk0lgqTLGx5RV6pw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFygs77K&md5=67b866ad533c01e3c2d8730b35d2b2e7</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.814186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.814186%26sid%3Dliteratum%253Aachs%26aulast%3DZoghbi%26aufirst%3DM.%2BE.%26aulast%3DMok%26aufirst%3DL.%26aulast%3DSwartz%26aufirst%3DD.%2BJ.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DFendley%26aufirst%3DG.%2BA.%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DAltenberg%26aufirst%3DG.%2BA.%26atitle%3DSubstrate-induced%2520conformational%2520changes%2520in%2520the%2520nucleotide-binding%2520domains%2520of%2520lipid%2520bilayer-associated%2520P-glycoprotein%2520during%2520ATP%2520hydrolysis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D20412%26epage%3D20424%26doi%3D10.1074%2Fjbc.M117.814186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follit, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span> <span> </span><span class="NLM_article-title">In silico screening for inhibitors of P-glycoprotein that target the nucleotide binding domains</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">716</span>â <span class="NLM_lpage">726</span>, <span class="refDoi">Â DOI: 10.1124/mol.114.095414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1124%2Fmol.114.095414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=25270578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVSmtbfM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=716-726&author=F.+K.+Brewerauthor=C.+A.+Follitauthor=P.+D.+Vogelauthor=J.+G.+Wise&title=In+silico+screening+for+inhibitors+of+P-glycoprotein+that+target+the+nucleotide+binding+domains&doi=10.1124%2Fmol.114.095414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">In silico screening for inhibitors of P-glycoprotein that target the nucleotide binding domains</span></div><div class="casAuthors">Brewer, Frances K.; Follit, Courtney A.; Vogel, Pia D.; Wise, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">716-726, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Multidrug resistances and the failure of chemotherapies are often caused by the expression or overexpression of ATP-binding cassette transporter proteins such as the multidrug resistance protein, P-glycoprotein (P-gp).  P-gp is expressed in the plasma membrane of many cell types and protects cells from accumulation of toxins.  P-gp uses ATP hydrolysis to catalyze the transport of a broad range of mostly hydrophobic compds. across the plasma membrane and out of the cell.  During cancer chemotherapy, the administration of therapeutics often selects for cells which overexpress P-gp, thereby creating populations of cancer cells resistant to a variety of chem. unrelated chemotherapeutics.  The present study describes extremely high-throughput, massively parallel in silico ligand docking studies aimed at identifying reversible inhibitors of ATP hydrolysis that target the nucleotide-binding domains of P-gp.  We used a structural model of human P-gp that we obtained from mol. dynamics expts. as the protein target for ligand docking.  We employed a novel approach of subtractive docking expts. that identified ligands that bound predominantly to the nucleotide-binding domains but not the drug-binding domains of P-gp.  Four compds. were found that inhibit ATP hydrolysis by P-gp.  Using ESR spectroscopy, we showed that at least three of these compds. affected nucleotide binding to the transporter.  These studies represent a successful proof of principle demonstrating the potential of targeted approaches for identifying specific inhibitors of P-gp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-lF2vv6jAJLVg90H21EOLACvtfcHk0lgqTLGx5RV6pw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVSmtbfM&md5=6775ef2590f40a0763f0386cce8f4613</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.095414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.095414%26sid%3Dliteratum%253Aachs%26aulast%3DBrewer%26aufirst%3DF.%2BK.%26aulast%3DFollit%26aufirst%3DC.%2BA.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26aulast%3DWise%26aufirst%3DJ.%2BG.%26atitle%3DIn%2520silico%2520screening%2520for%2520inhibitors%2520of%2520P-glycoprotein%2520that%2520target%2520the%2520nucleotide%2520binding%2520domains%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D86%26spage%3D716%26epage%3D726%26doi%3D10.1124%2Fmol.114.095414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Follit, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span> <span> </span><span class="NLM_article-title">In silico identified targeted inhibitors of P-glycoprotein overcome multidrug resistance in human cancer cells in culture</span>. <i>Pharmacol. Res. Perspect.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e00170</span> <span class="refDoi">Â DOI: 10.1002/prp2.170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fprp2.170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26516582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCjurjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2015&author=C.+A.+Follitauthor=F.+K.+Brewerauthor=J.+G.+Wiseauthor=P.+D.+Vogel&title=In+silico+identified+targeted+inhibitors+of+P-glycoprotein+overcome+multidrug+resistance+in+human+cancer+cells+in+culture&doi=10.1002%2Fprp2.170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">In silico identified targeted inhibitors of P-glycoprotein overcome multidrug resistance in human cancer cells in culture</span></div><div class="casAuthors">Follit, Courtney A.; Brewer, Frances K.; Wise, John G.; Vogel, Pia D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology Research & Perspectives</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e00170/1-e00170/13</span>CODEN:
                <span class="NLM_cas:coden">PRPHCA</span>;
        ISSN:<span class="NLM_cas:issn">2052-1707</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Failure of cancer chemotherapies is often linked to the over expression of ABC efflux transporters like the multidrug resistance P-glycoprotein (P-gp).  P-gp expression in cells leads to the elimination of a variety of chem. unrelated, mostly cytotoxic compds.  Administration of chemotherapeutics during therapy frequently selects for cells that over express P-gp and are therefore capable of robustly exporting diverse compds., including chemotherapeutics, from the cells.  P-gp thus confers multidrug resistance to a majority of drugs currently available for the treatment of cancers and diseases like HIV/AIDS.  The search for P-gp inhibitors for use as co-therapeutics to combat multidrug resistances has had little success to date.  In a previous study (Brewer et al., Mol Pharmacol 86: 716-726, 2014), we described how ultrahigh throughput computational searches led to the identification of four drug-like mols. that specifically interfere with the energy harvesting steps of substrate transport and inhibit P-gp catalyzed ATP hydrolysis in vitro.  In the present study, we demonstrate that three of these compds. reversed P-gp-mediated multidrug resistance of cultured prostate cancer cells to restore sensitivity comparable to naive prostate cancer cells to the chemotherapeutic drug, paclitaxel.  Potentiation concns. of the inhibitors were <3 Î¼mol/L.  The inhibitors did not exhibit significant toxicity to noncancerous cells at concns. where they reversed multidrug resistance in cancerous cells.  Our results indicate that these compds. with novel mechanisms of P-gp inhibition are excellent leads for the development of co-therapeutics for the treatment of multidrug resistances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokpFCstWfMq7Vg90H21EOLACvtfcHk0ljixl045IqSpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCjurjE&md5=92d1e01b1e99b1d73e3f21e60e8485aa</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fprp2.170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprp2.170%26sid%3Dliteratum%253Aachs%26aulast%3DFollit%26aufirst%3DC.%2BA.%26aulast%3DBrewer%26aufirst%3DF.%2BK.%26aulast%3DWise%26aufirst%3DJ.%2BG.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26atitle%3DIn%2520silico%2520identified%2520targeted%2520inhibitors%2520of%2520P-glycoprotein%2520overcome%2520multidrug%2520resistance%2520in%2520human%2520cancer%2520cells%2520in%2520culture%26jtitle%3DPharmacol.%2520Res.%2520Perspect.%26date%3D2015%26volume%3D3%26doi%3D10.1002%2Fprp2.170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nanayakkara, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follit, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span> <span> </span><span class="NLM_article-title">Targeted inhibitors of P-glycoprotein increase chemotherapeutic-induced mortality of multidrug resistant tumor cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">967</span>, <span class="refDoi">Â DOI: 10.1038/s41598-018-19325-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fs41598-018-19325-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=29343829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC1MvhslSrtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=967&author=A.+K.+Nanayakkaraauthor=C.+A.+Follitauthor=G.+Chenauthor=N.+S.+Williamsauthor=P.+D.+Vogelauthor=J.+G.+Wise&title=Targeted+inhibitors+of+P-glycoprotein+increase+chemotherapeutic-induced+mortality+of+multidrug+resistant+tumor+cells&doi=10.1038%2Fs41598-018-19325-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted inhibitors of P-glycoprotein increase chemotherapeutic-induced mortality of multidrug resistant tumor cells</span></div><div class="casAuthors">Nanayakkara Amila K; Follit Courtney A; Chen Gang; Vogel Pia D; Wise John G; Nanayakkara Amila K; Follit Courtney A; Chen Gang; Vogel Pia D; Wise John G; Williams Noelle S</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">967</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Overexpression of ATP-binding cassette (ABC) transporters is often linked to multidrug resistance (MDR) in cancer chemotherapies.  P-glycoprotein (P-gp) is one of the best studied drug transporters associated with MDR.  There are currently no approved drugs available for clinical use in cancer chemotherapies to reverse MDR by inhibiting P-glycoprotein.  Using computational studies, we previously identified several compounds that inhibit P-gp by targeting its nucleotide binding domain and avoiding its drug binding domains.  Several of these compounds showed successful MDR reversal when tested on a drug resistant prostate cancer cell line.  Using conventional two-dimensional cell culture of MDR ovarian and prostate cancer cells and three dimensional prostate cancer microtumor spheroids, we demonstrated here that co-administration with chemotherapeutics significantly decreased cell viability and survival as well as cell motility.  The P-gp inhibitors were not observed to be toxic on their own.  The inhibitors increased cellular retention of chemotherapeutics and reporter compounds known to be transport substrates of P-gp.  We also showed that these compounds are not transport substrates of P-gp and that two of the three inhibit P-gp, but not the closely related ABC transporter, ABCG2/BCRP.  The results presented suggest that these P-gp inhibitors may be promising leads for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7izekEYEuo42y4bB4mnvYfW6udTcc2ebbsQpyRbH7mbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvhslSrtg%253D%253D&md5=dd83baacadbdf694e71154a563b8c55f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-19325-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-19325-x%26sid%3Dliteratum%253Aachs%26aulast%3DNanayakkara%26aufirst%3DA.%2BK.%26aulast%3DFollit%26aufirst%3DC.%2BA.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DWilliams%26aufirst%3DN.%2BS.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26aulast%3DWise%26aufirst%3DJ.%2BG.%26atitle%3DTargeted%2520inhibitors%2520of%2520P-glycoprotein%2520increase%2520chemotherapeutic-induced%2520mortality%2520of%2520multidrug%2520resistant%2520tumor%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D967%26doi%3D10.1038%2Fs41598-018-19325-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltanshahi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawless, M. S.</span></span> <span> </span><span class="NLM_article-title">Virtual compound libraries: a new approach to decision making in molecular discovery research</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1010</span>â <span class="NLM_lpage">1023</span>, <span class="refDoi">Â DOI: 10.1021/ci9800209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9800209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK1cXmtVGgtbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=1010-1023&author=R.+D.+Cramerauthor=D.+E.+Pattersonauthor=R.+D.+Clarkauthor=F.+Soltanshahiauthor=M.+S.+Lawless&title=Virtual+compound+libraries%3A+a+new+approach+to+decision+making+in+molecular+discovery+research&doi=10.1021%2Fci9800209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Compound Libraries: A New Approach to Decision Making in Molecular Discovery Research</span></div><div class="casAuthors">Cramer, Richard D.; Patterson, David E.; Clark, Robert D.; Soltanshahi, Farhad; Lawless, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1010-1023</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual compd. libraries, descriptions of all of the structures that might be produced by specified transformations involving specified reagents, are esp. useful in mol. discovery when suitably fast and relevant searching techniques are available.  Issues to be considered include fundamental data structures, neighborhood searching principles, useful searching approaches and techniques, library definition and construction, algorithmic details of library comparison, and user interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjcDetEYSHo7Vg90H21EOLACvtfcHk0ljixl045IqSpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtVGgtbg%253D&md5=389423dd71db05df0470fffbf147aa39</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fci9800209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9800209%26sid%3Dliteratum%253Aachs%26aulast%3DCramer%26aufirst%3DR.%2BD.%26aulast%3DPatterson%26aufirst%3DD.%2BE.%26aulast%3DClark%26aufirst%3DR.%2BD.%26aulast%3DSoltanshahi%26aufirst%3DF.%26aulast%3DLawless%26aufirst%3DM.%2BS.%26atitle%3DVirtual%2520compound%2520libraries%253A%2520a%2520new%2520approach%2520to%2520decision%2520making%2520in%2520molecular%2520discovery%2520research%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D1010%26epage%3D1023%26doi%3D10.1021%2Fci9800209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de AraÃºjo-JÃºnior, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duranton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Justiano-Basaran, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugnier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourguignon, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Design of small-sized libraries by combinatorial assembly of linkers and functional groups to a given scaffold: application to the structure-based optimization of a phosphodiesterase 4 inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">3816</span>â <span class="NLM_lpage">3822</span>, <span class="refDoi">Â DOI: 10.1021/jm050063y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050063y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFKjsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3816-3822&author=M.+Krierauthor=J.+X.+de+Ara%C3%BAjo-J%C3%BAniorauthor=M.+Schmittauthor=J.+Durantonauthor=H.+Justiano-Basaranauthor=C.+Lugnierauthor=J.-J.+Bourguignonauthor=D.+Rognan&title=Design+of+small-sized+libraries+by+combinatorial+assembly+of+linkers+and+functional+groups+to+a+given+scaffold%3A+application+to+the+structure-based+optimization+of+a+phosphodiesterase+4+inhibitor&doi=10.1021%2Fjm050063y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Small-Sized Libraries by Combinatorial Assembly of Linkers and Functional Groups to a Given Scaffold: Application to the Structure-Based Optimization of a Phosphodiesterase 4 Inhibitor</span></div><div class="casAuthors">Krier, Mireille; De Araujo-Junior, Joao X.; Schmitt, Martine; Duranton, Jerome; Justiano-Basaran, Helene; Lugnier, Claire; Bourguignon, Jean-Jacques; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3816-3822</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Combinatorial chem. and library design have been reconciled by applying simple medicinal chem. concepts to virtual library design.  The herein reported "Scaffold-Linker-Functional Group" (SLF) approach has the aim to maximize diversity while minimizing the size of a scaffold-focused library with the aid of simple mol. variations in order to identify crit. pharmacophoric elements.  Straightforward rules define the way of assembling three building blocks: a conserved scaffold, a variable linker, and a variable functional group.  By carefully selecting a limited no. of functional groups not overlapping in pharmacophoric space, the size of the library is kept to a few hundred.  As an application of the SLF approach, a small-sized combinatorial library (320 compds.) was derived from the scaffold of the known phosphodiesterase 4 inhibitor zardaverine.  The most interesting analogs were further prioritized for synthesis and enzyme inhibition by FlexX docking to the X-ray structure of the enzyme, leading to a 900-fold increased affinity within nine synthesized compds. and a single screening round.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQS0szYq_dT7Vg90H21EOLACvtfcHk0ljixl045IqSpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFKjsbg%253D&md5=757e7fa03b4ed33f251fd33126e947c4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm050063y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050063y%26sid%3Dliteratum%253Aachs%26aulast%3DKrier%26aufirst%3DM.%26aulast%3Dde%2BAra%25C3%25BAjo-J%25C3%25BAnior%26aufirst%3DJ.%2BX.%26aulast%3DSchmitt%26aufirst%3DM.%26aulast%3DDuranton%26aufirst%3DJ.%26aulast%3DJustiano-Basaran%26aufirst%3DH.%26aulast%3DLugnier%26aufirst%3DC.%26aulast%3DBourguignon%26aufirst%3DJ.-J.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DDesign%2520of%2520small-sized%2520libraries%2520by%2520combinatorial%2520assembly%2520of%2520linkers%2520and%2520functional%2520groups%2520to%2520a%2520given%2520scaffold%253A%2520application%2520to%2520the%2520structure-based%2520optimization%2520of%2520a%2520phosphodiesterase%25204%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3816%26epage%3D3822%26doi%3D10.1021%2Fjm050063y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttamchandani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S. Q.</span></span> <span> </span><span class="NLM_article-title">Rapid assembly and in situ screening of bidentate inhibitors of protein tyrosine phosphatases</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">713</span>â <span class="NLM_lpage">716</span>, <span class="refDoi">Â DOI: 10.1021/ol052895w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol052895w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD28XntFGjsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=713-716&author=R.+Srinivasanauthor=M.+Uttamchandaniauthor=S.+Q.+Yao&title=Rapid+assembly+and+in+situ+screening+of+bidentate+inhibitors+of+protein+tyrosine+phosphatases&doi=10.1021%2Fol052895w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Assembly and in Situ Screening of Bidentate Inhibitors of Protein Tyrosine Phosphatases</span></div><div class="casAuthors">Srinivasan, Rajavel; Uttamchandani, Mahesh; Yao, Shao Q.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">713-716</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have successfully designed and synthesized a small library of protein tyrosine phosphatase (PTP) inhibitors, in which the so-called click chem. or Cu(I)-catalyzed 1,3-dipolar alkyne-azide coupling reaction was carried out for rapid assembly of 66 different bidentate compds.  Subsequent in situ enzymic screening revealed a potential PTP1B inhibitor (IC50 = 4.7 Î¼M) which is 10-100 fold more potent than other PTPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcj-jI1-_paLVg90H21EOLACvtfcHk0lgJXa8qS-V8vA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntFGjsA%253D%253D&md5=7fbe6b911bc26b5f30d1ff3d592091e1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fol052895w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol052895w%26sid%3Dliteratum%253Aachs%26aulast%3DSrinivasan%26aufirst%3DR.%26aulast%3DUttamchandani%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DS.%2BQ.%26atitle%3DRapid%2520assembly%2520and%2520in%2520situ%2520screening%2520of%2520bidentate%2520inhibitors%2520of%2520protein%2520tyrosine%2520phosphatases%26jtitle%3DOrg.%2520Lett.%26date%3D2006%26volume%3D8%26spage%3D713%26epage%3D716%26doi%3D10.1021%2Fol052895w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durrant, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">AutoClickChem: click chemistry in silico</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002397</span> <span class="refDoi">Â DOI: 10.1371/journal.pcbi.1002397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1371%2Fjournal.pcbi.1002397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=22438795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslahtLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=J.+D.+Durrantauthor=J.+A.+McCammon&title=AutoClickChem%3A+click+chemistry+in+silico&doi=10.1371%2Fjournal.pcbi.1002397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">AutoClickChem: click chemistry in silico</span></div><div class="casAuthors">Durrant, Jacob D.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1002397</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Academic researchers and many in industry often lack the financial resources available to scientists working in "big pharma".  High costs include those assocd. with high-throughput screening and chem. synthesis.  In order to address these challenges, many researchers have in part turned to alternate methodologies.  Virtual screening, for example, often substitutes for high-throughput screening and click chem. ensures that chem. synthesis is fast, cheap and comparatively easy.  Though both in silico screening and click chem. seek to make drug discovery more feasible, it is not yet routine to couple these two methodologies.  We here present a novel computer algorithm, called AutoClickChem, capable of performing many click-chem. reactions in silico.  AutoClickChem can be used to produce large combinatorial libraries of compd. models for use in virtual screens.  As the compds. of these libraries are constructed according to the reactions of click chem., they can be easily synthesized for subsequent testing in biochem. assays.  Addnl., in silico modeling of click-chem. products may prove useful in rational drug design and drug optimization.  AutoClickChem is based on the pymol. toolbox, a framework that may facilitate the development of future python-based programs that require the manipulation of mol. models.  Both the pymol. toolbox and AutoClickChem are released under the GNU General Public License version 3 and are available for download.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo44OEkiGkWVrVg90H21EOLACvtfcHk0lgJXa8qS-V8vA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslahtLo%253D&md5=53645fe62d0cc696c245b32bca2e2fe0</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002397%26sid%3Dliteratum%253Aachs%26aulast%3DDurrant%26aufirst%3DJ.%2BD.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DAutoClickChem%253A%2520click%2520chemistry%2520in%2520silico%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.pcbi.1002397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namiki, M.</span></span> <span> </span><span class="NLM_article-title">The establishment of two paclitaxel-resistant prostate cancer cell lines and the mechanisms of paclitaxel resistance with two cell lines</span>. <i>Prostate</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">955</span>â <span class="NLM_lpage">967</span>, <span class="refDoi">Â DOI: 10.1002/pros.20581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fpros.20581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=17440963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmvVKqt7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=955-967&author=M.+Takedaauthor=A.+Mizokamiauthor=K.+Mamiyaauthor=Y.+Q.+Liauthor=J.+Zhangauthor=E.+T.+Kellerauthor=M.+Namiki&title=The+establishment+of+two+paclitaxel-resistant+prostate+cancer+cell+lines+and+the+mechanisms+of+paclitaxel+resistance+with+two+cell+lines&doi=10.1002%2Fpros.20581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The establishment of two paclitaxel-resistant prostate cancer cell lines and the mechanisms of paclitaxel resistance with two cell lines</span></div><div class="casAuthors">Takeda, Masashi; Mizokami, Atsushi; Mamiya, Kiminori; Li, You Qiang; Zhang, Jian; Keller, Evan T.; Namiki, Mikio</div><div class="citationInfo"><span class="NLM_cas:title">Prostate (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">955-967</span>CODEN:
                <span class="NLM_cas:coden">PRSTDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-4137</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">BACKGROUND: Although paclitaxel is used for hormone-resistant prostate cancer, relapse definitely occurs later.  Details of the mol. mechanism responsible for paclitaxel-resistance remain unclear.  METHODS: We established paclitaxel-resistant cells, DU145-TxR and PC-3-TxR from parent DU145 and PC-3.  To characterize these cells, we examd. cross-resistance to other anticancer drugs.  Expression of several potential genes that had been related to drug-resistance was compared with parent cells by RT-PCR and Western blotting.  Methylation anal. of multiple drug resistance (MDR1) promoter was carried out using bisulfite-modified DNA from cell lines.  Knockdown expts. using small interfering RNA (siRNA) were also performed to confirm responsibility of drug-resistance.  Finally, cDNA microarray was performed to quantify gene expression in PC-3 and PC-3-TxR cells.  RESULTS: The IC50 for paclitaxel in DU145-TxR and PC-3-TxR was 34.0- and 43.4-fold higher than that in both parent cells, resp.  Both cells showed cross-resistance to some drugs, but not to VP-16 and cisplatin.  Methylation anal. revealed that methylated CpG sites of MDR1 promoter in DU145 and PC-3 cells were demethylated in DU145-TxR cells, but not in PC-3-TxR cells.  Knockdown of P-glycoprotein (P-gp), which was up-regulated in resistant cells, by MDR-1 siRNA restored paclitaxel sensitivity in DU145-TxR but not in PC-3-TxR, indicating that up-regulation of P-gp was not always main cause of paclitaxel-resistance.  Microarray anal. identified 201 (1.34%) up-regulated genes and 218 (1.45%) out of screened genes in PC-3-TxR.  CONCLUSIONS: Our data will provide mol. mechanisms of paclitaxel-resistance and be useful for screening target genes to diagnose paclitaxel sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCs6firNNZ87Vg90H21EOLACvtfcHk0lgJXa8qS-V8vA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmvVKqt7c%253D&md5=1b0e44bab963a7b82abc3eb6c979ca24</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fpros.20581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpros.20581%26sid%3Dliteratum%253Aachs%26aulast%3DTakeda%26aufirst%3DM.%26aulast%3DMizokami%26aufirst%3DA.%26aulast%3DMamiya%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DY.%2BQ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DE.%2BT.%26aulast%3DNamiki%26aufirst%3DM.%26atitle%3DThe%2520establishment%2520of%2520two%2520paclitaxel-resistant%2520prostate%2520cancer%2520cell%2520lines%2520and%2520the%2520mechanisms%2520of%2520paclitaxel%2520resistance%2520with%2520two%2520cell%2520lines%26jtitle%3DProstate%26date%3D2007%26volume%3D67%26spage%3D955%26epage%3D967%26doi%3D10.1002%2Fpros.20581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolstad, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span> <span> </span><span class="NLM_article-title">ZINC: A free tool to discover chemistry for biology</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1757</span>â <span class="NLM_lpage">1768</span>, <span class="refDoi">Â DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+A+free+tool+to+discover+chemistry+for+biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lhBblxfhZbKQg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520A%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1074</span>â <span class="NLM_lpage">1080</span>, <span class="refDoi">Â DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+tool+for+ligand+coordinate+and+restraint+generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0lhBblxfhZbKQg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3Delectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520tool%2520for%2520ligand%2520coordinate%2520and%2520restraint%2520generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span> <span> </span><span class="NLM_article-title">Specific 1,2-Hydride Shift in the Boron Trifluoride Catalyzed Reactions of Aromatic Aldehydes with Diazoacetonitrile: Simple Synthesis of Î²-Ketonitriles</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_fpage">6380</span>â <span class="NLM_lpage">6384</span>, <span class="refDoi">Â DOI: 10.1002/ejoc.201402901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fejoc.201402901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWjt7rL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&pages=6380-6384&author=Z.+Yangauthor=K.-I.+Sonauthor=S.+Liauthor=B.+Zhouauthor=J.+Xu&title=Specific+1%2C2-Hydride+Shift+in+the+Boron+Trifluoride+Catalyzed+Reactions+of+Aromatic+Aldehydes+with+Diazoacetonitrile%3A+Simple+Synthesis+of+%CE%B2-Ketonitriles&doi=10.1002%2Fejoc.201402901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Specific 1,2-Hydride Shift in the Boron Trifluoride Catalyzed Reactions of Aromatic Aldehydes with Diazoacetonitrile: Simple Synthesis of Î²-Ketonitriles</span></div><div class="casAuthors">Yang, Zhanhui; Son, Kwon-Il; Li, Siqi; Zhou, Bingnan; Xu, Jiaxi</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">6380-6384</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of Î²-ketonitriles RC(O)CH2CN [R = 3,4-(CH3)2C6H3, naphth-1-yl, thiophen-1-yl, furan-1-yl, etc.] was synthesized within 30 min under mild conditions through the BF3Â·OEt2-catalyzed addn. reactions of diazoacetonitrile to arom. aldehydes and subsequent 1,2-hydride shift.  This method is advantageous in that it is operationally simple, mild and metal-free conditions are used, the substrate scope is wide, and the products are obtained in moderate to high yields (up to 81%).  Addnl., Î³,Î´-unsatd. Î²-ketonitriles are also accessible by this method by using cinnamaldehydes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwV7CKaBQCHbVg90H21EOLACvtfcHk0lhBblxfhZbKQg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWjt7rL&md5=c81ae13233ada462fe2a9826d47afe53</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201402901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201402901%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DSon%26aufirst%3DK.-I.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DJ.%26atitle%3DSpecific%25201%252C2-Hydride%2520Shift%2520in%2520the%2520Boron%2520Trifluoride%2520Catalyzed%2520Reactions%2520of%2520Aromatic%2520Aldehydes%2520with%2520Diazoacetonitrile%253A%2520Simple%2520Synthesis%2520of%2520%25CE%25B2-Ketonitriles%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2014%26spage%3D6380%26epage%3D6384%26doi%3D10.1002%2Fejoc.201402901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, F. F.</span></span> <span> </span><span class="NLM_article-title">An efficient one-pot synthesis of N-(1,3-diphenyl-1H-pyrazol-5-yl)amides</span>. <i>J. Heterocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">831</span>â <span class="NLM_lpage">837</span>, <span class="refDoi">Â DOI: 10.1002/jhet.343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fjhet.343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Gis7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2010&pages=831-837&author=W.+N.+Suauthor=T.+P.+Linauthor=K.+M.+Chengauthor=K.+C.+Sungauthor=S.+K.+Linauthor=F.+F.+Wong&title=An+efficient+one-pot+synthesis+of+N-%281%2C3-diphenyl-1H-pyrazol-5-yl%29amides&doi=10.1002%2Fjhet.343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient one-pot synthesis of N-(1,3-diphenyl-1H-pyrazol-5-yl)amides</span></div><div class="casAuthors">Su, Wei-Nien; Lin, Tsung-Ping; Cheng, Kaung-Min; Sung, Kuan-Chin; Lin, Shao-Kai; Wong, Fung Fuh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">831-837</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">1943-5193</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A "one-pot" method for the synthesis of N-(1,3-diphenyl-1H-pyrazol-5-yl)amides was developed by cyclization of benzoylacetonitrile and phenylhydrazine in neat condition followed by acylation.  The corresponding N-(1,3-diphenyl-1H-pyrazol-5-yl)amides, e.g., I (R = H, Me, MeO, Cl, NO2), were provided in good to excellent yields (70-90%).  The significant advantages of the new synthetic method are excellent yields and simple work-up procedure without isolation and purifn. of intermediary 5-amino-1,3-diphenyl pyrazole.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCc88mn8mp5rVg90H21EOLACvtfcHk0lgrIvNzIsiTYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Gis7c%253D&md5=157fc13aa9fd5dc747d10f39709c4da7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjhet.343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.343%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DW.%2BN.%26aulast%3DLin%26aufirst%3DT.%2BP.%26aulast%3DCheng%26aufirst%3DK.%2BM.%26aulast%3DSung%26aufirst%3DK.%2BC.%26aulast%3DLin%26aufirst%3DS.%2BK.%26aulast%3DWong%26aufirst%3DF.%2BF.%26atitle%3DAn%2520efficient%2520one-pot%2520synthesis%2520of%2520N-%25281%252C3-diphenyl-1H-pyrazol-5-yl%2529amides%26jtitle%3DJ.%2520Heterocyclic%2520Chem.%26date%3D2010%26volume%3D47%26spage%3D831%26epage%3D837%26doi%3D10.1002%2Fjhet.343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilton, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motherwell, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selwood, D. L.</span></span> <span> </span><span class="NLM_article-title">Observations on the reactivity of thiyl radicals derived from 3,6-epidithiodiketopiperazine-2,5-diones and related congeners</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2239</span>â <span class="NLM_lpage">2242</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2005.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.bmcl.2005.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=15837301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlWqs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=2239-2242&author=S.+T.+Hiltonauthor=W.+B.+Motherwellauthor=P.+Potierauthor=C.+Pradetauthor=D.+L.+Selwood&title=Observations+on+the+reactivity+of+thiyl+radicals+derived+from+3%2C6-epidithiodiketopiperazine-2%2C5-diones+and+related+congeners&doi=10.1016%2Fj.bmcl.2005.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Observations on the reactivity of thiyl radicals derived from 3,6-epidithiodiketopiperazine-2,5-diones and related congeners</span></div><div class="casAuthors">Hilton, S. T.; Motherwell, W. B.; Potier, P.; Pradet, C.; Selwood, D. L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2239-2242</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A range of thiyl radicals derived from the reduced form of epidithiodiketopiperazines (ETPs) act as polarity reversal catalysts for the hydrosilylation of an enol lactone but not for H-atom abstraction from a model ribose ester.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSDJW9u-rjS7Vg90H21EOLACvtfcHk0lgrIvNzIsiTYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlWqs7g%253D&md5=a492f94d4a3448a54de140c8e1be00f6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DHilton%26aufirst%3DS.%2BT.%26aulast%3DMotherwell%26aufirst%3DW.%2BB.%26aulast%3DPotier%26aufirst%3DP.%26aulast%3DPradet%26aufirst%3DC.%26aulast%3DSelwood%26aufirst%3DD.%2BL.%26atitle%3DObservations%2520on%2520the%2520reactivity%2520of%2520thiyl%2520radicals%2520derived%2520from%25203%252C6-epidithiodiketopiperazine-2%252C5-diones%2520and%2520related%2520congeners%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D2239%26epage%3D2242%26doi%3D10.1016%2Fj.bmcl.2005.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krushinski, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beedle, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leander, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathbun, R. C.</span></span> <span> </span><span class="NLM_article-title">Structure-activity-relationships of (arylalkyl)imidazole anticonvulsants - comparison of the (fluorenylalkyl)imidazoles with nafimidone and denzimol</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1577</span>â <span class="NLM_lpage">1586</span>, <span class="refDoi">Â DOI: 10.1021/jm00159a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00159a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaL28XkvFOgurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=1577-1586&author=D.+W.+Robertsonauthor=J.+H.+Krushinskiauthor=E.+E.+Beedleauthor=J.+D.+Leanderauthor=D.+T.+Wongauthor=R.+C.+Rathbun&title=Structure-activity-relationships+of+%28arylalkyl%29imidazole+anticonvulsants+-+comparison+of+the+%28fluorenylalkyl%29imidazoles+with+nafimidone+and+denzimol&doi=10.1021%2Fjm00159a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships of (arylalkyl)imidazole anticonvulsants:  comparison of the (fluorenylalkyl)imidazoles with nafimidone and denzimol</span></div><div class="casAuthors">Robertson, David W.; Krushinski, Joseph H.; Beedle, E. E.; Leander, J. David; Wong, David T.; Rathbun, R. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1577-86</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The title compds., e.g. I [R = benzo[b]thien-2-yl, benzofuran-2-yl, phenanthren-2-yl, 9,10-dihydrophenanthren-3-yl, X = O; R = naphthalen-2-yl, X = H2] and II (R1 = H, 9-Me, 7-MeO, 7-Et; R2 = H, 2-Me, 4-Me; X = O, HO,H, OCH2CH2O), were prepd. as anticonvulsants.  Thus, 1-(9H-fluoren-2-yl)ethanone was treated with CuBr2 to give the 2-bromo deriv., which was treated with 2-methylimidazole to give II (R1 = H, R2 = 2-Me, X = O).  II (R1 = R2 = H, X = O) was twice as potent as nafimidone in inhibiting maximal electroshock seizures in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQjjdvfXE7krVg90H21EOLACvtfcHk0lgrIvNzIsiTYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XkvFOgurw%253D&md5=d5733db2e6ed41e3acda6cd1456e62ae</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm00159a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00159a004%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DD.%2BW.%26aulast%3DKrushinski%26aufirst%3DJ.%2BH.%26aulast%3DBeedle%26aufirst%3DE.%2BE.%26aulast%3DLeander%26aufirst%3DJ.%2BD.%26aulast%3DWong%26aufirst%3DD.%2BT.%26aulast%3DRathbun%26aufirst%3DR.%2BC.%26atitle%3DStructure-activity-relationships%2520of%2520%2528arylalkyl%2529imidazole%2520anticonvulsants%2520-%2520comparison%2520of%2520the%2520%2528fluorenylalkyl%2529imidazoles%2520with%2520nafimidone%2520and%2520denzimol%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1986%26volume%3D29%26spage%3D1577%26epage%3D1586%26doi%3D10.1021%2Fjm00159a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Meerloo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspers, G. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloos, J.</span></span> <span> </span><span class="NLM_article-title">Cell sensitivity assays: the MTT assay</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>731</i></span>,  <span class="NLM_fpage">237</span>â <span class="NLM_lpage">245</span>, <span class="refDoi">Â DOI: 10.1007/978-1-61779-080-5_20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1007%2F978-1-61779-080-5_20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=21516412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVyqtbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=731&publication_year=2011&pages=237-245&author=J.+van%0AMeerlooauthor=G.+J.+L.+Kaspersauthor=J.+Cloos&title=Cell+sensitivity+assays%3A+the+MTT+assay&doi=10.1007%2F978-1-61779-080-5_20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Cell sensitivity assays: the MTT assay</span></div><div class="casAuthors">van Meerloo, Johan; Kaspers, Gertjan J. L.; Cloos, Jacqueline</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">731</span>
        (<span class="NLM_cas:issue">Cancer Cell Culture</span>),
    <span class="NLM_cas:pages">237-245</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The MTT (3-[4,5-dimethylthiazol-2-yl]-2,5 di-Ph tetrazolium bromide) assay is based on the conversion of MTT into formazan crystals by living cells, which dets. mitochondrial activity.  Since for most cell populations the total mitochondrial activity is related to the no. of viable cells, this assay is broadly used to measure the in vitro cytotoxic effects of drugs on cell lines or primary patient cells.  In this chapter the protocol of the assay is described including important considerations relevant for each step of the assay as well as its limitations and possible applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprO-kUg91O_7Vg90H21EOLACvtfcHk0lgrIvNzIsiTYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVyqtbk%253D&md5=f88b391a65df5cb01b3bba486c55ce96</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-080-5_20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-080-5_20%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BMeerloo%26aufirst%3DJ.%26aulast%3DKaspers%26aufirst%3DG.%2BJ.%2BL.%26aulast%3DCloos%26aufirst%3DJ.%26atitle%3DCell%2520sensitivity%2520assays%253A%2520the%2520MTT%2520assay%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D731%26spage%3D237%26epage%3D245%26doi%3D10.1007%2F978-1-61779-080-5_20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berridge, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, A. S.</span></span> <span> </span><span class="NLM_article-title">Characterization of the cellular reduction of 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT): subcellular localization, substrate dependence, and involvement of mitochondrial electron transport in MTT reduction</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>303</i></span>,  <span class="NLM_fpage">474</span>â <span class="NLM_lpage">482</span>, <span class="refDoi">Â DOI: 10.1006/abbi.1993.1311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1006%2Fabbi.1993.1311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=8390225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK3sXkvFSmsL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=1993&pages=474-482&author=M.+V.+Berridgeauthor=A.+S.+Tan&title=Characterization+of+the+cellular+reduction+of+3-%284%2C5-dimethylthiazol-2-yl%29-2%2C5-diphenyltetrazolium+bromide+%28MTT%29%3A+subcellular+localization%2C+substrate+dependence%2C+and+involvement+of+mitochondrial+electron+transport+in+MTT+reduction&doi=10.1006%2Fabbi.1993.1311"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the cellular reduction of 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT): Subcellular localization, substrate dependence, and involvement of mitochondrial electron transport in MTT reduction</span></div><div class="casAuthors">Berridge, Michael V.; Tan, An S.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">474-82</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">The MTT assay, which is widely used to measure cell proliferation and to screen for anticancer drugs, is based on redn. of the tetrazolium salt MTT by actively growing cells to produce a blue formazan product.  Despite broad acceptance of this assay, neither the subcellular localization, nor the biochem. events involved in MTT redn. are known.  Mitochondrial involvement in MTT redn. has been inferred from studies with respiratory inhibitors using succinate as a substrate, but the contribution of this activity to overall cellular MTT redn. is unknown.  The bone marrow-derived cell line 32D was used to investigate the subcellular localization of MTT redn. by using succinate, NADH, and NADPH as substrates.  At optimum substrate concns., MTT redn. by whole cell homogenates was greatest with NADH and least with succinate, which accounted for <10% of the combined activities.  Using succinate, 95% of recoverable MTT reducing activity was in particulate fractions of the cell and 77% in the mitochondrial and light mitochondrial/lysosomal fractions.  When NADH and NADPH were used as substrates, increased amts. of MTT reducing activity were assocd. with sol. fractions of the cell and assocn. with mitochondrial fractions was less pronounced.  To further characterize MTT redn. by the mitochondrial fraction, respiratory chain inhibitors were used to explore involvement of electron transport in MTT redn.  Succinate-dependent mitochondrial MTT redn. was inhibited by 80% with chlorpromazine, 70% by antimycin A, and 85-90% by thenoyltrifluoroacetone (TTFA), but inhibition was not obsd. with rotenone at â¤2 Î¼M, Amytal, or azide.  These results suggest that when succinate is used as an electron donor, 70-80% of mitochondrial MTT redn. occurs subsequent to transfer of electrons from cytochrome c to cytochrome oxidase, but prior to the point of azide inhibition.  In contrast to succinate, NADPH-dependent mitochondrial MTT redn. was not affected by any of the respiratory inhibitors tested, and NADH-dependent redn. was only inhibited by chlorpromazine (40-50% at plateau concn.).  These results suggest that most cellular MTT redn. occurs outside the mitochondrial inner membrane and involves NADH and NADPH-dependent mechanisms that are insensitive to respiratory chain inhibitors.  This interpretation is supported by whole cell studies in which rotenone failed to affect basal and interleukin-3-stimulated MTT redn. at times up to 4 h but strongly inhibited DNA synthesis.  Apparently most cellular redn. of MTT occurs extramitochondrially and probably involves the pyridine nucleotide cofactors NADH and NADPH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmehIJ3dI6S7Vg90H21EOLACvtfcHk0lhKfLWcdyCKuw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvFSmsL0%253D&md5=2f559aaae05ea333d5a407b12c32a0d7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1006%2Fabbi.1993.1311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.1993.1311%26sid%3Dliteratum%253Aachs%26aulast%3DBerridge%26aufirst%3DM.%2BV.%26aulast%3DTan%26aufirst%3DA.%2BS.%26atitle%3DCharacterization%2520of%2520the%2520cellular%2520reduction%2520of%25203-%25284%252C5-dimethylthiazol-2-yl%2529-2%252C5-diphenyltetrazolium%2520bromide%2520%2528MTT%2529%253A%2520subcellular%2520localization%252C%2520substrate%2520dependence%252C%2520and%2520involvement%2520of%2520mitochondrial%2520electron%2520transport%2520in%2520MTT%2520reduction%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1993%26volume%3D303%26spage%3D474%26epage%3D482%26doi%3D10.1006%2Fabbi.1993.1311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kupcsik, L.</span></span> <span> </span><span class="NLM_article-title">Estimation of cell number based on metabolic activity: the MTT reduction assay</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>740</i></span>,  <span class="NLM_fpage">13</span>â <span class="NLM_lpage">19</span>, <span class="refDoi">Â DOI: 10.1007/978-1-61779-108-6_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1007%2F978-1-61779-108-6_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=21468963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntlakt7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=740&publication_year=2011&pages=13-19&author=L.+Kupcsik&title=Estimation+of+cell+number+based+on+metabolic+activity%3A+the+MTT+reduction+assay&doi=10.1007%2F978-1-61779-108-6_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of cell number based on metabolic activity: the MTT reduction assay</span></div><div class="casAuthors">Kupcsik, Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">740</span>
        (<span class="NLM_cas:issue">Mammalian Cell Viability</span>),
    <span class="NLM_cas:pages">13-19</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The MTT redn. assay is used to det. the level of metabolic activity in eukaryotic cells, including animal, plant, and fungal cells.  If the metabolic rate is const., the technique can be employed to count living cells in a sample.  Once it is set up, the method is very robust, and can be automatized to be applied on a large no. of samples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKpysUPR2UR7Vg90H21EOLACvtfcHk0lhKfLWcdyCKuw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntlakt7s%253D&md5=0c58d15edacb68f7f4e1f014f079d8ac</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-108-6_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-108-6_3%26sid%3Dliteratum%253Aachs%26aulast%3DKupcsik%26aufirst%3DL.%26atitle%3DEstimation%2520of%2520cell%2520number%2520based%2520on%2520metabolic%2520activity%253A%2520the%2520MTT%2520reduction%2520assay%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D740%26spage%3D13%26epage%3D19%26doi%3D10.1007%2F978-1-61779-108-6_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mannhold, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span> <span> </span><span class="NLM_article-title">Calculation of molecular lipophilicity: state-of-the-art and comparison of Log P methods on more than 96,000 compounds</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">861</span>â <span class="NLM_lpage">893</span>, <span class="refDoi">Â DOI: 10.1002/jps.21494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fjps.21494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=18683876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1eisLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2009&pages=861-893&author=R.+Mannholdauthor=G.+I.+Podaauthor=C.+Ostermannauthor=I.+V.+Tetko&title=Calculation+of+molecular+lipophilicity%3A+state-of-the-art+and+comparison+of+Log+P+methods+on+more+than+96%2C000+compounds&doi=10.1002%2Fjps.21494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of molecular lipophilicity: state-of-the-art and comparison of logP methods on more than 96,000 compounds</span></div><div class="casAuthors">Mannhold, Raimund; Poda, Gennadiy I.; Ostermann, Cluade; Tetko, Igor V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">861-893</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  We first review the state-of-the-art in development of logP prediction approaches falling in two major categories: substructure-based and property-based methods.  Then, we compare the predictive power of representative methods for one public (N = 266) and two in house datasets from Nycomed (N = 882) and Pfizer (N = 95809).  A total of 30 and 18 methods were tested for public and industrial datasets, resp.  Accuracy of models declined with the no. of nonhydrogen atoms.  The Arithmetic Av. Model (AAM), which predicts the same value (the arithmetic mean) for all compds., was used as a baseline model for comparison.  Methods with Root Mean Squared Error (RMSE) greater than RMSE produced by the AAM were considered as unacceptable.  The majority of analyzed methods produced reasonable results for the public dataset but only seven methods were successful on the both in house datasets.  We proposed a simple equation based on the no. of carbon atoms, NC, and the no. of hetero atoms, NHET: logP = 1.46(+0.02) + 0.11(Â±0.001) NC - 0.11(Â±0.001) NHEET.  This equation outperformed a large no. of programs benchmarked in this study.  Factors influencing the accuracy of logP predictions were elucidated and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR3RAtmKwEZ7Vg90H21EOLACvtfcHk0lhKfLWcdyCKuw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1eisLo%253D&md5=8ae19379316c226dc75ec057006b0e8c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fjps.21494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21494%26sid%3Dliteratum%253Aachs%26aulast%3DMannhold%26aufirst%3DR.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DOstermann%26aufirst%3DC.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26atitle%3DCalculation%2520of%2520molecular%2520lipophilicity%253A%2520state-of-the-art%2520and%2520comparison%2520of%2520Log%2520P%2520methods%2520on%2520more%2520than%252096%252C000%2520compounds%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2009%26volume%3D98%26spage%3D861%26epage%3D893%26doi%3D10.1002%2Fjps.21494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42717</span>, <span class="refDoi">Â DOI: 10.1038/srep42717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fsrep42717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=28256516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42717&author=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissADME%3A+a+free+web+tool+to+evaluate+pharmacokinetics%2C+drug-likeness+and+medicinal+chemistry+friendliness+of+small+molecules&doi=10.1038%2Fsrep42717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span></div><div class="casAuthors">Daina Antoine; Michielin Olivier; Zoete Vincent; Michielin Olivier; Michielin Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42717</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">To be effective as a drug, a potent molecule must reach its target in the body in sufficient concentration, and stay there in a bioactive form long enough for the expected biologic events to occur.  Drug development involves assessment of absorption, distribution, metabolism and excretion (ADME) increasingly earlier in the discovery process, at a stage when considered compounds are numerous but access to the physical samples is limited.  In that context, computer models constitute valid alternatives to experiments.  Here, we present the new SwissADME web tool that gives free access to a pool of fast yet robust predictive models for physicochemical properties, pharmacokinetics, drug-likeness and medicinal chemistry friendliness, among which in-house proficient methods such as the BOILED-Egg, iLOGP and Bioavailability Radar.  Easy efficient input and interpretation are ensured thanks to a user-friendly interface through the login-free website http://www.swissadme.ch.  Specialists, but also nonexpert in cheminformatics or computational chemistry can predict rapidly key parameters for a collection of molecules to support their drug discovery endeavours.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRl_Uk5dOMKs6qcneEtCi92fW6udTcc2eZ8LKSHzn10KLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D&md5=9715b8cb8a34b17c4c73ff69a5a8cc50</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fsrep42717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep42717%26sid%3Dliteratum%253Aachs%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissADME%253A%2520a%2520free%2520web%2520tool%2520to%2520evaluate%2520pharmacokinetics%252C%2520drug-likeness%2520and%2520medicinal%2520chemistry%2520friendliness%2520of%2520small%2520molecules%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D42717%26doi%3D10.1038%2Fsrep42717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lerner-Marmarosh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senior, A. E.</span></span> <span> </span><span class="NLM_article-title">Large scale purification of detergent-soluble P-glycoprotein from Pichia pastoris cells and characterization of nucleotide binding properties of wild-type, Walker A, and Walker B mutant proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">34711</span>â <span class="NLM_lpage">34718</span>, <span class="refDoi">Â DOI: 10.1074/jbc.274.49.34711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1074%2Fjbc.274.49.34711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10574938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK1MXnvFyrs7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=34711-34718&author=N.+Lerner-Marmaroshauthor=K.+Gimiauthor=I.+L.+Urbatschauthor=P.+Grosauthor=A.+E.+Senior&title=Large+scale+purification+of+detergent-soluble+P-glycoprotein+from+Pichia+pastoris+cells+and+characterization+of+nucleotide+binding+properties+of+wild-type%2C+Walker+A%2C+and+Walker+B+mutant+proteins&doi=10.1074%2Fjbc.274.49.34711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Large scale purification of detergent-soluble P-glycoprotein from Pichia pastoris cells and characterization of nucleotide binding properties of wild-type, Walker A, and Walker B mutant proteins</span></div><div class="casAuthors">Lerner-Marmarosh, Nicole; Gimi, Khursheed; Urbatsch, Ina L.; Gros, Philippe; Senior, Alan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">34711-34718</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp; mouse MDR3) was expressed in Pichia pastoris, grown in fermentor culture, and purified.  The final pure product is of high specific ATPase activity and is sol. at low detergent concn.  120 G of cells yielded 6 mg of pure Pgp; >4 kg of cells were obtained from a single fermentor run.  Properties of the pure protein were similar to those of previous prepns., except there was significant ATPase activity in absence of added lipid.  Mutant mouse MDR3 P-glycoproteins were purified by the same procedure after growth of cells in flask culture, with similar yields and purity.  This procedure should open up new avenues of structural, biophys., and biochem. studies of Pgp.  Equil. nucleotide-binding parameters of wild-type mouse MDR3 Pgp were studied using 2'-(3')-O-(2,4,6-trinitrophenyl)adenosine tri- and diphosphate.  Both analogs were found to bind with Kd in the low micromolar range, to a single class of site, with no evidence of cooperativity.  ATP displacement of the analogs was seen.  Similar binding was seen with K429R/K1072R and D551N/D1196N mutant mouse MDR3 Pgp, showing that these Walker A and B mutations had no significant effect on affinity or stoichiometry of nucleotide binding.  These residues, known to be crit. for catalysis, are concluded to be involved primarily in stabilization of the catalytic transition state in Pgp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWFsSz4RG5frVg90H21EOLACvtfcHk0lhVLj4MHGh1SQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvFyrs7k%253D&md5=2d3b3f025fc8a18438f26f9b3f705e68</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.49.34711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.49.34711%26sid%3Dliteratum%253Aachs%26aulast%3DLerner-Marmarosh%26aufirst%3DN.%26aulast%3DGimi%26aufirst%3DK.%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DSenior%26aufirst%3DA.%2BE.%26atitle%3DLarge%2520scale%2520purification%2520of%2520detergent-soluble%2520P-glycoprotein%2520from%2520Pichia%2520pastoris%2520cells%2520and%2520characterization%2520of%2520nucleotide%2520binding%2520properties%2520of%2520wild-type%252C%2520Walker%2520A%252C%2520and%2520Walker%2520B%2520mutant%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D34711%26epage%3D34718%26doi%3D10.1074%2Fjbc.274.49.34711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tombline, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muharemagic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senior, A. E.</span></span> <span> </span><span class="NLM_article-title">Expression, purification, and characterization of cysteine-free mouse P-glycoprotein</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>445</i></span>,  <span class="NLM_fpage">124</span>â <span class="NLM_lpage">128</span>, <span class="refDoi">Â DOI: 10.1016/j.abb.2005.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.abb.2005.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=16343415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlGisLfM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=445&publication_year=2006&pages=124-128&author=G.+Tomblineauthor=I.+L.+Urbatschauthor=N.+Virkauthor=A.+Muharemagicauthor=L.+B.+Whiteauthor=A.+E.+Senior&title=Expression%2C+purification%2C+and+characterization+of+cysteine-free+mouse+P-glycoprotein&doi=10.1016%2Fj.abb.2005.11.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification, and characterization of cysteine-free mouse P-glycoprotein</span></div><div class="casAuthors">Tombline, Gregory; Urbatsch, Ina L.; Virk, Navneet; Muharemagic, Alma; White, Lori Bartholomew; Senior, Alan E.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">445</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-128</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cysteine-free mouse MDR3 P-glycoprotein (Pgp) was constructed by mutagenesis of the 9 naturally occurring Cys residues to Ala.  The Cys-free protein was expressed in Pichia pastoris and purified.  The yield, purity, ATPase activity, Km(MgATP), and stimulation of ATPase by verapamil, were similar to that wild-type mouse Ppg.  Mouse Cys-free Pgp was superior in yield and stability to Cys-free human MDR1 Pgp.  Mutants Y1040A and Y1040C were constructed in mouse Cys-free Pgp background.  Both showed extremely low ATPase activity, strongly-impaired vanadate-trapping of ADP, and reduced photolabeling by 8-azido-ATP.  The results were consistent with the conclusion that Tyr-1040 was located in the MgATP-binding site in NBD2 and was required for correct binding and/or orientation of bound MgATP substrate in Pgp as previously suggested by x-ray structures of other ABC transporters and by sequencing of photolabeled Pgp.  The results also support the authors' previous conclusion that both catalytic sites must be intact for normal function in Pgp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTS_AdDoiAk7Vg90H21EOLACvtfcHk0lhVLj4MHGh1SQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlGisLfM&md5=42055a06913d3137948e6635e65dfb23</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2005.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2005.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DTombline%26aufirst%3DG.%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DVirk%26aufirst%3DN.%26aulast%3DMuharemagic%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DL.%2BB.%26aulast%3DSenior%26aufirst%3DA.%2BE.%26atitle%3DExpression%252C%2520purification%252C%2520and%2520characterization%2520of%2520cysteine-free%2520mouse%2520P-glycoprotein%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2006%26volume%3D445%26spage%3D124%26epage%3D128%26doi%3D10.1016%2Fj.abb.2005.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berridge, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callaghan, R.</span></span> <span> </span><span class="NLM_article-title">The molecular interaction of the high affinity reversal agent XR9576 with P-glycoprotein</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">403</span>â <span class="NLM_lpage">411</span>, <span class="refDoi">Â DOI: 10.1038/sj.bjp.0702807</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fsj.bjp.0702807" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10510451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK1MXmsFWgs70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=1999&pages=403-411&author=C.+Martinauthor=G.+Berridgeauthor=P.+Mistryauthor=C.+Higginsauthor=P.+Charltonauthor=R.+Callaghan&title=The+molecular+interaction+of+the+high+affinity+reversal+agent+XR9576+with+P-glycoprotein&doi=10.1038%2Fsj.bjp.0702807"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular interaction of the high affinity reversal agent XR9576 with P-glycoprotein</span></div><div class="casAuthors">Martin, Catherine; Berridge, Georgina; Mistry, Prakash; Higgins, Christopher; Charlton, Peter; Callaghan, Richard</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">403-411</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">1 The kinetics and nature of equil. binding were used to characterize the mol. interaction of the anthranilic acid deriv. [3H]-XR9576 with the multidrug resistance P-glycoprotein (P-gp).  XR9576 displayed specific high-affinity binding to P-gp (Bmax = 275 pmol mg-1, Kd=5.1 nM).  The transport substrates [3H]-vinblastine and [3H]-paclitaxel displayed 4 fold and 20 fold lower affinity resp. for P-gp.  The duration of action of XR9576 with P-gp was increased in comparison to that of vinblastine which displayed a slower rate of assocn. and a faster dissocn. rate.  2 The relative affinities of several modulators and transport substrates to interact with P-gp were detd. from displacement drug equil. binding assays.  Vinblastine and paclitaxel could only fractionally displace [3H]-XR9576 binding, displaying Ki values significantly different from their measured Kd values.  This suggests a non-competitive interaction between XR9576 and the P-gp substrates vinblastine and paclitaxel.  3 XR9576 was shown to be a potent modulator of P-gp mediated [3H]-vinblastine and [3H]-paclitaxel transport as it increased the steady-state accumulation of these cytotoxics in CHrB30 cells to levels obsd. in non-P-gp-expressing AuxB1 cells (EC50 = 487 Â± 50 nM).  This inhibition of drug transport is not mediated through competition for transport since [3H]-XR9576 accumulation was not influenced by P-gp expression or function.  4 These results demonstrate that the P-gp modulator XR9576 exhibits greater selectivity, duration of inhibition and potency of interaction with this transporter than any other reported modulators.  Several lines of evidence suggest that XR9576 inhibits P-gp function by binding at a site which is distinct from the site of interaction of transport substrates.  The two sites may be classified as serving modulatory or transport functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6HZ8wWLNGErVg90H21EOLACvtfcHk0lgdLG-Bn8_IiQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmsFWgs70%253D&md5=5d80afb78a34d30b4947e972d486a584</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0702807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0702807%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DBerridge%26aufirst%3DG.%26aulast%3DMistry%26aufirst%3DP.%26aulast%3DHiggins%26aufirst%3DC.%26aulast%3DCharlton%26aufirst%3DP.%26aulast%3DCallaghan%26aufirst%3DR.%26atitle%3DThe%2520molecular%2520interaction%2520of%2520the%2520high%2520affinity%2520reversal%2520agent%2520XR9576%2520with%2520P-glycoprotein%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1999%26volume%3D128%26spage%3D403%26epage%3D411%26doi%3D10.1038%2Fsj.bjp.0702807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Streckenbach, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repke, K. R. H.</span></span> <span> </span><span class="NLM_article-title">Analysis of phosphoryl transfer mechanism and catalytic centre geometries of transport ATPase by means of spin-labelled ATP</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>601</i></span>,  <span class="NLM_fpage">34</span>â <span class="NLM_lpage">46</span>, <span class="refDoi">Â DOI: 10.1016/0005-2736(80)90511-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2F0005-2736%2880%2990511-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=6250610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaL3cXmtVSjtbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=601&publication_year=1980&pages=34-46&author=B.+Streckenbachauthor=D.+Schwarzauthor=K.+R.+H.+Repke&title=Analysis+of+phosphoryl+transfer+mechanism+and+catalytic+centre+geometries+of+transport+ATPase+by+means+of+spin-labelled+ATP&doi=10.1016%2F0005-2736%2880%2990511-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of phosphoryl transfer mechanism and catalytic center geometries of transport ATPase by means of spin-labelled ATP</span></div><div class="casAuthors">Streckenbach, Barbara; Schwarz, Dieter; Repke, Kurt R. H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">601</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-46</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    </div><div class="casAbstract">In the presence of Na+, K+ and Mg2+, SL-ATP [3'-O-(1-oxyl-2,2,5,5-tetramethyl-3-carbonylpyrrolidine)-ATP] is not hydrolyzed by transport ATPase from 3 different sources.  In the presence of Na+ and Mg2+, SL-ATP reacts initially like ATP with the enzyme, as indicated by the prodn. of a similar ouabain-binding protein conformation.  With both nucleotides, this initial reaction includes the formation of the covalent enzyme-nucleotide complex through nucleophilic attack of the aspartate carboxyanion of the catalytic center on the terminal P atom of the triphosphate chain.  This produces the ouabain-binding conformation of the enzyme.  Unlike ATP, the covalent enzyme-SL-ATP complex resists further transformation.  In the presence of Na+, K+, and Mg2+, the influence of SL-ATP on ATP hydrolysis by transport ATPase depends on the ATP concn. chosen.  At low ATP concn., when the enzyme works as a Na+-ATPase, SL-ATP does not affect the rate of ATP cleavage.  At high ATP concn., however, when the enzyme works as a (Na+ + K+)-ATPase, SL-ATP reduces the rate of ATP hydrolysis to the level of Na+-ATPase activity, apparently due to the formation of the covalent enzyme-SL-ATP complex.  SL-ATP in the covalent enzyme-SL-ATP complex shows an ESR spectrum which is indistinguishable regarding the overall shape, the rotational correlation time, Ï, and the hyperfine coupling const., aN, from the ESR spectrum of free SL-ATP.  Consequently, the dimensions of the catalytic center cleft of transport ATPase provide the labeled group of SL-ATP, opposite to its 3'-O-esterification site at the ribose moiety, in a wide-cleft groove, enough free space for an essentially unhindered rotational mobility within an aq. environment, like that of the bulk medium.  Judged from literature data, similarly wide grooves exist in the catalytic center clefts of mitochondrial and myosin ATPases.  Thus, the structural unit forming the binding site for the AMP moiety of ATP in ATPases is similar to the structural unit forming the binding site for the AMP moiety of NAD and ADP in several dehydrogenases and kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-pIx38yRv8rVg90H21EOLACvtfcHk0lgdLG-Bn8_IiQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXmtVSjtbk%253D&md5=e6ef56b5f34855a0ebe74f3bb5daaedf</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0005-2736%2880%2990511-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2736%252880%252990511-8%26sid%3Dliteratum%253Aachs%26aulast%3DStreckenbach%26aufirst%3DB.%26aulast%3DSchwarz%26aufirst%3DD.%26aulast%3DRepke%26aufirst%3DK.%2BR.%2BH.%26atitle%3DAnalysis%2520of%2520phosphoryl%2520transfer%2520mechanism%2520and%2520catalytic%2520centre%2520geometries%2520of%2520transport%2520ATPase%2520by%2520means%2520of%2520spin-labelled%2520ATP%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1980%26volume%3D601%26spage%3D34%26epage%3D46%26doi%3D10.1016%2F0005-2736%2880%2990511-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delannoy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tombline, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senior, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span> <span> </span><span class="NLM_article-title">Nucleotide binding to the multidrug resistance P-glycoprotein as studied by ESR spectroscopy</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">14010</span>â <span class="NLM_lpage">14019</span>, <span class="refDoi">Â DOI: 10.1021/bi0512445</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0512445" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVGgtbbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=14010-14019&author=S.+Delannoyauthor=I.+L.+Urbatschauthor=G.+Tomblineauthor=A.+E.+Seniorauthor=P.+D.+Vogel&title=Nucleotide+binding+to+the+multidrug+resistance+P-glycoprotein+as+studied+by+ESR+spectroscopy&doi=10.1021%2Fbi0512445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleotide Binding to the Multidrug Resistance P-Glycoprotein as Studied by ESR Spectroscopy</span></div><div class="casAuthors">Delannoy, Sabine; Urbatsch, Ina L.; Tombline, Gregory; Senior, Alan E.; Vogel, Pia D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">14010-14019</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ESR spectroscopy using spin-labeled ATP was used to study nucleotide binding to and structural transitions within the multidrug resistance (MDR) P-glycoprotein, P-gp.  Spin-labeled ATP (SL-ATP) with the spin label attached to the ribose, was obsd. to be an excellent substrate analog for P-gp.  SL-ATP was hydrolyzed in a drug-stimulated fashion at about 14% of the rate for normal ATP and allowed reversible trapping of the enzyme in transition and ground states.  Equil. binding of a total of two nucleotides per P-gp was obsd. with a binding affinity of 366 Î¼M in the presence of Mg2+ but in the absence of transport substrates such as verapamil.  Binding of SL-ATP to wild-type P-gp in the presence of verapamil resulted in redn. of the protein-bound spin-label moiety, most likely due to a conformational transition within P-gp that positioned cysteines in close proximity to the spin label to allow chem. redn. of the radical.  We circumvented this problem by using a mutant of P-gp in which all naturally occurring cysteines were substituted for alanines.  Equil. binding of SL-ATP to this mutant P-gp resulted in max. binding of two nucleotides; the binding affinity was 223 Î¼M in the absence and 180 Î¼M in the presence of verapamil.  The corresponding ESR spectra of wild-type and Cys-less P-gp in the presence of SL-ATP indicate that a cysteine side chain of P-gp is located close to the ribose of the bound nucleotide.  Trapping SL-ATP as an AlFx-adduct resulted in ESR spectra that showed strong immobilization of the radical, supporting the formation of a closed conformation of P-gp in its transition state.  This study is the first to employ ESR spectroscopy with the use of spin-labeled nucleotide analogs to study P-glycoprotein.  The study shows that SL-ATP is an excellent substrate analog that will allow further exploration of structure and dynamics within the nucleotide binding domains of this important enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-N7KS3FsbLVg90H21EOLACvtfcHk0lgdLG-Bn8_IiQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVGgtbbP&md5=41f0dd9fdffa43775707de5fbeb71d7e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fbi0512445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0512445%26sid%3Dliteratum%253Aachs%26aulast%3DDelannoy%26aufirst%3DS.%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DTombline%26aufirst%3DG.%26aulast%3DSenior%26aufirst%3DA.%2BE.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26atitle%3DNucleotide%2520binding%2520to%2520the%2520multidrug%2520resistance%2520P-glycoprotein%2520as%2520studied%2520by%2520ESR%2520spectroscopy%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D14010%26epage%3D14019%26doi%3D10.1021%2Fbi0512445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nett, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmadel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trommer, W. E.</span></span> <span> </span><span class="NLM_article-title">Nucleotide binding sites on mitochondrial F1-ATPase. Electron spin resonance spectroscopy and photolabeling by azido-spin-labeled adenine nucleotides support an adenylate kinase-like orientation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">11982</span>â <span class="NLM_lpage">11986</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1318307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK38Xls1KmtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1992&pages=11982-11986&author=P.+D.+Vogelauthor=J.+H.+Nettauthor=H.+E.+Sauerauthor=K.+Schmadelauthor=R.+L.+Crossauthor=W.+E.+Trommer&title=Nucleotide+binding+sites+on+mitochondrial+F1-ATPase.+Electron+spin+resonance+spectroscopy+and+photolabeling+by+azido-spin-labeled+adenine+nucleotides+support+an+adenylate+kinase-like+orientation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleotide binding sites on mitochondrial F1-ATPase.  Electron spin resonance spectroscopy and photolabeling by azido-spin-labeled adenine nucleotides support an adenylate kinase-like orientation</span></div><div class="casAuthors">Vogel, Pia D.; Nett, Juergen H.; Sauer, Hubertus E.; Schmadel, Klaus; Cross, Richard L.; Trommer, Wolfgang E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11982-6</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">A spin-labeled photoaffinity ATP analog, 2-azido-2',3'-SL-ATP (2-Na3-SL-ATP; SL = 2,2,5,5-tetramethyl-3-pyrroline-1-oxyl-3-carboxylic acid ester) was specifically loaded at catalytic (exchangeable) or noncatalytic (nonexchangeable) nucleotide-binding sites on nucleotide-depleted mitochondrial F1-ATPase.  Photolysis of the enzyme complexes resulted in the specific modification of Î²-subunit Tyr-345 when the catalytic sites were occupied and Î²-subunit Tyr-368 when noncatalytic sites were filled.  These were the same amino acid assignments that were made previously using 2-azido-ATP.  The results demonstrated that the attachment of a spin label moiety to the ribose ring did not prevent proper binding of the analog at both types of nucleotide sites on F1-ATPase and suggested that the probe could be used for investigations of the nucleotide-binding sites by ESR spectroscopy.  The enzyme that was in complex with 2-N3-SL-ATP exhibited an ESR spectrum that was typical for highly immobilized nitroxyl radicals both in the dark or after photolysis.  Addnl. peaks in the high- and low-field regions arose due to dipolar spin interactions most likely involving a pair of catalytic and noncatalytic sites.  The 2 sites were calcd. to be â¼15 Ã apart.  This distance, obtained through ESR spectroscopy, combined with the finding that the 2 labeled amino acids were only 23 residues apart from each other, further supported an adenylate kinase-like arrangement of nucleotide binding sites on F1-ATPase where catalytic and noncatalytic sites are in close proximity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5m8Y7VQ_1pLVg90H21EOLACvtfcHk0lgdLG-Bn8_IiQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xls1KmtLY%253D&md5=5d859eae7b9ab881dbd1c4b7980ae696</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DP.%2BD.%26aulast%3DNett%26aufirst%3DJ.%2BH.%26aulast%3DSauer%26aufirst%3DH.%2BE.%26aulast%3DSchmadel%26aufirst%3DK.%26aulast%3DCross%26aufirst%3DR.%2BL.%26aulast%3DTrommer%26aufirst%3DW.%2BE.%26atitle%3DNucleotide%2520binding%2520sites%2520on%2520mitochondrial%2520F1-ATPase.%2520Electron%2520spin%2520resonance%2520spectroscopy%2520and%2520photolabeling%2520by%2520azido-spin-labeled%2520adenine%2520nucleotides%2520support%2520an%2520adenylate%2520kinase-like%2520orientation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1992%26volume%3D267%26spage%3D11982%26epage%3D11986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>â <span class="NLM_lpage">461</span>, <span class="refDoi">Â DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhM90EZnctpMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Automated docking of flexible ligands: applications of AutoDock</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">5</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1099-1352(199601)9:1<1::AID-JMR241>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2F%28SICI%291099-1352%28199601%299%3A1%3C1%3A%3AAID-JMR241%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=8723313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK28Xis1SqtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=1-5&author=D.+S.+Goodsellauthor=G.+M.+Morrisauthor=A.+J.+Olson&title=Automated+docking+of+flexible+ligands%3A+applications+of+AutoDock&doi=10.1002%2F%28SICI%291099-1352%28199601%299%3A1%3C1%3A%3AAID-JMR241%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking of flexible ligands: application of AutoDock</span></div><div class="casAuthors">Goodsell, David S.; Morris, Garrett M.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-5</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 23 refs. AutoDock is a suite of C programs used to predict the bound conformations of a small, flexible ligand to a macromol. target of known structure.  The technique combines simulated annealing for conformation searching with a rapid grid-based method of energy evaluation.  This paper reviews recent applications of the technique and describes the enhancement included in the current release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHBbJMLK-ctbVg90H21EOLACvtfcHk0lhM90EZnctpMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis1SqtrY%253D&md5=8f8f3964ba0c51da8f8b351e65a40a59</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291099-1352%28199601%299%3A1%3C1%3A%3AAID-JMR241%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291099-1352%2528199601%25299%253A1%253C1%253A%253AAID-JMR241%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutomated%2520docking%2520of%2520flexible%2520ligands%253A%2520applications%2520of%2520AutoDock%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D1996%26volume%3D9%26spage%3D1%26epage%3D5%26doi%3D10.1002%2F%28SICI%291099-1352%28199601%299%3A1%3C1%3A%3AAID-JMR241%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Distributed automated docking of flexible ligands to proteins: parallel applications of AutoDock 2.4</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">293</span>â <span class="NLM_lpage">304</span>, <span class="refDoi">Â DOI: 10.1007/BF00124499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1007%2FBF00124499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=8877701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK28XlsVOhsL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1996&pages=293-304&author=G.+M.+Morrisauthor=D.+S.+Goodsellauthor=R.+Hueyauthor=A.+J.+Olson&title=Distributed+automated+docking+of+flexible+ligands+to+proteins%3A+parallel+applications+of+AutoDock+2.4&doi=10.1007%2FBF00124499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Distributed automated docking of flexible ligands to proteins: parallel applications of AutoDock 2.4</span></div><div class="casAuthors">Morris, Garrett M.; Goodsell, David S.; Huey, Ruth; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-304</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">AutoDock 2.4 predicts the bound conformations of a small, flexible ligand to a nonflexible macromol. target of known structure.  The technique combines simulated annealing for conformation searching with a rapid grid-based method of energy evaluation based on the AMBER force field.  AutoDock has been optimized in performance without sacrificing accuracy; it incorporates many enhancements and addns., including an intuitive interface.  We have developed a set of tools for launching and analyzing many independent docking jobs in parallel on a heterogeneous network of UNIX-based workstations.  This paper describes the current release, and the results of a suite of diverse test systems.  We also present the results of a systematic investigation in to the effects of varying simulated-annealing parameters on mol. docking.  We show that even for ligands with a large no. of degrees of freedom, root-mean-square deviations of less than 1 Ã from the crystallog. conformation are obtained for the lowest-energy dockings, although fewer dockings find the crystallog. conformation when there are more degrees of freedom.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHQmD1We2g6LVg90H21EOLACvtfcHk0lhM90EZnctpMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XlsVOhsL0%253D&md5=0e88e41a8db0b67cef118620d0992452</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2FBF00124499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00124499%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DDistributed%2520automated%2520docking%2520of%2520flexible%2520ligands%2520to%2520proteins%253A%2520parallel%2520applications%2520of%2520AutoDock%25202.4%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D1996%26volume%3D10%26spage%3D293%26epage%3D304%26doi%3D10.1007%2FBF00124499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ãsterberg, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span> <span> </span><span class="NLM_article-title">Automated docking to multiple target structures: incorporation of protein mobility and structural water heterogeneity in AutoDock</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">34</span>â <span class="NLM_lpage">40</span>, <span class="refDoi">Â DOI: 10.1002/prot.10028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fprot.10028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=11746701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BD3MnpvVOlsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=34-40&author=F.+%C3%96sterbergauthor=G.+M.+Morrisauthor=M.+F.+Sannerauthor=A.+J.+Olsonauthor=D.+S.+Goodsell&title=Automated+docking+to+multiple+target+structures%3A+incorporation+of+protein+mobility+and+structural+water+heterogeneity+in+AutoDock&doi=10.1002%2Fprot.10028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking to multiple target structures: incorporation of protein mobility and structural water heterogeneity in AutoDock</span></div><div class="casAuthors">Osterberg Fredrik; Morris Garrett M; Sanner Michel F; Olson Arthur J; Goodsell David S</div><div class="citationInfo"><span class="NLM_cas:title">Proteins</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-40</span>
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">Protein motion and heterogeneity of structural waters are approximated in ligand-docking simulations, using an ensemble of protein structures.  Four methods of combining multiple target structures within a single grid-based lookup table of interaction energies are tested.  The method is evaluated using complexes of 21 peptidomimetic inhibitors with human immunodeficiency virus type 1 (HIV-1) protease.  Several of these structures show motion of an arginine residue, which is essential for binding of large inhibitors.  A structural water is also present in 20 of the structures, but it must be absent in the remaining one for proper binding.  Mean and minimum methods perform poorly, but two weighted average methods permit consistent and accurate ligand docking, using a single grid representation of the target protein structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVWVkt39Ar86Nc3HjJRoFAfW6udTcc2ebLywYj9OJS9bntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3MnpvVOlsg%253D%253D&md5=4ae2dd1cf13b1e0e0af7f41ddbfdc8ed</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fprot.10028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10028%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596sterberg%26aufirst%3DF.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26atitle%3DAutomated%2520docking%2520to%2520multiple%2520target%2520structures%253A%2520incorporation%2520of%2520protein%2520mobility%2520and%2520structural%2520water%2520heterogeneity%2520in%2520AutoDock%26jtitle%3DProteins%26date%3D2002%26volume%3D46%26spage%3D34%26epage%3D40%26doi%3D10.1002%2Fprot.10028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, R. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locher, K. P.</span></span> <span> </span><span class="NLM_article-title">Structure of a bacterial multidrug ABC transporter</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>443</i></span>,  <span class="NLM_fpage">180</span>â <span class="NLM_lpage">185</span>, <span class="refDoi">Â DOI: 10.1038/nature05155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fnature05155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=16943773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVansLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=443&publication_year=2006&pages=180-185&author=R.+J.+P.+Dawsonauthor=K.+P.+Locher&title=Structure+of+a+bacterial+multidrug+ABC+transporter&doi=10.1038%2Fnature05155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a bacterial multidrug ABC transporter</span></div><div class="casAuthors">Dawson, Roger J. P.; Locher, Kaspar P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">443</span>
        (<span class="NLM_cas:issue">7108</span>),
    <span class="NLM_cas:pages">180-185</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Multidrug transporters of the ABC family facilitate the export of diverse cytotoxic drugs across cell membranes.  This is clin. relevant, as tumor cells may become resistant to agents used in chemotherapy.  To understand the mol. basis of this process, we have detd. the 3.0 Ã crystal structure of a bacterial ABC transporter (Sav1866) from Staphylococcus aureus.  The homodimeric protein consists of 12 transmembrane helixes in an arrangement that is consistent with crosslinking studies and electron microscopic imaging of the human multidrug resistance protein MDR1, but critically different from that reported for the bacterial lipid flippase MsbA.  The obsd., outward-facing conformation reflects the ATP-bound state, with the two nucleotide-binding domains in close contact and the two transmembrane domains forming a central cavity-presumably the drug translocation pathway-that is shielded from the inner leaflet of the lipid bilayer and from the cytoplasm, but exposed to the outer leaflet and the extracellular space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4WSMSCWPqt7Vg90H21EOLACvtfcHk0lgXPBmAUYCcCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVansLY%253D&md5=a506c0275d59882632fc7fb5714dc3fc</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnature05155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05155%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DR.%2BJ.%2BP.%26aulast%3DLocher%26aufirst%3DK.%2BP.%26atitle%3DStructure%2520of%2520a%2520bacterial%2520multidrug%2520ABC%2520transporter%26jtitle%3DNature%26date%3D2006%26volume%3D443%26spage%3D180%26epage%3D185%26doi%3D10.1038%2Fnature05155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">VMD: visual molecular dynamics</span>. <i>J. Mol. Graph.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">33</span>â <span class="NLM_lpage">38</span>, <span class="refDoi">Â DOI: 10.1016/0263-7855(96)00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2F0263-7855%2896%2900018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=8744570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=33-38&author=W.+Humphreyauthor=A.+Dalkeauthor=K.+Schulten&title=VMD%3A+visual+molecular+dynamics&doi=10.1016%2F0263-7855%2896%2900018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">VDM: visual molecular dynamics</span></div><div class="casAuthors">Humphrey, William; Dalke, Andrew; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8, plates, 27-28</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">VMD is a mol. graphics program designed for the display and anal. of mol. assemblies, in particular, biopolymers such as proteins and nucleic acids.  VMD can simultaneously display any no. of structures using a wide variety of rendering styles and coloring methods.  Mols. are displayed as one or more "representations," in which each representation embodies a particular rendering method and coloring scheme for a selected subset of atoms.  The atoms displayed in each representation are chosen using an extensive atom selection syntax, which includes Boolean operators and regular expressions.  VMD provides a complete graphical user interface for program control, as well as a text interface using the Tcl embeddable parser to allow for complex scripts with variable substitution, control loops, and function calls.  Full session logging is supported, which produces a VMD command script for later playback.  High-resoln. raster images of displayed mols. may be produced by generating input scripts for use by a no. of photorealistic image-rendering applications.  VMD has also been expressly designed with the ability to animate mol. dynamics (MD) simulation trajectories, imported either from files or from a direct connection to a running MD simulation.  VMD is the visualization component of MDScope, a set of tools for interactive problem solving in structural biol., which also includes the parallel MD program NAMD, and the MDCOMM software used to connect the visualization and simulation programs, VMD is written in C++, using an object-oriented design; the program, including source code and extensive documentation, is freely available via anonymous ftp and through the World Wide Web.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0dd54QvBO97Vg90H21EOLACvtfcHk0lgXPBmAUYCcCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D&md5=1e3094ec3151fb85c5ff05f8505c78d5</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900018-5%26sid%3Dliteratum%253Aachs%26aulast%3DHumphrey%26aufirst%3DW.%26aulast%3DDalke%26aufirst%3DA.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DVMD%253A%2520visual%2520molecular%2520dynamics%26jtitle%3DJ.%2520Mol.%2520Graph.%26date%3D1996%26volume%3D14%26spage%3D33%26epage%3D38%26doi%3D10.1016%2F0263-7855%2896%2900018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varshney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, W. V.</span></span> <span> </span><span class="NLM_article-title">Computing smooth molecular surfaces</span>. <i>IEEE Comput. Graphics Appl.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">19</span>â <span class="NLM_lpage">25</span>, <span class="refDoi">Â DOI: 10.1109/38.310720</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1109%2F38.310720" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=19-25&author=A.+Varshneyauthor=F.+P.+Brooksauthor=W.+V.+Wright&title=Computing+smooth+molecular+surfaces&doi=10.1109%2F38.310720"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1109%2F38.310720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252F38.310720%26sid%3Dliteratum%253Aachs%26aulast%3DVarshney%26aufirst%3DA.%26aulast%3DBrooks%26aufirst%3DF.%2BP.%26aulast%3DWright%26aufirst%3DW.%2BV.%26atitle%3DComputing%2520smooth%2520molecular%2520surfaces%26jtitle%3DIEEE%2520Comput.%2520Graphics%2520Appl.%26date%3D1994%26volume%3D14%26spage%3D19%26epage%3D25%26doi%3D10.1109%2F38.310720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolinsky, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span> <span> </span><span class="NLM_article-title">PDB2PQR: an automated pipeline for the setup of PoissonâBoltzmann electrostatics calculations</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">W665</span>â <span class="NLM_lpage">W667</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkh381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1093%2Fnar%2Fgkh381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=15215472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W665-W667&author=T.+J.+Dolinskyauthor=J.+E.+Nielsenauthor=J.+A.+McCammonauthor=N.+A.+Baker&title=PDB2PQR%3A+an+automated+pipeline+for+the+setup+of+Poisson%E2%80%93Boltzmann+electrostatics+calculations&doi=10.1093%2Fnar%2Fgkh381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">PDB2PQR: An automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations</span></div><div class="casAuthors">Dolinsky, Todd J.; Nielsen, Jens E.; McCammon, J. Andrew; Baker, Nathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W665-W667</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Continuum solvation models, such as Poisson-Boltzmann and Generalized Born methods, have become increasingly popular tools for investigating the influence of electrostatics on biomol. structure, energetics and dynamics.  However, the use of such methods requires accurate and complete structural data as well as force field parameters such as at. charges and radii.  Unfortunately, the limiting step in continuum electrostatics calcns. is often the addn. of missing at. coordinates to mol. structures from the Protein Data Bank and the assignment of parameters to biomol. structures.  To address this problem, we have developed the PDB2PQR web service (http://agave.wustl.edu/pdb2pqr/).  This server automates many of the common tasks of prepg. structures for continuum electrostatics calcns., including adding a limited no. of missing heavy atoms to biomol. structures, estg. titrn. states and protonating biomols. in a manner consistent with favorable hydrogen bonding, assigning charge and radius parameters from a variety of force fields, and finally generating "PQR" output compatible with several popular computational biol. packages.  This service is intended to facilitate the setup and execution of electrostatics calcns. for both experts and non-experts and thereby broaden the accessibility to the biol. community of continuum electrostatics analyses of biomol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-GebFAmzzkrVg90H21EOLACvtfcHk0lhhKDcpJQCyLA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsbY%253D&md5=678d2d58f5cb2c1dd862e2d71cf9dbe1</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh381%26sid%3Dliteratum%253Aachs%26aulast%3DDolinsky%26aufirst%3DT.%2BJ.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DBaker%26aufirst%3DN.%2BA.%26atitle%3DPDB2PQR%253A%2520an%2520automated%2520pipeline%2520for%2520the%2520setup%2520of%2520Poisson%25E2%2580%2593Boltzmann%2520electrostatics%2520calculations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW665%26epage%3DW667%26doi%3D10.1093%2Fnar%2Fgkh381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolinsky, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span> <span> </span><span class="NLM_article-title">PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W522</span>â <span class="NLM_lpage">W525</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkm276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1093%2Fnar%2Fgkm276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=17488841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGltQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W522-W525&author=T.+J.+Dolinskyauthor=P.+Czodrowskiauthor=H.+Liauthor=J.+E.+Nielsenauthor=J.+H.+Jensenauthor=G.+Klebeauthor=N.+A.+Baker&title=PDB2PQR%3A+expanding+and+upgrading+automated+preparation+of+biomolecular+structures+for+molecular+simulations&doi=10.1093%2Fnar%2Fgkm276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations</span></div><div class="casAuthors">Dolinsky Todd J; Czodrowski Paul; Li Hui; Nielsen Jens E; Jensen Jan H; Klebe Gerhard; Baker Nathan A</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W522-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Real-world observable physical and chemical characteristics are increasingly being calculated from the 3D structures of biomolecules.  Methods for calculating pK(a) values, binding constants of ligands, and changes in protein stability are readily available, but often the limiting step in computational biology is the conversion of PDB structures into formats ready for use with biomolecular simulation software.  The continued sophistication and integration of biomolecular simulation methods for systems- and genome-wide studies requires a fast, robust, physically realistic and standardized protocol for preparing macromolecular structures for biophysical algorithms.  As described previously, the PDB2PQR web server addresses this need for electrostatic field calculations (Dolinsky et al., Nucleic Acids Research, 32, W665-W667, 2004).  Here we report the significantly expanded PDB2PQR that includes the following features: robust standalone command line support, improved pK(a) estimation via the PROPKA framework, ligand parameterization via PEOE_PB charge methodology, expanded set of force fields and easily incorporated user-defined parameters via XML input files, and improvement of atom addition and optimization code.  These features are available through a new web interface (http://pdb2pqr.sourceforge.net/), which offers users a wide range of options for PDB file conversion, modification and parameterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqIxhnOQkIHOxfW6udTcc2ebOYU00BxutCbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGltQ%253D%253D&md5=18883d2f9a6f30dba143a4a2ca80b0d8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm276%26sid%3Dliteratum%253Aachs%26aulast%3DDolinsky%26aufirst%3DT.%2BJ.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DBaker%26aufirst%3DN.%2BA.%26atitle%3DPDB2PQR%253A%2520expanding%2520and%2520upgrading%2520automated%2520preparation%2520of%2520biomolecular%2520structures%2520for%2520molecular%2520simulations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW522%26epage%3DW525%26doi%3D10.1093%2Fnar%2Fgkm276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sept, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holst, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Electrostatics of nanosystems: application to microtubules and the ribosome</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10037</span>â <span class="NLM_lpage">10041</span>, <span class="refDoi">Â DOI: 10.1073/pnas.181342398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1073%2Fpnas.181342398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=11517324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=10037-10041&author=N.+A.+Bakerauthor=D.+Septauthor=S.+Josephauthor=M.+J.+Holstauthor=J.+A.+McCammon&title=Electrostatics+of+nanosystems%3A+application+to+microtubules+and+the+ribosome&doi=10.1073%2Fpnas.181342398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatics of nanosystems: application to microtubules and the ribosome</span></div><div class="casAuthors">Baker, Nathan A.; Sept, David; Joseph, Simpson; Holst, Michael J.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10037-10041</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Evaluation of the electrostatic properties of biomols. has become a std. practice in mol. biophysics.  Foremost among the models used to elucidate the electrostatic potential is the Poisson-Boltzmann equation; however, existing methods for solving this equation have limited the scope of accurate electrostatic calcns. to relatively small biomol. systems.  Here we present the application of numerical methods to enable the trivially parallel soln. of the Poisson-Boltzmann equation for supramol. structures that are orders of magnitude larger in size.  As a demonstration of this methodol., electrostatic potentials have been calcd. for large microtubule and ribosome structures.  The results point to the likely role of electrostatics in a variety of activities of these structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_CAlhE0uX7Vg90H21EOLACvtfcHk0lhhKDcpJQCyLA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D&md5=1b861999ef12c6972e82e8ada0f387cb</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.181342398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.181342398%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DN.%2BA.%26aulast%3DSept%26aufirst%3DD.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DHolst%26aufirst%3DM.%2BJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DElectrostatics%2520of%2520nanosystems%253A%2520application%2520to%2520microtubules%2520and%2520the%2520ribosome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2001%26volume%3D98%26spage%3D10037%26epage%3D10041%26doi%3D10.1073%2Fpnas.181342398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jurrus, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Star, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felberg, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brookes, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liles, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohara, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolinsky, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konecny, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koes, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head-Gordon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kransky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holst, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCommon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span> <span> </span><span class="NLM_article-title">Improvements to the APBS biomolecular solvation software suite</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">112</span>â <span class="NLM_lpage">128</span>, <span class="refDoi">Â DOI: 10.1002/pro.3280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fpro.3280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=28836357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslSkt7vI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=112-128&author=E.+Jurrusauthor=D.+Engelauthor=K.+Starauthor=K.+Monsonauthor=J.+Brandiauthor=L.+E.+Felbergauthor=D.+H.+Brookesauthor=L.+Wilsonauthor=J.+Chenauthor=K.+Lilesauthor=M.+Chunauthor=P.+Liauthor=D.+W.+Goharaauthor=T.+Dolinskyauthor=R.+Konecnyauthor=D.+R.+Koesauthor=J.+E.+Nielsenauthor=T.+Head-Gordonauthor=W.+Gengauthor=R.+Kranskyauthor=G.-E.+Weiauthor=M.+J.+Holstauthor=J.+A.+McCommonauthor=N.+A.+Baker&title=Improvements+to+the+APBS+biomolecular+solvation+software+suite&doi=10.1002%2Fpro.3280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Improvements to the APBS biomolecular solvation software suite</span></div><div class="casAuthors">Jurrus, Elizabeth; Engel, Dave; Star, Keith; Monson, Kyle; Brandi, Juan; Felberg, Lisa E.; Brookes, David H.; Wilson, Leighton; Chen, Jiahui; Liles, Karina; Chun, Minju; Li, Peter; Gohara, David W.; Dolinsky, Todd; Konecny, Robert; Koes, David R.; Nielsen, Jens Erik; Head-Gordon, Teresa; Geng, Weihua; Krasny, Robert; Wei, Guo-Wei; Holst, Michael J.; McCammon, J. Andrew; Baker, Nathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-128</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Adaptive Poisson-Boltzmann Solver (APBS) software was developed to solve the equations of continuum electrostatics for large biomol. assemblages that have provided impact in the study of a broad range of chem., biol., and biomedical applications.  APBS addresses the three key technol. challenges for understanding solvation and electrostatics in biomedical applications: accurate and efficient models for biomol. solvation and electrostatics, robust and scalable software for applying those theories to biomol. systems, and mechanisms for sharing and analyzing biomol. electrostatics data in the scientific community.  To address new research applications and advancing computational capabilities, we have continually updated APBS and its suite of accompanying software since its release in 2001.  In this article, we discuss the models and capabilities that have recently been implemented within the APBS software package including a Poisson-Boltzmann anal. and a semi-anal. solver, an optimized boundary element solver, a geometry-based geometric flow solvation model, a graph theory-based algorithm for detg. pKa values, and an improved web-based visualization tool for viewing electrostatics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBO7hbYoyPZrVg90H21EOLACvtfcHk0ljXViNN7BKKNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslSkt7vI&md5=99651d125e38f4a85d453fecf0f71652</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fpro.3280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3280%26sid%3Dliteratum%253Aachs%26aulast%3DJurrus%26aufirst%3DE.%26aulast%3DEngel%26aufirst%3DD.%26aulast%3DStar%26aufirst%3DK.%26aulast%3DMonson%26aufirst%3DK.%26aulast%3DBrandi%26aufirst%3DJ.%26aulast%3DFelberg%26aufirst%3DL.%2BE.%26aulast%3DBrookes%26aufirst%3DD.%2BH.%26aulast%3DWilson%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLiles%26aufirst%3DK.%26aulast%3DChun%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DGohara%26aufirst%3DD.%2BW.%26aulast%3DDolinsky%26aufirst%3DT.%26aulast%3DKonecny%26aufirst%3DR.%26aulast%3DKoes%26aufirst%3DD.%2BR.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26aulast%3DHead-Gordon%26aufirst%3DT.%26aulast%3DGeng%26aufirst%3DW.%26aulast%3DKransky%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DG.-E.%26aulast%3DHolst%26aufirst%3DM.%2BJ.%26aulast%3DMcCommon%26aufirst%3DJ.%2BA.%26aulast%3DBaker%26aufirst%3DN.%2BA.%26atitle%3DImprovements%2520to%2520the%2520APBS%2520biomolecular%2520solvation%2520software%2520suite%26jtitle%3DProtein%2520Sci.%26date%3D2018%26volume%3D27%26spage%3D112%26epage%3D128%26doi%3D10.1002%2Fpro.3280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stone, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickey, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickey, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulson, D. F.</span></span> <span> </span><span class="NLM_article-title">Isolation of a human prostate carcinoma cell line (DU 145)</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">274</span>â <span class="NLM_lpage">281</span>, <span class="refDoi">Â DOI: 10.1002/ijc.2910210305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fijc.2910210305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=631930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADyaE1c7ivFajsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1978&pages=274-281&author=K.+R.+Stoneauthor=D.+D.+Mickeyauthor=H.+Wunderliauthor=G.+H.+Mickeyauthor=D.+F.+Paulson&title=Isolation+of+a+human+prostate+carcinoma+cell+line+%28DU+145%29&doi=10.1002%2Fijc.2910210305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of a human prostate carcinoma cell line (DU 145)</span></div><div class="casAuthors">Stone K R; Mickey D D; Wunderli H; Mickey G H; Paulson D F</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">274-81</span>
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    </div><div class="casAbstract">A long-term tissue culture cell line has been derived from a human prostate adenocarcinoma metastatic to the brain.  The cell line, DU 145, has been passaged 90 times in vitro over a period of 2 years.  The cells are epithelial, grow in isolated islands on plastic Petri dishes, and form colonies in soft agar suspension culture.  Karyotypic analysis demonstrates an aneuploid human karyotype with a modal chromosome number of 64.  Distinctive marker chromosomes (a translocation Y chromosome, metacentric minute chromosomes and three large acrocentic chromosomes) have been identified.  Electron microscopy of the original tumor tissue and of the tissue culture cell line show a remarkable similarity in cell organelle structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRbXe-vo_9hfa__Jf7-eMvJfW6udTcc2ebJ_EWjNMRDJbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE1c7ivFajsA%253D%253D&md5=128a9ca6ae610280b0e3b8e2c6cc04f7</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fijc.2910210305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.2910210305%26sid%3Dliteratum%253Aachs%26aulast%3DStone%26aufirst%3DK.%2BR.%26aulast%3DMickey%26aufirst%3DD.%2BD.%26aulast%3DWunderli%26aufirst%3DH.%26aulast%3DMickey%26aufirst%3DG.%2BH.%26aulast%3DPaulson%26aufirst%3DD.%2BF.%26atitle%3DIsolation%2520of%2520a%2520human%2520prostate%2520carcinoma%2520cell%2520line%2520%2528DU%2520145%2529%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D1978%26volume%3D21%26spage%3D274%26epage%3D281%26doi%3D10.1002%2Fijc.2910210305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breul, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hance, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crystal, R. G.</span></span> <span> </span><span class="NLM_article-title">Control of collagen production by human diploid lung fibroblasts</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>255</i></span>,  <span class="NLM_fpage">5250</span>â <span class="NLM_lpage">5260</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=7372635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaL3cXksVans7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=255&publication_year=1980&pages=5250-5260&author=S.+D.+Breulauthor=K.+H.+Bradleyauthor=A.+J.+Hanceauthor=M.+P.+Schaferauthor=R.+A.+Bergauthor=R.+G.+Crystal&title=Control+of+collagen+production+by+human+diploid+lung+fibroblasts"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Control of collagen production by human diploid lung fibroblasts</span></div><div class="casAuthors">Breul, Sally D.; Bradley, Kathryn H.; Hance, Allan J.; Schafer, Millie P.; Berg, Richard A.; Crystal, Ronald G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">255</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5250-60</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">To evaluate the control soft tissue fibroblasts have over collagen prodn., HFL-1, a diploid human lung cell strain, was studied during periods of rapid cell growth and relatively slow growth over 25 population doublings.  To minimize environmental influences, the extracellular milieu of the cells was kept const. throughout the study period.  Rates of collagen prodn. per cell unit time were quantitated by labeling HFL-1 with [14C]proline and measuring the prodn. of [14C]hydroxyproline after taking into consideration the sp. activity of [14C]proline within the free intracellular proline pool and the percent hydroxylation of proline residues in newly synthesized collagen.  Although the sp. activity of intracellular free proline and the percent hydroxylation of proline in collagen varied considerably depending on the growth rates of the cells, collagen prodn. by HFL-1 was const., even during periods of rapid cell growth.  Thus, under conditions of a stable environment, populations of soft tissue fibroblasts rigidly control their collagen prodn.  In cultures that maintained a const. doubling time, this stability was maintained over â¥25 population doublings, suggesting that on the av., collagen prodn. appears to be tightly controlled and dissocd. from the events and sequelae of cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyK2bRXCUwSrVg90H21EOLACvtfcHk0ljXViNN7BKKNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXksVans7s%253D&md5=73ba43348807e2bdd8fe59b18af080ea</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBreul%26aufirst%3DS.%2BD.%26aulast%3DBradley%26aufirst%3DK.%2BH.%26aulast%3DHance%26aufirst%3DA.%2BJ.%26aulast%3DSchafer%26aufirst%3DM.%2BP.%26aulast%3DBerg%26aufirst%3DR.%2BA.%26aulast%3DCrystal%26aufirst%3DR.%2BG.%26atitle%3DControl%2520of%2520collagen%2520production%2520by%2520human%2520diploid%2520lung%2520fibroblasts%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1980%26volume%3D255%26spage%3D5250%26epage%3D5260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardenas, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floreancig, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Brabander, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, M. G.</span></span> <span> </span><span class="NLM_article-title">Studies toward the unique pederin family member psymberin: structureâactivity relationships, biochemical studies, and genetics identify the mode-of-action of psymberin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">18998</span>â <span class="NLM_lpage">19003</span>, <span class="refDoi">Â DOI: 10.1021/ja3057002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja3057002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs7%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=18998-19003&author=C.-Y.+Wuauthor=Y.+Fengauthor=E.+R.+Cardenasauthor=N.+Williamsauthor=P.+E.+Floreancigauthor=J.+K.+De+Brabanderauthor=M.+G.+Roth&title=Studies+toward+the+unique+pederin+family+member+psymberin%3A+structure%E2%80%93activity+relationships%2C+biochemical+studies%2C+and+genetics+identify+the+mode-of-action+of+psymberin&doi=10.1021%2Fja3057002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Studies toward the Unique Pederin Family Member Psymberin: Structure-Activity Relationships, Biochemical Studies, and Genetics Identify the Mode-of-Action of Psymberin</span></div><div class="casAuthors">Wu, Cheng-Yang; Feng, Yu; Cardenas, Eduardo R.; Williams, Noelle; Floreancig, Paul E.; De Brabander, Jef K.; Roth, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18998-19003</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Psymberin is the only member of the pederin natural product family that contains a dihydroisocoumarin side chain.  Structural modifications of psymberin uncoupled inhibition of protein translation from cytotoxicity, suggesting that psymberin has more than one bioactivity.  A forward genetic screen in Caenorhabditis elegans was conducted to identify the mol. target(s) of psymberin.  Multiple independent psymberin-resistant mutants were isolated, each contg. the same point mutation in a gene encoding a ribosomal protein.  However, a psymberin-resistant mutant strain bearing this mutation was not cross-resistant to the pederin family member mycalamide A, which binds to the archaeal form of the same protein.  Thus, two pederin family members likely differ in how they bind the same mol. target.  The accumulation of psymberin in cells was sensitive to the stereochem. of the amide side chain at C4 or C8 and the presence of the dihydroisocoumarin side chain.  The observation that psymberin diastereomers or dihydroisocoumarin-truncated analogs lose all cytotoxic activity while retaining the ability to inhibit protein translation in a cell-free in vitro assay can be explained in the context of these differential cell uptake issues.  Finally, we also demonstrate that the blistering activity assocd. with pederin and other members of the family is not due to their protein synthesis inhibiting activity.  Unlike pederin and mycalamide, psymberin does not display irritant or blistering activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxBa84k1Y2prVg90H21EOLACvtfcHk0lhn3Pm1Nemupg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs7%252FI&md5=4056ff204d8ec50929401aa35f470622</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fja3057002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja3057002%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DC.-Y.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DCardenas%26aufirst%3DE.%2BR.%26aulast%3DWilliams%26aufirst%3DN.%26aulast%3DFloreancig%26aufirst%3DP.%2BE.%26aulast%3DDe%2BBrabander%26aufirst%3DJ.%2BK.%26aulast%3DRoth%26aufirst%3DM.%2BG.%26atitle%3DStudies%2520toward%2520the%2520unique%2520pederin%2520family%2520member%2520psymberin%253A%2520structure%25E2%2580%2593activity%2520relationships%252C%2520biochemical%2520studies%252C%2520and%2520genetics%2520identify%2520the%2520mode-of-action%2520of%2520psymberin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D18998%26epage%3D19003%26doi%3D10.1021%2Fja3057002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilke-Mounts, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner-Marmarosh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseau, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senior, A. E.</span></span> <span> </span><span class="NLM_article-title">Cysteines 431 and 1074 are responsible for inhibitory disulfide cross-linking between the two nucleotide-binding sites in human P-glycoprotein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">26980</span>â <span class="NLM_lpage">26987</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M010829200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1074%2Fjbc.M010829200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=11356825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsV2jt7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=26980-26987&author=I.+L.+Urbatschauthor=K.+Gimiauthor=S.+Wilke-Mountsauthor=N.+Lerner-Marmaroshauthor=M.+E.+Rousseauauthor=P.+Grosauthor=A.+E.+Senior&title=Cysteines+431+and+1074+are+responsible+for+inhibitory+disulfide+cross-linking+between+the+two+nucleotide-binding+sites+in+human+P-glycoprotein&doi=10.1074%2Fjbc.M010829200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteines 431 and 1074 are responsible for inhibitory disulfide cross-linking between the two nucleotide-binding sites in human P-glycoprotein</span></div><div class="casAuthors">Urbatsch, Ina L.; Gimi, Khursheed; Wilke-Mounts, Susan; Lerner-Marmarosh, Nicole; Rousseau, Marc-Etienne; Gros, Philippe; Senior, Alan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">26980-26987</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human wild-type and Cys-less P-glycoproteins were expressed in Pichia pastoris and purified in high yield in detergent-sol. form.  Both ran on SDS gels as a single 140-kDa band in the presence of reducing agent and showed strong verapamil-stimulated ATPase activity in the presence of added lipid.  The wild type showed spontaneous formation of higher mol. mass species in the absence of reducing agent, and its ATPase was activated by dithiothreitol.  Oxidn. with Cu2+ generated the same higher mol. mass species, primarily at 200 and â¼300 kDa, in high yield.  Crosslinking was reversed by dithiothreitol and prevented by pretreatment with N-ethylmaleimide.  Using proteins contg. different combinations of naturally occurring Cys residues, it was demonstrated that an inhibitory intramol. disulfide bond forms between Cys-431 and Cys-1074 (located in the Walker A sequences of nucleotide-binding sites 1 and 2, resp.), giving rise to the 200-kDa species.  In addn., dimeric P-glycoprotein species (â¼300 kDa) form by intermol. disulfide bonding between Cys-431 and Cys-1074.  The ready formation of the intramol. disulfide between Cys-431 and Cys-1074 establishes that the two nucleotide-binding sites of P-glycoprotein are structurally very close and capable of intimate functional interaction, consistent with available information on the catalytic mechanism.  Formation of such a disulfide in vivo could, in principle, underlie a regulatory mechanism and might provide a means of intervention to inhibit P-glycoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfw4Oa4Dx7BbVg90H21EOLACvtfcHk0lhn3Pm1Nemupg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsV2jt7s%253D&md5=8238a6cad5571519fe1120360467cd81</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M010829200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M010829200%26sid%3Dliteratum%253Aachs%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DGimi%26aufirst%3DK.%26aulast%3DWilke-Mounts%26aufirst%3DS.%26aulast%3DLerner-Marmarosh%26aufirst%3DN.%26aulast%3DRousseau%26aufirst%3DM.%2BE.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DSenior%26aufirst%3DA.%2BE.%26atitle%3DCysteines%2520431%2520and%25201074%2520are%2520responsible%2520for%2520inhibitory%2520disulfide%2520cross-linking%2520between%2520the%2520two%2520nucleotide-binding%2520sites%2520in%2520human%2520P-glycoprotein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D26980%26epage%3D26987%26doi%3D10.1074%2Fjbc.M010829200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, F. J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orelle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, A. L.</span></span> <span> </span><span class="NLM_article-title">Functional reconstitution of an ABC transporter in nanodiscs for use in electron paramagnetic resonance spectroscopy</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">9513</span>â <span class="NLM_lpage">9515</span>, <span class="refDoi">Â DOI: 10.1021/ja104047c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja104047c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFSqtrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=9513-9515&author=F.+J.+D.+Alvarezauthor=C.+Orelleauthor=A.+L.+Davidson&title=Functional+reconstitution+of+an+ABC+transporter+in+nanodiscs+for+use+in+electron+paramagnetic+resonance+spectroscopy&doi=10.1021%2Fja104047c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Reconstitution of an ABC Transporter in Nanodiscs for Use in Electron Paramagnetic Resonance Spectroscopy</span></div><div class="casAuthors">Alvarez, Frances Joan D.; Orelle, Cedric; Davidson, Amy L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">9513-9515</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ESR spectroscopy is a powerful biophys. technique for study of the structural dynamics of membrane proteins.  Many of these proteins interact with ligands or proteins on one or both sides of the membrane.  Membrane proteins are typically reconstituted in proteoliposomes to observe their function in a physiol. relevant environment.  However, membrane proteins can insert into liposomes in two different orientations, and surfaces facing the lumen of the vesicle can be inaccessible to ligands.  This heterogeneity can lead to subpopulations that do not respond to ligand binding, complicating EPR spectral anal., particularly for distance measurements.  Using the well-characterized maltose transporter, an ATP binding cassette (ABC) transporter that interacts with ligands on both sides of the membrane, the authors provide evidence that reconstitution into nanodisks, which are sol. disk-shaped phospholipid bilayers, is an ideal soln. to these problems.  The authors describe the functional reconstitution of the maltose transporter into nanodisks and demonstrate that this system is ideally suited to study conformational changes and intramol. distances by EPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMX4DDZdg-SrVg90H21EOLACvtfcHk0lhkHThUtCT0AQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFSqtrk%253D&md5=820ad086b6112fd50a1ad669b0b09540</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fja104047c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja104047c%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez%26aufirst%3DF.%2BJ.%2BD.%26aulast%3DOrelle%26aufirst%3DC.%26aulast%3DDavidson%26aufirst%3DA.%2BL.%26atitle%3DFunctional%2520reconstitution%2520of%2520an%2520ABC%2520transporter%2520in%2520nanodiscs%2520for%2520use%2520in%2520electron%2520paramagnetic%2520resonance%2520spectroscopy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D9513%26epage%3D9515%26doi%3D10.1021%2Fja104047c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayburt, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sligar, S. G.</span></span> <span> </span><span class="NLM_article-title">Membrane protein assembly into Nanodiscs</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>584</i></span>,  <span class="NLM_fpage">1721</span>â <span class="NLM_lpage">1727</span>, <span class="refDoi">Â DOI: 10.1016/j.febslet.2009.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.febslet.2009.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=19836392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFKqurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2010&pages=1721-1727&author=T.+H.+Bayburtauthor=S.+G.+Sligar&title=Membrane+protein+assembly+into+Nanodiscs&doi=10.1016%2Fj.febslet.2009.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane protein assembly into nanodisks</span></div><div class="casAuthors">Bayburt, Timothy H.; Sligar, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1721-1727</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nanodisks are sol. nanoscale phospholipid bilayers which can self-assemble integral membrane proteins for biophys., enzymic or structural investigations.  This means for rendering membrane proteins sol. at the single-mol. level offers advantages over liposomes or detergent micelles in terms of size, stability, ability to add genetically modifiable features to the nanodisk structure and ready access to both sides of the phospholipid bilayer domain.  Thus, the nanodisc system provides a novel platform for understanding membrane protein function.  Here, the authors provide an overview of the nanodisk approach and document through several examples many of the applications to the study of the structure and function of integral membrane proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphr-NB4w9pNbVg90H21EOLACvtfcHk0lhkHThUtCT0AQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFKqurw%253D&md5=1ace5001c844ff546e837251fdcb0774</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2009.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2009.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DBayburt%26aufirst%3DT.%2BH.%26aulast%3DSligar%26aufirst%3DS.%2BG.%26atitle%3DMembrane%2520protein%2520assembly%2520into%2520Nanodiscs%26jtitle%3DFEBS%2520Lett.%26date%3D2010%26volume%3D584%26spage%3D1721%26epage%3D1727%26doi%3D10.1016%2Fj.febslet.2009.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhart, R.</span></span> <span> </span><span class="NLM_article-title">ATPase of Escherichia coli: purification, dissociation, and reconstitution of the active complex from the isolated subunits</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">208</span>â <span class="NLM_lpage">216</span>, <span class="refDoi">Â DOI: 10.1021/bi00646a032</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00646a032" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaE28XlsFyrsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1976&pages=208-216&author=G.+Vogelauthor=R.+Steinhart&title=ATPase+of+Escherichia+coli%3A+purification%2C+dissociation%2C+and+reconstitution+of+the+active+complex+from+the+isolated+subunits&doi=10.1021%2Fbi00646a032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">ATPase of Escherichia coli:  purification, dissociation, and reconstitution of the active complex from the isolated subunits</span></div><div class="casAuthors">Vogel, Guenter; Steinhart, Roswithe</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">208-16</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">A simple procedure for the purifn. of Mg2+-stimulated ATPase of Escherichia coli by fractionation with poly(ethylene glycols) and gel filtration is described.  The enzyme restored ATPase-linked reactions to membrane prepns. lacking these activities.  Five different polypeptides (Î±, Î², Î³, Î´, Îµ) were obsd. in Na dodecyl sulfate electrophoresis.  Freezing in salt solns. split the enzyme complex into subunits which did not possess any catalytic activity.  The presence of different subunits was confirmed by electrophoretic and immunol. methods.  The active enzyme complex could be reconstituted by decreasing the ionic strength in the dissocd. sample.  Temp., pH, protein concn., and the presence of substrate were each important determinants of the rate and extent of reconstitution.  The dissocd. enzyme was sepd. by ion-exchange chromatog. into 2 major fragments.  Fragment IA had a mol. wt. of â¼100,000 and contained the Î±, Î³, and Îµ polypeptides.  The minor fragment, IB, had â¼ the same mol. wt. but contained, besides Î±, Î³, and Îµ, the Î´ polypeptide.  Fragment II, with a mol. wt. of â¼52,000, appeared to be identical with the Î² polypeptide.  ATPase activity could be reconstituted from fragments IA and II, whereas the capacity of the ATPase to drive energy-dependent processes in depleted membrane vesicles was only restored after incubation of these 2 fractions with fraction IB, which contained the Î´ subunit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpaMbB-0XbsrVg90H21EOLACvtfcHk0lhkHThUtCT0AQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XlsFyrsw%253D%253D&md5=6e763d83e76b35a1cdd6da8f5c5566e8</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fbi00646a032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00646a032%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DG.%26aulast%3DSteinhart%26aufirst%3DR.%26atitle%3DATPase%2520of%2520Escherichia%2520coli%253A%2520purification%252C%2520dissociation%252C%2520and%2520reconstitution%2520of%2520the%2520active%2520complex%2520from%2520the%2520isolated%2520subunits%26jtitle%3DBiochemistry%26date%3D1976%26volume%3D15%26spage%3D208%26epage%3D216%26doi%3D10.1021%2Fbi00646a032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 6 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Vigneshwaran Namasivayam, Katja Silbermann, Michael Wiese, Jens Pahnke, <span class="NLM_string-name hlFld-ContribAuthor">Sven Marcel Stefan</span>. </span><span class="cited-content_cbyCitation_article-title"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="387b786879">[emailÂ protected]</a>: Computer-Aided Pattern Analysis to Predict Multitarget ABC Transporter Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3350-3366. <a href="https://doi.org/10.1021/acs.jmedchem.0c02199" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02199</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02199%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DC%252540PA%25253A%252BComputer-Aided%252BPattern%252BAnalysis%252Bto%252BPredict%252BMultitarget%252BABC%252BTransporter%252BInhibitors%26aulast%3DNamasivayam%26aufirst%3DVigneshwaran%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D20122020%26date%3D16032021%26volume%3D64%26issue%3D6%26spage%3D3350%26epage%3D3366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ya-Sheng  Li</span>, <span class="hlFld-ContribAuthor ">Shen  Mao</span>, <span class="hlFld-ContribAuthor ">Dong-Sheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Can-Can  Wang</span>, <span class="hlFld-ContribAuthor ">Dan  Zu</span>, <span class="hlFld-ContribAuthor ">Xi  Yang</span>, <span class="hlFld-ContribAuthor ">Gui-Jun  Liu</span>, <span class="hlFld-ContribAuthor ">Si-Jia  Wang</span>, <span class="hlFld-ContribAuthor ">Bo  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-Ze  Bao</span>, <span class="hlFld-ContribAuthor ">Xin-Yi  Ye</span>, <span class="hlFld-ContribAuthor ">Bin  Wei</span>, <span class="hlFld-ContribAuthor ">Zi-Ning  Cui</span>, <span class="hlFld-ContribAuthor ">Jian-Wei  Chen</span>, <span class="hlFld-ContribAuthor ">Hong  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Rational design of phenyl thiophene (pyridine) derivatives that overcome P-glycoprotein mediated MDR in MCF-7/ADR cell. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>114 </em>, 105075. <a href="https://doi.org/10.1016/j.bioorg.2021.105075" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105075%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DRational%252Bdesign%252Bof%252Bphenyl%252Bthiophene%252B%252528pyridine%252529%252Bderivatives%252Bthat%252Bovercome%252BP-glycoprotein%252Bmediated%252BMDR%252Bin%252BMCF-7%25252FADR%252Bcell%26aulast%3DLi%26aufirst%3DYa-Sheng%26date%3D2021%26volume%3D114%26spage%3D105075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John E.  Bell</span>, <span class="hlFld-ContribAuthor ">Jessica K.  Bell</span>. </span><span class="cited-content_cbyCitation_article-title">Allosterism and Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-78. <a href="https://doi.org/10.1002/0471266949.bmc299" title="DOI URL">https://doi.org/10.1002/0471266949.bmc299</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc299%26sid%3Dliteratum%253Aachs%26atitle%3DAllosterism%252Band%252BDrug%252BDiscovery%26aulast%3DBell%26aufirst%3DJohn%2BE.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D78%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ya-Sheng  Li</span>, <span class="hlFld-ContribAuthor ">Xi  Yang</span>, <span class="hlFld-ContribAuthor ">Dong-Sheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Yue  Cai</span>, <span class="hlFld-ContribAuthor ">Zhi  Huang</span>, <span class="hlFld-ContribAuthor ">Rui  Wu</span>, <span class="hlFld-ContribAuthor ">Si-Jia  Wang</span>, <span class="hlFld-ContribAuthor ">Gui-Jun  Liu</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Xiao-Ze  Bao</span>, <span class="hlFld-ContribAuthor ">Xin-Yi  Ye</span>, <span class="hlFld-ContribAuthor ">Bin  Wei</span>, <span class="hlFld-ContribAuthor ">Zi-Ning  Cui</span>, <span class="hlFld-ContribAuthor ">Hong  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and bioactivity study on 5-phenylfuran derivatives as potent reversal agents against P-glycoprotein-mediated multidrug resistance in MCF-7/ADR cell. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>216 </em>, 113336. <a href="https://doi.org/10.1016/j.ejmech.2021.113336" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113336%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbioactivity%252Bstudy%252Bon%252B5-phenylfuran%252Bderivatives%252Bas%252Bpotent%252Breversal%252Bagents%252Bagainst%252BP-glycoprotein-mediated%252Bmultidrug%252Bresistance%252Bin%252BMCF-7%25252FADR%252Bcell%26aulast%3DLi%26aufirst%3DYa-Sheng%26date%3D2021%26volume%3D216%26spage%3D113336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vigneshwaran  Namasivayam</span>, <span class="hlFld-ContribAuthor ">Katja  Silbermann</span>, <span class="hlFld-ContribAuthor ">Jens  Pahnke</span>, <span class="hlFld-ContribAuthor ">Michael  Wiese</span>, <span class="hlFld-ContribAuthor ">Sven Marcel  Stefan</span>. </span><span class="cited-content_cbyCitation_article-title">Scaffold fragmentation and substructure hopping reveal potential, robustness, and limits of computer-aided pattern analysis (<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="b3f0f3e3f2">[emailÂ protected]</a>). </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2021,</strong> <em>19 </em>, 3269-3283. <a href="https://doi.org/10.1016/j.csbj.2021.05.018" title="DOI URL">https://doi.org/10.1016/j.csbj.2021.05.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2021.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2021.05.018%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DScaffold%252Bfragmentation%252Band%252Bsubstructure%252Bhopping%252Breveal%252Bpotential%25252C%252Brobustness%25252C%252Band%252Blimits%252Bof%252Bcomputer-aided%252Bpattern%252Banalysis%252B%252528C%252540PA%252529%26aulast%3DNamasivayam%26aufirst%3DVigneshwaran%26date%3D2021%26volume%3D19%26spage%3D3269%26epage%3D3283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wencheng  Xu</span>, <span class="hlFld-ContribAuthor ">Shuhe  Chen</span>, <span class="hlFld-ContribAuthor ">Xiaoqin  Wang</span>, <span class="hlFld-ContribAuthor ">Hongguang  Wu</span>, <span class="hlFld-ContribAuthor ">Haruki  Yamada</span>, <span class="hlFld-ContribAuthor ">Toshihiko  Hirano</span>. </span><span class="cited-content_cbyCitation_article-title">Bisbenzylisoquinoline alkaloids and P-glycoprotein function: A structure activity relationship study. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (12)
                                     , 115553. <a href="https://doi.org/10.1016/j.bmc.2020.115553" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115553%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DBisbenzylisoquinoline%252Balkaloids%252Band%252BP-glycoprotein%252Bfunction%25253A%252BA%252Bstructure%252Bactivity%252Brelationship%252Bstudy%26aulast%3DXu%26aufirst%3DWencheng%26date%3D2020%26volume%3D28%26issue%3D12%26spage%3D115553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0010.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Putative interactions of compound <b>29</b> at an allosteric inhibitory site on human P-glycoprotein. (A) Structure of compound <b>29</b>. For ease of reference, the âEasternâ and âWesternâ portions of the molecule are marked. (B) Compound <b>29</b> (shown in van der Waalsâ surfaces) docked to a potential allosteric site in the N-terminal nucleotide-binding domain (blue cartoon) of P-gp. This conformation has nucleotide-binding domains in a closed conformation and is similar to that of the Sav1866 X-ray structure (PDB accession: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HYD">2HYD</a>) published by Dawson and Locher.<a onclick="showRef(event, 'ref55 ref57'); return false;" href="javascript:void(0);" class="ref ref55 ref57">(55,57)</a> It was derived from the conformational dynamics investigations of Wise<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and McCormick et al.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The C-terminal nucleotide-binding domain is shown in a charcoal cartoon representation. Bound nucleotides are shown in licorice representations. (C) The P-gp inhibitor compound <b>29</b> (licorice) is shown docked to the putative allosteric site. The surface of P-gp is shown as a wireframe colored by electrostatic potential, and the view point is from within the protein. A large void is visible in the protein near the âWesternâ end of compound <b>29</b> distal to its cyclopropyl group). (D) Same as (C) except that the orientation of the molecule was turned approximately 30Â° around the <i>z</i> axis to show the docked molecule in a slightly different orientation. Small molecules are shown in the figure with nonpolar hydrogens removed for clarity. Images were created with VMD, the SURF surface renderer, and the Tachyon image renderer in VMD; the electrostatic surface of P-gp was calculated with pdb2pqr and the ABPS programs (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Retrosynthesis of the P-glycoprotein inhibitor, compound <b>29</b>, and the generalized scheme for creating variants of compound <b>29</b>. (A) Retrosynthetic scheme for compound <b>29</b>. (B) Generalized synthesis scheme used in the ChemGen program for the virtual synthesis of variants of compound <b>29</b>: In this reaction scheme, any amine can be used to react with chloroacetyl chloride to generate 2-chloroacetamide derivatives. Reaction of the latter with any thiol is then performed to create the sulfide-linked derivatives of which compound <b>29</b> is one. Computationally, large numbers of different amines and thiols can be used to generate large numbers of derivatives. In addition to these generalized schemes for variation of a given structure, localized substitutions of hydrogens for halides, for example, are quite easily accomplished by ChemGen (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0008.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Interactions of compound <b>29</b> group 1 variants at a putative allosteric inhibitory site on human P-glycoprotein. (A) The putative allosteric site on P-gp is shown in a surface representation with synthesized variants of compound <b>29</b> shown in licorice and colored by their atoms. Variants (labeled in the panel) are shown as in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> with the surface of P-glycoprotein colored by its calculated electrostatic potential. (B) The structures of the variant compounds <b>216</b>, <b>227</b>, <b>231</b>, <b>541</b>, and <b>551</b> identified using ChemGen are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0007.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Reversal of paclitaxel resistance by ChemGen-identified variants of the P-glycoprotein inhibitor, compound <b>29</b>. The prostate cancer cell line, DU145TXR, that overexpresses P-glycoprotein was treated with either paclitaxel (PTX) alone or paclitaxel and either 3, 5, 7, or 10 Î¼M P-gp inhibitors (upper left, upper right, lower left, and lower right panels, respectively). Metabolic activity assays using MTT were performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Calculated IC<sub>50</sub> values for paclitaxel in the presence of the respective compounds calculated from this data are reported in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cellular accumulation of the fluorescent dye, calcein, by inclusion of group 1 variants of the P-glycoprotein inhibitor, compound <b>29</b>, identified with ChemGen. The prostate cancer cell line, DU145TXR, that overexpresses P-glycoprotein was treated with calcein AM and without (left) or with (right) a 6 h preincubation with 2 Î¼M P-gp inhibitors indicated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Rationally designed <b>29</b>-variants (group 2 variants) docked at a putative allosteric inhibitory site on human P-glycoprotein. (A) The putative allosteric site on P-gp is shown in a surface representation with synthesized variants of compound <b>29</b> shown in licorice representations colored by their atoms (as indicated). Variants are shown as in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> with the surface of P-glycoprotein colored by its calculated electrostatic potential. (B) Structures of the rationally designed variant compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Reversal of paclitaxel resistance by rationally designed variants of the P-glycoprotein inhibitor <b>29</b>. The prostate cancer cell line, DU145TXR, that overexpresses P-glycoprotein was treated with either paclitaxel (PTX) or paclitaxel and either 3, 5, 7, or 10 Î¼M P-gp inhibitors (upper left, upper right, lower left, and lower right panels, respectively). Metabolic activity assays using MTT were performed as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Calculated IC<sub>50</sub> values for paclitaxel in the presence of the respective compounds calculated from this data are reported in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Cellular accumulation of the fluorescent dye, calcein, by inclusion of group 2 variants of the P-glycoprotein inhibitor, compound <b>29</b>. The prostate cancer cell line, DU145TXR, that overexpresses P-glycoprotein was treated with calcein AM and without (left) or with (right) a 6 h preincubation with 2 Î¼M indicated P-gp inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/medium/jm9b00966_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0002.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Putative allosteric inhibitory site on human P-glycoprotein with the ATPase inhibitor compound <b>29</b> bound. Compound <b>29</b> shown in a licorice representation with its solvent-excluded surface in a wire mesh as it was docked in the P-gp model shown in a blue cartoon. Shown in a red cartoon representation is the cryoEM-derived structure determined by Kim and Chen (2018, PDB accession <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0C">6V0C</a>) superposed on the P-gp model used for docking. One can clearly observe a steric clash of <b>29</b> with the cryoEM structure (black arrow). Residues involved in the steric clash with <b>29</b> (atoms within 2.0 Ã) are T422, E566, Q570, V584, S590, T591, and N594. Small molecules are shown in the figure with nonpolar hydrogens removed for clarity. Images were created with VMD with the MSMS surface renderer from M. Sanner and the Tachyon image renderer in VMD (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b00966/20191205/images/large/jm9b00966_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00966&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24374" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24374" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 68 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dos Santos, D. J. V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, M.-J. U.</span></span> <span> </span><span class="NLM_article-title">P-glycoprotein and membrane roles in multidrug resistance</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">929</span>â <span class="NLM_lpage">946</span>, <span class="refDoi">Â DOI: 10.4155/fmc.15.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.4155%2Ffmc.15.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26061109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpslGhu7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=929-946&author=R.+J.+Ferreiraauthor=D.+J.+V.+A.+dos+Santosauthor=M.-J.+U.+Ferreira&title=P-glycoprotein+and+membrane+roles+in+multidrug+resistance&doi=10.4155%2Ffmc.15.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">P-glycoprotein and membrane roles in multidrug resistance</span></div><div class="casAuthors">Ferreira, Ricardo J.; dos Santos, Daniel J. V. A.; Ferreira, Maria-Jose U.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">929-946</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Multidrug-resistance (MDR) phenomena are a worldwide health concern.  ATP-binding cassette efflux pumps as P-glycoprotein have been thoroughly studied in a frantic run to develop new efflux modulators capable to reverse MDR phenotypes.  The study of efflux pumps has provided some key aspects on drug extrusion, however the answers could not be found solely on ATP-binding cassette transporters.  Its counterpart - the plasma membrane - is now emerging as a crit. structure able to modify drug behavior and efflux pump activity.  Alterations in the membrane surrounding P-glycoprotein are now known to modulate drug efflux, with membrane-related biophys., biochem. and mech. aspects further increasing the complexity of an already multifaceted phenomena.  This review summarizes the main knowledge comprising the plasma membrane role in MDR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVK5Cuqc-j2rVg90H21EOLACvtfcHk0lgfTssD_nyAjA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpslGhu7c%253D&md5=bdcd93a9c7cdcb16c6cef975f810145f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.36%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%26aufirst%3DR.%2BJ.%26aulast%3Ddos%2BSantos%26aufirst%3DD.%2BJ.%2BV.%2BA.%26aulast%3DFerreira%26aufirst%3DM.-J.%2BU.%26atitle%3DP-glycoprotein%2520and%2520membrane%2520roles%2520in%2520multidrug%2520resistance%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D929%26epage%3D946%26doi%3D10.4155%2Ffmc.15.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharom, F. J.</span></span> <span> </span><span class="NLM_article-title">ABC multidrug transporters: structure, function and role in chemoresistance</span>. <i>Pharmacogenomics</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">105</span>â <span class="NLM_lpage">127</span>, <span class="refDoi">Â DOI: 10.2217/14622416.9.1.105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.2217%2F14622416.9.1.105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=18154452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFOjsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=105-127&author=F.+J.+Sharom&title=ABC+multidrug+transporters%3A+structure%2C+function+and+role+in+chemoresistance&doi=10.2217%2F14622416.9.1.105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">ABC multidrug transporters: structure, function and role in chemoresistance</span></div><div class="casAuthors">Sharom, Frances J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacogenomics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">105-127</span>CODEN:
                <span class="NLM_cas:coden">PARMFL</span>;
        ISSN:<span class="NLM_cas:issn">1462-2416</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  Three ATP-binding cassette (ABC)-superfamily multidrug efflux pumps are known to be responsible for chemoresistance; P-glycoprotein (ABCB1), MRP1 (ABCC1) and ABCG2 (BCRP).  These transporters play an important role in normal physiol. by protecting tissues from toxic xenobiotics and endogenous metabolites.  Hydrophobic amphipathic compds., including many clin. used drugs, interact with the substrate-binding pocket of these proteins via flexible hydrophobic and H-bonding interactions.  These efflux pumps are expressed in many human tumors, where they likely contribute to resistance to chemotherapy treatment.  However, the use of efflux-pump modulators in clin. cancer treatment has proved disappointing.  Single nucleotide polymorphisms in ABC drug-efflux pumps may play a role in responses to drug therapy and disease susceptibility.  The effect of various genotypes and haplotypes on the expression and function of these proteins is not yet clear, and their true impact remains controversial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodtANKweyrPrVg90H21EOLACvtfcHk0ljOsxJ_b33tgA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFOjsg%253D%253D&md5=7594f33bc3a524c65d3622b00750dbe6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2217%2F14622416.9.1.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252F14622416.9.1.105%26sid%3Dliteratum%253Aachs%26aulast%3DSharom%26aufirst%3DF.%2BJ.%26atitle%3DABC%2520multidrug%2520transporters%253A%2520structure%252C%2520function%2520and%2520role%2520in%2520chemoresistance%26jtitle%3DPharmacogenomics%26date%3D2008%26volume%3D9%26spage%3D105%26epage%3D127%26doi%3D10.2217%2F14622416.9.1.105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jaramillo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Saig, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, G. J.</span></span> <span> </span><span class="NLM_article-title">How to overcome ATP-binding cassette drug efflux transporter-mediated drug resistance?</span>. <i>Cancer Drug Resist.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">6</span>â <span class="NLM_lpage">29</span>, <span class="refDoi">Â DOI: 10.20517/cdr.2018.02</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.20517%2Fcdr.2018.02" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=6-29&author=A.+C.+Jaramilloauthor=F.+Al+Saigauthor=J.+Cloosauthor=G.+Jansenauthor=G.+J.+Peters&title=How+to+overcome+ATP-binding+cassette+drug+efflux+transporter-mediated+drug+resistance%3F&doi=10.20517%2Fcdr.2018.02"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.20517%2Fcdr.2018.02&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.20517%252Fcdr.2018.02%26sid%3Dliteratum%253Aachs%26aulast%3DJaramillo%26aufirst%3DA.%2BC.%26aulast%3DAl%2BSaig%26aufirst%3DF.%26aulast%3DCloos%26aufirst%3DJ.%26aulast%3DJansen%26aufirst%3DG.%26aulast%3DPeters%26aufirst%3DG.%2BJ.%26atitle%3DHow%2520to%2520overcome%2520ATP-binding%2520cassette%2520drug%2520efflux%2520transporter-mediated%2520drug%2520resistance%253F%26jtitle%3DCancer%2520Drug%2520Resist.%26date%3D2018%26volume%3D1%26spage%3D6%26epage%3D29%26doi%3D10.20517%2Fcdr.2018.02" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unadkat, J. D.</span></span> <span> </span><span class="NLM_article-title">Role of the breast cancer resistance protein (BCRP/ABCG2) in drug transport--an update</span>. <i>AAPS J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">65</span>â <span class="NLM_lpage">82</span>, <span class="refDoi">Â DOI: 10.1208/s12248-014-9668-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1208%2Fs12248-014-9668-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=25236865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFygu7fM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=65-82&author=Q.+Maoauthor=J.+D.+Unadkat&title=Role+of+the+breast+cancer+resistance+protein+%28BCRP%2FABCG2%29+in+drug+transport%2D%2Dan+update&doi=10.1208%2Fs12248-014-9668-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the breast cancer resistance protein (BCRP/ABCG2) in drug transport-an update</span></div><div class="casAuthors">Mao, Qingcheng; Unadkat, Jashvant D.</div><div class="citationInfo"><span class="NLM_cas:title">AAPS Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-82</span>CODEN:
                <span class="NLM_cas:coden">AJAOB6</span>;
        ISSN:<span class="NLM_cas:issn">1550-7416</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The human breast cancer resistance protein (BCRP, gene symbol ABCG2) is an ATP-binding cassette (ABC) efflux transporter.  It was so named because it was initially cloned from a multidrug-resistant breast cancer cell line where it was found to confer resistance to chemotherapeutic agents such as mitoxantrone and topotecan.  Since its discovery in 1998, the substrates of BCRP have been rapidly expanding to include not only therapeutic agents but also physiol. substances such as estrone-3-sulfate, 17Î²-estradiol 17-(Î²-D-glucuronide) and uric acid.  Likewise, at least hundreds of BCRP inhibitors have been identified.  Among normal human tissues, BCRP is highly expressed on the apical membranes of the placental syncytiotrophoblasts, the intestinal epithelium, the liver hepatocytes, the endothelial cells of brain microvessels, and the renal proximal tubular cells, contributing to the absorption, distribution, and elimination of drugs and endogenous compds. as well as tissue protection against xenobiotic exposure.  As a result, BCRP has now been recognized by the FDA to be one of the key drug transporters involved in clin. relevant drug disposition.  We published a highly-accessed review article on BCRP in 2005, and much progress has been made since then.  In this review, we provide an update of current knowledge on basic biochem. and pharmacol. functions of BCRP as well as its relevance to drug resistance and drug disposition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojBFIoPBT7CbVg90H21EOLACvtfcHk0ljOsxJ_b33tgA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFygu7fM&md5=c4b8d6e1286fc6340cf4a8159f525c22</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1208%2Fs12248-014-9668-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-014-9668-6%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DQ.%26aulast%3DUnadkat%26aufirst%3DJ.%2BD.%26atitle%3DRole%2520of%2520the%2520breast%2520cancer%2520resistance%2520protein%2520%2528BCRP%252FABCG2%2529%2520in%2520drug%2520transport--an%2520update%26jtitle%3DAAPS%2520J.%26date%3D2015%26volume%3D17%26spage%3D65%26epage%3D82%26doi%3D10.1208%2Fs12248-014-9668-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span> <span> </span><span class="NLM_article-title">Cancer stem cells and chemoresistance: The smartest survives the raid</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">145</span>â <span class="NLM_lpage">158</span>, <span class="refDoi">Â DOI: 10.1016/j.pharmthera.2016.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.pharmthera.2016.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26899500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVChtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2016&pages=145-158&author=J.+Zhao&title=Cancer+stem+cells+and+chemoresistance%3A+The+smartest+survives+the+raid&doi=10.1016%2Fj.pharmthera.2016.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer stem cells and chemoresistance: The smartest survives the raid</span></div><div class="casAuthors">Zhao, Jihe</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">145-158</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Chemoresistant metastatic relapse of minimal residual disease plays a significant role for poor prognosis of cancer.  Growing evidence supports a crit. role of cancer stem cell (CSC) behind the mechanisms for this deadly disease.  This review briefly introduces the basics of the conventional chemotherapies, updates the CSC theories, highlights the mol. and cellular mechanisms by which CSC smartly designs and utilizes multiple lines of self-defense to avoid being killed by chemotherapy, and concisely summarizes recent progress in studies on CSC-targeted therapies in the end, with the hope to help guide future research toward developing more effective therapeutic strategies to eradicate tumor cells in the patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS9bQmJ1OaX7Vg90H21EOLACvtfcHk0lgm189Jg0i71Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVChtrY%253D&md5=2e3676a52f325cdc7426c76d9eb4da81</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2016.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2016.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DJ.%26atitle%3DCancer%2520stem%2520cells%2520and%2520chemoresistance%253A%2520The%2520smartest%2520survives%2520the%2520raid%26jtitle%3DPharmacol.%2520Ther.%26date%3D2016%26volume%3D160%26spage%3D145%26epage%3D158%26doi%3D10.1016%2Fj.pharmthera.2016.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wijdeven, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assaraf, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neefjes, J.</span></span> <span> </span><span class="NLM_article-title">Old drugs, novel ways out: Drug resistance toward cytotoxic chemotherapeutics</span>. <i>Drug Resist Updates</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">65</span>â <span class="NLM_lpage">81</span>, <span class="refDoi">Â DOI: 10.1016/j.drup.2016.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.drup.2016.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=27620955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC2svgtVWqtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2016&pages=65-81&author=R.+H.+Wijdevenauthor=B.+Pangauthor=Y.+G.+Assarafauthor=J.+Neefjes&title=Old+drugs%2C+novel+ways+out%3A+Drug+resistance+toward+cytotoxic+chemotherapeutics&doi=10.1016%2Fj.drup.2016.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Old drugs, novel ways out: Drug resistance toward cytotoxic chemotherapeutics</span></div><div class="casAuthors">Wijdeven Ruud H; Pang Baoxu; Assaraf Yehuda G; Neefjes Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Drug resistance updates : reviews and commentaries in antimicrobial and anticancer chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">65-81</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Efficacy of chemotherapy in the treatment of distinct malignancies is often hampered by drug resistance arising in the tumor.  Understanding the molecular basis of drug resistance and translating this knowledge into personalized treatment decisions can enhance therapeutic efficacy and even curative outcome.  Over the years, multiple drug resistance mechanisms have been identified that enable tumors to cope with the damage instigated by a specific drug or group of anti-tumor agents.  Here we provide an overview of the molecular pathways leading to resistance against conventional anti-cancer drugs, with emphasis on the utility of these pathways for rational selection of treatments for individual cancer patients.  We further complement the review by discussing the pitfalls and difficulties in translating these findings into novel treatment strategies for cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQB6_n5sF8u3dts4TFFB11gfW6udTcc2eY-Mia86nqsSLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2svgtVWqtg%253D%253D&md5=e2c1879c6086cfd6ab7140a5f80af6d3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2016.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2016.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DWijdeven%26aufirst%3DR.%2BH.%26aulast%3DPang%26aufirst%3DB.%26aulast%3DAssaraf%26aufirst%3DY.%2BG.%26aulast%3DNeefjes%26aufirst%3DJ.%26atitle%3DOld%2520drugs%252C%2520novel%2520ways%2520out%253A%2520Drug%2520resistance%2520toward%2520cytotoxic%2520chemotherapeutics%26jtitle%3DDrug%2520Resist%2520Updates%26date%3D2016%26volume%3D28%26spage%3D65%26epage%3D81%26doi%3D10.1016%2Fj.drup.2016.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fruci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nobili, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locatelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alisi, A.</span></span> <span> </span><span class="NLM_article-title">Drug transporters and multiple drug resistance in the most common pediatric solid tumors</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">308</span>â <span class="NLM_lpage">316</span>, <span class="refDoi">Â DOI: 10.2174/1567205010666131212110948</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.2174%2F1567205010666131212110948" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26694726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC28XktlClsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=308-316&author=D.+Fruciauthor=W.+C.+S.+Choauthor=V.+Nobiliauthor=F.+Locatelliauthor=A.+Alisi&title=Drug+transporters+and+multiple+drug+resistance+in+the+most+common+pediatric+solid+tumors&doi=10.2174%2F1567205010666131212110948"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Transporters and Multiple Drug Resistance in the Most Common Pediatric Solid Tumors</span></div><div class="casAuthors">Fruci, Doriana; Cho, William C. S.; Romania, Paolo; Nobili, Valerio; Locatelli, Franco; Alisi, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">308-316</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Solid tumors account for approx. 30% of all pediatric cancers.  Although chemotherapy has largely contributed to strongly improve patient outcome, multidrug resistance (MDR) remains one of the major mechanisms limiting the overall survival.  The enhanced efflux rate of chemotherapeutic drugs from tumor cells through drug transporters is one of the most important mechanisms of MDR.  Drug transporters play a pivotal role in preserving the balance between sensitivity and resistance of tumor cells to anti-neoplastic drugs.  Their functional activities have been barely investigated in pediatric solid malignancies.  Here, we provide evidence from the current literatures on drug transporters and MDR in the most common types of pediatric solid tumors, including neuroblastoma, Wilms' tumor, rhabdomyosarcoma, retinoblastoma, medulloblastoma and hepatoblastoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO9B7XwflfBLVg90H21EOLACvtfcHk0lgm189Jg0i71Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktlClsLo%253D&md5=b7923a609527acc658e6c24c112ab858</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F1567205010666131212110948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1567205010666131212110948%26sid%3Dliteratum%253Aachs%26aulast%3DFruci%26aufirst%3DD.%26aulast%3DCho%26aufirst%3DW.%2BC.%2BS.%26aulast%3DNobili%26aufirst%3DV.%26aulast%3DLocatelli%26aufirst%3DF.%26aulast%3DAlisi%26aufirst%3DA.%26atitle%3DDrug%2520transporters%2520and%2520multiple%2520drug%2520resistance%2520in%2520the%2520most%2520common%2520pediatric%2520solid%2520tumors%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2016%26volume%3D17%26spage%3D308%26epage%3D316%26doi%3D10.2174%2F1567205010666131212110948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ierano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szakacs, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span> <span> </span><span class="NLM_article-title">The controversial role of ABC transporters in clinical oncology</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">209</span>â <span class="NLM_lpage">232</span>, <span class="refDoi">Â DOI: 10.1042/bse0500209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1042%2Fbse0500209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=21967059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlGrsb7O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=209-232&author=A.+Tamakiauthor=C.+Ieranoauthor=G.+Szakacsauthor=R.+W.+Robeyauthor=S.+E.+Bates&title=The+controversial+role+of+ABC+transporters+in+clinical+oncology&doi=10.1042%2Fbse0500209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The controversial role of ABC transporters in clinical oncology</span></div><div class="casAuthors">Tamaki, Akina; Ierano, Caterina; Szakacs, Gergely; Robey, Robert W.; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Essays in Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">209-232</span>CODEN:
                <span class="NLM_cas:coden">ESBIAV</span>;
        ISSN:<span class="NLM_cas:issn">0071-1365</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The phenomenon of multidrug resistance in cancer is often assocd. with the overexpression of the ABC (ATP-binding cassette) transporters Pgp (P-glycoprotein) (ABCB1), MRP1 (multidrug resistance-assocd. protein 1) (ABCC1) and ABCG2 [BCRP (breast cancer resistance protein)].  Since the discovery of Pgp over 35 years ago, studies have convincingly linked ABC transporter expression to poor outcome in several cancer types, leading to the development of transporter inhibitors.  Three generations of inhibitors later, we are still no closer to validating the 'Pgp hypothesis', the idea that increased chemotherapy efficacy can be achieved by inhibition of transporter-mediated efflux.  In this chapter, we highlight the difficulties and past failures encountered in the development of clin. inhibitors of ABC transporters.  We discuss the challenges that remain in our effort to exploit decades of work on ABC transporters in oncol.  In learning from past mistakes, it is hoped that ABC transporters can be developed as targets for clin. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTf-JsvF3tlLVg90H21EOLACvtfcHk0lg0ivutZHzNEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlGrsb7O&md5=3c63559612797c2339f046019985def9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2Fbse0500209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbse0500209%26sid%3Dliteratum%253Aachs%26aulast%3DTamaki%26aufirst%3DA.%26aulast%3DIerano%26aufirst%3DC.%26aulast%3DSzakacs%26aufirst%3DG.%26aulast%3DRobey%26aufirst%3DR.%2BW.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DThe%2520controversial%2520role%2520of%2520ABC%2520transporters%2520in%2520clinical%2520oncology%26jtitle%3DEssays%2520Biochem.%26date%3D2011%26volume%3D50%26spage%3D209%26epage%3D232%26doi%3D10.1042%2Fbse0500209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palmeira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasconcelos, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, M. M.</span></span> <span> </span><span class="NLM_article-title">Three decades of P-gp inhibitors: skimming through several generations and scaffolds</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1946</span>â <span class="NLM_lpage">2025</span>, <span class="refDoi">Â DOI: 10.2174/092986712800167392</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.2174%2F092986712800167392" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=22257057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsF2htbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1946-2025&author=A.+Palmeiraauthor=E.+Sousaauthor=M.+H.+Vasconcelosauthor=M.+M.+Pinto&title=Three+decades+of+P-gp+inhibitors%3A+skimming+through+several+generations+and+scaffolds&doi=10.2174%2F092986712800167392"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Three decades of P-gp inhibitors: skimming through several generations and scaffolds</span></div><div class="casAuthors">Palmeira, A.; Sousa, E.; Vasconcelos, M. H.; Pinto, M. M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1946-2025</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Many tumor cells become resistant to commonly used cytotoxic drugs due to the overexpression of ATP-binding cassette (ABC) transporters, namely P-glycoprotein (P-gp).  The discovery of the reversal of multidrug resistance (MDR) by verapamil occurred in 1981, and in 1968 MDR Chinese hamster cell lines were isolated for the first time.  Since then, P-gp inhibitors have been intensively studied as potential MDR reversers.  Initially, drugs to reverse MDR were not specifically developed for inhibiting P-gp; in fact, they had other pharmacol. properties, as well as a relatively low affinity for MDR transporters.  An example of this first generation P-gp inhibitors is verapamil.  The second generation included more specific with less side-effect inhibitors, such as dexverapamil or dexniguldipine.  A third generation of P-gp inhibitors comprised compds. such as tariquidar, with high affinity to P-gp at nanomolar concns.  These generations of inhibitors of P-gp have been examd. in preclin. and clin. studies; however, these trials have largely failed to demonstrate an improvement in therapeutic efficacy.  Therefore, new and innovative strategies, such as the fallback to natural products, the design of peptidomimetics and dual activity ligands emerged as a fourth generation of P-gp inhibitors.  The chem. of P-gp inhibitors, as well as their in vitro, in vivo and clin. trials are discussed, and the most recent advances concerning P-gp modulators are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooLS8EJNaESbVg90H21EOLACvtfcHk0lg0ivutZHzNEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsF2htbo%253D&md5=37ba7b1ce5f786a456edc7ed68f33f0b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F092986712800167392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712800167392%26sid%3Dliteratum%253Aachs%26aulast%3DPalmeira%26aufirst%3DA.%26aulast%3DSousa%26aufirst%3DE.%26aulast%3DVasconcelos%26aufirst%3DM.%2BH.%26aulast%3DPinto%26aufirst%3DM.%2BM.%26atitle%3DThree%2520decades%2520of%2520P-gp%2520inhibitors%253A%2520skimming%2520through%2520several%2520generations%2520and%2520scaffolds%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D1946%26epage%3D2025%26doi%3D10.2174%2F092986712800167392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Binkhathlan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavasanifar, A.</span></span> <span> </span><span class="NLM_article-title">P-glycoprotein inhibition as a therapeutic approach for overcoming multidrug resistance in cancer: current status and future perspectives</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">326</span>â <span class="NLM_lpage">346</span>, <span class="refDoi">Â DOI: 10.2174/15680096113139990076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.2174%2F15680096113139990076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=23369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVOjsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=326-346&author=Z.+Binkhathlanauthor=A.+Lavasanifar&title=P-glycoprotein+inhibition+as+a+therapeutic+approach+for+overcoming+multidrug+resistance+in+cancer%3A+current+status+and+future+perspectives&doi=10.2174%2F15680096113139990076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">P-glycoprotein inhibition as a therapeutic approach for overcoming multidrug resistance in cancer: current status and future perspectives</span></div><div class="casAuthors">Binkhathlan, Ziyad; Lavasanifar, Afsaneh</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">326-346</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  One of the major causes of failure in cancer chemotherapy is multidrug resistance (MDR), where cancer cells simultaneously become resistant to different anticancer drugs.  Over-expression of membrane efflux pumps like P-glycoprotein (P-gp) that recognizes different chemotherapeutic agents and transports them out of the cell, plays a major role in MDR.  The shortcoming of P-gp inhibitors in clinic was attributed to their non-specific action on P-gp and/or non-selective distribution to non-target organs that leads to intolerable side effects by the P-gp inhibitor at doses required for P-gp inhibition upon systemic administration.  Another major issue is the reduced elimination of P-gp substrates (e.g. anticancer drugs) and intolerable toxicities by anticancer drugs when co-administered with P-gp inhibitors.  To overcome these shortcomings, new generation of P-gp inhibitors with improved specificity for P-gp were developed.  More recently, attention was paid to the use of drug delivery systems primarily to restrict P-gp inhibition to tumor and reduce the non-selective inhibition of P-gp in non-target organs.  This review will provide an overview and update on the status of P-gp inhibition approaches and the role of drug delivery systems in overcoming P-gp mediated MDR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6kCA4RDN2s7Vg90H21EOLACvtfcHk0lg0ivutZHzNEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVOjsbY%253D&md5=9fc1e50504313ae7735049fcaadcb9ff</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F15680096113139990076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F15680096113139990076%26sid%3Dliteratum%253Aachs%26aulast%3DBinkhathlan%26aufirst%3DZ.%26aulast%3DLavasanifar%26aufirst%3DA.%26atitle%3DP-glycoprotein%2520inhibition%2520as%2520a%2520therapeutic%2520approach%2520for%2520overcoming%2520multidrug%2520resistance%2520in%2520cancer%253A%2520current%2520status%2520and%2520future%2520perspectives%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2013%26volume%3D13%26spage%3D326%26epage%3D346%26doi%3D10.2174%2F15680096113139990076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kathawala, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashby, C. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.-S.</span></span> <span> </span><span class="NLM_article-title">The modulation of ABC transporter-mediated multidrug resistance in cancer: A review of the past decade</span>. <i>Drug Resist. Updates</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">17</span>, <span class="refDoi">Â DOI: 10.1016/j.drup.2014.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.drup.2014.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=25554624" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC2MvjtFGmtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1-17&author=R.+J.+Kathawalaauthor=P.+Guptaauthor=C.+R.+Ashbyauthor=Z.-S.+Chen&title=The+modulation+of+ABC+transporter-mediated+multidrug+resistance+in+cancer%3A+A+review+of+the+past+decade&doi=10.1016%2Fj.drup.2014.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The modulation of ABC transporter-mediated multidrug resistance in cancer: a review of the past decade</span></div><div class="casAuthors">Kathawala Rishil J; Gupta Pranav; Ashby Charles R Jr; Chen Zhe-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Drug resistance updates : reviews and commentaries in antimicrobial and anticancer chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ATP-binding cassette (ABC) transporters represent one of the largest and oldest families of membrane proteins in all extant phyla from prokaryotes to humans, which couple the energy derived from ATP hydrolysis essentially to translocate, among various substrates, toxic compounds across the membrane.  The fundamental functions of these multiple transporter proteins include: (1) conserved mechanisms related to nutrition and pathogenesis in bacteria, (2) spore formation in fungi, and (3) signal transduction, protein secretion and antigen presentation in eukaryotes.  Moreover, one of the major causes of multidrug resistance (MDR) and chemotherapeutic failure in cancer therapy is believed to be the ABC transporter-mediated active efflux of a multitude of structurally and mechanistically distinct cytotoxic compounds across membranes.  It has been postulated that ABC transporter inhibitors known as chemosensitizers may be used in combination with standard chemotherapeutic agents to enhance their therapeutic efficacy.  The current paper reviews the advance in the past decade in this important domain of cancer chemoresistance and summarizes the development of new compounds and the re-evaluation of compounds originally designed for other targets as transport inhibitors of ATP-dependent drug efflux pumps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRNzfrj3o9ghtHxxy173TfNfW6udTcc2ebVghUF_GXZG7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvjtFGmtQ%253D%253D&md5=db6a95a46f45a829f02b1af34fc11cc4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2014.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2014.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DKathawala%26aufirst%3DR.%2BJ.%26aulast%3DGupta%26aufirst%3DP.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26aulast%3DChen%26aufirst%3DZ.-S.%26atitle%3DThe%2520modulation%2520of%2520ABC%2520transporter-mediated%2520multidrug%2520resistance%2520in%2520cancer%253A%2520A%2520review%2520of%2520the%2520past%2520decade%26jtitle%3DDrug%2520Resist.%2520Updates%26date%3D2015%26volume%3D18%26spage%3D1%26epage%3D17%26doi%3D10.1016%2Fj.drup.2014.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottesman, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastan, I. H.</span></span> <span> </span><span class="NLM_article-title">The Role of multidrug resistance efflux pumps in cancer: revisiting a JNCI publication exploring expression of the MDR1 (P-glycoprotein) gene</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">djv222</span>, <span class="refDoi">Â DOI: 10.1093/jnci/djv222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1093%2Fjnci%2Fdjv222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26286731" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2015&pages=djv222&author=M.+M.+Gottesmanauthor=I.+H.+Pastan&title=The+Role+of+multidrug+resistance+efflux+pumps+in+cancer%3A+revisiting+a+JNCI+publication+exploring+expression+of+the+MDR1+%28P-glycoprotein%29+gene&doi=10.1093%2Fjnci%2Fdjv222"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjv222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjv222%26sid%3Dliteratum%253Aachs%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26aulast%3DPastan%26aufirst%3DI.%2BH.%26atitle%3DThe%2520Role%2520of%2520multidrug%2520resistance%2520efflux%2520pumps%2520in%2520cancer%253A%2520revisiting%2520a%2520JNCI%2520publication%2520exploring%2520expression%2520of%2520the%2520MDR1%2520%2528P-glycoprotein%2529%2520gene%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2015%26volume%3D107%26spage%3Ddjv222%26doi%3D10.1093%2Fjnci%2Fdjv222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marin, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RodrÃ­guez-Macias, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DÃ­ez-MartÃ­n, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macias, R. I. R.</span></span> <span> </span><span class="NLM_article-title">Role of drug transport and metabolism in the chemoresistance of acute myeloid leukemia</span>. <i>Blood Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">55</span>â <span class="NLM_lpage">64</span>, <span class="refDoi">Â DOI: 10.1016/j.blre.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.blre.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26321049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOmt7bL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=55-64&author=J.+J.+G.+Marinauthor=O.+Brizauthor=G.+Rodr%C3%ADguez-Maciasauthor=J.+L.+D%C3%ADez-Mart%C3%ADnauthor=R.+I.+R.+Macias&title=Role+of+drug+transport+and+metabolism+in+the+chemoresistance+of+acute+myeloid+leukemia&doi=10.1016%2Fj.blre.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Role of drug transport and metabolism in the chemoresistance of acute myeloid leukemia</span></div><div class="casAuthors">Marin, Jose J. G.; Briz, Oscar; Rodriguez-Macias, Gabriela; Diez-Martin, Jose L.; Macias, Rocio I. R.</div><div class="citationInfo"><span class="NLM_cas:title">Blood Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-64</span>CODEN:
                <span class="NLM_cas:coden">BLOREB</span>;
        ISSN:<span class="NLM_cas:issn">0268-960X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Acute myeloid leukemia is a clonal but heterogeneous disease differing in mol. pathogenesis, clin. features and response to chemotherapy.  This latter frequently consists of a combination of cytarabine and anthracyclines, although etoposide, demethylating agents, and other drugs are also used.  Unfortunately, chemoresistance is a common and serious problem.  Multiple mechanisms account for impaired effectiveness of drugs and reduced levels of active agents in target cells.  The latter can be due to lower drug uptake, increased export or decreased intracellular proportion of active/inactive agent due to changes in the expression/function of enzymes responsible for the activation of pro-drugs and the inactivation of active agents.  Characterization of the "resistome", or profile of expressed genes accounting for multi-drug resistance (MDR) phenotype, would permit to predict the lack of response to chemotherapy and would help in the selection of the best pharmacol. regime for each patient and moment, and to develop strategies of chemosensitization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfZ9jGbdMhrrVg90H21EOLACvtfcHk0lhbT3D9YI_xRQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOmt7bL&md5=945e0f62a3800f906f24421dfea5eca6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.blre.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.blre.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DMarin%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DBriz%26aufirst%3DO.%26aulast%3DRodr%25C3%25ADguez-Macias%26aufirst%3DG.%26aulast%3DD%25C3%25ADez-Mart%25C3%25ADn%26aufirst%3DJ.%2BL.%26aulast%3DMacias%26aufirst%3DR.%2BI.%2BR.%26atitle%3DRole%2520of%2520drug%2520transport%2520and%2520metabolism%2520in%2520the%2520chemoresistance%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DBlood%2520Rev.%26date%3D2016%26volume%3D30%26spage%3D55%26epage%3D64%26doi%3D10.1016%2Fj.blre.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">List, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopecky, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willman, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persons, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slovak, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karanes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shurafa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appelbaum, F. R.</span></span> <span> </span><span class="NLM_article-title">Benefit of cyclosporine modulation of drug resistance in patients with poor-risk acute myeloid leukemia: a Southwest Oncology Group study</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">3212</span>â <span class="NLM_lpage">3220</span>, <span class="refDoi">Â DOI: 10.1182/blood.V98.12.3212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1182%2Fblood.V98.12.3212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=11719356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovVSgsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=3212-3220&author=A.+F.+Listauthor=K.+J.+Kopeckyauthor=C.+L.+Willmanauthor=D.+R.+Headauthor=D.+L.+Personsauthor=M.+L.+Slovakauthor=R.+Dorrauthor=C.+Karanesauthor=H.+E.+Hynesauthor=J.+H.+Doroshowauthor=M.+Shurafaauthor=F.+R.+Appelbaum&title=Benefit+of+cyclosporine+modulation+of+drug+resistance+in+patients+with+poor-risk+acute+myeloid+leukemia%3A+a+Southwest+Oncology+Group+study&doi=10.1182%2Fblood.V98.12.3212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Benefit of cyclosporine modulation of drug resistance in patients with poor-risk acute myeloid leukemia: a Southwest Oncology Group study</span></div><div class="casAuthors">List, Alan F.; Kopecky, Kenneth J.; Willman, Cheryl L.; Head, David R.; Persons, Diane L.; Slovak, Marilyn L.; Dorr, Robert; Karanes, Chatchada; Hynes, Harry E.; Doroshow, James H.; Shurafa, Muhammad; Appelbaum, Frederick R.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3212-3220</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Cyclosporine A (CsA) inhibits P-glycoprotein (Pgp)-mediated cellular export of anthracyclines at clin. achievable concns.  This randomized controlled trial was performed to test the benefit of CsA addn. to treatment with cytarabine and daunorubicin (DNR) in patients with poor-risk acute myeloid leukemia (AML).  A total of 226 patients were randomly assigned to sequential treatment with cytarabine and Infusional DNR with or without i.v. CsA.  Remitting patients received one course of consolidation chemotherapy that included DNR with or without CsA as assigned during induction.  Addn. of CsA significantly reduced the frequency of resistance to induction chemotherapy (31% vs. 47%, P = .0077).  Whereas the rate of complete remission was not significantly improved (39% vs. 33%, P = 14), relapse-free survival (34% vs. 9% at 2 yr, P = .031) and overall survival (22% vs. 12%, P = .046) were significantly increased with CsA.  The effect of CsA on survival was greatest in patients with moderate or bright Pgp expression (median 12 mo with CsA vs. 4 mo for controls) compared to patients with absent or low Pgp expression (median 6 mo in both arms).  The frequency of Induction deaths was 15% with CsA and 18% in controls.  Steady-state serum concns. of DNR (P = .0089) and daunorubicinol (P < .0001) were significantly higher in CsA-treated patients.  Survival (P = .0003) and induction response (P = .028) improved with increasing DNR concn. in CsA-treated patients but not in controls, suggesting a targeted Interaction by CsA to enhance anthracycline cytotoxicity.  These results indicate that addn. of CsA to an induction and consolidation regimen contg. infusional DNR significantly reduces resistance to DNR, prolongs the duration of remission, and improves overall survival in patients with poor-risk AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8IxQr3SlAlrVg90H21EOLACvtfcHk0lj6RykrPMO_0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovVSgsrc%253D&md5=6d53b5b27e1a51791db770858ac40047</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1182%2Fblood.V98.12.3212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V98.12.3212%26sid%3Dliteratum%253Aachs%26aulast%3DList%26aufirst%3DA.%2BF.%26aulast%3DKopecky%26aufirst%3DK.%2BJ.%26aulast%3DWillman%26aufirst%3DC.%2BL.%26aulast%3DHead%26aufirst%3DD.%2BR.%26aulast%3DPersons%26aufirst%3DD.%2BL.%26aulast%3DSlovak%26aufirst%3DM.%2BL.%26aulast%3DDorr%26aufirst%3DR.%26aulast%3DKaranes%26aufirst%3DC.%26aulast%3DHynes%26aufirst%3DH.%2BE.%26aulast%3DDoroshow%26aufirst%3DJ.%2BH.%26aulast%3DShurafa%26aufirst%3DM.%26aulast%3DAppelbaum%26aufirst%3DF.%2BR.%26atitle%3DBenefit%2520of%2520cyclosporine%2520modulation%2520of%2520drug%2520resistance%2520in%2520patients%2520with%2520poor-risk%2520acute%2520myeloid%2520leukemia%253A%2520a%2520Southwest%2520Oncology%2520Group%2520study%26jtitle%3DBlood%26date%3D2001%26volume%3D98%26spage%3D3212%26epage%3D3220%26doi%3D10.1182%2Fblood.V98.12.3212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaimes, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aller, S. G.</span></span> <span> </span><span class="NLM_article-title">Refined structures of mouse P-glycoprotein</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">34</span>â <span class="NLM_lpage">46</span>, <span class="refDoi">Â DOI: 10.1002/pro.2387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fpro.2387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=24155053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFGktLfN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2014&pages=34-46&author=J.+Liauthor=K.+F.+Jaimesauthor=S.+G.+Aller&title=Refined+structures+of+mouse+P-glycoprotein&doi=10.1002%2Fpro.2387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Refined structures of mouse P-glycoprotein</span></div><div class="casAuthors">Li, Jingzhi; Jaimes, Kimberly F.; Aller, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-46</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The recently detd. C. elegans P-glycoprotein (Pgp) structure revealed significant deviations compared to the original mouse Pgp structure, which suggested possible misinterpretations in the latter model.  To address this concern, we generated an exptl. electron d. map from single-wavelength anomalous dispersion phasing of an original mouse Pgp dataset to 3.8 Ã resoln.  The map exhibited significantly more detail compared to the original MAD map and revealed several regions of the structure that required de novo model building.  The improved drug-free structure was refined to 3.8 Ã resoln. with a 9.4 and 8.1% decrease in Rwork and Rfree, resp., (Rwork = 21.2%, Rfree = 26.6%) and a significant improvement in protein geometry.  The improved mouse Pgp model contains â¼95% of residues in the favorable Ramachandran region compared to only 57% for the original model.  The registry of six transmembrane helixes was cor., revealing amino acid residues involved in drug binding that were previously unrecognized.  Registry shifts (rotations and translations) for three transmembrane (TM)4 and TM5 and the addn. of three N-terminal residues were necessary, and were validated with new mercury labeling and anomalous Fourier d.  The cor. position of TM4, which forms the frame of a portal for drug entry, had backbone atoms shifted >6 Ã from their original positions.  The drug translocation pathway of mouse Pgp is 96% identical to human Pgp and is enriched in arom. residues that likely play a collective role in allowing a high degree of polyspecific substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreAlq0kWSXj7Vg90H21EOLACvtfcHk0lj6RykrPMO_0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFGktLfN&md5=7c6fd47f821dca3d47cf78c7d2e2020f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fpro.2387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2387%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJaimes%26aufirst%3DK.%2BF.%26aulast%3DAller%26aufirst%3DS.%2BG.%26atitle%3DRefined%2520structures%2520of%2520mouse%2520P-glycoprotein%26jtitle%3DProtein%2520Sci.%26date%3D2014%26volume%3D23%26spage%3D34%26epage%3D46%26doi%3D10.1002%2Fpro.2387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Molecular structure of human P-glycoprotein in the ATP-bound, outward-facing conformation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">915</span>, <span class="refDoi">Â DOI: 10.1126/science.aar7389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1126%2Fscience.aar7389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=29371429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVKgs7vN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=915&author=Y.+Kimauthor=J.+Chen&title=Molecular+structure+of+human+P-glycoprotein+in+the+ATP-bound%2C+outward-facing+conformation&doi=10.1126%2Fscience.aar7389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Co-infection of Ureaplasma urealyticum and human papilloma virus in asymptomatic sexually active individuals</span></div><div class="casAuthors">Kim, Sang Il; Yoon, Joo Hee; Park, Dong Choon; Lee, Dong Sup; Lee, Seung Ju; Choe, Hyun Sop; Hwikim, Jin; Park, Tae Chul; Lee, Sung Jong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Medical Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">915-920</span>CODEN:
                <span class="NLM_cas:coden">IJMSGZ</span>;
        ISSN:<span class="NLM_cas:issn">1449-1907</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">This study aimed to det. the role of asymptomatic bacterial sexually transmitted infections (STIs), such as Chlamydia trachomatis (Ct), Mycoplasma genitalium (Mg), Mycoplasma hominis (Mh), and Ureaplasma urealyticum (Uu) in human papillomavirus (HPV) infection.  In total, 264 asymptomatic outpatients aged between 21 and 80 years were prospectively enrolled in this study during routine gynecol. screening tests.  Specimens collected with a Cervex Brush were routinely analyzed with the Hybrid Capture 2 assay for HPV.  Simultaneously, a specimen obtained with an endocervical swab was used to detect Ct and Mg with a monoplex real-time polymerase chain reaction (PCR) and to confirm Mh and Uu with a Mycoplasma IST 2 kit.  The detection rates (%) of HPV, Ct, Mg, Mh, and Uu were 82/264 (31.1), 6/264 (2.3), 5/264 (1.9), 16/264 (6.1), and 95/264 (36.0), resp.  Of 95 Uu, 32 (33.7%) showed high d. colonization (HDC, â¥104 color-changing units/mL).  HDC-Uu was significantly assocd. with HPV infection (p=0.014, chi-square test).  Mg infection and Mh infection were not assocd. with HPV infection (p=0.981 and p=0.931, chi-square test).  Age was not assocd. with HPV infection or bacterial infection.  Our data suggested that asymptomatic HDC-Uu was closely assocd. with HPV infection.  Therefore, simultaneous evaluation for Uu and HPV should be performed during gynecol. screening, even in asymptomatic individuals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrADIFePxanKrVg90H21EOLACvtfcHk0lj6RykrPMO_0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVKgs7vN&md5=f7093af4dc290e028e859999e25b743f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.aar7389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aar7389%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DMolecular%2520structure%2520of%2520human%2520P-glycoprotein%2520in%2520the%2520ATP-bound%252C%2520outward-facing%2520conformation%26jtitle%3DScience%26date%3D2018%26spage%3D915%26doi%3D10.1126%2Fscience.aar7389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aller, S. G.</span></span> <span> </span><span class="NLM_article-title">Equilibrated atomic models of outward-facing P-glycoprotein and effect of ATP binding on structural dynamics</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">7880</span>, <span class="refDoi">Â DOI: 10.1038/srep07880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fsrep07880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=25600711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVCltrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=7880&author=L.+Panauthor=S.+G.+Aller&title=Equilibrated+atomic+models+of+outward-facing+P-glycoprotein+and+effect+of+ATP+binding+on+structural+dynamics&doi=10.1038%2Fsrep07880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Equilibrated Atomic Models of Outward-Facing P-glycoprotein and Effect of ATP Binding on Structural Dynamics</span></div><div class="casAuthors">Pan, Lurong; Aller, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7880</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) is an ATP-binding cassette (ABC) transporter that alternates between inward- and outward-facing conformations to capture and force substrates out of cells like a peristaltic pump.  The high degree of similarity in outward-facing structures across evolution of ABC transporters allowed construction of a high-confidence outward-facing Pgp at. model based on crystal structures of outward-facing Sav1866 and inward-facing Pgp.  The model adhered to previous exptl. detd. secondary- and tertiary- configurations during all-atom mol. dynamics simulations in the presence or absence of MgATP.  Three long lasting (>100 ns) meta-stable states were apparent in the presence of MgATP revealing new insights into alternating access.  The two ATP-binding pockets are highly asym. resulting in differential control of overall structural dynamics and allosteric regulation of the drug-binding pocket.  Equilibrated Pgp has a considerably different electrostatic profile compared to Sav1866 that implicates significant kinetic and thermodn. differences in transport mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonLxb3soLs7LVg90H21EOLACvtfcHk0lia9sOoSnWFHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVCltrY%253D&md5=197b7b93e90191f9347156277530b2a0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fsrep07880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep07880%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DL.%26aulast%3DAller%26aufirst%3DS.%2BG.%26atitle%3DEquilibrated%2520atomic%2520models%2520of%2520outward-facing%2520P-glycoprotein%2520and%2520effect%2520of%2520ATP%2520binding%2520on%2520structural%2520dynamics%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D7880%26doi%3D10.1038%2Fsrep07880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the multidrug transporter P-glycoprotein from Caenorhabditis elegans</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>490</i></span>,  <span class="NLM_fpage">566</span>â <span class="NLM_lpage">569</span>, <span class="refDoi">Â DOI: 10.1038/nature11448</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fnature11448" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=23000902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtlymu7%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=566-569&author=M.+S.+Jinauthor=M.+L.+Oldhamauthor=Q.+Zhangauthor=J.+Chen&title=Crystal+structure+of+the+multidrug+transporter+P-glycoprotein+from+Caenorhabditis+elegans&doi=10.1038%2Fnature11448"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the multidrug transporter P-glycoprotein from Caenorhabditis elegans</span></div><div class="casAuthors">Jin, Mi Sun; Oldham, Michael L.; Zhang, Qiuju; Chen, Jue</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">490</span>
        (<span class="NLM_cas:issue">7421</span>),
    <span class="NLM_cas:pages">566-569</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P-glycoprotein (P-gp) is an ATP-binding cassette transporter that confers multidrug resistance in cancer cells.  It also affects the absorption, distribution and clearance of cancer-unrelated drugs and xenobiotics.  For these reasons, the structure and function of P-gp have been studied extensively for decades.  Here we present biochem. characterization of P-gp from Caenorhabditis elegans and its crystal structure at a resoln. of 3.4 angstroems.  We find that the apparent affinities of P-gp for anticancer drugs actinomycin D and paclitaxel are approx. 4,000 and 100 times higher, resp., in the membrane bilayer than in detergent.  This affinity enhancement highlights the importance of membrane partitioning when a drug accesses the transporter in the membrane.  Furthermore, the transporter in the crystal structure opens its drug pathway at the level of the membrane's inner leaflet.  In the helixes flanking the opening to the membrane, we observe extended loops that may mediate drug binding, function as hinges to gate the pathway or both.  We also find that the interface between the transmembrane and nucleotide-binding domains, which couples ATP hydrolysis to transport, contains a ball-and-socket joint and salt bridges similar to the ATP-binding cassette importers, suggesting that ATP-binding cassette exporters and importers may use similar mechanisms to achieve alternating access for transport.  Finally, a model of human P-gp derived from the structure of C. elegans P-gp not only is compatible with decades of biochem. anal., but also helps to explain perplexing functional data regarding the Phe335Ala mutant.  These results increase our understanding of the structure and function of this important mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdu-UyIUtRP7Vg90H21EOLACvtfcHk0lia9sOoSnWFHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtlymu7%252FN&md5=73d67b98d098c05a3975ed6f371027cb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature11448&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11448%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DOldham%26aufirst%3DM.%2BL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520multidrug%2520transporter%2520P-glycoprotein%2520from%2520Caenorhabditis%2520elegans%26jtitle%3DNature%26date%3D2012%26volume%3D490%26spage%3D566%26epage%3D569%26doi%3D10.1038%2Fnature11448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, N. M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manolaridis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahlberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locher, K. P.</span></span> <span> </span><span class="NLM_article-title">Structure of the human multidrug transporter ABCG2</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span>,  <span class="NLM_fpage">504</span>â <span class="NLM_lpage">509</span>, <span class="refDoi">Â DOI: 10.1038/nature22345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fnature22345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=28554189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosFWntbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=546&publication_year=2017&pages=504-509&author=N.+M.+I.+Taylorauthor=I.+Manolaridisauthor=S.+M.+Jacksonauthor=J.+Kowalauthor=H.+Stahlbergauthor=K.+P.+Locher&title=Structure+of+the+human+multidrug+transporter+ABCG2&doi=10.1038%2Fnature22345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the human multidrug transporter ABCG2</span></div><div class="casAuthors">Taylor, Nicholas M. I.; Manolaridis, Ioannis; Jackson, Scott M.; Kowal, Julia; Stahlberg, Henning; Locher, Kaspar P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7659</span>),
    <span class="NLM_cas:pages">504-509</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">ABCG2 is a constitutively expressed ATP-binding cassette (ABC) transporter that protects many tissues against xenobiotic mols.  Its activity affects the pharmacokinetics of commonly used drugs and limits the delivery of therapeutics into tumor cells, thus contributing to multidrug resistance.  Here we present the structure of human ABCG2 detd. by cryo-electron microscopy, providing the first high-resoln. insight into a human multidrug transporter.  We visualize ABCG2 in complex with two antigen-binding fragments of the human-specific, inhibitory antibody 5D3 that recognizes extracellular loops of the transporter.  We observe two cholesterol mols. bound in the multidrug-binding pocket that is located in a central, hydrophobic, inward-facing translocation pathway between the transmembrane domains.  Combined with functional in vitro analyses, our results suggest a multidrug recognition and transport mechanism of ABCG2, rationalize disease-causing single nucleotide polymorphisms (SNP) and the allosteric inhibition by the 5D3 antibody, and provide the structural basis of cholesterol recognition by other G-subfamily ABC transporters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYOlfrPNtVhrVg90H21EOLACvtfcHk0lia9sOoSnWFHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosFWntbk%253D&md5=a3fd5157f639d2958e5abd996e2a6470</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature22345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22345%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DN.%2BM.%2BI.%26aulast%3DManolaridis%26aufirst%3DI.%26aulast%3DJackson%26aufirst%3DS.%2BM.%26aulast%3DKowal%26aufirst%3DJ.%26aulast%3DStahlberg%26aufirst%3DH.%26aulast%3DLocher%26aufirst%3DK.%2BP.%26atitle%3DStructure%2520of%2520the%2520human%2520multidrug%2520transporter%2520ABCG2%26jtitle%3DNature%26date%3D2017%26volume%3D546%26spage%3D504%26epage%3D509%26doi%3D10.1038%2Fnature22345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis of substrate recognition by the multidrug resistance protein MRP1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">1075</span>â <span class="NLM_lpage">1085.e9</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2017.01.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.cell.2017.01.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=28238471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlGrurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=1075-1085.e9&author=Z.+L.+Johnsonauthor=J.+Chen&title=Structural+basis+of+substrate+recognition+by+the+multidrug+resistance+protein+MRP1&doi=10.1016%2Fj.cell.2017.01.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Recognition by the Multidrug Resistance Protein MRP1</span></div><div class="casAuthors">Johnson, Zachary Lee; Chen, Jue</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1075-1085.e9</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The multidrug resistance protein MRP1 is an ATP-binding cassette (ABC) transporter that confers resistance to many anticancer drugs and plays a role in the disposition and efficacy of several opiates, antidepressants, statins, and antibiotics.  In addn., MRP1 regulates redox homeostasis, inflammation, and hormone secretion.  Using electron cryomicroscopy, we detd. the mol. structures of bovine MRP1 in two conformations: an apo form at 3.5 Ã without any added substrate and a complex form at 3.3 Ã with one of its physiol. substrates, leukotriene C4.  These structures show that by forming a single bipartite binding site, MRP1 can recognize a spectrum of substrates with different chem. structures.  We also obsd. large conformational changes induced by leukotriene C4, explaining how substrate binding primes the transporter for ATP hydrolysis.  Structural comparison of MRP1 and P-glycoprotein advances our understanding of the common and unique properties of these two important mols. in multidrug resistance to chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo87MmRBg2HsbVg90H21EOLACvtfcHk0liv5-KJu0wwMA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlGrurg%253D&md5=1799bd0d0180f8c524ebe97271a08c85</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.01.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.01.041%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DZ.%2BL.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520recognition%2520by%2520the%2520multidrug%2520resistance%2520protein%2520MRP1%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D1075%26epage%3D1085.e9%26doi%3D10.1016%2Fj.cell.2017.01.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span> <span> </span><span class="NLM_article-title">Catalytic transitions in the human MDR1 P-glycoprotein drug binding sites</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5125</span>â <span class="NLM_lpage">5141</span>, <span class="refDoi">Â DOI: 10.1021/bi300299z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi300299z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnslaht7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5125-5141&author=J.+G.+Wise&title=Catalytic+transitions+in+the+human+MDR1+P-glycoprotein+drug+binding+sites&doi=10.1021%2Fbi300299z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic Transitions in the Human MDR1 P-Glycoprotein Drug Binding Sites</span></div><div class="casAuthors">Wise, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5125-5141</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multidrug resistance proteins that belong to the ATP-binding cassette family like the human P-glycoprotein (ABCB1 or Pgp) are responsible for many failed cancer and antiviral chemotherapies because these membrane transporters remove the chemotherapeutics from the targeted cells.  Understanding the details of the catalytic mechanism of Pgp is therefore crit. to the development of inhibitors that might overcome these resistances.  In this work, targeted mol. dynamics techniques were used to elucidate catalytically relevant structures of Pgp.  Crystal structures of homologs in four different conformations were used as intermediate targets in the dynamics simulations.  Transitions from conformations that were wide open to the cytoplasm to transition state conformations that were wide open to the extracellular space were studied.  Twenty-six nonredundant transitional protein structures were identified from these targeted mol. dynamics simulations using evolutionary structure analyses.  Coupled movement of nucleotide binding domains (NBDs) and transmembrane domains (TMDs) that form the drug binding cavities were obsd.  Pronounced twisting of the NBDs as they approached each other as well as the quantification of a dramatic opening of the TMDs to the extracellular space as the ATP hydrolysis transition state was reached were obsd.  Docking interactions of 21 known transport ligands or inhibitors were analyzed with each of the 26 transitional structures.  Many of the docking results obtained here were validated by previously published biochem. detns.  As the ATP hydrolysis transition state was approached, drug docking in the extracellular half of the transmembrane domains seemed to be destabilized as transport ligand exit gates opened to the extracellular space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOOus-mymdBLVg90H21EOLACvtfcHk0liv5-KJu0wwMA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnslaht7k%253D&md5=0b1f63d47f37a3ea618d281c87650bc8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fbi300299z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi300299z%26sid%3Dliteratum%253Aachs%26aulast%3DWise%26aufirst%3DJ.%2BG.%26atitle%3DCatalytic%2520transitions%2520in%2520the%2520human%2520MDR1%2520P-glycoprotein%2520drug%2520binding%2520sites%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5125%26epage%3D5141%26doi%3D10.1021%2Fbi300299z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span> <span> </span><span class="NLM_article-title">Multiple drug transport pathways through human P-glycoprotein</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4374</span>â <span class="NLM_lpage">4390</span>, <span class="refDoi">Â DOI: 10.1021/acs.biochem.5b00018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b00018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=4374-4390&author=J.+W.+McCormickauthor=P.+D.+Vogelauthor=J.+G.+Wise&title=Multiple+drug+transport+pathways+through+human+P-glycoprotein&doi=10.1021%2Facs.biochem.5b00018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Drug Transport Pathways through Human P-Glycoprotein</span></div><div class="casAuthors">McCormick, James W.; Vogel, Pia D.; Wise, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">4374-4390</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-Glycoprotein (P-gp) is a plasma membrane efflux pump that is commonly assocd. with therapy resistances in cancers and infectious diseases.  P-gp can lower the intracellular concns. of many drugs to subtherapeutic levels by translocating them out of the cell.  Because of the broad range of substrates transported by P-gp, overexpression of P-gp causes multidrug resistance.  We reported previously on dynamic transitions of P-gp as it moved through conformations based on crystal structures of homologous ABCB1 proteins using in silico targeted mol. dynamics techniques.  We expanded these studies here by docking transport substrates to drug binding sites of P-gp in conformations open to the cytoplasm, followed by cycling the pump through conformations that opened to the extracellular space.  We obsd. reproducible transport of two substrates, daunorubicin and verapamil, by an av. of 11-12 Ã through the plane of the membrane as P-gp progressed through a catalytic cycle.  Methylpyrophosphate, a ligand that should not be transported by P-gp, did not show this movement through P-gp.  Drug binding to either of two subsites on P-gp appeared to det. the initial pathway used for drug movement through the membrane.  The specific side-chain interactions with drugs within each pathway seemed to be, at least in part, stochastic.  The docking and transport properties of a P-gp inhibitor, tariquidar, were also studied.  A mechanism of inhibition by tariquidar that involves stabilization of an outward open conformation with tariquidar bound in intracellular loops or at the drug binding domain of P-gp is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA2Lu18uESV7Vg90H21EOLACvtfcHk0liv5-KJu0wwMA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLrI&md5=af16b2b12607eb23bc9b88555f26a5a1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b00018%26sid%3Dliteratum%253Aachs%26aulast%3DMcCormick%26aufirst%3DJ.%2BW.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26aulast%3DWise%26aufirst%3DJ.%2BG.%26atitle%3DMultiple%2520drug%2520transport%2520pathways%2520through%2520human%2520P-glycoprotein%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D4374%26epage%3D4390%26doi%3D10.1021%2Facs.biochem.5b00018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frank, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgnia, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earl, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottesman, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambudkar, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span> <span> </span><span class="NLM_article-title">Cryo-EM analysis of the conformational landscape of human P-glycoprotein (ABCB1) during its catalytic cycle</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">35</span>â <span class="NLM_lpage">41</span>, <span class="refDoi">Â DOI: 10.1124/mol.116.104190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1124%2Fmol.116.104190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=27190212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVejs7rO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2016&pages=35-41&author=G.+A.+Frankauthor=S.+Shuklaauthor=P.+Raoauthor=M.+J.+Borgniaauthor=A.+Bartesaghiauthor=A.+Merkauthor=A.+Mobinauthor=L.+Esserauthor=L.+A.+Earlauthor=M.+M.+Gottesmanauthor=D.+Xiaauthor=S.+V.+Ambudkarauthor=S.+Subramaniam&title=Cryo-EM+analysis+of+the+conformational+landscape+of+human+P-glycoprotein+%28ABCB1%29+during+its+catalytic+cycle&doi=10.1124%2Fmol.116.104190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM analysis of the conformational landscape of human P-glycoprotein (ABCB1) during its catalytic cycle</span></div><div class="casAuthors">Frank, Gabriel A.; Shukla, Suneet; Rao, Prashant; Borgnia, Mario J.; Bartesaghi, Alberto; Merk, Alan; Mobin, Aerfa; Esser, Lothar; Earl, Lesley A.; Gottesman, Michael M.; Xia, Di; Ambudkar, Suresh V.; Subramaniam, Sriram</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-41</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The multidrug transporter P-glycoprotein (P-gp, ABCB1) is an ATP-dependent pump that mediates the efflux of structurally diverse drugs and xenobiotics across cell membranes, affecting drug pharmacokinetics and contributing to the development of multidrug resistance.  Structural information about the conformational changes in human P-gp during the ATP hydrolysis cycle has not been directly demonstrated, although mechanistic information has been inferred from biochem. and biophys. studies conducted with P-gp and its orthologs, or from structures of other ATP-binding cassette transporters.  Using single-particle cryo-electron microscopy, we report the surprising discovery that, in the absence of the transport substrate and nucleotides, human P-gp can exist in both open [nucleotide binding domains (NBDs) apart; inward facing] and closed (NBDs close; outward-facing) conformations.  We also probe conformational states of human P-gp during the catalytic cycle, and demonstrate that, following ATP hydrolysis, P-gp transitions through a complete closed conformation to a complete open conformation in the presence of ADP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1zy4LAdfsg7Vg90H21EOLACvtfcHk0lh35JTSDDdPSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVejs7rO&md5=8993b0baf446fde3801846cea3a141aa</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1124%2Fmol.116.104190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.116.104190%26sid%3Dliteratum%253Aachs%26aulast%3DFrank%26aufirst%3DG.%2BA.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DRao%26aufirst%3DP.%26aulast%3DBorgnia%26aufirst%3DM.%2BJ.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DMerk%26aufirst%3DA.%26aulast%3DMobin%26aufirst%3DA.%26aulast%3DEsser%26aufirst%3DL.%26aulast%3DEarl%26aufirst%3DL.%2BA.%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26aulast%3DXia%26aufirst%3DD.%26aulast%3DAmbudkar%26aufirst%3DS.%2BV.%26aulast%3DSubramaniam%26aufirst%3DS.%26atitle%3DCryo-EM%2520analysis%2520of%2520the%2520conformational%2520landscape%2520of%2520human%2520P-glycoprotein%2520%2528ABCB1%2529%2520during%2520its%2520catalytic%2520cycle%26jtitle%3DMol.%2520Pharmacol.%26date%3D2016%26volume%3D90%26spage%3D35%26epage%3D41%26doi%3D10.1124%2Fmol.116.104190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khunweeraphong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchler, K.</span></span> <span> </span><span class="NLM_article-title">The structure of the human ABC transporter ABCG2 reveals a novel mechanism for drug extrusion</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13767</span>, <span class="refDoi">Â DOI: 10.1038/s41598-017-11794-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fs41598-017-11794-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=29061978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC1M7js1yksQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=13767&author=N.+Khunweeraphongauthor=T.+Stocknerauthor=K.+Kuchler&title=The+structure+of+the+human+ABC+transporter+ABCG2+reveals+a+novel+mechanism+for+drug+extrusion&doi=10.1038%2Fs41598-017-11794-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of the human ABC transporter ABCG2 reveals a novel mechanism for drug extrusion</span></div><div class="casAuthors">Khunweeraphong Narakorn; Kuchler Karl; Stockner Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13767</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The human ABC transporter ABCG2 (Breast Cancer Resistance Protein, BCRP) is implicated in anticancer resistance, in detoxification across barriers and linked to gout.  Here, we generate a novel atomic model of ABCG2 using the crystal structure of ABCG5/G8.  Extensive mutagenesis verifies the structure, disclosing hitherto unrecognized essential residues and domains in the homodimeric ABCG2 transporter.  The elbow helix, the first intracellular loop (ICL1) and the nucleotide-binding domain (NBD) constitute pivotal elements of the architecture building the transmission interface that borders a central cavity which acts as a drug trap.  The transmission interface is stabilized by salt-bridge interactions between the elbow helix and ICL1, as well as within ICL1, which is essential to control the conformational switch of ABCG2 to the outward-open drug-releasing conformation.  Importantly, we propose that ICL1 operates like a molecular spring that holds the NBD dimer close to the membrane, thereby enabling efficient coupling of ATP hydrolysis during the catalytic cycle.  These novel mechanistic data open new opportunities to therapeutically target ABCG2 in the context of related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9zxYb5ouxkL0AGQpJZWpJfW6udTcc2eapTHArWlSjbbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7js1yksQ%253D%253D&md5=13b9d4f177ed8f979a173b5e2ce45fb8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-11794-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-11794-w%26sid%3Dliteratum%253Aachs%26aulast%3DKhunweeraphong%26aufirst%3DN.%26aulast%3DStockner%26aufirst%3DT.%26aulast%3DKuchler%26aufirst%3DK.%26atitle%3DThe%2520structure%2520of%2520the%2520human%2520ABC%2520transporter%2520ABCG2%2520reveals%2520a%2520novel%2520mechanism%2520for%2520drug%2520extrusion%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D13767%26doi%3D10.1038%2Fs41598-017-11794-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoghbi, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swartz, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendley, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altenberg, G. A.</span></span> <span> </span><span class="NLM_article-title">Substrate-induced conformational changes in the nucleotide-binding domains of lipid bilayer-associated P-glycoprotein during ATP hydrolysis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">20412</span>â <span class="NLM_lpage">20424</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M117.814186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1074%2Fjbc.M117.814186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=29018094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFygs77K" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=20412-20424&author=M.+E.+Zoghbiauthor=L.+Mokauthor=D.+J.+Swartzauthor=A.+Singhauthor=G.+A.+Fendleyauthor=I.+L.+Urbatschauthor=G.+A.+Altenberg&title=Substrate-induced+conformational+changes+in+the+nucleotide-binding+domains+of+lipid+bilayer-associated+P-glycoprotein+during+ATP+hydrolysis&doi=10.1074%2Fjbc.M117.814186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-induced conformational changes in the nucleotide-binding domains of lipid bilayer-associated P-glycoprotein during ATP hydrolysis</span></div><div class="casAuthors">Zoghbi, Maria E.; Mok, Leo; Swartz, Douglas J.; Singh, Anukriti; Fendley, Gregory A.; Urbatsch, Ina L.; Altenberg, Guillermo A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">20412-20424</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) is an efflux pump important in multidrug resistance of cancer cells and in detg. drug pharmacokinetics.  Pgp is a prototype ATP-binding cassette transporter with 2 nucleotide-binding domains (NBDs) that bind and hydrolyze ATP.  Conformational changes at the NBDs (the Pgp engines) lead to changes across Pgp transmembrane domains that result in substrate translocation.  According to current alternating access models (substrate-binding pocket accessible only to one side of the membrane at a time), binding of ATP promotes NBD dimerization, resulting in external accessibility of the drug-binding site (outward-facing, closed NBD conformation), and ATP hydrolysis leads to dissocn. of the NBDs with the subsequent return of the accessibility of the binding site to the cytoplasmic side (inward-facing, open NBD conformation).  However, previous work has not investigated these events under near-physiol. conditions in a lipid bilayer and in the presence of transport substrate.  Here, we used luminescence resonance energy transfer (LRET) to measure the distances between the 2 Pgp NBDs.  Pgp was labeled with LRET probes, reconstituted in lipid nanodisks, and the distance between the NBDs was measured at 37Â°.  In the presence of verapamil, a substrate that activates ATP hydrolysis, the NBDs of Pgp reconstituted in nanodiscs were never far apart during the hydrolysis cycle, and we never obsd. the NBD-NBD distances of tens of Ã that have previously been reported.  However, we found 2 main conformations that coexist in a dynamic equil. under all conditions studied.  These observations highlight the importance of performing studies of efflux pumps under near-physiol. conditions, in a lipid bilayer, at 37Â°, and during substrate-stimulated hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLqpb0i6gl3bVg90H21EOLACvtfcHk0lh35JTSDDdPSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFygs77K&md5=67b866ad533c01e3c2d8730b35d2b2e7</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.814186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.814186%26sid%3Dliteratum%253Aachs%26aulast%3DZoghbi%26aufirst%3DM.%2BE.%26aulast%3DMok%26aufirst%3DL.%26aulast%3DSwartz%26aufirst%3DD.%2BJ.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DFendley%26aufirst%3DG.%2BA.%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DAltenberg%26aufirst%3DG.%2BA.%26atitle%3DSubstrate-induced%2520conformational%2520changes%2520in%2520the%2520nucleotide-binding%2520domains%2520of%2520lipid%2520bilayer-associated%2520P-glycoprotein%2520during%2520ATP%2520hydrolysis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D20412%26epage%3D20424%26doi%3D10.1074%2Fjbc.M117.814186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follit, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span> <span> </span><span class="NLM_article-title">In silico screening for inhibitors of P-glycoprotein that target the nucleotide binding domains</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">716</span>â <span class="NLM_lpage">726</span>, <span class="refDoi">Â DOI: 10.1124/mol.114.095414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1124%2Fmol.114.095414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=25270578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVSmtbfM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=716-726&author=F.+K.+Brewerauthor=C.+A.+Follitauthor=P.+D.+Vogelauthor=J.+G.+Wise&title=In+silico+screening+for+inhibitors+of+P-glycoprotein+that+target+the+nucleotide+binding+domains&doi=10.1124%2Fmol.114.095414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">In silico screening for inhibitors of P-glycoprotein that target the nucleotide binding domains</span></div><div class="casAuthors">Brewer, Frances K.; Follit, Courtney A.; Vogel, Pia D.; Wise, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">716-726, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Multidrug resistances and the failure of chemotherapies are often caused by the expression or overexpression of ATP-binding cassette transporter proteins such as the multidrug resistance protein, P-glycoprotein (P-gp).  P-gp is expressed in the plasma membrane of many cell types and protects cells from accumulation of toxins.  P-gp uses ATP hydrolysis to catalyze the transport of a broad range of mostly hydrophobic compds. across the plasma membrane and out of the cell.  During cancer chemotherapy, the administration of therapeutics often selects for cells which overexpress P-gp, thereby creating populations of cancer cells resistant to a variety of chem. unrelated chemotherapeutics.  The present study describes extremely high-throughput, massively parallel in silico ligand docking studies aimed at identifying reversible inhibitors of ATP hydrolysis that target the nucleotide-binding domains of P-gp.  We used a structural model of human P-gp that we obtained from mol. dynamics expts. as the protein target for ligand docking.  We employed a novel approach of subtractive docking expts. that identified ligands that bound predominantly to the nucleotide-binding domains but not the drug-binding domains of P-gp.  Four compds. were found that inhibit ATP hydrolysis by P-gp.  Using ESR spectroscopy, we showed that at least three of these compds. affected nucleotide binding to the transporter.  These studies represent a successful proof of principle demonstrating the potential of targeted approaches for identifying specific inhibitors of P-gp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-lF2vv6jAJLVg90H21EOLACvtfcHk0lh4I3rgapE_NA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVSmtbfM&md5=6775ef2590f40a0763f0386cce8f4613</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.095414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.095414%26sid%3Dliteratum%253Aachs%26aulast%3DBrewer%26aufirst%3DF.%2BK.%26aulast%3DFollit%26aufirst%3DC.%2BA.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26aulast%3DWise%26aufirst%3DJ.%2BG.%26atitle%3DIn%2520silico%2520screening%2520for%2520inhibitors%2520of%2520P-glycoprotein%2520that%2520target%2520the%2520nucleotide%2520binding%2520domains%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D86%26spage%3D716%26epage%3D726%26doi%3D10.1124%2Fmol.114.095414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Follit, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span> <span> </span><span class="NLM_article-title">In silico identified targeted inhibitors of P-glycoprotein overcome multidrug resistance in human cancer cells in culture</span>. <i>Pharmacol. Res. Perspect.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e00170</span> <span class="refDoi">Â DOI: 10.1002/prp2.170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fprp2.170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=26516582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCjurjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2015&author=C.+A.+Follitauthor=F.+K.+Brewerauthor=J.+G.+Wiseauthor=P.+D.+Vogel&title=In+silico+identified+targeted+inhibitors+of+P-glycoprotein+overcome+multidrug+resistance+in+human+cancer+cells+in+culture&doi=10.1002%2Fprp2.170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">In silico identified targeted inhibitors of P-glycoprotein overcome multidrug resistance in human cancer cells in culture</span></div><div class="casAuthors">Follit, Courtney A.; Brewer, Frances K.; Wise, John G.; Vogel, Pia D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology Research & Perspectives</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e00170/1-e00170/13</span>CODEN:
                <span class="NLM_cas:coden">PRPHCA</span>;
        ISSN:<span class="NLM_cas:issn">2052-1707</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Failure of cancer chemotherapies is often linked to the over expression of ABC efflux transporters like the multidrug resistance P-glycoprotein (P-gp).  P-gp expression in cells leads to the elimination of a variety of chem. unrelated, mostly cytotoxic compds.  Administration of chemotherapeutics during therapy frequently selects for cells that over express P-gp and are therefore capable of robustly exporting diverse compds., including chemotherapeutics, from the cells.  P-gp thus confers multidrug resistance to a majority of drugs currently available for the treatment of cancers and diseases like HIV/AIDS.  The search for P-gp inhibitors for use as co-therapeutics to combat multidrug resistances has had little success to date.  In a previous study (Brewer et al., Mol Pharmacol 86: 716-726, 2014), we described how ultrahigh throughput computational searches led to the identification of four drug-like mols. that specifically interfere with the energy harvesting steps of substrate transport and inhibit P-gp catalyzed ATP hydrolysis in vitro.  In the present study, we demonstrate that three of these compds. reversed P-gp-mediated multidrug resistance of cultured prostate cancer cells to restore sensitivity comparable to naive prostate cancer cells to the chemotherapeutic drug, paclitaxel.  Potentiation concns. of the inhibitors were <3 Î¼mol/L.  The inhibitors did not exhibit significant toxicity to noncancerous cells at concns. where they reversed multidrug resistance in cancerous cells.  Our results indicate that these compds. with novel mechanisms of P-gp inhibition are excellent leads for the development of co-therapeutics for the treatment of multidrug resistances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokpFCstWfMq7Vg90H21EOLACvtfcHk0lh4I3rgapE_NA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCjurjE&md5=92d1e01b1e99b1d73e3f21e60e8485aa</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fprp2.170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprp2.170%26sid%3Dliteratum%253Aachs%26aulast%3DFollit%26aufirst%3DC.%2BA.%26aulast%3DBrewer%26aufirst%3DF.%2BK.%26aulast%3DWise%26aufirst%3DJ.%2BG.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26atitle%3DIn%2520silico%2520identified%2520targeted%2520inhibitors%2520of%2520P-glycoprotein%2520overcome%2520multidrug%2520resistance%2520in%2520human%2520cancer%2520cells%2520in%2520culture%26jtitle%3DPharmacol.%2520Res.%2520Perspect.%26date%3D2015%26volume%3D3%26doi%3D10.1002%2Fprp2.170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nanayakkara, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follit, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, J. G.</span></span> <span> </span><span class="NLM_article-title">Targeted inhibitors of P-glycoprotein increase chemotherapeutic-induced mortality of multidrug resistant tumor cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">967</span>, <span class="refDoi">Â DOI: 10.1038/s41598-018-19325-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fs41598-018-19325-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=29343829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC1MvhslSrtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=967&author=A.+K.+Nanayakkaraauthor=C.+A.+Follitauthor=G.+Chenauthor=N.+S.+Williamsauthor=P.+D.+Vogelauthor=J.+G.+Wise&title=Targeted+inhibitors+of+P-glycoprotein+increase+chemotherapeutic-induced+mortality+of+multidrug+resistant+tumor+cells&doi=10.1038%2Fs41598-018-19325-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted inhibitors of P-glycoprotein increase chemotherapeutic-induced mortality of multidrug resistant tumor cells</span></div><div class="casAuthors">Nanayakkara Amila K; Follit Courtney A; Chen Gang; Vogel Pia D; Wise John G; Nanayakkara Amila K; Follit Courtney A; Chen Gang; Vogel Pia D; Wise John G; Williams Noelle S</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">967</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Overexpression of ATP-binding cassette (ABC) transporters is often linked to multidrug resistance (MDR) in cancer chemotherapies.  P-glycoprotein (P-gp) is one of the best studied drug transporters associated with MDR.  There are currently no approved drugs available for clinical use in cancer chemotherapies to reverse MDR by inhibiting P-glycoprotein.  Using computational studies, we previously identified several compounds that inhibit P-gp by targeting its nucleotide binding domain and avoiding its drug binding domains.  Several of these compounds showed successful MDR reversal when tested on a drug resistant prostate cancer cell line.  Using conventional two-dimensional cell culture of MDR ovarian and prostate cancer cells and three dimensional prostate cancer microtumor spheroids, we demonstrated here that co-administration with chemotherapeutics significantly decreased cell viability and survival as well as cell motility.  The P-gp inhibitors were not observed to be toxic on their own.  The inhibitors increased cellular retention of chemotherapeutics and reporter compounds known to be transport substrates of P-gp.  We also showed that these compounds are not transport substrates of P-gp and that two of the three inhibit P-gp, but not the closely related ABC transporter, ABCG2/BCRP.  The results presented suggest that these P-gp inhibitors may be promising leads for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7izekEYEuo42y4bB4mnvYfW6udTcc2eaOlIs4mOtBQLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvhslSrtg%253D%253D&md5=dd83baacadbdf694e71154a563b8c55f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-19325-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-19325-x%26sid%3Dliteratum%253Aachs%26aulast%3DNanayakkara%26aufirst%3DA.%2BK.%26aulast%3DFollit%26aufirst%3DC.%2BA.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DWilliams%26aufirst%3DN.%2BS.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26aulast%3DWise%26aufirst%3DJ.%2BG.%26atitle%3DTargeted%2520inhibitors%2520of%2520P-glycoprotein%2520increase%2520chemotherapeutic-induced%2520mortality%2520of%2520multidrug%2520resistant%2520tumor%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D967%26doi%3D10.1038%2Fs41598-018-19325-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltanshahi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawless, M. S.</span></span> <span> </span><span class="NLM_article-title">Virtual compound libraries: a new approach to decision making in molecular discovery research</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1010</span>â <span class="NLM_lpage">1023</span>, <span class="refDoi">Â DOI: 10.1021/ci9800209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9800209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK1cXmtVGgtbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=1010-1023&author=R.+D.+Cramerauthor=D.+E.+Pattersonauthor=R.+D.+Clarkauthor=F.+Soltanshahiauthor=M.+S.+Lawless&title=Virtual+compound+libraries%3A+a+new+approach+to+decision+making+in+molecular+discovery+research&doi=10.1021%2Fci9800209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Compound Libraries: A New Approach to Decision Making in Molecular Discovery Research</span></div><div class="casAuthors">Cramer, Richard D.; Patterson, David E.; Clark, Robert D.; Soltanshahi, Farhad; Lawless, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1010-1023</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual compd. libraries, descriptions of all of the structures that might be produced by specified transformations involving specified reagents, are esp. useful in mol. discovery when suitably fast and relevant searching techniques are available.  Issues to be considered include fundamental data structures, neighborhood searching principles, useful searching approaches and techniques, library definition and construction, algorithmic details of library comparison, and user interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjcDetEYSHo7Vg90H21EOLACvtfcHk0liQdsbU5orJEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtVGgtbg%253D&md5=389423dd71db05df0470fffbf147aa39</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fci9800209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9800209%26sid%3Dliteratum%253Aachs%26aulast%3DCramer%26aufirst%3DR.%2BD.%26aulast%3DPatterson%26aufirst%3DD.%2BE.%26aulast%3DClark%26aufirst%3DR.%2BD.%26aulast%3DSoltanshahi%26aufirst%3DF.%26aulast%3DLawless%26aufirst%3DM.%2BS.%26atitle%3DVirtual%2520compound%2520libraries%253A%2520a%2520new%2520approach%2520to%2520decision%2520making%2520in%2520molecular%2520discovery%2520research%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D1010%26epage%3D1023%26doi%3D10.1021%2Fci9800209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de AraÃºjo-JÃºnior, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duranton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Justiano-Basaran, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugnier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourguignon, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Design of small-sized libraries by combinatorial assembly of linkers and functional groups to a given scaffold: application to the structure-based optimization of a phosphodiesterase 4 inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">3816</span>â <span class="NLM_lpage">3822</span>, <span class="refDoi">Â DOI: 10.1021/jm050063y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050063y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFKjsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3816-3822&author=M.+Krierauthor=J.+X.+de+Ara%C3%BAjo-J%C3%BAniorauthor=M.+Schmittauthor=J.+Durantonauthor=H.+Justiano-Basaranauthor=C.+Lugnierauthor=J.-J.+Bourguignonauthor=D.+Rognan&title=Design+of+small-sized+libraries+by+combinatorial+assembly+of+linkers+and+functional+groups+to+a+given+scaffold%3A+application+to+the+structure-based+optimization+of+a+phosphodiesterase+4+inhibitor&doi=10.1021%2Fjm050063y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Small-Sized Libraries by Combinatorial Assembly of Linkers and Functional Groups to a Given Scaffold: Application to the Structure-Based Optimization of a Phosphodiesterase 4 Inhibitor</span></div><div class="casAuthors">Krier, Mireille; De Araujo-Junior, Joao X.; Schmitt, Martine; Duranton, Jerome; Justiano-Basaran, Helene; Lugnier, Claire; Bourguignon, Jean-Jacques; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3816-3822</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Combinatorial chem. and library design have been reconciled by applying simple medicinal chem. concepts to virtual library design.  The herein reported "Scaffold-Linker-Functional Group" (SLF) approach has the aim to maximize diversity while minimizing the size of a scaffold-focused library with the aid of simple mol. variations in order to identify crit. pharmacophoric elements.  Straightforward rules define the way of assembling three building blocks: a conserved scaffold, a variable linker, and a variable functional group.  By carefully selecting a limited no. of functional groups not overlapping in pharmacophoric space, the size of the library is kept to a few hundred.  As an application of the SLF approach, a small-sized combinatorial library (320 compds.) was derived from the scaffold of the known phosphodiesterase 4 inhibitor zardaverine.  The most interesting analogs were further prioritized for synthesis and enzyme inhibition by FlexX docking to the X-ray structure of the enzyme, leading to a 900-fold increased affinity within nine synthesized compds. and a single screening round.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQS0szYq_dT7Vg90H21EOLACvtfcHk0liQdsbU5orJEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFKjsbg%253D&md5=757e7fa03b4ed33f251fd33126e947c4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm050063y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050063y%26sid%3Dliteratum%253Aachs%26aulast%3DKrier%26aufirst%3DM.%26aulast%3Dde%2BAra%25C3%25BAjo-J%25C3%25BAnior%26aufirst%3DJ.%2BX.%26aulast%3DSchmitt%26aufirst%3DM.%26aulast%3DDuranton%26aufirst%3DJ.%26aulast%3DJustiano-Basaran%26aufirst%3DH.%26aulast%3DLugnier%26aufirst%3DC.%26aulast%3DBourguignon%26aufirst%3DJ.-J.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DDesign%2520of%2520small-sized%2520libraries%2520by%2520combinatorial%2520assembly%2520of%2520linkers%2520and%2520functional%2520groups%2520to%2520a%2520given%2520scaffold%253A%2520application%2520to%2520the%2520structure-based%2520optimization%2520of%2520a%2520phosphodiesterase%25204%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3816%26epage%3D3822%26doi%3D10.1021%2Fjm050063y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttamchandani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S. Q.</span></span> <span> </span><span class="NLM_article-title">Rapid assembly and in situ screening of bidentate inhibitors of protein tyrosine phosphatases</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">713</span>â <span class="NLM_lpage">716</span>, <span class="refDoi">Â DOI: 10.1021/ol052895w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol052895w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD28XntFGjsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=713-716&author=R.+Srinivasanauthor=M.+Uttamchandaniauthor=S.+Q.+Yao&title=Rapid+assembly+and+in+situ+screening+of+bidentate+inhibitors+of+protein+tyrosine+phosphatases&doi=10.1021%2Fol052895w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Assembly and in Situ Screening of Bidentate Inhibitors of Protein Tyrosine Phosphatases</span></div><div class="casAuthors">Srinivasan, Rajavel; Uttamchandani, Mahesh; Yao, Shao Q.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">713-716</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have successfully designed and synthesized a small library of protein tyrosine phosphatase (PTP) inhibitors, in which the so-called click chem. or Cu(I)-catalyzed 1,3-dipolar alkyne-azide coupling reaction was carried out for rapid assembly of 66 different bidentate compds.  Subsequent in situ enzymic screening revealed a potential PTP1B inhibitor (IC50 = 4.7 Î¼M) which is 10-100 fold more potent than other PTPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcj-jI1-_paLVg90H21EOLACvtfcHk0liQdsbU5orJEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntFGjsA%253D%253D&md5=7fbe6b911bc26b5f30d1ff3d592091e1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fol052895w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol052895w%26sid%3Dliteratum%253Aachs%26aulast%3DSrinivasan%26aufirst%3DR.%26aulast%3DUttamchandani%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DS.%2BQ.%26atitle%3DRapid%2520assembly%2520and%2520in%2520situ%2520screening%2520of%2520bidentate%2520inhibitors%2520of%2520protein%2520tyrosine%2520phosphatases%26jtitle%3DOrg.%2520Lett.%26date%3D2006%26volume%3D8%26spage%3D713%26epage%3D716%26doi%3D10.1021%2Fol052895w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durrant, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">AutoClickChem: click chemistry in silico</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002397</span> <span class="refDoi">Â DOI: 10.1371/journal.pcbi.1002397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1371%2Fjournal.pcbi.1002397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=22438795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslahtLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=J.+D.+Durrantauthor=J.+A.+McCammon&title=AutoClickChem%3A+click+chemistry+in+silico&doi=10.1371%2Fjournal.pcbi.1002397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">AutoClickChem: click chemistry in silico</span></div><div class="casAuthors">Durrant, Jacob D.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1002397</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Academic researchers and many in industry often lack the financial resources available to scientists working in "big pharma".  High costs include those assocd. with high-throughput screening and chem. synthesis.  In order to address these challenges, many researchers have in part turned to alternate methodologies.  Virtual screening, for example, often substitutes for high-throughput screening and click chem. ensures that chem. synthesis is fast, cheap and comparatively easy.  Though both in silico screening and click chem. seek to make drug discovery more feasible, it is not yet routine to couple these two methodologies.  We here present a novel computer algorithm, called AutoClickChem, capable of performing many click-chem. reactions in silico.  AutoClickChem can be used to produce large combinatorial libraries of compd. models for use in virtual screens.  As the compds. of these libraries are constructed according to the reactions of click chem., they can be easily synthesized for subsequent testing in biochem. assays.  Addnl., in silico modeling of click-chem. products may prove useful in rational drug design and drug optimization.  AutoClickChem is based on the pymol. toolbox, a framework that may facilitate the development of future python-based programs that require the manipulation of mol. models.  Both the pymol. toolbox and AutoClickChem are released under the GNU General Public License version 3 and are available for download.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo44OEkiGkWVrVg90H21EOLACvtfcHk0lh1ti7Gd9LuUQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslahtLo%253D&md5=53645fe62d0cc696c245b32bca2e2fe0</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002397%26sid%3Dliteratum%253Aachs%26aulast%3DDurrant%26aufirst%3DJ.%2BD.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DAutoClickChem%253A%2520click%2520chemistry%2520in%2520silico%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.pcbi.1002397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namiki, M.</span></span> <span> </span><span class="NLM_article-title">The establishment of two paclitaxel-resistant prostate cancer cell lines and the mechanisms of paclitaxel resistance with two cell lines</span>. <i>Prostate</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">955</span>â <span class="NLM_lpage">967</span>, <span class="refDoi">Â DOI: 10.1002/pros.20581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fpros.20581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=17440963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmvVKqt7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=955-967&author=M.+Takedaauthor=A.+Mizokamiauthor=K.+Mamiyaauthor=Y.+Q.+Liauthor=J.+Zhangauthor=E.+T.+Kellerauthor=M.+Namiki&title=The+establishment+of+two+paclitaxel-resistant+prostate+cancer+cell+lines+and+the+mechanisms+of+paclitaxel+resistance+with+two+cell+lines&doi=10.1002%2Fpros.20581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The establishment of two paclitaxel-resistant prostate cancer cell lines and the mechanisms of paclitaxel resistance with two cell lines</span></div><div class="casAuthors">Takeda, Masashi; Mizokami, Atsushi; Mamiya, Kiminori; Li, You Qiang; Zhang, Jian; Keller, Evan T.; Namiki, Mikio</div><div class="citationInfo"><span class="NLM_cas:title">Prostate (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">955-967</span>CODEN:
                <span class="NLM_cas:coden">PRSTDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-4137</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">BACKGROUND: Although paclitaxel is used for hormone-resistant prostate cancer, relapse definitely occurs later.  Details of the mol. mechanism responsible for paclitaxel-resistance remain unclear.  METHODS: We established paclitaxel-resistant cells, DU145-TxR and PC-3-TxR from parent DU145 and PC-3.  To characterize these cells, we examd. cross-resistance to other anticancer drugs.  Expression of several potential genes that had been related to drug-resistance was compared with parent cells by RT-PCR and Western blotting.  Methylation anal. of multiple drug resistance (MDR1) promoter was carried out using bisulfite-modified DNA from cell lines.  Knockdown expts. using small interfering RNA (siRNA) were also performed to confirm responsibility of drug-resistance.  Finally, cDNA microarray was performed to quantify gene expression in PC-3 and PC-3-TxR cells.  RESULTS: The IC50 for paclitaxel in DU145-TxR and PC-3-TxR was 34.0- and 43.4-fold higher than that in both parent cells, resp.  Both cells showed cross-resistance to some drugs, but not to VP-16 and cisplatin.  Methylation anal. revealed that methylated CpG sites of MDR1 promoter in DU145 and PC-3 cells were demethylated in DU145-TxR cells, but not in PC-3-TxR cells.  Knockdown of P-glycoprotein (P-gp), which was up-regulated in resistant cells, by MDR-1 siRNA restored paclitaxel sensitivity in DU145-TxR but not in PC-3-TxR, indicating that up-regulation of P-gp was not always main cause of paclitaxel-resistance.  Microarray anal. identified 201 (1.34%) up-regulated genes and 218 (1.45%) out of screened genes in PC-3-TxR.  CONCLUSIONS: Our data will provide mol. mechanisms of paclitaxel-resistance and be useful for screening target genes to diagnose paclitaxel sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCs6firNNZ87Vg90H21EOLACvtfcHk0lh1ti7Gd9LuUQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmvVKqt7c%253D&md5=1b0e44bab963a7b82abc3eb6c979ca24</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fpros.20581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpros.20581%26sid%3Dliteratum%253Aachs%26aulast%3DTakeda%26aufirst%3DM.%26aulast%3DMizokami%26aufirst%3DA.%26aulast%3DMamiya%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DY.%2BQ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DE.%2BT.%26aulast%3DNamiki%26aufirst%3DM.%26atitle%3DThe%2520establishment%2520of%2520two%2520paclitaxel-resistant%2520prostate%2520cancer%2520cell%2520lines%2520and%2520the%2520mechanisms%2520of%2520paclitaxel%2520resistance%2520with%2520two%2520cell%2520lines%26jtitle%3DProstate%26date%3D2007%26volume%3D67%26spage%3D955%26epage%3D967%26doi%3D10.1002%2Fpros.20581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolstad, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span> <span> </span><span class="NLM_article-title">ZINC: A free tool to discover chemistry for biology</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1757</span>â <span class="NLM_lpage">1768</span>, <span class="refDoi">Â DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+A+free+tool+to+discover+chemistry+for+biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lh1ti7Gd9LuUQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520A%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1074</span>â <span class="NLM_lpage">1080</span>, <span class="refDoi">Â DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+tool+for+ligand+coordinate+and+restraint+generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0lh1ti7Gd9LuUQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3Delectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520tool%2520for%2520ligand%2520coordinate%2520and%2520restraint%2520generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span> <span> </span><span class="NLM_article-title">Specific 1,2-Hydride Shift in the Boron Trifluoride Catalyzed Reactions of Aromatic Aldehydes with Diazoacetonitrile: Simple Synthesis of Î²-Ketonitriles</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_fpage">6380</span>â <span class="NLM_lpage">6384</span>, <span class="refDoi">Â DOI: 10.1002/ejoc.201402901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fejoc.201402901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWjt7rL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&pages=6380-6384&author=Z.+Yangauthor=K.-I.+Sonauthor=S.+Liauthor=B.+Zhouauthor=J.+Xu&title=Specific+1%2C2-Hydride+Shift+in+the+Boron+Trifluoride+Catalyzed+Reactions+of+Aromatic+Aldehydes+with+Diazoacetonitrile%3A+Simple+Synthesis+of+%CE%B2-Ketonitriles&doi=10.1002%2Fejoc.201402901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Specific 1,2-Hydride Shift in the Boron Trifluoride Catalyzed Reactions of Aromatic Aldehydes with Diazoacetonitrile: Simple Synthesis of Î²-Ketonitriles</span></div><div class="casAuthors">Yang, Zhanhui; Son, Kwon-Il; Li, Siqi; Zhou, Bingnan; Xu, Jiaxi</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">6380-6384</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of Î²-ketonitriles RC(O)CH2CN [R = 3,4-(CH3)2C6H3, naphth-1-yl, thiophen-1-yl, furan-1-yl, etc.] was synthesized within 30 min under mild conditions through the BF3Â·OEt2-catalyzed addn. reactions of diazoacetonitrile to arom. aldehydes and subsequent 1,2-hydride shift.  This method is advantageous in that it is operationally simple, mild and metal-free conditions are used, the substrate scope is wide, and the products are obtained in moderate to high yields (up to 81%).  Addnl., Î³,Î´-unsatd. Î²-ketonitriles are also accessible by this method by using cinnamaldehydes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwV7CKaBQCHbVg90H21EOLACvtfcHk0ljolQzkWzKwdA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWjt7rL&md5=c81ae13233ada462fe2a9826d47afe53</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201402901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201402901%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DSon%26aufirst%3DK.-I.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DJ.%26atitle%3DSpecific%25201%252C2-Hydride%2520Shift%2520in%2520the%2520Boron%2520Trifluoride%2520Catalyzed%2520Reactions%2520of%2520Aromatic%2520Aldehydes%2520with%2520Diazoacetonitrile%253A%2520Simple%2520Synthesis%2520of%2520%25CE%25B2-Ketonitriles%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2014%26spage%3D6380%26epage%3D6384%26doi%3D10.1002%2Fejoc.201402901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, F. F.</span></span> <span> </span><span class="NLM_article-title">An efficient one-pot synthesis of N-(1,3-diphenyl-1H-pyrazol-5-yl)amides</span>. <i>J. Heterocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">831</span>â <span class="NLM_lpage">837</span>, <span class="refDoi">Â DOI: 10.1002/jhet.343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fjhet.343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Gis7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2010&pages=831-837&author=W.+N.+Suauthor=T.+P.+Linauthor=K.+M.+Chengauthor=K.+C.+Sungauthor=S.+K.+Linauthor=F.+F.+Wong&title=An+efficient+one-pot+synthesis+of+N-%281%2C3-diphenyl-1H-pyrazol-5-yl%29amides&doi=10.1002%2Fjhet.343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient one-pot synthesis of N-(1,3-diphenyl-1H-pyrazol-5-yl)amides</span></div><div class="casAuthors">Su, Wei-Nien; Lin, Tsung-Ping; Cheng, Kaung-Min; Sung, Kuan-Chin; Lin, Shao-Kai; Wong, Fung Fuh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">831-837</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">1943-5193</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A "one-pot" method for the synthesis of N-(1,3-diphenyl-1H-pyrazol-5-yl)amides was developed by cyclization of benzoylacetonitrile and phenylhydrazine in neat condition followed by acylation.  The corresponding N-(1,3-diphenyl-1H-pyrazol-5-yl)amides, e.g., I (R = H, Me, MeO, Cl, NO2), were provided in good to excellent yields (70-90%).  The significant advantages of the new synthetic method are excellent yields and simple work-up procedure without isolation and purifn. of intermediary 5-amino-1,3-diphenyl pyrazole.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCc88mn8mp5rVg90H21EOLACvtfcHk0ljolQzkWzKwdA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Gis7c%253D&md5=157fc13aa9fd5dc747d10f39709c4da7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjhet.343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.343%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DW.%2BN.%26aulast%3DLin%26aufirst%3DT.%2BP.%26aulast%3DCheng%26aufirst%3DK.%2BM.%26aulast%3DSung%26aufirst%3DK.%2BC.%26aulast%3DLin%26aufirst%3DS.%2BK.%26aulast%3DWong%26aufirst%3DF.%2BF.%26atitle%3DAn%2520efficient%2520one-pot%2520synthesis%2520of%2520N-%25281%252C3-diphenyl-1H-pyrazol-5-yl%2529amides%26jtitle%3DJ.%2520Heterocyclic%2520Chem.%26date%3D2010%26volume%3D47%26spage%3D831%26epage%3D837%26doi%3D10.1002%2Fjhet.343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilton, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motherwell, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selwood, D. L.</span></span> <span> </span><span class="NLM_article-title">Observations on the reactivity of thiyl radicals derived from 3,6-epidithiodiketopiperazine-2,5-diones and related congeners</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2239</span>â <span class="NLM_lpage">2242</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2005.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.bmcl.2005.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=15837301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlWqs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=2239-2242&author=S.+T.+Hiltonauthor=W.+B.+Motherwellauthor=P.+Potierauthor=C.+Pradetauthor=D.+L.+Selwood&title=Observations+on+the+reactivity+of+thiyl+radicals+derived+from+3%2C6-epidithiodiketopiperazine-2%2C5-diones+and+related+congeners&doi=10.1016%2Fj.bmcl.2005.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Observations on the reactivity of thiyl radicals derived from 3,6-epidithiodiketopiperazine-2,5-diones and related congeners</span></div><div class="casAuthors">Hilton, S. T.; Motherwell, W. B.; Potier, P.; Pradet, C.; Selwood, D. L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2239-2242</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A range of thiyl radicals derived from the reduced form of epidithiodiketopiperazines (ETPs) act as polarity reversal catalysts for the hydrosilylation of an enol lactone but not for H-atom abstraction from a model ribose ester.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSDJW9u-rjS7Vg90H21EOLACvtfcHk0lg5c54s8VYnJQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlWqs7g%253D&md5=a492f94d4a3448a54de140c8e1be00f6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DHilton%26aufirst%3DS.%2BT.%26aulast%3DMotherwell%26aufirst%3DW.%2BB.%26aulast%3DPotier%26aufirst%3DP.%26aulast%3DPradet%26aufirst%3DC.%26aulast%3DSelwood%26aufirst%3DD.%2BL.%26atitle%3DObservations%2520on%2520the%2520reactivity%2520of%2520thiyl%2520radicals%2520derived%2520from%25203%252C6-epidithiodiketopiperazine-2%252C5-diones%2520and%2520related%2520congeners%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D2239%26epage%3D2242%26doi%3D10.1016%2Fj.bmcl.2005.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krushinski, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beedle, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leander, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathbun, R. C.</span></span> <span> </span><span class="NLM_article-title">Structure-activity-relationships of (arylalkyl)imidazole anticonvulsants - comparison of the (fluorenylalkyl)imidazoles with nafimidone and denzimol</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1577</span>â <span class="NLM_lpage">1586</span>, <span class="refDoi">Â DOI: 10.1021/jm00159a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00159a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaL28XkvFOgurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=1577-1586&author=D.+W.+Robertsonauthor=J.+H.+Krushinskiauthor=E.+E.+Beedleauthor=J.+D.+Leanderauthor=D.+T.+Wongauthor=R.+C.+Rathbun&title=Structure-activity-relationships+of+%28arylalkyl%29imidazole+anticonvulsants+-+comparison+of+the+%28fluorenylalkyl%29imidazoles+with+nafimidone+and+denzimol&doi=10.1021%2Fjm00159a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships of (arylalkyl)imidazole anticonvulsants:  comparison of the (fluorenylalkyl)imidazoles with nafimidone and denzimol</span></div><div class="casAuthors">Robertson, David W.; Krushinski, Joseph H.; Beedle, E. E.; Leander, J. David; Wong, David T.; Rathbun, R. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1577-86</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The title compds., e.g. I [R = benzo[b]thien-2-yl, benzofuran-2-yl, phenanthren-2-yl, 9,10-dihydrophenanthren-3-yl, X = O; R = naphthalen-2-yl, X = H2] and II (R1 = H, 9-Me, 7-MeO, 7-Et; R2 = H, 2-Me, 4-Me; X = O, HO,H, OCH2CH2O), were prepd. as anticonvulsants.  Thus, 1-(9H-fluoren-2-yl)ethanone was treated with CuBr2 to give the 2-bromo deriv., which was treated with 2-methylimidazole to give II (R1 = H, R2 = 2-Me, X = O).  II (R1 = R2 = H, X = O) was twice as potent as nafimidone in inhibiting maximal electroshock seizures in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQjjdvfXE7krVg90H21EOLACvtfcHk0lg5c54s8VYnJQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XkvFOgurw%253D&md5=d5733db2e6ed41e3acda6cd1456e62ae</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm00159a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00159a004%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DD.%2BW.%26aulast%3DKrushinski%26aufirst%3DJ.%2BH.%26aulast%3DBeedle%26aufirst%3DE.%2BE.%26aulast%3DLeander%26aufirst%3DJ.%2BD.%26aulast%3DWong%26aufirst%3DD.%2BT.%26aulast%3DRathbun%26aufirst%3DR.%2BC.%26atitle%3DStructure-activity-relationships%2520of%2520%2528arylalkyl%2529imidazole%2520anticonvulsants%2520-%2520comparison%2520of%2520the%2520%2528fluorenylalkyl%2529imidazoles%2520with%2520nafimidone%2520and%2520denzimol%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1986%26volume%3D29%26spage%3D1577%26epage%3D1586%26doi%3D10.1021%2Fjm00159a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Meerloo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspers, G. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloos, J.</span></span> <span> </span><span class="NLM_article-title">Cell sensitivity assays: the MTT assay</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>731</i></span>,  <span class="NLM_fpage">237</span>â <span class="NLM_lpage">245</span>, <span class="refDoi">Â DOI: 10.1007/978-1-61779-080-5_20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1007%2F978-1-61779-080-5_20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=21516412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVyqtbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=731&publication_year=2011&pages=237-245&author=J.+van%0AMeerlooauthor=G.+J.+L.+Kaspersauthor=J.+Cloos&title=Cell+sensitivity+assays%3A+the+MTT+assay&doi=10.1007%2F978-1-61779-080-5_20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Cell sensitivity assays: the MTT assay</span></div><div class="casAuthors">van Meerloo, Johan; Kaspers, Gertjan J. L.; Cloos, Jacqueline</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">731</span>
        (<span class="NLM_cas:issue">Cancer Cell Culture</span>),
    <span class="NLM_cas:pages">237-245</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The MTT (3-[4,5-dimethylthiazol-2-yl]-2,5 di-Ph tetrazolium bromide) assay is based on the conversion of MTT into formazan crystals by living cells, which dets. mitochondrial activity.  Since for most cell populations the total mitochondrial activity is related to the no. of viable cells, this assay is broadly used to measure the in vitro cytotoxic effects of drugs on cell lines or primary patient cells.  In this chapter the protocol of the assay is described including important considerations relevant for each step of the assay as well as its limitations and possible applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprO-kUg91O_7Vg90H21EOLACvtfcHk0lg5c54s8VYnJQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVyqtbk%253D&md5=f88b391a65df5cb01b3bba486c55ce96</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-080-5_20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-080-5_20%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BMeerloo%26aufirst%3DJ.%26aulast%3DKaspers%26aufirst%3DG.%2BJ.%2BL.%26aulast%3DCloos%26aufirst%3DJ.%26atitle%3DCell%2520sensitivity%2520assays%253A%2520the%2520MTT%2520assay%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D731%26spage%3D237%26epage%3D245%26doi%3D10.1007%2F978-1-61779-080-5_20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berridge, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, A. S.</span></span> <span> </span><span class="NLM_article-title">Characterization of the cellular reduction of 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT): subcellular localization, substrate dependence, and involvement of mitochondrial electron transport in MTT reduction</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>303</i></span>,  <span class="NLM_fpage">474</span>â <span class="NLM_lpage">482</span>, <span class="refDoi">Â DOI: 10.1006/abbi.1993.1311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1006%2Fabbi.1993.1311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=8390225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK3sXkvFSmsL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=1993&pages=474-482&author=M.+V.+Berridgeauthor=A.+S.+Tan&title=Characterization+of+the+cellular+reduction+of+3-%284%2C5-dimethylthiazol-2-yl%29-2%2C5-diphenyltetrazolium+bromide+%28MTT%29%3A+subcellular+localization%2C+substrate+dependence%2C+and+involvement+of+mitochondrial+electron+transport+in+MTT+reduction&doi=10.1006%2Fabbi.1993.1311"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the cellular reduction of 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT): Subcellular localization, substrate dependence, and involvement of mitochondrial electron transport in MTT reduction</span></div><div class="casAuthors">Berridge, Michael V.; Tan, An S.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">474-82</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">The MTT assay, which is widely used to measure cell proliferation and to screen for anticancer drugs, is based on redn. of the tetrazolium salt MTT by actively growing cells to produce a blue formazan product.  Despite broad acceptance of this assay, neither the subcellular localization, nor the biochem. events involved in MTT redn. are known.  Mitochondrial involvement in MTT redn. has been inferred from studies with respiratory inhibitors using succinate as a substrate, but the contribution of this activity to overall cellular MTT redn. is unknown.  The bone marrow-derived cell line 32D was used to investigate the subcellular localization of MTT redn. by using succinate, NADH, and NADPH as substrates.  At optimum substrate concns., MTT redn. by whole cell homogenates was greatest with NADH and least with succinate, which accounted for <10% of the combined activities.  Using succinate, 95% of recoverable MTT reducing activity was in particulate fractions of the cell and 77% in the mitochondrial and light mitochondrial/lysosomal fractions.  When NADH and NADPH were used as substrates, increased amts. of MTT reducing activity were assocd. with sol. fractions of the cell and assocn. with mitochondrial fractions was less pronounced.  To further characterize MTT redn. by the mitochondrial fraction, respiratory chain inhibitors were used to explore involvement of electron transport in MTT redn.  Succinate-dependent mitochondrial MTT redn. was inhibited by 80% with chlorpromazine, 70% by antimycin A, and 85-90% by thenoyltrifluoroacetone (TTFA), but inhibition was not obsd. with rotenone at â¤2 Î¼M, Amytal, or azide.  These results suggest that when succinate is used as an electron donor, 70-80% of mitochondrial MTT redn. occurs subsequent to transfer of electrons from cytochrome c to cytochrome oxidase, but prior to the point of azide inhibition.  In contrast to succinate, NADPH-dependent mitochondrial MTT redn. was not affected by any of the respiratory inhibitors tested, and NADH-dependent redn. was only inhibited by chlorpromazine (40-50% at plateau concn.).  These results suggest that most cellular MTT redn. occurs outside the mitochondrial inner membrane and involves NADH and NADPH-dependent mechanisms that are insensitive to respiratory chain inhibitors.  This interpretation is supported by whole cell studies in which rotenone failed to affect basal and interleukin-3-stimulated MTT redn. at times up to 4 h but strongly inhibited DNA synthesis.  Apparently most cellular redn. of MTT occurs extramitochondrially and probably involves the pyridine nucleotide cofactors NADH and NADPH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmehIJ3dI6S7Vg90H21EOLACvtfcHk0li1he9Ab4IprA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvFSmsL0%253D&md5=2f559aaae05ea333d5a407b12c32a0d7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1006%2Fabbi.1993.1311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.1993.1311%26sid%3Dliteratum%253Aachs%26aulast%3DBerridge%26aufirst%3DM.%2BV.%26aulast%3DTan%26aufirst%3DA.%2BS.%26atitle%3DCharacterization%2520of%2520the%2520cellular%2520reduction%2520of%25203-%25284%252C5-dimethylthiazol-2-yl%2529-2%252C5-diphenyltetrazolium%2520bromide%2520%2528MTT%2529%253A%2520subcellular%2520localization%252C%2520substrate%2520dependence%252C%2520and%2520involvement%2520of%2520mitochondrial%2520electron%2520transport%2520in%2520MTT%2520reduction%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1993%26volume%3D303%26spage%3D474%26epage%3D482%26doi%3D10.1006%2Fabbi.1993.1311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kupcsik, L.</span></span> <span> </span><span class="NLM_article-title">Estimation of cell number based on metabolic activity: the MTT reduction assay</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>740</i></span>,  <span class="NLM_fpage">13</span>â <span class="NLM_lpage">19</span>, <span class="refDoi">Â DOI: 10.1007/978-1-61779-108-6_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1007%2F978-1-61779-108-6_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=21468963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntlakt7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=740&publication_year=2011&pages=13-19&author=L.+Kupcsik&title=Estimation+of+cell+number+based+on+metabolic+activity%3A+the+MTT+reduction+assay&doi=10.1007%2F978-1-61779-108-6_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of cell number based on metabolic activity: the MTT reduction assay</span></div><div class="casAuthors">Kupcsik, Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">740</span>
        (<span class="NLM_cas:issue">Mammalian Cell Viability</span>),
    <span class="NLM_cas:pages">13-19</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The MTT redn. assay is used to det. the level of metabolic activity in eukaryotic cells, including animal, plant, and fungal cells.  If the metabolic rate is const., the technique can be employed to count living cells in a sample.  Once it is set up, the method is very robust, and can be automatized to be applied on a large no. of samples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKpysUPR2UR7Vg90H21EOLACvtfcHk0li1he9Ab4IprA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntlakt7s%253D&md5=0c58d15edacb68f7f4e1f014f079d8ac</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-108-6_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-108-6_3%26sid%3Dliteratum%253Aachs%26aulast%3DKupcsik%26aufirst%3DL.%26atitle%3DEstimation%2520of%2520cell%2520number%2520based%2520on%2520metabolic%2520activity%253A%2520the%2520MTT%2520reduction%2520assay%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D740%26spage%3D13%26epage%3D19%26doi%3D10.1007%2F978-1-61779-108-6_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mannhold, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span> <span> </span><span class="NLM_article-title">Calculation of molecular lipophilicity: state-of-the-art and comparison of Log P methods on more than 96,000 compounds</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">861</span>â <span class="NLM_lpage">893</span>, <span class="refDoi">Â DOI: 10.1002/jps.21494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fjps.21494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=18683876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1eisLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2009&pages=861-893&author=R.+Mannholdauthor=G.+I.+Podaauthor=C.+Ostermannauthor=I.+V.+Tetko&title=Calculation+of+molecular+lipophilicity%3A+state-of-the-art+and+comparison+of+Log+P+methods+on+more+than+96%2C000+compounds&doi=10.1002%2Fjps.21494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of molecular lipophilicity: state-of-the-art and comparison of logP methods on more than 96,000 compounds</span></div><div class="casAuthors">Mannhold, Raimund; Poda, Gennadiy I.; Ostermann, Cluade; Tetko, Igor V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">861-893</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  We first review the state-of-the-art in development of logP prediction approaches falling in two major categories: substructure-based and property-based methods.  Then, we compare the predictive power of representative methods for one public (N = 266) and two in house datasets from Nycomed (N = 882) and Pfizer (N = 95809).  A total of 30 and 18 methods were tested for public and industrial datasets, resp.  Accuracy of models declined with the no. of nonhydrogen atoms.  The Arithmetic Av. Model (AAM), which predicts the same value (the arithmetic mean) for all compds., was used as a baseline model for comparison.  Methods with Root Mean Squared Error (RMSE) greater than RMSE produced by the AAM were considered as unacceptable.  The majority of analyzed methods produced reasonable results for the public dataset but only seven methods were successful on the both in house datasets.  We proposed a simple equation based on the no. of carbon atoms, NC, and the no. of hetero atoms, NHET: logP = 1.46(+0.02) + 0.11(Â±0.001) NC - 0.11(Â±0.001) NHEET.  This equation outperformed a large no. of programs benchmarked in this study.  Factors influencing the accuracy of logP predictions were elucidated and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR3RAtmKwEZ7Vg90H21EOLACvtfcHk0liFgFesjaZ1nQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1eisLo%253D&md5=8ae19379316c226dc75ec057006b0e8c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fjps.21494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21494%26sid%3Dliteratum%253Aachs%26aulast%3DMannhold%26aufirst%3DR.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DOstermann%26aufirst%3DC.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26atitle%3DCalculation%2520of%2520molecular%2520lipophilicity%253A%2520state-of-the-art%2520and%2520comparison%2520of%2520Log%2520P%2520methods%2520on%2520more%2520than%252096%252C000%2520compounds%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2009%26volume%3D98%26spage%3D861%26epage%3D893%26doi%3D10.1002%2Fjps.21494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42717</span>, <span class="refDoi">Â DOI: 10.1038/srep42717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fsrep42717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=28256516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42717&author=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissADME%3A+a+free+web+tool+to+evaluate+pharmacokinetics%2C+drug-likeness+and+medicinal+chemistry+friendliness+of+small+molecules&doi=10.1038%2Fsrep42717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span></div><div class="casAuthors">Daina Antoine; Michielin Olivier; Zoete Vincent; Michielin Olivier; Michielin Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42717</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">To be effective as a drug, a potent molecule must reach its target in the body in sufficient concentration, and stay there in a bioactive form long enough for the expected biologic events to occur.  Drug development involves assessment of absorption, distribution, metabolism and excretion (ADME) increasingly earlier in the discovery process, at a stage when considered compounds are numerous but access to the physical samples is limited.  In that context, computer models constitute valid alternatives to experiments.  Here, we present the new SwissADME web tool that gives free access to a pool of fast yet robust predictive models for physicochemical properties, pharmacokinetics, drug-likeness and medicinal chemistry friendliness, among which in-house proficient methods such as the BOILED-Egg, iLOGP and Bioavailability Radar.  Easy efficient input and interpretation are ensured thanks to a user-friendly interface through the login-free website http://www.swissadme.ch.  Specialists, but also nonexpert in cheminformatics or computational chemistry can predict rapidly key parameters for a collection of molecules to support their drug discovery endeavours.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRl_Uk5dOMKs6qcneEtCi92fW6udTcc2eZhon3d4WiK9bntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D&md5=9715b8cb8a34b17c4c73ff69a5a8cc50</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fsrep42717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep42717%26sid%3Dliteratum%253Aachs%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissADME%253A%2520a%2520free%2520web%2520tool%2520to%2520evaluate%2520pharmacokinetics%252C%2520drug-likeness%2520and%2520medicinal%2520chemistry%2520friendliness%2520of%2520small%2520molecules%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D42717%26doi%3D10.1038%2Fsrep42717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lerner-Marmarosh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senior, A. E.</span></span> <span> </span><span class="NLM_article-title">Large scale purification of detergent-soluble P-glycoprotein from Pichia pastoris cells and characterization of nucleotide binding properties of wild-type, Walker A, and Walker B mutant proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">34711</span>â <span class="NLM_lpage">34718</span>, <span class="refDoi">Â DOI: 10.1074/jbc.274.49.34711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1074%2Fjbc.274.49.34711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10574938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK1MXnvFyrs7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=34711-34718&author=N.+Lerner-Marmaroshauthor=K.+Gimiauthor=I.+L.+Urbatschauthor=P.+Grosauthor=A.+E.+Senior&title=Large+scale+purification+of+detergent-soluble+P-glycoprotein+from+Pichia+pastoris+cells+and+characterization+of+nucleotide+binding+properties+of+wild-type%2C+Walker+A%2C+and+Walker+B+mutant+proteins&doi=10.1074%2Fjbc.274.49.34711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Large scale purification of detergent-soluble P-glycoprotein from Pichia pastoris cells and characterization of nucleotide binding properties of wild-type, Walker A, and Walker B mutant proteins</span></div><div class="casAuthors">Lerner-Marmarosh, Nicole; Gimi, Khursheed; Urbatsch, Ina L.; Gros, Philippe; Senior, Alan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">34711-34718</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp; mouse MDR3) was expressed in Pichia pastoris, grown in fermentor culture, and purified.  The final pure product is of high specific ATPase activity and is sol. at low detergent concn.  120 G of cells yielded 6 mg of pure Pgp; >4 kg of cells were obtained from a single fermentor run.  Properties of the pure protein were similar to those of previous prepns., except there was significant ATPase activity in absence of added lipid.  Mutant mouse MDR3 P-glycoproteins were purified by the same procedure after growth of cells in flask culture, with similar yields and purity.  This procedure should open up new avenues of structural, biophys., and biochem. studies of Pgp.  Equil. nucleotide-binding parameters of wild-type mouse MDR3 Pgp were studied using 2'-(3')-O-(2,4,6-trinitrophenyl)adenosine tri- and diphosphate.  Both analogs were found to bind with Kd in the low micromolar range, to a single class of site, with no evidence of cooperativity.  ATP displacement of the analogs was seen.  Similar binding was seen with K429R/K1072R and D551N/D1196N mutant mouse MDR3 Pgp, showing that these Walker A and B mutations had no significant effect on affinity or stoichiometry of nucleotide binding.  These residues, known to be crit. for catalysis, are concluded to be involved primarily in stabilization of the catalytic transition state in Pgp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWFsSz4RG5frVg90H21EOLACvtfcHk0lgraJFgvkkJ4Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvFyrs7k%253D&md5=2d3b3f025fc8a18438f26f9b3f705e68</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.49.34711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.49.34711%26sid%3Dliteratum%253Aachs%26aulast%3DLerner-Marmarosh%26aufirst%3DN.%26aulast%3DGimi%26aufirst%3DK.%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DSenior%26aufirst%3DA.%2BE.%26atitle%3DLarge%2520scale%2520purification%2520of%2520detergent-soluble%2520P-glycoprotein%2520from%2520Pichia%2520pastoris%2520cells%2520and%2520characterization%2520of%2520nucleotide%2520binding%2520properties%2520of%2520wild-type%252C%2520Walker%2520A%252C%2520and%2520Walker%2520B%2520mutant%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D34711%26epage%3D34718%26doi%3D10.1074%2Fjbc.274.49.34711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tombline, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muharemagic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senior, A. E.</span></span> <span> </span><span class="NLM_article-title">Expression, purification, and characterization of cysteine-free mouse P-glycoprotein</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>445</i></span>,  <span class="NLM_fpage">124</span>â <span class="NLM_lpage">128</span>, <span class="refDoi">Â DOI: 10.1016/j.abb.2005.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.abb.2005.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=16343415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlGisLfM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=445&publication_year=2006&pages=124-128&author=G.+Tomblineauthor=I.+L.+Urbatschauthor=N.+Virkauthor=A.+Muharemagicauthor=L.+B.+Whiteauthor=A.+E.+Senior&title=Expression%2C+purification%2C+and+characterization+of+cysteine-free+mouse+P-glycoprotein&doi=10.1016%2Fj.abb.2005.11.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification, and characterization of cysteine-free mouse P-glycoprotein</span></div><div class="casAuthors">Tombline, Gregory; Urbatsch, Ina L.; Virk, Navneet; Muharemagic, Alma; White, Lori Bartholomew; Senior, Alan E.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">445</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-128</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cysteine-free mouse MDR3 P-glycoprotein (Pgp) was constructed by mutagenesis of the 9 naturally occurring Cys residues to Ala.  The Cys-free protein was expressed in Pichia pastoris and purified.  The yield, purity, ATPase activity, Km(MgATP), and stimulation of ATPase by verapamil, were similar to that wild-type mouse Ppg.  Mouse Cys-free Pgp was superior in yield and stability to Cys-free human MDR1 Pgp.  Mutants Y1040A and Y1040C were constructed in mouse Cys-free Pgp background.  Both showed extremely low ATPase activity, strongly-impaired vanadate-trapping of ADP, and reduced photolabeling by 8-azido-ATP.  The results were consistent with the conclusion that Tyr-1040 was located in the MgATP-binding site in NBD2 and was required for correct binding and/or orientation of bound MgATP substrate in Pgp as previously suggested by x-ray structures of other ABC transporters and by sequencing of photolabeled Pgp.  The results also support the authors' previous conclusion that both catalytic sites must be intact for normal function in Pgp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTS_AdDoiAk7Vg90H21EOLACvtfcHk0lgraJFgvkkJ4Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlGisLfM&md5=42055a06913d3137948e6635e65dfb23</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2005.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2005.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DTombline%26aufirst%3DG.%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DVirk%26aufirst%3DN.%26aulast%3DMuharemagic%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DL.%2BB.%26aulast%3DSenior%26aufirst%3DA.%2BE.%26atitle%3DExpression%252C%2520purification%252C%2520and%2520characterization%2520of%2520cysteine-free%2520mouse%2520P-glycoprotein%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2006%26volume%3D445%26spage%3D124%26epage%3D128%26doi%3D10.1016%2Fj.abb.2005.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berridge, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callaghan, R.</span></span> <span> </span><span class="NLM_article-title">The molecular interaction of the high affinity reversal agent XR9576 with P-glycoprotein</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">403</span>â <span class="NLM_lpage">411</span>, <span class="refDoi">Â DOI: 10.1038/sj.bjp.0702807</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fsj.bjp.0702807" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10510451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK1MXmsFWgs70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=1999&pages=403-411&author=C.+Martinauthor=G.+Berridgeauthor=P.+Mistryauthor=C.+Higginsauthor=P.+Charltonauthor=R.+Callaghan&title=The+molecular+interaction+of+the+high+affinity+reversal+agent+XR9576+with+P-glycoprotein&doi=10.1038%2Fsj.bjp.0702807"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular interaction of the high affinity reversal agent XR9576 with P-glycoprotein</span></div><div class="casAuthors">Martin, Catherine; Berridge, Georgina; Mistry, Prakash; Higgins, Christopher; Charlton, Peter; Callaghan, Richard</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">403-411</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">1 The kinetics and nature of equil. binding were used to characterize the mol. interaction of the anthranilic acid deriv. [3H]-XR9576 with the multidrug resistance P-glycoprotein (P-gp).  XR9576 displayed specific high-affinity binding to P-gp (Bmax = 275 pmol mg-1, Kd=5.1 nM).  The transport substrates [3H]-vinblastine and [3H]-paclitaxel displayed 4 fold and 20 fold lower affinity resp. for P-gp.  The duration of action of XR9576 with P-gp was increased in comparison to that of vinblastine which displayed a slower rate of assocn. and a faster dissocn. rate.  2 The relative affinities of several modulators and transport substrates to interact with P-gp were detd. from displacement drug equil. binding assays.  Vinblastine and paclitaxel could only fractionally displace [3H]-XR9576 binding, displaying Ki values significantly different from their measured Kd values.  This suggests a non-competitive interaction between XR9576 and the P-gp substrates vinblastine and paclitaxel.  3 XR9576 was shown to be a potent modulator of P-gp mediated [3H]-vinblastine and [3H]-paclitaxel transport as it increased the steady-state accumulation of these cytotoxics in CHrB30 cells to levels obsd. in non-P-gp-expressing AuxB1 cells (EC50 = 487 Â± 50 nM).  This inhibition of drug transport is not mediated through competition for transport since [3H]-XR9576 accumulation was not influenced by P-gp expression or function.  4 These results demonstrate that the P-gp modulator XR9576 exhibits greater selectivity, duration of inhibition and potency of interaction with this transporter than any other reported modulators.  Several lines of evidence suggest that XR9576 inhibits P-gp function by binding at a site which is distinct from the site of interaction of transport substrates.  The two sites may be classified as serving modulatory or transport functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6HZ8wWLNGErVg90H21EOLACvtfcHk0lgEHbMPJkXuFA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmsFWgs70%253D&md5=5d80afb78a34d30b4947e972d486a584</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0702807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0702807%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DBerridge%26aufirst%3DG.%26aulast%3DMistry%26aufirst%3DP.%26aulast%3DHiggins%26aufirst%3DC.%26aulast%3DCharlton%26aufirst%3DP.%26aulast%3DCallaghan%26aufirst%3DR.%26atitle%3DThe%2520molecular%2520interaction%2520of%2520the%2520high%2520affinity%2520reversal%2520agent%2520XR9576%2520with%2520P-glycoprotein%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1999%26volume%3D128%26spage%3D403%26epage%3D411%26doi%3D10.1038%2Fsj.bjp.0702807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Streckenbach, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repke, K. R. H.</span></span> <span> </span><span class="NLM_article-title">Analysis of phosphoryl transfer mechanism and catalytic centre geometries of transport ATPase by means of spin-labelled ATP</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>601</i></span>,  <span class="NLM_fpage">34</span>â <span class="NLM_lpage">46</span>, <span class="refDoi">Â DOI: 10.1016/0005-2736(80)90511-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2F0005-2736%2880%2990511-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=6250610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaL3cXmtVSjtbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=601&publication_year=1980&pages=34-46&author=B.+Streckenbachauthor=D.+Schwarzauthor=K.+R.+H.+Repke&title=Analysis+of+phosphoryl+transfer+mechanism+and+catalytic+centre+geometries+of+transport+ATPase+by+means+of+spin-labelled+ATP&doi=10.1016%2F0005-2736%2880%2990511-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of phosphoryl transfer mechanism and catalytic center geometries of transport ATPase by means of spin-labelled ATP</span></div><div class="casAuthors">Streckenbach, Barbara; Schwarz, Dieter; Repke, Kurt R. H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">601</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-46</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    </div><div class="casAbstract">In the presence of Na+, K+ and Mg2+, SL-ATP [3'-O-(1-oxyl-2,2,5,5-tetramethyl-3-carbonylpyrrolidine)-ATP] is not hydrolyzed by transport ATPase from 3 different sources.  In the presence of Na+ and Mg2+, SL-ATP reacts initially like ATP with the enzyme, as indicated by the prodn. of a similar ouabain-binding protein conformation.  With both nucleotides, this initial reaction includes the formation of the covalent enzyme-nucleotide complex through nucleophilic attack of the aspartate carboxyanion of the catalytic center on the terminal P atom of the triphosphate chain.  This produces the ouabain-binding conformation of the enzyme.  Unlike ATP, the covalent enzyme-SL-ATP complex resists further transformation.  In the presence of Na+, K+, and Mg2+, the influence of SL-ATP on ATP hydrolysis by transport ATPase depends on the ATP concn. chosen.  At low ATP concn., when the enzyme works as a Na+-ATPase, SL-ATP does not affect the rate of ATP cleavage.  At high ATP concn., however, when the enzyme works as a (Na+ + K+)-ATPase, SL-ATP reduces the rate of ATP hydrolysis to the level of Na+-ATPase activity, apparently due to the formation of the covalent enzyme-SL-ATP complex.  SL-ATP in the covalent enzyme-SL-ATP complex shows an ESR spectrum which is indistinguishable regarding the overall shape, the rotational correlation time, Ï, and the hyperfine coupling const., aN, from the ESR spectrum of free SL-ATP.  Consequently, the dimensions of the catalytic center cleft of transport ATPase provide the labeled group of SL-ATP, opposite to its 3'-O-esterification site at the ribose moiety, in a wide-cleft groove, enough free space for an essentially unhindered rotational mobility within an aq. environment, like that of the bulk medium.  Judged from literature data, similarly wide grooves exist in the catalytic center clefts of mitochondrial and myosin ATPases.  Thus, the structural unit forming the binding site for the AMP moiety of ATP in ATPases is similar to the structural unit forming the binding site for the AMP moiety of NAD and ADP in several dehydrogenases and kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-pIx38yRv8rVg90H21EOLACvtfcHk0lgEHbMPJkXuFA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXmtVSjtbk%253D&md5=e6ef56b5f34855a0ebe74f3bb5daaedf</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0005-2736%2880%2990511-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2736%252880%252990511-8%26sid%3Dliteratum%253Aachs%26aulast%3DStreckenbach%26aufirst%3DB.%26aulast%3DSchwarz%26aufirst%3DD.%26aulast%3DRepke%26aufirst%3DK.%2BR.%2BH.%26atitle%3DAnalysis%2520of%2520phosphoryl%2520transfer%2520mechanism%2520and%2520catalytic%2520centre%2520geometries%2520of%2520transport%2520ATPase%2520by%2520means%2520of%2520spin-labelled%2520ATP%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1980%26volume%3D601%26spage%3D34%26epage%3D46%26doi%3D10.1016%2F0005-2736%2880%2990511-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delannoy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tombline, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senior, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span> <span> </span><span class="NLM_article-title">Nucleotide binding to the multidrug resistance P-glycoprotein as studied by ESR spectroscopy</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">14010</span>â <span class="NLM_lpage">14019</span>, <span class="refDoi">Â DOI: 10.1021/bi0512445</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0512445" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVGgtbbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=14010-14019&author=S.+Delannoyauthor=I.+L.+Urbatschauthor=G.+Tomblineauthor=A.+E.+Seniorauthor=P.+D.+Vogel&title=Nucleotide+binding+to+the+multidrug+resistance+P-glycoprotein+as+studied+by+ESR+spectroscopy&doi=10.1021%2Fbi0512445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleotide Binding to the Multidrug Resistance P-Glycoprotein as Studied by ESR Spectroscopy</span></div><div class="casAuthors">Delannoy, Sabine; Urbatsch, Ina L.; Tombline, Gregory; Senior, Alan E.; Vogel, Pia D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">14010-14019</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ESR spectroscopy using spin-labeled ATP was used to study nucleotide binding to and structural transitions within the multidrug resistance (MDR) P-glycoprotein, P-gp.  Spin-labeled ATP (SL-ATP) with the spin label attached to the ribose, was obsd. to be an excellent substrate analog for P-gp.  SL-ATP was hydrolyzed in a drug-stimulated fashion at about 14% of the rate for normal ATP and allowed reversible trapping of the enzyme in transition and ground states.  Equil. binding of a total of two nucleotides per P-gp was obsd. with a binding affinity of 366 Î¼M in the presence of Mg2+ but in the absence of transport substrates such as verapamil.  Binding of SL-ATP to wild-type P-gp in the presence of verapamil resulted in redn. of the protein-bound spin-label moiety, most likely due to a conformational transition within P-gp that positioned cysteines in close proximity to the spin label to allow chem. redn. of the radical.  We circumvented this problem by using a mutant of P-gp in which all naturally occurring cysteines were substituted for alanines.  Equil. binding of SL-ATP to this mutant P-gp resulted in max. binding of two nucleotides; the binding affinity was 223 Î¼M in the absence and 180 Î¼M in the presence of verapamil.  The corresponding ESR spectra of wild-type and Cys-less P-gp in the presence of SL-ATP indicate that a cysteine side chain of P-gp is located close to the ribose of the bound nucleotide.  Trapping SL-ATP as an AlFx-adduct resulted in ESR spectra that showed strong immobilization of the radical, supporting the formation of a closed conformation of P-gp in its transition state.  This study is the first to employ ESR spectroscopy with the use of spin-labeled nucleotide analogs to study P-glycoprotein.  The study shows that SL-ATP is an excellent substrate analog that will allow further exploration of structure and dynamics within the nucleotide binding domains of this important enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-N7KS3FsbLVg90H21EOLACvtfcHk0ljQxPQoArJxsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVGgtbbP&md5=41f0dd9fdffa43775707de5fbeb71d7e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fbi0512445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0512445%26sid%3Dliteratum%253Aachs%26aulast%3DDelannoy%26aufirst%3DS.%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DTombline%26aufirst%3DG.%26aulast%3DSenior%26aufirst%3DA.%2BE.%26aulast%3DVogel%26aufirst%3DP.%2BD.%26atitle%3DNucleotide%2520binding%2520to%2520the%2520multidrug%2520resistance%2520P-glycoprotein%2520as%2520studied%2520by%2520ESR%2520spectroscopy%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D14010%26epage%3D14019%26doi%3D10.1021%2Fbi0512445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nett, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmadel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trommer, W. E.</span></span> <span> </span><span class="NLM_article-title">Nucleotide binding sites on mitochondrial F1-ATPase. Electron spin resonance spectroscopy and photolabeling by azido-spin-labeled adenine nucleotides support an adenylate kinase-like orientation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">11982</span>â <span class="NLM_lpage">11986</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1318307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK38Xls1KmtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1992&pages=11982-11986&author=P.+D.+Vogelauthor=J.+H.+Nettauthor=H.+E.+Sauerauthor=K.+Schmadelauthor=R.+L.+Crossauthor=W.+E.+Trommer&title=Nucleotide+binding+sites+on+mitochondrial+F1-ATPase.+Electron+spin+resonance+spectroscopy+and+photolabeling+by+azido-spin-labeled+adenine+nucleotides+support+an+adenylate+kinase-like+orientation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleotide binding sites on mitochondrial F1-ATPase.  Electron spin resonance spectroscopy and photolabeling by azido-spin-labeled adenine nucleotides support an adenylate kinase-like orientation</span></div><div class="casAuthors">Vogel, Pia D.; Nett, Juergen H.; Sauer, Hubertus E.; Schmadel, Klaus; Cross, Richard L.; Trommer, Wolfgang E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11982-6</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">A spin-labeled photoaffinity ATP analog, 2-azido-2',3'-SL-ATP (2-Na3-SL-ATP; SL = 2,2,5,5-tetramethyl-3-pyrroline-1-oxyl-3-carboxylic acid ester) was specifically loaded at catalytic (exchangeable) or noncatalytic (nonexchangeable) nucleotide-binding sites on nucleotide-depleted mitochondrial F1-ATPase.  Photolysis of the enzyme complexes resulted in the specific modification of Î²-subunit Tyr-345 when the catalytic sites were occupied and Î²-subunit Tyr-368 when noncatalytic sites were filled.  These were the same amino acid assignments that were made previously using 2-azido-ATP.  The results demonstrated that the attachment of a spin label moiety to the ribose ring did not prevent proper binding of the analog at both types of nucleotide sites on F1-ATPase and suggested that the probe could be used for investigations of the nucleotide-binding sites by ESR spectroscopy.  The enzyme that was in complex with 2-N3-SL-ATP exhibited an ESR spectrum that was typical for highly immobilized nitroxyl radicals both in the dark or after photolysis.  Addnl. peaks in the high- and low-field regions arose due to dipolar spin interactions most likely involving a pair of catalytic and noncatalytic sites.  The 2 sites were calcd. to be â¼15 Ã apart.  This distance, obtained through ESR spectroscopy, combined with the finding that the 2 labeled amino acids were only 23 residues apart from each other, further supported an adenylate kinase-like arrangement of nucleotide binding sites on F1-ATPase where catalytic and noncatalytic sites are in close proximity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5m8Y7VQ_1pLVg90H21EOLACvtfcHk0ljQxPQoArJxsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xls1KmtLY%253D&md5=5d859eae7b9ab881dbd1c4b7980ae696</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DP.%2BD.%26aulast%3DNett%26aufirst%3DJ.%2BH.%26aulast%3DSauer%26aufirst%3DH.%2BE.%26aulast%3DSchmadel%26aufirst%3DK.%26aulast%3DCross%26aufirst%3DR.%2BL.%26aulast%3DTrommer%26aufirst%3DW.%2BE.%26atitle%3DNucleotide%2520binding%2520sites%2520on%2520mitochondrial%2520F1-ATPase.%2520Electron%2520spin%2520resonance%2520spectroscopy%2520and%2520photolabeling%2520by%2520azido-spin-labeled%2520adenine%2520nucleotides%2520support%2520an%2520adenylate%2520kinase-like%2520orientation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1992%26volume%3D267%26spage%3D11982%26epage%3D11986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>â <span class="NLM_lpage">461</span>, <span class="refDoi">Â DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lgUovwpzlsd7Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Automated docking of flexible ligands: applications of AutoDock</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">5</span>, <span class="refDoi">Â DOI: 10.1002/(SICI)1099-1352(199601)9:1<1::AID-JMR241>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2F%28SICI%291099-1352%28199601%299%3A1%3C1%3A%3AAID-JMR241%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=8723313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK28Xis1SqtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=1-5&author=D.+S.+Goodsellauthor=G.+M.+Morrisauthor=A.+J.+Olson&title=Automated+docking+of+flexible+ligands%3A+applications+of+AutoDock&doi=10.1002%2F%28SICI%291099-1352%28199601%299%3A1%3C1%3A%3AAID-JMR241%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking of flexible ligands: application of AutoDock</span></div><div class="casAuthors">Goodsell, David S.; Morris, Garrett M.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-5</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 23 refs. AutoDock is a suite of C programs used to predict the bound conformations of a small, flexible ligand to a macromol. target of known structure.  The technique combines simulated annealing for conformation searching with a rapid grid-based method of energy evaluation.  This paper reviews recent applications of the technique and describes the enhancement included in the current release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHBbJMLK-ctbVg90H21EOLACvtfcHk0lgUovwpzlsd7Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis1SqtrY%253D&md5=8f8f3964ba0c51da8f8b351e65a40a59</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291099-1352%28199601%299%3A1%3C1%3A%3AAID-JMR241%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291099-1352%2528199601%25299%253A1%253C1%253A%253AAID-JMR241%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutomated%2520docking%2520of%2520flexible%2520ligands%253A%2520applications%2520of%2520AutoDock%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D1996%26volume%3D9%26spage%3D1%26epage%3D5%26doi%3D10.1002%2F%28SICI%291099-1352%28199601%299%3A1%3C1%3A%3AAID-JMR241%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Distributed automated docking of flexible ligands to proteins: parallel applications of AutoDock 2.4</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">293</span>â <span class="NLM_lpage">304</span>, <span class="refDoi">Â DOI: 10.1007/BF00124499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1007%2FBF00124499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=8877701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK28XlsVOhsL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1996&pages=293-304&author=G.+M.+Morrisauthor=D.+S.+Goodsellauthor=R.+Hueyauthor=A.+J.+Olson&title=Distributed+automated+docking+of+flexible+ligands+to+proteins%3A+parallel+applications+of+AutoDock+2.4&doi=10.1007%2FBF00124499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Distributed automated docking of flexible ligands to proteins: parallel applications of AutoDock 2.4</span></div><div class="casAuthors">Morris, Garrett M.; Goodsell, David S.; Huey, Ruth; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-304</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">AutoDock 2.4 predicts the bound conformations of a small, flexible ligand to a nonflexible macromol. target of known structure.  The technique combines simulated annealing for conformation searching with a rapid grid-based method of energy evaluation based on the AMBER force field.  AutoDock has been optimized in performance without sacrificing accuracy; it incorporates many enhancements and addns., including an intuitive interface.  We have developed a set of tools for launching and analyzing many independent docking jobs in parallel on a heterogeneous network of UNIX-based workstations.  This paper describes the current release, and the results of a suite of diverse test systems.  We also present the results of a systematic investigation in to the effects of varying simulated-annealing parameters on mol. docking.  We show that even for ligands with a large no. of degrees of freedom, root-mean-square deviations of less than 1 Ã from the crystallog. conformation are obtained for the lowest-energy dockings, although fewer dockings find the crystallog. conformation when there are more degrees of freedom.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHQmD1We2g6LVg90H21EOLACvtfcHk0lgUovwpzlsd7Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XlsVOhsL0%253D&md5=0e88e41a8db0b67cef118620d0992452</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2FBF00124499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00124499%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DDistributed%2520automated%2520docking%2520of%2520flexible%2520ligands%2520to%2520proteins%253A%2520parallel%2520applications%2520of%2520AutoDock%25202.4%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D1996%26volume%3D10%26spage%3D293%26epage%3D304%26doi%3D10.1007%2FBF00124499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ãsterberg, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span> <span> </span><span class="NLM_article-title">Automated docking to multiple target structures: incorporation of protein mobility and structural water heterogeneity in AutoDock</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">34</span>â <span class="NLM_lpage">40</span>, <span class="refDoi">Â DOI: 10.1002/prot.10028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fprot.10028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=11746701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BD3MnpvVOlsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=34-40&author=F.+%C3%96sterbergauthor=G.+M.+Morrisauthor=M.+F.+Sannerauthor=A.+J.+Olsonauthor=D.+S.+Goodsell&title=Automated+docking+to+multiple+target+structures%3A+incorporation+of+protein+mobility+and+structural+water+heterogeneity+in+AutoDock&doi=10.1002%2Fprot.10028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking to multiple target structures: incorporation of protein mobility and structural water heterogeneity in AutoDock</span></div><div class="casAuthors">Osterberg Fredrik; Morris Garrett M; Sanner Michel F; Olson Arthur J; Goodsell David S</div><div class="citationInfo"><span class="NLM_cas:title">Proteins</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-40</span>
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">Protein motion and heterogeneity of structural waters are approximated in ligand-docking simulations, using an ensemble of protein structures.  Four methods of combining multiple target structures within a single grid-based lookup table of interaction energies are tested.  The method is evaluated using complexes of 21 peptidomimetic inhibitors with human immunodeficiency virus type 1 (HIV-1) protease.  Several of these structures show motion of an arginine residue, which is essential for binding of large inhibitors.  A structural water is also present in 20 of the structures, but it must be absent in the remaining one for proper binding.  Mean and minimum methods perform poorly, but two weighted average methods permit consistent and accurate ligand docking, using a single grid representation of the target protein structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVWVkt39Ar86Nc3HjJRoFAfW6udTcc2ebQGgFcfRC1i7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3MnpvVOlsg%253D%253D&md5=4ae2dd1cf13b1e0e0af7f41ddbfdc8ed</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fprot.10028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10028%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596sterberg%26aufirst%3DF.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26atitle%3DAutomated%2520docking%2520to%2520multiple%2520target%2520structures%253A%2520incorporation%2520of%2520protein%2520mobility%2520and%2520structural%2520water%2520heterogeneity%2520in%2520AutoDock%26jtitle%3DProteins%26date%3D2002%26volume%3D46%26spage%3D34%26epage%3D40%26doi%3D10.1002%2Fprot.10028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, R. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locher, K. P.</span></span> <span> </span><span class="NLM_article-title">Structure of a bacterial multidrug ABC transporter</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>443</i></span>,  <span class="NLM_fpage">180</span>â <span class="NLM_lpage">185</span>, <span class="refDoi">Â DOI: 10.1038/nature05155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1038%2Fnature05155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=16943773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVansLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=443&publication_year=2006&pages=180-185&author=R.+J.+P.+Dawsonauthor=K.+P.+Locher&title=Structure+of+a+bacterial+multidrug+ABC+transporter&doi=10.1038%2Fnature05155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a bacterial multidrug ABC transporter</span></div><div class="casAuthors">Dawson, Roger J. P.; Locher, Kaspar P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">443</span>
        (<span class="NLM_cas:issue">7108</span>),
    <span class="NLM_cas:pages">180-185</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Multidrug transporters of the ABC family facilitate the export of diverse cytotoxic drugs across cell membranes.  This is clin. relevant, as tumor cells may become resistant to agents used in chemotherapy.  To understand the mol. basis of this process, we have detd. the 3.0 Ã crystal structure of a bacterial ABC transporter (Sav1866) from Staphylococcus aureus.  The homodimeric protein consists of 12 transmembrane helixes in an arrangement that is consistent with crosslinking studies and electron microscopic imaging of the human multidrug resistance protein MDR1, but critically different from that reported for the bacterial lipid flippase MsbA.  The obsd., outward-facing conformation reflects the ATP-bound state, with the two nucleotide-binding domains in close contact and the two transmembrane domains forming a central cavity-presumably the drug translocation pathway-that is shielded from the inner leaflet of the lipid bilayer and from the cytoplasm, but exposed to the outer leaflet and the extracellular space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4WSMSCWPqt7Vg90H21EOLACvtfcHk0lgh4sEPUb2JRQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVansLY%253D&md5=a506c0275d59882632fc7fb5714dc3fc</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnature05155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05155%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DR.%2BJ.%2BP.%26aulast%3DLocher%26aufirst%3DK.%2BP.%26atitle%3DStructure%2520of%2520a%2520bacterial%2520multidrug%2520ABC%2520transporter%26jtitle%3DNature%26date%3D2006%26volume%3D443%26spage%3D180%26epage%3D185%26doi%3D10.1038%2Fnature05155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">VMD: visual molecular dynamics</span>. <i>J. Mol. Graph.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">33</span>â <span class="NLM_lpage">38</span>, <span class="refDoi">Â DOI: 10.1016/0263-7855(96)00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2F0263-7855%2896%2900018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=8744570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=33-38&author=W.+Humphreyauthor=A.+Dalkeauthor=K.+Schulten&title=VMD%3A+visual+molecular+dynamics&doi=10.1016%2F0263-7855%2896%2900018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">VDM: visual molecular dynamics</span></div><div class="casAuthors">Humphrey, William; Dalke, Andrew; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8, plates, 27-28</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">VMD is a mol. graphics program designed for the display and anal. of mol. assemblies, in particular, biopolymers such as proteins and nucleic acids.  VMD can simultaneously display any no. of structures using a wide variety of rendering styles and coloring methods.  Mols. are displayed as one or more "representations," in which each representation embodies a particular rendering method and coloring scheme for a selected subset of atoms.  The atoms displayed in each representation are chosen using an extensive atom selection syntax, which includes Boolean operators and regular expressions.  VMD provides a complete graphical user interface for program control, as well as a text interface using the Tcl embeddable parser to allow for complex scripts with variable substitution, control loops, and function calls.  Full session logging is supported, which produces a VMD command script for later playback.  High-resoln. raster images of displayed mols. may be produced by generating input scripts for use by a no. of photorealistic image-rendering applications.  VMD has also been expressly designed with the ability to animate mol. dynamics (MD) simulation trajectories, imported either from files or from a direct connection to a running MD simulation.  VMD is the visualization component of MDScope, a set of tools for interactive problem solving in structural biol., which also includes the parallel MD program NAMD, and the MDCOMM software used to connect the visualization and simulation programs, VMD is written in C++, using an object-oriented design; the program, including source code and extensive documentation, is freely available via anonymous ftp and through the World Wide Web.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0dd54QvBO97Vg90H21EOLACvtfcHk0lgg1tKpYv_f9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D&md5=1e3094ec3151fb85c5ff05f8505c78d5</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900018-5%26sid%3Dliteratum%253Aachs%26aulast%3DHumphrey%26aufirst%3DW.%26aulast%3DDalke%26aufirst%3DA.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DVMD%253A%2520visual%2520molecular%2520dynamics%26jtitle%3DJ.%2520Mol.%2520Graph.%26date%3D1996%26volume%3D14%26spage%3D33%26epage%3D38%26doi%3D10.1016%2F0263-7855%2896%2900018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varshney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, W. V.</span></span> <span> </span><span class="NLM_article-title">Computing smooth molecular surfaces</span>. <i>IEEE Comput. Graphics Appl.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">19</span>â <span class="NLM_lpage">25</span>, <span class="refDoi">Â DOI: 10.1109/38.310720</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1109%2F38.310720" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=19-25&author=A.+Varshneyauthor=F.+P.+Brooksauthor=W.+V.+Wright&title=Computing+smooth+molecular+surfaces&doi=10.1109%2F38.310720"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1109%2F38.310720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252F38.310720%26sid%3Dliteratum%253Aachs%26aulast%3DVarshney%26aufirst%3DA.%26aulast%3DBrooks%26aufirst%3DF.%2BP.%26aulast%3DWright%26aufirst%3DW.%2BV.%26atitle%3DComputing%2520smooth%2520molecular%2520surfaces%26jtitle%3DIEEE%2520Comput.%2520Graphics%2520Appl.%26date%3D1994%26volume%3D14%26spage%3D19%26epage%3D25%26doi%3D10.1109%2F38.310720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolinsky, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span> <span> </span><span class="NLM_article-title">PDB2PQR: an automated pipeline for the setup of PoissonâBoltzmann electrostatics calculations</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">W665</span>â <span class="NLM_lpage">W667</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkh381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1093%2Fnar%2Fgkh381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=15215472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W665-W667&author=T.+J.+Dolinskyauthor=J.+E.+Nielsenauthor=J.+A.+McCammonauthor=N.+A.+Baker&title=PDB2PQR%3A+an+automated+pipeline+for+the+setup+of+Poisson%E2%80%93Boltzmann+electrostatics+calculations&doi=10.1093%2Fnar%2Fgkh381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">PDB2PQR: An automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations</span></div><div class="casAuthors">Dolinsky, Todd J.; Nielsen, Jens E.; McCammon, J. Andrew; Baker, Nathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W665-W667</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Continuum solvation models, such as Poisson-Boltzmann and Generalized Born methods, have become increasingly popular tools for investigating the influence of electrostatics on biomol. structure, energetics and dynamics.  However, the use of such methods requires accurate and complete structural data as well as force field parameters such as at. charges and radii.  Unfortunately, the limiting step in continuum electrostatics calcns. is often the addn. of missing at. coordinates to mol. structures from the Protein Data Bank and the assignment of parameters to biomol. structures.  To address this problem, we have developed the PDB2PQR web service (http://agave.wustl.edu/pdb2pqr/).  This server automates many of the common tasks of prepg. structures for continuum electrostatics calcns., including adding a limited no. of missing heavy atoms to biomol. structures, estg. titrn. states and protonating biomols. in a manner consistent with favorable hydrogen bonding, assigning charge and radius parameters from a variety of force fields, and finally generating "PQR" output compatible with several popular computational biol. packages.  This service is intended to facilitate the setup and execution of electrostatics calcns. for both experts and non-experts and thereby broaden the accessibility to the biol. community of continuum electrostatics analyses of biomol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-GebFAmzzkrVg90H21EOLACvtfcHk0lgg1tKpYv_f9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsbY%253D&md5=678d2d58f5cb2c1dd862e2d71cf9dbe1</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh381%26sid%3Dliteratum%253Aachs%26aulast%3DDolinsky%26aufirst%3DT.%2BJ.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DBaker%26aufirst%3DN.%2BA.%26atitle%3DPDB2PQR%253A%2520an%2520automated%2520pipeline%2520for%2520the%2520setup%2520of%2520Poisson%25E2%2580%2593Boltzmann%2520electrostatics%2520calculations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW665%26epage%3DW667%26doi%3D10.1093%2Fnar%2Fgkh381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolinsky, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span> <span> </span><span class="NLM_article-title">PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W522</span>â <span class="NLM_lpage">W525</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkm276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1093%2Fnar%2Fgkm276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=17488841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGltQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W522-W525&author=T.+J.+Dolinskyauthor=P.+Czodrowskiauthor=H.+Liauthor=J.+E.+Nielsenauthor=J.+H.+Jensenauthor=G.+Klebeauthor=N.+A.+Baker&title=PDB2PQR%3A+expanding+and+upgrading+automated+preparation+of+biomolecular+structures+for+molecular+simulations&doi=10.1093%2Fnar%2Fgkm276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations</span></div><div class="casAuthors">Dolinsky Todd J; Czodrowski Paul; Li Hui; Nielsen Jens E; Jensen Jan H; Klebe Gerhard; Baker Nathan A</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W522-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Real-world observable physical and chemical characteristics are increasingly being calculated from the 3D structures of biomolecules.  Methods for calculating pK(a) values, binding constants of ligands, and changes in protein stability are readily available, but often the limiting step in computational biology is the conversion of PDB structures into formats ready for use with biomolecular simulation software.  The continued sophistication and integration of biomolecular simulation methods for systems- and genome-wide studies requires a fast, robust, physically realistic and standardized protocol for preparing macromolecular structures for biophysical algorithms.  As described previously, the PDB2PQR web server addresses this need for electrostatic field calculations (Dolinsky et al., Nucleic Acids Research, 32, W665-W667, 2004).  Here we report the significantly expanded PDB2PQR that includes the following features: robust standalone command line support, improved pK(a) estimation via the PROPKA framework, ligand parameterization via PEOE_PB charge methodology, expanded set of force fields and easily incorporated user-defined parameters via XML input files, and improvement of atom addition and optimization code.  These features are available through a new web interface (http://pdb2pqr.sourceforge.net/), which offers users a wide range of options for PDB file conversion, modification and parameterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqIxhnOQkIHOxfW6udTcc2eaW47CWcI6JlLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGltQ%253D%253D&md5=18883d2f9a6f30dba143a4a2ca80b0d8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm276%26sid%3Dliteratum%253Aachs%26aulast%3DDolinsky%26aufirst%3DT.%2BJ.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DBaker%26aufirst%3DN.%2BA.%26atitle%3DPDB2PQR%253A%2520expanding%2520and%2520upgrading%2520automated%2520preparation%2520of%2520biomolecular%2520structures%2520for%2520molecular%2520simulations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW522%26epage%3DW525%26doi%3D10.1093%2Fnar%2Fgkm276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sept, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holst, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Electrostatics of nanosystems: application to microtubules and the ribosome</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10037</span>â <span class="NLM_lpage">10041</span>, <span class="refDoi">Â DOI: 10.1073/pnas.181342398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1073%2Fpnas.181342398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=11517324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=10037-10041&author=N.+A.+Bakerauthor=D.+Septauthor=S.+Josephauthor=M.+J.+Holstauthor=J.+A.+McCammon&title=Electrostatics+of+nanosystems%3A+application+to+microtubules+and+the+ribosome&doi=10.1073%2Fpnas.181342398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatics of nanosystems: application to microtubules and the ribosome</span></div><div class="casAuthors">Baker, Nathan A.; Sept, David; Joseph, Simpson; Holst, Michael J.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10037-10041</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Evaluation of the electrostatic properties of biomols. has become a std. practice in mol. biophysics.  Foremost among the models used to elucidate the electrostatic potential is the Poisson-Boltzmann equation; however, existing methods for solving this equation have limited the scope of accurate electrostatic calcns. to relatively small biomol. systems.  Here we present the application of numerical methods to enable the trivially parallel soln. of the Poisson-Boltzmann equation for supramol. structures that are orders of magnitude larger in size.  As a demonstration of this methodol., electrostatic potentials have been calcd. for large microtubule and ribosome structures.  The results point to the likely role of electrostatics in a variety of activities of these structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_CAlhE0uX7Vg90H21EOLACvtfcHk0lheYfO_bsk3vQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D&md5=1b861999ef12c6972e82e8ada0f387cb</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.181342398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.181342398%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DN.%2BA.%26aulast%3DSept%26aufirst%3DD.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DHolst%26aufirst%3DM.%2BJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DElectrostatics%2520of%2520nanosystems%253A%2520application%2520to%2520microtubules%2520and%2520the%2520ribosome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2001%26volume%3D98%26spage%3D10037%26epage%3D10041%26doi%3D10.1073%2Fpnas.181342398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jurrus, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Star, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felberg, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brookes, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liles, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohara, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolinsky, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konecny, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koes, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head-Gordon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kransky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holst, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCommon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span> <span> </span><span class="NLM_article-title">Improvements to the APBS biomolecular solvation software suite</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">112</span>â <span class="NLM_lpage">128</span>, <span class="refDoi">Â DOI: 10.1002/pro.3280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fpro.3280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=28836357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslSkt7vI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=112-128&author=E.+Jurrusauthor=D.+Engelauthor=K.+Starauthor=K.+Monsonauthor=J.+Brandiauthor=L.+E.+Felbergauthor=D.+H.+Brookesauthor=L.+Wilsonauthor=J.+Chenauthor=K.+Lilesauthor=M.+Chunauthor=P.+Liauthor=D.+W.+Goharaauthor=T.+Dolinskyauthor=R.+Konecnyauthor=D.+R.+Koesauthor=J.+E.+Nielsenauthor=T.+Head-Gordonauthor=W.+Gengauthor=R.+Kranskyauthor=G.-E.+Weiauthor=M.+J.+Holstauthor=J.+A.+McCommonauthor=N.+A.+Baker&title=Improvements+to+the+APBS+biomolecular+solvation+software+suite&doi=10.1002%2Fpro.3280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Improvements to the APBS biomolecular solvation software suite</span></div><div class="casAuthors">Jurrus, Elizabeth; Engel, Dave; Star, Keith; Monson, Kyle; Brandi, Juan; Felberg, Lisa E.; Brookes, David H.; Wilson, Leighton; Chen, Jiahui; Liles, Karina; Chun, Minju; Li, Peter; Gohara, David W.; Dolinsky, Todd; Konecny, Robert; Koes, David R.; Nielsen, Jens Erik; Head-Gordon, Teresa; Geng, Weihua; Krasny, Robert; Wei, Guo-Wei; Holst, Michael J.; McCammon, J. Andrew; Baker, Nathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-128</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Adaptive Poisson-Boltzmann Solver (APBS) software was developed to solve the equations of continuum electrostatics for large biomol. assemblages that have provided impact in the study of a broad range of chem., biol., and biomedical applications.  APBS addresses the three key technol. challenges for understanding solvation and electrostatics in biomedical applications: accurate and efficient models for biomol. solvation and electrostatics, robust and scalable software for applying those theories to biomol. systems, and mechanisms for sharing and analyzing biomol. electrostatics data in the scientific community.  To address new research applications and advancing computational capabilities, we have continually updated APBS and its suite of accompanying software since its release in 2001.  In this article, we discuss the models and capabilities that have recently been implemented within the APBS software package including a Poisson-Boltzmann anal. and a semi-anal. solver, an optimized boundary element solver, a geometry-based geometric flow solvation model, a graph theory-based algorithm for detg. pKa values, and an improved web-based visualization tool for viewing electrostatics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBO7hbYoyPZrVg90H21EOLACvtfcHk0ljqI5j45Lwi8Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslSkt7vI&md5=99651d125e38f4a85d453fecf0f71652</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fpro.3280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3280%26sid%3Dliteratum%253Aachs%26aulast%3DJurrus%26aufirst%3DE.%26aulast%3DEngel%26aufirst%3DD.%26aulast%3DStar%26aufirst%3DK.%26aulast%3DMonson%26aufirst%3DK.%26aulast%3DBrandi%26aufirst%3DJ.%26aulast%3DFelberg%26aufirst%3DL.%2BE.%26aulast%3DBrookes%26aufirst%3DD.%2BH.%26aulast%3DWilson%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLiles%26aufirst%3DK.%26aulast%3DChun%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DGohara%26aufirst%3DD.%2BW.%26aulast%3DDolinsky%26aufirst%3DT.%26aulast%3DKonecny%26aufirst%3DR.%26aulast%3DKoes%26aufirst%3DD.%2BR.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26aulast%3DHead-Gordon%26aufirst%3DT.%26aulast%3DGeng%26aufirst%3DW.%26aulast%3DKransky%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DG.-E.%26aulast%3DHolst%26aufirst%3DM.%2BJ.%26aulast%3DMcCommon%26aufirst%3DJ.%2BA.%26aulast%3DBaker%26aufirst%3DN.%2BA.%26atitle%3DImprovements%2520to%2520the%2520APBS%2520biomolecular%2520solvation%2520software%2520suite%26jtitle%3DProtein%2520Sci.%26date%3D2018%26volume%3D27%26spage%3D112%26epage%3D128%26doi%3D10.1002%2Fpro.3280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stone, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickey, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickey, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulson, D. F.</span></span> <span> </span><span class="NLM_article-title">Isolation of a human prostate carcinoma cell line (DU 145)</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">274</span>â <span class="NLM_lpage">281</span>, <span class="refDoi">Â DOI: 10.1002/ijc.2910210305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1002%2Fijc.2910210305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=631930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A280%3ADyaE1c7ivFajsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1978&pages=274-281&author=K.+R.+Stoneauthor=D.+D.+Mickeyauthor=H.+Wunderliauthor=G.+H.+Mickeyauthor=D.+F.+Paulson&title=Isolation+of+a+human+prostate+carcinoma+cell+line+%28DU+145%29&doi=10.1002%2Fijc.2910210305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of a human prostate carcinoma cell line (DU 145)</span></div><div class="casAuthors">Stone K R; Mickey D D; Wunderli H; Mickey G H; Paulson D F</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">274-81</span>
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    </div><div class="casAbstract">A long-term tissue culture cell line has been derived from a human prostate adenocarcinoma metastatic to the brain.  The cell line, DU 145, has been passaged 90 times in vitro over a period of 2 years.  The cells are epithelial, grow in isolated islands on plastic Petri dishes, and form colonies in soft agar suspension culture.  Karyotypic analysis demonstrates an aneuploid human karyotype with a modal chromosome number of 64.  Distinctive marker chromosomes (a translocation Y chromosome, metacentric minute chromosomes and three large acrocentic chromosomes) have been identified.  Electron microscopy of the original tumor tissue and of the tissue culture cell line show a remarkable similarity in cell organelle structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRbXe-vo_9hfa__Jf7-eMvJfW6udTcc2eYI_Rnu040pkbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE1c7ivFajsA%253D%253D&md5=128a9ca6ae610280b0e3b8e2c6cc04f7</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fijc.2910210305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.2910210305%26sid%3Dliteratum%253Aachs%26aulast%3DStone%26aufirst%3DK.%2BR.%26aulast%3DMickey%26aufirst%3DD.%2BD.%26aulast%3DWunderli%26aufirst%3DH.%26aulast%3DMickey%26aufirst%3DG.%2BH.%26aulast%3DPaulson%26aufirst%3DD.%2BF.%26atitle%3DIsolation%2520of%2520a%2520human%2520prostate%2520carcinoma%2520cell%2520line%2520%2528DU%2520145%2529%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D1978%26volume%3D21%26spage%3D274%26epage%3D281%26doi%3D10.1002%2Fijc.2910210305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breul, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hance, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crystal, R. G.</span></span> <span> </span><span class="NLM_article-title">Control of collagen production by human diploid lung fibroblasts</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>255</i></span>,  <span class="NLM_fpage">5250</span>â <span class="NLM_lpage">5260</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=7372635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaL3cXksVans7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=255&publication_year=1980&pages=5250-5260&author=S.+D.+Breulauthor=K.+H.+Bradleyauthor=A.+J.+Hanceauthor=M.+P.+Schaferauthor=R.+A.+Bergauthor=R.+G.+Crystal&title=Control+of+collagen+production+by+human+diploid+lung+fibroblasts"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Control of collagen production by human diploid lung fibroblasts</span></div><div class="casAuthors">Breul, Sally D.; Bradley, Kathryn H.; Hance, Allan J.; Schafer, Millie P.; Berg, Richard A.; Crystal, Ronald G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">255</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5250-60</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">To evaluate the control soft tissue fibroblasts have over collagen prodn., HFL-1, a diploid human lung cell strain, was studied during periods of rapid cell growth and relatively slow growth over 25 population doublings.  To minimize environmental influences, the extracellular milieu of the cells was kept const. throughout the study period.  Rates of collagen prodn. per cell unit time were quantitated by labeling HFL-1 with [14C]proline and measuring the prodn. of [14C]hydroxyproline after taking into consideration the sp. activity of [14C]proline within the free intracellular proline pool and the percent hydroxylation of proline residues in newly synthesized collagen.  Although the sp. activity of intracellular free proline and the percent hydroxylation of proline in collagen varied considerably depending on the growth rates of the cells, collagen prodn. by HFL-1 was const., even during periods of rapid cell growth.  Thus, under conditions of a stable environment, populations of soft tissue fibroblasts rigidly control their collagen prodn.  In cultures that maintained a const. doubling time, this stability was maintained over â¥25 population doublings, suggesting that on the av., collagen prodn. appears to be tightly controlled and dissocd. from the events and sequelae of cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyK2bRXCUwSrVg90H21EOLACvtfcHk0lg-S8DGj2qVfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXksVans7s%253D&md5=73ba43348807e2bdd8fe59b18af080ea</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBreul%26aufirst%3DS.%2BD.%26aulast%3DBradley%26aufirst%3DK.%2BH.%26aulast%3DHance%26aufirst%3DA.%2BJ.%26aulast%3DSchafer%26aufirst%3DM.%2BP.%26aulast%3DBerg%26aufirst%3DR.%2BA.%26aulast%3DCrystal%26aufirst%3DR.%2BG.%26atitle%3DControl%2520of%2520collagen%2520production%2520by%2520human%2520diploid%2520lung%2520fibroblasts%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1980%26volume%3D255%26spage%3D5250%26epage%3D5260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardenas, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floreancig, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Brabander, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, M. G.</span></span> <span> </span><span class="NLM_article-title">Studies toward the unique pederin family member psymberin: structureâactivity relationships, biochemical studies, and genetics identify the mode-of-action of psymberin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">18998</span>â <span class="NLM_lpage">19003</span>, <span class="refDoi">Â DOI: 10.1021/ja3057002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja3057002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs7%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=18998-19003&author=C.-Y.+Wuauthor=Y.+Fengauthor=E.+R.+Cardenasauthor=N.+Williamsauthor=P.+E.+Floreancigauthor=J.+K.+De+Brabanderauthor=M.+G.+Roth&title=Studies+toward+the+unique+pederin+family+member+psymberin%3A+structure%E2%80%93activity+relationships%2C+biochemical+studies%2C+and+genetics+identify+the+mode-of-action+of+psymberin&doi=10.1021%2Fja3057002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Studies toward the Unique Pederin Family Member Psymberin: Structure-Activity Relationships, Biochemical Studies, and Genetics Identify the Mode-of-Action of Psymberin</span></div><div class="casAuthors">Wu, Cheng-Yang; Feng, Yu; Cardenas, Eduardo R.; Williams, Noelle; Floreancig, Paul E.; De Brabander, Jef K.; Roth, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18998-19003</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Psymberin is the only member of the pederin natural product family that contains a dihydroisocoumarin side chain.  Structural modifications of psymberin uncoupled inhibition of protein translation from cytotoxicity, suggesting that psymberin has more than one bioactivity.  A forward genetic screen in Caenorhabditis elegans was conducted to identify the mol. target(s) of psymberin.  Multiple independent psymberin-resistant mutants were isolated, each contg. the same point mutation in a gene encoding a ribosomal protein.  However, a psymberin-resistant mutant strain bearing this mutation was not cross-resistant to the pederin family member mycalamide A, which binds to the archaeal form of the same protein.  Thus, two pederin family members likely differ in how they bind the same mol. target.  The accumulation of psymberin in cells was sensitive to the stereochem. of the amide side chain at C4 or C8 and the presence of the dihydroisocoumarin side chain.  The observation that psymberin diastereomers or dihydroisocoumarin-truncated analogs lose all cytotoxic activity while retaining the ability to inhibit protein translation in a cell-free in vitro assay can be explained in the context of these differential cell uptake issues.  Finally, we also demonstrate that the blistering activity assocd. with pederin and other members of the family is not due to their protein synthesis inhibiting activity.  Unlike pederin and mycalamide, psymberin does not display irritant or blistering activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxBa84k1Y2prVg90H21EOLACvtfcHk0lg-S8DGj2qVfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs7%252FI&md5=4056ff204d8ec50929401aa35f470622</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fja3057002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja3057002%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DC.-Y.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DCardenas%26aufirst%3DE.%2BR.%26aulast%3DWilliams%26aufirst%3DN.%26aulast%3DFloreancig%26aufirst%3DP.%2BE.%26aulast%3DDe%2BBrabander%26aufirst%3DJ.%2BK.%26aulast%3DRoth%26aufirst%3DM.%2BG.%26atitle%3DStudies%2520toward%2520the%2520unique%2520pederin%2520family%2520member%2520psymberin%253A%2520structure%25E2%2580%2593activity%2520relationships%252C%2520biochemical%2520studies%252C%2520and%2520genetics%2520identify%2520the%2520mode-of-action%2520of%2520psymberin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D18998%26epage%3D19003%26doi%3D10.1021%2Fja3057002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urbatsch, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilke-Mounts, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner-Marmarosh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseau, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senior, A. E.</span></span> <span> </span><span class="NLM_article-title">Cysteines 431 and 1074 are responsible for inhibitory disulfide cross-linking between the two nucleotide-binding sites in human P-glycoprotein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">26980</span>â <span class="NLM_lpage">26987</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M010829200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1074%2Fjbc.M010829200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=11356825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsV2jt7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=26980-26987&author=I.+L.+Urbatschauthor=K.+Gimiauthor=S.+Wilke-Mountsauthor=N.+Lerner-Marmaroshauthor=M.+E.+Rousseauauthor=P.+Grosauthor=A.+E.+Senior&title=Cysteines+431+and+1074+are+responsible+for+inhibitory+disulfide+cross-linking+between+the+two+nucleotide-binding+sites+in+human+P-glycoprotein&doi=10.1074%2Fjbc.M010829200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteines 431 and 1074 are responsible for inhibitory disulfide cross-linking between the two nucleotide-binding sites in human P-glycoprotein</span></div><div class="casAuthors">Urbatsch, Ina L.; Gimi, Khursheed; Wilke-Mounts, Susan; Lerner-Marmarosh, Nicole; Rousseau, Marc-Etienne; Gros, Philippe; Senior, Alan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">26980-26987</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human wild-type and Cys-less P-glycoproteins were expressed in Pichia pastoris and purified in high yield in detergent-sol. form.  Both ran on SDS gels as a single 140-kDa band in the presence of reducing agent and showed strong verapamil-stimulated ATPase activity in the presence of added lipid.  The wild type showed spontaneous formation of higher mol. mass species in the absence of reducing agent, and its ATPase was activated by dithiothreitol.  Oxidn. with Cu2+ generated the same higher mol. mass species, primarily at 200 and â¼300 kDa, in high yield.  Crosslinking was reversed by dithiothreitol and prevented by pretreatment with N-ethylmaleimide.  Using proteins contg. different combinations of naturally occurring Cys residues, it was demonstrated that an inhibitory intramol. disulfide bond forms between Cys-431 and Cys-1074 (located in the Walker A sequences of nucleotide-binding sites 1 and 2, resp.), giving rise to the 200-kDa species.  In addn., dimeric P-glycoprotein species (â¼300 kDa) form by intermol. disulfide bonding between Cys-431 and Cys-1074.  The ready formation of the intramol. disulfide between Cys-431 and Cys-1074 establishes that the two nucleotide-binding sites of P-glycoprotein are structurally very close and capable of intimate functional interaction, consistent with available information on the catalytic mechanism.  Formation of such a disulfide in vivo could, in principle, underlie a regulatory mechanism and might provide a means of intervention to inhibit P-glycoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfw4Oa4Dx7BbVg90H21EOLACvtfcHk0ljJMaveTtZC3w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsV2jt7s%253D&md5=8238a6cad5571519fe1120360467cd81</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M010829200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M010829200%26sid%3Dliteratum%253Aachs%26aulast%3DUrbatsch%26aufirst%3DI.%2BL.%26aulast%3DGimi%26aufirst%3DK.%26aulast%3DWilke-Mounts%26aufirst%3DS.%26aulast%3DLerner-Marmarosh%26aufirst%3DN.%26aulast%3DRousseau%26aufirst%3DM.%2BE.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DSenior%26aufirst%3DA.%2BE.%26atitle%3DCysteines%2520431%2520and%25201074%2520are%2520responsible%2520for%2520inhibitory%2520disulfide%2520cross-linking%2520between%2520the%2520two%2520nucleotide-binding%2520sites%2520in%2520human%2520P-glycoprotein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D26980%26epage%3D26987%26doi%3D10.1074%2Fjbc.M010829200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, F. J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orelle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, A. L.</span></span> <span> </span><span class="NLM_article-title">Functional reconstitution of an ABC transporter in nanodiscs for use in electron paramagnetic resonance spectroscopy</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">9513</span>â <span class="NLM_lpage">9515</span>, <span class="refDoi">Â DOI: 10.1021/ja104047c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja104047c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFSqtrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=9513-9515&author=F.+J.+D.+Alvarezauthor=C.+Orelleauthor=A.+L.+Davidson&title=Functional+reconstitution+of+an+ABC+transporter+in+nanodiscs+for+use+in+electron+paramagnetic+resonance+spectroscopy&doi=10.1021%2Fja104047c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Reconstitution of an ABC Transporter in Nanodiscs for Use in Electron Paramagnetic Resonance Spectroscopy</span></div><div class="casAuthors">Alvarez, Frances Joan D.; Orelle, Cedric; Davidson, Amy L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">9513-9515</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ESR spectroscopy is a powerful biophys. technique for study of the structural dynamics of membrane proteins.  Many of these proteins interact with ligands or proteins on one or both sides of the membrane.  Membrane proteins are typically reconstituted in proteoliposomes to observe their function in a physiol. relevant environment.  However, membrane proteins can insert into liposomes in two different orientations, and surfaces facing the lumen of the vesicle can be inaccessible to ligands.  This heterogeneity can lead to subpopulations that do not respond to ligand binding, complicating EPR spectral anal., particularly for distance measurements.  Using the well-characterized maltose transporter, an ATP binding cassette (ABC) transporter that interacts with ligands on both sides of the membrane, the authors provide evidence that reconstitution into nanodisks, which are sol. disk-shaped phospholipid bilayers, is an ideal soln. to these problems.  The authors describe the functional reconstitution of the maltose transporter into nanodisks and demonstrate that this system is ideally suited to study conformational changes and intramol. distances by EPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMX4DDZdg-SrVg90H21EOLACvtfcHk0ljJMaveTtZC3w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFSqtrk%253D&md5=820ad086b6112fd50a1ad669b0b09540</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fja104047c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja104047c%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez%26aufirst%3DF.%2BJ.%2BD.%26aulast%3DOrelle%26aufirst%3DC.%26aulast%3DDavidson%26aufirst%3DA.%2BL.%26atitle%3DFunctional%2520reconstitution%2520of%2520an%2520ABC%2520transporter%2520in%2520nanodiscs%2520for%2520use%2520in%2520electron%2520paramagnetic%2520resonance%2520spectroscopy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D9513%26epage%3D9515%26doi%3D10.1021%2Fja104047c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayburt, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sligar, S. G.</span></span> <span> </span><span class="NLM_article-title">Membrane protein assembly into Nanodiscs</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>584</i></span>,  <span class="NLM_fpage">1721</span>â <span class="NLM_lpage">1727</span>, <span class="refDoi">Â DOI: 10.1016/j.febslet.2009.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=10.1016%2Fj.febslet.2009.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=19836392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFKqurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2010&pages=1721-1727&author=T.+H.+Bayburtauthor=S.+G.+Sligar&title=Membrane+protein+assembly+into+Nanodiscs&doi=10.1016%2Fj.febslet.2009.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane protein assembly into nanodisks</span></div><div class="casAuthors">Bayburt, Timothy H.; Sligar, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1721-1727</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nanodisks are sol. nanoscale phospholipid bilayers which can self-assemble integral membrane proteins for biophys., enzymic or structural investigations.  This means for rendering membrane proteins sol. at the single-mol. level offers advantages over liposomes or detergent micelles in terms of size, stability, ability to add genetically modifiable features to the nanodisk structure and ready access to both sides of the phospholipid bilayer domain.  Thus, the nanodisc system provides a novel platform for understanding membrane protein function.  Here, the authors provide an overview of the nanodisk approach and document through several examples many of the applications to the study of the structure and function of integral membrane proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphr-NB4w9pNbVg90H21EOLACvtfcHk0lgAgMssswkiUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFKqurw%253D&md5=1ace5001c844ff546e837251fdcb0774</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2009.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2009.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DBayburt%26aufirst%3DT.%2BH.%26aulast%3DSligar%26aufirst%3DS.%2BG.%26atitle%3DMembrane%2520protein%2520assembly%2520into%2520Nanodiscs%26jtitle%3DFEBS%2520Lett.%26date%3D2010%26volume%3D584%26spage%3D1721%26epage%3D1727%26doi%3D10.1016%2Fj.febslet.2009.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhart, R.</span></span> <span> </span><span class="NLM_article-title">ATPase of Escherichia coli: purification, dissociation, and reconstitution of the active complex from the isolated subunits</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">208</span>â <span class="NLM_lpage">216</span>, <span class="refDoi">Â DOI: 10.1021/bi00646a032</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00646a032" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;key=1%3ACAS%3A528%3ADyaE28XlsFyrsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1976&pages=208-216&author=G.+Vogelauthor=R.+Steinhart&title=ATPase+of+Escherichia+coli%3A+purification%2C+dissociation%2C+and+reconstitution+of+the+active+complex+from+the+isolated+subunits&doi=10.1021%2Fbi00646a032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">ATPase of Escherichia coli:  purification, dissociation, and reconstitution of the active complex from the isolated subunits</span></div><div class="casAuthors">Vogel, Guenter; Steinhart, Roswithe</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">208-16</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">A simple procedure for the purifn. of Mg2+-stimulated ATPase of Escherichia coli by fractionation with poly(ethylene glycols) and gel filtration is described.  The enzyme restored ATPase-linked reactions to membrane prepns. lacking these activities.  Five different polypeptides (Î±, Î², Î³, Î´, Îµ) were obsd. in Na dodecyl sulfate electrophoresis.  Freezing in salt solns. split the enzyme complex into subunits which did not possess any catalytic activity.  The presence of different subunits was confirmed by electrophoretic and immunol. methods.  The active enzyme complex could be reconstituted by decreasing the ionic strength in the dissocd. sample.  Temp., pH, protein concn., and the presence of substrate were each important determinants of the rate and extent of reconstitution.  The dissocd. enzyme was sepd. by ion-exchange chromatog. into 2 major fragments.  Fragment IA had a mol. wt. of â¼100,000 and contained the Î±, Î³, and Îµ polypeptides.  The minor fragment, IB, had â¼ the same mol. wt. but contained, besides Î±, Î³, and Îµ, the Î´ polypeptide.  Fragment II, with a mol. wt. of â¼52,000, appeared to be identical with the Î² polypeptide.  ATPase activity could be reconstituted from fragments IA and II, whereas the capacity of the ATPase to drive energy-dependent processes in depleted membrane vesicles was only restored after incubation of these 2 fractions with fraction IB, which contained the Î´ subunit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpaMbB-0XbsrVg90H21EOLACvtfcHk0lgAgMssswkiUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XlsFyrsw%253D%253D&md5=6e763d83e76b35a1cdd6da8f5c5566e8</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fbi00646a032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00646a032%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DG.%26aulast%3DSteinhart%26aufirst%3DR.%26atitle%3DATPase%2520of%2520Escherichia%2520coli%253A%2520purification%252C%2520dissociation%252C%2520and%2520reconstitution%2520of%2520the%2520active%2520complex%2520from%2520the%2520isolated%2520subunits%26jtitle%3DBiochemistry%26date%3D1976%26volume%3D15%26spage%3D208%26epage%3D216%26doi%3D10.1021%2Fbi00646a032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HYD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HYD','PDB','2HYD'); return false;">PDB: 2HYD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0C','PDB','6V0C'); return false;">PDB: 6V0C</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i44"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00966">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_54453"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b00966?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00966</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Description of ChemGen programs and workflow, figure of chemical spaces of variants produced, figure of binding interactions of variants with P-glycoprotein, chemical synthesis scheme, synthetic procedures, analytical data: HRMS and NMR, and NMR spectra and molecular strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf">PDF</a>)</p></li><li><p class="inline">SMILES, inhibition of P-gp ATPase, and inhibition of ATP binding to P-gp of given molecules (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_002.csv">CSV</a>)</p></li><li><p class="inline">Human P-glycoprotein (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_001.pdf">jm9b00966_si_001.pdf (6.22 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_002.csv">jm9b00966_si_002.csv (1.24 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00966/suppl_file/jm9b00966_si_003.pdb">jm9b00966_si_003.pdb (711.07 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00966&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-23%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00966" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00966" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ac7f97c2621d6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
