
<html lang="en"     class="pb-page"  data-request-id="6e1394b0-ea07-4744-aa7a-24f0befbd26c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-9;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.0c00101;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design and Characterization of the First Selective and Potent Mechanism-Based Inhibitor of Cytochrome P450 4Z1" /></meta><meta name="dc.Creator" content="John P.  Kowalski" /></meta><meta name="dc.Creator" content="Matthew G.  McDonald" /></meta><meta name="dc.Creator" content="Robert D.  Pelletier" /></meta><meta name="dc.Creator" content="Helmut  Hanenberg" /></meta><meta name="dc.Creator" content="Constanze  Wiek" /></meta><meta name="dc.Creator" content="Allan E.  Rettie" /></meta><meta name="dc.Description" content="Mammary-tissue-restricted cytochrome P450 4Z1 (CYP4Z1) has garnered interest for its potential role in breast cancer progression. CYP4Z1-dependent metabolism of arachidonic acid preferentially gene..." /></meta><meta name="Description" content="Mammary-tissue-restricted cytochrome P450 4Z1 (CYP4Z1) has garnered interest for its potential role in breast cancer progression. CYP4Z1-dependent metabolism of arachidonic acid preferentially gene..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 17, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00101" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00101" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00101" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00101" /></link>
        
    
    

<title>Design and Characterization of the First Selective and Potent Mechanism-Based Inhibitor of Cytochrome P450 4Z1 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00101" /></meta><meta property="og:title" content="Design and Characterization of the First Selective and Potent Mechanism-Based Inhibitor of Cytochrome P450 4Z1" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0008.jpeg" /></meta><meta property="og:description" content="Mammary-tissue-restricted cytochrome P450 4Z1 (CYP4Z1) has garnered interest for its potential role in breast cancer progression. CYP4Z1-dependent metabolism of arachidonic acid preferentially generates 14,15-epoxyeicosatrienoic acid (14,15-EET), a metabolite known to influence cellular proliferation, migration, and angiogenesis. In this study, we developed time-dependent inhibitors of CYP4Z1 designed as fatty acid mimetics linked to the bioactivatable pharmacophore, 1-aminobenzotriazole (ABT). The most potent analogue, 8-[(1H-benzotriazol-1-yl)amino]octanoic acid (7), showed a 60-fold lower shifted-half-maximal inhibitory concentration (IC50) for CYP4Z1 compared to ABT, efficient mechanism-based inactivation of the enzyme evidenced by a KI = 2.2 μM and a kinact = 0.15 min–1, and a partition ratio of 14. Furthermore, 7 exhibited low off-target inhibition of other CYP isozymes. Finally, low micromolar concentrations of 7 inhibited 14,15-EET production in T47D breast cancer cells transfected with CYP4Z1. This first-generation, selective mechanism-based inhibitor (MBI) will be a useful molecular tool to probe the biochemical role of CYP4Z1 and its association with breast cancer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00101"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00101">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00101&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00101&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00101&amp;href=/doi/10.1021/acs.jmedchem.0c00101" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4824-4836</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00048" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00117" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design and Characterization of the First Selective and Potent Mechanism-Based Inhibitor of Cytochrome P450 4Z1</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">John P. Kowalski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John P. Kowalski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, School of Pharmacy, University of Washington, Seattle, Washington 98105, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+P.++Kowalski">John P. Kowalski</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0343-8425" title="Orcid link">http://orcid.org/0000-0002-0343-8425</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew G. McDonald</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew G. McDonald</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, School of Pharmacy, University of Washington, Seattle, Washington 98105, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+G.++McDonald">Matthew G. McDonald</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert D. Pelletier</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert D. Pelletier</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, School of Pharmacy, University of Washington, Seattle, Washington 98105, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+D.++Pelletier">Robert D. Pelletier</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Helmut Hanenberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Helmut Hanenberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pediatrics III, University Children’s Hospital Essen, University of Duisburg−Essen, 45122 Essen, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Helmut++Hanenberg">Helmut Hanenberg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Constanze Wiek</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Constanze Wiek</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Otorhinolaryngology and Head/Neck Surgery, Heinrich-Heine University, 40225 Düsseldorf, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Constanze++Wiek">Constanze Wiek</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Allan E. Rettie</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Allan E. Rettie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, School of Pharmacy, University of Washington, Seattle, Washington 98105, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#fd8f9889899498bd888ad3989988"><span class="__cf_email__" data-cfemail="7d0f18090914183d080a53181908">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Allan+E.++Rettie">Allan E. Rettie</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00101&amp;href=/doi/10.1021%2Facs.jmedchem.0c00101" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4824–4836</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 17, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 January 2020</li><li><span class="item_label"><b>Published</b> online</span>17 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00101" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00101</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4824%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJohn%2BP.%2BKowalski%252C%2BMatthew%2BG.%2BMcDonald%252C%2BRobert%2BD.%2BPelletier%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D9%26contentID%3Dacs.jmedchem.0c00101%26title%3DDesign%2Band%2BCharacterization%2Bof%2Bthe%2BFirst%2BSelective%2Band%2BPotent%2BMechanism-Based%2BInhibitor%2Bof%2BCytochrome%2BP450%2B4Z1%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4836%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00101"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">868</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00101" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design and Characterization of the First Selective and Potent Mechanism-Based Inhibitor of Cytochrome P450 4Z1&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;P. Kowalski&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;G. McDonald&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;D. Pelletier&quot;},{&quot;first_name&quot;:&quot;Helmut&quot;,&quot;last_name&quot;:&quot;Hanenberg&quot;},{&quot;first_name&quot;:&quot;Constanze&quot;,&quot;last_name&quot;:&quot;Wiek&quot;},{&quot;first_name&quot;:&quot;Allan&quot;,&quot;last_name&quot;:&quot;E. Rettie&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;4824-4836&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00101&quot;},&quot;abstract&quot;:&quot;Mammary-tissue-restricted cytochrome P450 4Z1 (CYP4Z1) has garnered interest for its potential role in breast cancer progression. CYP4Z1-dependent metabolism of arachidonic acid preferentially generates 14,15-epoxyeicosatrienoic acid (14,15-EET), a metabolite known to influence cellular proliferation, migration, and angiogenesis. In this study, we developed time-dependent inhibitors of CYP4Z1 designed as fatty acid mimetics linked to the bioactivatable pharmacophore, 1-aminobenzotriazole (ABT). The most potent analogue, 8-[(1H-benzotriazol-1-yl)amino]octanoic acid (7), showed a 60-fold lower shifted-half-maximal inhibitory concentration (IC50) for CYP4Z1 compared to ABT, efficient mechanism-based inactivation of the enzyme evidenced by a KI = 2.2 μM and a kinact = 0.15 min–1, and a partition ratio of 14. Furthermore, 7 exhibited low off-target inhibition of other CYP isozymes. Finally, low micromolar concentrations of 7 inhibited 14,15-EET production in T47D breast cancer cells transfected with CYP4Z1. This f&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00101&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00101" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00101&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00101" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00101&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00101" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00101&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00101&amp;href=/doi/10.1021/acs.jmedchem.0c00101" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00101" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00101" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00101%26sid%3Dliteratum%253Aachs%26pmid%3D32302132%26genre%3Darticle%26aulast%3DKowalski%26date%3D2020%26atitle%3DDesign%2Band%2BCharacterization%2Bof%2Bthe%2BFirst%2BSelective%2Band%2BPotent%2BMechanism-Based%2BInhibitor%2Bof%2BCytochrome%2BP450%2B4Z1%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D9%26spage%3D4824%26epage%3D4836%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292143" title="Metabolism">Metabolism</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/jmcmar.2020.63.issue-9/20200514/jmcmar.2020.63.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Mammary-tissue-restricted cytochrome P450 4Z1 (CYP4Z1) has garnered interest for its potential role in breast cancer progression. CYP4Z1-dependent metabolism of arachidonic acid preferentially generates 14,15-epoxyeicosatrienoic acid (14,15-EET), a metabolite known to influence cellular proliferation, migration, and angiogenesis. In this study, we developed time-dependent inhibitors of CYP4Z1 designed as fatty acid mimetics linked to the bioactivatable pharmacophore, 1-aminobenzotriazole (<b>ABT</b>). The most potent analogue, 8-[(1<i>H</i>-benzotriazol-1-yl)amino]octanoic acid (<b>7</b>), showed a 60-fold lower shifted-half-maximal inhibitory concentration (IC<sub>50</sub>) for CYP4Z1 compared to <b>ABT</b>, efficient mechanism-based inactivation of the enzyme evidenced by a <i>K</i><sub>I</sub> = 2.2 μM and a <i>k</i><sub>inact</sub> = 0.15 min<sup>–1</sup>, and a partition ratio of 14. Furthermore, <b>7</b> exhibited low off-target inhibition of other CYP isozymes. Finally, low micromolar concentrations of <b>7</b> inhibited 14,15-EET production in T47D breast cancer cells transfected with CYP4Z1. This first-generation, selective mechanism-based inhibitor (MBI) will be a useful molecular tool to probe the biochemical role of CYP4Z1 and its association with breast cancer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span> Note</h4><p class="last">After this paper was published ASAP on April 30, 2020, corrections were made to the TOC graphic, abstract graphic, and Figure 1. The corrected version was reposted May 14, 2020.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77687" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77687" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cytochrome P450 (CYP) enzymes are a critical family of hemoproteins that are involved in the metabolism of both xenobiotic and endogenous molecules.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In 2004, CYP4Z1 was cloned from a breast carcinoma line and subsequently shown by several groups to be upregulated in breast carcinoma.<a onclick="showRef(event, 'ref2 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref2 ref58 ref59">(2−4)</a> Other expression profiling studies also demonstrate consistently that the CYP4Z1 transcript level is associated with an elevated tumor grade and increased tumor aggressiveness.<a onclick="showRef(event, 'ref56 ref57 ref60'); return false;" href="javascript:void(0);" class="ref ref56 ref57 ref60">(5−7)</a> Some mechanistic insight has been provided by the finding that overexpression of CYP4Z1 promotes angiogenesis and tumor growth, both in vitro and in vivo.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(8)</a></div><div class="NLM_p">Many CYP4 enzymes are capable of metabolizing polyunsaturated fatty acids (PUFAs) to biological compounds that participate in a complex web of signaling.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(9)</a> Key metabolites of arachidonic acid (AA, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), namely, 20-hydroxyeicosatetraenoic acid (20-HETE) and 14,15-epoxyeicosatrienoic acid (14,15-EET), are thought to play roles in human cancer progression, by influencing angiogenesis, tumor growth, and metastasis.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(10−12)</a> We have recently established that 14,15-EET is a dominant metabolite of CYP4Z1-dependent AA metabolism and have speculated that a role for CYP4Z1 in breast cancer may involve this signaling ligand.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(13)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. CYP4Z1 metabolizes substrates internally. The primary oxidation positions for substrates arachidonic acid (AA), lauric acid (LA), and luciferin-benzyl ether (Luc-BE) are denoted by the red arrows, generating products 14,15-epoxyeicosatrienoic acid (14,15-EET), 8-hydroxy LA, and luciferin (via O-dealkylation), respectively.<a onclick="showRef(event, 'ref8 ref15 ref54'); return false;" href="javascript:void(0);" class="ref ref8 ref15 ref54">(13,20,34)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Breast cancer is one of the leading causes of death in women, with more than 2 million new cases diagnosed worldwide in 2018.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(14)</a> Novel therapeutic modalities are needed urgently, especially for recurrent and metastatic tumors.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(15)</a> Endogenous mammary-specific expression of CYP4Z1 offers a new avenue to explore for breast cancer treatment, and there has already been success with CYP inhibitors developed for anticancer therapy. For example, aromatase (CYP19) inhibitors, such as exemestane, letrozole, and anastrozole, are useful in breast cancer treatment because they effectively limit estrogen production.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(16)</a></div><div class="NLM_p last">Therefore, the goal of this study was to develop a potent and selective mechanism-based inhibitor (MBI) of CYP4Z1 to enable the biochemical characterization of this enzyme and help unravel the role that bioactive lipid metabolites play in breast cancer progression. Herein, we show that analogues of 1-aminobenzotriazole (<b>ABT</b>) are potent time-dependent inhibitors (TDIs) of CYP4Z1. Two analogues (<b>6</b>, <b>7</b>) showed pronounced selectivity as reversible inhibitors of CYP4Z1, compared to other CYP enzymes, and even higher selectivity when they were evaluated for their time-dependent effects on off-target CYPs. Further, we confirmed mechanism-based inhibition for our most potent and selective analogue, 8-[(1<i>H</i>-benzotriazol-1-yl)amino]octanoic acid (<b>7</b>), determined the kinetics of inactivation, and assessed the potency of <b>7</b> toward inhibition of CYP4Z1-mediated 14,15-EET production in a breast cancer cell line.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47052" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47052" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> CYP4Z1 Assay for Rapid Inhibitor Screening</h3><div class="NLM_p">A mammalian enzyme source was preferred, and, historically, HepG2 cells have been suitable for heterologous expression of CYPs due to their very low basal CYP levels and catalytically sufficient amounts of the co-enzymes cytochrome P450 reductase and cytochrome <i>b</i><sub>5</sub>.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(17,18)</a> Therefore, a CYP4Z1-expressing HepG2 cell line was generated via lentiviral transduction (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S1A</a>) and membranes prepared from these cells were used as the primary enzyme source for screening. This expression system provided an adequate amount of CYP4Z1, with no detectable enzyme in a control vector cell line, as analyzed by western blot (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S1B</a>). Quantification via carbon monoxide-binding spectra<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(19)</a> showed that HepG2 membrane preparations contained 1 μM active CYP4Z1 enzyme (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S1D</a>). To test for CYP4Z1 inhibition, we established a simple, sensitive functional assay using the recently identified CYP4Z1 substrate, luciferin-benzyl ether<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(20)</a> (Luc-BE, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Utilization of this probe substrate enabled higher throughput evaluation of inhibitor potency versus the use of more technically cumbersome fatty acid(s) such as laurate and arachidonate.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(13)</a> Luc-BE turnover to its luminescent metabolite, luciferin, is easily analyzed with Promega’s P450-Glo kit.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(21)</a> The reaction is reduced β-nicotinamide-adenine dinucleotide phosphate (NADPH)-dependent, its turnover is more than 500-fold higher than the background (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), and it displays apparent Michaelis–Menten kinetics (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). For the determination of half-maximal inhibitory concentration (IC<sub>50</sub>) values for CYP4Z1 inhibition, the probe substrate Luc-BE was used at a concentration equal to the determined <i>K</i><sub>m</sub> (29 μM). To evaluate both the reversible and time-dependent nature of potential inhibitors, we used a nondilution (also termed addition) method IC<sub>50</sub>-shift assay.<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(22−24)</a> Typically, if a compound elicits an IC<sub>50</sub> shift of ≥1.5, it is considered to be a time-dependent inhibitor of the enzyme being evaluated.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(25)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Probe substrate Luc-BE metabolism in the HepG2 membranes is both CYP4Z1- and NADPH-dependent. (B) CYP4Z1-HepG2 membrane metabolism of Luc-BE displays Michaelis–Menten kinetics at the substrate concentrations tested, with a <i>K</i><sub>m</sub> = 29 μM and a <i>V</i><sub>max</sub> = 20 pmol/pmol CYP4Z1/min; the Eadie–Hofstee plot (inset) shows the apparent linearity. Shown is the average ± standard deviation (SD) from at least three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Design and Synthesis of <b>ABT</b> Analogues for CYP4Z1 Inhibition</h3><div class="NLM_p">We undertook a targeted design strategy for a CYP4Z1 mechanism-based inhibitor to capitalize on potential selectivity benefits. The rationale for increased specificity is based on the following features: (i) reversible binding of the inhibitor to the enzyme must be of appropriate affinity, (ii) the inhibitor must also serve as a substrate for bioactivation by the target, and (iii) the reactive metabolite that is generated requires an active site target to inactivate the enzyme.<a onclick="showRef(event, 'ref1 ref21'); return false;" href="javascript:void(0);" class="ref ref1 ref21">(1,26)</a></div><div class="NLM_p">The known inhibitors of CYP4Z1 are promiscuous toward the CYP4 family and other hepatic CYP enzymes.<a onclick="showRef(event, 'ref8 ref15'); return false;" href="javascript:void(0);" class="ref ref8 ref15">(13,20)</a> Our preliminary work demonstrated that <b>ABT</b>, a mechanism-based pan-CYP inhibitor,<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(26,27)</a> displayed significant time-dependent inhibition of CYP4Z1. Thus, <b>ABT</b> appeared to offer a promising bioactivatable pharmacophore, and our data agreed with prior research, which indicated that <b>ABT</b> effectively inactivates AA-oxidizing CYPs.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(28)</a> Additionally, there has already been considerable precedent for exploration of <b>ABT</b> analogues, as prior studies have characterized <i>N</i>-benzyl-aminobenzotriazole, <i>N</i>-α-methylbenzyl-aminobenzotriazole, and <i>N</i>-α-ethylbenzyl-aminobenzotriazole as isozyme-selective MBIs of rabbit, guinea pig, and rat CYPs.<a onclick="showRef(event, 'ref24 ref25 ref26 ref53'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref53">(29−32)</a></div><div class="NLM_p">CYP4B1 is the only CYP4 member that has been crystallized;<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(33)</a> therefore, structural inferences that can be made about CYP4Z1 are limited. However, catalytic work with CYP4Z1 provided a useful basis for simple inhibitor design. Unlike most other CYP4 enzymes, CYP4Z1 catalyzes oxidations internally rather than at the ω-terminus.<a onclick="showRef(event, 'ref8 ref15 ref54'); return false;" href="javascript:void(0);" class="ref ref8 ref15 ref54">(13,20,34)</a> In addition to AA metabolism, the internal preference for oxidation is evident for the substrates, lauric acid (LA) and Luc-BE (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Therefore, we made simple chemical modifications to the <b>ABT</b> scaffold to generate CYP4Z1 inhibitors by adding carbon tails to <b>ABT</b> to incorporate structural elements of the known CYP4Z1 substrate, LA. Ideally, we wanted to take advantage of the relatively high mid-chain oxidation rate of LA to 8-hydroxy-LA (<i>V</i><sub>max</sub> = ∼40 min<sup>–1</sup>) in the hope that this would translate to CYP4Z1-mediated bioactivation of <b>ABT</b> analogues. The position of the exocyclic nitrogen that was expected to undergo P450-mediated activation in these new N-alkylated <b>ABT</b> congeners approximates the internal site of CYP4Z1-dependent metabolism of LA. Moreover, as illustrated by the docking of LA to the recently published CYP4Z1 homology model,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(13)</a> hydrophilic residues Asn381 and Ser383 are positioned to potentially engage with the free acid moiety of fatty acid substrates. This hypothesis was tested by evaluating <b>ABT</b> analogue inhibitors, which contained a terminal carboxylic acid group alongside inhibitors where that moiety had been removed.</div><div class="NLM_p">Neutral-chain analogues were prepared from readily available aliphatic aldehyde building blocks using a methodology based on that previously used in the synthesis of <i>N</i>-benzyl-aminobenzotriazole.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(29)</a> Acid-containing analogues were synthesized using terminal acid, primary alcohol starting materials and employing pyridinium chlorochromate (PCC)-mediated oxidation to generate the aldehyde moiety. Condensation of the aldehyde with <b>ABT</b> formed an imine, which was then reduced to the target compounds (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>ABT</b> Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Celite, PCC, CH<sub>2</sub>Cl<sub>2</sub>, room temperature (RT), 1 h, 21–25% (analogues <b>6</b>–<b>9</b>); (b) acetic acid, RT, 18 h, 43–89%; and (c) NaBH<sub>4</sub>, CH<sub>2</sub>Cl<sub>2</sub>, MeOH, RT, 2 h, 56–84%</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Evaluation of Time-Dependent Inhibition (TDI) of CYP4Z1 by <b>ABT</b> Analogues</h3><div class="NLM_p">Representative dose–response curves are provided in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, which shows the potency comparison between <b>ABT</b> and inhibitor <b>7</b> (all dose–response curves are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S2</a>, Supporting Information). IC<sub>50</sub> values for all <b>ABT</b> analogues are shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. <b>ABT</b> itself only weakly inhibited CYP4Z1 without a prior preincubation with NADPH. However, the addition of aliphatic tails (compounds <b>1</b>–<b>5</b>) dramatically increased affinity for CYP4Z1 over that of <b>ABT</b>, exemplified by a 50-fold decrease in the reversible IC<sub>50</sub> value from 154 to 2.9 μM for inhibitor <b>2</b>. While <b>ABT</b> itself showed a potent time-dependent effect for inhibition of CYP4Z1 (i.e., a 30 min preincubation of NADPH with CYP4Z1 produced a shifted-IC<sub>50</sub> of 12 μM), inhibitors <b>2</b>–<b>9</b> exhibited much lower shifted-IC<sub>50</sub> values that indicated up to 120-fold higher potency compared to <b>ABT</b>.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Time-dependent inhibition of CYP4Z1-mediated Luc-BE O-debenzylation by <b>ABT</b> (red) and inhibitor <b>7</b> (blue). Shown is the percent activity remaining versus inhibitor concentration, with (dashed lines) and without (solid lines) the presence of NADPH in a preincubation step. The <b>ABT</b> analogue <b>7</b> exhibited gains in the affinity and rate of enzyme inactivation for CYP4Z1 compared to the parent compound. IC<sub>50</sub> values are reported in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Shown is the average ± SD from three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> and Shifted-IC<sub>50</sub> Values for the Inhibition of CYP4Z1-Mediated Luc-BE O-Debenzylation by <b>ABT</b> and Its Analogues</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">inhibitor</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> ± SD (μM)</th><th class="colsep0 rowsep0" align="center" char="±">shifted-IC<sub>50</sub> ± SD (μM)</th><th class="colsep0 rowsep0" align="center" char=".">shift ratio<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ABT</b></td><td class="colsep0 rowsep0" align="char" char="±">154 ± 23</td><td class="colsep0 rowsep0" align="char" char="±">12 ± 0.8</td><td class="colsep0 rowsep0" align="char" char=".">13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char="±">19 ± 2.6</td><td class="colsep0 rowsep0" align="char" char="±">0.7 ± 0.06</td><td class="colsep0 rowsep0" align="char" char=".">27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char="±">2.9 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">0.1 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char="±">4.9 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char="±">12 ± 16</td><td class="colsep0 rowsep0" align="char" char="±">0.5 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char="±">46 ± 10</td><td class="colsep0 rowsep0" align="char" char="±">1.4 ± 0.4</td><td class="colsep0 rowsep0" align="char" char=".">32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char="±">41 ± 2.4</td><td class="colsep0 rowsep0" align="char" char="±">1.7 ± 0.06</td><td class="colsep0 rowsep0" align="char" char=".">24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char="±">5.9 ± 0.9</td><td class="colsep0 rowsep0" align="char" char="±">0.2 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char="±">4.5 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">0.2 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char="±">7.2 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">0.3 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">24</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub>/shifted-IC<sub>50</sub>.</p></div></div></div><div class="NLM_p last">Analysis of the terminal-acid-containing analogues showed similar gains in affinity over <b>ABT</b> itself for CYP4Z1 inhibition. The most potent congeners, <b>7</b> and <b>8</b> (with 8 and 10 carbon tails, respectively), exhibited shifted-IC<sub>50</sub> values of 200 nM, i.e., a 60-fold increase in potency over <b>ABT</b>. Interestingly, the addition of the acid moiety did not have a dramatic effect on CYP4Z1 inhibition potency (neither affinity nor inactivation) compared to the neutral-chain aliphatic variants. However, the benefits of adding a terminal acid group became apparent as counterscreening proceeded as described below. Finally, despite changes in apparent potencies for reversible and time-dependent inhibition across the entire analogue series, the shift ratios (IC<sub>50</sub>/shifted-IC<sub>50</sub>) varied only from 13 to 30 (omission of <b>3</b> narrowed this range to 23–30), which seems consistent with all inhibitors inactivating CYP4Z1 via a common mechanism.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Counterscreening against CYP4 Family Enzymes</h3><div class="NLM_p">Due to the complex nature of the role that fatty acid metabolites play in signaling in both healthy and disease states,<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(9,10)</a> nonspecific CYP4Z1 inhibitors such as <b>ABT</b><a onclick="showRef(event, 'ref15 ref23'); return false;" href="javascript:void(0);" class="ref ref15 ref23">(20,28)</a> and HET0016<a onclick="showRef(event, 'ref3 ref8 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref3 ref8 ref28 ref29">(8,13,35,36)</a> do not provide adequately discriminatory molecular tools with which to assess the physiological role of CYP4Z1. Consequently, we next evaluated the selectivity of the new <b>ABT</b> analogues against recombinantly expressed, and commercially available, CYP4 enzymes: CYP4A11, -4F2, -4F3a, -4F3b, and -4F12. As it is not commercially available, but was still of interest to us, CYP4F8 was expressed in HepG2 cells and membranes were prepared identically to that for CYP4Z1. Pro-luciferin probe substrates at concentrations ≤<i>K</i><sub>m</sub> (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>) were used for all CYP4 isozymes to determine inhibition, including CYP4F8 (as elucidated from in-house testing, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S8</a>), an isozyme for which limited substrate information has been published. A TDI assay at a single, moderate, concentration of 30 μM was utilized to assess whether any analogues significantly inhibited the off-target enzymes. A 30 min enzyme/NADPH preincubation step was used to assess time-dependent effects (akin to a full IC<sub>50</sub>-shift assay), which were modest for all enzymes and analogues tested (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Initial screening against off-target CYP4 isozymes was performed with inhibitors <b>2</b>–<b>9</b> (inhibitor <b>1</b> was excluded due to its low potency). The alkyl-modified analogues <b>2</b>–<b>5</b> inhibited CYP4F2, CYP4F3a, CYP4F3b, and CYP4F12 substantially, whereas the acid-modified analogues containing 10 and 12 carbon tails, <b>8</b> and <b>9</b>, respectively, substantially inhibited CYP4A11, CYP4F8, and CYP4F12. Importantly, however, two acid analogues <b>6</b> and <b>7</b> exhibited negligible or very low activity toward the CYP4 enzymes tested and were therefore selected for further study.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Time-dependent inhibition of CYP4A11-mediated Luc-4A O-demethylation; CYP4F2-, CYP4F3a-, and CYP4F3b-mediated Luc-4F2/3 O-(4-methylthio)-debenzylation; and CYP4F8- and CYP4F12-mediated Luc-BE O-debenzylation by <b>ABT</b> analogues screened at 30 μM. The pan-CYP4 inhibitor HET0016 was dosed at 0.5 μM as an inhibition control. All probe substrates were used at concentrations ≤<i>K</i><sub>m</sub>. All enzymes were recombinantly expressed in insect cells (Supersomes), except for CYP4F8, which was recombinantly expressed in HepG2 cells. The black and gray bars show the percent activity remaining when the inhibitor was preincubated for 30 min with enzyme plus and minus NADPH, respectively. Shown is the average ± SD from three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Evaluation of TDI for Inhibitors <b>6</b> and <b>7</b> against Off-Target CYP Isozymes</h3><div class="NLM_p">Full IC<sub>50</sub>-shift assays (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figures S3 and S4</a>) were performed with <b>6</b> and <b>7</b> to thoroughly assess off-target inhibition toward the other recombinantly expressed CYP4 enzymes (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). As determined from the specificity ratios (off-target CYP IC<sub>50</sub>/CYP4Z1 IC<sub>50</sub>), both <b>6</b> and <b>7</b> were comparatively poor reversible inhibitors of these isozymes compared to CYP4Z1. Relatively small shift ratios (∼1–3) were obtained for both inhibitors against the six CYP4 isozymes tested, indicating low propensity of <b>6</b> and <b>7</b> for enzyme-mediated bioactivation. Of note, any shift values <1 (indicative of inhibitors with lowered potency after a plus NADPH preincubation) likely result from significant metabolism of the inhibitor by the CYP to nonbioreactive species. The contrasting reactivities of <b>6</b> and <b>7</b> between these off-target isozymes and CYP4Z1 are readily apparent from the resultant TDI specificity ratios (off-target CYP shifted-IC<sub>50</sub>/CYP4Z1 shifted-IC<sub>50</sub>) that ranged from 137 to 2235. Therefore, CYP4Z1 selectivity becomes greatly enhanced as the result of TDI for these two inhibitors.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. IC<sub>50</sub> and Shifted-IC<sub>50</sub> Values for Inhibition of CYP4-Mediated Pro-luciferin Metabolism by Inhibitors <b>6</b> and <b>7</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">CYP</th><th class="colsep0 rowsep0" align="center" char=".">inhibitor</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> ± SD (μM)</th><th class="colsep0 rowsep0" align="center" char="±">shifted-IC<sub>50</sub> ± SD (μM)</th><th class="colsep0 rowsep0" align="center" char=".">shift ratio<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">specificity ratio<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">TDI specificity ratio<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">4A11</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="char" char="±">1330 ± 134</td><td class="colsep0 rowsep0" align="char" char="±">455 ± 20</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">268</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="char" char="±">282 ± 15</td><td class="colsep0 rowsep0" align="char" char="±">179 ± 51</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">48</td><td class="colsep0 rowsep0" align="char" char=".">895</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">4F2</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="char" char="±">1130 ± 114</td><td class="colsep0 rowsep0" align="char" char="±">647 ± 83</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">381</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="char" char="±">212 ± 15</td><td class="colsep0 rowsep0" align="char" char="±">221 ± 36</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">1105</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">4F3a</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="char" char="±">1240 ± 40</td><td class="colsep0 rowsep0" align="char" char="±">1370 ± 198</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">808</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="char" char="±">264 ± 31</td><td class="colsep0 rowsep0" align="char" char="±">447 ± 75</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">45</td><td class="colsep0 rowsep0" align="char" char=".">2235</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">4F3b</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="char" char="±">1520 ± 191</td><td class="colsep0 rowsep0" align="char" char="±">1690 ± 185</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">991</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="char" char="±">187 ± 6</td><td class="colsep0 rowsep0" align="char" char="±">213 ± 46</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">1065</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">4F8</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="char" char="±">1180 ± 137</td><td class="colsep0 rowsep0" align="char" char="±">778 ± 53</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">458</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="char" char="±">167 ± 23</td><td class="colsep0 rowsep0" align="char" char="±">57 ± 13</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">285</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">4F12</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="char" char="±">380 ± 39</td><td class="colsep0 rowsep0" align="char" char="±">233 ± 11</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">137</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="char" char="±">91 ± 17</td><td class="colsep0 rowsep0" align="char" char="±">46 ± 8</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">230</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub>/shifted-IC<sub>50</sub>.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Off-target CYP IC<sub>50</sub>/CYP4Z1 IC<sub>50</sub>.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Off-target CYP shifted IC<sub>50</sub>/CYP4Z1 shifted-IC<sub>50</sub>.</p></div></div></div><div class="NLM_p">Overall, inhibitor <b>7</b> possessed a higher degree of selectivity for CYP4Z1 than <b>6</b>. Additionally, although <b>7</b> showed relatively similar affinity across the off-target CYP4 isozymes, only CYP4F8 and CYP4F12 were well above the significance threshold for TDI (i.e., shift ratio ≥1.5). This indicates that these two enzymes are more likely to bind <b>7</b> in a position that leads directly to bioactivation. We suggest that this is the result of structural similarities with CYP4Z1 that are not present in the other CYP4 enzymes tested, namely, a lack of the active site glutamate residue necessary to form a covalent attachment with the heme.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(13)</a> A recent publication on the crystal structure of octane-bound CYP4B1, an ω-hydroxylase with a covalently attached heme, showed that this structural element is an important determinant of the enzyme’s (and most likely other similar CYP4 isozymes) regiospecificity for oxidation at the terminal position of alkyl-chain containing substrates.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(33)</a> Indeed, both CYP4F8 and CYP4F12 enzymes, neither of which possesses a covalently attached heme, primarily produce an internal ω-3-hydroxylated AA metabolite, 18-HETE,<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(37,38)</a> and thus seem to more closely mimic CYP4Z1 than the 20-HETE-producing CYP4 family members. However, although the shift ratios pass the threshold of significance for these two isozymes, they are ∼10-fold lower than the shift ratio observed for <b>7</b> and CYP4Z1. This results in CYP4F8 and CYP4F12 being inhibited 285-fold and 230-fold less, respectively, than CYP4Z1 after a preincubation with a cofactor. We speculate that this observation reflects the fact that 18-HETE is still three to four C–C bond lengths away from the major site of CYP4Z1-mediated AA metabolism and thus bioactivation of <b>7</b> is limited for CYP4F8 and CYP4F12 because the exocyclic nitrogen for this analogue is not positioned optimally for these two isozymes.</div><div class="NLM_p">Full IC<sub>50</sub>-shift assays were also performed with <b>6</b> and <b>7</b> to evaluate off-target inhibition toward hepatic CYP isozymes (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figures S5 and S6</a>), chosen based on the seven Food and Drug Administration (FDA)-recommended CYPs for drug–drug interaction (DDI) testing.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(39)</a> Human liver microsomes (HLMs) served as the enzyme source, and a cocktail method was employed to evaluate inhibition of CYP1A2, -2B6, -2C8, -2C9, -2C19, -2D6, and -3A4/5 concurrently (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Of note, in-house testing for inhibition, using isozyme-specific inhibitor probes, of metabolism by the seven hepatic CYPs in the cocktail assays yielded data in agreement with that reported in the literature<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(40)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S9</a>). Similar to the results obtained with the CYP4 isozymes, inhibitors <b>6</b> and <b>7</b> did not potently inhibit off-target hepatic CYPs, either reversibly or irreversibly, with specificity and TDI specificity ratios spanning 23–140 and 134–1650, respectively. For both inhibitors, the largest shift ratios were observed for CYP1A2, at 12.2 and 11.2 for <b>6</b> and <b>7</b>, respectively. The large degree of bioactivation observed may be explained by CYP1A2, like CYP4Z1, having a propensity itself for internal oxidation of PUFAs. The 14,15-EET synthase activity for CYP1A2 has been reported to be ∼0.54 pmol/pmol CYP1A2/min (at 50 μM AA)<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(41,42)</a> and is in a similar range to that observed for CYP4Z1 of 1.9 pmol/pmol CYP4Z1/min (at 75 μM AA).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(13)</a> Furthermore, all seven of the hepatic CYP isozymes that we analyzed have been shown previously to metabolize AA to 14,15-EET, albeit with a large range of efficiencies and regiospecificities.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(42)</a> Although the role of most AA-metabolizing CYPs in breast cancer progression is largely unknown, CYP3A4 expression has been specifically linked to decreased survival and is required for growth in several breast cancer cell lines.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(43)</a> Therefore, the low off-target inhibition of the generally liver-localized CYPs studied herein was a desired trait not only for broad selectivity purposes, but also for discriminatory probing of the role of CYP4Z1-dependent epoxygenase activity in a breast cancer setting. Overall, both inhibitors <b>6</b> and <b>7</b> demonstrated their lack of activity toward off-target enzymes primarily through low rates of bioactivation that resulted in modest time-dependent inhibition and high TDI specificity ratios. Due to its combined properties of high potency and excellent selectivity for CYP4Z1, inhibitor <b>7</b> was chosen for in-depth characterization as an MBI.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. IC<sub>50</sub> and Shifted-IC<sub>50</sub> Values for Inhibition of Hepatic CYP-Mediated Substrate Cocktail Metabolism by Inhibitors <b>6</b> and <b>7</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">CYP</th><th class="colsep0 rowsep0" align="center" char=".">inhibitor</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> ± SD (μM)</th><th class="colsep0 rowsep0" align="center" char="±">shifted-IC<sub>50</sub> ± SD (μM)</th><th class="colsep0 rowsep0" align="center" char=".">shift ratio<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">specificity ratio<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">TDI specificity ratio<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">1A2</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="left">2790 ± 272</td><td class="colsep0 rowsep0" align="char" char="±">228 ± 18</td><td class="colsep0 rowsep0" align="char" char=".">12.2</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">134</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="left">614 ± 117</td><td class="colsep0 rowsep0" align="char" char="±">55 ± 8</td><td class="colsep0 rowsep0" align="char" char=".">11.2</td><td class="colsep0 rowsep0" align="char" char=".">104</td><td class="colsep0 rowsep0" align="char" char=".">275</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">2B6</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="left">923 ± 152</td><td class="colsep0 rowsep0" align="char" char="±">258 ± 28</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">152</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="left">257 ± 68</td><td class="colsep0 rowsep0" align="char" char="±">62 ± 21</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="char" char=".">44</td><td class="colsep0 rowsep0" align="char" char=".">310</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">2C8</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="left">2310 ± 71</td><td class="colsep0 rowsep0" align="char" char="±">954 ± 31</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">561</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="left">306 ± 44</td><td class="colsep0 rowsep0" align="char" char="±">84 ± 16</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="char" char=".">420</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">2C9</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="char" char="±">1940 ± 56</td><td class="colsep0 rowsep0" align="char" char=".">>1.5</td><td class="colsep0 rowsep0" align="char" char=".">>73</td><td class="colsep0 rowsep0" align="char" char=".">1141</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="left">343 ± 88</td><td class="colsep0 rowsep0" align="char" char="±">330 ± 41</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="char" char=".">1650</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">2C19</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="char" char="±">1640 ± 189</td><td class="colsep0 rowsep0" align="char" char=".">>1.8</td><td class="colsep0 rowsep0" align="char" char=".">>73</td><td class="colsep0 rowsep0" align="char" char=".">965</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="left">759 ± 138</td><td class="colsep0 rowsep0" align="char" char="±">246 ± 62</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">129</td><td class="colsep0 rowsep0" align="char" char=".">1230</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">2D6</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="left">2850 ± 404</td><td class="colsep0 rowsep0" align="char" char="±">1710 ± 107</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="char" char=".">1006</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="left">826 ± 177</td><td class="colsep0 rowsep0" align="char" char="±">299 ± 86</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">140</td><td class="colsep0 rowsep0" align="char" char=".">1495</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">3A4/5</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="left">1310 ± 172</td><td class="colsep0 rowsep0" align="char" char="±">766 ± 29</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">451</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="left">244 ± 8</td><td class="colsep0 rowsep0" align="char" char="±">128 ± 10</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">41</td><td class="colsep0 rowsep0" align="char" char=".">640</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub>/shifted-IC<sub>50</sub>.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Off-target CYP IC<sub>50</sub>/CYP4Z1 IC<sub>50</sub>.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Off-target CYP shifted-IC<sub>50</sub>/CYP4Z1 shifted-IC<sub>50</sub>.</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Characterization of Mechanism-Based Inactivation of CYP4Z1 by <b>7</b></h3><div class="NLM_p">Mechanism-based inactivation is typically viewed as a characteristic to be designed away from in drug development, as the enzymes affected by this process are frequently off-target CYPs. When this occurs, DDIs are common liabilities as these off-target enzymes are rendered catalytically incompetent and must then be newly biosynthesized. However, numerous irreversible inhibitors (including MBIs) have emerged recently as therapeutics and have the benefits of increased biochemical efficiency, a longer duration of action, efficient inactivation of the target, and the potential to avoid some resistance mechanisms.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(44)</a> Examples of CYP MBI drugs include some aromatase (CYP19A) inhibitors that have shown efficacy in the clinic and are currently used for breast cancer treatment; an example is the steroidal mimetic exemestane.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(45)</a> Evaluating the characteristics of an MBI (not simply establishing TDI) enables insights into inhibitor structure–activity relationships and mechanisms of action, guides future analogue work, and provides a better understanding of the biochemistry of the target CYP. Therefore, the action of inhibitor <b>7</b> toward CYP4Z1 was considered in light of the Silverman criteria for an MBI.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(46)</a></div><div class="NLM_p">As seen in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, <b>7</b> displayed concentration- and time-dependent loss of enzyme activity, as first indicated by the single time-point assays described above. When the values for <i>k</i><sub>obs</sub> were plotted against concentrations of <b>7</b>, a <i>K</i><sub>I</sub> = 2.2 μM and a <i>k</i><sub>inact</sub> = 0.15 min<sup>–1</sup> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) were determined; the observed maximal rate of inactivation indicates saturation kinetics. Comparison with exemestane, where a <i>k</i><sub>inact</sub> for aromatase inhibition of 0.05 min<sup>–1</sup> has been determined, emphasizes how rapidly <b>7</b> inactivates CYP4Z1, although it must be noted that exemestane inactivation of aromatase is highly efficient due to its tight-binding affinity (<i>K</i><sub>I</sub> = 26 nM) for the enzyme.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(47)</a> To show irreversibility of mechanism-based inactivation, we assessed the fraction of activity remaining, from CYP4Z1 treated with <b>7</b>, before and after overnight dialysis to remove the excess inhibitor (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). No increased activity was observed after dialysis, indicating irreversible enzyme destruction. Multiple experiments confirmed the necessity of a catalytic step for the time-dependent inhibition of CYP4Z1 by <b>7</b>. This is directly shown in the dialysis experiment, where the lack of a preincubation step with the NADPH cofactor prior to the substrate turnover reaction resulted in no inactivation of the enzyme. The addition of hard or soft nucleophiles, at commonly used concentrations for reactive intermediate trapping experiments,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(48)</a> such as glutathione (GSH), <i>N</i>-acetyl-lysine (NAL), methoxyamine (MeONH<sub>2</sub>), <i>N</i>-acetyl-cysteine (NAC), semicarbazide (semicarb.), and potassium cyanide (KCN), did not abrogate inactivation of CYP4Z1 by <b>7</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Additionally, the presence of the reactive oxygen species-scavenging enzymes, catalase (CAT) and superoxide dismutase (SOD), did not rescue CYP4Z1 activity, eliminating the possibility of nonspecific, and nonenzymatic, oxidation reactions causing inactivation of CYP4Z1. These data indicate that inactivation of CYP4Z1 occurs prior to release of bioactivated <b>7</b> from the active site.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Characterization of mechanism-based inactivation of CYP4Z1 by <b>7</b>. Time- and concentration-dependent loss of CYP4Z1-mediated Luc-BE O-debenzylation activity from treatment with <b>7</b> (<i>K</i><sub>I</sub> = 2.2 μM, <i>k</i><sub>inact</sub> = 0.15 min<sup>–1</sup>) (A, B). The percent of CYP4Z1 activity remaining when inhibitor <b>7</b> was preincubated with enzyme plus (black bars) and minus (gray bars) NADPH was assessed both prior to (pre) and after (post) overnight dialysis. The TDI of CYP4Z1 was not affected by this dialysis step (C). The percent of CYP4Z1 activity remaining when inhibitor <b>7</b> was preincubated with enzyme plus NADPH was assessed with the following nucleophilic additives: glutathione (GSH), <i>N</i>-acetyl-lysine (NAL), methoxyamine (MeONH<sub>2</sub>), <i>N</i>-acetyl-cysteine (NAC), and semicarbazide (semicarb.) at 5 mM and potassium cyanide (KCN) at 1 mM. Inclusion of these trapping agents did not significantly abrogate the TDI of CYP4Z1 by <b>7</b>. Similarly, the addition of the reactive oxygen species-scavenging enzymes, catalase (CAT) and superoxide dismutase (SOD), did not affect the TDI of CYP4Z1 by <b>7</b> (D). In both (C) and (D), the minus NADPH sample controls exhibit cofactor dependency for the bioactivation reaction. A plot of the percent of CYP4Z1 activity, when inhibitor <b>7</b> was preincubated with enzyme plus NADPH, versus the concentration ratio of <b>7</b>/CYP4Z1 was used to determine the number of inactivator molecules converted and released as a product relative to each inactivation event. Extrapolation of the initial linear portion of the plot to the <i>x</i>-intercept (inset) shows a turnover number of 15, resulting in a partition ratio of ∼14 (E). For all assays, data shown is the average ± SD from three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Next, the partition ratio for inactivation of CYP4Z1 by <b>7</b> was determined (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). As a measure of efficiency, the partition ratio is the number of latent inactivator molecules converted and released as a product relative to each catalytic cycle resulting in enzyme inactivation.<a onclick="showRef(event, 'ref39 ref42'); return false;" href="javascript:void(0);" class="ref ref39 ref42">(46,49)</a> Partition ratios for MBIs of CYPs can span a large range, with values reported from 1 to more than 5000.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(50)</a> Therefore, the partition ratio of 14 for <b>7</b> lies toward the efficient end of this spectrum.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Inhibition of CYP4Z1 AA Metabolism in T47D Cells by <b>7</b></h3><div class="NLM_p">We next evaluated the inhibition of CYP4Z1-dependent metabolism of AA in breast-cancer-derived T47D cells.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(51)</a> Although a low level of CYP4Z1 mRNA is detectable in breast cancer cell lines, which may be induced ∼15-fold from dexamethasone treatment,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(52)</a> immortalization appears to eliminate the expression of endogenous CYP4Z1.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(8)</a> Therefore, T47D cells were modified to constitutively express CYP4Z1 (T47D-CYP4Z1) using a recombinant lentivirus. Western blot analysis showed ample CYP4Z1 expression in the CYP4Z1-T47D cells and no observable endogenous CYP4Z1 expression in the vector-T47D cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S1C</a>). Concentrations of free AA in human plasma span a wide range, from 2.7 to 50 μM.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(53,54)</a> Therefore, a 50 μM dose of AA was used to reflect this upper boundary. The previously characterized CYP4Z1 metabolites of AA, 19-HETE and 14,15-EET, along with the metabolite commonly generated by other CYP4 isozymes, 20-HETE, were monitored.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(13)</a> Also included in the AA metabolite profile was the degradation product 14,15-dihydroxy eicosatrienoic acid (14,15-DiHET), as this could be generated by 14,15-EET hydrolysis, both nonenzymatically, in the tissue culture environment, and enzymatically through the action of epoxide hydrolases potentially present in T47D cells.<a onclick="showRef(event, 'ref35 ref47'); return false;" href="javascript:void(0);" class="ref ref35 ref47">(42,55)</a></div><div class="NLM_p">The T47D-CYP4Z1 cells were exposed to <b>7</b>, incorporating a 24 h inhibitor preincubation step prior to the introduction of the substrate, to assess TDI. At 3 μM, potent inhibition of AA metabolism was observed (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Comparing peak area ratios (PARs) of metabolites divided by their respective deuterated internal standards showed that production of 14,15-EET, 19-HETE, and 14,15-DiHET was inhibited 83, 86, and 80%, respectively. When the inhibitor and the substrate were administered together to T47D-CYP4Z1 cells, without a preincubation step, 3 μM <b>7</b> produced lower levels of inhibition against production of 14,15-EET, 19-HETE, and 14,15-DiHET (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), as expected. When dosed at 1 μM, a comparable time-dependent effect was observed, with a ∼2-fold decrease in potency. Similar results were also attained when investigating AA metabolism in CYP4Z1-HepG2 membranes, where a time-dependent, ∼90% loss in CYP4Z1 activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S10</a>) was observed with inhibitor <b>7</b> utilized at 3 μM.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of CYP4Z1-mediated AA metabolism in T47D whole cells by <b>7</b>. Liquid chromatography–tandem mass spectrometry (LC–MS/MS) chromatograms showing the AA metabolite profile from CYP4Z1-T47D cells treated with dimethyl sulfoxide (DMSO) vehicle (bottom trace) or inhibitor <b>7</b> (middle trace) and vector-T47D cells treated with the vehicle (top trace). Cells were treated with 3 μM <b>7</b>, or the vehicle, and underwent a 24 h preincubation prior to the addition of 50 μM AA, which was followed by a 2 h incubation for substrate metabolism. This resulted in 14,15-EET, 19-HETE, and 14,15-DiHET metabolite formation being inhibited 83, 86, and 80%, respectively, as compared to vehicle-treated CYP4Z1-T47D cells. Vector-T47D cells dosed with the vehicle and 50 μM AA showed a minimal amount of background AA metabolism.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Inhibition of AA Metabolite Formation in CYP4Z1-T47D Cells by <b>7</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">14,15-EET</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">19-HETE</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">14,15-DiHET</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">[<b>7</b>] (μM)</th><th class="colsep0 rowsep0" align="center" char=".">minus<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a> (%)</th><th class="colsep0 rowsep0" align="center" char=".">plus<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (%)</th><th class="colsep0 rowsep0" align="center" char=".">minus (%)</th><th class="colsep0 rowsep0" align="center" char=".">plus (%)</th><th class="colsep0 rowsep0" align="center" char=".">minus (%)</th><th class="colsep0 rowsep0" align="center" char=".">plus (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">47</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="char" char=".">41</td><td class="colsep0 rowsep0" align="char" char=".">73</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">83</td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="char" char=".">86</td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="char" char=".">80</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Minus: pre-incubation without <b>7</b>.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Plus: 24 h preincubation with <b>7</b>.</p></div></div></div><div class="NLM_p">Finally, we confirmed the preference for CYP4Z1-mediated 14,15-EET production over that of 20-HETE (only a trace amount was detected, slightly above the lower limit of detection) in a mammalian whole-cell setting. Although other internal oxidation products from CYP4Z1 AA metabolism were observed in both the HepG2 membranes and T47D cells (decreasing proportionally with inhibitor and tentatively identified as 15-, 16-, 17-, and 18-HETE), these apparently minor metabolites were not characterized further here.</div><div class="NLM_p last">These data demonstrate the efficacy of inhibitor <b>7</b> in a whole-cell environment and are comparable to the results we obtained for the inhibition of CYP4Z1-mediated Luc-BE O-debenzylation in HepG2 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S11A</a>). Therefore, although not tested directly, potentially lower inhibitor permeability and stability in whole cells appear not to be major issues. Additionally, these results are not confounded by the possible cytotoxicity of inhibitor <b>7</b>, as a 3 μM treatment did not cause noticeable changes in viability 24 h later for any of the cell lines assayed (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S11B</a>). Although we hypothesize <b>7</b> to have antiproliferative effects against breast cancer cells expressing CYP4Z1, the specific conditions required to capture this effect (e.g., temporal, growth factor, and PUFA substrate variables) were not assessed here as it was beyond the scope of this study. To the best of our knowledge, no studies utilizing primary cells have been performed to interrogate CYP4Z1. However, due to its effectiveness in a breast-cancer-derived cell line for inhibiting the generation of the proangiogenic and pro-proliferative metabolite 14,15-EET, we posit that inhibitor <b>7</b> will be an effective molecular tool to probe the druggability of CYP4Z1 as a target for breast cancer therapy in varied in vitro settings.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In summary, we leveraged our knowledge of the product selectivity of CYP4Z1 to generate <b>ABT</b> analogues that are efficient time-dependent inhibitors of the enzyme with substantially increased potency over <b>ABT</b>. The design strategy of incorporating a terminal acid to produce fatty acid mimetics resulted in high selectivity for CYP4Z1 for two analogues, <b>6</b> and <b>7</b>. Further characterization, monitoring isozyme-specific inhibition in a large panel of off-target CYPs, identified <b>7</b> as the inhibitor with the best combination of isozyme selectivity and potency. <b>7</b> was further characterized as a relatively efficient mechanism-based inhibitor (<i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> = 0.07 min<sup>–1</sup> μM<sup>–1</sup>), with demonstrated time-dependent inhibition of CYP4Z1 in transfected breast-cancer-derived whole cells.</div><div class="NLM_p last">Future studies will investigate the mechanism of CYP4Z1 inactivation by <b>7</b> through trapping of reactive metabolite(s) and analysis of heme and apoprotein adducts. These first-generation inhibitors are potentially useful lead compounds for the development of more potent CYP4Z1 inhibitors with appropriate pharmacokinetic features to support in vivo studies and potential clinical applicability.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54227" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54227" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Information for Synthesis</h3><div class="NLM_p last">All solvents and reagents were purchased from commercial sources (Alfa Aesar, Acros Organics, AstaTech, Fisher, Oakwood Chemical, Sigma-Aldrich, and TCI) and used as received. All reactions were monitored by thin-layer chromatography, visualized by either UV (254 nm) or potassium permanganate stain. For isolation of purified compounds, normal phase chromatography was carried out on a Teledyne Isco CombiFlash system (Lincoln, NE). <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded at 25 °C in deuterated chloroform (CDCl<sub>3</sub>) on either a 499.73 MHz Agilent DD2 (Santa Clara, CA) or 499.96 MHz Bruker Avance series (Billerica, MA) spectrometer, respectively. Chemical shifts are reported below relative to the solvent peak in CDCl<sub>3</sub> at 7.26 ppm (<sup>1</sup>H NMR) or tetramethylsilane at 0.00 ppm (<sup>13</sup>C NMR). Coupling constants (<i>J</i>) are noted in hertz (Hz) and peak multiplicities as either a singlet (s), doublet (d), triplet (t), pentet (p), or multiplet (m). All <sup>1</sup>H and <sup>13</sup>C NMR spectra are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Supporting Information</a>. Accurate mass and purity analyses of the synthesized compounds were performed via ultra-performance liquid chromatography (UPLC)–mass spectrometry (MS) on a Waters Acquity UPLC (Milford, MA) coupled in tandem with a Waters Acquity tunable UV detector and a Thermo LTQ-Orbitrap mass spectrometer (San Jose, CA). Purity as assessed by UPLC–UV (254 and 280 nm) and <sup>1</sup>H NMR spectroscopy was ≥95% for all compounds.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Alcohol Oxidation (General Procedure I)</h3><div class="NLM_p last">A modified version of an existing method that has been published was utilized<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(56)</a> and was as follows: a dried and N<sub>2</sub>-purged round-bottom flask containing 1–2 g of Celite was charged with pyridinium chlorochromate (PCC, 3–5 mmol, 1.5 equiv), solvated in ∼50 mL of anhydrous dichloromethane. The appropriate primary alcohol (2–3 mmol), solvated in a minimal amount of anhydrous dichloromethane, was subsequently added to the reaction flask, and the mixture was stirred at room temperature under N<sub>2</sub> for 90 min. Next, anhydrous diethyl ether was added to the slurry and the mixture was filtered through a pad of Celite and rinsed with several portions of diethyl ether, and the solvent was removed from the combined organic fractions under vacuum. The residue was then washed thrice with portions of diethyl ether to harvest the soluble oxo-product, and the combined diethyl ether fractions were once again filtered through Celite and the solvent was removed under vacuum to afford the corresponding aldehyde.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Imine Formation (General Procedure II)</h3><div class="NLM_p last">Modified versions of existing methods that have been published were utilized for both general procedures II and III<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(29)</a> and were as follows: a dried and N<sub>2</sub>-purged round-bottom flask was charged with <b>ABT</b> (0.75 mmol), the appropriate aldehyde (4.5 mmol, 6.0 equiv), and ∼20 mL of glacial acetic acid. For inhibitors <b>6</b>–<b>9</b>, the oxo-product derived from general procedure I (0.5–0.8 mmol) was reacted with 1.5 equiv of <b>ABT</b>. The reaction mixture was stirred at room temperature overnight, and acetic acid was subsequently removed under vacuum. The crude product was purified via flash chromatography using a dichloromethane/ethyl acetate gradient to afford the corresponding imine intermediate (for analogues containing terminal carboxylic acid groups, 2% acetic acid was added to the mobile phase for improved chromatographic separation).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Imine Reduction (General Procedure III)</h3><div class="NLM_p">A dried round-bottom flask was charged with the appropriate imine intermediate (0.3–0.7 mmol), NaBH<sub>4</sub> (∼20 equiv), and ∼40 mL of a 2:1 solution of methanol/dichloromethane. The reaction mixture was stirred at room temperature for 1 h, and then, the solvents were removed under vacuum. The remaining mixture was solvated in a 20:1 solution of dichloromethane/methanol and then washed thrice with H<sub>2</sub>O; for the carboxylic acid-containing analogues, the pH of H<sub>2</sub>O was adjusted with 1 N HCl to ∼2 prior to washing. The organic fraction was dried over Na<sub>2</sub>SO<sub>4</sub> and filtered, and the crude product was purified via flash chromatography using a hexane/ethyl acetate gradient to afford the corresponding inhibitors <b>1</b>–<b>9</b>.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>N</i>-Butyl-1<i>H</i>-1,2,3-benzotriazol-1-amine (<b>1</b>)</h4><div class="NLM_p last">It was synthesized from butanal and <b>ABT</b> according to general procedures II and III, with respective yields of 89 and 80%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.02 (d, <i>J</i> = 8.4 Hz, 1H), 7.61 (d, <i>J</i> = 8.3 Hz, 1H), 7.50 (t, <i>J</i> = 7.9 Hz, 1H), 7.37 (t, <i>J</i> = 7.4 Hz, 1H), 3.42 (t, <i>J</i> = 7.1 Hz, 2H), 1.52 (p, <i>J</i> = 7.0 Hz, 2H), 1.46 (sx, <i>J</i> = 7.5 Hz, 2H), 0.93 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 144.4, 132.3, 127.7, 124.1, 119.9, 109.8, 52.91, 29.91, 19.96, 13.80; high-resolution mass spectrometry (HRMS) (positive electrospray ionization (ESI<sup>+</sup>)) <i>m</i>/<i>z</i> [M + H] calcd (C<sub>10</sub>H<sub>15</sub>N<sub>4</sub>) 191.1291, observed 191.1292, δ ppm 0.5.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>N</i>-Hexyl-1<i>H</i>-1,2,3-benzotriazol-1-amine (<b>2</b>)</h4><div class="NLM_p last">It was synthesized from hexanal and <b>ABT</b> according to general procedures II and III, with respective yields of 80 and 84%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.02 (d, <i>J</i> = 8.4 Hz, 1H), 7.61 (d, <i>J</i> = 8.3 Hz, 1H), 7.50 (t, <i>J</i> = 7.5 Hz, 1H), 7.37 (t, <i>J</i> = 7.2 Hz, 1H), 3.41 (t, <i>J</i> = 7.2 Hz, 2H), 1.53 (p, <i>J</i> = 7.3 Hz, 2H), 1.42 (p, <i>J</i> = 8.2 Hz, 2H), 1.29 (m, 4H), 0.88 (t, <i>J</i> = 6.9 Hz, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 144.4, 132.3, 127.6, 124.1, 119.9, 109.8, 53.22, 31.53, 27.84, 26.45, 22.52, 13.97; HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> [M + H] calcd (C<sub>12</sub>H<sub>19</sub>N<sub>4</sub>) 219.1604, observed 219.1606, δ ppm 0.9.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i>-Octyl-1<i>H</i>-1,2,3-benzotriazol-1-amine (<b>3</b>)</h4><div class="NLM_p last">It was synthesized from octanal and <b>ABT</b> according to general procedures II and III, with respective yields of 77 and 70%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.00 (d, <i>J</i> = 8.4 Hz, 1H), 7.60 (d, <i>J</i> = 8.3 Hz, 1H), 7.48 (t, <i>J</i> = 7.3 Hz, 1H), 7.35 (t, <i>J</i> = 7.2 Hz, 1H), 3.40 (t, <i>J</i> = 7.2 Hz, 2H), 1.52 (p, <i>J</i> = 7.3 Hz, 2H), 1.40 (p, <i>J</i> = 7.6 Hz, 2H), 1.26 (m, 8H), 0.87 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 144.4, 132.3, 127.7, 124.1, 119.9, 109.8, 53.27, 31.77, 29.31, 29.17, 27.88, 26.79, 22.62, 14.06; HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> [M + H] calcd (C<sub>14</sub>H<sub>23</sub>N<sub>4</sub>) 247.1917, observed 247.1922, δ ppm 2.0.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-Decyl-1<i>H</i>-1,2,3-benzotriazol-1-amine (<b>4</b>)</h4><div class="NLM_p last">It was synthesized from decanal and <b>ABT</b> according to general procedures II and III, with respective yields of 67 and 70%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.01 (d, <i>J</i> = 8.0 Hz, 1H), 7.60 (d, <i>J</i> = 8.3 Hz, 1H), 7.49 (t, <i>J</i> = 7.2 Hz, 1H), 7.36 (t, <i>J</i> = 8.1 Hz, 1H), 3.40 (t, <i>J</i> = 6.8 Hz, 2H), 1.52 (p, <i>J</i> = 7.1 Hz, 2H), 1.41 (p, <i>J</i> = 6.9 Hz, 2H), 1.25 (m, 12H), 0.87 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 144.4, 132.3, 127.6, 124.1, 119.9, 109.8, 53.24, 31.88, 29.52, 29.35, 29.28, 27.89, 26.79, 22.67, 14.10; HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> [M + H] calcd (C<sub>16</sub>H<sub>27</sub>N<sub>4</sub>) 275.2230, observed 275.2234, δ ppm 1.5.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-Dodecyl-1<i>H</i>-1,2,3-benzotriazol-1-amine (<b>5</b>)</h4><div class="NLM_p last">It was synthesized from dodecanal and <b>ABT</b> according to general procedures II and III, with respective yields of 65 and 74%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.02 (d, <i>J</i> = 8.4 Hz, 1H), 7.61 (d, <i>J</i> = 8.3 Hz, 1H), 7.50 (t, <i>J</i> = 7.7 Hz, 1H), 7.37 (t, <i>J</i> = 7.3 Hz, 1H), 3.41 (t, <i>J</i> = 7.2 Hz, 2H), 1.53 (p, <i>J</i> = 7.4 Hz, 2H), 1.41 (p, <i>J</i> = 7.6 Hz, 2H), 1.25 (m, 16H), 0.88 (t, <i>J</i> = 6.9 Hz, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 144.4, 132.3, 127.7, 124.1, 119.9, 109.8, 53.28, 31.92, 29.63, 29.56, 29.52, 29.35, 27.89, 26.79, 22.69, 14.11; HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> [M + H] calcd (C<sub>18</sub>H<sub>31</sub>N<sub>4</sub>) 303.2543, observed 303.2546, δ ppm 1.0.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 6-[(1<i>H</i>-1,2,3-Benzotriazol-1-yl)amino]hexanoic Acid (<b>6</b>)</h4><div class="NLM_p last">It was synthesized from 6-hydroxyhexanoic acid and <b>ABT</b> according to general procedures I, II, and III, with respective yields of 23, 50, and 76%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.01 (d, <i>J</i> = 8.4 Hz, 1H), 7.60 (d, <i>J</i> = 8.3 Hz, 1H), 7.49 (t, <i>J</i> = 7.8 Hz, 1H), 7.36 (t, <i>J</i> = 7.9 Hz, 1H), 3.42 (t, <i>J</i> = 6.8 Hz, 2H), 2.36 (t, <i>J</i> = 7.3 Hz, 2H), 1.66 (p, <i>J</i> = 7.4 Hz, 2H), 1.52 (m, 4H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 178.7, 144.3, 132.3, 127.9, 124.3, 119.9, 109.8, 52.87, 33.74, 27.56, 26.19, 24.34; HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> [M + H] calcd (C<sub>12</sub>H<sub>17</sub>N<sub>4</sub>O<sub>2</sub>) 249.1346, observed 249.1357, δ ppm 4.4.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 8-[(1<i>H</i>-1,2,3-Benzotriazol-1-yl)amino]octanoic Acid (<b>7</b>)</h4><div class="NLM_p last">It was synthesized from 8-hydroxyoctanoic acid and <b>ABT</b> according to general procedures I, II, and III, with respective yields of 25, 60, and 80%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.02 (d, <i>J</i> = 8.4 Hz, 1H), 7.60 (d, <i>J</i> = 8.3 Hz, 1H), 7.49 (t, <i>J</i> = 7.8 Hz, 1H), 7.36 (t, <i>J</i> = 7.5 Hz, 1H), 3.40 (t, <i>J</i> = 7.1 Hz, 2H), 2.34 (t, <i>J</i> = 7.5 Hz, 2H), 1.63 (p, <i>J</i> = 7.3 Hz, 2H), 1.53 (p, <i>J</i> = 7.4 Hz, 2H), 1.43 (p, <i>J</i> = 6.7 Hz, 2H), 1.33 (m, 4H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 179.3, 144.3, 132.3, 127.8, 124.3, 119.8, 109.8, 53.13, 33.99, 28.91, 28.88, 27.75, 26.53, 24.56; HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> [M + H] calcd (C<sub>14</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub>) 277.1659, observed 277.1670, δ ppm 4.0.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 10-[(1<i>H</i>-1,2,3-Benzotriazol-1-yl)amino]decanoic Acid (<b>8</b>)</h4><div class="NLM_p last">It was synthesized from 10-hydroxydecanoic acid and <b>ABT</b> according to general procedures I, II, and III, with respective yields of 21, 43, and 56%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.01 (d, <i>J</i> = 8.4 Hz, 1H), 7.60 (d, <i>J</i> = 8.3 Hz, 1H), 7.49 (t, <i>J</i> = 7.8 Hz, 1H), 7.36 (t, <i>J</i> = 7.5 Hz, 1H), 3.40 (t, <i>J</i> = 7.1 Hz, 2H), 2.34 (t, <i>J</i> = 7.5 Hz, 2H), 1.62 (p, <i>J</i> = 7.4 Hz, 2H), 1.52 (p, <i>J</i> = 7.5 Hz, 2H), 1.40 (p, <i>J</i> = 6.7 Hz, 2H), 1.28 (m, 8H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 179.2, 144.3, 132.2, 127.7, 124.2, 119.9, 109.8, 53.23, 33.96, 29.21, 29.08, 28.97, 27.82, 26.69, 24.65; HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> [M + H] calcd (C<sub>16</sub>H<sub>25</sub>N<sub>4</sub>O<sub>2</sub>) 305.1972, observed 305.1984, δ ppm 3.9.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 12-[(1<i>H</i>-1,2,3-Benzotriazol-1-yl)amino]dodecanoic Acid (<b>9</b>)</h4><div class="NLM_p last">It was synthesized from 12-hydroxydodecanoic acid and <b>ABT</b> according to general procedures I, II, and III, with respective yields of 22, 55, and 71%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.02 (d, <i>J</i> = 8.4 Hz, 1H), 7.61 (d, <i>J</i> = 8.3 Hz, 1H), 7.49 (t, <i>J</i> = 8.0 Hz, 1H), 7.36 (t, <i>J</i> = 7.8 Hz, 1H), 3.40 (t, <i>J</i> = 7.2 Hz, 2H), 2.35 (t, <i>J</i> = 7.5 Hz, 2H), 1.63 (p, <i>J</i> = 7.4 Hz, 2H), 1.53 (p, <i>J</i> = 7.5 Hz, 2H), 1.41 (p, <i>J</i> = 7.6 Hz, 2H), 1.26 (m, 12H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 179.5, 144.3, 132.2, 127.7, 124.2, 119.9, 109.8, 53.27, 34.06, 29.39, 29.32, 29.28, 29.16, 29.02, 27.83, 26.73, 24.70; HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd (C<sub>18</sub>H<sub>29</sub>N<sub>4</sub>O<sub>2</sub>) 333.2285, observed 333.2297, δ ppm 3.6.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> CYP4Z1-Mediated Luciferin-Benzyl Ether (Luc-BE) O-Debenzylation</h3><div class="NLM_p last">To compare Luc-BE (Promega, Madison, WI) O-debenzylation activity between the CYP4Z1-HepG2 (81 pmol CYP4Z1/mg membrane protein) and vector-HepG2 membrane stocks (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Method for Membrane Preparation</a>, Supporting Information),<a onclick="showRef(event, 'ref49 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51">(57−59)</a> metabolic reactions were set up in triplicate on ice and contained 0.6 mg/mL HepG2 membrane protein in 100 mM KPi, pH 7.4, in a final volume of 100 μL. Luc-BE was added to achieve a final concentration of 50 μM, and samples were equilibrated at 37 °C for 5 min. NADPH was added for a final concentration of 1 mM, or KPi was added for the minus NADPH controls. Metabolic reactions were allowed to proceed for 20 min at 37 °C. To assess the kinetics of CYP4Z1-mediated Luc-BE O-debenzylation, metabolic reactions were set up in triplicate on ice and contained 3 pmol CYP4Z1, in 100 mM KPi, pH 7.4, in a final volume of 100 μL. Luc-BE was added for final concentrations of 1, 5, 10, 25, 50, 75, 100, and 150 μM, and the reactions were equilibrated at 37 °C for 5 min. NADPH was added for a final concentration of 1 mM, and metabolism was allowed to proceed for 10 min at 37 °C.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Luminescence Assay (General Procedure IV)</h3><div class="NLM_p last">The CYP4Z1-mediated Luc-BE O-debenzylation reactions outlined above, those discussed below to assess inhibition of CYP4Z1 activity, and the other CYP-mediated pro-luciferin metabolism reactions were quenched (with P450-glo) and assayed as follows: reaction mixtures were transferred from their incubation tubes to white-walled 96-well plates containing an equal volume of P450-glo luciferase detection reagent (Promega, Madison, WI). Beetle luciferin metabolite standards (Promega), prepared in identical matrixes, were added at the same time as the metabolic reaction mixtures to the plates. After gently mixing and a 20 min incubation at room temperature, the plates were read on a Biotek Synergy HTX microplate reader (Winooski, VT). The luminescence generated (relative light units, RLUs) from the calibration luciferin standards was used to generate a standard curve to quantitate the metabolite concentrations in the reactions. These data were subsequently used to determine the rates of CYP4Z1 activity and for comparisons between inhibitor- and vehicle-treated reactions.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> CYP4 Isozyme IC<sub>50</sub>-Shift Experiments</h3><div class="NLM_p last">IC<sub>50</sub>-shift experiments to test inhibition of recombinantly expressed CYP4 isozymes were each performed separately, with duplicate technical replicates, and the experiments were repeated two to three times on separate days. Each metabolic reaction in the experiments contained one of the following: 4 pmol CYP4Z1 (HepG2 membranes), 1.5 pmol CYP4F8 (HepG2 membranes), and 4 pmol CYP4A11, CYP4F2, CYP4F3a, CYP4F3b, and CYP4F12 (Corning Supersomes, New York, NY). The total protein concentration for all reactions was ∼0.20–0.25 mg/mL, except for CYP4F8, which was 0.35 mg/mL due to the lower specific content of the enzyme in the HepG2 membrane preparation. Reactions were set up on ice containing the specific CYP4 isozyme in 100 mM KPi, pH 7.4, in a final volume of 200 μL. All inhibitors were solvated in DMSO to make 200× stocks and added to the reactions to achieve final concentrations spanning 0.001–100 μM (<b>1</b>–<b>9</b> for CYP4Z1), 0.03–1000 μM (<b>ABT</b> for CYP4Z1), 1–3000 μM (<b>6</b> for off-target CYP4 isozymes), or 1–1000 μM (<b>7</b> for off-target CYP4 isozymes). An equal volume of DMSO was added to all vehicle control samples, and DMSO concentrations did not exceed 0.5% v/v for all reactions. To assess time-dependent inhibition for the inhibitors, half of the metabolic reactions (for each isozyme) included a preincubation step plus the cofactor NADPH and the other half a preincubation step minus NADPH. The temporal and reagent addition scheme used was as follows: after a 5 min equilibration at 37 °C, NADPH was added from a 50× stock solution for a final concentration of 1 mM to the plus NADPH preincubation samples and an equal volume of buffer to the minus NADPH preincubation samples. After a subsequent 30 min preincubation at 37 °C, pro-luciferin probe substrates for each isozyme (detailed below) were added to both sets of preincubation reactions from 50× stocks (all at final concentrations ≤<i>K</i><sub>m</sub> values). Next, NADPH was added to the minus NADPH preincubation reactions and an equal volume of buffer to the plus NADPH preincubation reactions. The probe substrate reactions were allowed to proceed for 10 min at 37 °C, and reactions were assayed according to general procedure IV. The pro-luciferins (all from Promega) that were used as CYP4 isozyme probe substrates (concentrations) and the biotransformations that were monitored were as follows: CYP4Z1 Luc-BE (29 μM), CYP4F8 Luc-BE (12 μM), and CYP4F12 Luc-BE (23 μM), O-debenzylation; CYP4A11 Luc-4A (80 μM), O-demethylation; and CYP4F2, CYP4F3a, and CYP4F3b, Luc-4F2/3 (2 μM, due to substrate inhibition kinetics that was observed for all three enzymes, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S7</a>), O-(4-methylthio)-debenzylation. As no metabolite standard for Luc-4A or Luc-4F2/3 was available, enzymatic activity was assessed in a semiquantitative fashion for isozymes metabolizing these probe substrates. The counterscreening TDI experiment that is presented in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> used the same methods as detailed above, except with only a single concentration of 30 μM for inhibitors <b>2</b>–<b>9</b> and HET0016, as a general CYP4 inhibitor positive control, at 0.5 μM.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Hepatic CYP Isozyme IC<sub>50</sub>-Shift Experiments</h3><div class="NLM_p last">IC<sub>50</sub>-shift experiments to test inhibition of hepatic CYP isoforms by <b>6</b> and <b>7</b> were performed with duplicate technical replicates, and experiments were repeated three times on separate days. The assays utilized pooled human liver microsomes (HLMs) made up from eight random, individual donor livers that had been prepared for a previous study<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(60)</a> and followed a cocktail approach similar to previously published methods.<a onclick="showRef(event, 'ref19 ref33'); return false;" href="javascript:void(0);" class="ref ref19 ref33">(24,40)</a> The metabolic reactions contained HLM at 0.25 mg/mL total protein and inhibitor <b>6</b> or <b>7</b> at 1–3000 or 1–1000 μM, respectively, and used the same setup, timing, and reagent addition scheme as that followed for the CYP4 isozyme IC<sub>50</sub>-shift experiments. However, after the preincubation period, a cocktail of seven selective probe substrates (see below) combined was added from a 50× stock. The CYP isoforms evaluated, substrates (concentrations), and biotransformations were as follows: CYP1A2, phenacetin (100 μM), O-deethylation; CYP2B6, bupropion (12 μM), 1′-hydroxylation; CYP2C8, amodiaquine (1 μM), N-deethylation; CYP2C9, tolbutamide (100 μM), methyl-hydroxylation; CYP2C19, (<i>S</i>)-mephenytoin (50 μM), 4′-hydroxylation; CYP2D6, dextromethorphan (2.5 μM), O-demethylation; and CYP3A4/5, midazolam (2.5 μM), 1′-hydroxylation. All probe substrates were used at concentrations ≤<i>K</i><sub>m</sub>. After probe substrate turnover had proceeded for 10 min at 37 °C, reactions were subsequently terminated with an equal volume of ice-cold acetonitrile containing each deuterated internal standard (listed below) with a concentration of 200 nM and centrifuged at 15 000<i>g</i> for 10 min. A supernatant aliquot (5 μL) was analyzed by UPLC–MS/MS on a Waters Acquity UPLC connected to a Waters Xevo TQ-S instrument in ESI<sup>+</sup> mode, with the following settings: capillary, 3.1 kV; source offset, 60.0 V; source temperature, 150 °C; desolvation temperature, 350 °C; cone gas flow, 150 L/h; desolvation gas flow, 800 L/h; and collision gas flow, 0.15 mL/min. Cone voltage (CV) and collision energies (CEs) varied between analytes and are noted below. Analytes were separated chromatographically using a Waters HSS T3 column (2.1 × 100 mm<sup>2</sup>, 1.8 μm), starting with 95% mobile phase A (0.1% formic acid in H<sub>2</sub>O) and 5% mobile phase B (0.1% formic acid in acetonitrile) at a flow rate of 0.3 mL/min. After holding for 1 min, B was increased linearly from 5 to 60% between 1 and 7 min, immediately elevated to 95%, and then held at 95% between 7 and 8.5 min; the total run time was 11 min, following a 2.5 min equilibration period. The metabolites were detected by multiple reaction monitoring (MRM), with the following transitions, cone voltages (CVs), and collision energies (CEs): acetaminophen <i>m</i>/<i>z</i> 152.2 > 110, CV = 40 V, CE = 20 eV; hydroxybupropion <i>m</i>/<i>z</i> 256.1 > 238.1, CV = 32 V, CE = 10 eV; <i>N</i>-desethylamodiaquine <i>m</i>/<i>z</i> 328.1 > 283, CV = 40 V, CE = 20 eV; hydroxytolbutamide <i>m</i>/<i>z</i> 287.1 > 107, CV = 40 V, CE = 20 eV; 4-hydroxymephenytoin <i>m</i>/<i>z</i> 235.1 > 150.1, CV = 30 V, CE = 20 eV; dextrorphan <i>m</i>/<i>z</i> 258.1 > 157, CV = 40 V, CE = 30 eV; and 1′-hydroxymidazolam <i>m</i>/<i>z</i> 342.1 > 203, CV = 40 V, CE = 30 eV. The corresponding isotope-labeled internal standards and MRM transitions were as follows: acetaminophen-<i>d</i><sub>3</sub><i>m</i>/<i>z</i> 155.2 > 111.06, hydroxybupropion-<i>d</i><sub>6</sub><i>m</i>/<i>z</i> 262.2 > 244.2, <i>N</i>-desethylamodiaquine-<i>d</i><sub>5</sub><i>m</i>/<i>z</i> 333.1 > 283.1, hydroxytolbutamide-<i>d</i><sub>9</sub><i>m</i>/<i>z</i> 296.1 > 107, 4-hydroxymephenytoin-<i>d</i><sub>3</sub><i>m</i>/<i>z</i> 238.1 > 150.1, dextrorphan-<i>d</i><sub>3</sub><i>m</i>/<i>z</i> 261.2 > 157, and 1′-hydroxymidazolam-<i>d</i><sub>4</sub><i>m</i>/<i>z</i> 346.1 > 203.05. Metabolite standards, using identical matrix and workup conditions, were used to create standard curves that were utilized in quantitation of the metabolites and subsequent comparison of enzymatic activity between inhibitor- and vehicle-treated metabolic reactions.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Kinetics of CYP4Z1 Inactivation from <b>7</b></h3><div class="NLM_p last">Inhibition reactions were set up on ice in triplicate and contained 5 pmol CYP4Z1 in 100 mM KPi, pH 7.4, in a final volume of 100 μL. Inhibitor <b>7</b> was added to achieve final concentrations of 0.3, 1, 3, 6, 10, and 20 μM, and DMSO alone was utilized as a vehicle control. After a 5 min equilibration at 37 °C, the reactions were initiated by the addition of NADPH (1 mM final concentration). Aliquots of the inhibition reaction (10 μL) were taken at times 0, 3, 5, 10, 15, and 20 min and diluted 10-fold into a 100 μL probe substrate reaction containing Luc-BE, at 10× the determined <i>K</i><sub>m</sub> for CYP4Z1 (290 μM), in 100 mM KPi buffer with 1 mM NADPH. The probe substrate reactions progressed for 10 min at 37 °C and were quenched with an equal volume of ice-cold acetonitrile containing 200 nM Luc-4A as an internal standard and centrifuged at 15 000<i>g</i> for 10 min. Note that due to the timing constraints of assaying for the kinetics of enzyme inactivation, LC–MS/MS was utilized to detect the luciferin metabolite from CYP4Z1 Luc-BE O-debenzylation. A supernatant aliquot (3 μL) was analyzed by UPLC–MS/MS on a Waters Acquity UPLC connected to a Waters Xevo TQ-S instrument in ESI<sup>+</sup> mode, with the following settings: capillary, 2.8 kV; cone, 40.0 V; collision, 20.0 V; source offset, 60.0 V; source temperature, 150 °C; desolvation temperature, 350 °C; cone gas flow, 150 L/h; desolvation gas flow, 800 L/h; and collision gas flow, 0.15 mL/min. Analytes were separated chromatographically using a Waters BEH C18 column (2.1 × 50 mm<sup>2</sup>, 1.7 μm), starting with 95% mobile phase A (0.1% formic acid in H<sub>2</sub>O) and 5% mobile phase B (0.1% formic acid in acetonitrile) at a flow rate of 0.3 mL/min. After holding for 1 min, B was increased linearly from 5 to 95% between 1 and 4 min and held at 95% between 4 and 6 min; the total run time was 8.5 min, following a 2.5 min equilibration period. The metabolite luciferin was detected by MRM at <i>m</i>/<i>z</i> 281.0 > 234.92, and the internal standard Luc-4A was detected by MRM at <i>m</i>/<i>z</i> 289.1 > 201.96. The metabolite luciferin, using identical matrix and workup conditions, was used to create a standard curve that was utilized in quantitation of the metabolite concentration in the reactions and subsequent comparison of enzymatic activity between inhibitor- and vehicle-treated metabolic reactions. The natural logarithm of the remaining CYP4Z1 activity after treatment with varying concentrations of <b>7</b> was plotted versus preincubation time. The resultant differential slopes generated <i>k</i><sub>obs</sub> rates, which were then plotted against the concentrations of <b>7</b> to allow for the determination of the inactivation parameters <i>K</i><sub>I</sub> and <i>k</i><sub>inact</sub>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Examination of the Effects of Dialysis on CYP4Z1 Inactivation by <b>7</b></h3><div class="NLM_p last">Inhibition reactions were set up on ice in triplicate and contained 3 pmol CYP4Z1 in 100 mM KPi, pH 7.4, in a final volume of 100 μL. Inhibitor <b>7</b> was added for a final concentration of 3 μM, and DMSO was added to vehicle reactions. After a 5 min equilibration at 37 °C, the reactions were initiated by the addition of NADPH (1 mM final concentration) or KPi buffer only for the minus NADPH samples. The reactions were allowed to proceed for 15 min, and then, for the predialysis samples, aliquots (10 μL) were removed and diluted 10-fold into a 100 μL probe substrate reaction containing Luc-BE, at 10× the determined <i>K</i><sub>m</sub> for CYP4Z1 (290 μM), in 100 mM KPi buffer with 1 mM NADPH. The probe substrate reactions progressed for 10 min at 37 °C and were assayed according to general procedure IV. For the postdialysis samples (following the 15 min inhibition reaction), the entirety of the reaction mixture was immediately transferred to 10 000 molecular weight cutoff mini dialysis cups (Thermo Scientific Slide-A-Lyzer) suspended in 1 L of 100 mM KPi, pH 7.4, and incubated at 4 °C with gentle stirring overnight. The next morning, aliquots were removed, diluted into probe substrate reactions, and assayed as described for the predialysis samples.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Examination of the Effects of Trapping Agents and Reactive Oxygen Species-Scavenging Enzymes on CYP4Z1 Inactivation by <b>7</b></h3><div class="NLM_p last">Inhibition reactions were set up on ice in triplicate and contained 3 pmol CYP4Z1 in 100 mM KPi, pH 7.4, in a final volume of 100 μL. Inhibitor <b>7</b> was added for a final concentration of 3 μM, and DMSO was added to vehicle reactions. Reduced glutathione, <i>N</i>-acetyl-lysine, methoxyamine, <i>N</i>-acetyl-cysteine, and semicarbazide were added individually to reactions for final concentrations of 5 mM, and potassium cyanide was added to a reaction for a final concentration of 1 mM. Catalase (CAT) and superoxide dismutase (SOD), from Sigma (St. Louis, MO), were both added to an inhibition reaction together at 80 units each. Both inhibitor <b>7</b> and DMSO vehicle reactions were set up for each additive listed above. After a 5 min equilibration at 37 °C, the reactions were initiated by the addition of NADPH (1 mM final concentration) or KPi for the minus NADPH samples. The reactions were allowed to proceed for 15 min at 37 °C, and then aliquots of the inhibition reaction (10 μL) were diluted 10-fold into a 100 μL probe substrate reaction containing Luc-BE, at 10× the determined <i>K</i><sub>m</sub> for CYP4Z1 (290 μM), in 100 mM KPi buffer with 1 mM NADPH. The probe substrate reactions progressed for 10 min at 37 °C and were assayed according to general procedure IV.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Determination of the Partition Ratio</h3><div class="NLM_p last">The number of molecules of <b>7</b> metabolized per molecule of CYP4Z1 inactivated, the partition ratio, was estimated from a dilution method. Inhibition reactions were set up on ice in triplicate and contained 5 pmol CYP4Z1 in 100 mM KPi, pH 7.4, in a final volume of 100 μL. Inhibitor <b>7</b> was added at 11 different final concentrations spanning 0–50 μM. After a 5 min equilibration at 37 °C, the inactivation reactions were initiated by the addition of NADPH (1 mM final concentration). The reactions were allowed to proceed for 45 min at 37 °C to allow for maximal inactivation, and then, aliquots of the inhibition reaction (10 μL) were diluted 10-fold into a 100 μL probe substrate reaction containing Luc-BE, at 10× the determined <i>K</i><sub>m</sub> for CYP4Z1 (290 μM), in 100 mM KPi buffer with 1 mM NADPH. The probe substrate reactions progressed for 10 min at 37 °C and were assayed according to general procedure IV. The percentage of remaining CYP4Z1 activity was plotted versus the molar ratio of [<b>7</b>]/[CYP4Z1]. Extrapolation of the initial linear portion to the <i>x</i>-intercept generated the turnover number; the partition number was then calculated as the turnover number minus 1.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Inhibition of CYP4Z1-Mediated AA Metabolism in T47D Cells by <b>7</b></h3><div class="NLM_p last">T47D cells were modified to express CYP4Z1 (CYP4Z1-T47D), or a control vector (vector-T47D), in a similar fashion to the HepG2 cell lines. A schematic outline of the lentiviral vector system has been provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Supporting Information</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">Figure S1A</a>). Western blot analysis was performed as previously described.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(13)</a> The stably transduced cell lines were grown in tissue culture media consisting of Roswell Park Memorial Institute (RPMI)-1640, 1% glutamine, 10% fetal bovine serum (FBS), and 0.01 mg/mL insulin and selected with puromycin treatment (2 μg/mL) over a 4 day period to ensure homogeneous transduction profiles. Both CYP4Z1- and vector-T47D cell lines were seeded into six-well tissue-culture-treated plates at 2.5 × 10<sup>5</sup> cells/mL in a volume of 2 mL/well. The CYP4Z1-T47D cells were then dosed with inhibitor <b>7</b> to achieve final concentrations of 1 and 3 μM from 1000× DMSO stocks, or the DMSO vehicle was added (plus 24 h preincubation samples); vector-T47D cells were dosed with DMSO. The cells were then placed in a tissue culture incubator at 37 °C. After 24 h, AA was added to all wells for a final concentration of 50 μM from a 400× concentrated stock solvated in degassed ethanol. Inhibitor <b>7</b> was then added to the CYP4Z1-T47D cells, which had not been dosed the day prior (minus 24 h preincubation samples), at 1 and 3 μM, or the DMSO vehicle was added; vector-T47D cells were dosed with DMSO. Cells were placed back into the tissue culture incubator for 2 h to allow for CYP4Z1-mediated AA oxidation. Following this incubation period, media were collected from each well and 0.5 mL of H<sub>2</sub>O was added to the wells, which were then incubated at room temperature for 1 min. Subsequently, 1 mL of a 1:1 mixture of acetonitrile and methanol was added to each well and cells were scraped to remove them from the tissue culture dish. The aqueous and organic mixture was pooled with the respective media collections, and a further 1 mL of the acetonitrile/methanol mixture was added to rinse the wells of any remaining cells and/or extracted analytes. The combined aqueous and organic fractions were then subjected to a previously described workup,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(13)</a> and a supernatant aliquot (10 μL) was analyzed by UPLC–MS/MS on a Waters Acquity UPLC connected to a Waters Xevo TQ-S instrument in ESI<sup>–</sup> mode using the settings, chromatography gradient, and column as previously described by McDonald et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(13)</a> The oxidative AA metabolites 14,15-DiHET, 19-HETE, 20-HETE, and 14,15-EET were detected by MRM at <i>m</i>/<i>z</i> 337.10 > 207.00, 319.1 > 231, 319 > 245, and 319.1 > 219, respectively. The deuterated internal standards 14,15-DiHET-<i>d</i><sub>11</sub>, 20-HETE-<i>d</i><sub>6</sub>, and 14,15-EET-<i>d</i><sub>11</sub> were detected by MRM at <i>m</i>/<i>z</i> 348.30 > 207.20, 325.10 > 281.10, and 330.20 > 218.80, respectively. Relative inhibition of AA metabolite formation was determined by comparing the peak area ratios of the analytes divided by their respective deuterated internal standards from cells treated with <b>7</b> compared to the vehicle, at the various time points and conditions noted above. As no deuterated internal standard for 19-HETE is available, 20-HETE-<i>d</i><sub>6</sub> was utilized as the internal standard for this AA metabolite. The entire experiment was repeated under similar conditions and yielded comparable results (data not shown).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Data Analysis</h3><div class="NLM_p last">GraphPad Prism 7.00 (San Diego, CA) was used for graphing all results, estimating all kinetic parameters (<i>K</i><sub>m</sub>, <i>V</i><sub>max</sub>, <i>K</i><sub>I</sub>, and <i>k</i><sub>inact</sub>), and determining the half-maximal inhibitory concentration (IC<sub>50</sub>) values. IC<sub>50</sub> values from each individual experiment were averaged and standard deviations calculated. Mass spectral analysis was performed with Waters MassLynx v4.1 and Thermo Xcalibur v2.0. <sup>1</sup>H and <sup>13</sup>C NMR spectral analysis was performed with Bruker TopSpin v4.0.7.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00101" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55550" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55550" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00101?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00101</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">CYP4Z1 expression systems; method for membrane preparation; IC<sub>50</sub> and shifted-IC<sub>50</sub> determination for inhibition of CYP4Z1 by <b>1</b>–<b>9</b>, of CYP4 isoforms by <b>6</b> and <b>7</b>, and of hepatic CYP isoforms by <b>6</b> and <b>7</b>; Luc-4F2/3 metabolism kinetics for CYP4F2, CYP4F3a, and CYP4F3b; analysis of CYP4F8 metabolism of pro-luciferins; internal validation of the HLM cocktail inhibition assay; inhibition of CYP4Z1-mediated AA metabolism by <b>7</b> in HepG2 membranes; inhibition of CYP4Z1 in HepG2 cells and HepG2 and T47D cell viability; <sup>1</sup>H and <sup>13</sup>C NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings spreadsheet (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf">jm0c00101_si_001.pdf (4.06 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_002.csv">jm0c00101_si_002.csv (2.13 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00101" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74105" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74105" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Allan E. Rettie</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, School of Pharmacy, University of Washington, Seattle, Washington 98105, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#52203726263b371227257c373627"><span class="__cf_email__" data-cfemail="53213627273a361326247d363726">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John P. Kowalski</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, School of Pharmacy, University of Washington, Seattle, Washington 98105, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0343-8425" title="Orcid link">http://orcid.org/0000-0002-0343-8425</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew G. McDonald</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, School of Pharmacy, University of Washington, Seattle, Washington 98105, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert D. Pelletier</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, School of Pharmacy, University of Washington, Seattle, Washington 98105, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helmut Hanenberg</span> - <span class="hlFld-Affiliation affiliation">Department
of Pediatrics III, University Children’s Hospital Essen, University of Duisburg−Essen, 45122 Essen, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Constanze Wiek</span> - <span class="hlFld-Affiliation affiliation">Department
of Otorhinolaryngology and Head/Neck Surgery, Heinrich-Heine University, 40225 Düsseldorf, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3691-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i43">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80836" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80836" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Dale Whittington and J. Scott Edgar for assistance at the University of Washington, Department of Medicinal Chemistry, Mass Spectrometry Center. They also thank Adrienne Roehrich for assistance at the University of Washington, Department of Chemistry, NMR Facility. Finally, they thank Kent Kunze for helpful discussions regarding mechanism-based inactivation of CYP450s.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">CYP4Z1</td><td class="NLM_def"><p class="first last">cytochrome P450 4Z1</p></td></tr><tr><td class="NLM_term">KPi</td><td class="NLM_def"><p class="first last">potassium phosphate buffer</p></td></tr><tr><td class="NLM_term"><b>ABT</b></td><td class="NLM_def"><p class="first last">1-aminobenzotriazole</p></td></tr><tr><td class="NLM_term">Luc-BE</td><td class="NLM_def"><p class="first last">luciferin-benzyl ether</p></td></tr><tr><td class="NLM_term">NADPH</td><td class="NLM_def"><p class="first last">reduced β-nicotinamide-adenine dinucleotide phosphate</p></td></tr><tr><td class="NLM_term">MBI</td><td class="NLM_def"><p class="first last">mechanism-based inhibitor</p></td></tr><tr><td class="NLM_term">TDI</td><td class="NLM_def"><p class="first last">time-dependent inhibition</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">AA</td><td class="NLM_def"><p class="first last">arachidonic acid</p></td></tr><tr><td class="NLM_term">LA</td><td class="NLM_def"><p class="first last">lauric acid</p></td></tr><tr><td class="NLM_term">14,15-EET</td><td class="NLM_def"><p class="first last">14,15-epoxyeicosatrienoic acid</p></td></tr><tr><td class="NLM_term">20-HETE</td><td class="NLM_def"><p class="first last">20-hydroxyeicosatetraenoic acid</p></td></tr><tr><td class="NLM_term">PUFA</td><td class="NLM_def"><p class="first last">polyunsaturated fatty acid</p></td></tr><tr><td class="NLM_term">CAT</td><td class="NLM_def"><p class="first last">catalase</p></td></tr><tr><td class="NLM_term">SOD</td><td class="NLM_def"><p class="first last">superoxide dismutase</p></td></tr><tr><td class="NLM_term">GSH</td><td class="NLM_def"><p class="first last">glutathione</p></td></tr><tr><td class="NLM_term">NAL</td><td class="NLM_def"><p class="first last"><i>N</i>-acetyl-lysine</p></td></tr><tr><td class="NLM_term">MeONH<sub>2</sub></td><td class="NLM_def"><p class="first last">methoxyamine</p></td></tr><tr><td class="NLM_term">NAC</td><td class="NLM_def"><p class="first last"><i>N</i>-acetyl-cysteine</p></td></tr><tr><td class="NLM_term">KCN</td><td class="NLM_def"><p class="first last">potassium cyanide</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06805" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06805" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz de Montellano, P. R.</span></span> <i>Cytochrome P450: Structure, Mechanism, and Biochemistry</i>; <span class="NLM_publisher-name">Kluwer Academic Publishers</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2005</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1007%2Fb139087" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=P.+R.+Ortiz+de+Montellano&title=Cytochrome+P450%3A+Structure%2C+Mechanism%2C+and+Biochemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fb139087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252Fb139087%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26btitle%3DCytochrome%2520P450%253A%2520Structure%252C%2520Mechanism%252C%2520and%2520Biochemistry%26pub%3DKluwer%2520Academic%2520Publishers%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rieger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiessling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohayem, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Temme, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieber, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigle, B.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel mammary-restricted cytochrome P450, CYP4Z1, with overexpression in breast carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">2357</span>– <span class="NLM_lpage">2364</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-03-0849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1158%2F0008-5472.CAN-03-0849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15059886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1yjtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=2357-2364&author=M.+A.+Riegerauthor=R.+Ebnerauthor=D.+R.+Bellauthor=A.+Kiesslingauthor=J.+Rohayemauthor=M.+Schmitzauthor=A.+Temmeauthor=E.+P.+Rieberauthor=B.+Weigle&title=Identification+of+a+novel+mammary-restricted+cytochrome+P450%2C+CYP4Z1%2C+with+overexpression+in+breast+carcinoma&doi=10.1158%2F0008-5472.CAN-03-0849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Novel Mammary-Restricted Cytochrome P450, CYP4Z1, with Overexpression in Breast Carcinoma</span></div><div class="casAuthors">Rieger, Michael A.; Ebner, Reinhard; Bell, David R.; Kiessling, Andrea; Rohayem, Jacques; Schmitz, Marc; Temme, Achim; Rieber, E. Peter; Weigle, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2357-2364</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">By screening a transcriptome database for expressed sequence tags that are specifically expressed in mammary gland and breast carcinoma, the authors identified a new human cytochrome P 450 (CYP), termed CYP4Z1.  The cDNA was cloned from the breast carcinoma line SK-BR-3 and codes for a protein of 505 amino acids.  Moreover, a transcribed pseudogene CYP4Z2P that codes for a truncated CYP protein (340 amino acids) with 96% identity to CYP4Z1 was found in SK-BR-3.  CYP4Z1 and CYP4Z2P genes consisting of 12 exons are localized in head-to-head orientation on chromosome 1p33.  Tissue-specific expression was investigated using real-time reverse transcription PCR with normalized cDNA from 18 different human tissues.  CYP4Z1 mRNA was preferentially detected in breast carcinoma tissue and mammary gland, whereas only marginal expression was found in all other tested tissues.  Investigation of cDNA pairs from tumor/normal tissues obtained from 241 patients, including 50 breast carcinomas, confirmed the breast-restricted expression and showed a clear overexpression in 52% of breast cancer samples.  The expression profile of CYP4Z2P was similar to that of CYP4Z1 with preference in breast carcinoma and mammary gland but a lower expression level in general.  Immunoblot analyses with a specific antiserum for CYP4Z1 clearly demonstrated protein expression in mammary gland and breast carcinoma tissue specimens as well as in CYP4Z1-transduced cell lines.  Confocal laser-scanning microscopy of MCF-7 cells transfected with a fluorescent fusion protein CYP4Z1-enhanced green fluorescent protein and a subcellular fractionation showed localization to the endoplasmic reticulum as an integral membrane protein concordant for microsomal CYP enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGL-DjAe1fJrVg90H21EOLACvtfcHk0ljV5hZK0o44fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1yjtL4%253D&md5=e9000b09c4ca345038a79b42434fc40b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-03-0849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-03-0849%26sid%3Dliteratum%253Aachs%26aulast%3DRieger%26aufirst%3DM.%2BA.%26aulast%3DEbner%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DD.%2BR.%26aulast%3DKiessling%26aufirst%3DA.%26aulast%3DRohayem%26aufirst%3DJ.%26aulast%3DSchmitz%26aufirst%3DM.%26aulast%3DTemme%26aufirst%3DA.%26aulast%3DRieber%26aufirst%3DE.%2BP.%26aulast%3DWeigle%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520a%2520novel%2520mammary-restricted%2520cytochrome%2520P450%252C%2520CYP4Z1%252C%2520with%2520overexpression%2520in%2520breast%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D2357%26epage%3D2364%26doi%3D10.1158%2F0008-5472.CAN-03-0849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radvanyi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh-Sandhu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallichan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovitt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedyczak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gish, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zubovits, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hakimi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shortreed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrington, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oomen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tartaglia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berinstein, N. L.</span></span> <span> </span><span class="NLM_article-title">The gene associated with trichorhinophalangeal syndrome in humans is overexpressed in breast cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">11005</span>– <span class="NLM_lpage">11010</span>, <span class="refDoi"> DOI: 10.1073/pnas.0500904102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1073%2Fpnas.0500904102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=16043716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnvVWjurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=11005-11010&author=L.+Radvanyiauthor=D.+Singh-Sandhuauthor=S.+Gallichanauthor=C.+Lovittauthor=A.+Pedyczakauthor=G.+Malloauthor=K.+Gishauthor=K.+Kwokauthor=W.+Hannaauthor=J.+Zubovitsauthor=J.+Armesauthor=D.+Venterauthor=J.+Hakimiauthor=J.+Shortreedauthor=M.+Donovanauthor=M.+Parringtonauthor=P.+Dunnauthor=R.+Oomenauthor=J.+Tartagliaauthor=N.+L.+Berinstein&title=The+gene+associated+with+trichorhinophalangeal+syndrome+in+humans+is+overexpressed+in+breast+cancer&doi=10.1073%2Fpnas.0500904102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The gene associated with trichorhinophalangeal syndrome in humans is overexpressed in breast cancer</span></div><div class="casAuthors">Radvanyi, Laszlo; Singh-Sandhu, Devender; Gallichan, Scott; Lovitt, Corey; Pedyczak, Artur; Mallo, Gustavo; Gish, Kurt; Kwok, Kevin; Hanna, Wedad; Zubovits, Judith; Armes, Jane; Venter, Deon; Hakimi, Jalil; Shortreed, Jean; Donovan, Melinda; Parrington, Mark; Dunn, Pamela; Oomen, Ray; Tartaglia, James; Berinstein, Neil L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">11005-11010</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A comprehensive differential gene expression screen on a panel of 54 breast tumors and >200 normal tissue samples using DNA microarrays revealed 15 genes specifically overexpressed in breast cancer.  One of the most prevalent genes found was trichorhinophalangeal syndrome type 1 (TRPS-1), a gene previously shown to be assocd. with three rare autosomal dominant genetic disorders known as the trichorhinophalangeal syndromes.  A no. of corroborating methodologies, including in situ hybridization, e-Northern anal. using ORF EST (ORESTES) and Unigene EST abundance anal., immunoblot and immunofluorescence anal. of breast tumor cell lines, and immunohistochem., confirmed the microarray findings.  Immunohistochem. anal. found TRPS-1 protein expressed in >90% of early- and late-stage breast cancer, including ductal carcinoma in situ and invasive ductal, lobular, and papillary carcinomas.  The TRPS-1 gene is also immunogenic with processed and presented peptides activating T cells found after vaccination of HLA-A2.1 transgenic mouse.  Human T cell lines from HLA-A*0201+ female donors exhibiting TRPS-1-specific cytotoxic T lymphocyte activity could also be generated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY3pLDTU2xnrVg90H21EOLACvtfcHk0lj3MQ6HeDZu5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnvVWjurs%253D&md5=9ad3f97c19cb1c5c36053c9faa376cfb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0500904102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0500904102%26sid%3Dliteratum%253Aachs%26aulast%3DRadvanyi%26aufirst%3DL.%26aulast%3DSingh-Sandhu%26aufirst%3DD.%26aulast%3DGallichan%26aufirst%3DS.%26aulast%3DLovitt%26aufirst%3DC.%26aulast%3DPedyczak%26aufirst%3DA.%26aulast%3DMallo%26aufirst%3DG.%26aulast%3DGish%26aufirst%3DK.%26aulast%3DKwok%26aufirst%3DK.%26aulast%3DHanna%26aufirst%3DW.%26aulast%3DZubovits%26aufirst%3DJ.%26aulast%3DArmes%26aufirst%3DJ.%26aulast%3DVenter%26aufirst%3DD.%26aulast%3DHakimi%26aufirst%3DJ.%26aulast%3DShortreed%26aufirst%3DJ.%26aulast%3DDonovan%26aufirst%3DM.%26aulast%3DParrington%26aufirst%3DM.%26aulast%3DDunn%26aufirst%3DP.%26aulast%3DOomen%26aufirst%3DR.%26aulast%3DTartaglia%26aufirst%3DJ.%26aulast%3DBerinstein%26aufirst%3DN.%2BL.%26atitle%3DThe%2520gene%2520associated%2520with%2520trichorhinophalangeal%2520syndrome%2520in%2520humans%2520is%2520overexpressed%2520in%2520breast%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D11005%26epage%3D11010%26doi%3D10.1073%2Fpnas.0500904102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Tumoral expression of drug and xenobiotic metabolizing enzymes in breast cancer patients of different ethnicities with implications to personalized medicine</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">4747</span> <span class="refDoi"> DOI: 10.1038/s41598-017-04250-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1038%2Fs41598-017-04250-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=28684774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC1cjlt1Ortw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&author=Y.+Liauthor=A.+Steppiauthor=Y.+Zhouauthor=F.+Maoauthor=P.+C.+Millerauthor=M.+M.+Heauthor=T.+Zhaoauthor=Q.+Sunauthor=J.+Zhang&title=Tumoral+expression+of+drug+and+xenobiotic+metabolizing+enzymes+in+breast+cancer+patients+of+different+ethnicities+with+implications+to+personalized+medicine&doi=10.1038%2Fs41598-017-04250-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Tumoral expression of drug and xenobiotic metabolizing enzymes in breast cancer patients of different ethnicities with implications to personalized medicine</span></div><div class="casAuthors">Li Yan; Zhou Yidong; Mao Feng; Sun Qiang; Steppi Albert; Zhang Jinfeng; Miller Philip Craig; He Max M; He Max M; Zhao Tingting</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4747</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug and xenobiotic metabolizing enzymes (DXME) play important roles in drug responses and carcinogenesis.  Recent studies have found that expression of DXME in cancer cells significantly affects drug clearance and the onset of drug resistance.  In this study we compared the expression of DXME in breast tumor tissue samples from patients representing three ethnic groups: Caucasian Americans (CA), African Americans (AA), and Asian Americans (AS).  We further combined DXME gene expression data with eQTL data from the GTEx project and with allele frequency data from the 1000 Genomes project to identify SNPs that may be associated with differential expression of DXME genes.  We identified substantial differences among CA, AA, and AS populations in the expression of DXME genes and in activation of pathways involved in drug metabolism, including those involved in metabolizing chemotherapy drugs that are commonly used in the treatment of breast cancer.  These data suggest that differential expression of DXME may associate with health disparities in breast cancer outcomes observed among these three ethnic groups.  Our study suggests that development of personalized treatment strategies for breast cancer patients could be improved by considering both germline genotypes and tumor specific mutations and expression profiles related to DXME genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuRoK-jpntjJ6KlWarZTAzfW6udTcc2eadWVIGscElhrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjlt1Ortw%253D%253D&md5=afebeeb85f972ffb43e23f1fc1887c8e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-04250-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-04250-2%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSteppi%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DMao%26aufirst%3DF.%26aulast%3DMiller%26aufirst%3DP.%2BC.%26aulast%3DHe%26aufirst%3DM.%2BM.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DTumoral%2520expression%2520of%2520drug%2520and%2520xenobiotic%2520metabolizing%2520enzymes%2520in%2520breast%2520cancer%2520patients%2520of%2520different%2520ethnicities%2520with%2520implications%2520to%2520personalized%2520medicine%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26doi%3D10.1038%2Fs41598-017-04250-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, W. W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bureik, M.</span></span> <span> </span><span class="NLM_article-title">CYP4Z1 - A human cytochrome P450 enzyme that might hold the key to curing breast cancer</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2060</span>– <span class="NLM_lpage">2064</span>, <span class="refDoi"> DOI: 10.2174/1381612823666170207150156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.2174%2F1381612823666170207150156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=28176668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVShtrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=2060-2064&author=X.+Yangauthor=M.+Hutterauthor=W.+W.+B.+Gohauthor=M.+Bureik&title=CYP4Z1+-+A+human+cytochrome+P450+enzyme+that+might+hold+the+key+to+curing+breast+cancer&doi=10.2174%2F1381612823666170207150156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">CYP4Z1 - A Human Cytochrome P450 Enzyme that Might Hold the Key to Curing Breast Cancer</span></div><div class="casAuthors">Yang, Xu; Hutter, Michael; Bin Goh, Wilson Wen; Bureik, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2060-2064</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The human cytochrome P 450 (CYP) enzyme CYP4Z1 is a fatty acid hydroxylase which among human CYPs is unique for being much stronger expressed in the mammary gland than in all other tissues.  Moreover, it is strongly overexpressed in all subtypes of breast cancer, and some overexpression has also been found in other types of malignancies, such as ovarian, lung, and prostate cancers, resp.  Due to its unique expression pattern it is conceivable that this enzymes' activity might be exploited for a new therapeutic approach.  However, the main challenge for a CYP4Z1-based prodrug strategy (CBPS) for the treatment of breast cancer (and possibly other CYP4Z1-pos. malignancies) is the identification of candidate prodrugs that can be activated by this enzyme.  In this mini-review we summarize the current knowledge about the enzymic properties of the CYP4Z1 enzyme as well as on the expression pattern of the CYP4Z1 gene in both normal and cancer cells.  Moreover, we present the first homol. model of this enzyme and give an outlook on its potential use in cancer treatment strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9upj5h2gxrVg90H21EOLACvtfcHk0lj3MQ6HeDZu5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVShtrvE&md5=4f225fc40f0df5be9b3d3eae1715ea4b</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.2174%2F1381612823666170207150156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612823666170207150156%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHutter%26aufirst%3DM.%26aulast%3DGoh%26aufirst%3DW.%2BW.%2BB.%26aulast%3DBureik%26aufirst%3DM.%26atitle%3DCYP4Z1%2520-%2520A%2520human%2520cytochrome%2520P450%2520enzyme%2520that%2520might%2520hold%2520the%2520key%2520to%2520curing%2520breast%2520cancer%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2017%26volume%3D23%26spage%3D2060%26epage%3D2064%26doi%3D10.2174%2F1381612823666170207150156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patimalla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heys, S. D.</span></span> <span> </span><span class="NLM_article-title">Profiling the expression of cytochrome P450 in breast cancer</span>. <i>Histopathology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2559.2010.03606.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1111%2Fj.1365-2559.2010.03606.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=20716162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC3cjlsFGnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2010&pages=202-211&author=G.+I.+Murrayauthor=S.+Patimallaauthor=K.+N.+Stewartauthor=I.+D.+Millerauthor=S.+D.+Heys&title=Profiling+the+expression+of+cytochrome+P450+in+breast+cancer&doi=10.1111%2Fj.1365-2559.2010.03606.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling the expression of cytochrome P450 in breast cancer</span></div><div class="casAuthors">Murray Graeme I; Patimalla Siva; Stewart Keith N; Miller Iain D; Heys Steven D</div><div class="citationInfo"><span class="NLM_cas:title">Histopathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">202-11</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">AIMS:  The cytochrome P450s (P450) are key oxidative enzymes that metabolize many carcinogens and anticancer drugs.  Thus, these enzymes influence tumour development, tumour response to therapy and are putative tumour biomarkers.  The aim was to define the P450 expression profile in breast cancer and establish the significance of P450 expression in this tumour type.  METHODS AND RESULTS:  A tissue microarray containing 170 breast cancers of no special type was immunostained for a panel of 21 P450s.  The highest percentage of strong immunopositivity in breast cancers was seen for CYP4X1 (50.8%), CYP2S1 (37.5%) and CYP2U1 (32.2%), while CYP2J (98.6%) and CYP3A43 (70.7%) were the P450s that most frequently displayed no immunoreactivity.  CYP4V2 (P = 0.01), CYP4X1 (P = 0.01) and CYP4Z1 (P = 0.01) showed correlations with tumour grade.  CYP1B1 (P = 0.001), CYP3A5 (P = 0.001) and CYP51 (P = 0.005) showed the most significant correlations with oestrogen receptor status.  Correlations with survival were identified for CYP2S1 (P = 0.03), CYP3A4 (P = 0.025), CYP4V2 (P = 0.026) and CYP26A1 (P = 0.03), although none of these P450s was an independent marker of prognosis.  CONCLUSIONS:  This study has defined the expression profile of cytochrome P450s in breast cancer and may offer their potential application as biomarkers to aid decisions regarding optimal adjuvant hormonal therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT0ZpvUgyuFaDAyhZbj3lKdfW6udTcc2eaB3c7SK0dYybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjlsFGnsg%253D%253D&md5=96f08d44d30348e8511e91145b731b75</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2559.2010.03606.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2559.2010.03606.x%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DG.%2BI.%26aulast%3DPatimalla%26aufirst%3DS.%26aulast%3DStewart%26aufirst%3DK.%2BN.%26aulast%3DMiller%26aufirst%3DI.%2BD.%26aulast%3DHeys%26aufirst%3DS.%2BD.%26atitle%3DProfiling%2520the%2520expression%2520of%2520cytochrome%2520P450%2520in%2520breast%2520cancer%26jtitle%3DHistopathology%26date%3D2010%26volume%3D57%26spage%3D202%26epage%3D211%26doi%3D10.1111%2Fj.1365-2559.2010.03606.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Al-Esawi, N. S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geeran, A. R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alajeely, M. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Isawi, A. O. J.</span></span> <span> </span><span class="NLM_article-title">Expression of CYT4Z1 in breast carcinoma; Correlating with clinicopathological parameters</span>. <i>Biomed. Pharmacol. J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">511</span>, <span class="refDoi"> DOI: 10.13005/bpj/1912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.13005%2Fbpj%2F1912" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2020&pages=511&author=N.+S.+E.+Al-Esawiauthor=A.+R.+M.+Geeranauthor=M.+H.+J.+Alajeelyauthor=A.+O.+J.+Al-Isawi&title=Expression+of+CYT4Z1+in+breast+carcinoma%3B+Correlating+with+clinicopathological+parameters&doi=10.13005%2Fbpj%2F1912"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.13005%2Fbpj%2F1912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.13005%252Fbpj%252F1912%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Esawi%26aufirst%3DN.%2BS.%2BE.%26aulast%3DGeeran%26aufirst%3DA.%2BR.%2BM.%26aulast%3DAlajeely%26aufirst%3DM.%2BH.%2BJ.%26aulast%3DAl-Isawi%26aufirst%3DA.%2BO.%2BJ.%26atitle%3DExpression%2520of%2520CYT4Z1%2520in%2520breast%2520carcinoma%253B%2520Correlating%2520with%2520clinicopathological%2520parameters%26jtitle%3DBiomed.%2520Pharmacol.%2520J.%26date%3D2020%26volume%3D13%26spage%3D511%26doi%3D10.13005%2Fbpj%2F1912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falck, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span> <span> </span><span class="NLM_article-title">Increased expression of CYP4Z1 promotes tumor angiogenesis and growth in human breast cancer</span>. <i>Toxicol. Appl. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>264</i></span>,  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">83</span>, <span class="refDoi"> DOI: 10.1016/j.taap.2012.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.taap.2012.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=22841774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFGrsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=2012&pages=73-83&author=W.+Yuauthor=H.+Chaiauthor=Y.+Liauthor=H.+Zhaoauthor=X.+Xieauthor=H.+Zhengauthor=C.+Wangauthor=X.+Wangauthor=G.+Yangauthor=X.+Caiauthor=J.+R.+Falckauthor=J.+Yang&title=Increased+expression+of+CYP4Z1+promotes+tumor+angiogenesis+and+growth+in+human+breast+cancer&doi=10.1016%2Fj.taap.2012.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Increased expression of CYP4Z1 promotes tumor angiogenesis and growth in human breast cancer</span></div><div class="casAuthors">Yu, Wei; Chai, Hongyan; Li, Ying; Zhao, Haixia; Xie, Xianfei; Zheng, Hao; Wang, Chenlong; Wang, Xue; Yang, Guifang; Cai, Xiaojun; Falck, John R.; Yang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">264</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">73-83</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Cytochrome P 450 (CYP) 4Z1, a novel CYP4 family member, is over-expressed in human mammary carcinoma and assocd. with high-grade tumors and poor prognosis.  However, the precise role of CYP4Z1 in tumor progression is unknown.  Here, we demonstrate that CYP4Z1 overexpression promotes tumor angiogenesis and growth in breast cancer.  Stable expression of CYP4Z1 in T47D and BT-474 human breast cancer cells significantly increased mRNA expression and prodn. of vascular endothelial growth factor (VEGF)-A, and decreased mRNA levels and secretion of tissue inhibitor of metalloproteinase-2 (TIMP-2), without affecting cell proliferation and anchorage-independent cell growth in vitro.  Notably, the conditioned medium from CYP4Z1-expressing cells enhanced proliferation, migration and tube formation of human umbilical vein endothelial cells, and promoted angiogenesis in the zebrafish embryo and chorioallantoic membrane of the chick embryo.  In addn., there were lower levels of myristic acid and lauric acid, and higher contents of 20-hydroxyeicosatetraenoic acid (20-HETE) in CYP4Z1-expressing T47D cells compared with vector control.  CYP4Z1 overexpression significantly increased tumor wt. and microvessel d. by 2.6-fold and 1.9-fold in human tumor xenograft models, resp.  Moreover, CYP4Z1 transfection increased the phosphorylation of ERK1/2 and PI3K/Akt, while PI3K or ERK inhibitors and siRNA silencing reversed CYP4Z1-mediated changes in VEGF-A and TIMP-2 expression.  Conversely, HET0016, an inhibitor of the CYP4 family, potently inhibited the tumor-induced angiogenesis with assocd. changes in the intracellular levels of myristic acid, lauric acid and 20-HETE.  Collectively, these data suggest that increased CYP4Z1 expression promotes tumor angiogenesis and growth in breast cancer partly via PI3K/Akt and ERK1/2 activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI7xUctPOIvrVg90H21EOLACvtfcHk0lgm56aoC40Bxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFGrsrjO&md5=4b87a7625484e33ab82f9f7ba5b6ac92</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.taap.2012.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.taap.2012.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChai%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DCai%26aufirst%3DX.%26aulast%3DFalck%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DJ.%26atitle%3DIncreased%2520expression%2520of%2520CYP4Z1%2520promotes%2520tumor%2520angiogenesis%2520and%2520growth%2520in%2520human%2520breast%2520cancer%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2012%26volume%3D264%26spage%3D73%26epage%3D83%26doi%3D10.1016%2Fj.taap.2012.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edson, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">CYP4 enzymes as potential drug targets: focus on enzyme multiplicity, inducers and inhibitors, and therapeutic modulation of 20-hydroxyeicosatetraenoic acid (20-HETE) synthase and fatty acid omega-hydroxylase activities</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1429</span>– <span class="NLM_lpage">1440</span>, <span class="refDoi"> DOI: 10.2174/15680266113139990110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.2174%2F15680266113139990110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=23688133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WnsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=1429-1440&author=K.+Z.+Edsonauthor=A.+E.+Rettie&title=CYP4+enzymes+as+potential+drug+targets%3A+focus+on+enzyme+multiplicity%2C+inducers+and+inhibitors%2C+and+therapeutic+modulation+of+20-hydroxyeicosatetraenoic+acid+%2820-HETE%29+synthase+and+fatty+acid+omega-hydroxylase+activities&doi=10.2174%2F15680266113139990110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">CYP4 Enzymes As Potential Drug Targets: Focus on Enzyme Multiplicity, Inducers and Inhibitors, and Therapeutic Modulation of 20- Hydroxyeicosatetraenoic Acid (20-HETE) Synthase and Fatty Acid ω-Hydroxylase Activities</span></div><div class="casAuthors">Edson, Katheryne Z.; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1429-1440</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The Cytochrome P 450 4 (CYP4) family of enzymes in humans is comprised of thirteen isoenzymes that typically catalyze the ω -oxidn. of endogenous fatty acids and eicosanoids.  Several CYP4 enzymes can biosynthesize 20- hydroxyeicosatetraenoic acid, or 20-HETE, an important signaling eicosanoid involved in regulation of vascular tone and kidney reabsorption.  Addnl., accumulation of certain fatty acids is a hallmark of the rare genetic disorders, Refsum disease and X-ALD.  Therefore, modulation of CYP4 enzyme activity, either by inhibition or induction, is a potential strategy for drug discovery.  Here we review the substrate specificities, sites of expression, genetic regulation, and inhibition by exogenous chems. of the human CYP4 enzymes, and discuss the targeting of CYP4 enzymes in the development of new treatments for hypertension, stroke, certain cancers and the fatty acid-linked orphan diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNu1Vocf00NrVg90H21EOLACvtfcHk0lgm56aoC40Bxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WnsrrK&md5=065a5955167e84b0c5cb16d1f5f225c1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F15680266113139990110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F15680266113139990110%26sid%3Dliteratum%253Aachs%26aulast%3DEdson%26aufirst%3DK.%2BZ.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DCYP4%2520enzymes%2520as%2520potential%2520drug%2520targets%253A%2520focus%2520on%2520enzyme%2520multiplicity%252C%2520inducers%2520and%2520inhibitors%252C%2520and%2520therapeutic%2520modulation%2520of%252020-hydroxyeicosatetraenoic%2520acid%2520%252820-HETE%2529%2520synthase%2520and%2520fatty%2520acid%2520omega-hydroxylase%2520activities%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26spage%3D1429%26epage%3D1440%26doi%3D10.2174%2F15680266113139990110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edson, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totah, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">Cytochrome P450 omega-hydroxylases in inflammation and cancer</span>. <i>Adv. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1016/bs.apha.2015.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fbs.apha.2015.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=26233909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2015&pages=223-262&author=A.+L.+Johnsonauthor=K.+Z.+Edsonauthor=R.+A.+Totahauthor=A.+E.+Rettie&title=Cytochrome+P450+omega-hydroxylases+in+inflammation+and+cancer&doi=10.1016%2Fbs.apha.2015.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cytochrome P450 ω-hydroxylases in inflammation and cancer</span></div><div class="casAuthors">Johnson, Amanda L.; Edson, Katheryne Z.; Totah, Rheem A.; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Pharmacology (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">Cytochrome P450 Function and Pharmacological Roles in Inflammation and Cancer</span>),
    <span class="NLM_cas:pages">223-262</span>CODEN:
                <span class="NLM_cas:coden">ADPHEL</span>;
        ISSN:<span class="NLM_cas:issn">1054-3589</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Cytochrome P 450-dependent ω-hydroxylation is a prototypic metabolic reaction of CYP4 family members that is important for the elimination and bioactivation of not only therapeutic drugs, but also endogenous compds., principally fatty acids.  Eicosanoids, derived from arachidonic acid, are key substrates in the latter category.  Human CYP4 enzymes, mainly CYP4A11, CYP4F2, and CYP4F3B, hydroxylate arachidonic acid at the omega position to form 20-HETE, which has important effects in tumor progression and on angiogenesis and blood pressure regulation in the vasculature and kidney.  CYP4F3A in myeloid tissue catalyzes the ω-hydroxylation of leukotriene B4 to 20-hydroxy leukotriene B4, an inactivation process that is crit. for the regulation of the inflammatory response.  Here, we review the enzymol., tissue distribution, and substrate selectivity of human CYP4 ω-hydroxylases and their roles as catalysts for the formation and termination of the biol. effects of key eicosanoid metabolites in inflammation and cancer progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohzOfE_2I0hLVg90H21EOLACvtfcHk0lhagZ9CsFbY1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSht70%253D&md5=eea5e4ceba74de8e981748c50ae358e8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fbs.apha.2015.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.apha.2015.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DA.%2BL.%26aulast%3DEdson%26aufirst%3DK.%2BZ.%26aulast%3DTotah%26aufirst%3DR.%2BA.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DCytochrome%2520P450%2520omega-hydroxylases%2520in%2520inflammation%2520and%2520cancer%26jtitle%3DAdv.%2520Pharmacol.%26date%3D2015%26volume%3D74%26spage%3D223%26epage%3D262%26doi%3D10.1016%2Fbs.apha.2015.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z. H.</span></span> <span> </span><span class="NLM_article-title">14,15-EET induces the infiltration and tumor-promoting function of neutrophils to trigger the growth of minimal dormant metastases</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">43324</span>– <span class="NLM_lpage">43336</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.18632%2Foncotarget.9709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=27270316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FntlGhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=43324-43336&author=J.+Luoauthor=X.+X.+Fengauthor=C.+Luoauthor=Y.+Wangauthor=D.+Liauthor=Y.+Shuauthor=S.+S.+Wangauthor=J.+Qinauthor=Y.+C.+Liauthor=J.+M.+Zouauthor=D.+A.+Tianauthor=G.+M.+Zhangauthor=Z.+H.+Feng&title=14%2C15-EET+induces+the+infiltration+and+tumor-promoting+function+of+neutrophils+to+trigger+the+growth+of+minimal+dormant+metastases&doi=10.18632%2Foncotarget.9709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">14,15-EET induces the infiltration and tumor-promoting function of neutrophils to trigger the growth of minimal dormant metastases</span></div><div class="casAuthors">Luo Jing; Luo Chao; Wang Yu; Li Dong; Shu Yu; Wang Shan-Shan; Qin Jian; Li Yong-Chao; Zou Jiu-Ming; Zhang Gui-Mei; Feng Zuo-Hua; Feng Xin-Xia; Tian De-An</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">43324-43336</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Infiltrating neutrophils are known to promote in the development of tumor.  However, it is unclear whether and how neutrophils are involved in triggering the growth of dormant metastases.  Here we show that 14,15-epoxyeicosatrienoic acid (14,15-EET) can trigger the growth of dormant micrometastases by inducing neutrophilic infiltration and converting neutrophil function. 14,15-EET triggered neutrophil infiltration in metastatic lesions by activating STAT3 and JNK pathways to induce the expression of human IL-8 and murine CXCL15 in corresponding tumor cells.  The continuous expression of hIL-8/mCXCL15 was maintained by the sustained and enhanced activation of JNK pathway. 14,15-EET up-regulated miR-155 expression by activating STAT3 and JNK pathways. miR-155 in turn down-regulated the expression of SHIP1 and DET1, thus augmenting the activation of JNK and c-Jun.  Moreover, the function of neutrophils was converted from tumor-suppressing to tumor-promoting by 14,15-EET in vivo.  By inducing the production of G-CSF/IL-6 in vivo, 14,15-EET induced the enhancement of STAT3 activation in neutrophils to increase MMP-9 expression and decrease TRAIL expression.  Neutrophil-derived MMP-9 was required for 14,15-EET to induce angiogenesis during the growth of dormant micrometastases.  Depleting neutrophils or inhibiting hIL-8/mCXCL15 up-regulation resulted in the failure of 14,15-EET to promote the development of micrometastases.  These findings reveal a mechanism through which the infiltration and tumor-promoting function of neutrophils could be induced to trigger the growth of dormant metastases, which might be a driving force for the tumor recurrence based on dormant metastases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRhpIKq3LL6myEJCPKhJPuqfW6udTcc2ebKe8P55nca3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FntlGhsw%253D%253D&md5=6bebae354cacd572220eddae42b07ba4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9709%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%2BX.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DShu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%2BS.%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%2BC.%26aulast%3DZou%26aufirst%3DJ.%2BM.%26aulast%3DTian%26aufirst%3DD.%2BA.%26aulast%3DZhang%26aufirst%3DG.%2BM.%26aulast%3DFeng%26aufirst%3DZ.%2BH.%26atitle%3D14%252C15-EET%2520induces%2520the%2520infiltration%2520and%2520tumor-promoting%2520function%2520of%2520neutrophils%2520to%2520trigger%2520the%2520growth%2520of%2520minimal%2520dormant%2520metastases%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D43324%26epage%3D43336%26doi%3D10.18632%2Foncotarget.9709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panigrahy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeldin, D. C.</span></span> <span> </span><span class="NLM_article-title">EET signaling in cancer</span>. <i>Cancer Metastasis Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">540</span>, <span class="refDoi"> DOI: 10.1007/s10555-011-9315-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1007%2Fs10555-011-9315-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=22009066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1emt7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=525-540&author=D.+Panigrahyauthor=E.+R.+Greeneauthor=A.+Pozziauthor=D.+W.+Wangauthor=D.+C.+Zeldin&title=EET+signaling+in+cancer&doi=10.1007%2Fs10555-011-9315-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">EET signaling in cancer</span></div><div class="casAuthors">Panigrahy, Dipak; Greene, Emily R.; Pozzi, Ambra; Wang, Dao Wen; Zeldin, Darryl C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer and Metastasis Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">525-540</span>CODEN:
                <span class="NLM_cas:coden">CMRED4</span>;
        ISSN:<span class="NLM_cas:issn">0167-7659</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Inflammation and angiogenesis in the tumor microenvironment are increasingly implicated in tumorigenesis.  Endogenously produced lipid autacoids, locally acting small-mol. mediators, play a central role in inflammation and tissue homeostasis.  These lipid mediators, collectively referred to as eicosanoids, have recently been implicated in cancer.  Although eicosanoids, including prostaglandins and leukotrienes, are best known as products of arachidonic acid metab. by cyclooxygenases and lipoxygenases, arachidonic acid is also a substrate for another enzymic pathway, the cytochrome P 450 (CYP) system.  This eicosanoid pathway consists of two main branches: ω-hydroxylases which converts arachidonic acid to hydroxyeicosatetraenoic acids (HETEs) and epoxygenases which converts it to four regioisomeric epoxyeicosatrienoic acids (EETs; 5,6-EET, 8,9-EET, 11,12-EET, and 14,15-EET).  EETs regulate inflammation and vascular tone.  The bioactive EETs are produced predominantly in the endothelium and are mainly metabolized by sol. epoxide hydrolase to less active dihydroxyeicosatrienoic acids.  EET signaling was originally studied in conjunction with inflammatory and cardiovascular disease.  Arachidonic acid and its metabolites have recently stimulated great interest in cancer biol.  To date, most research on eicosanoids in cancer has focused on the COX and LOX pathways.  In contrast, the role of cytochrome P 450-derived eicosanoids, such as EETs and HETEs, in cancer has received little attention.  While CYP epoxygenases are expressed in human cancers and promote human cancer metastasis, the role of EETs (the direct products of CYP epoxygenases) in cancer remains poorly characterized.  In this review, the emerging role of EET signaling in angiogenesis, inflammation, and cancer is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMFV5maT9FHLVg90H21EOLACvtfcHk0lhagZ9CsFbY1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1emt7vL&md5=e6b8d1bef5d7f7a26eadb006a8b4d77c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs10555-011-9315-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10555-011-9315-y%26sid%3Dliteratum%253Aachs%26aulast%3DPanigrahy%26aufirst%3DD.%26aulast%3DGreene%26aufirst%3DE.%2BR.%26aulast%3DPozzi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DD.%2BW.%26aulast%3DZeldin%26aufirst%3DD.%2BC.%26atitle%3DEET%2520signaling%2520in%2520cancer%26jtitle%3DCancer%2520Metastasis%2520Rev.%26date%3D2011%26volume%3D30%26spage%3D525%26epage%3D540%26doi%3D10.1007%2Fs10555-011-9315-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amorosi, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitko, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalski, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totah, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">Expression and functional characterization of breast cancer-associated cytochrome P450 4Z1 in saccharomyces cerevisiae</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1364</span>– <span class="NLM_lpage">1371</span>, <span class="refDoi"> DOI: 10.1124/dmd.117.078188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.117.078188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=29018033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsVansL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=1364-1371&author=M.+G.+McDonaldauthor=S.+Rayauthor=C.+J.+Amorosiauthor=K.+A.+Sitkoauthor=J.+P.+Kowalskiauthor=L.+Pacoauthor=A.+Nathauthor=B.+Gallisauthor=R.+A.+Totahauthor=M.+J.+Dunhamauthor=D.+M.+Fowlerauthor=A.+E.+Rettie&title=Expression+and+functional+characterization+of+breast+cancer-associated+cytochrome+P450+4Z1+in+saccharomyces+cerevisiae&doi=10.1124%2Fdmd.117.078188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Expression and functional characterization of breast cancer-associated cytochrome P450 4Z1 in Saccharomyces cerevisiae</span></div><div class="casAuthors">McDonald, Matthew G.; Ray, Sutapa; Amorosi, Clara J.; Sitko, Katherine A.; Kowalski, John P.; Paco, Lorela; Nath, Abhinav; Gallis, Byron; Totah, Rheem A.; Dunham, Maitreya J.; Fowler, Douglas M.; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1364-1371</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">CYP4Z1 is an "orphan" cytochrome P 450 (P 450) enzyme that has provoked interest because of its hypothesized role in breast cancer through formation of the signaling mol. 20-hydroxyeicosatetraenoic acid (20-HETE).  We expressed human CYP4Z1 in Saccharomyces cerevisiae and evaluated its catalytic capabilities toward arachidonic and lauric acids (AA and LA).  Specific and sensitive mass spectrometry assays enabled discrimination of the regioselectivity of hydroxylation of these two fatty acids.  CYP4Z1 generated 7-, 8-, 9-, 10-, and 11-hydroxy LA, whereas the 12-hydroxy metabolite was not detected.  HET0016, the prototypic CYP4 inhibitor, only weakly inhibited laurate metabolite formation (IC50 ∼15 μM).  CYP4Z1 preferentially oxidized AA to the 14(S),15(R)-epoxide with high regioselectivity and stereoselectivity, a reaction that was also insensitive to HET0016, but neither 20-HETE nor 20-carboxy-AA were detectable metabolites.  Docking of LA and AA into a CYP4Z1 homol. model was consistent with this preference for internal fatty acid oxidn.  Thus, human CYP4Z1 has an inhibitor profile and product regioselectivity distinct from most other CYP4 enzymes, consistent with CYP4Z1's lack of a covalently linked heme.  These data suggest that, if CYP4Z1 modulates breast cancer progression, it does so by a mechanism other than direct prodn. of 20-HETE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYpQYvNbglrLVg90H21EOLACvtfcHk0lg8Hqs3W2uHCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsVansL4%253D&md5=cc68fbb4ebb3b5d9f808005fab0e5458</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1124%2Fdmd.117.078188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.117.078188%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DM.%2BG.%26aulast%3DRay%26aufirst%3DS.%26aulast%3DAmorosi%26aufirst%3DC.%2BJ.%26aulast%3DSitko%26aufirst%3DK.%2BA.%26aulast%3DKowalski%26aufirst%3DJ.%2BP.%26aulast%3DPaco%26aufirst%3DL.%26aulast%3DNath%26aufirst%3DA.%26aulast%3DGallis%26aufirst%3DB.%26aulast%3DTotah%26aufirst%3DR.%2BA.%26aulast%3DDunham%26aufirst%3DM.%2BJ.%26aulast%3DFowler%26aufirst%3DD.%2BM.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DExpression%2520and%2520functional%2520characterization%2520of%2520breast%2520cancer-associated%2520cytochrome%2520P450%25204Z1%2520in%2520saccharomyces%2520cerevisiae%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2017%26volume%3D45%26spage%3D1364%26epage%3D1371%26doi%3D10.1124%2Fdmd.117.078188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">World Cancer Research
Fund</span>. <a href="https://www.wcrf.org/dietandcancer/cancer-trends/breast-cancer-statistics" class="extLink">https://www.wcrf.org/dietandcancer/cancer-trends/breast-cancer-statistics</a> (accessed Oct, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=World+Cancer+Research%0AFund.+https%3A%2F%2Fwww.wcrf.org%2Fdietandcancer%2Fcancer-trends%2Fbreast-cancer-statistics+%28accessed+Oct%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capper, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rae, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auchus, R. J.</span></span> <span> </span><span class="NLM_article-title">The metabolism, analysis, and targeting of steroid hormones in breast and prostate cancer</span>. <i>Horm. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1007/s12672-016-0259-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1007%2Fs12672-016-0259-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=26969590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28XktlCitr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=149-164&author=C.+P.+Capperauthor=J.+M.+Raeauthor=R.+J.+Auchus&title=The+metabolism%2C+analysis%2C+and+targeting+of+steroid+hormones+in+breast+and+prostate+cancer&doi=10.1007%2Fs12672-016-0259-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The Metabolism, Analysis, and Targeting of Steroid Hormones in Breast and Prostate Cancer</span></div><div class="casAuthors">Capper, Cameron P.; Rae, James M.; Auchus, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Hormones & Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">149-164</span>CODEN:
                <span class="NLM_cas:coden">HCOABS</span>;
        ISSN:<span class="NLM_cas:issn">1868-8497</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Breast and prostate cancers are malignancies in which steroid hormones drive cellular proliferation.  Over the past century, this understanding has led to successful treatment strategies aimed to inhibit hormone-mediated tumor growth.  Nonetheless, disease relapse and progression still pose significant clin. problems, with recurrent and metastatic tumors often exhibiting resistance to current drug therapies.  The central role of androgens and estrogens in prostate and breast cancer etiol. explains not only why endocrine therapies are often initially successful but also why many tumors ultimately become resistant.  It is hypothesized that reducing the concn. of active hormones in the systemic circulation may be insufficient to block cancer progression, as this action selects for tumor cells that can generate active steroids from circulating precursors.  This review aims to highlight the currently known differences of steroid biosynthesis in normal physiol. vs. hormone-dependent cancers, modern approaches to the assessment and targeting of these pathways, and priorities for future research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-7vQDqCGJ37Vg90H21EOLACvtfcHk0lg8Hqs3W2uHCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktlCitr0%253D&md5=ad8c9d3ddd2e8534993da8b6ff0b427f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs12672-016-0259-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12672-016-0259-0%26sid%3Dliteratum%253Aachs%26aulast%3DCapper%26aufirst%3DC.%2BP.%26aulast%3DRae%26aufirst%3DJ.%2BM.%26aulast%3DAuchus%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520metabolism%252C%2520analysis%252C%2520and%2520targeting%2520of%2520steroid%2520hormones%2520in%2520breast%2520and%2520prostate%2520cancer%26jtitle%3DHorm.%2520Cancer%26date%3D2016%26volume%3D7%26spage%3D149%26epage%3D164%26doi%3D10.1007%2Fs12672-016-0259-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Njar, V. C.</span></span> <span> </span><span class="NLM_article-title">Targeting cytochrome P450 enzymes: a new approach in anti-cancer drug development</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">5047</span>– <span class="NLM_lpage">5060</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2007.05.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.bmc.2007.05.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=17544277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtlahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=5047-5060&author=R.+D.+Brunoauthor=V.+C.+Njar&title=Targeting+cytochrome+P450+enzymes%3A+a+new+approach+in+anti-cancer+drug+development&doi=10.1016%2Fj.bmc.2007.05.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cytochrome P450 enzymes: A new approach in anti-cancer drug development</span></div><div class="casAuthors">Bruno, Robert D.; Njar, Vincent C. O.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5047-5060</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cytochrome P450s (CYPs) represent a large class of heme-contg. enzymes that catalyze the metab. of multitudes of substrates both endogenous and exogenous.  Until recently, however, CYPs have been largely overlooked in cancer drug development, acknowledged only for their role in phase I metab. of chemotherapeutics.  The first successful strategy targeting CYP enzymes in cancer therapy was the development of potent inhibitors of CYP19 (aromatase) for the treatment of breast cancer.  Aromatase inhibitors ushered in a new era in hormone ablation therapy for estrogen dependent cancers, and have paved the way for similar strategies (i.e., inhibition of CYP17) that combat androgen-dependent prostate cancer.  Identification of CYPs involved in the inactivation of anticancer metabolites of vitamin D3 and vitamin A has triggered development of agents that target these enzymes as well.  The discovery of the over-expression of exogenous metabolizing CYPs, such as CYP1B1, in cancer cells has roused interest in the development of inhibitors for chemoprevention and of prodrugs designed to be activated by CYPs only in cancer cells.  Finally, the expression of CYPs within tumors has been utilized in the development of bioreductive mols. that are activated by CYPs only under hypoxic conditions.  This review offers the first comprehensive anal. of strategies in drug development that either inhibit or exploit CYP enzymes for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtuQVdXRAm27Vg90H21EOLACvtfcHk0lg8Hqs3W2uHCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtlahsbg%253D&md5=bf2eab080cc0a7f7623c7af9942c359d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.05.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.05.046%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DR.%2BD.%26aulast%3DNjar%26aufirst%3DV.%2BC.%26atitle%3DTargeting%2520cytochrome%2520P450%2520enzymes%253A%2520a%2520new%2520approach%2520in%2520anti-cancer%2520drug%2520development%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D5047%26epage%3D5060%26doi%3D10.1016%2Fj.bmc.2007.05.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westerink, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoonen, W. G.</span></span> <span> </span><span class="NLM_article-title">Cytochrome P450 enzyme levels in HepG2 cells and cryopreserved primary human hepatocytes and their induction in HepG2 cells</span>. <i>Toxicol. In Vitro</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1581</span>– <span class="NLM_lpage">1591</span>, <span class="refDoi"> DOI: 10.1016/j.tiv.2007.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.tiv.2007.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=17637504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWrsrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=1581-1591&author=W.+M.+Westerinkauthor=W.+G.+Schoonen&title=Cytochrome+P450+enzyme+levels+in+HepG2+cells+and+cryopreserved+primary+human+hepatocytes+and+their+induction+in+HepG2+cells&doi=10.1016%2Fj.tiv.2007.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cytochrome P450 enzyme levels in HepG2 cells and cryopreserved primary human hepatocytes and their induction in HepG2 cells</span></div><div class="casAuthors">Westerink, Walter M. A.; Schoonen, Willem G. E. J.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology in Vitro</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1581-1591</span>CODEN:
                <span class="NLM_cas:coden">TIVIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0887-2333</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Early in vitro toxicity screening might improve the success rate of new chem. entities in pharmaceutical development.  In previous studies, the advantage of cytotoxicity screening with the HepG2 cell line was shown.  Cytotoxicity could be identified for 70% of the compds. in these assays as compared with known toxicity in either in vitro assays in primary hepatocytes, in in vivo assays in rats, or in (pre-)clin. development in humans.  The low Phase I and II enzyme levels in HepG2 cells might were responsible for the fact that 30% of the compds. scored neg.  Therefore, we performed 2 follow-up studies in which Cytochrome P 450 (CYP) enzymes and Phase II metab. were examd.  In the present study, the transcript levels of CYP1A1, 1A2, 2A6, 2B6, 2C8, 2C9, 2C19, 2D6, 2E1, and 3A4 were measured with quant. PCR.  Results showed that transcripts of all CYPs were present in HepG2 cells, however, mRNA levels of most CYPs were dramatically lower than in primary human hepatocytes.  These results were confirmed with luminometric assays which were used to measure the enzyme activities of CYP1A1, 1A2, 2C9, and 3A4.  Regulation of CYP1A1, 1A2, 2B6, 2C8, 2D6, 2E1, and 3A4 by the aryl hydrocarbon receptor, pregnane X receptor and constitutive androstane receptor was studied in HepG2 cells at the mRNA and/or enzyme level.  Regulation of CYP1A1, 1A2, 2B6, and 3A4 mRNA levels was similar to the regulation in primary human hepatocytes.  In contrast, CYP2C8 mRNA levels are inducible in primary human hepatocytes, but not in HepG2 cells, after treatment with PXR/CAR activators.  Consistent with other studies, CYP2D6 and 2E1 transcript levels were not changed after treatment with AhR, PXR, and CAR activators.  Moreover, CYP1A1 and 1A2 enzyme levels could be induced by AhR agonists and CYP3A4 by PXR agonists.  As a consequence of the low levels of CYPs in HepG2 cells, cytotoxicity of several compds. might were missed or underestimated as compared with cytotoxicity in primary human hepatocytes.  Inducing HepG2 cells with particular receptor stimulators might lead to higher toxicity for several of the tested compds.  Compared to primary human hepatocytes, HepG2 cells are a relatively easy-to-handle tool to study the up-regulation of CYP1A1, 1A2, 2B6, and 3A4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ0IAUSBPeubVg90H21EOLACvtfcHk0liOWP-5RGUykA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWrsrvN&md5=b9d0fb486135d2ac6f9d58255204d1d3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2007.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2007.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DWesterink%26aufirst%3DW.%2BM.%26aulast%3DSchoonen%26aufirst%3DW.%2BG.%26atitle%3DCytochrome%2520P450%2520enzyme%2520levels%2520in%2520HepG2%2520cells%2520and%2520cryopreserved%2520primary%2520human%2520hepatocytes%2520and%2520their%2520induction%2520in%2520HepG2%2520cells%26jtitle%3DToxicol.%2520In%2520Vitro%26date%3D2007%26volume%3D21%26spage%3D1581%26epage%3D1591%26doi%3D10.1016%2Fj.tiv.2007.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kammerer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Küpper, J. H.</span></span> <span> </span><span class="NLM_article-title">NADPH-cytochrome P450 reductase expression and enzymatic activity in primary-like human hepatocytes and HepG2 cells for in vitro biotransformation studies</span>. <i>Clin. Hemorheol. Microcirc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.3233/CH-199226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.3233%2FCH-199226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=31561354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFGnurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2019&pages=249-260&author=C.+Schulzauthor=S.+Kammererauthor=J.+H.+K%C3%BCpper&title=NADPH-cytochrome+P450+reductase+expression+and+enzymatic+activity+in+primary-like+human+hepatocytes+and+HepG2+cells+for+in+vitro+biotransformation+studies&doi=10.3233%2FCH-199226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">NADPH-cytochrome P450 reductase expression and enzymatic activity in primary-like human hepatocytes and HepG2 cells for in vitro biotransformation studies</span></div><div class="casAuthors">Schulz, Christian; Kammerer, Sarah; Kuepper, Jan-Heiner; Wiggermann, P.; Krueger-Genge, A.; Jung, F.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Hemorheology and Microcirculation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">249-260</span>CODEN:
                <span class="NLM_cas:coden">CHMIFQ</span>;
        ISSN:<span class="NLM_cas:issn">1386-0291</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">BACKGROUND: : Human hepatocyte in vitro cell culture systems are important models for drug development and toxicol. studies in the context of liver xenobiotic metab.  Often, such culture systems are used to elucidate the biotransformation of xenobiotics or drugs and further investigate drug and drug metabolite effects on biol. systems in terms of potential therapeutic benefit or toxicity.  Human hepatocytes currently used for such in vitro studies are mostly primary cells or cell lines derived from liver cancers.  Both approaches have limitations such as low proliferation capacity and progressive dedifferentiation found in primary cells or lack of liver functions in cell lines, which makes it difficult to reliably predict biotransformation of xenobiotics in patients.  In order to overcome these limitations, HepaFH3 cells and Upcyte hepatocytes representing primary-like hepatocytes of the first and second generation are increasingly used.  Based on primary human hepatocyte cells transduced for stable expression of Upcyte proliferation genes, they are mitotically active and exhibit liver functions over an extended period, making them comparable to primary human hepatocytes.  These hepatocyte models show active liver metab. such as urea and glycogen formation as well as biotransformation of xenobiotics.  The latter is based on the expression, activity and inducibility of cytochrome P 450 enzymes (CYP) as essential phase I reaction components.  However, for further characterization in terms of performance and existing limitations, addnl. studies are needed to elucidate the mechanisms involved in phase I reactions.  One prerequisite is sufficient activity of microsomal NADPH-cytochrome P 450 reductase (POR) functionally connected as electron donor to those CYP enzymes.  OBJECTIVE: : For Upcyte hepatocytes and HepaFH3 cells, it is so far unknown to what extent POR is expressed, active, and may exert CYP-modulating effects.  Here we studied POR expression and corresponding enzyme activity in human hepatoblastoma cell line HepG2 and compared this with HepaFH3 and Upcyte hepatocytes representing proliferating primary-like hepatocytes.  METHODS: : POR expression of those hepatocyte models was detd. at mRNA and protein level using qRT-PCR, Western Blot and immunofluorescence staining.  Kinetic studies on POR activity in isolated microsomes were performed by a colorimetric method.  RESULTS: : The investigated hepatocyte models showed remarkable differences at the level of POR expression.  Compared to primary-like hepatocytes, POR expression of HepG2 cells was 4-fold higher at mRNA and 2-fold higher at protein level.  However, this higher expression did not correlate with corresponding enzyme activity levels in isolated microsomes, which were comparable between all cell systems tested.  A tendency of higher POR activity in HepG2 cells compared to HepaFH3 (p=0.0829) might be present.  Compared to primary human hepatocyte microsomes, POR activity was considerably lower in all hepatocyte models.  CONCLUSION: : In summary, our study revealed that POR expression and activity were clearly detectable in all in vitro hepatocyte models with the highest POR expression in cancer cell line HepG2.  However, POR activity was lower in tested hepatocyte models when compared to human primary hepatocyte microsomes.  Whether this was caused by e.g. polymorphisms or metabolic differences of investigated hepatocyte models will be target for future studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNB6apSkXyxbVg90H21EOLACvtfcHk0liOWP-5RGUykA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFGnurzL&md5=d3961203b786c32242c01c8e5427fdc9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3233%2FCH-199226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FCH-199226%26sid%3Dliteratum%253Aachs%26aulast%3DSchulz%26aufirst%3DC.%26aulast%3DKammerer%26aufirst%3DS.%26aulast%3DK%25C3%25BCpper%26aufirst%3DJ.%2BH.%26atitle%3DNADPH-cytochrome%2520P450%2520reductase%2520expression%2520and%2520enzymatic%2520activity%2520in%2520primary-like%2520human%2520hepatocytes%2520and%2520HepG2%2520cells%2520for%2520in%2520vitro%2520biotransformation%2520studies%26jtitle%3DClin.%2520Hemorheol.%2520Microcirc.%26date%3D2019%26volume%3D73%26spage%3D249%26epage%3D260%26doi%3D10.3233%2FCH-199226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guengerich, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Q.</span></span> <span> </span><span class="NLM_article-title">Measurement of cytochrome P450 and NADPH-cytochrome P450 reductase</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1245</span>– <span class="NLM_lpage">1251</span>, <span class="refDoi"> DOI: 10.1038/nprot.2009.121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1038%2Fnprot.2009.121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=19661994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVygsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=1245-1251&author=F.+P.+Guengerichauthor=M.+V.+Martinauthor=C.+D.+Sohlauthor=Q.+Cheng&title=Measurement+of+cytochrome+P450+and+NADPH-cytochrome+P450+reductase&doi=10.1038%2Fnprot.2009.121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement of cytochrome P 450 and NADPH-cytochrome P 450 reductase</span></div><div class="casAuthors">Guengerich, F. Peter; Martin, Martha V.; Sohl, Christal D.; Cheng, Qian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1245-1251</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cytochrome P 450 enzymes are important in the metab. of steroids, vitamins, carcinogens, drugs, and other compds.  Two of the commonly used assays in this field are the measurements of total P 450 and NADPH-P 450 reductase in biol. prepns.  A detailed protocol is presented for the measurement of P 450 by its spectral properties, along with a protocol for measuring NADPH-P 450 reductase by its NADPH-cytochrome c redn. activity.  Each assay can be completed in 5-10 min.  Detailed explanations for the rationale of particular sequences in the protocols are provided, along with potential confounding problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbeZdtyf5GjrVg90H21EOLACvtfcHk0liOWP-5RGUykA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVygsLnP&md5=f0d2167bf5427faee0fbe6f11c85e06d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2009.121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2009.121%26sid%3Dliteratum%253Aachs%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26aulast%3DMartin%26aufirst%3DM.%2BV.%26aulast%3DSohl%26aufirst%3DC.%2BD.%26aulast%3DCheng%26aufirst%3DQ.%26atitle%3DMeasurement%2520of%2520cytochrome%2520P450%2520and%2520NADPH-cytochrome%2520P450%2520reductase%26jtitle%3DNat.%2520Protoc.%26date%3D2009%26volume%3D4%26spage%3D1245%26epage%3D1251%26doi%3D10.1038%2Fnprot.2009.121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machalz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorensen, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bureik, M.</span></span> <span> </span><span class="NLM_article-title">Efficient substrate screening and inhibitor testing of human CYP4Z1 using permeabilized recombinant fission yeast</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2017.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.bcp.2017.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=28951277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOhsLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2017&pages=174-187&author=Q.+Yanauthor=D.+Machalzauthor=A.+Zollnerauthor=E.+J.+Sorensenauthor=G.+Wolberauthor=M.+Bureik&title=Efficient+substrate+screening+and+inhibitor+testing+of+human+CYP4Z1+using+permeabilized+recombinant+fission+yeast&doi=10.1016%2Fj.bcp.2017.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient substrate screening and inhibitor testing of human CYP4Z1 using permeabilized recombinant fission yeast</span></div><div class="casAuthors">Yan, Qi; Machalz, David; Zollner, Andy; Sorensen, Erik J.; Wolber, Gerhard; Bureik, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">174-187</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have established a protocol for the prepn. of permeabilized fission yeast cells (enzyme bags) that recombinantly express human cytochrome P 450 enzymes (CYPs).  A direct comparison of CYP3A4 activity gave an eightfold higher space-time yield for enzyme bag-catalyzed biotransformation as compared to whole-cell biotransformation, even though the total no. of cells employed was lower by a factor of 150.  Biotransformation of the luminogenic substrate Luciferin-H using CYP2C9-contg. enzyme bags proceeded efficiently and stably for 24 h.  CYP4Z1 is of interest because it is strongly overexpressed both in breast cancer cells and in breast cancer metastases; however, current knowledge about its catalytic properties is very limited.  Screening of CYP4Z1-contg. enzyme bags with 15 luminogenic substrates enabled us to identify two new hydroxylations and eleven ether cleavage reactions that are catalyzed by CYP4Z1.  By far the best substrate found in this study was Luciferin benzyl ether (Luciferin-BE).  On the basis of the recently published crystal structure of CYP4B1 we created a new homol. model of CYP4Z1 and performed mol. docking expts., which indicate that all active substrates show a highly similar binding geometry compared to the endogenous substrates.  The model predicts that Ser113, Ser222, Asn381, and Ser383 are key hydrogen bonding residues.  We also identified five new inhibitors of CYP4Z1: miconazole, econazole, aminobenzotriazole, tolazoline, and 1-benzylimidazole resp., with the last compd. being the most potent giving an IC50 value of 180 nM in our test system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLFJC9Ne1rhrVg90H21EOLACvtfcHk0liOWP-5RGUykA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOhsLzN&md5=f79fc6e4ec403d9198a7f8f8938bf575</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DMachalz%26aufirst%3DD.%26aulast%3DZollner%26aufirst%3DA.%26aulast%3DSorensen%26aufirst%3DE.%2BJ.%26aulast%3DWolber%26aufirst%3DG.%26aulast%3DBureik%26aufirst%3DM.%26atitle%3DEfficient%2520substrate%2520screening%2520and%2520inhibitor%2520testing%2520of%2520human%2520CYP4Z1%2520using%2520permeabilized%2520recombinant%2520fission%2520yeast%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D146%26spage%3D174%26epage%3D187%26doi%3D10.1016%2Fj.bcp.2017.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit16"><span>Promega
P450-Glo. <a href="https://www.promega.com/resources/protocols/technical-bulletins/101/p450-glo-assays-protocol/" class="extLink">https://www.promega.com/resources/protocols/technical-bulletins/101/p450-glo-assays-protocol/</a> (accessed Oct, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Promega%0AP450-Glo.+https%3A%2F%2Fwww.promega.com%2Fresources%2Fprotocols%2Ftechnical-bulletins%2F101%2Fp450-glo-assays-protocol%2F+%28accessed+Oct%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parkinson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yerino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsapple, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otradovec, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, B. W.</span></span> <span> </span><span class="NLM_article-title">An evaluation of the dilution method for identifying metabolism-dependent inhibitors of cytochrome P450 enzymes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1370</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1124/dmd.111.038596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.111.038596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=21525169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1yqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1370-1387&author=A.+Parkinsonauthor=F.+Kazmiauthor=D.+B.+Buckleyauthor=P.+Yerinoauthor=B.+L.+Parisauthor=J.+Holsappleauthor=P.+Torenauthor=S.+M.+Otradovecauthor=B.+W.+Ogilvie&title=An+evaluation+of+the+dilution+method+for+identifying+metabolism-dependent+inhibitors+of+cytochrome+P450+enzymes&doi=10.1124%2Fdmd.111.038596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An evaluation of the dilution method for identifying metabolism-dependent inhibitors of cytochrome P450 enzymes</span></div><div class="casAuthors">Parkinson, Andrew; Kazmi, Faraz; Buckley, David B.; Yerino, Phyllis; Paris, Brandy L.; Holsapple, Jeff; Toren, Paul; Otradovec, Steve M.; Ogilvie, Brian W.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1370-1387</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Metab.-dependent inhibition (MDI) of cytochrome P 450 is usually assessed in vitro by examg. whether the inhibitory potency of a drug candidate increases after a 30-min incubation with human liver microsomes (HLMs).  To augment the IC50 shift, many researchers incorporate a diln. step whereby the samples, after being preincubated for 30 min with a high concn. of HLMs (with and without NADPH), are dild. before measuring P 450 activity.  In the present study, we show that the greater IC50 shift assocd. with the diln. method is a consequence of data processing.  With the diln. method, IC50 values for direct-acting inhibitors vary with the diln. factor unless they are based on the final (postdilution) inhibitor concn., whereas the IC50 values for MDIs vary with the diln. factor unless they are based on the initial (predilution) concn.  When the latter data are processed on the final inhibitor concn., as is commonly done, the IC50 values for MDI (shifted IC50 values) decrease by the magnitude of the diln. factor.  The lower shifted IC50 values are a consequence of data processing, not enhanced P 450 inactivation.  In fact, for many MDIs, increasing the concn. of HLMs actually leads to considerably less P 450 inactivation because of inhibitor depletion and/or binding of the inhibitor to microsomes.  A true increase in P 450 inactivation and IC50 shift can be achieved by assessing MDI by a nondilution method and by decreasing the concn. of HLMs.  These results have consequences for the conduct of MDI studies and the development of cut-off criteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpORjFNdXEZzrVg90H21EOLACvtfcHk0lgf9MV-rDA9ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1yqt7g%253D&md5=f6555fc640d5b0e05b98ec36795a1984</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.038596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.038596%26sid%3Dliteratum%253Aachs%26aulast%3DParkinson%26aufirst%3DA.%26aulast%3DKazmi%26aufirst%3DF.%26aulast%3DBuckley%26aufirst%3DD.%2BB.%26aulast%3DYerino%26aufirst%3DP.%26aulast%3DParis%26aufirst%3DB.%2BL.%26aulast%3DHolsapple%26aufirst%3DJ.%26aulast%3DToren%26aufirst%3DP.%26aulast%3DOtradovec%26aufirst%3DS.%2BM.%26aulast%3DOgilvie%26aufirst%3DB.%2BW.%26atitle%3DAn%2520evaluation%2520of%2520the%2520dilution%2520method%2520for%2520identifying%2520metabolism-dependent%2520inhibitors%2520of%2520cytochrome%2520P450%2520enzymes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26spage%3D1370%26epage%3D1387%26doi%3D10.1124%2Fdmd.111.038596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henne, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VandenBrink, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aidasani, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stresser, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teffera, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonn, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B. K.</span></span> <span> </span><span class="NLM_article-title">Sequential metabolism of AMG 487, a novel CXCR3 antagonist, results in formation of quinone reactive metabolites that covalently modify CYP3A4 Cys239 and cause time-dependent inhibition of the enzyme</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1429</span>– <span class="NLM_lpage">1440</span>, <span class="refDoi"> DOI: 10.1124/dmd.112.045708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.112.045708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=22517972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOrs7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1429-1440&author=K.+R.+Henneauthor=T.+B.+Tranauthor=B.+M.+VandenBrinkauthor=D.+A.+Rockauthor=D.+K.+Aidasaniauthor=R.+Subramanianauthor=A.+K.+Masonauthor=D.+M.+Stresserauthor=Y.+Tefferaauthor=S.+G.+Wongauthor=M.+G.+Johnsonauthor=X.+Chenauthor=G.+R.+Tonnauthor=B.+K.+Wong&title=Sequential+metabolism+of+AMG+487%2C+a+novel+CXCR3+antagonist%2C+results+in+formation+of+quinone+reactive+metabolites+that+covalently+modify+CYP3A4+Cys239+and+cause+time-dependent+inhibition+of+the+enzyme&doi=10.1124%2Fdmd.112.045708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Sequential metabolism of AMG 487, a novel CXCR3 antagonist, results in formation of quinone reactive metabolites that covalently modify CYP3A4 Cys239 and cause time-dependent inhibition of the enzyme</span></div><div class="casAuthors">Henne, Kirk R.; Tran, Thuy B.; VandenBrink, Brooke M.; Rock, Dan A.; Aidasani, Divesh K.; Subramanian, Raju; Mason, Andrew K.; Stresser, David M.; Teffera, Yohannes; Wong, Simon G.; Johnson, Michael G.; Chen, Xiaoqi; Tonn, George R.; Wong, Bradley K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1429-1440</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">CYP3A4-mediated biotransformation of (R)-N-(1-(3-(4-ethoxyphenyl)-4-oxo-3,4-dihydropyrido[2,3-d]pyrimidin-2-yl)ethyl)-N-(pyridin-3-ylmethyl)-2-(4-(trifluoromethoxy)phenyl)acetamide (AMG 487) was previously shown to generate an inhibitory metabolite linked to dose- and time-dependent pharmacokinetics in humans.  Although in vitro activity loss assays failed to demonstrate CYP3A4 time-dependent inhibition (TDI) with AMG 487, its M2 phenol metabolite readily produced TDI when remaining activity was assessed using either midazolam or testosterone (KI = 0.73-0.74 μM, kinact = 0.088-0.099 min-1).  TDI investigations using an IC50 shift method successfully produced inhibition attributable to AMG 487, but only when preincubations were extended from 30 to 90 min.  The shift magnitude was ∼3× for midazolam activity, but no shift was obsd. for testosterone activity.  Subsequent partition ratio detns. conducted for M2 using recombinant CYP3A4 showed that inactivation was a relatively inefficient process (r = 36).  CYP3A4-mediated biotransformation of [3H]M2 in the presence of GSH led to identification of two new metabolites, M4 and M5, which shifted focus away from M2 being directly responsible for TDI.  M4 (hydroxylated M2) was further metabolized to form reactive intermediates that, upon reaction with GSH, produced isomeric adducts, collectively designated M5.  Incubations conducted in the presence of [18O]H2O confirmed incorporation of oxygen from O2 for the majority of M4 and M5 formed (>75%).  Further evidence of a primary role for M4 in CYP3A4 TDI was generated by protein labeling and proteolysis expts., in which M4 was found to be covalently bound to Cys239 of CYP3A4.  These investigations confirmed a primarily role for M4 in CYP3A4 inactivation, suggesting that a more complex metabolic pathway was responsible for generation of inhibitory metabolites affecting AMG 487 human pharmacokinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoskMDPPnKfuLVg90H21EOLACvtfcHk0lgf9MV-rDA9ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOrs7vO&md5=f695c134a3f06cfdf8cc1eb834ef8ccb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.045708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.045708%26sid%3Dliteratum%253Aachs%26aulast%3DHenne%26aufirst%3DK.%2BR.%26aulast%3DTran%26aufirst%3DT.%2BB.%26aulast%3DVandenBrink%26aufirst%3DB.%2BM.%26aulast%3DRock%26aufirst%3DD.%2BA.%26aulast%3DAidasani%26aufirst%3DD.%2BK.%26aulast%3DSubramanian%26aufirst%3DR.%26aulast%3DMason%26aufirst%3DA.%2BK.%26aulast%3DStresser%26aufirst%3DD.%2BM.%26aulast%3DTeffera%26aufirst%3DY.%26aulast%3DWong%26aufirst%3DS.%2BG.%26aulast%3DJohnson%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DTonn%26aufirst%3DG.%2BR.%26aulast%3DWong%26aufirst%3DB.%2BK.%26atitle%3DSequential%2520metabolism%2520of%2520AMG%2520487%252C%2520a%2520novel%2520CXCR3%2520antagonist%252C%2520results%2520in%2520formation%2520of%2520quinone%2520reactive%2520metabolites%2520that%2520covalently%2520modify%2520CYP3A4%2520Cys239%2520and%2520cause%2520time-dependent%2520inhibition%2520of%2520the%2520enzyme%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26spage%3D1429%26epage%3D1440%26doi%3D10.1124%2Fdmd.112.045708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oshikata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawase, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haramaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, H.</span></span> <span> </span><span class="NLM_article-title">Cocktail-substrate approach-based high-throughput assay for evaluation of direct and time-dependent inhibition of multiple cytochrome P450 isoforms</span>. <i>Drug Metab. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">198</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.2133/dmpk.DMPK-13-RG-093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.2133%2Fdmpk.DMPK-13-RG-093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=24172718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKjurrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=198-207&author=K.+Kozakaiauthor=Y.+Yamadaauthor=M.+Oshikataauthor=T.+Kawaseauthor=E.+Suzukiauthor=Y.+Haramakiauthor=H.+Taniguchi&title=Cocktail-substrate+approach-based+high-throughput+assay+for+evaluation+of+direct+and+time-dependent+inhibition+of+multiple+cytochrome+P450+isoforms&doi=10.2133%2Fdmpk.DMPK-13-RG-093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Cocktail-substrate approach-based high-throughput assay for evaluation of direct and time-dependent inhibition of multiple cytochrome P450 isoforms</span></div><div class="casAuthors">Kozakai, Kazumasa; Yamada, Yasuhiro; Oshikata, Motoji; Kawase, Taiji; Suzuki, Etsuko; Haramaki, Yukari; Taniguchi, Hideki</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Pharmacokinetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">198-207</span>CODEN:
                <span class="NLM_cas:coden">DMPRB8</span>;
        ISSN:<span class="NLM_cas:issn">1347-4367</span>.
    
            (<span class="NLM_cas:orgname">Japanese Society for the Study of Xenobiotics</span>)
        </div><div class="casAbstract">Avoiding drug-drug interactions (DDIs) mediated through inhibition of cytochrome P 450 (CYP) activity is highly desirable.  Direct inhibition (DI) of CYP through new chem. entities (NCEs) or time-dependent inhibition (TDI) through reactive metabolites should be elucidated at an early stage of drug discovery research.  In particular, TDI of CYP occurring through reactive metabolites may be irreversible and even sustained, causing far more serious DDIs for TDIs than for DIs.  Furthermore, it is important to ascertain whether an NCE inhibits multiple CYP isoforms.  Hence, using a cocktail-substrate approach that we previously established (in which the activity of 8 CYP isoforms is simultaneously evaluated in a single run), we evaluated the IC50 values of direct inhibitors and TDI parameters (kobs, shifted IC50, KI and kinact) of time-dependent inhibitors that affect multiple CYP isoforms.  The IC50 values for 8 CYP isoforms obtained using the cocktail-substrate approach were nearly identical to values previously reported.  The TDI parameters for CYP1A2, 2C9, 2C19, 2D6, and CYP3A4/5 obtained using the cocktail-substrate approach were also nearly identical to those obtained using a single-substrate approach.  Thus, the cocktail-substrate approach is useful for evaluating DI and TDI in the early stages of drug discovery and development processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBhvsFfgXk2rVg90H21EOLACvtfcHk0ligJhg8hjpR9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKjurrE&md5=47826700177358db6cfecee11cebc4b6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2133%2Fdmpk.DMPK-13-RG-093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2133%252Fdmpk.DMPK-13-RG-093%26sid%3Dliteratum%253Aachs%26aulast%3DKozakai%26aufirst%3DK.%26aulast%3DYamada%26aufirst%3DY.%26aulast%3DOshikata%26aufirst%3DM.%26aulast%3DKawase%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DE.%26aulast%3DHaramaki%26aufirst%3DY.%26aulast%3DTaniguchi%26aufirst%3DH.%26atitle%3DCocktail-substrate%2520approach-based%2520high-throughput%2520assay%2520for%2520evaluation%2520of%2520direct%2520and%2520time-dependent%2520inhibition%2520of%2520multiple%2520cytochrome%2520P450%2520isoforms%26jtitle%3DDrug%2520Metab.%2520Pharmacokinet.%26date%3D2014%26volume%3D29%26spage%3D198%26epage%3D207%26doi%3D10.2133%2Fdmpk.DMPK-13-RG-093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimm, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einolf, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomeir, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibert, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skordos, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonn, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Horn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">The conduct of in vitro studies to address time-dependent inhibition of drug-metabolizing enzymes: a perspective of the pharmaceutical research and manufacturers of America</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1355</span>– <span class="NLM_lpage">1370</span>, <span class="refDoi"> DOI: 10.1124/dmd.109.026716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.109.026716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=19359406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFenu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=1355-1370&author=S.+W.+Grimmauthor=H.+J.+Einolfauthor=S.+D.+Hallauthor=K.+Heauthor=H.+K.+Limauthor=K.+H.+Lingauthor=C.+Luauthor=A.+A.+Nomeirauthor=E.+Seibertauthor=K.+W.+Skordosauthor=G.+R.+Tonnauthor=R.+Van+Hornauthor=R.+W.+Wangauthor=Y.+N.+Wongauthor=T.+J.+Yangauthor=R.+S.+Obach&title=The+conduct+of+in+vitro+studies+to+address+time-dependent+inhibition+of+drug-metabolizing+enzymes%3A+a+perspective+of+the+pharmaceutical+research+and+manufacturers+of+America&doi=10.1124%2Fdmd.109.026716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The conduct of in vitro studies to address time-dependent inhibition of drug-metabolizing enzymes: a perspective of the Pharmaceutical Research and Manufacturers of America</span></div><div class="casAuthors">Grimm, Scott W.; Einolf, Heidi J.; Hall, Steven D.; He, Kan; Lim, Heng-Keang; Ling, Kah-Hiing John; Lu, Chuang; Nomeir, Amin A.; Seibert, Eleanore; Skordos, Konstantine W.; Tonn, George R.; Van Horn, Robert; Wang, Regina W.; Wong, Y. Nancy; Yang, Tian J.; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1355-1370</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Time-dependent inhibition (TDI) of cytochrome P 450 enzymes caused by new mol. entities (NMEs) is of concern because such compds. can be responsible for clin. relevant drug-drug interactions (DDI).  Although the biochem. underlying mechanism-based inactivation (MBI) of P 450 enzymes has been generally understood for several years, significant advances have been made only in the past few years regarding how in vitro time-dependent inhibition data can be used to understand and predict clin. DDI.  In this article, a team of scientists from 16 pharmaceutical research organizations that are member companies of the Pharmaceutical Research and Manufacturers of America offer a discussion of the phenomenon of TDI with emphasis on the lab. methods used in its measurement.  Results of an anonymous survey regarding pharmaceutical industry practices and strategies around TDI are reported.  Specific topics that still possess a high degree of uncertainty are raised, such as parameter ests. needed to make predictions of DDI magnitude from in vitro inactivation parameters.  A description of follow-up mechanistic expts. that can be done to characterize TDI are described.  A consensus recommendation regarding common practices to address TDI is included, the salient points of which include the use of a tiered approach wherein abbreviated assays are first used to det. whether NMEs demonstrate TDI or not, followed by more thorough inactivation studies for those that do to define the parameters needed for prediction of DDI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc57kHA3HAP7Vg90H21EOLACvtfcHk0ligJhg8hjpR9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFenu7o%253D&md5=7f74567b5c9298806f8a5cc58355b867</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fdmd.109.026716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.109.026716%26sid%3Dliteratum%253Aachs%26aulast%3DGrimm%26aufirst%3DS.%2BW.%26aulast%3DEinolf%26aufirst%3DH.%2BJ.%26aulast%3DHall%26aufirst%3DS.%2BD.%26aulast%3DHe%26aufirst%3DK.%26aulast%3DLim%26aufirst%3DH.%2BK.%26aulast%3DLing%26aufirst%3DK.%2BH.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DNomeir%26aufirst%3DA.%2BA.%26aulast%3DSeibert%26aufirst%3DE.%26aulast%3DSkordos%26aufirst%3DK.%2BW.%26aulast%3DTonn%26aufirst%3DG.%2BR.%26aulast%3DVan%2BHorn%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DR.%2BW.%26aulast%3DWong%26aufirst%3DY.%2BN.%26aulast%3DYang%26aufirst%3DT.%2BJ.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DThe%2520conduct%2520of%2520in%2520vitro%2520studies%2520to%2520address%2520time-dependent%2520inhibition%2520of%2520drug-metabolizing%2520enzymes%253A%2520a%2520perspective%2520of%2520the%2520pharmaceutical%2520research%2520and%2520manufacturers%2520of%2520America%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2009%26volume%3D37%26spage%3D1355%26epage%3D1370%26doi%3D10.1124%2Fdmd.109.026716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollenberg, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bumpus, N. N.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based inactivation of human cytochromes p450s: experimental characterization, reactive intermediates, and clinical implications</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.1021/tx7002504</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx7002504" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=189-205&author=P.+F.+Hollenbergauthor=U.+M.+Kentauthor=N.+N.+Bumpus&title=Mechanism-based+inactivation+of+human+cytochromes+p450s%3A+experimental+characterization%2C+reactive+intermediates%2C+and+clinical+implications&doi=10.1021%2Ftx7002504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-Based Inactivation of Human Cytochromes P450s: Experimental Characterization, Reactive Intermediates, and Clinical Implications</span></div><div class="casAuthors">Hollenberg, Paul F.; Kent, Ute M.; Bumpus, Namandje N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">189-205</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The P 450 type cytochromes are responsible for the metab. of a wide variety of xenobiotics and endogenous compds.  Although P 450-catalyzed reactions are generally thought to lead to detoxication of xenobiotics, the reactions can also produce reactive intermediates that can react with cellular macromols. leading to toxicity or that can react with the P450s that form them leading to irreversible (i.e., mechanism-based) inactivation.  This perspective describes the fundamentals of mechanism-based inactivation as it pertains to P 450 enzymes.  The exptl. approaches used to characterize mechanism-based inactivators are discussed, and the criteria required for a compd. to be classified as a mechanism-based inactivator are outlined.  The kinetic scheme for mechanism-based inactivation and the calcn. of the relevant kinetic consts. that describe a particular inactivation event are presented.  The structural aspects and important functional groups of several classes of mols. that have been found to impart mechanism-based inactivation upon metab. by P450s such as acetylenes, thiol-contg. compds. that include isothiocyanates, thiazolidinediones, and thiophenes, arylamines, quinones, furanocoumarins, and cyclic tertiary amines are described.  Emphasis throughout this perspective is placed on more recent findings with human P450s where the site of modification, whether it be the apoprotein or the heme moiety, and, at least in part, the identity of the reactive intermediate responsible for the loss in P 450 activity are known or inferred.  Recent advances in trapping procedures as well as new methods for identification of reactive intermediates are presented.  A variety of clin. important drugs that act as mechanism-based inactivators of P450s are discussed.  The irreversible inactivation of human P450s by these drugs has the potential for causing serious drug-drug interactions that may have severe toxicol. effects.  The clin. significance of inactivating human P450s for improving drug efficacy as well as drug safety is discussed along with the potential for exploiting mechanism-based inactivators of P450s for therapeutic benefits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC4MVcZv4wm7Vg90H21EOLACvtfcHk0lgyXKGIxMx4oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrvN&md5=0cb120c5da218dcc1e192a117bc2eac7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Ftx7002504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx7002504%26sid%3Dliteratum%253Aachs%26aulast%3DHollenberg%26aufirst%3DP.%2BF.%26aulast%3DKent%26aufirst%3DU.%2BM.%26aulast%3DBumpus%26aufirst%3DN.%2BN.%26atitle%3DMechanism-based%2520inactivation%2520of%2520human%2520cytochromes%2520p450s%253A%2520experimental%2520characterization%252C%2520reactive%2520intermediates%252C%2520and%2520clinical%2520implications%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D189%26epage%3D205%26doi%3D10.1021%2Ftx7002504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emoto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murase, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, K.</span></span> <span> </span><span class="NLM_article-title">In vitro inhibitory effect of 1-aminobenzotriazole on drug oxidations catalyzed by human cytochrome P450 enzymes: a comparison with SKF-525A and ketoconazole</span>. <i>Drug Metab. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.2133/dmpk.18.287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.2133%2Fdmpk.18.287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15618748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnt1yltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2003&pages=287-295&author=C.+Emotoauthor=S.+Muraseauthor=Y.+Sawadaauthor=B.+C.+Jonesauthor=K.+Iwasaki&title=In+vitro+inhibitory+effect+of+1-aminobenzotriazole+on+drug+oxidations+catalyzed+by+human+cytochrome+P450+enzymes%3A+a+comparison+with+SKF-525A+and+ketoconazole&doi=10.2133%2Fdmpk.18.287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro inhibitory effect of 1-aminobenzotriazole on drug oxidations catalyzed by human cytochrome P450 enzymes: A comparison with SKF-525A and ketoconazole</span></div><div class="casAuthors">Emoto, Chie; Murase, Shigeo; Sawada, Yasufusa; Jones, Barry C.; Iwasaki, Kazuhide</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Pharmacokinetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">287-295</span>CODEN:
                <span class="NLM_cas:coden">DMPRB8</span>;
        ISSN:<span class="NLM_cas:issn">1347-4367</span>.
    
            (<span class="NLM_cas:orgname">Japanese Society for the Study of Xenobiotics</span>)
        </div><div class="casAbstract">1-Aminobenzotriazole (ABT) is widely used as a non-specific inhibitor of animal cytochrome P 450 (CYP).  In the present study, the inhibitory effect of ABT was investigated on drug oxidns. catalyzed by human CYP isoforms.  This inhibitory effect was compared with that of SKF-525A, another non-specific inhibitor, and ketoconazole, a potent inhibitor of CYP3A.  Bacurovirus-expressed recombinant human CYP isoforms were used as an enzyme source.  The specific activities for human CYP isoforms are: phenacetin O-deethylation, for CYP1A2; diclofenac 4'-hydroxylation, for CYP2C9; S-mephenytoin 4'-hydroxylation, for CYP2C19; bufuralol 1'-hydroxylation, for CYP2D6; chlorzoxazone 6-hydroxylation, for CYP2E1; testosterone 6β-hydroxylation, nifedipine oxidn., and midazolam 1'-hydroxylation, for CYP3A4.  ABT inhibited both CYP1A2-dependent activity (Ki=330 μM) and CYP2E1-dependent activity (Ki=8.7 μM).  In contrast, SKF-525A weakly inhibited CYP1A2-dependent activities (46% inhibition at 1200 μM) and CYP2E1-dependent activities (65% inhibition at 1000 μM).  ABT exhibited the highest Ki value for CYP2C9-dependent diclofenac 4'-hydroxylation among those detd. by this assay (Ki=3500 μM).  Moreover, SKF-525A showed strong inhibition of CYP2D6-dependent bufuralol 1'-hydroxylation (Ki=0.043 μM).  Ketoconazole inhibited all tested drug oxidns., however, its inhibitory effect on CYP1A2-dependent activities was very weak (50% inhibition at 120 μM).  ABT, SKF-525A, and ketoconazole showed different selectivity and had a wide range of Ki values for the drug oxidns. catalyzed by human CYP enzymes.  Therefore, we conclude that inhibitory studies designed to predict the contribution of CYP enzymes to the metab. of certain compds. should be performed using multiple CYP inhibitors, such as ABT, SKF-525A, and ketoconazole.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWLwmZAR2feLVg90H21EOLACvtfcHk0lgyXKGIxMx4oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnt1yltw%253D%253D&md5=d32fc238d4bc63a2adf3dde029fab57a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.2133%2Fdmpk.18.287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2133%252Fdmpk.18.287%26sid%3Dliteratum%253Aachs%26aulast%3DEmoto%26aufirst%3DC.%26aulast%3DMurase%26aufirst%3DS.%26aulast%3DSawada%26aufirst%3DY.%26aulast%3DJones%26aufirst%3DB.%2BC.%26aulast%3DIwasaki%26aufirst%3DK.%26atitle%3DIn%2520vitro%2520inhibitory%2520effect%2520of%25201-aminobenzotriazole%2520on%2520drug%2520oxidations%2520catalyzed%2520by%2520human%2520cytochrome%2520P450%2520enzymes%253A%2520a%2520comparison%2520with%2520SKF-525A%2520and%2520ketoconazole%26jtitle%3DDrug%2520Metab.%2520Pharmacokinet.%26date%3D2003%26volume%3D18%26spage%3D287%26epage%3D295%26doi%3D10.2133%2Fdmpk.18.287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz
de Montellano, P. R.</span></span> <span> </span><span class="NLM_article-title">1-Aminobenzotriazole: A Mechanism-Based Cytochrome P450 Inhibitor and Probe of Cytochrome P450 Biology</span>. <i>Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">038</span> <span class="refDoi"> DOI: 10.4172/2161-0444.1000495</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.4172%2F2161-0444.1000495" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=P.+R.+Ortiz%0Ade+Montellano&title=1-Aminobenzotriazole%3A+A+Mechanism-Based+Cytochrome+P450+Inhibitor+and+Probe+of+Cytochrome+P450+Biology&doi=10.4172%2F2161-0444.1000495"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.4172%2F2161-0444.1000495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F2161-0444.1000495%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26atitle%3D1-Aminobenzotriazole%253A%2520A%2520Mechanism-Based%2520Cytochrome%2520P450%2520Inhibitor%2520and%2520Probe%2520of%2520Cytochrome%2520P450%2520Biology%26jtitle%3DMed.%2520Chem.%26date%3D2018%26volume%3D8%26doi%3D10.4172%2F2161-0444.1000495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bend, J. R.</span></span> <span> </span><span class="NLM_article-title">N-alkylaminobenzotriazoles as isozyme-selective suicide inhibitors of rabbit pulmonary microsomal cytochrome P-450</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=3724742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaL28XlvVKnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1986&pages=25-32&author=J.+M.+Mathewsauthor=J.+R.+Bend&title=N-alkylaminobenzotriazoles+as+isozyme-selective+suicide+inhibitors+of+rabbit+pulmonary+microsomal+cytochrome+P-450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">N-alkylaminobenzotriazoles as isozyme-selective suicide inhibitors of rabbit pulmonary microsomal cytochrome P-450</span></div><div class="casAuthors">Mathews, James M.; Bend, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-32</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">A series of N-alkylated 1-aminobenzotriazole (I) derivs. were synthesized as isoform-selective suicide inhibitors of cytochrome P 450 (P 450).  These included N-methyl-I, N-butyl-I (II), N-benzyl-I (III), and N-α-methylbenzyl-I (IV).  The suicide inhibitors showing the greatest potency and isoform selectivity were III and IV, compds. which included mol. features for P 450 inactivation (the I moiety) and similarity to benzphetamine.  I and its N-alkylated derivs. were tested as suicide inhibitors in rabbit lung microsomes, whose P 450 monooxygenase system has been well characterized in both untreated and β-naphthoflavone- or 2,3,7,8-tetrachlorodibenzo-p-dioxin-treated animals.  I (10 mM) destroyed up to 99% of the total P 450 content of lung microsomes of untreated rabbits.  At equimolar concns. (10 μM), I was less effective than the N-alkylated compds. for the inhibition of P 450 isoform 2-catalyzed benzphetamine N-demethylase (V); in fact, II, III, and IV completely inhibited V activity at this concn. and destroyed <40% of the total pulmonary P 450.  However, these compds. also inactivated 69-85% of isoform 6-catalyzed 7-ethoxyresorufin O-deethylase.  The most potent and isoform-selective suicide inhibitor prepd. was IV; at 1 μM concn., this compd. inhibited ∼80% of isoform 2-catalyzed and ∼20% of isoform 6-catalyzed monooxygenase activity, but spared P 450 isoform 6; at 2.5 μM concn., it caused a near-complete loss (96%) of V activity.  The partition ratio of IV, i.e., the molar ratio of inhibitor present to that of the P 450 destroyed, was 11, further demonstrating the potency of this compd.  Expts. with III- and phenobarbital-treated rats showed that the mechanism for suicidal inactivation of P 450 by this N-alkylated compd. was by benzyne release, the same mechanism demonstrated earlier for the parent compd., I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0KwdY-FVixrVg90H21EOLACvtfcHk0lhJ1g-MctQpeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XlvVKnu7o%253D&md5=965e34522e8ad29261b48002f60d465b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMathews%26aufirst%3DJ.%2BM.%26aulast%3DBend%26aufirst%3DJ.%2BR.%26atitle%3DN-alkylaminobenzotriazoles%2520as%2520isozyme-selective%2520suicide%2520inhibitors%2520of%2520rabbit%2520pulmonary%2520microsomal%2520cytochrome%2520P-450%26jtitle%3DMol.%2520Pharmacol.%26date%3D1986%26volume%3D30%26spage%3D25%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bend, J. R.</span></span> <span> </span><span class="NLM_article-title">N-aralkyl derivatives of 1-aminobenzotriazole as potent isozyme-selective mechanism-based inhibitors of rabbit pulmonary cytochrome P450 in vivo</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>265</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">285</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=8474011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaK3sXktVShtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1993&pages=281-285&author=J.+M.+Mathewsauthor=J.+R.+Bend&title=N-aralkyl+derivatives+of+1-aminobenzotriazole+as+potent+isozyme-selective+mechanism-based+inhibitors+of+rabbit+pulmonary+cytochrome+P450+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">N-Aralkyl derivatives of 1-aminobenzotriazole as potent isozyme-selective mechanism-based inhibitors of rabbit pulmonary cytochrome P450 in vivo</span></div><div class="casAuthors">Mathews, James M.; Bend, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">281-5</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    </div><div class="casAbstract">Two N-aralkylated (N-benzyl- and N-α-methylbenzyl-) derivs. of 1-aminobenzotriazole, a mechanism-based inhibitor of cytochrome P 450 with low isoenzyme selectivity, were previously shown to be potent and isoenzyme-selective suicide substrates for rabbit and guinea pig pulmonary P 450 in vitro (Mathews and Bend, 1986; Woodcroft et al., 1990).  These three compds. were compared as inhibitor in vivo after i.v. administration to rabbits treated with the cytochrome P 450 inducers β-naphthoflavone or phenobarbital.  By 1 h after administration of N-α-methylbenzyl-1-aminobenzotriazole (1 μmol/kg), 80% of P 450 2Bs-catalyzed benzphetamine N-demethylation in lung of β-naphthoflavone-treated rabbits was inactivated and about 35% of P 450 was lost without inhibition of P 450 1A1-catalyzed activity; at a dose of 10 or 100 μmol/kg, this compd. totally inactivated pulmonary P 450 2Bs activity while exerting minimal effects on benzphetamine N-demethylation activity (<20% inhibition) in liver of β-naphthoflavone-treated rabbits.  N-Benzyl-1-aminobenzotriazole was also an isoenzyme- and tissue-selective inhibitor of pulmonary P 450 2Bs in vivo.  Relatively high doses (100 μmol/kg) of these compds. were compared in phenobarbital-induced rabbits.  Virtually all (≥95%) of pulmonary P 450 2Bs-dependent activity was inhibited by the two N-aralkylated compds. (vs. 50% for 1-aminobenzotriazole).  At this dose, about 25% of hepatic P 450 was destroyed by all three compds., whereas 1-aminobenzotriazole and its N-benzyl and N-α-methylbenzyl derivs. inactivated 20, 50 and 85% of hepatic P 450 2Bs-selective benzphetamine N-demethylation activity, resp.  These results demonstrate that N-α-methylbenzyl-1-aminobenzotriazole and N-benzyl-1-aminobenzotriazole, but not 1-aminobenzotriazole, are highly selective for the inactivation of pulmonary P 450 2Bs in rabbits in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3KXaKWiR_hbVg90H21EOLACvtfcHk0lhJ1g-MctQpeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktVShtrc%253D&md5=18c12d1fe7fbc759ab10f96454eb72df</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMathews%26aufirst%3DJ.%2BM.%26aulast%3DBend%26aufirst%3DJ.%2BR.%26atitle%3DN-aralkyl%2520derivatives%2520of%25201-aminobenzotriazole%2520as%2520potent%2520isozyme-selective%2520mechanism-based%2520inhibitors%2520of%2520rabbit%2520pulmonary%2520cytochrome%2520P450%2520in%2520vivo%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1993%26volume%3D265%26spage%3D281%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woodcroft, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczepan, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knickle, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bend, J. R.</span></span> <span> </span><span class="NLM_article-title">Three N-aralkylated derivatives of 1-aminobenzotriazole as potent and isozyme-selective, mechanism-based inhibitors of guinea pig pulmonary cytochrome P-450 in vitro</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1037</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1981509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaK3MXpsVGksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1990&pages=1031-1037&author=K.+J.+Woodcroftauthor=E.+W.+Szczepanauthor=L.+C.+Knickleauthor=J.+R.+Bend&title=Three+N-aralkylated+derivatives+of+1-aminobenzotriazole+as+potent+and+isozyme-selective%2C+mechanism-based+inhibitors+of+guinea+pig+pulmonary+cytochrome+P-450+in+vitro"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Three N-aralkylated derivatives of 1-aminobenzotriazole as potent and isozyme-selective, mechanism-based inhibitors of guinea pig pulmonary cytochrome P-450 in vitro</span></div><div class="casAuthors">Woodcroft, Kimberley J.; Szczepan, Edward W.; Knickle, Leah C.; Bend, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1031-7</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    </div><div class="casAbstract">The potency and cytochrome P 450 (P 450) isoenzyme selectivity of 1-aminobenzotriazole (ABT) and three of its N-aralkylated analog N-benzyl-1-aminobenzotriazole (BBT), N-α-methylbenzyl-1-aminobenzotriazole (αMB), and the newly synthesized N-α-ethylbenzyl-1-aminobenzotriazole (αEB), as mechanism-based inhibitors were compared in pulmonary microsomes of untreated and β-naphthoflavone (β-NF)-induced guinea pigs.  All four compds. were suicide substrates for pulmonary P 450, resulting in the loss of spectrally assayed hemoprotein (up to 50%).  Monooxygenase activities were measured with isoenzyme-selective/specific substrates; the O-dealkylation of 7-pentoxyresorufin (PRF) for the guinea pig ortholog of rabbit P-450IIB4, the O-deethylation of 7-ethoxyresorufin for P-450Ia1, and the N-hydroxylation of the arom. amine 4-aminobiphenyl for P-450IVB1.  BBT, αMB, and αEB were selective for the suicidal inhibition of P-450IIB4; for example, 1 μM αMB inactivated 95% of P-450IIB4-, and approx. 10% of P-450IA1- and IVB1-catalyzed, activity in microsomes from β-NF-induced lungs.  Isoenzyme selectivity was approx. the same for αEB and slightly lower for BBT, which inactivated relatively more P-450IA1.  At low concns., 1 and 10 μM, resp., ABT preferentially inactivated P.  450IVB1, consistent with the efficient N-hydroxylation of arom. amines by this form of P 450.  αEB also was shown to efficiently inactivate P-450IIB4-catalyzed PRF activity in microsomes prepd. from liver of phenobarbital-induced guinea pigs.  Collectively, the data indicated that N-aralkylated derivs. of ABT are isoenzyme-selective, mechanism-based inhibitors in guinea pig lung, and that the order of potency is αMB = αEB > BBT > ABT for P-450IIB4.  This study also emphasized that the pulmonary P 450 monooxygenase systems of the guinea pig and rabbit are functionally very similar.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-puiCfrzFQLVg90H21EOLACvtfcHk0lhJ1g-MctQpeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXpsVGksA%253D%253D&md5=4dab44e28c4affff7104e1dab3a4b47c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWoodcroft%26aufirst%3DK.%2BJ.%26aulast%3DSzczepan%26aufirst%3DE.%2BW.%26aulast%3DKnickle%26aufirst%3DL.%2BC.%26aulast%3DBend%26aufirst%3DJ.%2BR.%26atitle%3DThree%2520N-aralkylated%2520derivatives%2520of%25201-aminobenzotriazole%2520as%2520potent%2520and%2520isozyme-selective%252C%2520mechanism-based%2520inhibitors%2520of%2520guinea%2520pig%2520pulmonary%2520cytochrome%2520P-450%2520in%2520vitro%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1990%26volume%3D18%26spage%3D1031%26epage%3D1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kent, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bend, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gage, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenberg, P. F.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based inactivation of cytochrome P450 2B1 by N-benzyl-1-aminobenzotriazole</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">600</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/tx960184o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx960184o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaK2sXjsVOjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1997&pages=600-608&author=U.+M.+Kentauthor=J.+R.+Bendauthor=B.+A.+Chamberlinauthor=D.+A.+Gageauthor=P.+F.+Hollenberg&title=Mechanism-based+inactivation+of+cytochrome+P450+2B1+by+N-benzyl-1-aminobenzotriazole&doi=10.1021%2Ftx960184o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based inactivation of cytochrome P450 2B1 by N-benzyl-1-aminobenzotriazole</span></div><div class="casAuthors">Kent, Ute M.; Bend, John R.; Chamberlin, Beverly A.; Gage, Douglas A.; Hollenberg, Paul F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">600-608</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The kinetics of inactivation of cytochrome P 450 2B1, the major phenobarbital inducible rat hepatic P 450, by N-benzyl-1-aminobenzotriazole (BBT) were characterized.  Purified, reconstituted P 450 2B1 7-ethoxy-4-(trifluoromethyl)coumarin (7-EFC) O-deethylase activity was inhibited by BBT in a mechanism-based manner.  The loss of O-deethylase activity followed pseudo-first-order kinetics and was NADPH and BBT dependent.  After a 5 min incubation, greater than 90% of the 2B1 activity was lost, whereas more than 70% of the ability of the reduced enzyme to bind CO was maintained.  Inclusion of 10 mM glutathione in the inactivation reaction lowered the rate of inactivation (kinactivation) and increased the partition ratio without significantly affecting the inactivator concn. required for half-maximal inactivation (KI).  The maximal rate const. for inactivation at 23° was 0.24 min-1 without and 0.15 min-1 with glutathione.  The apparent KI was 2 μM in both cases.  The extrapolated partition ratios were 4 and 9 without and with 10 mM glutathione, resp.  Consistent with mechanism-based inactivation, the loss of 7-EFC O-deethylase activity was irreversible, was not due to product inhibition, was saturable, and could be slowed by including increasing concns. of competing substrate.  However, the inactivated P 450 2B1 was still able to metabolize substrate if iodosobenzene was used as an alternate oxidant.  Inactivation of 2B1 with either N-[14C]-7-benzyl-1-aminobenzotriazole (BBT) or N-benzyl-1-amino-[14C]-2,3-benzotriazole resulted in the incorporation of covalent radiolabel into the apoprotein.  The stoichiometry of labeled metabolite adduct to protein was approx. 0.4:1 in both cases.  Identification of metabolites revealed the formation of and a new metabolite during catalysis of BBT by P 450 2B1.  Together, these data suggest that P 450 2B1 could be inactivated and labeled by more than one metabolite.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXT_y46KKqqLVg90H21EOLACvtfcHk0lhJ1g-MctQpeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjsVOjtLg%253D&md5=b9a41820c71af919f7dc57b4d505b74b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Ftx960184o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx960184o%26sid%3Dliteratum%253Aachs%26aulast%3DKent%26aufirst%3DU.%2BM.%26aulast%3DBend%26aufirst%3DJ.%2BR.%26aulast%3DChamberlin%26aufirst%3DB.%2BA.%26aulast%3DGage%26aufirst%3DD.%2BA.%26aulast%3DHollenberg%26aufirst%3DP.%2BF.%26atitle%3DMechanism-based%2520inactivation%2520of%2520cytochrome%2520P450%25202B1%2520by%2520N-benzyl-1-aminobenzotriazole%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1997%26volume%3D10%26spage%3D600%26epage%3D608%26doi%3D10.1021%2Ftx960184o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of cytochrome P450 4B1 (CYP4B1) monooxygenase complexed with octane discloses several structural adaptations for omega-hydroxylation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">5610</span>– <span class="NLM_lpage">5621</span>, <span class="refDoi"> DOI: 10.1074/jbc.M117.775494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1074%2Fjbc.M117.775494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=28167536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltl2nsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=5610-5621&author=M.+H.+Hsuauthor=B.+R.+Baerauthor=A.+E.+Rettieauthor=E.+F.+Johnson&title=The+crystal+structure+of+cytochrome+P450+4B1+%28CYP4B1%29+monooxygenase+complexed+with+octane+discloses+several+structural+adaptations+for+omega-hydroxylation&doi=10.1074%2Fjbc.M117.775494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The Crystal Structure of Cytochrome P450 4B1 (CYP4B1) Monooxygenase Complexed with Octane Discloses Several Structural Adaptations for ω-Hydroxylation</span></div><div class="casAuthors">Hsu, Mei-Hui; Baer, Brian R.; Rettie, Allan E.; Johnson, Eric F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5610-5621</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">P 450 family 4 fatty acid ω-hydroxylases preferentially oxygenate primary C-H bonds over adjacent, energetically favored secondary C-H bonds, but the mechanism explaining this intriguing preference is unclear.  To this end, the structure of rabbit P 450 4B1 complexed with its substrate octane was detd. by X-ray crystallog. to define features of the active site that contribute to a preference for ω-hydroxylation.  The structure indicated that octane is bound in a narrow active-site cavity that limits access of the secondary C-H bond to the reactive intermediate.  A highly conserved sequence motif on helix I contributes to positioning the terminal carbon of octane for ω-hydroxylation.  Glu-310 of this motif auto-catalytically forms an ester bond with the heme 5-Me, and the immobilized Glu-310 contributes to substrate positioning.  The preference for ω-hydroxylation was decreased in an E310A mutant having a shorter side chain, but the overall rates of metab. were retained.  E310D and E310Q substitutions having longer side chains exhibit lower overall rates, likely due to higher conformational entropy for these residues, but they retained high preferences for octane ω-hydroxylation.  Sequence comparisons indicated that active-site residues constraining octane for ω-hydroxylation are conserved in family 4 P450s.  Moreover, the heme 7-propionate is positioned in the active site and provides addnl. restraints on substrate binding.  In conclusion, P 450 4B1 exhibits structural adaptations for ω-hydroxylation that include changes in the conformation of the heme and changes in a highly conserved helix I motif that is assocd. with selective oxygenation of unactivated primary C-H bonds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtBwKWqT0GV7Vg90H21EOLACvtfcHk0ljoKfWPJuwUvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltl2nsLw%253D&md5=3cbaf4a1e166b765b6d9248a2a421495</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.775494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.775494%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DM.%2BH.%26aulast%3DBaer%26aufirst%3DB.%2BR.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26atitle%3DThe%2520crystal%2520structure%2520of%2520cytochrome%2520P450%25204B1%2520%2528CYP4B1%2529%2520monooxygenase%2520complexed%2520with%2520octane%2520discloses%2520several%2520structural%2520adaptations%2520for%2520omega-hydroxylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D5610%26epage%3D5621%26doi%3D10.1074%2Fjbc.M117.775494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zöllner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pistorius, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bureik, M.</span></span> <span> </span><span class="NLM_article-title">Human CYP4Z1 catalyzes the in-chain hydroxylation of lauric acid and myristic acid</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1515/BC.2009.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1515%2FBC.2009.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=19090726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BD1MzhsVenug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=313-317&author=A.+Z%C3%B6llnerauthor=C.+A.+Draganauthor=D.+Pistoriusauthor=R.+Mullerauthor=H.+B.+Bodeauthor=F.+T.+Petersauthor=H.+H.+Maurerauthor=M.+Bureik&title=Human+CYP4Z1+catalyzes+the+in-chain+hydroxylation+of+lauric+acid+and+myristic+acid&doi=10.1515%2FBC.2009.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Human CYP4Z1 catalyzes the in-chain hydroxylation of lauric acid and myristic acid</span></div><div class="casAuthors">Zollner Andy; Dragan Calin-Aurel; Pistorius Dominik; Muller Rolf; Bode Helge B; Peters Frank T; Maurer Hans H; Bureik Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Biological chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">313-7</span>
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    </div><div class="casAbstract">Overexpression of human CYP4Z1, a cytochrome P450 enzyme, has been correlated with poor prognosis in human cancer.  However, its catalytic properties are not yet known.  We expressed this P450 in Schizosaccharomyces pombe and demonstrate by whole-cell biotransformation assays CYP4Z1-dependent in-chain hydroxylation of lauric and myristic acid, which in both cases leads to the formation of four different monohydroxylated products at positions omega-2, omega-3, omega-4, and omega-5, respectively.  The CYP4Z1-expressing fission yeast should be a new valuable tool for testing cancer drugs or for the development of new prodrug strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQRV_GXi11L-WBAiYWy6n3SfW6udTcc2ebiGjd6FS9eUbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzhsVenug%253D%253D&md5=3cf9b55ac56400cb586afe9315c7320b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1515%2FBC.2009.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2009.030%26sid%3Dliteratum%253Aachs%26aulast%3DZ%25C3%25B6llner%26aufirst%3DA.%26aulast%3DDragan%26aufirst%3DC.%2BA.%26aulast%3DPistorius%26aufirst%3DD.%26aulast%3DMuller%26aufirst%3DR.%26aulast%3DBode%26aufirst%3DH.%2BB.%26aulast%3DPeters%26aufirst%3DF.%2BT.%26aulast%3DMaurer%26aufirst%3DH.%2BH.%26aulast%3DBureik%26aufirst%3DM.%26atitle%3DHuman%2520CYP4Z1%2520catalyzes%2520the%2520in-chain%2520hydroxylation%2520of%2520lauric%2520acid%2520and%2520myristic%2520acid%26jtitle%3DBiol.%2520Chem.%26date%3D2009%26volume%3D390%26spage%3D313%26epage%3D317%26doi%3D10.1515%2FBC.2009.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato-Watanabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kametani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomishima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kameo, K.</span></span> <span> </span><span class="NLM_article-title">HET0016, a potent and selective inhibitor of 20-HETE synthesizing enzyme</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0704101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1038%2Fsj.bjp.0704101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=11375247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktlSjurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2001&pages=325-329&author=N.+Miyataauthor=K.+Taniguchiauthor=T.+Sekiauthor=T.+Ishimotoauthor=M.+Sato-Watanabeauthor=Y.+Yasudaauthor=M.+Doiauthor=S.+Kametaniauthor=Y.+Tomishimaauthor=T.+Uekiauthor=M.+Satoauthor=K.+Kameo&title=HET0016%2C+a+potent+and+selective+inhibitor+of+20-HETE+synthesizing+enzyme&doi=10.1038%2Fsj.bjp.0704101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">HET0016, a potent and selective inhibitor of 20-HETE synthesizing enzyme</span></div><div class="casAuthors">Miyata, Noriyuki; Taniguchi, Kazuo; Seki, Takayuki; Ishimoto, Tsuyoshi; Sato-Watanabe, Mariko; Yasuda, Yoshiko; Doi, Mariko; Kametani, Shunichi; Tomishima, Yasumitsu; Ueki, Tomokazu; Sato, Masakazu; Kameo, Kazuya</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">325-329</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The present study examd. the inhibitory effects of N-Hydroxy-N'-(4-butyl-2-methylphenyl)-formamidine (HET0016) on the renal metab. of arachidonic acid by cytochrome P 450 (CYP) enzymes.  HET0016 exhibited a high degree of selectivity in inhibiting the formation of 20-hydroxy-5,8,11,14-eicosatetraenoic acid (20-HETE) in rat renal microsomes.  The IC50 value averaged 35±4 nM, whereas the IC50 value for inhibition of the formation of epoxyeicosatrienoic acids by HET0016 averaged 2800±300 nM.  In human renal microsomes, HET0016 potently inhibited the formation of 20-HETE with an IC50 value of 8.9±2.7 nM.  Higher concns. of HET0016 also inhibited the CYP2C9, CYP2D6 and CYP3A4-catalyzed substrates oxidn. with IC50 values of 3300, 83,900 and 71,000 nM.  The IC50 value for HET0016 on cyclooxygenase activity was 2300 nM.  These results indicate that HET0016 is a potent and selective inhibitor of CYP enzymes responsible for the formation of 20-HETE in man and rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn4OFmcinlNbVg90H21EOLACvtfcHk0ljoKfWPJuwUvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktlSjurs%253D&md5=1ab62d8d96df94e800be1307036fa9d1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0704101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0704101%26sid%3Dliteratum%253Aachs%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DTaniguchi%26aufirst%3DK.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DIshimoto%26aufirst%3DT.%26aulast%3DSato-Watanabe%26aufirst%3DM.%26aulast%3DYasuda%26aufirst%3DY.%26aulast%3DDoi%26aufirst%3DM.%26aulast%3DKametani%26aufirst%3DS.%26aulast%3DTomishima%26aufirst%3DY.%26aulast%3DUeki%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DM.%26aulast%3DKameo%26aufirst%3DK.%26atitle%3DHET0016%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%252020-HETE%2520synthesizing%2520enzyme%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2001%26volume%3D133%26spage%3D325%26epage%3D329%26doi%3D10.1038%2Fsj.bjp.0704101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakinuma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kameo, K.</span></span> <span> </span><span class="NLM_article-title">Pyrazole and isoxazole derivatives as new, potent, and selective 20-hydroxy-5,8,11,14-eicosatetraenoic acid synthase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">5416</span>– <span class="NLM_lpage">5427</span>, <span class="refDoi"> DOI: 10.1021/jm020557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVClt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5416-5427&author=T.+Nakamuraauthor=M.+Satoauthor=H.+Kakinumaauthor=N.+Miyataauthor=K.+Taniguchiauthor=K.+Bandoauthor=A.+Kodaauthor=K.+Kameo&title=Pyrazole+and+isoxazole+derivatives+as+new%2C+potent%2C+and+selective+20-hydroxy-5%2C8%2C11%2C14-eicosatetraenoic+acid+synthase+inhibitors&doi=10.1021%2Fjm020557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrazole and Isoxazole Derivatives as New, Potent, and Selective 20-Hydroxy-5,8,11,14-eicosatetraenoic Acid Synthase Inhibitors</span></div><div class="casAuthors">Nakamura, Toshio; Sato, Masakazu; Kakinuma, Hiroyuki; Miyata, Noriyuki; Taniguchi, Kazuo; Bando, Kagumi; Koda, Ayumi; Kameo, Kazuya</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5416-5427</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In a previous paper, we reported the N-hydroxyformamidine deriv. HET0016 as a potent and selective 20-HETE synthase inhibitor.  Despite its attraction as a potential therapeutic agent for cerebral diseases, the prepn. of an injectable formulation of HET0016 was limited by its poor soly. under neutral conditions and instability under acidic conditions.  The instability of HET0016 in acidic conditions is due to the N-hydroxyformamidine moiety, which is considered to be essential for the potent and selective activity seen in our previous study.  The activity was maintained when the N-hydroxyformamidine moiety was replaced by an imidazole ring, but this was assocd. with a loss of selectivity for cytochrome P450s (CYPs).  However, other azole derivs. such as isoxazole deriv. (IC50 value 38±10 nM) and pyrazole deriv. (IC50 value 23±12 nM) showed potent and selective activities with improved stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq52rANjA51OrVg90H21EOLACvtfcHk0lhiVFYveT-Tuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVClt7w%253D&md5=14f5b92c205e207b0bcccbed608f17c5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm020557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020557k%26sid%3Dliteratum%253Aachs%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DM.%26aulast%3DKakinuma%26aufirst%3DH.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DTaniguchi%26aufirst%3DK.%26aulast%3DBando%26aufirst%3DK.%26aulast%3DKoda%26aufirst%3DA.%26aulast%3DKameo%26aufirst%3DK.%26atitle%3DPyrazole%2520and%2520isoxazole%2520derivatives%2520as%2520new%252C%2520potent%252C%2520and%2520selective%252020-hydroxy-5%252C8%252C11%252C14-eicosatetraenoic%2520acid%2520synthase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5416%26epage%3D5427%26doi%3D10.1021%2Fjm020557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliw, E. H.</span></span> <span> </span><span class="NLM_article-title">cDna cloning and expression of CYP4F12, a novel human cytochrome P450</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1006/bbrc.2000.4191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1006%2Fbbrc.2000.4191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=11162607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1alsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2001&pages=892-897&author=J.+Bylundauthor=M.+Bylundauthor=E.+H.+Oliw&title=cDna+cloning+and+expression+of+CYP4F12%2C+a+novel+human+cytochrome+P450&doi=10.1006%2Fbbrc.2000.4191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">cDNA Cloning and Expression of CYP4F12, a Novel Human Cytochrome P450</span></div><div class="casAuthors">Bylund, Johan; Bylund, Mattias; Oliw, Ernst H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-897</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The cDNA of a novel human cytochrome P 450 was cloned from human liver by reverse transcription-polymerase chain reaction and designated CYP4F12.  The open reading frame coded for 524 amino acids, and the sequence could be aligned with 78-83% amino acid identity to the four human CYP4F enzymes (CYP4F2, CYP4F3, CYP4F8 and CYP4F11).  Northern blot anal. suggested three major transcripts of CYP4F12, which were detected in liver, kidney, colon, small intestine and heart.  The CYP4F12 gene contained 13 exons and was located at chromosome 19p13.1.  CYP4F12, expressed in yeast, oxidized arachidonic acid to 18-hydroxyarachidonic acid, and the ω-side chain of two stable prostaglandin (PG) H2 analogs (11,9-epoxymethano-PGH2 and 9,11-diazo-15-deoxy-PGH2).  CYP4F12 oxidized the ω-side chain of leukotriene B4, PGE2, PGF2α, PGH2, and 9,11-epoxymethano-PGH2 poorly.  Several CYP4F enzymes are important ω1- and ω2-hydroxylases of eicosanoids.  The physiol. function of CYP4F12 merits further investigation.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7lpkgnH2LSbVg90H21EOLACvtfcHk0lhiVFYveT-Tuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1alsA%253D%253D&md5=910c810a8b56a3444d062bc7d7886c89</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.2000.4191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.2000.4191%26sid%3Dliteratum%253Aachs%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DBylund%26aufirst%3DM.%26aulast%3DOliw%26aufirst%3DE.%2BH.%26atitle%3DcDna%2520cloning%2520and%2520expression%2520of%2520CYP4F12%252C%2520a%2520novel%2520human%2520cytochrome%2520P450%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2001%26volume%3D280%26spage%3D892%26epage%3D897%26doi%3D10.1006%2Fbbrc.2000.4191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hidestrand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingelman-Sundberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliw, E. H.</span></span> <span> </span><span class="NLM_article-title">Identification of CYP4F8 in human seminal vesicles as a prominent 19-hydroxylase of prostaglandin endoperoxides</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">21844</span>– <span class="NLM_lpage">21849</span>, <span class="refDoi"> DOI: 10.1074/jbc.M001712200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1074%2Fjbc.M001712200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10791960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD3cXltFGjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=21844-21849&author=J.+Bylundauthor=M.+Hidestrandauthor=M.+Ingelman-Sundbergauthor=E.+H.+Oliw&title=Identification+of+CYP4F8+in+human+seminal+vesicles+as+a+prominent+19-hydroxylase+of+prostaglandin+endoperoxides&doi=10.1074%2Fjbc.M001712200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of CYP4F8 in human seminal vesicles as a prominent 19-hydroxylase of prostaglandin endoperoxides</span></div><div class="casAuthors">Bylund, Johan; Hidestrand, Mats; Ingelman-Sundberg, Magnus; Oliw, Ernst H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">21844-21849</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A novel cytochrome P 450, CYP4F8, was recently cloned from human seminal vesicles.  CYP4F8 was expressed in yeast.  Recombinant CYP4F8 oxygenated arachidonic acid to (18R)-hydroxyarachidonate, whereas prostaglandin (PG) D2, PGE1, PGE2, PGF2α, and leukotriene B4 appeared to be poor substrates.  Three stable PGH2 analogs, 9,11-epoxymethano-PGH2 (U-44069), 11,9-epoxymethano-PGH2 (U-46619), and 9,11-diazo-15-deoxy-PGH2 (U-51605) were rapidly metabolized by ω2- and ω3-hydroxylation.  U-44069 was oxygenated with a Vmax of ∼260 pmol min-1 pmol P 450-1 and a Km of ∼7 μM.  PGH2 decomps. mainly to PGE2 in buffer and to PGF2α by redn. with SnCl2.  CYP4F8 metabolized PGH2 to 19-hydroxy-PGH2, which decompd. to 19-hydroxy-PGE2 in buffer and could be reduced to 19-hydroxy-PGF2α with SnCl2.  18-Hydroxy metabolites were also formed (∼17%).  PGH1 was metabolized to 19- and 18-hydroxy-PGH1 in the same way.  Microsomes of human seminal vesicles oxygenated arachidonate, U-44069, U-46619, U-51605, and PGH2, similar to CYP4F8.  (19R)-Hydroxy-PGE1 and (19R)-hydroxy-PGE2 are the main prostaglandins of human seminal fluid.  We propose that they are formed by CYP4F8-catalyzed ω2-hydroxylation of PGH1 and PGH2 in the seminal vesicles and isomerization to (19R)-hydroxy-PGE by PGE synthase.  CYP4F8 is the first described hydroxylase with specificity and catalytic competence for prostaglandin endoperoxides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4ft2M9da0IbVg90H21EOLACvtfcHk0lh6X9Oatn4wcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXltFGjt7g%253D&md5=4f4b78446a6fe9b40c9b27b803e19bc8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M001712200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M001712200%26sid%3Dliteratum%253Aachs%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DHidestrand%26aufirst%3DM.%26aulast%3DIngelman-Sundberg%26aufirst%3DM.%26aulast%3DOliw%26aufirst%3DE.%2BH.%26atitle%3DIdentification%2520of%2520CYP4F8%2520in%2520human%2520seminal%2520vesicles%2520as%2520a%2520prominent%252019-hydroxylase%2520of%2520prostaglandin%2520endoperoxides%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D21844%26epage%3D21849%26doi%3D10.1074%2Fjbc.M001712200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">U.S. Food & Drug Administration</span>. <a href="https://www.fda.gov/drugs/drug-interactions-labeling/drug-development-and-drug-interactions-table-substrates-inhibitors-and-inducers" class="extLink">https://www.fda.gov/drugs/drug-interactions-labeling/drug-development-and-drug-interactions-table-substrates-inhibitors-and-inducers</a> (accessed Oct, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Food+%26+Drug+Administration.+https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fdrug-interactions-labeling%2Fdrug-development-and-drug-interactions-table-substrates-inhibitors-and-inducers+%28accessed+Oct%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Breemen, R. B.</span></span> <span> </span><span class="NLM_article-title">High-throughput cytochrome P450 cocktail inhibition assay for assessing drug-drug and drug-botanical interactions</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1670</span>– <span class="NLM_lpage">1678</span>, <span class="refDoi"> DOI: 10.1124/dmd.115.065987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.115.065987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=26285764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28XislGhtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=1670-1678&author=G.+Liauthor=K.+Huangauthor=D.+Nikolicauthor=R.+B.+van+Breemen&title=High-throughput+cytochrome+P450+cocktail+inhibition+assay+for+assessing+drug-drug+and+drug-botanical+interactions&doi=10.1124%2Fdmd.115.065987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput cytochrome P450 cocktail inhibition assay for assessing drug-drug and drug-botanical interactions</span></div><div class="casAuthors">Li, Guannan; Huang, Ke; Nikolic, Dejan; van Breemen, Richard B.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1670-1678</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Detection of drug-drug interactions is essential during the early stages of drug discovery and development, and the understanding of drug-botanical interactions is important for the safe use of botanical dietary supplements.  Among the different forms of drug interactions that are known, inhibition of cytochrome P 450 (P 450) enzymes is the most common cause of drug-drug or drug-botanical interactions.  Therefore, a rapid and comprehensive mass spectrometry-based in vitro high-throughput P 450 cocktail inhibition assay was developed that uses 10 substrates simultaneously against nine CYP isoforms.  Including probe substrates for CYP1A2, CYP2A6, CYP2B6, CYP2C8, CYP2C9, CYP2C19, CYP2D6, CYP2E1, and two probes targeting different binding sites of CYP3A4/5, this cocktail simultaneously assesses at least as many P 450 enzymes as previous assays while remaining among the fastest due to short incubation times and rapid anal. using ultrahigh pressure liq. chromatog.-tandem mass spectrometry.  The method was validated using known inhibitors of each P 450 enzyme and then shown to be useful not only for single-compd. testing but also for the evaluation of potential drug-botanical interactions using the botanical dietary supplement licorice (Glycyrrhiza glabra) as an example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGRNxN4kxt-LVg90H21EOLACvtfcHk0lh6X9Oatn4wcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislGhtr8%253D&md5=386d973b1dd84f04d5b8b14415aea2bd</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.065987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.065987%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DNikolic%26aufirst%3DD.%26aulast%3Dvan%2BBreemen%26aufirst%3DR.%2BB.%26atitle%3DHigh-throughput%2520cytochrome%2520P450%2520cocktail%2520inhibition%2520assay%2520for%2520assessing%2520drug-drug%2520and%2520drug-botanical%2520interactions%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26spage%3D1670%26epage%3D1678%26doi%3D10.1124%2Fdmd.115.065987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corcos, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salaun, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amet, Y.</span></span> <span> </span><span class="NLM_article-title">Metabolism of eicosapentaenoic and docosahexaenoic acids by recombinant human cytochromes P450</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>471</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2008.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.abb.2008.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=18206980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislKktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=471&publication_year=2008&pages=116-125&author=M.+Ferauthor=Y.+Dreanoauthor=D.+Lucasauthor=L.+Corcosauthor=J.+P.+Salaunauthor=F.+Berthouauthor=Y.+Amet&title=Metabolism+of+eicosapentaenoic+and+docosahexaenoic+acids+by+recombinant+human+cytochromes+P450&doi=10.1016%2Fj.abb.2008.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of eicosapentaenoic and docosahexaenoic acids by recombinant human cytochromes P450</span></div><div class="casAuthors">Fer, Maude; Dreano, Yvonne; Lucas, Daniele; Corcos, Laurent; Salauen, Jean-Pierre; Berthou, Francois; Amet, Yolande</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">471</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">116-125</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Epoxidn. and hydroxylation of arachidonic acid (AA) are both catalyzed by cytochromes P 450s (CYPs).  The oxidized metabolites are known to be involved in the regulation of vascular tone and renal function.  By using a panel of 15 human recombinant CYPs, this study demonstrates that other polyunsatd. long-chain fatty acids (PUFA-LC), esp. the ω3 fatty acids eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA), are also epoxidized.  The regioselectivity of epoxidn. of four PUFA-LC by CYPs was investigated.  Among the several CYPs tested, CYP2C9/2C19 and 1A2 were the most efficient in EPA and DHA epoxidns.  It ensued that 10 μM of these two ω3 fatty acids decreased by more than 80% and 60%, resp., the formation by CYP2C9 of AA-epoxidized derivs.  These findings suggest that some physiol. effects of ω3 fatty acids may be due to a shift in the generation of active epoxidized metabolites of AA through CYP-mediated catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLiRIElYJIhLVg90H21EOLACvtfcHk0lh6X9Oatn4wcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislKktbc%253D&md5=b5c8e9cfd9f7ccdbbcc998eb0e015daa</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2008.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2008.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DFer%26aufirst%3DM.%26aulast%3DDreano%26aufirst%3DY.%26aulast%3DLucas%26aufirst%3DD.%26aulast%3DCorcos%26aufirst%3DL.%26aulast%3DSalaun%26aufirst%3DJ.%2BP.%26aulast%3DBerthou%26aufirst%3DF.%26aulast%3DAmet%26aufirst%3DY.%26atitle%3DMetabolism%2520of%2520eicosapentaenoic%2520and%2520docosahexaenoic%2520acids%2520by%2520recombinant%2520human%2520cytochromes%2520P450%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2008%26volume%3D471%26spage%3D116%26epage%3D125%26doi%3D10.1016%2Fj.abb.2008.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaspera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totah, R. A.</span></span> <span> </span><span class="NLM_article-title">Epoxyeicosatrienoic acids: formation, metabolism and potential role in tissue physiology and pathophysiology</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">757</span>– <span class="NLM_lpage">771</span>, <span class="refDoi"> DOI: 10.1517/17425250902932923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1517%2F17425250902932923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=19505190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsFamsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=757-771&author=R.+Kasperaauthor=R.+A.+Totah&title=Epoxyeicosatrienoic+acids%3A+formation%2C+metabolism+and+potential+role+in+tissue+physiology+and+pathophysiology&doi=10.1517%2F17425250902932923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Epoxyeicosatrienoic acids: formation, metabolism and potential role in tissue physiology and pathophysiology</span></div><div class="casAuthors">Kaspera, Ruediger; Totah, Rheem A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">757-771</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  CYP enzymes from the CYP2C and CYP2J subfamilies metabolize arachidonic acid in a regiospecific and stereoselective manner to eight epoxyeicosatrienoic acids (EETs).  Various EETs have been detected in the liver, as well as in many extrahepatic tissues, and have been implicated in numerous physiol. functions from cell signaling to vasodilation and angiogenesis.  This report reviews the sites of expression and activity of arachidonic acid epoxygenase CYP isoforms, as well as the physiol. role and metab. of EETs in various extrahepatic tissues.  Possible functions of EETs in tissue pathophysiol. and implications as potential drug targets are also discussed.  The most recent primary research literature on EET forming enzymes and the new physiol. functions of EETs in various tissues were reviewed.  Epoxyeicosatrienoic acids are important in maintaining the homeostasis and in responding to stress in various extra hepatic tissues.  It is not clear whether these effects are owing to EETs acting on a universal receptor or through a mechanism involving a second messenger.  A better understanding of the regulation of EET levels and their mechanism of action on various receptors will accelerate research aiming at developing therapeutic agents that target EET formation or metab. pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9X4GneX5N-LVg90H21EOLACvtfcHk0lgQ7Vve3Gub6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsFamsb8%253D&md5=2b011dc321a3b9a2bcd55275bd9437cd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F17425250902932923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425250902932923%26sid%3Dliteratum%253Aachs%26aulast%3DKaspera%26aufirst%3DR.%26aulast%3DTotah%26aufirst%3DR.%2BA.%26atitle%3DEpoxyeicosatrienoic%2520acids%253A%2520formation%252C%2520metabolism%2520and%2520potential%2520role%2520in%2520tissue%2520physiology%2520and%2520pathophysiology%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2009%26volume%3D5%26spage%3D757%26epage%3D771%26doi%3D10.1517%2F17425250902932923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesaros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuetz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donner, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falck, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capdevila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, D. A.</span></span> <span> </span><span class="NLM_article-title">CYP3A4 mediates growth of estrogen receptor-positive breast cancer cells in part by inducing nuclear translocation of phospho-Stat3 through biosynthesis of (+/-)-14,15-epoxyeicosatrienoic acid (EET)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">17543</span>– <span class="NLM_lpage">17559</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.198515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1074%2Fjbc.M110.198515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=21402692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVGjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=17543-17559&author=R.+Mitraauthor=Z.+Guoauthor=M.+Milaniauthor=C.+Mesarosauthor=M.+Rodriguezauthor=J.+Nguyenauthor=X.+Luoauthor=D.+Clarkeauthor=J.+Lambaauthor=E.+Schuetzauthor=D.+B.+Donnerauthor=N.+Puliauthor=J.+R.+Falckauthor=J.+Capdevilaauthor=K.+Guptaauthor=I.+A.+Blairauthor=D.+A.+Potter&title=CYP3A4+mediates+growth+of+estrogen+receptor-positive+breast+cancer+cells+in+part+by+inducing+nuclear+translocation+of+phospho-Stat3+through+biosynthesis+of+%28%2B%2F-%29-14%2C15-epoxyeicosatrienoic+acid+%28EET%29&doi=10.1074%2Fjbc.M110.198515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">CYP3A4 mediates growth of estrogen receptor-positive breast cancer cells in part by inducing nuclear translocation of phospho-stat3 through biosynthesis of (±)-14,15-epoxyeicosatrienoic acid (EET)</span></div><div class="casAuthors">Mitra, Ranjana; Guo, Zhijun; Milani, Monica; Mesaros, Clementina; Rodriguez, Mariangellys; Nguyen, Julia; Luo, Xianghua; Clarke, Duncan; Lamba, Jatinder; Schuetz, Erin; Donner, David B.; Puli, Narender; Falck, John R.; Capdevila, Jorge; Gupta, Kalpna; Blair, Ian A.; Potter, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">17543-17559</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">CYP3A4 expression in breast cancer correlates with decreased overall survival, but the mechanisms are unknown.  Cytochrome P 450 gene profiling by RNAi silencing demonstrates that CYP3A or 2C8 gene expression is specifically required for growth of the breast cancer lines MCF7, T47D, and MDA-MB-231.  CYP3A4 silencing blocks the cell cycle at the G2/M checkpoint and induces apoptosis in the MCF7 line, thereby inhibiting anchorage-dependent growth and survival.  CYP3A4 was profiled for NADPH-dependent arachidonic acid (AA) metab. and synthesized AA epoxygenase products (±)-8,9-, (±)-11,12-, and (±)-14,15-epoxyeicosatrienoic acid (EET) (total turnover of ∼2 pmol/pmol CYP3A4/min) but not hydroxylase products (±)-15-, (±)-19-, or 20-hydroxyeicosatetraenoic acid.  Furthermore, eicosanoid profiling revealed that MCF7 cells synthesize EETs in a CYP3A4-dependent manner.  The (±)-14,15-EET regioisomer selectively rescues breast cancer cells from CYP3A4 silencing in a concn.-dependent fashion and promotes mitogenesis and anchorage-dependent cloning.  Stat3 (Tyr-705) phosphorylation was inhibited by CYP3A4 silencing, providing a potential mechanism for CYP3A4 involvement in breast cancer cell growth.  Silencing Stat3 blocks breast cancer cell growth and abrogates (±)-14,15-EET-induced proliferation, indicating a Stat3 requirement for (±)-14,15-EET-mediated cell growth.  Although silencing of CYP3A4 reduces nuclear Tyr(P)-705-Stat3, (±)-14,15-EET restores this signaling process and promotes Tyr(P)-705-Stat3 translocation to the nucleus, suggesting that (±)-14,15-EET may be involved in an autocrine/paracrine pathway driving cell growth.  These studies indicate that CYP3A4 is a highly active AA epoxygenase that promotes Stat3-mediated breast cancer cell growth in part through (±)-14,15-EET biosynthesis.  Furthermore, these studies indicate an essential role for Stat3 as a mediator of epoxygenase activity in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0JlaMyFCsBrVg90H21EOLACvtfcHk0lgQ7Vve3Gub6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVGjtrs%253D&md5=0195525e306a8d31b5a35589f1cc8872</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.198515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.198515%26sid%3Dliteratum%253Aachs%26aulast%3DMitra%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DMilani%26aufirst%3DM.%26aulast%3DMesaros%26aufirst%3DC.%26aulast%3DRodriguez%26aufirst%3DM.%26aulast%3DNguyen%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DClarke%26aufirst%3DD.%26aulast%3DLamba%26aufirst%3DJ.%26aulast%3DSchuetz%26aufirst%3DE.%26aulast%3DDonner%26aufirst%3DD.%2BB.%26aulast%3DPuli%26aufirst%3DN.%26aulast%3DFalck%26aufirst%3DJ.%2BR.%26aulast%3DCapdevila%26aufirst%3DJ.%26aulast%3DGupta%26aufirst%3DK.%26aulast%3DBlair%26aufirst%3DI.%2BA.%26aulast%3DPotter%26aufirst%3DD.%2BA.%26atitle%3DCYP3A4%2520mediates%2520growth%2520of%2520estrogen%2520receptor-positive%2520breast%2520cancer%2520cells%2520in%2520part%2520by%2520inducing%2520nuclear%2520translocation%2520of%2520phospho-Stat3%2520through%2520biosynthesis%2520of%2520%2528%252B%252F-%2529-14%252C15-epoxyeicosatrienoic%2520acid%2520%2528EET%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D17543%26epage%3D17559%26doi%3D10.1074%2Fjbc.M110.198515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Strategies for discovering and derisking covalent, irreversible enzyme inhibitors</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.4155/fmc.10.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.4155%2Ffmc.10.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=20640225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFakur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=949-964&author=D.+S.+Johnsonauthor=E.+Weerapanaauthor=B.+F.+Cravatt&title=Strategies+for+discovering+and+derisking+covalent%2C+irreversible+enzyme+inhibitors&doi=10.4155%2Ffmc.10.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for discovering and derisking covalent, irreversible enzyme inhibitors</span></div><div class="casAuthors">Johnson, Douglas S.; Weerapana, Eranthie; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">949-964</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This article presents several covalent inhibitors, including examples of successful drugs, as well as highly selective, irreversible inhibitors of emerging therapeutic targets, such as fatty acid amide hydolase.  Covalent inhibitors have many desirable features, including increased biochem. efficiency of target disruption, less sensitivity toward pharmacokinetic parameters and increased duration of action that outlasts the pharmacokinetics of the compd.  Safety concerns that must be mitigated include lack of specificity and the potential immunogenicity of protein-inhibitor adduct(s).  Particular attention will be given to recent technologies, such as activity-based protein profiling, which allow one to define the proteome-wide selectivity patterns for covalent inhibitors in vitro and in vivo.  For instance, any covalent inhibitor can, in principle, be modified with a 'clickable' tag to generate an activity probe that is almost indistinguishable from the original agent.  These probes can be applied to any living system across a broad dose range to fully inventory their on and off targets.  The substantial no. of drugs on the market today that act by a covalent mechanism belies historical prejudices against the development of irreversibly acting therapeutic small mols.  Emerging proteomic technologies offer a means to systematically discriminate safe (selective) vs. deleterious (nonselective) covalent inhibitors and thus should inspire their future design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqm0whEybB-LVg90H21EOLACvtfcHk0lgQ7Vve3Gub6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFakur4%253D&md5=07e1f89418b168798ae5c1a9fa43e9c1</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.21%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DD.%2BS.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DStrategies%2520for%2520discovering%2520and%2520derisking%2520covalent%252C%2520irreversible%2520enzyme%2520inhibitors%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D949%26epage%3D964%26doi%3D10.4155%2Ffmc.10.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brueggemeier, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Cruz, E. S.</span></span> <span> </span><span class="NLM_article-title">Aromatase inhibitors in the treatment of breast cancer</span>. <i>Endocr. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1210/er.2004-0015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1210%2Fer.2004-0015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15814851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltVWru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=331-345&author=R.+W.+Brueggemeierauthor=J.+C.+Hackettauthor=E.+S.+Diaz-Cruz&title=Aromatase+inhibitors+in+the+treatment+of+breast+cancer&doi=10.1210%2Fer.2004-0015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatase inhibitors in the treatment of breast cancer</span></div><div class="casAuthors">Brueggemeier, Robert W.; Hackett, John C.; Diaz-Cruz, Edgar S.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine Reviews</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">331-345</span>CODEN:
                <span class="NLM_cas:coden">ERVIDP</span>;
        ISSN:<span class="NLM_cas:issn">0163-769X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review.  Estradiol, the most potent endogenous estrogen, is biosynthesized from androgens by the cytochrome P 450 enzyme complex called aromatase.  Aromatase is present in breast tissue, and intratumoral aromatase is the source of local estrogen prodn. in breast cancer tissues.  Inhibition of aromatase is an important approach for reducing growth-stimulatory effects of estrogens in estrogen-dependent breast cancer.  Steroidal inhibitors that have been developed to date build upon the basic androstenedione nucleus and incorporate chem. substituents at varying positions on the steroid.  Nonsteroidal aromatase inhibitors can be divided into three classes: aminoglutethimide-like mols., imidazole/triazole derivs., and flavonoid analogs.  Mechanism-based aromatase inhibitors are steroidal inhibitors that mimic the substrate, are converted by the enzyme to a reactive intermediate, and result in the inactivation of aromatase.  Both steroidal and non-steroidal aromatase inhibitors have shown clin. efficacy in the treatment of breast cancer.  The potent and selective third-generation aromatase inhibitors, anastrozole, letrozole, and exemestane, were introduced into the market as endocrine therapy in postmenopausal patients failing antiestrogen therapy alone or multiple hormonal therapies.  These agents are currently approved as first-line therapy for the treatment of postmenopausal women with metastatic estrogen-dependent breast cancer.  Several clin. studies of aromatase inhibitors are currently focusing on the use of these agents in the adjuvant setting for the treatment of early breast cancer.  Use of an aromatase inhibitor as initial therapy or after treatment with tamoxifen is now recommended as adjuvant hormonal therapy for a postmenopausal woman with hormone-dependent breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5rFlkckpA4bVg90H21EOLACvtfcHk0lhGnP2X8gp7nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltVWru7w%253D&md5=d65778c7f1aea9ad16a3d24f37dccb07</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1210%2Fer.2004-0015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fer.2004-0015%26sid%3Dliteratum%253Aachs%26aulast%3DBrueggemeier%26aufirst%3DR.%2BW.%26aulast%3DHackett%26aufirst%3DJ.%2BC.%26aulast%3DDiaz-Cruz%26aufirst%3DE.%2BS.%26atitle%3DAromatase%2520inhibitors%2520in%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DEndocr.%2520Rev.%26date%3D2005%26volume%3D26%26spage%3D331%26epage%3D345%26doi%3D10.1210%2Fer.2004-0015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. B.</span></span> <i>Mechanism-Based Enzyme Inactivation: Chemistry and Enzymology</i>; <span class="NLM_publisher-name">CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, FL</span>, <span class="NLM_year">1988</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1988&author=R.+B.+Silverman&title=Mechanism-Based+Enzyme+Inactivation%3A+Chemistry+and+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSilverman%26aufirst%3DR.%2BB.%26btitle%3DMechanism-Based%2520Enzyme%2520Inactivation%253A%2520Chemistry%2520and%2520Enzymology%26pub%3DCRC%2520Press%26date%3D1988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giudici, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ornati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briatico, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzzetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Salle, E.</span></span> <span> </span><span class="NLM_article-title">6-Methylenandrosta-1,4-diene-3,17-dione (FCE 24304): a new irreversible aromatase inhibitor</span>. <i>J. Steroid Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1016/0022-4731(88)90129-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2F0022-4731%2888%2990129-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=3386266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaL1cXks1OitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1988&pages=391-394&author=D.+Giudiciauthor=G.+Ornatiauthor=G.+Briaticoauthor=F.+Buzzettiauthor=P.+Lombardiauthor=E.+di+Salle&title=6-Methylenandrosta-1%2C4-diene-3%2C17-dione+%28FCE+24304%29%3A+a+new+irreversible+aromatase+inhibitor&doi=10.1016%2F0022-4731%2888%2990129-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">6-Methylenandrosta-1,4-diene-3,17-dione (FCE 24304): a new irreversible aromatase inhibitor</span></div><div class="casAuthors">Giudici, D.; Ornati, G.; Briatico, G.; Buzzetti, F.; Lombardi, P.; Di Salle, E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Steroid Biochemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1-6</span>),
    <span class="NLM_cas:pages">391-4</span>CODEN:
                <span class="NLM_cas:coden">JSTBBK</span>;
        ISSN:<span class="NLM_cas:issn">0022-4731</span>.
    </div><div class="casAbstract">FCE 24304 (6-methylenandrosta-1,4-diene-3,17-dione), a new irreversible aromatase inhibitor, was identified and characterized in vitro and in vivo.  The compd. caused time-dependent inactivation of human placental aromatase with a half-time of 13.9 min and Ki of 26 nM.  When tested in PMSG-treated rats, ovarian aromatase activity was reduced 24 h after dosing by both the s.c. (50% ED 1.8 mg/kg) and the oral (50% ED 3.7 mg/kg) routes.  No interference with 5α-reductase activity and no significant binding affinity for the estrogen receptor were found.  A slight binding affinity for the androgen receptor (relative binding affinity 0.2% of dihydrotestosterone) was obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0X7BVetfC_LVg90H21EOLACvtfcHk0lhGnP2X8gp7nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXks1OitLY%253D&md5=bb45165bda30bd0807cb0779f3ce8628</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2F0022-4731%2888%2990129-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-4731%252888%252990129-X%26sid%3Dliteratum%253Aachs%26aulast%3DGiudici%26aufirst%3DD.%26aulast%3DOrnati%26aufirst%3DG.%26aulast%3DBriatico%26aufirst%3DG.%26aulast%3DBuzzetti%26aufirst%3DF.%26aulast%3DLombardi%26aufirst%3DP.%26aulast%3Ddi%2BSalle%26aufirst%3DE.%26atitle%3D6-Methylenandrosta-1%252C4-diene-3%252C17-dione%2520%2528FCE%252024304%2529%253A%2520a%2520new%2520irreversible%2520aromatase%2520inhibitor%26jtitle%3DJ.%2520Steroid%2520Biochem.%26date%3D1988%26volume%3D30%26spage%3D391%26epage%3D394%26doi%3D10.1016%2F0022-4731%2888%2990129-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sodhi, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delarosa, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halladay, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driscoll, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dansette, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh, S. C.</span></span> <span> </span><span class="NLM_article-title">Inhibitory effects of trapping agents of sulfur drug reactive intermediates against major human cytochrome P450 isoforms</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>, <span class="NLM_elocation-id">1553</span> <span class="refDoi"> DOI: 10.3390/ijms18071553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.3390%2Fijms18071553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVehsrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&author=J.+K.+Sodhiauthor=E.+M.+Delarosaauthor=J.+S.+Halladayauthor=J.+P.+Driscollauthor=T.+Mulderauthor=P.+M.+Dansetteauthor=S.+C.+Khojasteh&title=Inhibitory+effects+of+trapping+agents+of+sulfur+drug+reactive+intermediates+against+major+human+cytochrome+P450+isoforms&doi=10.3390%2Fijms18071553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitory effects of trapping agents of sulfur drug reactive intermediates against major human cytochrome P450 isoforms</span></div><div class="casAuthors">Sodhi, Jasleen K.; Delarosa, Erlie Marie; Halladay, Jason S.; Driscoll, James P.; Mulder, Teresa; Dansette, Patrick M.; Khojasteh, S. Cyrus</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1553/1-1553/14</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In some cases, the formation of reactive species from the metab. of xenobiotics has been linked to toxicity and therefore it is imperative to detect potential bioactivation for candidate drugs during drug discovery.  Reactive species can covalently bind to trapping agents in in vitro incubations of compd. with human liver microsomes (HLM) fortified with β-NADP (NADPH), resulting in a stable conjugate of trapping agent and reactive species, thereby facilitating anal. detection and providing evidence of short-lived reactive metabolites.  Since reactive metabolites are typically generated by cytochrome P 450 (CYP) oxidn., it is important to ensure high concns. of trapping agents are not inhibiting the activities of CYP isoforms.  Here we assessed the inhibitory properties of fourteen trapping agents against the major human CYP isoforms (CYP1A2, 2C9, 2C19, 2D6 and 3A).  Based on our findings, eleven trapping agents displayed inhibition, three of which had IC50 values less than 1 mM (2-mercaptoethanol, N-methylmaleimide and N-ethylmaleimide (NEM)).  Three trapping agents (dimedone, N-acetyl-lysine and arsenite) did not inhibit CYP isoforms at concns. tested.  To illustrate effects of CYP inhibition by trapping agents on reactive intermediate trapping, an example drug (ticlopidine) and trapping agent (NEM) were chosen for further studies.  For the same amt. of ticlopidine (1 μM), increasing concns. of the trapping agent NEM (0.007-40 mM) resulted in a bell-shaped response curve of NEM-trapped ticlopidine S-oxide (TSO-NEM), due to CYP inhibition by NEM.  Thus, trapping studies should be designed to include several concns. of trapping agent to ensure optimal trapping of reactive metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4dfvGpPEhk7Vg90H21EOLACvtfcHk0lhGnP2X8gp7nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVehsrzJ&md5=4eb3d6f04c90fb2b68fa4f99b82d447f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.3390%2Fijms18071553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071553%26sid%3Dliteratum%253Aachs%26aulast%3DSodhi%26aufirst%3DJ.%2BK.%26aulast%3DDelarosa%26aufirst%3DE.%2BM.%26aulast%3DHalladay%26aufirst%3DJ.%2BS.%26aulast%3DDriscoll%26aufirst%3DJ.%2BP.%26aulast%3DMulder%26aufirst%3DT.%26aulast%3DDansette%26aufirst%3DP.%2BM.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26atitle%3DInhibitory%2520effects%2520of%2520trapping%2520agents%2520of%2520sulfur%2520drug%2520reactive%2520intermediates%2520against%2520major%2520human%2520cytochrome%2520P450%2520isoforms%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26doi%3D10.3390%2Fijms18071553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C.</span></span> <span> </span><span class="NLM_article-title">Suicide substrates for the alanine racemase of Escherichia coli B</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1313</span>– <span class="NLM_lpage">1321</span>, <span class="refDoi"> DOI: 10.1021/bi00600a028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00600a028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaE1cXktVahs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1978&pages=1313-1321&author=E.+Wangauthor=C.+Walsh&title=Suicide+substrates+for+the+alanine+racemase+of+Escherichia+coli+B&doi=10.1021%2Fbi00600a028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Suicide substrates for the alanine racemase of Escherichia coli B</span></div><div class="casAuthors">Wang, Elizabeth; Walsh, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1313-21</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Alanine racemase was purified ∼9000-fold from E. coli and found to be a dimer of 100,000 daltons, contg. 1 mol. pyridoxal phosphate/subunit.  The mol. basis for irrversible active site-directed inactivation of the enzyme by a no. of suicide substrates was examd.  Both D and L isomers of β-fluoroalanine and β-chloroalanine partition between (a) α,β elimination to pyruvate, ammonia, and halide ion or (b) inactivation.  No racemization is detectable.  The Vmax for pyruvate formation from L-chloroalanine is ∼50-fold lower than from the D isomer of chloroalanine or either fluoroalanine.  However, both enantiomeric pairs partition identically, ∼830 turnovers/inactivating event.  This invariant partition ratio suggests that a common intermediate, the eneamino acid-pyridoxal phosphate complex, is the species responsible for inactivation, probably by Michael attack from a nucleophilic residue at the enzyme active site.  In keeping with this idea, O-carbamoyl-D-serine and O-acetyl-D-serine also undergo enzyme-catalyzed elimination for 830 turnovers before causing irreversible inactivation, presumably from the same intermediate.  In contrast, the L isomers of O-carbamoyl- or O-acetlyserine do not eliminate nor do they induce inactivation, but serve merely as reversible, competitive inhibitors of the enzyme.  This suggests asym. binding regions for bulky β substituents at the active site and suggests D isomers of substituted β-alanines would be preferentially effective enzyme inactivators.  D-Cycloserine also inactivates alanine racemase in time-dependent fashion.  Thus, both natural antibiotics, O-carbamoyl-D-serine and D-cycloserine, previously reported as reversible alanine racemase inhibitors, are in fact suicide substrates along with the β-haloalanines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohIkuziPLwbLVg90H21EOLACvtfcHk0lhgxEKT-NKMAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXktVahs7c%253D&md5=0d32c8dfec57779e35308e0c360e3f06</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fbi00600a028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00600a028%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DE.%26aulast%3DWalsh%26aufirst%3DC.%26atitle%3DSuicide%2520substrates%2520for%2520the%2520alanine%2520racemase%2520of%2520Escherichia%2520coli%2520B%26jtitle%3DBiochemistry%26date%3D1978%26volume%3D17%26spage%3D1313%26epage%3D1321%26doi%3D10.1021%2Fbi00600a028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duczak, N.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brougham, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.</span></span> <span> </span><span class="NLM_article-title">Automated screening with confirmation of mechanism-based inactivation of CYP3A4, CYP2C9, CYP2C19, CYP2D6, and CYP1A2 in pooled human liver microsomes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1211</span>– <span class="NLM_lpage">1219</span>, <span class="refDoi"> DOI: 10.1124/dmd.104.003475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.104.003475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15860655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntVamsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=1211-1219&author=H.+K.+Limauthor=N.+Duczakauthor=L.+Broughamauthor=M.+Elliotauthor=K.+Patelauthor=K.+Chan&title=Automated+screening+with+confirmation+of+mechanism-based+inactivation+of+CYP3A4%2C+CYP2C9%2C+CYP2C19%2C+CYP2D6%2C+and+CYP1A2+in+pooled+human+liver+microsomes&doi=10.1124%2Fdmd.104.003475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Automated screening with confirmation of mechanism-based inactivation of CYP3A4, CYP2C9, CYP2C19, CYP2D6, and CYP1A2 in pooled human liver microsomes</span></div><div class="casAuthors">Lim, Heng-Keang; Duczak, Nicholas, Jr.; Brougham, Linda; Elliot, Michael; Patel, Krupa; Chan, Kelvin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1211-1219</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A strategy is proposed to profile compds. for mechanism-based inactivation of CYP3A4, CYP2C19, CYP2C9, CYP2D6, and CYP1A2 based on an apparent partition ratio screen.  Potent positives from the screen are confirmed by time- and concn.-dependent inactivation assays.  Quasi-irreversible inhibitions are then differentiated from irreversible inactivations by oxidn. with potassium ferricyanide and/or dialysis.  The three-step screening procedure has been validated with acceptable accuracy and precision for detection and confirmation of mechanism-based inactivators in drug discovery.  We report here the apparent partition ratios for 19 mechanism-based inactivators and four quasi-irreversible inhibitors obtained under the same exptl. conditions.  The apparent partition ratio screen was automated to provide throughput for detg. structure-mechanism-based inactivation relationships.  Information about reversibility can be used to assess potential toxicity mediated by covalent adducts, as well as the potential for pharmacokinetic drug-drug interactions.  Direct comparison of known mechanism-based inactivators and quasi-irreversible inhibitors, based on our screening of apparent partition ratios, has identified ritonavir, mibefradil, and azamulin as highly effective mechanism-based inactivators; e.g., 1 mol of CYP3A4 was inactivated on turnover of about 2 mol of compd.  Other mechanism-based inactivators we identified include bergamottin (CYP1A2 besides previously reported CYP3A4), troglitazone (CYP3A4), rosiglitazone (CYP3A4), and pioglitazone (CYP3A4).  Comparison of the apparent partition ratios and inactivation clearance data for the three glitazones suggests that the chromane moiety on troglitazone contributes to its greater potency for mechanism-based inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqMduVu8I82bVg90H21EOLACvtfcHk0lhgxEKT-NKMAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntVamsLc%253D&md5=354d611d2a5480fb79281f1eb6bd2e16</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1124%2Fdmd.104.003475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.104.003475%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DH.%2BK.%26aulast%3DDuczak%26aufirst%3DN.%26aulast%3DBrougham%26aufirst%3DL.%26aulast%3DElliot%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DK.%26atitle%3DAutomated%2520screening%2520with%2520confirmation%2520of%2520mechanism-based%2520inactivation%2520of%2520CYP3A4%252C%2520CYP2C9%252C%2520CYP2C19%252C%2520CYP2D6%252C%2520and%2520CYP1A2%2520in%2520pooled%2520human%2520liver%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26spage%3D1211%26epage%3D1219%26doi%3D10.1124%2Fdmd.104.003475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speirs, V.</span></span> <span> </span><span class="NLM_article-title">Choosing the right cell line for breast cancer research</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">215</span> <span class="refDoi"> DOI: 10.1186/bcr2889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1186%2Fbcr2889" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=21884641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FkvFCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&author=D.+L.+Hollidayauthor=V.+Speirs&title=Choosing+the+right+cell+line+for+breast+cancer+research&doi=10.1186%2Fbcr2889"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Choosing the right cell line for breast cancer research</span></div><div class="casAuthors">Holliday Deborah L; Speirs Valerie</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research : BCR</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">215</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer is a complex and heterogeneous disease.  Gene expression profiling has contributed significantly to our understanding of this heterogeneity at a molecular level, refining taxonomy based on simple measures such as histological type, tumour grade, lymph node status and the presence of predictive markers like oestrogen receptor and human epidermal growth factor receptor 2 (HER2) to a more sophisticated classification comprising luminal A, luminal B, basal-like, HER2-positive and normal subgroups.  In the laboratory, breast cancer is often modelled using established cell lines.  In the present review we discuss some of the issues surrounding the use of breast cancer cell lines as experimental models, in light of these revised clinical classifications, and put forward suggestions for improving their use in translational breast cancer research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9z00kKYf1zjdblLE_E9PWfW6udTcc2eaeoCDI9VNc1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FkvFCnuw%253D%253D&md5=9fbb9c5debbec50a5708666272fff46f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1186%2Fbcr2889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr2889%26sid%3Dliteratum%253Aachs%26aulast%3DHolliday%26aufirst%3DD.%2BL.%26aulast%3DSpeirs%26aufirst%3DV.%26atitle%3DChoosing%2520the%2520right%2520cell%2520line%2520for%2520breast%2520cancer%2520research%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2011%26volume%3D13%26doi%3D10.1186%2Fbcr2889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savas, Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span> <span> </span><span class="NLM_article-title">Conditional regulation of the human CYP4X1 and CYP4Z1 genes</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>436</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2005.02.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.abb.2005.02.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15797250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2MXis1GksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2005&pages=377-385&author=%C3%9C.+Savasauthor=M.-H.+Hsuauthor=K.+J.+Griffinauthor=D.+R.+Bellauthor=E.+F.+Johnson&title=Conditional+regulation+of+the+human+CYP4X1+and+CYP4Z1+genes&doi=10.1016%2Fj.abb.2005.02.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Conditional regulation of the human CYP4X1 and CYP4Z1 genes</span></div><div class="casAuthors">Savas, Uezen; Hsu, Mei-Hui; Griffin, Keith J.; Bell, David R.; Johnson, Eric F.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-385</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cytochrome P 450 genes (CYPs) encoding two new subfamilies designated CYP4X1 and CYP4Z1 were identified in the human genome and the Expressed Sequence Tags database.  Partial cDNAs encoding both P450s were isolated from human kidney and used to det. tissue distribution.  CYP4X1 was predominantly expressed in trachea and aorta, whereas CYP4Z1 mRNA was preferentially expressed in mammary tissue.  In T47-D cells, CYP4Z1 mRNA levels were induced by dexamethasone (14-fold) or by progesterone (10-fold).  The induction by these compds. was suppressed by co-treatment with the progesterone and glucocorticoid receptor antagonist mifepristone (RU486).  In the progesterone receptor neg. MCF-7 cells, CYP4Z1 mRNA was induced by dexamethasone but not by progesterone treatment.  CYP4Z1 mRNA levels were unaffected by 17β-estradiol.  In confluent cultures of human hepatoma HepG2 cells that stably express a mouse peroxisome proliferator activated receptor-α (PPARα) mutant, CYP4X1 mRNA was undetectable in vehicle-treated cells but was readily detectable following addn. of the PPARα agonist Wy14643.  This suggests that PPARα activation can affect human CYP4X1 gene transcription.  These results demonstrate selective tissue expression and implicate PPARα in CYP4X1 regulation, and the glucocorticoid and progesterone receptors in CYP4Z1 gene activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRacpMSaOSJbVg90H21EOLACvtfcHk0ljpWQtwrIr1Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXis1GksL0%253D&md5=07cd2eaba5741f1595d546fc62070851</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2005.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2005.02.022%26sid%3Dliteratum%253Aachs%26aulast%3DSavas%26aufirst%3D%25C3%259C.%26aulast%3DHsu%26aufirst%3DM.-H.%26aulast%3DGriffin%26aufirst%3DK.%2BJ.%26aulast%3DBell%26aufirst%3DD.%2BR.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26atitle%3DConditional%2520regulation%2520of%2520the%2520human%2520CYP4X1%2520and%2520CYP4Z1%2520genes%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2005%26volume%3D436%26spage%3D377%26epage%3D385%26doi%3D10.1016%2Fj.abb.2005.02.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishikiori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iizuka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichiba, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukushima, T.</span></span> <span> </span><span class="NLM_article-title">Determination of free fatty acids in human serum by HPLC with fluorescence detection</span>. <i>J. Chromatogr. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">537</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1093/chromsci/bmu081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1093%2Fchromsci%2Fbmu081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=25033982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlGrt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2015&pages=537-541&author=M.+Nishikioriauthor=H.+Iizukaauthor=H.+Ichibaauthor=K.+Sadamotoauthor=T.+Fukushima&title=Determination+of+free+fatty+acids+in+human+serum+by+HPLC+with+fluorescence+detection&doi=10.1093%2Fchromsci%2Fbmu081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of free fatty acids in human serum by HPLC with fluorescence detection</span></div><div class="casAuthors">Nishikiori, Minami; Iizuka, Hideaki; Ichiba, Hideaki; Sadamoto, Kiyomi; Fukushima, Takeshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chromatographic Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">537-541</span>CODEN:
                <span class="NLM_cas:coden">JCHSBZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-9665</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">It has been suggested that serum concns. of polyunsatd. essential fatty acids correlate with the symptoms or severity of various diseases, including depression and Alzheimer-type dementia, and that detn. of serum fatty acids might be important for disease diagnosis.  Thus, we developed to analyze serum fatty acids in healthy individuals by using a high-performance liq. chromatog. method with fluorescence detection, because free fatty acids have a carboxyl group that can be derivatized with a fluorescent reagent, 4-N,N-dimethylaminosulfonyl-7-N-(2-aminoethyl)amino-2,1,3-benzoxadiazole.  This approach could quantify five types of free fatty acids [α-linolenic acid (ALA), palmitoleic acid (PLA), arachidonic acid (AA), linoleic acid (LA) and oleic acid (OA)] in human serum.  The detection limits of the method were in the range of 2.29-4.75 fmol (signal-to-noise ratio 3), and abs. concns. of ALA, PLA, AA, LA and OA were 8.27 ± 1.04, 18.8 ± 2.95, 49.9 ± 4.03, 230 ± 18.1 and 201 ± 22.1 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKl5XFXTrRbVg90H21EOLACvtfcHk0ljpWQtwrIr1Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlGrt7bL&md5=0aae334c866789a70f7dc8473fedd5de</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fchromsci%2Fbmu081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fchromsci%252Fbmu081%26sid%3Dliteratum%253Aachs%26aulast%3DNishikiori%26aufirst%3DM.%26aulast%3DIizuka%26aufirst%3DH.%26aulast%3DIchiba%26aufirst%3DH.%26aulast%3DSadamoto%26aufirst%3DK.%26aulast%3DFukushima%26aufirst%3DT.%26atitle%3DDetermination%2520of%2520free%2520fatty%2520acids%2520in%2520human%2520serum%2520by%2520HPLC%2520with%2520fluorescence%2520detection%26jtitle%3DJ.%2520Chromatogr.%2520Sci.%26date%3D2015%26volume%3D53%26spage%3D537%26epage%3D541%26doi%3D10.1093%2Fchromsci%2Fbmu081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span> <span> </span><span class="NLM_article-title">An optimized high throughput clean-up method using mixed-mode SPE plate for the analysis of free arachidonic acid in plasma by LC-MS/MS</span>. <i>Int. J. Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2015</i></span>, <span class="NLM_elocation-id">374819</span> <span class="refDoi"> DOI: 10.1155/2015/374819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1155%2F2015%2F374819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=25873969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC2Mjkt12ktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&author=W.+Wangauthor=S.+Qinauthor=L.+Liauthor=X.+Chenauthor=Q.+Wangauthor=J.+Wei&title=An+optimized+high+throughput+clean-up+method+using+mixed-mode+SPE+plate+for+the+analysis+of+free+arachidonic+acid+in+plasma+by+LC-MS%2FMS&doi=10.1155%2F2015%2F374819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">An Optimized High Throughput Clean-Up Method Using Mixed-Mode SPE Plate for the Analysis of Free Arachidonic Acid in Plasma by LC-MS/MS</span></div><div class="casAuthors">Wang Wan; Qin Suzi; Li Linsen; Chen Xiaohua; Wang Qunjie; Wei Junfu</div><div class="citationInfo"><span class="NLM_cas:title">International journal of analytical chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2015</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">374819</span>
        ISSN:<span class="NLM_cas:issn">1687-8760</span>.
    </div><div class="casAbstract">A high throughput sample preparation method was developed utilizing mixed-mode solid phase extraction (SPE) in 96-well plate format for the determination of free arachidonic acid in plasma by LC-MS/MS.  Plasma was mixed with 3% aqueous ammonia and loaded into each well of 96-well plate.  After washing with water and methanol sequentially, 3% of formic acid in acetonitrile was used to elute arachidonic acid.  The collected fraction was injected onto a reversed phase column at 30°C with mobile phase of acetonitrile/water (70 : 30, v/v) and detected by LC-MS/MS coupled with electrospray ionization (ESI) in multiple reaction monitoring (MRM) mode.  The calibration curve ranged from 10 to 2500 ng/mL with sufficient linearity (r (2) = 0.9999).  The recoveries were in the range of 99.38% to 103.21% with RSD less than 6%.  The limit of detection is 3 ng/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS_GqxiFHx-9v_I8Jvzw4xCfW6udTcc2eZXznYYr8fU4bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mjkt12ktg%253D%253D&md5=af28e6023a9f80194361433eb784ce13</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1155%2F2015%2F374819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2015%252F374819%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DWei%26aufirst%3DJ.%26atitle%3DAn%2520optimized%2520high%2520throughput%2520clean-up%2520method%2520using%2520mixed-mode%2520SPE%2520plate%2520for%2520the%2520analysis%2520of%2520free%2520arachidonic%2520acid%2520in%2520plasma%2520by%2520LC-MS%252FMS%26jtitle%3DInt.%2520J.%2520Anal.%2520Chem.%26date%3D2015%26volume%3D2015%26doi%3D10.1155%2F2015%2F374819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Decker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giallonardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgener, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marowsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falck, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morisseau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammock, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruzdev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeldin, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arand, M.</span></span> <span> </span><span class="NLM_article-title">EH3 (ABHD9): the first member of a new epoxide hydrolase family with high activity for fatty acid epoxides</span>. <i>J. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2038</span>– <span class="NLM_lpage">2045</span>, <span class="refDoi"> DOI: 10.1194/jlr.M024448</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1194%2Fjlr.M024448" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=22798687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOmtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2012&pages=2038-2045&author=M.+Deckerauthor=M.+Adamskaauthor=A.+Croninauthor=F.+Di+Giallonardoauthor=J.+Burgenerauthor=A.+Marowskyauthor=J.+R.+Falckauthor=C.+Morisseauauthor=B.+D.+Hammockauthor=A.+Gruzdevauthor=D.+C.+Zeldinauthor=M.+Arand&title=EH3+%28ABHD9%29%3A+the+first+member+of+a+new+epoxide+hydrolase+family+with+high+activity+for+fatty+acid+epoxides&doi=10.1194%2Fjlr.M024448"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">EH3 (ABHD9): the first member of a new epoxide hydrolase family with high activity for fatty acid epoxides</span></div><div class="casAuthors">Decker, Martina; Adamska, Magdalena; Cronin, Annette; Di Giallonardo, Francesca; Burgener, Julia; Marowsky, Anne; Falck, John R.; Morisseau, Christophe; Hammock, Bruce D.; Gruzdev, Artiom; Zeldin, Darryl C.; Arand, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2038-2045</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology, Inc.</span>)
        </div><div class="casAbstract">Epoxide hydrolases are a small superfamily of enzymes important for the detoxification of chem. reactive xenobiotic epoxides and for the processing of endogenous epoxides that act as signaling mols.  Here, the identification of two human epoxide hydrolases, EH3 and EH4, are reported.  They share 45% sequence identity, thus representing a new family of mammalian epoxide hydrolases.  Quant. RT-PCR from mouse tissue indicates strongest EH3 expression in lung, skin, and upper gastrointestinal tract.  The recombinant enzyme shows a high turnover no. with 8,9-, 11,12-, and 14,15-epoxyeicosatrienoic acid (EET), as well as 9,10-epoxyoctadec-11-enoic acid (leukotoxin).  It is inhibited by a subclass of N,N'-disubstituted urea derivs., including 12-(3-adamantan-1-yl-ureido)-dodecanoic acid, 1-cyclohexyl-3-dodecylurea, and 1-(1-acetylpiperidin-4-yl)-3-(4-(trifluoromethoxy)phenyl)urea, compds. so far believed to be selective inhibitors of mammalian sol. epoxide hydrolase (sEH).  Its sensitivity to this subset of sEH inhibitors may have implications on the pharmacol. profile of these compds.  This is particularly relevant because sEH is a potential drug target, and clin. trials are under way exploring the value of sEH inhibitors in the treatment of hypertension and diabetes type II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpauMFKbDd0EbVg90H21EOLACvtfcHk0ljpWQtwrIr1Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOmtLjF&md5=09e15bebf9ad1061339f4fe38d00e08a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M024448&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M024448%26sid%3Dliteratum%253Aachs%26aulast%3DDecker%26aufirst%3DM.%26aulast%3DAdamska%26aufirst%3DM.%26aulast%3DCronin%26aufirst%3DA.%26aulast%3DDi%2BGiallonardo%26aufirst%3DF.%26aulast%3DBurgener%26aufirst%3DJ.%26aulast%3DMarowsky%26aufirst%3DA.%26aulast%3DFalck%26aufirst%3DJ.%2BR.%26aulast%3DMorisseau%26aufirst%3DC.%26aulast%3DHammock%26aufirst%3DB.%2BD.%26aulast%3DGruzdev%26aufirst%3DA.%26aulast%3DZeldin%26aufirst%3DD.%2BC.%26aulast%3DArand%26aufirst%3DM.%26atitle%3DEH3%2520%2528ABHD9%2529%253A%2520the%2520first%2520member%2520of%2520a%2520new%2520epoxide%2520hydrolase%2520family%2520with%2520high%2520activity%2520for%2520fatty%2520acid%2520epoxides%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2012%26volume%3D53%26spage%3D2038%26epage%3D2045%26doi%3D10.1194%2Fjlr.M024448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, A. I.</span>; <span class="NLM_string-name">Tatchell, A. R.</span>; <span class="NLM_string-name">Furnis, B. S.</span>; <span class="NLM_string-name">Hannaford, A. J.</span>; <span class="NLM_string-name">Smith, P. W. G.</span></span> <i>Vogel’s Textbook of Practical Organic Chemistry</i>, <span class="NLM_edition">5</span>th ed.; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=A.+I.+Vogel&author=A.+R.+Tatchell&author=B.+S.+Furnis&author=A.+J.+Hannaford&author=P.+W.+G.+Smith&title=Vogel%E2%80%99s+Textbook+of+Practical+Organic+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DA.%2BI.%26btitle%3DVogel%25E2%2580%2599s%2520Textbook%2520of%2520Practical%2520Organic%2520Chemistry%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roellecke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaisers, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarov-Yarovoy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramm, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanenberg, H.</span></span> <span> </span><span class="NLM_article-title">Identification of amino acid determinants in CYP4B1 for optimal catalytic processing of 4-ipomeanol</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>465</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1042/BJ20140813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1042%2FBJ20140813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=25247810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFWiu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=465&publication_year=2015&pages=103-114&author=C.+Wiekauthor=E.+M.+Schmidtauthor=K.+Roelleckeauthor=M.+Freundauthor=M.+Nakanoauthor=E.+J.+Kellyauthor=W.+Kaisersauthor=V.+Yarov-Yarovoyauthor=C.+M.+Krammauthor=A.+E.+Rettieauthor=H.+Hanenberg&title=Identification+of+amino+acid+determinants+in+CYP4B1+for+optimal+catalytic+processing+of+4-ipomeanol&doi=10.1042%2FBJ20140813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of amino acid determinants in CYP4B1 for optimal catalytic processing of 4-ipomeanol</span></div><div class="casAuthors">Wiek, Constanze; Schmidt, Eva M.; Roellecke, Katharina; Freund, Marcel; Nakano, Mariko; Kelly, Edward J.; Kaisers, Wolfgang; Yarov-Yarovoy, Vladimir; Kramm, Christof M.; Rettie, Allan E.; Hanenberg, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">465</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-114</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Mammalian CYP4B1 enzymes are cytochrome P 450 mono-oxygenases that are responsible for the bioactivation of several exogenous pro-toxins including 4-ipomeanol (4-IPO).  In contrast with the orthologous rabbit enzyme, we show here that native human CYP4B1 with a serine residue at position 427 is unable to bioactivate 4-IPO and does not cause cytotoxicity in HepG2 cells and primary human T-cells that overexpress these enzymes.  We also demonstrate that a proline residue in the meander region at position 427 in human CYP4B1 and 422 in rabbit CYP4B1 is important for protein stability and rescues the 4-IPO bioactivation of the human enzyme, but is not essential for the catalytic activity of the rabbit CYP4B1 protein.  Systematic substitution of native and p.S427P human CYP4B1 with peptide regions from the highly active rabbit enzyme reveals that 18 amino acids in the wild-type rabbit CYP4B1 protein are key for conferring high 4-IPO metabolizing activity.  Introduction of 12 of the 18 amino acids that are also present at corresponding positions in other human CYP4 family members into the p.S427P human CYP4B1 protein results in a mutant human enzyme (P+12) that is as stable and as active as the rabbit wild-type CYP4B1 protein.  These 12 mutations cluster in the predicted B-C loop through F-helix regions and reveal new amino acid regions important to P 450 enzyme stability.  Finally, by minimally re-engineering the human CYP4B1 enzyme for efficient activation of 4-IPO, we have developed a novel human suicide gene system that is a candidate for adoptive cellular therapies in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBPRWo3GQ9vrVg90H21EOLACvtfcHk0lgbFS1U3dEdaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFWiu7vI&md5=775cdbd19548aa4757dacf86b73eb3df</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1042%2FBJ20140813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20140813%26sid%3Dliteratum%253Aachs%26aulast%3DWiek%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DE.%2BM.%26aulast%3DRoellecke%26aufirst%3DK.%26aulast%3DFreund%26aufirst%3DM.%26aulast%3DNakano%26aufirst%3DM.%26aulast%3DKelly%26aufirst%3DE.%2BJ.%26aulast%3DKaisers%26aufirst%3DW.%26aulast%3DYarov-Yarovoy%26aufirst%3DV.%26aulast%3DKramm%26aufirst%3DC.%2BM.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26aulast%3DHanenberg%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520amino%2520acid%2520determinants%2520in%2520CYP4B1%2520for%2520optimal%2520catalytic%2520processing%2520of%25204-ipomeanol%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D465%26spage%3D103%26epage%3D114%26doi%3D10.1042%2FBJ20140813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roellecke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virts, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einholz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edson, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altvater, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Laer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheckenbach, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagenmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhardt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramm, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanenberg, H.</span></span> <span> </span><span class="NLM_article-title">Optimized human CYP4B1 in combination with the alkylator prodrug 4-ipomeanol serves as a novel suicide gene system for adoptive T-cell therapies</span>. <i>Gene Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1038/gt.2016.38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1038%2Fgt.2016.38" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=27092941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28XotFOrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=615-626&author=K.+Roelleckeauthor=E.+L.+Virtsauthor=R.+Einholzauthor=K.+Z.+Edsonauthor=B.+Altvaterauthor=C.+Rossigauthor=D.+von+Laerauthor=K.+Scheckenbachauthor=M.+Wagenmannauthor=D.+Reinhardtauthor=C.+M.+Krammauthor=A.+E.+Rettieauthor=C.+Wiekauthor=H.+Hanenberg&title=Optimized+human+CYP4B1+in+combination+with+the+alkylator+prodrug+4-ipomeanol+serves+as+a+novel+suicide+gene+system+for+adoptive+T-cell+therapies&doi=10.1038%2Fgt.2016.38"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Optimized human CYP4B1 in combination with the alkylator prodrug 4-ipomeanol serves as a novel suicide gene system for adoptive T-cell therapies</span></div><div class="casAuthors">Roellecke, K.; Virts, E. L.; Einholz, R.; Edson, K. Z.; Altvater, B.; Rossig, C.; von Laer, D.; Scheckenbach, K.; Wagenmann, M.; Reinhardt, D.; Kramm, C. M.; Rettie, A. E.; Wiek, C.; Hanenberg, H.</div><div class="citationInfo"><span class="NLM_cas:title">Gene Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">615-626</span>CODEN:
                <span class="NLM_cas:coden">GETHEC</span>;
        ISSN:<span class="NLM_cas:issn">0969-7128</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Engineering autologous or allogeneic T cells to express a suicide gene can control potential toxicity in adoptive T-cell therapies.  We recently reported the development of a novel human suicide gene system that is based on an orphan human cytochrome P 450 enzyme, CYP4B1, and the naturally occurring alkylator prodrug 4-ipomeanol.  The goal of this study was to systematically develop a clin. applicable self-inactivating lentiviral vector for efficient co-expression of CYP4B1 as an ER-located protein with two distinct types of cell surface proteins, either MACS selection genes for donor lymphocyte infusions after allogeneic stem cell transplantation or chimeric antigen receptors for retargeting primary T cells.  The U3 region of the myeloproliferative sarcoma virus in combination with the T2A site was found to drive high-level expression of our CYP4B1 mutant with truncated CD34 or CD271 as MACS suitable selection markers.  This lentiviral vector backbone was also well suited for co-expression of CYP4B1 with a codon-optimized CD19 chimeric antigen receptor (CAR) construct.  Finally, 4-ipomeanol efficiently induced apoptosis in primary T cells that co-express mutant CYP4B1 and the divergently located MACS selection and CAR genes.  In conclusion, we here developed a clin. suited lentiviral vector that supports high-level co-expression of cell surface proteins with a potent novel human suicide gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodG_hjPEjmZ7Vg90H21EOLACvtfcHk0lgbFS1U3dEdaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotFOrsrc%253D&md5=814f5d25968aa92252e01cf15ee980d5</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fgt.2016.38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fgt.2016.38%26sid%3Dliteratum%253Aachs%26aulast%3DRoellecke%26aufirst%3DK.%26aulast%3DVirts%26aufirst%3DE.%2BL.%26aulast%3DEinholz%26aufirst%3DR.%26aulast%3DEdson%26aufirst%3DK.%2BZ.%26aulast%3DAltvater%26aufirst%3DB.%26aulast%3DRossig%26aufirst%3DC.%26aulast%3Dvon%2BLaer%26aufirst%3DD.%26aulast%3DScheckenbach%26aufirst%3DK.%26aulast%3DWagenmann%26aufirst%3DM.%26aulast%3DReinhardt%26aufirst%3DD.%26aulast%3DKramm%26aufirst%3DC.%2BM.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26aulast%3DWiek%26aufirst%3DC.%26aulast%3DHanenberg%26aufirst%3DH.%26atitle%3DOptimized%2520human%2520CYP4B1%2520in%2520combination%2520with%2520the%2520alkylator%2520prodrug%25204-ipomeanol%2520serves%2520as%2520a%2520novel%2520suicide%2520gene%2520system%2520for%2520adoptive%2520T-cell%2520therapies%26jtitle%3DGene%2520Ther.%26date%3D2016%26volume%3D23%26spage%3D615%26epage%3D626%26doi%3D10.1038%2Fgt.2016.38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kowalski, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittington, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guttman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girhard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanenberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships for CYP4B1 bioactivation of 4-ipomeanol congeners: Direct correlation between cytotoxicity and trapped reactive intermediates</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2488</span>– <span class="NLM_lpage">2498</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrestox.9b00330</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.9b00330" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjtL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2019&pages=2488-2498&author=J.+P.+Kowalskiauthor=M.+G.+McDonaldauthor=D.+Whittingtonauthor=M.+Guttmanauthor=M.+Scianauthor=M.+Girhardauthor=H.+Hanenbergauthor=C.+Wiekauthor=A.+E.+Rettie&title=Structure-activity+relationships+for+CYP4B1+bioactivation+of+4-ipomeanol+congeners%3A+Direct+correlation+between+cytotoxicity+and+trapped+reactive+intermediates&doi=10.1021%2Facs.chemrestox.9b00330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships for CYP4B1 Bioactivation of 4-Ipomeanol Congeners: Direct Correlation between Cytotoxicity and Trapped Reactive Intermediates</span></div><div class="casAuthors">Kowalski, John P.; McDonald, Matthew G.; Whittington, Dale; Guttman, Miklos; Scian, Michele; Girhard, Marco; Hanenberg, Helmut; Wiek, Constanze; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2488-2498</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cytochrome P 450 4B1 (CYP4B1) has been explored as a candidate enzyme in suicide gene systems for its ability to bioactivate the natural product 4-ipomeanol (IPO) to a reactive species that causes cytotoxicity.  However, metabolic limitations of IPO necessitate discovery of new "pro-toxicant" substrates for CYP4B1.  In the present study, we examd. a series of synthetically facile N-alkyl-3-furancarboxamides for cytotoxicity in HepG2 cells expressing CYP4B1.  This compd. series maintains the furan "warhead" of IPO while replacing its alc. group with alkyl chains of varying length (C1 - C8).  Compds. with C3 - C6 carbon chain lengths showed similar potency to IPO (LD50 ∼ 5 μM).  Short chain analogs (<3 carbons) and long chain analogs (>6 carbons) exhibited reduced toxicity, resulting in a parabolic relationship between alkyl chain length and cytotoxicity.  A similar parabolic relationship was obsd. between alkyl chain length and reactive intermediate formation upon trapping of the putative enedial as a stable pyrrole adduct in incubations with purified recombinant rabbit CYP4B1 and common physiol. nucleophiles.  These parabolic relationships reflect the lower affinity of shorter chain compds. for CYP4B1 and increased ω-hydroxylation of the longer chain compds. by the enzyme.  Furthermore, modest time-dependent inhibition of CYP4B1 by N-pentyl-3-furancarboxamide was completely abolished when trapping agents were added, demonstrating escape of reactive intermediates from the enzyme after bioactivation.  An insulated CYP4B1 active site may explain the rarely obsd. direct correlation between adduct formation and cell toxicity reported here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUI6iSmCq-nrVg90H21EOLACvtfcHk0lgqoq5BTN6yKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjtL3P&md5=9a86a33de540c043ab2aa920cbaa4576</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.9b00330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.9b00330%26sid%3Dliteratum%253Aachs%26aulast%3DKowalski%26aufirst%3DJ.%2BP.%26aulast%3DMcDonald%26aufirst%3DM.%2BG.%26aulast%3DWhittington%26aufirst%3DD.%26aulast%3DGuttman%26aufirst%3DM.%26aulast%3DScian%26aufirst%3DM.%26aulast%3DGirhard%26aufirst%3DM.%26aulast%3DHanenberg%26aufirst%3DH.%26aulast%3DWiek%26aufirst%3DC.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DStructure-activity%2520relationships%2520for%2520CYP4B1%2520bioactivation%2520of%25204-ipomeanol%2520congeners%253A%2520Direct%2520correlation%2520between%2520cytotoxicity%2520and%2520trapped%2520reactive%2520intermediates%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2019%26volume%3D32%26spage%3D2488%26epage%3D2498%26doi%3D10.1021%2Facs.chemrestox.9b00330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">P450-Based drug-drug interactions of amiodarone and its metabolites: Diversity of inhibitory mechanisms</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1661</span>– <span class="NLM_lpage">1669</span>, <span class="refDoi"> DOI: 10.1124/dmd.115.065623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.115.065623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=26296708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28Xis1Khsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=1661-1669&author=M.+G.+McDonaldauthor=N.+T.+Auauthor=A.+E.+Rettie&title=P450-Based+drug-drug+interactions+of+amiodarone+and+its+metabolites%3A+Diversity+of+inhibitory+mechanisms&doi=10.1124%2Fdmd.115.065623"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">P450-based drug-drug interactions of amiodarone and its metabolites: diversity of inhibitory mechanisms</span></div><div class="casAuthors">McDonald, Matthew G.; Au, Nicholas T.; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1661-1669</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">In this study, IC50 shift and time-dependent inhibition (TDI) expts. were carried out to measure the ability of amiodarone (AMIO), and its circulating human metabolites, to reversibly and irreversibly inhibit CYP1A2, CYP2C9, CYP2D6, and CYP3A4 activities in human liver microsomes.  The [I]u/Ki,u values were calcd. and used to predict in vivo AMIO drug-drug interactions (DDIs) for pharmaceuticals metabolized by these four enzymes.  Based on these values, the minor metabolite N,N-didesethylamiodarone (DDEA) is predicted to be the major cause of DDIs with xenobiotics primarily metabolized by CYP1A2, CYP2C9, or CYP3A4, while AMIO and its N-monodesethylamiodarone (MDEA) deriv. are the most likely cause of interactions involving inhibition of CYP2D6 metab.  AMIO drug interactions predicted from the reversible inhibition of the four P 450 activities were found to be in good agreement with the magnitude of reported clin. DDIs with lidocaine, warfarin, metoprolol, and simvastatin.  The TDI expts. showed DDEA to be a potent inactivator of CYP1A2 (KI = 0.46 μM, kinact = 0.030 min-1), while MDEA was a moderate inactivator of both CYP2D6 (KI = 2.7 μM, kinact = 0.018 min-1) and CYP3A4 (KI = 2.6 μM, kinact = 0.016 min-1).  For DDEA and MDEA, mechanism-based inactivation appears to occur through formation of a metabolic intermediate complex.  Addnl. metabolic studies strongly suggest that CYP3A4 is the primary microsomal enzyme involved in the metab. of AMIO to both MDEA and DDEA.  In summary, these studies demonstrate both the diversity of inhibitory mechanisms with AMIO and the need to consider metabolites as the culprit in inhibitory P 450-based DDIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6wo6rgx9Ej7Vg90H21EOLACvtfcHk0lgqoq5BTN6yKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xis1Khsbg%253D&md5=712b2fec18f4c4aa416fd4d7d9d23819</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.065623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.065623%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DM.%2BG.%26aulast%3DAu%26aufirst%3DN.%2BT.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DP450-Based%2520drug-drug%2520interactions%2520of%2520amiodarone%2520and%2520its%2520metabolites%253A%2520Diversity%2520of%2520inhibitory%2520mechanisms%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26spage%3D1661%26epage%3D1669%26doi%3D10.1124%2Fdmd.115.065623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref58':['cit58'],'ref59':['cit59'],'ref56':['cit56'],'ref57':['cit57'],'ref60':['cit60'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref53':['cit53'],'ref27':['cit27'],'ref54':['cit54'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref55':['cit55'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">M. Saeed Mirzaei, Maxim V. Ivanov, Avat Arman Taherpour, <span class="NLM_string-name hlFld-ContribAuthor">Saber Mirzaei</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanism-Based Inactivation of Cytochrome P450 Enzymes: Computational Insights. </span><span class="cited-content_cbyCitation_journal-name">Chemical Research in Toxicology</span><span> <strong>2021,</strong> <em>34 </em>
                                    (4)
                                     , 959-987. <a href="https://doi.org/10.1021/acs.chemrestox.0c00483" title="DOI URL">https://doi.org/10.1021/acs.chemrestox.0c00483</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrestox.0c00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrestox.0c00483%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Research%2520in%2520Toxicology%26atitle%3DMechanism-Based%252BInactivation%252Bof%252BCytochrome%252BP450%252BEnzymes%25253A%252BComputational%252BInsights%26aulast%3DMirzaei%26aufirst%3DM.%2BSaeed%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D10112020%26date%3D26032021%26volume%3D34%26issue%3D4%26spage%3D959%26epage%3D987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John P.  Kowalski</span>, <span class="hlFld-ContribAuthor ">Robert D.  Pelletier</span>, <span class="hlFld-ContribAuthor ">Matthew G.  McDonald</span>, <span class="hlFld-ContribAuthor ">Edward J.  Kelly</span>, <span class="hlFld-ContribAuthor ">Allan E.  Rettie</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacokinetics, metabolism and off-target effects in the rat of 8-[(1H- benzotriazol-1-yl)amino]octanoic acid, a selective inhibitor of human cytochrome P450 4Z1:
              β
              -oxidation as a potential augmenting pathway for inhibition. </span><span class="cited-content_cbyCitation_journal-name">Xenobiotica</span><span> <strong>2021,</strong> <em>51 </em>
                                    (8)
                                     , 901-915. <a href="https://doi.org/10.1080/00498254.2021.1930281" title="DOI URL">https://doi.org/10.1080/00498254.2021.1930281</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/00498254.2021.1930281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F00498254.2021.1930281%26sid%3Dliteratum%253Aachs%26jtitle%3DXenobiotica%26atitle%3DPharmacokinetics%25252C%252Bmetabolism%252Band%252Boff-target%252Beffects%252Bin%252Bthe%252Brat%252Bof%252B8-%25255B%2525281H-%252Bbenzotriazol-1-yl%252529amino%25255Doctanoic%252Bacid%25252C%252Ba%252Bselective%252Binhibitor%252Bof%252Bhuman%252Bcytochrome%252BP450%252B4Z1%25253A%252B%2525CE%2525B2%252B-oxidation%252Bas%252Ba%252Bpotential%252Baugmenting%252Bpathway%252Bfor%252Binhibition%26aulast%3DKowalski%26aufirst%3DJohn%2BP.%26date%3D2021%26date%3D2021%26volume%3D51%26issue%3D8%26spage%3D901%26epage%3D915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hai  Liu</span>, <span class="hlFld-ContribAuthor ">Hai  Qin</span>, <span class="hlFld-ContribAuthor ">Yi  Zhou</span>, <span class="hlFld-ContribAuthor ">Yin  Yuan</span>, <span class="hlFld-ContribAuthor ">Yichen  Liu</span>, <span class="hlFld-ContribAuthor ">Ying  Chen</span>, <span class="hlFld-ContribAuthor ">Yue  Yang</span>, <span class="hlFld-ContribAuthor ">Haiwei  Ni</span>, <span class="hlFld-ContribAuthor ">Tao  Xi</span>, <span class="hlFld-ContribAuthor ">Lufeng  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">HET0016 attenuates the stemness of breast cancer cells through targeting CYP4Z1. </span><span class="cited-content_cbyCitation_journal-name">Molecular Carcinogenesis</span><span> <strong>2021,</strong> <em>60 </em>
                                    (6)
                                     , 413-426. <a href="https://doi.org/10.1002/mc.23302" title="DOI URL">https://doi.org/10.1002/mc.23302</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/mc.23302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmc.23302%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Carcinogenesis%26atitle%3DHET0016%252Battenuates%252Bthe%252Bstemness%252Bof%252Bbreast%252Bcancer%252Bcells%252Bthrough%252Btargeting%252BCYP4Z1%26aulast%3DLiu%26aufirst%3DHai%26date%3D2021%26date%3D2021%26volume%3D60%26issue%3D6%26spage%3D413%26epage%3D426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Machalz</span>, <span class="hlFld-ContribAuthor ">Hongjie  Li</span>, <span class="hlFld-ContribAuthor ">Wei  Du</span>, <span class="hlFld-ContribAuthor ">Shishir  Sharma</span>, <span class="hlFld-ContribAuthor ">Sijie  Liu</span>, <span class="hlFld-ContribAuthor ">Matthias  Bureik</span>, <span class="hlFld-ContribAuthor ">Gerhard  Wolber</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a novel potent cytochrome P450 CYP4Z1 inhibitor. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>215 </em>, 113255. <a href="https://doi.org/10.1016/j.ejmech.2021.113255" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113255</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113255%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bnovel%252Bpotent%252Bcytochrome%252BP450%252BCYP4Z1%252Binhibitor%26aulast%3DMachalz%26aufirst%3DDavid%26date%3D2021%26volume%3D215%26spage%3D113255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Barbara  Licznerska</span>, <span class="hlFld-ContribAuthor ">Wanda  Baer-Dubowska</span>. </span><span class="cited-content_cbyCitation_article-title">What might the presence of ‘orphan’ CYP450 isoforms in breast epithelial cells mean for the future of targeted therapeutics?. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Metabolism & Toxicology</span><span> <strong>2021,</strong> <em>17 </em>
                                    (2)
                                     , 135-137. <a href="https://doi.org/10.1080/17425255.2021.1844182" title="DOI URL">https://doi.org/10.1080/17425255.2021.1844182</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17425255.2021.1844182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17425255.2021.1844182%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Metabolism%2520%2526%2520Toxicology%26atitle%3DWhat%252Bmight%252Bthe%252Bpresence%252Bof%252B%2525E2%252580%252598orphan%2525E2%252580%252599%252BCYP450%252Bisoforms%252Bin%252Bbreast%252Bepithelial%252Bcells%252Bmean%252Bfor%252Bthe%252Bfuture%252Bof%252Btargeted%252Btherapeutics%25253F%26aulast%3DLicznerska%26aufirst%3DBarbara%26date%3D2021%26date%3D2020%26volume%3D17%26issue%3D2%26spage%3D135%26epage%3D137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. CYP4Z1 metabolizes substrates internally. The primary oxidation positions for substrates arachidonic acid (AA), lauric acid (LA), and luciferin-benzyl ether (Luc-BE) are denoted by the red arrows, generating products 14,15-epoxyeicosatrienoic acid (14,15-EET), 8-hydroxy LA, and luciferin (via O-dealkylation), respectively.<a onclick="showRef(event, 'ref8 ref15 ref54'); return false;" href="javascript:void(0);" class="ref ref8 ref15 ref54">(13,20,34)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Probe substrate Luc-BE metabolism in the HepG2 membranes is both CYP4Z1- and NADPH-dependent. (B) CYP4Z1-HepG2 membrane metabolism of Luc-BE displays Michaelis–Menten kinetics at the substrate concentrations tested, with a <i>K</i><sub>m</sub> = 29 μM and a <i>V</i><sub>max</sub> = 20 pmol/pmol CYP4Z1/min; the Eadie–Hofstee plot (inset) shows the apparent linearity. Shown is the average ± standard deviation (SD) from at least three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>ABT</b> Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Celite, PCC, CH<sub>2</sub>Cl<sub>2</sub>, room temperature (RT), 1 h, 21–25% (analogues <b>6</b>–<b>9</b>); (b) acetic acid, RT, 18 h, 43–89%; and (c) NaBH<sub>4</sub>, CH<sub>2</sub>Cl<sub>2</sub>, MeOH, RT, 2 h, 56–84%</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Time-dependent inhibition of CYP4Z1-mediated Luc-BE O-debenzylation by <b>ABT</b> (red) and inhibitor <b>7</b> (blue). Shown is the percent activity remaining versus inhibitor concentration, with (dashed lines) and without (solid lines) the presence of NADPH in a preincubation step. The <b>ABT</b> analogue <b>7</b> exhibited gains in the affinity and rate of enzyme inactivation for CYP4Z1 compared to the parent compound. IC<sub>50</sub> values are reported in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Shown is the average ± SD from three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Time-dependent inhibition of CYP4A11-mediated Luc-4A O-demethylation; CYP4F2-, CYP4F3a-, and CYP4F3b-mediated Luc-4F2/3 O-(4-methylthio)-debenzylation; and CYP4F8- and CYP4F12-mediated Luc-BE O-debenzylation by <b>ABT</b> analogues screened at 30 μM. The pan-CYP4 inhibitor HET0016 was dosed at 0.5 μM as an inhibition control. All probe substrates were used at concentrations ≤<i>K</i><sub>m</sub>. All enzymes were recombinantly expressed in insect cells (Supersomes), except for CYP4F8, which was recombinantly expressed in HepG2 cells. The black and gray bars show the percent activity remaining when the inhibitor was preincubated for 30 min with enzyme plus and minus NADPH, respectively. Shown is the average ± SD from three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Characterization of mechanism-based inactivation of CYP4Z1 by <b>7</b>. Time- and concentration-dependent loss of CYP4Z1-mediated Luc-BE O-debenzylation activity from treatment with <b>7</b> (<i>K</i><sub>I</sub> = 2.2 μM, <i>k</i><sub>inact</sub> = 0.15 min<sup>–1</sup>) (A, B). The percent of CYP4Z1 activity remaining when inhibitor <b>7</b> was preincubated with enzyme plus (black bars) and minus (gray bars) NADPH was assessed both prior to (pre) and after (post) overnight dialysis. The TDI of CYP4Z1 was not affected by this dialysis step (C). The percent of CYP4Z1 activity remaining when inhibitor <b>7</b> was preincubated with enzyme plus NADPH was assessed with the following nucleophilic additives: glutathione (GSH), <i>N</i>-acetyl-lysine (NAL), methoxyamine (MeONH<sub>2</sub>), <i>N</i>-acetyl-cysteine (NAC), and semicarbazide (semicarb.) at 5 mM and potassium cyanide (KCN) at 1 mM. Inclusion of these trapping agents did not significantly abrogate the TDI of CYP4Z1 by <b>7</b>. Similarly, the addition of the reactive oxygen species-scavenging enzymes, catalase (CAT) and superoxide dismutase (SOD), did not affect the TDI of CYP4Z1 by <b>7</b> (D). In both (C) and (D), the minus NADPH sample controls exhibit cofactor dependency for the bioactivation reaction. A plot of the percent of CYP4Z1 activity, when inhibitor <b>7</b> was preincubated with enzyme plus NADPH, versus the concentration ratio of <b>7</b>/CYP4Z1 was used to determine the number of inactivator molecules converted and released as a product relative to each inactivation event. Extrapolation of the initial linear portion of the plot to the <i>x</i>-intercept (inset) shows a turnover number of 15, resulting in a partition ratio of ∼14 (E). For all assays, data shown is the average ± SD from three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/medium/jm0c00101_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of CYP4Z1-mediated AA metabolism in T47D whole cells by <b>7</b>. Liquid chromatography–tandem mass spectrometry (LC–MS/MS) chromatograms showing the AA metabolite profile from CYP4Z1-T47D cells treated with dimethyl sulfoxide (DMSO) vehicle (bottom trace) or inhibitor <b>7</b> (middle trace) and vector-T47D cells treated with the vehicle (top trace). Cells were treated with 3 μM <b>7</b>, or the vehicle, and underwent a 24 h preincubation prior to the addition of 50 μM AA, which was followed by a 2 h incubation for substrate metabolism. This resulted in 14,15-EET, 19-HETE, and 14,15-DiHET metabolite formation being inhibited 83, 86, and 80%, respectively, as compared to vehicle-treated CYP4Z1-T47D cells. Vector-T47D cells dosed with the vehicle and 50 μM AA showed a minimal amount of background AA metabolism.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00101/20200507/images/large/jm0c00101_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00101&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i45">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81801" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81801" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz de Montellano, P. R.</span></span> <i>Cytochrome P450: Structure, Mechanism, and Biochemistry</i>; <span class="NLM_publisher-name">Kluwer Academic Publishers</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2005</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1007%2Fb139087" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=P.+R.+Ortiz+de+Montellano&title=Cytochrome+P450%3A+Structure%2C+Mechanism%2C+and+Biochemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fb139087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252Fb139087%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26btitle%3DCytochrome%2520P450%253A%2520Structure%252C%2520Mechanism%252C%2520and%2520Biochemistry%26pub%3DKluwer%2520Academic%2520Publishers%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rieger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiessling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohayem, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Temme, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieber, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigle, B.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel mammary-restricted cytochrome P450, CYP4Z1, with overexpression in breast carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">2357</span>– <span class="NLM_lpage">2364</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-03-0849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1158%2F0008-5472.CAN-03-0849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15059886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1yjtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=2357-2364&author=M.+A.+Riegerauthor=R.+Ebnerauthor=D.+R.+Bellauthor=A.+Kiesslingauthor=J.+Rohayemauthor=M.+Schmitzauthor=A.+Temmeauthor=E.+P.+Rieberauthor=B.+Weigle&title=Identification+of+a+novel+mammary-restricted+cytochrome+P450%2C+CYP4Z1%2C+with+overexpression+in+breast+carcinoma&doi=10.1158%2F0008-5472.CAN-03-0849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Novel Mammary-Restricted Cytochrome P450, CYP4Z1, with Overexpression in Breast Carcinoma</span></div><div class="casAuthors">Rieger, Michael A.; Ebner, Reinhard; Bell, David R.; Kiessling, Andrea; Rohayem, Jacques; Schmitz, Marc; Temme, Achim; Rieber, E. Peter; Weigle, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2357-2364</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">By screening a transcriptome database for expressed sequence tags that are specifically expressed in mammary gland and breast carcinoma, the authors identified a new human cytochrome P 450 (CYP), termed CYP4Z1.  The cDNA was cloned from the breast carcinoma line SK-BR-3 and codes for a protein of 505 amino acids.  Moreover, a transcribed pseudogene CYP4Z2P that codes for a truncated CYP protein (340 amino acids) with 96% identity to CYP4Z1 was found in SK-BR-3.  CYP4Z1 and CYP4Z2P genes consisting of 12 exons are localized in head-to-head orientation on chromosome 1p33.  Tissue-specific expression was investigated using real-time reverse transcription PCR with normalized cDNA from 18 different human tissues.  CYP4Z1 mRNA was preferentially detected in breast carcinoma tissue and mammary gland, whereas only marginal expression was found in all other tested tissues.  Investigation of cDNA pairs from tumor/normal tissues obtained from 241 patients, including 50 breast carcinomas, confirmed the breast-restricted expression and showed a clear overexpression in 52% of breast cancer samples.  The expression profile of CYP4Z2P was similar to that of CYP4Z1 with preference in breast carcinoma and mammary gland but a lower expression level in general.  Immunoblot analyses with a specific antiserum for CYP4Z1 clearly demonstrated protein expression in mammary gland and breast carcinoma tissue specimens as well as in CYP4Z1-transduced cell lines.  Confocal laser-scanning microscopy of MCF-7 cells transfected with a fluorescent fusion protein CYP4Z1-enhanced green fluorescent protein and a subcellular fractionation showed localization to the endoplasmic reticulum as an integral membrane protein concordant for microsomal CYP enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGL-DjAe1fJrVg90H21EOLACvtfcHk0lj2uXr86WK2mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1yjtL4%253D&md5=e9000b09c4ca345038a79b42434fc40b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-03-0849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-03-0849%26sid%3Dliteratum%253Aachs%26aulast%3DRieger%26aufirst%3DM.%2BA.%26aulast%3DEbner%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DD.%2BR.%26aulast%3DKiessling%26aufirst%3DA.%26aulast%3DRohayem%26aufirst%3DJ.%26aulast%3DSchmitz%26aufirst%3DM.%26aulast%3DTemme%26aufirst%3DA.%26aulast%3DRieber%26aufirst%3DE.%2BP.%26aulast%3DWeigle%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520a%2520novel%2520mammary-restricted%2520cytochrome%2520P450%252C%2520CYP4Z1%252C%2520with%2520overexpression%2520in%2520breast%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D2357%26epage%3D2364%26doi%3D10.1158%2F0008-5472.CAN-03-0849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radvanyi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh-Sandhu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallichan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovitt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedyczak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gish, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zubovits, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hakimi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shortreed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrington, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oomen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tartaglia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berinstein, N. L.</span></span> <span> </span><span class="NLM_article-title">The gene associated with trichorhinophalangeal syndrome in humans is overexpressed in breast cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">11005</span>– <span class="NLM_lpage">11010</span>, <span class="refDoi"> DOI: 10.1073/pnas.0500904102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1073%2Fpnas.0500904102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=16043716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnvVWjurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=11005-11010&author=L.+Radvanyiauthor=D.+Singh-Sandhuauthor=S.+Gallichanauthor=C.+Lovittauthor=A.+Pedyczakauthor=G.+Malloauthor=K.+Gishauthor=K.+Kwokauthor=W.+Hannaauthor=J.+Zubovitsauthor=J.+Armesauthor=D.+Venterauthor=J.+Hakimiauthor=J.+Shortreedauthor=M.+Donovanauthor=M.+Parringtonauthor=P.+Dunnauthor=R.+Oomenauthor=J.+Tartagliaauthor=N.+L.+Berinstein&title=The+gene+associated+with+trichorhinophalangeal+syndrome+in+humans+is+overexpressed+in+breast+cancer&doi=10.1073%2Fpnas.0500904102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The gene associated with trichorhinophalangeal syndrome in humans is overexpressed in breast cancer</span></div><div class="casAuthors">Radvanyi, Laszlo; Singh-Sandhu, Devender; Gallichan, Scott; Lovitt, Corey; Pedyczak, Artur; Mallo, Gustavo; Gish, Kurt; Kwok, Kevin; Hanna, Wedad; Zubovits, Judith; Armes, Jane; Venter, Deon; Hakimi, Jalil; Shortreed, Jean; Donovan, Melinda; Parrington, Mark; Dunn, Pamela; Oomen, Ray; Tartaglia, James; Berinstein, Neil L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">11005-11010</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A comprehensive differential gene expression screen on a panel of 54 breast tumors and >200 normal tissue samples using DNA microarrays revealed 15 genes specifically overexpressed in breast cancer.  One of the most prevalent genes found was trichorhinophalangeal syndrome type 1 (TRPS-1), a gene previously shown to be assocd. with three rare autosomal dominant genetic disorders known as the trichorhinophalangeal syndromes.  A no. of corroborating methodologies, including in situ hybridization, e-Northern anal. using ORF EST (ORESTES) and Unigene EST abundance anal., immunoblot and immunofluorescence anal. of breast tumor cell lines, and immunohistochem., confirmed the microarray findings.  Immunohistochem. anal. found TRPS-1 protein expressed in >90% of early- and late-stage breast cancer, including ductal carcinoma in situ and invasive ductal, lobular, and papillary carcinomas.  The TRPS-1 gene is also immunogenic with processed and presented peptides activating T cells found after vaccination of HLA-A2.1 transgenic mouse.  Human T cell lines from HLA-A*0201+ female donors exhibiting TRPS-1-specific cytotoxic T lymphocyte activity could also be generated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY3pLDTU2xnrVg90H21EOLACvtfcHk0lj2uXr86WK2mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnvVWjurs%253D&md5=9ad3f97c19cb1c5c36053c9faa376cfb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0500904102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0500904102%26sid%3Dliteratum%253Aachs%26aulast%3DRadvanyi%26aufirst%3DL.%26aulast%3DSingh-Sandhu%26aufirst%3DD.%26aulast%3DGallichan%26aufirst%3DS.%26aulast%3DLovitt%26aufirst%3DC.%26aulast%3DPedyczak%26aufirst%3DA.%26aulast%3DMallo%26aufirst%3DG.%26aulast%3DGish%26aufirst%3DK.%26aulast%3DKwok%26aufirst%3DK.%26aulast%3DHanna%26aufirst%3DW.%26aulast%3DZubovits%26aufirst%3DJ.%26aulast%3DArmes%26aufirst%3DJ.%26aulast%3DVenter%26aufirst%3DD.%26aulast%3DHakimi%26aufirst%3DJ.%26aulast%3DShortreed%26aufirst%3DJ.%26aulast%3DDonovan%26aufirst%3DM.%26aulast%3DParrington%26aufirst%3DM.%26aulast%3DDunn%26aufirst%3DP.%26aulast%3DOomen%26aufirst%3DR.%26aulast%3DTartaglia%26aufirst%3DJ.%26aulast%3DBerinstein%26aufirst%3DN.%2BL.%26atitle%3DThe%2520gene%2520associated%2520with%2520trichorhinophalangeal%2520syndrome%2520in%2520humans%2520is%2520overexpressed%2520in%2520breast%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D11005%26epage%3D11010%26doi%3D10.1073%2Fpnas.0500904102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Tumoral expression of drug and xenobiotic metabolizing enzymes in breast cancer patients of different ethnicities with implications to personalized medicine</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">4747</span> <span class="refDoi"> DOI: 10.1038/s41598-017-04250-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1038%2Fs41598-017-04250-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=28684774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC1cjlt1Ortw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&author=Y.+Liauthor=A.+Steppiauthor=Y.+Zhouauthor=F.+Maoauthor=P.+C.+Millerauthor=M.+M.+Heauthor=T.+Zhaoauthor=Q.+Sunauthor=J.+Zhang&title=Tumoral+expression+of+drug+and+xenobiotic+metabolizing+enzymes+in+breast+cancer+patients+of+different+ethnicities+with+implications+to+personalized+medicine&doi=10.1038%2Fs41598-017-04250-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Tumoral expression of drug and xenobiotic metabolizing enzymes in breast cancer patients of different ethnicities with implications to personalized medicine</span></div><div class="casAuthors">Li Yan; Zhou Yidong; Mao Feng; Sun Qiang; Steppi Albert; Zhang Jinfeng; Miller Philip Craig; He Max M; He Max M; Zhao Tingting</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4747</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug and xenobiotic metabolizing enzymes (DXME) play important roles in drug responses and carcinogenesis.  Recent studies have found that expression of DXME in cancer cells significantly affects drug clearance and the onset of drug resistance.  In this study we compared the expression of DXME in breast tumor tissue samples from patients representing three ethnic groups: Caucasian Americans (CA), African Americans (AA), and Asian Americans (AS).  We further combined DXME gene expression data with eQTL data from the GTEx project and with allele frequency data from the 1000 Genomes project to identify SNPs that may be associated with differential expression of DXME genes.  We identified substantial differences among CA, AA, and AS populations in the expression of DXME genes and in activation of pathways involved in drug metabolism, including those involved in metabolizing chemotherapy drugs that are commonly used in the treatment of breast cancer.  These data suggest that differential expression of DXME may associate with health disparities in breast cancer outcomes observed among these three ethnic groups.  Our study suggests that development of personalized treatment strategies for breast cancer patients could be improved by considering both germline genotypes and tumor specific mutations and expression profiles related to DXME genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuRoK-jpntjJ6KlWarZTAzfW6udTcc2ea7tnjFD2Ayurntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjlt1Ortw%253D%253D&md5=afebeeb85f972ffb43e23f1fc1887c8e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-04250-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-04250-2%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSteppi%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DMao%26aufirst%3DF.%26aulast%3DMiller%26aufirst%3DP.%2BC.%26aulast%3DHe%26aufirst%3DM.%2BM.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DTumoral%2520expression%2520of%2520drug%2520and%2520xenobiotic%2520metabolizing%2520enzymes%2520in%2520breast%2520cancer%2520patients%2520of%2520different%2520ethnicities%2520with%2520implications%2520to%2520personalized%2520medicine%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26doi%3D10.1038%2Fs41598-017-04250-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, W. W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bureik, M.</span></span> <span> </span><span class="NLM_article-title">CYP4Z1 - A human cytochrome P450 enzyme that might hold the key to curing breast cancer</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2060</span>– <span class="NLM_lpage">2064</span>, <span class="refDoi"> DOI: 10.2174/1381612823666170207150156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.2174%2F1381612823666170207150156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=28176668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVShtrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=2060-2064&author=X.+Yangauthor=M.+Hutterauthor=W.+W.+B.+Gohauthor=M.+Bureik&title=CYP4Z1+-+A+human+cytochrome+P450+enzyme+that+might+hold+the+key+to+curing+breast+cancer&doi=10.2174%2F1381612823666170207150156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">CYP4Z1 - A Human Cytochrome P450 Enzyme that Might Hold the Key to Curing Breast Cancer</span></div><div class="casAuthors">Yang, Xu; Hutter, Michael; Bin Goh, Wilson Wen; Bureik, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2060-2064</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The human cytochrome P 450 (CYP) enzyme CYP4Z1 is a fatty acid hydroxylase which among human CYPs is unique for being much stronger expressed in the mammary gland than in all other tissues.  Moreover, it is strongly overexpressed in all subtypes of breast cancer, and some overexpression has also been found in other types of malignancies, such as ovarian, lung, and prostate cancers, resp.  Due to its unique expression pattern it is conceivable that this enzymes' activity might be exploited for a new therapeutic approach.  However, the main challenge for a CYP4Z1-based prodrug strategy (CBPS) for the treatment of breast cancer (and possibly other CYP4Z1-pos. malignancies) is the identification of candidate prodrugs that can be activated by this enzyme.  In this mini-review we summarize the current knowledge about the enzymic properties of the CYP4Z1 enzyme as well as on the expression pattern of the CYP4Z1 gene in both normal and cancer cells.  Moreover, we present the first homol. model of this enzyme and give an outlook on its potential use in cancer treatment strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9upj5h2gxrVg90H21EOLACvtfcHk0ljC__svD8C2TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVShtrvE&md5=4f225fc40f0df5be9b3d3eae1715ea4b</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.2174%2F1381612823666170207150156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612823666170207150156%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHutter%26aufirst%3DM.%26aulast%3DGoh%26aufirst%3DW.%2BW.%2BB.%26aulast%3DBureik%26aufirst%3DM.%26atitle%3DCYP4Z1%2520-%2520A%2520human%2520cytochrome%2520P450%2520enzyme%2520that%2520might%2520hold%2520the%2520key%2520to%2520curing%2520breast%2520cancer%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2017%26volume%3D23%26spage%3D2060%26epage%3D2064%26doi%3D10.2174%2F1381612823666170207150156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patimalla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heys, S. D.</span></span> <span> </span><span class="NLM_article-title">Profiling the expression of cytochrome P450 in breast cancer</span>. <i>Histopathology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2559.2010.03606.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1111%2Fj.1365-2559.2010.03606.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=20716162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC3cjlsFGnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2010&pages=202-211&author=G.+I.+Murrayauthor=S.+Patimallaauthor=K.+N.+Stewartauthor=I.+D.+Millerauthor=S.+D.+Heys&title=Profiling+the+expression+of+cytochrome+P450+in+breast+cancer&doi=10.1111%2Fj.1365-2559.2010.03606.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling the expression of cytochrome P450 in breast cancer</span></div><div class="casAuthors">Murray Graeme I; Patimalla Siva; Stewart Keith N; Miller Iain D; Heys Steven D</div><div class="citationInfo"><span class="NLM_cas:title">Histopathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">202-11</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">AIMS:  The cytochrome P450s (P450) are key oxidative enzymes that metabolize many carcinogens and anticancer drugs.  Thus, these enzymes influence tumour development, tumour response to therapy and are putative tumour biomarkers.  The aim was to define the P450 expression profile in breast cancer and establish the significance of P450 expression in this tumour type.  METHODS AND RESULTS:  A tissue microarray containing 170 breast cancers of no special type was immunostained for a panel of 21 P450s.  The highest percentage of strong immunopositivity in breast cancers was seen for CYP4X1 (50.8%), CYP2S1 (37.5%) and CYP2U1 (32.2%), while CYP2J (98.6%) and CYP3A43 (70.7%) were the P450s that most frequently displayed no immunoreactivity.  CYP4V2 (P = 0.01), CYP4X1 (P = 0.01) and CYP4Z1 (P = 0.01) showed correlations with tumour grade.  CYP1B1 (P = 0.001), CYP3A5 (P = 0.001) and CYP51 (P = 0.005) showed the most significant correlations with oestrogen receptor status.  Correlations with survival were identified for CYP2S1 (P = 0.03), CYP3A4 (P = 0.025), CYP4V2 (P = 0.026) and CYP26A1 (P = 0.03), although none of these P450s was an independent marker of prognosis.  CONCLUSIONS:  This study has defined the expression profile of cytochrome P450s in breast cancer and may offer their potential application as biomarkers to aid decisions regarding optimal adjuvant hormonal therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT0ZpvUgyuFaDAyhZbj3lKdfW6udTcc2eZnnFVOR-F9YLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjlsFGnsg%253D%253D&md5=96f08d44d30348e8511e91145b731b75</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2559.2010.03606.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2559.2010.03606.x%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DG.%2BI.%26aulast%3DPatimalla%26aufirst%3DS.%26aulast%3DStewart%26aufirst%3DK.%2BN.%26aulast%3DMiller%26aufirst%3DI.%2BD.%26aulast%3DHeys%26aufirst%3DS.%2BD.%26atitle%3DProfiling%2520the%2520expression%2520of%2520cytochrome%2520P450%2520in%2520breast%2520cancer%26jtitle%3DHistopathology%26date%3D2010%26volume%3D57%26spage%3D202%26epage%3D211%26doi%3D10.1111%2Fj.1365-2559.2010.03606.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Al-Esawi, N. S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geeran, A. R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alajeely, M. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Isawi, A. O. J.</span></span> <span> </span><span class="NLM_article-title">Expression of CYT4Z1 in breast carcinoma; Correlating with clinicopathological parameters</span>. <i>Biomed. Pharmacol. J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">511</span>, <span class="refDoi"> DOI: 10.13005/bpj/1912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.13005%2Fbpj%2F1912" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2020&pages=511&author=N.+S.+E.+Al-Esawiauthor=A.+R.+M.+Geeranauthor=M.+H.+J.+Alajeelyauthor=A.+O.+J.+Al-Isawi&title=Expression+of+CYT4Z1+in+breast+carcinoma%3B+Correlating+with+clinicopathological+parameters&doi=10.13005%2Fbpj%2F1912"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.13005%2Fbpj%2F1912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.13005%252Fbpj%252F1912%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Esawi%26aufirst%3DN.%2BS.%2BE.%26aulast%3DGeeran%26aufirst%3DA.%2BR.%2BM.%26aulast%3DAlajeely%26aufirst%3DM.%2BH.%2BJ.%26aulast%3DAl-Isawi%26aufirst%3DA.%2BO.%2BJ.%26atitle%3DExpression%2520of%2520CYT4Z1%2520in%2520breast%2520carcinoma%253B%2520Correlating%2520with%2520clinicopathological%2520parameters%26jtitle%3DBiomed.%2520Pharmacol.%2520J.%26date%3D2020%26volume%3D13%26spage%3D511%26doi%3D10.13005%2Fbpj%2F1912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falck, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span> <span> </span><span class="NLM_article-title">Increased expression of CYP4Z1 promotes tumor angiogenesis and growth in human breast cancer</span>. <i>Toxicol. Appl. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>264</i></span>,  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">83</span>, <span class="refDoi"> DOI: 10.1016/j.taap.2012.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.taap.2012.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=22841774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFGrsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=2012&pages=73-83&author=W.+Yuauthor=H.+Chaiauthor=Y.+Liauthor=H.+Zhaoauthor=X.+Xieauthor=H.+Zhengauthor=C.+Wangauthor=X.+Wangauthor=G.+Yangauthor=X.+Caiauthor=J.+R.+Falckauthor=J.+Yang&title=Increased+expression+of+CYP4Z1+promotes+tumor+angiogenesis+and+growth+in+human+breast+cancer&doi=10.1016%2Fj.taap.2012.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Increased expression of CYP4Z1 promotes tumor angiogenesis and growth in human breast cancer</span></div><div class="casAuthors">Yu, Wei; Chai, Hongyan; Li, Ying; Zhao, Haixia; Xie, Xianfei; Zheng, Hao; Wang, Chenlong; Wang, Xue; Yang, Guifang; Cai, Xiaojun; Falck, John R.; Yang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">264</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">73-83</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Cytochrome P 450 (CYP) 4Z1, a novel CYP4 family member, is over-expressed in human mammary carcinoma and assocd. with high-grade tumors and poor prognosis.  However, the precise role of CYP4Z1 in tumor progression is unknown.  Here, we demonstrate that CYP4Z1 overexpression promotes tumor angiogenesis and growth in breast cancer.  Stable expression of CYP4Z1 in T47D and BT-474 human breast cancer cells significantly increased mRNA expression and prodn. of vascular endothelial growth factor (VEGF)-A, and decreased mRNA levels and secretion of tissue inhibitor of metalloproteinase-2 (TIMP-2), without affecting cell proliferation and anchorage-independent cell growth in vitro.  Notably, the conditioned medium from CYP4Z1-expressing cells enhanced proliferation, migration and tube formation of human umbilical vein endothelial cells, and promoted angiogenesis in the zebrafish embryo and chorioallantoic membrane of the chick embryo.  In addn., there were lower levels of myristic acid and lauric acid, and higher contents of 20-hydroxyeicosatetraenoic acid (20-HETE) in CYP4Z1-expressing T47D cells compared with vector control.  CYP4Z1 overexpression significantly increased tumor wt. and microvessel d. by 2.6-fold and 1.9-fold in human tumor xenograft models, resp.  Moreover, CYP4Z1 transfection increased the phosphorylation of ERK1/2 and PI3K/Akt, while PI3K or ERK inhibitors and siRNA silencing reversed CYP4Z1-mediated changes in VEGF-A and TIMP-2 expression.  Conversely, HET0016, an inhibitor of the CYP4 family, potently inhibited the tumor-induced angiogenesis with assocd. changes in the intracellular levels of myristic acid, lauric acid and 20-HETE.  Collectively, these data suggest that increased CYP4Z1 expression promotes tumor angiogenesis and growth in breast cancer partly via PI3K/Akt and ERK1/2 activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI7xUctPOIvrVg90H21EOLACvtfcHk0lj2gw_wJQ4paw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFGrsrjO&md5=4b87a7625484e33ab82f9f7ba5b6ac92</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.taap.2012.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.taap.2012.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChai%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DCai%26aufirst%3DX.%26aulast%3DFalck%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DJ.%26atitle%3DIncreased%2520expression%2520of%2520CYP4Z1%2520promotes%2520tumor%2520angiogenesis%2520and%2520growth%2520in%2520human%2520breast%2520cancer%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2012%26volume%3D264%26spage%3D73%26epage%3D83%26doi%3D10.1016%2Fj.taap.2012.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edson, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">CYP4 enzymes as potential drug targets: focus on enzyme multiplicity, inducers and inhibitors, and therapeutic modulation of 20-hydroxyeicosatetraenoic acid (20-HETE) synthase and fatty acid omega-hydroxylase activities</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1429</span>– <span class="NLM_lpage">1440</span>, <span class="refDoi"> DOI: 10.2174/15680266113139990110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.2174%2F15680266113139990110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=23688133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WnsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=1429-1440&author=K.+Z.+Edsonauthor=A.+E.+Rettie&title=CYP4+enzymes+as+potential+drug+targets%3A+focus+on+enzyme+multiplicity%2C+inducers+and+inhibitors%2C+and+therapeutic+modulation+of+20-hydroxyeicosatetraenoic+acid+%2820-HETE%29+synthase+and+fatty+acid+omega-hydroxylase+activities&doi=10.2174%2F15680266113139990110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">CYP4 Enzymes As Potential Drug Targets: Focus on Enzyme Multiplicity, Inducers and Inhibitors, and Therapeutic Modulation of 20- Hydroxyeicosatetraenoic Acid (20-HETE) Synthase and Fatty Acid ω-Hydroxylase Activities</span></div><div class="casAuthors">Edson, Katheryne Z.; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1429-1440</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The Cytochrome P 450 4 (CYP4) family of enzymes in humans is comprised of thirteen isoenzymes that typically catalyze the ω -oxidn. of endogenous fatty acids and eicosanoids.  Several CYP4 enzymes can biosynthesize 20- hydroxyeicosatetraenoic acid, or 20-HETE, an important signaling eicosanoid involved in regulation of vascular tone and kidney reabsorption.  Addnl., accumulation of certain fatty acids is a hallmark of the rare genetic disorders, Refsum disease and X-ALD.  Therefore, modulation of CYP4 enzyme activity, either by inhibition or induction, is a potential strategy for drug discovery.  Here we review the substrate specificities, sites of expression, genetic regulation, and inhibition by exogenous chems. of the human CYP4 enzymes, and discuss the targeting of CYP4 enzymes in the development of new treatments for hypertension, stroke, certain cancers and the fatty acid-linked orphan diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNu1Vocf00NrVg90H21EOLACvtfcHk0lgnWogZhEhhUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WnsrrK&md5=065a5955167e84b0c5cb16d1f5f225c1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F15680266113139990110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F15680266113139990110%26sid%3Dliteratum%253Aachs%26aulast%3DEdson%26aufirst%3DK.%2BZ.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DCYP4%2520enzymes%2520as%2520potential%2520drug%2520targets%253A%2520focus%2520on%2520enzyme%2520multiplicity%252C%2520inducers%2520and%2520inhibitors%252C%2520and%2520therapeutic%2520modulation%2520of%252020-hydroxyeicosatetraenoic%2520acid%2520%252820-HETE%2529%2520synthase%2520and%2520fatty%2520acid%2520omega-hydroxylase%2520activities%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26spage%3D1429%26epage%3D1440%26doi%3D10.2174%2F15680266113139990110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edson, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totah, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">Cytochrome P450 omega-hydroxylases in inflammation and cancer</span>. <i>Adv. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1016/bs.apha.2015.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fbs.apha.2015.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=26233909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2015&pages=223-262&author=A.+L.+Johnsonauthor=K.+Z.+Edsonauthor=R.+A.+Totahauthor=A.+E.+Rettie&title=Cytochrome+P450+omega-hydroxylases+in+inflammation+and+cancer&doi=10.1016%2Fbs.apha.2015.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cytochrome P450 ω-hydroxylases in inflammation and cancer</span></div><div class="casAuthors">Johnson, Amanda L.; Edson, Katheryne Z.; Totah, Rheem A.; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Pharmacology (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">Cytochrome P450 Function and Pharmacological Roles in Inflammation and Cancer</span>),
    <span class="NLM_cas:pages">223-262</span>CODEN:
                <span class="NLM_cas:coden">ADPHEL</span>;
        ISSN:<span class="NLM_cas:issn">1054-3589</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Cytochrome P 450-dependent ω-hydroxylation is a prototypic metabolic reaction of CYP4 family members that is important for the elimination and bioactivation of not only therapeutic drugs, but also endogenous compds., principally fatty acids.  Eicosanoids, derived from arachidonic acid, are key substrates in the latter category.  Human CYP4 enzymes, mainly CYP4A11, CYP4F2, and CYP4F3B, hydroxylate arachidonic acid at the omega position to form 20-HETE, which has important effects in tumor progression and on angiogenesis and blood pressure regulation in the vasculature and kidney.  CYP4F3A in myeloid tissue catalyzes the ω-hydroxylation of leukotriene B4 to 20-hydroxy leukotriene B4, an inactivation process that is crit. for the regulation of the inflammatory response.  Here, we review the enzymol., tissue distribution, and substrate selectivity of human CYP4 ω-hydroxylases and their roles as catalysts for the formation and termination of the biol. effects of key eicosanoid metabolites in inflammation and cancer progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohzOfE_2I0hLVg90H21EOLACvtfcHk0lhCB1oAg6gdPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSht70%253D&md5=eea5e4ceba74de8e981748c50ae358e8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fbs.apha.2015.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.apha.2015.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DA.%2BL.%26aulast%3DEdson%26aufirst%3DK.%2BZ.%26aulast%3DTotah%26aufirst%3DR.%2BA.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DCytochrome%2520P450%2520omega-hydroxylases%2520in%2520inflammation%2520and%2520cancer%26jtitle%3DAdv.%2520Pharmacol.%26date%3D2015%26volume%3D74%26spage%3D223%26epage%3D262%26doi%3D10.1016%2Fbs.apha.2015.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z. H.</span></span> <span> </span><span class="NLM_article-title">14,15-EET induces the infiltration and tumor-promoting function of neutrophils to trigger the growth of minimal dormant metastases</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">43324</span>– <span class="NLM_lpage">43336</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.18632%2Foncotarget.9709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=27270316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FntlGhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=43324-43336&author=J.+Luoauthor=X.+X.+Fengauthor=C.+Luoauthor=Y.+Wangauthor=D.+Liauthor=Y.+Shuauthor=S.+S.+Wangauthor=J.+Qinauthor=Y.+C.+Liauthor=J.+M.+Zouauthor=D.+A.+Tianauthor=G.+M.+Zhangauthor=Z.+H.+Feng&title=14%2C15-EET+induces+the+infiltration+and+tumor-promoting+function+of+neutrophils+to+trigger+the+growth+of+minimal+dormant+metastases&doi=10.18632%2Foncotarget.9709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">14,15-EET induces the infiltration and tumor-promoting function of neutrophils to trigger the growth of minimal dormant metastases</span></div><div class="casAuthors">Luo Jing; Luo Chao; Wang Yu; Li Dong; Shu Yu; Wang Shan-Shan; Qin Jian; Li Yong-Chao; Zou Jiu-Ming; Zhang Gui-Mei; Feng Zuo-Hua; Feng Xin-Xia; Tian De-An</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">43324-43336</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Infiltrating neutrophils are known to promote in the development of tumor.  However, it is unclear whether and how neutrophils are involved in triggering the growth of dormant metastases.  Here we show that 14,15-epoxyeicosatrienoic acid (14,15-EET) can trigger the growth of dormant micrometastases by inducing neutrophilic infiltration and converting neutrophil function. 14,15-EET triggered neutrophil infiltration in metastatic lesions by activating STAT3 and JNK pathways to induce the expression of human IL-8 and murine CXCL15 in corresponding tumor cells.  The continuous expression of hIL-8/mCXCL15 was maintained by the sustained and enhanced activation of JNK pathway. 14,15-EET up-regulated miR-155 expression by activating STAT3 and JNK pathways. miR-155 in turn down-regulated the expression of SHIP1 and DET1, thus augmenting the activation of JNK and c-Jun.  Moreover, the function of neutrophils was converted from tumor-suppressing to tumor-promoting by 14,15-EET in vivo.  By inducing the production of G-CSF/IL-6 in vivo, 14,15-EET induced the enhancement of STAT3 activation in neutrophils to increase MMP-9 expression and decrease TRAIL expression.  Neutrophil-derived MMP-9 was required for 14,15-EET to induce angiogenesis during the growth of dormant micrometastases.  Depleting neutrophils or inhibiting hIL-8/mCXCL15 up-regulation resulted in the failure of 14,15-EET to promote the development of micrometastases.  These findings reveal a mechanism through which the infiltration and tumor-promoting function of neutrophils could be induced to trigger the growth of dormant metastases, which might be a driving force for the tumor recurrence based on dormant metastases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRhpIKq3LL6myEJCPKhJPuqfW6udTcc2ebKHazwhbEVNbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FntlGhsw%253D%253D&md5=6bebae354cacd572220eddae42b07ba4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9709%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%2BX.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DShu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%2BS.%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%2BC.%26aulast%3DZou%26aufirst%3DJ.%2BM.%26aulast%3DTian%26aufirst%3DD.%2BA.%26aulast%3DZhang%26aufirst%3DG.%2BM.%26aulast%3DFeng%26aufirst%3DZ.%2BH.%26atitle%3D14%252C15-EET%2520induces%2520the%2520infiltration%2520and%2520tumor-promoting%2520function%2520of%2520neutrophils%2520to%2520trigger%2520the%2520growth%2520of%2520minimal%2520dormant%2520metastases%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D43324%26epage%3D43336%26doi%3D10.18632%2Foncotarget.9709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panigrahy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeldin, D. C.</span></span> <span> </span><span class="NLM_article-title">EET signaling in cancer</span>. <i>Cancer Metastasis Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">540</span>, <span class="refDoi"> DOI: 10.1007/s10555-011-9315-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1007%2Fs10555-011-9315-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=22009066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1emt7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=525-540&author=D.+Panigrahyauthor=E.+R.+Greeneauthor=A.+Pozziauthor=D.+W.+Wangauthor=D.+C.+Zeldin&title=EET+signaling+in+cancer&doi=10.1007%2Fs10555-011-9315-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">EET signaling in cancer</span></div><div class="casAuthors">Panigrahy, Dipak; Greene, Emily R.; Pozzi, Ambra; Wang, Dao Wen; Zeldin, Darryl C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer and Metastasis Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">525-540</span>CODEN:
                <span class="NLM_cas:coden">CMRED4</span>;
        ISSN:<span class="NLM_cas:issn">0167-7659</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Inflammation and angiogenesis in the tumor microenvironment are increasingly implicated in tumorigenesis.  Endogenously produced lipid autacoids, locally acting small-mol. mediators, play a central role in inflammation and tissue homeostasis.  These lipid mediators, collectively referred to as eicosanoids, have recently been implicated in cancer.  Although eicosanoids, including prostaglandins and leukotrienes, are best known as products of arachidonic acid metab. by cyclooxygenases and lipoxygenases, arachidonic acid is also a substrate for another enzymic pathway, the cytochrome P 450 (CYP) system.  This eicosanoid pathway consists of two main branches: ω-hydroxylases which converts arachidonic acid to hydroxyeicosatetraenoic acids (HETEs) and epoxygenases which converts it to four regioisomeric epoxyeicosatrienoic acids (EETs; 5,6-EET, 8,9-EET, 11,12-EET, and 14,15-EET).  EETs regulate inflammation and vascular tone.  The bioactive EETs are produced predominantly in the endothelium and are mainly metabolized by sol. epoxide hydrolase to less active dihydroxyeicosatrienoic acids.  EET signaling was originally studied in conjunction with inflammatory and cardiovascular disease.  Arachidonic acid and its metabolites have recently stimulated great interest in cancer biol.  To date, most research on eicosanoids in cancer has focused on the COX and LOX pathways.  In contrast, the role of cytochrome P 450-derived eicosanoids, such as EETs and HETEs, in cancer has received little attention.  While CYP epoxygenases are expressed in human cancers and promote human cancer metastasis, the role of EETs (the direct products of CYP epoxygenases) in cancer remains poorly characterized.  In this review, the emerging role of EET signaling in angiogenesis, inflammation, and cancer is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMFV5maT9FHLVg90H21EOLACvtfcHk0lhCB1oAg6gdPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1emt7vL&md5=e6b8d1bef5d7f7a26eadb006a8b4d77c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs10555-011-9315-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10555-011-9315-y%26sid%3Dliteratum%253Aachs%26aulast%3DPanigrahy%26aufirst%3DD.%26aulast%3DGreene%26aufirst%3DE.%2BR.%26aulast%3DPozzi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DD.%2BW.%26aulast%3DZeldin%26aufirst%3DD.%2BC.%26atitle%3DEET%2520signaling%2520in%2520cancer%26jtitle%3DCancer%2520Metastasis%2520Rev.%26date%3D2011%26volume%3D30%26spage%3D525%26epage%3D540%26doi%3D10.1007%2Fs10555-011-9315-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amorosi, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitko, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalski, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totah, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">Expression and functional characterization of breast cancer-associated cytochrome P450 4Z1 in saccharomyces cerevisiae</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1364</span>– <span class="NLM_lpage">1371</span>, <span class="refDoi"> DOI: 10.1124/dmd.117.078188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.117.078188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=29018033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsVansL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=1364-1371&author=M.+G.+McDonaldauthor=S.+Rayauthor=C.+J.+Amorosiauthor=K.+A.+Sitkoauthor=J.+P.+Kowalskiauthor=L.+Pacoauthor=A.+Nathauthor=B.+Gallisauthor=R.+A.+Totahauthor=M.+J.+Dunhamauthor=D.+M.+Fowlerauthor=A.+E.+Rettie&title=Expression+and+functional+characterization+of+breast+cancer-associated+cytochrome+P450+4Z1+in+saccharomyces+cerevisiae&doi=10.1124%2Fdmd.117.078188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Expression and functional characterization of breast cancer-associated cytochrome P450 4Z1 in Saccharomyces cerevisiae</span></div><div class="casAuthors">McDonald, Matthew G.; Ray, Sutapa; Amorosi, Clara J.; Sitko, Katherine A.; Kowalski, John P.; Paco, Lorela; Nath, Abhinav; Gallis, Byron; Totah, Rheem A.; Dunham, Maitreya J.; Fowler, Douglas M.; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1364-1371</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">CYP4Z1 is an "orphan" cytochrome P 450 (P 450) enzyme that has provoked interest because of its hypothesized role in breast cancer through formation of the signaling mol. 20-hydroxyeicosatetraenoic acid (20-HETE).  We expressed human CYP4Z1 in Saccharomyces cerevisiae and evaluated its catalytic capabilities toward arachidonic and lauric acids (AA and LA).  Specific and sensitive mass spectrometry assays enabled discrimination of the regioselectivity of hydroxylation of these two fatty acids.  CYP4Z1 generated 7-, 8-, 9-, 10-, and 11-hydroxy LA, whereas the 12-hydroxy metabolite was not detected.  HET0016, the prototypic CYP4 inhibitor, only weakly inhibited laurate metabolite formation (IC50 ∼15 μM).  CYP4Z1 preferentially oxidized AA to the 14(S),15(R)-epoxide with high regioselectivity and stereoselectivity, a reaction that was also insensitive to HET0016, but neither 20-HETE nor 20-carboxy-AA were detectable metabolites.  Docking of LA and AA into a CYP4Z1 homol. model was consistent with this preference for internal fatty acid oxidn.  Thus, human CYP4Z1 has an inhibitor profile and product regioselectivity distinct from most other CYP4 enzymes, consistent with CYP4Z1's lack of a covalently linked heme.  These data suggest that, if CYP4Z1 modulates breast cancer progression, it does so by a mechanism other than direct prodn. of 20-HETE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYpQYvNbglrLVg90H21EOLACvtfcHk0ljzvgqWQmF2Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsVansL4%253D&md5=cc68fbb4ebb3b5d9f808005fab0e5458</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1124%2Fdmd.117.078188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.117.078188%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DM.%2BG.%26aulast%3DRay%26aufirst%3DS.%26aulast%3DAmorosi%26aufirst%3DC.%2BJ.%26aulast%3DSitko%26aufirst%3DK.%2BA.%26aulast%3DKowalski%26aufirst%3DJ.%2BP.%26aulast%3DPaco%26aufirst%3DL.%26aulast%3DNath%26aufirst%3DA.%26aulast%3DGallis%26aufirst%3DB.%26aulast%3DTotah%26aufirst%3DR.%2BA.%26aulast%3DDunham%26aufirst%3DM.%2BJ.%26aulast%3DFowler%26aufirst%3DD.%2BM.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DExpression%2520and%2520functional%2520characterization%2520of%2520breast%2520cancer-associated%2520cytochrome%2520P450%25204Z1%2520in%2520saccharomyces%2520cerevisiae%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2017%26volume%3D45%26spage%3D1364%26epage%3D1371%26doi%3D10.1124%2Fdmd.117.078188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">World Cancer Research
Fund</span>. <a href="https://www.wcrf.org/dietandcancer/cancer-trends/breast-cancer-statistics" class="extLink">https://www.wcrf.org/dietandcancer/cancer-trends/breast-cancer-statistics</a> (accessed Oct, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=World+Cancer+Research%0AFund.+https%3A%2F%2Fwww.wcrf.org%2Fdietandcancer%2Fcancer-trends%2Fbreast-cancer-statistics+%28accessed+Oct%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capper, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rae, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auchus, R. J.</span></span> <span> </span><span class="NLM_article-title">The metabolism, analysis, and targeting of steroid hormones in breast and prostate cancer</span>. <i>Horm. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1007/s12672-016-0259-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1007%2Fs12672-016-0259-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=26969590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28XktlCitr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=149-164&author=C.+P.+Capperauthor=J.+M.+Raeauthor=R.+J.+Auchus&title=The+metabolism%2C+analysis%2C+and+targeting+of+steroid+hormones+in+breast+and+prostate+cancer&doi=10.1007%2Fs12672-016-0259-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The Metabolism, Analysis, and Targeting of Steroid Hormones in Breast and Prostate Cancer</span></div><div class="casAuthors">Capper, Cameron P.; Rae, James M.; Auchus, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Hormones & Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">149-164</span>CODEN:
                <span class="NLM_cas:coden">HCOABS</span>;
        ISSN:<span class="NLM_cas:issn">1868-8497</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Breast and prostate cancers are malignancies in which steroid hormones drive cellular proliferation.  Over the past century, this understanding has led to successful treatment strategies aimed to inhibit hormone-mediated tumor growth.  Nonetheless, disease relapse and progression still pose significant clin. problems, with recurrent and metastatic tumors often exhibiting resistance to current drug therapies.  The central role of androgens and estrogens in prostate and breast cancer etiol. explains not only why endocrine therapies are often initially successful but also why many tumors ultimately become resistant.  It is hypothesized that reducing the concn. of active hormones in the systemic circulation may be insufficient to block cancer progression, as this action selects for tumor cells that can generate active steroids from circulating precursors.  This review aims to highlight the currently known differences of steroid biosynthesis in normal physiol. vs. hormone-dependent cancers, modern approaches to the assessment and targeting of these pathways, and priorities for future research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-7vQDqCGJ37Vg90H21EOLACvtfcHk0ljzvgqWQmF2Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktlCitr0%253D&md5=ad8c9d3ddd2e8534993da8b6ff0b427f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs12672-016-0259-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12672-016-0259-0%26sid%3Dliteratum%253Aachs%26aulast%3DCapper%26aufirst%3DC.%2BP.%26aulast%3DRae%26aufirst%3DJ.%2BM.%26aulast%3DAuchus%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520metabolism%252C%2520analysis%252C%2520and%2520targeting%2520of%2520steroid%2520hormones%2520in%2520breast%2520and%2520prostate%2520cancer%26jtitle%3DHorm.%2520Cancer%26date%3D2016%26volume%3D7%26spage%3D149%26epage%3D164%26doi%3D10.1007%2Fs12672-016-0259-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Njar, V. C.</span></span> <span> </span><span class="NLM_article-title">Targeting cytochrome P450 enzymes: a new approach in anti-cancer drug development</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">5047</span>– <span class="NLM_lpage">5060</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2007.05.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.bmc.2007.05.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=17544277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtlahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=5047-5060&author=R.+D.+Brunoauthor=V.+C.+Njar&title=Targeting+cytochrome+P450+enzymes%3A+a+new+approach+in+anti-cancer+drug+development&doi=10.1016%2Fj.bmc.2007.05.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cytochrome P450 enzymes: A new approach in anti-cancer drug development</span></div><div class="casAuthors">Bruno, Robert D.; Njar, Vincent C. O.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5047-5060</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cytochrome P450s (CYPs) represent a large class of heme-contg. enzymes that catalyze the metab. of multitudes of substrates both endogenous and exogenous.  Until recently, however, CYPs have been largely overlooked in cancer drug development, acknowledged only for their role in phase I metab. of chemotherapeutics.  The first successful strategy targeting CYP enzymes in cancer therapy was the development of potent inhibitors of CYP19 (aromatase) for the treatment of breast cancer.  Aromatase inhibitors ushered in a new era in hormone ablation therapy for estrogen dependent cancers, and have paved the way for similar strategies (i.e., inhibition of CYP17) that combat androgen-dependent prostate cancer.  Identification of CYPs involved in the inactivation of anticancer metabolites of vitamin D3 and vitamin A has triggered development of agents that target these enzymes as well.  The discovery of the over-expression of exogenous metabolizing CYPs, such as CYP1B1, in cancer cells has roused interest in the development of inhibitors for chemoprevention and of prodrugs designed to be activated by CYPs only in cancer cells.  Finally, the expression of CYPs within tumors has been utilized in the development of bioreductive mols. that are activated by CYPs only under hypoxic conditions.  This review offers the first comprehensive anal. of strategies in drug development that either inhibit or exploit CYP enzymes for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtuQVdXRAm27Vg90H21EOLACvtfcHk0ljzvgqWQmF2Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtlahsbg%253D&md5=bf2eab080cc0a7f7623c7af9942c359d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.05.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.05.046%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DR.%2BD.%26aulast%3DNjar%26aufirst%3DV.%2BC.%26atitle%3DTargeting%2520cytochrome%2520P450%2520enzymes%253A%2520a%2520new%2520approach%2520in%2520anti-cancer%2520drug%2520development%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D5047%26epage%3D5060%26doi%3D10.1016%2Fj.bmc.2007.05.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westerink, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoonen, W. G.</span></span> <span> </span><span class="NLM_article-title">Cytochrome P450 enzyme levels in HepG2 cells and cryopreserved primary human hepatocytes and their induction in HepG2 cells</span>. <i>Toxicol. In Vitro</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1581</span>– <span class="NLM_lpage">1591</span>, <span class="refDoi"> DOI: 10.1016/j.tiv.2007.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.tiv.2007.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=17637504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWrsrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=1581-1591&author=W.+M.+Westerinkauthor=W.+G.+Schoonen&title=Cytochrome+P450+enzyme+levels+in+HepG2+cells+and+cryopreserved+primary+human+hepatocytes+and+their+induction+in+HepG2+cells&doi=10.1016%2Fj.tiv.2007.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cytochrome P450 enzyme levels in HepG2 cells and cryopreserved primary human hepatocytes and their induction in HepG2 cells</span></div><div class="casAuthors">Westerink, Walter M. A.; Schoonen, Willem G. E. J.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology in Vitro</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1581-1591</span>CODEN:
                <span class="NLM_cas:coden">TIVIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0887-2333</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Early in vitro toxicity screening might improve the success rate of new chem. entities in pharmaceutical development.  In previous studies, the advantage of cytotoxicity screening with the HepG2 cell line was shown.  Cytotoxicity could be identified for 70% of the compds. in these assays as compared with known toxicity in either in vitro assays in primary hepatocytes, in in vivo assays in rats, or in (pre-)clin. development in humans.  The low Phase I and II enzyme levels in HepG2 cells might were responsible for the fact that 30% of the compds. scored neg.  Therefore, we performed 2 follow-up studies in which Cytochrome P 450 (CYP) enzymes and Phase II metab. were examd.  In the present study, the transcript levels of CYP1A1, 1A2, 2A6, 2B6, 2C8, 2C9, 2C19, 2D6, 2E1, and 3A4 were measured with quant. PCR.  Results showed that transcripts of all CYPs were present in HepG2 cells, however, mRNA levels of most CYPs were dramatically lower than in primary human hepatocytes.  These results were confirmed with luminometric assays which were used to measure the enzyme activities of CYP1A1, 1A2, 2C9, and 3A4.  Regulation of CYP1A1, 1A2, 2B6, 2C8, 2D6, 2E1, and 3A4 by the aryl hydrocarbon receptor, pregnane X receptor and constitutive androstane receptor was studied in HepG2 cells at the mRNA and/or enzyme level.  Regulation of CYP1A1, 1A2, 2B6, and 3A4 mRNA levels was similar to the regulation in primary human hepatocytes.  In contrast, CYP2C8 mRNA levels are inducible in primary human hepatocytes, but not in HepG2 cells, after treatment with PXR/CAR activators.  Consistent with other studies, CYP2D6 and 2E1 transcript levels were not changed after treatment with AhR, PXR, and CAR activators.  Moreover, CYP1A1 and 1A2 enzyme levels could be induced by AhR agonists and CYP3A4 by PXR agonists.  As a consequence of the low levels of CYPs in HepG2 cells, cytotoxicity of several compds. might were missed or underestimated as compared with cytotoxicity in primary human hepatocytes.  Inducing HepG2 cells with particular receptor stimulators might lead to higher toxicity for several of the tested compds.  Compared to primary human hepatocytes, HepG2 cells are a relatively easy-to-handle tool to study the up-regulation of CYP1A1, 1A2, 2B6, and 3A4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ0IAUSBPeubVg90H21EOLACvtfcHk0liXEggEiPpvVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWrsrvN&md5=b9d0fb486135d2ac6f9d58255204d1d3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2007.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2007.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DWesterink%26aufirst%3DW.%2BM.%26aulast%3DSchoonen%26aufirst%3DW.%2BG.%26atitle%3DCytochrome%2520P450%2520enzyme%2520levels%2520in%2520HepG2%2520cells%2520and%2520cryopreserved%2520primary%2520human%2520hepatocytes%2520and%2520their%2520induction%2520in%2520HepG2%2520cells%26jtitle%3DToxicol.%2520In%2520Vitro%26date%3D2007%26volume%3D21%26spage%3D1581%26epage%3D1591%26doi%3D10.1016%2Fj.tiv.2007.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kammerer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Küpper, J. H.</span></span> <span> </span><span class="NLM_article-title">NADPH-cytochrome P450 reductase expression and enzymatic activity in primary-like human hepatocytes and HepG2 cells for in vitro biotransformation studies</span>. <i>Clin. Hemorheol. Microcirc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.3233/CH-199226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.3233%2FCH-199226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=31561354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFGnurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2019&pages=249-260&author=C.+Schulzauthor=S.+Kammererauthor=J.+H.+K%C3%BCpper&title=NADPH-cytochrome+P450+reductase+expression+and+enzymatic+activity+in+primary-like+human+hepatocytes+and+HepG2+cells+for+in+vitro+biotransformation+studies&doi=10.3233%2FCH-199226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">NADPH-cytochrome P450 reductase expression and enzymatic activity in primary-like human hepatocytes and HepG2 cells for in vitro biotransformation studies</span></div><div class="casAuthors">Schulz, Christian; Kammerer, Sarah; Kuepper, Jan-Heiner; Wiggermann, P.; Krueger-Genge, A.; Jung, F.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Hemorheology and Microcirculation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">249-260</span>CODEN:
                <span class="NLM_cas:coden">CHMIFQ</span>;
        ISSN:<span class="NLM_cas:issn">1386-0291</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">BACKGROUND: : Human hepatocyte in vitro cell culture systems are important models for drug development and toxicol. studies in the context of liver xenobiotic metab.  Often, such culture systems are used to elucidate the biotransformation of xenobiotics or drugs and further investigate drug and drug metabolite effects on biol. systems in terms of potential therapeutic benefit or toxicity.  Human hepatocytes currently used for such in vitro studies are mostly primary cells or cell lines derived from liver cancers.  Both approaches have limitations such as low proliferation capacity and progressive dedifferentiation found in primary cells or lack of liver functions in cell lines, which makes it difficult to reliably predict biotransformation of xenobiotics in patients.  In order to overcome these limitations, HepaFH3 cells and Upcyte hepatocytes representing primary-like hepatocytes of the first and second generation are increasingly used.  Based on primary human hepatocyte cells transduced for stable expression of Upcyte proliferation genes, they are mitotically active and exhibit liver functions over an extended period, making them comparable to primary human hepatocytes.  These hepatocyte models show active liver metab. such as urea and glycogen formation as well as biotransformation of xenobiotics.  The latter is based on the expression, activity and inducibility of cytochrome P 450 enzymes (CYP) as essential phase I reaction components.  However, for further characterization in terms of performance and existing limitations, addnl. studies are needed to elucidate the mechanisms involved in phase I reactions.  One prerequisite is sufficient activity of microsomal NADPH-cytochrome P 450 reductase (POR) functionally connected as electron donor to those CYP enzymes.  OBJECTIVE: : For Upcyte hepatocytes and HepaFH3 cells, it is so far unknown to what extent POR is expressed, active, and may exert CYP-modulating effects.  Here we studied POR expression and corresponding enzyme activity in human hepatoblastoma cell line HepG2 and compared this with HepaFH3 and Upcyte hepatocytes representing proliferating primary-like hepatocytes.  METHODS: : POR expression of those hepatocyte models was detd. at mRNA and protein level using qRT-PCR, Western Blot and immunofluorescence staining.  Kinetic studies on POR activity in isolated microsomes were performed by a colorimetric method.  RESULTS: : The investigated hepatocyte models showed remarkable differences at the level of POR expression.  Compared to primary-like hepatocytes, POR expression of HepG2 cells was 4-fold higher at mRNA and 2-fold higher at protein level.  However, this higher expression did not correlate with corresponding enzyme activity levels in isolated microsomes, which were comparable between all cell systems tested.  A tendency of higher POR activity in HepG2 cells compared to HepaFH3 (p=0.0829) might be present.  Compared to primary human hepatocyte microsomes, POR activity was considerably lower in all hepatocyte models.  CONCLUSION: : In summary, our study revealed that POR expression and activity were clearly detectable in all in vitro hepatocyte models with the highest POR expression in cancer cell line HepG2.  However, POR activity was lower in tested hepatocyte models when compared to human primary hepatocyte microsomes.  Whether this was caused by e.g. polymorphisms or metabolic differences of investigated hepatocyte models will be target for future studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNB6apSkXyxbVg90H21EOLACvtfcHk0liXEggEiPpvVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFGnurzL&md5=d3961203b786c32242c01c8e5427fdc9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3233%2FCH-199226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FCH-199226%26sid%3Dliteratum%253Aachs%26aulast%3DSchulz%26aufirst%3DC.%26aulast%3DKammerer%26aufirst%3DS.%26aulast%3DK%25C3%25BCpper%26aufirst%3DJ.%2BH.%26atitle%3DNADPH-cytochrome%2520P450%2520reductase%2520expression%2520and%2520enzymatic%2520activity%2520in%2520primary-like%2520human%2520hepatocytes%2520and%2520HepG2%2520cells%2520for%2520in%2520vitro%2520biotransformation%2520studies%26jtitle%3DClin.%2520Hemorheol.%2520Microcirc.%26date%3D2019%26volume%3D73%26spage%3D249%26epage%3D260%26doi%3D10.3233%2FCH-199226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guengerich, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Q.</span></span> <span> </span><span class="NLM_article-title">Measurement of cytochrome P450 and NADPH-cytochrome P450 reductase</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1245</span>– <span class="NLM_lpage">1251</span>, <span class="refDoi"> DOI: 10.1038/nprot.2009.121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1038%2Fnprot.2009.121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=19661994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVygsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=1245-1251&author=F.+P.+Guengerichauthor=M.+V.+Martinauthor=C.+D.+Sohlauthor=Q.+Cheng&title=Measurement+of+cytochrome+P450+and+NADPH-cytochrome+P450+reductase&doi=10.1038%2Fnprot.2009.121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement of cytochrome P 450 and NADPH-cytochrome P 450 reductase</span></div><div class="casAuthors">Guengerich, F. Peter; Martin, Martha V.; Sohl, Christal D.; Cheng, Qian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1245-1251</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cytochrome P 450 enzymes are important in the metab. of steroids, vitamins, carcinogens, drugs, and other compds.  Two of the commonly used assays in this field are the measurements of total P 450 and NADPH-P 450 reductase in biol. prepns.  A detailed protocol is presented for the measurement of P 450 by its spectral properties, along with a protocol for measuring NADPH-P 450 reductase by its NADPH-cytochrome c redn. activity.  Each assay can be completed in 5-10 min.  Detailed explanations for the rationale of particular sequences in the protocols are provided, along with potential confounding problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbeZdtyf5GjrVg90H21EOLACvtfcHk0liXEggEiPpvVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVygsLnP&md5=f0d2167bf5427faee0fbe6f11c85e06d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2009.121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2009.121%26sid%3Dliteratum%253Aachs%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26aulast%3DMartin%26aufirst%3DM.%2BV.%26aulast%3DSohl%26aufirst%3DC.%2BD.%26aulast%3DCheng%26aufirst%3DQ.%26atitle%3DMeasurement%2520of%2520cytochrome%2520P450%2520and%2520NADPH-cytochrome%2520P450%2520reductase%26jtitle%3DNat.%2520Protoc.%26date%3D2009%26volume%3D4%26spage%3D1245%26epage%3D1251%26doi%3D10.1038%2Fnprot.2009.121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machalz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorensen, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bureik, M.</span></span> <span> </span><span class="NLM_article-title">Efficient substrate screening and inhibitor testing of human CYP4Z1 using permeabilized recombinant fission yeast</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2017.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.bcp.2017.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=28951277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOhsLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2017&pages=174-187&author=Q.+Yanauthor=D.+Machalzauthor=A.+Zollnerauthor=E.+J.+Sorensenauthor=G.+Wolberauthor=M.+Bureik&title=Efficient+substrate+screening+and+inhibitor+testing+of+human+CYP4Z1+using+permeabilized+recombinant+fission+yeast&doi=10.1016%2Fj.bcp.2017.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient substrate screening and inhibitor testing of human CYP4Z1 using permeabilized recombinant fission yeast</span></div><div class="casAuthors">Yan, Qi; Machalz, David; Zollner, Andy; Sorensen, Erik J.; Wolber, Gerhard; Bureik, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">174-187</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have established a protocol for the prepn. of permeabilized fission yeast cells (enzyme bags) that recombinantly express human cytochrome P 450 enzymes (CYPs).  A direct comparison of CYP3A4 activity gave an eightfold higher space-time yield for enzyme bag-catalyzed biotransformation as compared to whole-cell biotransformation, even though the total no. of cells employed was lower by a factor of 150.  Biotransformation of the luminogenic substrate Luciferin-H using CYP2C9-contg. enzyme bags proceeded efficiently and stably for 24 h.  CYP4Z1 is of interest because it is strongly overexpressed both in breast cancer cells and in breast cancer metastases; however, current knowledge about its catalytic properties is very limited.  Screening of CYP4Z1-contg. enzyme bags with 15 luminogenic substrates enabled us to identify two new hydroxylations and eleven ether cleavage reactions that are catalyzed by CYP4Z1.  By far the best substrate found in this study was Luciferin benzyl ether (Luciferin-BE).  On the basis of the recently published crystal structure of CYP4B1 we created a new homol. model of CYP4Z1 and performed mol. docking expts., which indicate that all active substrates show a highly similar binding geometry compared to the endogenous substrates.  The model predicts that Ser113, Ser222, Asn381, and Ser383 are key hydrogen bonding residues.  We also identified five new inhibitors of CYP4Z1: miconazole, econazole, aminobenzotriazole, tolazoline, and 1-benzylimidazole resp., with the last compd. being the most potent giving an IC50 value of 180 nM in our test system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLFJC9Ne1rhrVg90H21EOLACvtfcHk0lh4e8nISf12yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOhsLzN&md5=f79fc6e4ec403d9198a7f8f8938bf575</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DMachalz%26aufirst%3DD.%26aulast%3DZollner%26aufirst%3DA.%26aulast%3DSorensen%26aufirst%3DE.%2BJ.%26aulast%3DWolber%26aufirst%3DG.%26aulast%3DBureik%26aufirst%3DM.%26atitle%3DEfficient%2520substrate%2520screening%2520and%2520inhibitor%2520testing%2520of%2520human%2520CYP4Z1%2520using%2520permeabilized%2520recombinant%2520fission%2520yeast%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D146%26spage%3D174%26epage%3D187%26doi%3D10.1016%2Fj.bcp.2017.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit16"><span>Promega
P450-Glo. <a href="https://www.promega.com/resources/protocols/technical-bulletins/101/p450-glo-assays-protocol/" class="extLink">https://www.promega.com/resources/protocols/technical-bulletins/101/p450-glo-assays-protocol/</a> (accessed Oct, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Promega%0AP450-Glo.+https%3A%2F%2Fwww.promega.com%2Fresources%2Fprotocols%2Ftechnical-bulletins%2F101%2Fp450-glo-assays-protocol%2F+%28accessed+Oct%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parkinson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yerino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsapple, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otradovec, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, B. W.</span></span> <span> </span><span class="NLM_article-title">An evaluation of the dilution method for identifying metabolism-dependent inhibitors of cytochrome P450 enzymes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1370</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1124/dmd.111.038596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.111.038596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=21525169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1yqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1370-1387&author=A.+Parkinsonauthor=F.+Kazmiauthor=D.+B.+Buckleyauthor=P.+Yerinoauthor=B.+L.+Parisauthor=J.+Holsappleauthor=P.+Torenauthor=S.+M.+Otradovecauthor=B.+W.+Ogilvie&title=An+evaluation+of+the+dilution+method+for+identifying+metabolism-dependent+inhibitors+of+cytochrome+P450+enzymes&doi=10.1124%2Fdmd.111.038596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An evaluation of the dilution method for identifying metabolism-dependent inhibitors of cytochrome P450 enzymes</span></div><div class="casAuthors">Parkinson, Andrew; Kazmi, Faraz; Buckley, David B.; Yerino, Phyllis; Paris, Brandy L.; Holsapple, Jeff; Toren, Paul; Otradovec, Steve M.; Ogilvie, Brian W.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1370-1387</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Metab.-dependent inhibition (MDI) of cytochrome P 450 is usually assessed in vitro by examg. whether the inhibitory potency of a drug candidate increases after a 30-min incubation with human liver microsomes (HLMs).  To augment the IC50 shift, many researchers incorporate a diln. step whereby the samples, after being preincubated for 30 min with a high concn. of HLMs (with and without NADPH), are dild. before measuring P 450 activity.  In the present study, we show that the greater IC50 shift assocd. with the diln. method is a consequence of data processing.  With the diln. method, IC50 values for direct-acting inhibitors vary with the diln. factor unless they are based on the final (postdilution) inhibitor concn., whereas the IC50 values for MDIs vary with the diln. factor unless they are based on the initial (predilution) concn.  When the latter data are processed on the final inhibitor concn., as is commonly done, the IC50 values for MDI (shifted IC50 values) decrease by the magnitude of the diln. factor.  The lower shifted IC50 values are a consequence of data processing, not enhanced P 450 inactivation.  In fact, for many MDIs, increasing the concn. of HLMs actually leads to considerably less P 450 inactivation because of inhibitor depletion and/or binding of the inhibitor to microsomes.  A true increase in P 450 inactivation and IC50 shift can be achieved by assessing MDI by a nondilution method and by decreasing the concn. of HLMs.  These results have consequences for the conduct of MDI studies and the development of cut-off criteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpORjFNdXEZzrVg90H21EOLACvtfcHk0lh4e8nISf12yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1yqt7g%253D&md5=f6555fc640d5b0e05b98ec36795a1984</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.038596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.038596%26sid%3Dliteratum%253Aachs%26aulast%3DParkinson%26aufirst%3DA.%26aulast%3DKazmi%26aufirst%3DF.%26aulast%3DBuckley%26aufirst%3DD.%2BB.%26aulast%3DYerino%26aufirst%3DP.%26aulast%3DParis%26aufirst%3DB.%2BL.%26aulast%3DHolsapple%26aufirst%3DJ.%26aulast%3DToren%26aufirst%3DP.%26aulast%3DOtradovec%26aufirst%3DS.%2BM.%26aulast%3DOgilvie%26aufirst%3DB.%2BW.%26atitle%3DAn%2520evaluation%2520of%2520the%2520dilution%2520method%2520for%2520identifying%2520metabolism-dependent%2520inhibitors%2520of%2520cytochrome%2520P450%2520enzymes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26spage%3D1370%26epage%3D1387%26doi%3D10.1124%2Fdmd.111.038596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henne, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VandenBrink, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aidasani, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stresser, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teffera, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonn, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B. K.</span></span> <span> </span><span class="NLM_article-title">Sequential metabolism of AMG 487, a novel CXCR3 antagonist, results in formation of quinone reactive metabolites that covalently modify CYP3A4 Cys239 and cause time-dependent inhibition of the enzyme</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1429</span>– <span class="NLM_lpage">1440</span>, <span class="refDoi"> DOI: 10.1124/dmd.112.045708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.112.045708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=22517972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOrs7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1429-1440&author=K.+R.+Henneauthor=T.+B.+Tranauthor=B.+M.+VandenBrinkauthor=D.+A.+Rockauthor=D.+K.+Aidasaniauthor=R.+Subramanianauthor=A.+K.+Masonauthor=D.+M.+Stresserauthor=Y.+Tefferaauthor=S.+G.+Wongauthor=M.+G.+Johnsonauthor=X.+Chenauthor=G.+R.+Tonnauthor=B.+K.+Wong&title=Sequential+metabolism+of+AMG+487%2C+a+novel+CXCR3+antagonist%2C+results+in+formation+of+quinone+reactive+metabolites+that+covalently+modify+CYP3A4+Cys239+and+cause+time-dependent+inhibition+of+the+enzyme&doi=10.1124%2Fdmd.112.045708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Sequential metabolism of AMG 487, a novel CXCR3 antagonist, results in formation of quinone reactive metabolites that covalently modify CYP3A4 Cys239 and cause time-dependent inhibition of the enzyme</span></div><div class="casAuthors">Henne, Kirk R.; Tran, Thuy B.; VandenBrink, Brooke M.; Rock, Dan A.; Aidasani, Divesh K.; Subramanian, Raju; Mason, Andrew K.; Stresser, David M.; Teffera, Yohannes; Wong, Simon G.; Johnson, Michael G.; Chen, Xiaoqi; Tonn, George R.; Wong, Bradley K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1429-1440</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">CYP3A4-mediated biotransformation of (R)-N-(1-(3-(4-ethoxyphenyl)-4-oxo-3,4-dihydropyrido[2,3-d]pyrimidin-2-yl)ethyl)-N-(pyridin-3-ylmethyl)-2-(4-(trifluoromethoxy)phenyl)acetamide (AMG 487) was previously shown to generate an inhibitory metabolite linked to dose- and time-dependent pharmacokinetics in humans.  Although in vitro activity loss assays failed to demonstrate CYP3A4 time-dependent inhibition (TDI) with AMG 487, its M2 phenol metabolite readily produced TDI when remaining activity was assessed using either midazolam or testosterone (KI = 0.73-0.74 μM, kinact = 0.088-0.099 min-1).  TDI investigations using an IC50 shift method successfully produced inhibition attributable to AMG 487, but only when preincubations were extended from 30 to 90 min.  The shift magnitude was ∼3× for midazolam activity, but no shift was obsd. for testosterone activity.  Subsequent partition ratio detns. conducted for M2 using recombinant CYP3A4 showed that inactivation was a relatively inefficient process (r = 36).  CYP3A4-mediated biotransformation of [3H]M2 in the presence of GSH led to identification of two new metabolites, M4 and M5, which shifted focus away from M2 being directly responsible for TDI.  M4 (hydroxylated M2) was further metabolized to form reactive intermediates that, upon reaction with GSH, produced isomeric adducts, collectively designated M5.  Incubations conducted in the presence of [18O]H2O confirmed incorporation of oxygen from O2 for the majority of M4 and M5 formed (>75%).  Further evidence of a primary role for M4 in CYP3A4 TDI was generated by protein labeling and proteolysis expts., in which M4 was found to be covalently bound to Cys239 of CYP3A4.  These investigations confirmed a primarily role for M4 in CYP3A4 inactivation, suggesting that a more complex metabolic pathway was responsible for generation of inhibitory metabolites affecting AMG 487 human pharmacokinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoskMDPPnKfuLVg90H21EOLACvtfcHk0li_eUX1po3jBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOrs7vO&md5=f695c134a3f06cfdf8cc1eb834ef8ccb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.045708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.045708%26sid%3Dliteratum%253Aachs%26aulast%3DHenne%26aufirst%3DK.%2BR.%26aulast%3DTran%26aufirst%3DT.%2BB.%26aulast%3DVandenBrink%26aufirst%3DB.%2BM.%26aulast%3DRock%26aufirst%3DD.%2BA.%26aulast%3DAidasani%26aufirst%3DD.%2BK.%26aulast%3DSubramanian%26aufirst%3DR.%26aulast%3DMason%26aufirst%3DA.%2BK.%26aulast%3DStresser%26aufirst%3DD.%2BM.%26aulast%3DTeffera%26aufirst%3DY.%26aulast%3DWong%26aufirst%3DS.%2BG.%26aulast%3DJohnson%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DTonn%26aufirst%3DG.%2BR.%26aulast%3DWong%26aufirst%3DB.%2BK.%26atitle%3DSequential%2520metabolism%2520of%2520AMG%2520487%252C%2520a%2520novel%2520CXCR3%2520antagonist%252C%2520results%2520in%2520formation%2520of%2520quinone%2520reactive%2520metabolites%2520that%2520covalently%2520modify%2520CYP3A4%2520Cys239%2520and%2520cause%2520time-dependent%2520inhibition%2520of%2520the%2520enzyme%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26spage%3D1429%26epage%3D1440%26doi%3D10.1124%2Fdmd.112.045708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oshikata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawase, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haramaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, H.</span></span> <span> </span><span class="NLM_article-title">Cocktail-substrate approach-based high-throughput assay for evaluation of direct and time-dependent inhibition of multiple cytochrome P450 isoforms</span>. <i>Drug Metab. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">198</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.2133/dmpk.DMPK-13-RG-093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.2133%2Fdmpk.DMPK-13-RG-093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=24172718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKjurrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=198-207&author=K.+Kozakaiauthor=Y.+Yamadaauthor=M.+Oshikataauthor=T.+Kawaseauthor=E.+Suzukiauthor=Y.+Haramakiauthor=H.+Taniguchi&title=Cocktail-substrate+approach-based+high-throughput+assay+for+evaluation+of+direct+and+time-dependent+inhibition+of+multiple+cytochrome+P450+isoforms&doi=10.2133%2Fdmpk.DMPK-13-RG-093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Cocktail-substrate approach-based high-throughput assay for evaluation of direct and time-dependent inhibition of multiple cytochrome P450 isoforms</span></div><div class="casAuthors">Kozakai, Kazumasa; Yamada, Yasuhiro; Oshikata, Motoji; Kawase, Taiji; Suzuki, Etsuko; Haramaki, Yukari; Taniguchi, Hideki</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Pharmacokinetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">198-207</span>CODEN:
                <span class="NLM_cas:coden">DMPRB8</span>;
        ISSN:<span class="NLM_cas:issn">1347-4367</span>.
    
            (<span class="NLM_cas:orgname">Japanese Society for the Study of Xenobiotics</span>)
        </div><div class="casAbstract">Avoiding drug-drug interactions (DDIs) mediated through inhibition of cytochrome P 450 (CYP) activity is highly desirable.  Direct inhibition (DI) of CYP through new chem. entities (NCEs) or time-dependent inhibition (TDI) through reactive metabolites should be elucidated at an early stage of drug discovery research.  In particular, TDI of CYP occurring through reactive metabolites may be irreversible and even sustained, causing far more serious DDIs for TDIs than for DIs.  Furthermore, it is important to ascertain whether an NCE inhibits multiple CYP isoforms.  Hence, using a cocktail-substrate approach that we previously established (in which the activity of 8 CYP isoforms is simultaneously evaluated in a single run), we evaluated the IC50 values of direct inhibitors and TDI parameters (kobs, shifted IC50, KI and kinact) of time-dependent inhibitors that affect multiple CYP isoforms.  The IC50 values for 8 CYP isoforms obtained using the cocktail-substrate approach were nearly identical to values previously reported.  The TDI parameters for CYP1A2, 2C9, 2C19, 2D6, and CYP3A4/5 obtained using the cocktail-substrate approach were also nearly identical to those obtained using a single-substrate approach.  Thus, the cocktail-substrate approach is useful for evaluating DI and TDI in the early stages of drug discovery and development processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBhvsFfgXk2rVg90H21EOLACvtfcHk0li_eUX1po3jBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKjurrE&md5=47826700177358db6cfecee11cebc4b6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2133%2Fdmpk.DMPK-13-RG-093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2133%252Fdmpk.DMPK-13-RG-093%26sid%3Dliteratum%253Aachs%26aulast%3DKozakai%26aufirst%3DK.%26aulast%3DYamada%26aufirst%3DY.%26aulast%3DOshikata%26aufirst%3DM.%26aulast%3DKawase%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DE.%26aulast%3DHaramaki%26aufirst%3DY.%26aulast%3DTaniguchi%26aufirst%3DH.%26atitle%3DCocktail-substrate%2520approach-based%2520high-throughput%2520assay%2520for%2520evaluation%2520of%2520direct%2520and%2520time-dependent%2520inhibition%2520of%2520multiple%2520cytochrome%2520P450%2520isoforms%26jtitle%3DDrug%2520Metab.%2520Pharmacokinet.%26date%3D2014%26volume%3D29%26spage%3D198%26epage%3D207%26doi%3D10.2133%2Fdmpk.DMPK-13-RG-093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimm, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einolf, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomeir, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibert, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skordos, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonn, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Horn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">The conduct of in vitro studies to address time-dependent inhibition of drug-metabolizing enzymes: a perspective of the pharmaceutical research and manufacturers of America</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1355</span>– <span class="NLM_lpage">1370</span>, <span class="refDoi"> DOI: 10.1124/dmd.109.026716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.109.026716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=19359406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFenu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=1355-1370&author=S.+W.+Grimmauthor=H.+J.+Einolfauthor=S.+D.+Hallauthor=K.+Heauthor=H.+K.+Limauthor=K.+H.+Lingauthor=C.+Luauthor=A.+A.+Nomeirauthor=E.+Seibertauthor=K.+W.+Skordosauthor=G.+R.+Tonnauthor=R.+Van+Hornauthor=R.+W.+Wangauthor=Y.+N.+Wongauthor=T.+J.+Yangauthor=R.+S.+Obach&title=The+conduct+of+in+vitro+studies+to+address+time-dependent+inhibition+of+drug-metabolizing+enzymes%3A+a+perspective+of+the+pharmaceutical+research+and+manufacturers+of+America&doi=10.1124%2Fdmd.109.026716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The conduct of in vitro studies to address time-dependent inhibition of drug-metabolizing enzymes: a perspective of the Pharmaceutical Research and Manufacturers of America</span></div><div class="casAuthors">Grimm, Scott W.; Einolf, Heidi J.; Hall, Steven D.; He, Kan; Lim, Heng-Keang; Ling, Kah-Hiing John; Lu, Chuang; Nomeir, Amin A.; Seibert, Eleanore; Skordos, Konstantine W.; Tonn, George R.; Van Horn, Robert; Wang, Regina W.; Wong, Y. Nancy; Yang, Tian J.; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1355-1370</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Time-dependent inhibition (TDI) of cytochrome P 450 enzymes caused by new mol. entities (NMEs) is of concern because such compds. can be responsible for clin. relevant drug-drug interactions (DDI).  Although the biochem. underlying mechanism-based inactivation (MBI) of P 450 enzymes has been generally understood for several years, significant advances have been made only in the past few years regarding how in vitro time-dependent inhibition data can be used to understand and predict clin. DDI.  In this article, a team of scientists from 16 pharmaceutical research organizations that are member companies of the Pharmaceutical Research and Manufacturers of America offer a discussion of the phenomenon of TDI with emphasis on the lab. methods used in its measurement.  Results of an anonymous survey regarding pharmaceutical industry practices and strategies around TDI are reported.  Specific topics that still possess a high degree of uncertainty are raised, such as parameter ests. needed to make predictions of DDI magnitude from in vitro inactivation parameters.  A description of follow-up mechanistic expts. that can be done to characterize TDI are described.  A consensus recommendation regarding common practices to address TDI is included, the salient points of which include the use of a tiered approach wherein abbreviated assays are first used to det. whether NMEs demonstrate TDI or not, followed by more thorough inactivation studies for those that do to define the parameters needed for prediction of DDI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc57kHA3HAP7Vg90H21EOLACvtfcHk0li_eUX1po3jBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFenu7o%253D&md5=7f74567b5c9298806f8a5cc58355b867</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fdmd.109.026716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.109.026716%26sid%3Dliteratum%253Aachs%26aulast%3DGrimm%26aufirst%3DS.%2BW.%26aulast%3DEinolf%26aufirst%3DH.%2BJ.%26aulast%3DHall%26aufirst%3DS.%2BD.%26aulast%3DHe%26aufirst%3DK.%26aulast%3DLim%26aufirst%3DH.%2BK.%26aulast%3DLing%26aufirst%3DK.%2BH.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DNomeir%26aufirst%3DA.%2BA.%26aulast%3DSeibert%26aufirst%3DE.%26aulast%3DSkordos%26aufirst%3DK.%2BW.%26aulast%3DTonn%26aufirst%3DG.%2BR.%26aulast%3DVan%2BHorn%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DR.%2BW.%26aulast%3DWong%26aufirst%3DY.%2BN.%26aulast%3DYang%26aufirst%3DT.%2BJ.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DThe%2520conduct%2520of%2520in%2520vitro%2520studies%2520to%2520address%2520time-dependent%2520inhibition%2520of%2520drug-metabolizing%2520enzymes%253A%2520a%2520perspective%2520of%2520the%2520pharmaceutical%2520research%2520and%2520manufacturers%2520of%2520America%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2009%26volume%3D37%26spage%3D1355%26epage%3D1370%26doi%3D10.1124%2Fdmd.109.026716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollenberg, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bumpus, N. N.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based inactivation of human cytochromes p450s: experimental characterization, reactive intermediates, and clinical implications</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.1021/tx7002504</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx7002504" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=189-205&author=P.+F.+Hollenbergauthor=U.+M.+Kentauthor=N.+N.+Bumpus&title=Mechanism-based+inactivation+of+human+cytochromes+p450s%3A+experimental+characterization%2C+reactive+intermediates%2C+and+clinical+implications&doi=10.1021%2Ftx7002504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-Based Inactivation of Human Cytochromes P450s: Experimental Characterization, Reactive Intermediates, and Clinical Implications</span></div><div class="casAuthors">Hollenberg, Paul F.; Kent, Ute M.; Bumpus, Namandje N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">189-205</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The P 450 type cytochromes are responsible for the metab. of a wide variety of xenobiotics and endogenous compds.  Although P 450-catalyzed reactions are generally thought to lead to detoxication of xenobiotics, the reactions can also produce reactive intermediates that can react with cellular macromols. leading to toxicity or that can react with the P450s that form them leading to irreversible (i.e., mechanism-based) inactivation.  This perspective describes the fundamentals of mechanism-based inactivation as it pertains to P 450 enzymes.  The exptl. approaches used to characterize mechanism-based inactivators are discussed, and the criteria required for a compd. to be classified as a mechanism-based inactivator are outlined.  The kinetic scheme for mechanism-based inactivation and the calcn. of the relevant kinetic consts. that describe a particular inactivation event are presented.  The structural aspects and important functional groups of several classes of mols. that have been found to impart mechanism-based inactivation upon metab. by P450s such as acetylenes, thiol-contg. compds. that include isothiocyanates, thiazolidinediones, and thiophenes, arylamines, quinones, furanocoumarins, and cyclic tertiary amines are described.  Emphasis throughout this perspective is placed on more recent findings with human P450s where the site of modification, whether it be the apoprotein or the heme moiety, and, at least in part, the identity of the reactive intermediate responsible for the loss in P 450 activity are known or inferred.  Recent advances in trapping procedures as well as new methods for identification of reactive intermediates are presented.  A variety of clin. important drugs that act as mechanism-based inactivators of P450s are discussed.  The irreversible inactivation of human P450s by these drugs has the potential for causing serious drug-drug interactions that may have severe toxicol. effects.  The clin. significance of inactivating human P450s for improving drug efficacy as well as drug safety is discussed along with the potential for exploiting mechanism-based inactivators of P450s for therapeutic benefits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC4MVcZv4wm7Vg90H21EOLACvtfcHk0lgH5cczOm-bLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrvN&md5=0cb120c5da218dcc1e192a117bc2eac7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Ftx7002504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx7002504%26sid%3Dliteratum%253Aachs%26aulast%3DHollenberg%26aufirst%3DP.%2BF.%26aulast%3DKent%26aufirst%3DU.%2BM.%26aulast%3DBumpus%26aufirst%3DN.%2BN.%26atitle%3DMechanism-based%2520inactivation%2520of%2520human%2520cytochromes%2520p450s%253A%2520experimental%2520characterization%252C%2520reactive%2520intermediates%252C%2520and%2520clinical%2520implications%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D189%26epage%3D205%26doi%3D10.1021%2Ftx7002504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emoto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murase, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, K.</span></span> <span> </span><span class="NLM_article-title">In vitro inhibitory effect of 1-aminobenzotriazole on drug oxidations catalyzed by human cytochrome P450 enzymes: a comparison with SKF-525A and ketoconazole</span>. <i>Drug Metab. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.2133/dmpk.18.287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.2133%2Fdmpk.18.287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15618748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnt1yltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2003&pages=287-295&author=C.+Emotoauthor=S.+Muraseauthor=Y.+Sawadaauthor=B.+C.+Jonesauthor=K.+Iwasaki&title=In+vitro+inhibitory+effect+of+1-aminobenzotriazole+on+drug+oxidations+catalyzed+by+human+cytochrome+P450+enzymes%3A+a+comparison+with+SKF-525A+and+ketoconazole&doi=10.2133%2Fdmpk.18.287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro inhibitory effect of 1-aminobenzotriazole on drug oxidations catalyzed by human cytochrome P450 enzymes: A comparison with SKF-525A and ketoconazole</span></div><div class="casAuthors">Emoto, Chie; Murase, Shigeo; Sawada, Yasufusa; Jones, Barry C.; Iwasaki, Kazuhide</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Pharmacokinetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">287-295</span>CODEN:
                <span class="NLM_cas:coden">DMPRB8</span>;
        ISSN:<span class="NLM_cas:issn">1347-4367</span>.
    
            (<span class="NLM_cas:orgname">Japanese Society for the Study of Xenobiotics</span>)
        </div><div class="casAbstract">1-Aminobenzotriazole (ABT) is widely used as a non-specific inhibitor of animal cytochrome P 450 (CYP).  In the present study, the inhibitory effect of ABT was investigated on drug oxidns. catalyzed by human CYP isoforms.  This inhibitory effect was compared with that of SKF-525A, another non-specific inhibitor, and ketoconazole, a potent inhibitor of CYP3A.  Bacurovirus-expressed recombinant human CYP isoforms were used as an enzyme source.  The specific activities for human CYP isoforms are: phenacetin O-deethylation, for CYP1A2; diclofenac 4'-hydroxylation, for CYP2C9; S-mephenytoin 4'-hydroxylation, for CYP2C19; bufuralol 1'-hydroxylation, for CYP2D6; chlorzoxazone 6-hydroxylation, for CYP2E1; testosterone 6β-hydroxylation, nifedipine oxidn., and midazolam 1'-hydroxylation, for CYP3A4.  ABT inhibited both CYP1A2-dependent activity (Ki=330 μM) and CYP2E1-dependent activity (Ki=8.7 μM).  In contrast, SKF-525A weakly inhibited CYP1A2-dependent activities (46% inhibition at 1200 μM) and CYP2E1-dependent activities (65% inhibition at 1000 μM).  ABT exhibited the highest Ki value for CYP2C9-dependent diclofenac 4'-hydroxylation among those detd. by this assay (Ki=3500 μM).  Moreover, SKF-525A showed strong inhibition of CYP2D6-dependent bufuralol 1'-hydroxylation (Ki=0.043 μM).  Ketoconazole inhibited all tested drug oxidns., however, its inhibitory effect on CYP1A2-dependent activities was very weak (50% inhibition at 120 μM).  ABT, SKF-525A, and ketoconazole showed different selectivity and had a wide range of Ki values for the drug oxidns. catalyzed by human CYP enzymes.  Therefore, we conclude that inhibitory studies designed to predict the contribution of CYP enzymes to the metab. of certain compds. should be performed using multiple CYP inhibitors, such as ABT, SKF-525A, and ketoconazole.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWLwmZAR2feLVg90H21EOLACvtfcHk0lgH5cczOm-bLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnt1yltw%253D%253D&md5=d32fc238d4bc63a2adf3dde029fab57a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.2133%2Fdmpk.18.287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2133%252Fdmpk.18.287%26sid%3Dliteratum%253Aachs%26aulast%3DEmoto%26aufirst%3DC.%26aulast%3DMurase%26aufirst%3DS.%26aulast%3DSawada%26aufirst%3DY.%26aulast%3DJones%26aufirst%3DB.%2BC.%26aulast%3DIwasaki%26aufirst%3DK.%26atitle%3DIn%2520vitro%2520inhibitory%2520effect%2520of%25201-aminobenzotriazole%2520on%2520drug%2520oxidations%2520catalyzed%2520by%2520human%2520cytochrome%2520P450%2520enzymes%253A%2520a%2520comparison%2520with%2520SKF-525A%2520and%2520ketoconazole%26jtitle%3DDrug%2520Metab.%2520Pharmacokinet.%26date%3D2003%26volume%3D18%26spage%3D287%26epage%3D295%26doi%3D10.2133%2Fdmpk.18.287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz
de Montellano, P. R.</span></span> <span> </span><span class="NLM_article-title">1-Aminobenzotriazole: A Mechanism-Based Cytochrome P450 Inhibitor and Probe of Cytochrome P450 Biology</span>. <i>Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">038</span> <span class="refDoi"> DOI: 10.4172/2161-0444.1000495</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.4172%2F2161-0444.1000495" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=P.+R.+Ortiz%0Ade+Montellano&title=1-Aminobenzotriazole%3A+A+Mechanism-Based+Cytochrome+P450+Inhibitor+and+Probe+of+Cytochrome+P450+Biology&doi=10.4172%2F2161-0444.1000495"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.4172%2F2161-0444.1000495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F2161-0444.1000495%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2Bde%2BMontellano%26aufirst%3DP.%2BR.%26atitle%3D1-Aminobenzotriazole%253A%2520A%2520Mechanism-Based%2520Cytochrome%2520P450%2520Inhibitor%2520and%2520Probe%2520of%2520Cytochrome%2520P450%2520Biology%26jtitle%3DMed.%2520Chem.%26date%3D2018%26volume%3D8%26doi%3D10.4172%2F2161-0444.1000495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bend, J. R.</span></span> <span> </span><span class="NLM_article-title">N-alkylaminobenzotriazoles as isozyme-selective suicide inhibitors of rabbit pulmonary microsomal cytochrome P-450</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=3724742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaL28XlvVKnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1986&pages=25-32&author=J.+M.+Mathewsauthor=J.+R.+Bend&title=N-alkylaminobenzotriazoles+as+isozyme-selective+suicide+inhibitors+of+rabbit+pulmonary+microsomal+cytochrome+P-450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">N-alkylaminobenzotriazoles as isozyme-selective suicide inhibitors of rabbit pulmonary microsomal cytochrome P-450</span></div><div class="casAuthors">Mathews, James M.; Bend, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-32</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">A series of N-alkylated 1-aminobenzotriazole (I) derivs. were synthesized as isoform-selective suicide inhibitors of cytochrome P 450 (P 450).  These included N-methyl-I, N-butyl-I (II), N-benzyl-I (III), and N-α-methylbenzyl-I (IV).  The suicide inhibitors showing the greatest potency and isoform selectivity were III and IV, compds. which included mol. features for P 450 inactivation (the I moiety) and similarity to benzphetamine.  I and its N-alkylated derivs. were tested as suicide inhibitors in rabbit lung microsomes, whose P 450 monooxygenase system has been well characterized in both untreated and β-naphthoflavone- or 2,3,7,8-tetrachlorodibenzo-p-dioxin-treated animals.  I (10 mM) destroyed up to 99% of the total P 450 content of lung microsomes of untreated rabbits.  At equimolar concns. (10 μM), I was less effective than the N-alkylated compds. for the inhibition of P 450 isoform 2-catalyzed benzphetamine N-demethylase (V); in fact, II, III, and IV completely inhibited V activity at this concn. and destroyed <40% of the total pulmonary P 450.  However, these compds. also inactivated 69-85% of isoform 6-catalyzed 7-ethoxyresorufin O-deethylase.  The most potent and isoform-selective suicide inhibitor prepd. was IV; at 1 μM concn., this compd. inhibited ∼80% of isoform 2-catalyzed and ∼20% of isoform 6-catalyzed monooxygenase activity, but spared P 450 isoform 6; at 2.5 μM concn., it caused a near-complete loss (96%) of V activity.  The partition ratio of IV, i.e., the molar ratio of inhibitor present to that of the P 450 destroyed, was 11, further demonstrating the potency of this compd.  Expts. with III- and phenobarbital-treated rats showed that the mechanism for suicidal inactivation of P 450 by this N-alkylated compd. was by benzyne release, the same mechanism demonstrated earlier for the parent compd., I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0KwdY-FVixrVg90H21EOLACvtfcHk0ljybz4QY7QOpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XlvVKnu7o%253D&md5=965e34522e8ad29261b48002f60d465b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMathews%26aufirst%3DJ.%2BM.%26aulast%3DBend%26aufirst%3DJ.%2BR.%26atitle%3DN-alkylaminobenzotriazoles%2520as%2520isozyme-selective%2520suicide%2520inhibitors%2520of%2520rabbit%2520pulmonary%2520microsomal%2520cytochrome%2520P-450%26jtitle%3DMol.%2520Pharmacol.%26date%3D1986%26volume%3D30%26spage%3D25%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bend, J. R.</span></span> <span> </span><span class="NLM_article-title">N-aralkyl derivatives of 1-aminobenzotriazole as potent isozyme-selective mechanism-based inhibitors of rabbit pulmonary cytochrome P450 in vivo</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>265</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">285</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=8474011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaK3sXktVShtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1993&pages=281-285&author=J.+M.+Mathewsauthor=J.+R.+Bend&title=N-aralkyl+derivatives+of+1-aminobenzotriazole+as+potent+isozyme-selective+mechanism-based+inhibitors+of+rabbit+pulmonary+cytochrome+P450+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">N-Aralkyl derivatives of 1-aminobenzotriazole as potent isozyme-selective mechanism-based inhibitors of rabbit pulmonary cytochrome P450 in vivo</span></div><div class="casAuthors">Mathews, James M.; Bend, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">281-5</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    </div><div class="casAbstract">Two N-aralkylated (N-benzyl- and N-α-methylbenzyl-) derivs. of 1-aminobenzotriazole, a mechanism-based inhibitor of cytochrome P 450 with low isoenzyme selectivity, were previously shown to be potent and isoenzyme-selective suicide substrates for rabbit and guinea pig pulmonary P 450 in vitro (Mathews and Bend, 1986; Woodcroft et al., 1990).  These three compds. were compared as inhibitor in vivo after i.v. administration to rabbits treated with the cytochrome P 450 inducers β-naphthoflavone or phenobarbital.  By 1 h after administration of N-α-methylbenzyl-1-aminobenzotriazole (1 μmol/kg), 80% of P 450 2Bs-catalyzed benzphetamine N-demethylation in lung of β-naphthoflavone-treated rabbits was inactivated and about 35% of P 450 was lost without inhibition of P 450 1A1-catalyzed activity; at a dose of 10 or 100 μmol/kg, this compd. totally inactivated pulmonary P 450 2Bs activity while exerting minimal effects on benzphetamine N-demethylation activity (<20% inhibition) in liver of β-naphthoflavone-treated rabbits.  N-Benzyl-1-aminobenzotriazole was also an isoenzyme- and tissue-selective inhibitor of pulmonary P 450 2Bs in vivo.  Relatively high doses (100 μmol/kg) of these compds. were compared in phenobarbital-induced rabbits.  Virtually all (≥95%) of pulmonary P 450 2Bs-dependent activity was inhibited by the two N-aralkylated compds. (vs. 50% for 1-aminobenzotriazole).  At this dose, about 25% of hepatic P 450 was destroyed by all three compds., whereas 1-aminobenzotriazole and its N-benzyl and N-α-methylbenzyl derivs. inactivated 20, 50 and 85% of hepatic P 450 2Bs-selective benzphetamine N-demethylation activity, resp.  These results demonstrate that N-α-methylbenzyl-1-aminobenzotriazole and N-benzyl-1-aminobenzotriazole, but not 1-aminobenzotriazole, are highly selective for the inactivation of pulmonary P 450 2Bs in rabbits in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3KXaKWiR_hbVg90H21EOLACvtfcHk0ljybz4QY7QOpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktVShtrc%253D&md5=18c12d1fe7fbc759ab10f96454eb72df</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMathews%26aufirst%3DJ.%2BM.%26aulast%3DBend%26aufirst%3DJ.%2BR.%26atitle%3DN-aralkyl%2520derivatives%2520of%25201-aminobenzotriazole%2520as%2520potent%2520isozyme-selective%2520mechanism-based%2520inhibitors%2520of%2520rabbit%2520pulmonary%2520cytochrome%2520P450%2520in%2520vivo%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1993%26volume%3D265%26spage%3D281%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woodcroft, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczepan, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knickle, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bend, J. R.</span></span> <span> </span><span class="NLM_article-title">Three N-aralkylated derivatives of 1-aminobenzotriazole as potent and isozyme-selective, mechanism-based inhibitors of guinea pig pulmonary cytochrome P-450 in vitro</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1037</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1981509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaK3MXpsVGksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1990&pages=1031-1037&author=K.+J.+Woodcroftauthor=E.+W.+Szczepanauthor=L.+C.+Knickleauthor=J.+R.+Bend&title=Three+N-aralkylated+derivatives+of+1-aminobenzotriazole+as+potent+and+isozyme-selective%2C+mechanism-based+inhibitors+of+guinea+pig+pulmonary+cytochrome+P-450+in+vitro"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Three N-aralkylated derivatives of 1-aminobenzotriazole as potent and isozyme-selective, mechanism-based inhibitors of guinea pig pulmonary cytochrome P-450 in vitro</span></div><div class="casAuthors">Woodcroft, Kimberley J.; Szczepan, Edward W.; Knickle, Leah C.; Bend, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1031-7</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    </div><div class="casAbstract">The potency and cytochrome P 450 (P 450) isoenzyme selectivity of 1-aminobenzotriazole (ABT) and three of its N-aralkylated analog N-benzyl-1-aminobenzotriazole (BBT), N-α-methylbenzyl-1-aminobenzotriazole (αMB), and the newly synthesized N-α-ethylbenzyl-1-aminobenzotriazole (αEB), as mechanism-based inhibitors were compared in pulmonary microsomes of untreated and β-naphthoflavone (β-NF)-induced guinea pigs.  All four compds. were suicide substrates for pulmonary P 450, resulting in the loss of spectrally assayed hemoprotein (up to 50%).  Monooxygenase activities were measured with isoenzyme-selective/specific substrates; the O-dealkylation of 7-pentoxyresorufin (PRF) for the guinea pig ortholog of rabbit P-450IIB4, the O-deethylation of 7-ethoxyresorufin for P-450Ia1, and the N-hydroxylation of the arom. amine 4-aminobiphenyl for P-450IVB1.  BBT, αMB, and αEB were selective for the suicidal inhibition of P-450IIB4; for example, 1 μM αMB inactivated 95% of P-450IIB4-, and approx. 10% of P-450IA1- and IVB1-catalyzed, activity in microsomes from β-NF-induced lungs.  Isoenzyme selectivity was approx. the same for αEB and slightly lower for BBT, which inactivated relatively more P-450IA1.  At low concns., 1 and 10 μM, resp., ABT preferentially inactivated P.  450IVB1, consistent with the efficient N-hydroxylation of arom. amines by this form of P 450.  αEB also was shown to efficiently inactivate P-450IIB4-catalyzed PRF activity in microsomes prepd. from liver of phenobarbital-induced guinea pigs.  Collectively, the data indicated that N-aralkylated derivs. of ABT are isoenzyme-selective, mechanism-based inhibitors in guinea pig lung, and that the order of potency is αMB = αEB > BBT > ABT for P-450IIB4.  This study also emphasized that the pulmonary P 450 monooxygenase systems of the guinea pig and rabbit are functionally very similar.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-puiCfrzFQLVg90H21EOLACvtfcHk0ljybz4QY7QOpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXpsVGksA%253D%253D&md5=4dab44e28c4affff7104e1dab3a4b47c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWoodcroft%26aufirst%3DK.%2BJ.%26aulast%3DSzczepan%26aufirst%3DE.%2BW.%26aulast%3DKnickle%26aufirst%3DL.%2BC.%26aulast%3DBend%26aufirst%3DJ.%2BR.%26atitle%3DThree%2520N-aralkylated%2520derivatives%2520of%25201-aminobenzotriazole%2520as%2520potent%2520and%2520isozyme-selective%252C%2520mechanism-based%2520inhibitors%2520of%2520guinea%2520pig%2520pulmonary%2520cytochrome%2520P-450%2520in%2520vitro%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1990%26volume%3D18%26spage%3D1031%26epage%3D1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kent, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bend, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gage, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenberg, P. F.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based inactivation of cytochrome P450 2B1 by N-benzyl-1-aminobenzotriazole</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">600</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/tx960184o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx960184o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaK2sXjsVOjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1997&pages=600-608&author=U.+M.+Kentauthor=J.+R.+Bendauthor=B.+A.+Chamberlinauthor=D.+A.+Gageauthor=P.+F.+Hollenberg&title=Mechanism-based+inactivation+of+cytochrome+P450+2B1+by+N-benzyl-1-aminobenzotriazole&doi=10.1021%2Ftx960184o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based inactivation of cytochrome P450 2B1 by N-benzyl-1-aminobenzotriazole</span></div><div class="casAuthors">Kent, Ute M.; Bend, John R.; Chamberlin, Beverly A.; Gage, Douglas A.; Hollenberg, Paul F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">600-608</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The kinetics of inactivation of cytochrome P 450 2B1, the major phenobarbital inducible rat hepatic P 450, by N-benzyl-1-aminobenzotriazole (BBT) were characterized.  Purified, reconstituted P 450 2B1 7-ethoxy-4-(trifluoromethyl)coumarin (7-EFC) O-deethylase activity was inhibited by BBT in a mechanism-based manner.  The loss of O-deethylase activity followed pseudo-first-order kinetics and was NADPH and BBT dependent.  After a 5 min incubation, greater than 90% of the 2B1 activity was lost, whereas more than 70% of the ability of the reduced enzyme to bind CO was maintained.  Inclusion of 10 mM glutathione in the inactivation reaction lowered the rate of inactivation (kinactivation) and increased the partition ratio without significantly affecting the inactivator concn. required for half-maximal inactivation (KI).  The maximal rate const. for inactivation at 23° was 0.24 min-1 without and 0.15 min-1 with glutathione.  The apparent KI was 2 μM in both cases.  The extrapolated partition ratios were 4 and 9 without and with 10 mM glutathione, resp.  Consistent with mechanism-based inactivation, the loss of 7-EFC O-deethylase activity was irreversible, was not due to product inhibition, was saturable, and could be slowed by including increasing concns. of competing substrate.  However, the inactivated P 450 2B1 was still able to metabolize substrate if iodosobenzene was used as an alternate oxidant.  Inactivation of 2B1 with either N-[14C]-7-benzyl-1-aminobenzotriazole (BBT) or N-benzyl-1-amino-[14C]-2,3-benzotriazole resulted in the incorporation of covalent radiolabel into the apoprotein.  The stoichiometry of labeled metabolite adduct to protein was approx. 0.4:1 in both cases.  Identification of metabolites revealed the formation of and a new metabolite during catalysis of BBT by P 450 2B1.  Together, these data suggest that P 450 2B1 could be inactivated and labeled by more than one metabolite.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXT_y46KKqqLVg90H21EOLACvtfcHk0ljybz4QY7QOpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjsVOjtLg%253D&md5=b9a41820c71af919f7dc57b4d505b74b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Ftx960184o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx960184o%26sid%3Dliteratum%253Aachs%26aulast%3DKent%26aufirst%3DU.%2BM.%26aulast%3DBend%26aufirst%3DJ.%2BR.%26aulast%3DChamberlin%26aufirst%3DB.%2BA.%26aulast%3DGage%26aufirst%3DD.%2BA.%26aulast%3DHollenberg%26aufirst%3DP.%2BF.%26atitle%3DMechanism-based%2520inactivation%2520of%2520cytochrome%2520P450%25202B1%2520by%2520N-benzyl-1-aminobenzotriazole%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1997%26volume%3D10%26spage%3D600%26epage%3D608%26doi%3D10.1021%2Ftx960184o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of cytochrome P450 4B1 (CYP4B1) monooxygenase complexed with octane discloses several structural adaptations for omega-hydroxylation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">5610</span>– <span class="NLM_lpage">5621</span>, <span class="refDoi"> DOI: 10.1074/jbc.M117.775494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1074%2Fjbc.M117.775494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=28167536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltl2nsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=5610-5621&author=M.+H.+Hsuauthor=B.+R.+Baerauthor=A.+E.+Rettieauthor=E.+F.+Johnson&title=The+crystal+structure+of+cytochrome+P450+4B1+%28CYP4B1%29+monooxygenase+complexed+with+octane+discloses+several+structural+adaptations+for+omega-hydroxylation&doi=10.1074%2Fjbc.M117.775494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The Crystal Structure of Cytochrome P450 4B1 (CYP4B1) Monooxygenase Complexed with Octane Discloses Several Structural Adaptations for ω-Hydroxylation</span></div><div class="casAuthors">Hsu, Mei-Hui; Baer, Brian R.; Rettie, Allan E.; Johnson, Eric F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5610-5621</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">P 450 family 4 fatty acid ω-hydroxylases preferentially oxygenate primary C-H bonds over adjacent, energetically favored secondary C-H bonds, but the mechanism explaining this intriguing preference is unclear.  To this end, the structure of rabbit P 450 4B1 complexed with its substrate octane was detd. by X-ray crystallog. to define features of the active site that contribute to a preference for ω-hydroxylation.  The structure indicated that octane is bound in a narrow active-site cavity that limits access of the secondary C-H bond to the reactive intermediate.  A highly conserved sequence motif on helix I contributes to positioning the terminal carbon of octane for ω-hydroxylation.  Glu-310 of this motif auto-catalytically forms an ester bond with the heme 5-Me, and the immobilized Glu-310 contributes to substrate positioning.  The preference for ω-hydroxylation was decreased in an E310A mutant having a shorter side chain, but the overall rates of metab. were retained.  E310D and E310Q substitutions having longer side chains exhibit lower overall rates, likely due to higher conformational entropy for these residues, but they retained high preferences for octane ω-hydroxylation.  Sequence comparisons indicated that active-site residues constraining octane for ω-hydroxylation are conserved in family 4 P450s.  Moreover, the heme 7-propionate is positioned in the active site and provides addnl. restraints on substrate binding.  In conclusion, P 450 4B1 exhibits structural adaptations for ω-hydroxylation that include changes in the conformation of the heme and changes in a highly conserved helix I motif that is assocd. with selective oxygenation of unactivated primary C-H bonds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtBwKWqT0GV7Vg90H21EOLACvtfcHk0lhOS-8iYS4BFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltl2nsLw%253D&md5=3cbaf4a1e166b765b6d9248a2a421495</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.775494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.775494%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DM.%2BH.%26aulast%3DBaer%26aufirst%3DB.%2BR.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26atitle%3DThe%2520crystal%2520structure%2520of%2520cytochrome%2520P450%25204B1%2520%2528CYP4B1%2529%2520monooxygenase%2520complexed%2520with%2520octane%2520discloses%2520several%2520structural%2520adaptations%2520for%2520omega-hydroxylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D5610%26epage%3D5621%26doi%3D10.1074%2Fjbc.M117.775494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zöllner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pistorius, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bureik, M.</span></span> <span> </span><span class="NLM_article-title">Human CYP4Z1 catalyzes the in-chain hydroxylation of lauric acid and myristic acid</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1515/BC.2009.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1515%2FBC.2009.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=19090726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BD1MzhsVenug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=313-317&author=A.+Z%C3%B6llnerauthor=C.+A.+Draganauthor=D.+Pistoriusauthor=R.+Mullerauthor=H.+B.+Bodeauthor=F.+T.+Petersauthor=H.+H.+Maurerauthor=M.+Bureik&title=Human+CYP4Z1+catalyzes+the+in-chain+hydroxylation+of+lauric+acid+and+myristic+acid&doi=10.1515%2FBC.2009.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Human CYP4Z1 catalyzes the in-chain hydroxylation of lauric acid and myristic acid</span></div><div class="casAuthors">Zollner Andy; Dragan Calin-Aurel; Pistorius Dominik; Muller Rolf; Bode Helge B; Peters Frank T; Maurer Hans H; Bureik Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Biological chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">313-7</span>
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    </div><div class="casAbstract">Overexpression of human CYP4Z1, a cytochrome P450 enzyme, has been correlated with poor prognosis in human cancer.  However, its catalytic properties are not yet known.  We expressed this P450 in Schizosaccharomyces pombe and demonstrate by whole-cell biotransformation assays CYP4Z1-dependent in-chain hydroxylation of lauric and myristic acid, which in both cases leads to the formation of four different monohydroxylated products at positions omega-2, omega-3, omega-4, and omega-5, respectively.  The CYP4Z1-expressing fission yeast should be a new valuable tool for testing cancer drugs or for the development of new prodrug strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQRV_GXi11L-WBAiYWy6n3SfW6udTcc2eYeeYntJj9nH7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzhsVenug%253D%253D&md5=3cf9b55ac56400cb586afe9315c7320b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1515%2FBC.2009.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2009.030%26sid%3Dliteratum%253Aachs%26aulast%3DZ%25C3%25B6llner%26aufirst%3DA.%26aulast%3DDragan%26aufirst%3DC.%2BA.%26aulast%3DPistorius%26aufirst%3DD.%26aulast%3DMuller%26aufirst%3DR.%26aulast%3DBode%26aufirst%3DH.%2BB.%26aulast%3DPeters%26aufirst%3DF.%2BT.%26aulast%3DMaurer%26aufirst%3DH.%2BH.%26aulast%3DBureik%26aufirst%3DM.%26atitle%3DHuman%2520CYP4Z1%2520catalyzes%2520the%2520in-chain%2520hydroxylation%2520of%2520lauric%2520acid%2520and%2520myristic%2520acid%26jtitle%3DBiol.%2520Chem.%26date%3D2009%26volume%3D390%26spage%3D313%26epage%3D317%26doi%3D10.1515%2FBC.2009.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato-Watanabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kametani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomishima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kameo, K.</span></span> <span> </span><span class="NLM_article-title">HET0016, a potent and selective inhibitor of 20-HETE synthesizing enzyme</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0704101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1038%2Fsj.bjp.0704101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=11375247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktlSjurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2001&pages=325-329&author=N.+Miyataauthor=K.+Taniguchiauthor=T.+Sekiauthor=T.+Ishimotoauthor=M.+Sato-Watanabeauthor=Y.+Yasudaauthor=M.+Doiauthor=S.+Kametaniauthor=Y.+Tomishimaauthor=T.+Uekiauthor=M.+Satoauthor=K.+Kameo&title=HET0016%2C+a+potent+and+selective+inhibitor+of+20-HETE+synthesizing+enzyme&doi=10.1038%2Fsj.bjp.0704101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">HET0016, a potent and selective inhibitor of 20-HETE synthesizing enzyme</span></div><div class="casAuthors">Miyata, Noriyuki; Taniguchi, Kazuo; Seki, Takayuki; Ishimoto, Tsuyoshi; Sato-Watanabe, Mariko; Yasuda, Yoshiko; Doi, Mariko; Kametani, Shunichi; Tomishima, Yasumitsu; Ueki, Tomokazu; Sato, Masakazu; Kameo, Kazuya</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">325-329</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The present study examd. the inhibitory effects of N-Hydroxy-N'-(4-butyl-2-methylphenyl)-formamidine (HET0016) on the renal metab. of arachidonic acid by cytochrome P 450 (CYP) enzymes.  HET0016 exhibited a high degree of selectivity in inhibiting the formation of 20-hydroxy-5,8,11,14-eicosatetraenoic acid (20-HETE) in rat renal microsomes.  The IC50 value averaged 35±4 nM, whereas the IC50 value for inhibition of the formation of epoxyeicosatrienoic acids by HET0016 averaged 2800±300 nM.  In human renal microsomes, HET0016 potently inhibited the formation of 20-HETE with an IC50 value of 8.9±2.7 nM.  Higher concns. of HET0016 also inhibited the CYP2C9, CYP2D6 and CYP3A4-catalyzed substrates oxidn. with IC50 values of 3300, 83,900 and 71,000 nM.  The IC50 value for HET0016 on cyclooxygenase activity was 2300 nM.  These results indicate that HET0016 is a potent and selective inhibitor of CYP enzymes responsible for the formation of 20-HETE in man and rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn4OFmcinlNbVg90H21EOLACvtfcHk0lh6c77rJIKBmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktlSjurs%253D&md5=1ab62d8d96df94e800be1307036fa9d1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0704101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0704101%26sid%3Dliteratum%253Aachs%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DTaniguchi%26aufirst%3DK.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DIshimoto%26aufirst%3DT.%26aulast%3DSato-Watanabe%26aufirst%3DM.%26aulast%3DYasuda%26aufirst%3DY.%26aulast%3DDoi%26aufirst%3DM.%26aulast%3DKametani%26aufirst%3DS.%26aulast%3DTomishima%26aufirst%3DY.%26aulast%3DUeki%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DM.%26aulast%3DKameo%26aufirst%3DK.%26atitle%3DHET0016%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%252020-HETE%2520synthesizing%2520enzyme%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2001%26volume%3D133%26spage%3D325%26epage%3D329%26doi%3D10.1038%2Fsj.bjp.0704101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakinuma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kameo, K.</span></span> <span> </span><span class="NLM_article-title">Pyrazole and isoxazole derivatives as new, potent, and selective 20-hydroxy-5,8,11,14-eicosatetraenoic acid synthase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">5416</span>– <span class="NLM_lpage">5427</span>, <span class="refDoi"> DOI: 10.1021/jm020557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVClt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5416-5427&author=T.+Nakamuraauthor=M.+Satoauthor=H.+Kakinumaauthor=N.+Miyataauthor=K.+Taniguchiauthor=K.+Bandoauthor=A.+Kodaauthor=K.+Kameo&title=Pyrazole+and+isoxazole+derivatives+as+new%2C+potent%2C+and+selective+20-hydroxy-5%2C8%2C11%2C14-eicosatetraenoic+acid+synthase+inhibitors&doi=10.1021%2Fjm020557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrazole and Isoxazole Derivatives as New, Potent, and Selective 20-Hydroxy-5,8,11,14-eicosatetraenoic Acid Synthase Inhibitors</span></div><div class="casAuthors">Nakamura, Toshio; Sato, Masakazu; Kakinuma, Hiroyuki; Miyata, Noriyuki; Taniguchi, Kazuo; Bando, Kagumi; Koda, Ayumi; Kameo, Kazuya</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5416-5427</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In a previous paper, we reported the N-hydroxyformamidine deriv. HET0016 as a potent and selective 20-HETE synthase inhibitor.  Despite its attraction as a potential therapeutic agent for cerebral diseases, the prepn. of an injectable formulation of HET0016 was limited by its poor soly. under neutral conditions and instability under acidic conditions.  The instability of HET0016 in acidic conditions is due to the N-hydroxyformamidine moiety, which is considered to be essential for the potent and selective activity seen in our previous study.  The activity was maintained when the N-hydroxyformamidine moiety was replaced by an imidazole ring, but this was assocd. with a loss of selectivity for cytochrome P450s (CYPs).  However, other azole derivs. such as isoxazole deriv. (IC50 value 38±10 nM) and pyrazole deriv. (IC50 value 23±12 nM) showed potent and selective activities with improved stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq52rANjA51OrVg90H21EOLACvtfcHk0lh6c77rJIKBmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVClt7w%253D&md5=14f5b92c205e207b0bcccbed608f17c5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm020557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020557k%26sid%3Dliteratum%253Aachs%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DM.%26aulast%3DKakinuma%26aufirst%3DH.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DTaniguchi%26aufirst%3DK.%26aulast%3DBando%26aufirst%3DK.%26aulast%3DKoda%26aufirst%3DA.%26aulast%3DKameo%26aufirst%3DK.%26atitle%3DPyrazole%2520and%2520isoxazole%2520derivatives%2520as%2520new%252C%2520potent%252C%2520and%2520selective%252020-hydroxy-5%252C8%252C11%252C14-eicosatetraenoic%2520acid%2520synthase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5416%26epage%3D5427%26doi%3D10.1021%2Fjm020557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliw, E. H.</span></span> <span> </span><span class="NLM_article-title">cDna cloning and expression of CYP4F12, a novel human cytochrome P450</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1006/bbrc.2000.4191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1006%2Fbbrc.2000.4191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=11162607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1alsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2001&pages=892-897&author=J.+Bylundauthor=M.+Bylundauthor=E.+H.+Oliw&title=cDna+cloning+and+expression+of+CYP4F12%2C+a+novel+human+cytochrome+P450&doi=10.1006%2Fbbrc.2000.4191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">cDNA Cloning and Expression of CYP4F12, a Novel Human Cytochrome P450</span></div><div class="casAuthors">Bylund, Johan; Bylund, Mattias; Oliw, Ernst H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-897</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The cDNA of a novel human cytochrome P 450 was cloned from human liver by reverse transcription-polymerase chain reaction and designated CYP4F12.  The open reading frame coded for 524 amino acids, and the sequence could be aligned with 78-83% amino acid identity to the four human CYP4F enzymes (CYP4F2, CYP4F3, CYP4F8 and CYP4F11).  Northern blot anal. suggested three major transcripts of CYP4F12, which were detected in liver, kidney, colon, small intestine and heart.  The CYP4F12 gene contained 13 exons and was located at chromosome 19p13.1.  CYP4F12, expressed in yeast, oxidized arachidonic acid to 18-hydroxyarachidonic acid, and the ω-side chain of two stable prostaglandin (PG) H2 analogs (11,9-epoxymethano-PGH2 and 9,11-diazo-15-deoxy-PGH2).  CYP4F12 oxidized the ω-side chain of leukotriene B4, PGE2, PGF2α, PGH2, and 9,11-epoxymethano-PGH2 poorly.  Several CYP4F enzymes are important ω1- and ω2-hydroxylases of eicosanoids.  The physiol. function of CYP4F12 merits further investigation.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7lpkgnH2LSbVg90H21EOLACvtfcHk0lh6c77rJIKBmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1alsA%253D%253D&md5=910c810a8b56a3444d062bc7d7886c89</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.2000.4191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.2000.4191%26sid%3Dliteratum%253Aachs%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DBylund%26aufirst%3DM.%26aulast%3DOliw%26aufirst%3DE.%2BH.%26atitle%3DcDna%2520cloning%2520and%2520expression%2520of%2520CYP4F12%252C%2520a%2520novel%2520human%2520cytochrome%2520P450%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2001%26volume%3D280%26spage%3D892%26epage%3D897%26doi%3D10.1006%2Fbbrc.2000.4191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hidestrand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingelman-Sundberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliw, E. H.</span></span> <span> </span><span class="NLM_article-title">Identification of CYP4F8 in human seminal vesicles as a prominent 19-hydroxylase of prostaglandin endoperoxides</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">21844</span>– <span class="NLM_lpage">21849</span>, <span class="refDoi"> DOI: 10.1074/jbc.M001712200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1074%2Fjbc.M001712200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10791960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD3cXltFGjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=21844-21849&author=J.+Bylundauthor=M.+Hidestrandauthor=M.+Ingelman-Sundbergauthor=E.+H.+Oliw&title=Identification+of+CYP4F8+in+human+seminal+vesicles+as+a+prominent+19-hydroxylase+of+prostaglandin+endoperoxides&doi=10.1074%2Fjbc.M001712200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of CYP4F8 in human seminal vesicles as a prominent 19-hydroxylase of prostaglandin endoperoxides</span></div><div class="casAuthors">Bylund, Johan; Hidestrand, Mats; Ingelman-Sundberg, Magnus; Oliw, Ernst H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">21844-21849</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A novel cytochrome P 450, CYP4F8, was recently cloned from human seminal vesicles.  CYP4F8 was expressed in yeast.  Recombinant CYP4F8 oxygenated arachidonic acid to (18R)-hydroxyarachidonate, whereas prostaglandin (PG) D2, PGE1, PGE2, PGF2α, and leukotriene B4 appeared to be poor substrates.  Three stable PGH2 analogs, 9,11-epoxymethano-PGH2 (U-44069), 11,9-epoxymethano-PGH2 (U-46619), and 9,11-diazo-15-deoxy-PGH2 (U-51605) were rapidly metabolized by ω2- and ω3-hydroxylation.  U-44069 was oxygenated with a Vmax of ∼260 pmol min-1 pmol P 450-1 and a Km of ∼7 μM.  PGH2 decomps. mainly to PGE2 in buffer and to PGF2α by redn. with SnCl2.  CYP4F8 metabolized PGH2 to 19-hydroxy-PGH2, which decompd. to 19-hydroxy-PGE2 in buffer and could be reduced to 19-hydroxy-PGF2α with SnCl2.  18-Hydroxy metabolites were also formed (∼17%).  PGH1 was metabolized to 19- and 18-hydroxy-PGH1 in the same way.  Microsomes of human seminal vesicles oxygenated arachidonate, U-44069, U-46619, U-51605, and PGH2, similar to CYP4F8.  (19R)-Hydroxy-PGE1 and (19R)-hydroxy-PGE2 are the main prostaglandins of human seminal fluid.  We propose that they are formed by CYP4F8-catalyzed ω2-hydroxylation of PGH1 and PGH2 in the seminal vesicles and isomerization to (19R)-hydroxy-PGE by PGE synthase.  CYP4F8 is the first described hydroxylase with specificity and catalytic competence for prostaglandin endoperoxides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4ft2M9da0IbVg90H21EOLACvtfcHk0lj-nCWICGZTsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXltFGjt7g%253D&md5=4f4b78446a6fe9b40c9b27b803e19bc8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M001712200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M001712200%26sid%3Dliteratum%253Aachs%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DHidestrand%26aufirst%3DM.%26aulast%3DIngelman-Sundberg%26aufirst%3DM.%26aulast%3DOliw%26aufirst%3DE.%2BH.%26atitle%3DIdentification%2520of%2520CYP4F8%2520in%2520human%2520seminal%2520vesicles%2520as%2520a%2520prominent%252019-hydroxylase%2520of%2520prostaglandin%2520endoperoxides%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D21844%26epage%3D21849%26doi%3D10.1074%2Fjbc.M001712200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">U.S. Food & Drug Administration</span>. <a href="https://www.fda.gov/drugs/drug-interactions-labeling/drug-development-and-drug-interactions-table-substrates-inhibitors-and-inducers" class="extLink">https://www.fda.gov/drugs/drug-interactions-labeling/drug-development-and-drug-interactions-table-substrates-inhibitors-and-inducers</a> (accessed Oct, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Food+%26+Drug+Administration.+https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fdrug-interactions-labeling%2Fdrug-development-and-drug-interactions-table-substrates-inhibitors-and-inducers+%28accessed+Oct%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Breemen, R. B.</span></span> <span> </span><span class="NLM_article-title">High-throughput cytochrome P450 cocktail inhibition assay for assessing drug-drug and drug-botanical interactions</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1670</span>– <span class="NLM_lpage">1678</span>, <span class="refDoi"> DOI: 10.1124/dmd.115.065987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.115.065987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=26285764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28XislGhtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=1670-1678&author=G.+Liauthor=K.+Huangauthor=D.+Nikolicauthor=R.+B.+van+Breemen&title=High-throughput+cytochrome+P450+cocktail+inhibition+assay+for+assessing+drug-drug+and+drug-botanical+interactions&doi=10.1124%2Fdmd.115.065987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput cytochrome P450 cocktail inhibition assay for assessing drug-drug and drug-botanical interactions</span></div><div class="casAuthors">Li, Guannan; Huang, Ke; Nikolic, Dejan; van Breemen, Richard B.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1670-1678</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Detection of drug-drug interactions is essential during the early stages of drug discovery and development, and the understanding of drug-botanical interactions is important for the safe use of botanical dietary supplements.  Among the different forms of drug interactions that are known, inhibition of cytochrome P 450 (P 450) enzymes is the most common cause of drug-drug or drug-botanical interactions.  Therefore, a rapid and comprehensive mass spectrometry-based in vitro high-throughput P 450 cocktail inhibition assay was developed that uses 10 substrates simultaneously against nine CYP isoforms.  Including probe substrates for CYP1A2, CYP2A6, CYP2B6, CYP2C8, CYP2C9, CYP2C19, CYP2D6, CYP2E1, and two probes targeting different binding sites of CYP3A4/5, this cocktail simultaneously assesses at least as many P 450 enzymes as previous assays while remaining among the fastest due to short incubation times and rapid anal. using ultrahigh pressure liq. chromatog.-tandem mass spectrometry.  The method was validated using known inhibitors of each P 450 enzyme and then shown to be useful not only for single-compd. testing but also for the evaluation of potential drug-botanical interactions using the botanical dietary supplement licorice (Glycyrrhiza glabra) as an example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGRNxN4kxt-LVg90H21EOLACvtfcHk0lj-nCWICGZTsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislGhtr8%253D&md5=386d973b1dd84f04d5b8b14415aea2bd</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.065987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.065987%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DNikolic%26aufirst%3DD.%26aulast%3Dvan%2BBreemen%26aufirst%3DR.%2BB.%26atitle%3DHigh-throughput%2520cytochrome%2520P450%2520cocktail%2520inhibition%2520assay%2520for%2520assessing%2520drug-drug%2520and%2520drug-botanical%2520interactions%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26spage%3D1670%26epage%3D1678%26doi%3D10.1124%2Fdmd.115.065987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corcos, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salaun, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amet, Y.</span></span> <span> </span><span class="NLM_article-title">Metabolism of eicosapentaenoic and docosahexaenoic acids by recombinant human cytochromes P450</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>471</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2008.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.abb.2008.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=18206980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislKktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=471&publication_year=2008&pages=116-125&author=M.+Ferauthor=Y.+Dreanoauthor=D.+Lucasauthor=L.+Corcosauthor=J.+P.+Salaunauthor=F.+Berthouauthor=Y.+Amet&title=Metabolism+of+eicosapentaenoic+and+docosahexaenoic+acids+by+recombinant+human+cytochromes+P450&doi=10.1016%2Fj.abb.2008.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of eicosapentaenoic and docosahexaenoic acids by recombinant human cytochromes P450</span></div><div class="casAuthors">Fer, Maude; Dreano, Yvonne; Lucas, Daniele; Corcos, Laurent; Salauen, Jean-Pierre; Berthou, Francois; Amet, Yolande</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">471</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">116-125</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Epoxidn. and hydroxylation of arachidonic acid (AA) are both catalyzed by cytochromes P 450s (CYPs).  The oxidized metabolites are known to be involved in the regulation of vascular tone and renal function.  By using a panel of 15 human recombinant CYPs, this study demonstrates that other polyunsatd. long-chain fatty acids (PUFA-LC), esp. the ω3 fatty acids eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA), are also epoxidized.  The regioselectivity of epoxidn. of four PUFA-LC by CYPs was investigated.  Among the several CYPs tested, CYP2C9/2C19 and 1A2 were the most efficient in EPA and DHA epoxidns.  It ensued that 10 μM of these two ω3 fatty acids decreased by more than 80% and 60%, resp., the formation by CYP2C9 of AA-epoxidized derivs.  These findings suggest that some physiol. effects of ω3 fatty acids may be due to a shift in the generation of active epoxidized metabolites of AA through CYP-mediated catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLiRIElYJIhLVg90H21EOLACvtfcHk0lj-nCWICGZTsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislKktbc%253D&md5=b5c8e9cfd9f7ccdbbcc998eb0e015daa</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2008.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2008.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DFer%26aufirst%3DM.%26aulast%3DDreano%26aufirst%3DY.%26aulast%3DLucas%26aufirst%3DD.%26aulast%3DCorcos%26aufirst%3DL.%26aulast%3DSalaun%26aufirst%3DJ.%2BP.%26aulast%3DBerthou%26aufirst%3DF.%26aulast%3DAmet%26aufirst%3DY.%26atitle%3DMetabolism%2520of%2520eicosapentaenoic%2520and%2520docosahexaenoic%2520acids%2520by%2520recombinant%2520human%2520cytochromes%2520P450%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2008%26volume%3D471%26spage%3D116%26epage%3D125%26doi%3D10.1016%2Fj.abb.2008.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaspera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totah, R. A.</span></span> <span> </span><span class="NLM_article-title">Epoxyeicosatrienoic acids: formation, metabolism and potential role in tissue physiology and pathophysiology</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">757</span>– <span class="NLM_lpage">771</span>, <span class="refDoi"> DOI: 10.1517/17425250902932923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1517%2F17425250902932923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=19505190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsFamsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=757-771&author=R.+Kasperaauthor=R.+A.+Totah&title=Epoxyeicosatrienoic+acids%3A+formation%2C+metabolism+and+potential+role+in+tissue+physiology+and+pathophysiology&doi=10.1517%2F17425250902932923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Epoxyeicosatrienoic acids: formation, metabolism and potential role in tissue physiology and pathophysiology</span></div><div class="casAuthors">Kaspera, Ruediger; Totah, Rheem A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">757-771</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  CYP enzymes from the CYP2C and CYP2J subfamilies metabolize arachidonic acid in a regiospecific and stereoselective manner to eight epoxyeicosatrienoic acids (EETs).  Various EETs have been detected in the liver, as well as in many extrahepatic tissues, and have been implicated in numerous physiol. functions from cell signaling to vasodilation and angiogenesis.  This report reviews the sites of expression and activity of arachidonic acid epoxygenase CYP isoforms, as well as the physiol. role and metab. of EETs in various extrahepatic tissues.  Possible functions of EETs in tissue pathophysiol. and implications as potential drug targets are also discussed.  The most recent primary research literature on EET forming enzymes and the new physiol. functions of EETs in various tissues were reviewed.  Epoxyeicosatrienoic acids are important in maintaining the homeostasis and in responding to stress in various extra hepatic tissues.  It is not clear whether these effects are owing to EETs acting on a universal receptor or through a mechanism involving a second messenger.  A better understanding of the regulation of EET levels and their mechanism of action on various receptors will accelerate research aiming at developing therapeutic agents that target EET formation or metab. pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9X4GneX5N-LVg90H21EOLACvtfcHk0liivm7mJAujaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsFamsb8%253D&md5=2b011dc321a3b9a2bcd55275bd9437cd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F17425250902932923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425250902932923%26sid%3Dliteratum%253Aachs%26aulast%3DKaspera%26aufirst%3DR.%26aulast%3DTotah%26aufirst%3DR.%2BA.%26atitle%3DEpoxyeicosatrienoic%2520acids%253A%2520formation%252C%2520metabolism%2520and%2520potential%2520role%2520in%2520tissue%2520physiology%2520and%2520pathophysiology%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2009%26volume%3D5%26spage%3D757%26epage%3D771%26doi%3D10.1517%2F17425250902932923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesaros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuetz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donner, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falck, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capdevila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, D. A.</span></span> <span> </span><span class="NLM_article-title">CYP3A4 mediates growth of estrogen receptor-positive breast cancer cells in part by inducing nuclear translocation of phospho-Stat3 through biosynthesis of (+/-)-14,15-epoxyeicosatrienoic acid (EET)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">17543</span>– <span class="NLM_lpage">17559</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.198515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1074%2Fjbc.M110.198515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=21402692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVGjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=17543-17559&author=R.+Mitraauthor=Z.+Guoauthor=M.+Milaniauthor=C.+Mesarosauthor=M.+Rodriguezauthor=J.+Nguyenauthor=X.+Luoauthor=D.+Clarkeauthor=J.+Lambaauthor=E.+Schuetzauthor=D.+B.+Donnerauthor=N.+Puliauthor=J.+R.+Falckauthor=J.+Capdevilaauthor=K.+Guptaauthor=I.+A.+Blairauthor=D.+A.+Potter&title=CYP3A4+mediates+growth+of+estrogen+receptor-positive+breast+cancer+cells+in+part+by+inducing+nuclear+translocation+of+phospho-Stat3+through+biosynthesis+of+%28%2B%2F-%29-14%2C15-epoxyeicosatrienoic+acid+%28EET%29&doi=10.1074%2Fjbc.M110.198515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">CYP3A4 mediates growth of estrogen receptor-positive breast cancer cells in part by inducing nuclear translocation of phospho-stat3 through biosynthesis of (±)-14,15-epoxyeicosatrienoic acid (EET)</span></div><div class="casAuthors">Mitra, Ranjana; Guo, Zhijun; Milani, Monica; Mesaros, Clementina; Rodriguez, Mariangellys; Nguyen, Julia; Luo, Xianghua; Clarke, Duncan; Lamba, Jatinder; Schuetz, Erin; Donner, David B.; Puli, Narender; Falck, John R.; Capdevila, Jorge; Gupta, Kalpna; Blair, Ian A.; Potter, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">17543-17559</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">CYP3A4 expression in breast cancer correlates with decreased overall survival, but the mechanisms are unknown.  Cytochrome P 450 gene profiling by RNAi silencing demonstrates that CYP3A or 2C8 gene expression is specifically required for growth of the breast cancer lines MCF7, T47D, and MDA-MB-231.  CYP3A4 silencing blocks the cell cycle at the G2/M checkpoint and induces apoptosis in the MCF7 line, thereby inhibiting anchorage-dependent growth and survival.  CYP3A4 was profiled for NADPH-dependent arachidonic acid (AA) metab. and synthesized AA epoxygenase products (±)-8,9-, (±)-11,12-, and (±)-14,15-epoxyeicosatrienoic acid (EET) (total turnover of ∼2 pmol/pmol CYP3A4/min) but not hydroxylase products (±)-15-, (±)-19-, or 20-hydroxyeicosatetraenoic acid.  Furthermore, eicosanoid profiling revealed that MCF7 cells synthesize EETs in a CYP3A4-dependent manner.  The (±)-14,15-EET regioisomer selectively rescues breast cancer cells from CYP3A4 silencing in a concn.-dependent fashion and promotes mitogenesis and anchorage-dependent cloning.  Stat3 (Tyr-705) phosphorylation was inhibited by CYP3A4 silencing, providing a potential mechanism for CYP3A4 involvement in breast cancer cell growth.  Silencing Stat3 blocks breast cancer cell growth and abrogates (±)-14,15-EET-induced proliferation, indicating a Stat3 requirement for (±)-14,15-EET-mediated cell growth.  Although silencing of CYP3A4 reduces nuclear Tyr(P)-705-Stat3, (±)-14,15-EET restores this signaling process and promotes Tyr(P)-705-Stat3 translocation to the nucleus, suggesting that (±)-14,15-EET may be involved in an autocrine/paracrine pathway driving cell growth.  These studies indicate that CYP3A4 is a highly active AA epoxygenase that promotes Stat3-mediated breast cancer cell growth in part through (±)-14,15-EET biosynthesis.  Furthermore, these studies indicate an essential role for Stat3 as a mediator of epoxygenase activity in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0JlaMyFCsBrVg90H21EOLACvtfcHk0liivm7mJAujaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVGjtrs%253D&md5=0195525e306a8d31b5a35589f1cc8872</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.198515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.198515%26sid%3Dliteratum%253Aachs%26aulast%3DMitra%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DMilani%26aufirst%3DM.%26aulast%3DMesaros%26aufirst%3DC.%26aulast%3DRodriguez%26aufirst%3DM.%26aulast%3DNguyen%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DClarke%26aufirst%3DD.%26aulast%3DLamba%26aufirst%3DJ.%26aulast%3DSchuetz%26aufirst%3DE.%26aulast%3DDonner%26aufirst%3DD.%2BB.%26aulast%3DPuli%26aufirst%3DN.%26aulast%3DFalck%26aufirst%3DJ.%2BR.%26aulast%3DCapdevila%26aufirst%3DJ.%26aulast%3DGupta%26aufirst%3DK.%26aulast%3DBlair%26aufirst%3DI.%2BA.%26aulast%3DPotter%26aufirst%3DD.%2BA.%26atitle%3DCYP3A4%2520mediates%2520growth%2520of%2520estrogen%2520receptor-positive%2520breast%2520cancer%2520cells%2520in%2520part%2520by%2520inducing%2520nuclear%2520translocation%2520of%2520phospho-Stat3%2520through%2520biosynthesis%2520of%2520%2528%252B%252F-%2529-14%252C15-epoxyeicosatrienoic%2520acid%2520%2528EET%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D17543%26epage%3D17559%26doi%3D10.1074%2Fjbc.M110.198515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Strategies for discovering and derisking covalent, irreversible enzyme inhibitors</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.4155/fmc.10.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.4155%2Ffmc.10.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=20640225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFakur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=949-964&author=D.+S.+Johnsonauthor=E.+Weerapanaauthor=B.+F.+Cravatt&title=Strategies+for+discovering+and+derisking+covalent%2C+irreversible+enzyme+inhibitors&doi=10.4155%2Ffmc.10.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for discovering and derisking covalent, irreversible enzyme inhibitors</span></div><div class="casAuthors">Johnson, Douglas S.; Weerapana, Eranthie; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">949-964</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This article presents several covalent inhibitors, including examples of successful drugs, as well as highly selective, irreversible inhibitors of emerging therapeutic targets, such as fatty acid amide hydolase.  Covalent inhibitors have many desirable features, including increased biochem. efficiency of target disruption, less sensitivity toward pharmacokinetic parameters and increased duration of action that outlasts the pharmacokinetics of the compd.  Safety concerns that must be mitigated include lack of specificity and the potential immunogenicity of protein-inhibitor adduct(s).  Particular attention will be given to recent technologies, such as activity-based protein profiling, which allow one to define the proteome-wide selectivity patterns for covalent inhibitors in vitro and in vivo.  For instance, any covalent inhibitor can, in principle, be modified with a 'clickable' tag to generate an activity probe that is almost indistinguishable from the original agent.  These probes can be applied to any living system across a broad dose range to fully inventory their on and off targets.  The substantial no. of drugs on the market today that act by a covalent mechanism belies historical prejudices against the development of irreversibly acting therapeutic small mols.  Emerging proteomic technologies offer a means to systematically discriminate safe (selective) vs. deleterious (nonselective) covalent inhibitors and thus should inspire their future design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqm0whEybB-LVg90H21EOLACvtfcHk0liivm7mJAujaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFakur4%253D&md5=07e1f89418b168798ae5c1a9fa43e9c1</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.21%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DD.%2BS.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DStrategies%2520for%2520discovering%2520and%2520derisking%2520covalent%252C%2520irreversible%2520enzyme%2520inhibitors%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D949%26epage%3D964%26doi%3D10.4155%2Ffmc.10.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brueggemeier, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Cruz, E. S.</span></span> <span> </span><span class="NLM_article-title">Aromatase inhibitors in the treatment of breast cancer</span>. <i>Endocr. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1210/er.2004-0015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1210%2Fer.2004-0015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15814851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltVWru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=331-345&author=R.+W.+Brueggemeierauthor=J.+C.+Hackettauthor=E.+S.+Diaz-Cruz&title=Aromatase+inhibitors+in+the+treatment+of+breast+cancer&doi=10.1210%2Fer.2004-0015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatase inhibitors in the treatment of breast cancer</span></div><div class="casAuthors">Brueggemeier, Robert W.; Hackett, John C.; Diaz-Cruz, Edgar S.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine Reviews</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">331-345</span>CODEN:
                <span class="NLM_cas:coden">ERVIDP</span>;
        ISSN:<span class="NLM_cas:issn">0163-769X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review.  Estradiol, the most potent endogenous estrogen, is biosynthesized from androgens by the cytochrome P 450 enzyme complex called aromatase.  Aromatase is present in breast tissue, and intratumoral aromatase is the source of local estrogen prodn. in breast cancer tissues.  Inhibition of aromatase is an important approach for reducing growth-stimulatory effects of estrogens in estrogen-dependent breast cancer.  Steroidal inhibitors that have been developed to date build upon the basic androstenedione nucleus and incorporate chem. substituents at varying positions on the steroid.  Nonsteroidal aromatase inhibitors can be divided into three classes: aminoglutethimide-like mols., imidazole/triazole derivs., and flavonoid analogs.  Mechanism-based aromatase inhibitors are steroidal inhibitors that mimic the substrate, are converted by the enzyme to a reactive intermediate, and result in the inactivation of aromatase.  Both steroidal and non-steroidal aromatase inhibitors have shown clin. efficacy in the treatment of breast cancer.  The potent and selective third-generation aromatase inhibitors, anastrozole, letrozole, and exemestane, were introduced into the market as endocrine therapy in postmenopausal patients failing antiestrogen therapy alone or multiple hormonal therapies.  These agents are currently approved as first-line therapy for the treatment of postmenopausal women with metastatic estrogen-dependent breast cancer.  Several clin. studies of aromatase inhibitors are currently focusing on the use of these agents in the adjuvant setting for the treatment of early breast cancer.  Use of an aromatase inhibitor as initial therapy or after treatment with tamoxifen is now recommended as adjuvant hormonal therapy for a postmenopausal woman with hormone-dependent breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5rFlkckpA4bVg90H21EOLACvtfcHk0ljJ-nD9EW7LnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltVWru7w%253D&md5=d65778c7f1aea9ad16a3d24f37dccb07</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1210%2Fer.2004-0015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fer.2004-0015%26sid%3Dliteratum%253Aachs%26aulast%3DBrueggemeier%26aufirst%3DR.%2BW.%26aulast%3DHackett%26aufirst%3DJ.%2BC.%26aulast%3DDiaz-Cruz%26aufirst%3DE.%2BS.%26atitle%3DAromatase%2520inhibitors%2520in%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DEndocr.%2520Rev.%26date%3D2005%26volume%3D26%26spage%3D331%26epage%3D345%26doi%3D10.1210%2Fer.2004-0015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. B.</span></span> <i>Mechanism-Based Enzyme Inactivation: Chemistry and Enzymology</i>; <span class="NLM_publisher-name">CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, FL</span>, <span class="NLM_year">1988</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1988&author=R.+B.+Silverman&title=Mechanism-Based+Enzyme+Inactivation%3A+Chemistry+and+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSilverman%26aufirst%3DR.%2BB.%26btitle%3DMechanism-Based%2520Enzyme%2520Inactivation%253A%2520Chemistry%2520and%2520Enzymology%26pub%3DCRC%2520Press%26date%3D1988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giudici, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ornati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briatico, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzzetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Salle, E.</span></span> <span> </span><span class="NLM_article-title">6-Methylenandrosta-1,4-diene-3,17-dione (FCE 24304): a new irreversible aromatase inhibitor</span>. <i>J. Steroid Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1016/0022-4731(88)90129-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2F0022-4731%2888%2990129-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=3386266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaL1cXks1OitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1988&pages=391-394&author=D.+Giudiciauthor=G.+Ornatiauthor=G.+Briaticoauthor=F.+Buzzettiauthor=P.+Lombardiauthor=E.+di+Salle&title=6-Methylenandrosta-1%2C4-diene-3%2C17-dione+%28FCE+24304%29%3A+a+new+irreversible+aromatase+inhibitor&doi=10.1016%2F0022-4731%2888%2990129-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">6-Methylenandrosta-1,4-diene-3,17-dione (FCE 24304): a new irreversible aromatase inhibitor</span></div><div class="casAuthors">Giudici, D.; Ornati, G.; Briatico, G.; Buzzetti, F.; Lombardi, P.; Di Salle, E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Steroid Biochemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1-6</span>),
    <span class="NLM_cas:pages">391-4</span>CODEN:
                <span class="NLM_cas:coden">JSTBBK</span>;
        ISSN:<span class="NLM_cas:issn">0022-4731</span>.
    </div><div class="casAbstract">FCE 24304 (6-methylenandrosta-1,4-diene-3,17-dione), a new irreversible aromatase inhibitor, was identified and characterized in vitro and in vivo.  The compd. caused time-dependent inactivation of human placental aromatase with a half-time of 13.9 min and Ki of 26 nM.  When tested in PMSG-treated rats, ovarian aromatase activity was reduced 24 h after dosing by both the s.c. (50% ED 1.8 mg/kg) and the oral (50% ED 3.7 mg/kg) routes.  No interference with 5α-reductase activity and no significant binding affinity for the estrogen receptor were found.  A slight binding affinity for the androgen receptor (relative binding affinity 0.2% of dihydrotestosterone) was obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0X7BVetfC_LVg90H21EOLACvtfcHk0ljJ-nD9EW7LnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXks1OitLY%253D&md5=bb45165bda30bd0807cb0779f3ce8628</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2F0022-4731%2888%2990129-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-4731%252888%252990129-X%26sid%3Dliteratum%253Aachs%26aulast%3DGiudici%26aufirst%3DD.%26aulast%3DOrnati%26aufirst%3DG.%26aulast%3DBriatico%26aufirst%3DG.%26aulast%3DBuzzetti%26aufirst%3DF.%26aulast%3DLombardi%26aufirst%3DP.%26aulast%3Ddi%2BSalle%26aufirst%3DE.%26atitle%3D6-Methylenandrosta-1%252C4-diene-3%252C17-dione%2520%2528FCE%252024304%2529%253A%2520a%2520new%2520irreversible%2520aromatase%2520inhibitor%26jtitle%3DJ.%2520Steroid%2520Biochem.%26date%3D1988%26volume%3D30%26spage%3D391%26epage%3D394%26doi%3D10.1016%2F0022-4731%2888%2990129-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sodhi, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delarosa, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halladay, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driscoll, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dansette, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh, S. C.</span></span> <span> </span><span class="NLM_article-title">Inhibitory effects of trapping agents of sulfur drug reactive intermediates against major human cytochrome P450 isoforms</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>, <span class="NLM_elocation-id">1553</span> <span class="refDoi"> DOI: 10.3390/ijms18071553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.3390%2Fijms18071553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVehsrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&author=J.+K.+Sodhiauthor=E.+M.+Delarosaauthor=J.+S.+Halladayauthor=J.+P.+Driscollauthor=T.+Mulderauthor=P.+M.+Dansetteauthor=S.+C.+Khojasteh&title=Inhibitory+effects+of+trapping+agents+of+sulfur+drug+reactive+intermediates+against+major+human+cytochrome+P450+isoforms&doi=10.3390%2Fijms18071553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitory effects of trapping agents of sulfur drug reactive intermediates against major human cytochrome P450 isoforms</span></div><div class="casAuthors">Sodhi, Jasleen K.; Delarosa, Erlie Marie; Halladay, Jason S.; Driscoll, James P.; Mulder, Teresa; Dansette, Patrick M.; Khojasteh, S. Cyrus</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1553/1-1553/14</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In some cases, the formation of reactive species from the metab. of xenobiotics has been linked to toxicity and therefore it is imperative to detect potential bioactivation for candidate drugs during drug discovery.  Reactive species can covalently bind to trapping agents in in vitro incubations of compd. with human liver microsomes (HLM) fortified with β-NADP (NADPH), resulting in a stable conjugate of trapping agent and reactive species, thereby facilitating anal. detection and providing evidence of short-lived reactive metabolites.  Since reactive metabolites are typically generated by cytochrome P 450 (CYP) oxidn., it is important to ensure high concns. of trapping agents are not inhibiting the activities of CYP isoforms.  Here we assessed the inhibitory properties of fourteen trapping agents against the major human CYP isoforms (CYP1A2, 2C9, 2C19, 2D6 and 3A).  Based on our findings, eleven trapping agents displayed inhibition, three of which had IC50 values less than 1 mM (2-mercaptoethanol, N-methylmaleimide and N-ethylmaleimide (NEM)).  Three trapping agents (dimedone, N-acetyl-lysine and arsenite) did not inhibit CYP isoforms at concns. tested.  To illustrate effects of CYP inhibition by trapping agents on reactive intermediate trapping, an example drug (ticlopidine) and trapping agent (NEM) were chosen for further studies.  For the same amt. of ticlopidine (1 μM), increasing concns. of the trapping agent NEM (0.007-40 mM) resulted in a bell-shaped response curve of NEM-trapped ticlopidine S-oxide (TSO-NEM), due to CYP inhibition by NEM.  Thus, trapping studies should be designed to include several concns. of trapping agent to ensure optimal trapping of reactive metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4dfvGpPEhk7Vg90H21EOLACvtfcHk0ljJ-nD9EW7LnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVehsrzJ&md5=4eb3d6f04c90fb2b68fa4f99b82d447f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.3390%2Fijms18071553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071553%26sid%3Dliteratum%253Aachs%26aulast%3DSodhi%26aufirst%3DJ.%2BK.%26aulast%3DDelarosa%26aufirst%3DE.%2BM.%26aulast%3DHalladay%26aufirst%3DJ.%2BS.%26aulast%3DDriscoll%26aufirst%3DJ.%2BP.%26aulast%3DMulder%26aufirst%3DT.%26aulast%3DDansette%26aufirst%3DP.%2BM.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26atitle%3DInhibitory%2520effects%2520of%2520trapping%2520agents%2520of%2520sulfur%2520drug%2520reactive%2520intermediates%2520against%2520major%2520human%2520cytochrome%2520P450%2520isoforms%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26doi%3D10.3390%2Fijms18071553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C.</span></span> <span> </span><span class="NLM_article-title">Suicide substrates for the alanine racemase of Escherichia coli B</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1313</span>– <span class="NLM_lpage">1321</span>, <span class="refDoi"> DOI: 10.1021/bi00600a028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00600a028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADyaE1cXktVahs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1978&pages=1313-1321&author=E.+Wangauthor=C.+Walsh&title=Suicide+substrates+for+the+alanine+racemase+of+Escherichia+coli+B&doi=10.1021%2Fbi00600a028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Suicide substrates for the alanine racemase of Escherichia coli B</span></div><div class="casAuthors">Wang, Elizabeth; Walsh, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1313-21</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Alanine racemase was purified ∼9000-fold from E. coli and found to be a dimer of 100,000 daltons, contg. 1 mol. pyridoxal phosphate/subunit.  The mol. basis for irrversible active site-directed inactivation of the enzyme by a no. of suicide substrates was examd.  Both D and L isomers of β-fluoroalanine and β-chloroalanine partition between (a) α,β elimination to pyruvate, ammonia, and halide ion or (b) inactivation.  No racemization is detectable.  The Vmax for pyruvate formation from L-chloroalanine is ∼50-fold lower than from the D isomer of chloroalanine or either fluoroalanine.  However, both enantiomeric pairs partition identically, ∼830 turnovers/inactivating event.  This invariant partition ratio suggests that a common intermediate, the eneamino acid-pyridoxal phosphate complex, is the species responsible for inactivation, probably by Michael attack from a nucleophilic residue at the enzyme active site.  In keeping with this idea, O-carbamoyl-D-serine and O-acetyl-D-serine also undergo enzyme-catalyzed elimination for 830 turnovers before causing irreversible inactivation, presumably from the same intermediate.  In contrast, the L isomers of O-carbamoyl- or O-acetlyserine do not eliminate nor do they induce inactivation, but serve merely as reversible, competitive inhibitors of the enzyme.  This suggests asym. binding regions for bulky β substituents at the active site and suggests D isomers of substituted β-alanines would be preferentially effective enzyme inactivators.  D-Cycloserine also inactivates alanine racemase in time-dependent fashion.  Thus, both natural antibiotics, O-carbamoyl-D-serine and D-cycloserine, previously reported as reversible alanine racemase inhibitors, are in fact suicide substrates along with the β-haloalanines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohIkuziPLwbLVg90H21EOLACvtfcHk0ljnr7jjcbTOeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXktVahs7c%253D&md5=0d32c8dfec57779e35308e0c360e3f06</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fbi00600a028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00600a028%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DE.%26aulast%3DWalsh%26aufirst%3DC.%26atitle%3DSuicide%2520substrates%2520for%2520the%2520alanine%2520racemase%2520of%2520Escherichia%2520coli%2520B%26jtitle%3DBiochemistry%26date%3D1978%26volume%3D17%26spage%3D1313%26epage%3D1321%26doi%3D10.1021%2Fbi00600a028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duczak, N.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brougham, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.</span></span> <span> </span><span class="NLM_article-title">Automated screening with confirmation of mechanism-based inactivation of CYP3A4, CYP2C9, CYP2C19, CYP2D6, and CYP1A2 in pooled human liver microsomes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1211</span>– <span class="NLM_lpage">1219</span>, <span class="refDoi"> DOI: 10.1124/dmd.104.003475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.104.003475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15860655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntVamsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=1211-1219&author=H.+K.+Limauthor=N.+Duczakauthor=L.+Broughamauthor=M.+Elliotauthor=K.+Patelauthor=K.+Chan&title=Automated+screening+with+confirmation+of+mechanism-based+inactivation+of+CYP3A4%2C+CYP2C9%2C+CYP2C19%2C+CYP2D6%2C+and+CYP1A2+in+pooled+human+liver+microsomes&doi=10.1124%2Fdmd.104.003475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Automated screening with confirmation of mechanism-based inactivation of CYP3A4, CYP2C9, CYP2C19, CYP2D6, and CYP1A2 in pooled human liver microsomes</span></div><div class="casAuthors">Lim, Heng-Keang; Duczak, Nicholas, Jr.; Brougham, Linda; Elliot, Michael; Patel, Krupa; Chan, Kelvin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1211-1219</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A strategy is proposed to profile compds. for mechanism-based inactivation of CYP3A4, CYP2C19, CYP2C9, CYP2D6, and CYP1A2 based on an apparent partition ratio screen.  Potent positives from the screen are confirmed by time- and concn.-dependent inactivation assays.  Quasi-irreversible inhibitions are then differentiated from irreversible inactivations by oxidn. with potassium ferricyanide and/or dialysis.  The three-step screening procedure has been validated with acceptable accuracy and precision for detection and confirmation of mechanism-based inactivators in drug discovery.  We report here the apparent partition ratios for 19 mechanism-based inactivators and four quasi-irreversible inhibitors obtained under the same exptl. conditions.  The apparent partition ratio screen was automated to provide throughput for detg. structure-mechanism-based inactivation relationships.  Information about reversibility can be used to assess potential toxicity mediated by covalent adducts, as well as the potential for pharmacokinetic drug-drug interactions.  Direct comparison of known mechanism-based inactivators and quasi-irreversible inhibitors, based on our screening of apparent partition ratios, has identified ritonavir, mibefradil, and azamulin as highly effective mechanism-based inactivators; e.g., 1 mol of CYP3A4 was inactivated on turnover of about 2 mol of compd.  Other mechanism-based inactivators we identified include bergamottin (CYP1A2 besides previously reported CYP3A4), troglitazone (CYP3A4), rosiglitazone (CYP3A4), and pioglitazone (CYP3A4).  Comparison of the apparent partition ratios and inactivation clearance data for the three glitazones suggests that the chromane moiety on troglitazone contributes to its greater potency for mechanism-based inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqMduVu8I82bVg90H21EOLACvtfcHk0ljnr7jjcbTOeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntVamsLc%253D&md5=354d611d2a5480fb79281f1eb6bd2e16</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1124%2Fdmd.104.003475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.104.003475%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DH.%2BK.%26aulast%3DDuczak%26aufirst%3DN.%26aulast%3DBrougham%26aufirst%3DL.%26aulast%3DElliot%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DK.%26atitle%3DAutomated%2520screening%2520with%2520confirmation%2520of%2520mechanism-based%2520inactivation%2520of%2520CYP3A4%252C%2520CYP2C9%252C%2520CYP2C19%252C%2520CYP2D6%252C%2520and%2520CYP1A2%2520in%2520pooled%2520human%2520liver%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26spage%3D1211%26epage%3D1219%26doi%3D10.1124%2Fdmd.104.003475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speirs, V.</span></span> <span> </span><span class="NLM_article-title">Choosing the right cell line for breast cancer research</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">215</span> <span class="refDoi"> DOI: 10.1186/bcr2889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1186%2Fbcr2889" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=21884641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FkvFCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&author=D.+L.+Hollidayauthor=V.+Speirs&title=Choosing+the+right+cell+line+for+breast+cancer+research&doi=10.1186%2Fbcr2889"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Choosing the right cell line for breast cancer research</span></div><div class="casAuthors">Holliday Deborah L; Speirs Valerie</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research : BCR</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">215</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer is a complex and heterogeneous disease.  Gene expression profiling has contributed significantly to our understanding of this heterogeneity at a molecular level, refining taxonomy based on simple measures such as histological type, tumour grade, lymph node status and the presence of predictive markers like oestrogen receptor and human epidermal growth factor receptor 2 (HER2) to a more sophisticated classification comprising luminal A, luminal B, basal-like, HER2-positive and normal subgroups.  In the laboratory, breast cancer is often modelled using established cell lines.  In the present review we discuss some of the issues surrounding the use of breast cancer cell lines as experimental models, in light of these revised clinical classifications, and put forward suggestions for improving their use in translational breast cancer research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9z00kKYf1zjdblLE_E9PWfW6udTcc2ebHrQepA-vakLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FkvFCnuw%253D%253D&md5=9fbb9c5debbec50a5708666272fff46f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1186%2Fbcr2889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr2889%26sid%3Dliteratum%253Aachs%26aulast%3DHolliday%26aufirst%3DD.%2BL.%26aulast%3DSpeirs%26aufirst%3DV.%26atitle%3DChoosing%2520the%2520right%2520cell%2520line%2520for%2520breast%2520cancer%2520research%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2011%26volume%3D13%26doi%3D10.1186%2Fbcr2889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savas, Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span> <span> </span><span class="NLM_article-title">Conditional regulation of the human CYP4X1 and CYP4Z1 genes</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>436</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2005.02.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1016%2Fj.abb.2005.02.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=15797250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BD2MXis1GksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2005&pages=377-385&author=%C3%9C.+Savasauthor=M.-H.+Hsuauthor=K.+J.+Griffinauthor=D.+R.+Bellauthor=E.+F.+Johnson&title=Conditional+regulation+of+the+human+CYP4X1+and+CYP4Z1+genes&doi=10.1016%2Fj.abb.2005.02.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Conditional regulation of the human CYP4X1 and CYP4Z1 genes</span></div><div class="casAuthors">Savas, Uezen; Hsu, Mei-Hui; Griffin, Keith J.; Bell, David R.; Johnson, Eric F.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-385</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cytochrome P 450 genes (CYPs) encoding two new subfamilies designated CYP4X1 and CYP4Z1 were identified in the human genome and the Expressed Sequence Tags database.  Partial cDNAs encoding both P450s were isolated from human kidney and used to det. tissue distribution.  CYP4X1 was predominantly expressed in trachea and aorta, whereas CYP4Z1 mRNA was preferentially expressed in mammary tissue.  In T47-D cells, CYP4Z1 mRNA levels were induced by dexamethasone (14-fold) or by progesterone (10-fold).  The induction by these compds. was suppressed by co-treatment with the progesterone and glucocorticoid receptor antagonist mifepristone (RU486).  In the progesterone receptor neg. MCF-7 cells, CYP4Z1 mRNA was induced by dexamethasone but not by progesterone treatment.  CYP4Z1 mRNA levels were unaffected by 17β-estradiol.  In confluent cultures of human hepatoma HepG2 cells that stably express a mouse peroxisome proliferator activated receptor-α (PPARα) mutant, CYP4X1 mRNA was undetectable in vehicle-treated cells but was readily detectable following addn. of the PPARα agonist Wy14643.  This suggests that PPARα activation can affect human CYP4X1 gene transcription.  These results demonstrate selective tissue expression and implicate PPARα in CYP4X1 regulation, and the glucocorticoid and progesterone receptors in CYP4Z1 gene activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRacpMSaOSJbVg90H21EOLACvtfcHk0lgS0f1EuLF21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXis1GksL0%253D&md5=07cd2eaba5741f1595d546fc62070851</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2005.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2005.02.022%26sid%3Dliteratum%253Aachs%26aulast%3DSavas%26aufirst%3D%25C3%259C.%26aulast%3DHsu%26aufirst%3DM.-H.%26aulast%3DGriffin%26aufirst%3DK.%2BJ.%26aulast%3DBell%26aufirst%3DD.%2BR.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26atitle%3DConditional%2520regulation%2520of%2520the%2520human%2520CYP4X1%2520and%2520CYP4Z1%2520genes%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2005%26volume%3D436%26spage%3D377%26epage%3D385%26doi%3D10.1016%2Fj.abb.2005.02.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishikiori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iizuka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichiba, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukushima, T.</span></span> <span> </span><span class="NLM_article-title">Determination of free fatty acids in human serum by HPLC with fluorescence detection</span>. <i>J. Chromatogr. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">537</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1093/chromsci/bmu081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1093%2Fchromsci%2Fbmu081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=25033982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlGrt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2015&pages=537-541&author=M.+Nishikioriauthor=H.+Iizukaauthor=H.+Ichibaauthor=K.+Sadamotoauthor=T.+Fukushima&title=Determination+of+free+fatty+acids+in+human+serum+by+HPLC+with+fluorescence+detection&doi=10.1093%2Fchromsci%2Fbmu081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of free fatty acids in human serum by HPLC with fluorescence detection</span></div><div class="casAuthors">Nishikiori, Minami; Iizuka, Hideaki; Ichiba, Hideaki; Sadamoto, Kiyomi; Fukushima, Takeshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chromatographic Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">537-541</span>CODEN:
                <span class="NLM_cas:coden">JCHSBZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-9665</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">It has been suggested that serum concns. of polyunsatd. essential fatty acids correlate with the symptoms or severity of various diseases, including depression and Alzheimer-type dementia, and that detn. of serum fatty acids might be important for disease diagnosis.  Thus, we developed to analyze serum fatty acids in healthy individuals by using a high-performance liq. chromatog. method with fluorescence detection, because free fatty acids have a carboxyl group that can be derivatized with a fluorescent reagent, 4-N,N-dimethylaminosulfonyl-7-N-(2-aminoethyl)amino-2,1,3-benzoxadiazole.  This approach could quantify five types of free fatty acids [α-linolenic acid (ALA), palmitoleic acid (PLA), arachidonic acid (AA), linoleic acid (LA) and oleic acid (OA)] in human serum.  The detection limits of the method were in the range of 2.29-4.75 fmol (signal-to-noise ratio 3), and abs. concns. of ALA, PLA, AA, LA and OA were 8.27 ± 1.04, 18.8 ± 2.95, 49.9 ± 4.03, 230 ± 18.1 and 201 ± 22.1 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKl5XFXTrRbVg90H21EOLACvtfcHk0lgS0f1EuLF21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlGrt7bL&md5=0aae334c866789a70f7dc8473fedd5de</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fchromsci%2Fbmu081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fchromsci%252Fbmu081%26sid%3Dliteratum%253Aachs%26aulast%3DNishikiori%26aufirst%3DM.%26aulast%3DIizuka%26aufirst%3DH.%26aulast%3DIchiba%26aufirst%3DH.%26aulast%3DSadamoto%26aufirst%3DK.%26aulast%3DFukushima%26aufirst%3DT.%26atitle%3DDetermination%2520of%2520free%2520fatty%2520acids%2520in%2520human%2520serum%2520by%2520HPLC%2520with%2520fluorescence%2520detection%26jtitle%3DJ.%2520Chromatogr.%2520Sci.%26date%3D2015%26volume%3D53%26spage%3D537%26epage%3D541%26doi%3D10.1093%2Fchromsci%2Fbmu081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span> <span> </span><span class="NLM_article-title">An optimized high throughput clean-up method using mixed-mode SPE plate for the analysis of free arachidonic acid in plasma by LC-MS/MS</span>. <i>Int. J. Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2015</i></span>, <span class="NLM_elocation-id">374819</span> <span class="refDoi"> DOI: 10.1155/2015/374819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1155%2F2015%2F374819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=25873969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A280%3ADC%252BC2Mjkt12ktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&author=W.+Wangauthor=S.+Qinauthor=L.+Liauthor=X.+Chenauthor=Q.+Wangauthor=J.+Wei&title=An+optimized+high+throughput+clean-up+method+using+mixed-mode+SPE+plate+for+the+analysis+of+free+arachidonic+acid+in+plasma+by+LC-MS%2FMS&doi=10.1155%2F2015%2F374819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">An Optimized High Throughput Clean-Up Method Using Mixed-Mode SPE Plate for the Analysis of Free Arachidonic Acid in Plasma by LC-MS/MS</span></div><div class="casAuthors">Wang Wan; Qin Suzi; Li Linsen; Chen Xiaohua; Wang Qunjie; Wei Junfu</div><div class="citationInfo"><span class="NLM_cas:title">International journal of analytical chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2015</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">374819</span>
        ISSN:<span class="NLM_cas:issn">1687-8760</span>.
    </div><div class="casAbstract">A high throughput sample preparation method was developed utilizing mixed-mode solid phase extraction (SPE) in 96-well plate format for the determination of free arachidonic acid in plasma by LC-MS/MS.  Plasma was mixed with 3% aqueous ammonia and loaded into each well of 96-well plate.  After washing with water and methanol sequentially, 3% of formic acid in acetonitrile was used to elute arachidonic acid.  The collected fraction was injected onto a reversed phase column at 30°C with mobile phase of acetonitrile/water (70 : 30, v/v) and detected by LC-MS/MS coupled with electrospray ionization (ESI) in multiple reaction monitoring (MRM) mode.  The calibration curve ranged from 10 to 2500 ng/mL with sufficient linearity (r (2) = 0.9999).  The recoveries were in the range of 99.38% to 103.21% with RSD less than 6%.  The limit of detection is 3 ng/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS_GqxiFHx-9v_I8Jvzw4xCfW6udTcc2eZleoTGIgnbQbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mjkt12ktg%253D%253D&md5=af28e6023a9f80194361433eb784ce13</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1155%2F2015%2F374819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2015%252F374819%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DWei%26aufirst%3DJ.%26atitle%3DAn%2520optimized%2520high%2520throughput%2520clean-up%2520method%2520using%2520mixed-mode%2520SPE%2520plate%2520for%2520the%2520analysis%2520of%2520free%2520arachidonic%2520acid%2520in%2520plasma%2520by%2520LC-MS%252FMS%26jtitle%3DInt.%2520J.%2520Anal.%2520Chem.%26date%3D2015%26volume%3D2015%26doi%3D10.1155%2F2015%2F374819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Decker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giallonardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgener, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marowsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falck, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morisseau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammock, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruzdev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeldin, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arand, M.</span></span> <span> </span><span class="NLM_article-title">EH3 (ABHD9): the first member of a new epoxide hydrolase family with high activity for fatty acid epoxides</span>. <i>J. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2038</span>– <span class="NLM_lpage">2045</span>, <span class="refDoi"> DOI: 10.1194/jlr.M024448</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1194%2Fjlr.M024448" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=22798687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOmtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2012&pages=2038-2045&author=M.+Deckerauthor=M.+Adamskaauthor=A.+Croninauthor=F.+Di+Giallonardoauthor=J.+Burgenerauthor=A.+Marowskyauthor=J.+R.+Falckauthor=C.+Morisseauauthor=B.+D.+Hammockauthor=A.+Gruzdevauthor=D.+C.+Zeldinauthor=M.+Arand&title=EH3+%28ABHD9%29%3A+the+first+member+of+a+new+epoxide+hydrolase+family+with+high+activity+for+fatty+acid+epoxides&doi=10.1194%2Fjlr.M024448"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">EH3 (ABHD9): the first member of a new epoxide hydrolase family with high activity for fatty acid epoxides</span></div><div class="casAuthors">Decker, Martina; Adamska, Magdalena; Cronin, Annette; Di Giallonardo, Francesca; Burgener, Julia; Marowsky, Anne; Falck, John R.; Morisseau, Christophe; Hammock, Bruce D.; Gruzdev, Artiom; Zeldin, Darryl C.; Arand, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2038-2045</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology, Inc.</span>)
        </div><div class="casAbstract">Epoxide hydrolases are a small superfamily of enzymes important for the detoxification of chem. reactive xenobiotic epoxides and for the processing of endogenous epoxides that act as signaling mols.  Here, the identification of two human epoxide hydrolases, EH3 and EH4, are reported.  They share 45% sequence identity, thus representing a new family of mammalian epoxide hydrolases.  Quant. RT-PCR from mouse tissue indicates strongest EH3 expression in lung, skin, and upper gastrointestinal tract.  The recombinant enzyme shows a high turnover no. with 8,9-, 11,12-, and 14,15-epoxyeicosatrienoic acid (EET), as well as 9,10-epoxyoctadec-11-enoic acid (leukotoxin).  It is inhibited by a subclass of N,N'-disubstituted urea derivs., including 12-(3-adamantan-1-yl-ureido)-dodecanoic acid, 1-cyclohexyl-3-dodecylurea, and 1-(1-acetylpiperidin-4-yl)-3-(4-(trifluoromethoxy)phenyl)urea, compds. so far believed to be selective inhibitors of mammalian sol. epoxide hydrolase (sEH).  Its sensitivity to this subset of sEH inhibitors may have implications on the pharmacol. profile of these compds.  This is particularly relevant because sEH is a potential drug target, and clin. trials are under way exploring the value of sEH inhibitors in the treatment of hypertension and diabetes type II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpauMFKbDd0EbVg90H21EOLACvtfcHk0li0WdRIVzFsyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOmtLjF&md5=09e15bebf9ad1061339f4fe38d00e08a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M024448&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M024448%26sid%3Dliteratum%253Aachs%26aulast%3DDecker%26aufirst%3DM.%26aulast%3DAdamska%26aufirst%3DM.%26aulast%3DCronin%26aufirst%3DA.%26aulast%3DDi%2BGiallonardo%26aufirst%3DF.%26aulast%3DBurgener%26aufirst%3DJ.%26aulast%3DMarowsky%26aufirst%3DA.%26aulast%3DFalck%26aufirst%3DJ.%2BR.%26aulast%3DMorisseau%26aufirst%3DC.%26aulast%3DHammock%26aufirst%3DB.%2BD.%26aulast%3DGruzdev%26aufirst%3DA.%26aulast%3DZeldin%26aufirst%3DD.%2BC.%26aulast%3DArand%26aufirst%3DM.%26atitle%3DEH3%2520%2528ABHD9%2529%253A%2520the%2520first%2520member%2520of%2520a%2520new%2520epoxide%2520hydrolase%2520family%2520with%2520high%2520activity%2520for%2520fatty%2520acid%2520epoxides%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2012%26volume%3D53%26spage%3D2038%26epage%3D2045%26doi%3D10.1194%2Fjlr.M024448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, A. I.</span>; <span class="NLM_string-name">Tatchell, A. R.</span>; <span class="NLM_string-name">Furnis, B. S.</span>; <span class="NLM_string-name">Hannaford, A. J.</span>; <span class="NLM_string-name">Smith, P. W. G.</span></span> <i>Vogel’s Textbook of Practical Organic Chemistry</i>, <span class="NLM_edition">5</span>th ed.; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=A.+I.+Vogel&author=A.+R.+Tatchell&author=B.+S.+Furnis&author=A.+J.+Hannaford&author=P.+W.+G.+Smith&title=Vogel%E2%80%99s+Textbook+of+Practical+Organic+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DA.%2BI.%26btitle%3DVogel%25E2%2580%2599s%2520Textbook%2520of%2520Practical%2520Organic%2520Chemistry%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roellecke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaisers, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarov-Yarovoy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramm, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanenberg, H.</span></span> <span> </span><span class="NLM_article-title">Identification of amino acid determinants in CYP4B1 for optimal catalytic processing of 4-ipomeanol</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>465</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1042/BJ20140813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1042%2FBJ20140813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=25247810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFWiu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=465&publication_year=2015&pages=103-114&author=C.+Wiekauthor=E.+M.+Schmidtauthor=K.+Roelleckeauthor=M.+Freundauthor=M.+Nakanoauthor=E.+J.+Kellyauthor=W.+Kaisersauthor=V.+Yarov-Yarovoyauthor=C.+M.+Krammauthor=A.+E.+Rettieauthor=H.+Hanenberg&title=Identification+of+amino+acid+determinants+in+CYP4B1+for+optimal+catalytic+processing+of+4-ipomeanol&doi=10.1042%2FBJ20140813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of amino acid determinants in CYP4B1 for optimal catalytic processing of 4-ipomeanol</span></div><div class="casAuthors">Wiek, Constanze; Schmidt, Eva M.; Roellecke, Katharina; Freund, Marcel; Nakano, Mariko; Kelly, Edward J.; Kaisers, Wolfgang; Yarov-Yarovoy, Vladimir; Kramm, Christof M.; Rettie, Allan E.; Hanenberg, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">465</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-114</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Mammalian CYP4B1 enzymes are cytochrome P 450 mono-oxygenases that are responsible for the bioactivation of several exogenous pro-toxins including 4-ipomeanol (4-IPO).  In contrast with the orthologous rabbit enzyme, we show here that native human CYP4B1 with a serine residue at position 427 is unable to bioactivate 4-IPO and does not cause cytotoxicity in HepG2 cells and primary human T-cells that overexpress these enzymes.  We also demonstrate that a proline residue in the meander region at position 427 in human CYP4B1 and 422 in rabbit CYP4B1 is important for protein stability and rescues the 4-IPO bioactivation of the human enzyme, but is not essential for the catalytic activity of the rabbit CYP4B1 protein.  Systematic substitution of native and p.S427P human CYP4B1 with peptide regions from the highly active rabbit enzyme reveals that 18 amino acids in the wild-type rabbit CYP4B1 protein are key for conferring high 4-IPO metabolizing activity.  Introduction of 12 of the 18 amino acids that are also present at corresponding positions in other human CYP4 family members into the p.S427P human CYP4B1 protein results in a mutant human enzyme (P+12) that is as stable and as active as the rabbit wild-type CYP4B1 protein.  These 12 mutations cluster in the predicted B-C loop through F-helix regions and reveal new amino acid regions important to P 450 enzyme stability.  Finally, by minimally re-engineering the human CYP4B1 enzyme for efficient activation of 4-IPO, we have developed a novel human suicide gene system that is a candidate for adoptive cellular therapies in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBPRWo3GQ9vrVg90H21EOLACvtfcHk0li0WdRIVzFsyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFWiu7vI&md5=775cdbd19548aa4757dacf86b73eb3df</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1042%2FBJ20140813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20140813%26sid%3Dliteratum%253Aachs%26aulast%3DWiek%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DE.%2BM.%26aulast%3DRoellecke%26aufirst%3DK.%26aulast%3DFreund%26aufirst%3DM.%26aulast%3DNakano%26aufirst%3DM.%26aulast%3DKelly%26aufirst%3DE.%2BJ.%26aulast%3DKaisers%26aufirst%3DW.%26aulast%3DYarov-Yarovoy%26aufirst%3DV.%26aulast%3DKramm%26aufirst%3DC.%2BM.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26aulast%3DHanenberg%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520amino%2520acid%2520determinants%2520in%2520CYP4B1%2520for%2520optimal%2520catalytic%2520processing%2520of%25204-ipomeanol%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D465%26spage%3D103%26epage%3D114%26doi%3D10.1042%2FBJ20140813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roellecke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virts, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einholz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edson, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altvater, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Laer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheckenbach, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagenmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhardt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramm, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanenberg, H.</span></span> <span> </span><span class="NLM_article-title">Optimized human CYP4B1 in combination with the alkylator prodrug 4-ipomeanol serves as a novel suicide gene system for adoptive T-cell therapies</span>. <i>Gene Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1038/gt.2016.38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1038%2Fgt.2016.38" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=27092941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28XotFOrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=615-626&author=K.+Roelleckeauthor=E.+L.+Virtsauthor=R.+Einholzauthor=K.+Z.+Edsonauthor=B.+Altvaterauthor=C.+Rossigauthor=D.+von+Laerauthor=K.+Scheckenbachauthor=M.+Wagenmannauthor=D.+Reinhardtauthor=C.+M.+Krammauthor=A.+E.+Rettieauthor=C.+Wiekauthor=H.+Hanenberg&title=Optimized+human+CYP4B1+in+combination+with+the+alkylator+prodrug+4-ipomeanol+serves+as+a+novel+suicide+gene+system+for+adoptive+T-cell+therapies&doi=10.1038%2Fgt.2016.38"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Optimized human CYP4B1 in combination with the alkylator prodrug 4-ipomeanol serves as a novel suicide gene system for adoptive T-cell therapies</span></div><div class="casAuthors">Roellecke, K.; Virts, E. L.; Einholz, R.; Edson, K. Z.; Altvater, B.; Rossig, C.; von Laer, D.; Scheckenbach, K.; Wagenmann, M.; Reinhardt, D.; Kramm, C. M.; Rettie, A. E.; Wiek, C.; Hanenberg, H.</div><div class="citationInfo"><span class="NLM_cas:title">Gene Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">615-626</span>CODEN:
                <span class="NLM_cas:coden">GETHEC</span>;
        ISSN:<span class="NLM_cas:issn">0969-7128</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Engineering autologous or allogeneic T cells to express a suicide gene can control potential toxicity in adoptive T-cell therapies.  We recently reported the development of a novel human suicide gene system that is based on an orphan human cytochrome P 450 enzyme, CYP4B1, and the naturally occurring alkylator prodrug 4-ipomeanol.  The goal of this study was to systematically develop a clin. applicable self-inactivating lentiviral vector for efficient co-expression of CYP4B1 as an ER-located protein with two distinct types of cell surface proteins, either MACS selection genes for donor lymphocyte infusions after allogeneic stem cell transplantation or chimeric antigen receptors for retargeting primary T cells.  The U3 region of the myeloproliferative sarcoma virus in combination with the T2A site was found to drive high-level expression of our CYP4B1 mutant with truncated CD34 or CD271 as MACS suitable selection markers.  This lentiviral vector backbone was also well suited for co-expression of CYP4B1 with a codon-optimized CD19 chimeric antigen receptor (CAR) construct.  Finally, 4-ipomeanol efficiently induced apoptosis in primary T cells that co-express mutant CYP4B1 and the divergently located MACS selection and CAR genes.  In conclusion, we here developed a clin. suited lentiviral vector that supports high-level co-expression of cell surface proteins with a potent novel human suicide gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodG_hjPEjmZ7Vg90H21EOLACvtfcHk0lgoxPU5tfPuLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotFOrsrc%253D&md5=814f5d25968aa92252e01cf15ee980d5</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fgt.2016.38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fgt.2016.38%26sid%3Dliteratum%253Aachs%26aulast%3DRoellecke%26aufirst%3DK.%26aulast%3DVirts%26aufirst%3DE.%2BL.%26aulast%3DEinholz%26aufirst%3DR.%26aulast%3DEdson%26aufirst%3DK.%2BZ.%26aulast%3DAltvater%26aufirst%3DB.%26aulast%3DRossig%26aufirst%3DC.%26aulast%3Dvon%2BLaer%26aufirst%3DD.%26aulast%3DScheckenbach%26aufirst%3DK.%26aulast%3DWagenmann%26aufirst%3DM.%26aulast%3DReinhardt%26aufirst%3DD.%26aulast%3DKramm%26aufirst%3DC.%2BM.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26aulast%3DWiek%26aufirst%3DC.%26aulast%3DHanenberg%26aufirst%3DH.%26atitle%3DOptimized%2520human%2520CYP4B1%2520in%2520combination%2520with%2520the%2520alkylator%2520prodrug%25204-ipomeanol%2520serves%2520as%2520a%2520novel%2520suicide%2520gene%2520system%2520for%2520adoptive%2520T-cell%2520therapies%26jtitle%3DGene%2520Ther.%26date%3D2016%26volume%3D23%26spage%3D615%26epage%3D626%26doi%3D10.1038%2Fgt.2016.38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kowalski, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittington, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guttman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girhard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanenberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships for CYP4B1 bioactivation of 4-ipomeanol congeners: Direct correlation between cytotoxicity and trapped reactive intermediates</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2488</span>– <span class="NLM_lpage">2498</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrestox.9b00330</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.9b00330" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjtL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2019&pages=2488-2498&author=J.+P.+Kowalskiauthor=M.+G.+McDonaldauthor=D.+Whittingtonauthor=M.+Guttmanauthor=M.+Scianauthor=M.+Girhardauthor=H.+Hanenbergauthor=C.+Wiekauthor=A.+E.+Rettie&title=Structure-activity+relationships+for+CYP4B1+bioactivation+of+4-ipomeanol+congeners%3A+Direct+correlation+between+cytotoxicity+and+trapped+reactive+intermediates&doi=10.1021%2Facs.chemrestox.9b00330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships for CYP4B1 Bioactivation of 4-Ipomeanol Congeners: Direct Correlation between Cytotoxicity and Trapped Reactive Intermediates</span></div><div class="casAuthors">Kowalski, John P.; McDonald, Matthew G.; Whittington, Dale; Guttman, Miklos; Scian, Michele; Girhard, Marco; Hanenberg, Helmut; Wiek, Constanze; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2488-2498</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cytochrome P 450 4B1 (CYP4B1) has been explored as a candidate enzyme in suicide gene systems for its ability to bioactivate the natural product 4-ipomeanol (IPO) to a reactive species that causes cytotoxicity.  However, metabolic limitations of IPO necessitate discovery of new "pro-toxicant" substrates for CYP4B1.  In the present study, we examd. a series of synthetically facile N-alkyl-3-furancarboxamides for cytotoxicity in HepG2 cells expressing CYP4B1.  This compd. series maintains the furan "warhead" of IPO while replacing its alc. group with alkyl chains of varying length (C1 - C8).  Compds. with C3 - C6 carbon chain lengths showed similar potency to IPO (LD50 ∼ 5 μM).  Short chain analogs (<3 carbons) and long chain analogs (>6 carbons) exhibited reduced toxicity, resulting in a parabolic relationship between alkyl chain length and cytotoxicity.  A similar parabolic relationship was obsd. between alkyl chain length and reactive intermediate formation upon trapping of the putative enedial as a stable pyrrole adduct in incubations with purified recombinant rabbit CYP4B1 and common physiol. nucleophiles.  These parabolic relationships reflect the lower affinity of shorter chain compds. for CYP4B1 and increased ω-hydroxylation of the longer chain compds. by the enzyme.  Furthermore, modest time-dependent inhibition of CYP4B1 by N-pentyl-3-furancarboxamide was completely abolished when trapping agents were added, demonstrating escape of reactive intermediates from the enzyme after bioactivation.  An insulated CYP4B1 active site may explain the rarely obsd. direct correlation between adduct formation and cell toxicity reported here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUI6iSmCq-nrVg90H21EOLACvtfcHk0lgoxPU5tfPuLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjtL3P&md5=9a86a33de540c043ab2aa920cbaa4576</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.9b00330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.9b00330%26sid%3Dliteratum%253Aachs%26aulast%3DKowalski%26aufirst%3DJ.%2BP.%26aulast%3DMcDonald%26aufirst%3DM.%2BG.%26aulast%3DWhittington%26aufirst%3DD.%26aulast%3DGuttman%26aufirst%3DM.%26aulast%3DScian%26aufirst%3DM.%26aulast%3DGirhard%26aufirst%3DM.%26aulast%3DHanenberg%26aufirst%3DH.%26aulast%3DWiek%26aufirst%3DC.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DStructure-activity%2520relationships%2520for%2520CYP4B1%2520bioactivation%2520of%25204-ipomeanol%2520congeners%253A%2520Direct%2520correlation%2520between%2520cytotoxicity%2520and%2520trapped%2520reactive%2520intermediates%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2019%26volume%3D32%26spage%3D2488%26epage%3D2498%26doi%3D10.1021%2Facs.chemrestox.9b00330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, A. E.</span></span> <span> </span><span class="NLM_article-title">P450-Based drug-drug interactions of amiodarone and its metabolites: Diversity of inhibitory mechanisms</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1661</span>– <span class="NLM_lpage">1669</span>, <span class="refDoi"> DOI: 10.1124/dmd.115.065623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=10.1124%2Fdmd.115.065623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=26296708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;key=1%3ACAS%3A528%3ADC%252BC28Xis1Khsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=1661-1669&author=M.+G.+McDonaldauthor=N.+T.+Auauthor=A.+E.+Rettie&title=P450-Based+drug-drug+interactions+of+amiodarone+and+its+metabolites%3A+Diversity+of+inhibitory+mechanisms&doi=10.1124%2Fdmd.115.065623"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">P450-based drug-drug interactions of amiodarone and its metabolites: diversity of inhibitory mechanisms</span></div><div class="casAuthors">McDonald, Matthew G.; Au, Nicholas T.; Rettie, Allan E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1661-1669</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">In this study, IC50 shift and time-dependent inhibition (TDI) expts. were carried out to measure the ability of amiodarone (AMIO), and its circulating human metabolites, to reversibly and irreversibly inhibit CYP1A2, CYP2C9, CYP2D6, and CYP3A4 activities in human liver microsomes.  The [I]u/Ki,u values were calcd. and used to predict in vivo AMIO drug-drug interactions (DDIs) for pharmaceuticals metabolized by these four enzymes.  Based on these values, the minor metabolite N,N-didesethylamiodarone (DDEA) is predicted to be the major cause of DDIs with xenobiotics primarily metabolized by CYP1A2, CYP2C9, or CYP3A4, while AMIO and its N-monodesethylamiodarone (MDEA) deriv. are the most likely cause of interactions involving inhibition of CYP2D6 metab.  AMIO drug interactions predicted from the reversible inhibition of the four P 450 activities were found to be in good agreement with the magnitude of reported clin. DDIs with lidocaine, warfarin, metoprolol, and simvastatin.  The TDI expts. showed DDEA to be a potent inactivator of CYP1A2 (KI = 0.46 μM, kinact = 0.030 min-1), while MDEA was a moderate inactivator of both CYP2D6 (KI = 2.7 μM, kinact = 0.018 min-1) and CYP3A4 (KI = 2.6 μM, kinact = 0.016 min-1).  For DDEA and MDEA, mechanism-based inactivation appears to occur through formation of a metabolic intermediate complex.  Addnl. metabolic studies strongly suggest that CYP3A4 is the primary microsomal enzyme involved in the metab. of AMIO to both MDEA and DDEA.  In summary, these studies demonstrate both the diversity of inhibitory mechanisms with AMIO and the need to consider metabolites as the culprit in inhibitory P 450-based DDIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6wo6rgx9Ej7Vg90H21EOLACvtfcHk0lgoxPU5tfPuLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xis1Khsbg%253D&md5=712b2fec18f4c4aa416fd4d7d9d23819</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.065623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.065623%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DM.%2BG.%26aulast%3DAu%26aufirst%3DN.%2BT.%26aulast%3DRettie%26aufirst%3DA.%2BE.%26atitle%3DP450-Based%2520drug-drug%2520interactions%2520of%2520amiodarone%2520and%2520its%2520metabolites%253A%2520Diversity%2520of%2520inhibitory%2520mechanisms%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26spage%3D1661%26epage%3D1669%26doi%3D10.1124%2Fdmd.115.065623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i42"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00101">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_38205"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00101?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00101</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">CYP4Z1 expression systems; method for membrane preparation; IC<sub>50</sub> and shifted-IC<sub>50</sub> determination for inhibition of CYP4Z1 by <b>1</b>–<b>9</b>, of CYP4 isoforms by <b>6</b> and <b>7</b>, and of hepatic CYP isoforms by <b>6</b> and <b>7</b>; Luc-4F2/3 metabolism kinetics for CYP4F2, CYP4F3a, and CYP4F3b; analysis of CYP4F8 metabolism of pro-luciferins; internal validation of the HLM cocktail inhibition assay; inhibition of CYP4Z1-mediated AA metabolism by <b>7</b> in HepG2 membranes; inhibition of CYP4Z1 in HepG2 cells and HepG2 and T47D cell viability; <sup>1</sup>H and <sup>13</sup>C NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings spreadsheet (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_001.pdf">jm0c00101_si_001.pdf (4.06 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00101/suppl_file/jm0c00101_si_002.csv">jm0c00101_si_002.csv (2.13 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00101&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-9%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00101%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00101" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a59667d99d1c7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
