
<html lang="en"     class="pb-page"  data-request-id="68028784-d095-43c8-baf7-f01150be7b88"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01183;issue:issue:10.1021/jmcmar.2020.63.issue-18;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Potent Small-Molecule SIRT6 Activators: Structure–Activity Relationship and Anti-Pancreatic Ductal Adenocarcinoma Activity" /></meta><meta name="dc.Creator" content="Xiuli  Chen" /></meta><meta name="dc.Creator" content="Weining  Sun" /></meta><meta name="dc.Creator" content="Shenzhen  Huang" /></meta><meta name="dc.Creator" content="Hailin  Zhang" /></meta><meta name="dc.Creator" content="Guifeng  Lin" /></meta><meta name="dc.Creator" content="Hui  Li" /></meta><meta name="dc.Creator" content="Jingxin  Qiao" /></meta><meta name="dc.Creator" content="Linli  Li" /></meta><meta name="dc.Creator" content="Shengyong  Yang" /></meta><meta name="dc.Description" content="SIRT6 activation is thought to be a promising target for the treatment of many diseases, particularly cancer. Herein, we report the discovery of a series of new small-molecule SIRT6 activators. Str..." /></meta><meta name="Description" content="SIRT6 activation is thought to be a promising target for the treatment of many diseases, particularly cancer. Herein, we report the discovery of a series of new small-molecule SIRT6 activators. Str..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 12, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01183" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01183" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01183" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01183" /></link>
        
    
    

<title>Discovery of Potent Small-Molecule SIRT6 Activators: Structure–Activity Relationship and Anti-Pancreatic Ductal Adenocarcinoma Activity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01183" /></meta><meta property="og:title" content="Discovery of Potent Small-Molecule SIRT6 Activators: Structure–Activity Relationship and Anti-Pancreatic Ductal Adenocarcinoma Activity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0018.jpeg" /></meta><meta property="og:description" content="SIRT6 activation is thought to be a promising target for the treatment of many diseases, particularly cancer. Herein, we report the discovery of a series of new small-molecule SIRT6 activators. Structure–activity relationship analyses led to the identification of the most potent compound, 2-(1-benzofuran-2-yl)-N-(diphenylmethyl) quinoline-4-carboxamide (12q), which showed an EC1.5 value of 0.58 ± 0.12 μM and an EC50 value of 5.35 ± 0.69 μM against SIRT6-dependent peptide deacetylation in FLUOR DE LYS assay. It exhibited weak or no activity against other HDAC family members as well as 415 kinases, indicating good selectivity for SIRT6. 12q significantly inhibited the proliferation and migration of pancreatic ductal adenocarcinoma (PDAC) cells in vitro. It also markedly suppressed the tumor growth in a PDAC tumor xenograft model. This compound showed attractive pharmacokinetic properties. Overall, 12q could be a good lead compound for the treatment of PDAC, and it is worthy of further study." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01183"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01183">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01183&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01183&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01183&amp;href=/doi/10.1021/acs.jmedchem.0c01183" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 10474-10495</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01125" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01335" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Potent Small-Molecule SIRT6 Activators: Structure–Activity Relationship and Anti-Pancreatic Ductal Adenocarcinoma Activity</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Xiuli Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiuli Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiuli++Chen">Xiuli Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Weining Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weining Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Drug Targeting and Drug Delivery System of Ministry of Education, West China School of Pharmacy, Sichuan University, Chengdu, Sichuan 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weining++Sun">Weining Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shenzhen Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shenzhen Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shenzhen++Huang">Shenzhen Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hailin Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hailin Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hailin++Zhang">Hailin Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guifeng Lin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guifeng Lin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guifeng++Lin">Guifeng Lin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hui Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hui Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hui++Li">Hui Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jingxin Qiao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jingxin Qiao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jingxin++Qiao">Jingxin Qiao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Linli Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Linli Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Drug Targeting and Drug Delivery System of Ministry of Education, West China School of Pharmacy, Sichuan University, Chengdu, Sichuan 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Linli++Li">Linli Li</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Shengyong Yang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shengyong Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#bac3dbd4ddc9c3fac9d9cf94dfdecf94d9d4"><span class="__cf_email__" data-cfemail="dca5bdb2bbafa59cafbfa9f2b9b8a9f2bfb2">[email protected]</span></a>. Phone: +86-28-85164063. Fax: +86-28-85164060.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shengyong++Yang">Shengyong Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5147-3746" title="Orcid link">http://orcid.org/0000-0001-5147-3746</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01183&amp;href=/doi/10.1021%2Facs.jmedchem.0c01183" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 10474–10495</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 12, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 July 2020</li><li><span class="item_label"><b>Published</b> online</span>12 August 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01183" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01183</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10474%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DXiuli%2BChen%252C%2BWeining%2BSun%252C%2BShenzhen%2BHuang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D18%26contentID%3Dacs.jmedchem.0c01183%26title%3DDiscovery%2Bof%2BPotent%2BSmall-Molecule%2BSIRT6%2BActivators%253A%2BStructure%25E2%2580%2593Activity%2BRelationship%2Band%2BAnti-Pancreatic%2BDuctal%2BAdenocarcinoma%2BActivity%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10495%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01183"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1665</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01183" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Potent Small-Molecule SIRT6 Activators: Structure–Activity Relationship and Anti-Pancreatic Ductal Adenocarcinoma Activity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xiuli&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Weining&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Shenzhen&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Hailin&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Guifeng&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Hui&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jingxin&quot;,&quot;last_name&quot;:&quot;Qiao&quot;},{&quot;first_name&quot;:&quot;Linli&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Shengyong&quot;,&quot;last_name&quot;:&quot;Yang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;10474-10495&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01183&quot;},&quot;abstract&quot;:&quot;SIRT6 activation is thought to be a promising target for the treatment of many diseases, particularly cancer. Herein, we report the discovery of a series of new small-molecule SIRT6 activators. Structure–activity relationship analyses led to the identification of the most potent compound, 2-(1-benzofuran-2-yl)-N-(diphenylmethyl) quinoline-4-carboxamide (12q), which showed an EC1.5 value of 0.58 ± 0.12 μM and an EC50 value of 5.35 ± 0.69 μM against SIRT6-dependent peptide deacetylation in FLUOR DE LYS assay. It exhibited weak or no activity against other HDAC family members as well as 415 kinases, indicating good selectivity for SIRT6. 12q significantly inhibited the proliferation and migration of pancreatic ductal adenocarcinoma (PDAC) cells in vitro. It also markedly suppressed the tumor growth in a PDAC tumor xenograft model. This compound showed attractive pharmacokinetic properties. Overall, 12q could be a good lead compound for the treatment of PDAC, and it is worthy of further study.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01183&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01183" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01183&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01183" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01183&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01183" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01183&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01183&amp;href=/doi/10.1021/acs.jmedchem.0c01183" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01183" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01183" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01183%26sid%3Dliteratum%253Aachs%26pmid%3D32787077%26genre%3Darticle%26aulast%3DChen%26date%3D2020%26atitle%3DDiscovery%2Bof%2BPotent%2BSmall-Molecule%2BSIRT6%2BActivators%253A%2BStructure%25E2%2580%2593Activity%2BRelationship%2Band%2BAnti-Pancreatic%2BDuctal%2BAdenocarcinoma%2BActivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D18%26spage%3D10474%26epage%3D10495%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291321" title="Bioactivity">Bioactivity</a>,</li><li><a href="/action/doSearch?ConceptID=292153" title="Tumors">Tumors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/jmcmar.2020.63.issue-18/20200924/jmcmar.2020.63.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">SIRT6 activation is thought to be a promising target for the treatment of many diseases, particularly cancer. Herein, we report the discovery of a series of new small-molecule SIRT6 activators. Structure–activity relationship analyses led to the identification of the most potent compound, 2-(1-benzofuran-2-yl)-<i>N</i>-(diphenylmethyl) quinoline-4-carboxamide (<b>12q</b>), which showed an EC<sub>1.5</sub> value of 0.58 ± 0.12 μM and an EC<sub>50</sub> value of 5.35 ± 0.69 μM against SIRT6-dependent peptide deacetylation in FLUOR DE LYS assay. It exhibited weak or no activity against other HDAC family members as well as 415 kinases, indicating good selectivity for SIRT6. <b>12q</b> significantly inhibited the proliferation and migration of pancreatic ductal adenocarcinoma (PDAC) cells <i>in vitro</i>. It also markedly suppressed the tumor growth in a PDAC tumor xenograft model. This compound showed attractive pharmacokinetic properties. Overall, <b>12q</b> could be a good lead compound for the treatment of PDAC, and it is worthy of further study.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34241" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34241" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Sirtuins are NAD<sup>+</sup>-dependent protein lysine deacetylases that participate in the regulation of many physiological processes, such as the cell cycle, metabolism, stress responses, and aging processes in prokaryotic and eukaryotic species.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> There are seven mammalian sirtuins, SIRT1-SIRT7, and they differ in their subcellular localization and in the substrate proteins they deacetylate.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Among these sirtuin family members, of particular importance is SIRT6. The basic functions of SIRT6 include hydrolyzing long-chain acylated substrates and catalyzing the deacetylation of <i>N</i><sup>ε</sup>-acetyl-lysines 9, 18, and 56 of histone H3 (H3K9ac, H3K18ac, and H3K56ac, respectively), which have been associated with many physiological and pathological phenotypes.<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14">(6−14)</a> The functional activation of SIRT6 might play an important role in prolonging lifespan and intervening in related diseases.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> A recent study demonstrated that SIRT6 is responsible for the more efficient DNA double-strand break repair in long-lived species, and increased SIRT6 activity corresponds to longer lifespans.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Numerous investigations have also shown that SIRT6 is involved in various diseases, such as neurodegenerative diseases,<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> diabetes,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> cardiovascular diseases,<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> and cancers.<a onclick="showRef(event, 'ref23 ref24 ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26 ref27 ref28">(23−28)</a> Notably, SIRT6 has been demonstrated to function as a tumor-suppressor gene in some cancers, implying that SIRT6 activation may have potential value in the treatment of these cancers.<a onclick="showRef(event, 'ref13 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref13 ref29 ref30">(13,29,30)</a> For example, SIRT6 was found to be critical for the suppression of pancreatic ductal adenocarcinoma (PDAC), one of the most lethal malignancies.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> SIRT6 inactivation promotes PDAC progression and metastasis through the upregulation of Lin28b, a negative regulator of let-7 microRNA. Knocking out the SIRT6 gene results in histone hyperacetylation and c-Myc recruitment at the Lin28b promoter and pronounced induction of Lin28b and downstream let-7 target genes, including IGF2BP1 and IGF2BP3.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In contrast, SIRT6 activation might suppress PDAC progression and reduce metastasis formation <i>via</i> the downregulation of Lin28b.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> SIRT6 activation is thus thought to be a promising target in PDAC treatment.</div><div class="NLM_p">You <i>et al.</i> reported the first synthetic SIRT6 activator, a pyrrolo[1,2-<i>a</i>]quinoxaline derivative (<b>UBCS039</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which binds at a distal region of the fatty acyl substrate site and “C site” of SIRT6.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> This compound showed an EC<sub>50</sub> value of 38 μM against SIRT6.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Recently, Huang <i>et al.</i> disclosed a new potent SIRT6 activator with cell activity, methyl 2-(<i>N</i>-(5-bromo-4-fluoro-2-methylphenyl)sulfamoyl)-5-(3,5-dichlorophenyl sulfonamido)benzoate (<b>MDL-800</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which displayed an EC<sub>50</sub> value of 10.3 μM against SIRT6.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a><b>MDL-800</b> exhibited considerable activity in decreasing H3K9ac and H3K56ac levels in human hepatocellular carcinoma (HCC) cells.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Obviously, the potency of these compounds is still not enough from the perspective of a good drug candidate. Therefore, more potent and selective SIRT6 activators with new skeletons are needed.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Reported synthetic SIRT6 activators.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To discover new SIRT6 activators, we performed a virtual screening against various commercial chemical databases, which led to the discovery of several hit compounds. Structural optimization to the most active hit compound was then carried out, and this process offered a number of SIRT6 activators. The most potent activator was subjected to further studies including assessment of its molecular level biological activities, selectivity, and <i>in vitro</i> and <i>in vivo</i> anti-PDAC activities.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36830" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36830" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Discovery of Hit Compounds with the Aid of Molecular Docking-Based Virtual Screening</h3><div class="NLM_p">To obtain new activators of SIRT6, molecular docking was adopted to screen commercial chemical databases including Specs, ChemDiv, Selleck, and MedChemExpress, as well as an in-house database. Here, the crystal structure of the SIRT6–<b>UBCS039</b> (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MF6">5MF6</a>) complex was used, and genetic optimization of ligand docking (GOLD) was adopted for molecular docking.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The SIRT6-specific acyl channel pocket was defined as the active site; this active site was very similar to the allosteric site of SIRT6 identified later by Huang <i>et al.</i><a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and was also found to accommodate SIRT6 modulators of the quercetin compound family.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> GoldScore<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and ID-Score<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> were utilized to estimate the strength of interactions between the compound and the protein receptor. The final ranking order of compounds was based on the consensus score of GoldScore and ID-Score.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> All the calculations were carried out with Discovery Studio (DS) 3.1 (Accelrys Inc., San Diego, CA, USA).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><div class="NLM_p">From the top ranked compounds, 40 compounds (<b>Hit01–Hit40</b>) were selected and purchased for biochemical assays. We first tested the activity of each of these compounds at a fixed concentration of 100 μM by the FLUOR DE LYS (FDL) assay using a fluorogenic AMC-myristoyl peptide substrate (Ac-EALPKK(Myr)-AMC).<a onclick="showRef(event, 'ref31 ref33 ref39'); return false;" href="javascript:void(0);" class="ref ref31 ref33 ref39">(31,33,39)</a> In this assay, four compounds (<b>Hit05</b>, <b>Hit07</b>, <b>Hit11</b>, and <b>Hit20</b>) showed clear activating effects at 100 μM against SIRT6 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B, and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Table S1</a>). We then measured the EC<sub>1.5</sub> values of these compounds; EC<sub>1.5</sub> is defined as the concentration of a compound at which the compound is able to increase the enzymatic activity by 50%.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> The calculated EC<sub>1.5</sub> values are 118.20 ± 1.33, 109.80 ± 1.09, 227.50 ± 1.31, and 27.14 ± 1.18 μM for <b>Hit05</b>, <b>Hit07</b>, <b>Hit11</b>, and <b>Hit20</b>, respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,C). Structural optimization and structure–activity relationship (SAR) analyses were then carried out on the most active compound, <b>Hit20</b>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>In vitro</i> SIRT6 enzymatic activity assay for the screened compounds. (A) Activation of SIRT6-dependent peptide demyristoylation by 40 selected compounds at 100 μM, determined with Ac-EALPKK(Myr)-AMC by the FDL assay. (B) Chemical structures of hit compounds <b>Hit05</b>, <b>Hit07</b>, <b>Hit11</b>, and <b>Hit20</b>. (C) Dose-dependent demyristoylation activities of SIRT6 by 4 hit compounds, determined with Ac-EALPKK(Myr)-AMC by the FDL assay. Data are presented as the mean ± s.d., <i>n</i> = 3 wells, from three independent experiments. (D) Target regions for the structural optimization and SAR analyses.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.2.  Structural Optimization and SAR Analyses of <b>Hit20</b></h3><div class="NLM_p">To facilitate the structural optimization, molecular docking was used again to predict the binding mode of <b>Hit20</b> with SIRT6. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Figure S1</a>, <b>Hit20</b> binds to the specific acyl channel pocket of SIRT6, a similar binding site of <b>UBCS039</b> and <b>MDL-801</b>.<a onclick="showRef(event, 'ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33">(31−33)</a> A face-to-face π–π interaction between the benzene ring of the quinoline scaffold and the phenyl ring of PHE86, and a σ–π interaction between the amino group of ALA-7 and the benzene ring of 3,4-dichlorobenzene are formed. We also noticed that there is some space not completely occupied around the 3,4-dichlorobenzene group (the 4-position carboxamide substituent on the quinoline, region I) and 3,4-dimethoxybenzene group (2-position on the quinoline, region II). It is expected that modifications to these two regions, for example, adding hydrogen bond donors or receptors, or increasing hydrophobic interaction, might benefit the improvement of bioactivity. Therefore, the subsequent structural optimization will focus on region I and region II (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). The SAR analyses are based on the activation potencies of the compounds against SIRT6, which were determined by an FDL assay with peptide Ac-EALPKK(Myr)-AMC as the substrate.</div><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6">2.2.1.  Impact of the 4-Position Carboxamide Substituents (Region I, R<sup>1</sup>) on the Quinoline</h4><div class="NLM_p">To examine the influence of 4-position carboxamide substituents (region I, R<sup>1</sup>) on the quinoline on the bioactivity, we fixed region II as the original 3,4-dimethoxy phenyl group and varied the group in region I. Here, region I groups chosen include phenyl groups with different substituents, as well as other fragments that might help increase hydrogen bonding interaction and/or hydrophobic interaction. Forty quinoline-4-carboxamide derivatives (<b>PC01–40</b>) with different R<sup>1</sup> substituents were purchased from commercial reagent companies. In addition, we also synthesized fifteen new quinoline-4-carboxamide derivatives (<b>5a–o</b>). The synthetic routes of <b>5a–o</b> are depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Condensation reactions of commercially available indolin-2,3-dione (<b>1</b>) and 3,4-dimethoxyacetophenone (<b>2</b>) in EtOH in the presence of an appropriate amount of KOH aqueous solution generated 2-(3,4-dimethoxyphenyl)quinoline-4-carboxylic acid (<b>3</b>) as an intermediate with a yield of 76%. Intermediate <b>3</b> was then reacted with various substituted anilines to offer final products <b>5a–o</b>. The final yields were between 45 and 79%.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>5a–o</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KOH, EtOH, 85 °C, 24 h; (b) (i) SOCl<sub>2</sub>, 107 °C, 5–6 h; (ii) R<sup>1</sup>-NH<sub>2</sub>, DMAP, DMF, RT, overnight.</p></p></figure><div class="NLM_p">The chemical structures and bioactivities of compounds <b>PC01–40</b> and <b>5a–o</b> are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Among compounds <b>PC01–24</b> and <b>5a–k</b> containing substituted phenyl groups at the R<sup>1</sup> position, eight compounds (<b>5a</b>, <b>PC12</b>, <b>PC14</b>, <b>PC17</b>, <b>5f–h</b>, and <b>5j</b>) displayed activation effects of >150% at 20 μM. EC<sub>1.5</sub> values of these compounds are 10.28 ± 2.38, 5.97 ± 1.99, 19.02 ± 1.14, 5.61 ± 1.24, 7.53 ± 1.31, 6.38 ± 1.98, 2.84 ± 0.37, and 4.77 ± 1.81 μM, respectively. For compounds <b>PC25–40</b> and <b>5l–o</b>, which contain a naphthalene ring, benzyl moiety, polyunsaturated ring, or heterocycle with larger substituents, six compounds (<b>5l</b>, <b>PC25–26</b>, and <b>PC28–30</b>) showed activation potencies of >150% at 20 μM. The EC<sub>1.5</sub> values of these compounds are 7.44 ± 2.08, 5.97 ± 1.98, 6.47 ± 4.40, 7.78 ± 1.60, 19.53 ± 1.35, and 8.14 ± 1.59 μM, respectively. Of all the fifty-five compounds, <b>5h</b>, which contains a 2-chloro-5-(trifluoromethyl)phenyl group, is the most active one in terms of the EC<sub>1.5</sub> values. We then fixed the R<sup>1</sup> substituent as a 2-chloro-5-(trifluoromethyl)phenyl group in the following optimization step.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Chemical Structures and Effect of Compounds <b>PC01–40</b> and <b>5a–o</b> on SIRT6-dependent Peptide Demyristoylation Activation</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0012.gif" alt="" id="GRAPHIC-d7e707-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0013.gif" alt="" id="gr11" /></img><div></div></div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9">2.2.2.  Effect of the 2-Position (Region II, R<sup>2</sup>) on the Quinoline</h4><div class="NLM_p">To explore the effect of the substituent at the 2-position (region II, R<sup>2</sup>) of quinoline, we installed a 5-member ring, 6-member ring, or benzohetercyclic ring in region II (R<sup>2</sup>) and fixed region I as the optimal 2-chloro-5-(trifluoromethyl)phenyl group. We purchased two quinoline-4-carboxamide derivatives with different substituents at R<sup>2</sup> (<b>PC41–42</b>) and synthesized thirteen new quinoline-4-carboxamide derivatives (<b>8a–m</b>). <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> illustrates the synthetic routes of compounds <b>8a–m</b>. Similar to before, the condensation reactions between indolin-2,3-dione (<b>1</b>) and various substituted acetophenones (<b>6a–m</b>) in EtOH in the presence of an appropriate amount of KOH aqueous solution generated 2-substituted quinolinone-4-carboxylic acids (<b>7a–m</b>) as intermediates. The final products (<b>8a–m</b>) were then obtained from the reactions of <b>7a–m</b> with commercially available 2-chloro-5-(trifluoromethyl) aniline.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>8a–m</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KOH, EtOH, 85 °C, 24 h; (b) (i) SOCl<sub>2</sub>, 107 °C, 5–6 h; (ii) 2-chloro-5-(trifluoromethyl)aniline, DMAP, DMF, RT, overnight.</p></p></figure><div class="NLM_p">The chemical structures and bioactivities of compounds <b>PC41–42</b> and <b>8a–m</b> are shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Compounds <b>PC41–42</b> and <b>8a</b>, containing a different substituted phenyl groups, did not activate SIRT6. Compound <b>8b</b> with 4-methoxyphenyl did activate SIRT6 with an EC<sub>1.5</sub> value of 20.72 ± 3.01 μM. Compounds <b>8d</b> and <b>8e</b>, bearing saturated five- or six-membered rings, also did not activate SIRT6. In addition, compounds <b>8f</b> and <b>8g</b>, which contain a (substituted) thiophene at the R<sup>2</sup> position, did not show obvious activity against SIRT6 at 20 μM. Interestingly, compounds <b>8h–m</b>, bearing a (substituted) furan or a benzo oxygen-containing heterocycle, displayed obvious activation effects, and <b>8j</b> is the most active one with an EC<sub>1.5</sub> value of 0.85 ± 0.36 μM.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Chemical Structures and Effect of Compounds <b>PC41–42</b> and <b>8a–m</b> on SIRT6-dependent Peptide Demyristoylation Activation</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0014.gif" alt="" id="GRAPHIC-d7e839-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0015.gif" alt="" id="gr12" /></img><div></div></div></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12">2.2.3.  Re-optimization of the 4-Position (Region I, R<sup>3</sup>) on the Quinoline</h4><div class="NLM_p">In the preceding SAR studies, we determined an optimal moiety (benzofuranyl group) for region II with region I fixed as 2-chloro-5-(trifluoromethyl)phenyl, which was also determined through optimization. To examine whether 2-chloro-5-(trifluoromethyl)phenyl is still optimal when region II is fixed as a benzofuranyl moiety, we synthesized seventeen compounds with different substituents at the 4-position (region I, R<sup>3</sup>) of the quinoline (<b>12a–q</b>). Compounds <b>12a–q</b> are prepared following the route outlined in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, which is very similar to that in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>12a–q</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KOH, EtOH, 85 °C, 24 h; (b) R<sup>3</sup>-NH<sub>2</sub>, HATU, HOBt, DMAP, 50 °C, overnight.</p></p></figure><div class="NLM_p">Bioactivities of compounds <b>12a–q</b> are shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. All the compounds showed obvious activities against SIRT6 at 20 μM. Compound <b>12q</b>, which contains a diphenylmethane group in region I, exhibited more potent activity than <b>8j</b>. The EC<sub>1.5</sub> value of compound <b>12q</b> is 0.72 ± 0.25 μM, which is 38-fold more potent than <b>Hit20</b>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Chemical Structure and Effect of Compounds <b>12a–q</b> on SIRT6-dependent Peptide Demyristoylation Activation</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0016.gif" alt="" id="GRAPHIC-d7e924-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0017.gif" alt="" id="gr13" /></img><div></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">2.3.  Bioactivity Validation of Compound <b>12q</b> by Various Biochemical and Biophysical Assays</h3><div class="NLM_p">To further validate the bioactivity of <b>12q</b><i>in vitro</i>, various biochemical and biophysical assays were performed, including FDL assays but with different substrate peptides, differential scanning fluorimetry (DSF) assays, isothermal titration calorimetry (ITC) assays, and surface plasmon resonance (SPR) assays. <b>MDL-800</b> was used as a reference compound because it is the most potent SIRT6 activator reported so far.</div><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16">2.3.1.  FDL Assay with Different Substrate Peptides</h4><div class="NLM_p">As indicated before,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> deacetylation is one of most important functions of SIRT6. We thus validated the influence of <b>12q</b> on the deacetylation activity of SIRT6. To this end, the FDL assay was used again, and acetyl peptide substrate (Ac-RYQK(Ac)-AMC) was used.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, compound <b>12q</b> dramatically enhanced SIRT6 deacetylation activity with an EC<sub>1.5</sub> value of 0.58 ± 0.12 μM, which is comparable to that of demyristoylation activity (0.72 ± 0.25 μM, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Dose-dependent curves of <b>12q</b> against SIRT6 deacetylation and demyristoylation are displayed in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B, respectively. Moreover, the calculated EC<sub>50</sub> values of <b>12q</b> are 5.35 ± 0.69 and 8.91 ± 1.81 μM for SIRT6 deacetylation and demyristoylation, respectively. Collectively, <b>12q</b> is able to enhance both the deacetylation activity and demyristoylation activity of SIRT6. Though the positive control <b>MDL-800</b> showed excellent SIRT6-dependant deacetylation activity, it could not enhance the demyristoylation activity of SIRT6 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), which is consistent with Huang’s result.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Bioactivities of <b>12q</b> against SIRT6 by various biochemical and biophysical assays. (A,B) Dose-activity curves of SIRT6-dependant deacetylation and demyristoylation of <b>12q</b>, determined with acetyl substrate peptide Ac-RYQK(Ac)-AMC and myristoyl substrate peptide Ac-EALPKK(Myr)-AMC. Data are presented as the mean ± s.d., <i>n</i> = 3 wells, from three independent experiments. (C) Changes in the thermodynamic stability of SIRT6 after the binding of <b>12q</b> or <b>MDL-800</b>. (D) ITC binding curves for SIRT6 and <b>12q</b>. (E) Representative SIRT6 binding curves and fit steady-state evaluation for <b>12q</b> using SPR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">One may notice that different peptides were used in deacetylation (Ac-RYQK(Ac)-AMC) and demyristoylation (Ac-EALPKK(Myr)-AMC) assays. To examine the possible influence of different peptides in FdL assays, we tested the activity of <b>12q</b> with Ac-RYQK(Myr)-AMC as the substrate. The measured EC<sub>1.5</sub> and EC<sub>50</sub> values are 0.49 ± 0.11 and 5.49 ± 0.57 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Figure S2</a>), respectively, which are comparable to the corresponding values in the cases of Ac-RYQK(Ac)-AMC and Ac-EALPKK(Myr)-AMC. All the results indicate that different peptide substrates have no significant impact on the bioactivity.</div></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17">2.3.2.  DSF Assay</h4><div class="NLM_p last">The bioactivity of <b>12q</b> was then validated by the DSF assay, which is a spectroscopic technique used to identify the thermal stability of a protein.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> In this assay, the recombinant human protein SIRT6 was used as the target protein. The thermal stability of SIRT6 in the presence or absence of small-molecule ligands was measured. The melting temperature (<i>T</i><sub>m</sub>) values of SIRT6 are 41.18 ± 0.10 and 42.19 ± 0.10 °C treated with dimethyl sulfoxide (DMSO) and <b>12q</b> at a concentration of 20 μM, respectively, indicating a Δ<i>T</i><sub>m</sub> value of 1.01 °C (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). At the same concentration, <b>MDL-800</b> displayed a Δ<i>T</i><sub>m</sub> value of 0.85 °C.</div></div><div id="sec2_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18">2.3.3.  ITC and SPR Assays</h4><div class="NLM_p last">ITC and SPR assays were adopted to measure the binding affinity of <b>12q</b> with SIRT6. In the ITC assay, <b>12q</b> showed an equilibrium dissociation constant (<i>K</i><sub>d</sub>) of 8.85 ± 1.89 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). The measured thermodynamic binding parameters are 14.33 ± 5.60 kcal mol<sup>–1</sup>, 71.20 cal mol<sup>–1</sup> K<sup>–1</sup>, −6.90 ± 5.60 kcal mol<sup>–1</sup>, and −21.23 kcal mol<sup>–1</sup> for Δ<i>H</i>, Δ<i>S</i>, Δ<i>G</i>, and −<i>T</i>Δ<i>S</i>, respectively. The stoichiometry factor (<i>N</i>) is 0.31 ± 0.098. In the SPR assay, compound <b>12q</b> displayed an equilibrium dissociation constant (<i>K</i><sub>d</sub>) of 5.52 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). <b>MDL-800</b> showed a <i>K</i><sub>d</sub> of 9.51 μM in the same assay (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Figure S3</a>).</div></div><div id="sec2_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19">2.3.4.  HPLC Assay</h4><div class="NLM_p last">The high-performance liquid chromatography (HPLC) assay was applied to test the SIRT6-dependant deacetylase activity of <b>12q</b>. Here, both peptide substrates Ac-RYQK(Ac)-AMC and Ac-EALPKK(Myr)-AMC were used. The results showed that <b>12q</b> could enhance the SIRT6-dependant deacetylation and demyristoylation activities (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Figure S4</a>).</div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20">2.4.  Selectivity of <b>12q</b></h3><div class="NLM_p">To test the selectivity of <b>12q</b>, we measured the half maximal inhibitory concentration (IC<sub>50</sub>) values of <b>12q</b> against other HDAC family members, including SIRT1-3, SIRT5, and HDAC1-11; SIRT4 and SIRT7 are not involved because they are not available for us at present. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, <b>12q</b> potently activated SIRT6 but showed no activity toward SIRT2, SIRT3, SIRT5, and HDAC1-11 at concentrations up to 200 μM. Although <b>12q</b> exhibited activity against SIRT1, the potency is almost 300-fold lower than that against SIRT6. We further tested the activity of <b>12q</b> against a panel of 415 recombinant human kinases, and <b>12q</b> showed no activity or very weak activity against these kinases (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Table S2</a>). These results indicated that <b>12q</b> is a selective activator of SIRT6 and is not a pan assay interference compound (PAIN).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Activities of <b>12q</b> against HDAC Family Members</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">target</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> μM</th><th class="colsep0 rowsep0" align="center">target</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT1</td><td class="colsep0 rowsep0" align="left">171.20 ± 14.98</td><td class="colsep0 rowsep0" align="left">HDAC4</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT2</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="left">HDAC5</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT3</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="left">HDAC6</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT5</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="left">HDAC7</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT6</td><td class="colsep0 rowsep0" align="left">0.58 ± 0.12 (EC<sub>1.5</sub>)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">HDAC8</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC1</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="left">HDAC9</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC2</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="left">HDAC10</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC3</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="left">HDAC11</td><td class="colsep0 rowsep0" align="left">>200</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">The activity of <b>12q</b> against SIRT6 was determined by the FDL assay with acetyl substrate peptide Ac-RYQK(Ac)-AMC.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22">2.5.  Predicted Binding Mode of <b>12q</b></h3><div class="NLM_p">Because we failed to obtain a crystal structure of the <b>12q</b>–SIRT6 complex, molecular docking was used again to predict the binding mode of <b>12q</b> and SIRT6. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> presents a predicted binding mode of <b>12q</b> with SIRT6. <b>12q</b> occupies the allosteric pocket, which is very similar to that of <b>UBCS039</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and <b>MDL-801</b><a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (also see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Figure S5</a>, a superposition of three complex structures). The oxygen atom of the carboxamide of <b>12q</b> forms one hydrogen bond with the amide group in the side chain of ASN4. The <i>N</i>-benzhydryl group of <b>12q</b> fits perfectly in the allosteric pocket <i>via</i> hydrophobic interactions with residues TYR5, VAL70, PHE82, PRO62, and PRO80. The benzofuran forms hydrophobic interactions with LYS160 and MET157. Two face-to-face π–π and one σ–π interactions are formed. One of the π–π interactions is between the benzene ring of the <i>N</i>-benzhydryl group and the benzene ring of residue PHE82, and the other π–π interaction is between the benzene ring of the quinoline and the benzene ring of residue PHE86. The σ–π interaction is between the benzene ring in the benzofuran group and the amino group in THR156 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). These interactions well explain the high potency of <b>12q</b>.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Predicted binding mode of <b>12q</b> with SIRT6 (the SIRT6 structure was taken from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Y2F">5Y2F</a>). <b>12q</b> (green), ADPR (apricot), and H3K9-Myr (magenta) are shown in the “zoomed-in” image.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23">2.6.  <i>In Vitro</i> Anti-PDAC Activities of Compound <b>12q</b></h3><div id="sec2_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24">2.6.1.  Antiviability and Antiproliferation Activities of <b>12q</b> against PDAC Cells</h4><div class="NLM_p">The MTT assay was used to measure the antiviability activity of <b>12q</b> against PDAC cells. Four human PDAC cell lines, PANC-1, BXPC-3, MIAPaCa-2, and AsPC-1, were chosen for this study. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, <b>12q</b> exhibited antiviability activity against all the tested cell lines in a dose-dependent manner, with IC<sub>50</sub> values of 4.13 ± 0.15, 8.27 ± 1.08, 7.10 ± 1.85, and 9.66 ± 1.13 μM, against PANC-1, BXPC-3, MIAPaCa-2, and AsPC-1, respectively. In this assay, <b>MDL-800</b> also showed activity but with relatively weak potency (IC<sub>50</sub> values of 35.19 ± 1.21, 57.96 ± 1.11, 55.65 ± 1.03, and 50.51 ± 1.01 μM for PANC-1, BXPC-3, MIAPaCa-2, and AsPC-1, respectively) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). To examine the tumor-specificity effect, we tested the antiviability activity of <b>12q</b> against eight cell lines of other tumor types, including two liver cancer cell lines (HEPG2 and BEL7402), one thyroid cancer cell line (OCUT-2C), one ovarian cancer cell line (SKOV-3), one colon cancer cell line (SW620), and three lung cancer cell lines (H526, H1975, and NCI-H446). Although <b>12q</b> displayed some activities against these tumor cell lines, the potencies are weaker compared with those of PDAC cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Table S3</a>), indicating a certain degree of selectivity for PDAC cells.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Antiproliferative activities of <b>12q</b> against pancreatic cancer cells <i>in vitro</i>. (A,B) Cells were treated with the indicated agents for 72 h and cell viability was measured by an MTT assay. Every experiment was carried out in triplicate. (C) Cells were incubated with various concentrations of the indicated agents for 18 days (BXPC-3) or 14 days (PANC-1). Then, the cells were stained with crystal violet and quantified. (D) Fluorescence microscopic appearance of EdU and Hoechst on BXPC-3 and PANC-1 cells after treatment with the indicated agents for 24 h. EdU-positive cells were quantified in 10 fields. Scale bars, 100 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The colony formation assay was performed to investigate the antiproliferation activity of <b>12q</b> against PDAC cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">S6A</a>, compound <b>12q</b> could inhibit the colony formation of PANC-1, BXPC-3, MIAPaCa-2, and AsPC-1 cells in a dose-dependent manner. The antiproliferation activity of <b>12q</b> was further confirmed by an EdU incorporation assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">S6B</a>).</div></div><div id="sec2_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25">2.6.2.  Cell Cycle Arrest and Apoptosis Induction Ability of <b>12q</b> against PDAC Cells</h4><div class="NLM_p last">Flow cytometry was adopted to detect cell cycle arrest and apoptotic cells upon treatment with <b>12q</b>. The results showed that compound <b>12q</b> caused a dose-dependent increase in the percentages of PANC-1 and BXPC-3 cells in the G2 phase (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Figure S7</a>), indicating that <b>12q</b> could cause PDAC cell arrest in the G2 phase. <b>12q</b> treatment also increased Annexin V<sup>+</sup> populations in a concentration-dependent manner, implying that <b>12q</b> could induce apoptosis in PANC-1 and BXPC-3 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Figure S8</a>).</div></div><div id="sec2_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26">2.6.3.  Antimigration Ability of <b>12q</b></h4><div class="NLM_p">A wound-healing assay was used to evaluate the antimigration ability of <b>12q</b><i>in vitro</i>. Four human PDAC cell lines, PANC-1, BXPC-3, MIAPaCa-2, and AsPC-1, were used for this experiment. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">S9</a>, <b>12q</b> significantly inhibited the migration of PDAC cells in a time- and dose-dependent manner, with migration inhibition rates reaching approximately 87.8, 70.0, 57.2, and 55.4% in the presence of 10 μmol/L <b>12q</b> at 24 h, respectively.</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Antimigration ability of <b>12q</b> and its bioactivities in intact cells. (A) Antimigration activity of <b>12q</b> against pancreatic cancer cells <i>in vitro</i>. BXPC-3 or PANC-1 cells were treated with various concentrations of the indicated agents. Scale bars, 100 μm. (B) Western blot analysis of SIRT6 and its substrates H3K9ac, H3K18ac, and H3K56ac and Lin28b, c-Myc, and IGF2BP3 protein expression in PANC-1 and BXPC-3 cells treated with the indicated doses of <b>12q</b> and <b>MDL-800</b> for 72 h. (C) CETSA was implemented in intact cells exposed to <b>12q</b> (25 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27">2.7.  Bioactivities of <b>12q</b> in Intact Cells</h3><div class="NLM_p">The cellular thermal shift assay (CETSA) was first used to investigate whether <b>12q</b> could bind to the SIRT6 protein in intact cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B, <b>12q</b> treatment efficiently protected SIRT6 protein from temperature-dependent degradation in PANC-1 and BXPC-3 cells, indicating that <b>12q</b> is able to bind to SIRT6. Western blot was used to examine the effect of <b>12q</b> on the acetylation status of H3K9ac, H3K18ac, and H3K56ac in intact cells; H3K9, H3K18, and H3K56 are the typical substrates of SIRT6. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C, <b>12q</b> decreased the levels of H3K9ac, H3K18ac, and H3K56ac in PANC-1 and BXPC-3 cells in a dose-dependent manner. <b>MDL-800</b> showed the same effect.</div><div class="NLM_p last">We further investigated the influence of <b>12q</b> on the Lin28b/let-7 signal pathway in PDAC cells. As indicated before, it has been reported that SIRT6 functional loss accelerates PDAC progression and metastasis <i>via</i> upregulation of Lin28b and c-Myc recruitment. As displayed in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C, <b>12q</b> treatment decreased the expression of Lin28b and IGF2BP3.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28">2.8.  <i>In Vivo</i> Effects of <b>12q</b> on Subcutaneous PANC-1 Tumor Xenograft</h3><div class="NLM_p">To evaluate the antitumor activity of <b>12q</b><i>in vivo</i>, we used the human pancreatic tumor xenograft model of PANC-1. The BALB/c female nude mice were orally administered 100 or 150 mg/kg/d of <b>12q</b> for 30 days. <b>MDL-800</b> (150 mg/kg/d) was used as a positive control. In all the treated group, tumor growth was slowed by <b>12q</b> in a dose-dependent manner, and a tumor inhibition rate of 90.25% at a dose of 150 mg/kg was observed (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B). No significant weight loss or toxicity was observed compared with the control group (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Figures S10 and S11</a>). <b>MDL-800</b> also showed considerable antitumor potency with a tumor inhibition rate of 78.03% at a dose of 150 mg/kg/d.</div><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Antitumor efficacy of <b>12q</b><i>in vivo</i>. (A) Mice-bearing PANC-1 tumor xenografts were treated with agents at the indicated dose or vehicle control alone over the designated treatment schedule once tumors reached the determined size (<i>n</i> = 6). Points, mean tumor volume; bars, SD. (B) Quantitative analysis of tumor volumes on the final study day. ***<i>P</i> < 0.001. (C) Tumor tissue from PANC-1 xenografts treated with vehicle control, <b>12q</b> (150 mg/kg), or reference agents at the dose mentioned above were evaluated by IHC. Scale bars, 50 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To elucidate the mechanism of <b>12q</b>-mediated antitumor efficacy <i>in vivo</i>, we conducted immunohistochemistry (IHC) analysis in the PANC-1 tumor model. As depicted in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C, <b>12q</b> potently inhibited the expression of Lin28b and c-Myc and led to a substantial decrease in tumor cell proliferation (Ki-67-positive cells) compared with the control group in the PANC-1 tumor model. The level of H3K9ac was lower in the <b>12q</b>-treated groups than in the vehicle group. These results revealed that <b>12q</b> may suppress PDAC xenograft tumor growth <i>in vivo</i> by activating the deacetylase activity of SIRT6.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29">2.9.  Preliminary PK Properties of <b>12q</b></h3><div class="NLM_p">The preliminary pharmacokinetic (PK) properties of compound <b>12q</b> were determined by oral administration to male Sprague-Dawley rats. The PK parameters of <b>12q</b> at a dose of 10 mg/kg are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. The area under the concentration–time curve (AUC<sub>(0–∞)</sub>) is 755.57 h·ng/mL. The maximum plasma concentration (<i>C</i><sub>max</sub>) is 98.45 ng/mL. The bioavailability (<i>F</i>) and half-life (<i>T</i><sub>1/2</sub>) are 4% and 7.52 h, respectively.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. <i>In Vitro</i> PK Profiles of Compound <b>12q</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">PK parameter</th><th class="colsep0 rowsep0" align="center">10 mg/kg p.o.<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="±">2 mg/kg i.v.<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL (L/h/kg)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">0.6 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>ss</sub> (L/kg)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">1112.8 ± 322.84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left">7.52 ± 1.44</td><td class="colsep0 rowsep0" align="char" char="±">9.06 ± 0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="left">2.00 ± 0.00</td><td class="colsep0 rowsep0" align="char" char="±">0.08 ± 0.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (ng/mL)</td><td class="colsep0 rowsep0" align="left">98.45 ± 3.62</td><td class="colsep0 rowsep0" align="char" char="±">5123.70 ± 905.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>(0–<i>t</i>)</sub> (h·ng/mL)</td><td class="colsep0 rowsep0" align="left">704.67 ± 80.47</td><td class="colsep0 rowsep0" align="char" char="±">3326.13 ± 476.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>(0–∞)</sub> (h·ng/mL)</td><td class="colsep0 rowsep0" align="left">755.57 ± 80.74</td><td class="colsep0 rowsep0" align="char" char="±">3381.49 ± 468.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left">4.24 ± 0.48</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Expressed as mean ± SD, <i>n</i> = 3.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">3.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68446" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68446" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this investigation, we identified a potent SIRT6 activator, 2-(1-benzofuran-2-yl)-<i>N</i>-(diphenylmethyl)quinoline-4-carboxamide (<b>12q</b>), through virtual screening, chemical synthesis, and SAR analyses. This compound showed EC<sub>1.5</sub> values of 0.58 ± 0.12 μM (EC<sub>50</sub> = 5.35 ± 0.69 μM) and 0.72 ± 0.25 μM (EC<sub>50</sub> = 8.91 ± 1.81 μM) against SIRT6 in the FDL assays with acetyl peptide and myristoyl peptide substrates, respectively. The bioactivity of <b>12q</b> was further validated by DSF, SPR, and ITC assays. In target selectivity assays, <b>12q</b> displayed excellent selectivity for SIRT6 against other tested SIRT family members and HDACs. In various <i>in vitro</i> assays including MTT, colony formation, flow cytometry, and wound-healing assays, <b>12q</b> exhibited considerable anti-PDAC activities. In the subcutaneous PANC-1 tumor xenograft model, <b>12q</b> showed potent anti-tumor activity. <b>12q</b> also exhibited fairly good PK properties. Overall, <b>12q</b> could be a promising lead compound for the treatment of pancreatic cancer and is worthy of further in-depth studies.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">4.  Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47711" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47711" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33">4.1.  Virtual Screening Based on Molecular Docking</h3><div class="NLM_p last">In this study, molecular docking was performed using GOLD 5.0.<a onclick="showRef(event, 'ref34 ref45'); return false;" href="javascript:void(0);" class="ref ref34 ref45">(34,45)</a> X-ray crystal structures of SIRT6 in the complex with <b>UBCS039</b> (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MF6">5MF6</a>) and <b>MDL-801</b> (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Y2F">5Y2F</a>) were used in the docking studies.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The Discovery Studio 3.1 (Accelrys, Inc. USA) software package was adopted to prepare the protein structure, including adding hydrogen atoms to the protein, removing water molecules, and assigning the force field (here, the CHARMm force field was adopted). The binding site was defined as a sphere containing residues within 10 Å of the ligand, an area large enough to cover the ligand-binding region. Compound libraries, including Specs, ChemDiv, Selleck, MedChemExpress, and an in-house database, were filtered by SMARTS-PAINS and other bad groups as reported<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and used for the virtual screening in this investigation.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34">4.2.  Chemistry Methods</h3><div class="NLM_p">Starting materials, reagents, and solvents used in the experiments were purchased from commercial vendors unless otherwise noted, and they were analytically pure or chemically pure. Anhydrous solvents were either dried by distillation under reduced pressure or purchased from J&K Scientific. All reactions were monitored by thin-layer chromatography (TLC) carried out on Merck silica gel plates (0.25 mm thick, 60F254) and visualization was achieved by UV light. <sup>1</sup>H NMR (proton nuclear magnetic resonance) spectra, <sup>13</sup>C NMR (carbon-13 nuclear magnetic resonance) spectra, and high-resolution mass spectrometry/electrospray ionization–MS (HRMS/ESI–MS) were used for characterization. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded with Bruker AVANCE III 400 MHz spectrometers, and chemical shifts are reported in parts per million (δ, ppm) relative to the internal reference tetramethylsilane (Me<sub>4</sub>Si). NMR spectra were acquired in CDCl<sub>3</sub> or DMSO-<i>d</i><sub>6</sub>. <sup>1</sup>H NMR data are reported as follows: chemical shift [multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, dd = doublet of doublets, dt = doublet of triplets, m = multiplet, br s = broad singlet), <i>J</i> = coupling constant(s) (Hz), integration]. <sup>13</sup>C NMR data are reported as follows: chemical shift [multiplicity (if not singlet), assignment (Cq = fully substituted carbon)]. HRMS and low-resolution ESI–MS data were recorded on an Agilent 1200-G6410A mass spectrometer using an ESI source. All the final compounds were purified to >95% purity, as determined by HPLC. HPLC analyses were performed on a Waters e2695 HPLC system with a Symmetry C18 column (4.6 mm × 250 mm, 5 μm). The binary solvent system (A/B) was as follows: methanol (A) and water (B), A/B = 80/20. The flow rate was 1.0 mL/min. The yields of all reactions refer to the purified products. All final compounds were routinely characterized by melting points (mp), as determined on a Mel-Temp melting point apparatus, and the reported values are uncorrected.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35">4.2.1.  General Procedure for the Preparation of the 2-Substituted Quinoline-4-carboxylic Acid (<b>3</b>, <b>7a–m</b>)<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a></h4><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36">4.2.1.1.  Example: 2-(3,4-Dimethoxyphenyl)quinoline-4-carboxylic Acid (<b>3</b>)</h5><div class="NLM_p last">The synthetic pathway for the preparation of compound <b>3</b> is depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Indolin-2,3-dione (<b>1</b>, 1.47 g, 20.0 mmol), 3,4-dimethoxyacetophenone (<b>2</b>, 2.16 g, 24.0 mmol), and KOH (1.68 g, 60 mmol) in ethanol (35 mL) were added to a 100 mL round bottom flask with stirring. The resulting mixture was heated to reflux at 85 °C for 48 h. After the reaction was complete, the solvent was removed under reduced pressure, and the residue was dissolved in water and washed twice with ethyl acetate. The aqueous layer was concentrated and adjusted to pH = 2 with concentrated HCl. The solid was separated by filtration, washed with water until neutral, dried, and purified by flash column chromatography over silica gel eluting with ethyl acetate and petroleum ether to obtain 2-(3,4-dimethoxyphenyl)quinoline-4-carboxylic acid (<b>3</b>) as a yellow solid in 76% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.95 (s, 1H), 8.59 (dd, <i>J</i> = 8.5, 1.3 Hz, 1H), 8.43 (s, 1H), 8.17–8.09 (m, 1H), 7.91 (d, <i>J</i> = 2.1 Hz, 1H), 7.87 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 7.83 (ddd, <i>J</i> = 8.4, 6.8, 1.5 Hz, 1H), 7.66 (ddd, <i>J</i> = 8.3, 6.8, 1.3 Hz, 1H), 7.14 (d, <i>J</i> = 8.4 Hz, 1H), 3.92 (s, 3H), 3.86 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 168.2, 156.0, 151.2, 149.6, 148.8, 138.1, 131.0, 130.5, 130.0, 127.7, 125.8, 123.6, 120.8, 119.2, 112.2, 110.7, 56.1. MS <i>m</i>/<i>z</i> (ESI): 310.11 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37">4.2.2.  General Procedure for the Preparation of the 2-Substituted Quinoline-4-carboxamide Derivatives (<b>5a–o</b> and <b>8a–m</b>)<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a></h4><div id="sec4_2_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38">4.2.2.1.  Example: <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(3,4-dimethoxyphenyl)quinoline-4-carboxamide (<b>5h</b>)</h5><div class="NLM_p last">The synthetic pathways for the preparation of compounds <b>5a–o</b> and <b>8a–m</b> are depicted in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. In a 25 mL round bottom flask, intermediate compound <b>3</b> (155 mg, 0.5 mmol) was dissolved in POCl<sub>3</sub> (6 mL) and heated to 107 °C for 4–5 h.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The progress of the reaction was monitored by TLC, and the reaction solution concentrated by rotary evaporation after the completion of the reaction. The obtained acid chloride compound was dissolved in dry dimethylformamide (DMF), and 3-amino-4-chlorobenzotrifluoride (4, 117 mg, 0.6 mmol) and 4-dimethylaminopyridine (DMAP) (12.2 mg, 0.1 mmol) were added, and the reaction was allowed to stand overnight. After the completion of the reaction, the reaction mixture was diluted with water and extracted with ethyl acetate. The organic layer was washed three times with saturated aqueous NaCl and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. <i>N</i>-[2-chloro-5-(trifluoromethyl)phenyl]-2-(3,4-dimethoxyphenyl)quinolinone-4-carboxamide (<b>5h</b>), yield 79%. mp 209.7–210.6 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.84 (s, 1H), 8.41 (s, 1H), 8.30 (d, <i>J</i> = 2.2 Hz, 1H), 8.26 (d, <i>J</i> = 8.2 Hz, 1H), 8.15 (d, <i>J</i> = 8.4 Hz, 1H), 7.97 (d, <i>J</i> = 1.8 Hz, 1H), 7.94 (dd, <i>J</i> = 8.4, 2.5 Hz, 1H), 7.88–7.81 (m, 2H), 7.71 (dd, <i>J</i> = 7.6, 2.1 Hz, 1H),7.65 (t, <i>J</i> = 7.7 Hz, 1H), 7.17 (d, <i>J</i> = 8.5 Hz, 1H), 3.94 (s, 3H), 3.87 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.6, 155.9, 151.2, 149.6, 142.5, 136.1, 133.6, 131.5, 131.2, 130.7, 129.9, 128.8, 128.5, 127.5, 125.6, 125.1, 124.6, 123.4, 122.8, 120.9, 117.4, 112.2, 110.8, 56.2, 56.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>19</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 487.1031; found, 487.1027; calcd for C<sub>25</sub>H<sub>18</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 509.0850; found, 509.0859.</div></div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39">4.2.3.  General Procedure for the Preparation of the 2-Substituted Quinoline-4-carboxamide Derivatives (<b>12a–q</b>)</h4><div id="sec4_2_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40">4.2.3.1.  Example: 2-(1-Benzofuran-2-yl)-<i>N</i>-[4-(trifluoromethyl)phenyl]quinoline-4-carboxamide (<b>12g</b>)</h5><div class="NLM_p last">The synthetic pathway for the preparation of compounds (<b>12a–q</b>) is depicted in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Similar to compound <b>3</b>, 2-(1-benzofuran-2-yl)quinoline-4-carboxylic acid (<b>7j</b>) was obtained from the condensation of indolin-2,3-dione (<b>1</b>) and 1-(benzofuran-2-yl)ethan-1-one (<b>6j</b>). The reaction of compound <b>7j</b> (289 mg, 1 mmol) with the proper commercially available substituted amine [e.g., <b>11g</b>, 4-(trifluoromethyl)aniline, 161 mg, 1 mmol] was stirred overnight at 50 °C with HATU (380 mg, 1 mmol) and HOBt (135 mg, 1 mmol) as condensation agents and DMAP (12 mg, 0.1 mmol) as a base. This two-step reaction provides compound <b>12g</b> as a white solid in 35% yield. mp 294.8–296.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.26 (s, 1H), 8.42 (s, 1H), 8.19 (dd, <i>J</i> = 16.9, 8.4 Hz, 2H), 8.05 (d, <i>J</i> = 8.4 Hz, 2H), 7.99 (s, 1H), 7.90 (ddd, <i>J</i> = 8.4, 6.8, 1.5 Hz, 1H), 7.82–7.77 (m, 4H), 7.72 (ddd, <i>J</i> = 8.3, 6.9, 1.3 Hz, 1H), 7.49–7.44 (m, 1H), 7.36 (t, <i>J</i> = 7.5 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.9, 155.5, 154.6, 148.4, 143.1, 142.8, 131.3, 130.0, 128.7, 128.5, 126.7, 126.2, 125.6, 124.9, 124.6, 124.2, 123.8, 123.5, 122.7, 120.5, 116.9, 112.2, 108.0. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 433.1158; found, 433.1166; calcd for C<sub>25</sub>H<sub>15</sub>F<sub>3</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 455.0978; found, 455.0982.</div></div><div id="sec4_2_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41">4.2.3.2.  2-(3,4-Dimethoxyphenyl)-<i>N</i>-(4-ethoxyphenyl)quinoline-4-carboxamide (<b>5a</b>)</h5><div class="NLM_p last">White solid. Yield 60%. mp 169.3–170.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.66 (s, 1H), 8.32 (s, 1H), 8.16–8.10 (m, 2H), 7.97 (d, <i>J</i> = 6.9 Hz, 2H), 7.81 (ddd, <i>J</i> = 8.3, 6.9, 1.4 Hz, 1H), 7.77–7.71 (m, 2H), 7.62 (ddd, <i>J</i> = 8.3, 6.8, 1.2 Hz, 1H), 7.18–7.11 (m, 1H), 6.97 (d, <i>J</i> = 9.0 Hz, 2H), 4.03 (q, <i>J</i> = 6.9 Hz, 2H), 3.93 (s, 3H), 3.86 (s, 3H), 1.34 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.4, 156.0, 155.5, 151.1, 149.6, 148.3, 143.6, 132.4, 131.3, 130.6, 129.9, 127.3, 125.6, 123.5, 122.0, 116.8, 114.9, 112.2, 110.9, 63.6, 56.2, 56.1, 15.2. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>26</sub>H<sub>25</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 429.1809; found, 429.1813; calcd for C<sub>26</sub>H<sub>24</sub>N<sub>2</sub>O<sub>4</sub>Na [M + Na]<sup>+</sup>, 451.1628; found, 451.1633.</div></div><div id="sec4_2_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42">4.2.3.3.  2-(3,4-Dimethoxyphenyl)-<i>N</i>-(3-methoxyphenyl)quinoline-4-carboxamide (<b>5b</b>)</h5><div class="NLM_p last">White solid. Yield 65%. mp 171.6–172.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.79 (s, 1H), 8.33 (s, 1H), 8.12 (dd, <i>J</i> = 11.6, 8.5 Hz, 2H), 7.98–7.95 (m, 2H), 7.84–7.80 (m, 1H), 7.67–7.59 (m, 1H), 7.56–7.52 (m, 1H), 7.39 (d, <i>J</i> = 8.2 Hz, 1H), 7.31 (d, <i>J</i> = 6.5 Hz, 1H), 7.15 (d, <i>J</i> = 8.5 Hz, 1H), 6.76 (dd, <i>J</i> = 11.6, 6.5 Hz, 1H), 3.93 (s, 3H), 3.86 (s, 3H), 3.78 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.0, 160.0, 156.0, 151.2, 149.6, 148.3, 143.4, 140.5, 131.2, 130.6, 130.1, 129.9, 127.4, 125.5, 123.3, 120.9, 116.9, 112.7, 112.2, 110.9, 110.1, 106.2, 56.2, 56.1, 55.6. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>23</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 415.1652; found, 415.1654; calcd for C<sub>25</sub>H<sub>22</sub>N<sub>2</sub>O<sub>4</sub>Na [M + Na]<sup>+</sup>, 437.1472; found, 437.1482.</div></div><div id="sec4_2_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43">4.2.3.4.  2-(3,4-Dimethoxyphenyl)-<i>N</i>-[4-(trifluoromethyl)phenyl]quinoline-4-carboxamide (<b>5c</b>)</h5><div class="NLM_p last">White solid. Yield 67%. mp 211.3–212.6 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ11.17 (s, 1H), 8.40 (s, 1H), 8.19–8.09 (m, 2H), 8.05 (d, <i>J</i> = 8.5 Hz, 2H), 7.98 (dq, <i>J</i> = 4.5, 2.1 Hz, 2H), 7.88–7.76 (m, 3H), 7.64 (ddd, <i>J</i> = 8.2, 6.9, 1.3 Hz, 1H), 7.15 (d, <i>J</i> = 9.0 Hz, 1H), 3.92 (s, 3H), 3.86 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.4, 156.0, 151.2, 149.6, 148.3, 142.9, 131.1, 130.7, 129.9, 127.5, 126.7, 126.6, 125.4, 124.8, 123.2, 121.0, 120.4, 117.0, 112.2, 110.9, 56.2, 56.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>20</sub>F<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 453.1421; found, 453.1423; calcd for C<sub>25</sub>H<sub>20</sub>F<sub>3</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 475.1240; found, 475.1252.</div></div><div id="sec4_2_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44">4.2.3.5.  2-(3,4-Dimethoxyphenyl)-<i>N</i>-(3-fluorophenyl)quinoline-4-carboxamide (<b>5d</b>)</h5><div class="NLM_p last">White solid. Yield 63%. mp 203.9–205.6 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.02 (s, 1H), 8.37 (s, 1H), 8.17–8.08 (m, 2H), 8.01–7.95 (m, 2H), 7.86–7.79 (m, 2H), 7.64 (ddd, <i>J</i> = 8.2, 6.8, 1.3 Hz, 1H), 7.59–7.54 (m, 1H), 7.50–7.42 (m, 1H), 7.15 (d, <i>J</i> = 9.0 Hz, 1H), 7.01 (td, <i>J</i> = 8.4, 2.6 Hz, 1H), 3.93 (s, 3H), 3.86 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.2, 156.0, 151.2, 149.6, 148.3, 143.1, 131.2, 131.1, 131.0, 130.7, 129.9, 127.5, 125.4, 123.2, 121.0, 117.0, 116.2, 112.2, 110.9, 56.2, 56.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>24</sub>H<sub>20</sub>FN<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 403.1452; found, 403.1460; calcd for C<sub>24</sub>H<sub>19</sub>FN<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 425.1272; found, 525.1276.</div></div><div id="sec4_2_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45">4.2.3.6.  <i>N</i>-(4-<i>tert</i>-Butylphenyl)-2-(3,4-dimethoxyphenyl)quinoline-4-carboxamide (<b>5e</b>)</h5><div class="NLM_p last">White solid. Yield 61%. mp 228.4–229.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.74 (s, 1H), 8.31 (s, 1H), 8.17–8.08 (m, 2H), 7.96 (d, <i>J</i> = 7.3 Hz, 2H), 7.82 (ddd, <i>J</i> = 8.4, 6.9, 1.4 Hz, 1H), 7.75 (d, <i>J</i> = 8.7 Hz, 1H), 7.62 (ddd, <i>J</i> = 8.1, 6.8, 1.3 Hz, 1H), 7.42 (d, <i>J</i> = 8.7 Hz, 2H), 7.14 (d, <i>J</i> = 8.9 Hz, 1H), 3.92 (s, 3H), 3.86 (s, 3H), 1.30 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.7, 156.0, 151.1, 149.6, 148.3, 147.0, 143.5, 136.8, 131.3, 130.6, 129.9, 127.3, 125.9, 125.5, 123.4, 120.9, 120.2, 116.9, 112.2, 110.9, 56.2, 56.1, 34.6, 31.7. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>28</sub>H<sub>29</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 441.2173; found, 441.2180; calcd for C<sub>28</sub>H<sub>28</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 463.1992; found, 463.1995.</div></div><div id="sec4_2_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46">4.2.3.7.  <i>N</i>-[2-Chloro-4-(trifluoromethyl)phenyl]-2-(3,4-dimethoxyphenyl)quinoline-4-carboxamide (<b>5f</b>)</h5><div class="NLM_p last">White solid. Yield 70%. mp 186.8–187.5 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.83 (s, 1H), 8.38 (s, 1H), 8.22 (dd, <i>J</i> = 8.4, 1.3 Hz, 1H), 8.15 (d, <i>J</i> = 8.4 Hz, 2H), 8.04 (d, <i>J</i> = 2.1 Hz, 1H), 7.99–7.92 (m, 2H), 7.90–7.81 (m, 2H), 7.66 (ddd, <i>J</i> = 8.2, 6.9, 1.3 Hz, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 1H), 3.93 (s, 3H), 3.87 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.5, 156.0, 151.2, 149.6, 148.3, 142.3, 138.9, 131.2, 130.7, 129.9, 129.3, 128.5, 127.5, 127.3, 125.4, 125.2, 123.4, 120.9, 117.4, 112.3, 110.9, 56.2, 56.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>19</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 487.1031; found, 487.1037; calcd for C<sub>25</sub>H<sub>18</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 509.0850; found, 509.0862.</div></div><div id="sec4_2_3_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47">4.2.3.8.  <i>N</i>-[3,5-Bis(trifluoromethyl)phenyl]-2-(3,4-dimethoxyphenyl)quinoline-4-carboxamide (<b>5g</b>)</h5><div class="NLM_p last">Off-white solid (yield 69%). mp 215.1–216.5 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.44 (s, 1H), 8.59–8.50 (m, 2H), 8.46 (s, 1H), 8.18 (dd, <i>J</i> = 15.6, 8.4 Hz, 2H), 7.98–7.95 (m, 2H), 7.91 (s, 1H), 7.88–7.81 (m, 1H), 7.65 (t, <i>J</i> = 7.6 Hz, 1H), 7.16 (d, <i>J</i> = 8.5 Hz, 1H), 3.93 (s, 3H), 3.87 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.7, 156.0, 151.3, 149.6, 148.4, 142.2, 141.2, 131.5, 131.2, 131.1, 130.8, 129.9, 127.6, 125.5, 125.1, 123.0, 122.3, 121.0, 120.3, 117.5, 117.2, 112.2, 110.9, 56.2, 56.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>26</sub>H<sub>19</sub>F<sub>6</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 521.1294; found, 521.1286; calcd for C<sub>26</sub>H<sub>18</sub>F<sub>6</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 543.1114; found, 543.1125.</div></div><div id="sec4_2_3_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48">4.2.3.9.  2-(3,4-Dimethoxyphenyl)-<i>N</i>-(4-fluoro-2-nitrophenyl)quinoline-4-carboxamide (<b>5i</b>)</h5><div class="NLM_p last">Alight yellow solid. Yield 45%. mp 203.9–205.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.23 (s, 1H), 8.35 (s, 1H), 8.23 (dd, <i>J</i> = 8.4, 1.4 Hz, 1H), 8.19–8.13 (m, 1H), 8.04 (dd, <i>J</i> = 8.5, 2.9 Hz, 1H), 7.97 (d, <i>J</i> = 2.1 Hz, 1H), 7.92 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 7.88–7.72 (m, 3H), 7.67 (ddd, <i>J</i> = 8.2, 6.8, 1.3 Hz, 1H), 7.18 (d, <i>J</i> = 8.5 Hz, 1H), 3.94 (s, 3H), 3.87 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.1, 160.3, 157.9, 155.9, 151.3, 149.6, 148.4, 144.6, 141.9, 131.1, 130.9, 130.0, 127.6, 125.4, 123.3, 121.8, 121.6, 120.9, 117.2, 113.1, 112.3, 110.8, 56.2, 56.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>24</sub>H<sub>19</sub>FN<sub>3</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 448.1303; found, 448.1309; calcd for C<sub>24</sub>H<sub>18</sub>FN<sub>3</sub>O<sub>5</sub>Na [M + Na]<sup>+</sup>, 470.1123; found, 470.1134.</div></div><div id="sec4_2_3_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49">4.2.3.10.  2-(3,4-Dimethoxyphenyl)-<i>N</i>-(2,4,6-trimethylphenyl)quinoline-4-carboxamide (<b>5j</b>)</h5><div class="NLM_p last">Off-white solid. Yield 59%. mp 222.8–223.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.12 (s, 1H), 8.26 (d, <i>J</i> = 6.7 Hz, 2H), 8.15 (d, <i>J</i> = 8.4 Hz, 1H), 7.97 (s, 1H), 7.91 (d, <i>J</i> = 8.4 Hz, 1H), 7.83 (t, <i>J</i> = 7.7 Hz, 1H), 7.66 (t, <i>J</i> = 7.7 Hz, 1H), 7.18 (d, <i>J</i> = 8.4 Hz, 1H), 6.99 (s, 2H), 3.93 (s, 3H), 3.87 (s, 3H), 2.33 (s, 6H), 2.28 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.9, 156.1, 151.2, 149.6, 148.4, 143.3, 136.5, 135.6, 132.3, 131.4, 130.7, 129.9, 129.0, 127.4, 125.6, 123.7, 121.0, 117.0, 112.3, 110.9, 56.2, 56.1, 21.0, 18.7. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>27</sub>H<sub>27</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 427.2016; found, 427.2021; calcd for C<sub>27</sub>H<sub>26</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 449.1836; found, 449.1845.</div></div><div id="sec4_2_3_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50">4.2.3.11.  <i>N</i>-(2,4-Di-<i>tert</i>-butyl-5-hydroxyphenyl)-2-(3,4-dimethoxyphenyl)quinoline-4-carboxamide (<b>5k</b>)</h5><div class="NLM_p last">White solid. Yield 58%. mp 254.1–255.5 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.08 (s, 1H), 9.33 (s, 1H), 8.27 (dd, <i>J</i> = 8.4, 1.4 Hz, 1H), 8.16 (d, <i>J</i> = 8.1 Hz, 2H), 7.92 (d, <i>J</i> = 2.1 Hz, 1H), 7.88–7.81 (m, 2H), 7.67 (ddd, <i>J</i> = 8.3, 6.8, 1.3 Hz, 1H), 7.25 (s, 1H), 7.20 (d, <i>J</i> = 8.5 Hz, 1H), 6.79 (s, 1H), 3.93 (s, 3H), 3.87 (s, 3H), 1.40 (br s, 18H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.9, 155.9, 154.3, 151.2, 149.6, 148.5, 143.1, 136.7, 134.4, 133.7, 131.4, 130.7, 130.0, 127.3, 125.5, 125.0, 123.6, 120.7, 119.4, 116.7, 112.4, 110.7, 56.1, 34.9, 31.7, 29.8. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>32</sub>H<sub>37</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 513.2748; found, 513.2754; calcd for C<sub>32</sub>H<sub>36</sub>N<sub>2</sub>O<sub>4</sub>Na [M + Na]<sup>+</sup>, 535.2567; found, 535.2565.</div></div><div id="sec4_2_3_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51">4.2.3.12.  2-(3,4-Dimethoxyphenyl)-<i>N</i>-(naphthalene-1-yl)quinoline-4-carboxamide (<b>5l</b>)</h5><div class="NLM_p last">Off-white solid. Yield 73%. mp 226.9–228.6 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.87 (s, 1H), 8.50 (s, 1H), 8.26 (d, <i>J</i> = 8.2 Hz, 1H), 8.22–8.16 (m, 2H), 8.02–8.01 (m, 3H), 7.93 (dd, <i>J</i> = 12.5, 7.8 Hz, 2H), 7.84 (t, <i>J</i> = 7.5 Hz, 1H), 7.69–7.56 (m, 4H), 7.17 (d, <i>J</i> = 7.1 Hz, 1H), 3.95 (s, 3H), 3.87 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.9, 153.1, 151.2, 149.6, 148.4, 143.4, 134.3, 133.5, 131.4, 130.6, 129.9, 128.9, 128.6, 127.4, 126.8, 126.7, 126.6, 126.1, 125.6, 123.7, 123.6, 121.1, 117.2, 112.2, 111.0, 56.2, 56.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>28</sub>H<sub>23</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 435.1703; found, 435.1714; calcd for C<sub>28</sub>H<sub>22</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 457.1523; found, 457.1532.</div></div><div id="sec4_2_3_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52">4.2.3.13.  2-(3,4-Dimethoxyphenyl)-<i>N</i>-(naphthalene-2-yl)quinoline-4-carboxamide (<b>5m</b>)</h5><div class="NLM_p last">Off-white solid. Yield 71%. mp 215.9–216.4 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.04 (s, 1H), 8.60 (d, <i>J</i> = 2.1 Hz, 1H), 8.41 (s, 1H), 8.17 (t, <i>J</i> = 8.3 Hz, 2H), 8.00–7.89 (m, 5H), 7.85–7.80 (m, 2H), 7.64 (t, <i>J</i> = 8.2 Hz, 1H), 7.53 (t, <i>J</i> = 7.0 Hz, 1H), 7.46 (t, <i>J</i> = 7.1 Hz, 1H), 7.15 (d, <i>J</i> = 8.8 Hz, 1H), 3.93 (s, 3H), 3.86 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.2, 156.1, 151.2, 149.6, 148.3, 143.4, 137.0, 133.8, 131.3, 130.7, 130.0, 128.9, 128.0, 127.4, 127.0, 125.5, 123.4, 121.0, 120.9, 117.0, 116.8, 112.2, 110.9, 56.2, 56.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>28</sub>H<sub>23</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 435.1703; found, 435.1718; calcd for C<sub>28</sub>H<sub>22</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 457.1523; found, 457.1548.</div></div><div id="sec4_2_3_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53">4.2.3.14.  Methyl (2<i>S</i>,3<i>S</i>)-3-[2-(3,4-Dimethoxyphenyl)quinoline-4-amido]bicycle[2.2.2]octane-2-carboxylate (<b>5n</b>)</h5><div class="NLM_p last">White solid. Yield 68%. mp 185.9–187.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.85 (d, <i>J</i> = 7.0 Hz, 1H), 8.09 (d, <i>J</i> = 8.4 Hz, 1H), 8.06 (s, 1H), 8.03 (d, <i>J</i> = 7.6 Hz, 1H), 7.93 (d, <i>J</i> = 2.0 Hz, 1H), 7.88 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.81–7.76 (m, 1H), 7.62–7.58 (m, 1H), 7.14 (d, <i>J</i> = 8.5 Hz, 1H), 4.50 (t, <i>J</i> = 6.7 Hz, 1H), 3.91 (s, 3H), 3.85 (s, 3H), 3.68 (s, 3H),2.66 (d, <i>J</i> = 6.5 Hz, 1H), 1.93–1.90 (m, 2H), 1.82–1.75 (m, 1H), 1.67–1.57 (m, 3H), 1.54–1.39 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 174.7, 166.9, 155.9, 151.1, 149.5, 148.2, 143.9, 131.3, 130.5, 129.8, 127.1, 125.5, 123.6, 120.9, 116.6, 112.2, 110.8, 53.2, 53.1, 52.2, 50.2, 48.5, 29.5, 28.5, 25.7, 24.3, 21.2, 19.5. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>28</sub>H<sub>31</sub>N<sub>2</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 475.2227; found, 475.2240; calcd for C<sub>28</sub>H<sub>30</sub>N<sub>2</sub>O<sub>5</sub>Na [M + Na]<sup>+</sup>, 497.2047; found, 497.2059.</div></div><div id="sec4_2_3_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54">4.2.3.15.  2-(3,4-Dimethoxyphenyl)-<i>N</i>-[(1<i>r</i>,3<i>s</i>,5<i>R</i>,7<i>S</i>)-3-hydroxyadamantan-1-yl]quinoline-4-carboxamide(<b>5o</b>)</h5><div class="NLM_p last">White solid. Yield 66%. mp 167.3–168.5 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.34 (s, 1H), 8.08 (d, <i>J</i> = 8.4 Hz, 1H), 8.01 (d, <i>J</i> = 8.7 Hz, 1H), 7.99 (s, 1H), 7.92 (d, <i>J</i> = 1.5 Hz, 1H), 7.88 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 7.77 (t, <i>J</i> = 7.6 Hz, 1H), 7.60 (t, <i>J</i> = 6.8 Hz, 1H), 7.12 (d, <i>J</i> = 8.5 Hz, 1H), 4.56 (s, 1H), 3.91 (s, 3H), 3.85 (s, 3H), 2.21 (br s, 2H), 2.06–1.98 (m, 6H), 1.65–1.48 (m, 6H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.8, 156.0, 151.1, 149.5, 148.2, 144.7, 131.4, 129.8, 127.0, 125.5, 123.7, 120.9, 116.1, 112.1, 110.9, 67.8, 56.2, 56.1, 55.1, 49.4, 44.7, 40.7, 35.4, 30.6. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>28</sub>H<sub>31</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 459.2278; found, 459.2295; calcd for C<sub>28</sub>H<sub>30</sub>N<sub>2</sub>O<sub>4</sub>Na [M + Na]<sup>+</sup>, 481.2098; found, 481.2123.</div></div><div id="sec4_2_3_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55">4.2.3.16.  2-(3-Fluoro-4-methoxyphenyl)quinoline-4-carboxylic Acid (<b>7a</b>)</h5><div class="NLM_p last">White solid. Yield 67%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.98 (s, 1H), 8.61 (dd, <i>J</i> = 8.6, 1.3 Hz, 1H), 8.45 (s, 1H), 8.23–8.11 (m, 3H), 7.84 (ddd, <i>J</i> = 8.4, 6.9, 1.4 Hz, 1H), 7.69 (ddd, <i>J</i> = 8.3, 6.8, 1.3 Hz, 1H), 7.34 (t, <i>J</i> = 8.7 Hz, 1H), 3.95 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 168.1, 154.8, 153.5, 151.1, 148.7, 138.2, 131.4, 130.7, 130.1, 128.1, 125.8, 124.3, 123.7, 119.1, 114.4, 56.6. MS <i>m</i>/<i>z</i> (ESI): 298.08 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56">4.2.3.17.  2-(4-Methoxyphenyl)quinoline-4-carboxylic Acid (<b>7b</b>)</h5><div class="NLM_p last">White solid. Yield 69%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.94 (s, 1H), 8.62 (d, <i>J</i> = 8.5 Hz, 1H), 8.42 (s, 1H), 8.28 (d, <i>J</i> = 8.4 Hz, 2H), 8.13 (d, <i>J</i> = 8.4 Hz, 1H), 7.83 (t, <i>J</i> = 7.7 Hz, 1H), 7.67 (t, <i>J</i> = 7.7 Hz, 1H), 7.13 (d, <i>J</i> = 8.5 Hz, 2H), 3.86 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 168.2, 161.4, 155.9, 148.9, 137.9, 130.8, 130.6, 130.0, 129.2, 127.7, 125.8, 123.6, 119.1, 114.8, 55.8. MS <i>m</i>/<i>z</i> (ESI): 280.10 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57">4.2.3.18.  2-(Pyridin-3-yl)quinoline-4-carboxylic Acid (<b>7c</b>)</h5><div class="NLM_p last">Colorless oily. Yield 43%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.47 (d, <i>J</i> = 2.3 Hz, 1H), 8.73 (dd, <i>J</i> = 4.8, 1.6 Hz, 1H), 8.66 (dt, <i>J</i> = 7.9, 1.8 Hz, 2H), 8.52 (s, 1H), 8.21 (dd, <i>J</i> = 8.5, 1.2 Hz, 1H), 7.89 (ddd, <i>J</i> = 8.4, 6.8, 1.4 Hz, 1H), 7.75 (ddd, <i>J</i> = 8.4, 6.8, 1.3 Hz, 1H), 7.61 (ddd, <i>J</i> = 8.0, 4.8, 0.9 Hz, 1H). MS <i>m</i>/<i>z</i> (ESI): 251.08 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58">4.2.3.19.  2-Cyclohexylquinoline-4-carboxylic Acid (<b>7d</b>)</h5><div class="NLM_p last">Colorless oily. Yield 55%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.84 (s, 1H), 8.63 (dd, <i>J</i> = 8.5, 1.4 Hz, 1H), 8.03 (dd, <i>J</i> = 8.5, 1.2 Hz, 1H), 7.84 (s, 1H), 7.78 (ddd, <i>J</i> = 8.4, 6.8, 1.5 Hz, 1H), 7.64 (ddd, <i>J</i> = 8.4, 6.9, 1.4 Hz, 1H), 2.94 (tt, <i>J</i> = 11.8, 3.4 Hz, 1H), 1.99–1.90 (m, 2H), 1.84 (dt, <i>J</i> = 12.7, 3.4 Hz, 2H), 1.78–1.70 (m, 1H), 1.64 (qd, <i>J</i> = 12.5, 3.2 Hz, 2H), 1.43 (qt, <i>J</i> = 12.4, 3.2 Hz, 2H), 1.31 (tt, <i>J</i> = 12.6, 3.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 168.2, 166.3, 148.5, 137.1, 130.0, 129.7, 127.4, 125.8, 123.6, 121.3, 46.6, 32.5, 26.4, 26.0. MS <i>m</i>/<i>z</i> (ESI): 256.13 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59">4.2.3.20.  2-Cyclopentylquinoline-4-carboxylic Acid (<b>7e</b>)</h5><div class="NLM_p last">Colorless oily. Yield 61%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.79 (s, 1H), 8.61 (dd, <i>J</i> = 8.5, 1.3 Hz, 1H), 8.02 (dd, <i>J</i> = 8.5, 1.2 Hz, 1H), 7.84 (s, 1H), 7.78 (ddd, <i>J</i> = 8.3, 6.8, 1.4 Hz, 1H), 7.64 (ddd, <i>J</i> = 8.3, 6.8, 1.3 Hz, 1H), 3.44 (p, <i>J</i> = 8.2 Hz, 1H), 2.17–2.03 (m, 2H), 1.95–1.77 (m, 4H), 1.70 (tt, <i>J</i> = 6.8, 3.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 168.2, 165.8, 148.4, 136.9, 130.1, 129.6, 127.4, 125.8, 123.6, 121.9, 48.0, 33.2, 26.0. MS <i>m</i>/<i>z</i> (ESI): 242.12 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60">4.2.3.21.  2-(Thiophen-2-yl)quinoline-4-carboxylic Acid (<b>7f</b>)</h5><div class="NLM_p last">The specific synthesis method was the same as that of the compound <b>3</b>, and the reaction was carried out to give a pale-yellow solid compound <b>7a</b>. Yield 70%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 14.03 (s, 1H), 8.58 (dd, <i>J</i> = 8.7, 1.3 Hz, 1H), 8.42 (s, 1H), 8.10 (dd, <i>J</i> = 3.7, 1.2 Hz, 1H), 8.05 (dt, <i>J</i> = 8.2, 1.0 Hz, 1H), 7.82 (ddd, <i>J</i> = 8.4, 6.9, 1.4 Hz, 1H), 7.79 (dd, <i>J</i> = 5.0, 1.1 Hz, 1H), 7.66 (ddd, <i>J</i> = 8.4, 6.9, 1.3 Hz, 1H), 7.25 (dd, <i>J</i> = 5.0, 3.7 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 167.9, 152.1, 148.6, 144.4, 138.2, 130.9, 130.7, 129.5, 129.2, 128.2, 127.9, 126.0, 123.8, 118.5. MS <i>m</i>/<i>z</i> (ESI): 256.05 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61">4.2.3.22.  2-(5-Methylthiophen-2-yl)quinoline-4-carboxylic Acid (<b>7g</b>)</h5><div class="NLM_p last">Pale-yellow solid. Yield 75%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.92 (s, 1H), 8.56 (dd, <i>J</i> = 8.5, 1.3 Hz, 1H), 8.34 (s, 1H), 8.00 (dd, <i>J</i> = 8.5, 1.2 Hz, 1H), 7.88 (d, <i>J</i> = 3.7 Hz, 1H), 7.80 (ddd, <i>J</i> = 8.4, 6.9, 1.4 Hz, 1H), 7.64 (ddd, <i>J</i> = 8.3, 6.9, 1.3 Hz, 1H), 6.94 (dd, <i>J</i> = 3.6, 1.2 Hz, 1H), 2.53 (br s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 167.9, 152.2, 148.6, 144.6, 142.0, 137.9, 130.8, 129.4, 128.4, 127.7, 127.6, 125.9, 123.6, 118.2, 15.9. MS <i>m</i>/<i>z</i> (ESI): 270.06 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62">4.2.3.23.  2-(Furan-2-yl)quinoline-4-carboxylic Acid (<b>7h</b>)</h5><div class="NLM_p last">Pale-yellow solid. Yield 70%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 14.02 (s, 1H), 8.65 (dd, <i>J</i> = 8.6, 1.3 Hz, 1H), 8.31 (s, 1H), 8.09 (dd, <i>J</i> = 8.5, 1.2 Hz, 1H), 8.00–7.97 (m, 1H), 7.84 (ddd, <i>J</i> = 8.4, 6.8, 1.4 Hz, 1H), 7.68 (ddd, <i>J</i> = 8.4, 6.9, 1.3 Hz, 1H), 7.46 (dd, <i>J</i> = 3.4, 0.7 Hz, 1H), 6.76 (dd, <i>J</i> = 3.5, 1.7 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 167.8, 152.9, 148.8, 148.5, 145.9, 137.7, 130.9, 129.8, 128.1, 126.0, 123.8, 118.4, 113.2, 111.8. MS <i>m</i>/<i>z</i> (ESI): 240.01 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63">4.2.3.24.  2-(5-Methylfuran-2-yl)quinoline-4-carboxylic Acid (<b>7i</b>)</h5><div class="NLM_p last">Pale-yellow solid. Yield 66%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.86 (s, 1H), 8.27 (d, <i>J</i> = 2.2 Hz, 1H), 8.20 (d, <i>J</i> = 8.3 Hz, 1H), 8.16 (s, 1H), 8.08 (d, <i>J</i> = 8.4 Hz, 1H), 7.91–7.79 (m, 2H), 7.73 (dd, <i>J</i> = 8.5, 2.3 Hz, 1H), 7.64 (dd, <i>J</i> = 8.4, 6.9 Hz, 1H), 7.36 (d, <i>J</i> = 3.3 Hz, 1H), 6.41 (d, <i>J</i> = 3.2 Hz, 1H), 2.46 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 167.8, 155.2, 151.4, 148.9, 148.5, 137.6, 130.7, 129.6, 127.7, 126.0, 123.6, 118.3, 113.1, 109.6, 14.1. MS <i>m</i>/<i>z</i> (ESI): 254.08 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64">4.2.3.25.  2-(1-Benzofuran-2-yl)quinoline-4-carboxylic Acid (<b>7j</b>)</h5><div class="NLM_p last">Yellow solid. Yield 69%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 14.10 (s, 1H), 8.71 (dd, <i>J</i> = 8.6, 1.3 Hz, 1H), 8.51 (s, 1H), 8.19 (dd, <i>J</i> = 8.5, 1.1 Hz, 1H), 7.94 (d, <i>J</i> = 0.9 Hz, 1H), 7.89 (ddd, <i>J</i> = 8.4, 6.8, 1.5 Hz, 1H), 7.83–7.77 (m, 2H), 7.74 (ddd, <i>J</i> = 8.3, 6.9, 1.3 Hz, 1H), 7.46 (ddd, <i>J</i> = 8.5, 7.2, 1.3 Hz, 1H), 7.35 (td, <i>J</i> = 7.5, 0.9 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 167.7, 155.5, 154.5, 148.9, 148.4, 138.1, 131.0, 130.0, 128.6, 126.6, 126.1, 124.3, 124.1, 122.7, 119.2, 112.2, 107.6. MS <i>m</i>/<i>z</i> (ESI): 290.08 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65">4.2.3.26.  2-(7-Methoxy-1-benzofuran-2-yl)quinoline-4-carboxylic Acid (<b>7k</b>)</h5><div class="NLM_p last">White solid. Yield 65%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.72 (dd, <i>J</i> = 8.6, 1.3 Hz, 1H), 8.48 (s, 1H), 8.19 (dd, <i>J</i> = 8.6, 1.2 Hz, 1H), 7.90 (s, 1H), 7.89 (ddd, <i>J</i> = 8.4, 6.8, 1.4 Hz, 1H), 7.74 (ddd, <i>J</i> = 8.4, 6.9, 1.3 Hz, 1H), 7.35 (dd, <i>J</i> = 7.8, 1.0 Hz, 1H), 7.27 (t, <i>J</i> = 7.8 Hz, 1H), 7.07 (dd, <i>J</i> = 7.9, 1.0 Hz, 1H), 4.03 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 167.7, 154.4, 149.0, 148.4, 145.7, 144.7, 138.0, 131.0, 130.3, 130.0, 128.7, 126.1, 124.9, 124.3, 119.1, 114.5, 108.6, 107.8, 56.3. MS <i>m</i>/<i>z</i> (ESI): 320.09 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66">4.2.3.27.  2-(2<i>H</i>-1,3-Benzodioxol-5-yl)quinoline-4-carboxylic Acid (<b>7l</b>)</h5><div class="NLM_p last">White solid. Yield 72%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.60 (dd, <i>J</i> = 8.5, 1.3 Hz, 1H), 8.39 (s, 1H), 8.12 (dd, <i>J</i> = 8.5, 1.2 Hz, 1H), 7.90–7.79 (m, 3H), 7.67 (ddd, <i>J</i> = 8.3, 6.8, 1.4 Hz, 1H), 7.10 (d, <i>J</i> = 8.1 Hz, 1H), 6.14 (s, 2H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 168.2, 155.7, 149.5, 148.7, 138.3, 132.7, 130.6, 130.1, 127.9, 125.8, 123.7, 122.3, 119.2, 109.1, 107.5, 102.0. MS <i>m</i>/<i>z</i> (ESI): 294.05 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67">4.2.3.28.  2-(2,3-Dihydro-1,4-benzodioxin-6-yl)quinoline-4-carboxylic Acid (<b>7m</b>)</h5><div class="NLM_p last">White solid. Yield 69%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.61 (dd, <i>J</i> = 8.5, 1.3 Hz, 1H), 8.37 (s, 1H), 8.12 (dd, <i>J</i> = 8.5, 1.2 Hz, 1H), 7.88–7.76 (m, 3H), 7.67 (ddd, <i>J</i> = 8.4, 6.9, 1.3 Hz, 1H), 7.04 (d, <i>J</i> = 8.3 Hz, 1H), 4.34 (br s, 4H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 168.1, 155.6, 148.8, 145.8, 144.3, 138.1, 131.7, 130.6, 130.1, 127.8, 125.8, 123.6, 120.9, 119.1, 118.0, 116.2, 64.9, 64.6. MS <i>m</i>/<i>z</i> (ESI): 308.09 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68">4.2.3.29.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(3-fluoro-4-methoxyphenyl)quinoline-4-carboxamide (<b>8a</b>)</h5><div class="NLM_p last">White solid. Yield 48%. mp 205.1–205.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.83 (s, 1H), 8.44 (s, 1H), 8.28–8.15 (m, 5H), 7.89–7.83 (m, 2H), 7.73 (dd, <i>J</i> = 8.3, 1.3 Hz, 1H), 7.68 (t, <i>J</i> = 8.0 Hz, 1H), 7.39 (t, <i>J</i> = 8.6 Hz, 1H), 3.96 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.4, 154.7, 153.5, 151.1, 149.4, 148.2, 142.6, 135.9, 133.7, 131.5, 130.9, 130.0, 128.9, 128.6, 127.8, 125.6, 125.2, 124.8, 124.4, 123.5, 117.3, 115.0, 114.5, 56.7. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>24</sub>H<sub>16</sub>ClF<sub>4</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 475.0831; found, 475.0845; calcd for C<sub>24</sub>H<sub>15</sub>ClF<sub>4</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 497.0650; found, 497.0658.</div></div><div id="sec4_2_3_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69">4.2.3.30.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(4-methoxyphenyl)quinoline-4-carboxamide (<b>8b</b>)</h5><div class="NLM_p last">White solid. Yield 56%. mp 205.7–207.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.83 (s, 1H), 8.39 (s, 1H), 8.33 (d, <i>J</i> = 8.8 Hz, 2H), 8.28 (d, <i>J</i> = 1.2 Hz, 1H), 8.25 (d, <i>J</i> = 8.0 Hz, 1H), 8.14 (d, <i>J</i> = 8.3 Hz, 1H), 7.89–7.82 (m, 2H), 7.73 (dd, <i>J</i> = 8.4, 1.6 Hz, 1H), 7.68–7.64 (m, 1H), 7.16 (d, <i>J</i> = 8.9 Hz, 2H), 3.87 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.5, 161.4, 155.9, 148.4, 142.4, 135.9, 133.7, 131.5, 131.0, 130.8, 129.9, 129.3, 128.9, 128.6, 127.5, 125.5, 125.3, 124.8, 123.3, 117.3, 114.8, 55.8. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>24</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 457.0925; found, 457.0951; calcd for C<sub>24</sub>H<sub>16</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 479.0745; found, 479.0740.</div></div><div id="sec4_2_3_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70">4.2.3.31.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(pyridin-3-yl)quinoline-4-carboxamide (<b>8c</b>)</h5><div class="NLM_p last">White solid. Yield 29%. mp 251.3–252.4 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.87 (s, 1H), 9.53 (s, 1H), 8.75 (d, <i>J</i> = 3.8 Hz, 1H), 8.70 (d, <i>J</i> = 7.9 Hz, 1H), 8.54 (s, 1H), 8.31 (t, <i>J</i> = 6.5 Hz, 2H), 8.22 (d, <i>J</i> = 8.3 Hz, 1H), 7.92–7.87 (m, 2H), 7.74 (t, <i>J</i> = 7.5 Hz, 2H), 7.64 (q, <i>J</i> = 7.7, 4.8 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.3, 154.2, 151.2, 148.9, 148.4, 142.9, 135.9, 135.1, 134.1, 133.7, 131.5, 131.1, 130.2, 128.9, 128.6, 128.4, 125.7, 125.2, 124.8, 124.5, 123.9, 117.8. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>22</sub>H<sub>14</sub>ClF<sub>3</sub>N<sub>3</sub>O [M + H]<sup>+</sup>, 428.0772; found, 428.0764; calcd for C<sub>22</sub>H<sub>13</sub>ClF<sub>3</sub>N<sub>3</sub>ONa [M + Na]<sup>+</sup>, 450.0591; found, 450.0600.</div></div><div id="sec4_2_3_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71">4.2.3.32.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-cyclohexylquinoline-4-carboxamide (<b>8d</b>)</h5><div class="NLM_p last">Colorless oily semi-solid. Yield 25%. mp 126.6–128.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.74 (s, 1H), 8.23–8.21 (m, 2H), 8.04 (d, <i>J</i> = 8.4 Hz, 1H), 7.86 (d, <i>J</i> = 8.4 Hz, 1H), 7.81–7.70 (m, 3H), 7.63 (t, <i>J</i> = 7.30 Hz, 1H), 2.95 (t, <i>J</i> = 11.6 Hz, 1H), 1.99 (d, <i>J</i> = 11.9 Hz, 2H), 1.86 (d, <i>J</i> = 12.8 Hz, 2H), 1.77–1.65 (m, 3H), 1.49–1.40 (m, 2H), 1.34 (d, <i>J</i> = 12.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.6, 166.3, 148.0, 141.9, 135.9, 133.8, 131.4, 130.2, 129.5, 128.9, 128.6, 127.1, 125.5, 125.4, 125.3, 124.7, 123.3, 122.7, 119.1, 46.9, 32.5, 26.4, 26.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>23</sub>H<sub>21</sub>ClF<sub>3</sub>N<sub>2</sub>O [M + H]<sup>+</sup>, 433.1289; found, 433.1251; calcd for C<sub>23</sub>H<sub>20</sub>ClF<sub>3</sub>N<sub>2</sub>ONa [M + Na]<sup>+</sup>, 455.1108; found, 455.1088.</div></div><div id="sec4_2_3_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72">4.2.3.33.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-cyclopentylquinoline-4-carboxamide (<b>8e</b>)</h5><div class="NLM_p last">Colorless oily semi-solid. Yield 23%. mp 187.9–189.6 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.76 (s, 1H), 8.25 (s, 1H), 8.21 (d, <i>J</i> = 8.4 Hz, 1H), 8.14 (s, 1H), 8.08–8.01 (m, 1H), 7.90–7.85 (m, 1H), 7.79 (ddd, <i>J</i> = 8.3, 6.8, 1.4 Hz, 1H), 7.75–7.70 (m, 1H), 7.64 (ddd, <i>J</i> = 8.2, 6.9, 1.3 Hz, 1H), 3.01–2.88 (m, 2H), 2.70–2.55 (m, 3H), 2.11–1.91 (m, 3H), 0.84 (q, <i>J</i> = 5.3, 4.4 Hz, 1H). HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>22</sub>H<sub>19</sub>ClF<sub>3</sub>N<sub>2</sub>O [M + H]<sup>+</sup>, 419.1133; found, 419.0959; calcd for C<sub>22</sub>H<sub>18</sub>ClF<sub>3</sub>N<sub>2</sub>ONa [M + Na]<sup>+</sup>, 441.0952; found, 441.0790.</div></div><div id="sec4_2_3_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73">4.2.3.34.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(thiophen-2-yl)quinoline-4-carboxamide (<b>8f</b>)</h5><div class="NLM_p last">White solid. Yield 51%. mp 252.2–253.6 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.85 (s, 1H), 8.39 (s, 1H), 8.28 (d, <i>J</i> = 1.2 Hz, 1H), 8.21 (d, <i>J</i> = 6.9 Hz, 1H), 8.08 (dd, <i>J</i> = 11.5, 5.7 Hz, 2H), 7.89–7.79 (m, 3H), 7.73 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.68–7.64 (m, 1H), 7.28 (dd, <i>J</i> = 4.9, 3.7 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.3, 152.1, 148.1, 144.6, 142.7, 135.8, 133.5, 131.5, 131.1, 130.7, 129.4, 129.1, 128.9, 128.6, 128.2, 127.7, 125.7, 125.1, 124.7, 123.6, 116.6. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>21</sub>H<sub>13</sub>ClF<sub>3</sub>N<sub>2</sub>OS [M + H]<sup>+</sup>, 433.0384; found, 433.0391; calcd for C<sub>21</sub>H<sub>12</sub>ClF<sub>3</sub>N<sub>2</sub>OSNa [M + Na]<sup>+</sup>, 455.0203; found, 455.0217.</div></div><div id="sec4_2_3_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74">4.2.3.35.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(5-methylthiophen-2-yl)quinoline-4-carboxamide (<b>8g</b>)</h5><div class="NLM_p last">White solid. Yield 46%. mp 216.1–217.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.84 (s, 1H), 8.31 (s, 1H), 8.27 (s, 1H), 8.20 (d, <i>J</i> = 8.1 Hz, 1H), 8.02 (d, <i>J</i> = 8.4 Hz, 1H), 7.88 (t, <i>J</i> = 6.9 Hz, 2H), 7.83–7.78 (m, 1H), 7.72 (dd, <i>J</i> = 8.4, 1.5 Hz, 1H), 7.63 (t, <i>J</i> = 8.1 Hz, 1H), 6.97 (dd, <i>J</i> = 3.6, 1.0 Hz, 1H), 2.54 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.3, 152.2, 148.1, 144.6, 142.5, 142.2, 135.9, 131.5, 130.9, 129.3, 128.9, 128.5, 128.4, 127.6, 127.4, 125.7, 125.5, 125.0, 124.7, 123.4, 116.3, 15.9. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>22</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>OS [M + H]<sup>+</sup>, 447.0540; found, 447.0541; calcd for C<sub>22</sub>H<sub>14</sub>ClF<sub>3</sub>N<sub>2</sub>OSNa [M + Na]<sup>+</sup>, 469.0360; found, 469.0440.</div></div><div id="sec4_2_3_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75">4.2.3.36.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(furan-2-yl)quinoline-4-carboxamide (<b>8h</b>)</h5><div class="NLM_p last">White solid. Yield 49%. mp 189.9–190.5 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.87 (s, 1H), 8.29–8.23 (m, 3H), 8.10 (d, <i>J</i> = 8.3 Hz, 1H), 8.00 (d, <i>J</i> = 1.0 Hz, 1H), 7.88–7.83 (m, 2H), 7.73 (dd, <i>J</i> = 8.9, 2.0 Hz, 1H), 7.69–7.65 (m, 1H), 7.46 (d, <i>J</i> = 3.3 Hz, 1H), 6.79 (dd, <i>J</i> = 5.5, 1.8 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.2, 153.1, 148.5, 148.3, 145.9, 142.6, 135.8, 133.7, 131.5, 131.1, 129.6, 128.9, 128.6, 127.8, 125.7, 125.3, 124.8, 123.5, 116.2, 113.3, 111.8. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>21</sub>H<sub>13</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 417.0612; found, 417.0589; calcd for C<sub>21</sub>H<sub>12</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 439.0432; found, 439.0428.</div></div><div id="sec4_2_3_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76">4.2.3.37.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(5-methylfuran-2-yl)quinoline-4-carboxamide (<b>8i</b>)</h5><div class="NLM_p last">White solid. Yield 55%. mp 208.9–209.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.85 (s, 1H), 8.27 (s, 1H), 8.21 (d, <i>J</i> = 8.2 Hz, 1H), 8.16 (s, 1H), 8.08 (d, <i>J</i> = 8.4 Hz, 1H), 7.88–7.80 (m, 2H), 7.72 (d, <i>J</i> = 8.8 Hz, 1H),7.64 (t, <i>J</i> = 7.4 Hz, 1H), 7.36 (d, <i>J</i> = 3.2 Hz, 1H), 6.40 (d, <i>J</i> = 2.6 Hz, 1H), 2.45 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.3, 155.2, 151.6, 148.5, 148.3, 142.5, 135.8, 133.7, 131.5, 131.0, 129.5, 128.9, 128.6, 127.4, 125.7, 125.2, 124.8, 123.3, 116.1, 113.2, 109.6, 14.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>22</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 431.0769; found, 431.0791; calcd for C<sub>22</sub>H<sub>14</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 453.0588; found, 455.0627.</div></div><div id="sec4_2_3_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77">4.2.3.38.  2-(1-Benzofuran-2-yl)-<i>N</i>-[2-chloro-5-(trifluoromethyl)phenyl]quinoline-4-carboxamide (<b>8j</b>)</h5><div class="NLM_p last">White solid. Yield 55%. mp 245.1–246.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.94 (s, 1H), 8.46 (s, 1H), 8.32 (s, 1H), 8.29 (d, <i>J</i> = 8.4 Hz, 1H), 8.21 (d, <i>J</i> = 8.40 Hz, 1H), 7.95 (s, 1H), 7.93–7.88 (m, 2H), 7.82 (dd, <i>J</i> = 7.7 Hz, 1.1 Hz, 1H), 7.78 (d, <i>J</i> = 8.3 Hz, 1H),7.77–7.72 (m, 2H), 7.49–7.45 (m, 1H), 7.36 (t, <i>J</i> = 9.5 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.1, 155.5, 154.9, 148.4, 142.8, 135.8, 133.7, 131.5, 131.3, 129.9, 128.9, 128.8, 128.6, 128.4, 126.7, 125.8, 125.3, 124.9, 124.8, 124.0, 122.7, 117.1, 112.2, 107.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 467.0769; found, 467.0782; calcd for C<sub>25</sub>H<sub>14</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 489.0583; found, 489.0616.</div></div><div id="sec4_2_3_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78">4.2.3.39.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(7-methoxy-1-benzofuran-2-yl)quinoline-4-carboxamide (<b>8k</b>)</h5><div class="NLM_p last">White solid. Yield 41%. mp 254.6–255.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.93 (s, 1H), 8.43 (s, 1H), 8.31 (t, <i>J</i> = 8.8 Hz, 2H), 8.21 (d, <i>J</i> = 8.4 Hz, 1H), 7.95–7.87 (m, 3H), 7.76–7.71 (m, 2H), 7.37 (d, <i>J</i> = 7.3 Hz, 1H), 7.27 (t, <i>J</i> = 7.9 Hz, 1H), 7.07 (d, <i>J</i> = 7.6 Hz, 1H), 4.02 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.1, 154.7, 148.4, 148.3, 145.7, 144.8, 142.9, 135.9, 133.6, 131.5, 131.3, 130.3, 129.9, 128.6, 128.4, 125.8, 125.3, 125.0, 124.7, 124.0, 122.8, 117.0, 114.6, 108.7, 108.1, 56.3. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>26</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 497.0874; found, 497.0833; calcd for C<sub>26</sub>H<sub>16</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 519.0694; found, 519.0660.</div></div><div id="sec4_2_3_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79">4.2.3.40.  2-(2<i>H</i>-1,3-Benzodioxol-5-yl)-<i>N</i>-[2-chloro-5-(trifluoromethyl)phenyl]quinoline-4-carboxamide (<b>8l</b>)</h5><div class="NLM_p last">White solid. Yield 56%. mp 196.5–197.7 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.81 (s, 1H), 8.39 (s, 1H), 8.27–8.24 (m, 2H), 8.14 (d, <i>J</i> = 8.4 Hz, 1H), 7.94–7.92 (m, 2H), 7.85 (dd, <i>J</i> = 17.9, 7.8 Hz, 2H), 7.73 (dd, <i>J</i> = 8.3, 1.3 Hz, 1H), 7.66 (t, <i>J</i> = 8.1 Hz, 1H), 7.14 (d, <i>J</i> = 8.6 Hz, 1H), 6.15 (s, 2H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.5, 155.6, 149.5, 148.7, 148.3, 142.4, 135.9, 133.7, 132.9, 131.5, 130.8, 129.9, 128.9, 128.6, 127.6, 125.5, 125.3, 124.7, 123.4, 122.4, 117.5, 109.1, 107.6, 102.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>24</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 471.0718; found, 471.0711; calcd for C<sub>24</sub>H<sub>14</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 493.0537; found, 493.0538.</div></div><div id="sec4_2_3_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80">4.2.3.41.  <i>N</i>-[2-Chloro-5-(trifluoromethyl)phenyl]-2-(2,3-dihydro-1,4-benzodioxin-6-yl)quinoline-4-carboxamide (<b>8m</b>)</h5><div class="NLM_p last">White solid. Yield 55%. mp 201.7–203.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.81 (s, 1H), 8.38 (s, 1H), 8.26 (t, <i>J</i> = 7.9 Hz, 2H), 8.13 (d, <i>J</i> = 8.4 Hz, 1H), 7.90–7.81 (m, 4H), 7.73 (dd, <i>J</i> = 8.5, 1.7 Hz, 1H), 7.66 (t, <i>J</i> = 7.6 Hz, 1H),7.07 (d, <i>J</i> = 8.4 Hz, 1H), 4.34 (s, 4H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.5, 155.6, 148.3, 145.8, 144.3, 142.3, 135.9, 133.8, 131.9, 131.5, 130.7, 129.9, 128.9, 128.6, 127.6, 125.5, 124.7, 123.4, 122.8, 121.0, 118.0, 117.3, 116.4, 64.9, 64.6. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 485.0874; found, 485.0866; calcd for C<sub>25</sub>H<sub>16</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 507.0694; found, 507.0700.</div></div><div id="sec4_2_3_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81">4.2.3.42.  2-(1-Benzofuran-2-yl)-<i>N</i>-(3,4-dichlorophenyl)quinoline-4-carboxamide (<b>12a</b>)</h5><div class="NLM_p last">White solid. Yield 37%. mp >290 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.19 (s, 1H), 8.40 (s, 1H), 8.28–8.14 (m, 3H), 8.01–7.95 (m, 1H), 7.90 (ddd, <i>J</i> = 8.4, 6.8, 1.5 Hz, 1H), 7.84–7.67 (m, 5H), 7.46 (ddd, <i>J</i> = 8.4, 7.2, 1.4 Hz, 1H), 7.41–7.33 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.7, 155.5, 154.6, 148.4, 142.9, 139.3, 131.6, 131.4, 131.3, 130.0, 128.7, 128.5, 126.7, 126.3, 125.7, 124.2, 123.7, 122.7, 121.8, 120.6, 116.9, 112.2, 108.0. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>24</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 433.0505; found, 433.0532; calcd for C<sub>24</sub>H<sub>14</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 455.0325; found, 455.0329.</div></div><div id="sec4_2_3_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82">4.2.3.43.  2-(1-Benzofuran-2-yl)-<i>N</i>-(2,3-dichlorophenyl)quinoline-4-carboxamide (<b>12b</b>)</h5><div class="NLM_p last">White solid. Yield 34%. mp >290 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.87 (s, 1H), 8.39 (s, 1H), 8.27 (d, <i>J</i> = 8.3 Hz, 1H), 8.20 (d, <i>J</i> = 8.4 Hz, 1H), 7.96 (s, 1H), 7.91 (ddd, <i>J</i> = 8.4, 6.9, 1.4 Hz, 1H), 7.81 (dd, <i>J</i> = 12.8, 8.1 Hz, 3H), 7.74 (ddd, <i>J</i> = 8.2, 6.9, 1.3 Hz, 1H), 7.65 (dd, <i>J</i> = 8.1, 1.5 Hz, 1H), 7.55–7.44 (m, 2H), 7.39–7.33 (m, 1H). HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>24</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 433.0505; found, 433.0507; calcd for C<sub>24</sub>H<sub>14</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 455.0325; found, 455.0329.</div></div><div id="sec4_2_3_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83">4.2.3.44.  2-(1-Benzofuran-2-yl)-<i>N</i>-(3-chloro-2-methylphenyl)quinoline-4-carboxamide (<b>12c</b>)</h5><div class="NLM_p last">White solid. Yield 35%. mp >290 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.65 (s, 1H), 8.41 (s, 1H), 8.22 (dd, <i>J</i> = 13.7, 8.4 Hz, 2H), 7.99 (s, 1H), 7.90 (ddd, <i>J</i> = 8.4, 6.8, 1.4 Hz, 1H), 7.81 (dd, <i>J</i> = 13.5, 8.0 Hz, 2H), 7.73 (ddd, <i>J</i> = 8.2, 6.8, 1.3 Hz, 1H), 7.59 (d, <i>J</i> = 7.8 Hz, 1H), 7.51–7.41 (m, 2H), 7.35 (q, <i>J</i> = 8.1 Hz, 2H), 2.40 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.9, 155.5, 154.7, 148.4, 143.4, 137.7, 134.4, 132.1, 131.3, 130.0, 128.8, 128.4, 127.6, 126.7, 126.0, 125.7, 124.2, 124.1, 122.7, 116.8, 112.2, 107.9, 15.9. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>18</sub>ClN<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 413.1051; found, 413.1055; calcd for C<sub>25</sub>H<sub>17</sub>ClN<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 435.0871; found, 435.0874.</div></div><div id="sec4_2_3_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84">4.2.3.45.  2-(1-Benzofuran-2-yl)-<i>N</i>-(2,5-dimethylphenyl)quinoline-4-carboxamide (<b>12d</b>)</h5><div class="NLM_p last">White solid. Yield 19%. mp >290 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.33 (s, 1H), 8.36 (s, 1H), 8.27–8.16 (m, 2H), 7.98 (d, <i>J</i> = 1.0 Hz, 1H), 7.90 (ddd, <i>J</i> = 8.4, 6.8, 1.4 Hz, 1H), 7.81 (dd, <i>J</i> = 12.2, 8.0 Hz, 2H), 7.73 (ddd, <i>J</i> = 8.2, 6.9, 1.3 Hz, 1H), 7.47 (ddd, <i>J</i> = 8.3, 7.2, 1.3 Hz, 1H), 7.44–7.40 (m, 1H), 7.40–7.32 (m, 1H), 7.21 (d, <i>J</i> = 7.7 Hz, 1H), 7.05 (dd, <i>J</i> = 7.7, 1.8 Hz, 1H), 2.35 (s, 3H), 2.32 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.6, 155.5, 154.7, 148.4, 143.9, 135.9, 135.7, 131.2, 130.8, 130.5, 129.9, 128.8, 128.3, 127.5, 127.2, 126.7, 125.8, 124.2, 122.7, 116.6, 112.2, 107.9, 21.0, 18.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>26</sub>H<sub>21</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 393.1598; found, 393.1600; calcd for C<sub>26</sub>H<sub>20</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 415.1417; found, 415.1425.</div></div><div id="sec4_2_3_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85">4.2.3.46.  2-(1-Benzofuran-2-yl)-<i>N</i>-(2,4,6-trimethylphenyl)quinoline-4-carboxamide (<b>12e</b>)</h5><div class="NLM_p last">White solid. Yield 21%. mp 266.8–268.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.21 (s, 1H), 8.30 (s, 1H), 8.25 (dd, <i>J</i> = 8.4, 1.3 Hz, 1H), 8.20 (d, <i>J</i> = 8.4 Hz, 1H), 7.99 (d, <i>J</i> = 0.9 Hz, 1H), 7.90 (ddd, <i>J</i> = 8.4, 6.9, 1.4 Hz, 1H), 7.85–7.78 (m, 2H), 7.74 (ddd, <i>J</i> = 8.3, 6.9, 1.2 Hz, 1H), 7.50–7.44 (m, 1H), 7.40–7.33 (m, 1H), 7.00 (s, 2H), 2.34 (s, 6H), 2.29 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.5, 155.5, 154.6, 148.5, 148.4, 143.9, 136.6, 135.5, 132.2, 131.3, 130.0, 129.0, 128.8, 128.4, 126.7, 125.8, 124.2, 122.7, 116.4, 112.2, 107.9, 21.0, 18.8. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>27</sub>H<sub>23</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 407.1754; found, 407.1760; calcd for C<sub>27</sub>H<sub>22</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 429.1573; found, 429.1572.</div></div><div id="sec4_2_3_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86">4.2.3.47.  2-(1-Benzofuran-2-yl)-<i>N</i>-(3-fluorophenyl)quinoline-4-carboxamide (<b>12f</b>)</h5><div class="NLM_p last">White solid. Yield 20%. mp 295.3–296.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.10 (s, 1H), 8.38 (s, 1H), 8.18 (dd, <i>J</i> = 14.9, 8.3 Hz, 2H), 8.05–7.66 (m, 6H), 7.62–7.30 (m, 4H), 7.03 (t, <i>J</i> = 8.6 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.7, 155.5, 154.6, 148.4, 143.2, 131.3, 130.0, 128.7, 128.5, 126.7, 125.7, 124.2, 123.8, 122.7, 116.8, 116.3, 112.2, 108.0. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>24</sub>H<sub>16</sub>FN<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 383.1190; found, 383.1194; calcd for C<sub>24</sub>H<sub>15</sub>FN<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 405.1010; found, 405.1017.</div></div><div id="sec4_2_3_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87">4.2.3.48.  2-(1-Benzofuran-2-yl)-<i>N</i>-[3,5-bis(trifluoromethyl)phenyl]quinoline-4-carboxamide (<b>12h</b>)</h5><div class="NLM_p last">White solid. Yield 26%. mp 271.2–272.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.52 (s, 1H), 8.51 (d, <i>J</i> = 17.0 Hz, 3H), 8.25 (dt, <i>J</i> = 21.1, 8.0 Hz, 2H), 8.04–7.89 (m, 3H), 7.80 (dd, <i>J</i> = 18.1, 7.9 Hz, 2H), 7.73 (t, <i>J</i> = 7.7 Hz, 1H), 7.53–7.44 (m, 1H), 7.36 (t, <i>J</i> = 7.5 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.2, 155.5, 154.6, 148.5, 148.4, 142.4, 141.1, 131.4, 131.2, 130.0, 128.7, 128.6, 126.7, 125.8, 124.2, 122.8, 120.4, 117.1, 112.2, 108.0. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>26</sub>H<sub>15</sub>F<sub>6</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 501.1032; found, 501.1031; calcd for C<sub>26</sub>H<sub>14</sub>F<sub>6</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 523.0852; found, 523.0868.</div></div><div id="sec4_2_3_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88">4.2.3.49.  2-(1-Benzofuran-2-yl)-<i>N</i>-[2-chloro-4-(trifluoromethyl)phenyl]quinoline-4-carboxamide (<b>12i</b>)</h5><div class="NLM_p last">White solid. Yield 44%. mp 249.1–249.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.96 (s, 1H), 8.44 (s, 1H), 8.22 (tdd, <i>J</i> = 21.0, 12.5, 6.0 Hz, 3H), 8.06 (d, <i>J</i> = 2.2 Hz, 1H), 7.98–7.69 (m, 6H), 7.49 (dt, <i>J</i> = 15.0, 7.6 Hz, 1H), 7.36 (t, <i>J</i> = 7.4 Hz, 1H). HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 467.0769; found, 467.0774; calcd for C<sub>25</sub>H<sub>14</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 489.0588; found, 489.0929.</div></div><div id="sec4_2_3_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89">4.2.3.50.  2-(1-Benzofuran-2-yl)-<i>N</i>-(4-ethoxyphenyl)quinoline-4-carboxamide (<b>12j</b>)</h5><div class="NLM_p last">White solid. Yield 36%. mp >290 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.74 (s, 1H), 8.34 (s, 1H), 8.18 (t, <i>J</i> = 7.6 Hz, 2H), 7.98 (s, 1H), 7.93–7.85 (m, 1H), 7.79 (dd, <i>J</i> = 12.2, 8.1 Hz, 2H), 7.74–7.68 (m, 3H), 7.46 (t, <i>J</i> = 7.9 Hz, 1H), 7.35 (t, <i>J</i> = 7.5 Hz, 1H), 6.98 (d, <i>J</i> = 8.9 Hz, 2H), 4.04 (q, <i>J</i> = 6.9 Hz, 2H), 1.35 (t, <i>J</i> = 7.0 Hz, 3H). HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>26</sub>H<sub>21</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 409.1547; found, 409.1547; calcd for C<sub>26</sub>H<sub>20</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 431.1366; found, 431.1220.</div></div><div id="sec4_2_3_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90">4.2.3.51.  2-(1-Benzofuran-2-yl)-<i>N</i>-[(3,4-dichlorophenyl)methyl]quinoline-4-carboxamide (<b>12k</b>)</h5><div class="NLM_p last">White solid. Yield 49%. mp 229.8–231.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.52 (t, <i>J</i> = 6.0 Hz, 1H), 8.23 (s, 1H), 8.20–8.12 (m, 2H), 7.94 (s, 1H), 7.87 (ddd, <i>J</i> = 8.3, 6.8, 1.4 Hz, 1H), 7.79 (dd, <i>J</i> = 13.6, 8.0 Hz, 2H), 7.74–7.64 (m, 3H), 7.51–7.41 (m, 2H), 7.35 (t, <i>J</i> = 7.5 Hz, 1H), 4.61 (d, <i>J</i> = 6.0 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 167.0, 155.5, 154.7, 148.4, 143.2, 140.7, 131.5, 131.2, 130.1, 129.9, 128.7, 128.4, 128.2, 126.6, 125.8, 124.1, 122.7, 116.6, 112.2, 107.8, 42.3. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>17</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 447.0662; found, 447.0724; calcd for C<sub>25</sub>H<sub>16</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 469.0481; found, 469.0484.</div></div><div id="sec4_2_3_52" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91">4.2.3.52.  2-(1-Benzofuran-2-yl)-<i>N</i>-[(4-methylphenyl)methyl]quinoline-4-carboxamide (<b>12l</b>)</h5><div class="NLM_p last">White solid. Yield 55%. mp 235.8–236.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.41 (t, <i>J</i> = 6.0 Hz, 1H), 8.18 (s, 1H), 8.16 (dt, <i>J</i> = 8.2, 1.6 Hz, 2H), 7.92 (d, <i>J</i> = 0.9 Hz, 1H), 7.86 (ddd, <i>J</i> = 8.5, 6.8, 1.4 Hz, 1H), 7.83–7.74 (m, 2H), 7.68 (ddd, <i>J</i> = 8.2, 6.9, 1.3 Hz, 1H), 7.45 (ddd, <i>J</i> = 8.4, 7.2, 1.4 Hz, 1H), 7.37–7.32 (m, 3H), 7.22 (s, 1H), 7.20 (s, 1H), 4.57 (d, <i>J</i> = 5.9 Hz, 2H), 2.32 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.8, 155.5, 154.7, 148.4, 148.3, 143.6, 136.6, 136.4, 131.1, 129.8, 129.5, 128.8, 128.1, 128.0, 126.6, 125.9, 124.2, 124.1, 122.7, 116.4, 112.2, 107.7, 43.0, 21.2. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>26</sub>H<sub>21</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 393.1598; found, 393.1601; calcd for C<sub>26</sub>H<sub>20</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 415.1417; found, 415.1427.</div></div><div id="sec4_2_3_53" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92">4.2.3.53.  2-(1-Benzofuran-2-yl)-<i>N</i>-[(4-fluorophenyl)methyl]quinoline-4-carboxamide (<b>12m</b>)</h5><div class="NLM_p last">White solid. Yield 53%. mp 225.6–226.7 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.50 (t, <i>J</i> = 5.9 Hz, 1H), 8.21 (s, 1H), 8.16 (dd, <i>J</i> = 8.3, 1.3 Hz, 2H), 7.93 (d, <i>J</i> = 1.0 Hz, 1H), 7.86 (ddd, <i>J</i> = 8.2, 6.8, 1.4 Hz, 1H), 7.83–7.74 (m, 2H), 7.68 (ddd, <i>J</i> = 8.2, 6.9, 1.2 Hz, 1H), 7.53–7.42 (m, 3H), 7.35 (td, <i>J</i> = 7.5, 1.0 Hz, 1H), 7.23 (t, <i>J</i> = 8.9 Hz, 2H), 4.60 (d, <i>J</i> = 5.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.8, 163.0, 160.6, 155.5, 154.7, 148.4, 148.3, 143.4, 135.7, 131.1, 130.0, 129.9, 128.7, 128.1, 126.6, 125.9, 124.2, 122.7, 116.5, 115.8, 115.6, 112.2, 107.8, 42.6. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>25</sub>H<sub>18</sub>FN<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 397.1347; found, 397.1349; calcd for C<sub>25</sub>H<sub>17</sub>FN<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 419.1166; found, 419.1168.</div></div><div id="sec4_2_3_54" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93">4.2.3.54.  2-(1-Benzofuran-2-yl)-<i>N</i>-(naphthalene-1-yl)quinoline-4-carboxamide (<b>12n</b>)</h5><div class="NLM_p last">White solid. Yield 38%. mp 286.9–288.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.98 (s, 1H), 8.52 (s, 1H), 8.30 (d, <i>J</i> = 8.4 Hz, 1H), 8.21 (s, 2H), 8.03 (t, <i>J</i> = 4.3 Hz, 2H), 7.92 (br s, 3H), 7.82 (t, <i>J</i> = 9.6 Hz, 2H), 7.74 (t, <i>J</i> = 7.1 Hz, 1H), 7.65–7.57 (m, 3H), 7.47 (t, <i>J</i> = 7.3 Hz, 1H), 7.36 (t, <i>J</i> = 7.0 Hz, 1H). HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>28</sub>H<sub>19</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 415.1441; found, 415.1451; calcd for C<sub>28</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 437.1260; found, 437.1272.</div></div><div id="sec4_2_3_55" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94">4.2.3.55.  2-(1-Benzofuran-2-yl)-<i>N</i>-[phenyl(pyridine-2-yl)methyl]quinoline-4-carboxamide (<b>12o</b>)</h5><div class="NLM_p last">White solid. Yield 33%. mp 222.5–223.4 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.87 (d, <i>J</i> = 8.2 Hz, 1H), 8.66–8.55 (m, 1H), 8.19 (s, 1H), 8.18–8.13 (m, 1H), 8.06 (d, <i>J</i> = 8.1 Hz, 1H), 7.94 (s, 1H), 7.85 (t, <i>J</i> = 7.7 Hz, 2H), 7.79 (t, <i>J</i> = 8.3 Hz, 2H), 7.64 (t, <i>J</i> = 7.6 Hz, 1H), 7.58 (d, <i>J</i> = 7.9 Hz, 1H), 7.50 (d, <i>J</i> = 7.4 Hz, 2H), 7.45 (t, <i>J</i> = 7.2 Hz, 1H), 7.41–7.27 (m, 5H), 6.55 (d, <i>J</i> = 8.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.5, 160.8, 155.5, 154.7, 149.6, 148.3, 143.7, 141.5, 137.6, 131.1, 129.8, 129.0, 128.8, 128.2, 128.1, 127.8, 126.6, 125.8, 124.3, 124.1, 123.0, 122.7, 122.2, 116.5, 112.2, 107.7, 59.1. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>30</sub>H<sub>22</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 456.1707; found, 456.1714; calcd for C<sub>30</sub>H<sub>21</sub>N<sub>3</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 478.1526; found, 478.1530.</div></div><div id="sec4_2_3_56" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95">4.2.3.56.  2-(1-Benzofuran-2-yl)-<i>N</i>-(4-phenoxyphenyl)quinoline-4-carboxamide (<b>12p</b>)</h5><div class="NLM_p last">White solid. Yield 38%. mp 264.5–265.8 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.97 (s, 1H), 8.37 (s, 1H), 8.20 (dd, <i>J</i> = 8.2, 3.5 Hz, 2H), 7.98 (s, 1H), 7.91–7.86 (m, 3H), 7.80 (dd, <i>J</i> = 12.8, 8.0 Hz, 2H), 7.73–7.69 (m, 1H), 7.48–7.33 (m, 4H), 7.16–7.10 (m, 3H), 7.03 (d, <i>J</i> = 7.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 165.2, 157.7, 155.5, 154.7, 153.0, 148.4, 143.6, 135.1, 131.2, 130.5, 129.9, 128.8, 128.4, 126.7, 125.8, 124.2, 124.0, 123.6, 122.7, 122.3, 120.0, 118.5, 116.8, 112.2, 107.9. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>30</sub>H<sub>21</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 457.1547; found, 457.1548; calcd for C<sub>30</sub>H<sub>20</sub>N<sub>2</sub>O<sub>3</sub>Na [M + Na]<sup>+</sup>, 479.1366; found, 479.1370.</div></div><div id="sec4_2_3_57" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96">4.2.3.57.  2-(1-Benzofuran-2-yl)-<i>N</i>-(diphenylmethyl)quinoline-4-carboxamide (<b>12q</b>)</h5><div class="NLM_p last">The specific synthesis method is the same as that of the compound <b>5h</b>, and the reaction mixture gives the compound <b>12q</b> as a white solid. Yield 54%, HPLC purity 96%. mp 248.5–249.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.92 (d, <i>J</i> = 8.6 Hz, 1H), 8.18 (s, 1H), 8.16 (d, <i>J</i> = 8.5 Hz, 1H), 7.98 (d, <i>J</i> = 8.3 Hz, 1H), 7.95 (s, 1H), 7.87–7.83 (m, 1H), 7.79 (t, <i>J</i> = 9.1 Hz, 2H), 7.65–7.61 (m, 1H), 7.49–7.28 (m, 12H), 6.54 (d, <i>J</i> = 8.6 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 166.3, 155.5, 154.7, 148.3, 143.7, 142.5, 131.1, 129.9, 129.0, 128.8, 128.1, 127.9, 127.7, 126.6, 125.6, 124.2, 124.1, 122.7, 116.4, 112.2, 107.8, 57.2. HRMS <i>m</i>/<i>z</i> (ESI): calcd for C<sub>31</sub>H<sub>23</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 455.1573; found, 455.1768; calcd for C<sub>31</sub>H<sub>22</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>, 477.1573; found, 477.1593.</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97">4.3.  Pharmacological Experimental Methods</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98">4.3.1.  Protein Production and Purification</h4><div class="NLM_p last">N-terminally his-tagged human SIRT6 constructs (residues 1–355, 1–318 and 13–308) using vector pQE80L.1 (internal pQE80 derivative with the TEV cleavage site) or vector pET151-D-TOPO in <i>Escherichia coli</i> M15 [pREP4] were used, which were kindly provided by Professor Clemens Steegborn (Program in Lehrstuhl Biochemie und Forschungszentrum für Biomakromoleküle Universitätsstr, Universität Bayreuth). Proteins were expressed and purified essentially as described previously.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The purified SIRT6 protein was stored at −80 °C in buffer containing 20 mM Na–HEPES pH 7.5, 100 mM NaCl, and 2 mM dithiothreitol (DTT).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99">4.3.2.  <i>In Vitro</i> Sirtuin Assay</h4><div class="NLM_p">Compounds were tested for their agonist activity against SIRT6 as described previously.<a onclick="showRef(event, 'ref39 ref43'); return false;" href="javascript:void(0);" class="ref ref39 ref43">(39,43)</a> For SIRT6 deacetylation, the 0.09 μg/mL in-house recombinant SIRT6 was incubated with different test compounds in the assay buffer (50 mM Tris-HCl, pH 8.0, 137 mM NaCl, 2.7 mM KCl, and 1 mM MgCl<sub>2</sub>) in black half-volume 96-well plates at 37 °C for 15 min. The mixture contained 320 μM substrate S5 (Ac-RYQK(Ac)-AMC), which was purchased from CASLO ApS (Denmark). NAD (1 mM) was added to initiate the reaction, and the mixture was incubated at 37 °C for 120 min. Next, the developer solution containing 6 μg/μL trypsin and 40 mM nicotinamide was added, and the solutions were incubated for 30 min at room temperature. The fluorescence was then measured with excitation and emission wavelengths of 380 and 440 nm, respectively, using a Bio Tek system (Bio Tek Instruments, Inc., Winooski, VT). For SIRT6 fatty-acid deacetylation, the 1 μM recombinant SIRT6 protein was incubated with different test compounds in the assay buffer (50 mM Tris-HCl, pH 8.0), bovine serum albumin (BSA) (1 mg/mL), and DTT (1 mM) in black, half-volume 96-well plates at 37 °C for 20 min. The mixture contained 10 μM Ac-EALPKK(Myr)-AMC peptide, which was purchased from ChinaPeptides Co., Ltd. (Suzhou, China). NAD (1 mM) was added to initiate the reaction, and the plates were incubated at 37 °C for 120 min. Next, the developer solution containing 5 mg/mL of trypsin and 8 mM nicotinamide was added, and the mixtures were incubated for 120 min at 37 °C. The fluorescence was then measured with excitation and emission wavelengths of 380 and 440 nm, respectively, using a Bio Tek system (Bio Tek Instruments, Inc., Winooski, VT). The experimental data were fitted in GraphPad Prism 5 to obtain inhibition or activity values using the following equations: inhibition % = (max – signal)/(max – min) × 100 or activity % = signal/(max – min) × 100.</div><div class="NLM_p">For SIRT6 deacetylation on Ac-RYQK(Ac)-AMC, the SIRT6 protein (60 μM) was incubated in a 100 μL of the reaction mixture (100 μM Ac-RYQK(Ac)-AMC, 3 mM NAD<sup>+</sup>, and assay buffer) with DMSO or 25 μM <b>12q</b> at 37 °C for 5 h. For SIRT6 demyristoylation on Ac-EALPKK(Myr)-AMC, the SIRT6 protein (10 μM) was incubated in a 100 μL of the reaction mixture (100 μM Ac-EALPKK(Myr)-AMC, 3 mM NAD<sup>+</sup>, and assay buffer) with DMSO or 25 μM <b>12q</b> at 37 °C for 2 h. The reactions were quenched with 100 mM HCl and 160 mM acetic acid at 37 °C for 30 min. After centrifugation at 12,000 rpm for 10 min, the supernatant was collected and analyzed by HPLC. HPLC analyses were performed on a ZORBAX Eclipse Plus C18 column (4.6 × 100 mm, 3.5 μm). The binary solvent system (A/B) was as follows: water with 0.1% trifluoroacetic acid (A) and acetonitrile with 0.1% trifluoroacetic acid (B).</div><div class="NLM_p last"><i>In vitro</i> enzymatic inhibition/activation assays of SIRT1-3 were also performed using the FDL assay provided by Shanghai ChemPartner Co., Ltd (Shanghai, China) and <i>in vitro</i> enzymatic inhibition/activation assay of SIRT5 was also performed using the FDL assay provided by BPS Bioscience Inc. (San Diego, California).</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100">4.3.3.  SPR Assay</h4><div class="NLM_p last">SPR technology-based binding assays were performed on a Biacore X100 instrument (GE Healthcare) at room temperature.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The SIRT6 proteins were immobilized on a CM5 sensor chip by a standard amide coupling procedure in 10 mM sodium acetate (pH 4.5). The activated surface was then brought into contact with the protein solution at a concentration of 100 μg/mL. The chip was equilibrated with 1.05× phosphate-buffered saline (PBS) buffer (10.5 mM phosphate (pH 7.4), 2.84 mM KCl, 143.85 mM NaCl, and 0.005% (v/v) surfactant P20) for 4 h. The compounds were serially diluted and injected for 60 s (contact phase) at a flow rate of 30 μL/min, followed by 120 s of the buffer flow (dissociation phase). The <i>K</i><sub>d</sub> values of the tested compounds were determined by BIA evaluation software (GE Healthcare).</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101">4.3.4.  DSF Assay</h4><div class="NLM_p last">DSF experiments were performed on a RT-PCR detection system (Bio-Rad CFX96) according to the known protocol.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> SYPRO orange (Sigma, 5000X concentrate in DMSO, S5692) was monitored using FRET filters at a wavelength of 492 nm for excitation and ROX filters at a wavelength of 610 nm for emission. Each reaction solution containing 2 μM SIRT6 proteins in buffer (20 mM HEPES, pH 7.4, and 100 mM NaCl), 5× SYPRO orange, and the test compounds in 10 μL, was heated from 25 to 95 °C. The fluorescence intensities were recorded every 1 °C/min and plotted as a function of temperature. The inflection point of the transition curve (<i>T</i><sub>m</sub>) was calculated by fitting the Boltzmann equation to the sigmoidal curve in GraphPad Prism 5.0.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102">4.3.5.  ITC Assay</h4><div class="NLM_p last">Experiments using ITC were carried out in a MicroCal iTC200 instrument at 25 °C.<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52,53)</a> The buffer conditions were 20 mM HEPES (pH 7.5), 100 mM NaCl, and 2 mM DTT. The titration was performed by injecting the proteins (400 μM) into a reaction cell containing the activators (80 μM). The thermodynamic binding parameters were extracted by nonlinear regression analysis of the binding isotherms (MicroCal Origin software, version 7.0). A single-site binding model was applied to determine the equilibrium dissociation constants (<i>K</i><sub>d</sub> values).</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103">4.3.6.  Cell Culture</h4><div class="NLM_p last">All the tumor cell lines used in this investigation were purchased from American Type Culture Collection (ATCC, Rockville, MD, USA). Cells were cultured in Dulbecco’s modified Eagle’s medium or RPMI1640 supplemented with 10% fetal bovine serum (Gibco, Eggenstein, Germany), 100 units/mL penicillin (HyClone), and streptomycin (HyClone) and grown in a 37 °C incubator with a humidified 5% CO<sub>2</sub> atmosphere.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104">4.3.7.  Proliferation Assay</h4><div class="NLM_p last">Cell proliferation assays were conducted as previously reported.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> A variety of human cancer cells were plated in triplicate in 96-well plates and were treated with the indicated concentrations of <b>12q</b> or other agents for 72 h, and cell viability was determined using an MTT (3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide) (Sigma-Aldrich, St. Louis, MO) assay. The IC<sub>50</sub> values were calculated using GraphPad Prism 5.01 software.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105">4.3.8.  Colony Formation Assay</h4><div class="NLM_p last">The cells were seeded in 6-well plates at a densities of 750, 500, 2200, and 500 cells per well for BXPC-3, PANC-1, AsPC-1, and MIAPaCa-2. After 24 h, different concentrations of <b>12q</b> or other agents were added. After 9 to 18 days of incubation, cells were fixed with 4% paraformaldehyde and stained with crystal violet. Colonies with >50 cells were counted under an inverted microscope.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106">4.3.9.  EdU Incorporation Assay</h4><div class="NLM_p last">This EdU incorporation assay was conducted using an EdU assay kit (KGA337-500, KeyGEN BioTECH). A total of 8000 (BXPC-3), 4600 (PANC-1), 7500 (AsPC-1), and 2500 (MIAPaCa-2) cells per well were seeded in 96-well plates and treated with different concentrations of <b>12q</b> or other agents for 24 h. Then, the proliferation of the cells was determined with an EdU-Apollo DNA proliferation detection kit according to the manufacturer’s instructions. The images were captured with a high-content screening instrument (Thermo) analyzed in HCC software.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107">4.3.10.  Migration Assay</h4><div class="NLM_p last">The migration assay was performed following the method reported previously.<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> BXPC-3, PANC-1, AsPC-1, or MIAPaCa-2 cells were seeded in 12-well plates and incubated at 37 °C under 5% CO<sub>2</sub>, and the cells were allowed to grow to monolayer confluence. Leaving the medium behind, the centers of the cell monolayer were scraped with a sterile, 200 μL pipette tip to create a denuded zone (gap) of constant width. Subsequently, the cell monolayer was washed with sterile PBS, and cells were exposed to various concentrations of <b>12q</b> or other agents. Images were taken using an Olympus inverted microscope after 24 h of incubation at 37 °C under 5% CO<sub>2</sub>. The migrated cells were quantified by manual counting, and the percentage of inhibition was expressed using untreated wells as 100%.</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108">4.3.11.  Cellular Thermal Shift Assay</h4><div class="NLM_p last">After treating PANC-1 cells or BXPC-3 cells with <b>12q</b> or DMSO for 72 h, the cells were collected and washed twice with PBS. The cells were resuspended in PBS solution containing 1 mmol/L cocktail. Divide each group of samples into multiple portions, corresponding to multiple temperature gradients. All the samples were heated at the corresponding temperature for 3 min and then taken out. The cells were lysed by repeated freezing and thawing 3 times. The sample was then centrifuged at 12,000 rpm and 4 °C for 15 min. The supernatants were analyzed by western blot. All experiments were performed in triplicate.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109">4.3.12.  Western Blot Analyses</h4><div class="NLM_p last">BXPC-3 and PANC-1 were incubated for 72 h in medium containing different concentrations of <b>12q</b> or other agents as previously described.<a onclick="showRef(event, 'ref27 ref33'); return false;" href="javascript:void(0);" class="ref ref27 ref33">(27,33)</a> Whole cell lysates were extracted with RIPA buffer (Beyotime, China) supplemented with a protease inhibitor cocktail (Sigma-Aldrich, Merck) and phenylmethylsulfonyl fluoride (Sigma-Aldrich, Merck). Protein concentrations were determined using a BCA protein assay kit (Heart, Xi’an). The protein extracts were separated by sodium dodecyl sulfate polyacrylamide gel electrophoresis on 12% polyacrylamide Tris-glycine gels and transferred onto a poly(vinylidene difluoride) (PVDF) membrane (Millipore). The PVDF membranes were then blocked with TBS-T containing 5% nonfat dry milk or 5% BSA with gentle shaking for 2 h at room temperature. Then, the membranes were incubated with antibodies including anti-SIRT6, anti-histone H3 (acetyl K9), anti-histone H3 (acetyl K18), anti-histone H3 (acetyl K56), anti-histone H3, anti-Lin28b, anti-Myc, anti-IGF2BP3, and anti-β-actin for 10 h at 4 °C. After three washes with TBS-T, the blots were incubated with the corresponding horseradish peroxidase-linked secondary antibodies (Zhong Shan Golden Bridge Biotechnology, China) for 1 h at 37 °C with gentle shaking. After three thorough washes with TBS-T, the blots were visualized with an enhanced luminol-based chemiluminescent substrate (Abbkine). Information regarding the primary antibodies can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">Table S4</a>.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110">4.3.13.  Cell Cycle Assays</h4><div class="NLM_p last">Cell cycle assays in PANC-1 and BXPC-3 cells were performed with a Cell Cycle Detection Kit (Keygentec, KGA511-KGA512) according to the manufacturer’s instructions. Briefly, PANC-1 and BXPC-3 cells were seeded in 6-well plates and treated with different concentrations of <b>12q</b> or <b>MDL-800</b> for 48 h. Then, the cells were incubated with 50 μg/mL propidium iodide (PI) for 30 min in the dark. The stained cells were then analyzed using a CytoFLEX flow cytometer (Beckman Coulter) and the resulting data were analyzed with cell cycle analysis software (Kaluza, Beckman Coulter).</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111">4.3.14.  Apoptosis Assay</h4><div class="NLM_p last">The apoptosis assays in PANC-1 and BXPC-3 cells were performed with an Annexin V-FITC Apoptosis Detection Kit (Keygentec, KGA105-KGA108) according to the manufacturer’s instructions. PANC-1 and BXPC-3 cells were seeded in 6-well plates and treated with different concentrations of <b>12q</b> or <b>MDL-800</b> for 48 h. Then, the cells were incubated with 5 μL of Annexin V-FITC and 5 μL of PI for 10 min in the dark. The stained cells were then analyzed and the resulting data were analyzed using a CytoFLEX flow cytometer (Beckman Coulter).</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112">4.3.15.  PK Study</h4><div class="NLM_p last">A 10 or 2 mg/mL dosing solution of <b>12q</b> was prepared by dissolution in physiological saline containing 5% DMSO, 10% solutol, and 85% saline with the pH adjusted to 7 for p.o. or i.v. administration in 6- to 8-week-old male Sprague-Dawley rats. At time points of 0.083, 0.25, 0.5, 1, 2, 4, 6, 8, and 24 h after dosing for p.o. or 0.25, 0.5, 1, 2, 4, 6, 8, 10, and 24 h after dosing for i.v., the blood samples was collected from each animal <i>via</i> a cardiac puncture, and plasma concentrations were determined using LC–MS/MS analysis. Noncompartmental PK parameters were fitted using DAS software (Enterprise, version 2.0, Mathematical Pharmacology Professional Committee of China).</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113">4.3.16.  <i>In Vivo</i> Models</h4><div class="NLM_p last">All animal experiments carried out were approved by the Animal Care and Use Committee of Sichuan University (Chengdu, Sichuan, China). BALB/c female nude mice (6- to 8-week-old) were purchased from Beijing HFK Bioscience Co., Ltd. (Beijing, China). To establish the PANC-1 xenograft model, 100 μL of a PANC-1 cell suspension (between 5 × 10<sup>6</sup> and 1 × 10<sup>7</sup> cells) was subcutaneously injected into the right flank region of female nude mice. After the tumors had grown to 130–150 mm<sup>3</sup>, the mice were randomly divided into 4 groups and administered different doses of the test compound or vehicle. The compounds were dissolved in 5% (v/v) DMSO (Sigma-Aldrich), 12.5% (v/v) castor oil (Sigma-Aldrich), 12.5% ethanol, and 70% saline. Tumor burden was monitored every 3 days using calipers. Tumor volume (TV) was calculated using the following formula: TV = length × width<sup>2</sup> × 0.5.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i114"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01183" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67153" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67153" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01183?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01183</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Activity data of 40 candidate compounds; predicted binding mode of <b>Hit20</b> with SIRT6; dose-dependent activities of SIRT6 demyristoylation of <b>12q</b>; predicted binding mode of <b>12q</b> with SIRT6; other experimental data of compound <b>12q</b><i>in vitro</i> and <i>in vivo</i>; and <sup>1</sup>H NMR spectra, <sup>13</sup>C NMR spectra, HRMS data, and HPLC traces for key target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Binding model of compound <b>12q</b> with SIRT6 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf">jm0c01183_si_001.pdf (9.54 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_002.csv">jm0c01183_si_002.csv (7.28 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_003.pdb">jm0c01183_si_003.pdb (244.49 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01183" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50346" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50346" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shengyong Yang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5147-3746" title="Orcid link">http://orcid.org/0000-0001-5147-3746</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ea938b848d9993aa99899fc48f8e9fc48984"><span class="__cf_email__" data-cfemail="4039212e273339003323356e2524356e232e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiuli Chen</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weining Sun</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Drug Targeting and Drug Delivery System of Ministry
of Education, West China School of Pharmacy, Sichuan University, Chengdu, Sichuan 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shenzhen Huang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hailin Zhang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guifeng Lin</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hui Li</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jingxin Qiao</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Linli Li</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Drug Targeting and Drug Delivery System of Ministry
of Education, West China School of Pharmacy, Sichuan University, Chengdu, Sichuan 610041, P. R. China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>X.C., W.S., and S.H. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i116">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56230" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56230" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Natural Science Foundation of China (81773633, 21772130, and 81930125), National Science and Technology Major Project (2018ZX09711002-014-002, 2018ZX09711002-011-019, 2018ZX09201018, 2018ZX09711003-003-006, and 2019ZX09301-135), and 1.3.5 project for disciplines of excellence, West China Hospital, Sichuan University (ZYGD18001 and ZY2017202). The authors thank Professor Clemens Steegborn (Program in Lehrstuhl Biochemie und Forschungszentrum für Biomakromoleküle Universitätsstr, Universität Bayreuth) for providing the N-Terminally his-tagged human SIRT6 constructs. The authors also thank Professor Jian Zhang (Shanghai Jiao-Tong University School of Medicine) for providing the myristoylated H3K9 (H3K9-Myr) peptide.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">HCC</td><td class="NLM_def"><p class="first last">hepatocellular carcinoma cell</p></td></tr><tr><td class="NLM_term">PKM2</td><td class="NLM_def"><p class="first last">pyruvate kinase M2</p></td></tr><tr><td class="NLM_term">PDAC</td><td class="NLM_def"><p class="first last">pancreatic ductal adenocarcinoma cell</p></td></tr><tr><td class="NLM_term">DS 3.1</td><td class="NLM_def"><p class="first last">Discovery Studio (DS) 3.1</p></td></tr><tr><td class="NLM_term">EC<sub>1.5</sub></td><td class="NLM_def"><p class="first last">concentration of compound required to increase enzyme activity by 50%</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">(3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide)</p></td></tr><tr><td class="NLM_term">EdU</td><td class="NLM_def"><p class="first last">5-ethynyl-2′-deoxyuridine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i118">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73336" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73336" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 56 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlicker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boanca, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steegborn, C.</span></span> <span> </span><span class="NLM_article-title">Structure-based development of novel sirtuin inhibitors</span>. <i>Aging</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">852</span>– <span class="NLM_lpage">872</span>, <span class="refDoi"> DOI: 10.18632/aging.100388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.18632%2Faging.100388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=21937767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCht7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=852-872&author=C.+Schlickerauthor=G.+Boancaauthor=M.+Lakshminarasimhanauthor=C.+Steegborn&title=Structure-based+development+of+novel+sirtuin+inhibitors&doi=10.18632%2Faging.100388"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based development of novel sirtuin inhibitors</span></div><div class="casAuthors">Schlicker, Christine; Boanca, Gina; Lakshminarasimhan, Mahadevan; Steegborn, Clemens</div><div class="citationInfo"><span class="NLM_cas:title">Aging</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">852-872</span>CODEN:
                <span class="NLM_cas:coden">AGINCN</span>;
        ISSN:<span class="NLM_cas:issn">1945-4589</span>.
    
            (<span class="NLM_cas:orgname">Impact Journals LLC</span>)
        </div><div class="casAbstract">Sirtuins are NAD+-dependent protein deacetylases regulating metab., stress responses, and aging processes.  Mammalia possess seven Sirtuin isoforms, Sirt1-7, which differ in their subcellular localization and in the substrate proteins they deacetylate.  The physiol. roles of Sirtuins and their potential use as therapeutic targets for metabolic and agingrelated diseases have spurred interest in the development of small-mol. Sirtuin modulators.  Here, we describe an approach exploiting the structures available for four human Sirtuins for the development of isoform-specific inhibitors.  Virtual docking of a compd. library into the peptide binding pockets of crystal structures of Sirt2, 3, 5 and 6 yielded compds. potentially discriminating between these isoforms.  Further characterization in activity assays revealed several inhibitory compds. with little isoform specificity, but also two compds. with micromolar potency and high specificity for Sirt2.  Structure comparison and the predicted, shared binding mode of the Sirt2-specific compds. indicate a pocket extending from the peptide-binding groove as target side enabling isoform specificity.  Our family-wide structure-based approach thus identified potent, Sirt2-specific inhibitors as well as lead structures and a target site for the development of compds. specific for other Sirtuin isoform, constituting an important step toward the identification of a complete panel of isoform-specific Sirtuin inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHw485oGKwoLVg90H21EOLACvtfcHk0li7eVJ-uzh7PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCht7fO&md5=77d30598d6b2e4aa536577b3b3a9e461</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.18632%2Faging.100388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Faging.100388%26sid%3Dliteratum%253Aachs%26aulast%3DSchlicker%26aufirst%3DC.%26aulast%3DBoanca%26aufirst%3DG.%26aulast%3DLakshminarasimhan%26aufirst%3DM.%26aulast%3DSteegborn%26aufirst%3DC.%26atitle%3DStructure-based%2520development%2520of%2520novel%2520sirtuin%2520inhibitors%26jtitle%3DAging%26date%3D2011%26volume%3D3%26spage%3D852%26epage%3D872%26doi%3D10.18632%2Faging.100388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imai, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaeberlein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarente, L.</span></span> <span> </span><span class="NLM_article-title">Transcriptional silencing and longevity protein Sir2 is an NAD-dependent histone deacetylase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>403</i></span>,  <span class="NLM_fpage">795</span>, <span class="refDoi"> DOI: 10.1038/35001622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2F35001622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10693811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhsVWiurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=795&author=S.-I.+Imaiauthor=C.+M.+Armstrongauthor=M.+Kaeberleinauthor=L.+Guarente&title=Transcriptional+silencing+and+longevity+protein+Sir2+is+an+NAD-dependent+histone+deacetylase&doi=10.1038%2F35001622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional silencing and longevity protein Sir2 is an NAD-dependent histone deacetylase</span></div><div class="casAuthors">Imai, Shin-Ichiro; Armstrong, Christopher M.; Kaebertein, Matt; Guarente, Leonard</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6771</span>),
    <span class="NLM_cas:pages">795-800</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Yeast Sir2 is a heterochromatin component that silences transcription at silent mating loci, telomeres and the ribosomal DNA, and that also suppresses recombination in the rDNA and extends replicative life span.  Mutational studies indicate that lysine 16 in the amino-terminal tail of histone H4 and lysines 9, 14 and 18 in H3 are critically important in silencing, whereas lysines 5, 8 and 12 of H4 have more redundant functions.  Lysines 9 and 14 of histone H3 and lysines 5, 8 and 16 of H4 are acetylated in active chromatin and hypoacetylated in silenced chromatin, and overexpression of Sir2 promotes global deacetylation of histones, indicating that Sir2 may be a histone deacetylase.  Deacetylation of lysine 16 of H4 is necessary for binding the silencing protein, Sir3 (ref. 8).  Here we show that yeast and mouse Sir2 proteins are NAD- dependent histone deacetylases, which deacetylate lysines 9 and 14 of H3 and specifically lysine 16 of H4.  Our anal. of two SIR2 mutations supports the idea that this deacetylase activity accounts for silencing, recombination suppression and extension of life span in vivo.  These findings provide a mol. framework of NAD-dependent histone deacetylation that connects metab., genomic silencing and aging in yeast and, perhaps, in higher eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF5dAUjTy39LVg90H21EOLACvtfcHk0liU-MPiSgihEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhsVWiurY%253D&md5=513e2dca05684d2e9823e249a83d4e16</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F35001622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35001622%26sid%3Dliteratum%253Aachs%26aulast%3DImai%26aufirst%3DS.-I.%26aulast%3DArmstrong%26aufirst%3DC.%2BM.%26aulast%3DKaeberlein%26aufirst%3DM.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DTranscriptional%2520silencing%2520and%2520longevity%2520protein%2520Sir2%2520is%2520an%2520NAD-dependent%2520histone%2520deacetylase%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D795%26doi%3D10.1038%2F35001622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, B. J.</span></span> <span> </span><span class="NLM_article-title">Seven sirtuins for seven deadly diseases ofaging</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2012.10.525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.freeradbiomed.2012.10.525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23104101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1KqsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=133-171&author=B.+J.+Morris&title=Seven+sirtuins+for+seven+deadly+diseases+ofaging&doi=10.1016%2Fj.freeradbiomed.2012.10.525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Seven sirtuins for seven deadly diseases of aging</span></div><div class="casAuthors">Morris, Brian J.</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">133-171</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Sirtuins are a class of NAD+-dependent deacetylases having beneficial health effects.  This extensive review describes the numerous intracellular actions of the seven mammalian sirtuins, their protein targets, intracellular localization, the pathways they modulate, and their role in common diseases of aging.  Selective pharmacol. targeting of sirtuins is of current interest in helping to alleviate global disease burden.  Since all sirtuins are activated by NAD+, strategies that boost NAD+ in cells are of interest.  While most is known about SIRT1, the functions of the six other sirtuins are now emerging.  Best known is the involvement of sirtuins in helping cells adapt energy output to match energy requirements.  SIRT1 and some of the other sirtuins enhance fat metab. and modulate mitochondrial respiration to optimize energy harvesting.  The AMP kinase/SIRT1-PGC-1α-PPAR axis and mitochondrial sirtuins appear pivotal to maintaining mitochondrial function.  Downregulation with aging explains much of the pathophysiol. that accumulates with aging.  Posttranslational modifications of sirtuins and their substrates affect specificity.  Although SIRT1 activation seems not to affect life span, activation of some of the other sirtuins might.  Since sirtuins are crucial to pathways that counter the decline in health that accompanies aging, pharmacol. agents that boost sirtuin activity have clin. potential in treatment of diabetes, cardiovascular disease, dementia, osteoporosis, arthritis, and other conditions.  In cancer, however, SIRT1 inhibitors could have therapeutic value.  Nutraceuticals such as resveratrol have a multiplicity of actions besides sirtuin activation.  Their net health benefit and relative safety may have originated from the ability of animals to survive environmental changes by utilizing these stress resistance chems. in the diet during evolution.  Each sirtuin forms a key hub to the intracellular pathways affected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8LdSgVF05MLVg90H21EOLACvtfcHk0liU-MPiSgihEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1KqsrrE&md5=6ea48068831cafbef89787a3536e872b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2012.10.525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2012.10.525%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DB.%2BJ.%26atitle%3DSeven%2520sirtuins%2520for%2520seven%2520deadly%2520diseases%2520ofaging%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2013%26volume%3D56%26spage%3D133%26epage%3D171%26doi%3D10.1016%2Fj.freeradbiomed.2012.10.525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sauve, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolberger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeke, J. D.</span></span> <span> </span><span class="NLM_article-title">The biochemistry of sirtuins</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.74.082803.133500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1146%2Fannurev.biochem.74.082803.133500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=16756498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD28XosVKhs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2006&pages=435-465&author=A.+A.+Sauveauthor=C.+Wolbergerauthor=V.+L.+Schrammauthor=J.+D.+Boeke&title=The+biochemistry+of+sirtuins&doi=10.1146%2Fannurev.biochem.74.082803.133500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The biochemistry of sirtuins</span></div><div class="casAuthors">Sauve, Anthony A.; Wolberger, Cynthia; Schramm, Vern L.; Boeke, Jef D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">435-465</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Sirtuins are a family of NAD-dependent protein deacetylases widely distributed in all phyla of life.  Accumulating evidence indicates that sirtuins are important regulators of organism life span.  In yeast, these unique enzymes regulate gene silencing by histone deacetylation and via formation of the novel compd., 2'-O-acetyl-ADP-ribose.  In multicellular organisms, sirtuins deacetylate histones and transcription factors that regulate stress, metab., and survival pathways.  The chem. mechanism of sirtuins provides novel opportunities for signaling and metabolic regulation of protein deacetylation.  The biol., chem., and structural characteristics of these unusual enzymes are discussed in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorq1GGNz7SSrVg90H21EOLACvtfcHk0liU-MPiSgihEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosVKhs7g%253D&md5=af3594af9dc2df013e4d1970f8ef7433</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.74.082803.133500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.74.082803.133500%26sid%3Dliteratum%253Aachs%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DWolberger%26aufirst%3DC.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26aulast%3DBoeke%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520biochemistry%2520of%2520sirtuins%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2006%26volume%3D75%26spage%3D435%26epage%3D465%26doi%3D10.1146%2Fannurev.biochem.74.082803.133500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chalkiadaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarente, L.</span></span> <span> </span><span class="NLM_article-title">The multifaceted functions of sirtuins in cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrc3985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnrc3985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26383140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFeisb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=608-624&author=A.+Chalkiadakiauthor=L.+Guarente&title=The+multifaceted+functions+of+sirtuins+in+cancer&doi=10.1038%2Fnrc3985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The multifaceted functions of sirtuins in cancer</span></div><div class="casAuthors">Chalkiadaki, Angeliki; Guarente, Leonard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The sirtuins (SIRTs; of which there are seven in mammals) are NAD+-dependent enzymes that regulate a large no. of cellular pathways and forestall the progression of ageing and age-assocd. diseases.  In recent years, the role of sirtuins in cancer biol. has become increasingly apparent, and growing evidence demonstrates that sirtuins regulate many processes that go awry in cancer cells, such as cellular metab., the regulation of chromatin structure and the maintenance of genomic stability.  In this article, we review recent advances in our understanding of how sirtuins affect cancer metab., DNA repair and the tumor microenvironment and how activating or inhibiting sirtuins may be important in preventing or treating cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpblQk6T-KatbVg90H21EOLACvtfcHk0lhAIsjS7mzhyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFeisb7L&md5=264b4e0b86556f0b72db33110ee3e2f5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrc3985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3985%26sid%3Dliteratum%253Aachs%26aulast%3DChalkiadaki%26aufirst%3DA.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DThe%2520multifaceted%2520functions%2520of%2520sirtuins%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrc3985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alders, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cetinbas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pajkrt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Koningsbruggen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christoffels, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannens, M. M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knegt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Etchegaray, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadreyev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Maarle, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">An inactivating mutation in the histone deacetylase SIRT6 causes human perinatal lethality</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1101/gad.307330.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1101%2Fgad.307330.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=29555651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsVaisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=373-388&author=C.+M.+Ferrerauthor=M.+Aldersauthor=A.+V.+Postmaauthor=S.+Parkauthor=M.+A.+Kleinauthor=M.+Cetinbasauthor=E.+Pajkrtauthor=A.+Glasauthor=S.+van+Koningsbruggenauthor=V.+M.+Christoffelsauthor=M.+M.+A.+M.+Mannensauthor=L.+Knegtauthor=J.-P.+Etchegarayauthor=R.+I.+Sadreyevauthor=J.+M.+Denuauthor=G.+Mostoslavskyauthor=M.+C.+van+Maarleauthor=R.+Mostoslavsky&title=An+inactivating+mutation+in+the+histone+deacetylase+SIRT6+causes+human+perinatal+lethality&doi=10.1101%2Fgad.307330.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">An inactivating mutation in the histone deacetylase SIRT6 causes human perinatal lethality</span></div><div class="casAuthors">Ferrer, Christina M.; Alders, Marielle; Postma, Alex V.; Park, Seonmi; Klein, Mark A.; Cetinbas, Murat; Pajkrt, Eva; Glas, Astrid; van Koningsbruggen, Silvana; Christoffels, Vincent M.; Mannens, Marcel M. A. M.; Knegt, Lia; Etchegaray, Jean-Pierre; Sadreyev, Ruslan I.; Denu, John M.; Mostoslavsky, Gustavo; van Maarle, Merel C.; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">373-388</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">It has been well established that histone and DNA modifications are crit. to maintaining the equil. between pluripotency and differentiation during early embryogenesis.  Mutations in key regulators of DNA methylation have shown that the balance between gene regulation and function is crit. during neural development in early years of life.  However, there have been no identified cases linking epigenetic regulators to aberrant human development and fetal demise.  Here, we demonstrate that a homozygous inactivating mutation in the histone deacetylase SIRT6 results in severe congenital anomalies and perinatal lethality in four affected fetuses.  In vitro, the amino acid change at Asp63 to a histidine results in virtually complete loss of H3K9 deacetylase and demyristoylase functions.  Functionally, SIRT6 D63H mouse embryonic stem cells (mESCs) fail to repress pluripotent gene expression, direct targets of SIRT6, and exhibit an even more severe phenotype than Sirt6-deficient ESCs when differentiated into embryoid bodies (EBs).  When terminally differentiated toward cardiomyocyte lineage, D63H mutant mESCs maintain expression of pluripotent genes and fail to form functional cardiomyocyte foci.  Last, human induced pluripotent stem cells (iPSCs) derived from D63H homozygous fetuses fail to differentiate into EBs, functional cardiomyocytes, and neural progenitor cells due to a failure to repress pluripotent genes.  Altogether, our study described a germline mutation in SIRT6 as a cause for fetal demise, defining SIRT6 as a key factor in human development and identifying the first mutation in a chromatin factor behind a human syndrome of perinatal lethality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXdn5zBcYq27Vg90H21EOLACvtfcHk0lhAIsjS7mzhyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsVaisbs%253D&md5=ea556b4ad909cdef66291247d0696c6b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1101%2Fgad.307330.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.307330.117%26sid%3Dliteratum%253Aachs%26aulast%3DFerrer%26aufirst%3DC.%2BM.%26aulast%3DAlders%26aufirst%3DM.%26aulast%3DPostma%26aufirst%3DA.%2BV.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DM.%2BA.%26aulast%3DCetinbas%26aufirst%3DM.%26aulast%3DPajkrt%26aufirst%3DE.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3Dvan%2BKoningsbruggen%26aufirst%3DS.%26aulast%3DChristoffels%26aufirst%3DV.%2BM.%26aulast%3DMannens%26aufirst%3DM.%2BM.%2BA.%2BM.%26aulast%3DKnegt%26aufirst%3DL.%26aulast%3DEtchegaray%26aufirst%3DJ.-P.%26aulast%3DSadreyev%26aufirst%3DR.%2BI.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26aulast%3DMostoslavsky%26aufirst%3DG.%26aulast%3Dvan%2BMaarle%26aufirst%3DM.%2BC.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DAn%2520inactivating%2520mutation%2520in%2520the%2520histone%2520deacetylase%2520SIRT6%2520causes%2520human%2520perinatal%2520lethality%26jtitle%3DGenes%2520Dev.%26date%3D2018%26volume%3D32%26spage%3D373%26epage%3D388%26doi%3D10.1101%2Fgad.307330.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Meter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seluanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbunova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span> <span> </span><span class="NLM_article-title">SIRT6 rescues the age related decline in base excision repair in a PARP1-dependent manner</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.4161/15384101.2014.980641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.4161%2F15384101.2014.980641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=25607651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVKrtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=269-276&author=Z.+Xuauthor=L.+Zhangauthor=W.+Zhangauthor=D.+Mengauthor=H.+Zhangauthor=Y.+Jiangauthor=X.+Xuauthor=M.+Van+Meterauthor=A.+Seluanovauthor=V.+Gorbunovaauthor=Z.+Mao&title=SIRT6+rescues+the+age+related+decline+in+base+excision+repair+in+a+PARP1-dependent+manner&doi=10.4161%2F15384101.2014.980641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 rescues the age related decline in base excision repair in a PARP1-dependent manner</span></div><div class="casAuthors">Xu, Zhu; Zhang, Lei; Zhang, Wenjun; Meng, Du; Zhang, Hongxia; Jiang, Ying; Xu, Xiaojun; Van Meter, Michael; Seluanov, Andrei; Gorbunova, Vera; Mao, Zhiyong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-276</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1551-4005</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">In principle, a decline in base excision repair (BER) efficiency with age should lead to genomic instability and ultimately contribute to the onset of the aging phenotype.  Although multiple studies have indicated a neg. link between aging and BER, the change of BER efficiency with age in humans has not been systematically analyzed.  Here, with foreskin fibroblasts isolated from 19 donors between 20 and 64 y of age, we report a significant decline of BER efficiency with age using a newly developed GFP reactivation assay.  We further obsd. a very strong neg. correlation between age and the expression levels of SIRT6, a factor which is known to maintain genomic integrity by improving DNA double strand break (DSB) repair.  Our mechanistic study suggests that, similar to the regulatory role that SIRT6 plays in DNA DSB repair, SIRT6 regulates BER in a PARP1-dependent manner.  Moreover, overexpression of SIRT6 rescues the decline of BER in aged fibroblasts.  In summary, our results uncovered the regulatory mechanisms of BER by SIRT6, suggesting that SIRT6 reactivation in aging tissues may help delay the process of aging through improving BER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra44iHCz1gIbVg90H21EOLACvtfcHk0lhAIsjS7mzhyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVKrtb7E&md5=c3bc7a61000bbfe34748efe2219575fd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4161%2F15384101.2014.980641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15384101.2014.980641%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMeng%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DVan%2BMeter%26aufirst%3DM.%26aulast%3DSeluanov%26aufirst%3DA.%26aulast%3DGorbunova%26aufirst%3DV.%26aulast%3DMao%26aufirst%3DZ.%26atitle%3DSIRT6%2520rescues%2520the%2520age%2520related%2520decline%2520in%2520base%2520excision%2520repair%2520in%2520a%2520PARP1-dependent%2520manner%26jtitle%3DCell%2520Cycle%26date%3D2015%26volume%3D14%26spage%3D269%26epage%3D276%26doi%3D10.4161%2F15384101.2014.980641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span> <span> </span><span class="NLM_article-title">SIRT6 deficiency results in developmental retardation in cynomolgus monkeys</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>560</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">665</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0437-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fs41586-018-0437-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=30135584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFOgsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=560&publication_year=2018&pages=661-665&author=W.+Zhangauthor=H.+Wanauthor=G.+Fengauthor=J.+Quauthor=J.+Wangauthor=Y.+Jingauthor=R.+Renauthor=Z.+Liuauthor=L.+Zhangauthor=Z.+Chenauthor=S.+Wangauthor=Y.+Zhaoauthor=Z.+Wangauthor=Y.+Yuanauthor=Q.+Zhouauthor=W.+Liauthor=G.-H.+Liuauthor=B.+Hu&title=SIRT6+deficiency+results+in+developmental+retardation+in+cynomolgus+monkeys&doi=10.1038%2Fs41586-018-0437-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 deficiency results in developmental retardation in cynomolgus monkeys</span></div><div class="casAuthors">Zhang, Weiqi; Wan, Haifeng; Feng, Guihai; Qu, Jing; Wang, Jiaqiang; Jing, Yaobin; Ren, Ruotong; Liu, Zunpeng; Zhang, Linlin; Chen, Zhiguo; Wang, Shuyan; Zhao, Yong; Wang, Zhaoxia; Yuan, Yun; Zhou, Qi; Li, Wei; Liu, Guang-Hui; Hu, Baoyang</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">560</span>
        (<span class="NLM_cas:issue">7720</span>),
    <span class="NLM_cas:pages">661-665</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">SIRT6 acts as a longevity protein in rodents1,2.  However, its biol. function in primates remains largely unknown.  Here we generate a SIRT6-null cynomolgus monkey (Macaca fascicularis) model using a CRISPR-Cas9-based approach.  SIRT6-deficient monkeys die hours after birth and exhibit severe prenatal developmental retardation.  SIRT6 loss delays neuronal differentiation by transcriptionally activating the long non-coding RNA H19 (a developmental repressor), and we were able to recapitulate this process in a human neural progenitor cell differentiation system.  SIRT6 deficiency results in histone hyperacetylation at the imprinting control region of H19, CTCF recruitment and upregulation of H19.  Our results suggest that SIRT6 is involved in regulating development in non-human primates, and may provide mechanistic insight into human perinatal lethality syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN2SKsLb0dF7Vg90H21EOLACvtfcHk0lijkI9MPS6d4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFOgsLrI&md5=2d28ce353a9c1bf941ebb94e2daa5a45</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0437-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0437-z%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DWan%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DQu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DJing%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DG.-H.%26aulast%3DHu%26aufirst%3DB.%26atitle%3DSIRT6%2520deficiency%2520results%2520in%2520developmental%2520retardation%2520in%2520cynomolgus%2520monkeys%26jtitle%3DNature%26date%3D2018%26volume%3D560%26spage%3D661%26epage%3D665%26doi%3D10.1038%2Fs41586-018-0437-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hine, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Meter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaidya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seluanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbunova, V.</span></span> <span> </span><span class="NLM_article-title">SIRT6 promotes DNA repair under stress by activating PARP1</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>332</i></span>,  <span class="NLM_fpage">1443</span>– <span class="NLM_lpage">1446</span>, <span class="refDoi"> DOI: 10.1126/science.1202723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1126%2Fscience.1202723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=21680843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntlOltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2011&pages=1443-1446&author=Z.+Maoauthor=C.+Hineauthor=X.+Tianauthor=M.+Van+Meterauthor=M.+Auauthor=A.+Vaidyaauthor=A.+Seluanovauthor=V.+Gorbunova&title=SIRT6+promotes+DNA+repair+under+stress+by+activating+PARP1&doi=10.1126%2Fscience.1202723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 promotes DNA repair under stress by activating PARP1</span></div><div class="casAuthors">Mao, Zhiyong; Hine, Christopher; Tian, Xiao; Van Meter, Michael; Au, Matthew; Vaidya, Amita; Seluanov, Andrei; Gorbunova, Vera</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">6036</span>),
    <span class="NLM_cas:pages">1443-1446</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Sirtuin 6 (SIRT6) is a mammalian homolog of the yeast Sir2 deacetylase.  Mice deficient for SIRT6 exhibit genome instability.  Here, we show that in mammalian cells subjected to oxidative stress SIRT6 is recruited to the sites of DNA double-strand breaks (DSBs) and stimulates DSB repair, through both nonhomologous end joining and homologous recombination.  Our results indicate that SIRT6 phys. assocs. with poly[ADP-ribose] polymerase 1 (PARP1) and mono-ADP-ribosylates PARP1 on lysine residue 521, thereby stimulating PARP1 poly-ADP-ribosylase activity and enhancing DSB repair under oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1cokFt_HuDbVg90H21EOLACvtfcHk0lijkI9MPS6d4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntlOltLg%253D&md5=3cf0f8ea3d92894f8217902197ce5500</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1202723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1202723%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DHine%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DVan%2BMeter%26aufirst%3DM.%26aulast%3DAu%26aufirst%3DM.%26aulast%3DVaidya%26aufirst%3DA.%26aulast%3DSeluanov%26aufirst%3DA.%26aulast%3DGorbunova%26aufirst%3DV.%26atitle%3DSIRT6%2520promotes%2520DNA%2520repair%2520under%2520stress%2520by%2520activating%2520PARP1%26jtitle%3DScience%26date%3D2011%26volume%3D332%26spage%3D1443%26epage%3D1446%26doi%3D10.1126%2Fscience.1202723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span> <span> </span><span class="NLM_article-title">Human SIRT6 promotes DNA end resection through CtIP deacetylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>329</i></span>,  <span class="NLM_fpage">1348</span>– <span class="NLM_lpage">1353</span>, <span class="refDoi"> DOI: 10.1126/science.1192049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1126%2Fscience.1192049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=20829486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFajs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=329&publication_year=2010&pages=1348-1353&author=A.+Kaidiauthor=B.+T.+Weinertauthor=C.+Choudharyauthor=S.+P.+Jackson&title=Human+SIRT6+promotes+DNA+end+resection+through+CtIP+deacetylation&doi=10.1126%2Fscience.1192049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Human SIRT6 Promotes DNA End Resection Through CtIP Deacetylation</span></div><div class="casAuthors">Kaidi, Abderrahmane; Weinert, Brian T.; Choudhary, Chunaram; Jackson, Stephen P.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">329</span>
        (<span class="NLM_cas:issue">5997</span>),
    <span class="NLM_cas:pages">1348-1353</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SIRT6 belongs to the sirtuin family of protein lysine deacetylases, which regulate aging and genome stability.  We found that human SIRT6 has a role in promoting DNA end resection, a crucial step in DNA double-strand break (DSB) repair by homologous recombination.  SIRT6 depletion impaired the accumulation of replication protein A (RPA) and single-stranded DNA (ssDNA) at DNA damage sites, reduced rates of homologous recombination, and sensitized cells to DSB-inducing agents.  We identified the DSB resection protein CtIP [C-terminal binding protein (CtBP) interacting protein] as a SIRT6 interaction partner and showed that SIRT6-dependent CtIP deacetylation promotes resection.  A nonacetylatable CtIP mutant alleviated the effect of SIRT6 depletion on resection, thus identifying CtIP as a key substrate by which SIRT6 facilitates DSB processing and homologous recombination.  These findings further clarify how SIRT6 promotes genome stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqYddumzd0v7Vg90H21EOLACvtfcHk0lijkI9MPS6d4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFajs7rF&md5=72055f022a4f82adfd8121c10b4fdc8c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1192049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1192049%26sid%3Dliteratum%253Aachs%26aulast%3DKaidi%26aufirst%3DA.%26aulast%3DWeinert%26aufirst%3DB.%2BT.%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26atitle%3DHuman%2520SIRT6%2520promotes%2520DNA%2520end%2520resection%2520through%2520CtIP%2520deacetylation%26jtitle%3DScience%26date%3D2010%26volume%3D329%26spage%3D1348%26epage%3D1353%26doi%3D10.1126%2Fscience.1192049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frendewey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auerbach, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenzuela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hottiger, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hursting, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarente, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demple, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yancopoulos, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, F. W.</span></span> <span> </span><span class="NLM_article-title">Genomic instability and aging-like phenotype in the absence of mammalian SIRT6</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2005.11.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2005.11.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=16439206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1KktLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=315-329&author=R.+Mostoslavskyauthor=K.+F.+Chuaauthor=D.+B.+Lombardauthor=W.+W.+Pangauthor=M.+R.+Fischerauthor=L.+Gellonauthor=P.+Liuauthor=G.+Mostoslavskyauthor=S.+Francoauthor=M.+M.+Murphyauthor=K.+D.+Millsauthor=P.+Patelauthor=J.+T.+Hsuauthor=A.+L.+Hongauthor=E.+Fordauthor=H.-L.+Chengauthor=C.+Kennedyauthor=N.+Nunezauthor=R.+Bronsonauthor=D.+Frendeweyauthor=W.+Auerbachauthor=D.+Valenzuelaauthor=M.+Karowauthor=M.+O.+Hottigerauthor=S.+Hurstingauthor=J.+C.+Barrettauthor=L.+Guarenteauthor=R.+Mulliganauthor=B.+Dempleauthor=G.+D.+Yancopoulosauthor=F.+W.+Alt&title=Genomic+instability+and+aging-like+phenotype+in+the+absence+of+mammalian+SIRT6&doi=10.1016%2Fj.cell.2005.11.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic instability and aging-like phenotype in the absence of mammalian SIRT6</span></div><div class="casAuthors">Mostoslavsky, Raul; Chua, Katrin F.; Lombard, David B.; Pang, Wendy W.; Fischer, Miriam R.; Gellon, Lionel; Liu, Pingfang; Mostoslavsky, Gustavo; Franco, Sonia; Murphy, Michael M.; Mills, Kevin D.; Patel, Parin; Hsu, Joyce T.; Hong, Andrew L.; Ford, Ethan; Cheng, Hwei-Ling; Kennedy, Caitlin; Nunez, Nomeli; Bronson, Roderick; Frendewey, David; Auerbach, Wojtek; Valenzuela, David; Karow, Margaret; Hottiger, Michael O.; Hursting, Stephen; Barrett, J. Carl; Guarente, Leonard; Mulligan, Richard; Demple, Bruce; Yancopoulos, George D.; Alt, Frederick W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">315-329</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Sir2 histone deacetylase functions as a chromatin silencer to regulate recombination, genomic stability, and aging in budding yeast.  Seven mammalian Sir2 homologs have been identified (SIRT1-SIRT7), and it has been speculated that some may have similar functions to Sir2.  Here, we demonstrate that SIRT6 is a nuclear, chromatin-assocd. protein that promotes resistance to DNA damage and suppresses genomic instability in mouse cells, in assocn. with a role in base excision repair (BER).  SIRT6-deficient mice are small and at 2-3 wk of age develop abnormalities that include profound lymphopenia, loss of s.c. fat, lordokyphosis, and severe metabolic defects, eventually dying at about 4 wk.  We conclude that one function of SIRT6 is to promote normal DNA repair, and that SIRT6 loss leads to abnormalities in mice that overlap with aging-assocd. degenerative processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-iTboO5w_ObVg90H21EOLACvtfcHk0lgkRfzB-II-HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1KktLc%253D&md5=14971074ed55f5382960b500370ecff2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.11.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.11.044%26sid%3Dliteratum%253Aachs%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DPang%26aufirst%3DW.%2BW.%26aulast%3DFischer%26aufirst%3DM.%2BR.%26aulast%3DGellon%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DMostoslavsky%26aufirst%3DG.%26aulast%3DFranco%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DM.%2BM.%26aulast%3DMills%26aufirst%3DK.%2BD.%26aulast%3DPatel%26aufirst%3DP.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26aulast%3DHong%26aufirst%3DA.%2BL.%26aulast%3DFord%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DH.-L.%26aulast%3DKennedy%26aufirst%3DC.%26aulast%3DNunez%26aufirst%3DN.%26aulast%3DBronson%26aufirst%3DR.%26aulast%3DFrendewey%26aufirst%3DD.%26aulast%3DAuerbach%26aufirst%3DW.%26aulast%3DValenzuela%26aufirst%3DD.%26aulast%3DKarow%26aufirst%3DM.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26aulast%3DHursting%26aufirst%3DS.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DGuarente%26aufirst%3DL.%26aulast%3DMulligan%26aufirst%3DR.%26aulast%3DDemple%26aufirst%3DB.%26aulast%3DYancopoulos%26aufirst%3DG.%2BD.%26aulast%3DAlt%26aufirst%3DF.%2BW.%26atitle%3DGenomic%2520instability%2520and%2520aging-like%2520phenotype%2520in%2520the%2520absence%2520of%2520mammalian%2520SIRT6%26jtitle%3DCell%26date%3D2006%26volume%3D124%26spage%3D315%26epage%3D329%26doi%3D10.1016%2Fj.cell.2005.11.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michishita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCord, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kioi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla-Nash, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusumoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawahara, T. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ried, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span> <span> </span><span class="NLM_article-title">SIRT6 is a histone H3 lysine 9 deacetylase that modulates telomeric chromatin</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>452</i></span>,  <span class="NLM_fpage">492</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/nature06736</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnature06736" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=18337721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjslCnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=452&publication_year=2008&pages=492-496&author=E.+Michishitaauthor=R.+A.+McCordauthor=E.+Berberauthor=M.+Kioiauthor=H.+Padilla-Nashauthor=M.+Damianauthor=P.+Cheungauthor=R.+Kusumotoauthor=T.+L.+A.+Kawaharaauthor=J.+C.+Barrettauthor=H.+Y.+Changauthor=V.+A.+Bohrauthor=T.+Riedauthor=O.+Gozaniauthor=K.+F.+Chua&title=SIRT6+is+a+histone+H3+lysine+9+deacetylase+that+modulates+telomeric+chromatin&doi=10.1038%2Fnature06736"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 is a histone H3 lysine 9 deacetylase that modulates telomeric chromatin</span></div><div class="casAuthors">Michishita, Eriko; McCord, Ronald A.; Berber, Elisabeth; Kioi, Mitomu; Padilla-Nash, Hesed; Damian, Mara; Cheung, Peggie; Kusumoto, Rika; Kawahara, Tiara L. A.; Barrett, J. Carl; Chang, Howard Y.; Bohr, Vilhelm A.; Ried, Thomas; Gozani, Or; Chua, Katrin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">452</span>
        (<span class="NLM_cas:issue">7186</span>),
    <span class="NLM_cas:pages">492-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Sir2 deacetylase regulates chromatin silencing and lifespan in Saccharomyces cerevisiae.  In mice, deficiency for the Sir2 family member SIRT6 leads to a shortened lifespan and a premature aging-like phenotype.  However, the mol. mechanisms of SIRT6 function are unclear.  SIRT6 is a chromatin-assocd. protein, but no enzymic activity of SIRT6 at chromatin has yet been detected, and the identity of physiol. SIRT6 substrates is unknown.  Here we show that the human SIRT6 protein is an NAD+-dependent, histone H3 lysine 9 (H3K9) deacetylase that modulates telomeric chromatin.  SIRT6 assocs. specifically with telomeres, and SIRT6 depletion leads to telomere dysfunction with end-to-end chromosomal fusions and premature cellular senescence.  Moreover, SIRT6-depleted cells exhibit abnormal telomere structures that resemble defects obsd. in Werner syndrome, a premature aging disorder.  At telomeric chromatin, SIRT6 deacetylates H3K9 and is required for the stable assocn. of WRN, the factor that is mutated in Werner syndrome.  We propose that SIRT6 contributes to the propagation of a specialized chromatin state at mammalian telomeres, which in turn is required for proper telomere metab. and function.  Our findings constitute the first identification of a physiol. enzymic activity of SIRT6, and link chromatin regulation by SIRT6 to telomere maintenance and a human premature aging syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYo1SMIJbbV7Vg90H21EOLACvtfcHk0lgkRfzB-II-HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjslCnurs%253D&md5=5aa9f5b39be29fa580012b718943a3d7</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature06736&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06736%26sid%3Dliteratum%253Aachs%26aulast%3DMichishita%26aufirst%3DE.%26aulast%3DMcCord%26aufirst%3DR.%2BA.%26aulast%3DBerber%26aufirst%3DE.%26aulast%3DKioi%26aufirst%3DM.%26aulast%3DPadilla-Nash%26aufirst%3DH.%26aulast%3DDamian%26aufirst%3DM.%26aulast%3DCheung%26aufirst%3DP.%26aulast%3DKusumoto%26aufirst%3DR.%26aulast%3DKawahara%26aufirst%3DT.%2BL.%2BA.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DChang%26aufirst%3DH.%2BY.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DRied%26aufirst%3DT.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DChua%26aufirst%3DK.%2BF.%26atitle%3DSIRT6%2520is%2520a%2520histone%2520H3%2520lysine%25209%2520deacetylase%2520that%2520modulates%2520telomeric%2520chromatin%26jtitle%3DNature%26date%3D2008%26volume%3D452%26spage%3D492%26epage%3D496%26doi%3D10.1038%2Fnature06736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sebastián, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwaans, B. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silberman, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gymrek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ram, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truelove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toiber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosentino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenson, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlynn, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacosa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiels, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">The histone deacetylase SIRT6 is a tumor suppressor that controls cancer metabolism</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1199</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.10.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2012.10.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23217706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaisr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2012&pages=1185-1199&author=C.+Sebasti%C3%A1nauthor=B.+M.+M.+Zwaansauthor=D.+M.+Silbermanauthor=M.+Gymrekauthor=A.+Gorenauthor=L.+Zhongauthor=O.+Ramauthor=J.+Trueloveauthor=A.+R.+Guimaraesauthor=D.+Toiberauthor=C.+Cosentinoauthor=J.+K.+Greensonauthor=A.+I.+MacDonaldauthor=L.+McGlynnauthor=F.+Maxwellauthor=J.+Edwardsauthor=S.+Giacosaauthor=E.+Guccioneauthor=R.+Weisslederauthor=B.+E.+Bernsteinauthor=A.+Regevauthor=P.+G.+Shielsauthor=D.+B.+Lombardauthor=R.+Mostoslavsky&title=The+histone+deacetylase+SIRT6+is+a+tumor+suppressor+that+controls+cancer+metabolism&doi=10.1016%2Fj.cell.2012.10.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The histone deacetylase SIRT6 is a tumor suppressor that controls cancer metabolism</span></div><div class="casAuthors">Sebastian, Carlos; Zwaans, Bernadette M. M.; Silberman, Dafne M.; Gymrek, Melissa; Goren, Alon; Zhong, Lei; Ram, Oren; Truelove, Jessica; Guimaraes, Alexander R.; Toiber, Debra; Cosentino, Claudia; Greenson, Joel K.; MacDonald, Alasdair I.; McGlynn, Liane; Maxwell, Fraser; Edwards, Joanne; Giacosa, Sofia; Guccione, Ernesto; Weissleder, Ralph; Bernstein, Bradley E.; Regev, Aviv; Shiels, Paul G.; Lombard, David B.; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1185-1199</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Reprogramming of cellular metab. is a key event during tumorigenesis.  Despite being known for decades (Warburg effect), the mol. mechanisms regulating this switch remained unexplored.  Here, we identify SIRT6 as a tumor suppressor that regulates aerobic glycolysis in cancer cells.  Importantly, loss of SIRT6 leads to tumor formation without activation of known oncogenes, whereas transformed SIRT6-deficient cells display increased glycolysis and tumor growth, suggesting that SIRT6 plays a role in both establishment and maintenance of cancer.  By using a conditional SIRT6 allele, we show that SIRT6 deletion in vivo increases the no., size, and aggressiveness of tumors.  SIRT6 also functions as a regulator of ribosome metab. by corepressing MYC transcriptional activity.  Lastly, Sirt6 is selectively downregulated in several human cancers, and expression levels of SIRT6 predict prognosis and tumor-free survival rates, highlighting SIRT6 as a crit. modulator of cancer metab.  Our studies reveal SIRT6 to be a potent tumor suppressor acting to suppress cancer metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6UVcycnyph7Vg90H21EOLACvtfcHk0lgkRfzB-II-HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaisr3J&md5=bb6df5c1d96df3ee5809773f91bb7ef5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.10.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.10.047%26sid%3Dliteratum%253Aachs%26aulast%3DSebasti%25C3%25A1n%26aufirst%3DC.%26aulast%3DZwaans%26aufirst%3DB.%2BM.%2BM.%26aulast%3DSilberman%26aufirst%3DD.%2BM.%26aulast%3DGymrek%26aufirst%3DM.%26aulast%3DGoren%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DRam%26aufirst%3DO.%26aulast%3DTruelove%26aufirst%3DJ.%26aulast%3DGuimaraes%26aufirst%3DA.%2BR.%26aulast%3DToiber%26aufirst%3DD.%26aulast%3DCosentino%26aufirst%3DC.%26aulast%3DGreenson%26aufirst%3DJ.%2BK.%26aulast%3DMacDonald%26aufirst%3DA.%2BI.%26aulast%3DMcGlynn%26aufirst%3DL.%26aulast%3DMaxwell%26aufirst%3DF.%26aulast%3DEdwards%26aufirst%3DJ.%26aulast%3DGiacosa%26aufirst%3DS.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DWeissleder%26aufirst%3DR.%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DShiels%26aufirst%3DP.%2BG.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DThe%2520histone%2520deacetylase%2520SIRT6%2520is%2520a%2520tumor%2520suppressor%2520that%2520controls%2520cancer%2520metabolism%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D1185%26epage%3D1199%26doi%3D10.1016%2Fj.cell.2012.10.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baeza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span> <span> </span><span class="NLM_article-title">Activation of the protein deacetylase SIRT6 by long-chain fatty acids and widespread deacylation by mammalian sirtuins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">31350</span>– <span class="NLM_lpage">31356</span>, <span class="refDoi"> DOI: 10.1074/jbc.c113.511261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1074%2Fjbc.c113.511261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=24052263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1yiur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=31350-31356&author=J.+L.+Feldmanauthor=J.+Baezaauthor=J.+M.+Denu&title=Activation+of+the+protein+deacetylase+SIRT6+by+long-chain+fatty+acids+and+widespread+deacylation+by+mammalian+sirtuins&doi=10.1074%2Fjbc.c113.511261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the protein deacetylase SIRT6 by long-chain fatty acids and widespread deacylation by mammalian sirtuins</span></div><div class="casAuthors">Feldman, Jessica L.; Baeza, Josue; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">31350-31356</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mammalian sirtuins (SIRT1 through SIRT7) are members of a highly conserved family of NAD-dependent protein deacetylases that function in metab., genome maintenance, and stress responses.  Emerging evidence suggests that some sirtuins display substrate specificity toward other acyl groups attached to the lysine ε-amine.  SIRT6 was recently reported to preferentially hydrolyze long-chain fatty acyl groups over acetyl groups.  Here, the authors investigated the catalytic ability of all sirtuins to hydrolyze 13 different acyl groups from histone H3 peptides, ranging in carbon length, satn., and chem. diversity.  The authors found that long-chain deacylation is a general feature of mammalian sirtuins, that SIRT1 and SIRT2 act as efficient decrotonylases, and that SIRT1, SIRT2, SIRT3, and SIRT4 can remove lipoic acid.  These results provided new insights into sirtuin function and a means for cellular removal of an expanding list of endogenous lysine modifications.  Given that SIRT6 is a poor deacetylase in vitro, but binds and prefers to hydrolyze long-chain acylated peptides, the authors hypothesized that binding of certain free fatty acids (FFAs) could stimulate deacetylation activity.  Indeed, the authors demonstrated that several biol. relevant FFAs (including myristic, oleic, and linoleic acids) at physiol. concns. induced up to a 35-fold increase in catalytic efficiency of SIRT6 but not SIRT1.  The activation mechanism was consistent with fatty acid inducing a conformation that binds acetylated H3 with greater affinity.  The binding of long-chain FFA and myristoylated H3 peptide was mutually exclusive.  The authors discussed the implications of discovering endogenous, small-mol. activators of SIRT6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0tBZ5loovPrVg90H21EOLACvtfcHk0li5uDcSn70zIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1yiur3M&md5=0bd3995be555fa210ff0bb5763e37681</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.c113.511261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.c113.511261%26sid%3Dliteratum%253Aachs%26aulast%3DFeldman%26aufirst%3DJ.%2BL.%26aulast%3DBaeza%26aufirst%3DJ.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DActivation%2520of%2520the%2520protein%2520deacetylase%2520SIRT6%2520by%2520long-chain%2520fatty%2520acids%2520and%2520widespread%2520deacylation%2520by%2520mammalian%2520sirtuins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D31350%26epage%3D31356%26doi%3D10.1074%2Fjbc.c113.511261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawahara, T. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michishita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adler, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCord, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ongaigui, K. C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span> <span> </span><span class="NLM_article-title">SIRT6 links histone H3 lysine 9 deacetylation to NF-κB-dependent gene expression and organismal life span</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2008.10.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2008.10.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=19135889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=62-74&author=T.+L.+A.+Kawaharaauthor=E.+Michishitaauthor=A.+S.+Adlerauthor=M.+Damianauthor=E.+Berberauthor=M.+Linauthor=R.+A.+McCordauthor=K.+C.+L.+Ongaiguiauthor=L.+D.+Boxerauthor=H.+Y.+Changauthor=K.+F.+Chua&title=SIRT6+links+histone+H3+lysine+9+deacetylation+to+NF-%CE%BAB-dependent+gene+expression+and+organismal+life+span&doi=10.1016%2Fj.cell.2008.10.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 links histone H3 lysine 9 deacetylation to NF-κB-dependent gene expression and organismal life span</span></div><div class="casAuthors">Kawahara, Tiara L. A.; Michishita, Eriko; Adler, Adam S.; Damian, Mara; Berber, Elisabeth; Lin, Meihong; McCord, Ron A.; Ongaigui, Kristine C. L.; Boxer, Lisa D.; Chang, Howard Y.; Chua, Katrin F.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-74</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Members of the sirtuin (SIRT) family of NAD-dependent deacetylases promote longevity in multiple organisms.  Deficiency of mammalian SIRT6 leads to shortened life span and an aging-like phenotype in mice, but the underlying mol. mechanisms are unclear.  Here we show that SIRT6 functions at chromatin to attenuate NF-κB signaling.  SIRT6 interacts with the NF-κB RELA subunit and deacetylates histone H3 lysine 9 (H3K9) at NF-κB target gene promoters.  In SIRT6-deficient cells, hyperacetylation of H3K9 at these target promoters is assocd. with increased RELA promoter occupancy and enhanced NF-κB-dependent modulation of gene expression, apoptosis, and cellular senescence.  Computational genomics analyses revealed increased activity of NF-κB-driven gene expression programs in multiple Sirt6-deficient tissues in vivo.  Moreover, haploinsufficiency of RelA rescues the early lethality and degenerative syndrome of Sirt6-deficient mice.  It has been proposed that SIRT6 attenuates NF-κB signaling via H3K9 deacetylation at chromatin, and hyperactive NF-κB signaling may contribute to premature and normal aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZDou29TSxW7Vg90H21EOLACvtfcHk0li5uDcSn70zIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOisbs%253D&md5=84c714f6faa67540c3ee7aeadc6526de</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.10.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.10.052%26sid%3Dliteratum%253Aachs%26aulast%3DKawahara%26aufirst%3DT.%2BL.%2BA.%26aulast%3DMichishita%26aufirst%3DE.%26aulast%3DAdler%26aufirst%3DA.%2BS.%26aulast%3DDamian%26aufirst%3DM.%26aulast%3DBerber%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DMcCord%26aufirst%3DR.%2BA.%26aulast%3DOngaigui%26aufirst%3DK.%2BC.%2BL.%26aulast%3DBoxer%26aufirst%3DL.%2BD.%26aulast%3DChang%26aufirst%3DH.%2BY.%26aulast%3DChua%26aufirst%3DK.%2BF.%26atitle%3DSIRT6%2520links%2520histone%2520H3%2520lysine%25209%2520deacetylation%2520to%2520NF-%25CE%25BAB-dependent%2520gene%2520expression%2520and%2520organismal%2520life%2520span%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D62%26epage%3D74%26doi%3D10.1016%2Fj.cell.2008.10.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanfi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naiman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amir, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peshti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar-Joseph, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, H. Y.</span></span> <span> </span><span class="NLM_article-title">The sirtuin SIRT6 regulates lifespan in male mice</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>483</i></span>,  <span class="NLM_fpage">218</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nature10815</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnature10815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=22367546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XislyntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=218-221&author=Y.+Kanfiauthor=S.+Naimanauthor=G.+Amirauthor=V.+Peshtiauthor=G.+Zinmanauthor=L.+Nahumauthor=Z.+Bar-Josephauthor=H.+Y.+Cohen&title=The+sirtuin+SIRT6+regulates+lifespan+in+male+mice&doi=10.1038%2Fnature10815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The sirtuin SIRT6 regulates lifespan in male mice</span></div><div class="casAuthors">Kanfi, Yariv; Naiman, Shoshana; Amir, Gail; Peshti, Victoria; Zinman, Guy; Nahum, Liat; Bar-Joseph, Ziv; Cohen, Haim Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7388</span>),
    <span class="NLM_cas:pages">218-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The significant increase in human lifespan during the past century confronts us with great medical challenges.  To meet these challenges, the mechanisms that det. healthy ageing must be understood and controlled.  Sirtuins are highly conserved deacetylases that have been shown to regulate lifespan in yeast, nematodes and fruitflies.  However, the role of sirtuins in regulating worm and fly lifespan has recently become controversial.  Moreover, the role of the seven mammalian sirtuins, SIRT1 to SIRT7 (homologues of the yeast sirtuin Sir2), in regulating lifespan is unclear.  Here we show that male, but not female, transgenic mice overexpressing Sirt6 (ref. 4) have a significantly longer lifespan than wild-type mice.  Gene expression anal. revealed significant differences between male Sirt6-transgenic mice and male wild-type mice: transgenic males displayed lower serum levels of insulin-like growth factor 1 (IGF1), higher levels of IGF-binding protein 1 and altered phosphorylation levels of major components of IGF1 signalling, a key pathway in the regulation of lifespan.  This study shows the regulation of mammalian lifespan by a sirtuin family member and has important therapeutic implications for age-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo08gRkOlbJDrVg90H21EOLACvtfcHk0lhxxQSUAVTJpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislyntbY%253D&md5=7f87716fd9e336558e1137081146f351</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature10815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10815%26sid%3Dliteratum%253Aachs%26aulast%3DKanfi%26aufirst%3DY.%26aulast%3DNaiman%26aufirst%3DS.%26aulast%3DAmir%26aufirst%3DG.%26aulast%3DPeshti%26aufirst%3DV.%26aulast%3DZinman%26aufirst%3DG.%26aulast%3DNahum%26aufirst%3DL.%26aulast%3DBar-Joseph%26aufirst%3DZ.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26atitle%3DThe%2520sirtuin%2520SIRT6%2520regulates%2520lifespan%2520in%2520male%2520mice%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D218%26epage%3D221%26doi%3D10.1038%2Fnature10815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firsanov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tombline, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldfarb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornwell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moody, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gladyshev, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seluanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbunova, V.</span></span> <span> </span><span class="NLM_article-title">SIRT6 is responsible for more efficient DNA double-strand break repair in long-lived species</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">638.e22</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.03.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2019.03.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=31002797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnslyhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=622-638.e22&author=X.+Tianauthor=D.+Firsanovauthor=Z.+Zhangauthor=Y.+Chengauthor=L.+Luoauthor=G.+Tomblineauthor=R.+Tanauthor=M.+Simonauthor=S.+Hendersonauthor=J.+Steffanauthor=A.+Goldfarbauthor=J.+Tamauthor=K.+Zhengauthor=A.+Cornwellauthor=A.+Johnsonauthor=J.-N.+Yangauthor=Z.+Maoauthor=B.+Mantaauthor=W.+Dangauthor=Z.+Zhangauthor=J.+Vijgauthor=A.+Wolfeauthor=K.+Moodyauthor=B.+K.+Kennedyauthor=D.+Bohmannauthor=V.+N.+Gladyshevauthor=A.+Seluanovauthor=V.+Gorbunova&title=SIRT6+is+responsible+for+more+efficient+DNA+double-strand+break+repair+in+long-lived+species&doi=10.1016%2Fj.cell.2019.03.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 Is Responsible for More Efficient DNA Double-Strand Break Repair in Long-Lived Species</span></div><div class="casAuthors">Tian, Xiao; Firsanov, Denis; Zhang, Zhihui; Cheng, Yang; Luo, Lingfeng; Tombline, Gregory; Tan, Ruiyue; Simon, Matthew; Henderson, Steven; Steffan, Janine; Goldfarb, Audrey; Tam, Jonathan; Zheng, Kitty; Cornwell, Adam; Johnson, Adam; Yang, Jiang-Nan; Mao, Zhiyong; Manta, Bruno; Dang, Weiwei; Zhang, Zhengdong; Vijg, Jan; Wolfe, Aaron; Moody, Kelsey; Kennedy, Brian K.; Bohmann, Dirk; Gladyshev, Vadim N.; Seluanov, Andrei; Gorbunova, Vera</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">622-638.e22</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">DNA repair has been hypothesized to be a longevity determinant, but the evidence for it is based largely on accelerated aging phenotypes of DNA repair mutants.  Here, using a panel of 18 rodent species with diverse lifespans, we show that more robust DNA double-strand break (DSB) repair, but not nucleotide excision repair (NER), co-evolves with longevity.  Evolution of NER, unlike DSB, is shaped primarily by sunlight exposure.  We further show that the capacity of the SIRT6 protein to promote DSB repair accounts for a major part of the variation in DSB repair efficacy between short- and long-lived species.  We dissected the mol. differences between a weak (mouse) and a strong (beaver) SIRT6 protein and identified five amino acid residues that are fully responsible for their differential activities.  Our findings demonstrate that DSB repair and SIRT6 have been optimized during the evolution of longevity, which provides new targets for anti-aging interventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWg7twatHGfLVg90H21EOLACvtfcHk0lhxxQSUAVTJpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnslyhtL8%253D&md5=27f36ed9062a64886f866c24c0c174e6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.03.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.03.043%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DX.%26aulast%3DFirsanov%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DTombline%26aufirst%3DG.%26aulast%3DTan%26aufirst%3DR.%26aulast%3DSimon%26aufirst%3DM.%26aulast%3DHenderson%26aufirst%3DS.%26aulast%3DSteffan%26aufirst%3DJ.%26aulast%3DGoldfarb%26aufirst%3DA.%26aulast%3DTam%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DCornwell%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DJ.-N.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DManta%26aufirst%3DB.%26aulast%3DDang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DVijg%26aufirst%3DJ.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DMoody%26aufirst%3DK.%26aulast%3DKennedy%26aufirst%3DB.%2BK.%26aulast%3DBohmann%26aufirst%3DD.%26aulast%3DGladyshev%26aufirst%3DV.%2BN.%26aulast%3DSeluanov%26aufirst%3DA.%26aulast%3DGorbunova%26aufirst%3DV.%26atitle%3DSIRT6%2520is%2520responsible%2520for%2520more%2520efficient%2520DNA%2520double-strand%2520break%2520repair%2520in%2520long-lived%2520species%26jtitle%3DCell%26date%3D2019%26volume%3D177%26spage%3D622%26epage%3D638.e22%26doi%3D10.1016%2Fj.cell.2019.03.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mook-Jung, I.</span></span> <span> </span><span class="NLM_article-title">p53-dependent SIRT6 expression protects Aβ42-induced DNA damage</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">25628</span>, <span class="refDoi"> DOI: 10.1038/srep25628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fsrep25628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=27156849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=25628&author=E.+S.+Jungauthor=H.+Choiauthor=H.+Songauthor=Y.+J.+Hwangauthor=A.+Kimauthor=H.+Ryuauthor=I.+Mook-Jung&title=p53-dependent+SIRT6+expression+protects+A%CE%B242-induced+DNA+damage&doi=10.1038%2Fsrep25628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">p53-dependent SIRT6 expression protects Aβ42-induced DNA damage</span></div><div class="casAuthors">Jung, Eun Sun; Choi, Hyunjung; Song, Hyundong; Hwang, Yu Jin; Kim, Ahbin; Ryu, Hoon; Inhee, Mook-Jung</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">25628</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is the most common type of dementia and age-related neurodegenerative disease.  Elucidating the cellular changes that occur during ageing is an important step towards understanding the pathogenesis and progression of neurodegenerative disorders.  SIRT6 is a member of the mammalian sirtuin family of anti-aging genes.  However, the relationship between SIRT6 and AD has not yet been elucidated.  Here, we report that SIRT6 protein expression levels are reduced in the brains of both the 5XFAD AD mouse model and AD patients.  Aβ42, a major component of senile plaques, decreases SIRT6 expression, and Aβ42-induced DNA damage is prevented by the overexpression of SIRT6 in HT22 mouse hippocampal neurons.  Also, there is a strong neg. correlation between Aβ42-induced DNA damage and p53 levels, a protein involved in DNA repair and apoptosis.  In addn., upregulation of p53 protein by Nutlin-3 prevents SIRT6 redn. and DNA damage induced by Aβ42.  Taken together, this study reveals that p53-dependent SIRT6 expression protects cells from Aβ42-induced DNA damage, making SIRT6 a promising new therapeutic target for the treatment of AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaThmGINJMprVg90H21EOLACvtfcHk0lg6R4ekgezMdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWlt7s%253D&md5=ef78344498ac9f200b3087f0bb4e094a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsrep25628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep25628%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DE.%2BS.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DHwang%26aufirst%3DY.%2BJ.%26aulast%3DKim%26aufirst%3DA.%26aulast%3DRyu%26aufirst%3DH.%26aulast%3DMook-Jung%26aufirst%3DI.%26atitle%3Dp53-dependent%2520SIRT6%2520expression%2520protects%2520A%25CE%25B242-induced%2520DNA%2520damage%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D25628%26doi%3D10.1038%2Fsrep25628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaluski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portillo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueberham, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toiber, D.</span></span> <span> </span><span class="NLM_article-title">Neuroprotective functions for the histone deacetylase SIRT6</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3052</span>– <span class="NLM_lpage">3062</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.celrep.2017.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=28355558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlt12ru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=3052-3062&author=S.+Kaluskiauthor=M.+Portilloauthor=A.+Besnardauthor=D.+Steinauthor=M.+Einavauthor=L.+Zhongauthor=U.+Ueberhamauthor=T.+Arendtauthor=R.+Mostoslavskyauthor=A.+Sahayauthor=D.+Toiber&title=Neuroprotective+functions+for+the+histone+deacetylase+SIRT6&doi=10.1016%2Fj.celrep.2017.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Neuroprotective Functions for the Histone Deacetylase SIRT6</span></div><div class="casAuthors">Kaluski, Shai; Portillo, Miguel; Besnard, Antoine; Stein, Daniel; Einav, Monica; Zhong, Lei; Ueberham, Uwe; Arendt, Thomas; Mostoslavsky, Raul; Sahay, Amar; Toiber, Debra</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3052-3062</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The histone deacetylase SIRT6 promotes DNA repair, but its activity declines with age with a concomitant accumulation of DNA damage.  Furthermore, SIRT6 knockout mice exhibit an accelerated aging phenotype and die prematurely.  Here, we report that brain-specific SIRT6-deficient mice survive but present behavioral defects with major learning impairments by 4 mo of age.  Moreover, the brains of these mice show increased signs of DNA damage, cell death, and hyperphosphorylated Tau-a crit. mark in several neurodegenerative diseases.  Mechanistically, SIRT6 regulates Tau protein stability and phosphorylation through increased activation of the kinase GSK3α/β.  Finally, SIRT6 mRNA and protein levels are reduced in patients with Alzheimer's disease.  Taken together, our results suggest that SIRT6 is crit. to maintain genomic stability in the brain and that its loss leads to toxic Tau stability and phosphorylation.  Therefore, SIRT6 and its downstream signaling could be targeted in Alzheimer's disease and age-related neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodlby_QkHa2bVg90H21EOLACvtfcHk0lg6R4ekgezMdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlt12ru78%253D&md5=949b4d56c910332ab0c823675afaa58d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DKaluski%26aufirst%3DS.%26aulast%3DPortillo%26aufirst%3DM.%26aulast%3DBesnard%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DD.%26aulast%3DEinav%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DUeberham%26aufirst%3DU.%26aulast%3DArendt%26aufirst%3DT.%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DSahay%26aufirst%3DA.%26aulast%3DToiber%26aufirst%3DD.%26atitle%3DNeuroprotective%2520functions%2520for%2520the%2520histone%2520deacetylase%2520SIRT6%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D18%26spage%3D3052%26epage%3D3062%26doi%3D10.1016%2Fj.celrep.2017.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Urso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toiber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vadysirisack, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinelli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wikstrom, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nir, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaitheesvaran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iliopoulos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurland, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dor, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirihai, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellisen, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">The histone deacetylase SIRT6 regulates glucose homeostasis via Hif1α</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">280</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.12.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2009.12.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=20141841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVWitLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2010&pages=280-293&author=L.+Zhongauthor=A.+D%E2%80%99Ursoauthor=D.+Toiberauthor=C.+Sebastianauthor=R.+E.+Henryauthor=D.+D.+Vadysirisackauthor=A.+Guimaraesauthor=B.+Marinelliauthor=J.+D.+Wikstromauthor=T.+Nirauthor=C.+B.+Clishauthor=B.+Vaitheesvaranauthor=O.+Iliopoulosauthor=I.+Kurlandauthor=Y.+Dorauthor=R.+Weisslederauthor=O.+S.+Shirihaiauthor=L.+W.+Ellisenauthor=J.+M.+Espinosaauthor=R.+Mostoslavsky&title=The+histone+deacetylase+SIRT6+regulates+glucose+homeostasis+via+Hif1%CE%B1&doi=10.1016%2Fj.cell.2009.12.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The histone deacetylase Sirt6 regulates glucose homeostasis via Hif1α</span></div><div class="casAuthors">Zhong, Lei; D'Urso, Agustina; Toiber, Debra; Sebastian, Carlos; Henry, Ryan E.; Vadysirisack, Douangsone D.; Guimaraes, Alexander; Marinelli, Brett; Wikstrom, Jakob D.; Nir, Tomer; Clish, Clary B.; Vaitheesvaran, Bhavapriya; Iliopoulos, Othon; Kurland, Irwin; Dor, Yuval; Weissleder, Ralph; Shirihai, Orian S.; Ellisen, Leif W.; Espinosa, Joaquin M.; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">280-293</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">SIRT6 is a member of a highly conserved family of NAD+-dependent deacetylases with various roles in metab., stress resistance, and life span.  SIRT6-deficient mice develop normally but succumb to a lethal hypoglycemia early in life; however, the mechanism underlying this hypoglycemia remained unclear.  Here, we demonstrate that SIRT6 functions as a histone H3K9 deacetylase to control the expression of multiple glycolytic genes.  Specifically, SIRT6 appears to function as a corepressor of the transcription factor Hif1α, a crit. regulator of nutrient stress responses.  Consistent with this notion, SIRT6-deficient cells exhibit increased Hif1α activity and show increased glucose uptake with upregulation of glycolysis and diminished mitochondrial respiration.  Our studies uncover a role for the chromatin factor SIRT6 as a master regulator of glucose homeostasis and may provide the basis for novel therapeutic approaches against metabolic diseases, such as diabetes and obesity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxNnBUuYtLELVg90H21EOLACvtfcHk0lixQJKcLRZDiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVWitLc%253D&md5=a6cef44265a7bbbc86343484b18b5e43</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.12.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.12.041%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DD%25E2%2580%2599Urso%26aufirst%3DA.%26aulast%3DToiber%26aufirst%3DD.%26aulast%3DSebastian%26aufirst%3DC.%26aulast%3DHenry%26aufirst%3DR.%2BE.%26aulast%3DVadysirisack%26aufirst%3DD.%2BD.%26aulast%3DGuimaraes%26aufirst%3DA.%26aulast%3DMarinelli%26aufirst%3DB.%26aulast%3DWikstrom%26aufirst%3DJ.%2BD.%26aulast%3DNir%26aufirst%3DT.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DVaitheesvaran%26aufirst%3DB.%26aulast%3DIliopoulos%26aufirst%3DO.%26aulast%3DKurland%26aufirst%3DI.%26aulast%3DDor%26aufirst%3DY.%26aulast%3DWeissleder%26aufirst%3DR.%26aulast%3DShirihai%26aufirst%3DO.%2BS.%26aulast%3DEllisen%26aufirst%3DL.%2BW.%26aulast%3DEspinosa%26aufirst%3DJ.%2BM.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DThe%2520histone%2520deacetylase%2520SIRT6%2520regulates%2520glucose%2520homeostasis%2520via%2520Hif1%25CE%25B1%26jtitle%3DCell%26date%3D2010%26volume%3D140%26spage%3D280%26epage%3D293%26doi%3D10.1016%2Fj.cell.2009.12.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundaresan, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillai, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravindra, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeevanandam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, C.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, M. P.</span></span> <span> </span><span class="NLM_article-title">The sirtuin SIRT6 blocks IGF-Akt signaling and development of cardiac hypertrophy by targeting c-Jun</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1643</span>– <span class="NLM_lpage">1650</span>, <span class="refDoi"> DOI: 10.1038/nm.2961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnm.2961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23086477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFeqs7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1643-1650&author=N.+R.+Sundaresanauthor=P.+Vasudevanauthor=L.+Zhongauthor=G.+Kimauthor=S.+Samantauthor=V.+Parekhauthor=V.+B.+Pillaiauthor=P.+V.+Ravindraauthor=M.+Guptaauthor=V.+Jeevanandamauthor=J.+M.+Cunninghamauthor=C.-X.+Dengauthor=D.+B.+Lombardauthor=R.+Mostoslavskyauthor=M.+P.+Gupta&title=The+sirtuin+SIRT6+blocks+IGF-Akt+signaling+and+development+of+cardiac+hypertrophy+by+targeting+c-Jun&doi=10.1038%2Fnm.2961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The sirtuin SIRT6 blocks IGF-Akt signaling and development of cardiac hypertrophy by targeting c-Jun</span></div><div class="casAuthors">Sundaresan, Nagalingam R.; Vasudevan, Prabhakaran; Zhong, Lei; Kim, Gene; Samant, Sadhana; Parekh, Vishwas; Pillai, Vinodkumar B.; Ravindra, P. V.; Gupta, Madhu; Jeevanandam, Valluvan; Cunningham, John M.; Deng, Chu-Xia; Lombard, David B.; Mostoslavsky, Raul; Gupta, Mahesh P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1643-1650</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Abnormal activation of insulin-like growth factor (IGF)-Akt signaling is implicated in the development of various diseases, including heart failure.  However, the mol. mechanisms that regulate activation of this signaling pathway are not completely understood.  Here we show that sirtuin 6 (SIRT6), a nuclear histone deacetylase, functions at the level of chromatin to directly attenuate IGF-Akt signaling.  SIRT6-deficient mice developed cardiac hypertrophy and heart failure, whereas SIRT6 transgenic mice were protected from hypertrophic stimuli, indicating that SIRT6 acts as a neg. regulator of cardiac hypertrophy.  SIRT6-deficient mouse hearts showed hyperactivation of IGF signaling-related genes and their downstream targets.  Mechanistically, SIRT6 binds to and suppresses the promoter of IGF signaling-related genes by interacting with c-Jun and deacetylating histone 3 at Lys9 (H3K9).  We also found reduced SIRT6 expression in human failing hearts.  These findings disclose a new link between SIRT6 and IGF-Akt signaling and implicate SIRT6 in the development of cardiac hypertrophy and failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-wZqCB1pYErVg90H21EOLACvtfcHk0lixQJKcLRZDiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFeqs7zL&md5=731cda8d921f4e6abdbdbf8159255fde</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnm.2961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2961%26sid%3Dliteratum%253Aachs%26aulast%3DSundaresan%26aufirst%3DN.%2BR.%26aulast%3DVasudevan%26aufirst%3DP.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DG.%26aulast%3DSamant%26aufirst%3DS.%26aulast%3DParekh%26aufirst%3DV.%26aulast%3DPillai%26aufirst%3DV.%2BB.%26aulast%3DRavindra%26aufirst%3DP.%2BV.%26aulast%3DGupta%26aufirst%3DM.%26aulast%3DJeevanandam%26aufirst%3DV.%26aulast%3DCunningham%26aufirst%3DJ.%2BM.%26aulast%3DDeng%26aufirst%3DC.-X.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DGupta%26aufirst%3DM.%2BP.%26atitle%3DThe%2520sirtuin%2520SIRT6%2520blocks%2520IGF-Akt%2520signaling%2520and%2520development%2520of%2520cardiac%2520hypertrophy%2520by%2520targeting%2520c-Jun%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D1643%26epage%3D1650%26doi%3D10.1038%2Fnm.2961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, R.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.-Q.</span></span> <span> </span><span class="NLM_article-title">Nmnat2 protects cardiomyocytes from hypertrophy via activation of SIRT6</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>586</i></span>,  <span class="NLM_fpage">866</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2012.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.febslet.2012.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=22449973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtF2js70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=866-874&author=Y.+Caiauthor=S.-S.+Yuauthor=S.-R.+Chenauthor=R.-B.+Piauthor=S.+Gaoauthor=H.+Liauthor=J.-T.+Yeauthor=P.-Q.+Liu&title=Nmnat2+protects+cardiomyocytes+from+hypertrophy+via+activation+of+SIRT6&doi=10.1016%2Fj.febslet.2012.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Nmnat2 protects cardiomyocytes from hypertrophy via activation of SIRT6</span></div><div class="casAuthors">Cai, Yi; Yu, Shan-Shan; Chen, Shao-Rui; Pi, Rong-Biao; Gao, Si; Li, Hong; Ye, Jian-Tao; Liu, Pei-Qing</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">866-874</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery of sirtuins (SIRT), a family of NAD (NAD)-dependent deacetylases, has indicated that intracellular NAD level is crucial for the hypertrophic response of cardiomyocytes.  NMN adenylyltransferase (Nmnat) is a central enzyme in NAD biosynthesis.  Here we revealed that Nmnat2 protein expression and enzyme activity were down-regulated during cardiac hypertrophy.  In neonatal rat cardiomyocytes, overexpression of Nmnat2 but not its catalytically inactive mutant blocked angiotensin II (Ang II)-induced cardiac hypertrophy, which was dependent on activation of SIRT6 through maintaining the intracellular NAD level.  Our results suggested that modulation of Nmnat2 activity may be beneficial in cardiac hypertrophy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppOcYUeItX4LVg90H21EOLACvtfcHk0lhoHSGL5rDfTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtF2js70%253D&md5=dd6d2142aa5bc16afffdb73cd705e4cd</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DS.-S.%26aulast%3DChen%26aufirst%3DS.-R.%26aulast%3DPi%26aufirst%3DR.-B.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DYe%26aufirst%3DJ.-T.%26aulast%3DLiu%26aufirst%3DP.-Q.%26atitle%3DNmnat2%2520protects%2520cardiomyocytes%2520from%2520hypertrophy%2520via%2520activation%2520of%2520SIRT6%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D866%26epage%3D874%26doi%3D10.1016%2Fj.febslet.2012.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Min, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakiri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatloukal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, E. F.</span></span> <span> </span><span class="NLM_article-title">Liver cancer initiation is controlled by AP-1 through SIRT6-dependent inhibition of survivin</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1203</span>– <span class="NLM_lpage">1211</span>, <span class="refDoi"> DOI: 10.1038/ncb2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fncb2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23041974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKisLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=1203-1211&author=L.+Minauthor=Y.+Jiauthor=L.+Bakiriauthor=Z.+Qiuauthor=J.+Cenauthor=X.+Chenauthor=L.+Chenauthor=H.+Scheuchauthor=H.+Zhengauthor=L.+Qinauthor=K.+Zatloukalauthor=L.+Huiauthor=E.+F.+Wagner&title=Liver+cancer+initiation+is+controlled+by+AP-1+through+SIRT6-dependent+inhibition+of+survivin&doi=10.1038%2Fncb2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Liver cancer initiation is controlled by AP-1 through SIRT6-dependent inhibition of survivin</span></div><div class="casAuthors">Min, Lihua; Ji, Yuan; Bakiri, Latifa; Qiu, Zhixin; Cen, Jin; Chen, Xiaotao; Chen, Lingli; Scheuch, Harald; Zheng, Hai; Qin, Lunxiu; Zatloukal, Kurt; Hui, Lijian; Wagner, Erwin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1203-1211</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Understanding stage-dependent oncogenic mechanisms is crit. to develop not only targeted therapies, but also diagnostic markers and preventive strategies.  The mechanisms acting during cancer initiation remain elusive, largely owing to a lack of suitable animal models and limited availability of human precancerous lesions.  Here we show using genetic mouse models specific for liver cancer initiation, that survival of initiated cancer cells is controlled by c-Jun, independently of p53, through suppressing c-Fos-mediated apoptosis.  Mechanistically, c-Fos induces SIRT6 transcription, which represses survivin by reducing histone H3K9 acetylation and NF-κB activation.  Importantly, increasing the level of SIRT6 or targeting the anti-apoptotic activity of survivin at the initiation stage markedly impairs cancer development.  Moreover, in human dysplastic liver nodules, but not in malignant tumors, a specific expression pattern with increased c-Jun-survivin and attenuated c-Fos-SIRT6 levels was identified.  These results reveal a regulatory network connecting stress response and histone modification in liver tumor initiation, which could be targeted to prevent liver tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHiziBDVkjvbVg90H21EOLACvtfcHk0lhoHSGL5rDfTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKisLnN&md5=bf66909a3f8db10890c8d76f0fc3e3b6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fncb2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2590%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DL.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DBakiri%26aufirst%3DL.%26aulast%3DQiu%26aufirst%3DZ.%26aulast%3DCen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DScheuch%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DZatloukal%26aufirst%3DK.%26aulast%3DHui%26aufirst%3DL.%26aulast%3DWagner%26aufirst%3DE.%2BF.%26atitle%3DLiver%2520cancer%2520initiation%2520is%2520controlled%2520by%2520AP-1%2520through%2520SIRT6-dependent%2520inhibition%2520of%2520survivin%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2012%26volume%3D14%26spage%3D1203%26epage%3D1211%26doi%3D10.1038%2Fncb2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marquardt, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashyap, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krupp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teufel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zechner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strand, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorgeirsson, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galle, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strand, S.</span></span> <span> </span><span class="NLM_article-title">Sirtuin-6–dependent genetic and epigenetic alterations are associated with poor clinical outcome in hepatocellular carcinoma patients</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1054</span>– <span class="NLM_lpage">1064</span>, <span class="refDoi"> DOI: 10.1002/hep.26413</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1002%2Fhep.26413" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23526469" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlOkt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2013&pages=1054-1064&author=J.+U.+Marquardtauthor=K.+Fischerauthor=K.+Bausauthor=A.+Kashyapauthor=S.+Maauthor=M.+Kruppauthor=M.+Linkeauthor=A.+Teufelauthor=U.+Zechnerauthor=D.+Strandauthor=S.+S.+Thorgeirssonauthor=P.+R.+Galleauthor=S.+Strand&title=Sirtuin-6%E2%80%93dependent+genetic+and+epigenetic+alterations+are+associated+with+poor+clinical+outcome+in+hepatocellular+carcinoma+patients&doi=10.1002%2Fhep.26413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Sirtuin-6-dependent genetic and epigenetic alterations are associated with poor clinical outcome in hepatocellular carcinoma patients</span></div><div class="casAuthors">Marquardt, Jens U.; Fischer, Kerstin; Baus, Katharina; Kashyap, Anubha; Ma, Shengyun; Krupp, Markus; Linke, Matthias; Teufel, Andreas; Zechner, Ulrich; Strand, Dennis; Thorgeirsson, Snorri S.; Galle, Peter R.; Strand, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1054-1064</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Sirtuin 6 (SIRT6) is a member of the sirtuin family of NAD+-dependent deacetylases.  Genetic deletion of Sirt6 in mice results in a severe degenerative phenotype with impaired liver function and premature death.  The role of SIRT6 in development and progression of hepatocellular carcinoma is currently unknown.  We first investigated SIRT6 expression in 153 primary human liver cancers and in normal and cirrhotic livers using microarray anal.  SIRT6 was significantly down-regulated in both cirrhotic livers and cancer.  A Sirt6 knockout (KO) gene expression signature was generated from primary hepatoctyes isolated from 3-wk-old Sirt6-deficient animals.  Sirt6-deficient hepatocytes showed up-regulation of established hepatocellular carcinoma (HCC) biomarkers alpha-fetoprotein (Afp), insulin-like growth factor 2 (Igf2), H19, and glypican-3.  Furthermore, decreased SIRT6 expression was obsd. in hepatoma cell lines that are known to be apoptosis-insensitive.  Re-expression of SIRT6 in HepG2 cells increased apoptosis sensitivity to CD95-stimulation or chemotherapy treatment.  Loss of Sirt6 was characterized by oncogenic changes, such as global hypomethylation, as well as metabolic changes, such as hypoglycemia and increased fat deposition.  The hepatocyte-specific Sirt6-KO signature had a prognostic impact and was enriched in patients with poorly differentiated tumors with high AFP levels as well as recurrent disease.  Finally, we demonstrated that the Sirt6-KO signature possessed a predictive value for tumors other than HCC (e.g., breast and lung cancer).  Conclusion: Loss of SIRT6 induces epigenetic changes that may be relevant to chronic liver disease and HCC development.  Down-regulation of SIRT6 and genes dysregulated by loss of SIRT6 possess oncogenic effects in hepatocarcinogenesis.  Our data demonstrate that deficiency in one epigenetic regulator predisposes a tumorigenic phenotype that ultimately has relevance for outcome of HCC and other cancer patients.  (Hepatol. 2013;53:1054-1064).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwQ-ICe0Bv57Vg90H21EOLACvtfcHk0lhItoEKVF3bOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlOkt7rO&md5=d596a2fe828beed5b73c3cea0fa68e85</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fhep.26413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.26413%26sid%3Dliteratum%253Aachs%26aulast%3DMarquardt%26aufirst%3DJ.%2BU.%26aulast%3DFischer%26aufirst%3DK.%26aulast%3DBaus%26aufirst%3DK.%26aulast%3DKashyap%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DKrupp%26aufirst%3DM.%26aulast%3DLinke%26aufirst%3DM.%26aulast%3DTeufel%26aufirst%3DA.%26aulast%3DZechner%26aufirst%3DU.%26aulast%3DStrand%26aufirst%3DD.%26aulast%3DThorgeirsson%26aufirst%3DS.%2BS.%26aulast%3DGalle%26aufirst%3DP.%2BR.%26aulast%3DStrand%26aufirst%3DS.%26atitle%3DSirtuin-6%25E2%2580%2593dependent%2520genetic%2520and%2520epigenetic%2520alterations%2520are%2520associated%2520with%2520poor%2520clinical%2520outcome%2520in%2520hepatocellular%2520carcinoma%2520patients%26jtitle%3DHepatology%26date%3D2013%26volume%3D58%26spage%3D1054%26epage%3D1064%26doi%3D10.1002%2Fhep.26413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cagnetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soncini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orecchioni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talarico, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minetto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guolo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Retali, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carminati, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clavio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miglino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamaschi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahimana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchosal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todoerti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passalacqua, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruzzone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nencioni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoli, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cea, M.</span></span> <span> </span><span class="NLM_article-title">Depletion of SIRT6 enzymatic activity increases acute myeloid leukemia cells’ vulnerability to DNA-damaging agents</span>. <i>Haematologica</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.3324/haematol.2017.176248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.3324%2Fhaematol.2017.176248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=29025907" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1aiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2018&pages=80-90&author=A.+Cagnettaauthor=D.+Sonciniauthor=S.+Orecchioniauthor=G.+Talaricoauthor=P.+Minettoauthor=F.+Guoloauthor=V.+Retaliauthor=N.+Colomboauthor=E.+Carminatiauthor=M.+Clavioauthor=M.+Miglinoauthor=M.+Bergamaschiauthor=A.+Nahimanaauthor=M.+Duchosalauthor=K.+Todoertiauthor=A.+Neriauthor=M.+Passalacquaauthor=S.+Bruzzoneauthor=A.+Nencioniauthor=F.+Bertoliniauthor=M.+Gobbiauthor=R.+M.+Lemoliauthor=M.+Cea&title=Depletion+of+SIRT6+enzymatic+activity+increases+acute+myeloid+leukemia+cells%E2%80%99+vulnerability+to+DNA-damaging+agents&doi=10.3324%2Fhaematol.2017.176248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of SIRT6 enzymatic activity increases acute myeloid leukemia cells vulnerability to DNA-damaging agents</span></div><div class="casAuthors">Cagnetta, Antonia; Soncini, Debora; Orecchioni, Stefania; Talarico, Giovanna; Minetto, Paola; Guolo, Fabio; Retali, Veronica; Colombo, Nicoletta; Carminati, Enrico; Clavio, Marino; Miglino, Maurizio; Bergamaschi, Micaela; Nahimana, Aimable; Duchosal, Michel; Todoerti, Katia; Neri, Antonino; Passalacqua, Mario; Bruzzone, Santina; Nencioni, Alessio; Bertolini, Francesco; Gobbi, Marco; Lemoli, Roberto M.; Cea, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Haematologica</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-90</span>CODEN:
                <span class="NLM_cas:coden">HAEMAX</span>;
        ISSN:<span class="NLM_cas:issn">1592-8721</span>.
    
            (<span class="NLM_cas:orgname">Ferrata Storti Foundation</span>)
        </div><div class="casAbstract">Genomic instability plays a pathol. role in various malignancies, including acute myeloid leukemia (AML), and thus represents a potential therapeutic target.  Recent studies demonstrate that SIRT6, a NAD+-dependent nuclear deacetylase, functions as genome-guardian by preserving DNA integrity in different tumor cells.  Here, we demonstrate that also CD34+ blasts from AML patients show ongoing DNA damage and SIRT6 overexpression.  Indeed, we identified a poor-prognostic subset of patients, with widespread instability, which relies on SIRT6 to compensate for DNA-replication stress.  As a result, SIRT6 depletion compromises the ability of leukemia cells to repair DNA double-strand breaks that, in turn, increases their sensitivity to daunorubicin and Ara-C, both in vitro and in vivo.  In contrast, low SIRT6 levels obsd. in normal CD34+ hematopoietic progenitors explain their weaker sensitivity to genotoxic stress.  Intriguingly, we have identified DNA-PKcs and CtIP deacetylation as crucial for SIRT6-mediated DNA repair.  Together, our data suggest that inactivation of SIRT6 in leukemia cells leads to disruption of DNA-repair mechanisms, genomic instability and aggressive AML.  This synthetic lethal approach, enhancing DNA damage while concomitantly blocking repair responses, provides the rationale for the clin. evaluation of SIRT6 modulators in the treatment of leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrm3PFOl7M6MrVg90H21EOLACvtfcHk0lhItoEKVF3bOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1aiurk%253D&md5=5838badf698489bbb92b758cad8f57ab</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3324%2Fhaematol.2017.176248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3324%252Fhaematol.2017.176248%26sid%3Dliteratum%253Aachs%26aulast%3DCagnetta%26aufirst%3DA.%26aulast%3DSoncini%26aufirst%3DD.%26aulast%3DOrecchioni%26aufirst%3DS.%26aulast%3DTalarico%26aufirst%3DG.%26aulast%3DMinetto%26aufirst%3DP.%26aulast%3DGuolo%26aufirst%3DF.%26aulast%3DRetali%26aufirst%3DV.%26aulast%3DColombo%26aufirst%3DN.%26aulast%3DCarminati%26aufirst%3DE.%26aulast%3DClavio%26aufirst%3DM.%26aulast%3DMiglino%26aufirst%3DM.%26aulast%3DBergamaschi%26aufirst%3DM.%26aulast%3DNahimana%26aufirst%3DA.%26aulast%3DDuchosal%26aufirst%3DM.%26aulast%3DTodoerti%26aufirst%3DK.%26aulast%3DNeri%26aufirst%3DA.%26aulast%3DPassalacqua%26aufirst%3DM.%26aulast%3DBruzzone%26aufirst%3DS.%26aulast%3DNencioni%26aufirst%3DA.%26aulast%3DBertolini%26aufirst%3DF.%26aulast%3DGobbi%26aufirst%3DM.%26aulast%3DLemoli%26aufirst%3DR.%2BM.%26aulast%3DCea%26aufirst%3DM.%26atitle%3DDepletion%2520of%2520SIRT6%2520enzymatic%2520activity%2520increases%2520acute%2520myeloid%2520leukemia%2520cells%25E2%2580%2599%2520vulnerability%2520to%2520DNA-damaging%2520agents%26jtitle%3DHaematologica%26date%3D2018%26volume%3D103%26spage%3D80%26epage%3D90%26doi%3D10.3324%2Fhaematol.2017.176248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagnetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulciniti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhasin, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monacelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballestrero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoli, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nencioni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span> <span> </span><span class="NLM_article-title">Evidence for a role of the histone deacetylase SIRT6 in DNA damage response of multiple myeloma cells</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">1138</span>– <span class="NLM_lpage">1150</span>, <span class="refDoi"> DOI: 10.1182/blood-2015-06-649970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1182%2Fblood-2015-06-649970" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26675349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Ggu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2016&pages=1138-1150&author=M.+Ceaauthor=A.+Cagnettaauthor=S.+Adamiaauthor=C.+Acharyaauthor=Y.-T.+Taiauthor=M.+Fulcinitiauthor=H.+Ohguchiauthor=A.+Munshiauthor=P.+Acharyaauthor=M.+K.+Bhasinauthor=L.+Zhongauthor=R.+Carrascoauthor=F.+Monacelliauthor=A.+Ballestreroauthor=P.+Richardsonauthor=M.+Gobbiauthor=R.+M.+Lemoliauthor=N.+Munshiauthor=T.+Hideshimaauthor=A.+Nencioniauthor=D.+Chauhanauthor=K.+C.+Anderson&title=Evidence+for+a+role+of+the+histone+deacetylase+SIRT6+in+DNA+damage+response+of+multiple+myeloma+cells&doi=10.1182%2Fblood-2015-06-649970"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for a role of the histone deacetylase SIRT6 in DNA damage response of multiple myeloma cells</span></div><div class="casAuthors">Cea, Michele; Cagnetta, Antonia; Adamia, Sophia; Acharya, Chirag; Tai, Yu-Tzu; Fulciniti, Mariateresa; Ohguchi, Hiroto; Munshi, Aditya; Acharya, Prakrati; Bhasin, Manoj K.; Zhong, Lei; Carrasco, Ruben; Monacelli, Fiammetta; Ballestrero, Alberto; Richardson, Paul; Gobbi, Marco; Lemoli, Roberto M.; Munshi, Nikhil; Hideshima, Teru; Nencioni, Alessio; Chauhan, Dharminder; Anderson, Kenneth C.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1138-1150</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Multiple myeloma (MM) is characterized by a highly unstable genome, with aneuploidy obsd. in nearly all patients.  The mechanism causing this karyotypic instability is largely unknown, but recent observations have correlated these abnormalities with dysfunctional DNA damage response.  Here, we show that the NAD+-dependent deacetylase SIRT6 is highly expressed in MM cells, as an adaptive response to genomic stability, and that high SIRT6 levels are assocd. with adverse prognosis.  Mechanistically, SIRT6 interacts with the transcription factor ELK1 and with the ERK signaling-related gene.  By binding to their promoters and deacetylating H3K9 at these sites, SIRT6 downregulates the expression of mitogen-activated protein kinase (MAPK) pathway genes, MAPK signaling, and proliferation.  In addn., inactivation of ERK2/p90RSK signaling triggered by high SIRT6 levels increases DNA repair via Chk1 and confers resistance to DNA damage.  Using genetic and biochem. studies in vitro and in human MM xenograft models, we show that SIRT6 depletion both enhances proliferation and confers sensitization to DNA-damaging agents.  Our findings therefore provide insights into the functional interplay between SIRT6 and DNA repair mechanisms, with implications for both tumorigenesis and the treatment of MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFsgwJVN8UIrVg90H21EOLACvtfcHk0lhItoEKVF3bOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Ggu77N&md5=5dfdc3bba615c8c760bb288066a7e7db</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1182%2Fblood-2015-06-649970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2015-06-649970%26sid%3Dliteratum%253Aachs%26aulast%3DCea%26aufirst%3DM.%26aulast%3DCagnetta%26aufirst%3DA.%26aulast%3DAdamia%26aufirst%3DS.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DTai%26aufirst%3DY.-T.%26aulast%3DFulciniti%26aufirst%3DM.%26aulast%3DOhguchi%26aufirst%3DH.%26aulast%3DMunshi%26aufirst%3DA.%26aulast%3DAcharya%26aufirst%3DP.%26aulast%3DBhasin%26aufirst%3DM.%2BK.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DCarrasco%26aufirst%3DR.%26aulast%3DMonacelli%26aufirst%3DF.%26aulast%3DBallestrero%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DGobbi%26aufirst%3DM.%26aulast%3DLemoli%26aufirst%3DR.%2BM.%26aulast%3DMunshi%26aufirst%3DN.%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DNencioni%26aufirst%3DA.%26aulast%3DChauhan%26aufirst%3DD.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26atitle%3DEvidence%2520for%2520a%2520role%2520of%2520the%2520histone%2520deacetylase%2520SIRT6%2520in%2520DNA%2520damage%2520response%2520of%2520multiple%2520myeloma%2520cells%26jtitle%3DBlood%26date%3D2016%26volume%3D127%26spage%3D1138%26epage%3D1150%26doi%3D10.1182%2Fblood-2015-06-649970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kugel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastián, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitamant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gladden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadreyev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardeesy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">SIRT6 suppresses pancreatic cancer through control of Lin28b</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>165</i></span>,  <span class="NLM_fpage">1401</span>– <span class="NLM_lpage">1415</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.04.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2016.04.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=27180906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvV2kt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=1401-1415&author=S.+Kugelauthor=C.+Sebasti%C3%A1nauthor=J.+Fitamantauthor=K.+N.+Rossauthor=S.+K.+Sahaauthor=E.+Jainauthor=A.+Gladdenauthor=K.+S.+Aroraauthor=Y.+Katoauthor=M.+N.+Riveraauthor=S.+Ramaswamyauthor=R.+I.+Sadreyevauthor=A.+Gorenauthor=V.+Deshpandeauthor=N.+Bardeesyauthor=R.+Mostoslavsky&title=SIRT6+suppresses+pancreatic+cancer+through+control+of+Lin28b&doi=10.1016%2Fj.cell.2016.04.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 suppresses pancreatic cancer through control of Lin28b</span></div><div class="casAuthors">Kugel, Sita; Sebastian, Carlos; Fitamant, Julien; Ross, Kenneth N.; Saha, Supriya K.; Jain, Esha; Gladden, Adrianne; Arora, Kshitij S.; Kato, Yasutaka; Rivera, Miguel N.; Ramaswamy, Sridhar; Sadreyev, Ruslan I.; Goren, Alon; Deshpande, Vikram; Bardeesy, Nabeel; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1401-1415</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin remodeling proteins are frequently dysregulated in human cancer, yet little is known about how they control tumorigenesis.  Here, we uncover an epigenetic program mediated by the NAD+-dependent histone deacetylase Sirtuin 6 (SIRT6) that is crit. for suppression of pancreatic ductal adenocarcinoma (PDAC), one of the most lethal malignancies.  SIRT6 inactivation accelerates PDAC progression and metastasis via upregulation of Lin28b, a neg. regulator of the let-7 microRNA.  SIRT6 loss results in histone hyperacetylation at the Lin28b promoter, Myc recruitment, and pronounced induction of Lin28b and downstream let-7 target genes, HMGA2, IGF2BP1, and IGF2BP3.  This epigenetic program defines a distinct subset with a poor prognosis, representing 30%-40% of human PDAC, characterized by reduced SIRT6 expression and an exquisite dependence on Lin28b for tumor growth.  Thus, we identify SIRT6 as an important PDAC tumor suppressor and uncover the Lin28b pathway as a potential therapeutic target in a molecularly defined PDAC subset.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IS8QdcElmbVg90H21EOLACvtfcHk0li_4bO592yVvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvV2kt7s%253D&md5=553ff1dc14d8a90aa11a708e34e1795b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.04.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.04.033%26sid%3Dliteratum%253Aachs%26aulast%3DKugel%26aufirst%3DS.%26aulast%3DSebasti%25C3%25A1n%26aufirst%3DC.%26aulast%3DFitamant%26aufirst%3DJ.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DSaha%26aufirst%3DS.%2BK.%26aulast%3DJain%26aufirst%3DE.%26aulast%3DGladden%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DK.%2BS.%26aulast%3DKato%26aufirst%3DY.%26aulast%3DRivera%26aufirst%3DM.%2BN.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DSadreyev%26aufirst%3DR.%2BI.%26aulast%3DGoren%26aufirst%3DA.%26aulast%3DDeshpande%26aufirst%3DV.%26aulast%3DBardeesy%26aufirst%3DN.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DSIRT6%2520suppresses%2520pancreatic%2520cancer%2520through%2520control%2520of%2520Lin28b%26jtitle%3DCell%26date%3D2016%26volume%3D165%26spage%3D1401%26epage%3D1415%26doi%3D10.1016%2Fj.cell.2016.04.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grozio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasigliè, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basile, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturla, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sociali, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soncini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poggi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoppoli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballestrero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrone, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruzzone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nencioni, A.</span></span> <span> </span><span class="NLM_article-title">The NAD+-dependent histone deacetylase SIRT6 promotes cytokine production and migration in pancreatic cancer cells by regulating Ca<sup>2+</sup> responses</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">40924</span>– <span class="NLM_lpage">40937</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.405837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1074%2Fjbc.m112.405837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23086953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslGjs7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=40924-40937&author=I.+Bauerauthor=A.+Grozioauthor=D.+Lasigli%C3%A8author=G.+Basileauthor=L.+Sturlaauthor=M.+Magnoneauthor=G.+Socialiauthor=D.+Sonciniauthor=I.+Caffaauthor=A.+Poggiauthor=G.+Zoppoliauthor=M.+Ceaauthor=G.+Feldmannauthor=R.+Mostoslavskyauthor=A.+Ballestreroauthor=F.+Patroneauthor=S.+Bruzzoneauthor=A.+Nencioni&title=The+NAD%2B-dependent+histone+deacetylase+SIRT6+promotes+cytokine+production+and+migration+in+pancreatic+cancer+cells+by+regulating+Ca2%2B+responses&doi=10.1074%2Fjbc.m112.405837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The NAD+-dependent Histone Deacetylase SIRT6 Promotes Cytokine Production and Migration in Pancreatic Cancer Cells by Regulating Ca2+ Responses</span></div><div class="casAuthors">Bauer, Inga; Grozio, Alessia; Lasiglie, Denise; Basile, Giovanna; Sturla, Laura; Magnone, Mirko; Sociali, Giovanna; Soncini, Debora; Caffa, Irene; Poggi, Alessandro; Zoppoli, Gabriele; Cea, Michele; Feldmann, Georg; Mostoslavsky, Raul; Ballestrero, Alberto; Patrone, Franco; Bruzzone, Santina; Nencioni, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">40924-40937</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cytokine secretion by cancer cells contributes to cancer-induced symptoms and angiogenesis.  Studies show that the sirtuin SIRT6 promotes inflammation by enhancing TNF expression.  Here, we aimed to det. whether SIRT6 is involved in conferring an inflammatory phenotype to cancer cells and to define the mechanisms linking SIRT6 to inflammation.  We show that SIRT6 enhances the expression of pro-inflammatory cyto-/chemokines, such as IL8 and TNF, and promotes cell migration in pancreatic cancer cells by enhancing Ca2+ responses.  Via its enzymic activity, SIRT6 increases the intracellular levels of ADP-ribose, an activator of the Ca2+ channel TRPM2.  In turn, TRPM2 and Ca2+ are shown to be involved in SIRT6-induced TNF and IL8 expression.  SIRT6 increases the nuclear levels of the Ca2+-dependent transcription factor, nuclear factor of activated T cells (NFAT), and cyclosporin A, a calcineurin inhibitor that reduces NFAT activity, reduces TNF and IL8 expression in SIRT6-overexpressing cells.  These results implicate a role for SIRT6 in the synthesis of Ca2+-mobilizing second messengers, in the regulation of Ca2+-dependent transcription factors, and in the expression of pro-inflammatory, pro-angiogenic, and chemotactic cytokines.  SIRT6 inhibition may help combat cancer-induced inflammation, angiogenesis, and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9ymWqzmiWv7Vg90H21EOLACvtfcHk0li_4bO592yVvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslGjs7nN&md5=59c3a2081b85bbbd3543643035388f4c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m112.405837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m112.405837%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DGrozio%26aufirst%3DA.%26aulast%3DLasigli%25C3%25A8%26aufirst%3DD.%26aulast%3DBasile%26aufirst%3DG.%26aulast%3DSturla%26aufirst%3DL.%26aulast%3DMagnone%26aufirst%3DM.%26aulast%3DSociali%26aufirst%3DG.%26aulast%3DSoncini%26aufirst%3DD.%26aulast%3DCaffa%26aufirst%3DI.%26aulast%3DPoggi%26aufirst%3DA.%26aulast%3DZoppoli%26aufirst%3DG.%26aulast%3DCea%26aufirst%3DM.%26aulast%3DFeldmann%26aufirst%3DG.%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DBallestrero%26aufirst%3DA.%26aulast%3DPatrone%26aufirst%3DF.%26aulast%3DBruzzone%26aufirst%3DS.%26aulast%3DNencioni%26aufirst%3DA.%26atitle%3DThe%2520NAD%252B-dependent%2520histone%2520deacetylase%2520SIRT6%2520promotes%2520cytokine%2520production%2520and%2520migration%2520in%2520pancreatic%2520cancer%2520cells%2520by%2520regulating%2520Ca2%252B%2520responses%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D40924%26epage%3D40937%26doi%3D10.1074%2Fjbc.m112.405837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span> <span> </span><span class="NLM_article-title">SIRT6 deacetylates PKM2 to suppress its nuclear localization and oncogenic functions</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">E538</span>– <span class="NLM_lpage">E547</span>, <span class="refDoi"> DOI: 10.1073/pnas.1520045113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1073%2Fpnas.1520045113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26787900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFShtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=E538-E547&author=A.+Bhardwajauthor=S.+Das&title=SIRT6+deacetylates+PKM2+to+suppress+its+nuclear+localization+and+oncogenic+functions&doi=10.1073%2Fpnas.1520045113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 deacetylates PKM2 to suppress its nuclear localization and oncogenic functions</span></div><div class="casAuthors">Bhardwaj, Abhishek; Das, Sanjeev</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">E538-E547</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SIRT6 (sirtuin 6) is a member of sirtuin family of deacetylases involved in diverse processes including genome stability, metabolic homeostasis, and tumorigenesis.  However, the role of SIRT6 deacetylase activity in its tumor-suppressor functions is not well understood.  Here we report that SIRT6 binds to and deacetylates nuclear PKM2 (pyruvate kinase M2) at the lysine 433 residue.  PKM2 is a glycolytic enzyme with nonmetabolic nuclear oncogenic functions.  SIRT6-mediated deacetylation results in PKM2 nuclear export.  We further have identified exportin 4 as the specific transporter mediating PKM2 nuclear export.  As a result of SIRT6-mediated deacetylation, PKM2 nuclear protein kinase and transcriptional coactivator functions are abolished.  Thus, SIRT6 suppresses PKM2 oncogenic functions, resulting in reduced cell proliferation, migration potential, and invasiveness.  Furthermore, studies in mouse tumor models demonstrate that PKM2 deacetylation is integral to SIRT6-mediated tumor suppression and inhibition of metastasis.  Addnl., reduced SIRT6 levels correlate with elevated nuclear acetylated PKM2 levels in increasing grades of hepatocellular carcinoma.  These findings provide key insights into the pivotal role of deacetylase activity in SIRT6 tumor-suppressor functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDt-i9uz1mb7Vg90H21EOLACvtfcHk0ljCxhJOl-Mxig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFShtb8%253D&md5=2c092b16b828cde50520835732deb04d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1520045113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1520045113%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DS.%26atitle%3DSIRT6%2520deacetylates%2520PKM2%2520to%2520suppress%2520its%2520nuclear%2520localization%2520and%2520oncogenic%2520functions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3DE538%26epage%3DE547%26doi%3D10.1073%2Fpnas.1520045113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span> <span> </span><span class="NLM_article-title">Lamin A is an endogenous SIRT6 activator and promotes SIRT6-mediated DNA repair</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1396</span>– <span class="NLM_lpage">1406</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.celrep.2015.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26549451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsl2nu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=1396-1406&author=S.+Ghoshauthor=B.+Liuauthor=Y.+Wangauthor=Q.+Haoauthor=Z.+Zhou&title=Lamin+A+is+an+endogenous+SIRT6+activator+and+promotes+SIRT6-mediated+DNA+repair&doi=10.1016%2Fj.celrep.2015.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Lamin A Is an Endogenous SIRT6 Activator and Promotes SIRT6-Mediated DNA Repair</span></div><div class="casAuthors">Ghosh, Shrestha; Liu, Baohua; Wang, Yi; Hao, Quan; Zhou, Zhongjun</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1396-1406</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The nuclear lamins are essential for various mol. events in the nucleus, such as chromatin organization, DNA replication, and provision of mech. support.  A specific point mutation in the LMNA gene creates a truncated prelamin A termed progerin, causing Hutchinson-Gilford progeria syndrome (HGPS).  SIRT6 deficiency leads to defective genomic maintenance and accelerated aging similar to HGPS, suggesting a potential link between lamin A and SIRT6.  Here, we report that lamin A is an endogenous activator of SIRT6 and facilitates chromatin localization of SIRT6 upon DNA damage.  Lamin A promotes SIRT6-dependent DNA-PKcs (DNA-PK catalytic subunit) recruitment to chromatin, CtIP deacetylation, and PARP1 mono-ADP ribosylation in response to DNA damage.  The presence of progerin jeopardizes SIRT6 activation and compromises SIRT6-mediated mol. events in response to DNA damage.  These data reveal a crit. role for lamin A in regulating SIRT6 activities, suggesting that defects in SIRT6 functions contribute to impaired DNA repair and accelerated aging in HGPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooBReO__6aJbVg90H21EOLACvtfcHk0ljCxhJOl-Mxig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsl2nu7fP&md5=dd94bde5e1267f6db3ce97b5d296d4da</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DZ.%26atitle%3DLamin%2520A%2520is%2520an%2520endogenous%2520SIRT6%2520activator%2520and%2520promotes%2520SIRT6-mediated%2520DNA%2520repair%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D13%26spage%3D1396%26epage%3D1406%26doi%3D10.1016%2Fj.celrep.2015.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">You, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kambach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meleshin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steegborn, C.</span></span> <span> </span><span class="NLM_article-title">Structural basis of sirtuin 6 activation by synthetic small molecules</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1011</span>, <span class="refDoi"> DOI: 10.1002/anie.201610082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1002%2Fanie.201610082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFahs7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=1007-1011&author=W.+Youauthor=D.+Rotiliauthor=T.-M.+Liauthor=C.+Kambachauthor=M.+Meleshinauthor=M.+Schutkowskiauthor=K.+F.+Chuaauthor=A.+Maiauthor=C.+Steegborn&title=Structural+basis+of+sirtuin+6+activation+by+synthetic+small+molecules&doi=10.1002%2Fanie.201610082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Sirtuin 6 Activation by Synthetic Small Molecules</span></div><div class="casAuthors">You, Weijie; Rotili, Dante; Li, Tie-Mei; Kambach, Christian; Meleshin, Marat; Schutkowski, Mike; Chua, Katrin F.; Mai, Antonello; Steegborn, Clemens</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1007-1011</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Sirtuins are protein deacylases regulating metab. and stress responses, and are implicated in aging-related diseases.  Small mol. activators for the human sirtuins Sirt1-7 are sought as chem. tools and potential therapeutics, such as for cancer.  Activators are available for Sirt1 and exploit its unique N-terminus, whereas drug-like activators for Sirt2-7 are lacking.  The authors synthesized and screened pyrrolo[1,2-a]quinoxaline derivs., yielding the first synthetic Sirt6 activators.  Biochem. assays show direct, substrate-independent compd. binding to the Sirt6 catalytic core and potent activation of Sirt6-dependent deacetylation of peptide substrates and complete nucleosomes.  Crystal structures of Sirt6/activator complexes reveal that the compds. bind to a Sirt6-specific acyl channel pocket and identify key interactions.  The authors' results establish potent Sirt6 activation with small mols. and provide a structural basis for further development of Sirt6 activators as tools and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfGfCtcpk-4bVg90H21EOLACvtfcHk0ljCxhJOl-Mxig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFahs7%252FL&md5=4eaa23343e806fdfb2397a8f662c0ef8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201610082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201610082%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DW.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DT.-M.%26aulast%3DKambach%26aufirst%3DC.%26aulast%3DMeleshin%26aufirst%3DM.%26aulast%3DSchutkowski%26aufirst%3DM.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DSteegborn%26aufirst%3DC.%26atitle%3DStructural%2520basis%2520of%2520sirtuin%25206%2520activation%2520by%2520synthetic%2520small%2520molecules%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D1007%26epage%3D1011%26doi%3D10.1002%2Fanie.201610082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">You, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steegborn, C.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the activation and inhibition of Sirtuin 6 by quercetin and its derivatives</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">19176</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-55654-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fs41598-019-55654-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=31844103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVeru7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=19176&author=W.+Youauthor=W.+Zhengauthor=S.+Weissauthor=K.+F.+Chuaauthor=C.+Steegborn&title=Structural+basis+for+the+activation+and+inhibition+of+Sirtuin+6+by+quercetin+and+its+derivatives&doi=10.1038%2Fs41598-019-55654-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the activation and inhibition of Sirtuin 6 by quercetin and its derivatives</span></div><div class="casAuthors">You, Weijie; Zheng, Wei; Weiss, Sandra; Chua, Katrin F.; Steegborn, Clemens</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19176</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mammalian Sirtuin 6 (Sirt6) is an NAD+-dependent protein deacylase regulating metab. and chromatin homeostasis.  Sirt6 activation protects against metabolic and aging-related diseases, and Sirt6 inhibition is considered a cancer therapy.  Available Sirt6 modulators show insufficient potency and specificity, and even partially contradictory Sirt6 effects were reported for the plant flavone quercetin.  To understand Sirt6 modulation by quercetin-based compds., we analyzed their binding and activity effects on Sirt6 and other Sirtuin isoforms and solved crystal structures of compd. complexes with Sirt6 and Sirt2.  We find that quercetin activates Sirt6 via the isoform-specific binding site for pyrrolo[1,2-a]quinoxalines.  Its inhibitory effect on other isoforms is based on an alternative binding site at the active site entrance.  Based on these insights, we identified isoquercetin as a ligand that can discriminate both sites and thus activates Sirt6 with increased specificity.  Furthermore, we find that quercetin derivs. that inhibit rather than activate Sirt6 exploit the same general Sirt6 binding site as the activators, identifying it as a versatile allosteric site for Sirt6 modulation.  Our results thus provide a structural basis for Sirtuin effects of quercetin-related compds. and helpful insights for Sirt6-targeted drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiCjD_hIEVCbVg90H21EOLACvtfcHk0ljCxhJOl-Mxig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVeru7fO&md5=c315a5654a7c33add2f7cd13a396891f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-55654-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-55654-1%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DW.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DWeiss%26aufirst%3DS.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DSteegborn%26aufirst%3DC.%26atitle%3DStructural%2520basis%2520for%2520the%2520activation%2520and%2520inhibition%2520of%2520Sirtuin%25206%2520by%2520quercetin%2520and%2520its%2520derivatives%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D19176%26doi%3D10.1038%2Fs41598-019-55654-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Identification of a cellularly active SIRT6 allosteric activator</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0150-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fs41589-018-0150-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=30374165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVCitL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=1118-1126&author=Z.+Huangauthor=J.+Zhaoauthor=W.+Dengauthor=Y.+Chenauthor=J.+Shangauthor=K.+Songauthor=L.+Zhangauthor=C.+Wangauthor=S.+Luauthor=X.+Yangauthor=B.+Heauthor=J.+Minauthor=H.+Huauthor=M.+Tanauthor=J.+Xuauthor=Q.+Zhangauthor=J.+Zhongauthor=X.+Sunauthor=Z.+Maoauthor=H.+Linauthor=M.+Xiaoauthor=Y.+E.+Chinauthor=H.+Jiangauthor=Y.+Xuauthor=G.+Chenauthor=J.+Zhang&title=Identification+of+a+cellularly+active+SIRT6+allosteric+activator&doi=10.1038%2Fs41589-018-0150-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a cellularly active SIRT6 allosteric activator</span></div><div class="casAuthors">Huang, Zhimin; Zhao, Junxing; Deng, Wei; Chen, Yingyi; Shang, Jialin; Song, Kun; Zhang, Lu; Wang, Chengxiang; Lu, Shaoyong; Yang, Xiuyan; He, Bin; Min, Jinrong; Hu, Hao; Tan, Minjia; Xu, Jianrong; Zhang, Qiufen; Zhong, Jie; Sun, Xiaoxiang; Mao, Zhiyong; Lin, Houwen; Xiao, Mingzhe; Chin, Y. Eugene; Jiang, Hualiang; Xu, Ying; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1118-1126</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">SIRT6, a member of the SIRT deacetylase family, is responsible for deacetylation of histone H3 Nε-acetyl-lysines 9 (H3K9ac) and 56 (H3K56ac).  As a tumor suppressor, SIRT6 has frequently been found to have low expression in various cancers.  Here, we report the identification of MDL-800, a selective SIRT6 activator.  MDL-800 increased the deacetylase activity of SIRT6 by up to 22-fold via binding to an allosteric site; this interaction led to a global decrease in H3K9ac and H3K56ac levels in human hepatocellular carcinoma (HCC) cells.  Consequently, MDL-800 inhibited the proliferation of HCC cells via SIRT6-driven cell-cycle arrest and was effective in a tumor xenograft model.  Together, these data demonstrate that pharmacol. activation of SIRT6 is a potential therapeutic approach for the treatment of HCC.  MDL-800 is a first-in-class small-mol. cellular SIRT6 activator that can be used to physiol. and pathol. investigate the roles of SIRT6 deacetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFPmr-laqlLVg90H21EOLACvtfcHk0ljkVz-2uT8bFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVCitL7O&md5=5fb5c1de5b6be0291ffa45debf72516e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0150-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0150-0%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DM.%26aulast%3DChin%26aufirst%3DY.%2BE.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520a%2520cellularly%2520active%2520SIRT6%2520allosteric%2520activator%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D1118%26epage%3D1126%26doi%3D10.1038%2Fs41589-018-0150-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0ljkVz-2uT8bFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-Y.</span></span> <span> </span><span class="NLM_article-title">ID-Score: a new empirical scoring function based on a comprehensive set of descriptors related to protein–ligand interactions</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">600</span>, <span class="refDoi"> DOI: 10.1021/ci300493w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300493w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1Sqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=592-600&author=G.-B.+Liauthor=L.-L.+Yangauthor=W.-J.+Wangauthor=L.-L.+Liauthor=S.-Y.+Yang&title=ID-Score%3A+a+new+empirical+scoring+function+based+on+a+comprehensive+set+of+descriptors+related+to+protein%E2%80%93ligand+interactions&doi=10.1021%2Fci300493w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">ID-Score: A New Empirical Scoring Function Based on a Comprehensive Set of Descriptors Related to Protein-Ligand Interactions</span></div><div class="casAuthors">Li, Guo-Bo; Yang, Ling-Ling; Wang, Wen-Jing; Li, Lin-Li; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">592-600</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Scoring functions have been widely used to assess protein-ligand binding affinity in structure-based drug discovery.  However, currently commonly used scoring functions face some challenges including poor correlation between calcd. scores and exptl. binding affinities, target-dependent performance, and low sensitivity to analogs.  In this account, we propose a new empirical scoring function termed ID-Score.  ID-Score was established based on a comprehensive set of descriptors related to protein-ligand interactions; these descriptors cover nine categories: van der Waals interaction, hydrogen-bonding interaction, electrostatic interaction, π-system interaction, metal-ligand bonding interaction, desolvation effect, entropic loss effect, shape matching, and surface property matching.  A total of 2278 complexes were used as the training set, and a modified support vector regression (SVR) algorithm was used to fit the exptl. binding affinities.  Evaluation results showed that ID-Score outperformed other selected commonly used scoring functions on a benchmark test set and showed considerable performance on a large independent test set.  ID-Score also showed a consistent higher performance across different biol. targets.  Besides, it could correctly differentiate structurally similar ligands, indicating higher sensitivity to analogs.  Collectively, the better performance of ID-Score enables it as a useful tool in assessing protein-ligand binding affinity in structure-based drug discovery as well as in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtuiAPD05vtLVg90H21EOLACvtfcHk0lhV-yoSNBxxTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1Sqsbg%253D&md5=1e001ed3da47d6970a5e3ac25eaf5b1c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci300493w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300493w%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-B.%26aulast%3DYang%26aufirst%3DL.-L.%26aulast%3DWang%26aufirst%3DW.-J.%26aulast%3DLi%26aufirst%3DL.-L.%26aulast%3DYang%26aufirst%3DS.-Y.%26atitle%3DID-Score%253A%2520a%2520new%2520empirical%2520scoring%2520function%2520based%2520on%2520a%2520comprehensive%2520set%2520of%2520descriptors%2520related%2520to%2520protein%25E2%2580%2593ligand%2520interactions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D592%26epage%3D600%26doi%3D10.1021%2Fci300493w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Houston, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkinshaw, M. D.</span></span> <span> </span><span class="NLM_article-title">Consensus docking: improving the reliability of docking in a virtual screening context</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1021/ci300399w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300399w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitFeksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=384-390&author=D.+R.+Houstonauthor=M.+D.+Walkinshaw&title=Consensus+docking%3A+improving+the+reliability+of+docking+in+a+virtual+screening+context&doi=10.1021%2Fci300399w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Consensus Docking: Improving the Reliability of Docking in a Virtual Screening Context</span></div><div class="casAuthors">Houston, Douglas R.; Walkinshaw, Malcolm D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">384-390</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based virtual screening relies on scoring the predicted binding modes of compds. docked into the target.  Because the accuracy of this scoring relies on the accuracy of the docking, methods that increase docking accuracy are valuable.  Here, we present a relatively straightforward method for improving the probability of identifying accurately docked poses.  The method is similar in concept to consensus scoring schemes, which have been shown to increase ranking power and thus hit rates, but combines information about predicted binding modes rather than predicted binding affinities.  The pose prediction success rate of each docking program alone was found in this trial to be 55% for Autodock, 58% for DOCK, and 64% for Vina.  By using more than one docking program to predict the binding pose, correct poses were identified in 82% or more of cases, a significant improvement.  In a virtual screen, these more reliably posed compds. can be preferentially advanced to subsequent scoring stages to improve hit rates.  Consensus docking can be easily introduced into established structure-based virtual screening methodologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGaDety9hNcrVg90H21EOLACvtfcHk0lhV-yoSNBxxTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitFeksbc%253D&md5=6f722446700235e241a339b8dd7e3755</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci300399w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300399w%26sid%3Dliteratum%253Aachs%26aulast%3DHouston%26aufirst%3DD.%2BR.%26aulast%3DWalkinshaw%26aufirst%3DM.%2BD.%26atitle%3DConsensus%2520docking%253A%2520improving%2520the%2520reliability%2520of%2520docking%2520in%2520a%2520virtual%2520screening%2520context%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D384%26epage%3D390%26doi%3D10.1021%2Fci300399w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of human TyrRS inhibitors by structure-based virtual screening, structural optimization, and bioassays</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">9323</span>– <span class="NLM_lpage">9330</span>, <span class="refDoi"> DOI: 10.1039/c9ra00458k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1039%2Fc9ra00458k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1emtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=9323-9330&author=S.+Huangauthor=X.+Wangauthor=G.+Linauthor=J.+Chengauthor=X.+Chenauthor=W.+Sunauthor=R.+Xiangauthor=Y.+Yuauthor=L.+Liauthor=S.+Yang&title=Discovery+of+human+TyrRS+inhibitors+by+structure-based+virtual+screening%2C+structural+optimization%2C+and+bioassays&doi=10.1039%2Fc9ra00458k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of human TyrRS inhibitors by structure-based virtual screening, structural optimization, and bioassays</span></div><div class="casAuthors">Huang, Shenzhen; Wang, Xiang; Lin, Guifeng; Cheng, Jie; Chen, Xiuli; Sun, Weining; Xiang, Rong; Yu, Yamei; Li, Linli; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">9323-9330</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The human tyrosyl transfer-RNA (tRNA) synthetase (TyrRS), which is well known for its essential aminoacylation function in protein synthesis, has been shown to translocate to the nucleus and protect against DNA damage caused by external stimuli.  Small mols. that can fit into the active site pocket of TyrRS are thought to affect the nuclear role.  The exploitation of TyrRS inhibitors has attracted attention recently.  In this investigation, we adopted a structure-based virtual screening strategy and subsequent structure-activity relationship anal. to discover new TyrRS inhibitors, and identified a potent compd. 5,7-dihydroxy-6,8-bis((3-hydroxyphenyl)thio)-2-phenyl-4H-chromen-4-one (compd. 11, Ki = 8.8 μM).  In intact HeLa cells, this compd. showed a protective effect against DNA damage.  Compd. 11 is a good lead compd. for the further development of drugs against disorders caused by DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIhbDn8NV6TrVg90H21EOLACvtfcHk0lhV-yoSNBxxTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXls1emtbw%253D&md5=12335cda0cd504adb26f2130467e2405</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc9ra00458k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc9ra00458k%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DG.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520human%2520TyrRS%2520inhibitors%2520by%2520structure-based%2520virtual%2520screening%252C%2520structural%2520optimization%252C%2520and%2520bioassays%26jtitle%3DRSC%2520Adv.%26date%3D2019%26volume%3D9%26spage%3D9323%26epage%3D9330%26doi%3D10.1039%2Fc9ra00458k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of new SIRT2 inhibitors by utilizing a consensus docking/scoring strategy and structure–activity relationship analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">679</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00714</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00714" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktlynu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=669-679&author=S.+Huangauthor=C.+Songauthor=X.+Wangauthor=G.+Zhangauthor=Y.+Wangauthor=X.+Jiangauthor=Q.+Sunauthor=L.+Huangauthor=R.+Xiangauthor=Y.+Huauthor=L.+Liauthor=S.+Yang&title=Discovery+of+new+SIRT2+inhibitors+by+utilizing+a+consensus+docking%2Fscoring+strategy+and+structure%E2%80%93activity+relationship+analysis&doi=10.1021%2Facs.jcim.6b00714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of New SIRT2 Inhibitors by Utilizing a Consensus Docking/Scoring Strategy and Structure-Activity Relationship Analysis</span></div><div class="casAuthors">Huang, Shenzhen; Song, Chunli; Wang, Xiang; Zhang, Guo; Wang, Yanlin; Jiang, Xiaojuan; Sun, Qizheng; Huang, Luyi; Xiang, Rong; Hu, Yiguo; Li, Linli; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-679</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SIRT2, which is a NAD+ (NAD) dependent deacetylase, has been demonstrated to play an important role in the occurrence and development of a variety of diseases such as cancer, ischemia-reperfusion, and neurodegenerative diseases.  Small mol. inhibitors of SIRT2 are thought to be potential interfering agents for relevant diseases.  Discovery of SIRT2 inhibitors has attracted much attention recently.  In this investigation, we adopted a consensus docking/scoring strategy to screen for novel SIRT2 inhibitors.  Structural optimization and structure-activity relationship (SAR) anal. were then carried out on highly potent compds. with new scaffolds, which led to the discovery of 2-((5-benzyl-5H-[1,2,4]triazino[5,6-b]indol-3-yl)thio)-N-(naphthalen-1-yl)acetamide (SR86).  This compd. showed good activity against SIRT2 with an IC50 value of 1.3 μM.  SR86 did not exhibit activity against SIRT1 and SIRT3, implying a good selectivity for SIRT2.  In in vitro cellular assays, SR86 displayed very good antiviability activity against breast cancer cell line MCF-7.  In Western blot assays, SR86 showed considerable activity in blocking the deacetylation of α-tubulin, which is a typical substrate of SIRT2.  Collectively, because of the new scaffold structure and good selectivity of SR86, it could serve as a promising lead compd., hence deserving further studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs8mGEkJHyfbVg90H21EOLACvtfcHk0liVktltTKvbpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktlynu74%253D&md5=84fedc2c19c521116e76bf16c558005a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00714%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520new%2520SIRT2%2520inhibitors%2520by%2520utilizing%2520a%2520consensus%2520docking%252Fscoring%2520strategy%2520and%2520structure%25E2%2580%2593activity%2520relationship%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D669%26epage%3D679%26doi%3D10.1021%2Facs.jcim.6b00714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhargava, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span> <span> </span><span class="NLM_article-title">A fluorogenic assay for screening Sirt6 modulators</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">5213</span>– <span class="NLM_lpage">5216</span>, <span class="refDoi"> DOI: 10.1039/c3ob41138a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1039%2Fc3ob41138a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23839075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFKrsrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=5213-5216&author=J.+Huauthor=B.+Heauthor=S.+Bhargavaauthor=H.+Lin&title=A+fluorogenic+assay+for+screening+Sirt6+modulators&doi=10.1039%2Fc3ob41138a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorogenic assay for screening Sirt6 modulators</span></div><div class="casAuthors">Hu, Jing; He, Bin; Bhargava, Shiva; Lin, Hening</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">5213-5216</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A fluorogenic high-throughput assay suitable for screening Sirt6 modulators is developed based on the recently discovered efficient activity of Sirt6 to hydrolyze myristoyl lysine.  Sirt6 modulators will be useful in investigating the function of Sirt6 and protein lysine fatty acylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2fZveXZ0sg7Vg90H21EOLACvtfcHk0liVktltTKvbpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFKrsrbM&md5=184d4df01307a1edefb8faceb6a5f7b7</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2Fc3ob41138a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob41138a%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DBhargava%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DH.%26atitle%3DA%2520fluorogenic%2520assay%2520for%2520screening%2520Sirt6%2520modulators%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D5213%26epage%3D5216%26doi%3D10.1039%2Fc3ob41138a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milne, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carney, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagne, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boss, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perni, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disch, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galonek, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iffland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedik, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinclair, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olefsky, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jirousek, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westphal, C. H.</span></span> <span> </span><span class="NLM_article-title">Small molecule activators of SIRT1 as therapeutics for the treatment of type 2 diabetes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">712</span>– <span class="NLM_lpage">716</span>, <span class="refDoi"> DOI: 10.1038/nature06261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnature06261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=18046409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=712-716&author=J.+C.+Milneauthor=P.+D.+Lambertauthor=S.+Schenkauthor=D.+P.+Carneyauthor=J.+J.+Smithauthor=D.+J.+Gagneauthor=L.+Jinauthor=O.+Bossauthor=R.+B.+Perniauthor=C.+B.+Vuauthor=J.+E.+Bemisauthor=R.+Xieauthor=J.+S.+Dischauthor=P.+Y.+Ngauthor=J.+J.+Nunesauthor=A.+V.+Lynchauthor=H.+Yangauthor=H.+Galonekauthor=K.+Israelianauthor=W.+Choyauthor=A.+Ifflandauthor=S.+Lavuauthor=O.+Medvedikauthor=D.+A.+Sinclairauthor=J.+M.+Olefskyauthor=M.+R.+Jirousekauthor=P.+J.+Elliottauthor=C.+H.+Westphal&title=Small+molecule+activators+of+SIRT1+as+therapeutics+for+the+treatment+of+type+2+diabetes&doi=10.1038%2Fnature06261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule activators of SIRT1 as therapeutics for the treatment of type 2 diabetes</span></div><div class="casAuthors">Milne, Jill C.; Lambert, Philip D.; Schenk, Simon; Carney, David P.; Smith, Jesse J.; Gagne, David J.; Jin, Lei; Boss, Olivier; Perni, Robert B.; Vu, Chi B.; Bemis, Jean E.; Xie, Roger; Disch, Jeremy S.; Ng, Pui Yee; Nunes, Joseph J.; Lynch, Amy V.; Yang, Hongying; Galonek, Heidi; Israelian, Kristine; Choy, Wendy; Iffland, Andre; Lavu, Siva; Medvedik, Oliver; Sinclair, David A.; Olefsky, Jerrold M.; Jirousek, Michael R.; Elliott, Peter J.; Westphal, Christoph H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7170</span>),
    <span class="NLM_cas:pages">712-716</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Calorie restriction extends lifespan and produces a metabolic profile desirable for treating diseases of aging such as type 2 diabetes.  SIRT1, an NAD+-dependent deacetylase, is a principal modulator of pathways downstream of calorie restriction that produce beneficial effects on glucose homeostasis and insulin sensitivity.  Resveratrol, a polyphenolic SIRT1 activator, mimics the antiaging effects of calorie restriction in lower organisms and in mice fed a high-fat diet ameliorates insulin resistance, increases mitochondrial content, and prolongs survival.  Here the authors describe the identification and characterization of small mol. activators of SIRT1 that are structurally unrelated to, and 1000-fold more potent than, resveratrol.  These compds. bind to the SIRT1 enzyme-peptide substrate complex at an allosteric site amino-terminal to the catalytic domain and lower the Michaelis const. for acetylated substrates.  In diet-induced obese and genetically obese mice, these compds. improve insulin sensitivity, lower plasma glucose, and increase mitochondrial capacity.  In Zucker fa/fa rats, hyperinsulinemic-euglycemic clamp studies demonstrate that SIRT1 activators improve whole-body glucose homeostasis and insulin sensitivity in adipose tissue, skeletal muscle and liver.  Thus, SIRT1 activation is a promising new therapeutic approach for treating diseases of aging such as type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZUMUqn50MbVg90H21EOLACvtfcHk0liVktltTKvbpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrM&md5=8343b182d3596cbd8eb175c8b9cdcace</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnature06261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06261%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DSchenk%26aufirst%3DS.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DGagne%26aufirst%3DD.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoss%26aufirst%3DO.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26aulast%3DVu%26aufirst%3DC.%2BB.%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DNg%26aufirst%3DP.%2BY.%26aulast%3DNunes%26aufirst%3DJ.%2BJ.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGalonek%26aufirst%3DH.%26aulast%3DIsraelian%26aufirst%3DK.%26aulast%3DChoy%26aufirst%3DW.%26aulast%3DIffland%26aufirst%3DA.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DMedvedik%26aufirst%3DO.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26aulast%3DOlefsky%26aufirst%3DJ.%2BM.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DElliott%26aufirst%3DP.%2BJ.%26aulast%3DWestphal%26aufirst%3DC.%2BH.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520SIRT1%2520as%2520therapeutics%2520for%2520the%2520treatment%2520of%2520type%25202%2520diabetes%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D712%26epage%3D716%26doi%3D10.1038%2Fnature06261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davoren, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nason, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dlugolenski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaChapelle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salafia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, D.</span></span> <span> </span><span class="NLM_article-title">Discovery and lead optimization of atropisomer D1 agonists with reduced desensitization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">11384</span>– <span class="NLM_lpage">11397</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01622</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01622" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2qtrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=11384-11397&author=J.+E.+Davorenauthor=D.+Nasonauthor=J.+Coeauthor=K.+Dlugolenskiauthor=C.+Helalauthor=A.+R.+Harrisauthor=E.+LaChapelleauthor=S.+Liangauthor=Y.+Liuauthor=R.+O%E2%80%99Connorauthor=C.+C.+Orozcoauthor=B.+K.+Raiauthor=M.+Salafiaauthor=B.+Samasauthor=W.+Xuauthor=R.+Kozakauthor=D.+Gray&title=Discovery+and+lead+optimization+of+atropisomer+D1+agonists+with+reduced+desensitization&doi=10.1021%2Facs.jmedchem.8b01622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Lead Optimization of Atropisomer D1 Agonists with Reduced Desensitization</span></div><div class="casAuthors">Davoren, Jennifer E.; Nason, Deane; Coe, Jotham; Dlugolenski, Keith; Helal, Christopher; Harris, Anthony R.; LaChapelle, Erik; Liang, Sidney; Liu, Yue; OConnor, Rebecca; Orozco, Christine C.; Rai, Brajesh K.; Salafia, Michelle; Samas, Brian; Xu, Wenjian; Kozak, Rouba; Gray, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11384-11397</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of D1 subtype-selective agonists with drug-like properties has been an enduring challenge for the greater part of 40 years.  All known D1-selective agonists are catecholamines that bring about receptor desensitization and undergo rapid metab., thus limiting their utility as a therapeutic for chronic illness such as schizophrenia and Parkinson's disease.  Our high-throughput screening efforts on D1 yielded a single non-catecholamine hit PF-4211 (6) that was developed into a series of potent D1 receptor agonist leads with high oral bioavailability and CNS penetration.  An important structural feature of this series is the locked biaryl ring system resulting in atropisomerism.  Disclosed herein is a summary of our hit-to-lead efforts on this series of D1 activators culminating in the discovery of atropisomer 31 (PF-06256142), a potent and selective orthosteric agonist of the D1 receptor that has reduced receptor desensitization relative to dopamine and other catechol-contg. agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2S5u6FmBDoLVg90H21EOLACvtfcHk0liB5c096Q9EAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2qtrvI&md5=ec84a23922f417fdeb1acd9beb3ad4bf</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01622%26sid%3Dliteratum%253Aachs%26aulast%3DDavoren%26aufirst%3DJ.%2BE.%26aulast%3DNason%26aufirst%3DD.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DDlugolenski%26aufirst%3DK.%26aulast%3DHelal%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DA.%2BR.%26aulast%3DLaChapelle%26aufirst%3DE.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%26aulast%3DOrozco%26aufirst%3DC.%2BC.%26aulast%3DRai%26aufirst%3DB.%2BK.%26aulast%3DSalafia%26aufirst%3DM.%26aulast%3DSamas%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DKozak%26aufirst%3DR.%26aulast%3DGray%26aufirst%3DD.%26atitle%3DDiscovery%2520and%2520lead%2520optimization%2520of%2520atropisomer%2520D1%2520agonists%2520with%2520reduced%2520desensitization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D11384%26epage%3D11397%26doi%3D10.1021%2Facs.jmedchem.8b01622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahusen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Ortiz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, W.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ki, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, C.-X.</span></span> <span> </span><span class="NLM_article-title">Hepatic-specific disruption of SIRT6 in mice results in fatty liver formation due to enhanced glycolysis and triglyceride synthesis</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2010.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cmet.2010.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=20816089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFamsbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=224-236&author=H.-S.+Kimauthor=C.+Xiaoauthor=R.-H.+Wangauthor=T.+Lahusenauthor=X.+Xuauthor=A.+Vassilopoulosauthor=G.+Vazquez-Ortizauthor=W.-I.+Jeongauthor=O.+Parkauthor=S.+H.+Kiauthor=B.+Gaoauthor=C.-X.+Deng&title=Hepatic-specific+disruption+of+SIRT6+in+mice+results+in+fatty+liver+formation+due+to+enhanced+glycolysis+and+triglyceride+synthesis&doi=10.1016%2Fj.cmet.2010.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatic-Specific Disruption of SIRT6 in Mice Results in Fatty Liver Formation Due to Enhanced Glycolysis and Triglyceride Synthesis</span></div><div class="casAuthors">Kim, Hyun-Seok; Xiao, Cuiying; Wang, Rui-Hong; Lahusen, Tyler; Xu, Xiaoling; Vassilopoulos, Athanassios; Vazquez-Ortiz, Guelaguetza; Jeong, Won-Il; Park, Ogyi; Ki, Sung Hwan; Gao, Bin; Deng, Chu-Xia</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">224-236</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Under various conditions, mammals have the ability to maintain serum glucose concn. within a narrow range.  SIRT1 plays an important role in regulating gluconeogenesis and fat metab.; however, the underlying mechanisms remain elusive.  Here, we show that SIRT1 forms a complex with FOXO3a and NRF1 on the SIRT6 promoter and pos. regulates expression of SIRT6, which, in turn, neg. regulates glycolysis, triglyceride synthesis, and fat metab. by deacetylating histone H3 lysine 9 in the promoter of many genes involved in these processes.  Liver-specific deletion of SIRT6 in mice causes profound alterations in gene expression, leading to increased glycolysis, triglyceride synthesis, reduced β oxidn., and fatty liver formation.  Human fatty liver samples exhibited significantly lower levels of SIRT6 than did normal controls.  Thus, SIRT6 plays a crit. role in fat metab. and may serve as a therapeutic target for treating fatty liver disease, the most common cause of liver dysfunction in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3p4P_dbupRLVg90H21EOLACvtfcHk0liB5c096Q9EAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFamsbnP&md5=5912efaedd47bfa7ba209e2681160f95</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2010.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2010.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DXiao%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DR.-H.%26aulast%3DLahusen%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DVassilopoulos%26aufirst%3DA.%26aulast%3DVazquez-Ortiz%26aufirst%3DG.%26aulast%3DJeong%26aufirst%3DW.-I.%26aulast%3DPark%26aufirst%3DO.%26aulast%3DKi%26aufirst%3DS.%2BH.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DDeng%26aufirst%3DC.-X.%26atitle%3DHepatic-specific%2520disruption%2520of%2520SIRT6%2520in%2520mice%2520results%2520in%2520fatty%2520liver%2520formation%2520due%2520to%2520enhanced%2520glycolysis%2520and%2520triglyceride%2520synthesis%26jtitle%3DCell%2520Metab.%26date%3D2010%26volume%3D12%26spage%3D224%26epage%3D236%26doi%3D10.1016%2Fj.cmet.2010.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kokkonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahnasto-Rilla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarho, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahtela-Kakkonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokkola, T.</span></span> <span> </span><span class="NLM_article-title">Studying SIRT6 regulation using H3K56 based substrate and small molecules</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2014.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.ejps.2014.06.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=25004411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Orur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2014&pages=71-76&author=P.+Kokkonenauthor=M.+Rahnasto-Rillaauthor=P.+Melliniauthor=E.+Jarhoauthor=M.+Lahtela-Kakkonenauthor=T.+Kokkola&title=Studying+SIRT6+regulation+using+H3K56+based+substrate+and+small+molecules&doi=10.1016%2Fj.ejps.2014.06.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Studying SIRT6 regulation using H3K56 based substrate and small molecules</span></div><div class="casAuthors">Kokkonen, Piia; Rahnasto-Rilla, Minna; Mellini, Paolo; Jarho, Elina; Lahtela-Kakkonen, Maija; Kokkola, Tarja</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71-76</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">SIRT6 is a modulator of chromatin structure having an important role in healthy ageing, and there is a crucial need to find specific modulators for it.  Therefore, the activity of SIRT6 should be studied using a variety of methods.  We examd. the capability of SIRT6 to deacetylate a set of five fluorogenic substrates based on p53 and histone H3 sequences.  The substrate designed around H3K56 deacetylation site exhibited the best signal-to-background ratio and was chosen for further studies.  Nicotinamide is a known inhibitor for sirtuins, and it was found to be less potent inhibitor for SIRT6 than it is for SIRT1.  In addn., we studied 15 other small mol. sirtuin modulators using the H3K56 based substrate.  EX-527, quercetin and three pseudopeptidic compds. were found to be the most potent SIRT6 inhibitors, exhibiting over 50% deacetylation inhibition.  These findings describe the first modulators of SIRT6 activity at the physiol. important H3K56 deacetylation site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0TYojzjbca7Vg90H21EOLACvtfcHk0liB5c096Q9EAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Orur3L&md5=93cc2b69650d1efebb5dda0128daa559</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2014.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2014.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DKokkonen%26aufirst%3DP.%26aulast%3DRahnasto-Rilla%26aufirst%3DM.%26aulast%3DMellini%26aufirst%3DP.%26aulast%3DJarho%26aufirst%3DE.%26aulast%3DLahtela-Kakkonen%26aufirst%3DM.%26aulast%3DKokkola%26aufirst%3DT.%26atitle%3DStudying%2520SIRT6%2520regulation%2520using%2520H3K56%2520based%2520substrate%2520and%2520small%2520molecules%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2014%26volume%3D63%26spage%3D71%26epage%3D76%26doi%3D10.1016%2Fj.ejps.2014.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0ljkhMlM2UZUvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Modeling water molecules in protein– ligand docking using GOLD</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">6504</span>– <span class="NLM_lpage">6515</span>, <span class="refDoi"> DOI: 10.1021/jm050543p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050543p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFCnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6504-6515&author=M.+L.+Verdonkauthor=G.+Chessariauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=J.+W.+M.+Nissinkauthor=R.+D.+Taylorauthor=R.+Taylor&title=Modeling+water+molecules+in+protein%E2%80%93+ligand+docking+using+GOLD&doi=10.1021%2Fjm050543p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Water Molecules in Protein-Ligand Docking Using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Chessari, Gianni; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Nissink, J. Willem M.; Taylor, Richard D.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6504-6515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We implemented a novel approach to score water mediation and displacement in the protein-ligand docking program GOLD.  The method allows water mols. to switch on and off and to rotate around their three principal axes.  A const. penalty, σp, representing the loss of rigid-body entropy, is added for water mols. that are switched on, hence rewarding water displacement.  We tested the methodol. in an extensive validation study.  First, σp is optimized against a training set of 58 protein-ligand complexes.  For this training set, our algorithm correctly predicts water mediation/displacement in ∼92% of the cases.  We obsd. small improvements in the quality of the predicted binding modes for water-mediated complexes.  In the second part of this work, an entirely independent set of 225 complexes is used.  For this test set, our algorithm correctly predicts water mediation/displacement in ∼93% of the cases.  Improvements in binding mode quality were obsd. for individual water-mediated complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKBGOf-rFSarVg90H21EOLACvtfcHk0ljkhMlM2UZUvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFCnu78%253D&md5=78b60f794d8af7266f0a37f6e2aea6b1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm050543p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050543p%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DModeling%2520water%2520molecules%2520in%2520protein%25E2%2580%2593%2520ligand%2520docking%2520using%2520GOLD%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6504%26epage%3D6515%26doi%3D10.1021%2Fjm050543p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abate-Pella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span> <span> </span><span class="NLM_article-title">Oxidative reactivities of 2-furylquinolines: ubiquitous scaffolds in common high-throughput screening libraries</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7419</span>– <span class="NLM_lpage">7430</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00930</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00930" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVykt7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7419-7430&author=M.+E.+Olsonauthor=D.+Abate-Pellaauthor=A.+L.+Perkinsauthor=M.+Liauthor=M.+A.+Carpenterauthor=A.+Rathoreauthor=R.+S.+Harrisauthor=D.+A.+Harki&title=Oxidative+reactivities+of+2-furylquinolines%3A+ubiquitous+scaffolds+in+common+high-throughput+screening+libraries&doi=10.1021%2Facs.jmedchem.5b00930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative reactivities of 2-furylquinolines: ubiquitous scaffolds in common high-throughput screening libraries</span></div><div class="casAuthors">Olson, Margaret E.; Abate-Pella, Daniel; Perkins, Angela L.; Li, Ming; Carpenter, Michael A.; Rathore, Anurag; Harris, Reuben S.; Harki, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7419-7430</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) was employed to discover APOBEC3G inhibitors, and multiple 2-furylquinolines, e.g. I, were found.  Dose-response assays with I from the HTS sample, as well as com. material, yielded similar confirmatory results.  Interestingly, freshly synthesized and DMSO-solubilized I was inactive.  Repeated screening of the DMSO aliquot of synthesized I revealed increasing APOBEC3G inhibitory activity with age, suggesting that I decomps. into an active inhibitor.  Lab. aging of I followed by anal. revealed that I undergoes oxidative decompn. in air, resulting from a [4+2] cycloaddn. between the furan of I and 1O2.  The resulting endoperoxide then undergoes addnl. transformations, highlighted by Baeyer-Villager rearrangements, to deliver lactam, carboxylic acid, and aldehyde products.  The endoperoxide also undergoes hydrolytic opening followed by further transformations to a bis-enone.  Eight structurally related analogs from HTS libraries were similarly reactive.  This study constitutes a cautionary tale to validate 2-furylquinolines for structure and stability prior to chem. optimization campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3HF7Xmelx8bVg90H21EOLACvtfcHk0ljkhMlM2UZUvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVykt7%252FN&md5=3f7855d8a577e0bf3d9a082cad37ac9a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00930%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DM.%2BE.%26aulast%3DAbate-Pella%26aufirst%3DD.%26aulast%3DPerkins%26aufirst%3DA.%2BL.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DCarpenter%26aufirst%3DM.%2BA.%26aulast%3DRathore%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DR.%2BS.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26atitle%3DOxidative%2520reactivities%2520of%25202-furylquinolines%253A%2520ubiquitous%2520scaffolds%2520in%2520common%2520high-throughput%2520screening%2520libraries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7419%26epage%3D7430%26doi%3D10.1021%2Facs.jmedchem.5b00930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahal, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joswig-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span> <span> </span><span class="NLM_article-title">Comparative study of the affinity and metabolism of type I and type II binding quinoline carboxamide analogues by cytochrome P450 3A4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">280</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1021/jm201207h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201207h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=280-290&author=U.+P.+Dahalauthor=C.+Joswig-Jonesauthor=J.+P.+Jones&title=Comparative+study+of+the+affinity+and+metabolism+of+type+I+and+type+II+binding+quinoline+carboxamide+analogues+by+cytochrome+P450+3A4&doi=10.1021%2Fjm201207h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm201207h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201207h%26sid%3Dliteratum%253Aachs%26aulast%3DDahal%26aufirst%3DU.%2BP.%26aulast%3DJoswig-Jones%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DComparative%2520study%2520of%2520the%2520affinity%2520and%2520metabolism%2520of%2520type%2520I%2520and%2520type%2520II%2520binding%2520quinoline%2520carboxamide%2520analogues%2520by%2520cytochrome%2520P450%25203A4%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D55%26spage%3D280%26epage%3D290%26doi%3D10.1021%2Fjm201207h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spinks, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleghorn, L. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlamb, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frearson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span> <span> </span><span class="NLM_article-title">Investigation of trypanothione reductase as a drug target in Trypanosoma brucei</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">2060</span>– <span class="NLM_lpage">2069</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200900262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1002%2Fcmdc.200900262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=19924760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWrs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=2060-2069&author=D.+Spinksauthor=E.+J.+Shanksauthor=L.+A.+T.+Cleghornauthor=S.+McElroyauthor=D.+Jonesauthor=D.+Jamesauthor=A.+H.+Fairlambauthor=J.+A.+Frearsonauthor=P.+G.+Wyattauthor=I.+H.+Gilbert&title=Investigation+of+trypanothione+reductase+as+a+drug+target+in+Trypanosoma+brucei&doi=10.1002%2Fcmdc.200900262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of Trypanothione Reductase as a Drug Target in Trypanosoma brucei</span></div><div class="casAuthors">Spinks, Daniel; Shanks, Emma J.; Cleghorn, Laura A. T.; McElroy, Stuart; Jones, Deuan; James, Daniel; Fairlamb, Alan H.; Frearson, Julie A.; Wyatt, Paul G.; Gilbert, Ian H.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2060-2069</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">There is an urgent need for new drugs for the treatment of tropical parasitic diseases such as human African trypanosomiasis, which is caused by Trypanosoma brucei.  The enzyme trypanothione reductase (TryR) is a potential drug target within these organisms.  Herein we report the screening of a 62 000 compd. library against T. brucei TryR.  Further work was undertaken to optimize potency and selectivity of two novel-compd. series arising from the enzymic and whole parasite screens and mammalian cell counter-screens.  Both of these series, contg. either a quinoline or pyrimidinopyrazine scaffold, yielded low micromolar inhibitors of the enzyme and growth of the parasite.  The challenges of inhibiting TryR with drug-like mols. is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-k1JvjbQWpLVg90H21EOLACvtfcHk0liENnjMAyRlzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWrs7%252FI&md5=444d2fed0ad1342ed8ee54c405549b4d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900262%26sid%3Dliteratum%253Aachs%26aulast%3DSpinks%26aufirst%3DD.%26aulast%3DShanks%26aufirst%3DE.%2BJ.%26aulast%3DCleghorn%26aufirst%3DL.%2BA.%2BT.%26aulast%3DMcElroy%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26aulast%3DFrearson%26aufirst%3DJ.%2BA.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26atitle%3DInvestigation%2520of%2520trypanothione%2520reductase%2520as%2520a%2520drug%2520target%2520in%2520Trypanosoma%2520brucei%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D2060%26epage%3D2069%26doi%3D10.1002%2Fcmdc.200900262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of 2-phenylquinoline-4-carboxamide derivatives as a new class of tubulin polymerization inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5939</span>– <span class="NLM_lpage">5951</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.bmc.2017.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=28988750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=5939-5951&author=L.+Zhuauthor=K.+Luoauthor=K.+Liauthor=Y.+Jinauthor=J.+Lin&title=Design%2C+synthesis+and+biological+evaluation+of+2-phenylquinoline-4-carboxamide+derivatives+as+a+new+class+of+tubulin+polymerization+inhibitors&doi=10.1016%2Fj.bmc.2017.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of 2-phenylquinoline-4-carboxamide derivatives as a new class of tubulin polymerization inhibitors</span></div><div class="casAuthors">Zhu, Li; Luo, Kaixiu; Li, Ke; Jin, Yi; Lin, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5939-5951</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of 2-phenylquinoline-4-carboxamide derivs. was synthesized, characterized and evaluated for its antiproliferative activity against five cancer cell lines, Hela, SK-OV-3, HCT116, A549 and MDA-MB-468, and a normal human fetal lung fibroblastic cell line, MRC-5.  Among them, compd. I displayed potent cytotoxic activity in vitro against SK-OV-3 and HCT116 cell lines with IC50 values of 0.5 and 0.2 μM, resp.  In general, the antiproliferative activity was correlated with the binding property of the colchicine binding site and inhibitory effect on tubulin polymn.  In addn., immunofluorescence and flow cytometry anal. revealed that selected compds. caused disruption of the mitotic spindle assembly and G2/M phase arrest of the cell cycle, which correlated with proliferation inhibitory activity.  Mol. docking anal. demonstrated the interaction of I at the colchicine binding site of tubulin.  These results indicate these compds. are promising inhibitors of tubulin polymn. for the potent treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu54Rs5g5vJrVg90H21EOLACvtfcHk0liENnjMAyRlzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzF&md5=f6ae769bfd1f7670e16dd23abcd32a69</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DJ.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%25202-phenylquinoline-4-carboxamide%2520derivatives%2520as%2520a%2520new%2520class%2520of%2520tubulin%2520polymerization%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D5939%26epage%3D5951%26doi%3D10.1016%2Fj.bmc.2017.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span> <span> </span><span class="NLM_article-title">Novel mitochondria-targeted and fluorescent DNA alkylation agents with highly selective activity against cancer cells</span>. <i>Dyes Pigm.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">107610</span>, <span class="refDoi"> DOI: 10.1016/j.dyepig.2019.107610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.dyepig.2019.107610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFOjs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2019&pages=107610&author=X.+Chenauthor=W.+Pengauthor=S.+Huangauthor=C.+Yangauthor=M.+Huauthor=S.+Yangauthor=S.+Yangauthor=Y.+Xieauthor=H.+Chenauthor=N.+Leiauthor=Y.+Luoauthor=K.+Li&title=Novel+mitochondria-targeted+and+fluorescent+DNA+alkylation+agents+with+highly+selective+activity+against+cancer+cells&doi=10.1016%2Fj.dyepig.2019.107610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mitochondria-targeted and fluorescent DNA alkylation agents with highly selective activity against cancer cells</span></div><div class="casAuthors">Chen, Xiuli; Peng, Wentao; Huang, Shenzhen; Yang, Chao; Hu, Mingxing; Yang, Shuping; Yang, Shengyong; Xie, Yongmei; Chen, Hao; Lei, Ning; Luo, Yi; Li, Kun</div><div class="citationInfo"><span class="NLM_cas:title">Dyes and Pigments</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107610</span>CODEN:
                <span class="NLM_cas:coden">DYPIDX</span>;
        ISSN:<span class="NLM_cas:issn">0143-7208</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of novel nitrogen mustard-based fluorescent compds. were reported with good mitochondrial targeting ability and DNA alkylation activity.  The results indicated that these compds. can be used directly as reporting and imaging agents for gel electrophoresis and flow cytometry without the necessity of addnl. fluorescent tagging agents.  Notably, the compd. CXL92 displayed highly selective activity against lung cancer cell line A549 with an IC50 value of 0.32 μM compared to normal cell (LO2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm9LdinJB4qrVg90H21EOLACvtfcHk0liENnjMAyRlzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFOjs7fP&md5=ab03d172c0f6041afb644f3cce45952f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.dyepig.2019.107610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dyepig.2019.107610%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLei%26aufirst%3DN.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DK.%26atitle%3DNovel%2520mitochondria-targeted%2520and%2520fluorescent%2520DNA%2520alkylation%2520agents%2520with%2520highly%2520selective%2520activity%2520against%2520cancer%2520cells%26jtitle%3DDyes%2520Pigm.%26date%3D2019%26volume%3D170%26spage%3D107610%26doi%3D10.1016%2Fj.dyepig.2019.107610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krzyzosiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigurdardottir, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolotti, A.</span></span> <span> </span><span class="NLM_article-title">Target-based discovery of an inhibitor of the regulatory phosphatase PPP1R15B</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1228.e19</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.06.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2018.06.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=30057111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2018&pages=1216-1228.e19&author=A.+Krzyzosiakauthor=A.+Sigurdardottirauthor=L.+Luhauthor=M.+Carraraauthor=I.+Dasauthor=K.+Schneiderauthor=A.+Bertolotti&title=Target-based+discovery+of+an+inhibitor+of+the+regulatory+phosphatase+PPP1R15B&doi=10.1016%2Fj.cell.2018.06.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Target-Based Discovery of an Inhibitor of the Regulatory Phosphatase PPP1R15B</span></div><div class="casAuthors">Krzyzosiak, Agnieszka; Sigurdardottir, Anna; Luh, Laura; Carrara, Marta; Das, Indrajit; Schneider, Kim; Bertolotti, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1216-1228.e19</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein phosphorylation is a prevalent and ubiquitous mechanism of regulation.  Kinases are popular drug targets, but identifying selective phosphatase inhibitors has been challenging.  Here, we used surface plasmon resonance to design a method to enable target-based discovery of selective serine/threonine phosphatase inhibitors.  The method targeted a regulatory subunit of protein phosphatase 1, PPP1R15B, a neg. regulator of proteostasis.  This yielded Raphin1, a selective inhibitor of R15B.  In cells, Raphin1 caused a rapid and transient accumulation of its phosphorylated substrate, resulting in a transient attenuation of protein synthesis.  In vitro, Raphin1 inhibits the recombinant R15B-PP1c holoenzyme, but not the closely related R15A-PP1c, by interfering with substrate recruitment.  Raphin1 was orally bioavailable, crossed the blood-brain barrier, and demonstrated efficacy in a mouse model of Huntington's disease.  This identifies R15B as a druggable target and provides a platform for target-based discovery of inhibitors of serine/threonine phosphatases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnL1IexJPGE7Vg90H21EOLACvtfcHk0lgarikg5yCDLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsLbL&md5=09415366927bac424f14e9cce05d9385</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.06.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.06.030%26sid%3Dliteratum%253Aachs%26aulast%3DKrzyzosiak%26aufirst%3DA.%26aulast%3DSigurdardottir%26aufirst%3DA.%26aulast%3DLuh%26aufirst%3DL.%26aulast%3DCarrara%26aufirst%3DM.%26aulast%3DDas%26aufirst%3DI.%26aulast%3DSchneider%26aufirst%3DK.%26aulast%3DBertolotti%26aufirst%3DA.%26atitle%3DTarget-based%2520discovery%2520of%2520an%2520inhibitor%2520of%2520the%2520regulatory%2520phosphatase%2520PPP1R15B%26jtitle%3DCell%26date%3D2018%26volume%3D174%26spage%3D1216%26epage%3D1228.e19%26doi%3D10.1016%2Fj.cell.2018.06.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poddar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surolia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, B.</span></span> <span> </span><span class="NLM_article-title">Sulfonamide drugs binding to the colchicine site of tubulin: thermodynamic analysis of the drug– tubulin interactions by isothermal titration calorimetry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1021/jm0494974</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0494974" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkvVGltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=547-555&author=M.+Banerjeeauthor=A.+Poddarauthor=G.+Mitraauthor=A.+Suroliaauthor=T.+Owaauthor=B.+Bhattacharyya&title=Sulfonamide+drugs+binding+to+the+colchicine+site+of+tubulin%3A+thermodynamic+analysis+of+the+drug%E2%80%93+tubulin+interactions+by+isothermal+titration+calorimetry&doi=10.1021%2Fjm0494974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfonamide Drugs Binding to the Colchicine Site of Tubulin: Thermodynamic Analysis of the Drug-Tubulin Interactions by Isothermal Titration Calorimetry</span></div><div class="casAuthors">Banerjee, Mithu; Poddar, Asim; Mitra, Gopa; Surolia, Avadhesha; Owa, Takashi; Bhattacharyya, Bhabatarak</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">547-555</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of several sulfonamide drugs paved the way toward the synthesis of 6 (N-[2-[(4-hydroxyphenyl)amino]-3-pyridinyl]-4-methoxybenzenesulfonamide, E7010) and 7 (N-(3-fluoro-4-methoxyphenyl)pentafluorobenzenesulfonamide, T138067), both of which inhibit tubulin polymn. and are under clin. development.  A series of diarylsulfonamides contg. an indole scaffold was also found to have antimitotic properties, but their mode of interactions with tubulin has remained unidentified so far.  In this study, the authors demonstrate that these sulfonamide drugs bind to the colchicine site of tubulin in a reversible manner.  They quenched intrinsic tryptophan fluorescence of tubulin presumably due to drug-induced conformational changes in the protein, but were unable to modulate GTPase activity of tubulin in contrast to colchicine that enhances the same enzymic activity.  Further investigation using isothermal titrn. calorimetry (ITC) revealed that 5 (N-(5-chloro-7-indolyl)-4-methoxybenzenesulfonamide) afforded a large pos. value of heat capacity change (ΔCp = +264 cal mol-1 K-1) on binding to tubulin, suggesting a substantial conformational transition in the protein along with partial enthalpy-entropy compensation.  On the other hand, the 2-chloro regioisomer 2 gave a large neg. value of ΔCp (-589 cal mol-1 K-1) along with complete enthalpy-entropy compensation.  This thermodn. profile was thought to be attributable to a prominent contribution of van der Waals interaction and hydrogen bonding between specific groups in the drug-tubulin complex.  These results indicate that a mere alteration in the position of a single substituent chlorine on the indole scaffold has a great influence on the drug-tubulin binding thermodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFXpusdp7zOrVg90H21EOLACvtfcHk0lgarikg5yCDLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkvVGltA%253D%253D&md5=c248bd33dc0d2831494869fe97b1cb41</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm0494974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0494974%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DM.%26aulast%3DPoddar%26aufirst%3DA.%26aulast%3DMitra%26aufirst%3DG.%26aulast%3DSurolia%26aufirst%3DA.%26aulast%3DOwa%26aufirst%3DT.%26aulast%3DBhattacharyya%26aufirst%3DB.%26atitle%3DSulfonamide%2520drugs%2520binding%2520to%2520the%2520colchicine%2520site%2520of%2520tubulin%253A%2520thermodynamic%2520analysis%2520of%2520the%2520drug%25E2%2580%2593%2520tubulin%2520interactions%2520by%2520isothermal%2520titration%2520calorimetry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D547%26epage%3D555%26doi%3D10.1021%2Fjm0494974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schnapp, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoevels, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakker, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span> <span> </span><span class="NLM_article-title">Comparative analysis of binding kinetics and thermodynamics of dipeptidyl peptidase-4 inhibitors and their relationship to structure</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7466</span>– <span class="NLM_lpage">7477</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00475</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00475" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SisbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7466-7477&author=G.+Schnappauthor=T.+Kleinauthor=Y.+Hoevelsauthor=R.+A.+Bakkerauthor=H.+Nar&title=Comparative+analysis+of+binding+kinetics+and+thermodynamics+of+dipeptidyl+peptidase-4+inhibitors+and+their+relationship+to+structure&doi=10.1021%2Facs.jmedchem.6b00475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative Analysis of Binding Kinetics and Thermodynamics of Dipeptidyl Peptidase-4 Inhibitors and Their Relationship to Structure</span></div><div class="casAuthors">Schnapp, Gisela; Klein, Thomas; Hoevels, Yvette; Bakker, Remko A.; Nar, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7466-7477</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding kinetics and thermodn. of dipeptidyl peptidase (DPP)-4 inhibitors (gliptins) were investigated using surface plasmon resonance and isothermal titrn. calorimetry.  Binding of gliptins to DPP-4 is a rapid electrostatically driven process.  Off-rates were generally slow partly because of reversible covalent bond formation by some gliptins, and partly because of strong and extensive interactions.  Binding of all gliptins is enthalpy-dominated due to strong ionic interactions and strong solvent-shielded hydrogen bonds.  Using a congeneric series of mols. which represented the intermediates in the lead optimization program of linagliptin, the onset of slow binding kinetics and development of the thermodn. repertoire were analyzed in the context of incremental changes of the chem. structures.  All compds. rapidly assocd., and therefore the optimization of affinity and residence time is highly correlated.  The major contributor to the increasing free energy of binding was a strong increase of binding enthalpy, whereas entropic contributions remained low and const. despite significant addn. of lipophilicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7s_u2QnM5OrVg90H21EOLACvtfcHk0lgarikg5yCDLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SisbrE&md5=95398602908df300f76c24677b5af218</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00475%26sid%3Dliteratum%253Aachs%26aulast%3DSchnapp%26aufirst%3DG.%26aulast%3DKlein%26aufirst%3DT.%26aulast%3DHoevels%26aufirst%3DY.%26aulast%3DBakker%26aufirst%3DR.%2BA.%26aulast%3DNar%26aufirst%3DH.%26atitle%3DComparative%2520analysis%2520of%2520binding%2520kinetics%2520and%2520thermodynamics%2520of%2520dipeptidyl%2520peptidase-4%2520inhibitors%2520and%2520their%2520relationship%2520to%2520structure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7466%26epage%3D7477%26doi%3D10.1021%2Facs.jmedchem.6b00475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span> <span> </span><span class="NLM_article-title">Novel mitochondria-targeted, nitrogen mustard-based DNA alkylation agents with near infrared fluorescence emission</span>. <i>Talanta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">888</span>– <span class="NLM_lpage">893</span>, <span class="refDoi"> DOI: 10.1016/j.talanta.2016.08.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.talanta.2016.08.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=27769499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2nsrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2016&pages=888-893&author=X.+Chenauthor=H.+Chenauthor=C.+Luauthor=C.+Yangauthor=X.+Yuauthor=K.+Liauthor=Y.+Xie&title=Novel+mitochondria-targeted%2C+nitrogen+mustard-based+DNA+alkylation+agents+with+near+infrared+fluorescence+emission&doi=10.1016%2Fj.talanta.2016.08.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mitochondria-targeted, nitrogen mustard-based DNA alkylation agents with near infrared fluorescence emission</span></div><div class="casAuthors">Chen, Xiuli; Chen, Hao; Lu, Chunyan; Yang, Chao; Yu, Xiaoqi; Li, Kun; Xie, Yongmei</div><div class="citationInfo"><span class="NLM_cas:title">Talanta</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">888-893</span>CODEN:
                <span class="NLM_cas:coden">TLNTA2</span>;
        ISSN:<span class="NLM_cas:issn">0039-9140</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this report, the design of two novel nitrogen mustard-based DNA crosslinking agents with fluorophores incorporated into the structure (TN-A(I) and TN-B (II)) is disclosed.  The results indicate that TN-A and TN-B can serve directly as both reporting and imaging agents for flow cytometry and gel electrophoresis without the necessity of another fluorescent tagging agents.  TN-A and TN-B both selectively locate in the mitochondria and exhibit good antitumor activity.  Notably, TN-A is the first DNA crosslinking agent with near IR fluorescence emission properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4XRlNbPJFqrVg90H21EOLACvtfcHk0lgM8v_p3rRXoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2nsrfL&md5=31a0a47f2ac33f53f08ec1a4a08ad2b7</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.talanta.2016.08.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.talanta.2016.08.051%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DY.%26atitle%3DNovel%2520mitochondria-targeted%252C%2520nitrogen%2520mustard-based%2520DNA%2520alkylation%2520agents%2520with%2520near%2520infrared%2520fluorescence%2520emission%26jtitle%3DTalanta%26date%3D2016%26volume%3D161%26spage%3D888%26epage%3D893%26doi%3D10.1016%2Fj.talanta.2016.08.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-Y.</span></span> <span> </span><span class="NLM_article-title">A preclinical evaluation of SKLB261, a multikinase inhibitor of EGFR/Src/VEGFR2, as a therapeutic agent against pancreatic cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-14-0485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1158%2F1535-7163.mct-14-0485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=25519702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlOjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=407-418&author=Y.+Panauthor=M.+Zhengauthor=L.+Zhongauthor=J.+Yangauthor=S.+Zhouauthor=Y.+Qinauthor=R.+Xiangauthor=Y.+Chenauthor=S.-Y.+Yang&title=A+preclinical+evaluation+of+SKLB261%2C+a+multikinase+inhibitor+of+EGFR%2FSrc%2FVEGFR2%2C+as+a+therapeutic+agent+against+pancreatic+cancer&doi=10.1158%2F1535-7163.mct-14-0485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A Preclinical Evaluation of SKLB261, a Multikinase Inhibitor of EGFR/Src/VEGFR2, as a Therapeutic Agent against Pancreatic Cancer</span></div><div class="casAuthors">Pan, Youli; Zheng, Mingwu; Zhong, Lei; Yang, Jiao; Zhou, Shu; Qin, Ya; Xiang, Rong; Chen, Yuzong; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">407-418</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The clin. prognosis of pancreatic cancer remains rather disappointing despite tremendous efforts in exploring medical treatments in the past two decades.  Development of more effective treatment strategies is still desperately needed to improve outcomes in patients with pancreatic cancer.  SKLB261 is a multikinase inhibitor obtained recently through a lead optimization.  In this investigation, we shall evaluate its anti-pancreatic cancer effects both in vitro and in vivo.  SKLB261 is a multikinase inhibitor potently inhibiting EGFR, Src, and VEGFR2 kinases.  It could significantly inhibit cell proliferation, migration, and invasion, and induce apoptosis in cellular assays of human pancreatic cancer cells that are sensitive or resistant to dasatinib and/or gemcitabine.  Western blot anal. showed that SKLB261 inhibited the activation of EGFR and Src kinases as well as their downstream signaling proteins, including FAK, ERK, and STAT3.  SKLB261 also showed potent antiangiogenic effects in transgenic zebrafish models.  In vivo, SKLB261 displayed more potent antitumor activities than dasatinib, gemcitabine, or erlotinib in pancreatic cancer xenografts, including BxPC-3, PANC-1, AsPC-1, and HPAC.  Furthermore, mice receiving SKLB261 therapy showed significant survival advantage compared with vehicle-treated and gemcitabine-treated groups in an exptl. metastasis model of pancreatic cancer.  These data, together with the good pharmacokinetic properties and low toxicity of this compd., provide a rationale for the ongoing clin. evaluation of SKLB261 in the treatment of pancreatic cancer.  Mol Cancer Ther; 14(2); 407-18. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOnyoK8YlU1LVg90H21EOLACvtfcHk0lgM8v_p3rRXoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlOjsbs%253D&md5=883fbc5c22d1a54b0a95ac2ff279a038</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-14-0485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-14-0485%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DQin%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.-Y.%26atitle%3DA%2520preclinical%2520evaluation%2520of%2520SKLB261%252C%2520a%2520multikinase%2520inhibitor%2520of%2520EGFR%252FSrc%252FVEGFR2%252C%2520as%2520a%2520therapeutic%2520agent%2520against%2520pancreatic%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3D407%26epage%3D418%26doi%3D10.1158%2F1535-7163.mct-14-0485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-Y.</span></span> <span> </span><span class="NLM_article-title">Preclinical evaluation of a novel orally available SRC/Raf/VEGFR2 inhibitor, SKLB646, in the treatment of triple-negative breast cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">378</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-15-0501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1158%2F1535-7163.mct-15-0501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26721945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslCmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=366-378&author=M.-W.+Zhengauthor=C.-H.+Zhangauthor=K.+Chenauthor=M.+Huangauthor=Y.-P.+Liauthor=W.-T.+Linauthor=R.-J.+Zhangauthor=L.+Zhongauthor=R.+Xiangauthor=L.-L.+Liauthor=X.-Y.+Liuauthor=Y.-Q.+Weiauthor=S.-Y.+Yang&title=Preclinical+evaluation+of+a+novel+orally+available+SRC%2FRaf%2FVEGFR2+inhibitor%2C+SKLB646%2C+in+the+treatment+of+triple-negative+breast+cancer&doi=10.1158%2F1535-7163.mct-15-0501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Evaluation of a Novel Orally Available SRC/Raf/VEGFR2 Inhibitor, SKLB646, in the Treatment of Triple-Negative Breast Cancer</span></div><div class="casAuthors">Zheng, Ming-Wu; Zhang, Chun-Hui; Chen, Kai; Huang, Mei; Li, Ya-Ping; Lin, Wan-Ting; Zhang, Rong-Jie; Zhong, Lei; Xiang, Rong; Li, Lin-Li; Liu, Xin-Yu; Wei, Yu-Quan; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">366-378</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is the most aggressive and deadly breast cancer subtype.  To date, chemotherapy is the only systemic therapy and prognosis remains poor.  Herein, we report the preclin. evaluation of SKLB646 in the treatment of TNBC; SKLB646 is a novel multiple kinase inhibitor developed by us recently.  This compd. potently inhibited SRC and VEGFR2 with IC50 values of 0.002 μmol/L and 0.012 μmol/L, resp.  It also considerably inhibited B-Raf and C-Raf with IC50 values of 0.022 and 0.019 μmol/L, resp.  It exhibited significant antiproliferation and antiviability activities against TNBC cell lines.  Studies of mechanism of action indicated that SKLB646 inhibited the activation of SRC signaling and blocked the MAPK signaling through inhibiting the Raf kinases.  Interestingly, SKLB646 dose dependently downregulated the expression of Fra1, a transcriptional factor that plays a crit. role in the epithelial-to-mesenchymal transition.  In addn., SKLB646 could inhibit HUVEC proliferation, migration, and invasion.  It effectively blocked the formation of intersegmental vessels in zebrafish embryos and displayed considerable antiangiogenic effects in the tumor-induced neovascularization zebrafish model.  In TNBC xenograft models, SKLB646 suppressed the tumor growth in a dose-dependent manner.  Moreover, SKLB646 could remarkably inhibit TNBC cell migration and invasion in vitro.  Furthermore, in an exptl. lung metastasis model, the overall survival time of groups treated with SKLB646 was much longer compared with the control-, dasatinib-, and paclitaxel-treated groups.  In a preliminary pharmacokinetic study, SKLB646 showed good pharmacokinetic properties.  Taken together, the preclin. data show that SKLB646 could be a promising lead compd. for the treatment of TNBC.  Mol Cancer Ther; 15(3); 366-78. 2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodSEWSFArBprVg90H21EOLACvtfcHk0lgM8v_p3rRXoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslCmsL8%253D&md5=5503431d68d044995a94faa8948f27ba</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-15-0501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-15-0501%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DM.-W.%26aulast%3DZhang%26aufirst%3DC.-H.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DY.-P.%26aulast%3DLin%26aufirst%3DW.-T.%26aulast%3DZhang%26aufirst%3DR.-J.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DL.-L.%26aulast%3DLiu%26aufirst%3DX.-Y.%26aulast%3DWei%26aufirst%3DY.-Q.%26aulast%3DYang%26aufirst%3DS.-Y.%26atitle%3DPreclinical%2520evaluation%2520of%2520a%2520novel%2520orally%2520available%2520SRC%252FRaf%252FVEGFR2%2520inhibitor%252C%2520SKLB646%252C%2520in%2520the%2520treatment%2520of%2520triple-negative%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D366%26epage%3D378%26doi%3D10.1158%2F1535-7163.mct-15-0501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Francesco Fiorentino, Antonello Mai, <span class="NLM_string-name hlFld-ContribAuthor">Dante Rotili</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging Therapeutic Potential of SIRT6 Modulators. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 9732-9758. <a href="https://doi.org/10.1021/acs.jmedchem.1c00601" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00601</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00601%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEmerging%252BTherapeutic%252BPotential%252Bof%252BSIRT6%252BModulators%26aulast%3DFiorentino%26aufirst%3DFrancesco%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01042021%26date%3D02072021%26volume%3D64%26issue%3D14%26spage%3D9732%26epage%3D9758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesco  Fiorentino</span>, <span class="hlFld-ContribAuthor ">Vincenzo  Carafa</span>, <span class="hlFld-ContribAuthor ">Gregorio  Favale</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>. </span><span class="cited-content_cbyCitation_article-title">The Two-Faced Role of SIRT6 in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (5)
                                     , 1156. <a href="https://doi.org/10.3390/cancers13051156" title="DOI URL">https://doi.org/10.3390/cancers13051156</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13051156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13051156%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DThe%252BTwo-Faced%252BRole%252Bof%252BSIRT6%252Bin%252BCancer%26aulast%3DFiorentino%26aufirst%3DFrancesco%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D5%26spage%3D1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Jiacheng  Wei</span>, <span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Ashfaq Ur  Rehman</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Jun  Pu</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of cryptic allosteric sites using reversed allosteric communication by a combined computational and experimental strategy. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     , 464-476. <a href="https://doi.org/10.1039/D0SC05131D" title="DOI URL">https://doi.org/10.1039/D0SC05131D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC05131D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC05131D%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DDiscovery%252Bof%252Bcryptic%252Ballosteric%252Bsites%252Busing%252Breversed%252Ballosteric%252Bcommunication%252Bby%252Ba%252Bcombined%252Bcomputational%252Band%252Bexperimental%252Bstrategy%26aulast%3DNi%26aufirst%3DDuan%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1%26spage%3D464%26epage%3D476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Reported synthetic SIRT6 activators.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>In vitro</i> SIRT6 enzymatic activity assay for the screened compounds. (A) Activation of SIRT6-dependent peptide demyristoylation by 40 selected compounds at 100 μM, determined with Ac-EALPKK(Myr)-AMC by the FDL assay. (B) Chemical structures of hit compounds <b>Hit05</b>, <b>Hit07</b>, <b>Hit11</b>, and <b>Hit20</b>. (C) Dose-dependent demyristoylation activities of SIRT6 by 4 hit compounds, determined with Ac-EALPKK(Myr)-AMC by the FDL assay. Data are presented as the mean ± s.d., <i>n</i> = 3 wells, from three independent experiments. (D) Target regions for the structural optimization and SAR analyses.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>5a–o</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KOH, EtOH, 85 °C, 24 h; (b) (i) SOCl<sub>2</sub>, 107 °C, 5–6 h; (ii) R<sup>1</sup>-NH<sub>2</sub>, DMAP, DMF, RT, overnight.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>8a–m</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KOH, EtOH, 85 °C, 24 h; (b) (i) SOCl<sub>2</sub>, 107 °C, 5–6 h; (ii) 2-chloro-5-(trifluoromethyl)aniline, DMAP, DMF, RT, overnight.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>12a–q</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KOH, EtOH, 85 °C, 24 h; (b) R<sup>3</sup>-NH<sub>2</sub>, HATU, HOBt, DMAP, 50 °C, overnight.</p></p></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Bioactivities of <b>12q</b> against SIRT6 by various biochemical and biophysical assays. (A,B) Dose-activity curves of SIRT6-dependant deacetylation and demyristoylation of <b>12q</b>, determined with acetyl substrate peptide Ac-RYQK(Ac)-AMC and myristoyl substrate peptide Ac-EALPKK(Myr)-AMC. Data are presented as the mean ± s.d., <i>n</i> = 3 wells, from three independent experiments. (C) Changes in the thermodynamic stability of SIRT6 after the binding of <b>12q</b> or <b>MDL-800</b>. (D) ITC binding curves for SIRT6 and <b>12q</b>. (E) Representative SIRT6 binding curves and fit steady-state evaluation for <b>12q</b> using SPR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Predicted binding mode of <b>12q</b> with SIRT6 (the SIRT6 structure was taken from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Y2F">5Y2F</a>). <b>12q</b> (green), ADPR (apricot), and H3K9-Myr (magenta) are shown in the “zoomed-in” image.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Antiproliferative activities of <b>12q</b> against pancreatic cancer cells <i>in vitro</i>. (A,B) Cells were treated with the indicated agents for 72 h and cell viability was measured by an MTT assay. Every experiment was carried out in triplicate. (C) Cells were incubated with various concentrations of the indicated agents for 18 days (BXPC-3) or 14 days (PANC-1). Then, the cells were stained with crystal violet and quantified. (D) Fluorescence microscopic appearance of EdU and Hoechst on BXPC-3 and PANC-1 cells after treatment with the indicated agents for 24 h. EdU-positive cells were quantified in 10 fields. Scale bars, 100 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Antimigration ability of <b>12q</b> and its bioactivities in intact cells. (A) Antimigration activity of <b>12q</b> against pancreatic cancer cells <i>in vitro</i>. BXPC-3 or PANC-1 cells were treated with various concentrations of the indicated agents. Scale bars, 100 μm. (B) Western blot analysis of SIRT6 and its substrates H3K9ac, H3K18ac, and H3K56ac and Lin28b, c-Myc, and IGF2BP3 protein expression in PANC-1 and BXPC-3 cells treated with the indicated doses of <b>12q</b> and <b>MDL-800</b> for 72 h. (C) CETSA was implemented in intact cells exposed to <b>12q</b> (25 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/medium/jm0c01183_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Antitumor efficacy of <b>12q</b><i>in vivo</i>. (A) Mice-bearing PANC-1 tumor xenografts were treated with agents at the indicated dose or vehicle control alone over the designated treatment schedule once tumors reached the determined size (<i>n</i> = 6). Points, mean tumor volume; bars, SD. (B) Quantitative analysis of tumor volumes on the final study day. ***<i>P</i> < 0.001. (C) Tumor tissue from PANC-1 xenografts treated with vehicle control, <b>12q</b> (150 mg/kg), or reference agents at the dose mentioned above were evaluated by IHC. Scale bars, 50 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c01183/20200918/images/large/jm0c01183_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01183&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i118">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 56 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlicker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boanca, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steegborn, C.</span></span> <span> </span><span class="NLM_article-title">Structure-based development of novel sirtuin inhibitors</span>. <i>Aging</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">852</span>– <span class="NLM_lpage">872</span>, <span class="refDoi"> DOI: 10.18632/aging.100388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.18632%2Faging.100388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=21937767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCht7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=852-872&author=C.+Schlickerauthor=G.+Boancaauthor=M.+Lakshminarasimhanauthor=C.+Steegborn&title=Structure-based+development+of+novel+sirtuin+inhibitors&doi=10.18632%2Faging.100388"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based development of novel sirtuin inhibitors</span></div><div class="casAuthors">Schlicker, Christine; Boanca, Gina; Lakshminarasimhan, Mahadevan; Steegborn, Clemens</div><div class="citationInfo"><span class="NLM_cas:title">Aging</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">852-872</span>CODEN:
                <span class="NLM_cas:coden">AGINCN</span>;
        ISSN:<span class="NLM_cas:issn">1945-4589</span>.
    
            (<span class="NLM_cas:orgname">Impact Journals LLC</span>)
        </div><div class="casAbstract">Sirtuins are NAD+-dependent protein deacetylases regulating metab., stress responses, and aging processes.  Mammalia possess seven Sirtuin isoforms, Sirt1-7, which differ in their subcellular localization and in the substrate proteins they deacetylate.  The physiol. roles of Sirtuins and their potential use as therapeutic targets for metabolic and agingrelated diseases have spurred interest in the development of small-mol. Sirtuin modulators.  Here, we describe an approach exploiting the structures available for four human Sirtuins for the development of isoform-specific inhibitors.  Virtual docking of a compd. library into the peptide binding pockets of crystal structures of Sirt2, 3, 5 and 6 yielded compds. potentially discriminating between these isoforms.  Further characterization in activity assays revealed several inhibitory compds. with little isoform specificity, but also two compds. with micromolar potency and high specificity for Sirt2.  Structure comparison and the predicted, shared binding mode of the Sirt2-specific compds. indicate a pocket extending from the peptide-binding groove as target side enabling isoform specificity.  Our family-wide structure-based approach thus identified potent, Sirt2-specific inhibitors as well as lead structures and a target site for the development of compds. specific for other Sirtuin isoform, constituting an important step toward the identification of a complete panel of isoform-specific Sirtuin inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHw485oGKwoLVg90H21EOLACvtfcHk0lh3x0wwqm6eYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCht7fO&md5=77d30598d6b2e4aa536577b3b3a9e461</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.18632%2Faging.100388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Faging.100388%26sid%3Dliteratum%253Aachs%26aulast%3DSchlicker%26aufirst%3DC.%26aulast%3DBoanca%26aufirst%3DG.%26aulast%3DLakshminarasimhan%26aufirst%3DM.%26aulast%3DSteegborn%26aufirst%3DC.%26atitle%3DStructure-based%2520development%2520of%2520novel%2520sirtuin%2520inhibitors%26jtitle%3DAging%26date%3D2011%26volume%3D3%26spage%3D852%26epage%3D872%26doi%3D10.18632%2Faging.100388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imai, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaeberlein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarente, L.</span></span> <span> </span><span class="NLM_article-title">Transcriptional silencing and longevity protein Sir2 is an NAD-dependent histone deacetylase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>403</i></span>,  <span class="NLM_fpage">795</span>, <span class="refDoi"> DOI: 10.1038/35001622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2F35001622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10693811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhsVWiurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=795&author=S.-I.+Imaiauthor=C.+M.+Armstrongauthor=M.+Kaeberleinauthor=L.+Guarente&title=Transcriptional+silencing+and+longevity+protein+Sir2+is+an+NAD-dependent+histone+deacetylase&doi=10.1038%2F35001622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional silencing and longevity protein Sir2 is an NAD-dependent histone deacetylase</span></div><div class="casAuthors">Imai, Shin-Ichiro; Armstrong, Christopher M.; Kaebertein, Matt; Guarente, Leonard</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6771</span>),
    <span class="NLM_cas:pages">795-800</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Yeast Sir2 is a heterochromatin component that silences transcription at silent mating loci, telomeres and the ribosomal DNA, and that also suppresses recombination in the rDNA and extends replicative life span.  Mutational studies indicate that lysine 16 in the amino-terminal tail of histone H4 and lysines 9, 14 and 18 in H3 are critically important in silencing, whereas lysines 5, 8 and 12 of H4 have more redundant functions.  Lysines 9 and 14 of histone H3 and lysines 5, 8 and 16 of H4 are acetylated in active chromatin and hypoacetylated in silenced chromatin, and overexpression of Sir2 promotes global deacetylation of histones, indicating that Sir2 may be a histone deacetylase.  Deacetylation of lysine 16 of H4 is necessary for binding the silencing protein, Sir3 (ref. 8).  Here we show that yeast and mouse Sir2 proteins are NAD- dependent histone deacetylases, which deacetylate lysines 9 and 14 of H3 and specifically lysine 16 of H4.  Our anal. of two SIR2 mutations supports the idea that this deacetylase activity accounts for silencing, recombination suppression and extension of life span in vivo.  These findings provide a mol. framework of NAD-dependent histone deacetylation that connects metab., genomic silencing and aging in yeast and, perhaps, in higher eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF5dAUjTy39LVg90H21EOLACvtfcHk0ljT80C9n6ODeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhsVWiurY%253D&md5=513e2dca05684d2e9823e249a83d4e16</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F35001622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35001622%26sid%3Dliteratum%253Aachs%26aulast%3DImai%26aufirst%3DS.-I.%26aulast%3DArmstrong%26aufirst%3DC.%2BM.%26aulast%3DKaeberlein%26aufirst%3DM.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DTranscriptional%2520silencing%2520and%2520longevity%2520protein%2520Sir2%2520is%2520an%2520NAD-dependent%2520histone%2520deacetylase%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D795%26doi%3D10.1038%2F35001622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, B. J.</span></span> <span> </span><span class="NLM_article-title">Seven sirtuins for seven deadly diseases ofaging</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2012.10.525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.freeradbiomed.2012.10.525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23104101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1KqsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=133-171&author=B.+J.+Morris&title=Seven+sirtuins+for+seven+deadly+diseases+ofaging&doi=10.1016%2Fj.freeradbiomed.2012.10.525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Seven sirtuins for seven deadly diseases of aging</span></div><div class="casAuthors">Morris, Brian J.</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">133-171</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Sirtuins are a class of NAD+-dependent deacetylases having beneficial health effects.  This extensive review describes the numerous intracellular actions of the seven mammalian sirtuins, their protein targets, intracellular localization, the pathways they modulate, and their role in common diseases of aging.  Selective pharmacol. targeting of sirtuins is of current interest in helping to alleviate global disease burden.  Since all sirtuins are activated by NAD+, strategies that boost NAD+ in cells are of interest.  While most is known about SIRT1, the functions of the six other sirtuins are now emerging.  Best known is the involvement of sirtuins in helping cells adapt energy output to match energy requirements.  SIRT1 and some of the other sirtuins enhance fat metab. and modulate mitochondrial respiration to optimize energy harvesting.  The AMP kinase/SIRT1-PGC-1α-PPAR axis and mitochondrial sirtuins appear pivotal to maintaining mitochondrial function.  Downregulation with aging explains much of the pathophysiol. that accumulates with aging.  Posttranslational modifications of sirtuins and their substrates affect specificity.  Although SIRT1 activation seems not to affect life span, activation of some of the other sirtuins might.  Since sirtuins are crucial to pathways that counter the decline in health that accompanies aging, pharmacol. agents that boost sirtuin activity have clin. potential in treatment of diabetes, cardiovascular disease, dementia, osteoporosis, arthritis, and other conditions.  In cancer, however, SIRT1 inhibitors could have therapeutic value.  Nutraceuticals such as resveratrol have a multiplicity of actions besides sirtuin activation.  Their net health benefit and relative safety may have originated from the ability of animals to survive environmental changes by utilizing these stress resistance chems. in the diet during evolution.  Each sirtuin forms a key hub to the intracellular pathways affected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8LdSgVF05MLVg90H21EOLACvtfcHk0ljT80C9n6ODeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1KqsrrE&md5=6ea48068831cafbef89787a3536e872b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2012.10.525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2012.10.525%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DB.%2BJ.%26atitle%3DSeven%2520sirtuins%2520for%2520seven%2520deadly%2520diseases%2520ofaging%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2013%26volume%3D56%26spage%3D133%26epage%3D171%26doi%3D10.1016%2Fj.freeradbiomed.2012.10.525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sauve, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolberger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeke, J. D.</span></span> <span> </span><span class="NLM_article-title">The biochemistry of sirtuins</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.74.082803.133500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1146%2Fannurev.biochem.74.082803.133500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=16756498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD28XosVKhs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2006&pages=435-465&author=A.+A.+Sauveauthor=C.+Wolbergerauthor=V.+L.+Schrammauthor=J.+D.+Boeke&title=The+biochemistry+of+sirtuins&doi=10.1146%2Fannurev.biochem.74.082803.133500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The biochemistry of sirtuins</span></div><div class="casAuthors">Sauve, Anthony A.; Wolberger, Cynthia; Schramm, Vern L.; Boeke, Jef D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">435-465</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Sirtuins are a family of NAD-dependent protein deacetylases widely distributed in all phyla of life.  Accumulating evidence indicates that sirtuins are important regulators of organism life span.  In yeast, these unique enzymes regulate gene silencing by histone deacetylation and via formation of the novel compd., 2'-O-acetyl-ADP-ribose.  In multicellular organisms, sirtuins deacetylate histones and transcription factors that regulate stress, metab., and survival pathways.  The chem. mechanism of sirtuins provides novel opportunities for signaling and metabolic regulation of protein deacetylation.  The biol., chem., and structural characteristics of these unusual enzymes are discussed in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorq1GGNz7SSrVg90H21EOLACvtfcHk0ljT80C9n6ODeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosVKhs7g%253D&md5=af3594af9dc2df013e4d1970f8ef7433</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.74.082803.133500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.74.082803.133500%26sid%3Dliteratum%253Aachs%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DWolberger%26aufirst%3DC.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26aulast%3DBoeke%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520biochemistry%2520of%2520sirtuins%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2006%26volume%3D75%26spage%3D435%26epage%3D465%26doi%3D10.1146%2Fannurev.biochem.74.082803.133500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chalkiadaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarente, L.</span></span> <span> </span><span class="NLM_article-title">The multifaceted functions of sirtuins in cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrc3985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnrc3985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26383140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFeisb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=608-624&author=A.+Chalkiadakiauthor=L.+Guarente&title=The+multifaceted+functions+of+sirtuins+in+cancer&doi=10.1038%2Fnrc3985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The multifaceted functions of sirtuins in cancer</span></div><div class="casAuthors">Chalkiadaki, Angeliki; Guarente, Leonard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The sirtuins (SIRTs; of which there are seven in mammals) are NAD+-dependent enzymes that regulate a large no. of cellular pathways and forestall the progression of ageing and age-assocd. diseases.  In recent years, the role of sirtuins in cancer biol. has become increasingly apparent, and growing evidence demonstrates that sirtuins regulate many processes that go awry in cancer cells, such as cellular metab., the regulation of chromatin structure and the maintenance of genomic stability.  In this article, we review recent advances in our understanding of how sirtuins affect cancer metab., DNA repair and the tumor microenvironment and how activating or inhibiting sirtuins may be important in preventing or treating cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpblQk6T-KatbVg90H21EOLACvtfcHk0ljT80C9n6ODeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFeisb7L&md5=264b4e0b86556f0b72db33110ee3e2f5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrc3985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3985%26sid%3Dliteratum%253Aachs%26aulast%3DChalkiadaki%26aufirst%3DA.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DThe%2520multifaceted%2520functions%2520of%2520sirtuins%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrc3985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alders, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cetinbas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pajkrt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Koningsbruggen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christoffels, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannens, M. M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knegt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Etchegaray, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadreyev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Maarle, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">An inactivating mutation in the histone deacetylase SIRT6 causes human perinatal lethality</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1101/gad.307330.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1101%2Fgad.307330.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=29555651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsVaisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=373-388&author=C.+M.+Ferrerauthor=M.+Aldersauthor=A.+V.+Postmaauthor=S.+Parkauthor=M.+A.+Kleinauthor=M.+Cetinbasauthor=E.+Pajkrtauthor=A.+Glasauthor=S.+van+Koningsbruggenauthor=V.+M.+Christoffelsauthor=M.+M.+A.+M.+Mannensauthor=L.+Knegtauthor=J.-P.+Etchegarayauthor=R.+I.+Sadreyevauthor=J.+M.+Denuauthor=G.+Mostoslavskyauthor=M.+C.+van+Maarleauthor=R.+Mostoslavsky&title=An+inactivating+mutation+in+the+histone+deacetylase+SIRT6+causes+human+perinatal+lethality&doi=10.1101%2Fgad.307330.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">An inactivating mutation in the histone deacetylase SIRT6 causes human perinatal lethality</span></div><div class="casAuthors">Ferrer, Christina M.; Alders, Marielle; Postma, Alex V.; Park, Seonmi; Klein, Mark A.; Cetinbas, Murat; Pajkrt, Eva; Glas, Astrid; van Koningsbruggen, Silvana; Christoffels, Vincent M.; Mannens, Marcel M. A. M.; Knegt, Lia; Etchegaray, Jean-Pierre; Sadreyev, Ruslan I.; Denu, John M.; Mostoslavsky, Gustavo; van Maarle, Merel C.; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">373-388</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">It has been well established that histone and DNA modifications are crit. to maintaining the equil. between pluripotency and differentiation during early embryogenesis.  Mutations in key regulators of DNA methylation have shown that the balance between gene regulation and function is crit. during neural development in early years of life.  However, there have been no identified cases linking epigenetic regulators to aberrant human development and fetal demise.  Here, we demonstrate that a homozygous inactivating mutation in the histone deacetylase SIRT6 results in severe congenital anomalies and perinatal lethality in four affected fetuses.  In vitro, the amino acid change at Asp63 to a histidine results in virtually complete loss of H3K9 deacetylase and demyristoylase functions.  Functionally, SIRT6 D63H mouse embryonic stem cells (mESCs) fail to repress pluripotent gene expression, direct targets of SIRT6, and exhibit an even more severe phenotype than Sirt6-deficient ESCs when differentiated into embryoid bodies (EBs).  When terminally differentiated toward cardiomyocyte lineage, D63H mutant mESCs maintain expression of pluripotent genes and fail to form functional cardiomyocyte foci.  Last, human induced pluripotent stem cells (iPSCs) derived from D63H homozygous fetuses fail to differentiate into EBs, functional cardiomyocytes, and neural progenitor cells due to a failure to repress pluripotent genes.  Altogether, our study described a germline mutation in SIRT6 as a cause for fetal demise, defining SIRT6 as a key factor in human development and identifying the first mutation in a chromatin factor behind a human syndrome of perinatal lethality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXdn5zBcYq27Vg90H21EOLACvtfcHk0liIcA8ukAZppA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsVaisbs%253D&md5=ea556b4ad909cdef66291247d0696c6b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1101%2Fgad.307330.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.307330.117%26sid%3Dliteratum%253Aachs%26aulast%3DFerrer%26aufirst%3DC.%2BM.%26aulast%3DAlders%26aufirst%3DM.%26aulast%3DPostma%26aufirst%3DA.%2BV.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DM.%2BA.%26aulast%3DCetinbas%26aufirst%3DM.%26aulast%3DPajkrt%26aufirst%3DE.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3Dvan%2BKoningsbruggen%26aufirst%3DS.%26aulast%3DChristoffels%26aufirst%3DV.%2BM.%26aulast%3DMannens%26aufirst%3DM.%2BM.%2BA.%2BM.%26aulast%3DKnegt%26aufirst%3DL.%26aulast%3DEtchegaray%26aufirst%3DJ.-P.%26aulast%3DSadreyev%26aufirst%3DR.%2BI.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26aulast%3DMostoslavsky%26aufirst%3DG.%26aulast%3Dvan%2BMaarle%26aufirst%3DM.%2BC.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DAn%2520inactivating%2520mutation%2520in%2520the%2520histone%2520deacetylase%2520SIRT6%2520causes%2520human%2520perinatal%2520lethality%26jtitle%3DGenes%2520Dev.%26date%3D2018%26volume%3D32%26spage%3D373%26epage%3D388%26doi%3D10.1101%2Fgad.307330.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Meter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seluanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbunova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span> <span> </span><span class="NLM_article-title">SIRT6 rescues the age related decline in base excision repair in a PARP1-dependent manner</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.4161/15384101.2014.980641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.4161%2F15384101.2014.980641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=25607651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVKrtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=269-276&author=Z.+Xuauthor=L.+Zhangauthor=W.+Zhangauthor=D.+Mengauthor=H.+Zhangauthor=Y.+Jiangauthor=X.+Xuauthor=M.+Van+Meterauthor=A.+Seluanovauthor=V.+Gorbunovaauthor=Z.+Mao&title=SIRT6+rescues+the+age+related+decline+in+base+excision+repair+in+a+PARP1-dependent+manner&doi=10.4161%2F15384101.2014.980641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 rescues the age related decline in base excision repair in a PARP1-dependent manner</span></div><div class="casAuthors">Xu, Zhu; Zhang, Lei; Zhang, Wenjun; Meng, Du; Zhang, Hongxia; Jiang, Ying; Xu, Xiaojun; Van Meter, Michael; Seluanov, Andrei; Gorbunova, Vera; Mao, Zhiyong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-276</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1551-4005</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">In principle, a decline in base excision repair (BER) efficiency with age should lead to genomic instability and ultimately contribute to the onset of the aging phenotype.  Although multiple studies have indicated a neg. link between aging and BER, the change of BER efficiency with age in humans has not been systematically analyzed.  Here, with foreskin fibroblasts isolated from 19 donors between 20 and 64 y of age, we report a significant decline of BER efficiency with age using a newly developed GFP reactivation assay.  We further obsd. a very strong neg. correlation between age and the expression levels of SIRT6, a factor which is known to maintain genomic integrity by improving DNA double strand break (DSB) repair.  Our mechanistic study suggests that, similar to the regulatory role that SIRT6 plays in DNA DSB repair, SIRT6 regulates BER in a PARP1-dependent manner.  Moreover, overexpression of SIRT6 rescues the decline of BER in aged fibroblasts.  In summary, our results uncovered the regulatory mechanisms of BER by SIRT6, suggesting that SIRT6 reactivation in aging tissues may help delay the process of aging through improving BER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra44iHCz1gIbVg90H21EOLACvtfcHk0liIcA8ukAZppA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVKrtb7E&md5=c3bc7a61000bbfe34748efe2219575fd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4161%2F15384101.2014.980641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15384101.2014.980641%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMeng%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DVan%2BMeter%26aufirst%3DM.%26aulast%3DSeluanov%26aufirst%3DA.%26aulast%3DGorbunova%26aufirst%3DV.%26aulast%3DMao%26aufirst%3DZ.%26atitle%3DSIRT6%2520rescues%2520the%2520age%2520related%2520decline%2520in%2520base%2520excision%2520repair%2520in%2520a%2520PARP1-dependent%2520manner%26jtitle%3DCell%2520Cycle%26date%3D2015%26volume%3D14%26spage%3D269%26epage%3D276%26doi%3D10.4161%2F15384101.2014.980641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span> <span> </span><span class="NLM_article-title">SIRT6 deficiency results in developmental retardation in cynomolgus monkeys</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>560</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">665</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0437-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fs41586-018-0437-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=30135584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFOgsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=560&publication_year=2018&pages=661-665&author=W.+Zhangauthor=H.+Wanauthor=G.+Fengauthor=J.+Quauthor=J.+Wangauthor=Y.+Jingauthor=R.+Renauthor=Z.+Liuauthor=L.+Zhangauthor=Z.+Chenauthor=S.+Wangauthor=Y.+Zhaoauthor=Z.+Wangauthor=Y.+Yuanauthor=Q.+Zhouauthor=W.+Liauthor=G.-H.+Liuauthor=B.+Hu&title=SIRT6+deficiency+results+in+developmental+retardation+in+cynomolgus+monkeys&doi=10.1038%2Fs41586-018-0437-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 deficiency results in developmental retardation in cynomolgus monkeys</span></div><div class="casAuthors">Zhang, Weiqi; Wan, Haifeng; Feng, Guihai; Qu, Jing; Wang, Jiaqiang; Jing, Yaobin; Ren, Ruotong; Liu, Zunpeng; Zhang, Linlin; Chen, Zhiguo; Wang, Shuyan; Zhao, Yong; Wang, Zhaoxia; Yuan, Yun; Zhou, Qi; Li, Wei; Liu, Guang-Hui; Hu, Baoyang</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">560</span>
        (<span class="NLM_cas:issue">7720</span>),
    <span class="NLM_cas:pages">661-665</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">SIRT6 acts as a longevity protein in rodents1,2.  However, its biol. function in primates remains largely unknown.  Here we generate a SIRT6-null cynomolgus monkey (Macaca fascicularis) model using a CRISPR-Cas9-based approach.  SIRT6-deficient monkeys die hours after birth and exhibit severe prenatal developmental retardation.  SIRT6 loss delays neuronal differentiation by transcriptionally activating the long non-coding RNA H19 (a developmental repressor), and we were able to recapitulate this process in a human neural progenitor cell differentiation system.  SIRT6 deficiency results in histone hyperacetylation at the imprinting control region of H19, CTCF recruitment and upregulation of H19.  Our results suggest that SIRT6 is involved in regulating development in non-human primates, and may provide mechanistic insight into human perinatal lethality syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN2SKsLb0dF7Vg90H21EOLACvtfcHk0lg5jv0xVe59AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFOgsLrI&md5=2d28ce353a9c1bf941ebb94e2daa5a45</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0437-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0437-z%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DWan%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DQu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DJing%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DG.-H.%26aulast%3DHu%26aufirst%3DB.%26atitle%3DSIRT6%2520deficiency%2520results%2520in%2520developmental%2520retardation%2520in%2520cynomolgus%2520monkeys%26jtitle%3DNature%26date%3D2018%26volume%3D560%26spage%3D661%26epage%3D665%26doi%3D10.1038%2Fs41586-018-0437-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hine, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Meter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaidya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seluanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbunova, V.</span></span> <span> </span><span class="NLM_article-title">SIRT6 promotes DNA repair under stress by activating PARP1</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>332</i></span>,  <span class="NLM_fpage">1443</span>– <span class="NLM_lpage">1446</span>, <span class="refDoi"> DOI: 10.1126/science.1202723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1126%2Fscience.1202723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=21680843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntlOltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2011&pages=1443-1446&author=Z.+Maoauthor=C.+Hineauthor=X.+Tianauthor=M.+Van+Meterauthor=M.+Auauthor=A.+Vaidyaauthor=A.+Seluanovauthor=V.+Gorbunova&title=SIRT6+promotes+DNA+repair+under+stress+by+activating+PARP1&doi=10.1126%2Fscience.1202723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 promotes DNA repair under stress by activating PARP1</span></div><div class="casAuthors">Mao, Zhiyong; Hine, Christopher; Tian, Xiao; Van Meter, Michael; Au, Matthew; Vaidya, Amita; Seluanov, Andrei; Gorbunova, Vera</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">6036</span>),
    <span class="NLM_cas:pages">1443-1446</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Sirtuin 6 (SIRT6) is a mammalian homolog of the yeast Sir2 deacetylase.  Mice deficient for SIRT6 exhibit genome instability.  Here, we show that in mammalian cells subjected to oxidative stress SIRT6 is recruited to the sites of DNA double-strand breaks (DSBs) and stimulates DSB repair, through both nonhomologous end joining and homologous recombination.  Our results indicate that SIRT6 phys. assocs. with poly[ADP-ribose] polymerase 1 (PARP1) and mono-ADP-ribosylates PARP1 on lysine residue 521, thereby stimulating PARP1 poly-ADP-ribosylase activity and enhancing DSB repair under oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1cokFt_HuDbVg90H21EOLACvtfcHk0lg5jv0xVe59AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntlOltLg%253D&md5=3cf0f8ea3d92894f8217902197ce5500</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1202723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1202723%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DHine%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DVan%2BMeter%26aufirst%3DM.%26aulast%3DAu%26aufirst%3DM.%26aulast%3DVaidya%26aufirst%3DA.%26aulast%3DSeluanov%26aufirst%3DA.%26aulast%3DGorbunova%26aufirst%3DV.%26atitle%3DSIRT6%2520promotes%2520DNA%2520repair%2520under%2520stress%2520by%2520activating%2520PARP1%26jtitle%3DScience%26date%3D2011%26volume%3D332%26spage%3D1443%26epage%3D1446%26doi%3D10.1126%2Fscience.1202723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span> <span> </span><span class="NLM_article-title">Human SIRT6 promotes DNA end resection through CtIP deacetylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>329</i></span>,  <span class="NLM_fpage">1348</span>– <span class="NLM_lpage">1353</span>, <span class="refDoi"> DOI: 10.1126/science.1192049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1126%2Fscience.1192049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=20829486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFajs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=329&publication_year=2010&pages=1348-1353&author=A.+Kaidiauthor=B.+T.+Weinertauthor=C.+Choudharyauthor=S.+P.+Jackson&title=Human+SIRT6+promotes+DNA+end+resection+through+CtIP+deacetylation&doi=10.1126%2Fscience.1192049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Human SIRT6 Promotes DNA End Resection Through CtIP Deacetylation</span></div><div class="casAuthors">Kaidi, Abderrahmane; Weinert, Brian T.; Choudhary, Chunaram; Jackson, Stephen P.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">329</span>
        (<span class="NLM_cas:issue">5997</span>),
    <span class="NLM_cas:pages">1348-1353</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SIRT6 belongs to the sirtuin family of protein lysine deacetylases, which regulate aging and genome stability.  We found that human SIRT6 has a role in promoting DNA end resection, a crucial step in DNA double-strand break (DSB) repair by homologous recombination.  SIRT6 depletion impaired the accumulation of replication protein A (RPA) and single-stranded DNA (ssDNA) at DNA damage sites, reduced rates of homologous recombination, and sensitized cells to DSB-inducing agents.  We identified the DSB resection protein CtIP [C-terminal binding protein (CtBP) interacting protein] as a SIRT6 interaction partner and showed that SIRT6-dependent CtIP deacetylation promotes resection.  A nonacetylatable CtIP mutant alleviated the effect of SIRT6 depletion on resection, thus identifying CtIP as a key substrate by which SIRT6 facilitates DSB processing and homologous recombination.  These findings further clarify how SIRT6 promotes genome stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqYddumzd0v7Vg90H21EOLACvtfcHk0lg5jv0xVe59AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFajs7rF&md5=72055f022a4f82adfd8121c10b4fdc8c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1192049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1192049%26sid%3Dliteratum%253Aachs%26aulast%3DKaidi%26aufirst%3DA.%26aulast%3DWeinert%26aufirst%3DB.%2BT.%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26atitle%3DHuman%2520SIRT6%2520promotes%2520DNA%2520end%2520resection%2520through%2520CtIP%2520deacetylation%26jtitle%3DScience%26date%3D2010%26volume%3D329%26spage%3D1348%26epage%3D1353%26doi%3D10.1126%2Fscience.1192049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frendewey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auerbach, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenzuela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hottiger, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hursting, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarente, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demple, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yancopoulos, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, F. W.</span></span> <span> </span><span class="NLM_article-title">Genomic instability and aging-like phenotype in the absence of mammalian SIRT6</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2005.11.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2005.11.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=16439206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1KktLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=315-329&author=R.+Mostoslavskyauthor=K.+F.+Chuaauthor=D.+B.+Lombardauthor=W.+W.+Pangauthor=M.+R.+Fischerauthor=L.+Gellonauthor=P.+Liuauthor=G.+Mostoslavskyauthor=S.+Francoauthor=M.+M.+Murphyauthor=K.+D.+Millsauthor=P.+Patelauthor=J.+T.+Hsuauthor=A.+L.+Hongauthor=E.+Fordauthor=H.-L.+Chengauthor=C.+Kennedyauthor=N.+Nunezauthor=R.+Bronsonauthor=D.+Frendeweyauthor=W.+Auerbachauthor=D.+Valenzuelaauthor=M.+Karowauthor=M.+O.+Hottigerauthor=S.+Hurstingauthor=J.+C.+Barrettauthor=L.+Guarenteauthor=R.+Mulliganauthor=B.+Dempleauthor=G.+D.+Yancopoulosauthor=F.+W.+Alt&title=Genomic+instability+and+aging-like+phenotype+in+the+absence+of+mammalian+SIRT6&doi=10.1016%2Fj.cell.2005.11.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic instability and aging-like phenotype in the absence of mammalian SIRT6</span></div><div class="casAuthors">Mostoslavsky, Raul; Chua, Katrin F.; Lombard, David B.; Pang, Wendy W.; Fischer, Miriam R.; Gellon, Lionel; Liu, Pingfang; Mostoslavsky, Gustavo; Franco, Sonia; Murphy, Michael M.; Mills, Kevin D.; Patel, Parin; Hsu, Joyce T.; Hong, Andrew L.; Ford, Ethan; Cheng, Hwei-Ling; Kennedy, Caitlin; Nunez, Nomeli; Bronson, Roderick; Frendewey, David; Auerbach, Wojtek; Valenzuela, David; Karow, Margaret; Hottiger, Michael O.; Hursting, Stephen; Barrett, J. Carl; Guarente, Leonard; Mulligan, Richard; Demple, Bruce; Yancopoulos, George D.; Alt, Frederick W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">315-329</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Sir2 histone deacetylase functions as a chromatin silencer to regulate recombination, genomic stability, and aging in budding yeast.  Seven mammalian Sir2 homologs have been identified (SIRT1-SIRT7), and it has been speculated that some may have similar functions to Sir2.  Here, we demonstrate that SIRT6 is a nuclear, chromatin-assocd. protein that promotes resistance to DNA damage and suppresses genomic instability in mouse cells, in assocn. with a role in base excision repair (BER).  SIRT6-deficient mice are small and at 2-3 wk of age develop abnormalities that include profound lymphopenia, loss of s.c. fat, lordokyphosis, and severe metabolic defects, eventually dying at about 4 wk.  We conclude that one function of SIRT6 is to promote normal DNA repair, and that SIRT6 loss leads to abnormalities in mice that overlap with aging-assocd. degenerative processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-iTboO5w_ObVg90H21EOLACvtfcHk0liEE0RjCRvtyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1KktLc%253D&md5=14971074ed55f5382960b500370ecff2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.11.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.11.044%26sid%3Dliteratum%253Aachs%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DPang%26aufirst%3DW.%2BW.%26aulast%3DFischer%26aufirst%3DM.%2BR.%26aulast%3DGellon%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DMostoslavsky%26aufirst%3DG.%26aulast%3DFranco%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DM.%2BM.%26aulast%3DMills%26aufirst%3DK.%2BD.%26aulast%3DPatel%26aufirst%3DP.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26aulast%3DHong%26aufirst%3DA.%2BL.%26aulast%3DFord%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DH.-L.%26aulast%3DKennedy%26aufirst%3DC.%26aulast%3DNunez%26aufirst%3DN.%26aulast%3DBronson%26aufirst%3DR.%26aulast%3DFrendewey%26aufirst%3DD.%26aulast%3DAuerbach%26aufirst%3DW.%26aulast%3DValenzuela%26aufirst%3DD.%26aulast%3DKarow%26aufirst%3DM.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26aulast%3DHursting%26aufirst%3DS.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DGuarente%26aufirst%3DL.%26aulast%3DMulligan%26aufirst%3DR.%26aulast%3DDemple%26aufirst%3DB.%26aulast%3DYancopoulos%26aufirst%3DG.%2BD.%26aulast%3DAlt%26aufirst%3DF.%2BW.%26atitle%3DGenomic%2520instability%2520and%2520aging-like%2520phenotype%2520in%2520the%2520absence%2520of%2520mammalian%2520SIRT6%26jtitle%3DCell%26date%3D2006%26volume%3D124%26spage%3D315%26epage%3D329%26doi%3D10.1016%2Fj.cell.2005.11.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michishita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCord, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kioi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla-Nash, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusumoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawahara, T. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ried, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span> <span> </span><span class="NLM_article-title">SIRT6 is a histone H3 lysine 9 deacetylase that modulates telomeric chromatin</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>452</i></span>,  <span class="NLM_fpage">492</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/nature06736</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnature06736" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=18337721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjslCnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=452&publication_year=2008&pages=492-496&author=E.+Michishitaauthor=R.+A.+McCordauthor=E.+Berberauthor=M.+Kioiauthor=H.+Padilla-Nashauthor=M.+Damianauthor=P.+Cheungauthor=R.+Kusumotoauthor=T.+L.+A.+Kawaharaauthor=J.+C.+Barrettauthor=H.+Y.+Changauthor=V.+A.+Bohrauthor=T.+Riedauthor=O.+Gozaniauthor=K.+F.+Chua&title=SIRT6+is+a+histone+H3+lysine+9+deacetylase+that+modulates+telomeric+chromatin&doi=10.1038%2Fnature06736"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 is a histone H3 lysine 9 deacetylase that modulates telomeric chromatin</span></div><div class="casAuthors">Michishita, Eriko; McCord, Ronald A.; Berber, Elisabeth; Kioi, Mitomu; Padilla-Nash, Hesed; Damian, Mara; Cheung, Peggie; Kusumoto, Rika; Kawahara, Tiara L. A.; Barrett, J. Carl; Chang, Howard Y.; Bohr, Vilhelm A.; Ried, Thomas; Gozani, Or; Chua, Katrin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">452</span>
        (<span class="NLM_cas:issue">7186</span>),
    <span class="NLM_cas:pages">492-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Sir2 deacetylase regulates chromatin silencing and lifespan in Saccharomyces cerevisiae.  In mice, deficiency for the Sir2 family member SIRT6 leads to a shortened lifespan and a premature aging-like phenotype.  However, the mol. mechanisms of SIRT6 function are unclear.  SIRT6 is a chromatin-assocd. protein, but no enzymic activity of SIRT6 at chromatin has yet been detected, and the identity of physiol. SIRT6 substrates is unknown.  Here we show that the human SIRT6 protein is an NAD+-dependent, histone H3 lysine 9 (H3K9) deacetylase that modulates telomeric chromatin.  SIRT6 assocs. specifically with telomeres, and SIRT6 depletion leads to telomere dysfunction with end-to-end chromosomal fusions and premature cellular senescence.  Moreover, SIRT6-depleted cells exhibit abnormal telomere structures that resemble defects obsd. in Werner syndrome, a premature aging disorder.  At telomeric chromatin, SIRT6 deacetylates H3K9 and is required for the stable assocn. of WRN, the factor that is mutated in Werner syndrome.  We propose that SIRT6 contributes to the propagation of a specialized chromatin state at mammalian telomeres, which in turn is required for proper telomere metab. and function.  Our findings constitute the first identification of a physiol. enzymic activity of SIRT6, and link chromatin regulation by SIRT6 to telomere maintenance and a human premature aging syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYo1SMIJbbV7Vg90H21EOLACvtfcHk0liEE0RjCRvtyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjslCnurs%253D&md5=5aa9f5b39be29fa580012b718943a3d7</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature06736&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06736%26sid%3Dliteratum%253Aachs%26aulast%3DMichishita%26aufirst%3DE.%26aulast%3DMcCord%26aufirst%3DR.%2BA.%26aulast%3DBerber%26aufirst%3DE.%26aulast%3DKioi%26aufirst%3DM.%26aulast%3DPadilla-Nash%26aufirst%3DH.%26aulast%3DDamian%26aufirst%3DM.%26aulast%3DCheung%26aufirst%3DP.%26aulast%3DKusumoto%26aufirst%3DR.%26aulast%3DKawahara%26aufirst%3DT.%2BL.%2BA.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DChang%26aufirst%3DH.%2BY.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DRied%26aufirst%3DT.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DChua%26aufirst%3DK.%2BF.%26atitle%3DSIRT6%2520is%2520a%2520histone%2520H3%2520lysine%25209%2520deacetylase%2520that%2520modulates%2520telomeric%2520chromatin%26jtitle%3DNature%26date%3D2008%26volume%3D452%26spage%3D492%26epage%3D496%26doi%3D10.1038%2Fnature06736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sebastián, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwaans, B. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silberman, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gymrek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ram, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truelove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toiber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosentino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenson, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlynn, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacosa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiels, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">The histone deacetylase SIRT6 is a tumor suppressor that controls cancer metabolism</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1199</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.10.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2012.10.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23217706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaisr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2012&pages=1185-1199&author=C.+Sebasti%C3%A1nauthor=B.+M.+M.+Zwaansauthor=D.+M.+Silbermanauthor=M.+Gymrekauthor=A.+Gorenauthor=L.+Zhongauthor=O.+Ramauthor=J.+Trueloveauthor=A.+R.+Guimaraesauthor=D.+Toiberauthor=C.+Cosentinoauthor=J.+K.+Greensonauthor=A.+I.+MacDonaldauthor=L.+McGlynnauthor=F.+Maxwellauthor=J.+Edwardsauthor=S.+Giacosaauthor=E.+Guccioneauthor=R.+Weisslederauthor=B.+E.+Bernsteinauthor=A.+Regevauthor=P.+G.+Shielsauthor=D.+B.+Lombardauthor=R.+Mostoslavsky&title=The+histone+deacetylase+SIRT6+is+a+tumor+suppressor+that+controls+cancer+metabolism&doi=10.1016%2Fj.cell.2012.10.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The histone deacetylase SIRT6 is a tumor suppressor that controls cancer metabolism</span></div><div class="casAuthors">Sebastian, Carlos; Zwaans, Bernadette M. M.; Silberman, Dafne M.; Gymrek, Melissa; Goren, Alon; Zhong, Lei; Ram, Oren; Truelove, Jessica; Guimaraes, Alexander R.; Toiber, Debra; Cosentino, Claudia; Greenson, Joel K.; MacDonald, Alasdair I.; McGlynn, Liane; Maxwell, Fraser; Edwards, Joanne; Giacosa, Sofia; Guccione, Ernesto; Weissleder, Ralph; Bernstein, Bradley E.; Regev, Aviv; Shiels, Paul G.; Lombard, David B.; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1185-1199</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Reprogramming of cellular metab. is a key event during tumorigenesis.  Despite being known for decades (Warburg effect), the mol. mechanisms regulating this switch remained unexplored.  Here, we identify SIRT6 as a tumor suppressor that regulates aerobic glycolysis in cancer cells.  Importantly, loss of SIRT6 leads to tumor formation without activation of known oncogenes, whereas transformed SIRT6-deficient cells display increased glycolysis and tumor growth, suggesting that SIRT6 plays a role in both establishment and maintenance of cancer.  By using a conditional SIRT6 allele, we show that SIRT6 deletion in vivo increases the no., size, and aggressiveness of tumors.  SIRT6 also functions as a regulator of ribosome metab. by corepressing MYC transcriptional activity.  Lastly, Sirt6 is selectively downregulated in several human cancers, and expression levels of SIRT6 predict prognosis and tumor-free survival rates, highlighting SIRT6 as a crit. modulator of cancer metab.  Our studies reveal SIRT6 to be a potent tumor suppressor acting to suppress cancer metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6UVcycnyph7Vg90H21EOLACvtfcHk0liPqSwFDx6dSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaisr3J&md5=bb6df5c1d96df3ee5809773f91bb7ef5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.10.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.10.047%26sid%3Dliteratum%253Aachs%26aulast%3DSebasti%25C3%25A1n%26aufirst%3DC.%26aulast%3DZwaans%26aufirst%3DB.%2BM.%2BM.%26aulast%3DSilberman%26aufirst%3DD.%2BM.%26aulast%3DGymrek%26aufirst%3DM.%26aulast%3DGoren%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DRam%26aufirst%3DO.%26aulast%3DTruelove%26aufirst%3DJ.%26aulast%3DGuimaraes%26aufirst%3DA.%2BR.%26aulast%3DToiber%26aufirst%3DD.%26aulast%3DCosentino%26aufirst%3DC.%26aulast%3DGreenson%26aufirst%3DJ.%2BK.%26aulast%3DMacDonald%26aufirst%3DA.%2BI.%26aulast%3DMcGlynn%26aufirst%3DL.%26aulast%3DMaxwell%26aufirst%3DF.%26aulast%3DEdwards%26aufirst%3DJ.%26aulast%3DGiacosa%26aufirst%3DS.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DWeissleder%26aufirst%3DR.%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DShiels%26aufirst%3DP.%2BG.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DThe%2520histone%2520deacetylase%2520SIRT6%2520is%2520a%2520tumor%2520suppressor%2520that%2520controls%2520cancer%2520metabolism%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D1185%26epage%3D1199%26doi%3D10.1016%2Fj.cell.2012.10.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baeza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span> <span> </span><span class="NLM_article-title">Activation of the protein deacetylase SIRT6 by long-chain fatty acids and widespread deacylation by mammalian sirtuins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">31350</span>– <span class="NLM_lpage">31356</span>, <span class="refDoi"> DOI: 10.1074/jbc.c113.511261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1074%2Fjbc.c113.511261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=24052263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1yiur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=31350-31356&author=J.+L.+Feldmanauthor=J.+Baezaauthor=J.+M.+Denu&title=Activation+of+the+protein+deacetylase+SIRT6+by+long-chain+fatty+acids+and+widespread+deacylation+by+mammalian+sirtuins&doi=10.1074%2Fjbc.c113.511261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the protein deacetylase SIRT6 by long-chain fatty acids and widespread deacylation by mammalian sirtuins</span></div><div class="casAuthors">Feldman, Jessica L.; Baeza, Josue; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">31350-31356</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mammalian sirtuins (SIRT1 through SIRT7) are members of a highly conserved family of NAD-dependent protein deacetylases that function in metab., genome maintenance, and stress responses.  Emerging evidence suggests that some sirtuins display substrate specificity toward other acyl groups attached to the lysine ε-amine.  SIRT6 was recently reported to preferentially hydrolyze long-chain fatty acyl groups over acetyl groups.  Here, the authors investigated the catalytic ability of all sirtuins to hydrolyze 13 different acyl groups from histone H3 peptides, ranging in carbon length, satn., and chem. diversity.  The authors found that long-chain deacylation is a general feature of mammalian sirtuins, that SIRT1 and SIRT2 act as efficient decrotonylases, and that SIRT1, SIRT2, SIRT3, and SIRT4 can remove lipoic acid.  These results provided new insights into sirtuin function and a means for cellular removal of an expanding list of endogenous lysine modifications.  Given that SIRT6 is a poor deacetylase in vitro, but binds and prefers to hydrolyze long-chain acylated peptides, the authors hypothesized that binding of certain free fatty acids (FFAs) could stimulate deacetylation activity.  Indeed, the authors demonstrated that several biol. relevant FFAs (including myristic, oleic, and linoleic acids) at physiol. concns. induced up to a 35-fold increase in catalytic efficiency of SIRT6 but not SIRT1.  The activation mechanism was consistent with fatty acid inducing a conformation that binds acetylated H3 with greater affinity.  The binding of long-chain FFA and myristoylated H3 peptide was mutually exclusive.  The authors discussed the implications of discovering endogenous, small-mol. activators of SIRT6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0tBZ5loovPrVg90H21EOLACvtfcHk0liPqSwFDx6dSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1yiur3M&md5=0bd3995be555fa210ff0bb5763e37681</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.c113.511261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.c113.511261%26sid%3Dliteratum%253Aachs%26aulast%3DFeldman%26aufirst%3DJ.%2BL.%26aulast%3DBaeza%26aufirst%3DJ.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DActivation%2520of%2520the%2520protein%2520deacetylase%2520SIRT6%2520by%2520long-chain%2520fatty%2520acids%2520and%2520widespread%2520deacylation%2520by%2520mammalian%2520sirtuins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D31350%26epage%3D31356%26doi%3D10.1074%2Fjbc.c113.511261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawahara, T. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michishita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adler, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCord, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ongaigui, K. C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span> <span> </span><span class="NLM_article-title">SIRT6 links histone H3 lysine 9 deacetylation to NF-κB-dependent gene expression and organismal life span</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2008.10.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2008.10.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=19135889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=62-74&author=T.+L.+A.+Kawaharaauthor=E.+Michishitaauthor=A.+S.+Adlerauthor=M.+Damianauthor=E.+Berberauthor=M.+Linauthor=R.+A.+McCordauthor=K.+C.+L.+Ongaiguiauthor=L.+D.+Boxerauthor=H.+Y.+Changauthor=K.+F.+Chua&title=SIRT6+links+histone+H3+lysine+9+deacetylation+to+NF-%CE%BAB-dependent+gene+expression+and+organismal+life+span&doi=10.1016%2Fj.cell.2008.10.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 links histone H3 lysine 9 deacetylation to NF-κB-dependent gene expression and organismal life span</span></div><div class="casAuthors">Kawahara, Tiara L. A.; Michishita, Eriko; Adler, Adam S.; Damian, Mara; Berber, Elisabeth; Lin, Meihong; McCord, Ron A.; Ongaigui, Kristine C. L.; Boxer, Lisa D.; Chang, Howard Y.; Chua, Katrin F.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-74</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Members of the sirtuin (SIRT) family of NAD-dependent deacetylases promote longevity in multiple organisms.  Deficiency of mammalian SIRT6 leads to shortened life span and an aging-like phenotype in mice, but the underlying mol. mechanisms are unclear.  Here we show that SIRT6 functions at chromatin to attenuate NF-κB signaling.  SIRT6 interacts with the NF-κB RELA subunit and deacetylates histone H3 lysine 9 (H3K9) at NF-κB target gene promoters.  In SIRT6-deficient cells, hyperacetylation of H3K9 at these target promoters is assocd. with increased RELA promoter occupancy and enhanced NF-κB-dependent modulation of gene expression, apoptosis, and cellular senescence.  Computational genomics analyses revealed increased activity of NF-κB-driven gene expression programs in multiple Sirt6-deficient tissues in vivo.  Moreover, haploinsufficiency of RelA rescues the early lethality and degenerative syndrome of Sirt6-deficient mice.  It has been proposed that SIRT6 attenuates NF-κB signaling via H3K9 deacetylation at chromatin, and hyperactive NF-κB signaling may contribute to premature and normal aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZDou29TSxW7Vg90H21EOLACvtfcHk0liPqSwFDx6dSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOisbs%253D&md5=84c714f6faa67540c3ee7aeadc6526de</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.10.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.10.052%26sid%3Dliteratum%253Aachs%26aulast%3DKawahara%26aufirst%3DT.%2BL.%2BA.%26aulast%3DMichishita%26aufirst%3DE.%26aulast%3DAdler%26aufirst%3DA.%2BS.%26aulast%3DDamian%26aufirst%3DM.%26aulast%3DBerber%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DMcCord%26aufirst%3DR.%2BA.%26aulast%3DOngaigui%26aufirst%3DK.%2BC.%2BL.%26aulast%3DBoxer%26aufirst%3DL.%2BD.%26aulast%3DChang%26aufirst%3DH.%2BY.%26aulast%3DChua%26aufirst%3DK.%2BF.%26atitle%3DSIRT6%2520links%2520histone%2520H3%2520lysine%25209%2520deacetylation%2520to%2520NF-%25CE%25BAB-dependent%2520gene%2520expression%2520and%2520organismal%2520life%2520span%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D62%26epage%3D74%26doi%3D10.1016%2Fj.cell.2008.10.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanfi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naiman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amir, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peshti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar-Joseph, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, H. Y.</span></span> <span> </span><span class="NLM_article-title">The sirtuin SIRT6 regulates lifespan in male mice</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>483</i></span>,  <span class="NLM_fpage">218</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nature10815</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnature10815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=22367546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XislyntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=218-221&author=Y.+Kanfiauthor=S.+Naimanauthor=G.+Amirauthor=V.+Peshtiauthor=G.+Zinmanauthor=L.+Nahumauthor=Z.+Bar-Josephauthor=H.+Y.+Cohen&title=The+sirtuin+SIRT6+regulates+lifespan+in+male+mice&doi=10.1038%2Fnature10815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The sirtuin SIRT6 regulates lifespan in male mice</span></div><div class="casAuthors">Kanfi, Yariv; Naiman, Shoshana; Amir, Gail; Peshti, Victoria; Zinman, Guy; Nahum, Liat; Bar-Joseph, Ziv; Cohen, Haim Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7388</span>),
    <span class="NLM_cas:pages">218-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The significant increase in human lifespan during the past century confronts us with great medical challenges.  To meet these challenges, the mechanisms that det. healthy ageing must be understood and controlled.  Sirtuins are highly conserved deacetylases that have been shown to regulate lifespan in yeast, nematodes and fruitflies.  However, the role of sirtuins in regulating worm and fly lifespan has recently become controversial.  Moreover, the role of the seven mammalian sirtuins, SIRT1 to SIRT7 (homologues of the yeast sirtuin Sir2), in regulating lifespan is unclear.  Here we show that male, but not female, transgenic mice overexpressing Sirt6 (ref. 4) have a significantly longer lifespan than wild-type mice.  Gene expression anal. revealed significant differences between male Sirt6-transgenic mice and male wild-type mice: transgenic males displayed lower serum levels of insulin-like growth factor 1 (IGF1), higher levels of IGF-binding protein 1 and altered phosphorylation levels of major components of IGF1 signalling, a key pathway in the regulation of lifespan.  This study shows the regulation of mammalian lifespan by a sirtuin family member and has important therapeutic implications for age-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo08gRkOlbJDrVg90H21EOLACvtfcHk0lhtiwo4ImO9rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislyntbY%253D&md5=7f87716fd9e336558e1137081146f351</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature10815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10815%26sid%3Dliteratum%253Aachs%26aulast%3DKanfi%26aufirst%3DY.%26aulast%3DNaiman%26aufirst%3DS.%26aulast%3DAmir%26aufirst%3DG.%26aulast%3DPeshti%26aufirst%3DV.%26aulast%3DZinman%26aufirst%3DG.%26aulast%3DNahum%26aufirst%3DL.%26aulast%3DBar-Joseph%26aufirst%3DZ.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26atitle%3DThe%2520sirtuin%2520SIRT6%2520regulates%2520lifespan%2520in%2520male%2520mice%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D218%26epage%3D221%26doi%3D10.1038%2Fnature10815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firsanov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tombline, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldfarb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornwell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moody, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gladyshev, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seluanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbunova, V.</span></span> <span> </span><span class="NLM_article-title">SIRT6 is responsible for more efficient DNA double-strand break repair in long-lived species</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">638.e22</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.03.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2019.03.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=31002797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnslyhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=622-638.e22&author=X.+Tianauthor=D.+Firsanovauthor=Z.+Zhangauthor=Y.+Chengauthor=L.+Luoauthor=G.+Tomblineauthor=R.+Tanauthor=M.+Simonauthor=S.+Hendersonauthor=J.+Steffanauthor=A.+Goldfarbauthor=J.+Tamauthor=K.+Zhengauthor=A.+Cornwellauthor=A.+Johnsonauthor=J.-N.+Yangauthor=Z.+Maoauthor=B.+Mantaauthor=W.+Dangauthor=Z.+Zhangauthor=J.+Vijgauthor=A.+Wolfeauthor=K.+Moodyauthor=B.+K.+Kennedyauthor=D.+Bohmannauthor=V.+N.+Gladyshevauthor=A.+Seluanovauthor=V.+Gorbunova&title=SIRT6+is+responsible+for+more+efficient+DNA+double-strand+break+repair+in+long-lived+species&doi=10.1016%2Fj.cell.2019.03.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 Is Responsible for More Efficient DNA Double-Strand Break Repair in Long-Lived Species</span></div><div class="casAuthors">Tian, Xiao; Firsanov, Denis; Zhang, Zhihui; Cheng, Yang; Luo, Lingfeng; Tombline, Gregory; Tan, Ruiyue; Simon, Matthew; Henderson, Steven; Steffan, Janine; Goldfarb, Audrey; Tam, Jonathan; Zheng, Kitty; Cornwell, Adam; Johnson, Adam; Yang, Jiang-Nan; Mao, Zhiyong; Manta, Bruno; Dang, Weiwei; Zhang, Zhengdong; Vijg, Jan; Wolfe, Aaron; Moody, Kelsey; Kennedy, Brian K.; Bohmann, Dirk; Gladyshev, Vadim N.; Seluanov, Andrei; Gorbunova, Vera</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">622-638.e22</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">DNA repair has been hypothesized to be a longevity determinant, but the evidence for it is based largely on accelerated aging phenotypes of DNA repair mutants.  Here, using a panel of 18 rodent species with diverse lifespans, we show that more robust DNA double-strand break (DSB) repair, but not nucleotide excision repair (NER), co-evolves with longevity.  Evolution of NER, unlike DSB, is shaped primarily by sunlight exposure.  We further show that the capacity of the SIRT6 protein to promote DSB repair accounts for a major part of the variation in DSB repair efficacy between short- and long-lived species.  We dissected the mol. differences between a weak (mouse) and a strong (beaver) SIRT6 protein and identified five amino acid residues that are fully responsible for their differential activities.  Our findings demonstrate that DSB repair and SIRT6 have been optimized during the evolution of longevity, which provides new targets for anti-aging interventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWg7twatHGfLVg90H21EOLACvtfcHk0lhtiwo4ImO9rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnslyhtL8%253D&md5=27f36ed9062a64886f866c24c0c174e6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.03.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.03.043%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DX.%26aulast%3DFirsanov%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DTombline%26aufirst%3DG.%26aulast%3DTan%26aufirst%3DR.%26aulast%3DSimon%26aufirst%3DM.%26aulast%3DHenderson%26aufirst%3DS.%26aulast%3DSteffan%26aufirst%3DJ.%26aulast%3DGoldfarb%26aufirst%3DA.%26aulast%3DTam%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DCornwell%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DJ.-N.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DManta%26aufirst%3DB.%26aulast%3DDang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DVijg%26aufirst%3DJ.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DMoody%26aufirst%3DK.%26aulast%3DKennedy%26aufirst%3DB.%2BK.%26aulast%3DBohmann%26aufirst%3DD.%26aulast%3DGladyshev%26aufirst%3DV.%2BN.%26aulast%3DSeluanov%26aufirst%3DA.%26aulast%3DGorbunova%26aufirst%3DV.%26atitle%3DSIRT6%2520is%2520responsible%2520for%2520more%2520efficient%2520DNA%2520double-strand%2520break%2520repair%2520in%2520long-lived%2520species%26jtitle%3DCell%26date%3D2019%26volume%3D177%26spage%3D622%26epage%3D638.e22%26doi%3D10.1016%2Fj.cell.2019.03.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mook-Jung, I.</span></span> <span> </span><span class="NLM_article-title">p53-dependent SIRT6 expression protects Aβ42-induced DNA damage</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">25628</span>, <span class="refDoi"> DOI: 10.1038/srep25628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fsrep25628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=27156849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=25628&author=E.+S.+Jungauthor=H.+Choiauthor=H.+Songauthor=Y.+J.+Hwangauthor=A.+Kimauthor=H.+Ryuauthor=I.+Mook-Jung&title=p53-dependent+SIRT6+expression+protects+A%CE%B242-induced+DNA+damage&doi=10.1038%2Fsrep25628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">p53-dependent SIRT6 expression protects Aβ42-induced DNA damage</span></div><div class="casAuthors">Jung, Eun Sun; Choi, Hyunjung; Song, Hyundong; Hwang, Yu Jin; Kim, Ahbin; Ryu, Hoon; Inhee, Mook-Jung</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">25628</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is the most common type of dementia and age-related neurodegenerative disease.  Elucidating the cellular changes that occur during ageing is an important step towards understanding the pathogenesis and progression of neurodegenerative disorders.  SIRT6 is a member of the mammalian sirtuin family of anti-aging genes.  However, the relationship between SIRT6 and AD has not yet been elucidated.  Here, we report that SIRT6 protein expression levels are reduced in the brains of both the 5XFAD AD mouse model and AD patients.  Aβ42, a major component of senile plaques, decreases SIRT6 expression, and Aβ42-induced DNA damage is prevented by the overexpression of SIRT6 in HT22 mouse hippocampal neurons.  Also, there is a strong neg. correlation between Aβ42-induced DNA damage and p53 levels, a protein involved in DNA repair and apoptosis.  In addn., upregulation of p53 protein by Nutlin-3 prevents SIRT6 redn. and DNA damage induced by Aβ42.  Taken together, this study reveals that p53-dependent SIRT6 expression protects cells from Aβ42-induced DNA damage, making SIRT6 a promising new therapeutic target for the treatment of AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaThmGINJMprVg90H21EOLACvtfcHk0lhtiwo4ImO9rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWlt7s%253D&md5=ef78344498ac9f200b3087f0bb4e094a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsrep25628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep25628%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DE.%2BS.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DHwang%26aufirst%3DY.%2BJ.%26aulast%3DKim%26aufirst%3DA.%26aulast%3DRyu%26aufirst%3DH.%26aulast%3DMook-Jung%26aufirst%3DI.%26atitle%3Dp53-dependent%2520SIRT6%2520expression%2520protects%2520A%25CE%25B242-induced%2520DNA%2520damage%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D25628%26doi%3D10.1038%2Fsrep25628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaluski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portillo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueberham, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toiber, D.</span></span> <span> </span><span class="NLM_article-title">Neuroprotective functions for the histone deacetylase SIRT6</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3052</span>– <span class="NLM_lpage">3062</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.celrep.2017.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=28355558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlt12ru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=3052-3062&author=S.+Kaluskiauthor=M.+Portilloauthor=A.+Besnardauthor=D.+Steinauthor=M.+Einavauthor=L.+Zhongauthor=U.+Ueberhamauthor=T.+Arendtauthor=R.+Mostoslavskyauthor=A.+Sahayauthor=D.+Toiber&title=Neuroprotective+functions+for+the+histone+deacetylase+SIRT6&doi=10.1016%2Fj.celrep.2017.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Neuroprotective Functions for the Histone Deacetylase SIRT6</span></div><div class="casAuthors">Kaluski, Shai; Portillo, Miguel; Besnard, Antoine; Stein, Daniel; Einav, Monica; Zhong, Lei; Ueberham, Uwe; Arendt, Thomas; Mostoslavsky, Raul; Sahay, Amar; Toiber, Debra</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3052-3062</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The histone deacetylase SIRT6 promotes DNA repair, but its activity declines with age with a concomitant accumulation of DNA damage.  Furthermore, SIRT6 knockout mice exhibit an accelerated aging phenotype and die prematurely.  Here, we report that brain-specific SIRT6-deficient mice survive but present behavioral defects with major learning impairments by 4 mo of age.  Moreover, the brains of these mice show increased signs of DNA damage, cell death, and hyperphosphorylated Tau-a crit. mark in several neurodegenerative diseases.  Mechanistically, SIRT6 regulates Tau protein stability and phosphorylation through increased activation of the kinase GSK3α/β.  Finally, SIRT6 mRNA and protein levels are reduced in patients with Alzheimer's disease.  Taken together, our results suggest that SIRT6 is crit. to maintain genomic stability in the brain and that its loss leads to toxic Tau stability and phosphorylation.  Therefore, SIRT6 and its downstream signaling could be targeted in Alzheimer's disease and age-related neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodlby_QkHa2bVg90H21EOLACvtfcHk0ljU8-3eI4fOmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlt12ru78%253D&md5=949b4d56c910332ab0c823675afaa58d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DKaluski%26aufirst%3DS.%26aulast%3DPortillo%26aufirst%3DM.%26aulast%3DBesnard%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DD.%26aulast%3DEinav%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DUeberham%26aufirst%3DU.%26aulast%3DArendt%26aufirst%3DT.%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DSahay%26aufirst%3DA.%26aulast%3DToiber%26aufirst%3DD.%26atitle%3DNeuroprotective%2520functions%2520for%2520the%2520histone%2520deacetylase%2520SIRT6%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D18%26spage%3D3052%26epage%3D3062%26doi%3D10.1016%2Fj.celrep.2017.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Urso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toiber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vadysirisack, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinelli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wikstrom, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nir, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaitheesvaran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iliopoulos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurland, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dor, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirihai, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellisen, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">The histone deacetylase SIRT6 regulates glucose homeostasis via Hif1α</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">280</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.12.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2009.12.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=20141841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVWitLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2010&pages=280-293&author=L.+Zhongauthor=A.+D%E2%80%99Ursoauthor=D.+Toiberauthor=C.+Sebastianauthor=R.+E.+Henryauthor=D.+D.+Vadysirisackauthor=A.+Guimaraesauthor=B.+Marinelliauthor=J.+D.+Wikstromauthor=T.+Nirauthor=C.+B.+Clishauthor=B.+Vaitheesvaranauthor=O.+Iliopoulosauthor=I.+Kurlandauthor=Y.+Dorauthor=R.+Weisslederauthor=O.+S.+Shirihaiauthor=L.+W.+Ellisenauthor=J.+M.+Espinosaauthor=R.+Mostoslavsky&title=The+histone+deacetylase+SIRT6+regulates+glucose+homeostasis+via+Hif1%CE%B1&doi=10.1016%2Fj.cell.2009.12.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The histone deacetylase Sirt6 regulates glucose homeostasis via Hif1α</span></div><div class="casAuthors">Zhong, Lei; D'Urso, Agustina; Toiber, Debra; Sebastian, Carlos; Henry, Ryan E.; Vadysirisack, Douangsone D.; Guimaraes, Alexander; Marinelli, Brett; Wikstrom, Jakob D.; Nir, Tomer; Clish, Clary B.; Vaitheesvaran, Bhavapriya; Iliopoulos, Othon; Kurland, Irwin; Dor, Yuval; Weissleder, Ralph; Shirihai, Orian S.; Ellisen, Leif W.; Espinosa, Joaquin M.; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">280-293</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">SIRT6 is a member of a highly conserved family of NAD+-dependent deacetylases with various roles in metab., stress resistance, and life span.  SIRT6-deficient mice develop normally but succumb to a lethal hypoglycemia early in life; however, the mechanism underlying this hypoglycemia remained unclear.  Here, we demonstrate that SIRT6 functions as a histone H3K9 deacetylase to control the expression of multiple glycolytic genes.  Specifically, SIRT6 appears to function as a corepressor of the transcription factor Hif1α, a crit. regulator of nutrient stress responses.  Consistent with this notion, SIRT6-deficient cells exhibit increased Hif1α activity and show increased glucose uptake with upregulation of glycolysis and diminished mitochondrial respiration.  Our studies uncover a role for the chromatin factor SIRT6 as a master regulator of glucose homeostasis and may provide the basis for novel therapeutic approaches against metabolic diseases, such as diabetes and obesity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxNnBUuYtLELVg90H21EOLACvtfcHk0ljU8-3eI4fOmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVWitLc%253D&md5=a6cef44265a7bbbc86343484b18b5e43</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.12.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.12.041%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DD%25E2%2580%2599Urso%26aufirst%3DA.%26aulast%3DToiber%26aufirst%3DD.%26aulast%3DSebastian%26aufirst%3DC.%26aulast%3DHenry%26aufirst%3DR.%2BE.%26aulast%3DVadysirisack%26aufirst%3DD.%2BD.%26aulast%3DGuimaraes%26aufirst%3DA.%26aulast%3DMarinelli%26aufirst%3DB.%26aulast%3DWikstrom%26aufirst%3DJ.%2BD.%26aulast%3DNir%26aufirst%3DT.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DVaitheesvaran%26aufirst%3DB.%26aulast%3DIliopoulos%26aufirst%3DO.%26aulast%3DKurland%26aufirst%3DI.%26aulast%3DDor%26aufirst%3DY.%26aulast%3DWeissleder%26aufirst%3DR.%26aulast%3DShirihai%26aufirst%3DO.%2BS.%26aulast%3DEllisen%26aufirst%3DL.%2BW.%26aulast%3DEspinosa%26aufirst%3DJ.%2BM.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DThe%2520histone%2520deacetylase%2520SIRT6%2520regulates%2520glucose%2520homeostasis%2520via%2520Hif1%25CE%25B1%26jtitle%3DCell%26date%3D2010%26volume%3D140%26spage%3D280%26epage%3D293%26doi%3D10.1016%2Fj.cell.2009.12.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundaresan, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillai, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravindra, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeevanandam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, C.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, M. P.</span></span> <span> </span><span class="NLM_article-title">The sirtuin SIRT6 blocks IGF-Akt signaling and development of cardiac hypertrophy by targeting c-Jun</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1643</span>– <span class="NLM_lpage">1650</span>, <span class="refDoi"> DOI: 10.1038/nm.2961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnm.2961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23086477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFeqs7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1643-1650&author=N.+R.+Sundaresanauthor=P.+Vasudevanauthor=L.+Zhongauthor=G.+Kimauthor=S.+Samantauthor=V.+Parekhauthor=V.+B.+Pillaiauthor=P.+V.+Ravindraauthor=M.+Guptaauthor=V.+Jeevanandamauthor=J.+M.+Cunninghamauthor=C.-X.+Dengauthor=D.+B.+Lombardauthor=R.+Mostoslavskyauthor=M.+P.+Gupta&title=The+sirtuin+SIRT6+blocks+IGF-Akt+signaling+and+development+of+cardiac+hypertrophy+by+targeting+c-Jun&doi=10.1038%2Fnm.2961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The sirtuin SIRT6 blocks IGF-Akt signaling and development of cardiac hypertrophy by targeting c-Jun</span></div><div class="casAuthors">Sundaresan, Nagalingam R.; Vasudevan, Prabhakaran; Zhong, Lei; Kim, Gene; Samant, Sadhana; Parekh, Vishwas; Pillai, Vinodkumar B.; Ravindra, P. V.; Gupta, Madhu; Jeevanandam, Valluvan; Cunningham, John M.; Deng, Chu-Xia; Lombard, David B.; Mostoslavsky, Raul; Gupta, Mahesh P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1643-1650</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Abnormal activation of insulin-like growth factor (IGF)-Akt signaling is implicated in the development of various diseases, including heart failure.  However, the mol. mechanisms that regulate activation of this signaling pathway are not completely understood.  Here we show that sirtuin 6 (SIRT6), a nuclear histone deacetylase, functions at the level of chromatin to directly attenuate IGF-Akt signaling.  SIRT6-deficient mice developed cardiac hypertrophy and heart failure, whereas SIRT6 transgenic mice were protected from hypertrophic stimuli, indicating that SIRT6 acts as a neg. regulator of cardiac hypertrophy.  SIRT6-deficient mouse hearts showed hyperactivation of IGF signaling-related genes and their downstream targets.  Mechanistically, SIRT6 binds to and suppresses the promoter of IGF signaling-related genes by interacting with c-Jun and deacetylating histone 3 at Lys9 (H3K9).  We also found reduced SIRT6 expression in human failing hearts.  These findings disclose a new link between SIRT6 and IGF-Akt signaling and implicate SIRT6 in the development of cardiac hypertrophy and failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-wZqCB1pYErVg90H21EOLACvtfcHk0ljU8-3eI4fOmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFeqs7zL&md5=731cda8d921f4e6abdbdbf8159255fde</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnm.2961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2961%26sid%3Dliteratum%253Aachs%26aulast%3DSundaresan%26aufirst%3DN.%2BR.%26aulast%3DVasudevan%26aufirst%3DP.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DG.%26aulast%3DSamant%26aufirst%3DS.%26aulast%3DParekh%26aufirst%3DV.%26aulast%3DPillai%26aufirst%3DV.%2BB.%26aulast%3DRavindra%26aufirst%3DP.%2BV.%26aulast%3DGupta%26aufirst%3DM.%26aulast%3DJeevanandam%26aufirst%3DV.%26aulast%3DCunningham%26aufirst%3DJ.%2BM.%26aulast%3DDeng%26aufirst%3DC.-X.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DGupta%26aufirst%3DM.%2BP.%26atitle%3DThe%2520sirtuin%2520SIRT6%2520blocks%2520IGF-Akt%2520signaling%2520and%2520development%2520of%2520cardiac%2520hypertrophy%2520by%2520targeting%2520c-Jun%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D1643%26epage%3D1650%26doi%3D10.1038%2Fnm.2961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, R.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.-Q.</span></span> <span> </span><span class="NLM_article-title">Nmnat2 protects cardiomyocytes from hypertrophy via activation of SIRT6</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>586</i></span>,  <span class="NLM_fpage">866</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2012.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.febslet.2012.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=22449973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtF2js70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=866-874&author=Y.+Caiauthor=S.-S.+Yuauthor=S.-R.+Chenauthor=R.-B.+Piauthor=S.+Gaoauthor=H.+Liauthor=J.-T.+Yeauthor=P.-Q.+Liu&title=Nmnat2+protects+cardiomyocytes+from+hypertrophy+via+activation+of+SIRT6&doi=10.1016%2Fj.febslet.2012.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Nmnat2 protects cardiomyocytes from hypertrophy via activation of SIRT6</span></div><div class="casAuthors">Cai, Yi; Yu, Shan-Shan; Chen, Shao-Rui; Pi, Rong-Biao; Gao, Si; Li, Hong; Ye, Jian-Tao; Liu, Pei-Qing</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">866-874</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery of sirtuins (SIRT), a family of NAD (NAD)-dependent deacetylases, has indicated that intracellular NAD level is crucial for the hypertrophic response of cardiomyocytes.  NMN adenylyltransferase (Nmnat) is a central enzyme in NAD biosynthesis.  Here we revealed that Nmnat2 protein expression and enzyme activity were down-regulated during cardiac hypertrophy.  In neonatal rat cardiomyocytes, overexpression of Nmnat2 but not its catalytically inactive mutant blocked angiotensin II (Ang II)-induced cardiac hypertrophy, which was dependent on activation of SIRT6 through maintaining the intracellular NAD level.  Our results suggested that modulation of Nmnat2 activity may be beneficial in cardiac hypertrophy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppOcYUeItX4LVg90H21EOLACvtfcHk0lgAv8odcdZ5IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtF2js70%253D&md5=dd6d2142aa5bc16afffdb73cd705e4cd</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DS.-S.%26aulast%3DChen%26aufirst%3DS.-R.%26aulast%3DPi%26aufirst%3DR.-B.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DYe%26aufirst%3DJ.-T.%26aulast%3DLiu%26aufirst%3DP.-Q.%26atitle%3DNmnat2%2520protects%2520cardiomyocytes%2520from%2520hypertrophy%2520via%2520activation%2520of%2520SIRT6%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D866%26epage%3D874%26doi%3D10.1016%2Fj.febslet.2012.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Min, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakiri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatloukal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, E. F.</span></span> <span> </span><span class="NLM_article-title">Liver cancer initiation is controlled by AP-1 through SIRT6-dependent inhibition of survivin</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1203</span>– <span class="NLM_lpage">1211</span>, <span class="refDoi"> DOI: 10.1038/ncb2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fncb2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23041974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKisLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=1203-1211&author=L.+Minauthor=Y.+Jiauthor=L.+Bakiriauthor=Z.+Qiuauthor=J.+Cenauthor=X.+Chenauthor=L.+Chenauthor=H.+Scheuchauthor=H.+Zhengauthor=L.+Qinauthor=K.+Zatloukalauthor=L.+Huiauthor=E.+F.+Wagner&title=Liver+cancer+initiation+is+controlled+by+AP-1+through+SIRT6-dependent+inhibition+of+survivin&doi=10.1038%2Fncb2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Liver cancer initiation is controlled by AP-1 through SIRT6-dependent inhibition of survivin</span></div><div class="casAuthors">Min, Lihua; Ji, Yuan; Bakiri, Latifa; Qiu, Zhixin; Cen, Jin; Chen, Xiaotao; Chen, Lingli; Scheuch, Harald; Zheng, Hai; Qin, Lunxiu; Zatloukal, Kurt; Hui, Lijian; Wagner, Erwin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1203-1211</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Understanding stage-dependent oncogenic mechanisms is crit. to develop not only targeted therapies, but also diagnostic markers and preventive strategies.  The mechanisms acting during cancer initiation remain elusive, largely owing to a lack of suitable animal models and limited availability of human precancerous lesions.  Here we show using genetic mouse models specific for liver cancer initiation, that survival of initiated cancer cells is controlled by c-Jun, independently of p53, through suppressing c-Fos-mediated apoptosis.  Mechanistically, c-Fos induces SIRT6 transcription, which represses survivin by reducing histone H3K9 acetylation and NF-κB activation.  Importantly, increasing the level of SIRT6 or targeting the anti-apoptotic activity of survivin at the initiation stage markedly impairs cancer development.  Moreover, in human dysplastic liver nodules, but not in malignant tumors, a specific expression pattern with increased c-Jun-survivin and attenuated c-Fos-SIRT6 levels was identified.  These results reveal a regulatory network connecting stress response and histone modification in liver tumor initiation, which could be targeted to prevent liver tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHiziBDVkjvbVg90H21EOLACvtfcHk0lgAv8odcdZ5IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKisLnN&md5=bf66909a3f8db10890c8d76f0fc3e3b6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fncb2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2590%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DL.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DBakiri%26aufirst%3DL.%26aulast%3DQiu%26aufirst%3DZ.%26aulast%3DCen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DScheuch%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DZatloukal%26aufirst%3DK.%26aulast%3DHui%26aufirst%3DL.%26aulast%3DWagner%26aufirst%3DE.%2BF.%26atitle%3DLiver%2520cancer%2520initiation%2520is%2520controlled%2520by%2520AP-1%2520through%2520SIRT6-dependent%2520inhibition%2520of%2520survivin%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2012%26volume%3D14%26spage%3D1203%26epage%3D1211%26doi%3D10.1038%2Fncb2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marquardt, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashyap, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krupp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teufel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zechner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strand, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorgeirsson, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galle, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strand, S.</span></span> <span> </span><span class="NLM_article-title">Sirtuin-6–dependent genetic and epigenetic alterations are associated with poor clinical outcome in hepatocellular carcinoma patients</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1054</span>– <span class="NLM_lpage">1064</span>, <span class="refDoi"> DOI: 10.1002/hep.26413</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1002%2Fhep.26413" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23526469" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlOkt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2013&pages=1054-1064&author=J.+U.+Marquardtauthor=K.+Fischerauthor=K.+Bausauthor=A.+Kashyapauthor=S.+Maauthor=M.+Kruppauthor=M.+Linkeauthor=A.+Teufelauthor=U.+Zechnerauthor=D.+Strandauthor=S.+S.+Thorgeirssonauthor=P.+R.+Galleauthor=S.+Strand&title=Sirtuin-6%E2%80%93dependent+genetic+and+epigenetic+alterations+are+associated+with+poor+clinical+outcome+in+hepatocellular+carcinoma+patients&doi=10.1002%2Fhep.26413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Sirtuin-6-dependent genetic and epigenetic alterations are associated with poor clinical outcome in hepatocellular carcinoma patients</span></div><div class="casAuthors">Marquardt, Jens U.; Fischer, Kerstin; Baus, Katharina; Kashyap, Anubha; Ma, Shengyun; Krupp, Markus; Linke, Matthias; Teufel, Andreas; Zechner, Ulrich; Strand, Dennis; Thorgeirsson, Snorri S.; Galle, Peter R.; Strand, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1054-1064</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Sirtuin 6 (SIRT6) is a member of the sirtuin family of NAD+-dependent deacetylases.  Genetic deletion of Sirt6 in mice results in a severe degenerative phenotype with impaired liver function and premature death.  The role of SIRT6 in development and progression of hepatocellular carcinoma is currently unknown.  We first investigated SIRT6 expression in 153 primary human liver cancers and in normal and cirrhotic livers using microarray anal.  SIRT6 was significantly down-regulated in both cirrhotic livers and cancer.  A Sirt6 knockout (KO) gene expression signature was generated from primary hepatoctyes isolated from 3-wk-old Sirt6-deficient animals.  Sirt6-deficient hepatocytes showed up-regulation of established hepatocellular carcinoma (HCC) biomarkers alpha-fetoprotein (Afp), insulin-like growth factor 2 (Igf2), H19, and glypican-3.  Furthermore, decreased SIRT6 expression was obsd. in hepatoma cell lines that are known to be apoptosis-insensitive.  Re-expression of SIRT6 in HepG2 cells increased apoptosis sensitivity to CD95-stimulation or chemotherapy treatment.  Loss of Sirt6 was characterized by oncogenic changes, such as global hypomethylation, as well as metabolic changes, such as hypoglycemia and increased fat deposition.  The hepatocyte-specific Sirt6-KO signature had a prognostic impact and was enriched in patients with poorly differentiated tumors with high AFP levels as well as recurrent disease.  Finally, we demonstrated that the Sirt6-KO signature possessed a predictive value for tumors other than HCC (e.g., breast and lung cancer).  Conclusion: Loss of SIRT6 induces epigenetic changes that may be relevant to chronic liver disease and HCC development.  Down-regulation of SIRT6 and genes dysregulated by loss of SIRT6 possess oncogenic effects in hepatocarcinogenesis.  Our data demonstrate that deficiency in one epigenetic regulator predisposes a tumorigenic phenotype that ultimately has relevance for outcome of HCC and other cancer patients.  (Hepatol. 2013;53:1054-1064).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwQ-ICe0Bv57Vg90H21EOLACvtfcHk0lgAv8odcdZ5IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlOkt7rO&md5=d596a2fe828beed5b73c3cea0fa68e85</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fhep.26413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.26413%26sid%3Dliteratum%253Aachs%26aulast%3DMarquardt%26aufirst%3DJ.%2BU.%26aulast%3DFischer%26aufirst%3DK.%26aulast%3DBaus%26aufirst%3DK.%26aulast%3DKashyap%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DKrupp%26aufirst%3DM.%26aulast%3DLinke%26aufirst%3DM.%26aulast%3DTeufel%26aufirst%3DA.%26aulast%3DZechner%26aufirst%3DU.%26aulast%3DStrand%26aufirst%3DD.%26aulast%3DThorgeirsson%26aufirst%3DS.%2BS.%26aulast%3DGalle%26aufirst%3DP.%2BR.%26aulast%3DStrand%26aufirst%3DS.%26atitle%3DSirtuin-6%25E2%2580%2593dependent%2520genetic%2520and%2520epigenetic%2520alterations%2520are%2520associated%2520with%2520poor%2520clinical%2520outcome%2520in%2520hepatocellular%2520carcinoma%2520patients%26jtitle%3DHepatology%26date%3D2013%26volume%3D58%26spage%3D1054%26epage%3D1064%26doi%3D10.1002%2Fhep.26413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cagnetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soncini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orecchioni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talarico, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minetto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guolo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Retali, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carminati, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clavio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miglino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamaschi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahimana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchosal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todoerti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passalacqua, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruzzone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nencioni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoli, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cea, M.</span></span> <span> </span><span class="NLM_article-title">Depletion of SIRT6 enzymatic activity increases acute myeloid leukemia cells’ vulnerability to DNA-damaging agents</span>. <i>Haematologica</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.3324/haematol.2017.176248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.3324%2Fhaematol.2017.176248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=29025907" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1aiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2018&pages=80-90&author=A.+Cagnettaauthor=D.+Sonciniauthor=S.+Orecchioniauthor=G.+Talaricoauthor=P.+Minettoauthor=F.+Guoloauthor=V.+Retaliauthor=N.+Colomboauthor=E.+Carminatiauthor=M.+Clavioauthor=M.+Miglinoauthor=M.+Bergamaschiauthor=A.+Nahimanaauthor=M.+Duchosalauthor=K.+Todoertiauthor=A.+Neriauthor=M.+Passalacquaauthor=S.+Bruzzoneauthor=A.+Nencioniauthor=F.+Bertoliniauthor=M.+Gobbiauthor=R.+M.+Lemoliauthor=M.+Cea&title=Depletion+of+SIRT6+enzymatic+activity+increases+acute+myeloid+leukemia+cells%E2%80%99+vulnerability+to+DNA-damaging+agents&doi=10.3324%2Fhaematol.2017.176248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of SIRT6 enzymatic activity increases acute myeloid leukemia cells vulnerability to DNA-damaging agents</span></div><div class="casAuthors">Cagnetta, Antonia; Soncini, Debora; Orecchioni, Stefania; Talarico, Giovanna; Minetto, Paola; Guolo, Fabio; Retali, Veronica; Colombo, Nicoletta; Carminati, Enrico; Clavio, Marino; Miglino, Maurizio; Bergamaschi, Micaela; Nahimana, Aimable; Duchosal, Michel; Todoerti, Katia; Neri, Antonino; Passalacqua, Mario; Bruzzone, Santina; Nencioni, Alessio; Bertolini, Francesco; Gobbi, Marco; Lemoli, Roberto M.; Cea, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Haematologica</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-90</span>CODEN:
                <span class="NLM_cas:coden">HAEMAX</span>;
        ISSN:<span class="NLM_cas:issn">1592-8721</span>.
    
            (<span class="NLM_cas:orgname">Ferrata Storti Foundation</span>)
        </div><div class="casAbstract">Genomic instability plays a pathol. role in various malignancies, including acute myeloid leukemia (AML), and thus represents a potential therapeutic target.  Recent studies demonstrate that SIRT6, a NAD+-dependent nuclear deacetylase, functions as genome-guardian by preserving DNA integrity in different tumor cells.  Here, we demonstrate that also CD34+ blasts from AML patients show ongoing DNA damage and SIRT6 overexpression.  Indeed, we identified a poor-prognostic subset of patients, with widespread instability, which relies on SIRT6 to compensate for DNA-replication stress.  As a result, SIRT6 depletion compromises the ability of leukemia cells to repair DNA double-strand breaks that, in turn, increases their sensitivity to daunorubicin and Ara-C, both in vitro and in vivo.  In contrast, low SIRT6 levels obsd. in normal CD34+ hematopoietic progenitors explain their weaker sensitivity to genotoxic stress.  Intriguingly, we have identified DNA-PKcs and CtIP deacetylation as crucial for SIRT6-mediated DNA repair.  Together, our data suggest that inactivation of SIRT6 in leukemia cells leads to disruption of DNA-repair mechanisms, genomic instability and aggressive AML.  This synthetic lethal approach, enhancing DNA damage while concomitantly blocking repair responses, provides the rationale for the clin. evaluation of SIRT6 modulators in the treatment of leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrm3PFOl7M6MrVg90H21EOLACvtfcHk0lgs9pj5xnVUfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1aiurk%253D&md5=5838badf698489bbb92b758cad8f57ab</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3324%2Fhaematol.2017.176248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3324%252Fhaematol.2017.176248%26sid%3Dliteratum%253Aachs%26aulast%3DCagnetta%26aufirst%3DA.%26aulast%3DSoncini%26aufirst%3DD.%26aulast%3DOrecchioni%26aufirst%3DS.%26aulast%3DTalarico%26aufirst%3DG.%26aulast%3DMinetto%26aufirst%3DP.%26aulast%3DGuolo%26aufirst%3DF.%26aulast%3DRetali%26aufirst%3DV.%26aulast%3DColombo%26aufirst%3DN.%26aulast%3DCarminati%26aufirst%3DE.%26aulast%3DClavio%26aufirst%3DM.%26aulast%3DMiglino%26aufirst%3DM.%26aulast%3DBergamaschi%26aufirst%3DM.%26aulast%3DNahimana%26aufirst%3DA.%26aulast%3DDuchosal%26aufirst%3DM.%26aulast%3DTodoerti%26aufirst%3DK.%26aulast%3DNeri%26aufirst%3DA.%26aulast%3DPassalacqua%26aufirst%3DM.%26aulast%3DBruzzone%26aufirst%3DS.%26aulast%3DNencioni%26aufirst%3DA.%26aulast%3DBertolini%26aufirst%3DF.%26aulast%3DGobbi%26aufirst%3DM.%26aulast%3DLemoli%26aufirst%3DR.%2BM.%26aulast%3DCea%26aufirst%3DM.%26atitle%3DDepletion%2520of%2520SIRT6%2520enzymatic%2520activity%2520increases%2520acute%2520myeloid%2520leukemia%2520cells%25E2%2580%2599%2520vulnerability%2520to%2520DNA-damaging%2520agents%26jtitle%3DHaematologica%26date%3D2018%26volume%3D103%26spage%3D80%26epage%3D90%26doi%3D10.3324%2Fhaematol.2017.176248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagnetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulciniti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhasin, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monacelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballestrero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoli, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nencioni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span> <span> </span><span class="NLM_article-title">Evidence for a role of the histone deacetylase SIRT6 in DNA damage response of multiple myeloma cells</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">1138</span>– <span class="NLM_lpage">1150</span>, <span class="refDoi"> DOI: 10.1182/blood-2015-06-649970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1182%2Fblood-2015-06-649970" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26675349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Ggu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2016&pages=1138-1150&author=M.+Ceaauthor=A.+Cagnettaauthor=S.+Adamiaauthor=C.+Acharyaauthor=Y.-T.+Taiauthor=M.+Fulcinitiauthor=H.+Ohguchiauthor=A.+Munshiauthor=P.+Acharyaauthor=M.+K.+Bhasinauthor=L.+Zhongauthor=R.+Carrascoauthor=F.+Monacelliauthor=A.+Ballestreroauthor=P.+Richardsonauthor=M.+Gobbiauthor=R.+M.+Lemoliauthor=N.+Munshiauthor=T.+Hideshimaauthor=A.+Nencioniauthor=D.+Chauhanauthor=K.+C.+Anderson&title=Evidence+for+a+role+of+the+histone+deacetylase+SIRT6+in+DNA+damage+response+of+multiple+myeloma+cells&doi=10.1182%2Fblood-2015-06-649970"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for a role of the histone deacetylase SIRT6 in DNA damage response of multiple myeloma cells</span></div><div class="casAuthors">Cea, Michele; Cagnetta, Antonia; Adamia, Sophia; Acharya, Chirag; Tai, Yu-Tzu; Fulciniti, Mariateresa; Ohguchi, Hiroto; Munshi, Aditya; Acharya, Prakrati; Bhasin, Manoj K.; Zhong, Lei; Carrasco, Ruben; Monacelli, Fiammetta; Ballestrero, Alberto; Richardson, Paul; Gobbi, Marco; Lemoli, Roberto M.; Munshi, Nikhil; Hideshima, Teru; Nencioni, Alessio; Chauhan, Dharminder; Anderson, Kenneth C.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1138-1150</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Multiple myeloma (MM) is characterized by a highly unstable genome, with aneuploidy obsd. in nearly all patients.  The mechanism causing this karyotypic instability is largely unknown, but recent observations have correlated these abnormalities with dysfunctional DNA damage response.  Here, we show that the NAD+-dependent deacetylase SIRT6 is highly expressed in MM cells, as an adaptive response to genomic stability, and that high SIRT6 levels are assocd. with adverse prognosis.  Mechanistically, SIRT6 interacts with the transcription factor ELK1 and with the ERK signaling-related gene.  By binding to their promoters and deacetylating H3K9 at these sites, SIRT6 downregulates the expression of mitogen-activated protein kinase (MAPK) pathway genes, MAPK signaling, and proliferation.  In addn., inactivation of ERK2/p90RSK signaling triggered by high SIRT6 levels increases DNA repair via Chk1 and confers resistance to DNA damage.  Using genetic and biochem. studies in vitro and in human MM xenograft models, we show that SIRT6 depletion both enhances proliferation and confers sensitization to DNA-damaging agents.  Our findings therefore provide insights into the functional interplay between SIRT6 and DNA repair mechanisms, with implications for both tumorigenesis and the treatment of MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFsgwJVN8UIrVg90H21EOLACvtfcHk0lgs9pj5xnVUfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Ggu77N&md5=5dfdc3bba615c8c760bb288066a7e7db</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1182%2Fblood-2015-06-649970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2015-06-649970%26sid%3Dliteratum%253Aachs%26aulast%3DCea%26aufirst%3DM.%26aulast%3DCagnetta%26aufirst%3DA.%26aulast%3DAdamia%26aufirst%3DS.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DTai%26aufirst%3DY.-T.%26aulast%3DFulciniti%26aufirst%3DM.%26aulast%3DOhguchi%26aufirst%3DH.%26aulast%3DMunshi%26aufirst%3DA.%26aulast%3DAcharya%26aufirst%3DP.%26aulast%3DBhasin%26aufirst%3DM.%2BK.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DCarrasco%26aufirst%3DR.%26aulast%3DMonacelli%26aufirst%3DF.%26aulast%3DBallestrero%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DGobbi%26aufirst%3DM.%26aulast%3DLemoli%26aufirst%3DR.%2BM.%26aulast%3DMunshi%26aufirst%3DN.%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DNencioni%26aufirst%3DA.%26aulast%3DChauhan%26aufirst%3DD.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26atitle%3DEvidence%2520for%2520a%2520role%2520of%2520the%2520histone%2520deacetylase%2520SIRT6%2520in%2520DNA%2520damage%2520response%2520of%2520multiple%2520myeloma%2520cells%26jtitle%3DBlood%26date%3D2016%26volume%3D127%26spage%3D1138%26epage%3D1150%26doi%3D10.1182%2Fblood-2015-06-649970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kugel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastián, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitamant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gladden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadreyev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardeesy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">SIRT6 suppresses pancreatic cancer through control of Lin28b</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>165</i></span>,  <span class="NLM_fpage">1401</span>– <span class="NLM_lpage">1415</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.04.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2016.04.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=27180906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvV2kt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=1401-1415&author=S.+Kugelauthor=C.+Sebasti%C3%A1nauthor=J.+Fitamantauthor=K.+N.+Rossauthor=S.+K.+Sahaauthor=E.+Jainauthor=A.+Gladdenauthor=K.+S.+Aroraauthor=Y.+Katoauthor=M.+N.+Riveraauthor=S.+Ramaswamyauthor=R.+I.+Sadreyevauthor=A.+Gorenauthor=V.+Deshpandeauthor=N.+Bardeesyauthor=R.+Mostoslavsky&title=SIRT6+suppresses+pancreatic+cancer+through+control+of+Lin28b&doi=10.1016%2Fj.cell.2016.04.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 suppresses pancreatic cancer through control of Lin28b</span></div><div class="casAuthors">Kugel, Sita; Sebastian, Carlos; Fitamant, Julien; Ross, Kenneth N.; Saha, Supriya K.; Jain, Esha; Gladden, Adrianne; Arora, Kshitij S.; Kato, Yasutaka; Rivera, Miguel N.; Ramaswamy, Sridhar; Sadreyev, Ruslan I.; Goren, Alon; Deshpande, Vikram; Bardeesy, Nabeel; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1401-1415</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin remodeling proteins are frequently dysregulated in human cancer, yet little is known about how they control tumorigenesis.  Here, we uncover an epigenetic program mediated by the NAD+-dependent histone deacetylase Sirtuin 6 (SIRT6) that is crit. for suppression of pancreatic ductal adenocarcinoma (PDAC), one of the most lethal malignancies.  SIRT6 inactivation accelerates PDAC progression and metastasis via upregulation of Lin28b, a neg. regulator of the let-7 microRNA.  SIRT6 loss results in histone hyperacetylation at the Lin28b promoter, Myc recruitment, and pronounced induction of Lin28b and downstream let-7 target genes, HMGA2, IGF2BP1, and IGF2BP3.  This epigenetic program defines a distinct subset with a poor prognosis, representing 30%-40% of human PDAC, characterized by reduced SIRT6 expression and an exquisite dependence on Lin28b for tumor growth.  Thus, we identify SIRT6 as an important PDAC tumor suppressor and uncover the Lin28b pathway as a potential therapeutic target in a molecularly defined PDAC subset.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IS8QdcElmbVg90H21EOLACvtfcHk0liExkXjBmt9Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvV2kt7s%253D&md5=553ff1dc14d8a90aa11a708e34e1795b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.04.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.04.033%26sid%3Dliteratum%253Aachs%26aulast%3DKugel%26aufirst%3DS.%26aulast%3DSebasti%25C3%25A1n%26aufirst%3DC.%26aulast%3DFitamant%26aufirst%3DJ.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DSaha%26aufirst%3DS.%2BK.%26aulast%3DJain%26aufirst%3DE.%26aulast%3DGladden%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DK.%2BS.%26aulast%3DKato%26aufirst%3DY.%26aulast%3DRivera%26aufirst%3DM.%2BN.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DSadreyev%26aufirst%3DR.%2BI.%26aulast%3DGoren%26aufirst%3DA.%26aulast%3DDeshpande%26aufirst%3DV.%26aulast%3DBardeesy%26aufirst%3DN.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DSIRT6%2520suppresses%2520pancreatic%2520cancer%2520through%2520control%2520of%2520Lin28b%26jtitle%3DCell%26date%3D2016%26volume%3D165%26spage%3D1401%26epage%3D1415%26doi%3D10.1016%2Fj.cell.2016.04.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grozio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasigliè, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basile, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturla, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sociali, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soncini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poggi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoppoli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballestrero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrone, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruzzone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nencioni, A.</span></span> <span> </span><span class="NLM_article-title">The NAD+-dependent histone deacetylase SIRT6 promotes cytokine production and migration in pancreatic cancer cells by regulating Ca<sup>2+</sup> responses</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">40924</span>– <span class="NLM_lpage">40937</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.405837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1074%2Fjbc.m112.405837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23086953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslGjs7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=40924-40937&author=I.+Bauerauthor=A.+Grozioauthor=D.+Lasigli%C3%A8author=G.+Basileauthor=L.+Sturlaauthor=M.+Magnoneauthor=G.+Socialiauthor=D.+Sonciniauthor=I.+Caffaauthor=A.+Poggiauthor=G.+Zoppoliauthor=M.+Ceaauthor=G.+Feldmannauthor=R.+Mostoslavskyauthor=A.+Ballestreroauthor=F.+Patroneauthor=S.+Bruzzoneauthor=A.+Nencioni&title=The+NAD%2B-dependent+histone+deacetylase+SIRT6+promotes+cytokine+production+and+migration+in+pancreatic+cancer+cells+by+regulating+Ca2%2B+responses&doi=10.1074%2Fjbc.m112.405837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The NAD+-dependent Histone Deacetylase SIRT6 Promotes Cytokine Production and Migration in Pancreatic Cancer Cells by Regulating Ca2+ Responses</span></div><div class="casAuthors">Bauer, Inga; Grozio, Alessia; Lasiglie, Denise; Basile, Giovanna; Sturla, Laura; Magnone, Mirko; Sociali, Giovanna; Soncini, Debora; Caffa, Irene; Poggi, Alessandro; Zoppoli, Gabriele; Cea, Michele; Feldmann, Georg; Mostoslavsky, Raul; Ballestrero, Alberto; Patrone, Franco; Bruzzone, Santina; Nencioni, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">40924-40937</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cytokine secretion by cancer cells contributes to cancer-induced symptoms and angiogenesis.  Studies show that the sirtuin SIRT6 promotes inflammation by enhancing TNF expression.  Here, we aimed to det. whether SIRT6 is involved in conferring an inflammatory phenotype to cancer cells and to define the mechanisms linking SIRT6 to inflammation.  We show that SIRT6 enhances the expression of pro-inflammatory cyto-/chemokines, such as IL8 and TNF, and promotes cell migration in pancreatic cancer cells by enhancing Ca2+ responses.  Via its enzymic activity, SIRT6 increases the intracellular levels of ADP-ribose, an activator of the Ca2+ channel TRPM2.  In turn, TRPM2 and Ca2+ are shown to be involved in SIRT6-induced TNF and IL8 expression.  SIRT6 increases the nuclear levels of the Ca2+-dependent transcription factor, nuclear factor of activated T cells (NFAT), and cyclosporin A, a calcineurin inhibitor that reduces NFAT activity, reduces TNF and IL8 expression in SIRT6-overexpressing cells.  These results implicate a role for SIRT6 in the synthesis of Ca2+-mobilizing second messengers, in the regulation of Ca2+-dependent transcription factors, and in the expression of pro-inflammatory, pro-angiogenic, and chemotactic cytokines.  SIRT6 inhibition may help combat cancer-induced inflammation, angiogenesis, and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9ymWqzmiWv7Vg90H21EOLACvtfcHk0liExkXjBmt9Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslGjs7nN&md5=59c3a2081b85bbbd3543643035388f4c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m112.405837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m112.405837%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DGrozio%26aufirst%3DA.%26aulast%3DLasigli%25C3%25A8%26aufirst%3DD.%26aulast%3DBasile%26aufirst%3DG.%26aulast%3DSturla%26aufirst%3DL.%26aulast%3DMagnone%26aufirst%3DM.%26aulast%3DSociali%26aufirst%3DG.%26aulast%3DSoncini%26aufirst%3DD.%26aulast%3DCaffa%26aufirst%3DI.%26aulast%3DPoggi%26aufirst%3DA.%26aulast%3DZoppoli%26aufirst%3DG.%26aulast%3DCea%26aufirst%3DM.%26aulast%3DFeldmann%26aufirst%3DG.%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DBallestrero%26aufirst%3DA.%26aulast%3DPatrone%26aufirst%3DF.%26aulast%3DBruzzone%26aufirst%3DS.%26aulast%3DNencioni%26aufirst%3DA.%26atitle%3DThe%2520NAD%252B-dependent%2520histone%2520deacetylase%2520SIRT6%2520promotes%2520cytokine%2520production%2520and%2520migration%2520in%2520pancreatic%2520cancer%2520cells%2520by%2520regulating%2520Ca2%252B%2520responses%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D40924%26epage%3D40937%26doi%3D10.1074%2Fjbc.m112.405837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span> <span> </span><span class="NLM_article-title">SIRT6 deacetylates PKM2 to suppress its nuclear localization and oncogenic functions</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">E538</span>– <span class="NLM_lpage">E547</span>, <span class="refDoi"> DOI: 10.1073/pnas.1520045113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1073%2Fpnas.1520045113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26787900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFShtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=E538-E547&author=A.+Bhardwajauthor=S.+Das&title=SIRT6+deacetylates+PKM2+to+suppress+its+nuclear+localization+and+oncogenic+functions&doi=10.1073%2Fpnas.1520045113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT6 deacetylates PKM2 to suppress its nuclear localization and oncogenic functions</span></div><div class="casAuthors">Bhardwaj, Abhishek; Das, Sanjeev</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">E538-E547</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SIRT6 (sirtuin 6) is a member of sirtuin family of deacetylases involved in diverse processes including genome stability, metabolic homeostasis, and tumorigenesis.  However, the role of SIRT6 deacetylase activity in its tumor-suppressor functions is not well understood.  Here we report that SIRT6 binds to and deacetylates nuclear PKM2 (pyruvate kinase M2) at the lysine 433 residue.  PKM2 is a glycolytic enzyme with nonmetabolic nuclear oncogenic functions.  SIRT6-mediated deacetylation results in PKM2 nuclear export.  We further have identified exportin 4 as the specific transporter mediating PKM2 nuclear export.  As a result of SIRT6-mediated deacetylation, PKM2 nuclear protein kinase and transcriptional coactivator functions are abolished.  Thus, SIRT6 suppresses PKM2 oncogenic functions, resulting in reduced cell proliferation, migration potential, and invasiveness.  Furthermore, studies in mouse tumor models demonstrate that PKM2 deacetylation is integral to SIRT6-mediated tumor suppression and inhibition of metastasis.  Addnl., reduced SIRT6 levels correlate with elevated nuclear acetylated PKM2 levels in increasing grades of hepatocellular carcinoma.  These findings provide key insights into the pivotal role of deacetylase activity in SIRT6 tumor-suppressor functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDt-i9uz1mb7Vg90H21EOLACvtfcHk0liExkXjBmt9Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFShtb8%253D&md5=2c092b16b828cde50520835732deb04d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1520045113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1520045113%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DS.%26atitle%3DSIRT6%2520deacetylates%2520PKM2%2520to%2520suppress%2520its%2520nuclear%2520localization%2520and%2520oncogenic%2520functions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3DE538%26epage%3DE547%26doi%3D10.1073%2Fpnas.1520045113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span> <span> </span><span class="NLM_article-title">Lamin A is an endogenous SIRT6 activator and promotes SIRT6-mediated DNA repair</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1396</span>– <span class="NLM_lpage">1406</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.celrep.2015.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26549451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsl2nu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=1396-1406&author=S.+Ghoshauthor=B.+Liuauthor=Y.+Wangauthor=Q.+Haoauthor=Z.+Zhou&title=Lamin+A+is+an+endogenous+SIRT6+activator+and+promotes+SIRT6-mediated+DNA+repair&doi=10.1016%2Fj.celrep.2015.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Lamin A Is an Endogenous SIRT6 Activator and Promotes SIRT6-Mediated DNA Repair</span></div><div class="casAuthors">Ghosh, Shrestha; Liu, Baohua; Wang, Yi; Hao, Quan; Zhou, Zhongjun</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1396-1406</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The nuclear lamins are essential for various mol. events in the nucleus, such as chromatin organization, DNA replication, and provision of mech. support.  A specific point mutation in the LMNA gene creates a truncated prelamin A termed progerin, causing Hutchinson-Gilford progeria syndrome (HGPS).  SIRT6 deficiency leads to defective genomic maintenance and accelerated aging similar to HGPS, suggesting a potential link between lamin A and SIRT6.  Here, we report that lamin A is an endogenous activator of SIRT6 and facilitates chromatin localization of SIRT6 upon DNA damage.  Lamin A promotes SIRT6-dependent DNA-PKcs (DNA-PK catalytic subunit) recruitment to chromatin, CtIP deacetylation, and PARP1 mono-ADP ribosylation in response to DNA damage.  The presence of progerin jeopardizes SIRT6 activation and compromises SIRT6-mediated mol. events in response to DNA damage.  These data reveal a crit. role for lamin A in regulating SIRT6 activities, suggesting that defects in SIRT6 functions contribute to impaired DNA repair and accelerated aging in HGPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooBReO__6aJbVg90H21EOLACvtfcHk0ljrwsO70Q8eGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsl2nu7fP&md5=dd94bde5e1267f6db3ce97b5d296d4da</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DZ.%26atitle%3DLamin%2520A%2520is%2520an%2520endogenous%2520SIRT6%2520activator%2520and%2520promotes%2520SIRT6-mediated%2520DNA%2520repair%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D13%26spage%3D1396%26epage%3D1406%26doi%3D10.1016%2Fj.celrep.2015.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">You, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kambach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meleshin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steegborn, C.</span></span> <span> </span><span class="NLM_article-title">Structural basis of sirtuin 6 activation by synthetic small molecules</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1011</span>, <span class="refDoi"> DOI: 10.1002/anie.201610082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1002%2Fanie.201610082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFahs7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=1007-1011&author=W.+Youauthor=D.+Rotiliauthor=T.-M.+Liauthor=C.+Kambachauthor=M.+Meleshinauthor=M.+Schutkowskiauthor=K.+F.+Chuaauthor=A.+Maiauthor=C.+Steegborn&title=Structural+basis+of+sirtuin+6+activation+by+synthetic+small+molecules&doi=10.1002%2Fanie.201610082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Sirtuin 6 Activation by Synthetic Small Molecules</span></div><div class="casAuthors">You, Weijie; Rotili, Dante; Li, Tie-Mei; Kambach, Christian; Meleshin, Marat; Schutkowski, Mike; Chua, Katrin F.; Mai, Antonello; Steegborn, Clemens</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1007-1011</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Sirtuins are protein deacylases regulating metab. and stress responses, and are implicated in aging-related diseases.  Small mol. activators for the human sirtuins Sirt1-7 are sought as chem. tools and potential therapeutics, such as for cancer.  Activators are available for Sirt1 and exploit its unique N-terminus, whereas drug-like activators for Sirt2-7 are lacking.  The authors synthesized and screened pyrrolo[1,2-a]quinoxaline derivs., yielding the first synthetic Sirt6 activators.  Biochem. assays show direct, substrate-independent compd. binding to the Sirt6 catalytic core and potent activation of Sirt6-dependent deacetylation of peptide substrates and complete nucleosomes.  Crystal structures of Sirt6/activator complexes reveal that the compds. bind to a Sirt6-specific acyl channel pocket and identify key interactions.  The authors' results establish potent Sirt6 activation with small mols. and provide a structural basis for further development of Sirt6 activators as tools and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfGfCtcpk-4bVg90H21EOLACvtfcHk0ljrwsO70Q8eGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFahs7%252FL&md5=4eaa23343e806fdfb2397a8f662c0ef8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201610082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201610082%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DW.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DT.-M.%26aulast%3DKambach%26aufirst%3DC.%26aulast%3DMeleshin%26aufirst%3DM.%26aulast%3DSchutkowski%26aufirst%3DM.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DSteegborn%26aufirst%3DC.%26atitle%3DStructural%2520basis%2520of%2520sirtuin%25206%2520activation%2520by%2520synthetic%2520small%2520molecules%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D1007%26epage%3D1011%26doi%3D10.1002%2Fanie.201610082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">You, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steegborn, C.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the activation and inhibition of Sirtuin 6 by quercetin and its derivatives</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">19176</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-55654-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fs41598-019-55654-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=31844103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVeru7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=19176&author=W.+Youauthor=W.+Zhengauthor=S.+Weissauthor=K.+F.+Chuaauthor=C.+Steegborn&title=Structural+basis+for+the+activation+and+inhibition+of+Sirtuin+6+by+quercetin+and+its+derivatives&doi=10.1038%2Fs41598-019-55654-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the activation and inhibition of Sirtuin 6 by quercetin and its derivatives</span></div><div class="casAuthors">You, Weijie; Zheng, Wei; Weiss, Sandra; Chua, Katrin F.; Steegborn, Clemens</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19176</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mammalian Sirtuin 6 (Sirt6) is an NAD+-dependent protein deacylase regulating metab. and chromatin homeostasis.  Sirt6 activation protects against metabolic and aging-related diseases, and Sirt6 inhibition is considered a cancer therapy.  Available Sirt6 modulators show insufficient potency and specificity, and even partially contradictory Sirt6 effects were reported for the plant flavone quercetin.  To understand Sirt6 modulation by quercetin-based compds., we analyzed their binding and activity effects on Sirt6 and other Sirtuin isoforms and solved crystal structures of compd. complexes with Sirt6 and Sirt2.  We find that quercetin activates Sirt6 via the isoform-specific binding site for pyrrolo[1,2-a]quinoxalines.  Its inhibitory effect on other isoforms is based on an alternative binding site at the active site entrance.  Based on these insights, we identified isoquercetin as a ligand that can discriminate both sites and thus activates Sirt6 with increased specificity.  Furthermore, we find that quercetin derivs. that inhibit rather than activate Sirt6 exploit the same general Sirt6 binding site as the activators, identifying it as a versatile allosteric site for Sirt6 modulation.  Our results thus provide a structural basis for Sirtuin effects of quercetin-related compds. and helpful insights for Sirt6-targeted drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiCjD_hIEVCbVg90H21EOLACvtfcHk0ljrwsO70Q8eGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVeru7fO&md5=c315a5654a7c33add2f7cd13a396891f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-55654-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-55654-1%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DW.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DWeiss%26aufirst%3DS.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DSteegborn%26aufirst%3DC.%26atitle%3DStructural%2520basis%2520for%2520the%2520activation%2520and%2520inhibition%2520of%2520Sirtuin%25206%2520by%2520quercetin%2520and%2520its%2520derivatives%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D19176%26doi%3D10.1038%2Fs41598-019-55654-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Identification of a cellularly active SIRT6 allosteric activator</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0150-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fs41589-018-0150-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=30374165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVCitL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=1118-1126&author=Z.+Huangauthor=J.+Zhaoauthor=W.+Dengauthor=Y.+Chenauthor=J.+Shangauthor=K.+Songauthor=L.+Zhangauthor=C.+Wangauthor=S.+Luauthor=X.+Yangauthor=B.+Heauthor=J.+Minauthor=H.+Huauthor=M.+Tanauthor=J.+Xuauthor=Q.+Zhangauthor=J.+Zhongauthor=X.+Sunauthor=Z.+Maoauthor=H.+Linauthor=M.+Xiaoauthor=Y.+E.+Chinauthor=H.+Jiangauthor=Y.+Xuauthor=G.+Chenauthor=J.+Zhang&title=Identification+of+a+cellularly+active+SIRT6+allosteric+activator&doi=10.1038%2Fs41589-018-0150-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a cellularly active SIRT6 allosteric activator</span></div><div class="casAuthors">Huang, Zhimin; Zhao, Junxing; Deng, Wei; Chen, Yingyi; Shang, Jialin; Song, Kun; Zhang, Lu; Wang, Chengxiang; Lu, Shaoyong; Yang, Xiuyan; He, Bin; Min, Jinrong; Hu, Hao; Tan, Minjia; Xu, Jianrong; Zhang, Qiufen; Zhong, Jie; Sun, Xiaoxiang; Mao, Zhiyong; Lin, Houwen; Xiao, Mingzhe; Chin, Y. Eugene; Jiang, Hualiang; Xu, Ying; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1118-1126</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">SIRT6, a member of the SIRT deacetylase family, is responsible for deacetylation of histone H3 Nε-acetyl-lysines 9 (H3K9ac) and 56 (H3K56ac).  As a tumor suppressor, SIRT6 has frequently been found to have low expression in various cancers.  Here, we report the identification of MDL-800, a selective SIRT6 activator.  MDL-800 increased the deacetylase activity of SIRT6 by up to 22-fold via binding to an allosteric site; this interaction led to a global decrease in H3K9ac and H3K56ac levels in human hepatocellular carcinoma (HCC) cells.  Consequently, MDL-800 inhibited the proliferation of HCC cells via SIRT6-driven cell-cycle arrest and was effective in a tumor xenograft model.  Together, these data demonstrate that pharmacol. activation of SIRT6 is a potential therapeutic approach for the treatment of HCC.  MDL-800 is a first-in-class small-mol. cellular SIRT6 activator that can be used to physiol. and pathol. investigate the roles of SIRT6 deacetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFPmr-laqlLVg90H21EOLACvtfcHk0ljGH1VEXjWzyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVCitL7O&md5=5fb5c1de5b6be0291ffa45debf72516e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0150-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0150-0%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DM.%26aulast%3DChin%26aufirst%3DY.%2BE.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520a%2520cellularly%2520active%2520SIRT6%2520allosteric%2520activator%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D1118%26epage%3D1126%26doi%3D10.1038%2Fs41589-018-0150-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0ljGH1VEXjWzyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-Y.</span></span> <span> </span><span class="NLM_article-title">ID-Score: a new empirical scoring function based on a comprehensive set of descriptors related to protein–ligand interactions</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">600</span>, <span class="refDoi"> DOI: 10.1021/ci300493w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300493w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1Sqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=592-600&author=G.-B.+Liauthor=L.-L.+Yangauthor=W.-J.+Wangauthor=L.-L.+Liauthor=S.-Y.+Yang&title=ID-Score%3A+a+new+empirical+scoring+function+based+on+a+comprehensive+set+of+descriptors+related+to+protein%E2%80%93ligand+interactions&doi=10.1021%2Fci300493w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">ID-Score: A New Empirical Scoring Function Based on a Comprehensive Set of Descriptors Related to Protein-Ligand Interactions</span></div><div class="casAuthors">Li, Guo-Bo; Yang, Ling-Ling; Wang, Wen-Jing; Li, Lin-Li; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">592-600</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Scoring functions have been widely used to assess protein-ligand binding affinity in structure-based drug discovery.  However, currently commonly used scoring functions face some challenges including poor correlation between calcd. scores and exptl. binding affinities, target-dependent performance, and low sensitivity to analogs.  In this account, we propose a new empirical scoring function termed ID-Score.  ID-Score was established based on a comprehensive set of descriptors related to protein-ligand interactions; these descriptors cover nine categories: van der Waals interaction, hydrogen-bonding interaction, electrostatic interaction, π-system interaction, metal-ligand bonding interaction, desolvation effect, entropic loss effect, shape matching, and surface property matching.  A total of 2278 complexes were used as the training set, and a modified support vector regression (SVR) algorithm was used to fit the exptl. binding affinities.  Evaluation results showed that ID-Score outperformed other selected commonly used scoring functions on a benchmark test set and showed considerable performance on a large independent test set.  ID-Score also showed a consistent higher performance across different biol. targets.  Besides, it could correctly differentiate structurally similar ligands, indicating higher sensitivity to analogs.  Collectively, the better performance of ID-Score enables it as a useful tool in assessing protein-ligand binding affinity in structure-based drug discovery as well as in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtuiAPD05vtLVg90H21EOLACvtfcHk0ljGH1VEXjWzyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1Sqsbg%253D&md5=1e001ed3da47d6970a5e3ac25eaf5b1c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci300493w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300493w%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-B.%26aulast%3DYang%26aufirst%3DL.-L.%26aulast%3DWang%26aufirst%3DW.-J.%26aulast%3DLi%26aufirst%3DL.-L.%26aulast%3DYang%26aufirst%3DS.-Y.%26atitle%3DID-Score%253A%2520a%2520new%2520empirical%2520scoring%2520function%2520based%2520on%2520a%2520comprehensive%2520set%2520of%2520descriptors%2520related%2520to%2520protein%25E2%2580%2593ligand%2520interactions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D592%26epage%3D600%26doi%3D10.1021%2Fci300493w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Houston, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkinshaw, M. D.</span></span> <span> </span><span class="NLM_article-title">Consensus docking: improving the reliability of docking in a virtual screening context</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1021/ci300399w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300399w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitFeksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=384-390&author=D.+R.+Houstonauthor=M.+D.+Walkinshaw&title=Consensus+docking%3A+improving+the+reliability+of+docking+in+a+virtual+screening+context&doi=10.1021%2Fci300399w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Consensus Docking: Improving the Reliability of Docking in a Virtual Screening Context</span></div><div class="casAuthors">Houston, Douglas R.; Walkinshaw, Malcolm D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">384-390</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based virtual screening relies on scoring the predicted binding modes of compds. docked into the target.  Because the accuracy of this scoring relies on the accuracy of the docking, methods that increase docking accuracy are valuable.  Here, we present a relatively straightforward method for improving the probability of identifying accurately docked poses.  The method is similar in concept to consensus scoring schemes, which have been shown to increase ranking power and thus hit rates, but combines information about predicted binding modes rather than predicted binding affinities.  The pose prediction success rate of each docking program alone was found in this trial to be 55% for Autodock, 58% for DOCK, and 64% for Vina.  By using more than one docking program to predict the binding pose, correct poses were identified in 82% or more of cases, a significant improvement.  In a virtual screen, these more reliably posed compds. can be preferentially advanced to subsequent scoring stages to improve hit rates.  Consensus docking can be easily introduced into established structure-based virtual screening methodologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGaDety9hNcrVg90H21EOLACvtfcHk0lhzVhwcr_nYCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitFeksbc%253D&md5=6f722446700235e241a339b8dd7e3755</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci300399w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300399w%26sid%3Dliteratum%253Aachs%26aulast%3DHouston%26aufirst%3DD.%2BR.%26aulast%3DWalkinshaw%26aufirst%3DM.%2BD.%26atitle%3DConsensus%2520docking%253A%2520improving%2520the%2520reliability%2520of%2520docking%2520in%2520a%2520virtual%2520screening%2520context%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D384%26epage%3D390%26doi%3D10.1021%2Fci300399w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of human TyrRS inhibitors by structure-based virtual screening, structural optimization, and bioassays</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">9323</span>– <span class="NLM_lpage">9330</span>, <span class="refDoi"> DOI: 10.1039/c9ra00458k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1039%2Fc9ra00458k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1emtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=9323-9330&author=S.+Huangauthor=X.+Wangauthor=G.+Linauthor=J.+Chengauthor=X.+Chenauthor=W.+Sunauthor=R.+Xiangauthor=Y.+Yuauthor=L.+Liauthor=S.+Yang&title=Discovery+of+human+TyrRS+inhibitors+by+structure-based+virtual+screening%2C+structural+optimization%2C+and+bioassays&doi=10.1039%2Fc9ra00458k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of human TyrRS inhibitors by structure-based virtual screening, structural optimization, and bioassays</span></div><div class="casAuthors">Huang, Shenzhen; Wang, Xiang; Lin, Guifeng; Cheng, Jie; Chen, Xiuli; Sun, Weining; Xiang, Rong; Yu, Yamei; Li, Linli; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">9323-9330</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The human tyrosyl transfer-RNA (tRNA) synthetase (TyrRS), which is well known for its essential aminoacylation function in protein synthesis, has been shown to translocate to the nucleus and protect against DNA damage caused by external stimuli.  Small mols. that can fit into the active site pocket of TyrRS are thought to affect the nuclear role.  The exploitation of TyrRS inhibitors has attracted attention recently.  In this investigation, we adopted a structure-based virtual screening strategy and subsequent structure-activity relationship anal. to discover new TyrRS inhibitors, and identified a potent compd. 5,7-dihydroxy-6,8-bis((3-hydroxyphenyl)thio)-2-phenyl-4H-chromen-4-one (compd. 11, Ki = 8.8 μM).  In intact HeLa cells, this compd. showed a protective effect against DNA damage.  Compd. 11 is a good lead compd. for the further development of drugs against disorders caused by DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIhbDn8NV6TrVg90H21EOLACvtfcHk0lhzVhwcr_nYCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXls1emtbw%253D&md5=12335cda0cd504adb26f2130467e2405</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc9ra00458k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc9ra00458k%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DG.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520human%2520TyrRS%2520inhibitors%2520by%2520structure-based%2520virtual%2520screening%252C%2520structural%2520optimization%252C%2520and%2520bioassays%26jtitle%3DRSC%2520Adv.%26date%3D2019%26volume%3D9%26spage%3D9323%26epage%3D9330%26doi%3D10.1039%2Fc9ra00458k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of new SIRT2 inhibitors by utilizing a consensus docking/scoring strategy and structure–activity relationship analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">679</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00714</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00714" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktlynu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=669-679&author=S.+Huangauthor=C.+Songauthor=X.+Wangauthor=G.+Zhangauthor=Y.+Wangauthor=X.+Jiangauthor=Q.+Sunauthor=L.+Huangauthor=R.+Xiangauthor=Y.+Huauthor=L.+Liauthor=S.+Yang&title=Discovery+of+new+SIRT2+inhibitors+by+utilizing+a+consensus+docking%2Fscoring+strategy+and+structure%E2%80%93activity+relationship+analysis&doi=10.1021%2Facs.jcim.6b00714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of New SIRT2 Inhibitors by Utilizing a Consensus Docking/Scoring Strategy and Structure-Activity Relationship Analysis</span></div><div class="casAuthors">Huang, Shenzhen; Song, Chunli; Wang, Xiang; Zhang, Guo; Wang, Yanlin; Jiang, Xiaojuan; Sun, Qizheng; Huang, Luyi; Xiang, Rong; Hu, Yiguo; Li, Linli; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-679</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SIRT2, which is a NAD+ (NAD) dependent deacetylase, has been demonstrated to play an important role in the occurrence and development of a variety of diseases such as cancer, ischemia-reperfusion, and neurodegenerative diseases.  Small mol. inhibitors of SIRT2 are thought to be potential interfering agents for relevant diseases.  Discovery of SIRT2 inhibitors has attracted much attention recently.  In this investigation, we adopted a consensus docking/scoring strategy to screen for novel SIRT2 inhibitors.  Structural optimization and structure-activity relationship (SAR) anal. were then carried out on highly potent compds. with new scaffolds, which led to the discovery of 2-((5-benzyl-5H-[1,2,4]triazino[5,6-b]indol-3-yl)thio)-N-(naphthalen-1-yl)acetamide (SR86).  This compd. showed good activity against SIRT2 with an IC50 value of 1.3 μM.  SR86 did not exhibit activity against SIRT1 and SIRT3, implying a good selectivity for SIRT2.  In in vitro cellular assays, SR86 displayed very good antiviability activity against breast cancer cell line MCF-7.  In Western blot assays, SR86 showed considerable activity in blocking the deacetylation of α-tubulin, which is a typical substrate of SIRT2.  Collectively, because of the new scaffold structure and good selectivity of SR86, it could serve as a promising lead compd., hence deserving further studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs8mGEkJHyfbVg90H21EOLACvtfcHk0lhzVhwcr_nYCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktlynu74%253D&md5=84fedc2c19c521116e76bf16c558005a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00714%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520new%2520SIRT2%2520inhibitors%2520by%2520utilizing%2520a%2520consensus%2520docking%252Fscoring%2520strategy%2520and%2520structure%25E2%2580%2593activity%2520relationship%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D669%26epage%3D679%26doi%3D10.1021%2Facs.jcim.6b00714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhargava, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span> <span> </span><span class="NLM_article-title">A fluorogenic assay for screening Sirt6 modulators</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">5213</span>– <span class="NLM_lpage">5216</span>, <span class="refDoi"> DOI: 10.1039/c3ob41138a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1039%2Fc3ob41138a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=23839075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFKrsrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=5213-5216&author=J.+Huauthor=B.+Heauthor=S.+Bhargavaauthor=H.+Lin&title=A+fluorogenic+assay+for+screening+Sirt6+modulators&doi=10.1039%2Fc3ob41138a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorogenic assay for screening Sirt6 modulators</span></div><div class="casAuthors">Hu, Jing; He, Bin; Bhargava, Shiva; Lin, Hening</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">5213-5216</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A fluorogenic high-throughput assay suitable for screening Sirt6 modulators is developed based on the recently discovered efficient activity of Sirt6 to hydrolyze myristoyl lysine.  Sirt6 modulators will be useful in investigating the function of Sirt6 and protein lysine fatty acylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2fZveXZ0sg7Vg90H21EOLACvtfcHk0lhrKuZRSpibpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFKrsrbM&md5=184d4df01307a1edefb8faceb6a5f7b7</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2Fc3ob41138a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob41138a%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DBhargava%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DH.%26atitle%3DA%2520fluorogenic%2520assay%2520for%2520screening%2520Sirt6%2520modulators%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D5213%26epage%3D5216%26doi%3D10.1039%2Fc3ob41138a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milne, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carney, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagne, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boss, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perni, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disch, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galonek, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iffland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedik, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinclair, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olefsky, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jirousek, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westphal, C. H.</span></span> <span> </span><span class="NLM_article-title">Small molecule activators of SIRT1 as therapeutics for the treatment of type 2 diabetes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">712</span>– <span class="NLM_lpage">716</span>, <span class="refDoi"> DOI: 10.1038/nature06261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnature06261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=18046409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=712-716&author=J.+C.+Milneauthor=P.+D.+Lambertauthor=S.+Schenkauthor=D.+P.+Carneyauthor=J.+J.+Smithauthor=D.+J.+Gagneauthor=L.+Jinauthor=O.+Bossauthor=R.+B.+Perniauthor=C.+B.+Vuauthor=J.+E.+Bemisauthor=R.+Xieauthor=J.+S.+Dischauthor=P.+Y.+Ngauthor=J.+J.+Nunesauthor=A.+V.+Lynchauthor=H.+Yangauthor=H.+Galonekauthor=K.+Israelianauthor=W.+Choyauthor=A.+Ifflandauthor=S.+Lavuauthor=O.+Medvedikauthor=D.+A.+Sinclairauthor=J.+M.+Olefskyauthor=M.+R.+Jirousekauthor=P.+J.+Elliottauthor=C.+H.+Westphal&title=Small+molecule+activators+of+SIRT1+as+therapeutics+for+the+treatment+of+type+2+diabetes&doi=10.1038%2Fnature06261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule activators of SIRT1 as therapeutics for the treatment of type 2 diabetes</span></div><div class="casAuthors">Milne, Jill C.; Lambert, Philip D.; Schenk, Simon; Carney, David P.; Smith, Jesse J.; Gagne, David J.; Jin, Lei; Boss, Olivier; Perni, Robert B.; Vu, Chi B.; Bemis, Jean E.; Xie, Roger; Disch, Jeremy S.; Ng, Pui Yee; Nunes, Joseph J.; Lynch, Amy V.; Yang, Hongying; Galonek, Heidi; Israelian, Kristine; Choy, Wendy; Iffland, Andre; Lavu, Siva; Medvedik, Oliver; Sinclair, David A.; Olefsky, Jerrold M.; Jirousek, Michael R.; Elliott, Peter J.; Westphal, Christoph H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7170</span>),
    <span class="NLM_cas:pages">712-716</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Calorie restriction extends lifespan and produces a metabolic profile desirable for treating diseases of aging such as type 2 diabetes.  SIRT1, an NAD+-dependent deacetylase, is a principal modulator of pathways downstream of calorie restriction that produce beneficial effects on glucose homeostasis and insulin sensitivity.  Resveratrol, a polyphenolic SIRT1 activator, mimics the antiaging effects of calorie restriction in lower organisms and in mice fed a high-fat diet ameliorates insulin resistance, increases mitochondrial content, and prolongs survival.  Here the authors describe the identification and characterization of small mol. activators of SIRT1 that are structurally unrelated to, and 1000-fold more potent than, resveratrol.  These compds. bind to the SIRT1 enzyme-peptide substrate complex at an allosteric site amino-terminal to the catalytic domain and lower the Michaelis const. for acetylated substrates.  In diet-induced obese and genetically obese mice, these compds. improve insulin sensitivity, lower plasma glucose, and increase mitochondrial capacity.  In Zucker fa/fa rats, hyperinsulinemic-euglycemic clamp studies demonstrate that SIRT1 activators improve whole-body glucose homeostasis and insulin sensitivity in adipose tissue, skeletal muscle and liver.  Thus, SIRT1 activation is a promising new therapeutic approach for treating diseases of aging such as type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZUMUqn50MbVg90H21EOLACvtfcHk0lhrKuZRSpibpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrM&md5=8343b182d3596cbd8eb175c8b9cdcace</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnature06261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06261%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DSchenk%26aufirst%3DS.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DGagne%26aufirst%3DD.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoss%26aufirst%3DO.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26aulast%3DVu%26aufirst%3DC.%2BB.%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DNg%26aufirst%3DP.%2BY.%26aulast%3DNunes%26aufirst%3DJ.%2BJ.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGalonek%26aufirst%3DH.%26aulast%3DIsraelian%26aufirst%3DK.%26aulast%3DChoy%26aufirst%3DW.%26aulast%3DIffland%26aufirst%3DA.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DMedvedik%26aufirst%3DO.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26aulast%3DOlefsky%26aufirst%3DJ.%2BM.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DElliott%26aufirst%3DP.%2BJ.%26aulast%3DWestphal%26aufirst%3DC.%2BH.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520SIRT1%2520as%2520therapeutics%2520for%2520the%2520treatment%2520of%2520type%25202%2520diabetes%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D712%26epage%3D716%26doi%3D10.1038%2Fnature06261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davoren, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nason, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dlugolenski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaChapelle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salafia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, D.</span></span> <span> </span><span class="NLM_article-title">Discovery and lead optimization of atropisomer D1 agonists with reduced desensitization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">11384</span>– <span class="NLM_lpage">11397</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01622</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01622" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2qtrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=11384-11397&author=J.+E.+Davorenauthor=D.+Nasonauthor=J.+Coeauthor=K.+Dlugolenskiauthor=C.+Helalauthor=A.+R.+Harrisauthor=E.+LaChapelleauthor=S.+Liangauthor=Y.+Liuauthor=R.+O%E2%80%99Connorauthor=C.+C.+Orozcoauthor=B.+K.+Raiauthor=M.+Salafiaauthor=B.+Samasauthor=W.+Xuauthor=R.+Kozakauthor=D.+Gray&title=Discovery+and+lead+optimization+of+atropisomer+D1+agonists+with+reduced+desensitization&doi=10.1021%2Facs.jmedchem.8b01622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Lead Optimization of Atropisomer D1 Agonists with Reduced Desensitization</span></div><div class="casAuthors">Davoren, Jennifer E.; Nason, Deane; Coe, Jotham; Dlugolenski, Keith; Helal, Christopher; Harris, Anthony R.; LaChapelle, Erik; Liang, Sidney; Liu, Yue; OConnor, Rebecca; Orozco, Christine C.; Rai, Brajesh K.; Salafia, Michelle; Samas, Brian; Xu, Wenjian; Kozak, Rouba; Gray, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11384-11397</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of D1 subtype-selective agonists with drug-like properties has been an enduring challenge for the greater part of 40 years.  All known D1-selective agonists are catecholamines that bring about receptor desensitization and undergo rapid metab., thus limiting their utility as a therapeutic for chronic illness such as schizophrenia and Parkinson's disease.  Our high-throughput screening efforts on D1 yielded a single non-catecholamine hit PF-4211 (6) that was developed into a series of potent D1 receptor agonist leads with high oral bioavailability and CNS penetration.  An important structural feature of this series is the locked biaryl ring system resulting in atropisomerism.  Disclosed herein is a summary of our hit-to-lead efforts on this series of D1 activators culminating in the discovery of atropisomer 31 (PF-06256142), a potent and selective orthosteric agonist of the D1 receptor that has reduced receptor desensitization relative to dopamine and other catechol-contg. agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2S5u6FmBDoLVg90H21EOLACvtfcHk0lhrKuZRSpibpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2qtrvI&md5=ec84a23922f417fdeb1acd9beb3ad4bf</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01622%26sid%3Dliteratum%253Aachs%26aulast%3DDavoren%26aufirst%3DJ.%2BE.%26aulast%3DNason%26aufirst%3DD.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DDlugolenski%26aufirst%3DK.%26aulast%3DHelal%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DA.%2BR.%26aulast%3DLaChapelle%26aufirst%3DE.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%26aulast%3DOrozco%26aufirst%3DC.%2BC.%26aulast%3DRai%26aufirst%3DB.%2BK.%26aulast%3DSalafia%26aufirst%3DM.%26aulast%3DSamas%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DKozak%26aufirst%3DR.%26aulast%3DGray%26aufirst%3DD.%26atitle%3DDiscovery%2520and%2520lead%2520optimization%2520of%2520atropisomer%2520D1%2520agonists%2520with%2520reduced%2520desensitization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D11384%26epage%3D11397%26doi%3D10.1021%2Facs.jmedchem.8b01622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahusen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Ortiz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, W.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ki, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, C.-X.</span></span> <span> </span><span class="NLM_article-title">Hepatic-specific disruption of SIRT6 in mice results in fatty liver formation due to enhanced glycolysis and triglyceride synthesis</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2010.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cmet.2010.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=20816089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFamsbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=224-236&author=H.-S.+Kimauthor=C.+Xiaoauthor=R.-H.+Wangauthor=T.+Lahusenauthor=X.+Xuauthor=A.+Vassilopoulosauthor=G.+Vazquez-Ortizauthor=W.-I.+Jeongauthor=O.+Parkauthor=S.+H.+Kiauthor=B.+Gaoauthor=C.-X.+Deng&title=Hepatic-specific+disruption+of+SIRT6+in+mice+results+in+fatty+liver+formation+due+to+enhanced+glycolysis+and+triglyceride+synthesis&doi=10.1016%2Fj.cmet.2010.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatic-Specific Disruption of SIRT6 in Mice Results in Fatty Liver Formation Due to Enhanced Glycolysis and Triglyceride Synthesis</span></div><div class="casAuthors">Kim, Hyun-Seok; Xiao, Cuiying; Wang, Rui-Hong; Lahusen, Tyler; Xu, Xiaoling; Vassilopoulos, Athanassios; Vazquez-Ortiz, Guelaguetza; Jeong, Won-Il; Park, Ogyi; Ki, Sung Hwan; Gao, Bin; Deng, Chu-Xia</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">224-236</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Under various conditions, mammals have the ability to maintain serum glucose concn. within a narrow range.  SIRT1 plays an important role in regulating gluconeogenesis and fat metab.; however, the underlying mechanisms remain elusive.  Here, we show that SIRT1 forms a complex with FOXO3a and NRF1 on the SIRT6 promoter and pos. regulates expression of SIRT6, which, in turn, neg. regulates glycolysis, triglyceride synthesis, and fat metab. by deacetylating histone H3 lysine 9 in the promoter of many genes involved in these processes.  Liver-specific deletion of SIRT6 in mice causes profound alterations in gene expression, leading to increased glycolysis, triglyceride synthesis, reduced β oxidn., and fatty liver formation.  Human fatty liver samples exhibited significantly lower levels of SIRT6 than did normal controls.  Thus, SIRT6 plays a crit. role in fat metab. and may serve as a therapeutic target for treating fatty liver disease, the most common cause of liver dysfunction in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3p4P_dbupRLVg90H21EOLACvtfcHk0lju9nEnzzzQuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFamsbnP&md5=5912efaedd47bfa7ba209e2681160f95</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2010.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2010.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DXiao%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DR.-H.%26aulast%3DLahusen%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DVassilopoulos%26aufirst%3DA.%26aulast%3DVazquez-Ortiz%26aufirst%3DG.%26aulast%3DJeong%26aufirst%3DW.-I.%26aulast%3DPark%26aufirst%3DO.%26aulast%3DKi%26aufirst%3DS.%2BH.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DDeng%26aufirst%3DC.-X.%26atitle%3DHepatic-specific%2520disruption%2520of%2520SIRT6%2520in%2520mice%2520results%2520in%2520fatty%2520liver%2520formation%2520due%2520to%2520enhanced%2520glycolysis%2520and%2520triglyceride%2520synthesis%26jtitle%3DCell%2520Metab.%26date%3D2010%26volume%3D12%26spage%3D224%26epage%3D236%26doi%3D10.1016%2Fj.cmet.2010.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kokkonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahnasto-Rilla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarho, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahtela-Kakkonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokkola, T.</span></span> <span> </span><span class="NLM_article-title">Studying SIRT6 regulation using H3K56 based substrate and small molecules</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2014.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.ejps.2014.06.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=25004411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Orur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2014&pages=71-76&author=P.+Kokkonenauthor=M.+Rahnasto-Rillaauthor=P.+Melliniauthor=E.+Jarhoauthor=M.+Lahtela-Kakkonenauthor=T.+Kokkola&title=Studying+SIRT6+regulation+using+H3K56+based+substrate+and+small+molecules&doi=10.1016%2Fj.ejps.2014.06.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Studying SIRT6 regulation using H3K56 based substrate and small molecules</span></div><div class="casAuthors">Kokkonen, Piia; Rahnasto-Rilla, Minna; Mellini, Paolo; Jarho, Elina; Lahtela-Kakkonen, Maija; Kokkola, Tarja</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71-76</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">SIRT6 is a modulator of chromatin structure having an important role in healthy ageing, and there is a crucial need to find specific modulators for it.  Therefore, the activity of SIRT6 should be studied using a variety of methods.  We examd. the capability of SIRT6 to deacetylate a set of five fluorogenic substrates based on p53 and histone H3 sequences.  The substrate designed around H3K56 deacetylation site exhibited the best signal-to-background ratio and was chosen for further studies.  Nicotinamide is a known inhibitor for sirtuins, and it was found to be less potent inhibitor for SIRT6 than it is for SIRT1.  In addn., we studied 15 other small mol. sirtuin modulators using the H3K56 based substrate.  EX-527, quercetin and three pseudopeptidic compds. were found to be the most potent SIRT6 inhibitors, exhibiting over 50% deacetylation inhibition.  These findings describe the first modulators of SIRT6 activity at the physiol. important H3K56 deacetylation site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0TYojzjbca7Vg90H21EOLACvtfcHk0lju9nEnzzzQuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Orur3L&md5=93cc2b69650d1efebb5dda0128daa559</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2014.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2014.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DKokkonen%26aufirst%3DP.%26aulast%3DRahnasto-Rilla%26aufirst%3DM.%26aulast%3DMellini%26aufirst%3DP.%26aulast%3DJarho%26aufirst%3DE.%26aulast%3DLahtela-Kakkonen%26aufirst%3DM.%26aulast%3DKokkola%26aufirst%3DT.%26atitle%3DStudying%2520SIRT6%2520regulation%2520using%2520H3K56%2520based%2520substrate%2520and%2520small%2520molecules%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2014%26volume%3D63%26spage%3D71%26epage%3D76%26doi%3D10.1016%2Fj.ejps.2014.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lju9nEnzzzQuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Modeling water molecules in protein– ligand docking using GOLD</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">6504</span>– <span class="NLM_lpage">6515</span>, <span class="refDoi"> DOI: 10.1021/jm050543p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050543p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFCnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6504-6515&author=M.+L.+Verdonkauthor=G.+Chessariauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=J.+W.+M.+Nissinkauthor=R.+D.+Taylorauthor=R.+Taylor&title=Modeling+water+molecules+in+protein%E2%80%93+ligand+docking+using+GOLD&doi=10.1021%2Fjm050543p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Water Molecules in Protein-Ligand Docking Using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Chessari, Gianni; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Nissink, J. Willem M.; Taylor, Richard D.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6504-6515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We implemented a novel approach to score water mediation and displacement in the protein-ligand docking program GOLD.  The method allows water mols. to switch on and off and to rotate around their three principal axes.  A const. penalty, σp, representing the loss of rigid-body entropy, is added for water mols. that are switched on, hence rewarding water displacement.  We tested the methodol. in an extensive validation study.  First, σp is optimized against a training set of 58 protein-ligand complexes.  For this training set, our algorithm correctly predicts water mediation/displacement in ∼92% of the cases.  We obsd. small improvements in the quality of the predicted binding modes for water-mediated complexes.  In the second part of this work, an entirely independent set of 225 complexes is used.  For this test set, our algorithm correctly predicts water mediation/displacement in ∼93% of the cases.  Improvements in binding mode quality were obsd. for individual water-mediated complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKBGOf-rFSarVg90H21EOLACvtfcHk0lg0MMslXHzktQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFCnu78%253D&md5=78b60f794d8af7266f0a37f6e2aea6b1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm050543p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050543p%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DModeling%2520water%2520molecules%2520in%2520protein%25E2%2580%2593%2520ligand%2520docking%2520using%2520GOLD%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6504%26epage%3D6515%26doi%3D10.1021%2Fjm050543p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abate-Pella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span> <span> </span><span class="NLM_article-title">Oxidative reactivities of 2-furylquinolines: ubiquitous scaffolds in common high-throughput screening libraries</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7419</span>– <span class="NLM_lpage">7430</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00930</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00930" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVykt7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7419-7430&author=M.+E.+Olsonauthor=D.+Abate-Pellaauthor=A.+L.+Perkinsauthor=M.+Liauthor=M.+A.+Carpenterauthor=A.+Rathoreauthor=R.+S.+Harrisauthor=D.+A.+Harki&title=Oxidative+reactivities+of+2-furylquinolines%3A+ubiquitous+scaffolds+in+common+high-throughput+screening+libraries&doi=10.1021%2Facs.jmedchem.5b00930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative reactivities of 2-furylquinolines: ubiquitous scaffolds in common high-throughput screening libraries</span></div><div class="casAuthors">Olson, Margaret E.; Abate-Pella, Daniel; Perkins, Angela L.; Li, Ming; Carpenter, Michael A.; Rathore, Anurag; Harris, Reuben S.; Harki, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7419-7430</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) was employed to discover APOBEC3G inhibitors, and multiple 2-furylquinolines, e.g. I, were found.  Dose-response assays with I from the HTS sample, as well as com. material, yielded similar confirmatory results.  Interestingly, freshly synthesized and DMSO-solubilized I was inactive.  Repeated screening of the DMSO aliquot of synthesized I revealed increasing APOBEC3G inhibitory activity with age, suggesting that I decomps. into an active inhibitor.  Lab. aging of I followed by anal. revealed that I undergoes oxidative decompn. in air, resulting from a [4+2] cycloaddn. between the furan of I and 1O2.  The resulting endoperoxide then undergoes addnl. transformations, highlighted by Baeyer-Villager rearrangements, to deliver lactam, carboxylic acid, and aldehyde products.  The endoperoxide also undergoes hydrolytic opening followed by further transformations to a bis-enone.  Eight structurally related analogs from HTS libraries were similarly reactive.  This study constitutes a cautionary tale to validate 2-furylquinolines for structure and stability prior to chem. optimization campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3HF7Xmelx8bVg90H21EOLACvtfcHk0lg0MMslXHzktQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVykt7%252FN&md5=3f7855d8a577e0bf3d9a082cad37ac9a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00930%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DM.%2BE.%26aulast%3DAbate-Pella%26aufirst%3DD.%26aulast%3DPerkins%26aufirst%3DA.%2BL.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DCarpenter%26aufirst%3DM.%2BA.%26aulast%3DRathore%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DR.%2BS.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26atitle%3DOxidative%2520reactivities%2520of%25202-furylquinolines%253A%2520ubiquitous%2520scaffolds%2520in%2520common%2520high-throughput%2520screening%2520libraries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7419%26epage%3D7430%26doi%3D10.1021%2Facs.jmedchem.5b00930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahal, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joswig-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span> <span> </span><span class="NLM_article-title">Comparative study of the affinity and metabolism of type I and type II binding quinoline carboxamide analogues by cytochrome P450 3A4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">280</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1021/jm201207h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201207h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=280-290&author=U.+P.+Dahalauthor=C.+Joswig-Jonesauthor=J.+P.+Jones&title=Comparative+study+of+the+affinity+and+metabolism+of+type+I+and+type+II+binding+quinoline+carboxamide+analogues+by+cytochrome+P450+3A4&doi=10.1021%2Fjm201207h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm201207h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201207h%26sid%3Dliteratum%253Aachs%26aulast%3DDahal%26aufirst%3DU.%2BP.%26aulast%3DJoswig-Jones%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DComparative%2520study%2520of%2520the%2520affinity%2520and%2520metabolism%2520of%2520type%2520I%2520and%2520type%2520II%2520binding%2520quinoline%2520carboxamide%2520analogues%2520by%2520cytochrome%2520P450%25203A4%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D55%26spage%3D280%26epage%3D290%26doi%3D10.1021%2Fjm201207h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spinks, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleghorn, L. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlamb, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frearson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span> <span> </span><span class="NLM_article-title">Investigation of trypanothione reductase as a drug target in Trypanosoma brucei</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">2060</span>– <span class="NLM_lpage">2069</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200900262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1002%2Fcmdc.200900262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=19924760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWrs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=2060-2069&author=D.+Spinksauthor=E.+J.+Shanksauthor=L.+A.+T.+Cleghornauthor=S.+McElroyauthor=D.+Jonesauthor=D.+Jamesauthor=A.+H.+Fairlambauthor=J.+A.+Frearsonauthor=P.+G.+Wyattauthor=I.+H.+Gilbert&title=Investigation+of+trypanothione+reductase+as+a+drug+target+in+Trypanosoma+brucei&doi=10.1002%2Fcmdc.200900262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of Trypanothione Reductase as a Drug Target in Trypanosoma brucei</span></div><div class="casAuthors">Spinks, Daniel; Shanks, Emma J.; Cleghorn, Laura A. T.; McElroy, Stuart; Jones, Deuan; James, Daniel; Fairlamb, Alan H.; Frearson, Julie A.; Wyatt, Paul G.; Gilbert, Ian H.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2060-2069</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">There is an urgent need for new drugs for the treatment of tropical parasitic diseases such as human African trypanosomiasis, which is caused by Trypanosoma brucei.  The enzyme trypanothione reductase (TryR) is a potential drug target within these organisms.  Herein we report the screening of a 62 000 compd. library against T. brucei TryR.  Further work was undertaken to optimize potency and selectivity of two novel-compd. series arising from the enzymic and whole parasite screens and mammalian cell counter-screens.  Both of these series, contg. either a quinoline or pyrimidinopyrazine scaffold, yielded low micromolar inhibitors of the enzyme and growth of the parasite.  The challenges of inhibiting TryR with drug-like mols. is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-k1JvjbQWpLVg90H21EOLACvtfcHk0lg0MMslXHzktQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWrs7%252FI&md5=444d2fed0ad1342ed8ee54c405549b4d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900262%26sid%3Dliteratum%253Aachs%26aulast%3DSpinks%26aufirst%3DD.%26aulast%3DShanks%26aufirst%3DE.%2BJ.%26aulast%3DCleghorn%26aufirst%3DL.%2BA.%2BT.%26aulast%3DMcElroy%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26aulast%3DFrearson%26aufirst%3DJ.%2BA.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26atitle%3DInvestigation%2520of%2520trypanothione%2520reductase%2520as%2520a%2520drug%2520target%2520in%2520Trypanosoma%2520brucei%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D2060%26epage%3D2069%26doi%3D10.1002%2Fcmdc.200900262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of 2-phenylquinoline-4-carboxamide derivatives as a new class of tubulin polymerization inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5939</span>– <span class="NLM_lpage">5951</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.bmc.2017.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=28988750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=5939-5951&author=L.+Zhuauthor=K.+Luoauthor=K.+Liauthor=Y.+Jinauthor=J.+Lin&title=Design%2C+synthesis+and+biological+evaluation+of+2-phenylquinoline-4-carboxamide+derivatives+as+a+new+class+of+tubulin+polymerization+inhibitors&doi=10.1016%2Fj.bmc.2017.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of 2-phenylquinoline-4-carboxamide derivatives as a new class of tubulin polymerization inhibitors</span></div><div class="casAuthors">Zhu, Li; Luo, Kaixiu; Li, Ke; Jin, Yi; Lin, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5939-5951</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of 2-phenylquinoline-4-carboxamide derivs. was synthesized, characterized and evaluated for its antiproliferative activity against five cancer cell lines, Hela, SK-OV-3, HCT116, A549 and MDA-MB-468, and a normal human fetal lung fibroblastic cell line, MRC-5.  Among them, compd. I displayed potent cytotoxic activity in vitro against SK-OV-3 and HCT116 cell lines with IC50 values of 0.5 and 0.2 μM, resp.  In general, the antiproliferative activity was correlated with the binding property of the colchicine binding site and inhibitory effect on tubulin polymn.  In addn., immunofluorescence and flow cytometry anal. revealed that selected compds. caused disruption of the mitotic spindle assembly and G2/M phase arrest of the cell cycle, which correlated with proliferation inhibitory activity.  Mol. docking anal. demonstrated the interaction of I at the colchicine binding site of tubulin.  These results indicate these compds. are promising inhibitors of tubulin polymn. for the potent treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu54Rs5g5vJrVg90H21EOLACvtfcHk0lgO0ZvL5IFNAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzF&md5=f6ae769bfd1f7670e16dd23abcd32a69</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DJ.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%25202-phenylquinoline-4-carboxamide%2520derivatives%2520as%2520a%2520new%2520class%2520of%2520tubulin%2520polymerization%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D5939%26epage%3D5951%26doi%3D10.1016%2Fj.bmc.2017.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span> <span> </span><span class="NLM_article-title">Novel mitochondria-targeted and fluorescent DNA alkylation agents with highly selective activity against cancer cells</span>. <i>Dyes Pigm.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">107610</span>, <span class="refDoi"> DOI: 10.1016/j.dyepig.2019.107610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.dyepig.2019.107610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFOjs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2019&pages=107610&author=X.+Chenauthor=W.+Pengauthor=S.+Huangauthor=C.+Yangauthor=M.+Huauthor=S.+Yangauthor=S.+Yangauthor=Y.+Xieauthor=H.+Chenauthor=N.+Leiauthor=Y.+Luoauthor=K.+Li&title=Novel+mitochondria-targeted+and+fluorescent+DNA+alkylation+agents+with+highly+selective+activity+against+cancer+cells&doi=10.1016%2Fj.dyepig.2019.107610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mitochondria-targeted and fluorescent DNA alkylation agents with highly selective activity against cancer cells</span></div><div class="casAuthors">Chen, Xiuli; Peng, Wentao; Huang, Shenzhen; Yang, Chao; Hu, Mingxing; Yang, Shuping; Yang, Shengyong; Xie, Yongmei; Chen, Hao; Lei, Ning; Luo, Yi; Li, Kun</div><div class="citationInfo"><span class="NLM_cas:title">Dyes and Pigments</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107610</span>CODEN:
                <span class="NLM_cas:coden">DYPIDX</span>;
        ISSN:<span class="NLM_cas:issn">0143-7208</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of novel nitrogen mustard-based fluorescent compds. were reported with good mitochondrial targeting ability and DNA alkylation activity.  The results indicated that these compds. can be used directly as reporting and imaging agents for gel electrophoresis and flow cytometry without the necessity of addnl. fluorescent tagging agents.  Notably, the compd. CXL92 displayed highly selective activity against lung cancer cell line A549 with an IC50 value of 0.32 μM compared to normal cell (LO2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm9LdinJB4qrVg90H21EOLACvtfcHk0lgO0ZvL5IFNAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFOjs7fP&md5=ab03d172c0f6041afb644f3cce45952f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.dyepig.2019.107610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dyepig.2019.107610%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLei%26aufirst%3DN.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DK.%26atitle%3DNovel%2520mitochondria-targeted%2520and%2520fluorescent%2520DNA%2520alkylation%2520agents%2520with%2520highly%2520selective%2520activity%2520against%2520cancer%2520cells%26jtitle%3DDyes%2520Pigm.%26date%3D2019%26volume%3D170%26spage%3D107610%26doi%3D10.1016%2Fj.dyepig.2019.107610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krzyzosiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigurdardottir, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolotti, A.</span></span> <span> </span><span class="NLM_article-title">Target-based discovery of an inhibitor of the regulatory phosphatase PPP1R15B</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1228.e19</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.06.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.cell.2018.06.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=30057111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2018&pages=1216-1228.e19&author=A.+Krzyzosiakauthor=A.+Sigurdardottirauthor=L.+Luhauthor=M.+Carraraauthor=I.+Dasauthor=K.+Schneiderauthor=A.+Bertolotti&title=Target-based+discovery+of+an+inhibitor+of+the+regulatory+phosphatase+PPP1R15B&doi=10.1016%2Fj.cell.2018.06.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Target-Based Discovery of an Inhibitor of the Regulatory Phosphatase PPP1R15B</span></div><div class="casAuthors">Krzyzosiak, Agnieszka; Sigurdardottir, Anna; Luh, Laura; Carrara, Marta; Das, Indrajit; Schneider, Kim; Bertolotti, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1216-1228.e19</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein phosphorylation is a prevalent and ubiquitous mechanism of regulation.  Kinases are popular drug targets, but identifying selective phosphatase inhibitors has been challenging.  Here, we used surface plasmon resonance to design a method to enable target-based discovery of selective serine/threonine phosphatase inhibitors.  The method targeted a regulatory subunit of protein phosphatase 1, PPP1R15B, a neg. regulator of proteostasis.  This yielded Raphin1, a selective inhibitor of R15B.  In cells, Raphin1 caused a rapid and transient accumulation of its phosphorylated substrate, resulting in a transient attenuation of protein synthesis.  In vitro, Raphin1 inhibits the recombinant R15B-PP1c holoenzyme, but not the closely related R15A-PP1c, by interfering with substrate recruitment.  Raphin1 was orally bioavailable, crossed the blood-brain barrier, and demonstrated efficacy in a mouse model of Huntington's disease.  This identifies R15B as a druggable target and provides a platform for target-based discovery of inhibitors of serine/threonine phosphatases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnL1IexJPGE7Vg90H21EOLACvtfcHk0lgO0ZvL5IFNAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsLbL&md5=09415366927bac424f14e9cce05d9385</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.06.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.06.030%26sid%3Dliteratum%253Aachs%26aulast%3DKrzyzosiak%26aufirst%3DA.%26aulast%3DSigurdardottir%26aufirst%3DA.%26aulast%3DLuh%26aufirst%3DL.%26aulast%3DCarrara%26aufirst%3DM.%26aulast%3DDas%26aufirst%3DI.%26aulast%3DSchneider%26aufirst%3DK.%26aulast%3DBertolotti%26aufirst%3DA.%26atitle%3DTarget-based%2520discovery%2520of%2520an%2520inhibitor%2520of%2520the%2520regulatory%2520phosphatase%2520PPP1R15B%26jtitle%3DCell%26date%3D2018%26volume%3D174%26spage%3D1216%26epage%3D1228.e19%26doi%3D10.1016%2Fj.cell.2018.06.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poddar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surolia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, B.</span></span> <span> </span><span class="NLM_article-title">Sulfonamide drugs binding to the colchicine site of tubulin: thermodynamic analysis of the drug– tubulin interactions by isothermal titration calorimetry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1021/jm0494974</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0494974" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkvVGltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=547-555&author=M.+Banerjeeauthor=A.+Poddarauthor=G.+Mitraauthor=A.+Suroliaauthor=T.+Owaauthor=B.+Bhattacharyya&title=Sulfonamide+drugs+binding+to+the+colchicine+site+of+tubulin%3A+thermodynamic+analysis+of+the+drug%E2%80%93+tubulin+interactions+by+isothermal+titration+calorimetry&doi=10.1021%2Fjm0494974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfonamide Drugs Binding to the Colchicine Site of Tubulin: Thermodynamic Analysis of the Drug-Tubulin Interactions by Isothermal Titration Calorimetry</span></div><div class="casAuthors">Banerjee, Mithu; Poddar, Asim; Mitra, Gopa; Surolia, Avadhesha; Owa, Takashi; Bhattacharyya, Bhabatarak</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">547-555</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of several sulfonamide drugs paved the way toward the synthesis of 6 (N-[2-[(4-hydroxyphenyl)amino]-3-pyridinyl]-4-methoxybenzenesulfonamide, E7010) and 7 (N-(3-fluoro-4-methoxyphenyl)pentafluorobenzenesulfonamide, T138067), both of which inhibit tubulin polymn. and are under clin. development.  A series of diarylsulfonamides contg. an indole scaffold was also found to have antimitotic properties, but their mode of interactions with tubulin has remained unidentified so far.  In this study, the authors demonstrate that these sulfonamide drugs bind to the colchicine site of tubulin in a reversible manner.  They quenched intrinsic tryptophan fluorescence of tubulin presumably due to drug-induced conformational changes in the protein, but were unable to modulate GTPase activity of tubulin in contrast to colchicine that enhances the same enzymic activity.  Further investigation using isothermal titrn. calorimetry (ITC) revealed that 5 (N-(5-chloro-7-indolyl)-4-methoxybenzenesulfonamide) afforded a large pos. value of heat capacity change (ΔCp = +264 cal mol-1 K-1) on binding to tubulin, suggesting a substantial conformational transition in the protein along with partial enthalpy-entropy compensation.  On the other hand, the 2-chloro regioisomer 2 gave a large neg. value of ΔCp (-589 cal mol-1 K-1) along with complete enthalpy-entropy compensation.  This thermodn. profile was thought to be attributable to a prominent contribution of van der Waals interaction and hydrogen bonding between specific groups in the drug-tubulin complex.  These results indicate that a mere alteration in the position of a single substituent chlorine on the indole scaffold has a great influence on the drug-tubulin binding thermodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFXpusdp7zOrVg90H21EOLACvtfcHk0liQaWEFkqik2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkvVGltA%253D%253D&md5=c248bd33dc0d2831494869fe97b1cb41</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm0494974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0494974%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DM.%26aulast%3DPoddar%26aufirst%3DA.%26aulast%3DMitra%26aufirst%3DG.%26aulast%3DSurolia%26aufirst%3DA.%26aulast%3DOwa%26aufirst%3DT.%26aulast%3DBhattacharyya%26aufirst%3DB.%26atitle%3DSulfonamide%2520drugs%2520binding%2520to%2520the%2520colchicine%2520site%2520of%2520tubulin%253A%2520thermodynamic%2520analysis%2520of%2520the%2520drug%25E2%2580%2593%2520tubulin%2520interactions%2520by%2520isothermal%2520titration%2520calorimetry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D547%26epage%3D555%26doi%3D10.1021%2Fjm0494974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schnapp, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoevels, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakker, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span> <span> </span><span class="NLM_article-title">Comparative analysis of binding kinetics and thermodynamics of dipeptidyl peptidase-4 inhibitors and their relationship to structure</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7466</span>– <span class="NLM_lpage">7477</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00475</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00475" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SisbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7466-7477&author=G.+Schnappauthor=T.+Kleinauthor=Y.+Hoevelsauthor=R.+A.+Bakkerauthor=H.+Nar&title=Comparative+analysis+of+binding+kinetics+and+thermodynamics+of+dipeptidyl+peptidase-4+inhibitors+and+their+relationship+to+structure&doi=10.1021%2Facs.jmedchem.6b00475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative Analysis of Binding Kinetics and Thermodynamics of Dipeptidyl Peptidase-4 Inhibitors and Their Relationship to Structure</span></div><div class="casAuthors">Schnapp, Gisela; Klein, Thomas; Hoevels, Yvette; Bakker, Remko A.; Nar, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7466-7477</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding kinetics and thermodn. of dipeptidyl peptidase (DPP)-4 inhibitors (gliptins) were investigated using surface plasmon resonance and isothermal titrn. calorimetry.  Binding of gliptins to DPP-4 is a rapid electrostatically driven process.  Off-rates were generally slow partly because of reversible covalent bond formation by some gliptins, and partly because of strong and extensive interactions.  Binding of all gliptins is enthalpy-dominated due to strong ionic interactions and strong solvent-shielded hydrogen bonds.  Using a congeneric series of mols. which represented the intermediates in the lead optimization program of linagliptin, the onset of slow binding kinetics and development of the thermodn. repertoire were analyzed in the context of incremental changes of the chem. structures.  All compds. rapidly assocd., and therefore the optimization of affinity and residence time is highly correlated.  The major contributor to the increasing free energy of binding was a strong increase of binding enthalpy, whereas entropic contributions remained low and const. despite significant addn. of lipophilicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7s_u2QnM5OrVg90H21EOLACvtfcHk0liQaWEFkqik2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SisbrE&md5=95398602908df300f76c24677b5af218</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00475%26sid%3Dliteratum%253Aachs%26aulast%3DSchnapp%26aufirst%3DG.%26aulast%3DKlein%26aufirst%3DT.%26aulast%3DHoevels%26aufirst%3DY.%26aulast%3DBakker%26aufirst%3DR.%2BA.%26aulast%3DNar%26aufirst%3DH.%26atitle%3DComparative%2520analysis%2520of%2520binding%2520kinetics%2520and%2520thermodynamics%2520of%2520dipeptidyl%2520peptidase-4%2520inhibitors%2520and%2520their%2520relationship%2520to%2520structure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7466%26epage%3D7477%26doi%3D10.1021%2Facs.jmedchem.6b00475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span> <span> </span><span class="NLM_article-title">Novel mitochondria-targeted, nitrogen mustard-based DNA alkylation agents with near infrared fluorescence emission</span>. <i>Talanta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">888</span>– <span class="NLM_lpage">893</span>, <span class="refDoi"> DOI: 10.1016/j.talanta.2016.08.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1016%2Fj.talanta.2016.08.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=27769499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2nsrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2016&pages=888-893&author=X.+Chenauthor=H.+Chenauthor=C.+Luauthor=C.+Yangauthor=X.+Yuauthor=K.+Liauthor=Y.+Xie&title=Novel+mitochondria-targeted%2C+nitrogen+mustard-based+DNA+alkylation+agents+with+near+infrared+fluorescence+emission&doi=10.1016%2Fj.talanta.2016.08.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mitochondria-targeted, nitrogen mustard-based DNA alkylation agents with near infrared fluorescence emission</span></div><div class="casAuthors">Chen, Xiuli; Chen, Hao; Lu, Chunyan; Yang, Chao; Yu, Xiaoqi; Li, Kun; Xie, Yongmei</div><div class="citationInfo"><span class="NLM_cas:title">Talanta</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">888-893</span>CODEN:
                <span class="NLM_cas:coden">TLNTA2</span>;
        ISSN:<span class="NLM_cas:issn">0039-9140</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this report, the design of two novel nitrogen mustard-based DNA crosslinking agents with fluorophores incorporated into the structure (TN-A(I) and TN-B (II)) is disclosed.  The results indicate that TN-A and TN-B can serve directly as both reporting and imaging agents for flow cytometry and gel electrophoresis without the necessity of another fluorescent tagging agents.  TN-A and TN-B both selectively locate in the mitochondria and exhibit good antitumor activity.  Notably, TN-A is the first DNA crosslinking agent with near IR fluorescence emission properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4XRlNbPJFqrVg90H21EOLACvtfcHk0liQaWEFkqik2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2nsrfL&md5=31a0a47f2ac33f53f08ec1a4a08ad2b7</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.talanta.2016.08.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.talanta.2016.08.051%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DY.%26atitle%3DNovel%2520mitochondria-targeted%252C%2520nitrogen%2520mustard-based%2520DNA%2520alkylation%2520agents%2520with%2520near%2520infrared%2520fluorescence%2520emission%26jtitle%3DTalanta%26date%3D2016%26volume%3D161%26spage%3D888%26epage%3D893%26doi%3D10.1016%2Fj.talanta.2016.08.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-Y.</span></span> <span> </span><span class="NLM_article-title">A preclinical evaluation of SKLB261, a multikinase inhibitor of EGFR/Src/VEGFR2, as a therapeutic agent against pancreatic cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-14-0485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1158%2F1535-7163.mct-14-0485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=25519702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlOjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=407-418&author=Y.+Panauthor=M.+Zhengauthor=L.+Zhongauthor=J.+Yangauthor=S.+Zhouauthor=Y.+Qinauthor=R.+Xiangauthor=Y.+Chenauthor=S.-Y.+Yang&title=A+preclinical+evaluation+of+SKLB261%2C+a+multikinase+inhibitor+of+EGFR%2FSrc%2FVEGFR2%2C+as+a+therapeutic+agent+against+pancreatic+cancer&doi=10.1158%2F1535-7163.mct-14-0485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A Preclinical Evaluation of SKLB261, a Multikinase Inhibitor of EGFR/Src/VEGFR2, as a Therapeutic Agent against Pancreatic Cancer</span></div><div class="casAuthors">Pan, Youli; Zheng, Mingwu; Zhong, Lei; Yang, Jiao; Zhou, Shu; Qin, Ya; Xiang, Rong; Chen, Yuzong; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">407-418</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The clin. prognosis of pancreatic cancer remains rather disappointing despite tremendous efforts in exploring medical treatments in the past two decades.  Development of more effective treatment strategies is still desperately needed to improve outcomes in patients with pancreatic cancer.  SKLB261 is a multikinase inhibitor obtained recently through a lead optimization.  In this investigation, we shall evaluate its anti-pancreatic cancer effects both in vitro and in vivo.  SKLB261 is a multikinase inhibitor potently inhibiting EGFR, Src, and VEGFR2 kinases.  It could significantly inhibit cell proliferation, migration, and invasion, and induce apoptosis in cellular assays of human pancreatic cancer cells that are sensitive or resistant to dasatinib and/or gemcitabine.  Western blot anal. showed that SKLB261 inhibited the activation of EGFR and Src kinases as well as their downstream signaling proteins, including FAK, ERK, and STAT3.  SKLB261 also showed potent antiangiogenic effects in transgenic zebrafish models.  In vivo, SKLB261 displayed more potent antitumor activities than dasatinib, gemcitabine, or erlotinib in pancreatic cancer xenografts, including BxPC-3, PANC-1, AsPC-1, and HPAC.  Furthermore, mice receiving SKLB261 therapy showed significant survival advantage compared with vehicle-treated and gemcitabine-treated groups in an exptl. metastasis model of pancreatic cancer.  These data, together with the good pharmacokinetic properties and low toxicity of this compd., provide a rationale for the ongoing clin. evaluation of SKLB261 in the treatment of pancreatic cancer.  Mol Cancer Ther; 14(2); 407-18. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOnyoK8YlU1LVg90H21EOLACvtfcHk0lh1l9oyFcEZpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlOjsbs%253D&md5=883fbc5c22d1a54b0a95ac2ff279a038</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-14-0485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-14-0485%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DQin%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.-Y.%26atitle%3DA%2520preclinical%2520evaluation%2520of%2520SKLB261%252C%2520a%2520multikinase%2520inhibitor%2520of%2520EGFR%252FSrc%252FVEGFR2%252C%2520as%2520a%2520therapeutic%2520agent%2520against%2520pancreatic%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3D407%26epage%3D418%26doi%3D10.1158%2F1535-7163.mct-14-0485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-Y.</span></span> <span> </span><span class="NLM_article-title">Preclinical evaluation of a novel orally available SRC/Raf/VEGFR2 inhibitor, SKLB646, in the treatment of triple-negative breast cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">378</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-15-0501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=10.1158%2F1535-7163.mct-15-0501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=26721945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslCmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=366-378&author=M.-W.+Zhengauthor=C.-H.+Zhangauthor=K.+Chenauthor=M.+Huangauthor=Y.-P.+Liauthor=W.-T.+Linauthor=R.-J.+Zhangauthor=L.+Zhongauthor=R.+Xiangauthor=L.-L.+Liauthor=X.-Y.+Liuauthor=Y.-Q.+Weiauthor=S.-Y.+Yang&title=Preclinical+evaluation+of+a+novel+orally+available+SRC%2FRaf%2FVEGFR2+inhibitor%2C+SKLB646%2C+in+the+treatment+of+triple-negative+breast+cancer&doi=10.1158%2F1535-7163.mct-15-0501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Evaluation of a Novel Orally Available SRC/Raf/VEGFR2 Inhibitor, SKLB646, in the Treatment of Triple-Negative Breast Cancer</span></div><div class="casAuthors">Zheng, Ming-Wu; Zhang, Chun-Hui; Chen, Kai; Huang, Mei; Li, Ya-Ping; Lin, Wan-Ting; Zhang, Rong-Jie; Zhong, Lei; Xiang, Rong; Li, Lin-Li; Liu, Xin-Yu; Wei, Yu-Quan; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">366-378</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is the most aggressive and deadly breast cancer subtype.  To date, chemotherapy is the only systemic therapy and prognosis remains poor.  Herein, we report the preclin. evaluation of SKLB646 in the treatment of TNBC; SKLB646 is a novel multiple kinase inhibitor developed by us recently.  This compd. potently inhibited SRC and VEGFR2 with IC50 values of 0.002 μmol/L and 0.012 μmol/L, resp.  It also considerably inhibited B-Raf and C-Raf with IC50 values of 0.022 and 0.019 μmol/L, resp.  It exhibited significant antiproliferation and antiviability activities against TNBC cell lines.  Studies of mechanism of action indicated that SKLB646 inhibited the activation of SRC signaling and blocked the MAPK signaling through inhibiting the Raf kinases.  Interestingly, SKLB646 dose dependently downregulated the expression of Fra1, a transcriptional factor that plays a crit. role in the epithelial-to-mesenchymal transition.  In addn., SKLB646 could inhibit HUVEC proliferation, migration, and invasion.  It effectively blocked the formation of intersegmental vessels in zebrafish embryos and displayed considerable antiangiogenic effects in the tumor-induced neovascularization zebrafish model.  In TNBC xenograft models, SKLB646 suppressed the tumor growth in a dose-dependent manner.  Moreover, SKLB646 could remarkably inhibit TNBC cell migration and invasion in vitro.  Furthermore, in an exptl. lung metastasis model, the overall survival time of groups treated with SKLB646 was much longer compared with the control-, dasatinib-, and paclitaxel-treated groups.  In a preliminary pharmacokinetic study, SKLB646 showed good pharmacokinetic properties.  Taken together, the preclin. data show that SKLB646 could be a promising lead compd. for the treatment of TNBC.  Mol Cancer Ther; 15(3); 366-78. 2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodSEWSFArBprVg90H21EOLACvtfcHk0lh1l9oyFcEZpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslCmsL8%253D&md5=5503431d68d044995a94faa8948f27ba</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-15-0501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-15-0501%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DM.-W.%26aulast%3DZhang%26aufirst%3DC.-H.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DY.-P.%26aulast%3DLin%26aufirst%3DW.-T.%26aulast%3DZhang%26aufirst%3DR.-J.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DL.-L.%26aulast%3DLiu%26aufirst%3DX.-Y.%26aulast%3DWei%26aufirst%3DY.-Q.%26aulast%3DYang%26aufirst%3DS.-Y.%26atitle%3DPreclinical%2520evaluation%2520of%2520a%2520novel%2520orally%2520available%2520SRC%252FRaf%252FVEGFR2%2520inhibitor%252C%2520SKLB646%252C%2520in%2520the%2520treatment%2520of%2520triple-negative%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D366%26epage%3D378%26doi%3D10.1158%2F1535-7163.mct-15-0501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MF6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MF6','PDB','5MF6'); return false;">PDB: 5MF6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Y2F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Y2F','PDB','5Y2F'); return false;">PDB: 5Y2F</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i114"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01183">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10780"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01183?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01183</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Activity data of 40 candidate compounds; predicted binding mode of <b>Hit20</b> with SIRT6; dose-dependent activities of SIRT6 demyristoylation of <b>12q</b>; predicted binding mode of <b>12q</b> with SIRT6; other experimental data of compound <b>12q</b><i>in vitro</i> and <i>in vivo</i>; and <sup>1</sup>H NMR spectra, <sup>13</sup>C NMR spectra, HRMS data, and HPLC traces for key target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf">PDF</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_002.csv">CSV</a>)</p></li><li><p class="inline">Binding model of compound <b>12q</b> with SIRT6 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_001.pdf">jm0c01183_si_001.pdf (9.54 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_002.csv">jm0c01183_si_002.csv (7.28 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01183/suppl_file/jm0c01183_si_003.pdb">jm0c01183_si_003.pdb (244.49 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01183&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01183%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-18%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01183" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ab76e7c4a3d4e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
