
<html lang="en"     class="pb-page"  data-request-id="2231f000-8a2a-4ca5-aa10-4758f2c008fe"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2013.56.issue-17;article:article:10.1021/jm400798j;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Selectivity Data: Assessment, Predictions, Concordance, and Implications" /></meta><meta name="dc.Creator" content="Cen  Gao" /></meta><meta name="dc.Creator" content="Suntara  Cahya" /></meta><meta name="dc.Creator" content="Christos A.  Nicolaou" /></meta><meta name="dc.Creator" content="Jibo  Wang" /></meta><meta name="dc.Creator" content="Ian A.  Watson" /></meta><meta name="dc.Creator" content="David J.  Cummins" /></meta><meta name="dc.Creator" content="Philip W.  Iversen" /></meta><meta name="dc.Creator" content="Michal  Vieth" /></meta><meta name="dc.Description" content="Could high-quality in silico predictions in drug discovery eventually replace part or most of experimental testing? To evaluate the agreement of selectivity data from different experimental or pred..." /></meta><meta name="Description" content="Could high-quality in silico predictions in drug discovery eventually replace part or most of experimental testing? To evaluate the agreement of selectivity data from different experimental or pred..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 30, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm400798j" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm400798j" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm400798j" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm400798j" /></link>
        
    
    

<title>Selectivity Data: Assessment, Predictions, Concordance, and Implications | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm400798j" /></meta><meta property="og:title" content="Selectivity Data: Assessment, Predictions, Concordance, and Implications" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0011.jpeg" /></meta><meta property="og:description" content="Could high-quality in silico predictions in drug discovery eventually replace part or most of experimental testing? To evaluate the agreement of selectivity data from different experimental or predictive sources, we introduce the new metric concordance minimum significant ratio (cMSR). Empowered by cMSR, we find the overall level of agreement between predicted and experimental data to be comparable to that found between experimental results from different sources. However, for molecules that are either highly selective or potent, the concordance between different experimental sources is significantly higher than the concordance between experimental and predicted values. We also show that computational models built from one data set are less predictive for other data sources and highlight the importance of bias correction for assessing selectivity data. Finally, we show that small-molecule target space relationships derived from different data sources and predictive models share overall similarity but can significantly differ in details." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm400798j"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm400798j">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm400798j&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm400798j&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm400798j&amp;href=/doi/10.1021/jm400798j" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 6991-7002</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm4008642" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm400802b" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Selectivity Data: Assessment, Predictions, Concordance, and Implications</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cen++Gao">Cen Gao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Suntara++Cahya">Suntara Cahya</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christos+A.++Nicolaou">Christos A. Nicolaou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jibo++Wang">Jibo Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ian+A.++Watson">Ian A. Watson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Cummins">David J. Cummins</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Philip+W.++Iversen">Philip W. Iversen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michal++Vieth">Michal Vieth</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="AFF-d2473e84-autogenerated"><span class="aff-text"><sup>†</sup>Discovery Chemistry, <sup>‡</sup>Discovery Statistics, <sup>§</sup>Advanced Analytics, Lilly Research Laboratories, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#a9dfc0ccddc1f6c4c0cac1c8c5e9c5c0c5c5d087cac6c4"><span class="__cf_email__" data-cfemail="42342b27362a1d2f2b212a232e022e2b2e2e3b6c212d2f">[email protected]</span></a>. Tel: 317-277-3959.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm400798j&amp;href=/doi/10.1021%2Fjm400798j" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 6991–7002</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 12, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 May 2013</li><li><span class="item_label"><b>Published</b> online</span>30 August 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 September 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm400798j" title="DOI URL">https://doi.org/10.1021/jm400798j</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6991%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCen%2BGao%252C%2BSuntara%2BCahya%252C%2BChristos%2BA.%2BNicolaou%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D17%26contentID%3Djm400798j%26title%3DSelectivity%2BData%253A%2BAssessment%252C%2BPredictions%252C%2BConcordance%252C%2Band%2BImplications%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7002%26publicationDate%3DSeptember%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm400798j"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1564</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm400798j" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Selectivity Data: Assessment, Predictions, Concordance, and Implications&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Cen&quot;,&quot;last_name&quot;:&quot;Gao&quot;},{&quot;first_name&quot;:&quot;Suntara&quot;,&quot;last_name&quot;:&quot;Cahya&quot;},{&quot;first_name&quot;:&quot;Christos&quot;,&quot;last_name&quot;:&quot;A. Nicolaou&quot;},{&quot;first_name&quot;:&quot;Jibo&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;A. Watson&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Cummins&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;W. Iversen&quot;},{&quot;first_name&quot;:&quot;Michal&quot;,&quot;last_name&quot;:&quot;Vieth&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;6991-7002&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm400798j&quot;},&quot;abstract&quot;:&quot;Could high-quality in silico predictions in drug discovery eventually replace part or most of experimental testing? To evaluate the agreement of selectivity data from different experimental or predictive sources, we introduce the new metric concordance minimum significant ratio (cMSR). Empowered by cMSR, we find the overall level of agreement between predicted and experimental data to be comparable to that found between experimental results from different sources. However, for molecules that are either highly selective or potent, the concordance between different experimental sources is significantly higher than the concordance between experimental and predicted values. We also show that computational models built from one data set are less predictive for other data sources and highlight the importance of bias correction for assessing selectivity data. Finally, we show that small-molecule target space relationships derived from different data sources and predictive models share overall similarity but can sign&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400798j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400798j" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400798j&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400798j" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400798j&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400798j" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm400798j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400798j&amp;href=/doi/10.1021/jm400798j" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm400798j" /></input><a href="/doi/pdf/10.1021/jm400798j" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm400798j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm400798j%26sid%3Dliteratum%253Aachs%26pmid%3D23937569%26genre%3Darticle%26aulast%3DGao%26date%3D2013%26atitle%3DSelectivity%2BData%253A%2BAssessment%252C%2BPredictions%252C%2BConcordance%252C%2Band%2BImplications%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D17%26spage%3D6991%26epage%3D7002%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=292031" title="Computational modeling">Computational modeling</a>,</li><li><a href="/action/doSearch?ConceptID=291466" title="Theoretical and computational chemistry">Theoretical and computational chemistry</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jmcmar.2013.56.issue-17/production/jmcmar.2013.56.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Could high-quality in silico predictions in drug discovery eventually replace part or most of experimental testing? To evaluate the agreement of selectivity data from different experimental or predictive sources, we introduce the new metric concordance minimum significant ratio (cMSR). Empowered by cMSR, we find the overall level of agreement between predicted and experimental data to be comparable to that found between experimental results from different sources. However, for molecules that are either highly selective or potent, the concordance between different experimental sources is significantly higher than the concordance between experimental and predicted values. We also show that computational models built from one data set are less predictive for other data sources and highlight the importance of bias correction for assessing selectivity data. Finally, we show that small-molecule target space relationships derived from different data sources and predictive models share overall similarity but can significantly differ in details.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">1 Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31826" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31826" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Understanding compound selectivity is one of the most critical components in all stages of targeted and phenotypic drug-discovery decision making. Early in the process, during target validation, a potent, selective chemical tool compound is often deemed critical to confirm the target’s role in the disease model.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In phenotypic discovery, multiple efforts have been focused on understanding the target and mechanism of action with both experimental<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and predictive profiling methods.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> In the lead optimization stage, animal toxicity data on optimized compounds is often linked to off-target activity,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> which can be identified from profiling data. In some instances, an understanding of adverse effects in humans is aided by the utilization of broad selectivity profiling, for example, using CEREP’s Bioprint approach.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Consequently, the informatics and computational communities, which historically focused on virtual screening aimed at single target activity prediction, have increasingly started to devote attention to polypharmacology prediction.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> One critical aspect of predicting and quantifying polypharmacology is the ability to understand the quality and applicability of predictions in various stages of decision making. In recent years, predictive modelers have been striving to improve the quality of in silico predictions.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Of note is the elegant Profile-QSAR method<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> that, in principle, can extend selectivity and activity prediction to a set of unknown but related targets. These approaches have been evaluated using a different set of quantitative metrics borrowed from the field of predictive modeling on single end-point problems and are not necessarily directly applicable to selectivity predictions.</div><div class="NLM_p">All machine learning and predictive modeling methods are fundamentally data driven, and as such the quality of the model produced, be it a single-target or a selectivity model, is in part dependent on the quality of the data used to develop it. Although data from multiple sources, such as different laboratories, may be combined to increase the size of the training and test data sets and thereby improve QSAR models, care must be taken to evaluate the concordance of different data sources.</div><div class="NLM_p last">In our previous work, we compared and contrasted kinase-selectivity data on the overlapping sets of compounds generated from four independent profiling sources.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> We found a relatively low level of concordance between different profiling results that, in turn, affected the general kinase and inhibitor space conclusions drawn from the data. Here, we extend this work to evaluate the concordance of computational models and experimental data, and we compare it to the concordance between experimental results from different sources. To quantify better the concordance in terms of single experiment variability, we introduce a new statistical metric, concordance minimum significant ratio (cMSR), inspired by a similar metric used in assay validation (<a class="ref internalNav" href="#sec2" aria-label="Theory and Methods">Theory and Methods</a>, section <a class="ref internalNav" href="#sec2_2" aria-label="2.2">2.2</a>). We illustrate the use of cMSR to evaluate different sources of experimental data as well as to compare experimental and predicted values on the same assay panel. We show the overall concordance between predicted and experimental data from the same source to be comparable to the concordance level between experimental results from different sources. In contrast, predictive models carry a lower level of agreement if used to predict experimental outcomes from other sources. We also note that the presence of bias (e.g., potency shifts between experimental data sources), which in our case varies for each target, needs to be accounted for. Although we provide a way to address bias correction, we realize that this adjustment might not be universally adequate and needs to be considered on a case-by-case basis. Finally, the identification of a compound’s primary target (i.e., the target where compound is most potent) is of interest in the analysis of phenotypic data. We found that two experimental sources have about a 50% overlap when the same primary targets are identified for compounds. In contrast to this, computational models have only a 20% agreement when compared with experimental results. The implications of our results span multiple distinct areas of drug discovery, including the comparison of assays from different sources, the utilization of computational models for selectivity assessment, and the limitations of predictive models built from literature data.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">2 Theory and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24478" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24478" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3">2.1 Background and Limitation of Using <i>R</i><sup>2</sup></h3><div class="NLM_p last">The concordance of different data sets and the performance of predictive models is typically measured using statistical metrics such as Pearson’s <i>R</i><sup>2</sup>, which measures the degree of linear association between two values (e.g., predicted and observed). To capture better the predictive power of a model, the cross-validated <i>R</i><sup>2</sup> is used, often denoted as <i>Q</i><sup>2</sup>. However, correlation-type metrics like these are affected by the range of the data, where a larger range in the data can artificially inflate the value of <i>R</i><sup>2</sup>. <a class="ref internalNav" href="#notes-1" aria-label="Figure S1">Figure S1</a> illustrates this effect, where two random variates were generated to have a correlation of 0.71 and a subset selected (at random) with a smaller range gives a correlation of 0.56. A metric that directly compares the accuracy of the prediction in the relevant units of measurement is more preferable to quantify both data the concordance and model performance. The root-mean squared error (RMSE) is a useful beginning toward that end.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.2 Concordance MSR (cMSR)</h3><div class="NLM_p">A common way to characterize the variability of assays during development is by using the MSR statistic, where MSR stands for minimum significant ratio.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> MSR is the smallest ratio of potencies between two compounds that is statistically significant. For example, the results of two different compounds tested in an assay with an MSR value of 3 would need to differ by more than 3-fold for the compounds to be deemed significantly different (at the chosen confidence level, alpha is usually 0.05).</div><div class="NLM_p">In this study, we are adapting the concept of MSR to assess the concordance between two sets of values originating from experimental measurements or in silico predictions. This new metric may be applied to measure the concordance between the screening results on the same target coming from different laboratories or companies or the concordance between experimental values and their corresponding predicted values. We refer to this statistic as the cMSR, where c is used to denote concordance. The formula to compute cMSR is given as follows<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/jm-2013-00798j_m001.gif" alt="" /></img></span>where ave = (∑<sub class="stack"><i>i</i>=1</sub><sup class="stack"><i>n</i></sup>log(<i>r</i><sub><i>i</i></sub>)/<i>n</i>)<sup>1/2</sup>, <i>r</i><sub><i>i</i></sub> is the ratio of two values (in concentration units, e.g., IC<sub>50</sub>) associated with the <i>i</i>th compound, sd = (∑<sub class="stack"><i>i</i>=1</sub><sup class="stack"><i>n</i></sup>(log(<i>r</i><sub><i>i</i></sub>) – ave)<sup>2</sup>/(<i>n</i> – 1))<sup>1/2</sup>, and the multiplier <i>k</i> = (1.645 + 0.325) <i>e</i><sup>–4.134(ave/sd)</sup>. The constant <i>k</i> is tuned for the 0.05 alpha level.</div><div class="NLM_p">It is noted that ave and sd are the sample average and standard deviation, respectively, of the log ratios of the two set of values of interest (in concentration units). They represent the average log difference between the two sets of values (i.e., bias) and the corresponding standard deviation around this bias (i.e., variability). It is assumed that the concentration values follow a typical log-normal distribution. If the results are already expressed on the logarithmic scale (e.g., −log<sub>10</sub>(IC<sub>50</sub>), often denoted as pIC<sub>50</sub>, p<i>K</i><sub>i</sub>, etc.), then ave = (∑<sub class="stack"><i>i</i>=1</sub><sup class="stack"><i>n</i></sup> ΔpIC<sub>50<i>i</i></sub>/<i>n</i>)<sup>1/2</sup> and sd = (∑<sub class="stack"><i>i</i>=1</sub><sup class="stack"><i>n</i></sup> (ΔpIC<sub>50<i>i</i></sub> – ave)<sup>2</sup>/(<i>n</i> – 1))<sup>1/2</sup>. It is noted that although related the cMSR is not equivalent to the root-mean standard error (RMSE) statistic. Further discussion regarding the difference between cMSR and RMSE can be found in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div><div class="NLM_p">As an extension to the MSR calculation for measuring the variability of a single assay, the cMSR measures both variability and bias between two sources of data (lab 1 vs lab 2, predicted vs experimental). In effect, the cMSR quantifies the magnitude of the variability and also penalizes any bias away from the ideal 1:1. When measuring the performance of individual assays with the MSR statistic, the bias does not come into the picture because we are quantifying the reproducibility of the assays within themselves. However, when we are to compare values between two different sources of data (different laboratories, predicted vs experimental, etc.), the bias becomes an integral part of the concordance measure in addition to variability. Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> illustrates how cMSR and <i>R</i><sup>2</sup> account for three different scenarios of bias between the prediction and the corresponding experimentally observed values: no bias, constant bias, and nonconstant bias. As shown, cMSR accounts for the potential bias between the data pair, unlike the ubiquitous <i>R</i><sup>2</sup> statistic. Additionally, the cMSR has a straightforward, intuitive interpretation: its value defines the degree of the fold difference where the majority (usually 95%) of the data points (e.g., compound activities) are expected to reside. As a comparison, Pearson’s <i>R</i><sup>2</sup> is a proportion of variability that is explained by the model (i.e., variability of model/(variability of model + error)); thus, 1 – <i>R</i><sup>2</sup> is the proportion of variability that is not explained by the model (i.e., the error as a proportion of the total variability). In contrast to <i>R</i><sup>2</sup>, cMSR is a direct measure of the errors expressed as fold difference to the expected range of the majority of the error. One can also contrast cMSR values to the MSR of the corresponding assays. For example, if an assay has an MSR of 3 and the cMSR of a predictive model built from the assay is 25, then it is easy to conclude that there is a lot more to be desired from the model. However, when the assay’s MSR is 5, a cMSR of 7 suggests a highly robust and potentially useful model. It is noted that per the definition of cMSR, it can be decomposed as cMSR = 10<sup>ave</sup> × 10<sup><i>k</i>sd</sup>, where the first part represents the mean fold difference (i.e., gross bias) between the data sources and the second part represents the variability component. It is often very informative to express the cMSR as a multiplication of the mean bias fold difference and the variability fold difference, especially when there is a significant gross bias between data sources.</div><div class="NLM_p">Although we focus most of our discussions in this Article on the basis of the cMSR metrics, we also provide <i>R</i><sup>2</sup> values where appropriate for readers preferring the latter metric as reference.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. cMSR and <i>R</i><sup>2</sup> values in the presence of varying bias between a set of predicted and observed values. The green line is the identity line, whereas the red line represents the best fitting line. Where there is (a) no bias, (b) constant bias (i.e., the prediction is consistently under or over predicted across the range of predicted values), and (c) nonconstant bias (i.e., an overestimation at lower end of prediction values and an under estimation at the higher end). In all cases, the <i>R</i><sup>2</sup> value is consistent at around 0.9. However, from a concordance standpoint, panel a is the most desirable outcome. In contrast to <i>R</i><sup>2</sup>, the cMSR reflects the potential bias in the data more accurately.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.3 Bias Correction</h3><div class="NLM_p last">The importance of bias between sources becomes even more critical when comparing and analyzing selectivity data because, in this case, the assays might have different baselines affecting the compound potencies. Having historical data for the compounds tested in both sources could help to estimate the systematic bias between assays, and in some cases this correction might hold well for future data. In general, however, extra caution should be taken when performing normalization between sources of data. Often, the availability of common results is very limited if any; thus, normalization should be done on a case-by-case basis, leveraging all available knowledge about the data. Is the bias of constant shift type or nonconstant? If it is nonconstant, then is it of linear or nonlinear type? If it is nonlinear, then is it due to potential differences in the dynamic ranges of the assays? These are some examples of questions that we need to answer before we can perform normalization satisfactorily. This issue is outside the scope of this work and therefore only a simple gross bias correction has been performed on the individual assay level to align the data better. In practice, different types of bias correction should be applied depending on what the individual assay calls for.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.4 Experimental Design</h3><div class="NLM_p">Our experiments used two sets of kinase-assay data collated previously:<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> an internal Lilly data set and data published by Metz et al.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The Lilly data set contains 2871 compounds across 100 kinases and 233 044 kinase-inhibitor pairs. The Metz data set contains 1473 compounds across 172 kinases and 106 653 kinase-inhibitor pairs. These two data sets are referred to as Exp(LLY) and Exp(Metz) in the remaining text. All data was converted to the negative log<sub>10</sub> of the IC<sub>50</sub> values (pIC<sub>50 </sub>= 6 −log<sub>10</sub>(IC<sub>50</sub>[μM]). <a class="ref internalNav" href="#notes-1" aria-label="Figure S2a">Figure S2a</a> provides a statistical summary of the two data sets. We computed the means of pIC<sub>50</sub> for each experimental source and target and noted significant target-dependent differences between the Lilly and Metz sources for training set data (<a class="ref internalNav" href="#notes-1" aria-label="Table S1">Table S1</a>). These biases, if not corrected, would lead to large discrepancies in comparing the results between sources (e.g., Lilly vs Metz data, see <a class="ref internalNav" href="#notes-1" aria-label="Table S2">Table S2</a>). In our experiments, we use common compounds between Lilly and Metz as a test set. As a result, the training sets for both sources do not have any common compounds. Thus, we adjust for these bias differences using a simple target-dependent mean correction for Metz experimental data (see <a class="ref internalNav" href="#notes-1" aria-label="Table S1">Table S1</a>). Although we realize that this is an oversimplified way of correcting source bias, in our case the training set (aka “historical”) biases are correlated (<i>R</i><sup>2</sup> = 0.51, last two columns in <a class="ref internalNav" href="#notes-1" aria-label="Table S1">Table S1</a>) and inform well on the test set biases (aka “future data”). The overlapping test data between the two sets consists of 32 compounds across 69 kinase targets, with a total of 1466 kinase-inhibitor pairs. Raw data for these 1466 inhibitor-target pairs is given in <a class="ref internalNav" href="#notes-1" aria-label="Table S5">Table S5</a>. For this set of overlapping data, we report data concordance using the cMSR metric as well as other metrics used in our previous communication: <i>R</i><sup>2</sup>, mean fold difference, and percentage of data within 3-fold.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p">For each of the 69 targets with overlapping data in both experimental sources, a predictive model was built using the SVMFP approach. As described in our previous communications,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> we used support vector machines (SVM)<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> to model continuous pIC<sub>50</sub> values, which were scaled to the −1 to 1 range for computational convenience. Support vector machine learning is a supervised learning method used for classification and regression.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> SVM classification models function by simultaneously minimizing the classification error while maximizing the geometric margin between classes. SVM regression models apply analogous concepts to a “tube” region around the observed values. We utilized the svm_lite package<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and a Tanimoto kernel function with circular fingerprints to a maximum radius of three bonds (ECC3 fingerprint). Atom typing was a function of four atomic attributes: hydrogen count, π electrons, aromaticity, and the number of heavy atom connections. To estimate the performance of the models, we performed 10 random 50/50 train/test cross-validation runs. As pointed out in our previous work,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> this and other training set cross validations tend not to be predictive of the test set performance for most metrics, including <i>R</i><sup>2</sup> (per-target cross-validation statistics are reported in <a class="ref internalNav" href="#notes-1" aria-label="Figure S3">Figure S3</a> and compared to the test set results). Interestingly, in contrast to <i>R</i><sup>2</sup>, the cross-validation cMSR metric appears to be concordant with the test set cMSR, with more than 50% of the cross-validated models with cMSR < 10 showing test set cMSR values <10.</div><div class="NLM_p">The 32 compounds common to both data sets were excluded from the training sets and served as our test set. Some of the 32 compounds are structurally related to the training-set molecule but most are quite different from the compounds in the training set (see <a class="ref internalNav" href="#notes-1" aria-label="Figure S2b">Figure S2b</a>, with a mean distance of 0.23 from the training sets). Later, we refer to the predicted results of these 32 compounds as Pred(LLY) and Pred(Metz), respectively. In all, the Lilly training set contains 2839 compounds and 160 859 kinase-inhibitor pairs, whereas the Metz training set contains 1400 compounds and 44 927 kinase-inhibitor pairs.</div><div class="NLM_p">Similar to the two experimental data sets, the concordance between the predicted activities and experimental data was assessed for all 1466 collated kinase-inhibitor pairs common between the Lilly and Metz data sets. It is worth noting that certain compound activities contain qualifiers (i.e., experimental data that are reported to be below or above the threshold values). The Lilly data set has an activity range between 1 nM to 20 μM. Activity outside of this range is reported with the qualifiers <1 nM or >20 μM. To assess the concordance of the activity levels better, we also looked into collated data without qualifiers (i.e., nonqualified results) separately. Out of the 1466 kinase-inhibitor pairs, 452 fall into this category. Finally, in addition to the collated data, we computed cMSR on a per-compound basis (32 compound bioprint group with up to 69 data points in each group) and a per-target basis (69 target bioprint group with up to 32 data points in each group).</div><div class="NLM_p last">Target-based bioprint clustering was performed using the Wards method<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> in R.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Sixty eight targets with more than 10 compound data points were used to generate the continuous Tanimoto distance matrix. The clustering result was laid out using the equal angle algorithm implemented internally and was graphically displayed using Cytoscape<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and Spotfire.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">3 Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">3.1 Concordance of Collated Target-Compound Data</h3><div class="NLM_p">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> provides the pairwise concordance metrics between the experimental and predicted data across the Lilly and Metz sources for 1466 kinase-inhibitor pairs. Scatter plots with activity distributions are shown in <a class="ref internalNav" href="#notes-1" aria-label="Figures S3 and S4">Figures S3 and S4</a>. cMSR between Exp(LLY) and Exp(Metz) is 12.6 for all data and increases to 23.3 (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) for nonqualified data. Both cMSR values indicate a relatively low overall concordance when compared to biological assays developed internally at Lilly, which typically have an MSR of around 3. This low concordance is consistent with our previously reported finding<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> obtained with traditional metrics (<i>R</i><sup>2</sup> and the percent of results within 3-fold). In addition, the mean fold difference shows an overall systematic bias between the two data sets. Without bias correction, Exp(Metz) is, on average, 3.5-fold (0.54 in log units) more potent than Exp(LLY) across all kinase-inhibitor pairs. This bias partially contributes to the large value of cMSR. <i>R</i><sup>2</sup> has a moderate value of 0.58 for Exp(LLY) and Exp(Metz), but it needs to be used with caution because the Lilly pIC<sub>50</sub> distribution deviates from normality (see the activity distribution in <a class="ref internalNav" href="#notes-1" aria-label="Figure S4a,b">Figure S4a,b</a>).</div><div class="NLM_p">The SVMFP-based predictive models built from the experimental data (models built from the Lilly training data compared to the Lilly experimental data and models built from the Metz data compared to the Metz experimental data) show a comparable cMSR (16.7 or 16.4), mean fold difference (−1.0 or 1.3), and percentage of results within 3-fold (68 or 62). Although the cMSR of these predictions is relatively high compared to the target assay MSR of 3, it is only slightly higher than the variability seen between Lilly and Metz experimental data with the corresponding cMSR of 12.6. This finding indicates that the overall level of agreement between predicted and experimental data is comparable to that found between the experimental results from different sources. However, the prediction models used on another source data have higher values in all metrics (i.e., predicting Metz experimental data using a model built on Lilly data and vice versa), with cMSR of 18.8 and 26.1, respectively.</div><div class="NLM_p">Note that the SVMFP methodology is just one of many ways to build predictive models. Although we use it in this work to exemplify the performance of predictive models, some generalizations might be specific to the method itself or the data sets we utilized. We also examined the performance of a random forest (RF) model with molecular descriptors and summarized the results in <a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>. On the basis of the results and our past experience,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> the SVMFP method is either comparable to or outperforms other models we have examined.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Concordance of All 1466 Kinase-Inhibitor Pairs between Different Sources Measured by Four Metrics<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">source 1</th><th class="colsep0 rowsep0" align="center">source 2</th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sup>2</sup>, all data</th><th class="colsep0 rowsep0" align="center" char=".">mean fold difference, all data<a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">cMSR, all data</th><th class="colsep0 rowsep0" align="center">percent within 3-fold, all data</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(LLY)</td><td class="colsep0 rowsep0" align="left">Exp(Metz)</td><td class="colsep0 rowsep0" align="char" char=".">0.58<a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">1.0<a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">12.6<a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">73<a class="ref internalNav" href="#t1fn3" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(LLY)</td><td class="colsep0 rowsep0" align="left">Pred(LLY)</td><td class="colsep0 rowsep0" align="char" char=".">0.46</td><td class="colsep0 rowsep0" align="char" char=".">–1.0</td><td class="colsep0 rowsep0" align="char" char=".">16.7</td><td class="colsep0 rowsep0" align="center">68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(Metz)</td><td class="colsep0 rowsep0" align="left">Pred(Metz)</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">16.4</td><td class="colsep0 rowsep0" align="center">62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(Metz)</td><td class="colsep0 rowsep0" align="left">Pred(LLY)</td><td class="colsep0 rowsep0" align="char" char=".">0.28</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">18.8</td><td class="colsep0 rowsep0" align="center">65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(LLY)</td><td class="colsep0 rowsep0" align="left">Pred(Metz)</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td><td class="colsep0 rowsep0" align="char" char=".">–1.3</td><td class="colsep0 rowsep0" align="char" char=".">26.1</td><td class="colsep0 rowsep0" align="center">51</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">Only numerical values are used in the comparisons after prefixes are eliminated.</p></div><div class="footnote" id="t1fn1"><sup>b</sup><p class="last">Mean fold difference was computed from a common data comparison. A negative sign indicates that source 1 has lower values than source 2.</p></div><div class="footnote" id="t1fn2"><sup>c</sup><p class="last">The numbers correspond to Lilly-Metz bias-corrected measured results from the Metz publication (values from <a class="ref internalNav" href="#notes-1" aria-label="Table S5">Table S5</a>). See <a class="ref internalNav" href="#notes-1" aria-label="Table S2">Table S2</a> for bias-uncorrected results.</p></div><div class="footnote" id="t1fn3"><sup>d</sup><p class="last">To comply with other metrics used in the table, the reported number for the percent of results within 3-fold indicates a comparison of the numerical results at its face value (e.g., activity of >1 μM is considered to be not within 3-fold of the activity of >5 μM, comparing 1 μM and 5 μM). If the prefixes are not eliminated before the comparison (e.g., >1 μM is considered to be the same as >5 μM and therefore within 3-fold), then this value becomes 77, which was used in our previous work.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></p></div></div></div><div class="NLM_p">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> shows the concordance values for the 452 kinase-inhibitor pairs with nonqualified data. With minor exceptions, all metrics show that nonqualified data has a significantly lower concordance. Thus, we find less agreement in cross-comparing or predicting the activity level of compounds rather than their inactivity because most qualified data came from compounds that are low in activity (i.e., IC<sub>50</sub> beyond the assay detection limit). The concordance trends between different data sources are similar to those for all data; however, the discrepancy between the two experimental sources deteriorates much less than for the predictive models. It is also of note that predictive models, on average, underestimate the experimental potency of active compounds by 2.3- to 3.6-fold.</div><div class="NLM_p">Both result sets suggest that the ability to estimate one’s own experimental outcome is not yet terribly satisfying, especially for active compounds. The problem could be complicated when one tries to increase the training set size by combining multiple data sources without proper normalization. The cMSR metric provides a more direct performance measure because we can compare directly the performance of the underlying assays as measured by MSR. The cMSR accounts for both the variability and the bias between sources of data. As we discussed earlier, bias can come in different varieties (constant vs nonconstant, linear vs nonlinear). The simple between-source bias correction that we attempted here worked to some extent, but a more thorough normalization procedure should be performed in practice before combining data from multiple sources. In some cases, it may well be decided that combining the data is not feasible because of uncertainty in the normalization process.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Concordance of Nonqualified 452 Kinase-Inhibitor Pairs Measured by Four Metrics<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">source 1</th><th class="colsep0 rowsep0" align="center">source 2</th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sup>2</sup>, nonqualified</th><th class="colsep0 rowsep0" align="center" char=".">mean fold difference, nonqualified</th><th class="colsep0 rowsep0" align="center" char=".">cMSR, nonqualified</th><th class="colsep0 rowsep0" align="center">percent within 3- fold, nonqualified</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(LLY)</td><td class="colsep0 rowsep0" align="left">Exp(Metz)</td><td class="colsep0 rowsep0" align="char" char=".">0.46</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">23.3</td><td class="colsep0 rowsep0" align="center">57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(LLY)</td><td class="colsep0 rowsep0" align="left">Pred(LLY)</td><td class="colsep0 rowsep0" align="char" char=".">0.35</td><td class="colsep0 rowsep0" align="char" char=".">–3.6</td><td class="colsep0 rowsep0" align="char" char=".">53.3</td><td class="colsep0 rowsep0" align="center">45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(Metz)</td><td class="colsep0 rowsep0" align="left">Pred(Metz)</td><td class="colsep0 rowsep0" align="char" char=".">0.23</td><td class="colsep0 rowsep0" align="char" char=".">–2.3</td><td class="colsep0 rowsep0" align="char" char=".">42.6</td><td class="colsep0 rowsep0" align="center">50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(Metz)</td><td class="colsep0 rowsep0" align="left">Pred(LLY)</td><td class="colsep0 rowsep0" align="char" char=".">0.14</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">62.9</td><td class="colsep0 rowsep0" align="center">43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Exp(LLY)</td><td class="colsep0 rowsep0" align="left">Pred(Metz)</td><td class="colsep0 rowsep0" align="char" char=".">0.17</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">73.1</td><td class="colsep0 rowsep0" align="center">41</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">Nonqualified data indicates a compound’s activity reported without the qualifiers < or >.</p></div></div></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">3.2 Concordance of Per-Compound Selectivity Profiles</h3><div class="NLM_p">Large screening panels are typically used to understand compound selectivity profiles and off-target activities. Heat maps are common ways to visually analyze such selectivity data.<a onclick="showRef(event, 'ref15 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref15 ref23 ref24">(15, 23, 24)</a> They allow a visual comparison of the similarity between two selectivity profiles of the same compound from two profiling sources versus the profile of a different compound from the same panel. Moreover, quantification of the per-compound selectivity profile allows one to examine better the differences between other compounds tested in the same panel.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Heat maps for 29 of the 32 compounds tested in at least 25 targets. Empty values are shown in gray. Targets in columns are arranged in alphabetical order. Pred(LLY), Exp(LLY), and Exp(Metz) are shown in the left, middle, and right panels, respectively. (b) Difference maps between Exp (LLY) and Pred (LLY) (left side) and Exp (LLY) and Exp(Metz) (right). Empty values are shown in gray. Per-compound cMSR values are listed on the map. Deeper purple indicates larger differences in activity between two sources, whereas lighter purple means smaller difference. In both figures, compounds are sorted by per-compound cMSR between Exp(LLY) and Pred(LLY). (c) Scatter plot showing the log<sub>10</sub> fold difference in activity of Pred(LLY)–Exp(LLY) versus Exp(Metz)–Exp(LLY). Kinase-inhibitor pairs that are within 3-fold in different data sources are colored accordingly: red for Pred(LLY)–Exp(LLY) and blue for Exp(Metz)–Exp(LLY). Of the total 1466 Exp(LLY) data, 777 (or 53%) are within 3–fold of both Exp(Metz) and Pred(LLY). These are colored magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>a shows the bioprint heat map for Pred(LLY), Exp(LLY), and Exp(Metz) with rows sorted by the per-compound cMSR between Exp(LLY) and Pred(LLY). (<i>R</i><sup>2</sup> sorting of the same panel is shown in <a class="ref internalNav" href="#notes-1" aria-label="Figure S6">Figure S6</a>.) The corresponding difference maps, colored by the log10 activity difference of Pred(LLY)–Exp(LLY) and Exp(LLY)–Exp(Metz), are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b. Intuitively, the top portions of both profiles (left and middle panels for Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>a) should be more similar than the bottom portions. The top 10 compounds on the left panel of Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b show smaller differences (light purple), indicating a higher similarity profile compared to compounds in the bottom.</div><div class="NLM_p">Although global metrics (cMSR or <i>R</i><sup>2</sup>) can assess overall bioprint similarity, they may not represent well the concordance between the outliers (e.g., the most potent selective compound). Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>a shows that Exp(LLY) and Exp (Metz) (middle and right panels, respectively) are more similar in terms of detecting potent, selective compounds (e.g., SGCV121 or SGCV97) despite the generally similar nature of the experimental and the same source of the predictive results. However, Pred(LLY) appears to predict better in the inactive space (i.e., the blue regions of heat map). This is likely due to the distribution of results in the experimental data sets trending toward the inactive space. For predictive results, such trends are captured during the model training process. To quantify this observation, we have determined the most potent activity and the associated target for 29 compounds with more >20 assay values from each profiling result (i.e., most likely the primary target for each compound). As shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, 11 of 29 compounds have identical “most likely targets” for Lilly and Metz’s experiment data sets. On the other side, Pred(LLY) data only shares three most likely targets with Exp(LLY) and four with Exp(Metz). Similarly, Pred(Metz) data identifies the same target in three cases as Exp(Metz) and five cases as Exp(LLY). In summary, both models significantly underperform when compared to in-between experimental agreement. Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> also shows literature reported primary targets for 24 compounds.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The Lilly and Metz experimental profiles correctly identified the primary target for 11 and 13 of the compounds, respectively, whereas the predictive models built from either data set identified only 5 and 4 times. These results indicate that, unlike the case of entire data sets, the in silico results are significantly less informative/accurate than the results from other sources of experimental profiling data for individual compounds. In addition, computational models utilizing structural descriptors tend to predict similar selectivity profiles for closely related compounds, whereas in some instances experimental profiles for these related compounds might differ significantly (see <a class="ref internalNav" href="#notes-1" aria-label="Figure S8">Figure S8</a> for graphical representation of bioprint-based dendrograms for compounds based on experimental and predicted profiles).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Most Potent Activity and the Corresponding Target of the 29 Compounds<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0006.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0007.gif" alt="" id="GRAPHIC-d2473e1001-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0008.gif" alt="" id="GRAPHIC-d2473e1002-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup>Table a</sup><p class="last">Compound ID is followed by its common synonym or Pubchem ID. Compound activity is shown as pIC<sub>50</sub>. If known, primary targets for 24 compounds are listed on the basis of Davis’s report<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> or literature searches. For each compound, data sources suggesting the same most potent target are shown in bold. Green cells indicate agreements between the data source and the reported primary targets.</p></div></div><div></div></div><div class="NLM_p last">Selectivity ratio is another popular approach to characterize the selectivity profile of compounds and therefore we investigated the in silico prediction of the selectivity ratio (results reported from individual value predictions) for each compound in each target-pair combination. The concordance of the selectivity ratios shows a trend similar to that seen in absolute values (<a class="ref internalNav" href="#notes-1" aria-label="Table S4">Table S4</a>). In addition, the visualization of compound clusters driven by their bioprint profiles suggests that such a clustering result is also highly dependent on the exact data source used (<a class="ref internalNav" href="#notes-1" aria-label="Figure S8">Figure S8</a>).</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">3.3 Concordance of the Per-Target Bioprint, Outsourcing Decisions</h3><div class="NLM_p">The distributions of the per-target cMSRs generated between different data sources are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>. They show overall trends consistent with the concordance of all data. Geometric means of cMSRs are the lowest between the two experimental sources (10.2) and predicted models within the same sources: 13.8 for Exp(LLY) and Pred(LLY) and 15.1 for Exp(Metz) and Pred(Metz). In silico predictions per target compared to experimental data from a different source are less concordant (mean of 16.2 for predicting Metz’s data using the Lilly-based model and 21.9 for predicting Lilly’s data using the Metz-based model). A number of targets have a cMSR value of 10 or less: 37 between the Lilly and Metz experimental data, 28 and 21 between the predicted in-source models and experimental data for Lilly and Metz, respectively, and 13 for Lilly data predicted from Metz-based models. The data presented in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> suggests that data concordance varies for each of Sugen kinase groups/subfamilies. For example, results for AGC and CAMK kinases are more concordant than those for tyrosine kinases (TK) between the two experimental data sets. A similar trend is observed when comparing the experimental data with predictive models. The lower concordance for TK kinases could be related to larger variability in assay protocols and larger diversity of inhibition modalities. Both can possibly contribute to the poor performance of predictive models.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Per-target log<sub>10</sub>(cMSR) distribution between sources (68 targets included with more than 10 common compounds values). Geometric means are shown for the cMSR for each pair of sources. Bars are colored by kinase Sugen groups.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Additional details are provided in <a class="ref internalNav" href="#notes-1" aria-label="Figure S9">Figure S9</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For the target computational models with a cMSR value of 10 or less, computational models might give reliable predictions to the experimental outcome. A practical question is whether we could a priori identify those potentially promising models during training protocols. To answer this question, we investigated the relationship between the cMSR metrics of the training set cross-validations and test set. In our data set, we observed good concordance between the cross-validation cMSR and test set cMSR for both Lilly (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>a) and Metz (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>b) data sets. For Lilly data, 28 or 61% of models with cross-validation cMSR <10 have cMSR <10 on test data. For Metz data, 16 of 31 or 52% models with cross-validation cMSR < 10 also have test set cMSR <10. In contrast, the <i>R</i><sup>2</sup> metrics from the cross-validations shown in <a class="ref internalNav" href="#notes-1" aria-label="Figure S3c,d">Figure S3c,d</a> do not show meaningful relationship with test set <i>R</i><sup>2</sup> values. Although these results might be specific to our data set, they also indicate that cMSR model metrics from cross-validation procedures might be a good predictor of future model performance.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation of cross-validation cMSR and test set cMSR for (a) Lilly and (b) Metz data sets. Models with cross-validation cMSR <10 are shown in red, models with test set cMSR <10 are shown in blue, and models with cMSR <10 in both the cross-validation and test set are shown in magenta. More detailed analysis of cross-validation results can be found in <a class="ref internalNav" href="#notes-1" aria-label="Figure S3">Figure S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Overall, per-target results point to the fact that higher selectivity concordance for computational models can be achieved only if all models, including the worst ones, can be improved to the desired quality. However, improvements in the overall experimental profiling concordance can be guided by cMSR given that vendors are flexible in panel customization assays and low concordance to internal standards can be eliminated. If the biases between individual targets are known and are time and compound independent, then the between-assays shift can be incorporated into data transformation given that the mean corrected concordance is good.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">3.4 Target Space Visualization from Different Bioprints</h3><div class="NLM_p">Target space relationships and visualization guided by small-molecule-based similarity has been the subject of multiple studies.<a onclick="showRef(event, 'ref5 ref27'); return false;" href="javascript:void(0);" class="ref ref5 ref27">(5, 27)</a> The small-molecule driven view of target space is a key driver in finding chemically tractable target combinations for designed polypharmacology.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In our previous research, we have pointed out that such target space relationships are both compound-set and profiling-data dependent.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Here, we have generated target dendrograms from bioprints of 28 compounds (listed in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> show the dendrograms for Pred(LLY), Exp(LLY), and Exp(Metz). The overall grouping of kinase targets is visually consistent among all three dendrograms, which, in turn, are consistent at a high level with sequence-based-derived kinase Sugen groups.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Although one can find a bit more similarity between predicted and experimental Lilly target space (for CMGC/green and TK/blue), the individual details of the dendrograms differ significantly between all three pictures. These differences, consistent with our initial observation,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> point out that details of compound-driven target space will strongly depend on the data source from which the relationship is derived (even if the same compound set is used).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Bioprint-driven target dendrograms for predicted Lilly (top), experimental Lilly (middle), and experimental Metz (bottom) profiling data sets. Sixty eight targets are colored by kinase subfamilies:<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> AGC, red; CAMK, magenta; CMGC, green; TK, blue; TKL, cyan; other, yellow. The target size is proportional to the number of the 29 compounds with <500 nM activity. The value varies from 0 to 15 for Exp(LLY) and 12 for Exp(Metz) data and from 0 to 7 for the predicted Lilly data. The underlying correlation between the target–target Tanimoto similarity matrix indicates a higher concordance between the Exp(LLY) and Pred(LLY) dendrogram (<i>R</i><sup>2 </sup>= 0.69) and lower concordance between the Exp(LLY) and Exp(Metz) dendrogram (<i>R</i><sup>2</sup> = 0.41). <a class="ref internalNav" href="#notes-1" aria-label="Figure S10">Figure S10</a> presents the scatter plots and data distributions for the target–target comparison.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">4 Discussion/Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43755" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43755" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this work, we have introduced a new metric for comparing profiling data and assessed our ability to predict selectivity profiles. Although the intersource experimental profiling data concordance is relatively low, our analysis revealed a number of interesting trends. Overall, the concordance of the in silico predictions built from the corresponding experimental data set is comparable to the concordance between the experimental results from different sources. At the same time, we found that predictive models produce different selectivity profiles for potent compounds compared to other source experiments. Specifically, the predictive models agreed on the most potent targets for approximately 20% of the compounds versus experimental, whereas different source experiments agreed on the most potent target identification for almost 50% of the compounds. We also showed that bioprint-based depiction of target and compound clustering differs for various source data and predictive models. Target space relationships generated from different experimental data sources show an overall similarity to the ones generated by in silico predictions; however, they substantially differ in details.</div><div class="NLM_p">The overall between-source profiling data concordance are between 4 (all data) to 8 (nonqualified data) times worse than typical, within assay experimental variability for active compounds. Our findings have implications for the decisions to generate or predict selectivity profiles from different laboratories or vendors. When individual compound-specific profiles are important, outsourcing profiles are favored. When profiling estimates for groups of compounds are needed, computational models built from the relevant source might be acceptable and preferred.</div><div class="NLM_p">We have also demonstrated that the computational models built from one data source are less predictive to inform on the data from another data source. Proper data normalization, and bias correction are needed before literature data sets can be utilized for model building. Because of the underlying data discrepancies between experiment sources, this is true even if the compounds used in the models appear chemically similar to the applicability domain. Finally, our findings indicate that additional research in selectivity prediction is required to enable the development of robust, useful in silico selectivity-prediction model methods and that data concordance may be one of the key considerations for achieving improved results.</div><div class="NLM_p last">It is our opinion that the goal of selectivity-prediction research in the near future is to accurately predict compound activity in each of the targets. We believe that the compound-based fingerprints used in this work, although providing a reasonable baseline to predict selectivity, also impose limits to the advancement of selectivity predictions beyond the current state. The incorporation of chemistry-independent physically relevant free-energy methods might provide an alternative capable to advance the predictions within experimental variability. Furthermore, in the case of specific drug-discovery projects, a more thorough assessment of compound selectivity profiles will include target-specific cellular activities or functional outcomes under physiologically relevant conditions. Although out of scope of the present work, the cMSR metric that we have introduced can be readily applied to the comparison of more complex end points.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16"><a href="/doi/suppl/10.1021/jm400798j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52334" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52334" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Methods of cMSR versus RMSE; mean values for training and test set experimental activities; bias-uncorrected results from Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> for all 1466 data points; tesults for the Random Forest (RF) models compared to SVMFP models with ECC3 fingerprints for all 1466 test set data; concordance of the selectivity ratios for the collated data 38 450 selectivity ratio pairs; test set raw data, experimental values, and predictions; simulated data to illustrate artificially an artificially inflated <i>R</i><sup>2</sup>; training set data distribution for both Lilly and Metz sources; nearest neighbor distribution in generalized fingerprint space for the common 32 compounds with respect to Lilly and Metz training sets; comparison of the training set 50-50 cross-validation results for Lilly and Metz data sets for the <i>R</i><sup>2</sup> and cMSR metrics; cumulative comparison of the training set (model aware) experimental versus predicted data for all kinase-inhibitor pairs for the Lilly training set and the Metz bias-uncorrected training set; graphical representation of all kinase-inhibitor pair correlations for the experimental and predicted results from different sources; heat maps for 29 of 32 compounds with more than 25 data points against 68 targets; 38 450 target pair compound log selectivity ratio correlations between the experimental and predicted data compound target–target pairs; bioprint-driven compound dendrograms for the predicted Lilly, experimental Lilly, and experimental Metz profiling data sets; per target log(cMSR) anova and distribution histograms; and comparison of the nonsymmetrical 2278 target–target Tanimoto similarities between the Lilly experimental, Metz experimental, and Lilly predicted values computed from the available 29 compound pIC<sub>50</sub>s. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400798j/suppl_file/jm400798j_si_001.pdf">jm400798j_si_001.pdf (886.67 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400798j/suppl_file/jm400798j_si_002.xlsx">jm400798j_si_002.xlsx (225.22 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm400798j" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26114" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26114" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michal Vieth</span> - <span class="hlFld-Affiliation affiliation">†Discovery Chemistry, ‡Discovery Statistics, §Advanced Analytics, Lilly Research Laboratories, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7a0c131f0e1225171319121b163a161316160354191517">vieth_m<span class="__cf_email__" data-cfemail="e48d878c8588a4888d88889dca878b89">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cen Gao</span> - <span class="hlFld-Affiliation affiliation">†Discovery Chemistry, ‡Discovery Statistics, §Advanced Analytics, Lilly Research Laboratories, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Suntara Cahya</span> - <span class="hlFld-Affiliation affiliation">†Discovery Chemistry, ‡Discovery Statistics, §Advanced Analytics, Lilly Research Laboratories, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christos A. Nicolaou</span> - <span class="hlFld-Affiliation affiliation">†Discovery Chemistry, ‡Discovery Statistics, §Advanced Analytics, Lilly Research Laboratories, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jibo Wang</span> - <span class="hlFld-Affiliation affiliation">†Discovery Chemistry, ‡Discovery Statistics, §Advanced Analytics, Lilly Research Laboratories, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian A. Watson</span> - <span class="hlFld-Affiliation affiliation">†Discovery Chemistry, ‡Discovery Statistics, §Advanced Analytics, Lilly Research Laboratories, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Cummins</span> - <span class="hlFld-Affiliation affiliation">†Discovery Chemistry, ‡Discovery Statistics, §Advanced Analytics, Lilly Research Laboratories, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip W. Iversen</span> - <span class="hlFld-Affiliation affiliation">†Discovery Chemistry, ‡Discovery Statistics, §Advanced Analytics, Lilly Research Laboratories, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d2473e1141-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Thibault Varin for helpful discussions and Dr. Quan Liao for implementation of the equal angle algorithm for dendrogram creation.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i18" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i18"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i19" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i19"> Abbreviations Used</h2><tr><td class="NLM_term">cMSR</td><td class="NLM_def"><p class="first last">concordance minimum significant ratio</p></td></tr><tr><td class="NLM_term">SVMFP</td><td class="NLM_def"><p class="first last">support vector machine fingerprint model</p></td></tr><tr><td class="NLM_term">RMSE</td><td class="NLM_def"><p class="first last">root-mean squared error</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03817" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03817" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 27 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0ljDqtPLPh6ZIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, K.</span><span> </span><span class="NLM_article-title">Assessment of chemical coverage of kinome space and its implications for kinase drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7898</span><span class="NLM_x">–</span> <span class="NLM_lpage">7914</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8011036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7898-7914&author=P.+Bamboroughauthor=D.+Drewryauthor=G.+Harperauthor=G.+K.+Smithauthor=K.+Schneider&title=Assessment+of+chemical+coverage+of+kinome+space+and+its+implications+for+kinase+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Chemical Coverage of Kinome Space and Its Implications for Kinase Drug Discovery</span></div><div class="casAuthors">Bamborough, Paul; Drewry, David; Harper, Gavin; Smith, Gary K.; Schneider, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7914</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">More than 500 compds. chosen to represent kinase inhibitor space have been screened against a panel of over 200 protein kinases.  Significant results include the identification of hits against new kinases including PIM1 and MPSK1, and the expansion of the inhibition profiles of several literature compds.  A detailed anal. of the data through the use of affinity fingerprints has produced findings with implications for biol. target selection, the choice of tool compds. for target validation, and lead discovery and optimization.  In a detailed examn. of the tyrosine kinases, interesting relationships have been found between targets and compds.  Taken together, these results show how broad cross-profiling can provide important insights to assist kinase drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBRI-arQk3t7Vg90H21EOLACvtfcHk0liFWr-scX6aOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI&md5=8d98c80fb157b987a19e32a1efb53103</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm8011036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8011036%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDrewry%26aufirst%3DD.%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DG.%2BK.%26aulast%3DSchneider%26aufirst%3DK.%26atitle%3DAssessment%2520of%2520chemical%2520coverage%2520of%2520kinome%2520space%2520and%2520its%2520implications%2520for%2520kinase%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7898%26epage%3D7914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettles, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span> </span><span class="NLM_article-title">“Bayes affinity fingerprints” improve retrieval rates in virtual screening and define orthogonal bioactivity space: When are multitarget drugs a feasible concept?</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">2445</span><span class="NLM_x">–</span> <span class="NLM_lpage">2456</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600197y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1yqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=2445-2456&author=A.+Benderauthor=J.+L.+Jenkinsauthor=M.+Glickauthor=Z.+Dengauthor=J.+H.+Nettlesauthor=J.+W.+Davies&title=%E2%80%9CBayes+affinity+fingerprints%E2%80%9D+improve+retrieval+rates+in+virtual+screening+and+define+orthogonal+bioactivity+space%3A+When+are+multitarget+drugs+a+feasible+concept%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">"Bayes Affinity Fingerprints" Improve Retrieval Rates in Virtual Screening and Define Orthogonal Bioactivity Space: When Are Multitarget Drugs a Feasible Concept?</span></div><div class="casAuthors">Bender, Andreas; Jenkins, Jeremy L.; Glick, Meir; Deng, Zhan; Nettles, James H.; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2445-2456</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conventional similarity searching of mols. compares single (or multiple) active query structures to each other in a relative framework, by means of a structural descriptor and a similarity measure.  While this often works well, depending on the target, the authors show here that retrieval rates can be improved considerably by incorporating an external framework describing ligand bioactivity space for comparisons ("Bayes affinity fingerprints").  Structures are described by Bayes scores for a ligand panel comprising about 1000 activity classes extd. from the WOMBAT database.  The comparison of structures is performed via the Pearson correlation coeff. of activity classes, i.e., the order in which two structures are similar to the panel activity classes.  Compd. retrieval on a recently published data set could be improved by as much as 24% relative (9% abs.).  Knowledge about the shape of the "bioactive chem. universe" is thus beneficial to identifying similar bioactivities.  Principal component anal. was employed to further analyze activity space with the objective to define orthogonal ligand bioactive chem. space, leading to nine major (roughly orthogonal) activity axes.  Employing only those nine activity classes, retrieval rates are still comparable to original Bayes affinity fingerprints; thus, the concept of orthogonal bioactive ligand chem. space was validated as being an information-rich but low-dimensional representation of bioactivity space.  Correlations between activity classes are a major determinant to gauge whether the desired multitarget activity of drugs is (on the basis of current knowledge) a feasible concept because it measures the extent to which activities can be optimized independently, or only by strongly influencing one another.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6IjlYwYDTwLVg90H21EOLACvtfcHk0liFWr-scX6aOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1yqsrs%253D&md5=47adb720154f45bda1738963d54c70a5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fci600197y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600197y%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DNettles%26aufirst%3DJ.%2BH.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3D%25E2%2580%259CBayes%2520affinity%2520fingerprints%25E2%2580%259D%2520improve%2520retrieval%2520rates%2520in%2520virtual%2520screening%2520and%2520define%2520orthogonal%2520bioactivity%2520space%253A%2520When%2520are%2520multitarget%2520drugs%2520a%2520feasible%2520concept%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D2445%26epage%3D2456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serrano, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemons, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J.</span><span> </span><span class="NLM_article-title">Chemogenomic data analysis: Prediction of small-molecule targets and the advent of biological fingerprints</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">719</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.2174%2F138620707782507313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=18045083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=719-731&author=A.+Benderauthor=D.+Youngauthor=J.+Jenkinsauthor=M.+Serranoauthor=D.+Mikhailovauthor=P.+Clemonsauthor=J.+Davies&title=Chemogenomic+data+analysis%3A+Prediction+of+small-molecule+targets+and+the+advent+of+biological+fingerprints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic data analysis: prediction of small-molecule targets and the advent of biological fingerprints</span></div><div class="casAuthors">Bender, Andreas; Young, Daniel W.; Jenkins, Jeremy L.; Serrano, Martin; Mikhailov, Dmitri; Clemons, Paul A.; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">719-731</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Chemogenomics comprises a systematic relationship between targets and ligands that are used as target modulators in living systems such as cells or organisms.  In recent years, data on small mol.-bioactivity relationships have become increasingly available, and consequently so have the no. of approaches used to translate bioactivity data into knowledge.  This review will focus on 2 aspects of chemogenomics.  Firstly, in cases such as cell-based screens, the question of which target(s) a compd. is modulating to cause the obsd. phenotype is crucial.  In silico target prediction tools can suggest likely biol. targets of small mols. via data mining in target-annotated chem. databases.  This review presents some of the current tools available for this task and shows some sample applications relevant to a pharmaceutical industry setting.  These applications are the prediction of false-positives in cell-based reporter gene assays, the prediction of targets by linking bioassay data with protein domain annotations, and the direct prediction of adverse reactions.  Secondly, in recent years a shift from structure-derived chem. descriptors to biol. descriptors has occurred.  Here, the effect of a compd. on a no. of biol. endpoints is used to make predictions about other properties, such as putative targets, assocd. adverse reactions, and pathways modulated by the compd.  This review further summarizes these "performance" descriptors and their applications, focusing on gene expression profiles and high-content screening data.  The advent of such biol. fingerprints suggests that the field of drug discovery is currently at a crossroads, where single target bioassay results are supplanted by multidimensional biol. fingerprints that reflect a new awareness of biol. networks and polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJuaT0KRiZF7Vg90H21EOLACvtfcHk0liFWr-scX6aOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOhsg%253D%253D&md5=4b3152898f48cc7021904a0c983f6068</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F138620707782507313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620707782507313%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DD.%26aulast%3DJenkins%26aufirst%3DJ.%26aulast%3DSerrano%26aufirst%3DM.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DClemons%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DJ.%26atitle%3DChemogenomic%2520data%2520analysis%253A%2520Prediction%2520of%2520small-molecule%2520targets%2520and%2520the%2520advent%2520of%2520biological%2520fingerprints%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2007%26volume%3D10%26spage%3D719%26epage%3D731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urban, L.</span><span> </span><span class="NLM_article-title">Large-scale prediction and testing of drug activity on side-effect targets</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">486</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnature11159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=22722194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=361-367&author=E.+Lounkineauthor=M.+J.+Keiserauthor=S.+Whitebreadauthor=D.+Mikhailovauthor=J.+Hamonauthor=J.+L.+Jenkinsauthor=P.+Lavanauthor=E.+Weberauthor=A.+K.+Doakauthor=S.+Coteauthor=B.+K.+Shoichetauthor=L.+Urban&title=Large-scale+prediction+and+testing+of+drug+activity+on+side-effect+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale prediction and testing of drug activity on side-effect targets</span></div><div class="casAuthors">Lounkine, Eugen; Keiser, Michael J.; Whitebread, Steven; Mikhailov, Dmitri; Hamon, Jacques; Jenkins, Jeremy L.; Lavan, Paul; Weber, Eckhard; Doak, Allison K.; Cote, Serge; Shoichet, Brian K.; Urban, Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">486</span>
        (<span class="NLM_cas:issue">7403</span>),
    <span class="NLM_cas:pages">361-367</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Discovering the unintended off-targets' that predict adverse drug reactions is daunting by empirical methods alone.  Drugs can act on several protein targets, some of which can be unrelated by conventional mol. metrics, and hundreds of proteins have been implicated in side effects.  Here we use a computational strategy to predict the activity of 656 marketed drugs on 73 unintended side-effect' targets.  Approx. half of the predictions were confirmed, either from proprietary databases unknown to the method or by new exptl. assays.  Affinities for these new off-targets ranged from 1 nM to 30 μM.  To explore relevance, we developed an assocn. metric to prioritize those new off-targets that explained side effects better than any known target of a given drug, creating a drug-target-adverse drug reaction network.  Among these new assocns. was the prediction that the abdominal pain side effect of the synthetic estrogen chlorotrianisene was mediated through its newly discovered inhibition of the enzyme cyclooxygenase-1.  The clin. relevance of this inhibition was borne out in whole human blood platelet aggregation assays.  This approach may have wide application to de-risking toxicol. liabilities in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5sVjfn4mgLLVg90H21EOLACvtfcHk0ljn6WRdiPIbZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D&md5=9c133b839d9527818a27a5834e4dfe11</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature11159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11159%26sid%3Dliteratum%253Aachs%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DLavan%26aufirst%3DP.%26aulast%3DWeber%26aufirst%3DE.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DCote%26aufirst%3DS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DUrban%26aufirst%3DL.%26atitle%3DLarge-scale%2520prediction%2520and%2520testing%2520of%2520drug%2520activity%2520on%2520side-effect%2520targets%26jtitle%3DNature%26date%3D2012%26volume%3D486%26spage%3D361%26epage%3D367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Krejsa, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogalski, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penzotti, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Migeon, J. C.</span><span> </span><span class="NLM_article-title">Predicting ADME properties and side effects: The BioPrint approach</span> <span class="citation_source-journal">Curr. Opin. Drug Discov. Devel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=12951810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFymsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2003&pages=470-480&author=C.+M.+Krejsaauthor=D.+Horvathauthor=S.+L.+Rogalskiauthor=J.+E.+Penzottiauthor=B.+Maoauthor=F.+Barbosaauthor=J.+C.+Migeon&title=Predicting+ADME+properties+and+side+effects%3A+The+BioPrint+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting ADME properties and side effects: The BioPrint approach</span></div><div class="casAuthors">Krejsa, Cecile M.; Horvath, Dragos; Rogalski, Sherri L.; Penzotti, Julie E.; Mao, Boryeu; Barbosa, Frederique; Migeon, Jacques C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">470-480</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Current Drugs</span>)
        </div><div class="casAbstract">A review.  Computational methods are increasingly used to streamline and enhance the lead discovery and optimization process.  However, accurate prediction of absorption, distribution, metab. and excretion (ADME) and adverse drug reactions (ADR) is often difficult, due to the complexity of underlying physiol. mechanisms.  Modeling approaches have been hampered by the lack of large, robust and standardized training datasets.  In an extensive effort to build such a dataset, the BioPrint database was constructed by systematic profiling of nearly all drugs available on the market, as well as numerous ref. compds.  The database is composed of several large datasets: compd. structures and mol. descriptors, in vitro ADME and pharmacol. profiles, and complementary clin. data including therapeutic use information, pharmacokinetics profiles and ADR profiles.  These data have allowed the development of computational tools designed to integrate a program of computational chem. into library design and lead development.  Models based on chem. structure are strengthened by in vitro results that can be used as addnl. compd. descriptors to predict complex in vivo endpoints.  The BioPrint pharmacoinformatics platform represents a systematic effort to accelerate the process of drug discovery, improve quant. structure-activity relationships and develop in vitro/in vivo assocns.  In this review, we will discuss the importance of training set size and diversity in model development, the implementation of linear and neighborhood modeling approaches, and the use of in silico methods to predict potential clin. liabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQ71juUoErrVg90H21EOLACvtfcHk0ljn6WRdiPIbZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFymsLw%253D&md5=f8db4e37cc895d67bf4a82efe021c5c2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrejsa%26aufirst%3DC.%2BM.%26aulast%3DHorvath%26aufirst%3DD.%26aulast%3DRogalski%26aufirst%3DS.%2BL.%26aulast%3DPenzotti%26aufirst%3DJ.%2BE.%26aulast%3DMao%26aufirst%3DB.%26aulast%3DBarbosa%26aufirst%3DF.%26aulast%3DMigeon%26aufirst%3DJ.%2BC.%26atitle%3DPredicting%2520ADME%2520properties%2520and%2520side%2520effects%253A%2520The%2520BioPrint%2520approach%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discov.%2520Devel.%26date%3D2003%26volume%3D6%26spage%3D470%26epage%3D480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruda, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abecassis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguiz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norval, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeons, F. R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidah, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constam, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetsel, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.-P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+C.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0ljn6WRdiPIbZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%2BC.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Ning, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karypis, G.</span><span> </span><span class="NLM_article-title">Improved machine learning models for predicting selective compounds</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">38</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci200346b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2kt77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=38-50&author=X.+Ningauthor=M.+Waltersauthor=G.+Karypis&title=Improved+machine+learning+models+for+predicting+selective+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Machine Learning Models for Predicting Selective Compounds</span></div><div class="casAuthors">Ning, Xia; Walters, Michael; Karypis, George</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-50</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of small potent compds. that selectively bind to the target under consideration with high affinities is a crit. step toward successful drug discovery.  However, there is still a lack of efficient and accurate computational methods to predict compd. selectivity properties.  In this paper, we propose a set of machine learning methods to do compd. selectivity prediction.  In particular, we propose a novel cascaded learning method and a multitask learning method.  The cascaded method decomps. the selectivity prediction into two steps, one model for each step, so as to effectively filter out nonselective compds.  The multitask method incorporates both activity and selectivity models into one multitask model so as to better differentiate compd. selectivity properties.  We conducted a comprehensive set of expts. and compared the results with those of other conventional selectivity prediction methods, and our results demonstrated that the cascaded and multitask methods significantly improve the selectivity prediction performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdkf0ZGDNBm7Vg90H21EOLACvtfcHk0ljwv0DeZyDg-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2kt77I&md5=2038119451a4a486b77457f966276e5e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci200346b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200346b%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DX.%26aulast%3DWalters%26aufirst%3DM.%26aulast%3DKarypis%26aufirst%3DG.%26atitle%3DImproved%2520machine%2520learning%2520models%2520for%2520predicting%2520selective%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D38%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Martin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansen, J.</span><span> </span><span class="NLM_article-title">Profile-QSAR: A novel meta-QSAR method that combines activities across the kinase family to accurately predict affinity, selectivity, and cellular activity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1942</span><span class="NLM_x">–</span> <span class="NLM_lpage">1956</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci1005004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1942-1956&author=E.+Martinauthor=P.+Mukherjeeauthor=D.+Sullivanauthor=J.+Jansen&title=Profile-QSAR%3A+A+novel+meta-QSAR+method+that+combines+activities+across+the+kinase+family+to+accurately+predict+affinity%2C+selectivity%2C+and+cellular+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity</span></div><div class="casAuthors">Martin, Eric; Mukherjee, Prasenjit; Sullivan, David; Jansen, Johanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1942-1956</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-QSAR is a novel 2D predictive model building method for kinases.  This "meta-QSAR" method models the activity of each compd. against a new kinase target as a linear combination of its predicted activities against a large panel of 92 previously studied kinases comprised from 115 assays.  Profile-QSAR starts with a sparse incomplete kinase by compd. (KxC) activity matrix, used to generate Bayesian QSAR models for the 92 "basis-set" kinases.  These Bayesian QSARs generate a complete "synthetic" KxC activity matrix of predictions.  These synthetic activities are used as "chem. descriptors" to train partial-least squares (PLS) models, from modest amts. of medium-throughput screening data, for predicting activity against new kinases.  The Profile-QSAR predictions for the 92 kinases (115 assays) gave a median external R2ext = 0.59 on 25% held-out test sets.  The method has proven accurate enough to predict pairwise kinase selectivities with a median correlation of R2ext = 0.61 for 958 kinase pairs with at least 600 common compds.  It has been further expanded by adding a "CkXC" cellular activity matrix to the KxC matrix to predict cellular activity for 42 kinase driven cellular assays with median R2ext = 0.58 for 24 target modulation assays and R2ext = 0.41 for 18 cell proliferation assays.  The 2D Profile-QSAR, along with the 3D Surrogate AutoShim, are the foundations of an internally developed iterative medium-throughput screening (IMTS) methodol. for virtual screening (VS) of compd. archives as an alternative to exptl. high-throughput screening (HTS).  The method has been applied to 20 actual prospective kinase projects.  Biol. results have so far been obtained in eight of them.  Q2 values ranged from 0.3 to 0.7.  Hit-rates at 10 uM for exptl. tested compds. varied from 25% to 80%, except in K5, which was a special case aimed specifically at finding "type II" binders, where none of the compds. were predicted to be active at 10 μM.  These overall results are particularly striking as chem. novelty was an important criterion in selecting compds. for testing.  The method is completely automated.  Predicted activities for nearly 4 million internal and com. compds. across 115 kinase assays and 42 cellular assays are stored in the corporate database.  Like computed phys. properties, this predicted kinase activity profile can be computed and stored as each compd. is registered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTg6KqlgEyLVg90H21EOLACvtfcHk0ljwv0DeZyDg-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D&md5=49422d01fd686a4befc9db7c123cfcce</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1005004%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DSullivan%26aufirst%3DD.%26aulast%3DJansen%26aufirst%3DJ.%26atitle%3DProfile-QSAR%253A%2520A%2520novel%2520meta-QSAR%2520method%2520that%2520combines%2520activities%2520across%2520the%2520kinase%2520family%2520to%2520accurately%2520predict%2520affinity%252C%2520selectivity%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1942%26epage%3D1956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, E.</span><span> </span><span class="NLM_article-title">Profile-QSAR and surrogate AutoShim protein-family modeling of proteases</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2430</span><span class="NLM_x">–</span> <span class="NLM_lpage">2440</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci300059d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WktLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2430-2440&author=P.+Mukherjeeauthor=E.+Martin&title=Profile-QSAR+and+surrogate+AutoShim+protein-family+modeling+of+proteases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR and Surrogate AutoShim Protein-Family Modeling of Proteases</span></div><div class="casAuthors">Mukherjee, Prasenjit; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2430-2440</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 2D Profile-QSAR and 3D Surrogate AutoShim protein-family virtual screening methods were originally developed for kinases.  They are the key components of an iterative medium-throughput screening alternative to expensive and time-consuming exptl. high-throughput screening.  Encouraged by the success with kinases, the S1-serine proteases were selected as a second protein family to tackle, based on the structural and SAR similarity among them, availability of structural and bioactivity data, and the current and future small-mol. drug discovery interest.  A validation study on 24 S1-serine protease assay data sets from 16 unique proteases gave very promising results.  Profile-QSAR gave a median RexT2 = 0.60 for 24 assay data sets, and pairwise selectivity modeling on 60 protease pairs gave a median RexT2 = 0.64, comparable to the performance for kinases.  A 17-structure universal ensemble S1-serine protease surrogate receptor for Autoshim was developed from a collection of ∼1500 x-ray structures.  The predictive performance on 24 S1-serine protease assays was good, with a median RexT2 = 0.41, but lower than had been obtained for kinases.  Anal. suggested that the higher structural diversity of the protease structures, as well as smaller assay data sets and fewer potent compds., both contributed to the decreased predictive power.  In a prospective virtual screening application, 32 compds. were ordered from a 1.5 million archive and tested in a biochem. assay.  Thirteen of the 32 compds. were active at IC50 ≤ 10 μM, a 41% hit-rate.  Three new scaffolds were identified which are being followed up with testing of addnl. analogs.  A SAR similarity anal. for this target against 13 other proteases also indicated two potential protease targets which were pos. and neg. correlated with the activity of the target protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr606feZGrYa7Vg90H21EOLACvtfcHk0ljwv0DeZyDg-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WktLY%253D&md5=5f24ff3fcdda7604cbd8177c1079e55e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci300059d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300059d%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DProfile-QSAR%2520and%2520surrogate%2520AutoShim%2520protein-family%2520modeling%2520of%2520proteases%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2430%26epage%3D2440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cahya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">M., V.</span><span> </span><span class="NLM_article-title">What general conclusions can we draw from kinase profiling data sets?</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1834</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1433</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1834&publication_year=2013&pages=1425-1433&author=J.+J.+Sutherlandauthor=C.+Gaoauthor=S.+Cahyaauthor=V.+M.&title=What+general+conclusions+can+we+draw+from+kinase+profiling+data+sets%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DCahya%26aufirst%3DS.%26aulast%3DM.%26aufirst%3DV.%26atitle%3DWhat%2520general%2520conclusions%2520can%2520we%2520draw%2520from%2520kinase%2520profiling%2520data%2520sets%253F%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2013%26volume%3D1834%26spage%3D1425%26epage%3D1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span> </span><span class="NLM_article-title">Three useful dimensions for domain applicability in QSAR models using random forest</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">814</span><span class="NLM_x">–</span> <span class="NLM_lpage">823</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci300004n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=814-823&author=R.+P.+Sheridan&title=Three+useful+dimensions+for+domain+applicability+in+QSAR+models+using+random+forest"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Three Useful Dimensions for Domain Applicability in QSAR Models Using Random Forest</span></div><div class="casAuthors">Sheridan, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">814-823</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One popular metric for estg. the accuracy of prospective quant. structure-activity relationship (QSAR) predictions is based on the similarity of the compd. being predicted to compds. in the training set from which the QSAR model was built.  More recent work in the field has indicated that other parameters might be equally or more important than similarity.  Here we make use of two addnl. parameters: the variation of prediction among random forest trees (less variation among trees indicates more accurate prediction) and the prediction itself (certain ranges of activity are intrinsically easier to predict than others).  The accuracy of prediction for a QSAR model, as measured by the root-mean-square error, can be estd. by cross-validation on the training set at the time of model-building and stored as a three-dimensional array of bins.  This is an obvious extension of the one-dimensional array of bins we previously proposed for similarity to the training set.  We show that using these three parameters simultaneously adds much more discrimination in prediction accuracy than any single parameter.  This approach can be applied to any QSAR method that produces an ensemble of models.  We also show that the root-mean-square errors produced by cross-validation are predictive of root-mean-square errors of compds. tested after the model was built.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoch9QmSOjtCrVg90H21EOLACvtfcHk0lhWtTX-oU1KNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWhsL4%253D&md5=85bb79c95e57ad6a95851b8533024a2d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci300004n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300004n%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26atitle%3DThree%2520useful%2520dimensions%2520for%2520domain%2520applicability%2520in%2520QSAR%2520models%2520using%2520random%2520forest%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D814%26epage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Eastwood, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farmen, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iversen, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craft, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smallwood, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garbison, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delapp, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, G. F.</span><span> </span><span class="NLM_article-title">The minimum significant ratio: A statistical parameter to characterize the reproducibility of potency estimates from concentration-response assays and estimation by replicate-experiment studies</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">253</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1177%2F1087057105285611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=16490778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A280%3ADC%252BD283mvVyhug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=253-261&author=B.+J.+Eastwoodauthor=M.+W.+Farmenauthor=P.+W.+Iversenauthor=T.+J.+Craftauthor=J.+K.+Smallwoodauthor=K.+E.+Garbisonauthor=N.+W.+Delappauthor=G.+F.+Smith&title=The+minimum+significant+ratio%3A+A+statistical+parameter+to+characterize+the+reproducibility+of+potency+estimates+from+concentration-response+assays+and+estimation+by+replicate-experiment+studies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The minimum significant ratio: a statistical parameter to characterize the reproducibility of potency estimates from concentration-response assays and estimation by replicate-experiment studies</span></div><div class="casAuthors">Eastwood Brian J; Farmen Mark W; Iversen Philip W; Craft Trelia J; Smallwood Jeffrey K; Garbison Kim E; Delapp Neil W; Smith Gerald F</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">253-61</span>
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    </div><div class="casAbstract">The authors show by illustration that procedures used to validate the reliability of single-concentration high-throughput screens such as the signal window and Z' factor do not ensure sufficient reliability in potency estimates from concentration response assays.  They develop the minimum significant ratio as a statistical parameter to characterize the fold change between 2 compounds run in the same experiment that can be considered a real difference and use this parameter to characterize the reliability of the assay.  They adapt methods described by Bland and Altman to develop a simple set of 2 experiments to estimate the minimum significant ratio and show that this protocol can identify assays that lack reproducibility.  The methods are then extended to validate the equivalency of the same assay run by multiple laboratories.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTcE509f3H8OukdrW1FkXG-fW6udTcc2eYSXEOiPSPf2bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD283mvVyhug%253D%253D&md5=df12ee03e938b7b16652a0ddd77a1a93</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1177%2F1087057105285611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057105285611%26sid%3Dliteratum%253Aachs%26aulast%3DEastwood%26aufirst%3DB.%2BJ.%26aulast%3DFarmen%26aufirst%3DM.%2BW.%26aulast%3DIversen%26aufirst%3DP.%2BW.%26aulast%3DCraft%26aufirst%3DT.%2BJ.%26aulast%3DSmallwood%26aufirst%3DJ.%2BK.%26aulast%3DGarbison%26aufirst%3DK.%2BE.%26aulast%3DDelapp%26aufirst%3DN.%2BW.%26aulast%3DSmith%26aufirst%3DG.%2BF.%26atitle%3DThe%2520minimum%2520significant%2520ratio%253A%2520A%2520statistical%2520parameter%2520to%2520characterize%2520the%2520reproducibility%2520of%2520potency%2520estimates%2520from%2520concentration-response%2520assays%2520and%2520estimation%2520by%2520replicate-experiment%2520studies%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D253%26epage%3D261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Metz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merta, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kifle, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Navigating the kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+kinome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0lhWtTX-oU1KNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D200%26epage%3D202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erickson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I.</span><span> </span><span class="NLM_article-title">Kinase inhibitor data modeling and de novo inhibitor design with fragment approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6456</span><span class="NLM_x">–</span> <span class="NLM_lpage">6466</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901147e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6456-6466&author=M.+Viethauthor=J.+Ericksonauthor=J.+Wangauthor=Y.+Websterauthor=M.+Maderauthor=R.+Higgsauthor=I.+Watson&title=Kinase+inhibitor+data+modeling+and+de+novo+inhibitor+design+with+fragment+approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm901147e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901147e%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DErickson%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWebster%26aufirst%3DY.%26aulast%3DMader%26aufirst%3DM.%26aulast%3DHiggs%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DI.%26atitle%3DKinase%2520inhibitor%2520data%2520modeling%2520and%2520de%2520novo%2520inhibitor%2520design%2520with%2520fragment%2520approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6456%26epage%3D6466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Erickson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Structure-guided expansion of kinase fragment libraries driven by support vector machine models</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1804</span><span class="NLM_x">, </span> <span class="NLM_fpage">642</span><span class="NLM_x">–</span> <span class="NLM_lpage">652</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1016%2Fj.bbapap.2009.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=20005305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1yms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1804&publication_year=2010&pages=642-652&author=J.+A.+Ericksonauthor=M.+M.+Maderauthor=I.+A.+Watsonauthor=Y.+W.+Websterauthor=R.+E.+Higgsauthor=M.+A.+Bellauthor=M.+Vieth&title=Structure-guided+expansion+of+kinase+fragment+libraries+driven+by+support+vector+machine+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided expansion of kinase fragment libraries driven by support vector machine models</span></div><div class="casAuthors">Erickson, Jon A.; Mader, Mary M.; Watson, Ian A.; Webster, Yue W.; Higgs, Richard E.; Bell, Michael A.; Vieth, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1804</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">642-652</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">This work outlines a new de novo design process for the creation of novel kinase inhibitor libraries.  It relies on a profiling paradigm that generates a substantial amt. of kinase inhibitor data from which highly predictive QSAR models can be constructed.  In addn., a broad diversity of X-ray structure information is needed for binding mode prediction.  This is important for scaffold and substituent site selection.  Borrowing from FBDD, the process involves fragmentation of known actives, proposition of binding mode hypotheses for the fragments, and model-driven recombination using a pharmacophore derived from known kinase inhibitor structures.  The support vector machine method, using Merck atom pair derived fingerprint descriptors, was used to build models from activity from 6 kinase assays.  These models were qualified prospectively by selecting and testing compds. from the internal compd. collection.  Overall hit and enrichment rates of 82% and 2.5%, resp., qualified the models for use in library design.  Using the process, 7 novel libraries were designed, synthesized and tested against these same 6 kinases.  The results showed excellent results, yielding a 92% hit rate for the 179 compds. that made up the 7 libraries.  The results of one library designed to include known literature compds., as well as an anal. of overall substituent frequency, are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF5GYJ7qB3MbVg90H21EOLACvtfcHk0ljrW_IXq6nCww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1yms7s%253D&md5=87d3531b362dd5ed32889efaf451862b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2009.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2009.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DErickson%26aufirst%3DJ.%2BA.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DWebster%26aufirst%3DY.%2BW.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DBell%26aufirst%3DM.%2BA.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DStructure-guided%2520expansion%2520of%2520kinase%2520fragment%2520libraries%2520driven%2520by%2520support%2520vector%2520machine%2520models%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2010%26volume%3D1804%26spage%3D642%26epage%3D652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Joachims, T.</span><span> </span><span class="NLM_article-title">Making large-scale support vector machine learning practical</span>. In  <span class="citation_source-book">Advances in Kernel Methods</span>; <span class="NLM_contrib-group">Scholkopf, B.; Burges, C. J. C.; Smola, A. J.</span>, Eds.;<span class="NLM_publisher-name"> MIT Press</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">1999</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&pages=169-184&author=T.+Joachimsauthor=B.+Scholkopf&author=C.+J.+C.+Burges&author=A.+J.+Smola&title=Advances+in+Kernel+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJoachims%26aufirst%3DT.%26atitle%3DMaking%2520large-scale%2520support%2520vector%2520machine%2520learning%2520practical%26btitle%3DAdvances%2520in%2520Kernel%2520Methods%26aulast%3DScholkopf%26aufirst%3DB.%26pub%3DMIT%2520Press%26date%3D1999%26spage%3D169%26epage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Doucet, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbault, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panaye, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, B.</span><span> </span><span class="NLM_article-title">Nonlinear SVM approaches to QSPR/QSAR studies and drug design</span> <span class="citation_source-journal">Curr. Comput.-Aided Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">289</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.2174%2F157340907782799372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVSqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=263-289&author=J.-P.+Doucetauthor=F.+Barbaultauthor=H.+Xiaauthor=A.+Panayeauthor=B.+Fan&title=Nonlinear+SVM+approaches+to+QSPR%2FQSAR+studies+and+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Nonlinear SVM approaches to QSPR/QSAR studies and drug design</span></div><div class="casAuthors">Doucet, Jean-Pierre; Barbault, Florent; Xia, Hairong; Panaye, Annick; Fan, Botao</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-289</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Recently, a new promising nonlinear method, the support vector machine (SVM), was proposed by Vapnik.  It rapidly found numerous applications in chem., biochem. and pharmacochem.  Several attempts using SVM in drug design have been reported.  It became an attractive nonlinear approach in this field.  In this review, the theor. basis of SVM in classification and regression is briefly described.  Its applications in QSPR/QSAR studies, and particularly in drug design are discussed.  Comparative studies with some linear and other nonlinear methods show SVM's high performance both in classification and correlation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlUWphlsyC4rVg90H21EOLACvtfcHk0ljrW_IXq6nCww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVSqtLk%253D&md5=5552439113da40b712c3467d7277bf95</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F157340907782799372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340907782799372%26sid%3Dliteratum%253Aachs%26aulast%3DDoucet%26aufirst%3DJ.-P.%26aulast%3DBarbault%26aufirst%3DF.%26aulast%3DXia%26aufirst%3DH.%26aulast%3DPanaye%26aufirst%3DA.%26aulast%3DFan%26aufirst%3DB.%26atitle%3DNonlinear%2520SVM%2520approaches%2520to%2520QSPR%252FQSAR%2520studies%2520and%2520drug%2520design%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2007%26volume%3D3%26spage%3D263%26epage%3D289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Ward, J. H.,  Jr.</span><span> </span><span class="NLM_article-title">Hierarchical grouping to optimize an objective function</span> <span class="citation_source-journal">J. Am. Stat. Assoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1963</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">236</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1080%2F01621459.1963.10500845" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1963&pages=236-244&author=J.+H.+Ward&title=Hierarchical+grouping+to+optimize+an+objective+function"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1080%2F01621459.1963.10500845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F01621459.1963.10500845%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DJ.%2BH.%26atitle%3DHierarchical%2520grouping%2520to%2520optimize%2520an%2520objective%2520function%26jtitle%3DJ.%2520Am.%2520Stat.%2520Assoc.%26date%3D1963%26volume%3D58%26spage%3D236%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">R Development Core Team</span> R: A Language and Environment for Statistical Computing; <span class="NLM_publisher-name">R Foundation for Statistical Computing</span>: <span class="NLM_publisher-loc">Vienna, Austria,</span><span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=R+Development+Core+Team+R%3A+A+Language+and+Environment+for+Statistical+Computing%3B+R+Foundation+for+Statistical+Computing%3A+Vienna%2C+Austria%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3D%2520R%253A%2520A%2520Language%2520and%2520Environment%2520for%2520Statistical%2520Computing%26pub%3DR%2520Foundation%2520for%2520Statistical%2520Computing%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Cline, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smoot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerami, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuchinsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landys, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christmas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avila-Campilo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creech, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanspers, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isserlin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Killcoyne, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lotia, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maere, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ono, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavlovic, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pico, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vailaya, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conklin, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hood, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuiper, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmulevich, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwikowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ideker, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bader, G. D.</span><span> </span><span class="NLM_article-title">Integration of biological networks and gene expression data using Cytoscape</span> <span class="citation_source-journal">Nat. Protocols</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">2366</span><span class="NLM_x">–</span> <span class="NLM_lpage">2382</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2366-2382&author=M.+S.+Clineauthor=M.+Smootauthor=E.+Ceramiauthor=A.+Kuchinskyauthor=N.+Landysauthor=C.+Workmanauthor=R.+Christmasauthor=I.+Avila-Campiloauthor=M.+Creechauthor=B.+Grossauthor=K.+Hanspersauthor=R.+Isserlinauthor=R.+Kelleyauthor=S.+Killcoyneauthor=S.+Lotiaauthor=S.+Maereauthor=J.+Morrisauthor=K.+Onoauthor=V.+Pavlovicauthor=A.+R.+Picoauthor=A.+Vailayaauthor=P.-L.+Wangauthor=A.+Adlerauthor=B.+R.+Conklinauthor=L.+Hoodauthor=M.+Kuiperauthor=C.+Sanderauthor=I.+Schmulevichauthor=B.+Schwikowskiauthor=G.+J.+Warnerauthor=T.+Idekerauthor=G.+D.+Bader&title=Integration+of+biological+networks+and+gene+expression+data+using+Cytoscape"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCline%26aufirst%3DM.%2BS.%26aulast%3DSmoot%26aufirst%3DM.%26aulast%3DCerami%26aufirst%3DE.%26aulast%3DKuchinsky%26aufirst%3DA.%26aulast%3DLandys%26aufirst%3DN.%26aulast%3DWorkman%26aufirst%3DC.%26aulast%3DChristmas%26aufirst%3DR.%26aulast%3DAvila-Campilo%26aufirst%3DI.%26aulast%3DCreech%26aufirst%3DM.%26aulast%3DGross%26aufirst%3DB.%26aulast%3DHanspers%26aufirst%3DK.%26aulast%3DIsserlin%26aufirst%3DR.%26aulast%3DKelley%26aufirst%3DR.%26aulast%3DKillcoyne%26aufirst%3DS.%26aulast%3DLotia%26aufirst%3DS.%26aulast%3DMaere%26aufirst%3DS.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DOno%26aufirst%3DK.%26aulast%3DPavlovic%26aufirst%3DV.%26aulast%3DPico%26aufirst%3DA.%2BR.%26aulast%3DVailaya%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DP.-L.%26aulast%3DAdler%26aufirst%3DA.%26aulast%3DConklin%26aufirst%3DB.%2BR.%26aulast%3DHood%26aufirst%3DL.%26aulast%3DKuiper%26aufirst%3DM.%26aulast%3DSander%26aufirst%3DC.%26aulast%3DSchmulevich%26aufirst%3DI.%26aulast%3DSchwikowski%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DG.%2BJ.%26aulast%3DIdeker%26aufirst%3DT.%26aulast%3DBader%26aufirst%3DG.%2BD.%26atitle%3DIntegration%2520of%2520biological%2520networks%2520and%2520gene%2520expression%2520data%2520using%2520Cytoscape%26jtitle%3DNat.%2520Protocols%26date%3D2007%26volume%3D2%26spage%3D2366%26epage%3D2382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span>Spotfire, 4.0; <span class="NLM_publisher-name">TIBCO</span>: <span class="NLM_publisher-loc">Somerville, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Spotfire%2C+4.0%3B+TIBCO%3A+Somerville%2C+MA%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSpotfire%252C%25204.0%26pub%3DTIBCO%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapland, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoorn, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Global mapping of pharmacological space</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">805</span><span class="NLM_x">–</span> <span class="NLM_lpage">15</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-15&author=G.+V.+Paoliniauthor=R.+H.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+mapping+of+pharmacological+space"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0lhf3VtA22muNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520mapping%2520of%2520pharmacological%2520space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Chemical fragments as foundations for understanding target space and activity prediction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2689</span><span class="NLM_x">–</span> <span class="NLM_lpage">2700</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701399f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2689-2700&author=J.+J.+Sutherlandauthor=R.+E.+Higgsauthor=I.+Watsonauthor=M.+Vieth&title=Chemical+fragments+as+foundations+for+understanding+target+space+and+activity+prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm701399f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701399f%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DWatson%26aufirst%3DI.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DChemical%2520fragments%2520as%2520foundations%2520for%2520understanding%2520target%2520space%2520and%2520activity%2520prediction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2689%26epage%3D2700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lhf3VtA22muNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">–</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lhf3VtA22muNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gragg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmerle, H.</span><span> </span><span class="NLM_article-title">Kinomics-structural biology and chemogenomics of kinase inhibitors and targets</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1697</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">257</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1697&publication_year=2004&pages=243-257&author=M.+Viethauthor=R.+E.+Higgsauthor=D.+H.+Robertsonauthor=M.+Shapiroauthor=E.+A.+Graggauthor=H.+Hemmerle&title=Kinomics-structural+biology+and+chemogenomics+of+kinase+inhibitors+and+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DShapiro%26aufirst%3DM.%26aulast%3DGragg%26aufirst%3DE.%2BA.%26aulast%3DHemmerle%26aufirst%3DH.%26atitle%3DKinomics-structural%2520biology%2520and%2520chemogenomics%2520of%2520kinase%2520inhibitors%2520and%2520targets%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2004%26volume%3D1697%26spage%3D243%26epage%3D257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Benedict W. J. Irwin, Julian R. Levell, Thomas M. Whitehead, Matthew D. Segall, <span class="NLM_string-name hlFld-ContribAuthor">Gareth J. Conduit</span>. </span><span class="cited-content_cbyCitation_article-title">Practical Applications of Deep Learning To Impute Heterogeneous Drug Discovery Data. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (6)
                                     , 2848-2857. <a href="https://doi.org/10.1021/acs.jcim.0c00443" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00443</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00443%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPractical%252BApplications%252Bof%252BDeep%252BLearning%252BTo%252BImpute%252BHeterogeneous%252BDrug%252BDiscovery%252BData%26aulast%3DIrwin%26aufirst%3DBenedict%2BW.%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D28042020%26date%3D10062020%26date%3D01062020%26volume%3D60%26issue%3D6%26spage%3D2848%26epage%3D2857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yadi Zhou, Suntara Cahya, Steven A. Combs, Christos A. Nicolaou, Jibo Wang, Prashant V. Desai, <span class="NLM_string-name hlFld-ContribAuthor">Jie Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Tunable Hyperparameters for Deep Neural Networks with Industrial ADME Data Sets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1005-1016. <a href="https://doi.org/10.1021/acs.jcim.8b00671" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00671</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00671%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DExploring%252BTunable%252BHyperparameters%252Bfor%252BDeep%252BNeural%252BNetworks%252Bwith%252BIndustrial%252BADME%252BData%252BSets%26aulast%3DZhou%26aufirst%3DYadi%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D30092018%26date%3D11012019%26date%3D26122018%26volume%3D59%26issue%3D3%26spage%3D1005%26epage%3D1016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">T. M. Whitehead, B. W. J. Irwin, P. Hunt, M. D. Segall, <span class="NLM_string-name hlFld-ContribAuthor">G. J. Conduit</span>. </span><span class="cited-content_cbyCitation_article-title">Imputation of Assay Bioactivity Data Using Deep Learning. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1197-1204. <a href="https://doi.org/10.1021/acs.jcim.8b00768" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00768</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00768%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImputation%252Bof%252BAssay%252BBioactivity%252BData%252BUsing%252BDeep%252BLearning%26aulast%3DWhitehead%26aufirst%3DT.%2BM.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D31102018%26date%3D21022019%26date%3D12022019%26volume%3D59%26issue%3D3%26spage%3D1197%26epage%3D1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christos A. Nicolaou, Christine Humblet, Hong Hu, Eva M. Martin, Frank C. Dorsey, Thomas M. Castle, Keith Ian Burton, Haitao Hu, Jorg Hendle, Michael J. Hickey, Joel Duerksen, Jibo Wang, <span class="NLM_string-name hlFld-ContribAuthor">Jon A. Erickson</span>. </span><span class="cited-content_cbyCitation_article-title">Idea2Data: Toward a New Paradigm for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (3)
                                     , 278-286. <a href="https://doi.org/10.1021/acsmedchemlett.8b00488" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00488</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00488%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdea2Data%25253A%252BToward%252Ba%252BNew%252BParadigm%252Bfor%252BDrug%252BDiscovery%26aulast%3DNicolaou%26aufirst%3DChristos%2BA.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D18102018%26date%3D04022019%26date%3D18022019%26date%3D04022019%26volume%3D10%26issue%3D3%26spage%3D278%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric J.  Martin</span>, <span class="hlFld-ContribAuthor ">Valery R.  Polyakov</span>, <span class="hlFld-ContribAuthor ">Li  Tian</span>, and <span class="hlFld-ContribAuthor ">Rolando C.  Perez</span>  . </span><span class="cited-content_cbyCitation_article-title">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (8)
                                     , 2077-2088. <a href="https://doi.org/10.1021/acs.jcim.7b00166" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00166</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00166%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProfile-QSAR%252B2.0%25253A%252BKinase%252BVirtual%252BScreening%252BAccuracy%252BComparable%252Bto%252BFour-Concentration%252BIC50s%252Bfor%252BRealistically%252BNovel%252BCompounds%26aulast%3DMartin%26aufirst%3DEric%2BJ.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D22032017%26date%3D26072017%26date%3D28082017%26date%3D26062017%26volume%3D57%26issue%3D8%26spage%3D2077%26epage%3D2088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olga A.  Tarasova</span>, <span class="hlFld-ContribAuthor ">Aleksandra F.  Urusova</span>, <span class="hlFld-ContribAuthor ">Dmitry A.  Filimonov</span>, <span class="hlFld-ContribAuthor ">Marc C.  Nicklaus</span>, <span class="hlFld-ContribAuthor ">Alexey V.  Zakharov</span>, and <span class="hlFld-ContribAuthor ">Vladimir V.  Poroikov</span>  . </span><span class="cited-content_cbyCitation_article-title">QSAR Modeling Using Large-Scale Databases: Case Study for HIV-1 Reverse Transcriptase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (7)
                                     , 1388-1399. <a href="https://doi.org/10.1021/acs.jcim.5b00019" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00019%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DQSAR%252BModeling%252BUsing%252BLarge-Scale%252BDatabases%25253A%252BCase%252BStudy%252Bfor%252BHIV-1%252BReverse%252BTranscriptase%252BInhibitors%26aulast%3DTarasova%26aufirst%3DOlga%2BA.%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D13012015%26date%3D29062015%26date%3D27072015%26date%3D05062015%26volume%3D55%26issue%3D7%26spage%3D1388%26epage%3D1399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Shen</span>, <span class="hlFld-ContribAuthor ">Christos A.  Nicolaou</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular property prediction: recent trends in the era of artificial intelligence. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2019,</strong> <em>32-33 </em>, 29-36. <a href="https://doi.org/10.1016/j.ddtec.2020.05.001" title="DOI URL">https://doi.org/10.1016/j.ddtec.2020.05.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2020.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2020.05.001%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DMolecular%252Bproperty%252Bprediction%25253A%252Brecent%252Btrends%252Bin%252Bthe%252Bera%252Bof%252Bartificial%252Bintelligence%26aulast%3DShen%26aufirst%3DJie%26date%3D2019%26volume%3D32-33%26spage%3D29%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dilip  Narayanan</span>, <span class="hlFld-ContribAuthor ">Osman A. B. S. M.  Gani</span>, <span class="hlFld-ContribAuthor ">Franz X. E.  Gruber</span>, <span class="hlFld-ContribAuthor ">Richard A.  Engh</span>. </span><span class="cited-content_cbyCitation_article-title">Data driven polypharmacological drug design for lung cancer: analyses for targeting ALK, MET, and EGFR. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2017,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-017-0229-8" title="DOI URL">https://doi.org/10.1186/s13321-017-0229-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-017-0229-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-017-0229-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DData%252Bdriven%252Bpolypharmacological%252Bdrug%252Bdesign%252Bfor%252Blung%252Bcancer%25253A%252Banalyses%252Bfor%252Btargeting%252BALK%25252C%252BMET%25252C%252Band%252BEGFR%26aulast%3DNarayanan%26aufirst%3DDilip%26date%3D2017%26date%3D2017%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Martin</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Richard A.  Engh</span>, <span class="hlFld-ContribAuthor ">Henrik  Moebitz</span>, <span class="hlFld-ContribAuthor ">Thibault  Varin</span>, <span class="hlFld-ContribAuthor ">Benoit  Roux</span>, <span class="hlFld-ContribAuthor ">Jens  Meiler</span>, <span class="hlFld-ContribAuthor ">Valerio  Berdini</span>, <span class="hlFld-ContribAuthor ">Alexander  Baumann</span>, <span class="hlFld-ContribAuthor ">Michal  Vieth</span>. </span><span class="cited-content_cbyCitation_article-title">Perspective on computational and structural aspects of kinase discovery from IPK2014. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</span><span> <strong>2015,</strong> <em>1854 </em>
                                    (10)
                                     , 1595-1604. <a href="https://doi.org/10.1016/j.bbapap.2015.03.014" title="DOI URL">https://doi.org/10.1016/j.bbapap.2015.03.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbapap.2015.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbapap.2015.03.014%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Proteins%2520and%2520Proteomics%26atitle%3DPerspective%252Bon%252Bcomputational%252Band%252Bstructural%252Baspects%252Bof%252Bkinase%252Bdiscovery%252Bfrom%252BIPK2014%26aulast%3DMartin%26aufirst%3DEric%26date%3D2015%26volume%3D1854%26issue%3D10%26spage%3D1595%26epage%3D1604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thibault  Varin</span>, <span class="hlFld-ContribAuthor ">Alexander G.  Godfrey</span>, <span class="hlFld-ContribAuthor ">Thierry  Masquelin</span>, <span class="hlFld-ContribAuthor ">Christos A.  Nicolaou</span>, <span class="hlFld-ContribAuthor ">David A.  Evans</span>, <span class="hlFld-ContribAuthor ">Michal  Vieth</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of selective RIO2 kinase small molecule ligand. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</span><span> <strong>2015,</strong> <em>1854 </em>
                                    (10)
                                     , 1630-1636. <a href="https://doi.org/10.1016/j.bbapap.2015.04.006" title="DOI URL">https://doi.org/10.1016/j.bbapap.2015.04.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbapap.2015.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbapap.2015.04.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Proteins%2520and%2520Proteomics%26atitle%3DDiscovery%252Bof%252Bselective%252BRIO2%252Bkinase%252Bsmall%252Bmolecule%252Bligand%26aulast%3DVarin%26aufirst%3DThibault%26date%3D2015%26volume%3D1854%26issue%3D10%26spage%3D1630%26epage%3D1636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark  Minie</span>, <span class="hlFld-ContribAuthor ">Gaurav  Chopra</span>, <span class="hlFld-ContribAuthor ">Geetika  Sethi</span>, <span class="hlFld-ContribAuthor ">Jeremy  Horst</span>, <span class="hlFld-ContribAuthor ">George  White</span>, <span class="hlFld-ContribAuthor ">Ambrish  Roy</span>, <span class="hlFld-ContribAuthor ">Kaushik  Hatti</span>, <span class="hlFld-ContribAuthor ">Ram  Samudrala</span>. </span><span class="cited-content_cbyCitation_article-title">CANDO and the infinite drug discovery frontier. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2014,</strong> <em>19 </em>
                                    (9)
                                     , 1353-1363. <a href="https://doi.org/10.1016/j.drudis.2014.06.018" title="DOI URL">https://doi.org/10.1016/j.drudis.2014.06.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2014.06.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2014.06.018%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DCANDO%252Band%252Bthe%252Binfinite%252Bdrug%252Bdiscovery%252Bfrontier%26aulast%3DMinie%26aufirst%3DMark%26date%3D2014%26volume%3D19%26issue%3D9%26spage%3D1353%26epage%3D1363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. cMSR and <i>R</i><sup>2</sup> values in the presence of varying bias between a set of predicted and observed values. The green line is the identity line, whereas the red line represents the best fitting line. Where there is (a) no bias, (b) constant bias (i.e., the prediction is consistently under or over predicted across the range of predicted values), and (c) nonconstant bias (i.e., an overestimation at lower end of prediction values and an under estimation at the higher end). In all cases, the <i>R</i><sup>2</sup> value is consistent at around 0.9. However, from a concordance standpoint, panel a is the most desirable outcome. In contrast to <i>R</i><sup>2</sup>, the cMSR reflects the potential bias in the data more accurately.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Heat maps for 29 of the 32 compounds tested in at least 25 targets. Empty values are shown in gray. Targets in columns are arranged in alphabetical order. Pred(LLY), Exp(LLY), and Exp(Metz) are shown in the left, middle, and right panels, respectively. (b) Difference maps between Exp (LLY) and Pred (LLY) (left side) and Exp (LLY) and Exp(Metz) (right). Empty values are shown in gray. Per-compound cMSR values are listed on the map. Deeper purple indicates larger differences in activity between two sources, whereas lighter purple means smaller difference. In both figures, compounds are sorted by per-compound cMSR between Exp(LLY) and Pred(LLY). (c) Scatter plot showing the log<sub>10</sub> fold difference in activity of Pred(LLY)–Exp(LLY) versus Exp(Metz)–Exp(LLY). Kinase-inhibitor pairs that are within 3-fold in different data sources are colored accordingly: red for Pred(LLY)–Exp(LLY) and blue for Exp(Metz)–Exp(LLY). Of the total 1466 Exp(LLY) data, 777 (or 53%) are within 3–fold of both Exp(Metz) and Pred(LLY). These are colored magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Per-target log<sub>10</sub>(cMSR) distribution between sources (68 targets included with more than 10 common compounds values). Geometric means are shown for the cMSR for each pair of sources. Bars are colored by kinase Sugen groups.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Additional details are provided in <a class="ref internalNav" href="#notes-1" aria-label="Figure S9">Figure S9</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation of cross-validation cMSR and test set cMSR for (a) Lilly and (b) Metz data sets. Models with cross-validation cMSR <10 are shown in red, models with test set cMSR <10 are shown in blue, and models with cMSR <10 in both the cross-validation and test set are shown in magenta. More detailed analysis of cross-validation results can be found in <a class="ref internalNav" href="#notes-1" aria-label="Figure S3">Figure S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/medium/jm-2013-00798j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Bioprint-driven target dendrograms for predicted Lilly (top), experimental Lilly (middle), and experimental Metz (bottom) profiling data sets. Sixty eight targets are colored by kinase subfamilies:<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> AGC, red; CAMK, magenta; CMGC, green; TK, blue; TKL, cyan; other, yellow. The target size is proportional to the number of the 29 compounds with <500 nM activity. The value varies from 0 to 15 for Exp(LLY) and 12 for Exp(Metz) data and from 0 to 7 for the predicted Lilly data. The underlying correlation between the target–target Tanimoto similarity matrix indicates a higher concordance between the Exp(LLY) and Pred(LLY) dendrogram (<i>R</i><sup>2 </sup>= 0.69) and lower concordance between the Exp(LLY) and Exp(Metz) dendrogram (<i>R</i><sup>2</sup> = 0.41). <a class="ref internalNav" href="#notes-1" aria-label="Figure S10">Figure S10</a> presents the scatter plots and data distributions for the target–target comparison.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-17/jm400798j/production/images/large/jm-2013-00798j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400798j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i20">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03683" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03683" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 27 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0lhI43b5Nq6wag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, K.</span><span> </span><span class="NLM_article-title">Assessment of chemical coverage of kinome space and its implications for kinase drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7898</span><span class="NLM_x">–</span> <span class="NLM_lpage">7914</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8011036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7898-7914&author=P.+Bamboroughauthor=D.+Drewryauthor=G.+Harperauthor=G.+K.+Smithauthor=K.+Schneider&title=Assessment+of+chemical+coverage+of+kinome+space+and+its+implications+for+kinase+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Chemical Coverage of Kinome Space and Its Implications for Kinase Drug Discovery</span></div><div class="casAuthors">Bamborough, Paul; Drewry, David; Harper, Gavin; Smith, Gary K.; Schneider, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7914</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">More than 500 compds. chosen to represent kinase inhibitor space have been screened against a panel of over 200 protein kinases.  Significant results include the identification of hits against new kinases including PIM1 and MPSK1, and the expansion of the inhibition profiles of several literature compds.  A detailed anal. of the data through the use of affinity fingerprints has produced findings with implications for biol. target selection, the choice of tool compds. for target validation, and lead discovery and optimization.  In a detailed examn. of the tyrosine kinases, interesting relationships have been found between targets and compds.  Taken together, these results show how broad cross-profiling can provide important insights to assist kinase drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBRI-arQk3t7Vg90H21EOLACvtfcHk0ljF6Jn7fh-21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI&md5=8d98c80fb157b987a19e32a1efb53103</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm8011036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8011036%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDrewry%26aufirst%3DD.%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DG.%2BK.%26aulast%3DSchneider%26aufirst%3DK.%26atitle%3DAssessment%2520of%2520chemical%2520coverage%2520of%2520kinome%2520space%2520and%2520its%2520implications%2520for%2520kinase%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7898%26epage%3D7914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettles, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span> </span><span class="NLM_article-title">“Bayes affinity fingerprints” improve retrieval rates in virtual screening and define orthogonal bioactivity space: When are multitarget drugs a feasible concept?</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">2445</span><span class="NLM_x">–</span> <span class="NLM_lpage">2456</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600197y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1yqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=2445-2456&author=A.+Benderauthor=J.+L.+Jenkinsauthor=M.+Glickauthor=Z.+Dengauthor=J.+H.+Nettlesauthor=J.+W.+Davies&title=%E2%80%9CBayes+affinity+fingerprints%E2%80%9D+improve+retrieval+rates+in+virtual+screening+and+define+orthogonal+bioactivity+space%3A+When+are+multitarget+drugs+a+feasible+concept%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">"Bayes Affinity Fingerprints" Improve Retrieval Rates in Virtual Screening and Define Orthogonal Bioactivity Space: When Are Multitarget Drugs a Feasible Concept?</span></div><div class="casAuthors">Bender, Andreas; Jenkins, Jeremy L.; Glick, Meir; Deng, Zhan; Nettles, James H.; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2445-2456</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conventional similarity searching of mols. compares single (or multiple) active query structures to each other in a relative framework, by means of a structural descriptor and a similarity measure.  While this often works well, depending on the target, the authors show here that retrieval rates can be improved considerably by incorporating an external framework describing ligand bioactivity space for comparisons ("Bayes affinity fingerprints").  Structures are described by Bayes scores for a ligand panel comprising about 1000 activity classes extd. from the WOMBAT database.  The comparison of structures is performed via the Pearson correlation coeff. of activity classes, i.e., the order in which two structures are similar to the panel activity classes.  Compd. retrieval on a recently published data set could be improved by as much as 24% relative (9% abs.).  Knowledge about the shape of the "bioactive chem. universe" is thus beneficial to identifying similar bioactivities.  Principal component anal. was employed to further analyze activity space with the objective to define orthogonal ligand bioactive chem. space, leading to nine major (roughly orthogonal) activity axes.  Employing only those nine activity classes, retrieval rates are still comparable to original Bayes affinity fingerprints; thus, the concept of orthogonal bioactive ligand chem. space was validated as being an information-rich but low-dimensional representation of bioactivity space.  Correlations between activity classes are a major determinant to gauge whether the desired multitarget activity of drugs is (on the basis of current knowledge) a feasible concept because it measures the extent to which activities can be optimized independently, or only by strongly influencing one another.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6IjlYwYDTwLVg90H21EOLACvtfcHk0ljF6Jn7fh-21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1yqsrs%253D&md5=47adb720154f45bda1738963d54c70a5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fci600197y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600197y%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DNettles%26aufirst%3DJ.%2BH.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3D%25E2%2580%259CBayes%2520affinity%2520fingerprints%25E2%2580%259D%2520improve%2520retrieval%2520rates%2520in%2520virtual%2520screening%2520and%2520define%2520orthogonal%2520bioactivity%2520space%253A%2520When%2520are%2520multitarget%2520drugs%2520a%2520feasible%2520concept%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D2445%26epage%3D2456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serrano, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemons, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J.</span><span> </span><span class="NLM_article-title">Chemogenomic data analysis: Prediction of small-molecule targets and the advent of biological fingerprints</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">719</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.2174%2F138620707782507313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=18045083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=719-731&author=A.+Benderauthor=D.+Youngauthor=J.+Jenkinsauthor=M.+Serranoauthor=D.+Mikhailovauthor=P.+Clemonsauthor=J.+Davies&title=Chemogenomic+data+analysis%3A+Prediction+of+small-molecule+targets+and+the+advent+of+biological+fingerprints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic data analysis: prediction of small-molecule targets and the advent of biological fingerprints</span></div><div class="casAuthors">Bender, Andreas; Young, Daniel W.; Jenkins, Jeremy L.; Serrano, Martin; Mikhailov, Dmitri; Clemons, Paul A.; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">719-731</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Chemogenomics comprises a systematic relationship between targets and ligands that are used as target modulators in living systems such as cells or organisms.  In recent years, data on small mol.-bioactivity relationships have become increasingly available, and consequently so have the no. of approaches used to translate bioactivity data into knowledge.  This review will focus on 2 aspects of chemogenomics.  Firstly, in cases such as cell-based screens, the question of which target(s) a compd. is modulating to cause the obsd. phenotype is crucial.  In silico target prediction tools can suggest likely biol. targets of small mols. via data mining in target-annotated chem. databases.  This review presents some of the current tools available for this task and shows some sample applications relevant to a pharmaceutical industry setting.  These applications are the prediction of false-positives in cell-based reporter gene assays, the prediction of targets by linking bioassay data with protein domain annotations, and the direct prediction of adverse reactions.  Secondly, in recent years a shift from structure-derived chem. descriptors to biol. descriptors has occurred.  Here, the effect of a compd. on a no. of biol. endpoints is used to make predictions about other properties, such as putative targets, assocd. adverse reactions, and pathways modulated by the compd.  This review further summarizes these "performance" descriptors and their applications, focusing on gene expression profiles and high-content screening data.  The advent of such biol. fingerprints suggests that the field of drug discovery is currently at a crossroads, where single target bioassay results are supplanted by multidimensional biol. fingerprints that reflect a new awareness of biol. networks and polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJuaT0KRiZF7Vg90H21EOLACvtfcHk0ljF6Jn7fh-21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOhsg%253D%253D&md5=4b3152898f48cc7021904a0c983f6068</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F138620707782507313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620707782507313%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DD.%26aulast%3DJenkins%26aufirst%3DJ.%26aulast%3DSerrano%26aufirst%3DM.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DClemons%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DJ.%26atitle%3DChemogenomic%2520data%2520analysis%253A%2520Prediction%2520of%2520small-molecule%2520targets%2520and%2520the%2520advent%2520of%2520biological%2520fingerprints%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2007%26volume%3D10%26spage%3D719%26epage%3D731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urban, L.</span><span> </span><span class="NLM_article-title">Large-scale prediction and testing of drug activity on side-effect targets</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">486</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnature11159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=22722194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=361-367&author=E.+Lounkineauthor=M.+J.+Keiserauthor=S.+Whitebreadauthor=D.+Mikhailovauthor=J.+Hamonauthor=J.+L.+Jenkinsauthor=P.+Lavanauthor=E.+Weberauthor=A.+K.+Doakauthor=S.+Coteauthor=B.+K.+Shoichetauthor=L.+Urban&title=Large-scale+prediction+and+testing+of+drug+activity+on+side-effect+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale prediction and testing of drug activity on side-effect targets</span></div><div class="casAuthors">Lounkine, Eugen; Keiser, Michael J.; Whitebread, Steven; Mikhailov, Dmitri; Hamon, Jacques; Jenkins, Jeremy L.; Lavan, Paul; Weber, Eckhard; Doak, Allison K.; Cote, Serge; Shoichet, Brian K.; Urban, Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">486</span>
        (<span class="NLM_cas:issue">7403</span>),
    <span class="NLM_cas:pages">361-367</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Discovering the unintended off-targets' that predict adverse drug reactions is daunting by empirical methods alone.  Drugs can act on several protein targets, some of which can be unrelated by conventional mol. metrics, and hundreds of proteins have been implicated in side effects.  Here we use a computational strategy to predict the activity of 656 marketed drugs on 73 unintended side-effect' targets.  Approx. half of the predictions were confirmed, either from proprietary databases unknown to the method or by new exptl. assays.  Affinities for these new off-targets ranged from 1 nM to 30 μM.  To explore relevance, we developed an assocn. metric to prioritize those new off-targets that explained side effects better than any known target of a given drug, creating a drug-target-adverse drug reaction network.  Among these new assocns. was the prediction that the abdominal pain side effect of the synthetic estrogen chlorotrianisene was mediated through its newly discovered inhibition of the enzyme cyclooxygenase-1.  The clin. relevance of this inhibition was borne out in whole human blood platelet aggregation assays.  This approach may have wide application to de-risking toxicol. liabilities in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5sVjfn4mgLLVg90H21EOLACvtfcHk0lgMtwsEqQOjHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D&md5=9c133b839d9527818a27a5834e4dfe11</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature11159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11159%26sid%3Dliteratum%253Aachs%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DLavan%26aufirst%3DP.%26aulast%3DWeber%26aufirst%3DE.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DCote%26aufirst%3DS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DUrban%26aufirst%3DL.%26atitle%3DLarge-scale%2520prediction%2520and%2520testing%2520of%2520drug%2520activity%2520on%2520side-effect%2520targets%26jtitle%3DNature%26date%3D2012%26volume%3D486%26spage%3D361%26epage%3D367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Krejsa, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogalski, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penzotti, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Migeon, J. C.</span><span> </span><span class="NLM_article-title">Predicting ADME properties and side effects: The BioPrint approach</span> <span class="citation_source-journal">Curr. Opin. Drug Discov. Devel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=12951810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFymsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2003&pages=470-480&author=C.+M.+Krejsaauthor=D.+Horvathauthor=S.+L.+Rogalskiauthor=J.+E.+Penzottiauthor=B.+Maoauthor=F.+Barbosaauthor=J.+C.+Migeon&title=Predicting+ADME+properties+and+side+effects%3A+The+BioPrint+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting ADME properties and side effects: The BioPrint approach</span></div><div class="casAuthors">Krejsa, Cecile M.; Horvath, Dragos; Rogalski, Sherri L.; Penzotti, Julie E.; Mao, Boryeu; Barbosa, Frederique; Migeon, Jacques C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">470-480</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Current Drugs</span>)
        </div><div class="casAbstract">A review.  Computational methods are increasingly used to streamline and enhance the lead discovery and optimization process.  However, accurate prediction of absorption, distribution, metab. and excretion (ADME) and adverse drug reactions (ADR) is often difficult, due to the complexity of underlying physiol. mechanisms.  Modeling approaches have been hampered by the lack of large, robust and standardized training datasets.  In an extensive effort to build such a dataset, the BioPrint database was constructed by systematic profiling of nearly all drugs available on the market, as well as numerous ref. compds.  The database is composed of several large datasets: compd. structures and mol. descriptors, in vitro ADME and pharmacol. profiles, and complementary clin. data including therapeutic use information, pharmacokinetics profiles and ADR profiles.  These data have allowed the development of computational tools designed to integrate a program of computational chem. into library design and lead development.  Models based on chem. structure are strengthened by in vitro results that can be used as addnl. compd. descriptors to predict complex in vivo endpoints.  The BioPrint pharmacoinformatics platform represents a systematic effort to accelerate the process of drug discovery, improve quant. structure-activity relationships and develop in vitro/in vivo assocns.  In this review, we will discuss the importance of training set size and diversity in model development, the implementation of linear and neighborhood modeling approaches, and the use of in silico methods to predict potential clin. liabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQ71juUoErrVg90H21EOLACvtfcHk0lgMtwsEqQOjHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFymsLw%253D&md5=f8db4e37cc895d67bf4a82efe021c5c2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrejsa%26aufirst%3DC.%2BM.%26aulast%3DHorvath%26aufirst%3DD.%26aulast%3DRogalski%26aufirst%3DS.%2BL.%26aulast%3DPenzotti%26aufirst%3DJ.%2BE.%26aulast%3DMao%26aufirst%3DB.%26aulast%3DBarbosa%26aufirst%3DF.%26aulast%3DMigeon%26aufirst%3DJ.%2BC.%26atitle%3DPredicting%2520ADME%2520properties%2520and%2520side%2520effects%253A%2520The%2520BioPrint%2520approach%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discov.%2520Devel.%26date%3D2003%26volume%3D6%26spage%3D470%26epage%3D480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruda, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abecassis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguiz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norval, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeons, F. R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidah, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constam, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetsel, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.-P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+C.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lgMtwsEqQOjHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%2BC.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Ning, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karypis, G.</span><span> </span><span class="NLM_article-title">Improved machine learning models for predicting selective compounds</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">38</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci200346b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2kt77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=38-50&author=X.+Ningauthor=M.+Waltersauthor=G.+Karypis&title=Improved+machine+learning+models+for+predicting+selective+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Machine Learning Models for Predicting Selective Compounds</span></div><div class="casAuthors">Ning, Xia; Walters, Michael; Karypis, George</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-50</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of small potent compds. that selectively bind to the target under consideration with high affinities is a crit. step toward successful drug discovery.  However, there is still a lack of efficient and accurate computational methods to predict compd. selectivity properties.  In this paper, we propose a set of machine learning methods to do compd. selectivity prediction.  In particular, we propose a novel cascaded learning method and a multitask learning method.  The cascaded method decomps. the selectivity prediction into two steps, one model for each step, so as to effectively filter out nonselective compds.  The multitask method incorporates both activity and selectivity models into one multitask model so as to better differentiate compd. selectivity properties.  We conducted a comprehensive set of expts. and compared the results with those of other conventional selectivity prediction methods, and our results demonstrated that the cascaded and multitask methods significantly improve the selectivity prediction performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdkf0ZGDNBm7Vg90H21EOLACvtfcHk0li8SVd-GgMvDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2kt77I&md5=2038119451a4a486b77457f966276e5e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci200346b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200346b%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DX.%26aulast%3DWalters%26aufirst%3DM.%26aulast%3DKarypis%26aufirst%3DG.%26atitle%3DImproved%2520machine%2520learning%2520models%2520for%2520predicting%2520selective%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D38%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Martin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansen, J.</span><span> </span><span class="NLM_article-title">Profile-QSAR: A novel meta-QSAR method that combines activities across the kinase family to accurately predict affinity, selectivity, and cellular activity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1942</span><span class="NLM_x">–</span> <span class="NLM_lpage">1956</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci1005004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1942-1956&author=E.+Martinauthor=P.+Mukherjeeauthor=D.+Sullivanauthor=J.+Jansen&title=Profile-QSAR%3A+A+novel+meta-QSAR+method+that+combines+activities+across+the+kinase+family+to+accurately+predict+affinity%2C+selectivity%2C+and+cellular+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity</span></div><div class="casAuthors">Martin, Eric; Mukherjee, Prasenjit; Sullivan, David; Jansen, Johanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1942-1956</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-QSAR is a novel 2D predictive model building method for kinases.  This "meta-QSAR" method models the activity of each compd. against a new kinase target as a linear combination of its predicted activities against a large panel of 92 previously studied kinases comprised from 115 assays.  Profile-QSAR starts with a sparse incomplete kinase by compd. (KxC) activity matrix, used to generate Bayesian QSAR models for the 92 "basis-set" kinases.  These Bayesian QSARs generate a complete "synthetic" KxC activity matrix of predictions.  These synthetic activities are used as "chem. descriptors" to train partial-least squares (PLS) models, from modest amts. of medium-throughput screening data, for predicting activity against new kinases.  The Profile-QSAR predictions for the 92 kinases (115 assays) gave a median external R2ext = 0.59 on 25% held-out test sets.  The method has proven accurate enough to predict pairwise kinase selectivities with a median correlation of R2ext = 0.61 for 958 kinase pairs with at least 600 common compds.  It has been further expanded by adding a "CkXC" cellular activity matrix to the KxC matrix to predict cellular activity for 42 kinase driven cellular assays with median R2ext = 0.58 for 24 target modulation assays and R2ext = 0.41 for 18 cell proliferation assays.  The 2D Profile-QSAR, along with the 3D Surrogate AutoShim, are the foundations of an internally developed iterative medium-throughput screening (IMTS) methodol. for virtual screening (VS) of compd. archives as an alternative to exptl. high-throughput screening (HTS).  The method has been applied to 20 actual prospective kinase projects.  Biol. results have so far been obtained in eight of them.  Q2 values ranged from 0.3 to 0.7.  Hit-rates at 10 uM for exptl. tested compds. varied from 25% to 80%, except in K5, which was a special case aimed specifically at finding "type II" binders, where none of the compds. were predicted to be active at 10 μM.  These overall results are particularly striking as chem. novelty was an important criterion in selecting compds. for testing.  The method is completely automated.  Predicted activities for nearly 4 million internal and com. compds. across 115 kinase assays and 42 cellular assays are stored in the corporate database.  Like computed phys. properties, this predicted kinase activity profile can be computed and stored as each compd. is registered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTg6KqlgEyLVg90H21EOLACvtfcHk0li8SVd-GgMvDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D&md5=49422d01fd686a4befc9db7c123cfcce</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1005004%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DSullivan%26aufirst%3DD.%26aulast%3DJansen%26aufirst%3DJ.%26atitle%3DProfile-QSAR%253A%2520A%2520novel%2520meta-QSAR%2520method%2520that%2520combines%2520activities%2520across%2520the%2520kinase%2520family%2520to%2520accurately%2520predict%2520affinity%252C%2520selectivity%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1942%26epage%3D1956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, E.</span><span> </span><span class="NLM_article-title">Profile-QSAR and surrogate AutoShim protein-family modeling of proteases</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2430</span><span class="NLM_x">–</span> <span class="NLM_lpage">2440</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci300059d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WktLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2430-2440&author=P.+Mukherjeeauthor=E.+Martin&title=Profile-QSAR+and+surrogate+AutoShim+protein-family+modeling+of+proteases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR and Surrogate AutoShim Protein-Family Modeling of Proteases</span></div><div class="casAuthors">Mukherjee, Prasenjit; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2430-2440</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 2D Profile-QSAR and 3D Surrogate AutoShim protein-family virtual screening methods were originally developed for kinases.  They are the key components of an iterative medium-throughput screening alternative to expensive and time-consuming exptl. high-throughput screening.  Encouraged by the success with kinases, the S1-serine proteases were selected as a second protein family to tackle, based on the structural and SAR similarity among them, availability of structural and bioactivity data, and the current and future small-mol. drug discovery interest.  A validation study on 24 S1-serine protease assay data sets from 16 unique proteases gave very promising results.  Profile-QSAR gave a median RexT2 = 0.60 for 24 assay data sets, and pairwise selectivity modeling on 60 protease pairs gave a median RexT2 = 0.64, comparable to the performance for kinases.  A 17-structure universal ensemble S1-serine protease surrogate receptor for Autoshim was developed from a collection of ∼1500 x-ray structures.  The predictive performance on 24 S1-serine protease assays was good, with a median RexT2 = 0.41, but lower than had been obtained for kinases.  Anal. suggested that the higher structural diversity of the protease structures, as well as smaller assay data sets and fewer potent compds., both contributed to the decreased predictive power.  In a prospective virtual screening application, 32 compds. were ordered from a 1.5 million archive and tested in a biochem. assay.  Thirteen of the 32 compds. were active at IC50 ≤ 10 μM, a 41% hit-rate.  Three new scaffolds were identified which are being followed up with testing of addnl. analogs.  A SAR similarity anal. for this target against 13 other proteases also indicated two potential protease targets which were pos. and neg. correlated with the activity of the target protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr606feZGrYa7Vg90H21EOLACvtfcHk0li8SVd-GgMvDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WktLY%253D&md5=5f24ff3fcdda7604cbd8177c1079e55e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci300059d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300059d%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DProfile-QSAR%2520and%2520surrogate%2520AutoShim%2520protein-family%2520modeling%2520of%2520proteases%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2430%26epage%3D2440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cahya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">M., V.</span><span> </span><span class="NLM_article-title">What general conclusions can we draw from kinase profiling data sets?</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1834</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1433</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1834&publication_year=2013&pages=1425-1433&author=J.+J.+Sutherlandauthor=C.+Gaoauthor=S.+Cahyaauthor=V.+M.&title=What+general+conclusions+can+we+draw+from+kinase+profiling+data+sets%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DCahya%26aufirst%3DS.%26aulast%3DM.%26aufirst%3DV.%26atitle%3DWhat%2520general%2520conclusions%2520can%2520we%2520draw%2520from%2520kinase%2520profiling%2520data%2520sets%253F%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2013%26volume%3D1834%26spage%3D1425%26epage%3D1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span> </span><span class="NLM_article-title">Three useful dimensions for domain applicability in QSAR models using random forest</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">814</span><span class="NLM_x">–</span> <span class="NLM_lpage">823</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci300004n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=814-823&author=R.+P.+Sheridan&title=Three+useful+dimensions+for+domain+applicability+in+QSAR+models+using+random+forest"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Three Useful Dimensions for Domain Applicability in QSAR Models Using Random Forest</span></div><div class="casAuthors">Sheridan, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">814-823</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One popular metric for estg. the accuracy of prospective quant. structure-activity relationship (QSAR) predictions is based on the similarity of the compd. being predicted to compds. in the training set from which the QSAR model was built.  More recent work in the field has indicated that other parameters might be equally or more important than similarity.  Here we make use of two addnl. parameters: the variation of prediction among random forest trees (less variation among trees indicates more accurate prediction) and the prediction itself (certain ranges of activity are intrinsically easier to predict than others).  The accuracy of prediction for a QSAR model, as measured by the root-mean-square error, can be estd. by cross-validation on the training set at the time of model-building and stored as a three-dimensional array of bins.  This is an obvious extension of the one-dimensional array of bins we previously proposed for similarity to the training set.  We show that using these three parameters simultaneously adds much more discrimination in prediction accuracy than any single parameter.  This approach can be applied to any QSAR method that produces an ensemble of models.  We also show that the root-mean-square errors produced by cross-validation are predictive of root-mean-square errors of compds. tested after the model was built.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoch9QmSOjtCrVg90H21EOLACvtfcHk0li34oxoZX75gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWhsL4%253D&md5=85bb79c95e57ad6a95851b8533024a2d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci300004n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300004n%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26atitle%3DThree%2520useful%2520dimensions%2520for%2520domain%2520applicability%2520in%2520QSAR%2520models%2520using%2520random%2520forest%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D814%26epage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Eastwood, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farmen, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iversen, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craft, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smallwood, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garbison, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delapp, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, G. F.</span><span> </span><span class="NLM_article-title">The minimum significant ratio: A statistical parameter to characterize the reproducibility of potency estimates from concentration-response assays and estimation by replicate-experiment studies</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">253</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1177%2F1087057105285611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=16490778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A280%3ADC%252BD283mvVyhug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=253-261&author=B.+J.+Eastwoodauthor=M.+W.+Farmenauthor=P.+W.+Iversenauthor=T.+J.+Craftauthor=J.+K.+Smallwoodauthor=K.+E.+Garbisonauthor=N.+W.+Delappauthor=G.+F.+Smith&title=The+minimum+significant+ratio%3A+A+statistical+parameter+to+characterize+the+reproducibility+of+potency+estimates+from+concentration-response+assays+and+estimation+by+replicate-experiment+studies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The minimum significant ratio: a statistical parameter to characterize the reproducibility of potency estimates from concentration-response assays and estimation by replicate-experiment studies</span></div><div class="casAuthors">Eastwood Brian J; Farmen Mark W; Iversen Philip W; Craft Trelia J; Smallwood Jeffrey K; Garbison Kim E; Delapp Neil W; Smith Gerald F</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">253-61</span>
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    </div><div class="casAbstract">The authors show by illustration that procedures used to validate the reliability of single-concentration high-throughput screens such as the signal window and Z' factor do not ensure sufficient reliability in potency estimates from concentration response assays.  They develop the minimum significant ratio as a statistical parameter to characterize the fold change between 2 compounds run in the same experiment that can be considered a real difference and use this parameter to characterize the reliability of the assay.  They adapt methods described by Bland and Altman to develop a simple set of 2 experiments to estimate the minimum significant ratio and show that this protocol can identify assays that lack reproducibility.  The methods are then extended to validate the equivalency of the same assay run by multiple laboratories.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTcE509f3H8OukdrW1FkXG-fW6udTcc2eZmiZXZkQxB0Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD283mvVyhug%253D%253D&md5=df12ee03e938b7b16652a0ddd77a1a93</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1177%2F1087057105285611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057105285611%26sid%3Dliteratum%253Aachs%26aulast%3DEastwood%26aufirst%3DB.%2BJ.%26aulast%3DFarmen%26aufirst%3DM.%2BW.%26aulast%3DIversen%26aufirst%3DP.%2BW.%26aulast%3DCraft%26aufirst%3DT.%2BJ.%26aulast%3DSmallwood%26aufirst%3DJ.%2BK.%26aulast%3DGarbison%26aufirst%3DK.%2BE.%26aulast%3DDelapp%26aufirst%3DN.%2BW.%26aulast%3DSmith%26aufirst%3DG.%2BF.%26atitle%3DThe%2520minimum%2520significant%2520ratio%253A%2520A%2520statistical%2520parameter%2520to%2520characterize%2520the%2520reproducibility%2520of%2520potency%2520estimates%2520from%2520concentration-response%2520assays%2520and%2520estimation%2520by%2520replicate-experiment%2520studies%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D253%26epage%3D261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Metz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merta, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kifle, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Navigating the kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+kinome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0li34oxoZX75gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D200%26epage%3D202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erickson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I.</span><span> </span><span class="NLM_article-title">Kinase inhibitor data modeling and de novo inhibitor design with fragment approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6456</span><span class="NLM_x">–</span> <span class="NLM_lpage">6466</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901147e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6456-6466&author=M.+Viethauthor=J.+Ericksonauthor=J.+Wangauthor=Y.+Websterauthor=M.+Maderauthor=R.+Higgsauthor=I.+Watson&title=Kinase+inhibitor+data+modeling+and+de+novo+inhibitor+design+with+fragment+approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm901147e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901147e%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DErickson%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWebster%26aufirst%3DY.%26aulast%3DMader%26aufirst%3DM.%26aulast%3DHiggs%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DI.%26atitle%3DKinase%2520inhibitor%2520data%2520modeling%2520and%2520de%2520novo%2520inhibitor%2520design%2520with%2520fragment%2520approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6456%26epage%3D6466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Erickson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Structure-guided expansion of kinase fragment libraries driven by support vector machine models</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1804</span><span class="NLM_x">, </span> <span class="NLM_fpage">642</span><span class="NLM_x">–</span> <span class="NLM_lpage">652</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1016%2Fj.bbapap.2009.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=20005305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1yms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1804&publication_year=2010&pages=642-652&author=J.+A.+Ericksonauthor=M.+M.+Maderauthor=I.+A.+Watsonauthor=Y.+W.+Websterauthor=R.+E.+Higgsauthor=M.+A.+Bellauthor=M.+Vieth&title=Structure-guided+expansion+of+kinase+fragment+libraries+driven+by+support+vector+machine+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided expansion of kinase fragment libraries driven by support vector machine models</span></div><div class="casAuthors">Erickson, Jon A.; Mader, Mary M.; Watson, Ian A.; Webster, Yue W.; Higgs, Richard E.; Bell, Michael A.; Vieth, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1804</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">642-652</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">This work outlines a new de novo design process for the creation of novel kinase inhibitor libraries.  It relies on a profiling paradigm that generates a substantial amt. of kinase inhibitor data from which highly predictive QSAR models can be constructed.  In addn., a broad diversity of X-ray structure information is needed for binding mode prediction.  This is important for scaffold and substituent site selection.  Borrowing from FBDD, the process involves fragmentation of known actives, proposition of binding mode hypotheses for the fragments, and model-driven recombination using a pharmacophore derived from known kinase inhibitor structures.  The support vector machine method, using Merck atom pair derived fingerprint descriptors, was used to build models from activity from 6 kinase assays.  These models were qualified prospectively by selecting and testing compds. from the internal compd. collection.  Overall hit and enrichment rates of 82% and 2.5%, resp., qualified the models for use in library design.  Using the process, 7 novel libraries were designed, synthesized and tested against these same 6 kinases.  The results showed excellent results, yielding a 92% hit rate for the 179 compds. that made up the 7 libraries.  The results of one library designed to include known literature compds., as well as an anal. of overall substituent frequency, are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF5GYJ7qB3MbVg90H21EOLACvtfcHk0li9FiGth6AoyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1yms7s%253D&md5=87d3531b362dd5ed32889efaf451862b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2009.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2009.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DErickson%26aufirst%3DJ.%2BA.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DWebster%26aufirst%3DY.%2BW.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DBell%26aufirst%3DM.%2BA.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DStructure-guided%2520expansion%2520of%2520kinase%2520fragment%2520libraries%2520driven%2520by%2520support%2520vector%2520machine%2520models%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2010%26volume%3D1804%26spage%3D642%26epage%3D652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Joachims, T.</span><span> </span><span class="NLM_article-title">Making large-scale support vector machine learning practical</span>. In  <span class="citation_source-book">Advances in Kernel Methods</span>; <span class="NLM_contrib-group">Scholkopf, B.; Burges, C. J. C.; Smola, A. J.</span>, Eds.;<span class="NLM_publisher-name"> MIT Press</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">1999</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&pages=169-184&author=T.+Joachimsauthor=B.+Scholkopf&author=C.+J.+C.+Burges&author=A.+J.+Smola&title=Advances+in+Kernel+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJoachims%26aufirst%3DT.%26atitle%3DMaking%2520large-scale%2520support%2520vector%2520machine%2520learning%2520practical%26btitle%3DAdvances%2520in%2520Kernel%2520Methods%26aulast%3DScholkopf%26aufirst%3DB.%26pub%3DMIT%2520Press%26date%3D1999%26spage%3D169%26epage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Doucet, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbault, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panaye, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, B.</span><span> </span><span class="NLM_article-title">Nonlinear SVM approaches to QSPR/QSAR studies and drug design</span> <span class="citation_source-journal">Curr. Comput.-Aided Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">289</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.2174%2F157340907782799372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVSqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=263-289&author=J.-P.+Doucetauthor=F.+Barbaultauthor=H.+Xiaauthor=A.+Panayeauthor=B.+Fan&title=Nonlinear+SVM+approaches+to+QSPR%2FQSAR+studies+and+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Nonlinear SVM approaches to QSPR/QSAR studies and drug design</span></div><div class="casAuthors">Doucet, Jean-Pierre; Barbault, Florent; Xia, Hairong; Panaye, Annick; Fan, Botao</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-289</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Recently, a new promising nonlinear method, the support vector machine (SVM), was proposed by Vapnik.  It rapidly found numerous applications in chem., biochem. and pharmacochem.  Several attempts using SVM in drug design have been reported.  It became an attractive nonlinear approach in this field.  In this review, the theor. basis of SVM in classification and regression is briefly described.  Its applications in QSPR/QSAR studies, and particularly in drug design are discussed.  Comparative studies with some linear and other nonlinear methods show SVM's high performance both in classification and correlation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlUWphlsyC4rVg90H21EOLACvtfcHk0li9FiGth6AoyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVSqtLk%253D&md5=5552439113da40b712c3467d7277bf95</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F157340907782799372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340907782799372%26sid%3Dliteratum%253Aachs%26aulast%3DDoucet%26aufirst%3DJ.-P.%26aulast%3DBarbault%26aufirst%3DF.%26aulast%3DXia%26aufirst%3DH.%26aulast%3DPanaye%26aufirst%3DA.%26aulast%3DFan%26aufirst%3DB.%26atitle%3DNonlinear%2520SVM%2520approaches%2520to%2520QSPR%252FQSAR%2520studies%2520and%2520drug%2520design%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2007%26volume%3D3%26spage%3D263%26epage%3D289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Ward, J. H.,  Jr.</span><span> </span><span class="NLM_article-title">Hierarchical grouping to optimize an objective function</span> <span class="citation_source-journal">J. Am. Stat. Assoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1963</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">236</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1080%2F01621459.1963.10500845" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1963&pages=236-244&author=J.+H.+Ward&title=Hierarchical+grouping+to+optimize+an+objective+function"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1080%2F01621459.1963.10500845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F01621459.1963.10500845%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DJ.%2BH.%26atitle%3DHierarchical%2520grouping%2520to%2520optimize%2520an%2520objective%2520function%26jtitle%3DJ.%2520Am.%2520Stat.%2520Assoc.%26date%3D1963%26volume%3D58%26spage%3D236%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">R Development Core Team</span> R: A Language and Environment for Statistical Computing; <span class="NLM_publisher-name">R Foundation for Statistical Computing</span>: <span class="NLM_publisher-loc">Vienna, Austria,</span><span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=R+Development+Core+Team+R%3A+A+Language+and+Environment+for+Statistical+Computing%3B+R+Foundation+for+Statistical+Computing%3A+Vienna%2C+Austria%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3D%2520R%253A%2520A%2520Language%2520and%2520Environment%2520for%2520Statistical%2520Computing%26pub%3DR%2520Foundation%2520for%2520Statistical%2520Computing%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Cline, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smoot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerami, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuchinsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landys, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christmas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avila-Campilo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creech, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanspers, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isserlin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Killcoyne, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lotia, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maere, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ono, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavlovic, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pico, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vailaya, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conklin, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hood, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuiper, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmulevich, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwikowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ideker, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bader, G. D.</span><span> </span><span class="NLM_article-title">Integration of biological networks and gene expression data using Cytoscape</span> <span class="citation_source-journal">Nat. Protocols</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">2366</span><span class="NLM_x">–</span> <span class="NLM_lpage">2382</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2366-2382&author=M.+S.+Clineauthor=M.+Smootauthor=E.+Ceramiauthor=A.+Kuchinskyauthor=N.+Landysauthor=C.+Workmanauthor=R.+Christmasauthor=I.+Avila-Campiloauthor=M.+Creechauthor=B.+Grossauthor=K.+Hanspersauthor=R.+Isserlinauthor=R.+Kelleyauthor=S.+Killcoyneauthor=S.+Lotiaauthor=S.+Maereauthor=J.+Morrisauthor=K.+Onoauthor=V.+Pavlovicauthor=A.+R.+Picoauthor=A.+Vailayaauthor=P.-L.+Wangauthor=A.+Adlerauthor=B.+R.+Conklinauthor=L.+Hoodauthor=M.+Kuiperauthor=C.+Sanderauthor=I.+Schmulevichauthor=B.+Schwikowskiauthor=G.+J.+Warnerauthor=T.+Idekerauthor=G.+D.+Bader&title=Integration+of+biological+networks+and+gene+expression+data+using+Cytoscape"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCline%26aufirst%3DM.%2BS.%26aulast%3DSmoot%26aufirst%3DM.%26aulast%3DCerami%26aufirst%3DE.%26aulast%3DKuchinsky%26aufirst%3DA.%26aulast%3DLandys%26aufirst%3DN.%26aulast%3DWorkman%26aufirst%3DC.%26aulast%3DChristmas%26aufirst%3DR.%26aulast%3DAvila-Campilo%26aufirst%3DI.%26aulast%3DCreech%26aufirst%3DM.%26aulast%3DGross%26aufirst%3DB.%26aulast%3DHanspers%26aufirst%3DK.%26aulast%3DIsserlin%26aufirst%3DR.%26aulast%3DKelley%26aufirst%3DR.%26aulast%3DKillcoyne%26aufirst%3DS.%26aulast%3DLotia%26aufirst%3DS.%26aulast%3DMaere%26aufirst%3DS.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DOno%26aufirst%3DK.%26aulast%3DPavlovic%26aufirst%3DV.%26aulast%3DPico%26aufirst%3DA.%2BR.%26aulast%3DVailaya%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DP.-L.%26aulast%3DAdler%26aufirst%3DA.%26aulast%3DConklin%26aufirst%3DB.%2BR.%26aulast%3DHood%26aufirst%3DL.%26aulast%3DKuiper%26aufirst%3DM.%26aulast%3DSander%26aufirst%3DC.%26aulast%3DSchmulevich%26aufirst%3DI.%26aulast%3DSchwikowski%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DG.%2BJ.%26aulast%3DIdeker%26aufirst%3DT.%26aulast%3DBader%26aufirst%3DG.%2BD.%26atitle%3DIntegration%2520of%2520biological%2520networks%2520and%2520gene%2520expression%2520data%2520using%2520Cytoscape%26jtitle%3DNat.%2520Protocols%26date%3D2007%26volume%3D2%26spage%3D2366%26epage%3D2382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span>Spotfire, 4.0; <span class="NLM_publisher-name">TIBCO</span>: <span class="NLM_publisher-loc">Somerville, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Spotfire%2C+4.0%3B+TIBCO%3A+Somerville%2C+MA%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSpotfire%252C%25204.0%26pub%3DTIBCO%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapland, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoorn, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Global mapping of pharmacological space</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">805</span><span class="NLM_x">–</span> <span class="NLM_lpage">15</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-15&author=G.+V.+Paoliniauthor=R.+H.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+mapping+of+pharmacological+space"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0liAyX6jv9tKhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520mapping%2520of%2520pharmacological%2520space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Chemical fragments as foundations for understanding target space and activity prediction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2689</span><span class="NLM_x">–</span> <span class="NLM_lpage">2700</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701399f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2689-2700&author=J.+J.+Sutherlandauthor=R.+E.+Higgsauthor=I.+Watsonauthor=M.+Vieth&title=Chemical+fragments+as+foundations+for+understanding+target+space+and+activity+prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm701399f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701399f%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DWatson%26aufirst%3DI.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DChemical%2520fragments%2520as%2520foundations%2520for%2520understanding%2520target%2520space%2520and%2520activity%2520prediction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2689%26epage%3D2700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0liAyX6jv9tKhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">–</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm400798j&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm400798j&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm400798j&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0liAyX6jv9tKhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gragg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmerle, H.</span><span> </span><span class="NLM_article-title">Kinomics-structural biology and chemogenomics of kinase inhibitors and targets</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1697</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">257</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1697&publication_year=2004&pages=243-257&author=M.+Viethauthor=R.+E.+Higgsauthor=D.+H.+Robertsonauthor=M.+Shapiroauthor=E.+A.+Graggauthor=H.+Hemmerle&title=Kinomics-structural+biology+and+chemogenomics+of+kinase+inhibitors+and+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DShapiro%26aufirst%3DM.%26aulast%3DGragg%26aufirst%3DE.%2BA.%26aulast%3DHemmerle%26aufirst%3DH.%26atitle%3DKinomics-structural%2520biology%2520and%2520chemogenomics%2520of%2520kinase%2520inhibitors%2520and%2520targets%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2004%26volume%3D1697%26spage%3D243%26epage%3D257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i16"><a href="/doi/suppl/10.1021/jm400798j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_45657"></div></div></div></div></div><hr /></hr><p class="last">Methods of cMSR versus RMSE; mean values for training and test set experimental activities; bias-uncorrected results from Table <a class="ref showTableEvent internalNav" href="#tbl1">1</a> for all 1466 data points; tesults for the Random Forest (RF) models compared to SVMFP models with ECC3 fingerprints for all 1466 test set data; concordance of the selectivity ratios for the collated data 38 450 selectivity ratio pairs; test set raw data, experimental values, and predictions; simulated data to illustrate artificially an artificially inflated <i>R</i><sup>2</sup>; training set data distribution for both Lilly and Metz sources; nearest neighbor distribution in generalized fingerprint space for the common 32 compounds with respect to Lilly and Metz training sets; comparison of the training set 50-50 cross-validation results for Lilly and Metz data sets for the <i>R</i><sup>2</sup> and cMSR metrics; cumulative comparison of the training set (model aware) experimental versus predicted data for all kinase-inhibitor pairs for the Lilly training set and the Metz bias-uncorrected training set; graphical representation of all kinase-inhibitor pair correlations for the experimental and predicted results from different sources; heat maps for 29 of 32 compounds with more than 25 data points against 68 targets; 38 450 target pair compound log selectivity ratio correlations between the experimental and predicted data compound target–target pairs; bioprint-driven compound dendrograms for the predicted Lilly, experimental Lilly, and experimental Metz profiling data sets; per target log(cMSR) anova and distribution histograms; and comparison of the nonsymmetrical 2278 target–target Tanimoto similarities between the Lilly experimental, Metz experimental, and Lilly predicted values computed from the available 29 compound pIC<sub>50</sub>s. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400798j/suppl_file/jm400798j_si_001.pdf">jm400798j_si_001.pdf (886.67 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400798j/suppl_file/jm400798j_si_002.xlsx">jm400798j_si_002.xlsx (225.22 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm400798j&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-17%3Barticle%3Aarticle%3A10.1021%2Fjm400798j%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm400798j" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799802bad313ca6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
