
<html lang="en"     class="pb-page"  data-request-id="efa476c3-1d5a-46c7-9400-8f9c8c05292f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c01638;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-21"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Rational Design of Right-Handed Heterogeneous Peptidomimetics as Inhibitors of Protein–Protein Interactions" /></meta><meta name="dc.Creator" content="Yan  Shi" /></meta><meta name="dc.Creator" content="Peng  Sang" /></meta><meta name="dc.Creator" content="Junhao  Lu" /></meta><meta name="dc.Creator" content="Pirada  Higbee" /></meta><meta name="dc.Creator" content="Lihong  Chen" /></meta><meta name="dc.Creator" content="Leixiang  Yang" /></meta><meta name="dc.Creator" content="Timothy  Odom" /></meta><meta name="dc.Creator" content="Gary  Daughdrill" /></meta><meta name="dc.Creator" content="Jiandong  Chen" /></meta><meta name="dc.Creator" content="Jianfeng  Cai" /></meta><meta name="dc.Description" content="Peptidomimetics have gained great attention for their function as protein–protein interaction (PPI) inhibitors. Herein, we report the design and investigation of a series of right-handed helical he..." /></meta><meta name="Description" content="Peptidomimetics have gained great attention for their function as protein–protein interaction (PPI) inhibitors. Herein, we report the design and investigation of a series of right-handed helical he..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 3, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01638" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01638" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01638" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01638" /></link>
        
    
    

<title>Rational Design of Right-Handed Heterogeneous Peptidomimetics as Inhibitors of Protein–Protein Interactions | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01638" /></meta><meta property="og:title" content="Rational Design of Right-Handed Heterogeneous Peptidomimetics as Inhibitors of Protein–Protein Interactions" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0008.jpeg" /></meta><meta property="og:description" content="Peptidomimetics have gained great attention for their function as protein–protein interaction (PPI) inhibitors. Herein, we report the design and investigation of a series of right-handed helical heterogeneous 1:1 α/Sulfono-γ-AA peptides as unprecedented inhibitors for p53-MDM2 and p53-MDMX. The most potent helical heterogeneous 1:1 α/Sulfono-γ-AA peptides were shown to bind tightly to MDM2 and MDMX, with Kd of 19.3 and 66.8 nM, respectively. Circular dichroism spectra, 2D-NMR spectroscopy, and the computational simulations suggested that these helical sulfono-γ-AA peptides could mimic the critical side chains of p53 and disrupt p53/MDM2 PPI effectively. It was noted that these 1:1 α/Sulfono-γ-AA peptides were completely resistant to proteolytic degradation, boosting their potential for biomedical applications. Furthermore, effective cellular activity is achieved by the stapled 1:1 α/Sulfono-γ-AA peptides, evidenced by significantly enhanced p53 transcriptional activity and much more induced level of MDM2 and p21. The 1:1 α/Sulfono-γ-AA peptides could be an alternative strategy to antagonize a myriad of PPIs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01638"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01638">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01638&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01638&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01638&amp;href=/doi/10.1021/acs.jmedchem.0c01638" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 13187-13196</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01635" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jmv063i021_1428478" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Rational Design of Right-Handed Heterogeneous Peptidomimetics as Inhibitors of Protein–Protein Interactions</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yan Shi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan Shi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Shi">Yan Shi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peng Sang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peng Sang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peng++Sang">Peng Sang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Junhao Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Junhao Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Oncology, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Junhao++Lu">Junhao Lu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pirada Higbee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pirada Higbee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cell Biology, Microbiology and Molecular Biology, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pirada++Higbee">Pirada Higbee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lihong Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lihong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Oncology, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lihong++Chen">Lihong Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Leixiang Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Leixiang Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Oncology, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Leixiang++Yang">Leixiang Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Timothy Odom</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Timothy Odom</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Timothy++Odom">Timothy Odom</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gary Daughdrill</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gary Daughdrill</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cell Biology, Microbiology and Molecular Biology, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#513635302436393523383d3d112422377f343524"><span class="__cf_email__" data-cfemail="d0b7b4b1a5b7b8b4a2b9bcbc90a5a3b6feb5b4a5">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gary++Daughdrill">Gary Daughdrill</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiandong Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiandong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Oncology, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#442e2d252a202b2a236a272c212a04292b22222d30306a2b3623"><span class="__cf_email__" data-cfemail="016b68606f656e6f662f6269646f416c6e67676875752f6e7366">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiandong++Chen">Jiandong Chen</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jianfeng Cai</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianfeng Cai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a7cdcec6c9c1c2c9c0c4c6cee7d2d4c189c2c3d2"><span class="__cf_email__" data-cfemail="a8c2c1c9c6cecdc6cfcbc9c1e8dddbce86cdccdd">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianfeng++Cai">Jianfeng Cai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3106-3306" title="Orcid link">http://orcid.org/0000-0003-3106-3306</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01638&amp;href=/doi/10.1021%2Facs.jmedchem.0c01638" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 13187–13196</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 3, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 September 2020</li><li><span class="item_label"><b>Published</b> online</span>3 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 November 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01638" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01638</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D13187%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYan%2BShi%252C%2BPeng%2BSang%252C%2BJunhao%2BLu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D21%26contentID%3Dacs.jmedchem.0c01638%26title%3DRational%2BDesign%2Bof%2BRight-Handed%2BHeterogeneous%2BPeptidomimetics%2Bas%2BInhibitors%2Bof%2BProtein%25E2%2580%2593Protein%2BInteractions%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D13196%26publicationDate%3DNovember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01638"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1015</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01638" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Rational Design of Right-Handed Heterogeneous Peptidomimetics as Inhibitors of Protein–Protein Interactions&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Shi&quot;},{&quot;first_name&quot;:&quot;Peng&quot;,&quot;last_name&quot;:&quot;Sang&quot;},{&quot;first_name&quot;:&quot;Junhao&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Pirada&quot;,&quot;last_name&quot;:&quot;Higbee&quot;},{&quot;first_name&quot;:&quot;Lihong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Leixiang&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Timothy&quot;,&quot;last_name&quot;:&quot;Odom&quot;},{&quot;first_name&quot;:&quot;Gary&quot;,&quot;last_name&quot;:&quot;Daughdrill&quot;},{&quot;first_name&quot;:&quot;Jiandong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jianfeng&quot;,&quot;last_name&quot;:&quot;Cai&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;13187-13196&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01638&quot;},&quot;abstract&quot;:&quot;Peptidomimetics have gained great attention for their function as protein–protein interaction (PPI) inhibitors. Herein, we report the design and investigation of a series of right-handed helical heterogeneous 1:1 α/Sulfono-γ-AA peptides as unprecedented inhibitors for p53-MDM2 and p53-MDMX. The most potent helical heterogeneous 1:1 α/Sulfono-γ-AA peptides were shown to bind tightly to MDM2 and MDMX, with Kd of 19.3 and 66.8 nM, respectively. Circular dichroism spectra, 2D-NMR spectroscopy, and the computational simulations suggested that these helical sulfono-γ-AA peptides could mimic the critical side chains of p53 and disrupt p53/MDM2 PPI effectively. It was noted that these 1:1 α/Sulfono-γ-AA peptides were completely resistant to proteolytic degradation, boosting their potential for biomedical applications. Furthermore, effective cellular activity is achieved by the stapled 1:1 α/Sulfono-γ-AA peptides, evidenced by significantly enhanced p53 transcriptional activity and much more induced level of MDM2 and &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01638&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01638" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01638&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01638" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01638&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01638" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01638&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01638&amp;href=/doi/10.1021/acs.jmedchem.0c01638" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01638" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01638" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01638%26sid%3Dliteratum%253Aachs%26pmid%3D33140956%26genre%3Darticle%26aulast%3DShi%26date%3D2020%26atitle%3DRational%2BDesign%2Bof%2BRight-Handed%2BHeterogeneous%2BPeptidomimetics%2Bas%2BInhibitors%2Bof%2BProtein%25E2%2580%2593Protein%2BInteractions%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D21%26spage%3D13187%26epage%3D13196%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290914" title="Peptidomimetics">Peptidomimetics</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291503" title="Organic polymers">Organic polymers</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/jmcmar.2020.63.issue-21/20201112/jmcmar.2020.63.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Peptidomimetics have gained great attention for their function as protein–protein interaction (PPI) inhibitors. Herein, we report the design and investigation of a series of right-handed helical heterogeneous 1:1 α/Sulfono-γ-AA peptides as unprecedented inhibitors for p53-MDM2 and p53-MDMX. The most potent helical heterogeneous 1:1 α/Sulfono-γ-AA peptides were shown to bind tightly to MDM2 and MDMX, with <i>K</i><sub>d</sub> of 19.3 and 66.8 nM, respectively. Circular dichroism spectra, 2D-NMR spectroscopy, and the computational simulations suggested that these helical sulfono-γ-AA peptides could mimic the critical side chains of p53 and disrupt p53/MDM2 PPI effectively. It was noted that these 1:1 α/Sulfono-γ-AA peptides were completely resistant to proteolytic degradation, boosting their potential for biomedical applications. Furthermore, effective cellular activity is achieved by the stapled 1:1 α/Sulfono-γ-AA peptides, evidenced by significantly enhanced p53 transcriptional activity and much more induced level of MDM2 and p21. The 1:1 α/Sulfono-γ-AA peptides could be an alternative strategy to antagonize a myriad of PPIs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57362" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57362" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the endeavor of searching and designing molecules that bind to important biological targets such as proteins with high affinity and specificity, peptides predominately serve as the leads.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> This is particularly true for the identification of molecules that modulate protein–protein interactions (PPIs) in which large, flat, and noncontiguous interfaces are involved. However, natural peptides are deemed not suitable for the therapeutic development due to their intrinsic drawbacks such as susceptibility to proteolytic degradation.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> To alleviate the problem, non-natural sequence-specific peptidomimetics have emerged to be a viable alternative strategy.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14">(4−14)</a> Compared to their natural counterparts, the peptidomimetics could not only retain or mimic the folding domain of the peptides such as helices but also display distinct structures and functions. Additionally, the chemodiversity is enhanced due to the ease of introducing diverse and unnatural functional side chains.<a onclick="showRef(event, 'ref3 ref15'); return false;" href="javascript:void(0);" class="ref ref3 ref15">(3,15)</a> Of course, thanks to their unnatural backbones, they are generally resistant to enzymatic hydrolysis, which enhances bioavailability.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Furthermore, peptidomimetic foldamers could mimic the primary, secondary, and even tertiary structures of peptides and proteins, and their potential application in biomedical science is enhanced through the structure-based design.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> In the past two decades, significant efforts have been made to develop new classes of peptidomimetics and successfully employ them into the study of biomolecular recognition and protein–protein interactions (PPIs).<a onclick="showRef(event, 'ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22 ref23">(19−23)</a> However, it remains an urgent task to exploit new peptidomimetic foldamers bearing new molecular scaffolds and frameworks due to current limited availability and application of peptidomimetics.</div><div class="NLM_p">To advance the field of peptidomimetics, we have recently developed a new class of foldamer, γ-AA peptides (oligomers of <i>N</i>-<u class="uu">a</u>cylated-<i>N</i>-<u class="uu">a</u>minoethyl amino acids), which were created based on the molecular scaffold γ-chiral PNA but with an enlarged chemodiversity to mimic peptides rather than nucleic acids.<a onclick="showRef(event, 'ref3 ref17'); return false;" href="javascript:void(0);" class="ref ref3 ref17">(3,17)</a> γ-AA peptides are highly resistant to enzymatic hydrolysis and have shown excellent cellular translocation capability.<a onclick="showRef(event, 'ref15 ref16 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref25 ref26 ref27">(15,16,25−27)</a> The modular synthesis of γ-AA peptides and their enormous chemodiversity make them excellent candidates for identification of valuable molecular probes or drug candidates through combinatorial library screening.<a onclick="showRef(event, 'ref15 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref15 ref28 ref29">(15,28,29)</a> More importantly, as a subclass of γ-AA peptides, sulfono-γ-AA peptides have been found to adopt a series of unprecedented helical secondary structures, stabilized by both intramolecular hydrogen bonding and the curved nature of sulfonamido moieties in the molecular framework,<a onclick="showRef(event, 'ref24 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref24 ref30 ref31 ref32">(24,30−32)</a> endowing them with superior folding propensity to the α-helix in solution. As such, sulfono-γ-AA peptides could be ideal candidates that could be rationally designed to mimic helical domains of proteins and disrupt medicinally related PPIs. Indeed, the homogeneous sulfono-γ-AA peptides, which form the left-handed 4<sub>14</sub> helical structure,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> have been shown to mimic the critical residues of the helical domain of BCL9 and disrupt the β-catenin/B cell lymphoma 9 protein–protein interaction (PPI)<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and to mimic GLP-1 to recognize GLP1-R and decrease blood glucose.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Recently, we have also demonstrated that they could be designed to mimic those important residues of the p53 helix and antagonize p53/MDM2 PPI.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> However, even if the results are highly promising, the mimicry of a right-handed α-helix by a left-handed helix may not be ideal, particularly when residues on the multiface of the helix are involved in interactions at the protein–protein interface. Previous structural studies show that 1:1 heterogeneous α/Sulfono-γ-AA peptides adopt a robust right-handed 4<sub>13</sub> windmill-shaped helices with a helical pitch of 5.34 Å (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A–E),<a onclick="showRef(event, 'ref30 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref34">(30,34)</a> implying their close similarity to the α-helix (pitch: 5.34 Å). Inspired by this desired structural information, we speculated that 1:1 heterogeneous α/Sulfono-γ-AA peptides could be a new class of helix mimetics as potential PPI inhibitors. In this paper, as a proof of concept, for the first time, we explored the heterogeneous 1:1 α/Sulfono-γ-AA peptides for the mimicry of the α-peptide helix. It is known that p53 is a tumor suppressor that plays a prominent role in the oncogenic transformation process and protects higher organisms from cancer. MDM2 and MDMX share considerable structural homology, binding to the N-terminus of p53 and leading to its transcriptional activity inhibition and degradation.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Recent findings suggest that overexpression of MDM2 and MDMX is a key factor that results in multiple human cancers, making them the promising target for antitumor drug development.<a onclick="showRef(event, 'ref22 ref36 ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref22 ref36 ref37 ref38 ref39 ref40">(22,36−40)</a> Hence, p53–MDM2 and p53–MDMX protein–protein interactions were selected as the model applications to evaluate the feasibility and practicality of our new developed foldamer.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Chemical structure of 1:1 α/Sulfono-γ-AA peptides. <b>a</b> and <b>b</b> denote the chiral side chain and the sulfonamido side chain from the building block, respectively. (B) The crystal structure of a 1:1 α/Sulfono-γ-AA peptide.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (C) Top view of panel (D,E). The schematic representation of the distribution of side chains from 1:1 α/Sulfono-γ-AA peptides. (D) Side view. (E) Top view.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05540" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05540" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Peptide Design and Activity Test</h3><div class="NLM_p">p53/MDM2 PPI has been a testing base for validation of helical peptidomimetics for the mimicry of the α-helix. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, three crucial residues of p53, Phe19, Trp23, and Leu26, on one face of the p53 helix, make multiple hydrophobic contacts at the deep pocket of the MDM2 cleft. It is demonstrated that molecules that can reproduce the binding modes of these three residues are expected to bind MDM2 tightly.<a onclick="showRef(event, 'ref41 ref42 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43 ref44 ref45">(41−45)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D, the pitch of 1:1 right-handed α/Sulfono-γ-AA peptides is 5.34 Å, essentially the same as that for the α-helix. Due to the characteristic of four side chains per turn (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E), we could select any face of the helix to mimic p53 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Superimposition of the p53 α-helix (purple) with a designed 1:1 α/Sulfono-γ-AA peptide (blue) (a PDB file is shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_002.pdb" class="ext-link">Supporting Information</a>). (B) The structure overlay of p53 (purple) binding and MDM2 (green) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>). (C) Overlay of the p53 helix (purple) with a 1:1 α/Sulfono-γ-AA peptide (blue) on the binding of MDM2 (green) (a PDB file is shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_002.pdb" class="ext-link">Supporting Information</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As an initial attempt, we randomly selected 2a, 4b, and 7 of 1:1 α/sulfono-γ-AA peptides to mimic F19, W23, and L26 in p53. 2a comes from the chiral side chain of the sulfono-γ-AA building block (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, compounds <b>1</b>–<b>3</b>) to mimic F19, 7 is the leucine amino acid residue to mimic L26, and 4b possesses sulfonyl side chains such as sulfonyl-2-naphthalene to mimic the W23 (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, compounds <b>1</b>–<b>3</b>). To our delight, the first peptide <b>1</b>, which contains the sulfonyl-2-naphthalene group, shows a <i>K</i><sub>d</sub> of 19.3 nM, revealing an almost 18-fold higher binding affinity toward MDM2 than the p53 (16–29) peptide (<i>K</i><sub>d</sub>, 360 nM, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf" class="ext-link">Figure S2</a>). It strongly suggested that the approach of 1:1 α/Sulfono-γ-AA peptides for the mimicry of the α-helix is viable. We next probed the effect of the side chains on positions 2a and 7. When the Trp group is used to mimic Phe on the 2a position (compound <b>2</b>, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), <i>K</i><sub>d</sub> = 82.7 nM, suggesting that although both indole and phenyl groups are hydrophobic, a phenyl group (compound <b>1</b>) is more preferable to mimic F19 in p53. Interestingly, replacing the Leu 7 in compound <b>1</b> with the cyclobutylmethyl group residue (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, compound <b>3</b>), the activity is still slightly weaker than that of <b>1</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, <i>K</i><sub>d</sub> = 72.6 nM), implying that Leu is the optimized residue. Subsequently, we explored the importance of the side chains at the position 4b. It is noted that compound <b>4</b> bearing the sulfonyl-1-naphthalene side chain induces a dramatic drop in the binding affinity (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, <i>K</i><sub>d</sub> = 482 nM), possibly due to the short group that could not make hydrophobic contact with the p53 binding domain of MDM2. The hypothesis is supported by the result that when changing this position to sulfonyl-TPE groups, the binding affinity of compound <b>5</b> becomes 6-fold potent as p53 (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, <i>K</i><sub>d</sub> = 62 nM). Instead of the naphthalene moiety, the effect of the phenyl group, as well as the substituent effect on its aromatic ring, was also investigated at the 4b position. Interestingly, although their binding affinities are all less than that of <b>1</b>, they bind more tightly to MDM2 than the p53 peptide.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Structures of the p53 and 1:1 α/Sulfono-γ-AA peptide <b>1</b>–<b>11</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">It is intriguing that the binding affinity reveals the relationship of 4-CF<sub>3</sub> > 4-Cl > 4-Br > 4-OCH<sub>3</sub> (compounds <b>9</b>, <b>7</b>, <b>8</b>, and <b>6</b>; <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). We speculated that the stronger electron-withdrawing group may facilitate the interaction with the MDM2 binding pocket. As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D, since the 1:1 α/Sulfono-γ-AA peptide helical scaffold has four faces, we postulated that residues on the other faces, such as 2b-5-8a and 3-6a-8b, could also be used for the mimicry of three hot-spot side chains F19, W23, and L26 in p53. As such, we synthesized compound <b>10</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) in which the side chains on the positions of 2b, 5, and 8a were designed to mimic F19, W23, and L26 in p53. To our disappointment, the binding affinity decreased sharply compared to that of <b>1</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, <i>K</i><sub>d</sub> = 653 nM). Also, if the side chains at the positions of 3, 6a, and 8b are used to mimic those three crucial residues in p53, although the activity is slightly improved (compound <b>11</b>, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, <i>K</i><sub>d</sub> = 344 nm) and comparable to that of the p53 peptide, it is still much less potent than compound <b>1</b>. These findings suggest that the precise arrangement of side chains on each helical face of 1:1 α/Sulfono-γ-AA peptides is not identical. In the meantime, the binding affinity is also highly affected by the neighboring side chains that are not directly involved in the binding to the target protein. The IC<sub>50</sub> values of these compounds are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf" class="ext-link">Figure S4</a>.</div><div class="NLM_p last">As MDMX is also involved in p53 binding and affecting the p53 signaling pathway, the binding affinities of these helical mimetics to MDMX are obtained. As shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf" class="ext-link">Figure S3</a>, most of the 1:1 α/Sulfono-γ-AA peptides have shown good <i>K</i><sub>d</sub> values to MDMX compared to p53, demonstrating that these peptides are potential dual-inhibitors to both MDM2 and MDMX.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> CD (Circular Dichroism) Spectra</h3><div class="NLM_p">In order to evaluate the helical propensity of these peptides in solution, the circular dichroism (CD) spectra are next recorded in PBS (100 μM) in the range of 195–260 nm. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, most of our compounds revealed a pronounced maximum at 198–202 nm and a minimum at 212 nm, suggesting that these peptides (<b>1</b>–<b>11</b>) adopt similar right-handed helical conformations.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Interestingly, a minimum of less than 200 nm was observed for p53 (16–29), signifying a random coil with almost no α-helix population. The data may explain why 1:1 α/Sulfono-γ-AA peptides are better MDM2 binders than p53.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. CD spectra of the p53 and compounds <b>1</b>–<b>11</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> HSQC NMR of Lead Peptide <b>9</b> in Complex with MDM2</h3><div class="NLM_p">As both <b>1</b> and <b>9</b> have comparable binding activities whereas <b>9</b> has slightly better solubility than <b>1</b>, we chose <b>9</b> for NMR study to further elucidate its binding to MDM2. Nuclear magnetic resonance (NMR) spectroscopy was used to determine if the compound <b>9</b> binding site on MDM2<sub>17–125</sub> was similar to the binding site for the p53 transactivation domain (TAD). <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A demonstrates the overlay of <sup>1</sup>H–<sup>15</sup>N HSQC spectra before (blue) and after (red) the addition of increasing concentrations of <b>9</b>. We measured the amide proton and nitrogen chemical shift changes of a uniformly <sup>15</sup>N-labeled sample of MDM2<sub>17–125</sub> after a stoichiometric excess of <b>9</b> was added. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B shows the average chemical shift changes in ppm for the amide proton and nitrogen resonances in MDM2<sub>17–125</sub>. The average chemical shift change for all the detectable MDM2<sub>17–125</sub> resonances when bound to <b>9</b> was 0.048 ppm. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D shows the structure of MDM2<sub>17–125</sub> in orange bound to p53TAD<sub>15–29</sub> in green.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C shows MDM2<sub>17–125</sub> residues with chemical shift changes greater than 0.048 ppm when bound to <b>9</b> colored red; 33 residues had chemical shift changes above the average. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D shows MDM2<sub>17–125</sub> residues with chemical shift changes greater than 0.048 ppm when bound to p53TAD<sub>1–73</sub> colored red; 52 residues had chemical shift changes above the average.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Overlay of <sup>1</sup>H–<sup>15</sup>N HSQC spectra for before (blue) and after (red) the addition of increasing concentrations of <b>9</b>. (B) Average chemical shift changes in ppm (parts per million) of the MDM2 p53 binding domain (residues 17–125) when bound to <b>9</b>. The 2° structure of MDM2<sub>17–125</sub> is shown below the graph. (C) Ribbon structure of MDM2<sub>17–125</sub> (orange) and p53TAD (green, residues 15–29) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>). Residues colored red have chemical shifts above the average of 0.048 ppm when bound to <b>9</b>. (D) Ribbon structure of MDM2<sub>17–125</sub> (orange) and p53TAD<sub>15–29</sub> (green) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>). Residues colored red have chemical shifts above the average of 0.048 ppm when bound to p53TAD<sub>1–73</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">MDM2<sub>17–125</sub> β<sub>1</sub> and β<sub>2</sub> had shifts in the presence of <b>9</b> and p53TAD<sub>1–73</sub>, as well as β<sub>3</sub>’. α<sub>1</sub>, β<sub>1</sub>’, β<sub>2</sub>’, and α<sub>2</sub>’ had more residues with shifts above the 0.048 ppm threshold in the presence of p53TAD<sub>1–73</sub> than <b>9</b> and β<sub>3</sub> had shifts only in the presence of <b>9</b> while α<sub>1</sub>’ had shifts only in the presence of p53TAD<sub>1–73</sub>. α<sub>2</sub> had shifts in the presence of both <b>9</b> and p53TAD<sub>1–73</sub>. For <b>9</b>, most of the large chemical shift changes were localized to the N-terminal half of α<sub>2</sub>. For p53TAD<sub>1–73</sub>, most of the large chemical shift changes were localized to the C-terminal half of α<sub>2</sub>. Based on these results, it appears that the binding site for <b>9</b> is similar to p53TAD<sub>1–73</sub> but not identical. We are currently using this data to optimize peptide design.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Stapled 1:1 α/Sulfono-γ-AA Peptide Induces Activation of p53 in Cells</h3><div class="NLM_p">To investigate the abilities of the compounds in activating p53 in cells, stapled 1:1 lactam-bridged α/Sulfono-γ-AA peptides <b>12</b>–<b>15</b> were synthesized via a side chain cross-linking strategy based on the most potent sequences <b>1</b> and <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–C) in the hope of enhancing cell permeability.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The IC<sub>50</sub>s of compounds <b>12</b>–<b>15</b> are 11.7, 9.4, 4.9, and 9.0 μM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf" class="ext-link">Figure S4</a>), which was better than those of corresponding linear compounds <b>1</b> and <b>2</b>. The luciferase reporter assay was employed to check the efficiency of wild-type p53 in activating the BP-100 MDM2 promoter, which constructed into the promoting sequence of luciferase (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf" class="ext-link">Figure S9</a>). Unlike the linear compounds <b>1</b> and <b>2</b>, enhanced luciferase activities (1.5–2-fold) were observed in the cells treated with stapled sequences. U2OS cells expressing endogenous wild-type p53 under negative regulation by both MDM2 and MDMX were treated with the stapled peptides. Treatment with all four stapled compounds led to increased p53 levels to various degrees. Modest induction of p53 target genes p21 and MDM2 were also detected, suggesting increased activities of p53 due to these compounds (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). The modest effect of the compounds in activating p53 in cell culture suggests that further optimization is needed to improve cell permeability and fully exploit their MDM2 binding ability. Overall, the results demonstrated the inhibitory function of these compounds for MDM2 in vitro, as well as their potential to be further developed into cell-permeable activators of p53 for cancer treatment.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Design and synthesis of the stapled sequences (a PDB file is shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_003.pdb" class="ext-link">Supporting Information</a>). (B) The structures of the stapled 1:1 α/Sulfono-γ-AA peptides <b>12</b>–<b>15</b>. (C) Overlay of the stapled 1:1 α/Sulfono-γ-AA peptide (blue) on the binding of MDM2 (a PDB file is shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_003.pdb" class="ext-link">Supporting Information</a>). (D) Activation of cellular p53 by compounds <b>1</b> and <b>2</b> and stapled p53 helical mimetics. U2OS osteosarcoma cells expressing endogenous wild-type p53 were treated with indicated compounds at 30 μM for 16 h. The cells were analyzed by Western blot for the level of p53 pathway markers.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Enzymatic Stability Study</h3><div class="NLM_p last">One distinctive characteristic of peptidomimetics is the remarkable resistance to enzymatic degradation, which has been proven in our previous study.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Herein, the stability of the 1:1 α/Sulfono-γ-AA peptide was evaluated by using p53 (16–29) as the control peptide. 0.1 mg/mL of compound <b>1</b> and p53 were incubated with 0.1 mg/mL pronase in 100 mM ammonium bicarbonate buffer (pH 7.8) at 37 °C for 24 h. After being monitored by HPLC-MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf" class="ext-link">Figures S6–S7</a>), the peptide p53 was found to be completely degraded by pronase as multiple unidentified peaks with no intact peptide remaining were detected. Meanwhile, the peptide <b>1</b> showed negligible degradation, indicating that these 1:1 α/Sulfono-γ-AA peptides were of great stability against proteolysis.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57816" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57816" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this paper, we have designed and synthesized a series of unprecedented right-handed helical 1:1 α/Sulfono-γ-AA peptide inhibitors based on the single crystal structure. The feasibility and practicality of the new helical peptidomimetic scaffolds were evaluated by mimicking the p53 helix for targeting MDM2 and MDMX. The activities of these peptides were evaluated by <i>K</i><sub>d</sub>. Besides, through the 2D-NMR experiment, the lead compound <b>9</b> was proven to interact with the p53-binding pocket of MDM2. The induction of p53 accumulation and its transcriptional targets p21 and MDM2 were also detected among the stapled 1:1 α/Sulfono-γ-AA peptide, suggesting that these peptides have the potential to be further developed into in vivo p53 activators. Furthermore, these peptidomimetics have shown great resistance toward the enzymatic degradation, which also make them to be promising therapeutic agents. This work provided a streamlined approach to discover potent peptidomimetic inhibitors of a myriad of protein–protein interactions.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34484" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34484" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> General Information</h3><div class="NLM_p last">Fmoc-protected amino acids were purchased from Chem-impex (Wood Dale, IL). Rink-Amide-MBHA resin (0.64 mmol/g) was purchased from GL Biochem (Shanghai, China). 1-Hydroxybenzotriazole wetted with no less than 20 wt % water (HOBt), 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDC), and <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) were purchased from Oakwood Chemical (Estill, SC). Tetraphenylethylene was purchased from Alfa Aesar. Chlorosulfonic acid was purchased from Sigma Aldrich. Solid phase synthesis was conducted in peptide synthesis vessels on a Burrell Wrist-Action shaker. γ-AA peptides were analyzed and purified on a Waters Breeze 2 HPLC system installed with both the analytic module (1 mL/min) and preparative module (16 mL/min), by employing a method using a 5–100% linear gradient of solvent B (0.1% TFA in acetonitrile) in solvent A (0.1% TFA in water) over 40 min followed by 100% solvent B over 10 min. Then the pure peak was collected and lyophilized on a Labcono lyophilizer. The purity of the compounds was determined to be >95% by analytical HPLC. High-resolution mass spectra were obtained on an Agilent 6220 using electrospray ionization time-of-flight (ESI-TOF). <sup>1</sup>H NMR spectra were recorded at 400 MHz using TMS as the internal standard. <sup>13</sup>C NMR spectra were recorded at 100 MHz using TMS as the internal standard. The multiplicities are reported as follows: singlet (s), doublet (d), doublet of doublets (dd), triplet (t), quartet (q), multiplet (m). Coupling constants are reported in hertz (Hz).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Preparation of Sulfono-γ-AA Building Blocks</h3><div class="NLM_p">The sulfono-γ-AA building blocks were synthesized based on a previous report.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf" class="ext-link">Scheme S1</a>, the initial starting materials were Fmoc-protected amino acids. Sulfono-γ-AA building blocks <b>4</b>–<b>10</b> and <b>15</b> were synthesized based on route 1, while sulfono-γ-AA building blocks <b>1</b>–<b>3</b> and <b>11</b>–<b>14</b> were synthesized based on route 2.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(4-(<i>tert</i>-butoxy)phenyl)propyl)-<i>n</i>-(naphthalen-2-ylsulfonyl)glycine (BB1)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.38 (d, <i>J</i> = 12 Hz, 1H), 8.05–8.13 (m, 4H), 7.82 (s, 2H), 7.63–7.65 (m, 3H), 7.23–7.35 (m, 6H), 7.04 (d, <i>J</i> = 8 Hz, 1H), 6.94 (d, <i>J</i> = 8 Hz, 1H), 6.71 (d, <i>J</i> = 8 Hz, 1H), 6.69 (d, <i>J</i> = 8 Hz, 1H), 4.12–4.18 (m, 2H), 3.8 (s, 2H), 3.52–3.57 (m, 2H), 3.36–3.43 (m, 1H), 3.14–3.23 (m, 1H), 2.71–2.84 (m, 1H), 1.11 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.8, 155.9, 153.6, 144.2, 141.1, 132.1, 130.4, 130.0, 129.8, 129.7, 129.2, 128.3, 128.1, 127.9, 127.8, 127.4, 125.6, 123.7, 123.0, 120.5, 115.3, 77.9, 65.6, 52.1, 51.9, 49.4, 47.0, 31.7, 28.9, 16.6. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>40</sub>H<sub>41</sub>N<sub>2</sub>O<sub>7</sub>S: 693.2634, found: 693.2650.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(4-(<i>tert</i>-butoxy)phenyl)propyl)-<i>N</i>-(naphthalen-1-ylsulfonyl)glycine (BB2)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.52 (d, <i>J</i> = 8 Hz, 1H), 8.18 (d, <i>J</i> = 4 Hz, 1H), 8.02–8.09 (m, 2H), 7.82 (s, 2H), 7.56–7.66 (m, 5H), 7.27–7.36 (m, 5H), 6.97 (d, <i>J</i> = 8 Hz, 1H), 6.85 (d, <i>J</i> = 8 Hz, 1H), 6.71 (d, <i>J</i> = 8 Hz, 2H), 6.58 (d, <i>J</i> = 4 Hz, 1H), 4.08–4.26 (m, 2H), 4.02–4.06 (m, 1H), 3.72 (s, 2H), 3.33–3.49 (m, 2H), 2.58–2.72 (m, 1H), 2.37–2.46 (m, 2H), 1.11 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.6, 155.9, 153.6, 144.2, 141.1, 135.4, 134.5, 130.3, 129.9, 129.4, 129.1, 128.5, 128.4, 128.0, 127.4, 127.2, 125.6, 124.9, 123.7, 120.5, 115.3, 77.9, 67.4, 65.7, 51.78, 49.2, 47.1, 37.4, 37.1, 31.7, 28.9, 27.7 HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>40</sub>H<sub>41</sub>N<sub>2</sub>O<sub>7</sub>S: 693.2634, found: 693.2650.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(4-(<i>tert</i>-butoxy)phenyl)propyl)-<i>N</i>-((4-(1,2,2-triphenylvinyl)phenyl)sulfonyl)glycine (BB3)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.85 (d, <i>J</i> = 8 Hz, 2H), 7.60 (d, <i>J</i> = 4 Hz, 2H), 7.47 (d, <i>J</i> = 4 Hz, 2H), 7.37 (s, 2H), 7.27–7.31 (m, 4H), 7.05–7.08 (m, 12H), 6.88–6.96 (m, 5H), 6.75 (d, <i>J</i> = 8 Hz, 2H), 6.62 (d, <i>J</i> = 8 Hz, 1H), 4.14–4.18 (m, 1H), 4.08–4.11 (m, 2H), 4.01–4.05 (m, 2H), 3.85–3.92 (m, 2H), 3.28–3.32 (m, 1H), 3.03–3.09 (m, 1H), 2.67–2.75 (m, 1H), 1.14 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.6, 155.9, 153.6, 148.1, 144.2, 143.0, 142.8, 141.1, 139.4, 137.4, 131.7, 131.1, 130.9, 130.1, 128.4, 128.3, 128.0, 127.4, 127.3, 126.9, 125.6, 123.8, 120.5, 77.9, 65.8, 51.7, 49.5, 47.1, 37.5, 31.7, 28.9 HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>56</sub>H<sub>53</sub>N<sub>2</sub>O<sub>7</sub>S: 897.3573, found: 897.3582.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-phenylpropyl)-<i>N</i>-((4-chlorophenyl)sulfonyl)glycine (BB4)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.82 (s, 1H), 7.83 (d, <i>J</i> = 8 Hz, 2H), 7.73 (d, <i>J</i> = 8 Hz, 2H), 7.47 (t, <i>J</i> = 4 Hz, 4H), 7.33–7.38 (m, 2H), 7.19–7.24 (m, 7H), 7.13 (d, <i>J</i> = 4 Hz, 1H), 4.13 (s, 1H), 4.07–4.09 (m, 2H), 3.97–4.02 (m, 1H), 3.84 (s, 1H), 3.35 (q, <i>J</i> = 12 Hz, 2H), 3.17 (q, <i>J</i> = 12 Hz, 1H), 2.83–2.87 (m, 1H), 2.55–2.61 (m, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.5, 155.9, 144.2, 141.1, 139.0, 138.8, 129.7, 129.5, 129.3, 128.5, 128.0, 127.4, 126.5, 125.6, 120.5, 65.7, 52.2, 51.7, 49.3, 47.1, 37.9 HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>32</sub>H<sub>30</sub>ClN<sub>2</sub>O<sub>6</sub>S: 605.1513, found: 605.1520.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-phenylpropyl)-<i>N</i>-((4-bromophenyl)sulfonyl)glycine (BB5)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.83 (s, 1H), 7.83 (d, <i>J</i> = 8 Hz, 2H), 7.72 (d, <i>J</i> = 8 Hz, 2H), 7.65 (d, <i>J</i> = 4 Hz, 2H), 7.54–7.56 (m, 2H), 7.36 (q, <i>J</i> = 12 Hz, 2H), 7.19–7.29 (m, 8H), 7.13 (d, <i>J</i> = 4 Hz, 1H), 4.13(s, 1H), 4.07–4.08 (m, 2H), 3.97–4.01 (m, 1H), 3.84 (s, 1H), 3.34 (q, <i>J</i> = 12 Hz, 1H), 3.18 (q, <i>J</i> = 12 Hz, 1H), 2.84 (q, <i>J</i> = 12 Hz, 1H), 2.55–2.61 (m, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.5, 155.9, 144.2, 141.1, 139.2, 139.0, 129.5, 129.4, 128.5, 128.0, 127.4, 126.5, 125.6, 120.5, 65.7, 52.2, 51.7, 49.3, 47.1, 37.9. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>32</sub>H<sub>30</sub>BrN<sub>2</sub>O<sub>6</sub>S: 649.1008, found: 649.1001.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-phenylpropyl)-<i>N</i>-((4-(trifluoromethyl)phenyl)sulfonyl)glycine (BB6)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.86 (s, 1H), 7.95 (d, <i>J</i> = 8 Hz, 2H), 7.88 (d, <i>J</i> = 8 Hz, 2H), 7.83 (d, <i>J</i> = 8 Hz, 2H), 7.55 (d, <i>J</i> = 8 Hz, 2H), 7.36 (q, <i>J</i> = 12 Hz, 2H), 7.19–7.29 (m, 7H), 7.13 (d, <i>J</i> = 4 Hz, 1H), 4.13(s, 1H), 4.07–4.08 (m, 2H), 3.96–4.01 (m, 1H), 3.86 (s, 1H), 3.39 (q, <i>J</i> = 12 Hz, 2H), 3.23 (q, <i>J</i> = 12 Hz, 1H), 2.85 (q, <i>J</i> = 12 Hz, 1H), 2.57–2.62 (m, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.4, 155.9, 144.2, 144.1, 143.8, 141.1, 138.9, 129.5, 128.5, 128.4, 127.9, 127.4, 126.7, 126.5, 125.6, 125.3, 122.7, 120.5, 65.7, 52.2, 51.7, 49.3, 47.1, 37.9. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>33</sub>H<sub>30</sub>F<sub>3</sub>N<sub>2</sub>O<sub>6</sub>S: 639.1777, found: 639.1780.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-phenylpropyl)-<i>N</i>-(methylsulfonyl)glycine (BB7)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.80 (d, <i>J</i> = 8 Hz, 2H), 7.53 (d, <i>J</i> = 8 Hz, 2H), 7.33 (t, <i>J</i> = 8 Hz, 2H), 7.23 (q, <i>J</i> = 12 Hz, 3H), 7.16 (d, <i>J</i> = 4 Hz, 4H), 7.08–7.11 (m, 1H), 4.12 (d, <i>J</i> = 8 Hz, 2H), 4.05–4.06 (m, 1H), 4.00–4.03 (m, 2H), 3.87 (s, 1H), 3.31 (q, <i>J</i> = 12 Hz, 1H), 3.14(q, <i>J</i> = 16 Hz, 1H), 2.88 (s, 3H), 2.79–2.853 (m, 1H), 2.49–2.55 (m, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 156.1, 144.2, 144.1, 139.1, 129.5, 128.5, 127.9, 127.4, 126.4, 125.5, 120.5, 65.7, 51.9, 51.6, 49.0, 47.1, 37.9. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>27</sub>H<sub>29</sub>N<sub>2</sub>O<sub>6</sub>S: 509.1746, found: 509.1750.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-4-methylpentyl)-<i>N</i>-(methylsulfonyl)glycine (BB8)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.80 (d, <i>J</i> = 8 Hz, 2H), 7.60 (d, <i>J</i> = 4 Hz, 2H), 7.34 (t, <i>J</i> = 4 Hz, 2H), 7.24–7.27 (m, 2H), 7.06 (d, <i>J</i> = 12 Hz, 1H), 4.27 (d, <i>J</i> = 8 Hz, 2H), 4.13–4.15 (m, 1H), 3.89 (s, 2H), 3.64 (s, 2H), 3.16 (q, <i>J</i> = 16 Hz, 1H), 3.01 (q, <i>J</i> = 12 Hz, 1H), 2.85 (s, 3H), 1.46–1.47 (m, 1H), 1.13–1.20 (m, 1H), 0.76 (q, <i>J</i> = 12 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.3144.3, 141.1, 127.9, 127.4, 125.5, 120.5, 65.5, 51.9, 48.8, 48.1, 47.2, 41.2, 24.6, 23.6, 21.9. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>24</sub>H<sub>31</sub>N<sub>2</sub>O<sub>6</sub>S: 475.1903, found: 475.1910.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-4-methylpentyl)-<i>N</i>-(isobutylsulfonyl)glycine (BB9)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.81 (d, <i>J</i> = 8 Hz, 2H), 7.61 (q, <i>J</i> = 8 Hz, 2H), 7.34 (t, <i>J</i> = 8 Hz, 2H), 7.23–7.27 (m, 2H), 7.08 (d, <i>J</i> = 12 Hz, 1H), 4.28–4.31 (m, 1H), 4.19–4.24 (m, 1H), 4.12–4.15 (m, 1H), 3.90 (s, 2H), 3.62–3.65 (m, 1H), 3.19 (q, <i>J</i> = 12 Hz, 1H), 3.05 (q, <i>J</i> = 16 Hz, 1H), 2.89 (t, <i>J</i> = 8 Hz, 2H), 1.96–2.04 (m, 1H), 1.46–1.48 (m, 1H), 1.14–1.21 (m, 2H), 0.9 (q, <i>J</i> = 8 Hz, 6H), 0.77 (q, <i>J</i> = 8 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.4, 144.3, 144.1, 141.1, 127.9, 127.4, 125.4, 120.5, 65.5, 59.3, 51.8, 48.6, 47.9, 47.2, 24.6, 23.6, 22.5, 21.9. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>27</sub>H<sub>37</sub>N<sub>2</sub>O<sub>6</sub>S: 517.2372, found: 517.2386.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-6-((<i>tert</i>-butoxycarbonyl)amino)hexyl)-<i>N</i>-(isobutylsulfonyl)glycine (10BB)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.81 (d, <i>J</i> = 8 Hz, 2H), 7.63 (q, <i>J</i> = 4 Hz, 2H), 7.34 (t, <i>J</i> = 8 Hz, 2H), 7.25–7.28 (m, 2H), 7.10 (d, <i>J</i> = 8 Hz, 1H), 6.66 (s, 1H), 4.19–4.26 (m, 2H), 4.14–4.16 (m, 1H), 3.92 (s, 2H), 3.55 (s, 2H), 3.20–3.24 (m, 1H), 3.05–3.11 (m, 1H), 2.90–2.95 (m, 2H), 2.83–2.88 (m, 2H), 1.98–2.05 (m, 1H), 1.29 (s, 9H), 1.18–1.24 (m, 5H), 0.9 (q, <i>J</i> = 8 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.4, 155.9, 144.2, 141.1, 128.0, 127.4, 125.6, 120.5, 77.8, 65.6, 59.4, 51.5, 49.9, 48.6, 48.5, 47.2, 31.9, 29.7, 28.6, 24.6, 23.1, 22.6. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>32</sub>H<sub>46</sub>N<sub>3</sub>O<sub>8</sub>S: 632.3006, found: 632.3009.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(1-(<i>tert</i>-butoxycarbonyl)-1<i>H</i>-indol-3-yl)propyl)-<i>N</i>-(methylsulfonyl)glycine (BB11)</h4><div class="NLM_p last">δ 7.96 (d, <i>J</i> = 8 Hz, 1H), 7.78 (q, <i>J</i> = 4 Hz, 2H), 7.62 (d, <i>J</i> = 8 Hz, 1H), 7.49 (d, <i>J</i> = 4 Hz, 3H), 7.30–7.33 (m, 3H), 7.23–7.26 (m, 1H), 7.17 (q, <i>J</i> = 16 Hz, 3H), 4.12–4.15 (m, 2H), 4.08–4.09 (m, 1H), 3.90 (s, 2H), 3.42 (q, <i>J</i> = 12 Hz, 1H), 3.23 (q, <i>J</i> = 12 Hz, 1H), 2.91 (s, 3H), 2.87 (s, 1H), 2.66–2.72 (m, 1H), 1.5 (s, 1H), 1.47 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 171.2, 158.9, 158.6, 156.3, 149.4, 144.1, 144.0, 141.1, 135.1, 130.9, 127.9, 127.8, 127.4, 125.5, 120.4, 117.8, 83.8, 65.9, 52.1, 50.1, 49.2, 47.1, 27.9. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>34</sub>H<sub>38</sub>N<sub>3</sub>O<sub>8</sub>S: 648.2380, found: 648.2371.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (<i>R</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(<i>tert</i>-butoxy)propyl)-<i>N</i>-((4-methoxyphenyl)sulfonyl)glycine (BB12)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.77 (d, <i>J</i> = 4 Hz, 6H), 7.33 (t, <i>J</i> = 4 Hz, 2H), 7.24–7.27 (m, 2H), 7.06 (d, <i>J</i> = 8 Hz, 1H), 6.68 (d, <i>J</i> = 8 Hz, 2H), 4.23–4.25 (m, 2H), 4.16 (d, <i>J</i> = 8 Hz 1H), 3.96–4.10 (m, 2H), 3.74 (s, 1H), 3.69 (s, 3H), 3.42 (d, <i>J</i> = 12 Hz 1H), 3.20–3.22 (m, 3H), 1.02 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.7, 162.8, 156.2, 144.2, 144.1, 131.5, 129.7, 128.0, 127.4, 125.6, 120.4, 114.6, 72.9, 65.9, 61.8, 55.9, 55.8, 51.1, 49.4, 47.1, 27.5. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>31</sub>H<sub>37</sub>N<sub>2</sub>O<sub>8</sub>S: 597.2271, found: 597.2268.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (<i>R</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(<i>tert</i>-butoxy)propyl)-<i>N</i>-(methylsulfonyl)glycine (BB13)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.82 (d, <i>J</i> = 8 Hz, 2H), 7.64 (d, <i>J</i> = 8 Hz, 2H), 7.36 (t, <i>J</i> = 4 Hz, 2H), 7.27 (t, <i>J</i> = 8 Hz, 2H), 7.11 (d, <i>J</i> = 8 Hz, 1H), 4.25–4.26 (m, 2H), 4.16–4.18 (m, 1H), 3.9–4.2 (m, 2H), 3.70 (s, 1H), 3.33–3.38 (m, 1H), 3.13–3.24 (m, 3H), 2.9 (s, 3H), 1.05 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.2, 144.2, 128.0, 127.4, 125.6, 120.5, 73.0, 65.8, 61.2, 50.9, 49.1, 48.9, 47.1, 27.6. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>25</sub>H<sub>33</sub>N<sub>2</sub>O<sub>7</sub>S: 505.2008, found: 505.2012.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-((2<i>R</i>,3<i>S</i>)-2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(<i>tert</i>-butoxy)butyl)-<i>N</i>-(benzylsulfonyl)glycine (BB14)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.81 (d, <i>J</i> = 8 Hz, 2H), 7.65 (d, <i>J</i> = 8 Hz, 2H), 7.34–7.35 (m, 5H), 7.28–7.29 (m, 3H), 7.17–7.24 (m, 2H), 4.27–4.38 (m, 4H), 4.17 (t, <i>J</i> = 8 Hz, 1H), 3.92–3.87 (m, 1H), 3.77 (s, 1H), 3.52–3.55 (m, 3H), 3.9 (t, <i>J</i> = 12 Hz, 1H), 1.07 (s, 9H), 0.88 (d, <i>J</i> = 8 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.5, 144.3, 141.1, 131.3, 130.1, 128.7, 128.5, 128.0, 127.4, 125.6, 125.5, 120.5, 120.4, 73.8, 67.2, 65.8, 58.5, 54.8, 49.2, 47.5, 47.3, 28.5, 17.9. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>32</sub>H<sub>39</sub>N<sub>2</sub>O<sub>7</sub>S: 595.2478, found: 595.2468.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)propyl)-<i>N</i>-((2-((<i>tert</i>-butoxycarbonyl)amino)ethyl)sulfonyl)glycine (BB15)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.81 (d, <i>J</i> = 8 Hz, 2H), 7.62 (s, 2H), 7.34 (t, <i>J</i> = 8 Hz, 2H), 7.26–7.28 (m, 2H), 7.16–7.18 (m, 1H), 6.83 (s, 1H), 4.23 (t, <i>J</i> = 8 Hz, 2H), 4.14 (s, 1H), 3.68 (s, 2H), 3.25 (s, 2H), 3.17 (s, 3H), 1.29 (s, 9H), 1.00 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.0, 155.7, 144.3, 141.1, 128.0, 127.4, 125.6, 120.5, 78.5, 65.7, 52.4, 51.5, 48.8, 47.2, 45.7, 35.2, 28.5, 18.6. HRMS (ESI) ([M + H]<sup>+</sup>) Calcd. for C<sub>27</sub>H<sub>36</sub>N<sub>3</sub>O<sub>8</sub>S: 562.2223, found: 562.2216.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Preparation of the α/Sulfono-γ-AA Peptide</h3><div class="NLM_p">The synthesis of the α/sulfono-γ-AA peptides was carried out on 150 mg of Rink Amide-MBHA resin (0.64 mmol/g) at room temperature (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf" class="ext-link">Scheme S2</a>). The resin was soaked in DMF for 5 min, and then the Fmoc protecting group was deprotected in 20% piperidine in DMF solution (2 mL, 10 min × 2). The resin was thoroughly washed with DCM (3 mL, 10 min × 3) and DMF (3 mL, 10 min × 3). After that, Fmoc-deprotected amino acid (2 equiv.), HOBt (4 equiv.), and DIC (4 equiv.) were added and shaken in 2 mL of DMF. After 3 h, the resin was washed with DCM and DMF and then treated with 20% piperidine/DMF solution (10 min × 2). Next, a solution of the α/Sulfono-γ-AA peptide building block (2 equiv.), HOBt (4 equiv.), and DIC (4 equiv.) in 2 mL of DMF was added and reacted for 4 h, and then the Fmoc protecting group was removed as stated above. The reaction cycles were repeated until the desired α/Sulfono-γ-AA peptides were synthesized. The N-terminus of the sequence was capped with acetic anhydride (1 mL) in pyridine (2 mL,15 min) followed by treatment with TFA/DCM (4 mL, 1:1, v/v) for 2.5 h. The cleavage solution was collected, and the resin was washed with DCM (3 mL × 2). The solution was combined and evaporated under air flow to give the crude product, which was analyzed and purified using a Waters HPLC system. The gradient eluting method of 5 to 100% of solvent B (0.1% TFA in acetonitrile) in A (0.1% TFA in water) over 40 min was performed. All the α/Sulfono-γ-AA peptides were obtained with moderate yields after prep-HPLC purification.</div><div class="NLM_p last">For the FITC-labeled α/sulfono-γ-AA peptide synthesis, after attaching the last α-amino acid, the Fmoc protecting group was then removed. FITC (1.2 equiv.) in 2 mL of DMF and DIPEA (6 equiv.) were added to the resin and shaken overnight to complete the reaction. After washing with DMF (×3) and DCM (×3), the resin was cleaved using TFA/DCM (6 mL, 1:1, v/v) for 3 h. Then, the pure FITC-labeled α/Sulfono-γ-AA peptides were obtained using the same abovementioned method.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Fluorescence Polarization Competition Assays</h3><div class="NLM_p last">The binding affinities (<i>K</i><sub>d</sub>) of the AA peptides were investigated by fluorescence polarization (FP). GST-MDM2-1-150 containing human MDM2 was expressed in <i>E. coli</i> as previously described by us. FP was carried out by incubating the 50 nM FITC-labeled AA peptide with MDM2/MDMX (0 to 1 μM) in 1× PBS with 0.1% Pluronic F-68. Dissociation constants (<i>K</i><sub>d</sub>) were determined by plotting the fluorescence anisotropy values as a function of protein concentration, and the plots were fitted to the following equation, where <i>L</i><sub>st</sub> is the concentration of the peptide, and <i>x</i> stands for the concentration of the protein. The experiments were performed in triplicate and repeated three times.<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_m001.gif" alt="" id="_i31" /></img></span></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Circular Dichroism</h3><div class="NLM_p last">Circular dichroism (CD) spectra were measured on an Aviv 215 circular dichroism spectrometer using a 1 mm path length quartz cuvette, and compound solutions in PBS buffer (or in trifluoroethanol) were prepared using dry weight of the lyophilized solid followed by dilution to give the desired concentration (100 uM) and solvent combination. 10 scans were averaged for each sample, and three independent experiments were conducted, and the spectra were averaged. The final spectra were normalized by subtracting the average blank spectra. Molar ellipticity [θ] (deg·cm<sup>2</sup>·dmol<sup>–1</sup>) was calculated using the equation<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_m002.gif" alt="" id="_i33" /></img></span>where θ<sub>obs</sub> is the measured ellipticity in millidegree, <i>n</i> is the number of side groups, <i>l</i> is the path length in centimeter (0.1 cm), and <i>c</i> is the concentration of the sulfono-γ-AA peptide in molar unit.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Luciferase Reporter Assay</h3><div class="NLM_p last">U2OS cells (p53 wild-type osteosarcoma) with stably integrated BP100-luc (p53-responsive luciferase reporter) and CMV-lacZ (internal control for cell mass and toxicity) were treated with compounds for 18 h. Luciferase and lacZ activities were determined, and the ratio of luc/lacZ indicates p53 transcriptional activity. Positive control Nutlin was used at 4 and 8 μM, respectively.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Western Blotting</h3><div class="NLM_p last">To detect proteins by Western blotting, cells were lysed in lysis buffer (50 mM Tris–HCl [pH 8.0], 150 mM NaCl, 0.5% NP-40, 1 mM phenylmethylsulfonyl fluoride [PMSF], 1× protease inhibitor cocktail) and centrifuged at 4 °C for 10 min at 14,000<i>g</i>. The supernatant was boiled in Laemmli sample buffer for 5 min and subjected to SDS-PAGE and Western blotting to detect the expression of proteins indicated in the figures using the corresponding antibodies.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Enzymatic Stability Study</h3><div class="NLM_p last">Lead compounds and peptide control p53 (0.1 mg/mL) were incubated with 0.1 mg/mL pronase in 100 mM ammonium bicarbonate buffer (pH 7.8) at 37 °C for 24 h. Then, the reaction mixtures were concentrated in a speed vacuum at medium temperature to remove water and ammonium bicarbonate. The resulting residues were redissolved in H<sub>2</sub>O/CH<sub>3</sub>CN and analyzed on a Waters analytical HPLC system with a 1 mL/min flow rate and 5 to 100% linear gradient of solvent B (0.1% TFA in acetonitrile) in A (0.1% TFA in water) over the duration of 50 min. The UV detector was set to 215 nm.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01638" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76342" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76342" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01638?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01638</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Preparation of sulfono-γ-AA building blocks; preparation of α/Sulfono-γ-AA peptides; fluorescence polarization competition assays; circular dichroism; enzymatic stability study; <sup>1</sup>H–<sup>15</sup>N HSQC NMR of lead peptide <b>9</b> in complex with MDM2; luciferase reporter assay; <sup>1</sup>H and <sup>13</sup>C NMR spectra of the sulfono-γ-AA peptide building blocks (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">PDB file of p53 α-helix with a designed 1:1 α/Sulfono-γ-AA peptide and MDM2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">PDB file of stapled sequence binding with MDM2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_004.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf">jm0c01638_si_001.pdf (3.86 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_002.pdb">jm0c01638_si_002.pdb (78.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_003.pdb">jm0c01638_si_003.pdb (81.87 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_004.csv">jm0c01638_si_004.csv (8.0 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01638" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76881" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76881" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gary Daughdrill</span> - <span class="hlFld-Affiliation affiliation">Department
of Cell Biology, Microbiology and Molecular Biology, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6f080b0e1a08070b1d0603032f1a1c09410a0b1a"><span class="__cf_email__" data-cfemail="a2c5c6c3d7c5cac6d0cbcecee2d7d1c48cc7c6d7">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiandong Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Oncology, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#42282b232c262d2c256c212a272c022f2d24242b36366c2d3025"><span class="__cf_email__" data-cfemail="8ee4e7efe0eae1e0e9a0ede6ebe0cee3e1e8e8e7fafaa0e1fce9">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianfeng Cai</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3106-3306" title="Orcid link">http://orcid.org/0000-0003-3106-3306</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e18b88808f87848f86828088a1949287cf848594"><span class="__cf_email__" data-cfemail="a8c2c1c9c6cecdc6cfcbc9c1e8dddbce86cdccdd">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Shi</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peng Sang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junhao Lu</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Oncology, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pirada Higbee</span> - <span class="hlFld-Affiliation affiliation">Department
of Cell Biology, Microbiology and Molecular Biology, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lihong Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Oncology, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leixiang Yang</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Oncology, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Timothy Odom</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Ave., Tampa, Florida 33620, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.S., P.S., J.L., and P.H. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75106" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75106" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The work was supported by NIH 9R01AI152416 and NIH 5R01AG056569.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last">ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">FITC</td><td class="NLM_def"><p class="first last">fluorescein isothiocyanate</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>d</sub></td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term">MDM2</td><td class="NLM_def"><p class="first last">mouse double minute 2</p></td></tr><tr><td class="NLM_term">MDMX</td><td class="NLM_def"><p class="first last">also known as MDM4</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18202" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18202" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 47 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S.</span></span> <span> </span><span class="NLM_article-title">Peptidomimetics</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1231</span>– <span class="NLM_lpage">1232</span>, <span class="refDoi"> DOI: 10.1021/ar800216e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800216e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Kiur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1231-1232&author=Y.-D.+Wuauthor=S.+Gellman&title=Peptidomimetics&doi=10.1021%2Far800216e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidomimetics</span></div><div class="casAuthors">Wu, Yun-Dong; Gellman, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1231-1232</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The emergence of peptidomimetics in research accounts during the past decade is outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz_q4rpSmjFrVg90H21EOLACvtfcHk0ljg_dQ_L0LiPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Kiur3K&md5=5d2218e70afc7a52a840091448822dd9</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Far800216e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800216e%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.-D.%26aulast%3DGellman%26aufirst%3DS.%26atitle%3DPeptidomimetics%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1231%26epage%3D1232%26doi%3D10.1021%2Far800216e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinhardt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, D. C.</span></span> <span> </span><span class="NLM_article-title">Development of α-Helical Calpain Probes by Mimicking a Natural Protein–Protein Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">17704</span>– <span class="NLM_lpage">17713</span>, <span class="refDoi"> DOI: 10.1021/ja307599z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja307599z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=17704-17713&author=H.+Joauthor=N.+Meinhardtauthor=Y.+Wuauthor=S.+Kulkarniauthor=X.+Huauthor=K.+E.+Lowauthor=P.+L.+Daviesauthor=W.+F.+DeGradoauthor=D.+C.+Greenbaum&title=Development+of+%CE%B1-Helical+Calpain+Probes+by+Mimicking+a+Natural+Protein%E2%80%93Protein+Interaction&doi=10.1021%2Fja307599z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Development of α-Helical Calpain Probes by Mimicking a Natural Protein-Protein Interaction</span></div><div class="casAuthors">Jo, Hyunil; Meinhardt, Nataline; Wu, Yibing; Kulkarni, Swapnil; Hu, Xiaozhen; Low, Kristin E.; Davies, Peter L.; DeGrado, William F.; Greenbaum, Doron C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">17704-17713</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly specific inhibitor of calpain was designed by mimicking a natural protein-protein interaction between calpain and its endogenous inhibitor calpastatin.  To enable this goal a new method of stabilizing an α-helix in a small peptide was established by screening 24 com. available cross-linkers for successful cysteine alkylation in a model peptide sequence.  The effects of crosslinking on the α-helicity of selected peptides were examd. by CD and NMR spectroscopy, and revealed structurally rigid cross-linkers to be the best at stabilizing α-helixes.  This strategy was applied to the design of inhibitors of calpain that are based on calpastatin, an intrinsically unstable polypeptide that becomes structured upon binding to the enzyme.  A two-turn α-helix that binds proximal to the active site cleft was stabilized, resulting in a potent and selective inhibitor for calpain.  The utility of this inhibitor was further expanded by developing irreversible calpain family activity-based probes (ABPs), which retained the specificity of the stabilized helical inhibitor.  It is believed that the inhibitor and ABPs will be useful for future investigation of calpains, while the crosslinking technique will enable exploration of other protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigseq6FFUU7Vg90H21EOLACvtfcHk0lh0SQvnAlE3fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrfK&md5=2824ee6e61d4bb0e4202696fc57f1fa0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fja307599z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja307599z%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DH.%26aulast%3DMeinhardt%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DKulkarni%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLow%26aufirst%3DK.%2BE.%26aulast%3DDavies%26aufirst%3DP.%2BL.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26aulast%3DGreenbaum%26aufirst%3DD.%2BC.%26atitle%3DDevelopment%2520of%2520%25CE%25B1-Helical%2520Calpain%2520Probes%2520by%2520Mimicking%2520a%2520Natural%2520Protein%25E2%2580%2593Protein%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D17704%26epage%3D17713%26doi%3D10.1021%2Fja307599z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: Design, Structure, and Applications</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00492</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00492" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=428-441&author=Y.+Shiauthor=P.+Tengauthor=P.+Sangauthor=F.+Sheauthor=L.+Weiauthor=J.+Cai&title=%CE%B3-AApeptides%3A+Design%2C+Structure%2C+and+Applications&doi=10.1021%2Facs.accounts.5b00492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: Design, structure, and applications</span></div><div class="casAuthors">Shi, Yan; Teng, Peng; Sang, Peng; She, Fengyu; Wei, Lulu; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">428-441</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The development of sequence-specific peptidomimetics has led to a variety of fascinating discoveries in chem. biol.  Many peptidomimetics can mimic primary, secondary, and even tertiary structure of peptides and proteins, and because of their unnatural backbones, they also possess significantly enhanced resistance to enzymic hydrolysis, improved bioavailability, and chemodiversity.  It is known that peptide nucleic acids (PNAs) are peptidic sequences developed for the mimicry of nucleic acids; however, their unique backbone as the mol. scaffold of peptidomimetics to mimic structure and function of bioactive peptides has not been investigated systematically.  As such, we recently developed a new class of peptidomimetics, "γ-AApeptides", based on the chiral γ-PNA backbone.  They are termed γ-AApeptides because they are the oligomers of γ-substituted-N-acylated-N-aminoethyl amino acids.  Similar to other classes of peptidomimetics, γ-AApeptides are also resistant to proteolytic degrdn. and possess the potential to enhance chemodiversity.  Moreover, in our scientific journey on the exploration of this class of peptidomimetics, we have discovered some intriguing structures and functions of γ-AApeptides.  In this account, we summarize the current development and application of γ-AApeptides with biol. potential.  Briefly, both linear and cyclic (either through head-to-tail or head-to-side-chain cyclization) γ-AApeptides with diverse functional groups can be synthesized easily on the solid phase using the synthetic protocol we developed. γ-AApeptides could mimic the primary structure of peptides, as they project the same no. of side chains as peptides of the same lengths.  For instance, they could mimic the Tat peptide to permeate cell membranes and bind to HIV RNA with high specificity and affinity.  Certain γ-AApeptides show similar activity to the RGD peptide and target integrin specifically on the cell surface.  γ-AApeptides with function akin to fMLF peptides are also identified.  More importantly, we found that γ-AApeptides can fold into discrete secondary structures, such as helical and β-turn-like structures.  Therefore, they could be rationally designed for a range of biol. applications.  For instance, γ-AApeptides can mimic host-defense peptides and display potent and broad-spectrum activity toward a panel of drug-resistant bacterial pathogens.  Meanwhile, because of their stability against proteolysis and their chemodiversity, γ-AApeptides are also amenable for combinatorial screening.  We demonstrate that, through combinatorial selection, certain γ-AApeptides are identified to inhibit Aβ40 peptide aggregation, suggesting their potential use as a mol. probe to intervene in Alzheimer's disease.  In addn., a few γ-AApeptides identified from the γ-AApeptide library have been shown to bind to the DNA-binding domain of STAT3 and antagonize STAT3/DNA interactions.  Our studies suggest that, with further studies and exploration on both structures and functions, γ-AApeptides may emerge to be a new class of peptidomimetics that play an important role in chem. biol. and biomedical sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrk8PLbKcOxrVg90H21EOLACvtfcHk0liQ_4PZhWjr6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D&md5=414ab3568ccfdcebb5863d876218804c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00492%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520Design%252C%2520Structure%252C%2520and%2520Applications%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D428%26epage%3D441%26doi%3D10.1021%2Facs.accounts.5b00492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span> <span> </span><span class="NLM_article-title">Universal Peptidomimetics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1021/ja1071916</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja1071916" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SrsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=462-477&author=E.+Koauthor=J.+Liuauthor=L.+M.+Perezauthor=G.+Luauthor=A.+Schaeferauthor=K.+Burgess&title=Universal+Peptidomimetics&doi=10.1021%2Fja1071916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Universal Peptidomimetics</span></div><div class="casAuthors">Ko, Eunhwa; Liu, Jing; Perez, Lisa M.; Lu, Genliang; Schaefer, Amber; Burgess, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">462-477</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptidomimetic scaffolds that can present side chains in conformations resembling those of amino acids in secondary structures without incurring excessive entropic or enthalpic penalties were discussed.  Compds. of this type were referred to as minimalist mimics.  The core hypothesis of the paper was that small sets of such scaffolds can be designed to analog local pairs of amino acids (including noncontiguous ones) in any secondary structure; i.e., they are universal peptidomimetics.  To illustrate this concept, a set of four peptidomimetic scaffolds, I-IV, were designed.  Libraries based on them were made bearing side chains corresponding to many of the protein-derived amino acids.  Modeling expts. were performed to give an indication of kinetic and thermodn. accessibilities of conformations that can mimic secondary structures.  Together, peptidomimetics based on these four scaffolds can adopt conformations that resemble almost any combination of local amino acid side chains in any secondary structure.  Universal peptidomimetics of this kind are likely to be most useful in the design of libraries for high-throughput screening against diverse targets.  Consequently, data arising from submission of these mols. to the NIH Mol. Libraries Small Mol. Repository (MLSMR) were outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjryjrFh6s0bVg90H21EOLACvtfcHk0liQ_4PZhWjr6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SrsbrM&md5=4c95046d25118b6d77e46668d88580b3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fja1071916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja1071916%26sid%3Dliteratum%253Aachs%26aulast%3DKo%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DPerez%26aufirst%3DL.%2BM.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DBurgess%26aufirst%3DK.%26atitle%3DUniversal%2520Peptidomimetics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D462%26epage%3D477%26doi%3D10.1021%2Fja1071916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, S. B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirshenbaum, K.</span></span> <span> </span><span class="NLM_article-title">Amphiphilic Cyclic Peptoids That Exhibit Antimicrobial Activity by Disrupting Staphylococcus aureus Membranes</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2013</i></span>,  <span class="NLM_fpage">3560</span>– <span class="NLM_lpage">3566</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201300077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fejoc.201300077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtFSrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2013&publication_year=2013&pages=3560-3566&author=M.+L.+Huangauthor=M.+A.+Bensonauthor=S.+B.+Y.+Shinauthor=V.+J.+Torresauthor=K.+Kirshenbaum&title=Amphiphilic+Cyclic+Peptoids+That+Exhibit+Antimicrobial+Activity+by+Disrupting+Staphylococcus+aureus+Membranes&doi=10.1002%2Fejoc.201300077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Amphiphilic cyclic peptoids that exhibit antimicrobial activity by disrupting Staphylococcus aureus membranes</span></div><div class="casAuthors">Huang, Mia L.; Benson, Meredith A.; Shin, Sung Bin Y.; Torres, Victor J.; Kirshenbaum, Kent</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2013</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3560-3566</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">There is a significant unmet need for new antimicrobial agents that can address antimicrobial resistance.  One promising group of antimicrobials is the antimicrobial peptides (AMPs) and their synthetic mimics.  In particular, synthetic sequence-specific oligomers of N-substituted glycine, termed "peptoids", have been found to show potent antimicrobial activity against bacterial pathogens in vitro and can act against the emergence of antimicrobial resistance.  In this study, the authors evaluate the antimicrobial activity of cyclic peptoid oligomers against clin. isolates of methicillin-resistant Staphylococcus aureus (MRSA).  The presence of the macrocyclic constraints can enforce a globally amphiphilic organization of the peptoid side-chains.  Several of these new amphiphilic compds. show potent and selective antimicrobial activity.  Electron microscopy expts. demonstrate that the peptoids target and damage the MRSA cytoplasmic membrane through the formation of pores.  These results substantiate the potential of peptoids as antimicrobial therapeutic agents for the treatment of S. aureus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokMvgO4N2wTLVg90H21EOLACvtfcHk0liP5pNnWA4KOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtFSrur0%253D&md5=bd4a8cc0af1010f04967e8b5576d0503</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201300077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201300077%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DM.%2BL.%26aulast%3DBenson%26aufirst%3DM.%2BA.%26aulast%3DShin%26aufirst%3DS.%2BB.%2BY.%26aulast%3DTorres%26aufirst%3DV.%2BJ.%26aulast%3DKirshenbaum%26aufirst%3DK.%26atitle%3DAmphiphilic%2520Cyclic%2520Peptoids%2520That%2520Exhibit%2520Antimicrobial%2520Activity%2520by%2520Disrupting%2520Staphylococcus%2520aureus%2520Membranes%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2013%26volume%3D2013%26spage%3D3560%26epage%3D3566%26doi%3D10.1002%2Fejoc.201300077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laursen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel-Andreasen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, C. A.</span></span> <span> </span><span class="NLM_article-title">β-Peptoid Foldamers at Last</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2696</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00257</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00257" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=2696-2704&author=J.+S.+Laursenauthor=J.+Engel-Andreasenauthor=C.+A.+Olsen&title=%CE%B2-Peptoid+Foldamers+at+Last&doi=10.1021%2Facs.accounts.5b00257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptoid foldamers at last</span></div><div class="casAuthors">Laursen, Jonas S.; Engel-Andreasen, Jens; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2696-2704</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  For a long time, peptides were considered unsuitable for drug development due to their inherently poor pharmacokinetic properties and proteolytic susceptibility.  However, this paradigm has changed significantly in the past decade with the approval of numerous antibodies and proteins as drugs.  In parallel, research in the field of synthetic mols. that are able to mimic or complement folding patterns exhibited by biopolymers, but are not recognized by proteases, have received considerable attention as well.  Such entities were coined "foldamers" by S. H. Gellman (1998).  Oligomers of N-alkylated 3-aminopropionic acid residues have been called β-peptoids due to their structural similarity to β-peptides and peptoids (N-alkylglycines), resp.  Because bona fide foldamer behavior has been demonstrated for both parent architectures, the authors wondered if the β-peptoids could serve as a successful addn. to the known ensemble of peptidomimetic foldamers.  When the authors entered this field, only the seminal description of libraries of β-peptoid dimers and trimers by B. C. Hamper et al. (1998) had been published a no. of years earlier.  Perhaps somewhat naively in retrospect, the authors envisioned that elongation of chain length combined with introduction of bulky α-chiral side-chains would deliver folded structures as reported for the α-peptoid counterparts.  Initially, the authors, and others, were unsuccessful in obtaining stable secondary structures of β-peptoid oligomers, and instead, these residues were either incorporated in cyclic structures or in combination with other types of residues to give peptidomimetic constructs with heterogeneous backbones.  Amphiphilic architectures with various membrane-targeting activities, such as mimics of antimicrobial peptides or cell-penetrating peptides, have thus been particularly successful.  The introduction of β-peptoid residues in histone deacetylase inhibitors mimicking non-ribosomal cyclotetrapeptides have also been reported.  Here, the authors sketch the scientific journey that ultimately delivered robustly folded β-peptoid oligomers.  Contributions involving biol. evaluation of peptidomimetic constructs contg. β-peptoid residues, as mentioned above, which were investigated leading up to these recently reported high-resoln. helical structures, are discussed.  On the basis of the work described here, the authors envision that β-peptoids will find future utility as peptidomimetics for biomedical investigation contg. both heterogeneous and homogeneous backbones.  The recent demonstration of control over the secondary structure of a homogeneous β-peptoid backbone now enables structure-based design of scaffolds with predictable display of desired functionalities in 3 dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT21EhTRrT-rVg90H21EOLACvtfcHk0liP5pNnWA4KOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyqsr%252FO&md5=61ec8510e6cf821887c1af9695ad22b8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00257%26sid%3Dliteratum%253Aachs%26aulast%3DLaursen%26aufirst%3DJ.%2BS.%26aulast%3DEngel-Andreasen%26aufirst%3DJ.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3D%25CE%25B2-Peptoid%2520Foldamers%2520at%2520Last%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D2696%26epage%3D2704%26doi%3D10.1021%2Facs.accounts.5b00257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horne, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketas, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klasse, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Structural and Biological Mimicry of Protein Surface Recognition by α/β-peptide Foldamers</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">14751</span>– <span class="NLM_lpage">14756</span>, <span class="refDoi"> DOI: 10.1073/pnas.0902663106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.0902663106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=19706443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFGrtrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=14751-14756&author=W.+S.+Horneauthor=L.+M.+Johnsonauthor=T.+J.+Ketasauthor=P.+J.+Klasseauthor=M.+Luauthor=J.+P.+Mooreauthor=S.+H.+Gellman&title=Structural+and+Biological+Mimicry+of+Protein+Surface+Recognition+by+%CE%B1%2F%CE%B2-peptide+Foldamers&doi=10.1073%2Fpnas.0902663106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biological mimicry of protein surface recognition by α/β-peptide foldamers</span></div><div class="casAuthors">Horne, W. Seth; Johnson, Lisa M.; Ketas, Thomas J.; Klasse, Per Johan; Lu, Min; Moore, John P.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">14751-14756, S14751/1-S14751/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Unnatural oligomers that can mimic protein surfaces offer a potentially useful strategy for blocking biomedically important protein-protein interactions.  Here we evaluate an approach based on combining α- and β-amino acid residues in the context of a polypeptide sequence from the HIV protein gp41, which represents an excellent testbed because of the wealth of available structural and biol. information.  We show that α/β-peptides can mimic structural and functional properties of a crit. gp41 subunit.  Phys. studies in soln., crystallog. data, and results from cell-fusion and virus-infectivity assays collectively indicate that the gp41-mimetic α/β-peptides effectively block HIV-cell fusion via a mechanism comparable to that of gp41-derived α-peptides.  An optimized α/β-peptide is far less susceptible to proteolytic degrdn. than is an analogous α-peptide.  Our findings show how a two-stage design approach, in which sequence-based α → β replacements are followed by site-specific backbone rigidification, can lead to phys. and biol. mimicry of a natural biorecognition process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAN0drYgQtaLVg90H21EOLACvtfcHk0liQWceHvvt35A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFGrtrfP&md5=e56297c79b7388ae517515c271c0fef3</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0902663106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0902663106%26sid%3Dliteratum%253Aachs%26aulast%3DHorne%26aufirst%3DW.%2BS.%26aulast%3DJohnson%26aufirst%3DL.%2BM.%26aulast%3DKetas%26aufirst%3DT.%2BJ.%26aulast%3DKlasse%26aufirst%3DP.%2BJ.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DMoore%26aufirst%3DJ.%2BP.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DStructural%2520and%2520Biological%2520Mimicry%2520of%2520Protein%2520Surface%2520Recognition%2520by%2520%25CE%25B1%252F%25CE%25B2-peptide%2520Foldamers%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D14751%26epage%3D14756%26doi%3D10.1073%2Fpnas.0902663106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berlicki, Ł.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilsl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wéber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mándity, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrele, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fülöp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, O.</span></span> <span> </span><span class="NLM_article-title">Unique α,β- and α,α,β,β-Peptide Foldamers Based on cis-β-Aminocyclopentanecarboxylic Acid</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2208</span>– <span class="NLM_lpage">2212</span>, <span class="refDoi"> DOI: 10.1002/anie.201107702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201107702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ymtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=2208-2212&author=%C5%81.+Berlickiauthor=L.+Pilslauthor=E.+W%C3%A9berauthor=I.+M.+M%C3%A1ndityauthor=C.+Cabreleauthor=T.+A.+Martinekauthor=F.+F%C3%BCl%C3%B6pauthor=O.+Reiser&title=Unique+%CE%B1%2C%CE%B2-+and+%CE%B1%2C%CE%B1%2C%CE%B2%2C%CE%B2-Peptide+Foldamers+Based+on+cis-%CE%B2-Aminocyclopentanecarboxylic+Acid&doi=10.1002%2Fanie.201107702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Unique α,β- and α,α,β,β-peptide foldamers based on cis-β-aminocyclopentanecarboxylic acid</span></div><div class="casAuthors">Berlicki, Lukasz; Pilsl, Ludwig; Weber, Edit; Mandity, Istvan M.; Cabrele, Chiara; Martinek, Tamas A.; Fueloep, Ferenc; Reiser, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2208-2212, S2208/1-S2208/29</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Non-natural oligomers with well-defined three-dimensional structures (foldamers) raise great interest owing to their ability to mimic and expand the world of biopolymer.  In this paper we present helical peptide foldamers with defined spatial distributions of their functional groups in polar solvents including water.  Stable helical structures based on α-amino and cis-β-aminocyclopentanecarboxylic acids have been design following the stereochem. pattering approach.  All the peptides were successfully obtained by solid phase peptide synthesis and catalytic hydrogenation.  CD, NMR and theor. calcns. were employed for detn. of foldamers. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptO-sm-FJwi7Vg90H21EOLACvtfcHk0liCvkNSU9AMFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ymtL0%253D&md5=7a96f663f67145f32e5519393dd6fa4d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107702%26sid%3Dliteratum%253Aachs%26aulast%3DBerlicki%26aufirst%3D%25C5%2581.%26aulast%3DPilsl%26aufirst%3DL.%26aulast%3DW%25C3%25A9ber%26aufirst%3DE.%26aulast%3DM%25C3%25A1ndity%26aufirst%3DI.%2BM.%26aulast%3DCabrele%26aufirst%3DC.%26aulast%3DMartinek%26aufirst%3DT.%2BA.%26aulast%3DF%25C3%25BCl%25C3%25B6p%26aufirst%3DF.%26aulast%3DReiser%26aufirst%3DO.%26atitle%3DUnique%2520%25CE%25B1%252C%25CE%25B2-%2520and%2520%25CE%25B1%252C%25CE%25B1%252C%25CE%25B2%252C%25CE%25B2-Peptide%2520Foldamers%2520Based%2520on%2520cis-%25CE%25B2-Aminocyclopentanecarboxylic%2520Acid%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D2208%26epage%3D2212%26doi%3D10.1002%2Fanie.201107702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span> <span> </span><span class="NLM_article-title">α-Aminoxy Acids: New Possibilities from Foldamers to Anion Receptors and Channels</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1428</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.1021/ar8001393</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar8001393" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFSmt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1428-1438&author=X.+Liauthor=Y.-D.+Wuauthor=D.+Yang&title=%CE%B1-Aminoxy+Acids%3A+New+Possibilities+from+Foldamers+to+Anion+Receptors+and+Channels&doi=10.1021%2Far8001393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">α-Aminoxy Acids: New Possibilities from Foldamers to Anion Receptors and Channels</span></div><div class="casAuthors">Li, Xiang; Wu, Yun-Dong; Yang, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1428-1438</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Naturally occurring peptides serve important functions as enzyme inhibitors, hormones, neurotransmitters, and immunomodulators in many physiol. processes including metab., digestion, pain sensitivity, and the immune response.  However, due to their conformational flexibility and poor bioavailability, such peptides are not generally viewed as useful therapeutic agents in clin. applications.  In an effort to solve these problems, chemists have developed peptidomimetic foldamers, unnatural oligomeric mols. that fold into rigid and well-defined secondary structures mimicking the structures and biol. functions of these natural peptides.  We have designed peptidomimetic foldamers that give predictable, backbone-controlled secondary structures irresp. of the nature of the side chains.  This account presents our efforts to develop a novel class of peptidomimetic foldamers comprising α-aminooxy acids and explore their applications in the simulation of ion recognition and transport processes in living systems.  Peptides constructed from α-aminooxy acids fold according to the following rules.  First, a strong intramol. eight-membered-ring hydrogen bond forms between adjacent α-aminooxy acid residues (the α N-O turn).  The chirality of the α-carbon, not the nature of the side chains, dets. the conformation of this chiral N-O turn.  Secondly, while homochiral oligomers of α-aminooxy acids form an extended helical structure (1.88 helix), heterochiral ones adopt a bent reverse turn structure.  Thirdly, in peptides of alternating α-amino acids and α-aminooxy acids, the seven-membered-ring intramol. hydrogen bond, i.e., the γ-turn, is initiated by a succeeding α N-O turn.  Thus, this type of peptide adopts a novel 7/8 helical structure.  In investigating the potential applications of α-aminooxy acids, we have found that the amide NH units of α-aminooxy acids are more acidic than are regular amide NH groups, which makes them better hydrogen bond donors when interacting with anions.  This property makes α-aminooxy acids ideal building blocks for the construction of anion receptors.  Indeed, we have constructed both cyclic and acyclic anion receptors that have strong affinities and good (enantio-)selectivities toward chloride (Cl-) and chiral carboxylate ions.  Taking advantage of these systems' preference for Cl- ions, we have also employed α-aminooxy acid units to construct a synthetic Cl- channel that can mediate the passage of Cl- ions across cell membranes.  Continued studies of these peptidomimetic systems built from α-aminooxy acids should lead to a broad range of applications in chem., biol., medicine, and materials science.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgWTc5dmlBYrVg90H21EOLACvtfcHk0liCvkNSU9AMFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFSmt7zK&md5=edf4a8e0559a6fce8a64d18bf21a6ebe</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Far8001393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far8001393%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.-D.%26aulast%3DYang%26aufirst%3DD.%26atitle%3D%25CE%25B1-Aminoxy%2520Acids%253A%2520New%2520Possibilities%2520from%2520Foldamers%2520to%2520Anion%2520Receptors%2520and%2520Channels%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1428%26epage%3D1438%26doi%3D10.1021%2Far8001393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisblum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palecek, S. P.</span></span> <span> </span><span class="NLM_article-title">Antifungal Activity from 14-Helical β-Peptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">12630</span>– <span class="NLM_lpage">12631</span>, <span class="refDoi"> DOI: 10.1021/ja064630y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja064630y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD28XptlGlu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=12630-12631&author=A.+J.+Karlssonauthor=W.+C.+Pomerantzauthor=B.+Weisblumauthor=S.+H.+Gellmanauthor=S.+P.+Palecek&title=Antifungal+Activity+from+14-Helical+%CE%B2-Peptides&doi=10.1021%2Fja064630y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Antifungal activity from 14-helical β-peptides</span></div><div class="casAuthors">Karlsson, Amy J.; Pomerantz, William C.; Weisblum, Bernard; Gellman, Samuel H.; Palecek, Sean P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">12630-12631</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have discovered that short β-peptides (9 or 10 residues) designed to adopt globally amphiphilic helical conformations display significant antifungal activity.  The most promising β-peptides cause little lysis of human red blood cells at concns. that kill Candida albicans, a common human fungal pathogen.  Since fungi are eukaryotes, discrimination between fungal and human cells is a significant finding.  Our β-peptides are active under assay conditions that mimic physiol. ionic strength; in contrast, α-helix-forming host-defense α-peptides are inactive against C. albicans under these conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkwF12HntvOrVg90H21EOLACvtfcHk0limKXAnSoY--Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptlGlu7k%253D&md5=f89c05797ccd944e54831f322634e4cb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja064630y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja064630y%26sid%3Dliteratum%253Aachs%26aulast%3DKarlsson%26aufirst%3DA.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%26aulast%3DWeisblum%26aufirst%3DB.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DPalecek%26aufirst%3DS.%2BP.%26atitle%3DAntifungal%2520Activity%2520from%252014-Helical%2520%25CE%25B2-Peptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D12630%26epage%3D12631%26doi%3D10.1021%2Fja064630y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lear, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodsdon, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Helical β-Peptide Inhibitors of the p53-hDM2 Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">9468</span>– <span class="NLM_lpage">9469</span>, <span class="refDoi"> DOI: 10.1021/ja031625a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja031625a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVSksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=9468-9469&author=J.+A.+Kritzerauthor=J.+D.+Learauthor=M.+E.+Hodsdonauthor=A.+Schepartz&title=Helical+%CE%B2-Peptide+Inhibitors+of+the+p53-hDM2+Interaction&doi=10.1021%2Fja031625a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Helical β-Peptide Inhibitors of the p53-hDM2 Interaction</span></div><div class="casAuthors">Kritzer, Joshua A.; Lear, James D.; Hodsdon, Michael E.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9468-9469</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HDM2 is recognized in vivo by a short α-helix within the p53 trans-activation domain (p53AD).  Disruption of the p53·hDM2 interaction is an important goal for cancer therapy.  A functional epitope comprised of three residues on one face of the p53AD helix (F19, W23, and L26) contributes heavily to the binding free energy.  We hypothesized that the p53AD functional epitope would be recapitulated if the side chains of F19, W23, and L26 were presented at successive positions three residues apart on a stabilized β3-peptide 14-helix.  Here, we report a set of β3-peptides that possess significant 14-helix structure in water; one recognizes a cleft on the surface of hDM2 with nanomolar affinity.  The strategy for β3-peptide design that we describe is general and may have advantages over one in which individual or multiple β-amino acid substitutions are introduced into a functional α-peptide, because it is based on homol. at the level of secondary structure, not primary sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVlZAGh6JntbVg90H21EOLACvtfcHk0limKXAnSoY--Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVSksLo%253D&md5=3fc9e87497a50588ee4591d5f84927ea</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fja031625a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja031625a%26sid%3Dliteratum%253Aachs%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26aulast%3DLear%26aufirst%3DJ.%2BD.%26aulast%3DHodsdon%26aufirst%3DM.%2BE.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DHelical%2520%25CE%25B2-Peptide%2520Inhibitors%2520of%2520the%2520p53-hDM2%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D9468%26epage%3D9469%26doi%3D10.1021%2Fja031625a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">β-Peptides: From Structure to Function</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">3219</span>– <span class="NLM_lpage">3232</span>, <span class="refDoi"> DOI: 10.1021/cr000045i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000045i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=3219-3232&author=R.+P.+Chengauthor=S.+H.+Gellmanauthor=W.+F.+DeGrado&title=%CE%B2-Peptides%3A+From+Structure+to+Function&doi=10.1021%2Fcr000045i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptides: From structure to function</span></div><div class="casAuthors">Cheng, Richard P.; Gellman, Samuel H.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3219-3232</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 187 refs., of the conformational properties and biol. functions, esp. as peptidomimetics, of β-peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTfycZ30AaKLVg90H21EOLACvtfcHk0lhCFTFPDspZOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D&md5=40cbf3f3d3c00ec8d6250adf407a663a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fcr000045i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000045i%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BP.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3D%25CE%25B2-Peptides%253A%2520From%2520Structure%2520to%2520Function%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D3219%26epage%3D3232%26doi%3D10.1021%2Fcr000045i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seebach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, J.</span></span> <span> </span><span class="NLM_article-title">β-Peptidic Peptidomimetics</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1366</span>– <span class="NLM_lpage">1375</span>, <span class="refDoi"> DOI: 10.1021/ar700263g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar700263g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvVKksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1366-1375&author=D.+Seebachauthor=J.+Gardiner&title=%CE%B2-Peptidic+Peptidomimetics&doi=10.1021%2Far700263g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptidic peptidomimetics</span></div><div class="casAuthors">Seebach, Dieter; Gardiner, James</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1366-1375</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  For more than a decade now, a search for answers to the following two questions has taken us on a new and exciting journey into the world of β- and γ-peptides: What happens if the O atoms in a 3i-helix of a polymeric chain composed of (R)-3-hydroxybutanoic acid are replaced by NH units.  What happens if 1 or 2 CH2 groups are introduced into each amino acid building block in the chain of a peptide or protein, thereby providing homologues of the proteinogenic α-amino acids.  Our journey has repeatedly thrown up surprises, continually expanding the potential of these classes of compd. and deepening our understanding of the structures, properties, and multifaceted functions of the natural "models" to which they are related.  β-Peptides differ from their natural counterparts, the α-peptides, by having CH2 groups inserted into every amino acid residue, either between the C=O groups and the α-C atoms (β3) or between the α-C and N atoms (β2).  The synthesis of these homologated proteinogenic amino acids and their assembly into β-peptides can be performed using known methods.  Despite the increased no. of possible conformers, the β-peptides form secondary structures (helixes, turns, sheets) even when the chain lengths are as short as 4 residues.  Furthermore, they are stable toward degrading and metabolizing enzymes in living organisms.  Linear, helical, and hairpin-type structures of β-peptides can now be designed in such a way that they resemble the characteristic and activity-related structural features ("epitopes") of corresponding natural peptides or protein sections.  This account presents examples of β-peptidic compds. binding, as agonists or antagonists (inhibitors), to (i) major histocompatibility complex (MHC) proteins (immune response), (ii) the lipid-transport protein SR-B1 (cholesterol uptake from the small intestine), (iii) the core (1-60) of interleukin-8 (inflammation), (iv) the oncoprotein RDM2, (v) the HIV gp41 fusion protein, (vi) G-protein-coupled somatostatin hsst receptors, (vii) the TNF immune response receptor CD40 (apoptosis), and (viii) DNA.  Short-chain β-peptides may be orally bioavailable and excreted from the body of mammals; long-chain β-peptides may require i.v. administration but will have longer half-lives of clearance.  It has been said that an interesting field of research distinguishes itself in that the results always throw up new questions; in this sense, the structural and biol. investigation of β-peptides has been a gold mine.  We expect that these peptidic peptidomimetics will play an increasing role in biomedical research and drug development in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0dEyPZmslHLVg90H21EOLACvtfcHk0lhCFTFPDspZOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvVKksrk%253D&md5=e679836f00843a02103caf63773ffa80</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Far700263g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far700263g%26sid%3Dliteratum%253Aachs%26aulast%3DSeebach%26aufirst%3DD.%26aulast%3DGardiner%26aufirst%3DJ.%26atitle%3D%25CE%25B2-Peptidic%2520Peptidomimetics%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1366%26epage%3D1375%26doi%3D10.1021%2Far700263g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espina, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stouten, P. F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duke, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mousa, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">Type II‘ to Type I β-Turn Swap Changes Specificity for Integrins</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1021/ja953163+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja953163%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADyaK28XltVGjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=293-294&author=A.+C.+Bachauthor=J.+R.+Espinaauthor=S.+A.+Jacksonauthor=P.+F.+W.+Stoutenauthor=J.+L.+Dukeauthor=S.+A.+Mousaauthor=W.+F.+DeGrado&title=Type+II%E2%80%98+to+Type+I+%CE%B2-Turn+Swap+Changes+Specificity+for+Integrins&doi=10.1021%2Fja953163%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Type II' to Type I β-Turn Swap Changes Specificity for Integrins</span></div><div class="casAuthors">Bach, Alvin C., II; Espina, J. Robert; Jackson, Sharon A.; Stouten, Pieter F. W.; Duke, Jodie L.; Mousa, Shaker A.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">293-4</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotcOCqaY7emrVg90H21EOLACvtfcHk0lh1CbPmEUKDRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XltVGjsA%253D%253D&md5=ded7ddfed0f53af3391fecf4607ca8d1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fja953163%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja953163%252B%26sid%3Dliteratum%253Aachs%26aulast%3DBach%26aufirst%3DA.%2BC.%26aulast%3DEspina%26aufirst%3DJ.%2BR.%26aulast%3DJackson%26aufirst%3DS.%2BA.%26aulast%3DStouten%26aufirst%3DP.%2BF.%2BW.%26aulast%3DDuke%26aufirst%3DJ.%2BL.%26aulast%3DMousa%26aufirst%3DS.%2BA.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DType%2520II%25E2%2580%2598%2520to%2520Type%2520I%2520%25CE%25B2-Turn%2520Swap%2520Changes%2520Specificity%2520for%2520Integrins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D293%26epage%3D294%26doi%3D10.1021%2Fja953163%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Challa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmagadda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">One-Bead–Two-Compound Thioether Bridged Macrocyclic γ-AApeptide Screening Library against EphA2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9290</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslKgtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9290-9298&author=Y.+Shiauthor=S.+Challaauthor=P.+Sangauthor=F.+Sheauthor=C.+Liauthor=G.+M.+Grayauthor=A.+Nimmagaddaauthor=P.+Tengauthor=T.+Odomauthor=Y.+Wangauthor=A.+van+der%0AVaartauthor=Q.+Liauthor=J.+Cai&title=One-Bead%E2%80%93Two-Compound+Thioether+Bridged+Macrocyclic+%CE%B3-AApeptide+Screening+Library+against+EphA2&doi=10.1021%2Facs.jmedchem.7b01280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">One-bead-two-compound thioether bridged macrocyclic γ-AApeptide screening library against EphA2</span></div><div class="casAuthors">Shi, Yan; Challa, Sridevi; Sang, Peng; She, Fengyu; Li, Chunpu; Gray, Geoffrey M.; Nimmagadda, Alekhya; Teng, Peng; Odom, Timothy; Wang, Yan; van der Vaart, Arjan; Li, Qi; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9290-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification of mol. ligands that recognize peptides or proteins is significant but poses a fundamental challenge in chem. biol. and biomedical sciences.  Development of cyclic peptidomimetic library is scarce, and thus discovery of cyclic peptidomimetic ligands for protein targets is rare.  Herein we report the unprecedented one-bead-two-compd. (OBTC) combinatorial library based on a novel class of the macrocyclic peptidomimetics γ-AApeptides.  In the library, we utilized the coding peptide tags synthesized with Dde-protected α-amino acids, which were orthogonal to solid phase synthesis of γ-AApeptides.  Employing the thioether linkage, the desired macrocyclic γ-AApeptides were found to be effective for ligand identification.  Screening the library against the receptor tyrosine kinase EphA2 led to the discovery of one lead compd. that tightly bound to EphA2 (Kd = 81 nM) and potently antagonized EphA2-mediated signaling.  This new approach of macrocyclic peptidomimetic library may lead to a novel platform for biomacromol. surface recognition and function modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE8_skSAKNvbVg90H21EOLACvtfcHk0lj3ql35d97Q-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslKgtLnO&md5=0c5c86232aa6afd699fd6de5128a1ecf</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01280%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DChalla%26aufirst%3DS.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DNimmagadda%26aufirst%3DA.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOne-Bead%25E2%2580%2593Two-Compound%2520Thioether%2520Bridged%2520Macrocyclic%2520%25CE%25B3-AApeptide%2520Screening%2520Library%2520against%2520EphA2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9290%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.7b01280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of β-catenin/B Cell Lymphoma 9 Protein–protein Interaction using α-helix–mimicking Sulfono-γ-AApeptide Inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10757</span>– <span class="NLM_lpage">10762</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819663116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.1819663116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=31088961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=10757-10762&author=P.+Sangauthor=M.+Zhangauthor=Y.+Shiauthor=C.+Liauthor=S.+Abdulkadirauthor=Q.+Liauthor=H.+Jiauthor=J.+Cai&title=Inhibition+of+%CE%B2-catenin%2FB+Cell+Lymphoma+9+Protein%E2%80%93protein+Interaction+using+%CE%B1-helix%E2%80%93mimicking+Sulfono-%CE%B3-AApeptide+Inhibitors&doi=10.1073%2Fpnas.1819663116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of β-catenin/B cell lymphoma 9 protein-protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span></div><div class="casAuthors">Sang, Peng; Zhang, Min; Shi, Yan; Li, Chunpu; Abdulkadir, Sami; Li, Qi; Ji, Haitao; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10757-10762</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The rational design of α-helix-mimicking peptidomimetics provides a streamlined approach to discover potent inhibitors for protein-protein interactions (PPIs).  However, designing cell-penetrating long peptidomimetic scaffolds equipped with various functional groups necessary for interacting with large protein-binding interfaces remains challenging.  This is particularly true for targeting β-catenin/BCL9 PPIs.  Here the authors designed a series of unprecedented helical sulfono-γ-AApeptides that mimic the binding mode of the α-helical HD2 domain of B Cell Lymphoma 9 (BCL9).  The studies show that sulfono-γ-AApeptides can structurally and functionally mimic the α-helical domain of BCL9 and selectively disrupt β-catenin/BCL9 PPIs with even higher potency.  More intriguingly, these sulfono-γ-AApeptides can enter cancer cells, bind with β-catenin and disrupt β-catenin/BCL9 PPIs, and exhibit excellent cellular activity, which is much more potent than the BCL9 peptide.  Furthermore, the enzymic stability studies demonstrate the remarkable stability of the helical sulfono-γ-AApeptides, with no degrdn. in the presence of pronase for 24 h, augmenting their biol. potential.  This work represents not only an example of helical sulfono-γ-AApeptides that mimic α-helix and disrupt protein-protein interactions, but also an excellent example of potent, selective, and cell-permeable unnatural foldameric peptidomimetics that disrupt the β-catenin/BCL9 PPI.  The design of helical sulfono-γ-AApeptides may lead to a new strategy to modulate a myriad of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pKlgpb-_U7Vg90H21EOLACvtfcHk0lj3ql35d97Q-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE&md5=145172aa8475bfe4a20b97d6b88d12dd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819663116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819663116%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520%25CE%25B2-catenin%252FB%2520Cell%2520Lymphoma%25209%2520Protein%25E2%2580%2593protein%2520Interaction%2520using%2520%25CE%25B1-helix%25E2%2580%2593mimicking%2520Sulfono-%25CE%25B3-AApeptide%2520Inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D10757%26epage%3D10762%26doi%3D10.1073%2Fpnas.1819663116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: Design, Synthesis and Evaluation</span>. <i>New J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1039/c0nj00943a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1039%2Fc0nj00943a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFKisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2011&pages=542-545&author=Y.+Niuauthor=Y.+Huauthor=X.+Liauthor=J.+Chenauthor=J.+Cai&title=%CE%B3-AApeptides%3A+Design%2C+Synthesis+and+Evaluation&doi=10.1039%2Fc0nj00943a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: design, synthesis and evaluation</span></div><div class="casAuthors">Niu, You-Hong; Hu, Yao-Gang; Li, Xiao-Long; Chen, Jian-Dong; Cai, Jian-Feng</div><div class="citationInfo"><span class="NLM_cas:title">New Journal of Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">542-545</span>CODEN:
                <span class="NLM_cas:coden">NJCHE5</span>;
        ISSN:<span class="NLM_cas:issn">1144-0546</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A new class of peptide mimics termed "γ-AApeptides" (derived from γ-chiral peptide nucleic acids, PNAs) has been described.  The design and synthesis of γ-AApeptides, and potential bioactivities towards p53/MDM2 interaction were demonstrated. γ-AApeptides were also found to be highly resistant to proteolysis.  The development of sequence-specific γ-AApeptides may lead to a family of peptidomimetics with a new framework for drug discovery or peptide/protein mimicry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJjedI1LYLnLVg90H21EOLACvtfcHk0ljVoUw4rv0Yqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFKisLw%253D&md5=bcc3c0c6b390ed5bc3b6019ed02191f9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1039%2Fc0nj00943a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0nj00943a%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520Design%252C%2520Synthesis%2520and%2520Evaluation%26jtitle%3DNew%2520J.%2520Chem.%26date%3D2011%26volume%3D35%26spage%3D542%26epage%3D545%26doi%3D10.1039%2Fc0nj00943a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Helical Antimicrobial Sulfono-γ-AApeptides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4802</span>– <span class="NLM_lpage">4811</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFWqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4802-4811&author=Y.+Liauthor=H.+Wuauthor=P.+Tengauthor=G.+Baiauthor=X.+Linauthor=X.+Zuoauthor=C.+Caoauthor=J.+Cai&title=Helical+Antimicrobial+Sulfono-%CE%B3-AApeptides&doi=10.1021%2Facs.jmedchem.5b00537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Antimicrobial Sulfono-γ-AApeptides</span></div><div class="casAuthors">Li, Yaqiong; Wu, Haifan; Teng, Peng; Bai, Ge; Lin, Xiaoyang; Zuo, Xiaobing; Cao, Chuanhai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4802-4811</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Host-defense peptides (HDPs) such as magainin 2 have emerged as potential therapeutic agents combating antibiotic resistance.  Inspired by their structures and mechanism of action, herein the authors report the first example of antimicrobial helical sulfono-γ-AApeptide foldamers.  The lead mol. displays broad-spectrum and potent antimicrobial activity against multi-drug-resistant Gram-pos. and Gram-neg. bacterial pathogens.  Time-kill studies and fluorescence microscopy suggest that sulfono-γ-AApeptides eradicate bacteria by taking a mode of action analogous to that of HDPs.  Clear structure-function relations exist in the studied sequences.  Longer sequences, presumably adopting more-defined helical structures, are more potent than shorter ones.  The sequence with less helical propensity in soln. could be more selective than the stronger helix-forming sequences.  Moreover, this class of antimicrobial agents are resistant to proteolytic degrdn.  These results may lead to the development of a new class of antimicrobial foldamers combating emerging antibiotic-resistant pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPELcUdcGfXrVg90H21EOLACvtfcHk0ljVoUw4rv0Yqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFWqsb4%253D&md5=5eab0445be6682f250016148fac54969</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00537%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DZuo%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHelical%2520Antimicrobial%2520Sulfono-%25CE%25B3-AApeptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4802%26epage%3D4811%26doi%3D10.1021%2Facs.jmedchem.5b00537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Hydrocarbon Stapled Peptides as Modulators of Biological Function</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1362</span>– <span class="NLM_lpage">1375</span>, <span class="refDoi"> DOI: 10.1021/cb501020r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb501020r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvFCgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1362-1375&author=P.+M.+Crommauthor=J.+Spiegelauthor=T.+N.+Grossmann&title=Hydrocarbon+Stapled+Peptides+as+Modulators+of+Biological+Function&doi=10.1021%2Fcb501020r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrocarbon Stapled Peptides as Modulators of Biological Function</span></div><div class="casAuthors">Cromm, Philipp M.; Spiegel, Jochen; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1362-1375</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Peptide-based drug discovery has experienced a significant upturn within the past decade since the introduction of chem. modifications and unnatural amino acids has allowed for overcoming some of the drawbacks assocd. with peptide therapeutics.  Strengthened by such features, modified peptides become capable of occupying a niche that emerges between the two major classes of today's therapeutics-small mols. (<500 Da) and biologics (>5000 Da).  Stabilized α-helixes have proven particularly successful at impairing disease-relevant PPIs previously considered "undruggable.".  Among those, hydrocarbon stapled α-helical peptides have emerged as a novel class of potential peptide therapeutics.  This review provides a comprehensive overview of the development and applications of hydrocarbon stapled peptides discussing the benefits and limitations of this technique.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKvg5aW_oBSbVg90H21EOLACvtfcHk0lih4Jfua7dYOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvFCgsb8%253D&md5=968add39a16b107bb83576b0a1d2c746</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fcb501020r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb501020r%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSpiegel%26aufirst%3DJ.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DHydrocarbon%2520Stapled%2520Peptides%2520as%2520Modulators%2520of%2520Biological%2520Function%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1362%26epage%3D1375%26doi%3D10.1021%2Fcb501020r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei
Araghi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, A. E.</span></span> <span> </span><span class="NLM_article-title">Rapid Optimization of Mcl-1 Inhibitors using Stapled Peptide Libraries Including Non-Natural Side Chains</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1238</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b01002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b01002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFeqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1238-1244&author=R.+Rezaei%0AAraghiauthor=J.+A.+Ryanauthor=A.+Letaiauthor=A.+E.+Keating&title=Rapid+Optimization+of+Mcl-1+Inhibitors+using+Stapled+Peptide+Libraries+Including+Non-Natural+Side+Chains&doi=10.1021%2Facschembio.5b01002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Optimization of Mcl-1 Inhibitors using Stapled Peptide Libraries Including Non-Natural Side Chains</span></div><div class="casAuthors">Rezaei Araghi, Raheleh; Ryan, Jeremy A.; Letai, Anthony; Keating, Amy E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1238-1244</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alpha helixes form a crit. part of the binding interface for many protein-protein interactions, and chem. stabilized synthetic helical peptides can be effective inhibitors of such helix-mediated complexes.  In particular, hydrocarbon stapling of peptides to generate constrained helixes can improve binding affinity and other peptide properties, but detg. the best stapled peptide variant often requires laborious trial and error.  Here, we describe the rapid discovery and optimization of a stapled-helix peptide that binds to Mcl-1, an antiapoptotic protein that is overexpressed in many chemoresistant cancers.  To accelerate discovery, we developed a peptide library synthesis and screening scheme capable of identifying subtle affinity differences among Mcl-1-binding stapled peptides.  We used our method to sample combinations of non-natural amino-acid substitutions that we introduced into Mcl-1 inhibitors in the context of a fixed helix-stabilizing hydrocarbon staple that increased peptide helical content and reduced proteolysis.  Peptides discovered in our screen contained surprising substitutions at sites that are conserved in natural binding partners.  Library-identified peptide M3d is the most potent mol. yet tested for selectively triggering mitochondrial permeabilization in Mcl-1 dependent cell lines.  Our library approach for optimizing helical peptide inhibitors can be readily applied to the study of other biomedically important targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnHiMFa67db7Vg90H21EOLACvtfcHk0ljvQP2dh0lBJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFeqs7s%253D&md5=176636fc021a45f95e93b7394dfaeca7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b01002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b01002%26sid%3Dliteratum%253Aachs%26aulast%3DRezaei%2BAraghi%26aufirst%3DR.%26aulast%3DRyan%26aufirst%3DJ.%2BA.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DKeating%26aufirst%3DA.%2BE.%26atitle%3DRapid%2520Optimization%2520of%2520Mcl-1%2520Inhibitors%2520using%2520Stapled%2520Peptide%2520Libraries%2520Including%2520Non-Natural%2520Side%2520Chains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1238%26epage%3D1244%26doi%3D10.1021%2Facschembio.5b01002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerlavais, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tovar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesavan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangurde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darlak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipovic, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feyfant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horstick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annis, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manning, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotouhi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, T. K.</span></span> <span> </span><span class="NLM_article-title">Stapled α–helical Peptide Drug Development: A Potent Dual Inhibitor of MDM2 and MDMX for p53-dependent Cancer Therapy</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">E3445</span>– <span class="NLM_lpage">E3454</span>, <span class="refDoi"> DOI: 10.1073/pnas.1303002110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.1303002110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=23946421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVKksLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=E3445-E3454&author=Y.+S.+Changauthor=B.+Gravesauthor=V.+Guerlavaisauthor=C.+Tovarauthor=K.+Packmanauthor=K.-H.+Toauthor=K.+A.+Olsonauthor=K.+Kesavanauthor=P.+Gangurdeauthor=A.+Mukherjeeauthor=T.+Bakerauthor=K.+Darlakauthor=C.+Elkinauthor=Z.+Filipovicauthor=F.+Z.+Qureshiauthor=H.+Caiauthor=P.+Berryauthor=E.+Feyfantauthor=X.+E.+Shiauthor=J.+Horstickauthor=D.+A.+Annisauthor=A.+M.+Manningauthor=N.+Fotouhiauthor=H.+Nashauthor=L.+T.+Vassilevauthor=T.+K.+Sawyer&title=Stapled+%CE%B1%E2%80%93helical+Peptide+Drug+Development%3A+A+Potent+Dual+Inhibitor+of+MDM2+and+MDMX+for+p53-dependent+Cancer+Therapy&doi=10.1073%2Fpnas.1303002110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Stapled α-helical peptide drug development: a potent dual inhibitor of MDM2 and MDMX for p53-dependent cancer therapy</span></div><div class="casAuthors">Chang, Yong S.; Graves, Bradford; Guerlavais, Vincent; Tovar, Christian; Packman, Kathryn; To, Kwong-Him; Olson, Karen A.; Kesavan, Kamala; Gangurde, Pranoti; Mukherjee, Aditi; Baker, Theresa; Darlak, Krzysztof; Elkin, Carl; Filipovic, Zoran; Qureshi, Farooq Z.; Cai, Hongliang; Berry, Pamela; Feyfant, Eric; Shi, Xiangguo E.; Horstick, James; Annis, D. Allen; Manning, Anthony M.; Fotouhi, Nader; Nash, Huw; Vassilev, Lyubomir T.; Sawyer, Tomi K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">E3445-E3454,SE3445/1-SE3445/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Stapled α-helical peptides have emerged as a promising new modality for a wide range of therapeutic targets.  Here, we report a potent and selective dual inhibitor of MDM2 and MDMX, ATSP-7041, which effectively activates the p53 pathway in tumors in vitro and in vivo.  Specifically, ATSP-7041 binds both MDM2 and MDMX with nanomolar affinities, shows submicromolar cellular activities in cancer cell lines in the presence of serum, and demonstrates highly specific, on-target mechanism of action.  A high resoln. (1.7-Å) X-ray crystal structure reveals its mol. interactions with the target protein MDMX, including multiple contacts with key amino acids as well as a role for the hydrocarbon staple itself in target engagement.  Most importantly, ATSP-7041 demonstrates robust p53-dependent tumor growth suppression in MDM2/MDMX-over-expressing xenograft cancer models, with a high correlation to on-target pharmacodynamic activity, and possesses favorable pharmacokinetic and tissue distribution properties.  Overall, ATSP-7041 demonstrates in vitro and in vivo proof-of-concept that stapled peptides can be developed as therapeutically relevant inhibitors of protein-protein interaction and may offer a viable modality for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFyYiS_tqgI7Vg90H21EOLACvtfcHk0ljvQP2dh0lBJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVKksLfN&md5=c7533088eb02a6c38ffb31a91fae9e2e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303002110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303002110%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%2BS.%26aulast%3DGraves%26aufirst%3DB.%26aulast%3DGuerlavais%26aufirst%3DV.%26aulast%3DTovar%26aufirst%3DC.%26aulast%3DPackman%26aufirst%3DK.%26aulast%3DTo%26aufirst%3DK.-H.%26aulast%3DOlson%26aufirst%3DK.%2BA.%26aulast%3DKesavan%26aufirst%3DK.%26aulast%3DGangurde%26aufirst%3DP.%26aulast%3DMukherjee%26aufirst%3DA.%26aulast%3DBaker%26aufirst%3DT.%26aulast%3DDarlak%26aufirst%3DK.%26aulast%3DElkin%26aufirst%3DC.%26aulast%3DFilipovic%26aufirst%3DZ.%26aulast%3DQureshi%26aufirst%3DF.%2BZ.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DBerry%26aufirst%3DP.%26aulast%3DFeyfant%26aufirst%3DE.%26aulast%3DShi%26aufirst%3DX.%2BE.%26aulast%3DHorstick%26aufirst%3DJ.%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DManning%26aufirst%3DA.%2BM.%26aulast%3DFotouhi%26aufirst%3DN.%26aulast%3DNash%26aufirst%3DH.%26aulast%3DVassilev%26aufirst%3DL.%2BT.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26atitle%3DStapled%2520%25CE%25B1%25E2%2580%2593helical%2520Peptide%2520Drug%2520Development%253A%2520A%2520Potent%2520Dual%2520Inhibitor%2520of%2520MDM2%2520and%2520MDMX%2520for%2520p53-dependent%2520Cancer%2520Therapy%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3DE3445%26epage%3DE3454%26doi%3D10.1073%2Fpnas.1303002110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">An Ultrahigh Affinity d-Peptide Antagonist Of MDM2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6237</span>– <span class="NLM_lpage">6241</span>, <span class="refDoi"> DOI: 10.1021/jm3005465</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3005465" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6237-6241&author=C.+Zhanauthor=L.+Zhaoauthor=X.+Weiauthor=X.+Wuauthor=X.+Chenauthor=W.+Yuanauthor=W.-Y.+Luauthor=M.+Pazgierauthor=W.+Lu&title=An+Ultrahigh+Affinity+d-Peptide+Antagonist+Of+MDM2&doi=10.1021%2Fjm3005465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An Ultrahigh Affinity d-Peptide Antagonist Of MDM2</span></div><div class="casAuthors">Zhan, Changyou; Zhao, Le; Wei, Xiaoli; Wu, Xueji; Chen, Xishan; Yuan, Weirong; Lu, Wei-Yue; Pazgier, Marzena; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6237-6241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncoprotein MDM2 neg. regulates the activity and stability of the p53 tumor suppressor and is an important mol. target for anticancer therapy.  Aided by mirror image phage display and native chem. ligation, we have previously discovered several proteolysis-resistant duodecimal d-peptide antagonists of MDM2, termed DPMI-α, β, γ.  The prototypic d-peptide inhibitor DPMI-α binds (25-109)MDM2 at an affinity of 220 nM and kills tumor cells in vitro and inhibits tumor growth in vivo by reactivating the p53 pathway.  Herein, we report the design of a superactive d-peptide antagonist of MDM2, termed DPMI-δ, of which the binding affinity for (25-109)MDM2 has been improved over DPMI-α by 3 orders of magnitude (Kd = 220 pM).  X-ray crystallog. studies validate DPMI-δ as an exceedingly potent inhibitor of the p53-MDM2 interaction, promising to be a highly attractive lead drug candidate for anticancer therapeutic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ_Li8SCEm3rVg90H21EOLACvtfcHk0lgRSe1S_Nih-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsL4%253D&md5=5dd937d5023470c7802f68dc7420af55</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm3005465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3005465%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DW.-Y.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DAn%2520Ultrahigh%2520Affinity%2520d-Peptide%2520Antagonist%2520Of%2520MDM2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6237%26epage%3D6241%26doi%3D10.1021%2Fjm3005465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grison, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aitken, D. J.</span></span> <span> </span><span class="NLM_article-title">An α-Helix-Mimicking 12,13-Helix: Designed α/β/γ-Foldamers as Selective Inhibitors of Protein–Protein Interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11096</span>– <span class="NLM_lpage">11100</span>, <span class="refDoi"> DOI: 10.1002/anie.201604517</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201604517" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1GmtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11096-11100&author=C.+M.+Grisonauthor=J.+A.+Milesauthor=S.+Robinauthor=A.+J.+Wilsonauthor=D.+J.+Aitken&title=An+%CE%B1-Helix-Mimicking+12%2C13-Helix%3A+Designed+%CE%B1%2F%CE%B2%2F%CE%B3-Foldamers+as+Selective+Inhibitors+of+Protein%E2%80%93Protein+Interactions&doi=10.1002%2Fanie.201604517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An α-Helix-Mimicking 12,13-Helix: Designed α/β/γ-Foldamers as Selective Inhibitors of Protein-Protein Interactions</span></div><div class="casAuthors">Grison, Claire M.; Miles, Jennifer A.; Robin, Sylvie; Wilson, Andrew J.; Aitken, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">11096-11100</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A major current challenge in bioorg. chem. is the identification of effective mimics of protein secondary structures that act as inhibitors of protein-protein interactions (PPIs).  In this work, trans-2-aminocyclobutanecarboxylic acid (tACBC) was used as the key β-amino acid component in the design of α/β/γ-peptides to structurally mimic a native α-helix.  Suitably functionalized α/β/γ-peptides assume an α-helix-mimicking 12,13-helix conformation in soln., exhibit enhanced proteolytic stability in comparison to the wild-type α-peptide parent sequence from which they are derived, and act as selective inhibitors of the p53/hDM2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx8w9r7WakLrVg90H21EOLACvtfcHk0li3xNLIbhpRUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1GmtbvM&md5=f7f8e78b97daa7452ae4e655056265c7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fanie.201604517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201604517%26sid%3Dliteratum%253Aachs%26aulast%3DGrison%26aufirst%3DC.%2BM.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DRobin%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DAitken%26aufirst%3DD.%2BJ.%26atitle%3DAn%2520%25CE%25B1-Helix-Mimicking%252012%252C13-Helix%253A%2520Designed%2520%25CE%25B1%252F%25CE%25B2%252F%25CE%25B3-Foldamers%2520as%2520Selective%2520Inhibitors%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11096%26epage%3D11100%26doi%3D10.1002%2Fanie.201604517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brzozowski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eswara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Helical Sulfono-γ-AApeptides with Aggregation-Induced Emission and Circularly Polarized Luminescence</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">12697</span>– <span class="NLM_lpage">12706</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b05329</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b05329" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVemurnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=12697-12706&author=Y.+Shiauthor=G.+Yinauthor=Z.+Yanauthor=P.+Sangauthor=M.+Wangauthor=R.+Brzozowskiauthor=P.+Eswaraauthor=L.+Wojtasauthor=Y.+Zhengauthor=X.+Liauthor=J.+Cai&title=Helical+Sulfono-%CE%B3-AApeptides+with+Aggregation-Induced+Emission+and+Circularly+Polarized+Luminescence&doi=10.1021%2Fjacs.9b05329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Sulfono-γ-AApeptides with Aggregation-Induced Emission and Circularly Polarized Luminescence</span></div><div class="casAuthors">Shi, Yan; Yin, Guangqiang; Yan, Zhiping; Sang, Peng; Wang, Minghui; Brzozowski, Robert; Eswara, Prahathees; Wojtas, Lukasz; Zheng, Youxuan; Li, Xiaopeng; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">12697-12706</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aggregation-induced emission (AIE) was intensively studied because of packing of small mols. and polymers; however, mid-mol.-wt. (1000-3000) mol. scaffold contg. a precise no. of AIE luminogens is rare.  Herein, we report the investigation of three tetraphenylethylene (TPE)-modified sulfono-γ-AApeptides in which multiple TPE moieties are conjugated to the chiral right-handed helical peptidomimetic backbone as functional side chains.  The crystal structure of the TPE-α/sulfono-γ-AA peptide 1 demonstrates that because of the rigid helical scaffold of the TPE-α/sulfono-γ-AA peptides, the intramol. rotations of the TPE with short linker are restricted, therefore leading to the boosted fluorescent emission in soln.  Peptides 2 and 3 exhibit aggregation-induced emission enhancement (AIEE), possibly because of the combination of both AIE and rotation restriction.  Moreover, because of their preoriented assembly induced by the right-handed helical scaffold, these emissive chiral luminogens show effective circularly polarized luminescence signals with high dissymmetry factor glum.  Finally, the amphiphilic nature of TPE-α/sulfono-γ-AA peptides could enable them to penetrate the bacterial membranes and exhibit strong fluorescence.  Their antimicrobial activity and labeling-free character could further augment their potential applications in both materials and biomedical sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygzhvkR3p3rVg90H21EOLACvtfcHk0lgi7H7zIFDMEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVemurnI&md5=8f859f2bf873cd9a02ef4576781b4362</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b05329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b05329%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DG.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DBrzozowski%26aufirst%3DR.%26aulast%3DEswara%26aufirst%3DP.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHelical%2520Sulfono-%25CE%25B3-AApeptides%2520with%2520Aggregation-Induced%2520Emission%2520and%2520Circularly%2520Polarized%2520Luminescence%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D12697%26epage%3D12706%26doi%3D10.1021%2Fjacs.9b05329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padhee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzzeo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Cellular Translocation of a γ-AApeptide Mimetic of Tat Peptide</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1534</span>, <span class="refDoi"> DOI: 10.1021/mp300070w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp300070w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1OisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=1529-1534&author=Y.+Niuauthor=G.+Baiauthor=H.+Wuauthor=R.+E.+Wangauthor=Q.+Qiaoauthor=S.+Padheeauthor=R.+Buzzeoauthor=C.+Caoauthor=J.+Cai&title=Cellular+Translocation+of+a+%CE%B3-AApeptide+Mimetic+of+Tat+Peptide&doi=10.1021%2Fmp300070w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular Translocation of a γ-AApeptide Mimetic of Tat Peptide</span></div><div class="casAuthors">Niu, Youhong; Bai, Ge; Wu, Haifan; Wang, Rongsheng E.; Qiao, Qiao; Padhee, Shruti; Buzzeo, Robert; Cao, Chuanhai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1529-1534</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cell-penetrating peptides including the trans-activating transcriptional activator (Tat) from HIV-1 have been used as carriers for intracellular delivery of a myriad of cargoes including drugs, mol. probes, DNAs and nanoparticles.  Utilizing fluorescence flow cytometry and confocal fluorescence microscopy, we demonstrate that a γ-AApeptide mimetic of Tat (48-57) can cross the cell membranes and enter the cytoplasm and nucleus of cells, with efficiency comparable to or better than that of Tat peptide (48-57).  Deletion of the four side chains of the γ-AApeptide attenuates translocation capability.  We also establish that the γ-AApeptide is even less toxic than the Tat peptide against mammalian cells.  In addn. to their low toxicity, γ-AApeptides are resistant to protease degrdn., which may prove to be advantageous over α-peptides for further development of mol. transporters for intracellular delivery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomoi3EtFaKb7Vg90H21EOLACvtfcHk0ljUS5IpA-3Atw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1OisLo%253D&md5=cbd49ecd5e45df1197ebb383887930ce</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fmp300070w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300070w%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DR.%2BE.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DPadhee%26aufirst%3DS.%26aulast%3DBuzzeo%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DCellular%2520Translocation%2520of%2520a%2520%25CE%25B3-AApeptide%2520Mimetic%2520of%2520Tat%2520Peptide%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2012%26volume%3D9%26spage%3D1529%26epage%3D1534%26doi%3D10.1021%2Fmp300070w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parag, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N. A.</span></span> <span> </span><span class="NLM_article-title">Stabilization of lncRNA GAS5 by a Small Molecule and Its Implications in Diabetic Adipocytes</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">330.e6</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1016%2Fj.chembiol.2018.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=30661991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVartb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=319-330.e6&author=Y.+Shiauthor=S.+Paragauthor=R.+Patelauthor=A.+Luiauthor=M.+Murrauthor=J.+Caiauthor=N.+A.+Patel&title=Stabilization+of+lncRNA+GAS5+by+a+Small+Molecule+and+Its+Implications+in+Diabetic+Adipocytes&doi=10.1016%2Fj.chembiol.2018.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of lncRNA GAS5 by a Small Molecule and Its Implications in Diabetic Adipocytes</span></div><div class="casAuthors">Shi, Yan; Parag, Sajan; Patel, Rekha; Lui, Ashley; Murr, Michel; Cai, Jianfeng; Patel, Niketa A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">319-330.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Long noncoding RNA (lncRNA) are regulatory RNAs >200 nt.  We previously showed that lncRNA GAS5 decreases significantly in serum of type 2 diabetes mellitus (T2DM) patients.  Hence, we sought to decipher the mol. mechanisms underlying the role of GAS5 in T2DM in adipose tissue.  Using CHIP-RIP, we demonstrate that GAS5 binds to promoter of insulin receptor to regulate its expression, and its depletion inhibits glucose uptake and insulin signaling.  Toward stabilizing GAS5 levels in T2DM, we incorporated a strategy to limit the degrdn. of GAS5 by blocking the interaction of GAS5 and UPF1 with a small mol. identified using OBTC screening strategy.  NP-C86 binds to GAS5 with high affinity, and increases GAS5 levels and glucose uptake in diabetic patient adipocytes.  As a broader impact, NP-C86 may be used as a mol. probe to investigate the intricacies of GAS5 in relevant biol. systems as it offers specificity, efficient cellular uptake and is non-cytotoxic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk-KPkGlzXi7Vg90H21EOLACvtfcHk0ljUS5IpA-3Atw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVartb8%253D&md5=5f7d6288486103bfe129bdd1a7d9e05d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DParag%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DLui%26aufirst%3DA.%26aulast%3DMurr%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DN.%2BA.%26atitle%3DStabilization%2520of%2520lncRNA%2520GAS5%2520by%2520a%2520Small%2520Molecule%2520and%2520Its%2520Implications%2520in%2520Diabetic%2520Adipocytes%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D319%26epage%3D330.e6%26doi%3D10.1016%2Fj.chembiol.2018.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borcherds, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daughdrill, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">α-Helix-Mimicking Sulfono-γ-AApeptide Inhibitors for p53–MDM2/MDMX Protein–Protein Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">975</span>– <span class="NLM_lpage">986</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00993</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00993" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1elsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=975-986&author=P.+Sangauthor=Y.+Shiauthor=J.+Luauthor=L.+Chenauthor=L.+Yangauthor=W.+Borcherdsauthor=S.+Abdulkadirauthor=Q.+Liauthor=G.+Daughdrillauthor=J.+Chenauthor=J.+Cai&title=%CE%B1-Helix-Mimicking+Sulfono-%CE%B3-AApeptide+Inhibitors+for+p53%E2%80%93MDM2%2FMDMX+Protein%E2%80%93Protein+Interactions&doi=10.1021%2Facs.jmedchem.9b00993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">α-Helix-Mimicking Sulfono-γ-AApeptide Inhibitors for p53-MDM2/MDMX Protein-Protein Interactions</span></div><div class="casAuthors">Sang, Peng; Shi, Yan; Lu, Junhao; Chen, Lihong; Yang, Leixiang; Borcherds, Wade; Abdulkadir, Sami; Li, Qi; Daughdrill, Gary; Chen, Jiandong; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">975-986</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of peptidomimetic scaffolds is a promising strategy for the inhibition of protein-protein interactions (PPIs).  Herein, we demonstrate that sulfono-γ-AApeptides can be rationally designed to mimic the p53 α-helix and inhibit p53-MDM2 PPIs.  The best inhibitor, with Kd and IC50 values of 26 nM and 0.891 μM toward MDM2, resp., is among the most potent unnatural peptidomimetic inhibitors disrupting the p53-MDM2/MDMX interaction.  Using fluorescence polarization assays, CD, NMR spectroscopy, and computational simulations, we demonstrate that sulfono-γ-AApeptides adopt helical structures resembling p53 and competitively inhibit the p53-MDM2 interaction by binding to the hydrophobic cleft of MDM2.  Intriguingly, the stapled sulfono-γ-AApeptides showed promising cellular activity by enhancing p53 transcriptional activity and inducing expression of MDM2 and p21.  Moreover, sulfono-γ-AApeptides exhibited remarkable resistance to proteolysis, augmenting their biol. potential.  Our results suggest that sulfono-γ-AApeptides are a new class of unnatural helical foldamers that disrupt PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEqdT_RC6b_LVg90H21EOLACvtfcHk0lgP6t3vG2KAVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1elsb0%253D&md5=422b83d0699b80ce888a7efd65e09ca7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00993%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DBorcherds%26aufirst%3DW.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DDaughdrill%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B1-Helix-Mimicking%2520Sulfono-%25CE%25B3-AApeptide%2520Inhibitors%2520for%2520p53%25E2%2580%2593MDM2%252FMDMX%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D975%26epage%3D986%26doi%3D10.1021%2Facs.jmedchem.9b00993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tipton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">gamma-AApeptide-based small-molecule ligands that inhibit Abeta aggregation</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5206</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1039/C3CC46685J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1039%2FC3CC46685J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=24158240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCjsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=5206-5208&author=H.+Wuauthor=Y.+Liauthor=G.+Baiauthor=Y.+Niuauthor=Q.+Qiaoauthor=J.+D.+Tiptonauthor=C.+Caoauthor=J.+Cai&title=gamma-AApeptide-based+small-molecule+ligands+that+inhibit+Abeta+aggregation&doi=10.1039%2FC3CC46685J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptide-based small-molecule ligands that inhibit Aβ aggregation</span></div><div class="casAuthors">Wu, Haifan; Li, Yaqiong; Bai, Ge; Niu, Youhong; Qiao, Qiao; Tipton, Jeremiah D.; Cao, Chuanhai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">5206-5208</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report the design, synthesis, characterization and evaluation of a novel class of γ-AApeptide one-bead-one-compd. (OBOC) library, from which a small γ-AApeptide was identified to effectively prevent and disassemble Aβ aggregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-8_VqKY2vrVg90H21EOLACvtfcHk0lgP6t3vG2KAVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCjsbw%253D&md5=f1750ac4d122bfa7ba92b23fa9250c63</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC3CC46685J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3CC46685J%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DTipton%26aufirst%3DJ.%2BD.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3Dgamma-AApeptide-based%2520small-molecule%2520ligands%2520that%2520inhibit%2520Abeta%2520aggregation%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D5206%26epage%3D5208%26doi%3D10.1039%2FC3CC46685J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Identification of novel inhibitors that disrupt STAT3-DNA interaction from a γ-AApeptide OBOC combinatorial library</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">8739</span>– <span class="NLM_lpage">8742</span>, <span class="refDoi"> DOI: 10.1039/C4CC03909B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1039%2FC4CC03909B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=24964402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVGnsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=8739-8742&author=P.+Tengauthor=X.+Zhangauthor=H.+Wuauthor=Q.+Qiaoauthor=S.+M.+Sebtiauthor=J.+Cai&title=Identification+of+novel+inhibitors+that+disrupt+STAT3-DNA+interaction+from+a+%CE%B3-AApeptide+OBOC+combinatorial+library&doi=10.1039%2FC4CC03909B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel inhibitors that disrupt STAT3-DNA interaction from a γ-AApeptide OBOC combinatorial library</span></div><div class="casAuthors">Teng, Peng; Zhang, Xiaolei; Wu, Haifan; Qiao, Qiao; Sebti, Said M.; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">63</span>),
    <span class="NLM_cas:pages">8739-8742</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">From a γ-AApeptide-based one-bead-one-compd. (OBOC) combinatorial library, the authors identified γ-AApeptides that can selectively inhibit STAT3-DNA interaction and suppress the expression levels of STAT3 target genes in intact cells.  The authors' results demonstrate that in addn. to the SH2 domain, the DNA binding domain of STAT3 is targetable for the development of a new generation of anti-cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVN0NpRV6tXrVg90H21EOLACvtfcHk0lgQg1p6-eCymw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVGnsrjM&md5=41c994aa9a985107fc1402b58d38223a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FC4CC03909B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC03909B%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520novel%2520inhibitors%2520that%2520disrupt%2520STAT3-DNA%2520interaction%2520from%2520a%2520%25CE%25B3-AApeptide%2520OBOC%2520combinatorial%2520library%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D8739%26epage%3D8742%26doi%3D10.1039%2FC4CC03909B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Hydrogen-Bonding-Driven 3D Supramolecular Assembly of Peptidomimetic Zipper</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">5661</span>– <span class="NLM_lpage">5665</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b11997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b11997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=5661-5665&author=P.+Tengauthor=Z.+Niuauthor=F.+Sheauthor=M.+Zhouauthor=P.+Sangauthor=G.+M.+Grayauthor=G.+Vermaauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=S.+Maauthor=J.+Cai&title=Hydrogen-Bonding-Driven+3D+Supramolecular+Assembly+of+Peptidomimetic+Zipper&doi=10.1021%2Fjacs.7b11997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen-Bonding-Driven 3D Supramolecular Assembly of Peptidomimetic Zipper</span></div><div class="casAuthors">Teng, Peng; Niu, Zheng; She, Fengyu; Zhou, Mi; Sang, Peng; Gray, Geoffrey M.; Verma, Gaurav; Wojtas, Lukasz; van der Vaart, Arjan; Ma, Shengqian; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5661-5665</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen-bonding-driven three-dimensional (3D) assembly of a peptidomimetic zipper has been established for the first time by using an α/AApeptide zipper that assembles into a de novo lattice arrangement through two layers of hydrogen-bonded linker-directed interactions.  Via a covalently bridged 1D 413-helix, drastic enhancement in stability has been achieved in the formed 3D cryst. supramol. architecture as evidenced by gas-sorption studies.  As the first example of an unnatural peptidic zipper, the dimensional augmentation of the zipper differs from metal-coordinated strategies, and may have general implications for the prepn. of peptidic functional materials for a variety of future applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6NOPVSQVnN7Vg90H21EOLACvtfcHk0lgQg1p6-eCymw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D&md5=75ec49a44f96107f9a2472698475157d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b11997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b11997%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DZ.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DVerma%26aufirst%3DG.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHydrogen-Bonding-Driven%25203D%2520Supramolecular%2520Assembly%2520of%2520Peptidomimetic%2520Zipper%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D5661%26epage%3D5665%26doi%3D10.1021%2Fjacs.7b11997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Tejada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjonaj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">De Novo Left-Handed Synthetic Peptidomimetic Foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9916</span>– <span class="NLM_lpage">9920</span>, <span class="refDoi"> DOI: 10.1002/anie.201805184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201805184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=9916-9920&author=F.+Sheauthor=P.+Tengauthor=A.+Peguero-Tejadaauthor=M.+Wangauthor=N.+Maauthor=T.+Odomauthor=M.+Zhouauthor=E.+Gjonajauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=De+Novo+Left-Handed+Synthetic+Peptidomimetic+Foldamers&doi=10.1002%2Fanie.201805184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">De novo left-handed synthetic peptidomimetic foldamers</span></div><div class="casAuthors">She, Fengyu; Teng, Peng; Peguero-Tejada, Alfredo; Wang, Minghui; Ma, Ning; Odom, Timothy; Zhou, Mi; Gjonaj, Erald; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9916-9920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of peptidomimetic helical foldamers with a wide repertoire of functions is of significant interest.  Herein, we report the X-ray crystal structures of a series of homogeneous L-sulfono-γ-AA foldamers and elucidate their folding conformation at the at. level.  Single-crystal X-ray crystallog. revealed that this class of oligomers fold into unprecedented dragon-boat-shaped and unexpected left-handed helixes, which are stabilized by the 14-hydrogen-bonding pattern present in all sequences.  These L-sulfono-γ-AA-peptides have a helical pitch of 5.1 Å and exactly four side chains per turn, and the side chains lie perfectly on top of each other along the helical axis.  2D NMR spectroscopy, computational simulations, and CD studies support the folding conformation in soln.  Our results provide a structural basis at the at. level for the design of novel biomimetics with a precise arrangement of functional groups in three dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2vbXzMss1rVg90H21EOLACvtfcHk0lhtVhroTp9EmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL&md5=c45cc7e77dfa2887f5fec63d8e3dbe7e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201805184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201805184%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DPeguero-Tejada%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGjonaj%26aufirst%3DE.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DDe%2520Novo%2520Left-Handed%2520Synthetic%2520Peptidomimetic%2520Foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D9916%26epage%3D9920%26doi%3D10.1002%2Fanie.201805184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7778</span>– <span class="NLM_lpage">7782</span>, <span class="refDoi"> DOI: 10.1002/anie.201903259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201903259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=7778-7782&author=P.+Tengauthor=G.+M.+Grayauthor=M.+Zhengauthor=S.+Singhauthor=X.+Liauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=Orthogonal+Halogen-Bonding-Driven+3D+Supramolecular+Assembly+of+Right-Handed+Synthetic+Helical+Peptides&doi=10.1002%2Fanie.201903259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span></div><div class="casAuthors">Teng, Peng; Gray, Geoffrey M.; Zheng, Mengmeng; Singh, Sylvia; Li, Xiaopeng; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7778-7782</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peptide-mediated self-assembly is a prevalent method for creating highly ordered supramol. architectures.  Herein, we report the first example of orthogonal C-X···X-C/C-X···π halogen bonding and hydrogen bonding driven cryst. architectures based on synthetic helical peptides bearing hybrids of L-sulfono-γ-AApeptides and natural amino acids.  The combination of halogen bonding, intra-/intermol. hydrogen bonding, and intermol. hydrophobic interactions enabled novel 3D supramol. assembly.  The orthogonal halogen bonding in the supramol. architecture exerts a novel mechanism for the self-assembly of synthetic peptide foldamers and gives new insights into mol. recognition, supramol. design, and rational design of biomimetic structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphs8_rPqx-zrVg90H21EOLACvtfcHk0lhtVhroTp9EmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D&md5=c61d38ec3f3a4e3d79d716a1c88a7ec3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201903259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201903259%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOrthogonal%2520Halogen-Bonding-Driven%25203D%2520Supramolecular%2520Assembly%2520of%2520Right-Handed%2520Synthetic%2520Helical%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D7778%26epage%3D7782%26doi%3D10.1002%2Fanie.201903259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amso, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen-Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">The Activity of Sulfono-γ- AApeptide Helical Foldamers That Mimic GLP-1</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">eaaz4988</span> <span class="refDoi"> DOI: 10.1126/sciadv.aaz4988</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1126%2Fsciadv.aaz4988" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=32440547" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&author=P.+Sangauthor=Z.+Zhouauthor=Y.+Shiauthor=C.+Leeauthor=Z.+Amsoauthor=D.+Huangauthor=T.+Odomauthor=V.+Nguyen-Tranauthor=W.+Shenauthor=J.+Cai&title=The+Activity+of+Sulfono-%CE%B3-+AApeptide+Helical+Foldamers+That+Mimic+GLP-1&doi=10.1126%2Fsciadv.aaz4988"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aaz4988&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aaz4988%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DAmso%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DNguyen-Tran%26aufirst%3DV.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DThe%2520Activity%2520of%2520Sulfono-%25CE%25B3-%2520AApeptide%2520Helical%2520Foldamers%2520That%2520Mimic%2520GLP-1%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26doi%3D10.1126%2Fsciadv.aaz4988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerrato, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Right-Handed Helical Foldamers Consisting of De Novo d-AApeptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">7363</span>– <span class="NLM_lpage">7369</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b03007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b03007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=7363-7369&author=P.+Tengauthor=N.+Maauthor=D.+C.+Cerratoauthor=F.+Sheauthor=T.+Odomauthor=X.+Wangauthor=L.+J.+Mingauthor=A.+van+der%0AVaartauthor=L.+Wojtasauthor=H.+Xuauthor=J.+Cai&title=Right-Handed+Helical+Foldamers+Consisting+of+De+Novo+d-AApeptides&doi=10.1021%2Fjacs.7b03007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Right-handed helical foldamers consisting of de novo D-AA-peptides</span></div><div class="casAuthors">Teng, Peng; Ma, Ning; Cerrato, Darrell Cole; She, Fengyu; Odom, Timothy; Wang, Xiang; Ming, Li-June; van der Vaart, Arjan; Wojtas, Lukasz; Xu, Hai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7363-7369</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New types of foldamer scaffolds are formidably challenging to design and synthesize, yet highly desirable as structural mimics of peptides/proteins with a wide repertoire of functions.  In particular, the development of peptidomimetic helical foldamers holds promise for new biomaterials, catalysts, and drug mols.  Unnatural L-sulfono-γ-AApeptides were recently developed and shown to have potential applications in both biomedical and material sciences.  However, D-sulfono-γ-AApeptides, the enantiomers of L-sulfono-γ-AApeptides, have never been studied due to the lack of high-resoln. three-dimensional structures to guide structure-based design.  Herein, we report the first synthesis and X-ray crystal structures of a series of 2:1 L-amino acid/D-sulfono-γ-AApeptide hybrid foldamers, and elucidate their folded conformation at the at. level.  Single-crystal X-ray crystallog. indicates that this class of oligomers folds into well-defined right-handed helixes with unique helical parameters.  The helical structures were consistent with data obtained from soln. 2D NMR, CD studies, and mol. dynamics simulations.  Our findings are expected to inspire the structure-based design of this type of unique folding biopolymers for biomaterials and biomedical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaGg_OJWsqYbVg90H21EOLACvtfcHk0liiwIQS053thw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D&md5=fef3f80500953f027687001f3e67f14b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b03007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b03007%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DCerrato%26aufirst%3DD.%2BC.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMing%26aufirst%3DL.%2BJ.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DRight-Handed%2520Helical%2520Foldamers%2520Consisting%2520of%2520De%2520Novo%2520d-AApeptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D7363%26epage%3D7369%26doi%3D10.1021%2Fjacs.7b03007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, D. P.</span></span> <span> </span><span class="NLM_article-title">Awakening Guardian Angels: Drugging the p53 Pathway</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">873</span>, <span class="refDoi"> DOI: 10.1038/nrc2763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1038%2Fnrc2763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=19935675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=862-873&author=C.+J.+Brownauthor=S.+Lainauthor=C.+S.+Vermaauthor=A.+R.+Fershtauthor=D.+P.+Lane&title=Awakening+Guardian+Angels%3A+Drugging+the+p53+Pathway&doi=10.1038%2Fnrc2763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Awakening guardian angels: Drugging the p53 pathway</span></div><div class="casAuthors">Brown, Christopher J.; Lain, Sonia; Verma, Chandra S.; Fersht, Alan R.; Lane, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">862-873</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Currently, around 11 million people are living with a tumor that contains an inactivating mutation of TP53 (the human gene that encodes p53) and another 11 million have tumors in which the p53 pathway is partially abrogated through the inactivation of other signaling or effector components.  The p53 pathway is therefore a prime target for new cancer drug development, and several original approaches to drug discovery that could have wide applications to drug development are being used.  In one approach, mols. that activate p53 by blocking protein-protein interactions with MDM2 are in early clin. development.  Remarkable progress has also been made in the development of p53-binding mols. that can rescue the function of certain p53 mutants.  Finally, cell-based assays are being used to discover compds. that exploit the p53 pathway by either seeking targets and compds. that show synthetic lethality with TP53 mutations or by looking for non-genotoxic activators of the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZW-elIe4w6bVg90H21EOLACvtfcHk0liiwIQS053thw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM&md5=072f48d2ed466b2deb9d2b999cc65a53</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrc2763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2763%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DC.%2BJ.%26aulast%3DLain%26aufirst%3DS.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26aulast%3DLane%26aufirst%3DD.%2BP.%26atitle%3DAwakening%2520Guardian%2520Angels%253A%2520Drugging%2520the%2520p53%2520Pathway%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D862%26epage%3D873%26doi%3D10.1038%2Fnrc2763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">Apamin as a Template for Structure-Based Rational Design of Potent Peptide Activators of p53</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">8712</span>– <span class="NLM_lpage">8715</span>, <span class="refDoi"> DOI: 10.1002/anie.200904550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.200904550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGju77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8712-8715&author=C.+Liauthor=M.+Pazgierauthor=M.+Liuauthor=W.+Y.+Luauthor=W.+Lu&title=Apamin+as+a+Template+for+Structure-Based+Rational+Design+of+Potent+Peptide+Activators+of+p53&doi=10.1002%2Fanie.200904550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Apamin as a Template for Structure-Based Rational Design of Potent Peptide Activators of p53</span></div><div class="casAuthors">Li, Chong; Pazgier, Marzena; Liu, Min; Lu, Wei-Yue; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">8712-8715, S8712/1-S8712/10</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The activity and stability of the tumor suppressor protein p53 is neg. regulated by the oncogenic proteins MDM2 and MDMX-the cellular process is initiated by MDM2/MDMX binding to the N-terminal transactivation domain of p53.  MDM2, an E3 ubiquitin ligase, primarily controls p53 stability by targeting the tumor suppressor protein for ubiquitin- mediated constitutive degrdn., whereas MDMX mainly functions as an effective transcriptional antagonist of p53 that blocks its ability to regulate responsive genes expression.  Antagonists that block the p-53-binding pocket of MDM2/ MDMX kill tumor cells both in vitro and in vivo by reactivating the p53 pathway, resulting in cell cycle arrest, senescence, or apoptosis.  MDM2 and MDMX act synergistically in tumor cells and have become highly attractive mol. targets for anticancer drug development.  Using phage display, we identified a 12-mer peptide, termed PMI (p-53-MDM2/MDMX inhibitor), with affinity for both MDM2 and MDMX in the low nM range.  Herein we report that grafting four residues of PMI crit. for MDM2/MDMX binding to apamin, a highly specific blocker of Ca2+-activated K+ channels of small conductance, converted the 18-amino acid residue bee-venom neurotoxin into several potent peptide inhibitors of the p-53-MDM2/MDMX interactions with different specificities.  The rational design of these apamin-derived p53 activators, termed stingins, was structurally validated by X-ray crystallog.  In conclusion, by grafting four hydrophobic residues, crit. for MDM2/MDMX binding, to the C-terminal a-helix of apamin, we successfully converted the bee-venom neurotoxin into a series of potent inhibitors of p53 interactions with MDM2 and MDMX.  Structural studies by X-ray crystallog. validated the mode of inhibition, that is apamin-derived stingins directly compete with p53 for MDM2/ MDMX binding.  Stingins represent a novel class of p53 activators and are superior in many aspects to the existing miniprotein antagonists of MDM2/MDMX.  Coupled with a therapeutically viable delivery modality, stingins may have the potential to be used as antitumor agents for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbV6YPq1xMkrVg90H21EOLACvtfcHk0li4p24OXYNceg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGju77P&md5=cfc5cff5f181b0178b6b1477e25eebb4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.200904550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200904550%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%2BY.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DApamin%2520as%2520a%2520Template%2520for%2520Structure-Based%2520Rational%2520Design%2520of%2520Potent%2520Peptide%2520Activators%2520of%2520p53%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D8712%26epage%3D8715%26doi%3D10.1002%2Fanie.200904550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">D-peptide Inhibitors of the p53–MDM2 Interaction for Ttargeted Molecular Therapy of Malignant Neoplasms</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">14321</span>– <span class="NLM_lpage">14326</span>, <span class="refDoi"> DOI: 10.1073/pnas.1008930107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.1008930107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=20660730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCkt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=14321-14326&author=M.+Liuauthor=C.+Liauthor=M.+Pazgierauthor=C.+Liauthor=Y.+Maoauthor=Y.+Lvauthor=B.+Guauthor=G.+Weiauthor=W.+Yuanauthor=C.+Zhanauthor=W.-Y.+Luauthor=W.+Lu&title=D-peptide+Inhibitors+of+the+p53%E2%80%93MDM2+Interaction+for+Ttargeted+Molecular+Therapy+of+Malignant+Neoplasms&doi=10.1073%2Fpnas.1008930107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">D-peptide inhibitors of the p53 - MDM2 interaction for targeted molecular therapy of malignant neoplasms</span></div><div class="casAuthors">Liu, Min; Li, Chong; Pazgier, Marzena; Li, Changqing; Mao, Yubin; Lv, Yifan; Gu, Bing; Wei, Gang; Yuan, Weirong; Zhan, Changyou; Lu, Wei-Yue; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">14321-14326, S14321/1-S14321/16</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The oncoproteins MDM2 and MDMX neg. regulate the activity and stability of the tumor suppressor protein p53, conferring tumor development and survival.  Antagonists targeting the p53-binding domains of MDM2 and MDMX kill tumor cells both in vitro and in vivo by reactivating the p53 pathway, promising a class of antitumor agents for cancer therapy.  Aided by native chem. ligation and mirror image phage display, the authors recently identified a D-peptide inhibitor of the p53-MDM2 interaction termed DPMI-α (TNWYANLEKLLR) that competes with p53 for MDM2 binding at an affinity of 219 nM.  Increased selection stringency resulted in a distinct D-peptide inhibitor termed DPMI-γ (DWWPLAFEALLR) that binds MDM2 at an affinity of 53 nM.  Structural studies coupled with mutational anal. verified the mode of action of these D-peptides as MDM2-dependent p53 activators.  Despite being resistant to proteolysis, both DPMI-α and DPMI-γ failed to actively traverse the cell membrane and, when conjugated to a cationic cell-penetrating peptide, were indiscriminately cytotoxic independently of p53 status.  When encapsulated in liposomes decorated with an integrin-targeting cyclic-RGD peptide, however DPMI-α exerted potent p53-dependent growth inhibitory activity against human glioblastoma in cell cultures and nude mouse xenograft models.  The authors' findings validate D-peptide antagonists of MDM2 as a class of p53 activators for targeted mol. therapy of malignant neoplasms harboring WT p53 and elevated levels of MDM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4a4riieVDmbVg90H21EOLACvtfcHk0li4p24OXYNceg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCkt73O&md5=fe67e00a00589a915106141be5bd8a4a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1008930107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1008930107%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DLv%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DZhan%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DW.-Y.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DD-peptide%2520Inhibitors%2520of%2520the%2520p53%25E2%2580%2593MDM2%2520Interaction%2520for%2520Ttargeted%2520Molecular%2520Therapy%2520of%2520Malignant%2520Neoplasms%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D14321%26epage%3D14326%26doi%3D10.1073%2Fpnas.1008930107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">A Left-Handed Solution to Peptide Inhibition of the p53–MDM2 Interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">3649</span>– <span class="NLM_lpage">3652</span>, <span class="refDoi"> DOI: 10.1002/anie.201000329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201000329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFWisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=3649-3652&author=M.+Liuauthor=M.+Pazgierauthor=C.+Liauthor=W.+Yuanauthor=C.+Liauthor=W.+Lu&title=A+Left-Handed+Solution+to+Peptide+Inhibition+of+the+p53%E2%80%93MDM2+Interaction&doi=10.1002%2Fanie.201000329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A Left-Handed Solution to Peptide Inhibition of the p53-MDM2 Interaction</span></div><div class="casAuthors">Liu, Min; Pazgier, Marzena; Li, Changqing; Yuan, Weirong; Li, Chong; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3649-3652, S3649/1-S3649/10</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein, we report the design, aided by mirror-image phage display (MIPD) and native chem. ligation (NCL) of a potent D-peptide ligand, termed DpMI-3 (TAWYANFEKLLR), of MDM2-the oncogenic E3 ubiquitin ligase that neg. regulates the activity and stability of the tumor suppressor protein p53.  X-ray crystallog. studies elucidated the structural basis for high-affinity D-peptide inhibition of the p-53-MDM2 interaction, and validated the mode of action of DpMI peptides as a novel class of p53 activators.  D-Peptide inhibitors are superior to many existing drug candidates in aspects such as potency, specificity, and particularly in vivo stability.  Coupled with a therapeutically viable delivery modality, DPMI-a or DPMI-P and its derivs. may have the potential to be developed into antitumor agents for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrdtmiMF1rbVg90H21EOLACvtfcHk0liR4EQAWgCENA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFWisLg%253D&md5=56738900a6934ef609bab1a7369eeaf2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fanie.201000329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201000329%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DA%2520Left-Handed%2520Solution%2520to%2520Peptide%2520Inhibition%2520of%2520the%2520p53%25E2%2580%2593MDM2%2520Interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26spage%3D3649%26epage%3D3652%26doi%3D10.1002%2Fanie.201000329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">Rapid Identification of Dual p53-MDM2/MDMX Interaction Inhibitors through Virtual Screening and Hit-based Substructure Search</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">9989</span>– <span class="NLM_lpage">9997</span>, <span class="refDoi"> DOI: 10.1039/C7RA00473G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1039%2FC7RA00473G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVOjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=9989-9997&author=S.+Chenauthor=X.+Liauthor=W.+Yuanauthor=Y.+Zouauthor=Z.+Guoauthor=Y.+Chaiauthor=W.+Lu&title=Rapid+Identification+of+Dual+p53-MDM2%2FMDMX+Interaction+Inhibitors+through+Virtual+Screening+and+Hit-based+Substructure+Search&doi=10.1039%2FC7RA00473G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid identification of dual p53-MDM2/MDMX interaction inhibitors through virtual screening and hit-based substructure search</span></div><div class="casAuthors">Chen, Si; Li, Xiang; Yuan, Weirong; Zou, Yan; Guo, Zhongwu; Chai, Yifeng; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">9989-9997</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Multi-target agents have garnered great interest over the past decade for their favorable therapeutic efficacy and drug resistance profiles.  Recently, dual inhibition of the p53 tumor suppressor interaction with its two neg. regulators MDM2 and MDMX has become an attractive anticancer approach as it can induce sustained MDM2/MDMX antagonism and robust p53 activation.  However, small mol. inhibitors with dual specificity against MDM2 and MDMX are difficult to design and are still scarce.  To identify novel scaffolds for dual inhibition of the p53-MDM2/MDMX interactions, we developed two five-point pharmacophore models for filtering the 2012 National Cancer Institute database, from which mol. docking was conducted to identify dual inhibitors.  We found 38 virtual hits and subjected them to a fluorescence polarization-based competitive binding assay, resulting in 10 active compds. of different scaffolds.  To further expand the chem. diversity of the initial hits, we performed a hit-based substructure search and identified NSC148171 from pharmacophore 1 as the most potent dual-specificity inhibitor with Ki values for MDM2 and MDMX at 0.62 and 4.6 μM.  All hits were subjected to inhibition assay of cancer cellular vitality and showed anti-proliferative activity roughly correlated with their Ki values.  This work not only yields several novel scaffolds for further structural and functional optimization of dual-specificity inhibitors of the p53-MDM2/MDMX interactions, but also showcases the power of our computational methods for small mol. anticancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZn-dghlQCLrVg90H21EOLACvtfcHk0liR4EQAWgCENA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVOjs7c%253D&md5=06c3f9e21fc11949a40d94dfdf9f74f2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2FC7RA00473G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7RA00473G%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DChai%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DRapid%2520Identification%2520of%2520Dual%2520p53-MDM2%252FMDMX%2520Interaction%2520Inhibitors%2520through%2520Virtual%2520Screening%2520and%2520Hit-based%2520Substructure%2520Search%26jtitle%3DRSC%2520Adv.%26date%3D2017%26volume%3D7%26spage%3D9989%26epage%3D9997%26doi%3D10.1039%2FC7RA00473G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monbo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarasov, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for High-affinity Peptide Inhibition of p53 Interactions with MDM2 and MDMX</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">4665</span>– <span class="NLM_lpage">4670</span>, <span class="refDoi"> DOI: 10.1073/pnas.0900947106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.0900947106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=19255450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktFOgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=4665-4670&author=M.+Pazgierauthor=M.+Liuauthor=G.+Zouauthor=W.+Yuanauthor=C.+Liauthor=C.+Liauthor=J.+Liauthor=J.+Monboauthor=D.+Zellaauthor=S.+G.+Tarasovauthor=W.+Lu&title=Structural+Basis+for+High-affinity+Peptide+Inhibition+of+p53+Interactions+with+MDM2+and+MDMX&doi=10.1073%2Fpnas.0900947106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for high-affinity peptide inhibition of p53 interactions with MDM2 and MDMX</span></div><div class="casAuthors">Pazgier, Marzena; Liu, Min; Zou, Guozhang; Yuan, Weirong; Li, Changqing; Li, Chong; Li, Jing; Monbo, Juahdi; Zella, Davide; Tarasov, Sergey G.; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4665-4670</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The oncoproteins MDM2 and MDMX neg. regulate the activity and stability of the tumor suppressor protein p53-a cellular process initiated by MDM2 and/or MDMX binding to the N-terminal transactivation domain of p53.  MDM2 and MDMX in many tumors confer p53 inactivation and tumor survival, and are important mol. targets for anticancer therapy.  We screened a duodecimal peptide phage library against site-specifically biotinylated p53-binding domains of human MDM2 and MDMX chem. synthesized via native chem. ligation, and identified several peptide inhibitors of the p53-MDM2/MDMX interactions.  The most potent inhibitor (TSFAEYWNLLSP), termed PMI, bound to MDM2 and MDMX at low nanomolar affinities - approx. 2 orders of magnitude stronger than the wild-type p53 peptide of the same length (ETFSDLWKLLPE).  We solved the crystal structures of synthetic MDM2 and MDMX, both in complex with PMI, at 1.6 Å resoln.  Comparative structural anal. identified an extensive, tightened intramol. H-bonding network in bound PMI that contributed to its conformational stability, thus enhanced binding to the 2 oncogenic proteins.  Importantly, the C-terminal residue Pro of PMI induced formation of a hydrophobic cleft in MDMX previously unseen in the structures of p53-bound MDM2 or MDMX.  Our findings deciphered the structural basis for high-affinity peptide inhibition of p53 interactions with MDM2 and MDMX, shedding new light on structure-based rational design of different classes of p53 activators for potential therapeutic use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9OZR5KzhWxbVg90H21EOLACvtfcHk0lgWtm1rpl26LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktFOgtro%253D&md5=d343128b1597cadfbb36200ac8d6a8a9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0900947106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0900947106%26sid%3Dliteratum%253Aachs%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZou%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMonbo%26aufirst%3DJ.%26aulast%3DZella%26aufirst%3DD.%26aulast%3DTarasov%26aufirst%3DS.%2BG.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DStructural%2520Basis%2520for%2520High-affinity%2520Peptide%2520Inhibition%2520of%2520p53%2520Interactions%2520with%2520MDM2%2520and%2520MDMX%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D4665%26epage%3D4670%26doi%3D10.1073%2Fpnas.0900947106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borcherds, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daughdrill, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Autoinhibition of MDMX by Intramolecular p53 Mimicry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">4624</span>– <span class="NLM_lpage">4629</span>, <span class="refDoi"> DOI: 10.1073/pnas.1420833112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.1420833112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=25825738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=4624-4629&author=L.+Chenauthor=W.+Borcherdsauthor=S.+Wuauthor=A.+Beckerauthor=E.+Schonbrunnauthor=G.+W.+Daughdrillauthor=J.+Chen&title=Autoinhibition+of+MDMX+by+Intramolecular+p53+Mimicry&doi=10.1073%2Fpnas.1420833112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Autoinhibition of MDMX by intramolecular p53 mimicry</span></div><div class="casAuthors">Chen, Lihong; Borcherds, Wade; Wu, Shaofang; Becker, Andreas; Schonbrunn, Ernst; Daughdrill, Gary W.; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4624-4629</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The p53 inhibitor MDMX is controlled by multiple stress signaling pathways.  Using a proteolytic fragment release (PFR) assay, we detected an intramol. interaction in MDMX that mechanistically mimics the interaction with p53, resulting in autoinhibition of MDMX.  This mimicry is mediated by a hydrophobic peptide located in a long disordered central segment of MDMX that has sequence similarity to the p53 transactivation domain.  NMR spectroscopy was used to show this hydrophobic peptide interacts with the N-terminal domain of MDMX in a structurally analogous manner to p53.  Mutation of two crit. tryptophan residues in the hydrophobic peptide disrupted the intramol. interaction and increased p53 binding, providing further evidence for mechanistic mimicry.  The PFR assay also revealed a second intramol. interaction between the RING domain and central region that regulates MDMX nuclear import.  These results establish the importance of intramol. interactions in MDMX regulation, and validate a new assay for the study of intramol. interactions in multidomain proteins with intrinsically disordered regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD5EJJ943LLVg90H21EOLACvtfcHk0lgWtm1rpl26LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsbs%253D&md5=7c465f4a11d5adc4e8a2e33f17ae1da0</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1420833112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1420833112%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DBorcherds%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DDaughdrill%26aufirst%3DG.%2BW.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DAutoinhibition%2520of%2520MDMX%2520by%2520Intramolecular%2520p53%2520Mimicry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D4624%26epage%3D4629%26doi%3D10.1073%2Fpnas.1420833112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Design and High-Resolution Structure of a β3-Peptide Bundle Catalyst</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">6810</span>– <span class="NLM_lpage">6813</span>, <span class="refDoi"> DOI: 10.1021/ja5013849</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja5013849" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1Ciu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=6810-6813&author=P.+S.+P.+Wangauthor=J.+B.+Nguyenauthor=A.+Schepartz&title=Design+and+High-Resolution+Structure+of+a+%CE%B23-Peptide+Bundle+Catalyst&doi=10.1021%2Fja5013849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Design and High-Resolution Structure of a β3-Peptide Bundle Catalyst</span></div><div class="casAuthors">Wang, Pam S. P.; Nguyen, Jennifer B.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6810-6813</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the widespread exploration of α-peptides as catalysts, there are few examples of β-peptides that alter a chem. transformation.  Previous work demonstrated that a special class of β3-peptides spontaneously self-assembles in water into discrete protein-like bundles possessing unique quaternary structures and exceptional thermodn. stability.  Here the authors describe β3-peptide bundles capable of both substrate binding and chem. catalysis-ester hydrolysis.  A combination of kinetic and high-resoln. structural anal. suggests an active site triad composed of residues from at least two strands of the octameric bundle structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJGELSSHUnErVg90H21EOLACvtfcHk0lg-uPvg2L2uFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1Ciu74%253D&md5=7489eb9f9626d5e958289732a577455b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja5013849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja5013849%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DP.%2BS.%2BP.%26aulast%3DNguyen%26aufirst%3DJ.%2BB.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DDesign%2520and%2520High-Resolution%2520Structure%2520of%2520a%2520%25CE%25B23-Peptide%2520Bundle%2520Catalyst%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D6810%26epage%3D6813%26doi%3D10.1021%2Fja5013849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cussol, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buratto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belorusova, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuville, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fribourg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolain, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goudreau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rochel, N.</span></span> <span> </span><span class="NLM_article-title">Structural basis for alpha-helix mimicry and inhibition of protein-protein interactions with oligourea foldamers</span>. <i>Angew Chem Int Ed Engl</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1002/anie.202008992</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.202008992" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=32935897" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=G.+Guichardauthor=L.+Cussolauthor=L.+Mauranauthor=J.+Burattoauthor=A.+Y.+Belorusovaauthor=M.+Neuvilleauthor=J.+Oszauthor=S.+Fribourgauthor=J.+Fremauxauthor=C.+Dolainauthor=S.+Goudreauauthor=N.+Rochel&title=Structural+basis+for+alpha-helix+mimicry+and+inhibition+of+protein-protein+interactions+with+oligourea+foldamers&doi=10.1002%2Fanie.202008992"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fanie.202008992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202008992%26sid%3Dliteratum%253Aachs%26aulast%3DGuichard%26aufirst%3DG.%26aulast%3DCussol%26aufirst%3DL.%26aulast%3DMauran%26aufirst%3DL.%26aulast%3DBuratto%26aufirst%3DJ.%26aulast%3DBelorusova%26aufirst%3DA.%2BY.%26aulast%3DNeuville%26aufirst%3DM.%26aulast%3DOsz%26aufirst%3DJ.%26aulast%3DFribourg%26aufirst%3DS.%26aulast%3DFremaux%26aufirst%3DJ.%26aulast%3DDolain%26aufirst%3DC.%26aulast%3DGoudreau%26aufirst%3DS.%26aulast%3DRochel%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520for%2520alpha-helix%2520mimicry%2520and%2520inhibition%2520of%2520protein-protein%2520interactions%2520with%2520oligourea%2520foldamers%26jtitle%3DAngew%2520Chem%2520Int%2520Ed%2520Engl%26date%3D2020%26doi%3D10.1002%2Fanie.202008992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Targeting Protein–protein Interactions: Lessons from p53/MDM2</span>. <i>Biopol.-Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">686</span>, <span class="refDoi"> DOI: 10.1002/bip.20741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fbip.20741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=17427181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKms7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2007&pages=657-686&author=J.+K.+Murrayauthor=S.+H.+Gellman&title=Targeting+Protein%E2%80%93protein+Interactions%3A+Lessons+from+p53%2FMDM2&doi=10.1002%2Fbip.20741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions: lessons from p53/MDM2</span></div><div class="casAuthors">Murray, Justin K.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">657-686</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The tremendous challenge of inhibiting therapeutically important protein-protein interactions has created the opportunity to extend traditional medicinal chem. to a new class of targets and to explore nontraditional strategies.  Here we review a widely studied system, the interaction between tumor suppressor p53 and its natural antagonist MDM2, for which both traditional and nontraditional approaches have been reported.  This system has been a testing ground for novel proteomimetic scaffold-based strategies, i.e., for attempts to mimic the recognition surface displayed by a folded protein with unnatural oligomers.  Retroinverso peptides, peptoids, terphenyls, β-hairpins, p-oligobenzamides, β-peptides, and miniproteins have all been explored as inhibitors of the p53/MDM2 interaction, and we focus on these oligomer-based efforts.  Traditional approaches have been successful as well, and we briefly review small mol. inhibitors along with other strategies for reactivation of the p53 pathway, for comparison with oligomer- based approaches.  We close with comments on an emerging dichotomy among protein-protein interaction targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy7SUsGRc4kbVg90H21EOLACvtfcHk0ljaUjChPsh8KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKms7nP&md5=8a70d8e515d8cae1284b9c3d1484eee3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fbip.20741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20741%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DTargeting%2520Protein%25E2%2580%2593protein%2520Interactions%253A%2520Lessons%2520from%2520p53%252FMDM2%26jtitle%3DBiopol.-Pept.%2520Sci.%26date%3D2007%26volume%3D88%26spage%3D657%26epage%3D686%26doi%3D10.1002%2Fbip.20741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lao, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarracino, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heindel, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Rational Design of Topographical Helix Mimics as Potent Inhibitors of Protein-protein Interactions</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">7877</span>– <span class="NLM_lpage">7888</span>, <span class="refDoi"> DOI: 10.1021/ja502310r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja502310r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1WltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=7877-7888&author=B.+B.+Laoauthor=K.+Drewauthor=D.+A.+Guarracinoauthor=T.+F.+Brewerauthor=D.+W.+Heindelauthor=R.+Bonneauauthor=P.+S.+Arora&title=Rational+Design+of+Topographical+Helix+Mimics+as+Potent+Inhibitors+of+Protein-protein+Interactions&doi=10.1021%2Fja502310r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Topographical Helix Mimics as Potent Inhibitors of Protein-Protein Interactions</span></div><div class="casAuthors">Lao, Brooke Bullock; Drew, Kevin; Guarracino, Danielle A.; Brewer, Thomas F.; Heindel, Daniel W.; Bonneau, Richard; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7877-7888</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interactions encompass large surface areas, but often a handful of key residues dominate the binding energy landscape.  Rationally designed small mol. scaffolds that reproduce the relative positioning and disposition of important binding residues, termed "hotspot residues", have been shown to successfully inhibit specific protein complexes.  Although this strategy has led to development of novel synthetic inhibitors of protein complexes, often direct mimicry of natural amino acid residues does not lead to potent inhibitors.  Exptl. screening of focused compd. libraries is used to further optimize inhibitors but the no. of possible designs that can be efficiently synthesized and exptl. tested in academic settings is limited.  We have applied the principles of computational protein design to optimization of nonpeptidic helix mimics as ligands for protein complexes.  We describe the development of computational tools to design helix mimetics from canonical and noncanonical residue libraries and their application to two therapeutically important protein-protein interactions: p53-MDM2 and p300-HIF1α.  The overall study provides a streamlined approach for discovering potent peptidomimetic inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNMadvCjMqobVg90H21EOLACvtfcHk0ljaUjChPsh8KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1WltLo%253D&md5=e19fc2448fcf0e6b71bdc208e1753209</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fja502310r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja502310r%26sid%3Dliteratum%253Aachs%26aulast%3DLao%26aufirst%3DB.%2BB.%26aulast%3DDrew%26aufirst%3DK.%26aulast%3DGuarracino%26aufirst%3DD.%2BA.%26aulast%3DBrewer%26aufirst%3DT.%2BF.%26aulast%3DHeindel%26aufirst%3DD.%2BW.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DRational%2520Design%2520of%2520Topographical%2520Helix%2520Mimics%2520as%2520Potent%2520Inhibitors%2520of%2520Protein-protein%2520Interactions%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D7877%26epage%3D7888%26doi%3D10.1021%2Fja502310r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kussie, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorina, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marechal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenbaas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavletich, N. P.</span></span> <span> </span><span class="NLM_article-title">Structure of the MDM2 Oncoprotein Bound to the p53 Tumor Suppressor Transactivation Domain</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">948</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1126/science.274.5289.948</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1126%2Fscience.274.5289.948" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=8875929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADyaK28XmslKnt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=948-953&author=P.+H.+Kussieauthor=S.+Gorinaauthor=V.+Marechalauthor=B.+Elenbaasauthor=J.+Moreauauthor=A.+J.+Levineauthor=N.+P.+Pavletich&title=Structure+of+the+MDM2+Oncoprotein+Bound+to+the+p53+Tumor+Suppressor+Transactivation+Domain&doi=10.1126%2Fscience.274.5289.948"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the MDM2 oncoprotein bound to the p53 tumor suppressor transactivation domain</span></div><div class="casAuthors">Kussie, Paul H.; Gorina, Svetlana; Marechal, Vincent; Elenbaas, Brian; Moreau, Jacque; Levine, Arnold J.; Pavletich, Nikola P.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5289</span>),
    <span class="NLM_cas:pages">948-953</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The MDM2 oncoprotein is a cellular inhibitor of the p53 tumor suppressor in that it can bind the transactivation domain of p53 and downregulate its ability to activate transcription.  In certain cancers, MDM2 amplification is a common event and contributes to the inactivation of p53.  The crystal structure of the 109-residue amino-terminal domain of MDM2 bound to a 15-residue transactivation domain peptide of p53 revealed that MDM2 has a deep hydrophobic cleft on which the p53 peptide binds as an amphipathic α helix.  The interface relies on the steric complementarity between the MDM2 cleft and the hydrophobic face of the p53 α helix and, in particular, on a triad of p53 amino acids - Phe19, Trp23, and Leu26 - which insert deep into the MDM2 cleft.  These same p53 residues are also involved in transactivation, supporting the hypothesis that MDM2 inactivates p53 by concealing its transactivation domain.  The structure also suggests that the amphipathic α helix may be a common structural motif in the binding of a diverse family of transactivation factors to the TATA-binding protein-assocd. factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotkRIRDnQnX7Vg90H21EOLACvtfcHk0liDeYwFe89MWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmslKnt7k%253D&md5=3702422b0a761617cff1498f47f30eeb</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5289.948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5289.948%26sid%3Dliteratum%253Aachs%26aulast%3DKussie%26aufirst%3DP.%2BH.%26aulast%3DGorina%26aufirst%3DS.%26aulast%3DMarechal%26aufirst%3DV.%26aulast%3DElenbaas%26aufirst%3DB.%26aulast%3DMoreau%26aufirst%3DJ.%26aulast%3DLevine%26aufirst%3DA.%2BJ.%26aulast%3DPavletich%26aufirst%3DN.%2BP.%26atitle%3DStructure%2520of%2520the%2520MDM2%2520Oncoprotein%2520Bound%2520to%2520the%2520p53%2520Tumor%2520Suppressor%2520Transactivation%2520Domain%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D948%26epage%3D953%26doi%3D10.1126%2Fscience.274.5289.948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murage, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J.-M.</span></span> <span> </span><span class="NLM_article-title">Development of Potent Glucagon-like Peptide-1 Agonists with High Enzyme Stability via Introduction of Multiple Lactam Bridges</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6412</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/jm100602m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100602m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVarsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6412-6420&author=E.+N.+Murageauthor=G.+Gaoauthor=A.+Biselloauthor=J.-M.+Ahn&title=Development+of+Potent+Glucagon-like+Peptide-1+Agonists+with+High+Enzyme+Stability+via+Introduction+of+Multiple+Lactam+Bridges&doi=10.1021%2Fjm100602m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent Glucagon-like Peptide-1 Agonists with High Enzyme Stability via Introduction of Multiple Lactam Bridges</span></div><div class="casAuthors">Murage, Eunice N.; Gao, Guangzu; Bisello, Alessandro; Ahn, Jung-Mo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6412-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glucagon-like peptide-1 (GLP-1) has the ability to lower the blood glucose level, and its regulatory functions make it an attractive therapeutic agent for the treatment of type 2 diabetes.  However, its rapid degrdn. by enzymes like dipeptidyl peptidase-IV (DPP-IV) and neutral endopeptidase (NEP) 24.11 severely compromises its effective clin. use.  Whereas specific DPP-IV inhibitors have been developed, NEP 24.11 targets multiple sites in the GLP-1 sequence, which makes it difficult to block.  To address this drawback, we have designed and synthesized conformationally constrained GLP-1 analogs by introducing multiple lactam bridges that stabilized both α-helixes in the N- and C-terminal regions simultaneously.  In addn. to improving the receptor activation capability (up to 5-fold) by fixing the α-helical conformations required for optimal receptor interaction, the introduced lactam bridges provided outstanding shielding over NEP 24.11 (half-life of >96 h).  These highly constrained peptides are the first examples of NEP 24.11-resistant GLP-1 analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGQuuDP1-5D7Vg90H21EOLACvtfcHk0liDeYwFe89MWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVarsbw%253D&md5=a7a8385aa91eae3f44d1231c8bb937d7</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm100602m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100602m%26sid%3Dliteratum%253Aachs%26aulast%3DMurage%26aufirst%3DE.%2BN.%26aulast%3DGao%26aufirst%3DG.%26aulast%3DBisello%26aufirst%3DA.%26aulast%3DAhn%26aufirst%3DJ.-M.%26atitle%3DDevelopment%2520of%2520Potent%2520Glucagon-like%2520Peptide-1%2520Agonists%2520with%2520High%2520Enzyme%2520Stability%2520via%2520Introduction%2520of%2520Multiple%2520Lactam%2520Bridges%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6412%26epage%3D6420%26doi%3D10.1021%2Fjm100602m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Shi</span>, <span class="hlFld-ContribAuthor ">Peng  Sang</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Cai</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of α-helix-mimicking sulfono-γ-AApeptides as p53−MDM2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (12)
                                     , 1021-1023. <a href="https://doi.org/10.4155/fmc-2021-0044" title="DOI URL">https://doi.org/10.4155/fmc-2021-0044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2021-0044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2021-0044%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B%2525CE%2525B1-helix-mimicking%252Bsulfono-%2525CE%2525B3-AApeptides%252Bas%252Bp53%2525E2%252588%252592MDM2%252Binhibitors%26aulast%3DShi%26aufirst%3DYan%26date%3D2021%26volume%3D13%26issue%3D12%26spage%3D1021%26epage%3D1023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Chemical structure of 1:1 α/Sulfono-γ-AA peptides. <b>a</b> and <b>b</b> denote the chiral side chain and the sulfonamido side chain from the building block, respectively. (B) The crystal structure of a 1:1 α/Sulfono-γ-AA peptide.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (C) Top view of panel (D,E). The schematic representation of the distribution of side chains from 1:1 α/Sulfono-γ-AA peptides. (D) Side view. (E) Top view.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Superimposition of the p53 α-helix (purple) with a designed 1:1 α/Sulfono-γ-AA peptide (blue) (a PDB file is shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_002.pdb" class="ext-link">Supporting Information</a>). (B) The structure overlay of p53 (purple) binding and MDM2 (green) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>). (C) Overlay of the p53 helix (purple) with a 1:1 α/Sulfono-γ-AA peptide (blue) on the binding of MDM2 (green) (a PDB file is shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_002.pdb" class="ext-link">Supporting Information</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Structures of the p53 and 1:1 α/Sulfono-γ-AA peptide <b>1</b>–<b>11</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. CD spectra of the p53 and compounds <b>1</b>–<b>11</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Overlay of <sup>1</sup>H–<sup>15</sup>N HSQC spectra for before (blue) and after (red) the addition of increasing concentrations of <b>9</b>. (B) Average chemical shift changes in ppm (parts per million) of the MDM2 p53 binding domain (residues 17–125) when bound to <b>9</b>. The 2° structure of MDM2<sub>17–125</sub> is shown below the graph. (C) Ribbon structure of MDM2<sub>17–125</sub> (orange) and p53TAD (green, residues 15–29) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>). Residues colored red have chemical shifts above the average of 0.048 ppm when bound to <b>9</b>. (D) Ribbon structure of MDM2<sub>17–125</sub> (orange) and p53TAD<sub>15–29</sub> (green) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>). Residues colored red have chemical shifts above the average of 0.048 ppm when bound to p53TAD<sub>1–73</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/medium/jm0c01638_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Design and synthesis of the stapled sequences (a PDB file is shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_003.pdb" class="ext-link">Supporting Information</a>). (B) The structures of the stapled 1:1 α/Sulfono-γ-AA peptides <b>12</b>–<b>15</b>. (C) Overlay of the stapled 1:1 α/Sulfono-γ-AA peptide (blue) on the binding of MDM2 (a PDB file is shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_003.pdb" class="ext-link">Supporting Information</a>). (D) Activation of cellular p53 by compounds <b>1</b> and <b>2</b> and stapled p53 helical mimetics. U2OS osteosarcoma cells expressing endogenous wild-type p53 were treated with indicated compounds at 30 μM for 16 h. The cells were analyzed by Western blot for the level of p53 pathway markers.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01638/20201105/images/large/jm0c01638_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01638&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38145" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38145" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 47 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S.</span></span> <span> </span><span class="NLM_article-title">Peptidomimetics</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1231</span>– <span class="NLM_lpage">1232</span>, <span class="refDoi"> DOI: 10.1021/ar800216e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800216e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Kiur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1231-1232&author=Y.-D.+Wuauthor=S.+Gellman&title=Peptidomimetics&doi=10.1021%2Far800216e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidomimetics</span></div><div class="casAuthors">Wu, Yun-Dong; Gellman, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1231-1232</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The emergence of peptidomimetics in research accounts during the past decade is outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz_q4rpSmjFrVg90H21EOLACvtfcHk0lhTEpDRFPfZRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Kiur3K&md5=5d2218e70afc7a52a840091448822dd9</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Far800216e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800216e%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.-D.%26aulast%3DGellman%26aufirst%3DS.%26atitle%3DPeptidomimetics%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1231%26epage%3D1232%26doi%3D10.1021%2Far800216e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinhardt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, D. C.</span></span> <span> </span><span class="NLM_article-title">Development of α-Helical Calpain Probes by Mimicking a Natural Protein–Protein Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">17704</span>– <span class="NLM_lpage">17713</span>, <span class="refDoi"> DOI: 10.1021/ja307599z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja307599z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=17704-17713&author=H.+Joauthor=N.+Meinhardtauthor=Y.+Wuauthor=S.+Kulkarniauthor=X.+Huauthor=K.+E.+Lowauthor=P.+L.+Daviesauthor=W.+F.+DeGradoauthor=D.+C.+Greenbaum&title=Development+of+%CE%B1-Helical+Calpain+Probes+by+Mimicking+a+Natural+Protein%E2%80%93Protein+Interaction&doi=10.1021%2Fja307599z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Development of α-Helical Calpain Probes by Mimicking a Natural Protein-Protein Interaction</span></div><div class="casAuthors">Jo, Hyunil; Meinhardt, Nataline; Wu, Yibing; Kulkarni, Swapnil; Hu, Xiaozhen; Low, Kristin E.; Davies, Peter L.; DeGrado, William F.; Greenbaum, Doron C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">17704-17713</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly specific inhibitor of calpain was designed by mimicking a natural protein-protein interaction between calpain and its endogenous inhibitor calpastatin.  To enable this goal a new method of stabilizing an α-helix in a small peptide was established by screening 24 com. available cross-linkers for successful cysteine alkylation in a model peptide sequence.  The effects of crosslinking on the α-helicity of selected peptides were examd. by CD and NMR spectroscopy, and revealed structurally rigid cross-linkers to be the best at stabilizing α-helixes.  This strategy was applied to the design of inhibitors of calpain that are based on calpastatin, an intrinsically unstable polypeptide that becomes structured upon binding to the enzyme.  A two-turn α-helix that binds proximal to the active site cleft was stabilized, resulting in a potent and selective inhibitor for calpain.  The utility of this inhibitor was further expanded by developing irreversible calpain family activity-based probes (ABPs), which retained the specificity of the stabilized helical inhibitor.  It is believed that the inhibitor and ABPs will be useful for future investigation of calpains, while the crosslinking technique will enable exploration of other protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigseq6FFUU7Vg90H21EOLACvtfcHk0ljgWa_0OC0bAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrfK&md5=2824ee6e61d4bb0e4202696fc57f1fa0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fja307599z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja307599z%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DH.%26aulast%3DMeinhardt%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DKulkarni%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLow%26aufirst%3DK.%2BE.%26aulast%3DDavies%26aufirst%3DP.%2BL.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26aulast%3DGreenbaum%26aufirst%3DD.%2BC.%26atitle%3DDevelopment%2520of%2520%25CE%25B1-Helical%2520Calpain%2520Probes%2520by%2520Mimicking%2520a%2520Natural%2520Protein%25E2%2580%2593Protein%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D17704%26epage%3D17713%26doi%3D10.1021%2Fja307599z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: Design, Structure, and Applications</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00492</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00492" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=428-441&author=Y.+Shiauthor=P.+Tengauthor=P.+Sangauthor=F.+Sheauthor=L.+Weiauthor=J.+Cai&title=%CE%B3-AApeptides%3A+Design%2C+Structure%2C+and+Applications&doi=10.1021%2Facs.accounts.5b00492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: Design, structure, and applications</span></div><div class="casAuthors">Shi, Yan; Teng, Peng; Sang, Peng; She, Fengyu; Wei, Lulu; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">428-441</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The development of sequence-specific peptidomimetics has led to a variety of fascinating discoveries in chem. biol.  Many peptidomimetics can mimic primary, secondary, and even tertiary structure of peptides and proteins, and because of their unnatural backbones, they also possess significantly enhanced resistance to enzymic hydrolysis, improved bioavailability, and chemodiversity.  It is known that peptide nucleic acids (PNAs) are peptidic sequences developed for the mimicry of nucleic acids; however, their unique backbone as the mol. scaffold of peptidomimetics to mimic structure and function of bioactive peptides has not been investigated systematically.  As such, we recently developed a new class of peptidomimetics, "γ-AApeptides", based on the chiral γ-PNA backbone.  They are termed γ-AApeptides because they are the oligomers of γ-substituted-N-acylated-N-aminoethyl amino acids.  Similar to other classes of peptidomimetics, γ-AApeptides are also resistant to proteolytic degrdn. and possess the potential to enhance chemodiversity.  Moreover, in our scientific journey on the exploration of this class of peptidomimetics, we have discovered some intriguing structures and functions of γ-AApeptides.  In this account, we summarize the current development and application of γ-AApeptides with biol. potential.  Briefly, both linear and cyclic (either through head-to-tail or head-to-side-chain cyclization) γ-AApeptides with diverse functional groups can be synthesized easily on the solid phase using the synthetic protocol we developed. γ-AApeptides could mimic the primary structure of peptides, as they project the same no. of side chains as peptides of the same lengths.  For instance, they could mimic the Tat peptide to permeate cell membranes and bind to HIV RNA with high specificity and affinity.  Certain γ-AApeptides show similar activity to the RGD peptide and target integrin specifically on the cell surface.  γ-AApeptides with function akin to fMLF peptides are also identified.  More importantly, we found that γ-AApeptides can fold into discrete secondary structures, such as helical and β-turn-like structures.  Therefore, they could be rationally designed for a range of biol. applications.  For instance, γ-AApeptides can mimic host-defense peptides and display potent and broad-spectrum activity toward a panel of drug-resistant bacterial pathogens.  Meanwhile, because of their stability against proteolysis and their chemodiversity, γ-AApeptides are also amenable for combinatorial screening.  We demonstrate that, through combinatorial selection, certain γ-AApeptides are identified to inhibit Aβ40 peptide aggregation, suggesting their potential use as a mol. probe to intervene in Alzheimer's disease.  In addn., a few γ-AApeptides identified from the γ-AApeptide library have been shown to bind to the DNA-binding domain of STAT3 and antagonize STAT3/DNA interactions.  Our studies suggest that, with further studies and exploration on both structures and functions, γ-AApeptides may emerge to be a new class of peptidomimetics that play an important role in chem. biol. and biomedical sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrk8PLbKcOxrVg90H21EOLACvtfcHk0ljjONV-cyg6cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D&md5=414ab3568ccfdcebb5863d876218804c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00492%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520Design%252C%2520Structure%252C%2520and%2520Applications%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D428%26epage%3D441%26doi%3D10.1021%2Facs.accounts.5b00492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span> <span> </span><span class="NLM_article-title">Universal Peptidomimetics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1021/ja1071916</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja1071916" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SrsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=462-477&author=E.+Koauthor=J.+Liuauthor=L.+M.+Perezauthor=G.+Luauthor=A.+Schaeferauthor=K.+Burgess&title=Universal+Peptidomimetics&doi=10.1021%2Fja1071916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Universal Peptidomimetics</span></div><div class="casAuthors">Ko, Eunhwa; Liu, Jing; Perez, Lisa M.; Lu, Genliang; Schaefer, Amber; Burgess, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">462-477</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptidomimetic scaffolds that can present side chains in conformations resembling those of amino acids in secondary structures without incurring excessive entropic or enthalpic penalties were discussed.  Compds. of this type were referred to as minimalist mimics.  The core hypothesis of the paper was that small sets of such scaffolds can be designed to analog local pairs of amino acids (including noncontiguous ones) in any secondary structure; i.e., they are universal peptidomimetics.  To illustrate this concept, a set of four peptidomimetic scaffolds, I-IV, were designed.  Libraries based on them were made bearing side chains corresponding to many of the protein-derived amino acids.  Modeling expts. were performed to give an indication of kinetic and thermodn. accessibilities of conformations that can mimic secondary structures.  Together, peptidomimetics based on these four scaffolds can adopt conformations that resemble almost any combination of local amino acid side chains in any secondary structure.  Universal peptidomimetics of this kind are likely to be most useful in the design of libraries for high-throughput screening against diverse targets.  Consequently, data arising from submission of these mols. to the NIH Mol. Libraries Small Mol. Repository (MLSMR) were outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjryjrFh6s0bVg90H21EOLACvtfcHk0lhqmsjvaHPSvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SrsbrM&md5=4c95046d25118b6d77e46668d88580b3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fja1071916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja1071916%26sid%3Dliteratum%253Aachs%26aulast%3DKo%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DPerez%26aufirst%3DL.%2BM.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DBurgess%26aufirst%3DK.%26atitle%3DUniversal%2520Peptidomimetics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D462%26epage%3D477%26doi%3D10.1021%2Fja1071916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, S. B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirshenbaum, K.</span></span> <span> </span><span class="NLM_article-title">Amphiphilic Cyclic Peptoids That Exhibit Antimicrobial Activity by Disrupting Staphylococcus aureus Membranes</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2013</i></span>,  <span class="NLM_fpage">3560</span>– <span class="NLM_lpage">3566</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201300077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fejoc.201300077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtFSrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2013&publication_year=2013&pages=3560-3566&author=M.+L.+Huangauthor=M.+A.+Bensonauthor=S.+B.+Y.+Shinauthor=V.+J.+Torresauthor=K.+Kirshenbaum&title=Amphiphilic+Cyclic+Peptoids+That+Exhibit+Antimicrobial+Activity+by+Disrupting+Staphylococcus+aureus+Membranes&doi=10.1002%2Fejoc.201300077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Amphiphilic cyclic peptoids that exhibit antimicrobial activity by disrupting Staphylococcus aureus membranes</span></div><div class="casAuthors">Huang, Mia L.; Benson, Meredith A.; Shin, Sung Bin Y.; Torres, Victor J.; Kirshenbaum, Kent</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2013</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3560-3566</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">There is a significant unmet need for new antimicrobial agents that can address antimicrobial resistance.  One promising group of antimicrobials is the antimicrobial peptides (AMPs) and their synthetic mimics.  In particular, synthetic sequence-specific oligomers of N-substituted glycine, termed "peptoids", have been found to show potent antimicrobial activity against bacterial pathogens in vitro and can act against the emergence of antimicrobial resistance.  In this study, the authors evaluate the antimicrobial activity of cyclic peptoid oligomers against clin. isolates of methicillin-resistant Staphylococcus aureus (MRSA).  The presence of the macrocyclic constraints can enforce a globally amphiphilic organization of the peptoid side-chains.  Several of these new amphiphilic compds. show potent and selective antimicrobial activity.  Electron microscopy expts. demonstrate that the peptoids target and damage the MRSA cytoplasmic membrane through the formation of pores.  These results substantiate the potential of peptoids as antimicrobial therapeutic agents for the treatment of S. aureus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokMvgO4N2wTLVg90H21EOLACvtfcHk0lhUj184EZ6HtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtFSrur0%253D&md5=bd4a8cc0af1010f04967e8b5576d0503</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201300077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201300077%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DM.%2BL.%26aulast%3DBenson%26aufirst%3DM.%2BA.%26aulast%3DShin%26aufirst%3DS.%2BB.%2BY.%26aulast%3DTorres%26aufirst%3DV.%2BJ.%26aulast%3DKirshenbaum%26aufirst%3DK.%26atitle%3DAmphiphilic%2520Cyclic%2520Peptoids%2520That%2520Exhibit%2520Antimicrobial%2520Activity%2520by%2520Disrupting%2520Staphylococcus%2520aureus%2520Membranes%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2013%26volume%3D2013%26spage%3D3560%26epage%3D3566%26doi%3D10.1002%2Fejoc.201300077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laursen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel-Andreasen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, C. A.</span></span> <span> </span><span class="NLM_article-title">β-Peptoid Foldamers at Last</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2696</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00257</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00257" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=2696-2704&author=J.+S.+Laursenauthor=J.+Engel-Andreasenauthor=C.+A.+Olsen&title=%CE%B2-Peptoid+Foldamers+at+Last&doi=10.1021%2Facs.accounts.5b00257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptoid foldamers at last</span></div><div class="casAuthors">Laursen, Jonas S.; Engel-Andreasen, Jens; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2696-2704</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  For a long time, peptides were considered unsuitable for drug development due to their inherently poor pharmacokinetic properties and proteolytic susceptibility.  However, this paradigm has changed significantly in the past decade with the approval of numerous antibodies and proteins as drugs.  In parallel, research in the field of synthetic mols. that are able to mimic or complement folding patterns exhibited by biopolymers, but are not recognized by proteases, have received considerable attention as well.  Such entities were coined "foldamers" by S. H. Gellman (1998).  Oligomers of N-alkylated 3-aminopropionic acid residues have been called β-peptoids due to their structural similarity to β-peptides and peptoids (N-alkylglycines), resp.  Because bona fide foldamer behavior has been demonstrated for both parent architectures, the authors wondered if the β-peptoids could serve as a successful addn. to the known ensemble of peptidomimetic foldamers.  When the authors entered this field, only the seminal description of libraries of β-peptoid dimers and trimers by B. C. Hamper et al. (1998) had been published a no. of years earlier.  Perhaps somewhat naively in retrospect, the authors envisioned that elongation of chain length combined with introduction of bulky α-chiral side-chains would deliver folded structures as reported for the α-peptoid counterparts.  Initially, the authors, and others, were unsuccessful in obtaining stable secondary structures of β-peptoid oligomers, and instead, these residues were either incorporated in cyclic structures or in combination with other types of residues to give peptidomimetic constructs with heterogeneous backbones.  Amphiphilic architectures with various membrane-targeting activities, such as mimics of antimicrobial peptides or cell-penetrating peptides, have thus been particularly successful.  The introduction of β-peptoid residues in histone deacetylase inhibitors mimicking non-ribosomal cyclotetrapeptides have also been reported.  Here, the authors sketch the scientific journey that ultimately delivered robustly folded β-peptoid oligomers.  Contributions involving biol. evaluation of peptidomimetic constructs contg. β-peptoid residues, as mentioned above, which were investigated leading up to these recently reported high-resoln. helical structures, are discussed.  On the basis of the work described here, the authors envision that β-peptoids will find future utility as peptidomimetics for biomedical investigation contg. both heterogeneous and homogeneous backbones.  The recent demonstration of control over the secondary structure of a homogeneous β-peptoid backbone now enables structure-based design of scaffolds with predictable display of desired functionalities in 3 dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT21EhTRrT-rVg90H21EOLACvtfcHk0ljHBoV88T13Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyqsr%252FO&md5=61ec8510e6cf821887c1af9695ad22b8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00257%26sid%3Dliteratum%253Aachs%26aulast%3DLaursen%26aufirst%3DJ.%2BS.%26aulast%3DEngel-Andreasen%26aufirst%3DJ.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3D%25CE%25B2-Peptoid%2520Foldamers%2520at%2520Last%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D2696%26epage%3D2704%26doi%3D10.1021%2Facs.accounts.5b00257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horne, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketas, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klasse, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Structural and Biological Mimicry of Protein Surface Recognition by α/β-peptide Foldamers</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">14751</span>– <span class="NLM_lpage">14756</span>, <span class="refDoi"> DOI: 10.1073/pnas.0902663106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.0902663106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=19706443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFGrtrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=14751-14756&author=W.+S.+Horneauthor=L.+M.+Johnsonauthor=T.+J.+Ketasauthor=P.+J.+Klasseauthor=M.+Luauthor=J.+P.+Mooreauthor=S.+H.+Gellman&title=Structural+and+Biological+Mimicry+of+Protein+Surface+Recognition+by+%CE%B1%2F%CE%B2-peptide+Foldamers&doi=10.1073%2Fpnas.0902663106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biological mimicry of protein surface recognition by α/β-peptide foldamers</span></div><div class="casAuthors">Horne, W. Seth; Johnson, Lisa M.; Ketas, Thomas J.; Klasse, Per Johan; Lu, Min; Moore, John P.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">14751-14756, S14751/1-S14751/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Unnatural oligomers that can mimic protein surfaces offer a potentially useful strategy for blocking biomedically important protein-protein interactions.  Here we evaluate an approach based on combining α- and β-amino acid residues in the context of a polypeptide sequence from the HIV protein gp41, which represents an excellent testbed because of the wealth of available structural and biol. information.  We show that α/β-peptides can mimic structural and functional properties of a crit. gp41 subunit.  Phys. studies in soln., crystallog. data, and results from cell-fusion and virus-infectivity assays collectively indicate that the gp41-mimetic α/β-peptides effectively block HIV-cell fusion via a mechanism comparable to that of gp41-derived α-peptides.  An optimized α/β-peptide is far less susceptible to proteolytic degrdn. than is an analogous α-peptide.  Our findings show how a two-stage design approach, in which sequence-based α → β replacements are followed by site-specific backbone rigidification, can lead to phys. and biol. mimicry of a natural biorecognition process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAN0drYgQtaLVg90H21EOLACvtfcHk0ljHBoV88T13Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFGrtrfP&md5=e56297c79b7388ae517515c271c0fef3</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0902663106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0902663106%26sid%3Dliteratum%253Aachs%26aulast%3DHorne%26aufirst%3DW.%2BS.%26aulast%3DJohnson%26aufirst%3DL.%2BM.%26aulast%3DKetas%26aufirst%3DT.%2BJ.%26aulast%3DKlasse%26aufirst%3DP.%2BJ.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DMoore%26aufirst%3DJ.%2BP.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DStructural%2520and%2520Biological%2520Mimicry%2520of%2520Protein%2520Surface%2520Recognition%2520by%2520%25CE%25B1%252F%25CE%25B2-peptide%2520Foldamers%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D14751%26epage%3D14756%26doi%3D10.1073%2Fpnas.0902663106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berlicki, Ł.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilsl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wéber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mándity, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrele, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fülöp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, O.</span></span> <span> </span><span class="NLM_article-title">Unique α,β- and α,α,β,β-Peptide Foldamers Based on cis-β-Aminocyclopentanecarboxylic Acid</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2208</span>– <span class="NLM_lpage">2212</span>, <span class="refDoi"> DOI: 10.1002/anie.201107702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201107702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ymtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=2208-2212&author=%C5%81.+Berlickiauthor=L.+Pilslauthor=E.+W%C3%A9berauthor=I.+M.+M%C3%A1ndityauthor=C.+Cabreleauthor=T.+A.+Martinekauthor=F.+F%C3%BCl%C3%B6pauthor=O.+Reiser&title=Unique+%CE%B1%2C%CE%B2-+and+%CE%B1%2C%CE%B1%2C%CE%B2%2C%CE%B2-Peptide+Foldamers+Based+on+cis-%CE%B2-Aminocyclopentanecarboxylic+Acid&doi=10.1002%2Fanie.201107702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Unique α,β- and α,α,β,β-peptide foldamers based on cis-β-aminocyclopentanecarboxylic acid</span></div><div class="casAuthors">Berlicki, Lukasz; Pilsl, Ludwig; Weber, Edit; Mandity, Istvan M.; Cabrele, Chiara; Martinek, Tamas A.; Fueloep, Ferenc; Reiser, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2208-2212, S2208/1-S2208/29</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Non-natural oligomers with well-defined three-dimensional structures (foldamers) raise great interest owing to their ability to mimic and expand the world of biopolymer.  In this paper we present helical peptide foldamers with defined spatial distributions of their functional groups in polar solvents including water.  Stable helical structures based on α-amino and cis-β-aminocyclopentanecarboxylic acids have been design following the stereochem. pattering approach.  All the peptides were successfully obtained by solid phase peptide synthesis and catalytic hydrogenation.  CD, NMR and theor. calcns. were employed for detn. of foldamers. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptO-sm-FJwi7Vg90H21EOLACvtfcHk0lhn6iRsINqJWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ymtL0%253D&md5=7a96f663f67145f32e5519393dd6fa4d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107702%26sid%3Dliteratum%253Aachs%26aulast%3DBerlicki%26aufirst%3D%25C5%2581.%26aulast%3DPilsl%26aufirst%3DL.%26aulast%3DW%25C3%25A9ber%26aufirst%3DE.%26aulast%3DM%25C3%25A1ndity%26aufirst%3DI.%2BM.%26aulast%3DCabrele%26aufirst%3DC.%26aulast%3DMartinek%26aufirst%3DT.%2BA.%26aulast%3DF%25C3%25BCl%25C3%25B6p%26aufirst%3DF.%26aulast%3DReiser%26aufirst%3DO.%26atitle%3DUnique%2520%25CE%25B1%252C%25CE%25B2-%2520and%2520%25CE%25B1%252C%25CE%25B1%252C%25CE%25B2%252C%25CE%25B2-Peptide%2520Foldamers%2520Based%2520on%2520cis-%25CE%25B2-Aminocyclopentanecarboxylic%2520Acid%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D2208%26epage%3D2212%26doi%3D10.1002%2Fanie.201107702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span> <span> </span><span class="NLM_article-title">α-Aminoxy Acids: New Possibilities from Foldamers to Anion Receptors and Channels</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1428</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.1021/ar8001393</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar8001393" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFSmt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1428-1438&author=X.+Liauthor=Y.-D.+Wuauthor=D.+Yang&title=%CE%B1-Aminoxy+Acids%3A+New+Possibilities+from+Foldamers+to+Anion+Receptors+and+Channels&doi=10.1021%2Far8001393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">α-Aminoxy Acids: New Possibilities from Foldamers to Anion Receptors and Channels</span></div><div class="casAuthors">Li, Xiang; Wu, Yun-Dong; Yang, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1428-1438</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Naturally occurring peptides serve important functions as enzyme inhibitors, hormones, neurotransmitters, and immunomodulators in many physiol. processes including metab., digestion, pain sensitivity, and the immune response.  However, due to their conformational flexibility and poor bioavailability, such peptides are not generally viewed as useful therapeutic agents in clin. applications.  In an effort to solve these problems, chemists have developed peptidomimetic foldamers, unnatural oligomeric mols. that fold into rigid and well-defined secondary structures mimicking the structures and biol. functions of these natural peptides.  We have designed peptidomimetic foldamers that give predictable, backbone-controlled secondary structures irresp. of the nature of the side chains.  This account presents our efforts to develop a novel class of peptidomimetic foldamers comprising α-aminooxy acids and explore their applications in the simulation of ion recognition and transport processes in living systems.  Peptides constructed from α-aminooxy acids fold according to the following rules.  First, a strong intramol. eight-membered-ring hydrogen bond forms between adjacent α-aminooxy acid residues (the α N-O turn).  The chirality of the α-carbon, not the nature of the side chains, dets. the conformation of this chiral N-O turn.  Secondly, while homochiral oligomers of α-aminooxy acids form an extended helical structure (1.88 helix), heterochiral ones adopt a bent reverse turn structure.  Thirdly, in peptides of alternating α-amino acids and α-aminooxy acids, the seven-membered-ring intramol. hydrogen bond, i.e., the γ-turn, is initiated by a succeeding α N-O turn.  Thus, this type of peptide adopts a novel 7/8 helical structure.  In investigating the potential applications of α-aminooxy acids, we have found that the amide NH units of α-aminooxy acids are more acidic than are regular amide NH groups, which makes them better hydrogen bond donors when interacting with anions.  This property makes α-aminooxy acids ideal building blocks for the construction of anion receptors.  Indeed, we have constructed both cyclic and acyclic anion receptors that have strong affinities and good (enantio-)selectivities toward chloride (Cl-) and chiral carboxylate ions.  Taking advantage of these systems' preference for Cl- ions, we have also employed α-aminooxy acid units to construct a synthetic Cl- channel that can mediate the passage of Cl- ions across cell membranes.  Continued studies of these peptidomimetic systems built from α-aminooxy acids should lead to a broad range of applications in chem., biol., medicine, and materials science.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgWTc5dmlBYrVg90H21EOLACvtfcHk0lhn6iRsINqJWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFSmt7zK&md5=edf4a8e0559a6fce8a64d18bf21a6ebe</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Far8001393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far8001393%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.-D.%26aulast%3DYang%26aufirst%3DD.%26atitle%3D%25CE%25B1-Aminoxy%2520Acids%253A%2520New%2520Possibilities%2520from%2520Foldamers%2520to%2520Anion%2520Receptors%2520and%2520Channels%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1428%26epage%3D1438%26doi%3D10.1021%2Far8001393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisblum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palecek, S. P.</span></span> <span> </span><span class="NLM_article-title">Antifungal Activity from 14-Helical β-Peptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">12630</span>– <span class="NLM_lpage">12631</span>, <span class="refDoi"> DOI: 10.1021/ja064630y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja064630y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD28XptlGlu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=12630-12631&author=A.+J.+Karlssonauthor=W.+C.+Pomerantzauthor=B.+Weisblumauthor=S.+H.+Gellmanauthor=S.+P.+Palecek&title=Antifungal+Activity+from+14-Helical+%CE%B2-Peptides&doi=10.1021%2Fja064630y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Antifungal activity from 14-helical β-peptides</span></div><div class="casAuthors">Karlsson, Amy J.; Pomerantz, William C.; Weisblum, Bernard; Gellman, Samuel H.; Palecek, Sean P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">12630-12631</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have discovered that short β-peptides (9 or 10 residues) designed to adopt globally amphiphilic helical conformations display significant antifungal activity.  The most promising β-peptides cause little lysis of human red blood cells at concns. that kill Candida albicans, a common human fungal pathogen.  Since fungi are eukaryotes, discrimination between fungal and human cells is a significant finding.  Our β-peptides are active under assay conditions that mimic physiol. ionic strength; in contrast, α-helix-forming host-defense α-peptides are inactive against C. albicans under these conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkwF12HntvOrVg90H21EOLACvtfcHk0lj2i0PHtQoeNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptlGlu7k%253D&md5=f89c05797ccd944e54831f322634e4cb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja064630y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja064630y%26sid%3Dliteratum%253Aachs%26aulast%3DKarlsson%26aufirst%3DA.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%26aulast%3DWeisblum%26aufirst%3DB.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DPalecek%26aufirst%3DS.%2BP.%26atitle%3DAntifungal%2520Activity%2520from%252014-Helical%2520%25CE%25B2-Peptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D12630%26epage%3D12631%26doi%3D10.1021%2Fja064630y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lear, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodsdon, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Helical β-Peptide Inhibitors of the p53-hDM2 Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">9468</span>– <span class="NLM_lpage">9469</span>, <span class="refDoi"> DOI: 10.1021/ja031625a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja031625a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVSksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=9468-9469&author=J.+A.+Kritzerauthor=J.+D.+Learauthor=M.+E.+Hodsdonauthor=A.+Schepartz&title=Helical+%CE%B2-Peptide+Inhibitors+of+the+p53-hDM2+Interaction&doi=10.1021%2Fja031625a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Helical β-Peptide Inhibitors of the p53-hDM2 Interaction</span></div><div class="casAuthors">Kritzer, Joshua A.; Lear, James D.; Hodsdon, Michael E.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9468-9469</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HDM2 is recognized in vivo by a short α-helix within the p53 trans-activation domain (p53AD).  Disruption of the p53·hDM2 interaction is an important goal for cancer therapy.  A functional epitope comprised of three residues on one face of the p53AD helix (F19, W23, and L26) contributes heavily to the binding free energy.  We hypothesized that the p53AD functional epitope would be recapitulated if the side chains of F19, W23, and L26 were presented at successive positions three residues apart on a stabilized β3-peptide 14-helix.  Here, we report a set of β3-peptides that possess significant 14-helix structure in water; one recognizes a cleft on the surface of hDM2 with nanomolar affinity.  The strategy for β3-peptide design that we describe is general and may have advantages over one in which individual or multiple β-amino acid substitutions are introduced into a functional α-peptide, because it is based on homol. at the level of secondary structure, not primary sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVlZAGh6JntbVg90H21EOLACvtfcHk0lj2i0PHtQoeNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVSksLo%253D&md5=3fc9e87497a50588ee4591d5f84927ea</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fja031625a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja031625a%26sid%3Dliteratum%253Aachs%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26aulast%3DLear%26aufirst%3DJ.%2BD.%26aulast%3DHodsdon%26aufirst%3DM.%2BE.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DHelical%2520%25CE%25B2-Peptide%2520Inhibitors%2520of%2520the%2520p53-hDM2%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D9468%26epage%3D9469%26doi%3D10.1021%2Fja031625a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">β-Peptides: From Structure to Function</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">3219</span>– <span class="NLM_lpage">3232</span>, <span class="refDoi"> DOI: 10.1021/cr000045i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000045i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=3219-3232&author=R.+P.+Chengauthor=S.+H.+Gellmanauthor=W.+F.+DeGrado&title=%CE%B2-Peptides%3A+From+Structure+to+Function&doi=10.1021%2Fcr000045i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptides: From structure to function</span></div><div class="casAuthors">Cheng, Richard P.; Gellman, Samuel H.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3219-3232</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 187 refs., of the conformational properties and biol. functions, esp. as peptidomimetics, of β-peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTfycZ30AaKLVg90H21EOLACvtfcHk0lgJxjrBVWYb6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D&md5=40cbf3f3d3c00ec8d6250adf407a663a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fcr000045i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000045i%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BP.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3D%25CE%25B2-Peptides%253A%2520From%2520Structure%2520to%2520Function%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D3219%26epage%3D3232%26doi%3D10.1021%2Fcr000045i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seebach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, J.</span></span> <span> </span><span class="NLM_article-title">β-Peptidic Peptidomimetics</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1366</span>– <span class="NLM_lpage">1375</span>, <span class="refDoi"> DOI: 10.1021/ar700263g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar700263g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvVKksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1366-1375&author=D.+Seebachauthor=J.+Gardiner&title=%CE%B2-Peptidic+Peptidomimetics&doi=10.1021%2Far700263g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptidic peptidomimetics</span></div><div class="casAuthors">Seebach, Dieter; Gardiner, James</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1366-1375</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  For more than a decade now, a search for answers to the following two questions has taken us on a new and exciting journey into the world of β- and γ-peptides: What happens if the O atoms in a 3i-helix of a polymeric chain composed of (R)-3-hydroxybutanoic acid are replaced by NH units.  What happens if 1 or 2 CH2 groups are introduced into each amino acid building block in the chain of a peptide or protein, thereby providing homologues of the proteinogenic α-amino acids.  Our journey has repeatedly thrown up surprises, continually expanding the potential of these classes of compd. and deepening our understanding of the structures, properties, and multifaceted functions of the natural "models" to which they are related.  β-Peptides differ from their natural counterparts, the α-peptides, by having CH2 groups inserted into every amino acid residue, either between the C=O groups and the α-C atoms (β3) or between the α-C and N atoms (β2).  The synthesis of these homologated proteinogenic amino acids and their assembly into β-peptides can be performed using known methods.  Despite the increased no. of possible conformers, the β-peptides form secondary structures (helixes, turns, sheets) even when the chain lengths are as short as 4 residues.  Furthermore, they are stable toward degrading and metabolizing enzymes in living organisms.  Linear, helical, and hairpin-type structures of β-peptides can now be designed in such a way that they resemble the characteristic and activity-related structural features ("epitopes") of corresponding natural peptides or protein sections.  This account presents examples of β-peptidic compds. binding, as agonists or antagonists (inhibitors), to (i) major histocompatibility complex (MHC) proteins (immune response), (ii) the lipid-transport protein SR-B1 (cholesterol uptake from the small intestine), (iii) the core (1-60) of interleukin-8 (inflammation), (iv) the oncoprotein RDM2, (v) the HIV gp41 fusion protein, (vi) G-protein-coupled somatostatin hsst receptors, (vii) the TNF immune response receptor CD40 (apoptosis), and (viii) DNA.  Short-chain β-peptides may be orally bioavailable and excreted from the body of mammals; long-chain β-peptides may require i.v. administration but will have longer half-lives of clearance.  It has been said that an interesting field of research distinguishes itself in that the results always throw up new questions; in this sense, the structural and biol. investigation of β-peptides has been a gold mine.  We expect that these peptidic peptidomimetics will play an increasing role in biomedical research and drug development in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0dEyPZmslHLVg90H21EOLACvtfcHk0lgJxjrBVWYb6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvVKksrk%253D&md5=e679836f00843a02103caf63773ffa80</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Far700263g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far700263g%26sid%3Dliteratum%253Aachs%26aulast%3DSeebach%26aufirst%3DD.%26aulast%3DGardiner%26aufirst%3DJ.%26atitle%3D%25CE%25B2-Peptidic%2520Peptidomimetics%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1366%26epage%3D1375%26doi%3D10.1021%2Far700263g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espina, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stouten, P. F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duke, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mousa, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">Type II‘ to Type I β-Turn Swap Changes Specificity for Integrins</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1021/ja953163+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja953163%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADyaK28XltVGjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=293-294&author=A.+C.+Bachauthor=J.+R.+Espinaauthor=S.+A.+Jacksonauthor=P.+F.+W.+Stoutenauthor=J.+L.+Dukeauthor=S.+A.+Mousaauthor=W.+F.+DeGrado&title=Type+II%E2%80%98+to+Type+I+%CE%B2-Turn+Swap+Changes+Specificity+for+Integrins&doi=10.1021%2Fja953163%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Type II' to Type I β-Turn Swap Changes Specificity for Integrins</span></div><div class="casAuthors">Bach, Alvin C., II; Espina, J. Robert; Jackson, Sharon A.; Stouten, Pieter F. W.; Duke, Jodie L.; Mousa, Shaker A.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">293-4</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotcOCqaY7emrVg90H21EOLACvtfcHk0lgOnJ61yEocOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XltVGjsA%253D%253D&md5=ded7ddfed0f53af3391fecf4607ca8d1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fja953163%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja953163%252B%26sid%3Dliteratum%253Aachs%26aulast%3DBach%26aufirst%3DA.%2BC.%26aulast%3DEspina%26aufirst%3DJ.%2BR.%26aulast%3DJackson%26aufirst%3DS.%2BA.%26aulast%3DStouten%26aufirst%3DP.%2BF.%2BW.%26aulast%3DDuke%26aufirst%3DJ.%2BL.%26aulast%3DMousa%26aufirst%3DS.%2BA.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DType%2520II%25E2%2580%2598%2520to%2520Type%2520I%2520%25CE%25B2-Turn%2520Swap%2520Changes%2520Specificity%2520for%2520Integrins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D293%26epage%3D294%26doi%3D10.1021%2Fja953163%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Challa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmagadda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">One-Bead–Two-Compound Thioether Bridged Macrocyclic γ-AApeptide Screening Library against EphA2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9290</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslKgtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9290-9298&author=Y.+Shiauthor=S.+Challaauthor=P.+Sangauthor=F.+Sheauthor=C.+Liauthor=G.+M.+Grayauthor=A.+Nimmagaddaauthor=P.+Tengauthor=T.+Odomauthor=Y.+Wangauthor=A.+van+der%0AVaartauthor=Q.+Liauthor=J.+Cai&title=One-Bead%E2%80%93Two-Compound+Thioether+Bridged+Macrocyclic+%CE%B3-AApeptide+Screening+Library+against+EphA2&doi=10.1021%2Facs.jmedchem.7b01280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">One-bead-two-compound thioether bridged macrocyclic γ-AApeptide screening library against EphA2</span></div><div class="casAuthors">Shi, Yan; Challa, Sridevi; Sang, Peng; She, Fengyu; Li, Chunpu; Gray, Geoffrey M.; Nimmagadda, Alekhya; Teng, Peng; Odom, Timothy; Wang, Yan; van der Vaart, Arjan; Li, Qi; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9290-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification of mol. ligands that recognize peptides or proteins is significant but poses a fundamental challenge in chem. biol. and biomedical sciences.  Development of cyclic peptidomimetic library is scarce, and thus discovery of cyclic peptidomimetic ligands for protein targets is rare.  Herein we report the unprecedented one-bead-two-compd. (OBTC) combinatorial library based on a novel class of the macrocyclic peptidomimetics γ-AApeptides.  In the library, we utilized the coding peptide tags synthesized with Dde-protected α-amino acids, which were orthogonal to solid phase synthesis of γ-AApeptides.  Employing the thioether linkage, the desired macrocyclic γ-AApeptides were found to be effective for ligand identification.  Screening the library against the receptor tyrosine kinase EphA2 led to the discovery of one lead compd. that tightly bound to EphA2 (Kd = 81 nM) and potently antagonized EphA2-mediated signaling.  This new approach of macrocyclic peptidomimetic library may lead to a novel platform for biomacromol. surface recognition and function modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE8_skSAKNvbVg90H21EOLACvtfcHk0lgHPnK7pMdazA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslKgtLnO&md5=0c5c86232aa6afd699fd6de5128a1ecf</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01280%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DChalla%26aufirst%3DS.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DNimmagadda%26aufirst%3DA.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOne-Bead%25E2%2580%2593Two-Compound%2520Thioether%2520Bridged%2520Macrocyclic%2520%25CE%25B3-AApeptide%2520Screening%2520Library%2520against%2520EphA2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9290%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.7b01280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of β-catenin/B Cell Lymphoma 9 Protein–protein Interaction using α-helix–mimicking Sulfono-γ-AApeptide Inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10757</span>– <span class="NLM_lpage">10762</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819663116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.1819663116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=31088961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=10757-10762&author=P.+Sangauthor=M.+Zhangauthor=Y.+Shiauthor=C.+Liauthor=S.+Abdulkadirauthor=Q.+Liauthor=H.+Jiauthor=J.+Cai&title=Inhibition+of+%CE%B2-catenin%2FB+Cell+Lymphoma+9+Protein%E2%80%93protein+Interaction+using+%CE%B1-helix%E2%80%93mimicking+Sulfono-%CE%B3-AApeptide+Inhibitors&doi=10.1073%2Fpnas.1819663116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of β-catenin/B cell lymphoma 9 protein-protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span></div><div class="casAuthors">Sang, Peng; Zhang, Min; Shi, Yan; Li, Chunpu; Abdulkadir, Sami; Li, Qi; Ji, Haitao; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10757-10762</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The rational design of α-helix-mimicking peptidomimetics provides a streamlined approach to discover potent inhibitors for protein-protein interactions (PPIs).  However, designing cell-penetrating long peptidomimetic scaffolds equipped with various functional groups necessary for interacting with large protein-binding interfaces remains challenging.  This is particularly true for targeting β-catenin/BCL9 PPIs.  Here the authors designed a series of unprecedented helical sulfono-γ-AApeptides that mimic the binding mode of the α-helical HD2 domain of B Cell Lymphoma 9 (BCL9).  The studies show that sulfono-γ-AApeptides can structurally and functionally mimic the α-helical domain of BCL9 and selectively disrupt β-catenin/BCL9 PPIs with even higher potency.  More intriguingly, these sulfono-γ-AApeptides can enter cancer cells, bind with β-catenin and disrupt β-catenin/BCL9 PPIs, and exhibit excellent cellular activity, which is much more potent than the BCL9 peptide.  Furthermore, the enzymic stability studies demonstrate the remarkable stability of the helical sulfono-γ-AApeptides, with no degrdn. in the presence of pronase for 24 h, augmenting their biol. potential.  This work represents not only an example of helical sulfono-γ-AApeptides that mimic α-helix and disrupt protein-protein interactions, but also an excellent example of potent, selective, and cell-permeable unnatural foldameric peptidomimetics that disrupt the β-catenin/BCL9 PPI.  The design of helical sulfono-γ-AApeptides may lead to a new strategy to modulate a myriad of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pKlgpb-_U7Vg90H21EOLACvtfcHk0li7G9YsfO5rCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE&md5=145172aa8475bfe4a20b97d6b88d12dd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819663116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819663116%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520%25CE%25B2-catenin%252FB%2520Cell%2520Lymphoma%25209%2520Protein%25E2%2580%2593protein%2520Interaction%2520using%2520%25CE%25B1-helix%25E2%2580%2593mimicking%2520Sulfono-%25CE%25B3-AApeptide%2520Inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D10757%26epage%3D10762%26doi%3D10.1073%2Fpnas.1819663116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: Design, Synthesis and Evaluation</span>. <i>New J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1039/c0nj00943a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1039%2Fc0nj00943a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFKisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2011&pages=542-545&author=Y.+Niuauthor=Y.+Huauthor=X.+Liauthor=J.+Chenauthor=J.+Cai&title=%CE%B3-AApeptides%3A+Design%2C+Synthesis+and+Evaluation&doi=10.1039%2Fc0nj00943a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: design, synthesis and evaluation</span></div><div class="casAuthors">Niu, You-Hong; Hu, Yao-Gang; Li, Xiao-Long; Chen, Jian-Dong; Cai, Jian-Feng</div><div class="citationInfo"><span class="NLM_cas:title">New Journal of Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">542-545</span>CODEN:
                <span class="NLM_cas:coden">NJCHE5</span>;
        ISSN:<span class="NLM_cas:issn">1144-0546</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A new class of peptide mimics termed "γ-AApeptides" (derived from γ-chiral peptide nucleic acids, PNAs) has been described.  The design and synthesis of γ-AApeptides, and potential bioactivities towards p53/MDM2 interaction were demonstrated. γ-AApeptides were also found to be highly resistant to proteolysis.  The development of sequence-specific γ-AApeptides may lead to a family of peptidomimetics with a new framework for drug discovery or peptide/protein mimicry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJjedI1LYLnLVg90H21EOLACvtfcHk0lgFhuJqjptVfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFKisLw%253D&md5=bcc3c0c6b390ed5bc3b6019ed02191f9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1039%2Fc0nj00943a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0nj00943a%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520Design%252C%2520Synthesis%2520and%2520Evaluation%26jtitle%3DNew%2520J.%2520Chem.%26date%3D2011%26volume%3D35%26spage%3D542%26epage%3D545%26doi%3D10.1039%2Fc0nj00943a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Helical Antimicrobial Sulfono-γ-AApeptides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4802</span>– <span class="NLM_lpage">4811</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFWqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4802-4811&author=Y.+Liauthor=H.+Wuauthor=P.+Tengauthor=G.+Baiauthor=X.+Linauthor=X.+Zuoauthor=C.+Caoauthor=J.+Cai&title=Helical+Antimicrobial+Sulfono-%CE%B3-AApeptides&doi=10.1021%2Facs.jmedchem.5b00537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Antimicrobial Sulfono-γ-AApeptides</span></div><div class="casAuthors">Li, Yaqiong; Wu, Haifan; Teng, Peng; Bai, Ge; Lin, Xiaoyang; Zuo, Xiaobing; Cao, Chuanhai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4802-4811</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Host-defense peptides (HDPs) such as magainin 2 have emerged as potential therapeutic agents combating antibiotic resistance.  Inspired by their structures and mechanism of action, herein the authors report the first example of antimicrobial helical sulfono-γ-AApeptide foldamers.  The lead mol. displays broad-spectrum and potent antimicrobial activity against multi-drug-resistant Gram-pos. and Gram-neg. bacterial pathogens.  Time-kill studies and fluorescence microscopy suggest that sulfono-γ-AApeptides eradicate bacteria by taking a mode of action analogous to that of HDPs.  Clear structure-function relations exist in the studied sequences.  Longer sequences, presumably adopting more-defined helical structures, are more potent than shorter ones.  The sequence with less helical propensity in soln. could be more selective than the stronger helix-forming sequences.  Moreover, this class of antimicrobial agents are resistant to proteolytic degrdn.  These results may lead to the development of a new class of antimicrobial foldamers combating emerging antibiotic-resistant pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPELcUdcGfXrVg90H21EOLACvtfcHk0lj9Ds26kVNq7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFWqsb4%253D&md5=5eab0445be6682f250016148fac54969</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00537%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DZuo%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHelical%2520Antimicrobial%2520Sulfono-%25CE%25B3-AApeptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4802%26epage%3D4811%26doi%3D10.1021%2Facs.jmedchem.5b00537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Hydrocarbon Stapled Peptides as Modulators of Biological Function</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1362</span>– <span class="NLM_lpage">1375</span>, <span class="refDoi"> DOI: 10.1021/cb501020r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb501020r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvFCgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1362-1375&author=P.+M.+Crommauthor=J.+Spiegelauthor=T.+N.+Grossmann&title=Hydrocarbon+Stapled+Peptides+as+Modulators+of+Biological+Function&doi=10.1021%2Fcb501020r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrocarbon Stapled Peptides as Modulators of Biological Function</span></div><div class="casAuthors">Cromm, Philipp M.; Spiegel, Jochen; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1362-1375</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Peptide-based drug discovery has experienced a significant upturn within the past decade since the introduction of chem. modifications and unnatural amino acids has allowed for overcoming some of the drawbacks assocd. with peptide therapeutics.  Strengthened by such features, modified peptides become capable of occupying a niche that emerges between the two major classes of today's therapeutics-small mols. (<500 Da) and biologics (>5000 Da).  Stabilized α-helixes have proven particularly successful at impairing disease-relevant PPIs previously considered "undruggable.".  Among those, hydrocarbon stapled α-helical peptides have emerged as a novel class of potential peptide therapeutics.  This review provides a comprehensive overview of the development and applications of hydrocarbon stapled peptides discussing the benefits and limitations of this technique.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKvg5aW_oBSbVg90H21EOLACvtfcHk0lj9Ds26kVNq7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvFCgsb8%253D&md5=968add39a16b107bb83576b0a1d2c746</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fcb501020r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb501020r%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSpiegel%26aufirst%3DJ.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DHydrocarbon%2520Stapled%2520Peptides%2520as%2520Modulators%2520of%2520Biological%2520Function%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1362%26epage%3D1375%26doi%3D10.1021%2Fcb501020r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei
Araghi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, A. E.</span></span> <span> </span><span class="NLM_article-title">Rapid Optimization of Mcl-1 Inhibitors using Stapled Peptide Libraries Including Non-Natural Side Chains</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1238</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b01002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b01002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFeqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1238-1244&author=R.+Rezaei%0AAraghiauthor=J.+A.+Ryanauthor=A.+Letaiauthor=A.+E.+Keating&title=Rapid+Optimization+of+Mcl-1+Inhibitors+using+Stapled+Peptide+Libraries+Including+Non-Natural+Side+Chains&doi=10.1021%2Facschembio.5b01002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Optimization of Mcl-1 Inhibitors using Stapled Peptide Libraries Including Non-Natural Side Chains</span></div><div class="casAuthors">Rezaei Araghi, Raheleh; Ryan, Jeremy A.; Letai, Anthony; Keating, Amy E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1238-1244</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alpha helixes form a crit. part of the binding interface for many protein-protein interactions, and chem. stabilized synthetic helical peptides can be effective inhibitors of such helix-mediated complexes.  In particular, hydrocarbon stapling of peptides to generate constrained helixes can improve binding affinity and other peptide properties, but detg. the best stapled peptide variant often requires laborious trial and error.  Here, we describe the rapid discovery and optimization of a stapled-helix peptide that binds to Mcl-1, an antiapoptotic protein that is overexpressed in many chemoresistant cancers.  To accelerate discovery, we developed a peptide library synthesis and screening scheme capable of identifying subtle affinity differences among Mcl-1-binding stapled peptides.  We used our method to sample combinations of non-natural amino-acid substitutions that we introduced into Mcl-1 inhibitors in the context of a fixed helix-stabilizing hydrocarbon staple that increased peptide helical content and reduced proteolysis.  Peptides discovered in our screen contained surprising substitutions at sites that are conserved in natural binding partners.  Library-identified peptide M3d is the most potent mol. yet tested for selectively triggering mitochondrial permeabilization in Mcl-1 dependent cell lines.  Our library approach for optimizing helical peptide inhibitors can be readily applied to the study of other biomedically important targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnHiMFa67db7Vg90H21EOLACvtfcHk0lhW5BpWG3qIkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFeqs7s%253D&md5=176636fc021a45f95e93b7394dfaeca7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b01002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b01002%26sid%3Dliteratum%253Aachs%26aulast%3DRezaei%2BAraghi%26aufirst%3DR.%26aulast%3DRyan%26aufirst%3DJ.%2BA.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DKeating%26aufirst%3DA.%2BE.%26atitle%3DRapid%2520Optimization%2520of%2520Mcl-1%2520Inhibitors%2520using%2520Stapled%2520Peptide%2520Libraries%2520Including%2520Non-Natural%2520Side%2520Chains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1238%26epage%3D1244%26doi%3D10.1021%2Facschembio.5b01002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerlavais, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tovar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesavan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangurde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darlak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipovic, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feyfant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horstick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annis, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manning, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotouhi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, T. K.</span></span> <span> </span><span class="NLM_article-title">Stapled α–helical Peptide Drug Development: A Potent Dual Inhibitor of MDM2 and MDMX for p53-dependent Cancer Therapy</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">E3445</span>– <span class="NLM_lpage">E3454</span>, <span class="refDoi"> DOI: 10.1073/pnas.1303002110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.1303002110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=23946421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVKksLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=E3445-E3454&author=Y.+S.+Changauthor=B.+Gravesauthor=V.+Guerlavaisauthor=C.+Tovarauthor=K.+Packmanauthor=K.-H.+Toauthor=K.+A.+Olsonauthor=K.+Kesavanauthor=P.+Gangurdeauthor=A.+Mukherjeeauthor=T.+Bakerauthor=K.+Darlakauthor=C.+Elkinauthor=Z.+Filipovicauthor=F.+Z.+Qureshiauthor=H.+Caiauthor=P.+Berryauthor=E.+Feyfantauthor=X.+E.+Shiauthor=J.+Horstickauthor=D.+A.+Annisauthor=A.+M.+Manningauthor=N.+Fotouhiauthor=H.+Nashauthor=L.+T.+Vassilevauthor=T.+K.+Sawyer&title=Stapled+%CE%B1%E2%80%93helical+Peptide+Drug+Development%3A+A+Potent+Dual+Inhibitor+of+MDM2+and+MDMX+for+p53-dependent+Cancer+Therapy&doi=10.1073%2Fpnas.1303002110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Stapled α-helical peptide drug development: a potent dual inhibitor of MDM2 and MDMX for p53-dependent cancer therapy</span></div><div class="casAuthors">Chang, Yong S.; Graves, Bradford; Guerlavais, Vincent; Tovar, Christian; Packman, Kathryn; To, Kwong-Him; Olson, Karen A.; Kesavan, Kamala; Gangurde, Pranoti; Mukherjee, Aditi; Baker, Theresa; Darlak, Krzysztof; Elkin, Carl; Filipovic, Zoran; Qureshi, Farooq Z.; Cai, Hongliang; Berry, Pamela; Feyfant, Eric; Shi, Xiangguo E.; Horstick, James; Annis, D. Allen; Manning, Anthony M.; Fotouhi, Nader; Nash, Huw; Vassilev, Lyubomir T.; Sawyer, Tomi K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">E3445-E3454,SE3445/1-SE3445/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Stapled α-helical peptides have emerged as a promising new modality for a wide range of therapeutic targets.  Here, we report a potent and selective dual inhibitor of MDM2 and MDMX, ATSP-7041, which effectively activates the p53 pathway in tumors in vitro and in vivo.  Specifically, ATSP-7041 binds both MDM2 and MDMX with nanomolar affinities, shows submicromolar cellular activities in cancer cell lines in the presence of serum, and demonstrates highly specific, on-target mechanism of action.  A high resoln. (1.7-Å) X-ray crystal structure reveals its mol. interactions with the target protein MDMX, including multiple contacts with key amino acids as well as a role for the hydrocarbon staple itself in target engagement.  Most importantly, ATSP-7041 demonstrates robust p53-dependent tumor growth suppression in MDM2/MDMX-over-expressing xenograft cancer models, with a high correlation to on-target pharmacodynamic activity, and possesses favorable pharmacokinetic and tissue distribution properties.  Overall, ATSP-7041 demonstrates in vitro and in vivo proof-of-concept that stapled peptides can be developed as therapeutically relevant inhibitors of protein-protein interaction and may offer a viable modality for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFyYiS_tqgI7Vg90H21EOLACvtfcHk0ljXpRW-Vz395w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVKksLfN&md5=c7533088eb02a6c38ffb31a91fae9e2e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303002110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303002110%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%2BS.%26aulast%3DGraves%26aufirst%3DB.%26aulast%3DGuerlavais%26aufirst%3DV.%26aulast%3DTovar%26aufirst%3DC.%26aulast%3DPackman%26aufirst%3DK.%26aulast%3DTo%26aufirst%3DK.-H.%26aulast%3DOlson%26aufirst%3DK.%2BA.%26aulast%3DKesavan%26aufirst%3DK.%26aulast%3DGangurde%26aufirst%3DP.%26aulast%3DMukherjee%26aufirst%3DA.%26aulast%3DBaker%26aufirst%3DT.%26aulast%3DDarlak%26aufirst%3DK.%26aulast%3DElkin%26aufirst%3DC.%26aulast%3DFilipovic%26aufirst%3DZ.%26aulast%3DQureshi%26aufirst%3DF.%2BZ.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DBerry%26aufirst%3DP.%26aulast%3DFeyfant%26aufirst%3DE.%26aulast%3DShi%26aufirst%3DX.%2BE.%26aulast%3DHorstick%26aufirst%3DJ.%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DManning%26aufirst%3DA.%2BM.%26aulast%3DFotouhi%26aufirst%3DN.%26aulast%3DNash%26aufirst%3DH.%26aulast%3DVassilev%26aufirst%3DL.%2BT.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26atitle%3DStapled%2520%25CE%25B1%25E2%2580%2593helical%2520Peptide%2520Drug%2520Development%253A%2520A%2520Potent%2520Dual%2520Inhibitor%2520of%2520MDM2%2520and%2520MDMX%2520for%2520p53-dependent%2520Cancer%2520Therapy%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3DE3445%26epage%3DE3454%26doi%3D10.1073%2Fpnas.1303002110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">An Ultrahigh Affinity d-Peptide Antagonist Of MDM2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6237</span>– <span class="NLM_lpage">6241</span>, <span class="refDoi"> DOI: 10.1021/jm3005465</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3005465" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6237-6241&author=C.+Zhanauthor=L.+Zhaoauthor=X.+Weiauthor=X.+Wuauthor=X.+Chenauthor=W.+Yuanauthor=W.-Y.+Luauthor=M.+Pazgierauthor=W.+Lu&title=An+Ultrahigh+Affinity+d-Peptide+Antagonist+Of+MDM2&doi=10.1021%2Fjm3005465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An Ultrahigh Affinity d-Peptide Antagonist Of MDM2</span></div><div class="casAuthors">Zhan, Changyou; Zhao, Le; Wei, Xiaoli; Wu, Xueji; Chen, Xishan; Yuan, Weirong; Lu, Wei-Yue; Pazgier, Marzena; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6237-6241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncoprotein MDM2 neg. regulates the activity and stability of the p53 tumor suppressor and is an important mol. target for anticancer therapy.  Aided by mirror image phage display and native chem. ligation, we have previously discovered several proteolysis-resistant duodecimal d-peptide antagonists of MDM2, termed DPMI-α, β, γ.  The prototypic d-peptide inhibitor DPMI-α binds (25-109)MDM2 at an affinity of 220 nM and kills tumor cells in vitro and inhibits tumor growth in vivo by reactivating the p53 pathway.  Herein, we report the design of a superactive d-peptide antagonist of MDM2, termed DPMI-δ, of which the binding affinity for (25-109)MDM2 has been improved over DPMI-α by 3 orders of magnitude (Kd = 220 pM).  X-ray crystallog. studies validate DPMI-δ as an exceedingly potent inhibitor of the p53-MDM2 interaction, promising to be a highly attractive lead drug candidate for anticancer therapeutic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ_Li8SCEm3rVg90H21EOLACvtfcHk0liVl88pYb2u1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsL4%253D&md5=5dd937d5023470c7802f68dc7420af55</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm3005465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3005465%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DW.-Y.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DAn%2520Ultrahigh%2520Affinity%2520d-Peptide%2520Antagonist%2520Of%2520MDM2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6237%26epage%3D6241%26doi%3D10.1021%2Fjm3005465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grison, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aitken, D. J.</span></span> <span> </span><span class="NLM_article-title">An α-Helix-Mimicking 12,13-Helix: Designed α/β/γ-Foldamers as Selective Inhibitors of Protein–Protein Interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11096</span>– <span class="NLM_lpage">11100</span>, <span class="refDoi"> DOI: 10.1002/anie.201604517</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201604517" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1GmtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11096-11100&author=C.+M.+Grisonauthor=J.+A.+Milesauthor=S.+Robinauthor=A.+J.+Wilsonauthor=D.+J.+Aitken&title=An+%CE%B1-Helix-Mimicking+12%2C13-Helix%3A+Designed+%CE%B1%2F%CE%B2%2F%CE%B3-Foldamers+as+Selective+Inhibitors+of+Protein%E2%80%93Protein+Interactions&doi=10.1002%2Fanie.201604517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An α-Helix-Mimicking 12,13-Helix: Designed α/β/γ-Foldamers as Selective Inhibitors of Protein-Protein Interactions</span></div><div class="casAuthors">Grison, Claire M.; Miles, Jennifer A.; Robin, Sylvie; Wilson, Andrew J.; Aitken, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">11096-11100</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A major current challenge in bioorg. chem. is the identification of effective mimics of protein secondary structures that act as inhibitors of protein-protein interactions (PPIs).  In this work, trans-2-aminocyclobutanecarboxylic acid (tACBC) was used as the key β-amino acid component in the design of α/β/γ-peptides to structurally mimic a native α-helix.  Suitably functionalized α/β/γ-peptides assume an α-helix-mimicking 12,13-helix conformation in soln., exhibit enhanced proteolytic stability in comparison to the wild-type α-peptide parent sequence from which they are derived, and act as selective inhibitors of the p53/hDM2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx8w9r7WakLrVg90H21EOLACvtfcHk0lgGwoR_uruanA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1GmtbvM&md5=f7f8e78b97daa7452ae4e655056265c7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fanie.201604517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201604517%26sid%3Dliteratum%253Aachs%26aulast%3DGrison%26aufirst%3DC.%2BM.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DRobin%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DAitken%26aufirst%3DD.%2BJ.%26atitle%3DAn%2520%25CE%25B1-Helix-Mimicking%252012%252C13-Helix%253A%2520Designed%2520%25CE%25B1%252F%25CE%25B2%252F%25CE%25B3-Foldamers%2520as%2520Selective%2520Inhibitors%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11096%26epage%3D11100%26doi%3D10.1002%2Fanie.201604517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brzozowski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eswara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Helical Sulfono-γ-AApeptides with Aggregation-Induced Emission and Circularly Polarized Luminescence</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">12697</span>– <span class="NLM_lpage">12706</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b05329</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b05329" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVemurnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=12697-12706&author=Y.+Shiauthor=G.+Yinauthor=Z.+Yanauthor=P.+Sangauthor=M.+Wangauthor=R.+Brzozowskiauthor=P.+Eswaraauthor=L.+Wojtasauthor=Y.+Zhengauthor=X.+Liauthor=J.+Cai&title=Helical+Sulfono-%CE%B3-AApeptides+with+Aggregation-Induced+Emission+and+Circularly+Polarized+Luminescence&doi=10.1021%2Fjacs.9b05329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Sulfono-γ-AApeptides with Aggregation-Induced Emission and Circularly Polarized Luminescence</span></div><div class="casAuthors">Shi, Yan; Yin, Guangqiang; Yan, Zhiping; Sang, Peng; Wang, Minghui; Brzozowski, Robert; Eswara, Prahathees; Wojtas, Lukasz; Zheng, Youxuan; Li, Xiaopeng; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">12697-12706</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aggregation-induced emission (AIE) was intensively studied because of packing of small mols. and polymers; however, mid-mol.-wt. (1000-3000) mol. scaffold contg. a precise no. of AIE luminogens is rare.  Herein, we report the investigation of three tetraphenylethylene (TPE)-modified sulfono-γ-AApeptides in which multiple TPE moieties are conjugated to the chiral right-handed helical peptidomimetic backbone as functional side chains.  The crystal structure of the TPE-α/sulfono-γ-AA peptide 1 demonstrates that because of the rigid helical scaffold of the TPE-α/sulfono-γ-AA peptides, the intramol. rotations of the TPE with short linker are restricted, therefore leading to the boosted fluorescent emission in soln.  Peptides 2 and 3 exhibit aggregation-induced emission enhancement (AIEE), possibly because of the combination of both AIE and rotation restriction.  Moreover, because of their preoriented assembly induced by the right-handed helical scaffold, these emissive chiral luminogens show effective circularly polarized luminescence signals with high dissymmetry factor glum.  Finally, the amphiphilic nature of TPE-α/sulfono-γ-AA peptides could enable them to penetrate the bacterial membranes and exhibit strong fluorescence.  Their antimicrobial activity and labeling-free character could further augment their potential applications in both materials and biomedical sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygzhvkR3p3rVg90H21EOLACvtfcHk0lhtwuDe9D-64A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVemurnI&md5=8f859f2bf873cd9a02ef4576781b4362</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b05329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b05329%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DG.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DBrzozowski%26aufirst%3DR.%26aulast%3DEswara%26aufirst%3DP.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHelical%2520Sulfono-%25CE%25B3-AApeptides%2520with%2520Aggregation-Induced%2520Emission%2520and%2520Circularly%2520Polarized%2520Luminescence%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D12697%26epage%3D12706%26doi%3D10.1021%2Fjacs.9b05329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padhee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzzeo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Cellular Translocation of a γ-AApeptide Mimetic of Tat Peptide</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1534</span>, <span class="refDoi"> DOI: 10.1021/mp300070w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp300070w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1OisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=1529-1534&author=Y.+Niuauthor=G.+Baiauthor=H.+Wuauthor=R.+E.+Wangauthor=Q.+Qiaoauthor=S.+Padheeauthor=R.+Buzzeoauthor=C.+Caoauthor=J.+Cai&title=Cellular+Translocation+of+a+%CE%B3-AApeptide+Mimetic+of+Tat+Peptide&doi=10.1021%2Fmp300070w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular Translocation of a γ-AApeptide Mimetic of Tat Peptide</span></div><div class="casAuthors">Niu, Youhong; Bai, Ge; Wu, Haifan; Wang, Rongsheng E.; Qiao, Qiao; Padhee, Shruti; Buzzeo, Robert; Cao, Chuanhai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1529-1534</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cell-penetrating peptides including the trans-activating transcriptional activator (Tat) from HIV-1 have been used as carriers for intracellular delivery of a myriad of cargoes including drugs, mol. probes, DNAs and nanoparticles.  Utilizing fluorescence flow cytometry and confocal fluorescence microscopy, we demonstrate that a γ-AApeptide mimetic of Tat (48-57) can cross the cell membranes and enter the cytoplasm and nucleus of cells, with efficiency comparable to or better than that of Tat peptide (48-57).  Deletion of the four side chains of the γ-AApeptide attenuates translocation capability.  We also establish that the γ-AApeptide is even less toxic than the Tat peptide against mammalian cells.  In addn. to their low toxicity, γ-AApeptides are resistant to protease degrdn., which may prove to be advantageous over α-peptides for further development of mol. transporters for intracellular delivery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomoi3EtFaKb7Vg90H21EOLACvtfcHk0ljVN51NPKR3qA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1OisLo%253D&md5=cbd49ecd5e45df1197ebb383887930ce</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fmp300070w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300070w%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DR.%2BE.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DPadhee%26aufirst%3DS.%26aulast%3DBuzzeo%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DCellular%2520Translocation%2520of%2520a%2520%25CE%25B3-AApeptide%2520Mimetic%2520of%2520Tat%2520Peptide%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2012%26volume%3D9%26spage%3D1529%26epage%3D1534%26doi%3D10.1021%2Fmp300070w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parag, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N. A.</span></span> <span> </span><span class="NLM_article-title">Stabilization of lncRNA GAS5 by a Small Molecule and Its Implications in Diabetic Adipocytes</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">330.e6</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1016%2Fj.chembiol.2018.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=30661991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVartb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=319-330.e6&author=Y.+Shiauthor=S.+Paragauthor=R.+Patelauthor=A.+Luiauthor=M.+Murrauthor=J.+Caiauthor=N.+A.+Patel&title=Stabilization+of+lncRNA+GAS5+by+a+Small+Molecule+and+Its+Implications+in+Diabetic+Adipocytes&doi=10.1016%2Fj.chembiol.2018.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of lncRNA GAS5 by a Small Molecule and Its Implications in Diabetic Adipocytes</span></div><div class="casAuthors">Shi, Yan; Parag, Sajan; Patel, Rekha; Lui, Ashley; Murr, Michel; Cai, Jianfeng; Patel, Niketa A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">319-330.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Long noncoding RNA (lncRNA) are regulatory RNAs >200 nt.  We previously showed that lncRNA GAS5 decreases significantly in serum of type 2 diabetes mellitus (T2DM) patients.  Hence, we sought to decipher the mol. mechanisms underlying the role of GAS5 in T2DM in adipose tissue.  Using CHIP-RIP, we demonstrate that GAS5 binds to promoter of insulin receptor to regulate its expression, and its depletion inhibits glucose uptake and insulin signaling.  Toward stabilizing GAS5 levels in T2DM, we incorporated a strategy to limit the degrdn. of GAS5 by blocking the interaction of GAS5 and UPF1 with a small mol. identified using OBTC screening strategy.  NP-C86 binds to GAS5 with high affinity, and increases GAS5 levels and glucose uptake in diabetic patient adipocytes.  As a broader impact, NP-C86 may be used as a mol. probe to investigate the intricacies of GAS5 in relevant biol. systems as it offers specificity, efficient cellular uptake and is non-cytotoxic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk-KPkGlzXi7Vg90H21EOLACvtfcHk0lg1Z6gMb2pJjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVartb8%253D&md5=5f7d6288486103bfe129bdd1a7d9e05d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DParag%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DLui%26aufirst%3DA.%26aulast%3DMurr%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DN.%2BA.%26atitle%3DStabilization%2520of%2520lncRNA%2520GAS5%2520by%2520a%2520Small%2520Molecule%2520and%2520Its%2520Implications%2520in%2520Diabetic%2520Adipocytes%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D319%26epage%3D330.e6%26doi%3D10.1016%2Fj.chembiol.2018.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borcherds, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daughdrill, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">α-Helix-Mimicking Sulfono-γ-AApeptide Inhibitors for p53–MDM2/MDMX Protein–Protein Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">975</span>– <span class="NLM_lpage">986</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00993</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00993" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1elsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=975-986&author=P.+Sangauthor=Y.+Shiauthor=J.+Luauthor=L.+Chenauthor=L.+Yangauthor=W.+Borcherdsauthor=S.+Abdulkadirauthor=Q.+Liauthor=G.+Daughdrillauthor=J.+Chenauthor=J.+Cai&title=%CE%B1-Helix-Mimicking+Sulfono-%CE%B3-AApeptide+Inhibitors+for+p53%E2%80%93MDM2%2FMDMX+Protein%E2%80%93Protein+Interactions&doi=10.1021%2Facs.jmedchem.9b00993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">α-Helix-Mimicking Sulfono-γ-AApeptide Inhibitors for p53-MDM2/MDMX Protein-Protein Interactions</span></div><div class="casAuthors">Sang, Peng; Shi, Yan; Lu, Junhao; Chen, Lihong; Yang, Leixiang; Borcherds, Wade; Abdulkadir, Sami; Li, Qi; Daughdrill, Gary; Chen, Jiandong; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">975-986</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of peptidomimetic scaffolds is a promising strategy for the inhibition of protein-protein interactions (PPIs).  Herein, we demonstrate that sulfono-γ-AApeptides can be rationally designed to mimic the p53 α-helix and inhibit p53-MDM2 PPIs.  The best inhibitor, with Kd and IC50 values of 26 nM and 0.891 μM toward MDM2, resp., is among the most potent unnatural peptidomimetic inhibitors disrupting the p53-MDM2/MDMX interaction.  Using fluorescence polarization assays, CD, NMR spectroscopy, and computational simulations, we demonstrate that sulfono-γ-AApeptides adopt helical structures resembling p53 and competitively inhibit the p53-MDM2 interaction by binding to the hydrophobic cleft of MDM2.  Intriguingly, the stapled sulfono-γ-AApeptides showed promising cellular activity by enhancing p53 transcriptional activity and inducing expression of MDM2 and p21.  Moreover, sulfono-γ-AApeptides exhibited remarkable resistance to proteolysis, augmenting their biol. potential.  Our results suggest that sulfono-γ-AApeptides are a new class of unnatural helical foldamers that disrupt PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEqdT_RC6b_LVg90H21EOLACvtfcHk0li-XN8z96eglQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1elsb0%253D&md5=422b83d0699b80ce888a7efd65e09ca7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00993%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DBorcherds%26aufirst%3DW.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DDaughdrill%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B1-Helix-Mimicking%2520Sulfono-%25CE%25B3-AApeptide%2520Inhibitors%2520for%2520p53%25E2%2580%2593MDM2%252FMDMX%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D975%26epage%3D986%26doi%3D10.1021%2Facs.jmedchem.9b00993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tipton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">gamma-AApeptide-based small-molecule ligands that inhibit Abeta aggregation</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5206</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1039/C3CC46685J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1039%2FC3CC46685J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=24158240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCjsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=5206-5208&author=H.+Wuauthor=Y.+Liauthor=G.+Baiauthor=Y.+Niuauthor=Q.+Qiaoauthor=J.+D.+Tiptonauthor=C.+Caoauthor=J.+Cai&title=gamma-AApeptide-based+small-molecule+ligands+that+inhibit+Abeta+aggregation&doi=10.1039%2FC3CC46685J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptide-based small-molecule ligands that inhibit Aβ aggregation</span></div><div class="casAuthors">Wu, Haifan; Li, Yaqiong; Bai, Ge; Niu, Youhong; Qiao, Qiao; Tipton, Jeremiah D.; Cao, Chuanhai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">5206-5208</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report the design, synthesis, characterization and evaluation of a novel class of γ-AApeptide one-bead-one-compd. (OBOC) library, from which a small γ-AApeptide was identified to effectively prevent and disassemble Aβ aggregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-8_VqKY2vrVg90H21EOLACvtfcHk0ljvP8raYA3mcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCjsbw%253D&md5=f1750ac4d122bfa7ba92b23fa9250c63</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC3CC46685J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3CC46685J%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DTipton%26aufirst%3DJ.%2BD.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3Dgamma-AApeptide-based%2520small-molecule%2520ligands%2520that%2520inhibit%2520Abeta%2520aggregation%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D5206%26epage%3D5208%26doi%3D10.1039%2FC3CC46685J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Identification of novel inhibitors that disrupt STAT3-DNA interaction from a γ-AApeptide OBOC combinatorial library</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">8739</span>– <span class="NLM_lpage">8742</span>, <span class="refDoi"> DOI: 10.1039/C4CC03909B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1039%2FC4CC03909B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=24964402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVGnsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=8739-8742&author=P.+Tengauthor=X.+Zhangauthor=H.+Wuauthor=Q.+Qiaoauthor=S.+M.+Sebtiauthor=J.+Cai&title=Identification+of+novel+inhibitors+that+disrupt+STAT3-DNA+interaction+from+a+%CE%B3-AApeptide+OBOC+combinatorial+library&doi=10.1039%2FC4CC03909B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel inhibitors that disrupt STAT3-DNA interaction from a γ-AApeptide OBOC combinatorial library</span></div><div class="casAuthors">Teng, Peng; Zhang, Xiaolei; Wu, Haifan; Qiao, Qiao; Sebti, Said M.; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">63</span>),
    <span class="NLM_cas:pages">8739-8742</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">From a γ-AApeptide-based one-bead-one-compd. (OBOC) combinatorial library, the authors identified γ-AApeptides that can selectively inhibit STAT3-DNA interaction and suppress the expression levels of STAT3 target genes in intact cells.  The authors' results demonstrate that in addn. to the SH2 domain, the DNA binding domain of STAT3 is targetable for the development of a new generation of anti-cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVN0NpRV6tXrVg90H21EOLACvtfcHk0ljmePkj0fvK2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVGnsrjM&md5=41c994aa9a985107fc1402b58d38223a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FC4CC03909B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC03909B%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520novel%2520inhibitors%2520that%2520disrupt%2520STAT3-DNA%2520interaction%2520from%2520a%2520%25CE%25B3-AApeptide%2520OBOC%2520combinatorial%2520library%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D8739%26epage%3D8742%26doi%3D10.1039%2FC4CC03909B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Hydrogen-Bonding-Driven 3D Supramolecular Assembly of Peptidomimetic Zipper</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">5661</span>– <span class="NLM_lpage">5665</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b11997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b11997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=5661-5665&author=P.+Tengauthor=Z.+Niuauthor=F.+Sheauthor=M.+Zhouauthor=P.+Sangauthor=G.+M.+Grayauthor=G.+Vermaauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=S.+Maauthor=J.+Cai&title=Hydrogen-Bonding-Driven+3D+Supramolecular+Assembly+of+Peptidomimetic+Zipper&doi=10.1021%2Fjacs.7b11997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen-Bonding-Driven 3D Supramolecular Assembly of Peptidomimetic Zipper</span></div><div class="casAuthors">Teng, Peng; Niu, Zheng; She, Fengyu; Zhou, Mi; Sang, Peng; Gray, Geoffrey M.; Verma, Gaurav; Wojtas, Lukasz; van der Vaart, Arjan; Ma, Shengqian; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5661-5665</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen-bonding-driven three-dimensional (3D) assembly of a peptidomimetic zipper has been established for the first time by using an α/AApeptide zipper that assembles into a de novo lattice arrangement through two layers of hydrogen-bonded linker-directed interactions.  Via a covalently bridged 1D 413-helix, drastic enhancement in stability has been achieved in the formed 3D cryst. supramol. architecture as evidenced by gas-sorption studies.  As the first example of an unnatural peptidic zipper, the dimensional augmentation of the zipper differs from metal-coordinated strategies, and may have general implications for the prepn. of peptidic functional materials for a variety of future applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6NOPVSQVnN7Vg90H21EOLACvtfcHk0lh12c0XZRDZJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D&md5=75ec49a44f96107f9a2472698475157d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b11997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b11997%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DZ.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DVerma%26aufirst%3DG.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHydrogen-Bonding-Driven%25203D%2520Supramolecular%2520Assembly%2520of%2520Peptidomimetic%2520Zipper%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D5661%26epage%3D5665%26doi%3D10.1021%2Fjacs.7b11997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Tejada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjonaj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">De Novo Left-Handed Synthetic Peptidomimetic Foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9916</span>– <span class="NLM_lpage">9920</span>, <span class="refDoi"> DOI: 10.1002/anie.201805184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201805184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=9916-9920&author=F.+Sheauthor=P.+Tengauthor=A.+Peguero-Tejadaauthor=M.+Wangauthor=N.+Maauthor=T.+Odomauthor=M.+Zhouauthor=E.+Gjonajauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=De+Novo+Left-Handed+Synthetic+Peptidomimetic+Foldamers&doi=10.1002%2Fanie.201805184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">De novo left-handed synthetic peptidomimetic foldamers</span></div><div class="casAuthors">She, Fengyu; Teng, Peng; Peguero-Tejada, Alfredo; Wang, Minghui; Ma, Ning; Odom, Timothy; Zhou, Mi; Gjonaj, Erald; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9916-9920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of peptidomimetic helical foldamers with a wide repertoire of functions is of significant interest.  Herein, we report the X-ray crystal structures of a series of homogeneous L-sulfono-γ-AA foldamers and elucidate their folding conformation at the at. level.  Single-crystal X-ray crystallog. revealed that this class of oligomers fold into unprecedented dragon-boat-shaped and unexpected left-handed helixes, which are stabilized by the 14-hydrogen-bonding pattern present in all sequences.  These L-sulfono-γ-AA-peptides have a helical pitch of 5.1 Å and exactly four side chains per turn, and the side chains lie perfectly on top of each other along the helical axis.  2D NMR spectroscopy, computational simulations, and CD studies support the folding conformation in soln.  Our results provide a structural basis at the at. level for the design of novel biomimetics with a precise arrangement of functional groups in three dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2vbXzMss1rVg90H21EOLACvtfcHk0liiSUYHeZmDAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL&md5=c45cc7e77dfa2887f5fec63d8e3dbe7e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201805184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201805184%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DPeguero-Tejada%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGjonaj%26aufirst%3DE.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DDe%2520Novo%2520Left-Handed%2520Synthetic%2520Peptidomimetic%2520Foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D9916%26epage%3D9920%26doi%3D10.1002%2Fanie.201805184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7778</span>– <span class="NLM_lpage">7782</span>, <span class="refDoi"> DOI: 10.1002/anie.201903259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201903259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=7778-7782&author=P.+Tengauthor=G.+M.+Grayauthor=M.+Zhengauthor=S.+Singhauthor=X.+Liauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=Orthogonal+Halogen-Bonding-Driven+3D+Supramolecular+Assembly+of+Right-Handed+Synthetic+Helical+Peptides&doi=10.1002%2Fanie.201903259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span></div><div class="casAuthors">Teng, Peng; Gray, Geoffrey M.; Zheng, Mengmeng; Singh, Sylvia; Li, Xiaopeng; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7778-7782</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peptide-mediated self-assembly is a prevalent method for creating highly ordered supramol. architectures.  Herein, we report the first example of orthogonal C-X···X-C/C-X···π halogen bonding and hydrogen bonding driven cryst. architectures based on synthetic helical peptides bearing hybrids of L-sulfono-γ-AApeptides and natural amino acids.  The combination of halogen bonding, intra-/intermol. hydrogen bonding, and intermol. hydrophobic interactions enabled novel 3D supramol. assembly.  The orthogonal halogen bonding in the supramol. architecture exerts a novel mechanism for the self-assembly of synthetic peptide foldamers and gives new insights into mol. recognition, supramol. design, and rational design of biomimetic structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphs8_rPqx-zrVg90H21EOLACvtfcHk0lgJvUqNFZBYtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D&md5=c61d38ec3f3a4e3d79d716a1c88a7ec3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201903259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201903259%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOrthogonal%2520Halogen-Bonding-Driven%25203D%2520Supramolecular%2520Assembly%2520of%2520Right-Handed%2520Synthetic%2520Helical%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D7778%26epage%3D7782%26doi%3D10.1002%2Fanie.201903259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amso, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen-Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">The Activity of Sulfono-γ- AApeptide Helical Foldamers That Mimic GLP-1</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">eaaz4988</span> <span class="refDoi"> DOI: 10.1126/sciadv.aaz4988</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1126%2Fsciadv.aaz4988" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=32440547" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&author=P.+Sangauthor=Z.+Zhouauthor=Y.+Shiauthor=C.+Leeauthor=Z.+Amsoauthor=D.+Huangauthor=T.+Odomauthor=V.+Nguyen-Tranauthor=W.+Shenauthor=J.+Cai&title=The+Activity+of+Sulfono-%CE%B3-+AApeptide+Helical+Foldamers+That+Mimic+GLP-1&doi=10.1126%2Fsciadv.aaz4988"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aaz4988&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aaz4988%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DAmso%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DNguyen-Tran%26aufirst%3DV.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DThe%2520Activity%2520of%2520Sulfono-%25CE%25B3-%2520AApeptide%2520Helical%2520Foldamers%2520That%2520Mimic%2520GLP-1%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26doi%3D10.1126%2Fsciadv.aaz4988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerrato, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Right-Handed Helical Foldamers Consisting of De Novo d-AApeptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">7363</span>– <span class="NLM_lpage">7369</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b03007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b03007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=7363-7369&author=P.+Tengauthor=N.+Maauthor=D.+C.+Cerratoauthor=F.+Sheauthor=T.+Odomauthor=X.+Wangauthor=L.+J.+Mingauthor=A.+van+der%0AVaartauthor=L.+Wojtasauthor=H.+Xuauthor=J.+Cai&title=Right-Handed+Helical+Foldamers+Consisting+of+De+Novo+d-AApeptides&doi=10.1021%2Fjacs.7b03007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Right-handed helical foldamers consisting of de novo D-AA-peptides</span></div><div class="casAuthors">Teng, Peng; Ma, Ning; Cerrato, Darrell Cole; She, Fengyu; Odom, Timothy; Wang, Xiang; Ming, Li-June; van der Vaart, Arjan; Wojtas, Lukasz; Xu, Hai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7363-7369</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New types of foldamer scaffolds are formidably challenging to design and synthesize, yet highly desirable as structural mimics of peptides/proteins with a wide repertoire of functions.  In particular, the development of peptidomimetic helical foldamers holds promise for new biomaterials, catalysts, and drug mols.  Unnatural L-sulfono-γ-AApeptides were recently developed and shown to have potential applications in both biomedical and material sciences.  However, D-sulfono-γ-AApeptides, the enantiomers of L-sulfono-γ-AApeptides, have never been studied due to the lack of high-resoln. three-dimensional structures to guide structure-based design.  Herein, we report the first synthesis and X-ray crystal structures of a series of 2:1 L-amino acid/D-sulfono-γ-AApeptide hybrid foldamers, and elucidate their folded conformation at the at. level.  Single-crystal X-ray crystallog. indicates that this class of oligomers folds into well-defined right-handed helixes with unique helical parameters.  The helical structures were consistent with data obtained from soln. 2D NMR, CD studies, and mol. dynamics simulations.  Our findings are expected to inspire the structure-based design of this type of unique folding biopolymers for biomaterials and biomedical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaGg_OJWsqYbVg90H21EOLACvtfcHk0lgEzLLySt6zvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D&md5=fef3f80500953f027687001f3e67f14b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b03007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b03007%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DCerrato%26aufirst%3DD.%2BC.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMing%26aufirst%3DL.%2BJ.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DRight-Handed%2520Helical%2520Foldamers%2520Consisting%2520of%2520De%2520Novo%2520d-AApeptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D7363%26epage%3D7369%26doi%3D10.1021%2Fjacs.7b03007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, D. P.</span></span> <span> </span><span class="NLM_article-title">Awakening Guardian Angels: Drugging the p53 Pathway</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">873</span>, <span class="refDoi"> DOI: 10.1038/nrc2763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1038%2Fnrc2763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=19935675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=862-873&author=C.+J.+Brownauthor=S.+Lainauthor=C.+S.+Vermaauthor=A.+R.+Fershtauthor=D.+P.+Lane&title=Awakening+Guardian+Angels%3A+Drugging+the+p53+Pathway&doi=10.1038%2Fnrc2763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Awakening guardian angels: Drugging the p53 pathway</span></div><div class="casAuthors">Brown, Christopher J.; Lain, Sonia; Verma, Chandra S.; Fersht, Alan R.; Lane, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">862-873</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Currently, around 11 million people are living with a tumor that contains an inactivating mutation of TP53 (the human gene that encodes p53) and another 11 million have tumors in which the p53 pathway is partially abrogated through the inactivation of other signaling or effector components.  The p53 pathway is therefore a prime target for new cancer drug development, and several original approaches to drug discovery that could have wide applications to drug development are being used.  In one approach, mols. that activate p53 by blocking protein-protein interactions with MDM2 are in early clin. development.  Remarkable progress has also been made in the development of p53-binding mols. that can rescue the function of certain p53 mutants.  Finally, cell-based assays are being used to discover compds. that exploit the p53 pathway by either seeking targets and compds. that show synthetic lethality with TP53 mutations or by looking for non-genotoxic activators of the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZW-elIe4w6bVg90H21EOLACvtfcHk0lhhkIXpOjQSRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM&md5=072f48d2ed466b2deb9d2b999cc65a53</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrc2763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2763%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DC.%2BJ.%26aulast%3DLain%26aufirst%3DS.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26aulast%3DLane%26aufirst%3DD.%2BP.%26atitle%3DAwakening%2520Guardian%2520Angels%253A%2520Drugging%2520the%2520p53%2520Pathway%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D862%26epage%3D873%26doi%3D10.1038%2Fnrc2763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">Apamin as a Template for Structure-Based Rational Design of Potent Peptide Activators of p53</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">8712</span>– <span class="NLM_lpage">8715</span>, <span class="refDoi"> DOI: 10.1002/anie.200904550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.200904550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGju77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8712-8715&author=C.+Liauthor=M.+Pazgierauthor=M.+Liuauthor=W.+Y.+Luauthor=W.+Lu&title=Apamin+as+a+Template+for+Structure-Based+Rational+Design+of+Potent+Peptide+Activators+of+p53&doi=10.1002%2Fanie.200904550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Apamin as a Template for Structure-Based Rational Design of Potent Peptide Activators of p53</span></div><div class="casAuthors">Li, Chong; Pazgier, Marzena; Liu, Min; Lu, Wei-Yue; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">8712-8715, S8712/1-S8712/10</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The activity and stability of the tumor suppressor protein p53 is neg. regulated by the oncogenic proteins MDM2 and MDMX-the cellular process is initiated by MDM2/MDMX binding to the N-terminal transactivation domain of p53.  MDM2, an E3 ubiquitin ligase, primarily controls p53 stability by targeting the tumor suppressor protein for ubiquitin- mediated constitutive degrdn., whereas MDMX mainly functions as an effective transcriptional antagonist of p53 that blocks its ability to regulate responsive genes expression.  Antagonists that block the p-53-binding pocket of MDM2/ MDMX kill tumor cells both in vitro and in vivo by reactivating the p53 pathway, resulting in cell cycle arrest, senescence, or apoptosis.  MDM2 and MDMX act synergistically in tumor cells and have become highly attractive mol. targets for anticancer drug development.  Using phage display, we identified a 12-mer peptide, termed PMI (p-53-MDM2/MDMX inhibitor), with affinity for both MDM2 and MDMX in the low nM range.  Herein we report that grafting four residues of PMI crit. for MDM2/MDMX binding to apamin, a highly specific blocker of Ca2+-activated K+ channels of small conductance, converted the 18-amino acid residue bee-venom neurotoxin into several potent peptide inhibitors of the p-53-MDM2/MDMX interactions with different specificities.  The rational design of these apamin-derived p53 activators, termed stingins, was structurally validated by X-ray crystallog.  In conclusion, by grafting four hydrophobic residues, crit. for MDM2/MDMX binding, to the C-terminal a-helix of apamin, we successfully converted the bee-venom neurotoxin into a series of potent inhibitors of p53 interactions with MDM2 and MDMX.  Structural studies by X-ray crystallog. validated the mode of inhibition, that is apamin-derived stingins directly compete with p53 for MDM2/ MDMX binding.  Stingins represent a novel class of p53 activators and are superior in many aspects to the existing miniprotein antagonists of MDM2/MDMX.  Coupled with a therapeutically viable delivery modality, stingins may have the potential to be used as antitumor agents for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbV6YPq1xMkrVg90H21EOLACvtfcHk0lgDOuMGxvH3kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGju77P&md5=cfc5cff5f181b0178b6b1477e25eebb4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.200904550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200904550%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%2BY.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DApamin%2520as%2520a%2520Template%2520for%2520Structure-Based%2520Rational%2520Design%2520of%2520Potent%2520Peptide%2520Activators%2520of%2520p53%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D8712%26epage%3D8715%26doi%3D10.1002%2Fanie.200904550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">D-peptide Inhibitors of the p53–MDM2 Interaction for Ttargeted Molecular Therapy of Malignant Neoplasms</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">14321</span>– <span class="NLM_lpage">14326</span>, <span class="refDoi"> DOI: 10.1073/pnas.1008930107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.1008930107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=20660730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCkt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=14321-14326&author=M.+Liuauthor=C.+Liauthor=M.+Pazgierauthor=C.+Liauthor=Y.+Maoauthor=Y.+Lvauthor=B.+Guauthor=G.+Weiauthor=W.+Yuanauthor=C.+Zhanauthor=W.-Y.+Luauthor=W.+Lu&title=D-peptide+Inhibitors+of+the+p53%E2%80%93MDM2+Interaction+for+Ttargeted+Molecular+Therapy+of+Malignant+Neoplasms&doi=10.1073%2Fpnas.1008930107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">D-peptide inhibitors of the p53 - MDM2 interaction for targeted molecular therapy of malignant neoplasms</span></div><div class="casAuthors">Liu, Min; Li, Chong; Pazgier, Marzena; Li, Changqing; Mao, Yubin; Lv, Yifan; Gu, Bing; Wei, Gang; Yuan, Weirong; Zhan, Changyou; Lu, Wei-Yue; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">14321-14326, S14321/1-S14321/16</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The oncoproteins MDM2 and MDMX neg. regulate the activity and stability of the tumor suppressor protein p53, conferring tumor development and survival.  Antagonists targeting the p53-binding domains of MDM2 and MDMX kill tumor cells both in vitro and in vivo by reactivating the p53 pathway, promising a class of antitumor agents for cancer therapy.  Aided by native chem. ligation and mirror image phage display, the authors recently identified a D-peptide inhibitor of the p53-MDM2 interaction termed DPMI-α (TNWYANLEKLLR) that competes with p53 for MDM2 binding at an affinity of 219 nM.  Increased selection stringency resulted in a distinct D-peptide inhibitor termed DPMI-γ (DWWPLAFEALLR) that binds MDM2 at an affinity of 53 nM.  Structural studies coupled with mutational anal. verified the mode of action of these D-peptides as MDM2-dependent p53 activators.  Despite being resistant to proteolysis, both DPMI-α and DPMI-γ failed to actively traverse the cell membrane and, when conjugated to a cationic cell-penetrating peptide, were indiscriminately cytotoxic independently of p53 status.  When encapsulated in liposomes decorated with an integrin-targeting cyclic-RGD peptide, however DPMI-α exerted potent p53-dependent growth inhibitory activity against human glioblastoma in cell cultures and nude mouse xenograft models.  The authors' findings validate D-peptide antagonists of MDM2 as a class of p53 activators for targeted mol. therapy of malignant neoplasms harboring WT p53 and elevated levels of MDM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4a4riieVDmbVg90H21EOLACvtfcHk0ljB7Rf6MzS7XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCkt73O&md5=fe67e00a00589a915106141be5bd8a4a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1008930107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1008930107%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DLv%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DZhan%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DW.-Y.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DD-peptide%2520Inhibitors%2520of%2520the%2520p53%25E2%2580%2593MDM2%2520Interaction%2520for%2520Ttargeted%2520Molecular%2520Therapy%2520of%2520Malignant%2520Neoplasms%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D14321%26epage%3D14326%26doi%3D10.1073%2Fpnas.1008930107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">A Left-Handed Solution to Peptide Inhibition of the p53–MDM2 Interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">3649</span>– <span class="NLM_lpage">3652</span>, <span class="refDoi"> DOI: 10.1002/anie.201000329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.201000329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFWisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=3649-3652&author=M.+Liuauthor=M.+Pazgierauthor=C.+Liauthor=W.+Yuanauthor=C.+Liauthor=W.+Lu&title=A+Left-Handed+Solution+to+Peptide+Inhibition+of+the+p53%E2%80%93MDM2+Interaction&doi=10.1002%2Fanie.201000329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A Left-Handed Solution to Peptide Inhibition of the p53-MDM2 Interaction</span></div><div class="casAuthors">Liu, Min; Pazgier, Marzena; Li, Changqing; Yuan, Weirong; Li, Chong; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3649-3652, S3649/1-S3649/10</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein, we report the design, aided by mirror-image phage display (MIPD) and native chem. ligation (NCL) of a potent D-peptide ligand, termed DpMI-3 (TAWYANFEKLLR), of MDM2-the oncogenic E3 ubiquitin ligase that neg. regulates the activity and stability of the tumor suppressor protein p53.  X-ray crystallog. studies elucidated the structural basis for high-affinity D-peptide inhibition of the p-53-MDM2 interaction, and validated the mode of action of DpMI peptides as a novel class of p53 activators.  D-Peptide inhibitors are superior to many existing drug candidates in aspects such as potency, specificity, and particularly in vivo stability.  Coupled with a therapeutically viable delivery modality, DPMI-a or DPMI-P and its derivs. may have the potential to be developed into antitumor agents for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrdtmiMF1rbVg90H21EOLACvtfcHk0lgBe82geUdvPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFWisLg%253D&md5=56738900a6934ef609bab1a7369eeaf2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fanie.201000329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201000329%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DA%2520Left-Handed%2520Solution%2520to%2520Peptide%2520Inhibition%2520of%2520the%2520p53%25E2%2580%2593MDM2%2520Interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26spage%3D3649%26epage%3D3652%26doi%3D10.1002%2Fanie.201000329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">Rapid Identification of Dual p53-MDM2/MDMX Interaction Inhibitors through Virtual Screening and Hit-based Substructure Search</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">9989</span>– <span class="NLM_lpage">9997</span>, <span class="refDoi"> DOI: 10.1039/C7RA00473G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1039%2FC7RA00473G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVOjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=9989-9997&author=S.+Chenauthor=X.+Liauthor=W.+Yuanauthor=Y.+Zouauthor=Z.+Guoauthor=Y.+Chaiauthor=W.+Lu&title=Rapid+Identification+of+Dual+p53-MDM2%2FMDMX+Interaction+Inhibitors+through+Virtual+Screening+and+Hit-based+Substructure+Search&doi=10.1039%2FC7RA00473G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid identification of dual p53-MDM2/MDMX interaction inhibitors through virtual screening and hit-based substructure search</span></div><div class="casAuthors">Chen, Si; Li, Xiang; Yuan, Weirong; Zou, Yan; Guo, Zhongwu; Chai, Yifeng; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">9989-9997</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Multi-target agents have garnered great interest over the past decade for their favorable therapeutic efficacy and drug resistance profiles.  Recently, dual inhibition of the p53 tumor suppressor interaction with its two neg. regulators MDM2 and MDMX has become an attractive anticancer approach as it can induce sustained MDM2/MDMX antagonism and robust p53 activation.  However, small mol. inhibitors with dual specificity against MDM2 and MDMX are difficult to design and are still scarce.  To identify novel scaffolds for dual inhibition of the p53-MDM2/MDMX interactions, we developed two five-point pharmacophore models for filtering the 2012 National Cancer Institute database, from which mol. docking was conducted to identify dual inhibitors.  We found 38 virtual hits and subjected them to a fluorescence polarization-based competitive binding assay, resulting in 10 active compds. of different scaffolds.  To further expand the chem. diversity of the initial hits, we performed a hit-based substructure search and identified NSC148171 from pharmacophore 1 as the most potent dual-specificity inhibitor with Ki values for MDM2 and MDMX at 0.62 and 4.6 μM.  All hits were subjected to inhibition assay of cancer cellular vitality and showed anti-proliferative activity roughly correlated with their Ki values.  This work not only yields several novel scaffolds for further structural and functional optimization of dual-specificity inhibitors of the p53-MDM2/MDMX interactions, but also showcases the power of our computational methods for small mol. anticancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZn-dghlQCLrVg90H21EOLACvtfcHk0li87FKSX6-XBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVOjs7c%253D&md5=06c3f9e21fc11949a40d94dfdf9f74f2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2FC7RA00473G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7RA00473G%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DChai%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DRapid%2520Identification%2520of%2520Dual%2520p53-MDM2%252FMDMX%2520Interaction%2520Inhibitors%2520through%2520Virtual%2520Screening%2520and%2520Hit-based%2520Substructure%2520Search%26jtitle%3DRSC%2520Adv.%26date%3D2017%26volume%3D7%26spage%3D9989%26epage%3D9997%26doi%3D10.1039%2FC7RA00473G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monbo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarasov, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for High-affinity Peptide Inhibition of p53 Interactions with MDM2 and MDMX</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">4665</span>– <span class="NLM_lpage">4670</span>, <span class="refDoi"> DOI: 10.1073/pnas.0900947106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.0900947106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=19255450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktFOgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=4665-4670&author=M.+Pazgierauthor=M.+Liuauthor=G.+Zouauthor=W.+Yuanauthor=C.+Liauthor=C.+Liauthor=J.+Liauthor=J.+Monboauthor=D.+Zellaauthor=S.+G.+Tarasovauthor=W.+Lu&title=Structural+Basis+for+High-affinity+Peptide+Inhibition+of+p53+Interactions+with+MDM2+and+MDMX&doi=10.1073%2Fpnas.0900947106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for high-affinity peptide inhibition of p53 interactions with MDM2 and MDMX</span></div><div class="casAuthors">Pazgier, Marzena; Liu, Min; Zou, Guozhang; Yuan, Weirong; Li, Changqing; Li, Chong; Li, Jing; Monbo, Juahdi; Zella, Davide; Tarasov, Sergey G.; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4665-4670</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The oncoproteins MDM2 and MDMX neg. regulate the activity and stability of the tumor suppressor protein p53-a cellular process initiated by MDM2 and/or MDMX binding to the N-terminal transactivation domain of p53.  MDM2 and MDMX in many tumors confer p53 inactivation and tumor survival, and are important mol. targets for anticancer therapy.  We screened a duodecimal peptide phage library against site-specifically biotinylated p53-binding domains of human MDM2 and MDMX chem. synthesized via native chem. ligation, and identified several peptide inhibitors of the p53-MDM2/MDMX interactions.  The most potent inhibitor (TSFAEYWNLLSP), termed PMI, bound to MDM2 and MDMX at low nanomolar affinities - approx. 2 orders of magnitude stronger than the wild-type p53 peptide of the same length (ETFSDLWKLLPE).  We solved the crystal structures of synthetic MDM2 and MDMX, both in complex with PMI, at 1.6 Å resoln.  Comparative structural anal. identified an extensive, tightened intramol. H-bonding network in bound PMI that contributed to its conformational stability, thus enhanced binding to the 2 oncogenic proteins.  Importantly, the C-terminal residue Pro of PMI induced formation of a hydrophobic cleft in MDMX previously unseen in the structures of p53-bound MDM2 or MDMX.  Our findings deciphered the structural basis for high-affinity peptide inhibition of p53 interactions with MDM2 and MDMX, shedding new light on structure-based rational design of different classes of p53 activators for potential therapeutic use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9OZR5KzhWxbVg90H21EOLACvtfcHk0licmyg0FschGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktFOgtro%253D&md5=d343128b1597cadfbb36200ac8d6a8a9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0900947106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0900947106%26sid%3Dliteratum%253Aachs%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZou%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMonbo%26aufirst%3DJ.%26aulast%3DZella%26aufirst%3DD.%26aulast%3DTarasov%26aufirst%3DS.%2BG.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DStructural%2520Basis%2520for%2520High-affinity%2520Peptide%2520Inhibition%2520of%2520p53%2520Interactions%2520with%2520MDM2%2520and%2520MDMX%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D4665%26epage%3D4670%26doi%3D10.1073%2Fpnas.0900947106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borcherds, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daughdrill, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Autoinhibition of MDMX by Intramolecular p53 Mimicry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">4624</span>– <span class="NLM_lpage">4629</span>, <span class="refDoi"> DOI: 10.1073/pnas.1420833112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1073%2Fpnas.1420833112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=25825738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=4624-4629&author=L.+Chenauthor=W.+Borcherdsauthor=S.+Wuauthor=A.+Beckerauthor=E.+Schonbrunnauthor=G.+W.+Daughdrillauthor=J.+Chen&title=Autoinhibition+of+MDMX+by+Intramolecular+p53+Mimicry&doi=10.1073%2Fpnas.1420833112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Autoinhibition of MDMX by intramolecular p53 mimicry</span></div><div class="casAuthors">Chen, Lihong; Borcherds, Wade; Wu, Shaofang; Becker, Andreas; Schonbrunn, Ernst; Daughdrill, Gary W.; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4624-4629</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The p53 inhibitor MDMX is controlled by multiple stress signaling pathways.  Using a proteolytic fragment release (PFR) assay, we detected an intramol. interaction in MDMX that mechanistically mimics the interaction with p53, resulting in autoinhibition of MDMX.  This mimicry is mediated by a hydrophobic peptide located in a long disordered central segment of MDMX that has sequence similarity to the p53 transactivation domain.  NMR spectroscopy was used to show this hydrophobic peptide interacts with the N-terminal domain of MDMX in a structurally analogous manner to p53.  Mutation of two crit. tryptophan residues in the hydrophobic peptide disrupted the intramol. interaction and increased p53 binding, providing further evidence for mechanistic mimicry.  The PFR assay also revealed a second intramol. interaction between the RING domain and central region that regulates MDMX nuclear import.  These results establish the importance of intramol. interactions in MDMX regulation, and validate a new assay for the study of intramol. interactions in multidomain proteins with intrinsically disordered regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD5EJJ943LLVg90H21EOLACvtfcHk0liTsIJGiqBqXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsbs%253D&md5=7c465f4a11d5adc4e8a2e33f17ae1da0</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1420833112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1420833112%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DBorcherds%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DDaughdrill%26aufirst%3DG.%2BW.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DAutoinhibition%2520of%2520MDMX%2520by%2520Intramolecular%2520p53%2520Mimicry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D4624%26epage%3D4629%26doi%3D10.1073%2Fpnas.1420833112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Design and High-Resolution Structure of a β3-Peptide Bundle Catalyst</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">6810</span>– <span class="NLM_lpage">6813</span>, <span class="refDoi"> DOI: 10.1021/ja5013849</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja5013849" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1Ciu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=6810-6813&author=P.+S.+P.+Wangauthor=J.+B.+Nguyenauthor=A.+Schepartz&title=Design+and+High-Resolution+Structure+of+a+%CE%B23-Peptide+Bundle+Catalyst&doi=10.1021%2Fja5013849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Design and High-Resolution Structure of a β3-Peptide Bundle Catalyst</span></div><div class="casAuthors">Wang, Pam S. P.; Nguyen, Jennifer B.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6810-6813</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the widespread exploration of α-peptides as catalysts, there are few examples of β-peptides that alter a chem. transformation.  Previous work demonstrated that a special class of β3-peptides spontaneously self-assembles in water into discrete protein-like bundles possessing unique quaternary structures and exceptional thermodn. stability.  Here the authors describe β3-peptide bundles capable of both substrate binding and chem. catalysis-ester hydrolysis.  A combination of kinetic and high-resoln. structural anal. suggests an active site triad composed of residues from at least two strands of the octameric bundle structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJGELSSHUnErVg90H21EOLACvtfcHk0lhdV8J38_h0gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1Ciu74%253D&md5=7489eb9f9626d5e958289732a577455b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja5013849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja5013849%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DP.%2BS.%2BP.%26aulast%3DNguyen%26aufirst%3DJ.%2BB.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DDesign%2520and%2520High-Resolution%2520Structure%2520of%2520a%2520%25CE%25B23-Peptide%2520Bundle%2520Catalyst%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D6810%26epage%3D6813%26doi%3D10.1021%2Fja5013849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cussol, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buratto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belorusova, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuville, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fribourg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolain, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goudreau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rochel, N.</span></span> <span> </span><span class="NLM_article-title">Structural basis for alpha-helix mimicry and inhibition of protein-protein interactions with oligourea foldamers</span>. <i>Angew Chem Int Ed Engl</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1002/anie.202008992</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fanie.202008992" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=32935897" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=G.+Guichardauthor=L.+Cussolauthor=L.+Mauranauthor=J.+Burattoauthor=A.+Y.+Belorusovaauthor=M.+Neuvilleauthor=J.+Oszauthor=S.+Fribourgauthor=J.+Fremauxauthor=C.+Dolainauthor=S.+Goudreauauthor=N.+Rochel&title=Structural+basis+for+alpha-helix+mimicry+and+inhibition+of+protein-protein+interactions+with+oligourea+foldamers&doi=10.1002%2Fanie.202008992"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fanie.202008992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202008992%26sid%3Dliteratum%253Aachs%26aulast%3DGuichard%26aufirst%3DG.%26aulast%3DCussol%26aufirst%3DL.%26aulast%3DMauran%26aufirst%3DL.%26aulast%3DBuratto%26aufirst%3DJ.%26aulast%3DBelorusova%26aufirst%3DA.%2BY.%26aulast%3DNeuville%26aufirst%3DM.%26aulast%3DOsz%26aufirst%3DJ.%26aulast%3DFribourg%26aufirst%3DS.%26aulast%3DFremaux%26aufirst%3DJ.%26aulast%3DDolain%26aufirst%3DC.%26aulast%3DGoudreau%26aufirst%3DS.%26aulast%3DRochel%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520for%2520alpha-helix%2520mimicry%2520and%2520inhibition%2520of%2520protein-protein%2520interactions%2520with%2520oligourea%2520foldamers%26jtitle%3DAngew%2520Chem%2520Int%2520Ed%2520Engl%26date%3D2020%26doi%3D10.1002%2Fanie.202008992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Targeting Protein–protein Interactions: Lessons from p53/MDM2</span>. <i>Biopol.-Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">686</span>, <span class="refDoi"> DOI: 10.1002/bip.20741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1002%2Fbip.20741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=17427181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKms7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2007&pages=657-686&author=J.+K.+Murrayauthor=S.+H.+Gellman&title=Targeting+Protein%E2%80%93protein+Interactions%3A+Lessons+from+p53%2FMDM2&doi=10.1002%2Fbip.20741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions: lessons from p53/MDM2</span></div><div class="casAuthors">Murray, Justin K.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">657-686</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The tremendous challenge of inhibiting therapeutically important protein-protein interactions has created the opportunity to extend traditional medicinal chem. to a new class of targets and to explore nontraditional strategies.  Here we review a widely studied system, the interaction between tumor suppressor p53 and its natural antagonist MDM2, for which both traditional and nontraditional approaches have been reported.  This system has been a testing ground for novel proteomimetic scaffold-based strategies, i.e., for attempts to mimic the recognition surface displayed by a folded protein with unnatural oligomers.  Retroinverso peptides, peptoids, terphenyls, β-hairpins, p-oligobenzamides, β-peptides, and miniproteins have all been explored as inhibitors of the p53/MDM2 interaction, and we focus on these oligomer-based efforts.  Traditional approaches have been successful as well, and we briefly review small mol. inhibitors along with other strategies for reactivation of the p53 pathway, for comparison with oligomer- based approaches.  We close with comments on an emerging dichotomy among protein-protein interaction targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy7SUsGRc4kbVg90H21EOLACvtfcHk0lj27pqamfZ1iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKms7nP&md5=8a70d8e515d8cae1284b9c3d1484eee3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fbip.20741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20741%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DTargeting%2520Protein%25E2%2580%2593protein%2520Interactions%253A%2520Lessons%2520from%2520p53%252FMDM2%26jtitle%3DBiopol.-Pept.%2520Sci.%26date%3D2007%26volume%3D88%26spage%3D657%26epage%3D686%26doi%3D10.1002%2Fbip.20741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lao, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarracino, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heindel, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Rational Design of Topographical Helix Mimics as Potent Inhibitors of Protein-protein Interactions</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">7877</span>– <span class="NLM_lpage">7888</span>, <span class="refDoi"> DOI: 10.1021/ja502310r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja502310r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1WltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=7877-7888&author=B.+B.+Laoauthor=K.+Drewauthor=D.+A.+Guarracinoauthor=T.+F.+Brewerauthor=D.+W.+Heindelauthor=R.+Bonneauauthor=P.+S.+Arora&title=Rational+Design+of+Topographical+Helix+Mimics+as+Potent+Inhibitors+of+Protein-protein+Interactions&doi=10.1021%2Fja502310r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Topographical Helix Mimics as Potent Inhibitors of Protein-Protein Interactions</span></div><div class="casAuthors">Lao, Brooke Bullock; Drew, Kevin; Guarracino, Danielle A.; Brewer, Thomas F.; Heindel, Daniel W.; Bonneau, Richard; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7877-7888</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interactions encompass large surface areas, but often a handful of key residues dominate the binding energy landscape.  Rationally designed small mol. scaffolds that reproduce the relative positioning and disposition of important binding residues, termed "hotspot residues", have been shown to successfully inhibit specific protein complexes.  Although this strategy has led to development of novel synthetic inhibitors of protein complexes, often direct mimicry of natural amino acid residues does not lead to potent inhibitors.  Exptl. screening of focused compd. libraries is used to further optimize inhibitors but the no. of possible designs that can be efficiently synthesized and exptl. tested in academic settings is limited.  We have applied the principles of computational protein design to optimization of nonpeptidic helix mimics as ligands for protein complexes.  We describe the development of computational tools to design helix mimetics from canonical and noncanonical residue libraries and their application to two therapeutically important protein-protein interactions: p53-MDM2 and p300-HIF1α.  The overall study provides a streamlined approach for discovering potent peptidomimetic inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNMadvCjMqobVg90H21EOLACvtfcHk0lj27pqamfZ1iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1WltLo%253D&md5=e19fc2448fcf0e6b71bdc208e1753209</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fja502310r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja502310r%26sid%3Dliteratum%253Aachs%26aulast%3DLao%26aufirst%3DB.%2BB.%26aulast%3DDrew%26aufirst%3DK.%26aulast%3DGuarracino%26aufirst%3DD.%2BA.%26aulast%3DBrewer%26aufirst%3DT.%2BF.%26aulast%3DHeindel%26aufirst%3DD.%2BW.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DRational%2520Design%2520of%2520Topographical%2520Helix%2520Mimics%2520as%2520Potent%2520Inhibitors%2520of%2520Protein-protein%2520Interactions%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D7877%26epage%3D7888%26doi%3D10.1021%2Fja502310r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kussie, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorina, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marechal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenbaas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavletich, N. P.</span></span> <span> </span><span class="NLM_article-title">Structure of the MDM2 Oncoprotein Bound to the p53 Tumor Suppressor Transactivation Domain</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">948</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1126/science.274.5289.948</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=10.1126%2Fscience.274.5289.948" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=8875929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADyaK28XmslKnt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=948-953&author=P.+H.+Kussieauthor=S.+Gorinaauthor=V.+Marechalauthor=B.+Elenbaasauthor=J.+Moreauauthor=A.+J.+Levineauthor=N.+P.+Pavletich&title=Structure+of+the+MDM2+Oncoprotein+Bound+to+the+p53+Tumor+Suppressor+Transactivation+Domain&doi=10.1126%2Fscience.274.5289.948"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the MDM2 oncoprotein bound to the p53 tumor suppressor transactivation domain</span></div><div class="casAuthors">Kussie, Paul H.; Gorina, Svetlana; Marechal, Vincent; Elenbaas, Brian; Moreau, Jacque; Levine, Arnold J.; Pavletich, Nikola P.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5289</span>),
    <span class="NLM_cas:pages">948-953</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The MDM2 oncoprotein is a cellular inhibitor of the p53 tumor suppressor in that it can bind the transactivation domain of p53 and downregulate its ability to activate transcription.  In certain cancers, MDM2 amplification is a common event and contributes to the inactivation of p53.  The crystal structure of the 109-residue amino-terminal domain of MDM2 bound to a 15-residue transactivation domain peptide of p53 revealed that MDM2 has a deep hydrophobic cleft on which the p53 peptide binds as an amphipathic α helix.  The interface relies on the steric complementarity between the MDM2 cleft and the hydrophobic face of the p53 α helix and, in particular, on a triad of p53 amino acids - Phe19, Trp23, and Leu26 - which insert deep into the MDM2 cleft.  These same p53 residues are also involved in transactivation, supporting the hypothesis that MDM2 inactivates p53 by concealing its transactivation domain.  The structure also suggests that the amphipathic α helix may be a common structural motif in the binding of a diverse family of transactivation factors to the TATA-binding protein-assocd. factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotkRIRDnQnX7Vg90H21EOLACvtfcHk0lilF4GqTZzDBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmslKnt7k%253D&md5=3702422b0a761617cff1498f47f30eeb</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5289.948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5289.948%26sid%3Dliteratum%253Aachs%26aulast%3DKussie%26aufirst%3DP.%2BH.%26aulast%3DGorina%26aufirst%3DS.%26aulast%3DMarechal%26aufirst%3DV.%26aulast%3DElenbaas%26aufirst%3DB.%26aulast%3DMoreau%26aufirst%3DJ.%26aulast%3DLevine%26aufirst%3DA.%2BJ.%26aulast%3DPavletich%26aufirst%3DN.%2BP.%26atitle%3DStructure%2520of%2520the%2520MDM2%2520Oncoprotein%2520Bound%2520to%2520the%2520p53%2520Tumor%2520Suppressor%2520Transactivation%2520Domain%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D948%26epage%3D953%26doi%3D10.1126%2Fscience.274.5289.948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murage, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J.-M.</span></span> <span> </span><span class="NLM_article-title">Development of Potent Glucagon-like Peptide-1 Agonists with High Enzyme Stability via Introduction of Multiple Lactam Bridges</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6412</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/jm100602m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100602m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVarsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6412-6420&author=E.+N.+Murageauthor=G.+Gaoauthor=A.+Biselloauthor=J.-M.+Ahn&title=Development+of+Potent+Glucagon-like+Peptide-1+Agonists+with+High+Enzyme+Stability+via+Introduction+of+Multiple+Lactam+Bridges&doi=10.1021%2Fjm100602m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent Glucagon-like Peptide-1 Agonists with High Enzyme Stability via Introduction of Multiple Lactam Bridges</span></div><div class="casAuthors">Murage, Eunice N.; Gao, Guangzu; Bisello, Alessandro; Ahn, Jung-Mo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6412-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glucagon-like peptide-1 (GLP-1) has the ability to lower the blood glucose level, and its regulatory functions make it an attractive therapeutic agent for the treatment of type 2 diabetes.  However, its rapid degrdn. by enzymes like dipeptidyl peptidase-IV (DPP-IV) and neutral endopeptidase (NEP) 24.11 severely compromises its effective clin. use.  Whereas specific DPP-IV inhibitors have been developed, NEP 24.11 targets multiple sites in the GLP-1 sequence, which makes it difficult to block.  To address this drawback, we have designed and synthesized conformationally constrained GLP-1 analogs by introducing multiple lactam bridges that stabilized both α-helixes in the N- and C-terminal regions simultaneously.  In addn. to improving the receptor activation capability (up to 5-fold) by fixing the α-helical conformations required for optimal receptor interaction, the introduced lactam bridges provided outstanding shielding over NEP 24.11 (half-life of >96 h).  These highly constrained peptides are the first examples of NEP 24.11-resistant GLP-1 analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGQuuDP1-5D7Vg90H21EOLACvtfcHk0lilF4GqTZzDBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVarsbw%253D&md5=a7a8385aa91eae3f44d1231c8bb937d7</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm100602m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100602m%26sid%3Dliteratum%253Aachs%26aulast%3DMurage%26aufirst%3DE.%2BN.%26aulast%3DGao%26aufirst%3DG.%26aulast%3DBisello%26aufirst%3DA.%26aulast%3DAhn%26aufirst%3DJ.-M.%26atitle%3DDevelopment%2520of%2520Potent%2520Glucagon-like%2520Peptide-1%2520Agonists%2520with%2520High%2520Enzyme%2520Stability%2520via%2520Introduction%2520of%2520Multiple%2520Lactam%2520Bridges%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6412%26epage%3D6420%26doi%3D10.1021%2Fjm100602m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR','PDB','1YCR'); return false;">PDB: 1YCR</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i37"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01638">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_40630"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01638?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01638</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Preparation of sulfono-γ-AA building blocks; preparation of α/Sulfono-γ-AA peptides; fluorescence polarization competition assays; circular dichroism; enzymatic stability study; <sup>1</sup>H–<sup>15</sup>N HSQC NMR of lead peptide <b>9</b> in complex with MDM2; luciferase reporter assay; <sup>1</sup>H and <sup>13</sup>C NMR spectra of the sulfono-γ-AA peptide building blocks (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf">PDF</a>)</p></li><li><p class="inline">PDB file of p53 α-helix with a designed 1:1 α/Sulfono-γ-AA peptide and MDM2 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_002.pdb">PDB</a>)</p></li><li><p class="inline">PDB file of stapled sequence binding with MDM2 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_004.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_001.pdf">jm0c01638_si_001.pdf (3.86 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_002.pdb">jm0c01638_si_002.pdb (78.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_003.pdb">jm0c01638_si_003.pdb (81.87 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01638/suppl_file/jm0c01638_si_004.csv">jm0c01638_si_004.csv (8.0 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01638&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01638%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-21" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01638" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679abe3618c421be","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
