
<html lang="en"     class="pb-page"  data-request-id="a63bf448-9010-4b40-b4b7-05ca2e0a3dd7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.1c00616;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Peptidomimetic α-Acyloxymethylketone Warheads with Six-Membered Lactam P1 Glutamine Mimic: SARS-CoV-2 3CL Protease Inhibition, Coronavirus Antiviral Activity, and in Vitro Biological Stability" /></meta><meta name="dc.Creator" content="Bing  Bai" /></meta><meta name="dc.Creator" content="Alexandr  Belovodskiy" /></meta><meta name="dc.Creator" content="Mostofa  Hena" /></meta><meta name="dc.Creator" content="Appan Srinivas  Kandadai" /></meta><meta name="dc.Creator" content="Michael A.  Joyce" /></meta><meta name="dc.Creator" content="Holly A.  Saffran" /></meta><meta name="dc.Creator" content="Justin A.  Shields" /></meta><meta name="dc.Creator" content="Muhammad Bashir  Khan" /></meta><meta name="dc.Creator" content="Elena  Arutyunova" /></meta><meta name="dc.Creator" content="Jimmy  Lu" /></meta><meta name="dc.Creator" content="Sardeev K.  Bajwa" /></meta><meta name="dc.Creator" content="Darren  Hockman" /></meta><meta name="dc.Creator" content="Conrad  Fischer" /></meta><meta name="dc.Creator" content="Tess  Lamer" /></meta><meta name="dc.Creator" content="Wayne  Vuong" /></meta><meta name="dc.Creator" content="Marco J.  van Belkum" /></meta><meta name="dc.Creator" content="Zhengxian  Gu" /></meta><meta name="dc.Creator" content="Fusen  Lin" /></meta><meta name="dc.Creator" content="Yanhua  Du" /></meta><meta name="dc.Creator" content="Jia  Xu" /></meta><meta name="dc.Creator" content="Mohammad  Rahim" /></meta><meta name="dc.Creator" content="Howard S.  Young" /></meta><meta name="dc.Creator" content="John C.  Vederas" /></meta><meta name="dc.Creator" content="D. Lorne  Tyrrell" /></meta><meta name="dc.Creator" content="M. Joanne  Lemieux" /></meta><meta name="dc.Creator" content="James A.  Nieman" /></meta><meta name="dc.Description" content="Recurring coronavirus outbreaks, such as the current COVID-19 pandemic, establish a necessity to develop direct-acting antivirals that can be readily administered and are active against a broad spe..." /></meta><meta name="Description" content="Recurring coronavirus outbreaks, such as the current COVID-19 pandemic, establish a necessity to develop direct-acting antivirals that can be readily administered and are active against a broad spe..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 9, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00616" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00616" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00616" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00616" /></link>
        
    
    

<title>Peptidomimetic α-Acyloxymethylketone Warheads with Six-Membered Lactam P1 Glutamine Mimic: SARS-CoV-2 3CL Protease Inhibition, Coronavirus Antiviral Activity, and in Vitro Biological Stability | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00616" /></meta><meta property="og:title" content="Peptidomimetic α-Acyloxymethylketone Warheads with Six-Membered Lactam P1 Glutamine Mimic: SARS-CoV-2 3CL Protease Inhibition, Coronavirus Antiviral Activity, and in Vitro Biological Stability" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0011.jpeg" /></meta><meta property="og:description" content="Recurring coronavirus outbreaks, such as the current COVID-19 pandemic, establish a necessity to develop direct-acting antivirals that can be readily administered and are active against a broad spectrum of coronaviruses. Described in this Article are novel α-acyloxymethylketone warhead peptidomimetic compounds with a six-membered lactam glutamine mimic in P1. Compounds with potent SARS-CoV-2 3CL protease and in vitro viral replication inhibition were identified with low cytotoxicity and good plasma and glutathione stability. Compounds 15e, 15h, and 15l displayed selectivity for SARS-CoV-2 3CL protease over CatB and CatS and superior in vitro SARS-CoV-2 antiviral replication inhibition compared with the reported peptidomimetic inhibitors with other warheads. The cocrystallization of 15l with SARS-CoV-2 3CL protease confirmed the formation of a covalent adduct. α-Acyloxymethylketone compounds also exhibited antiviral activity against an alphacoronavirus and non-SARS betacoronavirus strains with similar potency and a better selectivity index than remdesivir. These findings demonstrate the potential of the substituted heteroaromatic and aliphatic α-acyloxymethylketone warheads as coronavirus inhibitors, and the described results provide a basis for further optimization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00616"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00616">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00616&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00616&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00616&amp;href=/doi/10.1021/acs.jmedchem.1c00616" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00847" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02155" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Peptidomimetic α-Acyloxymethylketone Warheads with Six-Membered Lactam P1 Glutamine Mimic: SARS-CoV-2 3CL Protease Inhibition, Coronavirus Antiviral Activity, and <i>in Vitro</i> Biological Stability</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Bing Bai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bing Bai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Applied Virology Institute, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bing++Bai">Bing Bai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexandr Belovodskiy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexandr Belovodskiy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Applied Virology Institute, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexandr++Belovodskiy">Alexandr Belovodskiy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mostofa Hena</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mostofa Hena</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Applied Virology Institute, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mostofa++Hena">Mostofa Hena</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Appan Srinivas Kandadai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Appan Srinivas Kandadai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Applied Virology Institute, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Appan+Srinivas++Kandadai">Appan Srinivas Kandadai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael A. Joyce</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael A. Joyce</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Institute of Virology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+A.++Joyce">Michael A. Joyce</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Holly A. Saffran</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Holly A. Saffran</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Institute of Virology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Holly+A.++Saffran">Holly A. Saffran</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Justin A. Shields</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Justin A. Shields</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Institute of Virology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Justin+A.++Shields">Justin A. Shields</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Muhammad Bashir Khan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Muhammad Bashir Khan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Muhammad+Bashir++Khan">Muhammad Bashir Khan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elena Arutyunova</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elena Arutyunova</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Institute of Virology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elena++Arutyunova">Elena Arutyunova</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jimmy Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jimmy Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Institute of Virology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jimmy++Lu">Jimmy Lu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sardeev K. Bajwa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sardeev K. Bajwa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sardeev+K.++Bajwa">Sardeev K. Bajwa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Darren Hockman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darren Hockman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Applied Virology Institute, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darren++Hockman">Darren Hockman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Conrad Fischer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Conrad Fischer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Conrad++Fischer">Conrad Fischer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tess Lamer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tess Lamer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tess++Lamer">Tess Lamer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wayne Vuong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wayne Vuong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wayne++Vuong">Wayne Vuong</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-3439-8788" title="Orcid link">https://orcid.org/0000-0002-3439-8788</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marco J. van Belkum</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marco J. van Belkum</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marco+J.++van+Belkum">Marco J. van Belkum</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhengxian Gu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhengxian Gu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec (Shanghai) Co., Ltd., G Warehouse #101, No. 10 Building, #227 Meisheng Road, WaiGaoQiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhengxian++Gu">Zhengxian Gu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fusen Lin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fusen Lin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec (Shanghai) Co., Ltd., G Warehouse #101, No. 10 Building, #227 Meisheng Road, WaiGaoQiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fusen++Lin">Fusen Lin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanhua Du</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanhua Du</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec (Shanghai) Co., Ltd., G Warehouse #101, No. 10 Building, #227 Meisheng Road, WaiGaoQiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanhua++Du">Yanhua Du</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jia Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jia Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec (Shanghai) Co., Ltd., G Warehouse #101, No. 10 Building, #227 Meisheng Road, WaiGaoQiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jia++Xu">Jia Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mohammad Rahim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mohammad Rahim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rane Pharmaceuticals, Inc. 4290 91a Street NW, Edmonton, Alberta T6E 5V2, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mohammad++Rahim">Mohammad Rahim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Howard S. Young</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Howard S. Young</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Howard+S.++Young">Howard S. Young</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John C. Vederas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John C. Vederas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+C.++Vederas">John C. Vederas</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2996-0326" title="Orcid link">https://orcid.org/0000-0002-2996-0326</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">D. Lorne Tyrrell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">D. Lorne Tyrrell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Applied Virology Institute, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Li Ka Shing Institute of Virology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=D.+Lorne++Tyrrell">D. Lorne Tyrrell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">M. Joanne Lemieux</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">M. Joanne Lemieux</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Institute of Virology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=M.+Joanne++Lemieux">M. Joanne Lemieux</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4745-9153" title="Orcid link">https://orcid.org/0000-0003-4745-9153</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">James A. Nieman</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James A. Nieman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Li Ka Shing Applied Virology Institute, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div><div class="loa-info-affiliations-info">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#670d090e020a06092712060b0502151306490406"><span class="__cf_email__" data-cfemail="bbd1d5d2ded6dad5fbcedad7d9dec9cfda95d8da">[email protected]</span></a>. Tel: +1-(780) 492-8704.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+A.++Nieman">James A. Nieman</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-5087-0819" title="Orcid link">https://orcid.org/0000-0002-5087-0819</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00616&amp;href=/doi/10.1021%2Facs.jmedchem.1c00616" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 9, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 April 2021</li><li><span class="item_label"><b>Published</b> online</span>9 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00616" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00616</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DBing%2BBai%252C%2BAlexandr%2BBelovodskiy%252C%2BMostofa%2BHena%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00616%26title%3DPeptidomimetic%2B%25CE%25B1-Acyloxymethylketone%2BWarheads%2Bwith%2BSix-Membered%2BLactam%2BP1%2BGlutamine%2BMimic%253A%2BSARS-CoV-2%2B3CL%2BProtease%2BInhibition%252C%2BCoronavirus%2BAntiviral%2BActivity%252C%2Band%2Bin%2BVitro%2BBiological%2BStability%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D21%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00616"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1358</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00616" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Peptidomimetic α-Acyloxymethylketone Warheads with Six-Membered Lactam P1 Glutamine Mimic: SARS-CoV-2 3CL Protease Inhibition, Coronavirus Antiviral Activity, and in Vitro Biological Stability&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Bing&quot;,&quot;last_name&quot;:&quot;Bai&quot;},{&quot;first_name&quot;:&quot;Alexandr&quot;,&quot;last_name&quot;:&quot;Belovodskiy&quot;},{&quot;first_name&quot;:&quot;Mostofa&quot;,&quot;last_name&quot;:&quot;Hena&quot;},{&quot;first_name&quot;:&quot;Appan&quot;,&quot;last_name&quot;:&quot;Srinivas Kandadai&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;A. Joyce&quot;},{&quot;first_name&quot;:&quot;Holly&quot;,&quot;last_name&quot;:&quot;A. Saffran&quot;},{&quot;first_name&quot;:&quot;Justin&quot;,&quot;last_name&quot;:&quot;A. Shields&quot;},{&quot;first_name&quot;:&quot;Muhammad&quot;,&quot;last_name&quot;:&quot;Bashir Khan&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Arutyunova&quot;},{&quot;first_name&quot;:&quot;Jimmy&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Sardeev&quot;,&quot;last_name&quot;:&quot;K. Bajwa&quot;},{&quot;first_name&quot;:&quot;Darren&quot;,&quot;last_name&quot;:&quot;Hockman&quot;},{&quot;first_name&quot;:&quot;Conrad&quot;,&quot;last_name&quot;:&quot;Fischer&quot;},{&quot;first_name&quot;:&quot;Tess&quot;,&quot;last_name&quot;:&quot;Lamer&quot;},{&quot;first_name&quot;:&quot;Wayne&quot;,&quot;last_name&quot;:&quot;Vuong&quot;},{&quot;first_name&quot;:&quot;Marco&quot;,&quot;last_name&quot;:&quot;J. van Belkum&quot;},{&quot;first_name&quot;:&quot;Zhengxian&quot;,&quot;last_name&quot;:&quot;Gu&quot;},{&quot;first_name&quot;:&quot;Fusen&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Yanhua&quot;,&quot;last_name&quot;:&quot;Du&quot;},{&quot;first_name&quot;:&quot;Jia&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Mohammad&quot;,&quot;last_name&quot;:&quot;Rahim&quot;},{&quot;first_name&quot;:&quot;Howard&quot;,&quot;last_name&quot;:&quot;S. Young&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;C. Vederas&quot;},{&quot;first_name&quot;:&quot;D.&quot;,&quot;last_name&quot;:&quot;Lorne Tyrrell&quot;},{&quot;first_name&quot;:&quot;M.&quot;,&quot;last_name&quot;:&quot;Joanne Lemieux&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;A. Nieman&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00616&quot;},&quot;abstract&quot;:&quot;Recurring coronavirus outbreaks, such as the current COVID-19 pandemic, establish a necessity to develop direct-acting antivirals that can be readily administered and are active against a broad spectrum of coronaviruses. Described in this Article are novel α-acyloxymethylketone warhead peptidomimetic compounds with a six-membered lactam glutamine mimic in P1. Compounds with potent SARS-CoV-2 3CL protease and in vitro viral replication inhibition were identified with low cytotoxicity and good plasma and glutathione stability. Compounds 15e, 15h, and 15l displayed selectivity for SARS-CoV-2 3CL protease over CatB and CatS and superior in vitro SARS-CoV-2 antiviral replication inhibition compared with the reported peptidomimetic inhibitors with other warheads. The cocrystallization of 15l with SARS-CoV-2 3CL protease confirmed the formation of a covalent adduct. α-Acyloxymethylketone compounds also exhibited antiviral activity against an alphacoronavirus and non-SARS betacoronavirus strains with similar potency &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00616&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00616" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00616&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00616" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00616&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00616" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00616&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00616&amp;href=/doi/10.1021/acs.jmedchem.1c00616" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00616" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00616" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00616&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00616%26sid%3Dliteratum%253Aachs%26pmid%3D34242027%26genre%3Darticle%26aulast%3DBai%26date%3D2021%26atitle%3DPeptidomimetic%2B%25CE%25B1-Acyloxymethylketone%2BWarheads%2Bwith%2BSix-Membered%2BLactam%2BP1%2BGlutamine%2BMimic%253A%2BSARS-CoV-2%2B3CL%2BProtease%2BInhibition%252C%2BCoronavirus%2BAntiviral%2BActivity%252C%2Band%2Bin%2BVitro%2BBiological%2BStability%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292644" title="SARS-CoV-2">SARS-CoV-2</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0011.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Recurring coronavirus outbreaks, such as the current COVID-19 pandemic, establish a necessity to develop direct-acting antivirals that can be readily administered and are active against a broad spectrum of coronaviruses. Described in this Article are novel α-acyloxymethylketone warhead peptidomimetic compounds with a six-membered lactam glutamine mimic in P1. Compounds with potent SARS-CoV-2 3CL protease and <i>in vitro</i> viral replication inhibition were identified with low cytotoxicity and good plasma and glutathione stability. Compounds <b>15e</b>, <b>15h</b>, and <b>15l</b> displayed selectivity for SARS-CoV-2 3CL protease over CatB and CatS and superior <i>in vitro</i> SARS-CoV-2 antiviral replication inhibition compared with the reported peptidomimetic inhibitors with other warheads. The cocrystallization of <b>15l</b> with SARS-CoV-2 3CL protease confirmed the formation of a covalent adduct. α-Acyloxymethylketone compounds also exhibited antiviral activity against an alphacoronavirus and non-SARS betacoronavirus strains with similar potency and a better selectivity index than remdesivir. These findings demonstrate the potential of the substituted heteroaromatic and aliphatic α-acyloxymethylketone warheads as coronavirus inhibitors, and the described results provide a basis for further optimization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar"><issue-title>COVID-19</issue-title></a> special issue.</p></div><div class="extra-info-sec articleNote"><h4><span></span>  Note</h4><p class="first last">This article is made available via the <a href="/page/vi/chemistry_coronavirus_research" class="ext-link">ACS COVID-19 subset</a> for unrestricted RESEARCH re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62203" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62203" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Viruses in Group IV of the Baltimore classification system have a positive-sense single-stranded RNA genome.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Members of Group IV are responsible for a range of diseases that cause mortality and morbidity in birds and mammals. Many Group IV viruses also infect humans, including rhinoviruses, noroviruses, and coronaviruses. Human rhinoviruses are responsible for up to 40% of the seasonal common cold,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and human noroviruses cause acute gastroenteritis, resulting in an estimated 50 000 deaths worldwide per year.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The current global Coronavirus Disease 2019 (COVID-19) pandemic is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which is a member of the <i>Coronaviridae</i> family.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Other coronaviruses are responsible for a portion of common seasonal colds as well as 2003 Severe Acute Respiratory Syndrome (SARS, caused by the SARS-CoV-1), and 2012 Middle East Respiratory Syndrome (MERS, caused by the MERS-CoV). Coronaviruses have four genera with two, the alpha- and betacoronaviruses, descending from the bat viral gene pool, with SARS-CoV-1, MERS-CoV, and SARS-CoV-2 being betacoronaviruses.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a></div><div class="NLM_p">Rhinoviruses, noroviruses, and coronaviruses share a similar protease critical for processing the viral polyprotein required for viral replication in the host cells.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The peptide sequence cleaved by a protease is labeled as P3P2P1↓P1′P2′P3′, where the amide bond cleavage occurs between P1 and P1′, as indicated by the arrow, and the corresponding sites on the protease protein are referred to as S3S2S1S1′S2′S3′.<a onclick="showRef(event, 'cit8b'); return false;" href="javascript:void(0);" class="ref cit8b">(8b)</a> For rhinoviruses, noroviruses, and coronaviruses, this main protease is a cysteine protease that primarily hydrolyzes proteins between a P1 glutamine and a small P1′ amino acid, such as alanine or glycine.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> For rhinovirus, this protease is called 3C protease (3CP), whereas for norovirus and coronavirus, it is 3C-like protease (3CLP, 3CL<sup>pro</sup>, or M<sup>pro</sup>). Beyond P1 and P1′, significant differences in substrate specificity are observed. For example, Zhang and coworkers with a series of crystal structures observed significant shape and volume differences for S2 in different coronaviruses.<a onclick="showRef(event, 'cit8d'); return false;" href="javascript:void(0);" class="ref cit8d">(8d)</a></div><div class="NLM_p">Rhinovirus 3CP is the target of the clinically evaluated compound rupintrivir (<b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> During the rupintrivir drug discovery project at Agouron (Pfizer), a five-membered lactam at P1 was identified as a favorable glutamine replacement, where the peptidomimetic labels P1, P2, and P3 align with those of the substrates.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The five-membered lactam at P1 has been used in compounds directed against other Group IV viruses, besides rhinovirus 3CP, like norovirus 3CLP compound <b>2</b><a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and the published SARS-CoV-1 3CLP inhibitor <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The α,β-unsaturated ester moiety, referred to as the warhead, in <b>1</b> reacts with the catalytic sulfur of the cysteine protease to form an irreversible, covalent bond. Compounds <b>2</b> and <b>3</b> have different warheads, an aldehyde and a carbonyl of an α-hydroxymethylketone, respectively, that form reversible, covalent adducts with the sulfur.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Published protease inhibitors for rhinovirus 3CP (<b>1</b>, rupintrivir), norovirus 3CLP <b>2</b>, and SARS 3CLP (<b>3</b>, PF-835231).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2017, the Li Ka Shing Applied Virology Institute (LKSAVI) began to explore direct acting antivirals to treat norovirus infections. Different viral targets were examined, including the polymerase<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and peptidomimetic cysteine 3CLP inhibitors. Utilizing the literature at the time, various compounds were generated as part of the 3CLP portion of that program. What was not observed in the literature was that P2 cyclohexylalanine, such as in <b>2</b>, had cytotoxicity when combined with numerous different warhead types, potentially indicating significant cellular protease inhibition or metabolism of the cyclohexylalanine residue. Recently, others have also observed the cytotoxicity of cyclohexylalanine at P2.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">The outbreak of COVID-19 was first reported in December 2019 in Wuhan, China. To date, over 2.3 million people have died as a result of this ongoing pandemic.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Although fatalities resulting from COVID-19 can impact any age group, the elderly, those with underlying health conditions, and individuals who have poor or nonexistent health care access are the most likely to perish due to this disease.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Without intervention, tens of millions of people can be anticipated to die as a result of COVID-19. Although vaccines were recently approved and are being administered as rapidly as possible, there will still be a need for therapeutics for future coronavirus variants and strains that evade the immune system as well as for those individuals who are immunocompromised. Currently, the only direct-acting antiviral that has been authorized for use is remdesivir, which was originally in clinical trials as an Ebola treatment.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> It is administered intravenously and has demonstrated only modest efficacy for moderate COVID-19.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Novel therapeutics to treat coronavirus infections, especially ones with a different mechanism of action and that can be given orally, are needed not only now but also for potential future coronavirus outbreaks.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p">Shortly after COVID-19 was declared a pandemic, a few noncytotoxic compounds from the legacy LKSAVI norovirus project were tested for their anti-SARS-CoV-2 3CLP activity. Compounds <b>4</b> and <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) both had activity against 3CLP, with <b>5</b> demonstrating a submicromolar IC<sub>50</sub>. Modest antiviral activity was also observed for <b>4</b> and <b>5</b> in a SARS-CoV-2 plaque reduction assay (PRA), which led us to start examining derivatives.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Exploratory noncytotoxic, norovirus project compounds <b>4</b> and <b>5</b> that were active when screened against SARS-CoV-2 3CLP and viral replication. Structure of GC-373 and its bisulfite adduct GC-376 that are reported SARS-CoV-2 3CLP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite the success of targeting metalloproteases as well as serine, threonine, and aspartyl proteases, to date, no cysteine protease inhibitors have made it to market.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Clinical failures of cysteine-protease-targeting drugs like odanacatib and rupintrivir demonstrate the challenges of drug discovery, and yet both demonstrated acceptable safety profiles to be evaluated in clinical trials.<a onclick="showRef(event, 'ref9 ref19'); return false;" href="javascript:void(0);" class="ref ref9 ref19">(9,19)</a> It is worth noting that both drugs are covalent inhibitors and did not appear to have significant toxicity on short-term dosing, the treatment time frame anticipated for an antiviral. The recent growth in successful covalent drugs, including targeting cysteine at or near the active site, has challenged the “all covalent inhibitors are toxic” dogma.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20−22)</a> Covalent inhibition of serine proteases has proven successful by utilizing the nucleophilic nature of the oxygen in the active site to block catalysis.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> To successfully target the catalytic sulfur of cysteine in viral proteases, selectivity is required to avoid cytotoxicity as a result of inhibiting endogenous cysteine proteases. Excess reactivity will result in reaction with nucleophiles such as glutathione, which is present in the cytosol with a concentration range of 1–10 mM.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Optimal recognition elements for SARS-CoV-2 3CLP will help with selectivity and to reduce toxicity, but additional interaction and reactivity considerations will be critical to generate compounds with the antiviral potency, selectivity, and stability needed to become a safe drug.</div><div class="NLM_p">Since the pandemic started, many diverse peptidomimetics have been examined for their ability to inhibit SARS CoV-2 3CLP. This rapid and impressive mobilization has included repurposing peptidomimetic drugs of other proteases and SARS-CoV-2 3CLP-targeted inhibitors, and most of these efforts have already been reviewed.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Many of the SARS-CoV-2 3CLP-targeted inhibitors build on results from the closely related SARS-CoV-1 3CLP investigation.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The SARS-CoV-2 3CLP-targeted inhibitors have almost exclusively utilized a five-membered lactam in P1, with a few recent six-membered P1 lactam examples (vide infra). The general warhead classes of peptidomimetics that have been studied for SARS-CoV-2 3CLP inhibition are enoate, aldehyde, ketoamide, and ketone.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Some noteworthy recent articles include utilizing an aldehyde warhead with an indole P3 capping group by Dai et al. and an aldehyde warhead with P2 hepatitis C virus (HCV) protease proline derivatives by Qiao et al., who observed <i>in vivo</i> efficacy in a mouse model.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Other recent articles by Zhang et al. utilize a ketoamide<a onclick="showRef(event, 'cit8d ref29'); return false;" href="javascript:void(0);" class="ref cit8d ref29">(8d,29)</a> warhead with cyclopropylalanine P2 and a 2-pyridone capping group, and Jin et al. identified an enoate tetrapeptide inhibitor.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The most closely related recent article to the focus of the research we describe herein is from a group at Pfizer that developed peptidomimetic inhibitors, including <b>3</b>, against SARS-CoV-1<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and more recently tested them against SARS-CoV-2 3CLP. That study will be discussed further as follows.</div><div class="NLM_p">We decided to utilize a warhead that would generate an irreversible, covalent adduct. The α,β-unsaturated ester warhead of rupintrivir (<b>1</b>) reacts with the catalytic cysteine of rhinovirus 3CP in a Michael-like reaction to produce an irreversible adduct in the β-position.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> We became interested in the work of Krantz and coworkers,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> in particular, their study with cathepsin B (CatB), although they also reported some structural work with papain, both of which are cysteine proteases, with a different consensus sequence and thus different P1, P2, and capping group preferences.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Their CatB publications examined the leaving groups adjacent (α-position) to a ketone of an α-substituted methylketone.<a onclick="showRef(event, 'cit31a cit31b'); return false;" href="javascript:void(0);" class="ref cit31a cit31b">(31ab)</a> Krantz and coworkers observed that the p<i>K</i><sub>a</sub> of the leaving group correlated with the rate of irreversible, covalent inactivation. The identity of the α-leaving group has to be chosen judiciously. Chloromethylketones, although useful synthetic intermediates, tend to be too reactive and are known to react with glutathione and to demonstrate cytotoxicity.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> A fluoromethylketone is more selective, but concerns with the metabolism generating the extremely toxic metabolite fluoroacetic acid creates challenges with pursuing a pharmaceutical agent with this warhead.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Other leaving groups like diphenylphosphinyl, tretronoyl, and peptidylesters were examined with another cysteine protease, interleukin-1β converting enzyme, although no recent studies have been reported that study leaving group variations.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> Verhelst and coworkers examined a tetrapeptide with P1 glutamine, citrulline, and <i>N</i>,<i>N</i>-dimethylglutamine groups with a previously described 2,6-dimethylbenzoate α-cyloxymethylketone (AMK) as a molecular probe for SARS-CoV-2 3CLP inhibition. They reported a very low deactivation rate with these compounds and no antiviral results and concluded that the P1 residues were not optimal.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> We decided to explore the α-acyloxy group that allows the p<i>K</i><sub>a</sub> to be adjusted to balance the rapid irreversible adduct formation and the excessive reactivity that sequester glutathione or cause cytotoxicity. The potential mechanisms of acyloxy displacement from the α-methyl of the AMK are shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, and the product formed after inactivation has been characterized by X-ray crystallographic and nuclear magnetic resonance (NMR) studies.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Depending on the identity of R′ in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, the ester could be rapidly hydrolyzed by esterases, preventing the irreversible, covalent adduct from forming because the resulting hydroxyl group is not displaced. Krantz and coworkers named these AMK warheads “quiescent affinity labels”, and their work focused exclusively on benzoate esters and a few aliphatic esters.<a onclick="showRef(event, 'cit31a cit31b'); return false;" href="javascript:void(0);" class="ref cit31a cit31b">(31ab)</a> That CatB study identified 2,6-bistrifluoromethyl and 2,6-dichloromethyl benzoates as potent, irreversible inactivators. To our knowledge there are no examples of heteroaromatic esters and only two examples, methoxyacetate by Krantz and peptidylesters by Dai et al., of electron-withdrawing-group-substituted aliphatic esters that improve the leaving group potential of the ester.<a onclick="showRef(event, 'ref31 ref35'); return false;" href="javascript:void(0);" class="ref ref31 ref35">(31,35)</a> Beside these two exceptions, most of the previously studied esters are very hydrophobic and planar (aromatic), which will cause challenges in optimizing the physicochemical properties, such as the aqueous solubility, to obtain good pharmacokinetic and drug-like properties. The heteroaromatic ester group we selected for examination allows the incorporation of additional polar surface area to potentially overcome this. The reduction of planarity, such as reducing the number of sp<sup>2</sup> carbons, can also improve the solubility and other properties, and this was our rational for studying electron-withdrawing-group-substituted aliphatic esters. Because ester moieties can be rapidly cleaved by esterases <i>in vitro</i> and <i>in vivo</i>, the utilization of steric hindrance around the ester to slow hydrolysis is also an important consideration for good stability.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Potential mechanisms for the α-acyloxymethylketone covalent inhibition of cysteine proteases with the sulfur either directly displacing the acyloxygroup (red arrows) or the sulfur attacking the carbon of the ketone (blue arrows) followed by sulfur migration. Both mechanisms produce the same product.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We decided to keep the 4-methoxyindole capping group and leucine in the P2 position of <b>5</b>. On the basis of literature searches at the time and reviews,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> the six-membered glutamine mimic in P1 in <b>5</b> had not been examined against coronaviruses 3CLP, and so it was selected for P1, although it had been examined with retroviral aspartyl proteases (RVPs) and Enterovirus 71 3C proteases.<a onclick="showRef(event, 'ref2 ref38'); return false;" href="javascript:void(0);" class="ref ref2 ref38">(2,38)</a> With the remainder of the molecule held constant, a series of novel AMK-containing compounds were generated to assess their inhibition of both SARS-CoV-2 3CLP and SARS-CoV-2 replication, in addition to their cytotoxicity, stability (plasma and glutathione), cathepsin B and S selectivity, and ADME properties. After commencing our efforts, we were encouraged by the subsequent entry of PF-7304814, which is a phosphate prodrug for the hydroxymethylketone <b>3</b> reported in 2005,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> into clinical trials as an IV treatment for COVID-19.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> Hoffman and coworkers reported AMK derivatives of <b>3</b> but focused on benzoates and other AMK groups previously reported by Krantz.<a onclick="showRef(event, 'ref11 ref39'); return false;" href="javascript:void(0);" class="ref ref11 ref39">(11,39)</a> The derivatives examined by Hoffman et al. did not include 2,6-bis(trifluoromethyl)benzoate, heterocyclic, or electron-withdrawing aliphatic AMK warheads or the P1 six-membered lactam described herein. We examined <b>3</b> in our SARS-CoV-2 3CLP FRET assay and found that it had an IC<sub>50</sub> value of 18.9 nM, which is similar to <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). This reinforced our selection of <b>5</b> as a basis for studying AMK derivatives.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00756" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00756" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">The synthetic routes utilized to generate the compounds are shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> and were used with the six-membered lactam to generate <b>15a</b>–<b>15o</b> and with the five-membered P1 to generate <b>16m</b>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of α-Acyloxymethylketone Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) Sodium chloroacetate, Et<sub>3</sub>N, <i>t</i>-BuMgCl, THF, 63% yield (<i>n</i> = 1), 60% yield (<i>n</i> = 0); (ii) HCl in dioxane, CHCl<sub>3</sub>; (iii) Boc-<span class="smallcaps smallerCapital">l</span>-leucine, HATU, NMM, DMF, 54% yield (two steps, <i>n</i> = 1), 47% yield (two steps, <i>n</i> = 0); (iv) 4-methoxyindole-CO<sub>2</sub>H, HATU, NMM, DMF, 54% yield (two steps, <i>n</i> = 1), 42% (two steps, <i>n</i> = 0); (v) carboxylic acid, NaO<i>t</i>-Bu, NaI, DMF, 15–80% yield.</p></p></figure><div class="NLM_p">The synthesis of intermediates <b>7</b> and <b>8</b> was accomplished starting from <b>6</b> following the procedure of Zhai et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The conversion of the ester to the chloromethylketone utilized chemistry reported by Fukuyama and coworkers for their synthesis of a key intermediate of FR901483.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> They utilized an <i>N</i>-Boc-protected phenylalanine generating an optically active intermediate that, like <b>7</b> and <b>8</b>, also had an epimerizable α-stereocenter. In previous literature, the five-membered lactam <b>10</b> was formed from <b>8</b> by saponification, diazomethane coupling, and HCl treatment or a one-step treatment of lithium diisopropylamine with chloroiodomethane, the latter of which had poor reproducibility.<a onclick="showRef(event, 'ref11 ref39'); return false;" href="javascript:void(0);" class="ref ref11 ref39">(11,39)</a> In our hands, the one-step approach of Fukuyama more reliably produced <b>10</b> from <b>8</b> with a 60% yield and avoided the use of diazomethane. Using the same method with the six-membered lactam containing ester <b>7</b> generated the corresponding chloromethylketone <b>9</b> in 63% yield.</div><div class="NLM_p last">The chloromethylketones <b>9</b> and <b>10</b> were converted to <b>13</b> and <b>14</b>, respectively, following sequential deprotection and coupling protocols previously reported for <b>14</b>.<a onclick="showRef(event, 'ref11 ref39'); return false;" href="javascript:void(0);" class="ref ref11 ref39">(11,39)</a> Careful conversion of the chloromethylketone <b>13</b> to compounds <b>15a</b>–<b>15o</b> was required, as heating, extended reaction time, or excess base treatment produced a diastereomer, presumably at the stereogenic center next to the ketone. If the diastereomer formed, then it was readily observed by <sup>1</sup>H NMR and proved challenging to separate by silica gel chromatography. An example of this was compound <b>15o</b>, which was isolated with ∼25% epimer. The exact reason for this higher diastereomer amount is unclear. It could be a result of the longer time to form the product during the overnight reaction, the susceptibility of the product to epimerization, or an issue with the diastereomeric integrity of the <b>13</b> batch used for this reaction. Compounds <b>15a</b>–<b>15n</b> had significantly less diastereomer based on <sup>1</sup>H and <sup>19</sup>F NMR. (See the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Supporting Information</a>.) For one of the most active compounds, ester <b>15m</b>, the five-membered lactam in P1 was produced. The reaction of 4-trifluoromethyl-2,6-dimethylnicotinic acid with <b>14</b> produced <b>16m</b>, which was isolated with high diastereomeric integrity based on <sup>1</sup>H and <sup>19</sup>F NMR. (See the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Supporting Information</a>.)</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Biological Evaluation</h3><div class="NLM_p">Last year, a subset of authors of this manuscript reported the discovery that GC-373 and its bisulfite adduct GC-376 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) display good anti-SARS-CoV-2 3CLP and antiviral activity (Vero E6 host).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> These 3CLP inhibitors contain a benzyl carbamate capping group, a P1 five-membered lactam, and an aldehyde warhead. For this LKSAVI study, we collaborated with those researchers and utilized the same SARS-CoV-2 3CLP biochemical and virological methods and protocols reported but focused on distinct molecules with AMK warheads, a 4-methoxyindole cap, and a six-membered P1 lactam.</div><div class="NLM_p">The kinetics of protease inhibition by an AMK is complicated because the compounds can show differing degrees of both reversible and irreversible covalent inhibition.<a onclick="showRef(event, 'ref11 ref31 ref39'); return false;" href="javascript:void(0);" class="ref ref11 ref31 ref39">(11,31,39)</a> Even what appears to be reversible inhibition over longer timeframes produces a covalent adduct observed in cocrystal structures that is chemically irreversible.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Despite the complication of this bimodal nature, the ultimate goal is inhibition of coronavirus replication. To prevent viral replication <i>in vitro</i>, high levels of inhibition of the protease after only a short period are required to translate to robust antiviral activity, regardless of the degree of reversible and irreversible inhibition. Our plan was to identify the more active compounds based on the IC<sub>50</sub> values of the SARS-CoV-2 3CLP after a 30 min incubation period. We felt that this would provide insight into the team in an efficient manner. This approach is consistent with that recently reported by Thorarensen and coworkers at Pfizer, who observed that an IC<sub>50</sub> determination after an incubation period was both insightful and an efficient method to assess trends for irreversible, covalent inhibitor programs.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><div class="NLM_p">Cytotoxicity plays a key role in interpreting antiviral activity, as viruses require healthy host cells to effectively replicate. One potential mechanism of cytotoxicity is the inhibition of cellular proteases that impact protein processing, and low or no cytotoxicity will suggest a degree of selectivity over other important proteases in a cellular assay. The selectivity index (SI) is a measure of cytotoxicity divided by antiviral activity (CC<sub>50</sub>/EC<sub>50</sub>). Active compounds that cause cytotoxicity, such as broad-spectrum cellular protease inhibition, will show SARS-CoV-2 antiviral activity and also reduced viability, resulting in a low SI (<10). Active compounds with low or no cytotoxicity will have antiviral activity and high viability, resulting in a high SI (>100). The results for benchmark GC-376 and AMK compounds <b>15a</b> to <b>15o</b> are shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Compound GC-376, which is in clinical development as a feline infectious peritonitis (FIP) treatment and is now being considered as a COVID-19 treatment, has an IC<sub>50</sub> value of 190 nM for the SARS-CoV-2 3CLP.<a onclick="showRef(event, 'ref42 ref44'); return false;" href="javascript:void(0);" class="ref ref42 ref44">(42,44)</a> In comparison with a series of known protease inhibitors, GC-376 is one of the most potent inhibitors of SARS-CoV-2 3CLP.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> All of the AMK compounds tested, <b>15a</b>–<b>15o</b>, had even lower IC<sub>50</sub> values than GC-376 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) against SARS-CoV-2 3CLP.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Compound Inhibition of SARS-CoV-2 3CLP and Viral Replication</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0007.gif" alt="" id="gr6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0008.gif" alt="" id="gr7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">ChemDraw Prime version 16.0.1.4.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Measured p<i>K</i><sub>a</sub> value unless otherwise indicated.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">See the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for details, performed in triplicate with at least eight points in a single experiment.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">SARS-CoV-2 (Vero E6 host cells) with compounds screened at 10 μM and the active ones tested at multiple concentrations to determine EC<sub>50</sub>.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Literature p<i>K</i><sub>a</sub> value for the corresponding carboxylic acid from ref <a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a>.</p></div></div><div></div></div><div class="NLM_p">Compounds <b>15a</b>–<b>15e</b> are pivaloyl- and electron-withdrawing-group-substituted analogs designed to reduce the p<i>K</i><sub>a</sub> of the corresponding acid leaving group. The unsubstituted version, pivalate <b>15a</b>, showed an IC<sub>50</sub> of 64 nM. Krantz and coworkers, with different P1, P2, and capping groups targeting CatB, reported that the pivalate ester had a slow inactivation rate.<a onclick="showRef(event, 'cit31b'); return false;" href="javascript:void(0);" class="ref cit31b">(31b)</a> The related, but novel, warheads with a reduced p<i>K</i><sub>a</sub> for the corresponding acid are <b>15b</b>–<b>15e</b>, and each of these, with the exception of <b>15e</b>, were more active than <b>15a</b>; however, <b>15e</b> was still more active than GC-376. The benzoate derivative <b>15f</b>, also reported as a slow inactivator against CatB by Krantz,<a onclick="showRef(event, 'cit31b'); return false;" href="javascript:void(0);" class="ref cit31b">(31b)</a> had an essentially identical IC<sub>50</sub> to <b>15a</b> against SARS-CoV-2 3CLP. Substitutions on the 2,6-position to lower the p<i>K</i><sub>a</sub> of the corresponding acid resulted in reduced IC<sub>50</sub> values of 6.4 and 1.0 nM for the dichloro and bistrifluoromethyl compounds <b>15g</b> and <b>15h</b>, respectively. These were the best two benzoate ester inactivators reported by Krantz and coworkers for CatB.<a onclick="showRef(event, 'cit31b'); return false;" href="javascript:void(0);" class="ref cit31b">(31b)</a> These provided a comparison for our IC<sub>50</sub> value range for rapid inactivators. We then examined the use of pyridyl compounds, <b>15i</b>, <b>15j</b>, and <b>15k</b>, to reduce the p<i>K</i><sub>a</sub> of the corresponding acid leaving group compared with benzoate <b>15f</b>. Unexpectedly, we found, and subsequently confirmed on retest, that the nicotinate derivative <b>15j</b> was a potent inhibitor of SARS-CoV-2 3CLP, and its IC<sub>50</sub> value was in the range of the dichloro and bistrifluoromethyl derivatives, <b>15g</b> and <b>15h</b>. Substituted variations of <b>15j</b> were produced to improve better plasma stability (<i>vide infra</i>). Both <b>15l</b> and <b>15m</b> were less active against the SARS-CoV-2 3CLP than <b>15j</b>, but they were still about three times more active than the benzoate <b>15f</b>. Trimethyl-substituted pyrimidyl and pyrazole derivatives <b>15n</b> and <b>15o</b> were generated, but only the pyrimidyl derivative showed SARS-CoV-2 3CLP inhibition similar to <b>15l</b> and <b>15m</b>. The slopes of <i>k</i><sub>obs</sub> versus [I] graphs for five compounds, <b>15a</b>, <b>15e</b>, <b>15f</b>, <b>15h,</b> and <b>15l</b>, which represented a range of IC<sub>50</sub> and p<i>K</i><sub>a</sub> values, were determined (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Table S2</a>) and correlate well with the IC<sub>50</sub> values (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Chart S2</a>).</div><div class="NLM_p">Compound GC-376 had an EC<sub>50</sub> value of 0.9 μM in the PRA with a cytotoxicity of >100 μM, resulting in an SI of >111 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> with SARS-CoV-2 and Vero E6 host cells). Compounds <b>15a</b>–<b>15o</b> were initially screened at 10 μM, and any compound that did not reduce plaques by two-fold was assigned >10 μM. The compounds that significantly reduced the plaque numbers were examined in a concentration–response curve. Compound <b>15c</b> had some reduction in plaques but did not meet the criteria to perform a concentration–response curve, and thus it was assigned >5 μM. The compounds that showed notable antiviral activity were <b>15e</b>, <b>15g</b>, <b>15h</b>, <b>15l</b>, <b>15m</b>, and <b>15n</b>. No toxicity was observed up to the highest concentration tested (200 μM) against the Vero E6 and A549 cell-lines, resulting in a high SI for all active compounds. Compounds <b>15e</b>, <b>15g</b>, <b>15h</b>, <b>15l</b>, and <b>15m</b> were all more active than GC-376, and the most active of them, <b>15h</b> and <b>15l</b>, were established as some of the most potent SARS-CoV-2 antivirals reported.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Although numerous compounds had good SARS-CoV-2 3CLP IC<sub>50</sub> values, all of the compounds with a submicromolar EC<sub>50</sub> value in the PRA had a calculated LogP of 2.38 or greater. Compounds with a higher lipophilicity are anticipated to have higher permeability, which could explain why these compounds demonstrate better antiviral inhibition, despite similar protease IC<sub>50</sub> numbers.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Other factors, such as the stability in the presence of esterases, could also play a role during the PRA. Compounds with a lipophilicity of <3.0 are desirable,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and the calculated LogP values, with the exception of <b>15e</b>, suggested that the compounds might have reasonable absorption, distribution, metabolism, and excretion (ADME) properties.</div><div class="NLM_p">The compounds shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> all had the six-membered lactam at P1. To evaluate the influence of the glutamine mimic at P1, the five-membered lactam version of <b>15m</b> was synthesized, compound <b>16m</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Compound <b>16m</b> displayed an IC<sub>50</sub> against SARS-CoV-2 3CLP that was very close to that for <b>15m</b>, and <b>16m</b> showed a similar, or possibly slightly better, EC<sub>50</sub> in the PRA.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. SARS-CoV-2 3CLP and antiviral results for five-membered lactam <b>16m</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To assess the spectrum of activity of these AMK warhead-containing peptidomimetics, we screened selected compounds against different coronaviruses in a cytopathic effect (CPE) assay. <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> shows the results against CoV-229E (alphacoronavirus) and CoV-OC43 (betacoronavirus) with remdesivir utilized as a positive control.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Activity against Representative Alpha- and Betacoronaviruses</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cmpd</th><th class="colsep0 rowsep0" align="center" char=".">229E CPE AV EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">MRC5 CC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">OC43 CPE AV EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">Huh-7 CC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">remdesivir</td><td class="colsep0 rowsep0" align="char" char=".">0.035</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.020</td><td class="colsep0 rowsep0" align="center" char=".">10.37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15c</b></td><td class="colsep0 rowsep0" align="char" char=".">0.112</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.195</td><td class="colsep0 rowsep0" align="center" char=".">94.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15e</b></td><td class="colsep0 rowsep0" align="char" char=".">0.196</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.395</td><td class="colsep0 rowsep0" align="center" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15g</b></td><td class="colsep0 rowsep0" align="char" char=".">0.021</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.035</td><td class="colsep0 rowsep0" align="center" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15h</b></td><td class="colsep0 rowsep0" align="char" char=".">0.029</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.047</td><td class="colsep0 rowsep0" align="center" char=".">94.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15l</b></td><td class="colsep0 rowsep0" align="char" char=".">0.048</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.054</td><td class="colsep0 rowsep0" align="center" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15m</b></td><td class="colsep0 rowsep0" align="char" char=".">0.030</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.045</td><td class="colsep0 rowsep0" align="center" char=".">67.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16m</b></td><td class="colsep0 rowsep0" align="char" char=".">0.013</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.033</td><td class="colsep0 rowsep0" align="center" char=".">44.90</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Averages of two experiments with <i>Z</i>′ for 229E and OC43 consistently have been 0.79 ± 0.01 and 0.73 ± 0.01, respectively. 229E and OC43 CPE assays utilized MRC5 and Huh-7 host cells, respectively.</p></div></div></div><div class="NLM_p">Our compounds had antiviral activity against the CoV-229E alphacoronavirus (MRC5 host cells) with EC<sub>50</sub> values ranging from 13 to 196 nM, with <b>15g</b>, <b>15h</b>, <b>15m</b>, and <b>16m</b> having better activity than remdesivir. The cytotoxicity observed against the MRC5 cell line was >100 μM, which was the highest concentration tested, and was consistent with that observed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> against the Vero E6 and A549 cells. The SI was excellent for the compounds tested ranging from >510 to >7692 in the CoV-229E CPE assay.</div><div class="NLM_p last">Our compounds also had excellent antiviral activity against the CoV-OC43 betacoronavirus (Huh-7 host cells). The EC<sub>50</sub> values were slightly higher than those against CoV-229E, although a different cell line was also utilized (Huh-7 vs MRC5) in addition to the different strain. Remdesivir was a more active inhibitor of CoV-OC43 replication than all of the compounds tested, although <b>15g</b>, <b>15h</b>, <b>15l</b>, <b>15m</b>, and <b>16m</b> were only a few times less active. Remdesivir had a CC<sub>50</sub> value of 10.37 μM (SI = 518.5) against Huh-7 cells. Some of the compounds tested also had CC<sub>50</sub> values of <100 μM, although none were as low as the approved drug remdesivir. Compound <b>15m</b> and <b>16m</b> had the lowest CC<sub>50</sub> values, and yet their SIs were still 1502 and 1360, respectively, more than two times better than that of remdesivir.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> ADME Evaluation</h3><div class="NLM_p">In a search of the literature for the stability and ADME properties assessment of peptidomimetic AMK groups, we found only one instance of a peptidyl ester, which was very unstable.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> To determine if our AMK derivatives had acceptable properties for <i>in vivo</i> pharmacokinetic (PK) testing and efficacy studies, we assessed select compounds in an <i>in vitro</i> ADME panel (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) and stability studies. These compounds are not predicted to be charged at physiological pH, so the LogD was anticipated to be similar to the LogP. The calculated LogP (ChemDraw) for the compounds was in an acceptable range; however, as can be seen in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, the measured LogD (pH 7.4) was almost 2 units higher, all above 4, thus diminishing the probability that these compounds will have acceptable PK properties.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. ADME Properties for Select Compounds</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0009.gif" alt="" id="gr9" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0010.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">ChemDraw Prime version 16.0.1.4.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Shake flask method at 25 °C.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Average of two different experiments with efflux ratio = B to A/A to B.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Non-NADPH-dependent elimination of compound observed with microsomes and compound in the absence of NADPH.</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">Loss of compound was observed in medium without cells.</p></div><div class="footnote" id="t3fn6"><sup><sup>f</sup></sup><p class="last">Insufficient recovery due to nonspecific binding or instability in the matrix or other issues.</p></div></div><div></div></div><div class="NLM_p">The aliphatic ester <b>15c</b> was unstable in mouse plasma with a very short half life, suggesting that esterases present in the plasma are able to rapidly hydrolyze this hindered ester (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Compound <b>15g</b> was also somewhat unstable in plasma, with a half life under 3 h. These two compounds also showed non-NADPH-dependent metabolism in the presence of microsomes and low recovery in the Caco-2 permeability assay (not shown). When examined in the presence of glutathione, both <b>15c</b> and <b>15g</b> underwent significant hydrolysis in the pH 7.4 buffer over 24 h at 37 °C (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Table S1</a>), with virtually no glutathione adduct formation.</div><div class="NLM_p">Compounds <b>15l</b>, <b>15m</b>, and <b>16m</b> were all stable in mouse plasma, were not metabolized independent of NADPH, and had good Caco-2 assay recovery (not shown). All three had LogD over 4, so it is understandable that they had short half lives in the presence of microsomes and hepatocytes. Their lack of NADPH-independent metabolism and their short <i>T</i><sub>1/2</sub> in the presence of microsomes suggests that oxidative metabolism was the primary method of clearance of all three compounds. These three compounds also had good basolateral to apical (B to A) permeability of >20 × 10<sup>–6</sup> cm/s in the Caco-2 assay; however, significant efflux was observed, suggesting that these structures, or portions of these structures, were recognized by an efflux transporter. Compounds <b>15l</b> and <b>15m</b> both showed >80% stability in the presence of 10 mM glutathione (pH 7.4) after 24 h at 37 °C, indicating reasonable stability to glutathione-mediated clearance and nominal nondiscriminant thiol reactivity (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Table S1</a>).</div><div class="NLM_p">The hydrolysis of esters by esterases is influenced by a combination of steric and electronic factors.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Reducing the p<i>K</i><sub>a</sub> to more readily inhibit SARS-CoV-2 3CLP, such as in <b>15c</b> compared with <b>15a</b>, could also increase the rate of hydrolysis by an esterase in the plasma. To further understand the factors that might slow hydrolysis by esterases and not impact SARS-CoV-2 3CLP inhibition, we examined the rates of hydrolysis in human plasma (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The rapid hydrolysis of <b>15c</b> that occurred in mouse plasma also was observed in human plasma. In contrast, pivalate <b>15a</b> is very stable in human plasma, suggesting that the inductive influence of the CF<sub>3</sub> group in <b>15c</b> increased the hydrolysis rate despite the steric hindrance. Hydrolysis was extremely rapid for <b>15b</b>, further signifying that the electronic effects that reduced p<i>K</i><sub>a</sub> also contributed to the increased hydrolysis rate. Compound <b>15e</b> was significantly more stable than <b>15c</b> to esterases (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>) and was also more resistant to hydrolysis in the presence of glutathione (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Table S1</a>). The CF<sub>3</sub> group in <b>15e</b> and <b>15c</b> is expected to have a similar electronic influence, and thus the increase in the steric bulk of the cyclohexyl group over the <i>gem</i>-dimethyl is likely responsible for the reduced hydrolysis rate observed for <b>15e</b>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Human Plasma Stability for Select Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">human plasma stability</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cmpd</th><th class="colsep0 rowsep0" align="center" char=".">cmpd at 2 h (%)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (min)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15a</b></td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15b</b></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char="."><15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15c</b></td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15e</b></td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15f</b></td><td class="colsep0 rowsep0" align="char" char=".">67</td><td class="colsep0 rowsep0" align="char" char=".">209</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15h</b></td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15j</b></td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char="."><15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15l</b></td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15m</b></td><td class="colsep0 rowsep0" align="char" char=".">99</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Average of two runs in relation to internal standard and hydrolyzed product <b>5</b>.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Determined from four time points starting at 15 min and ending at 2 h.</p></div></div></div><div class="NLM_p">Benzoate esters are known to be substrates for esterases, as the results for <b>15f</b> confirm (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The 2,6-bistrifluoromethyl substitution in <b>15h</b> increased the stability in human plasma compared with the benzoate. The nicotinate <b>15j</b> was more unstable in human plasma than benzoate <b>15h</b>. Substitution of two methyls or a methyl and a trifluoromethyl adjacent to the ester of <b>15j</b> generated compounds <b>15l</b> and <b>15m</b>, respectively, which both showed excellent stability in human plasma.</div><div class="NLM_p">Literature 3CP and 3CLP peptidomimetics, such as <b>1</b>, <b>2</b>, and <b>3</b>, are reported to display low levels of inhibition of serine proteases and metalloproteases.<a onclick="showRef(event, 'ref2 ref10 ref39'); return false;" href="javascript:void(0);" class="ref ref2 ref10 ref39">(2,10,39)</a> Warheads, such as AMK and those in <b>1</b>, <b>2</b>, and <b>3</b>, are designed to react with the catalytic sulfur of the cysteine protease. In addition to assessing the glutathione reactivity (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Table S1</a>), it is prudent to also determine the selectivity over representative endogenous cysteine proteases, in particular, ones with similar specificities. For that reason, despite AMK compounds showing a high SI, we decided to measure their inhibition of two representative human cysteine proteases, cathepsin B (CatB) and cathepsin S (CatS). CatS has a similar consensus sequence to SARS-CoV-2 3CLP with specificities including a leucine in P2 and a glutamine in P1.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Analogs of <b>1</b> and hydroxymethylketone- moiety-containing <b>3</b> are reported to demonstrate moderate CatB inhibition, although CatS inhibition was not reported.<a onclick="showRef(event, 'ref2 ref39'); return false;" href="javascript:void(0);" class="ref ref2 ref39">(2,39)</a> Krantz and coworkers examined their AMK analogs as CatB inhibitors, further justifying the selection of CatB to examine selectivity.<a onclick="showRef(event, 'cit31a cit31b'); return false;" href="javascript:void(0);" class="ref cit31a cit31b">(31a,b)</a><a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> shows the results for select examples of our AMK compounds. For comparison we also screened an aldehyde warhead and norovirus 3CLP compound <b>2</b> (benchmark from the LKSAVI norovirus program) in addition to compound <b>5</b>, which is a P1 homologue of <b>3</b>.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Human Cathepsin B and S Inhibition by Select Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char="±">inhibition</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cmpd</th><th class="colsep0 rowsep0" align="center" char="±">CatB % at 1 μM</th><th class="colsep0 rowsep0" align="center" char="±">CatB IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="±">CatS % at 1 μM</th><th class="colsep0 rowsep0" align="center" char="±">CatS IC<sub>5</sub><sub>0</sub> (nM)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char="±">72.5 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">99.3 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char="±">26.5 ± 1.7</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">60.8 ± 4.4</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15e</b></td><td class="colsep0 rowsep0" align="char" char="±">38.6 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">2854 ± 71</td><td class="colsep0 rowsep0" align="char" char="±">84.2 ± 2.9</td><td class="colsep0 rowsep0" align="char" char="±">444 ± 29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15f</b></td><td class="colsep0 rowsep0" align="char" char="±">16.2 ± 7.4</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">41.0 ± 7.4</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15h</b></td><td class="colsep0 rowsep0" align="char" char="±">91.3 ± 1.6</td><td class="colsep0 rowsep0" align="char" char="±">464 ± 24</td><td class="colsep0 rowsep0" align="char" char="±">96.8 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">116 ± 11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15l</b></td><td class="colsep0 rowsep0" align="char" char="±">86.7 ± 1.7</td><td class="colsep0 rowsep0" align="char" char="±">201 ± 5</td><td class="colsep0 rowsep0" align="char" char="±">98.1 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">51.5 ± 11.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15m</b></td><td class="colsep0 rowsep0" align="char" char="±">95.0 ± 1.2</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">98.4 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">See the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for details. Performed in triplicate with six concentrations in a single experiment. Results were consistent with screening (triplicate) at 1 μM.</p></div></div></div><div class="NLM_p last">Compounds were screened at 1 μM with a 30 min incubation period to be consistent with SARS-CoV-2 3CLP inhibition results in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. CatS was inhibited by all of the compounds tested to a greater extent than CatB, suggesting a higher specificity of CatS for these P1 and P2 groups. Aldehyde <b>2</b> had the highest percent inhibition of CatS at 1 μM. The inhibition of CatB with compound <b>5</b> was reasonably consistent with the IC<sub>50</sub> value of 1.3 μM reported for analog <b>3</b>.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> For the tested AMK compounds, the slow inactivator benzoate <b>15f</b> displayed the lowest percent inhibition of 16.2 and 41.0% for CatB and CatS, respectively. AMK compounds <b>15e</b>, <b>15h</b>, <b>15l</b>, and <b>15m</b> all displayed significant inhibition of CatB and CatS at 1 μM, with <b>15h</b>, <b>15l</b>, and <b>15m</b> displaying high levels of inhibition against both. The AMK compound <b>15l</b>, which had a p<i>K</i><sub>a</sub> of 1.7 for the corresponding acid leaving group, had the lowest IC<sub>50</sub> values against CatB and CatS. Compound <b>15h</b>, with a slightly higher measured p<i>K</i><sub>a</sub> of 1.9, displayed good 464- and 116-fold selectivity for SARS-CoV-2 3CLP over CatB and CatS, respectively. Compound <b>15e</b>, with a measured p<i>K</i><sub>a</sub> of 3.4 for the corresponding acid leaving group, had the highest IC<sub>50</sub> values of 2854 and 444 nM for CatB and CatS, respectively, compared with <b>15l</b> and <b>15h</b>. Compound <b>15e</b> displayed some selectivity, 33- and 5.6-fold, for SARS-CoV-2 3CLP over CatB and CatS, respectively.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cocrystallization</h3><div class="NLM_p">To confirm that covalent inhibition was occurring, as depicted in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, we performed cocrystallization experiments. Compound <b>15l</b>, which has an IC<sub>50</sub> value of 19 nM, was cocrystallized with SARS-CoV-2 3CLP. The cocrystal structure obtained had a resolution of 2.15 Å and is shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7MBI">7MBI</a>).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0005.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cocrystal structure of chain C covalent adduct from <b>15l</b> reacting with SARS-CoV-2 3CLP (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7MBI">7MBI</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The SARS-CoV-2 3CLP–<b>15l</b> product crystallizes as a dimer, similar to other crystal structures of 3CLP. In the asymmetric unit, four protomers are found, and the <b>15l</b>–3CLP structure with the active sites of each protomer produced the same adduct, although with slightly different orientations; see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Figure S1</a> for an electron density map for all four protomers. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> shows the active site of one of the protomers (chain C). Overall, the orientation of this dipeptide in the P1 and P2 positions in the protease is similar to that of other inhibitors, yet the P3 position is extended compared with GC-376, for which P1 and P3 positions are stacked.<a onclick="showRef(event, 'ref39 ref42'); return false;" href="javascript:void(0);" class="ref ref39 ref42">(39,42)</a> Consistent with the mechanism in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, the 2,4,6-trimethylnicotinate group has been lost, and the sulfur of Cys145 has formed a covalent bond adjacent to the ketone with a C–S bond length of ∼1.75 Å in each protomer. This type of covalent bond does not support reversibility, and because the sulfur is bonded to the α-carbon instead of the carbonyl carbon, this puts it out of alignment with that of the peptide substrate. This translocated attachment could impact the interactions of the peptidomimetic with the protein. In the protomer in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, the ketone oxygen is within hydrogen-bonding distance of Cys145 amide (3.1 Å N to O) and Gly143 (3.2 Å N to O) of the oxyanion hole. The six-membered lactam is picking up the same H-bonding interactions in S1 as the five-membered lactam for GC-376 and <b>3</b> structures.<a onclick="showRef(event, 'ref39 ref42'); return false;" href="javascript:void(0);" class="ref ref39 ref42">(39,42)</a> The lactam N is 2.7 Å to the carboxylic group of Glu166 and is 2.9 Å to the O of Phe140. The lactam carbonyl is oriented directly toward imidazole NH of His163 (2.8 Å). The isobutyl group of P2 is in a hydrophobic S2 pocket formed from Met49, Met165, and His41. For the capping group (P3), the indole N–H and the carbonyl of the amide make an H-bond donor and acceptor pair of bonds with Glu166, which is consistent with <b>3</b> and with indole GC-376 analogs.<a onclick="showRef(event, 'ref39 ref44'); return false;" href="javascript:void(0);" class="ref ref39 ref44">(39,44)</a> In this protomer (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), the O of the methoxy is within H-bonding distance with Gln189 (3.0 Å N to O). The ligand plot of chain C is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Figure S1</a>. Despite the translocated attachment point, the H-bond interactions have been preserved in the active site of the different protomers.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57018" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57018" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Krantz and coworkers studied AMK warheads, primarily substituted benzoates, as part of a CatB study and found that 2,6-bistrifluoromethyl- and 2,6-dichloromethyl-substituted versions were rapid irreversible, covalent inhibitors.<a onclick="showRef(event, 'cit31b'); return false;" href="javascript:void(0);" class="ref cit31b">(31b)</a> They also found that benzoate and pivalate were slow inactivators. In our study, peptidomimetics <b>15g</b> and <b>15h</b>, designed for SARS-CoV-2 3CLP inhibition, resulted in single-digit nanomolar IC<sub>50</sub> values. Compound <b>15h</b> also showed good 464- and 116-fold selectivity for SARS-CoV-2 3CLP over CatB and CatS, respectively, and high plasma and glutathione stability, suggesting that 3CLP can be targeted selectively <i>in vivo</i> with AMK warheads. The slow inactivator esters, compounds <b>15a</b> and <b>15f</b>, both had an IC<sub>50</sub> ≈ 65 nM. This appeared consistent with Krantz’s observations and provided a gauge for comparing our compounds. Other drug-like properties must be considered for <i>in vivo</i> testing; for example, we found that 2,6-dichloromethylbenzoate derivative <b>15g</b> showed some instability in multiple <i>in vitro</i> stability assays. This instability is an important consideration, as 2,6-dichlorobenzoate AMK was examined in different efforts besides that of Krantz.<a onclick="showRef(event, 'ref34 ref39'); return false;" href="javascript:void(0);" class="ref ref34 ref39">(34,39)</a></div><div class="NLM_p">The electron-withdrawing-group-substituted aliphatic compounds <b>15b</b>–<b>15d</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) had lower IC<sub>50</sub> values against the SARS-CoV-2 3CLP than <b>15a</b> and <b>15f</b>, and one group, the 1-trifluoromethylcyclohex-1-yl of <b>15e</b>, had a slightly higher value. For <b>15a</b> to <b>15e</b>, there is a general trend of the lower p<i>K</i><sub>a</sub> of the leaving acid having a lower IC<sub>50</sub> value, but there are additional factors involved, as it is not a direct correlation. For example, the higher IC<sub>50</sub> of <b>15e</b> compared with that of <b>15a</b>, despite a 1.6 units lower p<i>K</i><sub>a</sub> of the corresponding carboxylic acid, might be a result of steric factors that cause an unfavorable interaction with 3CLP that impacts binding or hinders favorable alignment for inactivation. Compounds <b>15c</b> and <b>15e</b> also demonstrated good antiviral activity against both alpha- and betacoronavirus (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), with their SI values being >253. The instability of <b>15c</b> was observed in plasma (mouse and human) and in the presence of both microsomes and hepatocytes. Compound <b>15e</b> had significantly better human plasma stability, possibly indicating that a steric threshold was overcome to prevent rapid ester hydrolysis by esterases. In the presence of glutathione, compound <b>15e</b> also showed excellent stability to both hydrolysis and reaction with glutathione (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Table S1</a>). Compound <b>15e</b> had 33- and 5.6-fold selectivity for SARS-CoV-2 3CLP over CatB and CatS, respectively. The stability and antiviral activity of <b>15e</b> is encouraging; however, the high lipophilicity contribution of the 1-trifluoromethylcyclohex-1-yl will need to be reduced, especially considering the significantly higher measured LogD observed for analogs in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> compared with their calculated LogP.</div><div class="NLM_p">For the picolinic, nicotinic, and isonicotinic esters (<b>15i</b>, <b>15j</b>, and <b>15k</b>, respectively, in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) only the nicotinic ester showed a noteworthy 3CLP inhibition improvement compared with the benzoate <b>15f</b>. Attempts to explain this observation by modeling failed to shed light on this inhibition preference, again confirming that other factors besides p<i>K</i><sub>a</sub> are important. Nicotinate <b>15j</b> is very unstable in human plasma (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), even more so than benzoate <b>15f</b>. Substituting the nicotinate of <b>15j</b> to improve the stability resulted in analogs <b>15l</b> and <b>15m</b>, both of which had a four times higher IC<sub>50</sub> than <b>15j</b> but still superior inhibition to benzoate <b>15f</b>. Cocrystallizing <b>15l</b> with SARS-CoV-2 3CLP showed that an irreversible, covalent product is ultimately formed. Compounds <b>15l</b> and <b>15m</b> and the five-membered lactam <b>16m</b> had submicromolar EC<sub>50</sub> values in the PRA and showed excellent antiviral inhibition of 229E and OC43 (an alpha- and betacoronavirus, respectively). The activities of these three protease inhibitors in the CPE assays were similar to those of remdesivir (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Compounds <b>15m</b> and <b>16m</b> showed some cytotoxicity in the Huh-7 cell line, but their SI values were still superior to that for remdesivir. The glutathione stability of <b>15l</b> and <b>15m</b> was good (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Table S1</a>), as was their plasma stability (human and mouse). The low metabolic stability in the presence of both hepatocytes and microsomes (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) for <b>15l</b>, <b>15m</b>, and <b>16m</b> is attributed to the high measured lipophilicity. Although the permeabilities for <b>15l</b>, <b>15m</b>, and <b>16m</b> in the Caco-2 assay from the basolateral to the apical direction were good for all three compounds, efflux was an issue. Although the substituted nicotinic derivatives <b>15l</b> and <b>15m</b> have exciting activity with high SI values along with good plasma and glutathione stability, lipophilicity reduction will be required to improve the metabolic stability.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Modest selectivity was observed for <b>15l</b> for SARS-CoV-2 3CLP over CatB and CatS. The lower p<i>K</i><sub>a</sub> for the corresponding acid leaving group of <b>15l</b> might account for the lower selectivity, as it is anticipated to be more reactive. However, its higher IC<sub>50</sub> value than that of <b>15h</b> against SARS-CoV-2 3CLP, despite its lower p<i>K</i><sub>a</sub>, further reinforces that additional factors appear to be involved.</div><div class="NLM_p">Based on the properties of <b>15l</b>, trimethyl-substituted pyrimidyl and pyrazole were generated, derivatives <b>15n</b> and <b>15o</b>, respectively. Compound <b>15n</b>, despite an IC<sub>50</sub> value against SARS-CoV-2 3CLP close to that of <b>15l</b> and <b>15m</b>, was less active in the PRA. The <i>N</i>-methyl-substituted pyrazole <b>15o</b> had a similar IC<sub>50</sub> value to that of the reported slow inactivators, <b>15a</b> and <b>15f</b>. On the basis of the p<i>K</i><sub>a</sub> of the corresponding acid as a measure of the leaving group ability, compound <b>15o</b> with a 2 units lower p<i>K</i><sub>a</sub> should have improved inhibition compared with <b>15f</b>; however, it did not. The <i>N</i>-methyl potentially results in a less favorable interaction with the protein, negating any better leaving group ability.</div><div class="NLM_p">Taken together the results for <b>15a</b> to <b>15o</b> show that refinement of the warhead leaving group has significant effects on the inhibition of SARS-CoV-2 3CLP. The lack of direct correlation of IC<sub>50</sub> and p<i>K</i><sub>a</sub> of the corresponding leaving group indicates that other factors are important for determining the inhibition. Plotting <i>k</i><sub>obs</sub>/<i>I</i> versus p<i>K</i><sub>a</sub> for five compounds (<b>15a</b>, <b>15e</b>, <b>15f</b>, <b>15h</b>, and <b>15l</b>) also showed a trend toward lower p<i>K</i><sub>a</sub> leaving groups having a higher <i>k</i><sub>obs</sub>/<i>I</i> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Chart S1</a>), and the poor correlation also reinforces that other factors appear to be involved besides p<i>K</i><sub>a</sub>. Plotting <i>k</i><sub>obs</sub>/<i>I</i> versus the IC<sub>50</sub> value for the same compounds showed a good correlation of the lower IC<sub>50</sub> value having a higher <i>k</i><sub>obs</sub>/<i>I</i> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Chart S2</a>), which supports our decision to utilize IC<sub>50</sub> as a gauge of activity against SARS-CoV-2 3CLP.</div><div class="NLM_p">We found no reports of toxicity associated with the inhibition of other proteases for <b>2</b>, <b>3</b>, <b>3</b>’s prodrug PF-7304814, or FIP treatment GC-376, all of which have been examined <i>in vivo</i>. Compound <b>2</b> showed extremely potent CatS inhibition at 1 μM. High SI (CC<sub>50</sub>/EC<sub>50</sub>) ratios were obtained for the AMK compounds, and considering other factors, such as the free fraction, their bimodal nature (varying degree of reversibility versus irreversibility), the acute treatment time frame, and enzyme turnover considerations, it is unclear what level of selectivity against related human cysteine proteases, such as CatS, is required for a drug discovery program. Potent inhibitors of CatS have been examined in clinical trials and “are, in general, well-tolerated by patients and healthy volunteers”, although increased respiratory tract infection rates were observed in a small percentage of patients.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Regardless, compound <b>15e</b>, <b>15h</b>, and <b>15l</b> showed selectivity over CatB and CatS.</div><div class="NLM_p">Prior to commencing this work, we observed no publications examining six-membered P1 lactam as a coronavirus 3CLP inhibitor.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> Recently, variations with a different capping group and an aldehyde warhead were reported with modest SARS-CoV-1 and MERS 3CLP inhibition, and a patent application containing ketoamide warheads was published.<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a> The antiviral and protease results for <b>15m</b> and <b>16m</b> were similar, indicating that both lactams are reasonable glutamine mimics for SARS-CoV-2 3CLP. This observation is consistent with what was observed for rhinovirus 3CP peptidomimetics and with the recently published comparison with aldehyde warheads against MERS- and SARS-CoV-1 3CLP.<a onclick="showRef(event, 'ref2 ref53'); return false;" href="javascript:void(0);" class="ref ref2 ref53">(2,53)</a> This potentially represents a method to fine-tune the physiochemical properties of the peptidomimetic by selecting the five- or six-membered lactam in P1.</div><div class="NLM_p last">To achieve the ultimate goal of discovering a more readily administered 3CLP inhibitor to treat coronavirus infections, in particular, COVID-19 and its emerging variants, we will need to generate compounds with more favorable physicochemical properties. Accurately predicting the lipophilicity of the compounds before synthesis will be critical to generating compounds with improved properties. Upon the examination of some different lipophilicity prediction programs, it appears that Marvin LogD might provide a closer approximation of experimental values for these peptidomimetics. With only a small number of compounds with measured LogD values, this preliminary observation will have to be confirmed. On the basis of the encouraging results for the select AMK groups previously described, the formation of analogs with a reduction in the lipophilicity will be required to improve the stability to oxidative metabolism.<a onclick="showRef(event, 'cit48b'); return false;" href="javascript:void(0);" class="ref cit48b">(48b)</a> Compounds with CLogP > 2.38 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) were able to demonstrate EC<sub>50</sub> < 1 μM in the PRA assay. This equates to a measured LogD > 4 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). In reducing the lipophilicity to decrease the rate of oxidative metabolism, additional alterations will be required to further improve or maintain the permeability or reduce the recognition by efflux transporters. It will also be important to lower the molecular mass or to reduce H-bond donors to improve the permeability and possibly lessen the efflux ratio.<a onclick="showRef(event, 'ref47 cit48d'); return false;" href="javascript:void(0);" class="ref ref47 cit48d">(47,48d)</a> Amide bond replacement (as was done toward rupintrivir),<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> warhead refinement, and indole replacement represent opportunities to achieve this. To assess the AMK group <i>in vivo</i>, molecules with improved physicochemical properties that maintain antiviral activity will have to be identified. It is our hope that this work enables that.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Although vaccines are now approved to prevent COVID-19, research into antivirals to treat immunocompromised individuals, variants, and future outbreaks is required. Coronaviruses, and related viruses, have a well-conserved cysteine protease that can be targeted to discover novel direct-acting antivirals. This study previously identified unexplored α-acyloxymethylketone derivatives that exhibited potent inhibition of SARS-CoV-2 3CLP and SARS-CoV-2 replication <i>in vitro</i>. Compounds <b>15l</b> and <b>15m</b> had activity similar to that of the top inhibitors reported in the literature and excellent plasma stability, despite containing an ester. High SIs were observed. Good glutathione stability and selectivity for SARS-CoV-2 3CLP over CatB and CatS signifies that discriminant reactivity can be obtained for these warheads. The cocrystallization of <b>15l</b> with the SARS-CoV-2 3CLP confirmed that a covalent adduct is formed. Compounds <b>15h</b>, <b>15l</b>, <b>15m</b>, and <b>16m</b> also had good antiviral potency against different coronavirus strains (CoV-229E and CoV-OC43). Assessing a six-membered lactam against its five-membered P1 counterpart (<b>15m</b> and <b>16m</b>) demonstrates that these two groups appear to be comparable in activity. Testing the more active compounds in an ADME panel indicated issues with oxidative metabolism and efflux transporter recognition, with the higher measured lipophilicity than predicted being the basis for the observed metabolic instability. These studies lay the foundation for optimizing these and related α-acyloxymethylketone warheads to obtain acceptable properties to facilitate more advanced studies.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58675" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58675" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> SARS-CoV-2 Protease FRET Assay and Determination of IC<sub>50</sub> Values</h3><div class="NLM_p last">Pure SARS-CoV-2 3CLP was obtained as previously described in detail, and the enzymatic activity was confirmed according to an established protocol; see ref <a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a>. The FRET substrate was synthesized according to the methods previously described in ref <a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a>. The IC<sub>50</sub> values for each inhibitor tested were determined using methods described previously in ref <a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> via a FRET-based kinetic assay with a 30 min inhibitor incubation period prior to the addition of the substrate. GraphPad Prism software (GraphPad 8.3.1) was used to determine the IC<sub>50</sub>. Compounds were tested at eight different concentrations (100 to 0.00001 μM). Triplicate experiments were performed for each data point, the appropriate Hill slopes were generated, and the value was presented as a mean ± standard error.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> SARS-CoV-2 Plaque Reduction Assay and Analysis</h3><div class="NLM_p last">The EC<sub>50</sub> and cytotoxicity determinations were carried out using the methods previously described in detail in ref <a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> using SARS-CoV-2/Canada/VIDO 01/2020 and Vero E6 cells. Plaques were counted, and data were plotted as the percent inhibition versus log10 [compound] using Prism (GraphPad). Assays were performed in duplicate with the EC<sub>50</sub> determined with at least five different compound concentrations. EC<sub>50</sub> values were calculated using a nonlinear regression analysis, and the value was presented as a mean ± standard error.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> OC43 and 229E High-Throughput CPE Assay</h3><div class="NLM_p">HCoV 229E and OC43 strains were acquired from the ATCC. MRC5 and Huh7 cells were obtained from the ATCC and AppTec, respectively. MRC5 cells were maintained in minimum essential medium (Sigma) supplemented with 10% fetal bovine serum (FBS) (ExCell), 1% <span class="smallcaps smallerCapital">l</span>-glutamine (Gibco), 1% nonessential amino acid (NEAA) (Gibco), and 1% penicillin–streptomycin (HyClone). Minimum essential medium supplemented with 5% FBS, 1% <span class="smallcaps smallerCapital">l</span>-glutamine, 1% NEAA, and 1% penicillin–streptomycin was used as the assay medium. Huh7 cells were maintained in the Dulbecco’s modified Eagle’s medium (Gibco) supplemented with 10% FBS, 1% <span class="smallcaps smallerCapital">l</span>-glutamine, 1% NEAA, 1% sodium pyruvate (Gibco), and 1% penicillin–streptomycin.</div><div class="NLM_p">Cells were seeded in 96-well plates (20 000 MRC5 and 8000 Huh7 cells per well) in 100 μL per well of assay medium at an appropriate density and cultured at 37 °C and 5% CO<sub>2</sub> overnight. Next day, the test compound was diluted with assay medium and then added to the cells, 50 μL per well. Then, 50 μL per well of assay-medium-diluted virus was added. The final volume of the cell culture was 200 μL per well. The final concentration of dimethyl sulfoxide (DMSO) in the cell culture was 0.5%. The resulting cell culture was incubated at 35 (229E) or 33 °C (OC43) and 5% CO<sub>2</sub> for an additional 3 (229E) or 7 (OC43) days until the virus infection in the virus control (cells infected with virus, without compound treatment) displayed a significant CPE. The CPE was measured by CellTiter Glo following the manufacturer’s manual. The antiviral activity of compounds was calculated based on the protection against the virus-induced CPE at each concentration normalized by the virus control. The cytotoxicity of compounds was assessed under the same conditions, but without virus infection, in parallel. The cell viability was measured with CellTiter Glo following the manufacturer’s manual. The antiviral activity and cytotoxicity of compounds were expressed as the % inhibition and the % viability, respectively, and were calculated with the following formulas<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_m001.gif" alt="" id="_i21" /></img></span><span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_m002.gif" alt="" id="_i22" /></img></span>where raw data<sub>CPD</sub> represents the values of the sample-treated wells, average<sub>VC</sub> is the average value of the virus control, average<sub>CC</sub> is average value of the cell control (cells without virus infection or compound treatment), and average<sub>MC</sub> is the average value of the medium control (medium only) wells.</div><div class="NLM_p last">Compounds were tested at concentrations of 100, 20, 4, 0.8, 0.16, 0.032, 0.0064, and 0.00128 μM in duplicate. Remdesivir was used as a reference compound and was assayed starting from 1 and 100 μM for the EC<sub>50</sub> and CC<sub>50</sub> determinations, respectively. EC<sub>50</sub> and CC<sub>50</sub> values were calculated using the GraphPad Prism software with the equation “log(inhibitor) vs. response -- Variable slope (four parameters)”. The historic Z′ factor for the 229E and OC43 CPE assays is 0.79 ± 0.01 and 0.73 ± 0.01, respectively.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Crystallization of 3CLP with <b>15l</b></h3><div class="NLM_p last">The purified SARS-CoV-2 3CLP was dialyzed against buffer containing 10 mM NaCl and 5 mM Tris HCl pH 8.0 overnight at 4 °C. Protein was concentrated with a Millipore centrifugal filter (10 kDa MW cutoff) to a concentration of 8 mg/mL. Protein was incubated with 5 mM excess of inhibitor at 4 °C for 1 h prior to crystallization. The SARS-CoV-2 3CL protein was subjected to the JCSG plus and PACT crystallization screen (Molecular Dimensions), with hits identified under several conditions for <b>15l</b>. The best crystals were observed with sitting drop trays at room temperature at a ratio of 1:1 with mother liquor of 0.2 M sodium citrate tribasic dihydrate 0.1 M Bis-Tris propane pH 8.5 and 20% w/v PEG 3350. Prior to freezing, crystals were incubated with 22% glycerol as a cryoprotectant. Data collection took place at Stanford Synchrotron Radiation Lightsource (SSRL) (Menlo Park, CA) beamline 12-1 with Blu-Ice<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> using the Web-Ice interface<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and at Canadian Light Source, Inc. (CLSI) beamline 08B1-1 using MxDC.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Diffraction Data Collection, Phase Determination, Model Building, and Refinement</h3><div class="NLM_p last">X-ray diffraction data sets for SARS-CoV-2 3CLP and the inhibitor complex were collected at 100 K in a cold nitrogen stream using SSRL beamline 12-1 at a wavelength of 0.979460, equipped with an Eiger 16 M Pixel Array detector and using CLSI beamline 08B1-1 coupled to PILATUS 6M. Ten data sets were collected for the inhibitor, all of them were processed, and the best was selected based on data statistics. Energy-dispersive X-ray spectroscopy (XDS) and Scala were used for processing of the data sets. The diffraction data set of compound <b>15l</b> with SARS-CoV-2 3CLP was processed at a resolution of 2.15 Å in an orthorhombic space group <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Table S3</a>). The structure was determined by molecular replacement, with the crystal structure of the free enzyme of the SARS-CoV-2 3CLP (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTM">6WTM</a>)<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> as a search model, using the Phaser program from Phenix, version v1.18.1-3855. The first unliganded SARS-CoV-2 3CLP structure was reported by Zhang and coworkers.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The ligand was fit using Coot manually for the inhibitor in the density of the precalculated map from the Phenix refinement. The refinement of the structure was performed with phenix.refine in Phenix software. The structure refinement yielded reasonable values of final R<sub>work</sub> and R<sub>fre</sub>. The statistics of diffraction, data processing, and model refinement are given in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Supporting Information</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">Table S3</a>). The model was inspected with Ramachandran plots, and all showed good stereochemistry. The final model was displayed using PyMOL molecular graphics software (version 2.0 Schrödinger).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Chemistry</h3><div class="NLM_p">All reagents and solvents were used as purchased from commercial sources. Moisture-sensitive reactions were carried out under a nitrogen atmosphere in oven-dried glassware. Reactions were stirred and were performed at ambient temperature, unless otherwise indicated. The reaction progress was monitored by TLC (precoated silica gel aluminum plates containing a fluorescent indicator (F-254)) with UV (254 nm) or staining (ninhydrin or phosphomolybdic acid) or alternatively by LC/MS. <sup>1</sup>H NMR, <sup>13</sup>C NMR, and <sup>19</sup>F NMR spectra were recorded at ambient temperature, unless otherwise indicated, with a Bruker Avance III 600 MHz NMR spectrometer equipped with a Bruker 5 mm PABBO probe. DEPTQ was run for <sup>13</sup>C NMR. Chemical shifts are reported in ppm downfield from tetramethylsilane using residual solvent signals as the internal reference for <sup>1</sup>H NMR and <sup>13</sup>C NMR. <sup>19</sup>F NMR chemical shifts are reported relative to CFCl<sub>3</sub>. NMR spectra were processed utilizing an ACD/Spectrus processor (v2016.1.1, ACD/Laboratories) and Bruker TopSpin 4.0.6. Silica gel column purification was performed on a Biotage Isolera system with either Biotage or Silicycle cartridges. The LC/MS system used for monitoring the progress of the reactions and assessing the purity (absorbance at 254 nm) consisted of a Dionex ULTIMATE 3000 uHPLC module and a Thermo Scientific LTQ XL mass spectrometer with electrospray ionization and an Ion-Trap type of detector (alternating positive–negative mode). Separation was performed with a Thermo Scientific Accucore aQ C18 Polar End-capped LC column (100 mm × 2.1 mm; particle size 2.6 μm, 80 Å). The column was maintained at 30 °C. Commercial high-performance liquid chromatography (HPLC)-grade methanol and domestic “Millipore (Milli-Q)” filtered water were used for chromatography and were modified by the addition of 0.1% (v/v) of formic acid. The eluent was delivered at a constant flow rate of 0.4 mL/min, and the column was equilibrated for 5 min with the corresponding eluent prior to injection of the sample (1 to 3 μL). The gradient (Eluent A) was 45 to 95% methanol–water for 5.25 min followed by 5 min of isocratic 95% methanol–water. The purities of the final compounds were all >95%, as determined by UV (254 nm), and the corresponding <i>m</i>/<i>z</i> had the correct M + H or M – H signal with the appropriate isotope pattern. High-resolution mass spectrometry (HRMS) work was done on an LTQ Orbitrap XL (Thermo Scientific) apparatus operated in positive mode. Compound <b>2</b> was prepared as described in ref <a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a>.</div><div id="sec5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (3<i>S</i>)-3-[(<i>N</i>-{[(3-Chlorophenyl)methoxy]carbonyl}-<span class="smallcaps smallerCapital">l</span>-leucyl)amino]-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl Acetate (<b>4</b>)</h4><div class="NLM_p">To a solution of <b>14</b> (1.02 g, 3.35 mmol) in <i>N</i>,<i>N</i>-dimethylformamide (DMF) (8 mL) was added NaI (494 mg, 3.30 mmol) and potassium acetate (647 mg, 6.60 mmol), and the solution was allowed to sit overnight. Saturated aqueous NH<sub>4</sub>Cl solution was then added, and the mixture was extracted with EtOAc (3 × 40 mL). The combined organic layer was washed with a saturated aqueous NaHCO<sub>3</sub> solution and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 1–4% MeOH in dichloromethane (DCM), which generated (3<i>S</i>)-3-[(<i>tert</i>-butoxycarbonyl)amino]-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate (900 mg, 83% yield) as a light yellow foam. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 6.35 (br d, <i>J</i> = 6.3 Hz, 1H), 5.60 (br s, 1H), 4.97 (d, <i>J</i> = 17.1 Hz, 1H), 4.86 (d, <i>J</i> = 17.1 Hz, 1H), 4.40–4.32 (m, 1H), 3.43–3.34 (m, 2H), 2.53–2.41 (m, 2H), 2.18 (s, 3H), 2.13–2.06 (m, 1H), 1.95–1.84 (m, 2H), 1.47 (s, 9H).</div><div class="NLM_p last">To a solution of (3<i>S</i>)-3-[(<i>tert</i>-butoxycarbonyl)amino]-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate (251 mg, 0.764 mmol) in DCM (3 mL) cooled in an ice bath was added trifluoroacetic acid (TFA) (1.5 mL). After 5 min, the ice bath was removed. After 1 h, the solvent was removed under reduced pressure. The residue was coevaporated with DCM/ether (1:1, 2 × 20 mL) and dried under vacuum. To the resulting residue was added <i>N</i>-{[(3-chlorophenyl)methoxy]carbonyl}-<span class="smallcaps smallerCapital">l</span>-leucine (251 mg, 0.837 mmol) (prepared using the procedure in ref <a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a>) and anhydrous DCM (10 mL). The resulting solution was cooled in an ice bath; then, HATU (433 mg, 1.14 mmol) followed by triethylamine (Et<sub>3</sub>N) (0.250 mL, 1.83 mmol) was added. After 1 h, to the mixture was added an ice/saturated aqueous NaHCO<sub>3</sub> mixture (1:1, 10 mL). The two layers were separated, and the aqueous layer was extracted with DCM (2 × 30 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 1–3% MeOH in CHCl<sub>3</sub>, which generated <b>4</b> (170 mg, 44% yield) as a white solid. <i>R</i><sub>f</sub> = 0.24 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6,</sub> 334 K, D<sub>2</sub>O exchange) δ 7.40–7.32 (m, 3H), 7.31–7.27 (m, 1H), 5.08–5.00 (m, 2H), 4.84–4.74 (m, 2H), 4.44–4.38 (m, 1H), 4.08–4.02 (m, 1H), 3.18–3.11 (m, 1H), 3.10–3.03 (m, 1H), 2.31–2.22 (m, 1H), 2.14–2.07 (m, 1H), 2.07 (s, 3H), 2.01–1.93 (m, 1H), 1.70–1.57 (m, 3H), 1.55–1.43 (m, 2H), 0.90 (d, <i>J</i> = 6.6 Hz, 1H), 0.87 (d, <i>J</i> = 6.6 Hz, 1H). LC/MS: 5.24 min retention time, 98.2% pure. ESI-MS: 510 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>24</sub>H<sub>32</sub>ClN<sub>3</sub>O<sub>7</sub> + H 510.2007, found 510.2023.</div></div><div id="sec5_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-[(2<i>S</i>)-1-({(2<i>S</i>)-4-Hydroxy-3-oxo-1-[(3<i>S</i>)-2-oxopiperidin-3-yl]butan-2-yl}amino)-4-methyl-1-oxopentan-2-yl]-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>5</b>)</h4><div class="NLM_p">To oxo(phenyl)acetic acid (178 mg, 1.19 mmol) in anhydrous DMF (5 mL) was added sodium <i>tert</i>-butoxide (57.1 mg, 0.594 mmol). After 30 min, <b>13</b> (100 mg, 0.198 mmol) and NaI (59.4 mg, 0.396 mmol) were added. After 24 h, saturated brine (10 mL) was added, and the mixture was extracted with CHCl<sub>3</sub> (3 × 10 mL). The combined organic layer was washed with saturated brine (1 × 10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude was dissolved in CHCl<sub>3</sub> (10 mL) and loaded on silica gel column, and the product was purified by silica gel chromatography with a gradient of 0–6% MeOH in CHCl<sub>3</sub>, which afforded (3<i>S</i>)-3-{[<i>N</i>-(4-methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyloxo(phenyl)acetate (62.6 mg, 51% yield) as an off-white solid. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.1 Hz, 1H), 8.67 (d, <i>J</i> = 8.1 Hz, 1H), 8.48 (d, <i>J</i> = 7.6 Hz, 1H), 8.11 (dd, <i>J</i> = 1.0, 7.1 Hz, 1H), 7.86–7.79 (m, 1H), 7.67 (t, <i>J</i> = 7.8 Hz, 2H), 7.47 (br s, 1H), 7.38 (d, <i>J</i> = 1.7 Hz, 1H), 7.11 (t, <i>J</i> = 8.1 Hz, 1H), 7.02 (d, <i>J</i> = 8.2 Hz, 2H), 6.51 (d, <i>J</i> = 7.7 Hz, 1H), 5.31 (d, <i>J</i> = 17 Hz, 1H), 5.25 (d, <i>J</i> = 17 Hz, 1H), 4.61–4.54 (m, 1H), 4.52–4.41 (m, 1H), 3.89 (s, 3H), 3.15–3.04 (m, 2H), 2.35–2.14 (m, 2H), 1.92–1.81 (m, 1H), 1.79–1.65 (m, 4H), 1.65–1.48 (m, 2H), 1.46–1.30 (m, 1H), 0.96 (d, <i>J</i> = 6.3 Hz, 3H), 0.91 (d, <i>J</i> = 6.3 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 209.7, 172.7, 172.7, 161.0, 153.6, 137.8, 129.9, 124.4, 118.1, 105.4, 101.1, 99.2, 65.5, 55.1. 52.7, 51.5, 41.2, 40.2, 37.0, 31.4, 25.5, 24.4, 23.0, 21.4, 21.1. LC/MS: 6.21 min retention time, 96.3% pure. ESI-MS: 619 [M + H]<sup>+</sup>.</div><div class="NLM_p last">To a solution of (3<i>S</i>)-3-{[<i>N</i>-(4-methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyloxo(phenyl)acetate (42.6 mg, 0.069 mmol) in THF/MeOH (1:1, 10 mL) was added CsF (1.05 mg, 0.0069 mmol), and the solution was allowed to sit overnight. Saturated brine solution (10 mL) was then added, and the mixture was extracted with CHCl<sub>3</sub> (3 × 10 mL). The combined organic layer was washed with saturated brine (1 × 10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude was dissolved in CHCl<sub>3</sub> (5 mL) and loaded on a silica gel column, and the product was purified by silica gel chromatography with a gradient of 0 to 2% MeOH in CHCl<sub>3</sub>, which afforded <b>5</b> (18.9 mg, 56% yield) as an off-white solid. <i>R</i><sub>f</sub> = 0.18 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.56 (br s, 1H), 8.46 (d, <i>J</i> = 7.9, 1H), 8.40 (d, <i>J</i> = 7.6, 1H), 7.43 (br s, 1H), 7.36 (br s, 1H), 7.10 (br t, <i>J</i> = 7.7 Hz, 1H), 7.01 (br d, <i>J</i> = 7.9 Hz, 1H), 6.51 (br d, <i>J</i> = 7.5 Hz, 1H), 5.04 (br t, <i>J</i> = 5.6 Hz, 1H), 4.55–4.44 (m, 2H), 4.26 (dd, <i>J</i> = 6.0, 18.5 Hz, 1H), 4.15 (dd, <i>J</i> = 6.0, 18.5 Hz, 1H), 3.89 (s, 3H), 3.15–3.02 (m, 2H), 2.27–2.08 (m, 2H), 1.91–1.80 (m, 1H), 1.78–1.63 (m, 4H), 1.60–1.45 (m, 2H), 1.40–1.22 (m, 1H), 0.95 (br d, <i>J</i> = 6.0 Hz, 3H), 0.90 (br d, <i>J</i> = 6.0 Hz, 3H). LC/MS: 4.23 min retention time, 98.7% pure. ESI-MS: 487 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub> + H 487.2557, found 487.2565.</div></div><div id="sec5_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>tert</i>-Butyl {(2<i>S</i>)-4-Chloro-3-oxo-1-[(3<i>S</i>)-2-oxopiperidin-3-yl]butan-2-yl}carbamate (<b>9</b>)</h4><div class="NLM_p last">To a solution of <b>7</b> (7.00 g, 23.3 mmol) (prepared using methods in ref <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>), sodium chloroacetate (8.96 g, 76.9 mmol), and Et<sub>3</sub>N (10.7 mL, 76.9 mmol) in anhydrous tetrahydrofuran (THF) (200 mL) cooled in an ice bath was added a solution of 2.0 M <i>tert</i>-butylmagnesium chloride solution in THF (116.5 mL, 233 mmol) via a dropping funnel over 1.5 h.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> After 30 min, the ice bath was removed. After 18 h at room temperature, ice was then added, followed by the addition of a precooled 4 N HCl aqueous solution to carefully adjust the pH to 7. The two layers were separated, and the aqueous layer was extracted with EtOAc (3 × 150 mL). The combined organic phase was washed with brine (100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 50–100% EtOAc in hexanes, which generated <b>9</b> (4.7 g, 63% yield) as a white foam. <i>R</i><sub>f</sub> = 0.44 (5% MeOH in EtOAc) was a single spot when visualized with 10% phosphomolybdic acid in ethanol. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.51 (d, <i>J</i> = 7.5 Hz, 1H), 7.49 (br s, 1H), 4.65–4.56 (m, 2H), 4.23–4.16 (m, 1H), 3.15–3.07 (m, 2H), 2.18–2.10 (m, 1H), 2.05–2.00 (m, 1H), 1.92–1.86 (m, 1H), 1.80–1.73 (m, 1H), 1.71–1.64 (m, 1H), 1.61–1.52 (m, 1H), 1.40–1.37 (m, 9H), 1.36–1.34 (m, 1H).</div></div><div id="sec5_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>tert</i>-Butyl {(2<i>S</i>)-4-Chloro-3-oxo-1-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butan-2-yl}carbamate (<b>10</b>)</h4><div class="NLM_p last">To a solution of <b>8</b> (8.5 g, 29.7 mmol) (prepared using the method in ref <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>), sodium chloroacetate (11.4 g, 98.0 mmol), and Et<sub>3</sub>N (13.6 mL, 98.0 mmol) in anhydrous THF (250 mL) cooled in an ice bath was added a solution of 2.0 M <i>tert</i>-butylmagnesium chloride solution in THF (148.5 mL, 297 mmol) via a dropping funnel over 1.5 h.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> After 30 min, the ice bath was removed. After 18 h at room temperature, ice was then added, followed by a precooled 4 N HCl aqueous solution to carefully adjust the pH to 7. The two layers were separated, and the aqueous layer was extracted with EtOAc (3 × 150 mL). The combined organic phase was washed with brine (100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 0 to 1% MeOH in EtOAc, which generated <b>10</b> (5.45 g, 60% yield) as a white foam. The product was consistent with that produced by established methods.<a onclick="showRef(event, 'ref11 ref39'); return false;" href="javascript:void(0);" class="ref ref11 ref39">(11,39)</a></div></div><div id="sec5_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i><sup>2</sup>-(<i>tert</i>-Butoxycarbonyl)-<i>N</i>-{(2<i>S</i>)-4-chloro-3-oxo-1-[(3<i>S</i>)-2-oxopiperidin-3-yl]butan-2-yl}-<span class="smallcaps smallerCapital">l</span>-leucinamide (<b>11</b>)</h4><div class="NLM_p">A solution of <b>9</b> (3.70 g, 11.6 mmol) in CHCl<sub>3</sub> (50 mL) was cooled using an ice–water bath; then, 4 M HCl in 1,4-dioxane (14.5 mL) was added. After 30 min, the ice bath was removed, the mixture was warmed to room temperature, and the solution was allowed to sit overnight. An off-white precipitate was then formed, and the mixture was concentrated under reduced pressure (30 °C water bath temperature), coevaporated with acetonitrile (3 × 20 mL) and DCM (1 × 20 mL), and dried for an additional 1 h under reduced pressure to afford (3<i>S</i>)-3-[(2<i>S</i>)-2-amino-4-chloro-3-oxobutyl]piperidin-2-one hydrochloride salt (2.96 g) as a cream color solid. This material was used without further purification in the next step. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.55 (br s, 3H), 7.80 (br s, 1H), 4.90 (d, <i>J</i> = 16.9 Hz, 1H), 4.74 (d, <i>J</i> = 16.9 Hz, 1H), 4.37–4.29 (m, 1H), 3.17–3.07 (m, 2H), 2.47–2.38 (m, 1H), 2.12–2.05 (m, 1H), 2.01–1.88 (m, 2H), 1.83–1.74 (m, 1H), 1.68–1.55 (m, 2H), 1.50–1.36 (m, 1H).</div><div class="NLM_p last">To a solution of (3<i>S</i>)-3-[(2<i>S</i>)-2-amino-4-chloro-3-oxobutyl]piperidin-2-one hydrochloride salt (2.96 g, maximum of 11.6 mmol) and <i>N</i>-(<i>tert</i>-butylcarbonyl)-<span class="smallcaps smallerCapital">l</span>-leucine (5) (2.95 g, 12.8 mmol) in anhydrous DMF (50 mL) cooled in an ice bath was added HATU (4.85 g, 12.8 mmol). Then, <i>N</i>-methylmorpholine (NMM) (3.83 mL, 34.8 mmol) was added dropwise. After 45 min, an ice/saturated aqueous NaHCO<sub>3</sub> mixture (1:1, 200 mL) was added, and the resulting mixture was extracted with ethyl acetate (3 × 250 mL). The combined organic layer was washed with saturated brine solution (1 × 250 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was dissolved in CHCl<sub>3</sub> (20 mL), and the product was purified by silica gel chromatography with a gradient of 0–4% MeOH in CHCl<sub>3</sub>, which generated <b>11</b> (2.70 g, 54% yield) as a white foamy solid. <i>R</i><sub>f</sub> = 0.50 (5% MeOH in EtOAc) was a single spot when visualized with 10% phosphomolybdic acid in ethanol. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.45 (d, <i>J</i> = 7.9 Hz, 1H), 7.46 (br s, 1H), 7.04 (br d, <i>J</i> = 7.2 Hz, 1H), 4.61 (d, <i>J</i> = 16.7 Hz, 1H), 4.54 (d, <i>J</i> = 16.7 Hz, 1H), 4.50–4.39 (m, 1H), 3.93–3.86 (m, 1H), 3.14–3.06 (m, 2H), 2.22–2.09 (m, 2H), 1.90–1.83 (m, 1H), 1.80–1.57 (m, 4H), 1.57–1.48 (m, 1H), 1.48–1.41 (m, 1H), 1.38 (s, 9H), 1.34–1.23 (m, 1H), 0.89 (d, <i>J</i> = 6.6 Hz, 3H), 0.85 (d, <i>J</i> = 6.6 Hz, 3H).</div></div><div id="sec5_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i><sup>2</sup>-(<i>tert</i>-Butoxycarbonyl)-<i>N</i>-{(2<i>S</i>)-4-chloro-3-oxo-1-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butan-2-yl}-<span class="smallcaps smallerCapital">l</span>-leucinamide (<b>12</b>)</h4><div class="NLM_p last">This material was prepared using procedures in refs <a onclick="showRef(event, 'ref11 ref39'); return false;" href="javascript:void(0);" class="ref ref11 ref39">(11and39)</a>.</div></div><div id="sec5_6_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-[(2<i>S</i>)-1-({(2<i>S</i>)-4-Chloro-3-oxo-1-[(3<i>S</i>)-2-oxopiperidin-3-yl]butan-2-yl}amino)-4-methyl-1-oxopentan-2-yl]-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>13</b>)</h4><div class="NLM_p">A solution of <b>11</b> (2.70 g, 6.25 mmol) in CHCl<sub>3</sub> (25 mL) was cooled using an ice–water bath; then, 4 M HCl in 1,4-dioxane (10 mL) was added. After 30 min, the ice bath was removed, and the solution was allowed to sit overnight. A gummy precipitate was then formed. The mixture was then concentrated under reduced pressure (bath temperature at 35 °C), coevaporated with acetonitrile (3 × 20 mL) and DCM (1 × 20 mL), and dried under reduced pressure for 1 h to afford <i>N</i>-{(2<i>S</i>)-4-chloro-3-oxo-1-[(3<i>S</i>)-2-oxopiperidin-3-yl]butan-2-yl}-<span class="smallcaps smallerCapital">l</span>-leucinamide hydrochloride salt (2.30 g) as an off-white color solid. This material was used without further purification in the next step. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.13 (d, <i>J</i> = 7.5 Hz, 1H), 8.33 (br d, <i>J</i> = 3.8 Hz, 3H), 7.53 (br s, 1H), 4.68 (s, 2H), 4.56 (ddd, <i>J</i> = 4.0, 7.3, 11.1 Hz, 1H), 3.86–3.76 (m, 1H), 3.76–3.57 (m, 1H), 3.18–3.05 (m, 2H), 2.29–2.20 (m, 1H), 2.14 (ddd, <i>J</i> = 4.7, 11.2, 14.1 Hz, 1H), 1.97–1.89 (m, 1H), 1.78–1.64 (m, 4H), 1.64–1.51 (m, 1H), 1.43–1.35 (m, 1H), 0.93 (d, <i>J</i> = 6.4 Hz, 3H), 0.91 (d, <i>J</i> = 6.4 Hz, 3H).</div><div class="NLM_p last">To a solution of <i>N</i>-{(2<i>S</i>)-4-chloro-3-oxo-1-[(3<i>S</i>)-2-oxopiperidin-3-yl]butan-2-yl}-<span class="smallcaps smallerCapital">l</span>-leucinamide hydrochloride salt (2.30 g, 6.24 mmol) and 4-methoxy-1<i>H</i>-indole-2-carboxylic acid (1.31 g, 6.87 mmol) in anhydrous DMF (25 mL) cooled in an ice bath was added HATU (2.61 g, 6.87 mmol). Then, NMM (2.06 mL, 18.7 mmol) was added dropwise. After 45 min, the ice/saturated aqueous NaHCO<sub>3</sub> mixture (1,1, 150 mL) was added, and the resulting mixture was extracted with EtOAc (3 × 150 mL). The combined organic layer was washed with saturated brine (1 × 150 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was dissolved in CHCl<sub>3</sub> (25 mL), and the product was purified by silica gel column chromatography with a gradient of 0 to 4% MeOH in CHCl<sub>3</sub>. The resulting solid was triturated with EtOAc (10 mL) and filtered, and the solid was washed with EtOAc (2 × 3 mL), then dried under vacuum, which generated <b>13</b> (1.71 g, 54% yield) as an off-white solid. <i>R</i><sub>f</sub> = 0.48 (5% MeOH in EtOAc) was a single spot under UV. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.57 (d, <i>J</i> = 2.1 Hz, 1H), 8.60 (d, <i>J</i> = 7.7 Hz, 1H), 8.44 (d, <i>J</i> = 7.5 Hz, 1H), 7.46 (br s, 1H), 7.37 (dd, <i>J</i> = 0.8, 2.3 Hz, 1H), 7.09 (t, <i>J</i> = 8.1 Hz, 1H), 7.02 (d, <i>J</i> = 8.3 Hz, 1H), 6.51 (d, <i>J</i> = 7.3 Hz, 1H), 4.61 (d, <i>J</i> = 16.0 Hz, 1H), 4.57 (d, <i>J</i> = 16.0 Hz, 1H), 4.53–4.42 (m, 2H), 3.89 (s, 3H), 3.13–3.04 (m, 2H), 2.22–2.13 (m, 2H), 1.89–1.80 (m, 1H), 1.78–1.66 (m, 4H), 1.60–1.47 (m, 2H), 1.39–1.29 (m, 1H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.89 (d, <i>J</i> = 6.4 Hz, 3H). LC/MS: 5.19 min retention time, 98.2% pure. ESI-MS: 505 [M + H]<sup>+</sup>.</div></div><div id="sec5_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-[(2<i>S</i>)-1-({(2<i>S</i>)-4-chloro-3-oxo-1-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butan-2-yl}amino)-4-methyl-1-oxopentan-2-yl]-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>14</b>)</h4><div class="NLM_p last">This material was prepared using the procedures in refs <a onclick="showRef(event, 'ref11 ref39'); return false;" href="javascript:void(0);" class="ref ref11 ref39">(11and39)</a>.</div></div><div id="sec5_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 2,2-Dimethylpropanoate (<b>15a</b>)</h4><div class="NLM_p last">A mixture of 2,2-dimethylpropanoic acid (<b>10</b>) (48.5 mg, 0.475 mmol) and sodium <i>tert</i>-butoxide (22.8 mg, 0.238 mmol) in anhydrous DMF (3 mL) was stirred at room temperature for 30 min. To this, <b>13</b> (40.0 mg, 0.079 mmol) and NaI (23.8 mg, 0.158 mmol) were added. After 24 h, a saturated brine solution (10 mL) was added, and the mixture extracted with CHCl<sub>3</sub> (3 × 10 mL). The combined organic layer was washed with saturated brine solution (10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated reduced pressure. The residue was dissolved in CHCl<sub>3</sub> (5 mL), and the product was purified by silica gel column chromatography with a gradient of 0 to 1% MeOH in CHCl<sub>3</sub>, which afforded <b>15a</b> (14.9 mg, 33% yield) as an off-white solid. <i>R</i><sub>f</sub> = 0.33 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.3 Hz, 1H), 8.57 (d, <i>J</i> = 8.1 Hz, 1H), 8.44 (d, <i>J</i> = 7.7 Hz, 1H), 7.46 (br s, 1H), 7.37 (dd, <i>J</i> = 0.8, 2.3 Hz, 1H), 7.11 (t, <i>J</i> = 8.0 Hz, 1H), 7.01 (d, <i>J</i> = 8.3 Hz, 1H), 6.51 (d, <i>J</i> = 7.5 Hz, 1H), 4.87 (d, <i>J</i> = 17.1 Hz, 1H), 4.84 (d, <i>J</i> = 17.1 Hz, 1H), 4.51–4.42 (m, 2H), 3.89 (s, 3H), 3.14–3.04 (m, 2H), 2.24–2.15 (m, 2H), 1.88–1.80 (m, 1H), 1.77–1.64 (m, 4H), 1.60–1.47 (m, 2H), 1.38–1.27 (m, 1H), 1.18 (s, 9H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.90 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 203.0, 176.8, 172.9, 172.6, 161.2, 153.6, 137.8, 129.8, 124.4, 118.0, 105.4, 101.2, 99.2, 66.3, 55.1, 53.2, 51.6, 41.2, 40.0, 36.8, 31.2, 30.4, 26.9 (3C), 25.6, 24.4, 23.0, 21.3, 21.2. LC/MS: 6.68 min retention time, 98.2% pure. Anal. Calcd for C<sub>30</sub>H<sub>42</sub>N<sub>4</sub>O<sub>7</sub>·0.2H<sub>2</sub>O: C, 62.74; H, 7.44; N, 9.76. Found: C, 62.67; H, 7.46; N, 9.44. ESI-MS: 571 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>30</sub>H<sub>42</sub>N<sub>4</sub>O<sub>7</sub> + H 571.3132, found 571.3153.</div></div><div id="sec5_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 2-Cyano-2-methylpropanoate (<b>15b</b>)</h4><div class="NLM_p last">The procedure for <b>15a</b> was followed with <b>13</b> (40.0 mg, 0.0792 mmol), 2-cyano-2-methylpropanoic acid (60.7 mg, 0.537 mmol), sodium <i>tert</i>-butoxide (36.1 mg, 0.376 mmol), potassium iodide (47.4 mg, 0.286 mmol), and DMF (3 mL). The product was purified by column chromatography on silica gel (eluted with a gradient of 0–12% methanol in 1:1 chloroform/ethyl acetate mixture), followed by trituration with ether, which provided <b>15b</b> (25.9 mg, 56% yield) as a white powder. <i>R</i><sub>f</sub> = 0.32 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.0 Hz, 1H), 8.68–8.58 (m, 1H), 8.45 (d, <i>J</i> = 7.6 Hz, 1H), 7.48–7.41 (m, 1H), 7.37 (dd, <i>J</i> = 0.7, 2.1 Hz, 1H), 7.10 (t, <i>J</i> = 8.1 Hz, 1H), 7.01 (d, <i>J</i> = 8.2 Hz, 1H), 6.51 (d, <i>J</i> = 7.6 Hz, 1H),), 5.07 (d, <i>J</i> = 17.0 Hz, 1H), 5.01 (d, <i>J</i> = 17.0 Hz, 1H), 4.51–4.45 (m, 2H), 3.89 (s, 3H), 3.13–3.05 (m, 2H), 2.24–2.16 (m, 2H), 1.87–1.81 (m, 1H), 1.77–1.66 (m, 4H), 1.603 (s, 3H), 1.600 (s, 3H), 1.59–1.48 (m, 2H), 1.37–1.30 (m, 1H), 0.95 (d, <i>J</i> = 6.2 Hz, 3H), 0.90 (d, <i>J</i> = 6.2 Hz, 3H). LC/MS: 5.51 min retention time, 96.1% pure. ESI-MS 582 [M + H]<sup>+</sup>, 580 [M – H]<sup>−</sup>. Anal. Calcd for C<sub>30</sub>H<sub>39</sub>N<sub>5</sub>O<sub>7</sub>·0.25H<sub>2</sub>O: C, 61.74; H, 6.79; N, 11.95. Found: C, 61.49; H, 6.75; N, 11.78. HRMS (ESI+) calcd for C<sub>30</sub>H<sub>39</sub>N<sub>5</sub>O<sub>7</sub> + H 582.2928, found 582.2949.</div></div><div id="sec5_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (3<i>S</i>)-3-{[<i>N</i>-(4-methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 3,3,3-Trifluoro-2,2-dimethylpropanoate (<b>15c</b>)</h4><div class="NLM_p last">The procedure for <b>15a</b> was followed with <b>13</b> (39.8 mg, 0.0789 mmol), 3,3,3-trifluoro-2,2-dimethylpropanoic acid (91.2 mg, 0.584 mmol), sodium <i>tert</i>-butoxide (42.2 mg, 0.439 mmol), potassium iodide (22.9 mg, 0.138 mmol), and DMF (3 mL). The product was purified by silica gel column chromatography (eluted with a gradient of 0–12% methanol in a 1:1 chloroform/ethyl acetate mixture) followed by trituration with ether, which provided <b>15c</b> (27.0 mg, 55% yield) as an off-white powder. <i>R</i><sub>f</sub> = 0.34 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.0 Hz, 1H), 8.59 (d, <i>J</i> = 8.1 Hz, 1H), 8.45 (d, <i>J</i> = 7.7 Hz, 1H), 7.46 (brs, 1H), 7.37 (dd, <i>J</i> = 0.7, 2.0 Hz, 1H), 7.10 (dd, <i>J</i> = 7.5, 8.2 Hz, 1H), 7.01 (d, <i>J</i> = 8.2 Hz, 1H), 6.51 (d, <i>J</i> = 7.6 Hz, 1H), 5.00 (d, <i>J</i> = 17.0 Hz, 1H), 4.97 (d, <i>J</i> = 17.0 Hz, 1H), 4.50–4.44 (m, 2H), 3.89 (s, 3H), 3.12–3.05 (m, 2H), 2.24–2.16 (m, 2H), 1.87–1.81 (m, 1H), 1.76–1.66 (m, 4H), 1.59–1.48 (m, 2H), 1.42 (s, 6H), 1.37–1.30 (m, 1H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.90 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 202.1, 173.0, 172.7, 169.0, 161.2, 153.7, 137.9, 129.9, 126.4 (q, <i>J</i> = 281.9 Hz, CF<sub>3</sub>), 124.5, 118.1, 105.4, 101.2, 99.3, 67.5, 55.1, 53.2, 47.8 (q, <i>J</i> = 23.3 Hz), 51.7, 41.2, 40.1, 36.8, 31.1, 25.6, 24.5, 23.1, 21.3, 21.2, 19.3 (br., 2C). <sup>19</sup>F NMR (565 MHz, DMSO-<i>d</i><sub>6</sub>) δ −73.76 (s, 3F). LC/MS: 6.48 min retention time, 98.3% pure. ESI-MS 625 [M + H]<sup>+</sup>, 623 [M – H]<sup>−</sup>. HRMS (ESI+) calcd for C<sub>30</sub>H<sub>39</sub>F<sub>3</sub>N<sub>4</sub>O<sub>7</sub> + H 625.2849, found 625.2861.</div></div><div id="sec5_6_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 2-Methyl-1,3-dioxane-2-carboxylate (<b>15d</b>)</h4><div class="NLM_p last">A mixture of 2-methyl-1,3-dioxane-2-carboxylic acid (69.5 mg, 0.475 mmol) (prepared following the procedure in ref <a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a>) and sodium <i>tert</i>-butoxide (22.8 mg, 0.238 mmol) in anhydrous DMF (3 mL) was stirred at room temperature for 30 min. To this, <b>13</b> (40.1 mg, 0.079 mmol) and NaI (23.8 mg, 0.158 mmol) were added successively. After 24 h, saturated brine (10 mL) was added, and the resulting mixture was extracted with CHCl<sub>3</sub> (3 × 10 mL). The combined organic layer was washed with saturated brine solution (1 × 10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was dissolved in CHCl<sub>3</sub> (5 mL), and the product was purified by silica gel column chromatography with a gradient of 0–4% MeOH in DCM, which afforded <b>15d</b> (24.9 mg, 51% yield) as an off-white solid. <i>R</i><sub>f</sub> = 0.34 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.1 Hz, 1H), 8.60 (d, <i>J</i> = 8.1 Hz, 1H), 8.45 (d, <i>J</i> = 7.5 Hz, 1H), 7.46 (br s, 1H), 7.38 (dd, <i>J</i> = 0.8, 2.3 Hz, 1H), 7.11 (t, <i>J</i> = 8.1 Hz, 1H), 7.01 (d, <i>J</i> = 8.3 Hz, 1H), 6.51 (d, <i>J</i> = 7.3 Hz, 1H), 5.05 (d, <i>J</i> = 17.1 Hz, 1H), 5.01 (d, <i>J</i> = 17.1 Hz, 1H), 4.54–4.45 (m, 2H), 3.89 (s, 3H), 3.88–3.79 (m, 4H), 3.14–3.05 (m, 2H), 2.26–2.17 (m, 2H), 1.94–1.81 (m, 2H), 1.79–1.66 (m, 4H), 1.62–1.46 (m, 2H), 1.38 (s, 3H), 1.36–1.19 (m, 2H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.90 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 202.8, 173.0, 172.6, 169.5, 161.2, 153.6, 137.8, 129.8, 124.4, 118.1, 105.4, 101.2, 99.2, 97.8, 67.2, 62.3 (2C), 55.1, 53.3, 51.7, 41.2, 40.1, 36.8, 31.2, 26.2, 25.6, 24.4, 24.1, 23.0, 21.3, 21.2. LC/MS: 6.69 min retention time, 99.1% pure. ESI-MS: 615 [M + H]<sup>+</sup>. Anal. Calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>9</sub>·0.1H<sub>2</sub>O: C, 60.40; H, 6.90; N, 9.09. Found: C, 60.45; H, 6.92; N, 8.94. HRMS (ESI+) calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>9</sub> + H 615.3030, found 615.3049.</div></div><div id="sec5_6_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 1-(Trifluoromethyl)cyclohexane-1-carboxylate (<b>15e</b>)</h4><div class="NLM_p last">The procedure for <b>15a</b> was followed with <b>13</b> (50.1 mg, 0.0992 mmol), 1-(trifluoromethyl)cyclohexanecarboxylic acid (105.1 mg, 0.536 mmol), sodium <i>tert</i>-butoxide (44.3 mg, 0.461 mmol), potassium iodide (102 mg, 0.614 mmol), and DMF (3 mL). The product was purified by column chromatography (eluted with a gradient of 0–12% methanol in a 1:1 chloroform/ethyl acetate mixture) followed by trituration with ether, which provided <b>15e</b> (9.6 mg, 15% yield) as a yellow powder. <i>R</i><sub>f</sub> = 0.46 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.56 (br s, 1H), 8.65–8.30 (m, 2H), 7.48–7.32 (m, 2H), 7.10 (dd, <i>J</i> = 7.8, 8.0 Hz, 1H), 7.01 (d, <i>J</i> = 8.1 Hz, 1H), 6.51 (d, <i>J</i> = 7.6 Hz, 1H), 5.10–4.98 (m, 2H), 4.63–4.44 (m, 2H), 3.89 (s, 3H), 3.13–3.04 (m, 2H), 2.25–2.12 (m, 4H), 1.87–1.15 (m, 16H), 0.94 (d, <i>J</i> = 6.0 Hz, 3H), 0.89 (d, <i>J</i> = 6.0 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 202.1, 173.3, 172.8, 167.5, 161.3, 153.7, 137.9, 129.9, 125.8 (q, <i>J</i> = 283.3 Hz, CF<sub>3</sub>), 124.5, 118.1, 105.5, 101.3, 99.3, 67.9, 55.1, 53.3, 52.5 (q, <i>J</i> = 23.3 Hz), 51.7, 41.3, 40.1, 36.9, 31.2, 27.11, 27.06, 25.6, 24.5, 24.2, 23.1, 21.46, 21.44, 21.37, 21.2. <sup>19</sup>F NMR (565 MHz, DMSO-<i>d</i><sub>6</sub>) δ −73.56 (s, 3F). LC/MS: 7.04 min retention time, 97.5% pure. ESI-MS 665 [M + H]<sup>+</sup>, 663 [M – H]<sup>−</sup>. Anal. Calcd for C<sub>33</sub>H<sub>43</sub>F<sub>3</sub>N<sub>4</sub>O<sub>7</sub>·0.2H<sub>2</sub>O: C, 59.31; H, 6.55; N, 8.38. Found: C, 59.39; H, 6.53; N, 8.28. HRMS (ESI+) calcd for C<sub>33</sub>H<sub>43</sub>F<sub>3</sub>N<sub>4</sub>O<sub>7</sub> + H 665.3162, found 665.3197.</div></div><div id="sec5_6_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl Benzoate (<b>15f</b>)</h4><div class="NLM_p last">To a solution of benzoic acid (37.1 mg, 0.303 mmol) in DMF (3 mL) was added sodium <i>tert</i>-butoxide (19.2 mg, 0.199 mmol). After 15 min, a solution of <b>13</b> (50.1 mg, 0.099 mmol) in DMF (2 mL) was added followed by NaI (14.9 mg, 0.099 mmol), and the solution was allowed to sit overnight. Water (5 mL) was then added, and the resulting mixture was extracted with DCM (3 × 25 mL). The combined organic phase was washed with brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 0–10% MeOH in EtOAc, which generated <b>15f</b> (15 mg, 25% yield) as a white solid. <i>R</i><sub>f</sub> = 0.31 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.3 Hz, 1H), 8.64 (d, <i>J</i> = 7.9 Hz, 1H), 8.45 (d, <i>J</i> = 7.5 Hz, 1H), 8.01–7.96 (m, 2H), 7.71–7.68 (m, 1H), 7.59–7.52 (m, 2H), 7.47 (br s, 1H), 7.39–7.36 (m, 1H), 7.12–7.07 (m, 1H), 7.03–6.99 (m, 1H), 6.51 (d, <i>J</i> = 7.5 Hz, 1H), 5.16–5.11 (m, 2H), 4.56–4.46 (m, 2H), 3.88 (s, 3H), 3.13–3.06 (m, 2H), 2.27–2.19 (m, 2H), 1.88–1.82 (m, 1H), 1.79–1.66 (m, 4H), 1.62–1.48 (m, 2H), 1.38–1.32 (m, 1H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.90 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 202.9, 173.0, 172.6, 165.0, 161.2, 153.6, 137.8, 133.7, 129.9, 129.3 (2C), 129.0, 128.9 (2C), 124.4, 118.1, 105.4, 101.2, 99.2, 67.1, 55.1, 53.3, 51.7, 41.2, 40.1, 36.9, 31.2, 25.6, 24.4, 23.0, 21.4, 21.2. LC/MS: 6.19 min retention time, 98.3% pure. ESI-MS 591 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>32</sub>H<sub>38</sub>N<sub>4</sub>O<sub>7</sub> + H 591.2819, found 591.2825.</div></div><div id="sec5_6_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 2,6-Dichlorobenzoate (<b>15g</b>)</h4><div class="NLM_p last">After 30 min, to a solution of 2,6-dichlorobenzoic acid (101 mg, 0.529 mmol) and sodium <i>tert</i>-butoxide (29.7 mg, 0.309 mmol) in DMF (3 mL) was added <b>13</b> (49.8 mg, 0.0986 mmol) followed by potassium iodide (41.6 mg, 0.251 mmol), and the solution was allowed to sit overnight. The mixture was then diluted with DCM (15 mL) and washed with water (15 mL). The organic layer was poured into a Petri dish and left in the fume hood to dry. The residue was dissolved in a minimal amount of ethyl acetate and loaded on silica. Purification of the product by silica gel column chromatography (eluted with a gradient of 0–12% methanol in a 1:1 chloroform/ethyl acetate mixture) followed by recrystallization from ethyl acetate provided <b>15g</b> (17.8 mg, 27% yield) as a white powder. <i>R</i><sub>f</sub> = 0.46 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.55 (d, <i>J</i> = 2.1 Hz, 1H), 8.62 (d, <i>J</i> = 7.6 Hz, 1H), 8.45 (d, <i>J</i> = 7.6 Hz, 1H), 7.62–7.59 (m, 2H), 7.57–7.54 (m, 1H), 7.47–7.44 (m, 1H), 7.36 (dd, <i>J</i> = 0.9, 2.3 Hz, 1H), 7.10 (t, <i>J</i> = 8.0 Hz, 1H), 7.01 (d, <i>J</i> = 8.3 Hz, 1H), 6.51 (d, <i>J</i> = 8.0 Hz, 1H), 5.19 (d, <i>J</i> = 17.0 Hz, 1H), 5.16 (d, <i>J</i> = 17.0 Hz, 1H), 4.58–4.48 (m, 2H), 3.88 (s, 3H), 3.12–3.04 (m, 2H), 2.25–2.20 (m, 2H), 1.87–1.66 (m, 4H), 1.61–1.47 (m, 2H), 1.38–1.31 (m, 2H), 0.95 (d, <i>J</i> = 6.3 Hz, 3H), 0.90 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 201.4, 173.1, 172.8, 163.4, 161.2, 153.7, 137.9, 132.7, 132.1, 130.9 (2C), 129.9, 128.6 (2C), 124.5, 118.1, 105.5, 101.3, 99.3, 67.9, 55.1, 53.3, 51.7, 41.3, 40.1, 36.9, 31.1, 25.6, 24.5, 23.0, 21.5, 21.2. LC/MS: 6.33 min retention time, 96.8% pure. ESI-MS 659 [M + H]<sup>+</sup>, 657 [M – H]<sup>−</sup>. HRMS (ESI+) calcd for C<sub>32</sub>H<sub>36</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>7</sub> + H 659.2039, found 659.2070.</div></div><div id="sec5_6_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 2,6-Bis(trifluoromethyl)benzoate (<b>15h</b>)</h4><div class="NLM_p last">A mixture of 2,6-bis(trifluoromethyl)benzoic acid (122.7 mg, 0.475 mmol) and sodium <i>tert</i>-butoxide (22.8 mg, 0.238 mmol) in anhydrous DMF (3 mL) was stirred at room temperature for 30 min. To this, <b>13</b> (40.0 mg, 0.079 mmol) and NaI (23.8 mg, 0.158 mmol) were added successively. After 48 h, a saturated brine solution (10 mL) was added, and the mixture was extracted with CHCl<sub>3</sub> (3 × 10 mL). The combined organic layer was washed with saturated brine solution (10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was dissolved in CHCl<sub>3</sub> (5 mL) and loaded on a silica gel column, and the product was purified with a gradient of 0–2% MeOH in CHCl<sub>3</sub>, which afforded <b>15h</b> (31.9 mg, 55% yield) as an off-white solid. <i>R</i><sub>f</sub> = 0.40 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.55 (d, <i>J</i> = 2.1 Hz, 1H), 8.65 (d, <i>J</i> = 7.9 Hz, 1H), 8.45 (d, <i>J</i> = 7.7 Hz, 1H), 8.26 (d, <i>J</i> = 7.9 Hz, 2H), 8.02 (t, <i>J</i> = 8.0 Hz, 1H), 7.46 (br s, 1H), 7.37 (dd, <i>J</i> = 0.7, 2.2 Hz, 1H), 7.11 (t, J = 8.1 Hz, 1H), 7.02 (d, <i>J</i> = 8.2 Hz, 1H), 6.51 (d, <i>J</i> = 7.5 Hz, 1H), 5.21 (d, <i>J</i> = 17.1 Hz, 1H), 5.16 (d, <i>J</i> = 17.1 Hz, 1H), 4.60–4.49 (m, 2H), 3.89 (s, 3H), 3.15–3.03 (m, 2H), 2.28–2.19 (m, 2H), 1.87–1.79 (m, 1H), 1.77–1.63 (m, 4H), 1.62–1.55 (m, 1H), 1.55–1.46 (m, 1H), 1.37–1.28 (m, 1H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.90 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 200.8, 172.94, 172.6, 163.9, 161.1, 153.6, 137.8, 132.2, 131.0 (q, <i>J</i> = 4.2 Hz, 2C), 129.8, 128.7 (q, <i>J</i> = 2.1 Hz), 127.4 (q, <i>J</i> = 32.2 Hz, 2C), 124.4, 122.7 (q, <i>J</i> = 274.3 Hz, 2C), 118.1, 105.4, 101.2, 99.2, 68.2, 55.1, 53.1, 51.4, 41.2, 40.1, 36.8, 30.9, 25.6, 24.4, 22.9, 21.4, 21.1. <sup>19</sup>F NMR (565 MHz, DMSO-<i>d</i><sub>6</sub>) δ −58.28 to −58.36 (m, 6F). LC/MS: 6.49 min retention time, 97.1% pure. ESI-MS: 727 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>34</sub>H<sub>36</sub>F<sub>6</sub>N<sub>4</sub>O<sub>7</sub> + H 727.2567, found 727.2592.</div></div><div id="sec5_6_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butylpyridine-2-carboxylate (<b>15i</b>)</h4><div class="NLM_p last">To a solution of picolinic acid (41.1 mg, 0.333 mmol) in DMF (3 mL) was added sodium <i>tert</i>-butoxide (21.2 mg, 0.221 mmol). After 15 min, a solution of <b>13</b> (54.9 mg, 0.108 mmol) in DMF (2 mL) was added followed by the addition of NaI (16.0 mg, 0.107 mmol). The reaction mixture was stirred at room temperature overnight. Then, it was quenched by adding water (5 mL). The mixture was extracted with DCM (3 × 25 mL), and the combined organic phase was washed with brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The product was purified by column chromatography on silica gel with a gradient of 0–10% MeOH in EtOAc, which generated <b>15i</b> (12 mg, 20% yield) as a white solid. <i>R</i><sub>f</sub> = 0.21 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.59 (d, <i>J</i> = 1.9 Hz, 1H), 8.77–8.73 (m, 1H), 8.65 (d, <i>J</i> = 8.3 Hz, 1H), 8.46 (d, <i>J</i> = 7.5 Hz, 1H), 8.11–8.07 (m, 1H), 8.05–8.00 (m, 1H), 7.69 (ddd, <i>J</i> = 1.1, 4.7, 7.7 Hz, 1H), 7.47 (br s, 1H), 7.39–7.37 (m, 1H), 7.12–7.07 (m, 1H), 7.03–6.99 (m, 1H), 6.51 (d, <i>J</i> = 7.5 Hz, 1H), 5.22–5.13 (m, 2H), 4.58–4.53 (m, 1H), 4.52–4.46 (m, 1H), 3.89 (s, 3H), 3.13–3.06 (m, 2H), 2.28–2.21 (m, 2H), 1.89–1.83 (m, 1H), 1.78–1.68 (m, 4H), 1.62–1.48 (m, 2H), 1.40–1.33 (m, 1H), 0.96 (d, <i>J</i> = 6.4 Hz, 3H), 0.91 (d, <i>J</i> = 6.4 Hz, 3H). LC/MS: 5.19 min retention time, 96.2% pure. ESI-MS 592 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>31</sub>H<sub>37</sub>N<sub>5</sub>O<sub>7</sub> + H 592.2771, found 592.2782.</div></div><div id="sec5_6_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butylpyridine-3-carboxylate (<b>15j</b>)</h4><div class="NLM_p last">To a solution of nicotinic acid (37.0 mg, 0.301 mmol) in DMF (3 mL) was added sodium <i>tert</i>-butoxide (19.2 mg, 0.200 mmol). After 15 min, a solution of <b>13</b> (50.3 mg, 0.100 mmol) in DMF (2 mL) was added followed by the addition of NaI (15 mg, 0.10 mmol), and the solution was allowed to sit overnight. Water (5 mL) was then added, and the resulting mixture was extracted with DCM (3 × 25 mL). The combined organic layer was washed with brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 0–10% MeOH in EtOAc, which generated <b>15j</b> (28 mg, 47% yield) as an off-white solid. <i>R</i><sub>f</sub> = 0.25 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.2 Hz, 1H), 9.12 (dd, <i>J</i> = 0.8, 2.3 Hz, 1H), 8.86 (dd, <i>J</i> = 1.5, 4.9 Hz, 1H), 8.65 (d, <i>J</i> = 8.3 Hz, 1H), 8.46 (d, <i>J</i> = 7.5 Hz, 1H), 8.34–8.31 (m, 1H), 7.61 (ddd, <i>J</i> = 1.1, 4.8, 8.0 Hz, 1H), 7.48 (br s, 1H), 7.40–7.37 (m, 1H), 7.13–7.08 (m, 1H), 7.03–6.99 (m, 1H), 6.51 (d, <i>J</i> = 7.5 Hz, 1H), 5.21–5.13 (m, 2H), 4.59–4.52 (m, 1H), 4.52–4.47 (m, 1H), 3.89 (s, 3H), 3.13–3.06 (m, 2H), 2.27–2.21 (m, 2H), 1.89–1.83 (m, 1H), 1.78–1.67 (m, 4H), 1.61–1.48 (m, 2H), 1.39–1.31 (m, 1H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.90 (d, <i>J</i> = 6.8 Hz, 3H). LC/MS: 5.31 min retention time, 95.9% pure. ESI-MS 592 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>31</sub>H<sub>37</sub>N<sub>5</sub>O<sub>7</sub> + H 592.2771, found 592.2795.</div></div><div id="sec5_6_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butylpyridine-4-carboxylate (<b>15k</b>)</h4><div class="NLM_p last">To a solution of isonicotinic acid (37.1 mg, 0.301 mmol) in DMF (5 mL) was added sodium <i>tert</i>-butoxide (19.2 mg, 0.200 mmol). After 15 min, <b>13</b> (50.0 mg, 0.099 mmol) was added followed by KI (16.6 mg, 0.100 mmol). After 2 days, water (5 mL) was added, and the resulting mixture was extracted with EtOAc (3 × 20 mL). The combined organic phase was washed with brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 0–10% MeOH in EtOAc, which generated <b>15k</b> (31 mg, 52% yield) as a white solid. <i>R</i><sub>f</sub> = 0.32 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.3 Hz, 1H), 8.88–8.81 (m, 2H), 8.65 (d, <i>J</i> = 7.9 Hz, 1H), 8.46 (d, <i>J</i> = 7.9 Hz, 1H), 7.89–7.84 (m, 2H), 7.48 (br s, 1H), 7.39–7.37 (m, 1H), 7.13–7.08 (m, 1H), 7.04–6.99 (m, 1H), 6.52 (d, <i>J</i> = 7.5 Hz, 1H), 5.23–5.15 (m, 2H), 4.58–4.52 (m, 1H), 4.52–4.46 (m, 1H), 3.89 (s, 3H), 3.14–3.06 (m, 2H), 2.27–2.20 (m, 2H), 1.89–1.83 (m, 1H), 1.80–1.64 (m, 4H), 1.61–1.50 (m, 2H), 1.39–1.31 (m, 1H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.90 (d, <i>J</i> = 6.4 Hz, 3H). LC/MS: 5.36 min retention time, 95.4% pure. ESI-MS 592 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>31</sub>H<sub>37</sub>N<sub>5</sub>O<sub>7</sub> + H 592.2771, found 592.2797.</div></div><div id="sec5_6_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 2,4,6-Trimethylpyridine-3-carboxylate (<b>15l</b>)</h4><div class="NLM_p last">To a solution of 2,4,6-trimethylpyridine-3-carboxylic acid hydrochloride (81.3 mg, 0.403 mmol) in DMF (5 mL) was added sodium <i>tert</i>-butoxide (58.0 mg, 0.603 mmol). After 30 min, <b>13</b> (50.8 mg, 0.101 mmol) and then KI (16.6 mg, 0.100 mmol) was added. After 2 days, EtOAc (40 mL) was added, and the mixture was washed with water (3 × 10 mL) and brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 0–10% MeOH in CHCl<sub>3</sub>, which generated <b>15l</b> (38 mg, 60% yield) as a white solid. <i>R</i><sub>f</sub> = 0.31 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.57 (d, <i>J</i> = 2.3 Hz, 1H), 8.63 (d, <i>J</i> = 8.3 Hz, 1H), 8.45 (d, <i>J</i> = 7.9 Hz, 1H), 7.46 (br s, 1H), 7.39–7.36 (m, 1H), 7.12–7.08 (m, 1H), 7.05–7.00 (m, 2H), 6.51 (d, <i>J</i> = 7.2 Hz, 1H), 5.18–5.09 (m, 2H), 4.57–4.49 (m, 2H), 3.89 (s, 3H), 3.13–3.06 (m, 2H), 2.46 (s, 3H), 2.41 (s, 3H), 2.30 (s, 3H), 2.28–2.21 (m, 2H), 1.89–1.83 (m, 1H), 1.78–1.68 (m, 4H), 1.63–1.49 (m, 2H), 1.39–1.32 (m, 1H), 0.96 (d, <i>J</i> = 6.4 Hz, 3H), 0.91 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 202.8, 173.0, 172.6, 167.4, 161.2, 158.4, 154.2, 153.6, 145.2, 137.8, 129.8, 125.3, 124.4, 122.0, 118.1, 105.4, 101.2, 99.2, 67.3, 55.1, 53.3, 51.7, 41.2, 40.0, 36.8, 31.2, 25.6, 24.4, 23.8, 23.0, 22.7, 21.4, 21.2, 19.0. LC/MS: 4.39 min retention time, 99.5% pure. ESI-MS 634 [M + H]<sup>+</sup>. Anal. Calcd for C<sub>34</sub>H<sub>43</sub>N<sub>5</sub>O<sub>7</sub>·0.35H<sub>2</sub>O: C, 63.80; H, 6.88; N, 10.94. Found: C, 63.85; H, 6.93; N, 10.85. HRMS (ESI+) calcd for C<sub>34</sub>H<sub>43</sub>N<sub>5</sub>O<sub>7</sub> + H 634.3241, found 634.3239.</div></div><div id="sec5_6_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 2,6-Dimethyl-4-(trifluoromethyl)pyridine-3-carboxylate (<b>15m</b>)</h4><div class="NLM_p last">To a solution of 2,6-dimethyl-4-(trifluoromethyl)pyridine-3-carboxylic acid hydrochloride (102 mg, 0.400 mmol) (prepared using the procedures in refs <a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58and59)</a>) in DMF (5 mL) was added sodium <i>tert</i>-butoxide (58.1 mg, 0.605 mmol). After 30 min, <b>13</b> (50.1 mg, 0.099 mmol) followed by KI (16.6 mg, 0.100 mmol) was added. After 3 days, EtOAc (40 mL) was added, and the mixture was washed with water (3 × 10 mL) and brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 0–10% MeOH in CHCl<sub>3</sub>, which generated <b>15m</b> (23 mg, 33% yield) as a white solid. <i>R</i><sub>f</sub> = 0.36 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.56 (d, <i>J</i> = 2.3 Hz, 1H), 8.64 (d, <i>J</i> = 8.3 Hz, 1H), 8.45 (d, <i>J</i> = 7.5 Hz, 1H), 7.63 (s, 1H), 7.47 (br s, 1H), 7.38–7.36 (m, 1H), 7.12–7.08 (m, 1H), 7.03–6.99 (m, 1H), 6.51 (d, <i>J</i> = 7.2 Hz, 1H), 5.23–5.13 (m, 2H), 4.60–4.48 (m, 2H), 3.89 (s, 3H), 3.13–3.05 (m, 2H), 2.63 (s, 3H), 2.59 (s, 3H), 2.29–2.20 (m, 2H), 1.89–1.82 (m, 1H), 1.79–1.66 (m, 4H), 1.63–1.48 (m, 2H), 1.40–1.32 (m, 1H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.90 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 202.1, 173.0, 172.6, 165.5, 161.2, 161.1, 156.0, 153.6, 137.8, 134.2 (q, <i>J</i> = 32.6 Hz), 129.8, 124.40, 122.4 (q, <i>J</i> = 274.7 Hz), 121.4, 118.1, 116.8 (q, <i>J</i> = 4.5 Hz), 105.4, 101.2, 99.2, 67.9, 55.1, 53.2, 51.6, 41.2, 40.0, 36.8, 31.1, 25.6, 24.4, 24.0, 23.0, 22.4, 21.4, 21.2. <sup>19</sup>F NMR (565 MHz, DMSO-<i>d</i><sub>6</sub>) δ −60.74 (s, 3F). LC/MS: 6.37 min retention time, 96.9% pure. ESI-MS 688 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>34</sub>H<sub>40</sub>F<sub>3</sub>N<sub>5</sub>O<sub>7</sub> + H 688.2958, found 688.2975.</div></div><div id="sec5_6_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 2,4,6-Trimethylpyrimidine-5-carboxylate (<b>15n</b>)</h4><div class="NLM_p last">To 2,4,6-trimethylpyrimidine-5-carboxylic acid (82.3 mg, 0.495 mmol) in anhydrous DMF (3 mL) was added sodium <i>tert</i>-butoxide (23.8 mg, 0.247 mmol). After 30 min, <b>13</b> (50.1 mg, 0.099 mmol) and NaI (29.7 mg, 0.198 mmol) were added, and the solution was allowed to sit overnight. Saturated brine (10 mL) was then added, and the resulting mixture was extracted with CHCl<sub>3</sub> (3 × 10 mL). The combined organic layer was washed with saturated brine solution (1 × 10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was dissolved in CHCl<sub>3</sub> (5 mL) and loaded on a silica gel column, and the product was purified by silica gel column chromatography with a gradient of 0–6% MeOH in CHCl<sub>3</sub>, which afforded <b>15n</b> (38.1 mg, 61% yield) as an off-white solid. <i>R</i><sub>f</sub> = 0.21 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.57 (d, <i>J</i> = 2.3 Hz, 1H), 8.64 (d, <i>J</i> = 8.1 Hz, 1H), 8.46 (d, <i>J</i> = 7.5 Hz, 1H), 7.48–7.45 (m, 1H), 7.38 (dd, <i>J</i> = 0.8, 2.3 Hz, 1H), 7.12–7.08 (m, 1H), 7.01 (d, <i>J</i> = 8.1 Hz, 1H), 6.51 (d, <i>J</i> = 7.3 Hz, 1H), 5.23–5.14 (m, 2H), 4.59–4.53 (m, 1H), 4.53–4.48 (m, 1H), 3.89 (s, 3H), 3.14–3.05 (m, 2 H), 2.57 (s, 3H), 2.49 (s, 6H), 2.28–2.20 (m, 2H), 1.90–1.81 (m, 1H), 1.79–1.67 (m, 4H), 1.63–1.48 (m, 2H), 1.41–1.31 (m, 1H), 0.96 (d, <i>J</i> = 6.4 Hz, 3H), 0.91 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 202.7, 173.0, 172.6, 167.2, 166.3, 164.2, 161.2, 153.6, 137.8, 129.8, 124.4, 122.3, 118.0, 105.4, 101.2, 99.2, 67.7, 55.1, 53.3, 51.7, 41.2, 40.1, 40.0, 36.8, 31.2, 25.6, 25.5, 24.4, 23.0, 22.43 (2C), 21.37, 21.2. LC/MS: 5.71 min retention time, 98.5% pure. ESI-MS: 635 [M + H]<sup>+</sup>; Anal. Calcd for C<sub>33</sub>H<sub>43</sub>N<sub>6</sub>O<sub>7</sub>·0.35H<sub>2</sub>O: C, 61.83; H, 6.71; N, 13.11. Found: C, 61.80; H, 6.61; N, 12.92. HRMS (ESI+) calcd for C<sub>33</sub>H<sub>42</sub>N<sub>6</sub>O<sub>7</sub> + H 635.3193, found 635.3210.</div></div><div id="sec5_6_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopiperidin-3-yl]butyl 1,3,5-Trimethyl-1<i>H</i>-pyrazole-4-carboxylate (<b>15o</b>)</h4><div class="NLM_p last">To a solution of 1,3,5-trimethyl-1<i>H</i>-pyrazole-4-carboxylic acid (55.1 mg, 0.357 mmol) in DMF (1 mL) was added sodium <i>tert</i>-butoxide (17.1 mg, 0.178 mmol). After 10 min, <b>13</b> (45.0 mg, 0.0891 mmol) was added to the mixture followed by sodium iodide (26.7 mg, 0.178 mmol), and the solution was allowed to sit overnight. The mixture was then diluted with ethyl acetate/chloroform (75 mL/25 mL), and the resulting mixture was washed with water (25 mL) and brine (25 mL), dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The product was purified by column chromatography on silica with a gradient of 0 to 2% methanol in 1:1 chloroform/ethyl acetate, which generated <b>15o</b> (13.0 mg, 23% yield) as white solid. <i>R</i><sub>f</sub> = 0.25 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.57 (d, <i>J</i> = 2.1 Hz, 1H), 8.60 (d, <i>J</i> = 8.3 Hz, 1H), 8.44 (d, <i>J</i> = 8.3 Hz, 1H), 7.45 (br s, 1H), 7.37 (dd, <i>J</i> = 0.8, 2.3 Hz, 1H), 7.15–7.07 (m, 1H), 7.04–6.97 (m, 1H), 6.51 (d, <i>J</i> = 7.9 Hz, 1H), 5.09–4.94 (m, 2H), 4.56–4.44 (m, 2H), 3.89 (s, 3H), 3.69 (s, 3H), 2.44 (s, 3H), 2.26 (s, 3H), 1.92–1.78 (m, 2H), 1.78–1.65 (m, 4H), 1.62–1.46 (m, 4H), 1.43–1.25 (m, 2H), 0.95 (d, <i>J</i> = 6.0 Hz, 3H), 0.91 (d, <i>J</i> = 6.0 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 203.8, 173.4, 173.2, 163.2, 161.6, 154.1, 149.2, 144.6, 138.3, 130.3, 124.9, 118.5, 108.29, 108.27, 105.9, 101.7, 99.7, 66.2, 55.5, 53.7, 52.1, 41.7, 37.3, 36.3, 31.7, 26.1, 24.9, 23.5, 21.8, 21.6, 14.3, 11.3. LC/MS: 5.53 min retention time, 96.5% pure. ESI-MS: 623 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>32</sub>H<sub>42</sub>N<sub>6</sub>O<sub>7</sub> + H 623.3193, found 623.3232.</div></div><div id="sec5_6_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (3<i>S</i>)-3-{[<i>N</i>-(4-Methoxy-1<i>H</i>-indole-2-carbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Dimethyl-4-(trifluoromethyl)pyridine-3-carboxylate (<b>16m</b>)</h4><div class="NLM_p last">To a solution of 2,6-dimethyl-4-(trifluoromethyl)pyridine-3-carboxylic acid hydrochloride (102.4 mg, 0.401 mmol) (prepared using the procedures in refs <a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58and59)</a>) in DMF (5 mL) was added sodium <i>tert</i>-butoxide (58.5 mg, 0.609 mmol). After 30 min, <b>14</b> (50.4 mg, 0.103 mmol) followed by KI (16.6 mg, 0.100 mmol) was added. After 3 days, EtOAc (40 mL) was added, and the resulting mixture was washed with water (3 × 10 mL) and brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was purified by column chromatography on silica gel with a gradient of 0–10% MeOH in CHCl<sub>3</sub>, which generated <b>16m</b> (32 mg, 47% yield) as a white solid. <i>R</i><sub>f</sub> = 0.34 (5% MeOH in DCM). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.57 (d, <i>J</i> = 2.3 Hz, 1H), 8.65 (d, <i>J</i> = 7.9 Hz, 1H), 8.46 (d, <i>J</i> = 7.9 Hz, 1H), 7.66 (s, 1H), 7.63 (s, 1H). 7.39–7.36 (m, 1H), 7.14–7.07 (m, 1H), 7.04–6.98 (m, 1H), 6.51 (d, <i>J</i> = 7.2 Hz, 1H), 5.23–5.14 (m, 2H), 4.57–4.49 (m, 2H), 3.89 (s, 3H), 3.18–3.06 (m, 2H), 2.64 (s, 3H), 2.59 (s, 3H), 2.39–2.31 (m, 1H), 2.14–2.01 (m, 2H), 1.77–1.57 (m, 5H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 0.91 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 201.8, 178.3, 173.1, 165.5, 161.2, 161.2, 156.04, 153.6, 137.8, 134.2 (q, <i>J</i> = 33.0 Hz), 129.8, 124.4, 122.4 (q, <i>J</i> = 274.4 Hz), 121.4, 118.1, 116.8 (q, <i>J</i> = 4.1 Hz), 105.4, 101.2, 99.2, 67.8, 55.1, 54.9, 53.9, 51.6, 40.0, 37.3, 30.7, 27.1, 24.4, 24.0, 23.0, 22.4, 21.3. <sup>19</sup>F NMR (565 MHz, DMSO-<i>d</i><sub>6</sub>) δ −60.73 (s, 3F). LC/MS: 6.29 min retention time, 98.3% pure. ESI-MS 674 [M + H]<sup>+</sup>. HRMS (ESI+) calcd for C<sub>33</sub>H<sub>38</sub>F<sub>3</sub>N<sub>5</sub>O<sub>7</sub> + H 674.2802, found 674.2825.</div></div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> ADME</h3><div class="NLM_p">The ADME experiments were performed at WuXi AppTech (Shanghai) using standard, validated protocols, and the results are the mean of experiments done in at least duplicate. Positive controls were used, and they performed within the acceptable range.</div><div class="NLM_p">LogD was determined using a 100 mM phosphate buffer at pH 7.4 and 1-octanol using a shake flask method (shake 2 min and 1 h at a speed of 800 rpm at room temperature) and analysis by LC-MS/MS of the layers. The layers were diluted prior to analysis, and the LogD was equal to the log (mean octanol peak area ratio × octanol dilution factor/mean buffer peak area ratio × buffer layer dilution factor).</div><div class="NLM_p">Plasma stability was performed using pooled frozen CD-1 mouse plasma that was thawed at 37 °C prior to experiment. Compounds were tested at 2 μM concentration at 37 °C (2% final DMSO concentration). The concentration of compound relative to the internal standard was determined at time points of 0, 10, 30, 60, and 120 min after acetonitrile precipitation. The analysis was performed by LC-MS/MS relative to an internal standard after centrifuging the sample.</div><div class="NLM_p">Mouse (CD-1) Microsomal stability was performed at 37 °C with constant shaking. Incubation was done with and without the addition of NADPH with 0.5 mg protein/mL, compound concentration of 1 μM, control at 1 μM, DMSO 0.01%, and acetonitrile 0.99%. The compound concentration was determined by LC-MS/MS at 0, 5, 10, 20, 30, and 60 min.</div><div class="NLM_p">Mouse (CD-1) hepatocyte stability was performed 37 °C with constant shaking in a 95% humidified incubator with 5% CO<sub>2</sub>. Incubation was done with and without hepatocytes with 0.5 × 106 cells/mL, compound concentration of 1 μM, positive compound at 3 μM, DMSO 0.01%, and acetonitrile 0.90%. The compound concentration was determined by LC-MS/MS at 0, 15, 30, 60, and 90 min.</div><div class="NLM_p last">Caco-2 cells purchased from ATCC were seeded onto polyethylene (PET) membranes in 96-well Corning Insert plates at 1 × 10<sup>5</sup> cells/cm<sup>2</sup>, and the medium was refreshed every 4 to 5 days until the 21st to 28th day for confluent cell monolayer formation. The transport buffer was at pH 7.4, and the compounds were tested at 2.0 μM bidirectionally (DMSO concentration <1%). The plate was incubated for 2 h in a CO<sub>2</sub> incubator at 37 °C with 5% CO<sub>2</sub> at saturated humidity without shaking. The samples were mixed with acetonitrile with an internal standard, and the quantities of the compounds were determined by LC-MS/MS in starting, donor, and receiver solutions. The Caco-2 monolayer integrity was determined after each assay.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> p<i>K</i><sub>a</sub> Determination</h3><div class="NLM_p last">Nonliterature p<i>K</i><sub>a</sub> values were determined by a pH titration of 20–40 mM solution of compound in a 20:1 mixture of water/DMSO as reported by Albert and Serjeant.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The pH was measured with a Mettler Toledo SevenCompact pH/Ion S220 instrument that was calibrated with pH 1.00, 4.00, 7.00, and 10.00 standard buffers prior to each titration. The titrant (0.049 to 0.054 N NaOH) was delivered with a syringe pump at a 1 mL/min flow rate into the magnetically stirred solution. The carboxylic acids corresponding to compounds <b>15l</b>, <b>15m</b>, <b>15n</b>, and <b>15o</b> were converted into the hydrochloride salts prior to the titration to determine the p<i>K</i><sub>a</sub> of the carboxylic acid group.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Human Plasma Stability</h3><div class="NLM_p last">A test compound (1 μL of 5 mM DMSO solution) was incubated with human plasma (49 μL, Sigma-Aldrich, P9523-5 ML) at 37 °C for a period of 15, 30, 60, and/or 120 min. The final concentration of the compound was 100 μM with 2.0% DMSO. The incubation was terminated by the addition of 450 μL of acetonitrile containing reserpine (5 μM) as an internal standard. Following centrifugation at 15 000 rpm for 3 min, the concentration of the test compound along with the hydrolyzed product in the supernatant was quantified by LC/MS using the standard curve (Eluent E1 (isocratic 60% methanol–water for 6 min), Eluent E2 (isocratic 65% methanol–water for 6 min), or Eluent E3 (gradient of methanol–water, 65 to 85% in 2 min, followed by 4 min of isocratic methanol–water 85%)). The mean value of two independent experiments with two injections per sample is reported.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Human CatS Inhibition</h3><div class="NLM_p last">The commercial CatS (CTSS, EC 3.4.22.27) inhibitor fluorometric assay kit (ab185437) from Abcam (UK) and the accompanying instructions and protocols were utilized. A 96-well white plate with a flat bottom and PerkinElmer’s EnSpire multimode plate reader were used. Blank, enzyme, and inhibitor (supplied) controls were simultaneously tested for the inhibitors in triplicate. The test inhibitors were dissolved in and diluted in DMSO, and the DMSO final concentration was 1%. The compounds were screened at 1 μM, and for IC<sub>50</sub> determination, six different concentrations (4, 1, 0.25, 0.0625, 0.0156, and 0.0039 μM) were tested. Protocols from Abcam were followed, with the exception that incubation was for 30 min at room temperature instead of 15 min. All test inhibitors and controls were run in triplicate for each concentration. The slope of enzyme control (EC) and the slope of test inhibitors (S) were calculated (linear regression), and the % relative inhibition was determined, as in the kit instructions. IC<sub>50</sub> values were calculated using Prism 9 for Windows 64-bit (V 9.1.0) by GraphPad Software, and standard error is reported.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Human CatB Inhibition</h3><div class="NLM_p last">The commercial Cathepsin B (CTSB, EC 3.4.22.1) inhibitor fluorometric assay kit (K147-100) from BioVision (USA) and the accompanying instructions and protocols were utilized. The test was performed identical to CatS, other than the final concentrations and the fact that the BioVision protocols followed a 30 min incubation period instead of 15 min. For <b>15h</b> and <b>15l</b>, the final concentrations of 4, 1, 0.25, 0.0625, 0.0156, and 0.0039 μM were used, and for <b>15e</b>, 40, 10, 2.5, 0.625, 0.156, and 0.039 μM were used. All concentrations were performed in triplicate.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i55"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00616" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35007" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35007" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00616?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00616</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Glutathione stability studies (Table S1); protease, antiviral, and cytotoxicity graphs for <b>15l</b>; slopes of <i>k</i><sub>obs</sub> versus [I] for <b>15a</b>, <b>15e</b>, <b>15f</b>, <b>15h</b>, and <b>15l</b> (Table S2 and Charts S1 and S2); electron density for SARS-CoV-2 3CLP – <b>15l</b> active sites (Figure S1); SARS-CoV-2 3CLP – <b>15l</b> ligand plot (Figure S2) and crystallographic data (Table S3); and <sup>1</sup>H NMR, <sup>19</sup>F NMR, and HPLC for select compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds with biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf">jm1c00616_si_001.pdf (7.42 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_002.csv">jm1c00616_si_002.csv (2.25 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00616" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62847" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62847" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James A. Nieman</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Applied Virology Institute, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-5087-0819" title="Orcid link">https://orcid.org/0000-0002-5087-0819</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6e0400070b030f002e1b0f020c0b1c1a0f400d0f"><span class="__cf_email__" data-cfemail="cca6a2a5a9a1ada28cb9ada0aea9beb8ade2afad">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bing Bai</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Applied Virology Institute, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexandr Belovodskiy</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Applied Virology Institute, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mostofa Hena</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Applied Virology Institute, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Appan Srinivas Kandadai</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Applied Virology Institute, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael A. Joyce</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Institute of Virology, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Holly A. Saffran</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Institute of Virology, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justin A. Shields</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Institute of Virology, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Muhammad Bashir Khan</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Arutyunova</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Institute of Virology, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jimmy Lu</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Institute of Virology, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sardeev K. Bajwa</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darren Hockman</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Applied Virology Institute, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Conrad Fischer</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</span>; 
    <span>Present Address:
                        Department of Physical Sciences, Barry University, 11300 NE Second Avenue, Miami Shores, Florida 33161, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tess Lamer</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wayne Vuong</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-3439-8788" title="Orcid link">https://orcid.org/0000-0002-3439-8788</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marco J. van Belkum</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhengxian Gu</span> - <span class="hlFld-Affiliation affiliation">WuXi AppTec
(Shanghai) Co., Ltd., G Warehouse #101, No. 10 Building, #227 Meisheng Road, WaiGaoQiao
Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fusen Lin</span> - <span class="hlFld-Affiliation affiliation">WuXi AppTec
(Shanghai) Co., Ltd., G Warehouse #101, No. 10 Building, #227 Meisheng Road, WaiGaoQiao
Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanhua Du</span> - <span class="hlFld-Affiliation affiliation">WuXi AppTec
(Shanghai) Co., Ltd., G Warehouse #101, No. 10 Building, #227 Meisheng Road, WaiGaoQiao
Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jia Xu</span> - <span class="hlFld-Affiliation affiliation">WuXi AppTec
(Shanghai) Co., Ltd., G Warehouse #101, No. 10 Building, #227 Meisheng Road, WaiGaoQiao
Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohammad Rahim</span> - <span class="hlFld-Affiliation affiliation">Rane
Pharmaceuticals, Inc. 4290 91a Street NW, Edmonton, Alberta T6E 5V2, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Howard S. Young</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John C. Vederas</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2996-0326" title="Orcid link">https://orcid.org/0000-0002-2996-0326</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">D. Lorne Tyrrell</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Applied Virology Institute, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Li
Ka Shing Institute of Virology, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alberta T6G 2E1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">M. Joanne Lemieux</span> - <span class="hlFld-Affiliation affiliation">Li
Ka Shing Institute of Virology, University
of Alberta, Edmonton, Alberta T6G 2E1, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2H7, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4745-9153" title="Orcid link">https://orcid.org/0000-0003-4745-9153</a></span></div></li></ul></li><li></li><li><span class="author-information-subsection-header">Funding</span><p>Funding for the project was provided by Alberta Innovates (RES27408) and the Canadian Institutes of Health Research Rapid Research (VR3-172655).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i57">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13040" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13040" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank numerous colleagues at the University of Alberta including Vishwa Somayaji (Faculty of Pharmacy and Pharmaceutical Sciences) for running <sup>1</sup>H, <sup>13</sup>C, and <sup>19</sup>F NMR; Jack Moore (Alberta Proteomics and Mass Spectrometry Facility) for HMRS; and the Analytical and Instrumentation Laboratory (Department of Chemistry) for performing the elemental analyses. We also thank the DMPK group at WuXi AppTech for performing the ADME experiments reported in Table 3. We thank the staff at SSRL beamline 12-1, in particular Dr. Silvia Russi. Use of the Stanford Synchrotron Radiation Lightsource, SLAC National Accelerator Laboratory, is supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences under Contract No. DE-AC02-76SF00515. The SSRL Structural Molecular Biology Program is supported by the DOE Office of Biological and Environmental Research, and by the National Institutes of Health, National Institute of General Medical Sciences (P30GM133894). The contents of this publication are solely the responsibility of the authors and do not necessarily represent the official views of NIGMS or NIH.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">3CLP</td><td class="NLM_def"><p class="first last">3-chymotrypsin-like protease</p></td></tr><tr><td class="NLM_term">3CP</td><td class="NLM_def"><p class="first last">3-C protease</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, and excretion</p></td></tr><tr><td class="NLM_term">AMK</td><td class="NLM_def"><p class="first last">α-acyloxymethylketone</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyloxylcarbonyl</p></td></tr><tr><td class="NLM_term">CoV</td><td class="NLM_def"><p class="first last">coronavirus</p></td></tr><tr><td class="NLM_term">COVID-19</td><td class="NLM_def"><p class="first last">coronavirus disease 2019</p></td></tr><tr><td class="NLM_term">CPE</td><td class="NLM_def"><p class="first last">cytopathic effect</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">Et<sub>3</sub>N</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">FIP</td><td class="NLM_def"><p class="first last">feline infectious peritonitis</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino) methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">NMM</td><td class="NLM_def"><p class="first last"><i>N</i>-methylmorpholine</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">PRA</td><td class="NLM_def"><p class="first last">plaque reduction assay</p></td></tr><tr><td class="NLM_term">SARS</td><td class="NLM_def"><p class="first last">severe acute respiratory syndrome</p></td></tr><tr><td class="NLM_term">SI</td><td class="NLM_def"><p class="first last">selectivity index</p></td></tr><tr><td class="NLM_term"><i>t</i>Bu</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyl</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i59">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01841" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01841" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Baltimore, D.</span></span> <span> </span><span class="NLM_article-title">Expression of Animal Virus Genomes</span>. <i>Bacteriol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1128/br.35.3.235-241.1971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1128%2Fbr.35.3.235-241.1971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=4329869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaE38Xjt1WisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1971&pages=235-241&author=D.+Baltimore&title=Expression+of+Animal+Virus+Genomes&doi=10.1128%2Fbr.35.3.235-241.1971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of animal virus genomes</span></div><div class="casAuthors">Baltimore, David</div><div class="citationInfo"><span class="NLM_cas:title">Bacteriological Reviews</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">235-41</span>CODEN:
                <span class="NLM_cas:coden">BAREA8</span>;
        ISSN:<span class="NLM_cas:issn">0005-3678</span>.
    </div><div class="casAbstract">A review.  A comparison of gene expression in 3 viral systems indicates the diversity of behavior which exists.  The picornaviruses encode multiple protein species in a single species.  The rhabdoviruses and paramyxoviruses carry many genes in a single RNA but synthesize sep., complementary RNA species.  The RNA tumor viruses probably transfer their genetic information from RNA to DNA, where it can be expressed by DNA-dependent RNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdBXn3wvvXqbVg90H21EOLACvtfcHk0li8YQm7RcrHnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE38Xjt1WisA%253D%253D&md5=573e122f25b5127ddaa234b3822cf779</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1128%2Fbr.35.3.235-241.1971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fbr.35.3.235-241.1971%26sid%3Dliteratum%253Aachs%26aulast%3DBaltimore%26aufirst%3DD.%26atitle%3DExpression%2520of%2520Animal%2520Virus%2520Genomes%26jtitle%3DBacteriol.%2520Rev.%26date%3D1971%26volume%3D35%26spage%3D235%26epage%3D241%26doi%3D10.1128%2Fbr.35.3.235-241.1971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, J. J.</span></span> <span> </span><span class="NLM_article-title">Group IV Viruses: Single-Stranded (+)Sense RNA</span>. <i>Taxonomic Guide to Infectious Diseases</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">246</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-415895-5.00042-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2FB978-0-12-415895-5.00042-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=237-246&author=J.+J.+Berman&title=Group+IV+Viruses%3A+Single-Stranded+%28%2B%29Sense+RNA&doi=10.1016%2FB978-0-12-415895-5.00042-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-415895-5.00042-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-415895-5.00042-8%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DJ.%2BJ.%26atitle%3DGroup%2520IV%2520Viruses%253A%2520Single-Stranded%2520%2528%252B%2529Sense%2520RNA%26jtitle%3DTaxonomic%2520Guide%2520to%2520Infectious%2520Diseases%26date%3D2012%26spage%3D237%26epage%3D246%26doi%3D10.1016%2FB978-0-12-415895-5.00042-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marakovits, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejto, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harr, J. E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosa, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 4. Incorporation of P1 Lactam Moieties as L-Glutamine Replacements</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1213</span>– <span class="NLM_lpage">1224</span>, <span class="refDoi"> DOI: 10.1021/jm9805384</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9805384" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1213-1224&author=P.+S.+Dragovichauthor=T.+J.+Prinsauthor=R.+Zhouauthor=S.+E.+Webberauthor=J.+T.+Marakovitsauthor=S.+A.+Fuhrmanauthor=A.+K.+Patickauthor=D.+A.+Matthewsauthor=C.+A.+Leeauthor=C.+E.+Fordauthor=B.+J.+Burkeauthor=R.+A.+Rejtoauthor=T.+F.+Hendricksonauthor=T.+Tuntlandauthor=E.+L.+Brownauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=J.+E.+V.+Harrauthor=M.+B.+Kosaauthor=S.+T.+Worland&title=Structure-Based+Design%2C+Synthesis%2C+and+Biological+Evaluation+of+Irreversible+Human+Rhinovirus+3C+Protease+Inhibitors.+4.+Incorporation+of+P1+Lactam+Moieties+as+L-Glutamine+Replacements&doi=10.1021%2Fjm9805384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 4. Incorporation of P1 Lactam Moieties as L-Glutamine Replacements</span></div><div class="casAuthors">Dragovich, Peter S.; Prins, Thomas J.; Zhou, Ru; Webber, Stephen E.; Marakovits, Joseph T.; Fuhrman, Shella A.; Patick, Amy K.; Matthews, David A.; Lee, Caroline A.; Ford, Clifford E.; Burke, Benjamin J.; Rejto, Paul A.; Hendrickson, Thomas F.; Tuntland, Tove; Brown, Edward L.; Meador, James W., III; Ferre, Rose Ann; Harr, James E. V.; Kosa, Maha B.; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1213-1224</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-based design, chem. synthesis, and biol. evaluation of various human rhinovirus (HRV) 3C protease (3CP) inhibitors, e.g. I [R = CH2CHMe2, CHMe2; X = (S)-CH, (R)-CH, N; n = 1, 2] and II (R = CHMe2, CMe3; Y = NH, CH2) which incorporate P1 lactam moieties in lieu of an L-glutamine residue are described.  These compds. are comprised of a tripeptidyl or peptidomimetic binding determinant and an Et propenoate Michael acceptor moiety which forms an irreversible covalent adduct with the active site cysteine residue of the 3C enzyme.  The P1-lactam-contg. inhibitors display significantly increased 3CP inhibition activity along with improved antirhinoviral properties relative to corresponding L-glutamine-derived mols.  In addn., several lactam-contg. compds. exhibit excellent selectivity for HRV 3CP over several other serine and cysteine proteases and are not appreciably degraded by a variety of biol. agents.  Lactam II (R = CHMe2, Y = CH2, n = 1) (AG7088) was one of the most potent inhibitors (mean antirhinoviral EC90 ≈ 0.10 μM, n = 46 serotypes), and is shown to warrant addnl. preclin. development to explore its potential for use as an antirhinoviral agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlBuxFPwMr97Vg90H21EOLACvtfcHk0li8YQm7RcrHnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D&md5=0e2bf5a03d17ea371663a252e7b97b20</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm9805384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9805384%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DMarakovits%26aufirst%3DJ.%2BT.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBurke%26aufirst%3DB.%2BJ.%26aulast%3DRejto%26aufirst%3DR.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DHarr%26aufirst%3DJ.%2BE.%2BV.%26aulast%3DKosa%26aufirst%3DM.%2BB.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DStructure-Based%2520Design%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520Irreversible%2520Human%2520Rhinovirus%25203C%2520Protease%2520Inhibitors.%25204.%2520Incorporation%2520of%2520P1%2520Lactam%2520Moieties%2520as%2520L-Glutamine%2520Replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1213%26epage%3D1224%26doi%3D10.1021%2Fjm9805384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Center for Disease Control and Prevention</span>. <a href="https://www.cdc.gov/norovirus/trends-outbreaks/worldwide.html" class="extLink">https://www.cdc.gov/norovirus/trends-outbreaks/worldwide.html</a> (accessed 2021-02-06).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Center+for+Disease+Control+and+Prevention.+https%3A%2F%2Fwww.cdc.gov%2Fnorovirus%2Ftrends-outbreaks%2Fworldwide.html+%28accessed+2021-02-06%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginex, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maestro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrado-Gil, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Geijo, M.Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urquiza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramírez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span> <span> </span><span class="NLM_article-title">COVID-19: Drug Targets and Potential Treatments</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12359</span>– <span class="NLM_lpage">12386</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00606</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00606" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12359-12386&author=C.+Gilauthor=T.+Ginexauthor=I.+Maestroauthor=V.+Nozalauthor=L.+Barrado-Gilauthor=M.%C3%81.+Cuesta-Geijoauthor=J.+Urquizaauthor=D.+Ram%C3%ADrezauthor=C.+Alonsoauthor=N.+E.+Campilloauthor=A.+Martinez&title=COVID-19%3A+Drug+Targets+and+Potential+Treatments&doi=10.1021%2Facs.jmedchem.0c00606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19: Drug Targets and Potential Treatments</span></div><div class="casAuthors">Gil, Carmen; Ginex, Tiziana; Maestro, Ines; Nozal, Vanesa; Barrado-Gil, Lucia; Cuesta-Geijo, Miguel Angel; Urquiza, Jesus; Ramirez, David; Alonso, Covadonga; Campillo, Nuria E.; Martinez, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12359-12386</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Currently, the authors are immersed in a pandemic caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which severely threatens public health worldwide.  Until now, no drug or vaccine has been approved to treat the severe disease caused by this coronavirus, COVID-19.  The authors will focus on the main virus-based and host-based targets that can guide medicinal chem. efforts to discover new drugs for this devastating disease.  In principle, all CoVs enzymes and proteins involved in viral replication and the control of host cellular machineries are potentially druggable targets in the search for therapeutic options for SARS-CoV-2.  This perspective provides an overview of the main targets from a structural point of view, together with reported therapeutic compds. with activity against SARS-CoV-2 and/or other CoVs.  Also, the role of innate immune response to coronavirus infection and the related therapeutic options will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsg-YRktOknrVg90H21EOLACvtfcHk0liOlXLy6EmJdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I&md5=6a0dc17c5f601f09a92da933f81bbbea</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00606%26sid%3Dliteratum%253Aachs%26aulast%3DGil%26aufirst%3DC.%26aulast%3DGinex%26aufirst%3DT.%26aulast%3DMaestro%26aufirst%3DI.%26aulast%3DNozal%26aufirst%3DV.%26aulast%3DBarrado-Gil%26aufirst%3DL.%26aulast%3DCuesta-Geijo%26aufirst%3DM.%25C3%2581.%26aulast%3DUrquiza%26aufirst%3DJ.%26aulast%3DRam%25C3%25ADrez%26aufirst%3DD.%26aulast%3DAlonso%26aufirst%3DC.%26aulast%3DCampillo%26aufirst%3DN.%2BE.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCOVID-19%253A%2520Drug%2520Targets%2520and%2520Potential%2520Treatments%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12359%26epage%3D12386%26doi%3D10.1021%2Facs.jmedchem.0c00606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, S.K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, A.K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, C.C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, P. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, K.-Y.</span></span> <span> </span><span class="NLM_article-title">Molecular Epidemiology of Human Coronavirus OC43 Reveals Evolution of Different Genotypes over Time and Recent Emergence of a Novel Genotype due to Natural Recombination</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>85</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">11325</span>– <span class="NLM_lpage">11337</span>, <span class="refDoi"> DOI: 10.1128/JVI.05512-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1128%2FJVI.05512-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=21849456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCltr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=11325-11337&issue=21&author=S.K.+P.+Lauauthor=P.+Leeauthor=A.K.+L.+Tsangauthor=C.C.+Y.+Yipauthor=H.+Tseauthor=R.+A.+Leeauthor=L.-Y.+Soauthor=Y.-L.+Lauauthor=K.-H.+Chanauthor=P.+C.+Y.+Wooauthor=K.-Y.+Yuen&title=Molecular+Epidemiology+of+Human+Coronavirus+OC43+Reveals+Evolution+of+Different+Genotypes+over+Time+and+Recent+Emergence+of+a+Novel+Genotype+due+to+Natural+Recombination&doi=10.1128%2FJVI.05512-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular epidemiology of human coronavirus OC43 reveals evolution of different genotypes over time and recent emergence of a novel genotype due to natural recombination</span></div><div class="casAuthors">Lau, Susanna K. P.; Lee, Paul; Tsang, Alan K. L.; Yip, Cyril C. Y.; Tse, Herman; Lee, Rodney A.; So, Lok-Yee; Lau, Yu-Lung; Chan, Kwok-Hung; Woo, Patrick C. Y.; Yuen, Kwok-Yung</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11325-11337</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Although human coronavirus OC43-OC43 (HCoV-OC43) is the coronavirus most commonly assocd. with human infections, little is known about its mol. epidemiol. and evolution.  We conducted a mol. epidemiol. study to investigate different genotypes and potential recombination in HCoV-OC43.  Twenty-nine HCoV-OC43 strains from nasopharyngeal aspirates, collected from 2004 to 2011, were subjected to RNA-dependent RNA polymerase (RdRp), spike, and nucleocapsid gene anal.  Phylogenetic anal. showed at least three distinct clusters of HCoV-OC43, although 10 unusual strains displayed incongruent phylogenetic positions between RdRp and spike genes.  This suggested the presence of four HCoV-OC43 genotypes (A to D), with genotype D most likely arising from recombination.  The complete genome sequencing of two genotype C and D strains and bootscan anal. showed recombination events between genotypes B and C in the generation of genotype D.  Of the 29 strains, none belonged to the more ancient genotype A, 5 from 2004 belonged to genotype B, 15 from 2004 to 2006 belonged to genotype C, and 1 from 2004 and all 8 from 2008 to 2011 belonged to the recombinant genotype D.  Mol. clock anal. using spike and nucleocapsid genes dated the most recent common ancestor of all genotypes to the 1950s, genotype B and C to the 1980s, genotype B to the 1990s, and genotype C to the late 1990s to early 2000s, while the recombinant genotype D strains were detected as early as 2004.  This represents the first study to describe natural recombination in HCoV-OC43 and the evolution of different genotypes over time, leading to the emergence of novel genotype D, which is assocd. with pneumonia in our elderly population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSYvYE-NhLPbVg90H21EOLACvtfcHk0liOlXLy6EmJdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCltr3E&md5=ca95f357f42cea988b79ebbca43cf567</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FJVI.05512-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.05512-11%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DS.K.%2BP.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DTsang%26aufirst%3DA.K.%2BL.%26aulast%3DYip%26aufirst%3DC.C.%2BY.%26aulast%3DTse%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DR.%2BA.%26aulast%3DSo%26aufirst%3DL.-Y.%26aulast%3DLau%26aufirst%3DY.-L.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DWoo%26aufirst%3DP.%2BC.%2BY.%26aulast%3DYuen%26aufirst%3DK.-Y.%26atitle%3DMolecular%2520Epidemiology%2520of%2520Human%2520Coronavirus%2520OC43%2520Reveals%2520Evolution%2520of%2520Different%2520Genotypes%2520over%2520Time%2520and%2520Recent%2520Emergence%2520of%2520a%2520Novel%2520Genotype%2520due%2520to%2520Natural%2520Recombination%26jtitle%3DJ.%2520Virol.%26date%3D2011%26volume%3D85%26issue%3D21%26spage%3D11325%26epage%3D11337%26doi%3D10.1128%2FJVI.05512-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Centers for Disease Control and Prevention</span>. <a href="https://www.cdc.gov/coronavirus/types.html" class="extLink">https://www.cdc.gov/coronavirus/types.html</a> (accessed 2021-02-06).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Centers+for+Disease+Control+and+Prevention.+https%3A%2F%2Fwww.cdc.gov%2Fcoronavirus%2Ftypes.html+%28accessed+2021-02-06%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anirudhan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, L.</span></span> <span> </span><span class="NLM_article-title">Targeting SARS-CoV-2 Viral Proteases as a Therapeutic Strategy to Treat COVID-19</span>. <i>J. Med. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2722</span>– <span class="NLM_lpage">2734</span>, <span class="refDoi"> DOI: 10.1002/jmv.26814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fjmv.26814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33475167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjsVGgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2021&pages=2722-2734&author=V.+Anirudhanauthor=H.+Leeauthor=H.+Chengauthor=L.+Cooperauthor=L.+Rong&title=Targeting+SARS-CoV-2+Viral+Proteases+as+a+Therapeutic+Strategy+to+Treat+COVID-19&doi=10.1002%2Fjmv.26814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting SARS-CoV-2 viral proteases as a therapeutic strategy to treat COVID-19</span></div><div class="casAuthors">Anirudhan, Varada; Lee, Hyun; Cheng, Han; Cooper, Laura; Rong, Lijun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medical Virology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2722-2734</span>CODEN:
                <span class="NLM_cas:coden">JMVIDB</span>;
        ISSN:<span class="NLM_cas:issn">0146-6615</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The 21st century has witnessed three outbreaks of coronavirus (CoVs) infections caused by severe acute respiratory syndrome (SARS)-CoV, Middle East respiratory syndrome (MERS)-CoV, and SARS-CoV-2.  Coronavirus disease 2019 (COVID-19), caused by SARS-CoV-2, spreads rapidly and since the discovery of the first COVID-19 infection in Dec. 2019, has caused 1.2 million deaths worldwide and 226,777 deaths in the United States alone.  The high amino acid similarity between SARS-CoV and SARS-CoV-2 viral proteins supports testing therapeutic mols. that were designed to treat SARS infections during the 2003 epidemic.  In this review, we provide information on possible COVID-19 treatment strategies that act via inhibition of the two essential proteins of the virus, 3C-like protease (3CLpro) or papain-like protease (PLpro).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPAU7_-rnfw7Vg90H21EOLACvtfcHk0liOlXLy6EmJdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjsVGgtb4%253D&md5=4dcced5135ff23b9b1a9de385a4d952f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fjmv.26814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmv.26814%26sid%3Dliteratum%253Aachs%26aulast%3DAnirudhan%26aufirst%3DV.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DCooper%26aufirst%3DL.%26aulast%3DRong%26aufirst%3DL.%26atitle%3DTargeting%2520SARS-CoV-2%2520Viral%2520Proteases%2520as%2520a%2520Therapeutic%2520Strategy%2520to%2520Treat%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Virol.%26date%3D2021%26volume%3D93%26spage%3D2722%26epage%3D2734%26doi%3D10.1002%2Fjmv.26814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Long, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orr, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, J.</span></span> <span> </span><span class="NLM_article-title">A consensus sequence for substrate hydrolysis by rhinovirus 3C proteinase</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>258</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/0014-5793(89)81619-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2F0014-5793%2889%2981619-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=2556299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK3cXpsVCnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=258&publication_year=1989&pages=75-78&issue=1&author=A.+C.+Longauthor=D.+C.+Orrauthor=J.+M.+Cameronauthor=B.+M.+Dunnauthor=J.+Kay&title=A+consensus+sequence+for+substrate+hydrolysis+by+rhinovirus+3C+proteinase&doi=10.1016%2F0014-5793%2889%2981619-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8aR"><div class="casContent"><span class="casTitleNuber">8a</span><div class="casTitle"><span class="NLM_cas:atitle">A consensus sequence for substrate hydrolysis by rhinovirus 3C proteinase</span></div><div class="casAuthors">Long, Audrey C.; Orr, David C.; Cameron, Janet M.; Dunn, Ben M.; Kay, John</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">258</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-8</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    </div><div class="casAbstract">Kinetic consts. were detd. for the hydrolysis of a series of synthetic peptide substrates by recombinant rhinovirus (HRV 14) 3C proteinase.  Systematic removal or replacement of individual residues indicated that the min. sequence required for effective cleavage by the viral cysteine proteinase was P5-Val/Tyhr-P3-P2-Gln-Gly-Pro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU2X1pA77VGrVg90H21EOLACvtfcHk0lguTrbzCOIZXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXpsVCnsg%253D%253D&md5=c3a8c8e64b24d0e329e84270242207a6</span></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=10.1016%2F0014-5793%2889%2981619-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-5793%252889%252981619-9%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DA.%2BC.%26aulast%3DOrr%26aufirst%3DD.%2BC.%26aulast%3DCameron%26aufirst%3DJ.%2BM.%26aulast%3DDunn%26aufirst%3DB.%2BM.%26aulast%3DKay%26aufirst%3DJ.%26atitle%3DA%2520consensus%2520sequence%2520for%2520substrate%2520hydrolysis%2520by%2520rhinovirus%25203C%2520proteinase%26jtitle%3DFEBS%2520Lett.%26date%3D1989%26volume%3D258%26issue%3D1%26spage%3D75%26epage%3D78%26doi%3D10.1016%2F0014-5793%2889%2981619-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit8b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">May, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanathan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, K.K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korba, B.</span></span> <span> </span><span class="NLM_article-title">The P4-P2′ Amino Acids Surrounding Human Norovirus Polyprotein Cleavage Sites Define the Core Sequence Regulating Self-Processing Order</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>88</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">10738</span>– <span class="NLM_lpage">10747</span>, <span class="refDoi"> DOI: 10.1128/JVI.01357-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1128%2FJVI.01357-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=24991013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ygtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2014&pages=10738-10747&issue=18&author=J.+Mayauthor=P.+Viswanathanauthor=K.K.-S.+Ngauthor=A.+Medvedevauthor=B.+Korba&title=The+P4-P2%E2%80%B2+Amino+Acids+Surrounding+Human+Norovirus+Polyprotein+Cleavage+Sites+Define+the+Core+Sequence+Regulating+Self-Processing+Order&doi=10.1128%2FJVI.01357-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8bR"><div class="casContent"><span class="casTitleNuber">8b</span><div class="casTitle"><span class="NLM_cas:atitle">The P4-P2' amino acids surrounding human norovirus polyprotein cleavage sites define the core sequence regulating self-processing order</span></div><div class="casAuthors">May, Jared; Viswanathan, Prasanth; Ng, Kenneth K.-S.; Medvedev, Alexei; Korba, Brent</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10738-10747, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Noroviruses (NoV) are members of the family Caliciviridae.  The human NoV open reading frame 1 (ORF1) encodes a 200-kDa polyprotein which is cleaved by the viral 20-kDa 3C-like protease (Pro, NS6) into 6 nonstructural proteins that are necessary for viral replication.  The NoV ORF1 polyprotein is processed in a specific order, with "early" sites (NS1/2-3 and NS3-4) being cleaved rapidly and three "late" sites (NS4-5, NS5-6, and NS6-7) processed subsequently and less efficiently.  Previously, we demonstrated that the NoV polyprotein processing order is directly correlated with the efficiency of the enzyme, which is regulated by the primary amino acid sequences surrounding ORF1 cleavage sites.  Using fluorescence resonance energy transfer (FRET) peptides representing the NS2-3 and NS6-7 ORF1 cleavage sites, we now demonstrate that the amino acids spanning positions P4 to P2' (P4-P2') surrounding each site comprise the core sequence controlling NoV protease enzyme efficiency.  Furthermore, the NoV polyprotein self-processing order can be altered by interchanging this core sequence between NS2-3 and any of the three late sites in in vitro transcription-translation assays.  We also demonstrate that the nature of the side chain at the P3 position for the NS1/2-3 (Nterm/NTPase) site confers significant influence on enzyme catalysis (kcat and kcat/Km), a feature overlooked in previous structural studies.  Mol. modeling provides possible explanations for the P3 interactions with NoV protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXvwzhPOquPLVg90H21EOLACvtfcHk0lguTrbzCOIZXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ygtb3I&md5=c241cbf13509bcc9aa36a2729535937e</span></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1128%2FJVI.01357-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01357-14%26sid%3Dliteratum%253Aachs%26aulast%3DMay%26aufirst%3DJ.%26aulast%3DViswanathan%26aufirst%3DP.%26aulast%3DNg%26aufirst%3DK.K.-S.%26aulast%3DMedvedev%26aufirst%3DA.%26aulast%3DKorba%26aufirst%3DB.%26atitle%3DThe%2520P4-P2%25E2%2580%25B2%2520Amino%2520Acids%2520Surrounding%2520Human%2520Norovirus%2520Polyprotein%2520Cleavage%2520Sites%2520Define%2520the%2520Core%2520Sequence%2520Regulating%2520Self-Processing%2520Order%26jtitle%3DJ.%2520Virol.%26date%3D2014%26volume%3D88%26issue%3D18%26spage%3D10738%26epage%3D10747%26doi%3D10.1128%2FJVI.01357-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit8c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">What coronavirus 3C-like protease tells us: From structure, substrate selectivity, to inhibitor design</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1965</span>– <span class="NLM_lpage">1998</span>, <span class="refDoi"> DOI: 10.1002/med.21783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fmed.21783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33460213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtl2jsbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2021&pages=1965-1998&issue=4&author=M.+Xiongauthor=H.+Suauthor=W.+Zhaoauthor=H.+Xieauthor=Q.+Shaoauthor=Y.+Xu&title=What+coronavirus+3C-like+protease+tells+us%3A+From+structure%2C+substrate+selectivity%2C+to+inhibitor+design&doi=10.1002%2Fmed.21783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8cR"><div class="casContent"><span class="casTitleNuber">8c</span><div class="casTitle"><span class="NLM_cas:atitle">What coronavirus 3C-like protease tells us: From structure, substrate selectivity, to inhibitor design</span></div><div class="casAuthors">Xiong, Muya; Su, Haixia; Zhao, Wenfeng; Xie, Hang; Shao, Qiang; Xu, Yechun</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1965-1998</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The emergence of a variety of coronaviruses (CoVs) in the last decades has posed huge threats to human health.  Esp., the ongoing pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has led to more than 70 million infections and over 1.6 million of deaths worldwide in the past few months.  None of the efficacious antiviral agents against human CoVs have been approved yet. 3C-like protease (3CLpro) is an attractive target for antiviral intervention due to its essential role in processing polyproteins translated from viral RNA, and its conserved structural feature and substrate specificity among CoVs in spite of the sequence variation.  This review focuses on all available crystal structures of 12 CoV 3CLpros and their inhibitors, and intends to provide a comprehensive understanding of this protease from multiple aspects including its structural features, substrate specificity, inhibitor binding modes, and more importantly, to recapitulate the similarity and diversity among different CoV 3CLpros and the structure-activity relationship of various types of inhibitors.  Such an attempt could gain a deep insight into the inhibition mechanisms and drive future structure-based drug discovery targeting 3CLpros.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHnlfhAJGwSrVg90H21EOLACvtfcHk0lguTrbzCOIZXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtl2jsbvP&md5=d69289782b99eba0be757942e3172e56</span></div><a href="/servlet/linkout?suffix=cit8c&amp;dbid=16384&amp;doi=10.1002%2Fmed.21783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21783%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DWhat%2520coronavirus%25203C-like%2520protease%2520tells%2520us%253A%2520From%2520structure%252C%2520substrate%2520selectivity%252C%2520to%2520inhibitor%2520design%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2021%26volume%3D41%26issue%3D4%26spage%3D1965%26epage%3D1998%26doi%3D10.1002%2Fmed.21783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit8d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusov, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Brunn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wilde, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">α-Ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: structure-based design, synthesis, and activity assessment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4562</span>– <span class="NLM_lpage">4578</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01828</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01828" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4562-4578&issue=9&author=L.+Zhangauthor=D.+Linauthor=Y.+Kusovauthor=Y.+Nianauthor=Q.+Maauthor=J.+Wangauthor=A.+von+Brunnauthor=P.+Leyssenauthor=K.+Lankoauthor=J.+Neytsauthor=A.+de+Wildeauthor=E.+J.+Snijderauthor=H.+Liuauthor=R.+Hilgenfeld&title=%CE%B1-Ketoamides+as+broad-spectrum+inhibitors+of+coronavirus+and+enterovirus+replication%3A+structure-based+design%2C+synthesis%2C+and+activity+assessment&doi=10.1021%2Facs.jmedchem.9b01828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8dR"><div class="casContent"><span class="casTitleNuber">8d</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketoamides as Broad-Spectrum Inhibitors of Coronavirus and Enterovirus Replication: Structure-Based Design, Synthesis, and Activity Assessment</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Kusov, Yuri; Nian, Yong; Ma, Qingjun; Wang, Jiang; von Brunn, Albrecht; Leyssen, Pieter; Lanko, Kristina; Neyts, Johan; de Wilde, Adriaan; Snijder, Eric J.; Liu, Hong; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4562-4578</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate.  Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs.  In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic α-ketoamides as inhibitors of main and 3C proteases.  Six crystal structures of protease:inhibitor complexes were detd. as part of this study.  Compds. synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic.  Optimization of the P2 substituent of the α-ketoamides proved crucial for achieving near-equipotency against the three virus genera.  The best near-equipotent inhibitors, (I) [R1 = cyclopentylmethyl (P2) and R1 = cyclohexylmethyl (P2)], display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures.  In Huh7 cells, I [R1 = cyclohexylmethyl (P2)] exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWpcfl4wTRLLVg90H21EOLACvtfcHk0li4XaDdYJa2mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D&md5=60d91a8ecf916856a844d835ecf84d5f</span></div><a href="/servlet/linkout?suffix=cit8d&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01828%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DKusov%26aufirst%3DY.%26aulast%3DNian%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3Dvon%2BBrunn%26aufirst%3DA.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DLanko%26aufirst%3DK.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3Dde%2BWilde%26aufirst%3DA.%26aulast%3DSnijder%26aufirst%3DE.%2BJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3D%25CE%25B1-Ketoamides%2520as%2520broad-spectrum%2520inhibitors%2520of%2520coronavirus%2520and%2520enterovirus%2520replication%253A%2520structure-based%2520design%252C%2520synthesis%252C%2520and%2520activity%2520assessment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D9%26spage%3D4562%26epage%3D4578%26doi%3D10.1021%2Facs.jmedchem.9b01828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwaltney, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi-Burris, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gersten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsyu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span> <span> </span><span class="NLM_article-title">Randomized, Double-Blind, Placebo-Controlled Studies of Ruprintrivir Nasal Spray 2-Percent Suspension for Prevention and Treatment of Experimentally Induced Rhinovirus Colds in Healthy Volunteers</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3907</span>– <span class="NLM_lpage">3916</span>, <span class="refDoi"> DOI: 10.1128/AAC.47.12.3907-3916.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1128%2FAAC.47.12.3907-3916.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=14638501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsV2nu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2003&pages=3907-3916&issue=12&author=F.+G.+Haydenauthor=R.+B.+Turnerauthor=J.+M.+Gwaltneyauthor=K.+Chi-Burrisauthor=M.+Gerstenauthor=P.+Hsyuauthor=A.+K.+Patickauthor=G.+J.+Smithauthor=L.+S.+Zalman&title=Randomized%2C+Double-Blind%2C+Placebo-Controlled+Studies+of+Ruprintrivir+Nasal+Spray+2-Percent+Suspension+for+Prevention+and+Treatment+of+Experimentally+Induced+Rhinovirus+Colds+in+Healthy+Volunteers&doi=10.1128%2FAAC.47.12.3907-3916.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Phase II, randomized, double-blind, placebo-controlled studies of ruprintrivir nasal spray 2% suspension for prevention and treatment of experimentally induced rhinovirus colds in healthy volunteers</span></div><div class="casAuthors">Hayden, Frederick G.; Turner, Ronald B.; Gwaltney, Jack M.; Chi-burris, Kathy; Gersten, Merril; Hsyu, Poe; Patick, Amy K.; Smith, George J., III; Zalman, Leora S.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3907-3916</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Human rhinovirus (HRV) infections are usually self-limited but may be assocd. with serious consequences, particularly in those with asthma and chronic respiratory disease.  Effective antiviral agents are needed for preventing and treating HRV illnesses.  Ruprintrivir (Agouron Pharmaceuticals, Inc., San Diego, Calif.) selectively inhibits HRV 3C protease and shows potent, broad-spectrum anti-HRV activity in vitro.  We conducted three double-blind, placebo-controlled clin. trials in 202 healthy volunteers to assess the activity of ruprintrivir in exptl. HRV infection.  Subjects were randomized to receive intranasal ruprintrivir (8 mg) or placebo sprays as prophylaxis (two or five times daily [2×/day or 5×/day] for 5 days) starting 6 h before infection or as treatment (5×/day for 4 days) starting 24 h after infection.  Ruprintrivir prophylaxis reduced the proportion of subjects with pos. viral cultures (for 5×/day dosing groups, 44% for ruprintrivir treatment group vs. 70% for placebo treatment group [P = 0.03]; for 2×/day dosing groups, 60% for ruprintrivir group vs. 92% for placebo group [P = 0.004]) and viral titers but did not decrease the frequency of colds.  Ruprintrivir treatment reduced the mean total daily symptom score (2.2 for ruprintrivir treatment group and 3.3 for the placebo treatment group [P = 0.014]) by 33%.  Secondary endpoints, including viral titers, individual symptom scores, and nasal discharge wts., were also reduced by ruprintrivir treatment.  Overall, ruprintrivir was well tolerated; blood-tinged mucus and nasal passage irritation were the most common adverse effects reported.  Pharmacokinetic anal. of plasma and nasal ruprintrivir concns. revealed intranasal drug residence with minimal systemic absorption.  Results from these studies in exptl. rhinoviral infection support continued investigation of intranasal ruprintrivir in the setting of natural HRV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoudaxRiBJLg7Vg90H21EOLACvtfcHk0li4XaDdYJa2mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsV2nu7w%253D&md5=6d6c33f56af0c4f45145b291ace2dba1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FAAC.47.12.3907-3916.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.47.12.3907-3916.2003%26sid%3Dliteratum%253Aachs%26aulast%3DHayden%26aufirst%3DF.%2BG.%26aulast%3DTurner%26aufirst%3DR.%2BB.%26aulast%3DGwaltney%26aufirst%3DJ.%2BM.%26aulast%3DChi-Burris%26aufirst%3DK.%26aulast%3DGersten%26aufirst%3DM.%26aulast%3DHsyu%26aufirst%3DP.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26atitle%3DRandomized%252C%2520Double-Blind%252C%2520Placebo-Controlled%2520Studies%2520of%2520Ruprintrivir%2520Nasal%2520Spray%25202-Percent%2520Suspension%2520for%2520Prevention%2520and%2520Treatment%2520of%2520Experimentally%2520Induced%2520Rhinovirus%2520Colds%2520in%2520Healthy%2520Volunteers%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2003%26volume%3D47%26issue%3D12%26spage%3D3907%26epage%3D3916%26doi%3D10.1128%2FAAC.47.12.3907-3916.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti
Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerawarna, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uy, R. A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damalanka, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehzabeen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design and Optimization of Dipeptidyl Inhibitors of Norovirus 3CL Protease. Structure–Activity Relationships and Biochemical, X-ray Crystallographic, Cell-Based, and <i>In Vivo</i> Studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3144</span>– <span class="NLM_lpage">3155</span>, <span class="refDoi"> DOI: 10.1021/jm5019934</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5019934" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktlertr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3144-3155&author=A.+C.+Galasiti%0AKankanamalageauthor=Y.+Kimauthor=P.+M.+Weerawarnaauthor=R.+A.+Z.+Uyauthor=V.+C.+Damalankaauthor=S.+R.+Mandadapuauthor=K.+R.+Allistonauthor=N.+Mehzabeenauthor=K.+P.+Battaileauthor=S.+Lovellauthor=K.-O.+Changauthor=W.+C.+Groutas&title=Structure-Guided+Design+and+Optimization+of+Dipeptidyl+Inhibitors+of+Norovirus+3CL+Protease.+Structure%E2%80%93Activity+Relationships+and+Biochemical%2C+X-ray+Crystallographic%2C+Cell-Based%2C+and+In+Vivo+Studies&doi=10.1021%2Fjm5019934"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Dipeptidyl Inhibitors of Norovirus 3CL Protease. Structure-Activity Relationships and Biochemical, X-ray Crystallographic, Cell-Based, and In Vivo Studies</span></div><div class="casAuthors">Galasiti Kankanamalage, Anushka C.; Kim, Yunjeong; Weerawarna, Pathum M.; Uy, Roxanne Adeline Z.; Damalanka, Vishnu C.; Mandadapu, Sivakoteswara Rao; Alliston, Kevin R.; Mehzabeen, Nurjahan; Battaile, Kevin P.; Lovell, Scott; Chang, Kyeong-Ok; Groutas, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3144-3155</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Norovirus infection constitutes the primary cause of acute viral gastroenteritis.  There are currently no vaccines or norovirus-specific antiviral therapeutics available for the management of norovirus infection.  Norovirus 3C-like protease is essential for viral replication, consequently, inhibition of this enzyme is a fruitful avenue of investigation that may lead to the emergence of antinorovirus therapeutics.  The authors describe herein the optimization of dipeptidyl inhibitors of norovirus 3C-like protease using iterative SAR, x-ray crystallog., and enzyme and cell-based studies.  The authors also demonstrate herein in vivo efficacy of an inhibitor using the murine model of norovirus infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_rKoPrEayGbVg90H21EOLACvtfcHk0ljgeJ-Ih8rDOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktlertr4%253D&md5=fc3d5b27ee9241e5d76b7f040d3ca040</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm5019934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5019934%26sid%3Dliteratum%253Aachs%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DWeerawarna%26aufirst%3DP.%2BM.%26aulast%3DUy%26aufirst%3DR.%2BA.%2BZ.%26aulast%3DDamalanka%26aufirst%3DV.%2BC.%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DMehzabeen%26aufirst%3DN.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DK.-O.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26atitle%3DStructure-Guided%2520Design%2520and%2520Optimization%2520of%2520Dipeptidyl%2520Inhibitors%2520of%2520Norovirus%25203CL%2520Protease.%2520Structure%25E2%2580%2593Activity%2520Relationships%2520and%2520Biochemical%252C%2520X-ray%2520Crystallographic%252C%2520Cell-Based%252C%2520and%2520In%2520Vivo%2520Studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3144%26epage%3D3155%26doi%3D10.1021%2Fjm5019934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span>; <span class="NLM_string-name">Kania, R. S.</span>; <span class="NLM_string-name">Nieman, J. A.</span>; <span class="NLM_string-name">Planken, S. P.</span>; <span class="NLM_string-name">Smith, G. J.</span></span> <span> </span><span class="NLM_article-title">Anticoronaviral Compounds and Compositions, Their Pharmaceutical Uses and Materials For Their Synthesis</span>. <span class="NLM_patent">WO2005/113580</span>, May 9, <span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=R.+L.+Hoffman&author=R.+S.+Kania&author=J.+A.+Nieman&author=S.+P.+Planken&author=G.+J.+Smith&title=Anticoronaviral+Compounds+and+Compositions%2C+Their+Pharmaceutical+Uses+and+Materials+For+Their+Synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26atitle%3DAnticoronaviral%2520Compounds%2520and%2520Compositions%252C%2520Their%2520Pharmaceutical%2520Uses%2520and%2520Materials%2520For%2520Their%2520Synthesis%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freedman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tchesnokov, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, J. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinazi, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghton, M.</span></span> <span> </span><span class="NLM_article-title">Application of Molecular Dynamics Simulations to the Design of Nucleotide Inhibitors Binding to Norovirus Polymerase</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6566</span>– <span class="NLM_lpage">6578</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00742</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00742" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVCnsrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=6566-6578&author=H.+Freedmanauthor=J.+Kunduauthor=P.+E.+Tchesnokovauthor=J.+L.+M.+Lawauthor=J.+A.+Niemanauthor=R.+F.+Schinaziauthor=D.+L.+Tyrrellauthor=M.+Gotteauthor=M.+Houghton&title=Application+of+Molecular+Dynamics+Simulations+to+the+Design+of+Nucleotide+Inhibitors+Binding+to+Norovirus+Polymerase&doi=10.1021%2Facs.jcim.0c00742"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Molecular Dynamics Simulations to the Design of Nucleotide Inhibitors Binding to Norovirus Polymerase</span></div><div class="casAuthors">Freedman, Holly; Kundu, Juthika; Tchesnokov, Egor Petrovitch; Law, John Lok Man; Nieman, James A.; Schinazi, Raymond F.; Tyrrell, D. Lorne; Gotte, Matthias; Houghton, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6566-6578</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The RNA-dependent RNA polymerase (RdRp) of norovirus is an attractive target of antiviral agents aimed at providing protection against norovirus-assocd. gastroenteritis.  Here, we perform mol. dynamics simulations of the crystal structure of norovirus RdRp in complex with several known binders, as well as free-energy simulations by free-energy perturbation (FEP) to det. binding free energies of these mols. relative to the natural nucleotide substrates.  We det. exptl. EC50 values and nucleotide incorporation efficiencies for several of these compds.  Moreover, we investigate the mechanism of inhibition of some of these ligands.  Using FEP, we screened a virtual nucleotide library with 121 elements for binding to the polymerase and successfully identified two novel chain terminators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTlxYz2z6Pm7Vg90H21EOLACvtfcHk0ljgeJ-Ih8rDOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVCnsrzN&md5=18be63411c1858ef82c4f9a0542ac390</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00742%26sid%3Dliteratum%253Aachs%26aulast%3DFreedman%26aufirst%3DH.%26aulast%3DKundu%26aufirst%3DJ.%26aulast%3DTchesnokov%26aufirst%3DP.%2BE.%26aulast%3DLaw%26aufirst%3DJ.%2BL.%2BM.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DSchinazi%26aufirst%3DR.%2BF.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DGotte%26aufirst%3DM.%26aulast%3DHoughton%26aufirst%3DM.%26atitle%3DApplication%2520of%2520Molecular%2520Dynamics%2520Simulations%2520to%2520the%2520Design%2520of%2520Nucleotide%2520Inhibitors%2520Binding%2520to%2520Norovirus%2520Polymerase%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D6566%26epage%3D6578%26doi%3D10.1021%2Facs.jcim.0c00742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathnayake, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dampalla, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jesri, A.-R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashipathy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lushington, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Optimization of Dipeptidyl Inhibitors of Norovirus 3CL Protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">11945</span>– <span class="NLM_lpage">11963</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01252</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01252" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVOns7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=11945-11963&issue=20&author=A.+D.+Rathnayakeauthor=Y.+Kimauthor=C.+S.+Dampallaauthor=H.+N.+Nguyenauthor=A.-R.+M.+Jesriauthor=M.+M.+Kashipathyauthor=G.+H.+Lushingtonauthor=K.+P.+Battaileauthor=S.+Lovellauthor=K.-O.+Changauthor=W.+C.+Groutas&title=Structure-Guided+Optimization+of+Dipeptidyl+Inhibitors+of+Norovirus+3CL+Protease&doi=10.1021%2Facs.jmedchem.0c01252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Optimization of Dipeptidyl Inhibitors of Norovirus 3CL Protease</span></div><div class="casAuthors">Rathnayake, Athri D.; Kim, Yunjeong; Dampalla, Chamandi S.; Nguyen, Harry Nhat; Jesri, Abdul-Rahman M.; Kashipathy, Maithri M.; Lushington, Gerald H.; Battaile, Kevin P.; Lovell, Scott; Chang, Kyeong-Ok; Groutas, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">11945-11963</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acute gastroenteritis caused by noroviruses has a major impact on public health worldwide in terms of morbidity, mortality, and economic burden.  The disease impacts most severely immunocompromised patients, the elderly, and children.  The current lack of approved vaccines and small-mol. therapeutics for the treatment and prophylaxis of norovirus infections underscores the need for the development of norovirus-specific drugs.  The studies described herein entail the use of the gem-di-Me moiety as a means of improving the pharmacol. activity and physicochem. properties of a dipeptidyl series of transition state inhibitors of norovirus 3CL protease, an enzyme essential for viral replication.  Several compds. were found to be potent inhibitors of the enzyme in biochem. and cell-based assays.  The pharmacol. activity and cellular permeability of the inhibitors were found to be sensitive to the location of the gem-di-Me group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogL9olpw-8zLVg90H21EOLACvtfcHk0liFBHao8U_LoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVOns7jP&md5=0eb3b684894ddce9eea7491ae5d8e080</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01252%26sid%3Dliteratum%253Aachs%26aulast%3DRathnayake%26aufirst%3DA.%2BD.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DDampalla%26aufirst%3DC.%2BS.%26aulast%3DNguyen%26aufirst%3DH.%2BN.%26aulast%3DJesri%26aufirst%3DA.-R.%2BM.%26aulast%3DKashipathy%26aufirst%3DM.%2BM.%26aulast%3DLushington%26aufirst%3DG.%2BH.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DK.-O.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26atitle%3DStructure-Guided%2520Optimization%2520of%2520Dipeptidyl%2520Inhibitors%2520of%2520Norovirus%25203CL%2520Protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D20%26spage%3D11945%26epage%3D11963%26doi%3D10.1021%2Facs.jmedchem.0c01252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">World Health
Organization</span>. <a href="http://http:/www.who.int/emergencies/diseases/novel-coronavirus-2019" class="extLink">http:/www.who.int/emergencies/diseases/novel-coronavirus-2019</a> (accessed 2021-02-06).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=World+Health%0AOrganization.+http%3A%2Fwww.who.int%2Femergencies%2Fdiseases%2Fnovel-coronavirus-2019+%28accessed+2021-02-06%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamarthi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupte, A.</span></span> <span> </span><span class="NLM_article-title">The journey of remdesivir: from Ebola to COVID-19</span>. <i>Drugs in Context</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.7573/dic.2020-4-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.7573%2Fdic.2020-4-14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1-9&author=J.+Pardoauthor=A.+M.+Shuklaauthor=G.+Chamarthiauthor=A.+Gupte&title=The+journey+of+remdesivir%3A+from+Ebola+to+COVID-19&doi=10.7573%2Fdic.2020-4-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.7573%2Fdic.2020-4-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7573%252Fdic.2020-4-14%26sid%3Dliteratum%253Aachs%26aulast%3DPardo%26aufirst%3DJ.%26aulast%3DShukla%26aufirst%3DA.%2BM.%26aulast%3DChamarthi%26aufirst%3DG.%26aulast%3DGupte%26aufirst%3DA.%26atitle%3DThe%2520journey%2520of%2520remdesivir%253A%2520from%2520Ebola%2520to%2520COVID-19%26jtitle%3DDrugs%2520in%2520Context%26date%3D2020%26volume%3D9%26spage%3D1%26epage%3D9%26doi%3D10.7573%2Fdic.2020-4-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">U.S. Food and Drug Administration</span>. <a href="https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19" class="extLink">https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19</a> (accessed 2021-02-6).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Food+and+Drug+Administration.+https%3A%2F%2Fwww.fda.gov%2Fnews-events%2Fpress-announcements%2Ffda-approves-first-treatment-covid-19+%28accessed+2021-02-6%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheahan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pruijssers, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agostini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leist, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinnon, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappell, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluemling, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natchus, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saindane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolykhalov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harcourt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornburg, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanstrom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denison, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baric, R. S.</span></span> <span> </span><span class="NLM_article-title">An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 in human airway epithelial cell cultures and multiple coronaviruses in mice</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">eabb5883</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.abb5883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1126%2Fscitranslmed.abb5883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32253226" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=eabb5883&author=T.+P.+Sheahanauthor=A.+C.+Simsauthor=S.+Zhouauthor=R.+L.+Grahamauthor=A.+J.+Pruijssersauthor=M.+L.+Agostiniauthor=S.+R.+Leistauthor=A.+Schaferauthor=K.+H.+Dinnonauthor=L.+J.+Stevensauthor=J.+D.+Chappellauthor=X.+Luauthor=T.+M.+Hughesauthor=A.+S.+Georgeauthor=C.+S.+Hillauthor=S.+A.+Montgomeryauthor=A.+J.+Brownauthor=G.+R.+Bluemlingauthor=M.+G.+Natchusauthor=M.+Saindaneauthor=A.+A.+Kolykhalovauthor=G.+Painterauthor=J.+Harcourtauthor=A.+Taminauthor=N.+J.+Thornburgauthor=R.+Swanstromauthor=M.+R.+Denisonauthor=R.+S.+Baric&title=An+orally+bioavailable+broad-spectrum+antiviral+inhibits+SARS-CoV-2+in+human+airway+epithelial+cell+cultures+and+multiple+coronaviruses+in+mice&doi=10.1126%2Fscitranslmed.abb5883"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.abb5883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.abb5883%26sid%3Dliteratum%253Aachs%26aulast%3DSheahan%26aufirst%3DT.%2BP.%26aulast%3DSims%26aufirst%3DA.%2BC.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DGraham%26aufirst%3DR.%2BL.%26aulast%3DPruijssers%26aufirst%3DA.%2BJ.%26aulast%3DAgostini%26aufirst%3DM.%2BL.%26aulast%3DLeist%26aufirst%3DS.%2BR.%26aulast%3DSchafer%26aufirst%3DA.%26aulast%3DDinnon%26aufirst%3DK.%2BH.%26aulast%3DStevens%26aufirst%3DL.%2BJ.%26aulast%3DChappell%26aufirst%3DJ.%2BD.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DHughes%26aufirst%3DT.%2BM.%26aulast%3DGeorge%26aufirst%3DA.%2BS.%26aulast%3DHill%26aufirst%3DC.%2BS.%26aulast%3DMontgomery%26aufirst%3DS.%2BA.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DBluemling%26aufirst%3DG.%2BR.%26aulast%3DNatchus%26aufirst%3DM.%2BG.%26aulast%3DSaindane%26aufirst%3DM.%26aulast%3DKolykhalov%26aufirst%3DA.%2BA.%26aulast%3DPainter%26aufirst%3DG.%26aulast%3DHarcourt%26aufirst%3DJ.%26aulast%3DTamin%26aufirst%3DA.%26aulast%3DThornburg%26aufirst%3DN.%2BJ.%26aulast%3DSwanstrom%26aufirst%3DR.%26aulast%3DDenison%26aufirst%3DM.%2BR.%26aulast%3DBaric%26aufirst%3DR.%2BS.%26atitle%3DAn%2520orally%2520bioavailable%2520broad-spectrum%2520antiviral%2520inhibits%2520SARS-CoV-2%2520in%2520human%2520airway%2520epithelial%2520cell%2520cultures%2520and%2520multiple%2520coronaviruses%2520in%2520mice%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2020%26volume%3D12%26spage%3Deabb5883%26doi%3D10.1126%2Fscitranslmed.abb5883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B.</span></span> <span> </span><span class="NLM_article-title">Targeting proteases: success, failures and future prospects</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">799</span>, <span class="refDoi"> DOI: 10.1038/nrd2092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fnrd2092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=16955069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=785-799&author=B.+Turk&title=Targeting+proteases%3A+success%2C+failures+and+future+prospects&doi=10.1038%2Fnrd2092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18aR"><div class="casContent"><span class="casTitleNuber">18a</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting proteases: successes, failures and future prospects</span></div><div class="casAuthors">Turk, Boris</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">785-799</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Until fairly recently, proteases were considered primarily to be protein-degrading enzymes.  However, this view has dramatically changed and proteases are now seen as extremely important signalling mols. that are involved in numerous vital processes.  Protease signalling pathways are strictly regulated, and the dysregulation of protease activity can lead to pathologies such as cardiovascular and inflammatory diseases, cancer, osteoporosis and neurol. disorders.  Several small-mol. drugs targeting proteases are already on the market and many more are in development.  The status of human protease research and prospects for future protease-targeted drugs are reviewed here, with ref. to some key examples where protease drugs have succeeded or failed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa_X9u2oqlzrVg90H21EOLACvtfcHk0liFBHao8U_LoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltrs%253D&md5=00111ddbeafb283cbe88f7b0ee7bb404</span></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=10.1038%2Fnrd2092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2092%26sid%3Dliteratum%253Aachs%26aulast%3DTurk%26aufirst%3DB.%26atitle%3DTargeting%2520proteases%253A%2520success%252C%2520failures%2520and%2520future%2520prospects%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D785%26epage%3D799%26doi%3D10.1038%2Fnrd2092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit18b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span> <span> </span><span class="NLM_article-title">New Directions for Protease Inhibitors Directed Drug Discovery</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">579</span>, <span class="refDoi"> DOI: 10.1002/bip.22780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fbip.22780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=26584340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SkurjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2016&pages=563-579&author=Y.+Hamadaauthor=Y.+Kiso&title=New+Directions+for+Protease+Inhibitors+Directed+Drug+Discovery&doi=10.1002%2Fbip.22780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">New directions for protease inhibitors directed drug discovery</span></div><div class="casAuthors">Hamada, Yoshio; Kiso, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">563-579</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Proteases play crucial roles in various biol. processes, and their activities are essential for all living organisms-from viruses to humans.  Since their functions are closely assocd. with many pathogenic mechanisms, their inhibitors or activators are important mol. targets for developing treatments for various diseases.  Here, we describe drugs/drug candidates that target proteases, such as malarial plasmepsins, β-secretase, virus proteases, and dipeptidyl peptidase-4.  Previously, we reported inhibitors of aspartic proteases, such as renin, human immunodeficiency virus type 1 protease, human T-lymphotropic virus type I protease, plasmepsins, and β-secretase, as drug candidates for hypertension, adult T-cell leukemia, human T-lymphotropic virus type I-assocd. myelopathy, malaria, and Alzheimer's disease.  Our inhibitors are also described in this review article as examples of drugs that target proteases. © 2015 Wiley Periodicals, Inc.  Biopolymers (Pept Sci) 106: 563-579, 2016.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXnphbid6u97Vg90H21EOLACvtfcHk0lhgvSMxC7dVUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SkurjK&md5=352c1f1c60798f87da18044e7dfe2e2a</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1002%2Fbip.22780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22780%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DY.%26aulast%3DKiso%26aufirst%3DY.%26atitle%3DNew%2520Directions%2520for%2520Protease%2520Inhibitors%2520Directed%2520Drug%2520Discovery%26jtitle%3DBiopolymers%26date%3D2016%26volume%3D106%26spage%3D563%26epage%3D579%26doi%3D10.1002%2Fbip.22780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McClung, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donoghue, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papapoulos, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bone, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langdahl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saag, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiel, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallari, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marc, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonaca, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiviott, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Villiers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippuner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginster, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Portales, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanchetta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbini, C. A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cange, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grip, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyden, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DaSilva, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massaad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verbruggen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polis, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoch, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marc, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatine, M. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Odanacatib for the treatment of postmenopausal osteoporosis: results of the LOFT multicentre, randomised, double-blind, placebo-controlled trial and LOFT Extension study</span>. <i>Lancet Diabetes Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.1016/S2213-8587(19)30346-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2FS2213-8587%2819%2930346-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=31676222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVOnu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=899-911&issue=12&author=M.+R.+McClungauthor=M.+L.+O%E2%80%99Donoghueauthor=S.+E.+Papapoulosauthor=H.+Boneauthor=B.+Langdahlauthor=K.+G.+Saagauthor=I.+R.+Reidauthor=D.+P.+Kielauthor=I.+Cavallariauthor=P.+Marcauthor=M.+P.+Bonacaauthor=S.+D.+Wiviottauthor=T.+de+Villiersauthor=X.+Lingauthor=K.+Lippunerauthor=T.+Nakamuraauthor=J.-Y.+Reginsterauthor=J.+A.+Rodriguez-Portalesauthor=C.+Rouxauthor=J.+Zanchettaauthor=C.+A.+F.+Zerbiniauthor=J.-G.+Parkauthor=K.+Imauthor=A.+Cangeauthor=L.+T.+Gripauthor=N.+Heydenauthor=C.+DaSilvaauthor=D.+Cohnauthor=R.+Massaadauthor=B.+B.+Scottauthor=N.+Verbruggenauthor=D.+Gurnerauthor=D.+L.+Millerauthor=M.+L.+Blairauthor=A.+B.+Polisauthor=S.+A.+Stochauthor=A.+Santoraauthor=A.+Lombardiauthor=A.+T.+Leungauthor=K.+D.+Kaufmanauthor=S.+Marcauthor=M.+S.+Sabatine&title=Odanacatib+for+the+treatment+of+postmenopausal+osteoporosis%3A+results+of+the+LOFT+multicentre%2C+randomised%2C+double-blind%2C+placebo-controlled+trial+and+LOFT+Extension+study&doi=10.1016%2FS2213-8587%2819%2930346-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Odanacatib for the treatment of postmenopausal osteoporosis: results of the LOFT multicentre, randomised, double-blind, placebo-controlled trial and LOFT Extension study</span></div><div class="casAuthors">McClung, Michael R.; O'Donoghue, Michelle L.; Papapoulos, Socrates E.; Bone, Henry; Langdahl, Bente; Saag, Kenneth G.; Reid, Ian R.; Kiel, Douglas P.; Cavallari, Ilaria; Bonaca, Marc P.; Wiviott, Stephen D.; de Villiers, Tobias; Ling, Xu; Lippuner, Kurt; Nakamura, Toshitaka; Reginster, Jean-Yves; Rodriguez-Portales, Jose Adolfo; Roux, Christian; Zanchetta, Jose; Zerbini, Cristiano A. F.; Park, Jeong-Gun; Im, KyungAh; Cange, Abby; Grip, Laura T.; Heyden, Norman; DaSilva, Carolyn; Cohn, Dosinda; Massaad, Rachid; Scott, Boyd B.; Verbruggen, Nadia; Gurner, Deborah; Miller, Deborah L.; Blair, Micki L.; Polis, Adam B.; Stoch, S. Aubrey; Santora, Arthur; Lombardi, Antonio; Leung, Albert T.; Kaufman, Keith D.; Sabatine, Marc S.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Diabetes & Endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">899-911</span>CODEN:
                <span class="NLM_cas:coden">LDEAAS</span>;
        ISSN:<span class="NLM_cas:issn">2213-8587</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Odanacatib, a cathepsin K inhibitor, reduces bone resorption while maintaining bone formation.  Previous work has shown that odanacatib increases bone mineral d. in postmenopausal women with low bone mass.  We aimed to investigate the efficacy and safety of odanacatib to reduce fracture risk in postmenopausal women with osteoporosis.  The Long-term Odanacatib Fracture Trial (LOFT) was a multicentre, randomised, double-blind, placebo-controlled, event-driven study at 388 outpatient clinics in 40 countries.  Eligible participants were women aged at least 65 years who were postmenopausal for 5 years or more, with a femoral neck or total hip bone mineral d. T-score between -2.5 and -4.0 if no previous radiog. vertebral fracture, or between -1.5 and -4.0 with a previous vertebral fracture.  Women with a previous hip fracture, more than one vertebral fracture, or a T-score of less than -4.0 at the total hip or femoral neck were not eligible unless they were unable or unwilling to use approved osteoporosis treatment.  Participants were randomly assigned (1:1) to either oral odanacatib (50 mg once per wk) or matching placebo.  Randomisation was done using an interactive voice recognition system after stratification for previous radiog. vertebral fracture, and treatment was masked to study participants, investigators and their staff, and sponsor personnel.  If the study completed before 5 years of double-blind treatment, consenting participants could enrol in a double-blind extension study (LOFT Extension), continuing their original treatment assignment for up to 5 years from randomisation.  Primary endpoints were incidence of vertebral fractures as assessed using radiographs collected at baseline, 6 and 12 mo, yearly, and at final study visit in participants for whom evaluable radiograph images were available at baseline and at least one other timepoint, and hip and non-vertebral fractures adjudicated as being a result of osteoporosis as assessed by clin. history and radiograph.  Safety was assessed in participants who received at least one dose of study drug.  The adjudicated cardiovascular safety endpoints were a composite of cardiovascular death, myocardial infarction, or stroke, and new-onset atrial fibrillation or flutter.  Individual cardiovascular endpoints and death were also assessed.  LOFT and LOFT Extension are registered with ClinicalTrials.gov (no. NCT00529373) and the European Clin. Trials Database (EudraCT no. 2007-002693-66).  Between Sept 14, 2007, and Nov 17, 2009, we randomly assigned 16 071 evaluable patients to treatment: 8043 to odanacatib and 8028 to placebo.  After a median follow-up of 36.5 mo (IQR 34.43-40.15) 4297 women assigned to odanacatib and 3960 assigned to placebo enrolled in LOFT Extension (total median follow-up 47.6 mo, IQR 35.45-60.06).  In LOFT, cumulative incidence of primary outcomes for odanacatib vs. placebo were: radiog. vertebral fractures 3.7% (251/6770) vs. 7.8% (542/6910), hazard ratio (HR) 0.46, 95% CI 0.40-0.53; hip fractures 0.8% (65/8043) vs. 1.6% (125/8028), 0.53, 0.39-0.71; non-vertebral fractures 5.1% (412/8043) vs. 6.7% (541/8028), 0.77, 0.68-0.87; all p <0.0001.  Combined results from LOFT plus LOFT Extension for cumulative incidence of primary outcomes for odanacatib vs. placebo were: radiog. vertebral fractures 4.9% (341/6909) vs. 9.6% (675/7011), HR 0.48, 95% CI 0.42-0.55; hip fractures 1.1% (86/8043) vs. 2.0% (162/8028), 0.52, 0.40-0.67; non-vertebral fractures 6.4% (512/8043) vs. 8.4% (675/8028), 0.74, 0.66-0.83; all p <0.0001.  In LOFT, the composite cardiovascular endpoint of cardiovascular death, myocardial infarction, or stroke occurred in 273 (3.4%) of 8043 patients in the odanacatib group vs. 245 (3.1%) of 8028 in the placebo group (HR 1.12, 95% CI 0.95-1.34; p = 0.18).  New-onset atrial fibrillation or flutter occurred in 112 (1.4%) of 8043 patients in the odanacatib group vs. 96 (1.2%) of 8028 in the placebo group (HR 1.18, 0.90-1.55; p = 0.24).  Odanacatib was assocd. with an increased risk of stroke (1.7% [136/8043] vs 1.3% [104/8028], HR 1.32, 1.02-1.70; p = 0.034), but not myocardial infarction (0.7% [60/8043] vs 0.9% [74/8028], HR 0.82, 0.58-1.15; p = 0.26).  The HR for all-cause mortality was 1.13 (5.0% [401/8043] vs 4.4% [356/8028], 0.98-1.30; p = 0.10).  When data from LOFT Extension were included, the composite of cardiovascular death, myocardial infarction, or stroke occurred in significantly more patients in the odanacatib group than in the placebo group (401 [5.0%] of 8043 vs 343 [4.3%] of 8028, HR 1.17, 1.02-1.36; p = 0.029, as did stroke 2.3% [187/8043] vs 1.7% [137/8028], HR 1.37, 1.10-1.71; p = 0.0051).  Odanacatib reduced the risk of fracture, but was assocd. with an increased risk of cardiovascular events, specifically stroke, in postmenopausal women with osteoporosis.  Based on the overall balance between benefit and risk, the study's sponsor decided that they would no longer pursue development of odanacatib for treatment of osteoporosis.  Merck Sharp & Dohme Corp, a subsidiary of Merck & Co, Inc, Kenilworth, NJ, USA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcN107ZPXxyrVg90H21EOLACvtfcHk0lhgvSMxC7dVUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVOnu73O&md5=faee30ee846eeb2b368c1191dc9e9acd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS2213-8587%2819%2930346-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2213-8587%252819%252930346-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcClung%26aufirst%3DM.%2BR.%26aulast%3DO%25E2%2580%2599Donoghue%26aufirst%3DM.%2BL.%26aulast%3DPapapoulos%26aufirst%3DS.%2BE.%26aulast%3DBone%26aufirst%3DH.%26aulast%3DLangdahl%26aufirst%3DB.%26aulast%3DSaag%26aufirst%3DK.%2BG.%26aulast%3DReid%26aufirst%3DI.%2BR.%26aulast%3DKiel%26aufirst%3DD.%2BP.%26aulast%3DCavallari%26aufirst%3DI.%26aulast%3DMarc%26aufirst%3DP.%26aulast%3DBonaca%26aufirst%3DM.%2BP.%26aulast%3DWiviott%26aufirst%3DS.%2BD.%26aulast%3Dde%2BVilliers%26aufirst%3DT.%26aulast%3DLing%26aufirst%3DX.%26aulast%3DLippuner%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DReginster%26aufirst%3DJ.-Y.%26aulast%3DRodriguez-Portales%26aufirst%3DJ.%2BA.%26aulast%3DRoux%26aufirst%3DC.%26aulast%3DZanchetta%26aufirst%3DJ.%26aulast%3DZerbini%26aufirst%3DC.%2BA.%2BF.%26aulast%3DPark%26aufirst%3DJ.-G.%26aulast%3DIm%26aufirst%3DK.%26aulast%3DCange%26aufirst%3DA.%26aulast%3DGrip%26aufirst%3DL.%2BT.%26aulast%3DHeyden%26aufirst%3DN.%26aulast%3DDaSilva%26aufirst%3DC.%26aulast%3DCohn%26aufirst%3DD.%26aulast%3DMassaad%26aufirst%3DR.%26aulast%3DScott%26aufirst%3DB.%2BB.%26aulast%3DVerbruggen%26aufirst%3DN.%26aulast%3DGurner%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DD.%2BL.%26aulast%3DBlair%26aufirst%3DM.%2BL.%26aulast%3DPolis%26aufirst%3DA.%2BB.%26aulast%3DStoch%26aufirst%3DS.%2BA.%26aulast%3DSantora%26aufirst%3DA.%26aulast%3DLombardi%26aufirst%3DA.%26aulast%3DLeung%26aufirst%3DA.%2BT.%26aulast%3DKaufman%26aufirst%3DK.%2BD.%26aulast%3DMarc%26aufirst%3DS.%26aulast%3DSabatine%26aufirst%3DM.%2BS.%26atitle%3DOdanacatib%2520for%2520the%2520treatment%2520of%2520postmenopausal%2520osteoporosis%253A%2520results%2520of%2520the%2520LOFT%2520multicentre%252C%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%2520trial%2520and%2520LOFT%2520Extension%2520study%26jtitle%3DLancet%2520Diabetes%2520Endocrinol.%26date%3D2019%26volume%3D7%26issue%3D12%26spage%3D899%26epage%3D911%26doi%3D10.1016%2FS2213-8587%2819%2930346-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petter, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0lhgvSMxC7dVUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span> <span> </span><span class="NLM_article-title">Covalent Inhibition in Drug Discovery</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fcmdc.201900107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=30816012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFensb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=889-906&issue=9&author=A.+K.+Ghoshauthor=I.+Samantaauthor=A.+Mondalauthor=W.+R.+Liu&title=Covalent+Inhibition+in+Drug+Discovery&doi=10.1002%2Fcmdc.201900107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibition in Drug Discovery</span></div><div class="casAuthors">Ghosh, Avick Kumar; Samanta, Indranil; Mondal, Anushree; Liu, Wenshe Ray</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">889-906</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Although covalent inhibitors have been used as therapeutics for more than a century, there has been general resistance in the pharmaceutical industry against their further development due to safety concerns.  This inclination has recently been reverted after the development of a wide variety of covalent inhibitors to address human health conditions along with the US Food and Drug Administration (FDA) approval of several covalent therapeutics for use in humans.  Along with this exciting resurrection of an old drug discovery concept, this review surveys enzymes that can be targeted by covalent inhibitors for the treatment of human diseases.  We focus on protein kinases, RAS proteins, and a few other enzymes that have been studied extensively as targets for covalent inhibition, with the aim to address challenges in designing effective covalent drugs and to provide suggestions in the area that have yet to be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1p5IlI637AbVg90H21EOLACvtfcHk0ljC8i79W-qLFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFensb4%253D&md5=aaa9bd889193c02f8d9acdecaa45e64b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900107%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DSamanta%26aufirst%3DI.%26aulast%3DMondal%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26atitle%3DCovalent%2520Inhibition%2520in%2520Drug%2520Discovery%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26issue%3D9%26spage%3D889%26epage%3D906%26doi%3D10.1002%2Fcmdc.201900107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halford, B.</span></span> <span> </span><span class="NLM_article-title">Covalent drugs go from fringe field to fashionable endeavor</span>. <i>C&EN</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">43</span>),  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.47287/cen-09843-cover</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.47287%2Fcen-09843-cover" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2020&pages=28-33&issue=43&author=B.+Halford&title=Covalent+drugs+go+from+fringe+field+to+fashionable+endeavor&doi=10.47287%2Fcen-09843-cover"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.47287%2Fcen-09843-cover&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.47287%252Fcen-09843-cover%26sid%3Dliteratum%253Aachs%26aulast%3DHalford%26aufirst%3DB.%26atitle%3DCovalent%2520drugs%2520go%2520from%2520fringe%2520field%2520to%2520fashionable%2520endeavor%26jtitle%3DC%2526EN%26date%3D2020%26volume%3D98%26issue%3D43%26spage%3D28%26epage%3D33%26doi%3D10.47287%2Fcen-09843-cover" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallwitz, B.</span></span> <span> </span><span class="NLM_article-title">Clinical Use of DPP-4 Inhibitors</span>. <i>Front. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">389</span>, <span class="refDoi"> DOI: 10.3389/fendo.2019.00389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.3389%2Ffendo.2019.00389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=31275246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A280%3ADC%252BB3MzktlensA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=389&author=B.+Gallwitz&title=Clinical+Use+of+DPP-4+Inhibitors&doi=10.3389%2Ffendo.2019.00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Use of DPP-4 Inhibitors</span></div><div class="casAuthors">Gallwitz Baptist</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">389</span>
        ISSN:<span class="NLM_cas:issn">1664-2392</span>.
    </div><div class="casAbstract">DPP-4 inhibitors were introduced for the treatment of type 2 diabetes in 2006.  They stimulate insulin secretion and inhibit glucagon secretion by elevating endogenous GLP-1 concentrations without an intrinsic hypoglycaemia risk.  Their efficacy potential to lower HbA1c is in the range between 0.5 and 1.0% and their safety profile is favorable.  DPP-4 inhibitors are body weight neutral and they have demonstrated cardiovascular safety.  Most compounds can be used in impaired renal function.  Guidelines suggest the additional use of DPP-4 inhibitors after metformin failure in patients that do not require antidiabetic therapy with proven cardiovascular benefit.  Recently, DPP-4 inhibitors have increasingly replaced sulfonylureas as second line therapy after metformin failure and many metformin/DPP-4 inhibitor fixed dose combinations are available.  In later stages of type 2 diabetes, DPP-4 inhibitors are also recommended in the guidelines in triple therapies with metformin and SGLT-2 inhibitors or with metformin and insulin.  A treatment with DPP-4 inhibitors should be stopped when GLP-1 receptor agonists are used.  DPP-4 inhibitors can be used as monotherapy when metformin is contraindicated or not tolerated.  Some studies have shown value of initial metformin-DPP-4 inhibitor combination therapy in special populations.  This article gives an overview on the clinical use of DPP-4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7l-0AWRpXMoqdXsLcGTEUfW6udTcc2easyQpqPV-O5bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzktlensA%253D%253D&md5=76b007c33dc4180e5f22bbda6b98c25d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3389%2Ffendo.2019.00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffendo.2019.00389%26sid%3Dliteratum%253Aachs%26aulast%3DGallwitz%26aufirst%3DB.%26atitle%3DClinical%2520Use%2520of%2520DPP-4%2520Inhibitors%26jtitle%3DFront.%2520Endocrinol.%26date%3D2019%26volume%3D10%26spage%3D389%26doi%3D10.3389%2Ffendo.2019.00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forman, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinna, A.</span></span> <span> </span><span class="NLM_article-title">Glutathione: Overview of its protective roles, measurement, and biosynthesis</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">1–2</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2008.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.mam.2008.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=18796312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFelurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1-12&issue=1%E2%80%932&author=H.+J.+Formanauthor=H.+Zhangauthor=A.+Rinna&title=Glutathione%3A+Overview+of+its+protective+roles%2C+measurement%2C+and+biosynthesis&doi=10.1016%2Fj.mam.2008.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione: Overview of its protective roles, measurement, and biosynthesis</span></div><div class="casAuthors">Forman, Henry Jay; Zhang, Hongqiao; Rinna, Alessandra</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review is the introduction to a special issue concerning, glutathione (GSH), the most abundant low mol. wt. thiol compd. synthesized in cells.  GSH plays crit. roles in protecting cells from oxidative damage and the toxicity of xenobiotic electrophiles, and maintaining redox homeostasis.  Here, the functions and GSH and the sources of oxidants and electrophiles, the elimination of oxidants by redn. and electrophiles by conjugation with GSH are briefly described.  Methods of assessing GSH status in the cells are also described.  GSH synthesis and its regulation are addressed along with therapeutic approaches for manipulating GSH content that have been proposed.  The purpose here is to provide a brief overview of some of the important aspects of glutathione metab. as part of this special issue that will provide a more comprehensive review of the state of knowledge regarding this essential mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlWapNqV0GqLVg90H21EOLACvtfcHk0ljC8i79W-qLFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFelurY%253D&md5=c1b1aa2b03389bf1434af0762df78fe1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2008.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2008.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DForman%26aufirst%3DH.%2BJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DRinna%26aufirst%3DA.%26atitle%3DGlutathione%253A%2520Overview%2520of%2520its%2520protective%2520roles%252C%2520measurement%252C%2520and%2520biosynthesis%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2009%26volume%3D30%26issue%3D1%25E2%2580%25932%26spage%3D1%26epage%3D12%26doi%3D10.1016%2Fj.mam.2008.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="note"><p class="first last">Reviews of repurposed and targeted peptidomimetics:</p></div><div class="NLM_citation" id="cit25a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Citarella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scala, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piperno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micale, N.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 Mpro: A Potential Target for Peptidomimetics and Small-Molecule Inhibitors</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">607</span>, <span class="refDoi"> DOI: 10.3390/biom11040607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.3390%2Fbiom11040607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33921886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtlSnu7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=607&issue=11&author=A.+Citarellaauthor=A.+Scalaauthor=A.+Pipernoauthor=N.+Micale&title=SARS-CoV-2+Mpro%3A+A+Potential+Target+for+Peptidomimetics+and+Small-Molecule+Inhibitors&doi=10.3390%2Fbiom11040607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25aR"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro: a potential target for peptidomimetics and small-molecule inhibitors</span></div><div class="casAuthors">Citarella, Andrea; Scala, Angela; Piperno, Anna; Micale, Nicola</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">607</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The uncontrolled spread of the COVID-19 pandemic caused by the new coronavirus SARSCoV-2 during 2020-2021 is one of the most devastating events in the history, with remarkable impacts on the health, economic systems, and habits of the entire world population.  While some effective vaccines are nowadays approved and extensively administered, the long-term efficacy and safety of this line of intervention is constantly under debate as coronaviruses rapidly mutate and several SARSCoV-2 variants have been already identified worldwide.  Then, the WHO's main recommendations to prevent severe clin. complications by COVID-19 are still essentially based on social distancing and limitation of human interactions, therefore the identification of new target-based drugs became a priority.  Several strategies have been proposed to counteract such viral infection, including the repurposing of FDA already approved for the treatment of HIV, HCV, and EBOLA, inter alia.  Among the evaluated compds., inhibitors of the main protease of the coronavirus (Mpro) are becoming more and more promising candidates.  Mpro holds a pivotal role during the onset of the infection and its function is intimately related with the beginning of viral replication.  The interruption of its catalytic activity could represent a relevant strategy for the development of anti-coronavirus drugs.  SARS-CoV-2 Mpro is a peculiar cysteine protease of the coronavirus family, responsible for the replication and infectivity of the parasite.  This review offers a detailed anal. of the repurposed drugs and the newly synthesized mols. developed to date for the treatment of COVID-19 which share the common feature of targeting SARS-CoV-2 Mpro, as well as a brief overview of the main enzymic and cell-based assays to efficaciously screen such compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnucMIWuN98rVg90H21EOLACvtfcHk0lh-wcnTrPGJtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtlSnu7bO&md5=2e1820af6f975e353c5f2bcd3c1f31fb</span></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=10.3390%2Fbiom11040607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom11040607%26sid%3Dliteratum%253Aachs%26aulast%3DCitarella%26aufirst%3DA.%26aulast%3DScala%26aufirst%3DA.%26aulast%3DPiperno%26aufirst%3DA.%26aulast%3DMicale%26aufirst%3DN.%26atitle%3DSARS-CoV-2%2520Mpro%253A%2520A%2520Potential%2520Target%2520for%2520Peptidomimetics%2520and%2520Small-Molecule%2520Inhibitors%26jtitle%3DBiomolecules%26date%3D2021%26volume%3D11%26issue%3D11%26spage%3D607%26doi%3D10.3390%2Fbiom11040607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25b"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Yang K and Yang H Recent Progress in the Drug Development Targeting SARS-CoV-2 Main Protease as Treatment for COVID-19</span>. <i>Front. Mol. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">616341</span>, <span class="refDoi"> DOI: 10.3389/fmolb.2020.616341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.3389%2Ffmolb.2020.616341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33344509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXlsVWru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2020&pages=616341&author=W.+Cuiauthor=K.+Yangauthor=H.+Yang&title=Yang+K+and+Yang+H+Recent+Progress+in+the+Drug+Development+Targeting+SARS-CoV-2+Main+Protease+as+Treatment+for+COVID-19&doi=10.3389%2Ffmolb.2020.616341"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25bR"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in the drug development targeting SARS-CoV-2 main protease as treatment for COVID-19</span></div><div class="casAuthors">Cui, Wen; Yang, Kailin; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Molecular Biosciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">616341</span>CODEN:
                <span class="NLM_cas:coden">FMBRBS</span>;
        ISSN:<span class="NLM_cas:issn">2296-889X</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">The sudden outbreak of 2019 novel coronavirus (2019-nCoV, later named SARS-CoV-2) rapidly turned into an unprecedented pandemic of coronavirus disease 2019 (COVID-19).  This global healthcare emergency marked the third occurrence of a deadly coronavirus (CoV) into the human society after entering the new millennium, which overwhelmed the worldwide healthcare system and affected the global economy.  However, therapeutic options for COVID-19 are still very limited.  Developing drugs targeting vital proteins in viral life cycle is a feasible approach to overcome this dilemma.  Main protease (Mpro) plays a dominant role in processing CoV-encoded polyproteins which mediate the assembly of replication-transcription machinery and is thus recognized as an ideal antiviral target.  Here we summarize the recent progress in the discovery of anti-SARS-CoV-2 agents against Mpro.  Combining structural study, virtual screen, and exptl. screen, numerous therapeutic candidates including repurposed drugs and ab initio designed compds. have been proposed.  Such collaborative effort from the scientific community would accelerate the pace of developing efficacious treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptLZWCMNrk9LVg90H21EOLACvtfcHk0lh-wcnTrPGJtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXlsVWru7o%253D&md5=f03e7aed6bbe5cf3e7e57070544ecac2</span></div><a href="/servlet/linkout?suffix=cit25b&amp;dbid=16384&amp;doi=10.3389%2Ffmolb.2020.616341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmolb.2020.616341%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DYang%2520K%2520and%2520Yang%2520H%2520Recent%2520Progress%2520in%2520the%2520Drug%2520Development%2520Targeting%2520SARS-CoV-2%2520Main%2520Protease%2520as%2520Treatment%2520for%2520COVID-19%26jtitle%3DFront.%2520Mol.%2520Biosci.%26date%3D2020%26volume%3D7%26spage%3D616341%26doi%3D10.3389%2Ffmolb.2020.616341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25c"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cannalire, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beccari, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Leva, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summa, V.</span></span> <span> </span><span class="NLM_article-title">Targeting SARS-CoV-2 Proteases and Polymerase for COVID-19 Treatment: State of the Art and Future Opportunities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01140</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=R.+Cannalireauthor=C.+Cerchiaauthor=A.+R.+Beccariauthor=F.+S.+Di+Levaauthor=V.+Summa&title=Targeting+SARS-CoV-2+Proteases+and+Polymerase+for+COVID-19+Treatment%3A+State+of+the+Art+and+Future+Opportunities&doi=10.1021%2Facs.jmedchem.0c01140"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01140%26sid%3Dliteratum%253Aachs%26aulast%3DCannalire%26aufirst%3DR.%26aulast%3DCerchia%26aufirst%3DC.%26aulast%3DBeccari%26aufirst%3DA.%2BR.%26aulast%3DDi%2BLeva%26aufirst%3DF.%2BS.%26aulast%3DSumma%26aufirst%3DV.%26atitle%3DTargeting%2520SARS-CoV-2%2520Proteases%2520and%2520Polymerase%2520for%2520COVID-19%2520Treatment%253A%2520State%2520of%2520the%2520Art%2520and%2520Future%2520Opportunities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26doi%3D10.1021%2Facs.jmedchem.0c01140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manickam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S.-H.</span></span> <span> </span><span class="NLM_article-title">An Overview of Severe Acute Respiratory Syndrome–Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6595</span>– <span class="NLM_lpage">6628</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01461</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01461" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVyns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6595-6628&author=T.+Pillaiyarauthor=M.+Manickamauthor=V.+Namasivayamauthor=Y.+Hayashiauthor=S.-H.+Jung&title=An+Overview+of+Severe+Acute+Respiratory+Syndrome%E2%80%93Coronavirus+%28SARS-CoV%29+3CL+Protease+Inhibitors%3A+Peptidomimetics+and+Small+Molecule+Chemotherapy&doi=10.1021%2Facs.jmedchem.5b01461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Manickam, Manoj; Namasivayam, Vigneshwaran; Hayashi, Yoshio; Jung, Sang-Hun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6595-6628</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome (SARS) is caused by a newly emerged coronavirus that infected more than 8000 individuals and resulted in more than 800 (10-15%) fatalities in 2003.  The causative agent of SARS has been identified as a novel human coronavirus (SARS-CoV), and its viral protease, SARS-CoV 3CLpro, has been shown to be essential for replication and has hence been recognized as a potent drug target for SARS infection.  Currently, there is no effective treatment for this epidemic despite the intensive research that has been undertaken since 2003 (over 3500 publications).  This perspective focuses on the status of various efficacious anti-SARS-CoV 3CLpro chemotherapies discovered during the last 12 years (2003-2015) from all sources, including lab. synthetic methods, natural products, and virtual screening.  We describe here mainly peptidomimetic and small mol. inhibitors of SARS-CoV 3CLpro.  Attempts have been made to provide a complete description of the structural features and binding modes of these inhibitors under many conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB_JCXe-m0RLVg90H21EOLACvtfcHk0lh-wcnTrPGJtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVyns7c%253D&md5=6ce37076204280f64a7f362633fcee1a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01461%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DManickam%26aufirst%3DM.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DJung%26aufirst%3DS.-H.%26atitle%3DAn%2520Overview%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome%25E2%2580%2593Coronavirus%2520%2528SARS-CoV%2529%25203CL%2520Protease%2520Inhibitors%253A%2520Peptidomimetics%2520and%2520Small%2520Molecule%2520Chemotherapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6595%26epage%3D6628%26doi%3D10.1021%2Facs.jmedchem.5b01461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.-M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.-K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-based+design+of+antiviral+drug+candidates+targeting+the+SARS-CoV-2+main+protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0ljoZNQ8ye-wcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.-M.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.-K.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-based%2520design%2520of%2520antiviral%2520drug%2520candidates%2520targeting%2520the%2520SARS-CoV-2%2520main%2520protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Q.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y.-H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 Mpro inhibitors with antiviral activity in a transgenic mouse model</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">1374</span>– <span class="NLM_lpage">1378</span>, <span class="refDoi"> DOI: 10.1126/science.abf1611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1126%2Fscience.abf1611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33602867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2021&pages=1374-1378&author=J.+Qiaoauthor=Y.-S.+Liauthor=R.+Zengauthor=F.-L.+Liuauthor=R.-H.+Luoauthor=C.+Huangauthor=Y.-F.+Wangauthor=J.+Zhangauthor=B.+Quanauthor=C.+Shenauthor=X.+Maoauthor=X.+Liuauthor=W.+Sunauthor=W.+Yangauthor=X.+Niauthor=K.+Wangauthor=L.+Xuauthor=Z.-L.+Duanauthor=Q.-C.+Zouauthor=H.-L.+Zhangauthor=W.+Quauthor=Y.-H.-P.+Longauthor=M.-H.+Liauthor=R.-C.+Yangauthor=X.+Liuauthor=J.+Youauthor=Y.+Zhouauthor=R.+Yaoauthor=W.-P.+Liauthor=J.-M.+Liuauthor=P.+Chenauthor=Y.+Liuauthor=G.-F.+Linauthor=X.+Yangauthor=J.+Zouauthor=L.+Liauthor=Y.+Huauthor=G.-W.+Luauthor=W.-M.+Liauthor=Y.-Q.+Weiauthor=Y.-T.+Zhengauthor=J.+Leiauthor=S.+Yang&title=SARS-CoV-2+Mpro+inhibitors+with+antiviral+activity+in+a+transgenic+mouse+model&doi=10.1126%2Fscience.abf1611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro inhibitors with antiviral activity in a transgenic mouse model</span></div><div class="casAuthors">Qiao, Jingxin; Li, Yue-Shan; Zeng, Rui; Liu, Feng-Liang; Luo, Rong-Hua; Huang, Chong; Wang, Yi-Fei; Zhang, Jie; Quan, Baoxue; Shen, Chenjian; Mao, Xin; Liu, Xinlei; Sun, Weining; Yang, Wei; Ni, Xincheng; Wang, Kai; Xu, Ling; Duan, Zi-Lei; Zou, Qing-Cui; Zhang, Hai-Lin; Qu, Wang; Long, Yang-Hao-Peng; Li, Ming-Hua; Yang, Rui-Cheng; Liu, Xiaolong; You, Jing; Zhou, Yangli; Yao, Rui; Li, Wen-Pei; Liu, Jing-Ming; Chen, Pei; Liu, Yang; Lin, Gui-Feng; Yang, Xin; Zou, Jun; Li, Linli; Hu, Yiguo; Lu, Guang-Wen; Li, Wei-Min; Wei, Yu-Quan; Zheng, Yong-Tang; Lei, Jian; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6536</span>),
    <span class="NLM_cas:pages">1374-1378</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continually poses serious threats to global public health.  The main protease (Mpro) of SARS-CoV-2 plays a central role in viral replication.  We designed and synthesized 32 new bicycloproline-contg. Mpro inhibitors derived from either boceprevir or telaprevir, both of which are approved antivirals.  All compds. inhibited SARS-CoV-2 Mpro activity in vitro, with 50% inhibitory concn. values ranging from 7.6 to 748.5 nM.  The cocrystal structure of Mpro in complex with MI-23, one of the most potent compds., revealed its interaction mode.  Two compds. (MI-09 and MI-30) showed excellent antiviral activity in cell-based assays.  In a transgenic mouse model of SARS-CoV-2 infection, oral or i.p. treatment with MI-09 or MI-30 significantly reduced lung viral loads and lung lesions.  Both also displayed good pharmacokinetic properties and safety in rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVMv8s8s-oWLVg90H21EOLACvtfcHk0ljoZNQ8ye-wcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D&md5=555937b1776c8353e59601d567e2ebb9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fscience.abf1611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abf1611%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.-S.%26aulast%3DZeng%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DF.-L.%26aulast%3DLuo%26aufirst%3DR.-H.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DQuan%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DDuan%26aufirst%3DZ.-L.%26aulast%3DZou%26aufirst%3DQ.-C.%26aulast%3DZhang%26aufirst%3DH.-L.%26aulast%3DQu%26aufirst%3DW.%26aulast%3DLong%26aufirst%3DY.-H.-P.%26aulast%3DLi%26aufirst%3DM.-H.%26aulast%3DYang%26aufirst%3DR.-C.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DW.-P.%26aulast%3DLiu%26aufirst%3DJ.-M.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DG.-F.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DG.-W.%26aulast%3DLi%26aufirst%3DW.-M.%26aulast%3DWei%26aufirst%3DY.-Q.%26aulast%3DZheng%26aufirst%3DY.-T.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DSARS-CoV-2%2520Mpro%2520inhibitors%2520with%2520antiviral%2520activity%2520in%2520a%2520transgenic%2520mouse%2520model%26jtitle%3DScience%26date%3D2021%26volume%3D371%26spage%3D1374%26epage%3D1378%26doi%3D10.1126%2Fscience.abf1611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+%CE%B1-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0lgEa3Vang2CLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520%25CE%25B1-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2223-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fs41586-020-2223-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32272481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=289-293&author=Z.+Jinauthor=X.+Duauthor=Y.+Xuauthor=Y.+Dengauthor=L.+Liuauthor=Y.+Zhaoauthor=B.+Zhangauthor=X.+Liauthor=L.+Zhangauthor=C.+Pengauthor=Y.+Duanauthor=J.+Yuauthor=L.+Wangauthor=K.+Yangauthor=F.+Liuauthor=R.+Jiangauthor=X.+Yangauthor=T.+Youauthor=X.+Liuauthor=X.+Yangauthor=F.+Baiauthor=H.+Liuauthor=X.+Liuauthor=L.+W.+Guddatauthor=W.+Xuauthor=G.+Xiaoauthor=C.+Qinauthor=Z.+Shiauthor=H.+Jiangauthor=Z.+Raoauthor=H.+Yang&title=Structure+of+Mpro+from+SARS-CoV-2+and+discovery+of+its+inhibitors&doi=10.1038%2Fs41586-020-2223-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span></div><div class="casAuthors">Jin, Zhenming; Du, Xiaoyu; Xu, Yechun; Deng, Yongqiang; Liu, Meiqin; Zhao, Yao; Zhang, Bing; Li, Xiaofeng; Zhang, Leike; Peng, Chao; Duan, Yinkai; Yu, Jing; Wang, Lin; Yang, Kailin; Liu, Fengjiang; Jiang, Rendi; Yang, Xinglou; You, Tian; Liu, Xiaoce; Yang, Xiuna; Bai, Fang; Liu, Hong; Liu, Xiang; Guddat, Luke W.; Xu, Wenqing; Xiao, Gengfu; Qin, Chengfeng; Shi, Zhengli; Jiang, Hualiang; Rao, Zihe; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7811</span>),
    <span class="NLM_cas:pages">289-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A new coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is the etiol. agent responsible for the 2019-2020 viral pneumonia outbreak of coronavirus disease 2019 (COVID-19).  Currently, there are no targeted therapeutic agents for the treatment of this disease, and effective treatment options remain very limited.  Here, we describe the results of a program that aimed to rapidly discover lead compds. for clin. use, by combining structure-assisted drug design, virtual drug screening and high-throughput screening.  This program focused on identifying drug leads that target main protease (Mpro) of SARS-CoV-2: Mpro is a key enzyme of coronaviruses and has a pivotal role in mediating viral replication and transcription, making it an attractive drug target for SARS-CoV-2.  We identified a mechanism-based inhibitor (N3) by computer-aided drug design, and then detd. the crystal structure of Mpro of SARS-CoV-2 in complex with this compd.  Through a combination of structure-based virtual and high-throughput screening, we assayed more than 10,000 compds.-including approved drugs, drug candidates in clin. trials and other pharmacol. active compds.-as inhibitors of Mpro.  Six of these compds. inhibited Mpro, showing half-maximal inhibitory concn. values that ranged from 0.67 to 21.4μM.  One of these compds. (ebselen) also exhibited promising antiviral activity in cell-based assays.  Our results demonstrate the efficacy of our screening strategy, which can lead to the rapid discovery of drug leads with clin. potential in response to new infectious diseases for which no specific drugs or vaccines are available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFupuh6WIyW7Vg90H21EOLACvtfcHk0lgEa3Vang2CLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO&md5=b84f350fe9ce1109485df6caf814ba82</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2223-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2223-y%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DStructure%2520of%2520Mpro%2520from%2520SARS-CoV-2%2520and%2520discovery%2520of%2520its%2520inhibitors%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D289%26epage%3D293%26doi%3D10.1038%2Fs41586-020-2223-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copp, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauls, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span> <span> </span><span class="NLM_article-title">New Inhibitors of Cysteine Proteinases. Peptidyl Acyloxymethylketones and the Quiescent Nucleofuge Strategy</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">4429</span>– <span class="NLM_lpage">4431</span>, <span class="refDoi"> DOI: 10.1021/ja00221a062</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00221a062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaL1cXktlCguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=4429-4431&author=R.+A.+Smithauthor=L.+J.+Coppauthor=P.+J.+Colesauthor=H.+W.+Paulsauthor=V.+J.+Robinsonauthor=R.+W.+Spencerauthor=S.+B.+Heardauthor=A.+Krantz&title=New+Inhibitors+of+Cysteine+Proteinases.+Peptidyl+Acyloxymethylketones+and+the+Quiescent+Nucleofuge+Strategy&doi=10.1021%2Fja00221a062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31aR"><div class="casContent"><span class="casTitleNuber">31a</span><div class="casTitle"><span class="NLM_cas:atitle">New inhibitors of cysteine proteinases.  Peptidyl acyloxymethyl ketones and the quiescent nucleofuge strategy</span></div><div class="casAuthors">Smith, Roger A.; Copp, Leslie J.; Coles, Peter J.; Pauls, Henry W.; Robinson, Valerie J.; Spencer, Robin W.; Heard, Stephen B.; Krantz, Allen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4429-31</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Peptidyl acyloxymethyl ketones are potent and specific inhibitors of the cysteine proteinase, cathepsin B.  These inhibitors are affinity labels in which the peptide moiety serves to transport an aryl carboxylate nucleofuge that is uniquely reactive toward the active-site cysteine thiol of the target enzyme.  Heretofore, in constructing affinity labels the choice of leaving group was severely restricted.  The aryl carboxylate group offers considerable variation as a design element in that its binding affinity and reactivity can be controlled by substituent effects.  The apparent 2nd-order rate const. which characterizes the inhibition of bovine spleen cathepsin B by this series spans several orders of magnitude and in certain cases exceeds 106 M-1 s-1.  The inhibition is time-dependent, active-site directed, and irreversible.  The inactivation requires that the peptide component contain high affinity recognition elements for this enzyme and that the aryl carboxylate leaving group has a pKa <4.  Impressive inhibitory activity is also obsd. for certain peptidyl aryloxymethyl ketones.  NMR resonances of the inactivated adduct resulting from the action of representative 13C-labeled acyloxymethyl and aryloxymethyl ketone inhibitors on the model cysteine proteinase papain are consistent with the proposed mechanism of inactivation, involving displacement of the nucleofuge by the active-site cysteine to give a (cysteine-25)-thiomethyl ketone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0tvSCUqAffrVg90H21EOLACvtfcHk0lgEa3Vang2CLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktlCguro%253D&md5=12b38defc0f9731ff6adeebac78b0711</span></div><a href="/servlet/linkout?suffix=cit31a&amp;dbid=16384&amp;doi=10.1021%2Fja00221a062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00221a062%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DCopp%26aufirst%3DL.%2BJ.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DPauls%26aufirst%3DH.%2BW.%26aulast%3DRobinson%26aufirst%3DV.%2BJ.%26aulast%3DSpencer%26aufirst%3DR.%2BW.%26aulast%3DHeard%26aufirst%3DS.%2BB.%26aulast%3DKrantz%26aufirst%3DA.%26atitle%3DNew%2520Inhibitors%2520of%2520Cysteine%2520Proteinases.%2520Peptidyl%2520Acyloxymethylketones%2520and%2520the%2520Quiescent%2520Nucleofuge%2520Strategy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D4429%26epage%3D4431%26doi%3D10.1021%2Fja00221a062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit31b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copp, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, S. B.</span></span> <span> </span><span class="NLM_article-title">Peptidyl (Acyloxy)methyl Ketones and the Quiescent Affinity Label Concept: The Departing Group as a Variable Structure Element in the Design of Inactivators of Cysteine Proteinases</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">4678</span>– <span class="NLM_lpage">4687</span>, <span class="refDoi"> DOI: 10.1021/bi00233a007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00233a007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK3MXit1CgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1991&pages=4678-4687&author=A.+Krantzauthor=L.+J.+Coppauthor=P.+J.+Colesauthor=R.+A.+Smithauthor=S.+B.+Heard&title=Peptidyl+%28Acyloxy%29methyl+Ketones+and+the+Quiescent+Affinity+Label+Concept%3A+The+Departing+Group+as+a+Variable+Structure+Element+in+the+Design+of+Inactivators+of+Cysteine+Proteinases&doi=10.1021%2Fbi00233a007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31bR"><div class="casContent"><span class="casTitleNuber">31b</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl (acyloxy)methyl ketones and the quiescent affinity label concept:  the departing group as a variable structural element in the design of inactivators of cysteine proteinases</span></div><div class="casAuthors">Krantz, Allen; Copp, Leslie J.; Coles, Peter J.; Smith, Roger A.; Heard, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4678-87</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">(Acyloxy)methyl ketones, of general structure Z-[AA2]-[AA1]-CH2OCOAr (Z = benzyloxycarbonyl; AA = amino acid; Ar = aryl), were potent inactivators of the cysteine proteinase, cathepsin B.  These reagents were designed as affinity labels in which the dipeptidyl moiety serves as an affinity group (complementary to the S1 and S2 sites of the enzyme), while the (acyloxy)methyl ketone unit (-COCH2OCOR), contg. a weak leaving group in the form of a carboxylate nucleofuge, functions as the potentially reactive entity that labels the enzyme.  The inhibition was time-dependent, active-site directed, and irreversible.  The apparent 2nd-order rate const. (kinact/Kinact), which characterizes the inhibition of cathepsin B by this series, spanned several orders of magnitude and in certain cases exceeded 106 M-1 s-1.  The activity of this series of inhibitors was found to be exquisitely sensitive to the nature of the carboxylate leaving group as well as the affinity group.  A strong dependence of 2nd-order inactivation rate on leaving group pKa was uncovered for Z-Phe-Ala (acyloxy)methyl ketones [log(k/K) = -1.1 × pKa + 7.2.  Heretofore, in constructing affinity labels the choice of leaving group was quite restricted.  The aryl carboxylate group thus offers considerable variation as a design element in that both its binding affinity and reactivity can be controlled by substituent effects.  Specific peptidyl (acyloxy)methyl ketones thus represent prime examples of highly potent, chem. stable enzyme inhibitors with variable structural elements in both the affinity and departing groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprQo4CgIuJM7Vg90H21EOLACvtfcHk0lj9WyxlaQ8RGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXit1CgsLY%253D&md5=887882366c35b5cba8021d1d14eb47be</span></div><a href="/servlet/linkout?suffix=cit31b&amp;dbid=16384&amp;doi=10.1021%2Fbi00233a007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00233a007%26sid%3Dliteratum%253Aachs%26aulast%3DKrantz%26aufirst%3DA.%26aulast%3DCopp%26aufirst%3DL.%2BJ.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DHeard%26aufirst%3DS.%2BB.%26atitle%3DPeptidyl%2520%2528Acyloxy%2529methyl%2520Ketones%2520and%2520the%2520Quiescent%2520Affinity%2520Label%2520Concept%253A%2520The%2520Departing%2520Group%2520as%2520a%2520Variable%2520Structure%2520Element%2520in%2520the%2520Design%2520of%2520Inactivators%2520of%2520Cysteine%2520Proteinases%26jtitle%3DBiochemistry%26date%3D1991%26volume%3D30%26spage%3D4678%26epage%3D4687%26doi%3D10.1021%2Fbi00233a007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit31c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauls, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span> <span> </span><span class="NLM_article-title">Carbon-13 NMR characterization of the Papain Adduct Formed by Peptidyl Acyloxy-, Aryloxy, and Chloromethyl Ketone Irrversible Inhibitors</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/0045-2068(92)90024-W</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2F0045-2068%2892%2990024-W" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK38Xkt1agsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1992&pages=42-54&author=V.+J.+Robinsonauthor=H.+W.+Paulsauthor=P.+J.+Colesauthor=R.+A.+Smithauthor=A.+Krantz&title=Carbon-13+NMR+characterization+of+the+Papain+Adduct+Formed+by+Peptidyl+Acyloxy-%2C+Aryloxy%2C+and+Chloromethyl+Ketone+Irrversible+Inhibitors&doi=10.1016%2F0045-2068%2892%2990024-W"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31cR"><div class="casContent"><span class="casTitleNuber">31c</span><div class="casTitle"><span class="NLM_cas:atitle">Carbon-13 NMR characterization of the papain adduct formed by peptidyl acyloxy-, aryloxy-, and chloromethyl ketone irreversible inhibitors</span></div><div class="casAuthors">Robinson, Valerie J.; Pauls, Henry W.; Coles, Peter J.; Smith, Roger A.; Krantz, Allen</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-54</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    </div><div class="casAbstract">N-Benzoyloxycarbonyl-L-phenylalanyl-glycine-derived mesitoyloxy- and pentafluorophenoxymethyl ketones, incorporating 13C labels at the ketone carbonyl and α-keto methylene positions, were synthesized.  These compds. act as quiescent affinity labels of cysteine proteinases.  NMR studies established that these compds. form an irreversible covalent adduct with papain, via release of mesitoic acid or pentafluorophenol.  The same adduct was formed in each case, with 13C NMR signals at 214.7 ppm (COCH2) and 38.1 ppm (COCH2).  Identical spectra were obsd. for the adduct formed from the analogous 13C-labeled chloromethyl ketones; the structure of this type of adduct has been previously established by x-ray studies.  The NMR data are consistent with this structure, and provide a sensitive measure of the hydrogen bonding to the π-carbonyl.  Disruption of this hydrogen bonding by denaturation produced a 13C NMR shift of 10 ppm to higher field in the carbonyl carbon, to a chem. shift in accordance with that of a model thiomethyl ketone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqons7A9nr4NbVg90H21EOLACvtfcHk0lj9WyxlaQ8RGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xkt1agsrw%253D&md5=691e9b356f4262b75cdcb4042a583792</span></div><a href="/servlet/linkout?suffix=cit31c&amp;dbid=16384&amp;doi=10.1016%2F0045-2068%2892%2990024-W&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0045-2068%252892%252990024-W%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DV.%2BJ.%26aulast%3DPauls%26aufirst%3DH.%2BW.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DKrantz%26aufirst%3DA.%26atitle%3DCarbon-13%2520NMR%2520characterization%2520of%2520the%2520Papain%2520Adduct%2520Formed%2520by%2520Peptidyl%2520Acyloxy-%252C%2520Aryloxy%252C%2520and%2520Chloromethyl%2520Ketone%2520Irrversible%2520Inhibitors%26jtitle%3DBioorg.%2520Chem.%26date%3D1992%26volume%3D20%26spage%3D42%26epage%3D54%26doi%3D10.1016%2F0045-2068%2892%2990024-W" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yeo, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, S. C.</span></span> <span> </span><span class="NLM_article-title">The aminopeptidase inhibitor, z-CMK, is toxic and induces cell death in Jurkat T cells through oxidative stress</span>. <i>Toxicol. Mech. Methods</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1080/15376516.2017.1373882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1080%2F15376516.2017.1373882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=28849708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFWgsL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=157-166&issue=3&author=E.+H.+Yeoauthor=W.+L.+Gohauthor=S.+C.+Chow&title=The+aminopeptidase+inhibitor%2C+z-CMK%2C+is+toxic+and+induces+cell+death+in+Jurkat+T+cells+through+oxidative+stress&doi=10.1080%2F15376516.2017.1373882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The aminopeptidase inhibitor, z-L-CMK, is toxic and induces cell death in Jurkat T cells through oxidative stress</span></div><div class="casAuthors">Yeo, E. H.; Goh, W. L.; Chow, S. C.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Mechanisms and Methods</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">157-166</span>CODEN:
                <span class="NLM_cas:coden">TMMOCP</span>;
        ISSN:<span class="NLM_cas:issn">1537-6516</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The leucine aminopeptidase inhibitor, benzyloxycarbonyl-leucine-chloromethylketone (z-L-CMK), was found to be toxic and readily induce cell death in Jurkat T cells.  Dose-response studies show that lower concn. of z-L-CMK induced apoptosis in Jurkat T cells whereas higher concn. causes necrosis.  In z-L-CMK-induced apoptosis, both the initiator caspases (-8 and -9) and effector caspases (-3 and -6) were processed to their resp. subunits.  However, the caspases remained intact in z-L-CMK-induced necrosis.  The caspase inhibitor, z-VAD-FMK inhibited z-L-CMK-mediated apoptosis and caspase processing but has no effect on z-L-CMK-induced necrosis in Jurkat T cells.  The high mobility group protein B1 (HMGB1) protein was found to be released into the culture medium by the necrotic cells and not the apoptotic cells.  These results indicate that the necrotic cell death mediated by z-L-CMK at high concns. is via classical necrosis rather than secondary necrosis.  We also demonstrated that cell death mediated by z-L-CMK was assocd. with oxidative stress via the depletion of intracellular glutathione (GSH) and increase in reactive oxygen species (ROS), which was blocked by N-acetyl cysteine.  Taken together, the results demonstrated that z-L-CMK is toxic to Jurkat T cells and induces apoptosis at low concns., while at higher concns. the cells die of necrosis.  The toxic side effects in Jurkat T cells mediated by z-L-CMK are assocd. with oxidative stress via the depletion of GSH and accumulation of ROS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogQL4jAZ8JlbVg90H21EOLACvtfcHk0lj9WyxlaQ8RGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWgsL7P&md5=7b423409408370bd1a7dceffefae5a3b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1080%2F15376516.2017.1373882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15376516.2017.1373882%26sid%3Dliteratum%253Aachs%26aulast%3DYeo%26aufirst%3DE.%2BH.%26aulast%3DGoh%26aufirst%3DW.%2BL.%26aulast%3DChow%26aufirst%3DS.%2BC.%26atitle%3DThe%2520aminopeptidase%2520inhibitor%252C%2520z-CMK%252C%2520is%2520toxic%2520and%2520induces%2520cell%2520death%2520in%2520Jurkat%2520T%2520cells%2520through%2520oxidative%2520stress%26jtitle%3DToxicol.%2520Mech.%2520Methods%26date%3D2018%26volume%3D28%26issue%3D3%26spage%3D157%26epage%3D166%26doi%3D10.1080%2F15376516.2017.1373882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eichhold, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookfin, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taiwo, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehmeyer, K. R.</span></span> <span> </span><span class="NLM_article-title">Isolation and quantification of fluoroacetate in rat tissues, following dosing of Z-Phe-Ala-CH2-F, a peptidyl fluoromethyl ketone protease inhibitor</span>. <i>J. Pharm. Biomed. Anal.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1016/S0731-7085(97)00102-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2FS0731-7085%2897%2900102-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=9589405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK2sXnvFWqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1997&pages=459-467&issue=3&author=T.+H.+Eichholdauthor=E.+B.+Hookfinauthor=Y.+O.+Taiwoauthor=B.+Deauthor=K.+R.+Wehmeyer&title=Isolation+and+quantification+of+fluoroacetate+in+rat+tissues%2C+following+dosing+of+Z-Phe-Ala-CH2-F%2C+a+peptidyl+fluoromethyl+ketone+protease+inhibitor&doi=10.1016%2FS0731-7085%2897%2900102-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation and quantification of fluoroacetate in rat tissues, following dosing of Z-Phe-Ala-CH2-F, a peptidyl fluoromethyl ketone protease inhibitor</span></div><div class="casAuthors">Eichhold, Thomas H.; Hookfin, Erin B.; Taiwo, Yetunde O.; De, Biswanath; Wehmeyer, Kenneth R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">459-467</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Peptidyl fluoromethyl ketones (PFMK) are irreversible inhibitors of cathepsin B, a cysteine proteinase thought to be involved in the degrdn. of cartilage.  It has been speculated that PFMK inhibitors may metabolize in rodents to form fluoroacetate (FAC), an extremely toxic poison.  A highly selective and sensitive sepn. and detection scheme was developed to measure trace levels of FAC in rat tissues following PFMK dosing.  The procedure consisted of extg. FAC from tissue and spiking the ext. with [18O]2-fluoroacetate (18O-FAC) as an internal std.  FAC and 18O-FAC were further isolated from matrix components using ion-exchange, solid-phase extn.  The pentafluorobenzyl esters of FAC and 18O-FAC were formed to facilitate the chromatog. sepn.  Two-dimensional gas chromatog. coupled with selected-ion-monitoring detection provided the final measurement.  The assay had a limit of detection of 2 ng FAC per g tissue, and was capable of accurately quantitating as little as 10 ng FAC per g tissue with a S/N ratio of 40:1.  Linearity was established over two orders of magnitude, from 2-500 ng ml-1, with 5 μl injected on-column.  The method was used to demonstrate that FAC was formed in rats following dosing with Z-Phe-Ala-CH2-F, a PFMK cathepsin enzyme inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPmtRizUZf2bVg90H21EOLACvtfcHk0lidDjGoO38Kng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnvFWqsL8%253D&md5=ea69dcdcaff325ed19d40e94c6f96760</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0731-7085%2897%2900102-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0731-7085%252897%252900102-7%26sid%3Dliteratum%253Aachs%26aulast%3DEichhold%26aufirst%3DT.%2BH.%26aulast%3DHookfin%26aufirst%3DE.%2BB.%26aulast%3DTaiwo%26aufirst%3DY.%2BO.%26aulast%3DDe%26aufirst%3DB.%26aulast%3DWehmeyer%26aufirst%3DK.%2BR.%26atitle%3DIsolation%2520and%2520quantification%2520of%2520fluoroacetate%2520in%2520rat%2520tissues%252C%2520following%2520dosing%2520of%2520Z-Phe-Ala-CH2-F%252C%2520a%2520peptidyl%2520fluoromethyl%2520ketone%2520protease%2520inhibitor%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D1997%26volume%3D16%26issue%3D3%26spage%3D459%26epage%3D467%26doi%3D10.1016%2FS0731-7085%2897%2900102-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolle, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whipple, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osifo, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graybill, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helaszek, C. T.</span></span> <span> </span><span class="NLM_article-title">Aspartyl α-((Diphenylphosphinyl)oxy) methyl Ketones as Novel Inhibitors of Interleukin-1β Converting Enzyme. Utility of the Dephenylphosphinic Acid Leaving Group for the Inhibition of Cysteine Proteases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1021/jm00002a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00002a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK2MXivFOqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1995&pages=220-222&issue=38&author=R.+E.+Dolleauthor=J.+Singhauthor=D.+Whippleauthor=I.+K.+Osifoauthor=G.+Speierauthor=T.+L.+Graybillauthor=J.+S.+Gregoryauthor=A.+L.+Harrisauthor=C.+T.+Helaszek&title=Aspartyl+%CE%B1-%28%28Diphenylphosphinyl%29oxy%29+methyl+Ketones+as+Novel+Inhibitors+of+Interleukin-1%CE%B2+Converting+Enzyme.+Utility+of+the+Dephenylphosphinic+Acid+Leaving+Group+for+the+Inhibition+of+Cysteine+Proteases&doi=10.1021%2Fjm00002a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34aR"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Aspartyl α-((Diphenylphosphinyl)oxy)methyl Ketones as Novel Inhibitors of Interleukin-1β Converting Enzyme. Utility of the Diphenylphosphinic Acid Leaving Group for the Inhibition of Cysteine Proteases</span></div><div class="casAuthors">Dolle, Roland E.; Singh, Jasbir; Whipple, David; Osifo, I. Kelly; Speier, Gary; Graybill, Todd L.; Gregory, Jill S.; Harris, Alex L.; Helaszek, Carla T.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">220-2</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mono-, di- and tripeptide aspartyl α-((diphenylphosphinyl)oxy)methyl ketones are potent time-dependent inactivators of interleukin-1β converting enzyme (ICE).  Z-Val-Ala-Asp-CH2OP(O)Ph2 exhibits a second order rate const. (kobs/[I]) of 117,000 M-1 s-1.  By incorporating the (diphenylphosphinyl)oxy- (DPP) leaving group into the peptide scaffolds Z-Phe-Ala and Z-Leu-Phe, potent inhibitors of cathepsin B and calpain I were identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-O3lqW_Qo77Vg90H21EOLACvtfcHk0lidDjGoO38Kng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXivFOqtbo%253D&md5=ab67d2a8cb7e70900b8bf2f0cfd3a494</span></div><a href="/servlet/linkout?suffix=cit34a&amp;dbid=16384&amp;doi=10.1021%2Fjm00002a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00002a002%26sid%3Dliteratum%253Aachs%26aulast%3DDolle%26aufirst%3DR.%2BE.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DWhipple%26aufirst%3DD.%26aulast%3DOsifo%26aufirst%3DI.%2BK.%26aulast%3DSpeier%26aufirst%3DG.%26aulast%3DGraybill%26aufirst%3DT.%2BL.%26aulast%3DGregory%26aufirst%3DJ.%2BS.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DHelaszek%26aufirst%3DC.%2BT.%26atitle%3DAspartyl%2520%25CE%25B1-%2528%2528Diphenylphosphinyl%2529oxy%2529%2520methyl%2520Ketones%2520as%2520Novel%2520Inhibitors%2520of%2520Interleukin-1%25CE%25B2%2520Converting%2520Enzyme.%2520Utility%2520of%2520the%2520Dephenylphosphinic%2520Acid%2520Leaving%2520Group%2520for%2520the%2520Inhibition%2520of%2520Cysteine%2520Proteases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1995%26volume%3D38%26issue%3D38%26spage%3D220%26epage%3D222%26doi%3D10.1021%2Fjm00002a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit34b"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Graybill, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prouty, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speier, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolle, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helaszek, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ator, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strastes, J.</span></span> <span> </span><span class="NLM_article-title">α-((Tetronoyl)oxy)- and α-((Tetramoyl)oxy)methyl ketone inhibitors of the interleukin-1β converting enzyme (ICE)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(96)00574-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2FS0960-894X%2896%2900574-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK2sXls1eiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1997&pages=41-46&issue=1&author=T.+L.+Graybillauthor=C.+P.+Proutyauthor=G.+J.+Speierauthor=D.+Hoyerauthor=R.+E.+Dolleauthor=C.+T.+Helaszekauthor=M.+A.+Atorauthor=J.+Uhlauthor=J.+Strastes&title=%CE%B1-%28%28Tetronoyl%29oxy%29-+and+%CE%B1-%28%28Tetramoyl%29oxy%29methyl+ketone+inhibitors+of+the+interleukin-1%CE%B2+converting+enzyme+%28ICE%29&doi=10.1016%2FS0960-894X%2896%2900574-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34bR"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">α-[(Tetronoyl)oxy]- and α-[(tetramoyl)oxy]methyl ketone inhibitors of the interleukin-1β converting enzyme (ICE)</span></div><div class="casAuthors">Graybill, Todd L.; Prouty, Catherine P.; Speier, Gary J.; Hoyer, Denton; Dolle, Roland E.; Helaszek, Carla T.; Ator, Mark A.; Uhl, Joanne; Strasters, Joost</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-46</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Aryl-substituted tetronic acids, tetramic acids, and cyclic β-dicarbonyl moieties were evaluated as leaving groups in the peptidyl-COCH2-X type inhibitor.  Tripeptidyl aspartyl α-[(tetronyl)oxy]- and α-[(tetramoyl)oxy]methyl ketone derivs. demonstrate potent time-dependent inhibition (kobs/[I] 100,000-250,000 M-1s-1) of the cysteine protease ICE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQwc1Cqp_ESrVg90H21EOLACvtfcHk0lidDjGoO38Kng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXls1eiuw%253D%253D&md5=65fb1f34ab92351433675e2462c3a394</span></div><a href="/servlet/linkout?suffix=cit34b&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2896%2900574-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252896%252900574-4%26sid%3Dliteratum%253Aachs%26aulast%3DGraybill%26aufirst%3DT.%2BL.%26aulast%3DProuty%26aufirst%3DC.%2BP.%26aulast%3DSpeier%26aufirst%3DG.%2BJ.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DDolle%26aufirst%3DR.%2BE.%26aulast%3DHelaszek%26aufirst%3DC.%2BT.%26aulast%3DAtor%26aufirst%3DM.%2BA.%26aulast%3DUhl%26aufirst%3DJ.%26aulast%3DStrastes%26aufirst%3DJ.%26atitle%3D%25CE%25B1-%2528%2528Tetronoyl%2529oxy%2529-%2520and%2520%25CE%25B1-%2528%2528Tetramoyl%2529oxy%2529methyl%2520ketone%2520inhibitors%2520of%2520the%2520interleukin-1%25CE%25B2%2520converting%2520enzyme%2520%2528ICE%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1997%26volume%3D7%26issue%3D1%26spage%3D41%26epage%3D46%26doi%3D10.1016%2FS0960-894X%2896%2900574-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abeles, R. H.</span></span> <span> </span><span class="NLM_article-title">Inactivation of Cysteine Proteases by (Acyloxy)methyl Ketones Using S′-P′ Interactions</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">6498</span>– <span class="NLM_lpage">6502</span>, <span class="refDoi"> DOI: 10.1021/bi0002378</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0002378" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD3cXivVWhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=6498-6502&author=Y.+Daiauthor=L.+Hedstromauthor=R.+H.+Abeles&title=Inactivation+of+Cysteine+Proteases+by+%28Acyloxy%29methyl+Ketones+Using+S%E2%80%B2-P%E2%80%B2+Interactions&doi=10.1021%2Fbi0002378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Inactivation of Cysteine Proteases by (Acyloxy)methyl Ketones Using S'-P' Interactions</span></div><div class="casAuthors">Dai, Yong; Hedstrom, Lizbeth; Abeles, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6498-6502</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have synthesized (acyloxy)methyl ketone inactivators of papain, cathepsin B, and interleukin-1β conversion enzyme (ICE) that interact with both the S and S' sub-sites.  The value of kinact/Ki for these inactivators is strongly dependent on the leaving group.  For example, Z-Phe-Gly-CH2-X is a poor inactivator of papain when X is OCOCH3 (kinact/Ki = 2.5 M-1 s-1) but becomes a potent inactivator when X is OCO-L-Leu-Z (kinact/Ki = 11 000 M-1 s-1).  Since these leaving groups have similar chem. reactivities, the difference in potency must be attributed to interactions with the S' sites.  The potency of the leaving group correlates with the P' specificity of papain.  Similar results are also obsd. for the inactivation of cathepsin B by these compds.  A series of inactivators with the general structure Fmoc-L-Asp-CH2-X were designed to inactivate ICE.  No inhibition was obsd. when X was OCOCH3.  In contrast, ICE is inactivated when X is OCO-D-Pro-Z (kinact/Ki = 131 M-1 s-1).  These results demonstrate that S'-P' interactions can be utilized to increase the efficacy and selectivity of (acyloxy)methyl ketone inactivators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6mPpAnfktvrVg90H21EOLACvtfcHk0lhKWrelGjMs5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXivVWhs70%253D&md5=807bf12e74f3f5fd799eaf9f97e89a06</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi0002378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0002378%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DY.%26aulast%3DHedstrom%26aufirst%3DL.%26aulast%3DAbeles%26aufirst%3DR.%2BH.%26atitle%3DInactivation%2520of%2520Cysteine%2520Proteases%2520by%2520%2528Acyloxy%2529methyl%2520Ketones%2520Using%2520S%25E2%2580%25B2-P%25E2%2580%25B2%2520Interactions%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D6498%26epage%3D6502%26doi%3D10.1021%2Fbi0002378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
de Plassche, M. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barniol-Xicota, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhelst, S. H. L.</span></span> <span> </span><span class="NLM_article-title">Peptidyl Acyloxymethyl Ketones as Activity-Based Probes for the Main Protease of SARS-CoV-2</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3383</span>– <span class="NLM_lpage">3388</span>, <span class="refDoi"> DOI: 10.1002/cbic.202000371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fcbic.202000371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32717117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslKgsLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&pages=3383-3388&author=M.+A.+T.+van%0Ade+Plasscheauthor=M.+Barniol-Xicotaauthor=S.+H.+L.+Verhelst&title=Peptidyl+Acyloxymethyl+Ketones+as+Activity-Based+Probes+for+the+Main+Protease+of+SARS-CoV-2&doi=10.1002%2Fcbic.202000371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl Acyloxymethyl Ketones as Activity-Based Probes for the Main Protease of SARS-CoV-2</span></div><div class="casAuthors">van de Plassche, Merel A. T.; Barniol-Xicota, Marta; Verhelst, Steven H. L.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3383-3388</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The global pandemic caused by SARS-Co-V-2 calls for the fast development of antiviral drugs against this particular coronavirus.  Chem. tools to facilitate inhibitor discovery as well as detection of target engagement by hit or lead compds. from high-throughput screens are therefore in urgent need.  The authors here report novel, selective activity-based probes that enable detection of the SARS-Co-V-2 main protease.  The probes are based on acyloxymethyl ketone reactive electrophiles combined with a peptide sequence including unnatural amino acids that targets the nonprimed site of the main protease substrate binding cleft.  They are the first activity-based probes for the main protease of coronaviruses and display target labeling within a human proteome without background.  The authors expect that these reagents will be useful in the drug-development pipeline, not only for the current SARS-Co-V-2, but also for other coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBTq6Plv43lbVg90H21EOLACvtfcHk0lhKWrelGjMs5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslKgsLnJ&md5=3371c4fb01f0f852d78ccf7affe915ef</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fcbic.202000371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.202000371%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bde%2BPlassche%26aufirst%3DM.%2BA.%2BT.%26aulast%3DBarniol-Xicota%26aufirst%3DM.%26aulast%3DVerhelst%26aufirst%3DS.%2BH.%2BL.%26atitle%3DPeptidyl%2520Acyloxymethyl%2520Ketones%2520as%2520Activity-Based%2520Probes%2520for%2520the%2520Main%2520Protease%2520of%2520SARS-CoV-2%26jtitle%3DChemBioChem%26date%3D2020%26volume%3D21%26spage%3D3383%26epage%3D3388%26doi%3D10.1002%2Fcbic.202000371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jian, Y.</span></span> <span> </span><span class="NLM_article-title">The development of Coronavirus 3C-Like protease (3CLpro) inhibitors from 2010 to 2020</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>206</i></span>,  <span class="NLM_fpage">112711</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.ejmech.2020.112711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32810751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1CqtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=206&publication_year=2020&pages=112711&author=Y.+Liuauthor=C.+Liangauthor=L.+Xinauthor=X.+Renauthor=L.+Tianauthor=X.+Juauthor=H.+Liauthor=Y.+Wangauthor=Q.+Zhaoauthor=H.+Liuauthor=W.+Caoauthor=X.+Xieauthor=D.+Zhangauthor=Y.+Wangauthor=Y.+Jian&title=The+development+of+Coronavirus+3C-Like+protease+%283CLpro%29+inhibitors+from+2010+to+2020&doi=10.1016%2Fj.ejmech.2020.112711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The development of Coronavirus 3C-Like protease (3CLpro) inhibitors from 2010 to 2020</span></div><div class="casAuthors">Liu, Yuzhi; Liang, Chengyuan; Xin, Liang; Ren, Xiaodong; Tian, Lei; Ju, Xingke; Li, Han; Wang, Yongbo; Zhao, Qianqian; Liu, Hong; Cao, Wenqiang; Xie, Xiaolin; Zhang, Dezhu; Wang, Yu; Jian, Yanlin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">206</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112711</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  This review fully describes the coronavirus 3CLpro peptidomimetic inhibitors and nonpeptidic small mol. inhibitors developed from 2010 to 2020.  Specifically, the structural characteristics, binding modes and SARs of these 3CLpro inhibitors are expounded in detail by division into two categories: peptidomimetic inhibitors mainly utilize electrophilic warhead groups to covalently bind the 3CLpro Cys145 residue and thereby achieve irreversible inhibition effects, whereas nonpeptidic small mol. inhibitors mainly interact with residues in the S1', S1, S2 and S4 pockets via hydrogen bonds, hydrophobic bonds and van der Waals forces.  Based on the emerging PROTAC technol. and the existing 3CLpro inhibitors, 3CLpro PROTAC degraders are hypothesised to be next-generation anti-coronavirus drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi77_VV-QrrrVg90H21EOLACvtfcHk0lhKWrelGjMs5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1CqtLjI&md5=32ccbb34674ba26c24746117bd689284</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112711%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DXin%26aufirst%3DL.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DJu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DCao%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJian%26aufirst%3DY.%26atitle%3DThe%2520development%2520of%2520Coronavirus%25203C-Like%2520protease%2520%25283CLpro%2529%2520inhibitors%2520from%25202010%2520to%25202020%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D206%26spage%3D112711%26doi%3D10.1016%2Fj.ejmech.2020.112711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span> <span> </span><span class="NLM_article-title">Cyanohydrin as an Anchoring Group for Potent and Selective Inhibitors of Enterovirus 71 3C Protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9414</span>– <span class="NLM_lpage">9420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9414-9420&author=Y.+Zhaiauthor=X.+Zhaoauthor=Z.+Cuiauthor=M.+Wangauthor=Y.+Wangauthor=L.+Liauthor=Q.+Sunauthor=X.+Yangauthor=D.+Zengauthor=Y.+Liuauthor=Y.+Sunauthor=Z.+Louauthor=L.+Shangauthor=Z.+Yin&title=Cyanohydrin+as+an+Anchoring+Group+for+Potent+and+Selective+Inhibitors+of+Enterovirus+71+3C+Protease&doi=10.1021%2Facs.jmedchem.5b01013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Cyanohydrin as an Anchoring Group for Potent and Selective Inhibitors of Enterovirus 71 3C Protease</span></div><div class="casAuthors">Zhai, Yangyang; Zhao, Xiangshuai; Cui, Zhengjie; Wang, Man; Wang, Yaxin; Li, Linfeng; Sun, Qi; Yang, Xi; Zeng, Debin; Liu, Ying; Sun, Yuna; Lou, Zhiyong; Shang, Luqing; Yin, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9414-9420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyanohydrin derivs. as enterovirus 71 (EV71) 3C protease (3Cpro) inhibitors have been synthesized and assayed for their biochem. and antiviral activities.  Compared with the reported inhibitors, cyanohydrins (1S,2S,2'S,5S)-I and (1R,2S,2'S,5S)-I exhibited significantly improved activity and attractive selectivity profiles against other proteases, which were a result of the specific interactions between the cyanohydrin moiety and the catalytic site of 3Cpro.  Cyanohydrin as an anchoring group with high selectivity and excellent inhibitory activity represents a useful choice for cysteine protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxejr9UxByVLVg90H21EOLACvtfcHk0lhgF4nXEPc6Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FK&md5=e43c6b3002e7c61cf8419bc83782be79</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01013%26sid%3Dliteratum%253Aachs%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DCui%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DShang%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DZ.%26atitle%3DCyanohydrin%2520as%2520an%2520Anchoring%2520Group%2520for%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Enterovirus%252071%25203C%2520Protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9414%26epage%3D9420%26doi%3D10.1021%2Facs.jmedchem.5b01013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozminski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. J.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planken, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12725</span>– <span class="NLM_lpage">12747</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12725-12747&author=R.+L.+Hoffmanauthor=R.+S.+Kaniaauthor=M.+A.+Brothersauthor=F.+F.+Daviesauthor=R.+A.+Ferreauthor=K.+S.+Gajiwalaauthor=M.+Heauthor=R.+J.+Hoganauthor=K.+Kozminskiauthor=L.+Y.+Liauthor=J.+W.+Locknerauthor=J.+Louauthor=M.+T.+Marraauthor=L.+J.+Mitchellauthor=B.+W.+Murrayauthor=J.+A.+Niemanauthor=S.+Noellauthor=S.+P.+Plankenauthor=T.+Roweauthor=K.+Ryanauthor=G.+J.+Smithauthor=J.+E.+Solowiejauthor=C.+M.+Steppanauthor=B.+Taggart&title=Discovery+of+Ketone-Based+Covalent+Inhibitors+of+Coronavirus+3CL+Proteases+for+the+Potential+Therapeutic+Treatment+of+COVID-19&doi=10.1021%2Facs.jmedchem.0c01063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span></div><div class="casAuthors">Hoffman, Robert L.; Kania, Robert S.; Brothers, Mary A.; Davies, Jay F.; Ferre, Rose A.; Gajiwala, Ketan S.; He, Mingying; Hogan, Robert J.; Kozminski, Kirk; Li, Lilian Y.; Lockner, Jonathan W.; Lou, Jihong; Marra, Michelle T.; Mitchell Jr., Lennert J.; Murray, Brion W.; Nieman, James A.; Noell, Stephen; Planken, Simon P.; Rowe, Thomas; Ryan, Kevin; Smith III, George J.; Solowiej, James E.; Steppan, Claire M.; Taggart, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12725-12747</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002.  Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is crit. for coronavirus replication.  The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage.  A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallog. led to the identification of 3 and 4.  Preclin. expts. reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an i.v. treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_zn3ZDefOrVg90H21EOLACvtfcHk0lhgF4nXEPc6Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E&md5=7d5be169ad3de528496c862934d59881</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01063%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DDavies%26aufirst%3DF.%2BF.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHogan%26aufirst%3DR.%2BJ.%26aulast%3DKozminski%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLockner%26aufirst%3DJ.%2BW.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMarra%26aufirst%3DM.%2BT.%26aulast%3DMitchell%26aufirst%3DL.%2BJ.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DPlanken%26aufirst%3DS.%2BP.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DSolowiej%26aufirst%3DJ.%2BE.%26aulast%3DSteppan%26aufirst%3DC.%2BM.%26aulast%3DTaggart%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520Ketone-Based%2520Covalent%2520Inhibitors%2520of%2520Coronavirus%25203CL%2520Proteases%2520for%2520the%2520Potential%2520Therapeutic%2520Treatment%2520of%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12725%26epage%3D12747%26doi%3D10.1021%2Facs.jmedchem.0c01063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halford, B.</span></span> <span> </span><span class="NLM_article-title">Pfizer’s novel COVID-19 antiviral heads to clinical trials</span>. <i>C&EN</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">37</span>),  <span class="NLM_fpage">9</span>, <span class="refDoi"> DOI: 10.47287/cen-09837-buscon2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.47287%2Fcen-09837-buscon2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2020&pages=9&issue=37&author=B.+Halford&title=Pfizer%E2%80%99s+novel+COVID-19+antiviral+heads+to+clinical+trials&doi=10.47287%2Fcen-09837-buscon2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.47287%2Fcen-09837-buscon2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.47287%252Fcen-09837-buscon2%26sid%3Dliteratum%253Aachs%26aulast%3DHalford%26aufirst%3DB.%26atitle%3DPfizer%25E2%2580%2599s%2520novel%2520COVID-19%2520antiviral%2520heads%2520to%2520clinical%2520trials%26jtitle%3DC%2526EN%26date%3D2020%26volume%3D98%26issue%3D37%26spage%3D9%26doi%3D10.47287%2Fcen-09837-buscon2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ieda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuyama, T.</span></span> <span> </span><span class="NLM_article-title">Synthesis of the optically active key intermediate of FR901483</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4027</span>– <span class="NLM_lpage">4029</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2010.05.089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.tetlet.2010.05.089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVGks7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=4027-4029&author=S.+Iedaauthor=T.+Kanauthor=T.+Fukuyama&title=Synthesis+of+the+optically+active+key+intermediate+of+FR901483&doi=10.1016%2Fj.tetlet.2010.05.089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of the optically active key intermediate of FR901483</span></div><div class="casAuthors">Ieda, Shigeru; Kan, Toshiyuki; Fukuyama, Tohru</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">4027-4029</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Efficient synthesis of the tricyclic key intermediate I for (-)-FR 901483 was accomplished.  The precursor II of the intramol. aldol reaction was constructed by the Ugi 4CC reaction and subsequent intramol. Dieckmann condensation.  This approach allows a fully stereocontrolled total synthesis of (-)-FR 901483, which would provide various derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ-SssaT8bIbVg90H21EOLACvtfcHk0liiEBIyD_M72A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVGks7g%253D&md5=2267d7eab3f62ddc3bd55a6f5fa24fd4</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2010.05.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2010.05.089%26sid%3Dliteratum%253Aachs%26aulast%3DIeda%26aufirst%3DS.%26aulast%3DKan%26aufirst%3DT.%26aulast%3DFukuyama%26aufirst%3DT.%26atitle%3DSynthesis%2520of%2520the%2520optically%2520active%2520key%2520intermediate%2520of%2520FR901483%26jtitle%3DTetrahedron%2520Lett.%26date%3D2010%26volume%3D51%26spage%3D4027%26epage%3D4029%26doi%3D10.1016%2Fj.tetlet.2010.05.089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span> <span> </span><span class="NLM_article-title">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">4282</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18096-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fs41467-020-18096-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32855413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4282&issue=1&author=W.+Vuongauthor=M.+B.+Khanauthor=C.+Fischerauthor=E.+Arutyunovaauthor=T.+Lamerauthor=J.+Shieldsauthor=H.+A.+Saffranauthor=R.+T.+McKayauthor=M.+J.+van+Belkumauthor=M.+A.+Joyceauthor=H.+S.+Youngauthor=D.+L.+Tyrrellauthor=J.+C.+Vederasauthor=M.+J.+Lemieux&title=Feline+coronavirus+drug+inhibits+the+main+protease+of+SARS-CoV-2+and+blocks+virus+replication&doi=10.1038%2Fs41467-020-18096-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span></div><div class="casAuthors">Vuong, Wayne; Khan, Muhammad Bashir; Fischer, Conrad; Arutyunova, Elena; Lamer, Tess; Shields, Justin; Saffran, Holly A.; McKay, Ryan T.; van Belkum, Marco J.; Joyce, Michael A.; Young, Howard S.; Tyrrell, D. Lorne; Vederas, John C.; Lemieux, M. Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4282</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The main protease, Mpro (or 3CLpro) in SARS-CoV-2 is a viable drug target because of its essential role in the cleavage of the virus polypeptide.  Feline infectious peritonitis, a fatal coronavirus infection in cats, was successfully treated previously with a prodrug GC376, a dipeptide-based protease inhibitor.  We show the prodrug and its parent GC373, are effective inhibitors of the Mpro from both SARS-CoV and SARS-CoV-2 with IC50 values in the nanomolar range.  Crystal structures of SARS-CoV-2 Mpro with these inhibitors have a covalent modification of the nucleophilic Cys145.  NMR anal. reveals that inhibition proceeds via reversible formation of a hemithioacetal.  GC373 and GC376 are potent inhibitors of SARS-CoV-2 replication in cell culture.  They are strong drug candidates for the treatment of human coronavirus infections because they have already been successful in animals.  The work here lays the framework for their use in human trials for the treatment of COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMA7W4XtRQrbVg90H21EOLACvtfcHk0liiEBIyD_M72A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN&md5=c06d2dc9a65ae289505fe120d41d8885</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18096-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18096-2%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DW.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DShields%26aufirst%3DJ.%26aulast%3DSaffran%26aufirst%3DH.%2BA.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DJoyce%26aufirst%3DM.%2BA.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26atitle%3DFeline%2520coronavirus%2520drug%2520inhibits%2520the%2520main%2520protease%2520of%2520SARS-CoV-2%2520and%2520blocks%2520virus%2520replication%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26issue%3D1%26spage%3D4282%26doi%3D10.1038%2Fs41467-020-18096-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balbo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telliez, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, A. J.</span></span> <span> </span><span class="NLM_article-title">The advantages of describing covalent inhibitor in vitro potencies by IC<sub>50</sub> at a fixed time point: IC<sub>50</sub> determination of covalent inhibitors provides meaningful data to medicinal chemistry for SAR optimization</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">115865</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2020.115865</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.bmc.2020.115865" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33285410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFahsrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2021&pages=115865&author=A.+Thorarensenauthor=P.+Balboauthor=M.+E.+Bankerauthor=R.+M.+Czerwinskiauthor=M.+Kuhnauthor=T.+S.+Maurerauthor=J.-B.+Telliezauthor=F.+Vincentauthor=A.+J.+Wittwer&title=The+advantages+of+describing+covalent+inhibitor+in+vitro+potencies+by+IC50+at+a+fixed+time+point%3A+IC50+determination+of+covalent+inhibitors+provides+meaningful+data+to+medicinal+chemistry+for+SAR+optimization&doi=10.1016%2Fj.bmc.2020.115865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The advantages of describing covalent inhibitor in vitro potencies by IC50 at a fixed time point. IC50 determination of covalent inhibitors provides meaningful data to medicinal chemistry for SAR optimization</span></div><div class="casAuthors">Thorarensen, Atli; Balbo, Paul; Banker, Mary E.; Czerwinski, Robert M.; Kuhn, Max; Maurer, Tristan S.; Telliez, Jean-Baptiste; Vincent, Fabien; Wittwer, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">115865</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recent years have seen a resurgence in drug discovery efforts aimed at the identification of covalent inhibitors which has led to an explosion of literature reports in this area and most importantly new approved therapies.  These reports and breakthroughs highlight the significant investments made across the industry in SAR campaigns to optimize inhibitors.  The potency of covalent inhibitors is generally considered to be more accurately described by the time-independent kinetic parameter kinact/Ki rather than a by a simple IC50 since the latter is a time-dependent parameter.  Enzyme substrate concns. are an addnl. important factor to consider when attempting to translate parameters derived from enzymol. expts. to phenotypic behavior in a physiol. relevant cell-based system.  Theor. and exptl. investigations into the relationship between IC50, time, substrate concn. and Kinact/Ki provided us with an effective approach to provide meaningful data for SAR optimization.  The data we generated for our JAK3 irreversible covalent inhibitor program using IC50 values provided by enzyme assays with long incubations (>1h) coupled with physiol. substrate concn. provided the medicinal chemist with optimal information in a rapid and efficient manner.  We further document the wide applicability of this method by applying it to other enzymes systems where we have run covalent inhibitor programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_kQEA2V2QErVg90H21EOLACvtfcHk0liiEBIyD_M72A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFahsrbO&md5=e981d4fab453add5bcfea1dadf8168e4</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2020.115865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2020.115865%26sid%3Dliteratum%253Aachs%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DBalbo%26aufirst%3DP.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DCzerwinski%26aufirst%3DR.%2BM.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DMaurer%26aufirst%3DT.%2BS.%26aulast%3DTelliez%26aufirst%3DJ.-B.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DWittwer%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520advantages%2520of%2520describing%2520covalent%2520inhibitor%2520in%2520vitro%2520potencies%2520by%2520IC50%2520at%2520a%2520fixed%2520time%2520point%253A%2520IC50%2520determination%2520of%2520covalent%2520inhibitors%2520provides%2520meaningful%2520data%2520to%2520medicinal%2520chemistry%2520for%2520SAR%2520optimization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2021%26volume%3D29%26spage%3D115865%26doi%3D10.1016%2Fj.bmc.2020.115865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willoughby, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span> <span> </span><span class="NLM_article-title">Improved SARS-CoV-2 Mpro inhibitors based on feline coronavirus drug GC376: structure-activity studies and solubilization for enhanced bioavailability</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>222</i></span>,  <span class="NLM_fpage">113584</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2021.113584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.ejmech.2021.113584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=34118724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXht1yrt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=222&publication_year=2021&pages=113584&author=W.+Vuongauthor=C.+Fischerauthor=M.+B.+Khanauthor=M.+J.+van+Belkumauthor=T.+Lamerauthor=K.+D.+Willoughbyauthor=J.+Luauthor=E.+Arutyunovaauthor=M.+A.+Joyceauthor=H.+A.+Saffranauthor=J.+A.+Shieldsauthor=H.+S.+Youngauthor=J.+A.+Niemanauthor=D.+L.+Tyrrellauthor=M.+J.+Lemieuxauthor=J.+C.+Vederas&title=Improved+SARS-CoV-2+Mpro+inhibitors+based+on+feline+coronavirus+drug+GC376%3A+structure-activity+studies+and+solubilization+for+enhanced+bioavailability&doi=10.1016%2Fj.ejmech.2021.113584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44aR"><div class="casContent"><span class="casTitleNuber">44a</span><div class="casTitle"><span class="NLM_cas:atitle">Improved SARS-CoV-2 Mpro inhibitors based on feline antiviral drug GC376: Structural enhancements, increased solubility, and micellar studies</span></div><div class="casAuthors">Vuong, Wayne; Fischer, Conrad; Khan, Muhammad Bashir; van Belkum, Marco J.; Lamer, Tess; Willoughby, Kurtis D.; Lu, Jimmy; Arutyunova, Elena; Joyce, Michael A.; Saffran, Holly A.; Shields, Justin A.; Young, Howard S.; Nieman, James A.; Tyrrell, D. Lorne; Lemieux, M. Joanne; Vederas, John C.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">222</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113584</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Replication of SARS-CoV-2, the coronavirus causing COVID-19, requires a main protease (Mpro) to cleave viral proteins.  Consequently, Mpro is a target for antiviral agents.  We and others previously demonstrated that GC376, a bisulfite prodrug with efficacy as an anti-coronaviral agent in animals, is an effective inhibitor of Mpro in SARS-CoV-2.  We report structure-activity studies of improved GC376 derivs. with nanomolar affinities and therapeutic indexes >200.  Crystallog. structures of inhibitor-Mpro complexes reveal that an alternative binding pocket in Mpro, S4, accommodates the P3 position.  Alternative binding is induced by polar P3 groups or a nearby Me.  NMR and soly. studies with GC376 show that it exists as a mixt. of stereoisomers and forms colloids in aq. media at higher concns., a property not previously reported.  Replacement of its Na+ counter ion with choline greatly increases soly.  The phys., biochem., crystallog., and cellular data reveal new avenues for Mpro inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj9Iq0rzrDdrVg90H21EOLACvtfcHk0libXF_36SOh9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXht1yrt73I&md5=aa17ed00bc373ef06e402677d9a523dc</span></div><a href="/servlet/linkout?suffix=cit44a&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2021.113584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2021.113584%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DW.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DWilloughby%26aufirst%3DK.%2BD.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DJoyce%26aufirst%3DM.%2BA.%26aulast%3DSaffran%26aufirst%3DH.%2BA.%26aulast%3DShields%26aufirst%3DJ.%2BA.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26atitle%3DImproved%2520SARS-CoV-2%2520Mpro%2520inhibitors%2520based%2520on%2520feline%2520coronavirus%2520drug%2520GC376%253A%2520structure-activity%2520studies%2520and%2520solubilization%2520for%2520enhanced%2520bioavailability%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2021%26volume%3D222%26spage%3D113584%26doi%3D10.1016%2Fj.ejmech.2021.113584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit44b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span> <span> </span><span class="NLM_article-title">N-Terminal finger stabilizes the S1 Pocket for the reversible feline drug GC376 in SARS-CoV-2 M<sup>pro</sup> Dimer</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>433</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">167003</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2021.167003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.jmb.2021.167003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33895266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXpvFartbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=433&publication_year=2021&pages=167003&issue=13&author=E.+Arutyunovaauthor=M.+B.+Khanauthor=C.+Fischerauthor=J.+Luauthor=T.+Lamerauthor=W.+Vuongauthor=M.+J.+van+Belkumauthor=R.+T.+McKayauthor=D.+L.+Tyrrellauthor=J.+C.+Vederasauthor=H.+S.+Youngauthor=M.+J.+Lemieux&title=N-Terminal+finger+stabilizes+the+S1+Pocket+for+the+reversible+feline+drug+GC376+in+SARS-CoV-2+Mpro+Dimer&doi=10.1016%2Fj.jmb.2021.167003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44bR"><div class="casContent"><span class="casTitleNuber">44b</span><div class="casTitle"><span class="NLM_cas:atitle">N-terminal finger stabilizes the S1 pocket for the reversible feline drug GC376 in the SARS-CoV-2 Mpro dimer</span></div><div class="casAuthors">Arutyunova, Elena; Khan, Muhammad Bashir; Fischer, Conrad; Lu, Jimmy; Lamer, Tess; Vuong, Wayne; van Belkum, Marco J.; McKay, Ryan T.; Tyrrell, D. Lorne; Vederas, John C.; Young, Howard S.; Lemieux, M. Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">433</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">167003</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The main protease (Mpro, also known as 3CL protease) of SARS-CoV-2 is a high priority drug target in the development of antivirals to combat COVID-19 infections.  A feline coronavirus antiviral drug, GC376, has been shown to be effective in inhibiting the SARS-CoV-2 main protease and live virus growth.  As this drug moves into clin. trials, further characterization of GC376 with the main protease of coronaviruses is required to gain insight into the drug's properties, such as reversibility and broad specificity.  Reversibility is an important factor for therapeutic proteolytic inhibitors to prevent toxicity due to off-target effects.  Here we demonstrate that GC376 has nanomolar Ki values with the Mpro from both SARS-CoV-2 and SARS-CoV strains.  Restoring enzymic activity after inhibition by GC376 demonstrates reversible binding with both proteases.  In addn., the stability and thermodn. parameters of both proteases were studied to shed light on phys. chem. properties of these viral enzymes, revealing higher stability for SARS-CoV-2 Mpro.  The comparison of a new X-ray crystal structure of Mpro from SARS-CoV complexed with GC376 reveals similar mol. mechanism of inhibition compared to SARS-CoV-2 Mpro, and gives insight into the broad specificity properties of this drug.  In both structures, we observe domain swapping of the N-termini in the dimer of the Mpro, which facilitates coordination of the drug's P1 position.  These results validate that GC376 is a drug with an off-rate suitable for clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMfiKgDeiJRrVg90H21EOLACvtfcHk0libXF_36SOh9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXpvFartbg%253D&md5=fa586484a14c510731bba968126c6bf6</span></div><a href="/servlet/linkout?suffix=cit44b&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2021.167003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2021.167003%26sid%3Dliteratum%253Aachs%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DVuong%26aufirst%3DW.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26atitle%3DN-Terminal%2520finger%2520stabilizes%2520the%2520S1%2520Pocket%2520for%2520the%2520reversible%2520feline%2520drug%2520GC376%2520in%2520SARS-CoV-2%2520Mpro%2520Dimer%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2021%26volume%3D433%26issue%3D13%26spage%3D167003%26doi%3D10.1016%2Fj.jmb.2021.167003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szeto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarbet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1038/s41422-020-0356-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fs41422-020-0356-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32541865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1WmsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=678-692&author=C.+Maauthor=M.+D.+Saccoauthor=B.+Hurstauthor=J.+A.+Townsendauthor=Y.+Huauthor=T.+Szetoauthor=X.+Zhangauthor=B.+Tarbetauthor=M.+T.+Martyauthor=Y.+Chenauthor=J.+Wang&title=Boceprevir%2C+GC-376%2C+and+calpain+inhibitors+II%2C+XII+inhibit+SARS-CoV-2+viral+replication+by+targeting+the+viral+main+protease&doi=10.1038%2Fs41422-020-0356-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease</span></div><div class="casAuthors">Ma, Chunlong; Sacco, Michael Dominic; Hurst, Brett; Townsend, Julia Alma; Hu, Yanmei; Szeto, Tommy; Zhang, Xiujun; Tarbet, Bart; Marty, Michael Thomas; Chen, Yu; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">678-692</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A new coronavirus SARS-CoV-2, also called novel coronavirus 2019 (2019-nCoV), started to circulate among humans around Dec. 2019, and it is now widespread as a global pandemic.  The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.35% as of May 26, 2020.  There is no vaccine or antiviral available for SARS-CoV-2.  We report our discovery of inhibitors targeting the SARS-CoV-2 main protease (Mpro).  Using the FRET-based enzymic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II and XII were identified to have potent activity with single-digit to submicromolar IC50 values in the enzymic assay.  The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry.  Significantly, 4 compds. (boceprevir, GC-376, and calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC50 values ranging 0.49-3.37μM.  Notably, boceprevir and calpain inhibitors II and XII represent novel chemotypes that are distinct from known substrate-based peptidomimetic Mpro inhibitors.  A complex crystal structure of SARS-CoV-2 Mpro with GC-376, detd. at 2.15 Å resoln. with 3 protomers per asym. unit, revealed 2 unique binding configurations, shedding light on the mol. interactions and protein conformational flexibility underlying substrate and inhibitor binding by Mpro.  Overall, the compds. identified provide promising starting points for the further development of SARS-CoV-2 therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVnFC-t0uFW7Vg90H21EOLACvtfcHk0libXF_36SOh9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1WmsL%252FL&md5=225d518c313474cdfbcfe678d35e26a4</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41422-020-0356-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-020-0356-z%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DSacco%26aufirst%3DM.%2BD.%26aulast%3DHurst%26aufirst%3DB.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DSzeto%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DTarbet%26aufirst%3DB.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DBoceprevir%252C%2520GC-376%252C%2520and%2520calpain%2520inhibitors%2520II%252C%2520XII%2520inhibit%2520SARS-CoV-2%2520viral%2520replication%2520by%2520targeting%2520the%2520viral%2520main%2520protease%26jtitle%3DCell%2520Res.%26date%3D2020%26volume%3D30%26spage%3D678%26epage%3D692%26doi%3D10.1038%2Fs41422-020-0356-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mengist, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mekonnen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, T.</span></span> <span> </span><span class="NLM_article-title">Potency, Safety, and Pharmacokinetic Profiles of Potential Inhibitors Targeting SARS-CoV-2 Main Protease</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">630500</span>, <span class="refDoi"> DOI: 10.3389/fphar.2020.630500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.3389%2Ffphar.2020.630500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33597888" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=630500&author=H.+M.+Mengistauthor=D.+Mekonnenauthor=A.+Mohammedauthor=R.+Shiauthor=T.+Jin&title=Potency%2C+Safety%2C+and+Pharmacokinetic+Profiles+of+Potential+Inhibitors+Targeting+SARS-CoV-2+Main+Protease&doi=10.3389%2Ffphar.2020.630500"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2020.630500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2020.630500%26sid%3Dliteratum%253Aachs%26aulast%3DMengist%26aufirst%3DH.%2BM.%26aulast%3DMekonnen%26aufirst%3DD.%26aulast%3DMohammed%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DT.%26atitle%3DPotency%252C%2520Safety%252C%2520and%2520Pharmacokinetic%2520Profiles%2520of%2520Potential%2520Inhibitors%2520Targeting%2520SARS-CoV-2%2520Main%2520Protease%26jtitle%3DFront.%2520Pharmacol.%26date%3D2021%26volume%3D11%26spage%3D630500%26doi%3D10.3389%2Ffphar.2020.630500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahr, A.</span></span> <span> </span><span class="NLM_article-title">Lipophilicity and Its Relationship with Passive Drug Permeation</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">962</span>– <span class="NLM_lpage">977</span>, <span class="refDoi"> DOI: 10.1007/s11095-010-0303-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1007%2Fs11095-010-0303-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=21052797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlOqtrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2011&pages=962-977&author=X.+Liuauthor=B.+Testaauthor=A.+Fahr&title=Lipophilicity+and+Its+Relationship+with+Passive+Drug+Permeation&doi=10.1007%2Fs11095-010-0303-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47a</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilicity and Its Relationship with Passive Drug Permeation</span></div><div class="casAuthors">Liu, Xiangli; Testa, Bernard; Fahr, Alfred</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">962-977</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">In this review, we first summarize the structure and properties of biol. membranes and the routes of passive drug transfer through physiol. barriers.  Lipophilicity is then introduced in terms of the intermol. interactions it encodes.  Finally, lipophilicity indexes from isotropic solvent systems and from anisotropic membrane-like systems are discussed for their capacity to predict passive drug permeation across biol. membranes such as the intestinal epithelium, the blood-brain barrier (BBB) or the skin.  The broad evidence presented here shows that beyond the predictive power of lipophilicity parameters, the various intermol. forces they encode allow a mechanistic interpretation of passive drug permeation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxn2Ic1yX8SbVg90H21EOLACvtfcHk0ljk0hehFrZNGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlOqtrzK&md5=6bd2bb54a14574f31b046204f260595a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs11095-010-0303-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-010-0303-7%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DFahr%26aufirst%3DA.%26atitle%3DLipophilicity%2520and%2520Its%2520Relationship%2520with%2520Passive%2520Drug%2520Permeation%26jtitle%3DPharm.%2520Res.%26date%3D2011%26volume%3D28%26spage%3D962%26epage%3D977%26doi%3D10.1007%2Fs11095-010-0303-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit47b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinner, M. J.</span></span> <span> </span><span class="NLM_article-title">Getting Across the Cell Membrane: An Overview for Small Molecules, Peptides, and Proteins</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1266</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-2272-7_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1007%2F978-1-4939-2272-7_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=25560066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1Siur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1266&publication_year=2015&pages=29-53&author=N.+J.+Yangauthor=M.+J.+Hinner&title=Getting+Across+the+Cell+Membrane%3A+An+Overview+for+Small+Molecules%2C+Peptides%2C+and+Proteins&doi=10.1007%2F978-1-4939-2272-7_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47bR"><div class="casContent"><span class="casTitleNuber">47b</span><div class="casTitle"><span class="NLM_cas:atitle">Getting Across the Cell Membrane: An Overview for Small Molecules, Peptides, and Proteins</span></div><div class="casAuthors">Yang, Nicole J.; Hinner, Marlon J.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1266</span>
        (<span class="NLM_cas:issue">Site-Specific Protein Labeling</span>),
    <span class="NLM_cas:pages">29-53</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The ability to efficiently access cytosolic proteins is desired in both biol. research and medicine.  However, targeting intracellular proteins is often challenging, because to reach the cytosol, exogenous mols. must first traverse the cell membrane.  This review provides a broad overview of how certain mols. are thought to cross this barrier, and what kinds of approaches are being made to enhance the intracellular delivery of those that are impermeable.  We first discuss rules that govern the passive permeability of small mols. across the lipid membrane, and mechanisms of membrane transport that have evolved in nature for certain metabolites, peptides, and proteins.  Then, we introduce design strategies that have emerged in the development of small mols. and peptides with improved permeability.  Finally, intracellular delivery systems that have been engineered for protein payloads are surveyed.  Viewpoints from varying disciplines have been brought together to provide a cohesive overview of how the membrane barrier is being overcome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptS-9wM3Z8jbVg90H21EOLACvtfcHk0ljk0hehFrZNGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1Siur4%253D&md5=4002e2e8ab2fcc86d9ea400e1aa2a29b</span></div><a href="/servlet/linkout?suffix=cit47b&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2272-7_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-2272-7_3%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DN.%2BJ.%26aulast%3DHinner%26aufirst%3DM.%2BJ.%26atitle%3DGetting%2520Across%2520the%2520Cell%2520Membrane%253A%2520An%2520Overview%2520for%2520Small%2520Molecules%252C%2520Peptides%252C%2520and%2520Proteins%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1266%26spage%3D29%26epage%3D53%26doi%3D10.1007%2F978-1-4939-2272-7_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yukawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naven, R.</span></span> <span> </span><span class="NLM_article-title">Utility of Physicochemical Properties for the Prediction of Toxicological Outcomes: Takeda Perspective</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00536</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00536" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFKqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=203-209&issue=2&author=T.+Yukawaauthor=R.+Naven&title=Utility+of+Physicochemical+Properties+for+the+Prediction+of+Toxicological+Outcomes%3A+Takeda+Perspective&doi=10.1021%2Facsmedchemlett.9b00536"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48aR"><div class="casContent"><span class="casTitleNuber">48a</span><div class="casTitle"><span class="NLM_cas:atitle">Utility of Physicochemical Properties for the Prediction of Toxicological Outcomes: Takeda Perspective</span></div><div class="casAuthors">Yukawa, Tomoya; Naven, Russell</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">203-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The role that physicochem. properties play toward increasing the likelihood of toxicity findings in in vivo studies has been well reported, albeit sometimes with different conclusions.  We decided to understand the role that physicochem. properties play toward the prediction of in vivo toxicol. outcomes for Takeda chem. using 284 internal compds.  In support of the previously reported "3/75 rule", reducing lipophilicity of mols. decreases toxicity odds noticeably; however, we also found that the trend of toxicity odds is different between compds. classified by their ionization state.  For basic mols., the odds of in vivo toxicity outcomes were significantly impacted by both lipophilicity and polar surface area, whereas neutral mols. were impacted less so.  Through an anal. of several project-related compds., we herein demonstrate that the utilization of the 3/75 rule coupled with consideration of ionization state is a rational strategy for medicinal chem. design of safer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOaavww-aqYLVg90H21EOLACvtfcHk0ljk0hehFrZNGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFKqtrY%253D&md5=7d1f2112b22fcf45f59aef6b3658047d</span></div><a href="/servlet/linkout?suffix=cit48a&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00536&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00536%26sid%3Dliteratum%253Aachs%26aulast%3DYukawa%26aufirst%3DT.%26aulast%3DNaven%26aufirst%3DR.%26atitle%3DUtility%2520of%2520Physicochemical%2520Properties%2520for%2520the%2520Prediction%2520of%2520Toxicological%2520Outcomes%253A%2520Takeda%2520Perspective%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26issue%3D2%26spage%3D203%26epage%3D209%26doi%3D10.1021%2Facsmedchemlett.9b00536" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit48b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kiani, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jabeen, I.</span></span> <span> </span><span class="NLM_article-title">Lipophilic Metabolic Efficiency (LipMetE) and Drug Efficiency Indices to Explore the Metabolic Properties of the Substrates of Selected Cytochrome P450 Isoforms</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1021/acsomega.9b02344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.9b02344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisV2rtb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=179-188&author=Y.+S.+Kianiauthor=I.+Jabeen&title=Lipophilic+Metabolic+Efficiency+%28LipMetE%29+and+Drug+Efficiency+Indices+to+Explore+the+Metabolic+Properties+of+the+Substrates+of+Selected+Cytochrome+P450+Isoforms&doi=10.1021%2Facsomega.9b02344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48bR"><div class="casContent"><span class="casTitleNuber">48b</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Metabolic Efficiency (LipMetE) and Drug Efficiency Indices to Explore the Metabolic Properties of the Substrates of Selected Cytochrome P450 Isoforms</span></div><div class="casAuthors">Kiani, Yusra Sajid; Jabeen, Ishrat</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">179-188</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cytochrome P 450 (CYP450) enzymes belong to a superfamily of heme-contg. proteins that are involved in the metab. of structurally diverse endogenous and exogenous compds.  Various proof-of-concept studies indicate that metabolic stability and intrinsic clearance of CYP450 substrates are linked with the resp. lipophilicity (log P or log D).  This necessitates the normalization of lipophilicity (log P or log D) of a given CYP450 substrate with respect to its metabolic stability (LipMetE) and intrinsic clearance (log10CLint,u).  Therefore, in this article, the LipMetE values of already known substrates of CYP1A2, CYP2C9, CYP2C19, CYP2D6, and CYP3A4, including some marketed drugs, have been calcd. to elucidate the relationship between lipophilicity (log D7.4) and in vitro clearance.  Moreover, various drug efficiency metrics including lipophilic efficiency (LipE) and ligand efficiency (LE) have been evaluated, and the thresholds of these parameters have been defined for the CYP450 substrates exhibiting normalized LipMetE.  Our results indicate that for a given range of LipMetE, greater the log D value of the substrate the more avidly it binds to a given CYP450 enzyme and shows more intrinsic clearance (log10CLint,u).  Overall, the majority of the model substrates of CYP450 isoforms and already marketed drugs in our datasets exhibit log D7.4 values of ∼2.5 with LipMetE values in the range of 0-2.5 and LipE values of ≤3.  Overall, consideration of these parameters in ADME profiling could aid in reducing the drug failure rate in the later stages of clin. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgrXqwi9iUTbVg90H21EOLACvtfcHk0ljLkVpVvnxXMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisV2rtb%252FK&md5=04b499c771265b528a7f0ebf7577ab03</span></div><a href="/servlet/linkout?suffix=cit48b&amp;dbid=16384&amp;doi=10.1021%2Facsomega.9b02344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.9b02344%26sid%3Dliteratum%253Aachs%26aulast%3DKiani%26aufirst%3DY.%2BS.%26aulast%3DJabeen%26aufirst%3DI.%26atitle%3DLipophilic%2520Metabolic%2520Efficiency%2520%2528LipMetE%2529%2520and%2520Drug%2520Efficiency%2520Indices%2520to%2520Explore%2520the%2520Metabolic%2520Properties%2520of%2520the%2520Substrates%2520of%2520Selected%2520Cytochrome%2520P450%2520Isoforms%26jtitle%3DACS%2520Omega%26date%3D2020%26volume%3D5%26spage%3D179%26epage%3D188%26doi%3D10.1021%2Facsomega.9b02344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit48c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tinworth, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Facts, Patterns and Principles in Drug Discovery: Appraising the Rule of 5 with Measured Physicochemical Data</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">10091</span>– <span class="NLM_lpage">10108</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01596</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01596" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslShtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=10091-10108&author=C.+P.+Tinworthauthor=R.+J.+Young&title=Facts%2C+Patterns+and+Principles+in+Drug+Discovery%3A+Appraising+the+Rule+of+5+with+Measured+Physicochemical+Data&doi=10.1021%2Facs.jmedchem.9b01596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48cR"><div class="casContent"><span class="casTitleNuber">48c</span><div class="casTitle"><span class="NLM_cas:atitle">Facts, Patterns, and Principles in Drug Discovery: Appraising the Rule of 5 with Measured Physicochemical Data</span></div><div class="casAuthors">Tinworth, Christopher P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10091-10108</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The rule of 5 was designed to est. the likelihood of poor absorption or permeation, noting the impact of poor soly.  This Perspective explores the impact of various physicochem. descriptors and contemporary lipophilicity measurements on permeability and soly., showing that the distribution coeff. log D7.4 (rather than log P) is the most impactful parameter.  Mol. wt., almost invariably the defining characteristic of "beyond the rule of 5" compds., has little impact on soly. when log D7.4 measurements and aromaticity are considered.  Predicting permeation is more complex, given passive and carrier transport mechanisms; however, notable patterns of behavior are apparent, giving insight even "beyond the rule of 5".  Recommended best practices should involve using the facts (measurements) and the patterns they reveal to establish informative principles rather than fastidious rules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBT9kzvTWdTbVg90H21EOLACvtfcHk0ljLkVpVvnxXMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslShtb4%253D&md5=c93bbcd944dcb41c0d38d2a3881b8593</span></div><a href="/servlet/linkout?suffix=cit48c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01596%26sid%3Dliteratum%253Aachs%26aulast%3DTinworth%26aufirst%3DC.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DFacts%252C%2520Patterns%2520and%2520Principles%2520in%2520Drug%2520Discovery%253A%2520Appraising%2520the%2520Rule%2520of%25205%2520with%2520Measured%2520Physicochemical%2520Data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D10091%26epage%3D10108%26doi%3D10.1021%2Facs.jmedchem.9b01596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit48d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratkova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span> <span> </span><span class="NLM_article-title">Steering New Drug Discovery Campaigns: Permeability, Solubility and Physicochemical Properties in the bRo5 Chemical Space</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00581</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00581" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXislWhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=13-23&author=G.+Caronauthor=J.+Kihlbergauthor=G.+Goetzauthor=E.+Ratkovaauthor=V.+Poongavanamauthor=G.+Ermondi&title=Steering+New+Drug+Discovery+Campaigns%3A+Permeability%2C+Solubility+and+Physicochemical+Properties+in+the+bRo5+Chemical+Space&doi=10.1021%2Facsmedchemlett.0c00581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48dR"><div class="casContent"><span class="casTitleNuber">48d</span><div class="casTitle"><span class="NLM_cas:atitle">Steering New Drug Discovery Campaigns: Permeability, Solubility, and Physicochemical Properties in the bRo5 Chemical Space</span></div><div class="casAuthors">Caron, Giulia; Kihlberg, Jan; Goetz, Gilles; Ratkova, Ekaterina; Poongavanam, Vasanthanathan; Ermondi, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-23</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An increasing no. of drug discovery programs concern compds. in the beyond rule of 5 (bRo5) chem. space, such as cyclic peptides, macrocycles, and degraders.  Recent results show that common paradigms of property-based drug design need revision to be applied to larger and more flexible compds.  A virtual event entitled "Soly., permeability and physico-chem. properties in the bRo5 chem. space" was organized to provide preliminary guidance on how to make the discovery of oral drugs in the bRo5 space more effective.  The four speakers emphasized the importance of the bRo5 space as a source of new oral drugs and provided examples of exptl. and computational methods specifically tailored for design and optimization in this chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHq-tvhn7ITrVg90H21EOLACvtfcHk0lhc96ZSZxN7-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXislWhsw%253D%253D&md5=a8d6bd0c3fa5762ed9e4987e78aaf2bf</span></div><a href="/servlet/linkout?suffix=cit48d&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00581%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26aulast%3DGoetz%26aufirst%3DG.%26aulast%3DRatkova%26aufirst%3DE.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DErmondi%26aufirst%3DG.%26atitle%3DSteering%2520New%2520Drug%2520Discovery%2520Campaigns%253A%2520Permeability%252C%2520Solubility%2520and%2520Physicochemical%2520Properties%2520in%2520the%2520bRo5%2520Chemical%2520Space%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D12%26spage%3D13%26epage%3D23%26doi%3D10.1021%2Facsmedchemlett.0c00581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span> </span><span class="NLM_article-title">Dissociation Constants of Organic Acids
and Bases</span>. In  <i>CRC Handbook of Chemistry and
Physics</i>, <span class="NLM_edition">88</span>th ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Lide, D. R.</span></span>, Ed.; <span class="NLM_publisher-name">CRC Press</span>, <span class="NLM_year">2007</span>; pp  <span class="NLM_fpage">8-42</span>– <span class="NLM_lpage">8-51</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=8-42-8-51&author=D.+R.+Lide&title=CRC+Handbook+of+Chemistry+and%0APhysics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26atitle%3DDissociation%2520Constants%2520of%2520Organic%2520Acids%250Aand%2520Bases%26btitle%3DCRC%2520Handbook%2520of%2520Chemistry%2520and%250APhysics%26aulast%3DLide%26aufirst%3DD.%2BR.%26pub%3DCRC%2520Press%26date%3D2007%26spage%3D8-42%26epage%3D8-51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhareshwar, S. S.</span>; <span class="NLM_string-name">Stella, V. J.</span></span> <span> </span><span class="NLM_article-title">Chapter 3.2. Prodrugs of Alcohols and Phenols.</span> In  <i>Prodrugs: Challenges and Rewards Part 1</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stella, V. J.</span>, <span class="NLM_string-name">Borchardt, R. T.</span>, <span class="NLM_string-name">Hageman, M. J.</span>, <span class="NLM_string-name">Oliyai, R.</span>, <span class="NLM_string-name">Maag, H.</span>, <span class="NLM_string-name">Tilley, J. W.</span></span>, Eds.; <span class="NLM_series">Biotechnology: Pharmaceutical Aspects</span>  <span class="NLM_volume">V</span>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2007</span>; pp  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">99</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1007%2F978-0-387-49785-3_21" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=31-99&author=S.+S.+Dhareshwar&author=V.+J.+Stellaauthor=V.+J.+Stella&author=R.+T.+Borchardt&author=M.+J.+Hageman&author=R.+Oliyai&author=H.+Maag&author=J.+W.+Tilley&title=Prodrugs%3A+Challenges+and+Rewards+Part+1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-49785-3_21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-49785-3_21%26sid%3Dliteratum%253Aachs%26aulast%3DDhareshwar%26aufirst%3DS.%2BS.%26atitle%3DChapter%25203.2.%2520Prodrugs%2520of%2520Alcohols%2520and%2520Phenols.%26btitle%3DProdrugs%253A%2520Challenges%2520and%2520Rewards%2520Part%25201%26aulast%3DStella%26aufirst%3DV.%2BJ.%26volume%3DV%26pub%3DSpringer%26date%3D2007%26spage%3D31%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biniossek, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nägler, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, O.</span></span> <span> </span><span class="NLM_article-title">Proteomic Identification of Protease Cleavage Sites Characterizes Prime and Non-prime Specificity of Cysteine Cathepsin B L and S</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5363</span>– <span class="NLM_lpage">5373</span>, <span class="refDoi"> DOI: 10.1021/pr200621z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr200621z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKgsL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=5363-5373&author=M.+L.+Biniossekauthor=D.+K.+N%C3%A4glerauthor=C.+Becker-Paulyauthor=O.+Schilling&title=Proteomic+Identification+of+Protease+Cleavage+Sites+Characterizes+Prime+and+Non-prime+Specificity+of+Cysteine+Cathepsin+B+L+and+S&doi=10.1021%2Fpr200621z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Identification of Protease Cleavage Sites Characterizes Prime and Non-prime Specificity of Cysteine Cathepsins B, L, and S</span></div><div class="casAuthors">Biniossek, Martin L.; Naegler, Dorit K.; Becker-Pauly, Christoph; Schilling, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5363-5373</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cysteine cathepsins mediate proteome homeostasis and have pivotal functions in diseases such as cancer.  To better understand substrate recognition by cathepsins B, L, and S, we applied proteomic identification of protease cleavage sites (PICS) for simultaneous profiling of prime and non-prime specificity.  PICS profiling of cathepsin B endopeptidase specificity highlights strong selectivity for glycine in P3' due to an occluding loop blocking access to the primed subsites.  In P1', cathepsin B has a partial preference for phenylalanine, which is not found for cathepsins L and S.  Occurrence of P1' phenylalanine often coincides with arom. residues in P2.  For cathepsin L, PICS identifies 845 cleavage sites, representing the most comprehensive PICS profile to date.  Cathepsin L specificity is dominated by the canonical preference for arom. residues in P2 with limited contribution of prime-site selectivity determinants.  Profiling of cathepsins B and L with a shorter incubation time (4 h instead of 16 h) did not reveal time-dependency of individual specificity determinants.  Cathepsin S specificity was profiled at pH 6.0 and 7.5.  The PICS profiles at both pH values display a high degree of similarity.  Cathepsin S specificity is primarily guided by aliph. residues in P2 with limited importance of prime-site residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxwo4mQtEBrrVg90H21EOLACvtfcHk0lhc96ZSZxN7-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKgsL7F&md5=4bd86bc70546694933a2975c9b367450</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fpr200621z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr200621z%26sid%3Dliteratum%253Aachs%26aulast%3DBiniossek%26aufirst%3DM.%2BL.%26aulast%3DN%25C3%25A4gler%26aufirst%3DD.%2BK.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DSchilling%26aufirst%3DO.%26atitle%3DProteomic%2520Identification%2520of%2520Protease%2520Cleavage%2520Sites%2520Characterizes%2520Prime%2520and%2520Non-prime%2520Specificity%2520of%2520Cysteine%2520Cathepsin%2520B%2520L%2520and%2520S%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D5363%26epage%3D5373%26doi%3D10.1021%2Fpr200621z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samaha, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elgamal, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weldon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geraghty, R.</span></span> <span> </span><span class="NLM_article-title">Cathepsin S: investigating an old player in lung disease pathogenesis, comorbidities, and potential therapeutics</span>. <i>Respir. Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">111</span>, <span class="refDoi"> DOI: 10.1186/s12931-020-01381-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1186%2Fs12931-020-01381-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32398133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A280%3ADC%252BB38vlvFCqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&pages=111&author=R.+Brownauthor=S.+Nathauthor=A.+Loraauthor=G.+Samahaauthor=Z.+Elgamalauthor=R.+Kaiserauthor=C.+Taggartauthor=S.+Weldonauthor=R.+Geraghty&title=Cathepsin+S%3A+investigating+an+old+player+in+lung+disease+pathogenesis%2C+comorbidities%2C+and+potential+therapeutics&doi=10.1186%2Fs12931-020-01381-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin S: investigating an old player in lung disease pathogenesis, comorbidities, and potential therapeutics</span></div><div class="casAuthors">Brown Ryan; Taggart Clifford; Weldon Sinead; Nath Sridesh; Lora Alnardo; Samaha Ghassan; Elgamal Ziyad; Kaiser Ryan; Geraghty Patrick; Geraghty Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Respiratory research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Dysregulated expression and activity of cathepsin S (CTSS), a lysosomal protease and a member of the cysteine cathepsin protease family, is linked to the pathogenesis of multiple diseases, including a number of conditions affecting the lungs.  Extracellular CTSS has potent elastase activity and by processing cytokines and host defense proteins, it also plays a role in the regulation of inflammation.  CTSS has also been linked to G-coupled protein receptor activation and possesses an important intracellular role in major histocompatibility complex class II antigen presentation.  Modulated CTSS activity is also associated with pulmonary disease comorbidities, such as cancer, cardiovascular disease, and diabetes.  CTSS is expressed in a wide variety of immune cells and is biologically active at neutral pH.  Herein, we review the significance of CTSS signaling in pulmonary diseases and associated comorbidities.  We also discuss CTSS as a plausible therapeutic target and describe recent and current clinical trials examining CTSS inhibition as a means for treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfcwhyj6BL4yA3UY9EwTNrfW6udTcc2eaunQDiHcK3f7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38vlvFCqsg%253D%253D&md5=fab651d27425d6c6365a74cb32e24fd2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1186%2Fs12931-020-01381-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12931-020-01381-5%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DR.%26aulast%3DNath%26aufirst%3DS.%26aulast%3DLora%26aufirst%3DA.%26aulast%3DSamaha%26aufirst%3DG.%26aulast%3DElgamal%26aufirst%3DZ.%26aulast%3DKaiser%26aufirst%3DR.%26aulast%3DTaggart%26aufirst%3DC.%26aulast%3DWeldon%26aufirst%3DS.%26aulast%3DGeraghty%26aufirst%3DR.%26atitle%3DCathepsin%2520S%253A%2520investigating%2520an%2520old%2520player%2520in%2520lung%2520disease%2520pathogenesis%252C%2520comorbidities%252C%2520and%2520potential%2520therapeutics%26jtitle%3DRespir.%2520Res.%26date%3D2020%26volume%3D21%26spage%3D111%26doi%3D10.1186%2Fs12931-020-01381-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Insights into the Catalytic Mechanism of Middle East Respiratory Syndrome 3C-Like Protease and Severe Acute Respiratory Syndrome 3C-Like Protease</span>. <i>ACS Catal.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5871</span>– <span class="NLM_lpage">5890</span>, <span class="refDoi"> DOI: 10.1021/acscatal.0c00110</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscatal.0c00110" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXot1WmsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=5871-5890&author=H.+Wangauthor=S.+Heauthor=W.+Dengauthor=Y.+Zhangauthor=G.+Liauthor=J.+Sunauthor=W.+Zhaoauthor=Y.+Guoauthor=Z.+Yinauthor=D.+Liauthor=L.+Shang&title=Comprehensive+Insights+into+the+Catalytic+Mechanism+of+Middle+East+Respiratory+Syndrome+3C-Like+Protease+and+Severe+Acute+Respiratory+Syndrome+3C-Like+Protease&doi=10.1021%2Facscatal.0c00110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Insights into the Catalytic Mechanism of Middle East Respiratory Syndrome 3C-Like Protease and Severe Acute Respiratory Syndrome 3C-Like Protease</span></div><div class="casAuthors">Wang, Hao; He, Shuai; Deng, Weilong; Zhang, Ying; Li, Guobang; Sun, Jixue; Zhao, Wei; Guo, Yu; Yin, Zheng; Li, Dongmei; Shang, Luqing</div><div class="citationInfo"><span class="NLM_cas:title">ACS Catalysis</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5871-5890</span>CODEN:
                <span class="NLM_cas:coden">ACCACS</span>;
        ISSN:<span class="NLM_cas:issn">2155-5435</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Coronavirus 3C-like protease (3CLPro) is a highly conserved cysteine protease employing a catalytic dyad for its functions.  3CLPro is essential to the viral life cycle and, therefore, is an attractive target for developing antiviral agents.  However, the detailed catalytic mechanism of coronavirus 3CLPro remains largely unknown.  We took an integrated approach of employing X-ray crystallog., mutational studies, enzyme kinetics study and inhibitors to gain insights into the mechanism.  Such exptl. work is supplemented by computational studies include pre-reaction state anal. and ab initio calcn. of the crit. catalytic step, and mol. dynamic simulation of the wild-type and mutant enzymes.  Combined, such studies allowed us to identify a residue pair (Glu-His) and a conserved His as crit. for binding; a conserved GSCGS motif as important for the start of catalysis, a partial neg. charge cluster (PNCC) formed by Arg-Tyr-Asp as essential for catalysis, and a conserved water mol. mediating the remote interaction between PNCC and catalytic dyad.  The data collected and our new insights into the detailed mechanism have allowed us to achieve a good understanding of the difference in catalytic efficiency between 3CLPro from SARS and MERS, conduct mutational studies to improve catalytic activity by 8-fold, optimize existing inhibitors to improve potency by 4-fold, and identify a potential allosteric site for inhibitor design.  All such results reinforce each other to support the overall catalytic mechanism proposed herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTVgu_o7NTWrVg90H21EOLACvtfcHk0lhAxyxSLwMJ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXot1WmsL0%253D&md5=49037798a0e22f691f72be4132102c3f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facscatal.0c00110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscatal.0c00110%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DShang%26aufirst%3DL.%26atitle%3DComprehensive%2520Insights%2520into%2520the%2520Catalytic%2520Mechanism%2520of%2520Middle%2520East%2520Respiratory%2520Syndrome%25203C-Like%2520Protease%2520and%2520Severe%2520Acute%2520Respiratory%2520Syndrome%25203C-Like%2520Protease%26jtitle%3DACS%2520Catal.%26date%3D2020%26volume%3D10%26spage%3D5871%26epage%3D5890%26doi%3D10.1021%2Facscatal.0c00110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span>; <span class="NLM_string-name">Jiang, S.</span>; <span class="NLM_string-name">Dai, W.</span>; <span class="NLM_string-name">Lu, L.</span>; <span class="NLM_string-name">Peng, J.</span>; <span class="NLM_string-name">Xia, S.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Li, J.</span>; <span class="NLM_string-name">Jiang, J.</span>; <span class="NLM_string-name">Chen, K.</span></span> <span> </span><span class="NLM_article-title">Ketoamide Compound and Preparation Method, Pharmaceutical Composition and Use Thereof</span>. <span class="NLM_patent">WO2020/030143</span>, September 8, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=H.+Liu&author=S.+Jiang&author=W.+Dai&author=L.+Lu&author=J.+Peng&author=S.+Xia&author=J.+Wang&author=J.+Li&author=J.+Jiang&author=K.+Chen&title=Ketoamide+Compound+and+Preparation+Method%2C+Pharmaceutical+Composition+and+Use+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DKetoamide%2520Compound%2520and%2520Preparation%2520Method%252C%2520Pharmaceutical%2520Composition%2520and%2520Use%2520Thereof%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phizackerley, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, P.</span></span> <span> </span><span class="NLM_article-title">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1107/S0909049502015170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1107%2FS0909049502015170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=12409628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=401-406&author=T.+M.+McPhillipsauthor=S.+E.+McPhillipsauthor=H.+J.+Chiuauthor=A.+E.+Cohenauthor=A.+M.+Deaconauthor=P.+J.+Ellisauthor=E.+Garmanauthor=A.+Gonzalezauthor=N.+K.+Sauterauthor=R.+P.+Phizackerleyauthor=S.+M.+Soltisauthor=P.+Kuhn&title=Blu-Ice+and+the+Distributed+Control+System%3A+software+for+data+acquisition+and+instrument+control+at+macromolecular+crystallography+beamlines&doi=10.1107%2FS0909049502015170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span></div><div class="casAuthors">McPhillips, Timothy M.; McPhillips, Scott E.; Chiu, Hsiu Ju; Cohen, Aina E.; Deacon, Ashley M.; Ellis, Paul J.; Garman, Elspeth; Gonzalez, Ana; Sauter, Nicholas K.; Phizackerley, R. Paul; Soltis, S. Michael; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-406</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Blu-Ice and Distributed Control System (DCS) software packages were developed to provide unified control over the disparate hardware resources available at a macromol. crystallog. beamline.  Blu-Ice is a user interface that provides scientific experimenters and beamline support staff with intuitive graphical tools for collecting diffraction data and configuring beamlines for expts.  Blu-Ice communicates with the hardware at a beamline via DCS, an instrument-control and data-acquisition package designed to integrate hardware resources in a highly heterogeneous networked computing environment.  Together, Blu-Ice and DCS provide a flexible platform for increasing the ease of use, the level of automation and the remote accessibility of beamlines.  Blu-Ice and DCS are currently installed on four Stanford Synchrotron Radiation Lab. crystallog. beamlines and are being implemented at sister light sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVsQUIFJPBLVg90H21EOLACvtfcHk0lhAxyxSLwMJ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D&md5=c77228bd50890463f3e43e981a2e238a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0909049502015170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049502015170%26sid%3Dliteratum%253Aachs%26aulast%3DMcPhillips%26aufirst%3DT.%2BM.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DChiu%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DGarman%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DBlu-Ice%2520and%2520the%2520Distributed%2520Control%2520System%253A%2520software%2520for%2520data%2520acquisition%2520and%2520instrument%2520control%2520at%2520macromolecular%2520crystallography%2520beamlines%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2002%26volume%3D9%26spage%3D401%26epage%3D406%26doi%3D10.1107%2FS0909049502015170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moorhead, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltis, S. M.</span></span> <span> </span><span class="NLM_article-title">Web-Ice: Integrated Data Collection and Analysis for Macromolecular Crystallography</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1107/S0021889807057822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1107%2FS0021889807057822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmvFyntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=176-184&author=A.+Gonzalezauthor=P.+Moorheadauthor=S.+E.+McPhillipsauthor=J.+Songauthor=K.+Sharpauthor=J.+R.+Taylorauthor=P.+D.+Adamsauthor=N.+K.+Sauterauthor=S.+M.+Soltis&title=Web-Ice%3A+Integrated+Data+Collection+and+Analysis+for+Macromolecular+Crystallography&doi=10.1107%2FS0021889807057822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Web-Ice: integrated data collection and analysis for macromolecular crystallography</span></div><div class="casAuthors">Gonzalez, A.; Moorhead, P.; McPhillips, S. E.; Song, J.; Sharp, K.; Taylor, J. R.; Adams, P. D.; Sauter, N. K.; Soltis, S. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">176-184</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">New software tools are introduced to facilitate diffraction expts. involving large nos. of crystals.  While existing programs have long provided a framework for lattice indexing, Bragg spot integration, and symmetry detn., these initial data processing steps often require significant manual effort.  This limits the timely availability of data anal. needed for high-throughput procedures, including the selection of the best crystals from a large sample pool, and the calcn. of optimal data collection parameters to assure complete spot coverage with minimal radiation damage.  To make these protocols more efficient, a network of software applications and application servers has been developed, collectively known as Web-Ice.  When the package is installed at a crystallog. beamline, a programming interface allows the beamline control software (e.g. Blu-Ice, DCSS) to trigger data anal. automatically.  Results are organized based on a list of samples that the user provides, and are examd. within a Web page, accessible both locally at the beamline and remotely.  Optional programming interfaces permit the user to control data acquisition through the Web browser.  The system as a whole is implemented to support multiple users and multiple processors, and can be expanded to provide addnl. scientific functionality.  Web-Ice has a distributed architecture consisting of several stand-alone software components working together via a well defined interface.  Other synchrotrons or institutions may integrate selected components or the whole of Web-Ice with their own data acquisition software.  Updated information about current developments may be obtained at http://smb.slac.stanford.edu/research/developments/webice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCqAzIEM32NbVg90H21EOLACvtfcHk0ljLsSdOFaJivg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmvFyntA%253D%253D&md5=f34468ba2a5441684b9e6f220b4b372e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0021889807057822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807057822%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DMoorhead%26aufirst%3DP.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DSharp%26aufirst%3DK.%26aulast%3DTaylor%26aufirst%3DJ.%2BR.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26atitle%3DWeb-Ice%253A%2520Integrated%2520Data%2520Collection%2520and%2520Analysis%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2008%26volume%3D41%26spage%3D176%26epage%3D184%26doi%3D10.1107%2FS0021889807057822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wardrop, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velter, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forslund, R. E.</span></span> <span> </span><span class="NLM_article-title">Template-Directed C–H Insertion: Synthesis of the Dioxabicyclo[3.2.1]octane Core of the Zaragozic Acids</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">2261</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1021/ol0158361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0158361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1Wit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2001&pages=2261-2264&issue=15&author=D.+J.+Wardropauthor=A.+I.+Velterauthor=R.+E.+Forslund&title=Template-Directed+C%E2%80%93H+Insertion%3A+Synthesis+of+the+Dioxabicyclo%5B3.2.1%5Doctane+Core+of+the+Zaragozic+Acids&doi=10.1021%2Fol0158361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Template-Directed C-H Insertion: Synthesis of the Dioxabicyclo[3.2.1]octane Core of the Zaragozic Acids</span></div><div class="casAuthors">Wardrop, Duncan J.; Velter, Adriana I.; Forslund, Raymond E.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2261-2264</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The prepn. of (±)-(1α,3β,4β,5αβ,6β,7α)-4-benzyloxy-3,5-bis(benzyloxymethyl)-6-hydroxy-1-methyl-7-(triisopropylsilanyloxy)-2,8-dioxabicyclo[3.2.1]octan-6-ol, a model for the core of the zaragozic acids, is reported.  The pivotal reaction in this endeavor is the dirhodium(II)-catalyzed intramol. C-H bond insertion of 2-diazoacetyl-5-benzyloxy-4,6-bis(benzyloxymethyl)-2-methyl-1,3-dioxane (I), a transformation which generates four of the six stereocenters present in the core structure.  A novel method for the diastereoselective synthesis of pyruvic acid acetals was also developed and employed in the prepn. of I from a xylitol deriv.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpipb725ubcpLVg90H21EOLACvtfcHk0ljLsSdOFaJivg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1Wit7c%253D&md5=9b79ff69f8848c58417db38e9dffcdc0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fol0158361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0158361%26sid%3Dliteratum%253Aachs%26aulast%3DWardrop%26aufirst%3DD.%2BJ.%26aulast%3DVelter%26aufirst%3DA.%2BI.%26aulast%3DForslund%26aufirst%3DR.%2BE.%26atitle%3DTemplate-Directed%2520C%25E2%2580%2593H%2520Insertion%253A%2520Synthesis%2520of%2520the%2520Dioxabicyclo%255B3.2.1%255Doctane%2520Core%2520of%2520the%2520Zaragozic%2520Acids%26jtitle%3DOrg.%2520Lett.%26date%3D2001%26volume%3D3%26issue%3D15%26spage%3D2261%26epage%3D2264%26doi%3D10.1021%2Fol0158361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katsuyama, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funabiki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muramatsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, K.</span></span> <span> </span><span class="NLM_article-title">A Convenient and Regioselective Synthesis of 4-Trifluoromethylpyridines</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>1997</i></span>,  <span class="NLM_fpage">1321</span>– <span class="NLM_lpage">1324</span>, <span class="refDoi"> DOI: 10.1055/s-1997-1354</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1055%2Fs-1997-1354" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1997&publication_year=1997&pages=1321-1324&author=I.+Katsuyamaauthor=S.+Ogawaauthor=Y.+Yamaguchiauthor=K.+Funabikiauthor=M.+Matsuiauthor=H.+Muramatsuauthor=K.+Shibata&title=A+Convenient+and+Regioselective+Synthesis+of+4-Trifluoromethylpyridines&doi=10.1055%2Fs-1997-1354"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1055%2Fs-1997-1354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-1997-1354%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuyama%26aufirst%3DI.%26aulast%3DOgawa%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DFunabiki%26aufirst%3DK.%26aulast%3DMatsui%26aufirst%3DM.%26aulast%3DMuramatsu%26aufirst%3DH.%26aulast%3DShibata%26aufirst%3DK.%26atitle%3DA%2520Convenient%2520and%2520Regioselective%2520Synthesis%2520of%25204-Trifluoromethylpyridines%26jtitle%3DSynthesis%26date%3D1997%26volume%3D1997%26spage%3D1321%26epage%3D1324%26doi%3D10.1055%2Fs-1997-1354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, K.</span>; <span class="NLM_string-name">Tadaaki, O.</span>; <span class="NLM_string-name">Toru, M.</span></span> <span> </span><span class="NLM_article-title">Method for Producing Cyclic Diamine Derivative or Salt Thereof</span>. <span class="NLM_patent">WO2004/106323</span> (<span class="NLM_patent">EP1627875</span>), May 28, <span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=K.+Shibuya&author=O.+Tadaaki&author=M.+Toru&title=Method+for+Producing+Cyclic+Diamine+Derivative+or+Salt+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DShibuya%26aufirst%3DK.%26atitle%3DMethod%2520for%2520Producing%2520Cyclic%2520Diamine%2520Derivative%2520or%2520Salt%2520Thereof%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albert, A.</span>, <span class="NLM_string-name">Serjeant, E. P.</span></span> <i>The Determination of Ionization Constants. A Laboratory Manual</i>, <span class="NLM_edition">3</span>rd ed.; <span class="NLM_publisher-name">Chapman and Hall</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1984</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1007%2F978-94-009-5548-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1984&author=A.+Albert&author=E.+P.+Serjeant&title=The+Determination+of+Ionization+Constants.+A+Laboratory+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1007%2F978-94-009-5548-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-009-5548-6%26sid%3Dliteratum%253Aachs%26aulast%3DAlbert%26aufirst%3DA.%26btitle%3DThe%2520Determination%2520of%2520Ionization%2520Constants.%2520A%2520Laboratory%2520Manual%26pub%3DChapman%2520and%2520Hall%26date%3D1984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1a','cit1b'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8a','cit8b','cit8c','cit8d'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18a','cit18b'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25a','cit25b','cit25c'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31a','cit31b','cit31c'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34a','cit34b'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44a','cit44b'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47','cit47b'],'ref48':['cit48a','cit48b','cit48c','cit48d'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0011.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Published protease inhibitors for rhinovirus 3CP (<b>1</b>, rupintrivir), norovirus 3CLP <b>2</b>, and SARS 3CLP (<b>3</b>, PF-835231).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Exploratory noncytotoxic, norovirus project compounds <b>4</b> and <b>5</b> that were active when screened against SARS-CoV-2 3CLP and viral replication. Structure of GC-373 and its bisulfite adduct GC-376 that are reported SARS-CoV-2 3CLP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Potential mechanisms for the α-acyloxymethylketone covalent inhibition of cysteine proteases with the sulfur either directly displacing the acyloxygroup (red arrows) or the sulfur attacking the carbon of the ketone (blue arrows) followed by sulfur migration. Both mechanisms produce the same product.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of α-Acyloxymethylketone Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) Sodium chloroacetate, Et<sub>3</sub>N, <i>t</i>-BuMgCl, THF, 63% yield (<i>n</i> = 1), 60% yield (<i>n</i> = 0); (ii) HCl in dioxane, CHCl<sub>3</sub>; (iii) Boc-<span class="smallcaps smallerCapital">l</span>-leucine, HATU, NMM, DMF, 54% yield (two steps, <i>n</i> = 1), 47% yield (two steps, <i>n</i> = 0); (iv) 4-methoxyindole-CO<sub>2</sub>H, HATU, NMM, DMF, 54% yield (two steps, <i>n</i> = 1), 42% (two steps, <i>n</i> = 0); (v) carboxylic acid, NaO<i>t</i>-Bu, NaI, DMF, 15–80% yield.</p></p></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. SARS-CoV-2 3CLP and antiviral results for five-membered lactam <b>16m</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/medium/jm1c00616_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0005.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cocrystal structure of chain C covalent adduct from <b>15l</b> reacting with SARS-CoV-2 3CLP (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7MBI">7MBI</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00616/20210720/images/large/jm1c00616_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00616&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i59">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12347" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12347" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Baltimore, D.</span></span> <span> </span><span class="NLM_article-title">Expression of Animal Virus Genomes</span>. <i>Bacteriol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1128/br.35.3.235-241.1971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1128%2Fbr.35.3.235-241.1971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=4329869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaE38Xjt1WisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1971&pages=235-241&author=D.+Baltimore&title=Expression+of+Animal+Virus+Genomes&doi=10.1128%2Fbr.35.3.235-241.1971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of animal virus genomes</span></div><div class="casAuthors">Baltimore, David</div><div class="citationInfo"><span class="NLM_cas:title">Bacteriological Reviews</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">235-41</span>CODEN:
                <span class="NLM_cas:coden">BAREA8</span>;
        ISSN:<span class="NLM_cas:issn">0005-3678</span>.
    </div><div class="casAbstract">A review.  A comparison of gene expression in 3 viral systems indicates the diversity of behavior which exists.  The picornaviruses encode multiple protein species in a single species.  The rhabdoviruses and paramyxoviruses carry many genes in a single RNA but synthesize sep., complementary RNA species.  The RNA tumor viruses probably transfer their genetic information from RNA to DNA, where it can be expressed by DNA-dependent RNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdBXn3wvvXqbVg90H21EOLACvtfcHk0ljme6VMWZ9zCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE38Xjt1WisA%253D%253D&md5=573e122f25b5127ddaa234b3822cf779</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1128%2Fbr.35.3.235-241.1971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fbr.35.3.235-241.1971%26sid%3Dliteratum%253Aachs%26aulast%3DBaltimore%26aufirst%3DD.%26atitle%3DExpression%2520of%2520Animal%2520Virus%2520Genomes%26jtitle%3DBacteriol.%2520Rev.%26date%3D1971%26volume%3D35%26spage%3D235%26epage%3D241%26doi%3D10.1128%2Fbr.35.3.235-241.1971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, J. J.</span></span> <span> </span><span class="NLM_article-title">Group IV Viruses: Single-Stranded (+)Sense RNA</span>. <i>Taxonomic Guide to Infectious Diseases</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">246</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-415895-5.00042-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2FB978-0-12-415895-5.00042-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=237-246&author=J.+J.+Berman&title=Group+IV+Viruses%3A+Single-Stranded+%28%2B%29Sense+RNA&doi=10.1016%2FB978-0-12-415895-5.00042-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-415895-5.00042-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-415895-5.00042-8%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DJ.%2BJ.%26atitle%3DGroup%2520IV%2520Viruses%253A%2520Single-Stranded%2520%2528%252B%2529Sense%2520RNA%26jtitle%3DTaxonomic%2520Guide%2520to%2520Infectious%2520Diseases%26date%3D2012%26spage%3D237%26epage%3D246%26doi%3D10.1016%2FB978-0-12-415895-5.00042-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marakovits, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejto, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harr, J. E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosa, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 4. Incorporation of P1 Lactam Moieties as L-Glutamine Replacements</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1213</span>– <span class="NLM_lpage">1224</span>, <span class="refDoi"> DOI: 10.1021/jm9805384</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9805384" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1213-1224&author=P.+S.+Dragovichauthor=T.+J.+Prinsauthor=R.+Zhouauthor=S.+E.+Webberauthor=J.+T.+Marakovitsauthor=S.+A.+Fuhrmanauthor=A.+K.+Patickauthor=D.+A.+Matthewsauthor=C.+A.+Leeauthor=C.+E.+Fordauthor=B.+J.+Burkeauthor=R.+A.+Rejtoauthor=T.+F.+Hendricksonauthor=T.+Tuntlandauthor=E.+L.+Brownauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=J.+E.+V.+Harrauthor=M.+B.+Kosaauthor=S.+T.+Worland&title=Structure-Based+Design%2C+Synthesis%2C+and+Biological+Evaluation+of+Irreversible+Human+Rhinovirus+3C+Protease+Inhibitors.+4.+Incorporation+of+P1+Lactam+Moieties+as+L-Glutamine+Replacements&doi=10.1021%2Fjm9805384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 4. Incorporation of P1 Lactam Moieties as L-Glutamine Replacements</span></div><div class="casAuthors">Dragovich, Peter S.; Prins, Thomas J.; Zhou, Ru; Webber, Stephen E.; Marakovits, Joseph T.; Fuhrman, Shella A.; Patick, Amy K.; Matthews, David A.; Lee, Caroline A.; Ford, Clifford E.; Burke, Benjamin J.; Rejto, Paul A.; Hendrickson, Thomas F.; Tuntland, Tove; Brown, Edward L.; Meador, James W., III; Ferre, Rose Ann; Harr, James E. V.; Kosa, Maha B.; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1213-1224</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-based design, chem. synthesis, and biol. evaluation of various human rhinovirus (HRV) 3C protease (3CP) inhibitors, e.g. I [R = CH2CHMe2, CHMe2; X = (S)-CH, (R)-CH, N; n = 1, 2] and II (R = CHMe2, CMe3; Y = NH, CH2) which incorporate P1 lactam moieties in lieu of an L-glutamine residue are described.  These compds. are comprised of a tripeptidyl or peptidomimetic binding determinant and an Et propenoate Michael acceptor moiety which forms an irreversible covalent adduct with the active site cysteine residue of the 3C enzyme.  The P1-lactam-contg. inhibitors display significantly increased 3CP inhibition activity along with improved antirhinoviral properties relative to corresponding L-glutamine-derived mols.  In addn., several lactam-contg. compds. exhibit excellent selectivity for HRV 3CP over several other serine and cysteine proteases and are not appreciably degraded by a variety of biol. agents.  Lactam II (R = CHMe2, Y = CH2, n = 1) (AG7088) was one of the most potent inhibitors (mean antirhinoviral EC90 ≈ 0.10 μM, n = 46 serotypes), and is shown to warrant addnl. preclin. development to explore its potential for use as an antirhinoviral agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlBuxFPwMr97Vg90H21EOLACvtfcHk0ljme6VMWZ9zCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D&md5=0e2bf5a03d17ea371663a252e7b97b20</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm9805384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9805384%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DMarakovits%26aufirst%3DJ.%2BT.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBurke%26aufirst%3DB.%2BJ.%26aulast%3DRejto%26aufirst%3DR.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DHarr%26aufirst%3DJ.%2BE.%2BV.%26aulast%3DKosa%26aufirst%3DM.%2BB.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DStructure-Based%2520Design%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520Irreversible%2520Human%2520Rhinovirus%25203C%2520Protease%2520Inhibitors.%25204.%2520Incorporation%2520of%2520P1%2520Lactam%2520Moieties%2520as%2520L-Glutamine%2520Replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1213%26epage%3D1224%26doi%3D10.1021%2Fjm9805384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Center for Disease Control and Prevention</span>. <a href="https://www.cdc.gov/norovirus/trends-outbreaks/worldwide.html" class="extLink">https://www.cdc.gov/norovirus/trends-outbreaks/worldwide.html</a> (accessed 2021-02-06).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Center+for+Disease+Control+and+Prevention.+https%3A%2F%2Fwww.cdc.gov%2Fnorovirus%2Ftrends-outbreaks%2Fworldwide.html+%28accessed+2021-02-06%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginex, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maestro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrado-Gil, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Geijo, M.Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urquiza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramírez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span> <span> </span><span class="NLM_article-title">COVID-19: Drug Targets and Potential Treatments</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12359</span>– <span class="NLM_lpage">12386</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00606</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00606" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12359-12386&author=C.+Gilauthor=T.+Ginexauthor=I.+Maestroauthor=V.+Nozalauthor=L.+Barrado-Gilauthor=M.%C3%81.+Cuesta-Geijoauthor=J.+Urquizaauthor=D.+Ram%C3%ADrezauthor=C.+Alonsoauthor=N.+E.+Campilloauthor=A.+Martinez&title=COVID-19%3A+Drug+Targets+and+Potential+Treatments&doi=10.1021%2Facs.jmedchem.0c00606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19: Drug Targets and Potential Treatments</span></div><div class="casAuthors">Gil, Carmen; Ginex, Tiziana; Maestro, Ines; Nozal, Vanesa; Barrado-Gil, Lucia; Cuesta-Geijo, Miguel Angel; Urquiza, Jesus; Ramirez, David; Alonso, Covadonga; Campillo, Nuria E.; Martinez, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12359-12386</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Currently, the authors are immersed in a pandemic caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which severely threatens public health worldwide.  Until now, no drug or vaccine has been approved to treat the severe disease caused by this coronavirus, COVID-19.  The authors will focus on the main virus-based and host-based targets that can guide medicinal chem. efforts to discover new drugs for this devastating disease.  In principle, all CoVs enzymes and proteins involved in viral replication and the control of host cellular machineries are potentially druggable targets in the search for therapeutic options for SARS-CoV-2.  This perspective provides an overview of the main targets from a structural point of view, together with reported therapeutic compds. with activity against SARS-CoV-2 and/or other CoVs.  Also, the role of innate immune response to coronavirus infection and the related therapeutic options will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsg-YRktOknrVg90H21EOLACvtfcHk0lh8eQNVAzT3Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I&md5=6a0dc17c5f601f09a92da933f81bbbea</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00606%26sid%3Dliteratum%253Aachs%26aulast%3DGil%26aufirst%3DC.%26aulast%3DGinex%26aufirst%3DT.%26aulast%3DMaestro%26aufirst%3DI.%26aulast%3DNozal%26aufirst%3DV.%26aulast%3DBarrado-Gil%26aufirst%3DL.%26aulast%3DCuesta-Geijo%26aufirst%3DM.%25C3%2581.%26aulast%3DUrquiza%26aufirst%3DJ.%26aulast%3DRam%25C3%25ADrez%26aufirst%3DD.%26aulast%3DAlonso%26aufirst%3DC.%26aulast%3DCampillo%26aufirst%3DN.%2BE.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCOVID-19%253A%2520Drug%2520Targets%2520and%2520Potential%2520Treatments%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12359%26epage%3D12386%26doi%3D10.1021%2Facs.jmedchem.0c00606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, S.K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, A.K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, C.C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, P. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, K.-Y.</span></span> <span> </span><span class="NLM_article-title">Molecular Epidemiology of Human Coronavirus OC43 Reveals Evolution of Different Genotypes over Time and Recent Emergence of a Novel Genotype due to Natural Recombination</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>85</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">11325</span>– <span class="NLM_lpage">11337</span>, <span class="refDoi"> DOI: 10.1128/JVI.05512-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1128%2FJVI.05512-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=21849456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCltr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=11325-11337&issue=21&author=S.K.+P.+Lauauthor=P.+Leeauthor=A.K.+L.+Tsangauthor=C.C.+Y.+Yipauthor=H.+Tseauthor=R.+A.+Leeauthor=L.-Y.+Soauthor=Y.-L.+Lauauthor=K.-H.+Chanauthor=P.+C.+Y.+Wooauthor=K.-Y.+Yuen&title=Molecular+Epidemiology+of+Human+Coronavirus+OC43+Reveals+Evolution+of+Different+Genotypes+over+Time+and+Recent+Emergence+of+a+Novel+Genotype+due+to+Natural+Recombination&doi=10.1128%2FJVI.05512-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular epidemiology of human coronavirus OC43 reveals evolution of different genotypes over time and recent emergence of a novel genotype due to natural recombination</span></div><div class="casAuthors">Lau, Susanna K. P.; Lee, Paul; Tsang, Alan K. L.; Yip, Cyril C. Y.; Tse, Herman; Lee, Rodney A.; So, Lok-Yee; Lau, Yu-Lung; Chan, Kwok-Hung; Woo, Patrick C. Y.; Yuen, Kwok-Yung</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11325-11337</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Although human coronavirus OC43-OC43 (HCoV-OC43) is the coronavirus most commonly assocd. with human infections, little is known about its mol. epidemiol. and evolution.  We conducted a mol. epidemiol. study to investigate different genotypes and potential recombination in HCoV-OC43.  Twenty-nine HCoV-OC43 strains from nasopharyngeal aspirates, collected from 2004 to 2011, were subjected to RNA-dependent RNA polymerase (RdRp), spike, and nucleocapsid gene anal.  Phylogenetic anal. showed at least three distinct clusters of HCoV-OC43, although 10 unusual strains displayed incongruent phylogenetic positions between RdRp and spike genes.  This suggested the presence of four HCoV-OC43 genotypes (A to D), with genotype D most likely arising from recombination.  The complete genome sequencing of two genotype C and D strains and bootscan anal. showed recombination events between genotypes B and C in the generation of genotype D.  Of the 29 strains, none belonged to the more ancient genotype A, 5 from 2004 belonged to genotype B, 15 from 2004 to 2006 belonged to genotype C, and 1 from 2004 and all 8 from 2008 to 2011 belonged to the recombinant genotype D.  Mol. clock anal. using spike and nucleocapsid genes dated the most recent common ancestor of all genotypes to the 1950s, genotype B and C to the 1980s, genotype B to the 1990s, and genotype C to the late 1990s to early 2000s, while the recombinant genotype D strains were detected as early as 2004.  This represents the first study to describe natural recombination in HCoV-OC43 and the evolution of different genotypes over time, leading to the emergence of novel genotype D, which is assocd. with pneumonia in our elderly population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSYvYE-NhLPbVg90H21EOLACvtfcHk0lh8eQNVAzT3Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCltr3E&md5=ca95f357f42cea988b79ebbca43cf567</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FJVI.05512-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.05512-11%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DS.K.%2BP.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DTsang%26aufirst%3DA.K.%2BL.%26aulast%3DYip%26aufirst%3DC.C.%2BY.%26aulast%3DTse%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DR.%2BA.%26aulast%3DSo%26aufirst%3DL.-Y.%26aulast%3DLau%26aufirst%3DY.-L.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DWoo%26aufirst%3DP.%2BC.%2BY.%26aulast%3DYuen%26aufirst%3DK.-Y.%26atitle%3DMolecular%2520Epidemiology%2520of%2520Human%2520Coronavirus%2520OC43%2520Reveals%2520Evolution%2520of%2520Different%2520Genotypes%2520over%2520Time%2520and%2520Recent%2520Emergence%2520of%2520a%2520Novel%2520Genotype%2520due%2520to%2520Natural%2520Recombination%26jtitle%3DJ.%2520Virol.%26date%3D2011%26volume%3D85%26issue%3D21%26spage%3D11325%26epage%3D11337%26doi%3D10.1128%2FJVI.05512-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Centers for Disease Control and Prevention</span>. <a href="https://www.cdc.gov/coronavirus/types.html" class="extLink">https://www.cdc.gov/coronavirus/types.html</a> (accessed 2021-02-06).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Centers+for+Disease+Control+and+Prevention.+https%3A%2F%2Fwww.cdc.gov%2Fcoronavirus%2Ftypes.html+%28accessed+2021-02-06%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anirudhan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, L.</span></span> <span> </span><span class="NLM_article-title">Targeting SARS-CoV-2 Viral Proteases as a Therapeutic Strategy to Treat COVID-19</span>. <i>J. Med. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2722</span>– <span class="NLM_lpage">2734</span>, <span class="refDoi"> DOI: 10.1002/jmv.26814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fjmv.26814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33475167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjsVGgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2021&pages=2722-2734&author=V.+Anirudhanauthor=H.+Leeauthor=H.+Chengauthor=L.+Cooperauthor=L.+Rong&title=Targeting+SARS-CoV-2+Viral+Proteases+as+a+Therapeutic+Strategy+to+Treat+COVID-19&doi=10.1002%2Fjmv.26814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting SARS-CoV-2 viral proteases as a therapeutic strategy to treat COVID-19</span></div><div class="casAuthors">Anirudhan, Varada; Lee, Hyun; Cheng, Han; Cooper, Laura; Rong, Lijun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medical Virology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2722-2734</span>CODEN:
                <span class="NLM_cas:coden">JMVIDB</span>;
        ISSN:<span class="NLM_cas:issn">0146-6615</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The 21st century has witnessed three outbreaks of coronavirus (CoVs) infections caused by severe acute respiratory syndrome (SARS)-CoV, Middle East respiratory syndrome (MERS)-CoV, and SARS-CoV-2.  Coronavirus disease 2019 (COVID-19), caused by SARS-CoV-2, spreads rapidly and since the discovery of the first COVID-19 infection in Dec. 2019, has caused 1.2 million deaths worldwide and 226,777 deaths in the United States alone.  The high amino acid similarity between SARS-CoV and SARS-CoV-2 viral proteins supports testing therapeutic mols. that were designed to treat SARS infections during the 2003 epidemic.  In this review, we provide information on possible COVID-19 treatment strategies that act via inhibition of the two essential proteins of the virus, 3C-like protease (3CLpro) or papain-like protease (PLpro).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPAU7_-rnfw7Vg90H21EOLACvtfcHk0lj51mz7gJeIkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjsVGgtb4%253D&md5=4dcced5135ff23b9b1a9de385a4d952f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fjmv.26814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmv.26814%26sid%3Dliteratum%253Aachs%26aulast%3DAnirudhan%26aufirst%3DV.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DCooper%26aufirst%3DL.%26aulast%3DRong%26aufirst%3DL.%26atitle%3DTargeting%2520SARS-CoV-2%2520Viral%2520Proteases%2520as%2520a%2520Therapeutic%2520Strategy%2520to%2520Treat%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Virol.%26date%3D2021%26volume%3D93%26spage%3D2722%26epage%3D2734%26doi%3D10.1002%2Fjmv.26814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Long, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orr, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, J.</span></span> <span> </span><span class="NLM_article-title">A consensus sequence for substrate hydrolysis by rhinovirus 3C proteinase</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>258</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/0014-5793(89)81619-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2F0014-5793%2889%2981619-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=2556299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK3cXpsVCnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=258&publication_year=1989&pages=75-78&issue=1&author=A.+C.+Longauthor=D.+C.+Orrauthor=J.+M.+Cameronauthor=B.+M.+Dunnauthor=J.+Kay&title=A+consensus+sequence+for+substrate+hydrolysis+by+rhinovirus+3C+proteinase&doi=10.1016%2F0014-5793%2889%2981619-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8aR"><div class="casContent"><span class="casTitleNuber">8a</span><div class="casTitle"><span class="NLM_cas:atitle">A consensus sequence for substrate hydrolysis by rhinovirus 3C proteinase</span></div><div class="casAuthors">Long, Audrey C.; Orr, David C.; Cameron, Janet M.; Dunn, Ben M.; Kay, John</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">258</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-8</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    </div><div class="casAbstract">Kinetic consts. were detd. for the hydrolysis of a series of synthetic peptide substrates by recombinant rhinovirus (HRV 14) 3C proteinase.  Systematic removal or replacement of individual residues indicated that the min. sequence required for effective cleavage by the viral cysteine proteinase was P5-Val/Tyhr-P3-P2-Gln-Gly-Pro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU2X1pA77VGrVg90H21EOLACvtfcHk0lj51mz7gJeIkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXpsVCnsg%253D%253D&md5=c3a8c8e64b24d0e329e84270242207a6</span></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=10.1016%2F0014-5793%2889%2981619-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-5793%252889%252981619-9%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DA.%2BC.%26aulast%3DOrr%26aufirst%3DD.%2BC.%26aulast%3DCameron%26aufirst%3DJ.%2BM.%26aulast%3DDunn%26aufirst%3DB.%2BM.%26aulast%3DKay%26aufirst%3DJ.%26atitle%3DA%2520consensus%2520sequence%2520for%2520substrate%2520hydrolysis%2520by%2520rhinovirus%25203C%2520proteinase%26jtitle%3DFEBS%2520Lett.%26date%3D1989%26volume%3D258%26issue%3D1%26spage%3D75%26epage%3D78%26doi%3D10.1016%2F0014-5793%2889%2981619-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit8b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">May, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanathan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, K.K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korba, B.</span></span> <span> </span><span class="NLM_article-title">The P4-P2′ Amino Acids Surrounding Human Norovirus Polyprotein Cleavage Sites Define the Core Sequence Regulating Self-Processing Order</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>88</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">10738</span>– <span class="NLM_lpage">10747</span>, <span class="refDoi"> DOI: 10.1128/JVI.01357-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1128%2FJVI.01357-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=24991013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ygtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2014&pages=10738-10747&issue=18&author=J.+Mayauthor=P.+Viswanathanauthor=K.K.-S.+Ngauthor=A.+Medvedevauthor=B.+Korba&title=The+P4-P2%E2%80%B2+Amino+Acids+Surrounding+Human+Norovirus+Polyprotein+Cleavage+Sites+Define+the+Core+Sequence+Regulating+Self-Processing+Order&doi=10.1128%2FJVI.01357-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8bR"><div class="casContent"><span class="casTitleNuber">8b</span><div class="casTitle"><span class="NLM_cas:atitle">The P4-P2' amino acids surrounding human norovirus polyprotein cleavage sites define the core sequence regulating self-processing order</span></div><div class="casAuthors">May, Jared; Viswanathan, Prasanth; Ng, Kenneth K.-S.; Medvedev, Alexei; Korba, Brent</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10738-10747, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Noroviruses (NoV) are members of the family Caliciviridae.  The human NoV open reading frame 1 (ORF1) encodes a 200-kDa polyprotein which is cleaved by the viral 20-kDa 3C-like protease (Pro, NS6) into 6 nonstructural proteins that are necessary for viral replication.  The NoV ORF1 polyprotein is processed in a specific order, with "early" sites (NS1/2-3 and NS3-4) being cleaved rapidly and three "late" sites (NS4-5, NS5-6, and NS6-7) processed subsequently and less efficiently.  Previously, we demonstrated that the NoV polyprotein processing order is directly correlated with the efficiency of the enzyme, which is regulated by the primary amino acid sequences surrounding ORF1 cleavage sites.  Using fluorescence resonance energy transfer (FRET) peptides representing the NS2-3 and NS6-7 ORF1 cleavage sites, we now demonstrate that the amino acids spanning positions P4 to P2' (P4-P2') surrounding each site comprise the core sequence controlling NoV protease enzyme efficiency.  Furthermore, the NoV polyprotein self-processing order can be altered by interchanging this core sequence between NS2-3 and any of the three late sites in in vitro transcription-translation assays.  We also demonstrate that the nature of the side chain at the P3 position for the NS1/2-3 (Nterm/NTPase) site confers significant influence on enzyme catalysis (kcat and kcat/Km), a feature overlooked in previous structural studies.  Mol. modeling provides possible explanations for the P3 interactions with NoV protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXvwzhPOquPLVg90H21EOLACvtfcHk0lj51mz7gJeIkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ygtb3I&md5=c241cbf13509bcc9aa36a2729535937e</span></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1128%2FJVI.01357-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01357-14%26sid%3Dliteratum%253Aachs%26aulast%3DMay%26aufirst%3DJ.%26aulast%3DViswanathan%26aufirst%3DP.%26aulast%3DNg%26aufirst%3DK.K.-S.%26aulast%3DMedvedev%26aufirst%3DA.%26aulast%3DKorba%26aufirst%3DB.%26atitle%3DThe%2520P4-P2%25E2%2580%25B2%2520Amino%2520Acids%2520Surrounding%2520Human%2520Norovirus%2520Polyprotein%2520Cleavage%2520Sites%2520Define%2520the%2520Core%2520Sequence%2520Regulating%2520Self-Processing%2520Order%26jtitle%3DJ.%2520Virol.%26date%3D2014%26volume%3D88%26issue%3D18%26spage%3D10738%26epage%3D10747%26doi%3D10.1128%2FJVI.01357-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit8c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">What coronavirus 3C-like protease tells us: From structure, substrate selectivity, to inhibitor design</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1965</span>– <span class="NLM_lpage">1998</span>, <span class="refDoi"> DOI: 10.1002/med.21783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fmed.21783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33460213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtl2jsbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2021&pages=1965-1998&issue=4&author=M.+Xiongauthor=H.+Suauthor=W.+Zhaoauthor=H.+Xieauthor=Q.+Shaoauthor=Y.+Xu&title=What+coronavirus+3C-like+protease+tells+us%3A+From+structure%2C+substrate+selectivity%2C+to+inhibitor+design&doi=10.1002%2Fmed.21783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8cR"><div class="casContent"><span class="casTitleNuber">8c</span><div class="casTitle"><span class="NLM_cas:atitle">What coronavirus 3C-like protease tells us: From structure, substrate selectivity, to inhibitor design</span></div><div class="casAuthors">Xiong, Muya; Su, Haixia; Zhao, Wenfeng; Xie, Hang; Shao, Qiang; Xu, Yechun</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1965-1998</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The emergence of a variety of coronaviruses (CoVs) in the last decades has posed huge threats to human health.  Esp., the ongoing pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has led to more than 70 million infections and over 1.6 million of deaths worldwide in the past few months.  None of the efficacious antiviral agents against human CoVs have been approved yet. 3C-like protease (3CLpro) is an attractive target for antiviral intervention due to its essential role in processing polyproteins translated from viral RNA, and its conserved structural feature and substrate specificity among CoVs in spite of the sequence variation.  This review focuses on all available crystal structures of 12 CoV 3CLpros and their inhibitors, and intends to provide a comprehensive understanding of this protease from multiple aspects including its structural features, substrate specificity, inhibitor binding modes, and more importantly, to recapitulate the similarity and diversity among different CoV 3CLpros and the structure-activity relationship of various types of inhibitors.  Such an attempt could gain a deep insight into the inhibition mechanisms and drive future structure-based drug discovery targeting 3CLpros.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHnlfhAJGwSrVg90H21EOLACvtfcHk0lj51mz7gJeIkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtl2jsbvP&md5=d69289782b99eba0be757942e3172e56</span></div><a href="/servlet/linkout?suffix=cit8c&amp;dbid=16384&amp;doi=10.1002%2Fmed.21783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21783%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DWhat%2520coronavirus%25203C-like%2520protease%2520tells%2520us%253A%2520From%2520structure%252C%2520substrate%2520selectivity%252C%2520to%2520inhibitor%2520design%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2021%26volume%3D41%26issue%3D4%26spage%3D1965%26epage%3D1998%26doi%3D10.1002%2Fmed.21783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit8d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusov, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Brunn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wilde, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">α-Ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: structure-based design, synthesis, and activity assessment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4562</span>– <span class="NLM_lpage">4578</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01828</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01828" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4562-4578&issue=9&author=L.+Zhangauthor=D.+Linauthor=Y.+Kusovauthor=Y.+Nianauthor=Q.+Maauthor=J.+Wangauthor=A.+von+Brunnauthor=P.+Leyssenauthor=K.+Lankoauthor=J.+Neytsauthor=A.+de+Wildeauthor=E.+J.+Snijderauthor=H.+Liuauthor=R.+Hilgenfeld&title=%CE%B1-Ketoamides+as+broad-spectrum+inhibitors+of+coronavirus+and+enterovirus+replication%3A+structure-based+design%2C+synthesis%2C+and+activity+assessment&doi=10.1021%2Facs.jmedchem.9b01828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8dR"><div class="casContent"><span class="casTitleNuber">8d</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketoamides as Broad-Spectrum Inhibitors of Coronavirus and Enterovirus Replication: Structure-Based Design, Synthesis, and Activity Assessment</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Kusov, Yuri; Nian, Yong; Ma, Qingjun; Wang, Jiang; von Brunn, Albrecht; Leyssen, Pieter; Lanko, Kristina; Neyts, Johan; de Wilde, Adriaan; Snijder, Eric J.; Liu, Hong; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4562-4578</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate.  Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs.  In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic α-ketoamides as inhibitors of main and 3C proteases.  Six crystal structures of protease:inhibitor complexes were detd. as part of this study.  Compds. synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic.  Optimization of the P2 substituent of the α-ketoamides proved crucial for achieving near-equipotency against the three virus genera.  The best near-equipotent inhibitors, (I) [R1 = cyclopentylmethyl (P2) and R1 = cyclohexylmethyl (P2)], display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures.  In Huh7 cells, I [R1 = cyclohexylmethyl (P2)] exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWpcfl4wTRLLVg90H21EOLACvtfcHk0lg8PqjeuWt4bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D&md5=60d91a8ecf916856a844d835ecf84d5f</span></div><a href="/servlet/linkout?suffix=cit8d&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01828%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DKusov%26aufirst%3DY.%26aulast%3DNian%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3Dvon%2BBrunn%26aufirst%3DA.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DLanko%26aufirst%3DK.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3Dde%2BWilde%26aufirst%3DA.%26aulast%3DSnijder%26aufirst%3DE.%2BJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3D%25CE%25B1-Ketoamides%2520as%2520broad-spectrum%2520inhibitors%2520of%2520coronavirus%2520and%2520enterovirus%2520replication%253A%2520structure-based%2520design%252C%2520synthesis%252C%2520and%2520activity%2520assessment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D9%26spage%3D4562%26epage%3D4578%26doi%3D10.1021%2Facs.jmedchem.9b01828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwaltney, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi-Burris, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gersten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsyu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span> <span> </span><span class="NLM_article-title">Randomized, Double-Blind, Placebo-Controlled Studies of Ruprintrivir Nasal Spray 2-Percent Suspension for Prevention and Treatment of Experimentally Induced Rhinovirus Colds in Healthy Volunteers</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3907</span>– <span class="NLM_lpage">3916</span>, <span class="refDoi"> DOI: 10.1128/AAC.47.12.3907-3916.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1128%2FAAC.47.12.3907-3916.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=14638501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsV2nu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2003&pages=3907-3916&issue=12&author=F.+G.+Haydenauthor=R.+B.+Turnerauthor=J.+M.+Gwaltneyauthor=K.+Chi-Burrisauthor=M.+Gerstenauthor=P.+Hsyuauthor=A.+K.+Patickauthor=G.+J.+Smithauthor=L.+S.+Zalman&title=Randomized%2C+Double-Blind%2C+Placebo-Controlled+Studies+of+Ruprintrivir+Nasal+Spray+2-Percent+Suspension+for+Prevention+and+Treatment+of+Experimentally+Induced+Rhinovirus+Colds+in+Healthy+Volunteers&doi=10.1128%2FAAC.47.12.3907-3916.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Phase II, randomized, double-blind, placebo-controlled studies of ruprintrivir nasal spray 2% suspension for prevention and treatment of experimentally induced rhinovirus colds in healthy volunteers</span></div><div class="casAuthors">Hayden, Frederick G.; Turner, Ronald B.; Gwaltney, Jack M.; Chi-burris, Kathy; Gersten, Merril; Hsyu, Poe; Patick, Amy K.; Smith, George J., III; Zalman, Leora S.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3907-3916</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Human rhinovirus (HRV) infections are usually self-limited but may be assocd. with serious consequences, particularly in those with asthma and chronic respiratory disease.  Effective antiviral agents are needed for preventing and treating HRV illnesses.  Ruprintrivir (Agouron Pharmaceuticals, Inc., San Diego, Calif.) selectively inhibits HRV 3C protease and shows potent, broad-spectrum anti-HRV activity in vitro.  We conducted three double-blind, placebo-controlled clin. trials in 202 healthy volunteers to assess the activity of ruprintrivir in exptl. HRV infection.  Subjects were randomized to receive intranasal ruprintrivir (8 mg) or placebo sprays as prophylaxis (two or five times daily [2×/day or 5×/day] for 5 days) starting 6 h before infection or as treatment (5×/day for 4 days) starting 24 h after infection.  Ruprintrivir prophylaxis reduced the proportion of subjects with pos. viral cultures (for 5×/day dosing groups, 44% for ruprintrivir treatment group vs. 70% for placebo treatment group [P = 0.03]; for 2×/day dosing groups, 60% for ruprintrivir group vs. 92% for placebo group [P = 0.004]) and viral titers but did not decrease the frequency of colds.  Ruprintrivir treatment reduced the mean total daily symptom score (2.2 for ruprintrivir treatment group and 3.3 for the placebo treatment group [P = 0.014]) by 33%.  Secondary endpoints, including viral titers, individual symptom scores, and nasal discharge wts., were also reduced by ruprintrivir treatment.  Overall, ruprintrivir was well tolerated; blood-tinged mucus and nasal passage irritation were the most common adverse effects reported.  Pharmacokinetic anal. of plasma and nasal ruprintrivir concns. revealed intranasal drug residence with minimal systemic absorption.  Results from these studies in exptl. rhinoviral infection support continued investigation of intranasal ruprintrivir in the setting of natural HRV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoudaxRiBJLg7Vg90H21EOLACvtfcHk0lg8PqjeuWt4bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsV2nu7w%253D&md5=6d6c33f56af0c4f45145b291ace2dba1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FAAC.47.12.3907-3916.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.47.12.3907-3916.2003%26sid%3Dliteratum%253Aachs%26aulast%3DHayden%26aufirst%3DF.%2BG.%26aulast%3DTurner%26aufirst%3DR.%2BB.%26aulast%3DGwaltney%26aufirst%3DJ.%2BM.%26aulast%3DChi-Burris%26aufirst%3DK.%26aulast%3DGersten%26aufirst%3DM.%26aulast%3DHsyu%26aufirst%3DP.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26atitle%3DRandomized%252C%2520Double-Blind%252C%2520Placebo-Controlled%2520Studies%2520of%2520Ruprintrivir%2520Nasal%2520Spray%25202-Percent%2520Suspension%2520for%2520Prevention%2520and%2520Treatment%2520of%2520Experimentally%2520Induced%2520Rhinovirus%2520Colds%2520in%2520Healthy%2520Volunteers%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2003%26volume%3D47%26issue%3D12%26spage%3D3907%26epage%3D3916%26doi%3D10.1128%2FAAC.47.12.3907-3916.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti
Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerawarna, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uy, R. A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damalanka, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehzabeen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design and Optimization of Dipeptidyl Inhibitors of Norovirus 3CL Protease. Structure–Activity Relationships and Biochemical, X-ray Crystallographic, Cell-Based, and <i>In Vivo</i> Studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3144</span>– <span class="NLM_lpage">3155</span>, <span class="refDoi"> DOI: 10.1021/jm5019934</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5019934" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktlertr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3144-3155&author=A.+C.+Galasiti%0AKankanamalageauthor=Y.+Kimauthor=P.+M.+Weerawarnaauthor=R.+A.+Z.+Uyauthor=V.+C.+Damalankaauthor=S.+R.+Mandadapuauthor=K.+R.+Allistonauthor=N.+Mehzabeenauthor=K.+P.+Battaileauthor=S.+Lovellauthor=K.-O.+Changauthor=W.+C.+Groutas&title=Structure-Guided+Design+and+Optimization+of+Dipeptidyl+Inhibitors+of+Norovirus+3CL+Protease.+Structure%E2%80%93Activity+Relationships+and+Biochemical%2C+X-ray+Crystallographic%2C+Cell-Based%2C+and+In+Vivo+Studies&doi=10.1021%2Fjm5019934"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Dipeptidyl Inhibitors of Norovirus 3CL Protease. Structure-Activity Relationships and Biochemical, X-ray Crystallographic, Cell-Based, and In Vivo Studies</span></div><div class="casAuthors">Galasiti Kankanamalage, Anushka C.; Kim, Yunjeong; Weerawarna, Pathum M.; Uy, Roxanne Adeline Z.; Damalanka, Vishnu C.; Mandadapu, Sivakoteswara Rao; Alliston, Kevin R.; Mehzabeen, Nurjahan; Battaile, Kevin P.; Lovell, Scott; Chang, Kyeong-Ok; Groutas, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3144-3155</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Norovirus infection constitutes the primary cause of acute viral gastroenteritis.  There are currently no vaccines or norovirus-specific antiviral therapeutics available for the management of norovirus infection.  Norovirus 3C-like protease is essential for viral replication, consequently, inhibition of this enzyme is a fruitful avenue of investigation that may lead to the emergence of antinorovirus therapeutics.  The authors describe herein the optimization of dipeptidyl inhibitors of norovirus 3C-like protease using iterative SAR, x-ray crystallog., and enzyme and cell-based studies.  The authors also demonstrate herein in vivo efficacy of an inhibitor using the murine model of norovirus infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_rKoPrEayGbVg90H21EOLACvtfcHk0lgpA9koI1RYrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktlertr4%253D&md5=fc3d5b27ee9241e5d76b7f040d3ca040</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm5019934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5019934%26sid%3Dliteratum%253Aachs%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DWeerawarna%26aufirst%3DP.%2BM.%26aulast%3DUy%26aufirst%3DR.%2BA.%2BZ.%26aulast%3DDamalanka%26aufirst%3DV.%2BC.%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DMehzabeen%26aufirst%3DN.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DK.-O.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26atitle%3DStructure-Guided%2520Design%2520and%2520Optimization%2520of%2520Dipeptidyl%2520Inhibitors%2520of%2520Norovirus%25203CL%2520Protease.%2520Structure%25E2%2580%2593Activity%2520Relationships%2520and%2520Biochemical%252C%2520X-ray%2520Crystallographic%252C%2520Cell-Based%252C%2520and%2520In%2520Vivo%2520Studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3144%26epage%3D3155%26doi%3D10.1021%2Fjm5019934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span>; <span class="NLM_string-name">Kania, R. S.</span>; <span class="NLM_string-name">Nieman, J. A.</span>; <span class="NLM_string-name">Planken, S. P.</span>; <span class="NLM_string-name">Smith, G. J.</span></span> <span> </span><span class="NLM_article-title">Anticoronaviral Compounds and Compositions, Their Pharmaceutical Uses and Materials For Their Synthesis</span>. <span class="NLM_patent">WO2005/113580</span>, May 9, <span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=R.+L.+Hoffman&author=R.+S.+Kania&author=J.+A.+Nieman&author=S.+P.+Planken&author=G.+J.+Smith&title=Anticoronaviral+Compounds+and+Compositions%2C+Their+Pharmaceutical+Uses+and+Materials+For+Their+Synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26atitle%3DAnticoronaviral%2520Compounds%2520and%2520Compositions%252C%2520Their%2520Pharmaceutical%2520Uses%2520and%2520Materials%2520For%2520Their%2520Synthesis%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freedman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tchesnokov, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, J. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinazi, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghton, M.</span></span> <span> </span><span class="NLM_article-title">Application of Molecular Dynamics Simulations to the Design of Nucleotide Inhibitors Binding to Norovirus Polymerase</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6566</span>– <span class="NLM_lpage">6578</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00742</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00742" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVCnsrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=6566-6578&author=H.+Freedmanauthor=J.+Kunduauthor=P.+E.+Tchesnokovauthor=J.+L.+M.+Lawauthor=J.+A.+Niemanauthor=R.+F.+Schinaziauthor=D.+L.+Tyrrellauthor=M.+Gotteauthor=M.+Houghton&title=Application+of+Molecular+Dynamics+Simulations+to+the+Design+of+Nucleotide+Inhibitors+Binding+to+Norovirus+Polymerase&doi=10.1021%2Facs.jcim.0c00742"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Molecular Dynamics Simulations to the Design of Nucleotide Inhibitors Binding to Norovirus Polymerase</span></div><div class="casAuthors">Freedman, Holly; Kundu, Juthika; Tchesnokov, Egor Petrovitch; Law, John Lok Man; Nieman, James A.; Schinazi, Raymond F.; Tyrrell, D. Lorne; Gotte, Matthias; Houghton, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6566-6578</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The RNA-dependent RNA polymerase (RdRp) of norovirus is an attractive target of antiviral agents aimed at providing protection against norovirus-assocd. gastroenteritis.  Here, we perform mol. dynamics simulations of the crystal structure of norovirus RdRp in complex with several known binders, as well as free-energy simulations by free-energy perturbation (FEP) to det. binding free energies of these mols. relative to the natural nucleotide substrates.  We det. exptl. EC50 values and nucleotide incorporation efficiencies for several of these compds.  Moreover, we investigate the mechanism of inhibition of some of these ligands.  Using FEP, we screened a virtual nucleotide library with 121 elements for binding to the polymerase and successfully identified two novel chain terminators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTlxYz2z6Pm7Vg90H21EOLACvtfcHk0lgpA9koI1RYrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVCnsrzN&md5=18be63411c1858ef82c4f9a0542ac390</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00742%26sid%3Dliteratum%253Aachs%26aulast%3DFreedman%26aufirst%3DH.%26aulast%3DKundu%26aufirst%3DJ.%26aulast%3DTchesnokov%26aufirst%3DP.%2BE.%26aulast%3DLaw%26aufirst%3DJ.%2BL.%2BM.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DSchinazi%26aufirst%3DR.%2BF.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DGotte%26aufirst%3DM.%26aulast%3DHoughton%26aufirst%3DM.%26atitle%3DApplication%2520of%2520Molecular%2520Dynamics%2520Simulations%2520to%2520the%2520Design%2520of%2520Nucleotide%2520Inhibitors%2520Binding%2520to%2520Norovirus%2520Polymerase%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D6566%26epage%3D6578%26doi%3D10.1021%2Facs.jcim.0c00742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathnayake, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dampalla, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jesri, A.-R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashipathy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lushington, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Optimization of Dipeptidyl Inhibitors of Norovirus 3CL Protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">11945</span>– <span class="NLM_lpage">11963</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01252</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01252" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVOns7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=11945-11963&issue=20&author=A.+D.+Rathnayakeauthor=Y.+Kimauthor=C.+S.+Dampallaauthor=H.+N.+Nguyenauthor=A.-R.+M.+Jesriauthor=M.+M.+Kashipathyauthor=G.+H.+Lushingtonauthor=K.+P.+Battaileauthor=S.+Lovellauthor=K.-O.+Changauthor=W.+C.+Groutas&title=Structure-Guided+Optimization+of+Dipeptidyl+Inhibitors+of+Norovirus+3CL+Protease&doi=10.1021%2Facs.jmedchem.0c01252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Optimization of Dipeptidyl Inhibitors of Norovirus 3CL Protease</span></div><div class="casAuthors">Rathnayake, Athri D.; Kim, Yunjeong; Dampalla, Chamandi S.; Nguyen, Harry Nhat; Jesri, Abdul-Rahman M.; Kashipathy, Maithri M.; Lushington, Gerald H.; Battaile, Kevin P.; Lovell, Scott; Chang, Kyeong-Ok; Groutas, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">11945-11963</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acute gastroenteritis caused by noroviruses has a major impact on public health worldwide in terms of morbidity, mortality, and economic burden.  The disease impacts most severely immunocompromised patients, the elderly, and children.  The current lack of approved vaccines and small-mol. therapeutics for the treatment and prophylaxis of norovirus infections underscores the need for the development of norovirus-specific drugs.  The studies described herein entail the use of the gem-di-Me moiety as a means of improving the pharmacol. activity and physicochem. properties of a dipeptidyl series of transition state inhibitors of norovirus 3CL protease, an enzyme essential for viral replication.  Several compds. were found to be potent inhibitors of the enzyme in biochem. and cell-based assays.  The pharmacol. activity and cellular permeability of the inhibitors were found to be sensitive to the location of the gem-di-Me group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogL9olpw-8zLVg90H21EOLACvtfcHk0lgpA9koI1RYrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVOns7jP&md5=0eb3b684894ddce9eea7491ae5d8e080</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01252%26sid%3Dliteratum%253Aachs%26aulast%3DRathnayake%26aufirst%3DA.%2BD.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DDampalla%26aufirst%3DC.%2BS.%26aulast%3DNguyen%26aufirst%3DH.%2BN.%26aulast%3DJesri%26aufirst%3DA.-R.%2BM.%26aulast%3DKashipathy%26aufirst%3DM.%2BM.%26aulast%3DLushington%26aufirst%3DG.%2BH.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DK.-O.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26atitle%3DStructure-Guided%2520Optimization%2520of%2520Dipeptidyl%2520Inhibitors%2520of%2520Norovirus%25203CL%2520Protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D20%26spage%3D11945%26epage%3D11963%26doi%3D10.1021%2Facs.jmedchem.0c01252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">World Health
Organization</span>. <a href="http://http:/www.who.int/emergencies/diseases/novel-coronavirus-2019" class="extLink">http:/www.who.int/emergencies/diseases/novel-coronavirus-2019</a> (accessed 2021-02-06).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=World+Health%0AOrganization.+http%3A%2Fwww.who.int%2Femergencies%2Fdiseases%2Fnovel-coronavirus-2019+%28accessed+2021-02-06%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamarthi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupte, A.</span></span> <span> </span><span class="NLM_article-title">The journey of remdesivir: from Ebola to COVID-19</span>. <i>Drugs in Context</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.7573/dic.2020-4-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.7573%2Fdic.2020-4-14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1-9&author=J.+Pardoauthor=A.+M.+Shuklaauthor=G.+Chamarthiauthor=A.+Gupte&title=The+journey+of+remdesivir%3A+from+Ebola+to+COVID-19&doi=10.7573%2Fdic.2020-4-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.7573%2Fdic.2020-4-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7573%252Fdic.2020-4-14%26sid%3Dliteratum%253Aachs%26aulast%3DPardo%26aufirst%3DJ.%26aulast%3DShukla%26aufirst%3DA.%2BM.%26aulast%3DChamarthi%26aufirst%3DG.%26aulast%3DGupte%26aufirst%3DA.%26atitle%3DThe%2520journey%2520of%2520remdesivir%253A%2520from%2520Ebola%2520to%2520COVID-19%26jtitle%3DDrugs%2520in%2520Context%26date%3D2020%26volume%3D9%26spage%3D1%26epage%3D9%26doi%3D10.7573%2Fdic.2020-4-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">U.S. Food and Drug Administration</span>. <a href="https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19" class="extLink">https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19</a> (accessed 2021-02-6).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Food+and+Drug+Administration.+https%3A%2F%2Fwww.fda.gov%2Fnews-events%2Fpress-announcements%2Ffda-approves-first-treatment-covid-19+%28accessed+2021-02-6%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheahan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pruijssers, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agostini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leist, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinnon, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappell, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluemling, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natchus, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saindane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolykhalov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harcourt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornburg, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanstrom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denison, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baric, R. S.</span></span> <span> </span><span class="NLM_article-title">An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 in human airway epithelial cell cultures and multiple coronaviruses in mice</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">eabb5883</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.abb5883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1126%2Fscitranslmed.abb5883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32253226" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=eabb5883&author=T.+P.+Sheahanauthor=A.+C.+Simsauthor=S.+Zhouauthor=R.+L.+Grahamauthor=A.+J.+Pruijssersauthor=M.+L.+Agostiniauthor=S.+R.+Leistauthor=A.+Schaferauthor=K.+H.+Dinnonauthor=L.+J.+Stevensauthor=J.+D.+Chappellauthor=X.+Luauthor=T.+M.+Hughesauthor=A.+S.+Georgeauthor=C.+S.+Hillauthor=S.+A.+Montgomeryauthor=A.+J.+Brownauthor=G.+R.+Bluemlingauthor=M.+G.+Natchusauthor=M.+Saindaneauthor=A.+A.+Kolykhalovauthor=G.+Painterauthor=J.+Harcourtauthor=A.+Taminauthor=N.+J.+Thornburgauthor=R.+Swanstromauthor=M.+R.+Denisonauthor=R.+S.+Baric&title=An+orally+bioavailable+broad-spectrum+antiviral+inhibits+SARS-CoV-2+in+human+airway+epithelial+cell+cultures+and+multiple+coronaviruses+in+mice&doi=10.1126%2Fscitranslmed.abb5883"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.abb5883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.abb5883%26sid%3Dliteratum%253Aachs%26aulast%3DSheahan%26aufirst%3DT.%2BP.%26aulast%3DSims%26aufirst%3DA.%2BC.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DGraham%26aufirst%3DR.%2BL.%26aulast%3DPruijssers%26aufirst%3DA.%2BJ.%26aulast%3DAgostini%26aufirst%3DM.%2BL.%26aulast%3DLeist%26aufirst%3DS.%2BR.%26aulast%3DSchafer%26aufirst%3DA.%26aulast%3DDinnon%26aufirst%3DK.%2BH.%26aulast%3DStevens%26aufirst%3DL.%2BJ.%26aulast%3DChappell%26aufirst%3DJ.%2BD.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DHughes%26aufirst%3DT.%2BM.%26aulast%3DGeorge%26aufirst%3DA.%2BS.%26aulast%3DHill%26aufirst%3DC.%2BS.%26aulast%3DMontgomery%26aufirst%3DS.%2BA.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DBluemling%26aufirst%3DG.%2BR.%26aulast%3DNatchus%26aufirst%3DM.%2BG.%26aulast%3DSaindane%26aufirst%3DM.%26aulast%3DKolykhalov%26aufirst%3DA.%2BA.%26aulast%3DPainter%26aufirst%3DG.%26aulast%3DHarcourt%26aufirst%3DJ.%26aulast%3DTamin%26aufirst%3DA.%26aulast%3DThornburg%26aufirst%3DN.%2BJ.%26aulast%3DSwanstrom%26aufirst%3DR.%26aulast%3DDenison%26aufirst%3DM.%2BR.%26aulast%3DBaric%26aufirst%3DR.%2BS.%26atitle%3DAn%2520orally%2520bioavailable%2520broad-spectrum%2520antiviral%2520inhibits%2520SARS-CoV-2%2520in%2520human%2520airway%2520epithelial%2520cell%2520cultures%2520and%2520multiple%2520coronaviruses%2520in%2520mice%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2020%26volume%3D12%26spage%3Deabb5883%26doi%3D10.1126%2Fscitranslmed.abb5883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B.</span></span> <span> </span><span class="NLM_article-title">Targeting proteases: success, failures and future prospects</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">799</span>, <span class="refDoi"> DOI: 10.1038/nrd2092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fnrd2092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=16955069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=785-799&author=B.+Turk&title=Targeting+proteases%3A+success%2C+failures+and+future+prospects&doi=10.1038%2Fnrd2092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18aR"><div class="casContent"><span class="casTitleNuber">18a</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting proteases: successes, failures and future prospects</span></div><div class="casAuthors">Turk, Boris</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">785-799</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Until fairly recently, proteases were considered primarily to be protein-degrading enzymes.  However, this view has dramatically changed and proteases are now seen as extremely important signalling mols. that are involved in numerous vital processes.  Protease signalling pathways are strictly regulated, and the dysregulation of protease activity can lead to pathologies such as cardiovascular and inflammatory diseases, cancer, osteoporosis and neurol. disorders.  Several small-mol. drugs targeting proteases are already on the market and many more are in development.  The status of human protease research and prospects for future protease-targeted drugs are reviewed here, with ref. to some key examples where protease drugs have succeeded or failed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa_X9u2oqlzrVg90H21EOLACvtfcHk0lgsiQZYqFJmEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltrs%253D&md5=00111ddbeafb283cbe88f7b0ee7bb404</span></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=10.1038%2Fnrd2092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2092%26sid%3Dliteratum%253Aachs%26aulast%3DTurk%26aufirst%3DB.%26atitle%3DTargeting%2520proteases%253A%2520success%252C%2520failures%2520and%2520future%2520prospects%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D785%26epage%3D799%26doi%3D10.1038%2Fnrd2092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit18b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span> <span> </span><span class="NLM_article-title">New Directions for Protease Inhibitors Directed Drug Discovery</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">579</span>, <span class="refDoi"> DOI: 10.1002/bip.22780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fbip.22780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=26584340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SkurjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2016&pages=563-579&author=Y.+Hamadaauthor=Y.+Kiso&title=New+Directions+for+Protease+Inhibitors+Directed+Drug+Discovery&doi=10.1002%2Fbip.22780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">New directions for protease inhibitors directed drug discovery</span></div><div class="casAuthors">Hamada, Yoshio; Kiso, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">563-579</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Proteases play crucial roles in various biol. processes, and their activities are essential for all living organisms-from viruses to humans.  Since their functions are closely assocd. with many pathogenic mechanisms, their inhibitors or activators are important mol. targets for developing treatments for various diseases.  Here, we describe drugs/drug candidates that target proteases, such as malarial plasmepsins, β-secretase, virus proteases, and dipeptidyl peptidase-4.  Previously, we reported inhibitors of aspartic proteases, such as renin, human immunodeficiency virus type 1 protease, human T-lymphotropic virus type I protease, plasmepsins, and β-secretase, as drug candidates for hypertension, adult T-cell leukemia, human T-lymphotropic virus type I-assocd. myelopathy, malaria, and Alzheimer's disease.  Our inhibitors are also described in this review article as examples of drugs that target proteases. © 2015 Wiley Periodicals, Inc.  Biopolymers (Pept Sci) 106: 563-579, 2016.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXnphbid6u97Vg90H21EOLACvtfcHk0lgsiQZYqFJmEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SkurjK&md5=352c1f1c60798f87da18044e7dfe2e2a</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1002%2Fbip.22780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22780%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DY.%26aulast%3DKiso%26aufirst%3DY.%26atitle%3DNew%2520Directions%2520for%2520Protease%2520Inhibitors%2520Directed%2520Drug%2520Discovery%26jtitle%3DBiopolymers%26date%3D2016%26volume%3D106%26spage%3D563%26epage%3D579%26doi%3D10.1002%2Fbip.22780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McClung, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donoghue, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papapoulos, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bone, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langdahl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saag, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiel, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallari, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marc, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonaca, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiviott, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Villiers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippuner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginster, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Portales, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanchetta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbini, C. A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cange, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grip, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyden, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DaSilva, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massaad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verbruggen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polis, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoch, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marc, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatine, M. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Odanacatib for the treatment of postmenopausal osteoporosis: results of the LOFT multicentre, randomised, double-blind, placebo-controlled trial and LOFT Extension study</span>. <i>Lancet Diabetes Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.1016/S2213-8587(19)30346-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2FS2213-8587%2819%2930346-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=31676222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVOnu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=899-911&issue=12&author=M.+R.+McClungauthor=M.+L.+O%E2%80%99Donoghueauthor=S.+E.+Papapoulosauthor=H.+Boneauthor=B.+Langdahlauthor=K.+G.+Saagauthor=I.+R.+Reidauthor=D.+P.+Kielauthor=I.+Cavallariauthor=P.+Marcauthor=M.+P.+Bonacaauthor=S.+D.+Wiviottauthor=T.+de+Villiersauthor=X.+Lingauthor=K.+Lippunerauthor=T.+Nakamuraauthor=J.-Y.+Reginsterauthor=J.+A.+Rodriguez-Portalesauthor=C.+Rouxauthor=J.+Zanchettaauthor=C.+A.+F.+Zerbiniauthor=J.-G.+Parkauthor=K.+Imauthor=A.+Cangeauthor=L.+T.+Gripauthor=N.+Heydenauthor=C.+DaSilvaauthor=D.+Cohnauthor=R.+Massaadauthor=B.+B.+Scottauthor=N.+Verbruggenauthor=D.+Gurnerauthor=D.+L.+Millerauthor=M.+L.+Blairauthor=A.+B.+Polisauthor=S.+A.+Stochauthor=A.+Santoraauthor=A.+Lombardiauthor=A.+T.+Leungauthor=K.+D.+Kaufmanauthor=S.+Marcauthor=M.+S.+Sabatine&title=Odanacatib+for+the+treatment+of+postmenopausal+osteoporosis%3A+results+of+the+LOFT+multicentre%2C+randomised%2C+double-blind%2C+placebo-controlled+trial+and+LOFT+Extension+study&doi=10.1016%2FS2213-8587%2819%2930346-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Odanacatib for the treatment of postmenopausal osteoporosis: results of the LOFT multicentre, randomised, double-blind, placebo-controlled trial and LOFT Extension study</span></div><div class="casAuthors">McClung, Michael R.; O'Donoghue, Michelle L.; Papapoulos, Socrates E.; Bone, Henry; Langdahl, Bente; Saag, Kenneth G.; Reid, Ian R.; Kiel, Douglas P.; Cavallari, Ilaria; Bonaca, Marc P.; Wiviott, Stephen D.; de Villiers, Tobias; Ling, Xu; Lippuner, Kurt; Nakamura, Toshitaka; Reginster, Jean-Yves; Rodriguez-Portales, Jose Adolfo; Roux, Christian; Zanchetta, Jose; Zerbini, Cristiano A. F.; Park, Jeong-Gun; Im, KyungAh; Cange, Abby; Grip, Laura T.; Heyden, Norman; DaSilva, Carolyn; Cohn, Dosinda; Massaad, Rachid; Scott, Boyd B.; Verbruggen, Nadia; Gurner, Deborah; Miller, Deborah L.; Blair, Micki L.; Polis, Adam B.; Stoch, S. Aubrey; Santora, Arthur; Lombardi, Antonio; Leung, Albert T.; Kaufman, Keith D.; Sabatine, Marc S.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Diabetes & Endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">899-911</span>CODEN:
                <span class="NLM_cas:coden">LDEAAS</span>;
        ISSN:<span class="NLM_cas:issn">2213-8587</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Odanacatib, a cathepsin K inhibitor, reduces bone resorption while maintaining bone formation.  Previous work has shown that odanacatib increases bone mineral d. in postmenopausal women with low bone mass.  We aimed to investigate the efficacy and safety of odanacatib to reduce fracture risk in postmenopausal women with osteoporosis.  The Long-term Odanacatib Fracture Trial (LOFT) was a multicentre, randomised, double-blind, placebo-controlled, event-driven study at 388 outpatient clinics in 40 countries.  Eligible participants were women aged at least 65 years who were postmenopausal for 5 years or more, with a femoral neck or total hip bone mineral d. T-score between -2.5 and -4.0 if no previous radiog. vertebral fracture, or between -1.5 and -4.0 with a previous vertebral fracture.  Women with a previous hip fracture, more than one vertebral fracture, or a T-score of less than -4.0 at the total hip or femoral neck were not eligible unless they were unable or unwilling to use approved osteoporosis treatment.  Participants were randomly assigned (1:1) to either oral odanacatib (50 mg once per wk) or matching placebo.  Randomisation was done using an interactive voice recognition system after stratification for previous radiog. vertebral fracture, and treatment was masked to study participants, investigators and their staff, and sponsor personnel.  If the study completed before 5 years of double-blind treatment, consenting participants could enrol in a double-blind extension study (LOFT Extension), continuing their original treatment assignment for up to 5 years from randomisation.  Primary endpoints were incidence of vertebral fractures as assessed using radiographs collected at baseline, 6 and 12 mo, yearly, and at final study visit in participants for whom evaluable radiograph images were available at baseline and at least one other timepoint, and hip and non-vertebral fractures adjudicated as being a result of osteoporosis as assessed by clin. history and radiograph.  Safety was assessed in participants who received at least one dose of study drug.  The adjudicated cardiovascular safety endpoints were a composite of cardiovascular death, myocardial infarction, or stroke, and new-onset atrial fibrillation or flutter.  Individual cardiovascular endpoints and death were also assessed.  LOFT and LOFT Extension are registered with ClinicalTrials.gov (no. NCT00529373) and the European Clin. Trials Database (EudraCT no. 2007-002693-66).  Between Sept 14, 2007, and Nov 17, 2009, we randomly assigned 16 071 evaluable patients to treatment: 8043 to odanacatib and 8028 to placebo.  After a median follow-up of 36.5 mo (IQR 34.43-40.15) 4297 women assigned to odanacatib and 3960 assigned to placebo enrolled in LOFT Extension (total median follow-up 47.6 mo, IQR 35.45-60.06).  In LOFT, cumulative incidence of primary outcomes for odanacatib vs. placebo were: radiog. vertebral fractures 3.7% (251/6770) vs. 7.8% (542/6910), hazard ratio (HR) 0.46, 95% CI 0.40-0.53; hip fractures 0.8% (65/8043) vs. 1.6% (125/8028), 0.53, 0.39-0.71; non-vertebral fractures 5.1% (412/8043) vs. 6.7% (541/8028), 0.77, 0.68-0.87; all p <0.0001.  Combined results from LOFT plus LOFT Extension for cumulative incidence of primary outcomes for odanacatib vs. placebo were: radiog. vertebral fractures 4.9% (341/6909) vs. 9.6% (675/7011), HR 0.48, 95% CI 0.42-0.55; hip fractures 1.1% (86/8043) vs. 2.0% (162/8028), 0.52, 0.40-0.67; non-vertebral fractures 6.4% (512/8043) vs. 8.4% (675/8028), 0.74, 0.66-0.83; all p <0.0001.  In LOFT, the composite cardiovascular endpoint of cardiovascular death, myocardial infarction, or stroke occurred in 273 (3.4%) of 8043 patients in the odanacatib group vs. 245 (3.1%) of 8028 in the placebo group (HR 1.12, 95% CI 0.95-1.34; p = 0.18).  New-onset atrial fibrillation or flutter occurred in 112 (1.4%) of 8043 patients in the odanacatib group vs. 96 (1.2%) of 8028 in the placebo group (HR 1.18, 0.90-1.55; p = 0.24).  Odanacatib was assocd. with an increased risk of stroke (1.7% [136/8043] vs 1.3% [104/8028], HR 1.32, 1.02-1.70; p = 0.034), but not myocardial infarction (0.7% [60/8043] vs 0.9% [74/8028], HR 0.82, 0.58-1.15; p = 0.26).  The HR for all-cause mortality was 1.13 (5.0% [401/8043] vs 4.4% [356/8028], 0.98-1.30; p = 0.10).  When data from LOFT Extension were included, the composite of cardiovascular death, myocardial infarction, or stroke occurred in significantly more patients in the odanacatib group than in the placebo group (401 [5.0%] of 8043 vs 343 [4.3%] of 8028, HR 1.17, 1.02-1.36; p = 0.029, as did stroke 2.3% [187/8043] vs 1.7% [137/8028], HR 1.37, 1.10-1.71; p = 0.0051).  Odanacatib reduced the risk of fracture, but was assocd. with an increased risk of cardiovascular events, specifically stroke, in postmenopausal women with osteoporosis.  Based on the overall balance between benefit and risk, the study's sponsor decided that they would no longer pursue development of odanacatib for treatment of osteoporosis.  Merck Sharp & Dohme Corp, a subsidiary of Merck & Co, Inc, Kenilworth, NJ, USA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcN107ZPXxyrVg90H21EOLACvtfcHk0liERugTMrWdHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVOnu73O&md5=faee30ee846eeb2b368c1191dc9e9acd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS2213-8587%2819%2930346-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2213-8587%252819%252930346-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcClung%26aufirst%3DM.%2BR.%26aulast%3DO%25E2%2580%2599Donoghue%26aufirst%3DM.%2BL.%26aulast%3DPapapoulos%26aufirst%3DS.%2BE.%26aulast%3DBone%26aufirst%3DH.%26aulast%3DLangdahl%26aufirst%3DB.%26aulast%3DSaag%26aufirst%3DK.%2BG.%26aulast%3DReid%26aufirst%3DI.%2BR.%26aulast%3DKiel%26aufirst%3DD.%2BP.%26aulast%3DCavallari%26aufirst%3DI.%26aulast%3DMarc%26aufirst%3DP.%26aulast%3DBonaca%26aufirst%3DM.%2BP.%26aulast%3DWiviott%26aufirst%3DS.%2BD.%26aulast%3Dde%2BVilliers%26aufirst%3DT.%26aulast%3DLing%26aufirst%3DX.%26aulast%3DLippuner%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DReginster%26aufirst%3DJ.-Y.%26aulast%3DRodriguez-Portales%26aufirst%3DJ.%2BA.%26aulast%3DRoux%26aufirst%3DC.%26aulast%3DZanchetta%26aufirst%3DJ.%26aulast%3DZerbini%26aufirst%3DC.%2BA.%2BF.%26aulast%3DPark%26aufirst%3DJ.-G.%26aulast%3DIm%26aufirst%3DK.%26aulast%3DCange%26aufirst%3DA.%26aulast%3DGrip%26aufirst%3DL.%2BT.%26aulast%3DHeyden%26aufirst%3DN.%26aulast%3DDaSilva%26aufirst%3DC.%26aulast%3DCohn%26aufirst%3DD.%26aulast%3DMassaad%26aufirst%3DR.%26aulast%3DScott%26aufirst%3DB.%2BB.%26aulast%3DVerbruggen%26aufirst%3DN.%26aulast%3DGurner%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DD.%2BL.%26aulast%3DBlair%26aufirst%3DM.%2BL.%26aulast%3DPolis%26aufirst%3DA.%2BB.%26aulast%3DStoch%26aufirst%3DS.%2BA.%26aulast%3DSantora%26aufirst%3DA.%26aulast%3DLombardi%26aufirst%3DA.%26aulast%3DLeung%26aufirst%3DA.%2BT.%26aulast%3DKaufman%26aufirst%3DK.%2BD.%26aulast%3DMarc%26aufirst%3DS.%26aulast%3DSabatine%26aufirst%3DM.%2BS.%26atitle%3DOdanacatib%2520for%2520the%2520treatment%2520of%2520postmenopausal%2520osteoporosis%253A%2520results%2520of%2520the%2520LOFT%2520multicentre%252C%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%2520trial%2520and%2520LOFT%2520Extension%2520study%26jtitle%3DLancet%2520Diabetes%2520Endocrinol.%26date%3D2019%26volume%3D7%26issue%3D12%26spage%3D899%26epage%3D911%26doi%3D10.1016%2FS2213-8587%2819%2930346-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petter, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0liERugTMrWdHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span> <span> </span><span class="NLM_article-title">Covalent Inhibition in Drug Discovery</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fcmdc.201900107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=30816012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFensb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=889-906&issue=9&author=A.+K.+Ghoshauthor=I.+Samantaauthor=A.+Mondalauthor=W.+R.+Liu&title=Covalent+Inhibition+in+Drug+Discovery&doi=10.1002%2Fcmdc.201900107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibition in Drug Discovery</span></div><div class="casAuthors">Ghosh, Avick Kumar; Samanta, Indranil; Mondal, Anushree; Liu, Wenshe Ray</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">889-906</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Although covalent inhibitors have been used as therapeutics for more than a century, there has been general resistance in the pharmaceutical industry against their further development due to safety concerns.  This inclination has recently been reverted after the development of a wide variety of covalent inhibitors to address human health conditions along with the US Food and Drug Administration (FDA) approval of several covalent therapeutics for use in humans.  Along with this exciting resurrection of an old drug discovery concept, this review surveys enzymes that can be targeted by covalent inhibitors for the treatment of human diseases.  We focus on protein kinases, RAS proteins, and a few other enzymes that have been studied extensively as targets for covalent inhibition, with the aim to address challenges in designing effective covalent drugs and to provide suggestions in the area that have yet to be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1p5IlI637AbVg90H21EOLACvtfcHk0ljZIrLWmC4F_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFensb4%253D&md5=aaa9bd889193c02f8d9acdecaa45e64b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900107%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DSamanta%26aufirst%3DI.%26aulast%3DMondal%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26atitle%3DCovalent%2520Inhibition%2520in%2520Drug%2520Discovery%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26issue%3D9%26spage%3D889%26epage%3D906%26doi%3D10.1002%2Fcmdc.201900107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halford, B.</span></span> <span> </span><span class="NLM_article-title">Covalent drugs go from fringe field to fashionable endeavor</span>. <i>C&EN</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">43</span>),  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.47287/cen-09843-cover</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.47287%2Fcen-09843-cover" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2020&pages=28-33&issue=43&author=B.+Halford&title=Covalent+drugs+go+from+fringe+field+to+fashionable+endeavor&doi=10.47287%2Fcen-09843-cover"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.47287%2Fcen-09843-cover&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.47287%252Fcen-09843-cover%26sid%3Dliteratum%253Aachs%26aulast%3DHalford%26aufirst%3DB.%26atitle%3DCovalent%2520drugs%2520go%2520from%2520fringe%2520field%2520to%2520fashionable%2520endeavor%26jtitle%3DC%2526EN%26date%3D2020%26volume%3D98%26issue%3D43%26spage%3D28%26epage%3D33%26doi%3D10.47287%2Fcen-09843-cover" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallwitz, B.</span></span> <span> </span><span class="NLM_article-title">Clinical Use of DPP-4 Inhibitors</span>. <i>Front. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">389</span>, <span class="refDoi"> DOI: 10.3389/fendo.2019.00389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.3389%2Ffendo.2019.00389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=31275246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A280%3ADC%252BB3MzktlensA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=389&author=B.+Gallwitz&title=Clinical+Use+of+DPP-4+Inhibitors&doi=10.3389%2Ffendo.2019.00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Use of DPP-4 Inhibitors</span></div><div class="casAuthors">Gallwitz Baptist</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">389</span>
        ISSN:<span class="NLM_cas:issn">1664-2392</span>.
    </div><div class="casAbstract">DPP-4 inhibitors were introduced for the treatment of type 2 diabetes in 2006.  They stimulate insulin secretion and inhibit glucagon secretion by elevating endogenous GLP-1 concentrations without an intrinsic hypoglycaemia risk.  Their efficacy potential to lower HbA1c is in the range between 0.5 and 1.0% and their safety profile is favorable.  DPP-4 inhibitors are body weight neutral and they have demonstrated cardiovascular safety.  Most compounds can be used in impaired renal function.  Guidelines suggest the additional use of DPP-4 inhibitors after metformin failure in patients that do not require antidiabetic therapy with proven cardiovascular benefit.  Recently, DPP-4 inhibitors have increasingly replaced sulfonylureas as second line therapy after metformin failure and many metformin/DPP-4 inhibitor fixed dose combinations are available.  In later stages of type 2 diabetes, DPP-4 inhibitors are also recommended in the guidelines in triple therapies with metformin and SGLT-2 inhibitors or with metformin and insulin.  A treatment with DPP-4 inhibitors should be stopped when GLP-1 receptor agonists are used.  DPP-4 inhibitors can be used as monotherapy when metformin is contraindicated or not tolerated.  Some studies have shown value of initial metformin-DPP-4 inhibitor combination therapy in special populations.  This article gives an overview on the clinical use of DPP-4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7l-0AWRpXMoqdXsLcGTEUfW6udTcc2ebg_GYcQCOniLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzktlensA%253D%253D&md5=76b007c33dc4180e5f22bbda6b98c25d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3389%2Ffendo.2019.00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffendo.2019.00389%26sid%3Dliteratum%253Aachs%26aulast%3DGallwitz%26aufirst%3DB.%26atitle%3DClinical%2520Use%2520of%2520DPP-4%2520Inhibitors%26jtitle%3DFront.%2520Endocrinol.%26date%3D2019%26volume%3D10%26spage%3D389%26doi%3D10.3389%2Ffendo.2019.00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forman, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinna, A.</span></span> <span> </span><span class="NLM_article-title">Glutathione: Overview of its protective roles, measurement, and biosynthesis</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">1–2</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2008.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.mam.2008.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=18796312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFelurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1-12&issue=1%E2%80%932&author=H.+J.+Formanauthor=H.+Zhangauthor=A.+Rinna&title=Glutathione%3A+Overview+of+its+protective+roles%2C+measurement%2C+and+biosynthesis&doi=10.1016%2Fj.mam.2008.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione: Overview of its protective roles, measurement, and biosynthesis</span></div><div class="casAuthors">Forman, Henry Jay; Zhang, Hongqiao; Rinna, Alessandra</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review is the introduction to a special issue concerning, glutathione (GSH), the most abundant low mol. wt. thiol compd. synthesized in cells.  GSH plays crit. roles in protecting cells from oxidative damage and the toxicity of xenobiotic electrophiles, and maintaining redox homeostasis.  Here, the functions and GSH and the sources of oxidants and electrophiles, the elimination of oxidants by redn. and electrophiles by conjugation with GSH are briefly described.  Methods of assessing GSH status in the cells are also described.  GSH synthesis and its regulation are addressed along with therapeutic approaches for manipulating GSH content that have been proposed.  The purpose here is to provide a brief overview of some of the important aspects of glutathione metab. as part of this special issue that will provide a more comprehensive review of the state of knowledge regarding this essential mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlWapNqV0GqLVg90H21EOLACvtfcHk0ljZIrLWmC4F_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFelurY%253D&md5=c1b1aa2b03389bf1434af0762df78fe1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2008.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2008.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DForman%26aufirst%3DH.%2BJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DRinna%26aufirst%3DA.%26atitle%3DGlutathione%253A%2520Overview%2520of%2520its%2520protective%2520roles%252C%2520measurement%252C%2520and%2520biosynthesis%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2009%26volume%3D30%26issue%3D1%25E2%2580%25932%26spage%3D1%26epage%3D12%26doi%3D10.1016%2Fj.mam.2008.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="note"><p class="first last">Reviews of repurposed and targeted peptidomimetics:</p></div><div class="NLM_citation" id="rightTab-cit25a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Citarella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scala, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piperno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micale, N.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 Mpro: A Potential Target for Peptidomimetics and Small-Molecule Inhibitors</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">607</span>, <span class="refDoi"> DOI: 10.3390/biom11040607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.3390%2Fbiom11040607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33921886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtlSnu7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=607&issue=11&author=A.+Citarellaauthor=A.+Scalaauthor=A.+Pipernoauthor=N.+Micale&title=SARS-CoV-2+Mpro%3A+A+Potential+Target+for+Peptidomimetics+and+Small-Molecule+Inhibitors&doi=10.3390%2Fbiom11040607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25aR"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro: a potential target for peptidomimetics and small-molecule inhibitors</span></div><div class="casAuthors">Citarella, Andrea; Scala, Angela; Piperno, Anna; Micale, Nicola</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">607</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The uncontrolled spread of the COVID-19 pandemic caused by the new coronavirus SARSCoV-2 during 2020-2021 is one of the most devastating events in the history, with remarkable impacts on the health, economic systems, and habits of the entire world population.  While some effective vaccines are nowadays approved and extensively administered, the long-term efficacy and safety of this line of intervention is constantly under debate as coronaviruses rapidly mutate and several SARSCoV-2 variants have been already identified worldwide.  Then, the WHO's main recommendations to prevent severe clin. complications by COVID-19 are still essentially based on social distancing and limitation of human interactions, therefore the identification of new target-based drugs became a priority.  Several strategies have been proposed to counteract such viral infection, including the repurposing of FDA already approved for the treatment of HIV, HCV, and EBOLA, inter alia.  Among the evaluated compds., inhibitors of the main protease of the coronavirus (Mpro) are becoming more and more promising candidates.  Mpro holds a pivotal role during the onset of the infection and its function is intimately related with the beginning of viral replication.  The interruption of its catalytic activity could represent a relevant strategy for the development of anti-coronavirus drugs.  SARS-CoV-2 Mpro is a peculiar cysteine protease of the coronavirus family, responsible for the replication and infectivity of the parasite.  This review offers a detailed anal. of the repurposed drugs and the newly synthesized mols. developed to date for the treatment of COVID-19 which share the common feature of targeting SARS-CoV-2 Mpro, as well as a brief overview of the main enzymic and cell-based assays to efficaciously screen such compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnucMIWuN98rVg90H21EOLACvtfcHk0ljZIrLWmC4F_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtlSnu7bO&md5=2e1820af6f975e353c5f2bcd3c1f31fb</span></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=10.3390%2Fbiom11040607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom11040607%26sid%3Dliteratum%253Aachs%26aulast%3DCitarella%26aufirst%3DA.%26aulast%3DScala%26aufirst%3DA.%26aulast%3DPiperno%26aufirst%3DA.%26aulast%3DMicale%26aufirst%3DN.%26atitle%3DSARS-CoV-2%2520Mpro%253A%2520A%2520Potential%2520Target%2520for%2520Peptidomimetics%2520and%2520Small-Molecule%2520Inhibitors%26jtitle%3DBiomolecules%26date%3D2021%26volume%3D11%26issue%3D11%26spage%3D607%26doi%3D10.3390%2Fbiom11040607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25b"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Yang K and Yang H Recent Progress in the Drug Development Targeting SARS-CoV-2 Main Protease as Treatment for COVID-19</span>. <i>Front. Mol. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">616341</span>, <span class="refDoi"> DOI: 10.3389/fmolb.2020.616341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.3389%2Ffmolb.2020.616341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33344509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXlsVWru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2020&pages=616341&author=W.+Cuiauthor=K.+Yangauthor=H.+Yang&title=Yang+K+and+Yang+H+Recent+Progress+in+the+Drug+Development+Targeting+SARS-CoV-2+Main+Protease+as+Treatment+for+COVID-19&doi=10.3389%2Ffmolb.2020.616341"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25bR"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in the drug development targeting SARS-CoV-2 main protease as treatment for COVID-19</span></div><div class="casAuthors">Cui, Wen; Yang, Kailin; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Molecular Biosciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">616341</span>CODEN:
                <span class="NLM_cas:coden">FMBRBS</span>;
        ISSN:<span class="NLM_cas:issn">2296-889X</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">The sudden outbreak of 2019 novel coronavirus (2019-nCoV, later named SARS-CoV-2) rapidly turned into an unprecedented pandemic of coronavirus disease 2019 (COVID-19).  This global healthcare emergency marked the third occurrence of a deadly coronavirus (CoV) into the human society after entering the new millennium, which overwhelmed the worldwide healthcare system and affected the global economy.  However, therapeutic options for COVID-19 are still very limited.  Developing drugs targeting vital proteins in viral life cycle is a feasible approach to overcome this dilemma.  Main protease (Mpro) plays a dominant role in processing CoV-encoded polyproteins which mediate the assembly of replication-transcription machinery and is thus recognized as an ideal antiviral target.  Here we summarize the recent progress in the discovery of anti-SARS-CoV-2 agents against Mpro.  Combining structural study, virtual screen, and exptl. screen, numerous therapeutic candidates including repurposed drugs and ab initio designed compds. have been proposed.  Such collaborative effort from the scientific community would accelerate the pace of developing efficacious treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptLZWCMNrk9LVg90H21EOLACvtfcHk0lhf9N8WJBnURg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXlsVWru7o%253D&md5=f03e7aed6bbe5cf3e7e57070544ecac2</span></div><a href="/servlet/linkout?suffix=cit25b&amp;dbid=16384&amp;doi=10.3389%2Ffmolb.2020.616341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmolb.2020.616341%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DYang%2520K%2520and%2520Yang%2520H%2520Recent%2520Progress%2520in%2520the%2520Drug%2520Development%2520Targeting%2520SARS-CoV-2%2520Main%2520Protease%2520as%2520Treatment%2520for%2520COVID-19%26jtitle%3DFront.%2520Mol.%2520Biosci.%26date%3D2020%26volume%3D7%26spage%3D616341%26doi%3D10.3389%2Ffmolb.2020.616341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25c"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cannalire, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beccari, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Leva, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summa, V.</span></span> <span> </span><span class="NLM_article-title">Targeting SARS-CoV-2 Proteases and Polymerase for COVID-19 Treatment: State of the Art and Future Opportunities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01140</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=R.+Cannalireauthor=C.+Cerchiaauthor=A.+R.+Beccariauthor=F.+S.+Di+Levaauthor=V.+Summa&title=Targeting+SARS-CoV-2+Proteases+and+Polymerase+for+COVID-19+Treatment%3A+State+of+the+Art+and+Future+Opportunities&doi=10.1021%2Facs.jmedchem.0c01140"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01140%26sid%3Dliteratum%253Aachs%26aulast%3DCannalire%26aufirst%3DR.%26aulast%3DCerchia%26aufirst%3DC.%26aulast%3DBeccari%26aufirst%3DA.%2BR.%26aulast%3DDi%2BLeva%26aufirst%3DF.%2BS.%26aulast%3DSumma%26aufirst%3DV.%26atitle%3DTargeting%2520SARS-CoV-2%2520Proteases%2520and%2520Polymerase%2520for%2520COVID-19%2520Treatment%253A%2520State%2520of%2520the%2520Art%2520and%2520Future%2520Opportunities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26doi%3D10.1021%2Facs.jmedchem.0c01140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manickam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S.-H.</span></span> <span> </span><span class="NLM_article-title">An Overview of Severe Acute Respiratory Syndrome–Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6595</span>– <span class="NLM_lpage">6628</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01461</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01461" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVyns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6595-6628&author=T.+Pillaiyarauthor=M.+Manickamauthor=V.+Namasivayamauthor=Y.+Hayashiauthor=S.-H.+Jung&title=An+Overview+of+Severe+Acute+Respiratory+Syndrome%E2%80%93Coronavirus+%28SARS-CoV%29+3CL+Protease+Inhibitors%3A+Peptidomimetics+and+Small+Molecule+Chemotherapy&doi=10.1021%2Facs.jmedchem.5b01461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Manickam, Manoj; Namasivayam, Vigneshwaran; Hayashi, Yoshio; Jung, Sang-Hun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6595-6628</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome (SARS) is caused by a newly emerged coronavirus that infected more than 8000 individuals and resulted in more than 800 (10-15%) fatalities in 2003.  The causative agent of SARS has been identified as a novel human coronavirus (SARS-CoV), and its viral protease, SARS-CoV 3CLpro, has been shown to be essential for replication and has hence been recognized as a potent drug target for SARS infection.  Currently, there is no effective treatment for this epidemic despite the intensive research that has been undertaken since 2003 (over 3500 publications).  This perspective focuses on the status of various efficacious anti-SARS-CoV 3CLpro chemotherapies discovered during the last 12 years (2003-2015) from all sources, including lab. synthetic methods, natural products, and virtual screening.  We describe here mainly peptidomimetic and small mol. inhibitors of SARS-CoV 3CLpro.  Attempts have been made to provide a complete description of the structural features and binding modes of these inhibitors under many conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB_JCXe-m0RLVg90H21EOLACvtfcHk0lhf9N8WJBnURg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVyns7c%253D&md5=6ce37076204280f64a7f362633fcee1a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01461%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DManickam%26aufirst%3DM.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DJung%26aufirst%3DS.-H.%26atitle%3DAn%2520Overview%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome%25E2%2580%2593Coronavirus%2520%2528SARS-CoV%2529%25203CL%2520Protease%2520Inhibitors%253A%2520Peptidomimetics%2520and%2520Small%2520Molecule%2520Chemotherapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6595%26epage%3D6628%26doi%3D10.1021%2Facs.jmedchem.5b01461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.-M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.-K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-based+design+of+antiviral+drug+candidates+targeting+the+SARS-CoV-2+main+protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0lgSvNHNR3392g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.-M.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.-K.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-based%2520design%2520of%2520antiviral%2520drug%2520candidates%2520targeting%2520the%2520SARS-CoV-2%2520main%2520protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Q.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y.-H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 Mpro inhibitors with antiviral activity in a transgenic mouse model</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">1374</span>– <span class="NLM_lpage">1378</span>, <span class="refDoi"> DOI: 10.1126/science.abf1611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1126%2Fscience.abf1611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33602867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2021&pages=1374-1378&author=J.+Qiaoauthor=Y.-S.+Liauthor=R.+Zengauthor=F.-L.+Liuauthor=R.-H.+Luoauthor=C.+Huangauthor=Y.-F.+Wangauthor=J.+Zhangauthor=B.+Quanauthor=C.+Shenauthor=X.+Maoauthor=X.+Liuauthor=W.+Sunauthor=W.+Yangauthor=X.+Niauthor=K.+Wangauthor=L.+Xuauthor=Z.-L.+Duanauthor=Q.-C.+Zouauthor=H.-L.+Zhangauthor=W.+Quauthor=Y.-H.-P.+Longauthor=M.-H.+Liauthor=R.-C.+Yangauthor=X.+Liuauthor=J.+Youauthor=Y.+Zhouauthor=R.+Yaoauthor=W.-P.+Liauthor=J.-M.+Liuauthor=P.+Chenauthor=Y.+Liuauthor=G.-F.+Linauthor=X.+Yangauthor=J.+Zouauthor=L.+Liauthor=Y.+Huauthor=G.-W.+Luauthor=W.-M.+Liauthor=Y.-Q.+Weiauthor=Y.-T.+Zhengauthor=J.+Leiauthor=S.+Yang&title=SARS-CoV-2+Mpro+inhibitors+with+antiviral+activity+in+a+transgenic+mouse+model&doi=10.1126%2Fscience.abf1611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro inhibitors with antiviral activity in a transgenic mouse model</span></div><div class="casAuthors">Qiao, Jingxin; Li, Yue-Shan; Zeng, Rui; Liu, Feng-Liang; Luo, Rong-Hua; Huang, Chong; Wang, Yi-Fei; Zhang, Jie; Quan, Baoxue; Shen, Chenjian; Mao, Xin; Liu, Xinlei; Sun, Weining; Yang, Wei; Ni, Xincheng; Wang, Kai; Xu, Ling; Duan, Zi-Lei; Zou, Qing-Cui; Zhang, Hai-Lin; Qu, Wang; Long, Yang-Hao-Peng; Li, Ming-Hua; Yang, Rui-Cheng; Liu, Xiaolong; You, Jing; Zhou, Yangli; Yao, Rui; Li, Wen-Pei; Liu, Jing-Ming; Chen, Pei; Liu, Yang; Lin, Gui-Feng; Yang, Xin; Zou, Jun; Li, Linli; Hu, Yiguo; Lu, Guang-Wen; Li, Wei-Min; Wei, Yu-Quan; Zheng, Yong-Tang; Lei, Jian; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6536</span>),
    <span class="NLM_cas:pages">1374-1378</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continually poses serious threats to global public health.  The main protease (Mpro) of SARS-CoV-2 plays a central role in viral replication.  We designed and synthesized 32 new bicycloproline-contg. Mpro inhibitors derived from either boceprevir or telaprevir, both of which are approved antivirals.  All compds. inhibited SARS-CoV-2 Mpro activity in vitro, with 50% inhibitory concn. values ranging from 7.6 to 748.5 nM.  The cocrystal structure of Mpro in complex with MI-23, one of the most potent compds., revealed its interaction mode.  Two compds. (MI-09 and MI-30) showed excellent antiviral activity in cell-based assays.  In a transgenic mouse model of SARS-CoV-2 infection, oral or i.p. treatment with MI-09 or MI-30 significantly reduced lung viral loads and lung lesions.  Both also displayed good pharmacokinetic properties and safety in rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVMv8s8s-oWLVg90H21EOLACvtfcHk0lgSvNHNR3392g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D&md5=555937b1776c8353e59601d567e2ebb9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fscience.abf1611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abf1611%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.-S.%26aulast%3DZeng%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DF.-L.%26aulast%3DLuo%26aufirst%3DR.-H.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DQuan%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DDuan%26aufirst%3DZ.-L.%26aulast%3DZou%26aufirst%3DQ.-C.%26aulast%3DZhang%26aufirst%3DH.-L.%26aulast%3DQu%26aufirst%3DW.%26aulast%3DLong%26aufirst%3DY.-H.-P.%26aulast%3DLi%26aufirst%3DM.-H.%26aulast%3DYang%26aufirst%3DR.-C.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DW.-P.%26aulast%3DLiu%26aufirst%3DJ.-M.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DG.-F.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DG.-W.%26aulast%3DLi%26aufirst%3DW.-M.%26aulast%3DWei%26aufirst%3DY.-Q.%26aulast%3DZheng%26aufirst%3DY.-T.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DSARS-CoV-2%2520Mpro%2520inhibitors%2520with%2520antiviral%2520activity%2520in%2520a%2520transgenic%2520mouse%2520model%26jtitle%3DScience%26date%3D2021%26volume%3D371%26spage%3D1374%26epage%3D1378%26doi%3D10.1126%2Fscience.abf1611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+%CE%B1-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0ljVCb5_yify8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520%25CE%25B1-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2223-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fs41586-020-2223-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32272481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=289-293&author=Z.+Jinauthor=X.+Duauthor=Y.+Xuauthor=Y.+Dengauthor=L.+Liuauthor=Y.+Zhaoauthor=B.+Zhangauthor=X.+Liauthor=L.+Zhangauthor=C.+Pengauthor=Y.+Duanauthor=J.+Yuauthor=L.+Wangauthor=K.+Yangauthor=F.+Liuauthor=R.+Jiangauthor=X.+Yangauthor=T.+Youauthor=X.+Liuauthor=X.+Yangauthor=F.+Baiauthor=H.+Liuauthor=X.+Liuauthor=L.+W.+Guddatauthor=W.+Xuauthor=G.+Xiaoauthor=C.+Qinauthor=Z.+Shiauthor=H.+Jiangauthor=Z.+Raoauthor=H.+Yang&title=Structure+of+Mpro+from+SARS-CoV-2+and+discovery+of+its+inhibitors&doi=10.1038%2Fs41586-020-2223-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span></div><div class="casAuthors">Jin, Zhenming; Du, Xiaoyu; Xu, Yechun; Deng, Yongqiang; Liu, Meiqin; Zhao, Yao; Zhang, Bing; Li, Xiaofeng; Zhang, Leike; Peng, Chao; Duan, Yinkai; Yu, Jing; Wang, Lin; Yang, Kailin; Liu, Fengjiang; Jiang, Rendi; Yang, Xinglou; You, Tian; Liu, Xiaoce; Yang, Xiuna; Bai, Fang; Liu, Hong; Liu, Xiang; Guddat, Luke W.; Xu, Wenqing; Xiao, Gengfu; Qin, Chengfeng; Shi, Zhengli; Jiang, Hualiang; Rao, Zihe; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7811</span>),
    <span class="NLM_cas:pages">289-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A new coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is the etiol. agent responsible for the 2019-2020 viral pneumonia outbreak of coronavirus disease 2019 (COVID-19).  Currently, there are no targeted therapeutic agents for the treatment of this disease, and effective treatment options remain very limited.  Here, we describe the results of a program that aimed to rapidly discover lead compds. for clin. use, by combining structure-assisted drug design, virtual drug screening and high-throughput screening.  This program focused on identifying drug leads that target main protease (Mpro) of SARS-CoV-2: Mpro is a key enzyme of coronaviruses and has a pivotal role in mediating viral replication and transcription, making it an attractive drug target for SARS-CoV-2.  We identified a mechanism-based inhibitor (N3) by computer-aided drug design, and then detd. the crystal structure of Mpro of SARS-CoV-2 in complex with this compd.  Through a combination of structure-based virtual and high-throughput screening, we assayed more than 10,000 compds.-including approved drugs, drug candidates in clin. trials and other pharmacol. active compds.-as inhibitors of Mpro.  Six of these compds. inhibited Mpro, showing half-maximal inhibitory concn. values that ranged from 0.67 to 21.4μM.  One of these compds. (ebselen) also exhibited promising antiviral activity in cell-based assays.  Our results demonstrate the efficacy of our screening strategy, which can lead to the rapid discovery of drug leads with clin. potential in response to new infectious diseases for which no specific drugs or vaccines are available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFupuh6WIyW7Vg90H21EOLACvtfcHk0ljVCb5_yify8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO&md5=b84f350fe9ce1109485df6caf814ba82</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2223-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2223-y%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DStructure%2520of%2520Mpro%2520from%2520SARS-CoV-2%2520and%2520discovery%2520of%2520its%2520inhibitors%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D289%26epage%3D293%26doi%3D10.1038%2Fs41586-020-2223-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copp, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauls, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span> <span> </span><span class="NLM_article-title">New Inhibitors of Cysteine Proteinases. Peptidyl Acyloxymethylketones and the Quiescent Nucleofuge Strategy</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">4429</span>– <span class="NLM_lpage">4431</span>, <span class="refDoi"> DOI: 10.1021/ja00221a062</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00221a062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaL1cXktlCguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=4429-4431&author=R.+A.+Smithauthor=L.+J.+Coppauthor=P.+J.+Colesauthor=H.+W.+Paulsauthor=V.+J.+Robinsonauthor=R.+W.+Spencerauthor=S.+B.+Heardauthor=A.+Krantz&title=New+Inhibitors+of+Cysteine+Proteinases.+Peptidyl+Acyloxymethylketones+and+the+Quiescent+Nucleofuge+Strategy&doi=10.1021%2Fja00221a062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31aR"><div class="casContent"><span class="casTitleNuber">31a</span><div class="casTitle"><span class="NLM_cas:atitle">New inhibitors of cysteine proteinases.  Peptidyl acyloxymethyl ketones and the quiescent nucleofuge strategy</span></div><div class="casAuthors">Smith, Roger A.; Copp, Leslie J.; Coles, Peter J.; Pauls, Henry W.; Robinson, Valerie J.; Spencer, Robin W.; Heard, Stephen B.; Krantz, Allen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4429-31</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Peptidyl acyloxymethyl ketones are potent and specific inhibitors of the cysteine proteinase, cathepsin B.  These inhibitors are affinity labels in which the peptide moiety serves to transport an aryl carboxylate nucleofuge that is uniquely reactive toward the active-site cysteine thiol of the target enzyme.  Heretofore, in constructing affinity labels the choice of leaving group was severely restricted.  The aryl carboxylate group offers considerable variation as a design element in that its binding affinity and reactivity can be controlled by substituent effects.  The apparent 2nd-order rate const. which characterizes the inhibition of bovine spleen cathepsin B by this series spans several orders of magnitude and in certain cases exceeds 106 M-1 s-1.  The inhibition is time-dependent, active-site directed, and irreversible.  The inactivation requires that the peptide component contain high affinity recognition elements for this enzyme and that the aryl carboxylate leaving group has a pKa <4.  Impressive inhibitory activity is also obsd. for certain peptidyl aryloxymethyl ketones.  NMR resonances of the inactivated adduct resulting from the action of representative 13C-labeled acyloxymethyl and aryloxymethyl ketone inhibitors on the model cysteine proteinase papain are consistent with the proposed mechanism of inactivation, involving displacement of the nucleofuge by the active-site cysteine to give a (cysteine-25)-thiomethyl ketone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0tvSCUqAffrVg90H21EOLACvtfcHk0ljVCb5_yify8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktlCguro%253D&md5=12b38defc0f9731ff6adeebac78b0711</span></div><a href="/servlet/linkout?suffix=cit31a&amp;dbid=16384&amp;doi=10.1021%2Fja00221a062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00221a062%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DCopp%26aufirst%3DL.%2BJ.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DPauls%26aufirst%3DH.%2BW.%26aulast%3DRobinson%26aufirst%3DV.%2BJ.%26aulast%3DSpencer%26aufirst%3DR.%2BW.%26aulast%3DHeard%26aufirst%3DS.%2BB.%26aulast%3DKrantz%26aufirst%3DA.%26atitle%3DNew%2520Inhibitors%2520of%2520Cysteine%2520Proteinases.%2520Peptidyl%2520Acyloxymethylketones%2520and%2520the%2520Quiescent%2520Nucleofuge%2520Strategy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D4429%26epage%3D4431%26doi%3D10.1021%2Fja00221a062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit31b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copp, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, S. B.</span></span> <span> </span><span class="NLM_article-title">Peptidyl (Acyloxy)methyl Ketones and the Quiescent Affinity Label Concept: The Departing Group as a Variable Structure Element in the Design of Inactivators of Cysteine Proteinases</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">4678</span>– <span class="NLM_lpage">4687</span>, <span class="refDoi"> DOI: 10.1021/bi00233a007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00233a007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK3MXit1CgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1991&pages=4678-4687&author=A.+Krantzauthor=L.+J.+Coppauthor=P.+J.+Colesauthor=R.+A.+Smithauthor=S.+B.+Heard&title=Peptidyl+%28Acyloxy%29methyl+Ketones+and+the+Quiescent+Affinity+Label+Concept%3A+The+Departing+Group+as+a+Variable+Structure+Element+in+the+Design+of+Inactivators+of+Cysteine+Proteinases&doi=10.1021%2Fbi00233a007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31bR"><div class="casContent"><span class="casTitleNuber">31b</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl (acyloxy)methyl ketones and the quiescent affinity label concept:  the departing group as a variable structural element in the design of inactivators of cysteine proteinases</span></div><div class="casAuthors">Krantz, Allen; Copp, Leslie J.; Coles, Peter J.; Smith, Roger A.; Heard, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4678-87</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">(Acyloxy)methyl ketones, of general structure Z-[AA2]-[AA1]-CH2OCOAr (Z = benzyloxycarbonyl; AA = amino acid; Ar = aryl), were potent inactivators of the cysteine proteinase, cathepsin B.  These reagents were designed as affinity labels in which the dipeptidyl moiety serves as an affinity group (complementary to the S1 and S2 sites of the enzyme), while the (acyloxy)methyl ketone unit (-COCH2OCOR), contg. a weak leaving group in the form of a carboxylate nucleofuge, functions as the potentially reactive entity that labels the enzyme.  The inhibition was time-dependent, active-site directed, and irreversible.  The apparent 2nd-order rate const. (kinact/Kinact), which characterizes the inhibition of cathepsin B by this series, spanned several orders of magnitude and in certain cases exceeded 106 M-1 s-1.  The activity of this series of inhibitors was found to be exquisitely sensitive to the nature of the carboxylate leaving group as well as the affinity group.  A strong dependence of 2nd-order inactivation rate on leaving group pKa was uncovered for Z-Phe-Ala (acyloxy)methyl ketones [log(k/K) = -1.1 × pKa + 7.2.  Heretofore, in constructing affinity labels the choice of leaving group was quite restricted.  The aryl carboxylate group thus offers considerable variation as a design element in that both its binding affinity and reactivity can be controlled by substituent effects.  Specific peptidyl (acyloxy)methyl ketones thus represent prime examples of highly potent, chem. stable enzyme inhibitors with variable structural elements in both the affinity and departing groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprQo4CgIuJM7Vg90H21EOLACvtfcHk0lgYrgQvTg6Efw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXit1CgsLY%253D&md5=887882366c35b5cba8021d1d14eb47be</span></div><a href="/servlet/linkout?suffix=cit31b&amp;dbid=16384&amp;doi=10.1021%2Fbi00233a007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00233a007%26sid%3Dliteratum%253Aachs%26aulast%3DKrantz%26aufirst%3DA.%26aulast%3DCopp%26aufirst%3DL.%2BJ.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DHeard%26aufirst%3DS.%2BB.%26atitle%3DPeptidyl%2520%2528Acyloxy%2529methyl%2520Ketones%2520and%2520the%2520Quiescent%2520Affinity%2520Label%2520Concept%253A%2520The%2520Departing%2520Group%2520as%2520a%2520Variable%2520Structure%2520Element%2520in%2520the%2520Design%2520of%2520Inactivators%2520of%2520Cysteine%2520Proteinases%26jtitle%3DBiochemistry%26date%3D1991%26volume%3D30%26spage%3D4678%26epage%3D4687%26doi%3D10.1021%2Fbi00233a007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit31c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauls, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span> <span> </span><span class="NLM_article-title">Carbon-13 NMR characterization of the Papain Adduct Formed by Peptidyl Acyloxy-, Aryloxy, and Chloromethyl Ketone Irrversible Inhibitors</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/0045-2068(92)90024-W</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2F0045-2068%2892%2990024-W" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK38Xkt1agsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1992&pages=42-54&author=V.+J.+Robinsonauthor=H.+W.+Paulsauthor=P.+J.+Colesauthor=R.+A.+Smithauthor=A.+Krantz&title=Carbon-13+NMR+characterization+of+the+Papain+Adduct+Formed+by+Peptidyl+Acyloxy-%2C+Aryloxy%2C+and+Chloromethyl+Ketone+Irrversible+Inhibitors&doi=10.1016%2F0045-2068%2892%2990024-W"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31cR"><div class="casContent"><span class="casTitleNuber">31c</span><div class="casTitle"><span class="NLM_cas:atitle">Carbon-13 NMR characterization of the papain adduct formed by peptidyl acyloxy-, aryloxy-, and chloromethyl ketone irreversible inhibitors</span></div><div class="casAuthors">Robinson, Valerie J.; Pauls, Henry W.; Coles, Peter J.; Smith, Roger A.; Krantz, Allen</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-54</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    </div><div class="casAbstract">N-Benzoyloxycarbonyl-L-phenylalanyl-glycine-derived mesitoyloxy- and pentafluorophenoxymethyl ketones, incorporating 13C labels at the ketone carbonyl and α-keto methylene positions, were synthesized.  These compds. act as quiescent affinity labels of cysteine proteinases.  NMR studies established that these compds. form an irreversible covalent adduct with papain, via release of mesitoic acid or pentafluorophenol.  The same adduct was formed in each case, with 13C NMR signals at 214.7 ppm (COCH2) and 38.1 ppm (COCH2).  Identical spectra were obsd. for the adduct formed from the analogous 13C-labeled chloromethyl ketones; the structure of this type of adduct has been previously established by x-ray studies.  The NMR data are consistent with this structure, and provide a sensitive measure of the hydrogen bonding to the π-carbonyl.  Disruption of this hydrogen bonding by denaturation produced a 13C NMR shift of 10 ppm to higher field in the carbonyl carbon, to a chem. shift in accordance with that of a model thiomethyl ketone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqons7A9nr4NbVg90H21EOLACvtfcHk0lgYrgQvTg6Efw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xkt1agsrw%253D&md5=691e9b356f4262b75cdcb4042a583792</span></div><a href="/servlet/linkout?suffix=cit31c&amp;dbid=16384&amp;doi=10.1016%2F0045-2068%2892%2990024-W&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0045-2068%252892%252990024-W%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DV.%2BJ.%26aulast%3DPauls%26aufirst%3DH.%2BW.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DKrantz%26aufirst%3DA.%26atitle%3DCarbon-13%2520NMR%2520characterization%2520of%2520the%2520Papain%2520Adduct%2520Formed%2520by%2520Peptidyl%2520Acyloxy-%252C%2520Aryloxy%252C%2520and%2520Chloromethyl%2520Ketone%2520Irrversible%2520Inhibitors%26jtitle%3DBioorg.%2520Chem.%26date%3D1992%26volume%3D20%26spage%3D42%26epage%3D54%26doi%3D10.1016%2F0045-2068%2892%2990024-W" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yeo, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, S. C.</span></span> <span> </span><span class="NLM_article-title">The aminopeptidase inhibitor, z-CMK, is toxic and induces cell death in Jurkat T cells through oxidative stress</span>. <i>Toxicol. Mech. Methods</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1080/15376516.2017.1373882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1080%2F15376516.2017.1373882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=28849708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFWgsL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=157-166&issue=3&author=E.+H.+Yeoauthor=W.+L.+Gohauthor=S.+C.+Chow&title=The+aminopeptidase+inhibitor%2C+z-CMK%2C+is+toxic+and+induces+cell+death+in+Jurkat+T+cells+through+oxidative+stress&doi=10.1080%2F15376516.2017.1373882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The aminopeptidase inhibitor, z-L-CMK, is toxic and induces cell death in Jurkat T cells through oxidative stress</span></div><div class="casAuthors">Yeo, E. H.; Goh, W. L.; Chow, S. C.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Mechanisms and Methods</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">157-166</span>CODEN:
                <span class="NLM_cas:coden">TMMOCP</span>;
        ISSN:<span class="NLM_cas:issn">1537-6516</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The leucine aminopeptidase inhibitor, benzyloxycarbonyl-leucine-chloromethylketone (z-L-CMK), was found to be toxic and readily induce cell death in Jurkat T cells.  Dose-response studies show that lower concn. of z-L-CMK induced apoptosis in Jurkat T cells whereas higher concn. causes necrosis.  In z-L-CMK-induced apoptosis, both the initiator caspases (-8 and -9) and effector caspases (-3 and -6) were processed to their resp. subunits.  However, the caspases remained intact in z-L-CMK-induced necrosis.  The caspase inhibitor, z-VAD-FMK inhibited z-L-CMK-mediated apoptosis and caspase processing but has no effect on z-L-CMK-induced necrosis in Jurkat T cells.  The high mobility group protein B1 (HMGB1) protein was found to be released into the culture medium by the necrotic cells and not the apoptotic cells.  These results indicate that the necrotic cell death mediated by z-L-CMK at high concns. is via classical necrosis rather than secondary necrosis.  We also demonstrated that cell death mediated by z-L-CMK was assocd. with oxidative stress via the depletion of intracellular glutathione (GSH) and increase in reactive oxygen species (ROS), which was blocked by N-acetyl cysteine.  Taken together, the results demonstrated that z-L-CMK is toxic to Jurkat T cells and induces apoptosis at low concns., while at higher concns. the cells die of necrosis.  The toxic side effects in Jurkat T cells mediated by z-L-CMK are assocd. with oxidative stress via the depletion of GSH and accumulation of ROS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogQL4jAZ8JlbVg90H21EOLACvtfcHk0lgYrgQvTg6Efw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWgsL7P&md5=7b423409408370bd1a7dceffefae5a3b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1080%2F15376516.2017.1373882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15376516.2017.1373882%26sid%3Dliteratum%253Aachs%26aulast%3DYeo%26aufirst%3DE.%2BH.%26aulast%3DGoh%26aufirst%3DW.%2BL.%26aulast%3DChow%26aufirst%3DS.%2BC.%26atitle%3DThe%2520aminopeptidase%2520inhibitor%252C%2520z-CMK%252C%2520is%2520toxic%2520and%2520induces%2520cell%2520death%2520in%2520Jurkat%2520T%2520cells%2520through%2520oxidative%2520stress%26jtitle%3DToxicol.%2520Mech.%2520Methods%26date%3D2018%26volume%3D28%26issue%3D3%26spage%3D157%26epage%3D166%26doi%3D10.1080%2F15376516.2017.1373882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eichhold, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookfin, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taiwo, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehmeyer, K. R.</span></span> <span> </span><span class="NLM_article-title">Isolation and quantification of fluoroacetate in rat tissues, following dosing of Z-Phe-Ala-CH2-F, a peptidyl fluoromethyl ketone protease inhibitor</span>. <i>J. Pharm. Biomed. Anal.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1016/S0731-7085(97)00102-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2FS0731-7085%2897%2900102-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=9589405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK2sXnvFWqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1997&pages=459-467&issue=3&author=T.+H.+Eichholdauthor=E.+B.+Hookfinauthor=Y.+O.+Taiwoauthor=B.+Deauthor=K.+R.+Wehmeyer&title=Isolation+and+quantification+of+fluoroacetate+in+rat+tissues%2C+following+dosing+of+Z-Phe-Ala-CH2-F%2C+a+peptidyl+fluoromethyl+ketone+protease+inhibitor&doi=10.1016%2FS0731-7085%2897%2900102-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation and quantification of fluoroacetate in rat tissues, following dosing of Z-Phe-Ala-CH2-F, a peptidyl fluoromethyl ketone protease inhibitor</span></div><div class="casAuthors">Eichhold, Thomas H.; Hookfin, Erin B.; Taiwo, Yetunde O.; De, Biswanath; Wehmeyer, Kenneth R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">459-467</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Peptidyl fluoromethyl ketones (PFMK) are irreversible inhibitors of cathepsin B, a cysteine proteinase thought to be involved in the degrdn. of cartilage.  It has been speculated that PFMK inhibitors may metabolize in rodents to form fluoroacetate (FAC), an extremely toxic poison.  A highly selective and sensitive sepn. and detection scheme was developed to measure trace levels of FAC in rat tissues following PFMK dosing.  The procedure consisted of extg. FAC from tissue and spiking the ext. with [18O]2-fluoroacetate (18O-FAC) as an internal std.  FAC and 18O-FAC were further isolated from matrix components using ion-exchange, solid-phase extn.  The pentafluorobenzyl esters of FAC and 18O-FAC were formed to facilitate the chromatog. sepn.  Two-dimensional gas chromatog. coupled with selected-ion-monitoring detection provided the final measurement.  The assay had a limit of detection of 2 ng FAC per g tissue, and was capable of accurately quantitating as little as 10 ng FAC per g tissue with a S/N ratio of 40:1.  Linearity was established over two orders of magnitude, from 2-500 ng ml-1, with 5 μl injected on-column.  The method was used to demonstrate that FAC was formed in rats following dosing with Z-Phe-Ala-CH2-F, a PFMK cathepsin enzyme inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPmtRizUZf2bVg90H21EOLACvtfcHk0ljoxHnDDok1yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnvFWqsL8%253D&md5=ea69dcdcaff325ed19d40e94c6f96760</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0731-7085%2897%2900102-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0731-7085%252897%252900102-7%26sid%3Dliteratum%253Aachs%26aulast%3DEichhold%26aufirst%3DT.%2BH.%26aulast%3DHookfin%26aufirst%3DE.%2BB.%26aulast%3DTaiwo%26aufirst%3DY.%2BO.%26aulast%3DDe%26aufirst%3DB.%26aulast%3DWehmeyer%26aufirst%3DK.%2BR.%26atitle%3DIsolation%2520and%2520quantification%2520of%2520fluoroacetate%2520in%2520rat%2520tissues%252C%2520following%2520dosing%2520of%2520Z-Phe-Ala-CH2-F%252C%2520a%2520peptidyl%2520fluoromethyl%2520ketone%2520protease%2520inhibitor%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D1997%26volume%3D16%26issue%3D3%26spage%3D459%26epage%3D467%26doi%3D10.1016%2FS0731-7085%2897%2900102-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolle, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whipple, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osifo, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graybill, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helaszek, C. T.</span></span> <span> </span><span class="NLM_article-title">Aspartyl α-((Diphenylphosphinyl)oxy) methyl Ketones as Novel Inhibitors of Interleukin-1β Converting Enzyme. Utility of the Dephenylphosphinic Acid Leaving Group for the Inhibition of Cysteine Proteases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1021/jm00002a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00002a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK2MXivFOqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1995&pages=220-222&issue=38&author=R.+E.+Dolleauthor=J.+Singhauthor=D.+Whippleauthor=I.+K.+Osifoauthor=G.+Speierauthor=T.+L.+Graybillauthor=J.+S.+Gregoryauthor=A.+L.+Harrisauthor=C.+T.+Helaszek&title=Aspartyl+%CE%B1-%28%28Diphenylphosphinyl%29oxy%29+methyl+Ketones+as+Novel+Inhibitors+of+Interleukin-1%CE%B2+Converting+Enzyme.+Utility+of+the+Dephenylphosphinic+Acid+Leaving+Group+for+the+Inhibition+of+Cysteine+Proteases&doi=10.1021%2Fjm00002a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34aR"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Aspartyl α-((Diphenylphosphinyl)oxy)methyl Ketones as Novel Inhibitors of Interleukin-1β Converting Enzyme. Utility of the Diphenylphosphinic Acid Leaving Group for the Inhibition of Cysteine Proteases</span></div><div class="casAuthors">Dolle, Roland E.; Singh, Jasbir; Whipple, David; Osifo, I. Kelly; Speier, Gary; Graybill, Todd L.; Gregory, Jill S.; Harris, Alex L.; Helaszek, Carla T.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">220-2</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mono-, di- and tripeptide aspartyl α-((diphenylphosphinyl)oxy)methyl ketones are potent time-dependent inactivators of interleukin-1β converting enzyme (ICE).  Z-Val-Ala-Asp-CH2OP(O)Ph2 exhibits a second order rate const. (kobs/[I]) of 117,000 M-1 s-1.  By incorporating the (diphenylphosphinyl)oxy- (DPP) leaving group into the peptide scaffolds Z-Phe-Ala and Z-Leu-Phe, potent inhibitors of cathepsin B and calpain I were identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-O3lqW_Qo77Vg90H21EOLACvtfcHk0ljoxHnDDok1yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXivFOqtbo%253D&md5=ab67d2a8cb7e70900b8bf2f0cfd3a494</span></div><a href="/servlet/linkout?suffix=cit34a&amp;dbid=16384&amp;doi=10.1021%2Fjm00002a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00002a002%26sid%3Dliteratum%253Aachs%26aulast%3DDolle%26aufirst%3DR.%2BE.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DWhipple%26aufirst%3DD.%26aulast%3DOsifo%26aufirst%3DI.%2BK.%26aulast%3DSpeier%26aufirst%3DG.%26aulast%3DGraybill%26aufirst%3DT.%2BL.%26aulast%3DGregory%26aufirst%3DJ.%2BS.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DHelaszek%26aufirst%3DC.%2BT.%26atitle%3DAspartyl%2520%25CE%25B1-%2528%2528Diphenylphosphinyl%2529oxy%2529%2520methyl%2520Ketones%2520as%2520Novel%2520Inhibitors%2520of%2520Interleukin-1%25CE%25B2%2520Converting%2520Enzyme.%2520Utility%2520of%2520the%2520Dephenylphosphinic%2520Acid%2520Leaving%2520Group%2520for%2520the%2520Inhibition%2520of%2520Cysteine%2520Proteases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1995%26volume%3D38%26issue%3D38%26spage%3D220%26epage%3D222%26doi%3D10.1021%2Fjm00002a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit34b"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Graybill, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prouty, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speier, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolle, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helaszek, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ator, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strastes, J.</span></span> <span> </span><span class="NLM_article-title">α-((Tetronoyl)oxy)- and α-((Tetramoyl)oxy)methyl ketone inhibitors of the interleukin-1β converting enzyme (ICE)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(96)00574-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2FS0960-894X%2896%2900574-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADyaK2sXls1eiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1997&pages=41-46&issue=1&author=T.+L.+Graybillauthor=C.+P.+Proutyauthor=G.+J.+Speierauthor=D.+Hoyerauthor=R.+E.+Dolleauthor=C.+T.+Helaszekauthor=M.+A.+Atorauthor=J.+Uhlauthor=J.+Strastes&title=%CE%B1-%28%28Tetronoyl%29oxy%29-+and+%CE%B1-%28%28Tetramoyl%29oxy%29methyl+ketone+inhibitors+of+the+interleukin-1%CE%B2+converting+enzyme+%28ICE%29&doi=10.1016%2FS0960-894X%2896%2900574-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34bR"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">α-[(Tetronoyl)oxy]- and α-[(tetramoyl)oxy]methyl ketone inhibitors of the interleukin-1β converting enzyme (ICE)</span></div><div class="casAuthors">Graybill, Todd L.; Prouty, Catherine P.; Speier, Gary J.; Hoyer, Denton; Dolle, Roland E.; Helaszek, Carla T.; Ator, Mark A.; Uhl, Joanne; Strasters, Joost</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-46</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Aryl-substituted tetronic acids, tetramic acids, and cyclic β-dicarbonyl moieties were evaluated as leaving groups in the peptidyl-COCH2-X type inhibitor.  Tripeptidyl aspartyl α-[(tetronyl)oxy]- and α-[(tetramoyl)oxy]methyl ketone derivs. demonstrate potent time-dependent inhibition (kobs/[I] 100,000-250,000 M-1s-1) of the cysteine protease ICE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQwc1Cqp_ESrVg90H21EOLACvtfcHk0ljoxHnDDok1yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXls1eiuw%253D%253D&md5=65fb1f34ab92351433675e2462c3a394</span></div><a href="/servlet/linkout?suffix=cit34b&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2896%2900574-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252896%252900574-4%26sid%3Dliteratum%253Aachs%26aulast%3DGraybill%26aufirst%3DT.%2BL.%26aulast%3DProuty%26aufirst%3DC.%2BP.%26aulast%3DSpeier%26aufirst%3DG.%2BJ.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DDolle%26aufirst%3DR.%2BE.%26aulast%3DHelaszek%26aufirst%3DC.%2BT.%26aulast%3DAtor%26aufirst%3DM.%2BA.%26aulast%3DUhl%26aufirst%3DJ.%26aulast%3DStrastes%26aufirst%3DJ.%26atitle%3D%25CE%25B1-%2528%2528Tetronoyl%2529oxy%2529-%2520and%2520%25CE%25B1-%2528%2528Tetramoyl%2529oxy%2529methyl%2520ketone%2520inhibitors%2520of%2520the%2520interleukin-1%25CE%25B2%2520converting%2520enzyme%2520%2528ICE%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1997%26volume%3D7%26issue%3D1%26spage%3D41%26epage%3D46%26doi%3D10.1016%2FS0960-894X%2896%2900574-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abeles, R. H.</span></span> <span> </span><span class="NLM_article-title">Inactivation of Cysteine Proteases by (Acyloxy)methyl Ketones Using S′-P′ Interactions</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">6498</span>– <span class="NLM_lpage">6502</span>, <span class="refDoi"> DOI: 10.1021/bi0002378</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0002378" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD3cXivVWhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=6498-6502&author=Y.+Daiauthor=L.+Hedstromauthor=R.+H.+Abeles&title=Inactivation+of+Cysteine+Proteases+by+%28Acyloxy%29methyl+Ketones+Using+S%E2%80%B2-P%E2%80%B2+Interactions&doi=10.1021%2Fbi0002378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Inactivation of Cysteine Proteases by (Acyloxy)methyl Ketones Using S'-P' Interactions</span></div><div class="casAuthors">Dai, Yong; Hedstrom, Lizbeth; Abeles, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6498-6502</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have synthesized (acyloxy)methyl ketone inactivators of papain, cathepsin B, and interleukin-1β conversion enzyme (ICE) that interact with both the S and S' sub-sites.  The value of kinact/Ki for these inactivators is strongly dependent on the leaving group.  For example, Z-Phe-Gly-CH2-X is a poor inactivator of papain when X is OCOCH3 (kinact/Ki = 2.5 M-1 s-1) but becomes a potent inactivator when X is OCO-L-Leu-Z (kinact/Ki = 11 000 M-1 s-1).  Since these leaving groups have similar chem. reactivities, the difference in potency must be attributed to interactions with the S' sites.  The potency of the leaving group correlates with the P' specificity of papain.  Similar results are also obsd. for the inactivation of cathepsin B by these compds.  A series of inactivators with the general structure Fmoc-L-Asp-CH2-X were designed to inactivate ICE.  No inhibition was obsd. when X was OCOCH3.  In contrast, ICE is inactivated when X is OCO-D-Pro-Z (kinact/Ki = 131 M-1 s-1).  These results demonstrate that S'-P' interactions can be utilized to increase the efficacy and selectivity of (acyloxy)methyl ketone inactivators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6mPpAnfktvrVg90H21EOLACvtfcHk0lg2YSv9TiDZ2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXivVWhs70%253D&md5=807bf12e74f3f5fd799eaf9f97e89a06</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi0002378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0002378%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DY.%26aulast%3DHedstrom%26aufirst%3DL.%26aulast%3DAbeles%26aufirst%3DR.%2BH.%26atitle%3DInactivation%2520of%2520Cysteine%2520Proteases%2520by%2520%2528Acyloxy%2529methyl%2520Ketones%2520Using%2520S%25E2%2580%25B2-P%25E2%2580%25B2%2520Interactions%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D6498%26epage%3D6502%26doi%3D10.1021%2Fbi0002378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
de Plassche, M. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barniol-Xicota, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhelst, S. H. L.</span></span> <span> </span><span class="NLM_article-title">Peptidyl Acyloxymethyl Ketones as Activity-Based Probes for the Main Protease of SARS-CoV-2</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3383</span>– <span class="NLM_lpage">3388</span>, <span class="refDoi"> DOI: 10.1002/cbic.202000371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1002%2Fcbic.202000371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32717117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslKgsLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&pages=3383-3388&author=M.+A.+T.+van%0Ade+Plasscheauthor=M.+Barniol-Xicotaauthor=S.+H.+L.+Verhelst&title=Peptidyl+Acyloxymethyl+Ketones+as+Activity-Based+Probes+for+the+Main+Protease+of+SARS-CoV-2&doi=10.1002%2Fcbic.202000371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl Acyloxymethyl Ketones as Activity-Based Probes for the Main Protease of SARS-CoV-2</span></div><div class="casAuthors">van de Plassche, Merel A. T.; Barniol-Xicota, Marta; Verhelst, Steven H. L.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3383-3388</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The global pandemic caused by SARS-Co-V-2 calls for the fast development of antiviral drugs against this particular coronavirus.  Chem. tools to facilitate inhibitor discovery as well as detection of target engagement by hit or lead compds. from high-throughput screens are therefore in urgent need.  The authors here report novel, selective activity-based probes that enable detection of the SARS-Co-V-2 main protease.  The probes are based on acyloxymethyl ketone reactive electrophiles combined with a peptide sequence including unnatural amino acids that targets the nonprimed site of the main protease substrate binding cleft.  They are the first activity-based probes for the main protease of coronaviruses and display target labeling within a human proteome without background.  The authors expect that these reagents will be useful in the drug-development pipeline, not only for the current SARS-Co-V-2, but also for other coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBTq6Plv43lbVg90H21EOLACvtfcHk0lg2YSv9TiDZ2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslKgsLnJ&md5=3371c4fb01f0f852d78ccf7affe915ef</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fcbic.202000371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.202000371%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bde%2BPlassche%26aufirst%3DM.%2BA.%2BT.%26aulast%3DBarniol-Xicota%26aufirst%3DM.%26aulast%3DVerhelst%26aufirst%3DS.%2BH.%2BL.%26atitle%3DPeptidyl%2520Acyloxymethyl%2520Ketones%2520as%2520Activity-Based%2520Probes%2520for%2520the%2520Main%2520Protease%2520of%2520SARS-CoV-2%26jtitle%3DChemBioChem%26date%3D2020%26volume%3D21%26spage%3D3383%26epage%3D3388%26doi%3D10.1002%2Fcbic.202000371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jian, Y.</span></span> <span> </span><span class="NLM_article-title">The development of Coronavirus 3C-Like protease (3CLpro) inhibitors from 2010 to 2020</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>206</i></span>,  <span class="NLM_fpage">112711</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.ejmech.2020.112711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32810751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1CqtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=206&publication_year=2020&pages=112711&author=Y.+Liuauthor=C.+Liangauthor=L.+Xinauthor=X.+Renauthor=L.+Tianauthor=X.+Juauthor=H.+Liauthor=Y.+Wangauthor=Q.+Zhaoauthor=H.+Liuauthor=W.+Caoauthor=X.+Xieauthor=D.+Zhangauthor=Y.+Wangauthor=Y.+Jian&title=The+development+of+Coronavirus+3C-Like+protease+%283CLpro%29+inhibitors+from+2010+to+2020&doi=10.1016%2Fj.ejmech.2020.112711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The development of Coronavirus 3C-Like protease (3CLpro) inhibitors from 2010 to 2020</span></div><div class="casAuthors">Liu, Yuzhi; Liang, Chengyuan; Xin, Liang; Ren, Xiaodong; Tian, Lei; Ju, Xingke; Li, Han; Wang, Yongbo; Zhao, Qianqian; Liu, Hong; Cao, Wenqiang; Xie, Xiaolin; Zhang, Dezhu; Wang, Yu; Jian, Yanlin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">206</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112711</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  This review fully describes the coronavirus 3CLpro peptidomimetic inhibitors and nonpeptidic small mol. inhibitors developed from 2010 to 2020.  Specifically, the structural characteristics, binding modes and SARs of these 3CLpro inhibitors are expounded in detail by division into two categories: peptidomimetic inhibitors mainly utilize electrophilic warhead groups to covalently bind the 3CLpro Cys145 residue and thereby achieve irreversible inhibition effects, whereas nonpeptidic small mol. inhibitors mainly interact with residues in the S1', S1, S2 and S4 pockets via hydrogen bonds, hydrophobic bonds and van der Waals forces.  Based on the emerging PROTAC technol. and the existing 3CLpro inhibitors, 3CLpro PROTAC degraders are hypothesised to be next-generation anti-coronavirus drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi77_VV-QrrrVg90H21EOLACvtfcHk0lg2YSv9TiDZ2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1CqtLjI&md5=32ccbb34674ba26c24746117bd689284</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112711%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DXin%26aufirst%3DL.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DJu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DCao%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJian%26aufirst%3DY.%26atitle%3DThe%2520development%2520of%2520Coronavirus%25203C-Like%2520protease%2520%25283CLpro%2529%2520inhibitors%2520from%25202010%2520to%25202020%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D206%26spage%3D112711%26doi%3D10.1016%2Fj.ejmech.2020.112711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span> <span> </span><span class="NLM_article-title">Cyanohydrin as an Anchoring Group for Potent and Selective Inhibitors of Enterovirus 71 3C Protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9414</span>– <span class="NLM_lpage">9420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9414-9420&author=Y.+Zhaiauthor=X.+Zhaoauthor=Z.+Cuiauthor=M.+Wangauthor=Y.+Wangauthor=L.+Liauthor=Q.+Sunauthor=X.+Yangauthor=D.+Zengauthor=Y.+Liuauthor=Y.+Sunauthor=Z.+Louauthor=L.+Shangauthor=Z.+Yin&title=Cyanohydrin+as+an+Anchoring+Group+for+Potent+and+Selective+Inhibitors+of+Enterovirus+71+3C+Protease&doi=10.1021%2Facs.jmedchem.5b01013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Cyanohydrin as an Anchoring Group for Potent and Selective Inhibitors of Enterovirus 71 3C Protease</span></div><div class="casAuthors">Zhai, Yangyang; Zhao, Xiangshuai; Cui, Zhengjie; Wang, Man; Wang, Yaxin; Li, Linfeng; Sun, Qi; Yang, Xi; Zeng, Debin; Liu, Ying; Sun, Yuna; Lou, Zhiyong; Shang, Luqing; Yin, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9414-9420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyanohydrin derivs. as enterovirus 71 (EV71) 3C protease (3Cpro) inhibitors have been synthesized and assayed for their biochem. and antiviral activities.  Compared with the reported inhibitors, cyanohydrins (1S,2S,2'S,5S)-I and (1R,2S,2'S,5S)-I exhibited significantly improved activity and attractive selectivity profiles against other proteases, which were a result of the specific interactions between the cyanohydrin moiety and the catalytic site of 3Cpro.  Cyanohydrin as an anchoring group with high selectivity and excellent inhibitory activity represents a useful choice for cysteine protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxejr9UxByVLVg90H21EOLACvtfcHk0lh8-_gCcAhbrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FK&md5=e43c6b3002e7c61cf8419bc83782be79</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01013%26sid%3Dliteratum%253Aachs%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DCui%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DShang%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DZ.%26atitle%3DCyanohydrin%2520as%2520an%2520Anchoring%2520Group%2520for%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Enterovirus%252071%25203C%2520Protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9414%26epage%3D9420%26doi%3D10.1021%2Facs.jmedchem.5b01013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozminski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. J.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planken, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12725</span>– <span class="NLM_lpage">12747</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12725-12747&author=R.+L.+Hoffmanauthor=R.+S.+Kaniaauthor=M.+A.+Brothersauthor=F.+F.+Daviesauthor=R.+A.+Ferreauthor=K.+S.+Gajiwalaauthor=M.+Heauthor=R.+J.+Hoganauthor=K.+Kozminskiauthor=L.+Y.+Liauthor=J.+W.+Locknerauthor=J.+Louauthor=M.+T.+Marraauthor=L.+J.+Mitchellauthor=B.+W.+Murrayauthor=J.+A.+Niemanauthor=S.+Noellauthor=S.+P.+Plankenauthor=T.+Roweauthor=K.+Ryanauthor=G.+J.+Smithauthor=J.+E.+Solowiejauthor=C.+M.+Steppanauthor=B.+Taggart&title=Discovery+of+Ketone-Based+Covalent+Inhibitors+of+Coronavirus+3CL+Proteases+for+the+Potential+Therapeutic+Treatment+of+COVID-19&doi=10.1021%2Facs.jmedchem.0c01063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span></div><div class="casAuthors">Hoffman, Robert L.; Kania, Robert S.; Brothers, Mary A.; Davies, Jay F.; Ferre, Rose A.; Gajiwala, Ketan S.; He, Mingying; Hogan, Robert J.; Kozminski, Kirk; Li, Lilian Y.; Lockner, Jonathan W.; Lou, Jihong; Marra, Michelle T.; Mitchell Jr., Lennert J.; Murray, Brion W.; Nieman, James A.; Noell, Stephen; Planken, Simon P.; Rowe, Thomas; Ryan, Kevin; Smith III, George J.; Solowiej, James E.; Steppan, Claire M.; Taggart, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12725-12747</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002.  Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is crit. for coronavirus replication.  The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage.  A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallog. led to the identification of 3 and 4.  Preclin. expts. reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an i.v. treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_zn3ZDefOrVg90H21EOLACvtfcHk0lh8-_gCcAhbrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E&md5=7d5be169ad3de528496c862934d59881</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01063%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DDavies%26aufirst%3DF.%2BF.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHogan%26aufirst%3DR.%2BJ.%26aulast%3DKozminski%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLockner%26aufirst%3DJ.%2BW.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMarra%26aufirst%3DM.%2BT.%26aulast%3DMitchell%26aufirst%3DL.%2BJ.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DPlanken%26aufirst%3DS.%2BP.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DSolowiej%26aufirst%3DJ.%2BE.%26aulast%3DSteppan%26aufirst%3DC.%2BM.%26aulast%3DTaggart%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520Ketone-Based%2520Covalent%2520Inhibitors%2520of%2520Coronavirus%25203CL%2520Proteases%2520for%2520the%2520Potential%2520Therapeutic%2520Treatment%2520of%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12725%26epage%3D12747%26doi%3D10.1021%2Facs.jmedchem.0c01063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halford, B.</span></span> <span> </span><span class="NLM_article-title">Pfizer’s novel COVID-19 antiviral heads to clinical trials</span>. <i>C&EN</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">37</span>),  <span class="NLM_fpage">9</span>, <span class="refDoi"> DOI: 10.47287/cen-09837-buscon2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.47287%2Fcen-09837-buscon2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2020&pages=9&issue=37&author=B.+Halford&title=Pfizer%E2%80%99s+novel+COVID-19+antiviral+heads+to+clinical+trials&doi=10.47287%2Fcen-09837-buscon2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.47287%2Fcen-09837-buscon2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.47287%252Fcen-09837-buscon2%26sid%3Dliteratum%253Aachs%26aulast%3DHalford%26aufirst%3DB.%26atitle%3DPfizer%25E2%2580%2599s%2520novel%2520COVID-19%2520antiviral%2520heads%2520to%2520clinical%2520trials%26jtitle%3DC%2526EN%26date%3D2020%26volume%3D98%26issue%3D37%26spage%3D9%26doi%3D10.47287%2Fcen-09837-buscon2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ieda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuyama, T.</span></span> <span> </span><span class="NLM_article-title">Synthesis of the optically active key intermediate of FR901483</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4027</span>– <span class="NLM_lpage">4029</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2010.05.089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.tetlet.2010.05.089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVGks7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=4027-4029&author=S.+Iedaauthor=T.+Kanauthor=T.+Fukuyama&title=Synthesis+of+the+optically+active+key+intermediate+of+FR901483&doi=10.1016%2Fj.tetlet.2010.05.089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of the optically active key intermediate of FR901483</span></div><div class="casAuthors">Ieda, Shigeru; Kan, Toshiyuki; Fukuyama, Tohru</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">4027-4029</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Efficient synthesis of the tricyclic key intermediate I for (-)-FR 901483 was accomplished.  The precursor II of the intramol. aldol reaction was constructed by the Ugi 4CC reaction and subsequent intramol. Dieckmann condensation.  This approach allows a fully stereocontrolled total synthesis of (-)-FR 901483, which would provide various derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ-SssaT8bIbVg90H21EOLACvtfcHk0lh8-_gCcAhbrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVGks7g%253D&md5=2267d7eab3f62ddc3bd55a6f5fa24fd4</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2010.05.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2010.05.089%26sid%3Dliteratum%253Aachs%26aulast%3DIeda%26aufirst%3DS.%26aulast%3DKan%26aufirst%3DT.%26aulast%3DFukuyama%26aufirst%3DT.%26atitle%3DSynthesis%2520of%2520the%2520optically%2520active%2520key%2520intermediate%2520of%2520FR901483%26jtitle%3DTetrahedron%2520Lett.%26date%3D2010%26volume%3D51%26spage%3D4027%26epage%3D4029%26doi%3D10.1016%2Fj.tetlet.2010.05.089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span> <span> </span><span class="NLM_article-title">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">4282</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18096-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fs41467-020-18096-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32855413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4282&issue=1&author=W.+Vuongauthor=M.+B.+Khanauthor=C.+Fischerauthor=E.+Arutyunovaauthor=T.+Lamerauthor=J.+Shieldsauthor=H.+A.+Saffranauthor=R.+T.+McKayauthor=M.+J.+van+Belkumauthor=M.+A.+Joyceauthor=H.+S.+Youngauthor=D.+L.+Tyrrellauthor=J.+C.+Vederasauthor=M.+J.+Lemieux&title=Feline+coronavirus+drug+inhibits+the+main+protease+of+SARS-CoV-2+and+blocks+virus+replication&doi=10.1038%2Fs41467-020-18096-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span></div><div class="casAuthors">Vuong, Wayne; Khan, Muhammad Bashir; Fischer, Conrad; Arutyunova, Elena; Lamer, Tess; Shields, Justin; Saffran, Holly A.; McKay, Ryan T.; van Belkum, Marco J.; Joyce, Michael A.; Young, Howard S.; Tyrrell, D. Lorne; Vederas, John C.; Lemieux, M. Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4282</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The main protease, Mpro (or 3CLpro) in SARS-CoV-2 is a viable drug target because of its essential role in the cleavage of the virus polypeptide.  Feline infectious peritonitis, a fatal coronavirus infection in cats, was successfully treated previously with a prodrug GC376, a dipeptide-based protease inhibitor.  We show the prodrug and its parent GC373, are effective inhibitors of the Mpro from both SARS-CoV and SARS-CoV-2 with IC50 values in the nanomolar range.  Crystal structures of SARS-CoV-2 Mpro with these inhibitors have a covalent modification of the nucleophilic Cys145.  NMR anal. reveals that inhibition proceeds via reversible formation of a hemithioacetal.  GC373 and GC376 are potent inhibitors of SARS-CoV-2 replication in cell culture.  They are strong drug candidates for the treatment of human coronavirus infections because they have already been successful in animals.  The work here lays the framework for their use in human trials for the treatment of COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMA7W4XtRQrbVg90H21EOLACvtfcHk0ljSjC-9Mt0CWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN&md5=c06d2dc9a65ae289505fe120d41d8885</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18096-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18096-2%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DW.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DShields%26aufirst%3DJ.%26aulast%3DSaffran%26aufirst%3DH.%2BA.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DJoyce%26aufirst%3DM.%2BA.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26atitle%3DFeline%2520coronavirus%2520drug%2520inhibits%2520the%2520main%2520protease%2520of%2520SARS-CoV-2%2520and%2520blocks%2520virus%2520replication%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26issue%3D1%26spage%3D4282%26doi%3D10.1038%2Fs41467-020-18096-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balbo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telliez, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, A. J.</span></span> <span> </span><span class="NLM_article-title">The advantages of describing covalent inhibitor in vitro potencies by IC<sub>50</sub> at a fixed time point: IC<sub>50</sub> determination of covalent inhibitors provides meaningful data to medicinal chemistry for SAR optimization</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">115865</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2020.115865</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.bmc.2020.115865" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33285410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFahsrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2021&pages=115865&author=A.+Thorarensenauthor=P.+Balboauthor=M.+E.+Bankerauthor=R.+M.+Czerwinskiauthor=M.+Kuhnauthor=T.+S.+Maurerauthor=J.-B.+Telliezauthor=F.+Vincentauthor=A.+J.+Wittwer&title=The+advantages+of+describing+covalent+inhibitor+in+vitro+potencies+by+IC50+at+a+fixed+time+point%3A+IC50+determination+of+covalent+inhibitors+provides+meaningful+data+to+medicinal+chemistry+for+SAR+optimization&doi=10.1016%2Fj.bmc.2020.115865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The advantages of describing covalent inhibitor in vitro potencies by IC50 at a fixed time point. IC50 determination of covalent inhibitors provides meaningful data to medicinal chemistry for SAR optimization</span></div><div class="casAuthors">Thorarensen, Atli; Balbo, Paul; Banker, Mary E.; Czerwinski, Robert M.; Kuhn, Max; Maurer, Tristan S.; Telliez, Jean-Baptiste; Vincent, Fabien; Wittwer, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">115865</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recent years have seen a resurgence in drug discovery efforts aimed at the identification of covalent inhibitors which has led to an explosion of literature reports in this area and most importantly new approved therapies.  These reports and breakthroughs highlight the significant investments made across the industry in SAR campaigns to optimize inhibitors.  The potency of covalent inhibitors is generally considered to be more accurately described by the time-independent kinetic parameter kinact/Ki rather than a by a simple IC50 since the latter is a time-dependent parameter.  Enzyme substrate concns. are an addnl. important factor to consider when attempting to translate parameters derived from enzymol. expts. to phenotypic behavior in a physiol. relevant cell-based system.  Theor. and exptl. investigations into the relationship between IC50, time, substrate concn. and Kinact/Ki provided us with an effective approach to provide meaningful data for SAR optimization.  The data we generated for our JAK3 irreversible covalent inhibitor program using IC50 values provided by enzyme assays with long incubations (>1h) coupled with physiol. substrate concn. provided the medicinal chemist with optimal information in a rapid and efficient manner.  We further document the wide applicability of this method by applying it to other enzymes systems where we have run covalent inhibitor programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_kQEA2V2QErVg90H21EOLACvtfcHk0ljSjC-9Mt0CWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFahsrbO&md5=e981d4fab453add5bcfea1dadf8168e4</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2020.115865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2020.115865%26sid%3Dliteratum%253Aachs%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DBalbo%26aufirst%3DP.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DCzerwinski%26aufirst%3DR.%2BM.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DMaurer%26aufirst%3DT.%2BS.%26aulast%3DTelliez%26aufirst%3DJ.-B.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DWittwer%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520advantages%2520of%2520describing%2520covalent%2520inhibitor%2520in%2520vitro%2520potencies%2520by%2520IC50%2520at%2520a%2520fixed%2520time%2520point%253A%2520IC50%2520determination%2520of%2520covalent%2520inhibitors%2520provides%2520meaningful%2520data%2520to%2520medicinal%2520chemistry%2520for%2520SAR%2520optimization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2021%26volume%3D29%26spage%3D115865%26doi%3D10.1016%2Fj.bmc.2020.115865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willoughby, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span> <span> </span><span class="NLM_article-title">Improved SARS-CoV-2 Mpro inhibitors based on feline coronavirus drug GC376: structure-activity studies and solubilization for enhanced bioavailability</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>222</i></span>,  <span class="NLM_fpage">113584</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2021.113584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.ejmech.2021.113584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=34118724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXht1yrt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=222&publication_year=2021&pages=113584&author=W.+Vuongauthor=C.+Fischerauthor=M.+B.+Khanauthor=M.+J.+van+Belkumauthor=T.+Lamerauthor=K.+D.+Willoughbyauthor=J.+Luauthor=E.+Arutyunovaauthor=M.+A.+Joyceauthor=H.+A.+Saffranauthor=J.+A.+Shieldsauthor=H.+S.+Youngauthor=J.+A.+Niemanauthor=D.+L.+Tyrrellauthor=M.+J.+Lemieuxauthor=J.+C.+Vederas&title=Improved+SARS-CoV-2+Mpro+inhibitors+based+on+feline+coronavirus+drug+GC376%3A+structure-activity+studies+and+solubilization+for+enhanced+bioavailability&doi=10.1016%2Fj.ejmech.2021.113584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44aR"><div class="casContent"><span class="casTitleNuber">44a</span><div class="casTitle"><span class="NLM_cas:atitle">Improved SARS-CoV-2 Mpro inhibitors based on feline antiviral drug GC376: Structural enhancements, increased solubility, and micellar studies</span></div><div class="casAuthors">Vuong, Wayne; Fischer, Conrad; Khan, Muhammad Bashir; van Belkum, Marco J.; Lamer, Tess; Willoughby, Kurtis D.; Lu, Jimmy; Arutyunova, Elena; Joyce, Michael A.; Saffran, Holly A.; Shields, Justin A.; Young, Howard S.; Nieman, James A.; Tyrrell, D. Lorne; Lemieux, M. Joanne; Vederas, John C.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">222</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113584</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Replication of SARS-CoV-2, the coronavirus causing COVID-19, requires a main protease (Mpro) to cleave viral proteins.  Consequently, Mpro is a target for antiviral agents.  We and others previously demonstrated that GC376, a bisulfite prodrug with efficacy as an anti-coronaviral agent in animals, is an effective inhibitor of Mpro in SARS-CoV-2.  We report structure-activity studies of improved GC376 derivs. with nanomolar affinities and therapeutic indexes >200.  Crystallog. structures of inhibitor-Mpro complexes reveal that an alternative binding pocket in Mpro, S4, accommodates the P3 position.  Alternative binding is induced by polar P3 groups or a nearby Me.  NMR and soly. studies with GC376 show that it exists as a mixt. of stereoisomers and forms colloids in aq. media at higher concns., a property not previously reported.  Replacement of its Na+ counter ion with choline greatly increases soly.  The phys., biochem., crystallog., and cellular data reveal new avenues for Mpro inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj9Iq0rzrDdrVg90H21EOLACvtfcHk0lj3nuRXRMK9vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXht1yrt73I&md5=aa17ed00bc373ef06e402677d9a523dc</span></div><a href="/servlet/linkout?suffix=cit44a&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2021.113584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2021.113584%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DW.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DWilloughby%26aufirst%3DK.%2BD.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DJoyce%26aufirst%3DM.%2BA.%26aulast%3DSaffran%26aufirst%3DH.%2BA.%26aulast%3DShields%26aufirst%3DJ.%2BA.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26atitle%3DImproved%2520SARS-CoV-2%2520Mpro%2520inhibitors%2520based%2520on%2520feline%2520coronavirus%2520drug%2520GC376%253A%2520structure-activity%2520studies%2520and%2520solubilization%2520for%2520enhanced%2520bioavailability%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2021%26volume%3D222%26spage%3D113584%26doi%3D10.1016%2Fj.ejmech.2021.113584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit44b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span> <span> </span><span class="NLM_article-title">N-Terminal finger stabilizes the S1 Pocket for the reversible feline drug GC376 in SARS-CoV-2 M<sup>pro</sup> Dimer</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>433</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">167003</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2021.167003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1016%2Fj.jmb.2021.167003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33895266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXpvFartbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=433&publication_year=2021&pages=167003&issue=13&author=E.+Arutyunovaauthor=M.+B.+Khanauthor=C.+Fischerauthor=J.+Luauthor=T.+Lamerauthor=W.+Vuongauthor=M.+J.+van+Belkumauthor=R.+T.+McKayauthor=D.+L.+Tyrrellauthor=J.+C.+Vederasauthor=H.+S.+Youngauthor=M.+J.+Lemieux&title=N-Terminal+finger+stabilizes+the+S1+Pocket+for+the+reversible+feline+drug+GC376+in+SARS-CoV-2+Mpro+Dimer&doi=10.1016%2Fj.jmb.2021.167003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44bR"><div class="casContent"><span class="casTitleNuber">44b</span><div class="casTitle"><span class="NLM_cas:atitle">N-terminal finger stabilizes the S1 pocket for the reversible feline drug GC376 in the SARS-CoV-2 Mpro dimer</span></div><div class="casAuthors">Arutyunova, Elena; Khan, Muhammad Bashir; Fischer, Conrad; Lu, Jimmy; Lamer, Tess; Vuong, Wayne; van Belkum, Marco J.; McKay, Ryan T.; Tyrrell, D. Lorne; Vederas, John C.; Young, Howard S.; Lemieux, M. Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">433</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">167003</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The main protease (Mpro, also known as 3CL protease) of SARS-CoV-2 is a high priority drug target in the development of antivirals to combat COVID-19 infections.  A feline coronavirus antiviral drug, GC376, has been shown to be effective in inhibiting the SARS-CoV-2 main protease and live virus growth.  As this drug moves into clin. trials, further characterization of GC376 with the main protease of coronaviruses is required to gain insight into the drug's properties, such as reversibility and broad specificity.  Reversibility is an important factor for therapeutic proteolytic inhibitors to prevent toxicity due to off-target effects.  Here we demonstrate that GC376 has nanomolar Ki values with the Mpro from both SARS-CoV-2 and SARS-CoV strains.  Restoring enzymic activity after inhibition by GC376 demonstrates reversible binding with both proteases.  In addn., the stability and thermodn. parameters of both proteases were studied to shed light on phys. chem. properties of these viral enzymes, revealing higher stability for SARS-CoV-2 Mpro.  The comparison of a new X-ray crystal structure of Mpro from SARS-CoV complexed with GC376 reveals similar mol. mechanism of inhibition compared to SARS-CoV-2 Mpro, and gives insight into the broad specificity properties of this drug.  In both structures, we observe domain swapping of the N-termini in the dimer of the Mpro, which facilitates coordination of the drug's P1 position.  These results validate that GC376 is a drug with an off-rate suitable for clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMfiKgDeiJRrVg90H21EOLACvtfcHk0lj3nuRXRMK9vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXpvFartbg%253D&md5=fa586484a14c510731bba968126c6bf6</span></div><a href="/servlet/linkout?suffix=cit44b&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2021.167003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2021.167003%26sid%3Dliteratum%253Aachs%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DVuong%26aufirst%3DW.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26atitle%3DN-Terminal%2520finger%2520stabilizes%2520the%2520S1%2520Pocket%2520for%2520the%2520reversible%2520feline%2520drug%2520GC376%2520in%2520SARS-CoV-2%2520Mpro%2520Dimer%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2021%26volume%3D433%26issue%3D13%26spage%3D167003%26doi%3D10.1016%2Fj.jmb.2021.167003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szeto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarbet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1038/s41422-020-0356-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1038%2Fs41422-020-0356-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32541865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1WmsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=678-692&author=C.+Maauthor=M.+D.+Saccoauthor=B.+Hurstauthor=J.+A.+Townsendauthor=Y.+Huauthor=T.+Szetoauthor=X.+Zhangauthor=B.+Tarbetauthor=M.+T.+Martyauthor=Y.+Chenauthor=J.+Wang&title=Boceprevir%2C+GC-376%2C+and+calpain+inhibitors+II%2C+XII+inhibit+SARS-CoV-2+viral+replication+by+targeting+the+viral+main+protease&doi=10.1038%2Fs41422-020-0356-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease</span></div><div class="casAuthors">Ma, Chunlong; Sacco, Michael Dominic; Hurst, Brett; Townsend, Julia Alma; Hu, Yanmei; Szeto, Tommy; Zhang, Xiujun; Tarbet, Bart; Marty, Michael Thomas; Chen, Yu; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">678-692</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A new coronavirus SARS-CoV-2, also called novel coronavirus 2019 (2019-nCoV), started to circulate among humans around Dec. 2019, and it is now widespread as a global pandemic.  The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.35% as of May 26, 2020.  There is no vaccine or antiviral available for SARS-CoV-2.  We report our discovery of inhibitors targeting the SARS-CoV-2 main protease (Mpro).  Using the FRET-based enzymic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II and XII were identified to have potent activity with single-digit to submicromolar IC50 values in the enzymic assay.  The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry.  Significantly, 4 compds. (boceprevir, GC-376, and calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC50 values ranging 0.49-3.37μM.  Notably, boceprevir and calpain inhibitors II and XII represent novel chemotypes that are distinct from known substrate-based peptidomimetic Mpro inhibitors.  A complex crystal structure of SARS-CoV-2 Mpro with GC-376, detd. at 2.15 Å resoln. with 3 protomers per asym. unit, revealed 2 unique binding configurations, shedding light on the mol. interactions and protein conformational flexibility underlying substrate and inhibitor binding by Mpro.  Overall, the compds. identified provide promising starting points for the further development of SARS-CoV-2 therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVnFC-t0uFW7Vg90H21EOLACvtfcHk0lj3nuRXRMK9vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1WmsL%252FL&md5=225d518c313474cdfbcfe678d35e26a4</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41422-020-0356-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-020-0356-z%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DSacco%26aufirst%3DM.%2BD.%26aulast%3DHurst%26aufirst%3DB.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DSzeto%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DTarbet%26aufirst%3DB.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DBoceprevir%252C%2520GC-376%252C%2520and%2520calpain%2520inhibitors%2520II%252C%2520XII%2520inhibit%2520SARS-CoV-2%2520viral%2520replication%2520by%2520targeting%2520the%2520viral%2520main%2520protease%26jtitle%3DCell%2520Res.%26date%3D2020%26volume%3D30%26spage%3D678%26epage%3D692%26doi%3D10.1038%2Fs41422-020-0356-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mengist, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mekonnen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, T.</span></span> <span> </span><span class="NLM_article-title">Potency, Safety, and Pharmacokinetic Profiles of Potential Inhibitors Targeting SARS-CoV-2 Main Protease</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">630500</span>, <span class="refDoi"> DOI: 10.3389/fphar.2020.630500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.3389%2Ffphar.2020.630500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=33597888" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=630500&author=H.+M.+Mengistauthor=D.+Mekonnenauthor=A.+Mohammedauthor=R.+Shiauthor=T.+Jin&title=Potency%2C+Safety%2C+and+Pharmacokinetic+Profiles+of+Potential+Inhibitors+Targeting+SARS-CoV-2+Main+Protease&doi=10.3389%2Ffphar.2020.630500"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2020.630500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2020.630500%26sid%3Dliteratum%253Aachs%26aulast%3DMengist%26aufirst%3DH.%2BM.%26aulast%3DMekonnen%26aufirst%3DD.%26aulast%3DMohammed%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DT.%26atitle%3DPotency%252C%2520Safety%252C%2520and%2520Pharmacokinetic%2520Profiles%2520of%2520Potential%2520Inhibitors%2520Targeting%2520SARS-CoV-2%2520Main%2520Protease%26jtitle%3DFront.%2520Pharmacol.%26date%3D2021%26volume%3D11%26spage%3D630500%26doi%3D10.3389%2Ffphar.2020.630500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahr, A.</span></span> <span> </span><span class="NLM_article-title">Lipophilicity and Its Relationship with Passive Drug Permeation</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">962</span>– <span class="NLM_lpage">977</span>, <span class="refDoi"> DOI: 10.1007/s11095-010-0303-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1007%2Fs11095-010-0303-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=21052797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlOqtrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2011&pages=962-977&author=X.+Liuauthor=B.+Testaauthor=A.+Fahr&title=Lipophilicity+and+Its+Relationship+with+Passive+Drug+Permeation&doi=10.1007%2Fs11095-010-0303-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47a</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilicity and Its Relationship with Passive Drug Permeation</span></div><div class="casAuthors">Liu, Xiangli; Testa, Bernard; Fahr, Alfred</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">962-977</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">In this review, we first summarize the structure and properties of biol. membranes and the routes of passive drug transfer through physiol. barriers.  Lipophilicity is then introduced in terms of the intermol. interactions it encodes.  Finally, lipophilicity indexes from isotropic solvent systems and from anisotropic membrane-like systems are discussed for their capacity to predict passive drug permeation across biol. membranes such as the intestinal epithelium, the blood-brain barrier (BBB) or the skin.  The broad evidence presented here shows that beyond the predictive power of lipophilicity parameters, the various intermol. forces they encode allow a mechanistic interpretation of passive drug permeation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxn2Ic1yX8SbVg90H21EOLACvtfcHk0lh24RgqaOshiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlOqtrzK&md5=6bd2bb54a14574f31b046204f260595a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs11095-010-0303-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-010-0303-7%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DFahr%26aufirst%3DA.%26atitle%3DLipophilicity%2520and%2520Its%2520Relationship%2520with%2520Passive%2520Drug%2520Permeation%26jtitle%3DPharm.%2520Res.%26date%3D2011%26volume%3D28%26spage%3D962%26epage%3D977%26doi%3D10.1007%2Fs11095-010-0303-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit47b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinner, M. J.</span></span> <span> </span><span class="NLM_article-title">Getting Across the Cell Membrane: An Overview for Small Molecules, Peptides, and Proteins</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1266</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-2272-7_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1007%2F978-1-4939-2272-7_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=25560066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1Siur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1266&publication_year=2015&pages=29-53&author=N.+J.+Yangauthor=M.+J.+Hinner&title=Getting+Across+the+Cell+Membrane%3A+An+Overview+for+Small+Molecules%2C+Peptides%2C+and+Proteins&doi=10.1007%2F978-1-4939-2272-7_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47bR"><div class="casContent"><span class="casTitleNuber">47b</span><div class="casTitle"><span class="NLM_cas:atitle">Getting Across the Cell Membrane: An Overview for Small Molecules, Peptides, and Proteins</span></div><div class="casAuthors">Yang, Nicole J.; Hinner, Marlon J.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1266</span>
        (<span class="NLM_cas:issue">Site-Specific Protein Labeling</span>),
    <span class="NLM_cas:pages">29-53</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The ability to efficiently access cytosolic proteins is desired in both biol. research and medicine.  However, targeting intracellular proteins is often challenging, because to reach the cytosol, exogenous mols. must first traverse the cell membrane.  This review provides a broad overview of how certain mols. are thought to cross this barrier, and what kinds of approaches are being made to enhance the intracellular delivery of those that are impermeable.  We first discuss rules that govern the passive permeability of small mols. across the lipid membrane, and mechanisms of membrane transport that have evolved in nature for certain metabolites, peptides, and proteins.  Then, we introduce design strategies that have emerged in the development of small mols. and peptides with improved permeability.  Finally, intracellular delivery systems that have been engineered for protein payloads are surveyed.  Viewpoints from varying disciplines have been brought together to provide a cohesive overview of how the membrane barrier is being overcome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptS-9wM3Z8jbVg90H21EOLACvtfcHk0lh24RgqaOshiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1Siur4%253D&md5=4002e2e8ab2fcc86d9ea400e1aa2a29b</span></div><a href="/servlet/linkout?suffix=cit47b&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2272-7_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-2272-7_3%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DN.%2BJ.%26aulast%3DHinner%26aufirst%3DM.%2BJ.%26atitle%3DGetting%2520Across%2520the%2520Cell%2520Membrane%253A%2520An%2520Overview%2520for%2520Small%2520Molecules%252C%2520Peptides%252C%2520and%2520Proteins%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1266%26spage%3D29%26epage%3D53%26doi%3D10.1007%2F978-1-4939-2272-7_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yukawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naven, R.</span></span> <span> </span><span class="NLM_article-title">Utility of Physicochemical Properties for the Prediction of Toxicological Outcomes: Takeda Perspective</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00536</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00536" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFKqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=203-209&issue=2&author=T.+Yukawaauthor=R.+Naven&title=Utility+of+Physicochemical+Properties+for+the+Prediction+of+Toxicological+Outcomes%3A+Takeda+Perspective&doi=10.1021%2Facsmedchemlett.9b00536"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48aR"><div class="casContent"><span class="casTitleNuber">48a</span><div class="casTitle"><span class="NLM_cas:atitle">Utility of Physicochemical Properties for the Prediction of Toxicological Outcomes: Takeda Perspective</span></div><div class="casAuthors">Yukawa, Tomoya; Naven, Russell</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">203-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The role that physicochem. properties play toward increasing the likelihood of toxicity findings in in vivo studies has been well reported, albeit sometimes with different conclusions.  We decided to understand the role that physicochem. properties play toward the prediction of in vivo toxicol. outcomes for Takeda chem. using 284 internal compds.  In support of the previously reported "3/75 rule", reducing lipophilicity of mols. decreases toxicity odds noticeably; however, we also found that the trend of toxicity odds is different between compds. classified by their ionization state.  For basic mols., the odds of in vivo toxicity outcomes were significantly impacted by both lipophilicity and polar surface area, whereas neutral mols. were impacted less so.  Through an anal. of several project-related compds., we herein demonstrate that the utilization of the 3/75 rule coupled with consideration of ionization state is a rational strategy for medicinal chem. design of safer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOaavww-aqYLVg90H21EOLACvtfcHk0lh24RgqaOshiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFKqtrY%253D&md5=7d1f2112b22fcf45f59aef6b3658047d</span></div><a href="/servlet/linkout?suffix=cit48a&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00536&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00536%26sid%3Dliteratum%253Aachs%26aulast%3DYukawa%26aufirst%3DT.%26aulast%3DNaven%26aufirst%3DR.%26atitle%3DUtility%2520of%2520Physicochemical%2520Properties%2520for%2520the%2520Prediction%2520of%2520Toxicological%2520Outcomes%253A%2520Takeda%2520Perspective%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26issue%3D2%26spage%3D203%26epage%3D209%26doi%3D10.1021%2Facsmedchemlett.9b00536" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit48b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kiani, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jabeen, I.</span></span> <span> </span><span class="NLM_article-title">Lipophilic Metabolic Efficiency (LipMetE) and Drug Efficiency Indices to Explore the Metabolic Properties of the Substrates of Selected Cytochrome P450 Isoforms</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1021/acsomega.9b02344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.9b02344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisV2rtb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=179-188&author=Y.+S.+Kianiauthor=I.+Jabeen&title=Lipophilic+Metabolic+Efficiency+%28LipMetE%29+and+Drug+Efficiency+Indices+to+Explore+the+Metabolic+Properties+of+the+Substrates+of+Selected+Cytochrome+P450+Isoforms&doi=10.1021%2Facsomega.9b02344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48bR"><div class="casContent"><span class="casTitleNuber">48b</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Metabolic Efficiency (LipMetE) and Drug Efficiency Indices to Explore the Metabolic Properties of the Substrates of Selected Cytochrome P450 Isoforms</span></div><div class="casAuthors">Kiani, Yusra Sajid; Jabeen, Ishrat</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">179-188</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cytochrome P 450 (CYP450) enzymes belong to a superfamily of heme-contg. proteins that are involved in the metab. of structurally diverse endogenous and exogenous compds.  Various proof-of-concept studies indicate that metabolic stability and intrinsic clearance of CYP450 substrates are linked with the resp. lipophilicity (log P or log D).  This necessitates the normalization of lipophilicity (log P or log D) of a given CYP450 substrate with respect to its metabolic stability (LipMetE) and intrinsic clearance (log10CLint,u).  Therefore, in this article, the LipMetE values of already known substrates of CYP1A2, CYP2C9, CYP2C19, CYP2D6, and CYP3A4, including some marketed drugs, have been calcd. to elucidate the relationship between lipophilicity (log D7.4) and in vitro clearance.  Moreover, various drug efficiency metrics including lipophilic efficiency (LipE) and ligand efficiency (LE) have been evaluated, and the thresholds of these parameters have been defined for the CYP450 substrates exhibiting normalized LipMetE.  Our results indicate that for a given range of LipMetE, greater the log D value of the substrate the more avidly it binds to a given CYP450 enzyme and shows more intrinsic clearance (log10CLint,u).  Overall, the majority of the model substrates of CYP450 isoforms and already marketed drugs in our datasets exhibit log D7.4 values of ∼2.5 with LipMetE values in the range of 0-2.5 and LipE values of ≤3.  Overall, consideration of these parameters in ADME profiling could aid in reducing the drug failure rate in the later stages of clin. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgrXqwi9iUTbVg90H21EOLACvtfcHk0ljnJ94uD6O5EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisV2rtb%252FK&md5=04b499c771265b528a7f0ebf7577ab03</span></div><a href="/servlet/linkout?suffix=cit48b&amp;dbid=16384&amp;doi=10.1021%2Facsomega.9b02344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.9b02344%26sid%3Dliteratum%253Aachs%26aulast%3DKiani%26aufirst%3DY.%2BS.%26aulast%3DJabeen%26aufirst%3DI.%26atitle%3DLipophilic%2520Metabolic%2520Efficiency%2520%2528LipMetE%2529%2520and%2520Drug%2520Efficiency%2520Indices%2520to%2520Explore%2520the%2520Metabolic%2520Properties%2520of%2520the%2520Substrates%2520of%2520Selected%2520Cytochrome%2520P450%2520Isoforms%26jtitle%3DACS%2520Omega%26date%3D2020%26volume%3D5%26spage%3D179%26epage%3D188%26doi%3D10.1021%2Facsomega.9b02344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit48c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tinworth, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Facts, Patterns and Principles in Drug Discovery: Appraising the Rule of 5 with Measured Physicochemical Data</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">10091</span>– <span class="NLM_lpage">10108</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01596</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01596" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslShtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=10091-10108&author=C.+P.+Tinworthauthor=R.+J.+Young&title=Facts%2C+Patterns+and+Principles+in+Drug+Discovery%3A+Appraising+the+Rule+of+5+with+Measured+Physicochemical+Data&doi=10.1021%2Facs.jmedchem.9b01596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48cR"><div class="casContent"><span class="casTitleNuber">48c</span><div class="casTitle"><span class="NLM_cas:atitle">Facts, Patterns, and Principles in Drug Discovery: Appraising the Rule of 5 with Measured Physicochemical Data</span></div><div class="casAuthors">Tinworth, Christopher P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10091-10108</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The rule of 5 was designed to est. the likelihood of poor absorption or permeation, noting the impact of poor soly.  This Perspective explores the impact of various physicochem. descriptors and contemporary lipophilicity measurements on permeability and soly., showing that the distribution coeff. log D7.4 (rather than log P) is the most impactful parameter.  Mol. wt., almost invariably the defining characteristic of "beyond the rule of 5" compds., has little impact on soly. when log D7.4 measurements and aromaticity are considered.  Predicting permeation is more complex, given passive and carrier transport mechanisms; however, notable patterns of behavior are apparent, giving insight even "beyond the rule of 5".  Recommended best practices should involve using the facts (measurements) and the patterns they reveal to establish informative principles rather than fastidious rules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBT9kzvTWdTbVg90H21EOLACvtfcHk0ljnJ94uD6O5EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslShtb4%253D&md5=c93bbcd944dcb41c0d38d2a3881b8593</span></div><a href="/servlet/linkout?suffix=cit48c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01596%26sid%3Dliteratum%253Aachs%26aulast%3DTinworth%26aufirst%3DC.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DFacts%252C%2520Patterns%2520and%2520Principles%2520in%2520Drug%2520Discovery%253A%2520Appraising%2520the%2520Rule%2520of%25205%2520with%2520Measured%2520Physicochemical%2520Data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D10091%26epage%3D10108%26doi%3D10.1021%2Facs.jmedchem.9b01596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit48d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratkova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span> <span> </span><span class="NLM_article-title">Steering New Drug Discovery Campaigns: Permeability, Solubility and Physicochemical Properties in the bRo5 Chemical Space</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00581</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00581" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3MXislWhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=13-23&author=G.+Caronauthor=J.+Kihlbergauthor=G.+Goetzauthor=E.+Ratkovaauthor=V.+Poongavanamauthor=G.+Ermondi&title=Steering+New+Drug+Discovery+Campaigns%3A+Permeability%2C+Solubility+and+Physicochemical+Properties+in+the+bRo5+Chemical+Space&doi=10.1021%2Facsmedchemlett.0c00581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48dR"><div class="casContent"><span class="casTitleNuber">48d</span><div class="casTitle"><span class="NLM_cas:atitle">Steering New Drug Discovery Campaigns: Permeability, Solubility, and Physicochemical Properties in the bRo5 Chemical Space</span></div><div class="casAuthors">Caron, Giulia; Kihlberg, Jan; Goetz, Gilles; Ratkova, Ekaterina; Poongavanam, Vasanthanathan; Ermondi, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-23</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An increasing no. of drug discovery programs concern compds. in the beyond rule of 5 (bRo5) chem. space, such as cyclic peptides, macrocycles, and degraders.  Recent results show that common paradigms of property-based drug design need revision to be applied to larger and more flexible compds.  A virtual event entitled "Soly., permeability and physico-chem. properties in the bRo5 chem. space" was organized to provide preliminary guidance on how to make the discovery of oral drugs in the bRo5 space more effective.  The four speakers emphasized the importance of the bRo5 space as a source of new oral drugs and provided examples of exptl. and computational methods specifically tailored for design and optimization in this chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHq-tvhn7ITrVg90H21EOLACvtfcHk0ljnJ94uD6O5EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXislWhsw%253D%253D&md5=a8d6bd0c3fa5762ed9e4987e78aaf2bf</span></div><a href="/servlet/linkout?suffix=cit48d&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00581%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26aulast%3DGoetz%26aufirst%3DG.%26aulast%3DRatkova%26aufirst%3DE.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DErmondi%26aufirst%3DG.%26atitle%3DSteering%2520New%2520Drug%2520Discovery%2520Campaigns%253A%2520Permeability%252C%2520Solubility%2520and%2520Physicochemical%2520Properties%2520in%2520the%2520bRo5%2520Chemical%2520Space%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D12%26spage%3D13%26epage%3D23%26doi%3D10.1021%2Facsmedchemlett.0c00581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span> </span><span class="NLM_article-title">Dissociation Constants of Organic Acids
and Bases</span>. In  <i>CRC Handbook of Chemistry and
Physics</i>, <span class="NLM_edition">88</span>th ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Lide, D. R.</span></span>, Ed.; <span class="NLM_publisher-name">CRC Press</span>, <span class="NLM_year">2007</span>; pp  <span class="NLM_fpage">8-42</span>– <span class="NLM_lpage">8-51</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=8-42-8-51&author=D.+R.+Lide&title=CRC+Handbook+of+Chemistry+and%0APhysics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26atitle%3DDissociation%2520Constants%2520of%2520Organic%2520Acids%250Aand%2520Bases%26btitle%3DCRC%2520Handbook%2520of%2520Chemistry%2520and%250APhysics%26aulast%3DLide%26aufirst%3DD.%2BR.%26pub%3DCRC%2520Press%26date%3D2007%26spage%3D8-42%26epage%3D8-51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhareshwar, S. S.</span>; <span class="NLM_string-name">Stella, V. J.</span></span> <span> </span><span class="NLM_article-title">Chapter 3.2. Prodrugs of Alcohols and Phenols.</span> In  <i>Prodrugs: Challenges and Rewards Part 1</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stella, V. J.</span>, <span class="NLM_string-name">Borchardt, R. T.</span>, <span class="NLM_string-name">Hageman, M. J.</span>, <span class="NLM_string-name">Oliyai, R.</span>, <span class="NLM_string-name">Maag, H.</span>, <span class="NLM_string-name">Tilley, J. W.</span></span>, Eds.; <span class="NLM_series">Biotechnology: Pharmaceutical Aspects</span>  <span class="NLM_volume">V</span>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2007</span>; pp  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">99</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1007%2F978-0-387-49785-3_21" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=31-99&author=S.+S.+Dhareshwar&author=V.+J.+Stellaauthor=V.+J.+Stella&author=R.+T.+Borchardt&author=M.+J.+Hageman&author=R.+Oliyai&author=H.+Maag&author=J.+W.+Tilley&title=Prodrugs%3A+Challenges+and+Rewards+Part+1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-49785-3_21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-49785-3_21%26sid%3Dliteratum%253Aachs%26aulast%3DDhareshwar%26aufirst%3DS.%2BS.%26atitle%3DChapter%25203.2.%2520Prodrugs%2520of%2520Alcohols%2520and%2520Phenols.%26btitle%3DProdrugs%253A%2520Challenges%2520and%2520Rewards%2520Part%25201%26aulast%3DStella%26aufirst%3DV.%2BJ.%26volume%3DV%26pub%3DSpringer%26date%3D2007%26spage%3D31%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biniossek, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nägler, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker-Pauly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, O.</span></span> <span> </span><span class="NLM_article-title">Proteomic Identification of Protease Cleavage Sites Characterizes Prime and Non-prime Specificity of Cysteine Cathepsin B L and S</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5363</span>– <span class="NLM_lpage">5373</span>, <span class="refDoi"> DOI: 10.1021/pr200621z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr200621z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKgsL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=5363-5373&author=M.+L.+Biniossekauthor=D.+K.+N%C3%A4glerauthor=C.+Becker-Paulyauthor=O.+Schilling&title=Proteomic+Identification+of+Protease+Cleavage+Sites+Characterizes+Prime+and+Non-prime+Specificity+of+Cysteine+Cathepsin+B+L+and+S&doi=10.1021%2Fpr200621z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Identification of Protease Cleavage Sites Characterizes Prime and Non-prime Specificity of Cysteine Cathepsins B, L, and S</span></div><div class="casAuthors">Biniossek, Martin L.; Naegler, Dorit K.; Becker-Pauly, Christoph; Schilling, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5363-5373</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cysteine cathepsins mediate proteome homeostasis and have pivotal functions in diseases such as cancer.  To better understand substrate recognition by cathepsins B, L, and S, we applied proteomic identification of protease cleavage sites (PICS) for simultaneous profiling of prime and non-prime specificity.  PICS profiling of cathepsin B endopeptidase specificity highlights strong selectivity for glycine in P3' due to an occluding loop blocking access to the primed subsites.  In P1', cathepsin B has a partial preference for phenylalanine, which is not found for cathepsins L and S.  Occurrence of P1' phenylalanine often coincides with arom. residues in P2.  For cathepsin L, PICS identifies 845 cleavage sites, representing the most comprehensive PICS profile to date.  Cathepsin L specificity is dominated by the canonical preference for arom. residues in P2 with limited contribution of prime-site selectivity determinants.  Profiling of cathepsins B and L with a shorter incubation time (4 h instead of 16 h) did not reveal time-dependency of individual specificity determinants.  Cathepsin S specificity was profiled at pH 6.0 and 7.5.  The PICS profiles at both pH values display a high degree of similarity.  Cathepsin S specificity is primarily guided by aliph. residues in P2 with limited importance of prime-site residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxwo4mQtEBrrVg90H21EOLACvtfcHk0liwV8zw4HdXig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKgsL7F&md5=4bd86bc70546694933a2975c9b367450</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fpr200621z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr200621z%26sid%3Dliteratum%253Aachs%26aulast%3DBiniossek%26aufirst%3DM.%2BL.%26aulast%3DN%25C3%25A4gler%26aufirst%3DD.%2BK.%26aulast%3DBecker-Pauly%26aufirst%3DC.%26aulast%3DSchilling%26aufirst%3DO.%26atitle%3DProteomic%2520Identification%2520of%2520Protease%2520Cleavage%2520Sites%2520Characterizes%2520Prime%2520and%2520Non-prime%2520Specificity%2520of%2520Cysteine%2520Cathepsin%2520B%2520L%2520and%2520S%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D5363%26epage%3D5373%26doi%3D10.1021%2Fpr200621z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samaha, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elgamal, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weldon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geraghty, R.</span></span> <span> </span><span class="NLM_article-title">Cathepsin S: investigating an old player in lung disease pathogenesis, comorbidities, and potential therapeutics</span>. <i>Respir. Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">111</span>, <span class="refDoi"> DOI: 10.1186/s12931-020-01381-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1186%2Fs12931-020-01381-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=32398133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A280%3ADC%252BB38vlvFCqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&pages=111&author=R.+Brownauthor=S.+Nathauthor=A.+Loraauthor=G.+Samahaauthor=Z.+Elgamalauthor=R.+Kaiserauthor=C.+Taggartauthor=S.+Weldonauthor=R.+Geraghty&title=Cathepsin+S%3A+investigating+an+old+player+in+lung+disease+pathogenesis%2C+comorbidities%2C+and+potential+therapeutics&doi=10.1186%2Fs12931-020-01381-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin S: investigating an old player in lung disease pathogenesis, comorbidities, and potential therapeutics</span></div><div class="casAuthors">Brown Ryan; Taggart Clifford; Weldon Sinead; Nath Sridesh; Lora Alnardo; Samaha Ghassan; Elgamal Ziyad; Kaiser Ryan; Geraghty Patrick; Geraghty Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Respiratory research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Dysregulated expression and activity of cathepsin S (CTSS), a lysosomal protease and a member of the cysteine cathepsin protease family, is linked to the pathogenesis of multiple diseases, including a number of conditions affecting the lungs.  Extracellular CTSS has potent elastase activity and by processing cytokines and host defense proteins, it also plays a role in the regulation of inflammation.  CTSS has also been linked to G-coupled protein receptor activation and possesses an important intracellular role in major histocompatibility complex class II antigen presentation.  Modulated CTSS activity is also associated with pulmonary disease comorbidities, such as cancer, cardiovascular disease, and diabetes.  CTSS is expressed in a wide variety of immune cells and is biologically active at neutral pH.  Herein, we review the significance of CTSS signaling in pulmonary diseases and associated comorbidities.  We also discuss CTSS as a plausible therapeutic target and describe recent and current clinical trials examining CTSS inhibition as a means for treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfcwhyj6BL4yA3UY9EwTNrfW6udTcc2eZKYnq3ijz_e7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38vlvFCqsg%253D%253D&md5=fab651d27425d6c6365a74cb32e24fd2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1186%2Fs12931-020-01381-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12931-020-01381-5%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DR.%26aulast%3DNath%26aufirst%3DS.%26aulast%3DLora%26aufirst%3DA.%26aulast%3DSamaha%26aufirst%3DG.%26aulast%3DElgamal%26aufirst%3DZ.%26aulast%3DKaiser%26aufirst%3DR.%26aulast%3DTaggart%26aufirst%3DC.%26aulast%3DWeldon%26aufirst%3DS.%26aulast%3DGeraghty%26aufirst%3DR.%26atitle%3DCathepsin%2520S%253A%2520investigating%2520an%2520old%2520player%2520in%2520lung%2520disease%2520pathogenesis%252C%2520comorbidities%252C%2520and%2520potential%2520therapeutics%26jtitle%3DRespir.%2520Res.%26date%3D2020%26volume%3D21%26spage%3D111%26doi%3D10.1186%2Fs12931-020-01381-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Insights into the Catalytic Mechanism of Middle East Respiratory Syndrome 3C-Like Protease and Severe Acute Respiratory Syndrome 3C-Like Protease</span>. <i>ACS Catal.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5871</span>– <span class="NLM_lpage">5890</span>, <span class="refDoi"> DOI: 10.1021/acscatal.0c00110</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscatal.0c00110" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BB3cXot1WmsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=5871-5890&author=H.+Wangauthor=S.+Heauthor=W.+Dengauthor=Y.+Zhangauthor=G.+Liauthor=J.+Sunauthor=W.+Zhaoauthor=Y.+Guoauthor=Z.+Yinauthor=D.+Liauthor=L.+Shang&title=Comprehensive+Insights+into+the+Catalytic+Mechanism+of+Middle+East+Respiratory+Syndrome+3C-Like+Protease+and+Severe+Acute+Respiratory+Syndrome+3C-Like+Protease&doi=10.1021%2Facscatal.0c00110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Insights into the Catalytic Mechanism of Middle East Respiratory Syndrome 3C-Like Protease and Severe Acute Respiratory Syndrome 3C-Like Protease</span></div><div class="casAuthors">Wang, Hao; He, Shuai; Deng, Weilong; Zhang, Ying; Li, Guobang; Sun, Jixue; Zhao, Wei; Guo, Yu; Yin, Zheng; Li, Dongmei; Shang, Luqing</div><div class="citationInfo"><span class="NLM_cas:title">ACS Catalysis</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5871-5890</span>CODEN:
                <span class="NLM_cas:coden">ACCACS</span>;
        ISSN:<span class="NLM_cas:issn">2155-5435</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Coronavirus 3C-like protease (3CLPro) is a highly conserved cysteine protease employing a catalytic dyad for its functions.  3CLPro is essential to the viral life cycle and, therefore, is an attractive target for developing antiviral agents.  However, the detailed catalytic mechanism of coronavirus 3CLPro remains largely unknown.  We took an integrated approach of employing X-ray crystallog., mutational studies, enzyme kinetics study and inhibitors to gain insights into the mechanism.  Such exptl. work is supplemented by computational studies include pre-reaction state anal. and ab initio calcn. of the crit. catalytic step, and mol. dynamic simulation of the wild-type and mutant enzymes.  Combined, such studies allowed us to identify a residue pair (Glu-His) and a conserved His as crit. for binding; a conserved GSCGS motif as important for the start of catalysis, a partial neg. charge cluster (PNCC) formed by Arg-Tyr-Asp as essential for catalysis, and a conserved water mol. mediating the remote interaction between PNCC and catalytic dyad.  The data collected and our new insights into the detailed mechanism have allowed us to achieve a good understanding of the difference in catalytic efficiency between 3CLPro from SARS and MERS, conduct mutational studies to improve catalytic activity by 8-fold, optimize existing inhibitors to improve potency by 4-fold, and identify a potential allosteric site for inhibitor design.  All such results reinforce each other to support the overall catalytic mechanism proposed herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTVgu_o7NTWrVg90H21EOLACvtfcHk0liv8exCUTtcOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXot1WmsL0%253D&md5=49037798a0e22f691f72be4132102c3f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facscatal.0c00110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscatal.0c00110%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DShang%26aufirst%3DL.%26atitle%3DComprehensive%2520Insights%2520into%2520the%2520Catalytic%2520Mechanism%2520of%2520Middle%2520East%2520Respiratory%2520Syndrome%25203C-Like%2520Protease%2520and%2520Severe%2520Acute%2520Respiratory%2520Syndrome%25203C-Like%2520Protease%26jtitle%3DACS%2520Catal.%26date%3D2020%26volume%3D10%26spage%3D5871%26epage%3D5890%26doi%3D10.1021%2Facscatal.0c00110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span>; <span class="NLM_string-name">Jiang, S.</span>; <span class="NLM_string-name">Dai, W.</span>; <span class="NLM_string-name">Lu, L.</span>; <span class="NLM_string-name">Peng, J.</span>; <span class="NLM_string-name">Xia, S.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Li, J.</span>; <span class="NLM_string-name">Jiang, J.</span>; <span class="NLM_string-name">Chen, K.</span></span> <span> </span><span class="NLM_article-title">Ketoamide Compound and Preparation Method, Pharmaceutical Composition and Use Thereof</span>. <span class="NLM_patent">WO2020/030143</span>, September 8, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=H.+Liu&author=S.+Jiang&author=W.+Dai&author=L.+Lu&author=J.+Peng&author=S.+Xia&author=J.+Wang&author=J.+Li&author=J.+Jiang&author=K.+Chen&title=Ketoamide+Compound+and+Preparation+Method%2C+Pharmaceutical+Composition+and+Use+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DKetoamide%2520Compound%2520and%2520Preparation%2520Method%252C%2520Pharmaceutical%2520Composition%2520and%2520Use%2520Thereof%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phizackerley, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, P.</span></span> <span> </span><span class="NLM_article-title">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1107/S0909049502015170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1107%2FS0909049502015170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=12409628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=401-406&author=T.+M.+McPhillipsauthor=S.+E.+McPhillipsauthor=H.+J.+Chiuauthor=A.+E.+Cohenauthor=A.+M.+Deaconauthor=P.+J.+Ellisauthor=E.+Garmanauthor=A.+Gonzalezauthor=N.+K.+Sauterauthor=R.+P.+Phizackerleyauthor=S.+M.+Soltisauthor=P.+Kuhn&title=Blu-Ice+and+the+Distributed+Control+System%3A+software+for+data+acquisition+and+instrument+control+at+macromolecular+crystallography+beamlines&doi=10.1107%2FS0909049502015170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span></div><div class="casAuthors">McPhillips, Timothy M.; McPhillips, Scott E.; Chiu, Hsiu Ju; Cohen, Aina E.; Deacon, Ashley M.; Ellis, Paul J.; Garman, Elspeth; Gonzalez, Ana; Sauter, Nicholas K.; Phizackerley, R. Paul; Soltis, S. Michael; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-406</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Blu-Ice and Distributed Control System (DCS) software packages were developed to provide unified control over the disparate hardware resources available at a macromol. crystallog. beamline.  Blu-Ice is a user interface that provides scientific experimenters and beamline support staff with intuitive graphical tools for collecting diffraction data and configuring beamlines for expts.  Blu-Ice communicates with the hardware at a beamline via DCS, an instrument-control and data-acquisition package designed to integrate hardware resources in a highly heterogeneous networked computing environment.  Together, Blu-Ice and DCS provide a flexible platform for increasing the ease of use, the level of automation and the remote accessibility of beamlines.  Blu-Ice and DCS are currently installed on four Stanford Synchrotron Radiation Lab. crystallog. beamlines and are being implemented at sister light sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVsQUIFJPBLVg90H21EOLACvtfcHk0liv8exCUTtcOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D&md5=c77228bd50890463f3e43e981a2e238a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0909049502015170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049502015170%26sid%3Dliteratum%253Aachs%26aulast%3DMcPhillips%26aufirst%3DT.%2BM.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DChiu%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DGarman%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DBlu-Ice%2520and%2520the%2520Distributed%2520Control%2520System%253A%2520software%2520for%2520data%2520acquisition%2520and%2520instrument%2520control%2520at%2520macromolecular%2520crystallography%2520beamlines%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2002%26volume%3D9%26spage%3D401%26epage%3D406%26doi%3D10.1107%2FS0909049502015170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moorhead, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhillips, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltis, S. M.</span></span> <span> </span><span class="NLM_article-title">Web-Ice: Integrated Data Collection and Analysis for Macromolecular Crystallography</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1107/S0021889807057822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1107%2FS0021889807057822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmvFyntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=176-184&author=A.+Gonzalezauthor=P.+Moorheadauthor=S.+E.+McPhillipsauthor=J.+Songauthor=K.+Sharpauthor=J.+R.+Taylorauthor=P.+D.+Adamsauthor=N.+K.+Sauterauthor=S.+M.+Soltis&title=Web-Ice%3A+Integrated+Data+Collection+and+Analysis+for+Macromolecular+Crystallography&doi=10.1107%2FS0021889807057822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Web-Ice: integrated data collection and analysis for macromolecular crystallography</span></div><div class="casAuthors">Gonzalez, A.; Moorhead, P.; McPhillips, S. E.; Song, J.; Sharp, K.; Taylor, J. R.; Adams, P. D.; Sauter, N. K.; Soltis, S. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">176-184</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">New software tools are introduced to facilitate diffraction expts. involving large nos. of crystals.  While existing programs have long provided a framework for lattice indexing, Bragg spot integration, and symmetry detn., these initial data processing steps often require significant manual effort.  This limits the timely availability of data anal. needed for high-throughput procedures, including the selection of the best crystals from a large sample pool, and the calcn. of optimal data collection parameters to assure complete spot coverage with minimal radiation damage.  To make these protocols more efficient, a network of software applications and application servers has been developed, collectively known as Web-Ice.  When the package is installed at a crystallog. beamline, a programming interface allows the beamline control software (e.g. Blu-Ice, DCSS) to trigger data anal. automatically.  Results are organized based on a list of samples that the user provides, and are examd. within a Web page, accessible both locally at the beamline and remotely.  Optional programming interfaces permit the user to control data acquisition through the Web browser.  The system as a whole is implemented to support multiple users and multiple processors, and can be expanded to provide addnl. scientific functionality.  Web-Ice has a distributed architecture consisting of several stand-alone software components working together via a well defined interface.  Other synchrotrons or institutions may integrate selected components or the whole of Web-Ice with their own data acquisition software.  Updated information about current developments may be obtained at http://smb.slac.stanford.edu/research/developments/webice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCqAzIEM32NbVg90H21EOLACvtfcHk0liv8exCUTtcOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmvFyntA%253D%253D&md5=f34468ba2a5441684b9e6f220b4b372e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0021889807057822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807057822%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DMoorhead%26aufirst%3DP.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DSharp%26aufirst%3DK.%26aulast%3DTaylor%26aufirst%3DJ.%2BR.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26atitle%3DWeb-Ice%253A%2520Integrated%2520Data%2520Collection%2520and%2520Analysis%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2008%26volume%3D41%26spage%3D176%26epage%3D184%26doi%3D10.1107%2FS0021889807057822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wardrop, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velter, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forslund, R. E.</span></span> <span> </span><span class="NLM_article-title">Template-Directed C–H Insertion: Synthesis of the Dioxabicyclo[3.2.1]octane Core of the Zaragozic Acids</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">2261</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1021/ol0158361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0158361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1Wit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2001&pages=2261-2264&issue=15&author=D.+J.+Wardropauthor=A.+I.+Velterauthor=R.+E.+Forslund&title=Template-Directed+C%E2%80%93H+Insertion%3A+Synthesis+of+the+Dioxabicyclo%5B3.2.1%5Doctane+Core+of+the+Zaragozic+Acids&doi=10.1021%2Fol0158361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Template-Directed C-H Insertion: Synthesis of the Dioxabicyclo[3.2.1]octane Core of the Zaragozic Acids</span></div><div class="casAuthors">Wardrop, Duncan J.; Velter, Adriana I.; Forslund, Raymond E.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2261-2264</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The prepn. of (±)-(1α,3β,4β,5αβ,6β,7α)-4-benzyloxy-3,5-bis(benzyloxymethyl)-6-hydroxy-1-methyl-7-(triisopropylsilanyloxy)-2,8-dioxabicyclo[3.2.1]octan-6-ol, a model for the core of the zaragozic acids, is reported.  The pivotal reaction in this endeavor is the dirhodium(II)-catalyzed intramol. C-H bond insertion of 2-diazoacetyl-5-benzyloxy-4,6-bis(benzyloxymethyl)-2-methyl-1,3-dioxane (I), a transformation which generates four of the six stereocenters present in the core structure.  A novel method for the diastereoselective synthesis of pyruvic acid acetals was also developed and employed in the prepn. of I from a xylitol deriv.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpipb725ubcpLVg90H21EOLACvtfcHk0lhLLumrb5bUeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1Wit7c%253D&md5=9b79ff69f8848c58417db38e9dffcdc0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fol0158361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0158361%26sid%3Dliteratum%253Aachs%26aulast%3DWardrop%26aufirst%3DD.%2BJ.%26aulast%3DVelter%26aufirst%3DA.%2BI.%26aulast%3DForslund%26aufirst%3DR.%2BE.%26atitle%3DTemplate-Directed%2520C%25E2%2580%2593H%2520Insertion%253A%2520Synthesis%2520of%2520the%2520Dioxabicyclo%255B3.2.1%255Doctane%2520Core%2520of%2520the%2520Zaragozic%2520Acids%26jtitle%3DOrg.%2520Lett.%26date%3D2001%26volume%3D3%26issue%3D15%26spage%3D2261%26epage%3D2264%26doi%3D10.1021%2Fol0158361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katsuyama, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funabiki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muramatsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, K.</span></span> <span> </span><span class="NLM_article-title">A Convenient and Regioselective Synthesis of 4-Trifluoromethylpyridines</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>1997</i></span>,  <span class="NLM_fpage">1321</span>– <span class="NLM_lpage">1324</span>, <span class="refDoi"> DOI: 10.1055/s-1997-1354</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1055%2Fs-1997-1354" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1997&publication_year=1997&pages=1321-1324&author=I.+Katsuyamaauthor=S.+Ogawaauthor=Y.+Yamaguchiauthor=K.+Funabikiauthor=M.+Matsuiauthor=H.+Muramatsuauthor=K.+Shibata&title=A+Convenient+and+Regioselective+Synthesis+of+4-Trifluoromethylpyridines&doi=10.1055%2Fs-1997-1354"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1055%2Fs-1997-1354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-1997-1354%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuyama%26aufirst%3DI.%26aulast%3DOgawa%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DFunabiki%26aufirst%3DK.%26aulast%3DMatsui%26aufirst%3DM.%26aulast%3DMuramatsu%26aufirst%3DH.%26aulast%3DShibata%26aufirst%3DK.%26atitle%3DA%2520Convenient%2520and%2520Regioselective%2520Synthesis%2520of%25204-Trifluoromethylpyridines%26jtitle%3DSynthesis%26date%3D1997%26volume%3D1997%26spage%3D1321%26epage%3D1324%26doi%3D10.1055%2Fs-1997-1354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, K.</span>; <span class="NLM_string-name">Tadaaki, O.</span>; <span class="NLM_string-name">Toru, M.</span></span> <span> </span><span class="NLM_article-title">Method for Producing Cyclic Diamine Derivative or Salt Thereof</span>. <span class="NLM_patent">WO2004/106323</span> (<span class="NLM_patent">EP1627875</span>), May 28, <span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=K.+Shibuya&author=O.+Tadaaki&author=M.+Toru&title=Method+for+Producing+Cyclic+Diamine+Derivative+or+Salt+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DShibuya%26aufirst%3DK.%26atitle%3DMethod%2520for%2520Producing%2520Cyclic%2520Diamine%2520Derivative%2520or%2520Salt%2520Thereof%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albert, A.</span>, <span class="NLM_string-name">Serjeant, E. P.</span></span> <i>The Determination of Ionization Constants. A Laboratory Manual</i>, <span class="NLM_edition">3</span>rd ed.; <span class="NLM_publisher-name">Chapman and Hall</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1984</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;key=10.1007%2F978-94-009-5548-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1984&author=A.+Albert&author=E.+P.+Serjeant&title=The+Determination+of+Ionization+Constants.+A+Laboratory+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1007%2F978-94-009-5548-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-009-5548-6%26sid%3Dliteratum%253Aachs%26aulast%3DAlbert%26aufirst%3DA.%26btitle%3DThe%2520Determination%2520of%2520Ionization%2520Constants.%2520A%2520Laboratory%2520Manual%26pub%3DChapman%2520and%2520Hall%26date%3D1984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7MBI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7MBI','PDB','7MBI'); return false;">PDB: 7MBI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTM','PDB','6WTM'); return false;">PDB: 6WTM</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i55"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00616">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_83174"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00616?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00616</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Glutathione stability studies (Table S1); protease, antiviral, and cytotoxicity graphs for <b>15l</b>; slopes of <i>k</i><sub>obs</sub> versus [I] for <b>15a</b>, <b>15e</b>, <b>15f</b>, <b>15h</b>, and <b>15l</b> (Table S2 and Charts S1 and S2); electron density for SARS-CoV-2 3CLP – <b>15l</b> active sites (Figure S1); SARS-CoV-2 3CLP – <b>15l</b> ligand plot (Figure S2) and crystallographic data (Table S3); and <sup>1</sup>H NMR, <sup>19</sup>F NMR, and HPLC for select compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds with biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_001.pdf">jm1c00616_si_001.pdf (7.42 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00616/suppl_file/jm1c00616_si_002.csv">jm1c00616_si_002.csv (2.25 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00616&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00616%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00616" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b273d93c3ce2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
